<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/9/10/3395" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/9/10/3395.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/9/10/3395.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/9/10/3395.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/9/10/3395/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/9/10/3395.atom" />
<meta name="HW.pisa" content="ggg;9/10/3395" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Evolutionary Dynamics in Structured Populations Under Strong Population Genetic Forces" />
<meta name="DC.Identifier" content="10.1534/g3.119.400605" />
<meta name="DC.Date" content="2019-10-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2019 Feder et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="In the long-term neutral equilibrium, high rates of migration between subpopulations result in little population differentiation. However, in the short-term, even very abundant migration may not be enough for subpopulations to equilibrate immediately. In this study, we investigate dynamical patterns of short-term population differentiation in adapting populations via stochastic and analytical modeling through time. We characterize a regime in which selection and migration interact to create non-monotonic patterns of population differentiation over time when migration is weaker than selection, but stronger than drift. We demonstrate how these patterns can be leveraged to estimate high migration rates using approximate Bayesian computation. We apply this approach to estimate fast migration in a rapidly adapting intra-host Simian-HIV population sampled from different anatomical locations. We find differences in estimated migration rates between different compartments, even though all are above ![Formula][1]&lt;/img&gt; = 1. This work demonstrates how studying demographic processes on the timescale of selective sweeps illuminates processes too fast to leave signatures on neutral timescales.

 [1]: /embed/mml-math-1.gif" />
<meta name="DC.Contributor" content="Alison F. Feder" />
<meta name="DC.Contributor" content="Pleuni S. Pennings" />
<meta name="DC.Contributor" content="Joachim Hermisson" />
<meta name="DC.Contributor" content="Dmitri A. Petrov" />
<meta name="article:published_time" content="2019-10-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Evolutionary Dynamics in Structured Populations Under Strong Population Genetic Forces" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;In the long-term neutral equilibrium, high rates of migration between subpopulations result in little population differentiation. However, in the short-term, even very abundant migration may not be enough for subpopulations to equilibrate immediately. In this study, we investigate dynamical patterns of short-term population differentiation in adapting populations via stochastic and analytical modeling through time. We characterize a regime in which selection and migration interact to create non-monotonic patterns of population differentiation over time when migration is weaker than selection, but stronger than drift. We demonstrate how these patterns can be leveraged to estimate high migration rates using approximate Bayesian computation. We apply this approach to estimate fast migration in a rapidly adapting intra-host Simian-HIV population sampled from different anatomical locations. We find differences in estimated migration rates between different compartments, even though all are above &lt;math xmlns=&quot;http://www.w3.org/1998/Math/MathML&quot; id=&quot;me1&quot; overflow=&quot;scroll&quot;&gt;&lt;mrow&gt;&lt;msub&gt;&lt;mi&gt;N&lt;/mi&gt;&lt;mi&gt;e&lt;/mi&gt;&lt;/msub&gt;&lt;mi&gt;m&lt;/mi&gt;&lt;/mrow&gt;&lt;/math&gt; = 1. This work demonstrates how studying demographic processes on the timescale of selective sweeps illuminates processes too fast to leave signatures on neutral timescales.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2019/10/01" />
<meta name="citation_mjid" content="ggg;9/10/3395" />
<meta name="citation_id" content="9/10/3395" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/9/10/3395" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/9/10/3395.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/9/10/3395.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/9/10/3395.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.119.400605" />
<meta name="citation_pmid" content="31462443" />
<meta name="citation_volume" content="9" />
<meta name="citation_issue" content="10" />
<meta name="citation_num_pages" content="13" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="3395" />
<meta name="citation_lastpage" content="3407" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Alison F. Feder" />
<meta name="citation_author_institution" content="Department of Biology, Stanford University," />
<meta name="citation_author_institution" content="Department of Integrative Biology, University of California Berkeley," />
<meta name="citation_author_email" content="affeder@berkeley.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-2915-089X" />
<meta name="citation_author" content="Pleuni S. Pennings" />
<meta name="citation_author_institution" content="Department of Biology, San Francisco State University, and" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-8704-6578" />
<meta name="citation_author" content="Joachim Hermisson" />
<meta name="citation_author_institution" content="Faculty of Mathematics, University of Vienna" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-7476-9283" />
<meta name="citation_author" content="Dmitri A. Petrov" />
<meta name="citation_author_institution" content="Department of Biology, Stanford University," />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-3664-9130" />
<meta name="citation_reference" content="citation_journal_title=J. Virol.;citation_author=M. E. Abram;citation_author=A. L. Ferris;citation_author=W. Shao;citation_author=W. G. Alvord;citation_author=S. H. Hughes;citation_title=Nature, position, and frequency of mutations made in a single cycle of HIV-1 replication.;citation_volume=84;citation_year=2010;citation_pmid=20660205;citation_doi=10.1128/JVI.00915-10" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. Aeschbacher;citation_author=M. A. Beaumont;citation_author=A. Futschik;citation_title=A novel approach for choosing summary statistics in approximate Bayesian computation.;citation_volume=192;citation_year=2012;citation_pmid=22960215;citation_doi=10.1534/genetics.112.143164" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_author=B. E. Allman;citation_author=D. B. Weissman;citation_title=Hitchhiking in space: Ancestry in adapting, spatially extended populations.;citation_pages=722-734;citation_volume=72;citation_year=2018;citation_doi=10.1111/evo.13431" />
<meta name="citation_reference" content="citation_journal_title=Genet. Res.;citation_author=N. H. Barton;citation_title=The probability of fixation of a favoured allele in a subdivided population.;citation_pages=149-157;citation_volume=62;citation_year=1993;citation_doi=10.1017/S0016672300031748" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. A. Beaumont;citation_author=W. Zhang;citation_author=D. J. Balding;citation_title=Approximate Bayesian computation in population genetics.;citation_pages=2025-2035;citation_volume=162;citation_year=2002;citation_pmid=12524368" />
<meta name="citation_reference" content="citation_journal_title=Elife;citation_author=J. J. Berg;citation_author=A. Harpak;citation_author=N. Sinnott-Armstrong;citation_author=A. M. Joergensen;citation_author=H. Mostafavi;citation_title=Reduced signal for polygenic adaptation of height in uk biobank.;citation_pages=e39725;citation_volume=8;citation_year=2019;citation_doi=10.7554/eLife.39725" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_journal_abbrev=Evolution;citation_author=N. Bierne;citation_title=The distinctive footprints of local hitchhiking in a varied environment and global hitchhiking in a subdivided population.;citation_pages=3254-3272;citation_volume=64;citation_year=2010;citation_issue=11;citation_pmid=20550573;citation_doi=10.1111/j.1558-5646.2010.01050.x" />
<meta name="citation_reference" content="citation_journal_title=PLOS Comput. Biol.;citation_author=A.-F. Bitbol;citation_author=D. J. Schwab;citation_title=Quantifying the role of population subdivision in evolution on rugged fitness landscapes.;citation_pages=e1003778;citation_volume=10;citation_year=2014;citation_pmid=25122220;citation_doi=10.1371/journal.pcbi.1003778" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. P. Bollback;citation_author=T. L. York;citation_author=R. Nielsen;citation_title=Estimation of 2Nes From Temporal Allele Frequency Data;citation_pages=497-502;citation_volume=179;citation_year=2008;citation_issue=1;citation_pmid=18493066;citation_doi=10.1534/genetics.107.085019" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=G. S. Bradburd;citation_author=P. L. Ralph;citation_author=G. M. Coop;citation_title=A spatial framework for understanding population structure and admixture.;citation_pages=e1005703;citation_volume=12;citation_year=2016;citation_pmid=26771578;citation_doi=10.1371/journal.pgen.1005703" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=J. A. Coyne;citation_title=Genetics and speciation.;citation_pages=511-515;citation_volume=355;citation_year=1992;citation_issue=6360;citation_pmid=1741030;citation_doi=10.1038/355511a0" />
<meta name="citation_reference" content="citation_journal_title=Methods Ecol. Evol.;citation_author=K. Csilléry;citation_author=O. Francois;citation_author=M. G. B. Blum;citation_title=abc: an R package for approximate Bayesian computation (ABC).;citation_pages=475-479;citation_volume=3;citation_year=2012;citation_doi=10.1111/j.2041-210X.2011.00179.x" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. M. Desai;citation_author=D. S. Fisher;citation_title=Beneficial mutation–selection balance and the effect of linkage on positive selection.;citation_volume=176;citation_year=2007;citation_pmid=17483432;citation_doi=10.1534/genetics.106.067678" />
<meta name="citation_reference" content="citation_journal_title=Evol. Bioinform. Online;citation_author=G. Ewing;citation_author=A. Rodrigo;citation_title=Estimating population parameters using the structured serial coalescent with bayesian mcmc inference when some demes are hidden.;citation_pages=227-235;citation_volume=2;citation_year=2007;citation_doi=10.1177/117693430600200026" />
<meta name="citation_reference" content="citation_journal_title=PLoS Pathog.;citation_author=A. F. Feder;citation_author=C. Kline;citation_author=P. Polacino;citation_author=M. Cottrell;citation_author=A. D. Kashuba;citation_title=A spatio-temporal assessment of simian/human immunodeficiency virus (SHIV) evolution reveals a highly dynamic process within the host.;citation_pages=e1006358;citation_volume=13;citation_year=2017;citation_doi=10.1371/journal.ppat.1006358" />
<meta name="citation_reference" content="citation_title=Genetics;citation_author=A. F. Feder;citation_author=S. Kryazhimskiy;citation_author=J. B. Plotkin;citation_title=Identifying signatures of selection in genetic time series.;citation_volume=196;citation_year=2014;citation_pmid=24318534;citation_doi=10.1534/genetics.113.158220" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=C. Hauert;citation_author=M. Doebeli;citation_title=Spatial structure often inhibits the evolution of cooperation in the snowdrift game.;citation_pages=643-646;citation_volume=428;citation_year=2004;citation_issue=6983;citation_pmid=15074318;citation_doi=10.1038/nature02360" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. Hermisson;citation_author=P. S. Pennings;citation_title=Soft sweeps: molecular population genetics of adaptation from standing genetic variation.;citation_volume=169;citation_year=2005;citation_pmid=15716498;citation_doi=10.1534/genetics.104.036947" />
<meta name="citation_reference" content="citation_journal_title=Methods Ecol. Evol.;citation_author=J. Hermisson;citation_author=P. S. Pennings;citation_title=Soft sweeps and beyond: understanding the patterns and probabilities of selection footprints under rapid adaptation.;citation_pages=700-716;citation_volume=8;citation_year=2017;citation_doi=10.1111/2041-210X.12808" />
<meta name="citation_reference" content="citation_title=Genetics;citation_author=A. Iranmehr;citation_author=A. Akbari;citation_author=C. Schlötterer;citation_author=V. Bafna;citation_title=Clear: composition of likelihoods for evolve and resequence experiments.;citation_volume=206;citation_year=2017;citation_pmid=28396506;citation_doi=10.1534/genetics.116.197566" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. Jakobsson;citation_author=M. D. Edge;citation_author=N. A. Rosenberg;citation_title=The relationship between FST and the frequency of the most frequent allele.;citation_volume=193;citation_year=2013;citation_pmid=23172852;citation_doi=10.1534/genetics.112.144758" />
<meta name="citation_reference" content="citation_title=Population genetics and ecology;citation_title=Population subdivision and selection migration interaction;citation_pages=617-657;citation_year=1976" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=Y. Kim;citation_author=T. Maruki;citation_title=Hitchhiking effect of a beneficial mutation spreading in a subdivided population.;citation_volume=189;citation_year=2011;citation_pmid=21705748;citation_doi=10.1534/genetics.111.130203" />
<meta name="citation_reference" content="citation_title=Genetics;citation_author=K. M. Lee;citation_author=G. Coop;citation_title=Distinguishing among modes of convergent adaptation using population genomic data.;citation_volume=207;citation_year=2017;citation_pmid=29046403;citation_doi=10.1534/genetics.117.300417" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=E. A. Martens;citation_author=O. Hallatschek;citation_title=Interfering waves of adaptation promote spatial mixing.;citation_volume=189;citation_year=2011;citation_pmid=21900264;citation_doi=10.1534/genetics.111.130112" />
<meta name="citation_reference" content="citation_journal_title=New J. Phys.;citation_author=E. A. Martens;citation_author=R. Kostadinov;citation_author=C. C. Maley;citation_author=O. Hallatschek;citation_title=Spatial structure increases the waiting time for cancer.;citation_pages=115014;citation_volume=13;citation_year=2011;citation_pmid=22707911;citation_doi=10.1088/1367-2630/13/11/115014" />
<meta name="citation_reference" content="citation_journal_title=bioRxiv;citation_title=Fitness-valley crossing in subdivided asexual populations.;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=J. Novembre;citation_author=T. Johnson;citation_author=K. Bryc;citation_author=Z. Kutalik;citation_author=A. R. Boyko;citation_title=Genes mirror geography within Europe.;citation_pages=98-101;citation_volume=456;citation_year=2008;citation_issue=7218;citation_pmid=18758442;citation_doi=10.1038/nature07331" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=J. Paulose;citation_author=J. Hermisson;citation_author=O. Hallatschek;citation_title=Spatial soft sweeps: patterns of adaptation in populations with long-range dispersal.;citation_pages=e1007936;citation_volume=15;citation_year=2019;citation_doi=10.1371/journal.pgen.1007936" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=P. S. Pennings;citation_author=J. Hermisson;citation_title=Soft sweeps II—molecular population genetics of adaptation from recurrent mutation or migration.;citation_volume=23;citation_year=2006;citation_pmid=16520336;citation_doi=10.1093/molbev/msj117" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=P. Ralph;citation_author=G. Coop;citation_title=Parallel adaptation: one or many waves of advance of an advantageous allele?;citation_volume=186;citation_year=2010;citation_pmid=20660645;citation_doi=10.1534/genetics.110.119594" />
<meta name="citation_reference" content="citation_journal_title=Ann. Stat.;citation_author=D. B. Rubin;citation_title=Bayesianly justifiable and relevant frequency calculations for the applied statistician.;citation_pages=1151-1172;citation_volume=12;citation_year=1984;citation_doi=10.1214/aos/1176346785" />
<meta name="citation_reference" content="citation_title=Genetics;citation_author=J. G. Schraiber;citation_author=S. N. Evans;citation_author=M. Slatkin;citation_title=Bayesian inference of natural selection from allele frequency time series.;citation_volume=203;citation_year=2016;citation_pmid=27010022;citation_doi=10.1534/genetics.116.187278" />
<meta name="citation_reference" content="citation_journal_title=PLOS Comput. Biol.;citation_author=S. Sheehan;citation_author=Y. S. Song;citation_title=Deep learning for population genetic inference.;citation_pages=e1004845;citation_volume=12;citation_year=2016;citation_doi=10.1371/journal.pcbi.1004845" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=M. Slatkin;citation_author=T. Wiehe;citation_title=Genetic hitch-hiking in a subdivided population.;citation_pages=155-160;citation_volume=71;citation_year=1998;citation_issue=2;citation_pmid=9717437;citation_doi=10.1017/S001667239800319X" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=J. Terhorst;citation_author=C. Schlötterer;citation_author=Y. S. Song;citation_title=Multi-locus analysis of genomic time series data from experimental evolution.;citation_pages=e1005069;citation_volume=11;citation_year=2015;citation_pmid=25849855;citation_doi=10.1371/journal.pgen.1005069" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=K. Theys;citation_author=A. F. Feder;citation_author=M. Gelbart;citation_author=M. Hartl;citation_author=A. Stern;citation_title=Within-patient mutation frequencies reveal fitness costs of CpG dinucleotides and drastic amino acid changes in HIV.;citation_pages=e1007420;citation_volume=14;citation_year=2018;citation_doi=10.1371/journal.pgen.1007420" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. C. Whitlock;citation_title=Fixation probability and time in subdivided populations.;citation_pages=767-779;citation_volume=164;citation_year=2003;citation_pmid=12807795" />
<meta name="citation_reference" content="citation_year=1999" />
<meta name="citation_reference" content="citation_year=2019" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=B. A. Wilson;citation_author=P. S. Pennings;citation_author=D. A. Petrov;citation_title=Soft selective sweeps in evolutionary rescue.;citation_volume=205;citation_year=2017;citation_pmid=28213477;citation_doi=10.1534/genetics.116.191478" />
<meta name="citation_reference" content="citation_journal_title=Ann. Eugen.;citation_author=S. Wright;citation_title=The genetical structure of populations.;citation_pages=323-354;citation_volume=15;citation_year=1949;citation_pmid=24540312;citation_doi=10.1111/j.1469-1809.1949.tb02451.x" />
<meta name="citation_reference" content="citation_journal_title=Annual review of genetics;citation_journal_abbrev=Annu Rev Genet;citation_author=S. Wright;citation_title=The shifting balance theory and macroevolution.;citation_pages=1-20;citation_volume=16;citation_year=1982;citation_pmid=6760797;citation_doi=10.1146/annurev.ge.16.120182.000245" />
<meta name="citation_reference" content="citation_journal_title=Virus Evol.;citation_author=F. Zanini;citation_author=V. Puller;citation_author=J. Brodin;citation_author=J. Albert;citation_author=R. A. Neher;citation_title=In vivo mutation rates and the landscape of fitness costs of hiv-1.;citation_pages=vex003;citation_volume=3;citation_year=2017;citation_doi=10.1093/ve/vex003" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Evolutionary Dynamics in Structured Populations Under Strong Population Genetic Forces" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10.cover-source.jpg" />
<meta name="twitter:description" content="In the long-term neutral equilibrium, high rates of migration between subpopulations result in little population differentiation. However, in the short-term, even very abundant migration may not be enough for subpopulations to equilibrate immediately. In this study, we investigate dynamical patterns of short-term population differentiation in adapting populations via stochastic and analytical modeling through time. We characterize a regime in which selection and migration interact to create non-monotonic patterns of population differentiation over time when migration is weaker than selection, but stronger than drift. We demonstrate how these patterns can be leveraged to estimate high migration rates using approximate Bayesian computation. We apply this approach to estimate fast migration in a rapidly adapting intra-host Simian-HIV population sampled from different anatomical locations. We find differences in estimated migration rates between different compartments, even though all are above ![Formula][1]&lt;/img&gt; = 1. This work demonstrates how studying demographic processes on the timescale of selective sweeps illuminates processes too fast to leave signatures on neutral timescales.

 [1]: /embed/mml-math-1.gif" />
<meta name="og-title" property="og:title" content="Evolutionary Dynamics in Structured Populations Under Strong Population Genetic Forces" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/9/10/3395" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="In the long-term neutral equilibrium, high rates of migration between subpopulations result in little population differentiation. However, in the short-term, even very abundant migration may not be enough for subpopulations to equilibrate immediately. In this study, we investigate dynamical patterns of short-term population differentiation in adapting populations via stochastic and analytical modeling through time. We characterize a regime in which selection and migration interact to create non-monotonic patterns of population differentiation over time when migration is weaker than selection, but stronger than drift. We demonstrate how these patterns can be leveraged to estimate high migration rates using approximate Bayesian computation. We apply this approach to estimate fast migration in a rapidly adapting intra-host Simian-HIV population sampled from different anatomical locations. We find differences in estimated migration rates between different compartments, even though all are above ![Formula][1]&lt;/img&gt; = 1. This work demonstrates how studying demographic processes on the timescale of selective sweeps illuminates processes too fast to leave signatures on neutral timescales.

 [1]: /embed/mml-math-1.gif" />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10.cover-source.jpg" />
    <title>Evolutionary Dynamics in Structured Populations Under Strong Population Genetic Forces | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"GQk4eRiPy1Vf1X1bmUWG0HDL2ol7E-RcHKtwh7e2LLY","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"494279","apath":"\/ggg\/9\/10\/3395.atom","pisa":"ggg;9\/10\/3395","ac":{"\/ggg\/9\/10\/3395.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/9\/10\/3395.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;9\/10\/3395"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;9\/10\/3395"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/9\/10\/3395":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022WnE7VFJ5lZ7+q0bpDjKbQZacPEvhXnnIAMqqyQZGR8czUVRgYRAMsd2n7e\\\\\\\/uvoxgFCzCdFonULOudqne+QMOeBtRNpSsV2m6rDvOEClE5ZYAAnPFwzwLBgYutF4Bw1Vd0CsOaW1y5f\\\\\\\/gcLoXzmOMUzI+bqljJyrOgULCZNFg9\\\\\\\/UeKpFm5yQ9lAZ5nJDpmq4x4rBJzpe0x3q3i7DboGLH5BJI1dFc0AIGpy9o48CHybjKOWnd82Qr97x50bz9cU9G7idybHAQ\\\\\\\/1FT76Z2LVjsOSZfZbYOIZpwe9+dPy1yQAJfCDkntNCZ2WzDNNuzsBDhxpq\\\\\\\/UB7x39S45LTf7dawoceCCGABPIkkhS7+fabkVRjQn27W5HVA\\\\\\\/LGMngkpru1RSRuU+vz9sCvsi95q6L91W7ihdRX9uh\\\\\\\/oJAM10JavH6nR\\\\\\\/uHz5XqSxZu3lXKxQZmjUA2UXnZ7pS3i9Uv+3PiPdt64DDgBmlndBZArH2utcExWeQK66+qXkaeTj11eeDe\\\\\\\/Q70+Rl81NgWjoZUXwXJqCrjJXSpv2wffiisgkm9V5yE1y+O+99jm4tJ0JD\\\\\\\/YLhKPApIr2FWyAjExn\\\\\\\/Otepokyy4gSiI8PqlqBLRMd8FA+IE\\\\\\\/l6mZ\\\\\\\/sg9JC4PdY\\\\\\\/JUGxPXrvJXSIyXEQsSTrjvuq9tOSiblCghbbq3ODa4683UUw+DwtG77B9qV1pbLfYcL+HOTOBP8r\\\\\\\/A\\\\\\\/IIQ8u+QxrTOIh2z3VsQx0Z5PjZX769Nii1jPsitsVLslihkQLb+nJ45Dr9HCPSVFf0gFuyyktZUwMGgmaPjWrQcjE3ZscdYMhxcPM3DrTQKnyo8nFPsuCCSCy1VTyUvFmN4CtFLnAgfyPWWcIe4KYJRrU\\\\\\\/PAUY236o\\\\\\\/rNbC4R+zN2kIET1liVJ7+22PwVrw\\\\\\\/h+EJP54nouBtKN9BCprPJK7zMccHa9ICFN9Y6FY01xFsXY8v+\\\\\\\/riNRMWw8qXRjLSCcaOgweUus28rf0ZOGVL7oyeQHcExaursTy7jlCj3YU2LdNxdNHInziNCMGFcgSqmxEVo91Ir\\\\\\\/oWURPhcj0yUuUsr9cq2sDOQ\\\\\\\/d9gCFbdar9YiicnfL95T+OLPTFpDDYHxgDa3unGR7saXpU289kO3\\\\\\\/yX1R3v93PEjzrbzvtVVPJx9tXtK0iNZ2G2XF9gfgSL2D9LAT4s34l5j5+679QOfgaBALGjyVz2WDrPQNsGlifqoytZQ0OhtpNw7e11oUXirBgTBzpDgctB44OKPVdUj7Te1FJ\\\\\\\/3XDSzkmH4wh30GCrKl0zet1pJ\\\u0022,\\\u0022iv\\\u0022:\\\u0022n+8pCxdFRuLjP4S\\\\\\\/QvVjWw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022adaa31ebcc9aac1cdf67d0cdfecf947a16df90d0bd31576e848a154e8d4d51aa\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u002217YHGmTSj\\\\\\\/kPW+6gOGwuPwGz+4ZzFAzPS2C0wfKOmneHE8RUc5xTEnKm+ZDsaIOoQkMEbHLqrgR9ISedu9gwCFLEkZYMBniH9Pqd\\\\\\\/nfBMHg816lHBveMnHAQ7L5OwxvK1eRIBPpIhVXUNQEBvXn9UvWdR1XjkGAvrQ0pjBM9O1XbpMwehXVSbuzU6Ubcaves7fK4RCEnpwfFblQ\\\\\\\/vNN0g4YIVO\\\\\\\/UM3RnchWTOH2BNPEbcFYDpjn4GqtB8HIwcV8l6Fx7WGHyF5Kwp5YowgBfd9EXxE0o91oZsT8dc9zklSsXfTvyjGd4aFH8KWyIaRdUh+O8ZWQm+Cy+nDaehnSPCtg8A3o+KppRnkrhRPIERVnScCHhyQ0RSabToPpYXJAkw5J9qj48\\\\\\\/QVf3pWpC1t4UpEPlY8KSPHylw5Bqj17Xk2m\\\\\\\/fuZNk4Cbcq8MPspaW9WEuDyoxnTBFgf80Ym8Zs\\\\\\\/o1X8ngUnRaBMvSTmm+UJQXA6X8lQG83zpVbRBqmQnv7AM6rkK\\\\\\\/GRwz6lw\\\\\\\/Xc\\\\\\\/Cqf2\\\\\\\/4rqRh\\\\\\\/qQ922EI04dTZ+HJWApiJxX3YW6CqGcQGSY8iI06mznBcJ9w7K9\\\\\\\/46ZHT7vU00mwgMc41tKiIDIKegSF2HB3Mpsx4eY+quK8fRUzrE1asX1jvDclB3Vu\\\\\\\/UQ2wlJwiTkOLYQcnkDe3NzxOjuQz3iIetGVsOvvQEOMxfngcIp2xpOBou3Oohv+U6vjwUatM4Wajw5IApzQtcuL5tW16OO93Hjiae1+2CNP+hMZaOhRi1erfje9J4CBgoadMzdfQjflt2eQGCaucBzwhVLpfJnCVyupj0105Kvfkj+6E0z5QPZnX1BblmYv4FjzM6c2kdxq5DLMpQ2fxitAejcOXd4qNLAOEQ3K5uHC1sJG7S4PAnG3q5Mvg2AdSUBP2LixTJ9eXS7TLyu6o9DWdIP6ZZhP2jHqLgPX8LLbjdYUUJeurnB+yEyeDkf46IiVN80pB4k106077HDP\\\\\\\/7opjwcGYWQD\\\\\\\/cEJw1agUnIj\\\\\\\/5rKLytiyJMDWqbj6Wi3ZjHRMYJt5s1pWIYq21RERwxLdWmoz75f0jnsX3V+tyDIDiAFGCaHlsXNgeMlb5dGaWjD52bb9MWYy\\\\\\\/ad1gFBBU9Ymd29OctRoAPrdYR71gIt+LWm7Xm0TIw9AOlXi7o8AfEnPW2eg\\\\\\\/NjouM1mLgGoHFJzTBvyEleN+1ptvyQ7GiVnI2iH19HSBLjbMAxIgRpodBB7p6h3GUMs1BO8be6eZg5S4AHg8+0hMsz5qndt2wZ5SOLyNVrjANt4IxGsGmbGu2qeXzkQ\\\\\\\/0Cn5XOEZ3iUmyxORPOaxSGH9X8x9xodzxafzjHuXw0hCYh42suIFnU+ZerbygSPohS57CiZtm\\\\\\\/esfi8cPpOzEt2OGqzJCqYUA8MXhPMQFnm\\\\\\\/tXmcDusY3w7+PRItWgfwmyStbsIjotZkaS\\\\\\\/jFWchZhrBJ55d\\\\\\\/3O8weLQN8cE\\\\\\\/1qyDpumfuGW2sQPEtZNOMAkKrBl\\\\\\\/ixSM\\\\\\\/w16r6HNtqnWTD1suD+0VAnehDbD6f1PywSV285H\\\\\\\/dCYxXrJpGHhuGO4EQz+apo5LzYgGlyBPb2x5iqJiA9XPsRJJLUE3Pmw7CVoGFndBJSk1b9qy\\\\\\\/6GA2NdW5SYE98ioq+pGAvi4zZQmMBKW0RiS7Fqa6B+FLjcowElb+hoqtzrKg+4HgR9mfA1QAOZ+FstR6hbR0TZCLvPU2EJCUTkVH6ICdjdvgI86Di3xMzy6gn2vDwBUhgJkf1LIc5FnzodT3gnGY6B0Z0c10KQrIH9eg07s++G\\\\\\\/zIqvq82JoxJBWtYvmVpIIt+3v2nXhQC+qIis7IAFVLiW01uQ7Btdu2xggJpeGKtj6Zws9g5RkdplTH6iR9\\\\\\\/o\\\\\\\/VbvAtjJR4\\\\\\\/DjoLX9p28niGR8\\\\\\\/wDwUc\\\\\\\/76aCNBxtNbyuwyVCNxRso6MzHpjiEVgVhfUFIUBR6kRgCiDn1HVa3sNFqH8\\\\\\\/n\\\\\\\/tQ\\\\\\\/TgK8jr\\\\\\\/O554MmYrlfwvre7n+65zwVr9SUPsiUOlu0Ju7JOcYzqLQnXtksmqJBuNFJ\\\\\\\/3djhsec3QqIgPRhTw5nmHLkDkePt9eIZPCG9MD8ZmmpcxkDfVjdEX0pg4xa61+GA\\\\\\\/jesjMlogMVaqUNiSTMAnW+MAtxNd1kY+XUQpD8UtM7ezULVSWbLNIp\\\\\\\/NHaDjyCqt3xqkkY5zcPGltvanLNXKA+Nq2d0ViRsN7KHHWsJaXfBDa0Rdw90\\\\\\\/aVVbm+y0dg4+ezvhjGND5ks\\\\\\\/+DUCbhZBjsYN+IM+u5jIpdukiEAoGFmfY12ia96+5RyhpjcBm+ZfP8m6geDAtxtuunVCi1QTidvqZbJtkvLb1j3iemh7C0hEx7clatnYk0vD4Id3oSxyOCEonuPjBpv0xTy84Kb2cNUD0Hh2eryIoOIIOsSLtuF\\\\\\\/7e6lgpuQS4pzBbKCvri1ayeQ98ZrSiK5cqKPwqkLKjXBTa1+8HBSRsqdxjGf1OyTKtPBcBpWOuCjEZAt\\\\\\\/iYrlqYoPR9yYYxy8CVh\\\\\\\/UXvQvs\\\\\\\/6pVFen\\\\\\\/BQzSyG6GvUVxSiVgdwbb0Aq4yEeAmT1yBDntEIR\\\\\\\/1nJgFslE9JtQxuKWJjdz\\\\\\\/5GgPB8p+eQbznp7lEMMA2qxdQsUxtPvSAjKDLlUGAiUxEyPiTC7GwS1Yo+gD\\\\\\\/EacLtpjaZpQn4Vh2P6fiOsZwZYNDzHdruo8OWS\\\\\\\/kHVm2qjOpRzWj1jG1FPcVmhF7vO62GOmUH6YWhPI6gamBTfYAmPv6HMZwJyhOv9XULr5Zz7m5CvoO6\\\\\\\/0ItndmcOfimeSpxDCN\\\\\\\/vVjyIKOtL9U9dbkCbgCJJF2QDq2OkPi8NZHKgtICwK2fBn+r3FHIgBSdHgQleIAwlC5vsHkYwDTIG6ayZpUTGkm4uPLK1ZOLx0m3nBgL4CPpxQXe9yyx+xqQXp520sdtZApJAJnurx1ybQmSSb+fpHa+GW3xv0ZxSJHAd5W6s6V3XJXwnd06NKazcMICCQTEvePl8VmZwbIglzPTpzQ5Hy2nKOmwxFmWtg4xmiwuvd+kbua0RsTQ0RVWhzLjz1mIOtAYsUWlTZnyHUlooi6BEzSZFAggwPIyAFi66IjI7n1vi55MoOe\\\\\\\/\\\\\\\/SAtT8XZQzAGPAHV4mR4E6jZ1OVe5TlLHutyMDW3Z1823qs3P0JQPZjByTG02lDgg4V9szF1BTwcZAYi+dOcHxy8fLusH9wIyGcr8jyMGq\\\\\\\/tAVKno2MYfo1OggsyM23LcttJGRR4pyhi45hBlcKQViUhqiGIoAcYnXV3XViJxlLbsCB8V8J4LKuKWpiO0ZPILlLSc5MkR2lqfUpmuBz7GKGWWu1sQhEJyXCrcbcURDcC7k8Hy9U9lXVWB5dzJ8wj10ZLkgjrNkmviAuSr50epWyKIvycY0tCdSv++lqWyRqwaxcbPU3XP9NgO5qaSv\\\\\\\/WTVOtdvuujy87kCGYfaHlqqI+cg6EFF0RLx51Muyo84JhG9lsig\\\\\\\/TiQVrvd\\\\\\\/0AYu8dAgLsQHfreHiaXAMnrMQ1VDTJHvuCJvxO69bveKMSWCvP7eUwfL6wTEwiBIRKqLp1xIgxafwgnIEVgB4Y4yplf595T3J25bNR81vmiquUHq34cfq0p0vCkxCqaj57ENBI0pEBhBWVp5DWG14G0Ff9kKA3UYJ1jCTDTwmKIqXAt7oR75vPimNEKZJW35n6RnmoqkBRqheDxUz6B+xOA25vxcmhyIyyiV38BwXOcY3yPLYMgXQ3z1fBmT6xFpmimu2jNgqROP7gN1db4hqsuv1Wqu7258gD736RIAG9+sZejNkMPtX+6NkcNwFTYF2PZbVj9ayDncU7yl1zv22f2gv24THlT\\\\\\\/F9pZp8xTdupQL4RnC38KEkhdkg\\\\\\\/cn6IkFfIymdpT1l15m5lxfLzoAu6xGxW+RMLE6Zhj8AH7XSEY\\\\\\\/XFfzza8tTq\\\\\\\/8gGokJ32luXnzldR8Q9ViojQi0I\\\\\\\/\\\\\\\/NyO2veraZY0LcDPboCOHRnS5L97O2Wv3lpgYTS2J2y4DyryAh7A4VTjyUI7pWgwEswqvpbZ6oYo6N87oV6\\\\\\\/67zdIy6nCxGANBimI71MhSQlimYzqfF11D5KWeifzBF+wgWRL6uNs0SF3aWtY8X\\\\\\\/fyhjw2gOVp\\\\\\\/t3MP2wdWmsbj0lsyNUJyo1KcafN7yvwXmxHPmp0od1eMH3TBRVeXyxCY8bcWxRPZQTAOof7FwrPup9OPvmq89s65suyBGvAU7Wq+d3NaYu6G1gkJ3bo+dgMi6zlOVz\\\\\\\/Vv2k\\\\\\\/WsqHQU\\\\\\\/nLeku1Nbp43wgie2YV+NMjiz\\\\\\\/Lfbq0L0EemHXgwFrTnfoF9J\\\\\\\/7LQ\\\\\\\/IjeAwdmRvqDKOQ3rOjlhXZQzNHx6iQ33pLQpKr1zzUPHsYrwfvrgMGxRqiHigXiE4Vf8Xf++Vo1VgIEL98P0GPeu4bQ3GTegkpaunEo+Dm6RAZ\\\\\\\/FFnIb35E6BecXVoAExMq+xarCS03jThkM5OTYJhWj9WMZwanzPM625wJESY5VxK944YyExxpo91mUZAqdCKpoispiJd3kCArz4fUz839z5wJFS2Ga8PigT9VtzyvYgdij2DzmUc7YbSWKAd88le6s2HlkOYauPD6OE2mHVrvhEBf7VSkZPnjB5BsdvepHbyInqGFy03G\\\\\\\/sqoPMKecC963JhYmnzQsaT1TeXGLuS5OA+hc7gbZ1pJ2fgFb\\\\\\\/3VCWh0jsINMy\\\\\\\/mKUrnPSwQBXCLZVHy2cx8Rrf6lJQqzFv7mKtdLkPhK\\\\\\\/SgjjS7r16dN4\\\\\\\/i2lnpBe5L6E\\\\\\\/StlAY19a5jX46mq+qUjtDiqNGHHm4y6JBx\\\\\\\/KKEEVMwOSR1qaEVzAYy2arwJsgsCvUxd3Wahjn06RtMGIUTOzjJY97DnB97g1lD4jFgP87KKhDCFyi6dwWwtVtZKl7gE7n1qPTi+pne+6nGYqCPQSTWvp3su+9snyJuDqnZkcjSAh28F2S\\\\\\\/QTGJij8wirDyckyvZKkJ9i7AREI6538RVK4c6nfDR4YC9B7lvvGJ0C6n4ll\\\\\\\/j0SHyUAuti2lpY4tqNZXwfksn14alFCZea4KqK6z\\\\\\\/EKsgWNq9coU2LtSi7S3lYknnBjTEhJ9EsW41BiNy1jWe5OopmTFO1whoETm5wyVjwSmszjB\\\\\\\/w8Rh4h549WqEN5fTZ5dB3hYAodjutAocCIxEXLvMPpPM9n8FpyMccpRZMFzPxqWRlkz4Mn27wQcoTY1u1m4lmCY1qw1qvozX0os2sryEf0Uf8INgwO9Q4LcN91SyK4\\\\\\\/CkXIWaFxjcFC0Nbyzv2aCRktTrOERq4Asj0qtic9bk\\\\\\\/M\\\\\\\/D0WF\\\\\\\/\\\\\\\/yjEDwdWpBIC8GnsQt2gmpKLunp0Qjg2HAcfcFg4HWx8fAxJu34qOXAS30Mo7ofwZ2ten8XdQ4RUYQqpbt2OihJPRtJZXgBpVkVN1fYD0lrSpbiiFFLG\\\\\\\/oSnqmQUlJgOHsndd7xpofLu2gL8VaBciSeBbO2mkb694QZsg5C0dGFf+CdDLumdss5vHm67JL5O6CjjRWUnnXskzrutDIQo\\\\\\\/IfOYMSHtgBCd+wYec1ARwUavV6JVrLr0Qb8ssGwsRtx\\\\\\\/xWADV\\\\\\\/P1JMJvuuRc36OlfNfHl+V4PzOKQAJPAhRKvmlPGI0tq9RFOD8y1xxgDYocFA00dqveDNHLXxZMiHRNogEkPUVitFVCBuu+\\\\\\\/y9Fl5ZuFa8BdNXIi3s41jf8cv64I2QtBpfrPT5jW24iLE6f8PB+bo18rUGf3a4\\\\\\\/IvhSV\\\\\\\/Osf+yepdAdvcUBC3defS6hTDSznSql6Zto3dBfKAy0JeYixlWHbGBkuw8jmUdVSBYlhVF+ulNzIna11TYcpwNCPLXUMLYmibQHOKs4qyrQOayZIFFH7uGmE69yJ8vov0DAvH9C\\\\\\\/28wGdMaxKPkOBZcygM\\\\\\\/Kmq6\\\\\\\/\\\\\\\/vUw5pOeXWuYgjb4\\\\\\\/UWdFrwGDUuUwHj7Z1CBfm+u\\\\\\\/2E9K8jOwiJkhlkY1AWJNBxiFK6Ree4PxZB6S4SUxJJk2c6Xpz91ysXerQ8sUEa2a+SxLOkfZ\\\\\\\/aT7IUg3IYbONyRrCIyAiiMJ31cFbYYhiC\\\\\\\/KywKHRhAtDQJv+YzXMWC4UlB8gnRY+KWOKvGc7LraZvL7jwIUPMPJej9dbCDCHujxRaoddQ+DNtyjvQwo9e30WOZRlbHnq\\\\\\\/cc0dalZ5CE+6jiGhCfuao0+xI1Ifk3YBVMdvgWKPETqJsptGmV\\\\\\\/Kwaa57J5XdAdQlp2vHuMBBBu6kbZixVTZpM7uCbSNGqcYgfD1vom8bSRrKOoxFhdIc8n9DXHbiMrWOOAzqE5oSHJK5qUxjMCfLVA3NxIxQC4HOYbzlgAVdtxRR2Em9JZ17aWJKWoD\\\\\\\/GsZrvHX6yDxPs1PmSYHejBXMmixu9BSVfnSzd0kqwLSBO\\\\\\\/5MCU6FF7bu\\\\\\\/0ZsmPgiNc4Rfb3ci9IUyNoQJB7\\\\\\\/+og0FRjwxGirfnRD\\\\\\\/MCPALkKK+MKV7b4xahn+\\\\\\\/7+e9y5s7D0Kg7F78R5UxTHU4cWpqv4Hdge4RAJDCHXr9UbYtJ+iXea8FgkJwm0cQMhxEcKizX+6QA\\\\\\\/mCcKrs8NcUjMsyzU4jadftpf910m7lIdw4yHlwH2\\\\\\\/XiVBROh5K6ZGylVtd1RSwgVosUQ3ZJhnuebKw0FFUzEs+vdr\\\\\\\/jzoZKIZSZmfmVUIMDDVYj3ef\\\\\\\/dd9clKdOEqq8336ubGZ0TaUROLM\\\\\\\/xOP3PkA1iRXLODqEnXtSye3DTWD\\\\\\\/HmcZFrzCv2tqpBKHC1U6K+Qk0vzBYrWv22agbhZFdyHMXLV7V5EUP0K8UsIn1SAmKgYPw6DhJW4OOaqOMdBVI99i\\\\\\\/sQKH5BPY9l1\\\\\\\/RGofcIDcWOZ6rTewtuxfvwjO20pXKTrmugyw7YgRshRRVQQEHuDdMd\\\\\\\/rqRvhw0ImvBxvopquQSceKf4y8VRQqjngWJ6wVhZ2bfYmUhIsbzdKAmJXHYk29NV4L9A3GVjVFLref0DOr170GjAD32o56kpwsJzmY+R\\\\\\\/7MTU6sx++PZwXOvLyqF5LEFoFlh9WAs4iw1Aez4uW+IZj0eZDX4q9569GjGCj29OPbf8OzLIZcMeHwuC9cTemcudZSQIb5VUDmhcyV9ZkBp3Zo08A7yUdvQRPOTNrjAH76cG7UR9fs5rhHa90jtCq9RoFml6xLeoVb4WfzqbPCetohKIb5tk+RM1jnHAz3Q+wVEV+AQAymsFhjQchxWuJoXlS4id+S61xKdygAANSmLVTDY8r33Jg+S4WTx38r1IWtu5a23ZOnRxUaQdBIbxHytSJ+mFvb10RXACERNeudfVigPgclVOH6M0xnV+hfrR7dkxt5iCvyn2Y9nqzItk4sPzqk41x6nXyIbgMsINEqfuUJm9Wg2a2sJzsnyZk7ZVe\\\\\\\/9czwj5nhTbb\\\\\\\/N1mcjyWedL60dKhZiY5tHmiyNkpZGhiy8Xb9SbtlfMFFaJElM\\\\\\\/4cfo9Lq8\\\\\\\/nyx1WHY38w\\\\\\\/3qxui2aDzg61mgd\\\\\\\/EdB2MTgA1HN24PeUomdn6vUHi3t+47jFdIyCJsJcO+2R6m\\\\\\\/pTEABbEk8LsUB\\\\\\\/iv0XgZFF2Whad5yh8DQXSztHxTJNr+kyFnlg8Kcu+ciunfq9FZqQouZYG5eueaIFA1EYXWzWp2gfEq4+FiAQFKPBB4HjGKrmOGhpCEoj6PjpE5y1MxsXVqNP2UDIgXtiR1reqPoi7KpgYXhAZnGudaeYJCzl3hxKd\\\\\\\/MSagp8ni03Z49ezEB4G\\\\\\\/YszUQ4\\\\\\\/5nDFOTI98O6rEgmlyU7pK\\\\\\\/15fzIhvmajqu\\\\\\\/aVIJ301\\\\\\\/MewleSyjor+m\\\\\\\/B89hj+P7FtI8dlybwGZ+8+xcj38iNRpkpfPhTyc93smz71Vjd+wTHW5Yd1\\\\\\\/WbIda5Xa1ZvuN9q3IqSkSiBbG2Avtr+sv\\\\\\\/jojc1\\\\\\\/bqklSKc8M3aKVKO8tHHMhhrvmn\\\\\\\/odYJDQyEgd88REEI2eoDReITo1ijR4pZjtQskI3n4s9MI5fxJX0rSr6HJXmDJB896+mKk71AJWiJFxDVdsiUxOAocAAtH\\\\\\\/vQG+shsNfdkdf4BFhWNnnozBOc1zJSVhWJPX2RKnOOtIV5Bf3EITVstrFwcaOGPZ2LYupIdmskrOxnrrKbD+l+psAEFXaQ9RAJazTJZ5lrNt\\\\\\\/cc6HKeOlYXxBqL0zal3R32Rw\\\\\\\/Sa0IVxnTWC6PmuR2KP0yy9q\\\\\\\/CumpAFw2A6zu4hGKV7\\\\\\\/w6G702tErpTHfMsyZcw37qXpWTdpoJsnF5ZO95UfgYyYQ7dvCnwoxDSkfk+wJ9KdeElG2618AHSw3wo9bKZG5P3yVwbojpLg9PDI+1nlQkR\\\\\\\/j7mA0Vo7Kdsfwsh2eR5rHdSOP1vEfS2xyylmtqeEikQADk+tLGbZXtrEBVuBBVRTVk0BpIub5WaT9iED8jRqkXsz3DqdrsNQWepwUzo2IoJ1QUMyhw8GtaFkhXI7lPl0czxDcZAApucCGN2DD0WW\\\\\\\/CHrFO862zwQ3\\\\\\\/z5K5zgH9U1dmJIpmdFg2tBxCcoLZNNZ2cpxKx4ShafGCAx5xm3HIQz0bgHQZmlFSew1sTKFa4tYdWeozjoWj+uHkdkSNl7WzN7SPMXJ7DHLTk73\\\\\\\/dqb\\\\\\\/cLP5afCfuvv0T61QH5MAWQpZ3Yxhlnbs+Ygd44Tefyey\\\\\\\/X9TTVQSyETKBXMsGGKq0Azn97HtmiCsdC7cL62arGduxrXyUkNZOgXBPu687C3bU3nY4b2bF4c39qo7qob5GTePrj+Hdbq5hEE43\\\\\\\/SX1fIZUzxBbErBAqjUF6LiVp4GuZwKMKNB8x1o8Him8HTsy\\\\\\\/jaA5nsZS4gh4s40+pCE4ZgtvgN3Nl3480XPaZYp3Yri5yPJkjWzOlSQ4O7S9R4TkJdKlgD8c6jaT9mneqvAc9xysFHw0Ve6o4\\\\\\\/yTaLK0Qba17fvupSXn8qTZtq\\\\\\\/LvaAg67ykp01KcBS9hYhVNkqJKAcRkipz3UgyjO59yIFHyrQAlow7iiDXefNNq+lSV\\\\\\\/23ATO4xmJTk50nHTZj\\\\\\\/Wt+TFfL6WLK2XowhBiE\\\\\\\/voEU6oHTfPz8FA2fpobIzwAp7bTWRI9eCw1wwcAC026Vvgn0KittIU8MTVst0KboU4wtrFUbzR+4TK90KM3QO+2hxVm3F7bG7G8nMJAv5Z+v40c2uTu7gWl7gk7K7uTJMyR8PZtVANwaOnO9Pj4M3tw8bgC1n3Tn4+Psajxyivs8pQ85FDQn1SIVeEdjPUNwjNjTcmNYB1RBvaLLN+8Y+Fy7xjcbsEFzDeevnlVD1qsE14otRc7cPqP2Ngp1wrUCcH3XFmFrcfbBK9y5aaRY+IhZDiQh5x+BCpuaAooG3fbjB9Fknl+esWSUf5GlsdNMBC5287feLI8EOOnZL9trmVGKTtqDWLv88B8o2\\\\\\\/3xxxw0zCUKqffLRU9cMpGQLt8fGqM4bRBCzuQiAhpt9Kx75vQzHeHpYYRQ6n5EOtiYKTS2b6i1cPPgRnloASFqjjFQui67wuYLkTk5np04Sm1XyaYJ8qlbqwnSCOgg4pCUd7jP1VlGd67xSnxknVdjUSVBl4yta3zEozCXktIWjCvmnZLvBKifHwBzmYIMiD7dVb30SfbJHDK\\\\\\\/RHBLn2bOQf9mE3NM2NBSMZGTd0O9zXLXqSjII7hUkcagax2adPanQdnBv4Pa4EyYdDKQnWSs+5S4zEfKsS9p1Q9VIEflZatseFo4kjy23UoP0DMvzFOZoVuncGBgOKwXJVvjBhLTBY+Y+1hfxAcOb9SK95CPmTmSzWZw27pZAz4qLHTQq+j3kl+OfOyTAoQu39i2Uik6Et0Z9zaaziQ2E5JTJhUECmqODpqUZpz4nTmSOstek5TRQMfZjBlX8c9h5+3n6FRlMYsrXZmpFKjmrf1fH\\\\\\\/51ukH1AGZITLHUItm1GFdpHIivRuQidzIRzWYj1lBzLdd3sQfjAPyDkdpoex4RHd\\\\\\\/+UwbecQ4PF+7P0S9p5SmAPa\\\\\\\/pRRqr6sEMo5CfPMSIFqT0cwslvrg40\\\\\\\/rQ6uWm5ZrbqRmpkg8vEI+kymOfw7iGRA1SKA5jVcSxYnJkIk23Job2EuGLOZmBqhpvhWbwCb8Cwto\\\\\\\/KrT2ag0nNIre6x0HjvPzqp+WfEp8wJnus\\\\\\\/OqMjyOsBGGzd4srX1eQiR1lD2Q1L6LLsr+X8PZ4kkonHGH1A1\\\\\\\/kvWExslZcVqSQDYxz8Um+7U8GUrCmsVxymeDCdUc5eGjHM7bTqEPs33zD6mwjt+mh3XDFbgr2jIw5CgoqBTG8unk07klY1O8eqhDkAYVspYnlA9GSKO1vu9AQc37SUAmWJabB7GrQCOp\\\\\\\/EgdO8R6R4ZL+pSA3BlJHldOEyUtIerVbrgbgJR06oxUscBBfo4qqIiLoMJNp6hoAhkGpXIm8OfivcMvHcW\\\\\\\/KAzOqoIDgpFv\\\\\\\/\\\\\\\/rFId0UPT7XxoG28EBOTxbvSiQSV4u3RgXLEfMd6ZJFzDj34WFriM5n2x+GNOX8gB+Cl20Zc2b2G5io8DvVzzLC4xfmea84LTGAZY8D8t4tN0Vg+2+B\\\\\\\/3rsUMmhoJRX1XxItluzx9MMTjTSeNpgXe2rGeP1ND0WC54KRY3u\\\\\\\/de9j5\\\\\\\/CYwkImkuXOAWdTN8umGlRKUj7XQEpD1P7BgrehheL371TN0JDAKJ\\\\\\\/T6GKoc6+73u2GM16VHuran1O8T+9iA7QcmAXbo0J2oYVkOeMeiGoCmim8zx4P2wjlcyvyO9SAGHKmUClZFC4w7xzdtpYp6QsmPEmcTg2h+SXUFrKcxqJO7o+Z71VPLkpDSmlNnE4dY876GiFEjiuK4vIJoLqv6K0HavJ1Ogp+gm28xmOACDzmZwpn5Hjy8WQVEnVUGHCCdgqTb3QybxXNvzAakE79GHBisXob\\\\\\\/sVHwNUZzfe1vFNdhScz0QlOPpwm9QnQQNso8ps\\\\\\\/cGyeVBJvTXxGTqAT9xslz+rh22DE\\\\\\\/\\\\\\\/7Mcqg73MtkricYVfCzHhvu44EPiDhVznp1k12DnB+2jmfyPlhV25qQZrv\\\\\\\/m+eDw3tuBTNnNKztLvKvQqcQyA91MWr57vQnaDhk4jumVhrQs1h3ym+guJp3m0LySQPwR0iPSDej6HbOMxOdC\\\\\\\/MyHh4RcjtszVfPpcP3F9Yonbh2c+TUbgVyO09olenBpwd\\\\\\\/jhRcfOXDo8zhU8Yl1A2ddWwOpLaRy665Gvkd0h1YMxy7zG+SXE4e6LuLdpTm94L5F6MHRypiGfGu2uKmeXTLGdr9ZNtaJkL0e7rqgzFMicXujfvAIbfbeejsl7i71x\\\\\\\/wmRWsHtGOrW8jNu2HGLwOJxUskaDBqpDdQo+JmBiTXueZlJEYOc7\\\\\\\/EuaThMpXeey\\\\\\\/OIsiq3ITtkPvlOIwNVmY2kPi6wwLuTGSlvea0JsTHrCNeupzgpoY1HWIF3OgES2Er2LC2nZEPCT6l+x5leluHm\\\\\\\/mpIP0P\\\\\\\/81Y+ePsDY1NgMcMQDO3usNsgJyvaDpqvgZA9dbCFhY6Z42qkwlLg0j6AI+WgrBG9ubqBZ5KeH57hAtga50imf+9jM29D5B0XBSVXf0TTGJy3X378dCgXOu+6apbT1xKhH7\\\\\\\/9NmsQ82dAtfElyHKO3Qw6MO187tbaBIPUlVE3yOwOS6MXrC8vjzkq9evcBiRwAZ336ucz+\\\\\\\/kO\\\\\\\/bLeZ27k9ZHOyJh0grc5KhD3LlUTsEjZdYk3PMSe7j\\\\\\\/VQ9fkT6\\\\\\\/a5or1hBE+BvPHaphc\\\\\\\/TQJTwDwZM1jVcua9JWJGdCq0Aq6Xx5RO3AzpRQGh6G0Oe6d1bvQaU6YYrKqenKHcnApt6lscKpBTyhcmM7u+CWt+SE+Iu2L7fF002vr5qGZS6XbJ2HR7PLFpi0a2DzDiHwfZ8\\\\\\\/dZBXqeqRlgiw36aLVrtUBF9Dgn31uL8ZHm2IJm2p2IqLmooyvdNtueD8UMyZWGjUGh8TElkATZ5oBBKrjKymdV9ujqDY5PHZKd6lNPprIkbLwqH0ltdgE2Bfti0yYiCitPYOt9QRCWcGIP9WikLoYw1H2jBeE2QLjVzcKWQdoN34W30inyMWhOtIQLyHvh+83Y+CrW7xnmTWSOmqKiKRnWH8b9yUbV10zhrFuoQ6zUhEcxNiDUibBEmqvmPl4ITdF621wfgS1UNsTCj87fLzho98lFjROnGJ22aB+PxA5DegwGP\\\\\\\/+LZHMUkA49UCith26wYu\\\\\\\/G+v1JbVon2sXoIryXSupz\\\\\\\/MJ\\\\\\\/9IsWY5Cxn0+54cDtCa8Tzpkje7UU31YSUlCYYoyS3jplHVHS9ooRIWg5\\\\\\\/M0BqiNssnoVWpXPD0sgP5+huRMKu5FfvQv39Gzsi4u8Jm4wnyT7z67cTX1z9Nb+l67lDGfILq8ukqcJazAT3IzLOV6CjQVrf+IJi63qHWnaPfjsDtJ35t8B3RTCu52rTLwH40hxLeKso7TJidBDltrSHS0qE5siAXuubao8sB+GLaCmXGC2U5s7lGLdOoec7XS5pCJBfnTYzXlPTT7M\\\\\\\/UeGOJGcr69lBrzYL+AzcDukCyTuMeqF1M\\\\\\\/JKViUBZLQaDpGXfGKobXP\\\\\\\/B5C95otKjMhz6DpcNJJ+Ag2IgQFBbVBaNODPnBFCA7A29vUMHy3IaBSRSslK5ValA4Nw8QSVmxOegWAdqr3yrII7fZqr18s+H\\\\\\\/GViONu3rXjAuFw\\\\\\\/N0JBqrXw\\\\\\\/Q6GupGOdDMvZyozjFYh+o4wTDnvuk\\\\\\\/7BIaVtZgK4KVaPJcogGrxJXbaCCeML9eo4VbmQLiAhOCnA17o\\\\\\\/K4beQYBiuauobVHBiwnkCsd7oyUmMrKZgnK1q1K2sXPUDgvTRz+\\\\\\\/RcAubxIzuqOxTAhnGBFQio1X0l+gFCE5imHmgh0RAFG\\\\\\\/uBUIuieP1ccF3mKjurELhRr0z4B6rY71005CpaPUqTbhMZTnZLBgytQXaS6RY4ea54sLTJQ2qQ\\\\\\\/ozEHVL5jcIgtaRgd5yoj2M9r2gfWFBy+B0VvRqW2i02oG6jkvkdpHmIPKAuE41oSrbNYHMXIyQdeflUzJW+skvLwV4\\\\\\\/7RMccaOHZvvhCWGLM0uBlXzb1yK\\\\\\\/XnkiymNbgbsj522Nr3VBMtpiBFvlbBwpa4O+3rG6VuB58JtlhZz1ghXNAGSufTSrvzUI3fDPh6BiJMuKhbwd00BmhXC6rsb+9kmTkw7hiwnmwuYI22LY9EkAfUDE0EWeFmaiqhJ6NmYvJaWqQ97MNzA1OlKNsopL8D4gUhLMc1VcLjOz7WelPKtoSga6A7HxcrxxPefhEmKqNVbIHP5fnXBL5LdTT3CKGUlLJ5eLnn8L267DanlqTA3gHFiP9C6hDpyh4VGqEIhiqCgIQD38q4uj\\\\\\\/nfEBIr0AI5KtNghNbsifBI8g52neP7fPxfyBQWwHeuW169oTH34WTByXc7Yi6Bonb1yppFr7KyI7DhilQLn2PnQCbJjiV+sah8OAxyZf4RKxuqvVJRve\\\\\\\/+zWBwhDFpUK4fJpu4ohV8kQbq55wmK9\\\\\\\/\\\\\\\/vXsj4Ou6dQsVOJmkqer+5LXVxjzoXeh0SViuB27HM0f0ndDeRqwkXEt8sI5YmlH38xpvsfR5\\\\\\\/zgxscTuTTMgfq6MONiFlcXLZMT5BjAaKW2tugvXDimAgZFWxkYfnItHo9PrZ+ta+W\\\\\\\/P6ynmE+oHqunmuz5GIlJar\\\\\\\/LBPaJ9DExGBWvKz4IwByAvbX10Km40P45fne\\\\\\\/vrp+ldXRlMPfZzk2RPloFMyWXsmqaaleYLazVIBpvusxhJfin+jZXesAwF\\\\\\\/COHT3sbxzd2yhcXFhyFOpR8vSkJOnAcnSEjn+oFh6HpEV9N1YYj35Ieu63x8csAP2jDLyDGQbXqmrRryFCvIInr\\\\\\\/fbYYEetUtGbvrVTt4QcLD3zCLR8y9vQ36DuWQt6zZSIKllN5beHGFoq80DcOKvxipnbpe1cOqUbq0cahA8iKWiCvNgFq0RD1SGiuGC5icvsD5SgaCEgUZ39KSjCACt6PuJMutPgfJTo1q07Y3z1N3C6qZ1wHgGo4WXfZ5DE+0DZ7yNbEMtqdsI\\\\\\\/9wYaVo8muChe46omqciQ0hQoo3UbTDuL64NIvsgIlakcOz\\\\\\\/DIK15oXWYME1NoWgncNz7L2TjSRXKqWTZkzN2FgzCF7siH9sXKYXOjWU+c+75D9f3Zs3QeDWXhgrMAY4R9qPImUTqbCVr+tG0Rp4E61TOBR+ky8205WSnHGyduMceFk3Q40MG6DmTH675oF4qycXbNq28gbG6ktjFZu0g6TCBTd6f+xVRtLxHzBjpXyJCp5E6UDKl0I5J4CH5Az+GEZu0BN5QJrd+eweu9UD2XCAfR1iRH+2BtYpEgVzAfWvgxNFiKDtxm23SR2SlXEI4yjecslL2a\\\\\\\/JVran0y1Xuvd96JTJYFw+KhFSnnpI7\\\\\\\/w2scZTS7gyowaPZMDJkk36htApEIWkSk9qurXTPJV7q5xEqMlRzB1MrORaTr+Ngoqzoskuj9MKUXlPLf8SV3gt0VXTz2vSym+pMz9IWi2jLsP0waWNr5IIHFB9wQOuG29vlQ6DLEUW98CHIcazwvTmKlX0G8L1JN5p0yYSEzT1ERiRvLBzdNYIskpLwsVmrWtwdRLsWi\\\\\\\/rqZIMry5TfF56Nm68MAHcD4n+9bLHbyQPfHutSFBrYLLHl11DDtXPWLlI6N2uBqbXtlpAMkcg75pEI0i07sI7AMaicDEAcD01Q7Ho+LDG67dO5czlngRjHroy5oL\\\\\\\/g+\\\\\\\/KY+2oqJd56jNBX31APgR8IjJ6oGLooBYduZgQYpoZaqJIWEcu5s+RMT5JLoXzEOANzz5ASvjP17d2obe3ih\\\\\\\/I77VoNp4pWhoaAmV39LqmDyDG+jMxvavgyp1oGnXg0A\\\\\\\/81r38cv0OnfPoN1Dlg2+0H3unObnfLmF2iPaq47tHPe4ZdrUhOHI6hfVsApN4x6bsv9bnph9TVuEV3vkwjj0b+iSscyywSf7OdmIIPYcFaPSgC65Y020KAxCTZ30H\\\\\\\/59EZS56tH0KSibwErZ+CcroryNqEh8KVv6D8xBvCV2FC9fqttmPn1UeEpxdTzw8xXshpXOqoRa5sENLrn3wWg9hvasGdVQDs3RiRgg5tbmj70Sa4UqZfz1B5yFaPSG\\\\\\\/uN6Bda4r7oSVZeEJr\\\\\\\/nn8iyJ5XjJlmhTUKOmGb1\\\\\\\/IWviKOfPsalp5V3QINpuyicR\\\\\\\/10GbZ6LqWX33GBnL1Evb4+Eu0C0EGduybCAUGFqMtljwCIt8fcawXMJcjQFVLsO1OFvxoR69CEPtOrZThQhLgYJYWpkY9E+1+GDAyUMNoQcexviHuun2CWDfMKnJO4Wyv2g3MEW9Sx1Sq6oxtyX3SPq\\\\\\\/obsfRwyfJDZ\\\\\\\/Ar\\\\\\\/QAx1gBNVh\\\\\\\/ki+sEwoGQAGhDBfar6awLMqGZADc8BzIJ7UIbEWkR6rGCwnAtH73uWzSrYdnyFXXGijkfJm8gqQnfPi9LR7VvA4cWNzoI4\\\\\\\/VzPnfHPxAfoVAp9IkARB5J8zCKna5hNmOqv+4fSo77ndIHaOizEPOj0EEbQVXBjOL6bjbsl7ePKyd7pJoF0tzRnEWS2I9Kk8xc1mikRXZcHRxwQHHV4aJvVqwQSTPHZZjNOiXnOoESOEH9j0dg1DqgDcFAf4kqjppxWRwk8mrcjZfz42VZAFQ\\\\\\\/3dzrLPQW3f4e4MqgoeAiETf2vdDWGlBQ003IEq\\\\\\\/djLAQ0zKh0bvX53+7ySBbitEyAve+mCXVmJd84\\\\\\\/jqBne7+CzWjpTcLOtbkkdjmS8niKHc0pjtBUy+iTWEJibFtlQ526ZiscWEH8xVgugTx28NDCp0Wv2+OpBHmoI9jgyYU6rtGtNSQh9davQjlAAhJ93KA0I7vFv6acG2XK\\\\\\\/JOsCycSNs3bGBsULWqe8ZpPEH7uRePoGPeqx+pbGrU+MXaA5SmdRHYbypsFG28bt0Bsm4SAAoHTdBOuBtj+y2u2H8Y7NCoI1NrQ8MDosgHvXn53TawaQ6Zk5NV\\\\\\\/46FNbzQrvMrizkfrHpU4EC8jsiL\\\\\\\/NuhDo8jBBtYyxbxwJGHQD204Z052xaEGW8aqddlmMtSSYQbvX66zq9o\\\\\\\/cJFfbWt6NAGPpzqgmjgBK+ABluw0+zSlWMjgxNMP1bOTJkZyxWiO+ShVZl+jbGC3Bh82sUCIO3OdtgUH3bBpNz0J3M1rrwHu5rKKBhnBiCz8o4La87\\\\\\\/9LpskvsiDxLXCmMyVbHbNFJMBLzsDdMkLqF++naM3Q5XHNurKm0rddjx\\\\\\\/+LAA57bhQsShVZA5RVoPIVpe8mJOuNrnBDH3hNbQwdDWLD1s\\\\\\\/XJPPF0MHtVJJoJPmnMmKdM8AH25OwNcsLEXLw20gf2D0jDrK3a4mZ65Nz8njBsPSEhVvYH5AgQW4HEAlpDq\\\\\\\/VUP8OWEODFc3tymthi29mOXixnJzXFWDWiixoOb8A+4QlytG4UwW0a2IuMATLNKwOVZiZe2qBjcqgZYh4yNHCeXSlCOw3Tex9iOSs4f7s1E0uTMdyoU4C57cdMNXh2zYqo51Uu6u\\\\\\\/IwnSEY63gl\\\\\\\/4t6paTeYw7fJy5UGVBWNBdW\\\\\\\/NV4Dw5qYDiu7iwOzVAwyGZ9y+l4+BZBo2W36rB0ZJA2b3ZhScziuBae5tx8+T2bWkq7bjYlw1IUrYX1i5XCLnSfaQSXp0k5T55bBYbTfoDphG1hSTkIEnKJy01wQQGeNcpchur5PIEk+4acXLmltoImWx9erNcJm4Uaxc6YWNgCd5GHBJ3PNu\\\\\\\/XguPq7DLbDKlKEcpe+\\\\\\\/Pdh2734Mb\\\\\\\/X8+H1fq40c9nB3WoDayJ97zW5ZKpM5D\\\\\\\/4RrcdBgvFAuvATusm2dlpWUtrsU4bq124nLoe1P\\\\\\\/PS4UqxnYbJKoYlrT\\\\\\\/Liu6\\\\\\\/Nmd5KBdBsgVvo43NgHV8xr10wt0WZAQiEBVB2UKCcVNs+mjCWrn+HWc3D5dKZTEfUPuKDhj2xssIdf8LYDmYiQVwURjWLMZ9sI6abbZWDv3CHngN3g6YUHD3sFcubmec23cDZRmMUOAMpuaqR7R\\\\\\\/53ufuHHZk9by3DxOPr7DMgFDP8YdKSeXBLO7cKr1pyU2358uhJsE+AzYqTGhrIeAxtT2byvVzsjFFgAZ9JZZoDzfy4KwlDJUKd\\\\\\\/oa+hlJUqvtlH9B6M+VGEram7c\\\\\\\/gUseEu3HbfvBy2C9o\\\\\\\/546j272q9Uz52D05G4aai4V0+56\\\\\\\/4RWVlfEVGEa+i0SAD9Wcx\\\\\\\/kcY2b1D4hobsYCT3c3DdGgBlESSaEEaYcihMQ7+sq9eZf+Cz2mMeZRzBT4I3q+pbw4UWxn2\\\\\\\/jvDFtRsiU+w+fHDn108m62meFRSdmBZUjeDwMu+dra00HR44Iew6VMuBKR7qV6GJIMOP9e81vF0il6WWuWa8SVEWZXqnn7SIEW7nu7HZuwoAy9QXKIhcgu+EUKEkpHhCZqyM+9Eque7NlmxlUJCJV8gEHtl2pezU6KMOHKLpN0yVRQC9mDLOz3TCrj6ZvnEmC1NFIk+lzb1a9YH1oEE9v58f\\\\\\\/Rswl0fcBL2qOy\\\\\\\/tY+kg74QfA\\\\\\\/1m5Ksf1w1YJnTxeYuKHxktTYVeGK1KxVmh5Fp7Ta+VoQN9BzDdk1TmFQrttJP17lutYiHsAraPagiUf0Aj7CbbdMxve5yD5komIeKyzhhFjb6\\\\\\\/aYf9TJJoM2iW4FVJrNyylUeCojsVUI3NEbyFtyYE9cLL35UmQwS2whNIYXkg8Dva\\\\\\\/kLSlefacKVnmwWqEMyPsuzK8+bFDwu7oK\\\\\\\/5zfMkanEWFVD7QiTAi\\\\\\\/8q1yWZ8POTWCl5SoY5Wzj9IfBU0fTG6kXwH6iUAh80QhTWw+2HePGLf3c1jEC7KYgHa+kwHnuUHZp9Lqu0diX\\\\\\\/eSJ16FqdRMnkUZB+e8xc+1BaeUTvgujYtNGtVrL7ImTK2\\\\\\\/L6cr3BWW4amGrhr5IWbOmraKrQ8icTkmO5WMModQZcc4izR4j2e0SMNVoKmBLKF1QOGO6BGvnBzdSvrxE3xHbduqwvN1\\\\\\\/K8OvbhQQNHfAmwxucwHrcMb0jQLeK4JECus8OuxvgzuhEJJtv4zT8tW8xDsVmmKGdTGlakVpXAU7ObmcGg\\\\\\\/Vkg6WmituI+67ou6RdYyMRCz9lipz0ZcThGDPGsZ2G4nSP2MGWF5W75UPhK5oZEiHPwZf+Msd4xUPGdCUtGQvnfHxIL55vkNU5751DozK3hZhSJ20rtLJeqOj6+MCG2y6dpOofCDlzkgOCS+ZhrfhbuEhnuCuRoPGspAmAE3L4\\\\\\\/AMsRzSs7MTdoKngOePvJ9mVeFI2h9W1gWxQVjMA3IuyK0HejrvxCSB\\\\\\\/DTgNW4lxxGDbSuoBOjG5BaqoFox2jv7tGeG2w6IL+hoy33Wsv3GLuQoNKEFPwxW0YdZwGEpAqvdHGbqBdZlxWvmOlpqjb+RttFW9SiD9JMQi3CnHasCRba3eoOaW68tT\\\\\\\/e5NLETC6JoOMjYqdJ6GDeIURzx5rYE0jGmLQJMyXiRVMVEr\\\\\\\/WIO44\\\\\\\/wGurjdy8Z68Wzusu\\\\\\\/1iMJM77km3MSB5TgyKwvBAGlxRRVoq3PRGQF8wbYuhAS0FJ6kdaLHoJXnhvLQ41ia0euGp6SK9omYp0aHwmqw8hZmehlTKYW3zsm\\\\\\\/TNPrqdrlPA5dhTytixh6C0omgnIz6Y28kB2VB5vNEL3e5o7TpqS5xwkbXUO6TogYyc1GPgY3KBFEofVSmt6jAH5AyhjJj0xe+DPTlEmqKUgMupgywDWHgmso\\\\\\\/eMKC8gU004uEnWzELmAHB7bxgoo7bd7UpK4MNmrgHPgix3NGFg7VwPbPKdQZ9F9li8vDUe5NmKRlCHuBMJPQDQ\\\\\\\/ev2kbZRGDG9AX7sDcIec5c9DQJC9eeZ4RQCunfod7R40MCqurP3zAiKmfrd01TQ52U2yjEjHtjiYQIs58hVyecdGOiDdSaIqKLLiVEqS5xZk2Hb092u4e05PxIP5ewP+7coiaHWhljQ6CEepGT+7sNpURDO4UbCg5ENEAWuRLrUbJ9YdiMYUe2IqL3AmJDhhGl5fTYBLQ00YvM\\\\\\\/iU81RNEKMzlww0W\\\\\\\/9UGGT6lItINHCRbfC\\\\\\\/SxnXwxsyvKyb3WWUo5V3GVbEc0jNyODmGpQvw\\\\\\\/TYJLXVKIbxeeK9Aq1eZJWR0mR9x0eXbZZgnOH82ebSFKsBJqwdGd+buhFBCYArAEJnmKylhm3P56AWBSX\\\\\\\/BCD4uGvJSy\\\\\\\/GqmqA68Jl5zCbi2WcQIiuJyQl4WEVpMyY8lAv4N\\\\\\\/dWZY8JrLjAuQOVqjBynEbZYn9T4JWyoVgTkVPOh6vZpkWId6K85tvZcxWo4jpUufsr64kaE1LVDGJdo\\\\\\\/PD2rk\\\\\\\/BN4h4ZmuC86Z6RkizL0lMx0zTyJmVNV\\\\\\\/m2yWw6SLAihvLI8n1cOF2zfGYodmN9UBrh6Z4sb6+KI4iwxdg+4h8qLETH7DbSEbzL3hNUTJta5Lax3jBk9xHOek2UUaXqKbUFdn6FHsaK61l1a7hzMp0R2GO+oxT7QSh5LuokrazSh4KSBkxcY5OmX8QhrYWp3OZoIuAbykX1ZdQIoNJ6VQnbQGfCWSJoa2QHpuBVRYjrijkQHXk6rNO8K+jbtVYF3jaVKXpheHRVNTrEUDvfdiKHxPePoGW2pNhOthz59ILdOwg0lpAYAzNsH3NvoQF4oGcTi5RjjlD4TbFaSjj4XF8aMNYXsNrDetB8wzqoGfixSWt3xosnFcLfCEDBd4Kz4vVMYtUuk0j2dPskY\\\\\\\/6f7JDPgXGJpS8bOxGNYwm9URUXMzzeQ3uBlrqoj\\\\\\\/r0xEvp6i\\\\\\\/+EgxCKtlxDoY5HNu8Ikjwq13eCJsgySvgxz+8zVakvxPNxRAcldx1hb2FXedFSXCwIsyq88xtXYMcnd+OfH6EAhhvXT5w8ahxCDUXnLPwC9wI09FpLLBX5tWXY5vDlZOnQidSbkgBtUtPWRN3MgruNhzE6aaZ5lyjsdLJhYJLirRxa0Y1P+TFAe2n08MBc\\\\\\\/\\\\\\\/LR\\\\\\\/j6KJOOiWfvldwtS8tEGf5C4b\\\\\\\/tWZ12bdlpPm4DelIFKGyxaI2qTWbnVFz5oIMLceLSTuZz1ClbJFFD\\\\\\\/n\\\\\\\/zdNezigCXPq+dpuW36vOus2Krqip9SiQzyQ5f85AiKJwUH98LQ5CZ9arJcSCPoVM7vuCT37JRTKgwR9+O65TiMz9PxfvYslmvlV3RPJtNwW9pB1I3emiULYEl1ZYHeKH177\\\\\\\/1ZwkDb51TYoOES\\\\\\\/RXIiG2v0X1qepxQWwnEdHgn2AfIWlrPs4fg1V3IufiqnOG0FcRlJ4\\\\\\\/jyHpS99HJ9xYKd8vTPP\\\\\\\/xyJ1xtOPdSEOmH+ccW7aJjwCw7PwTQF4vRVaB0O3A0iLeDPUVIfA+aUm5XE0Z2EgR3eOcH0poyyivnVK9vhSiXqNITA1okYVW1Cxeqi8C\\\\\\\/nQCPpJTaZQDcLusnrHV+TyybGAgdrAgnjXFIEZ1ZH1eIOyIMIURagSEixDzT4wYd9c9cCFQVU+Bp5mll0H\\\\\\\/ci1tUrHz+T48jtt7kj2TBxG6EHBRNY9E3bAfnbI9ZC0CFIvcWkKXuz0IR8qwXUEMkO8yLRcbAXjLnH3wESNJqx9YNAuPpjnG9MnXbHwr9tjIQI\\\\\\\/JL18yWIGK60bYsYv+eUp5jenbjR6Kyl9OQROpidGDGyiv0sWmthsWVeq4pA0Sv9bdYDp3dl6SS95A2VAiF0rDHXPP0iQC\\\\\\\/dY4PmgyM2IyvuhUG9VyyNj+hsIebvttG9UG5nqe1pbFNI3crlo6DxPjGrKg+FeE1tkzhXA4qUhwzFo\\\\\\\/sGlYttX0i6exHUS7FLv7I8Ek48LvDKNHoiigZztkRJFssMn7RC5VZgwi8czFSsi+UmALGwaEfHoAqQ6gMirda+BwUHKau\\\\\\\/taRi28UoGyCyYS9l3glfXODfgoG91d0to4HoDxwRmYTRa1hOqjhLjqkS4uAzk5+v0a04\\\\\\\/KMcG6dn7CPSP97CWEY3WF0OMFT9nnIFsM0VEaSvuU9fbcUtoln6efX+hH6S593eC8BAsrRaEO+qnezWMLn09D4DjTkocAN5r1vJaID7XAIr085MOQ93usl7Sw1u6VMuzXo4PnofLLnsH2QVtH\\\\\\\/87ti+41KO\\\\\\\/4K+od7tUn0\\\\\\\/s6rCQthL5\\\\\\\/jCydqKiMCFTv6NGUZPOzZIIRyG8zpTqYvmPB\\\\\\\/vF4xKMAs+PaWjM1KMfx7ovPC+HzdYtVffPrTVTT24Ke2Ps\\\\\\\/craUoVYou22\\\\\\\/N7QbhNRc\\\\\\\/1oX4R+dYgrfbLRIwhkj6ahDQ0F\\\\\\\/s0Wm4Dlo6NfnCZp8MU3UT7r20eGYBnwuNkBCOhO6xbHLzPrcYcdtT90SWxyfiSm0fuLthZEtSuHzm8r\\\\\\\/8+SkE+K5v3rShkTNYC8\\\\\\\/w1cbVXKEAM56fNhI6C2LlPQTvba8LbOEfhG61c8+2c2A+QvSevFog900EIK9gpOQ1f3U36zp5Mh2mhvFnO66C1Y6yRYJ1jSpJQB1IjABCRzLFfjPRAWpxo+JCLVvWzhrN9wQkcrYZvSCr81y1R+u5TylYr0riyWXtVIRZOM+xm60cc3aRajeyksHojPO3enaLEtz+rXSwjRp4gcwc0sMXL0p6gEN117fsrG\\\\\\\/AbDaE\\\\\\\/IOKnIhVbIR2IFcvUIUw7Y5sD3VD9vYiN+POsdU9syukQmzQQoi+3bi8vF1FGwvok74g6TSBs1SlSOT3wKhPQL5ZlTi4Exbdrj7CjGIeW5qBVCzuCL6NH9nb0GXLixEAuEuUZs2HCXNs9F5kiLVtopYQvX+RVBvRZXd5xasr0D7NY18vY9HKHz\\\\\\\/G6y5n02jTtZI5QZpEohozoVPDxVstVIxCVaRAJqCf0x6A3GxlzO7nzWNfcgzbdooSmmeOv1rP2htLr1FYkoA4PRpKWG9+y5ObnMjafiW5RgErowJcFVTndzuVPssvG8e2Sop9rpGDQFlcDTIACV5ro+vVhZQ5GB5rs9ilIhrDJFzKy3kxafcKoQl3dDnW9sx\\\\\\\/OfBUkr2YwOCSGhNvivQJ2Up5DXxR7moPW+gWW80qfIM6wNpzklkLRSzl\\\\\\\/aQ3bBZjRPZXh9geGJNb3uBa4hKZC\\\\\\\/jKUSzJ\\\\\\\/WXyShdzQpfRGAB+zMH6DJ6yEMvzpRWYQAqSzXOkT0Z1Tmgo1g2Lz0S9ZETMyt4MVssx+avWvjPftBRgXh9B4Vquu6zdI+A7Yp5w\\\\\\\/tvuCjp\\\\\\\/lh5D\\\\\\\/bVWRg6NoG1aaJ3px4VjJ4VDJVgZ+lmsDa1pfeN9sQTytNKezdnzKSAx8yhxloN3XEYjuqAfONnTjgtRNyu\\\\\\\/LAHVrdZt4N0FaSIDaAZkikW2KAvXFLSQROWBAO6GXtXMKq65D8WntWmnihXxwLfX7vc95kQ9WqTKOUp9a1UUqRoon51xkTFdL8RRndwfqEuIL2Nl48UrpRAjs+ZJNT4xvCmySVkYqK8jP3W2DYt\\\\\\\/NMCACzVz594JirHFb8Of4\\\\\\\/ubUskgjh7i3cZstwwJnfbeTb1aTFF8pvWlC64PiUZ0HxGnmqJolT6nw6MouCUTb7JJATpXyepPm7A8SXB9a\\\\\\\/KKopuHCT3\\\\\\\/dgs5JQzYSSHnkYUxAQTExUHgYDYQAXQG59FiW5HSymPQSsZH0JizGGM2jzaN1p3Vxq5yOHT2pyTfBmEXVoeM4Hc03uWTuiYnRCJk7EceuLCyT9KaFE0\\\\\\\/fUWXjCSJUOLzUQD+tkvx62Ol\\\\\\\/\\\\\\\/I7DlhBkHGOE7w8ykDuzwcV\\\\\\\/SoKXxmsDTDktQ+zqLbGwCiaF0d8Tt1r4kALSIHO5AdnpFfJ7myOhJUacxD4Ef1vbTAa+USodihRhf5ndnGwB6KY3gxMZoG72ogqXBVpuhyXJPL+WdaKu23XCDSCNn1FteRhWSEHRGTCwzCKPfifuMufbSRbQvZ9+ZQ6XThRjz4VQyJEojoTRbPZH2O5w8bJr4PIKMLwBNF9c2+Qjtpka+9XDCtpf+NYJAtPoA71jsmnHJInFpwja9y2aNNW\\\\\\\/luEjX08wDN2YtVxKrTPymVvKJ71cq6ZfvzmMFI7LySq17Z+LJXwQ0K1Ozgf8gj0KngMR\\\\\\\/\\\\\\\/N1nvKT\\\\\\\/Xe9AsNF7\\\\\\\/xPMMG7pNo1NV83WMzFexucwvi\\\\\\\/m8HGMraI28osvVrxQ\\\\\\\/snffT7TmXSyZpy3CJih1+NJVISnBS3RxZmwf+4FILe9dxI8YP0BMZ9gzdx4TQFZLHwuO2OR5holGapjrc+1CDC2Q92ZNNR2mMMpuxkMaH0Ia5vrOwqB+0rSwkNis5WnTILJjRsTcQe89ajvIa9QImxwqCBVzHYXf6nn2cfEqjB97V69JcsTNKrvlrvBEQwLGl9dFjBvqyBJGfpavRR+KPofJYvQ0bv+5B8sy7ojalU2L62taJkSSysktn8PRZeXi6vwKziWDgmQW57WbMmn7m7kl1OZxCk6B2PRNK5ixuyti5VwJKSl222CEPVvRr+X1jIOIelV7mHyMeqPlaGlVVQGcWrJbTOLy0mdmlEwhe0u+zvYdvUDzubGl2RikVrwksYCE5ZCF92YbunEgrF00FEwLwWa5HX0qu+ZmO20WQL50p7ucqjjCpa3Fg0ENqYWE8EAXdcIL5m2pyoYIEnt4ftoty7WTNYtOBt4tIASpDnIHi7QHm+LDl933uvaFGckm0102ZolHLcGDtSiBXpVq9VVz2YrreHbkifbYzEF9j5UUiLnSxTJgXBbw0EiLjoYUjARCIf8LxyIGo1zTJ3UfcJJ59PvwShi+x0yd\\\\\\\/iYToWylTopDBCyPJy3NySju2ykUZJuEEKbrOcUSZ9MScC0CeRi4i6CjCJOBEmkOBkppEd7hKz1fvFVLesR7V2V5Ejor4utR699uAjVInWU9Uz9viALTiIGo9JFGaOBbwcZXAlPzBjHc1sHtHPOyRr8DW8do6vj7DAoqfQABiTsJbU2bpDuanT7l1a2FrG6O7p4xikwEOrzzSiXXhkCZ6TpFLSDSRgPoih+ZzwWDGeWJyPwl3I\\\\\\\/c48I51b9v0QywdnMMUwa2Toxk4qC\\\\\\\/m0wuoiyToDDBhxq+6hx98GTkP5vW4ZinL5wbTBww2tdZ8GTtSUzXXOMN\\\\\\\/2t\\\\\\\/SvGuXEzMB1jOGCzIQbArSCnFhv845ltyZyrCe3QVSQy\\\\\\\/\\\\\\\/cseVU2hiQ0n03HsjsKrpLkQL3EMG57WfXSVROdWlEKNtc\\\\\\\/RjIqvEaWYB+xIDSSCriu+gSSvjLptIzdvpYktD1DTWjDCdBg4TSSHrC1xmafpx8SfGRf0Om1ROSJHYWMGmjukx55aYfVVSz0AXrZeqn3n+rQAY2i301VEGN2desJo64bCpEmAfjRhNJ9BmY9wT1DPLB1NOjKYI3iT3JLagjnSGgowQCoUVyaA3z74+S7SjNBtpGv66aZoKYF\\\\\\\/oVil72w8yosdJVnZ9JkMY0S6FrVt\\\\\\\/k1rzLJ2gRZYSItV0iJvc3Xl0MIYI8nyIBo4P1c9TUkn5gZDP5gn82+G9mBFerTBw\\\\\\\/rwxuja2J+wb5VRP6gRYxT5yX6DmhNe1SFWImiWTKWOOePvhikp63ZKu6hGojO2UbD4RwjtCC54Kmi9hm+hw3CGWmxjyWFmlKMNi4kieegr1t1Lodu18fwOC4fTiO96vbxmoPCuuJYvp7Clm7UwaA0liJch4p+6YdPXMDKrnRuCLtx4e6MGegqEGJSSVxtao32bbysgMcXVN07T3rYSkOMw0aMFfC\\\\\\\/IP6BqN150ta7nsMvPflMBbDucoOc6FNfbueqVIRAzbqmQKoVK4FRM6pf7b+KzzwMT3QYEqNQwMkNdSB\\\\\\\/43OhJpNr8SCg4HU0mmv2WpgYCO6YcSqzcybl5jLGQafwqswb9SI4+FA0XO3CrEUJzIW7gZqFXv1C0tlLWfj4oVfOYl38oITHeeP+\\\\\\\/ZBnSPQaKimuZT54gESoenon22O1q9nPJUtp+SXeKgFVni0afVIsY6kQdB\\\\\\\/8O8w1S1mroUtSBJLgbVwGU+RhsEw678CvSdqQcwdRFd8StgoeT3ZnYcFoTULYoImlFMcCLamTDefqF8NaNYMGPm0CAxyHcr+Qgmz1ZYOLRqPdqh35NKSapNX0rBqfLceOhmoWDQy5WmccHkF3N8NhTmRRnZ+cbUj7bJbkyULQ+QxmlXwt8y6SW4aFSMPOIEamMTREfF03e3lLhl5tuuOciJrn0\\\\\\\/NU4QYHuAYAadix+gKECrh52OxrK\\\\\\\/USI3zKbWNiqd7s8GhSamSVlFuul\\\\\\\/HeIivpnGPBD83kOXyPkp20N7e6eHE26QEDEWlBvXgXFnLDfhXn06IuFFgZypUwjBmhK74NqzdXLQ6DoRDhpKS0ngNK0hu8yweH1AhD6JxEhhXlyszmWdh9LdnZucfz0LZbbkiYOjaDzQ+tr2wr3gj3uR+rOllduNjSRSGjsrZtlS6KKgA2RDiNN9MdnB3s9aX6+8l6PGDPJSlaxxQvXBm8etiiJOTgdqzzpWr+5jQP4izJS0XwuFqVGfWwlUGsV+a6vQQ8Jwwv\\\\\\\/t97JSi4KQjsBg0zIDgflZqZ2eKZ+PXmSHl6MrbHo5Ng3ZJsQ2SGgwrtHV9P\\\\\\\/x+es6+hSwleo4cNDRmENrcmngSdU+BAWLL1ulordWw9N0Gj5VVKvIcrcDA1dbPoLkStTda63PoXK+A3eBAL5S0dQQgZo27+PolMyHxg3UCjfZj128W9NEPS4uwJw3sehknZbN8eKVO7ZcvxSYy7vB1rEbr5DtgYhDJfufTfFJGvUs\\\\\\\/3hdoFKe+9qgJAoq2c7Tn\\\\\\\/J04hOFmO2f8Fs43pgHezMlgsvz+WIldDE3uz0nQfCOip4M8BZRCYzB35W1Z82ItojWkeTy6laBuuCZbIDPqcMJTac0tG\\\\\\\/DSMsLSrI4qrFNuLJhkFhpoVn9dtRS2ifjEC5l+Zkgsij5syEEd387nEVLA6AXIx1VXBLtyF5kbhN22noZTQ1HfSXpWeKeVTd2bPMEC6BxjnmjJ\\\\\\\/WXzQA+GjItaJnAjH0iKI1CgZ75qMY9PTe5FYWU1myrUpTUAFLs4Q6bx+m4B9DbhvdKvqP0MP31oD8O5bPaT3s4\\\\\\\/uT5u0eVvpblgc+K8O\\\\\\\/\\\\\\\/V1blccvF8ZrEbKme0RNz6OCXwQUVFtFyjmWswYneXp9SVAmCLis9BngM5f+hlm4uXMRdzPLWM6UyJmWvjRdXR2FXQNNuqIZ5vjDMNEfbmirqm8mfh6OIRLPZzWsAbfs8MVovD6uXv8HNw+2YqLC1I19AD1MkEfKLJVXAEVuzXW9vOrdeQIk+GB6+jWBR1xKdYTfVgd97ixHhs1H085ofDHXf6k4zwstm9Zkl4wdCSVBTrSYRnUso283lib4nEQ7J2ojrPl0zJXt+Rk3mAg7qKkO6M\\\\\\\/\\\\\\\/RUJXlwGBC++Kui+BCeB27\\\\\\\/OvqKiszuELAXHaQpnAbVrr3YDkBOtcX9OIaXpPPd09Zjs3vqUAhm7INXlaQLeWL67hhVlArDEDCfeVcTWkoF2n9xN5c0gnySyhoRvAV3yaJl9hcGmwsNiEi4P4+U7\\\\\\\/hjbZo7c2zZgKFmYMrh71BX7vqEXfwdTR743VZZv+hbNZkxFnmnmc70UWV7jj4hHnSPJyvAgmckQEQbpqtMCIvH\\\\\\\/U\\\\\\\/s0pN28M89jXtuHXEqLUUB+DZ10JxX\\\\\\\/gLQQ3ZVqBtrf8WB29MlUnfvTP3FDjUHV5E9LiP7eiboMQiFEe35BBTb1bDW0PZEJVmjhETiLIaTcpHxABDGBxejYzK2zCUyFHfmgtEFJ7LMfkhFm8CQWmTId1GmQGZCtdP9LhkheOlSFWMEmafEs0F4iToTzY0g0PoVhDAcrahS6fhpgnAxX1dFMwuT\\\\\\\/onQ1W5jn3GcAIsv8ACWz2gbDpj8qe8eHUVqERMp4bcOcuWUefkNJ\\\\\\\/fsz+tVDOF1X5P9XK0a9K7BqE4RwdVo0zOh0kjTyvt3+qaMVDIMw4aTeuFzvIY7+33A7bJUIXwOAJukL1y8bQTFQGA0lao8Ce+Qmfm7lg1bCAPfP5I\\\\\\\/JJYMWOqFn4At5c8msZNqA7r\\\\\\\/iSnR7v+e1ZFwqBsvAPVyR5Pr6FEUv64mo8\\\\\\\/s+AdLwMpe7qzDM3R8hPAtbcZ1KgoWw1XK5FVzC\\\\\\\/IIzyoFV0WjUH+Jmza4qy8yFgtViqFVyCcHNznqSwsvLDt6rvMGj0yfmPvypKLQElTk23KjLyP+nadeVx7S\\\\\\\/yE1jeB2nkbP55f0bDi6fgItuISooWnjDlAeaUPsmnD9xrYZwRxPIMwqzazbhD63+Q38CLfY8OLrF05cuOZFNtKs5VfqUNBlT+MfNuPuWapSWSKXJ5Vm+Tvpj6qyop3TRBpR\\\\\\\/wGvslGONAxtOCEpIQD3QAB7gHc9\\\\\\\/lP4fpr\\\\\\\/l5YiOw8iV+zJk68PmqkIMHHCbhBHWP1eSmyZOfozkzM+m86u7A4sP5hysm4jWERaIxj08eU4iFdcAz8MiSgcb3NUTbYzRgpuniES0FB4Wozpj7FGKtB\\\\\\\/LQDUkSJiPiN6VAiH40hxFg6Zlgomxv5C+Glg4MIF0PDOzsGLOSdlbZLwamjjO5An+\\\\\\\/meI3WimfL7IxwYZmKWooCan8UMzMcSjdF2J8gHla\\\\\\\/qjU8d3u6TB0CZkO4zJcrpJqvPB++BgIs52HjnYGFaVQ0HmV9CggO6Sy4oZMLH8299Nlio9sSAmAknRO4n3Hf7OyXezo+RqnlEKMKixot+ykfupTa1cA8hJNrDwF7ja9F8j8IvcmVjdT5xd5R1u6TpZD3SRh9DnZlJ5+U8QTR53NSiyTO6v5tezN7KDYHK7VrFKYSVFlsX69p26+p6PRYO3Epg6MxJe77a7jr95QaANKWxZc8XooZEvi52lqm785al36JSZHV1KuDAYHaCEMStzzZkoTC1N3h5qZ5ZN1OTBrso673B5B8cOdd4B+Zk\\\\\\\/hp19Ni27XtbVYXZkNpYuUdJ9A2BfwG\\\\\\\/OHb7jMV\\\\\\\/dbp1eD3z0pjb6LJiMDEt327GW5UH+omEtEtm4YDLkc+H1uLPS\\\\\\\/kzYJhlhmKdDkuHUDySvAdxLEUGjE9ov8l+dmEq2FM6vR+gxZ7VVY5hWUqyZK7AQ74ntaro7z2eP5qHGhip6YtAfOGdz+sR65rmbB6x4sT9tUtMvmCTcXa3uunRVUKy78P\\\\\\\/ak6Ptoxf+Jm1y9OJ4oJmX5FelmDFy\\\\\\\/3p7uAYVqD+2BCQMeHnMV2dl78S9ksyShbO+cInLfxPbT14Xt9AqKgoWyqRd2fUMvEYJN5553Mt93GdNxioJ1rEeoPIidU2ZudxrFzIC1r2U6WJeXR1+6lq89sbwbn2IeyNM8DWLbcyagVlPG2GnMgHGcjP7a796gyCZjbk7oUy5ZlBUOV3BAeS6wtVDkFaXs2z3BBuTU4X3sXLOJQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022PP1K7KOzTtDt60+L4hLB6w==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022a2de12f969116963238ab6e8129362e43093855c537ef3e2e85df0ab80eb03b7\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022cNYFCLMQQDB6ypkehxMU0UhiLBXWuCZ16ZuQPIf5sth+FrLqj2iH3B19kGC+Xv88Q6CN2S+LQao6ba1VM9o2vvoeprn13yHTWiQUzIi+OBdUsDyWeJAcVeeAK8R5qLJt8cxkbMEWeh4jzwoIRM\\\\\\\/PT9A1YbvdCtm\\\\\\\/b3fUILGO6lkeNv+W8AK66qQuvlOSH5ZZ0Yu4Rfm0zVFGHhafRk5uypPOglZPr+h6G9pk8KnRkVmjQ79L9uRPMZck7jyXM1ZV3Zc6hDGfZI4oSeq2vLjnthjraa+gjgCy8303KmZSJbB9HEikC5l5gcaV6GV7GtlzQds+Ah5Q6NlQSvzX0RS5iuNVxEy1m2ySiC7cpLLzvgee4OelKxFF2ZYeSYnmOkZbXqS+7FasggcrNBIXciIQ9LlIQt60+IGZ6ZzfSIme1YlVDok2nBzxAvs9NWCNCxsJ3RAtCYB7zbHBTf4ozHljrrrMfFBjH4tBgGbVxrvsAZVszVr8UWaYoZFfbQzUBcx+PJdBUg6y2zfX+EZs5Q\\\\\\\/+PYLz5hc2\\\\\\\/xoxfrS9b\\\\\\\/SACU6gsgK\\\\\\\/BMXGtpTbga2ysYxJ7\\\\\\\/SXOTbN\\\\\\\/OJiOFjElwH+6nxJLql5s+phkVU3wvXTGX4MtNZzWFURFINwJE7fWv\\\\\\\/7X\\\\\\\/ySs876E4CAwinxOGjVQOnR2IRL0KGCaoOD1pEElcUhecqr8V9O9Mvr00fQ790aYxb6H5vJVfNnZJZIrxj9g8z3yZeCJG9eAMw27Si0c1DEopIm\\\\\\\/rCVkmvTUTSnDKiDM47M8E6vvak4xVjgn27pI+UFA5Cjc2d7x+u4J4DFuUkrHktCqpmmfCiUFXXs0g5+\\\\\\\/XYkUXqjnNzMhlTPvodKZuKUkwG\\\\\\\/tN8Sqvwzu4Y9W7yp6ZXfsyElkpMTVixoQslNxOZWFNFpDRC0MT5um3lfAoDg+rnDs0cSXIaHT0vhIUW4U7a2OHbJ57XgGrvbmOmiSs+HwGBJqxyB4lehFgozZ99zGSsTZ2lXB1m6DEPbbdwSuoufpk+pY8SGhP2JaM03oowe5KDQVuSU+WsO9Cv5\\\\\\\/6P3sl46+pCPowTTEBwF4Iqz+Nsb0cKohoVpVbXBmidx9KINIwrx8OFgeJFVU\\\\\\\/tsWxmoyQu7XNSpWj1HQCWFeTOT4G2YLsfRHcYFJGMRymOaFdZVWBy0\\\\\\\/XSj7HydzsqgJTHvFoboWkP40uJtqAQ1HZSsZiACNaHXwOmtgW+qB7\\\\\\\/PYed31pPdgyJwW0mzjCNEX2cLx8ni2hGnvC59uffCkHuoTc7lneYhKNWpOZHKqljvZd3zZPtJcBin4wDGSgaUUYT2MH2\\\\\\\/uACUIHgivHxEJtECOlHl+M4VYtpFVNv4r6gm4iGCq4hrSFY9EySk7RRMZlfqNX5lYall0IVgafbq5I3biNYuqhiQ0wqfoZU9pT4Rqpmsn9oXLhGpMQ2f1fQasRI\\\\\\\/QCY\\\\\\\/4x1eX4c31Kb3qQXMWTp0RkmDlxYfFK3L6i+de\\\\\\\/rMDvqvZ0U\\\\\\\/gvmUtJ86EHrrROMl7Df4d\\\\\\\/1pb8xclLuYhPmD+M\\\\\\\/oEwzysSHbiWqnJPGXN0nbTk08JqUMn270+O7AU0HEqQBoWmzI0mWEfLM7zwswUSsr9mri2Amdi3ukn4QdWLVk3mqXeKri+iUt7yFucM3UIiHNdr+WB2rVghf5vM7fTWLUnOGm2+SqnZXIBYrEhwKonvLVOHjRxUBHlCwU5uQ1ChyVcFeIyYtKSZvme1zkJtaQY8BoBOe9FKnuiRDczx9xHl46J+lFgWS5rkJxtSStVscsmlSHn7w3oafrkdoOgN3G5r\\\\\\\/9C1i5LkOfxMqkoUR4q8QZ+JSrUORT3o5BRzKdmau6WPn1F3jj7gLWrpOs2TOmGcNf32s8a01ZYqrFdx3Sw3lsLHYZl8mql1EnMMyD73zb3qY9KJPLaAvUveDrKiQpUeqfvQnbOl6g\\\\\\\/vEZjO89Yxw0SEA6Su8ec6dOOGQb8GL91vplE5qtSvBwueq7QHEfLjaSnyTt\\\\\\\/\\\\\\\/9MsKpdv+rNhFdUavibZy3ONA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022nQPc7FGAGEeHN7sokXPxYQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002245e83c78106e726cd39a6326fe819a30f60b5fdf5b6f2a3354ff696b5152a5c5\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-494279 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/10/3395" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Pti90t-nor1deBw_rovO70AYvSJiGxz7fPj1Gp5M7II" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/10/3395" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-ZTpUQBCLEhiVktW12voOAvlAKqJ8Eky7pcwVOBcAbeE" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/10/3381" title="A Bayesian Genomic Multi-output Regressor Stacking Model for Predicting Multi-trait Multi-environment Plant Breeding Data" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/10/3409" title="Utilization of Tissue Ploidy Level Variation in de Novo Transcriptome Assembly of Pinus sylvestris" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494279" id="node494279" data-pisa="ggg;9/10/3395" data-pisa-master="ggg;g3.119.400605" data-apath="/ggg/9/10/3395.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Evolutionary Dynamics in Structured Populations Under Strong Population Genetic Forces</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-2915-089X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Alison F. Feder</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-8704-6578" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Pleuni S. Pennings</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0001-7476-9283" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Joachim Hermisson</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-3664-9130" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Dmitri A. Petrov</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3395-3407;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400605 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494279" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Alison F. Feder </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, Stanford University,</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Integrative Biology, University of California Berkeley,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alison%2BF.%2BFeder%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Feder%20AF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAlison%2BF.%2BFeder%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-2915-089X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Alison F. Feder</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:affeder@berkeley.edu" class="" data-icon-position="" data-hide-link-title="0">affeder@berkeley.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Pleuni S. Pennings </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Biology, San Francisco State University, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Pleuni%2BS.%2BPennings%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pennings%20PS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APleuni%2BS.%2BPennings%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-8704-6578" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Pleuni S. Pennings</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Joachim Hermisson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Faculty of Mathematics, University of Vienna</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joachim%2BHermisson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hermisson%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJoachim%2BHermisson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-7476-9283" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Joachim Hermisson</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Dmitri A. Petrov </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, Stanford University,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dmitri%2BA.%2BPetrov%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Petrov%20DA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADmitri%2BA.%2BPetrov%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-3664-9130" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Dmitri A. Petrov</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/9/10/3395" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:494279" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:494279/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/10/3395.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:494279" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:494279/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/10/3395.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:494279" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:494279/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/9/10/3395.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:494279" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:494279/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">In the long-term neutral equilibrium, high rates of migration between subpopulations result in little population differentiation. However, in the short-term, even very abundant migration may not be enough for subpopulations to equilibrate immediately. In this study, we investigate dynamical patterns of short-term population differentiation in adapting populations via stochastic and analytical modeling through time. We characterize a regime in which selection and migration interact to create non-monotonic patterns of population differentiation over time when migration is weaker than selection, but stronger than drift. We demonstrate how these patterns can be leveraged to estimate high migration rates using approximate Bayesian computation. We apply this approach to estimate fast migration in a rapidly adapting intra-host Simian-HIV population sampled from different anatomical locations. We find differences in estimated migration rates between different compartments, even though all are above <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-1.gif"/></noscript></span></span> = 1. This work demonstrates how studying demographic processes on the timescale of selective sweeps illuminates processes too fast to leave signatures on neutral timescales.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/approximate-bayesian-computation" class="hw-term hw-article-keyword hw-article-keyword-approximate-bayesian-computation" rel="nofollow">approximate Bayesian computation</a></li><li class="kwd"><a href="/keyword/fst" class="hw-term hw-article-keyword hw-article-keyword-fst" rel="nofollow">FST</a></li><li class="kwd"><a href="/keyword/migration" class="hw-term hw-article-keyword hw-article-keyword-migration" rel="nofollow">Migration</a></li><li class="kwd"><a href="/keyword/rapid-adaptation" class="hw-term hw-article-keyword hw-article-keyword-rapid-adaptation" rel="nofollow">Rapid adaptation</a></li><li class="kwd"><a href="/keyword/simian-hiv" class="hw-term hw-article-keyword hw-article-keyword-simian-hiv" rel="nofollow">Simian-HIV</a></li><li class="kwd"><a href="/keyword/spatial-structure" class="hw-term hw-article-keyword hw-article-keyword-spatial-structure" rel="nofollow">Spatial structure</a></li></ul><p id="p-4">A population’s structure in physical space can profoundly impact how it evolves. Structure can speed up or slow down evolution (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Barton 1993</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Whitlock 2003</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Martens <em>et al.</em> 2011</a>) and cause populations to reach evolutionary outcomes unlikely or impossible in well-mixed populations (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Wright 1982</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">McLaren 2016</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Bitbol and Schwab 2014</a>), including the maintenance of genetic diversity (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Karlin 1976</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Allman and Weissman 2018</a>) and the prevalence of cooperation (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Hauert and Doebeli 2004</a>), clonal interference (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Martens and Hallatschek 2011</a>) and speciation (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Coyne 1992</a>). Understanding population structure aids our interpretation of genetic data sampled from around the globe (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Bradburd <em>et al.</em> 2016</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Novembre <em>et al.</em> 2008</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Berg <em>et al.</em> 2019</a>). For these reasons, understanding and quantifying population structure has long been a goal of population biology.</p><p id="p-5">Despite population structure’s importance, classical population genetics results suggest that even little migration among subpopulations destroys much of the genetic evidence of its existence. For example, under neutral evolution, long-term population differentiation is a function of the product of effective population size (<span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-2.gif"/></noscript></span></span>) and migration rate per individual in the population (<em>m</em>). Specifically, Wright showed that population differentiation measure <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-3.gif"/></noscript></span></span> is equal to <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-4.gif"/></noscript></span></span> (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Wright 1949</a>) in an island model under neutral equilibrium. Therefore we might expect that when <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-5.gif"/></noscript></span></span> is much smaller than one, local drift overwhelms migration and <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-6.gif"/></noscript></span></span> is substantially elevated above zero (with up to a maximum value of one under certain conditions) and populations appear spatially structured. However, when <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-7.gif"/></noscript></span></span> is much larger than one, migration is effectively much faster than drift, <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-8.gif"/></noscript></span></span> approaches zero and the populations appear completely well-mixed. Although the island model framework has been criticized (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Whitlock <em>et al.</em> 1999</a>), the result is a population genetics classic and underlies all population genetics theory (and subsequent applications to data) under the modeling assumption of panmixia. Examples include detecting barriers to gene-flow by using patterns of local divergence to deduce locally reduced levels of genetic exchange.</p><p id="p-6">Importantly, Wright’s equation describes population equilibrium. In this equilibrium, even very few migrants can mix a large population if gene flow occurs over timescales of <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-9.gif"/></noscript></span></span> generations. However, a different picture emerges over shorter timescales (<em>i.e.</em>, less than <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-10.gif"/></noscript></span></span> generations) where ecological processes can have considerable non-equilibrium effects. For example, an ecological disturbance in a population can drastically and immediately perturb allele frequencies in different demes. The duration and magnitude of this departure from equilibrium will depend on the specifics of the disturbance, and the rate at which migration returns the populations to equilibrium.</p><p id="p-7">Non-instantaneous (but still rapid) perturbations in allele frequency that occur on a timescale much shorter than <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-11.gif"/></noscript></span></span> may also leave transient patterns in population differentiation. Strong positive selection represents one such process. Importantly, this holds not only for heterogeneous selection (which will lead to permanent differentiation patterns), but also for spatially homogeneous selection, which we will focus on here.</p><p id="p-8">Most straightforwardly, non-equilibrium population dynamics present a problem for interpreting population differentiation statistics, such as <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-12.gif"/></noscript></span></span>. Fast processes lead to signals of isolation which can be misinterpreted as weak migration or local adaptation preventing gene flow (<a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Whitlock 2003</a>). However, this also presents an opportunity: with dynamical data it may be possible to track non-equilibrium behavior of population differentiation over time after a strong perturbation to estimate migration.</p><p id="p-9">In this paper, we develop intuitions about subdivided populations adapting rapidly to a new selective pressure and connected to each other by migration over short timescales. Although our theory is more general, we start with a concrete example of how Simian-HIV evolves drug resistance across multiple tissues. This example demonstrates how strong selection, abundant mutation and fast migration can interact to create patterns of substantial yet transient population differentiation over short, clinically relevant timescales. We investigate the parameter regime where these forces are jointly strong and find that when migration is significantly faster than drift but slower than selection, characteristic, non-monotonic patterns of population differentiation with respect to time emerge. As a proof of concept that such dynamical patterns might be leveraged for estimation, we present an approximate Bayesian computation (ABC) approach that exploits population differentiation over time to estimate migration rates far above levels possible to determine with drifting alleles over long timescales. Finally, we return to the Simian-HIV data and estimate between tissue migration rates unresolvable in the neutral equilibrium regime.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Simulation</h3><p id="p-10">To gain intuition on the dynamics of populations under strong selection and migration, we performed stochastic simulations. Two populations of equal size <em>N</em> were instantiated with no standing genetic variation, under the assumption that beneficial mutations are costly before the change in environment.</p><p id="p-11">Each generation, four steps are simulated in each population to adjust the counts of the alleles in populations A and B respectively, <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-13.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-14.gif"/></noscript></span></span>:</p><div id="sec-3" class="subsection"><h4>Mutation:</h4><p id="p-12">Each population draws a Poisson distributed number of mutations (<span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-15.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-16.gif"/></noscript></span></span> (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Abram <em>et al.</em> 2010)</a>) which land on wildtype backgrounds and each create new, fully identified genetic lineages. All beneficial mutations confer identical fitness benefit <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-17.gif"/></noscript></span></span> relative to wildtype.</p></div><div id="sec-4" class="subsection"><h4>Selection:</h4><p id="p-13">Counts of different alleles in each population (<span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-18.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-19.gif"/></noscript></span></span>) are normalized into frequencies <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-20.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-21.gif"/></noscript></span></span>, respectively. The frequency of each mutational lineage is deterministically adjusted based on its fitness benefit: the frequency of each non-WT allele is multiplied by <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-22.gif"/></noscript></span></span>. Then, all frequencies are normalized so the sum of all alleles within each population equals 1.</p></div><div id="sec-5" class="subsection"><h4>Migration:</h4><p id="p-14">Allele frequencies are deterministically adjusted based on the migration rate, <em>m</em>:</p><span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-23.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-24.gif"/></noscript></span></span></div><div id="sec-6" class="subsection"><h4>Stochastic sampling:</h4><p id="p-15"><em>N</em> alleles each are sampled from <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-25.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-26.gif"/></noscript></span></span> with replacement to produce counts for the next generation, <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-27.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-28.gif"/></noscript></span></span>.</p></div></div><div id="sec-7" class="subsection"><h3>Approximate Bayesian computation</h3><p id="p-16">We employed approximate Bayesian computation (ABC) to estimate model parameters from simulated and real data (see below). We performed a two-step approximate Bayesian computation procedure in which we simulated <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-29.gif"/></noscript></span></span> forward trajectories with uniform <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-30.gif"/></noscript></span></span> priors where <span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-31.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-32.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-33.gif"/></noscript></span></span>. We restricted our <span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-34.gif"/></noscript></span></span> prior to values expected to produce mutations that would then have time to sweep in the population within the relevant time frame of 100 generations.</p><p id="p-17">We used rejection sampling, which accepts a certain percentage of trials (given by the tolerance) that minimize the Euclidean distance between the observed summary statistics and those summary statistics generated by the prior as implemented in the R package ‘abc’ (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Csilléry <em>et al.</em> 2012</a>). The parameter combinations that result in the lowest distances form the posterior. We first perform ABC to estimate the posteriors for <span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-35.gif"/></noscript></span></span> and <em>s</em>, using targeted summary statistics as discussed in <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Aeschbacher <em>et al.</em> (2012)</a>.</p><p id="p-18">For <span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-36.gif"/></noscript></span></span> we use the best fit <em>θ</em> under Ewens’ Sampling Formula for when the populations are combined at each time point. Note, combining two population allele frequencies with limited migration overestimates the value of <em>θ</em> (<em>i.e.</em>, increases diversity relative to a single population), but this does not affect our procedure because the best fit <em>θ</em> is used only as a summary statistic.</p><p id="p-19">To estimate <em>s</em>, we fit the observed frequency of the derived allele at the sampled time points to a logistic curve with y-intercept of <span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-37.gif"/></noscript></span></span> that minimized mean squared error from the observed data. In the cases of ties, we used the shallowest slope that could explain the data equally well.</p><p id="p-20">From our first fit ABC, we get posteriors over <span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-38.gif"/></noscript></span></span> and <em>s</em>. We then take the range of values within the 95% posterior for these two parameters and use those as a flat prior for a second fit (see Figure S12). For our second fit ABC, we use four summary statistics at each time point: <span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-39.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-40.gif"/></noscript></span></span>, the absolute value of the difference in heterozygosities between the two subpopulations, and the number of variants shared at any frequency. For a schematic illustration of the two-step process, see Figure S12. This two-step procedure drastically improved our estimates for <em>m</em> as compared to a one-step procedure (data not shown).</p></div><div id="sec-8" class="subsection"><h3>Computation of summary statistics</h3><p id="p-21">Compartments <em>A</em> and <em>B</em> were sampled with allele counts for the <em>l</em> alleles at the beneficial locus <span class="inline-formula" id="inline-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-41.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-42.gif"/></noscript></span></span>. The total number of alleles sampled in each population was <span class="inline-formula" id="inline-formula-41"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-43.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-42"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-44.gif"/></noscript></span></span> with <span class="inline-formula" id="inline-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-45.gif"/></noscript></span></span>. We use the <span class="inline-formula" id="inline-formula-44"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-46.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-46.gif"/></noscript></span></span> definition of <span class="inline-formula" id="inline-formula-45"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-47.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-47.gif"/></noscript></span></span>, based on the difference between within- and between-compartment heterozygosities:<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-48.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-48.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-46"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-49.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-49.gif"/></noscript></span></span> is the average within-compartment heterozygosity<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-50.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-50.gif"/></noscript></span></span>and <span class="inline-formula" id="inline-formula-47"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-51.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-51.gif"/></noscript></span></span> is the pooled-compartment heterozygosity<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-52.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-52.gif"/></noscript></span></span></p><p id="p-22"><span class="inline-formula" id="inline-formula-48"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-53.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-53.gif"/></noscript></span></span> is a renormalized <span class="inline-formula" id="inline-formula-49"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-54.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-54.gif"/></noscript></span></span> based on the maximal value it can reach accounting for the most frequent allele in the population<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-55.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-55.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-50"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-56.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-56.gif"/></noscript></span></span> is given in equations 15 and 16 of <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Jakobsson <em>et al.</em> (2013)</a>.</p><p id="p-23">The difference in heterozygosities is computed as</p><span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-57.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-57.gif"/></noscript></span></span></div><div id="sec-9" class="subsection"><h3>Model evaluation</h3><p id="p-24">We determine whether the posteriors from ABC are providing useful estimates by comparing the posteriors to the true model parameter. The most useful posteriors will have small variance and be centered around the true values. However, using either one of these metrics will lead to misidentification. For example, if we evaluated a posterior using the proportion of the time the 95% posterior interval contains the true value, a completely uniform estimate across the posterior should contain the true value 100% of the time while offering no information about the estimated parameter. Alternatively, if we evaluated a posterior using the size of the 95% interval, a very narrow posterior far away from the true value would score highly but provide misleading information. To balance confidence and accuracy, we use the mean-squared error computed in log space (as our posteriors are uniform on a log-scale) to find the average difference between each of <em>n</em> points (<span class="inline-formula" id="inline-formula-51"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-58.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-58.gif"/></noscript></span></span>) in our estimated posterior and our true test parameter <em>x</em>.</p><span class="disp-formula" id="disp-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-59.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-59.gif"/></noscript></span></span></div><div id="sec-10" class="subsection"><h3>Simian-HIV data</h3><p id="p-25">To explore a structured population with strong selection and migration, we analyze macaque T98133 from <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Feder <em>et al.</em> (2017)</a>. Briefly, a macaque was infected with RT-SHIVmne027, a simian immunodeficiency virus with an HIV-1 reverse transcriptase region and treated with FTC, a reverse transcriptase inhibitor. Drug resistance mutations to FTC occur within the reverse transcriptase region. The viral population was given 12 weeks to establish within the host before treatment began. Viral samples were collected from the gut, lymph node and blood plasma at 1, 3, 8 and 14 weeks after the onset of treatment. The authors performed single genome sequencing of the reverse transcriptase region, resulting in 800 bp regions containing relevant drug resistance mutations and complete linkage information and have approximately 30 sequences per time point and location.</p><p id="p-26">Generations in HIV/SHIV are approximately 24 hr, so we translated weeks 1, 3, 8 and 14 to generations 7, 21, 56 and 98, which we rounded to generations 5, 20, 50 and 100.</p><p id="p-27">We called derived haplotypes as unique if they contained a drug resistance mutation (M184V/I) and any other mutations at the initial time point at which drug resistance was observed (approximately generation 20) in at least <span class="inline-formula" id="inline-formula-52"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-60.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-60.gif"/></noscript></span></span> copies. Note, we replot <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figures 1</a> and <a id="xref-fig-7-1" class="xref-fig" href="#F7">7</a> with choices of <span class="inline-formula" id="inline-formula-53"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-61.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-61.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-54"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-62.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-62.gif"/></noscript></span></span> in Figures S8, S9, S10 and S11. 95% posteriors across the choices of <em>c</em> are also listed in Table S1. We excluded haplotypes appearing for the first time after generation 20, to avoid mutations occurring on the backgrounds of existing lineages, which our model does not consider. These later time point haplotypes were clustered back to the most common early time point haplotype with the minimal mutational distance. For example, if generation 20 contained three haplotypes with the mutations A, B and C: A, A+B and A+C, and the haplotype A+B+D was observed at generation 50, it was counted as haplotype A+B.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/10/3395/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Dynamics of drug resistance fixation across space and time in a treated Simian-HIV population. The top three rows shows diagrams of drug resistant haplotypes spreading in different sampling locations over time sampled at generations 7, 21, 49 and 98 after the onset of selection via the drug FTC in the gut, lymph node and blood plasma. Each color represents a distinct lineage separated by at least one mutation. The bottom-most panel plots pairwise  between pairs of sampling locations." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-146688292" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Dynamics of drug resistance fixation across space and time in a treated Simian-HIV population. The top three rows shows diagrams of drug resistant haplotypes spreading in different sampling locations over time sampled at generations 7, 21, 49 and 98 after the onset of selection via the drug FTC in the gut, lymph node and blood plasma. Each color represents a distinct lineage separated by at least one mutation. The bottom-most panel plots pairwise &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-63&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; between pairs of sampling locations.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/10/3395/F1.medium.gif" width="169" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/9/10/3395/F1.medium.gif" width="169" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/10/3395/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/10/3395/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494536" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-28" class="first-child">Dynamics of drug resistance fixation across space and time in a treated Simian-HIV population. The top three rows shows diagrams of drug resistant haplotypes spreading in different sampling locations over time sampled at generations 7, 21, 49 and 98 after the onset of selection via the drug FTC in the gut, lymph node and blood plasma. Each color represents a distinct lineage separated by at least one mutation. The bottom-most panel plots pairwise <span class="inline-formula" id="inline-formula-55"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/F1/embed/mml-math-63.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/F1/embed/mml-math-63.gif"/></noscript></span></span> between pairs of sampling locations.</p><div class="sb-div caption-clear"></div></div></div><p id="p-29">No additional known drug resistance mutations appeared after generation 20, although positive selection may still influence allele frequency trajectories.</p><p id="p-30">When we compute <span class="inline-formula" id="inline-formula-56"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-64.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-64.gif"/></noscript></span></span> pairwise between anatomical compartments (as in <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1D</a>), we treat each haplotype as equally different from every other haplotype, despite some sharing more alleles than others. For example, M184I and M184V+N255N+D177N and WT are all equally different from each other. We do this in line with our single mutation simulation model, in which all haplotypes are equally different from each other. Computing differentiation statistics that take into account conservation does not qualitatively change the patterns, and this question is considered in much more detail in <a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Feder <em>et al.</em> (2017)</a>. This convention is also used in the computation of all other summary statistics.</p></div><div id="sec-11" class="subsection"><h3>Data availability</h3><p id="p-31">All code to reproduce the analyses can be found at <a href="https://github.com/affeder/fst_dynamics">https://github.com/affeder/fst_dynamics</a>, and in the supplemental material. Sequences can be found in Genbank under accession numbers <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=MF029756&amp;atom=%2Fggg%2F9%2F10%2F3395.atom">MF029756</a>-<a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=MF030224&amp;atom=%2Fggg%2F9%2F10%2F3395.atom">MF030224</a> (plasma viral RNA) and <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=MF032109&amp;atom=%2Fggg%2F9%2F10%2F3395.atom">MF032109</a>-<a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=MF033066&amp;atom=%2Fggg%2F9%2F10%2F3395.atom">MF033066</a> (tissue viral RNA) for macaque <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=T98133.&amp;atom=%2Fggg%2F9%2F10%2F3395.atom">T98133.</a> Supplemental material available at FigShare: <a href="https://doi.org/10.25387/g3.9738875">https://doi.org/10.25387/g3.9738875</a>.</p></div></div><div class="section results-discussion" id="sec-12"><h2 class="">Results and Discussion</h2><div id="sec-13" class="subsection"><h3>A motivating example using Simian-HIV drug resistance evolution</h3><p id="p-32">Viruses infecting a multicellular host may be subject to both strong selection (via drugs and the immune system) and also fast migration between anatomical compartments (via the circulatory and/or lymphatic systems).</p><p id="p-33">A previous study found that among Simian-HIV within drug-treated macaques, multiple drug resistance mutations spread simultaneously across anatomical compartments with weak but significant evidence of population structure dynamically changing over time (<a id="xref-ref-15-3" class="xref-bibr" href="#ref-15">Feder <em>et al.</em> 2017</a>). In <a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>, we reproduce a simplified picture of this evolutionary process in one Simian-HIV-infected macaque (T98133) across three well-sampled tissues (lymph node, gut and plasma) at four time points (1, 3, 8 and 14 weeks after the onset of selection via the drug FTC). Approximately 30 single-genome sequences of the reverse transcriptase region of the <span class="inline-formula" id="inline-formula-57"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-65.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-65.gif"/></noscript></span></span> gene are taken per time point, per sampling location.</p><p id="p-34">Each population has several different variants of drug resistant viruses spreading simultaneously which can be seen both through different encodings of drug resistant types (<em>i.e.</em>, M184V <em>vs.</em> M184I) and also through linkage to different hitchhiking mutations (M184V <em>vs.</em> M184V + N255N + D177N) (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1</a>). All colors shown in <a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1</a> are resistant. Multiple spreading mutations suggest that the population mutation rate (<span class="inline-formula" id="inline-formula-58"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-66.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-66.gif"/></noscript></span></span>) is sufficiently large so as to produce soft selective sweeps (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Hermisson and Pennings 2005</a>). These mutations quickly displace wildtype virus across tissues almost entirely, confirming that selection to drug resistance is strong. Finally, migration is sufficiently fast to spread the same mutations around to different compartments. While some of these apparently spreading drug resistance mutants may be recurrent mutations, several are linked to multiple putatively neutral mutations, suggesting that they arose once and then moved between anatomical compartments. However, migration is not so fast that each compartment looks equivalent, as demonstrated by statistically significant elevated pairwise <span class="inline-formula" id="inline-formula-59"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-67.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-67.gif"/></noscript></span></span> between compartments at some of the points during the experiment (<a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1D</a>) (see Materials and Methods for details, and <a id="xref-ref-15-4" class="xref-bibr" href="#ref-15">Feder <em>et al.</em> (2017)</a> for a much more extensive description of population differentiation in this data). Notably, we observe non-monotonic patterns of <span class="inline-formula" id="inline-formula-60"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-68.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-68.gif"/></noscript></span></span> with respect to time, suggesting that some pairs of populations differentiate as they fix beneficial mutations, then re-equilibrate over time (plasma v. lymph node and lymph node v. gut).</p><p id="p-35">These patterns suggest qualitatively that the population genetic forces of mutation, migration and selection are all jointly strong. However, it remains quantitatively unclear how strong they are relative to each other. We therefore explore the dynamics of populations over short periods of time (including non-monotonicity of <span class="inline-formula" id="inline-formula-61"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-69.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-69.gif"/></noscript></span></span> relative to time) to better understand quantitatively how these population genetic parameters interact. We use many of the sampling attributes from the Simian-HIV example (≈100 generations, sampling depth of ≈30 genomes, ≈4 time points) as our reference in exploring this parameter space.</p><p id="p-36">Although we use these data as a motivating example, many ecologically important processes feature rapidly adapting and dynamically interconnected populations.</p></div><div id="sec-14" class="subsection"><h3>Building intuition about the interaction between strong selection and migration</h3><p id="p-37">We next explore how rapid adaptation in two subpopulations connected with migration can lead to signals of population differentiation over time.</p><p id="p-38"><a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a> shows two schematic scenarios through which a population divided into two subpopulations can fix beneficial variants. In both instances, a beneficial mutation will enter one population and begin to spread. However, the paths differ according to whether the first beneficial allele in the second subpopulation arrives through <em>de novo</em> mutation or via migration. Understanding differences in static signatures resulting from these two scenarios (in addition to selection from shared standing genetic variation between the two populations) are considered in <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Lee and Coop (2017)</a>.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/10/3395/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Schematic of allele frequencies and  across populations adapting in parallel. A. In a migrant-derived sweep, one population generates a beneficial allele via mutation significantly before the other population. The allele sweeps locally in its population of origin, and arrives in the alternative population due to migration before this population can fix its own variant. Here, the increase in  is due to divergent frequencies of the wildtype and derived allele, with the alternative population having a high frequency of the wildtype allele while the focal subpopulation has a high frequency of the derived allele. Migrant-derived sweeps have a short, symmetric spike in . B. In a locally-derived sweep, each population fixes its own variant (shown in red and blue) which increases in frequency locally. Over time, migration equilibrates the allele frequencies among the two populations. The increase in  is due to divergent frequencies among different derived alleles across populations, and  has a long-tailed, asymmetric trajectory over time." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-146688292" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Schematic of allele frequencies and &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-70&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; across populations adapting in parallel. A. In a migrant-derived sweep, one population generates a beneficial allele via mutation significantly before the other population. The allele sweeps locally in its population of origin, and arrives in the alternative population due to migration before this population can fix its own variant. Here, the increase in &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-71&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; is due to divergent frequencies of the wildtype and derived allele, with the alternative population having a high frequency of the wildtype allele while the focal subpopulation has a high frequency of the derived allele. Migrant-derived sweeps have a short, symmetric spike in &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-72&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;. B. In a locally-derived sweep, each population fixes its own variant (shown in red and blue) which increases in frequency locally. Over time, migration equilibrates the allele frequencies among the two populations. The increase in &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-73&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; is due to divergent frequencies among different derived alleles across populations, and &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-74&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; has a long-tailed, asymmetric trajectory over time.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/10/3395/F2.medium.gif" width="440" height="286"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/9/10/3395/F2.medium.gif" width="440" height="286"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/10/3395/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/10/3395/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494375" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-39" class="first-child">Schematic of allele frequencies and <span class="inline-formula" id="inline-formula-62"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/F2/embed/mml-math-70.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/F2/embed/mml-math-70.gif"/></noscript></span></span> across populations adapting in parallel. A. In a migrant-derived sweep, one population generates a beneficial allele via mutation significantly before the other population. The allele sweeps locally in its population of origin, and arrives in the alternative population due to migration before this population can fix its own variant. Here, the increase in <span class="inline-formula" id="inline-formula-63"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/F2/embed/mml-math-71.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/F2/embed/mml-math-71.gif"/></noscript></span></span> is due to divergent frequencies of the wildtype and derived allele, with the alternative population having a high frequency of the wildtype allele while the focal subpopulation has a high frequency of the derived allele. Migrant-derived sweeps have a short, symmetric spike in <span class="inline-formula" id="inline-formula-64"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/F2/embed/mml-math-72.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/F2/embed/mml-math-72.gif"/></noscript></span></span>. B. In a locally-derived sweep, each population fixes its own variant (shown in red and blue) which increases in frequency locally. Over time, migration equilibrates the allele frequencies among the two populations. The increase in <span class="inline-formula" id="inline-formula-65"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/F2/embed/mml-math-73.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/F2/embed/mml-math-73.gif"/></noscript></span></span> is due to divergent frequencies among different derived alleles across populations, and <span class="inline-formula" id="inline-formula-66"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/F2/embed/mml-math-74.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/F2/embed/mml-math-74.gif"/></noscript></span></span> has a long-tailed, asymmetric trajectory over time.</p><div class="sb-div caption-clear"></div></div></div><p id="p-40">In a “migrant-derived sweep” (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2A</a>), a beneficial mutation sweeps in one subpopulation and then spreads to the second subpopulation via migration. The mutation then sweeps in the second subpopulation. Population differentiation increases due to different frequencies of the same allele (or alleles) across subpopulations, and then disappears when the sweep completes in the second subpopulation. This case, which has been described previously in <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Slatkin and Wiehe (1998)</a>, <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Kim and Maruki (2011)</a> and <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Bierne (2010)</a>, is more likely when the influx of beneficial mutations is limited (<em>i.e.</em>, <span class="inline-formula" id="inline-formula-67"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-75.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-75.gif"/></noscript></span></span> low).</p><p id="p-41">When the population mutation rate is high, another path is also possible (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2B</a>). If many beneficial mutations enter the population each generation, then each subpopulation may produce its own beneficial variant (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2B1</a>.) which begins to rise in frequency locally (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2B2</a>). Population differentiation (as measured by <span class="inline-formula" id="inline-formula-68"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-76.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-76.gif"/></noscript></span></span>) increases due to sweeps of different alleles concurrently. However, over time, migration equilibrates the allele frequencies across the two populations (<a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2B3</a>), and <span class="inline-formula" id="inline-formula-69"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-77.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-77.gif"/></noscript></span></span> returns to long-term equilibrium levels. As we will explore later, the initial increase in differentiation and the rate of equilibration can be informative of the speed of migration.</p><p id="p-42">The shape of the <span class="inline-formula" id="inline-formula-70"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-78.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-78.gif"/></noscript></span></span> trajectory over time differs substantially between the two cases. In locally-derived sweeps, selective sweeps drive the initial increase of <span class="inline-formula" id="inline-formula-71"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-79.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-79.gif"/></noscript></span></span> and migration erodes population differentiation over time. In the case of the migrant-derived sweeps, selective sweeps drive both the increase and decrease in <span class="inline-formula" id="inline-formula-72"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-80.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-80.gif"/></noscript></span></span>, because <span class="inline-formula" id="inline-formula-73"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-81.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-81.gif"/></noscript></span></span> decreases as the sweep completes in the second subpopulation (See <a id="xref-fig-2-7" class="xref-fig" href="#F2">Figure 2A</a>). Migrant-derived sweeps are therefore substantially less informative than locally-derived sweeps about the rate of migration between two populations, because both the increase and decrease of <span class="inline-formula" id="inline-formula-74"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-82.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-82.gif"/></noscript></span></span> are on the timescale of selection.</p><p id="p-43">Whether a sweep will be locally- or migrant-derived is closely related to the probability of hard or soft sweeps (see <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Pennings and Hermisson (2006)</a>), and a full exploration of the conditions is left to a future study. Because <a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Kim and Maruki (2011)</a> describe migrant-derived sweep behavior in detail, we focus on locally-derived sweep behavior, although both types of sweeps occur in the model described below. Further, we believe a model of locally-derived beneficial mutations represents the more probable description of the dynamics in our motivating example of SHIV.</p><div id="sec-15" class="subsection"><h4>Model:</h4><p id="p-44">We consider a two-population island model with mutation, migration and selection. We model two haploid populations of size <em>N</em> with non-overlapping generations. Each population begins fixed with an identical wildtype allele (standing genetic variation is discussed below). Each generation, mutation introduces new beneficial mutations at a rate <span class="inline-formula" id="inline-formula-75"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-83.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-83.gif"/></noscript></span></span> at a single locus. Throughout the text, we focus on values of <span class="inline-formula" id="inline-formula-76"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-84.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-84.gif"/></noscript></span></span> where mutations are readily available to the population (<span class="inline-formula" id="inline-formula-77"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-85.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-85.gif"/></noscript></span></span>, and often <span class="inline-formula" id="inline-formula-78"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-86.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-86.gif"/></noscript></span></span>. Beneficial mutations possess an enhanced survival likelihood relative to wildtype (<span class="inline-formula" id="inline-formula-79"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-87.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-87.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-80"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-88.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-88.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-81"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-89.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-89.gif"/></noscript></span></span>, see Materials and Methods for full details). All beneficial mutations are neutral with respect to each other, and no individual can carry multiple mutations. This describes, for example, loss of function mutations where subsequent mutations in the same pathway have no effect and follows the modeling assumptions of <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Ralph and Coop (2010)</a>. Each generation, a proportion of the population <em>m</em> migrates symmetrically between the two subpopulations. Therefore, <span class="inline-formula" id="inline-formula-82"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-90.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-90.gif"/></noscript></span></span> individuals migrate in each population per generation.</p><p id="p-45">We investigate first the conditions under which the subpopulations will differentiate, as measured by the population differentiation statistic <span class="inline-formula" id="inline-formula-83"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-91.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-91.gif"/></noscript></span></span>.</p><p id="p-46"><a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a> shows a collection of stochastic simulations generated with <span class="inline-formula" id="inline-formula-84"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-92.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-92.gif"/></noscript></span></span>. Each colored region represents the spread of an allele in the two large populations experiencing strong positive selection (<span class="inline-formula" id="inline-formula-85"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-93.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-93.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-86"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-94.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-94.gif"/></noscript></span></span>). Red alleles originate in population A and blue alleles originate in population B. When the migration rate is low, each population fixes independently its own set of alleles (see <span class="inline-formula" id="inline-formula-87"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-95.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-95.gif"/></noscript></span></span> through <span class="inline-formula" id="inline-formula-88"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-96.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-96.gif"/></noscript></span></span>). When the migration rate is high, the populations are indistinguishable and represent a mix of alleles originating in both subpopulations (<em>i.e.</em>, <span class="inline-formula" id="inline-formula-89"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-97.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-97.gif"/></noscript></span></span>). Among intermediate migration rates, however, more complicated dynamics arise. The population differentiation statistic <span class="inline-formula" id="inline-formula-90"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-98.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-98.gif"/></noscript></span></span> initially rapidly increases and then slowly decreases. The non-monotonicity of these trajectories in time can be understood by considering three relevant timescales: 1) the relative forces of selection and migration during the sweep, 2) the relative forces of migration and drift following fixation, and 3) the relative timescales of migration and the sampling.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/10/3395/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Patterns of allele frequencies and  across populations adapting in parallel. Beneficial alleles originating in populations A and B (colored red and blue respectively) increase in frequency over the wildtype allele (in gray) creating patterns of differentiation in  shown on the bottom row. The patterns of differentiation are dependent on the migration rate connecting the two populations ( shown in columns,  varies as listed in column header.)" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-146688292" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Patterns of allele frequencies and &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-99&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; across populations adapting in parallel. Beneficial alleles originating in populations A and B (colored red and blue respectively) increase in frequency over the wildtype allele (in gray) creating patterns of differentiation in &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-100&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; shown on the bottom row. The patterns of differentiation are dependent on the migration rate connecting the two populations (&lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-101&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; shown in columns, &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-102&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; varies as listed in column header.)&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/10/3395/F3.medium.gif" width="440" height="317"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/9/10/3395/F3.medium.gif" width="440" height="317"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/10/3395/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/10/3395/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494318" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-47" class="first-child">Patterns of allele frequencies and <span class="inline-formula" id="inline-formula-91"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/F3/embed/mml-math-99.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/F3/embed/mml-math-99.gif"/></noscript></span></span> across populations adapting in parallel. Beneficial alleles originating in populations A and B (colored red and blue respectively) increase in frequency over the wildtype allele (in gray) creating patterns of differentiation in <span class="inline-formula" id="inline-formula-92"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/F3/embed/mml-math-100.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/F3/embed/mml-math-100.gif"/></noscript></span></span> shown on the bottom row. The patterns of differentiation are dependent on the migration rate connecting the two populations (<span class="inline-formula" id="inline-formula-93"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/F3/embed/mml-math-101.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/F3/embed/mml-math-101.gif"/></noscript></span></span> shown in columns, <span class="inline-formula" id="inline-formula-94"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/F3/embed/mml-math-102.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/F3/embed/mml-math-102.gif"/></noscript></span></span> varies as listed in column header.)</p><div class="sb-div caption-clear"></div></div></div><p id="p-48">The initial dynamics are determined by comparing the forces of migration and selection during the sweep. If selection overwhelms migration (<span class="inline-formula" id="inline-formula-95"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-103.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-103.gif"/></noscript></span></span>), each population at least partially fixes its own variant and <span class="inline-formula" id="inline-formula-96"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-104.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-104.gif"/></noscript></span></span> increases rapidly (here, <span class="inline-formula" id="inline-formula-97"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-105.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-105.gif"/></noscript></span></span>). Alleles spread within subpopulations faster than migration equilibrates these alleles across subpopulations. However, if migration is too fast compared to selection, little differentiation occurs at any point in the sweep (<span class="inline-formula" id="inline-formula-98"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-106.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-106.gif"/></noscript></span></span>).</p><p id="p-49">The subsequent decrease in <span class="inline-formula" id="inline-formula-99"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-107.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-107.gif"/></noscript></span></span> depends on the relative timescales of migration and both genetic drift and sampling post-sweep. In our model, all alleles share identical selective benefit and no secondary mutations occur on the background of the first mutation (although the first mutation can happen on the wildtype background multiple times). Therefore, after loss of the wildtype allele, all alleles are selectively neutral with respect to each other, and solely migration and drift govern the changes in their frequencies. <span class="inline-formula" id="inline-formula-100"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-108.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-108.gif"/></noscript></span></span> between the two populations will ultimately decrease to near the equilibrium predicted by Wright (<span class="inline-formula" id="inline-formula-101"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-109.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-109.gif"/></noscript></span></span>), but the rate of that equilibration depends on the migration. If migration is sufficiently fast compared to drift (<em>i.e.</em>, <span class="inline-formula" id="inline-formula-102"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-110.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-110.gif"/></noscript></span></span>), <span class="inline-formula" id="inline-formula-103"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-111.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-111.gif"/></noscript></span></span> will decrease faster than drift will move alleles within subpopulations. However, if migration is comparable to drift, the process will take significantly longer (on the order of <span class="inline-formula" id="inline-formula-104"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-112.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-112.gif"/></noscript></span></span> generations). Whether this decrease is observed depends on the time frame of population sampling, although this paper will be restricted to relatively short sampling periods (<em>i.e.</em>, 100 generations).</p><p id="p-50">These observations suggest that in adapting populations, distinctive patterns emerge depending on the relative values of migration (<em>m</em>), selection (<em>s</em>), genetic drift (<span class="inline-formula" id="inline-formula-105"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-113.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-113.gif"/></noscript></span></span>) and the sampling timescale. By examining these patterns over short timescales in genetic data, we may be able to estimate the values of <em>s</em>, <em>m</em> and <em>N</em> relative to each other in natural populations. This estimation will be a primary goal of the paper.</p><p id="p-51">An analog of <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a> with a lower population mutation rate (<span class="inline-formula" id="inline-formula-106"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-114.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-114.gif"/></noscript></span></span>) is shown in Figure S1, and displays example patterns of <span class="inline-formula" id="inline-formula-107"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-115.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-115.gif"/></noscript></span></span> in parameter regimes where migrant-derived sweeps are prevalent. If a similar picture is produced using neutral alleles, only the lowest migration rates produce differentiable patterns and elevated <span class="inline-formula" id="inline-formula-108"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-116.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-116.gif"/></noscript></span></span> (Figure S2), and it takes much longer for this differentiation to accrue (<em>i.e.</em>, on the order of <span class="inline-formula" id="inline-formula-109"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-117.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-117.gif"/></noscript></span></span> generations).</p><p id="p-52">The dynamic patterns of <span class="inline-formula" id="inline-formula-110"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-118.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-118.gif"/></noscript></span></span> under strong selection and migration suggest sweeping alleles across populations can provide insight into migration that occurs over much faster timescales than are relevant under neutrality. As in the neutral case, when migration is too slow compared to the rate of change in the population we can only say that migration rate belongs to a particular, slow regime. We encounter a similar situation when migration is too fast compared to selection. However, in the intermediate cases when selection and migration are on sufficiently similar timescales, more precise estimation appears possible. By observing alleles moving at different speeds due to different strengths of selection, we can move the boundaries of the bins and estimate more precisely migration at different speeds.</p><p id="p-53">This example also illustrates several additional practical points. 1) Because dynamical patterns in <span class="inline-formula" id="inline-formula-111"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-119.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-119.gif"/></noscript></span></span> are apparent when sweeping alleles are captured at multiple points in their trajectories, this approach depends on the availability of time series data. 2) Because differences in the allele frequencies must be measurably different, sampling depth will affect whether or not these signatures are present.</p><p id="p-54">Before dynamical patterns of population differentiation can be used to perform parameter estimation in a stochastic model, we first examine patterns of <span class="inline-formula" id="inline-formula-112"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-120.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-120.gif"/></noscript></span></span> under strong migration and selection in a deterministic setting. Specifically, we will explore analytically the parameter regimes that create transient and long-term population differentiation driven by locally-derived sweeps. We will see later that the population parameter regimes that create non-monotonic signatures of population differentiation with respect to time correspond to those in which we can successfully perform parameter estimation using approximate Bayesian computation.</p></div><div id="sec-16" class="subsection"><h4>Analytical approximation:</h4><p id="p-55">To study locally-derived sweeps, we consider the case in which each of two populations has a single copy of a local allele at the same locus (<em>i.e.</em>, frequency <span class="inline-formula" id="inline-formula-113"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-121.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-121.gif"/></noscript></span></span>) simultaneously. We ignore all subsequent mutation. Because the populations are symmetric, without loss of generality, we can consider the frequencies of a single subpopulation and determine what are the relative frequencies of the local and non-local allele at a time <em>t</em> (<span class="inline-formula" id="inline-formula-114"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-122.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-122.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-115"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-123.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-123.gif"/></noscript></span></span>, respectively). We consider the total number of derived alleles <span class="inline-formula" id="inline-formula-116"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-124.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-124.gif"/></noscript></span></span> and assume that <span class="inline-formula" id="inline-formula-117"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-125.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-125.gif"/></noscript></span></span> grows logistically.<span class="disp-formula" id="disp-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-126.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-126.gif"/></noscript></span></span>A differential equation describes the change in frequency of the non-local allele, <span class="inline-formula" id="inline-formula-118"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-127.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-127.gif"/></noscript></span></span>:<span class="disp-formula" id="disp-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-128.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-128.gif"/></noscript></span></span>We define <span class="inline-formula" id="inline-formula-119"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-129.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-129.gif"/></noscript></span></span>, the frequency of the non-local allele among all derived alleles. We solve to find<span class="disp-formula" id="disp-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-130.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-130.gif"/></noscript></span><span class="disp-formula-label">(1)</span></span>Closed form descriptions of <span class="inline-formula" id="inline-formula-120"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-131.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-131.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-121"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-132.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-132.gif"/></noscript></span></span> are given as follows:<span class="disp-formula" id="disp-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-133.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-133.gif"/></noscript></span><span class="disp-formula-label">(2)</span></span>We can investigate patterns of population differentiation using the <span class="inline-formula" id="inline-formula-122"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-134.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-134.gif"/></noscript></span></span> definition of <span class="inline-formula" id="inline-formula-123"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-135.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-135.gif"/></noscript></span></span>, representing the scaled relationship between intra- and inter-population heterozygosity (see Materials and Methods). If we assume that each population starts with its own allele at arbitrarily low frequency (<em>i.e.</em>, <span class="inline-formula" id="inline-formula-124"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-136.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-136.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-125"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-137.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-137.gif"/></noscript></span></span>), we can describe the dynamics of the two alleles over time via via the following equation (see supplemental text for more explicit derivation):<span class="disp-formula" id="disp-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-138.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-138.gif"/></noscript></span><span class="disp-formula-label">(3)</span></span>The predicted <span class="inline-formula" id="inline-formula-126"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-139.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-139.gif"/></noscript></span></span> trajectories match well the trajectories generated by simulations showing locally-derived sweeps (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). Because multiple circulating alleles constrain the possible values of <span class="inline-formula" id="inline-formula-127"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-140.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-140.gif"/></noscript></span></span> (<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Jakobsson <em>et al.</em> 2013</a>), all alleles originating from the same population from simulation are collapsed into a single allele so that the <span class="inline-formula" id="inline-formula-128"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-141.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-141.gif"/></noscript></span></span> magnitudes are also comparable.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/10/3395/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Simulated patterns of  over time share features with analytical predictions. Forward stochastic simulations of  between two populations undergoing parallel selection of strength s and connected by M variants per generation for a given set of parameters are shown in light, solid lines (100 replicates). All alleles originating in a single subpopulation are collapsed together for the purpose of computing . The median trajectory is shown in a dark solid line. Analytical predictions (equation 3) are shown in a dashed line. ()." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-146688292" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Simulated patterns of &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-142&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; over time share features with analytical predictions. Forward stochastic simulations of &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-143&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; between two populations undergoing parallel selection of strength &lt;em&gt;s&lt;/em&gt; and connected by &lt;em&gt;M&lt;/em&gt; variants per generation for a given set of parameters are shown in light, solid lines (100 replicates). All alleles originating in a single subpopulation are collapsed together for the purpose of computing &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-144&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;. The median trajectory is shown in a dark solid line. Analytical predictions (equation 3) are shown in a dashed line. (&lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-145&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/10/3395/F4.medium.gif" width="440" height="248"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/9/10/3395/F4.medium.gif" width="440" height="248"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/10/3395/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/10/3395/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494544" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-56" class="first-child">Simulated patterns of <span class="inline-formula" id="inline-formula-129"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/F4/embed/mml-math-142.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/F4/embed/mml-math-142.gif"/></noscript></span></span> over time share features with analytical predictions. Forward stochastic simulations of <span class="inline-formula" id="inline-formula-130"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/F4/embed/mml-math-143.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/F4/embed/mml-math-143.gif"/></noscript></span></span> between two populations undergoing parallel selection of strength <em>s</em> and connected by <em>M</em> variants per generation for a given set of parameters are shown in light, solid lines (100 replicates). All alleles originating in a single subpopulation are collapsed together for the purpose of computing <span class="inline-formula" id="inline-formula-131"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/F4/embed/mml-math-144.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/F4/embed/mml-math-144.gif"/></noscript></span></span>. The median trajectory is shown in a dark solid line. Analytical predictions (equation 3) are shown in a dashed line. (<span class="inline-formula" id="inline-formula-132"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/F4/embed/mml-math-145.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/F4/embed/mml-math-145.gif"/></noscript></span></span>).</p><div class="sb-div caption-clear"></div></div></div><p id="p-57">This equation also allows us to quantify the generation at which <span class="inline-formula" id="inline-formula-133"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-146.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-146.gif"/></noscript></span></span> will be maximized between the two subpopulations (<span class="inline-formula" id="inline-formula-134"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-147.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-147.gif"/></noscript></span></span>), and its maximum value (<span class="inline-formula" id="inline-formula-135"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-148.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-148.gif"/></noscript></span></span>:<span class="disp-formula" id="disp-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-149.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-149.gif"/></noscript></span><span class="disp-formula-label">(4)</span></span>and<span class="disp-formula" id="disp-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-150.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-150.gif"/></noscript></span><span class="disp-formula-label">(5)</span></span>From these equations, we can determine that the waiting time until the population reaches maximal <span class="inline-formula" id="inline-formula-136"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-151.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-151.gif"/></noscript></span></span> is dominated by a term inversely proportional to s. However, the maximal <span class="inline-formula" id="inline-formula-137"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-152.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-152.gif"/></noscript></span></span> at this point is determined by the ratio of <span class="inline-formula" id="inline-formula-138"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-153.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-153.gif"/></noscript></span></span>. The strength of selection controls largely when the <span class="inline-formula" id="inline-formula-139"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-154.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-154.gif"/></noscript></span></span> spike will happen, but the ratio of <span class="inline-formula" id="inline-formula-140"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-155.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-155.gif"/></noscript></span></span> governs the magnitude of the signal, with stronger selection relative to migration creating more pronounced spikes as a non-linear function of that proportion. Note that this equation is undefined when selection is not sufficiently stronger than migration (<span class="inline-formula" id="inline-formula-141"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-156.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-156.gif"/></noscript></span></span>).</p><p id="p-58">Below we will also use these equations to quantify the parameter regimes in which <span class="inline-formula" id="inline-formula-142"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-157.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-157.gif"/></noscript></span></span> will show non-monotonic patterns with respect to time by evaluating when <span class="inline-formula" id="inline-formula-143"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-158.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-158.gif"/></noscript></span></span> will be elevated sufficiently for it to be observed at a given depth, and also when <span class="inline-formula" id="inline-formula-144"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-159.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-159.gif"/></noscript></span></span> will decline appreciably over a relevant timescale by evaluating equation 3 some number of generations after <span class="inline-formula" id="inline-formula-145"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-160.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-160.gif"/></noscript></span></span>.</p><p id="p-59">Note that because this analytical framework assumes the simultaneous appearance of low frequency resistance mutations across populations, it only describes the dynamics seen when populations are non-mutation limited (<em>i.e.</em>, <span class="inline-formula" id="inline-formula-146"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-161.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-161.gif"/></noscript></span></span> large). Some of the discrepancies between the predicted and simulated trajectories stem from violation of this symmetry in simulations.</p></div></div><div id="sec-17" class="subsection"><h3>Patterns of population differentiation can be used for parameter estimation using approximate Bayesian computation</h3><p id="p-60">To translate these expected patterns into parameter estimates, we employ approximate Bayesian computation (ABC) (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Rubin 1984</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Beaumont <em>et al.</em> 2002</a>) because exact likelihood formulations cannot be worked out under complex demographic scenarios. This provides a framework for interpreting allele frequency trajectories in the context of underlying parameter regimes. Briefly, ABC works through simulating data under parameters drawn from a prior and then comparing the simulated data to observed data. The parameter combinations that generate simulated data most similar to the observed data form posterior distributions. Note, we do not intend this as a claim for the best way to estimate migration using selected alleles, but as a demonstration of extractable signal.</p><p id="p-61">We begin by demonstrating that migration information can be estimated from observed allele frequencies generated from our stochastic simulations using allele frequency differences in these connected populations over time. Recently, many methods have built up around the idea of estimating selection strength from allele frequency data in a single population (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Bollback <em>et al.</em> 2008</a>); <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Feder <em>et al.</em> 2014</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Terhorst <em>et al.</em> 2015</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Schraiber <em>et al.</em> 2016</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Iranmehr <em>et al.</em> 2017</a>). By leveraging this information across multiple populations, we can learn about selection and migration simultaneously. In particular, we imagine that we have allele frequency samples of <em>n</em> genomes before and after a sweep, and at some time point <em>T</em> generations later (here: <span class="inline-formula" id="inline-formula-147"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-162.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-162.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-148"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-163.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-163.gif"/></noscript></span></span>). These time points are chosen to match the Simian-HIV motivating example, but we provide a much more thorough analysis of how sampling time affects this approach below.</p><p id="p-62">Because we are tracking the frequencies of many alleles across multiple time points, we use summary statistics to characterize the observed dynamics. As noted in <a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Aeschbacher <em>et al.</em> (2012)</a>, the accuracy of ABC can be boosted when summary statistics are optimized separately for estimating different parameters, especially when there are only weak interactions between pairs of parameters. To jointly estimate <em>s</em>, <em>N</em> and <em>m</em>, we selected summary statistics that were useful for fitting the migration rate separately from <em>s</em> and <em>N</em>. We first performed a single round of ABC to estimate posteriors over <em>s</em> and <em>N</em> with these summaries. Then, we performed a second round of ABC to estimate <em>m</em> while restricting the priors over <em>s</em> and <em>N</em> to the posteriors from the first round fit (see Materials and Methods). We also fit <em>s</em> and <em>N</em> separately from each other and did not find a significant improvement in method performance (data not shown). To summarize information about <em>N</em>, we found the most likely <em>θ</em> using Ewens’ Sampling Formula for the combined allele frequencies at each sampled time point pooled across the populations. To summarize information about <em>s</em>, we assumed beneficial allele frequency <span class="inline-formula" id="inline-formula-149"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-164.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-164.gif"/></noscript></span></span> for a beneficial mutation at time 0 (which is a conservatively low frequency for intra-host standing genetic variation (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Zanini <em>et al.</em> 2017</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Theys <em>et al.</em> 2018</a>)) and determined the parameter <em>s</em> producing the logistic growth curve that minimized mean squared error from the observed data. To compute <em>M</em>, we used four parameters at each time point: <span class="inline-formula" id="inline-formula-150"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-165.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-165.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-151"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-166.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-166.gif"/></noscript></span></span>, the difference in heterozygosities between the two subpopulations and the number of shared alleles at any frequency. Reasoning and further details concerning these summary statistics can be found in the Materials and Methods.</p><div id="sec-18" class="subsection"><h4>Migration estimation depends on s:</h4><p id="p-63">To validate our ABC approach, we first simulated data under known parameters and then used our approach to re-estimate migration rates. We ran 200 forward simulations for each set of parameters and examined the resulting 95% posteriors over <em>m</em> to quantify the proportion of posteriors that contain the true <em>m</em> (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>). See Materials and Methods for full simulation details. We also quantify the probability that the posteriors contain incorrect values of <em>m</em> at varying distances away from the truth. We find the 95% posteriors contain the true migration rate with high probability (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>, indicated by the clustering of red around the <span class="inline-formula" id="inline-formula-152"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-167.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-167.gif"/></noscript></span></span> line) and do not contain migration rates far away from the truth (indicated by off <span class="inline-formula" id="inline-formula-153"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-168.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-168.gif"/></noscript></span></span>).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/10/3395/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="ABC procedure re-estimates true M values in simulated data. 200 simulations are run for each true value of m on the x-axis and 200 posteriors are produced. We plot the proportion of 95% posteriors for a given true m (shown on the x-axis) that include the various values of estimated m (shown on the y-axis). Values that are more red indicate that the posteriors successfully capture the test &#x2018;Estimated m&#x2019; value on the y-axis whereas blue values indicate that few posteriors include the y-axis value. Black horizontal lines indicate where the true value matches the y-axis &#x2018;Estimated m.&#x2019; Simulations are generated across a gradient of values for m and s (, sampling at generations (5, T = fixation, ." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-146688292" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;ABC procedure re-estimates true &lt;em&gt;M&lt;/em&gt; values in simulated data. 200 simulations are run for each true value of &lt;em&gt;m&lt;/em&gt; on the &lt;em&gt;x&lt;/em&gt;-axis and 200 posteriors are produced. We plot the proportion of 95% posteriors for a given true &lt;em&gt;m&lt;/em&gt; (shown on the &lt;em&gt;x&lt;/em&gt;-axis) that include the various values of estimated &lt;em&gt;m&lt;/em&gt; (shown on the &lt;em&gt;y&lt;/em&gt;-axis). Values that are more red indicate that the posteriors successfully capture the test &#x2018;Estimated &lt;em&gt;m&lt;/em&gt;&#x2019; value on the &lt;em&gt;y&lt;/em&gt;-axis whereas blue values indicate that few posteriors include the &lt;em&gt;y&lt;/em&gt;-axis value. Black horizontal lines indicate where the true value matches the &lt;em&gt;y&lt;/em&gt;-axis &#x2018;Estimated &lt;em&gt;m&lt;/em&gt;.&#x2019; Simulations are generated across a gradient of values for &lt;em&gt;m&lt;/em&gt; and &lt;em&gt;s&lt;/em&gt; (&lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-169&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;, sampling at generations (5, &lt;em&gt;T&lt;/em&gt; = fixation, &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-170&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/10/3395/F5.medium.gif" width="440" height="161"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/9/10/3395/F5.medium.gif" width="440" height="161"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/10/3395/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/10/3395/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494414" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-64" class="first-child">ABC procedure re-estimates true <em>M</em> values in simulated data. 200 simulations are run for each true value of <em>m</em> on the <em>x</em>-axis and 200 posteriors are produced. We plot the proportion of 95% posteriors for a given true <em>m</em> (shown on the <em>x</em>-axis) that include the various values of estimated <em>m</em> (shown on the <em>y</em>-axis). Values that are more red indicate that the posteriors successfully capture the test ‘Estimated <em>m</em>’ value on the <em>y</em>-axis whereas blue values indicate that few posteriors include the <em>y</em>-axis value. Black horizontal lines indicate where the true value matches the <em>y</em>-axis ‘Estimated <em>m</em>.’ Simulations are generated across a gradient of values for <em>m</em> and <em>s</em> (<span class="inline-formula" id="inline-formula-154"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/F5/embed/mml-math-169.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/F5/embed/mml-math-169.gif"/></noscript></span></span>, sampling at generations (5, <em>T</em> = fixation, <span class="inline-formula" id="inline-formula-155"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/F5/embed/mml-math-170.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/F5/embed/mml-math-170.gif"/></noscript></span></span>.</p><div class="sb-div caption-clear"></div></div></div><p id="p-65">There are instructive exceptions. First, when the migration rate is low, we have little power to distinguish values of <em>m</em> in this range. Low <em>m</em> results in near total differentiation between the two subpopulations (see <a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a>, <span class="inline-formula" id="inline-formula-156"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-171.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-171.gif"/></noscript></span></span>), so it is therefore unsurprising that we cannot predict the migration rate beyond bounding it. Similarly, this method cannot differentiate among very high migration rates. In this instance, the populations are entirely panmictic, and there is little signal to uncover (See <a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3</a>, <span class="inline-formula" id="inline-formula-157"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-172.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-172.gif"/></noscript></span></span>). Although within these regimes where allele frequencies from the two subpopulations appear panmictic or independent we have no ability to estimate migration rates specifically, this does allow us to bound fast migration rates much more precisely than is possible with neutral alleles. For example, if we were to observe identical equilibrium neutral allele frequencies among two subpopulations, we might conclude that <span class="inline-formula" id="inline-formula-158"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-173.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-173.gif"/></noscript></span></span>. However, if we observed identical allele frequencies over time subject to strong selection, this would suggest that <span class="inline-formula" id="inline-formula-159"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-174.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-174.gif"/></noscript></span></span>.</p><p id="p-66">The boundaries at which we enter these regimes of independence and panmixia depend on the strength of selection. We see that <span class="inline-formula" id="inline-formula-160"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-175.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-175.gif"/></noscript></span></span> results in apparent panmixia in many instances. However, when <span class="inline-formula" id="inline-formula-161"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-176.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-176.gif"/></noscript></span></span>, our method still accurately estimates migration rates when <span class="inline-formula" id="inline-formula-162"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-177.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-177.gif"/></noscript></span></span> (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5</a>). We test this more directly by computing the log MSE between the posteriors and the true <em>m</em> for a wide variety of selection strengths and migration rates (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6A</a>, see Materials and Methods for a detailed description of the log MSE). Consistent with the analysis in <a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5</a>, when migration is too high or too low relative to selection, the posterior is far from the truth. (Note, the relatively low MSE between the truth and the posterior among high values of <em>m</em> is driven by the limits on the prior, which is bounded above by <span class="inline-formula" id="inline-formula-163"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-178.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-178.gif"/></noscript></span></span>). However, the upper boundary at which the performance deteriorates increases with selection strength. Put another way, as selection strength increases, so do the rates of <em>m</em> that can be accurately estimated.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/10/3395/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Better estimation of higher migration rates among simulations with higher selection strength. For a fixed N, higher s values result in lower log MSEs (indicating successful estimation) for higher values of M. Each tile represents the median log-MSE for 100 trials computed with a given m and s. (, sampling at generations 2, T = fixation, ). B. Predictions of the dynamics of  over 100 generations under a variety of parameters for s and m. Each square in A. corresponds to the modeled dynamics in B. The parameter combinations predicted to lead to a 0.15 unit decrease in  after 30 generations (red) correspond to the parameter regimes where migration can be estimated with low MSE. The area circumscribed in black highlights this region in both A and B." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-146688292" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Better estimation of higher migration rates among simulations with higher selection strength. For a fixed &lt;em&gt;N&lt;/em&gt;, higher &lt;em&gt;s&lt;/em&gt; values result in lower log MSEs (indicating successful estimation) for higher values of &lt;em&gt;M&lt;/em&gt;. Each tile represents the median log-MSE for 100 trials computed with a given &lt;em&gt;m&lt;/em&gt; and &lt;em&gt;s&lt;/em&gt;. (&lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-179&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;, sampling at generations 2, &lt;em&gt;T&lt;/em&gt; = fixation, &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-180&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;). B. Predictions of the dynamics of &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-181&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; over 100 generations under a variety of parameters for &lt;em&gt;s&lt;/em&gt; and &lt;em&gt;m&lt;/em&gt;. Each square in A. corresponds to the modeled dynamics in B. The parameter combinations predicted to lead to a 0.15 unit decrease in &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-182&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; after 30 generations (red) correspond to the parameter regimes where migration can be estimated with low MSE. The area circumscribed in black highlights this region in both A and B.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/10/3395/F6.medium.gif" width="440" height="176"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/9/10/3395/F6.medium.gif" width="440" height="176"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/10/3395/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/10/3395/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494353" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-67" class="first-child">Better estimation of higher migration rates among simulations with higher selection strength. For a fixed <em>N</em>, higher <em>s</em> values result in lower log MSEs (indicating successful estimation) for higher values of <em>M</em>. Each tile represents the median log-MSE for 100 trials computed with a given <em>m</em> and <em>s</em>. (<span class="inline-formula" id="inline-formula-164"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/F6/embed/mml-math-179.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/F6/embed/mml-math-179.gif"/></noscript></span></span>, sampling at generations 2, <em>T</em> = fixation, <span class="inline-formula" id="inline-formula-165"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/F6/embed/mml-math-180.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/F6/embed/mml-math-180.gif"/></noscript></span></span>). B. Predictions of the dynamics of <span class="inline-formula" id="inline-formula-166"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/F6/embed/mml-math-181.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/F6/embed/mml-math-181.gif"/></noscript></span></span> over 100 generations under a variety of parameters for <em>s</em> and <em>m</em>. Each square in A. corresponds to the modeled dynamics in B. The parameter combinations predicted to lead to a 0.15 unit decrease in <span class="inline-formula" id="inline-formula-167"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/F6/embed/mml-math-182.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/F6/embed/mml-math-182.gif"/></noscript></span></span> after 30 generations (red) correspond to the parameter regimes where migration can be estimated with low MSE. The area circumscribed in black highlights this region in both A and B.</p><div class="sb-div caption-clear"></div></div></div><p id="p-68">These results are predicted by our analytical model of locally-derived sweeps. The parameter combinations that show a noticeable non-monotonic pattern in <span class="inline-formula" id="inline-formula-168"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-183.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-183.gif"/></noscript></span></span> over 100 generations (<em>i.e.</em>, <span class="inline-formula" id="inline-formula-169"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-184.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-184.gif"/></noscript></span></span> reaches a value greater than or equal to 0.15 followed by a decrease in <span class="inline-formula" id="inline-formula-170"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-185.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-185.gif"/></noscript></span></span> within 30 generations of <span class="inline-formula" id="inline-formula-171"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-186.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-186.gif"/></noscript></span></span>) are similar to those that can be estimated using the ABC approach (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6B</a>). This suggests that a primary signal identified by our ABC approach is a non-monotonic pattern of <span class="inline-formula" id="inline-formula-172"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-187.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-187.gif"/></noscript></span></span> over time. However, because the performance of the ABC approach deteriorates as other summary statistics are removed (data not shown), this is likely just one of several important signals for estimation.</p></div></div><div id="sec-19" class="subsection"><h3>Practical considerations in estimating m from data</h3><p id="p-69">The interaction between migration and selection is not the only factor that impacts our estimation. We find that the performance improves when the sampling is deeper, reflecting more accurate estimation of allele frequency differences (Figure S3). We also find that the choice of sampling time points matters for the estimation of specific migration rates. We assume that samples are taken before and after a sweep, and then at an additional time point <em>T</em> generations later. When <em>T</em> is small, performance is worse, presumably because migration has had less time to influence allele frequencies (Figure S4). When adding a fourth time point at generation 100, the method is fairly insensitive to the placement of the third time point (Figure S5).</p><p id="p-70">We also find that specific parameters of the ABC are important, namely the tolerance of accepted matches. We find that as tolerance becomes more stringent, the posteriors become narrower (Figure S6A). However, as the resulting posterior decreases in size due to lower tolerance, there is a decreased probability of capturing the true value in the posterior (Figure S6B). Nevertheless, decreasing the tolerance improves the matching of posteriors and the truth (Figure S6C), and we therefore choose a low tolerance (<span class="inline-formula" id="inline-formula-173"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-188.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-188.gif"/></noscript></span></span>) for our analyses.</p><p id="p-71">Finally, we explore how true asymmetries in the underlying model affect ABC performance in estimating migration rates (Figure S7). Similar to analyses above, we simulate adaptation to strong selection (<span class="inline-formula" id="inline-formula-174"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-189.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-189.gif"/></noscript></span></span>) across two subpopulations under a variety of migration rates, but now we allow the second subpopulation to differ in migration rate (<span class="inline-formula" id="inline-formula-175"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-190.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-190.gif"/></noscript></span></span>, Fig S7A-C), population size (<span class="inline-formula" id="inline-formula-176"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-191.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-191.gif"/></noscript></span></span>, Fig S7D-F) or selection strength (<span class="inline-formula" id="inline-formula-177"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-192.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-192.gif"/></noscript></span></span> Fig S7G-I). We estimate migration rates assuming no asymmetries.</p><p id="p-72">Our model is relatively robust to differences among migration rates between the two subpopulations (<span class="inline-formula" id="inline-formula-178"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-193.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-193.gif"/></noscript></span></span>). When <span class="inline-formula" id="inline-formula-179"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-194.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-194.gif"/></noscript></span></span>, the estimated migration rate is intermediate to the two rates, but has similar variance to the symmetric case (Figure S7A-C).</p><p id="p-73">Differences in population size between the two subpopulations increase the variance of the posteriors, but do not appear to bias the estimation of migration rates (Figure S7D-F).</p><p id="p-74">Differences among selection coefficients between the subpopulations create more complicated effects. When selection in one subpopulation is much smaller than the other (but selecting for the same variants), we are unable to estimate migration because the sweep does not complete across both compartments, and our estimation procedure conditions on a near full sweep (Figure S7G). When migration is sufficiently fast, migration from the subpopulation experiencing stronger selection contributes substantially to the sweep in the subpopulation experiencing weaker selection. As a result, when the populations are sampled, they appear more similar to each other than if both populations are increasing alleles due to strong, semi-independent selection. As a result, migration rates are systematically biased upwards, but the effect is small (Figure S7G-H). In addition, because the selection strength is lower in one subpopulation, the method has less resolution to differentiate among intermediate migration rates, as we observed in the investigation above. This also leads to an increase in variance among the estimated rates. We discuss how this compares to the case in which selection selects for different variants across compartments in the discussion.</p><div id="sec-20" class="subsection"><h4>Application to real data:</h4><p id="p-75">In the previous section, we have established that we can estimate population genetic parameters using a multi-step ABC method. In this section, we apply this method to real data that matches the structure of our ABC method (<a id="xref-ref-15-5" class="xref-bibr" href="#ref-15">Feder <em>et al.</em> 2017</a>) in order to estimate viral migration rates between pairs of different organs (blood plasma, lymph nodes, gut) of a Simian-HIV infected pigtailed macaque sampled over time (<a id="xref-fig-1-7" class="xref-fig" href="#F1">Figure 1</a>). The importance of estimating such rates has been previously investigated (although in the absence of selection) <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Ewing and Rodrigo (2007)</a>. The allele frequencies shown in <a id="xref-fig-1-8" class="xref-fig" href="#F1">Figure 1</a> are simplified trajectories of the full data to make it more comparable to our model. For full data processing choices (and fits when those choices are modified), see the Materials and Methods section.</p><p id="p-76">We find that estimates of <em>m</em> between the plasma and lymph node are very high: The distribution is flat above 1% of the population migrating each generation (<a id="xref-fig-7-2" class="xref-fig" href="#F7">Figure 7</a>, <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). The flatness of this distribution suggests that these subpopulations exist in the regime in which migration moves alleles faster than selection, and we find little differentiation that cannot be explained by sampling error (consistent with findings of panmixia in <a id="xref-ref-15-6" class="xref-bibr" href="#ref-15">Feder <em>et al.</em> (2017)</a>). Although a more specific migration rate cannot be estimated, posteriors suggest a migration rate above <span class="inline-formula" id="inline-formula-180"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-195.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-195.gif"/></noscript></span></span>.</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/10/3395/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Estimation of population parameter rates from intra-patient Simian-HIV data sampled from different subpopulations. The top row shows diagrams of drug resistant haplotypes spreading in different subpopulations over time sampled at generations 7, 21, 49 and 98 in the gut, lymph node and blood plasma. Each color represents a distinct lineage separated by at least one mutation. The resulting posteriors are given for the ABC procedure for comparisons between the plasma and gut (red) and plasma and lymph node (blue). . 95% posteriors for these distributions are shown in Table 1." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-146688292" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Estimation of population parameter rates from intra-patient Simian-HIV data sampled from different subpopulations. The top row shows diagrams of drug resistant haplotypes spreading in different subpopulations over time sampled at generations 7, 21, 49 and 98 in the gut, lymph node and blood plasma. Each color represents a distinct lineage separated by at least one mutation. The resulting posteriors are given for the ABC procedure for comparisons between the plasma and gut (red) and plasma and lymph node (blue). &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-196&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;. 95% posteriors for these distributions are shown in Table 1.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/10/3395/F7.medium.gif" width="440" height="166"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/9/10/3395/F7.medium.gif" width="440" height="166"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/10/3395/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/10/3395/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494406" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-77" class="first-child">Estimation of population parameter rates from intra-patient Simian-HIV data sampled from different subpopulations. The top row shows diagrams of drug resistant haplotypes spreading in different subpopulations over time sampled at generations 7, 21, 49 and 98 in the gut, lymph node and blood plasma. Each color represents a distinct lineage separated by at least one mutation. The resulting posteriors are given for the ABC procedure for comparisons between the plasma and gut (red) and plasma and lymph node (blue). <span class="inline-formula" id="inline-formula-181"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/F7/embed/mml-math-196.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/F7/embed/mml-math-196.gif"/></noscript></span></span>. 95% posteriors for these distributions are shown in <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>.</p><div class="sb-div caption-clear"></div></div></div><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/494454/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/494454/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Medians and 95% posteriors for parameter estimation in intra-macaque Simian-HIV populations</span><div class="sb-div caption-clear"></div></div></div><p id="p-79">However, the migration rate between the plasma and the gut is unimodal centered around 0.05% of the population migrating each generation (<a id="xref-fig-7-3" class="xref-fig" href="#F7">Figure 7</a>, <a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>) - half of the low end of the posterior between the lymph node and the plasma, yet higher than what would produce elevated <span class="inline-formula" id="inline-formula-185"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-200.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-200.gif"/></noscript></span></span> in drifting alleles at equilibrium (although <span class="inline-formula" id="inline-formula-186"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-201.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-201.gif"/></noscript></span></span> falls within the 95% posterior). These differences between the connectivity between the plasma and gut or lymph node are biologically consistent in their ordering, as we might expect the circulatory system to be more connected to the lymphatic system than a mucosal tissue.</p><p id="p-80">We estimate similar selection strength and population mutation rates across subpopulation comparisons (see <a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>). Across both comparisons, the selection strength is extremely strong (<span class="inline-formula" id="inline-formula-187"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-202.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-202.gif"/></noscript></span></span> - 3), which is necessary to explain the almost complete sweeps observed in the first 20 generations. The population mutation rate is sufficiently high that soft sweeps are likely (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Hermisson and Pennings 2017</a>). That we see no significant differences between the estimated population sizes makes intuitive sense given the consistency of sweep timing and diversity across the three populations. However, it is also possible that the size estimation across the pairs of compartments is driven by the size of the largest subpopulation, resulting in comparisons that are less dependent on the size of the smaller subpopulation. This would decrease our power to detect differences in compartment sizes in comparisons across ABC runs.</p><p id="p-81">Note, we make filtering choices to determine what is counted as a distinct haplotype (see Materials and Methods for full details). In the main text, we present intermediate filtering in which we create haplotypes from alleles observed at least twice before drug resistance sweeps (&gt;70%) (<em>i.e.</em>, alleles that are singletons before fixation are excluded). We can repeat the analysis for alleles observed in at least one copy (Figure S8) or at least five copies (Figure S9). Different haplotype thresholds unsurprisingly alter estimates of <span class="inline-formula" id="inline-formula-188"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-203.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-203.gif"/></noscript></span></span>, but posteriors are robust for estimates of <em>s</em> and <em>m</em>, likely due to the factorization procedure (Figures S10 and S11). Posteriors for all three allele frequency cutoffs are listed in Table S1.</p></div></div></div><div class="section discussion" id="sec-21"><h2 class="">Discussion</h2><p id="p-82">In this study, we characterize the dynamics of populations adapting under strong and uniform population genetic forces - abundant mutation, strong positive selection and fast migration between subpopulations. Specifically, we describe how beneficial mutations can spread locally within populations causing rapid population differentiation. Fast migration can then re-equilibrate the allele frequencies of the populations over short timescales. This can happen in two ways: either a beneficial mutation can sweep in one subpopulation and then establish in a second subpopulation via migration (migrant-derived sweeps), or each subpopulation can produce its own beneficial mutation(s) which ultimately equilibrate in frequency via migration (locally-derived sweeps).</p><p id="p-83">Although migrant-derived sweeps have been described previously (<a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Slatkin and Wiehe 1998</a>; <a id="xref-ref-23-3" class="xref-bibr" href="#ref-23">Kim and Maruki 2011</a>; <a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Bierne 2010</a>), locally-derived sweeps can help explain the apparent “softening” of hard sweeps in HIV, which initially show only a single beneficial haplotype, yet later multiple haplotypes appear (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Paulose <em>et al.</em> 2019</a>; <a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Ralph and Coop 2010</a>; <a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Hermisson and Pennings 2017</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Williams and Pennings 2019</a>). If each subpopulation fixes its own beneficial variant, which later mix, a locally hard sweep can appear to become soft. This is in contrast to the previously described “hardening” of soft sweeps, which can occur when demography causes only a single beneficial haplotype of a soft sweep to fix (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Wilson <em>et al.</em> 2017</a>).</p><p id="p-84">That allele frequencies can change dynamically over very short timescales has important implications for interpreting population differentiation statistics. In particular, data sampled at a particular time point can be misleading in determining population parameters if the underlying population is incorrectly assumed to be at equilibrium. Even with constant migration rates, samples at different time points could lead one to conclude that migration is either prevalent or rare, and migration rates much larger than <span class="inline-formula" id="inline-formula-189"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-204.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-204.gif"/></noscript></span></span> can lead to substantial population differentiation transiently. This suggests that merely observing elevated <span class="inline-formula" id="inline-formula-190"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-205.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-205.gif"/></noscript></span></span> is not a sufficient condition for diagnosing a low-migration system and understanding non-equilibrium dynamics of these statistics is important for analyzing data. Proper accounting for the covariance among selected alleles can help disentangle the signatures left by combined selection and migration (<a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Lee and Coop 2017</a>).</p><p id="p-85">To understand the conditions in which populations appear differentiated over short timescales following a selective sweep, we explore an analytical model of <span class="inline-formula" id="inline-formula-191"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-206.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-206.gif"/></noscript></span></span> between populations with strong migration, selection and mutation. In time series data, we find a variety of dynamics of elevated <span class="inline-formula" id="inline-formula-192"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-207.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-207.gif"/></noscript></span></span>: if selection is stronger than migration (<span class="inline-formula" id="inline-formula-193"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-208.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-208.gif"/></noscript></span></span>), <span class="inline-formula" id="inline-formula-194"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-209.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-209.gif"/></noscript></span></span> will increase over short timescales. If migration is faster than drift (<span class="inline-formula" id="inline-formula-195"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-210.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-210.gif"/></noscript></span></span>) and the sampling timescale, <span class="inline-formula" id="inline-formula-196"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-211.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-211.gif"/></noscript></span></span> will quickly decrease after the sweep. When both conditions are met, characteristic non-monotonic patterns of <span class="inline-formula" id="inline-formula-197"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-212.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-212.gif"/></noscript></span></span> emerge over experimentally tractable timescales. We introduce a modeling framework to further explore this diversity of patterns.</p><p id="p-86">After observing a diversity of patterns in <span class="inline-formula" id="inline-formula-198"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-213.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-213.gif"/></noscript></span></span> as function of the relationships between drift, selection and migration, we develop an iterated-ABC framework that estimates <em>N</em>, <em>s</em> and <em>m</em> jointly using summaries of population differentiation over time. We explore the regimes where estimation is accurate, and find that its accuracy can be predicted from our simple analytical model of <span class="inline-formula" id="inline-formula-199"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-214.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-214.gif"/></noscript></span></span>. In the parameter regimes where estimation is not accurate, we can bound migration rates above or below certain values by comparing the rate of migration to the processes of drift and selection. These broad regimes of estimation suggest that we can quantify migration rates that we might expect to be important among pathogen populations.</p><p id="p-87">While the model-based portion of this analysis focuses mainly on one metric - <span class="inline-formula" id="inline-formula-200"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-215.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-215.gif"/></noscript></span></span> - to explore patterns of differentiation between connected populations, the inference framework also exploits other summarized aspects of the data including counts of shared haplotypes, and alternative normalizations of heterozygosity. While we do not here disentangle exactly what each summary statistic contributes, this does suggest that subtle differences in allele frequency trajectories encode information that can be useful for estimation. This underscores the potential for machine learning in population genetic inference <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Sheehan and Song (2016)</a>.</p><p id="p-88">As an example of the inference approach, we estimate migration rates between viral populations inhabiting different organs in a Simian-HIV infected macaque treated with drugs. Consistent with the findings of the original study which used several metrics of population differentiation but did not attempt to estimate rates, we find differential connectivity between compartments of the body. In particular, we find that mucosal tissues (<em>i.e.</em>, the gut) have lower migration rates into the blood than the lymph nodes do. However, using our ABC framework, we can make this description of the intra-patient environment quantitative as opposed to simply qualitative. We estimate per-virus migration rates that can be used in future modeling studies.</p><p id="p-89">One aim of this analysis was to explain a conspicuous pattern observed during the first ≈100 generations after the onset of selection in spatially-sampled simian-HIV populations - a rapid increase in <span class="inline-formula" id="inline-formula-201"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-216.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3395/embed/mml-math-216.gif"/></noscript></span></span> that decays quickly over time. In this paper, we show that some of the key features of this pattern can be explained in a simple symmetric island model. Notably, these key features appear without assumptions of asymmetry among compartments or local adaptation (similar to the observation that distinct patterns of sweeps across subpopulations can exist without local adaptation in <a id="xref-ref-7-3" class="xref-bibr" href="#ref-7">Bierne (2010)</a>). However, what is necessary is a migration rate of the right order of magnitude, conditional on the strength of selection.</p><p id="p-90">Note, this does not imply that in the case of Simian-HIV the observed pattern cannot be produced in a different way. For example, selection strength or direction may differ among compartments caused by varying degrees of penetrance of antiretroviral drugs among organs of the body. If selection pressures differ between compartments (<em>i.e.</em>, a beneficial mutation in one compartment is neutral or even deleterious in the other), we are likely to underestimate migration rates, because allele frequencies will equilibrate more slowly between the two compartments, ultimately reaching migration-selection equilibrium. With strong and potentially temporally-varying local adaptation, we should be able to obtain the observed patterns with even much stronger migration.</p><p id="p-91">However, since complex selection scenarios are not necessary to produce key features of our observations, we need not assume they are present. Accordingly, we believe that the mechanisms that are described by our model provide a plausible explanation for the initial dynamics of population differentiation. This said, there is some circumstantial evidence that over longer time scales selection differences could play a role. For example, by generation 98, a different clone comes to dominate the gut compartment than the lymph node and plasma. Indeed, fitness differences among subsequent selective sweeps (neither of which are incorporated in the model) can explain this phenomenon.</p><p id="p-92">Other factors, such as asymmetries in mutation rates or compartment sizes could potentially lead to biases in the migration rates relative to our estimates (although likely to a lesser extent than differences in selection). Natural extensions to this model might incorporate differences in natural selection among populations, or include isolation-by-distance as opposed to a simple island model. An additional problem is that in non-mutation limited systems, not only do multiple mutations arise simultaneously, but populations quickly acquire double and triple mutants. It is therefore important that in future studies, we consider how these patterns of population differentiation might appear on a traveling wave of beneficial mutations, similar to what has been done in single populations (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Desai and Fisher 2007</a>).</p><p id="p-93">Our results also provide guidance for experimental design, particularly in non-mutation limited systems with strong selection and potentially high migration rates such as pathogen evolution, and pesticide and herbicide resistance. In particular, we find that the most useful way to glean information about migration rates from data are to include a temporal component to sampling. For migration estimation, it is considerably more useful to have three time points of 30 sequences than a single time point of 90 sequences. It is particularly useful to sample during or shortly after a selective sweep. From a practical perspective, the timing of such sweeps can be determined by phenotypic shifts in the population. If only a single time point directly after the sweep is available, this can provide information about whether migration is slower than selection. However, if an additional later time point is available, this potentially also gives information about whether migration is faster than drift. Having samples at both of these times provides both upper and lower bounds for high migration rates.</p><p id="p-94">To understand very rapid processes (such as fast migration) or the forces governing the population in its current state, it is insufficient to look at dynamics averaged out over long time periods. Selection provides an avenue to observe the current population state in time that neutral processes can miss. Using long term metrics (like comparing the rate of migration to the rate of drift) to investigate how much migration is happening in the moment invites numerous misinterpretations. When studying migration, as when studying all population processes, the correct timescale must be carefully considered.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-95">We would like to acknowledge Ilana Arbisser, Jamie Blundell, Graham Coop, Jonathan Kang, Grant Kinsler, Chuan Li, Michael MacLaren, Rohan Mehta, Noah Rosenberg, and Lawrence Urichio for useful discussions on the content of the manuscript, and two anonymous reviewers for improving the manuscript through their comments. This work was supported in part by the Miller Institute for Basic Research in Science at the University of California Berkeley (AFF), a Stanford Computational, Evolutionary and Human Genomics Trainee Fellowship (AFF), ABI-1458059 (DAP) and 3R35GM118165-03S1 (DAP).</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1"><a class="rev-xref" href="#xref-fn-1-1">↵</a><span class="fn-label">2</span> These authors contributed equally to this work.</p></li><li class="fn" id="fn-3"><p id="p-96">Supplemental material available at FigShare: <a href="https://doi.org/10.25387/g3.9738875">https://doi.org/10.25387/g3.9738875</a>.</p></li><li class="fn" id="fn-4"><p id="p-97"><em>Communicating editor: Y. Kim</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2019-06-04"><span class="received-label">Received </span>June 4, 2019.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2019-08-16"><span class="accepted-label">Accepted </span>August 16, 2019.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2019 Feder <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3395.1" data-doi="10.1128/JVI.00915-10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Abram</span>,  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. L.</span> <span class="cit-name-surname">Ferris</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Shao</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W. G.</span> <span class="cit-name-surname">Alvord</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. H.</span> <span class="cit-name-surname">Hughes</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Nature, position, and frequency of mutations made in a single cycle of HIV-1 replication.</span> <abbr class="cit-jnl-abbrev">J. Virol.</abbr> <span class="cit-vol">84</span>: <span class="cit-fpage">9864</span>–<span class="cit-lpage">9878</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1128/JVI.00915-10</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BVirol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.00915-10%26rft_id%253Dinfo%253Apmid%252F20660205%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI4NC8xOS85ODY0IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMC8zMzk1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3395.2" data-doi="10.1534/genetics.112.143164"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Aeschbacher</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. A.</span> <span class="cit-name-surname">Beaumont</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Futschik</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A novel approach for choosing summary statistics in approximate Bayesian computation.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">192</span>: <span class="cit-fpage">1027</span>–<span class="cit-lpage">1047</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.112.143164</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.143164%26rft_id%253Dinfo%253Apmid%252F22960215%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5Mi8zLzEwMjciO3M6NDoiYXRvbSI7czoxOToiL2dnZy85LzEwLzMzOTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3395.3" data-doi="10.1111/evo.13431"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Allman</span>,  <span class="cit-name-given-names">B. E.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. B.</span> <span class="cit-name-surname">Weissman</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Hitchhiking in space: Ancestry in adapting, spatially extended populations.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">72</span>: <span class="cit-fpage">722</span>–<span class="cit-lpage">734</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/evo.13431</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.volume%253D72%26rft.spage%253D722%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fevo.13431%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/evo.13431&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3395.4" data-doi="10.1017/S0016672300031748"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Barton</span>,  <span class="cit-name-given-names">N. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">The probability of fixation of a favoured allele in a subdivided population.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">62</span>: <span class="cit-fpage">149</span>–<span class="cit-lpage">157</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1017/S0016672300031748</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BRes.%26rft.volume%253D62%26rft.spage%253D149%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672300031748%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672300031748&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1993MJ62400007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3395.5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Beaumont</span>,  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Zhang</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. J.</span> <span class="cit-name-surname">Balding</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Approximate Bayesian computation in population genetics.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">162</span>: <span class="cit-fpage">2025</span>–<span class="cit-lpage">2035</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D162%26rft.spage%253D2025%26rft_id%253Dinfo%253Apmid%252F12524368%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=12524368&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3395.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000180502300043&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3395.6" data-doi="10.7554/eLife.39725"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Berg</span>,  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Harpak</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Sinnott-Armstrong</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. M.</span> <span class="cit-name-surname">Joergensen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Mostafavi</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">Reduced signal for polygenic adaptation of height in uk biobank.</span> <abbr class="cit-jnl-abbrev">Elife</abbr> <span class="cit-vol">8</span>: e39725.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.7554/eLife.39725</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DElife%26rft.volume%253D8%26rft.spage%253De39725%26rft_id%253Dinfo%253Adoi%252F10.7554%252FeLife.39725%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.7554/eLife.39725&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3395.7" data-doi="10.1111/j.1558-5646.2010.01050.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bierne</span>,  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The distinctive footprints of local hitchhiking in a varied environment and global hitchhiking in a subdivided population.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">64</span>: <span class="cit-fpage">3254</span>–<span class="cit-lpage">3272</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1558-5646.2010.01050.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.stitle%253DEvolution%26rft.aulast%253DBierne%26rft.auinit1%253DN.%26rft.volume%253D64%26rft.issue%253D11%26rft.spage%253D3254%26rft.epage%253D3272%26rft.atitle%253DThe%2Bdistinctive%2Bfootprints%2Bof%2Blocal%2Bhitchhiking%2Bin%2Ba%2Bvaried%2Benvironment%2Band%2Bglobal%2Bhitchhiking%2Bin%2Ba%2Bsubdivided%2Bpopulation.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1558-5646.2010.01050.x%26rft_id%253Dinfo%253Apmid%252F20550573%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1558-5646.2010.01050.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20550573&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3395.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000283377500016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3395.8" data-doi="10.1371/journal.pcbi.1003778"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bitbol</span>,  <span class="cit-name-given-names">A.-F.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. J.</span> <span class="cit-name-surname">Schwab</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Quantifying the role of population subdivision in evolution on rugged fitness landscapes.</span> <abbr class="cit-jnl-abbrev">PLOS Comput. Biol.</abbr> <span class="cit-vol">10</span>: <span class="cit-elocation-id">e1003778</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pcbi.1003778</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLOS%2BComput.%2BBiol.%26rft.volume%253D10%26rft.spage%253De1003778%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.1003778%26rft_id%253Dinfo%253Apmid%252F25122220%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.1003778&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25122220&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3395.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3395.9" data-doi="10.1534/genetics.107.085019"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bollback</span>,  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. L.</span> <span class="cit-name-surname">York</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Nielsen</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Estimation of 2nes from temporal allele frequency data.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">179</span>: <span class="cit-fpage">497</span>–<span class="cit-lpage">502</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.107.085019</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DBollback%26rft.auinit1%253DJ.%2BP.%26rft.volume%253D179%26rft.issue%253D1%26rft.spage%253D497%26rft.epage%253D502%26rft.atitle%253DEstimation%2Bof%2B2Nes%2BFrom%2BTemporal%2BAllele%2BFrequency%2BData%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.085019%26rft_id%253Dinfo%253Apmid%252F18493066%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTc5LzEvNDk3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMC8zMzk1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3395.10" data-doi="10.1371/journal.pgen.1005703"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bradburd</span>,  <span class="cit-name-given-names">G. S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. L.</span> <span class="cit-name-surname">Ralph</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">G. M.</span> <span class="cit-name-surname">Coop</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">A spatial framework for understanding population structure and admixture.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">12</span>: <span class="cit-elocation-id">e1005703</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1005703</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D12%26rft.spage%253De1005703%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1005703%26rft_id%253Dinfo%253Apmid%252F26771578%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1005703&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26771578&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3395.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3395.11" data-doi="10.1038/355511a0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Coyne</span>,  <span class="cit-name-given-names">J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">Genetics and speciation.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">355</span>: <span class="cit-fpage">511</span>–<span class="cit-lpage">515</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/355511a0</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DCoyne%26rft.auinit1%253DJ.%2BA.%26rft.volume%253D355%26rft.issue%253D6360%26rft.spage%253D511%26rft.epage%253D515%26rft.atitle%253DGenetics%2Band%2Bspeciation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F355511a0%26rft_id%253Dinfo%253Apmid%252F1741030%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/355511a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1741030&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3395.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3395.12" data-doi="10.1111/j.2041-210X.2011.00179.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Csilléry</span>,  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">O.</span> <span class="cit-name-surname">Francois</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. G. B.</span> <span class="cit-name-surname">Blum</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">abc: an R package for approximate Bayesian computation (ABC).</span> <abbr class="cit-jnl-abbrev">Methods Ecol. Evol.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">475</span>–<span class="cit-lpage">479</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.2041-210X.2011.00179.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2BEcol.%2BEvol.%26rft.volume%253D3%26rft.spage%253D475%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.2041-210X.2011.00179.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.2041-210X.2011.00179.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3395.13" data-doi="10.1534/genetics.106.067678"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Desai</span>,  <span class="cit-name-given-names">M. M.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. S.</span> <span class="cit-name-surname">Fisher</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Beneficial mutation–selection balance and the effect of linkage on positive selection.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">176</span>: <span class="cit-fpage">1759</span>–<span class="cit-lpage">1798</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.106.067678</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.106.067678%26rft_id%253Dinfo%253Apmid%252F17483432%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3Ni8zLzE3NTkiO3M6NDoiYXRvbSI7czoxOToiL2dnZy85LzEwLzMzOTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3395.14" data-doi="10.1177/117693430600200026"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ewing</span>,  <span class="cit-name-given-names">G.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Rodrigo</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Estimating population parameters using the structured serial coalescent with bayesian mcmc inference when some demes are hidden.</span> <abbr class="cit-jnl-abbrev">Evol. Bioinform. Online</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">227</span>–<span class="cit-lpage">235</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1177/117693430600200026</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvol.%2BBioinform.%2BOnline%26rft.volume%253D2%26rft.spage%253D227%26rft_id%253Dinfo%253Adoi%252F10.1177%252F117693430600200026%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1177/117693430600200026&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3395.15" data-doi="10.1371/journal.ppat.1006358"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Feder</span>,  <span class="cit-name-given-names">A. F.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Kline</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Polacino</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Cottrell</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. D.</span> <span class="cit-name-surname">Kashuba</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">A spatio-temporal assessment of simian/human immunodeficiency virus (SHIV) evolution reveals a highly dynamic process within the host.</span> <abbr class="cit-jnl-abbrev">PLoS Pathog.</abbr> <span class="cit-vol">13</span>: <span class="cit-elocation-id">e1006358</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.ppat.1006358</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BPathog.%26rft.volume%253D13%26rft.spage%253De1006358%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1006358%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1006358&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-book" id="cit-9.10.3395.16" data-doi="10.1534/genetics.113.158220"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Feder</span>,  <span class="cit-name-given-names">A. F.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Kryazhimskiy</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. B.</span> <span class="cit-name-surname">Plotkin</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Identifying signatures of selection in genetic time series.</span> <span class="cit-source">Genetics</span> <span class="cit-vol">196</span>: <span class="cit-fpage">509</span>–<span class="cit-lpage">522</span>. Erratum: 1559.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.113.158220</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.158220%26rft_id%253Dinfo%253Apmid%252F24318534%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk2LzIvNTA5IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMC8zMzk1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3395.17" data-doi="10.1038/nature02360"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hauert</span>,  <span class="cit-name-given-names">C.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Doebeli</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Spatial structure often inhibits the evolution of cooperation in the snowdrift game.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">428</span>: <span class="cit-fpage">643</span>–<span class="cit-lpage">646</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature02360</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DHauert%26rft.auinit1%253DC.%26rft.volume%253D428%26rft.issue%253D6983%26rft.spage%253D643%26rft.epage%253D646%26rft.atitle%253DSpatial%2Bstructure%2Boften%2Binhibits%2Bthe%2Bevolution%2Bof%2Bcooperation%2Bin%2Bthe%2Bsnowdrift%2Bgame.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature02360%26rft_id%253Dinfo%253Apmid%252F15074318%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature02360&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15074318&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3395.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000220697200041&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3395.18" data-doi="10.1534/genetics.104.036947"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hermisson</span>,  <span class="cit-name-given-names">J.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. S.</span> <span class="cit-name-surname">Pennings</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Soft sweeps: molecular population genetics of adaptation from standing genetic variation.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">169</span>: <span class="cit-fpage">2335</span>–<span class="cit-lpage">2352</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.104.036947</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.104.036947%26rft_id%253Dinfo%253Apmid%252F15716498%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2OS80LzIzMzUiO3M6NDoiYXRvbSI7czoxOToiL2dnZy85LzEwLzMzOTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3395.19" data-doi="10.1111/2041-210X.12808"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hermisson</span>,  <span class="cit-name-given-names">J.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. S.</span> <span class="cit-name-surname">Pennings</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Soft sweeps and beyond: understanding the patterns and probabilities of selection footprints under rapid adaptation.</span> <abbr class="cit-jnl-abbrev">Methods Ecol. Evol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">700</span>–<span class="cit-lpage">716</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/2041-210X.12808</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2BEcol.%2BEvol.%26rft.volume%253D8%26rft.spage%253D700%26rft_id%253Dinfo%253Adoi%252F10.1111%252F2041-210X.12808%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/2041-210X.12808&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-book" id="cit-9.10.3395.20" data-doi="10.1534/genetics.116.197566"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Iranmehr</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Akbari</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Schlötterer</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Bafna</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Clear: composition of likelihoods for evolve and resequence experiments.</span> <span class="cit-source">Genetics</span> <span class="cit-vol">206</span>: <span class="cit-fpage">1011</span>–<span class="cit-lpage">1023</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.116.197566</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.116.197566%26rft_id%253Dinfo%253Apmid%252F28396506%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjIwNi8yLzEwMTEiO3M6NDoiYXRvbSI7czoxOToiL2dnZy85LzEwLzMzOTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3395.21" data-doi="10.1534/genetics.112.144758"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jakobsson</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. D.</span> <span class="cit-name-surname">Edge</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">N. A.</span> <span class="cit-name-surname">Rosenberg</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The relationship between FST and the frequency of the most frequent allele.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">193</span>: <span class="cit-fpage">515</span>–<span class="cit-lpage">528</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.112.144758</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.144758%26rft_id%253Dinfo%253Apmid%252F23172852%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkzLzIvNTE1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMC8zMzk1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-book" id="cit-9.10.3395.22"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Karlin</span>,  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1976</span> <span class="cit-article-title">Population subdivision and selection migration interaction</span>, pp. <span class="cit-fpage">617</span>–<span class="cit-lpage">657</span> in <span class="cit-source">Population genetics and ecology</span>, <span class="cit-publ-name">Academic Press</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3395.23" data-doi="10.1534/genetics.111.130203"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>,  <span class="cit-name-given-names">Y.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Maruki</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Hitchhiking effect of a beneficial mutation spreading in a subdivided population.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">189</span>: <span class="cit-fpage">213</span>–<span class="cit-lpage">226</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.111.130203</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.130203%26rft_id%253Dinfo%253Apmid%252F21705748%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg5LzEvMjEzIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMC8zMzk1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-book" id="cit-9.10.3395.24" data-doi="10.1534/genetics.117.300417"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>,  <span class="cit-name-given-names">K. M.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Coop</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Distinguishing among modes of convergent adaptation using population genomic data.</span> <span class="cit-source">Genetics</span> <span class="cit-vol">207</span>: <span class="cit-fpage">1591</span>–<span class="cit-lpage">1619</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.117.300417</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.117.300417%26rft_id%253Dinfo%253Apmid%252F29046403%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjIwNy80LzE1OTEiO3M6NDoiYXRvbSI7czoxOToiL2dnZy85LzEwLzMzOTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3395.25" data-doi="10.1534/genetics.111.130112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Martens</span>,  <span class="cit-name-given-names">E. A.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">O.</span> <span class="cit-name-surname">Hallatschek</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Interfering waves of adaptation promote spatial mixing.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">189</span>: <span class="cit-fpage">1045</span>–<span class="cit-lpage">1060</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.111.130112</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.130112%26rft_id%253Dinfo%253Apmid%252F21900264%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE4OS8zLzEwNDUiO3M6NDoiYXRvbSI7czoxOToiL2dnZy85LzEwLzMzOTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3395.26" data-doi="10.1088/1367-2630/13/11/115014"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Martens</span>,  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Kostadinov</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. C.</span> <span class="cit-name-surname">Maley</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">O.</span> <span class="cit-name-surname">Hallatschek</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Spatial structure increases the waiting time for cancer.</span> <abbr class="cit-jnl-abbrev">New J. Phys.</abbr> <span class="cit-vol">13</span>: <span class="cit-elocation-id">115014</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1088/1367-2630/13/11/115014</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BJ.%2BPhys.%26rft.volume%253D13%26rft.spage%253D115014%26rft_id%253Dinfo%253Adoi%252F10.1088%252F1367-2630%252F13%252F11%252F115014%26rft_id%253Dinfo%253Apmid%252F22707911%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1088/1367-2630/13/11/115014&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22707911&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3395.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3395.27"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McLaren</span>,  <span class="cit-name-given-names">M. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Fitness-valley crossing in subdivided asexual populations.</span> <abbr class="cit-jnl-abbrev">bioRxiv</abbr>: 079624.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3395.28" data-doi="10.1038/nature07331"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Novembre</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Johnson</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Bryc</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Kutalik</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. R.</span> <span class="cit-name-surname">Boyko</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Genes mirror geography within europe.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">456</span>: <span class="cit-fpage">98</span>–<span class="cit-lpage">101</span>. Erratum: <span class="cit-fpage">98</span>–<span class="cit-lpage">101</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature07331</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DNovembre%26rft.auinit1%253DJ.%26rft.volume%253D456%26rft.issue%253D7218%26rft.spage%253D98%26rft.epage%253D101%26rft.atitle%253DGenes%2Bmirror%2Bgeography%2Bwithin%2BEurope.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature07331%26rft_id%253Dinfo%253Apmid%252F18758442%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature07331&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18758442&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3395.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000260674000048&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3395.29" data-doi="10.1371/journal.pgen.1007936"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Paulose</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Hermisson</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">O.</span> <span class="cit-name-surname">Hallatschek</span></span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">Spatial soft sweeps: patterns of adaptation in populations with long-range dispersal.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">15</span>: <span class="cit-elocation-id">e1007936</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1007936</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D15%26rft.spage%253De1007936%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1007936%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1007936&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3395.30" data-doi="10.1093/molbev/msj117"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pennings</span>,  <span class="cit-name-given-names">P. S.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Hermisson</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Soft sweeps II—molecular population genetics of adaptation from recurrent mutation or migration.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">1076</span>–<span class="cit-lpage">1084</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/msj117</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsj117%26rft_id%253Dinfo%253Apmid%252F16520336%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msj117&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16520336&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3395.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000237320800029&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3395.31" data-doi="10.1534/genetics.110.119594"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ralph</span>,  <span class="cit-name-given-names">P.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Coop</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Parallel adaptation: one or many waves of advance of an advantageous allele?</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">186</span>: <span class="cit-fpage">647</span>–<span class="cit-lpage">668</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.110.119594</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.119594%26rft_id%253Dinfo%253Apmid%252F20660645%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg2LzIvNjQ3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMC8zMzk1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3395.32" data-doi="10.1214/aos/1176346785"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rubin</span>,  <span class="cit-name-given-names">D. B.</span></span></li></ol><cite>, <span class="cit-pub-date">1984</span> <span class="cit-article-title">Bayesianly justifiable and relevant frequency calculations for the applied statistician.</span> <abbr class="cit-jnl-abbrev">Ann. Stat.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">1151</span>–<span class="cit-lpage">1172</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1214/aos/1176346785</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BStat.%26rft.volume%253D12%26rft.spage%253D1151%26rft_id%253Dinfo%253Adoi%252F10.1214%252Faos%252F1176346785%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1214/aos/1176346785&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1984TW89000001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-book" id="cit-9.10.3395.33" data-doi="10.1534/genetics.116.187278"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schraiber</span>,  <span class="cit-name-given-names">J. G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. N.</span> <span class="cit-name-surname">Evans</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Slatkin</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Bayesian inference of natural selection from allele frequency time series.</span> <span class="cit-source">Genetics</span> <span class="cit-vol">203</span>: <span class="cit-fpage">493</span>–<span class="cit-lpage">511</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.116.187278</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.116.187278%26rft_id%253Dinfo%253Apmid%252F27010022%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjAzLzEvNDkzIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMC8zMzk1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3395.34" data-doi="10.1371/journal.pcbi.1004845"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sheehan</span>,  <span class="cit-name-given-names">S.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y. S.</span> <span class="cit-name-surname">Song</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Deep learning for population genetic inference.</span> <abbr class="cit-jnl-abbrev">PLOS Comput. Biol.</abbr> <span class="cit-vol">12</span>: <span class="cit-elocation-id">e1004845</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pcbi.1004845</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLOS%2BComput.%2BBiol.%26rft.volume%253D12%26rft.spage%253De1004845%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.1004845%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.1004845&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3395.35" data-doi="10.1017/S001667239800319X"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Slatkin</span>,  <span class="cit-name-given-names">M.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Wiehe</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Genetic hitch-hiking in a subdivided population.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">71</span>: <span class="cit-fpage">155</span>–<span class="cit-lpage">160</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1017/S001667239800319X</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DSlatkin%26rft.auinit1%253DM.%26rft.volume%253D71%26rft.issue%253D2%26rft.spage%253D155%26rft.epage%253D160%26rft.atitle%253DGenetic%2Bhitch-hiking%2Bin%2Ba%2Bsubdivided%2Bpopulation.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS001667239800319X%26rft_id%253Dinfo%253Apmid%252F9717437%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S001667239800319X&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9717437&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3395.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000075333100007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3395.36" data-doi="10.1371/journal.pgen.1005069"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Terhorst</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Schlötterer</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y. S.</span> <span class="cit-name-surname">Song</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Multi-locus analysis of genomic time series data from experimental evolution.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-elocation-id">e1005069</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1005069</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D11%26rft.spage%253De1005069%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1005069%26rft_id%253Dinfo%253Apmid%252F25849855%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1005069&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25849855&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3395.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3395.37" data-doi="10.1371/journal.pgen.1007420"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Theys</span>,  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. F.</span> <span class="cit-name-surname">Feder</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Gelbart</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Hartl</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Stern</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Within-patient mutation frequencies reveal fitness costs of CpG dinucleotides and drastic amino acid changes in HIV.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">14</span>: <span class="cit-elocation-id">e1007420</span>. Erratum: 1007855.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1007420</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D14%26rft.spage%253De1007420%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1007420%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1007420&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3395.38"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Whitlock</span>,  <span class="cit-name-given-names">M. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Fixation probability and time in subdivided populations.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">164</span>: <span class="cit-fpage">767</span>–<span class="cit-lpage">779</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D164%26rft.spage%253D767%26rft_id%253Dinfo%253Apmid%252F12807795%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=12807795&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3395.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000183880000036&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3395.39"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Whitlock</span>,  <span class="cit-name-given-names">M. C.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. E.</span> <span class="cit-name-surname">McCauley</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> Indirect measures of gene flow and migration: F<sub>ST</sub> <sub>≠</sub> 1/(4Nm+ 1). Heredity 82: 117.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3395.40"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Williams</span>,  <span class="cit-name-given-names">K.-A.</span></span> and </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. S.</span> <span class="cit-name-surname">Pennings</span></span></li></ol><cite>, <span class="cit-pub-date">2019</span> Drug resistance evolution in hiv in the late 1990s: hard sweeps, soft sweeps, clonal interference and the accumulation of drug resistance mutations. bioRxiv 548198.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3395.41" data-doi="10.1534/genetics.116.191478"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wilson</span>,  <span class="cit-name-given-names">B. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. S.</span> <span class="cit-name-surname">Pennings</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. A.</span> <span class="cit-name-surname">Petrov</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Soft selective sweeps in evolutionary rescue.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">205</span>: <span class="cit-fpage">1573</span>–<span class="cit-lpage">1586</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.116.191478</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.116.191478%26rft_id%253Dinfo%253Apmid%252F28213477%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjIwNS80LzE1NzMiO3M6NDoiYXRvbSI7czoxOToiL2dnZy85LzEwLzMzOTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3395.42" data-doi="10.1111/j.1469-1809.1949.tb02451.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>,  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1949</span> <span class="cit-article-title">The genetical structure of populations.</span> <abbr class="cit-jnl-abbrev">Ann. Eugen.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">323</span>–<span class="cit-lpage">354</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1469-1809.1949.tb02451.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BEugen.%26rft.volume%253D15%26rft.spage%253D323%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1469-1809.1949.tb02451.x%26rft_id%253Dinfo%253Apmid%252F24540312%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1469-1809.1949.tb02451.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24540312&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3395.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3395.43" data-doi="10.1146/annurev.ge.16.120182.000245"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>,  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1982</span> <span class="cit-article-title">The shifting balance theory and macroevolution.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Genet.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">20</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1146/annurev.ge.16.120182.000245</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bgenetics%26rft.stitle%253DAnnu%2BRev%2BGenet%26rft.aulast%253DWright%26rft.auinit1%253DS.%26rft.volume%253D16%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.ge.16.120182.000245%26rft_id%253Dinfo%253Apmid%252F6760797%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.ge.16.120182.000245&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=6760797&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3395.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1982PV24400001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3395.44" data-doi="10.1093/ve/vex003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zanini</span>,  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Puller</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Brodin</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Albert</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. A.</span> <span class="cit-name-surname">Neher</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">In vivo mutation rates and the landscape of fitness costs of hiv-1.</span> <abbr class="cit-jnl-abbrev">Virus Evol.</abbr> <span class="cit-vol">3</span>: <span class="cit-elocation-id">vex003</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/ve/vex003</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirus%2BEvol.%26rft.volume%253D3%26rft.spage%253Dvex003%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fve%252Fvex003%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/ve/vex003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/9/10/3395.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/10/3381" title="A Bayesian Genomic Multi-output Regressor Stacking Model for Predicting Multi-trait Multi-environment Plant Breeding Data" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/10/3409" title="Utilization of Tissue Ploidy Level Variation in de Novo Transcriptome Assembly of Pinus sylvestris" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/9/10"><span>Volume 9 Issue 10, October 2019</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/9/10.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/9/10.cover-source.jpg?itok=yA8kifEd" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 9 (10)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/10/3395" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-nGImKaGroMhgRMddhZLCQzkDFiTNBlkx1NExPuLva_w" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/9/10/3395" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/9/10/3395" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/9/10/3395" class="active" data-icon-position="" data-hide-link-title="0">Evolutionary Dynamics in Structured Populations Under Strong Population Genetic Forces</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/494279" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-UwZP7Om5IWxiXnpw-coNdzR8MrdYkYPzvcRIuTrJzkk" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/10/3395.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/9/10/3395" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-YjVG_HL-vEJ6D5d0xQQTi3peR8TxEkGfiOKsyQ06JL0" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/9/10/3395" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/10/3395.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494279" id="node494279--2" data-pisa="ggg;9/10/3395" data-pisa-master="ggg;g3.119.400605" data-apath="/ggg/9/10/3395.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Evolutionary Dynamics in Structured Populations Under Strong Population Genetic Forces</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-2915-089X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Alison F. Feder</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-8704-6578" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Pleuni S. Pennings</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0001-7476-9283" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Joachim Hermisson</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-3664-9130" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Dmitri A. Petrov</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3395-3407;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400605 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494279--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Alison F. Feder </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, Stanford University,</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Integrative Biology, University of California Berkeley,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alison%2BF.%2BFeder%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Feder%20AF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAlison%2BF.%2BFeder%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-2915-089X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Alison F. Feder</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:affeder@berkeley.edu" class="" data-icon-position="" data-hide-link-title="0">affeder@berkeley.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Pleuni S. Pennings </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Biology, San Francisco State University, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Pleuni%2BS.%2BPennings%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pennings%20PS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APleuni%2BS.%2BPennings%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-8704-6578" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Pleuni S. Pennings</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Joachim Hermisson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Faculty of Mathematics, University of Vienna</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joachim%2BHermisson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hermisson%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJoachim%2BHermisson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-7476-9283" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Joachim Hermisson</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Dmitri A. Petrov </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, Stanford University,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dmitri%2BA.%2BPetrov%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Petrov%20DA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADmitri%2BA.%2BPetrov%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-3664-9130" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Dmitri A. Petrov</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/9/10/3395" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/9/10/3395&amp;title=Evolutionary%20Dynamics%20in%20Structured%20Populations%20Under%20Strong%20Population%20Genetic%20Forces" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/9/10/3395&amp;title=Evolutionary%20Dynamics%20in%20Structured%20Populations%20Under%20Strong%20Population%20Genetic%20Forces" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/9/10/3395&amp;title=Evolutionary%20Dynamics%20in%20Structured%20Populations%20Under%20Strong%20Population%20Genetic%20Forces" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/9/10/3395&amp;text=Evolutionary%20Dynamics%20in%20Structured%20Populations%20Under%20Strong%20Population%20Genetic%20Forces" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/9/10/3395&amp;title=Evolutionary%20Dynamics%20in%20Structured%20Populations%20Under%20Strong%20Population%20Genetic%20Forces" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/9/10/3395&amp;t=Evolutionary%20Dynamics%20in%20Structured%20Populations%20Under%20Strong%20Population%20Genetic%20Forces" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/9/10/3395&amp;title=Evolutionary%20Dynamics%20in%20Structured%20Populations%20Under%20Strong%20Population%20Genetic%20Forces" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/9/10/3395&amp;title=Evolutionary%20Dynamics%20in%20Structured%20Populations%20Under%20Strong%20Population%20Genetic%20Forces" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/494279/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494279" id="node494279--3" data-pisa="ggg;9/10/3395" data-pisa-master="ggg;g3.119.400605" data-apath="/ggg/9/10/3395.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Evolutionary Dynamics in Structured Populations Under Strong Population Genetic Forces</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-2915-089X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Alison F. Feder</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-8704-6578" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Pleuni S. Pennings</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0001-7476-9283" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Joachim Hermisson</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-3664-9130" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Dmitri A. Petrov</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3395-3407;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400605 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494279--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Alison F. Feder </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, Stanford University,</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Integrative Biology, University of California Berkeley,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alison%2BF.%2BFeder%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Feder%20AF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAlison%2BF.%2BFeder%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-2915-089X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Alison F. Feder</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:affeder@berkeley.edu" class="" data-icon-position="" data-hide-link-title="0">affeder@berkeley.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Pleuni S. Pennings </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Biology, San Francisco State University, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Pleuni%2BS.%2BPennings%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pennings%20PS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APleuni%2BS.%2BPennings%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-8704-6578" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Pleuni S. Pennings</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Joachim Hermisson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Faculty of Mathematics, University of Vienna</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joachim%2BHermisson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hermisson%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJoachim%2BHermisson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-7476-9283" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Joachim Hermisson</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Dmitri A. Petrov </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, Stanford University,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dmitri%2BA.%2BPetrov%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Petrov%20DA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADmitri%2BA.%2BPetrov%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-3664-9130" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Dmitri A. Petrov</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=3395&amp;rft.epage=3407&amp;rft.atitle=Evolutionary%20Dynamics%20in%20Structured%20Populations%20Under%20Strong%20Population%20Genetic%20Forces&amp;rft.volume=9&amp;rft.issue=10&amp;rft.date=2019-10-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Feder%2C+Alison+F.&amp;rft.au=Pennings%2C+Pleuni+S.&amp;rft.au=Hermisson%2C+Joachim&amp;rft.au=Petrov%2C+Dmitri+A."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/494279/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/494279/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/494279/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/494279/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/494279/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/494279/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/494279/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/494279/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/494279/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/494279/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/494279/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/494279/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/9/10/3395" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-12" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results and Discussion</a></li><li class="even"><a href="#sec-21" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/9/10/3395.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/9/10/3395.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/9/10/3395.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
