<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/9/10/3129" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/9/10/3129.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/9/10/3129.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/9/10/3129.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/9/10/3129/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/9/10/3129.atom" />
<meta name="HW.pisa" content="ggg;9/10/3129" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="QTG-Finder: A Machine-Learning Based Algorithm To Prioritize Causal Genes of Quantitative Trait Loci in Arabidopsis and Rice" />
<meta name="DC.Identifier" content="10.1534/g3.119.400319" />
<meta name="DC.Date" content="2019-10-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2019 Lin et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Linkage mapping is one of the most commonly used methods to identify genetic loci that determine a trait. However, the loci identified by linkage mapping may contain hundreds of candidate genes and require a time-consuming and labor-intensive fine mapping process to find the causal gene controlling the trait. With the availability of a rich assortment of genomic and functional genomic data, it is possible to develop a computational method to facilitate faster identification of causal genes. We developed QTG-Finder, a machine learning based algorithm to prioritize causal genes by ranking genes within a quantitative trait locus (QTL). Two predictive models were trained separately based on known causal genes in Arabidopsis and rice. An independent validation analysis showed that the models could recall about 64% of Arabidopsis and 79% of rice causal genes when the top 20% ranked genes were considered. The top 20% ranked genes can range from 10 to 100 genes, depending on the size of a QTL. The models can prioritize different types of traits though at different efficiency. We also identified several important features of causal genes including paralog copy number, being a transporter, being a transcription factor, and containing SNPs that cause premature stop codon. This work lays the foundation for systematically understanding characteristics of causal genes and establishes a pipeline to predict causal genes based on public data." />
<meta name="DC.Contributor" content="Fan Lin" />
<meta name="DC.Contributor" content="Jue Fan" />
<meta name="DC.Contributor" content="Seung Y. Rhee" />
<meta name="article:published_time" content="2019-10-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="QTG-Finder: A Machine-Learning Based Algorithm To Prioritize Causal Genes of Quantitative Trait Loci in Arabidopsis and Rice" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Linkage mapping is one of the most commonly used methods to identify genetic loci that determine a trait. However, the loci identified by linkage mapping may contain hundreds of candidate genes and require a time-consuming and labor-intensive fine mapping process to find the causal gene controlling the trait. With the availability of a rich assortment of genomic and functional genomic data, it is possible to develop a computational method to facilitate faster identification of causal genes. We developed QTG-Finder, a machine learning based algorithm to prioritize causal genes by ranking genes within a quantitative trait locus (QTL). Two predictive models were trained separately based on known causal genes in Arabidopsis and rice. An independent validation analysis showed that the models could recall about 64% of Arabidopsis and 79% of rice causal genes when the top 20% ranked genes were considered. The top 20% ranked genes can range from 10 to 100 genes, depending on the size of a QTL. The models can prioritize different types of traits though at different efficiency. We also identified several important features of causal genes including paralog copy number, being a transporter, being a transcription factor, and containing SNPs that cause premature stop codon. This work lays the foundation for systematically understanding characteristics of causal genes and establishes a pipeline to predict causal genes based on public data.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2019/10/01" />
<meta name="citation_mjid" content="ggg;9/10/3129" />
<meta name="citation_id" content="9/10/3129" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/9/10/3129" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/9/10/3129.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/9/10/3129.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/9/10/3129.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.119.400319" />
<meta name="citation_pmid" content="31358562" />
<meta name="citation_volume" content="9" />
<meta name="citation_issue" content="10" />
<meta name="citation_num_pages" content="10" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="3129" />
<meta name="citation_lastpage" content="3138" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Fan Lin" />
<meta name="citation_author_institution" content="Department of Plant Biology, Carnegie Institution for Science, Stanford, California 94305" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-3742-5291" />
<meta name="citation_author" content="Jue Fan" />
<meta name="citation_author_institution" content="Department of Plant Biology, Carnegie Institution for Science, Stanford, California 94305" />
<meta name="citation_author" content="Seung Y. Rhee" />
<meta name="citation_author_institution" content="Department of Plant Biology, Carnegie Institution for Science, Stanford, California 94305" />
<meta name="citation_author_email" content="srhee@carnegiescience.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-7572-4762" />
<meta name="citation_reference" content="citation_journal_title=Trends Plant Sci.;citation_author=J. L. Araus;citation_author=S. C. Kefauver;citation_author=M. Zaman-Allah;citation_author=M. S. Olsen;citation_author=J. E. Cairns;citation_title=Translating High-Throughput Phenotyping into Genetic Gain.;citation_pages=451-466;citation_volume=23;citation_year=2018;citation_doi=10.1016/j.tplants.2018.02.001" />
<meta name="citation_reference" content="citation_journal_title=BMC Plant Biol.;citation_author=J. W. Bargsten;citation_author=J. P. Nap;citation_author=G. F. Sanchez-Perez;citation_author=A. D. van Dijk;citation_title=Prioritization of candidate genes in QTL regions based on associations between traits and biological processes.;citation_pages=330;citation_volume=14;citation_year=2014;citation_doi=10.1186/s12870-014-0330-3" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=I. Baxter;citation_author=J. N. Brazelton;citation_author=D. Yu;citation_author=Y. S. Huang;citation_author=B. Lahner;citation_title=A coastal cline in sodium accumulation in Arabidopsis thaliana is driven by natural variation of the sodium transporter AtHKT1;1.;citation_pages=e1001193-e1001193;citation_volume=6;citation_year=2010;citation_issue=11;citation_pmid=21085628;citation_doi=10.1371/journal.pgen.1001193" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=L. Bentsink;citation_author=J. Hanson;citation_author=C. J. Hanhart;citation_author=H. Blankestijn-de Vries;citation_author=C. Coltrane;citation_title=Natural variation for seed dormancy in Arabidopsis is regulated by additive genetic and molecular pathways.;citation_volume=107;citation_year=2010;citation_pmid=20145108;citation_doi=10.1073/pnas.1000410107" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=J. Bergelson;citation_author=F. Roux;citation_title=Towards identifying genes underlying ecologically relevant traits in Arabidopsis thaliana.;citation_pages=867-879;citation_volume=11;citation_year=2010;citation_issue=12;citation_pmid=21085205;citation_doi=10.1038/nrg2896" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=B. Brachi;citation_author=N. Faure;citation_author=M. Horton;citation_author=E. Flahauw;citation_author=A. Vazquez;citation_title=Linkage and association mapping of Arabidopsis thaliana flowering time in nature.;citation_pages=e1000940-e1000940;citation_volume=6;citation_year=2010;citation_issue=5;citation_pmid=20463887;citation_doi=10.1371/journal.pgen.1000940" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=E. S. Buckler;citation_author=J. B. Holland;citation_author=P. J. Bradbury;citation_author=C. B. Acharya;citation_author=P. J. Brown;citation_title=The Genetic Architecture of Maize Flowering Time;citation_pages=714-718;citation_volume=325;citation_year=2009;citation_issue=5941;citation_pmid=19661422;citation_doi=10.1126/science.1174276" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=O. Carlborg;citation_author=C. S. Haley;citation_title=Epistasis: too often neglected in complex trait studies?;citation_pages=618-625;citation_volume=5;citation_year=2004;citation_issue=8;citation_pmid=15266344;citation_doi=10.1038/nrg1407" />
<meta name="citation_reference" content="citation_journal_abbrev=Fly;citation_author=P. Cingolani;citation_author=A. Platts;citation_author=L. L. Wang;citation_author=M. Coon;citation_author=T. Nguyen;citation_title=A program for annotating and predicting the effects of single nucleotide polymorphisms, SnpEff: SNPs in the genome of Drosophila melanogaster strain w1118; iso-2; iso-3.;citation_pages=80-92;citation_volume=6;citation_year=2012;citation_issue=2;citation_pmid=22728672;citation_doi=10.4161/fly.19695" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_title=1,135 Genomes Reveal the Global Pattern of Polymorphism in Arabidopsis thaliana.;citation_pages=481-491;citation_volume=166;citation_year=2016;citation_pmid=27293186;citation_doi=10.1016/j.cell.2016.05.063" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=G. L. Conte;citation_author=M. E. Arnegard;citation_author=J. Best;citation_author=Y. F. Chan;citation_author=F. C. Jones;citation_title=Extent of QTL Reuse During Repeated Phenotypic Divergence of Sympatric Threespine Stickleback.;citation_volume=201;citation_year=2015;citation_pmid=26384359;citation_doi=10.1534/genetics.115.182550" />
<meta name="citation_reference" content="citation_journal_title=Proc. Biol. Sci.;citation_author=G. L. Conte;citation_author=M. E. Arnegard;citation_author=C. L. Peichel;citation_author=D. Schluter;citation_title=The probability of genetic parallelism and convergence in natural populations.;citation_volume=279;citation_year=2012;citation_pmid=23075840;citation_doi=10.1098/rspb.2012.2146" />
<meta name="citation_reference" content="citation_journal_title=Mach. Learn.;citation_author=C. Cortes;citation_author=V. Vapnik;citation_title=Support-vector networks.;citation_pages=273-297;citation_volume=20;citation_year=1995;citation_doi=10.1007/BF00994018" />
<meta name="citation_reference" content="citation_journal_title=Front. Plant Sci.;citation_author=A. V. Daware;citation_author=R. Srivastava;citation_author=A. K. Singh;citation_author=S. K. Parida;citation_author=A. K. Tyagi;citation_title=Regional Association Analysis of MetaQTLs Delineates Candidate Grain Size Genes in Rice.;citation_pages=807;citation_volume=8;citation_year=2017;citation_doi=10.3389/fpls.2017.00807" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=R. C. Deo;citation_author=G. Musso;citation_author=M. Tasan;citation_author=P. Tang;citation_author=A. Poon;citation_title=Prioritizing causal disease genes using unbiased genomic features.;citation_pages=534;citation_volume=15;citation_year=2014;citation_pmid=25633252;citation_doi=10.1186/s13059-014-0534-8" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. J. Dinka;citation_author=M. A. Campbell;citation_author=T. Demers;citation_author=M. N. Raizada;citation_title=Predicting the size of the progeny mapping population required to positionally clone a gene.;citation_volume=176;citation_year=2007;citation_pmid=17565938;citation_doi=10.1534/genetics.107.074377" />
<meta name="citation_reference" content="citation_journal_title=Curr. Opin. Plant Biol.;citation_author=N. Fahlgren;citation_author=M. A. Gehan;citation_author=I. Baxter;citation_title=Lights, camera, action: high-throughput plant phenotyping is ready for a close-up.;citation_pages=93-99;citation_volume=24;citation_year=2015;citation_pmid=25733069;citation_doi=10.1016/j.pbi.2015.02.006" />
<meta name="citation_reference" content="The Food and Agriculture Organization, 2009 How to feed the world in 2050. Available at: http://www.fao.org/fileadmin/templates/wsfs/docs/expert_paper/How_to_Feed_the_World_in_2050.pdf." />
<meta name="citation_reference" content="citation_journal_title=Nat. Commun.;citation_author=S. Gelfman;citation_author=Q. Wang;citation_author=K. M. McSweeney;citation_author=Z. Ren;citation_author=F. La Carpia;citation_title=Annotating pathogenic non-coding variants in genic regions.;citation_pages=236;citation_volume=8;citation_year=2017;citation_doi=10.1038/s41467-017-00141-2" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=S. Gotz;citation_author=J. M. Garcia-Gomez;citation_author=J. Terol;citation_author=T. D. Williams;citation_author=S. H. Nagaraj;citation_title=High-throughput functional annotation and data mining with the Blast2GO suite.;citation_volume=36;citation_year=2008;citation_pmid=18445632;citation_doi=10.1093/nar/gkn176" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=C. E. Grant;citation_author=T. L. Bailey;citation_author=W. S. Noble;citation_title=FIMO: scanning for occurrences of a given motif.;citation_volume=27;citation_year=2011;citation_pmid=21330290;citation_doi=10.1093/bioinformatics/btr064" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=Y. Hattori;citation_author=K. Nagai;citation_author=S. Furukawa;citation_author=X.-J. Song;citation_author=R. Kawano;citation_title=The ethylene response factors SNORKEL1 and SNORKEL2 allow rice to adapt to deep water.;citation_pages=1026-1030;citation_volume=460;citation_year=2009;citation_issue=7258;citation_pmid=19693083;citation_doi=10.1038/nature08258" />
<meta name="citation_reference" content="He, H., 2014 Environmental Regulation of Seed Performance. Dissertation. Wageningen University, Wageningen University." />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=L. A. Hindorff;citation_author=P. Sethupathy;citation_author=H. A. Junkins;citation_author=E. M. Ramos;citation_author=J. P. Mehta;citation_title=Potential etiologic and functional implications of genome-wide association loci for human diseases and traits.;citation_volume=106;citation_year=2009;citation_pmid=19474294;citation_doi=10.1073/pnas.0903103106" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=F. Hormozdiari;citation_author=G. Kichaev;citation_author=W. Y. Yang;citation_author=B. Pasaniuc;citation_author=E. Eskin;citation_title=Identification of causal genes for complex traits.;citation_volume=31;citation_year=2015;citation_pmid=26072484;citation_doi=10.1093/bioinformatics/btv240" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=Y. F. Huang;citation_author=B. Gulko;citation_author=A. Siepel;citation_title=Fast, scalable prediction of deleterious noncoding variants from functional and population genomic data.;citation_pages=618-624;citation_volume=49;citation_year=2017;citation_pmid=28288115;citation_doi=10.1038/ng.3810" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=K. A. Jagadeesh;citation_author=A. M. Wenger;citation_author=M. J. Berger;citation_author=H. Guturu;citation_author=P. D. Stenson;citation_title=M-CAP eliminates a majority of variants of uncertain significance in clinical exomes at high sensitivity.;citation_pages=1581-1586;citation_volume=48;citation_year=2016;citation_pmid=27776117;citation_doi=10.1038/ng.3703" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=C. E. Jones;citation_author=A. L. Brown;citation_author=U. Baumann;citation_title=Estimating the annotation error rate of curated GO database sequence annotations.;citation_pages=170-170;citation_volume=8;citation_year=2007;citation_pmid=17519041;citation_doi=10.1186/1471-2105-8-170" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=P. Jones;citation_author=D. Binns;citation_author=H. Y. Chang;citation_author=M. Fraser;citation_author=W. Li;citation_title=InterProScan 5: genome-scale protein function classification.;citation_volume=30;citation_year=2014;citation_pmid=24451626;citation_doi=10.1093/bioinformatics/btu031" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=M. Kircher;citation_author=D. M. Witten;citation_author=P. Jain;citation_author=B. J. O’Roak;citation_author=G. M. Cooper;citation_title=A general framework for estimating the relative pathogenicity of human genetic variants.;citation_pages=310-315;citation_volume=46;citation_year=2014;citation_pmid=24487276;citation_doi=10.1038/ng.2892" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=I. Lee;citation_author=B. Ambaru;citation_author=P. Thakkar;citation_author=E. M. Marcotte;citation_author=S. Y. Rhee;citation_title=Rational association of genes with traits using a genome-scale gene network for Arabidopsis thaliana.;citation_pages=149-156;citation_volume=28;citation_year=2010;citation_issue=2;citation_pmid=20118918;citation_doi=10.1038/nbt.1603" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=I. Lee;citation_author=Y. S. Seo;citation_author=D. Coltrane;citation_author=S. Hwang;citation_author=T. Oh;citation_title=Genetic dissection of the biotic stress response using a genome-scale gene network for rice.;citation_volume=108;citation_year=2011;citation_pmid=22042862;citation_doi=10.1073/pnas.1110384108" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=P. H. Leinonen;citation_author=D. L. Remington;citation_author=J. Leppala;citation_author=O. Savolainen;citation_title=Genetic basis of local adaptation and flowering time variation in Arabidopsis lyrata.;citation_pages=709-723;citation_volume=22;citation_year=2013;citation_doi=10.1111/j.1365-294X.2012.05678.x" />
<meta name="citation_reference" content="citation_journal_title=Molecular genetics and genomics :  MGG;citation_journal_abbrev=Mol Genet Genomics;citation_author=X. Q. Liu;citation_author=L. Wang;citation_author=S. Chen;citation_author=F. Lin;citation_author=Q. H. Pan;citation_title=Genetic and physical mapping of Pi36(t), a novel rice blast resistance gene located on rice chromosome 8.;citation_pages=394-401;citation_volume=274;citation_year=2005;citation_issue=4;citation_pmid=16151856;citation_doi=10.1007/s00438-005-0032-5" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell;citation_author=J. P. Lloyd;citation_author=A. E. Seddon;citation_author=G. D. Moghe;citation_author=M. C. Simenc;citation_author=S.-H. Shiu;citation_title=Characteristics of Plant Essential Genes Allow for within- and between-Species Prediction of Lethal Mutant Phenotypes.;citation_volume=27;citation_year=2015;citation_pmid=26286535;citation_doi=10.1105/tpc.15.00051" />
<meta name="citation_reference" content="citation_journal_title=Annual review of genetics;citation_journal_abbrev=Annu Rev Genet;citation_author=T. F. C. Mackay;citation_title=THE GENETIC ARCHITECTURE OF QUANTITATIVE TRAITS;citation_pages=303-339;citation_volume=35;citation_year=2001;citation_issue=1;citation_pmid=11700286;citation_doi=10.1146/annurev.genet.35.102401.090633" />
<meta name="citation_reference" content="citation_journal_title=Nat. Rev. Genet.;citation_author=T. F. C. Mackay;citation_title=Epistasis and Quantitative Traits: Using Model Organisms to Study Gene-Gene Interactions.;citation_pages=22-33;citation_volume=15;citation_year=2014;citation_pmid=24296533;citation_doi=10.1038/nrg3627" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=L. Mansueto;citation_author=R. R. Fuentes;citation_author=F. N. Borja;citation_author=J. Detras;citation_author=J. M. Abriol-Santos;citation_title=Rice SNP-seek database update: new SNPs, indels, and queries.;citation_volume=45;citation_year=2017;citation_pmid=27899667;citation_doi=10.1093/nar/gkw1135" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_author=A. Martin;citation_author=V. Orgogozo;citation_title=The Loci of repeated evolution: a catalog of genetic hotspots of phenotypic variation.;citation_pages=1235-1250;citation_volume=67;citation_year=2013;citation_pmid=23617905;citation_doi=10.1111/evo.12081" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=F. Mordelet;citation_author=J. P. Vert;citation_title=ProDiGe: Prioritization Of Disease Genes with multitask machine learning from positive and unlabeled examples.;citation_pages=389-389;citation_volume=12;citation_year=2011;citation_pmid=21977986;citation_doi=10.1186/1471-2105-12-389" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=H. Motte;citation_author=A. Vercauteren;citation_author=S. Depuydt;citation_author=S. Landschoot;citation_author=D. Geelen;citation_title=Combining linkage and association mapping identifies RECEPTOR-LIKE PROTEIN KINASE1 as an essential Arabidopsis shoot regeneration gene.;citation_volume=111;citation_year=2014;citation_pmid=24850864;citation_doi=10.1073/pnas.1404978111" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell;citation_author=B. T. Moyers;citation_title=Camoco: A Net for the Sea of Candidate Genes.;citation_volume=30;citation_year=2018;citation_pmid=30509899;citation_doi=10.1105/tpc.18.00908" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=P. C. Ng;citation_author=S. Henikoff;citation_title=SIFT: predicting amino acid changes that affect protein function;citation_pages=3812-3814;citation_volume=31;citation_year=2003;citation_issue=13;citation_pmid=12824425;citation_doi=10.1093/nar/gkg509" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=S. V. Nuzhdin;citation_author=C. L. Dilda;citation_author=T. F. Mackay;citation_title=The Genetic Architecture of Selection Response: Inferences From Fine-Scale Mapping of Bristle Number Quantitative Trait Loci in Drosophila melanogaster;citation_pages=1317-1331;citation_volume=153;citation_year=1999;citation_issue=3;citation_pmid=10545462" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=S. P. Otto;citation_author=C. D. Jones;citation_title=Detecting the Undetected: Estimating the Total Number of Loci Underlying a Quantitative Trait;citation_pages=2093-2107;citation_volume=156;citation_year=2000;citation_issue=4;citation_pmid=11102398" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiol.;citation_author=N. Panchy;citation_author=M. Lehti-Shiu;citation_author=S. H. Shiu;citation_title=Evolution of Gene Duplication in Plants.;citation_volume=171;citation_year=2016;citation_pmid=27288366;citation_doi=10.1104/pp.16.00523" />
<meta name="citation_reference" content="citation_journal_title=J. Mach. Learn. Res.;citation_author=F. Pedregosa;citation_author=G. Varoquaux;citation_author=A. Gramfort;citation_author=V. Michel;citation_author=B. Thirion;citation_title=Scikit-learn: Machine Learning in Python.;citation_pages=2825-2830;citation_volume=12;citation_year=2011;citation_pmid=23755062;citation_doi=10.1016/j.cpc.2010.04.018" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=C. Perez-Iratxeta;citation_author=P. Bork;citation_author=M. A. Andrade;citation_title=Association of genes to genetically inherited diseases using data mining.;citation_pages=316-319;citation_volume=31;citation_year=2002;citation_issue=3;citation_pmid=12006977;citation_doi=10.1038/ng895" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=K. S. Pollard;citation_author=M. J. Hubisz;citation_author=K. R. Rosenbloom;citation_author=A. Siepel;citation_title=Detection of nonneutral substitution rates on mammalian phylogenies.;citation_volume=20;citation_year=2010;citation_pmid=19858363;citation_doi=10.1101/gr.097857.109" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=A. L. Price;citation_author=N. A. Zaitlen;citation_author=D. Reich;citation_author=N. Patterson;citation_title=New approaches to population stratification in genome-wide association studies.;citation_pages=459-463;citation_volume=11;citation_year=2010;citation_issue=7;citation_pmid=20548291;citation_doi=10.1038/nrg2813" />
<meta name="citation_reference" content="citation_journal_title=Nat. Methods;citation_author=G. R. Ritchie;citation_author=I. Dunham;citation_author=E. Zeggini;citation_author=P. Flicek;citation_title=Functional annotation of noncoding sequence variants.;citation_pages=294-296;citation_volume=11;citation_year=2014;citation_pmid=24487584;citation_doi=10.1038/nmeth.2832" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell;citation_author=R. J. Schaefer;citation_author=J. M. Michno;citation_author=J. Jeffers;citation_author=O. Hoekenga;citation_author=B. Dilkes;citation_title=Integrating coexpression networks with GWAS to prioritize causal genes in maize.;citation_volume=30;citation_year=2018;citation_pmid=30413654;citation_doi=10.1105/tpc.18.00299" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiol.;citation_author=P. Schläpfer;citation_author=P. Zhang;citation_author=C. Wang;citation_author=T. Kim;citation_author=M. Banf;citation_title=Genome-Wide Prediction of Metabolic Enzymes, Pathways, and Gene Clusters in Plants [erratum in Plant Physiol. 176: 2583].;citation_volume=173;citation_year=2017;citation_pmid=28228535;citation_doi=10.1104/pp.16.01942" />
<meta name="citation_reference" content="citation_journal_title=Trends in genetics :  TIG;citation_journal_abbrev=Trends Genet;citation_author=A. B. Singleton;citation_author=J. Hardy;citation_author=B. J. Traynor;citation_author=H. Houlden;citation_title=Towards a complete resolution of the genetic architecture of disease.;citation_pages=438-442;citation_volume=26;citation_year=2010;citation_issue=10;citation_pmid=20813421;citation_doi=10.1016/j.tig.2010.07.004" />
<meta name="citation_reference" content="citation_journal_title=Plant Signal. Behav.;citation_author=R. Takahashi;citation_author=K. Bashir;citation_author=Y. Ishimaru;citation_author=N. K. Nishizawa;citation_author=H. Nakanishi;citation_title=The role of heavy-metal ATPases, HMAs, in zinc and cadmium transport in rice.;citation_pages=1605-1607;citation_volume=7;citation_year=2012;citation_pmid=23072989;citation_doi=10.4161/psb.22454" />
<meta name="citation_reference" content="citation_journal_title=IEEE Trans. Pattern Anal. Mach. Intell.;citation_author=H. Tin Kam;citation_title=The random subspace method for constructing decision forests.;citation_pages=832-844;citation_volume=20;citation_year=1998;citation_doi=10.1109/34.709601" />
<meta name="citation_reference" content="citation_journal_title=Brief. Bioinform.;citation_author=W. G. Touw;citation_author=J. R. Bayjanov;citation_author=L. Overmars;citation_author=L. Backus;citation_author=J. Boekhorst;citation_title=Data mining in the Life Sciences with Random Forest: a walk in the park or lost in the jungle?;citation_volume=14;citation_year=2013;citation_pmid=22786785;citation_doi=10.1093/bib/bbs034" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=M. R. Tuinstra;citation_author=G. Ejeta;citation_author=P. B. Goldsbrough;citation_title=Heterogeneous inbred family (HIF) analysis: a method for developing near-isogenic lines that differ at quantitative trait loci.;citation_pages=1005-1011;citation_volume=95;citation_year=1997;citation_doi=10.1007/s001220050654" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell;citation_author=J. Van de Velde;citation_author=K. S. Heyndrickx;citation_author=K. Vandepoele;citation_title=Inference of Transcriptional Networks in Arabidopsis through Conserved Noncoding Sequence Analysis.;citation_volume=26;citation_year=2014;citation_pmid=24989046;citation_doi=10.1105/tpc.114.127001" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiology;citation_journal_abbrev=Plant Physiol.;citation_author=D. Weigel;citation_author=M. Nordborg;citation_title=Natural Variation in Arabidopsis. How Do We Find the Causal Genes?;citation_pages=567-568;citation_volume=138;citation_year=2005;citation_issue=2;citation_pmid=15955918;citation_doi=10.1104/pp.104.900157" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_author=M. T. Weirauch;citation_author=A. Yang;citation_author=M. Albu;citation_author=A. G. Cote;citation_author=A. Montenegro-Montero;citation_title=Determination and Inference of Eukaryotic Transcription Factor Sequence Specificity.;citation_pages=1431-1443;citation_volume=158;citation_year=2014;citation_pmid=25215497;citation_doi=10.1016/j.cell.2014.08.009" />
<meta name="citation_reference" content="citation_journal_title=Trends Genet.;citation_author=M. Wellenreuther;citation_author=B. Hansson;citation_title=Detecting Polygenic Evolution: Problems, Pitfalls, and Promises.;citation_pages=155-164;citation_volume=32;citation_year=2016;citation_doi=10.1016/j.tig.2015.12.004" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=K. Xu;citation_author=X. Xu;citation_author=T. Fukao;citation_author=P. Canlas;citation_author=R. Maghirang-Rodriguez;citation_title=Sub1A is an ethylene-response-factor-like gene that confers submergence tolerance to rice.;citation_pages=705-708;citation_volume=442;citation_year=2006;citation_issue=7103;citation_pmid=16900200;citation_doi=10.1038/nature04920" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=S. Xu;citation_title=Theoretical Basis of the Beavis Effect;citation_pages=2259-2268;citation_volume=165;citation_year=2003;citation_issue=4;citation_pmid=14704201;citation_doi=10.1038/hdy.1992.131" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=H. Y. Yang;citation_author=A. Q. You;citation_author=Z. F. Yang;citation_author=F. Zhang;citation_author=R. F. He;citation_title=High-resolution genetic mapping at the Bph15 locus for brown planthopper resistance in rice (Oryza sativa L.).;citation_pages=182-191;citation_volume=110;citation_year=2004;citation_issue=1;citation_pmid=15549231;citation_doi=10.1007/s00122-004-1844-0" />
<meta name="citation_reference" content="citation_journal_title=Plant Breed.;citation_author=Z. G. Yin;citation_author=H. D. Qi;citation_author=Q. S. Chen;citation_author=Z. G. Zhang;citation_author=H. W. Jiang;citation_title=Soybean plant height QTL mapping and meta-analysis for mining candidate genes.;citation_pages=688-698;citation_volume=136;citation_year=2017;citation_doi=10.1111/pbr.12500" />
<meta name="citation_reference" content="citation_journal_title=Rice (N. Y.);citation_author=J.-i. Yonemaru;citation_author=T. Yamamoto;citation_author=S. Fukuoka;citation_author=Y. Uga;citation_author=K. Hori;citation_title=Q-TARO: QTL Annotation Rice Online Database.;citation_pages=194-203;citation_volume=3;citation_year=2010;citation_doi=10.1007/s12284-010-9041-z" />
<meta name="citation_reference" content="citation_year=2004" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=Q. Zhao;citation_author=Q. Feng;citation_author=H. Lu;citation_author=Y. Li;citation_author=A. Wang;citation_title=Pan-genome analysis highlights the extent of genomic variation in cultivated and wild rice.;citation_pages=278-284;citation_volume=50;citation_year=2018;citation_doi=10.1038/s41588-018-0041-z" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="QTG-Finder: A Machine-Learning Based Algorithm To Prioritize Causal Genes of Quantitative Trait Loci in Arabidopsis and Rice" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10.cover-source.jpg" />
<meta name="twitter:description" content="Linkage mapping is one of the most commonly used methods to identify genetic loci that determine a trait. However, the loci identified by linkage mapping may contain hundreds of candidate genes and require a time-consuming and labor-intensive fine mapping process to find the causal gene controlling the trait. With the availability of a rich assortment of genomic and functional genomic data, it is possible to develop a computational method to facilitate faster identification of causal genes. We developed QTG-Finder, a machine learning based algorithm to prioritize causal genes by ranking genes within a quantitative trait locus (QTL). Two predictive models were trained separately based on known causal genes in Arabidopsis and rice. An independent validation analysis showed that the models could recall about 64% of Arabidopsis and 79% of rice causal genes when the top 20% ranked genes were considered. The top 20% ranked genes can range from 10 to 100 genes, depending on the size of a QTL. The models can prioritize different types of traits though at different efficiency. We also identified several important features of causal genes including paralog copy number, being a transporter, being a transcription factor, and containing SNPs that cause premature stop codon. This work lays the foundation for systematically understanding characteristics of causal genes and establishes a pipeline to predict causal genes based on public data." />
<meta name="og-title" property="og:title" content="QTG-Finder: A Machine-Learning Based Algorithm To Prioritize Causal Genes of Quantitative Trait Loci in Arabidopsis and Rice" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/9/10/3129" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Linkage mapping is one of the most commonly used methods to identify genetic loci that determine a trait. However, the loci identified by linkage mapping may contain hundreds of candidate genes and require a time-consuming and labor-intensive fine mapping process to find the causal gene controlling the trait. With the availability of a rich assortment of genomic and functional genomic data, it is possible to develop a computational method to facilitate faster identification of causal genes. We developed QTG-Finder, a machine learning based algorithm to prioritize causal genes by ranking genes within a quantitative trait locus (QTL). Two predictive models were trained separately based on known causal genes in Arabidopsis and rice. An independent validation analysis showed that the models could recall about 64% of Arabidopsis and 79% of rice causal genes when the top 20% ranked genes were considered. The top 20% ranked genes can range from 10 to 100 genes, depending on the size of a QTL. The models can prioritize different types of traits though at different efficiency. We also identified several important features of causal genes including paralog copy number, being a transporter, being a transcription factor, and containing SNPs that cause premature stop codon. This work lays the foundation for systematically understanding characteristics of causal genes and establishes a pipeline to predict causal genes based on public data." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10.cover-source.jpg" />
    <title>QTG-Finder: A Machine-Learning Based Algorithm To Prioritize Causal Genes of Quantitative Trait Loci in Arabidopsis and Rice | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings, {"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"1J95wEbpGNaO61__X7KD7x4oFmB-wVHGr6GL01OT7hA","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"494281","apath":"\/ggg\/9\/10\/3129.atom","pisa":"ggg;9\/10\/3129","ac":{"\/ggg\/9\/10\/3129.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/9\/10\/3129.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;9\/10\/3129"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;9\/10\/3129"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/9\/10\/3129.figures-only":true,"\/content\/9\/10\/3129":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022tzcFuIgVUG\\\\\\\/UoVxm2kphAyYkwcM24Ip3Qz1DNm1GtaMzODQZFfy80IZfhwVZvCvCp2KG8ZVGGO2qaNm2m4xdb1OoM4k2sDTbzBoxDOcULI8lxpZe+oFBoQHeNj66ftX+PHmPS6CtDbLkb4IrasvuMHnNEKbynruarT1uAMEe+yoztmPMgFKhjkfYu8TDsnVUIAJ7DEn4N1BJ8N6CtsD9lY0\\\\\\\/7ISnxXlQZlc\\\\\\\/8sicM1H09B74LOpgYJd62IBJJaNQTU28x8bcV2ZEO43D+8tan9FhGM6+q8VxLsQAJB6Yz35YglmZQ8CEmZ2Sa+eC++u8zw7NbUbw7mPl\\\\\\\/tgIu\\\\\\\/ZtiJPIWoy2uByQ337bHQnHAjHrb+KyDL0jj0kQX1jocVmFyjT\\\\\\\/lsCeTMKZY08Jph4ekzD9KATCvQDYOduKZB+W14KdMH1k1f3wl+6FRcJ0IIryuKrBZ2hlFTPnk1zlWu0gPF4XaLFc4TRvZk7Eh3ohrb\\\\\\\/xLZJ89TEAHumBsvTL+EWbli84b0zTlQFf+4RqV7VO6ImwdbBKmmM32GHPjRj6wjCuk6XNaEKKtCdLaRxiIjw7wqKEFsA+7peEqxanGJBR865uBED0Jzu0iZ6E7KB\\\\\\\/cc1qaFgRMTlxgtvR+KN2BS0KDFKEzmzK4A4ytzgXNvsgVjFmPifiGqSA76ECAeEMS5rnb9C3T1B2B\\\\\\\/Z6B\\\\\\\/X0gBX6KePRtgohIIMJSnKIjRb3sBXtLMhNVhn7fWTi7BPlOc7qt8NbCE+yxmGSgBeOZRHc2UTF\\\\\\\/oIiq2hs5gQ9+NYNQpJgD7e3ujKi+O2rckAONUW9sX\\\\\\\/U7L61T\\\\\\\/AfYdyQzYsr+yz1jMcL92Z6qotLgnin0vEA27zFsWsKSM3voLlDcfWrOqQ321sW6ktBYoJzhBrX2bBfvzK\\\\\\\/k\\\\\\\/w\\\\\\\/hi6o9rLEj0d3RawDFP8qtlnmN07h50mAQHYaDBf6I4Q+WHP4OldbpMMJs6vHzznpdH1VYAq1+pYes\\\\\\\/LOOvKSA4fQdjOyR7uAfOsCUWF12p5pn3PcHkfVNeaXA0HnzKZyMufbFFZjctsiVix1GSlsixfRPIUxx3KcTr54RlScLPdrryE6WbLZGAqgL3KYUBteI75+ncI45fE9IyjMaa9DUR3+LCasE8sVcwd\\\\\\\/yFqzZW7xeKC90+xBIKRMPAiJLxXCMUcOG70AjT5ASULlWUwG1BB6iCGkYJ9q+e1xo33Kx\\\\\\\/rPHEL\\\\\\\/UsRTwJVn48wieKKQCnzrjH5lZLmsk5CO40QMa6sRn861XuVHz\\\\\\\/FD+mXOexXiD6tRhsVg\\\u0022,\\\u0022iv\\\u0022:\\\u0022pB0LaDsXuPf\\\\\\\/RQRROlNYaw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00223723b9ba9d390d4588bd66021823a9cd3d6ab4e8692d26dbb4c7e7765e5409b9\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00224oH5VKys82eD30DRIb01q75cf+xLnRroJeHaMro+k4JZd1lseX4IjPaZAzMyWd1WV7Tl6Ghdd2Lul9VoDk4cnwnqgKFVZY\\\\\\\/+hD92Cbr77MP7Z2bwSPZb5Jqb0IhFIoqq2xfVx77LvcYF4eI0HPpZGZCRzObrlnZbP6MBYCZsa\\\\\\\/rH\\\\\\\/iG0d9pWAqBpAJxmuFGyI4EiX\\\\\\\/\\\\\\\/8zDrU+dhg4VF+gmhxmwCiIks++CfQzpFP4enBvGU\\\\\\\/8HNWRdUy9xkMJVTcnNSKvUcHyl5k9Oq12XB3DrMZnESN8qKi\\\\\\\/h5kXS4Go1vSrHc45WVmoOaIhNOMcyLA\\\\\\\/eFwz3MmHpOLhA7d2sZIxFpOWntHmMHJ8EroWWe9PeYhIfvauSzG7A6lcKN0RyVJUxLT+zrNJ8od201c4vMO5FZwOLc02ObwOvREQHU\\\\\\\/bBdLKzW2BPYM\\\\\\\/\\\\\\\/tpT+5Khjve97hl2Ylc8dEYibIAjTHWyIkZMYrBvcrKbTlxD90watpbN3xdI7jpSVtheiKttR0K7Ll93fkJdfbSJmgFL1u52PO76HTUML956ijfdzox6bIUVdSQwaGrc+IZO5Qq0wC8dmVCYmpCaF8iiZM4jIJmjc9C2nEnS6SX1N+g0Ks2NRgHip\\\\\\\/7pYQNqL\\\\\\\/sUFQgJzrVAJ4scJQuUjVPW3C4OXjj0It6fJI+2DLlJ3A3qYnAtmmnxfvnjtO\\\\\\\/1D2uTVEy1G+teqjyDLGn1trdOGK7QwzBia9wgHW4TGeZQnT2sprMn4jj0FYdxRy21dQCxtRR4YxWv1O0Sd2jb\\\\\\\/mFCjL6l8YXR4poH+I4kkJSSwUVfNla65abGIsDzXdMuxBftez3jjXzFEWB9FCxF3eS6nkUj8vPyfCzBVDUJNYAq1Mf6ELq3trz1EQF\\\\\\\/zj2ImVLEF1egxNffOxYDkZyMEbUb5r0jdC0l+Gxl5DuQALgKztaOApRqq+motMhRxIQjuw1fmFM9MN0gGfHD7pTSgNq4414TnP+Q\\\\\\\/kFZS7n3K4oghrivXXWs31OUl5iuyu9D8818ehp0f8ArWhXPUBywLjQ3AwPHmsABYiuEHW\\\\\\\/xumEhPWXiNAoXEljrSWVR\\\\\\\/hX5F27MPbr\\\\\\\/ld3RcInSdQHgotB\\\\\\\/8kx\\\\\\\/GnFvmUWZHhbz25RkRNBj6env4JtkKNbHNTnp6Ay9aP0mkIK2jdfEA6FjYj7NnlKVBbNrtzLCl+1IBof6s\\\\\\\/FNVabHP1qdM5X8i0X88lOrBStO8CYjaFQabrayQ3nrlqlgUsPTL5ssOgRRHOvg3r1Bu3h2PBshZJD415B1qu79NZb8wBmVw+33FrukKa92NLguHWp4Ed+Fe3xAqLWWLAif1iB6AI\\\\\\\/yPhNwh4VhsSid8MsVv+ux0d2wz+K4Q8RaO5jEEdThGM2z4vvv+k6oee2OaDoTAnNEIPoMfPjHgcPXi+sIKXjedrH\\\\\\\/rXvUFBwHdOO84t0qEJSdXw6ThlWfFJ8fkIAsXFzW+OrisON4pLg116nqRHG3u7u+Z3EhdCZE4zP0L+MHNKWvXP2xQG1pBhl+vaVSjMgjZv7R2MIsFXi1CF+Ho9RYkQ7Pob8JMO+rKeNF7N5rMdHKHR+RLqH9tTvNPWDCuboe\\\\\\\/G\\\\\\\/DnBV\\\\\\\/Q8q8LuHfblkvC22PQxaJIyt\\\\\\\/UwXPJoN+vffMI3JFWZdpXwxUqmfE2HpgH+d6E7fBtOJFNoNAbHc\\\\\\\/SgXeMHxVwpQvZWiQE7l\\\\\\\/Xwh6VaBGEIP8tK+52l9Hj9PonUJTuY7Sjktq+mvpeVzACLkylzdxiXr2YzmdIq7\\\\\\\/dgRfKtP3Pt4CMqP7qwHxQn3p5Pq42aaSC78Omh1aohNaopcu4zuGKKQhV4umZapw1EVsActif\\\\\\\/F47vetl\\\\\\\/jUQULJ3nTLkmyYjlaA1ahfoGoyW8LJGDPJK2B8vPHaJcEhO28n1+WMAWO4p+x2ZpDGRdGsNbCr8BSupTGSIvZqnaE5mtQuXxGixvb4Pm\\\\\\\/zUHp8rISvx0HJTABkLSV9b5K44APcuzgctjqX4OyTc7nqmJVVjveZTIUL0npQaqeImEIcen71c1vM0zVMZ+6ip06uEolnhWeII6kHYRUVr5dAjSXFah0k\\\\\\\/Ezdx7Fykl0J3hShPkY3LZGpmMaCGgsrAT4BmtwU32h6vBTrrziDHv9SFZuWh9c4N\\\\\\\/66sHb9+zWIfXd0E53b+VWpqdLQibxNC+jbZFp77Ty5c0t+fjYIBqUZvSFBkGzHuLt6NziUWtmAc04KPjb\\\\\\\/9Chv0ed8ZdzQ15KKeIplmpSfTFrxIbE1iNUHjBwXWrysyWRyCHdTwuAgeXdVYjZAHg0IP\\\\\\\/tl5LD1iAOThfz8mORKNvitVthkGlsi0G39ARLts6PCZPfpBxgcbXoXjROLGy\\\\\\\/DIUhB0rkAJi8d7PuCxXJ7QfhGw8Alfv9TG4MBn2yk3x5wBhLJoRvQUBxRHmxEd3eZ+amaGbAnwO+iE7hMEkEN2o7u9d94fyosfjZT4DhE4BX1sB4S92CQBBPnl2JGMt41l4wyw3l1\\\\\\\/ODsQwvSE1HAqMcBGFmHXuoqUlNaMECLeJ\\\\\\\/tnWNk\\\\\\\/6eP087xulqnxGj2MtU8iATqNepLRFiXC1G61kGgTVDhJvgLE+OqhyPFj3j18dPBfRUi7eMV3qunSrMtuN8chnTL7GJiZRFhGkydVYVeczSvTq0WIOVA9BGsyBtt0Lu\\\\\\\/xndj+bwx\\\\\\\/6QR\\\\\\\/OKhI23t3yWGNjaJvxou5WDmCoNMNZ3WbgKhsANb0I5BEM6qrTfWLYyVEoddn9RMyk8pnS5lOcagVYmhcvD4c3xo5JmgbNHogISYpiuBbQp7REOg8GTKOIdjbSjFoSyCnryXMpktEJ\\\\\\\/gR+v3MSGK5SmhfU96FbpCPHzYrFy0IjWMkU81pel2ythtGvXfbky0bV7eBKEcAsv\\\\\\\/JpgG3imV4qpQYrE1RxJe6OIWegiIVVItgFuYalSuaE610s2TpwLyWNu4YbqvW0eke6ObAW9Hd3hD0MlGFdggOoIYJpOY2kx0sI+1v+rOgKqNymtuDZswwhm6pWXZuAoferIEGlST3sOjYGVGM5E1aOJMjIc1AXT8CnE5seWUMK\\\\\\\/isvRHHmOpK8WT9OfLChW\\\\\\\/J\\\\\\\/snHPmINd\\\\\\\/I2yUQqXVmKa1aJvzqSD5g3Voef+0q+JM5jd5WI68KrViCRbU8jsoZRweQqKAB6LOv+ULI\\\\\\\/vATDMPUTrEInLh4z9L\\\\\\\/fqdtyhND1EXWlf7TQpo+fZ2JBY870Ev\\\\\\\/TseMvAHuy85uGAJvnw0PoeNvf\\\\\\\/7CKh2F7yZhZYMrkFm3arezJdOcai24EasoEgF7RSjsX5CriivRsu\\\\\\\/dYsC+oVkj5LjLFHqC1JV61BrybHFJwt\\\\\\\/QAlfClQnOb2Ddr9l8jH+VDm2zQWFiLs4ezDVZ6t5h5NDfT4NcGn9S+HDBwK1OWsIkmlHkW06b6pmBS0eEpM0Thi0da5K3iEjip1n5rV0KbHx1nw5iIXVph+r4dbd4Hi2UeoMuaBbs5T\\\\\\\/BgDcmURLLwo+tpQ5jU6Jv6WqZo+e1L6o5h7MHei9IYk54HcU1FCIKN0LkI1n3ol5OuvIdxzwr9T2R2F\\\\\\\/PAhugVFOLOLqUEDNdWTqCF1d6FyGwIULVZpo9tikVNrlX2Fdb7QDmq7ql6OWY0mc7yEMB4waZpsB8gbLp56F\\\\\\\/TUb58YRECSsGP+WBhfw+wzCZ\\\\\\\/qfUQjpLpYfmWlDqait8oGkVEbyYTJWYBMJvK2ttL4QJmgqlK896P8m1wbYK8l1cz8ixfhiUqzWO3vwmk\\\\\\\/lu5RYJQ06h+XkoaXPeByj6NFTufBzVSIvBLzN3o7qU5yK\\\\\\\/5f9Xy0wBWeeZGiaE7qIZ0WFJFKdfflF+or+EsKV\\\\\\\/Vtg9fC5DuzaHdkmgBXUGEsjgLq\\\\\\\/UrILE6b5Od84i3E8LDTIWY3aYtw97tgLUtgDJDshYMNbvVGtwxh0zXWfgEmTb4nNlk5eGLwErAI1wtre9AEm9ch+uAXC0vZHnStmU5K6sMJa3UvCFV78lQZb\\\\\\\/nal+OGPcx7Gk0\\\\\\\/Qd\\\\\\\/Fz1ayLJDBSe6eFPoyIajhFv6Mhj29HVGUL2PW+LP+NCagDPLji4kMxZREps7fnoSvievb9yzQQnZJDBJBV8Xiist7Aqg2yR7I7YzDRW1kKY0rYn91HKrH7RwZUgl600YTZd64bJAMZHXcWPbA9uD0k5oxhNBwx4mif878q5\\\\\\\/M8RImOMSwSUj91ek7oVCUOJV3weP\\\\\\\/Go0VFkofjRWchqZH9AC96olTcEmHNrnpXCcnfZoNeuWkR6Pt2KvulslD4TpVoWZ7Tm3SkvC8vQEhm1FDAa8CnpVqWo0HQdm4ZK6LWNuF\\\\\\\/v\\\\\\\/mh+L22Waz3GLmrudobFvTReHvyYETSYacPVZsxPutcrMCd3LJsgc+\\\\\\\/kKAwv\\\\\\\/BhefUGi4H6GNAFPhpHkOIjxfvPuyyavZ0vZNSL9LC+0P+kkVkn9CGzz8VETJiTANa7Dot1mx7rFS2zpDc9wENyzNCiazhKgTqR2DQbdtM3pKd2QeyVW3oxrmyrrC0RpV1zs9xa3Hsj8BI4mqsP3TnTwCJY9Q9ycH++y\\\\\\\/05dGmmMMpJtAfy0UJsLkcqyqTD8d7ST3EfQRG6Pes24QWn6TNuLo3t4wjUThYgXwDmYgAt+Nz+XwFycZA0tTC7OT5Js+zn5syUY64HqBFkYpwIhraPZ02urBhbVKVpGtrEVMSkq6\\\\\\\/4pHWal6ljmh00mFgHX0IG11ZCG3koyFYhFdYsgZ\\\\\\\/5rr6w\\\\\\\/DUeN7XpcTFuo6Jgc8IQY5dFNhtg8FA+QyK9JGAXjAOZAoqPSkqs\\\\\\\/oCUvp0j7SBnfk8DLmAeCKhWLHcbfE5zpE0Gf\\\\\\\/dcJzguz3G+41XvA0lVuT2BiJm2a+GaNCpI9IAGSWF4kvqBq4Vwh3gKbYcqGt8jLEO\\\\\\\/HN7lNvcJaj0fK4ae41biMZGyXah5E5aFlvFEcQDCO5TXMk4gl7qrvvvs1RMWDlB3vs7TNrMCpQPId3Bwlcl2jpF2JlB3JKyrh1tZABp2wu0+pkWfLRWOhf52u69UIBWDYEU8UN4kO44yYRTmm24VVeapONbLd2Yd3+cYyCf7WI8gN\\\\\\\/Wc25ilpwk95jkMFsB1Pre6xJdvaaHZ9AB4DQqM6\\\\\\\/Tad0QATM4FClzqo2GhNLVsP9VEcQDq+Ln+PxG2YqnuZnXc7gUJCVwtuNPLNpCK9RHbP8oOUFO4vwcgy04tfB6mTd1lkE+4jqFffyCtjWFXBZyY+C4MFc7u8yZcJR\\\\\\\/6SEVix44R6Tx3i\\\\\\\/5m8lo9d0C8JExO4ILxVXYqLjfFAO\\\\\\\/9BfY52n2hzaFurwiuh1Ek\\\\\\\/ZU0TAENAoiY2Xx0YvWhG4bNn6SuMC0RI9GKYniw4yppoBF5trqSSlnEfrGWSbXZWA9\\\\\\\/Rti2\\\\\\\/R129UJpIfaeJa\\\\\\\/45D\\\\\\\/vkNXvfSMRV73pGAjITe3lbL2NekklmGYKYOCaOvJG\\\\\\\/JnzyODzLUdBmXlBuxGLWWv9DzYkCFUkOaya7PAkQkeLCj9iz5Ajj4jqlZiJzagVBvkBTXxAm6jBnesBdMr6asyCA9Ivl6XyWMOY1t9FkX1pLEb3QHZrfSjW4REwRe\\\\\\\/m0jDoykgIevSTj8JSo9fv6PfFfPNCLX0jdm3xgUtbYcdSJ2hnt1YSsvIVSU9ljzYdqOEOF6DOt48SAVFfn2qeNUHd9HI6nlbKhM6g6IxuqvMmd\\\\\\\/9jGSbynsJW3zfmzCMSIYJneaYRE81yUTFJtV9Bhz3hJZZL8fN7eBy5zfCe3c9jOJODTU7WUHublL2vrQDhhNJGfVxCUjXVWqCvrBTys\\\\\\\/OpEdGO3\\\\\\\/XLEElma9mGc0Evq8pUrPxMSTZhqtGHxamki2Nrnyx3NLL3LY293ysEMf75+zS2jA5JAQxjc50q8kMMExaTGyCSKMadIZ6uTKk\\\\\\\/vRFi8dhA968DDxfrVW\\\\\\\/IPYbcJqayy\\\\\\\/w5IhJjNuVIv61CIxFVMNoyLsD5ecO0bL8LmQMf5j2iteUiASF6wnWZ9iC2t1AgQFH8005FwDCk6yc4Abws3OC6VHgYA8I0JwaxHyAQ66SZgx+TvcY7HNI6RG1wxxr2QdwyOzS0YhJJzEL+\\\\\\\/sk1AI+tw5lKFNG9P81FXtHTeTerHohxioZmf7KArQ1stc0KEs5GOT+sPofaq9htbyc6TgIkdjXoD2Z7+5O+NMYBymCdHL9bM\\\\\\\/Iw3Oow4y19Sn99wQfE8i93XUOvS1TlY1WrcUkXOTNN0WOrJG30nHbtUat7seej9TGN4D81Zr7uKWdiozWwfOYHDj1FC7VMafpcCTxnAXedB0ALnXkG3D+u4LN\\\\\\\/jQyz6KRJREp4CtkK8olfmLQhTAJ8kWfvf2UBA0MWPt2LVtqpN\\\\\\\/6R8a6HPfjLifzQu\\\\\\\/+AMdBLJQ3klFF0DwC4RIFywcSwL2IsTI9VUj0oTFgkOwzzqBaksSHI+YEB1bj7xJ5ovmbUGxW4aHJv\\\\\\\/knh3WKv\\\\\\\/7wmBJ6WWQ7lKY3Etc0\\\\\\\/bp4JohoCaKbHLDtOAXhWaM0OTd4lntsxEyA7MPjSkbApmnRmDiQ91fQEZ+4Fdyio5\\\\\\\/rsPx1FgXZMsCZW2WhuEWse8ay7d5rh5PjWI2AXpa1tjLN22A7c8QtfLbTGTfdmHCSStgF3hixvnOsFOEBIg4A8\\\\\\\/jlzRLNbIoP60w9WUsrupMIlCxDtwI+d6Lv4oPqGzOu\\\\\\\/SOnbm8jpVQKMjontLBDilvu9BqBBlPhSK7x5iHdgtk9aQyHnaGtsur9eIdqJmyZBQCeC8uEUGJkE9hLtOPGGagHXSK8nGTtvf0xBH6X5MNNm80wUCSo7lDKjgzDyPP\\\\\\\/vb7uA2\\\\\\\/np2Vyn61a7CgCUX5MpdmS3WPZGuunFJowO7NqbifSF6sbTnD4tOL5OnR7j\\\\\\\/p8pfSpKdaABZTnPfbXjg1AS87m2g1c8pJQUFGGTwKxN8fJ5O\\\\\\\/m609Stc6Lel6qlc0GxNs\\\\\\\/z\\\\\\\/rAp4u7K9K3dSi26pxk3CB7d2NiVKzSKAs+xc7wkUhGmVMhLZdZ5eidj3gGRFjY6WpeAqJjhKq3lbDgXjUOIjlt8j9ElYyuFe\\\\\\\/Ob+UJ8ZUcJN93MMoBdA1x4qqQtpS\\\\\\\/ByuPhNqV4KVQCQKS\\\\\\\/GVeeBAaQGC9fd3ndESorBlHEnVv7x6G9WeFCAGIGa8hbvLUK4FrmhwCSeQtSfI8yY+0CWVH3ni\\\\\\\/QkwEm48Ppw4jmVni\\\\\\\/J\\\\\\\/tRBIpoQQVfhWrNEjlmDQxYJernwJaT6SuwrTZGz0p4LxboOLxuFQb9IHU1lwpjRxr5QbzPcmfVgo4DfsTIVyp6OTTgcBe\\\\\\\/suiZh\\\\\\\/ltbP\\\\\\\/6\\\\\\\/aB89+hYOxp8sQHTFKVV8dWUuTcP5Xt9A8vm6DtaqjGLgD8PZHzFmyQ5XffcZr2RarQZsVIz138ahn03lvRwAyeaKkOdu54xsAQ23Bc0k5iOmByAxEzgnjrxaWT6u+4gofCuCUioFjiZbx2QIvldB2cc5QSUe\\\\\\\/290KykIFf3STgfVjmihOwD2T6y3+FXNyIu3Dxr1wr99QcDlJ8IBuMyK7bJa427yYSt\\\\\\\/Ck27+d0Kd3KjXsGxHDSaBziXy7wEfjybX8RPULldkn700gskETWCjG8lO2NTXm6lxrEulBeZlTkZ5hRxfB9EdaEtkUt+amJwdUdZhUZah1lMzhFHkY8oGaXpIlgqbSP\\\\\\\/YVhWuPz9e9tVQHyThY7q6\\\\\\\/8KeEnWeKyuNhNiP\\\\\\\/RoJ1NJtz37L3ldh42+NUi\\\\\\\/0IsA4YkzWCgRXtgQWY2BZwhVP1JHDVcULqTeS1UfE\\\\\\\/QHv22EZCXlOAns4WQ0nXwQcERP3wMeUNJyMx+Nxb\\\\\\\/gBheBB0Uusk+Q7QhU3xwv+MfKlfHkrtmQZuZNvD2\\\\\\\/SQOGFue7+4xR7pNLjFO8UfwbsJJ0dwt6ZhN\\\\\\\/w2qYgcfzKzf5Xe+Ew7d3ndqdzTt3azZpfEp\\\\\\\/qR3mLKZXue0o+OFKgguyn3n1V7oVNaQo2rEVECaZ607UO3vIz+2G5+KTM+wwXbFWSZmIWuYmVmTPIk5ATd55aVDPuFdsnCouwgqomDPWLmsdG33BMkXOBIbFOyfJtVRgWJ9+Mcl6TpzV9jFPAu+tLVrnByfdLDxDONmXP8k5\\\\\\\/vNBdFNeTF7zYxNyI3UYIL5CL+Fv5DrulhgypX2kbwzp+hvwrRkk47niCDc9pjOwC0uMbbvdhnn5haBzcuvHEmyNae2zFmsuLuvzQK3tZPFnOf9wJmG8GZ+EoEvsVVAgzhkS\\\\\\\/UEnR0t4K1Hesq9J9h2bvoE8F+eqza+wahbS4bmyfWI0Y2vCOGMaxrfZMJ3OqicWeDD17VP2dyVl3INTlIiSK3qeTB0zWUfTBzRo15+tgIGDZZFGyBrCRqpsdJOnIT3TR9OkogCGoS4vXdVG4n4uj2zwsmVS7MsM9XmC1xjtv9f8Zh6RLUfs3Ak0ugkI6yv3b9qv6c1qfEAnEqB0axRlr3KIZCz6rABcZYi0mPfZV9MRdEPs4LaaZu1xP0660PDOQxXkkWA\\\\\\\/Fltzs8TdSg+1s\\\\\\\/DtOD\\\\\\\/bSBr3xuucOdmMOt\\\\\\\/yDMiaLPBv5lyxK4RWb9CqbdmksOfk\\\\\\\/zoNmCoxYB+Hn1nQ4diwmtxl99iCeCuAEEqlbIMY4WlcHHUa4bwjJdKabFJw5a6TrQMOW7jcoZqjSaOErP5zYBKO5+ioRvrEd2HGSVL\\\\\\\/yF6qDfMvDCCLyiMP3WxouVA605darkpwJ9A8Wes8cfLS2WVZVz\\\\\\\/RjCASFWJwDsWhNV9H6UWp++TkZ5vJReJSHlUHCV5YqNJWly5dLEezgm+iq+NyggVAS7IZlhnp43S1YLNbv78agdAyBzgwy0diuxQEWePKwtJanhxHgS0bWGhF2GYoG96T22rUt6rXcsmEYiOMgTxSeGn5nUW0Y1RcLVPLDnqV6HotxRb79V7TBl8sypZkLGty7nGw+c3hwDBUvL+WHLsIOUpOM6o7O270G5h1q9t\\\\\\\/+vfYU5rkQtK7ETAPPY8RrpLHS0Gzi8\\\\\\\/EXQCueI+xtoGTDZrzr5V6Li1ZO\\\\\\\/FNhpU\\\\\\\/Ug+NFW3GkQ6y4M7\\\\\\\/XCMBveimZcRE6lj\\\\\\\/\\\\\\\/eVaSbNO35yLw4uwSxgi3z5cf1B5jjrJQAhTNIH2xAQRWh3AJwIW\\\\\\\/2TafqQOLVEqQ6DQyFJbsCYwtKlL6xUEOrdq13uCMNxn+hs8Tg+eNBGC\\\\\\\/3Uq+0Q2zegbBn9kWLeU6R9wCao0J7twfHlDEXxlj2f3FVTbBafCmX8XBNegf+4yvrcoGNVpdR45zOuvgwMepKlwJnbbbXwmh58M3zi4qbmxz8B0bNrqe3uMl75MPlatBEPvxQrsb1np+Z0BybSMAQGO5Gbfp4Jwp4yjfnUO9HFYrG9VmM83goD9PM83pDzKX55+6vxilzn5j1EkFwX42jh6Lzq21At+wt8KNVFPOkuYyoBLpzZj5bNpYTSXTb+ddKUMuqGeT67ofwS5FpDY4Sg\\\\\\\/11FE7UonifWd64cSO2bXgX+09SDszAdlS30YU6AfSau+9uZE7Bfl+qkUZRX\\\\\\\/+3RjeekGXGz2ObaRcGr4TqGMPmJDP7N\\\\\\\/PIBFrrCqPKvqLVX1f7LbodaGbw\\\\\\\/zTYEiIcTAiHyQolG+g5S+Vtst7obxIHRopR1epLjaO+CYSNetzMnCll\\\\\\\/NUvYoSCFb6xB+juO9xa4b5WhqtUpPlokjc\\\\\\\/XwwTxGPweQl78xe0ZqrkZdJGUms5Mz2ky1ckr\\\\\\\/TtLot562VUJAYjwLCKFr\\\\\\\/v2HZ3ox9ti4+ECwQ8UlbWOYl9LBKvaOaD8SjxHFjPb6bae5q34d0gZvtAeYKuLvMedvKXgRR+p7aBlhRy4DojxW3okDDvkPR9Otrpxt9VGFUD5zLf\\\\\\\/WRFUbrkOrFZdy21ZANeEOauntsOXWWDYnW7k848VMmWmR2\\\\\\\/USSgV718jUfDoQ753DmJUXVrQjbeuOlKquRn8h6D\\\\\\\/CmEBamG4qZEHuKf9zrppgpDQOxDgQWitd8MOcWTD\\\\\\\/MPH+Psbq\\\\\\\/nlOTrSJi\\\\\\\/DLSWfhzXFPUzsV1mWIaQIG3D10mOlVrpnqmLTeWtB\\\\\\\/yTZgDGLI\\\\\\\/QiDb5BaBNJAAcfJKKu+bLxruZ1kyAcr3LWwbA1a11OKZqJCPrdrd1XCbMloTA9\\\\\\\/XmZOrfJpxsT8DuV65MoM1UAFfL65wHxlsYCL8ALuTS3wNxBy92yT4JrqIPTI4Nygn1uAqyGSKq9lstSg9Ip+eKh8JgfR4kpz2JWX7KrAR1NdxZ9CYNlzUd2b3EOaWVwIA9Y5R28YpL3ZlnNspjCAyUyCiA5ajfwWUF34DG4xywEUOFp2dE4aXMtcC82cY2y\\\\\\\/k2pkW0y6A4ooYwyuDX2LRq964OPTHdvZqVD7E9NuEsiNbvoF4ejQTRCMc0vGSPogTZSdcK5adO+IIuRao6IJBOAbwPKtjLXAmbCdJ5tk1gCDjFzWGD9lAaut2FopNj68OiXFuUPz5Ixk1p0I11KHIZAyCZ3KOY8DEwNiZy\\\\\\\/M\\\\\\\/auQt3dBVmG5sqsdxGfeXz\\\\\\\/CMRKvrX6kZvn5+weoJKPVFW7SFiIzPt4PV\\\\\\\/EUvbzfBf+FaNmklJENjIHThdcy4UY5cMZ3Voqdoig9aqoSVq\\\\\\\/AxLEC0DUzvUJPZp0qARVakMiWA25gOywpAunEz7iiFideQbNepysXf4eN6jUYGC85\\\\\\\/gfi7+xcV2m5pyhT\\\\\\\/rZKgQjada4GNi1dh4aTbCaDfC0RFq1MO8FzyzLWI12JnLCpWiVNvgPVpdBjknKCS3RjMm5s2UDnWn9oqMWYywctHFCrFWwgnHA1Wo\\\\\\\/iIsWmlr2OEtKKrCXTttUicqdx\\\\\\\/UXTzJWqqwGyAB4wwVP9UmUXtYVnQKWFVdAJ7lGYkGJsiocbRGspWGjaW89IhTDWVhuRYx6VuDthtfcnFVw4nsZVguqYRb9Nlwm494KGU42ibWfmOlNpraMAtDcZmBEDejweEO8y5mNyqLg8ZLQZ6JMnypqVjdX7Ib\\\\\\\/qTjkee2W3\\\\\\\/4iSTfBceauCSv4OUKXog9ZeNzRNXjrTSYBxjh6brj4HfyThDaGTfdchKt2TBvboPgBnp\\\\\\\/1pBPI6P5nDsEATdLAP0dijm5FdsGDqTTxcKYCSb0YGYF0+RPMOY9fchYBfp9Qzu1dG3ixdVRJfNRgsgRvZmCy529CJeud6JOAHXFGZQJjUvlYWQevbZNSZGF86CKQWfETWSyKqUVvaehrl\\\\\\\/noF4pen8I+v4QWZNA42yNSXsCG9f9DvXUhn6R+i2jlp0vLQowLEAlabhw5lLWt5ZOSQ1B9NLM\\\\\\\/YroC97C51Z\\\\\\\/YdSF+NgVF17ghJ+Sh4z4szh7CFTN9tJNGXPHE4DNTZo1dlmNrL0q2SzyzwzN2ZyfimZuyi2LscKil1ZaHV1vNlztyzNHcr9xirBds1eHaeF9ZC1hKNsWVV\\\\\\\/WuMf6izGDcRNK6I0bxwz6Hn8j7NYHAolpYqOAyUwVVjrFhSBgjuazaq7aoOL7OmKjfqeXHIcMM5mJNrwIuIqXeHQ0OulGSE\\\\\\\/Jk4OrX41I1u59JjfKMnngv+GiFdGVsTenALYZDhiBKwhNvAOx5MYvEM+Nqhr\\\\\\\/7qmw8sM6nKISIWIuopgrCPlCmXGuWdlzvCUJkVJR0Q7r0VWYDfsg+8L9jJKNyO\\\\\\\/+blp2byy+81XTBBOIOJNckH3wbNgGZKBlEEfnXxJdVHWXs4hK5lUW1\\\\\\\/kOTjTW2YS6rrBE4m65LgqBiWs4etizZJ7vUhIRJrGFw2nPIR2LXFa67bu6r6o9bcqzydq4yyt2vCt\\\\\\\/nwT0H4btJARf28dekDb97MJ7FoJn5UZFAlJa8PAH42Q3mlMkgqjc+eXmC2vtquaW3v+91B\\\\\\\/d1VcRdlCZ2JH73wwIR5sOHtuCHdWyoCY8AzlLpQxX8YpJBHNv3LACps4V67k3QxXuSGHjQtRSCMvtVC7+JrYnJneyV9R4iMOqJ3UY6QwFA3617C\\\\\\\/vOT\\\\\\\/+qwsRxiuCdqKznQSiDaClGHK2vCw7S2xnJo\\\\\\\/5Gpev8g9bC8b6Gqxi51i7YtX7wR70De9GlpzfdO1BiLoIH9neIY+lEOwsjftFWdD96Gy1lq3nNLt9pwR2eqkhEiEGaAvoaQybYDXFVYDHBsGoCf1RQe1vMf9I8xZONsqpejVCq9xicKTDPkvHMnbWBVTEAEHRoerQxtUaeqNBVeyVdpVtbez7DU5HH7wG+g4tGLiH3HYy6VSZQjHqkvuRlnFl3uh7ujQkzZ3YEqPxAF7ZgPjvlAkBFgj4qTp0kgklx3hIt2Z35KhCpyzE+D7TCFGe126dJel2fC\\\\\\\/IYw6CT85TGjxpBqpuq2iqI3NGWa3TDZqERPfXS6EfI1ruPQkP7kK35ctyrXsrXnITkSEmBPab9pT7MEcuXjBRa3f5x\\\\\\\/0ti34o7WCs16Zd7k01W5uh6wonTTbvxmaDD2G3264zyLKs4KbuUWZjApoLkny1u8sAnrKoUvZPzl4HfKGDYz5tkS4nyFpfYocSCxNOcTx1tDnNOmY+SfWQe8u3CDE8bGtKofSBnUBsCbQ1+5B\\\\\\\/nu\\\\\\\/NJveo9MICK+wkV+kczhcKTEqbtpwCy4JaLmWe8AieDCYSbfv+lrw5safpuUAuUNxLgsPJvFH27cwlpBQdzNNf09UGe2mVMEFa76Rbbuy6uMlR724Sl8Cdq6LiVL1YG3WCaoaiOZs7DqiM8IW+XLcJgsOBSkw8TA5w6R35X2ySY9nermkJ+EB2uej1NN7MmYMtLgR0XmS5dfhNIjuPjo3T21WKFy7TGQZZRRahzUUVq8lPE7rEjef7TVivFuDgxC+07n79B8knM4dn8LUAC0NaI3pTIyokIJ8n0LABiiXG6hsp6l60M3d+szemvWlOYpDyB0LsnLiJ3QDH1s8DYcQ\\\\\\\/q5ymJuZdvWEy3NlSn3718en8WJ7fK\\\\\\\/4LtSt8BdQUw5\\\\\\\/fUWeV7Slds9vgW8fOsg2oC1DszRj0+sPUWpBPIZQ4fZ5zqAG\\\\\\\/I76x7eCbBBf5ZsIC4tb\\\\\\\/aFxQ7g6l6eRCJ1SVrFsqGHZllIu9vYrPjIgtr0iJeX7gSjKXS1CfPG2GcfUO5PPTtt7GuNskBf1MnpdkWXXFd5RQ4I0yXgP6lan+oXS9X5XzOeKD9d0kYUGb6Z748RyW2Rx7HIHIfXvK5AXung8ST8+nxCBlHoDjWNev783iC5xLnzH+i3X9yzuC8jz2ReNV7gaZ+K22U06MRGNuPd6GZ2uGphZINxTi7\\\\\\\/xfjapoeDjj6ns+APXi0174EDQqng+BXDbThemqGI8niEa4h3EqSzmGQYCOZ+\\\\\\\/69iL8vBEmNiV4Rbrx66WQKjOPTBfe58eH\\\\\\\/rUbKI\\\\\\\/dV1B6NzozJUEgwtubeP5Pc4GtZ7KSP5RWTf+eWCGbqqLIxNhmAGKSUUc7etWEl0iZwqil2hAjELHQIXugGCnISK2mD2fld4XKWZweS+6QJZ4FzL5rMqVBrnmI7F9GpSxNUQ0gbqtAnj+SkQS\\\\\\\/XlWqDWGpcz4ri8+3939fpcOQ8ouABWNcRo9dDh0XcVN6SLvf2XXSPXeXLR3vHX3CkyUX6Fj1BLDq2OvlDY9x2E0NvSTw1EnN1p\\\\\\\/3Fm3TH3rvRk24\\\\\\\/QGJOdBI76HESxwcT0sfL\\\\\\\/hHnry\\\\\\\/nFcbAfev5hjMGenABAO8\\\\\\\/zYAbuXMk1jkSUCxfNuIs6ND9yy1FBEW9+HRm2vl9srGfqKm7LSM\\\\\\\/XhigVKrjbVnnrb4GVSrRBi3el7pOnq3KgRzi07HhB375fXvm\\\\\\\/gc+hELdM55OPfmTvkVlHSe908XJ2YudJOYvwktginMTngRoloa4uVWUSXo8WKV81DArP88ebNRFh1DJTqsFO9RZncjK9dON86awPJYBeyptNURGghkHIe8qUpfmt91o0efILMTRbejtt6tCl9z9wuUtAJraGhJik1IKCDdi+oz9CDRigm34o8F9VbEHsP6kcxjfLrSWTirY9xHdajmKnB2zoJGWueSYUIPybdSKIKwWo1HQXZ4KpG4dCttjxhwsw6GhBTio3Lrxb3TutCgwwpQkI7y3UbABcvRjSkw5vNwOxzSxx5PKvZtjBZ+OIrJG\\\\\\\/JsPOvOHfUtskEuGtlU\\\\\\\/ayu6E54yX6ecd0D9TLFUquQNwbWxaaIuAzpDLV7Pkrx9vYWU7Is6+HP6wclxV0dSKfQcW2W9YHs6TmxYvcGCV0b6G6zD986c+pNsjbDOktYYY3uS+Rl5rA8n2LuJJRRuqn13vUiHrJJF+4HYoHZELMQfZ\\\\\\\/WDExx57rde+Hpf5cNttmHvkoPCIT94ZPtoQhyBIHUTJ\\\\\\\/z0EOq2jsNq9xX8ozmrGF06H3O1Qkfc\\\\\\\/fwk9VllspyqCFLGLJ6rK\\\\\\\/z92B+OF4S9imub8eN9VU76uacowlCtig4sqyIqa\\\\\\\/lepTBQXDEDZwuTqKYJgMLHKH9U7WfLBRVZosoO9jsU50QgBenWEMN6sE8VOEfLBhsPZqQathv7BCW3xYe\\\\\\\/yBJi5PdUHp8ByMNLl1P8lO+NXXh4zCvP8zuHVZHn7eR\\\\\\\/bASqD3IABeKq1GzHV1MdJt7Gs1+SoBfCRjRQ5WTwVK\\\\\\\/1g1aJ7lnPZEEsxgUHmh7D6A7XhRe5QVxS6v8QOgPd\\\\\\\/Hi27sVvhB+gKqs8Ld4HmOOu3BoGiDQANP8paBrp3KU8JzbI+JPkz\\\\\\\/44u8\\\\\\\/F+GGHEb4SHbVix3CH3V3enzcQxAfG\\\\\\\/t7hgRWjDa8si86\\\\\\\/RQnnoqK7cksxZexFDL3kSP0EOzbm7af+UnMkKbcMyCbhlBrVhx+Kp058zoaImaXZ1XdoUMogWg+smnESeoFBQJOGc1g0a+76rc9IkFP2TJLhd7xYsBbBIYLLh+Rfkp3jz2Q15OXeCW0eJodBpluay9+OfrekMNwCP0SbpB0cUDnPS4fPiKNj0U4Qn\\\\\\\/iVHEPX7nEwmwChB0zUytQjlDfPUJiQosj6Jqyk\\\\\\\/HLT19iWb6FdQRiE26K7J0TdJKdKvZ\\\\\\\/FYUe9VahJqdegJnjHlnfTe5kZkwt+9raAxd5n51oWmptjdMGJ\\\\\\\/DODhbkxw52dQowRXUZjallng1ZZr\\\\\\\/ZxzJPLV5XNmfBlpKeisyY06p9gzgJFFESME1k9MN7gc9rgvmrSTO+RhiNQBw+nMhGEjFGGWiPHyg+B5EEITwZxn5BQPn4MsAYCypPMPq\\\\\\\/POt0YCtI5+bUwFD5Adpuor9CtR+26wRmK9DCBgDfome1bNFobXXIDeB8RDNKAA235QDeqe2bJM0u4id7W9DufPn\\\\\\\/3rB\\\\\\\/\\\\\\\/35GzYmhEjXaN1M2T0u9RyZk58ldGlx2RsLb9GvVvWb8A1Pr\\\\\\\/HLE842nILs8l0jptu7zcrjxvZU3hDp9cC+aR6iV6+LaGE4HqMLBLbSUqZ\\\\\\\/sMgiLKjHZbwfThnNI8z7HZzOwgKKZqKQayqmwJ5bfp47O2jzE1+DVVvogIqIBBdzFjHz3WqBYhOZqCOQNL2Wf0EpX3kgL9+73EDqMYjackKuFmryKX8tZKFlWTFB7TcWH+9A8DbPOqIpvJ0E5qSxZ4VyMhokgh4fgj6YrOTVnXvAWIQXuuCi8Bko1pwxfBnNGtvaMDQS31Fru0MHP9Duq9Umz0Iq1MLl7idZuDYDY5zlPB+BerKQHOf10XPWhVCMUAFv0aIK+5w8whWMnyYh96z\\\\\\\/4Vl\\\\\\\/VRqpC7yToco5gNoGcyPQ6ta9cQYiOEdBIEuxfTKGG7SEqAc8\\\\\\\/YXytwFHELsdC6dC9vE+GhInz2tNS3lyBqcTnliaFMKYmQzdKZP1u62KmUdtGg34aGLol\\\\\\\/sOTMlyD3yrUGeY\\\\\\\/wo25fGGlM3SL5a6dAXooB3Iw9L7fxmmAGVvdHUUR1RWq3dUfQUGJSo5kB6DAxx+Rpjm3D0kj7ybC2zV3ewNvrr3GXXNfwUCQOvYY\\\\\\\/GMAXatxilewDp5Qas29AeFo7hY36IiRLY+xDLlfFb5ZJpVuUhuvrBMFKMywDyKujczuZYGO39\\\\\\\/rZFbIUKRjziHOXxIDiXxs4HzsRsR4e\\\\\\\/zZXiwg8hjDEAn\\\\\\\/5OA179HJTKAgEtjiilGgGP6VUtWd9uH\\\\\\\/wFY2\\\\\\\/GuSNTSdQyysS\\\\\\\/99E2jrAFlJT7F8a+EpusF9wq9\\\\\\\/kJW7FKadSEff1sC\\\\\\\/kDB2Leq+0Alxz\\\\\\\/HqSeWoj2lGCNYlttICVYk3ZbDiP81QrNE5xwMF\\\\\\\/QF6oduYattBaCAHjMjvYey5C4anDjkNSoaS6iLPZfSaPzkVuYBIRogFhdq\\\\\\\/zIZatB0dWnkNT6ZRQPdABKuqqKUfmj5SufopoMWIn5tfaIVfslO+QZ63C77HBigpsYkRebBHsXS1sj8elglEkiJS7QTfIc+cOV9bBOwfpVgcjd5TqGQ30CF4N1pr8MoXRyOagxxysQ5ZEkWUclN2s4zk+0o4vcBLPN5kcfQAc8OswoG\\\\\\\/PBDyCOqt89A3q1wa5koQgqrouFeiKJPY6wHqyrECwXevP9v08dr4eKqeb2v6cdsUXQjpjcdnZX1MUbsMSXerTiN3E513\\\\\\\/90NIL5vBxmmJKxpP5FcAl3C7iv4YlOrRKIGenvTYMIPFeXkRGLT4u2bqVxO7g+xiM2yWRfF9oPA+EjHQAuWthQ\\\\\\\/nQMV0GJPfk25mroyxVZeh2KJSWfS\\\\\\\/qgXGXKYD1DZeJM6nGUDzo3uI0qV7HPY0xRi6EzLnWf65FiOnbzw286VB2ALsy4K6RgiTh23K\\\\\\\/sIiO\\\\\\\/RzVIzObnXW0K2qpr876uxkSAX1mbDoHinmnDHQOrgd9S60uTN73Dew9mzzz5MsUSZEOciZPlh7t34tsKqQ5WLc1wFYz4GFUObngg\\\\\\\/jR9QImXKByu\\\\\\\/HJFlzHsOnVYaFjSp1aTsNhLzg\\\\\\\/T1UxEjLp2xUMii85ZryXnIRW8p0Hri7HucNlfYiMxgKp\\\\\\\/f5W49889r96Pu\\\\\\\/YYHZMyA10gPzG1+wlnS+pt7VbJ+dz+ejAgchAEt0oUhuMZaNDryxDxf7PEx+Giseg0t\\\\\\\/X3RnbB99mOoZ1eBjEJM27MvBOliLe42oBm+mhlx+vit+n1BOnh0+yd6LJGto8cJBRHJ0psf80jQCCWfAQIpzY1V0jAobPAsGIXQCnBHYIn4ClcWdkmoIoE8O9Z9l26wtUgeyAUX5QkOtRsapGUXcILHz4aRIgUTym5MhUXuVFiItAEC\\\\\\\/yni3TKE\\\\\\\/EtzHWPH9QBxeF0e8CI53+uBJbMcPTNITnFTDh0ltNp5dNsls2FrBFF8p2A5GDlkdFKxCBISjIasRLkdOyrClO4mdoU2Gfgvqx2NY8UxBZkvL2C0nxmArhtvyabTqgwTsWsqsAGYKRVtizK1jzOs86obuGiny+xuAzMi9KwrHi32fvjLTLAtHFLBtvGc3Z0qZKwqfocctOWl6Nn2u\\\\\\\/TxeyMhWsVraDZLMXPvffpzewfozFovca+1gihZSfKqyaJmAH0J4AiSszNrPYEcQZCSYfb2Ft9Nfdp43LofacoMXh7gSEeqFnVKd2LXwymEVJnhl5L4cYa7aa0vxU42TH+NXPiHs7bkVYOEZqn+kFOkiM5bPtkDvw6GgA0YFiF9oyz5iTE3KG8TBiWdtfexe2WCU2ccbR\\\\\\\/tFu9HcDK5P+muQ9yLoOrpbmJnJucWlqe9UIRposOrG98xiO8ECC9Pd5tM4jXOklxDmD2J6Mh7KKDqCBo\\\\\\\/6FK3+f3ECeBteBBs20YGDo+Qe4z4NFEWjFUmaEoRBvoeBiV8XuE10cjb+kPzMFgLe6t5Tgxqyf3sO\\\\\\\/tuOVfeluYj6q4oYjM7knQ20e0wmd6LQD5xcK65kW8hvvaHmN7TOIntrxJrRA04LyzSpOM9SfeV9HtcGhQ0zgG9q8Ixf71z72LOsmSMOt+xtfJfhKofkdg8tlOKf4PIbSqp4TFjZ6cx+m4eokrOu\\\\\\\/D3DlwSSq8xhPmv4uB9np3tL7qUvBQ45jxmk\\\\\\\/Y3r1zS8b1iU28TekWGDkZLvRAgG0G25s1pWETjXpXhmGYQ\\\\\\\/JYnFc36\\\\\\\/zRTPAPZnG8IPHr1caPM0J1AzUGCbDmqvWiLkSRElrxMbd1rKGk3Epb2vH76V5UZB1SPjNX5j8aGNoGs3xLR43e2VuFKESofjoqtVar0DpuOjJ28yiVuw+\\\\\\\/66QQWa2E+opDQD7UzeAieia0Z2mKbQ0lz8KSrCqSRXiAC7ATjMlLqiGI8gmxYjv9IdZ5GlYX01tmn\\\\\\\/994AIagBh7RymlDI7G6Jhww+L7QSjvLkmG1HWV0neSt+gl0AueZyVIRU5MVU37IyAK8ytaMTjGflvJLrQVJ8SWqqca9Gm+LJcrX5xCCQZq6X1Y\\\\\\\/BNeEDmu8+fHUAKbg\\\\\\\/N\\\\\\\/ovwKgssQhTcgGYLahe9KFa+6iE2X2kQSJrhZBjiFq3JUc0wfnH8POdbMQBsJKvZTSlgbdei89nkmTaZbnJFETo403dtDYyeILfRx7+jPv83kYBzIAWNKFc8iGDjsfMqqn3I8ejs2KVmMCBESSONf0sXudz0CRooeL778UxOAYITf2kvSmn2kjVHBbin2TGT\\\\\\\/WNGuG4elMINkl13Um1EVc9vc1vYBme0I0Frggr2CnVcf26R\\\\\\\/EQxlX6+ixpvX\\\\\\\/3FMf+UZG6e+QUfONqKln\\\\\\\/G9yxwi9lykdPqd7O+M3rY9dUgpT3YnDU1jPGQp+jxE4bdYDGyzrerR3hltGn0UitVViutotWKjaT0orjG3WKdNObFSThW942QfilKEO156Olb5E+AN4I47KjnMmsQ4TYrpTONDcEQ050N2pT\\\\\\\/AqtacWD+gZHY2hzWrEYuERQk3lPEPlPogAliw+Dj02RHet\\\\\\\/Ox1Qem5pNVyI1Vwg5tMyJx0zGHgDsK22qOstvPrRfrr8hq1j6zpRPhmdU7a1CAy5b+GNuGtJHotv65PoL6LwjMGmDERihaYz5SwsCGv8sqKht8nKaf02CIWTrVBidToXJ9aFYs\\\\\\\/7evAFyKR015Dp07Rb9NkRaA03Ze+yUBU2i6nEZOk3lvoeL6kALfqpx0IJ31b+61eAA2oQ9I1MhbsFTgIGqPejhgGZnswQmcfigSbDxRiT9+9lOn+9w+BCMXoBAnrVwU14BeV9\\\\\\\/ojX39TPZfCCO+\\\\\\\/6VFtlusSGP2ooduQb82o26JQWjPnHdHY6dRFsw6eDZrM9MGo4HzSwcXpEY0Bze48FyOb\\\\\\\/LkUglnsS4DZMDUMwWIyM9WJdd\\\\\\\/uYZBcK8LmoGgYNGeLg3nfs8W\\\\\\\/pKE8agp+ubKPp8\\\\\\\/I8SAuEwcCVmfxB5qbPYjOKE6JFaF2UScTgYvNu+GdQx9VTALq0os65VpxgGAUWIrwdKJcidMfeUImNfgltNquB5knYDVtZyfMhZt2+n\\\\\\\/jBUMwa8jzJqO6LPV9k\\\\\\\/wRfuqpM1\\\\\\\/il0vwbWjh4iwn4VYNmPxlsEqddjbqm9EibGR4PkR3+ueyWxwyZb+Vngl24s233ei88C9RheQS6SZ6ba4ysupzqo3Lg8hWAHFVF+BdIn0A6tjrjkTbdXx\\\\\\\/VmM\\\\\\\/9xi9zp1MviOEADJWmlDxdRPaFZsr0LArHct3Hdp2SnMDX079z6r5Flc66KszvUJ7y49gRg4UxnQyrdqHrlDEboBiTDbv8louc1mIEFgM5RKhu08uY7kR5qn+1lUfAT8O9MYv5bwRgZlwOgdIPHNl1IpxDgosqKlclG55UX46ejEBFOF9bXcbwZBSr42Cb8AjUIV2JG1nACliBQ1ZjJFqQfcrKmZOirhkDFEQMA4yIxXOwabWABz+FZauS5jErjGoiESiUvZXyFKmTcW\\\\\\\/ec\\\\\\\/fB8M3XuKjxUwMQZIx5KGw938vSSHDwlBuENwj1W5uH+8Pqva70+QapvSai8M748tz7w4zzXqxSU\\\\\\\/NwATPStIa9yNHHp5FlyZsBeUGoBshC7TohXSgo2Lj12JNzIgYavA4fPjf8P7tgq+dHZa3afn9ikAnOVxjnJGz9hrquycWk7qJ\\\\\\\/nN7eLMMl2hnDcKnKZjsuX6278erriy1O7+KDVydyygiVIhQ9hmmWQSfNcsE1\\\\\\\/V2YxHhLVQ\\\\\\\/t2y\\\\\\\/PeQLNXJR7fSI5ZdqD+8vIcJ7Hv0hzDJYoBa\\\\\\\/K4pfsIFfatFnk4spRRndFVj5\\\\\\\/GCsXNnrF1Nwh5wMF2q2Tl0n1kUUyt3uRBEv6Re9PiG7y8kS3f5QVI1CVgbxtjnaRRVlFe5J16szj2SI3KBvuesx2g1wm8zZ9fQIKuepvnB7g\\\\\\\/Atl+d8e7mKno2jTPl6w6JyWep3dtpGrwbt0\\\\\\\/QpXrqfp\\\\\\\/ltOobs63A8LJLcnDjj3mwK3QcYh+owYYeYx5Ryn3domLvSlmSFycpiT\\\\\\\/nAJr3N3SZqa3Z1od7aTrHm4IdKzOkUqiIYsdB4s4+UILbFJS5cba4IK+VrHfVHHo+stFMLwxR68HscqNg2pfyV\\\\\\\/BQKWcK7W7GFWmqUOJ\\\\\\\/7oaQe55ZU48SvsjU69mqpaxfAKvVyK2fq4kYg322LKt6VAXRi3vtGKEzMGeU3+n\\\\\\\/lJ\\\\\\\/262sxtyBleP+Y2mj8WpyOVWTWmQ6ONnoCMIWc\\\\\\\/KiC9T5qBYdRa+rtPSni4dSxRF4Q70NYwaFgOiUizw7DoPybs8d2nj5N9PsrAT6kKAGWRGv+85YUzkZErbjGnUprgBvMX5Prmororkwk8uDPRf8+zux3Hd3XfE\\\\\\\/8ml4i5Jkw+KqSvAdb6CVbYpS2NQyaAYqclGCMyZpvUi\\\\\\\/EBJlAP+vNN2kIC3tvciomaItyWPwjezbGcy3rtK9mPZGGXOtg\\\\\\\/bkib9g0IqNxL8XxmkCiU5Gurj4lmUK43dKgeNqCfeZqg1pkyUnKhS2j5t5JNkyanGZ1gNxmQdf8HIpn8LUrDIKvKJ7\\\\\\\/lHkC0ZZzfKOYey8ox9aK1j8l116bYZ1c1tgroveCOT+UFzQrxVeQAbNTf4OsHr0wzdxCMf+yg+qsB072UBKZjPK93kf6FMV0Ezsfnc3lJhfxYn\\\\\\\/LjYY3\\\\\\\/kFXZzSJ4B4tsmMi6gQ1RADNuG2m4CXvUB18azwmGsYJ7oFu6UBIM78dgNoliF6aqJjc7Mjzxh\\\\\\\/kmT6++9tW7DlHkvQVlpGFZCwSVXcKnWEifq6BsmogaiaOVXhLZitSaVgTyTQlG5vuVeENu6EdvdpzgrKE0l9yS6IzlHnzvyKREzu\\\\\\\/pHKE\\\\\\\/M9EDesICk1FlcVeLKq1a0HssVNzk0UDcdNtgxMY8BsAem98chr5POEAOkarK5VffRyAa\\\\\\\/jUbzkTo\\\\\\\/IgJO57y3Q4EMguFEeU5B9onAq57GQa9ieoz4Ug68dwI3tQaI0UaFYQCIsrhzfvmXIK+nqTpKXAva7SetrcqVRQ6mVlxPvq\\\\\\\/RhOV23SxNbvMqpagam8R98HzMXFojqvasKauAlRT9UnOi0Ja9dmt9O3KTZ4wpkjmtqwy6BdHePafNmxYRpMLVf5RKmuCtWgb226KD\\\\\\\/E6gO\\\\\\\/GgGxNgvvqcNkWKE7AM2Zz1Ev7lAoInG7xt0SZXZeAdEe7kN3BP1AfwC08BHehGLtE4GEoo8XlrvF6a3xRsU6ayuiMd17kquy1iyM5BhiFiLXpztZ0kGxNiz8WxRl62vu36qsfaOT+L9i7zpxUwyJnam85wQ6rVHWsGiP8UYzYBBKUVkFBXybjPOUaaqwCkHNfxVKGirg+nfXuelm1gRQupHY6UNPLP2G5sjuaPU6+Ybe5JNMgXTOWHq4ufAzmCRnDC22urjESDLaaYcCKy7GeQ6E+QYLjj4I0sCzSCcmsoh9II9kSZx94XOQaCIZ9yTJDAwHz8FNg09qZVD1MIC7\\\\\\\/AGocMuNxs81ZO9T0N329sVvBPJtURRku9E6uLBwPDPN52RcL4KCLLrmHIvRPR6BpvGUPBPBd6WLEzPR0DulG3LACmaUIDlOplSd2ZdURmn8ER9nW7PlX8Daw50Gkj\\\\\\\/kL9yWZrTWVh1mXuRFMja\\\\\\\/qDKvrulxFlu1reLNTsdQeBkmJXLgx+BNiMgcoa2UAHGDsLktvl5rX6A5yNwOTAsibuTL3JOPLQDpnbXh6j8LL6eDowTARGvvU2zVIJXG4McoZ9SM1MDo\\\\\\\/lqcaBZ9G1aMEQB39Pg32js5HBdA3lN17Awa05Ggme8iaZOJv5OJziIzKDzEnXDhWxiFE5P7qb1tOrLu\\\\\\\/lynDgqo6MJnHnSqiM3twnqk9TIgNJi2mws9bz7KJXloqfy8uNbD\\\\\\\/I7aqEUM4j1W5Ri1ubni61SZYf+SbL99U9ywnmirtAOlzyzqR4\\\\\\\/MpnfMX\\\\\\\/yIG17seY+QmkOqbV5f\\\\\\\/sNmw7877TcF6loFPJX7CrZR0nepuAc1TyoUcXtSy4DQUQePo7lWqbMlC0lrTs2FP5sQA0a2BeEG7pmOI6H+RT98I9wF+8ISPh2opqc\\\\\\\/fRVR56JR4eMf93HHY5JOMi65iwv4aBMGJkbuPovxsdigtWbsIweCwhKaUEvP23wtB4o1MRBJKw3w5At1\\\\\\\/IPc3j3gzT4TnrcggCfXaxgzBIRszn5ccOS6Xt69\\\\\\\/1FpCbt+EwJSAZMmKIKuNTpbb0yN7kLQqb0fGJW6VokS5Hj6tnStZhxiTzKsO3ix7TpbsLFVVb7dzPhT3YYRnQQ\\\\\\\/RmI51fBjROTjIp3xoveqlD6\\\\\\\/t0HmL8yzxRSAOpO1Zn2U+swrNDROIQdTkj020SRMoOkJJ66ksSJOcYjLPKv\\\\\\\/exGx0aUOA35FtpaM++BsZG6iICERMMu3TSdU+3IBGlUR7qAKr7UoxJNtGZmzIZC4EmmGCW+kQJZ9ZqN7voN1ODQUnf\\\\\\\/zQz\\\\\\\/8wJwYk1LKMgitT+71JelzxPmTCTJiJXJyG3aP5M5VNQUpHRlaazAF9Si\\\\\\\/ynvK5QDWo7FmUHq++91Ctvh7497JSvP98uwgRprZSUol\\\\\\\/z1LNjv7rwvEglWm8TtTLYTyFMPNQyNUdEMbi+tg+721KBbyAM7iMuw4wt60xFNYkSnYr7Y7485atsyy1XKMTuyOxm0lG8sPlJf20qOP9ZmZizXNosDMezXpwGeq+fptspmS8nY\\\\\\\/hP0tAMlvEabQFeTAcTNfVHL3QHsAquGWhIYGtb7ECpuevHaSM7it289uecBmTSNMkQ+10r4mMYlmwpmdSWS64iFWBaLKIWOtBMMgSvLyFK+1C0FADdPA4A4+kxbd906qYNqFAto1wUSPdutWRCqkLMUJ9vsQSm46ZFRi1TQThowksVohxuM+2Cp6HiUntXP40oa4jQKHPiPk550sGQ7xB+R84Fq5Blx\\\\\\\/sesBVQ\\\\\\\/pzHwjEwwmrq1s3aRv1wtsRyHeFm3QqsgMP4gBHpLVAKEny1kqFLeqn\\\\\\\/fON2+O1iMRyXvhmEkK9oaWHqr2+Mcudoi4dQ+jNLbqb5VNgP4svEU5pog2YFnHmJnfG2MhGzurt3rVRtsA\\\\\\\/McDRkRUT4j8u7ogii0yDUqnKCvwsWWmXBYJ5Fy6ZjaxopapZwLQYETJjLb2CsQQCHyM0EVdQUQ6JfP6h2Pt6LcEqy6Vl\\\\\\\/IQNi4EI4ddI+bfnDI2tijpopMylWe9P8NdFeRCz3gGMBO938chVW5rCI2H46bxa51ZJdSJJqi15WjDMuAmg5TZq6G0s3mYe4XN657Oo1VxTLnZBEpm9MI\\\\\\\/Im2aLptu3NhTZEhZAB1lk8Z7OpURo9qKr3x\\\\\\\/\\\\\\\/DO4IV9iEcgewFfLcJaVLxBdlu429ZOibF1sk+gZ\\\\\\\/+Xvr7TrQkF48MyHknvmUnw3kTWeFmdPBtrJxNS3tYQywcS9kNxqFVb\\\\\\\/hFKrcza8zGczlSRHkPIDTAnzmLK7vqeDmPMyv990LEeh9xAk0e2v4pz\\\\\\\/QpNgaIZzKZRTG1brme0PKud7IIIhX21jk8L1kzYxE7BiEPwqAJ05YQ8dBBZwExqmnaoSBdbzAuatobNwQO8ehDLtSc9bn9weucxu7OaWYVT0ZNxL8t6e4G44h9GF830klK8RTbqOpHLvOunzwkNcjdui\\\\\\\/39n+IQ56LTs\\\\\\\/B19oZ72TGuX4fa+m2mbbg7B0l9we\\\\\\\/GSlXI0rmGtgM1l+tgCgd5\\\\\\\/QSQSHaOLioanpblw8qTKOxgkj61kR9fqlgIU0SyN2WVDE3BcpRKRNpE6wrBx9eAYQqbnIGthekoVM\\\\\\\/LqzCL63nZBWlotmXh5YmXkVYEtYgQ2+osjmSNPPDmA2DumBQR7il8xZVDYgkIohpBxEQ7ISrS7CnA4jnyhbj\\\\\\\/DfdUtmzNm5KbXYkwnTosjo0YvJWpOU2zYrvMuKd3V6DSOA6WnScjm+8hZD54KnQ8wMHdzIk+qajIzFCl5h7YW0R15k5bBgMb1I3qxO2nrEWyFpv9VZ2K\\\\\\\/0Rt4nnb5d5MMGpJ5jgmPZm3rmEctNn2tCOX5FTNzIO01orK0GYb8aDoOTKMDPFwJCoEkNuQbgs3Jxomw2dNkMova9Xbq+KCrtGdSRjoCryqsfMqYB0uzwW6Vtx9L3URw3vuMAmngw433pfmEMqIem+XokgOW85vlWdKjaQmWtig3CaTGZ76ImYxK8rR8JllgHpZ+ih3HW0EsInzSiY8V6h77i2ZEtNrbcg1G5OuihrCbMDWV\\\\\\\/eQg22Now+B8tObmZ+kabhQShzE7aIPf+9DqLtPkiT+95DZkhA9b9B+YbzVgwRQKXuleceiXd5psKIWFHzykooif3e6bZDerO15zWs6jJUW0zSZ80dfiSZ4b3bDesqHmRhkMynAOFPxf7XtuyqTEKllwifPuKUgsz3ckgJPYgliftThoBYG7jlm6tk5mVaT2qpqGPzPi4BmHZOHsJekMGex3Rhl+GwzMQuBNey6QS+5SfbuFBJhoXolb6JvS0jOSaCzpdfSxHjETwHhU\\\\\\\/c4zbPGUTOfOrn6sI1dp431b2GLUmDgmT\\\\\\\/2U8qglltNo+177l7HGEFOUzoLKzER70nM7SIiJFjt8tKZN+0rB+ib6+xmS60CWluMBrNMxfAuaNaoN73hJ\\\\\\\/tDPkmBPqq+ETY+fWQR2WCyOQ34P0xuKXzQxKLJanC3FjK688WJzg2MpylkpOvWaPHLvWulDEQYGENw4dscCuIQtdvnOxu2pMpKB\\\\\\\/Qu9UxGGMUSpUTbW+sYyAmdODk+8WAAEy4leGpTs1etS8NerhLnUn77UVgvig4BMrq\\\\\\\/VpjBk\\\\\\\/iQ2nUp8YHxOaQT2jr+zPHq3tQveGrV57wj0LZW5nIO6FE4Ps\\\\\\\/v55J4bhHLVKejAc6NbvpWnpViqi2yvTpaweohGFQ+LuvPYwvgbbnHl45GIAZLva0BAXomXQ7xBLdo0aVN+xfrFleHg2wXj3ndcm\\\\\\\/dD\\\\\\\/i1vcLG27oZsD9qRtivqLH300pILM7BQqx6rhFH\\\\\\\/1p4eBkP5jLufD\\\\\\\/ZxnFMpLF\\\\\\\/WYEZqK7iuk19OTx2krHU0JzD11HRURb6srWYq8etc9QIabOO9Ft5Sy4OyODwgGGvkhx1rXghVHotZSUdBZCgKJPXfW1XjZ6jOYZ7du6g9PbIc0BDX9nDAfqSsUnCytofo4WgAiXbiWMXP5jKjbzaAidEgyTOZIVveq7+QWcpQBExeOJLLJfzXUNOXohN5O5HlGOTbcpWF57L+\\\\\\\/GhRkHTIUPjeBGAtmriTEAHxm\\\\\\\/R2Q14klelrGhtKCqS5n8jB8AL3B7Z4Q28WfwSeidgj3NPOset2IWxb7K3U7IF7L836gsPsWljhjL\\\\\\\/jV9WRuhILzSn5wbm8xPBFO0p6HeVuA7Nmh7hKFTSiAE\\\\\\\/1tV8JfiZr4DjzS20YtZ\\\\\\\/RwqOVWADyZCR2QWUSmwy\\\\\\\/oEDhvYKxZQArbGrw1bd5d04KWK+HjmXoYAOtrS5FlsiP8JkdUgO4a6a\\\\\\\/Hvj0BmPnP+YTg2JAd8AVRQoImKYjsKqbbxAUdt8vCFJxh+zu0cjEngeQSL252g7AFlVfLGpDjHwV3K6wigPFGYM412D2BWO46oEALrpUdIg5wFnelc4eqJlUHBrpbZV1B5FtJniwEsr\\\\\\\/prJa8vApBDnMdY5ow1aQW4LC3Gzispjlq\\\\\\\/j2Jmq2mm\\\\\\\/14jLwkb6N47PcpjXxTqeeHw0ZQoXO0e\\\\\\\/pDVwACPfTrgyl7jR2BnJhq79S1DH+WNPYREW55C2vmoWUX6j83XIHMyUW3t9ii4mN2jUbiXrkNkTPi5jVAiyJDMQTxuyvJ5yEIYGKKkUhR5P9Q3vHBk8PedQp+ySxary0xULQKqWeOIARZSKp+Xq9PIlbvQ6vWqCN6lm0KcvrnN1jl1D+nbCjaugz4ldAEErC2zkgRiqlSRY9lpHBnGeXEoI6OoJzasSZYxlqzaS3S6HSDn0Zi0qqW2ugZrzqWLBQE+CylhL\\\\\\\/DlKPlfYoejuaDMZArRr0r8EY\\\\\\\/ZYRw9nqfkNzoD+K9wJlswesed4Lvk1jAUiAwN\\\\\\\/yGiSUufvFqec0PzT+Zqkf3mimQF47ULXJKUeCR\\\\\\\/OzNFCYIvWWTxU8aOMt8wDtFtzgMeqNul2lOAA8xqxDxZi5XzWPQbzr5HYlAj6JERaA1luFGkCTKMSvsW8eXg4mtbElg2KayrayqmZ64XwN4mYaITkoTzD\\\\\\\/NoS27xI86aiXatkNFZlZvstEVcA0YqWkdsiKW56M8aMlSZUQNREuB8ePplWG4O\\\\\\\/shCv8us\\\\\\\/pRgbb6Ry40sOuCz0v9BED08T7w0RNrQPHUYE61XsvEEGsOHAirkSAxxuDriEvO0LPv9RIWWJf\\\\\\\/4E06\\\\\\\/waAIYzqGqtaSJ41CivAOAR7wPNwTvZoeRXufeHsmxsHWMZf0FWG1zJ\\\\\\\/SwuwPsn4Zo\\\\\\\/9zdbWaT0akwKNAPxVt6UDOj8MXJbq0aRtUU0GcbObw2\\\\\\\/N63zdBcE4I5zcMXo6REFpBgyBdHGIfuzZigy7C5mlix8jc9cgf0gRzHwyqnP29G6vqcvDnBIVEpKfSE\\\\\\\/N5CQbLYxVKi8RDAKW9EldqpQzekK+9JNimndXdmJARINGl4QOykLP5asqjIMRV14VHFdODPH3wK5LQb3QTyW79vaA8Vg1jIhJrG1TX02i0Pba4fh7Y\\\\\\\/4CUvpHmm0L+g6IuMILJa\\\\\\\/ce1zp2OARbsnKL\\\\\\\/TaWkHCpJ0\\\\\\\/oIfdedsTWXkxhD5\\\\\\\/2TFIK3I9Hsxp\\\\\\\/mkwUKYbxpMizGDSNrkmqTnqU1jIkc8v8lwevo7n\\\\\\\/W5QPOyI91Rajm31RsQ4cjztPDwMUP2mQnmc4xfRtP4GJvup47MPav7s42P0elk3YTW3hxD2pESh9c8u9riVVMS+VvI2UEd4r7RBztrGoB6hka5piHuRDeu+ZL5H\\\\\\\/YRZZRpe8krQjccltt31oRVwcfalZNIV0OdMODvg\\\\\\\/5ITbdW8GQHf1r5pbusjjAjPj\\\\\\\/MZWap4y3IyiwFg12l\\\\\\\/TizTTKYCAbTwKe5d3j7XDFqqBs\\\\\\\/64TE1aSYQ3XUESAryjb3amLvLQQZ7WlB7h+N3UaO80vfhmg9aEYAOILdhBo1S7EaiC+zJ2qk1+Kbv\\\\\\\/Iol8B2eDOmQ7Lk7CgVT7hPp6j16PC8144GXGQg\\\\\\\/hG77vmSBPv0Bnnv3hYylCCGO+coYoi0sjNAE\\\\\\\/tcvdTUoYD4xgiJGVqk+xbrBnXNxK4XbUJJ7pyIy7uUdYnTX1RiUVmaSptuFHNETvkgRbV4YefTjG8k7R0fNSXWbElGypR\\\\\\\/I0ddWDm3IjDsPL0zh9xEgqQActg5sEyUEXiFucZS0jTkBJ\\\\\\\/TGcNq4UiR1rk\\\\\\\/qL48lcSapF8c2ZwhkJmyY0qybAh\\\\\\\/Aq1b1vJQZ8dXA00d+OGz+yXe9TGgxaE70hJmP2RyL3XKf7znvba9by79Q6jz+StPWLEKQwrLmYI52u5xHqx5DGJIhxJiumYmbcwSPIvcYpz2D7RqFG1WppRpaTu31uqgAet7ziAORGnGdilJ\\\\\\\/M3eSKmE4kqF7BDkHEc0w8Dsl5gHNLI2fmfslo\\\\\\\/V39lPK7ne\\\\\\\/oECfY5gCMsWQKdJD4drzS\\\\\\\/ZCYBYYUj2TzoJmlp3oX3oiPVyeL2pITKJvKVxaqbSiTMryrVdlt3e0aY3STXhkxayUR1ZXZD7VH5iIxHVGhPxMnVyLjv9DPrvUQweUMsz5jMzkRNyfOjHCY0hocs1a79HlSoKr3H9JjAaPku5Eq+gi0HVUa2MbqMM+L+41N0f34aA4BGDse505NGroPMY3QBVjJE7MXnO00Zy+deo3dMwynHsgqW5XKMgpZubrl0Z8TEQLh1uKa6CAhONRaoRlLhMU0Y6Lkg0RkUIq1wX8EF94iAUTytzTM3tCO4fok+yiOt4\\\\\\\/8Zi9xVcZAcna+vKxj3Vee7+PjVUmPswuTCydoXp+nrH9ZQj9P\\\\\\\/5PlBy0WGTDsbVCT8PLJlmnYTOMxiQ6lkhchsDEZNd0HV\\\\\\\/3PYEtm21JkXjeaAb\\\\\\\/0v3F13D5+VgTKSN8jtSxmDlMZV94\\\\\\\/QGqGIbJ6RSnH6u40UiVo5CExu1OVmUXPesjCdggf\\\\\\\/+BK7XvWKcSl7hE9NdvYcUSJBUZmR2i1hek996q2FtyXqaMY59svMvHoFxNJUknRowZSmeVNXlhGLENT2l2ipA5GqS7B2I5M\\\\\\\/vjW+PAjCVaQBUp3mj6Zk02LdUHR\\\u0022,\\\u0022iv\\\u0022:\\\u0022nkqSvkNVb9XTJXIn2aUyaQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002273d29a124c5cb3c38ce8a2f4843c51d8d30afaeb2abfe354286ef80dd9b76c98\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022QzzWd3fnYGF1St1y7VC7\\\\\\\/m1Ih1yEJ4rA6J7ioivoD5107wSn6GYxpV6P\\\\\\\/23aFTf3sYwMJyyDH2JYTEOb6eQWFgGV350yoOOxUXOvlA1H4SRRKcBbTbUOA6aCnqJ6jncCSLQ81oFFgD6fTch9FQ5nRW82AOzuGh\\\\\\\/K5unDT1rTwp4dBlNXHLAaFXmXJivtIyb11NxDAw3dIICoqgHSbG2cOjbYKT5P\\\\\\\/aldDESJFuFJ3sC\\\\\\\/FZBbNcA6WOGkhgkHm9TzyXolsBnWmCuFRFXQadxwaYbZh1FarGRp18UuzYciT\\\\\\\/s0GmScwUl8CrBXvnRIrX1P25CulRVp7I\\\\\\\/fG7cmp\\\\\\\/QkcKbX+v60Z72x387iqqWY6mv5xB03dOwRq+q80TIvV3T68xQa14IK\\\\\\\/sMcWnaOMxmNy\\\\\\\/hHeWeaGG+PS\\\\\\\/h1aY+QW\\\\\\\/PGGhKM4KaB4sCDeg4kFx4x9KUEuehLM8CQPott12+0dnx7\\\\\\\/2G54IKXyBZ5LGVmH84U7BUHZ+raOGFUeXq6OVQ06q\\\\\\\/mmre8sSaUGuQDnOOtUjMLYQaysE+ImuW2ebLmta8kCtzbWJI1DugSpgtqdfaVTkUiBfxb75v42AvpnlChFEPwNSm3j8jU6lRr\\\\\\\/7CHgAI6ECkKq80l7NQnT4ZiBEaiAxxR66nf9W7922VJoNWZiEuBv5unMYxrb61URVFBMkANawEV+uIwfjmKdD7H10xfqaJKihqi7U4ofFsf3R1wWlttfa4+RJT2tMMMfUb9sst2xMr8sVCMTvW2WZ2XNySZuc4evUYJ8xoW+jOSOW+xZ\\\\\\\/azZGpVVjb11BGYMls83W9SBwci436l5yx+HqoH4G\\\\\\\/\\\\\\\/OQ300CHSvfPozoOXipQOHyjzJo3r2C9GSW59b1cykwPbvmUgrCnj5PFM3UUbYxzU\\\\\\\/L03tUnw9KGL6fumM9hjNcmohgamnqujkFdE\\\\\\\/VLYmlqEKYBOcD8iFu\\\\\\\/Pw\\\\\\\/Q9lg0dBAD1wpSoMgnz6DKX3a7j13x\\\\\\\/BOdAwsGOgOIOAL8LklaZbDtSWSOQNacTDdLSmYJ6W9O5EH1WxI\\\\\\\/47jOAo9PX4AkkSEOCGNDZUdG6ZKWjAayr3E4Dhhpbg8epGMJDwaV+CrrkiePhNGq7pZlM7X\\\\\\\/QAaWjf+GI+ttNcHPFjWuC+GHfp\\\\\\\/SgcFlxLanO1meZhniZDTITLjcNUZ0ak5i2mgVILVyjXsy6T7tERhAHbvu5Kigm1PUd\\\\\\\/iF+vmE3BGX+b\\\\\\\/Y6ntrw0zL3d3H1JWl2SQG6gLT7yStZZxHChvcoAz7aHg\\\\\\\/MfywLcQlrFKw2DnuEhzTa3QasruLqDKG8fazmC3Q5PHvMFOy9csT+MM1YRXiwvzKibZ4jcVZ7pt1GwxPiA56BBnbx65zYEEzQq\\\\\\\/ga25ObUg2R501oF9o5nxCqrjRFyQm8tWIKMX74guYsE7SfXT4cV5GaSCsDfie\\\\\\\/WnUeZviO8xDq3vywzoZTyp5\\\\\\\/VTWeYVaazOncJ6DWduHf3PuRlTR54feXgVH+FJrT3\\\\\\\/PRZ2p\\\\\\\/KhW4qlhycyULFyfwFTkoxDqJ96mbBvaQllGUQaAmXIUMtrZoVVoo\\\\\\\/B5pSpTTaOpXrAMzHiDtT+kHYlegvI9GP\\\\\\\/k53XIfgnW9Elv\\\\\\\/NHDzFWvz2woNb\\\\\\\/aXzNLqzgAo2WX+VLzhQ0Y4kcbJLn5k6emSgEeN\\\\\\\/swCHw8zA2n688hmP0JTKqrfcNoE62FDAsWdYrLwQL\\\\\\\/Sw\\\\\\\/qpMnsjgjzI5aZtZavTe80UfbSpuGf64iL\\\\\\\/CrjKGMywfbiIWYX3hjh94uSSrd6AtVlKdC+I4g5COwLyjlqQNFBYt+2itifxSBCFFb7fbSwPUUkGZS3+qUSIYRzqf0qu5tzUj+fqvoL6DpSU8T6FX3XDBwo\\\\\\\/fY9BesCO5vCEalWVKiH12FygoGTKDOZ8k4sDx0Tx9447qZqofrczBp2M+Td8+zk4FxyHCMEx5bpFz4Wqn8mnr4m\\\\\\\/Uy8ArxH\\\\\\\/slwl7aNRrYF2MaI8MHyBXzLCG1tdrHncWbuUwowhO5PnkyUNFDcr1V9kcmTp+pfufnva4AP3U5DATbFsqJsfG8H20WPYxrXxqA\\\\\\\/6IkXYKEJN\\\u0022,\\\u0022iv\\\u0022:\\\u0022yZVuishClGsc6HTVPdluPQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00220c2fa324abaedaedc3f934e65f038cabc35e27542e46db570d75cadee9297f41\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-494281 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/10/3129" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-cY2d-OTRTKZWbOHH8ZxI5hmeJAmy4Uk-JITglm3ra4M" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/10/3129" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form--YEorwXdQ7lFaC46aSgUCoxkgDSwMxRzLITcmVYNjKw" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/10/3119" title="Transformation with Artificial Chromosomes in Oxytricha trifallax and Their Applications" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/10/3139" title="Genome-Wide Association and Gene Co-expression Network Analyses Reveal Complex Genetics of Resistance to Goss’s Wilt of Maize" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494281" id="node494281" data-pisa="ggg;9/10/3129" data-pisa-master="ggg;g3.119.400319" data-apath="/ggg/9/10/3129.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">QTG-Finder: A Machine-Learning Based Algorithm To Prioritize Causal Genes of Quantitative Trait Loci in Arabidopsis and Rice</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-3742-5291" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Fan Lin</span>, <span class="highwire-citation-author" data-delta="1">Jue Fan</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-7572-4762" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Seung Y. Rhee</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3129-3138;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400319 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494281" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Fan Lin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Plant Biology, Carnegie Institution for Science, Stanford, California 94305</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Fan%2BLin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lin%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AFan%2BLin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-3742-5291" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Fan Lin</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jue Fan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Plant Biology, Carnegie Institution for Science, Stanford, California 94305</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jue%2BFan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fan%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJue%2BFan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Seung Y. Rhee </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Plant Biology, Carnegie Institution for Science, Stanford, California 94305</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Seung%2BY.%2BRhee%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rhee%20SY&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASeung%2BY.%2BRhee%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-7572-4762" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Seung Y. Rhee</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:srhee@carnegiescience.edu" class="" data-icon-position="" data-hide-link-title="0">srhee@carnegiescience.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/9/10/3129" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:494281" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:494281/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/10/3129.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:494281" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:494281/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/10/3129.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:494281" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:494281/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/9/10/3129.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:494281" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:494281/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Linkage mapping is one of the most commonly used methods to identify genetic loci that determine a trait. However, the loci identified by linkage mapping may contain hundreds of candidate genes and require a time-consuming and labor-intensive fine mapping process to find the causal gene controlling the trait. With the availability of a rich assortment of genomic and functional genomic data, it is possible to develop a computational method to facilitate faster identification of causal genes. We developed QTG-Finder, a machine learning based algorithm to prioritize causal genes by ranking genes within a quantitative trait locus (QTL). Two predictive models were trained separately based on known causal genes in Arabidopsis and rice. An independent validation analysis showed that the models could recall about 64% of Arabidopsis and 79% of rice causal genes when the top 20% ranked genes were considered. The top 20% ranked genes can range from 10 to 100 genes, depending on the size of a QTL. The models can prioritize different types of traits though at different efficiency. We also identified several important features of causal genes including paralog copy number, being a transporter, being a transcription factor, and containing SNPs that cause premature stop codon. This work lays the foundation for systematically understanding characteristics of causal genes and establishes a pipeline to predict causal genes based on public data.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/arabidopsis" class="hw-term hw-article-keyword hw-article-keyword-arabidopsis" rel="nofollow">Arabidopsis</a></li><li class="kwd"><a href="/keyword/causal-gene" class="hw-term hw-article-keyword hw-article-keyword-causal-gene" rel="nofollow">causal gene</a></li><li class="kwd"><a href="/keyword/machine-learning" class="hw-term hw-article-keyword hw-article-keyword-machine-learning" rel="nofollow">machine learning</a></li><li class="kwd"><a href="/keyword/quantitative-trait-loci" class="hw-term hw-article-keyword hw-article-keyword-quantitative-trait-loci" rel="nofollow">quantitative trait loci</a></li><li class="kwd"><a href="/keyword/rice" class="hw-term hw-article-keyword hw-article-keyword-rice" rel="nofollow">rice</a></li></ul><p id="p-3">As the world’s population expands, food security faces a major challenge in the near future. By 2050, world population is projected to grow by 34%, which will require a 70% increase of global food production to meet the demand (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">The Food and Agriculture Organization 2009</a>). To catch up with the growing global food demand, it is important to improve the efficiency of arable land usage by developing better crops.</p><p id="p-4">Many agriculturally and medically important traits are quantitative and controlled by multiple genetic loci. Examples include plant height, grain yield, and flowering time in plants and common disorders such as cancer, diabetes, and hypertension in humans. The variation in quantitative traits allows organisms to adapt to various environments (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Baxter <em>et al.</em> 2010</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Leinonen <em>et al.</em> 2013</a>). Quantitative traits are determined by a combination of genetic complexity and environmental factors (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Mackay 2001</a>). The genetic complexity of quantitative traits comes from the involvement of multiple quantitative trait loci (QTL) and the non-additive interactions among them (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Carlborg and Haley 2004</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Mackay 2014</a>). Causal genes of QTL are genes whose differences in the DNA sequence or state cause a phenotypic variation in parental genotypes and are supported by multiple lines of experimental evidence including mutational analysis, transgenic complementation, and deficiency complementation (<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Weigel and Nordborg 2005</a>). To understand the evolutionary forces and molecular mechanisms that shape the genetic architectures of adaptive traits, we need to identify all major causal genes that contribute to phenotypic variation of the traits and elucidate the molecular mechanisms of their actions. Achieving this goal will facilitate rational engineering of plant traits and more accurate prediction of the effects of their modifications on the engineered plant.</p><p id="p-5">QTL linkage mapping and genome wide association study (GWAS) are two common approaches used to identify QTL, each with its own strengths and limitations. Both mapping approaches are based on the co-segregation of a trait and genetic variants in a population. The population for linkage mapping is usually the progeny of parental plants that differ in a trait, such as an F2 population or recombinant inbred lines (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bergelson and Roux 2010</a>). GWAS mapping uses a natural population that has a heritable variation of a trait. Compared to GWAS, linkage mapping does not suffer from issues like population structure and suffers less from rare alleles (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Bergelson and Roux 2010</a>). For example, the most significant seed dormancy QTL <em>DOG1</em> identified by linkage mapping was not identified by GWAS, likely due to the rarity of the strong allele in the GWAS population (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bentsink <em>et al.</em> 2010</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">He 2014</a>). Confounding population structure can cause a high false positive rate in GWAS, though some methods have been developed to ameliorate it (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Price <em>et al.</em> 2010</a>). However, efforts to correct it could result in a higher false negative rate (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Brachi <em>et al.</em> 2010</a>). Linkage mapping is less prone to these issues, but it cannot identify QTL of minor effects when the sample size is small (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Otto and Jones 2000</a>; <a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Xu 2003</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Martin and Orgogozo 2013</a>; <a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Wellenreuther and Hansson 2016</a>).</p><p id="p-6">For QTL identified by linkage mapping, finding causal genes underlying them is still a big bottleneck (<a id="xref-ref-5-3" class="xref-bibr" href="#ref-5">Bergelson and Roux 2010</a>). In a typical rice linkage mapping, the size of a QTL can range from 200kb- 3Mb, which can harbor tens to hundreds of genes depending on the mapping population and gene density (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Bargsten <em>et al.</em> 2014</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Daware <em>et al.</em> 2017</a>). Even in the post-genomic era where all the genes in the genome are uncovered, identifying QTL causal genes is not straightforward since many QTL either contain no obvious candidate genes or too many genes potentially relevant for the trait (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Nuzhdin <em>et al.</em> 1999</a>). Therefore, despite the many QTL that have been reported in plants, only a few have been studied at the molecular level.</p><p id="p-7">To narrow down the range of candidate genes in a QTL region, conventional fine mapping is reliable but time-consuming and labor-intensive. The basis of fine mapping is to create a population that has more recombination events within a QTL in order to identify a smaller genomic segment that co-segregates with the trait. However, the enormous time and labor required for creating and screening a population of progenies limits the usage of this method (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Tuinstra <em>et al.</em> 1997</a>). Depending on the frequency of recombination, thousands of progenies may need to be genotyped to get to a gene-scale resolution (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Dinka <em>et al.</em> 2007</a>). For example, 1,160 progenies were screened to identify the <em>Pi36</em> gene in rice and as many as 18,994 progenies were screened to identify the causal gene of <em>Bph15</em> in rice (<a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Yang <em>et al.</em> 2004</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Liu <em>et al.</em> 2005</a>). The high cost associated with genotyping and phenotyping makes it challenging to apply fine mapping to all QTL.</p><p id="p-8">Alternative approaches to refine the candidate list of causal genes include meta-analysis, joint linkage-association analysis, and other computational methods including machine-learning algorithms. The first two approaches require either the availability of many QTL studies on similar traits or an additional association mapping experiment (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Buckler <em>et al.</em> 2009</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Motte <em>et al.</em> 2014</a>; <a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Yin <em>et al.</em> 2017</a>). Computational methods including machine-learning algorithms have been developed to prioritize disease associated genes and genetic variants in human (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Perez-Iratxeta <em>et al.</em> 2002</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Kircher <em>et al.</em> 2014</a>; <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Ritchie <em>et al.</em> 2014</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Hormozdiari <em>et al.</em> 2015</a>). To distinguish disease-associated from non-associated variants, a variety of information has been used, including the effect of polymorphism (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Ng and Henikoff 2003</a>; <a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Kircher <em>et al.</em> 2014</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Gelfman <em>et al.</em> 2017</a>), sequence conservation (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Pollard <em>et al.</em> 2010</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Huang <em>et al.</em> 2017</a>), regulatory information (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Deo <em>et al.</em> 2014</a>), expression profile (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Mordelet and Vert 2011</a>; <a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Deo <em>et al.</em> 2014</a>), Gene Ontology (GO) (<a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Mordelet and Vert 2011</a>), KEGG pathway (<a id="xref-ref-40-3" class="xref-bibr" href="#ref-40">Mordelet and Vert 2011</a>), and publications (<a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">Perez-Iratxeta <em>et al.</em> 2002</a>). In contrast, only two causal gene prioritization approaches are available for plants. One method was developed for GWAS in maize based on co-expression networks (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Schaefer <em>et al.</em> 2018</a>). Another method was developed for linkage mapping based on biological process GOs (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Bargsten <em>et al.</em> 2014</a>). To date, no machine-learning approach using multiple data types has been developed to address this problem.</p><p id="p-9">Here, we built a supervised learning algorithm to prioritize QTL causal genes using known causal genes in <em>Arabidopsis thaliana</em> (Arabidopsis) and <em>Oryza sativa</em> (rice) and a suite of publicly available genetic and genomic data. For each species, we trained a predictive model using features based on polymorphism data, function annotation, co-function network, and paralog copy number. By testing the models on an independent set of known causal genes, we demonstrated its efficacy in prioritizing causal genes.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Data sources and features used in QTG-Finder</h3><p id="p-10">Twenty-eight features were extracted from published genome-scale data, which included eight polymorphism features, seventeen functional annotation features, one co-function network feature and two evolutionary features (Supplementary Table S3).</p><p id="p-11">Arabidopsis polymorphism data of 1,135 accessions was downloaded from 1001 Genomes Project (<a href="https://1001genomes.org">https://1001genomes.org</a>) (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">1001 Genomes Consortium 2016</a>) and rice polymorphism data of 3,010 cultivars was downloaded from Rice SNP-Seek Database (<a href="http://snp-seek.irri.org">http://snp-seek.irri.org</a>) (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Mansueto <em>et al.</em> 2017</a>). We used SIFT4G (v 2.4) (<a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Ng and Henikoff 2003</a>) and SnpEff (v 4.3r) (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Cingolani <em>et al.</em> 2012</a>) to annotate the raw polymorphism data. The number of non-synonymous SNP as annotated by SIFT4G was normalized to protein length and used as a numeric feature (normalized_nonsyn_SNP). Non-synonymous SNPs at conserved protein sequences were predicted to cause deleterious amino acid changes by SIFT4G. The presence of deleterious non-synonymous SNPs in a gene was used as a binary feature (is_nonsyn_deleterious). If a gene contained any deleterious non-synonymous SNPs, the “is_nonsyn_deleterious” feature was set to 1, otherwise it was set to 0. Other binary polymorphism features such as “is_start_lost” (start codon lost) and “is_start_gained” (start codon gained) were extracted from SnpEff annotations in the same way. For “is_SNP_cis”, the Position Weight Matrices of <em>cis</em>-elements were downloaded from CIS-BP database (Build 1.02) (<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Weirauch <em>et al.</em> 2014</a>) and mapped to 1kb upstream of all genes in the genome using FIMO (v 4.12.0) (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Grant <em>et al.</em> 2011</a>). The <em>cis</em>-elements with a matching score above 55 were imported into SnpEff library to annotate the SNPs. This matching score cutoff was determined by a cross-validation as described later.</p><p id="p-12">Functional annotation features were binary features based on GO (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Gotz <em>et al.</em> 2008</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Jones <em>et al.</em> 2014</a>) and Plant Metabolic Network (PMN) (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Schläpfer <em>et al.</em> 2017</a>). Arabidopsis and rice genes were annotated by Blast2GO (BLAST+ 2.2.29) and InterProScan (v 5.3-46.0). The molecular function GOs were then converted to high-level functional groups such as transcription factor, receptor, kinase, transporter, and enzyme to mitigate the effect of some inaccurate annotations (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Jones <em>et al.</em> 2007</a>). To assess the performance of this approach, we compared aggregated high-level GO annotations from Blast2GO with aggregated high-level curated GO annotations from AmiGO (<a href="http://current.geneontology.org/products/pages/downloads.html">http://current.geneontology.org/products/pages/downloads.html</a>) and un-aggregated GO annotations. Genes annotated as enzymes were further classified into 13 PMN metabolic domains such as carbohydrate metabolism and nucleotide metabolism (<a id="xref-ref-53-2" class="xref-bibr" href="#ref-53">Schläpfer <em>et al.</em> 2017</a>). Unclassified genes in PMN were classified as “is_other_metabolism”. Genes annotated as enzymes by GO but not present in PMN databases are either enzymes involved in macromolecule metabolic processes or enzymes without a specific function assigned. Since the majority of them are involved in macromolecule metabolic processes, we named this group as “is_macromolecule_metabolism”.</p><p id="p-13">Co-function networks of Arabidopsis and rice were retrieved from AraNet and RiceNet (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Lee <em>et al.</em> 2010</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Lee <em>et al.</em> 2011</a>). The sum of all the edge weights of a gene was used as the “network_weight” feature. We used the sum of edge weights because hub genes have been proposed to be hotspots of phenotypic variation (<a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Martin and Orgogozo 2013</a>).</p><p id="p-14">Paralog copy number (paralog_copy_number) and essential gene prediction (is_essential_gene) were taken from a previous publication (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Lloyd <em>et al.</em> 2015</a>).</p></div><div id="sec-3" class="subsection"><h3>Arabidopsis and rice causal genes used for training and independent validation</h3><p id="p-15">For model training and cross-validation, curated causal genes from Martin and Orgogozo were used as positives for algorithm training (<a id="xref-ref-39-3" class="xref-bibr" href="#ref-39">Martin and Orgogozo 2013</a>). In total, 60 Arabidopsis and 45 rice causal genes were used as the initial training set (Supplementary Tables S1 and S2). We curated and included gene identifiers and trait categories in these tables (Supplementary Methods). For literature validation, we performed a further literature curation and found eleven Arabidopsis and eighteen rice causal genes, which were not included in the Martin and Orgogozo list (Supplementary Methods and Supplementary Table S8).</p><p id="p-16">The QTL regions used for independent validation were obtained from previously published studies. Even though some studies fine mapped the QTL, we still used the original QTL regions instead of the fine-mapped regions since our method was developed to replace fine mapping. We included all genes between the markers that were used to define a QTL for prioritization. When the genome locations of the markers were not provided in the publication, we searched their genome locations in Gramene marker database (<a href="https://archive.gramene.org/db/markers/marker_view">https://archive.gramene.org/db/markers/marker_view</a>).</p></div><div id="sec-4" class="subsection"><h3>Algorithm training and parameter optimization</h3><p id="p-17">The QTG-Finder algorithm was developed in Python (v 3.6) with the ‘sklearn’ package (v 0.19.0) (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Pedregosa <em>et al.</em> 2011</a>). We developed an extended 5-fold cross-validation framework (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>) to evaluate training performance and optimize model parameters.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/10/3129/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Model training and optimization based on cross-validation. (A) model training and cross-validation framework. We randomly selected negatives from the genome and iterated to maximize the combinations of training and testing data. (B) The ROC curve of Arabidopsis and rice models after parameter optimization. True and false positive rates were based on the average of all iterations. The gray diagonal line indicates the expected performance based on random guessing. The number in parentheses indicates Area Under the ROC Curve (AUC-ROC)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1569807890" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Model training and optimization based on cross-validation. (A) model training and cross-validation framework. We randomly selected negatives from the genome and iterated to maximize the combinations of training and testing data. (B) The ROC curve of Arabidopsis and rice models after parameter optimization. True and false positive rates were based on the average of all iterations. The gray diagonal line indicates the expected performance based on random guessing. The number in parentheses indicates Area Under the ROC Curve (AUC-ROC).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/10/3129/F1.medium.gif" width="440" height="191"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/9/10/3129/F1.medium.gif" width="440" height="191"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/10/3129/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/10/3129/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494426" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-18" class="first-child">Model training and optimization based on cross-validation. (A) model training and cross-validation framework. We randomly selected negatives from the genome and iterated to maximize the combinations of training and testing data. (B) The ROC curve of Arabidopsis and rice models after parameter optimization. True and false positive rates were based on the average of all iterations. The gray diagonal line indicates the expected performance based on random guessing. The number in parentheses indicates Area Under the ROC Curve (AUC-ROC).</p><div class="sb-div caption-clear"></div></div></div><p id="p-19">For the 5-fold cross validation, curated causal genes were used as positives and the other genes from the genome were used as negatives. The positives were randomly re-split into training and testing positives in a 4:1 ratio and in an iterative manner. Training and testing positives were combined with different sets of negative genes that were randomly selected from the rest of the genome. To increase the combination of positives and negatives, we re-split the positives 50 times randomly and selected negatives 50 times. This number of iterations ensured greater than 99% probability that every positive sample co-occurred with every negative at least once in the training or testing set during the cross-validation process. The probability of co-occurrence was calculated as Equation 1. <em>P<sub>co</sub></em> is the probability of co-occurrence of a positive and a negative in a testing or training set. <em>N</em> is the total number of negative samples. <em>n</em> is the number of negative samples selected as testing or training samples. <em>R</em> is the number of iterations used to re-split the positive set. <em>C</em> is the number of cross-validation folds that contains a positive sample. <em>C</em> was set to 4 for the training set and 1 for the testing test. <em>S</em> is the number of iterations to randomly select the negative set.<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3129/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3129/embed/mml-math-1.gif"/></noscript></span><span class="disp-formula-label">(1)</span></span>We tested different classifiers and parameters and optimized the model based on Area Under the Curve of the Receiver Operating Characteristic (AUC-ROC). The average AUC-ROC from all iterations was used to evaluate training performance. We tested three classifiers: Random Forest, naïve Bayes, and Support Vector Machine (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Cortes and Vapnik 1995</a>; <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Tin Kam 1998</a>; <a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Zhang 2004</a>) (Supplementary Figure S1). For Random Forest, we tuned the number of trees and the maximum number of features for each tree based on AUC-ROC (Supplementary Figure S2). We used 100 trees and a max_feature of 9 for Random Forest. For Support Vector Machine, the RBF kernel was used and the C parameter was tuned. Random Forest was chosen for further analysis since its performance was slightly better than the other two classifiers. The ratio of positives and negatives in training data were also tuned to maximize cross-validation AUC-ROC (Supplementary Figure S3). The best performing positives:negatives ratio was 1:20 for Arabidopsis and 1:5 for rice. For testing, a positives:negatives ratio of 1:200 was used since it is close to the average ratio of causal and non-causal genes in real QTL.</p></div><div id="sec-5" class="subsection"><h3>Feature importance analysis</h3><p id="p-20">We implemented a leave-one-out analysis to evaluate feature importance. This method was based on the change of AUC-ROC (ΔAUC-ROC) when leaving out one feature from the models. The same cross-validation framework was used for this analysis. For each iteration, we calculated AUC-ROC on the original and the leave-one-out models developed with the same training and testing datasets. The ΔAUC-ROC was calculated by subtracting the leave-one-out AUC-ROC from the original AUC-ROC. With the results from all iterations, we calculated the average ΔAUC-ROC for each feature.</p></div><div id="sec-6" class="subsection"><h3>Independent literature validation</h3><p id="p-21">For validation, we applied the models to an independent set of causal genes that were curated from recent literature and not used for cross-validation. The models were trained by all known causal genes from the initial list and negatives were randomly selected from the rest of the genome. Model training was repeated 5,000 times using resampled training negatives from the genome in combination with the same set of known causal genes. The 5,000 iterations were conducted to ensure that there was &gt;99% probability that each gene in the genome was selected at least once. We applied the models to each of the independent causal gene and all other genes located within the QTL. All genes within the QTL were ranked based on the frequency of being predicted as a causal gene.</p></div><div id="sec-7" class="subsection"><h3>Model performance for multiple QTL</h3><p id="p-22">To understand performance of the model when it was applied to multiple QTL of the same trait, we conducted simulations. We calculated the probability of including at least <em>K</em> causal genes within the prioritized list at a given cut-off of the rank percentile when applying the models to a total of <em>N</em> QTL with Equation 2. <em>p</em> is the probability of a known causal gene to be included at a particular cutoff of the prioritized list using the independent set of causal genes found in the literature. <em>x</em> is the number of causal genes included in the cut-off.</p><span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3129/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3129/embed/mml-math-2.gif"/></noscript></span><span class="disp-formula-label">(2)</span></span></div><div id="sec-8" class="subsection"><h3>Trait category analysis</h3><p id="p-23">The trait category analysis was performed in a similar way as the independent literature validation except using different training and testing sets. Each curated causal gene was tested once. For each round, one curated causal gene was removed from the training set. Then the model was trained and applied to rank the removed causal gene and 200 flanking genes.</p></div><div id="sec-9" class="subsection"><h3>Data Availability</h3><p id="p-24">The source code for QTG-Finder and related analyses such as cross-validation, feature importance analysis, and trait category analysis are available at <a href="https://github.com/carnegie/QTG_Finder">https://github.com/carnegie/QTG_Finder</a>. Supplemental material available at FigShare: <a href="https://doi.org/10.25387/g3.8968769">https://doi.org/10.25387/g3.8968769</a>.</p></div></div><div class="section results" id="sec-10"><h2 class="">Results</h2><div id="sec-11" class="subsection"><h3>QTG-Finder: a machine-learning algorithm to prioritize causal genes</h3><p id="p-25">We developed the QTG-Finder algorithm to accelerate finding causal genes from QTL data and generated two predictive models in Arabidopsis and rice. These two species were selected for model training since they have the largest number of QTL causal genes (QTGs) that have been discovered by fine mapping and map-based cloning in plants (<a id="xref-ref-39-4" class="xref-bibr" href="#ref-39">Martin and Orgogozo 2013</a>). For model training, we selected 60 Arabidopsis and 45 rice causal genes as a positive set (<a id="xref-ref-39-5" class="xref-bibr" href="#ref-39">Martin and Orgogozo, 2013</a>, Supplementary Tables S1 and S2). The negative set was a set of genes randomly selected from the rest of the genome. To train the models, we used 28 Arabidopsis features and 27 rice features, including polymorphism features, functional categories of genes, function inference from co-function networks, gene essentiality, and paralog copy number (Supplementary Tables S3, S4 and S5). These features were generally independent from each other; most have a Pearson’s correlation coefficient &lt;0.2 (Supplementary Figure S4).</p><p id="p-26">We devised an extended cross-validation framework to optimize the models (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1A</a>). With this framework, we evaluated the training performance with AUC-ROC and optimized parameters. We used AUC-ROC for model optimization since our goal is not only to identify causal genes (true positives) in the prioritized list but also reduce the number of candidates by eliminating non-causal genes (true negatives) from the prioritized list. To find the optimal parameters, we compared the AUC-ROC of different machine-learning classifiers, modeling parameters, the ratio of positive:negative genes in the training set, and different methods to generate GO features (Supplementary Figures S2, S3, S4, and S5). Random Forest was selected as the classifier since it was less prone to over-fitting and performed better than the other classifiers tested (Supplementary Figure S1). After optimization, AUC-ROC for the Arabidopsis and rice models were 0.86 and 0.73, respectively (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1B</a>). The optimized models were also evaluated by confusion matrix (Supplementary Table S6). The true positive and true negative rates calculated from the confusion matrix indicated that the model was better at classifying non-causal genes than causal genes.</p><p id="p-27">Since the positive training set used was relatively small, we also evaluated the relationship between training performance and size of the training set. The AUC-ROC increased as a larger training set was used. Interestingly, maximum gain in the AUC-ROC was achieved with 20 causal genes for the traits represented by the training set (Supplementary Figure S6).</p></div><div id="sec-12" class="subsection"><h3>Important features for predicting causal genes</h3><p id="p-28">With the optimized models, we asked which features were important for causal gene prediction. Since Random Forest uses features and their interactions for classification (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Touw <em>et al.</em> 2013</a>), the importance of a feature cannot be measured by simple enrichment or depletion of a single feature in causal genes. Therefore, we evaluated feature importance based on the change of AUC-ROC (<strong>Δ</strong>AUC-ROC) when excluding a feature from the model (<a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Lloyd <em>et al.</em> 2015</a>). When an important feature is excluded from the model, the AUC-ROC should decrease.</p><p id="p-29">For both Arabidopsis and rice models, eight features decreased AUC-ROC when removed (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a> and Supplementary Table S7). The six most important features for Arabidopsis were paralog copy number, transporter, the number of non-synonymous SNPs normalized to protein length (normalized_nonsyn_SNP), receptor, transcription factor, and SNPs causing premature stop codon (is_stop_gained) (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2A</a>). The six most important features for rice were paralog copy number, macromolecule metabolism, network weight sum, transcription factor, transporter, and SNPs causing premature stop codon (is_stop_gained). Four out of the six most important features were consistent between Arabidopsis and rice models, which were paralog copy number, transporter, transcription factor, and SNPs causing premature stop codon.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/10/3129/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Important features of causal genes and their enrichment or depletion relative to the genome background. (A) Feature importance as indicated by the change of AUC-ROC (&#x394;AUC-ROC) when excluding each feature. The &#x394;AUC-ROC indicates the average value of all iterations. Error bars indicate standard deviation. The features with a name that starts with &#x201C;is_&#x201D; are binary variables. (B) The enrichment or depletion of the top 6 features in Arabidopsis and rice models. The enrichment/depletion were indicated by the ratio of causal genes to genome background. ns, not shown because the feature is not one of the top 6 features in that species." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1569807890" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Important features of causal genes and their enrichment or depletion relative to the genome background. (A) Feature importance as indicated by the change of AUC-ROC (&#x394;AUC-ROC) when excluding each feature. The &#x394;AUC-ROC indicates the average value of all iterations. Error bars indicate standard deviation. The features with a name that starts with &#x201C;is_&#x201D; are binary variables. (B) The enrichment or depletion of the top 6 features in Arabidopsis and rice models. The enrichment/depletion were indicated by the ratio of causal genes to genome background. ns, not shown because the feature is not one of the top 6 features in that species.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/10/3129/F2.medium.gif" width="244" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/9/10/3129/F2.medium.gif" width="244" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/10/3129/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/10/3129/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494485" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-30" class="first-child">Important features of causal genes and their enrichment or depletion relative to the genome background. (A) Feature importance as indicated by the change of AUC-ROC (ΔAUC-ROC) when excluding each feature. The ΔAUC-ROC indicates the average value of all iterations. Error bars indicate standard deviation. The features with a name that starts with “is_” are binary variables. (B) The enrichment or depletion of the top 6 features in Arabidopsis and rice models. The enrichment/depletion were indicated by the ratio of causal genes to genome background. ns, not shown because the feature is not one of the top 6 features in that species.</p><div class="sb-div caption-clear"></div></div></div><p id="p-31">For the six most important features in Arabidopsis and rice, we examined their ratio in known causal genes <em>vs.</em> randomly selected genes in the genome (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2B</a>). Compared to other genes in the genome, the causal genes in both species tended to have more paralogs, higher frequency of being a transporter or a transcription factor, and higher frequency of containing SNPs that cause premature stop codons. In addition, Arabidopsis causal genes were more likely to be a receptor and rice causal genes were more likely to be a non-hub gene.</p><p id="p-32">The rest of the features contributed less to, but did not impair much, the model performance (<strong>Δ</strong>AUC-ROC&lt; 0.02). Since there was no strong evidence that they impair prediction, we did not remove them from the models for further analysis.</p></div><div id="sec-13" class="subsection"><h3>Validating QTG-Finder by ranking an independent set of QTL genes</h3><p id="p-33">To assess the predictability of QTG-Finder models, we searched the literature for a separate set of known causal genes from the initial training set. We found eleven Arabidopsis and eighteen rice genes that are likely causal genes underlying QTL when interpreting linkage mapping with additional evidence such as functional complementation, fine mapping, joint linkage-association analysis or genetic analyses (Supplementary Table S8). These causal genes were not used for model training or cross-validation.</p><p id="p-34">To examine model performance independently, we applied the QTG-Finder models to this new set of causal genes. For each known causal gene, we ranked all the genes within its QTL region, based on the frequency of being predicted as a causal gene from 5,000 iterations. Since the number of genes in a QTL region varies, we used a gene’s rank percentile for evaluation. The rank percentile of a gene indicates the percentage of QTL genes that had higher ranks than the gene of interest.</p><p id="p-35">Based on the rank percentile of these known causal genes, we evaluated model performance at different cutoffs of rank percentile such as 5%, 10%, or 20%. We calculated the percentage of known causal genes being recalled at different cutoffs (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>). The top 20% of the ranked genes included seven Arabidopsis (∼64%) and fourteen rice (∼79%) causal genes (Supplementary Table S8). This set included 10-100 non-causal genes. With a more stringent cutoff of 5%, four Arabidopsis (∼27%) and three rice (∼26%) causal genes were prioritized. We examined the molecule types, trait categories, and features of the eight known causal genes (4 Arabidopsis and 4 rice) that were not prioritized within the top 20%, but did not observe any special trend.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/10/3129/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Model performance at different thresholds. (A) Percentage of recalled causal genes of a single QTL at different rank thresholds. Dashed lines indicate the background of random selections. (B-C) The probability of causal gene recall when analyzing multiple QTL simultaneously." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1569807890" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Model performance at different thresholds. (A) Percentage of recalled causal genes of a single QTL at different rank thresholds. Dashed lines indicate the background of random selections. (B-C) The probability of causal gene recall when analyzing multiple QTL simultaneously.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/10/3129/F3.medium.gif" width="440" height="179"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/9/10/3129/F3.medium.gif" width="440" height="179"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/10/3129/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/10/3129/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494493" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-36" class="first-child">Model performance at different thresholds. (A) Percentage of recalled causal genes of a single QTL at different rank thresholds. Dashed lines indicate the background of random selections. (B-C) The probability of causal gene recall when analyzing multiple QTL simultaneously.</p><div class="sb-div caption-clear"></div></div></div><p id="p-37">We also asked whether the different strengths of experimental evidence of the causal genes affected the model performance. Causal genes were grouped based on the type of supporting experimental evidence (Table S6). The first group included genes with weak supporting evidence such as mutational analysis. The second group contained genes supported by stronger evidence such as fine mapping, functional complementation, and joint linkage-associate analysis. The average rank percentile of the two groups was similar; 18% for the first group and 16% for the second.</p><p id="p-38">Since most linkage mapping studies identify multiple QTL, we asked what the probability is of identifying causal genes from multiple QTL simultaneously. We calculated a theoretical model performance on multiple QTL identification as the probability of identifying causal genes for at least <em>N</em> QTL when applying the model to all QTL of a trait (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3B and C</a>). For example, assuming there were five QTL (<em>N</em> = 5) of a trait identified by a linkage mapping study and each QTL contained one causal gene. For the Arabidopsis model, the probability of identifying at least one causal gene would be 99% when the top 20% genes of all QTL were tested experimentally. The probability of identifying all five causal genes would be 10% when the top 20% cutoff was used. We further compared the performance of all three cutoffs, top 20%, top 10%, and top 5%. The probability of identifying at least one out of five causal genes would be no less than 80% for all three cutoffs. However, the probability to recall at least four out of five causal genes at top 20% would be 40%, 14% (at top 10%), and 2% (at top 5%). Therefore, a less stringent cutoff, top 20%, performs much better than a more stringent cutoff if one is interested in finding most of the causal genes or causal genes of a particular QTL. However, if the goal is to identify any causal gene, then screening the top 5% of all QTL may be a more strategic approach.</p><p id="p-39">To compare our results with an existing QTL prioritization method for rice (<a id="xref-ref-2-3" class="xref-bibr" href="#ref-2">Bargsten <em>et al.</em> 2014</a>), we examined how genes in our rice validation set were prioritized in that study. Only three out of eighteen genes were prioritized as candidates when the top 9% genes in QTL regions were considered. For QTG-Finder, eight out eighteen genes were prioritized as candidates when the top 9% genes were considered.</p></div><div id="sec-14" class="subsection"><h3>Trait type preference of QTG-Finder models</h3><p id="p-40">Since the training set included genes for different types of traits at an imbalanced ratio, we asked how QTG-Finder models would work for each type of traits (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4A</a>). The independent validation in the previous section was based on causal genes related to plant development and disease resistance (Supplementary Table S8). However, this validation set was not large enough for a systematic analysis and did not have any abiotic-stress-related causal genes. Therefore, we performed a rank analysis for different trait categories using the known causal genes from the initial training set (60 for Arabidopsis and 45 for rice). For this rank analysis, each causal gene was removed from the training set once and used for a rank test. The removed causal gene and its 200 neighboring genes in the genome were used as a testing set. We applied the models to each testing set to obtain the rank for each causal gene. Then we calculated the average rank for the causal genes in the four trait categories: development, abiotic stress, biotic stress and “other”. The “other” category included traits in seed hull color, oil composition, necrosis, etc. (Supplementary Tables S1 and S2).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/10/3129/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Performance comparison across trait categories. (A) Trait categories of known causal genes from the training set. (B) The rank percentile of causal genes of different trait categories. Each causal gene and 200 neighboring genes were used as testing set only once. All other known causal genes were used for training. Each dot indicates a known causal gene. The gray dashed line indicates 20% rank percentile. The trait categories of causal genes are defined in Tables S1 and S2." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1569807890" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Performance comparison across trait categories. (A) Trait categories of known causal genes from the training set. (B) The rank percentile of causal genes of different trait categories. Each causal gene and 200 neighboring genes were used as testing set only once. All other known causal genes were used for training. Each dot indicates a known causal gene. The gray dashed line indicates 20% rank percentile. The trait categories of causal genes are defined in Tables S1 and S2.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/10/3129/F4.medium.gif" width="216" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/9/10/3129/F4.medium.gif" width="216" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/10/3129/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/10/3129/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494498" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-41" class="first-child">Performance comparison across trait categories. (A) Trait categories of known causal genes from the training set. (B) The rank percentile of causal genes of different trait categories. Each causal gene and 200 neighboring genes were used as testing set only once. All other known causal genes were used for training. Each dot indicates a known causal gene. The gray dashed line indicates 20% rank percentile. The trait categories of causal genes are defined in Tables S1 and S2.</p><div class="sb-div caption-clear"></div></div></div><p id="p-42">Model performance varied for different trait categories. Both abiotic and biotic stress traits had better performance than developmental traits (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4B</a>). This could be because the developmental trait category has more diversified traits and genes than the abiotic and biotic stress trait category. In addition, the Arabidopsis model performed slightly better than the rice model for all trait categories. This trait category analysis can guide us to determine rank cutoffs when applying models to different types of traits.</p></div></div><div class="section discussion" id="sec-15"><h2 class="">Discussion</h2><p id="p-43">Linkage mapping is a useful tool to identify the genomic regions responsible for many agriculturally and medically important traits. However, it is not straightforward to identify the genes that cause phenotypic variation of the trait from these genomic regions. The discovery of causal genes still requires time-consuming and labor-intensive fine mapping. In this study, we developed a machine-learning algorithm to reduce the number of candidates to be tested experimentally in order to accelerate the discovery of causal genes.</p><div id="sec-16" class="subsection"><h3>A machine-learning algorithm to prioritize QTL causal genes</h3><p id="p-44">Several causal variant or gene prioritization methods have been developed for human data but not many in plants (<a id="xref-ref-2-4" class="xref-bibr" href="#ref-2">Bargsten <em>et al.</em> 2014</a>; <a id="xref-ref-30-3" class="xref-bibr" href="#ref-30">Kircher <em>et al.</em> 2014</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Jagadeesh <em>et al.</em> 2016</a>; <a id="xref-ref-52-2" class="xref-bibr" href="#ref-52">Schaefer <em>et al.</em> 2018</a>). Most prioritization methods have been developed for GWAS mapping in human, an organism where linkage mapping cannot be performed. However, linkage mapping can capture rare alleles and has been broadly used to study quantitative traits of livestock, crops, and model organisms. A causal gene prioritization is especially helpful for large QTL identified by linkage mapping, which can constitute tens to thousands of genes. One method has been developed in rice to prioritize causal genes for linkage mapping (<a id="xref-ref-2-5" class="xref-bibr" href="#ref-2">Bargsten <em>et al.</em> 2014</a>). This method is based on the hypothesis that causal genes from multiple QTL of the same trait are more likely to have the same biological process GO terms, and therefore genes with overrepresented biological process GOs were prioritized as causal genes. However, this method gives no predictions for ∼15% of traits and lack an unbiased performance evaluation since the same set of causal genes was used to determine the cutoff and evaluate performance. We evaluated performance of this GO-based method with the causal genes from the validation set used in this study. The GO-based method identified fewer causal genes compared to QTG-Finder when a similar fraction of QTL was prioritized. Another method named Camoco has been developed in maize to prioritize causal genes for GWAS (<a id="xref-ref-52-3" class="xref-bibr" href="#ref-52">Schaefer <em>et al.</em> 2018</a>). Camoco prioritizes genes based on the relative strength and degree of co-expression among genes near GWAS peaks. The success of this method depends highly on the gene expression dataset being appropriate for the trait of interest. For example, an expression dataset of root tissues may work better for root-related traits than shoot-related traits. In addition, this method may not be able to capture causal genes that are transiently expressed or expressed at low levels (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Moyers 2018</a>). Since each of these approaches utilizes different sets of information, they may be used in conjunction with the QTG-Finder.</p><p id="p-45">In this study, we built a supervised learning algorithm using multiple features and validated its efficacy with an independent dataset from the literature. The models could accelerate the discovery of causal genes by ranking all the genes in a QTL region and prioritizing the top 5%, 10%, or 20% genes, which are most likely to contain the causal gene, for experimental testing. Based on an assessment using independent data in the literature, we calculated the performance when applying the models to all QTL of a trait and compared three cutoffs (top 5%, 10%, and 20%). The less stringent cutoff (top 20%) had a higher chance to find more causal genes (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3B and C</a>) but yielded more candidates that needed to be tested by experiments. The more stringent cutoff (top 5%) had a lower chance to find all causal genes but yielded a smaller set of candidates to test. The probability for the models to find at least one causal gene is high for all three cutoffs. If the goal were to find at least one causal gene for functional studies and the particular QTL regions did not matter, the 5% cutoff would be more efficient. If the goal were to discover all causal genes and understand the genetic architecture of a trait, the 20% cutoff would be better. Similarly, if a particular QTL were of interest for discovering the underlying causal gene, the 20% cutoff would be better.</p><p id="p-46">There are several conceptual and practical advantages of QTG-Finder algorithm. First, this algorithm combines multiple types of publically available data including polymorphisms, function annotations, co-function network and other genomic data, which have not been applied to prioritize causal genes from linkage mapping studies. Second, models were trained on causal genes from various traits and can be applied to several types of traditional traits, though the prioritization efficiency was not equivalent. Third, validation from the literature provides guidance on what proportion of genes to prioritize in practice rather than arbitrarily selecting a threshold. Fourth, the models treat each QTL independently and have the flexibility to prioritize a specific QTL of interest.</p><p id="p-47">Two limitations of this study are the small number of known causal genes in plants and the impurity of negative set used for model training. As a positive dataset, we used 60 Arabidopsis and 45 rice causal genes that have been verified by map-based-cloning. Even though the positive dataset are of high quality, the sample size may not be large enough to represent all categories or features of causal genes and therefore lead to ascertainment bias. The models may perform better on over-represented gene categories or features in the training set. A larger positive training set will mitigate this bias. For example, the qTARO database is a useful resource to find potential new causal genes for rice, though these genes would need to be curated further (<a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Yonemaru <em>et al.</em> 2010</a>). The negative set was composed of genes randomly selected from the rest of the genome. Though we excluded known causal genes, there could still be some uncharacterized causal genes. As a result of these limitations, 20% cutoff will still yield ∼100 candidates for large QTL, which is challenging for experimental characterization unless at least a medium-throughput phenotyping method is available. Fortunately, plant science is entering an era of high-throughput phenotyping with advances in automation, computation and sensor technology (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Fahlgren <em>et al.</em> 2015</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Araus <em>et al.</em> 2018</a>). Our study establishes an extendable framework that can be easily updated with new training datasets and features. As more causal genes are uncovered, the new data can be easily incorporated to improve the models.</p><p id="p-48">The current models included genes in the reference genomes of rice and Arabidopsis. Even though the majority of causal genes are present in the reference genomes, there are exceptions. For example, <em>SUB1A</em>, <em>SNORKEL1</em>, and <em>SNORKEL2</em> are causal genes absent in the rice reference genome Nipponbare (<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Xu <em>et al.</em> 2006</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Hattori <em>et al.</em> 2009</a>). Those genes cannot be predicted with the current models. In the future, this could be addressed by using pan-genome gene information and presence–absence variation (<a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Zhao <em>et al.</em> 2018</a>).</p></div><div id="sec-17" class="subsection"><h3>Important features for predicting QTL causal genes</h3><p id="p-49">Many causal genes were repeatedly found to cause phenotypic variation of similar traits, which is also known as genetic hotspots of phenotypic variation or gene reuse (<a id="xref-ref-39-6" class="xref-bibr" href="#ref-39">Martin and Orgogozo 2013</a>). By examining 1,008 causal alleles in animals, plants, and yeasts, Martin and Orgogozo found <em>de novo</em> mutations to occur repeatedly at certain genes or orthologous loci and causing similar phenotypic variations either among lineages or within a single lineage. The prevalence of gene reuse suggests that causal genes are likely to have some genetic and genomic characteristics that allow them to be repeatedly used for phenotypic variation. The mechanism for gene reuse is not clear but it may be influenced by factors such as the availability of standing genetic variation, mutation rate, pleiotropic constraint, and epistatic interactions of a gene (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Conte <em>et al.</em> 2012</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Conte <em>et al.</em> 2015</a>).</p><p id="p-50">While many QTL causal genes have been cloned, their features have not been systematically examined before. Instead of evaluating each feature individually, we trained Random Forest models and evaluated feature importance for all features by adopting the leave-one-out strategy. Several of the most important features were consistent between Arabidopsis and rice models: containing SNPs that cause a premature stop codon, paralog copy number, being a transporter, and being a transcription factor.</p><p id="p-51">We extracted polymorphism features from re-sequencing data of many accessions, which provide more information about the existence of standing genetic variation in the species than the polymorphism data used for linkage mapping, which typically comes from two parental lines. DNA polymorphisms such as nonsense SNPs, deleterious non-synonymous SNPs, SNPs at <em>cis</em>-regulatory elements, and SNPs at splice junctions have been used as features to classify causal and non-causal variants of human diseases (<a id="xref-ref-30-4" class="xref-bibr" href="#ref-30">Kircher <em>et al.</em> 2014</a>; <a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Jagadeesh <em>et al.</em> 2016</a>). These SNPs can directly affect the function or expression of a gene and therefore are more likely to be causal than the rest of the SNPs. Our results indicate Arabidopsis and rice causal genes were more likely to carry a SNP that causes premature stop codon (nonsense SNP) than an average gene in the genome. We also found Arabidopsis causal genes were more likely to have more non-synonymous SNPs than an average gene in the genome. Besides the high impact SNPs in coding regions, we also examined polymorphisms in non-coding regions since about 90% of human trait/disease-associated SNPs are located outside of coding regions (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Hindorff <em>et al.</em> 2009</a>). The SNPs at <em>cis</em>-regulatory elements did not show a high feature importance in our algorithm, although this might be due to limited exploration of non-coding sequences in plants. The CIS-BP database contains <em>cis</em>-elements of 44% of the transcription factors in Arabidopsis (<a id="xref-ref-61-2" class="xref-bibr" href="#ref-61">Weirauch <em>et al.</em> 2014</a>). Developing a more accurate and complete map of functional non-coding regions based on conserved noncoding sequences (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Van de Velde <em>et al.</em> 2014</a>) will potentially make non-coding polymorphism features more useful for prioritizing causal genes in the future. The SNPs linked to causal SNPs might add background noise and reduce the capability to distinguish causal genes from non-causal genes. This could be a reason why half of the polymorphism features were not significantly enriched in the causal genes (Supplemental Table S3).</p><p id="p-52">Paralogs contribute to the evolution of plant traits by providing functional divergence that gives plants the potential to adapt to complex environments (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Panchy <em>et al.</em> 2016</a>). Through evolution, genes involved in signal transduction and stress response have retained more paralogs while essential genes like DNA gyrase A have retained fewer paralogs (<a id="xref-ref-35-3" class="xref-bibr" href="#ref-35">Lloyd <em>et al.</em> 2015</a>; <a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">Panchy <em>et al.</em> 2016</a>). By acquiring new functions or sub-functions, paralogs allow plants to sense and handle different environmental conditions in a more comprehensive and adjustable way. For example, the eight paralogous heavy metal ATPases (HMAs) in Arabidopsis are all involved in heavy metal transport but have different substrate preferences, tissue expression patterns, and subcellular compartment locations (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Takahashi <em>et al.</em> 2012</a>). Three of them, <em>HMA3</em>, <em>HMA4</em>, <em>HMA5</em>, are known causal genes of QTL identified by linkage mapping. The known causal genes we analyzed have more paralog copies than other genes in the genome. This suggests that many plant causal genes are playing a role in providing more phenotypic tuning parameters to allow plants to adapt to complex environments.</p><p id="p-53">When the training set is small, there is a possibility of ascertainment bias. For example, the GO features, is_transporter and is_transcription_factor, may be considered as important features because of their enrichment in the current training set. We will have more confidence of the importance of features when more known causal genes become available.</p><p id="p-54">The important features of causal genes identified by linkage mapping may be different from those identified by GWAS. Given the difference of the two genetic approaches, linkage mapping tends to identify large-effect alleles of protein-coding regions, while GWAS tends to identify common alleles with a wider range of effect sizes at protein-coding regions or non-coding regions (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Singleton <em>et al.</em> 2010</a>). Therefore, whether the features used in this study can be applied to GWAS remains open. It would be interesting to systematically compare causal genes identified by linkage mapping and GWAS in the future.</p><p id="p-55">Overall, QTG-Finder is a novel machine-learning pipeline to prioritize causal genes for QTL identified by linkage mapping. We trained QTG-Finder models for Arabidopsis and rice based on known causal genes from each species, respectively. By utilizing information like polymorphisms, function annotations, co-function networks, and paralog copy numbers, the models can rank QTL genes to prioritize causal genes. Our independent literature validation demonstrates that the models can recall about 64% of causal genes for Arabidopsis and 79% for rice when the top 20% of ranked QTL genes were considered. The algorithm is applicable to any traditional quantitative traits but the performance was different for each trait type. Since QTG-Finder is a machine-learning based pipeline, extending the training set and adding features can easily expand and improve the models. We envision that frameworks like QTG-Finder can accelerate the discovery of novel quantitative trait genes by reducing the number of candidate genes and efforts of experimental testing.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-56">We thank Dr. John Lloyd and Dr. Shin-Han Shiu for sharing the data of rice essential gene prediction. We thank Kevin Radja for testing the source code and giving useful comments.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-57">Supplemental material available at FigShare: <a href="https://doi.org/10.25387/g3.8968769">https://doi.org/10.25387/g3.8968769</a>.</p></li><li class="fn" id="fn-2"><p id="p-58"><em>Communicating editor: D. J. de Koning</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2019-05-02"><span class="received-label">Received </span>May 2, 2019.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2019-07-23"><span class="accepted-label">Accepted </span>July 23, 2019.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2019 Lin <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3129.1" data-doi="10.1016/j.tplants.2018.02.001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Araus</span>,  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. C.</span> <span class="cit-name-surname">Kefauver</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Zaman-Allah</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. S.</span> <span class="cit-name-surname">Olsen</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. E.</span> <span class="cit-name-surname">Cairns</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Translating High-Throughput Phenotyping into Genetic Gain.</span> <abbr class="cit-jnl-abbrev">Trends Plant Sci.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">451</span>–<span class="cit-lpage">466</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.tplants.2018.02.001</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BPlant%2BSci.%26rft.volume%253D23%26rft.spage%253D451%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tplants.2018.02.001%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tplants.2018.02.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3129.2" data-doi="10.1186/s12870-014-0330-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bargsten</span>,  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. P.</span> <span class="cit-name-surname">Nap</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G. F.</span> <span class="cit-name-surname">Sanchez-Perez</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. D.</span> <span class="cit-name-surname">van Dijk</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Prioritization of candidate genes in QTL regions based on associations between traits and biological processes.</span> <abbr class="cit-jnl-abbrev">BMC Plant Biol.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">330</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12870-014-0330-3</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BPlant%2BBiol.%26rft.volume%253D14%26rft.spage%253D330%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12870-014-0330-3%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12870-014-0330-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3129.3" data-doi="10.1371/journal.pgen.1001193"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Baxter</span>,  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. N.</span> <span class="cit-name-surname">Brazelton</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Yu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y. S.</span> <span class="cit-name-surname">Huang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Lahner</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">A coastal cline in sodium accumulation in <em>Arabidopsis thaliana</em> is friven by natural variation of the sodium transporter <em>AtHKT1;1</em>.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">6</span>: <span class="cit-elocation-id">e1001193</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1001193</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DBaxter%26rft.auinit1%253DI.%26rft.volume%253D6%26rft.issue%253D11%26rft.spage%253De1001193%26rft.epage%253De1001193%26rft.atitle%253DA%2Bcoastal%2Bcline%2Bin%2Bsodium%2Baccumulation%2Bin%2BArabidopsis%2Bthaliana%2Bis%2Bdriven%2Bby%2Bnatural%2Bvariation%2Bof%2Bthe%2Bsodium%2Btransporter%2BAtHKT1%253B1.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1001193%26rft_id%253Dinfo%253Apmid%252F21085628%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1001193&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21085628&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3129.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3129.4" data-doi="10.1073/pnas.1000410107"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bentsink</span>,  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Hanson</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. J.</span> <span class="cit-name-surname">Hanhart</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Blankestijn-de Vries</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Coltrane</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Natural variation for seed dormancy in Arabidopsis is regulated by additive genetic and molecular pathways.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">107</span>: <span class="cit-fpage">4264</span>–<span class="cit-lpage">4269</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.1000410107</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1000410107%26rft_id%253Dinfo%253Apmid%252F20145108%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTA3LzkvNDI2NCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzkvMTAvMzEyOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3129.5" data-doi="10.1038/nrg2896"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bergelson</span>,  <span class="cit-name-given-names">J.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Roux</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Towards identifying genes underlying ecologically relevant traits in <em>Arabidopsis thaliana</em>.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">867</span>–<span class="cit-lpage">879</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg2896</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DBergelson%26rft.auinit1%253DJ.%26rft.volume%253D11%26rft.issue%253D12%26rft.spage%253D867%26rft.epage%253D879%26rft.atitle%253DTowards%2Bidentifying%2Bgenes%2Bunderlying%2Becologically%2Brelevant%2Btraits%2Bin%2BArabidopsis%2Bthaliana.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2896%26rft_id%253Dinfo%253Apmid%252F21085205%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2896&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21085205&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3129.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3129.6" data-doi="10.1371/journal.pgen.1000940"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brachi</span>,  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Faure</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Horton</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Flahauw</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Vazquez</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Linkage and association mapping of <em>Arabidopsis thaliana</em> flowering time in nature.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">6</span>: <span class="cit-elocation-id">e1000940</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1000940</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DBrachi%26rft.auinit1%253DB.%26rft.volume%253D6%26rft.issue%253D5%26rft.spage%253De1000940%26rft.epage%253De1000940%26rft.atitle%253DLinkage%2Band%2Bassociation%2Bmapping%2Bof%2BArabidopsis%2Bthaliana%2Bflowering%2Btime%2Bin%2Bnature.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000940%26rft_id%253Dinfo%253Apmid%252F20463887%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000940&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20463887&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3129.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3129.7" data-doi="10.1126/science.1174276"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Buckler</span>,  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. B.</span> <span class="cit-name-surname">Holland</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. J.</span> <span class="cit-name-surname">Bradbury</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. B.</span> <span class="cit-name-surname">Acharya</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. J.</span> <span class="cit-name-surname">Brown</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The Genetic Architecture of Maize Flowering Time.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">325</span>: <span class="cit-fpage">714</span>–<span class="cit-lpage">718</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.1174276</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DBuckler%26rft.auinit1%253DE.%2BS.%26rft.volume%253D325%26rft.issue%253D5941%26rft.spage%253D714%26rft.epage%253D718%26rft.atitle%253DThe%2BGenetic%2BArchitecture%2Bof%2BMaize%2BFlowering%2BTime%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1174276%26rft_id%253Dinfo%253Apmid%252F19661422%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjUvNTk0MS83MTQiO3M6NDoiYXRvbSI7czoxOToiL2dnZy85LzEwLzMxMjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3129.8" data-doi="10.1038/nrg1407"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Carlborg</span>,  <span class="cit-name-given-names">O.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. S.</span> <span class="cit-name-surname">Haley</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Epistasis: too often neglected in complex trait studies?</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">618</span>–<span class="cit-lpage">625</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg1407</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DCarlborg%26rft.auinit1%253DO.%26rft.volume%253D5%26rft.issue%253D8%26rft.spage%253D618%26rft.epage%253D625%26rft.atitle%253DEpistasis%253A%2Btoo%2Boften%2Bneglected%2Bin%2Bcomplex%2Btrait%2Bstudies%253F%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg1407%26rft_id%253Dinfo%253Apmid%252F15266344%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg1407&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15266344&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3129.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000223039200016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3129.9" data-doi="10.4161/fly.19695"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cingolani</span>,  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Platts</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L. L.</span> <span class="cit-name-surname">Wang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Coon</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Nguyen</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A program for annotating and predicting the effects of single nucleotide polymorphisms, SnpEff: SNPs in the genome of <em>Drosophila melanogaster</em> strain w1118; iso-2; iso-3.</span> <abbr class="cit-jnl-abbrev">Fly (Austin)</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">80</span>–<span class="cit-lpage">92</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.4161/fly.19695</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DFly%26rft.aulast%253DCingolani%26rft.auinit1%253DP.%26rft.volume%253D6%26rft.issue%253D2%26rft.spage%253D80%26rft.epage%253D92%26rft.atitle%253DA%2Bprogram%2Bfor%2Bannotating%2Band%2Bpredicting%2Bthe%2Beffects%2Bof%2Bsingle%2Bnucleotide%2Bpolymorphisms%252C%2BSnpEff%253A%2BSNPs%2Bin%2Bthe%2Bgenome%2Bof%2BDrosophila%2Bmelanogaster%2Bstrain%2Bw1118%253B%2Biso-2%253B%2Biso-3.%26rft_id%253Dinfo%253Adoi%252F10.4161%252Ffly.19695%26rft_id%253Dinfo%253Apmid%252F22728672%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.4161/fly.19695&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22728672&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3129.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000305965500003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3129.10" data-doi="10.1016/j.cell.2016.05.063"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">1001 Genomes Consortium</span></li></ol><cite>. <span class="cit-pub-date">2016</span> <span class="cit-article-title">1,135 Genomes Reveal the Global Pattern of Polymorphism in <em>Arabidopsis thaliana</em>.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">166</span>: <span class="cit-fpage">481</span>–<span class="cit-lpage">491</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.cell.2016.05.063</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D166%26rft.spage%253D481%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2016.05.063%26rft_id%253Dinfo%253Apmid%252F27293186%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2016.05.063&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27293186&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3129.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3129.11" data-doi="10.1534/genetics.115.182550"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Conte</span>,  <span class="cit-name-given-names">G. L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. E.</span> <span class="cit-name-surname">Arnegard</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Best</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y. F.</span> <span class="cit-name-surname">Chan</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F. C.</span> <span class="cit-name-surname">Jones</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Extent of QTL Reuse During Repeated Phenotypic Divergence of Sympatric Threespine Stickleback.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">201</span>: <span class="cit-fpage">1189</span>–<span class="cit-lpage">1200</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.115.182550</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.115.182550%26rft_id%253Dinfo%253Apmid%252F26384359%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjIwMS8zLzExODkiO3M6NDoiYXRvbSI7czoxOToiL2dnZy85LzEwLzMxMjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3129.12" data-doi="10.1098/rspb.2012.2146"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Conte</span>,  <span class="cit-name-given-names">G. L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. E.</span> <span class="cit-name-surname">Arnegard</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. L.</span> <span class="cit-name-surname">Peichel</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Schluter</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The probability of genetic parallelism and convergence in natural populations.</span> <abbr class="cit-jnl-abbrev">Proc. Biol. Sci.</abbr> <span class="cit-vol">279</span>: <span class="cit-fpage">5039</span>–<span class="cit-lpage">5047</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1098/rspb.2012.2146</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BBiol.%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frspb.2012.2146%26rft_id%253Dinfo%253Apmid%252F23075840%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1098/rspb.2012.2146&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23075840&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3129.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3129.13" data-doi="10.1007/BF00994018"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cortes</span>,  <span class="cit-name-given-names">C.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Vapnik</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Support-vector networks.</span> <abbr class="cit-jnl-abbrev">Mach. Learn.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">273</span>–<span class="cit-lpage">297</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/BF00994018</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMach.%2BLearn.%26rft.volume%253D20%26rft.spage%253D273%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00994018%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00994018&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1995RX35400003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3129.14" data-doi="10.3389/fpls.2017.00807"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Daware</span>,  <span class="cit-name-given-names">A. V.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Srivastava</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. K.</span> <span class="cit-name-surname">Singh</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. K.</span> <span class="cit-name-surname">Parida</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. K.</span> <span class="cit-name-surname">Tyagi</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Regional Association Analysis of MetaQTLs Delineates Candidate Grain Size Genes in Rice.</span> <abbr class="cit-jnl-abbrev">Front. Plant Sci.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">807</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fpls.2017.00807</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BPlant%2BSci.%26rft.volume%253D8%26rft.spage%253D807%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffpls.2017.00807%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fpls.2017.00807&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3129.15" data-doi="10.1186/s13059-014-0534-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Deo</span>,  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Musso</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Tasan</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Tang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Poon</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Prioritizing causal disease genes using unbiased genomic features.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">534</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s13059-014-0534-8</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D15%26rft.spage%253D534%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-014-0534-8%26rft_id%253Dinfo%253Apmid%252F25633252%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13059-014-0534-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25633252&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3129.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3129.16" data-doi="10.1534/genetics.107.074377"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dinka</span>,  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. A.</span> <span class="cit-name-surname">Campbell</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Demers</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. N.</span> <span class="cit-name-surname">Raizada</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Predicting the size of the progeny mapping population required to positionally clone a gene.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">176</span>: <span class="cit-fpage">2035</span>–<span class="cit-lpage">2054</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.107.074377</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.074377%26rft_id%253Dinfo%253Apmid%252F17565938%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3Ni80LzIwMzUiO3M6NDoiYXRvbSI7czoxOToiL2dnZy85LzEwLzMxMjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3129.17" data-doi="10.1016/j.pbi.2015.02.006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fahlgren</span>,  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. A.</span> <span class="cit-name-surname">Gehan</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Baxter</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Lights, camera, action: high-throughput plant phenotyping is ready for a close-up.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Plant Biol.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">93</span>–<span class="cit-lpage">99</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.pbi.2015.02.006</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BOpin.%2BPlant%2BBiol.%26rft.volume%253D24%26rft.spage%253D93%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.pbi.2015.02.006%26rft_id%253Dinfo%253Apmid%252F25733069%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.pbi.2015.02.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25733069&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3129.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3129.18"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">The Food and Agriculture Organization</span>, <span class="cit-pub-date">2009</span> How to feed the world in 2050. Available at: <a href="http://www.fao.org/fileadmin/templates/wsfs/docs/expert_paper/How_to_Feed_the_World_in_2050.pdf">http://www.fao.org/fileadmin/templates/wsfs/docs/expert_paper/How_to_Feed_the_World_in_2050.pdf</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3129.19" data-doi="10.1038/s41467-017-00141-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gelfman</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Q.</span> <span class="cit-name-surname">Wang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. M.</span> <span class="cit-name-surname">McSweeney</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Ren</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">La Carpia</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Annotating pathogenic non-coding variants in genic regions.</span> <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">236</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/s41467-017-00141-2</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun.%26rft.volume%253D8%26rft.spage%253D236%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41467-017-00141-2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41467-017-00141-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3129.20" data-doi="10.1093/nar/gkn176"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gotz</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">Garcia-Gomez</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Terol</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. D.</span> <span class="cit-name-surname">Williams</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. H.</span> <span class="cit-name-surname">Nagaraj</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">High-throughput functional annotation and data mining with the Blast2GO suite.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">3420</span>–<span class="cit-lpage">3435</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkn176</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkn176%26rft_id%253Dinfo%253Apmid%252F18445632%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkn176&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18445632&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3129.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000257183200025&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3129.21" data-doi="10.1093/bioinformatics/btr064"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Grant</span>,  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. L.</span> <span class="cit-name-surname">Bailey</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">W. S.</span> <span class="cit-name-surname">Noble</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">FIMO: scanning for occurrences of a given motif.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">1017</span>–<span class="cit-lpage">1018</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btr064</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtr064%26rft_id%253Dinfo%253Apmid%252F21330290%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btr064&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21330290&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3129.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000289162000022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3129.22" data-doi="10.1038/nature08258"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hattori</span>,  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Nagai</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Furukawa</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.-J.</span> <span class="cit-name-surname">Song</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Kawano</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The ethylene response factors <em>SNORKEL1</em> and <em>SNORKEL2</em> allow rice to adapt to deep water.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">460</span>: <span class="cit-fpage">1026</span>–<span class="cit-lpage">1030</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature08258</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DHattori%26rft.auinit1%253DY.%26rft.volume%253D460%26rft.issue%253D7258%26rft.spage%253D1026%26rft.epage%253D1030%26rft.atitle%253DThe%2Bethylene%2Bresponse%2Bfactors%2BSNORKEL1%2Band%2BSNORKEL2%2Ballow%2Brice%2Bto%2Badapt%2Bto%2Bdeep%2Bwater.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature08258%26rft_id%253Dinfo%253Apmid%252F19693083%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature08258&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19693083&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3129.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000269085500037&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-thesis" id="cit-9.10.3129.23"><div class="cit-metadata unstructured">He, H., 2014 Environmental Regulation of Seed Performance. Dissertation. Wageningen University, Wageningen University.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3129.24" data-doi="10.1073/pnas.0903103106"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hindorff</span>,  <span class="cit-name-given-names">L. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Sethupathy</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H. A.</span> <span class="cit-name-surname">Junkins</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. M.</span> <span class="cit-name-surname">Ramos</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. P.</span> <span class="cit-name-surname">Mehta</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Potential etiologic and functional implications of genome-wide association loci for human diseases and traits.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">106</span>: <span class="cit-fpage">9362</span>–<span class="cit-lpage">9367</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.0903103106</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0903103106%26rft_id%253Dinfo%253Apmid%252F19474294%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTA2LzIzLzkzNjIiO3M6NDoiYXRvbSI7czoxOToiL2dnZy85LzEwLzMxMjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3129.25" data-doi="10.1093/bioinformatics/btv240"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hormozdiari</span>,  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Kichaev</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W. Y.</span> <span class="cit-name-surname">Yang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Pasaniuc</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Eskin</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Identification of causal genes for complex traits.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">i206</span>–<span class="cit-lpage">i213</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btv240</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtv240%26rft_id%253Dinfo%253Apmid%252F26072484%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btv240&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26072484&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3129.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3129.26" data-doi="10.1038/ng.3810"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>,  <span class="cit-name-given-names">Y. F.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Gulko</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Siepel</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Fast, scalable prediction of deleterious noncoding variants from functional and population genomic data.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">618</span>–<span class="cit-lpage">624</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.3810</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D49%26rft.spage%253D618%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.3810%26rft_id%253Dinfo%253Apmid%252F28288115%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.3810&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28288115&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3129.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3129.27" data-doi="10.1038/ng.3703"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jagadeesh</span>,  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. M.</span> <span class="cit-name-surname">Wenger</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. J.</span> <span class="cit-name-surname">Berger</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Guturu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. D.</span> <span class="cit-name-surname">Stenson</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">M-CAP eliminates a majority of variants of uncertain significance in clinical exomes at high sensitivity.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">1581</span>–<span class="cit-lpage">1586</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.3703</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D48%26rft.spage%253D1581%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.3703%26rft_id%253Dinfo%253Apmid%252F27776117%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.3703&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27776117&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3129.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3129.28" data-doi="10.1186/1471-2105-8-170"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>,  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. L.</span> <span class="cit-name-surname">Brown</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">U.</span> <span class="cit-name-surname">Baumann</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Estimating the annotation error rate of curated GO database sequence annotations.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">170</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2105-8-170</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DJones%26rft.auinit1%253DC.%2BE.%26rft.volume%253D8%26rft.spage%253D170%26rft.epage%253D170%26rft.atitle%253DEstimating%2Bthe%2Bannotation%2Berror%2Brate%2Bof%2Bcurated%2BGO%2Bdatabase%2Bsequence%2Bannotations.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-8-170%26rft_id%253Dinfo%253Apmid%252F17519041%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-8-170&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17519041&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3129.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3129.29" data-doi="10.1093/bioinformatics/btu031"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>,  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Binns</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H. Y.</span> <span class="cit-name-surname">Chang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Fraser</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Li</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">InterProScan 5: genome-scale protein function classification.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">1236</span>–<span class="cit-lpage">1240</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btu031</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu031%26rft_id%253Dinfo%253Apmid%252F24451626%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btu031&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24451626&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3129.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000336095100007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3129.30" data-doi="10.1038/ng.2892"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kircher</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. M.</span> <span class="cit-name-surname">Witten</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Jain</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. J.</span> <span class="cit-name-surname">O’Roak</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G. M.</span> <span class="cit-name-surname">Cooper</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A general framework for estimating the relative pathogenicity of human genetic variants.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">310</span>–<span class="cit-lpage">315</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.2892</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D46%26rft.spage%253D310%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2892%26rft_id%253Dinfo%253Apmid%252F24487276%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2892&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24487276&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3129.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3129.31" data-doi="10.1038/nbt.1603"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>,  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Ambaru</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Thakkar</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. M.</span> <span class="cit-name-surname">Marcotte</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. Y.</span> <span class="cit-name-surname">Rhee</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Rational association of genes with traits using a genome-scale gene network for <em>Arabidopsis thaliana</em>.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">149</span>–<span class="cit-lpage">156</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nbt.1603</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DLee%26rft.auinit1%253DI.%26rft.volume%253D28%26rft.issue%253D2%26rft.spage%253D149%26rft.epage%253D156%26rft.atitle%253DRational%2Bassociation%2Bof%2Bgenes%2Bwith%2Btraits%2Busing%2Ba%2Bgenome-scale%2Bgene%2Bnetwork%2Bfor%2BArabidopsis%2Bthaliana.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.1603%26rft_id%253Dinfo%253Apmid%252F20118918%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.1603&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20118918&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3129.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000274317200023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3129.32" data-doi="10.1073/pnas.1110384108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>,  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y. S.</span> <span class="cit-name-surname">Seo</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Coltrane</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Hwang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Oh</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genetic dissection of the biotic stress response using a genome-scale gene network for rice.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">18548</span>–<span class="cit-lpage">18553</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.1110384108</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1110384108%26rft_id%253Dinfo%253Apmid%252F22042862%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA4LzQ1LzE4NTQ4IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMC8zMTI5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3129.33" data-doi="10.1111/j.1365-294X.2012.05678.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Leinonen</span>,  <span class="cit-name-given-names">P. H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. L.</span> <span class="cit-name-surname">Remington</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Leppala</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">O.</span> <span class="cit-name-surname">Savolainen</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genetic basis of local adaptation and flowering time variation in <em>Arabidopsis lyrata</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">709</span>–<span class="cit-lpage">723</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1365-294X.2012.05678.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D22%26rft.spage%253D709%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2012.05678.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2012.05678.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000313726300013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3129.34" data-doi="10.1007/s00438-005-0032-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>,  <span class="cit-name-given-names">X. Q.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Wang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Chen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Lin</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">Q. H.</span> <span class="cit-name-surname">Pan</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Genetic and physical mapping of <em>Pi36(t)</em>, a novel rice blast resistance gene located on rice chromosome 8.</span> <abbr class="cit-jnl-abbrev">Mol. Genet. Genomics</abbr> <span class="cit-vol">274</span>: <span class="cit-fpage">394</span>–<span class="cit-lpage">401</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00438-005-0032-5</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bgenetics%2Band%2Bgenomics%2B%253A%2B%2BMGG%26rft.stitle%253DMol%2BGenet%2BGenomics%26rft.aulast%253DLiu%26rft.auinit1%253DX.%2BQ.%26rft.volume%253D274%26rft.issue%253D4%26rft.spage%253D394%26rft.epage%253D401%26rft.atitle%253DGenetic%2Band%2Bphysical%2Bmapping%2Bof%2BPi36%2528t%2529%252C%2Ba%2Bnovel%2Brice%2Bblast%2Bresistance%2Bgene%2Blocated%2Bon%2Brice%2Bchromosome%2B8.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00438-005-0032-5%26rft_id%253Dinfo%253Apmid%252F16151856%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00438-005-0032-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16151856&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3129.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000233380500008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3129.35" data-doi="10.1105/tpc.15.00051"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lloyd</span>,  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. E.</span> <span class="cit-name-surname">Seddon</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G. D.</span> <span class="cit-name-surname">Moghe</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. C.</span> <span class="cit-name-surname">Simenc</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.-H.</span> <span class="cit-name-surname">Shiu</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Characteristics of Plant Essential Genes Allow for within- and between-Species Prediction of Lethal Mutant Phenotypes.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">2133</span>–<span class="cit-lpage">2147</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1105/tpc.15.00051</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.15.00051%26rft_id%253Dinfo%253Apmid%252F26286535%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjk6IjI3LzgvMjEzMyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzkvMTAvMzEyOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3129.36" data-doi="10.1146/annurev.genet.35.102401.090633"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mackay</span>,  <span class="cit-name-given-names">T. F. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">The genetic architecture of quantitative traits.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Genet.</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">303</span>–<span class="cit-lpage">339</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1146/annurev.genet.35.102401.090633</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bgenetics%26rft.stitle%253DAnnu%2BRev%2BGenet%26rft.aulast%253DMackay%26rft.auinit1%253DT.%2BF.%2BC.%26rft.volume%253D35%26rft.issue%253D1%26rft.spage%253D303%26rft.epage%253D339%26rft.atitle%253DTHE%2BGENETIC%2BARCHITECTURE%2BOF%2BQUANTITATIVE%2BTRAITS%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.genet.35.102401.090633%26rft_id%253Dinfo%253Apmid%252F11700286%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.genet.35.102401.090633&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11700286&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3129.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000172991500012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3129.37" data-doi="10.1038/nrg3627"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mackay</span>,  <span class="cit-name-given-names">T. F. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Epistasis and Quantitative Traits: Using Model Organisms to Study Gene-Gene Interactions.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">22</span>–<span class="cit-lpage">33</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg3627</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BGenet.%26rft.volume%253D15%26rft.spage%253D22%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3627%26rft_id%253Dinfo%253Apmid%252F24296533%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3627&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24296533&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3129.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3129.38" data-doi="10.1093/nar/gkw1135"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mansueto</span>,  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. R.</span> <span class="cit-name-surname">Fuentes</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F. N.</span> <span class="cit-name-surname">Borja</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Detras</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">Abriol-Santos</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Rice SNP-seek database update: new SNPs, indels, and queries.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">D1075</span>–<span class="cit-lpage">D1081</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkw1135</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkw1135%26rft_id%253Dinfo%253Apmid%252F27899667%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkw1135&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27899667&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3129.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3129.39" data-doi="10.1111/evo.12081"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Martin</span>,  <span class="cit-name-given-names">A.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Orgogozo</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The Loci of repeated evolution: a catalog of genetic hotspots of phenotypic variation.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">67</span>: <span class="cit-fpage">1235</span>–<span class="cit-lpage">1250</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.volume%253D67%26rft.spage%253D1235%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fevo.12081%26rft_id%253Dinfo%253Apmid%252F23617905%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/evo.12081&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23617905&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3129.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000318234900001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3129.40" data-doi="10.1186/1471-2105-12-389"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mordelet</span>,  <span class="cit-name-given-names">F.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. P.</span> <span class="cit-name-surname">Vert</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">ProDiGe: Prioritization Of Disease Genes with multitask machine learning from positive and unlabeled examples.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">389</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2105-12-389</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DMordelet%26rft.auinit1%253DF.%26rft.volume%253D12%26rft.spage%253D389%26rft.epage%253D389%26rft.atitle%253DProDiGe%253A%2BPrioritization%2BOf%2BDisease%2BGenes%2Bwith%2Bmultitask%2Bmachine%2Blearning%2Bfrom%2Bpositive%2Band%2Bunlabeled%2Bexamples.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-12-389%26rft_id%253Dinfo%253Apmid%252F21977986%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-12-389&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21977986&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3129.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3129.41" data-doi="10.1073/pnas.1404978111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Motte</span>,  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Vercauteren</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Depuydt</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Landschoot</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Geelen</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Combining linkage and association mapping identifies <em>RECEPTOR-LIKE PROTEIN KINASE1</em> as an essential Arabidopsis shoot regeneration gene.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">111</span>: <span class="cit-fpage">8305</span>–<span class="cit-lpage">8310</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.1404978111</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1404978111%26rft_id%253Dinfo%253Apmid%252F24850864%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTExLzIyLzgzMDUiO3M6NDoiYXRvbSI7czoxOToiL2dnZy85LzEwLzMxMjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3129.42" data-doi="10.1105/tpc.18.00908"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Moyers</span>,  <span class="cit-name-given-names">B. T.</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Camoco: A Net for the Sea of Candidate Genes.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">2889</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1105/tpc.18.00908</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.18.00908%26rft_id%253Dinfo%253Apmid%252F30509899%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjEwOiIzMC8xMi8yODg5IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMC8zMTI5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3129.43" data-doi="10.1093/nar/gkg509"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ng</span>,  <span class="cit-name-given-names">P. C.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Henikoff</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">SIFT: predicting amino acid changes that affect protein function.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">3812</span>–<span class="cit-lpage">3814</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkg509</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DNg%26rft.auinit1%253DP.%2BC.%26rft.volume%253D31%26rft.issue%253D13%26rft.spage%253D3812%26rft.epage%253D3814%26rft.atitle%253DSIFT%253A%2Bpredicting%2Bamino%2Bacid%2Bchanges%2Bthat%2Baffect%2Bprotein%2Bfunction%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkg509%26rft_id%253Dinfo%253Apmid%252F12824425%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkg509&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12824425&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3129.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000183832900117&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3129.44"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nuzhdin</span>,  <span class="cit-name-given-names">S. V.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. L.</span> <span class="cit-name-surname">Dilda</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. F.</span> <span class="cit-name-surname">Mackay</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">The genetic architecture of selection response. Inferences from fine-scale mapping of bristle number quantitative trait loci in <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">153</span>: <span class="cit-fpage">1317</span>–<span class="cit-lpage">1331</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DNuzhdin%26rft.auinit1%253DS.%2BV.%26rft.volume%253D153%26rft.issue%253D3%26rft.spage%253D1317%26rft.epage%253D1331%26rft.atitle%253DThe%2BGenetic%2BArchitecture%2Bof%2BSelection%2BResponse%253A%2BInferences%2BFrom%2BFine-Scale%2BMapping%2Bof%2BBristle%2BNumber%2BQuantitative%2BTrait%2BLoci%2Bin%2BDrosophila%2Bmelanogaster%26rft_id%253Dinfo%253Apmid%252F10545462%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1My8zLzEzMTciO3M6NDoiYXRvbSI7czoxOToiL2dnZy85LzEwLzMxMjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3129.45"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Otto</span>,  <span class="cit-name-given-names">S. P.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. D.</span> <span class="cit-name-surname">Jones</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Detecting the undetected: Estimating the total number of loci underlying a quantitative trait.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">156</span>: <span class="cit-fpage">2093</span>–<span class="cit-lpage">2107</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DOtto%26rft.auinit1%253DS.%2BP.%26rft.volume%253D156%26rft.issue%253D4%26rft.spage%253D2093%26rft.epage%253D2107%26rft.atitle%253DDetecting%2Bthe%2BUndetected%253A%2BEstimating%2Bthe%2BTotal%2BNumber%2Bof%2BLoci%2BUnderlying%2Ba%2BQuantitative%2BTrait%26rft_id%253Dinfo%253Apmid%252F11102398%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1Ni80LzIwOTMiO3M6NDoiYXRvbSI7czoxOToiL2dnZy85LzEwLzMxMjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3129.46" data-doi="10.1104/pp.16.00523"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Panchy</span>,  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Lehti-Shiu</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. H.</span> <span class="cit-name-surname">Shiu</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Evolution of Gene Duplication in Plants.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">171</span>: <span class="cit-fpage">2294</span>–<span class="cit-lpage">2316</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.16.00523%26rft_id%253Dinfo%253Apmid%252F27288366%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czoxMDoiMTcxLzQvMjI5NCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzkvMTAvMzEyOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3129.47" data-doi="10.1016/j.cpc.2010.04.018"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pedregosa</span>,  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Varoquaux</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Gramfort</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Michel</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Thirion</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Scikit-learn: Machine Learning in Python.</span> <abbr class="cit-jnl-abbrev">J. Mach. Learn. Res.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">2825</span>–<span class="cit-lpage">2830</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BMach.%2BLearn.%2BRes.%26rft.volume%253D12%26rft.spage%253D2825%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cpc.2010.04.018%26rft_id%253Dinfo%253Apmid%252F23755062%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cpc.2010.04.018&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23755062&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3129.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3129.48" data-doi="10.1038/ng895"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Perez-Iratxeta</span>,  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Bork</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. A.</span> <span class="cit-name-surname">Andrade</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Association of genes to genetically inherited diseases using data mining.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">316</span>–<span class="cit-lpage">319</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng895</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DPerez-Iratxeta%26rft.auinit1%253DC.%26rft.volume%253D31%26rft.issue%253D3%26rft.spage%253D316%26rft.epage%253D319%26rft.atitle%253DAssociation%2Bof%2Bgenes%2Bto%2Bgenetically%2Binherited%2Bdiseases%2Busing%2Bdata%2Bmining.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng895%26rft_id%253Dinfo%253Apmid%252F12006977%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng895&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12006977&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3129.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000176494100023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3129.49" data-doi="10.1101/gr.097857.109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pollard</span>,  <span class="cit-name-given-names">K. S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. J.</span> <span class="cit-name-surname">Hubisz</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. R.</span> <span class="cit-name-surname">Rosenbloom</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Siepel</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Detection of nonneutral substitution rates on mammalian phylogenies.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">110</span>–<span class="cit-lpage">121</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.097857.109</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.097857.109%26rft_id%253Dinfo%253Apmid%252F19858363%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjIwLzEvMTEwIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMC8zMTI5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3129.50" data-doi="10.1038/nrg2813"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Price</span>,  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N. A.</span> <span class="cit-name-surname">Zaitlen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Reich</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Patterson</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">New approaches to population stratification in genome-wide association studies.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">459</span>–<span class="cit-lpage">463</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg2813</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DPrice%26rft.auinit1%253DA.%2BL.%26rft.volume%253D11%26rft.issue%253D7%26rft.spage%253D459%26rft.epage%253D463%26rft.atitle%253DNew%2Bapproaches%2Bto%2Bpopulation%2Bstratification%2Bin%2Bgenome-wide%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2813%26rft_id%253Dinfo%253Apmid%252F20548291%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2813&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20548291&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3129.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000278998500008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3129.51" data-doi="10.1038/nmeth.2832"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ritchie</span>,  <span class="cit-name-given-names">G. R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Dunham</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Zeggini</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Flicek</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Functional annotation of noncoding sequence variants.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">294</span>–<span class="cit-lpage">296</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nmeth.2832</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BMethods%26rft.volume%253D11%26rft.spage%253D294%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.2832%26rft_id%253Dinfo%253Apmid%252F24487584%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.2832&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24487584&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3129.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000332086100022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3129.52" data-doi="10.1105/tpc.18.00299"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schaefer</span>,  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">Michno</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Jeffers</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">O.</span> <span class="cit-name-surname">Hoekenga</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Dilkes</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Integrating coexpression networks with GWAS to prioritize causal genes in maize.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">2922</span>–<span class="cit-lpage">2942</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1105/tpc.18.00299</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.18.00299%26rft_id%253Dinfo%253Apmid%252F30413654%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjEwOiIzMC8xMi8yOTIyIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMC8zMTI5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3129.53" data-doi="10.1104/pp.16.01942"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schläpfer</span>,  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Zhang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Wang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Kim</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Banf</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Genome-Wide Prediction of Metabolic Enzymes, Pathways, and Gene Clusters in Plants [erratum in Plant Physiol. 176: 2583].</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">173</span>: <span class="cit-fpage">2041</span>–<span class="cit-lpage">2059</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1104/pp.16.01942</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.16.01942%26rft_id%253Dinfo%253Apmid%252F28228535%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czoxMDoiMTczLzQvMjA0MSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzkvMTAvMzEyOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3129.54" data-doi="10.1016/j.tig.2010.07.004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Singleton</span>,  <span class="cit-name-given-names">A. B.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Hardy</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. J.</span> <span class="cit-name-surname">Traynor</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Houlden</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Towards a complete resolution of the genetic architecture of disease.</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">438</span>–<span class="cit-lpage">442</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.tig.2010.07.004</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DSingleton%26rft.auinit1%253DA.%2BB.%26rft.volume%253D26%26rft.issue%253D10%26rft.spage%253D438%26rft.epage%253D442%26rft.atitle%253DTowards%2Ba%2Bcomplete%2Bresolution%2Bof%2Bthe%2Bgenetic%2Barchitecture%2Bof%2Bdisease.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2010.07.004%26rft_id%253Dinfo%253Apmid%252F20813421%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tig.2010.07.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20813421&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3129.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000282403200003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3129.55" data-doi="10.4161/psb.22454"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Takahashi</span>,  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Bashir</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Ishimaru</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N. K.</span> <span class="cit-name-surname">Nishizawa</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Nakanishi</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The role of heavy-metal ATPases, HMAs, in zinc and cadmium transport in rice.</span> <abbr class="cit-jnl-abbrev">Plant Signal. Behav.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">1605</span>–<span class="cit-lpage">1607</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.4161/psb.22454</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BSignal.%2BBehav.%26rft.volume%253D7%26rft.spage%253D1605%26rft_id%253Dinfo%253Adoi%252F10.4161%252Fpsb.22454%26rft_id%253Dinfo%253Apmid%252F23072989%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.4161/psb.22454&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23072989&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3129.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3129.56" data-doi="10.1109/34.709601"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tin Kam</span>,  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">The random subspace method for constructing decision forests.</span> <abbr class="cit-jnl-abbrev">IEEE Trans. Pattern Anal. Mach. Intell.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">832</span>–<span class="cit-lpage">844</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1109/34.709601</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DIEEE%2BTrans.%2BPattern%2BAnal.%2BMach.%2BIntell.%26rft.volume%253D20%26rft.spage%253D832%26rft_id%253Dinfo%253Adoi%252F10.1109%252F34.709601%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1109/34.709601&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3129.57" data-doi="10.1093/bib/bbs034"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Touw</span>,  <span class="cit-name-given-names">W. G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. R.</span> <span class="cit-name-surname">Bayjanov</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Overmars</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Backus</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Boekhorst</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Data mining in the Life Sciences with Random Forest: a walk in the park or lost in the jungle?</span> <abbr class="cit-jnl-abbrev">Brief. Bioinform.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">315</span>–<span class="cit-lpage">326</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bib/bbs034</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBrief.%2BBioinform.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbib%252Fbbs034%26rft_id%253Dinfo%253Apmid%252F22786785%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bib/bbs034&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22786785&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3129.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3129.58" data-doi="10.1007/s001220050654"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tuinstra</span>,  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Ejeta</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. B.</span> <span class="cit-name-surname">Goldsbrough</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Heterogeneous inbred family (HIF) analysis: a method for developing near-isogenic lines that differ at quantitative trait loci.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">95</span>: <span class="cit-fpage">1005</span>–<span class="cit-lpage">1011</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s001220050654</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D95%26rft.spage%253D1005%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs001220050654%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s001220050654&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1997YF71500037&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3129.59" data-doi="10.1105/tpc.114.127001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Van de Velde</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. S.</span> <span class="cit-name-surname">Heyndrickx</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Vandepoele</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Inference of Transcriptional Networks in Arabidopsis through Conserved Noncoding Sequence Analysis.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">2729</span>–<span class="cit-lpage">2745</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1105/tpc.114.127001</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.114.127001%26rft_id%253Dinfo%253Apmid%252F24989046%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjk6IjI2LzcvMjcyOSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzkvMTAvMzEyOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3129.60" data-doi="10.1104/pp.104.900157"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Weigel</span>,  <span class="cit-name-given-names">D.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Nordborg</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Natural variation in Arabidopsis. How do we find the causal genes?</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">138</span>: <span class="cit-fpage">567</span>–<span class="cit-lpage">568</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1104/pp.104.900157</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiology%26rft.stitle%253DPlant%2BPhysiol.%26rft.aulast%253DWeigel%26rft.auinit1%253DD.%26rft.volume%253D138%26rft.issue%253D2%26rft.spage%253D567%26rft.epage%253D568%26rft.atitle%253DNatural%2BVariation%2Bin%2BArabidopsis.%2BHow%2BDo%2BWe%2BFind%2Bthe%2BCausal%2BGenes%253F%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.104.900157%26rft_id%253Dinfo%253Apmid%252F15955918%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czo5OiIxMzgvMi81NjciO3M6NDoiYXRvbSI7czoxOToiL2dnZy85LzEwLzMxMjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3129.61" data-doi="10.1016/j.cell.2014.08.009"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Weirauch</span>,  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Yang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Albu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. G.</span> <span class="cit-name-surname">Cote</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Montenegro-Montero</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Determination and Inference of Eukaryotic Transcription Factor Sequence Specificity.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">158</span>: <span class="cit-fpage">1431</span>–<span class="cit-lpage">1443</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.cell.2014.08.009</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D158%26rft.spage%253D1431%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2014.08.009%26rft_id%253Dinfo%253Apmid%252F25215497%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2014.08.009&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25215497&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3129.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000343094200022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3129.62" data-doi="10.1016/j.tig.2015.12.004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wellenreuther</span>,  <span class="cit-name-given-names">M.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Hansson</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Detecting Polygenic Evolution: Problems, Pitfalls, and Promises.</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">155</span>–<span class="cit-lpage">164</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.tig.2015.12.004</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BGenet.%26rft.volume%253D32%26rft.spage%253D155%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2015.12.004%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tig.2015.12.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3129.63" data-doi="10.1038/nature04920"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>,  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Xu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Fukao</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Canlas</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Maghirang-Rodriguez</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title"><em>Sub1A</em> is an ethylene-response-factor-like gene that confers submergence tolerance to rice.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">442</span>: <span class="cit-fpage">705</span>–<span class="cit-lpage">708</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature04920</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DXu%26rft.auinit1%253DK.%26rft.volume%253D442%26rft.issue%253D7103%26rft.spage%253D705%26rft.epage%253D708%26rft.atitle%253DSub1A%2Bis%2Ban%2Bethylene-response-factor-like%2Bgene%2Bthat%2Bconfers%2Bsubmergence%2Btolerance%2Bto%2Brice.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature04920%26rft_id%253Dinfo%253Apmid%252F16900200%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature04920&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16900200&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3129.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000239630200050&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3129.64" data-doi="10.1038/hdy.1992.131"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>,  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Theoretical basis of the Beavis effect.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">165</span>: <span class="cit-fpage">2259</span>–<span class="cit-lpage">2268</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DXu%26rft.auinit1%253DS.%26rft.volume%253D165%26rft.issue%253D4%26rft.spage%253D2259%26rft.epage%253D2268%26rft.atitle%253DTheoretical%2BBasis%2Bof%2Bthe%2BBeavis%2BEffect%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.1992.131%26rft_id%253Dinfo%253Apmid%252F14704201%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2NS80LzIyNTkiO3M6NDoiYXRvbSI7czoxOToiL2dnZy85LzEwLzMxMjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3129.65" data-doi="10.1007/s00122-004-1844-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>,  <span class="cit-name-given-names">H. Y.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. Q.</span> <span class="cit-name-surname">You</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z. F.</span> <span class="cit-name-surname">Yang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Zhang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. F.</span> <span class="cit-name-surname">He</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">High-resolution genetic mapping at the Bph15 locus for brown planthopper resistance in rice (<em>Oryza sativa</em> L.).</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">110</span>: <span class="cit-fpage">182</span>–<span class="cit-lpage">191</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-004-1844-0</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DYang%26rft.auinit1%253DH.%26rft.volume%253D110%26rft.issue%253D1%26rft.spage%253D182%26rft.epage%253D191%26rft.atitle%253DHigh-resolution%2Bgenetic%2Bmapping%2Bat%2Bthe%2BBph15%2Blocus%2Bfor%2Bbrown%2Bplanthopper%2Bresistance%2Bin%2Brice%2B%2528Oryza%2Bsativa%2BL.%2529.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-004-1844-0%26rft_id%253Dinfo%253Apmid%252F15549231%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-004-1844-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15549231&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3129.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000226629900019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3129.66" data-doi="10.1111/pbr.12500"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yin</span>,  <span class="cit-name-given-names">Z. G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H. D.</span> <span class="cit-name-surname">Qi</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Q. S.</span> <span class="cit-name-surname">Chen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z. G.</span> <span class="cit-name-surname">Zhang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H. W.</span> <span class="cit-name-surname">Jiang</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Soybean plant height QTL mapping and meta-analysis for mining candidate genes.</span> <abbr class="cit-jnl-abbrev">Plant Breed.</abbr> <span class="cit-vol">136</span>: <span class="cit-fpage">688</span>–<span class="cit-lpage">698</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/pbr.12500</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BBreed.%26rft.volume%253D136%26rft.spage%253D688%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fpbr.12500%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/pbr.12500&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3129.67" data-doi="10.1007/s12284-010-9041-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yonemaru</span>,  <span class="cit-name-given-names">J.-i.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Yamamoto</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Fukuoka</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Uga</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Hori</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Q-TARO: QTL Annotation Rice Online Database.</span> <abbr class="cit-jnl-abbrev">Rice (N. Y.)</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">194</span>–<span class="cit-lpage">203</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s12284-010-9041-z</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRice%2B%2528N.%2BY.%2529%26rft.volume%253D3%26rft.spage%253D194%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs12284-010-9041-z%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s12284-010-9041-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-confproc" id="cit-9.10.3129.68"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> The Optimality of Naive Bayes. <em>Proc. FLAIRS</em>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3129.69" data-doi="10.1038/s41588-018-0041-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>,  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Q.</span> <span class="cit-name-surname">Feng</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Lu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Li</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Wang</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Pan-genome analysis highlights the extent of genomic variation in cultivated and wild rice.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">50</span>: <span class="cit-fpage">278</span>–<span class="cit-lpage">284</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/s41588-018-0041-z</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D50%26rft.spage%253D278%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41588-018-0041-z%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41588-018-0041-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/9/10/3129.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/10/3119" title="Transformation with Artificial Chromosomes in Oxytricha trifallax and Their Applications" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/10/3139" title="Genome-Wide Association and Gene Co-expression Network Analyses Reveal Complex Genetics of Resistance to Goss’s Wilt of Maize" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/9/10"><span>Volume 9 Issue 10, October 2019</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/9/10.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/9/10.cover-source.jpg?itok=yA8kifEd" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 9 (10)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/10/3129.figures-only" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Ue9ns3Q4QZb7vtDcfsBKOXjhU74juTlwAhOJE8Q2-Hk" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/9/10/3129" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/9/10/3129" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/9/10/3129" class="active" data-icon-position="" data-hide-link-title="0">QTG-Finder: A Machine-Learning Based Algorithm To Prioritize Causal Genes of Quantitative Trait Loci in Arabidopsis and Rice</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/494281" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-MitLxlK8jJwZNjz5tNQeb2hPwUmveQJJ3wphNLIn2IE" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/10/3129.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/9/10/3129" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-ygKi1savmGCRty54BV2oA2BLBhgXqc0RgikquQrtrLk" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/9/10/3129" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/10/3129.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494281" id="node494281--2" data-pisa="ggg;9/10/3129" data-pisa-master="ggg;g3.119.400319" data-apath="/ggg/9/10/3129.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">QTG-Finder: A Machine-Learning Based Algorithm To Prioritize Causal Genes of Quantitative Trait Loci in Arabidopsis and Rice</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-3742-5291" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Fan Lin</span>, <span class="highwire-citation-author" data-delta="1">Jue Fan</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-7572-4762" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Seung Y. Rhee</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3129-3138;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400319 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494281--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Fan Lin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Plant Biology, Carnegie Institution for Science, Stanford, California 94305</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Fan%2BLin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lin%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AFan%2BLin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-3742-5291" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Fan Lin</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jue Fan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Plant Biology, Carnegie Institution for Science, Stanford, California 94305</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jue%2BFan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fan%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJue%2BFan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Seung Y. Rhee </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Plant Biology, Carnegie Institution for Science, Stanford, California 94305</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Seung%2BY.%2BRhee%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rhee%20SY&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASeung%2BY.%2BRhee%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-7572-4762" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Seung Y. Rhee</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:srhee@carnegiescience.edu" class="" data-icon-position="" data-hide-link-title="0">srhee@carnegiescience.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/9/10/3129" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/9/10/3129&amp;title=QTG-Finder%3A%20A%20Machine-Learning%20Based%20Algorithm%20To%20Prioritize%20Causal%20Genes%20of%20Quantitative%20Trait%20Loci%20in%20Arabidopsis%20and%20Rice" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/9/10/3129&amp;title=QTG-Finder%3A%20A%20Machine-Learning%20Based%20Algorithm%20To%20Prioritize%20Causal%20Genes%20of%20Quantitative%20Trait%20Loci%20in%20Arabidopsis%20and%20Rice" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/9/10/3129&amp;title=QTG-Finder%3A%20A%20Machine-Learning%20Based%20Algorithm%20To%20Prioritize%20Causal%20Genes%20of%20Quantitative%20Trait%20Loci%20in%20Arabidopsis%20and%20Rice" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/9/10/3129&amp;text=QTG-Finder%3A%20A%20Machine-Learning%20Based%20Algorithm%20To%20Prioritize%20Causal%20Genes%20of%20Quantitative%20Trait%20Loci%20in%20Arabidopsis%20and%20Rice" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/9/10/3129&amp;title=QTG-Finder%3A%20A%20Machine-Learning%20Based%20Algorithm%20To%20Prioritize%20Causal%20Genes%20of%20Quantitative%20Trait%20Loci%20in%20Arabidopsis%20and%20Rice" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/9/10/3129&amp;t=QTG-Finder%3A%20A%20Machine-Learning%20Based%20Algorithm%20To%20Prioritize%20Causal%20Genes%20of%20Quantitative%20Trait%20Loci%20in%20Arabidopsis%20and%20Rice" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/9/10/3129&amp;title=QTG-Finder%3A%20A%20Machine-Learning%20Based%20Algorithm%20To%20Prioritize%20Causal%20Genes%20of%20Quantitative%20Trait%20Loci%20in%20Arabidopsis%20and%20Rice" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/9/10/3129&amp;title=QTG-Finder%3A%20A%20Machine-Learning%20Based%20Algorithm%20To%20Prioritize%20Causal%20Genes%20of%20Quantitative%20Trait%20Loci%20in%20Arabidopsis%20and%20Rice" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/494281/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494281" id="node494281--3" data-pisa="ggg;9/10/3129" data-pisa-master="ggg;g3.119.400319" data-apath="/ggg/9/10/3129.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">QTG-Finder: A Machine-Learning Based Algorithm To Prioritize Causal Genes of Quantitative Trait Loci in Arabidopsis and Rice</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-3742-5291" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Fan Lin</span>, <span class="highwire-citation-author" data-delta="1">Jue Fan</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-7572-4762" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Seung Y. Rhee</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3129-3138;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400319 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494281--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Fan Lin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Plant Biology, Carnegie Institution for Science, Stanford, California 94305</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Fan%2BLin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lin%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AFan%2BLin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-3742-5291" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Fan Lin</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jue Fan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Plant Biology, Carnegie Institution for Science, Stanford, California 94305</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jue%2BFan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fan%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJue%2BFan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Seung Y. Rhee </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Plant Biology, Carnegie Institution for Science, Stanford, California 94305</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Seung%2BY.%2BRhee%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rhee%20SY&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASeung%2BY.%2BRhee%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-7572-4762" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Seung Y. Rhee</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:srhee@carnegiescience.edu" class="" data-icon-position="" data-hide-link-title="0">srhee@carnegiescience.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=3129&amp;rft.epage=3138&amp;rft.atitle=QTG-Finder%3A%20A%20Machine-Learning%20Based%20Algorithm%20To%20Prioritize%20Causal%20Genes%20of%20Quantitative%20Trait%20Loci%20in%20Arabidopsis%20and%20Rice&amp;rft.volume=9&amp;rft.issue=10&amp;rft.date=2019-10-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Lin%2C+Fan&amp;rft.au=Fan%2C+Jue&amp;rft.au=Rhee%2C+Seung+Y."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/494281/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/494281/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/494281/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/494281/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/494281/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/494281/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/494281/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/494281/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/494281/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/494281/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/494281/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/494281/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/9/10/3129" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-10" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-15" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/9/10/3129.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/9/10/3129.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/9/10/3129.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__c-64NG7CB_fwZbyex-B49GkX1AvVESfu7M9yxTNDG_o__Uvi1liJMEKbHtHHv_1fo_FCLvimvbq9jMwPcCPR364Q__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
