<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/9/10/3273" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/9/10/3273.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/9/10/3273.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/9/10/3273.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/9/10/3273/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/9/10/3273.atom" />
<meta name="HW.pisa" content="ggg;9/10/3273" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="IslandCafe: Compositional Anomaly and Feature Enrichment Assessment for Delineation of Genomic Islands" />
<meta name="DC.Identifier" content="10.1534/g3.119.400562" />
<meta name="DC.Date" content="2019-10-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright Â© 2019 Jani, Azad. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="One of the evolutionary forces driving bacterial genome evolution is the acquisition of clusters of genes through horizontal gene transfer (HGT). These genomic islands may confer adaptive advantages to the recipient bacteria, such as, the ability to thwart antibiotics, become virulent or hypervirulent, or acquire novel metabolic traits. Methods for detecting genomic islands either search for markers or features typical of islands or examine anomaly in oligonucleotide composition against the genome background. The former tends to underestimate, missing islands that have the markers either lost or degraded, while the latter tends to overestimate, due to their inability to discriminate compositional atypicality arising because of HGT from those that are a consequence of other biological factors. We propose here a framework that exploits the strengths of both these approaches while bypassing the pitfalls of either. Genomic islands lacking markers are identified by their association with genomic islands with markers. This was made possible by performing marker enrichment and phyletic pattern analyses within an integrated framework of recursive segmentation and clustering. The proposed method, IslandCafe, compared favorably with frequently used methods for genomic island detection on synthetic test datasets and on a test-set of known islands from 15 well-characterized bacterial species. Furthermore, IslandCafe identified novel islands with imprints of likely horizontal acquisition." />
<meta name="DC.Contributor" content="Mehul Jani" />
<meta name="DC.Contributor" content="Rajeev K. Azad" />
<meta name="article:published_time" content="2019-10-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="IslandCafe: Compositional Anomaly and Feature Enrichment Assessment for Delineation of Genomic Islands" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;One of the evolutionary forces driving bacterial genome evolution is the acquisition of clusters of genes through horizontal gene transfer (HGT). These genomic islands may confer adaptive advantages to the recipient bacteria, such as, the ability to thwart antibiotics, become virulent or hypervirulent, or acquire novel metabolic traits. Methods for detecting genomic islands either search for markers or features typical of islands or examine anomaly in oligonucleotide composition against the genome background. The former tends to underestimate, missing islands that have the markers either lost or degraded, while the latter tends to overestimate, due to their inability to discriminate compositional atypicality arising because of HGT from those that are a consequence of other biological factors. We propose here a framework that exploits the strengths of both these approaches while bypassing the pitfalls of either. Genomic islands lacking markers are identified by their association with genomic islands with markers. This was made possible by performing marker enrichment and phyletic pattern analyses within an integrated framework of recursive segmentation and clustering. The proposed method, IslandCafe, compared favorably with frequently used methods for genomic island detection on synthetic test datasets and on a test-set of known islands from 15 well-characterized bacterial species. Furthermore, IslandCafe identified novel islands with imprints of likely horizontal acquisition.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2019/10/01" />
<meta name="citation_mjid" content="ggg;9/10/3273" />
<meta name="citation_id" content="9/10/3273" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/9/10/3273" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/9/10/3273.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/9/10/3273.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/9/10/3273.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.119.400562" />
<meta name="citation_pmid" content="31387857" />
<meta name="citation_volume" content="9" />
<meta name="citation_issue" content="10" />
<meta name="citation_num_pages" content="13" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="3273" />
<meta name="citation_lastpage" content="3285" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Mehul Jani" />
<meta name="citation_author_institution" content="Department of Biological Sciences and BioDiscovery Institute and" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-2486-5696" />
<meta name="citation_author" content="Rajeev K. Azad" />
<meta name="citation_author_institution" content="Department of Biological Sciences and BioDiscovery Institute and" />
<meta name="citation_author_institution" content="Department of Mathematics, University of North Texas, Denton, Texas 76203" />
<meta name="citation_author_email" content="Rajeev.Azad@unt.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-6874-6146" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=S. F. Altschul;citation_author=W. Gish;citation_author=W. Miller;citation_author=E. W. Myers;citation_author=D. J. Lipman;citation_title=Basic local alignment search tool.;citation_pages=403-410;citation_volume=215;citation_year=1990;citation_issue=3;citation_pmid=2231712;citation_doi=10.1006/jmbi.1990.9999" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=A. J. Arvey;citation_author=R. K. Azad;citation_author=A. Raval;citation_author=J. G. Lawrence;citation_title=Detection of genomic islands via segmental genome heterogeneity.;citation_volume=37;citation_year=2009;citation_pmid=19589805;citation_doi=10.1093/nar/gkp576" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Comput Biol;citation_author=R. K. Azad;citation_author=J. G. Lawrence;citation_title=Use of artificial genomes in assessing methods for atypical gene detection.;citation_pages=e56-e56;citation_volume=1;citation_year=2005;citation_issue=6;citation_pmid=16292353;citation_doi=10.1371/journal.pcbi.0010056" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=R. K. Azad;citation_author=J. Li;citation_title=Interpreting genomic data via entropic dissection.;citation_volume=41;citation_year=2013;citation_pmid=23036836;citation_doi=10.1093/nar/gks917" />
<meta name="citation_reference" content="citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=J. Proteomics Bioinform.;citation_author=D. Che;citation_author=H. Wang;citation_author=J. Fazekas;citation_author=B. Chen;citation_title=An accurate genomic island prediction method for sequenced bacterial and archaeal genomes.;citation_pages=214-221;citation_volume=7;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=D. M. de Brito;citation_author=V. Maracaja-Coutinho;citation_author=S. T. de Farias;citation_author=L. V. Batista;citation_author=T. G. do RÃªgo;citation_title=A novel method to predict genomic islands based on mean shift clustering algorithm.;citation_pages=e0146352;citation_volume=11;citation_year=2016;citation_doi=10.1371/journal.pone.0146352" />
<meta name="citation_reference" content="citation_journal_title=Journal of Clinical Microbiology;citation_journal_abbrev=J. Clin. Microbiol.;citation_author=M. Denton;citation_author=N. J. Todd;citation_author=K. G. Kerr;citation_author=P. M. Hawkey;citation_author=J. M. Littlewood;citation_title=Molecular Epidemiology of Stenotrophomonas maltophilia Isolated from Clinical Specimens from Patients with Cystic Fibrosis and Associated Environmental Samples;citation_pages=1953-1958;citation_volume=36;citation_year=1998;citation_issue=7;citation_pmid=9650943" />
<meta name="citation_reference" content="citation_journal_title=Journal of Bacteriology;citation_journal_abbrev=J. Bacteriol.;citation_author=U. Dobrindt;citation_author=F. Agerer;citation_author=K. Michaelis;citation_author=A. Janka;citation_author=C. Buchrieser;citation_title=Analysis of Genome Plasticity in Pathogenic and Commensal Escherichia coli Isolates by Use of DNA Arrays;citation_pages=1831-1840;citation_volume=185;citation_year=2003;citation_issue=6;citation_pmid=12618447;citation_doi=10.1128/JB.185.6.1831-1840.2003" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Microbiology;citation_journal_abbrev=Nat Rev Microbiol;citation_author=U. Dobrindt;citation_author=B. Hochhut;citation_author=U. Hentschel;citation_author=J. Hacker;citation_title=Genomic islands in pathogenic and environmental microorganisms.;citation_pages=414-424;citation_volume=2;citation_year=2004;citation_issue=5;citation_pmid=15100694;citation_doi=10.1038/nrmicro884" />
<meta name="citation_reference" content="citation_journal_title=Emerging infectious diseases;citation_journal_abbrev=Emerg Infect Dis;citation_author=B. Doublet;citation_author=R. Lailler;citation_author=D. Meunier;citation_author=A. Brisabois;citation_author=D. Boyd;citation_title=Variant Salmonella genomic island 1 antibiotic resistance gene cluster in Salmonella enterica serovar Albany.;citation_pages=585-591;citation_volume=9;citation_year=2003;citation_issue=5;citation_pmid=12737743;citation_doi=10.3201/eid0905.020609" />
<meta name="citation_reference" content="citation_year=1998" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=R. D. Finn;citation_author=J. Clements;citation_author=S. R. Eddy;citation_title=HMMER web server: Interactive sequence similarity searching.;citation_volume=39;citation_year=2011;citation_pmid=21593126;citation_doi=10.1093/nar/gkr367" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=R. D. Finn;citation_author=P. Coggill;citation_author=R. Y. Eberhardt;citation_author=S. R. Eddy;citation_author=J. Mistry;citation_title=The pfam protein families database: Towards a more sustainable future.;citation_volume=44;citation_year=2015;citation_pmid=26673716;citation_doi=10.1093/nar/gkv1344" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=H. Ganesan;citation_author=A. S. Rakitianskaia;citation_author=C. F. Davenport;citation_author=B. Tummler;citation_author=O. N. Reva;citation_title=The SeqWord Genome Browser: an online tool for the identification and visualization of atypical regions of bacterial genomes through oligonucleotide usage.;citation_pages=333-333;citation_volume=9;citation_year=2008;citation_pmid=18687122;citation_doi=10.1186/1471-2105-9-333" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=S. Garcia-Vallve;citation_author=E. Guzman;citation_author=M. A. Montero;citation_author=A. Romeu;citation_title=HGT-DB: a database of putative horizontally transferred genes in prokaryotic complete genomes;citation_pages=187-189;citation_volume=31;citation_year=2003;citation_issue=1;citation_pmid=12519978;citation_doi=10.1093/nar/gkg004" />
<meta name="citation_reference" content="citation_journal_title=Physical review. E, Statistical, nonlinear, and soft matter physics;citation_journal_abbrev=Phys Rev E Stat Nonlin Soft Matter Phys;citation_author=I. Grosse;citation_author=P. Bernaola-GalvÃ¡n;citation_author=P. Carpena;citation_author=R. RomÃ¡n-RoldÃ¡n;citation_author=J. Oliver;citation_title=Analysis of symbolic sequences using the Jensen-Shannon divergence.;citation_pages=041905-041905;citation_volume=65;citation_year=2002;citation_issue=4 Pt 1;citation_pmid=12005871;citation_doi=10.1103/PhysRevE.65.041905" />
<meta name="citation_reference" content="citation_journal_title=EMBO reports;citation_journal_abbrev=EMBO Rep.;citation_author=J. Hacker;citation_author=E. Carniel;citation_title=Ecological fitness, genomic islands and bacterial pathogenicity: A Darwinian view of the evolution of microbes;citation_pages=376-381;citation_volume=2;citation_year=2001;citation_issue=5;citation_pmid=11375927;citation_doi=10.1093/embo-reports/kve097" />
<meta name="citation_reference" content="citation_journal_title=J. Bacteriol.;citation_author=M. T. Holden;citation_author=H. M. Seth-Smith;citation_author=L. C. Crossman;citation_author=M. Sebaihia;citation_author=S. D. Bentley;citation_title=The genome of burkholderia cenocepacia J2315, an epidemic pathogen of cystic fibrosis patients.;citation_volume=191;citation_year=2009;citation_pmid=18931103;citation_doi=10.1128/JB.01230-08" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_journal_abbrev=Bioinformatics;citation_author=W. Hsiao;citation_author=I. Wan;citation_author=S. J. Jones;citation_author=F. S. Brinkman;citation_title=IslandPath: aiding detection of genomic islands in prokaryotes;citation_pages=418-420;citation_volume=19;citation_year=2003;citation_issue=3;citation_pmid=12584130;citation_doi=10.1093/bioinformatics/btg004" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=W. W. Hsiao;citation_author=K. Ung;citation_author=D. Aeschliman;citation_author=J. Bryan;citation_author=B. B. Finlay;citation_title=Evidence of a large novel gene pool associated with prokaryotic genomic islands.;citation_pages=e62-e62;citation_volume=1;citation_year=2005;citation_issue=5;citation_pmid=16299586;citation_doi=10.1371/journal.pgen.0010062" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=D. Hyatt;citation_author=G. Chen;citation_author=P. F. LoCascio;citation_author=M. L. Land;citation_author=F. W. Larimer;citation_title=Prodigal: prokaryotic gene recognition and translation initiation site identification.;citation_pages=119-119;citation_volume=11;citation_year=2010;citation_pmid=20211023;citation_doi=10.1186/1471-2105-11-119" />
<meta name="citation_reference" content="citation_journal_title=ACM SIGBioinformatics Record;citation_author=M. Jani;citation_author=R. K. Azad;citation_title=Information entropy based methods for genome comparison.;citation_pages=1-4;citation_volume=3;citation_year=2013;citation_doi=10.1145/2500124.2500126" />
<meta name="citation_reference" content="citation_journal_title=Front. Microbiol.;citation_author=M. Jani;citation_author=K. Mathee;citation_author=R. K. Azad;citation_title=Identification of novel genomic islands in liverpool epidemic strain of pseudomonas aeruginosa using segmentation and clustering.;citation_pages=1210;citation_volume=7;citation_year=2016;citation_doi=10.3389/fmicb.2016.01210" />
<meta name="citation_reference" content="citation_journal_title=FEMS Microbiol. Rev.;citation_author=M. Juhas;citation_author=J. R. van der Meer;citation_author=M. Gaillard;citation_author=R. M. Harding;citation_author=D. W. Hood;citation_title=Genomic islands: Tools of bacterial horizontal gene transfer and evolution.;citation_volume=33;citation_year=2009;citation_pmid=19178566;citation_doi=10.1111/j.1574-6976.2008.00136.x" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=M. G. Langille;citation_author=F. S. Brinkman;citation_title=IslandViewer: An integrated interface for computational identification and visualization of genomic islands.;citation_volume=25;citation_year=2009;citation_pmid=19151094;citation_doi=10.1093/bioinformatics/btp030" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=M. G. Langille;citation_author=W. W. Hsiao;citation_author=F. S. Brinkman;citation_title=Evaluation of genomic island predictors using a comparative genomics approach.;citation_pages=329-329;citation_volume=9;citation_year=2008;citation_pmid=18680607;citation_doi=10.1186/1471-2105-9-329" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Microbiology;citation_journal_abbrev=Nat Rev Microbiol;citation_author=M. G. Langille;citation_author=W. W. Hsiao;citation_author=F. S. Brinkman;citation_title=Detecting genomic islands using bioinformatics approaches.;citation_pages=373-382;citation_volume=8;citation_year=2010;citation_issue=5;citation_pmid=20395967;citation_doi=10.1038/nrmicro2350" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=J. G. Lawrence;citation_author=H. Ochman;citation_title=Molecular archaeology of the Escherichia coli genome;citation_pages=9413-9417;citation_volume=95;citation_year=1998;citation_issue=16;citation_pmid=9689094;citation_doi=10.1073/pnas.95.16.9413" />
<meta name="citation_reference" content="citation_journal_title=Plasmid;citation_journal_abbrev=Plasmid;citation_author=A. M. Osborn;citation_author=D. BÃ¶ltner;citation_title=When phage, plasmids, and transposons collide: genomic islands, and conjugative- and mobilizable-transposons as a mosaic continuum.;citation_pages=202-212;citation_volume=48;citation_year=2002;citation_issue=3;citation_pmid=12460536;citation_doi=10.1016/S0147-619X(02)00117-8" />
<meta name="citation_reference" content="citation_journal_title=In silico biology;citation_journal_abbrev=In Silico Biol;citation_author=S. Pundhir;citation_author=H. Vijayvargiya;citation_author=A. Kumar;citation_title=PredictBias: a server for the identification of genomic and pathogenicity islands in prokaryotes.;citation_pages=223-234;citation_volume=8;citation_year=2008;citation_issue=3-4;citation_pmid=19032158" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=W. D. Reiter;citation_author=P. Palm;citation_author=S. Yeats;citation_title=Transfer RNA genes frequently serve as integration sites for prokaryotic genetic elements;citation_pages=1907-1914;citation_volume=17;citation_year=1989;citation_issue=5;citation_pmid=2467253;citation_doi=10.1093/nar/17.5.1907" />
<meta name="citation_reference" content="citation_journal_title=Infection and Immunity;citation_journal_abbrev=Infect. Immun.;citation_author=J. M. Reyrat;citation_author=V. Pelicic;citation_author=B. Gicquel;citation_author=R. Rappuoli;citation_title=Counterselectable Markers: Untapped Tools for Bacterial Genetics and Pathogenesis;citation_pages=4011-4017;citation_volume=66;citation_year=1998;citation_issue=9;citation_pmid=9712740" />
<meta name="citation_reference" content="citation_journal_title=Clinical Microbiology Reviews;citation_journal_abbrev=Clin. Microbiol. Rev.;citation_author=H. Schmidt;citation_author=M. Hensel;citation_title=Pathogenicity Islands in Bacterial Pathogenesis;citation_pages=14-56;citation_volume=17;citation_year=2004;citation_issue=1;citation_pmid=14726454;citation_doi=10.1128/CMR.17.1.14-56.2004" />
<meta name="citation_reference" content="citation_journal_title=J. Biotechnol.;citation_author=S. C. Soares;citation_author=H. Geyik;citation_author=R. T. Ramos;citation_author=P. H. de SÃ¡;citation_author=E. G. Barbosa;citation_title=GIPSy: Genomic island prediction software.;citation_pages=2-11;citation_volume=232;citation_year=2016;citation_doi=10.1016/j.jbiotec.2015.09.008" />
<meta name="citation_reference" content="citation_journal_title=Nat. Rev. Genet.;citation_author=S. M. Soucy;citation_author=J. Huang;citation_author=J. P. Gogarten;citation_title=Horizontal gene transfer: Building the web of life.;citation_pages=472-482;citation_volume=16;citation_year=2015;citation_pmid=26184597;citation_doi=10.1038/nrg3962" />
<meta name="citation_reference" content="citation_journal_title=Physical review. E, Statistical, nonlinear, and soft matter physics;citation_journal_abbrev=Phys Rev E Stat Nonlin Soft Matter Phys;citation_author=V. Thakur;citation_author=R. K. Azad;citation_author=R. Ramaswamy;citation_title=Markov models of genome segmentation.;citation_pages=011915-011915;citation_volume=75;citation_year=2007;citation_issue=1 Pt 1;citation_pmid=17358192;citation_doi=10.1103/PhysRevE.75.011915" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=G. S. Vernikos;citation_author=J. Parkhill;citation_title=Interpolated variable order motifs for identification of horizontally acquired DNA: Revisiting the salmonella pathogenicity islands.;citation_volume=22;citation_year=2006;citation_pmid=16837528;citation_doi=10.1093/bioinformatics/btl369" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=S. Waack;citation_author=O. Keller;citation_author=R. Asper;citation_author=T. Brodag;citation_author=C. Damm;citation_title=Score-based prediction of genomic islands in prokaryotic genomes using hidden Markov models.;citation_pages=142-142;citation_volume=7;citation_year=2006;citation_pmid=16542435;citation_doi=10.1186/1471-2105-7-142" />
<meta name="citation_reference" content="citation_year=2011" />
<meta name="citation_reference" content="citation_journal_title=Brief. Bioinform.;citation_author=W. Wei;citation_author=F. Gao;citation_author=M. Z. Du;citation_author=H. L. Hua;citation_author=J. Wang;citation_title=Zisland explorer: Detect genomic islands by combining homogeneity and heterogeneity properties.;citation_pages=357-366;citation_volume=18;citation_year=2017;citation_doi=10.1093/bib/bbw019" />
<meta name="citation_reference" content="citation_journal_title=Journal of Medical Microbiology;citation_journal_abbrev=J Med Microbiol;citation_author=C. Winstanley;citation_title=Spot the difference: applications of subtractive hybridisation to the study of bacterial pathogens;citation_pages=459-467;citation_volume=51;citation_year=2002;citation_issue=6;citation_pmid=12018652;citation_doi=10.1099/0022-1317-51-6-459" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=C. Winstanley;citation_author=M. G. Langille;citation_author=J. L. Fothergill;citation_author=I. Kukavica-Ibrulj;citation_author=C. Paradis-Bleau;citation_title=Newly introduced genomic prophage islands are critical determinants of in vivo competitiveness in the liverpool epidemic strain of pseudomonas aeruginosa.;citation_volume=19;citation_year=2009;citation_pmid=19047519;citation_doi=10.1101/gr.086082.108" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="IslandCafe: Compositional Anomaly and Feature Enrichment Assessment for Delineation of Genomic Islands" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10.cover-source.jpg" />
<meta name="twitter:description" content="One of the evolutionary forces driving bacterial genome evolution is the acquisition of clusters of genes through horizontal gene transfer (HGT). These genomic islands may confer adaptive advantages to the recipient bacteria, such as, the ability to thwart antibiotics, become virulent or hypervirulent, or acquire novel metabolic traits. Methods for detecting genomic islands either search for markers or features typical of islands or examine anomaly in oligonucleotide composition against the genome background. The former tends to underestimate, missing islands that have the markers either lost or degraded, while the latter tends to overestimate, due to their inability to discriminate compositional atypicality arising because of HGT from those that are a consequence of other biological factors. We propose here a framework that exploits the strengths of both these approaches while bypassing the pitfalls of either. Genomic islands lacking markers are identified by their association with genomic islands with markers. This was made possible by performing marker enrichment and phyletic pattern analyses within an integrated framework of recursive segmentation and clustering. The proposed method, IslandCafe, compared favorably with frequently used methods for genomic island detection on synthetic test datasets and on a test-set of known islands from 15 well-characterized bacterial species. Furthermore, IslandCafe identified novel islands with imprints of likely horizontal acquisition." />
<meta name="og-title" property="og:title" content="IslandCafe: Compositional Anomaly and Feature Enrichment Assessment for Delineation of Genomic Islands" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/9/10/3273" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="One of the evolutionary forces driving bacterial genome evolution is the acquisition of clusters of genes through horizontal gene transfer (HGT). These genomic islands may confer adaptive advantages to the recipient bacteria, such as, the ability to thwart antibiotics, become virulent or hypervirulent, or acquire novel metabolic traits. Methods for detecting genomic islands either search for markers or features typical of islands or examine anomaly in oligonucleotide composition against the genome background. The former tends to underestimate, missing islands that have the markers either lost or degraded, while the latter tends to overestimate, due to their inability to discriminate compositional atypicality arising because of HGT from those that are a consequence of other biological factors. We propose here a framework that exploits the strengths of both these approaches while bypassing the pitfalls of either. Genomic islands lacking markers are identified by their association with genomic islands with markers. This was made possible by performing marker enrichment and phyletic pattern analyses within an integrated framework of recursive segmentation and clustering. The proposed method, IslandCafe, compared favorably with frequently used methods for genomic island detection on synthetic test datasets and on a test-set of known islands from 15 well-characterized bacterial species. Furthermore, IslandCafe identified novel islands with imprints of likely horizontal acquisition." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10.cover-source.jpg" />
    <title>IslandCafe: Compositional Anomaly and Feature Enrichment Assessment for Delineation of Genomic Islands | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"es-r3O8pvh3QdQr4XAroVodWM7Tpws4S8sOMQa7i7f4","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"494276","apath":"\/ggg\/9\/10\/3273.atom","pisa":"ggg;9\/10\/3273","ac":{"\/ggg\/9\/10\/3273.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/9\/10\/3273.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;9\/10\/3273"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;9\/10\/3273"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/9\/10\/3273":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022qbGQMy6FmiRcXk+q+Wua09u1TKmJ8ULsilpHTVLEQMQBsBUwoYBrm3Mx5YordJJD8VK4M3KwhPa3cuX2dVqLhjkpRM0FTceExtmevvSdITQ\\\\\\\/dRu7EJ8eyT2sNFE9M6tomQ7JhkTyn0POaBCagVSVhU3MWLclcZ9m+7DdgIYhmo3lGLDrZ\\\\\\\/jydQZb53TFK4+O92CCFgbn7iIW1LdsTJLL2rWe4AdkhKRHgHPGziXTsNo7x5fNPOizmY8k\\\\\\\/mjWk+a1CpKMHDYGZsNL3pSrHJkYZScUR+8bXyNUbHKuo5bdhvsLV60pIiVZ6TDkfBYcCn6ixesGLexZ7PDOvJkOtRGjTrU4Pbt9IaUGk0TTkRp3+18Pgh4GzEePwNfQPtLpHFXoCfdXfgR25KRspkGHiTuhuwwzZJcRvggjGgrS9Ci4wAE7i0kl9cqq3ytOVbLOxJDKDj1CLwlRf\\\\\\\/HiwQzJ8EF8nyssHqjAbeRw\\\\\\\/JFaWltAQZT+cYXf8mO\\\\\\\/olp0RMmdd1wR5ZQ44o0WTrquNUXT3ddqZDPyJVhsWN9TUU6MoHLyHlTRSPbd5tTx9S2oxJcUrn06uz4BQpQ+2vkodFDyfwbb2\\\\\\\/BZAa4PyazxvPCx34OlG0fiWGe6gkZ0UBZP9Im7sU2oanrLcMtqln3Avzoueziy+gL8X9D4ogQ86mBRiij2krDyDYn5A++k0rlAUKWUV1SqmzGUGe+owTPCJFQKwh9wWU59ZSI8I3jlf6SkF83F+00vBN3EXP7ZKi9fW1UJe6wL+yGItzfI9PKmzKwRSTtvdlkaIWEWgj2N\\\\\\\/In+KYhHMVb1bmnd3TaeSJ3O1wnM8wSMvacBnHGpq0jwP+KCaxk0banr6Kksf0byZ52HwDEkOSfh3aHy4eWQkzd0Wq9mS6dUE6AxV0mUCKwRvrXxO3KhAs8DeODQJ3\\\\\\\/MGpikcTvjxjuhDXEG4VCCrxG5wvNhULKsI2kJo1O3W+Ch7V0AS9MWYSLPK+A2ELAmwN9PtT54kVS7qX\\\\\\\/xAD8fsyAqqGWhum2xcU10tDo2dKctYRxC9b26NYodfNvNgBPKrN3dN4WYuOpbvCo4zVTlCQhlqCLXvmpGNuixc59nP9qkYdr1ffsxOVyikt695Z0nIJAby4LiwljlZT1036W0IOkCpa0hcmjW53U1kqwaaBD2LHJPo3+O5Lxg7d6iBb0IMpfVHdq\\\\\\\/2jKGMrTHALijBJKZXbTHGA\\\\\\\/zAiH2syAW32vaE5DRhixvTsr1PdygeG00sEi6AAGzbQxVp7oOfeyr7II3I6XY00BX\\\u0022,\\\u0022iv\\\u0022:\\\u0022ODE9D1LXNnXPuKvhr3Goaw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022e85401dbd73b032414dfe190c5274fe5ee6206817c1de81f187dd7dd94d6d85d\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022LHbE0BhOB8hquLEPjLbtny1z6kQw8AMfD7LekBe0OCT7pdfn3TYBELhPzhUgTLxzbX2nh3Jymy7XCnGLpDNBsASy8eaB5aqgV3G6XGAOuGuQYHv8B3F2ImL8p7QSdJonfcXqOtbNZPwZ5kTx1uRR+sEqz6N\\\\\\\/nwdJFT60wE57walVC1YAcd3wQBhh6qVmHTrQJAR\\\\\\\/AQ29eSxlGnkqFjkL1uag6Wpeia\\\\\\\/XLeTtua5kq7xIUxUPOK1ILLG78pUo97CeEacFin+EKli6tMPTtY4r8d9YQi\\\\\\\/oHcgvdihk7YhaR2rWAs6aZ6tFfxqmvqRNt5UdRvEWRgrgPQltHhYcCZxLFsXeRzgYawggd+83Rp1\\\\\\\/QvcZvb2niYpGHf++Q9T+4Edc1ppQBQrU\\\\\\\/dodkREXGkKkWfhxGQI5osX2c4WHAcHTFuLgEzhdnJRffutzJJ727ZBgLEcGZ8huybuyUr\\\\\\\/ujS9v5N4vw2Y4gryZBZLb1+NtpRIKkvcWNBC2SylD\\\\\\\/B3L06KSU7kqbgaHIiPdIH7xTjotvxZUoq3UaZLZ\\\\\\\/c2NTvRDjPGgHHq95ModvdfxFtbjyoQBNvYhnbmTPWOGB7d6s1BPGCyqOwzMFhAf6tk6X6J5WNXxa4hvp\\\\\\\/LBofuQs4nEXQQ6qBlE3My3UVnKy9+wBHUYsP9AfGSqWa5lkwcIjOt4xerhCQNHDOSpWybsjxPK95QfePLqGqHUU3bgTvRu2z6nzUydqgGbomzYh+ib4UjnKKLiYm4xptcgYaneIEGKeYBQLuXtXMC62g6Yo7Ol5VPKov3QsACFuydKfyr41xfyexQ8fguk+FMQKeVNCedlA2eoc\\\\\\\/qkMyRXirb4yHZLKuWLLJ4Zc1Fz\\\\\\\/bEVt+Oi+P7hw4jVN+aDSFQ3x63lI6KWtgQczcR1v6JoI+DflNjRe+znXRCvHyV4zp\\\\\\\/D60n5EU35Sezf5BNvjqLC4l+3lyuJD\\\\\\\/AJtWbfC+Jr6O\\\\\\\/LqdbPRQeQCk91lEGsYH23w6vDilz\\\\\\\/EAWd0AjRM6bS0QPdwiP7oTvMUs6o4AZA+aG73aXCHU8DmGhuU3y8mDIQoewXhXDFB+N2zyHpR8LIueiXQkpmFAk\\\\\\\/RwiLshn1Qm58YuKDY9dDsa+EPH7u+ru1Paj9vPggfB4vRwxjKuwXLUEh+d8FdrFn+kvXNWKfszTboazNs5Ou4LAJF7Sk8gaRcSXWcqcfS+mFv9eVBNN8qHKjbqL67SZc+LYROvgOT9YOLhoHBchystXlQwccEhu0IkNCDzh6NJywv6+UnYKNcLbsU5gaIaZ6jHl5RwhD7ieweyLJV\\\\\\\/Vla9JWp0992nrEeO4b8xzQkDINSF5HW9fWfYhZYoTAy4JgOcDQImv7jwmcEipY902LoNdgNt2NXUcHmfb9FJvBG\\\\\\\/U2KYoQ2WRTqZ8Ca6me6fvvYSKdqQJfvzTWpxCmKIdyJy2\\\\\\\/qOZ5rDrd71BIqbC+pcU3VD4mKLB4we\\\\\\\/rbHqgVs6YKJREGiDulzCPHSyzZzmuaxciL4wDjxN35CJ1qCYBGMuDRpa3aXtId5RYJJCJlK7xvz4e4qurAfQFfnIrZ4CJLKB7E\\\\\\\/CbSCIYBYbW3nNiJjjSVXgI9FAXRpwzHTaElOloyzi+mnrnx6NcyHB7bvHfHCfpI\\\\\\\/Y7wIdGGkRbfgToo4GWb7WDa7onuLNknJdSjf8IB5\\\\\\\/XbZ1XzvFZorg15K7ta4W0fgPMS36z8xZi5xgfGrPhJNz5ZfF6Hz+NssseP5z1+esn+QiMzoIXjwPXPZD++7EX1jzC7v5WpCfA+nGcUcC6n166yie05OdEyiOibNglQcIFYpIwAtD+o776VgnUVevqQM7IA96+y12Ey+WB1Yvh5JI5JRreAyzAeXfGPoWRnrxBruWkeZ2EMTmliUa6CevTEEAbyj6\\\\\\\/yEdPRGqDzPf7zrKG8VCQN+lAFQzb9s8Fjzu+rTHIViSJGorsBVBLM9+isxtCBFaRr\\\\\\\/NeCqE5o3FrJW2vpuRScZFCpbnL7Uw+1iSbVBz7Ivu9Ug6GvZsEsf9Gq0eHDU6g1tcTClRhu0v0KYSRdA3M5vmhqwtIrzcFVFy1xCC\\\\\\\/ZHAAOPxhFujKp4z7+bqg7mtes+3jSeCutycDYMFGQXlDCgeEiNFkLDM8oQCzvsjYnLH8csegim43V7WQF4GVAfN6+8exFe+D4lHe6ROWnTAdVCB\\\\\\\/hgQa3985om7PmslfupjDLRZj1SjDPJcvHj9QzeKQaq9TR28v3hdHCovugH2\\\\\\\/qNGQVYQcZuwt\\\\\\\/cnYFaq201TInscCDaAQZMV52my5bvvwWp4GimQrQ8xIcybsxN2u15FaeOCndfJ5ewOpX1HI959XTn9ltcetMp8ynBhTLfhRx0AS9M6OyWpIWtife5yB7Xjb\\\\\\\/CecqPU52OWMcGsllHIJGEp083B50r41Prn\\\\\\\/yohP6u4KiU3TlVS6NFX3vhZXreVDeWl60aycbPr\\\\\\\/Km5VuitTkdY\\\\\\\/8kopvCTGGLQ18iaHc+JvQ3TWT7KDjdq56NmzWrEzuY4HHUUSqDAlRJ8BOu5vosv0VLe29UQFIQPG2ogvzCP\\\\\\\/HJ7kI0g40db\\\\\\\/wKTh34aeoD2QD27HQ59m8K9jq4I0TddAzz4b\\\\\\\/cCi3nd3E9W+WZjvNE6gXRr5N2ngsl2QdtWzHc6k5Ia2+0p1Tr6wkdOQGA53zw2Uh3BO8ZW58SblGZEXiXXvUAmU2aaWqoP\\\\\\\/Xujrmtu\\\\\\\/iUTMpy9K4D3ERA2Jrum+bPx2+zpWLjJDqFZNgrPc0lP8sHSHhAvObx0vGZF24VQO3pMUjuLgW8HBJQzNJuu2ez0JSTGI3cR+5axBmJAGkDCb1LA79Z2m9aoq\\\\\\\/gyM53PQujQuRHXZxfce8KiK3isPIN+VDV+k1a6ubL6zNivUc2j1\\\\\\\/34A4qhgLS+USqh\\\\\\\/al9wIxjSThmbpI4U\\\\\\\/KfyQTW9+mmHgTZF0R9+tyk+OflXw7FmORvWSgHpDofw0DsFlqAHGJBZ4g7ZzcwoP8zNRqlNbb6mKu7XfBcKI1MRPBlEwaeLwB43HD8Qkb5NnizYkJ36EmNzQ9h6O7wb3\\\\\\\/aD23rLrTfRn1s\\\\\\\/ttGrGDbArtzXnb8y7qURoOBAMU6bxq83fPs20FzuSABafs7KPml7T9BV\\\\\\\/dh3MPUbpZu8VVkpt37z\\\\\\\/dnAY+HU0vM3+GFHcC3t1pOcElR6Xtql2LHAu1TXyvNWwBIB3XojtSOoqnzofVzTdF+QVah3qBn+PUGm9VW0Gec9caUJeWBeXNeVgD1lZ235Jr5XxSnXfGjBgN9eOc3twHwnrPxLhQOwSnbB5OTKPrPF+a9tqjR0f2DEm0LnyQDYX45Tf8bRZnvZxQQtgYPi6\\\\\\\/a\\\\\\\/WW7pI0tUWEiSOnQWGsvbmEGKpKeXG4PDEujqtzMj3yBrGhMlal5+Ky0qpp2jI2YlzTJlZ3iwhV\\\\\\\/aOwlSMyzGWAZ9qvQ+DaEcPX2vrJWVGMdmMumshsjA7vDU6f+d\\\\\\\/1RgTI6ozgmP0I2bNWKfm5GEe6qY\\\\\\\/kwyqsthtwdhMbCNOhxokqJ2WhAVj5Pvof4YsQY05LzvAiGvNRy2WFzlv8fVoFTokQBxPPDgtw+fDXWR4x\\\\\\\/+jONxQO0NtvxKwyum0IEUdLV1RCWp4u8Ks\\\\\\\/89HmNpHnZ\\\\\\\/yTeSrqZUNoDsn1hZq5I9k649Qt\\\\\\\/82TYiaMotKCX1A3I4bAy3Cz7EfyWd1Db70Fy2BRltfPWAGpX9JkNdo+xA6QpJWIeCM9i6WYR8JMrChA3bVTE8Kb9da+VKJvM3bwWf+sZDQxng9l3qibiwPfLYoUn5GeJ40CG4St9JUocTNR3mWrT1IlxxUcVw5kIniUpZeFs0DbhGnUdcvZrGWtVwPN7X8b6Wn2psB\\\\\\\/MZcQnLeX31h00Jo3J9MFE6tijkGNKPa+qhKQAfLndQmJVD0qAl7LqSfm5EhmvqL0ND3S1wPNkPkdtK5ZLpMR7j0fCd7gEQW2Rrn0q3HdnksnwAgfooMHu08macauwowd3FgRhd8khBOS2IjpQfrPyb1fltVgRu5BFkCl\\\\\\\/G9W7Z5FIjSvA6a9VzvRVBus5NVKOSyVwHSOEHd4MOZe8gyvpwOQcuObPlPqM6\\\\\\\/p3dvk\\\\\\\/nlTN67NQkeEXu6dlg4fvCtXGC6RJd2+X7nZsvtpZ\\\\\\\/711M7WHFwtt666RtYoykKg9QXcRseJ2yiQr4RQblYG6+\\\\\\\/Q40TE+1w0JvccDkDRqHbMaXXLxiEgNqW96aJlwPb5pVBTV+Y8e57Naa3d0sYT\\\\\\\/TGrG6CCC55x+9XMGV0g0HURqLQdEsZ\\\\\\\/lwe6Llrv2eSfpvmAV+izif53qvbB8jWAZtOrV4\\\\\\\/yRRsNzmjs5FxX6c9XNYcnjPaggTf2ZyLaqE\\\\\\\/tG4Tr6uiTdLnX3tV2OZvbGM0P6huA5Ds78rHLrharvWhMztsvKJJMUJx9sBi3jHT7x+ertxas\\\\\\\/zpii8\\\\\\\/+\\\\\\\/FjvwIMIOwBgVcb4\\\\\\\/p5BQ3+h6th7wVFZT4azwygxHWYQsNJLDL7rW52Q3mX+NGb0XqUCZB0KKy83Hbu\\\\\\\/BO+ahENaYfJ23cganWgXfSBTI7cZTotXpHAqd0qER2w7fnDGhPVDyF\\\\\\\/KbWC+qwKM2LC9FL2rUpsXz4o6BpC\\\\\\\/qQqzYdlIEyn5VUAXMChKiUCeh2dvQORkFoStrGpWx7DLfSE\\\\\\\/eTxv\\\\\\\/\\\\\\\/daMsnazXZsNQcwOyBV3PNVUY0zzwZ+ZaFvwlZ7NNriSn2RtaNi\\\\\\\/UPCTBWg\\\\\\\/RXUw9s\\\\\\\/M+ybTnHfbSN8KApJDFkxZ6k40+sUk3cRr+5v35RZ5tCl\\\\\\\/hxquHqC6aQjdhHLzCkLleQ135UEWuLh70E4Z493wItbIMFs4\\\\\\\/3xKSjWJoUMZOJN3eCy2ZKdNLMFUJ4HWaSHoYgDzgCZGPsi1dJzU1+J11I9fVuqeEGkK9VfAbYg6FwUGXO3bvNYQpVJoEIjEeVOSS\\\\\\\/pY0qB+xQMyXrd+zc2aQXJCKPfZleePN\\\\\\\/yjLX2BeSOPLzbAU\\\\\\\/8Qq1PsBwZPRwLjngryimksfjrJD4iSNTge+c490zYuOWt7XFXdQZddr3m8dS+9nUsVVjObVc9WurWhI1hmE2zpYKBwXC3eGNZa1JbqwsYiJhGcPE\\\\\\\/psf9kk4DWDh+y+DTvYARYOFzeNnGoiFWc1L9sj2g+5uOJry\\\\\\\/qZIURkFQRGQvSmvOjzPIMvmLa30r90hBGMYYZCVKNZr77kOrBvLtcdlTxU+fCwP\\\\\\\/Hq2ry5YW1YdpgwC4DZEoMZLAsDKZILHKz5ZjWafqd4W+\\\\\\\/GvuPKO8sH54ZI0FlM5UGKfuEss+U9MFozszQWOB1DjIjPobozZ4\\\\\\\/Tu\\\\\\\/kQzqqB2cBE7zBH75co49NrdELinspSOzJ6VNxEJC26MifXqs59UDn3BaOBbIzn2+8pwWdPr55L3X7Yq7zZFykAX3T4cgoKCHphg9yjMw2UsGW5A6\\\\\\\/W3icCZ7DhluAiVu21CaKEjzXv9jiWLJeYReBimIIQNF3mU\\\\\\\/+CzHIgtJ3bkKh2DPh1aA\\\\\\\/wj8opaxMFvfnlHSm\\\\\\\/nUufIfDJCT5+EzaX6f2RMwGMzQOCgp68sFO1HlkVORY3Fy3u+TgG+mr1B9Gz5DEmaCMxzOljQQQ111MknkqMv8WrOgobsXOJths7YV0ukHjm5eG\\\\\\\/Caa4DTLH8kIfupB4yH2bxBABQIs7MI78eEcjXiZV80pP27zvWEq+DBvnhD3VCPy91zW26n+Uuzx2B833I2z+Y1L4lIPAKuU4G36bXyduVZPcxlBp3tTmf0sP6XDRkgIq2Tpu6PxcBZwNInUxPQd2B4EsZdDIYA5cFDV1rFZb3LfIsk\\\\\\\/6JvxAYT3GN2VKCWiNa2gXUqyg7EBmdGyh8a7v+rrjHvfglUHiMvE0ZE\\\\\\\/qUFO11555hp3VP9FPsgGqGRbg5l+v4PwJXj3X9Sg7tpSxeIi+Op9E8v0kLTKC36+WAz+pDS9VE0F6\\\\\\\/DDOmEEDheyMuDgPHYF9xKSexST0o5RYz0e\\\\\\\/Gneoj0NVsS3oOUru2pvzsZPAJEtlHsmgGVc10GI9Ged0\\\\\\\/mmtLI9J1y0nQFTYK9gZBXCj6nyJfxgxpm8yT\\\\\\\/jyl0z2L6VBHBo9YTVEqTFK6Riyhx7JXUUYg+rZetO4Ag69\\\\\\\/Ur4ROmoS9Ed7ggkBUAfWnGMCcGdueOYUjVTDNGNU+WDodifM482uOTG375AHDEmeTC9kuepfEHqsKn7eFDPke8EgiFtJdsxIVtj3jX7KY0uciJShjsn0u6WGdqZmL\\\\\\\/xgyCtir21bg7g2s2ieP0A5kqO+3pkgXSCMD0dM+znAypB8Yl3d5QzYvqUlr87gohGNu+MbhkWP9m8ohkRMFTp7IrkJo3eA+uwCUDy8M082nt0I7Abt91gfKE4lPr0r3pCFhuafz+Dq0sxWN\\\\\\\/IfRaXJ3ERiqO6PrSqkyteF7ZOAPKxqFnk8S9pvEMVSQQqFSZ+lSBVZm+in1eq\\\\\\\/fd6A0anb+Xm8cBx7HDn0HqsTO\\\\\\\/cqytEM1c7EzxpZ8evPDvBSv+9awJ9ynUXwePL8y4hCeZkB4LvRs7UKnJGBzNDrpzzfzXxyv76Fe5fWqvd7LWNPiyQvQCThHMtY1v47KiOIy8dQ9PShgJRGkGNc5BoUwTY6oB7DA9htWw0JFH1+zRLzB6myD\\\\\\\/H1zGVxz45ddeOjHPxJrNsEIBT57ArjS+Yv1BYo\\\\\\\/8V1hULuTch+bb8Ce40B+wEeP9smgfMddAGEKL95PKNkluyC2a+uG8Z2V7rm0B9wb3\\\\\\\/cAf98vs4X0IqzmgiufaDT5JdKLUeqmD440z2t12h4vBXVergHz8wq8Ct0rTHAvJ6J6dvTftuL+dNaIAeHetABk1r5hdI4oPG36J9qPRsvG7bCVL+E9te1DjqlCtzaF9gZrivabuhoZ9\\\\\\\/rqXxwMf7PYgH39a6VpunttHk+EBctly6GOBmHkEu\\\\\\\/ypcD7DNI11JxDUEgrQbYrQkb4wOqImAKDdeGciyhF2EN2oviLw2GoC0ZkzpM3VCIDH3DZEaXsJ8SbN+0lYfm4oFPgqrdJOb9F170CUQwQImuTwM\\\\\\\/IxUdmdh6rWN5csvFyjE1F0kSneR7qi5Gq\\\\\\\/8Ka4+MSvm6wASU9Go41JhUBHnPBwzibtoWT63MMFrz7ve4vauqmzv3DmOR3bZlD96IovWOw22+AOgxehQWTgLMmFbKK8TLJbGohejj1ecVAyrMERiFvEbJMI+qiNKW3fCRlnzARA2vmLVfhLpJrsyP\\\\\\\/c4ItCdUFF5XF2eM0Od5nRClgEX7mQn54trgps\\\\\\\/6zeoPfsVRw2pRuX32oeNmfgcU8ThpUw5MwZWri31MChZYM5E+l1glftKpW7vYXr3k81ZzORNn9sUrHQRqeRRJKNigNP0\\\\\\\/Rc4SZ8zm9D2zO0WnsDh+VP5Pvp2lUdfDz2LlDf\\\\\\\/kJhuS60Iw6VjR3w5wordJhBxYX38x5c7C54vvgdXH7eOHlCZ1M\\\\\\\/WO4Xfy7mYbSxHWA\\\\\\\/ak3K\\\\\\\/LAH0mxppe7xte75nei683r0OVddqrkje5DkXBGuaiUAF0Vjt1bpoXdJQt6ZNBct5o20\\\\\\\/rS0ctUSgdKxwxAnbj38dQDnNG3ZDJfJNsMtnX+NHMCTY9g0eAZRf0X5sP5onUs6hovh9ImbGcgLo65BtyYIhZvSLX82JbtQKlcsO4wOnFDejeB8m72jLC+p9X5LPiyAykUoifXYZaQ8xKWt+J\\\\\\\/0xQuMh2Z6sIbotmOBwjwhigk+L6vRWC6+kcVjCS0K\\\\\\\/\\\\\\\/d6XCyKSoKIMA9fZVrXKnfeUtS\\\\\\\/lBMgW9TEYxvugKHZM21+JMaeh9+S9DiRNK6KHSPkCF8PARbPlPeXQbl0fxkK\\\\\\\/de+9umFOLInDjpFfxVxyVA\\\\\\\/3pFxuCYLGH0apJbG4yJaGlgqDYZ2kx641kk2gn4Tb7lgGmjUSOV+BRz\\\\\\\/tTBNdYSBilZtLkST2BMfQ4vYRhp2YLxrE50Iro6sNlwZTQBETJT+wLU6Us3+VO4BA7p67OLrtlHO4aDujwdLYeA1hsAm0app70vyAe+m0QQQL1QOaagJUOxM929EjCpNMYzlQg\\\\\\\/lljVVQFMvCiFw2SWRz4tpSl1GLG17n9y5gNomPnxyfy\\\\\\\/9ipx\\\\\\\/6F7jd4nl\\\\\\\/fZcNJRVXSck2lh7jCBKhb5cC+\\\\\\\/dCAV+K5hekjj6oXp\\\\\\\/TDl+BX2fwM4dUaYM2kg6QIjgLX1JfHU0OZurfriUga\\\\\\\/DffqYJ+CC5XT2TSsAIi3fjprvYo4DCwbfJWBXTjr09lh1f7ADRV9ekCXzy2Na31TKWkRFKYY\\\\\\\/HylKrKLyczusctH31drfPdFG+XjGPy9Oa7JaC0bu1IDkXFZWNwdMFZkLsDS3OvEN3ayWisDuusnrFEMIWEx1JLihDCnKjv3nUHUZOiVIS84woT\\\\\\\/B20v2GOr2kwpRFEMKOaTIigqWHIOiiHg06lP7k\\\\\\\/uxvs36+Uf7W7cho9\\\\\\\/7Ut2bQUpQwTVLCZIQ1hqt3pTUWOUU7F8obkwmxM7m9et3p\\\\\\\/PxPDR\\\\\\\/WAhFyXTzXAZscUtOYITV5ayS5xgHPrYbZC+Az7qhi0Y5SVejqLXEK\\\\\\\/fYlTkgMBWdK000\\\\\\\/VhHkqqwYMJWMkqsGnkMJA2OcUWHWqM\\\\\\\/1FUR8QrQ0RZH2mBsDxMvwI22kW2ASLig4ujqKOzSjvz3J\\\\\\\/qf+vfPVd\\\\\\\/0jLNp9tI\\\\\\\/\\\\\\\/H7LAzGHN4qqWSePbLmwIhYWUtP2GyNYXGme07PjjGWTiRHPQ0CdNy71Paeznwxj\\\\\\\/mvC576SfhhWPlrsiKzKGGhnzRcQbDRrqriZbvF5uGs8mAsXh1cxOOz9Xiu6OvMQKsXH4hz2JEj7OHC86LQOFGHEjlvJ2BDAsTU6xl4Sk2rmaAU0S\\\\\\\/YvJP+iyqMiCh0M17DAaalPxKt\\\\\\\/C4jH+u8Gmu\\\\\\\/y1\\\\\\\/IeGsbTaDIrsfUrC6z4VYZCEpawBI0RjagFJD0W2ZKIPn7u9xa\\\\\\\/eaPVx3gFaQ3BN+6ZigpURpds7RzPKUGNV9D2PfQ5pbNRfEmtuMHVse9lyOegLK92MRwjPSf5YVPfwzYakb7MvO1rlvtJ+8\\\\\\\/20G2nQqE9bvuwyLnZw7hJyYwRNw7BxUKYro5o4oL2sdLsU6+OyuXbaV5fH85i4Ytkn56CVPkF5zZvDYzDjzvaoN0bRBNSHreb2uJvwhWEaSD9939A8qYN8OwF+JGDrAkAE+3slm2Kls9qtlEBIbMar4r6Cj4Qy4PLqc4GThT++plgyhxuQny2yeJzfeZkK90j3ItD13d0PfNi0c2IyfBgzTM4z4zuYVPvN9TuaaK5+nQLiemak6Xq2R7836UT704Ul9Dxs1o2R55fBFdwL0l0cjCJp82PnnCB0+OT\\\\\\\/MKkKOHpvGJ1yWfRI\\\\\\\/u9foiVBC+ztjKheYkG\\\\\\\/1kJCFEWBCbW9Wk0oygkkUVKve8IimI73ytVV0koCBglCp4O+x4Npi\\\\\\\/HLitxDb7tdObiM4ap8POiPvaq+KNLpmWzVMhL5S1nlCUzYRWaskBz4U93j\\\\\\\/lRGOcX8S1nCntOwmKMFAz6d2A1kGmLcsDhQRf+fBf+RSI7Hs+hwHU1v3WVmnRpzvsVqxGlhi+FZ4KFYE55Qe5iF5PAwdRsh23uaGK+\\\\\\\/ke4lzEEd+688gviWJL\\\\\\\/ld2kHK3XGyjgKiKDzaQPdFz63alcZ7l0\\\\\\\/RYZvMZS6gfLrnXsVSzXh73N6fSIwlrioePuWzUzRyNYNB3WxPvpeAWLxj6A+FdiGxpYopm6tw7uSlOlDNTU0t6FpYG3ht6czhxlj1WW65t2grWlhA3aD7Xw5l9S74rhgg5WeRHRvFfBjjHIabTFqnvjgEteTpjO8y6c1P4T4vZ3G5MeXTwUt9cL0iEV7PP+k8TWYb5jSagCvkPSLVcDg6MWh1AmPDym\\\\\\\/d33czZh9jpQCW4lRM+7tH4ptJxv763HEQWr4WswsdkACwMxFRjapM8Ly1mPZ97QVZcNcVzIzGItEtOwmQTNe5HAp6pomLc9VbmzVr4PJosyq2hkai0POzJWV9GINyqWVVnwvkqzokwsaSYpTOruTojIfkT1RmACZSPMVUclp9a5iAjhet63mRoNPHOa313LJJ9qivatI0FpIo+gHGPMTAQ5blZ7V7C82HfQ\\\\\\\/a\\\\\\\/VVWlnfPgRAKjBJvskE3ic4Lbx+QVgt3GWYy8vq7KWPt+rt\\\\\\\/tndkdcLLp2LiMInbd6qqPt\\\\\\\/QBfcM5DyI03ys3uer4KNHIrKZjooArV\\\\\\\/pZYeVAUfeZYDniQ9Q1EfSJZBhxuchIcnPp1A+6vM2oeSB4feKHuCM6K2Yz3RFvIvxCoAdkL9xqwBWukPT6Uhs0oYhoK1+Qy0+cWsLaf6bWtV+KoCTvT4o6ehd8hT59aQsrKw5qgBGNfjiG4lwMc26ELb26GVqO+vA8ggDQM3KFp67N8H0\\\\\\\/RXNX\\\\\\\/A5doh9XwIB6WP8B\\\\\\\/GZVjzkVZOAnreVAbRmU1E6JNMWiJYXfGGjHIjDVk4McT79zt+R2hoeQ4KrGTdasNFaBwpFMQbWcWzgJIQhXjgjN976JS\\\\\\\/VR5WUZ3PoQYl3mUxxvI4VoIb0tMv6hEY4qmvE4lmfGBHYxOI69RTx8PYD+I9Kjwe0P63D+mIJZFtkt1Do+AQ6pcnlnkWPOCtnrG883PzUfWqk4U4vfF8JMeNIV4PCOpWZVFGNAHnr4sFmvLcbZjwziS9\\\\\\\/XWKXskDhyXmPuVO7\\\\\\\/\\\\\\\/emdZ1ZwRCriduK4DknzX1gxIbbgV3hGuuWodhVkrNYko5zsoaqxx5Md5XTAshRscKTnHCPsKatV5ohND\\\\\\\/GzBh+mJjTqPXcmnUqeWujN6e6ulYJxhfnYEmZwRG6A8uD6Ke4V2+vTq4Ph9d6UNZ0+bOW6X06nGsfQV4K9jHBmehbo+rwF7QQAhCwSPrT0+uoXnuMVOWGRkWHRMHbNronFeiZRzYyCwhRomvWBezXfppWkoJxXLQgEpzxOXTInwD7CUE2W3kAUJCQ9WqOn94vlBsMMOz+5D+Yw\\\\\\\/cIXcijSo4JJCep6uOsnTBWXfGxEOI2y1VNk9Ar9o+pu\\\\\\\/h6Ywc+vK3pG5pfto9j9Je3uyXzwXt3TzfBV0NSXfMdbQ9qc5CyDd6GCsTKkC5uGUR7WiM72RkxffqQUxdhjyuqocppj45jgvhcfPYRKIKNPsm+OMh8stzBucY157Y5Ie4hcKDWD09w651fAkzavb8Ogtc7KKiJzWubV19MPlxDvyjZEbrqgrNWZUzTi3exSqQ\\\\\\\/x1rkIAx0P++itrAFA7j3DQ42OV+bkpvlw+tyJdRn12mzDqVytNp0WQXFKQW3lGmtev2BRsHcfBX\\\\\\\/mHunHwNRIV5hCVJ+Qxg4+oDiDpuNnhwrA2qLttAq9lvdU2Vee8RGhD6evT3Zhq7edxfW6yICwZT0cqCpoEcagGhCrattvr40oolfPtZlTz3zbwK2bCCCGpJo\\\\\\\/m7eJ8+W2oRcYdofQmmaNf0YkPQw20EBdymAFWfAhQBnzw\\\\\\\/94OEyXAJEiB3HgPgPJrx+nDwBsLFYH0ut\\\\\\\/3fatIGWF3nxRkxqRoZN8t1TRdYRBJeLpGP\\\\\\\/ANtIQ8Fb+ETJMWHQIVLO0tS9jD+8r2Ign85qP8aSuMyzrFV0LdDt3BYaItr+wO6rS53SAUV1nfO+6eWSUD4FR9pKYQ7r+TGln8g5oRlH7h9DjGez178\\\\\\\/zeRKEvwaDVUGPX7DI7JlkUBePfOqeLHRGf8itOeWVU+EGxF\\\\\\\/1uO8javzvqilLB3DI0zzTlgAMFzs5JFfJzYXXtFnBH52N1nFvn4HesIngqh3UwLE5e43Hwr+xg8JvMwKtwMAyYsdnJYDLkKjJyn9RiS\\\\\\\/8rOxMTcaz69uXduvnp5YvklB2ouKGVQTvOGu93E8oMO5+ERrgPeK51drT0Wdx\\\\\\\/VBHpmulxwWLC61TxHHjNs4Chfd77dR9pdbhZQAZuiIhQTznMa7K+C7HHd\\\\\\\/iCqKgrgkmmpjQQUn6UgY\\\\\\\/hcooj+x7ZkIdjfazbv05bJnw1FjKX1iXR8YIvMt3EKf7IzbL71Jmdb6RoFTFOYL2P3dgVOM2QTWO5oPyOR\\\\\\\/POuUurPcPv9bWxxvi6gmtIVNmb79s6MerigLv0Tc75NgaAN0b\\\\\\\/LiWGJimxu\\\\\\\/6RGy5Flx+VrduNC6FRou7KceFpPd1kFlvlSH14Ju27b8NaLjXNCgXSGMX97PiO+lfMIQU6dsPfRhSG8GN2wmjmvw4WLU9qZ9Q7q1tylI5IVIWn+A3tj+uBPmbvdSb+VBj5qoWtsBr9OulFZ2ZB+isCGN5Fu3eEmnAZdTL46vNl6LxUiU7FwlPNcUdYhe0zs\\\\\\\/F+09HoREk1qcti7sjDqMlVGJMklzOW5gNtCjmiEk4\\\\\\\/xnLsJQD7kzZMETd5bidNvg6vxyX6Z+hBrRzPnVNJ00hsC7cuOsCYQWZAdhvdf0r7D0NT5PCD10IJzhX40d+0Wg1mIj5T5Nqd3iOipZUg01y2AXks3Gi5B0VgMkVp0i1hL4PV+o6\\\\\\\/Cw3wOfOvxLxojEAYQtp7tXw8A1EEm6UYv\\\\\\\/FWYhmKN8D65GkJbHoDZ6VZlXsQC4yj9asRphfmz3+F7xhJk2648WSxCK\\\\\\\/jhLjRC9mN8VrL+Frw+kxecYargvgGgQi91U0cXJdoBsVRjFHQjwsjR+O676cni+6AIxowiOYwZqx5T5swRimkQq78aUY6rp4u0AQ0m6wQw9ws76NMSPOrEBeDT8m0yKkqZAjvJIVJPi8C8AXrWfvAkdqTUkWczKkHhtygti5gGJtkOuWoL9YXOJ6KAOmdpUWLCtSK4b1swiymiUKk4WD5qT99ScVSe\\\\\\\/\\\\\\\/ONJxH4bdHxfKE6PT7K7TuOEVmfT+HaKV8G8irLuDEbij1\\\\\\\/eJZEJkyjLSXrPcJvEAWS9DBImRT8jKdBM1AkocyIRaIn8UPO0YDZ9bh3KijAN\\\\\\\/Zi+\\\\\\\/sL0CAgB6s8R2wmBZH12PWHOBxp6JsqEtAkPwuSBfDIYT3zIwnJYhYW5BEY9eNZXncOjRe2lEtYFgyMVn5+2u8AWmBTzt3LSnHSLu6zCbWEHyzsVkYCjuBhzTk5YT+5FXRAb23lcbEVDqOKGTldYZMrjTJCv9IHlVbMCuF983AaNlE5C0V1BrGi1qncbyFqiGpGdkYZnF0Xn6aydituHT6eJv6a2DfZhgHrPHnEDYPQzFwXUAuWP4Yr82n1FbRqxXkOvYr8jkHnRhf3yF1IDuX3Wqbd+\\\\\\\/6eih5zd3mij6LaRP8anPg6+bd8yQYGSyF385ic0N+qboTc0R2fBAlAsgib2+oCg5YrYgOziEdb7Lei+zQ3mUPKVECpXwuPqNLKgZd+cSsWoEB1LT7485XdXjYps61E01sLWky1oxdAVdtEFPYNrhZHgmoGpejiI\\\\\\\/f\\\\\\\/Ow7PUNnbCSRlfX7NSvOTNynIxRRG0vymv2mIKeOU+hqnubaoqjdGtJro4KoKYb+aYvzFsqbITBRI0\\\\\\\/cPZ7NVyplx9Kcm9x4m4rRUW3tjTfR6o6AFdd1e8DKImPtqZwNharN+2BoRv1OTzVUyZgQFdwTMS2oZ055qfs9BEuZG4sBvcD2839dcn34+9uahheSY72sZXD99qePsHWwmSKEp5Z05MQtiyX1oLrdeuocnlR8tYN8VJsSU9yw7m0Bo3LkleYaa0Cq03ibX0D\\\\\\\/ekzo0vZmKO3A\\\\\\\/hVajuNByfjUFGGzdJ75w7FhpGrCAA1mNPP\\\\\\\/+cU+oBD\\\\\\\/iExsaitUkzEeGKn5gzZdMGacnqdB8bRPYaV0Q0ngbUFbT7pUK46vNfBKHMq7PaB01TNek7xCqPUl8DWqQCDGFMzKNgzwUdwSFjPV98MX+30lAJiR1XdxuUEZMwV0ixJZCgVssMtCSJxradaYjF1HG0FPhKfk3wsUTWMrXeZb3q3SFvSvlei9vh3ez5XNqqx9fG\\\\\\\/gLnlCZ\\\\\\\/mcVWMTXtAapVAJBj3ob3uPsRBWUHDVAernBsNf3TO5XkKuzKJ29k8EeC9uzG0cBLzMoR1vpBAnOpYwtlkvyxYDsl6qmvnbYKYscZsHv7Oz9VC06FXX2k\\\\\\\/QLBeQEOK1zJGMLx3y3cUHB8xtCZw97aFMcVDRQ0zdORWful2SO+wddbp69xfxD+PrRcIKaeVxas7zNXo+Th3AdGrAI49h1pCu88DGrInRNHQPRbBxEci7J+XyZTkL0pwPyO6DbnZPnlD8sbBfEO0erK8sZwRQIQR3J8XUJNylYwSKhAj0nENSoyGj3H8fDcqTJyuxN6uf72IIj3HCYIcmxNQi7V6QtPu05Basg86VgJABl6iDAaQ4DWmWPTdFkfF3UllN9jY4RYzsoXAEaLBiz0aZlmGpSLdsXAppYAurVg7wnNdYTqaM3xZvFA0L0zNZyIg6\\\\\\\/Kv7RKRjIbkqgPdPM8+Mplz0ycOQBgjo696NhcrKRJJhKixZBHHEsw++uU8Q92hORTzHW7uTFOvXuqTfMkaHzat0bLW492K3Dq23zlUU3tNzgLfPxQ1ohC61ei2BFZHy7mf+fotnDuHteJdODK0b6OOr3x8X1V3r4dYfFCSem0FcMPkxHbzekG6W5gy44wwcwbHWEHhCXZZfXquwbENKheoW\\\\\\\/7ixgVI4VUunFXkk1GKaiCDOUiDwNIb6fH+9\\\\\\\/xevXEZDtGsEGEnTElm0\\\\\\\/LJJ3bzW0woLCeI7EmyGgGZWto\\\\\\\/0L8JTqs9hEeJxlnAdvCJ3x8o7rOYgnNORML\\\\\\\/sdZQBsAhwTSiCEel7EAmS6aL8tznhd5rMozZLnuVzRaY8qKuTBe2Z5Kgv87w4ByuRlj5CAd+JY4IuL9olLPLyVQTgSFSbvwfR1UhDJEGSAw7NWSd6p9pl78gHtMH\\\\\\\/T\\\\\\\/lLC+vjzHasD4sCmw4sadNcs37W3XuLi3kevvMnjmZwZ4T5dHYp96xaHWqOwU+uyq9btD6X\\\\\\\/L1n46GqpXVdJfegXnRcdI8QAC1YlmKy0u\\\\\\\/y6o0tKSMs6wi8uZLSF0Jk7XJ3Xv40S+SiiAnbAbGMfNRWrjf0LwHycrcAjqlWl5b2VVCLTd11\\\\\\\/o5iYG73zoUs1Jsak8ZCD\\\\\\\/Qq7FGrHPI7VyxCPifzAJZhTjt4jZUY7RvqBOANtnMcjrmvLIWKn+DBo3MY57wnUZ9Ax57He8GzaPLndcrXjdaV+8ooYS1kABg6wWrPEdh\\\\\\\/PBUsev0QVGQEjtalPNkRaidFMVDDT67CeRmlJxAaiFGY8yIv5xLiWBJbdlKvjRkLpLQIOHwQMMUm9itWTvEgDF0aXeatrIFWIieOEjRJYR24sHYzWQhSsKl3xIG6BKVRKecr0167bm6qxN0O+Z5E6my3wtd1okfCUXG01VqMyNgG0wDB30Z1F6J12N2sho0Pj4Na4vo2CkjG4Q6LOwdQ7HZ+CEDc3yIC85yfAj943kwWwlFjF3StdOBiueioHiB1tuGldvqcDqbzM7eCKsrA3PPIF1TBbyPfz6leLRzOQn9+kUzjA0w0PMKACf7jWY9MzrkPH103lyO2aZjID8eSlQxbznQGOMizPfzRCbr9sRYnNBRp7Us3J4Y6MQ2K6e561SQDlv4ARRTQdEZsF3RZjyWkNL6ExozlSFFwQJpskhxFdpRUxjuCee6JsUnE7rZAUWuRPswGbNfagCh08BzpgnnQ1VQcK9OcNmHY8m1DAczgtPEtoopTuh94km+FIU5Bmm0ml11+8MRb5N7YV5PwdwYCAClMLw4DvaBfenDeSTIlcf9II2LVfoFkq2qVAXG5hzYmV1K6BTE0++lI6djqeFVWEKmtvb9\\\\\\\/GhoFetVUbrvmqph+d3F+FOHXTb74CWUo0kUVQgU3zb0ktdHcwd79V0Vgu5RAiHAgcQXlb0L8fEdG5f\\\\\\\/PcXUpx1SSwLYYsW5FF5gB+Y4BsPZ4DQX\\\\\\\/IH9H2awL0QghFL\\\\\\\/3JwXe6lMfslOQ4fOLrrRcRO8taB3af3Bz2WhCJjsO1dlp3TrMCkU6Kbw0pdz6jyIYxBV0I9hW5a9M\\\\\\\/8Jv1DwBA3brrDhLfaH\\\\\\\/10sPfev8mwrvVP3\\\\\\\/C3Kdn4E2TGi+aRVgEE+fQbkQAHnR0r\\\\\\\/RmLE3BQeiB1uo90+uO0DeihecdpRCMAC+CShifGZJlv21rTpQ51LOa8GJkkd231EYdsM3mCZ6nOlQ4KEv09PkUTtvTgLTh+hz78tVY9GAgCPSFtgaWlIM+g6osQDtMMwUsdiFGgVLwmBiENV+JVL05XiJxMNMwdAT0cSLb4Xf3uoXflmYSYWKVFzfyfi+7p9qRUi3hUu8bDhn6+6xfB5HD9o641Gk06M\\\\\\\/OGjBM5OFnpeBY9AQLoMJIztv63tjDX4O+2\\\\\\\/sziDjjQBN2tjoM3b9B70IRUjvSvfp7F7SGUd1CyF+2xslml\\\\\\\/l6jm43ByW3W9yzCcDkrZB6V5YFfk36FAOqVq\\\\\\\/L\\\\\\\/xkNVVw7oxupAEHzxGn31fxALkQE33+m2tqauMDvDJNtaMhB1UPBvJAlHs4V7OGGOiQmlINP1T5WS\\\\\\\/rwdQ5Fw+tNPZqb1ArrTDVJY3RqwjcW3o3O3wLdVBWiQ8dtfcgSe633s8OYgQqNtFdlU1GaT\\\\\\\/kF\\\\\\\/p\\\\\\\/kupR8Ul\\\\\\\/DXEWzCa4E00oLedsoOrvJ3sK1n92ejcTb1KL75afqUPMqWtgn9FEBWj0I8GxYB3thzvfysBCtEOsPA009BzhpJE4HzMvG8AqpIKiwQSf8FoLlOLHILxeMEeUaGeCJo7x3TNzNdjQIaJovSCuTGLY7YRk6BxN+XtW7IkrwLxilBcwLAIrqs7j5mY7Pg\\\\\\\/K1YSnv0hkzZ\\\\\\\/AJ3y6BL1BjMIZLXy\\\\\\\/vOeXrHuR8dVYuwY8uFg4Pnr9Utwd93j4IM986dztuFmO2159v7mofhsk++y1nEcuG0gjSfeZ3XSUCW8\\\\\\\/5bbPdQknzAhG3gp6gV4Ey\\\\\\\/TmUrcrjDekYuBNa0AMAce3rrAU9TQjbvMubogj1aV1YeVsxfz3tv1hpfkHKhbOfspgi4zbGxCmXItUVnKMHXXgU5maiQHYNrWhtCb49C+aMPcFvwpfkZydicdOlBHaTuSrV+c\\\\\\\/LR3PdYQZ6oasUlfxvCez693qrZxQV9Z0UDdoQBp0oux6QI6Lul05b3AY1vtl4Ur38LFzTbg7BI+TGQIE56PsGmHSyUb5wh2oMe5K6IisR8zD4+yOtteL82SkcDic4aHigYrw0Pxy4P6CAORMr0YpIRI9hVA2TwJLk3+2dXSCDhCPF+RW71Lj8EIWcjJj6LzH6xb9nl0wb2w2dy8g5ahY6FEQunYNBC\\\\\\\/5iniQfXNwg3TcN8fAiXthupPUUMVekcQbwJ7QtB3V7cC7GR4PUPA\\\\\\\/OQ1F4HGG0FqKUDxusskCVpZlNgG0\\\\\\\/LFunuX4HAnFf5p3HKEQT0G+t3x32uSb9WZz4Yd9cc0Fn9gjHpC1VxjoMqAC3XeorIJoQxk1\\\\\\\/pn8ZzwUOJjmpdOOy+RQj8pnmwrLBJ1XXyiyOpbQ4e309sOl9d7zWQfOXgDcqcfWEzhGQ2Rd5W0XWsalacdqVj0+p8\\\\\\\/c\\\\\\\/3NQImyjTYhe8CHw+JeNTG2zwtAwP4Jog7asiDRkAXeFOZga8f21Kerjq8amArDVkcbT5DkLRFNHXrniIZwXlD2mjmZYecjbJvYHflZhcEv7iNlmgSFVJlaUxdN6RtCri7WsLz8PlKN+l0xeMu6kaH2a2GPhmDtkXvcE4DCu8Sd67lcqE+TsIeg8\\\\\\\/7Q0J7Z5Dh1JSEZXvX2DSWKzltifU5CVmwoG1IMG4NMd72m1Bi2vtaiSBftptx2CQpSrbgqPVUbcYxViriem2B1RKO96z0iv6lpJ40G3XpB6Bgkyv19bydhiQRGso7Gk6f0PACREbynefycbRjKlo2Et1kJK6I1teRG4tiXd1bAv+Tc4n\\\\\\\/ZzFAeQwMMocqRZ0lSW\\\\\\\/bloUT4XNLlp2PbB6p87J1u8T2Hew1PJfwr5iUq4rSs519tc\\\\\\\/U+\\\\\\\/SPVq5tZ5kznE46NcjANf32ubJF6cQiOQIIj+u4LI4QF+BhIxysx1Tr39qH6Kz\\\\\\\/lhmqMoVABj0P\\\\\\\/49wnOwa3b4YDHLyd4RsV2OIaRhyC084iUIAPYF5RfTd+k0ZAbaY8qdIXDnAakqP3C15wZ+WFIfKF8dZQgtVQr1nfAwkwiOXcliGzcpRCQ02AwCLVEJQvV8we3yN1UEwxvQqQbMcVlywZ5agTMObkW3rVjxCWn4IOaIewNdXQso1mog\\\\\\\/22q3hlhr7pHDx6tWTZ27xh1XP5ZC1131WZl5A88qq6UIIOGkcFX7v7Od9QjUu4c1x8zKeJkSqmVKzYLHFEBTjQEo2twE6PV44Mh\\\\\\\/hyK+N9igatWYwfbqYtr5huva1A6qkJMS+DZoj+OV+nAz2DuA4i7Bqe5Qj6DaFO\\\\\\\/EbbiMHqpP9beLnImLex2kwbCI0nYy729lqI7cdMaguhi5pPxJUoFwWdo7EaLxIj2w2vCxE6lbS2RzHucClyB2xD+GxSOK7od5Y9IXsuJS6v5o6WHtCeDxV7NdNpwy7VO4XuSFWnOtim4imrb4ZRNISiVzzTe5tnwQdVk6g8Oej8+8ut6ubXWY3\\\\\\\/GYmY3rTzck35Cwf3yW3wqhn3EGxbif23p9u2mrJX\\\\\\\/uKc4dVg9JYOajK1RSU93C+31japT0LBgOKhYm+V4QCo4yU3+3IWXQ4idTn0OGNetPhbFg1UT9vqV5nn\\\\\\\/SfYYgTKVDtudJpEwemopszFljTJWAzzWnyQHybU0knmzwlW6SDEVIiXKtG2B0UpiklgggJuxha8Xe6QuEhD\\\\\\\/vZKWVP93KmUvjO74TrlkuVp0EpFRxoVeD9bC43hZLTD1jELgZ\\\\\\\/8rvvtK\\\\\\\/sY0Tol+6wdQU1qj22NbxtEIGvFfKKpd8A1gjqvTiVbijevIbtcnPobqZcDL65viaB9wF2zi5zpOl5e0GgMixEcM8TN2SPaT3G\\\\\\\/vBzDkfX3apwRz4MWD8ZMi6MbOpMoXajSMdEmP8rWSSJasTSmCHjwnIkLdZNovWAxCQ6SRZ1QwN9f1+A+9jNhyv5aZhIZj0lMZXLqK6pSs4JCVS69uEv9c5JZOD4mzcFSiwlWzmX79Y6FSUU6Th6exm1Zth8GhZ78qJ\\\\\\\/VnRep9kZHNHRtmI1kJo80oFEuGFH8G7V2zeFqHAvTqLxqvUaXirvP\\\\\\\/GgVHqqepnBQRsZVQFvLPHkCNJeS0Z1Hxb91Qois+ATyfVmbhDW5OkiCJsGp3a4LfaLitzMAu\\\\\\\/lGJxKl50qoNZOvSkWH9JayQ6Z6GV8hgPJlp10llQYHKOXQKwWYXuI4kRybgrwE1YHBlHyZa0H4nzhA1DNeNc4AIom48YCfAXlO2\\\\\\\/W+7dVTrWYKbsDuzDm4AmNnXCFS5+kGI5jWvQ+5WIHE1rw0kynsceMNjleCWLNeTq6fXfWC\\\\\\\/h465PQGTtlQb5TcDFwhD9qIELMtjYYQBz5miJ2AgsyPpO1+cdBEUDL2Th+HSMIdAKZU6IqrD5iYPi\\\\\\\/FO7O6rLV9Wm7xAdseL71ld6QgQ0kp6nwDAE2rHq4i+zHjbipqMmjPsm7GT8+gu6gdx15OGrf787mnUMlvZ3VdSfDl7XY\\\\\\\/d\\\\\\\/zSUZYjaSLbhYb5kXFwfMOtAxBYgD7v5+B73Ivc3Onh75Szsu9a\\\\\\\/DLqHhq6oYqSGBewZmTpUq5apnPc+yzdyYj8DXPYmhWXQbFaO5\\\\\\\/96Z0FM3C2nIJVYgi\\\\\\\/h77KKJxk8t8xxR2ACaw4G4zfn2Cz1PKFk7cDxJRwc\\\\\\\/AkT4GjreE54S\\\\\\\/2dGY9z3A6UDggkSJ9e3n+kRyNvnM8AAcv44oU+\\\\\\\/hCRT85TWQNPkkFwTutvx4MzX8VhRlXG6i3nP9GKE92AX5hE+wX2kL5o56Mea83nE3xAhPpAaardSH\\\\\\\/IthxYXsovUVu2fmxuIW\\\\\\\/dR3vsr5CNtWxI5qY8mlR7+603jw64UU0xpiOq\\\\\\\/yi4cNnaB2IOroZ8KFoyVwlKl3ZtNLE6gCmF9RvyyrwHevXv6iVQQ4ixCGN6VgMsIqzfvWGV6teE82wEn8Rvv5iruAp8oTTUboUMdteITioXH4GRVCvHDuRVqxt1e20\\\\\\\/2BaLFJmFCQqMzsonzTsfVljjKQstpj+3JKNYCcMdwXwUIp8HMvBEEmZNjVKX4nf1lPK6ryhJrbO\\\\\\\/8qPOmGfkMU6ibMc9XOePeTe+IkXfewqsXHX3GD23o8lZq9kAlO75agQ+\\\\\\\/Kg3zAyKqhxrcnGlDvk0hlw7UDfKHaFtwsYH9\\\\\\\/tNjT8tpVcpmrdcgFgE6mWWAVSU9UeIwxCwkpTk8fLqPMIpS\\\\\\\/X90NTmDoKO7M0RYPc3J1brNc5KMqC0cLbLSXcoER5AWxNZALojnaTz+s\\\\\\\/N5\\\\\\\/W0i7DubFdNORX2F5gYmRvNkUpEluwlBRy\\\\\\\/NTppE5A1XozE6xWEpuDVng3pMV47oWcjawoufTlNVvEStNd0QiOI7yFo1uvEPmVvFtLBBzj3addhZDp\\\\\\\/tiaE2jBzNC658UhoWCn0vFpZllWA3WU5tWxy+yHnKOS0ShEm1OlM3n6KxBdRJyANOp6yxUrLgaP0vG4RJFBVFMoBKobxH49tBS+WxnntQa6JwZyPT2M4AgGJBOANUx9Ii9xLRBXgHjHFfNhaaN+l1cgDa\\\\\\\/ESX8GtoWVhvZlOq7nJEajOXddfBXV8QyQ6RFx1Yq8mz1ch1g7JoI9Ar2BWHmpuNT02qwiI1loaSQg98UploTVvv9HqJ\\\\\\\/NRVZlOMM2wUm6xYqrAQ5+sk2Uh2Jm1Fk+HYQ0JxsIogMhzSoqL\\\\\\\/k7TeUpb30xHaQ+Jmb5SUjbfaoxwkU+ytFEdbc0HdM2cCWY78pdvEtY9DK9HHkRfvbYNVzH6DWduDq8r+bvRr9G6TlPPIQOJFrPUebUzUUwf5nN+AhszT53e6WAAq\\\\\\\/WPs1AGtS6+uCISvZpM2mKuuQtIfT\\\\\\\/d6Wz3WdNxxkFfBeAjQw95tzZpXOHiPdQ+11x+5Vq\\\\\\\/GHZCY23giuWPzi3VExhYzXh2EebQodaBQQ8ZYmNspDa6jl\\\\\\\/dg\\\\\\\/vTjPARfJshnIVEwj01oWdTpUPcQSuCVdGNJp2RgPsnjY9W\\\\\\\/MSyRlEmu0nCO2eQHwTUaYGkICqU1b+KojzuTibQQ5MGCDYtCjC2Ny8DBKbHMhhHyuYKbmZLKOYvPeENTCDaR7f94EJGlKAz2NFWhHOjXVOxepgeAjwpP9m\\\\\\\/8JMnb2XrXEkM7CP0owggqo93SqL8D4FVwr2m8e3qk5dMN5I9P0BYZ2ez6wkC9+oKiPbeBPxecFFpi9enxWs2Dxf27TGpCa1oe5tFFqqX\\\\\\\/ZvTZZ9ADUIHn1ADeFi+sPn87ZwNdmKMMdEl3NJp4l6SOuTQ1qNhqATreQzDJJEE3JVQtjaAVgXo+cbhDCW6J42vEZFlBD6zR0gSGjj3XtZO+35jQOZbcS13nz0\\\\\\\/Coe5CvfQ7WK0X7sh1Mjacu8Domk\\\\\\\/Q9aw3aXh\\\\\\\/Ta0OSFLe+5rUS0kuQDN+O+1PMexWe1TlYkr6M1K\\\\\\\/n1WBufGE+hAYxRZY30\\\\\\\/3g0lV5Bfp5HCc4G\\\\\\\/VMafir1UEx8wsqLc5eAbE0uE+sv3s\\\\\\\/gf9N6XCt0OP1H0sJv3kdvMme3t1buiJltTxfzLSS9KJmsTNTD7BxDfnPKktEgJoFbPf4fe0beaFxwyke8KZYMeJgIM1U0\\\\\\\/iJ12dXr\\\\\\\/+nozF+Ihxo+kTTpXVPrLsc\\\\\\\/BW88cQ3HmV6CyfWMUKm3mxrVeDOJQkqfpdqA4rySm4IiNm00ndAf47MQdfxboSVjKZfME1yRHdioasjrCCBr51fWrifKqiKFSMRDuLyZ7FWYqA2xDYQ+YaPYzR1KJ0TAyK\\\\\\\/SNLLCpIJNcf1EmDjNZNfvqlouUjSte\\\\\\\/qbBWsPTv6JJRjstnUMkdEnj5ZByBUCUJq+4bGTSy7QbKHRSj5fs10p2GdY\\\\\\\/MIr78neZcytKIYd5NZCuS\\\\\\\/HCtTnLIzP0lU9izyceJkh+t38d0l6RNX2ECMdDNstdP7bfXxAuJdQ2kukZXpwPWjtLHNh+nOahFSILwg6bUGA+uKe113QXl4r4fo8CoKETw0svU9QcMNMFl6CMFCkSK1jxrlcZieUyOs+RnXf9FERn6uJDtOqUjkyKg64\\\\\\\/p5XC0p9efUP1n67X7NfMDDh5nSXC8uNxifXaH2pP0ReO589RMCrf9cqdx627QiZTNjjbQTGr4R09Q55fyjYii8hCI\\\\\\\/4S5o\\\\\\\/JCwW48KemgErkxsoNgSSrOGOnMigMCKA\\\\\\\/9FgOIFIRw3rcYqNs4xGKixuX\\\\\\\/AhmpmqWGUz5uag\\\\\\\/SZRkKwgvPeIW3Ywcx5pPmSqdKuQqEiwD2QkTe7pcHI9eNcPkUzBmUkL4TVKcYf9wv51IvkwPywM0sq5vnIKNwfhwmUph1YG1GNWgqqdZQTTc03LbuwAyzrR7tvqK1ARGisu0phsh\\\\\\\/LndJKQiKEVdkVAKVOugKMpaZYml1yxtU4L8MiE3kF9Z23cI18jU7D8uRBMoEAHXYiFK9vVdsypdQCRR4nPMfK3g8iBaOsKG2VGLJGTXu4vNQbbe2bS73v7YWza3py3LIzBZMghTWJRZtzxzkv3zkK3q7WJR5+SVmCqjyk6dQVRbil2H1851Bwt3c3E6bYU8F7BDK4yYTuGjWkHsBU0a1\\\\\\\/F21vGgGag7thmKvuGYprfOOCdW7w4NRMdddgaffhtpufS+E4v4nH7a9cNgTeEahmlxvkxXaS+5mXvteeAbRWePPOJSJh4vGExIZcgEcIXMTnDtCj+YUhSU7VhwMnGQ1ukapxsj2SXd4SlfoUlDt+V75QPOOzbIEMHhfq7YZ9\\\\\\\/JfZTImapwiz2Q04eVv0Awa+QDOIOIKwrBeo0mWizitJIGwLKUFqs7uMnMU06tjvNP4vO4CLrmeP24sUStGzN9hkXiI5P1Xt0wPIQS8kiujkJOwO06iCb+gOffns3RyMsp2sjnq\\\\\\\/Jac\\\\\\\/SozRRmnyk\\\\\\\/Slo35e6EbOH7SBwVwI4w8+0QprvOxUJ8lbBvAekJ8vsWma2+xhCvpqXwrTOgFBn9U6mp9uhIz1ZoM3oqw2I3fSYB3GcdwEfqBz0zaDr9bABhwUOuMABLSjR2W35yDKzbc5MJ9GfKWW9O0\\\\\\\/0O\\\\\\\/lS6IvL4ZwbBzzfFJBS+qNOjkdzu7nlTuTTBCnm+xxvIPngdsiHOO7WGB2j2MTBJNL7uH+4tLIpkMqfvDvKW9nRFjV9zsGs7VF5nwx1vlezIMyMfm4\\\\\\\/cs8p7AbBOxSlWbGTbp26RZB4tAVlJZ14nNLaLz7Mjz8R6FKPuKZtgDsAm\\\\\\\/bVa+H8XmCJuhhxshgTQYbXFMGNq7br2RNbchjHaOnAheVmrgnyc2qHm6nv5ZIUze\\\\\\\/cHDVAWjHMPKmkFyTuNC2wsDEesJs+sZkBg4eO4gxiF+CqMBNbZCemtNL+Cdg3EphKuJyeLN+PGaVI74hkZo4wHVI5gFbWR4fYRPPha3dYA8HCBoJhBCugxgEfEgu7WvBSKOLBSf7uOG1U\\\\\\\/kUzPyAkXHErI5T72aZhsQUXgotP5HMuOE835Q0eMpzD9bdY23vfZlr1lp+Ij\\\\\\\/7ljnQubPz8RCGXlnCd5t7httr+qeB9lQqFaVID\\\\\\\/xKSqGmg7MkNXMYxRA\\\\\\\/HAwrI\\\\\\\/zoPjjTgQkiBCuv4QyYzHt1cQdl79ZuiIyDDRr3ThITqlspUa3dwHqpxnH0d3Og3WK0o9MerbXgUOsAaSD7xAQMyb\\\\\\\/4x49FUYxYUD3V2zZIxlHQmoIByyeFRmeTUa9LJAOJTlijUdugYXrU1ir914u1SYlbc2LKe8+udlyRSFTckWeCn9TGjR8UBk82fHq1Ehm8Bg4NURCZDxEX\\\\\\\/RRnA1NttNXg9wHbzS8et0EvUS1CH\\\\\\\/jOW5y\\\\\\\/bSUpD4gY6oB8giBG3C3IGikve0P0yqQOwA9nyeIxKkLAWv7WM\\\\\\\/N+8lZmqod8OsOaGdajcn9K44+ibLeWGiRSZeWtvxeJO7eTss7ofLvyyFN7vtE\\\\\\\/2W7jrxxMfrA7zgraYMwgeNxTlqTX8ZeCybVLgQu8OGxN9t6T2CP+nbYokkAICjVuxQp6KIow1vN+ElVNT+7gOd6yyPiu37mrjI\\\\\\\/AUHfFlmXc6Zsz7qAPU+vZHeiwZsHuSSy\\\\\\\/86BQp698iHjRkdYScvAjGfch7QHp8P5Emkp4v0XG+4DG1FwPA+D4yakLkBFSPdb+DeSADYL6QvVE7Y4YBQvH+9m6zNYQeAVEMLc03lUdJWRnUocNSfAmH\\\\\\\/ZyF43gRKXrYRGKl2fJWM0JqGcWP\\\\\\\/4H1h6jvpXsh9u1jDs\\\\\\\/0BxzrN+JZoCiq52AoOvqD8oBa3jgQ\\\\\\\/2ISf5Ueb533el4nMeDXWgY6i11\\\\\\\/G\\\\\\\/aW0zIn\\\\\\\/PCEBKgo1cb4IC2l1MdoMZU+\\\\\\\/Em+xle1N5GmjcSoV09SE2doSz+9\\\\\\\/a2VUHguJlGiEdgDteH8Z+aSmtdwA52r73O+PmlqJ07G\\\\\\\/dAGS6dIWIf7y\\\\\\\/jEXKsy3NRmr\\\\\\\/DLhhovveN5pwHoz94JhfxdcwK9bCSAspwBS9g7ctW7RYAqGfbysGBjV1oePYgmKvpZ01ggahxaghLc2oL9UQobON86TqhLZT7hO91yfdKlHQVyNni3Cs9VLDmMFsedO0VZcqYyrmp\\\\\\\/X2PX8hNvzbZC72UU4w6WBgF3iqKQzIY7qqsDMu1tMuoaDYh7vs2HE4wvUzmMk2BOOhPV7NUc0ul8B3u+F0xvVji7Q9RjNL\\\\\\\/bRuQoxzv+UvzcvxHRANs9+\\\\\\\/a0ZeiNOTvQrvO\\\\\\\/EPwYMuNCERVfN+x8JCg51c6\\\\\\\/GKBJDJkkEiEHp\\\\\\\/z4IrSx+iTR+2TqNMKQHI3nTidswNTgclvO5A3yt15f4REZOICGLrHL9etQS+KztYEawoXUY\\\\\\\/J6S63kkXaH4AT2V6jnElsBZ4WQ3iLTpAe4D3P2ykvFpD\\\\\\\/ha9N+etYETjGRQvrkIfHyQJ9rrXQH7rB4sudehCS6Je0N9LNZo462riiijlITNzNO+mRYcMi0bgzW2D4tdQGW0XhJsvLg1FgdNuGMv0FbHDUakfnLqEMoIDrgZpl9rgt5m+aymHCdu9ohI45rLHZ\\\\\\\/GlUFld4PHhem5zINSbDw1d\\\\\\\/alqxggYAAsRvS4lMZqJB5ujypy7jj2rRWQ6pt1LixWM0\\\\\\\/z3zEgFAENbxVdd7L5BIFNK8AvUBsNt+iNDWndWuKDkGPmx6S67D\\\\\\\/1DIihgv\\\\\\\/r\\\\\\\/4s4EwpQUZnZP0yqpNxRXWpT207veiF5BEOmUXbJJ+IXwimhV5AA0Ny\\\\\\\/oOG1E4dR5S21P0N2xC9l75CJw4HR8ah2zaKqXLpUMWEch9LLahzpm+DUSyqbNEmqte7l5qIqfuFPz7jz4bRlYTVeu1ELISw7RPQbWNzl6u8b0ryKStl35Z5nn3bNVkfI74ZyZzxDjr20qN0XHUFkvtupMFmg5cKcYxHw7ecMdBtMB6YZdU+eP8O0ux2fGUSTkMqe2TGxkLRrcB1T6VTXTryXPnbJD4mcy6aF6m2AqXvuAzK1A\\\\\\\/9wtMrfJe5fPjo4CwgSCQepu5GYDKjQattQ64i80hIN9jj4ozxp5nmIYI58488aJiymDGIyJXBStEjCAGBXKhfPJwRA1kdct\\\\\\\/ECYD\\\\\\\/meRO1C3ujH5Hj\\\\\\\/4qXGikN+rl1g951x31RYiYr4+O9jVvs1sdqwHihaZk+Qp4yJdYBLJOf+S4basQ9mzAFH6SAavAOo1p+As+FT18FmnGgLIjz0PROP5UKRzHPRe0aJVWHVMmLuuqlvgSWfuSeL87IED0+72tv0tjlPzBqmEwRzLyJGxGvbSNhhKGWXGXNQQMEUpa55g33Sq\\\\\\\/rmKG6jnCf53bDB9jOLjfLaXW+7uIY0sqITFSF1eni9m1PQnFLuG6\\\\\\\/ZBOvJ0x7vUaoJ6V4WGf+Adem67wbOA\\\\\\\/FvJGXrqZ39GIX9zsVOrsnYYeEWPEwK23TjaXyAH7CKb6Mlfe0bcaB9AWkWT4G\\\\\\\/feP4fxoj1NGrNnlI5MT+fduCeBMmt69oasRAAYPbf9OLxCIZLdA+oGtjESNmml9Tsy3dANCOOv6z1Ed033\\\\\\\/unriZNDXSqFjvqlJWDchFvZJYkStOmkrzxllUHDO9auNqb9406Uni3wHYj7eJ9ibwbQuXWUEPQwcHOGEdQWtCINIRhvwDc2aBi9BU2Xh4f4fsvtB3naNLnzeKu5T+JXWndmXuAzC1D+3i7HEMe+QWz0Q4GxcMb9D6iDSAlOeglZYkTaA9tRvgw8dKHPjlN8zAdsi3idfL9W+n1nXBQ58IKJw7Nm8wOoi6wzPM1OkYrHud0LRgQhWEZyH9AFxXZeW6VTSxJTVSW8K3Zdj3KJ+4wKerSGkwHRlp\\\\\\\/xBo39zi9JfVw3BnfNkyK3x7GpK+Okx1SLLQynckuKBTQ+4OOTP2JVo76cLF4\\\\\\\/1w\\\\\\\/oYlx5Epsd8Whtr4+mCNlZ1sCoc2+\\\\\\\/JcZtVdxJ1srnkZG9UWEiMgcoIPXRNrLFAqYkh3gXW+8VdwroingSAE2Aff23Bc3TbkwCQ322\\\\\\\/Ko98nTE+QLxcqCkBR265qqRkUz1LKXTtPGbfzXRLQwE0Wrk2puKetIhZwZXurZnhwaFKtK1erFf\\\\\\\/49xsfwQZqqkrOxne7rK2Im3PgRNXbARTEaPLCCzDeXuBtJkFG1AYUVNGCRX9EOWkHqJBW7PEjmoVs1UAickD1SniPuoIPztrlEqM\\\\\\\/jV0moKh5fE4o3zKftVkMiYRkH2tG1PoVbIGxbW0Bg4tLmJgtxEBait8hSKr5Ea+HpgXzwFwttjAQB8zcbk67DVwuf0Gjg+hmYrH\\\\\\\/ZbYB53dSBoLaL+aELhrvHjcLbBgitFZjKUps1HGtRGIpF1TeTqE2YMeSSwvy\\\\\\\/VE87sY6QRsC+2iP39SnCVKZfVtLZJXztZzbZ48wV9oWJ69LMGRa\\\\\\\/o5xAeK28xYzEt0aAJ4nkOj8C9pVJD8rVc3pZqkSHiflOqg1bTLuexOnhatYvjwlw+lFvLqgrCjeCus95dSfFCRI6YnDeXJHUTdvODaGCSVmZ7e8NKbYcZwwb+0TxNpvpV9b7rKf5rAl4TiOV6J9ZJQt0Yj99d3h3VCGcpSL4woIm1FXu33JfbaZ85gt9amMj6wuWT0YmoRI1qfN5hlQutBpcpdhzCC9opX28jS9F2EyerCkj6bIhCdflubpP0+16++MH5KZBYnxq2WGyz3c2MmzA3hJiraG1roOzRyb+ywD2m88X5RQun+CjZNpJTepmlqwXHfcSk2JwbtEQLuF5MWZnEsHxf\\\\\\\/q3DDkzMa\\\\\\\/oxJPLdysIqY2nnu0+PfllsbvRoM3AVMNQzNeHNj2Qj7\\\\\\\/YFztmFmvRcS89jfQg30a5wb02ftfAX6diHmmFHZxvLxt1WrrLZSzFbpNrjY9fDhCQarrGTQFSixYr1YZlaIISJt+9HLDQdPuGvI6OVIZjgkqoWpiPFYvRkOqI2LqdC0JxybnX8B4\\\\\\\/o32BZ\\\\\\\/OYHLYEfhDsr93w89pMqd783+EPZNykvKteVY3T6G4xyhjF6cEhb8mBExQTNjgxgaIT7F7kDMiK4kM\\\\\\\/NnmkH95585KMBsgO69Cuhni6ewya6Jmxpo8P+sSrEuS1y62VM+Mw9Hyq8XBwtT9RsCugcwsaqzoObh6xjUjMIX7gEK\\\\\\\/LG+UiP\\\\\\\/aDfhmaiHd3IzHH91u3u0GV1HO6nMgbQ4z1HbnvffFw+2Q\\\\\\\/1alvJ5+eMQZNb2X81IR+gLPxpOVQxwSIm0eozVSPbH85AN4OW2teZv1DUs9z0YL3bAf+ghJUZfoxYNCGEDgARspoxpUNIyt6px1ilEZY2mo3KrQraUZkMHi\\\\\\\/peySOfub4Eu2xQCQPx7ssYRuO7IjMXevLi3oyiRQA+uR2jXVAZna1brvGQ2CZatgddSYZSmJPquRD5lDN1VHtcTXOJwBbcXTJgn0KNDPuwHeR48VDeqY0BUNnt4NY5QG\\\\\\\/ait2kPh+v\\\\\\\/01CbvTe9uj5MxlmS0QdzWHp2jKyzl0p48XZaczMQJg9Zns3E+i0u3YAZAhFcDeYbF0h95DGDxKlGno7bDgzMN50C\\\\\\\/BubwMjvVt\\\\\\\/YjPS18i2AkFVp4vwziJYLrbmBLjZRgqaWwM093j5aiRJb6VtlSqQLtstlvr8h861becKmHOduO90mb03YJ7qgbBPtWqXngAR4S6amD8J8xCJRbWbaUf1ygWsQRXHDcjHQJt\\\\\\\/A0JcFN8R3JhU+keWEa8rovVm7bvRZ7Y\\\\\\\/V8Tyv2jgCJn8Y9ZX3MrdkSF4u1g89liNOgV1VmWlIrxYRi38HQPc4ieASy1b0roI\\\\\\\/V+P+290mC8DTjMdMvMJd6accmy9B9Z9Jb0SsSFLONVUZgS69F4\\\\\\\/t4Ki5D9vYirPaF3M4N5+z4WzPqhUWJf8sGPkXWTyr6We7Ak\\\\\\\/7s4rOFEUeJSeHaVM8NlBYG9kv3F1xg3QVg==\\\u0022,\\\u0022iv\\\u0022:\\\u0022j3AOMsf61OxYDMJF0wbAjA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002295f3bc2479c05bd805b2af72c2fe6e18bb2cac5a9bf0217a10765d0ac5efbb4d\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022OkxuKUOO+wYr8N0cUd+R3gzTaFlzvCObSkti9NIbHEqBzPHPWCQsUrzOnqSRxruWa\\\\\\\/Rcmh8rgcEOLfWkJAN9777PQeUlLgWMv\\\\\\\/S4gYmIH\\\\\\\/VxTbRpH8HTNS1+5wSa9lERqv27WSKtzMV8cQrgO57U2EHK\\\\\\\/8Nm8BiQL8xoeo4LCjBTuGIx19e\\\\\\\/XqYCg5nqvHq3K\\\\\\\/yR+K0g810G+fWbUW13XOpWAtHpT9E5y5W3EbgYrNw4cvnM0QEdqO5NrKeuNoM9oI2q4Frsjbu\\\\\\\/zAWdNkrskRTmhr5N\\\\\\\/Mb\\\\\\\/7mkROoiUdmCVf86ASRe5obGb0MfAx0ItLs600i+Wx\\\\\\\/2HhlfcXHvzxNPhlGCTfLPGSSwhy814lvfBNwFc1i0z58CKC6ECMtG4S+eP98A1MRkXq5GJiKqy4VOxu6vh6J7K51PWTBfsFPjbrhVt5jOiW01vbefG8pCU5+jQ2oEjEBqHIuJt0KigsT6mNXqN9m4wY24oSIWe8fwtn17aJQ1ketfdZdlulrNr\\\\\\\/ETlex8ChvoXme1VVdZSRZV0FQqeK4UTPsNeOpcKDUmlTKzwe\\\\\\\/P\\\\\\\/xiFr+1iqv2E4BTTmetpUD4BZJ3CwOZqg4A1ege5xzQNaaqBVjzdnqxDPLgRQyrtjJxbpxXJbUzPyfe1Yg3Biga2UyXFDOCR\\\\\\\/zvQ\\\\\\\/IbxpiOG6kK72VoiTJ8b+1TRYuiidiGnRJFSNU0fEDChFieanylRp8lX7bV0ODUHzX6oOmBDcooPuHihhmeMJMgMeng54pO2ep5p3Zc5gN9goY6ot6MI\\\\\\\/ZzhtqpeflHUQNbaNiV3PVlPQh97shNOcNBm03saWme94JqmWJFynTkJPjCiqFGWBn0jxc6qjO6B0AiS5rYBWOOsBwp\\\\\\\/05CzSry4R2r9C7iGZFFIVdRYzN72oPGf0tmg1ATiopVckBqLTJXuu\\\\\\\/fjUy08i6Y8e+PLfu48BwAQE83h4ZVHzy+fLPjkUBtjuoJEBvB+AUrezwgnEPthv3GGM34upU0Yd7Sboa99IN5rGNVP7WE6fSEk22W\\\\\\\/0CLJ43H9698+b4D3FT7akbEUHjStK2lykpokWJevwZIH3uXWl3yeYt04ojV4omYLCa5tMNvjU6YneSMdYC2Xuda1ZVdhk9\\\\\\\/jS3THOH1Hmo\\\\\\\/lxgaC93JKtM+Uv+ukb9oopZXAYxF8f4iaDJ4ykiXDN9fksn\\\\\\\/5nRHMB2zZF7no\\\\\\\/q1zcyLcFb8YWmn1EFU2p3DHP82Ru8kZSW\\\\\\\/FO4yhHEEMFM\\\\\\\/n3qI4MgV4DknS7ddC7pPq+LoIsnlDWADfQOEW48cohvJnYojcInAUS+ATYPbzcYI9ql5rpB3oz9xcUDPGOpD\\\\\\\/B\\\\\\\/6moZlvRHH86ULpZpO1ghKNYVOd6aL\\\\\\\/uKyFL0E2K5jPl+WE8Vu2GboGa0KRLFc8rOQgt9iePQ4H1POU9\\\\\\\/j0xrhfhXjQsLUc7c708DKofKC1q1NawcNvh\\\\\\\/kLLMtNR1kGtfHhs5uN8Ayhub60VIsxUndmBqCyEeZeS7HHMYbU9Squf\\\\\\\/Ex1brH3\\\\\\\/PADkjjUExamJzbMBxQWe6Qb+ZNHQ\\\\\\\/4GYlUvmenZbOdIh5h\\\\\\\/fb3ltkMLqUt05Xw8F33J2ywxXdAzzKIIFgrkrRUtUF1sAxOIrgWKmvsa05TroW9wmku2Wyf5Gn\\\\\\\/LxuIiV55ZQFNowf94pEYiTWnc2GkI2iV7ZmAwqU7eucrV9FfydIKDwge8moxamAlu7sCvdNqe0Wj0lPATytCwTDo5laUlBa+kzO4ziblHLy5IwS+x+03\\\\\\\/iDwIWsDul87QgRZixyNVjDjeKRsyWOwhnajaWyFmg71cZLAN5A3MRNlR2SaL2eESLIK44xrvoe8Nn7WTZ9mDDno3ZBHs2EXH6brlO9EjQ9ZJnPwEyD1PaJf8Z4KdyKZuBpmFoXNUxNkPOce6IzcrPijKxJJ6jCtdSAJb36ggXNoAaSR\\\\\\\/tUe7oHcAxXiWLqScvXgq3IHsMU5LPmWTT\\\\\\\/jRRM3gWMZmFbwJT9TZsg5\\\\\\\/OScpB1DMQayn6ZjGMdPZuOJAJg==\\\u0022,\\\u0022iv\\\u0022:\\\u0022zAub5cRl9jl6+RYf+1fimw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022c0b3d76f237ca3dc707afab8125601c9f816632d6f1b0ea0519485cb1300bdab\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-494276 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/10/3273" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-vSZFgGAmmXeM9BAyHyxEteI-PwgpfkVhiLGtGga9hzQ" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/10/3273" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-4ZQ4xQ3vBtQ31YE5dinVaW1_DkQUsMJLOruKCB3EMKw" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/10/3263" title="De Novo Genome Assembly and Comparative Genomics of the Barley Leaf Rust Pathogen Puccinia hordei Identifies Candidates for Three Avirulence Genes" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/10/3287" title="Comparative Transcriptomic Analysis Reveals Molecular Profiles of Central Nervous System in Maternal Diapause Induction of Locusta migratoria" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494276" id="node494276" data-pisa="ggg;9/10/3273" data-pisa-master="ggg;g3.119.400562" data-apath="/ggg/9/10/3273.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">IslandCafe: Compositional Anomaly and Feature Enrichment Assessment for Delineation of Genomic Islands</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-2486-5696" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Mehul Jani</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-6874-6146" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Rajeev K. Azad</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3273-3285;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400562 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494276" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Mehul Jani </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biological Sciences and BioDiscovery Institute and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mehul%2BJani%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jani%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMehul%2BJani%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-2486-5696" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Mehul Jani</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Rajeev K. Azad </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biological Sciences and BioDiscovery Institute and</div><div class='author-affiliation'><span class='nlm-sup'>â </span>Department of Mathematics, University of North Texas, Denton, Texas 76203</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rajeev%2BK.%2BAzad%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Azad%20RK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARajeev%2BK.%2BAzad%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-6874-6146" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Rajeev K. Azad</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:Rajeev.Azad@unt.edu" class="" data-icon-position="" data-hide-link-title="0">Rajeev.Azad@unt.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/9/10/3273" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:494276" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:494276/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/10/3273.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:494276" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:494276/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/10/3273.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:494276" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:494276/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/9/10/3273.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:494276" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:494276/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">One of the evolutionary forces driving bacterial genome evolution is the acquisition of clusters of genes through horizontal gene transfer (HGT). These genomic islands may confer adaptive advantages to the recipient bacteria, such as, the ability to thwart antibiotics, become virulent or hypervirulent, or acquire novel metabolic traits. Methods for detecting genomic islands either search for markers or features typical of islands or examine anomaly in oligonucleotide composition against the genome background. The former tends to underestimate, missing islands that have the markers either lost or degraded, while the latter tends to overestimate, due to their inability to discriminate compositional atypicality arising because of HGT from those that are a consequence of other biological factors. We propose here a framework that exploits the strengths of both these approaches while bypassing the pitfalls of either. Genomic islands lacking markers are identified by their association with genomic islands with markers. This was made possible by performing marker enrichment and phyletic pattern analyses within an integrated framework of recursive segmentation and clustering. The proposed method, IslandCafe, compared favorably with frequently used methods for genomic island detection on synthetic test datasets and on a test-set of known islands from 15 well-characterized bacterial species. Furthermore, IslandCafe identified novel islands with imprints of likely horizontal acquisition.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/genomic-island" class="hw-term hw-article-keyword hw-article-keyword-genomic-island" rel="nofollow">genomic island</a></li><li class="kwd"><a href="/keyword/horizontal-gene-transfer" class="hw-term hw-article-keyword hw-article-keyword-horizontal-gene-transfer" rel="nofollow">horizontal gene transfer</a></li><li class="kwd"><a href="/keyword/mobile-elements" class="hw-term hw-article-keyword hw-article-keyword-mobile-elements" rel="nofollow">mobile elements</a></li><li class="kwd"><a href="/keyword/segmentation" class="hw-term hw-article-keyword hw-article-keyword-segmentation" rel="nofollow">segmentation</a></li><li class="kwd"><a href="/keyword/clustering" class="hw-term hw-article-keyword hw-article-keyword-clustering" rel="nofollow">clustering</a></li></ul><p id="p-3">Genomic islands (GIs) are clusters of functionally related genes that are mobilized across organisms through mechanisms other than vertical inheritance, namely, transformation, conjugation, transduction, cell fusion, and gene transfer agents (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Juhas <em>et al.</em> 2009</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Soucy, Huang, &amp; Gogarten 2015</a>). Horizontal gene acquisitions enable microbes to modulate their repertoire of genes. GIs are an important instrument that plays a major role in the evolution of microorganisms. GIs may be involved in adaptability, fitness and competitiveness (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Dobrindt <em>et al.</em> 2003</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Dobrindt <em>et al.</em> 2004</a>), and may be important clinically as these can be enriched in genes that can lead to emergence of new pathogens or antibiotic resistant strains.</p><p id="p-4">Because of the versatile functions of GIs and their impact on the evolution of microbes and on the interactions between microbes and their hosts, significant efforts have g7one into quantification of the presence of GIs and understanding of their impact on the microbial evolution (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Hacker &amp; Carniel 2001</a>; <a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Juhas <em>et al.</em> 2009</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Langille, M. G., Hsiao, &amp; Brinkman 2008</a>). GIs have traditionally been deciphered using wet lab approaches that utilize subtractive hybridization or suppression subtractive hybridization, or DNA-DNA hybridization, enabling genome comparison to identify strain specific sequences (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Dobrindt <em>et al.</em> 2004</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Winstanley 2002</a>). Another approach called island probing (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Reyrat, Pelicic, Gicquel, &amp; Rappuoli 1998</a>) uses counterselectable markers (<a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">Dobrindt <em>et al.</em> 2004</a>). However, exhaustive quantification of GIs using wet lab techniques is not feasible, in particular of the GIs that could have integrated into the genomes of close relatives. Furthermore, this could be time consuming and expensive. Whole genome sequencing and comparative genomics, spurred by the recent advances in sequencing, have enabled high-throughput genome-wide studies to localize mobile genomic elements and quantify their influence on microbial evolution. A number of computational methods that perform either standalone (single genome) or comparative (multiple genomes) analyses to identify GIs have been developed in recent years (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Che, Wang, Fazekas, &amp; Chen 2014</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">de Brito, Maracaja-Coutinho, de Farias, Batista, &amp; do RÃªgo 2016</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Ganesan, Rakitianskaia, Davenport, Tummler, &amp; Reva 2008</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Hsiao, W., Wan, Jones, &amp; Brinkman 2003</a>; <a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Langille <em>et al.</em> 2008</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Langille, M. G. &amp; Brinkman 2009</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Pundhir, Vijayvargiya, &amp; Kumar 2008</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Vernikos &amp; Parkhill 2006</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Waack <em>et al.</em> 2006</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Wang, Fazekas, Booth, Liu, &amp; Che 2011</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Wei <em>et al.</em> 2017</a>). Among the most frequently used approaches are those that search for characteristic markers, such as tRNA genes in the host genome where GIs may integrate, direct repeats that flank them, integrase genes, transfer genes, phage-related sequences, conjugative elements, and other mobile genetic elements such as IS elements. Integration of GI splits the tRNA gene but is regenerated by the island sequences; GI boundaries could thus be localized by identifying the regenerated tRNA and its displaced fragment using sequence alignment (<a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Hsiao <em>et al.</em> 2003</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Reiter, Palm, &amp; Yeats 1989</a>). Further, GIs often possess genes involved in mobility and integration/recombination, such as those encoding integrases, transposases, recombinases and excisionases (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Doublet <em>et al.</em> 2003</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Langille <em>et al.</em> 2010</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Osborn &amp; BÃ¶ltner 2002</a>). Sequences flanking GIs may also bear evidence of GI integration. Phage insertion at tRNA sites often results in direct repeat sequences (<a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Langille <em>et al.</em> 2010</a>). GIs may also harbor genes involved in pathogenicity and drug resistance, and may be enriched in genes with yet unknown functions (âhypothetical proteinsâ) or of phage origin (<a id="xref-ref-28-3" class="xref-bibr" href="#ref-28">Langille <em>et al.</em> 2010</a>). However, this approach gives a conservative estimate of islands, missing those that integrate at sites other than tRNA or tmRNA genes, or missing islands deficient in markers that might have been eliminated by subsequent evolutionary decay of the islands.</p><p id="p-5">Another class of methods exploits the atypical compositional features of GIs, such as unusual GC content or oligonucleotide composition or codon usage pattern. These atypical features reflect the compositional biases of the donors and can thus be identified as distinct signals against the recipient genome background. These methods often invoke either bottom-up gene by gene approach, <em>e.g.</em>, SIGI-HMM that examines codon usage bias, (<a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Waack <em>et al.</em> 2006</a>; <a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Wei <em>et al.</em> 2017</a>) or moving window approach, <em>e.g.</em>, AlienHunter and SeqWord that use oligomer frequencies within a sliding window (<a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Vernikos &amp; Parkhill 2006</a>) and Mean Shift Genomic Island Predictor (MSGIP) that detects windows of genes with atypical composition (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">de Brito <em>et al.</em> 2016</a>). Because of the variable compositional character of GIs, weakly atypical genes or windows are often misclassified, which complicates the detection of island boundaries. Furthermore, the moving window methods are sensitive to window sizeâ smaller size increases stochastic fluctuations while larger size diminishes resolution. These methods are also prone to misclassifying genes or regions that display anomalous composition for reasons other than horizontal acquisition.</p><p id="p-6">To minimize false positives, combinations of these approaches have also been proposed. IslandPath-Dimob uses presence of mobility genes, t-RNA insertion sites, and dinucleotide compositional bias to predict GIs (<a id="xref-ref-20-3" class="xref-bibr" href="#ref-20">Hsiao <em>et al.</em> 2003</a>). IslandViewer (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bertelli <em>et al.</em> 2017</a>; <a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Langille &amp; Brinkman 2009</a>) is an integrated interface that combines the predictions of different methods including SIGI-HMM and IslandPath-Dimob to localize GIs with high confidence. GIHunter (<a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Wang <em>et al.</em> 2011</a>) uses eight GI-associated features including gene density, intergenic distance, phage genes, tRNA genes, genes encoding integrase or transposase, highly expressed genes, and the AlienHunter score. These methods could be highly specific, though at the cost of numerous false negatives. In contrast, PredictBias (<a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Pundhir <em>et al.</em> 2008</a>), a moving window approach, is designed to be more sensitive- genomic regions with unusual codon usage and either atypical GC composition or atypical dinucleotide composition, or regions with an abundance of virulence genes, are deemed GIs.</p><p id="p-7">Among the comparative genomics or phylogenetic methods, one approach is to search for genomic regions with limited phylogenetic distribution, that is, those regions that are absent from the genomes of close relatives are inferred as GIs. IslandPick (<a id="xref-ref-27-3" class="xref-bibr" href="#ref-27">Langille <em>et al.</em> 2008</a>) is an example of this class of methods. This approach, however, requires multiple strains of closely related species. Lineage-specific gene loss may further confound the inference.</p><p id="p-8">It was recently suggested that GIs could be more robustly identified using a top-down approach that allows examination of genes <em>en mass</em>e (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Arvey, Azad, Raval, &amp; Lawrence 2009</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Azad, R. K. &amp; Li 2013</a>). The premise of this approach is simultaneous analysis of GI harbored genes, both weakly atypical and strongly atypical, in order to decipher the GI structure more robustly. One such method, MJSD, which is based on a recursive segmentation procedure, indeed delineated GIs significantly better than other methods (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Arvey <em>et al.</em> 2009</a>). This has spurred development of more segmentation based methods, including Zisland Explorer that uses cumulative GC profile for GI identification (<a id="xref-ref-41-3" class="xref-bibr" href="#ref-41">Wei <em>et al.</em> 2017</a>) and GEMINI that utilizes segment context information within an integrated segmentation and clustering framework to delineate GIs.</p><p id="p-9">Although further development of more sensitive, top-down methods for GI detection is sorely needed, there are significant challenges that must be overcome. The segmentation procedure is often followed by an agglomerative clustering procedure for grouping of compositionally similar segments (<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Azad &amp; Li 2013</a>). This makes possible identification of âtypicalâ (the most abundant, potentially vertically inherited or native) and âatypicalâ (potentially horizontally acquired or alien) genomes by virtue of the cluster size, with the largest cluster representing the genome backbone (<em>i.e.</em>, the native genome) while the others representing the accessory or alien genome. Although this makes possible an unbiased analysis of a genome, agglomerative clustering of segments is fraught with risks. Grouping of segments at stricter stringencies may render clusters that are âpureâ, <em>i.e.</em>, harboring either native or alien DNA segments but not both, however, numerous native clusters as well as alien clusters thus generated could complicate the reconstruction of the genome structure. Relaxing the stringency may result in undesirable mergers of clusters. This is particularly true in this case where weakly typical, native segments may coalesce into one or more clusters that are deemed distinct from the cluster harboring strongly typical, native segments; allowing these native clusters to merge by relaxing the stringency may cause clusters harboring weakly atypical, alien segments to coalesce with the native clusters. This may also result in undesirable mergers of clusters representing closely related but distinct donors. On the other hand, certain class of native segments, <em>e.g.</em>, those harboring highly expressed genes of apparently unusual codon usage, may coalesce into clusters of their own and may not merge with the backbone cluster even at relaxed stringencies. One way to circumvent this is to utilize biological information, such as segment context information, for cluster merger after generating apparently pure clusters at a stringent clustering threshold (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Jani, Mathee, &amp; Azad 2016</a>). Further experiments show that whereas this may work for some strains, this is not universally applicable for any choice of threshold (further discussed below).</p><p id="p-10">Deconstruction of segmental structure underlying a genome is critical to deciphering GIs, as the data from recent studies suggest (<a id="xref-ref-4-3" class="xref-bibr" href="#ref-4">Azad &amp; Li 2013</a>; <a id="xref-ref-41-4" class="xref-bibr" href="#ref-41">Wei <em>et al.</em> 2017</a>). Furthermore, for a sensitive and robust GI detection, GIs that have lost the identifying features or markers must also be identified and atypical segments that are compositionally deviant for reasons other than HGT must be eliminated from predictions. The success of such procedures depends on robust grouping of genes or segments of similar ancestry or origin. In our survey of the methods for GI detection, we observed that none of the current methods has the ability to accomplish this. We therefore embarked on developing an approach that could utilize biological information specific to GIs within an integrated framework of segmentation and clustering. We hypothesized that GIs lacking the identifying features or markers could be identified if analyzed together with GIs enriched in these markers. Our hypothesis is based on the assumption that not all GIs originating from a donor taxon would have lost many or most identifying markers contemporaneously, and if this is true, GIs deficient in markers could be identified by virtue of their association (<em>i.e.</em>, origin) with GIs enriched in markers. This association could be explored in a number of ways; here, we infer the association based on similar compositional bias shared by GIs originating from a specific donor taxon. Based on this premise, after obtaining clusters at a stringent threshold, if a cluster is found enriched in GI specific markers (assessed against whole genome background), all segments within the cluster, whether enriched or not in markers, are deemed GIs (or, parts of GIs or the alien genome). On the other hand, if a cluster is not enriched in GI specific markers, all segments within the cluster are deemed ânon-GIsâ, provided genes harbored by these segments do not display aberrant phyletic pattern. We tested our hypothesis on synthetic test datasets and on a comprehensive dataset of known genomic islands from 15 well-characterized bacterial species. Our method, IslandCafe, named after detection of <span class="underline">Island</span> by <span class="underline">C</span>ompositional <span class="underline">a</span>nomaly and <span class="underline">f</span>eature <span class="underline">e</span>nrichment, compared favorably with frequently used methods for genomic island detection. In what follows, we further elaborate on our method and the genomes that were used for assessment, present our results, and conclude with a discussion.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Genome sequences and annotations</h3><p id="p-11">The complete genome sequences and annotations of 15 bacteria representing a host of taxa, namely, <em>Acinetobacter baumannii</em> AYE (NC_010410.1), <em>Bartonella tribocorum</em> CIP 105476 (NC_ 010161.1), <em>Bordetella petrii</em> DSM 12804 (NC_010170.1), <em>Burkholderia cenocepia</em> J2315 (NC_011000.1, NC_011001.1 and NC_011002.1), <em>Burkholderia pseudomallei</em> K96243 (NC_006350.1 and NC_006351.1), <em>Clavibacter michiganensis</em> NCPPB 382 (NC_009480.1), <em>Corynebacterium diptheriae</em> NCTC 13129 (NC_002935.2), <em>Escherichia coli</em> CFT073 (NC_004431.1), <em>Mesorhizobium loti</em> MAFF 303099 (NC_002678.2), <em>Proteus mirabilis</em> HI4320 (NC_010554.1), <em>Pseudomonas aeruginosa</em> LESB58 (NC_011770.1), <em>Salmonella enterica serovar Typhi</em> CT18 (NC_003198.1), <em>Staphylococcus aureus</em> USA300 FPR3757 (NC_007793.1), <em>Streptococcus equi</em> 4047 (NC_012471.1), and <em>Vibrio cholerae</em> O1 biovar eltor str. N16961 (NC_002505.1 and NC_002506.1) were obtained from NCBI FTP site (<a href="ftp://ftp.ncbi.nlm.nih.gov/genomes/archive/old_genbank/Bacteria/">ftp://ftp.ncbi.nlm.nih.gov/genomes/archive/old_genbank/Bacteria/</a>). These bacteria were previously well-studied for the presence of genomic islands; genomic island coordinates and the supporting evidence from these studies are provided in Table S1.</p></div><div id="sec-3" class="subsection"><h3>Synthetic genomes</h3><p id="p-12">In addition to the aforementioned genomes, synthetic genomes were constructed for assessment for IslandCafe and other methods. <em>Burkholderia cenocepacia</em> J2315 was selected as the recipient organism. All GI prediction methods considered in this study were then applied to catalog putative GIs in <em>B. cenocepacia</em> J2315s primary chromosome (chromosome 1). <em>B. cenocepacia</em> J2315 genomic sequences that were not predicted as GIs by any method were extracted and concatenated. This provided a conservative backbone (core) genome of <em>B. cenocepacia</em> J2315. Backbone genomes were obtained similarly for selected donors, namely <em>Pseudomonas aeruginosa</em> LESB58 and <em>Stenotrophomonas maltophilia</em> D457 (secondary chromosomes, where present, were not considered for extraction of the backbone genomes). A synthetic genome was constructed by simulating transfer of segments from the backbone genomes of donors into the <em>B. cenocepacia</em> J2315 backbone genome. 12 Segments of size 30 Kbp, 50 Kbp, and 80 Kbp were randomly extracted from the backbone genome of each donor and inserted into the <em>B. cenocepacia</em> backbone genome; these GIs in a synthetic genome represented â¼16% of the original <em>B. cenocepacia</em> J2315 chromosome 1. Multiple synthetic genome replicates were thus obtained and three test datasets were then generated by randomly introducing marker genes from donor genomes in 25%, 50%, and 75% of the GIs in synthetic genomes, respectively. It is possible that some of the sequences sampled from donor backbone genomes for insertion might already have one or more marker genes, so the GIs in the synthetic genome selected for marker introduction contain at least one marker gene, following insertion of a marker gene (transposase) into each of these GIs.</p></div><div id="sec-4" class="subsection"><h3>Current genomic island prediction tools</h3><p id="p-13">We assessed IslandCafe against ten frequently used GI prediction methods, namely, IslandPick (<a id="xref-ref-27-4" class="xref-bibr" href="#ref-27">Langille <em>et al.</em> 2008</a>), GIHunter (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Che <em>et al.</em> 2014</a>), IslandPath-Dimob (<a id="xref-ref-20-4" class="xref-bibr" href="#ref-20">Hsiao <em>et al.</em> 2003</a>), MSGIP (<a id="xref-ref-7-3" class="xref-bibr" href="#ref-7">de Brito <em>et al.</em> 2016</a>), SIGI-HMM (<a id="xref-ref-39-3" class="xref-bibr" href="#ref-39">Waack <em>et al.</em> 2006</a>), Zisland Explorer (<a id="xref-ref-41-5" class="xref-bibr" href="#ref-41">Wei <em>et al.</em> 2017</a>), AlienHunter (<a id="xref-ref-38-3" class="xref-bibr" href="#ref-38">Vernikos &amp; Parkhill 2006</a>), PredictBias (<a id="xref-ref-31-3" class="xref-bibr" href="#ref-31">Pundhir <em>et al.</em> 2008</a>), SeqWord (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Ganesan <em>et al.</em> 2008</a>), and IslandViewer (<a id="xref-ref-26-3" class="xref-bibr" href="#ref-26">Langille &amp; Brinkman 2009</a>). These include both well-established and recently developed methods. MSGIP, SIGI-HMM, Zisland Explorer, AlienHunter, and SeqWord search for regions with anomalous codon usage and/or oligonucleotide composition. IslandPath-Dimob and GIHunter search for regions harboring GI specific markers and combine this with compositional signatures to predict GIs. PredictBias bases its prediction on compositional bias; in addition, it outputs segments that are enriched in virulence genes. IslandPick identifies putative GIs via sequence comparison, and IslandViewer combines predictions by three different methods, namely, SIGI-HMM, IslandPath-DIMOB, and IslandPick.</p></div><div id="sec-5" class="subsection"><h3>Compositional anomaly assessment</h3><p id="p-14">Our proposed method, IslandCafe, combines compositional anomaly, a hallmark of GIs, with the functional or structural features that characterize GIs within the framework of a recursive segmentation and agglomerative clustering procedure. This integrative framework allows fragmentation of a genome into compositionally homogeneous segments that are then segregated into clusters reflecting their potentially shared ancestries (<a id="xref-ref-4-4" class="xref-bibr" href="#ref-4">Azad &amp; Li 2013</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Jani &amp; Azad 2013</a>). The binary segmentation process entails division of a genome into two segments at a position where the compositional difference between the resulting segments is maximum, provided this difference is deemed statistically significant. The compositional difference is quantified using an information-entropy based divergence measure, namely, Jensen-Shannon divergence generalized within the framework of Markov chain model of order <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3273/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3273/embed/mml-math-1.gif"/></noscript></span></span> defined as (<a id="xref-ref-2-3" class="xref-bibr" href="#ref-2">Arvey <em>et al.</em> 2009</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Thakur, Azad, &amp; Ramaswamy 2007</a>),<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3273/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3273/embed/mml-math-2.gif"/></noscript></span></span><span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3273/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3273/embed/mml-math-3.gif"/></noscript></span></span>) denotes the divergence between two sequence segments represented by the respective probability distributions <em>p</em><sub>1</sub> and <em>p</em><sub>2</sub>, each comprising the marginal and conditional (transitional) probability distributions. <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3273/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3273/embed/mml-math-4.gif"/></noscript></span></span> is the entropy function for Markov source of order m, defined as, <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3273/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3273/embed/mml-math-5.gif"/></noscript></span></span>. <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3273/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3273/embed/mml-math-6.gif"/></noscript></span></span> is the probability for transitioning from oligonucleotide w to the succeeding nucleotide x,and <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3273/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3273/embed/mml-math-7.gif"/></noscript></span></span> is the probability of oligonucleotide w of length <em>m</em> that defines that order of the Markov model. <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3273/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3273/embed/mml-math-8.gif"/></noscript></span></span> is the weight factor assigned to <em>p<sub>i</sub></em>.</p><p id="p-15">The statistical significance of a value of the divergence measure is assessed using the probability distribution of this measure in random sequences, which has been shown to approximate the chi-square distribution function (<a id="xref-ref-2-4" class="xref-bibr" href="#ref-2">Arvey <em>et al.</em> 2009</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Grosse <em>et al.</em> 2002</a>),</p><p id="p-16"><span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3273/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3273/embed/mml-math-9.gif"/></noscript></span></span> where, <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3273/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3273/embed/mml-math-10.gif"/></noscript></span></span> is the degrees of freedom, with alphabet size <em>k</em> = 4 for DNA sequences, and <em>L</em> is the length of the sequence to be segmented. Probability distribution of the maximum value of this measure has been shown to approximate chi-square distribution function with fitting parameters (<a id="xref-ref-2-5" class="xref-bibr" href="#ref-2">Arvey <em>et al.</em> 2009</a>; <a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Grosse <em>et al.</em> 2002</a>; <a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Thakur <em>et al.</em> 2007</a>):<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3273/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3273/embed/mml-math-11.gif"/></noscript></span></span><em>Î²</em> and <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3273/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3273/embed/mml-math-12.gif"/></noscript></span></span> are the fitting parameters whose values are estimated by fitting the above equation to the empirical distributions obtained via Monte Carlo simulations. The recursion of segmentation is halted when none of the segments can be divided further. Since hyper-segmentation is allowed for precise delineation of GI boundaries, an agglomerative clustering of similar segments is performed at a relatively relaxed clustering stringency in two steps (<a id="xref-ref-4-5" class="xref-bibr" href="#ref-4">Azad &amp; Li 2013</a>). In the first step, compositionally similar, contiguous segments are identified and merged, thus restoring the underlying segmental structure. The second clustering step allows grouping of compositionally similar clusters recursively, performed within the same framework of statistical hypothesis testing (<a id="xref-ref-4-6" class="xref-bibr" href="#ref-4">Azad &amp; Li 2013</a>). This procedure, by virtue of its ability to segregate segments of different lineages, recovers the backbone (vertically inherited) genome as the largest cluster (typically with more than half of the genome), whereas smaller clusters represent the potential donor taxa (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Garcia-Vallve, Guzman, Montero, &amp; Romeu 2003</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Lawrence &amp; Ochman 1998</a>). A significant challenge, however, lies here in classifying the segments by their origins after the segmental structure of a genome is uncovered (<em>i.e.</em>, after the first clustering step). Often, native segments are grouped into two or more clusters, with the largest harboring the strongly typical native segments. The weakly typical native segments tend to form cluster(s) of their own. Relaxing the statistical threshold to allow merger of these clusters could also result in merger of one or more alien clusters with the native clusters. Robust merger of clusters is critical to the success of this class of methods as any cluster-level misclassification renders all segments within the cluster misclassified. By combining this statistical framework with GI specific marker enrichment and phyletic pattern assessment, we show here that this problem can be satisfactorily addressed.</p></div><div id="sec-6" class="subsection"><h3>Feature enrichment and phyletic pattern analysis</h3><p id="p-17">After generating clusters at a stringent clustering threshold within the statistical hypothesis testing framework, we first perform an enrichment analysis to identify clusters enriched in features or markers that are typically associated with GIs. We posit that weakly typical, native clusters are not enriched in these features, whereas alien clusters, both strongly atypical and weakly atypical, are enriched. IslandCafe identifies markers that typically characterize GI (Table S2) by parsing the input genome annotation file. Given a genome sequence, IslandCafe compiles markers by first employing Prodigal (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Hyatt <em>et al.</em> 2010</a>) to predict genes and then HMMER (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Eddy 1998</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Finn, Clements, &amp; Eddy 2011</a>) to annotate markers. A custom Pfam HMM database was created by performing search for GI markers using keywords such as transposase in the Pfam database (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Finn <em>et al.</em> 2015</a>). The custom database was manually curated to eliminate any profile HMMs not representing GI markers. This resulted in a local database of 449 profile HMMs representing marker gene families (Table S3). Genes from a genome of interest are probed against this database and those with âhitsâ in the database with expect values 0.01 or less (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Hsiao <em>et al.</em> 2005</a>) are annotated as marker genes.</p><p id="p-18">Following segmentation and clustering performed for each genome, parts of the native or backbone genome are first identified by the largest cluster; the smaller clusters are then examined and processed as follows. If a cluster is found enriched in marker genes (one and half fold relative to the genome), it is deemed an alien cluster. Non-enriched clusters are deemed alien only if their genes display aberrant phyletic pattern, <em>i.e.</em>, absence from majority of the close relatives of the genome harboring the gene of interest. Aberrant phyletic distribution is assessed by aligning the sequence of the protein encoded by the gene of interest against all protein sequences originating from the same taxon as the gene of interest using BLASTP (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Altschul, Gish, Miller, Myers, &amp; Lipman 1990</a>). A custom database of protein sequences for all bacteria at the NCBI ftp site (<a href="http://ftp://ftp.ncbi.nlm.nih.gov/genomes/archive/old_genbank/Bacteria/">ftp://ftp.ncbi.nlm.nih.gov/genomes/archive/old_genbank/Bacteria/</a>) was constructed to accomplish this. If a query protein returns BLAST hits with &gt;70% identity and query coverage in less than half of the organisms from the same genus as the organism with the query sequence, the gene encoding the query protein is deemed phyletically atypical or aberrant. We thus identify weakly typical, smaller native clusters based on enrichment and phyletic pattern, that is, those lacking marker enrichment and aberrant phyletic distribution, and merge them to the largest cluster harboring the strongly typical native segments. The remaining clusters, enriched in markers or displaying atypical gene distribution, are deemed alien. GIs are identified as contiguous alien segments, 8 Kbp or more in size (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Schmidt &amp; Hensel 2004</a>). An illustrative diagram of the pipeline implemented in IslandCafe is shown in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/10/3273/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Schematic representation of IslandCafe&#x2019;s protocol for identifying genomic islands. A genome is divided recursively into compositionally homogeneous segments (&#x201C;Segmentation&#x201D;, here each segment is shown by a colored horizontal thick line; the red vertical line indicates position where the divergence is maximum). &#x201C;Contiguous clustering&#x201D; is the first round of agglomerative clustering procedure, which entails identification and merger of compositionally similar contiguous segments (contiguous similar segments are shown in a circle). &#x201C;Non-contiguous clustering&#x201D; is the second round of agglomerative clustering procedure, which entails recursive grouping of compositionally similar segments, including non-contiguous similar segments (single lines and groups of lines (circles) of same color are now merged into a single cluster). &#x201C;Feature enrichment and phyletic distribution analyses&#x201D; entails identification of GI specific features in each segment (shown as &#x394; on a segment) and identification of genes with aberrant phyletic distribution (shown as on a segment). &#x201C;Clustering based on feature enrichment and phyletic distribution&#x201D; involves identification and merger of native clusters (blue and orange clusters that are not enriched in GI specific markers or lack genes with aberrant phyletic distribution) while precluding undesirable mergers of alien clusters. IslandCafe calls any segments or chains of contiguous segments, 8 Kbp or greater, belonging to alien clusters as GIs." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1719961398" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Schematic representation of IslandCafe&#x2019;s protocol for identifying genomic islands. A genome is divided recursively into compositionally homogeneous segments (&#x201C;Segmentation&#x201D;, here each segment is shown by a colored horizontal thick line; the red vertical line indicates position where the divergence is maximum). &#x201C;Contiguous clustering&#x201D; is the first round of agglomerative clustering procedure, which entails identification and merger of compositionally similar contiguous segments (contiguous similar segments are shown in a circle). &#x201C;Non-contiguous clustering&#x201D; is the second round of agglomerative clustering procedure, which entails recursive grouping of compositionally similar segments, including non-contiguous similar segments (single lines and groups of lines (circles) of same color are now merged into a single cluster). &#x201C;Feature enrichment and phyletic distribution analyses&#x201D; entails identification of GI specific features in each segment (shown as &#x394; on a segment) and identification of genes with aberrant phyletic distribution (shown as on a segment). &#x201C;Clustering based on feature enrichment and phyletic distribution&#x201D; involves identification and merger of native clusters (blue and orange clusters that are not enriched in GI specific markers or lack genes with aberrant phyletic distribution) while precluding undesirable mergers of alien clusters. IslandCafe calls any segments or chains of contiguous segments, 8 Kbp or greater, belonging to alien clusters as GIs.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/10/3273/F1.medium.gif" width="440" height="314"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/9/10/3273/F1.medium.gif" width="440" height="314"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/10/3273/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/10/3273/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494359" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-19" class="first-child">Schematic representation of IslandCafeâs protocol for identifying genomic islands. A genome is divided recursively into compositionally homogeneous segments (âSegmentationâ, here each segment is shown by a colored horizontal thick line; the red vertical line indicates position where the divergence is maximum). âContiguous clusteringâ is the first round of agglomerative clustering procedure, which entails identification and merger of compositionally similar contiguous segments (contiguous similar segments are shown in a circle). âNon-contiguous clusteringâ is the second round of agglomerative clustering procedure, which entails recursive grouping of compositionally similar segments, including non-contiguous similar segments (single lines and groups of lines (circles) of same color are now merged into a single cluster). âFeature enrichment and phyletic distribution analysesâ entails identification of GI specific features in each segment (shown as Î on a segment) and identification of genes with aberrant phyletic distribution (shown as on a segment). âClustering based on feature enrichment and phyletic distributionâ involves identification and merger of native clusters (blue and orange clusters that are not enriched in GI specific markers or lack genes with aberrant phyletic distribution) while precluding undesirable mergers of alien clusters. IslandCafe calls any segments or chains of contiguous segments, 8 Kbp or greater, belonging to alien clusters as GIs.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-7" class="subsection"><h3>Performance assessment</h3><p id="p-20">Comparative assessment of IslandCafe was performed against ten frequently used GI prediction methods at both nucleotide-level (<em>i.e.</em>, ability to identify GI nucleotides) and island-level (<em>i.e.</em>, ability to identify a GI as a single segment or a chain of contiguous segments). In assessing the island-level accuracy, if a method predicts several segments spanning a known GI, only the largest overlapping segment is considered. If the largest overlapping segment is more than twice the size of the corresponding GI, it is not considered a true prediction. The standard performance metrics, namely, recall (or sensitivity), precision, average performance, F-measure, performance coefficient, and Matthews correlation coefficient (MCC), as defined below, were used for performance assessment.<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3273/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3273/embed/mml-math-13.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3273/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3273/embed/mml-math-14.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3273/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3273/embed/mml-math-15.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3273/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3273/embed/mml-math-16.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3273/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3273/embed/mml-math-17.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3273/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3273/embed/mml-math-18.gif"/></noscript></span></span>Here TP, TN, FP, FN are the numbers of true positives, true negatives, false positives, and false negatives respectively. All these performance metrics have values lying between 0 and 1, which are shown either as fraction or percentage in the following sections. Together these metrics quantify the accuracy of a method in identifying GIs or discriminating GIs from non-GIs. Note that we didnât consider Classification Accuracy, defined as (TP+TN)/(TP+TN+FP+FN), for assessment here, as this is not an appropriate metric for an unbalanced dataset, which is the case here for both synthetic and datasets; <em>e.g.</em>, real dataset has â¼91% non-GI nucleotides and only â¼9% GI nucleotides. As the number of GI nucleotides is much smaller (â¼6.6 Mb across all 15 species) than the nucleotides in the rest of the genomes (â¼69.7 Mb), a method that identifies only few or even no GI nucleotides will have a Classification Accuracy of â¼90%. In contrast, the accuracy measures such as Recall and Precision and their combination, the F-measure, are considered appropriate for unbalanced datasets.</p></div><div id="sec-8" class="subsection"><h3>Data Availability</h3><p id="p-21">IslandCafe is an open source collaborative initiative available in the GitHub repository (<a href="https://github.com/mehuljani/IslandCafe">https://github.com/mehuljani/IslandCafe</a>). The following data are listed in the supplementary table: Evidence from the literature supporting presence of GIs in the genomes used for this study (Table S1), list of genes often associated with GIs (Table S2), list of PFAM ids used to make database for identifying GI specific features or marker genes (Table S3), performance of segmentation and clustering algorithm in identifying GIs in 15 bacterial species at various segmentation and clustering thresholds (Table S4), performance of segmentation and clustering algorithm after including marker gene enrichment and phylogenetic modules (Table S5), performance of GI prediction tools in identifying nucleotides belonging to GIs and identifying GIs in synthetic <em>B.cenocepacia</em> genome dataset with 25% of all GIs harboring marker genes (Table S6), performance of genomic island prediction tools in identifying nucleotides belonging to GIs and identifying GIs in synthetic <em>B. cenocepacia</em> genome dataset with 50% of all GIs harboring marker genes (Table S7), performance of genomic island prediction tools in identifying nucleotides belonging to GIs and identifying GIs in synthetic <em>B. cenocepacia</em> genome dataset with 75% of all GIs harboring marker genes (Table S8), performance of genomic island prediction tools in classifying nucleotides as belonging to GIs and non-GIs for 15 bacterial species (Table S9), performance evaluation of genomic island prediction tools for identifying islands with at least 75% overlap between the predicted segment and the corresponding known island in 15 bacterial species (Table S10), performance evaluation of genomic island prediction tools for identifying islands with at least 95% overlap between the predicted segment and the corresponding known island in 15 bacterial species (Table S11), performance of GI prediction methods in identifying GIs in 15 bacterial species at 50% cutoff of overlap between the predicted segment and the corresponding known GI (Table S12), performance of GI prediction methods (assessed by F-measure) in identifying GIs in 15 bacterial species at 75% cutoff of overlap between the predicted segment and the corresponding known GI (Table S13), performance of GI prediction methods (assessed by F-measure) in identifying GIs in 15 bacterial species at 95% cutoff of overlap between the predicted segment and the corresponding known GI (Table S14), coordinates of GIs predicted by CAFE in the 15 representative genomes (Table S15), and grouping of GIs lacking marker genes with GIs harboring marker genes arising from a donor within a distinct cluster (Table S16). Supplemental material available at FigShare: <a href="https://doi.org/10.25387/g3.9255608">https://doi.org/10.25387/g3.9255608</a>.</p></div></div><div class="section results" id="sec-9"><h2 class="">Results</h2><div id="sec-10" class="subsection"><h3>Segmentation and clustering in GI detection</h3><p id="p-22">An integrated segmentation and clustering approach was shown to work well in deciphering GIs in bacterial genomes (<a id="xref-ref-4-7" class="xref-bibr" href="#ref-4">Azad &amp; Li 2013</a>). Comparative assessment with other methods on an artificial chimeric <em>E. coli</em> genome and a well-understood <em>Salmonella enterica typhi</em> CT18 genome demonstrated the power of this method in delineating large structures such as GIs in bacterial genomes (<a id="xref-ref-4-8" class="xref-bibr" href="#ref-4">Azad &amp; Li 2013</a>). Here, we assessed the same approach on a broader dataset, including 15 different species as listed in Materials section above. Both segmentation and clustering thresholds were varied. Segmentation threshold (significance level) was varied from 1E-2 to 1E-20, and similarly, the clustering thresholds were varied in this range for each segmentation threshold to identify the optimal setting of the program (Table S4). The highest accuracies on the combined test set of GIs and non-GIs from all 15 species were 0.55, 0.34, and 0.50 when considering average performance, performance coefficient, and F-measure respectively as the overall accuracy parameter (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). Results for individual genomes show that the optimal performance was obtained at different threshold settings for these genomes, which may differ with the optimal threshold setting for the entire dataset (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). This suggests that a âuniversalâ optimal parameter setting is difficult to realize within this framework.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/494352/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/494352/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Performance of segmentation and clustering algorithm in identifying GI nucleotides in 15 representative genomes. The values of performance metrics are shown for the optimal threshold setting for each genome and also for the optimal setting for the entire test dataset (all 15 genomes individually run at same thresholds). Contiguous clustering threshold refers to the significance threshold used in the first round of agglomerative clustering procedure to identify and merge compositionally similar contiguous segments. Non-contiguous clustering threshold refers to the significance threshold used in the second round of agglomerative clustering procedure to recursively group compositionally similar segments, including non-contiguous similar segments</span><div class="sb-div caption-clear"></div></div></div><p id="p-24">We further analyzed these genomes using the current version of this program that additionally used the segment context information (<a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Jani <em>et al.</em> 2016</a>) to improve GI detection in <em>Pseudomonas aeruginosa</em> LESB58. This strain was previously reported to harbor a number of pathogenicity and resistance islands (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Winstanley, C. <em>et al.</em> 2009</a>). As the genome of <em>Pseudomonas aeruginosa</em> LESB58 is well-characterized, with at least four GIs experimentally validated, the cluster configurations for this genome were earlier examined at different threshold settings, from a stringent setting generating numerous pure clusters to a relaxed setting generating hybrid clusters (harboring both native and alien segments) in addition to the pure clusters. Simultaneous formation of two large native clusters was observed, the largest with â¼62.5% of the genome harbored strongly typical, native segments and the other with â¼24.7% of the genome harbored compositionally ambiguous (weakly typical) native segments, whereas the third largest cluster with â¼7.1% of the genome harbored alien segments. Numerous smaller clusters were also formed in the process. Attempt to merge the large native clusters by successively relaxing the clustering stringency results in the merger of the largest alien cluster with the largest (native) cluster. This highlights the difficulty in segregating native and alien segments based on statistical approach alone. Earlier we had proposed utilizing segment context information to circumvent this limitation (<a id="xref-ref-24-3" class="xref-bibr" href="#ref-24">Jani <em>et al.</em> 2016</a>). This was based on the observation that compositionally ambiguous native segments are sparsely distributed within the genome. While this provided a cue to merge native clusters in the LESB58 genome, our analysis on the broader genome set showed that this approach may not work for some genomes, specifically when the clusters have too few segments to allow this statistical analysis. For example, the application of the recursive segmentation and clustering procedure to the <em>Vibrio cholerae O1 biovar eltor</em> str. N16961 Chr. 2 (1.07 Mb) genome resulted in seven clusters, however, the three largest clusters have only 2-3 segments each, precluding a reliable inference based on segment context information.</p></div><div id="sec-11" class="subsection"><h3>Utilizing feature enrichment and phyletic distribution in GI detection</h3><p id="p-25">Although segmentation and clustering provide a powerful tool for segregating compositionally similar segments, and in the process identifying the potential GIs, grouping of compositionally distinct native clusters while precluding undesirable mergers of alien and native clusters is a major bottleneck that must be addressed in order to take the state-of-the-art in GI prediction to new heights. We posit that since it is almost impossible to accomplish this task by tinkering with the algorithmic parameters within the hypothesis testing framework, a significant advance in the field could be possible by utilizing any biological information that could complement the compositional information encoded in GIs. We therefore attempted to complement the composition-based segmentation and clustering with functional or structural information embodied in GIs and the phyletic distribution of GI harbored genes. Functional features or markers that are frequently observed in GIs include genes that are associated with integration/recombination and transposition, phage metabolism, plasmid, and insertion (Table S2). While these features have previously been used to localize GIs, here we use this information for cluster merger, particularly to merge weakly typical, native cluster(s) with the strongly typical, native cluster. We postulate that alien clusters, particularly the large ones, are replete with GIs and therefore, are enriched in the aforementioned features that characterize GIs. On the other hand, the weakly typical native clusters are expected to be depleted of GI specific features. We use this difference in the enrichment of functional/structural features in the clusters to identify weakly typical, native clusters and merge them with the strongly typical, native cluster. This also allows us to eliminate spurious predictions arising from clusters with genes that are compositionally atypical for reasons other than HGT. For example, highly expressed native genes often have an unusual codon usage or compositional bias and are therefore prone to be misclassified as alien. However, as these clusters are not enriched in GI specific features, the proposed approach classifies them as native, thus minimizing false positives.</p><p id="p-26">The presence of GI specific features was quantified for each atypical cluster outputted by the segmentation and clustering algorithm, and if an atypical cluster was enriched in these features, it was deemed alien. Substantial performance improvement was observed following this procedure (Table S5). Further improvement was achieved when non-enriched atypical clusters were examined for the distribution of their genes in close relatives and deemed alien if their genes displayed aberrant phyletic pattern (Table S5).</p></div><div id="sec-12" class="subsection"><h3>Comparative assessment of IslandCafe with other GI prediction methods</h3><div id="sec-13" class="subsection"><h4>Rationale for comparative assessment at both nucleotide and island level:</h4><p id="p-27">Most previous studies evaluated GI prediction toolsâ performance based on their ability to classify genes as GI- or non-GI-borne genes (<a id="xref-ref-6-3" class="xref-bibr" href="#ref-6">Che <em>et al.</em> 2014</a>; <a id="xref-ref-41-6" class="xref-bibr" href="#ref-41">Wei <em>et al.</em> 2017</a>). However, GIs are clusters of functionally related genes that are acquired <em>in toto</em> rather than in multiple horizontal transfers of single genes at a genomic locus. Following acquisition, a GI may again be mobilized within or across genomes as an intact unit. It is therefore important that the GI prediction methods identify islands as single structures. Often, the GI prediction methods are unable to identify GIs as single, continuous entities; the rather fragmented predictions could be missing not just the large chunks of GIs but could also be overestimating the number of GIs in a genome. On the contrary, some methods predict large structures that may overlap with only small portion of known GIs. In order to assess the ability of method to recover a known GI as a single segment or a chain of contiguous segments, we considered here, for the GI level assessment, only the predicted segment (or chain of contiguous segments) with the largest overlap with the known GI as the true prediction by a method, provided this prediction satisfies a preset criterion (see below). This allows a fair assessment of the ability of a method to reconstruct GI structure within a genome; a reasonably robust method is expected to identify a GI as a single segment (or a chain of contiguous segments) with substantial overlap with the GI and also comparable in size to the GI it overlaps with.</p><p id="p-28">We evaluated IslandCafe against other GI prediction tools for localizing GIs at 50%, 75%, and 95% cutoffs, where an X% cutoff means that X% of a GI should be identified with the predicted segment (or chain of contiguous segments) in order for the prediction to be deemed a success. Thus, the GI level performance assessment was accomplished by obtaining the recall of a method as the fraction of known GIs that were correctly identified as single segments (or chains of contiguous segments) at X% cutoff, and the precision of a method as the fraction of all predicted GIs that were true predictions (known GIs) at X% cutoff. The harmonic mean of these two metrics defined the F-measure that was used as a single accuracy measure for GI-level performance assessment. Note that this evaluation procedure was followed only for GI level assessment; the nucleotide level performance evaluation was still done by assessing whether known GI and non-GI nucleotides were correctly classified using the assessment measures described in the Materials and Methods section. While the latter does not attest to a methodâs ability to decipher the GI structure, it does provide information on the fraction of a genome that is deemed GIs or parts of GIs by a method. Together, these two evaluation procedures provide insights into the ability of the methods to predict GI based accessory genome and the GIs themselves. As GIs are often acquired in single evolutionary events, GI prediction also provides an estimate of the horizontal transfer events involving GIs, an important information that sheds light on evolutionary processes shaping bacterial genomes.</p></div><div id="sec-14" class="subsection"><h4>Using synthetic genomes for assessment:</h4><p id="p-29">We first evaluated the GI prediction methods for their ability to detect GIs in synthetic genomes. As the evolutionary history of the segments (<em>i.e.</em>, island or non-island) is known in synthetic genomes, these provide a valid dataset for evaluating GI prediction methods. The donors, <em>P. aeruginosa</em> LESB58 and <em>S. maltophilia</em> D457, represent the class Gammaproteobacteria, distinct from the class Betaproteobacteria that the recipient <em>B. cenocepacia</em> J2315 belongs to. Both the donors and the recipient <em>B. cenocepacia</em> J2315 used in constructing synthetic genomes were previously observed cohabiting the lungs of the cystic fibrosis patients (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Denton, Todd, Kerr, Hawkey, &amp; Littlewood 1998</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Holden <em>et al.</em> 2009</a>; <a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Winstanley <em>et al.</em> 2009</a>), and could potentially be exchanging DNAs; indeed horizontal gene transfers among bacteria cohabiting the cystic fibrosis lungs have previously been reported (<a id="xref-ref-24-4" class="xref-bibr" href="#ref-24">Jani <em>et al.</em> 2016</a>). The performance of the GI prediction methods was assessed on these test data, by averaging over 5 synthetic genome replicates.</p></div><div id="sec-15" class="subsection"><h4>Classification of GI and non-GI nucleotides in synthetic genomes:</h4><p id="p-30">IslandCafe outperformed other methods by 23â93% in F-measure, 17â94% in Average Performance, and 34â88% in Performance Coefficient on the dataset with 25% marker gene GIs (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>, <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a> and Table S6), and similarly on the dataset with 50% marker gene GIs (by 14â96% in F-measure, <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2B</a>, Table S7) and on the dataset with 75% marker gene GIs (by 9â94% in F-measure, <a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2C</a>, Table S8). IslandCafeâs MCC was also significantly higher, <em>e.g.</em>, 0.93 <em>vs.</em> 0.70 of the next best performing method AlienHunter on the dataset with 25% marker gene GIs (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>). In general, the performance of marker based methods improved with increasing number of GIs with markers; IslandPath-Dimobâs and IslandViewerâs F-measure for identifying GI nucleotides increased from 0.41 and 0.69 (25% GIs with marker genes) to 0.81 and 0.85 (75% GIs with marker genes) respectively (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2</a>, Table S6-S8). As expected, the performance of the methods based solely on composition did not vary significantly with changes in marker abundance; among these methods, AlienHunter attained the highest nucleotide level F-measure of 0.70 (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2</a>, Tables S6-S8). IslandCafe could attain high sensitivity and specificity, thus outperforming other methods in identifying GI nucleotides in all synthetic datasets with nucleotide-level F-measure values of 0.93, 0.96 and 0.94 for 25%, 50%, and 75% of GIs with marker genes respectively (<a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2</a>, Tables S6-S8).</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/494324/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/494324/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Comparative assessment of genomic island prediction methods in classifying GI and non-GI nucleotides in a synthetic <em>Burkholderia cenocepia</em> genome having 25% GIs with marker genes. The values of the performance metrics were obtained by averaging over 5 synthetic genome replicates. Highest value for each performance metric is shown shaded</span><div class="sb-div caption-clear"></div></div></div><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/10/3273/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Assessment of genomic island prediction tools on synthetic Burkholderia cenocepia genomes. F- measure values (Y-axis; averaged over 5 synthetic genome replicates) are shown for different GI prediction methods and datasets when markers are artificially introduced into A) 25%, B) 50%, and C) 75% of all GIs in the genomes, for nucleotide level, and for island level at different overlap cutoffs (numbers in parenthesis on the X-axis; Z% cutoff means that Z% of a known GI should be identified with the predicted segment (or chain of contiguous segments) in order for the prediction to be deemed a success; see text for further details). Segmentation, contiguous clustering and non-contiguous clustering were performed at thresholds 10&#x2212;10, 10&#x2212;13, and 10&#x2212;13 respectively. Methods based on composition, phylogenetics, and marker gene detection are shown with diagonal brick fill in bars. Bars with a dotted fill show methods using both composition and presence of marker genes for identifying GIs. Methods relying only on sequence composition are shown with upward diagonal lines, and method based only on sequence comparison (IslandPick) is shown with grid." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1719961398" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Assessment of genomic island prediction tools on synthetic &lt;em&gt;Burkholderia cenocepia&lt;/em&gt; genomes. F- measure values (Y-axis; averaged over 5 synthetic genome replicates) are shown for different GI prediction methods and datasets when markers are artificially introduced into A) 25%, B) 50%, and C) 75% of all GIs in the genomes, for nucleotide level, and for island level at different overlap cutoffs (numbers in parenthesis on the X-axis; Z% cutoff means that Z% of a known GI should be identified with the predicted segment (or chain of contiguous segments) in order for the prediction to be deemed a success; see text for further details). Segmentation, contiguous clustering and non-contiguous clustering were performed at thresholds 10&lt;sup&gt;&#x2212;10&lt;/sup&gt;, 10&lt;sup&gt;&#x2212;13&lt;/sup&gt;, and 10&lt;sup&gt;&#x2212;13&lt;/sup&gt; respectively. Methods based on composition, phylogenetics, and marker gene detection are shown with diagonal brick fill in bars. Bars with a dotted fill show methods using both composition and presence of marker genes for identifying GIs. Methods relying only on sequence composition are shown with upward diagonal lines, and method based only on sequence comparison (IslandPick) is shown with grid.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/10/3273/F2.medium.gif" width="440" height="251"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/9/10/3273/F2.medium.gif" width="440" height="251"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/10/3273/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/10/3273/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494302" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-31" class="first-child">Assessment of genomic island prediction tools on synthetic <em>Burkholderia cenocepia</em> genomes. F- measure values (Y-axis; averaged over 5 synthetic genome replicates) are shown for different GI prediction methods and datasets when markers are artificially introduced into A) 25%, B) 50%, and C) 75% of all GIs in the genomes, for nucleotide level, and for island level at different overlap cutoffs (numbers in parenthesis on the X-axis; Z% cutoff means that Z% of a known GI should be identified with the predicted segment (or chain of contiguous segments) in order for the prediction to be deemed a success; see text for further details). Segmentation, contiguous clustering and non-contiguous clustering were performed at thresholds 10<sup>â10</sup>, 10<sup>â13</sup>, and 10<sup>â13</sup> respectively. Methods based on composition, phylogenetics, and marker gene detection are shown with diagonal brick fill in bars. Bars with a dotted fill show methods using both composition and presence of marker genes for identifying GIs. Methods relying only on sequence composition are shown with upward diagonal lines, and method based only on sequence comparison (IslandPick) is shown with grid.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-16" class="subsection"><h4>Identification of GI in synthetic genomes:</h4><p id="p-32">At island level, IslandCafe outperformed other methods at all cutoffs on the dataset with 25% marker gene GIs, bettering IslandViewer and IslandPath-Dimob by 54â65% and 20â31% respectively in F-measure (<a id="xref-fig-2-7" class="xref-fig" href="#F2">Figure 2A</a>, Table S6). IslandCafe displayed better performance on the dataset with 50% marker gene GIs as well, outperforming IslandViewer and IslandPath-Dimob by 41â49% and 4â12% respectively in F-measure (<a id="xref-fig-2-8" class="xref-fig" href="#F2">Figure 2B</a>, Table S7). On the dataset with 75% marker gene GIs, while IslandCafe outperformed all methods at the 50% cutoff, it was outperformed by IslandPath-Dimob by 5% in F-measure at the 75% and 95% cutoffs (<a id="xref-fig-2-9" class="xref-fig" href="#F2">Figure 2C</a>, Table S8). Notably, in all cases, whereas IslandCafe balanced the Recall and Precision very well, other methods produced large differences between Recall and Precision (Table S6-S8).</p></div><div id="sec-17" class="subsection"><h4>Using real genomes for assessment:</h4><p id="p-33">To assess the performance of IslandCafe and other GI prediction tools on genuine genomes, we used our compiled dataset of known GIs; these GIs have previously been used for assessment of GI prediction tools (<a id="xref-ref-24-5" class="xref-bibr" href="#ref-24">Jani <em>et al.</em> 2016</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Soares <em>et al.</em> 2016</a>; <a id="xref-ref-41-7" class="xref-bibr" href="#ref-41">Wei <em>et al.</em> 2017</a>). Although Zisland Explorer (<a id="xref-ref-41-8" class="xref-bibr" href="#ref-41">Wei <em>et al.</em> 2017</a>) was assessed on <em>Cronobacter sakazakii</em> ATCC BAA-894 as well, we did not include it due to the lack of supporting evidence for the reported GIs. This test dataset has a total of 138 GIs, 109 of which harbor markers. As with synthetic genomes, each method was evaluated for its ability to identify GI nucleotides and the GIs themselves as single segments or chains of contiguous segments.</p></div><div id="sec-18" class="subsection"><h4>Classification of GI and non-GI nucleotides in real genomes:</h4><p id="p-34">IslandCafe balanced sensitivity and specificity well (Recall and Precision respectively in <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>) in identifying the GI borne nucleotides, thus outperforming other methods by up to 49% in F-measure, up to 48% in Average performance, and up to 40% in Performance Coefficient (<a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>, <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). Both IslandCafe and IslandViewer performed comparably, with the former slightly outperforming the latter (by â¼1â3% in overall accuracy assessment) (<a id="xref-table-wrap-3-3" class="xref-table" href="#T3">Table 3</a>). IslandCafe, however, outperformed IslandPath-Dimob by larger margins (by â¼2â7% in overall accuracy assessment, <a id="xref-table-wrap-3-4" class="xref-table" href="#T3">Table 3</a>). GIHunter was found to be most sensitive, identifying â¼87% of GI nucleotides, however it had a precision of only 45%. On the other hand, IslandPath-Dimob had the highest precision of 67%, however, it could identify only 53% of the GI nucleotides. Zisland Explorer and IslandViewer balanced sensitivity and precision (as indicated by small difference between Recall and Precision values), however, both Recall and Precision values were relatively lower for ZislandExplorer (45% and 56% respectively, compared to 71% and 61% by IslandCafe, <a id="xref-table-wrap-3-5" class="xref-table" href="#T3">Table 3</a>).</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/494502/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/494502/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Comparative assessment of genomic island prediction methods in classifying GI and non-GI nucleotides in 15 representative bacterial genomes. Highest value for each performance metric is shown shaded</span><div class="sb-div caption-clear"></div></div></div><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/10/3273/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Assessment of genomic island prediction tools in identifying GI nucleotides and GIs in 15 representative bacterial genomes. F-measure values for nucleotide level and island level performance assessment are shown on the Y-axis for different GI prediction methods. Island level F-measure values are shown for different overlap cutoffs (numbers in parenthesis on the X-axis; Z% cutoff means that Z% of a known GI should be identified with the predicted segment (or chain of contiguous segments) in order for the prediction to be deemed a success; see text for further details). Segmentation, contiguous clustering, and non-contiguous clustering were performed at thresholds 2&#xB7;10&#x2212;1, 10&#x2212;5, and 10&#x2212;3 respectively. Methods based on composition, phylogenetics, and marker gene detection are shown with diagonal brick fill in bars. Bars with a dotted fill show methods using both composition and presence of marker genes for identifying GIs. Methods relying only on sequence composition are shown with upward diagonal lines, and method based only on sequence comparison (IslandPick) is shown with grid." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1719961398" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Assessment of genomic island prediction tools in identifying GI nucleotides and GIs in 15 representative bacterial genomes. F-measure values for nucleotide level and island level performance assessment are shown on the Y-axis for different GI prediction methods. Island level F-measure values are shown for different overlap cutoffs (numbers in parenthesis on the X-axis; Z% cutoff means that Z% of a known GI should be identified with the predicted segment (or chain of contiguous segments) in order for the prediction to be deemed a success; see text for further details). Segmentation, contiguous clustering, and non-contiguous clustering were performed at thresholds 2&#xB7;10&lt;sup&gt;&#x2212;1&lt;/sup&gt;, 10&lt;sup&gt;&#x2212;5&lt;/sup&gt;, and 10&lt;sup&gt;&#x2212;3&lt;/sup&gt; respectively. Methods based on composition, phylogenetics, and marker gene detection are shown with diagonal brick fill in bars. Bars with a dotted fill show methods using both composition and presence of marker genes for identifying GIs. Methods relying only on sequence composition are shown with upward diagonal lines, and method based only on sequence comparison (IslandPick) is shown with grid.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/10/3273/F3.medium.gif" width="440" height="279"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/9/10/3273/F3.medium.gif" width="440" height="279"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/10/3273/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/10/3273/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494445" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-35" class="first-child">Assessment of genomic island prediction tools in identifying GI nucleotides and GIs in 15 representative bacterial genomes. F-measure values for nucleotide level and island level performance assessment are shown on the Y-axis for different GI prediction methods. Island level F-measure values are shown for different overlap cutoffs (numbers in parenthesis on the X-axis; Z% cutoff means that Z% of a known GI should be identified with the predicted segment (or chain of contiguous segments) in order for the prediction to be deemed a success; see text for further details). Segmentation, contiguous clustering, and non-contiguous clustering were performed at thresholds 2Â·10<sup>â1</sup>, 10<sup>â5</sup>, and 10<sup>â3</sup> respectively. Methods based on composition, phylogenetics, and marker gene detection are shown with diagonal brick fill in bars. Bars with a dotted fill show methods using both composition and presence of marker genes for identifying GIs. Methods relying only on sequence composition are shown with upward diagonal lines, and method based only on sequence comparison (IslandPick) is shown with grid.</p><div class="sb-div caption-clear"></div></div></div><p id="p-36">The comparative genomics approach, as in IslandPick, did not perform as well as the composition based approaches, including IslandCafe (<a id="xref-table-wrap-3-6" class="xref-table" href="#T3">Table 3</a>). The performance of this class of methods is contingent upon the genomes selected for comparison and the availability of closely related genomes. Though IslandPath-Dimob that relies on the presence of mobility genes was most precise, it is, however, prone to miss true GIs (<em>e.g.</em>, those that may have lost the mobility genes), and therefore suffers from relatively low sensitivity. High sensitivity of GIHunter can be attributed to the large size of its predicted GIs (average size of GIs predicted by GIHunter is 58.2 Kbp compared to 44.9 Kbp by IslandCafe). PredictBias predicted 1268 GIs, highest among all methods (compare with IslandCafeâs 172) and therefore, as expected, has relatively high recall but low precision (<a id="xref-table-wrap-3-7" class="xref-table" href="#T3">Table 3</a>).</p><p id="p-37">Although IslandCafe and IslandViewer performed well in balancing the sensitivity and precision on an aggregated dataset comprised of genomes from 15 species, yielding higher overall accuracy, the performance of GI prediction tools may however vary on individual genomes or chromosomes. The performance of a method may depend on a number of factors, such as, compositional variability including variations in GC-content, availability of closely related genomes, size and amount of GIs in a genome, and perhaps how well a genome is annotated. IslandCafe performed well in identifying nucleotides belonging to GIs in <em>Vibrio cholerae</em>, <em>Streptococcus equi</em> and <em>Bordetella petrii</em> (F-measure and MCC &gt; 0.7) (Table S9). It also performed better than other methods on <em>Mesorhizhobium loti</em> (highest values for all accuracy metrics). All GI prediction methods had difficulty finding GI nucleotides in <em>Salmonella enterica</em> (highest F-measure value was 0.56 by ZislandExplorer) and <em>Corynebacterium diptheriae</em> (highest F-measure value was 0.6 by IslandPath-Dimob; Table S9). These results highlight the complementary strengths of GI prediction methods. Overall, IslandCafe outperformed other GI prediction methods in identifying nucleotides of the known GIs. Furthermore, IslandCafe displayed highest, performance coefficient, F-measure, and MCC for three genomes, more than any other methods.</p></div><div id="sec-19" class="subsection"><h4>Identification of GIs in real genomes:</h4><p id="p-38">At island level, highest accuracy was attained by IslandCafe for all cutoffs. At the 50% cutoff, IslandCafe outperformed other methods by up to 46% in F-measure (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a> and <a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>). IslandCafe was also most precise, outperforming other methods by up to 42% in Precision. IslandCafe outperformed the next best performing method IslandPath-Dimob by 11% in F-measure, bettering both Recall and Precision by 10% and 11% respectively. At 60% Recall, IslandViewer was most sensitive, outperforming IslandCafe by â¼1%, however, its Precision was only 19% compared to 47% of IslandCafe. Of the 442 predictions by IslandViewer, 83 were known GIs at the 50% cutoff. In contrast, of the 172 predictions by IslandCafe, 81 were known GIs. Zisland Explorer predicted fewer GIs than IslandCafe, however, it could identify only 43 knowns GIs (<a id="xref-table-wrap-4-2" class="xref-table" href="#T4">Table 4</a>). IslandCafe outperformed other methods at the 75% and 95% cutoffs as well, with the highest F-measure (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a>, Tables S10 and S11), outperforming the next best methods IslandPath-Dimob by 13% and GIHunter by 10% for 75% and 95% cutoffs respectively. At both 75% and 95% cutoffs, IslandCafe was again the most precise method, while PredictBias and IslandViewer were most sensitive (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3</a>, Table S10 and S11).</p><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/494401/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/494401/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">Performance assessment of genomic island prediction methods in identifying GIs in 15 bacterial species at the 50% overlap cutoff. At this cutoff, 50% or more of a known GI should be identified with the predicted segment (or chain of contiguous segments) in order for the prediction to be deemed a success (see Tables S9 and S10 for 75% and 95% overlap cutoffs respectively, and the text for further details). Highest value for each performance metric is shown shaded</span><div class="sb-div caption-clear"></div></div></div><p id="p-39">We also assessed the performance of GI prediction tools in localizing GIs in each genome. IslandCafe displayed highest F-measure for six genomes at 50% overlap cutoff as well as at 75% cutoff (Tables S12 and 13), more than any other methods. IslandCafe also had the highest average F-measure (per genome) at 50% and 75% cutoffs (Tables S12 and S13). At 95% cutoff, IslandCafe had the highest F-measure for three genomes and the highest F-measure per genome (Table S14). At this cutoff, IslandViewer had the highest F-measure for six genomes (Table S14). At the island level, IslandCafe thus had highest values of F-measure for up to six genomes out of fifteen, and the highest overall accuracy. Comparative assessment of IslandCafeâs performance at both nucleotide and island levels highlights its strength in identifying GIs as contiguous elements.</p><p id="p-40">Results from the nucleotide and island level assessment demonstrate the overall superior performance of IslandCafe over other methods. Although IslandCafe outperformed IslandViewer slightly in identifying nucleotides of known GIs in real genomes, its performance in identifying known GIs as single segments was substantially better than that of IslandViewer. IslandCafe compared favorably with IslandPath-Dimob as well at both nucleotide and island levels (note that both these programs require the presence of markers in their predictions). Considering that some programs such as IslandViewer take a rather holistic approach by combining several GI prediction methods to raise the accuracy bar, we show here there is still scope for substantial improvement in standalone statistical methods by utilizing biological information to augment their power of discrimination. Notably, our analysis revealed the complementary strengths of different methods, which is further elaborated in the Discussion section below.</p></div></div><div id="sec-20" class="subsection"><h3>Analysis of novel genomic islands</h3><p id="p-41">We examined GIs predicted by IslandCafe that do not overlap with known GIs. IslandCafe identified 72 novel GIs in 15 bacteria we studied (Table S15). These islands display atypical composition and were identified within clusters that were either marker-enriched or displayed unusual phyletic pattern, providing multiple lines of evidence in support of their possible horizontal acquisition. Of the 72 novel GIs identified by IslandCafe, 20 have marker genes such as transposase, integrase or phage related genes (Table S15). The compositional, phylogenetic and marker analyses performed by IslandCafe suggest that these âfalse positivesâ are likely true positives, and accuracy rendered by IslandCafe is likely much higher. This may apply to other methods as well, particularly those that rely on multiple lines of evidence. However, as revealed in this study, almost all existing methods have difficulty recovering the GI structure (see island level accuracies in <a id="xref-fig-2-10" class="xref-fig" href="#F2">Figures 2</a> and <a id="xref-fig-3-5" class="xref-fig" href="#F3">3</a>, <a id="xref-table-wrap-4-3" class="xref-table" href="#T4">Table 4</a> and, Tables S10 and S11). With a relatively much better performance in island recovery, IslandCafe offers new opportunities for researchers to investigate the novel GIs, perhaps alongside novel predictions by other promising methods such as IslandPath-Dimob.</p></div></div><div class="section discussion" id="sec-21"><h2 class="">Discussion</h2><p id="p-42">Although top-down, recursive segmentation is a powerful tool for genome analysis, adapting this approach to delineate GIs necessitates parameter optimization, robust cluster merger, and elimination of spurious predictions. In this study, we show that the combining diverse evidence helps in minimizing the sensitivity of parametric methods on parameter setting. Significance thresholds that result in optimal performance of the segmentation and clustering often vary between bacterial taxa, however, by incorporating GI specific feature enrichment and phyletic distribution analyses within this framework, we identified parameters that were globally applicable on a diverse set of bacterial species in delineating GIs with high accuracy (segmentation threshold: 2â10<sup>â1</sup>, contiguous clustering threshold: 10<sup>â5</sup>, and non-contiguous clustering threshold:10<sup>â3</sup>). Thus suggests that once clusters are reliably populated within a broad range of stringent thresholds, feature enrichment and phyletic distribution analyses suffice to group native clusters of apparently variable composition and thus enable robust detection of GIs. This could reflect on overall performance as is apparent in IslandCafeâs higher single accuracy metric values in identifying known GIs from 15 representative genomes in comparison to segmentation-clustering; the values for nucleotide-level Average Performance, Performance Coefficient, F-measure, and MCC were 0.66, 0.49, 0.66, and 0.62 respectively for the former, while 0.55, 0.34, 0.50, and 0.47 respectively for the latter (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Tables 1</a> and <a id="xref-table-wrap-3-8" class="xref-table" href="#T3">3</a>). In genome-wise comparison as well, segmentation-clustering at genome specific optimal parameter setting could not perform better than IslandCafe at its single, universal parametric setting on a majority of genomes (<a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a> and Table S8). Although genome specific optimal parameter setting is hard to realize for yet uncharacterized genomes and the quest will always be for a universal parameter setting (the âdefault settingâ) that yields the best overall performance, future efforts could focus on organism or species specific parameter setting where feasible to further enhance the performance and take the GI prediction to even greater heights.</p><p id="p-43">Since compositional disparity is a strong indicator of recent HGT (<a id="xref-ref-38-4" class="xref-bibr" href="#ref-38">Vernikos &amp; Parkhill 2006</a>), a host of methods exploiting the compositional or codon usage biases have been developed to localize alien genes or GIs (<a id="xref-ref-7-4" class="xref-bibr" href="#ref-7">de Brito <em>et al.</em> 2016</a>; <a id="xref-ref-24-6" class="xref-bibr" href="#ref-24">Jani <em>et al.</em> 2016</a>; <a id="xref-ref-39-4" class="xref-bibr" href="#ref-39">Waack <em>et al.</em> 2006</a>; <a id="xref-ref-41-9" class="xref-bibr" href="#ref-41">Wei <em>et al.</em> 2017</a>). Methods or visualizations tools that search for signals, such as, the remnants of GI integration in the recipient genome and markers or features associated with GIs have also been developed (<a id="xref-ref-20-5" class="xref-bibr" href="#ref-20">Hsiao <em>et al.</em> 2003</a>; <a id="xref-ref-31-4" class="xref-bibr" href="#ref-31">Pundhir <em>et al.</em> 2008</a>). Attempts have also been made to combine the sensitive compositional approach with the conservative signal sensor approach (<a id="xref-ref-26-4" class="xref-bibr" href="#ref-26">Langille &amp; Brinkman 2009</a>). Our approach to detect GIs via clustering and enrichment stands out among the current methods in its ability to perform unbiased detection of GIs through grouping of genomic segments of similar origin based on composition and enrichment. GIs that are weakly atypical or those that lack identifying markers are difficult to identify; clustering allows their identification via association with GIs from the same source. Marker-deficient GIs are prone to be missed by the conventional approaches. These GIs could group with marker-enriched GIs originating from a donor source within our proposed framework, and therefore, could be identified based on association. Marker-deficient GIs from a source, if allowed to be grouped and analyzed together, as in IslandCafe, may even display significantly greater enrichment than the native clusters. The predicted segments with only one or two markers are likely to be missed by conventional methods that base their predictions on abundance, unlike IslandCafe that bases its inference on grouping and enrichment. An example is shown in Table S16, where 20 segments (annotated GI 1 â GI 20) from the backbone genomes of three donors, namely, <em>D. thermolithotrophum</em>, <em>F. nucleatum</em>, and <em>M. agalactiae</em> were inserted into the backbone genome of <em>A. baumannii</em>. <em>D. thermolithotrophum</em> and <em>F. nucleatum</em> contributed 7 GIs each, while <em>M. agalactiae</em> contributed 6 GIs; half of these GIs lacked any markers. Application of optimized IslandCafe to this synthetic <em>A. baumannii</em> genome with 20 GIs revealed a robust grouping of GIs from a donor source, both with and without markers. All seven GIs from <em>D. thermolithotrophum</em> were assigned to a single cluster (Cluster ID 2, Table S16). The two <em>F. nucleatum</em> clusters (Cluster IDs 6 and 7) were also not contaminated with GIs from other sources. All 6 GIs from <em>M. agalactiae</em> were assigned to a single cluster (Cluster ID 8). This demonstrates the ability of IslandCafe to identify marker devoid GIs via their association with GIs with markers through a pipeline of segmentation, clustering, and enrichment.</p><p id="p-44">We further emphasize the augmentation of generic attributes of this pipeline through incorporation of a phylogenetic module in posteriority to enable identification of marker deficient GI clusters. By incorporating both enrichment and phylogenetic modules within an integrated framework of segmentation and clustering, we have strived to develop an approach that can work well on just sequenced, yet uncharacterized genomes. Though segmentation and clustering identify the native genome well, yet in several clusters, with some very small. These smaller native clusters show atypical composition for reasons other than HGT, however, these clusters are likely not enriched in GI specific features and/or lack unusual phyletic pattern. Therefore, IslandCafe takes a two-pronged approach (enrichment and phyletic distribution) to address this. First, it attempts to minimize false positives by allowing merger of non-enriched clusters into the largest (native) cluster and in parallel, it attempts to minimize false negatives by ensuring that those non-enriched clusters that display unusual phyletic pattern are not merged. This augments the precision substantially, without a significant decrease in recall, resulting in significant improvement in overall accuracy (<em>e.g.</em>, F-measure). The power of IslandCafe thus lies in its ability to precisely delineate GIs due to the segmentation approach, identify GIs devoid of markers by their association with marker-enriched GIs (in clusters), and identify GIs displaying unusual phyletic pattern by association again (in clusters). Whereas many predicted GIs are expected to display atypical composition <em>and</em> marker enrichment or unusual phyletic pattern, some with only atypical composition could also be identified because of association, deciphered via clustering. IslandCafe, in principle, is thus more precise than segmentation and clustering approach, while retaining the generic attributes of this approach. Our results indeed support this (compare IslandCafeâs Recall and Precision of 0.71 and 0.61 with the respective 0.72 and 0.39 by Segmentation and Clustering; <a id="xref-table-wrap-1-5" class="xref-table" href="#T1">Tables 1</a> and <a id="xref-table-wrap-3-9" class="xref-table" href="#T3">3</a>).</p><p id="p-45">As different methods often test different hypotheses, this may lead to non-converging predictions from these programs. Reconciliation of divergent sets of predictions is a challenge, however, this also indicates that no single method alone can address this problem and a holistic approach that combines the complementary strengths of different methods must be explored. We suggest to the readers, based on our comparative analysis, to collate the predictions from IslandCafe and IslandViewer into a single set for their genome of interest. Both programs base their predictions on multiple lines of evidence thereby reducing the likelihood of generating false positives. As they clearly showed complementarity by outperforming each other substantially on different sets of genomes (Tables S9, S12-S14), union of their predictions could substantially augment the sensitivity with only marginal or negligible additions of false positives. We further recommend, based on IslandCafeâs demonstrated ability to more efficiently recover the GI structure (<a id="xref-fig-2-11" class="xref-fig" href="#F2">Figures 2</a> and <a id="xref-fig-3-6" class="xref-fig" href="#F3">3</a>), that IslandCafeâs predictions be relied upon where there are disagreements between the two programs on the structures of the predicted GIs. In summary, we demonstrated the robustness and universality of the simple approach implemented in IslandCafe by assessing on synthetic test datasets and on a set of well-understood genomes sampled from different bacterial lineages. Overall, IslandCafe was found to be most accurate among the GI prediction methods, however, none of the methods could outperform all others on all genomes considered in this study. Our results reveal the complementarity of different approaches and suggest usage of compositional, phylogenetic, and functional or structural features in concert to comprehensively catalog GIs. We show here that enriching a statistical framework with biological information is a step forward in more robust delineation of GIs. Such frameworks will become even more relevant as more information encoded within genomes is deciphered and utilized within these frameworks. Future efforts could also focus on developing integrative approaches by exploiting the complementarity of different methods, which enables boosting the sensitivity without compromising the specificity (see, for example, <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Azad &amp; Lawrence 2005</a>).</p></div><div class="section ack" id="ack-1"><h2 class="">ACKNOWLEDGMENTS</h2><p id="p-46">We acknowledge the support by the University of North Texas for accomplishing this project.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-2"><p id="p-47">Supplemental material available at FigShare: <a href="https://doi.org/10.25387/g3.9255608">https://doi.org/10.25387/g3.9255608</a>.</p></li><li class="fn" id="fn-3"><p id="p-48"><em>Communicating editor: A. Rokas</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2019-04-02"><span class="received-label">Received </span>April 2, 2019.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2019-08-02"><span class="accepted-label">Accepted </span>August 2, 2019.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright Â© 2019 Jani, Azad</li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.10.3273.1" data-doi="10.1006/jmbi.1990.9999"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Altschul</span>,  <span class="cit-name-given-names">S. F.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Gish</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Miller</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. W.</span> <span class="cit-name-surname">Myers</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. J.</span> <span class="cit-name-surname">Lipman</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span>â<span class="cit-article-title">Basic local alignment search tool.</span> <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">215</span>: <span class="cit-fpage">403</span>â<span class="cit-lpage">410</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/S0022-2836(05)80360-2</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DAltschul%26rft.auinit1%253DS.%2BF.%26rft.volume%253D215%26rft.issue%253D3%26rft.spage%253D403%26rft.epage%253D410%26rft.atitle%253DBasic%2Blocal%2Balignment%2Bsearch%2Btool.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fjmbi.1990.9999%26rft_id%253Dinfo%253Apmid%252F2231712%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/jmbi.1990.9999&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2231712&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3273.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1990ED16700008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.10.3273.2" data-doi="10.1093/nar/gkp576"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Arvey</span>,  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. K.</span> <span class="cit-name-surname">Azad</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Raval</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. G.</span> <span class="cit-name-surname">Lawrence</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">Detection of genomic islands via segmental genome heterogeneity.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">5255</span>â<span class="cit-lpage">5266</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkp576</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkp576%26rft_id%253Dinfo%253Apmid%252F19589805%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkp576&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19589805&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3273.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000271569000003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.10.3273.3" data-doi="10.1371/journal.pcbi.0010056"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Azad</span>,  <span class="cit-name-given-names">R. K.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. G.</span> <span class="cit-name-surname">Lawrence</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span>â<span class="cit-article-title">Use of artificial genomes in assessing methods for atypical gene detection.</span> <abbr class="cit-jnl-abbrev">PLOS Comput. Biol.</abbr> <span class="cit-vol">1</span>: <span class="cit-elocation-id">e56</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pcbi.0010056</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BComput%2BBiol%26rft.aulast%253DAzad%26rft.auinit1%253DR.%2BK.%26rft.volume%253D1%26rft.issue%253D6%26rft.spage%253De56%26rft.epage%253De56%26rft.atitle%253DUse%2Bof%2Bartificial%2Bgenomes%2Bin%2Bassessing%2Bmethods%2Bfor%2Batypical%2Bgene%2Bdetection.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.0010056%26rft_id%253Dinfo%253Apmid%252F16292353%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.0010056&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16292353&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3273.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.10.3273.4" data-doi="10.1093/nar/gks917"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Azad</span>,  <span class="cit-name-given-names">R. K.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Li</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span>â<span class="cit-article-title">Interpreting genomic data via entropic dissection.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">41</span>: <span class="cit-elocation-id">e23</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gks917</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgks917%26rft_id%253Dinfo%253Apmid%252F23036836%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gks917&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23036836&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3273.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.10.3273.5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bertelli</span>,  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. R.</span> <span class="cit-name-surname">Laird</span>, </span></li><li><span class="cit-auth"> <span class="cit-name-given-names">K. P.</span> <span class="cit-name-surname">Williams</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. Y.</span> <span class="cit-name-surname">Lau</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Hoad</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span>âIslandViewer 4: Expanded prediction of genomic islands for larger-scale datasets. Nucleic Acids Res. 45: W30âW35.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.10.3273.6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Che</span>,  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Wang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Fazekas</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Chen</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span>â<span class="cit-article-title">An accurate genomic island prediction method for sequenced bacterial and archaeal genomes.</span> <abbr class="cit-jnl-abbrev">J. Proteomics Bioinform.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">214</span>â<span class="cit-lpage">221</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BProteomics%2BBioinform.%26rft.volume%253D7%26rft.spage%253D214%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.10.3273.7" data-doi="10.1371/journal.pone.0146352"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de Brito</span>,  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Maracaja-Coutinho</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. T.</span> <span class="cit-name-surname">de Farias</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L. V.</span> <span class="cit-name-surname">Batista</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. G.</span> <span class="cit-name-surname">do RÃªgo</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span>â<span class="cit-article-title">A novel method to predict genomic islands based on mean shift clustering algorithm.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">11</span>: <span class="cit-elocation-id">e0146352</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0146352</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D11%26rft.spage%253De0146352%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0146352%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0146352&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.10.3273.8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Denton</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N. J.</span> <span class="cit-name-surname">Todd</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. G.</span> <span class="cit-name-surname">Kerr</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. M.</span> <span class="cit-name-surname">Hawkey</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">Littlewood</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span>â<span class="cit-article-title">Molecular epidemiology of stenotrophomonas maltophilia isolated from clinical specimens from patients with cystic fibrosis and associated environmental samples.</span> <abbr class="cit-jnl-abbrev">J. Clin. Microbiol.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">1953</span>â<span class="cit-lpage">1958</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BClinical%2BMicrobiology%26rft.stitle%253DJ.%2BClin.%2BMicrobiol.%26rft.aulast%253DDenton%26rft.auinit1%253DM.%26rft.volume%253D36%26rft.issue%253D7%26rft.spage%253D1953%26rft.epage%253D1958%26rft.atitle%253DMolecular%2BEpidemiology%2Bof%2BStenotrophomonas%2Bmaltophilia%2BIsolated%2Bfrom%2BClinical%2BSpecimens%2Bfrom%2BPatients%2Bwith%2BCystic%2BFibrosis%2Band%2BAssociated%2BEnvironmental%2BSamples%26rft_id%253Dinfo%253Apmid%252F9650943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamNtIjtzOjU6InJlc2lkIjtzOjk6IjM2LzcvMTk1MyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzkvMTAvMzI3My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.10.3273.9" data-doi="10.1128/JB.185.6.1831-1840.2003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dobrindt</span>,  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Agerer</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Michaelis</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Janka</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Buchrieser</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span>â<span class="cit-article-title">Analysis of genome plasticity in pathogenic and commensal escherichia coli isolates by use of DNA arrays.</span> <abbr class="cit-jnl-abbrev">J. Bacteriol.</abbr> <span class="cit-vol">185</span>: <span class="cit-fpage">1831</span>â<span class="cit-lpage">1840</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1128/JB.185.6.1831-1840.2003</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBacteriology%26rft.stitle%253DJ.%2BBacteriol.%26rft.aulast%253DDobrindt%26rft.auinit1%253DU.%26rft.volume%253D185%26rft.issue%253D6%26rft.spage%253D1831%26rft.epage%253D1840%26rft.atitle%253DAnalysis%2Bof%2BGenome%2BPlasticity%2Bin%2BPathogenic%2Band%2BCommensal%2BEscherichia%2Bcoli%2BIsolates%2Bby%2BUse%2Bof%2BDNA%2BArrays%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJB.185.6.1831-1840.2003%26rft_id%253Dinfo%253Apmid%252F12618447%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6MTA6IjE4NS82LzE4MzEiO3M6NDoiYXRvbSI7czoxOToiL2dnZy85LzEwLzMyNzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.10.3273.10" data-doi="10.1038/nrmicro884"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dobrindt</span>,  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Hochhut</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">U.</span> <span class="cit-name-surname">Hentschel</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Hacker</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span>â<span class="cit-article-title">Genomic islands in pathogenic and environmental microorganisms.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Microbiol.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">414</span>â<span class="cit-lpage">424</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrmicro884</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BMicrobiology%26rft.stitle%253DNat%2BRev%2BMicrobiol%26rft.aulast%253DDobrindt%26rft.auinit1%253DU.%26rft.volume%253D2%26rft.issue%253D5%26rft.spage%253D414%26rft.epage%253D424%26rft.atitle%253DGenomic%2Bislands%2Bin%2Bpathogenic%2Band%2Benvironmental%2Bmicroorganisms.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrmicro884%26rft_id%253Dinfo%253Apmid%252F15100694%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrmicro884&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15100694&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3273.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000221589700015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.10.3273.11" data-doi="10.3201/eid0905.020609"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Doublet</span>,  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Lailler</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Meunier</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Brisabois</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Boyd</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span>â<span class="cit-article-title">Variant salmonella genomic island 1 antibiotic resistance gene cluster in salmonella enterica serovar albany.</span> <abbr class="cit-jnl-abbrev">Emerg. Infect. Dis.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">585</span>â<span class="cit-lpage">591</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3201/eid0905.020609</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEmerging%2Binfectious%2Bdiseases%26rft.stitle%253DEmerg%2BInfect%2BDis%26rft.aulast%253DDoublet%26rft.auinit1%253DB.%26rft.volume%253D9%26rft.issue%253D5%26rft.spage%253D585%26rft.epage%253D591%26rft.atitle%253DVariant%2BSalmonella%2Bgenomic%2Bisland%2B1%2Bantibiotic%2Bresistance%2Bgene%2Bcluster%2Bin%2BSalmonella%2Benterica%2Bserovar%2BAlbany.%26rft_id%253Dinfo%253Adoi%252F10.3201%252Feid0905.020609%26rft_id%253Dinfo%253Apmid%252F12737743%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3201/eid0905.020609&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12737743&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3273.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000182569700012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">âµ</a><div class="cit ref-cit ref-web" id="cit-9.10.3273.12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eddy</span>,  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span>âHMMER userâs guide, version December 1998. St. Louis. <a href="http://hmmer.wustl.edu/">http://hmmer.wustl.edu/</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.10.3273.13" data-doi="10.1093/nar/gkr367"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Finn</span>,  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Clements</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. R.</span> <span class="cit-name-surname">Eddy</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span>â<span class="cit-article-title">HMMER web server: Interactive sequence similarity searching.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">W29</span>â<span class="cit-lpage">W37</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkr367</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkr367%26rft_id%253Dinfo%253Apmid%252F21593126%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkr367&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21593126&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3273.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000292325300006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.10.3273.14" data-doi="10.1093/nar/gkv1344"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Finn</span>,  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Coggill</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. Y.</span> <span class="cit-name-surname">Eberhardt</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. R.</span> <span class="cit-name-surname">Eddy</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Mistry</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span>â<span class="cit-article-title">The pfam protein families database: Towards a more sustainable future.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">D279</span>â<span class="cit-lpage">D285</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkv1344</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkv1344%26rft_id%253Dinfo%253Apmid%252F26673716%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkv1344&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26673716&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3273.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.10.3273.15" data-doi="10.1186/1471-2105-9-333"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ganesan</span>,  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. S.</span> <span class="cit-name-surname">Rakitianskaia</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. F.</span> <span class="cit-name-surname">Davenport</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Tummler</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">O. N.</span> <span class="cit-name-surname">Reva</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span>â<span class="cit-article-title">The SeqWord genome browser: An online tool for the identification and visualization of atypical regions of bacterial genomes through oligonucleotide usage.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">333</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2105-9-333</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DGanesan%26rft.auinit1%253DH.%26rft.volume%253D9%26rft.spage%253D333%26rft.epage%253D333%26rft.atitle%253DThe%2BSeqWord%2BGenome%2BBrowser%253A%2Ban%2Bonline%2Btool%2Bfor%2Bthe%2Bidentification%2Band%2Bvisualization%2Bof%2Batypical%2Bregions%2Bof%2Bbacterial%2Bgenomes%2Bthrough%2Boligonucleotide%2Busage.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-9-333%26rft_id%253Dinfo%253Apmid%252F18687122%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-9-333&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18687122&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3273.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.10.3273.16" data-doi="10.1093/nar/gkg004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Garcia-Vallve</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Guzman</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. A.</span> <span class="cit-name-surname">Montero</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Romeu</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span>â<span class="cit-article-title">HGT-DB: A database of putative horizontally transferred genes in prokaryotic complete genomes.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">187</span>â<span class="cit-lpage">189</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkg004</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DGarcia-Vallve%26rft.auinit1%253DS.%26rft.volume%253D31%26rft.issue%253D1%26rft.spage%253D187%26rft.epage%253D189%26rft.atitle%253DHGT-DB%253A%2Ba%2Bdatabase%2Bof%2Bputative%2Bhorizontally%2Btransferred%2Bgenes%2Bin%2Bprokaryotic%2Bcomplete%2Bgenomes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkg004%26rft_id%253Dinfo%253Apmid%252F12519978%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkg004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12519978&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3273.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000181079700042&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.10.3273.17" data-doi="10.1103/PhysRevE.65.041905"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Grosse</span>,  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Bernaola-GalvÃ¡n</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Carpena</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">RomÃ¡n-RoldÃ¡n</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Oliver</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span>â<span class="cit-article-title">Analysis of symbolic sequences using the jensen-shannon divergence.</span> <abbr class="cit-jnl-abbrev">Phys. Rev. E Stat. Nonlin. Soft Matter Phys.</abbr> <span class="cit-vol">65</span>: <span class="cit-elocation-id">041905</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1103/PhysRevE.65.041905</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhysical%2Breview.%2BE%252C%2BStatistical%252C%2Bnonlinear%252C%2Band%2Bsoft%2Bmatter%2Bphysics%26rft.stitle%253DPhys%2BRev%2BE%2BStat%2BNonlin%2BSoft%2BMatter%2BPhys%26rft.aulast%253DGrosse%26rft.auinit1%253DI.%26rft.volume%253D65%26rft.issue%253D4%2BPt%2B1%26rft.spage%253D041905%26rft.epage%253D041905%26rft.atitle%253DAnalysis%2Bof%2Bsymbolic%2Bsequences%2Busing%2Bthe%2BJensen-Shannon%2Bdivergence.%26rft_id%253Dinfo%253Adoi%252F10.1103%252FPhysRevE.65.041905%26rft_id%253Dinfo%253Apmid%252F12005871%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1103/PhysRevE.65.041905&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12005871&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3273.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.10.3273.18" data-doi="10.1093/embo-reports/kve097"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hacker</span>,  <span class="cit-name-given-names">J.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Carniel</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span>â<span class="cit-article-title">Ecological fitness, genomic islands and bacterial pathogenicity. A darwinian view of the evolution of microbes.</span> <abbr class="cit-jnl-abbrev">EMBO Rep.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">376</span>â<span class="cit-lpage">381</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/embo-reports/kve097</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEMBO%2Breports%26rft.stitle%253DEMBO%2BRep.%26rft.aulast%253DHacker%26rft.auinit1%253DJ.%26rft.volume%253D2%26rft.issue%253D5%26rft.spage%253D376%26rft.epage%253D381%26rft.atitle%253DEcological%2Bfitness%252C%2Bgenomic%2Bislands%2Band%2Bbacterial%2Bpathogenicity%253A%2BA%2BDarwinian%2Bview%2Bof%2Bthe%2Bevolution%2Bof%2Bmicrobes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fembo-reports%252Fkve097%26rft_id%253Dinfo%253Apmid%252F11375927%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NToiZW1ib3IiO3M6NToicmVzaWQiO3M6NzoiMi81LzM3NiI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzkvMTAvMzI3My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.10.3273.19" data-doi="10.1128/JB.01230-08"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Holden</span>,  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H. M.</span> <span class="cit-name-surname">Seth-Smith</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L. C.</span> <span class="cit-name-surname">Crossman</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Sebaihia</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. D.</span> <span class="cit-name-surname">Bentley</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">The genome of burkholderia cenocepacia J2315, an epidemic pathogen of cystic fibrosis patients.</span> <abbr class="cit-jnl-abbrev">J. Bacteriol.</abbr> <span class="cit-vol">191</span>: <span class="cit-fpage">261</span>â<span class="cit-lpage">277</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1128/JB.01230-08</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBacteriol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJB.01230-08%26rft_id%253Dinfo%253Apmid%252F18931103%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6OToiMTkxLzEvMjYxIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMC8zMjczLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.10.3273.20" data-doi="10.1093/bioinformatics/btg004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hsiao</span>,  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Wan</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. J.</span> <span class="cit-name-surname">Jones</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">F. S.</span> <span class="cit-name-surname">Brinkman</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span>â<span class="cit-article-title">IslandPath: Aiding detection of genomic islands in prokaryotes.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">418</span>â<span class="cit-lpage">420</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btg004</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DHsiao%26rft.auinit1%253DW.%26rft.volume%253D19%26rft.issue%253D3%26rft.spage%253D418%26rft.epage%253D420%26rft.atitle%253DIslandPath%253A%2Baiding%2Bdetection%2Bof%2Bgenomic%2Bislands%2Bin%2Bprokaryotes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtg004%26rft_id%253Dinfo%253Apmid%252F12584130%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btg004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12584130&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3273.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000181303000015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.10.3273.21" data-doi="10.1371/journal.pgen.0010062"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hsiao</span>,  <span class="cit-name-given-names">W. W.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Ung</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Aeschliman</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Bryan</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. B.</span> <span class="cit-name-surname">Finlay</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span>â<span class="cit-article-title">Evidence of a large novel gene pool associated with prokaryotic genomic islands.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">1</span>: <span class="cit-elocation-id">e62</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.0010062</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DHsiao%26rft.auinit1%253DW.%2BW.%26rft.volume%253D1%26rft.issue%253D5%26rft.spage%253De62%26rft.epage%253De62%26rft.atitle%253DEvidence%2Bof%2Ba%2Blarge%2Bnovel%2Bgene%2Bpool%2Bassociated%2Bwith%2Bprokaryotic%2Bgenomic%2Bislands.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.0010062%26rft_id%253Dinfo%253Apmid%252F16299586%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.0010062&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16299586&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3273.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.10.3273.22" data-doi="10.1186/1471-2105-11-119"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hyatt</span>,  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Chen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. F.</span> <span class="cit-name-surname">LoCascio</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. L.</span> <span class="cit-name-surname">Land</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F. W.</span> <span class="cit-name-surname">Larimer</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title">Prodigal: Prokaryotic gene recognition and translation initiation site identification.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">119</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2105-11-119</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DHyatt%26rft.auinit1%253DD.%26rft.volume%253D11%26rft.spage%253D119%26rft.epage%253D119%26rft.atitle%253DProdigal%253A%2Bprokaryotic%2Bgene%2Brecognition%2Band%2Btranslation%2Binitiation%2Bsite%2Bidentification.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-11-119%26rft_id%253Dinfo%253Apmid%252F20211023%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-11-119&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20211023&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3273.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.10.3273.23" data-doi="10.1145/2500124.2500126"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jani</span>,  <span class="cit-name-given-names">M.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. K.</span> <span class="cit-name-surname">Azad</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span>â<span class="cit-article-title">Information entropy based methods for genome comparison.</span> <abbr class="cit-jnl-abbrev">ACM SIGBioinformatics Record</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">1</span>â<span class="cit-lpage">4</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1145/2500124.2500126</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DACM%2BSIGBioinformatics%2BRecord%26rft.volume%253D3%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1145%252F2500124.2500126%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1145/2500124.2500126&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.10.3273.24" data-doi="10.3389/fmicb.2016.01210"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jani</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Mathee</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. K.</span> <span class="cit-name-surname">Azad</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span>â<span class="cit-article-title">Identification of novel genomic islands in liverpool epidemic strain of pseudomonas aeruginosa using segmentation and clustering.</span> <abbr class="cit-jnl-abbrev">Front. Microbiol.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">1210</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fmicb.2016.01210</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BMicrobiol.%26rft.volume%253D7%26rft.spage%253D1210%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffmicb.2016.01210%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fmicb.2016.01210&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.10.3273.25" data-doi="10.1111/j.1574-6976.2008.00136.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Juhas</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. R.</span> <span class="cit-name-surname">van der Meer</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Gaillard</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. M.</span> <span class="cit-name-surname">Harding</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. W.</span> <span class="cit-name-surname">Hood</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">Genomic islands: Tools of bacterial horizontal gene transfer and evolution.</span> <abbr class="cit-jnl-abbrev">FEMS Microbiol. Rev.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">376</span>â<span class="cit-lpage">393</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1574-6976.2008.00136.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEMS%2BMicrobiol.%2BRev.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1574-6976.2008.00136.x%26rft_id%253Dinfo%253Apmid%252F19178566%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1574-6976.2008.00136.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19178566&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3273.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000263357200006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.10.3273.26" data-doi="10.1093/bioinformatics/btp030"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Langille</span>,  <span class="cit-name-given-names">M. G.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">F. S.</span> <span class="cit-name-surname">Brinkman</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">IslandViewer: An integrated interface for computational identification and visualization of genomic islands.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">664</span>â<span class="cit-lpage">665</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btp030</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp030%26rft_id%253Dinfo%253Apmid%252F19151094%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp030&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19151094&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3273.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000263834600017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.10.3273.27" data-doi="10.1186/1471-2105-9-329"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Langille</span>,  <span class="cit-name-given-names">M. G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W. W.</span> <span class="cit-name-surname">Hsiao</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">F. S.</span> <span class="cit-name-surname">Brinkman</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span>â<span class="cit-article-title">Evaluation of genomic island predictors using a comparative genomics approach.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">329</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2105-9-329</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DLangille%26rft.auinit1%253DM.%2BG.%26rft.volume%253D9%26rft.spage%253D329%26rft.epage%253D329%26rft.atitle%253DEvaluation%2Bof%2Bgenomic%2Bisland%2Bpredictors%2Busing%2Ba%2Bcomparative%2Bgenomics%2Bapproach.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-9-329%26rft_id%253Dinfo%253Apmid%252F18680607%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-9-329&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18680607&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3273.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.10.3273.28" data-doi="10.1038/nrmicro2350"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Langille</span>,  <span class="cit-name-given-names">M. G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W. W.</span> <span class="cit-name-surname">Hsiao</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">F. S.</span> <span class="cit-name-surname">Brinkman</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title">Detecting genomic islands using bioinformatics approaches.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Microbiol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">373</span>â<span class="cit-lpage">382</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrmicro2350</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BMicrobiology%26rft.stitle%253DNat%2BRev%2BMicrobiol%26rft.aulast%253DLangille%26rft.auinit1%253DM.%2BG.%26rft.volume%253D8%26rft.issue%253D5%26rft.spage%253D373%26rft.epage%253D382%26rft.atitle%253DDetecting%2Bgenomic%2Bislands%2Busing%2Bbioinformatics%2Bapproaches.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrmicro2350%26rft_id%253Dinfo%253Apmid%252F20395967%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrmicro2350&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20395967&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3273.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.10.3273.29" data-doi="10.1073/pnas.95.16.9413"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lawrence</span>,  <span class="cit-name-given-names">J. G.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Ochman</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span>â<span class="cit-article-title">Molecular archaeology of the escherichia coli genome.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">95</span>: <span class="cit-fpage">9413</span>â<span class="cit-lpage">9417</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.95.16.9413</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DLawrence%26rft.auinit1%253DJ.%2BG.%26rft.volume%253D95%26rft.issue%253D16%26rft.spage%253D9413%26rft.epage%253D9417%26rft.atitle%253DMolecular%2Barchaeology%2Bof%2Bthe%2BEscherichia%2Bcoli%2Bgenome%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.95.16.9413%26rft_id%253Dinfo%253Apmid%252F9689094%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiOTUvMTYvOTQxMyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzkvMTAvMzI3My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.10.3273.30" data-doi="10.1016/S0147-619X(02)00117-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Osborn</span>,  <span class="cit-name-given-names">A. M.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">BÃ¶ltner</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span>â<span class="cit-article-title">When phage, plasmids, and transposons collide: Genomic islands, and conjugative-and mobilizable-transposons as a mosaic continuum.</span> <abbr class="cit-jnl-abbrev">Plasmid</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">202</span>â<span class="cit-lpage">212</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/S0147-619X(02)00117-8</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlasmid%26rft.stitle%253DPlasmid%26rft.aulast%253DOsborn%26rft.auinit1%253DA.%2BM.%26rft.volume%253D48%26rft.issue%253D3%26rft.spage%253D202%26rft.epage%253D212%26rft.atitle%253DWhen%2Bphage%252C%2Bplasmids%252C%2Band%2Btransposons%2Bcollide%253A%2Bgenomic%2Bislands%252C%2Band%2Bconjugative-%2Band%2Bmobilizable-transposons%2Bas%2Ba%2Bmosaic%2Bcontinuum.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0147-619X%252802%252900117-8%26rft_id%253Dinfo%253Apmid%252F12460536%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0147-619X(02)00117-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12460536&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3273.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000179823700006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.10.3273.31"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pundhir</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Vijayvargiya</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Kumar</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span>â<span class="cit-article-title">PredictBias: A server for the identification of genomic and pathogenicity islands in prokaryotes.</span> <abbr class="cit-jnl-abbrev">In Silico Biol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">223</span>â<span class="cit-lpage">234</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DIn%2Bsilico%2Bbiology%26rft.stitle%253DIn%2BSilico%2BBiol%26rft.aulast%253DPundhir%26rft.auinit1%253DS.%26rft.volume%253D8%26rft.issue%253D3-4%26rft.spage%253D223%26rft.epage%253D234%26rft.atitle%253DPredictBias%253A%2Ba%2Bserver%2Bfor%2Bthe%2Bidentification%2Bof%2Bgenomic%2Band%2Bpathogenicity%2Bislands%2Bin%2Bprokaryotes.%26rft_id%253Dinfo%253Apmid%252F19032158%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=19032158&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3273.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.10.3273.32" data-doi="10.1093/nar/17.5.1907"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Reiter</span>,  <span class="cit-name-given-names">W. D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Palm</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Yeats</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span>â<span class="cit-article-title">Transfer RNA genes frequently serve as integration sites for prokaryotic genetic elements.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">1907</span>â<span class="cit-lpage">1914</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/17.5.1907</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DReiter%26rft.auinit1%253DW.-D.%26rft.volume%253D17%26rft.issue%253D5%26rft.spage%253D1907%26rft.epage%253D1914%26rft.atitle%253DTransfer%2BRNA%2Bgenes%2Bfrequently%2Bserve%2Bas%2Bintegration%2Bsites%2Bfor%2Bprokaryotic%2Bgenetic%2Belements%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F17.5.1907%26rft_id%253Dinfo%253Apmid%252F2467253%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/17.5.1907&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2467253&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3273.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1989T691100010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.10.3273.33"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Reyrat</span>,  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Pelicic</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Gicquel</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Rappuoli</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span>â<span class="cit-article-title">Counterselectable markers: Untapped tools for bacterial genetics and pathogenesis.</span> <abbr class="cit-jnl-abbrev">Infect. Immun.</abbr> <span class="cit-vol">66</span>: <span class="cit-fpage">4011</span>â<span class="cit-lpage">4017</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInfection%2Band%2BImmunity%26rft.stitle%253DInfect.%2BImmun.%26rft.aulast%253DReyrat%26rft.auinit1%253DJ.-M.%26rft.volume%253D66%26rft.issue%253D9%26rft.spage%253D4011%26rft.epage%253D4017%26rft.atitle%253DCounterselectable%2BMarkers%253A%2BUntapped%2BTools%2Bfor%2BBacterial%2BGenetics%2Band%2BPathogenesis%26rft_id%253Dinfo%253Apmid%252F9712740%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiaWFpIjtzOjU6InJlc2lkIjtzOjk6IjY2LzkvNDAxMSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzkvMTAvMzI3My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.10.3273.34" data-doi="10.1128/CMR.17.1.14-56.2004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schmidt</span>,  <span class="cit-name-given-names">H.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Hensel</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span>â<span class="cit-article-title">Pathogenicity islands in bacterial pathogenesis.</span> <abbr class="cit-jnl-abbrev">Clin. Microbiol. Rev.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">14</span>â<span class="cit-lpage">56</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1128/CMR.17.1.14-56.2004</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DClinical%2BMicrobiology%2BReviews%26rft.stitle%253DClin.%2BMicrobiol.%2BRev.%26rft.aulast%253DSchmidt%26rft.auinit1%253DH.%26rft.volume%253D17%26rft.issue%253D1%26rft.spage%253D14%26rft.epage%253D56%26rft.atitle%253DPathogenicity%2BIslands%2Bin%2BBacterial%2BPathogenesis%26rft_id%253Dinfo%253Adoi%252F10.1128%252FCMR.17.1.14-56.2004%26rft_id%253Dinfo%253Apmid%252F14726454%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiY21yIjtzOjU6InJlc2lkIjtzOjc6IjE3LzEvMTQiO3M6NDoiYXRvbSI7czoxOToiL2dnZy85LzEwLzMyNzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.10.3273.35" data-doi="10.1016/j.jbiotec.2015.09.008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Soares</span>,  <span class="cit-name-given-names">S. C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Geyik</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. T.</span> <span class="cit-name-surname">Ramos</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. H.</span> <span class="cit-name-surname">de SÃ¡</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. G.</span> <span class="cit-name-surname">Barbosa</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span>â<span class="cit-article-title">GIPSy: Genomic island prediction software.</span> <abbr class="cit-jnl-abbrev">J. Biotechnol.</abbr> <span class="cit-vol">232</span>: <span class="cit-fpage">2</span>â<span class="cit-lpage">11</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.jbiotec.2015.09.008</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiotechnol.%26rft.volume%253D232%26rft.spage%253D2%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jbiotec.2015.09.008%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jbiotec.2015.09.008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.10.3273.36" data-doi="10.1038/nrg3962"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Soucy</span>,  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Huang</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. P.</span> <span class="cit-name-surname">Gogarten</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span>â<span class="cit-article-title">Horizontal gene transfer: Building the web of life.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">472</span>â<span class="cit-lpage">482</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg3962</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BGenet.%26rft.volume%253D16%26rft.spage%253D472%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3962%26rft_id%253Dinfo%253Apmid%252F26184597%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3962&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26184597&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3273.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.10.3273.37" data-doi="10.1103/PhysRevE.75.011915"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thakur</span>,  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. K.</span> <span class="cit-name-surname">Azad</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Ramaswamy</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span>â<span class="cit-article-title">Markov models of genome segmentation.</span> <abbr class="cit-jnl-abbrev">Phys. Rev. E Stat. Nonlin. Soft Matter Phys.</abbr> <span class="cit-vol">75</span>: <span class="cit-elocation-id">011915</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1103/PhysRevE.75.011915</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhysical%2Breview.%2BE%252C%2BStatistical%252C%2Bnonlinear%252C%2Band%2Bsoft%2Bmatter%2Bphysics%26rft.stitle%253DPhys%2BRev%2BE%2BStat%2BNonlin%2BSoft%2BMatter%2BPhys%26rft.aulast%253DThakur%26rft.auinit1%253DV.%26rft.volume%253D75%26rft.issue%253D1%2BPt%2B1%26rft.spage%253D011915%26rft.epage%253D011915%26rft.atitle%253DMarkov%2Bmodels%2Bof%2Bgenome%2Bsegmentation.%26rft_id%253Dinfo%253Adoi%252F10.1103%252FPhysRevE.75.011915%26rft_id%253Dinfo%253Apmid%252F17358192%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1103/PhysRevE.75.011915&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17358192&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3273.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.10.3273.38" data-doi="10.1093/bioinformatics/btl369"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vernikos</span>,  <span class="cit-name-given-names">G. S.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Parkhill</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span>â<span class="cit-article-title">Interpolated variable order motifs for identification of horizontally acquired DNA: Revisiting the salmonella pathogenicity islands.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">2196</span>â<span class="cit-lpage">2203</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btl369</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtl369%26rft_id%253Dinfo%253Apmid%252F16837528%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btl369&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16837528&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3273.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000240588800004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.10.3273.39" data-doi="10.1186/1471-2105-7-142"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Waack</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">O.</span> <span class="cit-name-surname">Keller</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Asper</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Brodag</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Damm</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span>â<span class="cit-article-title">Score-based prediction of genomic islands in prokaryotic genomes using hidden markov models.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">142</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2105-7-142</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DWaack%26rft.auinit1%253DS.%26rft.volume%253D7%26rft.spage%253D142%26rft.epage%253D142%26rft.atitle%253DScore-based%2Bprediction%2Bof%2Bgenomic%2Bislands%2Bin%2Bprokaryotic%2Bgenomes%2Busing%2Bhidden%2BMarkov%2Bmodels.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-7-142%26rft_id%253Dinfo%253Apmid%252F16542435%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-7-142&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16542435&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3273.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">âµ</a><div class="cit ref-cit ref-web" id="cit-9.10.3273.40"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Fazekas</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Booth</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Q.</span> <span class="cit-name-surname">Liu</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Che</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span>âAn integrative approach for genomic island prediction in prokaryotic genomes. International Symposium on Bioinformatics Research and Applications. Springer, Berlin, Heidelberg. 404â415.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.10.3273.41" data-doi="10.1093/bib/bbw019"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wei</span>,  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Gao</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. Z.</span> <span class="cit-name-surname">Du</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H. L.</span> <span class="cit-name-surname">Hua</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Wang</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span>â<span class="cit-article-title">Zisland explorer: Detect genomic islands by combining homogeneity and heterogeneity properties.</span> <abbr class="cit-jnl-abbrev">Brief. Bioinform.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">357</span>â<span class="cit-lpage">366</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bib/bbw019</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBrief.%2BBioinform.%26rft.volume%253D18%26rft.spage%253D357%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbib%252Fbbw019%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bib/bbw019&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.10.3273.42" data-doi="10.1099/0022-1317-51-6-459"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Winstanley</span>,  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span>â<span class="cit-article-title">Spot the difference: Applications of subtractive hybridisation to the study of bacterial pathogens.</span> <abbr class="cit-jnl-abbrev">J. Med. Microbiol.</abbr> <span class="cit-vol">51</span>: <span class="cit-fpage">459</span>â<span class="cit-lpage">467</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1099/0022-1317-51-6-459</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMedical%2BMicrobiology%26rft.stitle%253DJ%2BMed%2BMicrobiol%26rft.aulast%253DWINSTANLEY%26rft.auinit1%253DC.%26rft.volume%253D51%26rft.issue%253D6%26rft.spage%253D459%26rft.epage%253D467%26rft.atitle%253DSpot%2Bthe%2Bdifference%253A%2Bapplications%2Bof%2Bsubtractive%2Bhybridisation%2Bto%2Bthe%2Bstudy%2Bof%2Bbacterial%2Bpathogens%26rft_id%253Dinfo%253Adoi%252F10.1099%252F0022-1317-51-6-459%26rft_id%253Dinfo%253Apmid%252F12018652%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1099/0022-1317-51-6-459&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12018652&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3273.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000175519200002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.10.3273.43" data-doi="10.1101/gr.086082.108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Winstanley</span>,  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. G.</span> <span class="cit-name-surname">Langille</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. L.</span> <span class="cit-name-surname">Fothergill</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Kukavica-Ibrulj</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Paradis-Bleau</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">Newly introduced genomic prophage islands are critical determinants of in vivo competitiveness in the liverpool epidemic strain of pseudomonas aeruginosa.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">12</span>â<span class="cit-lpage">23</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.086082.108</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.086082.108%26rft_id%253Dinfo%253Apmid%252F19047519%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjc6IjE5LzEvMTIiO3M6NDoiYXRvbSI7czoxOToiL2dnZy85LzEwLzMyNzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/9/10/3273.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/10/3263" title="De Novo Genome Assembly and Comparative Genomics of the Barley Leaf Rust Pathogen Puccinia hordei Identifies Candidates for Three Avirulence Genes" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/10/3287" title="Comparative Transcriptomic Analysis Reveals Molecular Profiles of Central Nervous System in Maternal Diapause Induction of Locusta migratoria" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/9/10"><span>Volume 9 Issue 10, October 2019</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/9/10.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/9/10.cover-source.jpg?itok=yA8kifEd" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 9 (10)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/10/3273" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-aJ8WyDyA8JcNSPJzMaDRQmmArwwNGXL1i3D89uBz6Pw" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/9/10/3273" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/9/10/3273" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/9/10/3273" class="active" data-icon-position="" data-hide-link-title="0">IslandCafe: Compositional Anomaly and Feature Enrichment Assessment for Delineation of Genomic Islands</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/494276" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-QaRLBdfR7tU1akV8Nvj0gDdgZqAOTGEsuphce_HmYWI" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/10/3273.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/9/10/3273" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-KpMRAH3uNNfURgMs3NyF0k9M4Ug_-Ij1658xcGt150U" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/9/10/3273" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/10/3273.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494276" id="node494276--2" data-pisa="ggg;9/10/3273" data-pisa-master="ggg;g3.119.400562" data-apath="/ggg/9/10/3273.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">IslandCafe: Compositional Anomaly and Feature Enrichment Assessment for Delineation of Genomic Islands</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-2486-5696" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Mehul Jani</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-6874-6146" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Rajeev K. Azad</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3273-3285;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400562 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494276--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Mehul Jani </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biological Sciences and BioDiscovery Institute and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mehul%2BJani%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jani%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMehul%2BJani%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-2486-5696" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Mehul Jani</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Rajeev K. Azad </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biological Sciences and BioDiscovery Institute and</div><div class='author-affiliation'><span class='nlm-sup'>â </span>Department of Mathematics, University of North Texas, Denton, Texas 76203</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rajeev%2BK.%2BAzad%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Azad%20RK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARajeev%2BK.%2BAzad%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-6874-6146" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Rajeev K. Azad</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:Rajeev.Azad@unt.edu" class="" data-icon-position="" data-hide-link-title="0">Rajeev.Azad@unt.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/9/10/3273" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/9/10/3273&amp;title=IslandCafe%3A%20Compositional%20Anomaly%20and%20Feature%20Enrichment%20Assessment%20for%20Delineation%20of%20Genomic%20Islands" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/9/10/3273&amp;title=IslandCafe%3A%20Compositional%20Anomaly%20and%20Feature%20Enrichment%20Assessment%20for%20Delineation%20of%20Genomic%20Islands" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/9/10/3273&amp;title=IslandCafe%3A%20Compositional%20Anomaly%20and%20Feature%20Enrichment%20Assessment%20for%20Delineation%20of%20Genomic%20Islands" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/9/10/3273&amp;text=IslandCafe%3A%20Compositional%20Anomaly%20and%20Feature%20Enrichment%20Assessment%20for%20Delineation%20of%20Genomic%20Islands" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/9/10/3273&amp;title=IslandCafe%3A%20Compositional%20Anomaly%20and%20Feature%20Enrichment%20Assessment%20for%20Delineation%20of%20Genomic%20Islands" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/9/10/3273&amp;t=IslandCafe%3A%20Compositional%20Anomaly%20and%20Feature%20Enrichment%20Assessment%20for%20Delineation%20of%20Genomic%20Islands" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/9/10/3273&amp;title=IslandCafe%3A%20Compositional%20Anomaly%20and%20Feature%20Enrichment%20Assessment%20for%20Delineation%20of%20Genomic%20Islands" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/9/10/3273&amp;title=IslandCafe%3A%20Compositional%20Anomaly%20and%20Feature%20Enrichment%20Assessment%20for%20Delineation%20of%20Genomic%20Islands" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/494276/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494276" id="node494276--3" data-pisa="ggg;9/10/3273" data-pisa-master="ggg;g3.119.400562" data-apath="/ggg/9/10/3273.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">IslandCafe: Compositional Anomaly and Feature Enrichment Assessment for Delineation of Genomic Islands</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-2486-5696" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Mehul Jani</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-6874-6146" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Rajeev K. Azad</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3273-3285;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400562 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494276--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Mehul Jani </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biological Sciences and BioDiscovery Institute and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mehul%2BJani%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jani%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMehul%2BJani%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-2486-5696" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Mehul Jani</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Rajeev K. Azad </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biological Sciences and BioDiscovery Institute and</div><div class='author-affiliation'><span class='nlm-sup'>â </span>Department of Mathematics, University of North Texas, Denton, Texas 76203</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rajeev%2BK.%2BAzad%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Azad%20RK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARajeev%2BK.%2BAzad%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-6874-6146" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Rajeev K. Azad</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:Rajeev.Azad@unt.edu" class="" data-icon-position="" data-hide-link-title="0">Rajeev.Azad@unt.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=3273&amp;rft.epage=3285&amp;rft.atitle=IslandCafe%3A%20Compositional%20Anomaly%20and%20Feature%20Enrichment%20Assessment%20for%20Delineation%20of%20Genomic%20Islands&amp;rft.volume=9&amp;rft.issue=10&amp;rft.date=2019-10-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Jani%2C+Mehul&amp;rft.au=Azad%2C+Rajeev+K."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/494276/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/494276/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/494276/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/494276/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/494276/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/494276/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/494276/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/494276/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/494276/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/494276/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/494276/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/494276/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/9/10/3273" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-9" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-21" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">ACKNOWLEDGMENTS</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/9/10/3273.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/9/10/3273.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/9/10/3273.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
