<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/9/10/3369" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/9/10/3369.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/9/10/3369.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/9/10/3369.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/9/10/3369/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/9/10/3369.atom" />
<meta name="HW.pisa" content="ggg;9/10/3369" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Predicting Longitudinal Traits Derived from High-Throughput Phenomics in Contrasting Environments Using Genomic Legendre Polynomials and B-Splines" />
<meta name="DC.Identifier" content="10.1534/g3.119.400346" />
<meta name="DC.Date" content="2019-10-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright Â© 2019 Momen et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Recent advancements in phenomics coupled with increased output from sequencing technologies can create the platform needed to rapidly increase abiotic stress tolerance of crops, which increasingly face productivity challenges due to climate change. In particular, high-throughput phenotyping (HTP) enables researchers to generate large-scale data with temporal resolution. Recently, a random regression model (RRM) was used to model a longitudinal rice projected shoot area (PSA) dataset in an optimal growth environment. However, the utility of RRM is still unknown for phenotypic trajectories obtained from stress environments. Here, we sought to apply RRM to forecast the rice PSA in control and water-limited conditions under various longitudinal cross-validation scenarios. To this end, genomic Legendre polynomials and B-spline basis functions were used to capture PSA trajectories. Prediction accuracy declined slightly for the water-limited plants compared to control plants. Overall, RRM delivered reasonable prediction performance and yielded better prediction than the baseline multi-trait model. The difference between the results obtained using Legendre polynomials and that using B-splines was small; however, the former yielded a higher prediction accuracy. Prediction accuracy for forecasting the last five time points was highest when the entire trajectory from earlier growth stages was used to train the basis functions. Our results suggested that it was possible to decrease phenotyping frequency by only phenotyping every other day in order to reduce costs while minimizing the loss of prediction accuracy. This is the first study showing that RRM could be used to model changes in growth over time under abiotic stress conditions." />
<meta name="DC.Contributor" content="Mehdi Momen" />
<meta name="DC.Contributor" content="Malachy T. Campbell" />
<meta name="DC.Contributor" content="Harkamal Walia" />
<meta name="DC.Contributor" content="Gota Morota" />
<meta name="article:published_time" content="2019-10-01" />
<meta name="article:section" content="Genomic Prediction" />
<meta name="citation_title" content="Predicting Longitudinal Traits Derived from High-Throughput Phenomics in Contrasting Environments Using Genomic Legendre Polynomials and B-Splines" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Recent advancements in phenomics coupled with increased output from sequencing technologies can create the platform needed to rapidly increase abiotic stress tolerance of crops, which increasingly face productivity challenges due to climate change. In particular, high-throughput phenotyping (HTP) enables researchers to generate large-scale data with temporal resolution. Recently, a random regression model (RRM) was used to model a longitudinal rice projected shoot area (PSA) dataset in an optimal growth environment. However, the utility of RRM is still unknown for phenotypic trajectories obtained from stress environments. Here, we sought to apply RRM to forecast the rice PSA in control and water-limited conditions under various longitudinal cross-validation scenarios. To this end, genomic Legendre polynomials and B-spline basis functions were used to capture PSA trajectories. Prediction accuracy declined slightly for the water-limited plants compared to control plants. Overall, RRM delivered reasonable prediction performance and yielded better prediction than the baseline multi-trait model. The difference between the results obtained using Legendre polynomials and that using B-splines was small; however, the former yielded a higher prediction accuracy. Prediction accuracy for forecasting the last five time points was highest when the entire trajectory from earlier growth stages was used to train the basis functions. Our results suggested that it was possible to decrease phenotyping frequency by only phenotyping every other day in order to reduce costs while minimizing the loss of prediction accuracy. This is the first study showing that RRM could be used to model changes in growth over time under abiotic stress conditions.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2019/10/01" />
<meta name="citation_mjid" content="ggg;9/10/3369" />
<meta name="citation_id" content="9/10/3369" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/9/10/3369" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/9/10/3369.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/9/10/3369.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/9/10/3369.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.119.400346" />
<meta name="citation_pmid" content="31427454" />
<meta name="citation_volume" content="9" />
<meta name="citation_issue" content="10" />
<meta name="citation_num_pages" content="12" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Genomic Prediction" />
<meta name="citation_firstpage" content="3369" />
<meta name="citation_lastpage" content="3380" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Mehdi Momen" />
<meta name="citation_author_institution" content="Department of Animal and Poultry Sciences, Virginia Polytechnic Institute and State University, Blacksburg, VA, 24061 and" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-2562-2741" />
<meta name="citation_author" content="Malachy T. Campbell" />
<meta name="citation_author_institution" content="Department of Animal and Poultry Sciences, Virginia Polytechnic Institute and State University, Blacksburg, VA, 24061 and" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-8257-3595" />
<meta name="citation_author" content="Harkamal Walia" />
<meta name="citation_author_institution" content="Department of Agronomy and Horticulture, University of Nebraska-Lincoln, Lincoln, NE, 68583" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-9712-5824" />
<meta name="citation_author" content="Gota Morota" />
<meta name="citation_author_institution" content="Department of Animal and Poultry Sciences, Virginia Polytechnic Institute and State University, Blacksburg, VA, 24061 and" />
<meta name="citation_author_email" content="morota@vt.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-3567-6911" />
<meta name="citation_reference" content="citation_title=Selected Papers of Hirotugu Akaike;citation_author=H. Akaike;citation_title=A new look at the statistical model identification;citation_year=1974;citation_doi=10.1007/978-1-4612-1694-0_16" />
<meta name="citation_reference" content="citation_journal_title=New Phytol.;citation_author=P. A. Arnold;citation_author=L. E. Kruuk;citation_author=A. B. Nicotra;citation_title=How to analyse plant phenotypic plasticity in response to a changing climate.;citation_pages=1235-1241;citation_volume=222;citation_year=2019;citation_doi=10.1111/nph.15656" />
<meta name="citation_reference" content="citation_journal_abbrev=J Anim Breed Genet;citation_author=F. Baldi;citation_author=M. Alencar;citation_author=L. G. Albuquerque;citation_title=Random regression analyses using B-splines functions to model growth from birth to adult age in Canchim cattle.;citation_pages=433-441;citation_volume=127;citation_year=2010;citation_issue=6;citation_pmid=21077967;citation_doi=10.1111/j.1439-0388.2010.00873.x" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=S. R. Browning;citation_author=B. L. Browning;citation_title=Rapid and accurate haplotype phasing and missing-data inference for whole-genome association studies by use of localized haplotype clustering.;citation_pages=1084-1097;citation_volume=81;citation_year=2007;citation_issue=5;citation_pmid=17924348;citation_doi=10.1086/521987" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiol.;citation_author=M. T. Campbell;citation_author=A. C. Knecht;citation_author=B. Berger;citation_author=C. J. Brien;citation_author=D. Wang;citation_title=Integrating image-based phenomics and association analysis to dissect the genetic architecture of temporal salinity responses in rice.;citation_volume=168;citation_year=2015;citation_pmid=26111541;citation_doi=10.1104/pp.15.00450" />
<meta name="citation_reference" content="citation_journal_title=PLANT DIRECT;citation_author=M. T. Campbell;citation_author=H. Walia;citation_author=G. Morota;citation_pages=00080;citation_volume=2;citation_year=2018;citation_doi=10.1002/pld3.80" />
<meta name="citation_reference" content="citation_title=LEVERAG BREED VALU OBTAIN FROM RANDOM REGRESS MODEL FOR GENET INFER OF LONGITUDIN TRAIT PLANT GENOM;citation_author=M. Campbell;citation_author=M. Momen;citation_author=H. Walia;citation_author=G. Morota;citation_title=LEVERAGING BREEDING VALUES OBTAINED FROM RANDOM REGRESSION MODELS FOR GENETIC INFERENCE OF LONGITUDINAL TRAITS. PLANT GENOME;citation_volume=12;citation_year=2019;citation_doi=10.3835/plantgenome2018.10.0075" />
<meta name="citation_reference" content="citation_journal_title=J. Agric. Biol. Environ. Stat.;citation_author=B. R. Cullis;citation_author=A. B. Smith;citation_author=N. E. Coombes;citation_title=On the design of early generation variety trials with correlated data.;citation_pages=381-393;citation_volume=11;citation_year=2006;citation_doi=10.1198/108571106X154443" />
<meta name="citation_reference" content="citation_title=A Practical Guide to Splines;citation_volume=Vol. 27;citation_year=2001" />
<meta name="citation_reference" content="citation_journal_title=Trends in plant science;citation_journal_abbrev=Trends Plant Sci;citation_author=R. T. Furbank;citation_author=M. Tester;citation_title=Phenomics--technologies to relieve the phenotyping bottleneck.;citation_pages=635-644;citation_volume=16;citation_year=2011;citation_issue=12;citation_pmid=22074787;citation_doi=10.1016/j.tplants.2011.09.005" />
<meta name="citation_reference" content="citation_journal_title=Comput. Electron. Agric.;citation_author=Y. Ge;citation_author=G. Bai;citation_author=V. Stoerger;citation_author=J. C. Schnable;citation_title=Temporal dynamics of maize plant growth, water use, and leaf water content using automated high throughput rgb and hyperspectral imaging.;citation_pages=625-632;citation_volume=127;citation_year=2016;citation_doi=10.1016/j.compag.2016.07.028" />
<meta name="citation_reference" content="citation_journal_abbrev=Plant Methods;citation_author=M. R. Golzarian;citation_author=R. A. Frick;citation_author=K. Rajendran;citation_author=B. Berger;citation_author=S. Roy;citation_title=Accurate inference of shoot biomass from high-throughput images of cereal plants.;citation_pages=2-2;citation_volume=7;citation_year=2011;citation_pmid=21284859;citation_doi=10.1186/1746-4811-7-2" />
<meta name="citation_reference" content="citation_journal_title=Journal of Animal Science;citation_journal_abbrev=J ANIM SCI;citation_author=C. Henderson;citation_author=R. Quaas;citation_title=Multiple Trait Evaluation Using Relatives' Records;citation_pages=1188-1197;citation_volume=43;citation_year=1976;citation_issue=6;citation_doi=10.2527/jas1976.4361188x" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=J. T. Howard;citation_author=S. Jiao;citation_author=F. Tiezzi;citation_author=Y. Huang;citation_author=K. A. Gray;citation_title=Genome-wide association study on legendre random regression coefficients for the growth and feed intake trajectory on Duroc Boars.;citation_pages=59;citation_volume=16;citation_year=2015;citation_doi=10.1186/s12863-015-0218-8" />
<meta name="citation_reference" content="citation_journal_title=Journal of Animal Science;citation_journal_abbrev=J ANIM SCI;citation_author=H. Iwaisaki;citation_author=S. Tsuruta;citation_author=I. Misztal;citation_author=J. Bertrand;citation_title=Genetic parameters estimated with multitrait and linear spline-random regression models using Gelbvieh early growth data;citation_pages=757-763;citation_volume=83;citation_year=2005;citation_issue=4;citation_pmid=15753329;citation_doi=10.2527/2005.834757x" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=J. Jamrozik;citation_author=L. Schaeffer;citation_title=Estimates of Genetic Parameters for a Test Day Model with Random Regressions for Yield Traits of First Lactation Holsteins;citation_pages=762-770;citation_volume=80;citation_year=1997;citation_issue=4;citation_pmid=9149971;citation_doi=10.3168/jds.S0022-0302(97)75996-4" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. Kirkpatrick;citation_author=D. Lofsvold;citation_author=M. Bulmer;citation_title=Analysis of the inheritance, selection and evolution of growth trajectories.;citation_pages=979-993;citation_volume=124;citation_year=1990;citation_pmid=2323560" />
<meta name="citation_reference" content="citation_journal_title=J. Exp. Bot.;citation_author=A. C. Knecht;citation_author=M. T. Campbell;citation_author=A. Caprez;citation_author=D. R. Swanson;citation_author=H. Walia;citation_title=Image Harvest: an open-source platform for high-throughput plant image processing and analysis.;citation_volume=67;citation_year=2016;citation_pmid=27141917;citation_doi=10.1093/jxb/erw176" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. Marchal;citation_author=C. D. Schlichting;citation_author=R. Gobin;citation_author=P. Balandier;citation_author=F. Millier;citation_title=Deciphering hybrid larch reaction norms using random regression. G3: Genes, Genomes;citation_pages=21-32;citation_volume=9;citation_year=2019" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=K. Meyer;citation_title=Estimating covariance functions for longitudinal data using a random regression model.;citation_pages=221;citation_volume=30;citation_year=1998;citation_doi=10.1051/gse:19980302" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=K. Meyer;citation_title=Random regression analyses using B-splines to model growth of Australian Angus cattle.;citation_pages=473-500;citation_volume=37;citation_year=2005;citation_issue=5;citation_pmid=16093011;citation_doi=10.1051/gse:2005012" />
<meta name="citation_reference" content="citation_journal_abbrev=J Zhejiang Univ Sci B;citation_author=K. Meyer;citation_title=WOMBAT: a tool for mixed model analyses in quantitative genetics by restricted maximum likelihood (REML).;citation_pages=815-821;citation_volume=8;citation_year=2007;citation_issue=11;citation_pmid=17973343;citation_doi=10.1631/jzus.2007.B0815" />
<meta name="citation_reference" content="citation_journal_title=Livest. Prod. Sci.;citation_author=K. Meyer;citation_author=W. B. Hill;citation_title=Estimation of genetic and phenotypic covariance functions for longitudinal or repeated records by restricted maximum likelihood.;citation_pages=185-200;citation_volume=47;citation_year=1997;citation_doi=10.1016/S0301-6226(96)01414-5" />
<meta name="citation_reference" content="citation_journal_title=Philosophical Transactions of the Royal Society B: Biological Sciences;citation_journal_abbrev=Phil Trans R Soc B;citation_author=K. Meyer;citation_author=M. Kirkpatrick;citation_title=Up hill, down dale: quantitative genetics of curvaceous traits;citation_pages=1443-1455;citation_volume=360;citation_year=2005;citation_issue=1459;citation_pmid=16048787;citation_doi=10.1098/rstb.2005.1681" />
<meta name="citation_reference" content="citation_journal_abbrev=J Anim Breed Genet;citation_author=I. Misztal;citation_title=Properties of random regression models using linear splines.;citation_pages=74-80;citation_volume=123;citation_year=2006;citation_issue=2;citation_pmid=16533360;citation_doi=10.1111/j.1439-0388.2006.00582.x" />
<meta name="citation_reference" content="citation_pages=743-744;citation_volume=volume 33;citation_year=2002" />
<meta name="citation_reference" content="citation_title=Linear models for the prediction of animal breeding values;citation_author=R. A. Mrode;citation_year=2014;citation_doi=10.1079/9781780643915.0000" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=S. Purcell;citation_author=B. Neale;citation_author=K. Todd-Brown;citation_author=L. Thomas;citation_author=M. A. Ferreira;citation_title=PLINK: a tool set for whole-genome association and population-based linkage analyses.;citation_pages=559-575;citation_volume=81;citation_year=2007;citation_issue=3;citation_pmid=17701901;citation_doi=10.1086/519795" />
<meta name="citation_reference" content="citation_year=2016" />
<meta name="citation_reference" content="citation_journal_abbrev=J Anim Breed Genet;citation_author=L. Schaeffer;citation_author=J. Jamrozik;citation_title=Random regression models: a longitudinal perspective.;citation_pages=145-146;citation_volume=125;citation_year=2008;citation_issue=3;citation_pmid=18479264;citation_doi=10.1111/j.1439-0388.2008.00748.x" />
<meta name="citation_reference" content="citation_journal_title=Livest. Prod. Sci.;citation_author=L. R. Schaeffer;citation_title=Application of random regression models in animal breeding.;citation_pages=35-45;citation_volume=86;citation_year=2004;citation_doi=10.1016/S0301-6226(03)00151-9" />
<meta name="citation_reference" content="citation_journal_title=Ann. Stat.;citation_author=G. Schwarz;citation_title=Estimating the dimension of a model.;citation_pages=461-464;citation_volume=6;citation_year=1978;citation_doi=10.1214/aos/1176344136" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=J. Sun;citation_author=J. E. Rutkoski;citation_author=J. A. Poland;citation_author=J. Crossa;citation_author=J.-L. Jannink;citation_title=Multitrait, random regression, or simple repeatability model in high-throughput phenotyping data improve genomic prediction for wheat grain yield.;citation_pages=1;citation_volume=10;citation_year=2017;citation_doi=10.3835/plantgenome2016.11.0111" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=M. Tester;citation_author=P. Langridge;citation_title=Breeding Technologies to Increase Crop Production in a Changing World;citation_pages=818-822;citation_volume=327;citation_year=2010;citation_issue=5967;citation_pmid=20150489;citation_doi=10.1126/science.1183700" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=J. H. Van der Werf;citation_author=M. E. Goddard;citation_author=K. Meyer;citation_title=The Use of Covariance Functions and Random Regressions for Genetic Evaluation of Milk Production Based on Test Day Records;citation_pages=3300-3308;citation_volume=81;citation_year=1998;citation_issue=12;citation_pmid=9891276;citation_doi=10.3168/jds.S0022-0302(98)75895-3" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=P. M. VanRaden;citation_title=Efficient Methods to Compute Genomic Predictions;citation_pages=4414-4423;citation_volume=91;citation_year=2008;citation_issue=11;citation_pmid=18946147;citation_doi=10.3168/jds.2007-0980" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=I. White;citation_author=R. Thompson;citation_author=S. Brotherstone;citation_title=Genetic and Environmental Smoothing of Lactation Curves with Cubic Splines;citation_pages=632-638;citation_volume=82;citation_year=1999;citation_issue=3;citation_pmid=10194684;citation_doi=10.3168/jds.S0022-0302(99)75277-X" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Commun;citation_author=K. Zhao;citation_author=C.-W. Tung;citation_author=G. C. Eizenga;citation_author=M. H. Wright;citation_author=M. L. Ali;citation_title=Genome-wide association mapping reveals a rich genetic architecture of complex traits in Oryza sativa.;citation_pages=467-467;citation_volume=2;citation_year=2011;citation_pmid=21915109;citation_doi=10.1038/ncomms1467" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Predicting Longitudinal Traits Derived from High-Throughput Phenomics in Contrasting Environments Using Genomic Legendre Polynomials and B-Splines" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10.cover-source.jpg" />
<meta name="twitter:description" content="Recent advancements in phenomics coupled with increased output from sequencing technologies can create the platform needed to rapidly increase abiotic stress tolerance of crops, which increasingly face productivity challenges due to climate change. In particular, high-throughput phenotyping (HTP) enables researchers to generate large-scale data with temporal resolution. Recently, a random regression model (RRM) was used to model a longitudinal rice projected shoot area (PSA) dataset in an optimal growth environment. However, the utility of RRM is still unknown for phenotypic trajectories obtained from stress environments. Here, we sought to apply RRM to forecast the rice PSA in control and water-limited conditions under various longitudinal cross-validation scenarios. To this end, genomic Legendre polynomials and B-spline basis functions were used to capture PSA trajectories. Prediction accuracy declined slightly for the water-limited plants compared to control plants. Overall, RRM delivered reasonable prediction performance and yielded better prediction than the baseline multi-trait model. The difference between the results obtained using Legendre polynomials and that using B-splines was small; however, the former yielded a higher prediction accuracy. Prediction accuracy for forecasting the last five time points was highest when the entire trajectory from earlier growth stages was used to train the basis functions. Our results suggested that it was possible to decrease phenotyping frequency by only phenotyping every other day in order to reduce costs while minimizing the loss of prediction accuracy. This is the first study showing that RRM could be used to model changes in growth over time under abiotic stress conditions." />
<meta name="og-title" property="og:title" content="Predicting Longitudinal Traits Derived from High-Throughput Phenomics in Contrasting Environments Using Genomic Legendre Polynomials and B-Splines" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/9/10/3369" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Recent advancements in phenomics coupled with increased output from sequencing technologies can create the platform needed to rapidly increase abiotic stress tolerance of crops, which increasingly face productivity challenges due to climate change. In particular, high-throughput phenotyping (HTP) enables researchers to generate large-scale data with temporal resolution. Recently, a random regression model (RRM) was used to model a longitudinal rice projected shoot area (PSA) dataset in an optimal growth environment. However, the utility of RRM is still unknown for phenotypic trajectories obtained from stress environments. Here, we sought to apply RRM to forecast the rice PSA in control and water-limited conditions under various longitudinal cross-validation scenarios. To this end, genomic Legendre polynomials and B-spline basis functions were used to capture PSA trajectories. Prediction accuracy declined slightly for the water-limited plants compared to control plants. Overall, RRM delivered reasonable prediction performance and yielded better prediction than the baseline multi-trait model. The difference between the results obtained using Legendre polynomials and that using B-splines was small; however, the former yielded a higher prediction accuracy. Prediction accuracy for forecasting the last five time points was highest when the entire trajectory from earlier growth stages was used to train the basis functions. Our results suggested that it was possible to decrease phenotyping frequency by only phenotyping every other day in order to reduce costs while minimizing the loss of prediction accuracy. This is the first study showing that RRM could be used to model changes in growth over time under abiotic stress conditions." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10.cover-source.jpg" />
    <title>Predicting Longitudinal Traits Derived from High-Throughput Phenomics in Contrasting Environments Using Genomic Legendre Polynomials and B-Splines | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings, {"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"0L5XlbEuf-JvBuK1u2_zhRQSVFhvV7eVdGcjakK7uiQ","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"494296","apath":"\/ggg\/9\/10\/3369.atom","pisa":"ggg;9\/10\/3369","ac":{"\/ggg\/9\/10\/3369.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/9\/10\/3369.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;9\/10\/3369"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;9\/10\/3369"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/9\/10\/3369":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022\\\\\\\/nAFCT8SjD1PXN6T3T8gr\\\\\\\/A6HCBg2RE0OuAiY32HQi1o4Vp1e5wiU7A4ypwZjZpqq0P9adzMfWu5yQgisPalT5yaWN2gTAgcthimq1utLnRGVwQDDUIRpQqINsUtyBzUC8FsjI9CjzVa\\\\\\\/a19rv5SzFDF5NW0X772r9vdeOY8\\\\\\\/GXqfYVtcvP6xSAMAnBaKFqQyPiIJZOYLiou8G8+Op6yCHs3GbKxBTZdK6AQbCCBfpySsSNZUh8WlULIFOvQKwqNY8pfsriWLUSPu5BLB7+s+63wrTIA1eU5qSl5QHCo4gjML5xjsJL9uPLja06xX1kE0Uh930jEbjRKGNhqzb1qyoXI6eBUMa10OwHjfUHsiERs8+N4xZ25DB2GnDznSeV79cI1bzPkeBpXnEjI4d+LtvJnqjaJ5nUpwAPWC6DBuLMhIbo+KiFgLeKvhWv5Tw8W36RhO0zGDlwTI2bkfmkgayaCYWcnzrAjHiIIy8VKexqw+OEtaInJbxDTLdlDQlAeCxVpKe9C89Wn+GagqhG\\\\\\\/e5wj9RZKII5RkAGjOaKokscc+z1cVGW+6a1X2hhkXccWxZB0ejZbdTT5XO\\\\\\\/TGpd74MVRI7z0e7DUn1RDQ6QqCympIHKMjXftPa5o8a12HjCHVENcLSw8+kizfZwuBADMsrkjt\\\\\\\/eRPuZbujiCJLJAH3BEmZ8Fsz1Yt8Z1\\\\\\\/qNbrOSOlWiae5ifhMdeZvhKZM2tH3mzOlmVzGHuy0Ctmc6EdsDb\\\\\\\/QLUksJdMVq2636bnYTcw31UpczSHRuOEHeeeRwSzsuLC3unzA4aTrlXQMZ\\\\\\\/7xdQD7PupASNWYu5BzhcaldHMixGYOVWGWyNeH1hwOludt5+92VRbjVVUNMrJFgRpnJrMXtAQwdT8+e+aGpQqoK\\\\\\\/xKKre5HRXE01sbK4wP90jWLpITXVlrT6IgS9L+yxGiIdcgCBKmKwyBqDVMj80NNyCxrGIrF1+SwGIkyuQZTMHSPmh50TrvTCO8zdTZu4NgEsQZgqrQezf\\\\\\\/w9yIqZjyASELB6vxEF52kBducfTXfRf1DlR7iNs3ufC0Lf2EZFCaZrPixGfwcvqsPRczjTkdHxjSKah8VFY82cxd42eR4VFA76\\\\\\\/MdmIBwEkepsmCELad67iDIsBNMWoGg6PDdt3ssUl+P4oT1+29qTHfZZuQ2zCwMG9YRIk136DK+FOxPESgzCZbY3BmLBvqOeTs0Ff6FJ9OsBPv3cV0YBSF+Yp1peDWVsY8XraAu0JdEch9mKB\\\\\\\/APyghy1NPaQWIbA+jFu72f\\\u0022,\\\u0022iv\\\u0022:\\\u0022dC9hcyHawJESucMqlnX\\\\\\\/QA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002262a17c41b5d4101c3a315582f580d29c9ddc8bccb3d622c3fc4293483693ba7c\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022oKD7oLzPpj+FEzZT4atztz9DGmOHK8dyFh7Ab\\\\\\\/UBJmqsZvJ6kCmZuUu9sddHojh7yvZauOTW10JpKG+qgAg2QLUPa\\\\\\\/zcPsVlmOHz9Ir\\\\\\\/IPYsRfFkqNLQYmLchEnsn\\\\\\\/mgr6Ulz2EgNOG8C3b5dPgsWRBJ2EvvO7VsLcK8MottsX4jdilHhbWle93dK5ch0aHTJO6hFaSXRgGMzvksOdyj\\\\\\\/U8FDSvorTFDqmLdayFt3CP+lCwhU4p7\\\\\\\/DNqyGNOXN7xPmsZyufH8YF3Wk93W695T3dVfzeeyA+Nqo09Ux8dwhfQJVaF0R85Nd+kwPONjFIev47txMGzl+\\\\\\\/rUb8DD3qZ1MHqTy9uTCCbDkQ5YoHyQ2L9Du87nnKAGlOgldtvnFB99HT1nq\\\\\\\/SpnoNpyWgdOIPlRnGMxFouE2PVWrjkLbHZwpu3LPxSMViK8pcdpraXOzN9uYrCn4txuzk6YNe1tZI79GNAihl\\\\\\\/eEA7VZ\\\\\\\/O5Z0PG5EO9oa3x0lIQcJDcNi4YtWcCGkC7s0cYBXu172m39CpTxPSOQNR+WBLoY6nFZd0KbiR2WUBrld0PJgvdAiMdKsj6qzCx6gwcgtu9K8Mz94QpltL8853J4D0wVhD4eiyM47dK2A4RNRf000i1BVAys21nOU0E+PHHWbo4ArtB7bpgXMAL29jJaYdFtYDGFqD4ZKEPG1Xu0HWVhhu6RN0LzQObxvi3MnmbYCr01D1Ab4MXmlI98PVZFJJVzK6wdEEj2WlPPQuCMCVqimgmqd\\\\\\\/V5cr6aDedexbqMbUYOQSGnKYzuRomj5jUaXmNSBO4oTGwMVVR4pN4E2U5wEQrCKp2eRJIMSbdptZYHsA384DolwHILge8t\\\\\\\/Legs\\\\\\\/XMY1g9p8jEfnlOb2aapqFVAMKnlvcbD0SpU3LrCb1SKeERdi+v\\\\\\\/T9ISSLfJ2RWo5nlCb4QLi1hPaF13RPkRjtioTgiPtq8K4F76P1vP5+b2pcm2rHDGl6OBpbhWx1TSEmhr4QssJWxgivKtUmRnZoYdnWZ\\\\\\\/Ud6DB9lOTxivfIRxyxKDe1RyPXaS1BJWAE\\\\\\\/znxvSuHCEwfshemDQ7aae+Oya4JOOk9fUYgwnWs4JSxN6upUA6ptt9gnHV9ph8QMOiEY2ylQkcyMM4VFPleSiMQgjXMKLyZ+Oi7Ykc2ZetITI\\\\\\\/dYcyUyGeOIsLk+taD3+lEfLc0yvfyGKlg0G7B+IAJ46cOqxE5fRmGeeAHTJ85aEgK04Y6zAzavdF5NudevfGNG9Ejes1v9KhKLt5IXi+NMDRnvZKXqOpXuuNTMYQiCoXN6ywVh+0LrKbdrOSXP917mXsE0K2unItPKFaZbWd+2Lpx3FD5bXFwJIrIPDccUIPVfNtnOB1STszpGmwHw+C1+4WMMfQlkW09vr8z+m9aV+v1jnmgZ\\\\\\\/rLpf61DvlKe74djIOnBcRTGEU1dQYko6c9wTwSJjragn7oo5jIq4E6Pie9NGi7+voPuV\\\\\\\/Ju0rWZ4hrQmW9raqNRA7KsBXFWe\\\\\\\/l5pevysRfd7BLrpBXAl4MWJS0E1Xoos6ujmsR1qc4R4sUfomWzWarjDfX+WQO1dlXV1Hb33W3TGHMA\\\\\\\/LJnAryOn8lgvYIpRSa9A\\\\\\\/pfoBNUDNG4z3khhP3wuQkLTMfn9eL+KmgtrPF6XOzmdIGXH3JsPdBaRH6dcUz0oBz4r4\\\\\\\/PfG+1SGpUW0Aa9GKnXp7sqtsPO0KS1Op9GnxuGg0da10QIsGE\\\\\\\/EQog+5+xRJyNHNTx2fCmFyA\\\\\\\/XApCTF7o2Kp59xcosL0b3CBGtxCB7fOQtsxlBxvX3FSDawc38PcPjGlVYgnsZsaKbmKuhWiNlGhpFT71KFCdwG\\\\\\\/AvD6l1BcOe\\\\\\\/x87TSKz8QFwQvS9JBSJ+fFnlnUrATc58nyum132rseAdRyLy3M9iWyeC6NmZOkESvTKJ3lYbeyOaQdw8Alz1RZcpNusCr6E0kH7yzK2KEKWSj9cMvFb8KZ8RGX2uLcOSflE7Jv2JC\\\\\\\/\\\\\\\/lrUrtaY1cqnyuhbAdDbtUlupxGACLfKMgynQRlKMD7DOQNLGfc8goL30PN5xGc+bCC+ojC1+Tr6JCBGHHUVnhudiNwHnUM1YYzHh1InPKqJEEOqhcqNKf\\\\\\\/8hUDNubvaSdSld341bZDc4YHcB5Zv2bN9129yX\\\\\\\/4+kobQi8\\\\\\\/Ds+uMetawDdo+Ffs1UyWI+4l9yskns\\\\\\\/cOUj5uxQMXcA3xfkEimOaj+Z23RuIKuAJDPse8jR4YAlaYASV2YJPfHDR\\\\\\\/Y1xehDh\\\\\\\/VHdW8XgN4jUgmAU8pRGSKHt1nKZGOofDSd4Y+SlWyaGojPy82+AvzmJgKb2jG24HMhhJiVarumlwZv8SYTtDai3FbW1tgXz8zk3gRR3o49Cw4TILNqROZl4AUUvm0CydLx9iJLHUbdDU0OL5CPZ0a44gtWb5p\\\\\\\/t+CgCT+EUke66x+zlFgA3XBnwuJ2Jpj\\\\\\\/eLbAO9kQpgdisAH+ompzV625yCb96ir0IXA8WnvGG5QYiCx8C8sNQFncEJ93Tu9T2FrRE8XGpL3XU\\\\\\\/ktHDAEIrGGYB\\\\\\\/gtFKJz16dnsNje8BE8bWnwdkZoKa+2OEIQmdqstbLOhf\\\\\\\/CPjZN3Rx3ECDJ5d7pmDOMYmTWhJsmwipR2+jxXyZFTUDk1NWV+JpfNor6mBjrNMdwH3W5mafZQw8cHo82VsO4dzqDDt\\\\\\\/HR6+GYwrezq873zJYxXgfaCmuSHgfbttWqFgrP1bZUm\\\\\\\/oJbaxHXmJyVHYWwclAZPgrq2q5Vah8HuJ4qlJmXpFKuYcunylo0dxq4InafLEEDOBm9fhorM4iYqR7XyvyGXQ8IZ9E\\\\\\\/VUaXEcIWk4xza\\\\\\\/bqL1xuEVUPfv1oU9rW6hF4l0ULFJGUbINhSs5nUcxmV7tXFmen+p1ZqqbFX0eH\\\\\\\/SFAxEyM+8B\\\\\\\/n7DgOhjW8OOhzTI8YaY3NA\\\\\\\/lGXaBiswcl8CYkbDwRd12DD8XMQ4gLg2AZbdA5TYulTjCIkEoHoyoq91HWvCpbiMunQrc737m71ebw70T1HZjzOLlY26bnrw+mLThdI3FrKGZy6llkZRKbs4IO8TdkqEgqw8DrfJCVPGZ\\\\\\\/mMgDVGcDKyoPGgsnIdEHy8yysuIzHzsf+0XGejVEVp+W8JKfU1lPKcrCeoROKQKeETRGZt67sxJvlL95FOirE5tTO3ONHyxdPQlvnPICoYMoUsAL2sId2w+JH7bbgdoYnCLeaMb8VabvREwhFym1zjy83rjKDJOAvC8UO0euqmjUcfkogRaagBMtOQFGMA4a3x0PsaMBEbMe+Igm\\\\\\\/ePWPnvWekdBog6LQ+BNoSmG1oQ99Brw\\\\\\\/6VgU+LgAo6j9L173qnNIFPN5qNMf+xPQfDJ2V3eTK+sWFNG9r1TrCN1wrsVIp2wELOWw4P3wazc4w02unlngKrorjYkV0h5M+VT0k52xfaMk2qBE\\\\\\\/b3FGifj3EBfZQTKHVsgIrIwBfN9T8nbRj5fIFCPb+XiN\\\\\\\/kE8OKJ6tbyDPPDGbZRdizQaLAwvEfmXinl\\\\\\\/qgouivD\\\\\\\/+YkDqdgvcwyetNc36q9lQfhvFraUu53HOQaT\\\\\\\/BM8bQsiWVN8NKsInXSTpeW7goD0oMsQUmfIlpaE2\\\\\\\/4c91MsPrn9y61etzjQtT2PbVSYo4fcjJCSUtthAjAt79XUkpBg9bjo4apJA5Mq05XB2Kf577X+PBsRMeQbq2CtirJ8860y9xvhfVADU509tTITqArYZp7YTji6SBxnMAF88LxNtvH1MUjJdqtrd\\\\\\\/vNuHk4uiqtzz46S4IUxd+LxD+pRPZlhBtTG2qa4LuZO8JSdc\\\\\\\/aDU1aU3UcTjJs1cD84Yww\\\\\\\/ciRwFHAVPcCwD6I6SU9Jis7CmrHS27ua6dXXB9SAvd5U0sdgyB7OnKgJn1RkI9Z8NPX7LfQr\\\\\\\/AXeUeWOLAEd7b6S8w+Tn9RZHX9ep2qB+SVOVFfj1Uiu4oya9aLucdslZjAewjErdhK9xJmU8MIJF7jnox3HZHoR7rgUXoiWNGz0QkldQiPy9rg+65Z6IVKYhSEgTLSUo8\\\\\\\/fyZa4ww43DyomskpHFoEAyG741tSqvvw8j+hZzIchbbPdyoqQxrMYSU0TnJ4YtcoHGbZdFVp\\\\\\\/WU\\\\\\\/aW7AFQAluk0X4byp2naDKK+pW3BmW6Okw3zm86nyNwzWn2onU96FIr4so4sBS8fJk1NU4xx2K4a5gI1vmlb5VOExP+rzSZdXQui7zVgNM5YP2OkB\\\\\\\/6+lxHOckgqz7SJGuMFXovj5\\\\\\\/feHmeo4UXKoOPpzeBSsgtBrRwB7xm1McHEEXXD0pzDpsk0sm+z1e+w8JkhczO58ii65csEx2Tn8hduyxkxMKhjyVcWsjVod6DEv9VCOcpGyLhnvK35N\\\\\\\/MTfez7sIKQaWZrjzGY6ropDXMkxc\\\\\\\/DrwPjLA8gPNwnI85\\\\\\\/OWTPpKuzfs\\\\\\\/rr2Oiki1uKoXct\\\\\\\/dmGNFzzDCSQD51lKqpSmh672rbb6P\\\\\\\/gU9Y6W1bJ\\\\\\\/0E+hbzL+a2mfgOhqsrgLVxiBHNBeEle9CRV92QRZvm+IpVmHu\\\\\\\/hi6ZTvjLoXz8lc+DsseK2Se\\\\\\\/m41wXcXeGh1AnFJnwWGCpot8j7nsUH2RdgKDkwdWSK\\\\\\\/jEWLau2UsqzTJ\\\\\\\/EqH3GY1NGPrO2Pmw4u1SHhtNiBk+WEdgrT7\\\\\\\/cuZrVDws4c91FCJwJ\\\\\\\/21QxgVy11IwYyQVkW8s+hhN+XL7P6MBoa9bzy6mBazjYbEKWJxa9oQF7H\\\\\\\/EAeekhrqfKoBwCCwf2KISNCsOjjCDWnKxQijgRwPsVKgaV0sns68KtGVjgzWzJY8lKEqKRrrqw4XQ1Kbt6exs4Z2iOKLkaMxdI5SGlQZdgluBNF3Fur4ZYZU6T7nO5E4DgIqvKSRHwoEWJIHYbgshtJFfopdfD3hP7t7WoW0RfLFEAGIMy\\\\\\\/9eYm4OYE70p6mZA7UZaPgQNCs5lomjtZNjahNZDBrZz3XiB9lfCvsO9rNYDH4G6FsOF37Fm4lMcHVUZyTJuPnw3LOzfvenFVRZN2wuQoURmPNptRh51XW25Jj7jnRvIWnGxH1VBMCR3BFe2bLKOhyUbG2gwqeFuiT0GwUxf0I6ZcOBQb9na\\\\\\\/BMW499zry6Ppn0vKgD1EY62tAAdhnxYYHb0xpfkB0ju1StMAHgfn+pStg51jtQkniS4dwXuYqCvts3YqPF11ANkC+Kay4kULuwfcmnQmiBbdNCGvKDpz7DBTLUtEnURG8jk6pNfzBg7Ax+4dpnQuJCo\\\\\\\/Mhz7tdG4We11zuot6Rg7iOzM9SEcsvKIp1EVbY+VSrbhtYikDtwh0mrsXOdiZbd9zNOH\\\\\\\/PGNV4XyWC8UqyF92gh\\\\\\\/BpspvCOcjLkCg\\\\\\\/KgL9hhb1euvqOJZ2RDPqsuPKVU2mnT467+9ylie1sDbcLaxYWesfQY+0KRM1WOLN3D5oEC3YjmG988swEHTSMOSBisKsIWZqPSxTT2AkqnhDYL1Iijui7XRYyAFqZ6WbDqcykqQWwO3BAkg\\\\\\\/7bnKXsXOAos4YinbTejWhRtlQ7Vwic1x\\\\\\\/Lsw33yeebVqw2EKGQxAaVvcNUB0Pdy0Flx46G593AN8vMN9L8AtCYhoDRF11Hg0TwUMh+0KUe0kQWU+k8+v3TsA+5Su+OcE7X7Vh8oEshHUSO6PId0cINKw5yJhBm8Ap5h8Vy\\\\\\\/dQwOHUqWKHdeanPT8REY39VYoTgc8pfFceWtsJRYOX92NnWedaeXhOLyAVkh2jbuEwrKlRPIB8pOsuCRU6DvPfaAfoQr66JCGdXMl3NTi5efNF\\\\\\\/A5pwf4Dsq\\\\\\\/cYcr5R2TEX4yJLwmdYQHBjxfXzwWKug+lc\\\\\\\/BgDqqWFJvDbwS2Z0dhxMnP2CI88n7khqBJKrD0UBArop6mjsqtSp5EkviH8FCXGVBAbGPeD2SDvsdxAV+nw42R4bbrVLPypwu6ILYFbI2tu+mOnOk6oJySq9E0d7kT\\\\\\\/s\\\\\\\/S753GVJIvPwJ0yn3ax5DwTGjQ4DGmGtfPq0HUFkTjleX1yPMHai0BbBv1PGWWCz\\\\\\\/qHgy\\\\\\\/Xn7LkJynRg\\\\\\\/GB6q609dTh2orll93gzAphJvkWACRdEzF5eFMxpLcphB61KN\\\\\\\/8tzfHzAcM+Axz+PuUXqgSxoVyW67ck3vstqiq10DNQPUGPu29pm4zCoMt+vzfhytG8clm6umPuPzQoSC6EiahvVJKFgw5qGlJL7HXT8\\\\\\\/tfq03IcFJtozEbgvZbUnospa4gGnYc1Q0l9JHoJefH+Dt\\\\\\\/YOL5Jkk8jRnl7Nn5wGFLLQ\\\\\\\/zrs1M15B\\\\\\\/MYD4HVgdaF0AlIDhsZxVGNUsjfljsb\\\\\\\/MjxgibXO7PZqrZiaeYOaX1n5OYsZhm3D7tMvYaW0R+uHk3Ep+31HLdO+oDbqv1fXAGiZXFxST2WOi8JfB12f\\\\\\\/MhMF4NxR+WQCmutQYNI8M30kwGtgDmHmom+uYtG0lu3BbJBjXSTvUWtPo52NNp+kAmKYkLMQYtRDboaRzNKXx6rQ9G3756+0\\\\\\\/1X+1NAXzW24yGgUbqZyBdf\\\\\\\/SVXMj0FtAq6GbKP+LhcCdeZmv\\\\\\\/fPRTuhzmO6602TujEfkVyqxyT7Z1V261hVi0IsmsOPhwngpbsmfMlW528eIUMLJcQooJVslcTvXROf9VfJQudPA5eP\\\\\\\/58bVmcPwcKiZBcrJSkphV3\\\\\\\/RdIEUQGLD1ediP9J5N3mRWKZBZ2wVZxQ\\\\\\\/U9p5ssDu\\\\\\\/XGenkrFhvN8XIF8rKeBOqzHT7IMYLsGQ477BN\\\\\\\/MpOlSHha72RJCEl9mo2F0jeN6DPdLDwN5aukc6H0s3EuyH66I++8X0WnLOtO\\\\\\\/fYKFKMsI3FB4imOYHUBXMibHmZE+2uEvB1LFYCE3KacdPkzLsCL5oe7H2WaWqUCfRFAeHBqvDMqF4KcvjB\\\\\\\/8F81NbC\\\\\\\/QKEzAKe7dyX4\\\\\\\/o6Min3Yi3VlJmDIhL5wYTQ2LfmP7OyRp4anbBG9vdFzXN9JaCjOY5uGtV0Vusz8MlTENeM1vc1F3HwilCF7xMybrmYUAd6dO5YB9Q1DWczNCe5YbyxQRFnbMZg5P4IXilEtv07eDkpeKaC6x3FyScvHInHGax1PDdUovdyzYhTYD8SmG\\\\\\\/pzyu2MjV2yZOje\\\\\\\/97jlvCTyQa7IiRpUrLvd+0CSY9AuW729so7ZxSxtDw8lrxh\\\\\\\/ElhBrnUq59\\\\\\\/LXBx\\\\\\\/cvoSOPm1n1S7z5iFEFSPGzNCHDayiEWHaRczG4GWbwll3Uo7LfWS7dKUSOIyZKqCQ25xiOS2m4l1UbDhuZnKAVnmf+u30euksZCcHw5Evk2yGBWFZI0TnLk7TXYdsEPrhULiLPKmseelhXOC\\\\\\\/aSv7tuGZmN1pCuuV06pMu66BJhvAa7CjeyuOOWJ+Y+yjJs9nyw\\\\\\\/\\\\\\\/ZzZk1wvWq1Nr3rg9qcM+C+ne3qBXsqk\\\\\\\/uCN2aqoQxaQkoicgBbbVyrLcgCORpeGUb7Nf+g3prjh9icHbdmp85681ZYeWhgeufmw43dTScztddj7GHTakLmOAsYs1WnKT\\\\\\\/vAwRX0h76pwqwPJfFUczUG\\\\\\\/wfpNZjBSnfG2lqYANOJomxBsm599vZUHgGL7EgBoNQbkKsp969ufRnJkU32nmtTRIk51hEb2uEbSCyT9rROc0uOh0bjss2Giwk3PpA1hdsE1z6KkH5hMQ7bvHmE+x1mPEbjgJdy7qwZOBQqsM1iv5QzV6BdTOOLe3iPdV1UKL40wTtypOi9cO3ULpo5X3bQ8sgNmNA7r3\\\\\\\/j+zHVP\\\\\\\/p3adYy\\\\\\\/JdTwqoDwa+AcvWMr3pRBVPkbsMQKiDRE+rCsdh\\\\\\\/qnI+dfgfwxZ8gg1OlrFTlx\\\\\\\/x4oxtm0Vdt3UCBvXwIP+m2LaNGiybLy1TyDYsNteAHGUplCS7V60uQk0zK\\\\\\\/IC0daK4MAK8JCcq4WTXnjRiyQtFfL6yFy0LXzaYiSg5pqW1IXpRrPJWJs5L4VkT\\\\\\\/YHdh3HYaBu7ipTS1qn7qNHi2y43LuOAcbduNLcMqFuKcPGU\\\\\\\/xA9OOzYmexw96BtE3BYTnapu04ibR1g5V6S3TtQbHAgOug74JAaOgefiABjBqCaoa+qBEVEUumBjbGLutjd9cTvjEtaHzClmzYF9mp7PyPlpDMpEbpytp6r36\\\\\\\/1xHI03art6QX649wH3H\\\\\\\/FutG26OJA30\\\\\\\/19SD6yLwEckSXIwjC3KYZ0cn3+c8W5S3MeMh+deCYxLPzYsnoFvx+UDQunouWsckQCCTh5D2lzBv1zKBSPvqrWYL+w9M2pynFt9SGiBzHnNKvMLpgPeBS0bTEjADGDW942txY1qKtzOKVm4lz\\\\\\\/RsGvMhGBv1dPipd9WGNpylQqMYC329KtFX15ZIRpFSuQLPmviffM\\\\\\\/ECOPhqKmYvGr2paHHJJ97H8FwohULCNJpv7ax+jIVjV8c42MY\\\\\\\/TcfgQ+Ev8joB5oQTPCOsWYcftrIyFM+MHBjXhi6sQKiRKD9Nl689K8GjEe5KD+kwRiqKhJ3kaT9QYWzWsD3Hfe+uysblOV5wKk8u+N2wyoG65axgp92LsQi4f0sLdVcO4E2xydKZE3AXA94a\\\\\\\/LN06zEMATOqJ3GATuReoCb\\\\\\\/VutdnebjevtIivLqlL4Y4P6r92pIzD3sXIsJYrcF1Zzhtpj0lIQ9kvo83TnQcGtFx6Fojjjr7uyjzioTurK1XzJW+OKAuKU83tHd33LoM5BqlyRSOWq6z5yTu9PScgFKgrPf0+M\\\\\\\/7j7HccjoPu+TNZNHHsB1XAA27F4B1pF0iOpnUthbsYRGwp19JDQGclhQDfMRgDej2yGfap2yd82L3FPzO0ML9tm2cS73qw4zZo87WTu5HZtVb01+v4DK18CkACx+dhEn00Zl4Re5hjgaXP21gShX5\\\\\\\/sENDvpvAvpSGVhP71w7cZSWQ7l9l8UYZkm0JSmbU\\\\\\\/SlXYZlFnVcoQjMAPgSNY1rAxCS8W0dOMMhUtrZZoh7PccwoQODJzBLs21qZXWFMox6F\\\\\\\/HwCp1UWYGxYE+\\\\\\\/eQ44RaIOGHC4evj8Fg+TXwYCoUDkh2yil9CGEYmkZFMErRVPUSy9+PVJbfwXKtCs62fZR079ygnHhEPqZITDFIVJtmaybN+ybIjCztRD6gCAuiaW1w5CZ3pWevcEIsYwrJv0FQ0nXSaeAIWTNXjvmQ9QXYIJ7x1JUQz+WXg8vhFxcTgzyyMFooOP8JnYJwJghR6z95me94kQ1uDc77UjrPq41mVGkAJBrO+33KvpPCrm6zODwGy4PJdT5Z++tyqwmS36PCHMs40vQF9WPRnx8pLENLynl4zA8cqq5pOSWdixjTQ3DIOyi4nE07NfdB+\\\\\\\/MoM+KqQIA3\\\\\\\/St7NkwoZ34e2Dl3h8Lvv9gMIknOP\\\\\\\/YUq5XbbeB5uFNF48IDOp+c0qDHk7bi0l+tpx0fsMdr4L02ybJHoPRxCa708VZEEJ6SRrA58wGMy\\\\\\\/Sm+D8MXSqJ5\\\\\\\/8l2KjL7uSlGAfpLWsxRWKk8oGawVk3wfBhymGbi2rfL52L2n7NWTBajitQe+IlcsXLybnE6zALmHcpvhx9k61Fow1c5LbAfxzK7BoJa8hynpNYnP01kpiTyUODOkxANi1leYWjTCJIF3535p8mob8pe3jK3fZ54GNZBMqIwAzufmuoOHgT0ewz+k\\\\\\\/fZ7I7mygxLLNibBsB6LqYqchiO61jlTKxBTkXeaB3BtM2hSewQjwpWKFmZTrpZ3ka+GQuw3VnhIHWUNYAU3RMz0lQDi\\\\\\\/5Jh+C6cnvUVglXrapqqviRQIh9+7oywcu3VVoeVCSLpgmrtZymFZKcKc3x6EzTxnCHlMILWp2AAma8\\\\\\\/kR9H0wkQzFR+6ainaa2uw600nipxpIzMNiRcUSZgta50+7r\\\\\\\/MDOgL6om12gA5eqST7zJ2ukXHFN0qV0BIC5Jh\\\\\\\/ZYepyUbW7rm6PofaDLc9b6+ekdd\\\\\\\/hUIyHlBXvBu8ii0yJCboGe3ahC8GbSmg4u28K7A+2lgT4xx5T33Nf7Mu39g2XG5Ua9S69LkW\\\\\\\/pcf9Txs8a2hwV1qSizvWjmo25NeFEBzUI8LVqL5gadnZT\\\\\\\/B90cHAyxmom\\\\\\\/pgDzybowyKwVOZFP1KfRsxGuKFss4LByxrINjL8NuOiVehTepBNTz2WFEje36IoUr4rl9A5EQ1Th7kA3YXhTuxUKa6E6LOF1QdWL9NQNo7Co1UmJdWmN85eJarHuGm1SUxggQt7m6A5UmkM2lPSfTJrLPvpnNZdBK6XALX9OyaY1DTltsteBBUmHyXM0HtUw6oMmGHiGVV8aJ8BD+uMBDBFfUe8E9irPNr2zDqSoA2Q3CXS3yvEohQMjt9aMQaQnPo\\\\\\\/gXaccbJjPo9BmPUiV+EsbVjIymnrVL\\\\\\\/KPBQaABbOwe2N8eW9okvfxLYRU+LBYAGZxZ4t\\\\\\\/72PGq+bnO0ah1qX2YV4hw3yIq9a0AM8XIenoSkqEla+vPOq2zcH2eH+wKHZrIi00HBtIBn\\\\\\\/\\\\\\\/Ev7jE6Izdvt+24RwEjJ61IURmaA975sog+aNpBbvYqOmWTfDwfwasrwvd5E6GS\\\\\\\/R1r6umAlef4jmtEUCBkJAlg95lmGg00b51VtQmCXCsJZcCuGeR0W4mPM1UGWuezvelIGvPZA78gq+Fx7U0fgGE0HnVriFMvmuXuNHEVcV6mjNoIXo2hOaDdZrkMy1M3QjcxkGCf8UZ6eweb+Bt24+MKxlrYRV\\\\\\\/P7HlFKC2PwDjipmF2Y+Z+y7tQH1NJCNv4fipFr3PHelD14DbB8+4dp2L6Q5l2yPYCnhsJJ9A5zHOH9NjENc8mC1haFbeJk20ueaSC9Fe\\\\\\\/k3lviEqT7nKhltqsm95HbBmbdE0u+oGr59mss1GN8abVatA7P17eZlBk8Og5Z8M2hoRSK7ALsB3Ri2bZRudVnBc2F9EGgRxDB5HCzlTiEfn3Q20JEGg6soM4rYReKSMqZs+AaDGoHKp2OvlBktsjlzKUFv3PJE6xLJyEVjZfbbojq4xzZ9PAhAZlBLGjMJhdmwUBxOWtkb15kSbb09cGFJ1JjlIzzER7YMZ0amFe+1qHcX56L9N2zmdi9HhbKfCIK8A1hTCmVgBqv61Bimh9jY8kjY0S4s3LpmHiKLVmWU2q7clo60ngS9\\\\\\\/UAyw8OLc7Ecq3cZnlZH8K334m4PpgTaPN3Wj2d3BfA8R4XrJXknWMbxl6We8WOksZZXlc+tHc6+bLWQXM+AZL\\\\\\\/KDoFZfCixsNAjGz4S+Ozatjk14RepQz9k289DL2SNG4XQlLfgJhS0zSrRNwllygJfYuNNcDFlJBxLDf1Ob2\\\\\\\/xlfJH78nYBJNJ5kjsCq7ijAsN3Ga2YHKiUx5g9TbNhaigbw+PnbGu2tD9MpcYVy+XdWk\\\\\\\/fS9qc1BQRVDedm7ccmQHgscyWgdjZDvl81Bnl0DHa4gf9kssn4r9\\\\\\\/kRui3K2lM6zPbUP\\\\\\\/SeH+93wkUSwfmhX4lgpvSjcnaqk4w\\\\\\\/1rBxumPrTPdXO0vPBK4jh7+L2EFa6eKSAIDAcx5MXP0OKordeRPXjwc6LVq01PtzKnss9Ts+RfzfzG4F6z+GbFGfO3G\\\\\\\/6Wm9rLkTuKJRjhPBfj\\\\\\\/tF70O3VaBz+SLSxufNkwq+U\\\\\\\/Lll7p9Sop8zIS6V5fd9Pbk31b50w1oJQF4C8tczywTy9tk+STM+9Ro3\\\\\\\/EVM4W3YoWdQCV7ol2D+NklOnampULnorLEtGXCKKUaZ3TOJbVKcUbbMBIMGHBPoVC2O5qxGmQu+pYNjDW\\\\\\\/XKSWMiOk86Bp4uDdAGVKBJktL\\\\\\\/8nshkuOYIvgk8nY4ftt523nHzXJJPJa+SPITh1ronDS3h4PhIJH3f1PKO2AVlLC7W\\\\\\\/PMMdcBA+0GaNpH5HTm0JvPDKdGtclsCuIEJjqG4MjX8OGfIsi31Kt37MBwJd44XKy5OWzbH1HrC1tFrZglIheq8s0ed2LWd4JTFntJDLfWp8vBhArn3efludY95LYhu+kYw+BTcpzo9kHhfGovoM5YyecQ9jnpPfFL3sATa92LynVAd5R\\\\\\\/7UNpA\\\\\\\/NrGvRexNvu5r2jyw\\\\\\\/wBOLWzrpGoAby9WXyaV1BYkJTb1UtiIrLXQKCs2SXs+x8aKFoH9AANIiR0kOlsGu7ytp3kViTvlzZNr9cdhL26WZmo1hlNfRnGfVD1Ow4NjN\\\\\\\/Zl34oOxzQvoRK2nB6qWm4Hn68RnnjsKbsXCzTF5HGwBCaizIgyCWOX5Xi1Cw9BRcOglAbmErWcLrEe53DbdKf8nso3ZS98HMTEEZjtWab2o4L8FvEPMcC2ZzsAR3D56xSA\\\\\\\/MYrdHX4xSWDQ0YZ3mjL4kOkKwdFi+PoXSdG3StIjfq1gXju1gY7x7Vfc6YIluDqHr+Pf4n+vpmwY0D9zVyOJb9Ue1cU82DvKT6fwM6uva0ctiiZjremeYkTs2PCoPQujBKLZd3AG7rhxVH7A6A43LKXqjELWzU35huSUPD8YbuxlgQBbRJsXZAUgUhdg4iXI0iLaXKHrhAS\\\\\\\/uAHkpRb0zwM48rt+i9ZEhSC70lfAX7rnexhCUNzheiOGi0hgP1KNFUrbFFXFjSaaf0qjSIxv9ZN1FRizTWy3aoOeqS9e\\\\\\\/4OeQ6Vvqcb14607RFcd8ouIvoo1CJhCOZISswjKoioEVkeLaS\\\\\\\/tAnTks4xRw+cqqHdFtP9AnjJuhIxsIEXRuX7Qfvz33D5XjtZrrvETXJTx3PJfH8O41Q8\\\\\\\/WdG8SXcdpf9RM+X+NBiCsvfPjbFMmgd2ZSfaXRQ4XYakk3HOyxCyc7FZxVSufqs6NkPvKHDxktyDHZgzothEAGin30ARFlZdgmVSfku1uvjNL5kRTQ99I6bsLoCIvqq5dzRFunN3fDkJVyCoVCbHMRGQ3XYoHw5epkH6+wtVG0qDUAlyxH8sx+MJmizmZw+t43PEzov+tVM7wyQ5vUPi8xwzzZuMrScTeIJRzywb20wSFsYb6CwJd8b5S\\\\\\\/tV+X7OL6TXKkt4uiYxlmxsxnHbo83HGw0IwFd2F\\\\\\\/LXwyKUkUIP3hPs12hBgrvG2EDJydyRlXPn10S5uNk3pEdvjoG5Q29st4dpgvLtmYGpbG1x9TQL\\\\\\\/L37\\\\\\\/x7rrE7RSbP9TGuYOhfuVzgMXJwHejhs0IzMZx4lArO5MRjlYQ58qVbwvDElXxj0nfCUXNuL7O1uyXx9SvNPIoGGx3ywa9lc5VbOyPvuWGxbInJJxnmy1LdICuHRqfZl3mkdpXo6O7NYBCsTxKBQmUQnisnnK066NRs2BaRzgbF6ebvy88mSf5DQOfB\\\\\\\/FUL2aRHRNZFW43nR17s4xrzVUbOlgQXsLu8g3TfWJFKgbWIP2D40bUZrS+WBfEtEdBumCAM66w29BW42I+ygKbp7PhmucSUFucGJ1+r+uFS694I9SKXmmD38HKnj0itPP3V0EValwT78ujP0\\\\\\\/hWuTkap4sLyXSMegAR0nNuoPIx+oPbl1BTjPZGIgbPJ18axWkQmqxbXOY\\\\\\\/dfmO5u6FHFtkViAd70QypDD0+P7emuxnxRfyI+xVuqdm8sMk02uCIzrX9eOT66fQZ3YwgvIsdxm0DL+JK953nXn4gk1zDa\\\\\\\/vpjEipWUz0k7UAlDQpWJYiXQHiJLwqlKJ5AWx4zuiH2LE0KckEhUGCvU0Ir6P9v0wOFFe8PP86mOoKCTz6XzjSD4zIaZgJo8R1IoEfMUCeDEQvTELAd6XnKORiuaKxbsacIr6JmUQWhp0N5Q7xDIGWDad4jiRHIWeD6sf6lp4sqw10eomuSQcxdAltOU324sEZHQlnZ\\\\\\\/Ptd6GVWY4L4qKGAbnMNLj8+1OuO0hexZcSGPWtifqnmM5PCtW\\\\\\\/K0vympX7Iw3k12ZPwvRjPwS4EKSQ4XfA0F1J\\\\\\\/W9AOAb+LOXp2WAtK4ILsajaTfg2++xBqInH\\\\\\\/zsC1PdHweeAW2uC1\\\\\\\/RlDQG8QwuZeQh2l56rtO8UwO+\\\\\\\/eLMdPe\\\\\\\/QLEPkL95ZmSMlR\\\\\\\/crLcMjdUKaY\\\\\\\/u4KnT+nkyOHVJT5M\\\\\\\/5pOjcGAiI3k42K2KHV05v4GMvW2VI8ZQKgTQNIfC8lmNYLxHU0Z14gXrahEUMvLAdy6oIF6RlGLN4oAzKkULa4Pd5E6275Lqg29WKe4\\\\\\\/\\\\\\\/2NXqhylMt3q4G0TqpgkX\\\\\\\/LoUToFPzf\\\\\\\/PLIFPHYKYjLyWHTfqpzv+En\\\\\\\/6Xlu3uiRxcWIftUy0wma07\\\\\\\/sXThjRoI+HY6dbhyGNK0l+I3F6WxZTwzTKYutNAwGXMGxR6mb69zZ6p\\\\\\\/k\\\\\\\/HydSerRqVbKxWu5Rig9qJTe\\\\\\\/+O1\\\\\\\/ZlzJpwXVfV4TyxwajMUpYMpLnKNbLreYvt37e5YZ8iWdKYjv338U1s7JRfJ0zlZjDX7Vx7y3wKQiBzQAPnoEO1MhUHyliIDlWvPsRLbbUO0RBul54TpcYsUpPd21QQUB8RZh7Y\\\\\\\/jG3MaldDIzKp04CbmizPsQUaLofWDdo3gjITj\\\\\\\/\\\\\\\/qVfYvk5AC1KfmpPiC5ol0k7XrF3dl7+LmT9ndOSd9eVdJuNzv5tXqMgKSmolWiuZ7pUcwOt+wJ2qmYlIFj2cctkjd6NlW6sVua\\\\\\\/l6bpQuhYtcKHvAkr+ynVYu7OnbilmiJaeK0bYZo1LHBH5npNvMMwRdNACd05UifpMKzsbxTU1E9X2vX5jGbGqKg+s07n32R+szAuBX1ijw0sgyS4Aws0keCCtWE4gSvF61l+Jy4+7s8xyRe10LkLu942ZgOHAXf5Lmo\\\\\\\/NX4U4Pw3CiAvXAWgCBhGFT2wz2NKoIL5\\\\\\\/5RNP8DcWluqSef9tOMqVMly9F1a0rLLsm4KWPE5PKcs8C\\\\\\\/lIk0AE1We9fkzoDchzQPtwzhS4qgrvJlWC3Zi5ymMQBgs1jNvCYoE3ImeteaXwbIOpPZfmqqSkfo4hbQJiDWhhktAmLCM79Pzs0ULRjioeA09wg2n56i9WzscFZo0Sfp227ovzhFnURIFYczG6HCragRXopZsDth01E9WeOtsyNcxzfHXqwY9BGJ+eVZmgfV\\\\\\\/vEVC7tQWAQMsQvc3pyknlm1cx6LpVDBluVw76MqFD0FzEmmJrK\\\\\\\/Ey4kycbyRSWPWTXt4A70Ob2JoZZKZu7P3OZsp3BMlu4xyDFPWt4FaPAno2EjkXOZfNyxLGEP89\\\\\\\/EgSZz3IfuBWsePt0VKd0pho1yoY7vHwT3VAs6fL4T9KBs\\\\\\\/M2vVHPu8uJX9dweHEOJiiQmV2En1tT4AeB1BjadXYXRDiToMnZGZnPS2jOPZzeA9xRNTpKp6pG+BTg6ntLhU4ANNdrZ19W8B+WvLayCqLR4B0xPs54pqVjnTdEbjYuTWc6K00aX5j2jDex9lsUWgbLJh3Xp41u+p91L+3bkm44jaNKLlOQ78vqPhC1HB3XvhcKNuRvIs270BmJIg8AwhwHVzlo\\\\\\\/R8Qh7UXpvQARoMIZqAfAwboLiWkOCzJBPomzmy8OVUt\\\\\\\/HJW2+A\\\\\\\/5tHq+FGC8MJV\\\\\\\/UqJxWEwmmeXR8+O1\\\\\\\/VsAyDzjae1g++jCgEjOEz8UVgsnA03WUY0mwkHtIOzbAfsZ8QcHcfchM7MTJiAQf\\\\\\\/EXB6B\\\\\\\/IfQvzyr2QtD9vgvDBrlg4vTT\\\\\\\/nN+USuv3wOy315Xh3lH1080aeD8YR7ztl4pPZMAt6Cany873u\\\\\\\/7W0FwlBMhZipMGlHqlMcfSVvEFqUWA\\\\\\\/dJ3ziutxY8jDSGB5IvZG8nriX7S9ZYAG12hMu1JfAEY0FvfcDvL1Flo0SOLlIula1c8tvuVpStQyKadv2PRVPNuiDP53ORhTnaNBe38+hFon8z7CXVAH\\\\\\\/kR8mYNkwiAN5K2evFHrSJCKipp61ssoi3JmSGsG+bgJPuFiTENfiF6lPfBBGBj\\\\\\\/DSE3RsvsesMdiTUrxDqdkWJbCkhPnpryEGMflDMjQvNTiF+fHbY43mpgblmTpU+YhjkEvI4ZiLoOSKbkcTxBtBcEPsOOSgRGRE3EHQy7oM\\\\\\\/IjqjMkgWmZt9+62scLo+D9zWDNrCeRVZJNS4ATRF1pwCSTtHt3KxfQ5ZXAg77fwljNH8madGoC1Ya4lsTRylpeyCHFb4xb7RI1RH\\\\\\\/ugLCTyyDwbXfaevuDwFDKKaatJmdE1sPw7rpRu9tahHuMF9O8zylFB6CW8o1VjZQMovWjUHSDwBUMISTUkv6jbCtEfU4Wrs0IYgCQbSy1ur5sG7kETa0JJhdbxLXYUd6Rkp+zENTHj3uOya\\\\\\\/SZGm75T93F4FxJkGMiAxkjiKNflN6R6g\\\\\\\/gntGys8vtkl8vYdqYCaCPAln01ON0oJzVPl3d15iKUJw4yuGS3VynWLd19g\\\\\\\/wSt6dnNuESABUMdU+28fdZX7h61GOU8niSf771LbufEhPCAjrC\\\\\\\/ciNb7jH+CN\\\\\\\/4OYQXxYhaZVFDKB1dt6zCYL1i4dTn4EaBObFpqG79+Sa1n54QKsSwmdYY2HRQKPHpPkrLUBpdCJL39SCiMozYBzoM97UAo28NUYTxtoqmGvJnhWByTMtPc1u2Fd2Fcu\\\\\\\/mbLyniCeG8SZZIzHRz79jSt4fQBBaiFVUBFeF7IqEt\\\\\\\/\\\\\\\/zC76g0uk6XHR88TtapfZo\\\\\\\/RS+qUUb3pulVIp6RNhrEpZjqpdcgKnsou8wn+rxYQ7CdrhAgXMsz9g6rm+iUDBDtSEpZKBsN3F+ibU24zrFqux+7SWD2jHC\\\\\\\/M+pnyNgqEDJgzYolPXBp2T2c977Zhqv5p5vJm2EPyNmG3Behtl91jlXeauar9h5IK3fS\\\\\\\/POmpV0bSCDWgDHEmsbQWUSAPv+58XUzPUOsE1J2vRqBWQUGMhnDHYkakczW10bGz5JFGT7wQ+fuauN9INcu1zsCfNfG2aHOfE9jUVKI6Yw6n+s\\\\\\\/5s+yvUt8zlr3lPq9KU3vg56i\\\\\\\/0JRLJYZ8gbACBOGUeimmSZLzJl3qM+Pz\\\\\\\/4XJk16EAUFVBYBRbhc9CFdJFbjZbFcscKoo\\\\\\\/T2I3h2aLT4DK4LICGXruQPv51a+zpK\\\\\\\/96w7W3+8Cwo8LKg2HUT+EvL30p9irzJplrAGximgICc1F8Q2S89+PSUHJcviciy39cNuirO4pTgzz9xRJx4o7P4meMKnjqqVE+GEDLYlDeECXwqyPSxqTpSu8o8y5smhzOpkHUqeo74UX\\\\\\\/BF\\\\\\\/brJvY7tcLZ4wW9jEPr\\\\\\\/0yB7eArw8pldHBubABMF2RVu9LWa+2Pr9T8VDxqKJ3kd0q7pknhBS60gBEjpzUyfJ0FaAlviJyDtir1VUh3WIy5c5xYmPtyqflDUcrKgDUMgJIZPnpF6o4jPL+7Pr0lb0Qbjp47RBRTze3Kg6\\\\\\\/F4OJz36SlT5HbR0NJVapaDs8QsUGtcmOQK6M6sKP2Qz1EYpQIx1HHsoqbdKXPW4chMTUIdkVG5qy5\\\\\\\/COa8lyQG4DG2GwBPKHmfX4D1XmHU7DMTWCkrJIj3UffegcahcaXP\\\\\\\/fIwYxvSDkmnB949ipF+NKhayzeznkSffBUBOEWs+F2wN7pY4TmQRwCly4grW5MztPAqvJNVZHqGFJ2dfqNSBNMHOzIlx7zOAu\\\\\\\/AfJq15skUqBPn1ZyL8gVQV0AKdxG+On5r7tiFLx1hLK+zgOS5VZWpSTjub+8eWb9fk14qe5kbehvejFLj0nHtYeso+GyNEVFYBr+qUmJAA46ImEAA+WqyJ35CHItW77v1jQPBEeW1OFfLlnaZoHPrpARj1R0GCmmN1pRF0LVn\\\\\\\/AgdWIwq9rx1EzC3gtLMRzQNVqMVW\\\\\\\/bli+NuDYzEkqH6e\\\\\\\/cVaoDmCj3S1FBz7kJM4fiyJ0sV6BPir7m5hb1dEkKwyGy1uvLOeitN6XeReAPvEa\\\\\\\/xTFwRhK8rR3WddNuznOw9iRpWak\\\\\\\/kV5CZlfEhT25rrqDU6cZ7BNxS7JJ5Pc0ZepJRlQl8dBj5y5yI3gUQ+tKs0X3YP9B0fL3dcrbAIcRN0g131kBzc+lIVlJF4UHvywR1LLDJk2sVGtrafGLTBa7JW8XrNbq1AcYVbv0w26RHSckEgaDIZuA\\\\\\\/rgVqdffSvAJpB2+\\\\\\\/hmMw98VCQoFg+TJj3+S5BJL6V5kjTgVBicQdf3+2sqLQ0vKPU8pGXr6lioa3QyRDCh4YmV4\\\\\\\/5QJCdUTdcN2O\\\\\\\/bQPqTZ6\\\\\\\/9fQ0PAEbupIThHk1fExeAX7SJlYTNm5tuMsPsyjdGTWCWULOYnJ4t1XWBk+Y4uBcuZPK8cR0uGHQSWGuNif5DMPWMDvTcwZhPPx6\\\\\\\/J1UkACsmtC5O9NBWgIgw2blsZGHdUGaj3k2qEaJjYNH8pIdMsZ94DITmZla+BvPO8v1eNlnuGEHH\\\\\\\/ik92RkYIkQ4Lmui85NOP+vAclDSJAgzlXtonLB+D1BwT\\\\\\\/yhxWaotDtPnht19\\\\\\\/StWM7uXCtHRHSLvJtz8lza2OlQpaVZgnXuKZsvW2tIkK72q4lQfUu3TInmY7zLDeOiHH\\\\\\\/tWB019O6xe9Pk5qbmI3OnJ9H+tRQc\\\\\\\/9r9M303zLWMIbkOJGmtb\\\\\\\/WX+lDoTGjQ7bCPMhiTtwjh9EL5Pvc4AbZHJ5Hj4ArGfgzv9oF\\\\\\\/4vK4PKlYEKb6YPGHfisRBNCjQScgBK67Z1PvcEZ31gg\\\\\\\/pKGzameOJXQ5Z22ba4kSp6NKetBYk5iPmfR33R3Ibv2eITtafqSWDw1BhZS9nx+LyqejE2VP5woZBXgnjUBrBuXomJOelVlHBzOLf03zandSF2kzcApDjuHwqQX7N94sUE3Rn1ThsCJqVg7A\\\\\\\/FEAUDP7x8jeKjM81vdYRTh6+pv9qVICplIJBz4NrpZ2R3NBwuX4jwtpR+oJqaP8dqVX5V17euk0CNaiEzeqLo2YUYX\\\\\\\/8wMJneXqjPzfdJhtoIfQV82EdRkySGTzS07ahlwnuTerXt1Qki9SlRmvnxGdaj6aUZwVXU8wCZsItj6\\\\\\\/H6rZlM9F3OS0mG66OtAFsw3In0I9dLcCsfH7KiVHHCPvlFd+eXGczf0TInzBK7Mu+a\\\\\\\/wH1onQqilxtRrjLcJmesM3135lHuTUtYaONgmj337cqOCKXg3fC24WnJnYSaewLcQT8+pMPqIkYA3XnTtsxoGn4wPW8inDLpTgtrt8euW4BYx0f1AbyFid44Of0rewJthfKhQrpGV1wLgL60tY8d3nQe5su+7a8xOr+qVZ63up954SXazDPn557nxGcPrieQL3IoG4QdH42z7TABD3z8G7Kfm25opsUOC3glY6J7QwPEljxcVKlROYnWJsfrslBRLDjm+NnJRnBr+fr3RYxP8PZYDXYn9WYh\\\\\\\/YsYUU6ylER3qYGZLx98c9tfXaoWA6UsKLhRoNfmoBHzR0XSDeUwtz1jj3HvKCvUJds1r0JpmYT0ihk8gkSHmew7UGJNTPN2PAYyX4NDPNBLyNmIZBRnDQjPVin8T7bAyvo8OdSdL+ICAAo4d\\\\\\\/77QkF24v3oW0wzCCm4ub6NnYpMg7v2QYiv2T0WRb3gCNyWBu9p\\\\\\\/k2drltRXKPI03juXZ\\\\\\\/8qXyusa+fbmgnviVlu2eMP9Hc\\\\\\\/OGMTl8Nd6vHrLEN1AJGTFA1eEQmW60K0mw1F7UCUi77HF78IuceKuPqCvZPguTBsd3XC+l6P2LYQS2\\\\\\\/E3OtYHglvNT5vYh8UGxRKRNmHDCYpy7aWerY6xRCvnlIDK+i\\\\\\\/ySj+dcUnlhdzGNu1TF7XwL\\\\\\\/fYixnZtxHoaFgaUz83QzcpF95dCxeXQvSlqgqFOpkTm5hOKg65yvEACkuVfoJKVg9EHrC93v8nB1jesxGMlZztbDtUd96RlWDVd9UgxrS31OJgLY3LS73J1kupiR2uFdmiNquVAnCYtyT4pSwe3cqchPew2q\\\\\\\/B\\\\\\\/vzkzU\\\\\\\/mha9YcHfGbW450epagmW1PNiGGHNaDYJ7SYSMLy4SQlpMMnHhudJkVtTkzf+cDcABtGpanEND1bfoTK5TpnuqbybGYe7ethRoSQ7aF2XO1ZipR4d5KNH+ugq12rqJ0ISHJQMa2B\\\\\\\/Hkw15e8eVCAk62xsh9xWJG1tDgOVUehmZ6KG1M7068gAY2Jdgs7Sa82klS0bSnMhnsCYPtXsFtQGkcPIQUBbqOO\\\\\\\/G+Bgvmx9pf4SwbfuGB\\\\\\\/lafdDOMLzWMWeeoVXH6FjmcHuOiMxRHRZcR++kB4aRoimYbcmNB4Bq8mjx21xRk3k6FmNEFE7kl4go0hnvqXFX8X5Ck5iBCYJT1DTHJ2NngU6MPaf7AJO5phSsnxiKMPFliV\\\\\\\/wiWx56BlzSmQLb6i2zfgeZ9rm9jiCrf9FrjbY2rFhds4JQpxYqj2CNskKaUNC6ogZQffqgJtgFrMLGNhVm+0vM84+1P8EBAFSlYVzBdgjmD5rAlg1hqreBmuDFnVtuaqRjvEfwnsXPjqReAwx9eDLyLJ8SGzmmgBrp8hsDN9yU5P2oOfOQIk6LU3X1DtzuQaCZtE1CI01U2qQ6SXYqtf\\\\\\\/xgPOnaNfxJn6FfdUv6Bciv\\\\\\\/WokUH0g1nf1Wp2urgwqs9I7AXfGvjDZoGuri8QaPIk0URMKyCo6QrBqPZzV2MVXh0rdYUWZzYaPMWIYEP96ho0KX\\\\\\\/O8El30Ag3vq\\\\\\\/mEu34uG9KpXYD0YKDjQvUu\\\\\\\/\\\\\\\/FUWKe+gk0CUU+F7aP3lksCFjBUi4R9mUGtdgYY\\\\\\\/I+rkjdrvnNvIkZltKdhTEPu0Y3Kd36dhyxRiocpaPPfinx3826w0ZN13oJQo6iClpNmGHH0fJN\\\\\\\/lX5ZqdwYpdmKwuwOxtsniLTURtX9ufYp7i8QV9ktFLB7im2Ajg5qxEoAlHBSDeOY5r1p+xDt5+Cv48dN28Fcgva35Xc3lt1tACm+fjmRClyeV5X78ojpH0i0sqbt5kvEXCaZhCmLqLDot5R78SBXRuOzFAQHCAJJMh1GUfpubJnqv9B\\\\\\\/yEW+jGOuP9ndQW1EaetwDTS\\\\\\\/JSczrsA8bJR6uuXciCKi8YEfERR12IUJ9F8w5MC2z5h+oupHEymQYpNx7117k42bgyn7YAK49sqg0ZWj2I7uCCeCSSRksFSE2TmSejj78+pzCu569rOfGn98yTk\\\\\\\/bafAy4NIJWoIwwHlGEEqEcPp936xOJ11Olp1zT6ud2xzlIN7mU5DwGR1PiL3YG\\\\\\\/KKFksiaBwa3\\\\\\\/XD3nOO0O+JIfVcAJG\\\\\\\/BAbxuehbp3qb\\\\\\\/1K7HNTJZYe7Thf9ZhfKKmUnjIr2fSDXPQ+8sSfOmx5Bcmsa8aHwzzGTNRUAsYsD1WWvhVt8eiZzW9il9Qmwk+YWy97A\\\\\\\/LOzcFqP4gw0ss3d0lmzulKQYJ2g\\\\\\\/DYIlgt2wLXp9fForRTg7fA1Gdf0onCO3dI\\\\\\\/IC6tgUxbPUacynAiGECQd8PJLKVHDHLv2A0s8q+S95blKDstArjk8btUrH52RKs9FB5iFFfFrzZTODDCS5JjhbQSsd8RuqsoIu\\\\\\\/IlmNLnK5+gwyfn6Xl5nem7p5OpJPExU6L9qLzSO5IMqleUY7thCOCbN1hV2ANiPtjeR8SJzEGvZBehFYIvCr5hLtSRWwwiUYrR8twOZkUSdNRZ\\\\\\\/uxvSGkyUshJ8NkxgWRDKtjuThKyNtrDRgxu6fP595WL5zOGmCIaNCaIIVNhAGvPSBeH5cFTskiC7BcGyBrEptFqbu63UxLX5rG7SJ2ic6pAhMSvNt7AW6e4JHArzCcWzuUS92OS3F5FxSRXpFftBQBapQlIyj8snLrXJnjmjfP3qpt58cTn5NQ\\\\\\\/M0a\\\\\\\/veaKQbHKME\\\\\\\/wiolt13fOQfaL1uQfZtlu6JRWrVbwopLqToAWYEL8uJ28jjjbk10DSnQ+LRET2cSYjqpA4TzxZdyHXpkRyVlAJIcZHBQXydCU1nSO4VZb8pI6vNcrlMf5u7kSMga12uZVzNQScI8rK+vOyfabSE5nP52Y\\\\\\\/2Ks5XHnid2PjQ6xjkJ8Z5\\\\\\\/0+HwG2mylnJSTin6j5rlVjcnzQ8fzCI+JR4O0rqgDRXYB+YF4133wYGkS4TndmNOlKGOYRxBJAbySS8WOTGA0Jf6HAGfZapxrko2mDxnvtLpeixR1OA7n9bhVZgyeOCXFJzmdyUwndVuo7p5DkiDiVncrAVNtr0gkXKwzHSy6VhFTW30UZOgbwZAbumA9hFdw\\\\\\\/Vv1whXO0d4ZLJ6RZnyK59Kf8uDNpTOlZDjCfdqDyredtQfeZgG940EAN55M342lTGo\\\\\\\/AIsVf9hPVtT+wx+JW0QmzcK6krc3jptTlar4IbGvVvlwtfd6PRaKggqX7sQ+nZYeZ\\\\\\\/UfgKx3caP\\\\\\\/cMHWH1xRItH00A0RN6TniCojY9nXPxpvd76d2eh6R4yNVEcv2DXoVCCU3zR2mPUGU3P8el9EWggfhMEssQr1wW9g6CAa8LKU8jzakj8uDLOvkEsNWG1iNoQiCYG1HXF3Ty8gYGP874Eq8OHQrvQU1LAACy2Oj+7ZSgDitx\\\\\\\/jhWRjFEBgqim0UiBc\\\\\\\/+ZUArbtFI8o\\\\\\\/7Mnk97TmlxMw8dGeMMQQeVuD070F+qoWo46Ov\\\\\\\/hOHmJ\\\\\\\/yYP4518EpzuZ\\\\\\\/0nrKtWSyqWvlkWsEFVqoGm+sQByOa+cqDBT6ktB8o9uMF2ZvE+TkQTV\\\\\\\/ytoJXizScVfPAtBW9EpC7vtEdnjsVKVMaihz8toOWJnVQ9jnq8iEZn9DhpMRP3g2mcsIBsiGwczAdHbJmMAlbyGQGW5S+YGSIoV2GajJ00MfC6+18dTVdm1a41moN7DaIF2JxZYDNbww5w3RW98e0gX1drtXpQCWhGwnMSmGiK6dhaDwBBkgxgZF8+bpN+V4Vm82cVnHEuoJzEwmDnD4pEw3ZZ2FGyeINuf375t1w28+eUf0lJNX1T5y+ITKZrnhZswSAG5XDZ1OL2d5wc6B3HkQIQ9w0jvAsmGZp7j59hiBBgp7plnIuW0U+4T0+XVn4LMv02\\\\\\\/HaVc+xkPZH7BqIT30BdzfQBl8GquatH+JzJdlM+gf7h9SepTUICGQcDSQP5kFqCC\\\\\\\/j4H2m3+E8vGcmNNI38ycaUq3QDVHPiLqYaqbqkXaKvloHBJLFR12beOI1VbYi9Tq7dr7mRIkkIfyyViwYzN4tIZLLS7V3T2PwyftAD5lbrNgaj\\\\\\\/hgdpWfbyDi7N\\\\\\\/4rFFvP\\\\\\\/ZXOOY\\\\\\\/hpqnRA7mALchwNVhG1oQrVl+bg0WFgILs3pVjlQPKW\\\\\\\/Hyb9W3gMiNHrJDXCtndL0+s3P94Jgw6XvBcR+y8Dzo15IFuDmLtM3amJQGTSXYWV3gY1ozC57m+8ja3Hidlci8gNocyhkJDt5XIG2yjkhXyDXgLUhF0S8KZx1zYBakisU8mVUHXPXNrPsrINfj4\\\\\\\/+B4yBuHXd9VygssdZrWNptu4aAq7xgaOxf25+ijwmLUsz6dhwwV29jLc4o0IaiuW5IkCoKpmpDMQ1ok02D3HBJ9TIVYDuroKM7JwWkB8ZCXFbxgPf5eK1SA\\\\\\\/AEo5sPtEWkB6HdjMzqDy3GI6VG8uLfTtUjBLV5XVteprvhdp9rhKOxJh5LcRHTxUSeIc42697sjyEgAudEXdye5vqdLcHl9Dm5L0q3vCHlUr7Vza79TcMbj83l3rSCkOd6KNxfELWL9TwMNTrPgwNBH3auJeRu4gSFu14EQdZDoalXFTwmAWC+8petsHr9bLJMqfqZWOsiGI7kkyEHhF6Tm9PpsQXrV6LFy3SyNhiaFAcdxHddp0XIwxABEl7lGJo+ZclmxXhVu\\\\\\\/HcPAeHTfJiNMeVAiDxw+M7q5UGFtL1CSFsCopmHmocdy\\\\\\\/orKOqSyC5nqBew6wusmE2d20AiRdAbSxSNR6j8HxC+aJepv8mQExEqcH4ORxJRdpf0OmNP2RzZezgKQ3cy+C9xhE1s3\\\\\\\/Cim\\\\\\\/7P8EnUnb3ls7vAlBdKx0d+pfxonPRGRrWPB\\\\\\\/ZSNuE+fQ8ArYFXvo6tqIGzogDEbrPWbZyuMvHyoYB\\\\\\\/9s2nZhf25zgZ4FIHzP4wkhcO4g38ZBHOSUrDqLi7LRBWvMxI4HPnJqko5b4IeRz31M31K5Ycz5aVxCcu4pR1XV6d\\\\\\\/vm1wLD5YkE\\\\\\\/p02VBb6TaJ07dpJtFKWqBF6pQiNi3rMCaSXO0fCR+D+Drwch5ecsYkcNFB4fEPT7xSlGmpPuoV5mriUvR6MV27BNenB5KnRCIJOHdDtJtwpvDlbQLZuXjVxZ\\\\\\\/cXr9mWZA8o0gjxH5znwxn5bYXRCrillM6CadZhFcn066ENMSNER8\\\\\\\/OJfmisFBQW1biOt5ur9j9SQe6DFZKi\\\\\\\/phEPjmrfJLBl4HVnEUFXfaERY8VDjtcnt7SGS6Z9yhrvIRgf9P2pD7SR85OMYxWo+dhMeXAHHncdSV6rBB4KPcomdVdg81S4ejEAGdzk2aV69sT0msT1jFhKHl\\\\\\\/atXcHGi4UrsJ\\\\\\\/X+H4RmZf5ZzSKO3i6i8gbi7ReNKMiO29h3cXSWMKvfkLNxxgjt3F\\\\\\\/Pb99GJEg9pp6HOxZh21UI4J1BYnSxPeCHZfOBkEFXqv5vRfIYlKdzC8HGKMSDlp0XdgkgLLBmSJR\\\\\\\/9qJSWgFWG+aZbD6Wsp8GsffhwWDP6RBTLZuTxljsW3TYWuRzN\\\\\\\/3iGbMLqVwPLgVGbMx4ls\\\\\\\/9pu6zBWn\\\\\\\/TcBhadkQ6wTj2ttcaKMfW8S9++Ahw210bYhRzMHdHoTyIsh1UdrdDitoiq0nv4PMwltYJXT7fTD5xUw8QOmfDSF+n\\\\\\\/49nDKMAddKjaYs6MRiOCUQZUJAE6ZcSovm8dOK51Y\\\\\\\/albUU+RIJ5H4Ptiz5lRk050cuxv0diE+c82xNvk0k604aKppSNIuAnQGT2gFxJ2ccEfeyi9B\\\\\\\/37UFPkl3U5U+B1zweaEwv8jf36+roAguKh0G9\\\\\\\/7n5oxiQ6p6xCtgfpz0qtMvCNijd8SxgmzzH8wvi4uxBnuHOx\\\\\\\/o+W\\\\\\\/6oSqZc8u0SNNO+oT7qPfhHGtoIcHdesqJnQTS5RCpW5PKwcTeiGPx4apGWwQwZmQvTjjKqiyu7i+NNqSkSbcOIxl\\\\\\\/0SaoEmrqkjMRTw4rrQjFzVWxzkNkz8pqangF4uABn\\\\\\\/ZjmQZ1wYphtbcetdaGQdLBhLhG2zS6bOxFZAZbwQ1RDqwoa1D\\\\\\\/IMyZe+BX38tEMcBk0b8Q5TqnIw5y3zUyUzMm7+qICK+qZd2jrglGSGVNtQwS5OzGLb7XNglewNPQQ959jfznCItEQpKV4Zk4UBxqyhgKIM0\\\\\\\/jRGXEy\\\\\\\/rSziZEBqSUI4XNOTfX397gBG\\\\\\\/BB\\\\\\\/28\\\\\\\/70\\\\\\\/e6FX8tziw9rncFIlIP+EYnU+S0mhAU8VQvcBfYmqJQmR8Jy33xWrymv2NBbwNTZ36xVKBu57L7Ie0eKxJWwMN9tP3Tng6Pu6G0+KW0pZJZ473SrFoe6r031gobQFJaj1RS76pla+Qn1OPSgiM8v1gcF8rk5jH+R3vQLwIdjfZj2bZskb7MaejSFLuuDNA4Ed6XunTnva9kB0k7MsOP4v9+\\\\\\\/CMfXzgGkt4ZUnxW85dvTbUpSWVB5tLhkwhWvQOaoaOBf\\\\\\\/O0PThL83wuU\\\\\\\/KmwhFe4rQfChxn8\\\\\\\/bmGIEuwcSt\\\\\\\/gIH3tEW5ASXI9iokxGzinJYjRs+BMyk0buyrERoZz0DU\\\\\\\/zm4ElCCPEZg2HEVPmxkpzgWgQY5gjlXI7\\\\\\\/TXyYhaWSMGBI2w31LG3XQrutKkU5O59yHft84rvBwucbY9XAyeYvxAE3edTs4mtf7W6iqn25nwPNROoBluK1KY+XktkmGbWLiJ8YwqyULOilQ+6PQ8UbITCIyRLZv+TnHsxAle1ZqUk6oEKZreFfZhToC+lijsnRx1XQ5QHmnrn8ZKRpwHVtOhCswWZNMr2Ms4BasgRspcgPcsmFkKzURf3bO5B7k60qUGCf7bAhRIaz9ma8OSso\\\\\\\/M\\\\\\\/YDjpPmn+PWpawxq8riezD\\\\\\\/1pvpSXSDMCksIK7Zu4qi7INR1b97I\\\\\\\/\\\\\\\/McmUHA\\\\\\\/Xc7nXkQLNLXwFIBZOPwgFPAint1juSRprSe06l+aQt9a94kbBYLyii84zrlyc1TpBBpx0IjwZ6a8VqU+uAcl0cWCYEq8ssvA7GwPvQ5DYk+GNrf6h4lgaM86bgP3MREbWTha2u5TsFXrIJXRzf5rBz13\\\\\\\/SZqB02OUWUJDS1zjXZ5zo+iFXBQmeUDgxVVsxbKUQ71Q+i8PFvxLAFydkBxXd61PV4y5K8IhQ4kDQhIyairc2Gb3yAVDf7IzIKf4\\\\\\\/DnrJGHtYIUenPey0LZnoc+vzigUHRVaqCHJalPM26M2+zaxIhlXuXmiuGtAVVrPM5Sr2S1BznFdKYtwwUzf34Zlk9YzeKmwMHPYAuE3dR1ScLfEFluaJANxEIV33c5I+aFtz0Jr8X\\\\\\\/\\\\\\\/QM7xC+B6aCMc\\\\\\\/8ycYfRuhBetDhpvczN4nLPqbBw5SZvJyOrVaAxPezIprGSEb3IecCsHl\\\\\\\/ALfYsVJG4Hp6MCBtVwemMsdxp4LjFAaos6xIcpaAJa0z\\\\\\\/IRQFLA7poaKGBC8hYiiJF8cHFrSB5j4x9tfVA3NN9LFPl0QW1ZDO+pg7075zVjdk+oUgBJiC1\\\\\\\/TE7YsKjfrZ2RBLfoA+2KWa4QFkemwCJBfqUz5Biv1t5QDfBCjuQCH0xKu7z\\\\\\\/u9ewGSz0+OZXNBbB+8NAJirgPWOOSIGobqI1nDsNriVGR1B2+NAi9aJLbW7eHGK69XjIkVnf3NUmVrq2jA8d7e0JRtJ75y+Q+Q8UVTSUMsd8UmI8to\\\\\\\/7oeDj7KHdXbE7HLl7ub1KJ7+QFz3HQtJtpER8O6Tb+tjV7wUYrrv\\\\\\\/spT6mWs+Amtrk5wRivcktIj1OJM8rcTHqE3woh6OkJi5EX6ERF2WFNA71f1J77VujPw+s\\\\\\\/z4gutNBRFfPlybaCS2Dkv1CVV51H3SnIhWTdX7lsGIPUvBABOKtGyPi5vsX+RxpMQcBmqdMynRyphufluqWUuUjzPqwkZGk1+QKhDuBJuFhbLnhRnphN1bauwildXNOPajM\\\\\\\/NxcINz9iDm3cHPTiofz+WwaWUEe1AVQutcnWKJL1qtyJIjIPl59tSqMp+qB15k8BUx4V6X\\\\\\\/WWm8EIWPA47AL04PMLPE2aOXQdRLCT4kOPMX6iT6kawOG5sBxbX1CIAVjkbVQF+lZYRpR4tPY24STZLviazmMAFb3OzveaFkFYgTF\\\\\\\/h8wI3Oy16nHOjX+3QkTpAiN\\\\\\\/Tcv1G7lDNTI53XHBmD3xew8y9Ixp21KIiL7i8NxXzgcu17h\\\\\\\/sKwgj5qmJTcigwubdZ5jpynupFWbyht7LcsSUU89CMUoAlQYrQv63uUMYjCEjuvReKvcdO30P8Ipfajmlj\\\\\\\/b9S+g+swKuBWgNevipWCuLw1Shg2WXyHdDcVTLktMpt3vclNfTTp5X7pmPxUSWTSfD40g3fsMqBembracnavf\\\\\\\/hFC3RmD4rg3DGfZvgYVtgpysfcWPD0Oy58dERu6DvEjWeh4iWuHoPHIeXBZhur6gFg63zxB4L0v+cAPAQAcKCwthG9NZodGJWTbbB9ZAmcDzJyyB1e1PhUa+6M7YYp+XBT6oq\\\\\\\/w5ZBJo9yVLpUG1ygQJZuaKuhlDRQPJMS21McxkA3H5MGBVVhNjztJ6d5V\\\\\\\/yeS4Q4JGFVGhmaJLTmsMd47\\\\\\\/Hm8e02DAAikDBKh7k\\\\\\\/H7+67UG23rz6nN8NZZh5uav5v+gkv1OgmajpImCQinxnXaa1cdUtwzMMGFQc9\\\\\\\/EjFVX8u+OyvZcAq7XW62d3WDFj0B8zADtpry8UQMVgq0CbCTX2uS\\\\\\\/D2UKoPh5PZaDXrQ6VQLvjyEiWMAxX2wj9jxz9COAPkrO7sew8Gpv4y4\\\\\\\/\\\\\\\/UeFwU6j2\\\\\\\/mrW1pA7U+4yb4yFYxyurkoIVENhmEt9zLZO\\\\\\\/bFCdkyrza8CRl3YaNCy9T6wgUVVVLjsLLH9KPUwWE59gBJPyL7rnOeAaiy277A\\\\\\\/Iog\\\\\\\/QfFAZTWhn+KFiVLzDcnVYAJWWGMKjAdpazBDriUOIl66IAJJwNVh9J5M3qMzK+1C8J9lZInyFhOUU\\\\\\\/Vr5K7Y5JkwTVr9rev1XrhYtmuAWcmO8Vs1rG\\\\\\\/p8lyiV0GVeelTV4VP5ngj+2pdYL1CLb8sXvm0Zr6UXYlRQChababXRyAjeJd9XU6XG5NP4ZGAOgyAwa53+fnwdRoalTw5ucoJRwttWR4K3ddr5V4ryLAk6EVEokUtNB2cuvLN6Uk7BaJMOmqQ0L+GIhHEESIpKzRYEkj35eXUPIg5\\\\\\\/9DcW3Wg1a6oXyOdMQQ8EXc8U5s+s+dG5eXe+tOgqtx0eYFb6Ti03rT9hXU2X+ZRV\\\\\\\/rw1KI2I3hjAELE0Wss1v9n2zBvsGYITrxb7UDi2Sd7hPCmB2efLI5inATC0uVzICaWrigaSUaRZKsuTTbk0vp4CztgPqulV3zevVKMxTgV47wZCsoU6ga38mS\\\\\\\/PemnnlckjAlFPgidbUUZ47QYsY10iJL6+L95pgLdxlWu6Zq8bEv2VUGVT\\\u0022,\\\u0022iv\\\u0022:\\\u0022ZZwuOqO8C0H171qmgLn06Q==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00227097dad2b0f22f42984a760b54d4d66d824dcecc3ac52540400c0bd268f66a25\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00221LCqXYimf2LbzBeNsu9RF4+l\\\\\\\/yoTFTp1afd4e8dDaVkSinwday1jssmfkHBlfd1HYCG1KNmXpPJt+AVwsOXX9bAEn5Mry8ZvWinWkdmfuG+irOSZuXj5cCVt7RgAgICrLI5cDLJY+ULAFnmveZHV6ISycUhwcdhpKZzBvQXsli\\\\\\\/x1exuhjyxMul8\\\\\\\/3n4AKEsLBrS1\\\\\\\/Kct0Lr2FUyPKBsu8mEf+1ecgysnll9MyHWJLr5OR2kzHYMc2DBCTApuZhVnYVL5GXRq5nUODwI2Jmubd1v4OeQ3trSYdxr8kyGXQWI+aNGid6ErGec6TpPClaApkLwwMKBcpsNbWnWqqEiGkyTwH1PLnJBF1e3zZEgwQnrXyRaBY9NERlJqG5wQgY5ZoHhmq4pmMnfgEZsWvkZ1cfh2Hi87fh5NGlkr20i+LhiwAjQVUnMTi7+5bgPGSlbzS7JHReXTdaHx2x7sdeOIzqNbsA69BWCRAGRHrql88AKF4DYE3DntFHnQyuxk02Mq0Z\\\\\\\/BHmAlXbh8qBzB6E9qdtA0MggJ89NFAKdww3TC5lJFN1SsDNAdtvzxPpiclbOYOY1yLnm+\\\\\\\/+0IsPDuX92rJuFZsdbiGWdsl2BsyQqwM4MfJUwDL+9u5krlMUYGljO3kL4plO9\\\\\\\/wyzk3MF7NjX6cgcD2RKy8wV4xyNtAZqEWzu794cNAXa1ILrox2z+1CAcmizpPRv2Op2g48Zyv2ktDl946N6DDCwORFVwDJrT+sXvNPjj\\\\\\\/hF5v8EpdT\\\\\\\/RH5ANcS2pkS0QVne+9iiozdmeyOnlFK4Qdxpga5ccGot8Ca8jk8dGXTn6\\\\\\\/SvwywXwKZLUsp5cgU7qLTiXABxkyupOoYD4Q5I+kNuf16ZL1VU8mCe0kBwNJ494W7pBj2sFxHCZn1UqGTedg+caHTx\\\\\\\/\\\\\\\/I4vwEVDFr7YjNIUEqx2+FY7WemEp95qYPvNmy\\\\\\\/iOUkl0MZ7XpAT8BDEg3q8GEW4XO13giC7f5TEtV69wCiCb1ofpS+yDlnXDm9yoHav2yC9C0ZrB5e1c9hPuby+f1kqnE93Vk58EixEcYmFQVcSDqXgkiSZHgJ1k7qSryfROpq8xkD69c+Z3VJE1VrJDwpVbZ8wiRKCu146kn0yAWrT0pJlHfjvxAP3ZgMm23kcDRP6JlfgdgRYXXrWqwODlleQahJK9iwfQkPIUe6+hmZA1RBOjT1y9Nmgv9oEeBQw+Kvo8BGnG\\\\\\\/Ar9jTTUYEYOu9ys\\\\\\\/HpKJJvtmPtk4pEFBLJHImjThULY0192RCUDCYyOBPQbDAJKb60ucfdV\\\\\\\/7ArPPh0m5Q9Qxre+Bi9u7wU07afGSR7qOXqKaB6dHclRbwtH5toaMYOCEh7PzBGbWk3\\\\\\\/\\\\\\\/I\\\\\\\/03rX3R8lPHTkEtBkPznvpSa96VeqoJrmWMpv4esQ8\\\\\\\/8XTHhL4ZBMe2TkAoPajsiM1a+eRJwDQ3qRgf9bzdmKT9J+s3VU0kLUWFi32kQH8kX\\\\\\\/N98N9Od9pgFAY1LP+EhRvVQ7MA\\\\\\\/eSCPwYtzkyFn\\\\\\\/1mn1XAReefCcTEefR7tnHWSIWe6c4DXQE3VrMBZYwOZ4uUUj6BT2yAWEdyT0mrV0DNTcgmOHQKcUheheNIX+Shzxkea1xulxNqs9zYC3wFmfETH7rP9j2ROlyRKZoy3eH3+EB4invsBxL70WrsweG+cor0nvmG1LFvUYTIBvtqZS70zXOc4j78NnA8sSu+MUoI9Er\\\\\\\/TZX2UDR8tUmcnXuV05PK5qrEOOlkug+C4XR76bWRS1NVknFQp0dFVNTuDWTEQbC9Ivoq2bCrwdna9yU7hUMDdH\\\\\\\/0BDSMDzRXqYYx4Kr3eHKgdr1vHx+k542\\\\\\\/esPqdMXw6f6UXv1l5BDcoT6e7r5zeJV8Hve75o8JWEDz8dXS2xtFsqyo9RHNPTkt+l7ZnLrxtXmBaWi02UI\\\\\\\/5GDsE8xMnANW6wv6ornpGW5HzSSk7mnYZaa0mAW\\\\\\\/VZSqeielHFvp8FuXZZtOvGOcsSRWQAEd2A98sVmhgHvC9A\\\\\\\/0hFeFFKjeJ6R1eIAZRVg+Jx\\\\\\\/APH9g4902kIHXuaRtAhgnRyBv\\\u0022,\\\u0022iv\\\u0022:\\\u0022ezVCIVLgAlp6lmkCi+NFCA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022faa3e7da4560b2ffdae7415edcb7daba2a0de7c8d8b10f148b52c2a06f46979a\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-494296 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-genomic-prediction">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/10/3369" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-6DCYJBx28BKuKva1zY2aAshArRl5FN5HZmjrGwIjqA0" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/10/3369" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-aAAW5Qkvl3vb5zB3_1iap2xYeE2n7sYADIdbj6EFnlI" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/10/3359" title="Exon 3 of the NUMB Gene Emerged in the Chordate Lineage Coopting the NUMB Protein to the Regulation of MDM2" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/10/3381" title="A Bayesian Genomic Multi-output Regressor Stacking Model for Predicting Multi-trait Multi-environment Plant Breeding Data" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494296" id="node494296" data-pisa="ggg;9/10/3369" data-pisa-master="ggg;g3.119.400346" data-apath="/ggg/9/10/3369.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Predicting Longitudinal Traits Derived from High-Throughput Phenomics in Contrasting Environments Using Genomic Legendre Polynomials and B-Splines</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-2562-2741" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Mehdi Momen</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-8257-3595" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Malachy T. Campbell</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-9712-5824" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Harkamal Walia</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-3567-6911" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Gota Morota</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3369-3380;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400346 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494296" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Mehdi Momen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Animal and Poultry Sciences, Virginia Polytechnic Institute and State University, Blacksburg, VA, 24061 and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mehdi%2BMomen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Momen%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMehdi%2BMomen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-2562-2741" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Mehdi Momen</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Malachy T. Campbell </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Animal and Poultry Sciences, Virginia Polytechnic Institute and State University, Blacksburg, VA, 24061 and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Malachy%2BT.%2BCampbell%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Campbell%20MT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMalachy%2BT.%2BCampbell%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-8257-3595" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Malachy T. Campbell</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Harkamal Walia </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>Department of Agronomy and Horticulture, University of Nebraska-Lincoln, Lincoln, NE, 68583</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Harkamal%2BWalia%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Walia%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AHarkamal%2BWalia%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-9712-5824" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Harkamal Walia</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Gota Morota </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Animal and Poultry Sciences, Virginia Polytechnic Institute and State University, Blacksburg, VA, 24061 and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gota%2BMorota%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Morota%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGota%2BMorota%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-3567-6911" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Gota Morota</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:morota@vt.edu" class="" data-icon-position="" data-hide-link-title="0">morota@vt.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/9/10/3369" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:494296" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:494296/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/10/3369.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:494296" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:494296/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/10/3369.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:494296" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:494296/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/9/10/3369.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:494296" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:494296/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Recent advancements in phenomics coupled with increased output from sequencing technologies can create the platform needed to rapidly increase abiotic stress tolerance of crops, which increasingly face productivity challenges due to climate change. In particular, high-throughput phenotyping (HTP) enables researchers to generate large-scale data with temporal resolution. Recently, a random regression model (RRM) was used to model a longitudinal rice projected shoot area (PSA) dataset in an optimal growth environment. However, the utility of RRM is still unknown for phenotypic trajectories obtained from stress environments. Here, we sought to apply RRM to forecast the rice PSA in control and water-limited conditions under various longitudinal cross-validation scenarios. To this end, genomic Legendre polynomials and B-spline basis functions were used to capture PSA trajectories. Prediction accuracy declined slightly for the water-limited plants compared to control plants. Overall, RRM delivered reasonable prediction performance and yielded better prediction than the baseline multi-trait model. The difference between the results obtained using Legendre polynomials and that using B-splines was small; however, the former yielded a higher prediction accuracy. Prediction accuracy for forecasting the last five time points was highest when the entire trajectory from earlier growth stages was used to train the basis functions. Our results suggested that it was possible to decrease phenotyping frequency by only phenotyping every other day in order to reduce costs while minimizing the loss of prediction accuracy. This is the first study showing that RRM could be used to model changes in growth over time under abiotic stress conditions.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/genomic-prediction" class="hw-term hw-article-keyword hw-article-keyword-genomic-prediction" rel="nofollow">genomic prediction</a></li><li class="kwd"><a href="/keyword/phenomics" class="hw-term hw-article-keyword hw-article-keyword-phenomics" rel="nofollow">phenomics</a></li><li class="kwd"><a href="/keyword/longitudinal-modeling" class="hw-term hw-article-keyword hw-article-keyword-longitudinal-modeling" rel="nofollow">longitudinal modeling</a></li><li class="kwd"><a href="/keyword/random-regression" class="hw-term hw-article-keyword hw-article-keyword-random-regression" rel="nofollow">random regression</a></li><li class="kwd"><a href="/keyword/time-series" class="hw-term hw-article-keyword hw-article-keyword-time-series" rel="nofollow">time series</a></li><li class="kwd"><a href="/keyword/genpred" class="hw-term hw-article-keyword hw-article-keyword-genpred" rel="nofollow">GenPred</a></li><li class="kwd"><a href="/keyword/shared-data-resources" class="hw-term hw-article-keyword hw-article-keyword-shared-data-resources" rel="nofollow">Shared Data Resources</a></li></ul><div class="section other1" id="sec-1"><h2 class="">Background</h2><p id="p-3">Plant biology has become a large-scale, data-rich field with the development of high-throughput technologies for genomics and phenomics. This has increased the feasibility of data driven approaches to be applied to address the challenge of developing climate-resilient crops (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Tester and Langridge 2010</a>). Crop responses to environmental changes are highly dynamic and have a strong temporal component. Such responses are also known as function-valued traits, for which means and covariances along the trajectory change continuously. Single time point measurements of phenotypes, however, only provide a snapshot, posing a series of challenges for research efforts aimed at understanding the ability of the plant to mount a tolerant response to an environmental constraint. Advancements in high-throughput phenotyping (HTP) technologies have enabled the automated collection of measurements at high temporal resolution to produce high density image data that can capture a plethora of morphological and physiological measurements (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Furbank and Tester 2011</a>). In particular, image-based phenotyping has been deemed a game changer because conventional phenotyping is laborious and often involves destructive methods, precluding repeated sampling over time from the same individual (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Ge <em>et al.</em> 2016</a>). More importantly, these HTP systems offer greater potential to uncover the time-specific molecular events driven by important genes that have yet to be discovered in genome-wide association studies (GWAS) or to perform forecasting of future phenotypes in longitudinal genomic prediction. Thus, integrating these HTP data into quantitative genetics has the potential to increase the rate of genetic gain in crops. However, to take full advantage of such opportunities, novel statistical methods that can fully leverage time series HTP data need to be developed.</p><p id="p-4">Recently, <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Campbell <em>et al.</em> (2018)</a> used a random regression model (RRM) to perform genomic prediction for longitudinal HTP traits in controlled environments, such as greenhouses, using Legendre polynomials as the choice of a basis function to model dependencies across time. They also demonstrated that RRM could be used to achieve reasonable prediction accuracy in a cross-validation (CV) framework to forecast future phenotypes based on information from earlier growth stages. RRM also enables the calculation of (co)variances and genetic values at any time between the beginning and end of the trajectory, even including time points that lack phenotypic information. This study showed that RRM could effectively describe the temporal dynamics of genetic signals by accounting for mean and covariance structures that are subjected to change over time (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Kirkpatrick <em>et al.</em> 1990</a>). However, the utility of RRM for plants under an abiotic stress environment is not explored. This is a critical unknown as the crop productivity is greatly limited by environmental challenges such as drought and heat stress. In addition to the Legendre polynomials, spline functions can be used to describe the relationships between image-based phenomics and genomics data in longitudinal modeling (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">White <em>et al.</em> 1999</a>). In particular, B-spline functions have been used to study a variety of traits, such as growth records, in animal breeding in terms of model goodness of fit using pedigree data (<em>e.g.</em>, <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Meyer 2005</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Baldi <em>et al.</em> 2010</a>); however, its application to HTP data in plants and its predictive ability from a CV perspective remains untested.</p><p id="p-5">Here we present our results from the performance of RRM applied to HTP temporal shoot biomass data in response to control and water-limited conditions using Legendre polynomials and spline functions. We selected drought stress because water limitation significantly impacts shoot growth and is the major limitation for agricultural productivity and global food security.</p></div><div class="section materials-methods" id="sec-2"><h2 class="">Materials and Methods</h2><div id="sec-3" class="subsection"><h3>Plant materials and greenhouse conditions</h3><p id="p-6">Three hundred fifty-seven accessions <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-1.gif"/></noscript></span></span> of the rice (<em>O. Sativa</em>) diversity panel 1 (RDP1) were selected for this study (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Zhao <em>et al.</em> 2011</a>). Seeds were surface sterilized with Thiram fungicide and germinated on moist paper towels in plastic boxes for three days. For each accession, three uniformly germinated seedlings were selected and transplanted to pots (150mm diameter Ã 200 mm height) filled with 2.5 kg of UC Mix. Square containers were placed below each pot to allow water to collect. The plants were grown in saturated soil on greenhouse benches prior to phenotyping.</p><p id="p-7">All lines were genotyped with 44,000 single nucleotide polymorphisms (SNPs) (<a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Zhao <em>et al.</em> 2011</a>). We used PLINK v1.9 software (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Purcell <em>et al.</em> 2007</a>) to remove SNPs with a call rate â¤ 0.95 and a minor allele frequency â¤ 0.05. Missing genotypes were imputed using Beagle software version 3.3.2 (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Browning and Browning 2007</a>). Finally, 34,993 SNPs were retained for further analysis.</p></div><div id="sec-4" class="subsection"><h3>Experimental design and drought treatment</h3><p id="p-8">All experiments were conducted at the Plant Accelerator, Australian Plant Phenomics Facility, at the University of Adelaide, SA, Australia. The panel was phenotyped for a digital metric representing shoot growth over 20 days of progressive drought using an image-based phenomics platform. Each plant was phenotyped daily over a period of 20 days and the imaging interval was â¼24 hr. The details of the experimental design are provided in <a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Campbell <em>et al.</em> (2018)</a>. Briefly, each experiment consisted of 357 accessions from RDP1 and was repeated three times from February to April 2016. Two smart-greenhouses were used for each experiment. In each smart-greenhouse, the accessions were distributed across 432 pots positioned across 24 lanes. The experimental design used in this study is similar to the partially replicated paired design (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Cullis <em>et al.</em> 2006</a>). Here, a set of test entries is replicated while the remaining test entries are unreplicated. This design was slightly modified to accommodate the different water treatments and allow comparison of treatments within each accession. To this end, each accession was assigned to two consecutive pots, and the water treatments were randomly assigned to each pot. In each experiment, 54 accessions were randomly selected and replicated twice.</p><p id="p-9">Seven days after transplant (DAT), plants were thinned to one seedling per pot. Two layers of blue mesh were placed on top of the pots to reduce evaporation. The plants were loaded on to the imaging system and were watered to 90% field capacity (FC) DAT. On the 13 DAT, each pot was watered to 90% and was imaged to obtain an initial phenotype before the onset of drought. One plant from each pair was randomly selected for drought treatment. Water was withheld from drought plants until 10% FC, and after which water was applied to maintain 10% FC. For the duration of the experiment, the control plants were maintained at 100% FC.</p></div><div id="sec-5" class="subsection"><h3>Statistical analysis of phenotypic data</h3><p id="p-10">Visible images were processed, and digital features were extracted using the open-source Python library Image Harvest (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Knecht <em>et al.</em> 2016</a>). The sum of plant pixels from the two sides and one top view of red/green/blue (RGB) images was summed and used as a measure of shoot biomass. This digital phenotype is referred to as the projected shoot area (PSA) throughout this study. Several studies have reported a high correlation between PSA estimates and shoot biomass (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Campbell <em>et al.</em> 2015</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Golzarian <em>et al.</em> 2011</a>; <a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Knecht <em>et al.</em> 2016</a>). Prior to downstream analyses, outlier plants at each time point were detected for each trait using the 1.5 interquartile range rule, and potential outliers were plotted along with their treatment counterparts and inspected visually. Plants that exhibited abnormal growth patterns were removed. In total, we used records from 2,415 plants across both treatments. Briefly, there were 1,208-1,209 records in control for each day of imaging and 1,205-1,206 records for each day in low water treatment. All accessions had at least two records for each treatment-day block.</p><p id="p-11">Raw phenotypic measurements were adjusted for downstream genetic analyses prior to fitting RRM. Best linear unbiased estimators (BLUE) were computed for each accession by fitting experimental effect with three levels and replication within experiment for some of the accessions. We postulated that observations at each time point follow the model <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-2.gif"/></noscript></span></span>, where X and Z are <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-3.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-4.gif"/></noscript></span></span> orders of incident matrices linking observations (<span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-5.gif"/></noscript></span></span>) to systematic effects (<em>f</em>) and number of accessions (<em>n</em>), respectively, y is an <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-6.gif"/></noscript></span></span> vector of observations at each time point, Î² is a <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-7.gif"/></noscript></span></span> vector of systematic effects, u is a <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-8.gif"/></noscript></span></span> vector of accession effects, and Ïµ is an <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-9.gif"/></noscript></span></span> vector of residuals with <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-10.gif"/></noscript></span></span>, where I is an identity matrix. This was followed by fitting a RRM-based genomic prediction approach to predict phenotypes as described below.</p></div><div id="sec-6" class="subsection"><h3>Random regression model</h3><p id="p-12">We conducted quantitative genetics modeling of image-derived phenotypes using a RRM to assess how well we could predict dynamic genetic signals. The RRM assumes that genetic effects and genetic variances are not constant and can vary continuously across the trajectory. This leads to better prediction of time-dependent complex traits by estimating heterogeneous single nucleotide polymorphism (SNP) effects across the trajectory. Specifically, we viewed the trajectory of digital image-processed longitudinal records as an infinite-dimensional characteristic that could be modeled by a smooth function (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Meyer and Hill 1997</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Van der Werf <em>et al.</em> 1998</a>). Changes in PSA (BLUEs) over time were modeled through Legendre polynomials and B-splines of time at phenotyping. The general formula for the RRM was as follows:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-11.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-12.gif"/></noscript></span></span> is a time covariate coefficient defined by a basis function evaluated at time point <em>t</em> belonging to the <em>j</em>th accession; <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-13.gif"/></noscript></span></span> is a <em>k</em>th fixed regression coefficient for the populationâs mean growth trajectory; <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-14.gif"/></noscript></span></span> is a <em>k</em>th random regression coefficient associated with the additive genetic effects of the <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-15.gif"/></noscript></span></span> accession; <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-16.gif"/></noscript></span></span> is the number of random regression parameters for fixed effect time trajectories; <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-17.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-18.gif"/></noscript></span></span> are the number of random regression parameters for random effects; and <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-19.gif"/></noscript></span></span> is a <em>k</em>th permanent environmental random regression coefficient for the accession <em>j</em>. The starting values of index <em>k</em>, and <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-20.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-21.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-22.gif"/></noscript></span></span> are defined separately for Legendre polynomials and B-splines below.</p><p id="p-13">In the matrix notation, the above equation can be rewritten as<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-23.gif"/></noscript></span></span>where Î² is a vector of solutions for fixed regressions; u is the additive genetic random regression coefficients; <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-24.gif"/></noscript></span></span> is the permanent environmental random regression coefficients; Ïµ is the residuals; and X, Z, and Q are corresponding incident matrices. Here, <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-25.gif"/></noscript></span></span> was defined as the resemblance between records of an individual due to non-random environmental effects that are persistent across the 20 time points. We assumed a multivariate-Gaussian distribution and the variance-covariance structure of<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-26.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-27.gif"/></noscript></span></span> is the genomic relationship matrix of <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">VanRaden (2008)</a>, where <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-28.gif"/></noscript></span></span> represents a centered and standardized marker matrix and <em>p</em> is the number of markers; <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-29.gif"/></noscript></span></span> is the covariance function between the random regression coefficients for the additive genetic effect; â is the Kronecker product; <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-30.gif"/></noscript></span></span> is the covariance function between the random regression coefficients for the permanent environmental effects; and <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-31.gif"/></noscript></span></span> is a diagonal matrix of heterogeneous residuals varying across times, where <span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-32.gif"/></noscript></span></span> is the residual variance.</p></div><div id="sec-7" class="subsection"><h3>Choice of basis function</h3><p id="p-14">The choice of the basis function to model the shape of the longitudinal measurements is critical. An ideal basis function has adequate potential to capture real patterns of changes in variance along a continuous scale (time) for a given trait (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Meyer and Kirkpatrick 2005</a>). In this study, we used RRM with two basis functions, <em>i.e.</em>, Legendre polynomials (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Meyer 1998</a>) and B-splines (<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Meyer 2005</a>), to describe line-specific curves for the PSA trajectory over the day of imaging.</p><div id="sec-8" class="subsection"><h4>Legendre polynomials:</h4><p id="p-15">Applying parametric shape functions for covariates of time is challenging because these covariates tend to generate high correlations among trajectories (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Mrode 2014</a>). For this reason, fitting Legendre polynomials of time at recording as covariables is a common choice to model growth curves because these polynomials greatly reduce the correlations between estimated random regression coefficients and make no prior assumptions regarding the shape of the longitudinal curve. This function has been used widely in animal breeding for many years (<em>e.g.</em>, <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Jamrozik and Schaeffer 1997</a>) and has recently been used in plant breeding as well (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Sun <em>et al.</em> 2017</a>; <a id="xref-ref-6-3" class="xref-bibr" href="#ref-6">Campbell <em>et al.</em> 2018</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Marchal <em>et al.</em> 2019</a>). Suppose <em>d</em> is the order of fit or degree of the polynomial. Legendre polynomials evaluated at the standardized time points were computed as <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-33.gif"/></noscript></span></span>, where M is the <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-34.gif"/></noscript></span></span> by <span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-35.gif"/></noscript></span></span> matrix containing the polynomials of the standardized time covariate <span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-36.gif"/></noscript></span></span> and Î is the <span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-37.gif"/></noscript></span></span> matrix of Legendre polynomial coefficients (<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Kirkpatrick <em>et al.</em> 1990</a>). Here, <span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-38.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-39.gif"/></noscript></span></span> because PSA was measured for 20 days. We chose the same orders of polynomials for fixed, additive, and permanent environmental coefficients as previously described <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Schaeffer (2016)</a>. We compared linear (<span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-40.gif"/></noscript></span></span>) and quadratic (<span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-41.gif"/></noscript></span></span>) Legendre polynomials in this study. Thus, the numbers of regression coefficients were <span class="inline-formula" id="inline-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-42.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-43.gif"/></noscript></span></span> for linear and quadratic Legendre polynomials, respectively.</p></div><div id="sec-9" class="subsection"><h4>B-splines:</h4><p id="p-16">Spline functions consist of individual segments of polynomials joined at specific points called knots. B-splines first require determination of the total number of knots <em>K</em>. Although a large number of knots will increase complexity, too few knots will decrease accuracy. This basis function is reported to offer several advantages, including better numerical properties compared with polynomials, especially when there are high genetic variances at the extremes of the trajectory period, negative correlations between the most distant time point measurements, and a small number of records, particularly at the last stage of the trajectory (<a id="xref-ref-21-3" class="xref-bibr" href="#ref-21">Meyer 2005</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Misztal 2006</a>). Here, we used equidistant knots, and the B-spline function was computed from Cox-de Boorâs recursion formula (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">De Boor 2001</a>). Given a preconsidered knot sequence of time <em>t</em>, the covariables for B-splines of degree <span class="inline-formula" id="inline-formula-41"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-44.gif"/></noscript></span></span> were defined by assuming values of unity for all points in a given interval or zero otherwise. For the <em>i</em>th interval given by knots<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-45.gif"/></noscript></span></span>where <em>T</em> is the threshold in time interval. According to <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">De Boor (2001)</a>, the matrix Î¦ of B-spline for higher-order polynomials can be defined by recursion<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-46.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-46.gif"/></noscript></span></span>This indicates that a B-spline of degree <em>d</em> is simply a function of B-splines of degree <span class="inline-formula" id="inline-formula-42"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-47.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-47.gif"/></noscript></span></span>. Note that the number of random regression coefficients depends on the number of knots and order of polynomials for B-splines. In general, the number of regression coefficients is given by <span class="inline-formula" id="inline-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-48.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-48.gif"/></noscript></span></span> (<a id="xref-ref-21-4" class="xref-bibr" href="#ref-21">Meyer 2005</a>). In this study, we fitted linear B-splines with <span class="inline-formula" id="inline-formula-44"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-49.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-49.gif"/></noscript></span></span> or <span class="inline-formula" id="inline-formula-45"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-50.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-50.gif"/></noscript></span></span> knots to divide the time points into equally spaced intervals. The same number of knots was considered for fixed trajectories, additive genetic, and permanent environmental coefficients. Thus, the numbers of regression coefficients were three <span class="inline-formula" id="inline-formula-46"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-51.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-51.gif"/></noscript></span></span> and four <span class="inline-formula" id="inline-formula-47"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-52.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-52.gif"/></noscript></span></span> for <span class="inline-formula" id="inline-formula-48"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-53.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-53.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-49"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-54.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-54.gif"/></noscript></span></span> knots, respectively.</p></div></div><div id="sec-10" class="subsection"><h3>Goodness of model fit</h3><p id="p-17">The goodness of fit of RRM was assessed by computing the Akaikeâs information criterion (AIC) (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Akaike 1974</a>) and the SchwarzâBayesian information criterion (BIC) (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Schwarz 1978</a>). The best model was selected based on the largest AIC and BIC values after multiplying by -1/2. We used the Wombat software to fit RMM in this study (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Meyer 2007</a>).</p></div><div id="sec-11" class="subsection"><h3>Cross-validation scenarios</h3><p id="p-18">As graphically represented in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>, three different CV scenarios were designed to train the RRM. In all scenarios, prediction accuracy was evaluated by computing Pearson correlations between predicted genetic values and PSA in the testing set. Each of the CV scenarios is described below.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/10/3369/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Pictorial representation of three cross-validation schemes used for predicting longitudinal projected shoot area (PSA) using a random regression model coupled with Legendre polynomials and B-splines. The data set consisted of 357 lines. CV1: 179 lines were used as the training set to predict PSA for the remaining 178 lines. Here, all 20 time points in the training set were used to predict PSA at each of 20 time points for a new set of lines. CV2: The data set was split into two longitudinal stages. The model was trained using the earlier growth stages to predict PSA at the second part of growth stages. We increased the number of time points used for training in a sequential manner. CV3: This was used to evaluate the impact of phenotyping frequency in the training data set on longitudinal prediction accuracy. Observations on odd days were used (CV3A), Observations on even days were used (CV3B), and keep one and delete two consecutive time points (CV3C). TP: time points." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2108681262" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Pictorial representation of three cross-validation schemes used for predicting longitudinal projected shoot area (PSA) using a random regression model coupled with Legendre polynomials and B-splines. The data set consisted of 357 lines. CV1: 179 lines were used as the training set to predict PSA for the remaining 178 lines. Here, all 20 time points in the training set were used to predict PSA at each of 20 time points for a new set of lines. CV2: The data set was split into two longitudinal stages. The model was trained using the earlier growth stages to predict PSA at the second part of growth stages. We increased the number of time points used for training in a sequential manner. CV3: This was used to evaluate the impact of phenotyping frequency in the training data set on longitudinal prediction accuracy. Observations on odd days were used (CV3A), Observations on even days were used (CV3B), and keep one and delete two consecutive time points (CV3C). TP: time points.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/10/3369/F1.medium.gif" width="440" height="429"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/9/10/3369/F1.medium.gif" width="440" height="429"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/10/3369/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/10/3369/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494459" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-19" class="first-child">Pictorial representation of three cross-validation schemes used for predicting longitudinal projected shoot area (PSA) using a random regression model coupled with Legendre polynomials and B-splines. The data set consisted of 357 lines. CV1: 179 lines were used as the training set to predict PSA for the remaining 178 lines. Here, all 20 time points in the training set were used to predict PSA at each of 20 time points for a new set of lines. CV2: The data set was split into two longitudinal stages. The model was trained using the earlier growth stages to predict PSA at the second part of growth stages. We increased the number of time points used for training in a sequential manner. CV3: This was used to evaluate the impact of phenotyping frequency in the training data set on longitudinal prediction accuracy. Observations on odd days were used (CV3A), Observations on even days were used (CV3B), and keep one and delete two consecutive time points (CV3C). TP: time points.</p><div class="sb-div caption-clear"></div></div></div><div id="sec-12" class="subsection"><h4>CV1:</h4><p id="p-20">In the first CV scenario (CV1), the whole data set was divided into two subsets, <em>i.e.</em>, training and testing sets, each including 179 and 178 accessions, respectively. All 20 time points in the training set were fit to the RRM using Legendre polynomials and B-splines, and we predicted phenotypic values of 20 time points for lines in the testing set. Random assignment of individuals into the training and testing sets was repeated 10 times. The equation for CV1 was set up in the following manner. The time-specific genetic value of the <em>i</em>th individual in the training set was computed as <span class="inline-formula" id="inline-formula-50"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-55.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-55.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-51"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-56.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-56.gif"/></noscript></span></span> is the estimated genetic value of the individual <em>i</em> at time <em>t</em>; <span class="inline-formula" id="inline-formula-52"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-57.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-57.gif"/></noscript></span></span> is the <em>t</em>th row vector of the <span class="inline-formula" id="inline-formula-53"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-58.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-58.gif"/></noscript></span></span> matrix Î¦; and <span class="inline-formula" id="inline-formula-54"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-59.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-59.gif"/></noscript></span></span> is the <em>i</em>th column vector of the <span class="inline-formula" id="inline-formula-55"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-60.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-60.gif"/></noscript></span></span> matrix u. Then, a vector of predicted genetic values of individuals in the testing set at time <em>t</em> was obtained as <span class="inline-formula" id="inline-formula-56"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-61.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-61.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-57"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-62.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-62.gif"/></noscript></span></span> is the genomic relationship matrix between the testing and training set and <span class="inline-formula" id="inline-formula-58"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-63.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-63.gif"/></noscript></span></span> is the inverse of genomic relationship matrix between the training set. Because CV1 is not a forecasting task, a standard multi-trait model (MTM) was also fitted as a baseline model considering longitudinal traits as different traits (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Henderson and Quaas 1976</a>).<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-64.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-64.gif"/></noscript></span></span>where Î² is a vector of systematic effects; u is the vector of additive genetic values; Ïµ is the residuals; and X and Z are corresponding incident matrices. The joint distribution of u and Ïµ follows multivariate normal<span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-65.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-65.gif"/></noscript></span></span>where G is the genomic relationship matrix, I is the identity matrix, and <span class="inline-formula" id="inline-formula-59"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-66.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-66.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-60"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-67.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-67.gif"/></noscript></span></span> refer to <span class="inline-formula" id="inline-formula-61"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-68.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-68.gif"/></noscript></span></span> dimensional additive genetic and residual variance-covariance matrices, respectively. In other words, the diagonal and off-diagonal elements of Î£ include variance at each time point and covariance between time points, respectively. The BLUPF90 family of programs was used to fit MTM with 20 traits (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Misztal <em>et al.</em> 2002</a>).</p></div><div id="sec-13" class="subsection"><h4>CV2:</h4><p id="p-21">The second CV scenario (CV2) was related to forecasting future phenotypes from longitudinal traits at early time points. Individuals in the training set were used to forecast their yet-to-be observed PSA values at later time points from information available at earlier time points. The first quarter of the time points {<em>t</em> = 1, 2, 3, 4, 5} was used as the training set, and the remaining time points {<em>t</em> = 6, 7,â¯, 20} were predicted for each line in the training set. This was followed by sequentially increasing the number of time points used to train the model so that in the last run, three quarters of the time points {<em>t</em> = 1, 2, â¯ 15} were used in the training set to forecast phenotypes at the last quarter of time points {<em>t</em> = 16, 17, 18, 19, 20}. This CV scenario was designed to find a sufficient set of earlier time points to obtain reasonable longitudinal prediction accuracy and is known as walk forward validation. We set up the CV2 equation by first estimating the random regression coefficient matrix u using <span class="inline-formula" id="inline-formula-62"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-69.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-69.gif"/></noscript></span></span>, which was computed from time point 1 to time point <em>t</em>. The prediction of future time points <span class="inline-formula" id="inline-formula-63"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-70.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-70.gif"/></noscript></span></span> (<span class="inline-formula" id="inline-formula-64"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-71.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-71.gif"/></noscript></span></span>) for an individual <em>i</em> was carried out by <span class="inline-formula" id="inline-formula-65"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-72.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-72.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-66"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-73.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-73.gif"/></noscript></span></span> is the <span class="inline-formula" id="inline-formula-67"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-74.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-74.gif"/></noscript></span></span>th row vector of <span class="inline-formula" id="inline-formula-68"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-75.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-75.gif"/></noscript></span></span> by <span class="inline-formula" id="inline-formula-69"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-76.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-76.gif"/></noscript></span></span> matrix Î¦; and <span class="inline-formula" id="inline-formula-70"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-77.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-77.gif"/></noscript></span></span> is the <em>i</em>th column vector of the number of random regression coefficients by <em>n</em> matrix u.</p></div><div id="sec-14" class="subsection"><h4>CV3:</h4><p id="p-22">The third CV scenario (CV3) was designed to evaluate whether it was possible to reduce the phenotyping frequency while still maintaining a high prediction accuracy for the last quarter of observations. We used the last case in CV2 such that time points {<em>t</em> = 1, 2, â¯, 15} were used for the training set to forecast the last quarter of observations {<em>t</em> = 16, 17, 18, 19, 20}. We then reduced the number of time points used in the training set as follows: A, observations on odd days {<em>t</em> = 1, 3, â¯, 15} were used; B, observations on even days {<em>t</em> = 2, 4, â¯, 14} were used; C, keep one and delete two consecutive time points. In CV2 and CV3 scenarios, half of the individuals were randomly selected to fit and evaluate the model, and the analysis was repeated 10 times. If the loss of prediction accuracy was minimal, it was possible to reduce the phenotyping cost. The equation for CV3 was set up in the same way as that for CV2.</p></div></div><div id="sec-15" class="subsection"><h3>Data availability</h3><p id="p-23">Phenotypic data used herein are available in Supplementary File S1 at Figshare. Genotypic data regarding the rice accessions can be downloaded from the rice diversity panel website (<a href="http://www.ricediversity.org/">http://www.ricediversity.org/</a>) and also available in Supplementary File S2 at Figshare. Supplemental material available at FigShare: <a href="https://doi.org/10.25387/g3.9383543">https://doi.org/10.25387/g3.9383543</a>.</p></div></div><div class="section results" id="sec-16"><h2 class="">Results</h2><div id="sec-17" class="subsection"><h3>Assessing model fit</h3><p id="p-24"><a id="xref-fig-2-1" class="xref-fig" href="#F2">Figures 2A</a> and <a id="xref-fig-2-2" class="xref-fig" href="#F2">2B</a> show the box plots of the original PSA and BLUE for the phenotypic trajectories over the 20 days of imaging for control and water-limited conditions. The PSA for control and water-limited plants diverged significantly six days after initiating the drought treatment (<em>p</em> &lt; 0.0025). Supplementary Figure 1 (File S3) shows the linear or quadratic forms of Legendre polynomials and three and four knot-based B-spline curves over 20 days of imaging. For Legendre polynomials, intercept, linear, and quadratic coefficients are represented in black, red, and green, respectively. For B-spline, knot 1, knot 2, and knot 3 are represented in black, red, and green, respectively.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/10/3369/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="A: Box plots of projected shoot area (PSA) over the 20 days of imaging in two environments: controlled and water-limited conditions. B: Best linear unbiased estimators over the 20 days of imaging in two environments: controlled and water-limited conditions." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2108681262" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;A: Box plots of projected shoot area (PSA) over the 20 days of imaging in two environments: controlled and water-limited conditions. B: Best linear unbiased estimators over the 20 days of imaging in two environments: controlled and water-limited conditions.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/10/3369/F2.medium.gif" width="440" height="245"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/9/10/3369/F2.medium.gif" width="440" height="245"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/10/3369/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/10/3369/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494444" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-25" class="first-child">A: Box plots of projected shoot area (PSA) over the 20 days of imaging in two environments: controlled and water-limited conditions. B: Best linear unbiased estimators over the 20 days of imaging in two environments: controlled and water-limited conditions.</p><div class="sb-div caption-clear"></div></div></div><p id="p-26"><a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a> summarizes the goodness of fits of RRM coupled with linear and quadratic Legendre polynomials and B-spline functions in control and water-limited conditions. For the Legendre polynomials, quadratic forms require more parameters to be estimated compared with linear forms. Similar to observation for B-splines, the presence of a greater number of knots suggested that there were more parameters to be estimated. Under control conditions, the best goodness of fit was obtained by linear Legendre polynomials, followed by linear B-splines with three knots, linear B-splines with four knots, and quadratic Legendre polynomials according to AIC scores. According to BIC scores, linear Legendre polynomials performed the best, followed by linear B-splines with three knots, quadratic Legendre polynomials, and linear B-splines with four knots. Under water-limited conditions, the best goodness of fit was given by linear Legendre polynomials, followed by linear B-splines with three knots, quadratic Legendre polynomials, and linear B-splines with four knots for both AIC and BIC scores. The number of parameters in the model varied from 26 to 40.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/494358/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/494358/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Assessing goodness of fit for two random regression models (Legendre polynomials and B-splines) used to predict projected shoot area measured over 20 time points</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-18" class="subsection"><h3>Cross-validation</h3><p id="p-28">The results from CV1 are shown in <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>. This CV was designed to evaluate the accuracy of predicting testing set individuals using all time points. Under control conditions, MTM performed relatively better than RRM up to day 3. The prediction accuracy of RRM increased subsequently and after the 10<sup>th</sup> day of imaging, the best prediction was given by linear Legendre, followed by quadratic Legendre, linear B-spline with three knots, and linear B-spine with four knots. Overall, RRM performed better than MTM, and linear Legendre was the best prediction machine throughout the growth stages. Under water-limited conditions, prediction accuracy was lower compared with those of control conditions. All RRM delivered higher prediction than MTM except for the first two time points. Although Legendre polynomials performed better than B-splines until day 7, the difference between these approaches became negligible afterward.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/10/3369/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Prediction accuracy obtained from cross-validation 1 scenario. Total of 179 lines were used as the training set to predict PSA for the remaining 178 lines. Here, all 20 time points in the training set were used to predict PSA at each of 20 time points for a new set of lines. LEGL: linear Legendre polynomials; LEGQ: quadratic Legendre polynomials; BSPL3: linear B-splines with three knots; BSPL4: linear B-spline with four knots; MTM: multi-trait model." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2108681262" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Prediction accuracy obtained from cross-validation 1 scenario. Total of 179 lines were used as the training set to predict PSA for the remaining 178 lines. Here, all 20 time points in the training set were used to predict PSA at each of 20 time points for a new set of lines. LEGL: linear Legendre polynomials; LEGQ: quadratic Legendre polynomials; BSPL3: linear B-splines with three knots; BSPL4: linear B-spline with four knots; MTM: multi-trait model.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/10/3369/F3.medium.gif" width="440" height="241"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/9/10/3369/F3.medium.gif" width="440" height="241"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/10/3369/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/10/3369/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494335" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-29" class="first-child">Prediction accuracy obtained from cross-validation 1 scenario. Total of 179 lines were used as the training set to predict PSA for the remaining 178 lines. Here, all 20 time points in the training set were used to predict PSA at each of 20 time points for a new set of lines. LEGL: linear Legendre polynomials; LEGQ: quadratic Legendre polynomials; BSPL3: linear B-splines with three knots; BSPL4: linear B-spline with four knots; MTM: multi-trait model.</p><div class="sb-div caption-clear"></div></div></div><p id="p-30"><a id="xref-fig-4-1" class="xref-fig" href="#F4">Figures 4</a> and <a id="xref-fig-5-1" class="xref-fig" href="#F5">5</a> show the accuracy of CV2 under control and water-limited conditions, respectively. This CV was designed to test how much information from the previous time points was required to achieve reasonable prediction accuracy at later growth stages. Under control conditions, we found that the best prediction for the last five time points was achieved when using information from all prior time points (15/5 CV2 subscenario). This suggested that having more information from previous time points to train the model would result in higher prediction accuracy. Using the first five time points to train the model resulted in the worse prediction (5/15 CV2 subscenario). Thus, it is likely that the prediction accuracy in RRM declined because we attempted to estimate numerous parameters from only five time points. Legendre polynomials yielded better and more stable prediction than B-splines. We observed a similar trend under water-limited conditions; that is, using more previous time points to train the model resulted in higher prediction accuracy. However, the accuracy of prediction was unstable and decreased dramatically. There was no noticeable difference between the Legendre polynomials and B-splines in terms of performance.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/10/3369/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Prediction accuracy of cross-validation scenario 2 in control conditions. Each line depicts the different number of training and testing sets partitioning at the time point levels. LEGL: linear Legendre polynomials; LEGQ: quadratic Legendre polynomials; BSPL3: linear B-splines with three knots; BSPL4: linear B-spline with four knots." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2108681262" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Prediction accuracy of cross-validation scenario 2 in control conditions. Each line depicts the different number of training and testing sets partitioning at the time point levels. LEGL: linear Legendre polynomials; LEGQ: quadratic Legendre polynomials; BSPL3: linear B-splines with three knots; BSPL4: linear B-spline with four knots.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/10/3369/F4.medium.gif" width="440" height="351"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/9/10/3369/F4.medium.gif" width="440" height="351"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/10/3369/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/10/3369/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494468" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-31" class="first-child">Prediction accuracy of cross-validation scenario 2 in control conditions. Each line depicts the different number of training and testing sets partitioning at the time point levels. LEGL: linear Legendre polynomials; LEGQ: quadratic Legendre polynomials; BSPL3: linear B-splines with three knots; BSPL4: linear B-spline with four knots.</p><div class="sb-div caption-clear"></div></div></div><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/10/3369/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Prediction accuracy of cross-validation scenario 2 in water-limited conditions. Each line depicts the different number of training and testing sets partitioning at the time point levels. LEGL: linear Legendre polynomials; LEGQ: quadratic Legendre polynomials; BSPL3: linear B-splines with three knots; BSPL4: linear B-spline with four knots." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2108681262" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Prediction accuracy of cross-validation scenario 2 in water-limited conditions. Each line depicts the different number of training and testing sets partitioning at the time point levels. LEGL: linear Legendre polynomials; LEGQ: quadratic Legendre polynomials; BSPL3: linear B-splines with three knots; BSPL4: linear B-spline with four knots.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/10/3369/F5.medium.gif" width="440" height="353"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/9/10/3369/F5.medium.gif" width="440" height="353"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/10/3369/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/10/3369/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494306" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-32" class="first-child">Prediction accuracy of cross-validation scenario 2 in water-limited conditions. Each line depicts the different number of training and testing sets partitioning at the time point levels. LEGL: linear Legendre polynomials; LEGQ: quadratic Legendre polynomials; BSPL3: linear B-splines with three knots; BSPL4: linear B-spline with four knots.</p><div class="sb-div caption-clear"></div></div></div><p id="p-33"><a id="xref-fig-6-1" class="xref-fig" href="#F6">Figures 6</a> and <a id="xref-fig-7-1" class="xref-fig" href="#F7">7</a> show the CV3 accuracy under control and water-limited conditions, respectively. We designed this CV to evaluate whether it was possible to reduce phenotyping frequency and phenotyping costs without sacrificing prediction accuracy. Under control conditions, the prediction accuracy of CV3A, CV3B, and CV3C all decreased relative to the benchmark scenario in CV2, where all of the first 15 time points were used for the training set to forecast the last five time points. Although removing two consecutive time points did not improve performance (CV3C), the prediction accuracy from phenotyping every other day was still relatively high (CV3A and CV3B). In general, the linear B-splines performed the best, and differences between scenarios were minimal. Under water-limited conditions, we observed the same trend, but the prediction accuracy was more unstable and decreased relative to control conditions. The quadratic Legendre polynomials and B-splines with four knots did not perform well, possibly due to overfitting.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/10/3369/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Prediction accuracy of cross-validation scenario 3 in control conditions. A: only observations in the odd days were used; B: only observations in the even days were used; C: keep one and delete two consecutive time points; CV2: use all available previous time points; LEGL: linear Legendre polynomials; LEGQ: quadratic Legendre polynomials; BSPL3: linear B-splines with three knots; BSPL4: linear B-spline with four knots." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2108681262" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Prediction accuracy of cross-validation scenario 3 in control conditions. A: only observations in the odd days were used; B: only observations in the even days were used; C: keep one and delete two consecutive time points; CV2: use all available previous time points; LEGL: linear Legendre polynomials; LEGQ: quadratic Legendre polynomials; BSPL3: linear B-splines with three knots; BSPL4: linear B-spline with four knots.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/10/3369/F6.medium.gif" width="440" height="297"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/9/10/3369/F6.medium.gif" width="440" height="297"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/10/3369/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/10/3369/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494474" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-34" class="first-child">Prediction accuracy of cross-validation scenario 3 in control conditions. A: only observations in the odd days were used; B: only observations in the even days were used; C: keep one and delete two consecutive time points; CV2: use all available previous time points; LEGL: linear Legendre polynomials; LEGQ: quadratic Legendre polynomials; BSPL3: linear B-splines with three knots; BSPL4: linear B-spline with four knots.</p><div class="sb-div caption-clear"></div></div></div><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/10/3369/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Prediction accuracy of cross-validation scenario 3 in water-limited conditions. A: only observations in the odd days were used; B: only observations in the even days were used; C: keep one and delete two consecutive time points; CV2: use all available previous time points; LEGL: linear Legendre polynomials; LEGQ: quadratic Legendre polynomials; BSPL3: linear B-splines with three knots; BSPL4: linear B-spline with four knots." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2108681262" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Prediction accuracy of cross-validation scenario 3 in water-limited conditions. A: only observations in the odd days were used; B: only observations in the even days were used; C: keep one and delete two consecutive time points; CV2: use all available previous time points; LEGL: linear Legendre polynomials; LEGQ: quadratic Legendre polynomials; BSPL3: linear B-splines with three knots; BSPL4: linear B-spline with four knots.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/10/3369/F7.medium.gif" width="440" height="295"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/9/10/3369/F7.medium.gif" width="440" height="295"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/10/3369/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/10/3369/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494479" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-35" class="first-child">Prediction accuracy of cross-validation scenario 3 in water-limited conditions. A: only observations in the odd days were used; B: only observations in the even days were used; C: keep one and delete two consecutive time points; CV2: use all available previous time points; LEGL: linear Legendre polynomials; LEGQ: quadratic Legendre polynomials; BSPL3: linear B-splines with three knots; BSPL4: linear B-spline with four knots.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-19"><h2 class="">Discussion</h2><p id="p-36">Image-based automated HTP technologies offer great potential for characterizing multi-faceted phenotypes at high temporal resolution. The use of HTP platforms plays a pivotal role in accelerating breeding efforts by providing the temporal resolution needed for capturing adaptive responses to environmental challenges, but the development of statistical methodologies to analyze image-based function-valued phenotypes has not kept pace with our ability to generate HTP data. Because phenomics and genomics landscapes for plants are constantly advancing, parallel efforts are required to develop tools for integrating diverse genomic and phenomic datasets characterized by high temporal resolution in genetic analysis. Rice is one of the most drought sensitive cereal crops, resulting in substantial yield losses. With predictions for greater climatic shifts in the future and increasing competition for fresh water resources, research that leverages the full potential of genomics and phenomics is needed to elucidate the genetic and physiological basis of drought tolerance. However, there is currently a lack of information regarding the modeling of temporal HTP data.</p><p id="p-37">RRM identifies the effects of heterogeneous SNPs that transiently influence key traits and translates this to prediction of phenotypes. The main idea behind RRM is to describe subject-specific curves through basis functions (<a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Meyer and Kirkpatrick 2005</a>). Although RRM has been successfully applied to pedigree-based animal breeding (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Schaeffer and Jamrozik 2008</a>), its utility is largely limited to evaluating goodness-of-fit for candidate models rather than CV-based prediction, and its integration into HTP data has not been reported. In this study, we coupled HTP data with high-density genomic infromation to carry out longitudinal prediction by capturing time-specific genetic signals. A diverse panel of rice accessions subjected to drought stress was used to illustrate the utility of the RRM for evaluating Legendre polynomials and B-splines of time at recording.</p><div id="sec-20" class="subsection"><h3>Longitudinal prediction</h3><p id="p-38">We found that it was possible to model longitudinal PSA responses under water-limited conditions, albeit with decreased prediction accuracy compared with that of the control. We also placed particular emphasis on comparing two basis functions, <em>i.e.</em>, Legendre polynomials and B-splines. To the best of our knowledge, the current study is the first to use a B-spline function to evaluate longitudinal prediction accuracy in the RRM applied to HTP data. Linear B-spline functions with <span class="inline-formula" id="inline-formula-97"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-104.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-104.gif"/></noscript></span></span> (two segments) or <span class="inline-formula" id="inline-formula-98"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-105.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-105.gif"/></noscript></span></span> knots (three segments) were used. B-splines have been reported to have better numerical properties (<em>e.g.</em>, lower computational requirement and faster convergence) than Legendre polynomials because each coefficient of a function affects only a part of the trajectory and can be used to estimate genetic parameters more smoothly while still adequately capturing the features of dynamic data (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Iwaisaki <em>et al.</em> 2005</a>; <a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Baldi <em>et al.</em> 2010</a>).</p><p id="p-39">We observed differences in prediction accuracy across models during early growth stages; however, differences were incremental when predicting later growth stages in the CV1 scenario, in which the training and testing sets were partitioned based on individuals. Overall, linear Legendre polynomials performed the best and was clearly an advancement over the MTM. Prediction performance in CV2, in which the training and testing sets were partitioned according to growth stages rather individuals, showed that it was possible to predict future phenotypes from information available from earlier trajectories. Here, linear and quadratic Legendre polynomials produced the highest and most stable prediction accuracy under control conditions, whereas linear B-splines with three knots performed better in the water-limited environment. The final scenario (CV3) demonstrated that we could decrease the phenotyping frequency by only phenotyping every other day to reduce the phenotyping cost while minimizing the loss of prediction accuracy. In this case, linear B-spline with three knots performed relatively well.</p><p id="p-40">B-spline functions require two parameters (the position of the knots and the number of knots) to be tuned. The position of knots can be chosen based on a trajectory pattern such that more knots are placed for rapidly changing time points, whereas less knots are placed for time points with slow changes (<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Misztal 2006</a>). Thus, the position of knots should be carefully chosen if the number of phenotyped individuals varies substantially at each growth stage. We chose equidistant knots in the current study because all accessions were phenotyped on the same days during the trajectory. The number of knots determines the number of segments fitted. When more knots are specified, the model becomes more complex. Although we used <span class="inline-formula" id="inline-formula-99"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-106.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-106.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-100"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-107.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3369/embed/mml-math-107.gif"/></noscript></span></span> based on previous literature and a visual inspection of the observed phenotypic trajectory, further investigations are warranted to explore the impact of the number of knots on prediction accuracy. The performance of quadratic B-spline functions was not evaluated in the current study because we encountered convergence issues, possibly due to the small sample size. In general, we found that the advantages of B-splines in inferential tasks compared with Legendre polynomials were not shown clearly in terms of prediction. This is likely because PSA trajectories were relatively simple exponential or monotonically increasing trajectories without obvious inflection points, indicating that the potential of B-splines was not able to be fully exploited in the current study.</p></div><div id="sec-21" class="subsection"><h3>Choice of parameters</h3><p id="p-41">We also found that ranking the models according to AIC and BIC revealed only mild agreement with prediction performance evaluated by CV, suggesting that the RRM that gives the best goodness-of-fit is not guaranteed to deliver the best prediction and vice versa. The choice for the order of fit or the number of knots is arguably the most challenging modeling aspect in the RRM. In the majority of literature describing the RRM, this parameter is mainly chosen based on AIC, BIC, or the eigendecomposition of the covariance matrix. The major issue regarding this approach is that there is a tendency to simply pick a model with the highest order of fit or the largest number of knots. However, this study, suggests finding the best parameter in terms of prediction accuracy obtained from CV.</p></div><div id="sec-22" class="subsection"><h3>Future perspective</h3><p id="p-42">We anticipate that the current work will guide us to conduct genomic selection of economically important traits on the longitudinal scale for the purpose of breeding crops that are better adapted to new environments or to less favorable challenging climatic conditions. Although in the current study, our aim was to assess RRM for genomic prediction of shoot biomass under contrasting water regimes, these frameworks can be extended to any time-resolved phenotype, provided there are enough time points with complete or partial records. Owing to the accessibility of HTP platforms in the public sector as well as the growing availability of unmanned aerial vehicles and other autonomous field-based platforms, many breeding programs are currently generating temporal phenotypic data. Although the temporal phenotypes themselves may not be a target of selection, these data can be utilized to improve selection for conventional end-point phenotypes such as yield. For instance, <a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Sun <em>et al.</em> (2017)</a> used parameters from RRM as covariates in a mixed model to improve prediction for yield in drought-stressed environments in wheat.</p><p id="p-43">The identification of genomic components over trajectories will provide information regarding the optimum time points to maximize cost-effective selection or to design a genome-assisted breeding program aiming to change the shape of the longitudinal response curve (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Schaeffer 2004</a>). Using our approach, we could evaluate all changes in plant biomass accumulation during the course of the experiment, in contrast to single time point analyses. Thus, we expect that RRM analysis will become the norm for modeling trajectories of function-valued HTP data because such approaches could be considered an extension of the widely used genomic best linear unbiased prediction model for time series data. Lastly, the utility of the RRM does not preclude its use in other applications. For example, the RRM offers a new avenue for performing longitudinal GWAS (<em>e.g.</em>, <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Howard <em>et al.</em> 2015</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Campbell <em>et al.</em> 2019</a>) and genotype-by-environment interactions using the reaction norm (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Arnold <em>et al.</em> 2019</a>). In summary, an RRM using Legendre polynomial or spline functions could be an effective option for modeling trait trajectories of HTP data and may have potential applications in characterizing phenotypic plasticity in plants.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-44">This work was supported by the National Science Foundation under Grant Number 1736192 to HW and GM, and Virginia Polytechnic Institute and State University startup funds to GM. MTC and HW designed and conducted the experiments. MM analyzed the data. MM and GM conceived the idea and wrote the manuscript. MTC and HW discussed results and revised the manuscript. GM supervised and directed the study. All authors read and approved the manuscript.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-2"><p id="p-45">Supplemental material available at FigShare: <a href="https://doi.org/10.25387/g3.9383543">https://doi.org/10.25387/g3.9383543</a>.</p></li><li class="fn" id="fn-3"><p id="p-46"><em>Communicating editor: J. Holland</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2019-05-13"><span class="received-label">Received </span>May 13, 2019.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2019-08-15"><span class="accepted-label">Accepted </span>August 15, 2019.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright Â© 2019 Momen <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">âµ</a><div class="cit ref-cit ref-book" id="cit-9.10.3369.1" data-doi="10.1007/978-1-4612-1694-0_16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Akaike</span>,  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">1974</span>â<span class="cit-article-title">A new look at the statistical model identification</span>, pp. <span class="cit-fpage">215</span>â<span class="cit-lpage">222</span> in <span class="cit-source">Selected Papers of Hirotugu Akaike</span>, <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">Berlin</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/978-1-4612-1694-0_16</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSelected%2BPapers%2Bof%2BHirotugu%2BAkaike%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-1-4612-1694-0_16%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-1-4612-1694-0_16&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.10.3369.2" data-doi="10.1111/nph.15656"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Arnold</span>,  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L. E.</span> <span class="cit-name-surname">Kruuk</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. B.</span> <span class="cit-name-surname">Nicotra</span></span></li></ol><cite>, <span class="cit-pub-date">2019</span>â<span class="cit-article-title">How to analyse plant phenotypic plasticity in response to a changing climate.</span> <abbr class="cit-jnl-abbrev">New Phytol.</abbr> <span class="cit-vol">222</span>: <span class="cit-fpage">1235</span>â<span class="cit-lpage">1241</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/nph.15656</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytol.%26rft.volume%253D222%26rft.spage%253D1235%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fnph.15656%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/nph.15656&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.10.3369.3" data-doi="10.1111/j.1439-0388.2010.00873.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Baldi</span>,  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Alencar</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">L. G.</span> <span class="cit-name-surname">Albuquerque</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title">Random regression analyses using b-splines functions to model growth from birth to adult age in canchim cattle.</span> <abbr class="cit-jnl-abbrev">J. Anim. Breed. Genet.</abbr> <span class="cit-vol">127</span>: <span class="cit-fpage">433</span>â<span class="cit-lpage">441</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1439-0388.2010.00873.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DJ%2BAnim%2BBreed%2BGenet%26rft.aulast%253DBaldi%26rft.auinit1%253DF.%26rft.volume%253D127%26rft.issue%253D6%26rft.spage%253D433%26rft.epage%253D441%26rft.atitle%253DRandom%2Bregression%2Banalyses%2Busing%2BB-splines%2Bfunctions%2Bto%2Bmodel%2Bgrowth%2Bfrom%2Bbirth%2Bto%2Badult%2Bage%2Bin%2BCanchim%2Bcattle.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1439-0388.2010.00873.x%26rft_id%253Dinfo%253Apmid%252F21077967%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1439-0388.2010.00873.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21077967&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3369.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.10.3369.4" data-doi="10.1086/521987"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Browning</span>,  <span class="cit-name-given-names">S. R.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. L.</span> <span class="cit-name-surname">Browning</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span>â<span class="cit-article-title">Rapid and accurate haplotype phasing and missing-data inference for whole-genome association studies by use of localized haplotype clustering.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">81</span>: <span class="cit-fpage">1084</span>â<span class="cit-lpage">1097</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1086/521987</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DBrowning%26rft.auinit1%253DS.%2BR.%26rft.volume%253D81%26rft.issue%253D5%26rft.spage%253D1084%26rft.epage%253D1097%26rft.atitle%253DRapid%2Band%2Baccurate%2Bhaplotype%2Bphasing%2Band%2Bmissing-data%2Binference%2Bfor%2Bwhole-genome%2Bassociation%2Bstudies%2Bby%2Buse%2Bof%2Blocalized%2Bhaplotype%2Bclustering.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F521987%26rft_id%253Dinfo%253Apmid%252F17924348%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/521987&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17924348&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3369.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000250480900018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.10.3369.5" data-doi="10.1104/pp.15.00450"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Campbell</span>,  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. C.</span> <span class="cit-name-surname">Knecht</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Berger</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. J.</span> <span class="cit-name-surname">Brien</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Wang</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span>â<span class="cit-article-title">Integrating image-based phenomics and association analysis to dissect the genetic architecture of temporal salinity responses in rice.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">168</span>: <span class="cit-fpage">1476</span>â<span class="cit-lpage">1489</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1104/pp.15.00450</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.15.00450%26rft_id%253Dinfo%253Apmid%252F26111541%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czoxMDoiMTY4LzQvMTQ3NiI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzkvMTAvMzM2OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.10.3369.6" data-doi="10.1002/pld3.80"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Campbell</span>,  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Walia</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Morota</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> Utilizing random regression models for genomic prediction of a longitudinal trait derived from high-throughput phenotyping. Plant Direct 2: 1-11<strong>:</strong><span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/pld3.80</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLANT%2BDIRECT%26rft.volume%253D2%26rft.spage%253D00080%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fpld3.80%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/pld3.80&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">âµ</a><div class="cit ref-cit ref-book" id="cit-9.10.3369.7" data-doi="10.3835/plantgenome2018.10.0075"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Campbell</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Momen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Walia</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Morota</span></span></li></ol><cite>, <span class="cit-pub-date">2019</span>â<span class="cit-article-title">Leveraging breeding values obtained from random regression models for genetic inference of longitudinal traits. Plant Genome</span> <span class="cit-vol">12</span>: 180075.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2018.10.0075</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLEVERAG%2BBREED%2BVALU%2BOBTAIN%2BFROM%2BRANDOM%2BREGRESS%2BMODEL%2BFOR%2BGENET%2BINFER%2BOF%2BLONGITUDIN%2BTRAIT%2BPLANT%2BGENOM%26rft.volume%253D12%26rft.atitle%253DLEVERAGING%2BBREEDING%2BVALUES%2BOBTAINED%2BFROM%2BRANDOM%2BREGRESSION%2BMODELS%2BFOR%2BGENETIC%2BINFERENCE%2BOF%2BLONGITUDINAL%2BTRAITS.%2BPLANT%2BGENOME%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2018.10.0075%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2018.10.0075&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.10.3369.8" data-doi="10.1198/108571106X154443"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cullis</span>,  <span class="cit-name-given-names">B. R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. B.</span> <span class="cit-name-surname">Smith</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">N. E.</span> <span class="cit-name-surname">Coombes</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span>â<span class="cit-article-title">On the design of early generation variety trials with correlated data.</span> <abbr class="cit-jnl-abbrev">J. Agric. Biol. Environ. Stat.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">381</span>â<span class="cit-lpage">393</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1198/108571106X154443</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAgric.%2BBiol.%2BEnviron.%2BStat.%26rft.volume%253D11%26rft.spage%253D381%26rft_id%253Dinfo%253Adoi%252F10.1198%252F108571106X154443%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1198/108571106X154443&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">âµ</a><div class="cit ref-cit ref-book" id="cit-9.10.3369.9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">De Boor</span>,  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span>â<span class="cit-source">A Practical Guide to Splines</span>, <span class="cit-vol">Vol. 27</span>, <span class="cit-edition">Revised Edition</span>. <span class="cit-publ-name">Springer-Verlag</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.10.3369.10" data-doi="10.1016/j.tplants.2011.09.005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Furbank</span>,  <span class="cit-name-given-names">R. T.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Tester</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span>â<span class="cit-article-title">Phenomics-technologies to relieve the phenotyping bottleneck.</span> <abbr class="cit-jnl-abbrev">Trends Plant Sci.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">635</span>â<span class="cit-lpage">644</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.tplants.2011.09.005</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bplant%2Bscience%26rft.stitle%253DTrends%2BPlant%2BSci%26rft.aulast%253DFurbank%26rft.auinit1%253DR.%2BT.%26rft.volume%253D16%26rft.issue%253D12%26rft.spage%253D635%26rft.epage%253D644%26rft.atitle%253DPhenomics--technologies%2Bto%2Brelieve%2Bthe%2Bphenotyping%2Bbottleneck.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tplants.2011.09.005%26rft_id%253Dinfo%253Apmid%252F22074787%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tplants.2011.09.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22074787&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3369.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000298724900001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.10.3369.11" data-doi="10.1016/j.compag.2016.07.028"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ge</span>,  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Bai</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Stoerger</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. C.</span> <span class="cit-name-surname">Schnable</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span>â<span class="cit-article-title">Temporal dynamics of maize plant growth, water use, and leaf water content using automated high throughput rgb and hyperspectral imaging.</span> <abbr class="cit-jnl-abbrev">Comput. Electron. Agric.</abbr> <span class="cit-vol">127</span>: <span class="cit-fpage">625</span>â<span class="cit-lpage">632</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.compag.2016.07.028</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DComput.%2BElectron.%2BAgric.%26rft.volume%253D127%26rft.spage%253D625%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.compag.2016.07.028%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.compag.2016.07.028&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.10.3369.12" data-doi="10.1186/1746-4811-7-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Golzarian</span>,  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. A.</span> <span class="cit-name-surname">Frick</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Rajendran</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Berger</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Roy</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span>â<span class="cit-article-title">Accurate inference of shoot biomass from high-throughput images of cereal plants.</span> <abbr class="cit-jnl-abbrev">Plant Methods</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">2</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1746-4811-7-2</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPlant%2BMethods%26rft.aulast%253DGolzarian%26rft.auinit1%253DM.%2BR.%26rft.volume%253D7%26rft.spage%253D2%26rft.epage%253D2%26rft.atitle%253DAccurate%2Binference%2Bof%2Bshoot%2Bbiomass%2Bfrom%2Bhigh-throughput%2Bimages%2Bof%2Bcereal%2Bplants.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1746-4811-7-2%26rft_id%253Dinfo%253Apmid%252F21284859%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1746-4811-7-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21284859&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3369.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.10.3369.13" data-doi="10.2527/jas1976.4361188x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Henderson</span>,  <span class="cit-name-given-names">C.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Quaas</span></span></li></ol><cite>, <span class="cit-pub-date">1976</span>â<span class="cit-article-title">Multiple trait evaluation using relativesâ records.</span> <abbr class="cit-jnl-abbrev">J. Anim. Sci.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">1188</span>â<span class="cit-lpage">1197</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2527/jas1976.4361188x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BAnimal%2BScience%26rft.stitle%253DJ%2BANIM%2BSCI%26rft.aulast%253DHenderson%26rft.auinit1%253DC.%2BR.%26rft.volume%253D43%26rft.issue%253D6%26rft.spage%253D1188%26rft.epage%253D1197%26rft.atitle%253DMultiple%2BTrait%2BEvaluation%2BUsing%2BRelatives%2527%2BRecords%26rft_id%253Dinfo%253Adoi%252F10.2527%252Fjas1976.4361188x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2527/jas1976.4361188x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1976CP41700009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.10.3369.14" data-doi="10.1186/s12863-015-0218-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Howard</span>,  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Jiao</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Tiezzi</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Huang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. A.</span> <span class="cit-name-surname">Gray</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span>â<span class="cit-article-title">Genome-wide association study on legendre random regression coefficients for the growth and feed intake trajectory on Duroc Boars.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">59</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12863-015-0218-8</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D16%26rft.spage%253D59%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12863-015-0218-8%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12863-015-0218-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.10.3369.15" data-doi="10.2527/2005.834757x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Iwaisaki</span>,  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Tsuruta</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Misztal</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Bertrand</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span>â<span class="cit-article-title">Genetic parameters estimated with multitrait and linear spline-random regression models using gelbvieh early growth data.</span> <abbr class="cit-jnl-abbrev">J. Anim. Sci.</abbr> <span class="cit-vol">83</span>: <span class="cit-fpage">757</span>â<span class="cit-lpage">763</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2527/2005.834757x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BAnimal%2BScience%26rft.stitle%253DJ%2BANIM%2BSCI%26rft.aulast%253DIwaisaki%26rft.auinit1%253DH.%26rft.volume%253D83%26rft.issue%253D4%26rft.spage%253D757%26rft.epage%253D763%26rft.atitle%253DGenetic%2Bparameters%2Bestimated%2Bwith%2Bmultitrait%2Band%2Blinear%2Bspline-random%2Bregression%2Bmodels%2Busing%2BGelbvieh%2Bearly%2Bgrowth%2Bdata%26rft_id%253Dinfo%253Adoi%252F10.2527%252F2005.834757x%26rft_id%253Dinfo%253Apmid%252F15753329%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2527/2005.834757x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15753329&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3369.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.10.3369.16" data-doi="10.3168/jds.S0022-0302(97)75996-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jamrozik</span>,  <span class="cit-name-given-names">J.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Schaeffer</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span>â<span class="cit-article-title">Estimates of genetic parameters for a test day model with random regressions for yield traits of first lactation holsteins.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">80</span>: <span class="cit-fpage">762</span>â<span class="cit-lpage">770</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3168/jds.S0022-0302(97)75996-4</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DJamrozik%26rft.auinit1%253DJ.%26rft.volume%253D80%26rft.issue%253D4%26rft.spage%253D762%26rft.epage%253D770%26rft.atitle%253DEstimates%2Bof%2BGenetic%2BParameters%2Bfor%2Ba%2BTest%2BDay%2BModel%2Bwith%2BRandom%2BRegressions%2Bfor%2BYield%2BTraits%2Bof%2BFirst%2BLactation%2BHolsteins%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.S0022-0302%252897%252975996-4%26rft_id%253Dinfo%253Apmid%252F9149971%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.S0022-0302(97)75996-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9149971&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3369.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1997WW56600020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.10.3369.17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kirkpatrick</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Lofsvold</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Bulmer</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span>â<span class="cit-article-title">Analysis of the inheritance, selection and evolution of growth trajectories.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">124</span>: <span class="cit-fpage">979</span>â<span class="cit-lpage">993</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D124%26rft.spage%253D979%26rft_id%253Dinfo%253Apmid%252F2323560%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=2323560&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3369.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.10.3369.18" data-doi="10.1093/jxb/erw176"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Knecht</span>,  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. T.</span> <span class="cit-name-surname">Campbell</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Caprez</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. R.</span> <span class="cit-name-surname">Swanson</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Walia</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span>â<span class="cit-article-title">Image Harvest: an open-source platform for high-throughput plant image processing and analysis.</span> <abbr class="cit-jnl-abbrev">J. Exp. Bot.</abbr> <span class="cit-vol">67</span>: <span class="cit-fpage">3587</span>â<span class="cit-lpage">3599</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/jxb/erw176</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BExp.%2BBot.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjxb%252Ferw176%26rft_id%253Dinfo%253Apmid%252F27141917%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/jxb/erw176&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27141917&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3369.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.10.3369.19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Marchal</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. D.</span> <span class="cit-name-surname">Schlichting</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Gobin</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Balandier</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Millier</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2019</span>â<span class="cit-article-title">Deciphering hybrid larch reaction norms using random regression. G3: Genes, Genomes</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">21</span>â<span class="cit-lpage">32</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D9%26rft.spage%253D21%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.10.3369.20" data-doi="10.1051/gse:19980302"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meyer</span>,  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span>â<span class="cit-article-title">Estimating covariance functions for longitudinal data using a random regression model.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">221</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1297-9686-30-3-221</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D30%26rft.spage%253D221%26rft_id%253Dinfo%253Adoi%252F10.1051%252Fgse%253A19980302%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1051/gse:19980302&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000076236100002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.10.3369.21" data-doi="10.1051/gse:2005012"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meyer</span>,  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span>â<span class="cit-article-title">Random regression analyses using B-splines to model growth of australian angus cattle.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">473</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1297-9686-37-6-473</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DMeyer%26rft.auinit1%253DK.%26rft.volume%253D37%26rft.issue%253D5%26rft.spage%253D473%26rft.epage%253D500%26rft.atitle%253DRandom%2Bregression%2Banalyses%2Busing%2BB-splines%2Bto%2Bmodel%2Bgrowth%2Bof%2BAustralian%2BAngus%2Bcattle.%26rft_id%253Dinfo%253Adoi%252F10.1051%252Fgse%253A2005012%26rft_id%253Dinfo%253Apmid%252F16093011%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1051/gse:2005012&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16093011&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3369.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000231207200001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.10.3369.22" data-doi="10.1631/jzus.2007.B0815"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meyer</span>,  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span>â<span class="cit-article-title">Wombat - A tool for mixed model analyses in quantitative genetics by restricted maximum likelihood (reml).</span> <abbr class="cit-jnl-abbrev">J. Zhejiang Univ. Sci. B</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">815</span>â<span class="cit-lpage">821</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1631/jzus.2007.B0815</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DJ%2BZhejiang%2BUniv%2BSci%2BB%26rft.aulast%253DMeyer%26rft.auinit1%253DK.%26rft.volume%253D8%26rft.issue%253D11%26rft.spage%253D815%26rft.epage%253D821%26rft.atitle%253DWOMBAT%253A%2Ba%2Btool%2Bfor%2Bmixed%2Bmodel%2Banalyses%2Bin%2Bquantitative%2Bgenetics%2Bby%2Brestricted%2Bmaximum%2Blikelihood%2B%2528REML%2529.%26rft_id%253Dinfo%253Adoi%252F10.1631%252Fjzus.2007.B0815%26rft_id%253Dinfo%253Apmid%252F17973343%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1631/jzus.2007.B0815&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17973343&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3369.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.10.3369.23" data-doi="10.1016/S0301-6226(96)01414-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meyer</span>,  <span class="cit-name-given-names">K.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">W. B.</span> <span class="cit-name-surname">Hill</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span>â<span class="cit-article-title">Estimation of genetic and phenotypic covariance functions for longitudinal or repeated records by restricted maximum likelihood.</span> <abbr class="cit-jnl-abbrev">Livest. Prod. Sci.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">185</span>â<span class="cit-lpage">200</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/S0301-6226(96)01414-5</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLivest.%2BProd.%2BSci.%26rft.volume%253D47%26rft.spage%253D185%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0301-6226%252896%252901414-5%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0301-6226(96)01414-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.10.3369.24" data-doi="10.1098/rstb.2005.1681"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meyer</span>,  <span class="cit-name-given-names">K.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Kirkpatrick</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span>â<span class="cit-article-title">Up hill, down dale: quantitative genetics of curvaceous traits.</span> <abbr class="cit-jnl-abbrev">Philos. Trans. R. Soc. Lond. B Biol. Sci.</abbr> <span class="cit-vol">360</span>: <span class="cit-fpage">1443</span>â<span class="cit-lpage">1455</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1098/rstb.2005.1681</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhilosophical%2BTransactions%2Bof%2Bthe%2BRoyal%2BSociety%2BB%253A%2BBiological%2BSciences%26rft.stitle%253DPhil%2BTrans%2BR%2BSoc%2BB%26rft.aulast%253DMeyer%26rft.auinit1%253DK.%26rft.volume%253D360%26rft.issue%253D1459%26rft.spage%253D1443%26rft.epage%253D1455%26rft.atitle%253DUp%2Bhill%252C%2Bdown%2Bdale%253A%2Bquantitative%2Bgenetics%2Bof%2Bcurvaceous%2Btraits%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frstb.2005.1681%26rft_id%253Dinfo%253Apmid%252F16048787%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1098/rstb.2005.1681&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16048787&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3369.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.10.3369.25" data-doi="10.1111/j.1439-0388.2006.00582.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Misztal</span>,  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span>â<span class="cit-article-title">Properties of random regression models using linear splines.</span> <abbr class="cit-jnl-abbrev">J. Anim. Breed. Genet.</abbr> <span class="cit-vol">123</span>: <span class="cit-fpage">74</span>â<span class="cit-lpage">80</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1439-0388.2006.00582.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DJ%2BAnim%2BBreed%2BGenet%26rft.aulast%253DMisztal%26rft.auinit1%253DI.%26rft.volume%253D123%26rft.issue%253D2%26rft.spage%253D74%26rft.epage%253D80%26rft.atitle%253DProperties%2Bof%2Brandom%2Bregression%2Bmodels%2Busing%2Blinear%2Bsplines.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1439-0388.2006.00582.x%26rft_id%253Dinfo%253Apmid%252F16533360%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1439-0388.2006.00582.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16533360&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3369.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">âµ</a><div class="cit ref-cit ref-confproc" id="cit-9.10.3369.26"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Misztal</span>,  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Tsuruta</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Strabel</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Auvray</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Druet</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> Blupf90 and related programs (bgf90). In <em>Proceedings of the 7th World Congress on Genetics Applied to Livestock Production</em>, <span class="cit-vol">volume 33</span>. <span class="cit-fpage">743</span>â<span class="cit-lpage">744</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">âµ</a><div class="cit ref-cit ref-book" id="cit-9.10.3369.27" data-doi="10.1079/9781780643915.0000"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mrode</span>,  <span class="cit-name-given-names">R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span>â<span class="cit-source">Linear models for the prediction of animal breeding values</span>, <span class="cit-publ-name">CABI</span>, <span class="cit-publ-loc">Oxfordshire, UK</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1079/9781780643915.0000</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLinear%2Bmodels%2Bfor%2Bthe%2Bprediction%2Bof%2Banimal%2Bbreeding%2Bvalues%26rft_id%253Dinfo%253Adoi%252F10.1079%252F9781780643915.0000%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1079/9781780643915.0000&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.10.3369.28" data-doi="10.1086/519795"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Purcell</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Neale</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Todd-Brown</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Thomas</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. A.</span> <span class="cit-name-surname">Ferreira</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span>â<span class="cit-article-title">Plink: a tool set for whole-genome association and population-based linkage analyses.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">81</span>: <span class="cit-fpage">559</span>â<span class="cit-lpage">575</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1086/519795</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DPurcell%26rft.auinit1%253DS.%26rft.volume%253D81%26rft.issue%253D3%26rft.spage%253D559%26rft.epage%253D575%26rft.atitle%253DPLINK%253A%2Ba%2Btool%2Bset%2Bfor%2Bwhole-genome%2Bassociation%2Band%2Bpopulation-based%2Blinkage%2Banalyses.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F519795%26rft_id%253Dinfo%253Apmid%252F17701901%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/519795&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17701901&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3369.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">âµ</a><div class="cit ref-cit ref-web" id="cit-9.10.3369.29"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schaeffer</span>,  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <em>Random regression models</em>. Available in <a href="http://animalbiosciences.uoguelph.ca/">http://animalbiosciences. uoguelph. ca/</a><sup>â¼</sup> lrs/BOOKS/rrmbook. pdf.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.10.3369.30" data-doi="10.1111/j.1439-0388.2008.00748.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schaeffer</span>,  <span class="cit-name-given-names">L.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Jamrozik</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span>â<span class="cit-article-title">Random regression models: a longitudinal perspective.</span> <abbr class="cit-jnl-abbrev">J. Anim. Breed. Genet.</abbr> <span class="cit-vol">125</span>: <span class="cit-fpage">145</span>â<span class="cit-lpage">146</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1439-0388.2008.00748.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DJ%2BAnim%2BBreed%2BGenet%26rft.aulast%253DSchaeffer%26rft.auinit1%253DL.%2BR.%26rft.volume%253D125%26rft.issue%253D3%26rft.spage%253D145%26rft.epage%253D146%26rft.atitle%253DRandom%2Bregression%2Bmodels%253A%2Ba%2Blongitudinal%2Bperspective.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1439-0388.2008.00748.x%26rft_id%253Dinfo%253Apmid%252F18479264%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1439-0388.2008.00748.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18479264&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3369.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.10.3369.31" data-doi="10.1016/S0301-6226(03)00151-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schaeffer</span>,  <span class="cit-name-given-names">L. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span>â<span class="cit-article-title">Application of random regression models in animal breeding.</span> <abbr class="cit-jnl-abbrev">Livest. Prod. Sci.</abbr> <span class="cit-vol">86</span>: <span class="cit-fpage">35</span>â<span class="cit-lpage">45</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/S0301-6226(03)00151-9</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLivest.%2BProd.%2BSci.%26rft.volume%253D86%26rft.spage%253D35%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0301-6226%252803%252900151-9%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0301-6226(03)00151-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.10.3369.32" data-doi="10.1214/aos/1176344136"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schwarz</span>,  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">1978</span>â<span class="cit-article-title">Estimating the dimension of a model.</span> <abbr class="cit-jnl-abbrev">Ann. Stat.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">461</span>â<span class="cit-lpage">464</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1214/aos/1176344136</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BStat.%26rft.volume%253D6%26rft.spage%253D461%26rft_id%253Dinfo%253Adoi%252F10.1214%252Faos%252F1176344136%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1214/aos/1176344136&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1978EQ63300014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.10.3369.33" data-doi="10.3835/plantgenome2016.11.0111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. E.</span> <span class="cit-name-surname">Rutkoski</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. A.</span> <span class="cit-name-surname">Poland</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Crossa</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.-L.</span> <span class="cit-name-surname">Jannink</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span>â<span class="cit-article-title">Multitrait, random regression, or simple repeatability model in high-throughput phenotyping data improve genomic prediction for wheat grain yield.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">10</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2016.11.0111</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D18%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2016.11.0111%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2016.11.0111&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.10.3369.34" data-doi="10.1126/science.1183700"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tester</span>,  <span class="cit-name-given-names">M.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Langridge</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title">Breeding technologies to increase crop production in a changing world.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">327</span>: <span class="cit-fpage">818</span>â<span class="cit-lpage">822</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.1183700</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DTester%26rft.auinit1%253DM.%26rft.volume%253D327%26rft.issue%253D5967%26rft.spage%253D818%26rft.epage%253D822%26rft.atitle%253DBreeding%2BTechnologies%2Bto%2BIncrease%2BCrop%2BProduction%2Bin%2Ba%2BChanging%2BWorld%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1183700%26rft_id%253Dinfo%253Apmid%252F20150489%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjcvNTk2Ny84MTgiO3M6NDoiYXRvbSI7czoxOToiL2dnZy85LzEwLzMzNjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.10.3369.35" data-doi="10.3168/jds.S0022-0302(98)75895-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Van der Werf</span>,  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. E.</span> <span class="cit-name-surname">Goddard</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Meyer</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span>â<span class="cit-article-title">The use of covariance functions and random regressions for genetic evaluation of milk production based on test day records.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">81</span>: <span class="cit-fpage">3300</span>â<span class="cit-lpage">3308</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3168/jds.S0022-0302(98)75895-3</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DVan%2BDer%2BWerf%26rft.auinit1%253DJ.H.J.%26rft.volume%253D81%26rft.issue%253D12%26rft.spage%253D3300%26rft.epage%253D3308%26rft.atitle%253DThe%2BUse%2Bof%2BCovariance%2BFunctions%2Band%2BRandom%2BRegressions%2Bfor%2BGenetic%2BEvaluation%2Bof%2BMilk%2BProduction%2BBased%2Bon%2BTest%2BDay%2BRecords%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.S0022-0302%252898%252975895-3%26rft_id%253Dinfo%253Apmid%252F9891276%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.S0022-0302(98)75895-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9891276&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3369.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000077836700022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.10.3369.36" data-doi="10.3168/jds.2007-0980"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">VanRaden</span>,  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span>â<span class="cit-article-title">Efficient methods to compute genomic predictions.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">4414</span>â<span class="cit-lpage">4423</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3168/jds.2007-0980</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DVanRaden%26rft.auinit1%253DP.%2BM.%26rft.volume%253D91%26rft.issue%253D11%26rft.spage%253D4414%26rft.epage%253D4423%26rft.atitle%253DEfficient%2BMethods%2Bto%2BCompute%2BGenomic%2BPredictions%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2007-0980%26rft_id%253Dinfo%253Apmid%252F18946147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2007-0980&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18946147&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3369.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000260277200035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.10.3369.37" data-doi="10.3168/jds.S0022-0302(99)75277-X"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">White</span>,  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Thompson</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Brotherstone</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span>â<span class="cit-article-title">Genetic and environmental smoothing of lactation curves with cubic splines.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">82</span>: <span class="cit-fpage">632</span>â<span class="cit-lpage">638</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3168/jds.S0022-0302(99)75277-X</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DWhite%26rft.auinit1%253DI.M.S.%26rft.volume%253D82%26rft.issue%253D3%26rft.spage%253D632%26rft.epage%253D638%26rft.atitle%253DGenetic%2Band%2BEnvironmental%2BSmoothing%2Bof%2BLactation%2BCurves%2Bwith%2BCubic%2BSplines%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.S0022-0302%252899%252975277-X%26rft_id%253Dinfo%253Apmid%252F10194684%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.S0022-0302(99)75277-X&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10194684&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3369.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000079242900026&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.10.3369.38" data-doi="10.1038/ncomms1467"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>,  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.-W.</span> <span class="cit-name-surname">Tung</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G. C.</span> <span class="cit-name-surname">Eizenga</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. H.</span> <span class="cit-name-surname">Wright</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. L.</span> <span class="cit-name-surname">Ali</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span>â<span class="cit-article-title">Genome-wide association mapping reveals a rich genetic architecture of complex traits in <em>Oryza sativa</em>.</span> <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">467</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ncomms1467</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BCommun%26rft.aulast%253DZhao%26rft.auinit1%253DK.%26rft.volume%253D2%26rft.spage%253D467%26rft.epage%253D467%26rft.atitle%253DGenome-wide%2Bassociation%2Bmapping%2Breveals%2Ba%2Brich%2Bgenetic%2Barchitecture%2Bof%2Bcomplex%2Btraits%2Bin%2BOryza%2Bsativa.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms1467%26rft_id%253Dinfo%253Apmid%252F21915109%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms1467&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21915109&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3369.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/9/10/3369.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/10/3359" title="Exon 3 of the NUMB Gene Emerged in the Chordate Lineage Coopting the NUMB Protein to the Regulation of MDM2" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/10/3381" title="A Bayesian Genomic Multi-output Regressor Stacking Model for Predicting Multi-trait Multi-environment Plant Breeding Data" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/9/10"><span>Volume 9 Issue 10, October 2019</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/9/10.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/9/10.cover-source.jpg?itok=yA8kifEd" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 9 (10)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/10/3369" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-8U6rtgK1CQHoqPVg1UNi6Xz5c4rbls-ofiQUVs2XH1w" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/genomic-prediction" class="" data-icon-position="" data-hide-link-title="0">Genomic Prediction</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/9/10/3369" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/9/10/3369" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/9/10/3369" class="active" data-icon-position="" data-hide-link-title="0">Predicting Longitudinal Traits Derived from High-Throughput Phenomics in Contrasting Environments Using Genomic Legendre Polynomials and B-Splines</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/494296" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-ET2Gx-Z1fgcKzMENTezJdILSP-vjFT79YBddyV0WfqA" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/10/3369.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/9/10/3369" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-2-B1I42LcydD8I-GrSC0-ITo1W1iDGwZhwcPd5-vRNI" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/9/10/3369" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/10/3369.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494296" id="node494296--2" data-pisa="ggg;9/10/3369" data-pisa-master="ggg;g3.119.400346" data-apath="/ggg/9/10/3369.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Predicting Longitudinal Traits Derived from High-Throughput Phenomics in Contrasting Environments Using Genomic Legendre Polynomials and B-Splines</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-2562-2741" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Mehdi Momen</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-8257-3595" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Malachy T. Campbell</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-9712-5824" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Harkamal Walia</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-3567-6911" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Gota Morota</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3369-3380;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400346 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494296--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Mehdi Momen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Animal and Poultry Sciences, Virginia Polytechnic Institute and State University, Blacksburg, VA, 24061 and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mehdi%2BMomen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Momen%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMehdi%2BMomen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-2562-2741" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Mehdi Momen</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Malachy T. Campbell </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Animal and Poultry Sciences, Virginia Polytechnic Institute and State University, Blacksburg, VA, 24061 and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Malachy%2BT.%2BCampbell%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Campbell%20MT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMalachy%2BT.%2BCampbell%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-8257-3595" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Malachy T. Campbell</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Harkamal Walia </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>Department of Agronomy and Horticulture, University of Nebraska-Lincoln, Lincoln, NE, 68583</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Harkamal%2BWalia%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Walia%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AHarkamal%2BWalia%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-9712-5824" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Harkamal Walia</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Gota Morota </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Animal and Poultry Sciences, Virginia Polytechnic Institute and State University, Blacksburg, VA, 24061 and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gota%2BMorota%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Morota%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGota%2BMorota%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-3567-6911" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Gota Morota</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:morota@vt.edu" class="" data-icon-position="" data-hide-link-title="0">morota@vt.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/9/10/3369" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/9/10/3369&amp;title=Predicting%20Longitudinal%20Traits%20Derived%20from%20High-Throughput%20Phenomics%20in%20Contrasting%20Environments%20Using%20Genomic%20Legendre%20Polynomials%20and%20B-Splines" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/9/10/3369&amp;title=Predicting%20Longitudinal%20Traits%20Derived%20from%20High-Throughput%20Phenomics%20in%20Contrasting%20Environments%20Using%20Genomic%20Legendre%20Polynomials%20and%20B-Splines" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/9/10/3369&amp;title=Predicting%20Longitudinal%20Traits%20Derived%20from%20High-Throughput%20Phenomics%20in%20Contrasting%20Environments%20Using%20Genomic%20Legendre%20Polynomials%20and%20B-Splines" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/9/10/3369&amp;text=Predicting%20Longitudinal%20Traits%20Derived%20from%20High-Throughput%20Phenomics%20in%20Contrasting%20Environments%20Using%20Genomic%20Legendre%20Polynomials%20and%20B-Splines" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/9/10/3369&amp;title=Predicting%20Longitudinal%20Traits%20Derived%20from%20High-Throughput%20Phenomics%20in%20Contrasting%20Environments%20Using%20Genomic%20Legendre%20Polynomials%20and%20B-Splines" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/9/10/3369&amp;t=Predicting%20Longitudinal%20Traits%20Derived%20from%20High-Throughput%20Phenomics%20in%20Contrasting%20Environments%20Using%20Genomic%20Legendre%20Polynomials%20and%20B-Splines" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/9/10/3369&amp;title=Predicting%20Longitudinal%20Traits%20Derived%20from%20High-Throughput%20Phenomics%20in%20Contrasting%20Environments%20Using%20Genomic%20Legendre%20Polynomials%20and%20B-Splines" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/9/10/3369&amp;title=Predicting%20Longitudinal%20Traits%20Derived%20from%20High-Throughput%20Phenomics%20in%20Contrasting%20Environments%20Using%20Genomic%20Legendre%20Polynomials%20and%20B-Splines" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/494296/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494296" id="node494296--3" data-pisa="ggg;9/10/3369" data-pisa-master="ggg;g3.119.400346" data-apath="/ggg/9/10/3369.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Predicting Longitudinal Traits Derived from High-Throughput Phenomics in Contrasting Environments Using Genomic Legendre Polynomials and B-Splines</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-2562-2741" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Mehdi Momen</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-8257-3595" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Malachy T. Campbell</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-9712-5824" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Harkamal Walia</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-3567-6911" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Gota Morota</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3369-3380;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400346 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494296--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Mehdi Momen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Animal and Poultry Sciences, Virginia Polytechnic Institute and State University, Blacksburg, VA, 24061 and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mehdi%2BMomen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Momen%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMehdi%2BMomen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-2562-2741" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Mehdi Momen</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Malachy T. Campbell </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Animal and Poultry Sciences, Virginia Polytechnic Institute and State University, Blacksburg, VA, 24061 and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Malachy%2BT.%2BCampbell%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Campbell%20MT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMalachy%2BT.%2BCampbell%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-8257-3595" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Malachy T. Campbell</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Harkamal Walia </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>Department of Agronomy and Horticulture, University of Nebraska-Lincoln, Lincoln, NE, 68583</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Harkamal%2BWalia%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Walia%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AHarkamal%2BWalia%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-9712-5824" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Harkamal Walia</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Gota Morota </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Animal and Poultry Sciences, Virginia Polytechnic Institute and State University, Blacksburg, VA, 24061 and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gota%2BMorota%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Morota%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGota%2BMorota%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-3567-6911" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Gota Morota</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:morota@vt.edu" class="" data-icon-position="" data-hide-link-title="0">morota@vt.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=3369&amp;rft.epage=3380&amp;rft.atitle=Predicting%20Longitudinal%20Traits%20Derived%20from%20High-Throughput%20Phenomics%20in%20Contrasting%20Environments%20Using%20Genomic%20Legendre%20Polynomials%20and%20B-Splines&amp;rft.volume=9&amp;rft.issue=10&amp;rft.date=2019-10-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Momen%2C+Mehdi&amp;rft.au=Campbell%2C+Malachy+T.&amp;rft.au=Walia%2C+Harkamal&amp;rft.au=Morota%2C+Gota"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/494296/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/494296/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/494296/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/494296/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/494296/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/494296/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/494296/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/494296/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/494296/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/494296/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/494296/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/494296/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495864" id="node495864" data-pisa="ggg;10/3/1113" data-pisa-master="ggg;g3.119.400968" data-apath="/ggg/10/3/1113.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1113" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Multi-trait Genomic Prediction Model Increased the Predictive Ability for Agronomic and Malting Quality Traits in Barley (<em>Hordeum vulgare</em> L.)</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495543" id="node495543" data-pisa="ggg;10/2/769" data-pisa-master="ggg;g3.119.400759" data-apath="/ggg/10/2/769.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/2/769" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Novel Bayesian Networks for Genomic Prediction of Developmental Traits in Biomass Sorghum</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495557" id="node495557" data-pisa="ggg;10/2/539" data-pisa-master="ggg;g3.119.400866" data-apath="/ggg/10/2/539.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/2/539" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Optimized Genetic Testing for Polledness in Multiple Breeds of Cattle</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Genomic%20Prediction" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Genomic Prediction</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/9/10/3369" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Background</a></li><li class="odd"><a href="#sec-2" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="even"><a href="#sec-16" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="odd"><a href="#sec-19" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/9/10/3369.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/9/10/3369.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/9/10/3369.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
