<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/9/10/3333" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/9/10/3333.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/9/10/3333.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/9/10/3333.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/9/10/3333/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/9/10/3333.atom" />
<meta name="HW.pisa" content="ggg;9/10/3333" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Genomic Prediction Including SNP-Specific Variance Predictors" />
<meta name="DC.Identifier" content="10.1534/g3.119.400381" />
<meta name="DC.Date" content="2019-10-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2019 Mouresan et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="The increasing amount of available biological information on the markers can be used to inform the models applied for genomic selection to improve predictions. The objective of this study was to propose a general model for genomic selection using a link function approach within the hierarchical generalized linear model framework (hglm) that can include external information on the markers. These models can be fitted using the well-established hglm package in R. We also present an R package (CodataGS) to fit these models, which is significantly faster than the hglm package. Simulated data were used to validate the proposed model. We tested categorical, continuous and combination models where the external information on the markers was related to 1) the location of the QTL on the genome with varying degree of uncertainty, 2) the relationship of the markers with the QTL calculated as the LD between them, and 3) a combination of both. The proposed models showed improved accuracies from 3.8% up to 23.2% compared to the SNP-BLUP method in a simulated population derived from a base population with 100 individuals. Moreover, the proposed categorical model was tested on a dairy cattle dataset for two traits (Milk Yield and Fat Percentage). These results also showed improved accuracy compared to SNP-BLUP, especially for the Fat% trait. The performance of the proposed models depended on the genetic architecture of the trait, as traits that deviate from the infinitesimal model benefited more from the external information. Also, the gain in accuracy depended on the degree of uncertainty of the external information provided to the model. The usefulness of these type of models is expected to increase with time as more accurate information on the markers becomes available." />
<meta name="DC.Contributor" content="Elena Flavia Mouresan" />
<meta name="DC.Contributor" content="Maria Selle" />
<meta name="DC.Contributor" content="Lars Rönnegård" />
<meta name="article:published_time" content="2019-10-01" />
<meta name="article:section" content="Genomic Prediction" />
<meta name="citation_title" content="Genomic Prediction Including SNP-Specific Variance Predictors" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;The increasing amount of available biological information on the markers can be used to inform the models applied for genomic selection to improve predictions. The objective of this study was to propose a general model for genomic selection using a link function approach within the hierarchical generalized linear model framework (hglm) that can include external information on the markers. These models can be fitted using the well-established hglm package in R. We also present an R package (CodataGS) to fit these models, which is significantly faster than the hglm package. Simulated data were used to validate the proposed model. We tested categorical, continuous and combination models where the external information on the markers was related to 1) the location of the QTL on the genome with varying degree of uncertainty, 2) the relationship of the markers with the QTL calculated as the LD between them, and 3) a combination of both. The proposed models showed improved accuracies from 3.8% up to 23.2% compared to the SNP-BLUP method in a simulated population derived from a base population with 100 individuals. Moreover, the proposed categorical model was tested on a dairy cattle dataset for two traits (Milk Yield and Fat Percentage). These results also showed improved accuracy compared to SNP-BLUP, especially for the Fat% trait. The performance of the proposed models depended on the genetic architecture of the trait, as traits that deviate from the infinitesimal model benefited more from the external information. Also, the gain in accuracy depended on the degree of uncertainty of the external information provided to the model. The usefulness of these type of models is expected to increase with time as more accurate information on the markers becomes available.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2019/10/01" />
<meta name="citation_mjid" content="ggg;9/10/3333" />
<meta name="citation_id" content="9/10/3333" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/9/10/3333" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/9/10/3333.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/9/10/3333.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/9/10/3333.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.119.400381" />
<meta name="citation_pmid" content="31467030" />
<meta name="citation_volume" content="9" />
<meta name="citation_issue" content="10" />
<meta name="citation_num_pages" content="11" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Genomic Prediction" />
<meta name="citation_firstpage" content="3333" />
<meta name="citation_lastpage" content="3343" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Elena Flavia Mouresan" />
<meta name="citation_author_institution" content="Department of Animal Breeding and Genetics, Swedish University of Agricultural Sciences, Sweden, 75007," />
<meta name="citation_author_email" content="elena.flavia.mouresan@slu.se" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-1335-7610" />
<meta name="citation_author" content="Maria Selle" />
<meta name="citation_author_institution" content="Department of Mathematical Sciences, Norwegian University of Science and Technology, Norway, 7491, and" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-2062-3235" />
<meta name="citation_author" content="Lars Rönnegård" />
<meta name="citation_author_institution" content="Department of Animal Breeding and Genetics, Swedish University of Agricultural Sciences, Sweden, 75007," />
<meta name="citation_author_institution" content="School of Technology and Business Studies, Dalarna University, Sweden, 79188" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-1057-5401" />
<meta name="citation_reference" content="citation_journal_title=Genet Sel Evol: GSE;citation_author=R. Abdollahi-Arpanahi;citation_author=G. Morota;citation_author=B. D. Valente;citation_author=A. Kranis;citation_author=G. J. M. Rosa;citation_title=Differential contribution of genomic regions to marked genetic variation and prediction of quantitative traits in broiler chickens.;citation_pages=10;citation_volume=48;citation_year=2016;citation_doi=10.1186/s12711-016-0187-z" />
<meta name="citation_reference" content="citation_journal_title=J. R. Stat. Soc. Ser. C Appl. Stat.;citation_author=M. Aitkin;citation_title=Modelling variance heterogeneity in normal regression using GLIM.;citation_pages=332-339;citation_volume=36;citation_year=1987" />
<meta name="citation_reference" content="citation_journal_title=PLOS Comput. Biol.;citation_author=W. S. Bush;citation_author=J. H. Moore;citation_title=Chapter 11: Genome-Wide Association Studies.;citation_pages=e1002822;citation_volume=8;citation_year=2012;citation_pmid=23300413;citation_doi=10.1371/journal.pcbi.1002822" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. Carneiro;citation_author=N. Ferrand;citation_author=M. W. Nachman;citation_title=Recombination and Speciation: Loci near Centromeres Are More Differentiated than Loci near Telomeres between Subspecies of the European Rabbit (Oryctolagus Cuniculus).;citation_volume=181;citation_year=2009;citation_pmid=19015539;citation_doi=10.1534/genetics.108.096826" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=R. Caspi;citation_author=T. Altman;citation_author=K. Dreher;citation_author=C. A. Fulcher;citation_author=P. Subhraveti;citation_title=The MetaCyc database of metabolic pathways and enzymes and the BioCyc collection of pathway/genome databases.;citation_volume=40;citation_year=2012;citation_pmid=22102576;citation_doi=10.1093/nar/gkr1014" />
<meta name="citation_reference" content="citation_year=2011" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=H. D. Daetwyler;citation_author=A. Capitan;citation_author=H. Pausch;citation_author=P. Stothard;citation_author=R. van Binsbergen;citation_title=Whole-genome sequencing of 234 bulls facilitates mapping of monogenic and complex traits in cattle.;citation_pages=858-865;citation_volume=46;citation_year=2014;citation_pmid=25017103;citation_doi=10.1038/ng.3034" />
<meta name="citation_reference" content="citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=H. D. Daetwyler;citation_author=R. Pong-Wong;citation_author=B. Villanueva;citation_author=J. A. Woolliams;citation_title=The impact of genetic architecture on genome-wide evaluation methods.;citation_volume=185;citation_year=2010;citation_pmid=20407128;citation_doi=10.1534/genetics.110.116855" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=G. de los Campos;citation_author=D. Sorensen;citation_author=D. Gianola;citation_title=Genomic heritability: What is it?;citation_pages=e1005048;citation_volume=11;citation_year=2015;citation_pmid=25942577;citation_doi=10.1371/journal.pgen.1005048" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=G. de los Campos;citation_author=H. Naya;citation_author=D. Gianola;citation_author=J. Crossa;citation_author=A. Legarra;citation_title=Predicting quantitative traits with regression models for dense molecular markers and pedigree.;citation_volume=182;citation_year=2009;citation_pmid=19293140;citation_doi=10.1534/genetics.109.101501" />
<meta name="citation_reference" content="citation_journal_title=Journal of Animal Science;citation_journal_abbrev=J ANIM SCI;citation_author=D. N. Do;citation_author=L. L. G. Janss;citation_author=J. Jensen;citation_author=H. N. Kadarmideen;citation_title=SNP annotation-based whole genomic prediction and selection: An application to feed efficiency and its component traits in pigs;citation_pages=2056-2063;citation_volume=93;citation_year=2015;citation_issue=5;citation_doi=10.2527/jas.2014-8640" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=M. Erbe;citation_author=B. J. Hayes;citation_author=L. K. Matukumalli;citation_author=S. Goswami;citation_author=P. J. Bowman;citation_title=Improving accuracy of genomic predictions within and between dairy cattle breeds with imputed high-density single nucleotide polymorphism panels.;citation_pages=4114-4129;citation_volume=95;citation_year=2012;citation_issue=7;citation_pmid=22720968;citation_doi=10.3168/jds.2011-5019" />
<meta name="citation_reference" content="citation_title=Introduction to Quantitative Genetics. Ed. 4. Longmans Green, Harlow, Essex.;citation_year=1996" />
<meta name="citation_reference" content="citation_journal_title=BMC Genet.;citation_author=N. Gao;citation_author=J. Li;citation_author=J. He;citation_author=G. Xiao;citation_author=Y. Luo;citation_title=Improving accuracy of genomic prediction by genetic architecture based priors in a Bayesian model.;citation_pages=120;citation_volume=16;citation_year=2015;citation_doi=10.1186/s12863-015-0278-9" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=D. Gianola;citation_title=Priors in whole-genome regression: The Bayesian alphabet returns.;citation_volume=194;citation_year=2013;citation_pmid=23636739;citation_doi=10.1534/genetics.113.151753" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=O. González-Recio;citation_author=D. Gianola;citation_author=G. J. M. Rosa;citation_author=K. A. Weigel;citation_author=A. Kranis;citation_title=Genome-assisted prediction of a quantitative trait measured in parents and progeny: application to food conversion rate in chickens.;citation_pages=3-3;citation_volume=41;citation_year=2009;citation_pmid=19284693;citation_doi=10.1186/1297-9686-41-3" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=K. L. Gunderson;citation_author=F. J. Steemers;citation_author=G. Lee;citation_author=L. G. Mendoza;citation_author=M. S. Chee;citation_title=A genome-wide scalable SNP genotyping assay using microarray technology.;citation_pages=549-554;citation_volume=37;citation_year=2005;citation_issue=5;citation_pmid=15838508;citation_doi=10.1038/ng1547" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=D. Habier;citation_author=R. L. Fernando;citation_author=J. C. M. Dekkers;citation_title=The Impact of Genetic Relationship Information on Genome-Assisted Breeding Values;citation_pages=2389-2397;citation_volume=177;citation_year=2007;citation_issue=4;citation_pmid=18073436;citation_doi=10.1534/genetics.107.081190" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=D. Habier;citation_author=R. L. Fernando;citation_author=K. Kizilkaya;citation_author=D. J. Garrick;citation_title=Extension of the bayesian alphabet for genomic selection.;citation_pages=186-186;citation_volume=12;citation_year=2011;citation_pmid=21605355;citation_doi=10.1186/1471-2105-12-186" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=B. J. Hayes;citation_author=J. Pryce;citation_author=A. J. Chamberlain;citation_author=P. J. Bowman;citation_author=M. E. Goddard;citation_title=Genetic architecture of complex traits and accuracy of genomic prediction: coat colour, milk-fat percentage, and type in Holstein cattle as contrasting model traits.;citation_pages=e1001139-e1001139;citation_volume=6;citation_year=2010;citation_issue=9;citation_pmid=20927186;citation_doi=10.1371/journal.pgen.1001139" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=B. J. Hayes;citation_author=P. J. Bowman;citation_author=A. J. Chamberlain;citation_author=M. E. Goddard;citation_title=Invited review: Genomic selection in dairy cattle: Progress and challenges;citation_pages=433-443;citation_volume=92;citation_year=2009;citation_issue=2;citation_pmid=19164653;citation_doi=10.3168/jds.2008-1646" />
<meta name="citation_reference" content="citation_journal_title=Biosystems;citation_journal_abbrev=Biosystems;citation_author=M. Hecker;citation_author=S. Lambeck;citation_author=S. Toepfer;citation_author=E. van Someren;citation_author=R. Guthke;citation_title=Gene regulatory network inference: data integration in dynamic models-a review.;citation_pages=86-103;citation_volume=96;citation_year=2009;citation_issue=1;citation_pmid=19150482;citation_doi=10.1016/j.biosystems.2008.12.004" />
<meta name="citation_reference" content="citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=Z. L. Hu;citation_author=C. A. Park;citation_author=X. L. Wu;citation_author=J. M. Reecy;citation_title=Animal QTLdb: An improved database tool for livestock animal QTL/association data dissemination in the post-genome era.;citation_volume=41;citation_year=2013;citation_pmid=23180796;citation_doi=10.1093/nar/gks1150" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=F. Jaffrezic;citation_author=I. M. S. White;citation_author=R. Thompson;citation_author=W. G. Hill;citation_title=A Link Function Approach to Model Heterogeneity of Residual Variances Over Time in Lactation Curve Analyses;citation_pages=1089-1093;citation_volume=83;citation_year=2000;citation_issue=5;citation_pmid=10821584;citation_doi=10.3168/jds.S0022-0302(00)74973-3" />
<meta name="citation_reference" content="citation_journal_title=Livest. Sci.;citation_author=H. Kadarmideen;citation_title=Genomics to systems biology in animal and veterinary sciences: Progress, lessons and opportunities.;citation_pages=232-248;citation_volume=166;citation_year=2014;citation_doi=10.1016/j.livsci.2014.04.028" />
<meta name="citation_reference" content="citation_year=2008" />
<meta name="citation_reference" content="citation_journal_title=BMC GENOM;citation_author=L. Koufariotis;citation_author=Y. P. Chen;citation_author=S. Bolormaa;citation_author=B. J. Hayes;citation_author=A. J. Schork;citation_pages=436;citation_volume=15;citation_year=2014;citation_doi=10.1186/1471-2164-15-436" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=T. I. Lee;citation_author=N. J. Rinaldi;citation_author=F. Robert;citation_author=D. T. Odom;citation_author=Z. Bar-Joseph;citation_title=Transcriptional Regulatory Networks in Saccharomyces cerevisiae;citation_pages=799-804;citation_volume=298;citation_year=2002;citation_issue=5594;citation_pmid=12399584;citation_doi=10.1126/science.1075090" />
<meta name="citation_reference" content="citation_journal_title=J. R. Stat. Soc. B;citation_author=Y. Lee;citation_author=J. A. Nelder;citation_title=Hierarchical Generalized Linear Models.;citation_pages=619-678;citation_volume=58;citation_year=1996" />
<meta name="citation_reference" content="citation_journal_title=Can. J. Stat.;citation_author=Y. Lee;citation_author=J. A. Nelder;citation_title=Generalized Linear Models for the Analysis of Quality-Improvement Experiments.;citation_pages=95-105;citation_volume=26;citation_year=1998;citation_doi=10.2307/3315676" />
<meta name="citation_reference" content="citation_title=Generalized Linear Models with Random Effects: Unified Analysis via H-likelihood;citation_author=Y. Lee;citation_author=J. A. Nelder;citation_author=Y. Pawitan;citation_year=2006;citation_doi=10.1201/9781420011340" />
<meta name="citation_reference" content="citation_title=Data Analysis Using Hierarchical Generalized Linear Models with R;citation_author=Y. Lee;citation_author=L. Rönnegård;citation_author=M. Noh;citation_year=2017;citation_doi=10.1201/9781315211060" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. Legarra;citation_author=C. Robert-Granié;citation_author=E. Manfredi;citation_author=J. M. Elsen;citation_title=Performance of Genomic Selection in Mice.;citation_volume=180;citation_year=2008;citation_pmid=18757934;citation_doi=10.1534/genetics.108.088575" />
<meta name="citation_reference" content="citation_journal_abbrev=Genetics;citation_author=T. Luan;citation_author=J. A. Woolliams;citation_author=S. Lien;citation_author=M. Kent;citation_author=M. Svendsen;citation_title=The accuracy of Genomic Selection in Norwegian red cattle assessed by cross-validation.;citation_pages=1119-1126;citation_volume=183;citation_year=2009;citation_issue=3;citation_pmid=19704013;citation_doi=10.1534/genetics.109.107391" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=I. M. MacLeod;citation_author=P. J. Bowman;citation_author=C. J. Vander Jagt;citation_author=M. Haile-Mariam;citation_author=K. E. Kemper;citation_title=Exploiting biological priors and sequence variants enhances QTL discovery and genomic prediction of complex traits.;citation_pages=144;citation_volume=17;citation_year=2016;citation_pmid=26920147;citation_doi=10.1186/s12864-016-2443-6" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=T. H. E. Meuwissen;citation_author=B. J. Hayes;citation_author=M. E. Goddard;citation_title=Prediction of total genetic value using genome-wide dense marker maps.;citation_pages=1819-1829;citation_volume=157;citation_year=2001;citation_pmid=11290733" />
<meta name="citation_reference" content="citation_journal_title=BMC GENOM;citation_author=G. Morota;citation_author=R. Abdollahi-Arpanahi;citation_author=A. Kranis;citation_author=D. Gianola;citation_pages=109;citation_volume=15;citation_year=2014;citation_doi=10.1186/1471-2164-15-109" />
<meta name="citation_reference" content="citation_journal_abbrev=J Anim Breed Genet;citation_author=W. M. Muir;citation_title=Comparison of genomic and traditional BLUP-estimated breeding value accuracy and selection response under alternative trait and genomic parameters.;citation_pages=342-355;citation_volume=124;citation_year=2007;citation_issue=6;citation_pmid=18076471;citation_doi=10.1111/j.1439-0388.2007.00700.x" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=T. Ostersen;citation_author=O. F. Christensen;citation_author=M. Henryon;citation_author=B. Nielsen;citation_author=G. Su;citation_title=Deregressed EBV as the response variable yield more reliable genomic predictions than traditional EBV in pure-bred pigs.;citation_pages=38-38;citation_volume=43;citation_year=2011;citation_pmid=22070746;citation_doi=10.1186/1297-9686-43-38" />
<meta name="citation_reference" content="citation_year=2010" />
<meta name="citation_reference" content="citation_journal_title=Genet Sel Evol: GSE;citation_author=L. Rönnegård;citation_author=M. Felleki;citation_author=F. Fikse;citation_author=H. A. Mulder;citation_author=E. Strandberg;citation_title=Genetic heterogeneity of residual variance - estimation of variance components using double hierarchical generalized linear models.;citation_pages=8;citation_volume=42;citation_year=2010a;citation_doi=10.1186/1297-9686-42-8" />
<meta name="citation_reference" content="citation_journal_title=R J.;citation_author=L. Rönnegård;citation_author=X. Shen;citation_author=M. Alam;citation_title=hglm: A Package for Fitting Hierarchical Generalized Linear Models.;citation_pages=20-28;citation_volume=2;citation_year=2010;citation_doi=10.32614/RJ-2010-009" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=A. J. Schork;citation_author=W. K. Thompson;citation_author=P. Pham;citation_author=A. Torkamani;citation_author=J. C. Roddey;citation_title=All SNPs are not created equal: genome-wide association studies reveal a consistent pattern of enrichment among functionally annotated SNPs.;citation_pages=e1003449;citation_volume=9;citation_year=2013;citation_pmid=23637621;citation_doi=10.1371/journal.pgen.1003449" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Comput Biol;citation_author=R. Shalgi;citation_author=D. Lieber;citation_author=M. Oren;citation_author=Y. Pilpel;citation_title=Global and local architecture of the mammalian microRNA-transcription factor regulatory network.;citation_pages=e131-e131;citation_volume=3;citation_year=2007;citation_issue=7;citation_pmid=17630826;citation_doi=10.1371/journal.pcbi.0030131" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=X. Shen;citation_author=M. Alam;citation_author=F. Fikse;citation_author=L. Rönnegård;citation_title=A Novel Generalized Ridge Regression Method for Quantitative Genetics.;citation_volume=193;citation_year=2013;citation_pmid=23335338;citation_doi=10.1534/genetics.112.146720" />
<meta name="citation_reference" content="citation_journal_title=J. Anim. Sci.;citation_author=W. M. Snelling;citation_author=R. A. Cushman;citation_author=J. W. Keele;citation_author=C. Maltecca;citation_author=M. G. Thomas;citation_title=BREEDING AND GENETICS SYMPOSIUM: Networks and pathways to guide genomic selection.;citation_volume=91;citation_year=2013;citation_pmid=23097404;citation_doi=10.2527/jas.2012-5784" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=A. K. Sonesson;citation_author=T. H. E. Meuwissen;citation_title=Testing strategies for genomic selection in aquaculture breeding programs.;citation_pages=37-37;citation_volume=41;citation_year=2009;citation_pmid=19566932;citation_doi=10.1186/1297-9686-41-37" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=D. Sorensen;citation_author=R. Waagepetersen;citation_title=Normal linear models with genetically structured residual variance heterogeneity: a case study.;citation_pages=207-222;citation_volume=82;citation_year=2003;citation_issue=3;citation_pmid=15134199;citation_doi=10.1017/S0016672303006426" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=I. Strandén;citation_author=D. J. Garrick;citation_title=Technical note: Derivation of equivalent computing algorithms for genomic predictions and reliabilities of animal merit;citation_pages=2971-2975;citation_volume=92;citation_year=2009;citation_issue=6;citation_pmid=19448030;citation_doi=10.3168/jds.2008-1929" />
<meta name="citation_reference" content="citation_journal_title=J. Dairy Sci.;citation_author=G. Su;citation_author=O. F. Christensen;citation_author=L. Janss;citation_author=M. S. Lund;citation_title=Comparison of genomic predictions using genomic relationship matrices built with different weighting factors to account for locus/specific variances.;citation_pages=6547-6559;citation_volume=97;citation_year=2014;citation_pmid=25129495;citation_doi=10.3168/jds.2014-8210" />
<meta name="citation_reference" content="citation_journal_title=Genet Sel Evol: GSE;citation_author=L. Tusell;citation_author=H. Gilbert;citation_author=J. Riquet;citation_author=M. J. Mercat;citation_author=A. Legarra;citation_title=Pedigree and genomic evaluation of pigs using a terminal-cross model.;citation_pages=32;citation_volume=48;citation_year=2016;citation_doi=10.1186/s12711-016-0211-3" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=M. G. Usai;citation_author=M. E. Goddard;citation_author=B. J. Hayes;citation_title=LASSO with cross-validation for genomic selection.;citation_pages=427-436;citation_volume=91;citation_year=2009;citation_issue=6;citation_pmid=20122298;citation_doi=10.1017/S0016672309990334" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=P. M. VanRaden;citation_author=C. P. Van Tassell;citation_author=G. R. Wiggans;citation_author=T. S. Sonstegard;citation_author=R. D. Schnabel;citation_title=Invited Review: Reliability of genomic predictions for North American Holstein bulls;citation_pages=16-24;citation_volume=92;citation_year=2009;citation_issue=1;citation_pmid=19109259;citation_doi=10.3168/jds.2008-1514" />
<meta name="citation_reference" content="citation_journal_title=Genet Sel Evol: GSE;citation_author=A. Wolc;citation_author=H. H. Zhao;citation_author=J. Arango;citation_author=P. Settar;citation_author=J. E. Fulton;citation_title=Response and inbreeding from a genomic selection experiment in layer chickens.;citation_pages=59;citation_volume=47;citation_year=2015;citation_doi=10.1186/s12711-015-0133-5" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=J. Yang;citation_author=T. A. Manolio;citation_author=L. R. Pasquale;citation_author=E. Boerwinkle;citation_author=N. Caporaso;citation_title=Genome partitioning of genetic variation for complex traits using common SNPs.;citation_pages=519-525;citation_volume=43;citation_year=2011;citation_issue=6;citation_pmid=21552263;citation_doi=10.1038/ng.823" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=W. Yang;citation_author=R. J. Tempelman;citation_title=A Bayesian antedependence model for whole genome prediction.;citation_volume=190;citation_year=2012;citation_pmid=22135352;citation_doi=10.1534/genetics.111.131540" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=Z. Zhang;citation_author=U. Ober;citation_author=M. Erbe;citation_author=H. Zhang;citation_author=N. Gao;citation_title=Improving the accuracy of Whole Genome Prediction for Complex Traits using the results of Genome Wide Association Studies.;citation_pages=e93017;citation_volume=9;citation_year=2014;citation_pmid=24663104;citation_doi=10.1371/journal.pone.0093017" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=Z. Zhang;citation_author=M. Erbe;citation_author=J. He;citation_author=U. Ober;citation_author=N. Gao;citation_title=Accuracy of whole-genome prediction using a genetic architecture-enhanced variance-covariance matrix. G3: Genes, Genomes;citation_pages=615-627;citation_volume=5;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=Z. Zhang;citation_author=J. F. Liu;citation_author=X. D. Ding;citation_author=P. Bijma;citation_author=D. J. de Koning;citation_title=Best linear unbiased prediction of genomic breeding values using trait-specific marker-derived relationship matrix.;citation_pages=e12648;citation_volume=5;citation_year=2010;citation_pmid=20844593;citation_doi=10.1371/journal.pone.0012648" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Genomic Prediction Including SNP-Specific Variance Predictors" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10.cover-source.jpg" />
<meta name="twitter:description" content="The increasing amount of available biological information on the markers can be used to inform the models applied for genomic selection to improve predictions. The objective of this study was to propose a general model for genomic selection using a link function approach within the hierarchical generalized linear model framework (hglm) that can include external information on the markers. These models can be fitted using the well-established hglm package in R. We also present an R package (CodataGS) to fit these models, which is significantly faster than the hglm package. Simulated data were used to validate the proposed model. We tested categorical, continuous and combination models where the external information on the markers was related to 1) the location of the QTL on the genome with varying degree of uncertainty, 2) the relationship of the markers with the QTL calculated as the LD between them, and 3) a combination of both. The proposed models showed improved accuracies from 3.8% up to 23.2% compared to the SNP-BLUP method in a simulated population derived from a base population with 100 individuals. Moreover, the proposed categorical model was tested on a dairy cattle dataset for two traits (Milk Yield and Fat Percentage). These results also showed improved accuracy compared to SNP-BLUP, especially for the Fat% trait. The performance of the proposed models depended on the genetic architecture of the trait, as traits that deviate from the infinitesimal model benefited more from the external information. Also, the gain in accuracy depended on the degree of uncertainty of the external information provided to the model. The usefulness of these type of models is expected to increase with time as more accurate information on the markers becomes available." />
<meta name="og-title" property="og:title" content="Genomic Prediction Including SNP-Specific Variance Predictors" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/9/10/3333" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="The increasing amount of available biological information on the markers can be used to inform the models applied for genomic selection to improve predictions. The objective of this study was to propose a general model for genomic selection using a link function approach within the hierarchical generalized linear model framework (hglm) that can include external information on the markers. These models can be fitted using the well-established hglm package in R. We also present an R package (CodataGS) to fit these models, which is significantly faster than the hglm package. Simulated data were used to validate the proposed model. We tested categorical, continuous and combination models where the external information on the markers was related to 1) the location of the QTL on the genome with varying degree of uncertainty, 2) the relationship of the markers with the QTL calculated as the LD between them, and 3) a combination of both. The proposed models showed improved accuracies from 3.8% up to 23.2% compared to the SNP-BLUP method in a simulated population derived from a base population with 100 individuals. Moreover, the proposed categorical model was tested on a dairy cattle dataset for two traits (Milk Yield and Fat Percentage). These results also showed improved accuracy compared to SNP-BLUP, especially for the Fat% trait. The performance of the proposed models depended on the genetic architecture of the trait, as traits that deviate from the infinitesimal model benefited more from the external information. Also, the gain in accuracy depended on the degree of uncertainty of the external information provided to the model. The usefulness of these type of models is expected to increase with time as more accurate information on the markers becomes available." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10.cover-source.jpg" />
    <title>Genomic Prediction Including SNP-Specific Variance Predictors | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"zAEK8bLdudJ8FPyjrXwzuuiH8hhpBrpGNnjdWGncwzM","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"494292","apath":"\/ggg\/9\/10\/3333.atom","pisa":"ggg;9\/10\/3333","ac":{"\/ggg\/9\/10\/3333.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/9\/10\/3333.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;9\/10\/3333"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;9\/10\/3333"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/9\/10\/3333":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022OD1YrtaiWM9LGssF9xMsUC6M7HejVGBmK1qcXdBgtB5qEUrTBkjzXORKMQob3HFdoOmjpbNJqbgFcNHiEvBymZxPl24E6l+OWGqe+Ue7Gy3qs+y\\\\\\\/HBxoWVrqBBaXgjMcjd48wVM5Er6b9osSsh1+BiVDS9zXCmh2tmmW2WcRyVRGyZXkS5WnsuU\\\\\\\/zwREWeNWmSRUm9UKagMAByi4WiskQxaYXvZG16bR9\\\\\\\/OTO5Ss+2xPEga5RCZOxHhb24TtY5zrwcboXDBbgUx0\\\\\\\/MAxH0dVNJqozgzibdstePUa22QV+uDye+4oy1hnZz9lZHerWyzMxnWIqRWV35V4qvhWSxmPyDjyu3WDla1B2r3SqdNFIvkBfAtUJRwP0kXatRUX+Qr8UBd2EIdkgWds\\\\\\\/WlA+T\\\\\\\/T3dWzi\\\\\\\/5HMFS2UZ39vHWTw\\\\\\\/t272F\\\\\\\/OScq2bijyEatrXp0a5ZYvzHsC91ZeIwvSxb0DNt9V7Rc1tI7JIcgWt7drooXUHd38tT\\\\\\\/xNsi648L82KmeGkl\\\\\\\/D32XhjwQCjRis2F7LEm6crhjfWZ7oPmJokSGFYMbP1ud7zXxum7\\\\\\\/hXPIJ9\\\\\\\/CakLCTcyOD9j358X6F++nmNLBE7Z\\\\\\\/cfwyDgYqbqjJHIh2GV28lJsVbeHKA0PdL816TtUV287HVBkBrStZrR2aB5t5D0VYCOhr0jaiPLTcvmFvV84pVl6\\\\\\\/NOhoARzeUW2vxYVLnL2HpBTKjewvmG5b+b+SdP\\\\\\\/c383X7NNpuygAc\\\\\\\/bun3x+TVvSQyEGYEyx3tF69UO\\\\\\\/BhkuonvGHWNy2wYPs1cA\\\\\\\/3JPf1\\\\\\\/mvzA\\\\\\\/OQMHFvik1BVvFuFPNoeKb8fkdWSl1QqHrn6EwAsuMnJXi\\\\\\\/O+g1P2XuBu7LYZMgSZZPJniesDj5PCT5RKMzKUYb0Ax7uKH64qIPStD18+PVGTMjT7PtH4B9O2AN+UjnAHGSa5ajvrpb8ToVLEmLRjTz9R81CyAguJwZgj5ghMdwRmn84ATYD2Uo7kqDoJFY6DFloLtxg1Tq2wSejt9UfiPTd8bHYZe39mF56YqjNUJkPXni3sCbCIav+A9GDubH4fjbM2ZQj91IdSjfpHnq3lUlJJzSUr2UhhJ+tWCHUQJAfnn06tJbqCU0xXW0p+0qxLhXtA6aBfwSDYKjiiUHBf6dZzlpfSvfkGf6Td0DYOYQRMM7f8ey1YFtPeeLnl0SJh0FCQvdVAltWoaZTx3yDVIkq3nUUNcC9CHAvWL+aiipI1+aRZl+3A7wSOZjqTJLINOMuxZcfRNon9vrSLT8CiPupXve\\\\\\\/\\\u0022,\\\u0022iv\\\u0022:\\\u0022yQEdfSt6gbum+uao+ifpuA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022a430b2b785448b5d429bc8fa5dd5e062d6b5c97a3a61b58000349fb7d4100986\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022ROUU6b8GEUTVcfeB4d4czVO9tMaM+2CM34UYkpMx8mlJ1XkunHjOjHYWHF\\\\\\\/1PRpMisoj6S5w\\\\\\\/8yiFUj26ccA5bfEQFAgexYpa+47Ah7Ui3F4ATrd6G8Aqn3NSCKQFXVJGvdfrcj67pMS7boao3wT6fUWr9k4EEBSg7XZ2n+WR4l+hM7fejtXjCxC1Yq6Y74aspEMT2ZsS+rtH\\\\\\\/32golaltE82Txcgyo6+9UHaM7503c0+R8jmneS2xFl2KLy0rX4zdJdPaQ+30isKP6s2jGSaSZ3oGzxDmfuQI+tZik18sh14pFKpXa\\\\\\\/Koujdvwq9fBHUWAtSdz4Z0IkvXy7m50IgF09jrfISdnWdGoaKcFtLGr6MBs3Da+OJDZ+7kZblWcfAIKpbhKBf9Fg1nxfYmxE1STA+xGAjbgnYtSwVffVKgG1v8nvJB0IosVYGMp9tnLMaLLP46Til\\\\\\\/eisw7ZzbQ7OQISZ1sYYO1HJ4rY4ugy1R1\\\\\\\/7+GOG1uGLdxO5qT7Ynsw9yMVPFk9z3A24TL9ubEXWpRaNcA8\\\\\\\/LFDAmyz\\\\\\\/\\\\\\\/ZjWjIpCArfTdrwcaxP34S54d1pwCgWtsCbVJ3lcwokmJhJGp4eBc0hvYUeEyCpnKGN7rzHqsBtcBK\\\\\\\/64C7ey9ud9EJCTyRi601UGGYHInRH+pTtDFByVKz8tY\\\\\\\/MgAPTtLx9qikwyXME65fmNrLiS27k\\\\\\\/AidIFl4IvWwSGGSheGcH+I3BgtiIUcrCTyTX2dMpOl6yaqqpVgJWjAZzZuC9SLAikArFk0+xdKxDSb9vlkok4CDj\\\\\\\/D0GWAcM6api2QdQowhmNfIx6gVT6o88inT\\\\\\\/fj3WQQfiTmBrBkydrHggRDtHml4WZcEVCJ0mKNzV2\\\\\\\/7DSxjh+yDYNmNAmAVEh2Wx+6ZiBVFRCwEmetl\\\\\\\/Qf+7qdtk8m\\\\\\\/qyc0yISh\\\\\\\/zUMU32Z06AQMVDYU0Le3uKkE2m4NH+GWFocaCSWUinuHIZ4DCKg5QatPAuU8dxzeHn4eKqUP9jya78lzQ9PykxX9EdVffZALy7C3I4BXYnoLzsJJyM0tmvZ5jYD++HQxLeL5jfc9p3bjgTvONJ5HgH2yQuOz\\\\\\\/TXmJlsOsUDZA\\\\\\\/BHecBeTN8eD6T0g9DJXAQ9tLX50jcun0G7EGwob0yRAeuCWD6be2wQwMiOGDotyn04r6MRgFQ7jC7UQlyd2bBNqOaH2qisfBR2jG2xLYjiKL+gODoBdM9BI9ZWDY\\\\\\\/v0ILbRD4W\\\\\\\/lWbusjhXIYfKN5CGNJc6rDq7vWOZZgS3rpnc6QlPhz6r5qLCA0opy6Nx698QCFBAu4d5gtvK1oOklBQJwQG\\\\\\\/lPWgbEWBS3DrOeNHi8ek0Tf6qvgBOxdV+\\\\\\\/AcCW62o4+pfEKc5SUTZNqufUyLEkD\\\\\\\/XrGOr6DmFb\\\\\\\/7ZMioAuIxbq6bxj2UO8L\\\\\\\/g7he+ARZHhbZ\\\\\\\/vrYMkzTKEHIgj7cep6OWVCGWXNlriHG\\\\\\\/mmmiBVPHg3c1SUnvVLPpkL4PWLyuoqt66ldwASWR2kMcYye0k\\\\\\\/Cj5GFTGjYaanOWGIhN8ocNZEVZp5Vo65J6AnRcfYbzJ4wgEv7p6zIwVcfLk0p8jeuQnBaWyOkysnBW3YbHwwSqyfhAj\\\\\\\/4\\\\\\\/\\\\\\\/JyM65xVBJCOnmMrcQ5b4n+5EylBLd5Ua\\\\\\\/QCH7rl+LCcSecItHLFJb2XeiwF5hTa7V7XfORuFNEiSNWqMJFycjELdJ8mUQpTUGT2zEJglr9mTnCGiFefbaLm06Y3z1nt89zIGPVvO\\\\\\\/wn9jnisNx+pbUgzXfqWH+RGT3SLKQREqsGkVwTOOQvoo9vaJkhCzadfl0i0IVVwofsJhzC295RItaXQtlrO8rS3CN3AqcK08lbSGdYlLioyEQCh5SsvaMMNBgzMIi9mP1YeVm7ees3qRMn32CoK55vEtkpF4fPbxgCEhxdIvX+jkqZFRhm0Y4MQcQcm5nntFRnIxV8cPXAhSsd5\\\\\\\/fBrXgVbXjK5Zhe\\\\\\\/Bu7qL9GMps2FtkpTWXJWVv9HGRgdXKccuzZXc8dTiY6tYqRSQquFBegWtvkRtorLgDdNplprUxAqycCRW4itfkBcXXzJ9ATFyJEy0pxKNgW1W\\\\\\\/QvuIA4vlXiJ5f32nqzFKriAPu4l+hTGKocRFYfgsvVHHLdTXdXMK6ICXxEOPXM92eW3L3CMPD0l53+E0MtOdBU8Vq6nPLR+bSKjJ\\\\\\\/\\\\\\\/Y\\\\\\\/z1bSeSl6u6cjhSzu72IQlezMhZcWq5xKTLebo4s\\\\\\\/3LUP7HiIAaMo0mFvGYM5h8++p60ngc80vcRHJ6arEyEtuTUhv9Je2aOnrAIfQ9fGq\\\\\\\/drg2CoJzDbohIABs6wJ2LtMfoOztSLiYDoaW+4S+oxQ+yrwPwmXHlF2OMUu0gqf3G3RraIKRsNFyA7BdVv3g8LoEQH\\\\\\\/xIzhZOadHNn9ZEhIHyuDA+zTPzAOFVdNeepONiYxtHrm\\\\\\\/MgdGY2KTU\\\\\\\/DCdLFsD8zYVHw3SpK\\\\\\\/nq0uZaLo2v+iqb4K2tkJ11AOkx8iDwYKjnNQVGkfRPKVs\\\\\\\/6CBKqgV2KCAPWRJAoPboUVPFlSOzqoYI+WJUIW7kAT4wx3It9GZhgI+QI\\\\\\\/yjZAe405mtG41tWcjUe1t9TnLZjMyhOCDMgzdWTk3PansUIpUSTVWB2RZ6HdwzlB9Z2tsWA6bHEV7lsBvzMRMsVZp6tPyriZvaG3876os4ClNUn+3T58CWT6XMCbXS4Vg8A1aeuLpidpgYFydVLhcRCttn9qMiUfckyQ8gK7Ves\\\\\\\/qpetay3XRb6Q9rM11ccusaKwg3y19QnkTC+TKAzaEoK2ZoBGrN\\\\\\\/TS+BO2yjliVDc85AKba3qKF\\\\\\\/cWL0dRDAZnU5HzJSvRI8NTl9JbTH\\\\\\\/tf0A2jswDsXw28Zr\\\\\\\/u6pGBQUzYFgBjy6FEfpKfLl7EYYOJICU\\\\\\\/05vU7bCQqHPT2n64tOAjp02sZEtsJ4zeEqsKKHfkv5N+tSOqzBHqLp6KEdcMtqEBJkIThfCPSqRTdQXNY82JWdgH65vjvgEw1u6KF7BzLUYM\\\\\\\/pA8P81ZHxjdFO3gIG2vTI5qfjqB7fRZ\\\\\\\/E0GkOt\\\\\\\/3kFIOVkzMo9\\\\\\\/1ACD4UkdmkOvPaB+qVCMEewq5njwqJzUgrKS3rsbt5JXBC63uU9HtGbT\\\\\\\/Ln\\\\\\\/tdicyqX79w81OSsgMUl7pY+cDQRfm9yled88a29Cxle6JFuqsuSe+D49s1zOCmbiRBz43\\\\\\\/Z9hgXYSguP9wAftbkO3l0xARkR2IMhOfYh1zPuvhVyABVPgezPKX1X8piTqVLlxy1UVX6Q+FsTpMHY1z3EbavEXsXH8O0UMRy3apOkCnr\\\\\\\/Y7wiHOeBe4CE0mwCYH\\\\\\\/UhcDiess3awD5YFc2cWyDQscpxR3f+LDZVUMEgkjAco+GfQkc4OclfFO7J127obvFA9fQBLTIwRDKULmNeVGszxnH\\\\\\\/F73SbZdJ2FS1rp3oq+WKs+12olIA8ZMKnRDhe3wK5x65Yes2E1dRsD6EarGLSLd83n6C1LKFRRBXNMILiWIZb\\\\\\\/LVW3QdzcqRLFG6PzRrYsl26SByJdRww9GzWTQMALZPuxfDnvaZgwVIAgsLwlLNHsBOtdmm7y46yYC8kYDcTfQIHFugqWF6zHfL8PTuj0FjRtkwKTWV1rpmkZoEtxDLmYQgty4K6VLEJq\\\\\\\/slAmuefFWE6kA0o7PAkGGoDGiz1yGRknteGHJiDxcWZV5M4ZTlLnAwv0mJF8I8kiMnGb7D1xgkr7j2cnVI3qiZxrqPRAA+KZbWRyark74POome9BCzC16euKmGQsc3EcprXPhd4ix98KBwUMl5w1QOMlBHlsQE+qnaOQWmnwSU7BRTX3zXLDuMFjmRE+j7aYYMJC\\\\\\\/pPCM48dnehbGVI47w\\\\\\\/frosPCPVaJ\\\\\\\/XJ4DWf\\\\\\\/XVSFdwm9rA51iFiAesoY0gIW1neJgj9m+Lv8xviNkOyx++tthSLJhEt\\\\\\\/oI+jNTqhpF3o591djjf9OKwPxjlOxOBi8ki2uIGkNh6\\\\\\\/0ByblUoO0dRfGV3B8uznpZEAWDp4ixEoFmtFUPZO1lQ3kKc5tSHSkclfGbb+S6jjbQYrZyPsk5VLmRDGeeyQzmL93pTRCWeNM\\\\\\\/UcKv3sbS8W+Y73537igpgJp0fSW39G0fo43Z2P\\\\\\\/v7UDeBQbjGCT\\\\\\\/jR3EOTgYQv7v9dlsq9rdD\\\\\\\/fZ7kgnPKCrdJPMFYoBEh3AvQwZe1qS\\\\\\\/RRJhTkwM3NrHor50FxBeQXeCReQ48XZKFZFfgrfHqcCjPDAZ+yHnc3pUz0uWF8f6j7EkbpRMIZH6VVWHlgjkPLM9CPgvV1PxVk6B5K9ZBBjk\\\\\\\/eLQClo4joU9I2qsvgCqDyCRBIK82DMZzZ1\\\\\\\/MIE+gBu7g9cPWUNirqjRqhtqrdedcc3l41ohbzMby2xgSFcXQF0HQvjgnm6CZg38XuOOMA+Wi1KAszfQsoUo+izhpO7eW0ugYZ2IVyzciPwkFWTDgDpxw1ubiSd+I5hEfaI8YQlxccE9L9W9j+5DNritahx3XengEFBbeRE3oJd6wbCVi+yKt1yW8YJ66A3F+qjUslEXNiLNIngJFcvIWTAh73tP+xPHbjo4N3XWdfRqIjAn7ikoY96Mtbj0aZctW00PFjBByBKCkFoCo+1hdL5\\\\\\\/HE8R+XMNnaOsPq6PQ6YIIIaemQ16+wxBtFBCo7V3KswoqfJih\\\\\\\/v+RaKxtdXPZvDxBoJBLOOXLPbzUo+INb6Qe2QMb+4\\\\\\\/Qef+K\\\\\\\/AAYpya1kNWAtdllp8JtfQJtWlhKuYBAoRkhEDmrPMKYckihlZHLLYrM3l\\\\\\\/GN7KitRtnGVfmwDwRb5GIVHuHsdWnuRQ\\\\\\\/+yYQtvbsuXzEE00h1gNCUy7wv4yJTA\\\\\\\/5jxB3VhrebceGGbtzGUmipVxkaXlnkJusdFo2oMIkotZeVQ1Pir5Sp7RRY3fdugAW1rD4kcjljJXSRezVVy7JMyszikper4pIRBfMdOBECWNcufc7SPP44dbTqlQdbanoAX0bBqtQPJT2TkAqz1NrWd1eT5s\\\\\\\/62MUSyozu4rtg8sklV76yAi1KkL\\\\\\\/iOEd35D+lLfNdQcBWgklhQEO0mT0yPk5HmQJLLkhIjTP39XFZSj7MHdLf8krvFVwizdcgAE6dpkyGoygbDqX5AjU2yMai\\\\\\\/mU1TeF6BhUKwiq6EO9E0rPeTC7A4\\\\\\\/4FcAndzVcAFtuvJSSU8YghPAOqd+WJSP\\\\\\\/wHZ6X8W9jeqEiufjVY4B1XudJGOo+BDgQATP80HYTv80YBQAzwBKec88O8Vu4khMCbecq80jtW54Wv7dA8X5g1vR5L8OdTjHr4GIk1kEARGkfHvikVJeMGF6tj1Jkofb+CBzTavgXSX2bOeBfMVy16AcJP0kB9\\\\\\\/TiJnPEmDga2fRjMdTKAr0s4CboASVDN+ClM0fRs9TQF\\\\\\\/OwnsftJHGI5icKg++x22GnIaC\\\\\\\/\\\\\\\/b84sFESA8xpYK5I\\\\\\\/VhcNd4mgjeoNNxjfBG0DnBsl1S6W3ktZRMaXOpF22VufgGwG+m7UWbfGlpklknyt1qE1IuB1C0yu2PhsWHhFNsiGFe+taP95oQ2beralQtVBVj+NN7tEOgYSkk1aqjrsG0s1ZIMdM5Ifiqcj88YfdhlrcPZZDAKzwP5UBvd4YCrIJIn3ehvSEsFyGe+Qbof63hbdRNFgHxdwnic7I6HUlA4KLD7VrqpqKNEBGg0lDE67Znx6dkxfkJKjHVvFep2y65NAK9aqRGqIhhcElwkfMlV7dLo2B4wtThnYlNzterrEzKAcAoorfEKcRKZDmRD19QW3gkWvK\\\\\\\/Ecfoekzl5WynGeGvie7WOE44P81PnJn1AtSF2QlcvQq\\\\\\\/\\\\\\\/1S5dAi3EVdONoLTp36GGKRy3If8j\\\\\\\/zBAT0jEbkKlsz+jdyHCdw6+NEiud1sprLV+O\\\\\\\/ZtrVR4ftKabzDaP+7SwL1Fs2JXN1gElN7VR04SL5ZUiql3gfhqP4LrG72WiB+x4+BweRD0WjycozDO\\\\\\\/q1AVgP9q7Pa2m4jx2TMH4Xf2MSPHkdhkrUQOjwLxOXYPkY4VG5d4\\\\\\\/jeF4XEWzlG74Rn5WMyeFWnHQiV35wGJR28o4YsGD7b\\\\\\\/fe5HR\\\\\\\/kbWRYqgcm3525jGE7SIOZw2y\\\\\\\/lX8GR5478WNcXTCv2o3HF+dnLho2eFdQbKZRlpFf5uKVwDhC57aHUmez+p8zKIHkVzjzJhcG8aTCJRzbLYf4GntCRP+ldWLhTtPCpYfA4QAfPduKq7fWcvcRXgA8E\\\\\\\/3wJODDcmogrbcKpynp+BAu\\\\\\\/3wHZkG2z0vKHCd3zRCcgC4vBMaqLac6oMDhf2WpS1\\\\\\\/EuWHm76uXOq0dfkiQyM9cIvNz1ptXGqQX0j2coLiv5IsXHjd9fwTIZIKifkay0Ky89wLhCZtPP8UFBxpOEHb94SoH720hTguX\\\\\\\/scv+mwTvdaXfJlSjq96jafuRvI2EQ0KToEruUg9D0oBnFZd0ajIL78+Pg\\\\\\\/GsrK1DJnSRLk9NiN4FKy8AWJGc3X001CcbBXWStpuRVL4xZwsHfa6HkYoNf04gspTugdHyWIYvxueqDvUrWxutjQpFkbs1DYwd6AY+TPT6Bq2HwokOAX6jgaz8WT0DsV0wHk0ICLhyPP\\\\\\\/YPmOzlsqBiyHIG0JsliGYEY4uU8swsxFc2C7QNEkUOmY6PxMeOn8FQwMHjFTriKOD\\\\\\\/fqxji3Hg2L88MuiQl6P+J55faqMcIiyDCHCjc\\\\\\\/J3y+fFGhUjyZxqmi1r1mysXbEYvlkl7bTOs\\\\\\\/t\\\\\\\/jFxzrJuUHfkTUMPAhWok+S3Fw63F4TbIx5QpIq9k15tc7ZP86qjoTdnK7OqFDCZv51QWrEq66XyPcVrIbDbse7LcgSIoIYJu3NyzKYY6QkqyBggThkM+P9lt\\\\\\\/wZt1xoEUDtKKesTpkCcmCiOe+DKv9GSe3zeNhxi8mWu07pSd3aGSRKmv6RLu5OwJrorVBcxKy5KjLyjd85b3DC9y5Nfwqad+lRlaBwhidLs76tHkST+DzaCVjjZ2dMRvLV\\\\\\\/thXdIjZHzuh8c5MWeVt96o42Aj4w0dNSjHhobgx+6XPFu+Cnw5mVnjTXeg83F5HKiWqxGI9vkifUOWAGJ76y11jVPkxXd91hXeJOxK6EXiRGDZjEWpUHG4KfF1ofZR7hbRZN8ivoiWgfFYhZolOg91HeSLi5IGID\\\\\\\/FMz8Kzp1Mq6EVHCgF928tBPt6vGmbARg4MOulcouyX0sSJb7BmQNwOZqOvwMYWKmRs6AaEwkqtba6y2aIlNexAvaN0\\\\\\\/J5B15HhiDYU487\\\\\\\/LJQ7eFhbxmI6X8Chmj+3bSZ9zurjdMJLFRnHiQ9\\\\\\\/prSs87vsE5FL1eea\\\\\\\/\\\\\\\/nV5x2rIP2vkJUwnfxM+XD+Jdx4QkK5DeeOESdwdiPHDnyOqmRyaupM4NRb+ILJ3CVtmaO16RywjgSRFEIhr1fCVl5fnE4wkiWdvxWCk7uIBqFFyY8pR5wmvTDH\\\\\\\/exA0aBP+sdZD1JXsjFimNJCEdzptcsVlrBBj65HPfDQxIFwP+d6nxAJsS1UP6JtQtvTg9CHZ4QoeTAq7kNfep\\\\\\\/WJnuELX\\\\\\\/Tfrh8K92BNet7rog5lESOSyU9u9MxHSI6QFmnF34kwvxDIkL3xOVz15k\\\\\\\/YHQM9A+x88g59PXRCy455E3bkdHFjkQJ2DzH9bnUAYPWQ5VSC3JBEe34BJbdLECwFtnqNVFjf\\\\\\\/PleYIFI4poPMNREJOnn43JQWMqFYIPDIw+a01vEQY39CoWFR0qA+569f86mMnCZTw7\\\\\\\/w3e3D9hrfth9+UitXxtpEJ4psOcCmR9f073ZRKRWjl1WgH1nY10SPcmExRp7U6UMPZ5t8ztnJffhIPClknnF\\\\\\\/iRYJzOIimBqbKDREWJt2kjlDWLCOn\\\\\\\/6nYM3GtspKsuCdRgfhF4vNgG452ZOBoXwNB4tbcS6WB++xXyucTOdmDNbyrt8yRcxuZdaEPfNi7SJ4PEWv4V1qmmDSyxTijxbqzgr6b6fvYMfJ1Ax6nH0a7kPyoUefj21jblqG3ixIHcTcO0L\\\\\\\/1A9wpymMvjeN9uthvc7r8U8AcZI05eeaYofqAFLzfXoqUUomkGt34ARQHZi3sHrwEM\\\\\\\/iH56fveM7SmDhAK83q6EKJeenRdlK7U4\\\\\\\/QSV+yOW5mVFKXxI44bHYWagXgINNhJk+68fQ0qaOuELv+VE1b68JJ0+H8Pq7rFMxVt4voJm41xY5pFCzavA9jab6wVkGXEu+qKu\\\\\\\/DXbi\\\\\\\/rocdkpavcShTGns2KT81ZXFGa29Sv\\\\\\\/473YH1ajOTSfCUt9PqsYiVZS1sWHrQGM7i5PnBRfm8P0a\\\\\\\/6LDI\\\\\\\/RacAytIrP\\\\\\\/Zjn6DMyAHiWh1vTReaOYzvqS5RckI26llUf1b8XUKi3IeYNp4sLqHCwN5KnbL\\\\\\\/UVBleHEzMS055IcD+Nck3\\\\\\\/CeUhZz5ZDN8lc8xz9WoSTgo0QXEsg3t8euhbDhEm6ukpaNYJSsC97pGMWfF9wFGBRNU\\\\\\\/TpGL0TL379Kwyc7wW0ANme5oXHr4MtgCIvKtbCyuw2NsyecNxS0HKbf6cilfoeGTZMB4ptq+fO5jKDhrwh73fg8MEHQ248jeTWZsrUCQPDyaY1NJXS1z4miwvI8\\\\\\\/f+IXfUEJB\\\\\\\/Uy883RInviB2ZyeEx2Ymq59juemn6IxXpT+eO6QHFPhR5gTnfAMeLt+39HbzY6x6M30fGnoX5JdAS5TzkIsMeH7xFMEJukAQko5p1K+JUn+1k47WbUuUzsVHa3PonJ763rlMwmQUIEnkFes0z4qGjkUy23Eghw+Mo7ti3IsxLLx6I4ngPjS6eSaYuGeWY3hDY1wrPjcJXMPO7F0QvXWe3BP52js4oBeV1Ua7LSPsywpodlJ\\\\\\\/SOyw4fIYVOrIv9Fb+gWRGhNMAe36c0iQ8NutuNuqotnJZ82Omko0Zlcu\\\\\\\/mojXL91hKGb5Ortm0+oRUysfo0yF24MK0h1kkM6IPqTZHFqZ1P2J04g5GD7uJ\\\\\\\/U59Sy3oKEV1wJ6iWjmKsjWiP\\\\\\\/num74N4FOm6c3H\\\\\\\/HjMONLmvv23NAXReBq5blAf2JHY20bn7bgvl2BQk4zZTktxfN1ErBAEA9USR0+I4iJ52bxwGfbrhIKCeZzSKe5OcJ5wCO\\\\\\\/tBjGIRCcd0MK7o2rVpKzXJpux0W4PpthR\\\\\\\/KDiXVBO9VYXog6r+wnXu2UGx98VnFdVBbK6vLJMOSEdoz\\\\\\\/Qtn7Fn4OqtPfWT2GR3iXMjeEOZKHqkebSgsfIR7ZZaMSsp8zNguMHt56iuo6ReYqgIkKDM6waPUkX505qmE4VQhLytd74eokrhf1\\\\\\\/zi3bYQXPXrf6D0AZAfi6+VnFrWnubZYGvGoAIqjNlC70IhlpKEWpib8jjY3uMW\\\\\\\/pssG7RW0e1+Gfo\\\\\\\/qdxdnS7Xv9ELZuz51AZIuVJq6URY3Lbml6hQU9HNB4wkRYWnXzuxf\\\\\\\/k\\\\\\\/6SvzbDGVv6pgVw6qydolrG89FYLkDvIrzW+Z1aTalZMrljRDCOPGhB3AH0Onaspihk1nPcPnwi\\\\\\\/CsiXWEL\\\\\\\/ZyKyV3qJnttpwiEGa05rHrqwJNVZYqVcv0Ir2tqQWBeTp0MvYTPUe4gf5QD+Kvbmg\\\\\\\/waL4xdaf7kJS3P6h6oRLKD40J61R4l4adV5faR8nGBWCoNXecjdLYu02NzrmLnlhQUyPqc\\\\\\\/3J+BZMogxnPDlqUJPVkDAFNoRLCp58J34ZPV0pLf1IyX4Bw\\\\\\\/BI1ekVpARJ7GhNVFg9z\\\\\\\/yBQryMnu4Aju4B8RHKUPVC\\\\\\\/yOZm4sL5iwjTQ3Yn4Wq8aZ\\\\\\\/fJHNXCjn6Ri5H65XOcG16vl5RuKeNbxuKBbApRI9YOlQZxiFBQm86frZromXRYN6TshCj8fjA1EvmAanLHPwKg1ti8MpfsrArd\\\\\\\/1rIzp9BTBbnRMRXL9tEibqA6RvrkRgKhW\\\\\\\/hvNI9Qdo+Thh4k2bJgh24WKiRd6htgo3mieAJqTdihVvVJYo3yQfiQbqgFhnzqS4ZXpZDf2Tfbu2sO9t4eivAw0+A4RKnhjED3x1d82UXiU15+O52HZyYb0AdLiVF1XBiH95hKkYfsD4hIY1ZMHUWkYidVNIX5wCTYTnoTPq2Y02p26bc92Lw\\\\\\\/1V2ncn5u2Cktsa6V3MqeHpCPcBUv537LJ6EQYQ8Lj\\\\\\\/9solh6TVockjBTHggexeWiX2Pax+JuSOcovDnYEO6YAw7UPWMw2UGGZkVOMcsTgBsJqku6HtPpYMMjBj6\\\\\\\/Sj6u7IqTEDkvZCiitOavVmCNV\\\\\\\/sQQLm9sXZq1+1Vo7jEITZ8Td39QzLiYSa1AByqlSZsAr2wYdRJmV5eVHi75MVzeE5s4caezTBJHFzxas8Vnxp0T1phIxk+XmW6ECYuUUGnMj4kDM4DKIVueoCkQwLkH4AsU61ZB+wKoNeEimrzHVALr2lfsPpU98t3U\\\\\\\/NM2ec5Y2QJwzEO0WR0fMdoBBSKJaejbdW5dBz6kqw6BNwAGV4QkuEDmqOotKKHLYKl4xmZo4p58VyiMkZC8sPHesfQSVxBxQdN4tY6OcKOqsaRasUHXEM90BbpN21AWvBlEUTzXSLbuM2Xu8N94f5nVOhtJAjAIF4jv6vqUr4GjIgsk6XlGJmX+pmMcYYsNZJxBUrEzaTA9pwdkVWj5YgVHbr9LDmJb6METJuJi7njwyRc774Xf0iF\\\\\\\/zWk\\\\\\\/YBFlcqYf9gy4EXlSQFYOL2xlV3UeJkWr5tcPH20gSigcartE7QiZbyJkOHz6fyV+6zVecJFBup+YXByrDTcDaKpwoHISuCauSt8scZ7D+GDmosmpF31hMAs3MJIGKtifVbh6\\\\\\\/ixZMD7hDRpkdT8BHnrnKk5hfDhB9+MtjcD9\\\\\\\/K7O3w95AHfF8RjBiV9ofCcLd\\\\\\\/4WWaGOxb6jM6P9WfT6Avi\\\\\\\/4lV\\\\\\\/uN1ME6aD\\\\\\\/zKNwGOEfwXYbiWbo\\\\\\\/toOOC\\\\\\\/icQkXfu\\\\\\\/5Fnfdul9ABG\\\\\\\/+5BtR4Jcko49t7e\\\\\\\/0P6SFsRCbMFHVm2tOQLbZ7Og+t50o8oJnTppf38Tci4uYtYV\\\\\\\/seA5BfNfw76LKdaui9sfnPADx0AO7Ts3x82FDlsjPxdhsKXJzC2FETkQt9Eiz1XHTgq1yS294eA7SZmF1bikeZTjyVjI\\\\\\\/\\\\\\\/HyUqHg\\\\\\\/tvRnVpWjD0oLCf8xUpOtGTHAtyDn7C2VtifI6DCSb+tYRTvenwwW3XNKB5cYRz4eW8umfHZEaOxPVXiGTMkZ0819mTkK+2mq0SJRmpY50SuRpIoo4QmWoUA\\\\\\\/iei9hrGNClkvJsCIzvg2CJQ0jpLmwWjrjkssnlsoy9CgwmnbEys2ix1XDbVj90jjkI4yzDTas2IXtr8rUShKPT3kKSibHdm1w1E14UTUAsJZT9+pCjcmlHFhmreowHF+04Vn+oq86trew1Dt4p4GCt8oDCp3BBg6iFX3MCSMo6JqWHFLFMb0tsHzC+7\\\\\\\/zQdbV9lqRN2vWpz8YXw8HLSF83e4pKGH3KA0h6GecCtV\\\\\\\/fWqAFhLiXri7Hzasy\\\\\\\/OVfijuMiKSs8K5TciwOzO8OzdPuvj0rZ4UFJKA8e1uBfzYL2gcdOTpLfbsYfR7XgNFqTZOvv0YimzXDS6mmzvZgVF7razKRx3AGogeGkcqP3fyCqu63w+jYs1VYsxCKaZkT\\\\\\\/ji+YfmWjHxAUv89tJRJrlXSF3hzPc\\\\\\\/MWrKI8a4Vf4qNTdQfA+QaD61p+qmviMujWdA1Hv2DE3de37VSlxAm8vyumbk7Jn\\\\\\\/W2Wks1L3r+AIlSRnKgPRBNQeYvJOJZsGprHF4aUQyQ1gifVgO2JGb3cfN6mHUbQlp18DAwIdUuwa\\\\\\\/yxnOApf2F7yDWkQH5us51Ed\\\\\\\/Dlv1LAZx3PeXA0bHLHPTYVaykgNchLRVtfociEDfZbFPNP6pA0kCtrnFoVa0i7feU1fZC4y+btgBotNceZwtc3a5E0yxr2WgVGWKV4DpBvY3ZSQvfDJOD1no+7mniP3OJli8WxN\\\\\\\/xuIkrvnTQprt+l8pwHsaJe6chJ9hF37myFKM4Uv\\\\\\\/cNWzZdxf98nVov+KqwQIf+yjnjJYZR93PZ8\\\\\\\/6d5x\\\\\\\/8ncyPgrh4WB99\\\\\\\/w9vdDLN\\\\\\\/w86wRULijtHv0sqX9L\\\\\\\/N3y8gGqS0XSunUi\\\\\\\/Ed4CA3x9Nc7rwaAyWFAF1uf4R5wQQeqfRO31sVgMxSbqmfLWaRlfKBg5MYH3uZo7LLCOn5YELRzr1DZTXm8C8+WYW53VHid8+SOkzEIy5jirj\\\\\\\/09oilIp5cyxxsB62lD2jw9LWENy8+GxfBbwzD8Ayo04hRn3f4NO6qEKJK1ezVJvIC6PDDPKn2IIuTl675pFevJGIY6\\\\\\\/0Nd6jGZ0g9SLbsjc68BR3ZfERAGqMrEmQK0fWip3G0PSnuvViWqFbYM1LE9K\\\\\\\/JX2Mg5SECNZKKWAUuE1VuhsrLF+b8UbJO7SdkdioD2ZprdpKMQStXM7M088LLniiwwif+8HX1n+eUB3LWWbNFbB71n4gF5EYk1PVbgbEsFKwad+2iQucb+e+gVmjM+Q9xOOGasCdTaTrRl5Gi3yXUPSp+W2tg0ERERYVRVs2HAM4qhTnCIg+P1ye+5FMtol8KWUmjo2WRiyjUrnRkpXsQuy3xaEr8x5YNZ2ijLqrGJviUt6Xjz1zLQonHMl\\\\\\\/vlwBTzhRecRl7UBtgATUd74G6Qhnlxern+D1pSCqkj7bolUPbZytzhbgo7DAJs6YdfPGtpJ9YyqZa6GKRhL0eTnhG7EhhyJmhEEdGbQKXoffbUZVsNMlcr3x\\\\\\\/EbAFK8NAwV05qKvz7VR6lbJjjJjDlAEYEreM\\\\\\\/tAEE0WjH2p8Tb7bDVd7GRrKKILW6U4CJMc72gDGaCYm3N1O3coiWhUGuCyRgKtGET87I5WdGJc379wfENnsVKHGnj4SQ17ffIybYJfx6pkQTBzSfBhppgJ0XiJC4q5DJBKYOFIBYYnO8CTWTaO2eKRyjbEf2p6a5T+PXRS0AJMiju0gGnpnJLsZMc\\\\\\\/DN11qMZhUTEUE75AHPwgQuEODfIrFPK8cy4qyDYoYB3oq8fuhbVmxvtwWV571HBVDszxA8eq4LmIxiG2IAccWdbVT6EUNMxc0RGUQRxNn363\\\\\\\/Qh3CM4Qs1RaAFcsBnEZFF0H\\\\\\\/tCyu4Dvuo4iF2GDyVtw7QSpBTc4Qr15YwY\\\\\\\/69Y6zooPI31MnszDtbp2inq2FYcBYZ9RdTJ+n3jj7TjwidC901j6JxvCUUcIFVmJEvUCnMbUI5bkJR04jyR6lHba9S5+M+ScY5Qtb3U5VkEVayYKpcDR5eDP6oK2NCn9ADtSxEQAG987zl6xeKmUc7ESIGpPyfg7WcBp5GkA0DQ5nOaWHqxLafluX40OYVZf35VRI7XiB2Ahz41q9A29l+VjjdCDj03jb\\\\\\\/TJp1uAs6h9NbM0KOpsXjLSSmieD2QUpGnASOO\\\\\\\/\\\\\\\/ms4K8DI97Rc3vUJKmJ2Natk\\\\\\\/GpQsi9yuINS6CXHxvyVuZ7+EOGepUW8VhJ3y5rdyW409XEH6kLL1pzl85iHeUEoBTZVIa27BZS0\\\\\\\/Tvmt4A51SFPBtK5FXgVm2V46pfwG\\\\\\\/Cnm52tIPcEgPpbki8KEozfyzc\\\\\\\/aws2y93v+SmEVfNu3bboBkaq31KVrD9zaZrlhoNImHIu\\\\\\\/krHMt1mY8zEDvh2eUeYEaZ0zmzQUY6ufKV9Bher6tGZxnI6RH3Snk3Gn\\\\\\\/wEZjBtnbSJbU\\\\\\\/tO593FTOfaR6jrVVhAQlx3uzaSGfyrYWmMVpGI7Un19kFGqgrf0Nlk3Ge2Lqu5P4j4IwVGxhwNABuhf40NSH6djyQC+LGbZV9\\\\\\\/6yN+sQ2winp5DZMAKaDBDFkDXJpyP3CcLRmuKQfAHdLoBAywQZLSdAmmPv1QHa8M2EXSHqP6kSX1\\\\\\\/rnL\\\\\\\/yZa9EjkMV9AasYzehvf7RUNCwSWo1ggmV8Zlkz1whTc9UZL+Dus+4V3OrQhqjSA7zczab8DQK\\\\\\\/Q7obHmeMCTyk7ByMQev3QpEZYzqrLsHzdcs+3DnWn0IzoLuhFFRD+4IQWhudw+KPbpGYlIZRc4pR1NA8YLswmLFeVRRQW\\\\\\\/eU5CUnjkK\\\\\\\/6YpMWp5\\\\\\\/BmwCIRwAH4dki0XTf7L5ewPABrnh\\\\\\\/KtmREYIwXimmUl35QfbidCJEACdzZX368gLVsDcKuMokfjEZq1ldfafXgoRXMDsh\\\\\\\/+4UvEgfvYzUUUUvrpfNxgYvujY6yEIfBCXGa6Lci5BIeB\\\\\\\/5PWmjrKVkH\\\\\\\/QYyPSNaofTW9bt4H7oPCQ7c5XrxJlIm3C5LfOvRX7mX77yz1tIGBQxtRHHKTnklbIN5qxiCqePeDXyUUdFN7OCEsFGQCKHg8nd\\\\\\\/OP33wzi\\\\\\\/cNy5KZgU8kc993ypbtcPOKgJxssnUHTGonu+aANTSPvulSuWJuO7tbpJnbb0Udz2y0bcPRfcM0ofDZsXJ1KMJ7cU+aC97whHKpG0mH0l4L2BU0uGGKg9Bqqh6Sj2furgfjgwGlVrbEPGS1MvknVubswE1nt8SUqrbL1tGG22xRPFbcFhQ\\\\\\\/ZUR6XHxNZ8fhNITx94\\\\\\\/\\\\\\\/WvdLRFqlBs2upUq1I57uUhQb25FCklyY1E+JKogJZaEPw145zHhjq0Pl1Z4qpvsYu6+7BQ8zwn4Edcw6jfPM5uAp8vVGsWTD1TVtdxz8jkkahEhEfkNoOOQW7Mrp3k14zv0P\\\\\\\/FYxAdzfX9KCWUMDYFPTShpTORrvnDSLbA3tuafGkgd3UpaTjEV1xv2RMAPnIg5vbi+qucqAiuoBXBV4w0LyI2n8g6BaPPSdl60DOcbc\\\\\\\/0AtlGvoF8BYVH\\\\\\\/v\\\\\\\/S0XOxQXbxRxyIrVuGR5p\\\\\\\/voOejrWnPwFW1uBRxIjfh4j+nWMkq\\\\\\\/0z0Ss3rMNhWoG9L+QHerFZkzTDP87rX8sfdZu+N2ClGfAizp5hFkvDhwQDuPiz2UOY6P956pIpR5WFTXfp4SMzZdtUeTNbEtPw6x3ZF3o1WZuCVkjrpSbIs1gL8ktA\\\\\\\/prMF+wwoiqQUBDnqnQtuhFpnwgei4yNNhd73WtxZEaziY0O9WGyDilf4EMgDG6lgdXCU3Ok\\\\\\\/jYltOvDaikMxbTxco\\\\\\\/xdzf+tBVY9MS0ziu2aT6TmN4EdozO4V+jCjtSEfNiiDf4na\\\\\\\/UvBwBZeEAParuohUupfiSTjZc7RBdvkmFADrHj5Eo4T9ADxy8GtEdYLXQ9H2RZS6pR3WIAcNVx9p8+ZZXVgrVB6d75vO61PB3+cvdqORYcS6+F64WzBT85mB9hN6kEIe1EnW4MN0MA1ryUUBKOZBeN0Trj5vygT6RqisLmXK+qThGPOKMius0WG0vyH2iM\\\\\\\/sXSLc+d4XSqe8NyE3MGnUH02Hlvonc0P+RvThNiD4IHr2phYS8DUQottNZyU2JTK38dTo7vv2CHL+X5wSncXBlXyX1WWEBWdfPnPx3mgyL9ECFEwqDy3l51xejJz477sQdLeDNqlWKAoFk2pIdc8EbQ+0cXw9bscOAkLiipGruG02BQKbn5uS645PJ6TTGtxNfABi6MpEe1lyFfCo50ncG8TwrJTInnSfILFhPLIrqOe2HBOsU9Iy\\\\\\\/OwvVAoIV6DEMfHi1KH8WE51b4X7OChTWeOsbmL8k7l1EF2VpCGCRIYEzQH6JSkMfjm7rIQQtewKpY6eTm6dFXlDaK7p5HEBJh1qC+ECVqWdxI36Fnhr+7BdZSufz8n1o1Kv6YJCccHCbxRgOyUXqkA1RctEefrY7tvo+amsfdhQ8u1HAIfEtePpSpexlcf4quaaEyp9hGGo13Yfcwux4I6B\\\\\\\/o4aMkqFk5pnb6JR2NEegODMgAVEIFHD\\\\\\\/CLFgMykBYx9TB\\\\\\\/g8\\\\\\\/gdEdAPlEMT2dT5smAX\\\\\\\/fQAi\\\\\\\/XtYr0H7Dk6V0AsWdatnfjynyuKxA\\\\\\\/KHEiQHvKhhqEk1AKrHJytEB+gfftMDzJBNbZzQhnH6KMMv7k136b8qyHvxmQi19mrtica\\\\\\\/wXg3ObQh2TYs5rw8OXCFREa+ZWYRr3pwXidnyffWFmWAcXvSZMWdMQRFJgd2Uwir+72C\\\\\\\/YNkXLTAzUzkKEodRAiDiNyWe+7q3MoNefdvsmDLjdcPlZqA1jP7Z3D7r7A5mt3yPlLGZ9sQi1ndpqtxLXM7tLJZhrztQMKQ\\\\\\\/2CwbhXZy6UBxWffB2TTwYzB8XGoNBf1w8qk9i2Mxmg0HbyI7CdyQf1jKQpWy\\\\\\\/xCM7kdajddTCYRlH7EvpQR60asQ+FW1GVC7N7HLy7qUHB+B1FCmS22AwVGM9RWhXVUdDv\\\\\\\/MSP9hlKJXjuAmMgl7buR+\\\\\\\/ccyxU4\\\\\\\/stHUDb1HdFid1l0OZg0FUfx7K1KVOsluxGjnyWx9jYrQR12UhgNmoL3z5+exNmlPeohRLDl1wSUHlEA4YDh\\\\\\\/ScX1TrTC3Js9sTTwdx24u6I9YuMIRBRKPzILJiY6CvBwah3AeIjuWssAumOcSYCKfHnxTJGnYfpp4eM\\\\\\\/H3aXHx+OpEuGAVc50j\\\\\\\/5XAdVsQcfjl\\\\\\\/mEeqtKpDGuQTq7B33db5O+ra\\\\\\\/ugLrBzBfDfGW9+yb9e0WHAgesQzjBEcrbEo4L4hxZTHi8PqzXQ1w+uea4lp28Q1q5rjkIu2xSFAW0BPsuTClBrfy2R1uPxaJob8W\\\\\\\/nYO4lWdEeyGgMVykn2KDAgXtx9DmjfbRz3KOwodC++VyJAA6DtXZgBXqkMxObfNpnFzuYyTpbNfE6YJ50pcYTFOZccijCol7dHXKG+HvszSlPRmv+v9w4Wtc44gYoiow6Hoa79EXLaDgEp10eFIn2Tu7MA\\\\\\\/eSq+odc1ZUjMy4e5FQlty\\\\\\\/OEgYUDHEetmg3+u6e+4oUia1VQDclIuonPSBUlBk4HC7zOU0NAxXyzu\\\\\\\/ZciPoOTHrkoPKt+2GhvoB9deLX4JzBrrfciciD9Gs6t+aKME8HFaSo0jMsIu+Kp\\\\\\\/6sPQ9A0j6202huCQYvDuPrKSJONjuHZX\\\\\\\/kqJXHXPpmlKgydLdhG557RB4gYbOBdxTEmqchkasAqTPAebLS9GyHDC+AKOjBCM+vaWKKis4oQMLaOKixVzk59dvhlu9M76AtmqzwD\\\\\\\/JfGDGXgY9LF7rrWD6zqQfAKSBRTrtTDFvDaxPBmkr\\\\\\\/qIcu0Z9Zfc8DoDt4y2AZ1iyYMfzSsTduho2HBnUDjqYrMGrqV51PpVJyoT\\\\\\\/G9kVOYi4+OnJFTpBcxTu6ljTP9tqqwx05Qsax+2IllDy\\\\\\\/mUCftyNKIJrNZH0DjQc47e5\\\\\\\/ehrcxYVSAGvutiP8\\\\\\\/78zxvS8zr+8dkELP9+4WAJlKQqIVkjM2gIc+bsE\\\\\\\/WDHCQMdDb5ZEChWxSEDfDRIHptqgLA+vXptYTAOsQPvYU8IFEy7iR\\\\\\\/8jbTD778STMckVZR0jkosHKSFD+Ym3tbqE1bmuIfPwjiiE\\\\\\\/UU4ISc4dgdG8HsmqKNlp4rlr4bXdkRzIbFkNwP8P\\\\\\\/S6DxrDLhU7qdvLCf9Fmid9aTd6GpMURBGWAkHpc2Is1fG23S8gP+s\\\\\\\/5Kx6HTPxlw3FWjJOYJISUWZAMd4Y2hG6WR66LpcF5d9NVbMx3+HHoR\\\\\\\/VW++YwqNQesWYFxLY4Ot0h766Mf27tspQeHNmKQTG\\\\\\\/3R97J1qeBZ9LAkMKgz8w\\\\\\\/SboEjGN8mbZuAj8xNZOlYtVV8WKXMKvX0ap1ZL8ONNceN3iButMsGCFxDQvzt0SS\\\\\\\/+Gxreruv1SXjpmo6oGHPXS8z1RZ0z3mRheXE6roTXM1zGv7XgiaWUXAgIiamHUpjLWUuGhvX7W4VSt6M0rrPqQS15Ued\\\\\\\/eXxFi5FIAi5NQ0P4jsA6JgC4iOqdi\\\\\\\/3TQc9bQ2rpLyEewqCgvd\\\\\\\/Ebml0fntipI4knOKYNwJfjEl5DF8ITKSCMOcucgDtLmtt+fGSqnfWBOfGxRH320GQg2\\\\\\\/R3QUao5BqZCxBCG8fLyAOgc4uvDVRLqzTxN7g2ji9LPhLSP2Ab+W1HYfn9eBjPY4Xfe4jPkX1MjdH0EZgDpayaP95PG+b2l71RRAzbAPHnyOG65mZI31\\\\\\\/e5+hiwil63B7V68lkZpC15s4nP901nzutikhlA0Dh1izPH73qjDs5\\\\\\\/wDGV5L0cBumhxgmczZmV\\\\\\\/ZnFNiKAit3UcsjtTmroOL9fytWwbkwKYp1viZxerK\\\\\\\/vutfveptd6ORWet8Xf\\\\\\\/F0nhzfIMbd58A\\\\\\\/KaJY\\\\\\\/omjZ6sC3WF41dAwQOkRRsQTcKvFXeT6Dn81xuhdPuPpnCKA996vMdhijVZ5KBGMGk8svj5xmSkYPcFrwVYjRyj0Vh1EcstZ2UkZESo9FBBsibsKUY+ze5PhrDhhGEYj2w01ANvoAzTrVE540RnPXKGx\\\\\\\/TbwMdTxuVReTN7nw8qxib\\\\\\\/ssI4aIBYQJ6\\\\\\\/3EqxsfLKV2cIyOVy18ySP+axWGYOe9Do3BaV0tKEr8HePr2Jm3IUCA9sm65V5O4K4Kz3BxBB7DYQMByiN3VWWA0P3KKHqmsNhQfcb4C6FFz7hDL6L4xx8edaKak5fr82BcnAxgzlNBYeRlJHdiheYuNv9xiBGkDZLGKvA4ki+HYadI1y4m9\\\\\\\/NA7gV7PXGfOA07kKdwZUOnTTbR2EJTBmKQVqOTzQQbXBi6KyICwlDVucBNShnIbVBbOiapi8OyhASug1t94sAkhAp26NMgQ2mtEe\\\\\\\/uhDwg7wFCx3MFHgE4yYJ9sQVEbp+Th50g4uIGq8KUZgLPDU9DYTUcpfcfO\\\\\\\/Cyt8CvDRfA0gEuVOnEDOr3wIZYQ3x4l28EWYw7FuG8Me+ZxLEoclLzg7UB7IPNk8t\\\\\\\/qcmkiA12dKfEUZcDpahTnPU02hAWLYnuLtjPc8Z+VyrbpEIq2Fl8K23SzI1rDJ12BOEwmMXcKouTQy3feDKN3fIBJL4d8Ybzny\\\\\\\/Y99Y9tU8eLS61q9GURoLCikPuwaVsnuio9ui0Iwgz0qHpqqpOyHCUpzbuNRMmjAOtWelvLtanviRSEZuVXcCfJbr9\\\\\\\/mATldKXvgVwTtbcQ33OgvQVMbCG6o\\\\\\\/1uXstxDGf+pughYrADJMd4DBeGEBLDnD0EoRTfINqhvzOUdUlD9ZXInKo4mo+js9pgT0OgO3sTwblQ7ZRREQsx\\\\\\\/4zvZ1k3F5RCoQeMDwiTfwigqb6ZbY4eRqhsveyziXkjf8S5hYQ4BKxbCo2RH+DIpx1REwV\\\\\\\/Hc4jv0aTy6ytNF1+J3XjXTdLcipJFLgIJNZ6+L0s4fDK4hXLegLz9a0d3u7v\\\\\\\/s4T1BEPn74vUVOHmRnaZnOPswyoKxVEqC1gRfeNHc0wFomhNONYURhvE8uDwfoHGLzURx\\\\\\\/8s+\\\\\\\/J2FPpp1pJkPaZZwBuYBlqGZz2J8yXf1HRHGQMbCPh7nViFvsPRuyU3tKkYFNt8lAcVeSfOrlBdCydEvF8gE4XAHP61rbR3hvLeyEFGMbQhHttrgu4XxeLdyEPRIVfRvdJJS4vasKfIi\\\\\\\/Z5lEpxebSLDziN1E7Gt2OGjk0V8HG96YnzgyVxc7LeBGPEDAl5nToR1EqfCarb1uwFC2TbVK2i9fkb1X+ZTdSL6OrlhVjAX7ptVtPekLVs99R7XOaa702q8kW09+oq5iqAxRZ8kSI1\\\\\\\/zoQyKVH1I78GKFlTBcNfIqmt\\\\\\\/+4\\\\\\\/o2LCz0RLKY8CUvUrtKffYBBFHoEdvWl3N7PwYbVKmw1\\\\\\\/9p\\\\\\\/Vtx1sfcjiP5Q0S0F6pgGGC1exLvXtJykJLK8rGSuvSRtmPnC7\\\\\\\/RiE5IsRVRUtirNDx7ObPlHwje7W5wwc9nYRCY8AgHcIvmHyFhQmTutiVwk5f91RHT8ENax4r2i1F2JBwAQp\\\\\\\/WzBtbP91xS\\\\\\\/lxjB1JwwpwRHctX82LwP\\\\\\\/BgSiPhZBsMj5PxeLyZyslFQ4hVesOdx10ZX70qiI42\\\\\\\/oZYkg8WXoA9\\\\\\\/jkYUL+nbWnjJMdgl3qg4yVpjvOoIOxGPejaTtM3CGgbFq42Xkz7wRPY\\\\\\\/nKisUEdOrGQdifYJr96LVYNVdFN2M6FZDetww+paoBcT3UP6guVhKcvjwjtpI\\\\\\\/Ndv+AIIYWIlU5yadMyLPCP33tkt3nWt7W6uMKcy894wr6Ru2dmG0ItEarShsnqkQZ0BBVKPF2qEByxcGaiFIPouiQYacom66yFey5riNEeQH2Ib\\\\\\\/LVWmmJN9qq7THCjwhwEF96R2KFhgOdRA1\\\\\\\/54tZBUDzsw4hOStols7wM9si+BWIhvYsKrQbXqhCn48ptlNQPmZoR6oGOWUFawz\\\\\\\/qUFbQaZdBxlTGJ\\\\\\\/wrnafNUT0Vmzbn4GpriUaILP97LezD+5FVK3KrRZMwyAPXlBTrJQvr7PLa2LAQg4xfF2LQ11nanpUqUjGqUK2zKA9abIMI3QOsjOtUUlXYv\\\\\\\/vi4pT1+4SZYm7PWR09vt6l7cZ6Z2P45ORFRYGoEySUZDgg6CXe+w8kRh1LGrj\\\\\\\/VRcu5u9ODvR5paQZyzj1pTr1AE93ktiH2ahvKNYxwK98FDZnjLERyF1HwduEOk6irULo3oUGIuxzDikInKSgxvP13oeNmnRNp\\\\\\\/SnmDkJ5AJE96sL5PUGVhkxwE0ilBxQtebJ1ZevOQsPBYSL5ljhNH+ZtAQICfmZNrlmJwnN+di6\\\\\\\/0u8D1NJEMgSDiytOSAR1+\\\\\\\/uLPmPc4EmEwH6fKr\\\\\\\/Hc50lwp0ukfJcsZyyZElxjD2czh0k1HingsKYQB7j52jqcvS3o9y09SB0+UgSdVq4E\\\\\\\/ghFPwNL1whIAKfQNxVFH7Ig1SwroPFIzwlU+8woR2XO2LqAknLL92J8VxMI0kEtSiwCUgIo6vFflBLiPlzeR5iAkuYMxahrF5fWTeHhGa7kw6ZGeRWN7xxSsU0eextP8V5zO\\\\\\\/wFHdGWqrHv8LGb6GCToAV9BNYifLp2XvUgn\\\\\\\/lAhXI8S692qdTAnt2N1Wo7KLmLkbmly5TM9A2gond5NsGcanwk5r56V2x\\\\\\\/acP5Sya363NEtm9e33rmhMCdVnGk8WANT4SIhmx6Lk6sPBGAjGoDRUswNFqyaCxTPBTGvYoM8BZBGTCte76JICPPCJ9jD9WzQ+Au8TSF8tZyn\\\\\\\/eNTESSbkXYv+6tr8MRQ3HnnPrcZn56ovkGp3TrtJAaATQ24N5BtCmLXj86MCujbg2hryzZIP2D8MHI35eZ+GAJxEKoj18jxOV7KFtukn2dTW2ZLNJnAAA3Hvfc36qhmxW11gEXlPkWValJNm78JNog8aAdjhL185YKFuEvkgPyKvW4h11M3dpOvYe7Y4Tjg1H97C0XQ5cUq2BDYhkgKtNbgsXrV3Qpc8FWqbvqdCFLsUUeMxxikc0K4JoMk5lDGclg2URmTNpt45NmZzf+ux7MaflJVahzW3sCbtPvdqKZtvqM4CeM4+QjM1vw4rDjlnw1WUA6yOnnzGHO2L1n9q22\\\\\\\/1ff4PhyMdmEj2I1pO5dJsZ+KjXQrFa\\\\\\\/DfnL4\\\\\\\/2uMIyxKG6VaEl+gmIijqa1UpRLvsgUt4FzJWGHZJ7lcPHThrPE4zIgIxGJ8JvpNxmEECYJZiAF+deMfYQ\\\\\\\/Dfj5CLIDqyY6tLma3upxJb04aeutyIF9pP5nSYrK6WjjXOUOXa7bIvi17NZaJ+UYi8Ej5qdLjURsyIY2N200Kj7auYh18vI2dxKPxidqVmSD5JJMyX4ZnhNiMiG1n\\\\\\\/xGvjT\\\\\\\/rqbPQIkIVRmPiEcnwSigPEro\\\\\\\/Vnw8EmhocGxMDO29K9khzZf2Xinmg+ynZizs++evZCOyukbpCq+bJTWH\\\\\\\/gIWZMxG7guXAGPj2r2\\\\\\\/mNUlmG8DmRxyoddiGMrKCQ7DaGaVb51BmT47NbT+frrOw5kr\\\\\\\/xsVE7iAQQeMO6YeZ8eLIjlKF9uUXCDShtgOyp+teEvJVbofkggQFPFSZxmGd3a14Kfmn++hqu2tyPEVIQs37uTjhkrchFJu\\\\\\\/qO0MrDPxEwQfCYw5mOJeQyHIH6HmcNwV2UsJ9l3pOH7JmEmeDdhYrI313P7NFVuWO2vZchD8zfCT8RdVceX6itMEfNagmNm36VuFljdARAZ0oZGKKR+nHGO4ECUDrl6mIoColHST7khGUHp47mj8KZLW4Qk4gPF9DrPycM6W5VKhrgeFbN0PK0Vwu+07C617dbhFsOrKBi34rf7bYT8tQJrNIhWHXhwhA\\\\\\\/a4y7ubiwkuIvDxvFUb+wq+9CNCGHGelzeGJCTgyk8qEw9kIob9Xle7r4dflEbPf8H2r\\\\\\\/M0do8HpKGGwl2mUUeT8As6iC7R+fSnIHvG31C1zdOv6oBOqpEYqcDjJn30IwleqG4EdDFDt+u0et9PSTlT09y2m9ouCBQdQGtsX3kOukiF+B73xoikAZyhy1UycroBP0A0SPix2aMUl3fEiGE1VqAXaEA3HIcH+kqQVFwZCh5i5rxHSvTJH0LPFr0u5hy0iniIt077sLK10ppfzFQMbQQRtyFoNnH6OwD0wX9DTkALVcAZQ7yPWLeUrPfdk78Iyy9BxD9QZ42x53alJ3RKAceExI5OOBs3rf3e60JsZv4BO0pGNC1YvRioV0bvEEcKdiLsQRqVXyRSgIowYM0FhVTGoobx8YAACc7OjOSZWeK\\\\\\\/s9Nnv9Y880Soa908B4U99ME5A22hOwThSdxWOdYHsTKeH6lQG6BwON2klSca9Gyq2hfavd+zGeq6nJKTJw6xr+xftY0BZ5saxLG8QX4Pwj\\\\\\\/Br1nW3Yx0XfhEcDbfGwOAbOFtTasLtSnOLqujfjbLAUL8ljGwRhZzAy7L1ThH1WQf46IKrNVLqlyrpoYGdnD0M4U4kwSGwlh\\\\\\\/gxFqdhhdWDSxq4ngFpALxlCQCPBRu+bekXK1MPffLyI0g8MHSp\\\\\\\/TpqChhYpKEzlMGruCSIh9n2ggAVma\\\\\\\/baoGi3A8ossF86oJrrR\\\\\\\/5dwXBKbVrd0A+4dYVYtEGhPUMA\\\\\\\/IfK6nsRnTu1jS8B6YU5t2WZ76VwoP+A+P6gqwGI0NsUfy3pMRkSowEVtjAcr0BOWMbg8En2IMSKGr\\\\\\\/1wEI9auoz7iecvkPrQt5Bgc9vXeWsjAgJNDvQ93RFTnq\\\\\\\/FCtngJHOUp5fmv7ssvWQzrQUAnZlOvDgqNnOJo8b7Kqr+adOSDeZPz8BEQdngZiRTwA5V1bk\\\\\\\/yhG7VABZlOMi9G1z1\\\\\\\/rDYCdVZhi7f+PK8mffFY7zdy7qeeDSPC\\\\\\\/DLx0GpicHWkonB82UCA2A0sYxt2Rh5tWjK9sZ8TUvJV7joD30foIKm\\\\\\\/mOrwwew0AAb4c8JoiOz6kC0buk6+M7Li7D2xlsciqOfXD\\\\\\\/wLsx8yyjGANNIv5ZrQ0EqNnmf4T2DLqgMpMD2KIh98LyCM2AJzV3CQKq8W4BER0wHulhcgag3Zkno7zqGlfbvtRNrwPaTCiXzTcVbfwnZkqRkJ3YQCyGkLrxeaCrK7rK6hAo48H+spvTTAn1gKuHpmhHzTiICZ85\\\\\\\/whULljpS69Yy4xDHx9BtQOBD+W\\\\\\\/Q45IKzdRI1RvySFC68CLGbpGmAhdLRKJYS8qaJE6Nc\\\\\\\/f8dkRRd7ZBlp9NquZe1mHqxiWk8Cu7Tvd0U4xc0TM5iZHrIDDCGk0ToY6DjIAjHRBW8AAlQiIC70y7bfl4UlIb+JfY0EKuU\\\\\\\/jVFxuO\\\\\\\/BStfhLEGdmw8loIOzczJk5iYj+c+i5WKPMEsY5DziqF8dFJPJF5vEjEfo0GDKhYB8scKwhjy1GTK3BxzOYLDFQFSB4g+ZaovU6XRUaYJwqdqosOGvknDwffrD8E866HZOZNXqco1i3wFY14wA335O+r9f\\\\\\\/tCmelQkpJAUdAGUPsHelaFHwQIoYAk6hts1D1QrXt+qJHQ6vOPH47zyhtOk\\\\\\\/ITKvedhnnHzYxsUJVOLPzvANjI0N9FB0L9GI8gmUYBrkIoV4ri8JYStmhgEya88orudAeOJ9fBgak+xG4qkBMpTsZtmHGWLyNhy38Qsrus4GEdj13ugVo0ZYYz8dzw0XSisrZ6k+3ofbQVd6XBS6mYw3NVkNRxmMmUzRKLr1v+qPM8dZD1UcioZAqSvjdM+VPezYxMM4Sbdq366Jvrfd+pa22mQwiKlRlOnQ4vvdLETvXhfmu20IR+8s9YSAX8QnYX74WC777ZTe68ySbrKWYxGP8+gOmWERMYc+C0\\\\\\\/Ted1fklSf1GDdDciHWDviaYFVR\\\\\\\/F2U28GdOQxhlQzPcxsLrBExdW57ggULdF8BeGE7DB6f0uncj0gz6iAF\\\\\\\/o5e6lYzXhLAXLxygayb7NlClg1trgyPIkQuwGgLLnvOb8aMlZk1hPo7NpzEQ+6curuEpyEpmmmYx01y2UXPsF4e61GUolIlA\\\\\\\/zuRg6c5h8WeJoE8rjtheDeb+M2cz2PUq7VDrTTFuzU8m7VT4lTxQdRmz3SUTbSHgFFsVIA+eBBUC9EVVZ4LeunoohDrZav8zR7T4u2s6\\\\\\\/xm28me1RUY9P7OpyaeVQg7hiEdV6hRJiD5vWWEJtKkRMMXg6Xv5pidHwgNYQuRwh6Y5ID3+FQFDD6LGqw9YxfaWsyVuSFdg9ULxxYyFzPWhqLSYeYMq9mL9cMoFGq0VhdMHXnOLKwbgey3zt+ShTbE0GqR5GPwknTBeRyiykCLWlXf9J2NCnBcqPLvrwJfXn\\\\\\\/TZKDqZnL15RLzvOHCCvEm32il1w57UyNGzOVXMhwzdyH4eLd7jXWCbrKVc32UtfLYWj22zmPnrVWTMK4eczoVFt2R9IxoWuCb3kV2DeyKyl0YnXE6hzCQYssu4NPiq9jMa0VuoDdqnQ\\\\\\\/7zKAr97XM50a9Ag5xY6yY611RFi6iptFVJEnN\\\\\\\/N2Tlk6Odh\\\\\\\/7UFaZPYX8DqBDWarzXq4NTxliGWYJsno9VQsuVWfx07Oqn4sxkC9TawQVP8nmnAymcbshcPtJiIFCz\\\\\\\/tv+ucqO4xonWBo0JqAL4fPds1qyZ1rmTkwi+u4nn9Q8jEbEmnLzaJOAtdnWIaZrtFNFfGodsHlKqGfVDSQH7JycX03YY3nYRrzAvdaFPfWFtTCuh5EdcB\\\\\\\/fsCjgeM42UW3KS5eR2l1W8CIrDvRViC\\\\\\\/RbiL25oomqHKSy5NyXliLhj72qSMPClVTtXNi09CiAu9a\\\\\\\/Vw70A5NIyPFlT8s8O01k360LpXsDw5t3+3d8u19UKtUuXn09uz2hGVpHpAHqLbO77DdOD9v9Ty2mM2TaMiJRE\\\\\\\/NqcufQQbLTWU8k9Pua27XWh\\\\\\\/aEvU96PhjWYftK8LVsJKmYd5hQQP9IeA9hysWVH5npfQQkDEIhKWYQ3iQ8x7Hz9Xcd2xVfLFHb4XeKvhqb273x5Z1Ae9Qy86iSAxgRj\\\\\\\/KDnpllJHiXfRFZlK60qgcYBPp14UTAX5xqAISQWOlWTp4j27B1p+HZtTUp\\\\\\\/xOe4C00KGnxrrb\\\\\\\/DGevHQX79nb25k2+RUMuHNCUDz4LCodq45cygOw1IB27y9K2O2cLBlf6z5dVv7FtdKap\\\\\\\/Ms0slX1Nzfmv3\\\\\\\/xaztT17wjsKWhtWaWQBXCk7Jn6t2SM9yu7mhzx6O0tBY+38B4lSiuDiMVMpMe3sTAG\\\\\\\/b8dwGKKem+HVfzG0sogVBRUc\\\\\\\/lmkix8ktL\\\\\\\/8PqD6F5hAB1ILZDRqEzVTEE1xOsxvwDL8gAK5aXiXqeqhbviXvjWkkgqm1sa5oHf2FhTWOB6dxSKj0J4J0RkFNJmTaWyZb47\\\\\\\/hnh6f5eDLa6whFP\\\\\\\/3\\\\\\\/t5yHqlISWl90Mg7lTQpmAHb1tRI7\\\\\\\/Pv4FAbObybGnqTkY4uzDU71UZYsaEeeW6CfbqekKT8K5WGok\\\\\\\/6VkQ322O2Tkf2KZPp5JiA9XGE6XZS7iDpZR7AmElveObRTjlIcBxPD2G5K4Kl5ZQuEjqk3H2OMH+ToCgREpMEmgO1L1pmvI3EYmKvNG3dByVdImeamDTUNEtFJ5LzmjptLa3BaYv4mQFNUmYKnucBpNWiW12FTfhYJG0dsFMFH7nMgIqJDQpveh5S3NvvVaHZXj16yMkwOxT2O\\\\\\\/iFIlsqGwc4lbeZAPrzY+If7RUnqWYbRtETjXK2pVyUm3SBFynWozYtA7dsQOwCP5UlZ1XF6B3WGuBh8Bzh8i1F991365GsqvsOfBVh9QY6lS7CVwqBNv286LM8Cb+wX7bK+k51rXbd4EPadN2kQEDQp9G32vtWNqObcQdahRYg\\\\\\\/+ewvJhF9fq3TDVgFhJLsUCjxxtHT9iAsO5tKv8pjXb1rd4AByfqhTGeRGZtEYpNrNHcDcDeXX9Q4hnokxvNnMXLCq3acG35KgX7m+O6c+OEIRtfjVA3RPj5lSP6+6boNAgOiwT65zHwQT1ny2HZCVedizO8HlzzVXdz6SR0m87RMh4TwMM6EQCvh2KO809w4DFxo0US4sU6mN1LiB\\\\\\\/oMCd1yG4R8YsUvBCR3vudDLPYcBJhGULWm12vk7RlPXXR+5Qzram+0VMbypoDiRasTTfHhKBXDmIWxZaHUKLgi6SVuxtX5Yx7Mg\\\\\\\/Ql+4DXFI1EVs2n1X2bmXDQxz1HdldkldAFUzpryQ61SzNacYh\\\\\\\/zdpgqVxMDXnvmEVP5Lq9g09X2LnexoHONCyMflH2ln5D1LZ3htFKekv1ilHLJgkFdd6zOc4beDWghg6g8erbXlYM47IU2PzcFHEIRykv7W1La2PyUh5qLNX+korH2vbGdQlApGEENMY090zMsVKXB7wEqfWAVuNAR1+UahuLo5A7ccb1apH48vmH3K+qtAN+WvkE3OkFZlkCYZ5s\\\\\\\/C2BWY9jpHKJR8iJeGAibOfsDX30ZUFzfTjOUSIp5ZTgRwiXJea18m5Hp72C9NE4mJ2uh+QgYW4CLUjfGeMT8d\\\\\\\/m2ahRZnMp+qzCP7wUp+xmhtktlMP7xJItKEDTBOzdTeU6jrwNq1Ni3Ve3L\\\\\\\/JZIEr8XX7sqaqhslgn5kRjV\\\\\\\/DywD1KIOJD9UJyaxggUkxGE4f0dTysS5rG2JzDlDG5gN8RuUxyAqak7RtfOWiLInFIeXIZLAz4eoY2wAsGsUpwl1ykwWooIaiSGwccYUMWEQ1vz9cNvQigxR+srPQZmAsrna\\\\\\\/\\\\\\\/\\\\\\\/New+T2eIEZQIHKUul1xHgMlw2Zpc0j+i3PCB9lXRePH4Uja6SsPdB1+8KH\\\\\\\/O3x12gPlfelUrn2ellxIDiyDLICd6Q2Nu5A63PsdcCuhTZjel2Af5SDcTm33F3tJhesHsbHerX5YkgHrye05jtKbPyg9iXspxRfKQlHGvnSrU3HpjPytWO5I7YWkfXt2F\\\\\\\/uLYsiCfGJewSGN2EPHbX8AZDB8Gk4OKcNLK1QBEA9IYsI+Z3aiFxg5dRch848CJASQTNTSCO7XfiIQX9YazRc5ZkRvuYtO4Gn4VyLrCCSIvdUATgfIUWUITJgN9GGjKhlajZB\\\\\\\/ra3zEnKaZnIfWTmSsrXUqFnh1j2ozNV057bwdgVdgoDi64CRtl1hXFdibkioi0REth9Ng0hZaXaSgfR9GdJiGTypYY5zxUxsDTNk6MZLqfzDrsgGp1fsrPqVUrVkBSDC8fzo4PmBUPQs8Keo7cZRmMkSS9vO0q4SZ89AG3X\\\\\\\/FKsUbrfvrU6VHrQAPXbIP3QW1nr2btbssCwqRLzECgxcyhhvTl9cgnhrZMqN+I9+fmF58UGtMDPS3dJDv\\\\\\\/H1l1T0gOXHKVPLP4M7qgFgLkhw3Ns55sbXJcvrtszIph8qu35ldqhvSOAcXy3wlWzE\\\\\\\/\\\\\\\/I4dGmImSgRDASDOyTPBg30e4NFc0EP1pHDPD8FvkWhk9Y1V10amtxgOCVLtNcSwELaKwkSUW6XWnRsF43Xs9T\\\\\\\/9jDOz5iJTnHXDrChK8AXIv+\\\\\\\/tbFlHlVFBNXLOfttzTwsSbjiY9a2rQFfEk02ZTKjIu+TstshUdDqVJr9AmINZNVNVWVIxpafiXgdtTJDJ8Dp1lx\\\\\\\/yspH2Uk4kxeZV4PyZYMiOxr1OEllV4NI6qIxOmGEKvsf+PNkGGmpZJEpNFYL41\\\\\\\/qv5NYeCVFISxQK8\\\\\\\/seZNCOm\\\\\\\/PkPbr7JTL2HAKnDx9faIbMQ24iTkU6Bz8nomE0j+ic3aowjKOOHcApbXE3gRCq9t8XKIQQ6HHtIJa19cQYx6Xolfp082CjFjOVa9iuZ1XCRskiaQsBzIlH3r\\\\\\\/ELCLb8fQD2dJkgTb1TBUJ3WDv65u\\\\\\\/wc0AsZ2VrVvWjRioGKnDI3M9p5bR+E8KCcaI9fXvEh93A9VB9str9\\\\\\\/D0omjwaggcZ8u2vZjzwtBvw3NxlVKiD2TcANJ6RR7KQMYtieVX9wQvAvrUrPV\\\\\\\/ohdu\\\\\\\/TvOYBuQNO5J8Cf5RBLTNRieGGjQS5JF8K++6+T3Pt2DU86pqJlhF2vrRiXdLNMXTrc3G21nTCA+RyX2jvK5gf75tp\\\\\\\/gO3hTgGYkalehbuNamqlq6J94jA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022QKO2drps3hs+mZttQDgH4g==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022cd807f1023949bc5394c95d7f1d398c6c86e56791e9fa770d6f94435ec418320\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022qKdgu1T1rUs1yDySMVNZ\\\\\\\/6CtkHC3Ui+B3xBOAR1\\\\\\\/40SOG+P5nMYOF+4gl8LZAlL3nqzvMVF6dRcmU3zDsD38W9DUKwDNLLWZMeyPEMjBIrhLul52uDSfeQPVIiKYXSTnKWEMTomxZyfXo++WviC4DfiVppw6kUAnH0ynMfVtsLsaX92uhiBiewE++6T9m9zSzcBLpKXsgdSAa9nfADfNh8KJcQyUrXSYKW5RLVVQmHVdATa7+XMvuT7MwLWFF6mX8R8GcN7DAvHs+nAZE\\\\\\\/sp6fCD4\\\\\\\/VyAEMhVzTAI3XS9GlwlWPMeLPS1O0+aphLmJlKlmZXW6FeR2tLmZWkngwrSq\\\\\\\/zHidvI1KGM5Z8RdNUyxDw8ZsbB6lbhOh2W\\\\\\\/\\\\\\\/NlYsUpluIUbaqcfdwOSWXYprzxvWBIV656dAXsk1NsuAbWJhrzCnc60zT6LX9KB68pZy51UG1DV5lKKuJOfrnFT8r0spsCe+FFFx2QCVwRVGuaoJI0\\\\\\\/V1vU+X63THVQS4aXcGtp9qAW4qBcV2ufRPmPg6\\\\\\\/BOMY3HUpgWC\\\\\\\/P3xrvUnT\\\\\\\/VQYETDtYM1Tf1Lmn3N9XN8hGJ\\\\\\\/BYWDIlDWfTI3759ComPFYqZIQr+mEGLOzbQolq2SayYGqTWn3+hz+OonIEgNUAjyrAx2fXWqbSYs\\\\\\\/h0eOch5LKN7zjHsfyv3aexUA\\\\\\\/AWzyfGHZMUAt7kSf2nB4qXlbl99zp7wt7hk\\\\\\\/aGzABVZkxxP0N56IEIkWr2dnQUrOFxJdS104EoLF2Nlp4UW\\\\\\\/9bbGhZJIxqlF+KqX+Ze6clbbYoCZx8budfaQYwuYJKPPoXSTImlzpkY\\\\\\\/udPAWaWBavE8UuBD7HjK8bK0rbEUC7YHlB5+n+W5loU945ZaTViNqe8Lf1y2JKZ6mdEDvLmhQWUt\\\\\\\/N4q2ywmIU+vP7YEr\\\\\\\/YAEwiLymtMlsDB\\\\\\\/dNeJtD6W\\\\\\\/qWCkcoGPo\\\\\\\/DRGuWE8Yqy5kH7Gt82yZiLUnvtZA6J9TihFpxhPYK2TQSMfggzCj1LIRYfcfW8jWGN0ahh104Y+2qeMZjPmn3nvc4sXnJHNN6enSktOc6ICR6qBKzKedZLk2lZXHsuyR6wWqvv91fZ3\\\\\\\/v+Ygpaonem7Q+rLbx1pCIO42ajAeb3oPYup5CQ5mqisq2fJuKus2W57UAWjwt0xmPTvh32OzEmqdSK8Kcbh4bO7H18fwUYrYwPf8u93Zx6TfrvuDr3Yq9KO082Yd3XrDWpR+1+tgOOydwYZI4Z2As0J+zM9ynrnIfXWBG8r1R9fRkYg\\\\\\\/Aw73AoFpg4j0f1iE2Zl7L834zPLKpn4P1fnN\\\\\\\/zu758TESiHZKv3fKuF0VU1YdrDnFAI11OXvUiaGC13h0I8EALzLiR2\\\\\\\/eI5nDVxlIJ7zuyNPPG3dxz7dmKF0dHnt8sOvXqRw8LNqukokovwyWY4xy8jQVKnSQoT2LChqE5tTzxOoIXxXUflvavdzdHpYEgakuLlZQSgEa4VxigQLUIMEqKUffMP6DM4\\\\\\\/u+4hR3fPK8w3aNBCG1i4WJOrgff6NXbrpbPj4zUuFHsTg3SgSMbbhdo+EuMrBljUAyv\\\\\\\/BT2B6okKmF2Nr4KXpdKHuC+xNq9wWGhD1HBxFwsVtQ857fNzvegTItHHrdfluAcSaKXq2Ydau\\\\\\\/D3lSOVEeYacf2pvqNeaON4N3OJS4vWaPi0Z29kUMymX3saw2TDVQSUujlpK4\\\\\\\/FtcVrerv1Bz5+P8\\\\\\\/EcceM1jzlwzqp\\\\\\\/9QOFUzTLARReC13\\\\\\\/McbUwMitMZ6VjeIduCHFmFKumSI6ruBCRXYQN\\\\\\\/PTg3JQAlgoYz9nis\\\\\\\/W1d0uz6WwSEAkkwRxj\\\\\\\/QqSoczj3BR4CfwGSILcbSpuhXS01LX7NTsQASYuQ1ivqkT50RLmaqqJju278I\\\\\\\/o82aA+9d\\\\\\\/QPjztw98DCUAGCd2LPUsOZ0bve85Qsr0QWQAh59LBjmoaLq9\\\\\\\/87x7BRgZYKu3xA3rLBFaC7\\\\\\\/M8ePgNJwM9FG0cBuUaPE5FZj6xpiHbW5kavlafYKygY7rEWMRg==\\\u0022,\\\u0022iv\\\u0022:\\\u0022w7WI3zTKt0aug3AX2xosYg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002259ff078a8c1084797f49f472c85ea4adbeefb322bbc994a380837ca64add5df7\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-494292 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-genomic-prediction">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/10/3333" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-VoABzwhZSZ3ytDfQhIc_XwD6s-f82AxRhiJJl0O60qI" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/10/3333" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-My_-cFGXk1a9aw6_W-LyEQFhrJwkUlLZfiHx_mZGd2M" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/10/3315" title="Disease Resistance Genetics and Genomics in Octoploid Strawberry" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/10/3345" title="Alternative Lengthening of Telomeres in the Budding Yeast Naumovozyma castellii" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494292" id="node494292" data-pisa="ggg;9/10/3333" data-pisa-master="ggg;g3.119.400381" data-apath="/ggg/9/10/3333.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genomic Prediction Including SNP-Specific Variance Predictors</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-1335-7610" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Elena Flavia Mouresan</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-2062-3235" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Maria Selle</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-1057-5401" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Lars Rönnegård</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3333-3343;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400381 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494292" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Elena Flavia Mouresan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Animal Breeding and Genetics, Swedish University of Agricultural Sciences, Sweden, 75007,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Elena%2BFlavia%2BMouresan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mouresan%20EF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AElena%2BFlavia%2BMouresan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-1335-7610" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Elena Flavia Mouresan</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:elena.flavia.mouresan@slu.se" class="" data-icon-position="" data-hide-link-title="0">elena.flavia.mouresan@slu.se</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Maria Selle </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Mathematical Sciences, Norwegian University of Science and Technology, Norway, 7491, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Maria%2BSelle%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Selle%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMaria%2BSelle%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-2062-3235" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Maria Selle</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Lars Rönnegård </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Animal Breeding and Genetics, Swedish University of Agricultural Sciences, Sweden, 75007,</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>School of Technology and Business Studies, Dalarna University, Sweden, 79188</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lars%2BR%C3%B6nneg%C3%A5rd%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=R%C3%B6nneg%C3%A5rd%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALars%2BR%25C3%25B6nneg%25C3%25A5rd%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-1057-5401" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Lars Rönnegård</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/9/10/3333" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:494292" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:494292/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/10/3333.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:494292" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:494292/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/10/3333.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:494292" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:494292/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/9/10/3333.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:494292" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:494292/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">The increasing amount of available biological information on the markers can be used to inform the models applied for genomic selection to improve predictions. The objective of this study was to propose a general model for genomic selection using a link function approach within the hierarchical generalized linear model framework (hglm) that can include external information on the markers. These models can be fitted using the well-established hglm package in R. We also present an R package (CodataGS) to fit these models, which is significantly faster than the hglm package. Simulated data were used to validate the proposed model. We tested categorical, continuous and combination models where the external information on the markers was related to 1) the location of the QTL on the genome with varying degree of uncertainty, 2) the relationship of the markers with the QTL calculated as the LD between them, and 3) a combination of both. The proposed models showed improved accuracies from 3.8% up to 23.2% compared to the SNP-BLUP method in a simulated population derived from a base population with 100 individuals. Moreover, the proposed categorical model was tested on a dairy cattle dataset for two traits (Milk Yield and Fat Percentage). These results also showed improved accuracy compared to SNP-BLUP, especially for the Fat% trait. The performance of the proposed models depended on the genetic architecture of the trait, as traits that deviate from the infinitesimal model benefited more from the external information. Also, the gain in accuracy depended on the degree of uncertainty of the external information provided to the model. The usefulness of these type of models is expected to increase with time as more accurate information on the markers becomes available.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/blup" class="hw-term hw-article-keyword hw-article-keyword-blup" rel="nofollow">BLUP</a></li><li class="kwd"><a href="/keyword/hglm" class="hw-term hw-article-keyword hw-article-keyword-hglm" rel="nofollow">hglm</a></li><li class="kwd"><a href="/keyword/codatags" class="hw-term hw-article-keyword hw-article-keyword-codatags" rel="nofollow">CodataGS</a></li><li class="kwd"><a href="/keyword/external-information" class="hw-term hw-article-keyword hw-article-keyword-external-information" rel="nofollow">external information</a></li><li class="kwd"><a href="/keyword/genomic-prediction" class="hw-term hw-article-keyword hw-article-keyword-genomic-prediction" rel="nofollow">Genomic Prediction</a></li><li class="kwd"><a href="/keyword/genpred" class="hw-term hw-article-keyword hw-article-keyword-genpred" rel="nofollow">GenPred</a></li><li class="kwd"><a href="/keyword/shared-data-resources" class="hw-term hw-article-keyword hw-article-keyword-shared-data-resources" rel="nofollow">Shared Data Resources</a></li></ul><p id="p-3">The identification of a large number of Single Nucleotide Polymorphisms (SNPs) along the genome, as a by-product of the sequencing efforts (<em>e.g.</em>, <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Daetwyler <em>et al.</em> 2014</a>) and the development of SNP-chip genotyping technology (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Gunderson <em>et al.</em> 2005</a>) have made genotyping of thousands of markers affordable at low cost. <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Meuwissen <em>et al.</em> (2001)</a> foresaw these breakthroughs in technology and proposed a new method of selection in animal breeding denoted as Genomic Selection (GS). This method has been tested through simulation studies (<a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Meuwissen <em>et al.</em> 2001</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Muir 2007</a>) and cross validation with real data in different species such as mice (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Legarra <em>et al.</em> 2008</a>), dairy cattle (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Luan <em>et al.</em> 2009</a>; <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">VanRaden <em>et al.</em> 2009</a>), aquaculture (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Sonesson and Meuwissen 2009</a>) and poultry (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">González-Recio <em>et al.</em> 2009</a>). Nowadays, GS has become part of the routine breeding schemes in dairy cattle (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Hayes <em>et al.</em> 2009</a>) and other species including pigs (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Ostersen <em>et al.</em> 2011</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Hidalgo <em>et al.</em> 2015</a>; <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Tusell <em>et al.</em> 2016</a>) and poultry (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Wolc <em>et al.</em> 2015</a>).</p><p id="p-4">Several statistical models have been proposed for genomic prediction using whole-genome markers. The most popular method provides best linear unbiased predictions (BLUP) of marker effects (<a id="xref-ref-38-3" class="xref-bibr" href="#ref-38">Meuwissen <em>et al.</em> 2001</a>) by assuming that the marker effects come from a Gaussian distribution with constant variance and every marker can have an effect on the analyzed trait. This method is referred to either as GBLUP or SNP-BLUP depending on the implementation. Biologically, it seems more reasonable to assume that some of the markers are in linkage disequilibrium (LD) with a causative gene or a quantitative trait locus (QTL) and therefore can capture their effect on the studied trait, whereas some markers are not in LD with any gene and should therefore not capture any effect. To achieve this idea, several methods have been developed to incorporate different prior assumptions on the genetic architecture of the trait. For this family of methods, often referred to as the Bayesian Alphabet (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Gianola 2013</a>), it is assumed that the genetic effects of the SNPs follow alternative distributions like a t-distribution (Bayes A) (<a id="xref-ref-38-4" class="xref-bibr" href="#ref-38">Meuwissen <em>et al.</em> 2001</a>), a double exponential distribution (Bayes LASSO) (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">de los Campos <em>et al.</em> 2009</a>; <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Usai <em>et al.</em> 2009</a>) or a mixture of distributions (<em>i.e.</em>, Bayes B, Bayes Cπ, Bayes R) (<a id="xref-ref-38-5" class="xref-bibr" href="#ref-38">Meuwissen <em>et al.</em> 2001</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Habier <em>et al.</em> 2011</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Erbe <em>et al.</em> 2012</a>). The prior assumptions of these methods are rather arbitrary and their performance relies heavily on the model assumptions capturing accurately the true genetic architecture of the trait of interest (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Daetwyler <em>et al.</em> 2010</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Hayes <em>et al.</em> 2010</a>).</p><p id="p-5">Whole-genome sequencing of individuals has facilitated the detection of genetic variants that can be used for GS. Currently, in <em>Bos Taurus</em> cattle ∼28 million genetic variants have been reported (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Daetwyler <em>et al.</em> 2013</a>). This large number of polymorphic markers comes with a major challenge in terms of computational speed and memory. One way to deal with this problem is to make use of the biological information available on the markers, <em>e.g.</em>, to annotate the markers in classes based on genome location or functionality and prioritize those classes that show a higher probability of containing trait associated markers. <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Koufariotis <em>et al.</em> (2014)</a> showed that protein coding regions explain significantly more variation than similar number of randomly chosen markers across many traits in cattle. Moreover, in a study by <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Schork <em>et al.</em> (2013)</a>, the upstream and downstream classes showed significant enrichment in trait associated variants suggesting that these classes can potentially have important regulatory functions. In the same line, <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Yang <em>et al.</em> (2011)</a> stated that genic regions contributed more additive genetic variance than non-genic regions for human traits. However, <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Do <em>et al.</em> (2015)</a> found that the contribution to total genomic variance per SNP among the annotated classes was similar for all regions in a feed efficiency study in pigs.</p><p id="p-6">Several authors have also investigated the predictive ability of models based on annotation classes. Using kernel methods, <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Morota <em>et al.</em> (2014)</a> and <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Abdollahi-Arpanahi <em>et al.</em> (2016)</a> showed that a whole-genome approach provided better predictive ability than that obtained from classes of genomic regions considered separately. Likewise, <a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Do <em>et al.</em> (2015)</a> using GBLUP and Bayesian methods (Bayes A, B and Cπ) found that classification of SNPs by genomic annotation had little impact on the accuracy of prediction for feed efficiency traits in pigs.</p><p id="p-7">Apart from genome annotation information, other biological information is available on the SNPs. QTL databases are available for most livestock species (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Hu <em>et al.</em> 2013</a>) and Genome-Wide Association Studies (GWAS) (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Bush and Moore 2012</a>) have identified a great number of trait-associated markers. Moreover, metabolic and signaling pathways (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Kanehisa <em>et al.</em> 2008</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Croft <em>et al.</em> 2011</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Caspi <em>et al.</em> 2012</a>) and gene regulatory networks (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Lee <em>et al.</em> 2002</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Shalgi <em>et al.</em> 2007</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Hecker <em>et al.</em> 2009</a>) can also provide valuable insight to the underlying biology of the traits of interest (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Snelling <em>et al.</em> 2013</a>). A rather new tool that has been developed to incorporate existing knowledge of the genetic architecture of complex traits into a GS model is BLUP|GA, <em>i.e.</em>, “BLUP approach given the Genetic Architecture” (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Zhang <em>et al.</em> 2014</a>). This tool uses publicly available GWAS results and showed improved prediction accuracies compared to traditional GBLUP and Bayes B methods. Also, a similar approach was developed by <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Kadarmideen (2014)</a> (system genomic BLUP, - sgBLUP-) where SNPs with known biological role were explicitly modeled in addition to conventional random SNP effects in SNP-BLUP or GBLUP methods. Along with the BLUP approaches, several Bayesian methods were also developed. Bayes Bπ (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Gao <em>et al.</em> 2015</a>) is a modified version of Bayes B (<a id="xref-ref-38-6" class="xref-bibr" href="#ref-38">Meuwissen <em>et al.</em> 2001</a>) able to utilize locus-specific priors. In their study, the authors obtained locus-specific priors from variance analysis (ANOVA) based on information from each single marker separately and the results showed improved accuracy and decreased bias compared to Bayes B and Bayes Cπ. In a similar way, <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">MacLeod <em>et al.</em> (2016)</a> proposed a modification to the BayesR method (<a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Erbe <em>et al.</em> 2012</a>) that incorporates prior biological knowledge. This method provides a flexible approach to improve the accuracy of genomic prediction and QTL discovery taking advantage of available biological knowledge. The basic idea of previously developed methods is to group SNPs into those having a biological function and those with an unknown function. Both the BLUP|GA and BayesBπ methods, also include continuous weights for all, or a subset of markers. For the BLUP|GA method, weights computed using trait-specific GWAS results are used to construct the genomic relationship matrix, whereas in BayesBπ the weights are computed from single-SNP ANOVA analyses.</p><p id="p-8">Although a large number of methods have been developed already for GS, a general BLUP method to include explanatory variables for SNP-specific variances that allow both continuous and class variables seems to be missing. Here we propose a general model using a link function approach within the hierarchical generalized linear model framework (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Lee <em>et al.</em> 2006</a>). The algorithm proposed by <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Lee and Nelder (1996)</a> is used, where the hierarchical generalized linear model is fitted by iterating between augmented generalized linear models. With this approach, rather complex models can be fitted using a single deterministic fitting algorithm (see <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Rönnegård <em>et al.</em> 2010a</a>, <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">2010b</a>).</p><p id="p-9">The aim of the paper is to assess the accuracy for such models including predictors for SNP variances, with special emphasis on the effect of the trait’s genetic architecture and LD structure on estimation accuracy. We present a family of models where the SNP variances can be modeled using both, categorical and continuous predictors, or a combination of the two. The computation time of these models is also studied and a new, faster R package (CodataGS) to fit these models is presented.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Data simulation</h3><p id="p-10">Data were simulated to evaluate the models. Four different scenarios for QTL variance distribution were simulated under three different genetic architectures in which the number of QTL per chromosome was 10, 20 or 100. For each combination of scenario and genetic architecture, 100 simulation replicates were produced. This section describes the simulations in detail.</p><p id="p-11">A base population was simulated of 100 individuals that evolved under random mating for 400 non-overlapping generations (generation -399 to 0) maintaining the population size constant. After the 400 historical generations, two more generations were simulated, still under random mating and expanding the population size from 100 to 200 individuals per generation. Generation 1 was used as training set and generation 2 as validation set. The genome comprised of two chromosomes of 1 Morgan each with 8,800 loci, evenly distributed across the genome. In the base population alleles were coded as 0 or 1 with equal probability resulting in intermediate average allele frequencies. In the first generation, 1,000 loci per chromosome were selected randomly among those loci with a Minor allele frequency (MAF) higher than 0.05 to simulate the SNP marker panel. The same loci were used for validation in generation 2.</p><p>To simulate phenotypes in generation 1 (training set), <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3333/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3333/embed/mml-math-1.gif"/></noscript></span></span> loci were selected randomly excluding loci that were on the edge of the chromosome and those with a MAF lower than 0.05. In order to simulate different scenarios of genetic architecture underlying the trait, the number of QTL (<span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3333/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3333/embed/mml-math-2.gif"/></noscript></span></span>) varied between 10, 20 and 100 per chromosome. Moreover, the QTL effects, <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3333/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3333/embed/mml-math-3.gif"/></noscript></span></span>, <em>j</em> = 1, …, <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3333/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3333/embed/mml-math-4.gif"/></noscript></span></span>, were assumed to be normally distributed with mean 0 and varying variance assigned in one of the following ways:</p><ul class="list-simple " id="list-1"><li id="list-item-1"><p id="p-13"><strong>Scenario 0 (Sc0):</strong> <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3333/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3333/embed/mml-math-5.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3333/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3333/embed/mml-math-6.gif"/></noscript></span></span></p></li><li id="list-item-2"><p id="p-14"><strong>Scenario 1 (Sc1):</strong> <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3333/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3333/embed/mml-math-7.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3333/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3333/embed/mml-math-8.gif"/></noscript></span></span>, with probability 0.5, and <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3333/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3333/embed/mml-math-9.gif"/></noscript></span></span>, with probability 0.5</p></li><li id="list-item-3"><p id="p-15"><strong>Scenario 2 (Sc2):</strong> <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3333/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3333/embed/mml-math-10.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3333/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3333/embed/mml-math-11.gif"/></noscript></span></span>, if <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3333/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3333/embed/mml-math-12.gif"/></noscript></span></span> belonged to chromosome 1, and <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3333/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3333/embed/mml-math-13.gif"/></noscript></span></span>, if <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3333/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3333/embed/mml-math-14.gif"/></noscript></span></span> belonged to chromosome 2</p></li></ul><p>Here, <em>e</em> is the natural number and therefore the variance can take values between <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3333/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3333/embed/mml-math-15.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3333/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3333/embed/mml-math-16.gif"/></noscript></span></span> The difference between the scenarios Sc1 and Sc2 is that in Sc1 heterogeneous QTL effects are allowed on the same chromosome and may be in linkage disequilibrium with each other. On the other hand, in Sc2 the two different types of QTL are located on different chromosomes to ensure low LD between them.</p><ul class="list-simple " id="list-2"><li id="list-item-4"><p id="p-16"><strong>Scenario 3 (Sc3):</strong> <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3333/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3333/embed/mml-math-17.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3333/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3333/embed/mml-math-18.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3333/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3333/embed/mml-math-19.gif"/></noscript></span></span> is the position of QTL <em>j</em> and <em>f</em> is a function of relative distance to the chromosome edge. Consequently, <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3333/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3333/embed/mml-math-20.gif"/></noscript></span></span> take values between <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3333/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3333/embed/mml-math-21.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3333/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3333/embed/mml-math-22.gif"/></noscript></span></span>. This scenario is motivated by the finding that fitness genes tend to be located closer to the center of the chromosomes (see <em>e.g.</em>, <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Carneiro <em>et al.</em> (2009)</a> and references therein).</p></li></ul><p>For each scenario, the three separate genetic architectures were simulated, <em>i.e.</em>, with 10, 20 or 100 QTL per chromosome. In order for the results from the different scenarios and genetic architectures to be comparable, the total genetic variance was scaled to 1.0. In this way, the obtained traits were either controlled by a small number of QTL with medium-large effects or by a large number of QTL with small effects.</p><p id="p-17">In generation 1 (training set) phenotypes were simulated for all 200 individuals as:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3333/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3333/embed/mml-math-23.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3333/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3333/embed/mml-math-24.gif"/></noscript></span></span> is the phenotype of individual <em>i</em>, <em>μ</em> is a fixed effect which was set equal to 0, <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3333/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3333/embed/mml-math-25.gif"/></noscript></span></span> is the genotype for the <em>j<sup>th</sup></em> QTL coded as 0, 1 or 2 for the homozygote, heterozygote and the alternative homozygote respectively for individual <em>i</em> in generation 1, <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3333/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3333/embed/mml-math-26.gif"/></noscript></span></span> is the simulated normally distributed <em>j<sup>th</sup></em> QTL effect as described above, and <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3333/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3333/embed/mml-math-27.gif"/></noscript></span></span> is the residual effect of the <em>i<sup>th</sup></em> individual normally distributed with mean 0 and the appropriate variance <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3333/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3333/embed/mml-math-28.gif"/></noscript></span></span> in order to create a trait with heritability of 0.2.</p><p id="p-18">Generation 2 was used as validation set where true genomic breeding values (TBVs) were computed as:<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3333/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3333/embed/mml-math-29.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3333/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3333/embed/mml-math-30.gif"/></noscript></span></span> is the QTL genotype for QTL <em>j</em> and individual <em>i</em> for this generation.</p></div><div id="sec-3" class="subsection"><h3>Genomic evaluation</h3><p id="p-19">To estimate the SNP effects, the marker panel of 1,000 SNPs per chromosome mentioned above was used and the following model was assumed:<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3333/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3333/embed/mml-math-31.gif"/></noscript></span><span class="disp-formula-label">(1)</span></span>where <span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3333/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3333/embed/mml-math-32.gif"/></noscript></span></span> is the phenotype of individual <em>i</em>, <em>μ</em> is a fixed effect, <em>p</em> is the total number of SNPs, <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3333/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3333/embed/mml-math-33.gif"/></noscript></span></span> is the genotype of the SNP <em>j</em> for individual <em>i</em> coded as 0, 1 or 2, <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3333/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3333/embed/mml-math-34.gif"/></noscript></span></span> is the residual effect, and<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3333/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3333/embed/mml-math-35.gif"/></noscript></span><span class="disp-formula-label">(2)</span></span>is the <em>j<sup>th</sup></em> SNP effect normally distributed with mean 0 and variance<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3333/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3333/embed/mml-math-36.gif"/></noscript></span><span class="disp-formula-label">(3)</span></span>where <span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3333/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3333/embed/mml-math-37.gif"/></noscript></span></span> is a linear predictor for the SNP-specific variance the components of which are explained in the following section.</p></div><div id="sec-4" class="subsection"><h3>Evaluation models</h3><p id="p-20">The linear predictor for variance (<span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3333/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3333/embed/mml-math-38.gif"/></noscript></span></span>) allows to incorporate any type of external information about the SNP variance, making it possible to assign the same variance for all SNPs, a subgroup of SNPs or assign a unique variance for each SNP. We used this linear predictor for variance to introduce external information on the SNPs into the models and the predictive performance of different prior assumptions was tested. The log link ensures a positive variance (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Aitkin 1987</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Lee and Nelder 1998</a>) and due to its computational robustness is a common choice of link function in variance modeling (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Jaffrezic <em>et al.</em> 2000</a>; <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Sorensen and Waagepetersen 2003</a>; <a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Rönnegård <em>et al.</em> 2010a</a>). By using a Gamma generalized linear model with a log link, the score function for this model is equivalent to the score function of the REML likelihood in a linear mixed model (<a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Lee and Nelder 1996</a>, <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Lee <em>et al.</em> 2017</a> page 91) and therefore produces REML estimates of the variance components. Furthermore, especially for variances close to zero the likelihood will be more symmetric on a logarithmic scale than on an untransformed scale, and thereby gives better standard errors for the fitted variance components.</p><p>The models tested in this study were:</p><ol class="list-ord " id="list-3"><li id="list-item-5"><p id="p-22"><strong>SNP-BLUP</strong>: In the traditional model the variance of the markers is assumed to be equal for all markers and therefore <span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3333/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3333/embed/mml-math-39.gif"/></noscript></span></span> in the linear predictor for the variance for all markers.</p></li><li id="list-item-6"><p id="p-23">Categorical models (<strong>W10</strong>, <strong>W20</strong> and <strong>W40</strong>): For these models the genome was divided into non-overlapping windows of 10, 20 or 40 SNPs. Then, all the SNPs within a given window were given the value <span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3333/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3333/embed/mml-math-40.gif"/></noscript></span></span> if they contained a QTL and <span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3333/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3333/embed/mml-math-41.gif"/></noscript></span></span> if they did not. Hence, a study with known regions harboring the QTL was mimicked, where these regions were known with varying degree of uncertainty.</p></li><li id="list-item-7"><p id="p-24">Continuous model (<strong>LD</strong>): For this model, following <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Yang and Tempelman (2012)</a> and <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Rönnegård and Lee (2010)</a>, the linkage disequilibrium (LD) between a SNP and a QTL was calculated as <span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3333/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3333/embed/mml-math-42.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3333/embed/mml-math-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3333/embed/mml-math-43.gif"/></noscript></span></span> (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Falconer and Mackay 1996</a>), <span class="inline-formula" id="inline-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3333/embed/mml-math-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3333/embed/mml-math-44.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3333/embed/mml-math-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3333/embed/mml-math-45.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-41"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3333/embed/mml-math-46.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3333/embed/mml-math-46.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-42"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3333/embed/mml-math-47.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3333/embed/mml-math-47.gif"/></noscript></span></span> are the allele frequencies of the SNP and QTL, <span class="inline-formula" id="inline-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3333/embed/mml-math-48.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3333/embed/mml-math-48.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-44"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3333/embed/mml-math-49.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3333/embed/mml-math-49.gif"/></noscript></span></span> are the homozygous haplotype frequencies and <span class="inline-formula" id="inline-formula-45"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3333/embed/mml-math-50.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3333/embed/mml-math-50.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-46"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3333/embed/mml-math-51.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3333/embed/mml-math-51.gif"/></noscript></span></span> are the heterozygous haplotype frequencies. Then, each SNP was assigned the value of <span class="inline-formula" id="inline-formula-47"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3333/embed/mml-math-52.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3333/embed/mml-math-52.gif"/></noscript></span></span>. The relationship between SNPs and QTL was modeled in such way that markers in higher LD with one or more QTL would be given more importance in the model compared to other markers not in LD with any QTL.</p></li><li id="list-item-8"><p id="p-25">Combination of categorical and continuous models (<strong>W10-LD</strong>, <strong>W20-LD</strong> and <strong>W40-LD</strong>): In these models the genome was divided into windows as in the previous categorical models but the SNPs located within a window that harbored a QTL were given the value of the LD with the QTL instead of 1. The model could, therefore, differentiate between SNPs not only based on location but also based on the relationship with the real QTL.</p><p id="p-26"><a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a> gives an overview of all simulated scenarios and models tested. Each scenario was simulated with 10, 20 and 100 QTL per chromosome as described previously.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/494321/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/494321/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">SUMMARY OF MODELS TESTED FOR EACH SCENARIO OF GENETIC ARCHITECTURE SIMULATED</span><div class="sb-div caption-clear"></div></div></div></li><li id="list-item-9"><p id="p-29">Additional models (<strong>W10-2var</strong>, <strong>W20-2var</strong>, <strong>W40-2var</strong>, <strong>Dis</strong>, <strong>W10-Dis</strong>, <strong>W20-Dis</strong> and <strong>W40-Dis</strong>): The previously described models include external information on the physical location of the QTL relative to the SNPs or/and the relationship of the SNPs with the QTL but they do not include any information about the QTL variance. Therefore, a few additional models were created based on the particular parameters used for the simulation of each genetic architecture scenario. These models are defined as follows.</p><ol class="list-alphalower " id="list-4"><li id="list-item-10"><p id="p-30">For the scenarios where the QTL effects came from distributions with two different variances (Sc1 and Sc2) we assumed this information was known and we expanded the linear predictor to <span class="inline-formula" id="inline-formula-48"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3333/embed/mml-math-53.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3333/embed/mml-math-53.gif"/></noscript></span></span> in order to accommodate for more variances (in the models W10-2var, W20-2var, and W40-2var). The genome was divided in non-overlapping windows as before and SNPs associated with a QTL with variance <span class="inline-formula" id="inline-formula-49"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3333/embed/mml-math-54.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3333/embed/mml-math-54.gif"/></noscript></span></span> was assigned <span class="inline-formula" id="inline-formula-50"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3333/embed/mml-math-55.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3333/embed/mml-math-55.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-51"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3333/embed/mml-math-56.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3333/embed/mml-math-56.gif"/></noscript></span></span>, while if it was associated with a QTL with variance <span class="inline-formula" id="inline-formula-52"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3333/embed/mml-math-57.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3333/embed/mml-math-57.gif"/></noscript></span></span> it was assigned <span class="inline-formula" id="inline-formula-53"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3333/embed/mml-math-58.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3333/embed/mml-math-58.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-54"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3333/embed/mml-math-59.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3333/embed/mml-math-59.gif"/></noscript></span></span>. If a SNP was located within a window with no QTL then both <span class="inline-formula" id="inline-formula-55"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3333/embed/mml-math-60.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3333/embed/mml-math-60.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-56"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3333/embed/mml-math-61.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3333/embed/mml-math-61.gif"/></noscript></span></span> had a value of 0.</p></li><li id="list-item-11"><p id="p-31">For Sc3, we used the distance of the markers from the edge of the chromosome as external information either as a continuous variable (Dis) or within windows (W10-Dis, W20-Dis and W40-Dis), since the QTL variances were simulated in the same way.</p></li></ol></li></ol></div><div id="sec-5" class="subsection"><h3>German Holstein population data</h3><p id="p-32">To demonstrate the model on real data, we used a German Holstein genomic prediction population consisting of 5024 bulls (<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Zhang <em>et al.</em> 2015</a>). Three traits were measured, where the first two had highly significant QTL from a GWAS. Including this information as explanatory variables for the SNP-specific variances was expected to improve genomic selection. We were also able to compare our results with <a id="xref-ref-60-2" class="xref-bibr" href="#ref-60">Zhang <em>et al.</em> (2015)</a>, who have developed the algorithm BLUP|GA that includes information about genetic architecture by building trait-specific genomic covariance matrices.</p><p id="p-33">All bulls had been genotyped and we used the 42,373 SNPs with minor allele frequency above 0.01. For the three traits, which were milk yield, milk fat percentage and somatic cell score, <a id="xref-ref-60-3" class="xref-bibr" href="#ref-60">Zhang <em>et al.</em> (2015)</a> provide highly reliable estimated breeding values (EBVs) for all bulls from previous studies (<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Hu <em>et al.</em> 2013</a>; <a id="xref-ref-59-2" class="xref-bibr" href="#ref-59">Zhang <em>et al.</em> 2014</a>). The EBVs for milk yield and milk fat percentage were used as phenotypes.</p><p id="p-34">We chose to fit the model 1) <strong>SNP-BLUP</strong> and models 2) <strong>W11</strong> and <strong>W41</strong>, with windows of size 11 and 41 SNPs centered around candidate QTL peaks. To find candidate QTL, we performed GWAS, correcting for genomic relationship using estimated residual and additive genetic variance from GBLUP. All SNPs from GWAS with p-value less than 10¯<sup>5</sup> were considered a candidate QTL. For milk yield we identified 6 candidate QTL peaks and for the fat percentage we identified 5 candidate QTL peaks, which were used as the center of the windows.</p></div><div id="sec-6" class="subsection"><h3>Hglm method and CodataGS</h3><p id="p-35">The estimation of the SNP effects was performed by fitting the model described by equations 1-3 that allows both continuous and categorical predictors for the SNP-specific variances, or a combination of continuous and categorical predictors. We tested a few examples of external information on the SNPs and these models were fitted using the <strong>hglm</strong> package in R (<a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Rönnegård <em>et al.</em> 2010b</a>). In the <strong>hglm</strong> package the linear predictor for variance <span class="inline-formula" id="inline-formula-57"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3333/embed/mml-math-62.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3333/embed/mml-math-62.gif"/></noscript></span></span> is specified using the <em>X.rand.disp</em> option in the hglm function and the function estimates SNP effects (example of the command line to call the hglm function with the option <em>X.rand.disp</em> can be found in the Supplementary File S1 line 156).</p><p id="p-36">When the number of markers largely exceeds the number of individuals, the computational speed and memory requirements can be improved by fitting individual effects (<em>i.e.</em>, EGBVs) in an equivalent model instead of SNP effects (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Strandén and Garrick 2009</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Shen <em>et al.</em> 2013</a>). This equivalent model, which uses the external information on each SNP in the same way as in the <strong>hglm</strong> package, was implemented in the R package <strong>CodataGS</strong> and is available on CRAN (<a href="https://cran.r-project.org/web/packages/CodataGS">https://cran.r-project.org/web/packages/CodataGS</a>). The theory is explained in the Supplementary File S3. The CodataGS R package was used for the analysis of the German Holstein population data.</p></div><div id="sec-7" class="subsection"><h3>Accuracy</h3><p id="p-37">The predictive ability of all models was evaluated as the correlation of the estimated genomic breeding values (EGBVs) and the true genomic breeding values (TGBVs) for the validation set (Generation 2). For each simulation setup, 100 replicates were generated. The convergence of the models varied from 71 to 100% and results are presented for those replicates where all models converged. For the German Holstein population, we performed a fivefold cross-validation with all bulls randomly separated in four groups of 1005 and one group of 1004 with both model 1) and 2). Each group served as a test set while the rest of the groups were used to estimate the SNP effects. The predictive ability was measured as the correlation between the EBVs and the phenotypes of the testing individuals.</p></div><div id="sec-8" class="subsection"><h3>Data availability</h3><p id="p-38">Simulation of the data that support the findings is possible through the attached simulation code in File S1 and File S2 (Functions for the simulation) deposited at figshare. The simulation code and the methodology described previously are sufficient to reproduce the results of this study. The analysis program CodataGS used to apply the alternative models on the Holstein dataset is available at <a href="https://cran.r-project.org/web/packages/CodataGS">https://cran.r-project.org/web/packages/CodataGS</a>. Supplemental material available at FigShare: <a href="https://doi.org/10.25387/g3.9247832">https://doi.org/10.25387/g3.9247832</a>.</p></div></div><div class="section results" id="sec-9"><h2 class="">Results</h2><p id="p-39"><a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a> contains different versions of the model tested. The fitted SNP effects obtained from <strong>hglm</strong> for one simulation replicate under scenario Sc0 with 10 QTL per chromosome are presented in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>. The R code to reproduce <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a> is found in Supplementary File S1 (along with File S2). The results show how the fitted SNP effects may change between model specifications. For example, it can be observed that with increasing window size the estimated effects tend to be spread between more SNPs.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/10/3333/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Simulated QTL effects (black dots) and fitted SNP effects under SNP-BLUP and 7 alternative models (Categorical: W10, W20 and W40, Continuous: LD, Combination: W10-LD, W20-LD and W40-LD) for one simulation replicate under simulation scenario Sc0 with 10 QTL per chromosome underlying the trait." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-589083922" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Simulated QTL effects (black dots) and fitted SNP effects under SNP-BLUP and 7 alternative models (Categorical: W10, W20 and W40, Continuous: LD, Combination: W10-LD, W20-LD and W40-LD) for one simulation replicate under simulation scenario Sc0 with 10 QTL per chromosome underlying the trait.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/10/3333/F1.medium.gif" width="325" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/9/10/3333/F1.medium.gif" width="325" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/10/3333/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/10/3333/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494530" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-40" class="first-child">Simulated QTL effects (black dots) and fitted SNP effects under SNP-BLUP and 7 alternative models (Categorical: W10, W20 and W40, Continuous: LD, Combination: W10-LD, W20-LD and W40-LD) for one simulation replicate under simulation scenario Sc0 with 10 QTL per chromosome underlying the trait.</p><div class="sb-div caption-clear"></div></div></div><div id="sec-10" class="subsection"><h3>Model performance</h3><p id="p-41"><a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a> shows the accuracies of the predicted EGBVs in the validation set (generation 2) for scenario 0 (Sc0) with 10 QTL per chromosome underlying the trait. In general, the alternative models performed better than SNP-BLUP. The categorical models yielded higher accuracies compared to the SNP-BLUP model by 14.3% (0.670 ± 0.013), 11.9% (0.656 ± 0.012) and 8.4% (0.635 ± 0.012) for the models W10, W20 and W40, respectively. Nonetheless, we observe that the advantage of the categorical models over the SNP-BLUP decreased with increasing window sizes. Moreover, the continuous model (LD) resulted in higher accuracy than the SNP-BLUP or the categorical models with an increase of 22.4% (0.717 ± 0.011) in accuracy with respect to the SNP-BLUP. Similarly, the combination models performed 20.6% (W10-LD, 0.707 ± 0.013) 21.8% (W20-LD, 0.714 ± 0.013) and 23.2% (W40-LD, 0.722 ±0.013) better than the SNP-BLUP model. Contrary to the categorical models, the combination models maintained the gain in accuracy with increasing window size. The alternative models provided unbiased predictions while the SNP-BLUP showed upward bias (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>). Finally, the mean squared error of prediction (MSEP) in the validation set improved with the alternative models compared to the SNP-BLUP, indicating that predictions are closer to the true breeding values in the alternative models compared with the SNP-BLUP.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/494541/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/494541/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">ACCURACY AND BIAS OF THE PREDICTED EGBVS IN THE VALIDATION SET (GENERATION 2) FOR THE SCENARIO 0 (SC0) WITH 10 QTLS PER CHROMOSOME UNDERLYING THE TRAIT</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-11" class="subsection"><h3>Effect of number of simulated QTL</h3><p id="p-43">In order to investigate the performance of the alternative models for traits with different genetic architectures we simulated a trait controlled by an increasing number of QTL with each having a decreasing effect. As an overview, the accuracies of the different models in Sc0 with 20 and 100 QTL per chromosome are visualized in <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a> together with the results from 10 QTL per chromosome. The advantage of the alternative models over the SNP-BLUP model decreased with increasing number of QTL controlling the trait. When the number of QTL underlying the trait is 20 QTL per chromosome, the accuracies obtained were 9.6%, 7.5% and 3.8% better than the SNP-BLUP for the W10, W20 and W40 models, respectively. The continuous model resulted in a gain of 12.9% in accuracy while the combination models performed slightly better than all the alternative models yielding gains in accuracy of 14%, 14.2% and 13.5% for the W10-LD, W20-LD and W40-LD models, respectively. Finally, in the case of 100 QTL per chromosome, all models performed roughly the same as SNP-BLUP, yielding accuracies between 0.583 ± 0.012 (W40) and 0.599 ± 0.011 (W10-LD).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/10/3333/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Accuracies obtained under different cases of genetic architecture of the trait for SNP-BLUP and the alternative models." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-589083922" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Accuracies obtained under different cases of genetic architecture of the trait for SNP-BLUP and the alternative models.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/10/3333/F2.medium.gif" width="440" height="192"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/9/10/3333/F2.medium.gif" width="440" height="192"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/10/3333/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/10/3333/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494407" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-44" class="first-child">Accuracies obtained under different cases of genetic architecture of the trait for SNP-BLUP and the alternative models.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-12" class="subsection"><h3>Effect of variance of the QTL effects</h3><p id="p-45">The genetic architecture of a trait does not only depend on the number of QTL that affect the trait. For example, mutations can affect protein coding regions or regulatory regions and these mutations can have a bigger or smaller effect on the trait. Therefore we can assume that their effects come from a mixture of distributions with varying variance over the genome. For this purpose we simulated several scenarios where the QTL effects were drawn from a mixture of distributions (see Sc1 – Sc3 in Materials and Methods). We compared the performance of all models under all scenarios of QTL effect variances and all cases of number of QTL affecting the trait (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). In general the models performed similarly under Sc1, Sc2 and Sc3 as in Sc0. Small differences were observed in the case of 10 QTL per chromosome where all models performed slightly better in Sc0 and Sc2 (QTL effects from a low variance distribution on chromosome 1 and from high variance distribution on chromosome 2) compared with the results from Sc1 and Sc3. Nonetheless, this minimum difference disappeared quickly with increasing number of QTL per chromosomes. The external information included in the alternative models was related to the position of the QTL on the genome and/or the relationship of the SNPs with the QTL (LD), but no information about the distribution of the variance itself was included. Therefore, we fitted additional models that considered the way the QTL were simulated (see linear predictor 5: Additional models Material and Methods, and Supplementary file Table S1). For Sc1 and Sc2 we extended the linear predictor (<span class="inline-formula" id="inline-formula-58"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3333/embed/mml-math-63.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3333/embed/mml-math-63.gif"/></noscript></span></span>) to accommodate for two types of variances for the SNPs in windows that harbored a QTL assuming that we knew beforehand the distribution variance of the effect of that QTL and, as before, we tested 3 different window sizes (10, 20 and 40 SNPs per window). The results showed that these additional models performed similarly as the categorical models (W10, W20 and W40) under all cases of genetic architecture simulated. The only exception to these results was for the Sc2 with 100 QTL per chromosome where additional models showed a small increase in accuracy compared to all other models (Supplementary files, Figure S1). For the Sc3 we used the distance of the SNP from the edge of the chromosome as external information, either as a continuous variable or within windows. Similarly as before, the additional models that included information on the simulated distribution variance of the QTL did not perform better than the alternative models. The combined models (W10-Dis, W20-Dis and W40-Dis) performed the same as the categorical models while the continuous model (Dis) showed no benefit compared to the alternative models or the SNP-BLUP model under any simulation scenario of genetic architecture.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/10/3333/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Accuracies obtained from SNP-BLUP model and alternative models under all simulated scenarios and genetic architectures." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-589083922" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Accuracies obtained from SNP-BLUP model and alternative models under all simulated scenarios and genetic architectures.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/10/3333/F3.medium.gif" width="414" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/9/10/3333/F3.medium.gif" width="414" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/10/3333/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/10/3333/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494477" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-46" class="first-child">Accuracies obtained from SNP-BLUP model and alternative models under all simulated scenarios and genetic architectures.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-13" class="subsection"><h3>Computation time</h3><p id="p-47">When the number of markers exceeds the number of individuals, the computational speed and memory requirements can be an important drawback for the use of such models. A solution to this problem is to fit individual effects (<em>i.e.</em>, EGBVs) in an equivalent model instead of SNP effects. In this study all evaluations were performed using the <strong>hglm</strong> R package that fits SNP effects. For a larger number of SNPs the computations would be unfeasible and an equivalent model which uses the external information on each SNP in the same way as in the <strong>hglm</strong> package was implemented in the R package <strong>CodataGS</strong> (<a href="https://cran.r-project.org/web/packages/CodataGS">https://cran.r-project.org/web/packages/CodataGS</a>). The theory is explained in the Supplementary File S3. Fitting individual effects instead of SNP effects resulted in largely improved run time of all models. For a training population of 200 individuals with 2,000 SNP markers, fitting SNP effects (<strong>hglm</strong>) required on average 9.35 sec per iteration while fitting individual effects (<strong>CodataGS</strong>) required only 0.46 sec per iteration (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). The improved speed and memory requirements of the equivalent model can be considerably beneficial since the usual size of the training sets is much larger than the one used here (thousands of individuals with tens of thousands of SNPs). Nonetheless, the speed performance of the equivalent model depends heavily on the number of individuals and the relationship between time and number of individuals is not linear but rather exponential (Supplementary Figure S2).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/10/3333/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Time of execution (seconds per iteration) of SNP-BLUP and alternative models from hglm package and CodataGS package." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-589083922" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Time of execution (seconds per iteration) of SNP-BLUP and alternative models from hglm package and CodataGS package.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/10/3333/F4.medium.gif" width="440" height="197"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/9/10/3333/F4.medium.gif" width="440" height="197"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/10/3333/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/10/3333/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494431" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-48" class="first-child">Time of execution (seconds per iteration) of SNP-BLUP and alternative models from hglm package and CodataGS package.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-14" class="subsection"><h3>German Holstein population results</h3><p id="p-49">To demonstrate the model on real data, we used a German Holstein population consisting of 5024 bulls (<a id="xref-ref-60-4" class="xref-bibr" href="#ref-60">Zhang <em>et al.</em> 2015</a>). We chose to fit the model 1) <strong>SNP-BLUP</strong> and models 2) <strong>W11</strong> and <strong>W41</strong> with windows of sizes 11 and 41 SNPs centered around candidate QTL peaks. We obtained the candidate QTL peaks after performing a GWAS, correcting for genomic relationship using estimated residual and additive genetic variance from GBLUP. All SNPs from the GWAS with p-value less than 10¯<sup>5</sup> were considered a candidate QTL. For milk yield (MY) we identified 6 candidate QTL peaks and for the fat percentage (Fat%) we identified 5 candidate QTL peaks, which were used as the center of the windows.</p><p id="p-50"><a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a> shows the average accuracies obtained from the SNP-BLUP and W41 models for two traits (MY and Fat%) in the fivefold-cross-validation analysis. We present only the results from the W41 model as the model W11 yielded very similar accuracies as the W41 model. For both traits the W41 model yielded higher accuracies than the SNP-BLUP. The W41 model showed a higher advantage in predictive ability for the trait Fat% yielding an accuracy of 0.862 compared to the 0.811 obtained from the SNP-BLUP model. The results for the MY trait were similar but the predictive advantage of the W41 model was lower compared to the Fat% trait (accuracy of 0.785 from the W41 model over 0.771 from the SNP-BLUP model).</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/494354/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/494354/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">MEAN ACCURACY (STANDARD ERROR) OF THE PREDICTED EGBVS IN A 5-FOLD CROSS VALIDATION ANALYSIS USING THE GERMAN HOLSTEIN DATA FOR TWO TRAITS</span><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-15"><h2 class="">Discussion</h2><p id="p-52">The knowledge on the genetic architecture of different traits, and SNP-specific biological information, is increasing rapidly and several authors have proposed methods for genomic selection that can make use of this available biological information to improve selection accuracy (<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Zhang <em>et al.</em> 2010</a>; <a id="xref-ref-59-3" class="xref-bibr" href="#ref-59">Zhang <em>et al.</em> 2014</a>; <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Su <em>et al.</em> 2014</a>). In this line, this study proposes a general model using a link function approach within the hierarchical generalized linear model framework (<a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Lee <em>et al.</em> 2006</a>) to include biological external information into the model. Following <a id="xref-ref-61-2" class="xref-bibr" href="#ref-61">Zhang <em>et al.</em> (2010)</a>, we used a base population of 100 individuals in our simulation study. This is a rather small population size and the results should therefore be extrapolated to larger effective population sizes with caution.</p><p id="p-53">All the results in the current study use the same general model (described by equations 1 – 3) for predicting breeding values. The alternative models in <a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>, including SNP-BLUP, are fitted within this single framework and in the results the accuracies of the alternative models are compared. There are numerous Bayesian models not included within this framework that may be of interest to compare with. However, we use SNP-BLUP as a basic model to compare the results to and study the accuracies of models that make use of external information on the SNPs.</p><p id="p-54">A very attractive feature of the method proposed in this study is that it provides a flexible way to model the SNP variances using a linear predictor (equation 3). Any type of existing knowledge on the SNP markers can be utilized and potentially increase the predictive ability of the model. In this study we investigated the performance of external information related to the position of the QTL on the genome and the relationship of the SNP markers with the QTL and we showed that the inclusion of such information can improve the predicting ability of genomic selection. From our results we identified two main factors that influence the performance of such models, the genetic architecture of the trait and the quality/accuracy of the external information.</p><p id="p-55">In the models W10, W20 and W40, the causative effect is assumed to be within a window and does not assume that the exact position of the causative mutation is known. This model should be suitable for genomic prediction where external information from QTL studies is included. For the LD model and the combined models (W10-LD, W20-LD and W40-LD) it is assumed that the position of the causative SNP is known. Especially in plant breeding, there is a need to include major genes, whose positions are accurately known, in genomic prediction. For such cases the models including LD information combines marker assisted selection and genomic selection in a dynamic way.</p><p id="p-56">We investigated models with three different window sizes that were suitable for our simulated data. For applications on real data the optimal number of markers to be included in each window, in terms of prediction accuracy, will depend on marker density and the genetic architecture. In our application on the dairy cattle data the optimal number of markers within a window was not assessed statistically, but since the marker map was much denser than in the simulated data we chose the model with the largest number of markers, <em>i.e.</em>, window size 40.</p><div id="sec-16" class="subsection"><h3>Genetic architecture of the trait</h3><p id="p-57">The performance of several alternative models in our study was better compared to the SNP-BLUP method when the trait was controlled by a small number of QTL with medium-large effects. The advantage of these models was reduced with increasing number of QTL with smaller effects. However, the alternative models did not result in lower accuracies compared to the SNP-BLUP model. The reason is that as the estimated effect of the external information on the SNP variances approaches zero the model reduces to a SNP-BLUP model. Furthermore, as the number of QTL that control the trait increases, the external information on SNPs becomes more similar among the SNPs. For example, for the categorical models, a QTL is located within most or all defined windows and as a result all SNPs get the same weight in the model. Moreover, most or all SNPs are in LD with a QTL at similar levels. Consequently, the alternative models turn into a SNP-BLUP model. These results are in agreement with the findings of <a id="xref-ref-61-3" class="xref-bibr" href="#ref-61">Zhang <em>et al.</em> (2010)</a>. In their simulation study they investigated the performance of a BLUP model with weighted G matrix and showed that for traits controlled by high number of QTL the traditional GBLUP and their method performed similarly. This effect has also been observed in studies on real data (<a id="xref-ref-59-4" class="xref-bibr" href="#ref-59">Zhang <em>et al.</em> 2014</a>). Analyzing three dairy cattle traits (Milk Yield (MY), Fat percentage (FP) and Somatic Cell Count (SCC)) these authors found that traits controlled by a few QTL with large effects (MY and FP) perform better under models with external information on the SNPs while the SCC trait, that is controlled by many QTL evenly distributed along the genome, performed better under the standard GBLUP model.</p><p id="p-58">In our simulation study we created different genetic architectures for the trait with respect not only to the number of the QTL affecting the trait but also to the distribution of the QTL effects and their variances (see Material and Methods). Our results showed that this aspect did not affect the performance of the alternative models. Moreover, the additional models that included information on the variance distribution across the genome were not able to provide any benefit, contrary to methods that assume mixtures of distributions for the SNP markers like Bayesian methods (<a id="xref-ref-13-3" class="xref-bibr" href="#ref-13">Erbe <em>et al.</em> 2012</a>).</p></div><div id="sec-17" class="subsection"><h3>External information</h3><p id="p-59">In this study we investigated the performance of models that include information on the location of the QTL on the genome (categorical models) and thereby tried to mimic the external information available on the QTL databases and the different window sizes resemble the degree of uncertainty of a QTL region. Our results indicate that this type of external information has the potential to improve the accuracy of genomic selection and that the degree of improvement is inversely related to the degree of uncertainty on the QTL region. The usefulness of the QTL database information has been demonstrated by <a id="xref-ref-59-5" class="xref-bibr" href="#ref-59">Zhang <em>et al.</em> (2014)</a>. In their study these authors searched for reported QTL on the traits under consideration (Fat percentage, milk yield and somatic cell score for dairy cattle and several traits for rice) and after a quality control to avoid the possible false positive reports they included this information into a GBLUP model. For most of the examined traits an increase in accuracy was observed, especially for the traits that showed a characteristic genetic architecture. The discovery of new QTL or the causative mutations is expected to increase in the future with the use of whole genome sequence and the development of new methods for analysis and as a consequence the information available will become more accurate.</p><p id="p-60">The external information that proved to be more valuable in this study was the LD estimates between the SNPs and the QTL. In the standard GBLUP method, markers in linkage equilibrium (LE) to the causative QTL tend to capture effects due to family relationship, whereas mainly markers in LD capture the QTL effects themselves (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Habier <em>et al.</em> 2007</a>, <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">de los Campos <em>et al.</em> 2015</a>). In the BayesB model (<a id="xref-ref-38-7" class="xref-bibr" href="#ref-38">Meuwissen <em>et al.</em> 2001</a>), the prior for the SNP variances is a mixture of two distributions that tends to group markers into two classes: those in LD and those in LE with the QTL. By modeling the two classes of markers better predictions for unrelated individuals can be obtained. In other studies, LD information has been incorporated in a model for the marker variances, which smooths the effects between markers in close LD (<em>e.g.</em>, the Bayesian antedependence model by <a id="xref-ref-58-2" class="xref-bibr" href="#ref-58">Yang and Tempelman 2012</a>, and the double hierarchical generalized linear model by <a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Rönnegård and Lee 2010</a>), and thereby captures the QTL effects rather than family information. These models give better predictions than GBLUP when individuals are unrelated and the total number of QTL is small. This is in line with our findings where the models including LD between markers and QTL resulted in improved prediction accuracies, especially when the number of simulated QTL was small. Finally, the results obtained from the combined models indicate that information on the real relationship between markers and QTL can compensate for the loss of information due to the uncertainty of the QTL report.</p><p id="p-61">The prior of BayesB is rather general because it does not use any external information on the SNPs, whereas the model we propose gives more specific information about each SNP. Since the information on each SNP is more specific in our model its performance compared to GBPLUP and BayesB is expected to improve as the number of individuals in the training set decreases, in line with the results of <a id="xref-ref-60-5" class="xref-bibr" href="#ref-60">Zhang <em>et al.</em> (2015</a>, Supplementary Table 1) .</p><p id="p-62">The model applied in <a id="xref-ref-60-6" class="xref-bibr" href="#ref-60">Zhang <em>et al.</em> (2015)</a> is BLUP|GA and was developed in <a id="xref-ref-59-6" class="xref-bibr" href="#ref-59">Zhang <em>et al.</em> (2014)</a>. It includes external data on SNPs in the model and has similarities to our model since both methods fit trait-specific genomic relationship matrices. In the BLUP|GA method SNPs are divided into two groups by the user. In the first group there is a single genetic variance for all SNPs and in the second group SNP-specific variances are modeled as proportional to user-specific weights. Furthermore, the ratio between the variances for the two groups is also user-specified. This is indeed similar to our proposed method, but with some significant differences. The method that we propose uses a regression approach where covariates are specified by the user, whereas all model parameters are estimated. The covariates can include negative values in our method but the SNP variances will still be positive because the genetic variances are modeled using a logarithmic link function. By specifying covariates rather than weights for the SNP variances, hopefully, our proposed method will also be user friendly and the implementation in the CodataGS package (<a href="https://cran.r-project.org/web/packages/CodataGS">https://cran.r-project.org/web/packages/CodataGS</a>) fits rather well with the regression framework in R.</p></div><div id="sec-18" class="subsection"><h3>CONCLUSIONS</h3><p id="p-63">In this study we investigated the potential benefit of external information on improving the accuracy of genomic selection. In conclusion, using external information to model SNP-specific variances can provide gains in accuracy compared to the traditional SNP-BLUP. Nonetheless, the level of gain depends on the genetic architecture of the trait of interest and the quality of the external information on the SNP markers. The usefulness of these type of models is expected to increase with time as more accurate information on the SNPs becomes available. Finally, our analysis on real data indicated that the proposed method has potential but further studies are required to confirm the advantage of this approach.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-64">This project was supported by the Mistra Biotech project, a research program financed by Mistra – the Swedish foundation for strategic environmental research, and the Swedish University of Agricultural Sciences, SLU. M. Selle acknowledges the financial support given by the Research Council of Norway, grant number 250362.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-5"><p id="p-65">Supplemental material available at FigShare: <a href="https://doi.org/10.25387/g3.9247832">https://doi.org/10.25387/g3.9247832</a>.</p></li><li class="fn" id="fn-6"><p id="p-66"><em>Communicating editor: L. McIntyre</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2019-05-30"><span class="received-label">Received </span>May 30, 2019.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2019-08-09"><span class="accepted-label">Accepted </span>August 9, 2019.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2019 Mouresan <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3333.1" data-doi="10.1186/s12711-016-0187-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Abdollahi-Arpanahi</span>,  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Morota</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. D.</span> <span class="cit-name-surname">Valente</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Kranis</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G. J. M.</span> <span class="cit-name-surname">Rosa</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Differential contribution of genomic regions to marked genetic variation and prediction of quantitative traits in broiler chickens.</span> <abbr class="cit-jnl-abbrev">Genet Sel Evol: GSE</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">10</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12711-016-0187-z</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet%2BSel%2BEvol%253A%2BGSE%26rft.volume%253D48%26rft.spage%253D10%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12711-016-0187-z%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12711-016-0187-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3333.2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Aitkin</span>,  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span> <span class="cit-article-title">Modelling variance heterogeneity in normal regression using GLIM.</span> <abbr class="cit-jnl-abbrev">J. R. Stat. Soc. Ser. C Appl. Stat.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">332</span>–<span class="cit-lpage">339</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BR.%2BStat.%2BSoc.%2BSer.%2BC%2BAppl.%2BStat.%26rft.volume%253D36%26rft.spage%253D332%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3333.3" data-doi="10.1371/journal.pcbi.1002822"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bush</span>,  <span class="cit-name-given-names">W. S.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. H.</span> <span class="cit-name-surname">Moore</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Chapter 11: Genome-Wide Association Studies.</span> <abbr class="cit-jnl-abbrev">PLOS Comput. Biol.</abbr> <span class="cit-vol">8</span>: <span class="cit-elocation-id">e1002822</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pcbi.1002822</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLOS%2BComput.%2BBiol.%26rft.volume%253D8%26rft.spage%253De1002822%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.1002822%26rft_id%253Dinfo%253Apmid%252F23300413%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.1002822&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23300413&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3333.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=23300413&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3333.4" data-doi="10.1534/genetics.108.096826"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Carneiro</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Ferrand</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. W.</span> <span class="cit-name-surname">Nachman</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Recombination and Speciation: Loci near Centromeres Are More Differentiated than Loci near Telomeres between Subspecies of the European Rabbit (Oryctolagus Cuniculus).</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">181</span>: <span class="cit-fpage">593</span>–<span class="cit-lpage">606</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.108.096826</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.108.096826%26rft_id%253Dinfo%253Apmid%252F19015539%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTgxLzIvNTkzIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMC8zMzMzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3333.5" data-doi="10.1093/nar/gkr1014"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Caspi</span>,  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Altman</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Dreher</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. A.</span> <span class="cit-name-surname">Fulcher</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Subhraveti</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The MetaCyc database of metabolic pathways and enzymes and the BioCyc collection of pathway/genome databases.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">D742</span>–<span class="cit-lpage">D753</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkr1014</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkr1014%26rft_id%253Dinfo%253Apmid%252F22102576%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkr1014&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22102576&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3333.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000298601300112&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3333.6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Croft</span>,  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">O’Kelly</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Wu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Haw</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Gillespie</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> Reactome: A Database of Reactions, Pathways and Biological Processes. Nucleic Acids Res. 39 (SUPPL. 1).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3333.7" data-doi="10.1038/ng.3034"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Daetwyler</span>,  <span class="cit-name-given-names">H. D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Capitan</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Pausch</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Stothard</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">van Binsbergen</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Whole-genome sequencing of 234 bulls facilitates mapping of monogenic and complex traits in cattle.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">858</span>–<span class="cit-lpage">865</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.3034</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D46%26rft.spage%253D858%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.3034%26rft_id%253Dinfo%253Apmid%252F25017103%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.3034&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25017103&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3333.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3333.8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Daetwyler</span>,  <span class="cit-name-given-names">H. D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. P. L.</span> <span class="cit-name-surname">Calus</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Pong-Wong</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">de los Campos</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">Hickey</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> Genomic Prediction in Animals and Plants: Simulation of Data, Validation, Reporting, and Benchmarking. Genetics 193: 347 LP-365.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3333.9" data-doi="10.1534/genetics.110.116855"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Daetwyler</span>,  <span class="cit-name-given-names">H. D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Pong-Wong</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Villanueva</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. A.</span> <span class="cit-name-surname">Woolliams</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The impact of genetic architecture on genome-wide evaluation methods.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">185</span>: <span class="cit-fpage">1021</span>–<span class="cit-lpage">1031</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.110.116855</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.116855%26rft_id%253Dinfo%253Apmid%252F20407128%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE4NS8zLzEwMjEiO3M6NDoiYXRvbSI7czoxOToiL2dnZy85LzEwLzMzMzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3333.10" data-doi="10.1371/journal.pgen.1005048"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>,  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Sorensen</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Gianola</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genomic heritability: What is it?</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-elocation-id">e1005048</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1005048</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D11%26rft.spage%253De1005048%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1005048%26rft_id%253Dinfo%253Apmid%252F25942577%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1005048&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25942577&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3333.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3333.11" data-doi="10.1534/genetics.109.101501"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>,  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Naya</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Gianola</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Crossa</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Legarra</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Predicting quantitative traits with regression models for dense molecular markers and pedigree.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">182</span>: <span class="cit-fpage">375</span>–<span class="cit-lpage">385</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.109.101501</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.109.101501%26rft_id%253Dinfo%253Apmid%252F19293140%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTgyLzEvMzc1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMC8zMzMzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3333.12" data-doi="10.2527/jas.2014-8640"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Do</span>,  <span class="cit-name-given-names">D. N.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L. L. G.</span> <span class="cit-name-surname">Janss</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Jensen</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">H. N.</span> <span class="cit-name-surname">Kadarmideen</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">SNP Annotation-Based Whole Genomic Prediction and Selection : An Application to Feed Efficiency and Its Component Traits in Pigs.</span> <abbr class="cit-jnl-abbrev">J. Anim. Sci.</abbr> <span class="cit-vol">93</span>: <span class="cit-fpage">2056</span>–<span class="cit-lpage">2063</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2527/jas.2014-8640</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BAnimal%2BScience%26rft.stitle%253DJ%2BANIM%2BSCI%26rft.aulast%253DDo%26rft.auinit1%253DD.%2BN.%26rft.volume%253D93%26rft.issue%253D5%26rft.spage%253D2056%26rft.epage%253D2063%26rft.atitle%253DSNP%2Bannotation-based%2Bwhole%2Bgenomic%2Bprediction%2Band%2Bselection%253A%2BAn%2Bapplication%2Bto%2Bfeed%2Befficiency%2Band%2Bits%2Bcomponent%2Btraits%2Bin%2Bpigs%26rft_id%253Dinfo%253Adoi%252F10.2527%252Fjas.2014-8640%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToiYW5pbWFsc2NpIjtzOjU6InJlc2lkIjtzOjk6IjkzLzUvMjA1NiI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzkvMTAvMzMzMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3333.13" data-doi="10.3168/jds.2011-5019"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Erbe</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. J.</span> <span class="cit-name-surname">Hayes</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L. K.</span> <span class="cit-name-surname">Matukumalli</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Goswami</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. J.</span> <span class="cit-name-surname">Bowman</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Improving accuracy of genomic predictions within and between dairy cattle breeds with imputed high-density single nucleotide polymorphism panels.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">95</span>: <span class="cit-fpage">4114</span>–<span class="cit-lpage">4129</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3168/jds.2011-5019</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DErbe%26rft.auinit1%253DM.%26rft.volume%253D95%26rft.issue%253D7%26rft.spage%253D4114%26rft.epage%253D4129%26rft.atitle%253DImproving%2Baccuracy%2Bof%2Bgenomic%2Bpredictions%2Bwithin%2Band%2Bbetween%2Bdairy%2Bcattle%2Bbreeds%2Bwith%2Bimputed%2Bhigh-density%2Bsingle%2Bnucleotide%2Bpolymorphism%2Bpanels.%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2011-5019%26rft_id%253Dinfo%253Apmid%252F22720968%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2011-5019&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22720968&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3333.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000305363100062&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3333.14"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Falconer</span>,  <span class="cit-name-given-names">D. S.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. F. C.</span> <span class="cit-name-surname">Mckay</span></span>, </li></ol><cite><span class="cit-pub-date">1996</span> <span class="cit-article-title">Introduction to Quantitative Genetics. Ed. 4. Longmans Green, Harlow, Essex.</span> </cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3333.15" data-doi="10.1186/s12863-015-0278-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gao</span>,  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Li</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">He</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Xiao</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Luo</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Improving accuracy of genomic prediction by genetic architecture based priors in a Bayesian model.</span> <abbr class="cit-jnl-abbrev">BMC Genet.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">120</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12863-015-0278-9</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenet.%26rft.volume%253D16%26rft.spage%253D120%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12863-015-0278-9%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12863-015-0278-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3333.16" data-doi="10.1534/genetics.113.151753"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>,  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Priors in whole-genome regression: The Bayesian alphabet returns.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">194</span>: <span class="cit-fpage">573</span>–<span class="cit-lpage">596</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.113.151753</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.151753%26rft_id%253Dinfo%253Apmid%252F23636739%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk0LzMvNTczIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMC8zMzMzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3333.17" data-doi="10.1186/1297-9686-41-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">González-Recio</span>,  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Gianola</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G. J. M.</span> <span class="cit-name-surname">Rosa</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. A.</span> <span class="cit-name-surname">Weigel</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Kranis</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Genome-Assisted Prediction of a Quantitative Trait Measured in Parents and Progeny: Application to Food Conversion Rate in Chickens.</span> <abbr class="cit-jnl-abbrev">Genet Sel Evol: GSE</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">3</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1297-9686-41-3</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DGonzalez-Recio%26rft.auinit1%253DO.%26rft.volume%253D41%26rft.spage%253D3%26rft.epage%253D3%26rft.atitle%253DGenome-assisted%2Bprediction%2Bof%2Ba%2Bquantitative%2Btrait%2Bmeasured%2Bin%2Bparents%2Band%2Bprogeny%253A%2Bapplication%2Bto%2Bfood%2Bconversion%2Brate%2Bin%2Bchickens.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-41-3%26rft_id%253Dinfo%253Apmid%252F19284693%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-41-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19284693&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3333.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3333.18" data-doi="10.1038/ng1547"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gunderson</span>,  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F. J.</span> <span class="cit-name-surname">Steemers</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Lee</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L. G.</span> <span class="cit-name-surname">Mendoza</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. S.</span> <span class="cit-name-surname">Chee</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">A genome-wide scalable SNP genotyping assay using microarray technology.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">549</span>–<span class="cit-lpage">554</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng1547</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DGunderson%26rft.auinit1%253DK.%2BL.%26rft.volume%253D37%26rft.issue%253D5%26rft.spage%253D549%26rft.epage%253D554%26rft.atitle%253DA%2Bgenome-wide%2Bscalable%2BSNP%2Bgenotyping%2Bassay%2Busing%2Bmicroarray%2Btechnology.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1547%26rft_id%253Dinfo%253Apmid%252F15838508%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1547&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15838508&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3333.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000228862300027&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3333.19" data-doi="10.1534/genetics.107.081190"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Habier</span>,  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. L.</span> <span class="cit-name-surname">Fernando</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. C. M.</span> <span class="cit-name-surname">Dekkers</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">The impact of genetic relationship information on genome-assisted breeding values.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">177</span>: <span class="cit-fpage">2389</span>–<span class="cit-lpage">2397</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.107.081190</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DHabier%26rft.auinit1%253DD.%26rft.volume%253D177%26rft.issue%253D4%26rft.spage%253D2389%26rft.epage%253D2397%26rft.atitle%253DThe%2BImpact%2Bof%2BGenetic%2BRelationship%2BInformation%2Bon%2BGenome-Assisted%2BBreeding%2BValues%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.081190%26rft_id%253Dinfo%253Apmid%252F18073436%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3Ny80LzIzODkiO3M6NDoiYXRvbSI7czoxOToiL2dnZy85LzEwLzMzMzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3333.20" data-doi="10.1186/1471-2105-12-186"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Habier</span>,  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. L.</span> <span class="cit-name-surname">Fernando</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Kizilkaya</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. J.</span> <span class="cit-name-surname">Garrick</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Extension of the bayesian alphabet for genomic selection.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">186</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2105-12-186</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DHabier%26rft.auinit1%253DD.%26rft.volume%253D12%26rft.spage%253D186%26rft.epage%253D186%26rft.atitle%253DExtension%2Bof%2Bthe%2Bbayesian%2Balphabet%2Bfor%2Bgenomic%2Bselection.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-12-186%26rft_id%253Dinfo%253Apmid%252F21605355%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-12-186&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21605355&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3333.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3333.21" data-doi="10.1371/journal.pgen.1001139"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>,  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Pryce</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. J.</span> <span class="cit-name-surname">Chamberlain</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. J.</span> <span class="cit-name-surname">Bowman</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. E.</span> <span class="cit-name-surname">Goddard</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genetic Architecture of Complex Traits and Accuracy of Genomic Prediction: Coat Colour, Milk-Fat Percentage, and Type in Holstein Cattle as Contrasting Model Traits.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">6</span>: <span class="cit-elocation-id">e1001139</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1001139</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DHayes%26rft.auinit1%253DB.%2BJ.%26rft.volume%253D6%26rft.issue%253D9%26rft.spage%253De1001139%26rft.epage%253De1001139%26rft.atitle%253DGenetic%2Barchitecture%2Bof%2Bcomplex%2Btraits%2Band%2Baccuracy%2Bof%2Bgenomic%2Bprediction%253A%2Bcoat%2Bcolour%252C%2Bmilk-fat%2Bpercentage%252C%2Band%2Btype%2Bin%2BHolstein%2Bcattle%2Bas%2Bcontrasting%2Bmodel%2Btraits.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1001139%26rft_id%253Dinfo%253Apmid%252F20927186%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1001139&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20927186&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3333.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3333.22" data-doi="10.3168/jds.2008-1646"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>,  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. J.</span> <span class="cit-name-surname">Bowman</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. J.</span> <span class="cit-name-surname">Chamberlain</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. E.</span> <span class="cit-name-surname">Goddard</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Invited review: Genomic selection in dairy cattle: progress and challenges.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">92</span>: <span class="cit-fpage">433</span>–<span class="cit-lpage">443</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3168/jds.2008-1646</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DHayes%26rft.auinit1%253DB.%2BJ.%26rft.volume%253D92%26rft.issue%253D2%26rft.spage%253D433%26rft.epage%253D443%26rft.atitle%253DInvited%2Breview%253A%2BGenomic%2Bselection%2Bin%2Bdairy%2Bcattle%253A%2BProgress%2Band%2Bchallenges%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2008-1646%26rft_id%253Dinfo%253Apmid%252F19164653%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2008-1646&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19164653&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3333.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000262654900001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3333.23" data-doi="10.1016/j.biosystems.2008.12.004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hecker</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Lambeck</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Toepfer</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">van Someren</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Guthke</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Gene regulatory network inference: Data integration in dynamic models-a review.</span> <abbr class="cit-jnl-abbrev">Biosystems</abbr> <span class="cit-vol">96</span>: <span class="cit-fpage">86</span>–<span class="cit-lpage">103</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.biosystems.2008.12.004</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiosystems%26rft.stitle%253DBiosystems%26rft.aulast%253DHecker%26rft.auinit1%253DM.%26rft.volume%253D96%26rft.issue%253D1%26rft.spage%253D86%26rft.epage%253D103%26rft.atitle%253DGene%2Bregulatory%2Bnetwork%2Binference%253A%2Bdata%2Bintegration%2Bin%2Bdynamic%2Bmodels-a%2Breview.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.biosystems.2008.12.004%26rft_id%253Dinfo%253Apmid%252F19150482%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.biosystems.2008.12.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19150482&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3333.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000264693500011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3333.24"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hidalgo</span>,  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. W. M.</span> <span class="cit-name-surname">Bastiaansen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. S.</span> <span class="cit-name-surname">Lopes</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Harlizius</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. A. M.</span> <span class="cit-name-surname">Groenen</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> Accuracy of predicted genomic breeding values in purebred and crossbred pigs. G3 (Bethesda) 5: 1575–1583.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3333.25" data-doi="10.1093/nar/gks1150"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>,  <span class="cit-name-given-names">Z. L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. A.</span> <span class="cit-name-surname">Park</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X. L.</span> <span class="cit-name-surname">Wu</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">Reecy</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Animal QTLdb: An improved database tool for livestock animal QTL/association data dissemination in the post-genome era.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">D871</span>–<span class="cit-lpage">D879</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gks1150</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgks1150%26rft_id%253Dinfo%253Apmid%252F23180796%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gks1150&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23180796&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3333.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000312893300124&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3333.26" data-doi="10.3168/jds.S0022-0302(00)74973-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jaffrezic</span>,  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">I. M. S.</span> <span class="cit-name-surname">White</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Thompson</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">W. G.</span> <span class="cit-name-surname">Hill</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">A Link Function Approach to Model Heterogeneity of Residual Variances Over Time in Lactation Curve Analyses.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">83</span>: <span class="cit-fpage">1089</span>–<span class="cit-lpage">1093</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3168/jds.S0022-0302(00)74973-3</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DJaffrezic%26rft.auinit1%253DF.%26rft.volume%253D83%26rft.issue%253D5%26rft.spage%253D1089%26rft.epage%253D1093%26rft.atitle%253DA%2BLink%2BFunction%2BApproach%2Bto%2BModel%2BHeterogeneity%2Bof%2BResidual%2BVariances%2BOver%2BTime%2Bin%2BLactation%2BCurve%2BAnalyses%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.S0022-0302%252800%252974973-3%26rft_id%253Dinfo%253Apmid%252F10821584%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.S0022-0302(00)74973-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10821584&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3333.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000087005900020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3333.27" data-doi="10.1016/j.livsci.2014.04.028"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kadarmideen</span>,  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genomics to systems biology in animal and veterinary sciences: Progress, lessons and opportunities.</span> <abbr class="cit-jnl-abbrev">Livest. Sci.</abbr> <span class="cit-vol">166</span>: <span class="cit-fpage">232</span>–<span class="cit-lpage">248</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.livsci.2014.04.028</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLivest.%2BSci.%26rft.volume%253D166%26rft.spage%253D232%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.livsci.2014.04.028%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.livsci.2014.04.028&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3333.28"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kanehisa</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Araki</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Goto</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Hattori</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Hirakawa</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> KEGG for Linking Genomes to Life and the Environment. Nucleic Acids Res. 36 (SUPPL. 1).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3333.29" data-doi="10.1186/1471-2164-15-436"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Koufariotis</span>,  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y. P.</span> <span class="cit-name-surname">Chen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Bolormaa</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. J.</span> <span class="cit-name-surname">Hayes</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. J.</span> <span class="cit-name-surname">Schork</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> Regulatory and Coding Genome Regions Are Enriched for Trait Associated Variants in Dairy and Beef Cattle. BMC Genomics 15 (1). BioMed Central: 436.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2164-15-436</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGENOM%26rft.volume%253D15%26rft.spage%253D436%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-15-436%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-15-436&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3333.30" data-doi="10.1126/science.1075090"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>,  <span class="cit-name-given-names">T. I.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N. J.</span> <span class="cit-name-surname">Rinaldi</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Robert</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. T.</span> <span class="cit-name-surname">Odom</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Bar-Joseph</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Transcriptional regulatory networks in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">298</span>: <span class="cit-fpage">799</span>–<span class="cit-lpage">804</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.1075090</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DLee%26rft.auinit1%253DT.%2BI.%26rft.volume%253D298%26rft.issue%253D5594%26rft.spage%253D799%26rft.epage%253D804%26rft.atitle%253DTranscriptional%2BRegulatory%2BNetworks%2Bin%2BSaccharomyces%2Bcerevisiae%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1075090%26rft_id%253Dinfo%253Apmid%252F12399584%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIyOTgvNTU5NC83OTkiO3M6NDoiYXRvbSI7czoxOToiL2dnZy85LzEwLzMzMzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3333.31"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>,  <span class="cit-name-given-names">Y.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. A.</span> <span class="cit-name-surname">Nelder</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Hierarchical Generalized Linear Models.</span> <abbr class="cit-jnl-abbrev">J. R. Stat. Soc. B</abbr> <span class="cit-vol">58</span>: <span class="cit-fpage">619</span>–<span class="cit-lpage">678</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BR.%2BStat.%2BSoc.%2BB%26rft.volume%253D58%26rft.spage%253D619%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3333.32" data-doi="10.2307/3315676"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>,  <span class="cit-name-given-names">Y.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. A.</span> <span class="cit-name-surname">Nelder</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Generalized Linear Models for the Analysis of Quality-Improvement Experiments.</span> <abbr class="cit-jnl-abbrev">Can. J. Stat.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">95</span>–<span class="cit-lpage">105</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2307/3315676</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCan.%2BJ.%2BStat.%26rft.volume%253D26%26rft.spage%253D95%26rft_id%253Dinfo%253Adoi%252F10.2307%252F3315676%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/3315676&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-book" id="cit-9.10.3333.33" data-doi="10.1201/9781420011340"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>,  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. A.</span> <span class="cit-name-surname">Nelder</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Pawitan</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-source">Generalized Linear Models with Random Effects: Unified Analysis via H-likelihood</span>, <span class="cit-publ-name">Chapman &amp; Hall/CRC</span>, <span class="cit-publ-loc">Boca Raton</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1201/9781420011340</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGeneralized%2BLinear%2BModels%2Bwith%2BRandom%2BEffects%253A%2BUnified%2BAnalysis%2Bvia%2BH-likelihood%26rft_id%253Dinfo%253Adoi%252F10.1201%252F9781420011340%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1201/9781420011340&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-book" id="cit-9.10.3333.34" data-doi="10.1201/9781315211060"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>,  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Rönnegård</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Noh</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-source">Data Analysis Using Hierarchical Generalized Linear Models with R</span>, <span class="cit-publ-name">Chapman and Hall/CRC</span>, <span class="cit-publ-loc">Boca Raton</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1201/9781315211060</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DData%2BAnalysis%2BUsing%2BHierarchical%2BGeneralized%2BLinear%2BModels%2Bwith%2BR%26rft_id%253Dinfo%253Adoi%252F10.1201%252F9781315211060%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1201/9781315211060&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3333.35" data-doi="10.1534/genetics.108.088575"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Legarra</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Robert-Granié</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Manfredi</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">Elsen</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Performance of Genomic Selection in Mice.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">180</span>: <span class="cit-fpage">611</span>–<span class="cit-lpage">618</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.108.088575</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.108.088575%26rft_id%253Dinfo%253Apmid%252F18757934%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTgwLzEvNjExIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMC8zMzMzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3333.36" data-doi="10.1534/genetics.109.107391"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Luan</span>,  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. A.</span> <span class="cit-name-surname">Woolliams</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Lien</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Kent</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Svendsen</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The Accuracy of Genomic Selection in Norwegian Red Cattle Assessed by Cross-Validation.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">183</span>: <span class="cit-fpage">1119</span>–<span class="cit-lpage">1126</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.109.107391</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DGenetics%26rft.aulast%253DLuan%26rft.auinit1%253DT.%26rft.volume%253D183%26rft.issue%253D3%26rft.spage%253D1119%26rft.epage%253D1126%26rft.atitle%253DThe%2Baccuracy%2Bof%2BGenomic%2BSelection%2Bin%2BNorwegian%2Bred%2Bcattle%2Bassessed%2Bby%2Bcross-validation.%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.109.107391%26rft_id%253Dinfo%253Apmid%252F19704013%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/genetics.109.107391&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19704013&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3333.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000272295800029&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3333.37" data-doi="10.1186/s12864-016-2443-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">MacLeod</span>,  <span class="cit-name-given-names">I. M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. J.</span> <span class="cit-name-surname">Bowman</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. J.</span> <span class="cit-name-surname">Vander Jagt</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Haile-Mariam</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. E.</span> <span class="cit-name-surname">Kemper</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Exploiting biological priors and sequence variants enhances QTL discovery and genomic prediction of complex traits.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">144</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12864-016-2443-6</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D17%26rft.spage%253D144%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864-016-2443-6%26rft_id%253Dinfo%253Apmid%252F26920147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12864-016-2443-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26920147&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3333.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3333.38"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>,  <span class="cit-name-given-names">T. H. E.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. J.</span> <span class="cit-name-surname">Hayes</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. E.</span> <span class="cit-name-surname">Goddard</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Prediction of total genetic value using genome-wide dense marker maps.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">157</span>: <span class="cit-fpage">1819</span>–<span class="cit-lpage">1829</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D157%26rft.spage%253D1819%26rft_id%253Dinfo%253Apmid%252F11290733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=11290733&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3333.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000168223400036&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3333.39" data-doi="10.1186/1471-2164-15-109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Morota</span>,  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Abdollahi-Arpanahi</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Kranis</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Gianola</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> Genome-Enabled Prediction of Quantitative Traits in Chickens Using Genomic Annotation. BMC Genomics 15 (1). BioMed Central: 109.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2164-15-109</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGENOM%26rft.volume%253D15%26rft.spage%253D109%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-15-109%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-15-109&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3333.40" data-doi="10.1111/j.1439-0388.2007.00700.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Muir</span>,  <span class="cit-name-given-names">W. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Comparison of Genomic and Traditional BLUP-Estimated Breeding Value Accuracy and Selection Response under Alternative Trait and Genomic Parameters.</span> <abbr class="cit-jnl-abbrev">J. Anim. Breed. Genet.</abbr> <span class="cit-vol">124</span>: <span class="cit-fpage">342</span>–<span class="cit-lpage">355</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1439-0388.2007.00700.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DJ%2BAnim%2BBreed%2BGenet%26rft.aulast%253DMuir%26rft.auinit1%253DW.%2BM.%26rft.volume%253D124%26rft.issue%253D6%26rft.spage%253D342%26rft.epage%253D355%26rft.atitle%253DComparison%2Bof%2Bgenomic%2Band%2Btraditional%2BBLUP-estimated%2Bbreeding%2Bvalue%2Baccuracy%2Band%2Bselection%2Bresponse%2Bunder%2Balternative%2Btrait%2Band%2Bgenomic%2Bparameters.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1439-0388.2007.00700.x%26rft_id%253Dinfo%253Apmid%252F18076471%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1439-0388.2007.00700.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18076471&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3333.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000251432800004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3333.41" data-doi="10.1186/1297-9686-43-38"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ostersen</span>,  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">O. F.</span> <span class="cit-name-surname">Christensen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Henryon</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Nielsen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Su</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Deregressed EBV as the Response Variable Yield More Reliable Genomic Predictions than Traditional EBV in Pure-Bred Pigs.</span> <abbr class="cit-jnl-abbrev">Genet Sel Evol: GSE</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">38</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1297-9686-43-38</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DOstersen%26rft.auinit1%253DT.%26rft.volume%253D43%26rft.spage%253D38%26rft.epage%253D38%26rft.atitle%253DDeregressed%2BEBV%2Bas%2Bthe%2Bresponse%2Bvariable%2Byield%2Bmore%2Breliable%2Bgenomic%2Bpredictions%2Bthan%2Btraditional%2BEBV%2Bin%2Bpure-bred%2Bpigs.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-43-38%26rft_id%253Dinfo%253Apmid%252F22070746%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-43-38&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22070746&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3333.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-confproc" id="cit-9.10.3333.42"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rönnegård</span>,  <span class="cit-name-given-names">L.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Lee</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> Hierarchical generalized linear models have a great potential in genetics and animal breeding. In proceedings: World Congress on Genetics Applied to Livestock Production, Leipzig, Germany.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3333.43" data-doi="10.1186/1297-9686-42-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rönnegård</span>,  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Felleki</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Fikse</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H. A.</span> <span class="cit-name-surname">Mulder</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Strandberg</span></span></li></ol><cite>, <span class="cit-pub-date">2010a</span> <span class="cit-article-title">Genetic heterogeneity of residual variance - estimation of variance components using double hierarchical generalized linear models.</span> <abbr class="cit-jnl-abbrev">Genet Sel Evol: GSE</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">8</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1297-9686-42-8</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet%2BSel%2BEvol%253A%2BGSE%26rft.volume%253D42%26rft.spage%253D8%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-42-8%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-42-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3333.44" data-doi="10.32614/RJ-2010-009"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rönnegård</span>,  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Shen</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Alam</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span><span class="cit-comment">b</span> <span class="cit-article-title">hglm: A Package for Fitting Hierarchical Generalized Linear Models.</span> <abbr class="cit-jnl-abbrev">R J.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">20</span>–<span class="cit-lpage">28</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.32614/RJ-2010-009</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DR%2BJ.%26rft.volume%253D2%26rft.spage%253D20%26rft_id%253Dinfo%253Adoi%252F10.32614%252FRJ-2010-009%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.32614/RJ-2010-009&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3333.45" data-doi="10.1371/journal.pgen.1003449"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schork</span>,  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W. K.</span> <span class="cit-name-surname">Thompson</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Pham</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Torkamani</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. C.</span> <span class="cit-name-surname">Roddey</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">All SNPs are not created equal: genome-wide association studies reveal a consistent pattern of enrichment among functionally annotated SNPs.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">9</span>: <span class="cit-elocation-id">e1003449</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1003449</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D9%26rft.spage%253De1003449%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003449%26rft_id%253Dinfo%253Apmid%252F23637621%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003449&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23637621&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3333.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3333.46" data-doi="10.1371/journal.pcbi.0030131"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shalgi</span>,  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Lieber</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Oren</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Pilpel</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Global and local architecture of the mammalian microRNA-transcription factor regulatory network.</span> <abbr class="cit-jnl-abbrev">PLOS Comput. Biol.</abbr> <span class="cit-vol">3</span>: <span class="cit-elocation-id">e131</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pcbi.0030131</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BComput%2BBiol%26rft.aulast%253DShalgi%26rft.auinit1%253DR.%26rft.volume%253D3%26rft.issue%253D7%26rft.spage%253De131%26rft.epage%253De131%26rft.atitle%253DGlobal%2Band%2Blocal%2Barchitecture%2Bof%2Bthe%2Bmammalian%2BmicroRNA-transcription%2Bfactor%2Bregulatory%2Bnetwork.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.0030131%26rft_id%253Dinfo%253Apmid%252F17630826%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.0030131&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17630826&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3333.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3333.47" data-doi="10.1534/genetics.112.146720"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shen</span>,  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Alam</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Fikse</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Rönnegård</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">A Novel Generalized Ridge Regression Method for Quantitative Genetics.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">193</span>: <span class="cit-fpage">1255</span>–<span class="cit-lpage">1268</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.112.146720</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.146720%26rft_id%253Dinfo%253Apmid%252F23335338%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5My80LzEyNTUiO3M6NDoiYXRvbSI7czoxOToiL2dnZy85LzEwLzMzMzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3333.48" data-doi="10.2527/jas.2012-5784"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Snelling</span>,  <span class="cit-name-given-names">W. M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. A.</span> <span class="cit-name-surname">Cushman</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. W.</span> <span class="cit-name-surname">Keele</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Maltecca</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. G.</span> <span class="cit-name-surname">Thomas</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">BREEDING AND GENETICS SYMPOSIUM: Networks and pathways to guide genomic selection.</span> <abbr class="cit-jnl-abbrev">J. Anim. Sci.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">537</span>–<span class="cit-lpage">552</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2527/jas.2012-5784</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAnim.%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.2527%252Fjas.2012-5784%26rft_id%253Dinfo%253Apmid%252F23097404%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2527/jas.2012-5784&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23097404&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3333.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3333.49" data-doi="10.1186/1297-9686-41-37"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sonesson</span>,  <span class="cit-name-given-names">A. K.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. H. E.</span> <span class="cit-name-surname">Meuwissen</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Testing Strategies for Genomic Selection in Aquaculture Breeding Programs.</span> <abbr class="cit-jnl-abbrev">Genet Sel Evol: GSE</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">37</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1297-9686-41-37</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DSonesson%26rft.auinit1%253DA.%2BK.%26rft.volume%253D41%26rft.spage%253D37%26rft.epage%253D37%26rft.atitle%253DTesting%2Bstrategies%2Bfor%2Bgenomic%2Bselection%2Bin%2Baquaculture%2Bbreeding%2Bprograms.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-41-37%26rft_id%253Dinfo%253Apmid%252F19566932%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-41-37&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19566932&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3333.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3333.50" data-doi="10.1017/S0016672303006426"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sorensen</span>,  <span class="cit-name-given-names">D.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Waagepetersen</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Normal linear models with genetically structured residual variance heterogeneity: a case study.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">82</span>: <span class="cit-fpage">207</span>–<span class="cit-lpage">222</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1017/S0016672303006426</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DSorensen%26rft.auinit1%253DD.%26rft.volume%253D82%26rft.issue%253D3%26rft.spage%253D207%26rft.epage%253D222%26rft.atitle%253DNormal%2Blinear%2Bmodels%2Bwith%2Bgenetically%2Bstructured%2Bresidual%2Bvariance%2Bheterogeneity%253A%2Ba%2Bcase%2Bstudy.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672303006426%26rft_id%253Dinfo%253Apmid%252F15134199%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672303006426&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15134199&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3333.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000220642000006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3333.51" data-doi="10.3168/jds.2008-1929"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Strandén</span>,  <span class="cit-name-given-names">I.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. J.</span> <span class="cit-name-surname">Garrick</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Technical Note: Derivation of Equivalent Computing Algorithms for Genomic Predictions and Reliabilities of Animal Merit.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">92</span>: <span class="cit-fpage">2971</span>–<span class="cit-lpage">2975</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3168/jds.2008-1929</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DStranden%26rft.auinit1%253DI.%26rft.volume%253D92%26rft.issue%253D6%26rft.spage%253D2971%26rft.epage%253D2975%26rft.atitle%253DTechnical%2Bnote%253A%2BDerivation%2Bof%2Bequivalent%2Bcomputing%2Balgorithms%2Bfor%2Bgenomic%2Bpredictions%2Band%2Breliabilities%2Bof%2Banimal%2Bmerit%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2008-1929%26rft_id%253Dinfo%253Apmid%252F19448030%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2008-1929&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19448030&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3333.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3333.52" data-doi="10.3168/jds.2014-8210"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Su</span>,  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">O. F.</span> <span class="cit-name-surname">Christensen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Janss</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. S.</span> <span class="cit-name-surname">Lund</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Comparison of genomic predictions using genomic relationship matrices built with different weighting factors to account for locus/specific variances.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">97</span>: <span class="cit-fpage">6547</span>–<span class="cit-lpage">6559</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3168/jds.2014-8210</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BDairy%2BSci.%26rft.volume%253D97%26rft.spage%253D6547%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2014-8210%26rft_id%253Dinfo%253Apmid%252F25129495%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2014-8210&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25129495&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3333.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3333.53" data-doi="10.1186/s12711-016-0211-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tusell</span>,  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Gilbert</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Riquet</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. J.</span> <span class="cit-name-surname">Mercat</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Legarra</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Pedigree and genomic evaluation of pigs using a terminal-cross model.</span> <abbr class="cit-jnl-abbrev">Genet Sel Evol: GSE</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">32</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12711-016-0211-3</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet%2BSel%2BEvol%253A%2BGSE%26rft.volume%253D48%26rft.spage%253D32%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12711-016-0211-3%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12711-016-0211-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3333.54" data-doi="10.1017/S0016672309990334"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Usai</span>,  <span class="cit-name-given-names">M. G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. E.</span> <span class="cit-name-surname">Goddard</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. J.</span> <span class="cit-name-surname">Hayes</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">LASSO with cross-validation for genomic selection.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">427</span>–<span class="cit-lpage">436</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1017/S0016672309990334</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DUsai%26rft.auinit1%253DM.%2BG.%26rft.volume%253D91%26rft.issue%253D6%26rft.spage%253D427%26rft.epage%253D436%26rft.atitle%253DLASSO%2Bwith%2Bcross-validation%2Bfor%2Bgenomic%2Bselection.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672309990334%26rft_id%253Dinfo%253Apmid%252F20122298%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672309990334&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20122298&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3333.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3333.55" data-doi="10.3168/jds.2008-1514"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">VanRaden</span>,  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. P.</span> <span class="cit-name-surname">Van Tassell</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G. R.</span> <span class="cit-name-surname">Wiggans</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. S.</span> <span class="cit-name-surname">Sonstegard</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. D.</span> <span class="cit-name-surname">Schnabel</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Invited Review: Reliability of Genomic Predictions for North American Holstein Bulls.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">92</span>: <span class="cit-fpage">16</span>–<span class="cit-lpage">24</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3168/jds.2008-1514</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DVanRaden%26rft.auinit1%253DP.%2BM.%26rft.volume%253D92%26rft.issue%253D1%26rft.spage%253D16%26rft.epage%253D24%26rft.atitle%253DInvited%2BReview%253A%2BReliability%2Bof%2Bgenomic%2Bpredictions%2Bfor%2BNorth%2BAmerican%2BHolstein%2Bbulls%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2008-1514%26rft_id%253Dinfo%253Apmid%252F19109259%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2008-1514&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19109259&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3333.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000261948600002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3333.56" data-doi="10.1186/s12711-015-0133-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wolc</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H. H.</span> <span class="cit-name-surname">Zhao</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Arango</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Settar</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. E.</span> <span class="cit-name-surname">Fulton</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Response and inbreeding from a genomic selection experiment in layer chickens.</span> <abbr class="cit-jnl-abbrev">Genet Sel Evol: GSE</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">59</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12711-015-0133-5</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet%2BSel%2BEvol%253A%2BGSE%26rft.volume%253D47%26rft.spage%253D59%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12711-015-0133-5%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12711-015-0133-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3333.57" data-doi="10.1038/ng.823"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. A.</span> <span class="cit-name-surname">Manolio</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L. R.</span> <span class="cit-name-surname">Pasquale</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Boerwinkle</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Caporaso</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genome Partitioning of Genetic Variation for Complex Traits Using Common SNPs.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">519</span>–<span class="cit-lpage">525</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.823</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DYang%26rft.auinit1%253DJ.%26rft.volume%253D43%26rft.issue%253D6%26rft.spage%253D519%26rft.epage%253D525%26rft.atitle%253DGenome%2Bpartitioning%2Bof%2Bgenetic%2Bvariation%2Bfor%2Bcomplex%2Btraits%2Busing%2Bcommon%2BSNPs.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.823%26rft_id%253Dinfo%253Apmid%252F21552263%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.823&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21552263&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3333.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3333.58" data-doi="10.1534/genetics.111.131540"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>,  <span class="cit-name-given-names">W.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. J.</span> <span class="cit-name-surname">Tempelman</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A Bayesian antedependence model for whole genome prediction.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">190</span>: <span class="cit-fpage">1491</span>–<span class="cit-lpage">1501</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.111.131540</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.131540%26rft_id%253Dinfo%253Apmid%252F22135352%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5MC80LzE0OTEiO3M6NDoiYXRvbSI7czoxOToiL2dnZy85LzEwLzMzMzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3333.59" data-doi="10.1371/journal.pone.0093017"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">U.</span> <span class="cit-name-surname">Ober</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Erbe</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Zhang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Gao</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Improving the accuracy of Whole Genome Prediction for Complex Traits using the results of Genome Wide Association Studies.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>: <span class="cit-elocation-id">e93017</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0093017</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De93017%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0093017%26rft_id%253Dinfo%253Apmid%252F24663104%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0093017&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24663104&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3333.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3333.60"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Erbe</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">He</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">U.</span> <span class="cit-name-surname">Ober</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Gao</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Accuracy of whole-genome prediction using a genetic architecture-enhanced variance-covariance matrix. G3: Genes, Genomes</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">615</span>–<span class="cit-lpage">627</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D5%26rft.spage%253D615%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3333.61" data-doi="10.1371/journal.pone.0012648"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. F.</span> <span class="cit-name-surname">Liu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X. D.</span> <span class="cit-name-surname">Ding</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Bijma</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. J.</span> <span class="cit-name-surname">de Koning</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Best linear unbiased prediction of genomic breeding values using trait-specific marker-derived relationship matrix.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">5</span>: <span class="cit-elocation-id">e12648</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0012648</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D5%26rft.spage%253De12648%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0012648%26rft_id%253Dinfo%253Apmid%252F20844593%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0012648&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20844593&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3333.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/9/10/3333.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/10/3315" title="Disease Resistance Genetics and Genomics in Octoploid Strawberry" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/10/3345" title="Alternative Lengthening of Telomeres in the Budding Yeast Naumovozyma castellii" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/9/10"><span>Volume 9 Issue 10, October 2019</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/9/10.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/9/10.cover-source.jpg?itok=yA8kifEd" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 9 (10)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/10/3333" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-KPyRy1nevB3QXy0jpK4GeusIiQnOKOhjElLPlsOaEbU" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/genomic-prediction" class="" data-icon-position="" data-hide-link-title="0">Genomic Prediction</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/9/10/3333" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/9/10/3333" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/9/10/3333" class="active" data-icon-position="" data-hide-link-title="0">Genomic Prediction Including SNP-Specific Variance Predictors</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/494292" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-BB3YlKcSDRn7aoX3IpFhWOxlBJ9cVXTCx3y0kZcybjc" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/10/3333.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/9/10/3333" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-Ls1dsUYY8DkuoC6NwAvjwzpv2DG0oQyjPruam8XlkaQ" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/9/10/3333" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/10/3333.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494292" id="node494292--2" data-pisa="ggg;9/10/3333" data-pisa-master="ggg;g3.119.400381" data-apath="/ggg/9/10/3333.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genomic Prediction Including SNP-Specific Variance Predictors</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-1335-7610" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Elena Flavia Mouresan</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-2062-3235" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Maria Selle</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-1057-5401" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Lars Rönnegård</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3333-3343;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400381 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494292--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Elena Flavia Mouresan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Animal Breeding and Genetics, Swedish University of Agricultural Sciences, Sweden, 75007,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Elena%2BFlavia%2BMouresan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mouresan%20EF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AElena%2BFlavia%2BMouresan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-1335-7610" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Elena Flavia Mouresan</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:elena.flavia.mouresan@slu.se" class="" data-icon-position="" data-hide-link-title="0">elena.flavia.mouresan@slu.se</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Maria Selle </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Mathematical Sciences, Norwegian University of Science and Technology, Norway, 7491, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Maria%2BSelle%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Selle%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMaria%2BSelle%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-2062-3235" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Maria Selle</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Lars Rönnegård </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Animal Breeding and Genetics, Swedish University of Agricultural Sciences, Sweden, 75007,</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>School of Technology and Business Studies, Dalarna University, Sweden, 79188</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lars%2BR%C3%B6nneg%C3%A5rd%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=R%C3%B6nneg%C3%A5rd%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALars%2BR%25C3%25B6nneg%25C3%25A5rd%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-1057-5401" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Lars Rönnegård</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/9/10/3333" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/9/10/3333&amp;title=Genomic%20Prediction%20Including%20SNP-Specific%20Variance%20Predictors" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/9/10/3333&amp;title=Genomic%20Prediction%20Including%20SNP-Specific%20Variance%20Predictors" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/9/10/3333&amp;title=Genomic%20Prediction%20Including%20SNP-Specific%20Variance%20Predictors" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/9/10/3333&amp;text=Genomic%20Prediction%20Including%20SNP-Specific%20Variance%20Predictors" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/9/10/3333&amp;title=Genomic%20Prediction%20Including%20SNP-Specific%20Variance%20Predictors" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/9/10/3333&amp;t=Genomic%20Prediction%20Including%20SNP-Specific%20Variance%20Predictors" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/9/10/3333&amp;title=Genomic%20Prediction%20Including%20SNP-Specific%20Variance%20Predictors" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/9/10/3333&amp;title=Genomic%20Prediction%20Including%20SNP-Specific%20Variance%20Predictors" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/494292/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494292" id="node494292--3" data-pisa="ggg;9/10/3333" data-pisa-master="ggg;g3.119.400381" data-apath="/ggg/9/10/3333.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genomic Prediction Including SNP-Specific Variance Predictors</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-1335-7610" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Elena Flavia Mouresan</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-2062-3235" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Maria Selle</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-1057-5401" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Lars Rönnegård</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3333-3343;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400381 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494292--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Elena Flavia Mouresan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Animal Breeding and Genetics, Swedish University of Agricultural Sciences, Sweden, 75007,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Elena%2BFlavia%2BMouresan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mouresan%20EF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AElena%2BFlavia%2BMouresan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-1335-7610" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Elena Flavia Mouresan</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:elena.flavia.mouresan@slu.se" class="" data-icon-position="" data-hide-link-title="0">elena.flavia.mouresan@slu.se</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Maria Selle </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Mathematical Sciences, Norwegian University of Science and Technology, Norway, 7491, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Maria%2BSelle%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Selle%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMaria%2BSelle%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-2062-3235" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Maria Selle</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Lars Rönnegård </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Animal Breeding and Genetics, Swedish University of Agricultural Sciences, Sweden, 75007,</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>School of Technology and Business Studies, Dalarna University, Sweden, 79188</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lars%2BR%C3%B6nneg%C3%A5rd%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=R%C3%B6nneg%C3%A5rd%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALars%2BR%25C3%25B6nneg%25C3%25A5rd%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-1057-5401" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Lars Rönnegård</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=3333&amp;rft.epage=3343&amp;rft.atitle=Genomic%20Prediction%20Including%20SNP-Specific%20Variance%20Predictors&amp;rft.volume=9&amp;rft.issue=10&amp;rft.date=2019-10-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Mouresan%2C+Elena+Flavia&amp;rft.au=Selle%2C+Maria&amp;rft.au=R%C3%B6nneg%C3%A5rd%2C+Lars"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/494292/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/494292/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/494292/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/494292/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/494292/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/494292/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/494292/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/494292/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/494292/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/494292/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/494292/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/494292/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495864" id="node495864" data-pisa="ggg;10/3/1113" data-pisa-master="ggg;g3.119.400968" data-apath="/ggg/10/3/1113.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1113" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Multi-trait Genomic Prediction Model Increased the Predictive Ability for Agronomic and Malting Quality Traits in Barley (<em>Hordeum vulgare</em> L.)</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495543" id="node495543" data-pisa="ggg;10/2/769" data-pisa-master="ggg;g3.119.400759" data-apath="/ggg/10/2/769.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/2/769" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Novel Bayesian Networks for Genomic Prediction of Developmental Traits in Biomass Sorghum</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495557" id="node495557" data-pisa="ggg;10/2/539" data-pisa-master="ggg;g3.119.400866" data-apath="/ggg/10/2/539.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/2/539" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Optimized Genetic Testing for Polledness in Multiple Breeds of Cattle</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Genomic%20Prediction" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Genomic Prediction</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/9/10/3333" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-9" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-15" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/9/10/3333.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/9/10/3333.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/9/10/3333.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
