<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/9/10/3139" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/9/10/3139.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/9/10/3139.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/9/10/3139.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/9/10/3139/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/9/10/3139.atom" />
<meta name="HW.pisa" content="ggg;9/10/3139" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Genome-Wide Association and Gene Co-expression Network Analyses Reveal Complex Genetics of Resistance to Goss’s Wilt of Maize" />
<meta name="DC.Identifier" content="10.1534/g3.119.400347" />
<meta name="DC.Date" content="2019-10-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2019 Singh et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Goss’s bacterial wilt and leaf blight is a disease of maize caused by the gram positive bacterium Clavibacter michiganensis subsp. nebraskensis ( Cmn ). First discovered in Nebraska, Goss’s wilt has now spread to major maize growing states in the United States and three provinces in Canada. Previous studies conducted using elite maize inbred lines and their hybrids have shown that resistance to Goss’s wilt is a quantitative trait. The objective of this study was to further our understanding of the genetic basis of resistance to Goss’s wilt by using a combined approach of genome-wide association mapping and gene co-expression network analysis. Genome-wide association analysis was accomplished using a diversity panel consisting of 555 maize inbred lines and a set of 450 recombinant inbred lines (RILs) from three bi-parental mapping populations, providing the most comprehensive screening of Goss’s wilt resistance to date. Three SNPs in the diversity panel and 10 SNPs in the combined dataset, including the diversity panel and RILs, were found to be significantly associated with Goss’s wilt resistance. Each significant SNP explained 1–5% of the phenotypic variation for Goss’s wilt (total of 8–11%). To augment the results of genome-wide association mapping and help identify candidate genes, a time course RNA sequencing experiment was conducted using resistant (N551) and susceptible (B14A) maize inbred lines. Gene co-expression network analysis of this time course experiment identified one module of 141 correlated genes that showed differential regulation in response to Cmn inoculations in both resistant and susceptible lines. SNPs inside and flanking these genes explained 13.3% of the phenotypic variation. Among 1,000 random samples of genes, only 8% of samples explained more phenotypic variance for Goss’s wilt resistance than those implicated by the co-expression network analysis. While a statistically significant enrichment was not observed ( P &lt; 0.05), these results suggest a possible role for these genes in quantitative resistance at the field level and warrant more research on combining gene co-expression network analysis with quantitative genetic analyses to dissect complex disease resistance traits. The results of the GWAS and co-expression analysis both support the complex nature of resistance to this important disease of maize." />
<meta name="DC.Contributor" content="Amritpal Singh" />
<meta name="DC.Contributor" content="Guangyong Li" />
<meta name="DC.Contributor" content="Alex B. Brohammer" />
<meta name="DC.Contributor" content="Diego Jarquin" />
<meta name="DC.Contributor" content="Candice N. Hirsch" />
<meta name="DC.Contributor" content="James R. Alfano" />
<meta name="DC.Contributor" content="Aaron J. Lorenz" />
<meta name="article:published_time" content="2019-10-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Genome-Wide Association and Gene Co-expression Network Analyses Reveal Complex Genetics of Resistance to Goss’s Wilt of Maize" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Goss’s bacterial wilt and leaf blight is a disease of maize caused by the gram positive bacterium &lt;i&gt;Clavibacter michiganensis&lt;/i&gt; subsp. &lt;i&gt;nebraskensis&lt;/i&gt; (&lt;i&gt;Cmn&lt;/i&gt;). First discovered in Nebraska, Goss’s wilt has now spread to major maize growing states in the United States and three provinces in Canada. Previous studies conducted using elite maize inbred lines and their hybrids have shown that resistance to Goss’s wilt is a quantitative trait. The objective of this study was to further our understanding of the genetic basis of resistance to Goss’s wilt by using a combined approach of genome-wide association mapping and gene co-expression network analysis. Genome-wide association analysis was accomplished using a diversity panel consisting of 555 maize inbred lines and a set of 450 recombinant inbred lines (RILs) from three bi-parental mapping populations, providing the most comprehensive screening of Goss’s wilt resistance to date. Three SNPs in the diversity panel and 10 SNPs in the combined dataset, including the diversity panel and RILs, were found to be significantly associated with Goss’s wilt resistance. Each significant SNP explained 1–5% of the phenotypic variation for Goss’s wilt (total of 8–11%). To augment the results of genome-wide association mapping and help identify candidate genes, a time course RNA sequencing experiment was conducted using resistant (N551) and susceptible (B14A) maize inbred lines. Gene co-expression network analysis of this time course experiment identified one module of 141 correlated genes that showed differential regulation in response to &lt;i&gt;Cmn&lt;/i&gt; inoculations in both resistant and susceptible lines. SNPs inside and flanking these genes explained 13.3% of the phenotypic variation. Among 1,000 random samples of genes, only 8% of samples explained more phenotypic variance for Goss’s wilt resistance than those implicated by the co-expression network analysis. While a statistically significant enrichment was not observed (&lt;i&gt;P&lt;/i&gt; &amp;lt; 0.05), these results suggest a possible role for these genes in quantitative resistance at the field level and warrant more research on combining gene co-expression network analysis with quantitative genetic analyses to dissect complex disease resistance traits. The results of the GWAS and co-expression analysis both support the complex nature of resistance to this important disease of maize.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2019/10/01" />
<meta name="citation_mjid" content="ggg;9/10/3139" />
<meta name="citation_id" content="9/10/3139" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/9/10/3139" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/9/10/3139.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/9/10/3139.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/9/10/3139.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.119.400347" />
<meta name="citation_pmid" content="31362973" />
<meta name="citation_volume" content="9" />
<meta name="citation_issue" content="10" />
<meta name="citation_num_pages" content="14" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="3139" />
<meta name="citation_lastpage" content="3152" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Amritpal Singh" />
<meta name="citation_author_institution" content="Department of Agronomy and Plant Genetics, University of Minnesota, St. Paul, MN 55108" />
<meta name="citation_author" content="Guangyong Li" />
<meta name="citation_author_institution" content="Department of Plant Pathology, University of Nebraska, Lincoln, NE 68503-0722" />
<meta name="citation_author_institution" content="Center for Plant Science Innovation, University of Nebraska, Lincoln, NE 68588-0660, and" />
<meta name="citation_author" content="Alex B. Brohammer" />
<meta name="citation_author_institution" content="Department of Agronomy and Plant Genetics, University of Minnesota, St. Paul, MN 55108" />
<meta name="citation_author" content="Diego Jarquin" />
<meta name="citation_author_institution" content="Department of Agronomy and Horticulture, University of Nebraska, Lincoln, NE 68503-0915" />
<meta name="citation_author" content="Candice N. Hirsch" />
<meta name="citation_author_institution" content="Department of Agronomy and Plant Genetics, University of Minnesota, St. Paul, MN 55108" />
<meta name="citation_author" content="James R. Alfano" />
<meta name="citation_author_institution" content="Department of Plant Pathology, University of Nebraska, Lincoln, NE 68503-0722" />
<meta name="citation_author_institution" content="Center for Plant Science Innovation, University of Nebraska, Lincoln, NE 68588-0660, and" />
<meta name="citation_author" content="Aaron J. Lorenz" />
<meta name="citation_author_institution" content="Department of Agronomy and Plant Genetics, University of Minnesota, St. Paul, MN 55108" />
<meta name="citation_author_email" content="lore0149@umn.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-4361-1683" />
<meta name="citation_reference" content="citation_year=2011" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=D. H. Alexander;citation_author=J. Novembre;citation_author=K. Lange;citation_title=Fast model-based estimation of ancestry in unrelated individuals.;citation_volume=19;citation_year=2009;citation_pmid=19648217;citation_doi=10.1101/gr.094052.109" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=S. Anders;citation_author=P. T. Pyl;citation_author=W. Huber;citation_title=HTSeq—a Python framework to work with high-throughput sequencing data.;citation_volume=31;citation_year=2015;citation_pmid=25260700;citation_doi=10.1093/bioinformatics/btu638" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=T. Asai;citation_author=G. Tena;citation_author=J. Plotnikova;citation_author=M. R. Willmann;citation_author=W.-L. Chiu;citation_title=MAP kinase signalling cascade in Arabidopsis innate immunity.;citation_pages=977-983;citation_volume=415;citation_year=2002;citation_issue=6875;citation_pmid=11875555;citation_doi=10.1038/415977a" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=P. J. Balint-Kurti;citation_author=J. Yang;citation_author=G. V. Esbroeck;citation_author=J. Jung;citation_author=M. E. Smith;citation_title=Use of a Maize Advanced Intercross Line for Mapping of QTL for Northern Leaf Blight Resistance and Multiple Disease Resistance;citation_pages=458-466;citation_volume=50;citation_year=2010;citation_issue=2;citation_doi=10.2135/cropsci2009.02.0066" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=J. C. Barrett;citation_author=B. Fry;citation_author=J. Maller;citation_author=M. J. Daly;citation_title=Haploview: analysis and visualization of LD and haplotype maps.;citation_volume=21;citation_year=2005;citation_pmid=15297300;citation_doi=10.1093/bioinformatics/bth457" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=J. M. Benson;citation_author=J. A. Poland;citation_author=B. M. Benson;citation_author=E. L. Stromberg;citation_author=R. J. Nelson;citation_title=Resistance to gray leaf spot of maize: genetic architecture and mechanisms elucidated through nested association mapping and near-isogenic line analysis.;citation_pages=e1005045;citation_volume=11;citation_year=2015;citation_pmid=25764179;citation_doi=10.1371/journal.pgen.1005045" />
<meta name="citation_reference" content="citation_journal_abbrev=Front Plant Sci;citation_author=V. Bonardi;citation_author=J. L. Dangl;citation_title=How complex are intracellular immune receptor signaling complexes?;citation_pages=237-237;citation_volume=3;citation_year=2012;citation_pmid=23109935;citation_doi=10.3389/fpls.2012.00237" />
<meta name="citation_reference" content="citation_year=2009" />
<meta name="citation_reference" content="citation_journal_title=Plant Dis. Rep.;citation_author=A. Calub;citation_author=W. A. Compton;citation_author=C. O. Gardner;citation_author=M. L. Schuster;citation_title=Reaction of 113 corn inbreds to leaf freckles and wilt of corn.;citation_pages=956-960;citation_volume=58;citation_year=1974" />
<meta name="citation_reference" content="citation_journal_title=Phytopathology;citation_author=M. L. Carson;citation_author=Z. W. Wicks;citation_title=Relationship between Leaf Freckles and Wilt Severity and Yield Losses in Closely Related Maize Hybrids.;citation_pages=95-98;citation_volume=81;citation_year=1991;citation_doi=10.1094/Phyto-81-95" />
<meta name="citation_reference" content="citation_journal_title=Plant J.;citation_author=N. Christie;citation_author=A. A. Myburg;citation_author=F. Joubert;citation_author=S. L. Murray;citation_author=M. Carstens;citation_title=Systems genetics reveals a transcriptional network associated with susceptibility in the maize–grey leaf spot pathosystem.;citation_pages=746-763;citation_volume=89;citation_year=2017;citation_doi=10.1111/tpj.13419" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=J. S. Cooper;citation_author=P. J. Balint-Kurti;citation_author=T. M. Jamann;citation_title=Identification of Quantitative Trait Loci for Goss’s Wilt of Maize.;citation_pages=1192-1200;citation_volume=58;citation_year=2018;citation_doi=10.2135/cropsci2017.10.0618" />
<meta name="citation_reference" content="citation_journal_title=The Plant journal :  for cell and molecular biology;citation_journal_abbrev=Plant J;citation_author=G. Doehlemann;citation_author=R. Wahl;citation_author=R. J. Horst;citation_author=L. M. Voll;citation_author=B. Usadel;citation_title=Reprogramming a maize plant: transcriptional and metabolic changes induced by the fungal biotroph Ustilago maydis.;citation_pages=181-195;citation_volume=56;citation_year=2008;citation_issue=2;citation_pmid=18564380;citation_doi=10.1111/j.1365-313X.2008.03590.x" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=Z. Du;citation_author=X. Zhou;citation_author=Y. Ling;citation_author=Z. Zhang;citation_author=Z. Su;citation_title=agriGO: a GO analysis toolkit for the agricultural community.;citation_volume=38;citation_year=2010;citation_pmid=20435677;citation_doi=10.1093/nar/gkq310" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=R. J. Elshire;citation_author=J. C. Glaubitz;citation_author=Q. Sun;citation_author=J. A. Poland;citation_author=K. Kawamoto;citation_title=A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species.;citation_pages=e19379-e19379;citation_volume=6;citation_year=2011;citation_issue=5;citation_pmid=21573248;citation_doi=10.1371/journal.pone.0019379" />
<meta name="citation_reference" content="citation_title=PQR database;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=F1000Prime Rep.;citation_author=B. G. Forde;citation_author=M. R. Roberts;citation_title=Glutamate receptor-like channels in plants: a role as amino acid sensors in plant defence?;citation_pages=37;citation_volume=6;citation_year=2014;citation_doi=10.12703/P6-37" />
<meta name="citation_reference" content="citation_journal_title=Plant Dis.;citation_author=A. Friskop;citation_author=K. Kinzer;citation_author=M. McConnell;citation_author=Z. Liu;citation_author=K. Korus;citation_title=First Report of Goss’s Bacterial Leaf Blight and Wilt of Corn Caused by Clavibacter michiganensis subsp. nebraskensis in North Dakota.;citation_pages=1739;citation_volume=98;citation_year=2014;citation_doi=10.1094/PDIS-04-14-0397-PDN" />
<meta name="citation_reference" content="citation_journal_title=Maize Genet. Coop. Newsl.;citation_author=C. O. Gardner;citation_author=M. L. Schuster;citation_title=Genetic studies of susceptibility to bacterial leaf freckles and wilt, Corynebacterium nebraskense.;citation_pages=155-157;citation_volume=47;citation_year=1974" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=J. C. Glaubitz;citation_author=T. M. Casstevens;citation_author=F. Lu;citation_author=J. Harriman;citation_author=R. J. Elshire;citation_title=TASSEL-GBS: a high capacity genotyping by sequencing analysis pipeline.;citation_pages=e90346;citation_volume=9;citation_year=2014;citation_pmid=24587335;citation_doi=10.1371/journal.pone.0090346" />
<meta name="citation_reference" content="citation_journal_title=Am. J. Hum. Genet.;citation_author=A. Gusev;citation_author=S. H. Lee;citation_author=G. Trynka;citation_author=H. Finucane;citation_author=B. J. Vilhjálmsson;citation_title=Partitioning Heritability of Regulatory and Cell-Type-Specific Variants across 11 Common Diseases.;citation_pages=535-552;citation_volume=95;citation_year=2014;citation_pmid=25439723;citation_doi=10.1016/j.ajhg.2014.10.004" />
<meta name="citation_reference" content="citation_journal_title=Plant Dis.;citation_author=P. K. Hosack;citation_author=L. E. Sweets;citation_author=G. L. Miller;citation_author=M. J. Calcutt;citation_author=B. E. Arenz;citation_title=First Report of Goss’s Bacterial Wilt and Leaf Blight on Corn Caused by Clavibacter michiganensis subsp. nebraskensis in Missouri.;citation_pages=2158;citation_volume=100;citation_year=2016;citation_doi=10.1094/PDIS-02-16-0227-PDN" />
<meta name="citation_reference" content="citation_journal_title=Plant Dis.;citation_author=R. J. Howard;citation_author=M. W. Harding;citation_author=J. Lynn;citation_author=L. M. Kawchuk;citation_author=N. M. Rasmussen;citation_title=First Report of Goss’s Bacterial Wilt and Leaf Blight on Corn Caused by Clavibacter michiganensis subsp. nebraskensis in Alberta, Canada.;citation_pages=1034;citation_volume=99;citation_year=2015;citation_doi=10.1094/PDIS-11-14-1117-PDN" />
<meta name="citation_reference" content="citation_year=2007" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=M. Jakobsson;citation_author=N. A. Rosenberg;citation_title=CLUMPP: a cluster matching and permutation program for dealing with label switching and multimodality in analysis of population structure.;citation_volume=23;citation_year=2007;citation_pmid=17485429;citation_doi=10.1093/bioinformatics/btm233" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=J. D. G. Jones;citation_author=J. L. Dangl;citation_title=The plant immune system.;citation_pages=323-329;citation_volume=444;citation_year=2006;citation_issue=7117;citation_pmid=17108957;citation_doi=10.1038/nature05286" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=D. Kim;citation_author=G. Pertea;citation_author=C. Trapnell;citation_author=H. Pimentel;citation_author=R. Kelley;citation_title=TopHat2: accurate alignment of transcriptomes in the presence of insertions, deletions and gene fusions.;citation_pages=R36;citation_volume=14;citation_year=2013;citation_pmid=23618408;citation_doi=10.1186/gb-2013-14-4-r36" />
<meta name="citation_reference" content="citation_journal_title=Plant Dis.;citation_author=K. A. Korus;citation_author=A. D. Timmerman;citation_author=R. D. French-Monar;citation_author=T. A. Jackson;citation_title=First Report of Goss’s Bacterial Wilt and Leaf Blight (Clavibacter michiganensis subsp nebraskensis) of Corn in Texas.;citation_pages=73;citation_volume=95;citation_year=2011;citation_doi=10.1094/PDIS-07-10-0541" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=K. L. Kump;citation_author=P. J. Bradbury;citation_author=R. J. Wisser;citation_author=E. S. Buckler;citation_author=A. R. Belcher;citation_title=Genome-wide association study of quantitative resistance to southern leaf blight in the maize nested association mapping population.;citation_pages=163-168;citation_volume=43;citation_year=2011;citation_issue=2;citation_pmid=21217757;citation_doi=10.1038/ng.747" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=P. Langfelder;citation_author=S. Horvath;citation_title=WGCNA: an R package for weighted correlation network analysis.;citation_pages=559-559;citation_volume=9;citation_year=2008;citation_pmid=19114008;citation_doi=10.1186/1471-2105-9-559" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=A. Lanubile;citation_author=A. Ferrarini;citation_author=V. Maschietto;citation_author=M. Delledonne;citation_author=A. Marocco;citation_title=Functional genomic analysis of constitutive and inducible defense responses to Fusarium verticillioides infection in maize genotypes with contrasting ear rot resistance.;citation_pages=710;citation_volume=15;citation_year=2014;citation_pmid=WOS:00034104;citation_doi=10.1186/1471-2164-15-710" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=C. Lippert;citation_author=J. Listgarten;citation_author=Y. Liu;citation_author=C. M. Kadie;citation_author=R. I. Davidson;citation_title=FaST linear mixed models for genome-wide association studies.;citation_pages=833-835;citation_volume=8;citation_year=2011;citation_issue=10;citation_pmid=21892150;citation_doi=10.1038/nmeth.1681" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=M. Liu;citation_author=J. Gao;citation_author=F. Yin;citation_author=G. Gong;citation_author=C. Qin;citation_title=Transcriptome Analysis of Maize Leaf Systemic Symptom Infected by Bipolaris zeicola.;citation_pages=e0119858;citation_volume=10;citation_year=2015;citation_doi=10.1371/journal.pone.0119858" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=Y. Liu;citation_author=Y. Guo;citation_author=C. Ma;citation_author=D. Zhang;citation_author=C. Wang;citation_title=Transcriptome analysis of maize resistance to Fusarium graminearum.;citation_pages=477;citation_volume=17;citation_year=2016;citation_doi=10.1186/s12864-016-2780-5" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=M. I. Love;citation_author=W. Huber;citation_author=S. Anders;citation_title=Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2.;citation_pages=550;citation_volume=15;citation_year=2014;citation_pmid=25516281;citation_doi=10.1186/s13059-014-0550-8" />
<meta name="citation_reference" content="citation_journal_title=Plant Dis.;citation_author=D. Malvick;citation_author=R. Syverson;citation_author=D. Mollov;citation_author=C. A. Ishimaru;citation_title=Goss’s Bacterial Blight and Wilt of Corn Caused by Clavibacter michiganensis subsp. nebraskensis Occurs in Minnesota.;citation_pages=1064;citation_volume=94;citation_year=2010;citation_doi=10.1094/PDIS-94-8-1064A" />
<meta name="citation_reference" content="citation_journal_title=CUTADAPT REMOVES ADAPTER SEQUENCES FROM HIGHTHROUGHPUT SEQUENCING READS;citation_author=M. Martin;citation_pages=10;citation_volume=17;citation_year=2011;citation_pmid=25452271;citation_doi=10.14806/ej.17.1.200" />
<meta name="citation_reference" content="citation_journal_title=Maize Genet. Coop. Newsl.;citation_author=P. R. Martin;citation_author=C. O. Gardner;citation_author=A. G. Calub;citation_author=M. L. Schuster;citation_title=Inheritance ofsusceptibility and tolerance to leaf freckles and wilt (Corynebacterium nebraskense) of corn.;citation_pages=137-138;citation_volume=49;citation_year=1975" />
<meta name="citation_reference" content="citation_journal_title=BMC Plant Biol.;citation_author=J. Meyer;citation_author=D. K. Berger;citation_author=S. A. Christensen;citation_author=S. L. Murray;citation_title=RNA-Seq analysis of resistant and susceptible sub-tropical maize lines reveals a role for kauralexins in resistance to grey leaf spot disease, caused by Cercospora zeina.;citation_pages=197;citation_volume=17;citation_year=2017;citation_doi=10.1186/s12870-017-1137-9" />
<meta name="citation_reference" content="citation_journal_title=Sci. Rep.;citation_author=V. J. Miranda;citation_author=W. F. Porto;citation_author=G. D. R. Fernandes;citation_author=R. Pogue;citation_author=D. O. Nolasco;citation_title=Comparative transcriptomic analysis indicates genes associated with local and systemic resistance to Colletotrichum graminicola in maize.;citation_pages=2483;citation_volume=7;citation_year=2017;citation_doi=10.1038/s41598-017-02298-8" />
<meta name="citation_reference" content="citation_year=2012" />
<meta name="citation_reference" content="citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Plant Health Prog.;citation_author=D. Mueller;citation_author=K. Wise;citation_author=A. Sisson;citation_author=T. Allen;citation_author=G. Bergstrom;citation_title=CORN YIELD LOSS ESTIMATES DUE TO DISEASES IN THE UNITED STATES AND ONTARIO, CANADA FROM 2012 TO 2015.;citation_pages=211-222;citation_volume=17;citation_year=2016;citation_doi=10.1094/PHP-RS-16-0030" />
<meta name="citation_reference" content="citation_journal_title=Phytopathology;citation_author=E. N. Ngong-Nassah;citation_author=M. L. Carson;citation_author=Z. W. Wicks;citation_title=Inheritance of Resistance to Leaf Freckles and Wilt Caused by Clavibacter-Michiganense Subsp Nebraskense in Early Maturing Maize Inbred Lines.;citation_pages=142-146;citation_volume=82;citation_year=1992;citation_doi=10.1094/Phyto-82-142" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_journal_abbrev=Bioinformatics;citation_author=E. Paradis;citation_author=J. Claude;citation_author=K. Strimmer;citation_title=APE: Analyses of Phylogenetics and Evolution in R language;citation_pages=289-290;citation_volume=20;citation_year=2004;citation_issue=2;citation_pmid=14734327;citation_doi=10.1093/bioinformatics/btg412" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=J. A. Poland;citation_author=P. J. Bradbury;citation_author=E. S. Buckler;citation_author=R. J. Nelson;citation_title=Genome-wide nested association mapping of quantitative resistance to northern leaf blight in maize.;citation_volume=108;citation_year=2011;citation_pmid=21482771;citation_doi=10.1073/pnas.1010894108" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=S. Purcell;citation_author=B. Neale;citation_author=K. Todd-Brown;citation_author=L. Thomas;citation_author=M. A. R. Ferreira;citation_title=PLINK: a tool set for whole-genome association and population-based linkage analyses.;citation_pages=559-575;citation_volume=81;citation_year=2007;citation_issue=3;citation_pmid=17701901;citation_doi=10.1086/519795" />
<meta name="citation_reference" content="citation_year=2012" />
<meta name="citation_reference" content="citation_journal_title=Maize Genet. Coop. Newsl.;citation_author=T. R. Rocheford;citation_author=C. A. Gardner;citation_author=A. K. Vidaver;citation_title=Chromosomal location of additional genes for resistance to Corynebacterium (Clavibacter) michiganense ssp. nebraskense.;citation_pages=73-74;citation_volume=60;citation_year=1985" />
<meta name="citation_reference" content="citation_journal_title=Journal of Heredity;citation_journal_abbrev=J Hered;citation_author=T. R. Rocheford;citation_author=C. O. Gardner;citation_author=A. K. Vidaver;citation_title=Genetic Studies of Resistance in Maize (Zea mays L.) to Goss's Bacterial Wilt and Blight (Clavibacter michiganense ssp. nebraskense);citation_pages=351-356;citation_volume=80;citation_year=1989;citation_issue=5;citation_doi=10.1093/oxfordjournals.jhered.a110873" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=M. C. Romay;citation_author=M. J. Millard;citation_author=J. C. Glaubitz;citation_author=J. A. Peiffer;citation_author=K. L. Swarts;citation_title=Comprehensive genotyping of the USA national maize inbred seed bank.;citation_pages=R55;citation_volume=14;citation_year=2013;citation_pmid=23759205;citation_doi=10.1186/gb-2013-14-6-r55" />
<meta name="citation_reference" content="citation_journal_title=Plant Dis.;citation_author=G. Ruhl;citation_author=K. Wise;citation_author=T. Creswell;citation_author=A. Leonberger;citation_author=C. Speers;citation_title=First Report of Goss’s Bacterial Wilt and Leaf Blight on Corn Caused by Clavibacter michiganensis subsp. nebraskensis in Indiana.;citation_pages=841;citation_volume=93;citation_year=2009;citation_doi=10.1094/PDIS-93-8-0841B" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=W. K. Russell;citation_title=Registration of N551 and N552 Parental Inbred Lines of Maize;citation_pages=1014-1015;citation_volume=46;citation_year=2006;citation_issue=2;citation_doi=10.2135/cropsci2005.08-0254" />
<meta name="citation_reference" content="citation_journal_title=Molecular Biology and Evolution;citation_journal_abbrev=Mol Biol Evol;citation_author=N. Saitou;citation_author=M. Nei;citation_title=The neighbor-joining method: a new method for reconstructing phylogenetic trees;citation_pages=406-425;citation_volume=4;citation_year=1987;citation_issue=4;citation_pmid=3447015;citation_doi=10.1093/oxfordjournals.molbev.a040454" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Pathog;citation_author=M. Sato;citation_author=K. Tsuda;citation_author=L. Wang;citation_author=J. Coller;citation_author=Y. Watanabe;citation_title=Network modeling reveals prevalent negative regulatory relationships between signaling sectors in Arabidopsis immune signaling.;citation_pages=e1001011-e1001011;citation_volume=6;citation_year=2010;citation_issue=7;citation_pmid=20661428;citation_doi=10.1371/journal.ppat.1001011" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=C. M. Schaefer;citation_author=R. Bernardo;citation_title=Genomewide Association Mapping of Flowering Time, Kernel Composition, and Disease Resistance in Historical Minnesota Maize Inbreds.;citation_pages=2518-2529;citation_volume=53;citation_year=2013;citation_doi=10.2135/cropsci2013.02.0121" />
<meta name="citation_reference" content="citation_year=1972" />
<meta name="citation_reference" content="citation_journal_title=EMBO J.;citation_author=C. Segonzac;citation_author=A. P. Macho;citation_author=M. Sanmartín;citation_author=V. Ntoukakis;citation_author=J. J. Sánchez-Serrano;citation_title=Negative control of BAK1 by protein phosphatase 2A during plant innate immunity.;citation_pages=2069-2079;citation_volume=33;citation_year=2014;citation_pmid=25085430;citation_doi=10.15252/embj.201488698" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=A. Singh;citation_author=A. P. Andersen;citation_author=T. A. Jackson-Zeims;citation_author=A. J. Lorenz;citation_title=Mapping quantitative trait loci for resistance to Goss’s bacterial wilt and leaf blight in North American Maize by joint linkage analysis.;citation_pages=2306;citation_volume=56;citation_year=2016;citation_doi=10.2135/cropsci2015.09.0543" />
<meta name="citation_reference" content="citation_journal_title=Plant Dis.;citation_author=R. Singh;citation_author=C. Hollier;citation_author=T. Burks;citation_author=R. Frazier;citation_title=First Report of Goss’s Wilt of Corn Caused by Clavibacter michiganensis subsp. nebraskensis in Louisiana.;citation_pages=1268;citation_volume=99;citation_year=2015;citation_doi=10.1094/PDIS-08-14-0807-PDN" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=J. D. Storey;citation_author=R. Tibshirani;citation_title=Statistical significance for genomewide studies.;citation_volume=100;citation_year=2003;citation_pmid=12883005;citation_doi=10.1073/pnas.1530509100" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=K. Swarts;citation_author=H. Li;citation_author=R. Navarro;citation_author=J. Alberto;citation_author=D. An;citation_title=Novel methods to optimize genotypic imputation for low-coverage, next-generation sequence data in crop plants.;citation_pages=12;citation_volume=7;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=J. Am. Soc. Hortic. Sci.;citation_author=C. L. Treat;citation_author=W. F. Tracy;citation_title=Inheritance of resistance to Goss’s wilt in sweet corn.;citation_pages=672-674;citation_volume=115;citation_year=1990;citation_doi=10.21273/JASHS.115.4.672" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=C. L. Treat;citation_author=W. F. Tracy;citation_author=P. N. Drolsom;citation_author=J. G. Coors;citation_title=Inheritance of Resistance to Goss's Wilt in Maize;citation_pages=893-896;citation_volume=30;citation_year=1990;citation_issue=4;citation_doi=10.2135/cropsci1990.0011183X003000040027x" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=N. Wang;citation_author=J. M. Akey;citation_author=K. Zhang;citation_author=R. Chakraborty;citation_author=L. Jin;citation_title=Distribution of recombination crossovers and the origin of haplotype blocks: the interplay of population history, recombination, and mutation.;citation_pages=1227-1234;citation_volume=71;citation_year=2002;citation_issue=5;citation_pmid=12384857;citation_doi=10.1086/344398" />
<meta name="citation_reference" content="citation_journal_title=Genome;citation_author=J. Wang;citation_author=Y. Zhang;citation_author=J. Du;citation_author=X. Pan;citation_author=L. Ma;citation_title=Combined analysis of genome-wide expression profiling of maize (Zea mays L.) leaves infected with Ustilago maydis.;citation_pages=505-513;citation_volume=61;citation_year=2018;citation_doi=10.1139/gen-2017-0226" />
<meta name="citation_reference" content="citation_year=1981" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=J. Yang;citation_author=B. Benyamin;citation_author=B. P. McEvoy;citation_author=S. Gordon;citation_author=A. K. Henders;citation_title=Common SNPs explain a large proportion of the heritability for human height.;citation_pages=565-569;citation_volume=42;citation_year=2010;citation_issue=7;citation_pmid=20562875;citation_doi=10.1038/ng.608" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=Y. Yu;citation_author=J. Shi;citation_author=X. Li;citation_author=J. Liu;citation_author=Q. Geng;citation_title=Transcriptome analysis reveals the molecular mechanisms of the defense response to gray leaf spot disease in maize.;citation_pages=742;citation_volume=19;citation_year=2018;citation_doi=10.1186/s12864-018-5072-4" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=Z.-L. Zheng;citation_author=Y. Zhao;citation_title=Transcriptome comparison and gene coexpression network analysis provide a systems view of citrus response to ‘CandidatusLiberibacter asiaticus’ infection.;citation_pages=27;citation_volume=14;citation_year=2013;citation_pmid=23324561;citation_doi=10.1186/1471-2164-14-27" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=C. T. Zila;citation_author=L. F. Samayoa;citation_author=R. Santiago;citation_author=A. Butrón;citation_author=J. B. Holland;citation_title=A genome-wide association study reveals genes associated with fusarium ear rot resistance in a maize core diversity panel.;citation_volume=3;citation_year=2013;citation_pmid=24048647;citation_doi=10.1534/g3.113.007328" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Genome-Wide Association and Gene Co-expression Network Analyses Reveal Complex Genetics of Resistance to Goss’s Wilt of Maize" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10.cover-source.jpg" />
<meta name="twitter:description" content="Goss’s bacterial wilt and leaf blight is a disease of maize caused by the gram positive bacterium Clavibacter michiganensis subsp. nebraskensis ( Cmn ). First discovered in Nebraska, Goss’s wilt has now spread to major maize growing states in the United States and three provinces in Canada. Previous studies conducted using elite maize inbred lines and their hybrids have shown that resistance to Goss’s wilt is a quantitative trait. The objective of this study was to further our understanding of the genetic basis of resistance to Goss’s wilt by using a combined approach of genome-wide association mapping and gene co-expression network analysis. Genome-wide association analysis was accomplished using a diversity panel consisting of 555 maize inbred lines and a set of 450 recombinant inbred lines (RILs) from three bi-parental mapping populations, providing the most comprehensive screening of Goss’s wilt resistance to date. Three SNPs in the diversity panel and 10 SNPs in the combined dataset, including the diversity panel and RILs, were found to be significantly associated with Goss’s wilt resistance. Each significant SNP explained 1–5% of the phenotypic variation for Goss’s wilt (total of 8–11%). To augment the results of genome-wide association mapping and help identify candidate genes, a time course RNA sequencing experiment was conducted using resistant (N551) and susceptible (B14A) maize inbred lines. Gene co-expression network analysis of this time course experiment identified one module of 141 correlated genes that showed differential regulation in response to Cmn inoculations in both resistant and susceptible lines. SNPs inside and flanking these genes explained 13.3% of the phenotypic variation. Among 1,000 random samples of genes, only 8% of samples explained more phenotypic variance for Goss’s wilt resistance than those implicated by the co-expression network analysis. While a statistically significant enrichment was not observed ( P &lt; 0.05), these results suggest a possible role for these genes in quantitative resistance at the field level and warrant more research on combining gene co-expression network analysis with quantitative genetic analyses to dissect complex disease resistance traits. The results of the GWAS and co-expression analysis both support the complex nature of resistance to this important disease of maize." />
<meta name="og-title" property="og:title" content="Genome-Wide Association and Gene Co-expression Network Analyses Reveal Complex Genetics of Resistance to Goss’s Wilt of Maize" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/9/10/3139" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Goss’s bacterial wilt and leaf blight is a disease of maize caused by the gram positive bacterium Clavibacter michiganensis subsp. nebraskensis ( Cmn ). First discovered in Nebraska, Goss’s wilt has now spread to major maize growing states in the United States and three provinces in Canada. Previous studies conducted using elite maize inbred lines and their hybrids have shown that resistance to Goss’s wilt is a quantitative trait. The objective of this study was to further our understanding of the genetic basis of resistance to Goss’s wilt by using a combined approach of genome-wide association mapping and gene co-expression network analysis. Genome-wide association analysis was accomplished using a diversity panel consisting of 555 maize inbred lines and a set of 450 recombinant inbred lines (RILs) from three bi-parental mapping populations, providing the most comprehensive screening of Goss’s wilt resistance to date. Three SNPs in the diversity panel and 10 SNPs in the combined dataset, including the diversity panel and RILs, were found to be significantly associated with Goss’s wilt resistance. Each significant SNP explained 1–5% of the phenotypic variation for Goss’s wilt (total of 8–11%). To augment the results of genome-wide association mapping and help identify candidate genes, a time course RNA sequencing experiment was conducted using resistant (N551) and susceptible (B14A) maize inbred lines. Gene co-expression network analysis of this time course experiment identified one module of 141 correlated genes that showed differential regulation in response to Cmn inoculations in both resistant and susceptible lines. SNPs inside and flanking these genes explained 13.3% of the phenotypic variation. Among 1,000 random samples of genes, only 8% of samples explained more phenotypic variance for Goss’s wilt resistance than those implicated by the co-expression network analysis. While a statistically significant enrichment was not observed ( P &lt; 0.05), these results suggest a possible role for these genes in quantitative resistance at the field level and warrant more research on combining gene co-expression network analysis with quantitative genetic analyses to dissect complex disease resistance traits. The results of the GWAS and co-expression analysis both support the complex nature of resistance to this important disease of maize." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10.cover-source.jpg" />
    <title>Genome-Wide Association and Gene Co-expression Network Analyses Reveal Complex Genetics of Resistance to Goss’s Wilt of Maize | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"r6TOltLBvpiwlwJB6WNbOD_dkgO80HogEk3bRgLshw0","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"494277","apath":"\/ggg\/9\/10\/3139.atom","pisa":"ggg;9\/10\/3139","ac":{"\/ggg\/9\/10\/3139.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/9\/10\/3139.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;9\/10\/3139"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;9\/10\/3139"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/9\/10\/3139":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022bV9rt0NgLt\\\\\\\/pAzb3ET26DgbkMOMO9FhbqkELW8s9a8BjSHIqMl1u\\\\\\\/2EhF7UG8uVp\\\\\\\/9e6wZqifgiP4OE5YqzS4qBwow71Dpw7L8famzrxOD\\\\\\\/cTuJWl24e36D+LsRSficS0isAnBteN8zk3JjYpD6tiQzQ6coh45dAiOaTH0dZ\\\\\\\/LuhW8Ez7rgUFXkxwBNg0s8o0C6upILGjfhNd2LRdNwMKWSxyvvwYaxWxl+8Fz0SPl4E5mfekMUZ90dSpVdzvgrWVSM5Xk9+H5AyjKyI1NcNRZGzzfMgRmWK7vtNwAa\\\\\\\/q\\\\\\\/2rM6ttaetrGaQyAZa6bvbX6vDEARV6wZYniVL4CgX7Y\\\\\\\/WKkyybUnEDfwdImumst5EdZ3YSWiWEcPnAfCMUFP47PopSXwIRuCCxiWCuVhRbb38sPlK1dPMOoQViZC\\\\\\\/JMLLPqK2ey8T0jmmZ1hwIfD6yR2bymUWqQL4rfwoHlqh7bviMC4Ktx0zPvCywtj2LfYu2hTQGi3fwAkEX\\\\\\\/FlUvH3uB9mReMfFff1KFCU53IeJ8wRiihTGUdvauCH08WQcN98XGOlUTFaV2eh0BNBSaDNmoNiqYyICMUXMzn+fTkAbWqx9jMZ2uXDl6PEI\\\\\\\/wReIKmNSPyx4z5yMBAycmfu6j4xXQIXAwwqyVwdIe3U1wE+XZps\\\\\\\/1dJr9FOL7deLRMyM0NBzHyKsKt2gX5ChnjTvWx49Y1OZ+6yi18llcXtlWxa1g1rwfSyzP\\\\\\\/xU4OHcTFLBmu7tbxYjdpO1jV9Pv\\\\\\\/2v\\\\\\\/TeYWJpV2rPYaFcrf94FXLlJz\\\\\\\/cqiAjQ+V5eifXhpFztVaPQym14uljgRe6Nz8LnxH\\\\\\\/JhRu77dfMtfGjRHAuSvKUqAk+iKBGonlZnqvxm7qpAt6zW9Ig6AYYMdE1WfMoRmS5zVYwolEaXgkDNd67nXR7iVxpzCeqqZsJEoVNCg1+WL4YcwMcBxCjxo\\\\\\\/V66n09ue38aF1zPvDk8tr6eHHRFocPL+WdUFJRSb7x9FqzoxA3IVs4jZXzdMg358Qp9sS+RreeNWlIUkfaymvmBvpUhvpqqUc8c4v749gCN2aff8mA+1UhW9sBzpsePDWwrwrDiEI352qo4wEmLBFxBUTLu6X4hi9z9LvJhEfDpo1SfpfOplHWCPjpy+VTiuQhMTbbY5rttdU5NOjAIe+3ZbFQ0w+S6hDEGNXi8vpswyZdOsPMO0hqcw8h3nShCB9JsYV4iOrkTHBw+5V57Sm1YoIbgUpGMoW5z2YLJySHwuu2KNb2n\\\\\\\/S85fPKdL0GQPaXmS5JWb\\\u0022,\\\u0022iv\\\u0022:\\\u0022nrLAOy+RrzbbW0oEaFj3Bg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022257fca68054a5e4258e4bd02351221aea0591abe2bc90f5ac718de5e76c307aa\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022a8rLDO8wL71mJiflsrDigGAWZpl4LnsYW24zjTdAVI74OKf6ISc4s6KBL\\\\\\\/dO\\\\\\\/NyAfJ77dtIEI\\\\\\\/fcn6vAL0L9vHDpNsvJcnumF9SGQHpm+unADinhBKiG1ByZCwTH5pqCtZItcTpWczPPOyrOIN1unZnIZ4xyDQjvStaz3fn2reFkkvsIlhWLNZtPY3v+L0T0lbUwuHCpaiUkQJ8YgZvH6k3b\\\\\\\/QUYM34+aVoFVNQ1DzS7MVtOMW\\\\\\\/GKj9wxw8dvSJBx84c7V22Vuur5oqegFNxIdpJSm+knBB+\\\\\\\/fogZlJ5aC0rBMwiNL8wL3i0x9hSVYUKFaJ+w3HktHfvbouzPYGn7FfiW21CIXV4Wup2xH20ykKL8wYPZD+WOvyauSb+n\\\\\\\/eBaGZts+HqeM8gtFzN8AZ9vPcI\\\\\\\/ioGnQdYNvZU9n+PvwwpQKKDDK+1nLgTwZuAZosQ8URRajgC0mmIYWzIYfpv\\\\\\\/6et1ObT2snpFMRsmShLgSt2kg2urPr0nFmMm8+D6cvbkcqhg1i0L3aPdkGcRz1BBr7WgszhMwRg7ev2KycHvFtUBkao\\\\\\\/2KdATr4wisIE46asvAsaGUHr5SV+FvnEl8mn49pymyoL7qdbkkPGyDu+BrvQprvS59yInshSiI9hiV+K6\\\\\\\/7O84Z5VYX62xt5bPGX\\\\\\\/0zIzpd5f44hZXJU+6V8Q739zdRCyIM2kC1kABKwfU\\\\\\\/hBx48voQx8nWYHXALHfLiZWwrEKiBlS1jyt5DphZOQtT11qgCY5h3k2Rrxx2WluvZFnn7JBF2UVLaAmp5i0XLzmX37+DPgG9U0qg\\\\\\\/Z41yrO69ODtPCVLMSwlHxsnh67PP3CYv1i2oEe7Fr475uD5BejRA896wuZ674luS5jyre7+nCJc7SuauUX1TbciYtZ2FZLo74IODU8stBUMK+o69qhHelNZYHmhxWBQ5kh0Ia07H\\\\\\\/Js7Gczn6BziIcE8VSBP5gJ35vlKyV9vl4ZnqwfgnAH+i+mLXjldAeYsSx2A0N6XAeKFQ9q1GLIGRe8iQMHgTU0mqSYNOtDg35BhEy+38NKXvA20vP9SHIbPjFrCqgVnJDHbAn8t9emkizmtPeGUPSENl82xK2FDRuXfISybJHv8p1qku8lfMy4KViNdyKzKvZI7j80NLSCPJh5zbcCtII5pwBSwaIB14f1WFxyVHz7kFTQ1FhvfLpn\\\\\\\/mF3w4kyTieqc\\\\\\\/lQmCfOGMWaJ+z0BOg2F5YGrLvwP14GLgUOSYNd4EzPjCuk8G\\\\\\\/pKA1m+B\\\\\\\/cj71jnnoYPUsllQwyLwYSnnuGFlhtLj6VJ9hM0BCUbGUJVIzKZ74wg8DUqwr7SDxzIOmZJDxbe9d6rfSM7ZJA\\\\\\\/WuWgcmuhB\\\\\\\/xsslZIhWGEmwir1VmRtpNRnkYC02rjftGEbJPtaTgXYMEatiJV0\\\\\\\/YJda9BXGqMuOZNu2K8ZhVrmqmf++QpZ3i11ZX+1itQhk7EvbPG8MpbrbWZGuJk9TmnXhQ6mcrF0gFX9vxpg7Ihml\\\\\\\/A\\\\\\\/Nw3LDuzdObZOhLmO7ngCPiXj24w5SHDpL8UkF36DQOxF552yfJnx9eyruCcgWSuQkLw3rfPmKaBdIOaW6+ZrsTHT\\\\\\\/4okO5WPSHiku\\\\\\\/NJTbiUsC3+yIQTDvkU5FQp4gjW7N3+3WpKlXmglwOdILF497yn4LsysZstG\\\\\\\/Y2FwBThnVr8ALvSrr4XBo4AQ8pIAwn+vCxLLf6iNcPAPX2BLL953rsn1egAdPLRzimPsBwqIapU7invKSCODK7iSAgcy4YLPeFCRxQkyk5BKFI4xzL8tzuedzQY3eJ64CvFKXpXv813uQNQQPefWOm2Eyo2\\\\\\\/qoNSh112ExkDdXBiYh+9wZM9+oJ4sdyFkijAd6oKRwhJ9R\\\\\\\/bMLkAcmzH4Gj\\\\\\\/tIe13BI37lu1nokzSs7K+9L4KJ8bwNwxvidueEUL3MQiel\\\\\\\/fjiPZleDCnHSyLDeZBVvY9zoZARkkSxIeabvNW5NAuQ5cwFPMxCD0eC99KPWIOuIvMhJZJRB1Ywm9x\\\\\\\/DwNvh4\\\\\\\/sLIImI0mkOcToIFbQGI6iM83l51RV7qJ9+COVhprM4hpwKNi7sn+98s9JZF+cVwoDGtoYUe7hMHKx8RK77ud+0gvLYrzEgXXJ+tTRM6ZdUfyMK4PHA9RYitPeCH\\\\\\\/4Jvqgn2CKb5ZkKkgR5vg43pmW9\\\\\\\/uJC5aWWHx7LZZhyjRFuKMCwxL3MaDWLDsxsWEboIZVJspB7KG4gfM9sBoq2sIS+rLA8Rg2HporXNH+Pu9f\\\\\\\/M8TXuaEDnI8rEf47sl2BO+3lFz6nC91mCXB69zjU7qxLAn00hBNrwoNKrTbV2qwethbNuzJ03OJAonsI1trqp3fOamz3ueorg5rFJnw6ZS1mHo3RS35JBcOhAwVRXi\\\\\\\/t6dH\\\\\\\/3g+szSHIvUJa1GADWYi+Ze+PtiDz8GJ1nkLIBx9ZfI+2n3uQikQfgCcQDeXWRtRwy0v8mTkr9BVdlC6h61tAI7h09Bw2zZNRe65ftj9T72gjdDbhAIsrikibQjJSdmdIWWy7Ktoxoxie1nUhr3lQLjqwATCaWI50dpU3MM9K3Vdmf0OMsRTsKcHhpGyezJKb0NDZ\\\\\\\/7\\\\\\\/5TytffSysl\\\\\\\/OQkAIdohGM7d7Oer+vRg6i+ygeeRogm8DGB3UrFyn9SeROBkAI\\\\\\\/S5xvsyhil9SL0qO1lUokz39tBWht2AJ66oENOOdIgCWNil4pDh7EZ5eA9mVYScsGrtJGB3c4Hpy\\\\\\\/UZoX5VHlXF4X6XJv1G7POsaJRSvqJ2uRSWmC1Hti0HwymijptpGFrUTcipVFcJtxctt9nNeL8kcmoZmIfgVQ4j8Uzh9+yBx0MA+ODf79yOCZtYJD73V7S9vqwU2xU5h8Wjy4eA9nr0fyc4HsrY8cwpZMRkTToMUr1dkzBQYINzvUH1uWYW\\\\\\\/IrJBiE+sohnPDlaTJbl8qz8s+9RzswwGRtonVQtWczyJOFIelIg4RlUmWsWJS5Lpr5y\\\\\\\/rmDNCTs65lUxT2N7LtuWl4cASB2AqrAAcjcbJdBfjhC9cFfzUELz3v+\\\\\\\/StyJVyEkkGQ2464RgcEwtzfxLpjY3EWPxNGg3jRV+1C0aPAVS4FqoLnd8DmS2teVwn80v+acLG8SzWGEp1q2M5nzo1Ahk0atF12\\\\\\\/Pkk3qQDI+QJtOW32Fg3uRQiitTkOGbYr7KiDJom2FNOF6ieFBbAzZLFIkhD3aksdjsb66bLR8s+BiIrw3FrKPDmlAYLIMgdcrh0mS6riCMzacRT65HA2Ij7GCoRIECvvqlN1g406DUDJtPJMeLTMWzcUer8ChyxBlARYFIhbuNk3oJQKhyf7Bblk\\\\\\\/I+w+bsV6+DaPA1\\\\\\\/JDgqVrl1XyuB3euZT69OsG689IVF1Ww63gbXTi6BIeAJ7Q3YMtvkMHi+3JYLu0bUEY+JxrIZNQYP1WPzuM3vP225xv7a1EVgW7sY2WtALR92PcjUb3x7jLoRAoaYJMZ69K0FIyty7Alq4Xf5iyKGu\\\\\\\/BOt7k48BvxPP4C2EnKkdeuoP9zUb3Q9bY6tYsVNc142WvAIPKphDAYYjvRft7AIW+0doeTe+JM2Jh5OW7ib\\\\\\\/aFQTacp+rP40QEpm0xwiCIz+484h5zmx9DL598JkfV4nGOAKi3LDAirBPS\\\\\\\/pwIJFx76834+Tpv2Qh3LpcBXwglHyZI9OxeG0xM69EfcP0XJPM56hWOmbg+rmDrsqLNHofzKehiqfmB\\\\\\\/kgmGwjdw6hHt1UhVb275gXhu+nLcZbYs6h5FrJF+zpOQelRAWPKIDp3ttqLLPu83DyzXdFw0HxFlbdsoyW1SvRCN0wyF5YJIji\\\\\\\/cTp7bD2hzQqiyIxbjO8hmF96YDQyBgLNyPzlkGit0Kb\\\\\\\/p1UKWwK5XqynYKKUiptBdq5tMiXmW1wTt86NywOilUzoXOp0LDYP2hMll0QuQBBT2k6Bol29V+uP8zUOdEdp14UAqYGdIxQE60NONdsHezpShDFD0NRjtxPK\\\\\\\/OsHz6HgxWeHPJJRLzBETouVJ1UvQn5U4I0PR3ex4AZHDlz6ANkULdNrCXWDkQzJV04Hy1re9wjV\\\\\\\/6Imsb7KDO1b3mG0fa6kzSbzfW6JDVVyuhKYKNm26C3+0eul0B\\\\\\\/ZBb12ej05Q4UXEu4kY99B95pjJKO3EUGBcOu0DGwQK98dOHpgTSWmtZUvJhvtilCKYRHSGbZjqEJVXSSzjCaGW58DtFd+8SXasBz37fFFcGZ58FyKLwjEBkJQissX6Pinq\\\\\\\/ALL8SbA5NswqIVdGk3iYSfnOOkOHTgAtlpKqOIcxo4RCxaMSGadU8JjTkNhdamzkhDhreaggciVFNGGlc\\\\\\\/9g7ADNKuscD8Y+QT\\\\\\\/N1jjJBNxjZz35WQB4AE6ZG7myEYrtl1JoyuJDnYR0FNLJwEdFkzjpnpwxHeGATA5X4BqxY3bhZCdDZHMYsKIEyIbF7soqzTuMcThOMH+HHwNDlCJ9Ird2VcPllCHCJx7AcDN38QXhLXQKFtHC444yr0\\\\\\\/jxorJ64XyKCRVvpjcAJZPo7GPdffXsioOEkFE96xAbxt33QQx3jfZf+0lP1S9pYETfjlZeWVAG6SURfNrTNt0jEkcalTQQ+EZGLL7+L3N+WzftCWaa8xNyaLR5x7ne2q+kjBRmPw5VNrpf8X8H10AUN7+pYcT\\\\\\\/O1QMgeYfCkPrN106x4ieYtwfPMGVK\\\\\\\/vgXqIt7oZcCp\\\\\\\/0ITvz\\\\\\\/27nCf3IBrs0hvz\\\\\\\/X+IuAjWECveBPRLy8HVSu2r5ljQyXwFQtbmlsUS4JRApQOi+io+Ww5wMBUtMBr9xxcmd6GsIAoNt8vrEwRvFXpxOI1OvYhwQmwmjx+VVEzeKfZkFla4h2j\\\\\\\/dqrTbiMX6HlJFydtXtyAC43+vM5BOSPKEjwiCGzys9zZhhtyXiqDbLpB\\\\\\\/1936VDStqq+i5EFU7qJ3ItY3IQwg\\\\\\\/YYxbLWLydPdD9jDHmhWXDtK236fJmCJZuWIl4m0UY3nmxhVaCqxgVPJ17MdqaactOsQbBQWhG8KEYYfAYpXDG+2tl\\\\\\\/tZ633iOTtn9hoYBgqgAvdONVDSskrf8UJ+pbFzBF6XX8f0Hne0nWpgJxvl2eIeE4daIXgFNKuHk5K4U9NGzvF09X2zLkOvchNy6FuJCehareJm8fgnyABkxW4gwamvnj0hgOg73Xvvgzb4Hehl2ri+e0SzQfsnXd3RsTaGLb3CAuSDSuCDJHLxBTEVV8D6oBs4xSuETrQXvS4oiYi26D8jm7Z81cfSFByfdAwVgAVECRt31TP5GE2nPIQHBuNDzAtUqOzvtn\\\\\\\/sXrK2sU01B5+vXtWk2PY2xlovSJ6JoFE54H0PV34PbjFnpKQtIa0JWwxuk3A3upkVnqUwH0fjdAe+e44H1C923A87fPazsK2FkOJ76tDrs6jFFK6d1WrW+H8aYHOATWWA+V8YU1a7OHUrLzySm98\\\\\\\/va\\\\\\\/c1TjSTQSgv6XUMrXG2eBw56Rioq1vZvQNipEpzYEF96KHa9\\\\\\\/O7j9e4Kfh0E8CWF1MOkyxHrKF7\\\\\\\/ujanDQZrnWfa5fm646D5GuPHpOpqm4PSwkPq7vOY0tC4XxA4cclbtuKo8DoZtXMugOZCraPy+8sV6rC+ztenLGd72wU9GYz0Qw\\\\\\\/rv943glP1TxspChgIo+S2V7KNLe\\\\\\\/wjTZDdgeSB9+vWRRKhJjQeSDfx1qZkjIMXE4leD2K0HBoLcJPA+Nfk1JTtTxcst4tISzpObG\\\\\\\/b8i3U7SI2BQdlRtWL0U4wp4doQdWyqv\\\\\\\/HqeNZcejSw1vkFN1V3prfg\\\\\\\/5flIgemGTJIP\\\\\\\/8QW3d2WFRY\\\\\\\/0An0U\\\\\\\/h7w4g3GWd9k9WdDb9aVZsnqbEyLuATc16XXir4WvydlIH4wSM4zg2pgfxtvH0luH1K9Qz+KjfsoNwzGy\\\\\\\/Y8Rf+ChzEGqCF8qeTzLu\\\\\\\/m\\\\\\\/wMsRfGO5mDrdDetQ8iZyXlhSWwyDBl4yswMxk\\\\\\\/dWhZ5q+Y+z9g0cUvC3VMOD6398Biy9EDgQ8rofWcovlv64O4KJ8iiqKWhHlpM5Ka92xIjorCQYKsH6ooNhrTlFXuwb0cpPKW364HKLgi6jqh528cVmN02ZTyudMFHxRWe8P5xbNWW9fYlrI7l9RQuWp0oR+5TY\\\\\\\/Z+GK7cr4LJ\\\\\\\/AMuMbW2q5J49JRCCyv3Uva8Qah\\\\\\\/Vl\\\\\\\/RtI9jkNH0azhQC48Fr9YsQfs1KgOR9uP768hbYvo2KIrjpdZGcgP2jo0AKDgIKFSGbr2IqSsWo7y9avS9HoTLwskV+Y2S0c7BHz0FuWBx1XwJgqgsOjrg3Q4v4BeLB4MdQAMLPW7iuTVJM8VQBvziLyMNEIWW760lJf7z+kD25tG3tfmWcpQ8JXGFdR2oi7a3Hi1vglaHcnlN3N3YkrxJFc6sJkag7f4RnxOP4OE6witxPx+PU0NgEsNnLoU3sVWqEHpEp1sPLZI7MfHrwRDkoHmp3AJ81SdJ6U8MaAIgOINJksBCYPIjJkLXtbEvs8XYb2mGiAPrs9Fa9Mjqc\\\\\\\/SPi\\\\\\\/C6Dg3YF9wszAnpj7rBS38mYMlsfadzH3kQiujfNlhwP1TKLRFSpfKqKDyKoCixoGDUiAxyoQU2OQuCLjcKYfd3pN+0kEMHW8I6Y4Td976+E5M8b42SFedO77ScOcbZgeeVBR7d9O0y4MbzngVKLqVhjOBJk2L\\\\\\\/wpeLi+r3tTIwrNSlVzbRw5hdPYhNZ4aDXZ3gCVM2ymCoqxcfe8yuFWWi5fCUr4EoumxfiIJ1RHY66m6TRDt24oUzEQT03\\\\\\\/30CrbSLaK2Sb7qMFDK9hp21mOl1J6vuGoAlbwYuoPKw65U\\\\\\\/iujqBsPQcW5qAgSWzUgA9+hrkHUE+0aBys2YQY4SP5VpUlQ1FM4PuPzhExeDJ9ty92EdNKw6joNQsitrCJ7WekhXjMjnUVToGZthRBkpBtp1n0v9B2w04wU+6M0SDhd2spRYiq5eZHfO+kRMLXUbdGa5GQhA\\\\\\\/k5endTtdo3HQr6C3yivt1hM+dTKYtlgqTWS3Gv9LF\\\\\\\/FoCRhwj5H2cLu+T+dCW8fuGU+M7qVX9KRxvsx5c291rUtq\\\\\\\/l0brtdEbPKaR4VM967481r0OxM3ZBv7Mq9RFoPeOBjm7CpAy9pZCncf9Z1ZrwDZ\\\\\\\/WL9mNRc6qhH8TNnwk2\\\\\\\/I5NFwFNTFXmpXUL0ZCz8FNpjNuaE+7I5BUKN2lb7eQOoQNUn\\\\\\\/umkaZRvQ9mEoehrQ07wo8ANCGtZOkp3MCrk6DuHftThSjAe5av6MCZJZbbPkWp5+ZXnC6XNvnlhQkJ218UG4RMClTtc+UEOLqhDDw9BPX4IndBGsjYwdIvmOg\\\\\\\/MFqTUnceD\\\\\\\/mL4xcA9kjJQhTF0IyybgeSYUnRx1gobll4HkthtVuxwa9HYJte+2JZsG1omWXCQn1W2xiq7sjRkff7JMAMOaBTrGhxADJ1pYbV7TN4yjEuJnR2YP+5ZOkmRrZFd8SwgZlQWdTt8T0KAzZm1AO4yPzhnBfUCQioog6mv5xqf\\\\\\\/UJ2G1oPQIufl4ENhxrNsJgwtC5hhxj9QbbSeZgPQpo4\\\\\\\/4cSbAduWSO4pa2jQzK0ocFJsrDwa+VmTLLJHCUyVz6EqR+MHYISu9Kz\\\\\\\/QzW0Sd9jtH2ZpEnmEU\\\\\\\/oM3tGFPvzWNNYKcTIwyZIqO0NcCkdz6XxSI3RI3znfIfYJsusooUVgqs0Ir9ATE1\\\\\\\/MMfIGPu88els1EO8S4oWBgFCj\\\\\\\/hYi\\\\\\\/n0mTOgr84qBvr\\\\\\\/hQdNWqP2Z+NNP+UTvQPEB9dn0IAS0mN\\\\\\\/7pF+1OxRFTOjO93ev14sVUSgXn4tn5OD42n2JasAxl+IIRDCJUZaLmTLFsr8iY7HoAQtwehd\\\\\\\/LRpgJKzPCTG8M0p\\\\\\\/WbtKRLA1VWN7miSMTLDuZYu6rilRP6kImg3om1P+QS0UbZGP5ZXWGjZ3JUBbMzPygybKtq7qIFG4GjY80Y8tfoh\\\\\\\/qUkvv\\\\\\\/yt\\\\\\\/J0tlWxrsIZfHGskl1E\\\\\\\/\\\\\\\/hBz5oQMKnwIf42JWb7LRG0QPTabGNXiM02Sshmqz\\\\\\\/C1PxsbWOAHBFnm6dsVy0OxpQFSA0JuxoS1sm+rVHpbbm4Ux6P3DdIlJfOw6h+fPnw\\\\\\\/xr8SnROLMdf69bebe\\\\\\\/Pqtgl1gwlH+EJ0muGtsev7c\\\\\\\/MlI1AUXpR+sURysnolLEoRKb8cCzQ+8WD3sYSzoJoPwE17V+UgdNJhLA2gSVTSSEHlamkNHkNt633+u5H4EXV9FZRofhOPx3zC3eE\\\\\\\/mpkCd88DT9iOn2+xsVgWmDNejAfexr4hi1iTRJFmXlyjtr45agJoNLeOHHYORgNp79gUuMSNGBPtVJvHmfvE+3P+glRUSIRr5coD5YxtBFqFxr8ZhfUcV5F4Puu\\\\\\\/uEBASO6ribjWbjW1rZWDDBDmlGS4uK\\\\\\\/p68YOCUWvB2XiosSGWqimP2qkrEaklM8Hpdf8cRi6IJbNgYViXESUdFXuseHv1Xn2jkGNqAHTiw9Bzz2jqLKdOsPKWOp6\\\\\\\/cKZRJCtEzI09EPPtAcFJlF30ifrM82MrF+w7o2Ewp\\\\\\\/SKSOHqGhaI4IVz0t38aWimJkd9nOzuGjrWUy\\\\\\\/ppD0MVwPV2ye1ADgYmrf+1rf\\\\\\\/GE0qGTzZJNssRKJBomCUZdjRsJIy+bu3aD+\\\\\\\/2h\\\\\\\/Xe6qwyk5RNXn04YRXVRCkSbGHHmVob8ifX3grP8J48dMKGmYmBLpLGtvHMpKhKJMIo5xTezwYG9DDkXWuhs4l0BxIxtnwKuqk1b4eckojIW17omuy9iDOA7VBICtzRegV00KA17vHh9ocm6LIM+Gcc9aFBjM5h14I40cOgLqDOQmHvZYHaexujo7YWOvXpMyA2acLS6T0xDJHSwbSLDhoYoCYP5kVskS+wWfnmOOP43FQxI8pAuUAuaqrkmBMQFRHxFVC6OmNZeJvw+pXdxEgbbmJbhe6i2IAXc5z9ILu8s+1aIM6\\\\\\\/pb13ywc5yN8XgP6cnFMWRDDjfs9mHK2ZD\\\\\\\/lOQuhk9Alzql3v2pJUGg0YhVblJOaoRa+74ShdCOL4cQ5AN47htItq4NCmGWfb78hTq+CX2mHpowbgl0tmHRvJpbgl3svRqAWUHj1mF+xUwliD828KWxCWDxZsMiDlzjlEpOiYPV9e81LxLEUPy7xSB6q\\\\\\\/y76uNmHlj8vN46SNriRv31iHG7f5weEDhlPy8SPKBgKAOu2iS\\\\\\\/XhuR0l66\\\\\\\/9Dg9vS4ajwVWhgCjV+7SG6IzW0SMd2449OlfXzWS+3Wr27jeYXFpX05wYYcxvMktC+TCw4637LsStDk2IO5f1G2YQha1mT6jUPRxAwcHpbm17Jqxcrz7sk\\\\\\\/FmA7XzKVSupY1Ja0VTo1L8esCl77\\\\\\\/tl7loCYBGHy3KsH74i59LJf8QxxNKeKsg3K3AMR8l8NjN0P0GK7+0R0eRQA6NtOyS5roYLWmket1gWwGxCzH0il9F8LTb8zGicGYR9xCPO1CT39Vf\\\\\\\/0SDi4wbxY6SyvshfQ\\\\\\\/oCnI4wI920kgJ205p\\\\\\\/wYQ9NqzuICoK8gUVVfM4X31zqrByfLOLPDaBI6s5rP19D5xzppU7hM3yhDFrR+YrJyV+zjf8sXDp2VGe8kqQ9q9SJRZ2ThNEyBdUWzJOB6s11RxzKuceq8XtGfV4SAcx+r71J40TQ4Pe6bEqkNXC\\\\\\\/JMTL2vgbJW1TaPCwQpbn5iFFZB0IrYkIVBs2f0\\\\\\\/7qo73UzPQizT0\\\\\\\/iAJCmFIcgN8hjC0bpgJz4oyUi6iBDeNcALflqUQKXOOtX0KfskAoqv1fuX5n62WObYrmXC8iUVQZ5cdN16ewPD7ZKYaiGPLDsc8pvP0199R+M0TbAjjHkZOccmfyLyffbqoD+MVXjqTYmSMGv0VkALHUsKYHxM1zY6S82nX\\\\\\\/+YkbKcVKc0UTPosfQQ0iAH+K33KVLXtFgUFYcVaoYXiq2o8CUASpFYXwKqTyugzqdXtKWWo6WbLTvlt5RInaOu8hcy9Zn4yfdyElHU94aW1ltcL07N3K5pyhH9gnqcU6nEsxv3o5fNfNLYaWvvaW55QyiupNuNAy078JqD+Y9ONUCHDAGctSSD6SsCdEK3nNvJk4C8G4N5osSzddJwRScQMuTLWZ8e\\\\\\\/XCtkC8+KbRnptn79xJWhrSkPtXPcKwnPBUaIgwcZGFwuP6fGgIDz\\\\\\\/yd0c7xWbQXFMoR0la\\\\\\\/v1UZ4rWJHIeDOkIB183cxt3P+N8wXi6MmkxgfUTw7YOKw+j\\\\\\\/nwz5RWgyg69F5tei4cSotMtiThSZyf2UG9OxuL88+RnkWpAJlDNumvUp5D8YBaVdcmBLu5Jxdh1vGpqMSCkm+\\\\\\\/cNTsS5JFvEFQiyqZeUc01bEZOXrWQCjiwgknW+OCiPhKWe0\\\\\\\/75uMrAqWGkeSV4utDK9v8TDxndhpFc30fCbW9d8sgnDhgBbhezgNUD+TCtU+\\\\\\\/nBY8rp2RbjvwnxaiXj\\\\\\\/yWFaMPMDb3ALvqVVjZ8ZVdT0SLl9F3GC7QioZnr\\\\\\\/QWq\\\\\\\/yLWtnQfERvaoadEbN89tiYw84H9VpulFCh7F+8WJpTL+yRhTutJEL90c45CNN4Pm9bvpetPHJXaCnCtepDSauJ+CqTbagy5rWRRJBLjd+Ee7NVg+xJDG6QFnjJuUmilkJ6logIWrHJ9Te1TMGMBsTwL\\\\\\\/YewXMeMJv+wUResODyvaxVG7OORkqZAhIciDMaqeNGlr0eWLn+I8SHVpRYFZ\\\\\\\/LTzgzpTQalydNQUAf41BW9CD01rbQBcPwyD1xCeEbQgKDFQyGa8sydSgYxG+mr+SNI3dPEbOp64X\\\\\\\/8e9szNEwvHi8B+JBNfrewuKPfpfJ069pAaqJXvK15Et5ZJQOGX8Aq0JRDNcl+GZth8HnwelchaI3bNKDD3QcpN7nyMZLxaQOG1AUQpbGHQWkojeGF\\\\\\\/IPg0qoanhURtjowTXMrdfEpWVEGcBBgnJqOFNNyVYqhXiZYpO\\\\\\\/jglToYdYbB\\\\\\\/BFkvTDSIYXQZck19\\\\\\\/7pzIWZuibuIBXS1ptdyp9EoEjSjw246vCytthETqr+sUotYuTNg0yxD3AcpfupHSp5\\\\\\\/9lo7l9IG9mwI\\\\\\\/Zo6AfqImq8kgT45TbsZZsoRpbK\\\\\\\/C1j8NoxPVRSYWvv8pL\\\\\\\/3uc9Ho84AOy1soD1H0lLf\\\\\\\/U0Aq2Br52idg73yC74mruGX+QScxwBBAneFDl1BugVqLKfodajnsPKQMYhFgcU8AddZaL7oepLSeqKUc4VOsvoI8QAlaICIec0qeJVpg3pB1lSkvAmC6d9NjXXMu88F1n7g+FuxUyMjed3i9RBrsnwNVKEIRB1KoKr7Ic2YxUKTzBETbTZ2s8A0nPqX6f1FvfNYdg9phNdwoORTKHI44L2PS8vHncuo278rncQmj0dkiqkiuwiF2Bat7VF3YIMpLNB3s3htN1FaZ48F9K\\\\\\\/4hIDayHOuTZidlxxmMS5Ai2j4dhpXi8zU0J0oV4BviIQMINJ5mcmpt9TIAkr7NN2j01NGmHDJ9ewHYsB7YwDizmZ76eeAUnWlwJ7cX7pCmNMHSKVwgLbVPqU4O9I1jYlivu6C4YbSNagdkmsNgL94xXKX9JYuYLOHf232arEoeVXTdPZJuTd\\\\\\\/9xa6hlgM56eylzgd2sSXJTGx3NC5J+v+kGh48XSqgq24tft0VX1tlKeLo1VS3nVlRk1jTXe3z70AoA4Fe\\\\\\\/jnTN7IZy\\\\\\\/Ih4Bs7Mx5fdVCdXfE6fhQCM2POHrui37m6fuyOR3iRrykQNED3HhDcbG2r112rrVyw6abTBiyUi4kUMW2Ls8vyCDCZ4n8N6ZIY4TopG6ITkAkuB8CpHsOjsK3lflOI2vfL51Q4KIejpB667dUrF0ckbkTTE4QsLLaXZGp4ASWS3xcd1qlFUEEWtz2up6jchQCYcIA9Ax+At\\\\\\\/Unj9m\\\\\\\/Qg0hECokNo49T+W9KZgh\\\\\\\/MJJ8YAxRQaIYPuxh79uf6uFx\\\\\\\/DGsAhQHV94mUS8pIFgO\\\\\\\/2vZpalrv3Rmg2xHpd8Al0d8KX5uiwShHkQafa1AtnA68mTUbli+xMzACSwPvPaXlMnnTpL\\\\\\\/S0zpnvI8ofQs7t5iSlZ\\\\\\\/THDzx4Yt96NSBeFploJnf2hqwbm4ayQSZnO5gjTsw5B+Rjb8HIwrNhGPPQteSwo1mdEfVlNgJCeIr+aXwxKHQR9qfxhMg4\\\\\\\/gudgjiKqBrPhsvhS296nergtJ2Oa2LA7etVOIfAUeF43VCFd6TL+Yr80RILI4tSRh91g20l5TXGExQx\\\\\\\/90qibNsXplWUxXYUdoqLPPtCZV+6RLmLyJCgBVKy4siCcYLpY6VO91tAcaUwSKq70RniTOJPf9BbZu3x3S\\\\\\\/OCwqnDJns3wY4yT2YNce5fhceisVLmyG0gR\\\\\\\/jog6xhlkbuJmhRmnwJ1zdPRaq8rWp9C7pIcOpYDUPoCyDeL8WLrzZ5af3hlna+w\\\\\\\/Pw3LhU8QydTb9\\\\\\\/mGmIrpeO7juF3IX\\\\\\\/3xvnkIqCZH8DGiWZG2SlQGkDn0BLO0KPKCUbS1bfgOzKVQf9GJT52s5KJzOqqTpEnrOVMWPZjw78nr+a56+1nFnii4s62pmarTN46VSO16yab74eATYUI9aXmdCbWqo1257jjorhWc4rKV5eEeiGulSaSJAWVYhxu9g1InpZhbrurG6t4vlZ2X+73sACeqICHD\\\\\\\/h8vThpAvqBl0yGQftGspoWcZm124g0EKixsFaVYJe+4aYRWrhney8fzk9Hko4TEc6AxLz45lV\\\\\\\/gydDqg68GTfXzvP6dThVucJLmSoX0a47NJK+NX1GewScTT3EwouqaiO\\\\\\\/rKHxihZmcqFg1ZleqS9VKZlHpPEi67w895BV80Vbr5zqhRskO9zuiR8rfErwflGJR6niMa6fulHifRVOl7XbAF960nyHdZP9xfh36BqIIAh1s6so8DuEgEVi5HgCassvTL1n1MC8Re5Iysr0hAn+NC3gL9q9yzdzvWugT\\\\\\\/Fs39XqGgcukGSOhFaZevPq92gseEZF6g\\\\\\\/muQCX5r2+oQsaUv5wdJh3B7rp8t6Uvnk3IAhSittetgro73gF8iqo2lhgEVpYnlhjEUCuSG88lPzhdNfxUmnap+0HzqwGC\\\\\\\/Fu3y8IBJ1EPsTBDv8geK+CoDcalAR\\\\\\\/UoOBhdtgOV+0mBk4Osv3UAmFsVsS1WaLLXgXvJyj2i\\\\\\\/7zYgRpqEE10K2e4eyv4bvrUEh502lVqcd2xFYNEVTmeoUYz7TsSM6RbdHYW0d\\\\\\\/FWBItSZIXK4vYLWGtiC\\\\\\\/lqPXlgwyyY2VN+5GTQlDLmK40d\\\\\\\/eNFNBrWs6SYgbcILdgoLRIEKhoUqkh4BC0mZUb5MXzShX5zrH28Urb5130Ftkg+pSULlmz5zRg7xIspXivcxyj5u38MEInOXj2+HEwpqBuZ8DjXrptDmgpnGqDoDzQdZCFXdhdk+6KbLa\\\\\\\/pIIoz0X1A8D48X73ZLJtZP4aKLvAf+Tk7fcT0iB2xX8Gnc6\\\\\\\/p4o\\\\\\\/74KGCUyuBzybb77luqck5PrAAoJrvFTLouEg2SJCXHxwTPu2MAD\\\\\\\/d19xNuYt0KCyXYAIvWgsxAoe2m8XgP6H0hMgH1mAS+QgWDuv6TVydUoKes34hVeXsHf0swn3BP6PDqXD4q9wsCN\\\\\\\/YsWDLp2f4x78TRbV2u906bFrV0YwxaYLV3UZRurJe7NDJUgfJxVW5vNszWzwVXZrP\\\\\\\/wqZ2EndomvYVPNun6JcdpKsTZPDys10xoU9UnsFcCVu\\\\\\\/cxFvTCHCpOZT+FGqpsOYTKgrf5zfmV6i4RiLmXlG6VS4gRokGHznw05zokkMJJxqi0YUdnsPq\\\\\\\/gZ+q76\\\\\\\/cd7vRmuvZGyrC8lT5tDsSNaLIiIrbf6lfYWkHE3soiDz8AtWzD4pMSCiVEdjszjJTfvk2kKazews1MXdBmj+hOwpKT8Uk1CQf4KeazLkUbXwRCALbogaCUD4tDzIV\\\\\\\/vHRVGeSDbgk04onX4\\\\\\\/hymBIPX0HTqDRbaqW5a8MbVt9BBXTkSA7Ml1NjKNr6KUnqiroUz38WxJFZ6yJuDv3OmDFX3tF2nRCKqqlauw\\\\\\\/ZhfQglhBvyArUEBmSb3B6g3\\\\\\\/31njhluL5UsZN7RVykNLtNFHkYeA8XWNvHqwG3GguazuMZ0\\\\\\\/65AJL0XRcr1gl9s2uebsr39OOHtSxJC5CiVvIVTLBoxLoKxSsPjKhFN4uSItCI7XMRgfdzuaUX72p+9LAnC6xVn+B7dZHhGQblvezsE\\\\\\\/QX59KGsTOs5nI5s0vvDJsDte1wh4sJOJ0Y62Y2f5LvU+XYPR\\\\\\\/tIsDTBBTrq9h4k0iflQXEzS+oR3Ev7UPB0srHoiEdeWRPkPIMo0E6OK3skebgkgw0cxMjSeW\\\\\\\/wCoLbkAFvCCmvilBJ0b1OlipKoP+NavRtCOHh9oTpaaMWB7VEnH2wAOWgwZ9l8Kyxjn+MTdVpVremGr7Jg2\\\\\\\/N\\\\\\\/QfkZk21YOk0xb6kU7SjweigAkOYH\\\\\\\/\\\\\\\/9JhLIyQ18w0L9WCwtwzi24kLW9e3ZchcGTetZzmSQrRLG4zGbBghWPVDGAkvtTGO+bsYtHN9gWJmOFhYnSy7oskH9pprEsPj9Mtg6+eFCApjbxn8JvqdnpEIeStBOuBIAUuViTm7yj6I5u1HGcpYrMJyhy\\\\\\\/fXEiUYjc2rFRgi1+ld7H2CkyNVt7wd7SLHz2WrT9vsMBaU2fHEQaYUnnnwmvoGRpvat4VQ95OQ1fhGO0QLdh1t8KT+UFflJ6AGY9xXfjN+kHWu6UBgyzWfgRjiE0zqIZVf1gPHSbsqbDsVzs7sgZWwE0h1l7aZNwSpKLvEq8z5CrN8zlnbqJMpnXr6KKcOy0pNVBMAdchFs+Fg9T0aItAnBHE62TZGZaVtXKTN5IxVshwOH9sg7pm5D5QkmwUsiZLhUp6usVTriOGiNLOITsasxG8E1mgltd4Shz6LrfkLVDJLHMzZ3o08lSxBF16FU3djG\\\\\\\/FXTqDDnEZ4nSNjgTqm0QTxffyzFVROLWef3C0ungKwG2BBnArAiBA71h9fuh9LAhndBOCqeL+lcbA0XtswPsfdk4d0+XqVl8fFwgawle1Wp8Ymwx1x6m3gvLYaRWkAnZxBqunoNWFxERoWi0WXwOKSicNTOS\\\\\\\/ukFyc1ZYyEhTWtbVJovXffK4kuI0s89ousC7Ibary28hSeHPh+naPzBZrX1XlkmSL84roEkxcA8FcUNqCjE35v+ntmeimwM7R5pXE8kDvE6rZdRekTrPHa6KwweZFOeJNTKYrX0LkQ0OxKcdYfHQPzYM02pjjPI1kGkCmDnV1\\\\\\\/gl6q6EnbcQSOVw+R2GM1SIZV2HMI7mMDGMxoLvo7LKOSOKHb7bsY0TTo5DHmyu\\\\\\\/g1G6lbMEkmA\\\\\\\/sqCYVpS3kkRYm36yudZh3eakC+yX+VpldTIjLVE7rns870Wx0aQvB5zrbgJt2v+VVZ0x7RlI963+0Oq3P1cRgSWKYavMP9ZsZWZZp48LdTLZVABhAbXUd4cIoTjIKLHADI0Jbh5+7TkIHcJ\\\\\\\/Jim0xhm+D7LM4cFhuH9HVAeNsn37wJS3WR3ivIOTIwXnKmJAgWAUJi0tXY31U2a6a8fG9whDp0XVMWoGn9pBaALtv4us02yodg0xCX8iaWI4ejFaQE0vSfqOixLd+kOBKkBloVKWFjyqzStIPr4zoysuXR38Gc2r\\\\\\\/ls20TFZdij6owxvDOLLRq0ThhNuAvX0D8RLz74N8V7NELex4PZq6w\\\\\\\/OzkMoKGEHvwC52mpWaZRI7kbZOb0IjB3g2hryTTizqeq6b3R+Sr+gUh9fE3HLO6h4ZRjY6ZsHAElpyDrLdkHENrTcqd6FF7TEOklD5Rpt9sjDlf2wx4kytF2GrC+a9U1wd31EMtXmu117aqE0Bbdah30LKsqntqFxB2M2a9qYln28Wwgqr7aKS6y+iSpCh6xObd2RN5c02zVEMqE9OwX9Cgn4Gp7TTzvgsrGA94JqyUVEG0Orl1Lp7NGRrT\\\\\\\/GHYFJAJybyROPG5M9qX9cX0pRoOsrAjUThFTmLnPHw7SVEEpTMP4BtGVyeRpNqDfV+9kIsvBgmWEf7x++sSYWGsoBBuqtXntLuIPsCj9wUnV1ajuQLo\\\\\\\/Vm\\\\\\\/M4VeNZ9ThLmPOBbeOeAbl7Zzw\\\\\\\/+OwSO\\\\\\\/522JiJmFhrwE7bEFsOOeyYzLul1idNjADCamJ3YpEAA15plynQATfqoco3cHhYmUOOUbS4ZrzfUhanqW8+nMdWiqSS5txGMl8gT9UnQJBi3GPqHtTtsnMxGNOLBbBzBGSNPnvfLwsVq4sukwrT0CrpaTsc6sLj8Kz5ZWTgxV6byqhBXc6+WeDV4NhhJEWog8BQyvpihHXHci5pWEHhN1eaqMC9tx4fYbRcTko+Us0u8gK0SMCzHuPwMDmLzUh4WcH4ytQR+8iPyd6ofsKFuyV7kpZL4wUxEpInUxpZSKEOPhUyGc2bTCbUYotxNJQJkVEK6cuMrPa6\\\\\\\/2fdIk7VNI\\\\\\\/EWmg8gamHz8Fp8gOZtMS0iXpRE3eO\\\\\\\/tr+Htpsv0uiCOEuSxHXefM5\\\\\\\/px60jpYY1qsgx63h83LmRaU\\\\\\\/wpmwOhVDnkJrBlUcFdYzSnOCK9f+o6gY7IvsgYBVWp4vRONKcxylRfDbSn5uUFQxWaHzSHnZWVeIKdNkV6scpiyqvAZZSqzBRFWt94O9Pv68vrx0bNyAwOM0vZF22uSbz+dIumuHrCKmHX9w43REGXBAkdFjRHPcEHVk6Wv9ntmJiGEDJo030FDqy7dy5d7MFQmn6aBKHpelaLtT2wda\\\\\\\/t98amqXRo0kMI825nJuC2vqzRpYsZyRIwaOIoz39kC+JehbKJ54kD\\\\\\\/9IoPx70+It0055YVfjMWsggxAaLF1AGzi78EqaiabIO5ZbfIi\\\\\\\/5ohU\\\\\\\/vuMPrdmKkdNEdm7SyQX0Si74+hlgUu2qYG2cX55N+JthQWV0RjYvUyz6\\\\\\\/B2EGxIKYVU8w7fYq1MpQEd0RxNQhwbhl9sYW0wrGX0iQVIhDkIkMjvMfYr2TrGN9eRgPweKgU0KwDcsZytLB3t2IT7MNliKm0K6mk6qvYAvYW0bAnKqJDKDZUwERTbi8dPd\\\\\\\/he6tNwV606C6hQTGfGgJYSP1rCXy6ZXF7lNZE9VWNJI6oXvQ\\\\\\\/IC2xjNCq2sbqLE5AnFAfeY02OsxDRw\\\\\\\/abUO9nVg1ZG2OcHI7jKHRLvICr6u9RFDPeFsNR2qYFaHueWx1ZlmbEMM5GuD7GSIw8HAt6GMxM3DjxpeCrA6pxlDze\\\\\\\/NxOAahBVhqiVNBTnu5+1Au35B0WlB17SxfecOUHTcxbGtB\\\\\\\/OP8FAE4uJTsZqo7K9mq5sPQxhuC5WE81SC6WAP1L79hofBdBrq0aZCN5Lcgurrz8VMzZDlUylXbNYfXpGCXnIkG9Lok2PB5Uipe8qLsxA5f3JlBj8YyCA4AoPwop4tduQZikSnUhmYCsqjTP1\\\\\\\/s\\\\\\\/ohJu32ZkV0Y9N47Ark0BkRBTh\\\\\\\/CaqKJppFPTOlPFmcqJayzdLODSjqOlQ3xwQ2j6NnVSu\\\\\\\/ZceCbhsivas5TvoB9upOeSWzuibR2zRdyU\\\\\\\/2WlZByIPR1LiB2cCb4B1G+a5A1qgQSzpKt90ytksGSaYTmxCj0SYMqQF73TCz9NDye5KmNJ9NMkaCQovUiJgZihSk2RZYGATSSVIZD6mx\\\\\\\/Q8BZNPUof557xx2TaN9R6Wtwjos1Q4JduKDRLvAB7jYYJr+YUfpfEmwwW8uxj3kpZrlYGc6dFD5+z30S7WPslWq34s0xYfR43eW63DERlxc669JjnUffoj+G\\\\\\\/6pvcrEgiazaEeph7rGq+VNOqddwYmuGoDc9V1XJubAHGZ823N0z9u33VB+R0uU7IYDAn2jZsjmHRysH5BEtWDY1a5RPFG36QzlpvIijhQLjbgyV9dNA576S\\\\\\\/C6+\\\\\\\/vfzMsIDcfn04YK38n9s6pGskXLhXp8rQvvdL7nO0qte0fRK37KYUZXzqQGqkhHCHPCieu\\\\\\\/Rm\\\\\\\/45wx6UUUeBF91OjMzGhKXxfnKlsDnBvN6d74zySFjLZEC1yqEu5H0tfvaF\\\\\\\/Bt1KT9E4Zs1VWSmegEtc6NRm7DQyC6g1a8wsXb19eVOtF3U7GSLpCBTY+WhlnhRZ203W4Avxcm00cH48mNz39FW59iHBSJem65SXNOuIb4g3SlD9G5o0fmKIwZFfrfc8+0B6wfnHk6EnzxXppYKMd3fElsh2wrqeCtmhhRhTEgYxRIcNE\\\\\\\/Re7T8HDsKtu2OYXFGQ+kYuyvltrX+WT1DiyBAkEnBeExoIVxwQBMAyGab61LhQao8jF4xaR8lOYvxSvr3wT\\\\\\\/dDz7wVNd0V4ap76iRIxa2ilhN2peRjnPJk+LJ3cA49dws4s6PvcVOjdrm3OFDOvojB8LSQsoandq\\\\\\\/HICyL+3XilenAT2jNehAeuQAObI5gp04ljGcC3gMi9uSjotmkLp8\\\\\\\/VNHSzMyPfhn03uOxnvLCwJTKgP9X+VqWlKPDXnCa4DqwoWz15ofR3wmsegWSAaEIfcsmqhwl4xsmR6GiE8eklIGofY80qTRq87u1f28VNRhr8wfonIZd9e27wgzvIeRecBrWcUufiynq7R9buGfuSD1lruSD1Rxuxm2vZ6Lfhd2hibXfP1uloa\\\\\\\/iiyBMn9fCqM2G99X9tQUYTNsIx1Kie\\\\\\\/m5WJDR4VgrYKOEM0zHdlLCpFcVESEg4u0KgBcU+TCz95bv48di0Fv9EZjomWw2PlQ0ahH3sHFx4pu08yguzm76cJCCD9qhNvX5vHY4p+palhnYObqind8B\\\\\\\/Y\\\\\\\/YLqeiYs7gBR0Vr3P+7xL44Lj94UnY9kNt9aMU1uITyVFTfF6rfOxryKlVGPFqToO1XaAr\\\\\\\/MPrOG7id2hsCLrFyCtjGZPgt+BIiOOSgMlUlET\\\\\\\/HfOib+aVXb5ia4EuEpAbo6vb2pvJ0JyXuxOsXW8KHsEKQWfkwHW5aFMSGfa\\\\\\\/ZRPd8hIufMUdA6q\\\\\\\/tk5xZGwyUMkAfMhgOzgHGgsBPy5jiImffiNjWzbbMQXx8OTANXXz+5h2CuN0vFN7Q7CotbHlBxzFufiSfI+CurPLvZvB+oodMQv3fE4Wzg9XnSiqs4s7znoO1ngxgUYn0XYuCgt\\\\\\\/O3XAdMVoUv+7sPs2CFJZqbMb6SsMfAZZpgpwKWtbbhvg5sUjzQgo+82IsDjmynGDJrsVAVhWZj+vrlJ8fxrVTBw\\\\\\\/\\\\\\\/BMABTB7MFYCnRFat+LltpNcakzh6uZBVGbVJ3tSLz8ExWahzUIa\\\\\\\/trlhBJRD2x93WCJVk2+NWjUHXcMifThRtHsxz54GUBlQmS8ceXqns2PttB6y0jkWtsopZCo6no3rxD5wha1NY2JyP7EyN5lv6JGrURJ+V6eViucduHuKpoNzONkkhjSJIFZa2gApN6wrD+C8+dpQhWHL4fp1odIQmP9b2U\\\\\\\/B\\\\\\\/nVTjTGOF20hEwLD8MMXgRDYFGUrgcrqdaQfkHlq\\\\\\\/lxIvfFaq9WIdEJD\\\\\\\/nycS0SKY\\\\\\\/X5sovyJ0bYkjyCgxhWfMOitQ8iHOUfUwaYmpG5Ux+y2PLP72EsQCvU4KKa7dTM9pguIvBVHvD7yfNSPr730IaALZ6BJ4sreOyuhFUit06NmXSdmAl22HWLfYkpjzjLZvzD3ZlOi3qyQqAb+TRZGiYqXCmSAydHBRDPrF2ZukovyYVHl14m3VDAncu8209wbxVCzX2XQPeG0IMQYcxXHltzWTmRtG8VQwocBCGFDi0kDQS2pQXua\\\\\\\/4ud22v8kTEPIYPHGE9SNeJdR7JOzUui21KTGXeePDm4CZF1mkXSP926epb3df5u2BJ68ddjtr0dSgsRL9d8RyiSYrELlIo0Zr5uERteyM9\\\\\\\/ZRfuk73rG5rSehy4e7HnaZZLjRuJTQu5b57qvrrfIK9B8H2xqx\\\\\\\/LqLNGkBBDO90VtG9mbQtlYIR2xHN2htcfPDOF2p4gmBORgxeAy4w+XLjTPAj3KL7cvamwMLnyK0jW3NowPOqZU12q76BuV2WYzGu\\\\\\\/\\\\\\\/2\\\\\\\/W3IYBDSZj\\\\\\\/vVDCYk46FR+73z13oLhcy752pie7PBRZ6E+iBZZWwD3yXtvK5uYNbE+fJbo7MP2r+3mB60bX6dICY2a8iBnenniwe4csmfvsk3AsJ4PVnt+DkHgijGsW4XVNXbj9AKthABl\\\\\\\/mDcDO+ZMY\\\\\\\/z5EDi1a\\\\\\\/t8C3WorXtoP97RwdqfagR8Ft5X\\\\\\\/HNienp4JXAAKUn1OMfafeOrS9E9n2+jnZfvKHH1czLSx4K2j3AMMzL9jW45eLoy5TBi2x6qZN9rq\\\\\\\/V9ODfC93MEAviUApX6Se70PascaFOiIczOlLrvZ8y3JSd3vk8x6at5\\\\\\\/wztEmnfytXs9xskoR9\\\\\\\/tVyTFkTGGqqI273a6jKKGN7KeAogI5olBNx6sukq+7KGsQpnvg1jHhltWCAoTsBFvbUHG+zM50d3Nm7qLOdGVsIxpWAzvg\\\\\\\/XTTYnlrKyPS5sX9fvaq0ocKpJoJDbNjs+StLDM\\\\\\\/+WAW5nN8hISKYd53K59cI1EwEcHH4M+B76VKYZAKJVUSKovC92vXG3DsZAYA1kwaiW7NTY8NPod1kd37VoVMig8FZ3ePnACdGfzaZpjqboRyakUuhvv17R1229b9BHF7lvHPoUpAqFricy2z32t3TpaknLQ5Jfw3vPpkPKgcfD\\\\\\\/tiD\\\\\\\/fhcZXa8mpz1U4jcdt57Dia3lgzOFq2n6PcyJhqvlQbK5I5zOsPd5lGlIpmrUP1qClIojfp+BdnNdVJDTzXQq9YUL6TVyLnWJr4v+v+qmajqLjmYt7hCfyaU\\\\\\\/sFuQjdXYj0wounmqPFAikWOCptH1rfs6rWOY42cJ9vMUZNH5DrnCxr4r04D4ehDnly1yEAqMLqYLnurnDopi3vH\\\\\\\/IaUuVhWN5kQWQaslLwICE5ahBcR6fGLzWzK97UZneCcPTyYtr1BtkokgoJPkd4RSAz9lq0JbGW3c8OwfxQyvJMXnL6yQbN87+ZnUxSHmcKPsKQQNkzLJf3s2WNjXOkZB2PLjWOWFvjhwiYvlaGlctOeHhH3o0Zd2K\\\\\\\/uh\\\\\\\/\\\\\\\/NwkbLJoDlJL0pimwlWAmyqTjAzoVvtwNMhbuLC4g8icVvJWQiAucUmoURn2iqAys6vkbowy2w0brdXwA8Zag9XmNES1GTx4VteUGPdTcz6HJv2SitXQIbWNmSkFthMdKWdxlAAL04n5Pdxm24G5xqY1VO9QWWT0rMZNyZIM9B+Iyqa8udINfDbou+ZrUV5GZoqSvsJcn9aXumSXShvZ4qE+GBUzsnUb6P7mIxrvqd7oVd72SNeSpq3VkfwUdx1FsjKQhutF3rirm\\\\\\\/DD6YHvspL+dkNf1UrRtYbw5g9RE7DZ3WhOG\\\\\\\/KJaeAOfyqok9\\\\\\\/ey\\\\\\\/gtXMDk6cfBovI05tk7R15+dxS7QbjwnORkotrPvHDvKQ7L2ULYJn\\\\\\\/XzYbzdGsnxJQNOkJhRoid6MYUjuFd156mgWrqky56pKnhqvu6fQQkb1DA\\\\\\\/VmIcNqNYUFG5Q9XLOtuQWpdYQTrAloowq2SyRIAs4hlRH7e952l+CWjkWLj+ItObSkpE5Jkj4i4k\\\\\\\/PGT15xo9R3\\\\\\\/ZvekN6qgvsDZpNsmjNL4ikDTJPhWAG80CKabkiCvCdh9K9tGEAjIfSLyVLh\\\\\\\/8jjrBk2R6Bi8TLl4xxc7K5KiWJb7Jp\\\\\\\/weKiEfeEI17hV7Y\\\\\\\/txi6Q7WQa6RcXJTBi+x+RRkKeak287e5EhrRVYfTaxvnxj\\\\\\\/2xBcSFKHQfvKtUalJIbgTIbR00+sgvOjF\\\\\\\/93vbeqWLpsRcYaD9AcrSzAGddrTljftraVDKtCdX2aBF2BOTEv\\\\\\\/sGTnCif2\\\\\\\/uVrRBRCWkd1ENf8DY4qDJs\\\\\\\/wcqfc+ACQzOQBQjKH4oIwNqEgc7eI00PoDDEvx1JEGvAdlU0iarvlwLvuNelKdDH2SxkP4M8QEgfkGO\\\\\\\/PoGqxrlQMTGv0dTgUPDgt9yu7PIcyv5LHR\\\\\\\/a1vE0m9Z0QoCXyBu67K4qIdjaB0KJqlQLmCaAGc1rXhzvC8AnS4VoQViBqvibb1v0rpJnwo2RgDL\\\\\\\/MY8SrRE\\\\\\\/LSy03SHHok4J4w0VDSAgV5F4ibG+sisvn3qsIzHpCEJ0PZeuNOHusa0fklBoLVXXViRJq4IllR1Khiza09yRPq9eRa7ZRLC5FtutbNV9Actn3YjeEjA\\\\\\\/ly8jI7ypXDcpXpMcdtwJQC0rteG1cVTfyUG\\\\\\\/2IT8aQuXbjx5+KUgYp4xe5bi0B5wu4fM74s+OnYVBwPZGYlQyO1+1yq\\\\\\\/FxzhaBV+LqztrTV77RNchme1jNwn0Mijn6XtkyFlluL8EH3KQnfm304oiHUGhV7hXiSTWGQ\\\\\\\/qUEBYas1xMvEwPDgEFs1FeZarU07fd67QsOP9HNcMeY\\\\\\\/ApmjmMNtFAKSRP\\\\\\\/Zifysrs9sHnON6bP81Za1OLoG+I6tA3dI3CZjrxe+44ldEej0VO+hBajZFgnxOjt3bnM095fuFImC\\\\\\\/AN57v1cpvy2y\\\\\\\/SG9AmJQ8I2V5zUaFywKxoWE4EBLMJsH0Q7wsEysFu+mVBYQb33wbysnifoOQVOIAmzuV+21IdAfD5KxfmjE0fjmGQdhxocr2LfXj9TgMLMP\\\\\\\/by1p75cpnJF4plxkJvnKuSWNgr4PQD14pGESGFkpMMb4OiQabsauAPCpcel\\\\\\\/+XVkP4ieKhmqOCwFjewk8n9K0ycbOkonCAer+s5mzKlD7BYkddIX90kj11t96ZFa8BN\\\\\\\/9bXIt3JQyMM+Ej9\\\\\\\/87zLnIEEYjzDfZeGwtm91frgjN8Gwpbf5cU42RkicIlFm79gTW+mpGzn30mM0ZrV1tsl\\\\\\\/b1eyu+YYnIH7eLzMaGKkyqcFloHJF8rU5PKW1MTIcqMgtJk14dSRn8jY6fcnfyq2x06whXQFaUwvuvqsXlZI1LXS79ZyNfRX9REZ\\\\\\\/PS5eLytc\\\\\\\/myGJygTy5SD0rOFVg6+8ggwGZi+BMSzHVCBoEf06AqBY9ZbMj8SgdPH9LFKrotxp0PlIuPu5x2G3DWpT9Jc9oA2\\\\\\\/WT6E5thZQ41tT5DkPwy3IYHqumXjMwN9zcksKQvAkVCvSUHsCJPrgidSa8LaQJ1RSlCkBswx49H7vOg+85FWHX8dznlwMyTEhgjOuIp7Jv9Y0tle4mwHqy5klJ1PJuKe+IhKbNH5xawgdpR6sBAplXUhPWsX0nYS4eL7KjkQPVqyxSNUkxNPmiZahtremyjFsUUOcIw17Z5OswUEF0JWcp3oEvxF1EoE5RdrShhdPZtVn\\\\\\\/RTP1lgXrzDLyUouhtIL2oJZn+\\\\\\\/5LDlhALm\\\\\\\/MjaGQiR82fsV3BERPVLO7qAvrg+C8I6uWfXQ1OTJ6RFBlxcQN0UoF\\\\\\\/U8wedBJD4Krkfm6mY2c6h2uFwuCzaRB+ALIKxBOpQy\\\\\\\/ZqZOZtfJDb+oo\\\\\\\/e6k1nBi5xkBAxjxB3AyI5w\\\\\\\/JHnuEZlcJdDlSqjOHcjv6XVoh2dohoJcjDs4Ve\\\\\\\/rz5cPmAqCsNbH7EGaaDc6h6yQpEuVHu2QHXtf\\\\\\\/UDkZsM8mPilDSKLuMqhrf+qCOkf197I9BOUo4P6XlS7eZP2C2njVaQ\\\\\\\/JFUyHqKgOLKgN0XKils2BiDcOr7XIgFgYkeYm63jZB2WqN1qA288HI9lGwMuVKt5IpWe5JHvO4C7iWdu6fl0IODoPamxPZLfSmEsSZjZMQzNDClWnlpmuVp0MrPiNBZvakxqpTJCx1+BLK0HTv6RdT1Gr\\\\\\\/+f7bRBfgNQgxpYRB1VU93ESNxrgaMXLGRLD0idDgNBTrUlM1HkJ9f1oBwh\\\\\\\/g8qY6xiGkZvbOJOvZdL2taqV\\\\\\\/1R1D6H5vzTQNhMITNnvtAK+EHSAFvOs4TpaGunvWWzV9sJE\\\\\\\/ZKpPZX9vrWT9wDpOHpjkeZhgt\\\\\\\/xW2W\\\\\\\/KAohRPDizHURxgYW9YIU0mqXkQx1rqn00LREhY2SjYFVcAZJ2bT1fCHP311tReff+uO9L8mzLBWtAf4aNQ3+TJU0f3o\\\\\\\/k9cJaT6K3CRCukGAro6\\\\\\\/sibzCHg8P0uf6a\\\\\\\/emr9eyXuhfWm4onXoGLXjyXlYh7bV4GalpkZEY3HA0vyqPmYjX3MAvjWCP36mClojqoinFP5vvxkM0carv10Q\\\\\\\/WwwUi4JcOJciC6ceOvZLm6EvLn8XuEanmdpF0eitGZ+fHuCH48pi4nQSCiQPvK\\\\\\\/iyKFQ+x2tQ\\\\\\\/7TWr7w6v\\\\\\\/2AgtOO+WCGClFLRyIj5w8oAlXK8UQRlW0AoVe3O+YWnz8TjzhCJN+u8bgssc8UYaOhIUZzEbsqY9+34sVBQKL5Y9wJvP+ilCilw6ooyan8CNkyR697GxcMcyHcY8Nn7DRYL89yjB083kOVaVzNCj4sytI86FyCyuvOjFxpgc9HroskOv42Puz0UbfYG4PEB9u+m4UyHNu9ek6diMimckiMzgrTZ1gJ\\\\\\\/w\\\\\\\/XGexnhU0tLQxXgPbCR1DRpqyNupcD7LKbxoRRpDkA\\\\\\\/eFfSufjPbJu5hsdxRkKfqltJc6P3buJ4+Aq+WVEBSii4s6NX0weayyrm9k5QZBEQMNmg+4gJD\\\\\\\/m0WSN8h09oU2lpYNzusDlM+RV8l+KWG\\\\\\\/P9mmPrbKEgefh181y0sv16ruLYvllL+xlNRMyJPwc7Fd2jBh+HwknIrQrj\\\\\\\/QjXjPMF2e0oSL9fIq9u+3YcqutDKgUrHAzF3azIGT8+oBQmSC1uFbZ8anJu+541f\\\\\\\/fHHamnzbH6TNDqKkVd4\\\\\\\/hioqn+Fn9U9GIdZHlnFxjmomR84E4S7kAS61+WT5Jh\\\\\\\/\\\\\\\/Cfm1oX2mb\\\\\\\/lVJISWRlRtFcD+PQswxRjdjVADC5mdPLQG7+JPhhtioo+nukMgpyME8IbfWpYUi9OM3jbtC0wLlTBzV0Y8xftkJpTxhCZTOEwLfZG3S4wBXmBT0rZi9uEVflZk34Hj15pv42gIJubAFz1G4ANItUulbcMv3A+qtkC+7ybu+jvCXvRPg3Tv1Eof58xEtPpijrhK2xbBQBgEVzDBL6pjbYx6tipD4jOmRnmoDMLJDFNmGRdRobE5t8XWk3qH+FzRD\\\\\\\/FQHkP0z3o+dbPQxe9+fFiaM2smInWPiHMGGymYATY3Lzig\\\\\\\/5LZPxKSMrgs2lmDMGTbS2PuLEak+gUfswOo8Fzpa9zBrS6n28UgFLUFt0LGODpyjfIOahNageBVWI27bmtO9nnVjZ+tqchzS4rCb9NnQpEXCfTY+GkvLd0hxhI++8h2oiMmS\\\\\\\/8aSrnn3p0pFonkVxrW0vBG6FR27WDIQQowfx7rxUWFUE3f57Nz9mUU1KhNu71szk8w\\\\\\\/TVrhfZo\\\\\\\/Y219Bz18y+kC6KLQpGD4bSGumEIjRxrszhK1wtuxp5tqqK+wLm6VMMkNZdg\\\\\\\/HeD\\\\\\\/N7DGdidp4fPanvaOrqnwCUGQB5oX1Ys\\\\\\\/sKRpadCVj4YjuFZcOnXUQSst\\\\\\\/rJzPCynYhPK\\\\\\\/EDLi3XSoaLOC6uUgN4yoqGnYYEF9QNJj6956MjwSENgKn8YNQzYsgvv\\\\\\\/MYrgArE+BHEvwqwUDkA1s91ZWGUdz0Dz5flnlrskPyXNOyuP4KOZf\\\\\\\/DXAqUUb7i1C+7cIoiRiHZP9jWJYZLUtEAemcORGPLBiN\\\\\\\/uMbj556SrOGzLFW9kjTexKgUzGg9qD4r3r+GkZg1WZtVSMq6X10CG2erQ\\\\\\\/bxT3Teil2yd4akoEVtZNzCvwJrhOdgIxH\\\\\\\/dWobvyVTJrS9JpiVsO89eyyCTaAxj3e8O7b+AkLZW24E295BStwiJP5I47GIugAaZ6CWPw4IJHkbsMvkK6ctYOteoeUNDYt0fVrcvroqu6KhypNqHEAn8SYBd3jlpTD7jV0t64MT\\\\\\\/ks\\\\\\\/8pSpUu510K67S2fXb9Q\\\\\\\/KxovdE3naspAK4l0yqhDdmLNxEnFqqyGI7t+\\\\\\\/51\\\\\\\/C3Hwhi0MXaIaVDw2gVgMkwQ\\\\\\\/59HmhpNeAyaT7XX+XwUMnU9GO\\\\\\\/ZmjHk+gQinevpXfzI357hNTsfDZ3L\\\\\\\/Q2448jdWpa4w8yPSbWxo5E2EtgPSz0lLIFJU89VcaUTzQ8UVAyUBcIN+D6yP6BgyW9rWNW1F12QXjlVb39jLI3CXohbhRkHTRFA3Us9ryZ7xyGSHyK6MXWEuY17UeHim1qxFBAHA12XkQb9gBXJwZQXdztPjDgujN1DyJRndFTS+oy0eKmN\\\\\\\/5trs3CBgUzzxoKyeLKmNBFbeY135aQcgX8oSjgpEBEFOr1Q5TNzfjEHq2zceHJ9gLZpNklb1NTWm0Z\\\\\\\/KVEAlunaamnLnAlnHng11zJa0p16B16o+0JrF3h1dKYKKg16IaD+cxEssV98giBm8\\\\\\\/w+HoQ0xuD+dqw0pQ4pQa3oYKgdbUN+teNLZ7nvC+BQjFfFgYsoseVEva7ZDuzsOykMYVhptd8fTzRr5bhyaJZndEWpWPoDwDgOG3f1+RVsYiwDqpcvZKzZhATzHYRSsX9sD5APs\\\\\\\/nAPBNj8m4Ej9wEe\\\\\\\/GZyUT+vLHcytnWGhCExd7XjQL1sEHVESlMMNZAZjPVILaMCfepYllQZSsGcV45r15PWr0TtLhsy2dZXhXig+ZbSg7QWylZky1Wiaequqw2J8hEaOq4kQzTVswcN3kzdNzbCM2STaVpvr5jbpUx+srnLnL7GR9MI7En174Kc9TWLl6npXgUjGD8QevW9OwKI0SUq5nfjC3mdP5WkZHylVf9tJexbdndDY+vvJkebFQVKhU6EouG4VJH+wY9ZZG3cQdwwLyjrTtOPUH60SPM3p3YEjr3qImCW+QmRRkxmHTKmtYyu5TalPZ+2Pba6MsDncveChPryp92RLq7v+yvmKeFxiOnPjcN3QZGemtR36adJwNJgiOIslN\\\\\\\/hMprmbxeCtKTDzxqFvSVxjQMM1gX0Q46\\\\\\\/0U1MMSRfDi7X9Fsyo\\\\\\\/jJ7ZBcL2sVLpdfv0xqJ7PrtvqIbpgZklQrpfQbVHrChacJ25MsSq65N48y6mgFk4ij7vZJlG6yL5vC8uivF8wJxYLeBI1nAF0HYBl4mWQHxbT6YsjvJJHiGuiSLfc0Dm+0HBB\\\\\\\/7RD5zHVEiL4fNsnkQjcia0KZ9hccG782g3ke+\\\\\\\/0lWcXoRKgy1MU1RxAFwkhx6eRJ3im42kCRDp0G5\\\\\\\/IHvLcBCYq3CghVkO1VqwRVpy+exHHHRxlv2kaL1n0\\\\\\\/nadJyvPQgjWtl8C\\\\\\\/g9ny6w2Eiz5EqfZqKB2PQnG1P8DyHiZpmHi30e3bDiAzRHilgYJaOjPC2wPnDi\\\\\\\/Y0x9yTAzeR7qWz1JGbVv0fsFS1ZTSmuC6Il\\\\\\\/YSRTy0bjcUm4Y+MIEtKuX1LRX\\\\\\\/CxOe8MJux7dN+8Ymvb9aM5cWjfQB5s4QQfrEha6QkBN879o\\\\\\\/W13R6Bok00NNuxrBFCfwpMv57e8IaZv4MH6op8D1de1SqpPykxolEixfIrTth76lDYopBxpFs3Z9iq3ogi8ZbXn\\\\\\\/kImOTHLmeDGHxtI1QCeC3QoKe9NgcHaiCSfpJKMV4JqsliWDbVlOZbtiB5QDF2tBtrLXW+pslhJpWVC8XjanGRLp3vx6+9yxYQBOew\\\\\\\/jMKPkhJqYUJqUIZQR7PeJ4jBh7esHHBHNx8M9ANgWKJkiLViUKhnMicdG1PBuLYMpWQFxqIYOM9MZeds52a3fbNHUF9KuDc5K6vHqXyx0h3iJ6XuDBU+9TcahdInZUqYOgR08CplmefJnUTE2aAWBDUkReOSfIn92P3x3TZD4Gn8SHDSTdKxNxM2dgsuxm93sT54YY0IGZsmctcLe5u5OSqfxG3HY1g14s52o7H3fzolHkiL+SSu8DIBnta9k9kWMeyeLvlY2SErCQ5aM3N5A53o\\\\\\\/5gRf3Kth9TXQk99KTVPSBQkGx\\\\\\\/tZ35UviaweR4RdjzdSlxeSvmhMB24RphJQNdmJTPteqtdULx6W3mQ2mKwpjRJsHiC3L1L4l\\\\\\\/UXbJv9UbzNn\\\\\\\/9i0daD5UDEk480Gr2zaYm6AZa1PGLkVuoE\\\\\\\/y6exdLXEmYbggZIYq8vKGLRi0RMwPKNt1MddVZvNeYxBEWmBR1\\\\\\\/Rg1gpe+FLgTEvf8YsH4BOgwAlMuK8kfYItKHiv\\\\\\\/oxPv4T1hcFZQ+JCQhnaK0hUcZkeSyWTWZLHS57pxy2IuEJda1ZShVx+6N7VJss\\\\\\\/\\\\\\\/QP46dZzJKNhfdXKwc08oRnutc0+CafbFPP\\\\\\\/4OVSSl2lwoaGjY1GNEVJ69lcbc7sxrVIxu+EZl1hKyh3HcqsLVmU66zoBFULiEwVMjd3PklpunQtLsCSdWFQ8HCFh1Fhi2b4XEkoWz12W1646RfeuPBB1jwvj+unACFAxAgRvoQhDzX6mWn3EYAvLp495iafFgHwJgddaDETbwykCV6M\\\\\\\/WdLLq57iLRfnqfSad58\\\u0022,\\\u0022iv\\\u0022:\\\u00227v\\\\\\\/8\\\\\\\/6oUoR8wqWdh4TvnTw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00227072cc9bc4c622787aa5fb0e5bdd9043ec94f69704cbd2c44365aa7b5e171ee5\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022W\\\\\\\/Uwu\\\\\\\/l7v+MyY3w841Z0cCoQzfnpAUsb44gFnElIXY03MNDBUTqaG1g0nlLO+hpEaQnlT9IofKTA2i14Q5GVbKcZPoKMVakXA8yc6N1CTMT287hWP83UTn02LrbsRU0ksujQe27h3z45gm9AgjO9jNc12nr+CG8VGlvy9ekbef+hCd7rS\\\\\\\/yeEzFDAt8J7nU\\\\\\\/l3O5odwiIhfYJTua+20QYFw2tlArgw98AIKhmhbCcqk3K100CnsFyPK50cyDcdvxgV3AbqL\\\\\\\/YwyVMp4Y+l5haHgG6NBMER1GmCP79wAxP4IwTNkPWSLlE5Pk1E9gzh9NWJvZT5J4ABMG8zBeZhtMGssFHwPdo2lcDFh9k13qnyYNG98sixmuCd1fCX\\\\\\\/wyzGkmAT2EmjpbjLH8JbS+FK\\\\\\\/H4B11fZgNtEpRBlzjntEFh2VjYec0hQluFBHSyEZ4NckKwJcl7kFvEji\\\\\\\/aCI1z62xP1vA0Z4XpoHsKG5i33hZJIjZKhYwEOWTvFhq+tN7dVD1RACBP4KZ1XkFqzgTvXQ9MrGxco64+ifQ5l7\\\\\\\/urukji7y\\\\\\\/vtp8V6hxVD781qMl1Kd4LNvJHehosQ1KjUwn7pCD2lTl2pgfNLnB2asukAyl\\\\\\\/cgiqGuqDqQJ5ITrB8AC+Blhwg8dMojam8TU00tHle1PO43OUimlyhVvoHFlSMLVl\\\\\\\/gmNSEiYx08vu9WvZxmhjEzsQSuMfK0iK5ficboB4THrXLUsMv05a0GnDTX\\\\\\\/+zb6rncO+eoEHaSbxOGJhTDANfR6G7ZsP9\\\\\\\/AJfGpxD72O9ZhMX6J5hjWdmmTJW+KN1fJnBX2lHOeyObyvMx3BA0zqjLMjUE\\\\\\\/3v5oKnh4g2j51JSrbtw99UHSXjW0Zb0nCjSe4h4B2A1uzvLdNhu9ZYZMPoksJbfWgoOh\\\\\\\/Iv+yYYBSu+6Ogwh6kCXxvY1m4g6tCY+2f1EbCYnpWUoEJzaNDuDTXlqIp5\\\\\\\/ybVzrM4X4\\\\\\\/OyNT15M1MRrw1pdKMukIihtaPipCgvqiu0XML8KhiZ7uw4vu+PP7GquhQm\\\\\\\/XJYXhIqSaZ7MU3eEpsTUp2pZNYxraNEVoGe9NqbDj1IXJ3sgTOIeb1vo6nO8hNJWvE28N4KZOJOlqczKR+IaDxLSzuR4C0L5u1csxXf6W8RUQ4oWXFHMbhj0GS0I0xw5NcVGEEHrug1M+HGM0GjfH4A\\\\\\\/hZLElHYbwZrSHHIdPVXnhteRLcfz0cVSQY6RXIiuKIYQg9FtJWIZj8Opvbb4Ds4iKR3cBaIn6LvUIwj2hsYUPED6U+q3Un0D\\\\\\\/uobzapyMBOM0M6lGPtS0yS7GpzhWw+O6snlCKv1J6lAzTh9OzOpEg\\\\\\\/zXcbFcFBwRp4n+9pLpNRhqFF8SJPVQbURpYfcjrKkDg8AomY4f+YpbKmAcU19pv21WL1ZjYtDPmm6OKdKR2+fQSt7mnIRPAqrl23shufuCWo6vUOuPEMXJOACKMoqnQXT2VS5j2Ahpd65+UO7skN4N5LB+PDEqvao91YXBVrg6bTPPJ0Hrxj6j3UadTcOsNgKaHfc33\\\\\\\/J+Dowgw5RKIoYQ0Ls8E5bb\\\\\\\/L0iNU9sTPieFwl+xmV0Y9wofcQ8lwHj310BpMh4GGIpaBfE3mytBNoSQh8a+7eCFMSLLqZQTAxBuLCojF1swTI8QsiSb07Ly6J0jAA9a7SD8NY9XAtkb\\\\\\\/k5ani87rqfrSKE\\\\\\\/L9xtDTaTQWiDzCsdXyuRKrMvcNhCIQbp\\\\\\\/3K\\\\\\\/IVVmaTgnfjjFaJlwb78yZRA63TxAY7vWENf\\\\\\\/I4qL0BAmrB5M0OY7FxakJNjfv69MDFSAbFCcGQ\\\\\\\/wHAytfBjUw8QjKvF7lDnjyYLqNo3u0EJq0x89S0h7jrWtncKpyNv+oXLmDNGROzUWX\\\\\\\/Py0Bneql2XnUhUh2c+vQGNiFYtPVHKjx8qyxaTWxh2aVSPGpTTPrqcXIwQq+YQCZpnH3NBy9qnvJWL0viNCi+SkvrDxExrcGFIR+0AB00hZxE03muA9mjPJHi9bLCJb8SERwJkOp6eHjmGusG7yfzW1gNjpwRHWr3Rt07MCIibfUdjJt\\\u0022,\\\u0022iv\\\u0022:\\\u0022MFBRWOY2FX5pWnCae+BZpQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00228419e108ef02c4ca1b7e4a3405a11d9117b9398247554d7f433f9202037521de\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-494277 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/10/3139" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-RsZ7lfbesH_SFOJjTbsaZCHVTBROySNLMA6wMB7DcXk" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/10/3139" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-3Fa6AA8l12kmaJrq6hihzBFu1c0iEP83EwhHTW5WMng" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/10/3129" title="QTG-Finder: A Machine-Learning Based Algorithm To Prioritize Causal Genes of Quantitative Trait Loci in Arabidopsis and Rice" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/10/3153" title="Multi-trait Improvement by Predicting Genetic Correlations in Breeding Crosses" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494277" id="node494277" data-pisa="ggg;9/10/3139" data-pisa-master="ggg;g3.119.400347" data-apath="/ggg/9/10/3139.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genome-Wide Association and Gene Co-expression Network Analyses Reveal Complex Genetics of Resistance to Goss’s Wilt of Maize</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Amritpal Singh</span>, <span class="highwire-citation-author" data-delta="1">Guangyong Li</span>, <span class="highwire-citation-author" data-delta="2">Alex B. Brohammer</span>, <span class="highwire-citation-author" data-delta="3">Diego Jarquin</span>, <span class="highwire-citation-author" data-delta="4">Candice N. Hirsch</span>, <span class="highwire-citation-author" data-delta="5">James R. Alfano</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0002-4361-1683" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Aaron J. Lorenz</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3139-3152;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400347 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494277" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Amritpal Singh </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy and Plant Genetics, University of Minnesota, St. Paul, MN 55108</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Amritpal%2BSingh%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Singh%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAmritpal%2BSingh%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Guangyong Li </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Plant Pathology, University of Nebraska, Lincoln, NE 68503-0722</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Center for Plant Science Innovation, University of Nebraska, Lincoln, NE 68588-0660, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Guangyong%2BLi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Li%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGuangyong%2BLi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Alex B. Brohammer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy and Plant Genetics, University of Minnesota, St. Paul, MN 55108</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alex%2BB.%2BBrohammer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Brohammer%20AB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlex%2BB.%2BBrohammer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Diego Jarquin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Agronomy and Horticulture, University of Nebraska, Lincoln, NE 68503-0915</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Diego%2BJarquin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jarquin%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADiego%2BJarquin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Candice N. Hirsch </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy and Plant Genetics, University of Minnesota, St. Paul, MN 55108</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Candice%2BN.%2BHirsch%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hirsch%20CN&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACandice%2BN.%2BHirsch%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">James R. Alfano </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Plant Pathology, University of Nebraska, Lincoln, NE 68503-0722</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Center for Plant Science Innovation, University of Nebraska, Lincoln, NE 68588-0660, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=James%2BR.%2BAlfano%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Alfano%20JR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJames%2BR.%2BAlfano%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Aaron J. Lorenz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy and Plant Genetics, University of Minnesota, St. Paul, MN 55108</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Aaron%2BJ.%2BLorenz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lorenz%20AJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAaron%2BJ.%2BLorenz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-4361-1683" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Aaron J. Lorenz</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:lore0149@umn.edu" class="" data-icon-position="" data-hide-link-title="0">lore0149@umn.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/9/10/3139" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:494277" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:494277/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/10/3139.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:494277" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:494277/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/10/3139.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:494277" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:494277/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/9/10/3139.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:494277" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:494277/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Goss’s bacterial wilt and leaf blight is a disease of maize caused by the gram positive bacterium <em>Clavibacter michiganensis</em> subsp. <em>nebraskensis</em> (<em>Cmn</em>). First discovered in Nebraska, Goss’s wilt has now spread to major maize growing states in the United States and three provinces in Canada. Previous studies conducted using elite maize inbred lines and their hybrids have shown that resistance to Goss’s wilt is a quantitative trait. The objective of this study was to further our understanding of the genetic basis of resistance to Goss’s wilt by using a combined approach of genome-wide association mapping and gene co-expression network analysis. Genome-wide association analysis was accomplished using a diversity panel consisting of 555 maize inbred lines and a set of 450 recombinant inbred lines (RILs) from three bi-parental mapping populations, providing the most comprehensive screening of Goss’s wilt resistance to date. Three SNPs in the diversity panel and 10 SNPs in the combined dataset, including the diversity panel and RILs, were found to be significantly associated with Goss’s wilt resistance. Each significant SNP explained 1–5% of the phenotypic variation for Goss’s wilt (total of 8–11%). To augment the results of genome-wide association mapping and help identify candidate genes, a time course RNA sequencing experiment was conducted using resistant (N551) and susceptible (B14A) maize inbred lines. Gene co-expression network analysis of this time course experiment identified one module of 141 correlated genes that showed differential regulation in response to <em>Cmn</em> inoculations in both resistant and susceptible lines. SNPs inside and flanking these genes explained 13.3% of the phenotypic variation. Among 1,000 random samples of genes, only 8% of samples explained more phenotypic variance for Goss’s wilt resistance than those implicated by the co-expression network analysis. While a statistically significant enrichment was not observed (<em>P</em> &lt; 0.05), these results suggest a possible role for these genes in quantitative resistance at the field level and warrant more research on combining gene co-expression network analysis with quantitative genetic analyses to dissect complex disease resistance traits. The results of the GWAS and co-expression analysis both support the complex nature of resistance to this important disease of maize.</p></div><ul class="kwd-group"><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/maize-zea-mays" class="hw-term hw-article-keyword hw-article-keyword-maize-zea-mays" rel="nofollow">Maize (<em xmlns:default="http://www.w3.org/1999/xhtml">Zea mays</em>)</a></li><li class="kwd"><a href="/keyword/genome-wide-association-mapping" class="hw-term hw-article-keyword hw-article-keyword-genome-wide-association-mapping" rel="nofollow">genome-wide association mapping</a></li><li class="kwd"><a href="/keyword/quantitative-disease-resistance" class="hw-term hw-article-keyword hw-article-keyword-quantitative-disease-resistance" rel="nofollow">quantitative disease resistance</a></li><li class="kwd"><a href="/keyword/goss%E2%80%99s-wilt" class="hw-term hw-article-keyword hw-article-keyword-goss&#x2019;s-wilt" rel="nofollow">Goss’s wilt</a></li><li class="kwd"><a href="/keyword/weighted-gene-co-expression-network-analysis" class="hw-term hw-article-keyword hw-article-keyword-weighted-gene-co-expression-network-analysis" rel="nofollow">weighted gene co-expression network analysis</a></li><li class="kwd"><a href="/keyword/clavibacter" class="hw-term hw-article-keyword hw-article-keyword-clavibacter" rel="nofollow">Clavibacter</a></li><li class="kwd"><a href="/keyword/quantitative-trait-loci" class="hw-term hw-article-keyword hw-article-keyword-quantitative-trait-loci" rel="nofollow">quantitative trait loci</a></li><li class="kwd"><a href="/keyword/genotyping-sequencing" class="hw-term hw-article-keyword hw-article-keyword-genotyping-by-sequencing" rel="nofollow">genotyping-by-sequencing</a></li><li class="kwd"><a href="/keyword/single-nucleotide-polymorphism" class="hw-term hw-article-keyword hw-article-keyword-single-nucleotide-polymorphism" rel="nofollow">single nucleotide polymorphism</a></li></ul><p id="p-3">Goss’s bacterial wilt and leaf blight of maize, caused by a gram positive bacterium <em>Clavibacter michiganensis</em> subsp. <em>nebraskensis</em> (<em>Cmn</em>), can dramatically reduce grain yield of maize if infection becomes severe enough. Studies using artificial inoculations of <em>Cmn</em> conducted during the 1990s showed yield losses up to 44% (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Carson and Wicks 1991</a>), and yield losses of 50% or more have been reported by producers (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Robertson 2012</a>). Goss’ wilt was first discovered on one farm in southcentral Nebraska in 1969 (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Schuster <em>et al.</em> 1972</a>). Over the next decade, Goss’s wilt spread to 58 counties in Nebraska and 34 counties across seven states, becoming a major concern for maize producers (<a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Wysong <em>et al.</em> 1981</a>). The incorporation of host plant resistance into maize hybrids greatly reduced the incidence of Goss’s wilt, with it being only sporadically observed in the far western Corn Belt throughout the late 1980s until the mid-2000s (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Jackson <em>et al.</em> 2007</a>). Goss’s wilt suddenly re-emerged around 2006 as indicated by a large increase in the number of samples diagnosed with Goss’s wilt submitted to plant disease diagnostic clinics (<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Jackson <em>et al.</em> 2007</a>). Since 2006, Goss’s wilt has continued to spread throughout North American maize growing regions, having been reported in Nebraska, Iowa, Colorado, Missouri, Indiana, Illinois, Kansas, Minnesota, North Dakota, South Dakota, Wisconsin, Texas, and Louisiana. In Canada it has been confirmed in Alberta, Manitoba, and Ontario (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Ruhl <em>et al.</em> 2009</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Malvick <em>et al.</em> 2010</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Korus <em>et al.</em> 2011</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">EPPO 2014</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Friskop <em>et al.</em> 2014</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Howard <em>et al.</em> 2015</a>; <a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Singh <em>et al.</em> 2015</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Hosack <em>et al.</em> 2016</a>). In recent years Goss’s wilt has ranked among major diseases of maize in North America in terms of estimated yield losses, with estimates ranging from 38.5 million bushels in 2012 to 215.9 million bushels in 2014 in major maize producing U.S. states and Canada (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Mueller and Wise 2012</a>, <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">2014</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Mueller <em>et al.</em> 2016</a>).</p><p id="p-4">Identifying sources of resistance to Goss’s wilt and breeding resistance into maize hybrids remains a viable strategy to reduce yield losses. A great degree of variation in the level of resistance to Goss’s wilt exists in maize. Maize inbred lines have shown variable response to Goss’s wilt varying from resistant, intermediate, to highly susceptible based on screenings in the 1970s and 1980s with a limited number of inbred lines (<a id="xref-ref-58-2" class="xref-bibr" href="#ref-58">Schuster <em>et al.</em> 1972</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Calub <em>et al.</em> 1974</a>; <a id="xref-ref-68-2" class="xref-bibr" href="#ref-68">Wysong <em>et al.</em> 1981</a>, <a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Treat <em>et al.</em> 1990</a>). For example, B14 and its derived lines, such as A619, have been found to be generally susceptible; Oh43 was reported to be moderately susceptible; and Mo17 was reported to be resistant (<a id="xref-ref-58-3" class="xref-bibr" href="#ref-58">Schuster <em>et al.</em> 1972</a>; <a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Calub <em>et al.</em> 1974</a>). Studies using classical mating designs, including diallels and generation means analyses, have indicated that resistance to Goss’s wilt is under polygenic control, with additive genetic variation being the predominant form of genetic variation (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Gardner and Schuster 1974</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Martin <em>et al.</em> 1975</a>; <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Rocheford <em>et al.</em> 1989</a>; <a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Treat and Tracy 1990</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Ngong-Nassah <em>et al.</em> 1992</a>).</p><p id="p-5">Efforts to identify molecular markers linked to resistance to help unravel the genetic architecture of Goss’s wilt have been lacking until recently because the incidence of this disease was only sporadic during the time plant geneticists and breeders were developing and adopting molecular markers, QTL mapping, and marker-assisted selection (circa late 1980s to 2000s). The re-emergence of Goss’s wilt has increased interest in identifying sources of resistance, molecular markers that could be used for selection, and genes controlling resistance. Using three bi-parental maize populations connected by a common parent, <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Singh <em>et al.</em> (2016)</a> identified eleven QTL of small effect, half of which were population specific. <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Schaefer and Bernardo (2013)</a> performed genome-wide association mapping for Goss’s wilt and identified several SNPs associated with resistance, each explaining no more than 10% of the phenotypic variation. The diversity panel used by <a id="xref-ref-57-2" class="xref-bibr" href="#ref-57">Schaefer and Bernardo (2013)</a>, however, was limited in size and diversity, being comprised of only elite maize inbred lines. This limited diversity, combined with relatively few molecular markers, lowered the mapping resolution achieved by this study. Most recently, <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Cooper <em>et al.</em> (2018)</a> used the Intermated B73 X Mo17 mapping population and related introgression lines to map QTL controlling Goss’s wilt resistance. These authors detected several of the previously mapped QTL, including a QTL on chromosome 1 that overlapped with a locus known to be important in resistance to multiple diseases of maize (<a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Cooper <em>et al.</em> 2018</a>).</p><p id="p-6">Transcriptomic approaches, such as RNA-seq, can help identify resistance candidate genes by quantifying changes in gene expression levels after resistance and susceptible plants have been exposed to a pathogen. Several studies, however most of them involving fungal pathogens of maize, have investigated the response of maize to its pathogens using transcriptomics (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Lanubile <em>et al.</em> 2014</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Liu <em>et al.</em> 2015</a>, <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">2016</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Miranda <em>et al.</em> 2017</a>). From these studies, several genes have been reported to be differentially expressed in resistant and susceptible lines in response to <em>Fusarium</em> and <em>Bipolaris</em> species with enrichment in functions including pathogen recognition and signaling, growth and development, plant hormone signal transduction, and defense (<a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Liu <em>et al.</em> 2016</a>; <a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Lanubile <em>et al.</em> 2014</a>; <a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Liu <em>et al.</em> 2015</a>). Study of gene expression changes in near-isogenic lines for disease resistance QTL to <em>Fusarium</em> further revealed two different mechanism of resistance provided by the QTL, where one QTL conferred resistance through the expression of defense related genes and the other imparted resistance by affecting auxin signaling and transport (<a id="xref-ref-35-3" class="xref-bibr" href="#ref-35">Liu <em>et al.</em> 2016</a>). Systemic acquired resistance in maize to <em>Colletotrichum</em> was found to be controlled by genes involved in salicylic acid pathway and chromatin modification (<a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Miranda <em>et al.</em> 2017</a>).</p><p id="p-7">Recently published transcription profiling of resistant and susceptible maize genotypes (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Doehlemann <em>et al.</em> 2008</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Meyer <em>et al.</em> 2017</a>; <a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Yu <em>et al.</em> 2018</a>), expression QTL analysis (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Christie <em>et al.</em> 2017</a>), and meta-analysis of expression profiling studies (<a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Wang <em>et al.</em> 2018</a>) for <em>Cercospora zeina</em>, <em>C. zeae-maydis</em>, and <em>Ustilago maydis</em> pathogens also revealed complex defense response and molecular interaction with maize. Different approaches revealed changes in hormonal signaling such as jasmonic acid, and salicylic acid, as well as oxidative reduction in response to pathogens. Studies also pointed toward implication of genes involved in primary defense such as leucine rich repeat receptor like kinases, PAMPs, flavonoid and terpenoid biosynthesis genes, calmodulin-like proteins, and genes encoding pathogenesis related proteins.</p><p id="p-8">Combing GWAS and transcriptomic analyses provides a powerful systems approach to characterize the genetic basis of resistance to this pathogen of maize. The objectives of this study were: 1) Characterize the genetic variation for Goss’s wilt within a large panel of diverse inbred lines; 2) Discover genomic regions and candidate genes controlling variation for resistance to Goss’s wilt; 3) Discover co-expressed modules of genes that show changes in gene expression patterns between a resistant and susceptible maize line in response to inoculation with <em>Cmn</em>, and compare these genes to those implicated by GWAS; 4) Quantify the amount of variation in field resistance explained by differentially expressed gene modules.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Germplasm and selection of diversity panel for genome-wide association analysis</h3><p id="p-9">Two sets of germplasm were used for the GWAS: a diversity panel of 555 inbred lines and three bi-parental linkage mapping populations selected from the maize nested association mapping (NAM) population. The diversity panel of 555 maize inbred lines was selected from a larger set of 2,815 maize inbred lines genotyped by <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Romay <em>et al.</em> (2013)</a>. This panel consisted of lines from Stiff Stalk Synthetic (SSS), non-SSS, tropical, popcorn, and sweet corn genetic backgrounds. Subpopulation classification (<em>i.e.</em>, SSS, non-SSS, popcorn) of each line was according to <a id="xref-ref-52-2" class="xref-bibr" href="#ref-52">Romay <em>et al.</em> (2013)</a>. The first year of Goss’s wilt resistance evaluation included 400 inbred lines selected from the total set of 2,815 inbred lines. To arrive at this set of 400, the initial set was first reduced to 900 lines by retaining only those that reached mid-silk within four days of B73 according to mid-silk growing degree data provided by <a id="xref-ref-52-3" class="xref-bibr" href="#ref-52">Romay <em>et al.</em> (2013)</a>. This was done to reduce confounding effects that variation in days to flowering could have on disease ratings. A <em>k</em>-means clustering analysis was then applied to SNP data to classify the 900 lines into 400 clusters, and one line from each of the 400 clusters was randomly selected. Seed of the 400 inbred lines was obtained from the North Central Regional Plant Introduction Station (NCRPIS) in Ames, IA and increased during the summer of 2014 in the breeding nursery through self-pollination. Insufficient statistical power to detect associations in the first year of this study prompted an increase in the panel size for evaluations in 2015. The panel evaluated in 2015 consisted of 555 lines, which was formed by the addition of 155 inbred lines from the set of 900 for which adequate seed quantities were immediately available. No specific criteria were applied to selection of these additional 155 lines beyond immediate seed availability.</p><p id="p-10">The three bi-parental RIL populations selected from the NAM population were B73 x Oh43, B73 x HP301, and B73 x P39. These same populations were used in an earlier study to identify QTL for Goss’s wilt through linkage mapping (<a id="xref-ref-60-2" class="xref-bibr" href="#ref-60">Singh <em>et al.</em> 2016</a>). While the common parent, B73, is moderately resistant, the other three parents were found to be comparatively susceptible to Goss’s wilt in a preliminary screening. Moreover, Oh43, HP301, and P39 are dent, popcorn, and sweetcorn types, respectively, which allowed for the discovery of Goss’s wilt resistance alleles from three distinct genetic backgrounds. In 2012, 195 RILs from the B73 x Oh43 population were screened for Goss’s wilt. In 2013, 172 RILs from the B73 x Oh43 population, 141 RILs from the B73 x HP301 population, and 125 RILs from the B73 x P39 population were evaluated. In 2014, 174 RILs from the B73 x Oh43 population, 143 RILs from the B73 x HP301 population, and 124 RILs from the B73 x P39 population were phenotyped for Goss’s wilt. The lines were evaluated using a completely randomized field design with replicated checks as described earlier (<a id="xref-ref-60-3" class="xref-bibr" href="#ref-60">Singh <em>et al.</em> 2016</a>).</p></div><div id="sec-3" class="subsection"><h3>Genotypic data</h3><p id="p-11">ZeaGBSv1.0 dataset consisting of 681,257 GBS SNPs published by <a id="xref-ref-52-4" class="xref-bibr" href="#ref-52">Romay <em>et al.</em> (2013)</a> downloaded from <a href="http://panzea.org">panzea.org</a> in 2013 was used to select the diversity panel. In 2014, ZeaGBSv2.7 version dataset of 955,690 GBS SNPs was made available for maize germplasm public germplasm including NAM RILs at <a href="http://panzea.org">panzea.org</a>, which was then downloaded and used for the genome-wide association analysis. Briefly, the GBS data made available by <a id="xref-ref-52-5" class="xref-bibr" href="#ref-52">Romay <em>et al.</em> (2013)</a> was obtained using the <em>ApeKI</em> restriction enzyme as previously described (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Elshire <em>et al.</em> 2011</a>). A reference genome-based GBS pipeline in TASSEL 5.0 was used for SNP discovery with standard parameters as applied to maize Discovery Build was used to call the SNPs (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Glaubitz <em>et al.</em> 2014</a>). The imputed version of the ZeaGBSv2.7 dataset, imputed using Fast Inbred Line Library ImputatioN (FILLIN) method, was used (<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Swarts <em>et al.</em> 2014</a>).</p><p id="p-12">Markers with more than 80% missing data and a minor allele frequency (MAF) less than 0.05 were filtered out, leaving 342,237 SNPs for analysis. This filtered and imputed SNP dataset was used for principal component analysis (PCA), LD decay, and GWAS analyses.</p></div><div id="sec-4" class="subsection"><h3>Diversity panel characterization</h3><p id="p-13">Population structure within the diversity panel was visually assessed using PCA and ADMIXTURE, a software for model-based estimation of ancestry (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Alexander <em>et al.</em> 2009</a>). A cross validation procedure implemented in ADMIXTURE was used to initially choose the optimum number of subpopulations (K) by minimizing the cross validation error. Multiple runs of ADMIXTURE were conducted at different values of K ranging from 3 to 20. Twenty replications were performed for each value of K. Cluster memberships for each replicate were aligned using the cluster matching software CLUMPP (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Jakobsson and Rosenberg 2007</a>). A plot of cross validation error <em>vs.</em> K was examined, but an exact optimum could not be determined (Supplementary Figure 1). K was set to three based on visual inspection of a PCA plot of PC1 <em>vs.</em> PC2 (Supplementary Figure 2a), subpopulation membership plots from ADMIXTURE differing in K (Supplementary Figure 2b), and subpopulation information from <a id="xref-ref-52-6" class="xref-bibr" href="#ref-52">Romay <em>et al.</em> (2013)</a>. The average cluster membership across twenty replications of the ADMIXTURE analysis was used as a covariate in the GWAS model to account for population structure.</p><p id="p-14">Decay of linkage disequilibrium (LD) was assessed as pairwise R<sup>2</sup> between the SNP markers within 10 kb windows using PLINK v1.07 available at <a href="http://zzz.bwh.harvard.edu/plink/">http://zzz.bwh.harvard.edu/plink/</a> (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Purcell <em>et al.</em> 2007</a>). To assess the relationship among the lines of the diversity panel and visualize the clustering of inbred lines according to Goss’s wilt resistance, a Hamming distance matrix was created using PLINK in which the distance was calculated as 1-IBS, where IBS is the identity-by-state coefficient. The distance matrix was used to create a neighbor joining tree using the method of <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Saitou and Nei (1987)</a> as implemented in <em>ape</em> package in R (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Paradis <em>et al.</em> 2004</a>).</p></div><div id="sec-5" class="subsection"><h3>Goss’s wilt phenotyping and disease nursery</h3><p id="p-15">In 2014, the diversity panel of 400 inbred lines was planted in a Goss’s wilt nursery at the Agricultural Research and Development Center of the University of Nebraska in Mead, NE. Plots were arranged in a randomized complete block design with three replications. Susceptible line B14A, and two susceptible and two resistant proprietary check inbred lines from Dow AgroSciences were included to assess disease development. In 2015, the diversity panel of 555 lines was planted at the same location using the same experimental design as in 2014. Inoculations with <em>Cmn</em> were carried out following the same procedure as described previously (<a id="xref-ref-60-4" class="xref-bibr" href="#ref-60">Singh <em>et al.</em> 2016</a>). Briefly, wounds were created on plant leaves with motorized weed whippers and <em>Cmn</em> inoculum was sprayed within seconds of injuring the plants to ensure infection. Disease ratings were recorded 15, 30, and 45 days after inoculations (DAI). A disease rating scale of 1 to 9 on a whole plot basis was used in this study, where 1 represents complete resistance, 2 indicates disease spread less than approximately 5 cm from the point of inoculation, 3 represents limited spread but more than 5 cm from the point of inoculation, 4 indicates a large spread with lesions often extending to middle of the leaf, 5 indicates systemic infection and lesions on un-inoculated leaves, 6 indicates blight of un-inoculated leaves and wilting of plants, 7 indicates severe blight and wilt, 8 indicate severe blight and severe wilt with limited green tissue on leaves and stems of plants, and 9 represents a completely dead plot (<a id="xref-ref-60-5" class="xref-bibr" href="#ref-60">Singh <em>et al.</em> 2016</a>).</p></div><div id="sec-6" class="subsection"><h3>Analysis of phenotypic data</h3><p id="p-16">Three visual ratings taken after inoculation at 15, 30, and 45 DAI were combined to calculate WMD scores. For calculation of WMD, the average of two consecutive ratings was taken and multiplied by the number of days between the two ratings. These values were summed and divided by the total number of days spanning the first and last rating (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Balint-Kurti <em>et al.</em> 2010</a>; <a id="xref-ref-60-6" class="xref-bibr" href="#ref-60">Singh <em>et al.</em> 2016</a>). Analysis of variance (ANOVA) was conducted on WMD values using ASReml-R (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Butler <em>et al.</em> 2009</a>) by fitting the following model:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3139/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3139/embed/mml-math-1.gif"/></noscript></span><span class="disp-formula-label">[1]</span></span>Where, <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3139/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3139/embed/mml-math-2.gif"/></noscript></span></span> represents the WMD value, μ is the grand mean, <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3139/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3139/embed/mml-math-3.gif"/></noscript></span></span>is the effect of genotype <em>i</em>, <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3139/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3139/embed/mml-math-4.gif"/></noscript></span></span> is the effect for year <em>j</em>, <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3139/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3139/embed/mml-math-5.gif"/></noscript></span></span>is the effect of the <em>k</em><sup>th</sup> replication nested within the <em>j</em><sup>th</sup> year, <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3139/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3139/embed/mml-math-6.gif"/></noscript></span></span>is the interaction effect between inbred line and year, and <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3139/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3139/embed/mml-math-7.gif"/></noscript></span></span>is the residual. All effects except for the residual were treated as fixed effects. Best linear unbiased estimates (BLUEs) for WMD of the inbred lines were calculated. For estimating heritability, inbred line, inbred line-by-year interaction, and residual variances were estimated using ASReml-R by fitting these effects as IID random effects. Plot-based heritability was calculated as <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3139/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3139/embed/mml-math-8.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3139/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3139/embed/mml-math-9.gif"/></noscript></span></span> is the variance among inbred lines, <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3139/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3139/embed/mml-math-10.gif"/></noscript></span></span>is the inbred-by-year interaction variance, and <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3139/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3139/embed/mml-math-11.gif"/></noscript></span></span> is the residual variance.</p></div><div id="sec-7" class="subsection"><h3>Genome-wide association model</h3><p id="p-17">Genome-wide association mapping was performed using the following mixed linear model:<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3139/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3139/embed/mml-math-12.gif"/></noscript></span><span class="disp-formula-label">[2]</span></span>where <strong>y</strong> is a vector of WMD BLUEs of the inbred lines; β is a vector of fixed effects, including an intercept and subpopulation (K = 3) effects; <strong>X</strong> is an incidence matrix relating β to <strong>y</strong> and contains subpopulation membership probabilities output from ADMIXTURE; <em>m</em> is the fixed SNP effect of the SNP being tested; <strong>w</strong> is a vector indicating the allelic state of each inbred line for the marker being tested; <strong>u</strong> is a vector of random polygenic effects where <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3139/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3139/embed/mml-math-13.gif"/></noscript></span></span> and <strong>G</strong> is a genomic relationship matrix (GRM) calculated using the marker data; <strong>Z</strong> is a design matrix relating <strong>u</strong> to <strong>y</strong>; and <strong>e</strong> is a vector of random residuals where <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3139/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3139/embed/mml-math-14.gif"/></noscript></span></span>. Model [2] was implemented using the factored spectrally transformed linear mixed model (FaST-LMM) algorithm, including calculation of the GRM (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Lippert <em>et al.</em> 2011</a>). Parameter -fileSim was added to FaST-LMM command to obtain the GRM which uses a realized relationship matrix from SNPs to calculate the GRM.</p><p id="p-18">The linear mixed model described was also applied to the combined dataset of diversity panel (N = 555) and bi-parental populations (N = 450). The model was modified to include a fixed environmental effect to account for the different environments in which these sets of germplasm were evaluated. Also, subpopulation effects were extended to include subpopulation effects for each of the three bi-parental populations. The subpopulation effect incidence matrix, <em>X</em>, which included subpopulation memberships as described above, was extended by adding three columns for each of the bi-parental populations, where each column contained a 1 when the RIL in that row belonged to the corresponding bi-parental population and a 0 in all other columns.</p><p id="p-19">In order to declare SNPs as significantly associated with WMD in GWAS analyses, a false-discovery rate (FDR) based on a q-value of 0.1 was used (<a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Storey and Tibshirani 2003</a>). To calculate the percent variation explained by significant SNPs (<span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3139/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3139/embed/mml-math-15.gif"/></noscript></span></span>) after accounting for subpopulation effects, a multiple regression model was fit including WMD BLUEs as the dependent variable, and subpopulation effects and those of significant SNPs as independent variables (full model). A model including only subpopulation effects as independent variables was also fit (reduced model), and <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3139/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3139/embed/mml-math-16.gif"/></noscript></span></span> was calculated as the difference in variation explained between the full and reduced models.</p><p id="p-20">The total variation in WMD BLUEs within the diversity panel that could be explained by all genotyped SNPs (polygenic background effects) was calculated using a variance component approach (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Gusev <em>et al.</em> 2014</a>):<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3139/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3139/embed/mml-math-17.gif"/></noscript></span><span class="disp-formula-label">[3]</span></span>where <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3139/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3139/embed/mml-math-18.gif"/></noscript></span></span> is the BLUE for inbred lines estimating using model [1], <em>u</em> is a random polygenic background effect where <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3139/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3139/embed/mml-math-19.gif"/></noscript></span></span>), and <em>e</em> is residual. The proportion of variation explained by all genotyped SNPs was calculated as <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3139/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3139/embed/mml-math-20.gif"/></noscript></span></span> .</p></div><div id="sec-8" class="subsection"><h3>Haplotype analysis</h3><p id="p-21">Once significant SNPs were identified by the GWAS, a haplotype analysis was performed in the regions of the genome surrounding the significant SNPs using the software Haploview (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Barrett <em>et al.</em> 2005</a>). The SNPs within 10 kb of each significant SNP were included initially to conduct the haplotype block analysis. The regions were extended beyond 10 kb for haplotype block analyses if the haplotype block length in a region was greater than 10 kb. Haplotype blocks were defined according to the four gamete rule (<a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Wang <em>et al.</em> 2002</a>). A cutoff of 1% was used to define the haplotype block boundaries, meaning that if a fourth two-SNP haplotype allele was observed at a frequency of greater than 1%, recombination was assumed to have occurred between the SNPs that formed the haplotype. Allele frequencies of haplotype alleles were examined within each subpopulation of the diversity panel to determine the allele frequency differences among subpopulations.</p></div><div id="sec-9" class="subsection"><h3>Plant materials and inoculations for transcriptome profiling</h3><p id="p-22">Transcriptome profiles were obtained for B14A and N551 under both control and <em>Cmn</em> inoculated conditions. B14A was found to be susceptible and N551 was found to be resistant to <em>Cmn</em> in the field screening described above (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1c</a>). Mean field ratings of B14A and N551 were 3.4 and 1.0, respectively. B14A and N551 both belong to the stiff stalk heterotic group (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1b</a>) and are related to one another as B14 was a founder of the synthetic population from which N551 was derived (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Russell 2006</a>). The IBS, as calculated from SNPs, between N551 and B14A was 0.82 which was greater than the average IBS of 0.67 in the diversity panel.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/10/3139/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genetic relationship among the inbred lines of the diversity panel and distribution of Goss&#x2019;s wilt among the inbred lines. (a) Histogram showing the distribution of weighted mean disease BLUEs for inbred lines included in the diversity panel. Vertical dotted lines display the check lines B73 and B14A; (b) Neighbor joining tree of 555 lines of the diversity panel created from a distance matrix calculated using the GBS SNP data. Labels are colored coded by the six subpopulation groups within the diversity panel; (c) The same neighbor joining tree as in (b) but labels are color coded according to Goss&#x2019;s weighted mean disease values. Red color indicates that a line is susceptible and green color represents resistant lines. Specific groups are zoomed to indicate trends in Goss&#x2019;s wilt resistance distribution. For example, most of the B14-related lines were susceptible while Mo17-related lines were relatively resistant. N551 is present in the B14 zoomed window along with B14A." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-59671720" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Genetic relationship among the inbred lines of the diversity panel and distribution of Goss&#x2019;s wilt among the inbred lines. (a) Histogram showing the distribution of weighted mean disease BLUEs for inbred lines included in the diversity panel. Vertical dotted lines display the check lines B73 and B14A; (b) Neighbor joining tree of 555 lines of the diversity panel created from a distance matrix calculated using the GBS SNP data. Labels are colored coded by the six subpopulation groups within the diversity panel; (c) The same neighbor joining tree as in (b) but labels are color coded according to Goss&#x2019;s weighted mean disease values. Red color indicates that a line is susceptible and green color represents resistant lines. Specific groups are zoomed to indicate trends in Goss&#x2019;s wilt resistance distribution. For example, most of the B14-related lines were susceptible while Mo17-related lines were relatively resistant. N551 is present in the B14 zoomed window along with B14A.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/10/3139/F1.medium.gif" width="210" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/9/10/3139/F1.medium.gif" width="210" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/10/3139/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/10/3139/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494515" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-23" class="first-child">Genetic relationship among the inbred lines of the diversity panel and distribution of Goss’s wilt among the inbred lines. (a) Histogram showing the distribution of weighted mean disease BLUEs for inbred lines included in the diversity panel. Vertical dotted lines display the check lines B73 and B14A; (b) Neighbor joining tree of 555 lines of the diversity panel created from a distance matrix calculated using the GBS SNP data. Labels are colored coded by the six subpopulation groups within the diversity panel; (c) The same neighbor joining tree as in (b) but labels are color coded according to Goss’s weighted mean disease values. Red color indicates that a line is susceptible and green color represents resistant lines. Specific groups are zoomed to indicate trends in Goss’s wilt resistance distribution. For example, most of the B14-related lines were susceptible while Mo17-related lines were relatively resistant. N551 is present in the B14 zoomed window along with B14A.</p><div class="sb-div caption-clear"></div></div></div><p id="p-24">To perform inoculations, seeds of the inbred lines were planted in plastic inserts and placed in a greenhouse for two weeks before inoculations. Inoculations were carried out when the plants were at the V2 stage (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Abendroth <em>et al.</em> 2011</a>). For inoculations, plants were transported to the lab and were inoculated using a vacuum infiltration method with the Welch 1400 Duo Seal Vacuum Pump. Inoculum was prepared from <em>Cmn</em> isolate 12038 from the Alfano Lab (University of Nebraska-Lincoln), which was tested and determined to be virulent on maize. The bacterial cells were suspended in 10mM MgCl<sub>2</sub> for measuring concentration and then were mixed into distilled water. The inoculum bacteria concentration was set to 1 × 10<sup>8</sup> colony forming units with a spectrophotometer. To increase the surface tension of the suspension, 0.005% Tween20 was added to the inoculum. Control inoculations were done with water in place of the <em>Cmn</em> inoculum and samples were also collected from these controls at each time point. The plants were placed upside down into the inoculum and the vacuum was applied to each plant for three minutes.</p><p id="p-25">Leaf samples were collected at 0, 8, and 15 hr post inoculations (hpi). All above ground leaves were cut with sterilized scissors and immediately placed in liquid nitrogen after wrapping in aluminum foil. These time points were chosen to evaluate genes that change expression patterns early in response to <em>Cmn</em> infection and contribute to primary defense response. In a preliminary RT-PCR experiment designed to determine optimal hpi, the pathogen responsive genes PR1 and PR5 were expressed within 12 to 15 hr. A previous study in Arabidopsis found that genes involved in early defense signaling responded to elicitor flg22 as early as 30 min after treatment (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Asai <em>et al.</em> 2002</a>).</p><p id="p-26">Three biological replicates, with one plant representing a replicate, were included for each line, treatment, and hpi combination, resulting in a total of 30 samples (Supplementary Table 1). RNA was isolated using the Qiagen RNeasy mini kit (Qiagen, Valencia, CA, USA) based on the manufacturer’s protocol, and was purified using the Qiagen RNA clean-up protocol according to the manufacturer instructions.</p></div><div id="sec-10" class="subsection"><h3>Library preparation and sequencing</h3><p id="p-27">RNA samples were submitted to the University of Minnesota Genomics Center for library preparation and sequencing. RNA sizing, quantification, and purity assessments were done with an Agilent Bioanalyzer (Agilent, Santa Clara, CA). Standard 36 dual indexed TruSeq RNA libraries were created. Libraries were sequenced on the Illumina HiSeq 2500 instrument using v4 chemistry (Illumina, Inc., San Diego, CA). Single end 50 bp reads were generated from the sequencing runs.</p></div><div id="sec-11" class="subsection"><h3>Sequence quality control and transcript abundance estimates</h3><p id="p-28">Read quality was determined using FastQC version 0.11.5 (<a href="https://www.bioinformatics.babraham.ac.uk/projects/fastqc/">https://www.bioinformatics.babraham.ac.uk/projects/fastqc/</a>). High contents of Illumina Universal and TruSeq adapters were detected and these adapters were removed with CutAdapt version 1.8.1 (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Martin 2011</a>). The quality cutoff value was set to 20 and a minimum processed read length value of 20 was used in CutAdapt while processing the reads. After adapter trimming, the reads were aligned to the B73 v4 reference genome using a splice-site aware aligner TopHat2 version 2.0.13 (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Kim <em>et al.</em> 2013</a>) with a minimum intron size (-i) of 5 and a maximum intron size (-I) of 60000. The B73 v4 genome assembly was downloaded from Gramene Release 33 (<a href="http://www.gramene.org">http://www.gramene.org</a>). Read counting was performed with HTSeq version 0.5.3 (Anders <em>et al.</em> 2015<strong>)</strong> with strand specific option (-s) set to no, the feature type (-t) set to gene, the mode for handling overlapping reads (-m) set to union, and a minimum alignment quality (-a) of 20.</p></div><div id="sec-12" class="subsection"><h3>Gene co-expression network anaylsis</h3><p id="p-29">The raw expression matrix of 39,324 genes was filtered to remove genes with consistently no counts across the samples. The filtered count matrix was read into the DESeq2 R package and log2(x+1) transformed (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Love <em>et al.</em> 2014</a>). A coefficient of variation (CV) filter was applied to filter out the genes with consistently low expression across the samples in which the genes with CV less than 0.5 were removed. After applying the transformation and CV filter, the expression matrix with 9,303 genes comprisong 26 samples out of 30 sequenced, was processed with R package WGCNA to identify gene co-expression modules (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Langfelder and Horvath 2008</a>). The blockwiseConsensusModules function from the WGCNA package was used to identify modules with tree cut height 0.4 and all other options set to default. WGCNA is a systems biology approach which uses the global gene expression matrix for all the genes to group the genes that show similar expression pattern together into co-expression ‘modules’.</p><p id="p-30">Module eigen genes that were representative of the expression profile of all the genes inside each module were visualized by a heat map to find trends in expression profiles across the control and treated samples. The normalized and transformed expression matrix was centered and standardized to calculate the Z-scores and the expression profile of all the genes inside each module across the control and treated samples was plotted using parallel coordinate plots. A gene ontology (GO) analysis was performed using the AgriGO server to test if any of the modules were enriched in functions related to defense (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Du <em>et al.</em> 2010</a>).</p><p id="p-31">It was hypothesized that the gene modules responding to inoculation with <em>Cmn</em> would explain a disproportionate amount of variation in WMD as compared to a bootstrapped set of random genes. GBS SNPs from the GWAS panel that fell inside genes of such module(s) and those that flanked within one kb (determined based on genome-wide average LD decay) were used to calculate a GRM, according to [3], and the proportion of phenotypic variation in Goss’s wilt resistance explained was calculated using a variance component approach (<a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Gusev <em>et al.</em> 2014</a>). To establish a baseline for determining whether the genes implicated by the gene module(s) explain a disproportionate amount of phenotypic variance, variance component approach was applied to 1,000 random sets of an equal number of genes as in the module of interest. It is important to note that the sampling unit is the gene, and all SNPs inside each sampled gene were used to calculate the GRM, meaning that the number of SNPs could vary across gene samples. To compare this approach to an approach where the number of SNPs was well held constant rather than genes, 1000 random sets of 83 SNPs were also sampled and used to generate a null distribution of variation explained using the variance component approach.</p></div><div id="sec-13" class="subsection"><h3>Data availability</h3><p id="p-32">Phenotypic data used in this study is available at figshare GSA portal. FileS1 contains raw phenotypic data collected from 555 diversity panel lines, FileS2 contains Goss’s wilt BLUEs for diversity panel inbred lines, FileS3 contains subpopulation memberships (population structure) of 555 diversity panel lines computed using ADMIXTURE, FileS4 is contains raw phenotypic data for biparental populations, FileS5 contains Goss’s wilt BLUEs (1005 lines) used in Combined GWAS model, FileS6 contains membership assignments or population structure information used in Combined GWAS model, and FileS7 contains gene expression data used for WGCNA. Genotypic data used in this study was obtained from <a href="https://www.panzea.org/genotypes">https://www.panzea.org/genotypes</a>. Data are now available at CyVerse Data Store. CyVerse data store file path is /iplant/home/shared/panzea/genotypes/GBS/v27/. Supplemental material available at FigShare: <a href="https://doi.org/10.25387/g3.7884779">https://doi.org/10.25387/g3.7884779</a>.</p></div></div><div class="section results" id="sec-14"><h2 class="">Results</h2><div id="sec-15" class="subsection"><h3>Significant pnenotypic variability for resistance to Goss’s wilt exists among inbred lines</h3><p id="p-33">A diverse panel of 555 maize inbred lines was screened for Goss’s wilt resistance over two years, forming the largest and most diverse set of lines screened for resistance to this gram positive bacterial disease to date. Variation in flowering time was restricted to reduce the confounding effects of flowering time on resistance (see Materials and Methods). Resistance scores for each plot were taken at multiple time points, which were then summarized by calculating the “weighted mean disease” (WMD). A skewed distribution toward resistance was observed (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1a</a>), similar to what was observed in an earlier evaluation of bi-parental mapping populations (<a id="xref-ref-60-7" class="xref-bibr" href="#ref-60">Singh <em>et al.</em> 2016</a>). Leaf blight symptoms were common following inoculation, while only a few plots showed severe wilting. In an analysis of variance on the diversity panel experiment, inbred line and inbred line-by-year interaction effects were found to be significant (<em>P</em> &lt; 0.05). Broad-sense heritability on a plot basis was high (<em>H</em><sup>2</sup> = 0.75), indicating a combination of measurement precision and a large amount of genetic variation for Goss’s wilt WMD in this panel.</p></div><div id="sec-16" class="subsection"><h3>Genetically similar lines vary in their level of resistance to Goss’s wilt</h3><p id="p-34">After filtering on the basis of SNP quality and minor allele frequency, a total of 342,237 SNP calls on the diversity panel remained. Linkage disequilibrium between SNPs was found to rapidly decay to 0.2 within 1000 bp, suggesting a high amount of diversity in the chosen panel. Principal component analysis and ADMIXTURE analysis (K = 3) using the SNP dataset revealed subpopulations within the diversity panel (Supplementary Figure 2). The first principal component (PC1) separated lines according to stiff stalk, non-stiff, and popcorn subpopulations. A few sweet corn and tropical lines included in the panel also clustered among themselves (Supplementary Figure 2a).</p><p id="p-35">A neighbor joining tree of the diversity panel lines was created to visualize the subpopulations and distribution of Goss’s wilt resistance by subpopulation. Color coding of lines by subpopulation clearly indicated that the lines clustered according to subpopulation, as expected (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1b</a>). No strong pattern in the distribution of lines according to Goss’s wilt resistance score was observed (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1c</a>) as groups of highly related lines included both those that were resistant and susceptible. Although elite popcorn breeding lines are generally highly susceptible to Goss’s wilt (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Rocheford <em>et al.</em> 1985</a>), the popcorn lines screened as part of this diversity panel ranged from resistant to highly susceptible, indicating a high degree of variation even within popcorn (<a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1c</a>). It was found that lines closely related to Mo17 were generally resistant, and most of the lines related to B14 were moderately to highly susceptible, in agreement with previous findings (<a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">Calub <em>et al.</em> 1974</a>).</p></div><div id="sec-17" class="subsection"><h3>Genetic variation in Goss’s wilt resistance is controlled by many loci of small effect</h3><p id="p-36">Association analysis within the diversity panel identified three SNPs on chromosome 5 that passed a FDR cutoff of 0.10, with two of the SNPs being within just 17 bp of each other (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>, <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2a</a>). The amount of phenotypic variation for Goss’s wilt resistance explained by individual SNPs within the diversity panel was 3–5% (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). Together the three significant SNPs explained 8% of the phenotypic variation after accounting for population structure. In contrast, the polygenic genetic variance, modeled by the genomic relationship matrix (GRM) calculated using all SNPs, explained 64% of the phenotypic variation for Goss’s wilt resistance within the diversity panel.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/494467/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/494467/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Information about significant SNPs associated with Goss’s wilt resistance in the diversity panel and combined dataset</span><div class="sb-div caption-clear"></div></div></div><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/10/3139/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Manhattan plots showing results from genome-wide association mapping. (a) GWAS results on the diversity panel (N =555) displayed in a Manhattan plot where the y-axis is the negative log10 of p-values for the SNPs from model [2]. Associations that passed the false-discovery rate of 0.10 are colored green; (b) genome-wide association mapping results on the combined dataset (N =1005) displayed in a Manhattan plot where the y-axis is the -log10(P) for the SNPs from model [2]. Associations that passed the false-discovery rate of 0.10 are colored green." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-59671720" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Manhattan plots showing results from genome-wide association mapping. (a) GWAS results on the diversity panel (N =555) displayed in a Manhattan plot where the y-axis is the negative log&lt;sub&gt;10&lt;/sub&gt; of p-values for the SNPs from model [2]. Associations that passed the false-discovery rate of 0.10 are colored green; (b) genome-wide association mapping results on the combined dataset (N =1005) displayed in a Manhattan plot where the y-axis is the -log&lt;sub&gt;10&lt;/sub&gt;(P) for the SNPs from model [2]. Associations that passed the false-discovery rate of 0.10 are colored green.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/10/3139/F2.medium.gif" width="363" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/9/10/3139/F2.medium.gif" width="363" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/10/3139/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/10/3139/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494483" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-45" class="first-child">Manhattan plots showing results from genome-wide association mapping. (a) GWAS results on the diversity panel (N =555) displayed in a Manhattan plot where the y-axis is the negative log<sub>10</sub> of p-values for the SNPs from model [2]. Associations that passed the false-discovery rate of 0.10 are colored green; (b) genome-wide association mapping results on the combined dataset (N =1005) displayed in a Manhattan plot where the y-axis is the -log<sub>10</sub>(P) for the SNPs from model [2]. Associations that passed the false-discovery rate of 0.10 are colored green.</p><div class="sb-div caption-clear"></div></div></div><p id="p-46">We subsequently conducted GWAS using a combined dataset that included both the diversity panel and three RIL populations. The phenotypic data for the three RIL populations was the same as that analyzed by <a id="xref-ref-60-8" class="xref-bibr" href="#ref-60">Singh <em>et al.</em> (2016)</a>. Ten SNPs were found to be significant at a FDR of 0.10 in this analysis; these SNPs were on chromosomes 1, 2, and 5 (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>, <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2b</a>). Each of the SNPs identified in the combined dataset explained a small amount of phenotypic variation for Goss’s wilt resistance, ranging from 1 to 3%. Physical positions of the significant SNPs found in GWAS of the combined dataset were compared to the QTL intervals of QTL mapping conducted using only the bi-parental families with fewer SNPs by <a id="xref-ref-60-9" class="xref-bibr" href="#ref-60">Singh <em>et al.</em> (2016)</a>. Significant SNPs in GWAS on chromosome 2 co-localized with QTL peak from joint-linkage mapping and on chromosome 5 co-localized with QTL peak of the B73 x HP301 bi-parental population reported by <a id="xref-ref-60-10" class="xref-bibr" href="#ref-60">Singh <em>et al.</em> (2016)</a> (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). However, certain QTL were not detected, including QTL detected in both joint linkage mapping and within individual families (<em>e.g.</em>, chromosome 4, 9, and 10). Other QTL specific to one bi-parental family were not detected in GWAS as well (<em>e.g.</em>, chromosome 6, and 7).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/10/3139/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparison of physical positions of QTL detected in bi-parental linkage mapping and GWAS. Bi-parental linkage mapping was conducted by Singh et al. (2016), and significant SNPs from GWAS in the combined dataset co-located on chromosomes 1, 2, and 5. The x-axis of each plot represents the physical position of each SNP, and the y-axis displays the negative log10 of p-values for each SNP included in the GWAS. Gray colored solid points represent all SNPs used in GWAS. Significant SNPs in the GWAS are indicated by green dots, and 2-LOD support interval of QTL detected by Singh et al. (2016) are shown by the red or blue windows." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-59671720" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Comparison of physical positions of QTL detected in bi-parental linkage mapping and GWAS. Bi-parental linkage mapping was conducted by Singh &lt;em&gt;et al.&lt;/em&gt; (2016), and significant SNPs from GWAS in the combined dataset co-located on chromosomes 1, 2, and 5. The x-axis of each plot represents the physical position of each SNP, and the y-axis displays the negative log&lt;sub&gt;10&lt;/sub&gt; of p-values for each SNP included in the GWAS. Gray colored solid points represent all SNPs used in GWAS. Significant SNPs in the GWAS are indicated by green dots, and 2-LOD support interval of QTL detected by Singh &lt;em&gt;et al.&lt;/em&gt; (2016) are shown by the red or blue windows.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/10/3139/F3.medium.gif" width="440" height="243"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/9/10/3139/F3.medium.gif" width="440" height="243"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/10/3139/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/10/3139/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494451" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-47" class="first-child">Comparison of physical positions of QTL detected in bi-parental linkage mapping and GWAS. Bi-parental linkage mapping was conducted by <a id="xref-ref-60-11" class="xref-bibr" href="#ref-60">Singh <em>et al.</em> (2016)</a>, and significant SNPs from GWAS in the combined dataset co-located on chromosomes 1, 2, and 5. The x-axis of each plot represents the physical position of each SNP, and the y-axis displays the negative log<sub>10</sub> of p-values for each SNP included in the GWAS. Gray colored solid points represent all SNPs used in GWAS. Significant SNPs in the GWAS are indicated by green dots, and 2-LOD support interval of QTL detected by <a id="xref-ref-60-12" class="xref-bibr" href="#ref-60">Singh <em>et al.</em> (2016)</a> are shown by the red or blue windows.</p><div class="sb-div caption-clear"></div></div></div><p id="p-48">Identification of candidate genes could contribute to an enhanced understanding of the biology underlying resistance to Goss’s wilt and provide a starting point for future gene confirmation studies. A candidate gene analysis was performed by extending a window around a given significant SNP until the average LD decayed to 0.20 on both sides of the significant SNP. If the defined window around the significant SNPs overlapped with the position of a gene, the gene was declared as a potential candidate. Several protein coding genes were identified within these windows with notable functions such as nucleic acid binding, zinc ion binding, electron transport, and a glutamate receptor protein (<a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>). In each of the regions with significant SNPs, the number of candidate genes ranged from zero to three (<a id="xref-table-wrap-1-5" class="xref-table" href="#T1">Table 1</a>). One candidate gene on chromosome 5 was found to have a function directly related to plant defense. This gene codes for glutamate receptor protein and is a member of the glutamate receptor-like gene family (<em>GRLs</em>) that has been reported to play a role in plant defense response (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Forde and Roberts 2014</a>). Other candidate genes did not have an apparent direct role in defense to pathogens (<a id="xref-table-wrap-1-6" class="xref-table" href="#T1">Table 1</a>).</p></div><div id="sec-18" class="subsection"><h3>Haplotype block analysis of Goss’s wilt resistance QTL</h3><p id="p-49">A haplotype block analysis was conducted on each genomic region harboring significant SNPs to assess local LD decay rate and distribution of haplotype alleles among subpopulations. The haplotype blocks at the chromosome 1 and 5 regions were smaller in size due to low LD in these regions, while chromosome 2 had larger haplotype blocks (Supplementary Figure 3). Four of the five significant SNPs on chromosome 2 were located in block 1 (5 kb), while the fifth SNP (1,977,875 bp away) was excluded from the haplotype block analysis in this region (Supplementary Figure 3b). The chromosome 2 region was investigated further to look at the haplotype allele frequency in the diversity panel as a whole as well as within individual subpopulations. Multiple alleles were observed at each haplotype block within the region of interest on chromosome 2. The allelic effects were small, as the WMD of the lines carrying each of the haplotype allele ranged from 2.3 to 3.0 (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). Allele A1 had the lowest mean Goss’s wilt WMD, and it was the most common allele in all subpopulations except for popcorn in which it had a frequency of only 0.02. Allele A2 had a higher Goss’s wilt WMD and a high frequency among the popcorn lines (93.8% of lines). The allele with the highest Goss’s wilt WMD, A3, had a relatively high frequency in the stiff stalk (29% of lines) and non-stiff stalk (20% of lines) subpopulations compared to the popcorn subpopulation (4.2% of lines), indicating there may be an allele carried by many stiff stalk and non-stiff stalk inbred lines that confers higher susceptibility than the allele common in the popcorn subpopulation (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>).</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/494487/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/494487/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Haplotype allele frequency in the diversity panel as a whole, and within individual subpopulations (stiff stalk, non-stiff stalk, popcorn, and unclassified). Haplotype blocks 1-4 are those at the chromosome 2 QTL region detected in the combined analysis of the diversity panel and RILs as shown in supplementary figure 2. Mean weighted mean disease (WMD) of the lines carrying each haplotype allele are presented</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-19" class="subsection"><h3>A module of co-expressed genes showed responsiveness to inoculations with CMN</h3><p id="p-55">In addition to allelic variation, expression variation can contribute significantly to trait variation. To identify genes with expression variation that may be contributing to resistance to Goss’s wilt, a time course RNA sequencing experiment of a relatively resistant inbred line (N551) and susceptible inbred line (B14A) under control and infected conditions was performed. These lines were chosen for this experiment because they showed different levels of resistance to Goss’s wilt, and they are highly related to one another, reducing confounding genetic background effects (<a id="xref-fig-1-7" class="xref-fig" href="#F1">Figures 1b</a> and <a id="xref-fig-1-8" class="xref-fig" href="#F1">1c</a>). Weighted gene co-expression network analysis identified eight modules of co-expressed genes in the time course experiment. The number of genes included in each module ranged from 30 in module eight to 1,557 in module 1, with 7,373 genes not clustering with any other genes. A heatmap of the eigen genes of the modules provides a high-level overview of each module, allowing identification of trends in gene expression changes of possible biological importance. We did not find any modules showing changes in gene expression in response to infection that were specific to a genotype (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4a</a>). However, module two showed changes in gene expression in both lines in response to <em>Cmn</em> (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4a</a>). In both N551 and B14A, the 141 genes that clustered into module two showed a decrease in expression at eight hpi, and an even further decrease in expression at 15 hpi (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4a</a>). Further evaluation of genes inside the second module from parallel coordinate plots revealed that most genes were down-regulated in B14A and N551 at eight and 15 hpi with <em>Cmn</em> (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4b</a>). Module one was genotype specific module and rest of the modules appear to indicate variation at the sample level.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/10/3139/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Expression patterns of eight modules identified using weighted gene co-expression network analysis (WGCNA) across different samples. (a) Heatmap of eigen genes of eight modules. An eigen gene is representative of the gene expression pattern of genes inside that module. Labels of the columns are inbred line_treatment_hours_rep; (b) Normalized expression of genes inside module two obtained from WGCNA across all the samples. This module showed changes in gene expression in both susceptible line B14A as well as resistant line N551 in response to Cmn; (c) Distribution of phenotypic variance explained by SNPs inside and flanking the 1,000 random samples of genes. As a comparison, phenotypic variation explained by SNPs inside and flanking the genes of module two is depicted by red vertical dashed line." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-59671720" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Expression patterns of eight modules identified using weighted gene co-expression network analysis (WGCNA) across different samples. (a) Heatmap of eigen genes of eight modules. An eigen gene is representative of the gene expression pattern of genes inside that module. Labels of the columns are inbred line_treatment_hours_rep; (b) Normalized expression of genes inside module two obtained from WGCNA across all the samples. This module showed changes in gene expression in both susceptible line B14A as well as resistant line N551 in response to &lt;em&gt;Cmn&lt;/em&gt;; (c) Distribution of phenotypic variance explained by SNPs inside and flanking the 1,000 random samples of genes. As a comparison, phenotypic variation explained by SNPs inside and flanking the genes of module two is depicted by red vertical dashed line.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/10/3139/F4.medium.gif" width="294" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/9/10/3139/F4.medium.gif" width="294" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/10/3139/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/10/3139/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494378" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-56" class="first-child">Expression patterns of eight modules identified using weighted gene co-expression network analysis (WGCNA) across different samples. (a) Heatmap of eigen genes of eight modules. An eigen gene is representative of the gene expression pattern of genes inside that module. Labels of the columns are inbred line_treatment_hours_rep; (b) Normalized expression of genes inside module two obtained from WGCNA across all the samples. This module showed changes in gene expression in both susceptible line B14A as well as resistant line N551 in response to <em>Cmn</em>; (c) Distribution of phenotypic variance explained by SNPs inside and flanking the 1,000 random samples of genes. As a comparison, phenotypic variation explained by SNPs inside and flanking the genes of module two is depicted by red vertical dashed line.</p><div class="sb-div caption-clear"></div></div></div><p id="p-57">A gene ontology (GO) enrichment analysis revealed that the genes of the second module were not enriched for defense functions. Most of the significant Biological Process related GO terms pointed toward regulation of metabolism (Supplementary Table 2). Checking the functional annotations of genes inside the second module revealed several protein coding genes including protease inhibitor, wound induced proteins, Thiazole biosynthetic enzyme, aquaporin, DRE-binding protein, etc. (Supplementary Table 3).</p><p id="p-58">We wanted to test whether SNPs within and flanking (up to one kb on either side) the genes in module two explained a disproportionate amount of the variance in Goss’s wilt resistance observed in the diversity panel to help determine if these genes are important contributors to genetic variance for quantitative disease resistance. The physical positions of the SNPs used in the GWAS were aligned with the physical positions of the genes that clustered into module two. Twenty genes in module two had at least one SNP within or flanking them, with the remaining 121 genes having no SNPs within or flanking them. The total number of SNPs found was 83. These 83 SNPs were used to calculate a GRM, which was then fit into a linear mixed model to estimate the variance in WMD explained by these SNPs (<a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Yang <em>et al.</em> 2010</a>; <a id="xref-ref-22-3" class="xref-bibr" href="#ref-22">Gusev <em>et al.</em> 2014</a>). The SNPs of module two explained 13.3% of the phenotypic variation for WMD, which is higher than the total percent variation explained by the significant GWAS SNPs. To determine if the variance explained by the SNPs of genes included in module two was greater than expected by a random set of genes, additional GRMs were calculated using SNPs inside and flanking 1,000 random samples of 20 genes. The number of SNPs in the 1,000 random samples ranged from 0 to 199 with a mean of 39 SNPs. The percent variation explained by the SNPs among the 1,000 random gene samples ranged from 0.0 to 64.8%. Only 8.6% of the 1,000 random samples explained more variation than the module two SNPs (<a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4c</a>), providing suggestive, but not strong nor statistically significant evidence for a role of these genes in Goss’s wilt resistance at the field level. A similar analysis was performed where 1000 random sets of SNPs were sampled in which the number of SNPs inside and flanking genes (83) was held constant rather than the number of genes. The variation explained by these 1000 sets of SNPs ranged from 5 to 20%, with 9.4% of the random samples explaining more variation than the 83 SNPs implicated by the gene expression analysis. These results led to the same conclusion as the results obtained where the gene was considered as the sampling unit.</p></div></div><div class="section discussion" id="sec-20"><h2 class="">Discussion</h2><p id="p-59">Genome-wide association mapping, made possible by the widest screening of Goss’s wilt resistance reported to-date, and gene co-expression network analysis were used to increase our knowledge of the genetic basis of resistance to Goss’s wilt of maize. Goss’s wilt is an important disease of maize that has not been studied extensively using modern technologies of molecular genetics as compared to other maize diseases such as NCLB, SCLB, and gray leaf spot (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Kump <em>et al.</em> 2011</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Poland <em>et al.</em> 2011</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Benson <em>et al.</em> 2015</a>), only three published studies have used molecular markers to study the genetic basis of Goss’s wilt resistance (<a id="xref-ref-57-3" class="xref-bibr" href="#ref-57">Schaefer and Bernardo 2013</a>; <a id="xref-ref-60-13" class="xref-bibr" href="#ref-60">Singh <em>et al.</em> 2016</a>; <a id="xref-ref-13-3" class="xref-bibr" href="#ref-13">Cooper <em>et al.</em> 2018</a>). Both GWAS and gene co-expression network analysis approaches pointed toward a complex nature of resistance to Goss’s wilt, which is in agreement with the complex nature of plant signaling and defense processes as studied using model organisms (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Jones and Dangl 2006</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Bonardi and Dangl 2012</a>) and hypothesized to be the case for Goss’s wilt based on initial genetic studies (<a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Martin <em>et al.</em> 1975</a>; <a id="xref-ref-51-2" class="xref-bibr" href="#ref-51">Rocheford <em>et al.</em> 1989</a>; <a id="xref-ref-65-2" class="xref-bibr" href="#ref-65">Treat <em>et al.</em> 1990</a>).</p><p id="p-60">A great degree of variation for Goss’s wilt resistance was found within subpopulations, with relatively little variation between subpopulations in most cases. Our expectation was that inbred lines within certain subpopulations would be more resistant on average as compared to inbred lines from other subpopulations. This was, however, not observed in this study as resistant and susceptible lines were distributed among all the groups (<a id="xref-fig-1-9" class="xref-fig" href="#F1">Figure 1c</a>). The popcorn subpopulation was more susceptible overall, but relatively resistant popcorn inbred lines were still found. Lines related to inbred line B14 tended to be susceptible whereas the lines related to Mo17 were found to be generally resistant, as has been reported in previous studies (<a id="xref-ref-58-4" class="xref-bibr" href="#ref-58">Schuster <em>et al.</em> 1972</a>; <a id="xref-ref-10-4" class="xref-bibr" href="#ref-10">Calub <em>et al.</em> 1974</a>).</p><p id="p-61">The GWAS and calculated proportion of variance explained by all genotyped SNPs points toward a highly polygenic genetic architecture underlying Goss’s wilt resistance. Similar to our previous QTL mapping study (<a id="xref-ref-60-14" class="xref-bibr" href="#ref-60">Singh <em>et al.</em> 2016</a>), each of the significant SNPs associated with Goss’s wilt resistance in the GWAS explained a small amount of the phenotypic variation. For other important leaf diseases of maize including southern corn leaf blight, northern corn leaf blight, and gray leaf spot several small effect SNPs have been associated with resistance (<a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Kump <em>et al.</em> 2011</a>; <a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">Poland <em>et al.</em> 2011</a>; <a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Benson <em>et al.</em> 2015</a>). Thirty-two QTL with small additive effects that together explained 80% and 93% of the phenotypic and genotypic variation, respectively, for southern corn leaf blight were reported in a GWAS of maize NAM population (<a id="xref-ref-30-3" class="xref-bibr" href="#ref-30">Kump <em>et al.</em> 2011</a>). Similarly, 29 QTL were identified for northern corn leaf blight in the NAM population that explained 77% and 96% of the phenotypic and genetic variance, respectively, for northern corn leaf blight, with each QTL being of small effect (<a id="xref-ref-47-3" class="xref-bibr" href="#ref-47">Poland <em>et al.</em> 2011</a>). The studies using the NAM population explained greater total phenotypic variation because of the nested linkage mapping design of the NAM population combined with a much larger population size. Nevertheless, the polygenic variance accounted for 64% of the phenotypic variation for Goss’s wilt resistance in the diversity panel and thus a genomic selection approach may be more effective in breeding for Goss’s wilt resistance compared to selecting for individual QTL. A similar conclusion was reported in a <em>Fusarium</em> ear rot study in maize in which the GRM explained nearly 50% of the variation for <em>Fusarium</em> ear rot and only 1.3–3% of the variation was explained by individual significant SNPs (<a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">Zila <em>et al.</em> 2013</a>).</p><p id="p-62">Combined GWAS (diversity panel and bi-parental populations together) increased the power of GWAS and more genomic regions were associated with Goss’s wilt resistance as compared to GWAS of the diversity panel only. However, there were some QTL regions that were detected in joint-linkage and linkage mapping of individual bi-parental populations conducted by <a id="xref-ref-60-15" class="xref-bibr" href="#ref-60">Singh <em>et al.</em> (2016)</a> that were not detected by combined GWAS. One possible cause of not detecting certain QTL in combined GWAS approach could be different predominant linkage phases between SNPs and causal polymorphisms between bi-parental populations and the diversity panel, possibly causing effects to be cancelled out. Also, because the number of markers was much larger for GWAS, the higher statistical threshold of GWAS needed to keep the experiment-wise false positive error rate in check likely resulted in failure to declare significance for smaller effect QTL.</p><p id="p-63">In addition to the variation at the allelic level, differential regulation of genes in response to pathogen may contribute to the phenotypic variation. Furthermore, the number of host genes regulated by disease resistance is a basic systems biology question that has been explored multiple times with different plant pathosystems. In a gene-co-expression network analysis of response of Citrus to bacterium <em>Candidatus liberibacter</em> spp., 3,507 genes were suggested to play a role in defense (<a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">Zheng and Zhao 2013</a>). Similarly, gene co-expression network analysis of genes regulated by immune and hypersensitive resistance responses to <em>Blumeria graminis</em> f. sp <em>tritici</em> in diploid wheat <em>Triticum urartu</em> indicated that 3,900 and 4,100 genes may be involved in the above two types of resistance responses, respectively (<a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">Zhang <em>et al.</em> 2016</a>). In the present study, we identified one module that corresponded to response of maize inbred lines to <em>Cmn</em> were identified consisting of 141 co-related genes. Most of the genes in this module showed decreased expression in susceptible and resistant inbred lines at at 8 and 15 hpi. This result corroborates with previous findings from molecular and co-expression network studies showing that plant immunity can be controlled by negative regulation of a certain set of genes (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Sato <em>et al.</em> 2010</a>; <a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Segonzac <em>et al.</em> 2014</a>). Particularly, specific subunits of protein Ser/Thr phosphatases have been involved in negative regulation of defense signaling at different steps (<a id="xref-ref-59-2" class="xref-bibr" href="#ref-59">Segonzac <em>et al.</em> 2014</a>). Negative regulation of certain genes within a plant immune system are required for effective functioning by preventing over activation of the immune system which may cause auto immune responses such as cell death, thus reducing plant fitness (<a id="xref-ref-56-2" class="xref-bibr" href="#ref-56">Sato <em>et al.</em> 2010</a>).</p><p id="p-64">A search for overlap between genes implicated by the WGCNA and variation explained in Goss’s resistance at the field level was performed to determine the extent to which genes in module two are important to variation in resistance among inbred lines. None of the eight candidate genes identified in the GWAS analysis overlapped with the genes inside of module two. Several reasons could explain this lack of strong correspondence between GWAS and WGCNA. The power of the GWAS was low such that it could not be expected to detect all the phenotypic variation due to possible low frequencies of the causal alleles and rapid LD decay in this panel. Second, failure of WGCNA implicated genes to explain more variation in Goss's wilt resistance than random samples of genes could be because the variation in gene expression of these genes is not related to polymorphisms within the genes, but rather to polymorphisms in <em>cis</em>- or <em>trans</em>-acting transcription factors. Third, it is possible that the gene expression differences between the two chosen inbred lines, B14A and N551, are not representative of the gene expression patterns in the diversity panel as a whole. Fourthly, the genes contributing to quantitative disease resistance in the field, 15 days or more after inoculation may not be the same, or have little overlap with, genes responding to infection within hours during the seedling stages. Finally, while a mixture of five isolates was used for field inoculations of the GWAS, only one isolate was used to inoculate B14A and N551 in the lab.</p><p id="p-65">Using a variance component approach, we found that the genes implicated in the WGCNA did not explain a disproportionate amount of the variance in Goss’s wilt resistance in the diversity panel using a significance level of <em>P</em> &lt; 0.05. However, only 8% of 1,000 random genes samples explained more phenotypic variance than those genes implicated by the WGCNA, suggesting a possible role of these genes. More research is needed to precisely determine the role of these genes in quantitative Goss’s resistance at the field level. More generally, these results suggest a possible useful role for combining WGCNA with quantitative genetic analyses to determine genes underlying variation for complex disease resistance traits in crops. More research to refine these methods to attain this overall goal is warranted.</p><p id="p-66">Goss’s wilt and leaf blight of maize is an economically important disease, yet very few studies have been performed with the aim of elucidating the genetic basis of host resistance using modern genomic and transcriptomic techniques. This study explored an extensive amount of genetic variation for Goss’s wilt resistance and identified SNPs associated with resistance using a GWAS approach. The amount of variation attributed to the background polygenic effect through a genomic relationship matrix was eightfold higher than that explained by statistically significant QTL alone. These results suggest the genetic architecture of this disease is highly polygenic, which is consistent with the findings of our previous linkage mapping study and those using classical mating designs. Several biological processes may be involved in quantitative resistance to Goss’s wilt which were revealed by GO analysis and functional annotations of module two genes. Results from this study provide important information about the number of QTL, effect size of QTL, and the candidate genes underlying Goss’s wilt resistance, and provides a critical step toward further elucidating the genetic mechanisms of Goss’s wilt resistance. Due to the presence of several small effect QTL that together contribute to resistance to Goss’s wilt, a single QTL likely cannot be targeted for incorporating resistance into maize germplasm through marker-assisted selection. Rather, a genome-wide selection approach for population improvement maybe a better strategy for imparting resistance to Goss’s wilt into maize breeding programs.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-67">We are thankful to Dow AgroSciences for funding this research. ABB was supported by the DuPont Pioneer Bill Kuhn Honorary Fellowship and the MnDRIVE Global Food Ventures Graduate Student Fellowship. The Minnesota Supercomputing Institute (MSI) at the University of Minnesota and Holland Computing Center at the University of Nebraska provided computational resources that contributed to this research. We appreciate the help received from Nonoy Bandillo, Dnyaneshwar C. Kadam, Jon Luetchens, M.D. Liakat Ali, and Collin Lamkey for planting and inoculating the Goss’s wilt experiments. We are also thankful to Jackson lab members: Tamra Jackson, Jae Brungardt, Brad Tharnish, and Cody Kaarstad for providing inoculum. The U.S. National Plant Germplasm System and North Central Regional Plant Introduction Station provided seeds of germplasm used in this study, for which we are very grateful.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-14"><p id="p-68">Supplemental material available at FigShare: <a href="https://doi.org/10.25387/g3.7884779">https://doi.org/10.25387/g3.7884779</a>.</p></li><li class="fn" id="fn-15"><p id="p-69"><em>Communicating editor: A. Paterson</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2019-03-23"><span class="received-label">Received </span>March 23, 2019.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2019-07-23"><span class="accepted-label">Accepted </span>July 23, 2019.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2019 Singh <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3139.1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Abendroth</span>,  <span class="cit-name-given-names">L. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. W.</span> <span class="cit-name-surname">Elmore</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. J.</span> <span class="cit-name-surname">Boyer</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. K.</span> <span class="cit-name-surname">Marlay</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> Corn Growth and Development Iowa State University Press, Ames, <a href="https://store.extension.iastate.edu/product/6065">https://store.extension.iastate.edu/product/6065</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3139.2" data-doi="10.1101/gr.094052.109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Alexander</span>,  <span class="cit-name-given-names">D. H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Novembre</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Lange</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Fast model-based estimation of ancestry in unrelated individuals.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">1655</span>–<span class="cit-lpage">1664</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.094052.109</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.094052.109%26rft_id%253Dinfo%253Apmid%252F19648217%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjE5LzkvMTY1NSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzkvMTAvMzEzOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-9.10.3139.3" data-doi="10.1093/bioinformatics/btu638"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Anders</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. T.</span> <span class="cit-name-surname">Pyl</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Huber</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">HTSeq—a Python framework to work with high-throughput sequencing data.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">166</span>–<span class="cit-lpage">169</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu638%26rft_id%253Dinfo%253Apmid%252F25260700%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btu638&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25260700&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3139.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000347832300003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3139.4" data-doi="10.1038/415977a"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Asai</span>,  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Tena</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Plotnikova</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. R.</span> <span class="cit-name-surname">Willmann</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.-L.</span> <span class="cit-name-surname">Chiu</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">MAP kinase signalling cascade in Arabidopsis innate immunity.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">415</span>: <span class="cit-fpage">977</span>–<span class="cit-lpage">983</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/415977a</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DAsai%26rft.auinit1%253DT.%26rft.volume%253D415%26rft.issue%253D6875%26rft.spage%253D977%26rft.epage%253D983%26rft.atitle%253DMAP%2Bkinase%2Bsignalling%2Bcascade%2Bin%2BArabidopsis%2Binnate%2Bimmunity.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F415977a%26rft_id%253Dinfo%253Apmid%252F11875555%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/415977a&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11875555&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3139.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000174075000031&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3139.5" data-doi="10.2135/cropsci2009.02.0066"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Balint-Kurti</span>,  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Yang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G. V.</span> <span class="cit-name-surname">Esbroeck</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Jung</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. E.</span> <span class="cit-name-surname">Smith</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Use of a Maize Advanced Intercross Line for Mapping of QTL for Northern Leaf Blight Resistance and Multiple Disease Resistance.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">50</span>: <span class="cit-fpage">458</span>–<span class="cit-lpage">466</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2009.02.0066</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DBalint-Kurti%26rft.auinit1%253DP.%2BJ.%26rft.volume%253D50%26rft.issue%253D2%26rft.spage%253D458%26rft.epage%253D466%26rft.atitle%253DUse%2Bof%2Ba%2BMaize%2BAdvanced%2BIntercross%2BLine%2Bfor%2BMapping%2Bof%2BQTL%2Bfor%2BNorthern%2BLeaf%2BBlight%2BResistance%2Band%2BMultiple%2BDisease%2BResistance%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2009.02.0066%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2009.02.0066&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000275564500004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3139.6" data-doi="10.1093/bioinformatics/bth457"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Barrett</span>,  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Fry</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Maller</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. J.</span> <span class="cit-name-surname">Daly</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Haploview: analysis and visualization of LD and haplotype maps.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">263</span>–<span class="cit-lpage">265</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/bth457</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbth457%26rft_id%253Dinfo%253Apmid%252F15297300%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bth457&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15297300&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3139.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000226308500016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3139.7" data-doi="10.1371/journal.pgen.1005045"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Benson</span>,  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. A.</span> <span class="cit-name-surname">Poland</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. M.</span> <span class="cit-name-surname">Benson</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. L.</span> <span class="cit-name-surname">Stromberg</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. J.</span> <span class="cit-name-surname">Nelson</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Resistance to gray leaf spot of maize: genetic architecture and mechanisms elucidated through nested association mapping and near-isogenic line analysis.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-elocation-id">e1005045</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1005045</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D11%26rft.spage%253De1005045%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1005045%26rft_id%253Dinfo%253Apmid%252F25764179%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1005045&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25764179&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3139.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3139.8" data-doi="10.3389/fpls.2012.00237"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bonardi</span>,  <span class="cit-name-given-names">V.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. L.</span> <span class="cit-name-surname">Dangl</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">How complex are intracellular immune receptor signaling complexes?</span> <abbr class="cit-jnl-abbrev">Front. Plant Sci.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">237</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fpls.2012.00237</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DFront%2BPlant%2BSci%26rft.aulast%253DBonardi%26rft.auinit1%253DV.%26rft.volume%253D3%26rft.spage%253D237%26rft.epage%253D237%26rft.atitle%253DHow%2Bcomplex%2Bare%2Bintracellular%2Bimmune%2Breceptor%2Bsignaling%2Bcomplexes%253F%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffpls.2012.00237%26rft_id%253Dinfo%253Apmid%252F23109935%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fpls.2012.00237&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23109935&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3139.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3139.9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Butler</span>,  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Cullis</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Gilmour</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Gogel</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <em>{</em><em>ASReml}-R reference manual</em>, QE02001, ISSN 0812-0005.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3139.10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Calub</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W. A.</span> <span class="cit-name-surname">Compton</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. O.</span> <span class="cit-name-surname">Gardner</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. L.</span> <span class="cit-name-surname">Schuster</span></span></li></ol><cite>, <span class="cit-pub-date">1974</span> <span class="cit-article-title">Reaction of 113 corn inbreds to leaf freckles and wilt of corn.</span> <abbr class="cit-jnl-abbrev">Plant Dis. Rep.</abbr> <span class="cit-vol">58</span>: <span class="cit-fpage">956</span>–<span class="cit-lpage">960</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BDis.%2BRep.%26rft.volume%253D58%26rft.spage%253D956%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3139.11" data-doi="10.1094/Phyto-81-95"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Carson</span>,  <span class="cit-name-given-names">M. L.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z. W.</span> <span class="cit-name-surname">Wicks</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">Relationship between Leaf Freckles and Wilt Severity and Yield Losses in Closely Related Maize Hybrids.</span> <abbr class="cit-jnl-abbrev">Phytopathology</abbr> <span class="cit-vol">81</span>: <span class="cit-fpage">95</span>–<span class="cit-lpage">98</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1094/Phyto-81-95</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhytopathology%26rft.volume%253D81%26rft.spage%253D95%26rft_id%253Dinfo%253Adoi%252F10.1094%252FPhyto-81-95%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1094/Phyto-81-95&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3139.12" data-doi="10.1111/tpj.13419"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Christie</span>,  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. A.</span> <span class="cit-name-surname">Myburg</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Joubert</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. L.</span> <span class="cit-name-surname">Murray</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Carstens</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Systems genetics reveals a transcriptional network associated with susceptibility in the maize–grey leaf spot pathosystem.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">89</span>: <span class="cit-fpage">746</span>–<span class="cit-lpage">763</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/tpj.13419</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BJ.%26rft.volume%253D89%26rft.spage%253D746%26rft_id%253Dinfo%253Adoi%252F10.1111%252Ftpj.13419%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/tpj.13419&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3139.13" data-doi="10.2135/cropsci2017.10.0618"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cooper</span>,  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. J.</span> <span class="cit-name-surname">Balint-Kurti</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. M.</span> <span class="cit-name-surname">Jamann</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Identification of Quantitative Trait Loci for Goss’s Wilt of Maize.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">58</span>: <span class="cit-fpage">1192</span>–<span class="cit-lpage">1200</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2017.10.0618</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D58%26rft.spage%253D1192%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2017.10.0618%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2017.10.0618&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3139.14" data-doi="10.1111/j.1365-313X.2008.03590.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Doehlemann</span>,  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Wahl</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. J.</span> <span class="cit-name-surname">Horst</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L. M.</span> <span class="cit-name-surname">Voll</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Usadel</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Reprogramming a maize plant: transcriptional and metabolic changes induced by the fungal biotroph Ustilago maydis.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">56</span>: <span class="cit-fpage">181</span>–<span class="cit-lpage">195</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1365-313X.2008.03590.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2Bjournal%2B%253A%2B%2Bfor%2Bcell%2Band%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BJ%26rft.aulast%253DDoehlemann%26rft.auinit1%253DG.%26rft.volume%253D56%26rft.issue%253D2%26rft.spage%253D181%26rft.epage%253D195%26rft.atitle%253DReprogramming%2Ba%2Bmaize%2Bplant%253A%2Btranscriptional%2Band%2Bmetabolic%2Bchanges%2Binduced%2Bby%2Bthe%2Bfungal%2Bbiotroph%2BUstilago%2Bmaydis.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-313X.2008.03590.x%26rft_id%253Dinfo%253Apmid%252F18564380%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-313X.2008.03590.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18564380&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3139.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000259914500001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3139.15" data-doi="10.1093/nar/gkq310"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Du</span>,  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Zhou</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Ling</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Zhang</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Su</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">agriGO: a GO analysis toolkit for the agricultural community.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">W64</span>–<span class="cit-lpage">W70</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkq310</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkq310%26rft_id%253Dinfo%253Apmid%252F20435677%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkq310&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20435677&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3139.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000284148900012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3139.16" data-doi="10.1371/journal.pone.0019379"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Elshire</span>,  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. C.</span> <span class="cit-name-surname">Glaubitz</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Q.</span> <span class="cit-name-surname">Sun</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. A.</span> <span class="cit-name-surname">Poland</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Kawamoto</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A Robust, Simple Genotyping-by-Sequencing (GBS) Approach for High Diversity Species.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">6</span>: <span class="cit-elocation-id">e19379</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0019379</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DElshire%26rft.auinit1%253DR.%2BJ.%26rft.volume%253D6%26rft.issue%253D5%26rft.spage%253De19379%26rft.epage%253De19379%26rft.atitle%253DA%2Brobust%252C%2Bsimple%2Bgenotyping-by-sequencing%2B%2528GBS%2529%2Bapproach%2Bfor%2Bhigh%2Bdiversity%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0019379%26rft_id%253Dinfo%253Apmid%252F21573248%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0019379&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21573248&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3139.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-book" id="cit-9.10.3139.17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">EPPO</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-source">PQR database</span>, <span class="cit-publ-name">European and Mediterranean Plant Protection Organization</span>, <span class="cit-publ-loc">Paris, France</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3139.18" data-doi="10.12703/P6-37"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Forde</span>,  <span class="cit-name-given-names">B. G.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. R.</span> <span class="cit-name-surname">Roberts</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Glutamate receptor-like channels in plants: a role as amino acid sensors in plant defence?</span> <abbr class="cit-jnl-abbrev">F1000Prime Rep.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">37</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.12703/P6-37</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DF1000Prime%2BRep.%26rft.volume%253D6%26rft.spage%253D37%26rft_id%253Dinfo%253Adoi%252F10.12703%252FP6-37%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.12703/P6-37&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3139.19" data-doi="10.1094/PDIS-04-14-0397-PDN"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Friskop</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Kinzer</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">McConnell</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Liu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Korus</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">First Report of Goss’s Bacterial Leaf Blight and Wilt of Corn Caused by Clavibacter michiganensis subsp. nebraskensis in North Dakota.</span> <abbr class="cit-jnl-abbrev">Plant Dis.</abbr> <span class="cit-vol">98</span>: <span class="cit-fpage">1739</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1094/PDIS-04-14-0397-PDN</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BDis.%26rft.volume%253D98%26rft.spage%253D1739%26rft_id%253Dinfo%253Adoi%252F10.1094%252FPDIS-04-14-0397-PDN%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1094/PDIS-04-14-0397-PDN&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3139.20"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gardner</span>,  <span class="cit-name-given-names">C. O.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. L.</span> <span class="cit-name-surname">Schuster</span></span></li></ol><cite>, <span class="cit-pub-date">1974</span> <span class="cit-article-title">Genetic studies of susceptibility to bacterial leaf freckles and wilt, Corynebacterium nebraskense.</span> <abbr class="cit-jnl-abbrev">Maize Genet. Coop. Newsl.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">155</span>–<span class="cit-lpage">157</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMaize%2BGenet.%2BCoop.%2BNewsl.%26rft.volume%253D47%26rft.spage%253D155%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3139.21" data-doi="10.1371/journal.pone.0090346"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Glaubitz</span>,  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. M.</span> <span class="cit-name-surname">Casstevens</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Lu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Harriman</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. J.</span> <span class="cit-name-surname">Elshire</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">TASSEL-GBS: a high capacity genotyping by sequencing analysis pipeline.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>: <span class="cit-elocation-id">e90346</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0090346</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De90346%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0090346%26rft_id%253Dinfo%253Apmid%252F24587335%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0090346&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24587335&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3139.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3139.22" data-doi="10.1016/j.ajhg.2014.10.004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gusev</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. H.</span> <span class="cit-name-surname">Lee</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Trynka</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Finucane</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. J.</span> <span class="cit-name-surname">Vilhjálmsson</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Partitioning Heritability of Regulatory and Cell-Type-Specific Variants across 11 Common Diseases.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">95</span>: <span class="cit-fpage">535</span>–<span class="cit-lpage">552</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.ajhg.2014.10.004</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BJ.%2BHum.%2BGenet.%26rft.volume%253D95%26rft.spage%253D535%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajhg.2014.10.004%26rft_id%253Dinfo%253Apmid%252F25439723%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ajhg.2014.10.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25439723&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3139.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3139.23" data-doi="10.1094/PDIS-02-16-0227-PDN"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hosack</span>,  <span class="cit-name-given-names">P. K.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L. E.</span> <span class="cit-name-surname">Sweets</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G. L.</span> <span class="cit-name-surname">Miller</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. J.</span> <span class="cit-name-surname">Calcutt</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. E.</span> <span class="cit-name-surname">Arenz</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">First Report of Goss’s Bacterial Wilt and Leaf Blight on Corn Caused by Clavibacter michiganensis subsp. nebraskensis in Missouri.</span> <abbr class="cit-jnl-abbrev">Plant Dis.</abbr> <span class="cit-vol">100</span>: <span class="cit-fpage">2158</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1094/PDIS-02-16-0227-PDN</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BDis.%26rft.volume%253D100%26rft.spage%253D2158%26rft_id%253Dinfo%253Adoi%252F10.1094%252FPDIS-02-16-0227-PDN%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1094/PDIS-02-16-0227-PDN&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3139.24" data-doi="10.1094/PDIS-11-14-1117-PDN"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Howard</span>,  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. W.</span> <span class="cit-name-surname">Harding</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Lynn</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L. M.</span> <span class="cit-name-surname">Kawchuk</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">N. M.</span> <span class="cit-name-surname">Rasmussen</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">First Report of Goss’s Bacterial Wilt and Leaf Blight on Corn Caused by Clavibacter michiganensis subsp. nebraskensis in Alberta, Canada.</span> <abbr class="cit-jnl-abbrev">Plant Dis.</abbr> <span class="cit-vol">99</span>: <span class="cit-fpage">1034</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1094/PDIS-11-14-1117-PDN</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BDis.%26rft.volume%253D99%26rft.spage%253D1034%26rft_id%253Dinfo%253Adoi%252F10.1094%252FPDIS-11-14-1117-PDN%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1094/PDIS-11-14-1117-PDN&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3139.25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jackson</span>,  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. M.</span> <span class="cit-name-surname">Harveson</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. K.</span> <span class="cit-name-surname">Vidaver</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> Goss’s bacterial wilt and leaf blight of corn. NebGuide Coop Ext Serv IANR Univ Nebr Linc. G1675, <a href="http://extensionpublications.unl.edu/assets/pdf/g1675.pdf">http://extensionpublications.unl.edu/assets/pdf/g1675.pdf</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3139.26" data-doi="10.1093/bioinformatics/btm233"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jakobsson</span>,  <span class="cit-name-given-names">M.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">N. A.</span> <span class="cit-name-surname">Rosenberg</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">CLUMPP: a cluster matching and permutation program for dealing with label switching and multimodality in analysis of population structure.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">1801</span>–<span class="cit-lpage">1806</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btm233</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtm233%26rft_id%253Dinfo%253Apmid%252F17485429%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btm233&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17485429&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3139.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000249248300012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3139.27" data-doi="10.1038/nature05286"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>,  <span class="cit-name-given-names">J. D. G.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. L.</span> <span class="cit-name-surname">Dangl</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">The plant immune system.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">444</span>: <span class="cit-fpage">323</span>–<span class="cit-lpage">329</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature05286</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DJones%26rft.auinit1%253DJ.%2BD.%26rft.volume%253D444%26rft.issue%253D7117%26rft.spage%253D323%26rft.epage%253D329%26rft.atitle%253DThe%2Bplant%2Bimmune%2Bsystem.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature05286%26rft_id%253Dinfo%253Apmid%252F17108957%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature05286&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17108957&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3139.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000242018300039&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3139.28" data-doi="10.1186/gb-2013-14-4-r36"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>,  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Pertea</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Trapnell</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Pimentel</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Kelley</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">TopHat2: accurate alignment of transcriptomes in the presence of insertions, deletions and gene fusions.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">R36</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/gb-2013-14-4-r36</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D14%26rft.spage%253DR36%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2013-14-4-r36%26rft_id%253Dinfo%253Apmid%252F23618408%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2013-14-4-r36&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23618408&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3139.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3139.29" data-doi="10.1094/PDIS-07-10-0541"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Korus</span>,  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. D.</span> <span class="cit-name-surname">Timmerman</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. D.</span> <span class="cit-name-surname">French-Monar</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. A.</span> <span class="cit-name-surname">Jackson</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">First Report of Goss’s Bacterial Wilt and Leaf Blight (Clavibacter michiganensis subsp nebraskensis) of Corn in Texas.</span> <abbr class="cit-jnl-abbrev">Plant Dis.</abbr> <span class="cit-vol">95</span>: <span class="cit-fpage">73</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1094/PDIS-07-10-0541</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BDis.%26rft.volume%253D95%26rft.spage%253D73%26rft_id%253Dinfo%253Adoi%252F10.1094%252FPDIS-07-10-0541%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1094/PDIS-07-10-0541&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3139.30" data-doi="10.1038/ng.747"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kump</span>,  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. J.</span> <span class="cit-name-surname">Bradbury</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. J.</span> <span class="cit-name-surname">Wisser</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. S.</span> <span class="cit-name-surname">Buckler</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. R.</span> <span class="cit-name-surname">Belcher</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genome-wide association study of quantitative resistance to southern leaf blight in the maize nested association mapping population.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">163</span>–<span class="cit-lpage">168</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.747</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DKump%26rft.auinit1%253DK.%2BL.%26rft.volume%253D43%26rft.issue%253D2%26rft.spage%253D163%26rft.epage%253D168%26rft.atitle%253DGenome-wide%2Bassociation%2Bstudy%2Bof%2Bquantitative%2Bresistance%2Bto%2Bsouthern%2Bleaf%2Bblight%2Bin%2Bthe%2Bmaize%2Bnested%2Bassociation%2Bmapping%2Bpopulation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.747%26rft_id%253Dinfo%253Apmid%252F21217757%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.747&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21217757&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3139.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000286623800017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3139.31" data-doi="10.1186/1471-2105-9-559"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Langfelder</span>,  <span class="cit-name-given-names">P.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Horvath</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">WGCNA: an R package for weighted correlation network analysis.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">559</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2105-9-559</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DLangfelder%26rft.auinit1%253DP.%26rft.volume%253D9%26rft.spage%253D559%26rft.epage%253D559%26rft.atitle%253DWGCNA%253A%2Ban%2BR%2Bpackage%2Bfor%2Bweighted%2Bcorrelation%2Bnetwork%2Banalysis.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-9-559%26rft_id%253Dinfo%253Apmid%252F19114008%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-9-559&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19114008&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3139.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3139.32" data-doi="10.1186/1471-2164-15-710"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lanubile</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Ferrarini</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Maschietto</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Delledonne</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Marocco</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Functional genomic analysis of constitutive and inducible defense responses to Fusarium verticillioides infection in maize genotypes with contrasting ear rot resistance.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">710</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2164-15-710</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D15%26rft.spage%253D710%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-15-710%26rft_id%253Dinfo%253Apmid%252FWOS%253A00034104%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-15-710&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=WOS:00034104&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3139.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3139.33" data-doi="10.1038/nmeth.1681"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lippert</span>,  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Listgarten</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Liu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. M.</span> <span class="cit-name-surname">Kadie</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. I.</span> <span class="cit-name-surname">Davidson</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">FaST linear mixed models for genome-wide association studies.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">833</span>–<span class="cit-lpage">835</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nmeth.1681</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DLippert%26rft.auinit1%253DC.%26rft.volume%253D8%26rft.issue%253D10%26rft.spage%253D833%26rft.epage%253D835%26rft.atitle%253DFaST%2Blinear%2Bmixed%2Bmodels%2Bfor%2Bgenome-wide%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1681%26rft_id%253Dinfo%253Apmid%252F21892150%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1681&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21892150&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3139.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000295358000017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3139.34" data-doi="10.1371/journal.pone.0119858"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Gao</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Yin</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Gong</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Qin</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Transcriptome Analysis of Maize Leaf Systemic Symptom Infected by Bipolaris zeicola.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">10</span>: <span class="cit-elocation-id">e0119858</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0119858</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D10%26rft.spage%253De0119858%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0119858%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0119858&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3139.35" data-doi="10.1186/s12864-016-2780-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>,  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Guo</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Ma</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Zhang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Wang</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Transcriptome analysis of maize resistance to Fusarium graminearum.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">477</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12864-016-2780-5</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D17%26rft.spage%253D477%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864-016-2780-5%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12864-016-2780-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3139.36" data-doi="10.1186/s13059-014-0550-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Love</span>,  <span class="cit-name-given-names">M. I.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Huber</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Anders</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">550</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s13059-014-0550-8</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D15%26rft.spage%253D550%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-014-0550-8%26rft_id%253Dinfo%253Apmid%252F25516281%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13059-014-0550-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25516281&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3139.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3139.37" data-doi="10.1094/PDIS-94-8-1064A"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Malvick</span>,  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Syverson</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Mollov</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. A.</span> <span class="cit-name-surname">Ishimaru</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Goss’s Bacterial Blight and Wilt of Corn Caused by Clavibacter michiganensis subsp. nebraskensis Occurs in Minnesota.</span> <abbr class="cit-jnl-abbrev">Plant Dis.</abbr> <span class="cit-vol">94</span>: <span class="cit-fpage">1064</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1094/PDIS-94-8-1064A</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BDis.%26rft.volume%253D94%26rft.spage%253D1064%26rft_id%253Dinfo%253Adoi%252F10.1094%252FPDIS-94-8-1064A%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1094/PDIS-94-8-1064A&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3139.38" data-doi="10.14806/ej.17.1.200"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Martin</span>,  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> Cutadapt removes adapter sequences from high-throughput sequencing reads. EMBnet.journal 17: 10–12.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.14806/ej.17.1.200</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCUTADAPT%2BREMOVES%2BADAPTER%2BSEQUENCES%2BFROM%2BHIGHTHROUGHPUT%2BSEQUENCING%2BREADS%26rft.volume%253D17%26rft.spage%253D10%26rft_id%253Dinfo%253Adoi%252F10.14806%252Fej.17.1.200%26rft_id%253Dinfo%253Apmid%252F25452271%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.14806/ej.17.1.200&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25452271&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3139.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3139.39"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Martin</span>,  <span class="cit-name-given-names">P. R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. O.</span> <span class="cit-name-surname">Gardner</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. G.</span> <span class="cit-name-surname">Calub</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. L.</span> <span class="cit-name-surname">Schuster</span></span></li></ol><cite>, <span class="cit-pub-date">1975</span> <span class="cit-article-title">Inheritance ofsusceptibility and tolerance to leaf freckles and wilt (Corynebacterium nebraskense) of corn.</span> <abbr class="cit-jnl-abbrev">Maize Genet. Coop. Newsl.</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">137</span>–<span class="cit-lpage">138</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMaize%2BGenet.%2BCoop.%2BNewsl.%26rft.volume%253D49%26rft.spage%253D137%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3139.40" data-doi="10.1186/s12870-017-1137-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meyer</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. K.</span> <span class="cit-name-surname">Berger</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. A.</span> <span class="cit-name-surname">Christensen</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. L.</span> <span class="cit-name-surname">Murray</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">RNA-Seq analysis of resistant and susceptible sub-tropical maize lines reveals a role for kauralexins in resistance to grey leaf spot disease, caused by Cercospora zeina.</span> <abbr class="cit-jnl-abbrev">BMC Plant Biol.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">197</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12870-017-1137-9</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BPlant%2BBiol.%26rft.volume%253D17%26rft.spage%253D197%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12870-017-1137-9%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12870-017-1137-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3139.41" data-doi="10.1038/s41598-017-02298-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Miranda</span>,  <span class="cit-name-given-names">V. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W. F.</span> <span class="cit-name-surname">Porto</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G. D. R.</span> <span class="cit-name-surname">Fernandes</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Pogue</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. O.</span> <span class="cit-name-surname">Nolasco</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Comparative transcriptomic analysis indicates genes associated with local and systemic resistance to Colletotrichum graminicola in maize.</span> <abbr class="cit-jnl-abbrev">Sci. Rep.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">2483</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/s41598-017-02298-8</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BRep.%26rft.volume%253D7%26rft.spage%253D2483%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41598-017-02298-8%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41598-017-02298-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3139.42"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mueller</span>,  <span class="cit-name-given-names">D.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Wise</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> Corn disease loss estimates from the United States and Ontario, Canada. Purdue Ext. Publ. BP-96–12—W.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3139.43"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mueller</span>,  <span class="cit-name-given-names">D.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Wise</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> Corn disease loss estimates from the United States and Ontario, Canada. Purdue Ext. Publ. BP-96–14—W.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3139.44" data-doi="10.1094/PHP-RS-16-0030"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mueller</span>,  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Wise</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Sisson</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Allen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Bergstrom</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Corn yield loss estimates due to diseases in the United States and Ontario, Canada from 2012 to 2015.</span> <abbr class="cit-jnl-abbrev">Plant Health Prog.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">211</span>–<span class="cit-lpage">222</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1094/PHP-RS-16-0030</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BHealth%2BProg.%26rft.volume%253D17%26rft.spage%253D211%26rft.atitle%253DCORN%2BYIELD%2BLOSS%2BESTIMATES%2BDUE%2BTO%2BDISEASES%2BIN%2BTHE%2BUNITED%2BSTATES%2BAND%2BONTARIO%252C%2BCANADA%2BFROM%2B2012%2BTO%2B2015.%26rft_id%253Dinfo%253Adoi%252F10.1094%252FPHP-RS-16-0030%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1094/PHP-RS-16-0030&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3139.45" data-doi="10.1094/Phyto-82-142"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ngong-Nassah</span>,  <span class="cit-name-given-names">E. N.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. L.</span> <span class="cit-name-surname">Carson</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z. W.</span> <span class="cit-name-surname">Wicks</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">Inheritance of Resistance to Leaf Freckles and Wilt Caused by Clavibacter-Michiganense Subsp Nebraskense in Early Maturing Maize Inbred Lines.</span> <abbr class="cit-jnl-abbrev">Phytopathology</abbr> <span class="cit-vol">82</span>: <span class="cit-fpage">142</span>–<span class="cit-lpage">146</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1094/Phyto-82-142</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhytopathology%26rft.volume%253D82%26rft.spage%253D142%26rft_id%253Dinfo%253Adoi%252F10.1094%252FPhyto-82-142%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1094/Phyto-82-142&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3139.46" data-doi="10.1093/bioinformatics/btg412"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Paradis</span>,  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Claude</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Strimmer</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">APE: Analyses of Phylogenetics and Evolution in R language.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">289</span>–<span class="cit-lpage">290</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btg412</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DParadis%26rft.auinit1%253DE.%26rft.volume%253D20%26rft.issue%253D2%26rft.spage%253D289%26rft.epage%253D290%26rft.atitle%253DAPE%253A%2BAnalyses%2Bof%2BPhylogenetics%2Band%2BEvolution%2Bin%2BR%2Blanguage%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtg412%26rft_id%253Dinfo%253Apmid%252F14734327%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btg412&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14734327&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3139.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000188389700026&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3139.47" data-doi="10.1073/pnas.1010894108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>,  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. J.</span> <span class="cit-name-surname">Bradbury</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. S.</span> <span class="cit-name-surname">Buckler</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. J.</span> <span class="cit-name-surname">Nelson</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genome-wide nested association mapping of quantitative resistance to northern leaf blight in maize.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">6893</span>–<span class="cit-lpage">6898</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.1010894108</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1010894108%26rft_id%253Dinfo%253Apmid%252F21482771%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTA4LzE3LzY4OTMiO3M6NDoiYXRvbSI7czoxOToiL2dnZy85LzEwLzMxMzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3139.48" data-doi="10.1086/519795"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Purcell</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Neale</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Todd-Brown</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Thomas</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. A. R.</span> <span class="cit-name-surname">Ferreira</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">PLINK: a tool set for whole-genome association and population-based linkage analyses.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">81</span>: <span class="cit-fpage">559</span>–<span class="cit-lpage">575</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1086/519795</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DPurcell%26rft.auinit1%253DS.%26rft.volume%253D81%26rft.issue%253D3%26rft.spage%253D559%26rft.epage%253D575%26rft.atitle%253DPLINK%253A%2Ba%2Btool%2Bset%2Bfor%2Bwhole-genome%2Bassociation%2Band%2Bpopulation-based%2Blinkage%2Banalyses.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F519795%26rft_id%253Dinfo%253Apmid%252F17701901%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/519795&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17701901&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3139.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-confproc" id="cit-9.10.3139.49"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Robertson</span>,  <span class="cit-name-given-names">A. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> Goss’s wilt: a 2012 recap and looking ahead to 2013. Proc 2013 Wis. Crop Manag. Conf. 52.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3139.50"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rocheford</span>,  <span class="cit-name-given-names">T. R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. A.</span> <span class="cit-name-surname">Gardner</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. K.</span> <span class="cit-name-surname">Vidaver</span></span></li></ol><cite>, <span class="cit-pub-date">1985</span> <span class="cit-article-title">Chromosomal location of additional genes for resistance to Corynebacterium (Clavibacter) michiganense ssp. nebraskense.</span> <abbr class="cit-jnl-abbrev">Maize Genet. Coop. Newsl.</abbr> <span class="cit-vol">60</span>: <span class="cit-fpage">73</span>–<span class="cit-lpage">74</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMaize%2BGenet.%2BCoop.%2BNewsl.%26rft.volume%253D60%26rft.spage%253D73%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3139.51" data-doi="10.1093/oxfordjournals.jhered.a110873"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rocheford</span>,  <span class="cit-name-given-names">T. R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. O.</span> <span class="cit-name-surname">Gardner</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. K.</span> <span class="cit-name-surname">Vidaver</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">Genetic studies of resistance inmaize (Zea mays L.) to Goss’s bacterial wilt and blight (Clavibacter michiganensis subsp. nebraskensis).</span> <abbr class="cit-jnl-abbrev">J. Hered.</abbr> <span class="cit-vol">80</span>: <span class="cit-fpage">351</span>–<span class="cit-lpage">356</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/oxfordjournals.jhered.a110873</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BHeredity%26rft.stitle%253DJ%2BHered%26rft.aulast%253DRocheford%26rft.auinit1%253DT.%2BR.%26rft.volume%253D80%26rft.issue%253D5%26rft.spage%253D351%26rft.epage%253D356%26rft.atitle%253DGenetic%2BStudies%2Bof%2BResistance%2Bin%2BMaize%2B%2528Zea%2Bmays%2BL.%2529%2Bto%2BGoss%2527s%2BBacterial%2BWilt%2Band%2BBlight%2B%2528Clavibacter%2Bmichiganense%2Bssp.%2Bnebraskense%2529%26rft_id%253Dinfo%253Adoi%252F10.1093%252Foxfordjournals.jhered.a110873%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/oxfordjournals.jhered.a110873&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3139.52" data-doi="10.1186/gb-2013-14-6-r55"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Romay</span>,  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. J.</span> <span class="cit-name-surname">Millard</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. C.</span> <span class="cit-name-surname">Glaubitz</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. A.</span> <span class="cit-name-surname">Peiffer</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. L.</span> <span class="cit-name-surname">Swarts</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Comprehensive genotyping of the USA national maize inbred seed bank.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">R55</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/gb-2013-14-6-r55</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D14%26rft.spage%253DR55%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2013-14-6-r55%26rft_id%253Dinfo%253Apmid%252F23759205%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2013-14-6-r55&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23759205&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3139.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3139.53" data-doi="10.1094/PDIS-93-8-0841B"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ruhl</span>,  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Wise</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Creswell</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Leonberger</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Speers</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">First Report of Goss’s Bacterial Wilt and Leaf Blight on Corn Caused by Clavibacter michiganensis subsp. nebraskensis in Indiana.</span> <abbr class="cit-jnl-abbrev">Plant Dis.</abbr> <span class="cit-vol">93</span>: <span class="cit-fpage">841</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1094/PDIS-93-8-0841B</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BDis.%26rft.volume%253D93%26rft.spage%253D841%26rft_id%253Dinfo%253Adoi%252F10.1094%252FPDIS-93-8-0841B%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1094/PDIS-93-8-0841B&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3139.54" data-doi="10.2135/cropsci2005.08-0254"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Russell</span>,  <span class="cit-name-given-names">W. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Registration of N551 and N552 Parental Inbred Lines of Maize.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">1014</span>–<span class="cit-lpage">1015</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2005.08-0254</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DRussell%26rft.auinit1%253DW.K.%26rft.volume%253D46%26rft.issue%253D2%26rft.spage%253D1014%26rft.epage%253D1015%26rft.atitle%253DRegistration%2Bof%2BN551%2Band%2BN552%2BParental%2BInbred%2BLines%2Bof%2BMaize%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2005.08-0254%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2005.08-0254&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3139.55" data-doi="10.1093/oxfordjournals.molbev.a040454"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Saitou</span>,  <span class="cit-name-given-names">N.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Nei</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span> <span class="cit-article-title">The neighbor-joining method: a new method for reconstructing phylogenetic trees.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">406</span>–<span class="cit-lpage">425</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft.stitle%253DMol%2BBiol%2BEvol%26rft.aulast%253DSaitou%26rft.auinit1%253DN%26rft.volume%253D4%26rft.issue%253D4%26rft.spage%253D406%26rft.epage%253D425%26rft.atitle%253DThe%2Bneighbor-joining%2Bmethod%253A%2Ba%2Bnew%2Bmethod%2Bfor%2Breconstructing%2Bphylogenetic%2Btrees%26rft_id%253Dinfo%253Adoi%252F10.1093%252Foxfordjournals.molbev.a040454%26rft_id%253Dinfo%253Apmid%252F3447015%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/oxfordjournals.molbev.a040454&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=3447015&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3139.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1987J406700007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3139.56" data-doi="10.1371/journal.ppat.1001011"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sato</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Tsuda</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Wang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Coller</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Watanabe</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Network Modeling Reveals Prevalent Negative Regulatory Relationships between Signaling Sectors in Arabidopsis Immune Signaling.</span> <abbr class="cit-jnl-abbrev">PLoS Pathog.</abbr> <span class="cit-vol">6</span>: <span class="cit-elocation-id">e1001011</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.ppat.1001011</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BPathog%26rft.aulast%253DSato%26rft.auinit1%253DM.%26rft.volume%253D6%26rft.issue%253D7%26rft.spage%253De1001011%26rft.epage%253De1001011%26rft.atitle%253DNetwork%2Bmodeling%2Breveals%2Bprevalent%2Bnegative%2Bregulatory%2Brelationships%2Bbetween%2Bsignaling%2Bsectors%2Bin%2BArabidopsis%2Bimmune%2Bsignaling.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1001011%26rft_id%253Dinfo%253Apmid%252F20661428%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1001011&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20661428&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3139.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3139.57" data-doi="10.2135/cropsci2013.02.0121"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schaefer</span>,  <span class="cit-name-given-names">C. M.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Bernardo</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genomewide Association Mapping of Flowering Time, Kernel Composition, and Disease Resistance in Historical Minnesota Maize Inbreds.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">53</span>: <span class="cit-fpage">2518</span>–<span class="cit-lpage">2529</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2013.02.0121</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D53%26rft.spage%253D2518%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2013.02.0121%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2013.02.0121&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-confproc" id="cit-9.10.3139.58"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schuster</span>,  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Hoff</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Mandel</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Lazar</span></span></li></ol><cite>, <span class="cit-pub-date">1972</span> Leaf freckles and wilt, a new corn disease. 27th Ann Corn Sorghum Res. Congr Chic. Ill Dec12–14 176–191.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3139.59" data-doi="10.15252/embj.201488698"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Segonzac</span>,  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. P.</span> <span class="cit-name-surname">Macho</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Sanmartín</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Ntoukakis</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. J.</span> <span class="cit-name-surname">Sánchez-Serrano</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Negative control of BAK1 by protein phosphatase 2A during plant innate immunity.</span> <abbr class="cit-jnl-abbrev">EMBO J.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">2069</span>–<span class="cit-lpage">2079</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.15252/embj.201488698</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEMBO%2BJ.%26rft.volume%253D33%26rft.spage%253D2069%26rft_id%253Dinfo%253Adoi%252F10.15252%252Fembj.201488698%26rft_id%253Dinfo%253Apmid%252F25085430%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.15252/embj.201488698&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25085430&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3139.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3139.60" data-doi="10.2135/cropsci2015.09.0543"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Singh</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. P.</span> <span class="cit-name-surname">Andersen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. A.</span> <span class="cit-name-surname">Jackson-Zeims</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. J.</span> <span class="cit-name-surname">Lorenz</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Mapping quantitative trait loci for resistance to Goss’s bacterial wilt and leaf blight in North American Maize by joint linkage analysis.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">56</span>: <span class="cit-fpage">2306</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2015.09.0543</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D56%26rft.spage%253D2306%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2015.09.0543%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2015.09.0543&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3139.61" data-doi="10.1094/PDIS-08-14-0807-PDN"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Singh</span>,  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Hollier</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Burks</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Frazier</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">First Report of Goss’s Wilt of Corn Caused by Clavibacter michiganensis subsp. nebraskensis in Louisiana.</span> <abbr class="cit-jnl-abbrev">Plant Dis.</abbr> <span class="cit-vol">99</span>: <span class="cit-fpage">1268</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1094/PDIS-08-14-0807-PDN</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BDis.%26rft.volume%253D99%26rft.spage%253D1268%26rft_id%253Dinfo%253Adoi%252F10.1094%252FPDIS-08-14-0807-PDN%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1094/PDIS-08-14-0807-PDN&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3139.62" data-doi="10.1073/pnas.1530509100"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Storey</span>,  <span class="cit-name-given-names">J. D.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Tibshirani</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Statistical significance for genomewide studies.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">100</span>: <span class="cit-fpage">9440</span>–<span class="cit-lpage">9445</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.1530509100</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1530509100%26rft_id%253Dinfo%253Apmid%252F12883005%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTAwLzE2Lzk0NDAiO3M6NDoiYXRvbSI7czoxOToiL2dnZy85LzEwLzMxMzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3139.63"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Swarts</span>,  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Li</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Navarro</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Alberto</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">An</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Novel methods to optimize genotypic imputation for low-coverage, next-generation sequence data in crop plants.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">12</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D7%26rft.spage%253D12%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3139.64" data-doi="10.21273/JASHS.115.4.672"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Treat</span>,  <span class="cit-name-given-names">C. L.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">W. F.</span> <span class="cit-name-surname">Tracy</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">Inheritance of resistance to Goss’s wilt in sweet corn.</span> <abbr class="cit-jnl-abbrev">J. Am. Soc. Hortic. Sci.</abbr> <span class="cit-vol">115</span>: <span class="cit-fpage">672</span>–<span class="cit-lpage">674</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.21273/JASHS.115.4.672</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAm.%2BSoc.%2BHortic.%2BSci.%26rft.volume%253D115%26rft.spage%253D672%26rft_id%253Dinfo%253Adoi%252F10.21273%252FJASHS.115.4.672%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.21273/JASHS.115.4.672&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3139.65" data-doi="10.2135/cropsci1990.0011183X003000040027x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Treat</span>,  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W. F.</span> <span class="cit-name-surname">Tracy</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. N.</span> <span class="cit-name-surname">Drolsom</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. G.</span> <span class="cit-name-surname">Coors</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">Inheritance of Resistance to Gosss Wilt in Maize.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">893</span>–<span class="cit-lpage">896</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci1990.0011183X003000040027x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DTreat%26rft.auinit1%253DC.%2BL.%26rft.volume%253D30%26rft.issue%253D4%26rft.spage%253D893%26rft.epage%253D896%26rft.atitle%253DInheritance%2Bof%2BResistance%2Bto%2BGoss%2527s%2BWilt%2Bin%2BMaize%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci1990.0011183X003000040027x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci1990.0011183X003000040027x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3139.66" data-doi="10.1086/344398"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">Akey</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Zhang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Chakraborty</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Jin</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Distribution of Recombination Crossovers and the Origin of Haplotype Blocks: The Interplay of Population History, Recombination, and Mutation.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">71</span>: <span class="cit-fpage">1227</span>–<span class="cit-lpage">1234</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1086/344398</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DWang%26rft.auinit1%253DN.%26rft.volume%253D71%26rft.issue%253D5%26rft.spage%253D1227%26rft.epage%253D1234%26rft.atitle%253DDistribution%2Bof%2Brecombination%2Bcrossovers%2Band%2Bthe%2Borigin%2Bof%2Bhaplotype%2Bblocks%253A%2Bthe%2Binterplay%2Bof%2Bpopulation%2Bhistory%252C%2Brecombination%252C%2Band%2Bmutation.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F344398%26rft_id%253Dinfo%253Apmid%252F12384857%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/344398&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12384857&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3139.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000178884300023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3139.67" data-doi="10.1139/gen-2017-0226"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Zhang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Du</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Pan</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Ma</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Combined analysis of genome-wide expression profiling of maize (Zea mays L.) leaves infected with Ustilago maydis.</span> <abbr class="cit-jnl-abbrev">Genome</abbr> <span class="cit-vol">61</span>: <span class="cit-fpage">505</span>–<span class="cit-lpage">513</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1139/gen-2017-0226</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%26rft.volume%253D61%26rft.spage%253D505%26rft_id%253Dinfo%253Adoi%252F10.1139%252Fgen-2017-0226%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1139/gen-2017-0226&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-confproc" id="cit-9.10.3139.68"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wysong</span>,  <span class="cit-name-given-names">D. S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Doupnik</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Lane</span></span></li></ol><cite>, <span class="cit-pub-date">1981</span> Goss’s wilt and corn lethal necrosis - can they become a major problem? Proc. Annu. Corn Sorghum Ind. Res. Conf. 36: 104–152.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3139.69" data-doi="10.1038/ng.608"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Benyamin</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. P.</span> <span class="cit-name-surname">McEvoy</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Gordon</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. K.</span> <span class="cit-name-surname">Henders</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Common SNPs explain a large proportion of the heritability for human height.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">565</span>–<span class="cit-lpage">569</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.608</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DYang%26rft.auinit1%253DJ.%26rft.volume%253D42%26rft.issue%253D7%26rft.spage%253D565%26rft.epage%253D569%26rft.atitle%253DCommon%2BSNPs%2Bexplain%2Ba%2Blarge%2Bproportion%2Bof%2Bthe%2Bheritability%2Bfor%2Bhuman%2Bheight.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.608%26rft_id%253Dinfo%253Apmid%252F20562875%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.608&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20562875&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3139.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000279242400007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3139.70" data-doi="10.1186/s12864-018-5072-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>,  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Shi</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Li</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Liu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Q.</span> <span class="cit-name-surname">Geng</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Transcriptome analysis reveals the molecular mechanisms of the defense response to gray leaf spot disease in maize.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">742</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12864-018-5072-4</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D19%26rft.spage%253D742%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864-018-5072-4%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12864-018-5072-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3139.71"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Zheng</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Li</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Li</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Liu</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> Coexpression network analysis of the genes regulated by two types of resistance responses to powdery mildew in wheat. Sci. Rep. 6: srep23805.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/srep23805</span></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference  in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3139.72" data-doi="10.1186/1471-2164-14-27"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zheng</span>,  <span class="cit-name-given-names">Z.-L.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Zhao</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Transcriptome comparison and gene coexpression network analysis provide a systems view of citrus response to ‘CandidatusLiberibacter asiaticus’ infection.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">27</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2164-14-27</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D14%26rft.spage%253D27%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-14-27%26rft_id%253Dinfo%253Apmid%252F23324561%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-14-27&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23324561&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3139.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference  in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3139.73" data-doi="10.1534/g3.113.007328"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zila</span>,  <span class="cit-name-given-names">C. T.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L. F.</span> <span class="cit-name-surname">Samayoa</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Santiago</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Butrón</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. B.</span> <span class="cit-name-surname">Holland</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">A genome-wide association study reveals genes associated with fusarium ear rot resistance in a maize core diversity panel.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">2095</span>–<span class="cit-lpage">2104</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.113.007328</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.113.007328%26rft_id%253Dinfo%253Apmid%252F24048647%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjk6IjMvMTEvMjA5NSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzkvMTAvMzEzOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/9/10/3139.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/10/3129" title="QTG-Finder: A Machine-Learning Based Algorithm To Prioritize Causal Genes of Quantitative Trait Loci in Arabidopsis and Rice" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/10/3153" title="Multi-trait Improvement by Predicting Genetic Correlations in Breeding Crosses" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/9/10"><span>Volume 9 Issue 10, October 2019</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/9/10.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/9/10.cover-source.jpg?itok=yA8kifEd" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 9 (10)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/10/3139" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-9v9y52_hrAEKKDThmIVQl8tE84CjWQ4TE5i-yKdyLQQ" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/9/10/3139" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/9/10/3139" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/9/10/3139" class="active" data-icon-position="" data-hide-link-title="0">Genome-Wide Association and Gene Co-expression Network Analyses Reveal Complex Genetics of Resistance to Goss’s Wilt of Maize</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/494277" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-nVrvxvPN4SbCx0H6iZqT0mTPI5tl_IoU-SSeBSTEWSM" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/10/3139.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/9/10/3139" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-5kQ6ujDCp0j1nmP_rchVXEc3K8z9LUYTafuIGTaNGAI" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/9/10/3139" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/10/3139.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494277" id="node494277--2" data-pisa="ggg;9/10/3139" data-pisa-master="ggg;g3.119.400347" data-apath="/ggg/9/10/3139.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genome-Wide Association and Gene Co-expression Network Analyses Reveal Complex Genetics of Resistance to Goss’s Wilt of Maize</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Amritpal Singh</span>, <span class="highwire-citation-author" data-delta="1">Guangyong Li</span>, <span class="highwire-citation-author" data-delta="2">Alex B. Brohammer</span>, <span class="highwire-citation-author" data-delta="3">Diego Jarquin</span>, <span class="highwire-citation-author" data-delta="4">Candice N. Hirsch</span>, <span class="highwire-citation-author" data-delta="5">James R. Alfano</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0002-4361-1683" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Aaron J. Lorenz</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3139-3152;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400347 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494277--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Amritpal Singh </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy and Plant Genetics, University of Minnesota, St. Paul, MN 55108</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Amritpal%2BSingh%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Singh%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAmritpal%2BSingh%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Guangyong Li </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Plant Pathology, University of Nebraska, Lincoln, NE 68503-0722</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Center for Plant Science Innovation, University of Nebraska, Lincoln, NE 68588-0660, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Guangyong%2BLi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Li%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGuangyong%2BLi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Alex B. Brohammer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy and Plant Genetics, University of Minnesota, St. Paul, MN 55108</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alex%2BB.%2BBrohammer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Brohammer%20AB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlex%2BB.%2BBrohammer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Diego Jarquin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Agronomy and Horticulture, University of Nebraska, Lincoln, NE 68503-0915</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Diego%2BJarquin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jarquin%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADiego%2BJarquin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Candice N. Hirsch </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy and Plant Genetics, University of Minnesota, St. Paul, MN 55108</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Candice%2BN.%2BHirsch%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hirsch%20CN&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACandice%2BN.%2BHirsch%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">James R. Alfano </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Plant Pathology, University of Nebraska, Lincoln, NE 68503-0722</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Center for Plant Science Innovation, University of Nebraska, Lincoln, NE 68588-0660, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=James%2BR.%2BAlfano%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Alfano%20JR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJames%2BR.%2BAlfano%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Aaron J. Lorenz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy and Plant Genetics, University of Minnesota, St. Paul, MN 55108</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Aaron%2BJ.%2BLorenz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lorenz%20AJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAaron%2BJ.%2BLorenz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-4361-1683" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Aaron J. Lorenz</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:lore0149@umn.edu" class="" data-icon-position="" data-hide-link-title="0">lore0149@umn.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/9/10/3139" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/9/10/3139&amp;title=Genome-Wide%20Association%20and%20Gene%20Co-expression%20Network%20Analyses%20Reveal%20Complex%20Genetics%20of%20Resistance%20to%20Goss%E2%80%99s%20Wilt%20of%20Maize" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/9/10/3139&amp;title=Genome-Wide%20Association%20and%20Gene%20Co-expression%20Network%20Analyses%20Reveal%20Complex%20Genetics%20of%20Resistance%20to%20Goss%E2%80%99s%20Wilt%20of%20Maize" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/9/10/3139&amp;title=Genome-Wide%20Association%20and%20Gene%20Co-expression%20Network%20Analyses%20Reveal%20Complex%20Genetics%20of%20Resistance%20to%20Goss%E2%80%99s%20Wilt%20of%20Maize" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/9/10/3139&amp;text=Genome-Wide%20Association%20and%20Gene%20Co-expression%20Network%20Analyses%20Reveal%20Complex%20Genetics%20of%20Resistance%20to%20Goss%E2%80%99s%20Wilt%20of%20Maize" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/9/10/3139&amp;title=Genome-Wide%20Association%20and%20Gene%20Co-expression%20Network%20Analyses%20Reveal%20Complex%20Genetics%20of%20Resistance%20to%20Goss%E2%80%99s%20Wilt%20of%20Maize" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/9/10/3139&amp;t=Genome-Wide%20Association%20and%20Gene%20Co-expression%20Network%20Analyses%20Reveal%20Complex%20Genetics%20of%20Resistance%20to%20Goss%E2%80%99s%20Wilt%20of%20Maize" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/9/10/3139&amp;title=Genome-Wide%20Association%20and%20Gene%20Co-expression%20Network%20Analyses%20Reveal%20Complex%20Genetics%20of%20Resistance%20to%20Goss%E2%80%99s%20Wilt%20of%20Maize" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/9/10/3139&amp;title=Genome-Wide%20Association%20and%20Gene%20Co-expression%20Network%20Analyses%20Reveal%20Complex%20Genetics%20of%20Resistance%20to%20Goss%E2%80%99s%20Wilt%20of%20Maize" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/494277/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494277" id="node494277--3" data-pisa="ggg;9/10/3139" data-pisa-master="ggg;g3.119.400347" data-apath="/ggg/9/10/3139.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genome-Wide Association and Gene Co-expression Network Analyses Reveal Complex Genetics of Resistance to Goss’s Wilt of Maize</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Amritpal Singh</span>, <span class="highwire-citation-author" data-delta="1">Guangyong Li</span>, <span class="highwire-citation-author" data-delta="2">Alex B. Brohammer</span>, <span class="highwire-citation-author" data-delta="3">Diego Jarquin</span>, <span class="highwire-citation-author" data-delta="4">Candice N. Hirsch</span>, <span class="highwire-citation-author" data-delta="5">James R. Alfano</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0002-4361-1683" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Aaron J. Lorenz</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3139-3152;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400347 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494277--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Amritpal Singh </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy and Plant Genetics, University of Minnesota, St. Paul, MN 55108</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Amritpal%2BSingh%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Singh%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAmritpal%2BSingh%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Guangyong Li </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Plant Pathology, University of Nebraska, Lincoln, NE 68503-0722</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Center for Plant Science Innovation, University of Nebraska, Lincoln, NE 68588-0660, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Guangyong%2BLi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Li%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGuangyong%2BLi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Alex B. Brohammer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy and Plant Genetics, University of Minnesota, St. Paul, MN 55108</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alex%2BB.%2BBrohammer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Brohammer%20AB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlex%2BB.%2BBrohammer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Diego Jarquin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Agronomy and Horticulture, University of Nebraska, Lincoln, NE 68503-0915</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Diego%2BJarquin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jarquin%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADiego%2BJarquin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Candice N. Hirsch </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy and Plant Genetics, University of Minnesota, St. Paul, MN 55108</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Candice%2BN.%2BHirsch%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hirsch%20CN&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACandice%2BN.%2BHirsch%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">James R. Alfano </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Plant Pathology, University of Nebraska, Lincoln, NE 68503-0722</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Center for Plant Science Innovation, University of Nebraska, Lincoln, NE 68588-0660, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=James%2BR.%2BAlfano%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Alfano%20JR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJames%2BR.%2BAlfano%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Aaron J. Lorenz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy and Plant Genetics, University of Minnesota, St. Paul, MN 55108</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Aaron%2BJ.%2BLorenz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lorenz%20AJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAaron%2BJ.%2BLorenz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-4361-1683" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Aaron J. Lorenz</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:lore0149@umn.edu" class="" data-icon-position="" data-hide-link-title="0">lore0149@umn.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=3139&amp;rft.epage=3152&amp;rft.atitle=Genome-Wide%20Association%20and%20Gene%20Co-expression%20Network%20Analyses%20Reveal%20Complex%20Genetics%20of%20Resistance%20to%20Goss%E2%80%99s%20Wilt%20of%20Maize&amp;rft.volume=9&amp;rft.issue=10&amp;rft.date=2019-10-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Singh%2C+Amritpal&amp;rft.au=Li%2C+Guangyong&amp;rft.au=Brohammer%2C+Alex+B.&amp;rft.au=Jarquin%2C+Diego&amp;rft.au=Hirsch%2C+Candice+N.&amp;rft.au=Alfano%2C+James+R.&amp;rft.au=Lorenz%2C+Aaron+J."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/494277/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/494277/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/494277/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/494277/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/494277/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/494277/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/494277/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/494277/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/494277/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/494277/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/494277/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/494277/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/9/10/3139" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-14" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-20" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/9/10/3139.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/9/10/3139.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/9/10/3139.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
