<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/9/10/3153" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/9/10/3153.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/9/10/3153.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/9/10/3153.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/9/10/3153/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/9/10/3153.atom" />
<meta name="HW.pisa" content="ggg;9/10/3153" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Multi-trait Improvement by Predicting Genetic Correlations in Breeding Crosses" />
<meta name="DC.Identifier" content="10.1534/g3.119.400406" />
<meta name="DC.Date" content="2019-10-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2019 Neyhart et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="The many quantitative traits of interest to plant breeders are often genetically correlated, which can complicate progress from selection. Improving multiple traits may be enhanced by identifying parent combinations – an important breeding step – that will deliver more favorable genetic correlations ( rG ). Modeling the segregation of genomewide markers with estimated effects may be one method of predicting rG in a cross, but this approach remains untested. Our objectives were to: (i) use simulations to assess the accuracy of genomewide predictions of rG and the long-term response to selection when selecting crosses on the basis of such predictions; and (ii) empirically measure the ability to predict genetic correlations using data from a barley ( Hordeum vulgare L.) breeding program. Using simulations, we found that the accuracy to predict rG was generally moderate and influenced by trait heritability, population size, and genetic correlation architecture ( i.e. , pleiotropy or linkage disequilibrium). Among 26 barley breeding populations, the empirical prediction accuracy of rG was low (-0.012) to moderate (0.42), depending on trait complexity. Within a simulated plant breeding program employing indirect selection, choosing crosses based on predicted rG increased multi-trait genetic gain by 11–27% compared to selection on the predicted cross mean. Importantly, when the starting genetic correlation was negative, such cross selection mitigated or prevented an unfavorable response in the trait under indirect selection. Prioritizing crosses based on predicted genetic correlation can be a feasible and effective method of improving unfavorably correlated traits in breeding programs." />
<meta name="DC.Contributor" content="Jeffrey L. Neyhart" />
<meta name="DC.Contributor" content="Aaron J. Lorenz" />
<meta name="DC.Contributor" content="Kevin P. Smith" />
<meta name="article:published_time" content="2019-10-01" />
<meta name="article:section" content="Genomic Prediction" />
<meta name="citation_title" content="Multi-trait Improvement by Predicting Genetic Correlations in Breeding Crosses" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;The many quantitative traits of interest to plant breeders are often genetically correlated, which can complicate progress from selection. Improving multiple traits may be enhanced by identifying parent combinations – an important breeding step – that will deliver more favorable genetic correlations (&lt;i&gt;r&lt;sub&gt;G&lt;/sub&gt;&lt;/i&gt;). Modeling the segregation of genomewide markers with estimated effects may be one method of predicting &lt;i&gt;r&lt;sub&gt;G&lt;/sub&gt;&lt;/i&gt; in a cross, but this approach remains untested. Our objectives were to: (i) use simulations to assess the accuracy of genomewide predictions of &lt;i&gt;r&lt;sub&gt;G&lt;/sub&gt;&lt;/i&gt; and the long-term response to selection when selecting crosses on the basis of such predictions; and (ii) empirically measure the ability to predict genetic correlations using data from a barley (&lt;i&gt;Hordeum vulgare&lt;/i&gt; L.) breeding program. Using simulations, we found that the accuracy to predict &lt;i&gt;r&lt;sub&gt;G&lt;/sub&gt;&lt;/i&gt; was generally moderate and influenced by trait heritability, population size, and genetic correlation architecture (&lt;i&gt;i.e.&lt;/i&gt;, pleiotropy or linkage disequilibrium). Among 26 barley breeding populations, the empirical prediction accuracy of &lt;i&gt;r&lt;sub&gt;G&lt;/sub&gt;&lt;/i&gt; was low (-0.012) to moderate (0.42), depending on trait complexity. Within a simulated plant breeding program employing indirect selection, choosing crosses based on predicted &lt;i&gt;r&lt;sub&gt;G&lt;/sub&gt;&lt;/i&gt; increased multi-trait genetic gain by 11–27% compared to selection on the predicted cross mean. Importantly, when the starting genetic correlation was negative, such cross selection mitigated or prevented an unfavorable response in the trait under indirect selection. Prioritizing crosses based on predicted genetic correlation can be a feasible and effective method of improving unfavorably correlated traits in breeding programs.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2019/10/01" />
<meta name="citation_mjid" content="ggg;9/10/3153" />
<meta name="citation_id" content="9/10/3153" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/9/10/3153" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/9/10/3153.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/9/10/3153.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/9/10/3153.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.119.400406" />
<meta name="citation_pmid" content="31358561" />
<meta name="citation_volume" content="9" />
<meta name="citation_issue" content="10" />
<meta name="citation_num_pages" content="13" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Genomic Prediction" />
<meta name="citation_firstpage" content="3153" />
<meta name="citation_lastpage" content="3165" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Jeffrey L. Neyhart" />
<meta name="citation_author_institution" content="Department of Agronomy and Plant Genetics, University of Minnesota, St. Paul, MN 55108" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-1991-5310" />
<meta name="citation_author" content="Aaron J. Lorenz" />
<meta name="citation_author_institution" content="Department of Agronomy and Plant Genetics, University of Minnesota, St. Paul, MN 55108" />
<meta name="citation_author" content="Kevin P. Smith" />
<meta name="citation_author_institution" content="Department of Agronomy and Plant Genetics, University of Minnesota, St. Paul, MN 55108" />
<meta name="citation_author_email" content="smith376@umn.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-8253-3326" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=E. Adeyemo;citation_author=R. Bernardo;citation_title=Predicting Genetic Variance from Genomewide Marker Effects Estimated from a Diverse Panel of Maize Inbreds.;citation_pages=583-590;citation_volume=59;citation_year=2019;citation_doi=10.2135/cropsci2018.08.0525" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=D. Akdemir;citation_author=W. Beavis;citation_author=R. Fritsche-Neto;citation_author=A. K. Singh;citation_author=J. Isidro-Sánchez;citation_title=Multi-objective optimized genomic breeding strategies for sustainable food improvement.;citation_volume=122;citation_year=2019;citation_doi=10.1038/s41437-018-0147-1" />
<meta name="citation_reference" content="citation_year=2019" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=N. Bandillo;citation_author=D. Jarquin;citation_author=Q. Song;citation_author=R. Nelson;citation_author=P. Cregan;citation_title=A Population Structure and Genome-Wide Association Analysis on the USDA Soybean Germplasm Collection.;citation_volume=8;citation_year=2015;citation_doi=10.3835/plantgenome2015.04.0024" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=G. L. Bennett;citation_author=L. A. Swiger;citation_title=GENETIC VARIANCE AND CORRELATION AFTER SELECTION FOR TWO TRAITS BY INDEX, INDEPENDENT CULLING LEVELS AND EXTREME SELECTION;citation_pages=763-775;citation_volume=94;citation_year=1980;citation_issue=3;citation_pmid=7399261" />
<meta name="citation_reference" content="citation_title=Breeding for Quantitative Traits in Plants;citation_year=2010" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=R. Bernardo;citation_title=Genomewide Selection of Parental Inbreds: Classes of Loci and Virtual Biparental Populations.;citation_pages=2586;citation_volume=54;citation_year=2014;citation_doi=10.2135/cropsci2014.01.0088" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=Y. Beyene;citation_author=K. Semagn;citation_author=S. Mugo;citation_author=A. Tarekegne;citation_author=R. Babu;citation_title=Genetic Gains in Grain Yield Through Genomic Selection in Eight Bi-parental Maize Populations under Drought Stress.;citation_pages=154-163;citation_volume=55;citation_year=2015;citation_doi=10.2135/cropsci2014.07.0460" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=V. C. Blake;citation_author=C. Birkett;citation_author=D. E. Matthews;citation_author=D. L. Hane;citation_author=P. Bradbury;citation_title=The Triticeae Toolbox: Combining Phenotype and Genotype Data to Advance Small-Grains Breeding.;citation_pages=1-10;citation_volume=9;citation_year=2016;citation_doi=10.3835/plantgenome2016.05.0047" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=M. Bohn;citation_author=H. F. Utz;citation_author=A. E. Melchinger;citation_title=Genetic Similarities among Winter Wheat Cultivars Determined on the Basis of RFLPs, AFLPs, and SSRs and Their Use for Predicting Progeny Variance;citation_pages=228-237;citation_volume=39;citation_year=1999;citation_issue=1;citation_doi=10.2135/cropsci1999.0011183X003900010035x" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=B. B. Bohren;citation_author=W. G. Hill;citation_author=A. Robertson;citation_title=Some observations on asymmetrical correlated responses to selection.;citation_pages=44-57;citation_volume=7;citation_year=1966;citation_issue=1;citation_pmid=5906491;citation_doi=10.1017/S0016672300009460" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_author=E. A. Boyle;citation_author=Y. I. Li;citation_author=J. K. Pritchard;citation_title=An Expanded View of Complex Traits : From Polygenic to Omnigenic.;citation_pages=1177-1186;citation_volume=169;citation_year=2017;citation_pmid=28622505;citation_doi=10.1016/j.cell.2017.05.038" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_journal_abbrev=Bioinformatics;citation_author=K. W. Broman;citation_author=H. Wu;citation_author=S. Sen;citation_author=G. A. Churchill;citation_title=R/qtl: QTL mapping in experimental crosses;citation_pages=889-890;citation_volume=19;citation_year=2003;citation_issue=7;citation_pmid=12724300;citation_doi=10.1093/bioinformatics/btg112" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=B. Bulik-Sullivan;citation_author=H. K. Finucane;citation_author=V. Anttila;citation_author=A. Gusev;citation_author=F. R. Day;citation_title=An atlas of genetic correlations across human diseases and traits.;citation_pages=1236-1241;citation_volume=47;citation_year=2015;citation_pmid=26414676;citation_doi=10.1038/ng.3406" />
<meta name="citation_reference" content="citation_journal_title=Am. Nat.;citation_author=M. G. Bulmer;citation_title=The Effect of Selection on Genetic Variability.;citation_pages=201-211;citation_volume=105;citation_year=1971;citation_doi=10.1086/282718" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=M. P. Calus;citation_author=R. F. Veerkamp;citation_title=Accuracy of multi-trait genomic selection using different methods.;citation_pages=26-26;citation_volume=43;citation_year=2011;citation_pmid=21729282;citation_doi=10.1186/1297-9686-43-26" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=S. Chao;citation_author=J. Dubcovsky;citation_author=J. Dvořák;citation_author=M.-C. Luo;citation_author=S. P. Baenziger;citation_title=Population- and genome-specific patterns of linkage disequilibrium and SNP variation in spring and winter wheat (Triticum aestivum L.).;citation_pages=1-17;citation_volume=11;citation_year=2011" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=R. K. Chi;citation_author=S. A. Eberhart;citation_author=L. H. Penny;citation_title=COVARIANCES AMONG RELATIVES IN A MAIZE VARIETY (ZEA MAYS  L.);citation_pages=511-520;citation_volume=63;citation_year=1969;citation_issue=2;citation_pmid=17248468" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=T. J. Close;citation_author=P. R. Bhat;citation_author=S. Lonardi;citation_author=Y. Wu;citation_author=N. Rostoks;citation_title=Development and implementation of high-throughput SNP genotyping in barley.;citation_pages=582-582;citation_volume=10;citation_year=2009;citation_pmid=19961604;citation_doi=10.1186/1471-2164-10-582" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=E. Combs;citation_author=R. Bernardo;citation_title=Genomewide selection to introgress semidwarf maize germplasm into US Corn Belt inbreds.;citation_pages=1427-1436;citation_volume=53;citation_year=2013;citation_doi=10.2135/cropsci2012.11.0666" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=G. Covarrubias-Pazaran;citation_title=Genome-Assisted prediction of quantitative traits using the r package sommer.;citation_pages=e0156744;citation_volume=11;citation_year=2016;citation_doi=10.1371/journal.pone.0156744" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=H. D. Daetwyler;citation_author=B. Villanueva;citation_author=J. A. Woolliams;citation_title=Accuracy of predicting the genetic risk of disease using a genome-wide approach.;citation_pages=e3395-e3395;citation_volume=3;citation_year=2008;citation_issue=10;citation_pmid=18852893;citation_doi=10.1371/journal.pone.0003395" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=H. D. Daetwyler;citation_author=R. Pong-Wong;citation_author=B. Villanueva;citation_author=J. Woolliams;citation_title=The impact of genetic architecture on genome-wide evaluation methods.;citation_volume=185;citation_year=2010;citation_pmid=20407128;citation_doi=10.1534/genetics.110.116855" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=Y. Deng;citation_author=W. Pan;citation_title=Testing Genetic Pleiotropy with GWAS Summary Statistics for Marginal and Conditional Analyses.;citation_volume=207;citation_year=2017;citation_pmid=28971959;citation_doi=10.1534/genetics.117.300347" />
<meta name="citation_reference" content="citation_title=Introduction to Quantitative Genetics;citation_year=1996" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. Felsenstein;citation_title=THE EFFECT OF LINKAGE ON DIRECTIONAL SELECTION;citation_pages=349-363;citation_volume=52;citation_year=1965;citation_issue=2;citation_pmid=5861564" />
<meta name="citation_reference" content="citation_journal_title=Trans. R. Soc. Edinb.;citation_author=R. A. Fisher;citation_title=The Correlation between Relatives on the Supposition of Mendelian Inheritance.;citation_pages=399-433;citation_volume=52;citation_year=1919;citation_doi=10.1017/S0080456800012163" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=D. Habier;citation_author=R. L. Fernando;citation_author=K. Kizilkaya;citation_author=D. J. Garrick;citation_title=Extension of the bayesian alphabet for genomic selection.;citation_pages=186-186;citation_volume=12;citation_year=2011;citation_pmid=21605355;citation_doi=10.1186/1471-2105-12-186" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=M. T. Hamblin;citation_author=T. J. Close;citation_author=P. R. Bhat;citation_author=S. Chao;citation_author=J. G. Kling;citation_title=Population Structure and Linkage Disequilibrium in U.S. Barley Germplasm: Implications for Association Mapping;citation_pages=556-566;citation_volume=50;citation_year=2010;citation_issue=2;citation_doi=10.2135/cropsci2009.04.0198" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=J.-L. Jannink;citation_title=Dynamics of long-term genomic selection.;citation_pages=35-35;citation_volume=42;citation_year=2010;citation_pmid=20712894;citation_doi=10.1186/1297-9686-42-35" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=Y. Jia;citation_author=J.-L. Jannink;citation_title=Multiple-Trait Genomic Selection Methods Increase Genetic Value Prediction Accuracy.;citation_volume=192;citation_year=2012;citation_pmid=23086217;citation_doi=10.1534/genetics.112.144246" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=A. Korte;citation_author=B. J. Vilhjálmsson;citation_author=V. Segura;citation_author=A. Platt;citation_author=Q. Long;citation_title=A mixed-model approach for genome-wide association studies of correlated traits in structured populations.;citation_pages=1066-1071;citation_volume=44;citation_year=2012;citation_issue=9;citation_pmid=22902788;citation_doi=10.1038/ng.2376" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=R. Lande;citation_title=The genetic correlation between characters maintained by selection, linkage and inbreeding.;citation_pages=309-320;citation_volume=44;citation_year=1984;citation_issue=3;citation_pmid=6530140;citation_doi=10.1017/S0016672300026549" />
<meta name="citation_reference" content="citation_title=Efficiency of marker-assisted selection in the improvement of quantitative traits. Genetics;citation_pages=743-756;citation_volume=124;citation_year=1990" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=S. H. Lee;citation_author=J. Yang;citation_author=M. E. Goddard;citation_author=P. M. Visscher;citation_author=N. R. Wray;citation_title=Estimation of pleiotropy between complex diseases using single-nucleotide polymorphism-derived genomic relationships and restricted maximum likelihood.;citation_volume=28;citation_year=2012;citation_pmid=22843982;citation_doi=10.1093/bioinformatics/bts474" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=C. Lehermeier;citation_author=S. Teyssèdre;citation_author=C.-C. Schön;citation_title=Genetic Gain Increases by Applying the Usefulness Criterion with Improved Variance Prediction in Selection of Crosses.;citation_volume=207;citation_year=2017;citation_pmid=29038144;citation_doi=10.1534/genetics.117.300403" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=L. Lian;citation_author=A. Jacobson;citation_author=S. Zhong;citation_author=R. Bernardo;citation_title=Prediction of genetic variance in biparental maize populations: Genomewide marker effects vs. mean genetic variance in prior populations.;citation_pages=1181-1188;citation_volume=55;citation_year=2015;citation_doi=10.2135/cropsci2014.10.0729" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=A. J. Lorenz;citation_author=K. P. Smith;citation_author=J.-L. Jannink;citation_title=Potential and Optimization of Genomic Selection for Fusarium Head Blight Resistance in Six-Row Barley.;citation_pages=1609-1621;citation_volume=52;citation_year=2012;citation_doi=10.2135/cropsci2011.09.0503" />
<meta name="citation_reference" content="citation_title=Genetics and Analysis of Quantitative Traits;citation_year=1998" />
<meta name="citation_reference" content="citation_journal_title=Philosophical Transactions of the Royal Society B: Biological Sciences;citation_journal_abbrev=Phil Trans R Soc B;citation_author=T. F. C. Mackay;citation_title=Mutations and quantitative genetic variation: lessons from Drosophila;citation_pages=1229-1239;citation_volume=365;citation_year=2010;citation_issue=1544;citation_pmid=20308098;citation_doi=10.1098/rstb.2009.0315" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=T. H. E. Meuwissen;citation_author=B. J. Hayes;citation_author=M. E. Goddard;citation_title=Prediction of Total Genetic Value Using Genome-Wide Dense Marker Maps;citation_pages=1819-1829;citation_volume=157;citation_year=2001;citation_issue=4;citation_pmid=11290733;citation_doi=10.1093/nar/gkv1344" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=M. Mohammadi;citation_author=T. Tiede;citation_author=K. P. Smith;citation_title=PopVar: A Genome-Wide Procedure for Predicting Genetic Variance and Correlated Response in Biparental Breeding Populations.;citation_pages=2068-2077;citation_volume=55;citation_year=2015;citation_doi=10.2135/cropsci2015.01.0030" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome J.;citation_author=M. Muñoz-Amatriaín;citation_author=M. J. Moscou;citation_author=P. R. Bhat;citation_author=J. T. Svensson;citation_author=J. Bartoš;citation_title=An Improved Consensus Linkage Map of Barley Based on Flow-Sorted Chromosomes and Single Nucleotide Polymorphism Markers.;citation_pages=238-249;citation_volume=4;citation_year=2011;citation_doi=10.3835/plantgenome2011.08.0023" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=J. L. Neyhart;citation_author=K. P. Smith;citation_title=Validating Genomewide Predictions of Genetic Variance in a Contemporary Breeding Program.;citation_pages=1062;citation_volume=59;citation_year=2019;citation_doi=10.2135/cropsci2018.11.0716" />
<meta name="citation_reference" content="citation_author=T. Osthushenrich;citation_author=M. Frisch;citation_author=E. Herzog;citation_title=Genomic selection of crossing partners on basis of the expected mean and variance of their derived lines.;citation_pages=e0188839;citation_volume=12;citation_year=2017;citation_doi=10.1371/journal.pone.0188839" />
<meta name="citation_reference" content="citation_title=R: A Language and Environment for Statistical Computing;citation_year=2018" />
<meta name="citation_reference" content="citation_journal_title=Biometrics;citation_author=A. Robertson;citation_title=The Sampling Variance of the Genetic Correlation Coefficient.;citation_pages=469-485;citation_volume=15;citation_year=1959;citation_doi=10.2307/2527750" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=A. H. Sallam;citation_author=J. B. Endelman;citation_author=J.-L. Jannink;citation_author=K. P. Smith;citation_title=Assessing Genomic Selection Prediction Accuracy in a Dynamic Barley Breeding Population.;citation_volume=8;citation_year=2015;citation_doi=10.3835/plantgenome2014.05.0020" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=D. J. Schaid;citation_author=X. Tong;citation_author=B. Larrabee;citation_author=R. B. Kennedy;citation_author=G. A. Poland;citation_title=Statistical methods for testing genetic pleiotropy.;citation_volume=204;citation_year=2016;citation_pmid=27527515;citation_doi=10.1534/genetics.116.189308" />
<meta name="citation_reference" content="citation_title=Bericht über die Arbeitstagung der Vereinigung Österreichischer Pflanzenzüchter;citation_title=F1-leistung und elternwahl euphyder züchtung von selbstbefruchtern;citation_pages=243-248;citation_year=1975" />
<meta name="citation_reference" content="citation_journal_title=J. Sci. Food Agric.;citation_author=N. W. Simmonds;citation_title=The relation between yield and protein in cereal grain.;citation_pages=309-315;citation_volume=67;citation_year=1995;citation_doi=10.1002/jsfa.2740670306" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=J. A. Sleper;citation_author=R. Bernardo;citation_title=Genomewide selection for unfavorably correlated traits in maize.;citation_pages=1587-1593;citation_volume=58;citation_year=2018;citation_doi=10.2135/cropsci2017.12.0719" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=E. Souza;citation_author=M. E. Sorrells;citation_title=Prediction of progeny variation in oat from parental genetic relationships.;citation_pages=233-241;citation_volume=82;citation_year=1991;citation_doi=10.1007/BF00226219" />
<meta name="citation_reference" content="citation_journal_title=Mol. Breed.;citation_author=T. Tiede;citation_author=L. Kumar;citation_author=M. Mohammadi;citation_author=K. P. Smith;citation_title=Predicting genetic variance in bi-parental breeding populations is more accurate when explicitly modeling the segregation of informative genomewide markers.;citation_pages=199;citation_volume=35;citation_year=2015;citation_doi=10.1007/s11032-015-0390-6" />
<meta name="citation_reference" content="citation_journal_title=Mol. Breed.;citation_author=T. Tiede;citation_author=K. P. Smith;citation_title=Evaluation and retrospective optimization of genomic selection for yield and disease resistance in spring barley.;citation_pages=55;citation_volume=38;citation_year=2018;citation_doi=10.1007/s11032-018-0820-3" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=H. F. Utz;citation_author=M. Bohn;citation_author=A. E. Melchinger;citation_title=Predicting Progeny Means and Variances of Winter Wheat Crosses from Phenotypic Values of Their Parents;citation_pages=1470-1478;citation_volume=41;citation_year=2001;citation_issue=5;citation_doi=10.2135/cropsci2001.4151470x" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=B. Villanueva;citation_author=B. W. Kennedy;citation_title=Effect of selection on genetic parameters of correlated traits.;citation_pages=746-752;citation_volume=80;citation_year=1990;citation_doi=10.1007/BF00224187" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=V. Wimmer;citation_author=C. Lehermeier;citation_author=T. Albrecht;citation_author=H. J. Auinger;citation_author=Y. Wang;citation_title=Genome-wide prediction of traits with different genetic architecture through efficient variable selection.;citation_volume=195;citation_year=2013;citation_pmid=23934883;citation_doi=10.1534/genetics.113.150078" />
<meta name="citation_reference" content="citation_journal_title=Crop J.;citation_author=J. Yao;citation_author=D. Zhao;citation_author=X. Chen;citation_author=Y. Zhang;citation_author=J. Wang;citation_title=Use of genomic selection and breeding simulation in cross prediction for improvement of yield and quality in wheat (Triticum aestivum L.).;citation_pages=353-365;citation_volume=6;citation_year=2018;citation_doi=10.1016/j.cj.2018.05.003" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. Zhong;citation_author=J.-L. Jannink;citation_title=Using quantitative trait loci results to discriminate among crosses on the basis of their progeny mean and variance.;citation_volume=177;citation_year=2007;citation_pmid=17660556;citation_doi=10.1534/genetics.107.075358" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Multi-trait Improvement by Predicting Genetic Correlations in Breeding Crosses" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10.cover-source.jpg" />
<meta name="twitter:description" content="The many quantitative traits of interest to plant breeders are often genetically correlated, which can complicate progress from selection. Improving multiple traits may be enhanced by identifying parent combinations – an important breeding step – that will deliver more favorable genetic correlations ( rG ). Modeling the segregation of genomewide markers with estimated effects may be one method of predicting rG in a cross, but this approach remains untested. Our objectives were to: (i) use simulations to assess the accuracy of genomewide predictions of rG and the long-term response to selection when selecting crosses on the basis of such predictions; and (ii) empirically measure the ability to predict genetic correlations using data from a barley ( Hordeum vulgare L.) breeding program. Using simulations, we found that the accuracy to predict rG was generally moderate and influenced by trait heritability, population size, and genetic correlation architecture ( i.e. , pleiotropy or linkage disequilibrium). Among 26 barley breeding populations, the empirical prediction accuracy of rG was low (-0.012) to moderate (0.42), depending on trait complexity. Within a simulated plant breeding program employing indirect selection, choosing crosses based on predicted rG increased multi-trait genetic gain by 11–27% compared to selection on the predicted cross mean. Importantly, when the starting genetic correlation was negative, such cross selection mitigated or prevented an unfavorable response in the trait under indirect selection. Prioritizing crosses based on predicted genetic correlation can be a feasible and effective method of improving unfavorably correlated traits in breeding programs." />
<meta name="og-title" property="og:title" content="Multi-trait Improvement by Predicting Genetic Correlations in Breeding Crosses" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/9/10/3153" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="The many quantitative traits of interest to plant breeders are often genetically correlated, which can complicate progress from selection. Improving multiple traits may be enhanced by identifying parent combinations – an important breeding step – that will deliver more favorable genetic correlations ( rG ). Modeling the segregation of genomewide markers with estimated effects may be one method of predicting rG in a cross, but this approach remains untested. Our objectives were to: (i) use simulations to assess the accuracy of genomewide predictions of rG and the long-term response to selection when selecting crosses on the basis of such predictions; and (ii) empirically measure the ability to predict genetic correlations using data from a barley ( Hordeum vulgare L.) breeding program. Using simulations, we found that the accuracy to predict rG was generally moderate and influenced by trait heritability, population size, and genetic correlation architecture ( i.e. , pleiotropy or linkage disequilibrium). Among 26 barley breeding populations, the empirical prediction accuracy of rG was low (-0.012) to moderate (0.42), depending on trait complexity. Within a simulated plant breeding program employing indirect selection, choosing crosses based on predicted rG increased multi-trait genetic gain by 11–27% compared to selection on the predicted cross mean. Importantly, when the starting genetic correlation was negative, such cross selection mitigated or prevented an unfavorable response in the trait under indirect selection. Prioritizing crosses based on predicted genetic correlation can be a feasible and effective method of improving unfavorably correlated traits in breeding programs." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10.cover-source.jpg" />
    <title>Multi-trait Improvement by Predicting Genetic Correlations in Breeding Crosses | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"pobfTYtTHEhnaEr20nT7e1GN5RxWw3LahwKmVAILjZQ","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"494269","apath":"\/ggg\/9\/10\/3153.atom","pisa":"ggg;9\/10\/3153","ac":{"\/ggg\/9\/10\/3153.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/9\/10\/3153.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;9\/10\/3153"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;9\/10\/3153"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/9\/10\/3153":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022cVVeA04qIjkg7dNI5K803A2Ohha8\\\\\\\/7VFCL5EFCG4MomZK6fn\\\\\\\/hJ74j\\\\\\\/5yOlngZKEW1C31NJDNFuZZRMGmmj7mi4jdfIuikKSiMJhlIfsC+RY1Mx+ihAiYEibYRmg0ZxSlJYw0uUbjR6Dq8ceswlOlGDdAnMsMeSuCUhtfJfm4vbTI49OsQVhd7YOdvwDpDLIcYgHKkKsu\\\\\\\/n9UNbL2RP+Ylcl+RuGUxCr+0AXy+dpgjiJ+K4J7Lu97R8ot089hVBfE\\\\\\\/MhUoG3EpjnK8CuEwPfPeTx8ETUn1KRDVtzDu+YcNXFgcu9qed0xTBXhyhQJMgn6LmtbZRIZC15UNgZJJ7GrfawzcBuQW5Zi5nOF+Shi0WbzpDwEAIlyhiXBwz1\\\\\\\/CNBbVZS1xb5Sumwys6HVweNebqHZEWFbZrBIphYRpERSYV7JxkZGr59CGcDc07refDX9I3TMUZm5CmQK2XvUFbO3IwEvoBfzVYnYNN1ewzHmYZvlYiQrdGT9rJZB8KEWLLhwkp46eXpA78B+iu0BxRuUmeMGMtKA5uzVoQ7u9VmAEejNYznMxERCSgqW4VhkWFpxaJbQEXCOVeLK+ugJVddAGSBcIc7khsCFuecl15lthQ\\\\\\\/9S6Apch7nguj4r0zV3I\\\\\\\/8ntBL\\\\\\\/2kRkW7gAF2k1UoPqgXFhdgngfdsYMgDecU6p\\\\\\\/ULQa5zjtDca9le1BAGc0fv+p3Z7uZC5evUGCD1QrPgagbT1k4tWrW+e5mEOggUbQGmdCxDAtYchPruO73gFg8N9dOxoU05lYSm3PX1rC8CkqhbMcLt6P0c\\\\\\\/Yfyn0GIqfr0lYFeKYylUXypeev7yOzbdr53fNi2RHE9XN0IxS5afLRQLO+7VKP1mBeTV7STyfQ+K3x7yjYZPCX9PHeHVkHUK3Hwb43JUhcQZloL4qwuhHfe1rYahgnPOPiJXO2ah8POylrT3VYmdYDj3Z\\\\\\\/UbPeKqtlt8KqRKbntSN4YVk1qJNIBDeLx7CZamt32DJb38zgqRj2LX0DmqrL8lJkTpRqMDMegkijQX2LOGyZR9UTt2KW4BzQgYkHP4t+Mho1uccFnw8IWxYU7vnTSDd+3UjYR4nKUWlCvoT0OPTjURKcyhQd4C6cVuN73MFZuLNzds2qB6vWcNLQGflFSF6NvYNWV4UFonv44lWOexn0NmXRFqLb2Vd4Sui2Dcoqih5KKdQouYe7wYYMKYyMoHRa16jusCdMUU9kxuJtWatwDBGQp5qigMuaSkzwox8k+Y4\\\\\\\/AratxR\\\\\\\/7tG+3zfoPDwufjmZN\\\u0022,\\\u0022iv\\\u0022:\\\u0022vm2mnKH3GAWWS2JGrjje7w==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022b13f2863b431a6e31df8a24d33fce9cf77fbfc778539b74f44c99e8049fddbe2\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00229mB0XZqSkYy\\\\\\\/vikFBm5ws5d5c1IR+\\\\\\\/E8YI4mQEvhMc\\\\\\\/tB07dk8ojn0i4t2vmb3mqIJQ65bCVmyoRYcbQtjpQkM0Qv6zeZyqy49w\\\\\\\/Cd9Nuvq8NHoqMVWu3HBk\\\\\\\/wbIT9TtbotTWcfCfUyUmtea0jYDjzxZq4bXr0OjFZlwx1g2YvC+MHNcPATVNy3qvHckG6G4gIk5Dv2o8GdHwU\\\\\\\/iRpfcU5GF3gZM\\\\\\\/AUgdh9yzoCOJ7dWzdUaf\\\\\\\/zUx2cmjmgoooaWfPQzdZFH7z882\\\\\\\/hW\\\\\\\/eNvRSLIjtDyhHOO4F3Tnnxb32cKJnojCf9e\\\\\\\/kSWPgbpObnyqY5zdT5S\\\\\\\/ExcLfVADNf3xK6vWgSByQEdAzPJiL13eACQWgc9Ihvb3RSm9wsL5Vu3MhDdwxcVrDCJG1slzTNQtdfZN2T193ARg5QaMFIPDekFfHedeChn4mP+eMjEl29bub9ULi8ZhKwWO3R\\\\\\\/tK0e\\\\\\\/WaW+9N81Cy7rR6iid63C\\\\\\\/q626gJmlS3nVjd1Pqu\\\\\\\/62z+\\\\\\\/bxNP679CpV7U7Wxg5QlSQ0sPnVWcG3RRJR2hsh94d5XmuXXn\\\\\\\/OtIK1hjSnHvnmuZZJOF+0\\\\\\\/0LkWXOnSD\\\\\\\/ic6a3UM1H6V4ugs5r0g4lbinHe\\\\\\\/PY8CzUPb4cYrfjTiQrJCpyWbGrEdxNWsNht5qEiNYc+0zsxPWFiCdAqPeABxzgHrXKV4V5SixhskOOwRgnr5DFkHe83trumtD1GUvEaAvh5fcm\\\\\\\/y+wLC5bMTZMa4j1sxxG1P0f+eV4tGiEMJzHg1W+t4hiUvl+NWlBefVac\\\\\\\/h0F7QHQRTwcF\\\\\\\/oaB8jb08dHZgo\\\\\\\/l8RBi64818hmJEKQIocMI+mgAC++cYCvomtC7Z7syN8zrp9Sbx2GzInY9QDAyqZ4nzknvyaLA0QzoST\\\\\\\/TXnoix63xri84Ok6BCiPdHqio0VFvYT70eLlwHytF28Fg6DuGE+EYbIEr3ieAzUn\\\\\\\/nT0RHkPcP9+brmbzRHmOlGZqlUDIEtrxpVl1difposqwXrYrrAvZinMAGB+CtBa9IeK4FAc3eLTbhVPt0oO7FZVOC\\\\\\\/Z+7hILauOWlEBxLuU8g25qIjjJxHbftQ4PRSeID2+2O+EPKOiGuXtaU1f+xkPKnhhNVyF9qleOrbxhxpA92Gh6Jm31M7kBLA+OURIsNtRLDGb6Bl10NnWiiKXvceNc4Q5jykU9g6prYObjOY2DK6dsjY1RScq3RJohowqkoFsX9GR11hBzXGc1LdGLOVumEJXLqRLolrqea\\\\\\\/+aj9ajgO1LP\\\\\\\/dL7QaoCgqJjEbk5wxipBjSqHOtumFYyfQ\\\\\\\/BKwvR7msRDqOhpWz4osi0Gj9go1ROjs7mRBpBy9mBLjW3t3\\\\\\\/5u\\\\\\\/DX+XZ+Y264NvI5tUEm37exDuZImnPTlg73J9sGwMQyVDr8jX0rZeSqGRnzHsDmRuYsGQvYPlx9DlzTf+pQcIM54XCvQAm6IDYqHp4YiL6e5orJND2CNLpNtenQA9iRHmX\\\\\\\/gg8LpLXbCjCOaOkTAEMxAx+K7uoOO91wHrwjMJzxTo0pq3IGtM8Wz5OqCdx5LmUUUg6V70dgtig5STAVqaI1QrGtZhh13R2SsXIpjsvVzUyTNIHORSq18rvrH16\\\\\\\/8c+eKfzW9P7lVDK97W8tx+W0uALywbTvuvlrH1GVCd069MaoI1ckXut4F+t38LHFJ3S61whzK+OpqqDL5lw3rMNaJ0MinFkII6DkK6B7MqrkmPxM7kO3Zqx9Fx\\\\\\\/XgPtb\\\\\\\/VmFnDHC9reTC59fPU43d5mWZtXw0fdZWhgL5WNecms0085jSm9IgvbIx2rlQLbVdv0BmnmADHEZXg+2v5lwOQk3s1TQQ2xugoj0SOKtxqyeRatpQ65tNKoafwVneuccOTINGxsP5laL32EuwmB9n3OKi897X9G1gt0BvuNu0gdqdmu1aEXWDUezlpg9w+4YrdbM9qRy13bIZZKD5u\\\\\\\/lcJgxvZaldwJ2JNWf4uW+EupGOqeNSr8JZyguvVGUAqwZSg13X0egS4cKvvIb5E\\\\\\\/AwzGPSAg6JQiiL5AoP280W7ldTTMRn+PyavqYx81nPhvmozID72e3STx8DFqX4SvfJY2HAC2+1srCsyj4ShdK3b3z\\\\\\\/R\\\\\\\/vqXfumYpaaiRS3y4mLlzglWiTkA1k8KHj6iptxOfdnvYWahcckkUAwQIfrFtVAGnKVWN2PsavjZgVLO+qhVN7\\\\\\\/eRxJcvqy38U\\\\\\\/EfVs8oPbcK2OMcKArKRMtC7uweWqrxWZcJzh8VG4dK2Acpv9hykWnBTFrCVj2FT8NE+xquN8282kZnjXxcFzCa6nW1DrKpWGeMWAIaORZidP9UAbbBbvjqtJiqzpRzfUuV5LlO7SKP5NdoFndJ6xZtf1+sUvbexWDFuKkW6hv5ldGwbbVWacAatuOK7vTWQjsCmDP\\\\\\\/q4XkT1kHS9Y7rxxw56+n+An6zQ3dzD9y1nebUB\\\\\\\/02eUxAn4Gqz6h1dNvUYodHoob9iq9tqxyQ1Ufu374BZ3pI\\\\\\\/0KXaMxYT1SUfnD8zoImliRKWfUrxpNgWlbxkC2g9KFKfZmpWpttrGXS+7xTk7QIwf90a4LtiA6lf7UVhZ9B2Zflwwd8x8GGV4f81A3n4wPhGzt1vbDY8XWO4Oa1deiOsqxePkKBttm3\\\\\\\/fJCJplwmzLClVYTIsCnMRer\\\\\\\/I3VYtHWGUfy5nKI90Mn8pAsEe0Og7HyJhPApzj179BZTVo1sqK1qsBfa2IMVW7H8RlYeBtSz\\\\\\\/3zaXYBGzqRu4smsQsuWDCBtQTTIwabkMrD2dVlyDPr4E0xk+eiyoEShIod7GaXjrHuZHW\\\\\\\/aimlIpZOHDEbJtxyNuWmO7z0BmFvi7cNJrT7vSKJQp\\\\\\\/JKRLRZViOwZfjIfBG2e38cbH1WyI03ZN1925gY69fniCqvnFThMorqFhXboKwhFTTdYXu07Zgw3y5WFoJoqQ4tH9NrNpPhzSnXGFCM6Vr4lSvQWA2HYnfcYHpvEQze8Hq+CTsKeczA2kf4nlT\\\\\\\/uEQ4fkphklZCiYSSq+Al\\\\\\\/Ay+8ORJ9+DLjCgwVfmeRONmaiyOYa8xQnyF+gfiMvMUXnLTsQ8LecxtjYM2zb9xD6UNKvmBP0i+z0CV2z8QByCEWS8+2gPxvugc2V6iOqrMnSz5qendcbjY962evc2ICLdCifI2Uk\\\\\\\/uLLSurmspkUZLBAts03vSx0602VGtovQy7yKPU3WtR9iCsxwVSznMNLs0GGjPDMA9M3YfsQ2rR6yms2zcUtcJ5EWtuNsXJfjkHlzzolypW152bRr82UOqEOfiq+QzcG8wg9bugMavqwduTOiQZHDoX4VajfXXvo6oZd0rp15fJ7THrXmowj\\\\\\\/20nS6yQJ97vV9qhOH0ls6\\\\\\\/hFSUBDp7RkdI\\\\\\\/I263krO1o+\\\\\\\/J3TBItnlc0TMmT3XXJL2OWWQW80Ukm01HgyY440c9ZXhk0LRNJKWmZrHimQDoE9ApCV1sTpaFkzgBQCQvBT\\\\\\\/EgjaUJglHwkSZM0TIoE5Me32mwd84HP9z7d1xIlcKr5HxZ3xtfZ3KNaGxncl1V\\\\\\\/diKhwdYbKCxTbQuucoZjUcyxk4gqpcGDhMFbAGyPCE33mdtRO24Jk07DICTzyXlmK\\\\\\\/2tr\\\\\\\/jCTrJnITVEUqBiXWm4oLyfLJ1DgxlR4k7YqFAcQeiq\\\\\\\/at5OWN06CYHDu+Pfxi8+PmM+Yyx9rcmQtZ5pXySf7RyHF1e1q7sCAOjjEz4wv8Dypz2S5DWZIk3Eqw+0nqV+U3xkW3juh7CryX5DxEBDEW+dkjCCqLKs42Rkr+RZZlVVYhD9Qi3fxF4ADHNYPLMJ+MGFeAc\\\\\\\/LpfEYpXt0e8vpEckQ5NRJwRQyONH+R\\\\\\\/uyOf75jOyTUEZqSETPgD0P9RgG0j7QDM+ILdUoIUnSd+YfVcTb0h8aaKXZQtnyTeBZaHJkm8n2T4zjpkdZi1MymKN1AesWVJMiXcKUB26mJcxcqzz5sC4s6rrEych50uqTk3XESi4r2JdJAdVQ51ymXgWL9JKWinYSKs7u3fGggBX0eHiF7ne6Hc64fyrsUNeoQMD38uACBEtI4W0edPPexoCztLJcbpYCWjih6g1VPhgxmy0dSFhS5F78JsMCTbMZGPYUwRdZ6v3xuWXrh7xqsbvos9lMIdaEYwD9zG0JOJYJ3O+ovJ67HLmcYQwHbwHKVEk1Oq8L9vRcqc67dr4PpTXRlpj07BPAyhvYqqTBQ4dBPf7D1K8peJQ3bdfcjDjTTgAcnwfXuQRWb5R4c3lmv4A+2UBb8MQuO7M1VTBvXL1Hheg\\\\\\\/cN5e0o0keY0mfPdI2wQthdPIfCQjnVxBSFk46GJm5Fc+CStXGgt2VAIsGsWKgqhsDCjXoCJycNqYmSW42Ke1Woh6UkOrJomYWnROhVwVh56\\\\\\\/QorOwblHxYgxVRL0PR5qLmEXmMHGGr0qA2MvPI3RAr1ELc\\\\\\\/+3wEyhFhV0+nGkUZdT8XqWFIkHOfROvnXAQVg2HIWYDkW\\\\\\\/3U7KeUvu6CcfMlr08TweLmZ3C6+0WAIj9bhDlAm+FxPh9o6H6oJeRDEJ9SRKRiQytwKJH9d5OBFxByZE991iN2fl6t6ULmm\\\\\\\/o8fWJTE77FjtDQPtJ\\\\\\\/gEaOh8prjBMaHG7qWwzmR0UXKVmNBh+XHMLccvUw2Lqi7zUnaB0kMPjFia1HgutJ+IS7z1\\\\\\\/ClXFbu98cRPeVtedyjz56KpbxCTEhlKgfPAxzNST5BDBrN1ehFKsAxn0yCszGSoERnjbMsJnld0Hh+kYkJGl8Ny0\\\\\\\/C0lHOVyyK8afkawGijdevLqmRk7kNIBI+lPVxKMFS2vMJ5fneDmQINCvVlZptVG\\\\\\\/uFCkj+n0q4N57BX+b7xV98VrtZhJ5gWimC7p3u+5d0ZC+sM9uHFoPMAmZ8OTJFxRtn8QptO\\\\\\\/1sjd9MwWEWcVzIIkWVR+bR5RvOvAn3Pa3AAHBOspszPEsTI5dE6Abcb5bDGSsgkvJS+Hxb9+j6uSDCx0akWLFdbW\\\\\\\/r7OMmmKnnFm13bO443xhhXONCX92XRZB2KsMgmyqjbPY+s2lJ8qqo\\\\\\\/7iJihHs0X7E0KC1oxI0ZVUx0thX+Roy1ref2OcjwKHgJ+KF2aWTfsA7MtUs8jmyx7fQpCJKZ\\\\\\\/LNTJeOCYYtrpzBpjgWkE1yGpf5j4ZOqdHeULeo025hhTfibKZjDnrIqhXc+FLLk5XWXrGvDfmub+YAlc5iWglhqA8jZCYT+YKA4iWplsJjJPP7o49c4zlM5MY9Xm+qLQvSuNTWITJNoAbdJR8YGv0Cd9rYnyUKUkTTGQtLphYt\\\\\\\/z+P7uJMade7d6sCZmyG4RO5IxlA0uN0umHHMhrum4cYfseNSomv6JDL4pdt\\\\\\\/J0+2Y+MPqZKLBbNyWt\\\\\\\/nv5uteCGP01yLa+3gCHEIVglC8UdK1bMJVDfmVLmmaT7qCkv1tQwdQG2ANVtiO8yCLSwCFxFD7yyduph4nTq65+pm6oT8+9oO\\\\\\\/jtv++TLFS5EsGC4RsO4xeChp56WQ1d4lz5xZ\\\\\\\/sCb98R0kPA0OHjJ\\\\\\\/vQCS8L7hVIc357F2G2xvE8ZS0TYVUCFbek7O7axQzLrm0qeD6SoY3pRwp5NT8s2IHyp7sJuXhztTxRm1hnWxi+8hlJ2PicJ3+C3hJ3ARM9TEoSwFsVAyxDRVynUBjQiF5Tt+rt5W6qivGmFljhFf6cOAdgd9JMl7zDCqPvIFKSwuCr73pkUEfe\\\\\\\/cIanWsUR4+ZkmkAFnTr+nSCoAqGZsrLDtVNJQcaR6jYB49BLevEebWcqyfZ93U0taRID1bBxTXlAUdXQit76H9wWekpEyohjK5vmc\\\\\\\/\\\\\\\/eua\\\\\\\/CFnivuC+ReTxgcIZYRWJBceJmsl\\\\\\\/T58sqqXUWUnn3gurgM+tngQ2CnLzLIz9h1rc04S7Neb6c1s\\\\\\\/3qZNL6AFP3UjX3x\\\\\\\/tjg7TeDGn1fYlr\\\\\\\/DD+VRto\\\\\\\/VgNDoqji0UovxPXkQKyr1gFOqiR1Socy8PEC\\\\\\\/9KfSr2iXfUoApdKOvB2luaIKtMEYJg4QofYZMv1EmMF+X+oi9110dPtuC9sQEi5dqZsCjqvSpUtqjcBWLi2jlwck9ceSO\\\\\\\/\\\\\\\/6knpG27h\\\\\\\/yCq4c7dB\\\\\\\/hw\\\\\\\/CBNEKzjP3DmkxyjTT05UGykcipziJCo4WCrWAfOTKRw3U5YbDO4nzwQThZovKonJkfPB+lgSZmL3q4QhiOcGlXQITgcbSbOLXW8DwNAZrRWQl2XCHqEx3iXEhoo\\\\\\\/DG6OpvcZln\\\\\\\/AXpQvfCsFrpiGX9BQhbV8YKIpTvKjuZETO0WkfryV2Gk8GFJKWAcxbWDnTR3c5S5D7v98mmh5IKw+0qZF83Gn6UVm3HDBYXNybVVGqqfcaON4dgjSlLs6jp7n2XmaZ++EefuKdZEDnKDiZWyAXLmqKKLTmP+zOVH6qqyKmnldptZgK4B+CPtqlYYFKmU167z1U+rEsO5uv0U\\\\\\\/daGf+uQB3ToaUx3ZwwT+aM1I6uTVyVsy0BpVFe\\\\\\\/QXiQ+i4rzcqxGZiC\\\\\\\/GK5QuzZd1QPVmcK8+rzVKmY101ergy4NiTxBOJzKt6AfsJZ1BtZUz8Gw6ig4yHSXxok+eyoi1iz9JNs9f8LEMrFoK0JqHYqalmq6j1QwDSoE98ymXIEVKN1TW4RlPA5TgjCIJha3uV52q4ZuA21Q9oW3DFTb8FaZDGTlBd\\\\\\\/YoIJWD7m51j2T8cPq544LSsps0xqHdCgUcFzxNri9umCCrcpjCM4Oq7qTHIutZav43PgcF7vOMfr68bqaZPcsgtYLXNVnyN9sgKmrbbyJpNCrokQwn0MSBLsanIfMRzbU6cbSlNLRINBXV1B5PwTFlk+r+R\\\\\\\/ZQIiMAM+qO7BwfopC1HbqXuKMlVVFCWMIGTDoW6mLL1I10n3ZJG6e4C3hbv30d++++1WVKlqbVEbhlBu0blT\\\\\\\/RTB6LIOtyqUN1DYilRhy+RW4WRJrN+W4yCGYqp6InOska6mwUASYTr54kYLrY\\\\\\\/Pefge+Xmzde\\\\\\\/e8rFdVzMLrJkUoV+wudF1hLznrsu62Bkf7Q288U52+8SEdPwvPMZM\\\\\\\/693CIuwMaZ5xuYU+xGJZNcQElxZhifsVNFioqrBtj8eyUEGNGeU+5QgkLQPxs9hl17u3oDamXbJp05R8ubSuomiW5BWlkiBzgzPY3OBq9y7h0Kh\\\\\\\/Yc2+Tsg4MfvjW7z9I5239Pim98DpulARusVHcnq5naZoPgD1hT+7CeiBQPZVx4JlnXliRhkw+cQtosAAuX940ysX10\\\\\\\/rvyKrFFzGxPVgaRXqRPAAtckmsslML8TC\\\\\\\/fZdQmEOS9HJRo+2AKUF5u1MPZykHVYzM22LpfEOM4rCh99v+bGkO8UJaXl4pvZzM3NuGWDbYo7+5E9PO4yFohne9gd7a1Ps2SpEnTYABsWIDnS1ww5h0s1HkeHKJgwXrr\\\\\\\/JjebSw5TlzpD\\\\\\\/3+yO1MaVTS1aEoB9Hu0dxG8QD78\\\\\\\/4IYUkq4ihJusBH3H1+bpaa8eqszESO1Ydf1mv1jXjOTgK2BRn+J81vM9fzdzlcUNt6d\\\\\\\/jshptOwv7VpXLuMXrGlOQe6jiv3PA4B2yigh4JHZlNgf8K7Z5s8JOuSOy4kY2qbLPkq8+GOiOuyMt4WT2m4s9aChqKdOLHboSBBmfHVpjhLdZ0eI3hIexUmisn+wDPfqj\\\\\\\/GEY3RcN119ODa7NybzJIY91Mk39iLUX84bVXnDTREHPDzd8q7t6O07l1SNZIlWSJjqKz7n3q2VFvJ5IxbZUgeVj4bEJSBnKsTVh+C94Kc53RuteOSMe1dHZFiiLg76RxJACD0kGMuUfS38u7M56X44Iur7cFBlhnLcx+kzzsOduw5xqubDkCMacu6quGGtScHiXkWUbXhYcH9xC5BplyXI11Hv4UqxO\\\\\\\/zG7WIhFYq9vIMCiWrk5HQDAsHGeKCHqY+680fqni6i99iwWwn+vf6BPQ97Ll1fFyKeGewry5YrfZ52msQV87jCHuKMnHMm+tN073wx7I2VHEVR0mDevEls7ekYyqAwlYUb69nmPzirKP1cN+WJCMFAaY\\\\\\\/Ksv259yhk7ZtAeuYAiOxH7iBU5xXEP8yL3kt4LJJnNc0Ma3oI+XGKnHy1xE4Ld2nyho4XHA5lLE0PV0rmjn71A+0DI0OwFROIwFjNQ38hqw1dYCYDm+82lmXWvBGBfubY6Zx2IRYSdvvXd+DMZKkgVHvmacqGn9Onclgt7X8OjF9YXkEdC1CFK7MRw3dDj0UX83k1Ds1VRQ55nsnTHKuPoYsjjpb8NfABoIZ7cDJgFU+KlKCn1jJZlKMQK97xvwq2v18IlaVGO1D9erNhCX2F+odg7wXYVe69x7UQzQAN8mvE\\\\\\\/tPvowxu8Yr7ZNhbiqg3soeLj9AuuyYGMmU6HC0tFTbJaiU\\\\\\\/jDtw5clSrEhQmkhYnSSeUHGEm0bNl5jmUnctuu1pNAFKF23V+9x7BdLMRIdv8+tDDOAdnTi7+G2Yc1zMWCL1CcNdUFpWFNAYzb8ROcPn1u+XB\\\\\\\/9ITzaOYSbEH8tNgHdPEifSg4aSZRT8VBRGZeVO6znAPBJXoZUxfyS93JZwKOi+bInQdJs5qZ1CRzSlpc9TNUou7XAztWzLE9nFGk7t0MIv2EJiqf5aiSLTU+OzADjce5Tt349PFw+jTEPB2TL0M4mkloWmVyl5+TrxLc4\\\\\\\/aL0WCRYIoQZpwGgyVffvLnGI02+Bmi28vDQ10rtTTneseAUc8ECtpxfLwbBwVrMc54THgYRox9OGhqtHD1bLRohdiCPZqTU+KUBSZp5LD4Hoy6lBQByWjb6j0AQUduv5RdcHRUxbG7\\\\\\\/CtWarjRuBj64KhMm5GxXG0PXzMlSTET9c7y1ulUufgWaQNxncVT8SLUnnBEnjJDYAydV+992I441SORKPIqiJy93p11aDT74vnaHZkVcj+1jKNdE07AVenQ0FxmiOdHGFGMsxDHecNEiwO6wRulUCd4ULn3dWX1oA7JGiFvR8GnZ3Z2252HuAj3jFM2u7+XEbpDsSdqLlpewwbdAiMv\\\\\\\/w7Nv9aQ2nOeCZy3OECjRqU2lwYsvKhrEEXJo5qtoXOuyUfppbbSxwEWRc8Cg9m8buPIW8oYgHpskbB7rT98La\\\\\\\/KKuNCSGEQedA86eknj5SnkQ2mN+DRWHlIl+jCKCf2V\\\\\\\/T5pqLZlxIqmcWrwBPgBQ86miRbSCrQh3rljvpGG4WRpoQ5cbWc07f+e9D7rb5A78F+kDj71+OdVay6pO44z2o4JT\\\\\\\/MMKSvQiYBrOo+jNMWJv43krAFDQvnoJMtZ67N3djf5UxcDIfdraU+UaEOSmy3ha8ZCAAi0R6Aa2PY1QCpNNe4DOfqWiy\\\\\\\/tPbBNzGGYuNeftr0uM6aHWcVSFXd2CEr0T4AqE6bIBw7z5giFWv2ovP7aaNbdF\\\\\\\/IEbGlRTXlwuV7Xk2USUUTWhWitb4fFyII50M440jMuZDktcFOK99B4Zs9FYMOjwlgtglPU76zPeoTtas+U24cDi0ZXlIBMVfxzzgdEGVXPy7zbASkEQ0TbsFxmogESf0NOEU3TQxZaeuRnwkePXV\\\\\\\/bReqFBvqsBfZopiHa8w0GNwg6uLdWbKQgjyBs3gG6fI9u06fs1NMIV5jQSre1b9ZhQgr+QoRAFBFCW0QBfep3Rdmc7i3dDx7iPA3Dqw5NEJzfMiXFVWA8PcwQWdJfL8wyABOn7\\\\\\\/qXErsAucf\\\\\\\/T+IExuxPVjkHGwJincDyXQ9+QwtHiNWQTJmJaNM0jX7r2Kc5EiIVG87rpeND3ZNMuieIdIrBPRhrbtbArN7ppRdjt0aSaakqdothcJgFaoitIL4iK9zzz\\\\\\\/j5yc27VRKDzES5R1xKK17mdRu2GzX4CId2PpwivVlIirRzRofZXCDBFvMZKnch7QUBTtFP4R3iCt0JShMRLPWUrXYk5+bBP0wvpEaN\\\\\\\/eVU9daCILOlSGd7tYxEte\\\\\\\/OmQikiDrr2TCxbW8jeMfFpsj8Dr6b\\\\\\\/TYlGtAIvGxZNyHOZdv2aYe5bVST3uD7q+uTVGeSmXeceE52UIwOK\\\\\\\/Sw3MqRM3URMNxc78B364yudVjvJtgghHqQZvaH+JYdZ8B0r00QDl2R7yI1lB6qpJWXZXF2alIzxwR01Ly6uYy2IUcbQ0hZEk4fko+jm4QUu3KLlgMW7zl6ww2QJZYLAP8mQrTxkPHqOCokTBeqdEVzkUOg1TOWXyklLcIVux2\\\\\\\/Dr0szpffqqDWzDT2PdumHBHDdFYLXH7R54KRGzvQg+zEXeoqhFwlb7yM0KxlfWa5FwAUmFX1tL1y10zH7WPTJFq4xkdyOr9Ne3T7gg9tSYfTBOVM0DdaxQ7ENcOScIo9SkxseT0VKLvkjMvVOUSvVINT1yJpmvbLSriV\\\\\\\/LOzYW7rRIF+TcrF3uafxHOG+AgWtSgbyVyxFx4oYav1yr9wBTF\\\\\\\/qSowbKxUBRdksV9B43hveqAm2+kPStjhpRZ48Sg1k+Z8hBy4Kssjod+7+SlOQc9g8dAhU8Fy9S2BNm1QXIQ\\\\\\\/mxO+z7Bg31fg40NXlZgRjcC3EAblPE4DOyM\\\\\\\/V2DGLQTbKiT63\\\\\\\/LnM0U3BEguCGX3R8B6n2bavUgmT++WqUvMA5\\\\\\\/UozbFu\\\\\\\/DmjUHTyyqGXZQX4EgPzALNNJW3cpLCuzVVkSnIWmIv7y7AzHoqDfwNPrw8E+mLK29TExeTYVYjCaZgNRiIoQh+BapWthbIysCnJfWv\\\\\\\/s4MKaH\\\\\\\/SFuTBO72u3K9c6b9fmH3Al5aubeCv2ZXtDofero5M9kddbTxOD0hglTpOR5pbaqp9dK0lDQ3BRyxgt9J8Fbx\\\\\\\/MznCRifMDAwi+3Ajb5nVK4AyKts6BMYIfKD2tazk3yUGFrLmkMncR35iHCC8BQ8pRnAbi7BMzmfEm7QszcaDTct+EG0BobnJ+Z2oOCbok7DaNDi7BZ4aeDDV4m7OqQw2Pon9ToAywP9a5mkJNzYS9k1N2aaT3Tye1XHaVLtdrx3\\\\\\\/pPWUAIN4LZ0lDa09JOgV5ozSSqVl79UYjbi+dOj4Z7X5CHRRciJZBwaxEyM\\\\\\\/m8dsLn9A\\\\\\\/4q8x5gVT11nGz9\\\\\\\/XT3uaRSOcsm73mTbxFP\\\\\\\/Y2st60AGbV35CxXaL7V22svoRWnBoaBmz8o+43Dep\\\\\\\/ijXSbQ0oAfB51RFDeZsMeUD0e9Sll043m7M4MUxDKq3HmxSAy8i1NhxT+P\\\\\\\/n9Y0medEqpT1WP8+7XUhhElQW2VXTqXpU42x0YLNnSrFxkJ4TbpaUsJsB\\\\\\\/5EOYSh9ttTdWn9vhB6xszOMG54dPvLCAN5kwXVCILG34xKXz+AgeRSbbk\\\\\\\/Ti1XM5\\\\\\\/oV25vUNgMKlHjLSbw599nwEUVJ4OKIwofOI9CeRWcQPPXQsVwmS7OdCaZPIZfFwzBeEnL8p0SrMNKwutnmb5kQFsuPOimDkdVFBpGeDmb9ik4yHPVo5uW7mv60AOH8kOKEeADxLBBtJeNXFW5rCfUuSxECnYu+QswDcj8lUkd\\\\\\\/Mh5qoVzAMP3qmIbaR1CsBBZPe7JQlGDb4nGpOrUidcmIWgoe66dWg71guhiKzXMAMtEHYmneYM6AYue\\\\\\\/3PPs3CFs\\\\\\\/KakE03vRmPJjxmZTiqPSAh0PvBrhDdD5T8AkGEeexv\\\\\\\/OmlklRyFOApvLcMyGxYG69WSLl79jYuNEE5CqcGEUjPZKQBcT8UvPUv9YjLS5UoqSq89\\\\\\\/Uz6+CObx52r6phkArGwXNK81N6k2nvXxHI83RudfBWRCtbBiXnuAQTZar3vm2T\\\\\\\/qpX0GIWRXRnEJfW+aUYRDGQ+23iq4F1l9pZXEoMkM4B1t\\\\\\\/IIFZrmD8qjWuS2RVJi4gxf1slTVhbJMLabYiLTmq76exlDeeJ8\\\\\\\/ShIh5DCwio5kr6k3H1Emcrcto\\\\\\\/Agnu0+Wm7sAFlW2rnK6en3SoHNlkuJiuQDLtWu9z1hzrAai6TkQ7h6Yb4DPHvP49ZPrNH6ulx2BEnhX2H3YRDFRZj4\\\\\\\/ePwBZ4MuarhqgCMGnUiTMz6ZKsJ+FcawTiM88v7USSX7whcBMYRW0PTmOtpmBOp2y5HnLHZVEWGIaQhskqGxt0+V2TIZ7ZbfClwxTWT2MSfFveL6thvpnW8jMCkMxhgHG+Rz8PHPX2MOFd99mm\\\\\\\/Gfc66MgewgZJiEVKH7KDCjgyyjEcoWeaEYxVq5PjMKRHRQT88dQw89cPpQCsn61dvI69+Fm92x5uOcopPL743bM6KvuOQo5hz3N\\\\\\\/cT1jNsGIGdaKADXQsyrC6xCXZEldu1HrwlS9E3+bkOrwUyc3mM4SKOqh80U4YbK5h2z6sY7ZkgvjWFKQ\\\\\\\/8cePAeORbrSjMfxbzjvGaIugs9gsyaAd1ZseSJRvP7Zit3r\\\\\\\/TDRL2HEFcaQQ5GsX34MJ3omK81jiNYmQK\\\\\\\/zJjZOxIaUxfXCJDxGq5bLLl2OYVwkKYB\\\\\\\/Ob5XQuaubmfetlNNE9nbozXE6kMdeYwGfSkFMzc2TyF7bsMj8Uc+s7drho76cFJT\\\\\\\/WMgGXJxch1VspHfvX+1wHMGtIj9mhvcdnZBOOA5yh2iM7vlCepgVggNI3g+mVbloXUEVuCXUQeHACcPgoT68jygpgp1lAPrBSRvTp1Zhi813igLc6JR7vl9MgtS5+LQnBkKtF0RraU3wmBAVl0ybOOnF7loUCqjqiIqygyYWnd1NUyC0ZQadvyozDtdErNmRvXoz77qKD1OC0679JMXHl7z1BngkFdJYmqZvzo1rvWrmhtXvMPFcdATVIplCjXktV8rLPjNR\\\\\\\/QgMaBaFM3f+HmAJc3kh3LwzC3Si4V6Eb6NM2\\\\\\\/VxOyqikTAbymZpUuy5WgEZp2V4CPTBB7DrOZ9eItInZaJDVg31RXGHJ07SrKBOmxIksMFSy4KYrJWn5nTH2qO\\\\\\\/JqvhFA9J4k+dXwk9kCaF4CmldLmyWi75hTdfexn37xTgqIcJHcubSeOLx74G62pJz9UbGd8yg13WNMQ7ODRNXggFyMMlES73Xc18mzABJ5mDkGvLxWF\\\\\\\/LaFzS1D9z4ck+vwxzjGoGR1OPaMNqnOkmUj\\\\\\\/dSnq7hGGLdV1ejNRUsMUWcm1NrZ6LM8nRr1KJNeazLohMeE75AloZsk+E2Krs6LZ9+\\\\\\\/rQ1gA1ux+BJdqNf6c\\\\\\\/eyzJthaCuc4qmbaVv0dmxgCr9C35qbLUIvyMizYrUUIOl4kcQ20z\\\\\\\/qk5gC5tzM878Pg\\\\\\\/oJUr\\\\\\\/V6K00oSvVJY+l1RBbgZlBLYKoqtaKqIIskZElptlZWrjauzoOEleCBzo35\\\\\\\/BnG9dy9dJ\\\\\\\/OURpFqCowhwVQsrOJ4caXqlkBJpLmqG21\\\\\\\/NSquKufjgJb69Au0bgS08vAC5jvhRr7aM9L+Nxt7QByX9Mw80mqqvYSTblS8xTpVWTMQUSMXrHDsi4U202Uxt1AMW9Nx6BGWzDC0bDBAxEpXmHaXm+7mxtZ7BG9M7rZGvguQ0l1xx4ao1NCIpTG0BwbyLyvH3fB29S8gLQm1H57WBYsTBRd2J0V+Nxg2xF9po2oPJlSNbSYlh5IeIHarFAKr+KONamquvHbnAo\\\\\\\/Qm5uuRvFlESQ8nBEatqcT6V140ZwF7dsHvoRC0AC87UEHENpqH47AOp\\\\\\\/5Bqla0JEIADXSR2rTuOcDNtDAwzLpxGf5A4fyPQD050Ti+h1qNNV2nRCxt9+goWXPcWZmwkMaKM4cmXAkwu2KmbE3RqSSO+JPma2uP7tIygksw0kaiuSuCYh4BWHOKMUKVrhYHF9APmPMUCAajhBg7ojDtRyw8nhVkQ7t35lvnE6EM4w81UUsplRVgvF364MM0uC\\\\\\\/xQxvKeEVlFAgrhtFzKtOxCyejBKDrxpK3hJmMmPeCUyM4w72oM7ohv858Lx5H1CCUEn+YUFtIUpJIyCbSEXm1\\\\\\\/XPeKMxBX6dzepxJJ+zY6vjTJrK3lWxHmfSUw5c1XKfJj1p1EcoUccNv6VnOftf7hPthNTsCW3lZr3UftE3Gg03F74Fy8MgJudgPLa9qAsbaDTexnEdmftXlnCPAxYMTyts5OHubFCqWw3fr5Tg7ogqfI8kYp4PzmN87upzHAcqxKpZhnBfqmwoXS8n4w+p3aNfbkQo5Roy2XvA31OZIM5GQ23u6epfXBwjY+JfcvjdVQEz011Crr\\\\\\\/rna6sVdebUuJgHkRGXAN1PBqED4qqZ2DHADeBAobhDmdFaGHRFkBXfLiQRwWmDBsbd+yeTLxRI7OiEdvfFcaThR6DyrlolIx8ijHUOhlTc2hPlhJROnhb3zxhhF6UsDbSGbANWsYABIl967GhRsOOiVdnciha8jb06I1XvCgeAp+yKsLdMYc6+V8HDWOAknrgO+LzIPcF+SQC9lwLTobxrz+4U+NQoRQSQ\\\\\\\/MRBfBaysrq3b7oTCI2hP9K08LK2W4MVwAaETC\\\\\\\/ZRSC8zsUgRlnIdT4jPg2egRCbFIqMLiVvfxG1WhVqnP9bOKmoR9Cu2bBjN1hm3ffclVzQvXzz5W3yl3gjbk0DnQdis4lDasOfGtKKHrm0cuZ1YfeDQKiQgu11ABXP8EQDoAJ2bi++AcGLwdF82THto7ZZ5NG9yy2FkHTmk\\\\\\\/MuGWFUdM+E5ejfgBcX6PQCEAaLSLj3kSv0aTNURdW6l74IsK\\\\\\\/12VJZtrVRyr0rnm0v4UTuwA\\\\\\\/PmiXkiV7Pd9cgWQFE1ozYpZrLLcSXgxouK+cQxYcGf3vVJmEOv9Cbj2Uqu2FAeoY5YOpNL8CdpPA43+gAOi1FECTS3Zd5A4u3+7HIq4mB68m4zUg\\\\\\\/jncXEGk3fMLBHSIacC4IivLlrX3S3oegztztgqRNymqSMOLZQ5N\\\\\\\/2ei73tFV3J\\\\\\\/y7e9VOEayW3vKTNb9dSLYyKCWmtfQQnZcw5lcnucCf2kDJLoCXrcs4vm+veyFQ+JJ+ZPg3KNumVKJJoRAa5nIaZZCgDzx1QbbSTVa5ZiG08e\\\\\\\/G6nrKTA+dLxrLkAcHK2YXQx9HQNUfGLukLnBO\\\\\\\/ZlIvf4XgXati\\\\\\\/fz+LW9fDNqauDsAgB938oP3n6TvbhyjmjUAGOV3X\\\\\\\/+XnZlRW0DUqNiWjHegee\\\\\\\/+epJq1mgOuqR6XMfg1hes1BhQl7Au1VyCBw6FQ9q8m6\\\\\\\/3CLx+Khxc8GVSnDaUTUKU0XyntwzqnIeoE0XwzZBJe9H0oHfI5d2zfij\\\\\\\/cKcLLGsdR\\\\\\\/pyDw62xszAP9Lj8Td6FwAFvccLagXMqHcidcloBezzb6XfmnonFZqj9xQ3WAVb\\\\\\\/f5+jx1LqT1rbsxOtHdTa6O7ySoaD87LIQqbybhV3khOZK48gVkoa7obnpMqqG\\\\\\\/rWoN7ieYpSBqOldtCg0mwfxaSPktJHyT9LCZFMbIGlVoxoHR0BBa4rTOuf2q83UPEPMZ2YvjqX92gitTO0iz90nwylgRGdIQTymWMGy4T59ONUOY6iw3Y1r4cH7xD1WRXPQTuq22VeCiZZ2IVayHsoWpw4MhOIop0dmzvS2tRmmNKgvVkiyfRcJaX\\\\\\\/4Y347J1iCbe\\\\\\\/Hr50moW\\\\\\\/U1SrUAjJgyWRtDe4zI9HJqTrTLtLL7fs\\\\\\\/MXMW8grcC1a19PJY8s9BmhUfOXCZEjlLAvGTETHdyyDnS9io5FYuSIPtAHPqXPbO+Z9tEpNexEmDVngFSaF6l2u1zCMMozMbxaJSZ+p+ccQCJJSjmapZzPqRu2h8ZhUWaRTjM9xZNbo5JgUo2XEHUpEIZpiXFKZ88+WaW\\\\\\\/LsWMzvHo7vo+ca6XOfbC7I8U4jnLBGFHJaI6A8PJ58jLjU\\\\\\\/E4cdUuaKWt9AzyFphFjnSuueodw1vVSuKmPf1SLwehZgHGrzpLVrSk11Iu1yfBIcWa2XaJTccCXGVtPDQiiaHjRjG3C0Fsj6FtYKfoIWL3z0CIRE84mrnzKnQFCkW0OG\\\\\\\/CC5h9WzDnGA\\\\\\\/DsEGQHdnwcgb552v+TeTS0LfJEC8zKCLf65cNvQf68R8uilWv14ksK7cFX8YycxP+Ko9AR\\\\\\\/W66gc9pX9qSiVDufYx03SpG6v2koKi3d+VIboVU048T7a5ihUuozh090\\\\\\\/Fm5SxacuafgCrmuqH4qW6MS0TjkuqWYR9IP3LF\\\\\\\/8gmEhu4229I4wAdlOzS5dRS+lvyr+ubHeBESE5X7V7TLiGpWTF8PqWh\\\\\\\/TsOmnPmnmsjZd921wUAhp2EMxIPkjTutNcBM+VPGNgeaTygDrS3NYAKrPq\\\\\\\/ymGZPcMJLp4CTHDAiGZJVMg0SUgIBxzwD7p4EqOWU1xetj5nCWnmWdUVPMOkg52W5XxIpC5Q3CnOX6PDRGL5EH4BjtIY7xjcm8x78PqLA9WGjwsYJHIJid886FavAmIrjegJNxKOQOFeBwFhgz506IUzGALqDS0FwrFBFQSysZnUUaMSlg3ilzOgdN2GiTfspnmnVyS061sAY+j9OOjD85uKmDDoGjmyl6quaIDrcDQfbqdA6A+Yhw\\\\\\\/PegskPNMREyDSjKdACtGAOX6Y35is7AEhMR9NpzRu44Zr5BoObp6H2WudQ\\\\\\\/vgZIdnPboI7CUoyBTOiEkqPZSuPcOrrgaEMk\\\\\\\/y5Fuh2tH7VOHBEfHpcLx9kbBsugDXWmMc5tBjnlTBl5eR58cyPnz4AfaQ8hmTzHuDoLx+buHXFX8wWG\\\\\\\/Zc1rVN5aLj62XJxq4KobV8v00Fz5FlmyqIFvli7yhAq0nsJ7lmTXap+m8ra\\\\\\\/y4sLaXe9xygp7rHjQWEPJvU8v2NxZ9KFkaEW1bCIh9BHKZsnXl0F0pci2x6fzmbgPQKqmjZR03a0kV08dVwpIqd6PI+Kk6VMAImjySEhYnWXgf9WwvidZgpDm28mSER8BosB3hJ4xpDgMuWVm5bkfaYDVu0qjEDd6KneXa08pVwx2jqCk\\\\\\\/w1q05JeDxghNaSqlH1IqpFcTFupCQyybpYsRhuqytXWqM+QOLD4t3JV0KdK3vIZQjnNU5sHdBylWiiWNJF8ibm2PAazu7QQu2o5fPAhhkhZoTYNxiEnSCguDpTTItrYod8w47VPahOBoT\\\\\\\/4QaOpQvUSGwexYrsoHJ\\\\\\\/r8Vg4pmhv\\\\\\\/aoXRqHBnV9NC+rCfEK\\\\\\\/dtG86Kdplh7T7BQzaIUF6ltIBhPg+VD8G7ZE3XZXx6eCn2DXRpA6cPLUZThrnQEC3eGjQ8E18VDeBxPcX+LUP8TT2dKouAxu4e2MiCGb4CbQkHvEHlsJ2McIHCYkfrgzCquIdCOQomQz8UBgMaSPYOwecVG\\\\\\\/2MlWVp6hX6t46etw5oHYP6dBX5aYrK6KbTTYl0CMscANxy28hx4kbbIp4MyzVTg7oNdb21Le9C22INHFbgecVUm77Vf1mtho0snM81+WjQokA26zqkXCn5Vd+PeSA1eEC2W7SSNYUmG30Fkp0QLeCZkgUCjdE+fbZ8jKfdz7MyWTbHe0Fo9LWY9itoaiGSLQi\\\\\\\/lejRTMrQxICvwqHIpaVjFaWJPzValq8eqyt2qF4oNDLJwJ8fR\\\\\\\/FkVLzxPQ8EVhOr6JmiUpZ5IgdLht3zJI1etiIMCykJ+yJOnLi9mzzqjQQibsGoXemCJgnvX0pK7PivfwgrND5lOORKnmN2sOqV7Pj2rR2HLh+anQBDqIM47ZBxS8L7ZdptHZRPpqTcAGJ0VZLeXL5tlTCollFQ2Gv6ofoCumYB4RU7D0blvbSNxoVveuY9jdeLRTuypLsQeCLEex\\\\\\\/AIROho\\\\\\\/9IQDThKFaoOUckd6Py+JiC00J6LBv7T0pWV7rFxZVAke0U7bgzWsh0YuSmqcjCEUNbIzdy8srs4abWXkN7KnIcwHOkE9KDphmyNo+D3dDY+Uk6YZZr3nyZ1UuepMBMRFH7wda05o8lNZZ\\\\\\\/HyFLC4GZBAiUGnXwFpsFqfoL6El6lWbbCrr94TVyQpVoeQdyvYqXj8MRUhDtKsLWlXnNpv9ZO5my9lQAf+GIus2sSMGaM4KAuv13\\\\\\\/LtXhVNH5Z4TdqGn6qtdCARQI3TNS5qxn49XT850zBfL2FFd\\\\\\\/buwfofeCmMuDISDYC9X\\\\\\\/+npbzrtVP1S4XveXDvnAk2jCVk9MyXb\\\\\\\/PEj89tJsfyOhvH2oxE3VHxKhrhMUaukz25wArx5ebNqcpC82Sf1DYDEgz9MYKhDGVeWoOz+ad\\\\\\\/ujoiBJSJjI6YpP9hxEfGRm7er\\\\\\\/R6LyvaldIdYrtF8a4mUkC81X1qiDE61M+sqUZ8\\\\\\\/hImFMl9l9aAoTBKC7\\\\\\\/ecrK8qZX+T\\\\\\\/FmoJz3thgGndXuSYwtbOimqanmiJR6tmaVxJrwHmzyc9y+3p1lEnJrOmLIzZAOl7stJiztfCheJo7KtQ7AUcXfCqoSrrIblHyeuGMQlC0VSU3LYGEl7PR5qPZrGPP6\\\\\\\/kXab8KPl1xaUuhcPjxvkidf8Jg\\\\\\\/ERVvot7XLKXgYbxnTnaE2WXobM95r81uc5h0Wk6I3da7KTPXRxFDhzCikhNW8rhGNHUQO32+CYT0UzPntQgtIm+pejvOeE+L62s0PCzowc2MtrYDrblC+l54uBGCDPuBocG6++cz6aCrWyOJZYvZ1SGhqPCOXQMRCNhlVV49B8lys\\\\\\\/hfsa+8ughPPJ+gIq+RZLCGcUTCaIuiPSu4yYAXQDw42hpC9IurJJZl1Gy1DeXEYagAhw1Sp+yMxn1DK7FDVCEybJyYgq30k\\\\\\\/tRoffWrWF4jkt5yZaXQCnhvV1c6A0rP3WV+AZLCJ4G49ZJMVhVRg1UKvmvmqRISXx40tIpabzCvScfPrY+boy28ujsKL2IaB1yhVeL4j6BsE55tMFgFR3HX8qiAjAQsYKalhiBqibMxeWijfAtJHhelJ+8RIrJmp8SiNtVrn5wadPNrsBik2RAkn1A61+6x4BYJ5dIr7kwmwc+RLg0zsGQ+NIaxc4bSYMzOtdg+\\\\\\\/rO\\\\\\\/Vmet9FyEgULQK5VfKHoePSRgMx16wOG6OHsU+C2k2ZUDaweiF7BWDCAy5IDyZ5WI98uJdECiB\\\\\\\/6MTv27f\\\\\\\/B04pyd70bwbPpAJLwD9wlZjX7R3giFLhgmJBjQaehvJkluOd3wobDbLaHSxnFKxlql0+O+ZlbLbYGLtA4vEd88Nt+r8Gx4FHdUJP+qvKm+Z4V\\\\\\\/X\\\\\\\/oR4wunRtWCkdS5U6HjPP0U3uNO2VSQ4QAQUy17MmqNpv9WDWs5ifQ9RYP4LHle8sKeuTbN0QnzuSLI92sXhUVyJ+2b7leugcEm8gSL584kgNMe1WQeVwt75\\\\\\\/vuy4I71QVkSGHcnAoo8QNowneNGuTokFDRs4d360ROkigEQCSmP12Oe3MkknT4UBTgeGBf1WK6hNhPGzfIubRQeyNTsC0i2b+JYHBye0\\\\\\\/9TVSYAtDJ2g9r1acLscuXwsWJYvmnoK9LrcalLImXEgs1uEHTrlT\\\\\\\/NxBPuoscXcnv4Jw+vKtxaShEw9vuRW4Po7NPtDV+d+BhIAWDNmwFDCNRRNUG44BDZpONLDKb6d19o5GkfBU+Blie8iC2Cl595kB3iRXsISokZd6HXm+rr76lSMySc+OehTM4yA13H+C3H+5+X22eMkAiczChnGxNowyhOuiJMIHdGAai1Jr+iyKxn7m3scBAw8BP5fH8Cs2RwuAdYOHozCUb8RSwEb9k+QYpZSmosrzC+6QglkjJl3ZCLt\\\\\\\/UknPTC54NAV2WNAN+EgZH4+r3NkoSy8hchydRNdMZp\\\\\\\/tZUxFYtYZHrsv8JgVgzt0bR52P1ejLLUjWXR1koHo5SBYaQcv1bOugv7H4xzuaNR9ZTBuFiDzjCOw\\\\\\\/as0BSv5EtqkinGsk2vj3JE6EbqbzlE\\\\\\\/7lvQZGI4P6u+K9yFUl2tfLp\\\\\\\/R5ecDOOToVj5bgkH9O2thcAcQKzwy6JN\\\\\\\/U1i\\\\\\\/u6I1PD55nk+AF\\\\\\\/Dqac6ZMWrqiNWb\\\\\\\/wtuIY0pBPDkrwjuXdDZusW7HpsWk9XWAFWOQsiTqQF8hpeny8ZyJyYDJSbyv\\\\\\\/3iT12V1nEj3n\\\\\\\/ouzi5ajU18qWTzQ3bk\\\\\\\/QNCIkhYxe6soH7w4XaoCCfvMNFhJ9Jv33T9G8wvsNQG76wilnkoQ5Hr2VRq4xFfzAjNj5U+mu31u3Z6TlyQ9XDUoaNfF03DET\\\\\\\/d9URt5wcALcE4yzhLb07+G0ITk+CailoDWiw4\\\\\\\/67ZmvSyZ6L7uk6NNbGqDZJdE1gFTZ2aLE2VIMOHsDfMbeo0NZSoayb+RWtSvq\\\\\\\/U4sB+YU2g+tkhbLu36Let4ys7qg03xE5nZRiz3INo6qZiO4KNP\\\\\\\/ngBxgN5mgw\\\\\\\/bBMBfvpEZU2fdGK4c3PGj\\\\\\\/mjV5wvEQJQXslI\\\\\\\/3Bxo6caL5DKvSmTr\\\\\\\/FR55RLYlLRynL7r\\\\\\\/67uTk4wyUJoG0Y6x367\\\\\\\/IxawpeXDebGFoeOqvAPg9jNQqZxjICzsA25k8yrWNW43Mm2AkNIvcjf5Ep1N8BKxUuLbUgK79scbKv5U0KGDOm8aeOv9rQEqPV1GmWyUKvSQFwSWzw7EdDNjabT6ZTM3AYSEFeovITcTIeEoYl2V8OOyIXUYItDpQ9l9foYPUXOPuNp7OsjI1teq4Q3eRJtrELiAIELU+1+4HX+1q6u5AtDnG2+wPTarnOHDWqNQ35IaN\\\\\\\/imPu1G3EyVv0IP2Jud2GsigsBCZjWWQwObA5xkdDoGpPWkl2wBEvOIUuK+V8xKaO6osXirEQyoMexHw3QcvNMTZfhqpyPnrrA3yYkFBOps7yHfZ55uF0LJIeq4FcSmsmwjmXuXGz7JYXIMXU+VaD7G2jFrXyREsBoehLfhmf3W1JmxiGMhs6nsB6nuunBzBXJHpgzPiMF3E4T29I01EKkafkV4XLJH1OhDd54sy81OTLasOM+wPy+GHvi4WjdHNhs3+RRskPVJ3PtsApP9nJtRzHy4JvBI5SVmqP1C9qwi3ve7PD9ARz4Ls8W9MomUYlqV6LtYv0HLXr9R0GHcZFPqu+k8rILIrecUCjnJ2pPD1XGKEtYoLO2xOhl0WBeeE05z+VsxL619FMrKy23LXr4bFYtXLEO+QltoqL56iEXqQXTX7CIVdg5s766PPAqghu9SavdTvEXGfD3OU3w5yvfA5fhwZ0Jk0VLkjXxE\\\\\\\/tCguuxugRNJgIg2RigkYptc8ybstVY7hoQNeBBI4uoNw6XT8QqXitUpmeK52UPwYuJ9kajQ\\\\\\\/r3WNnETDuy6A6qKeBHOFSILpM+H33Z3zKTRWvfzcWVXgJDd+x2G5wcJ4Cn\\\\\\\/dtDPf50Bhgpr0VGT7QwCABfcDFaDqp94+5AEpOrdAPgTJvrxI78H\\\\\\\/A5H0tG\\\\\\\/IG7QCjoU1ATHbLfm2mNzMSzoMiFKq62nv2fZgSKLxHztVdG6QZ5enzEFRvy8uxbsBhoPvT9xV06LCUOtwL6uDc4Zs5Y05NpzRn7n19ybNyf2nS6MgMlDamnSgApnTtDXlGCXIu1GxpfYX82jnxFAMSoGZ0vT6W\\\\\\\/LnfMSmNVZsnGlE+MOKNeHfhQqiYTcdqRyg547sI8GE1lSzUUhGMMBOIwQeBpN8wbZI8QR+t1lIy4IeCFy+slm5bdTww9mi7npAdkWn1Gv50yld7Kn2CYS9L3dzDYt1QSNhutjCkP6Jo\\\\\\\/ZDJbbBmZ24oMYb0YX4LnKLiH+BY2cLJN6Ye\\\\\\\/XwoHj\\\\\\\/FWJJtdIbJePcFQf1sFIXhkA38qySfnm2hIRKFs\\\\\\\/UQ+d4jt5Xs6YdPRxuvw4m8ngMLp\\\\\\\/JZS3GeG1coJ99UUoeh\\\\\\\/ZuaOzlgsDZb8QXZ4NApbD37ERW9uOHJQNOnjelV\\\\\\\/krn8KOR3RXsP+iMAq5hqdIcAztsAZUbaNmjEl+Y33TsVQ7rQ2X\\\\\\\/kykmpda4mxO4UmZ8cQp6i9KtaTmBeXE4cnA64bGMmnJUZPkogv7\\\\\\\/Fb\\\\\\\/sYRYUZW7k4pQF25SWRaRyqH0JYqf+UknmDcnfj9+AdGulgR\\\\\\\/AKKDIIdcmFhW3dFGmQJKJH3ug24I31f1ZSBFXoK+oVdx18sO5nQN6z3aJ0j4AoVzGg\\\\\\\/AwzoaRAfGrO7EiwnR7NaqzJUrlJPLLx5uA1xcVjM44Y08uQeQ8s1rq7EDNzXem26pZUL\\\\\\\/8q5X4p1szyIuT5iSkDX5SM+EgeaeSkVAMiSeTCo6\\\\\\\/PAei7W7ckxqLuy1t69AevpN0NSRsUR7fcTB4gPRD+fDQu5xt40e\\\\\\\/nr2shp8UDuLZB2HGtUeD\\\\\\\/K\\\\\\\/JD4Sqwg7Z5PpaQgs3IYIhnSWTYav92Suv23ZFyablWkRK2eGSPElLalxg+kakW4tC9HTNfhNs1yVu0AXqEcf39II789STECicQ56Ajhea7wtm3U9qjEEQbLHhXPdTHTuxxKSb\\\\\\\/cssyE1v3NSbovZzyCFZ2v0NvQsrYmXEdieyq6xt1RNK\\\\\\\/lNEpW2RMSKjKYk2JTOt80kNEnGMCUy45Qd4m5z6A5e+JrUcRQJgE7VWoHwPQiBoLUPrX\\\\\\\/NqAcZNZsoTSDhu+VrWCZGjfmaCw\\\\\\\/zR7PCYb3TJa58tsNCh06nfHeJmo4xgy0Nl3dHTbfG4HU\\\\\\\/9UC8koPw+VsPsmh+PwRLKPIz\\\\\\\/jT3b5jG6y4rEqUnxC\\\\\\\/tdnDvtADWTpXPSVpTvGcqE1++QAt\\\\\\\/gjzBP1WpEDLged4fsVPcQJQdEFyFnQmdoU12gJqns2FCZcmevi3EJn1NS6TnKD1SpiKS2tuohtA0oLH7RtZ3DSIVfKx16zeiRGd1+wlcEnO9n6lV7TKWBWBrnRmEjriOcDmhV89ZMAEq6MD6YZIk7HHO1QkJcm24xLx8tx4Uf3O6G\\\\\\\/V5FkaaR\\\\\\\/4cct9HjPhk4xuOMS2gM6WSgj8SkgbGIRpmN2ptZo0NYHI2S1MdB14JEGsE4rm9NNSQmClsg5+dBzZDbjcTEb\\\\\\\/ARL9+DIz\\\\\\\/C5zzM9I8JXlmtjVewI7xsP0EdpXI7ybQffuksjkEkiPJnizghf7Pt0wQK1ecVVF2dTTL7zpTRvAkVbughsEY6ql78jR4m8X+UB5xCzZJ342dvUL7sMcQ8liBtHtppBXcVcbLH3BcV7RO0OrF3U\\\\\\\/YQo1icAadzXbZGLW44LIBgfq5pZikum9X+bY\\\\\\\/jDniah+RS9D0PsCd8miHaxtesc+neQJFpPISlX6Gpo4aJEFkhs6AVcmTB8M5YOpJjVt1q+b7Rpy22+Fz+jZ5AOSzRBvfmGDBCOfrJpu4o\\\\\\\/gJb10712uB5FA2PnvDRGqyc9Xb+u+ZchX5pCPMmM8uYahU8\\\\\\\/\\\\\\\/EkMVj0Pv+UTWFQLc6Q2rG8ciYpoXEfVwl8pMBdccKjoVcxZcEjPCYOjZ7IdaN9XdVFRc50bC7MomxW4Bib6bGufT542FoS1qvSbjDgMsFKnYsUNRRqsvD\\\\\\\/pevQufiGHCg0fa\\\\\\\/FlIItSwTnleWCkF9KXH1j0Bxm11gWAznGGHHKNsrWisPoWr+WdPPyChwpPTs\\\\\\\/2+EjVIG2aGePhZMIZiF2Mi+JrNNyo6KD4fa0ElSxMJAXkZbQ\\\\\\\/pwRj2kKiurMI06fDQy+QE5UJZsxsPIaI+kiSGfJOh+ZbsYQ\\\\\\\/piTz7SqE+asa+tW+3\\\\\\\/JlSqCOWmVfkmG5O3j\\\\\\\/Da8qliHwychJLDig8tPO\\\\\\\/i87LwcN2uGt4SXQZ9wptJLuD5qEh8A9wOP4YRHbPEsBokp\\\\\\\/GmwCfnMjy06LBh2RZphyWDJRy\\\\\\\/WM51yN3h7CyiC6+1p4ZEASkF83oz4ab3mc\\\\\\\/oQ1F3kArqkwai7YwUFYkvH8ANu2Ii47GBX1wXu+PA37\\\\\\\/GUVqNWjS9em5sjudU3dCfojG9c29m5Ljb+rwRAwOvZ4NtPaCS\\\\\\\/hPnZ0sytZ4jU1103VAINZQUq7G\\\\\\\/HxOw7bAvmLe12pjymuYZ1Jv5Xp9QqKnEODt\\\\\\\/98ieta9iFTrrXCmh8o2IcPwTgcyKdUdBKsnuy5zVEAj\\\\\\\/KThRYViqJfZpe9DOY2ZxKyXusEm2CUqDn0OiTLpFnus1dmWcJZRD7WDIRft3LWRIMNPETtqipZIgT\\\\\\\/xEE65awvn8biF5F\\\\\\\/4\\\\\\\/q9edeAvynwBDIaQY22hDIPxj0qEODTWQfBO4RX+8qLgnTU0uCgGbL0ZTMJAb0J33lNioQ+kE5JAPk9C\\\\\\\/1TngO1yyXVMqrkMRzSXBrybNxSws3cadmEG4jqNCo391N7Ba6ydTqc1Gfc7IPX6NEcKszKhVb7fsLWjfqeYmxAM4obC3Ffy7yeEIYSQBM5037znBjm2Kmz+mN19rqbN7xPmwUZSDwE2ZZ0POgRA0arbcwJ2\\\\\\\/RSeJ97T+fd4JsXw6zBw9RrUZ7kpyknBtlmR5tLmxJmZt5uHGDiqP4Npkbu3M5jw6rbeZlwd1B1eeyFY+3LYKREBWDzyZqa5QnQ8OkpVL6vJTbbG2LDoutq\\\\\\\/4r16cJYwVW9JuIEqX2xLcpmXRGJHQnR1CSB\\\\\\\/V1N4j3o72OsM3mWioFUU86wrcwkHjuIB0agjNxTuOCtp16l+pTO4pd1zrJftSPOkgOI8pudyC45hqoqyK7SiOK7WB1LpYJsphX6SinSy2Ddi7L49dhP3WLvoiR4YH7f7Q0Ef\\\\\\\/2FSQo3MRmuqzgTScRr2KwXJl9rdGuHm0aNVu63uEmxwnjqH3bylWui5e6t8Tbwn4ovLmfGxC6UUoPdM27\\\\\\\/Z1bvMIvX5xe3jVEyKaZh2AAwVf9EdkKATg2aU6OI6cL4t1X5Q2AwixW16YnzIYG2ao8kBCLbaejBoyOrnN0YtWF5ibwLwAO\\\\\\\/vqBUdyQ3VcBIOeQ0kETr1xgDshTr2SxNizY6MheI1Ennby5SEBRIwYcmkJz9E683Bb2UzKuLzcCsz4ka2yM8D5wh4LP8RGNIcMvZvwYcxbyqzKRlMWYzgta\\\\\\\/FZ51\\\\\\\/wq2BFcZCyUdteg\\\\\\\/7OWkiOT89i1TIiaCC8hUuj5gml5pDXn46ZoRKZAAQuH+hlRFpt3RQgFSOPAb\\\\\\\/5mU1pTY2pAUYx5rCeZsNOOc3INjQJZnhxOVq02oJhGAwWuaqjeO7l\\\\\\\/v\\\\\\\/d\\\\\\\/2tYvOCWv4MRnmcfBCcyN1XKDEmZyHgr2VtjOr2mWN9BtD3fmpIpeCAs00\\\\\\\/NIrD30Eys214XuCfaIs1z\\\\\\\/+hV93yMGQ+rmQDpqI+yuOPn8FiJIWaxRgwVInj16cbWE4EwvdRn440Y8aGgYUpOXRyGcgvC9LYFYYBFXmY0tmU+oruG4mBPl2G1jdAJwX0TkC0BsiralMaJ0\\\\\\\/GUdS7Q6PPRWlFKUnyOu+gdmHTfU5ruVN4IIdH1tfY\\\\\\\/JGrgebHS6yEg965bd\\\\\\\/kZ8YPFxlRXKQ6oXW+zpzn5nge3u6tkJOuR8HMigMZ0xIMMkmDby4iatLbPY1eX9p9KHme1lPgd4c4Zj5alEGpDE7vQhyjARCfUEwYQ3Bf50aqHeTUUGqCJ\\\\\\\/wt14RumQzRYiOWuFBh29R814YM5u6dsUSdskVHkuHsr1jXWIuwpiXleTLJgS2HFkfhQTBYMBRt3dpMNMcHjkIiTq+ofFKEP+EMLYXlS0ym6bzwc\\\\\\\/AxJ8kZAergRV+Sfp6DpB9SVxOruTHrp2sAuKPDfibPDoxt6viZepR3CceLQ2XedgBEhHiJQm5gjes858A8ip6HOQBY24QyNiwROtxbFOJU6Gp9NRmCbbUlIITsRWW8ZiOxpvLEl1Gu60EjvqebLFHj4D8m50QjNSd7QUgHW7aSX+1X8mc\\\\\\\/pMJ2O0fKaRuD2J1j\\\\\\\/Yt1oGZB9veYvD5Cf7q+ocaI0Mb5bGiDz7DYfs\\\\\\\/jowTAO\\\\\\\/ePd+xhm+9gyv56X\\\\\\\/fY1pQP\\\\\\\/cFJG0ikZ\\\\\\\/QRWndRkgPCYXvdXIXcWbghZYVNj\\\\\\\/0j5VUI3wdPH9PFWKHSuIfLZmpe+XvA\\\\\\\/1gvHF0o7cHoyBdnHtNBdmacM\\\\\\\/steqn9LXVYrlpTdYjdjlhsuOcYdsUl7MGOI4QaYOCkN9mcYxdxiyhZENwzp8OjxeHWBc77gE1iKiOo1J0Ms5uO0Qc8CsZaQUX1maONnN22FlF1PKZLaYU4Wo5QYpa0FY5tIBkbrgQsUfi8rY7d4uRBExGM0YQu5GDsNV06XAeQYSZ2HFYCpmizUZX7k0n0npkjghYIGQv5NeXEKhroLwLp6lSxJ0goj0GPe8nPVUs2h\\\\\\\/Obf+r9JXonhBXC2r47YSBkxELvyzMBQj494Y+QMSHcY9\\\\\\\/QNeQz185OpMzCK\\\\\\\/DuMUv\\\\\\\/U2FRqrWVVuOu76tKk8gNLMuhxNWWW5vWQvoazzRo\\\\\\\/mnKEZt5ChZKTDUMZ4mqoaajYSEvclJfXOg8yccSmvy7PKCXDAl8tm6TRUu7Fe1BOQCO8PBlHnvT\\\\\\\/byvct2Od1EAkmz0Lk0G7jibs2BGrJBe4UUxMZZcrJU0zESlEtUd3wVAN1fY\\\\\\\/idPL3lgiaI7eJBH9pfosuoEQZxNIGOd8Nxwmr0J35TmOWBWVffoP98949cwRC8fWTlqOZPIB69A0eSK8vwFDWbZcjnXyOPTQofLyJV\\\\\\\/uZLVVlFtWkezzMOYgKvz8dwHP77VCQTwB1eDLpmd46vPmW9lA3zuStxdqKbenVHtyc9vu0mJY8y2ub75mPME\\\\\\\/OSm1AvYi+6X9eG2LZctI7\\\\\\\/SJFXv2QLzrw6bxbVnzF2Zsc8g5kqjBye2neWfqKSpp6QlqGcItk9fRyirbhFLwa7En81vK2wP1A08ZCUfmfgmqz\\\\\\\/P979HlYTDkBDPZg8auMBJWYV5tLB565v9exr3uSe89lwPVGKAJ8pDtAt2XNNVL+MWa4JYLzxXeWtMsX4KSZkzGrK+XrZRkKsvIvzp6vF1sLv5qn8GC30STL83FmwiaRUW7tyvCIwk9P9SZ5YYf1fo7QpHpIM+wo8JbWUc69uJbzGfvaIQ7+vY1GbltdKPA4q7ICzhNa0wJCq5BUlwxVjogwAptM9wdWMdlRdgkI\\\\\\\/XjvViM6oG3yZegY5aHeRssVBJ0NJX2E0SWVEOhddBR\\\\\\\/ziw+oN4izHZLZkGcHgBbSDfDbq1R02IJ78FIUvazb76TIWE91D0e9zSiZmRzdXSWmOE+ki0KVODIzNOuNoTgtZNpDAf92TPLn9YCh8C2+W6\\\\\\\/rn5rBdW1p6spIVl9qux8Vt8Xz6ZpayGUHnCfUgaOVm3gyXWG\\\\\\\/VkMeu9HF00JT7kZeDWsh8MG\\\\\\\/sTmNtHOPW257jf2d9SHHwgs+y\\\\\\\/8mBpEcqD97z4V84g+zv3INSBY\\\\\\\/jB+ZbFcBQypqI4CT7VRwNoPqnkPO6fW9U7fFz3O8WJHmpKNt5nQ1L8ubbQ+AYIDV8nijAhP0+ZrHm+5YHfC14aCKTZloqqH9hINIiVXRjJqLL9waWE8CWfhtcj9JHb8BKhdnu3lJMpSoenotWWgsKr71caILJuZg1tkpsMM3jl6EwMrKF9R4rpMK5Cjp0EZsrvca5F1pQAuPl1E8wbw7KJ8VLB5UkNaWj9\\\\\\\/iGEOF08DNVN767nNzpOT\\\\\\\/cpoktA887M66lgpfRzfOrXAs3QH+dl5jPs\\\\\\\/E+f4\\\\\\\/7KKn77tNatvBpHQtNZvMW023Gh3OJv9JM4u7vEQ0nblwevKEM7i0gapuUAtJS0S4NVz\\\\\\\/DwcY3IoJaTzbRn3TV\\\\\\\/mWbKi6eWrvS48LItvp4DchUNsV4UZtbNttZS50PqqyOKFwAjS9Iys156ZapzhonjJm1tR2Fy1Cog2CMLSyYlhMEE0TgYvMSeWJTrKjC+338ul6Mey1SqK\\\\\\\/wv8gHwvzLUEHVbWlRwLpibdb5hPTgSIHD4F+4f3uIzs9dQaSMvXNk6AQJMKAVZrnruvlQEo8RuKZFGIW5H\\\\\\\/ymP2tNXzNeg64En8QGi0MYbNa8a6GRRKYF4pc5xdc\\\\\\\/iviCab5ZzmrHE3XSPtdsS6cNcyOMvfpiKW9ZI2jAnIa8OHgzygcbDhg4uZUPeY490kokOuXUrN+pQKUSKpwe6PAK1bzdKaLnGylF6T2tjua7RXkN2+Ji3yNtj0XAgvFcsIlz3La\\\\\\\/V6jZjJzeDLHs4CSYbC6JCZ0e+jmrHxT58gFUrCWoaJZliJBNuYMXeWkmiy41zuyc7jhVf4bAu12npXzPM0ZYspNneLAT5yrVjdLBuYdSsVU6OJXFuA7IbjpdBhMHoODCncmh2amOuvnU3j4B0RFx2nf98stXtbjDl+nccfdaU6v1kyR5bPmlnxyVO6vWjeLdFYxqARczEEioadXXUxQqcGFHI48lk58uzw4zZ2a2SXl\\\\\\\/q6BYJH8fyJChBD4NUYPRufjJA7sC9npjMGBzFv5jeiwyFJ3ivUVaErHuvLsYP0Rm62buWOfk078qbHzf3w58FAie8jn9\\\\\\\/Ch+9+P5IOIAgPYKN8HMMO1yGawGeqk6794gcS36EQqSQBm2hTAZk8uyP+mW6Fu3mDf6MEUkzHid1b5GS\\\\\\\/uJ1xvV9wGWhjaaLG7FMXZ3Wpe4tXdqnps6x8LgAytPaIfMNw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022HA3SCL8uBXBXBnanBmKaMQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022da13ec38ed8d959dfe5ac8cbb75fae19d32782e89f193c614cfd3c46b0ade90f\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022Q0i9UY4jMtO+IZXihSVKNmkA6MH7gCXjDDgVAS2V8SZ2zXWR6tMO6Ti\\\\\\\/pw1edZBhtugE\\\\\\\/Yn3xDLb+bO64zTtyy3WoQtIiXpVR\\\\\\\/1wzvNs5alcLIYKH+PujfJGqePQC746IdE2iRavdSb0crd4DA06uLflqikGhXp1rkZRoZHYV5FoJBB5ch2NqTzyuUWrof6N3ngX6fHP5BMLP1EvLEtNsTxXZtvl+wt8V1qDoUa0ZIUgjoTUoPxK6QRgAFnkVfFicq5ZzBtW8dfkEeN6Z36zgPGLXF79mU\\\\\\\/c9bVaZiDYCX2VXvLE8zowhzmUXCqP8c8oPGXlGSA33MIINnazochcKDVl+Kam0hC1bb5eDWIqDxutlZrDAMwgIzoRKSKcS9F5Vr4RUrK028nLvzAF8WvgTnL+h2XKatl7z1JVwM7InqhVVRZl4FZu+oaDsY6hO0e0EDtAuRYBZGT\\\\\\\/L866UAUQl5HfAWdYHX9aHqKKIFwFQwEmLQ6nsGnixi48ZaWfQLNsRyFdzYrJGyTo\\\\\\\/V3Yz9GdA0sKINNhNrquM8UoB7QVc1XdgqBUQCPu+KDcyadkdYt5SYja14excvz+69IvmJROI7o5SW3Y4vkBSJ6LIp70HjY2WaY3wad1hcc24z8Z\\\\\\\/ggucp9EAftRtuByxNZeDTaTtm9fIMfkb5v+9Pt4l+CCS0ud2dF8vi\\\\\\\/mLmNZj7sgSDRoagtGU9YlX\\\\\\\/QG0E8Axxbx1QwO7r+HSgzJ2u0179O+BsONUyh7+98kgm3HrYl7TtaiPBnfI8MmouaHAiTLGQv1HhFjoH6JQFPv8hUta+XMH7AGq8tEhseMdokxOTvuiIimpMFRwE9Xn3pmLt0rh9lQtuLUP62PVVfh\\\\\\\/kmmV5nGvJH2sQQN\\\\\\\/MHdGZ\\\\\\\/933ElSZkU\\\\\\\/ubIjyTn7zYL4I2dCrLBUeNHky73piNvRfUlRg1EVeHSZllDtfGEwGzcjPP1uzW8J1XKLpx2vgn\\\\\\\/hjBrNX9EaYD79dBiZLbq14CArUH7KSGIzPJ73oIoyfsk0RFhQZu5nEoB0c656YbmBSAP92YGJWmvOndd15XnZrcKfHiG7apb4hEbccqe1Q3tYE28g9JxeJ1uba9PS51Led\\\\\\\/JjYUaAsGcIN71XJYvmczSujip5YKdCXFaZt4Zs4xi1EuQcF4w0COcNZCDfM2sely64tw80SKKUSmNY\\\\\\\/cU8fl0iOS0KlqboNfgf2My1mKWOxg4Z7RX0P0b6k2TH0\\\\\\\/w0ETb5DzJWYZvUjs1WAeXyVmLpCMcBuRUHNonhii3Mxtzb\\\\\\\/v5xU4ivi\\\\\\\/+ZsvdmN3mwJgnj+e9qplx\\\\\\\/KBfEXzKSavADr2l5gD5kaLeeinVQuchlSayiryTlLujYoFCBlN1pKPSe8I45QxfA5WoemfK2n\\\\\\\/35UyyZ1AgSSKVfrM0oVFryOJZbr51ffarhhsh8hEbvdYpaKDylpyN51pnj1u602b46FlKqSvKPEldsS0ZXwix7Q5XGQ9+2Hegd1bdOPIBHNvYvLpPsmq0TYI2IVh+UG90x3tlf+glRf2P4zq\\\\\\\/wsdCdOa+S02bJc0zbKvxJIIlXZhKHKYOhFm97IrNxhXMdB5s1iS5XZPo81s6wJzujZtIt5ST6KbvImM7XsGsM+M+U\\\\\\\/W26nW\\\\\\\/zA9TSzUl325NgUh2GMQElTX\\\\\\\/O+Evoa4rR9t\\\\\\\/9F1YvXTkSCxQwFyMnXz5OsgKtEaBWFrIeOmF+800IsiqoPUx8rpcTHRTLLfMrcDR9w21+PtVmpDbytabfV0NvRWJvo21SMVG0xpBGaYyRHq9NzS8l8lTDcnyKmEt5AmttChJWfO1MGKom75sD33KYALamjgsF4ec+OZ9aUBvsW0BOvTpteedDRegyx7WU9Y7GBpbL6pSp7Q54M2WLdTzKgjZyaJ5UfyOscVKTE4yPYg6L+mtICk1egl1hV4uTGZ9gMrSmIs6AM4FkKtyP4cP9uVSpVO3PZfyVks7mFNO24HaYfNzC8g6KIcMoOZtXwsfa9v9wqMRdlLQq3TDldhKuBIFwW6+Ng==\\\u0022,\\\u0022iv\\\u0022:\\\u0022EvDya9boYxmeZACTr2KBDA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002224e74083c7ea5f83a135c535f5b51c598600084375674513e56c0eec723caad9\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-494269 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-genomic-prediction">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/10/3153" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-TxfgrOoVkSwnJNxKK5zEiHuLzOgjj2-olVhJhKyHhvE" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/10/3153" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-8TltP_A5hVVdhooPrvnIZyXRv3LZ8ej2a-UFPL8Y-to" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/10/3139" title="Genome-Wide Association and Gene Co-expression Network Analyses Reveal Complex Genetics of Resistance to Goss’s Wilt of Maize" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/10/3167" title="Genome Size Evolution Differs Between Drosophila Subgenera with Striking Differences in Male and Female Genome Size in Sophophora" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494269" id="node494269" data-pisa="ggg;9/10/3153" data-pisa-master="ggg;g3.119.400406" data-apath="/ggg/9/10/3153.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Multi-trait Improvement by Predicting Genetic Correlations in Breeding Crosses</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-1991-5310" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jeffrey L. Neyhart</span>, <span class="highwire-citation-author" data-delta="1">Aaron J. Lorenz</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0001-8253-3326" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Kevin P. Smith</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3153-3165;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400406 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494269" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jeffrey L. Neyhart </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Agronomy and Plant Genetics, University of Minnesota, St. Paul, MN 55108</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jeffrey%2BL.%2BNeyhart%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Neyhart%20JL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJeffrey%2BL.%2BNeyhart%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-1991-5310" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jeffrey L. Neyhart</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Aaron J. Lorenz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Agronomy and Plant Genetics, University of Minnesota, St. Paul, MN 55108</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Aaron%2BJ.%2BLorenz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lorenz%20AJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAaron%2BJ.%2BLorenz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Kevin P. Smith </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Agronomy and Plant Genetics, University of Minnesota, St. Paul, MN 55108</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kevin%2BP.%2BSmith%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Smith%20KP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKevin%2BP.%2BSmith%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-8253-3326" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Kevin P. Smith</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:smith376@umn.edu" class="" data-icon-position="" data-hide-link-title="0">smith376@umn.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/9/10/3153" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:494269" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:494269/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/10/3153.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:494269" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:494269/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/10/3153.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:494269" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:494269/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/9/10/3153.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:494269" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:494269/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">The many quantitative traits of interest to plant breeders are often genetically correlated, which can complicate progress from selection. Improving multiple traits may be enhanced by identifying parent combinations – an important breeding step – that will deliver more favorable genetic correlations (<em>r<sub>G</sub></em>). Modeling the segregation of genomewide markers with estimated effects may be one method of predicting <em>r<sub>G</sub></em> in a cross, but this approach remains untested. Our objectives were to: (i) use simulations to assess the accuracy of genomewide predictions of <em>r<sub>G</sub></em> and the long-term response to selection when selecting crosses on the basis of such predictions; and (ii) empirically measure the ability to predict genetic correlations using data from a barley (<em>Hordeum vulgare</em> L.) breeding program. Using simulations, we found that the accuracy to predict <em>r<sub>G</sub></em> was generally moderate and influenced by trait heritability, population size, and genetic correlation architecture (<em>i.e.</em>, pleiotropy or linkage disequilibrium). Among 26 barley breeding populations, the empirical prediction accuracy of <em>r<sub>G</sub></em> was low (-0.012) to moderate (0.42), depending on trait complexity. Within a simulated plant breeding program employing indirect selection, choosing crosses based on predicted <em>r<sub>G</sub></em> increased multi-trait genetic gain by 11–27% compared to selection on the predicted cross mean. Importantly, when the starting genetic correlation was negative, such cross selection mitigated or prevented an unfavorable response in the trait under indirect selection. Prioritizing crosses based on predicted genetic correlation can be a feasible and effective method of improving unfavorably correlated traits in breeding programs.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/genetic-correlation" class="hw-term hw-article-keyword hw-article-keyword-genetic-correlation" rel="nofollow">genetic correlation</a></li><li class="kwd"><a href="/keyword/genomewide-prediction" class="hw-term hw-article-keyword hw-article-keyword-genomewide-prediction" rel="nofollow">genomewide prediction</a></li><li class="kwd"><a href="/keyword/cross-selection" class="hw-term hw-article-keyword hw-article-keyword-cross-selection" rel="nofollow">cross selection</a></li><li class="kwd"><a href="/keyword/simulation" class="hw-term hw-article-keyword hw-article-keyword-simulation" rel="nofollow">simulation</a></li><li class="kwd"><a href="/keyword/barley" class="hw-term hw-article-keyword hw-article-keyword-barley" rel="nofollow">barley</a></li><li class="kwd"><a href="/keyword/genomic-prediction" class="hw-term hw-article-keyword hw-article-keyword-genomic-prediction" rel="nofollow">Genomic Prediction</a></li><li class="kwd"><a href="/keyword/genpred" class="hw-term hw-article-keyword hw-article-keyword-genpred" rel="nofollow">GenPred</a></li><li class="kwd"><a href="/keyword/shared-data-resources" class="hw-term hw-article-keyword hw-article-keyword-shared-data-resources" rel="nofollow">Shared Data Resources</a></li></ul><p id="p-3">Quantitative traits often exhibit complex relationships with one another, with ramifications for disease epidemiology, evolutionary processes, and plant and animal improvement. These relationships may manifest as genetic correlations, which can be caused by shared genetic influence (<em>i.e.</em>, pleiotropy) or the non-random association of alleles (<em>i.e.</em>, linkage disequilibrium) (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Lynch and Walsh 1998</a>). Investigations in quantitative genetics commonly assume that many loci of small effect govern traits [<em>i.e.</em> “infinitesimal model” (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Fisher 1919</a>)]. This suggest that a large proportion of the genome should contribute to phenotypic variation, a hypothesis that has been supported by recent genome-wide analyses of complex traits (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Mackay 2010</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Boyle <em>et al.</em> 2017</a>). If true for multiple complex traits, a natural corollary follows that pleiotropy or close linkage of trait-specific genes is widespread. Recent studies attempting to identify quantitative trait loci (QTL) influencing multiple traits using dense genomewide markers have provided support for this idea, reporting extensive pleiotropy or strong genetic correlations (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Korte <em>et al.</em> 2012</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Lee <em>et al.</em> 2012</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Bulik-Sullivan <em>et al.</em> 2015</a>; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Schaid <em>et al.</em> 2016</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Deng and Pan 2017</a>).</p><p id="p-4">Plant breeders routinely select on multiple traits, but progress can be complicated by genetic correlations. If two traits are favorably correlated, selection can simultaneously improve both by tandem selection, indirect selection, or a trait index (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Bernardo 2010</a>). Unfavorable correlations, meanwhile, are common and often the bane of the breeder. In crop improvement, notorious examples include grain yield and grain protein content in wheat (<em>Triticum aestivum</em> L.; <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Simmonds 1995</a>), grain yield and plant height in maize (<em>Zea mays</em> L.; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Chi <em>et al.</em> 1969</a>), and seed protein and oil content in soybean (<em>Glycine max</em> L.; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bandillo <em>et al.</em> 2015</a>). The directions of such correlations imply an unfavorable response in one trait when selecting on another (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Falconer and Mackay 1996</a>), and the underlying cause will impact the prospects of long-term improvement. Selection on traits with shared, antagonistic genetic influence is functionally constrained, but correlations induced by linkage disequilibrium are transient and can be disrupted by recombination (<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Falconer and Mackay 1996</a>; <a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Lynch and Walsh 1998</a>).</p><p id="p-5">Genomewide selection has become popular among plant breeders as a method of predicting the merit of unphenotyped individuals using genomewide markers and a phenotyped training population (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Meuwissen <em>et al.</em> 2001</a>). Typical prediction models are univariate (<em>i.e.</em>, one trait), but multivariate models have recently been explored as a means of borrowing information from genetically correlated traits and improving the prediction accuracy of both traits (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Calus and Veerkamp 2011</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Jia and Jannink 2012</a>). Selection on multiple traits using predicted breeding values would proceed as if using phenotypic values, relying on procedures such as tandem selection, independent culling levels, or the construction of a trait index (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Bernardo 2010</a>), with most studies of multi-trait genomewide selection using the latter (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Combs and Bernardo 2013</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Beyene <em>et al.</em> 2015</a>; <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Sleper and Bernardo 2018</a>; <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Tiede and Smith 2018</a>).</p><p id="p-6">These models and selection methods implicitly assume that the breeding population has already been developed from selected parents. Therefore, the genetic variance of each trait, and the genetic correlation between traits, both of which determine the direct or correlated response to selection (<a id="xref-ref-25-3" class="xref-bibr" href="#ref-25">Falconer and Mackay 1996</a>), are fixed parameters of the population. In addition to more accurate selection within an established population, multi-trait genetic gain could be increased by developing better populations through deliberate selection of parent combinations with a more ideal mean, larger genetic variance, and more favorable genetic correlation. Typically, breeders select parents using the expected population mean, which can reliably be predicted as the mean of the two parents (<a id="xref-ref-6-3" class="xref-bibr" href="#ref-6">Bernardo 2010</a>). Recently developed methods that rely on <em>in silico</em> simulations (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Bernardo 2014</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Mohammadi <em>et al.</em> 2015</a>) or deterministic equations (<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Zhong and Jannink 2007</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Lehermeier <em>et al.</em> 2017</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Osthushenrich <em>et al.</em> 2017</a>) have been proposed to predict the genetic variance in a potential cross. These procedures model the expected segregation of genomewide markers with estimated effects, and early validation experiments suggest that such procedures may be useful (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Lian <em>et al.</em> 2015</a>; <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Tiede <em>et al.</em> 2015</a>; <a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">Osthushenrich <em>et al.</em> 2017</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Neyhart and Smith 2019</a>).</p><p id="p-7">Predictions of the population mean and genetic variance could be used to discriminate among potential crosses on the basis of the expected mean of selected progeny in those crosses. This can be quantified by the usefulness criterion (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Schnell and Utz 1975</a>), or the superior progeny mean (<a id="xref-ref-60-2" class="xref-bibr" href="#ref-60">Zhong and Jannink 2007</a>). The superior progeny mean assumes selection on a single trait, yet if two traits are genetically correlated, a response to selection would also be expected in a second trait. This “correlated progeny mean,” as we will refer to it, could be used to further distinguish ideal crosses as long as the genetic correlation is known or can be predicted. Though much research has focused on predicting the genetic variance in breeding crosses (<em>e.g.</em>, <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Souza and Sorrells 1991</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Bohn <em>et al.</em> 1999</a>; <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Utz <em>et al.</em> 2001</a>), little work has addressed predicting the genetic correlation. The simulation approach codified by <a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Mohammadi <em>et al.</em> (2015)</a> generates such predictions, but their accuracy and utility remain unexplored, and the use of simulations can be computationally burdensome for a large number of potential crosses.</p><p id="p-8">The ideal selection of crosses to simultaneously improve multiple traits has been the focus of recent research. <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Allier <em>et al.</em> (2019)</a> presented theory and a deterministic equation to predict the genetic correlation between two traits in multi- or bi-parental populations. While they applied this equation to the case of parental contribution (treated as a quantitative trait) correlated with an agronomic trait of interest, the theory could be generalized to two or more traits in the traditional sense. Additionally, <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Akdemir <em>et al.</em> (2019)</a> applied a multi-objective optimized breeding strategy in simulations to select parent combinations and improve two unfavorably correlated traits. This approach solves the multiple objective optimization problem of maximizing the genetic gain of two or more traits while constraining inbreeding. Their maximization objective accounts for both the predicted mean and genetic variance of a cross, but does not consider the predicted genetic correlation between traits. In theory, such information could be included in this optimization framework, as long as predictions are accurate.</p><p id="p-9">The objectives of this study were to (i) use simulations to assess the accuracy of genomewide predictions of genetic correlations and the long-term response to selection when selecting crosses on the basis of superior/correlated progeny means; and (ii) empirically measure the ability to predict genetic correlations using data from a barley (<em>Hordeum vulgare</em> L.) breeding program.</p><div class="section methods-materials" id="sec-1"><h2 class="">Methods and Materials</h2><div id="sec-2" class="subsection"><h3>Theory</h3><p id="p-10">Below, we first outline a deterministic prediction of the genetic variance of a single trait and the correlation between traits in a recombinant inbred line (RIL) population assuming two fully inbred parents, bi-allelic QTL, and no dominance or epistasis. This derivation follows the notation presented in <a id="xref-ref-60-3" class="xref-bibr" href="#ref-60">Zhong and Jannink (2007)</a>; others have determined equations for the expected genetic variance in bi-parental populations of intermediate selfing generations (<a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Lehermeier <em>et al.</em> 2017</a>) or multi-parent populations (<a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Allier <em>et al.</em> 2019</a>), and this derivation could be applied to such circumstances. We then use these predictions to determine the superior progeny mean and correlated progeny mean for a cross.</p><p id="p-11">Suppose that <em>L</em><sub>(</sub><em><sub>k</sub></em><sub>)</sub> QTL influence the <em>k</em>th quantitative trait, and in the <em>m</em>th cross <em>L<sub>m</sub></em><sub>(</sub><em><sub>k</sub></em><sub>)</sub> QTL are segregating for that trait (where <em>L<sub>m</sub></em><sub>(</sub><em><sub>k</sub></em><sub>)</sub> ≤ <em>L</em><sub>(</sub><em><sub>k</sub></em><sub>)</sub>). The expected genetic variance in the cross is the sum of the variance of each locus plus the covariance between pairs of loci. As noted in <a id="xref-ref-60-4" class="xref-bibr" href="#ref-60">Zhong and Jannink (2007)</a>, the genetic variance in cross <em>m</em> is<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-1.gif"/></noscript></span><span class="disp-formula-label">(1)</span></span>where <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-2.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-3.gif"/></noscript></span></span> are the allele substitution effects of the <em>i</em>th and <em>j</em>th loci, respectively and <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-4.gif"/></noscript></span></span> is the recombination fraction between the <em>i</em>th and <em>j</em>th loci. As expected, loci that are genetically unlinked (<em>i.e.</em>, independent, <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-5.gif"/></noscript></span></span>) will have a covariance of 0. The covariance can be generalized across coupling and repulsion phase linkage by setting the allele substitution effects <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-6.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-7.gif"/></noscript></span></span> to those of the first parent and <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-8.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-9.gif"/></noscript></span></span> to those of the second parent (<a id="xref-ref-60-5" class="xref-bibr" href="#ref-60">Zhong and Jannink 2007</a>).</p><p id="p-12">The single-trait covariance term in <a id="xref-disp-formula-1-1" class="xref-disp-formula" href="#disp-formula-1">Equation (1)</a> can be modified to calculate the expected genetic covariance between two traits, 1 and 2:<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-10.gif"/></noscript></span><span class="disp-formula-label">(2)</span></span>where <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-11.gif"/></noscript></span></span> is the recombination fraction between the <em>i</em>th locus of trait 1 and the <em>j</em>th locus of trait 2, <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-12.gif"/></noscript></span></span> is the allele substitution effect of the <em>i</em>th locus of trait 1 and <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-13.gif"/></noscript></span></span> is the allele substitution effect of the <em>j</em>th locus of trait 2. Using the expected genetic variance of each trait calculated from <a id="xref-disp-formula-1-2" class="xref-disp-formula" href="#disp-formula-1">Equation (1)</a> and the expected covariance between traits from <a id="xref-disp-formula-2-1" class="xref-disp-formula" href="#disp-formula-2">Equation (2)</a>, the expected genetic correlation is<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-14.gif"/></noscript></span><span class="disp-formula-label">(3)</span></span>With estimates of the genetic variance for two traits and the genetic correlation between the traits, we can rely on established theory to estimate the superior progeny mean (<span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-15.gif"/></noscript></span></span>) and correlated progeny mean in a cross. For trait 1, assumed under direct selection, the superior progeny mean is<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-16.gif"/></noscript></span><span class="disp-formula-label">(4)</span></span>where <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-17.gif"/></noscript></span></span> is the expected mean of trait 1 in the cross (estimated as the mean breeding value of the parents) and <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-18.gif"/></noscript></span></span> is the standardized selection coefficient. It is worth noting that the deviation from <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-19.gif"/></noscript></span></span> in <a id="xref-disp-formula-4-1" class="xref-disp-formula" href="#disp-formula-4">Equation (4)</a> is the same as the direct response to selection, <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-20.gif"/></noscript></span></span>, when the heritability is 1. The correlated response of the second trait, after selection on the first, is <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-21.gif"/></noscript></span></span>, which, when expressed as a deviation from the expected mean of the second trait, becomes the correlated progeny mean:<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-22.gif"/></noscript></span><span class="disp-formula-label">(5)</span></span>As with phenotypic values of two traits in a population, estimates of the superior progeny mean and correlated progeny mean could be used to select crosses that maximize the genetic gain for both traits, through independent culling levels or index selection (<a id="xref-ref-6-4" class="xref-bibr" href="#ref-6">Bernardo 2010</a>).</p><p id="p-13">The equations above assume that the loci under consideration are the true QTL influencing the quantitative traits. Since the effects of such QTL are usually unknown, the estimated effects of genomewide markers in linkage disequilibrium with QTL can be used to make predictions (<a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Meuwissen <em>et al.</em> 2001</a>). This is the basis of <em>in silico</em> methods to predict genetic variance and genetic correlation, such as the R package <em>PopVar</em> (<a id="xref-ref-42-3" class="xref-bibr" href="#ref-42">Mohammadi <em>et al.</em> 2015</a>). The advantage of the deterministic equations is computational speed (about 130-fold faster, data not shown), with a high or perfect correlation between predicted values (Figure S1).</p></div><div id="sec-3" class="subsection"><h3>Simulations</h3><p id="p-14">We conducted two simulations to assess the utility of predicting the genetic correlation in a breeding cross. Our simulations were based on observed marker genotypes of 1,570 North American two-row spring barley lines genotyped with 3,072 single nucleotide polymorphism (SNP) markers (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Close <em>et al.</em> 2009</a>), with genetic positions according to a consensus linkage map (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Muñoz-Amatriaín <em>et al.</em> 2011</a>); all data were obtained from the Triticeae Toolbox (<a href="https://triticeaetoolbox.org/barley/">https://triticeaetoolbox.org/barley/</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Blake <em>et al.</em> 2016</a>). We used empirical marker data to initiate our simulations in order to reflect the observed patterns of LD within North American barley breeding lines (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Jannink 2010</a>). Marker genotypes were arbitrarily coded as -1, 0, 1, where -1 was homozygous for the second allele, 0 was heterozygous, and 1 was homozygous for the first allele. After removing monomorphic and redundant SNPs (identical genotype calls and genetic positions), and SNPs and lines with more than 10% missing data, we were left with a marker matrix of 1,565 lines and 2,309 SNPs. We set the few heterozygous genotypes to missing and imputed missing calls using the mode across each SNP. The genetic map positions of completely coincident SNPs (<em>i.e.</em>, due to low genetic resolution) were jittered by adding a small value (1 × 10<sup>−6</sup> cM). These data were used to define the genetic architecture of the simulated quantitative traits and form the initial pool from which to establish a base training population.</p><div id="sec-4" class="subsection"><h4>Simulation 1 – Accuracy of predicting genetic correlations:</h4><p id="p-15">In the first simulation experiment, we assessed the conditions influencing prediction accuracy of genetic correlations. We perturbed the heritabilities of two traits, the architecture defining the traits (<em>i.e.</em>, number of QTL) and genetic correlation, the initial genetic correlation, and the base/training population size (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). Simulations were initiated by drawing 200 SNP markers to act as QTL. For each trait, 100 – <em>L</em> QTL were assigned an effect of 0, where <em>L</em> was the effective number of QTL (30 or 100). QTL effects were defined by a geometric series, as proposed by <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Lande and Thompson (1990)</a>: for the <em>k</em>th QTL, the value of the favorable homozygote was <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-23.gif"/></noscript></span></span>, the value of the heterozygote was 0, and the value of the unfavorable homozygote was <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-24.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-25.gif"/></noscript></span></span>. The first allele at each QTL was randomly assigned to be favorable or unfavorable and larger values were considered favorable for both traits. This randomization was performed independently for each trait.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/494398/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/494398/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">In our two simulation experiments, we modified the heritability (<em>h<sup>2</sup></em>) and number of quantitative trait loci (<em>N<sub>QTL</sub></em>) of two traits, the starting genetic correlation (<em>r<sub>G(0)</sub></em>), correlation architecture, size of a training population (<em>N<sub>TP</sub></em>), and model used to predicted genomewide marker effects</span><div class="sb-div caption-clear"></div></div></div><p id="p-16">Genetic correlations were generated according to three different architecture types: pleiotropy, tight linkage, or loose linkage. For simplicity, we assumed that the genetic architecture was governed entirely by one of the types. Under pleiotropy, the sampled QTL effects were first stored in an <em>L</em> × 2 matrix, A. The desired genetic correlation in the base population (<span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-26.gif"/></noscript></span></span>) was achieved by multiplying matrix A by the Choleski decomposition of the variance-covariance matrix Σ, which contained 1 on the diagonal and <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-27.gif"/></noscript></span></span> on the off-diagonal. This resulted in a set of QTL with pleiotropic effects that varied in both magnitude and sign for the two traits. Under tight linkage and loose linkage, each SNP sampled to be an effective QTL for the first trait was paired with another SNP that was sampled – with restrictions – to be an effective QTL for the second trait. For tight linkage, this second SNP was restricted to within 5 cM of the first SNP, and for loose linkage, this second SNP was restricted to between 25 cM and 35 cM of the first SNP. The QTL effects were again stored in the <em>L</em> × 2 matrix A, where each row was a pair of QTL, and subsequently adjusted as above. Effects of QTL influencing the second trait were then multiplied by matrix R, which contained estimates of linkage disequilibrium (LD; measured as the pairwise correlation, <em>r</em>, between genotype states in the base population) between QTL influencing the first trait and QTL influencing the second trait. This adjustment resulted in base genetic correlations that approximately matched the target, <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-28.gif"/></noscript></span></span> (Figure S2).</p><p id="p-17">The base/training population was first generated by randomly sampling <em>N<sub>TP</sub></em> individuals from the simulation starting material. For each trait, the genotypic value of an individual was calculated as the sum of the QTL allele effects carried by that individual, and the genetic variance was calculated as the variance among genotypic values. Phenotypic values were simulated by adding independent normally distributed deviations to the genotypic values to achieve a starting entry-mean heritability of <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-29.gif"/></noscript></span></span> (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>) with no residual covariance between traits. Individuals were assumed to be phenotyped in three environments with one replication, and the mean phenotypic value was used for genomewide prediction. Marker effects were predicted using the univariate model:<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-30.gif"/></noscript></span><span class="disp-formula-label">(6)</span></span>where <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-31.gif"/></noscript></span></span> was the phenotypic mean of the <em>i</em>th individual for the <em>p</em>th trait, <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-32.gif"/></noscript></span></span> was the population mean for the <em>p</em>th trait, <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-33.gif"/></noscript></span></span> was the allelic state of the <em>m</em>th marker in the <em>i</em>th individual (<em>i.e.</em>, -1, 0, or 1), <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-34.gif"/></noscript></span></span> was the predicted effect of the <em>m</em>th marker for the <em>p</em>th trait, and <span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-35.gif"/></noscript></span></span> was the associated error. We used two models to predict marker effects: ridge-regression best linear unbiased prediction (RR-BLUP) and BayesCπ (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Habier <em>et al.</em> 2011</a>). Potential crosses were generated by randomly sampling 50 pairs of individuals from the base population. We predicted the genetic correlation (<span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-36.gif"/></noscript></span></span>) for each potential cross using <a id="xref-disp-formula-1-3" class="xref-disp-formula" href="#disp-formula-1">Equations (1)</a>, (<a id="xref-disp-formula-2-2" class="xref-disp-formula" href="#disp-formula-2">2</a>), and (<a id="xref-disp-formula-3-1" class="xref-disp-formula" href="#disp-formula-3">3</a>), where α was substituted with <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-37.gif"/></noscript></span></span>. The expected genetic correlation (<span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-38.gif"/></noscript></span></span>) was similarly computed but instead using the known QTL effects instead of the predicted marker effects. Prediction accuracy was defined as the correlation between the predicted and expected genetic correlations. As a comparison, we also assessed predictions of the trait-specific mean (<span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-39.gif"/></noscript></span></span>) and genetic variance (<span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-40.gif"/></noscript></span></span>) in each cross. Each condition of this simulation was replicated 100 times.</p></div><div id="sec-5" class="subsection"><h4>Simulation 2 – Correlated response to selection:</h4><p id="p-18">We conducted a second simulation experiment to measure the long-term response of two correlated traits under different cross selection strategies. The range of perturbed parameters was smaller than in the first simulation (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>), though simulations were initiated as described above. We assumed that a breeder wanted to simultaneously increase the genotypic value of two quantitative traits through indirect selection, therefore positive genetic correlations were favorable. The base/training population was created by randomly sampling <em>N<sub>TP</sub></em> = 600 individuals from the simulation starting material. Informed by the results of the first simulation (see below), we used only the simpler RR-BLUP model to predict marker effects. Potential parents for the first breeding cycle were selected by determining the best 30 individuals in training population based on the predicted genotypic values of the first trait (the primary trait under direct selection). We predicted the mean, genetic variance, and genetic correlation for all possible non-reciprocal crosses between the potential parents. We then calculated the superior progeny mean of the first trait and the correlated progeny mean of the second trait using the predicted parameters and <a id="xref-disp-formula-4-2" class="xref-disp-formula" href="#disp-formula-4">Equations (4)</a> and (<a id="xref-disp-formula-5-1" class="xref-disp-formula" href="#disp-formula-5">5</a>) with a standardized selection coefficient of <span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-41.gif"/></noscript></span></span> (<em>i.e.</em>, selection of the best 5%).</p><p id="p-19">Twenty crosses were selected based on i) equal-weight sum of the normalized predicted superior progeny mean of the first trait and predicted correlated response in the second trait <span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-42.gif"/></noscript></span></span>, ii) the predicted cross mean of the primary trait <span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-43.gif"/></noscript></span></span>, or iii) random selection. We will subsequently refer to the non-random cross selection methods by the abbreviations CPM (correlated/superior progeny mean) or FM (family, or cross, mean). From the selected crosses, families of 50 recombinant inbred lines were simulated using the <em>qtl</em> R package (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Broman <em>et al.</em> 2003</a>). Recombination events were sampled according to the genetic map (<a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Muñoz-Amatriaín <em>et al.</em> 2011</a>), with the assumption of no crossover interference or mutation. This resulted in a pool of 1,000 selection candidates. Finally, 50 potential parents for the next breeding cycle were chosen from these candidates using predicted genotypic values of the first trait. We simulated 10 cycles of recurrent selection (outlined in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>), during which marker effect estimates remained unchanged. Along with the standardized selection response for each trait, we also tracked the genetic variance of each trait, the genetic correlation between traits, the frequency of favorable, unfavorable, and antagonistic (<em>i.e.</em>, two alleles with opposite effect) QTL haplotypes, and the proportion of QTL with fixed alleles. Simulations were replicated 250 times, and we report the mean and 95% approximate confidence interval for each measured variable.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/10/3153/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="In the recurrent selection simulation (Simulation 2), a training population (TP) was sampled and used to predict genomewide marker effects. In the first cycle, potential parents were identified using direct selection on predicted genotypic values (PGVs) of the first trait. Crosses were selected by one of three methods and were used to simulate selection candidates. Potential parents of the next cycle were determined using direct selection on the first trait. Ten breeding cycles were simulated. Any processes that relied on the predicted marker effects are noted with a blue/gray box." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-211462471" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;In the recurrent selection simulation (Simulation 2), a training population (TP) was sampled and used to predict genomewide marker effects. In the first cycle, potential parents were identified using direct selection on predicted genotypic values (PGVs) of the first trait. Crosses were selected by one of three methods and were used to simulate selection candidates. Potential parents of the next cycle were determined using direct selection on the first trait. Ten breeding cycles were simulated. Any processes that relied on the predicted marker effects are noted with a blue/gray box.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/10/3153/F1.medium.gif" width="280" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/9/10/3153/F1.medium.gif" width="280" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/10/3153/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/10/3153/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494355" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-20" class="first-child">In the recurrent selection simulation (Simulation 2), a training population (TP) was sampled and used to predict genomewide marker effects. In the first cycle, potential parents were identified using direct selection on predicted genotypic values (PGVs) of the first trait. Crosses were selected by one of three methods and were used to simulate selection candidates. Potential parents of the next cycle were determined using direct selection on the first trait. Ten breeding cycles were simulated. Any processes that relied on the predicted marker effects are noted with a blue/gray box.</p><div class="sb-div caption-clear"></div></div></div></div></div><div id="sec-6" class="subsection"><h3>Empirical validation</h3><p id="p-21">To empirically validate predictions of genetic correlations, we used phenotypic and genotypic data from a barley breeding program. The details of data generation are described elsewhere (<a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Neyhart and Smith 2019</a>), but we include a brief overview below. A training population (TP) of 175 two-row spring barley lines was genotyped with 6,361 SNP markers and phenotyped in four location-year environments for heading date (a proxy for flowering time), Fusarium head blight (FHB) severity (caused by the fungal pathogen <em>Fusarium graminearum</em> Schwabe), and plant height. To estimate the genetic correlation between pairs of traits in the TP, we regressed the phenotypic observations of two traits, <span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-44.gif"/></noscript></span></span> on individuals (genotypes), environments, and their interaction, where <em>i</em> indexes genotypes, <em>j</em> environments, and <em>p</em> traits. We fitted a bi-variate mixed model:<span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-45.gif"/></noscript></span><span class="disp-formula-label">(7)</span></span>where <span class="inline-formula" id="inline-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-46.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-46.gif"/></noscript></span></span> is the population mean, <span class="inline-formula" id="inline-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-47.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-47.gif"/></noscript></span></span>the fixed environmental effect, <span class="inline-formula" id="inline-formula-41"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-48.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-48.gif"/></noscript></span></span> the random effect of genotypes, <span class="inline-formula" id="inline-formula-42"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-49.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-49.gif"/></noscript></span></span> the random interaction effect of genotypes and environment, and <span class="inline-formula" id="inline-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-50.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-50.gif"/></noscript></span></span> the associated error. The distribution of random effects was assumed to be multivariate normal such that: <span class="inline-formula" id="inline-formula-44"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-51.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-51.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-45"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-52.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-52.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-46"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-53.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-53.gif"/></noscript></span></span>, where G is the realized genomic relationship matrix, <span class="inline-formula" id="inline-formula-47"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-54.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-54.gif"/></noscript></span></span> is an incidence matrix for individuals, <span class="inline-formula" id="inline-formula-48"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-55.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-55.gif"/></noscript></span></span> is an incidence matrix for environments, I is an identity matrix, ⊗ denotes the Kronecker product between matrices and ° denotes the Hadamard product between matrices. The genotype, genotype-environment interaction, and residuals covariance structures are<span class="disp-formula" id="disp-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-56.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-56.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-57.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-57.gif"/></noscript></span></span>and<span class="disp-formula" id="disp-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-58.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-58.gif"/></noscript></span></span>Models were fitted using the R package <em>sommer</em> (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Covarrubias-Pazaran 2016</a>) and the genetic correlation was estimated from elements in Σ using <a id="xref-disp-formula-3-2" class="xref-disp-formula" href="#disp-formula-3">Equation (3)</a>.</p><p id="p-22">Marker effects were estimated from marker genotypes and phenotypic best linear unbiased estimates (BLUES; <em>i.e.</em>, genotypic means from a model accounting for genotype, environment, and the interaction) of the TP using the univariate model in <a id="xref-disp-formula-6-1" class="xref-disp-formula" href="#disp-formula-6">Equation (6)</a>. Among all potential non-reciprocal crosses between 813 offspring of the TP (<em>n</em> = 330,078), we used estimated marker effects and the R package <em>PopVar</em> to predict the genetic correlation for each pair of traits. (Predictions were generated early in the study using this package, so for consistency we used those values, and not those generated using the deterministic equations above.) Twenty-six crosses were made based on the predictions, producing “validation families” ranging from 28 to 160 F<sub>5</sub> lines. The criteria for selecting crosses are described in <a id="xref-ref-44-3" class="xref-bibr" href="#ref-44">Neyhart and Smith (2019)</a> and rested primarily on predictions of genetic variance for several traits relevant for the breeding program. Validation families were phenotyped for the same three traits in 2 or 4 environments. Observations of heading date and plant height were recorded for all families, but due to logistical constraints of the inoculated disease nursery, only 14 families were phenotyped for FHB severity.</p><p id="p-23">Validation family phenotypes were used to estimate the observed genetic correlation in each family. We fitted a model modified from <a id="xref-disp-formula-7-1" class="xref-disp-formula" href="#disp-formula-7">Equation (7)</a>:<span class="disp-formula" id="disp-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-59.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-59.gif"/></noscript></span><span class="disp-formula-label">(8)</span></span>where <span class="inline-formula" id="inline-formula-49"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-60.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-60.gif"/></noscript></span></span>was the BLUE of genotypes in a validation family and other terms as described above. The distribution of random genotypic effects was assumed multivariate normal such that <span class="inline-formula" id="inline-formula-50"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-61.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-61.gif"/></noscript></span></span>, where the genomic relationship matrix in <a id="xref-disp-formula-7-2" class="xref-disp-formula" href="#disp-formula-7">Equation (7)</a> was replaced by an identity matrix, since validation families were ungenotyped and equally related within a family. Predictive ability was measured as the correlation between predicted and estimated genetic correlations across validation families, and the significance of this coefficient was tested using 1,000 bootstrapping replicates. Note that predictive ability is calculated by comparing predictions with phenotype-based observations, whereas prediction accuracy compares predictions with the true genotypic parameter (unobservable in our empirical experiment).</p></div><div id="sec-7" class="subsection"><h3>Data availability</h3><p id="p-24">Marker data for initiating the simulations and all data used in the empirical validation experiment is available from the Triticeae Toolbox (T3; <a href="https://triticeaetoolbox.org/barley/">https://triticeaetoolbox.org/barley/</a>). Genomewide marker data for empirical validation is also available from the Data Repository for the University of Minnesota (DRUM): <a href="https://conservancy.umn.edu/handle/11299/204785">https://conservancy.umn.edu/handle/11299/204785</a>. All simulations and analyses were performed in R (v. 3.5.1; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">R Core Team 2018</a>) and relevant scripts are located in the GitHub repository <a href="https://github.com/UMN-BarleyOatSilphium/GenCorPrediction">https://github.com/UMN-BarleyOatSilphium/GenCorPrediction</a>. Instructions are included in this repository for downloading data from T3. Supplemental material available at FigShare: <a href="https://doi.org/10.25387/g3.8980055">https://doi.org/10.25387/g3.8980055</a>.</p></div></div><div class="section results" id="sec-8"><h2 class="">Results</h2><div id="sec-9" class="subsection"><h3>Factors influencing predictions of genetic correlation</h3><p id="p-25">In our simulation, prediction accuracy for the genetic correlation in potential crosses was most influenced by trait heritability, training population size (<em>N<sub>TP</sub></em>), and genetic architecture. We provide a cross-section of results in <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>, and all results for the first simulation are displayed in Figure S3 and Table S1. Accuracy increased additively as a function of the heritability of both simulated traits, but only reached a maximum of about 0.81 under the most ideal conditions (Figure S3, Table S1). On average, accuracy increased by about 1.5-fold when moving from <em>N<sub>TP</sub></em> = 150 to <em>N<sub>TP</sub></em> = 600. We generally did not observe a pattern of diminishing returns when increasing <em>N<sub>TP</sub></em>, though some evidence of that pattern was present under the tight linkage architecture (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/10/3153/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The prediction accuracy of the genetic correlation in a cross increased with larger training populations and greater trait heritability. Accuracy was also influenced by genetic correlation architecture (loose linkage, tight linkage, or pleiotropy), prediction model [BayesC&#x3C0; (solid) or RR-BLUP (dashed)], and number of QTL [30 (navy) or 100 (orange)]. Lines denote the mean of 100 simulation replicates, and ribbons denote a 95% confidence interval. Results are restricted to a base genetic correlation of 0.5. (See Figure S3 and Table S1 for complete results.)" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-211462471" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;The prediction accuracy of the genetic correlation in a cross increased with larger training populations and greater trait heritability. Accuracy was also influenced by genetic correlation architecture (loose linkage, tight linkage, or pleiotropy), prediction model [BayesC&#x3C0; (solid) or RR-BLUP (dashed)], and number of QTL [30 (navy) or 100 (orange)]. Lines denote the mean of 100 simulation replicates, and ribbons denote a 95% confidence interval. Results are restricted to a base genetic correlation of 0.5. (See Figure S3 and Table S1 for complete results.)&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/10/3153/F2.medium.gif" width="440" height="429"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/9/10/3153/F2.medium.gif" width="440" height="429"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/10/3153/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/10/3153/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494428" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-26" class="first-child">The prediction accuracy of the genetic correlation in a cross increased with larger training populations and greater trait heritability. Accuracy was also influenced by genetic correlation architecture (loose linkage, tight linkage, or pleiotropy), prediction model [BayesCπ (solid) or RR-BLUP (dashed)], and number of QTL [30 (navy) or 100 (orange)]. Lines denote the mean of 100 simulation replicates, and ribbons denote a 95% confidence interval. Results are restricted to a base genetic correlation of 0.5. (See Figure S3 and Table S1 for complete results.)</p><div class="sb-div caption-clear"></div></div></div><p id="p-27">Between all correlation architectures, tight linkage resulted in the highest prediction accuracy, followed by loose linkage and then pleiotropy. The difference in accuracy under tight linkage <em>vs.</em> loose linkage was on average 0.071 (17%) and this difference under loose linkage <em>vs.</em> pleiotropy was 0.017 (6.2%). With tight linkage and loose linkage genetic architectures, accuracy was slightly higher when 100 <em>vs.</em> 30 QTL influenced both traits (a difference of about 0.04, or 10%), but the reverse was true under pleiotropy, where the accuracy was about 8% lower (a difference of about 0.03) with more QTL (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a>, Table S1). Interestingly, an interaction was apparent between the genetic architecture and the prediction model. Under pleiotropy and tight linkage, there was a slight advantage to using the BayesCπ model over RR-BLUP, particularly when 30 QTL influenced each trait. This difference was quite slim, however, with a boost to accuracy of only about 0.015 (3%).</p><p id="p-28">The family mean and genetic variance of each trait in potential crosses were almost always predicted with greater accuracy than the genetic correlation (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>, Table S1). In general, the family mean was predicted most accurately, followed by the genetic variance and the genetic correlation. (The genetic variance of one trait was occasionally predicted more accurately than the family mean of another trait, but only if the heritability of the first trait was much less than the second.) This trend was consistent across training population sizes, prediction models, and genetic architectures. The average (and range in) prediction accuracy was 0.87 (0.64, 1.0) for the family mean, 0.66 (0.34, 0.96) for the genetic variance, and 0.48 (0.18, 0.81) for the genetic correlation.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/10/3153/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Parameters of a cross were predicted with varying degrees of accuracy in our simulation. Predictions of the cross mean (, blue) were most accurate, followed by genetic variance (, orange) and genetic correlation (, red); this ranking was consistent across trait heritabilities, training population size, and genetic correlation architecture (loose linkage, tight linkage, or pleiotropy). Lines denote the mean of 100 simulation replicates, and ribbons denote a 95% confidence interval. Results are restricted to a base genetic correlation of 0.5, RR-BLUP prediction model, and 100 QTL. (See Table S1 for complete results.)" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-211462471" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Parameters of a cross were predicted with varying degrees of accuracy in our simulation. Predictions of the cross mean (&lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-62&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;, blue) were most accurate, followed by genetic variance (&lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-63&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;, orange) and genetic correlation (&lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-64&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;, red); this ranking was consistent across trait heritabilities, training population size, and genetic correlation architecture (loose linkage, tight linkage, or pleiotropy). Lines denote the mean of 100 simulation replicates, and ribbons denote a 95% confidence interval. Results are restricted to a base genetic correlation of 0.5, RR-BLUP prediction model, and 100 QTL. (See Table S1 for complete results.)&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/10/3153/F3.medium.gif" width="440" height="431"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/9/10/3153/F3.medium.gif" width="440" height="431"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/10/3153/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/10/3153/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494455" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-29" class="first-child">Parameters of a cross were predicted with varying degrees of accuracy in our simulation. Predictions of the cross mean (<span class="inline-formula" id="inline-formula-51"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/F3/embed/mml-math-62.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/F3/embed/mml-math-62.gif"/></noscript></span></span>, blue) were most accurate, followed by genetic variance (<span class="inline-formula" id="inline-formula-52"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/F3/embed/mml-math-63.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/F3/embed/mml-math-63.gif"/></noscript></span></span>, orange) and genetic correlation (<span class="inline-formula" id="inline-formula-53"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/F3/embed/mml-math-64.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/F3/embed/mml-math-64.gif"/></noscript></span></span>, red); this ranking was consistent across trait heritabilities, training population size, and genetic correlation architecture (loose linkage, tight linkage, or pleiotropy). Lines denote the mean of 100 simulation replicates, and ribbons denote a 95% confidence interval. Results are restricted to a base genetic correlation of 0.5, RR-BLUP prediction model, and 100 QTL. (See Table S1 for complete results.)</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-10" class="subsection"><h3>Empirical validation of predicted genetic correlations</h3><p id="p-30">We used genomewide markers and phenotypic data to empirically estimate the genetic correlation for three pairs of quantitative traits in our 175-line training population (TP). The genetic correlation was -0.84 between Fusarium head blight (FHB) severity and heading date, -0.44 between FHB severity and plant height, and 0.48 between heading date and plant height (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). These estimates were reflected in predictions of the cross mean and genetic correlation of 330,078 potential crosses (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>, <a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>). The average predicted genetic correlation among the potential crosses was -0.54 for FHB severity and heading date, -0.26 for FHB severity and plant height, and 0.24 for heading date and plant height. Though the genetic correlations between FHB severity and both heading date and plant height were unfavorable (earlier-flowering, shorter, and disease resistant plants are desirable), predictions implied that progress could be made by selecting populations with more favorable genetic correlations. For instance, more than 2,400 (0.73%) potential crosses were predicted to have a favorable (<em>i.e.</em>, positive) correlation between FHB severity and heading date.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/494470/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/494470/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Estimated genetic correlation in the empirical barley training population (<span class="inline-formula" id="inline-formula-54"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/T2/embed/mml-math-65.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/T2/embed/mml-math-65.gif"/></noscript></span></span>) and the mean and range (in parentheses) of the predicted genetic correlations among 330,078 potential crosses (<span class="inline-formula" id="inline-formula-55"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/T2/embed/mml-math-66.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/T2/embed/mml-math-66.gif"/></noscript></span></span>) for three pairs of quantitative traits</span><div class="sb-div caption-clear"></div></div></div><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/10/3153/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="In the barley breeding population used to empirically validate predictions, the relationship between the predicted cross means () of three pairs of traits for n = 330,078 potential crosses mirrored the overall distribution of predicted genetic correlations (), shaded from red (negative) to blue (positive). FHB, Fusarium head blight." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-211462471" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;In the barley breeding population used to empirically validate predictions, the relationship between the predicted cross means (&lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-69&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;) of three pairs of traits for &lt;em&gt;n&lt;/em&gt; = 330,078 potential crosses mirrored the overall distribution of predicted genetic correlations (&lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-70&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;), shaded from red (negative) to blue (positive). FHB, Fusarium head blight.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/10/3153/F4.medium.gif" width="440" height="175"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/9/10/3153/F4.medium.gif" width="440" height="175"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/10/3153/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/10/3153/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494323" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-32" class="first-child">In the barley breeding population used to empirically validate predictions, the relationship between the predicted cross means (<span class="inline-formula" id="inline-formula-58"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/F4/embed/mml-math-69.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/F4/embed/mml-math-69.gif"/></noscript></span></span>) of three pairs of traits for <em>n</em> = 330,078 potential crosses mirrored the overall distribution of predicted genetic correlations (<span class="inline-formula" id="inline-formula-59"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/F4/embed/mml-math-70.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/F4/embed/mml-math-70.gif"/></noscript></span></span>), shaded from red (negative) to blue (positive). FHB, Fusarium head blight.</p><div class="sb-div caption-clear"></div></div></div><p id="p-33">The mean (and range) of estimated genetic correlations among the validation families was -0.18 (-0.72, 0.58) for FHB severity and heading date, -0.038 (-0.67, 0.64) for FHB severity and plant height, and -0.13 (-0.64, 0.69) for heading date and plant height (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>). Estimates of predictive ability for genetic correlations ranged from -0.012 to 0.41 (<a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>). We could only validate predictions of the correlation between heading date and plant height, where all 26 validation families (VF) were phenotyped. The predictive abilities for remaining trait combinations were not significantly different from zero (<em>P</em> &gt; 0.05; bootstrapping). The ability to predict the genetic correlation appeared to coincide with the heritability of both traits; the entry-mean heritability in the TP (and in the VF) was 0.45 (0.11) for FHB severity, 0.96 (0.78) for heading date, and 0.52 (0.74) for plant height (<a id="xref-ref-44-4" class="xref-bibr" href="#ref-44">Neyhart and Smith 2019</a>).</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/494422/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/494422/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Number of phenotyped validation families, mean and range (in parentheses) of observed genetic correlations, and the predictive ability, measured as the correlation between the predicted (<span class="inline-formula" id="inline-formula-60"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/T3/embed/mml-math-71.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/T3/embed/mml-math-71.gif"/></noscript></span></span>) and observed (<span class="inline-formula" id="inline-formula-61"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/T3/embed/mml-math-72.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/T3/embed/mml-math-72.gif"/></noscript></span></span>) genetic correlation, for each of three pairs of traits. A 95% confidence interval (in parentheses) for the predictive ability was estimated from 1,000 bootstrapping replicates</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-11" class="subsection"><h3>Long-term response with different cross selection strategies</h3><p id="p-37">Our second simulation showed that the genetic gain for two correlated quantitative traits was impacted by the base genetic correlation, the genetic architecture, and the strategy to select crosses (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>). We found little difference in the outcome when the heritability of the second trait was 0.6; therefore, we highlight results when the heritability of the second trait was 0.3, a more realistic situation for indirect selection (<a id="xref-ref-6-5" class="xref-bibr" href="#ref-6">Bernardo 2010</a>). When measuring progress via a trait index (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5A</a>), we found that selecting crosses based on the predicted correlated superior progeny mean (CPM) resulted in a greater response than using the predicted cross mean (FM) or by random selection. Under all genetic architectures, the advantage of imposing non-random cross selection became clear after 1 breeding cycle. Subsequently, after 3 cycles, selecting crosses based on CPM resulted in higher gain than by selecting based on FM. Only after 9 – 10 cycles did random cross selection achieve equivalent or superior genetic gain compared with FM selection, though it never outperformed selection using CPM.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/10/3153/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Selecting crosses on the predicted correlated/superior progeny mean (CPM, orange) led to a greater long-term response (in units of genetic standard deviations) compared to selection on the predicted family mean (FM), blue) or by random selection (gray). This was true for a two-trait index (A) and both traits individually (B) across three correlation architectures and two base genetic correlations (rG(0)). Lines denote the mean of 250 simulation replicates, and the ribbon denotes a 95% confidence interval." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-211462471" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Selecting crosses on the predicted correlated/superior progeny mean (CPM, orange) led to a greater long-term response (in units of genetic standard deviations) compared to selection on the predicted family mean (FM), blue) or by random selection (gray). This was true for a two-trait index (A) and both traits individually (B) across three correlation architectures and two base genetic correlations (&lt;em&gt;r&lt;sub&gt;G(0)&lt;/sub&gt;&lt;/em&gt;). Lines denote the mean of 250 simulation replicates, and the ribbon denotes a 95% confidence interval.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/10/3153/F5.medium.gif" width="440" height="220"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/9/10/3153/F5.medium.gif" width="440" height="220"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/10/3153/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/10/3153/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494524" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-38" class="first-child">Selecting crosses on the predicted correlated/superior progeny mean (CPM, orange) led to a greater long-term response (in units of genetic standard deviations) compared to selection on the predicted family mean (FM), blue) or by random selection (gray). This was true for a two-trait index (A) and both traits individually (B) across three correlation architectures and two base genetic correlations (<em>r<sub>G(0)</sub></em>). Lines denote the mean of 250 simulation replicates, and the ribbon denotes a 95% confidence interval.</p><div class="sb-div caption-clear"></div></div></div><p id="p-39">Gain from selection, and marginal differences between selection methods, depended on the genetic architecture and correlation. The final genetic gain was, on average, less when the genetic correlation was negative. With pleiotropic architecture, the reduction in genetic gain from a correlation of 0.5 to -0.5 was more severe (a roughly 100% decrease) than with tight or loose linkage architectures (a roughly 20% decrease). After 10 cycles, the marginal genetic response (based on an index) when using CPM <em>vs.</em> FM cross selection ranged from 0.30 (11%), with tight linkage architecture and positive correlation, to 0.53 (27%), with loose linkage architecture and negative correlation.</p><p id="p-40">When considering traits individually, we found that much of the advantage of selecting crosses on CPM was realized in the correlated response of the second trait (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5B</a>). Selection on CPM or FM yielded similar responses in the first trait, except under pleiotropic architecture and a positive genetic correlation. Again, random cross selection led to the lowest response, though in some cases it became equivalent to selection on CPM or FM after 10 breeding cycles. Genetic gain was consistent for the primary trait, with a plateau reached after 6 – 8 cycles of selection. Conversely, we observed a rapid plateauing in the genetic gain of the second trait when the architecture was not pleiotropic. Here, when the genetic correlation was positive, the correlated response in the second trait was also positive, as expected, with a greater response under CPM selection. When the genetic correlation was negative, FM or random cross selection led to a negative response in the second trait, which was prevented under CPM selection (<a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5B</a>). Under pleiotropic conditions, the correlated response in the second trait matched expectations given the genetic correlation; however, CPM selection increased the positive response with positive correlation and mitigated the negative response with negative correlation.</p><p id="p-41">As expected, the genetic variance for both traits decreased over cycles of selection (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6A</a>), and by cycle 10, most had been exhausted. Although genetic variance for the first trait declined similarly under varying architectures, the loss of variance for the second trait was more precipitous when the architecture was defined by linkage <em>vs.</em> pleiotropy. Under the latter, genetic variance for the second trait was reduced at a rate comparable to the first trait (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6A</a>). We found that cross selection by FM always led to the most rapid reduction of genetic variance, while this decay was slower when selecting on CPM and slower yet with random mating. This ranking among selection methods was very apparent for the first trait (and for the second trait under pleiotropic architecture), but marginal differences were much less for the second trait.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/10/3153/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genetic variances (A) and the genetic correlation (B) in the simulated indirect selection breeding program were influenced by the cross selection method (family mean, FM, blue; correlated/superior progeny mean, CPM, orange; random, gray), genetic architecture, and base genetic correlation (rG(0)). All components declined in absolute value over cycles of selection, but the rate depended on the cross selection method and genetic architecture. Lines denote the mean of 250 simulation replicates, and the ribbon denotes a 95% confidence interval." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-211462471" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Genetic variances (A) and the genetic correlation (B) in the simulated indirect selection breeding program were influenced by the cross selection method (family mean, FM, blue; correlated/superior progeny mean, CPM, orange; random, gray), genetic architecture, and base genetic correlation (&lt;em&gt;r&lt;sub&gt;G(0)&lt;/sub&gt;&lt;/em&gt;). All components declined in absolute value over cycles of selection, but the rate depended on the cross selection method and genetic architecture. Lines denote the mean of 250 simulation replicates, and the ribbon denotes a 95% confidence interval.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/10/3153/F6.medium.gif" width="251" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/9/10/3153/F6.medium.gif" width="251" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/10/3153/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/10/3153/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494521" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-42" class="first-child">Genetic variances (A) and the genetic correlation (B) in the simulated indirect selection breeding program were influenced by the cross selection method (family mean, FM, blue; correlated/superior progeny mean, CPM, orange; random, gray), genetic architecture, and base genetic correlation (<em>r<sub>G(0)</sub></em>). All components declined in absolute value over cycles of selection, but the rate depended on the cross selection method and genetic architecture. Lines denote the mean of 250 simulation replicates, and the ribbon denotes a 95% confidence interval.</p><div class="sb-div caption-clear"></div></div></div><p id="p-43">The genetic correlation in the breeding population consistently declined in absolute value, moving toward zero under all simulated conditions (<a id="xref-fig-6-3" class="xref-fig" href="#F6">Figure 6B</a>). The genetic architecture impacted the rate of change, with the most rapid movement under loose linkage, followed by tight linkage and then pleiotropy. The correlation initially became more negative when selection was imposed on the base population (<em>i.e.</em>, cycle 0 to cycle 1). This change was much larger when the base correlation (<span class="inline-formula" id="inline-formula-64"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-75.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-75.gif"/></noscript></span></span>) was positive; indeed, under non-pleiotropic architecture, the genetic correlation became near-zero, or negative, after 1 cycle of selection. Conversely, when <span class="inline-formula" id="inline-formula-65"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-76.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-76.gif"/></noscript></span></span> was negative, the genetic correlation moved more steadily toward 0. When <span class="inline-formula" id="inline-formula-66"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-77.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-77.gif"/></noscript></span></span> was negative, we found that selecting crosses on CPM usually led to a more negative genetic correlation than FM or random selection, particularly in the first 5 breeding cycles.</p><p id="p-44">Changes in haplotype frequency were greatest when loose linkage defined the correlation architecture, followed by tight linkage and pleiotropy (<a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7A</a>). The change in haplotype frequencies was more limited with negative <span class="inline-formula" id="inline-formula-67"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-78.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-78.gif"/></noscript></span></span>, particularly under pleiotropic architecture. Selecting crosses on CPM led to a significantly higher increase in the frequency of favorable haplotypes and a corresponding decrease in the frequency of unfavorable haplotypes. Further, with non-pleiotropic architecture we observed a greater reduction in the frequency of antagonistic haplotypes when selecting crosses by CPM than other methods. As expected, the frequency of antagonistic haplotypes did not change when the architecture was defined by pleiotropy (<a id="xref-fig-7-2" class="xref-fig" href="#F7">Figure 7A</a>). Selection increasingly drove QTL to fixation (<a id="xref-fig-7-3" class="xref-fig" href="#F7">Figure 7B</a>), but the rate of fixation was uneven for different cross selection methods. Choosing crosses on FM led to the highest fixation rate, followed by CPM and then random mating. There was a slightly higher fixation rate with positive genetic correlation than with negative genetic correlation, and the fixation rates for QTL influencing the first trait or second trait were roughly equivalent.</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/10/3153/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The change in frequency of two-trait quantitative trait locus (QTL) haplotypes and proportion of fixed QTL depended on the cross selection method genetic architecture, and base genetic correlation (rG(0)). (A) The increase in frequency of favorable (solid) haplotypes and decrease in frequency of unfavorable (dashed) and antagonistic (dotted) haplotypes was always greater when selecting crosses on the correlated/superior progeny mean (CPM, orange). (B) The rate of QTL fixation for both trait 1 (solid) and trait 2 (dotted) was always greatest with cross selection on the family mean (FM, blue), followed by CPM and random mating (gray). Lines denote the mean of 250 simulation replicates, and the ribbon denotes a 95% confidence interval." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-211462471" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The change in frequency of two-trait quantitative trait locus (QTL) haplotypes and proportion of fixed QTL depended on the cross selection method genetic architecture, and base genetic correlation (&lt;em&gt;r&lt;sub&gt;G(0)&lt;/sub&gt;&lt;/em&gt;). (A) The increase in frequency of favorable (solid) haplotypes and decrease in frequency of unfavorable (dashed) and antagonistic (dotted) haplotypes was always greater when selecting crosses on the correlated/superior progeny mean (CPM, orange). (B) The rate of QTL fixation for both trait 1 (solid) and trait 2 (dotted) was always greatest with cross selection on the family mean (FM, blue), followed by CPM and random mating (gray). Lines denote the mean of 250 simulation replicates, and the ribbon denotes a 95% confidence interval.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/10/3153/F7.medium.gif" width="220" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/9/10/3153/F7.medium.gif" width="220" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/10/3153/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/10/3153/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494522" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-45" class="first-child">The change in frequency of two-trait quantitative trait locus (QTL) haplotypes and proportion of fixed QTL depended on the cross selection method genetic architecture, and base genetic correlation (<em>r<sub>G(0)</sub></em>). (A) The increase in frequency of favorable (solid) haplotypes and decrease in frequency of unfavorable (dashed) and antagonistic (dotted) haplotypes was always greater when selecting crosses on the correlated/superior progeny mean (CPM, orange). (B) The rate of QTL fixation for both trait 1 (solid) and trait 2 (dotted) was always greatest with cross selection on the family mean (FM, blue), followed by CPM and random mating (gray). Lines denote the mean of 250 simulation replicates, and the ribbon denotes a 95% confidence interval.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-12"><h2 class="">Discussion</h2><div id="sec-13" class="subsection"><h3>Predictions of genetic correlations are feasible with reliable training data</h3><p id="p-46">Our first simulation measured the prediction accuracy of genetic correlations as a function of training population (TP) size, trait heritability, prediction model, and genetic architecture. Our results implicated the usual suspects driving genomewide prediction accuracy for individual traits. Increasing the TP size and the heritability of both traits improved accuracy, an expected result given the importance of these parameters (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Daetwyler <em>et al.</em> 2008</a>; <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Wimmer <em>et al.</em> 2013</a>).</p><p id="p-47">The impact of genetic architecture was curious. Genetic correlations caused by loose linkage or pleiotropy led to lower prediction accuracies compared to architecture defined by tight linkage (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2</a>). We hypothesize that the same phenomenon, albeit with opposite effect, is responsible. Genomewide prediction models (<em>i.e.</em>, RR-BLUP or BayesCπ) assume that many more markers than true QTL have non-zero effect on both traits. Our approach to predicting genetic correlations relies on the recombination and segregation of these markers, implying that all contribute to variability in genetic variance and covariance, while only true QTL generate this variability. With pleiotropy, this would manifest as a downward bias in the covariance between traits, as we predict a greater possibility of recombination between markers than what is possible for the true QTL. Under loose linkage, an opposite, upward bias in covariance would be expected, as the many markers with uneven trait effects would be predicted to co-segregate more often than what is possible for the true QTL. Indeed, when we calculated the average bias of the predicted genetic correlations, we observed a roughly 30% upward bias under loose linkage and an opposite 30% downward bias under pleiotropy (Figure S4). Additionally, though the bias in predicting genetic covariance was always negative, it was less so under loose linkage than under pleiotropy. This bias, particularly if uneven across predicted crosses, could lead to the observed loss in accuracy. Practically, the impact of genetic architecture may be less important, since architecture is generally immutable and the effect on prediction accuracy is small (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2</a>).</p><p id="p-48">Though prediction accuracies were not appreciably different between models, it is worth mentioning potential causes and impacts. With fewer QTL, we observed higher prediction accuracies under the BayesCπ model, an unsurprising result given the known advantage of such models with heterogenous contributions of SNPs to total variance (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Daetwyler <em>et al.</em> 2010</a>); however, differences between models were smaller with architectures defined by more QTL. Although not observed in our simulations, we might expect the genetic diversity in the training population to play a role in prediction accuracy. Lower diversity, defined by lower effective population sizes and fewer distinct haplotype blocks, will lead to a spreading of the variance across few sets of markers in high LD (<a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Daetwyler <em>et al.</em> 2008</a>, <a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">2010</a>). This spreading would also exacerbate the observed upward bias under loose linkage architecture and downward bias under pleiotropic architecture. Further, the larger haplotype blocks accompanying lower diversity would cloud differences in predictions between tight linkage and loose linkage architectures. Predictions of genetic correlations in barley and other inbreeding crops may be particularly sensitive to this problem, given the long-range persistence of LD observed in these species (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Hamblin <em>et al.</em> 2010</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Chao <em>et al.</em> 2011</a>).</p><p id="p-49">Differences in the accuracy to predict the three parameters of a potential cross (<em>i.e.</em>, mean, genetic variance, and genetic correlation) are attributable to the nature of each statistic and have practical implications. Greater accuracy when predicting the cross mean <em>vs.</em> genetic variance was predicted by theory (<a id="xref-ref-60-6" class="xref-bibr" href="#ref-60">Zhong and Jannink 2007</a>) and has been observed empirically (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Adeyemo and Bernardo 2019</a>; <a id="xref-ref-44-5" class="xref-bibr" href="#ref-44">Neyhart and Smith 2019</a>); this trend is expected because the genetic variance, a second-order statistic, will be more adversely impacted by error in marker effect estimates. Similarly, the accuracy of the genetic correlation, a ratio of second-order statistics with large sampling variance (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Robertson 1959</a>), will be even more adversely affected. Even at large TP sizes, the predictions of the genetic correlation were only as accurate as those of the genetic variance at modest TP size and never as accurate as those of the cross mean (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>). Practically, this suggests that very large TPs are needed for such predictions to be useful, a prospect that may be prohibitive for a plant breeder. Further research could be directed toward applying Bayesian approaches, such as the “posterior mean variance” method of <a id="xref-ref-36-3" class="xref-bibr" href="#ref-36">Lehermeier <em>et al.</em> (2017)</a>, to improve the accuracy and bias of predicted genetic correlations, particularly at small TP sizes.</p></div><div id="sec-14" class="subsection"><h3>Selecting crosses using predicted genetic correlations increases multi-trait response</h3><p id="p-50">Under a simulated breeding program focused on multi-trait recurrent indirect selection, we showed that the long-term genetic gain for two traits was greatest when crosses were selected on predicted correlated/superior progeny means (CPM). This was true under all conditions of genetic architecture, trait heritability, and base genetic correlation (<span class="inline-formula" id="inline-formula-68"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-79.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-79.gif"/></noscript></span></span>) (<a id="xref-fig-5-5" class="xref-fig" href="#F5">Figure 5</a>). Further, cross selection based on CPM was superior to selection on the predicted cross mean (FM) or random mating, standard choices in programs using “best-by-best” breeding for cultivar development or in recurrent selection (<a id="xref-ref-6-6" class="xref-bibr" href="#ref-6">Bernardo 2010</a>).</p><p id="p-51">The greater multi-trait selection response achieved under CPM cross selection can be attributed to many drivers, including genetic variance and correlation, haplotype and allele frequencies, and LD. We discuss their impacts below. Compared to FM, selection on CPM led to higher genetic variance for both traits (<a id="xref-fig-6-4" class="xref-fig" href="#F6">Figure 6A</a>). When selecting on CPM, particularly at the relatively high selection intensity used in our simulation (<em>i</em> = 0.05; <em>k<sub>sp</sub></em> = 2.06), more weight is given to the predicted genetic variance <em>vs.</em> the predicted mean (<a id="xref-disp-formula-4-3" class="xref-disp-formula" href="#disp-formula-4">Equations 4</a> and <a id="xref-disp-formula-5-2" class="xref-disp-formula" href="#disp-formula-5">5</a>). Segregation of QTL is explicitly driving the prediction of genetic variance in our approach, and an emphasis on variance may keep small or moderate-effect QTL at intermediate frequency, at which variance is maximized (<a id="xref-ref-39-3" class="xref-bibr" href="#ref-39">Lynch and Walsh 1998</a>). Therefore, when selecting on CPM, we might expect a short-term loss in genetic gain for long-term benefit. Indeed, we note a small deficit in selection response in the first two breeding cycles relative to FM selection, particularly for the trait under direct selection (<a id="xref-fig-5-6" class="xref-fig" href="#F5">Figure 5B</a>). Selecting crosses using FM likely emphasizes the rapid increase in frequency of beneficial alleles at large and moderate-effect QTL, leading to fixation of unfavorable small-effect QTL due to drift or linkage (<a id="xref-fig-7-4" class="xref-fig" href="#F7">Figure 7B</a>). Practically, the maintenance of genetic variance under CPM selection suggests that genetic gain may be sustained beyond 10 breeding cycles (<a id="xref-fig-5-7" class="xref-fig" href="#F5">Figure 5</a>).</p><p id="p-52">Changes in LD and haplotype frequencies are likely driving the movement toward zero of the genetic correlation in the breeding population. After the first breeding cycle, we observed a sharp trend toward, or persistence of, more negative genetic correlations (<a id="xref-fig-6-5" class="xref-fig" href="#F6">Figure 6B</a>). This could be the product of negative covariance generated due to LD (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Felsenstein 1965</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Bulmer 1971</a>; <a id="xref-ref-25-4" class="xref-bibr" href="#ref-25">Falconer and Mackay 1996</a>) and the simultaneous fixation of favorable haplotypes or QTL and maintenance of antagonistic haplotypes or QTL at intermediate frequency (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bennett and Swiger 1980</a>; <a id="xref-ref-25-5" class="xref-bibr" href="#ref-25">Falconer and Mackay 1996</a>). Though genetic correlations became similar at later cycles (<a id="xref-fig-6-6" class="xref-fig" href="#F6">Figure 6B</a>), the less rapid movement toward zero of genetic correlations when selecting crosses using CPM is curious. This pattern, more apparent with positive <span class="inline-formula" id="inline-formula-69"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-80.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3153/embed/mml-math-80.gif"/></noscript></span></span>, may be due to similar forces influencing the genetic variance. Selection on CPM combined with stronger selection intensity weighs the predicted genetic correlation and genetic variance of each trait (<a id="xref-disp-formula-4-4" class="xref-disp-formula" href="#disp-formula-4">Equations 4</a> and <a id="xref-disp-formula-5-3" class="xref-disp-formula" href="#disp-formula-5">5</a>). As above, this would value the maintenance of segregating QTL in the population, in agreement with our observations (<a id="xref-fig-7-5" class="xref-fig" href="#F7">Figure 7B</a>). Co-segregating QTL for both traits would impact the genetic covariance to a greater degree than the genetic variances (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Bohren <em>et al.</em> 1966</a>; <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Villanueva and Kennedy 1990</a>), leading to the observed differences in genetic correlation (<a id="xref-fig-6-7" class="xref-fig" href="#F6">Figure 6B</a>). We would expect the linkage maintaining covariance in the short-term to be broken down by recombination, which may help explain why the genetic correlation moved more rapidly toward zero under the loose linkage genetic architecture (<a id="xref-fig-6-8" class="xref-fig" href="#F6">Figure 6B</a>).</p><p id="p-53">The general trends in genetic correlation over cycles can be explained by the genetic architecture. With pleiotropy, the movement toward – or maintenance of – a negative correlation is due to the fixation of favorable QTL and presence of antagonistic QTL. Absent pleiotropy, the correlation is due entirely to LD (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Lande 1984</a>; <a id="xref-ref-39-4" class="xref-bibr" href="#ref-39">Lynch and Walsh 1998</a>), which is degraded by recombination, eventually moving the genetic correlation toward zero (<a id="xref-ref-57-2" class="xref-bibr" href="#ref-57">Villanueva and Kennedy 1990</a>). Though our results confirm this under the loose linkage and tight linkage architecture (<a id="xref-fig-6-9" class="xref-fig" href="#F6">Figure 6B</a>), the final genetic correlation is slightly negative, and more so with tight linkage. This is likely due to fixation of antagonistic QTL haplotypes, which, particularly when tightly linked, can effectively act as pleiotropic loci (<a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Lande 1984</a>) and are subject to the same competing forces mentioned earlier.</p><p id="p-54">Our approach to cross selection for simultaneously improvement of multiple traits is one of several that have recently been proposed. Selection using CPM takes advantage of predictions of the genetic correlation between two traits (<a id="xref-disp-formula-4-5" class="xref-disp-formula" href="#disp-formula-4">Equations 4</a> and <a id="xref-disp-formula-5-4" class="xref-disp-formula" href="#disp-formula-5">5</a>). In a similar vein, <a id="xref-ref-3-3" class="xref-bibr" href="#ref-3">Allier <em>et al.</em> (2019)</a> proposed selecting crosses that were predicted to maximize the response for a trait under direct selection and produce the most favorable correlated response in parental contribution (treated as the indirect trait). Their method extends predictions of genetic correlations to three- and four-way crosses, a generalization beyond our described approach for bi-parental populations. Instead of predicting genetic correlations, a multi-trait index could be constructed and crosses could be selected on the basis of the superior progeny mean of that index (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Yao <em>et al.</em> 2018</a>). Though this simplifies the number of parameters to be predicted (a single genetic variance <em>vs.</em> many pairwise correlations), it uses information differently from the CPM method. Finally, predictions of genetic correlations or correlated responses could enhance multi-objective cross optimization procedures (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Akdemir <em>et al.</em> 2019</a>).</p><p id="p-55">The results of our simulation bode well for implementation in a breeding program. Notably, we observed that the advantage of selecting crosses on CPM was apparent even when the genetic correlation was negative (<em>i.e.</em>, unfavorable). This condition is often encountered by breeders and would typically discourage the use of indirect selection (<a id="xref-ref-6-7" class="xref-bibr" href="#ref-6">Bernardo 2010</a>); however, we demonstrated that CPM cross selection can mitigate any negative response in the trait under indirect selection when the genetic correlation is negative.</p></div><div id="sec-15" class="subsection"><h3>Application in a breeding program</h3><p id="p-56">To demonstrate its feasibility under more realistic conditions, we generated predictions of genetic correlations among populations in a barley breeding program. For two pairs of traits with moderately or strongly unfavorable correlations, we identified many crosses with favorable predicted correlations (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>, <a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>), suggesting that specific crosses could be targeted to improve multiple traits simultaneously. This would rely on accurately discriminating among crosses, and we attempted to validate predictions of genetic correlations using empirical data of breeding populations.</p><p id="p-57">Though we were only able to validate predictions for one pair of traits (<a id="xref-table-wrap-3-3" class="xref-table" href="#T3">Table 3</a>), we observed that predictive ability seemed to be associated with the heritability of both traits, in agreement with the results from our first simulation. Of course, trait heritability may influence accuracy beyond unreliable marker effect estimates. With less heritable traits, the correlation among environmental effects is expected to have a greater influence on the phenotypic correlation (<a id="xref-ref-39-5" class="xref-bibr" href="#ref-39">Lynch and Walsh 1998</a>). It is not difficult to imagine how shared environment could influence the observed correlations. For instance, environmental stresses might stunt the growth of plants and promote earlier flowering, creating a positive correlation between these traits. Additionally, plants that flower later or are taller may avoid the soil-borne <em>F. graminearum</em> inoculum, potentially leading to artificial negative correlations between the traits. The results of our simulations and empirical experiment confirm that, as in any other implementation of genomewide selection, reliable phenotypic data are paramount.</p><p id="p-58">The modest size of our TP (<em>n</em> = 175) likely constrained prediction accuracy, as suggested in our first simulation (<a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2</a>). Previous genomewide selection research, including those focused on barley, often report a pattern of diminishing returns when predicting line means with ever-larger TPs (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Lorenz <em>et al.</em> 2012</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Sallam <em>et al.</em> 2015</a>). A breeding program using a smaller TP, perhaps as a resource-saving measure, may be ill-equipped to utilize predictions of genetic correlations. The size of our TP was a function of the early stage of implementing genomewide selection in the breeding program, and we might expect that as more individuals are phenotyped and genotyped, the size of the training dataset will become more satisfactory.</p><p id="p-59">Fortunately, the barrier for a breeder to incorporate predictions of genetic correlations is low. First, the data required for predictions (phenotypes, marker genotypes, and a genetic map) are commonly available in many breeding programs. Second, it is relatively inexpensive, in both time and computing power, to generate such predictions. Thus, it is possible that this procedure can be an additional tool for breeders to make decisions. We note, however, that validating predictions of genetic correlations requires phenotypic data on many large families, leading to population sizes that are generally unrealistic for a breeding program (<a id="xref-ref-6-8" class="xref-bibr" href="#ref-6">Bernardo 2010</a>). It is likely that these predictions, if implemented, will not be routinely validated, unlike genomewide predictions of genotypic means, and this lack of feedback may prove discouraging for breeders. Nevertheless, further work is necessary to demonstrate empirically the utility of selecting crosses informed by predictions of genetic correlations, with emphasis on the response to selection for two potentially unfavorably correlated traits.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-60">We thank Ed Schiefelbein, Guillermo Velasquez, and Karen Beaubien for technical support during population development, phenotypic data collection, and genotyping. Thanks go to Ruth Dill-Macky for supplying <em>F. graminearum</em> inoculum for St. Paul and to Madeline Smith and Joseph Wodarek for managing the FHB trial in Crookston, MN. We are grateful to Austin Case, Jo Heuschele, John Hill Price, Ian McNish, Becky Zhong, and Alexander Susko for assistance and encouragement. Resources from the Minnesota Supercomputing Institute were used to complete this project. This research was supported by the U.S. Wheat and Barley Scab Initiative, the Minnesota Department of Agriculture, Rahr Malting Company, the Brewers Association, the American Malting Barley Association, and USDA-NIFA Grant #2018-67011-28075.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-5"><p id="p-61">Supplemental material available at FigShare: <a href="https://doi.org/10.25387/g3.8980055">https://doi.org/10.25387/g3.8980055</a>.</p></li><li class="fn" id="fn-6"><p id="p-62"><em>Communicating editor: D. J. de Koning</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2019-03-27"><span class="received-label">Received </span>March 27, 2019.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2019-07-25"><span class="accepted-label">Accepted </span>July 25, 2019.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2019 Neyhart <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3153.1" data-doi="10.2135/cropsci2018.08.0525"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Adeyemo</span>,  <span class="cit-name-given-names">E.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Bernardo</span></span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">Predicting Genetic Variance from Genomewide Marker Effects Estimated from a Diverse Panel of Maize Inbreds.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">59</span>: <span class="cit-fpage">583</span>–<span class="cit-lpage">590</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2018.08.0525</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D59%26rft.spage%253D583%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2018.08.0525%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2018.08.0525&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3153.2" data-doi="10.1038/s41437-018-0147-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Akdemir</span>,  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Beavis</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Fritsche-Neto</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. K.</span> <span class="cit-name-surname">Singh</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Isidro-Sánchez</span></span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">Multi-objective optimized genomic breeding strategies for sustainable food improvement.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">122</span>: <span class="cit-fpage">672</span>–<span class="cit-lpage">683</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/s41437-018-0147-1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41437-018-0147-1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41437-018-0147-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3153.3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Allier</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Moreau</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Charcosset</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Teyssèdre</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Lehermeier</span></span></li></ol><cite>, <span class="cit-pub-date">2019</span> Usefulness Criterion and Post-selection Parental Contributions in Multi-parental Crosses: Application to Polygenic Trait Introgression. G3 (Bethesda) 9: 1469–1479. <a href="https://doi.org/https://doi.org/10.1534/g3.119.400129">https://doi.org/https://doi.org/10.1534/g3.119.400129</a></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3153.4" data-doi="10.3835/plantgenome2015.04.0024"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bandillo</span>,  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Jarquin</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Q.</span> <span class="cit-name-surname">Song</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Nelson</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Cregan</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">A Population Structure and Genome-Wide Association Analysis on the USDA Soybean Germplasm Collection.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">0</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2015.04.0024</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2015.04.0024%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2015.04.0024&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3153.5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bennett</span>,  <span class="cit-name-given-names">G. L.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">L. A.</span> <span class="cit-name-surname">Swiger</span></span></li></ol><cite>, <span class="cit-pub-date">1980</span> <span class="cit-article-title">Genetic variance and correlation after selection for two traits by index, independent culling levels, and extreme selection.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">94</span>: <span class="cit-fpage">763</span>–<span class="cit-lpage">775</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DBennett%26rft.auinit1%253DG.%2BL.%26rft.volume%253D94%26rft.issue%253D3%26rft.spage%253D763%26rft.epage%253D775%26rft.atitle%253DGENETIC%2BVARIANCE%2BAND%2BCORRELATION%2BAFTER%2BSELECTION%2BFOR%2BTWO%2BTRAITS%2BBY%2BINDEX%252C%2BINDEPENDENT%2BCULLING%2BLEVELS%2BAND%2BEXTREME%2BSELECTION%26rft_id%253Dinfo%253Apmid%252F7399261%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiOTQvMy83NjMiO3M6NDoiYXRvbSI7czoxOToiL2dnZy85LzEwLzMxNTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-book" id="cit-9.10.3153.6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bernardo</span>,  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-source">Breeding for Quantitative Traits in Plants</span>, <span class="cit-publ-name">Stemma Press</span>, <span class="cit-publ-loc">Woodbury, Minnesota</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3153.7" data-doi="10.2135/cropsci2014.01.0088"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bernardo</span>,  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genomewide Selection of Parental Inbreds: Classes of Loci and Virtual Biparental Populations.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">54</span>: <span class="cit-fpage">2586</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2014.01.0088</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D54%26rft.spage%253D2586%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2014.01.0088%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2014.01.0088&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3153.8" data-doi="10.2135/cropsci2014.07.0460"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Beyene</span>,  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Semagn</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Mugo</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Tarekegne</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Babu</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genetic Gains in Grain Yield Through Genomic Selection in Eight Bi-parental Maize Populations under Drought Stress.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">55</span>: <span class="cit-fpage">154</span>–<span class="cit-lpage">163</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2014.07.0460</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D55%26rft.spage%253D154%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2014.07.0460%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2014.07.0460&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3153.9" data-doi="10.3835/plantgenome2016.05.0047"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Blake</span>,  <span class="cit-name-given-names">V. C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Birkett</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. E.</span> <span class="cit-name-surname">Matthews</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. L.</span> <span class="cit-name-surname">Hane</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Bradbury</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">The Triticeae Toolbox: Combining Phenotype and Genotype Data to Advance Small-Grains Breeding.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">10</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2014.12.0099</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D9%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2016.05.0047%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2016.05.0047&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3153.10" data-doi="10.2135/cropsci1999.0011183X003900010035x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bohn</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H. F.</span> <span class="cit-name-surname">Utz</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. E.</span> <span class="cit-name-surname">Melchinger</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Genetic Similarities among Winter Wheat Cultivars Determined on the Basis of RFLPs, AFLPs, and SSRs and Their Use for Predicting Progeny Variance.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">228</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci1999.0011183X003900010035x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DBohn%26rft.auinit1%253DM.%26rft.volume%253D39%26rft.issue%253D1%26rft.spage%253D228%26rft.epage%253D237%26rft.atitle%253DGenetic%2BSimilarities%2Bamong%2BWinter%2BWheat%2BCultivars%2BDetermined%2Bon%2Bthe%2BBasis%2Bof%2BRFLPs%252C%2BAFLPs%252C%2Band%2BSSRs%2Band%2BTheir%2BUse%2Bfor%2BPredicting%2BProgeny%2BVariance%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci1999.0011183X003900010035x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci1999.0011183X003900010035x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000078747300035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3153.11" data-doi="10.1017/S0016672300009460"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bohren</span>,  <span class="cit-name-given-names">B. B.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W. G.</span> <span class="cit-name-surname">Hill</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Robertson</span></span></li></ol><cite>, <span class="cit-pub-date">1966</span> <span class="cit-article-title">Some observations on asymmetrical correlated responses to selection.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">44</span>–<span class="cit-lpage">57</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1017/S0016672300009460</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DBohren%26rft.auinit1%253DB.%2BB.%26rft.volume%253D7%26rft.issue%253D1%26rft.spage%253D44%26rft.epage%253D57%26rft.atitle%253DSome%2Bobservations%2Bon%2Basymmetrical%2Bcorrelated%2Bresponses%2Bto%2Bselection.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672300009460%26rft_id%253Dinfo%253Apmid%252F5906491%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672300009460&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=5906491&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3153.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A19667422100005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3153.12" data-doi="10.1016/j.cell.2017.05.038"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Boyle</span>,  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y. I.</span> <span class="cit-name-surname">Li</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. K.</span> <span class="cit-name-surname">Pritchard</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">An Expanded View of Complex Traits : From Polygenic to Omnigenic.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">169</span>: <span class="cit-fpage">1177</span>–<span class="cit-lpage">1186</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.cell.2017.05.038</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D169%26rft.spage%253D1177%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2017.05.038%26rft_id%253Dinfo%253Apmid%252F28622505%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2017.05.038&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28622505&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3153.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3153.13" data-doi="10.1093/bioinformatics/btg112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Broman</span>,  <span class="cit-name-given-names">K. W.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Wu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Sen</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">G. A.</span> <span class="cit-name-surname">Churchill</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">R/qtl: QTL mapping in experimental crosses.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">889</span>–<span class="cit-lpage">890</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btg112</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DBroman%26rft.auinit1%253DK.%2BW.%26rft.volume%253D19%26rft.issue%253D7%26rft.spage%253D889%26rft.epage%253D890%26rft.atitle%253DR%252Fqtl%253A%2BQTL%2Bmapping%2Bin%2Bexperimental%2Bcrosses%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtg112%26rft_id%253Dinfo%253Apmid%252F12724300%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btg112&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12724300&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3153.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000182634800014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3153.14" data-doi="10.1038/ng.3406"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bulik-Sullivan</span>,  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H. K.</span> <span class="cit-name-surname">Finucane</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Anttila</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Gusev</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F. R.</span> <span class="cit-name-surname">Day</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">An atlas of genetic correlations across human diseases and traits.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">1236</span>–<span class="cit-lpage">1241</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.3406</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D47%26rft.spage%253D1236%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.3406%26rft_id%253Dinfo%253Apmid%252F26414676%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.3406&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26414676&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3153.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3153.15" data-doi="10.1086/282718"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bulmer</span>,  <span class="cit-name-given-names">M. G.</span></span></li></ol><cite>, <span class="cit-pub-date">1971</span> <span class="cit-article-title">The Effect of Selection on Genetic Variability.</span> <abbr class="cit-jnl-abbrev">Am. Nat.</abbr> <span class="cit-vol">105</span>: <span class="cit-fpage">201</span>–<span class="cit-lpage">211</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1086/282718</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BNat.%26rft.volume%253D105%26rft.spage%253D201%26rft_id%253Dinfo%253Adoi%252F10.1086%252F282718%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/282718&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1971J801400001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3153.16" data-doi="10.1186/1297-9686-43-26"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Calus</span>,  <span class="cit-name-given-names">M. P.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. F.</span> <span class="cit-name-surname">Veerkamp</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Accuracy of multi-trait genomic selection using different methods.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">26</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1297-9686-43-26</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DCalus%26rft.auinit1%253DM.%2BP.%26rft.volume%253D43%26rft.spage%253D26%26rft.epage%253D26%26rft.atitle%253DAccuracy%2Bof%2Bmulti-trait%2Bgenomic%2Bselection%2Busing%2Bdifferent%2Bmethods.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-43-26%26rft_id%253Dinfo%253Apmid%252F21729282%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-43-26&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21729282&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3153.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3153.17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chao</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Dubcovsky</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Dvořák</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.-C.</span> <span class="cit-name-surname">Luo</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. P.</span> <span class="cit-name-surname">Baenziger</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Population- and genome-specific patterns of linkage disequilibrium and SNP variation in spring and winter wheat (Triticum aestivum L.).</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">17</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D11%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3153.18"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chi</span>,  <span class="cit-name-given-names">R. K.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. A.</span> <span class="cit-name-surname">Eberhart</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">L. H.</span> <span class="cit-name-surname">Penny</span></span></li></ol><cite>, <span class="cit-pub-date">1969</span> <span class="cit-article-title">Covariances Among Relatives in a Maize Variety (Zea mays L.).</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">63</span>: <span class="cit-fpage">511</span>–<span class="cit-lpage">520</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DChi%26rft.auinit1%253DR.%2BK.%26rft.volume%253D63%26rft.issue%253D2%26rft.spage%253D511%26rft.epage%253D520%26rft.atitle%253DCOVARIANCES%2BAMONG%2BRELATIVES%2BIN%2BA%2BMAIZE%2BVARIETY%2B%2528ZEA%2BMAYS%2B%2BL.%2529%26rft_id%253Dinfo%253Apmid%252F17248468%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo4OiJnZW5ldGljcyI7czo1OiJyZXNpZCI7czo4OiI2My8yLzUxMSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzkvMTAvMzE1My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3153.19" data-doi="10.1186/1471-2164-10-582"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Close</span>,  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. R.</span> <span class="cit-name-surname">Bhat</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Lonardi</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Wu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Rostoks</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Development and implementation of high-throughput SNP genotyping in barley.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">582</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2164-10-582</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DClose%26rft.auinit1%253DT.%2BJ.%26rft.volume%253D10%26rft.spage%253D582%26rft.epage%253D582%26rft.atitle%253DDevelopment%2Band%2Bimplementation%2Bof%2Bhigh-throughput%2BSNP%2Bgenotyping%2Bin%2Bbarley.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-10-582%26rft_id%253Dinfo%253Apmid%252F19961604%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-10-582&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19961604&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3153.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3153.20" data-doi="10.2135/cropsci2012.11.0666"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Combs</span>,  <span class="cit-name-given-names">E.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Bernardo</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genomewide selection to introgress semidwarf maize germplasm into US Corn Belt inbreds.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">53</span>: <span class="cit-fpage">1427</span>–<span class="cit-lpage">1436</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2012.11.0666</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D53%26rft.spage%253D1427%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2012.11.0666%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2012.11.0666&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3153.21" data-doi="10.1371/journal.pone.0156744"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Covarrubias-Pazaran</span>,  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genome-Assisted prediction of quantitative traits using the r package sommer.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">11</span>: <span class="cit-elocation-id">e0156744</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0156744</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D11%26rft.spage%253De0156744%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0156744%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0156744&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3153.22" data-doi="10.1371/journal.pone.0003395"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Daetwyler</span>,  <span class="cit-name-given-names">H. D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Villanueva</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. A.</span> <span class="cit-name-surname">Woolliams</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Accuracy of predicting the genetic risk of disease using a genome-wide approach.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">3</span>: <span class="cit-elocation-id">e3395</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0003395</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DDaetwyler%26rft.auinit1%253DH.%2BD.%26rft.volume%253D3%26rft.issue%253D10%26rft.spage%253De3395%26rft.epage%253De3395%26rft.atitle%253DAccuracy%2Bof%2Bpredicting%2Bthe%2Bgenetic%2Brisk%2Bof%2Bdisease%2Busing%2Ba%2Bgenome-wide%2Bapproach.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0003395%26rft_id%253Dinfo%253Apmid%252F18852893%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0003395&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18852893&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3153.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3153.23" data-doi="10.1534/genetics.110.116855"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Daetwyler</span>,  <span class="cit-name-given-names">H. D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Pong-Wong</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Villanueva</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Woolliams</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The impact of genetic architecture on genome-wide evaluation methods.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">185</span>: <span class="cit-fpage">1021</span>–<span class="cit-lpage">1031</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.110.116855</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.116855%26rft_id%253Dinfo%253Apmid%252F20407128%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE4NS8zLzEwMjEiO3M6NDoiYXRvbSI7czoxOToiL2dnZy85LzEwLzMxNTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3153.24" data-doi="10.1534/genetics.117.300347"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Deng</span>,  <span class="cit-name-given-names">Y.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Pan</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Testing Genetic Pleiotropy with GWAS Summary Statistics for Marginal and Conditional Analyses.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">207</span>: <span class="cit-fpage">1285</span>–<span class="cit-lpage">1299</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.117.300347</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.117.300347%26rft_id%253Dinfo%253Apmid%252F28971959%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjIwNy80LzEyODUiO3M6NDoiYXRvbSI7czoxOToiL2dnZy85LzEwLzMxNTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-book" id="cit-9.10.3153.25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Falconer</span>,  <span class="cit-name-given-names">D. S.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. F. C.</span> <span class="cit-name-surname">Mackay</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-source">Introduction to Quantitative Genetics</span>, <span class="cit-publ-name">Pearson Prentice Hall</span>, <span class="cit-publ-loc">Harlow, Essex</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3153.26"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Felsenstein</span>,  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1965</span> <span class="cit-article-title">The Effect of Linkage on Directional Selection.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">349</span>–<span class="cit-lpage">363</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DFelsenstein%26rft.auinit1%253DJ.%26rft.volume%253D52%26rft.issue%253D2%26rft.spage%253D349%26rft.epage%253D363%26rft.atitle%253DTHE%2BEFFECT%2BOF%2BLINKAGE%2BON%2BDIRECTIONAL%2BSELECTION%26rft_id%253Dinfo%253Apmid%252F5861564%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo4OiJnZW5ldGljcyI7czo1OiJyZXNpZCI7czo4OiI1Mi8yLzM0OSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzkvMTAvMzE1My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3153.27" data-doi="10.1017/S0080456800012163"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fisher</span>,  <span class="cit-name-given-names">R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1919</span> <span class="cit-article-title">The Correlation between Relatives on the Supposition of Mendelian Inheritance.</span> <abbr class="cit-jnl-abbrev">Trans. R. Soc. Edinb.</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">399</span>–<span class="cit-lpage">433</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1017/S0080456800012163</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrans.%2BR.%2BSoc.%2BEdinb.%26rft.volume%253D52%26rft.spage%253D399%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0080456800012163%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0080456800012163&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3153.28" data-doi="10.1186/1471-2105-12-186"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Habier</span>,  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. L.</span> <span class="cit-name-surname">Fernando</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Kizilkaya</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. J.</span> <span class="cit-name-surname">Garrick</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Extension of the bayesian alphabet for genomic selection.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">186</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2105-12-186</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DHabier%26rft.auinit1%253DD.%26rft.volume%253D12%26rft.spage%253D186%26rft.epage%253D186%26rft.atitle%253DExtension%2Bof%2Bthe%2Bbayesian%2Balphabet%2Bfor%2Bgenomic%2Bselection.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-12-186%26rft_id%253Dinfo%253Apmid%252F21605355%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-12-186&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21605355&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3153.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3153.29" data-doi="10.2135/cropsci2009.04.0198"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hamblin</span>,  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. J.</span> <span class="cit-name-surname">Close</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. R.</span> <span class="cit-name-surname">Bhat</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Chao</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. G.</span> <span class="cit-name-surname">Kling</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Population Structure and Linkage Disequilibrium in U.S. Barley Germplasm: Implications for Association Mapping.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">50</span>: <span class="cit-fpage">556</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2009.04.0198</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DHamblin%26rft.auinit1%253DM.%2BT.%26rft.volume%253D50%26rft.issue%253D2%26rft.spage%253D556%26rft.epage%253D566%26rft.atitle%253DPopulation%2BStructure%2Band%2BLinkage%2BDisequilibrium%2Bin%2BU.S.%2BBarley%2BGermplasm%253A%2BImplications%2Bfor%2BAssociation%2BMapping%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2009.04.0198%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2009.04.0198&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000275564500013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3153.30" data-doi="10.1186/1297-9686-42-35"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>,  <span class="cit-name-given-names">J.-L.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Dynamics of long-term genomic selection.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">35</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1297-9686-42-35</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DJannink%26rft.auinit1%253DJ.%2BL.%26rft.volume%253D42%26rft.spage%253D35%26rft.epage%253D35%26rft.atitle%253DDynamics%2Bof%2Blong-term%2Bgenomic%2Bselection.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-42-35%26rft_id%253Dinfo%253Apmid%252F20712894%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-42-35&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20712894&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3153.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3153.31" data-doi="10.1534/genetics.112.144246"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jia</span>,  <span class="cit-name-given-names">Y.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.-L.</span> <span class="cit-name-surname">Jannink</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Multiple-Trait Genomic Selection Methods Increase Genetic Value Prediction Accuracy.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">192</span>: <span class="cit-fpage">1513</span>–<span class="cit-lpage">1522</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.112.144246</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.144246%26rft_id%253Dinfo%253Apmid%252F23086217%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5Mi80LzE1MTMiO3M6NDoiYXRvbSI7czoxOToiL2dnZy85LzEwLzMxNTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3153.32" data-doi="10.1038/ng.2376"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Korte</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. J.</span> <span class="cit-name-surname">Vilhjálmsson</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Segura</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Platt</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Q.</span> <span class="cit-name-surname">Long</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A mixed-model approach for genome-wide association studies of correlated traits in structured populations.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">1066</span>–<span class="cit-lpage">1071</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.2376</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DKorte%26rft.auinit1%253DA.%26rft.volume%253D44%26rft.issue%253D9%26rft.spage%253D1066%26rft.epage%253D1071%26rft.atitle%253DA%2Bmixed-model%2Bapproach%2Bfor%2Bgenome-wide%2Bassociation%2Bstudies%2Bof%2Bcorrelated%2Btraits%2Bin%2Bstructured%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2376%26rft_id%253Dinfo%253Apmid%252F22902788%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2376&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22902788&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3153.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3153.33" data-doi="10.1017/S0016672300026549"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lande</span>,  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1984</span> <span class="cit-article-title">The genetic correlation between characters maintained by selection, linkage and inbreeding.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">309</span>–<span class="cit-lpage">320</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1017/S0016672300026549</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DLande%26rft.auinit1%253DR.%26rft.volume%253D44%26rft.issue%253D3%26rft.spage%253D309%26rft.epage%253D320%26rft.atitle%253DThe%2Bgenetic%2Bcorrelation%2Bbetween%2Bcharacters%2Bmaintained%2Bby%2Bselection%252C%2Blinkage%2Band%2Binbreeding.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672300026549%26rft_id%253Dinfo%253Apmid%252F6530140%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672300026549&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=6530140&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3153.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1984ACP8900007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3153.34"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lande</span>,  <span class="cit-name-given-names">R.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Thompson</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">Efficiency of marker-assisted selection in the improvement of quantitative traits. Genetics</span> <span class="cit-vol">124</span>: <span class="cit-fpage">743</span>–<span class="cit-lpage">756</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3153.35" data-doi="10.1093/bioinformatics/bts474"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>,  <span class="cit-name-given-names">S. H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Yang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. E.</span> <span class="cit-name-surname">Goddard</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. M.</span> <span class="cit-name-surname">Visscher</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">N. R.</span> <span class="cit-name-surname">Wray</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Estimation of pleiotropy between complex diseases using single-nucleotide polymorphism-derived genomic relationships and restricted maximum likelihood.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">2540</span>–<span class="cit-lpage">2542</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/bts474</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbts474%26rft_id%253Dinfo%253Apmid%252F22843982%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bts474&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22843982&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3153.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000309687500025&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3153.36" data-doi="10.1534/genetics.117.300403"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lehermeier</span>,  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Teyssèdre</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.-C.</span> <span class="cit-name-surname">Schön</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Genetic Gain Increases by Applying the Usefulness Criterion with Improved Variance Prediction in Selection of Crosses.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">207</span>: <span class="cit-fpage">1651</span>–<span class="cit-lpage">1661</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.117.300403</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.117.300403%26rft_id%253Dinfo%253Apmid%252F29038144%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjIwNy80LzE2NTEiO3M6NDoiYXRvbSI7czoxOToiL2dnZy85LzEwLzMxNTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3153.37" data-doi="10.2135/cropsci2014.10.0729"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lian</span>,  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Jacobson</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Zhong</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Bernardo</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Prediction of genetic variance in biparental maize populations: Genomewide marker effects <em>vs.</em> mean genetic variance in prior populations.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">55</span>: <span class="cit-fpage">1181</span>–<span class="cit-lpage">1188</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2014.10.0729</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D55%26rft.spage%253D1181%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2014.10.0729%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2014.10.0729&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3153.38" data-doi="10.2135/cropsci2011.09.0503"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lorenz</span>,  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. P.</span> <span class="cit-name-surname">Smith</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.-L.</span> <span class="cit-name-surname">Jannink</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Potential and Optimization of Genomic Selection for Fusarium Head Blight Resistance in Six-Row Barley.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">1609</span>–<span class="cit-lpage">1621</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2011.09.0503</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D52%26rft.spage%253D1609%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2011.09.0503%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2011.09.0503&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000305804900016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-book" id="cit-9.10.3153.39"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>,  <span class="cit-name-given-names">M.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Walsh</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-source">Genetics and Analysis of Quantitative Traits</span>, <span class="cit-publ-name">Sinauer Associates, Inc.</span>, <span class="cit-publ-loc">Sunderland, MA</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3153.40" data-doi="10.1098/rstb.2009.0315"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mackay</span>,  <span class="cit-name-given-names">T. F. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Mutations and quantitative genetic variation: Lessons from Drosophila.</span> <abbr class="cit-jnl-abbrev">Philos. Trans. R. Soc. B</abbr> <span class="cit-vol">365</span>: <span class="cit-fpage">1229</span>–<span class="cit-lpage">1239</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1098/rstb.2009.0315</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhilosophical%2BTransactions%2Bof%2Bthe%2BRoyal%2BSociety%2BB%253A%2BBiological%2BSciences%26rft.stitle%253DPhil%2BTrans%2BR%2BSoc%2BB%26rft.aulast%253DMackay%26rft.auinit1%253DT.%2BF.%2BC.%26rft.volume%253D365%26rft.issue%253D1544%26rft.spage%253D1229%26rft.epage%253D1239%26rft.atitle%253DMutations%2Band%2Bquantitative%2Bgenetic%2Bvariation%253A%2Blessons%2Bfrom%2BDrosophila%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frstb.2009.0315%26rft_id%253Dinfo%253Apmid%252F20308098%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1098/rstb.2009.0315&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20308098&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3153.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3153.41" data-doi="10.1093/nar/gkv1344"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>,  <span class="cit-name-given-names">T. H. E.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. J.</span> <span class="cit-name-surname">Hayes</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. E.</span> <span class="cit-name-surname">Goddard</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Prediction of total genetic value using genome-wide dense marker maps.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">157</span>: <span class="cit-fpage">1819</span>–<span class="cit-lpage">1829</span>. <a href="https://doi.org/111290733">https://doi.org/111290733</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMeuwissen%26rft.auinit1%253DT.%2BH.%2BE.%26rft.volume%253D157%26rft.issue%253D4%26rft.spage%253D1819%26rft.epage%253D1829%26rft.atitle%253DPrediction%2Bof%2BTotal%2BGenetic%2BValue%2BUsing%2BGenome-Wide%2BDense%2BMarker%2BMaps%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkv1344%26rft_id%253Dinfo%253Apmid%252F11290733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1Ny80LzE4MTkiO3M6NDoiYXRvbSI7czoxOToiL2dnZy85LzEwLzMxNTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3153.42" data-doi="10.2135/cropsci2015.01.0030"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mohammadi</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Tiede</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. P.</span> <span class="cit-name-surname">Smith</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">PopVar: A Genome-Wide Procedure for Predicting Genetic Variance and Correlated Response in Biparental Breeding Populations.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">55</span>: <span class="cit-fpage">2068</span>–<span class="cit-lpage">2077</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2015.01.0030</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D55%26rft.spage%253D2068%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2015.01.0030%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2015.01.0030&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3153.43" data-doi="10.3835/plantgenome2011.08.0023"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Muñoz-Amatriaín</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. J.</span> <span class="cit-name-surname">Moscou</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. R.</span> <span class="cit-name-surname">Bhat</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. T.</span> <span class="cit-name-surname">Svensson</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Bartoš</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">An Improved Consensus Linkage Map of Barley Based on Flow-Sorted Chromosomes and Single Nucleotide Polymorphism Markers.</span> <abbr class="cit-jnl-abbrev">Plant Genome J.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">238</span>–<span class="cit-lpage">249</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2011.08.0023</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%2BJ.%26rft.volume%253D4%26rft.spage%253D238%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2011.08.0023%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2011.08.0023&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000312661700008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3153.44" data-doi="10.2135/cropsci2018.11.0716"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Neyhart</span>,  <span class="cit-name-given-names">J. L.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. P.</span> <span class="cit-name-surname">Smith</span></span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">Validating Genomewide Predictions of Genetic Variance in a Contemporary Breeding Program.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">59</span>: <span class="cit-fpage">1062</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2018.11.0716</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D59%26rft.spage%253D1062%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2018.11.0716%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2018.11.0716&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-book" id="cit-9.10.3153.45" data-doi="10.1371/journal.pone.0188839"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Osthushenrich</span>,  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Frisch</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Herzog</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Genomic selection of crossing partners on basis of the expected mean and variance of their derived lines.</span> PLoS One <span class="cit-vol">12</span>: <span class="cit-fpage">e0188839</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0188839</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.volume%253D12%26rft.spage%253De0188839%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0188839%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0188839&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-book" id="cit-9.10.3153.46"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">R Core Team</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-source">R: A Language and Environment for Statistical Computing</span>, <span class="cit-publ-name">R Foundation for Statistical Computing</span>, <span class="cit-publ-loc">Vienna, Austria</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3153.47" data-doi="10.2307/2527750"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Robertson</span>,  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1959</span> <span class="cit-article-title">The Sampling Variance of the Genetic Correlation Coefficient.</span> <abbr class="cit-jnl-abbrev">Biometrics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">469</span>–<span class="cit-lpage">485</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2307/2527750</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiometrics%26rft.volume%253D15%26rft.spage%253D469%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2527750%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2527750&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1959WX52300011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3153.48" data-doi="10.3835/plantgenome2014.05.0020"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sallam</span>,  <span class="cit-name-given-names">A. H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. B.</span> <span class="cit-name-surname">Endelman</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.-L.</span> <span class="cit-name-surname">Jannink</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. P.</span> <span class="cit-name-surname">Smith</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Assessing Genomic Selection Prediction Accuracy in a Dynamic Barley Breeding Population.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">0</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2014.05.0020</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2014.05.0020%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2014.05.0020&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3153.49" data-doi="10.1534/genetics.116.189308"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schaid</span>,  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Tong</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Larrabee</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. B.</span> <span class="cit-name-surname">Kennedy</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G. A.</span> <span class="cit-name-surname">Poland</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Statistical methods for testing genetic pleiotropy.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">204</span>: <span class="cit-fpage">483</span>–<span class="cit-lpage">497</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.116.189308</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.116.189308%26rft_id%253Dinfo%253Apmid%252F27527515%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjA0LzIvNDgzIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMC8zMTUzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-book" id="cit-9.10.3153.50"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schnell</span>,  <span class="cit-name-given-names">F. W.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">H. F.</span> <span class="cit-name-surname">Utz</span></span></li></ol><cite>, <span class="cit-pub-date">1975</span> <span class="cit-article-title">F1-leistung und elternwahl euphyder züchtung von selbstbefruchtern</span>, pp. <span class="cit-fpage">243</span>–<span class="cit-lpage">248</span> in <span class="cit-source">Bericht über die Arbeitstagung der Vereinigung Österreichischer Pflanzenzüchter</span>, <span class="cit-publ-name">BAL Gumpenstein</span>, <span class="cit-publ-loc">Gumpenstein, Austria</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3153.51" data-doi="10.1002/jsfa.2740670306"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Simmonds</span>,  <span class="cit-name-given-names">N. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">The relation between yield and protein in cereal grain.</span> <abbr class="cit-jnl-abbrev">J. Sci. Food Agric.</abbr> <span class="cit-vol">67</span>: <span class="cit-fpage">309</span>–<span class="cit-lpage">315</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/jsfa.2740670306</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BSci.%2BFood%2BAgric.%26rft.volume%253D67%26rft.spage%253D309%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fjsfa.2740670306%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/jsfa.2740670306&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1995QM87300005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3153.52" data-doi="10.2135/cropsci2017.12.0719"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sleper</span>,  <span class="cit-name-given-names">J. A.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Bernardo</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Genomewide selection for unfavorably correlated traits in maize.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">58</span>: <span class="cit-fpage">1587</span>–<span class="cit-lpage">1593</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2017.12.0719</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D58%26rft.spage%253D1587%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2017.12.0719%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2017.12.0719&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3153.53" data-doi="10.1007/BF00226219"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Souza</span>,  <span class="cit-name-given-names">E.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. E.</span> <span class="cit-name-surname">Sorrells</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">Prediction of progeny variation in oat from parental genetic relationships.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">82</span>: <span class="cit-fpage">233</span>–<span class="cit-lpage">241</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/BF00226219</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D82%26rft.spage%253D233%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00226219%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00226219&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3153.54" data-doi="10.1007/s11032-015-0390-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tiede</span>,  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Kumar</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Mohammadi</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. P.</span> <span class="cit-name-surname">Smith</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Predicting genetic variance in bi-parental breeding populations is more accurate when explicitly modeling the segregation of informative genomewide markers.</span> <abbr class="cit-jnl-abbrev">Mol. Breed.</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">199</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s11032-015-0390-6</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBreed.%26rft.volume%253D35%26rft.spage%253D199%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11032-015-0390-6%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11032-015-0390-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3153.55" data-doi="10.1007/s11032-018-0820-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tiede</span>,  <span class="cit-name-given-names">T.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. P.</span> <span class="cit-name-surname">Smith</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Evaluation and retrospective optimization of genomic selection for yield and disease resistance in spring barley.</span> <abbr class="cit-jnl-abbrev">Mol. Breed.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">55</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s11032-018-0820-3</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBreed.%26rft.volume%253D38%26rft.spage%253D55%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11032-018-0820-3%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11032-018-0820-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3153.56" data-doi="10.2135/cropsci2001.4151470x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Utz</span>,  <span class="cit-name-given-names">H. F.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Bohn</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. E.</span> <span class="cit-name-surname">Melchinger</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Predicting progeny means and variances of winter wheat crosses from phenotypic values of their parents.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">1470</span>–<span class="cit-lpage">1478</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2001.4151470x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DUtz%26rft.auinit1%253DH.%2BF.%26rft.volume%253D41%26rft.issue%253D5%26rft.spage%253D1470%26rft.epage%253D1478%26rft.atitle%253DPredicting%2BProgeny%2BMeans%2Band%2BVariances%2Bof%2BWinter%2BWheat%2BCrosses%2Bfrom%2BPhenotypic%2BValues%2Bof%2BTheir%2BParents%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2001.4151470x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2001.4151470x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3153.57" data-doi="10.1007/BF00224187"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Villanueva</span>,  <span class="cit-name-given-names">B.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. W.</span> <span class="cit-name-surname">Kennedy</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">Effect of selection on genetic parameters of correlated traits.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">80</span>: <span class="cit-fpage">746</span>–<span class="cit-lpage">752</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/BF00224187</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D80%26rft.spage%253D746%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00224187%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00224187&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3153.58" data-doi="10.1534/genetics.113.150078"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wimmer</span>,  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Lehermeier</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Albrecht</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H. J.</span> <span class="cit-name-surname">Auinger</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Wang</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genome-wide prediction of traits with different genetic architecture through efficient variable selection.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">195</span>: <span class="cit-fpage">573</span>–<span class="cit-lpage">587</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.113.150078</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.150078%26rft_id%253Dinfo%253Apmid%252F23934883%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk1LzIvNTczIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMC8zMTUzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3153.59" data-doi="10.1016/j.cj.2018.05.003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yao</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Zhao</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Chen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Zhang</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Wang</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Use of genomic selection and breeding simulation in cross prediction for improvement of yield and quality in wheat (Triticum aestivum L.).</span> <abbr class="cit-jnl-abbrev">Crop J.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">353</span>–<span class="cit-lpage">365</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.cj.2018.05.003</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BJ.%26rft.volume%253D6%26rft.spage%253D353%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cj.2018.05.003%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cj.2018.05.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3153.60" data-doi="10.1534/genetics.107.075358"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhong</span>,  <span class="cit-name-given-names">S.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.-L.</span> <span class="cit-name-surname">Jannink</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Using quantitative trait loci results to discriminate among crosses on the basis of their progeny mean and variance.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">177</span>: <span class="cit-fpage">567</span>–<span class="cit-lpage">576</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.107.075358</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.075358%26rft_id%253Dinfo%253Apmid%252F17660556%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTc3LzEvNTY3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMC8zMTUzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/9/10/3153.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/10/3139" title="Genome-Wide Association and Gene Co-expression Network Analyses Reveal Complex Genetics of Resistance to Goss’s Wilt of Maize" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/10/3167" title="Genome Size Evolution Differs Between Drosophila Subgenera with Striking Differences in Male and Female Genome Size in Sophophora" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/9/10"><span>Volume 9 Issue 10, October 2019</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/9/10.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/9/10.cover-source.jpg?itok=yA8kifEd" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 9 (10)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/10/3153" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-6Ys6TDcrTb64v5SHtKXfZ0scoieeAqhPml1gbfY-jTg" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/genomic-prediction" class="" data-icon-position="" data-hide-link-title="0">Genomic Prediction</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/9/10/3153" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/9/10/3153" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/9/10/3153" class="active" data-icon-position="" data-hide-link-title="0">Multi-trait Improvement by Predicting Genetic Correlations in Breeding Crosses</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/494269" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-6d9NHvrbTnK9NGO3Pwbcfri6aYRgtjHXHH86nSXTkn0" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/10/3153.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/9/10/3153" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-V5m04_ayA_EYNV9DKSV1GjkS0SGWG2GHYZyLfUz9W0I" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/9/10/3153" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/10/3153.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494269" id="node494269--2" data-pisa="ggg;9/10/3153" data-pisa-master="ggg;g3.119.400406" data-apath="/ggg/9/10/3153.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Multi-trait Improvement by Predicting Genetic Correlations in Breeding Crosses</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-1991-5310" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jeffrey L. Neyhart</span>, <span class="highwire-citation-author" data-delta="1">Aaron J. Lorenz</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0001-8253-3326" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Kevin P. Smith</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3153-3165;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400406 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494269--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jeffrey L. Neyhart </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Agronomy and Plant Genetics, University of Minnesota, St. Paul, MN 55108</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jeffrey%2BL.%2BNeyhart%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Neyhart%20JL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJeffrey%2BL.%2BNeyhart%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-1991-5310" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jeffrey L. Neyhart</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Aaron J. Lorenz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Agronomy and Plant Genetics, University of Minnesota, St. Paul, MN 55108</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Aaron%2BJ.%2BLorenz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lorenz%20AJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAaron%2BJ.%2BLorenz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Kevin P. Smith </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Agronomy and Plant Genetics, University of Minnesota, St. Paul, MN 55108</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kevin%2BP.%2BSmith%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Smith%20KP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKevin%2BP.%2BSmith%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-8253-3326" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Kevin P. Smith</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:smith376@umn.edu" class="" data-icon-position="" data-hide-link-title="0">smith376@umn.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/9/10/3153" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/9/10/3153&amp;title=Multi-trait%20Improvement%20by%20Predicting%20Genetic%20Correlations%20in%20Breeding%20Crosses" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/9/10/3153&amp;title=Multi-trait%20Improvement%20by%20Predicting%20Genetic%20Correlations%20in%20Breeding%20Crosses" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/9/10/3153&amp;title=Multi-trait%20Improvement%20by%20Predicting%20Genetic%20Correlations%20in%20Breeding%20Crosses" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/9/10/3153&amp;text=Multi-trait%20Improvement%20by%20Predicting%20Genetic%20Correlations%20in%20Breeding%20Crosses" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/9/10/3153&amp;title=Multi-trait%20Improvement%20by%20Predicting%20Genetic%20Correlations%20in%20Breeding%20Crosses" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/9/10/3153&amp;t=Multi-trait%20Improvement%20by%20Predicting%20Genetic%20Correlations%20in%20Breeding%20Crosses" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/9/10/3153&amp;title=Multi-trait%20Improvement%20by%20Predicting%20Genetic%20Correlations%20in%20Breeding%20Crosses" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/9/10/3153&amp;title=Multi-trait%20Improvement%20by%20Predicting%20Genetic%20Correlations%20in%20Breeding%20Crosses" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/494269/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494269" id="node494269--3" data-pisa="ggg;9/10/3153" data-pisa-master="ggg;g3.119.400406" data-apath="/ggg/9/10/3153.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Multi-trait Improvement by Predicting Genetic Correlations in Breeding Crosses</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-1991-5310" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jeffrey L. Neyhart</span>, <span class="highwire-citation-author" data-delta="1">Aaron J. Lorenz</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0001-8253-3326" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Kevin P. Smith</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3153-3165;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400406 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494269--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jeffrey L. Neyhart </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Agronomy and Plant Genetics, University of Minnesota, St. Paul, MN 55108</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jeffrey%2BL.%2BNeyhart%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Neyhart%20JL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJeffrey%2BL.%2BNeyhart%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-1991-5310" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jeffrey L. Neyhart</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Aaron J. Lorenz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Agronomy and Plant Genetics, University of Minnesota, St. Paul, MN 55108</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Aaron%2BJ.%2BLorenz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lorenz%20AJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAaron%2BJ.%2BLorenz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Kevin P. Smith </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Agronomy and Plant Genetics, University of Minnesota, St. Paul, MN 55108</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kevin%2BP.%2BSmith%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Smith%20KP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKevin%2BP.%2BSmith%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-8253-3326" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Kevin P. Smith</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:smith376@umn.edu" class="" data-icon-position="" data-hide-link-title="0">smith376@umn.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=3153&amp;rft.epage=3165&amp;rft.atitle=Multi-trait%20Improvement%20by%20Predicting%20Genetic%20Correlations%20in%20Breeding%20Crosses&amp;rft.volume=9&amp;rft.issue=10&amp;rft.date=2019-10-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Neyhart%2C+Jeffrey+L.&amp;rft.au=Lorenz%2C+Aaron+J.&amp;rft.au=Smith%2C+Kevin+P."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/494269/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/494269/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/494269/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/494269/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/494269/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/494269/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/494269/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/494269/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/494269/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/494269/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/494269/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/494269/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495864" id="node495864" data-pisa="ggg;10/3/1113" data-pisa-master="ggg;g3.119.400968" data-apath="/ggg/10/3/1113.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1113" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Multi-trait Genomic Prediction Model Increased the Predictive Ability for Agronomic and Malting Quality Traits in Barley (<em>Hordeum vulgare</em> L.)</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495543" id="node495543" data-pisa="ggg;10/2/769" data-pisa-master="ggg;g3.119.400759" data-apath="/ggg/10/2/769.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/2/769" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Novel Bayesian Networks for Genomic Prediction of Developmental Traits in Biomass Sorghum</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495557" id="node495557" data-pisa="ggg;10/2/539" data-pisa-master="ggg;g3.119.400866" data-apath="/ggg/10/2/539.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/2/539" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Optimized Genetic Testing for Polledness in Multiple Breeds of Cattle</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Genomic%20Prediction" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Genomic Prediction</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/9/10/3153" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Methods and Materials</a></li><li class="odd"><a href="#sec-8" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-12" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/9/10/3153.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/9/10/3153.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/9/10/3153.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
