<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/9/10/3057" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/9/10/3057.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/9/10/3057.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/9/10/3057.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/9/10/3057.atom" />
<meta name="HW.pisa" content="ggg;9/10/3057" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Whole Genome Sequence of the Commercially Relevant Mushroom Strain Agaricus bisporus var. bisporus ARP23" />
<meta name="DC.Identifier" content="10.1534/g3.119.400563" />
<meta name="DC.Date" content="2019-10-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2019 OConnor et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Agaricus bisporus is an extensively cultivated edible mushroom. Demand for cultivation is continuously growing and difficulties associated with breeding programs now means strains are effectively considered monoculture. While commercial growing practices are highly efficient and tightly controlled, the over-use of a single strain has led to a variety of disease outbreaks from a range of pathogens including bacteria, fungi and viruses. To address this, the Agaricus Resource Program (ARP) was set up to collect wild isolates from diverse geographical locations through a bounty-driven scheme to create a repository of wild Agaricus germplasm. One of the strains collected, Agaricus bisporus var. bisporus ARP23, has been crossed extensively with white commercial varieties leading to the generation of a novel hybrid with a dark brown pileus commonly referred to as ‘Heirloom’. Heirloom has been successfully implemented into commercial mushroom cultivation. In this study the whole genome of Agaricus bisporus var. bisporus ARP23 was sequenced and assembled with Illumina and PacBio sequencing technology. The final genome was found to be 33.49 Mb in length and have significant levels of synteny to other sequenced Agaricus bisporus strains. Overall, 13,030 putative protein coding genes were located and annotated. Relative to the other A. bisporus genomes that are currently available, Agaricus bisporus var. bisporus ARP23 is the largest A. bisporus strain in terms of gene number and genetic content sequenced to date. Comparative genomic analysis shows that the A. bisporus mating loci in unifactorial and unsurprisingly highly conserved between strains. The lignocellulolytic gene content of all A. bisporus strains compared is also very similar. Our results show that the pangenome structure of A. bisporus is quite diverse with between 60–70% of the total protein coding genes per strain considered as being orthologous and syntenically conserved. These analyses and the genome sequence described herein are the starting point for more detailed molecular analyses into the growth and phenotypical responses of Agaricus bisporus var. bisporus ARP23 when challenged with economically important mycoviruses." />
<meta name="DC.Contributor" content="Eoin O’Connor" />
<meta name="DC.Contributor" content="Jamie McGowan" />
<meta name="DC.Contributor" content="Charley G. P. McCarthy" />
<meta name="DC.Contributor" content="Aniça Amini" />
<meta name="DC.Contributor" content="Helen Grogan" />
<meta name="DC.Contributor" content="David A. Fitzpatrick" />
<meta name="article:published_time" content="2019-10-01" />
<meta name="article:section" content="Genome Report" />
<meta name="citation_title" content="Whole Genome Sequence of the Commercially Relevant Mushroom Strain Agaricus bisporus var. bisporus ARP23" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;&lt;i&gt;Agaricus bisporus&lt;/i&gt; is an extensively cultivated edible mushroom. Demand for cultivation is continuously growing and difficulties associated with breeding programs now means strains are effectively considered monoculture. While commercial growing practices are highly efficient and tightly controlled, the over-use of a single strain has led to a variety of disease outbreaks from a range of pathogens including bacteria, fungi and viruses. To address this, the Agaricus Resource Program (ARP) was set up to collect wild isolates from diverse geographical locations through a bounty-driven scheme to create a repository of wild &lt;i&gt;Agaricus&lt;/i&gt; germplasm. One of the strains collected, &lt;i&gt;Agaricus bisporus&lt;/i&gt; var. &lt;i&gt;bisporus&lt;/i&gt; ARP23, has been crossed extensively with white commercial varieties leading to the generation of a novel hybrid with a dark brown pileus commonly referred to as ‘Heirloom’. Heirloom has been successfully implemented into commercial mushroom cultivation. In this study the whole genome of &lt;i&gt;Agaricus bisporus&lt;/i&gt; var. &lt;i&gt;bisporus&lt;/i&gt; ARP23 was sequenced and assembled with Illumina and PacBio sequencing technology. The final genome was found to be 33.49 Mb in length and have significant levels of synteny to other sequenced &lt;i&gt;Agaricus bisporus&lt;/i&gt; strains. Overall, 13,030 putative protein coding genes were located and annotated. Relative to the other &lt;i&gt;A. bisporus&lt;/i&gt; genomes that are currently available, &lt;i&gt;Agaricus bisporus&lt;/i&gt; var. &lt;i&gt;bisporus&lt;/i&gt; ARP23 is the largest &lt;i&gt;A. bisporus&lt;/i&gt; strain in terms of gene number and genetic content sequenced to date. Comparative genomic analysis shows that the &lt;i&gt;A. bisporus&lt;/i&gt; mating loci in unifactorial and unsurprisingly highly conserved between strains. The lignocellulolytic gene content of all &lt;i&gt;A. bisporus&lt;/i&gt; strains compared is also very similar. Our results show that the pangenome structure of &lt;i&gt;A. bisporus&lt;/i&gt; is quite diverse with between 60–70% of the total protein coding genes per strain considered as being orthologous and syntenically conserved. These analyses and the genome sequence described herein are the starting point for more detailed molecular analyses into the growth and phenotypical responses of &lt;i&gt;Agaricus bisporus&lt;/i&gt; var. &lt;i&gt;bisporus&lt;/i&gt; ARP23 when challenged with economically important mycoviruses.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2019/10/01" />
<meta name="citation_mjid" content="ggg;9/10/3057" />
<meta name="citation_id" content="9/10/3057" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/9/10/3057" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/9/10/3057.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/9/10/3057.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/9/10/3057.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.119.400563" />
<meta name="citation_pmid" content="31371382" />
<meta name="citation_volume" content="9" />
<meta name="citation_issue" content="10" />
<meta name="citation_num_pages" content="10" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Genome Report" />
<meta name="citation_firstpage" content="3057" />
<meta name="citation_lastpage" content="3066" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Eoin O’Connor" />
<meta name="citation_author_institution" content="Genome Evolution Laboratory, Department of Biology, Maynooth University, Maynooth, Co. Kildare, Ireland," />
<meta name="citation_author_institution" content="Teagasc Food Research Centre, Ashtown, Dublin 15, D15 KN3K, Ireland," />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-4626-5605" />
<meta name="citation_author" content="Jamie McGowan" />
<meta name="citation_author_institution" content="Genome Evolution Laboratory, Department of Biology, Maynooth University, Maynooth, Co. Kildare, Ireland," />
<meta name="citation_author_institution" content="Human Health Research Institute, Maynooth University, Maynooth, Co. Kildare, Ireland, and" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-4240-5175" />
<meta name="citation_author" content="Charley G. P. McCarthy" />
<meta name="citation_author_institution" content="Genome Evolution Laboratory, Department of Biology, Maynooth University, Maynooth, Co. Kildare, Ireland," />
<meta name="citation_author_institution" content="Human Health Research Institute, Maynooth University, Maynooth, Co. Kildare, Ireland, and" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-3880-9978" />
<meta name="citation_author" content="Aniça Amini" />
<meta name="citation_author_institution" content="Sylvan-Somycel (ESSC - Unité 2), ZI SUD, rue Lavoisier, BP 25, 37130 Langeais, France" />
<meta name="citation_author" content="Helen Grogan" />
<meta name="citation_author_institution" content="Teagasc Food Research Centre, Ashtown, Dublin 15, D15 KN3K, Ireland," />
<meta name="citation_author" content="David A. Fitzpatrick" />
<meta name="citation_author_institution" content="Genome Evolution Laboratory, Department of Biology, Maynooth University, Maynooth, Co. Kildare, Ireland," />
<meta name="citation_author_institution" content="Human Health Research Institute, Maynooth University, Maynooth, Co. Kildare, Ireland, and" />
<meta name="citation_author_email" content="david.fitzpatrick@mu.ie" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-7345-6998" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=S. F. Altschul;citation_author=T. L. Madden;citation_author=A. A. Schäffer;citation_author=J. Zhang;citation_author=Z. Zhang;citation_title=Gapped BLAST and PSI-BLAST: a new generation of protein database search programs.;citation_volume=25;citation_year=1997;citation_pmid=9254694;citation_doi=10.1093/nar/25.17.3389" />
<meta name="citation_reference" content="citation_journal_title=Nat. Biotechnol.;citation_author=S. Arora;citation_author=B. Steuernagel;citation_author=K. Gaurav;citation_author=S. Chandramohan;citation_author=Y. Long;citation_title=Resistance gene cloning from a wild crop relative by sequence capture and association genetics.;citation_pages=139-143;citation_volume=37;citation_year=2019;citation_doi=10.1038/s41587-018-0007-9" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=M. Ashburner;citation_author=C. A. Ball;citation_author=J. A. Blake;citation_author=D. Botstein;citation_author=H. Butler;citation_title=Gene ontology: tool for the unification of biology. The Gene Ontology Consortium.;citation_pages=25-29;citation_volume=25;citation_year=2000;citation_issue=1;citation_pmid=10802651;citation_doi=10.1038/75556" />
<meta name="citation_reference" content="citation_journal_title=Database (Oxford);citation_author=R. Balakrishnan;citation_author=J. Park;citation_author=K. Karra;citation_author=B. C. Hitz;citation_author=G. Binkley;citation_title=YeastMine-An integrated data warehouse for Saccharomyces cerevisiae data as a multipurpose tool-kit.;citation_volume=2012;citation_year=2012;citation_pmid=22434830;citation_doi=10.1093/database/bar062" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=G. Benson;citation_title=Tandem repeats finder: A program to analyze DNA sequences.;citation_volume=27;citation_year=1999;citation_pmid=9862982;citation_doi=10.1093/nar/27.2.573" />
<meta name="citation_reference" content="citation_journal_title=BMC Bioinformatics;citation_author=M. Boetzer;citation_author=W. Pirovano;citation_title=SSPACE-LongRead: scaffolding bacterial draft genomes using long read sequence information.;citation_pages=211;citation_volume=15;citation_year=2014;citation_pmid=24950923;citation_doi=10.1186/1471-2105-15-211" />
<meta name="citation_reference" content="citation_title=Mushroom Biology and Mushroom Products. ;citation_title=The Two Life Cycles of Agaricus bisporus;citation_pages=57-66;citation_year=1996" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=C. Camacho;citation_author=G. Coulouris;citation_author=V. Avagyan;citation_author=N. Ma;citation_author=J. Papadopoulos;citation_title=BLAST+: architecture and applications.;citation_pages=421-421;citation_volume=10;citation_year=2009;citation_pmid=20003500;citation_doi=10.1186/1471-2105-10-421" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=S. Capella-Gutierrez;citation_author=J. M. Silla-Martinez;citation_author=T. Gabaldon;citation_title=trimAl: a tool for automated alignment trimming in large-scale phylogenetic analyses.;citation_volume=25;citation_year=2009;citation_pmid=19505945;citation_doi=10.1093/bioinformatics/btp348" />
<meta name="citation_reference" content="citation_year=2018" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=H. Dempewolf;citation_author=G. Baute;citation_author=J. Anderson;citation_author=B. Kilian;citation_author=C. Smith;citation_title=Past and future use of wild relatives in crop breeding.;citation_pages=1070-1082;citation_volume=57;citation_year=2017;citation_doi=10.2135/cropsci2016.10.0885" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=R. C. Edgar;citation_title=MUSCLE: multiple sequence alignment with high accuracy and high throughput;citation_pages=1792-1797;citation_volume=32;citation_year=2004;citation_issue=5;citation_pmid=15034147;citation_doi=10.1093/nar/gkh340" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=D. M. Emms;citation_author=S. Kelly;citation_title=OrthoFinder: solving fundamental biases in whole genome comparisons dramatically improves orthogroup inference accuracy.;citation_pages=157;citation_volume=16;citation_year=2015;citation_pmid=26243257;citation_doi=10.1186/s13059-015-0721-2" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=A. J. Enright;citation_author=S. Van Dongen;citation_author=C. A. Ouzounis;citation_title=An efficient algorithm for large-scale detection of protein families;citation_pages=1575-1584;citation_volume=30;citation_year=2002;citation_issue=7;citation_pmid=11917018;citation_doi=10.1093/nar/30.7.1575" />
<meta name="citation_reference" content="citation_journal_title=Fungal Genet. Biol.;citation_author=C. Férandon;citation_author=J. Xu;citation_author=G. Barroso;citation_title=The 135 kbp mitochondrial genome of Agaricus bisporus is the largest known eukaryotic reservoir of group I introns and plasmid-related sequences.;citation_pages=85-91;citation_volume=55;citation_year=2013;citation_pmid=23428625;citation_doi=10.1016/j.fgb.2013.01.009" />
<meta name="citation_reference" content="citation_journal_title=Euphytica;citation_author=J. T. Fletcher;citation_title=Disease resistance in protected crops and mushrooms.;citation_pages=33-49;citation_volume=63;citation_year=1992;citation_doi=10.1007/BF00023910" />
<meta name="citation_reference" content="citation_journal_abbrev=Fungal Biol;citation_author=M. Foulongne-Oriol;citation_author=A. Rodier;citation_author=T. Rousseau;citation_author=M. Largeteau;citation_author=J. M. Savoie;citation_title=Quantitative genetics to dissect the fungal-fungal interaction between Lecanicillium verticillium and the white button mushroom Agaricus bisporus.;citation_pages=421-431;citation_volume=115;citation_year=2011;citation_issue=4-5;citation_pmid=21530924;citation_doi=10.1016/j.funbio.2011.02.017" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=D. E. Fouts;citation_author=L. Brinkac;citation_author=E. Beck;citation_author=J. Inman;citation_author=G. Sutton;citation_title=PanOCT: Automated clustering of orthologs using conserved gene neighborhood for pan-genomic analysis of bacterial strains and closely related species.;citation_volume=40;citation_year=2012;citation_pmid=22904089;citation_doi=10.1093/nar/gks757" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=W. Gao;citation_author=J. J. Baars;citation_author=O. Dolstra;citation_author=R. G. Visser;citation_author=A. S. Sonnenberg;citation_title=Genetic variation and combining ability analysis of bruising sensitivity in agaricus bisporus.;citation_pages=e76826;citation_volume=8;citation_year=2013;citation_pmid=24116171;citation_doi=10.1371/journal.pone.0076826" />
<meta name="citation_reference" content="citation_journal_title=Mycological research;citation_journal_abbrev=Mycol Res;citation_author=H. M. Grogan;citation_author=B. A. T. Adie;citation_author=R. H. Gaze;citation_author=M. P. Challen;citation_author=P. R. Mills;citation_title=Double-stranded RNA elements associated with the MVX disease of Agaricus bisporus.;citation_pages=147-154;citation_volume=107;citation_year=2003;citation_issue=Pt 2;citation_pmid=12747325;citation_doi=10.1017/S0953756203007202" />
<meta name="citation_reference" content="citation_journal_title=Commun. Integr. Biol.;citation_author=P. Grognet;citation_author=P. Silar;citation_title=Maintaining heterokaryosis in pseudo-homothallic fungi.;citation_pages=e994382;citation_volume=8;citation_year=2015;citation_doi=10.4161/19420889.2014.994382" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=T. Hackl;citation_author=R. Hedrich;citation_author=J. Schultz;citation_author=F. Förster;citation_title=Proovread: Large-scale high-accuracy PacBio correction through iterative short read consensus.;citation_volume=30;citation_year=2014;citation_pmid=25015988;citation_doi=10.1093/bioinformatics/btu392" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=D. T. Hoang;citation_author=O. Chernomor;citation_author=A. von Haeseler;citation_author=B. Q. Minh;citation_author=L. S. Vinh;citation_title=UFBoot2: Improving the Ultrafast Bootstrap Approximation.;citation_pages=518-522;citation_volume=35;citation_year=2018;citation_pmid=29077904;citation_doi=10.1093/molbev/msx281" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=K. J. Hoff;citation_author=S. Lange;citation_author=A. Lomsadze;citation_author=M. Borodovsky;citation_author=M. Stanke;citation_title=BRAKER1: Unsupervised RNA-Seq-based genome annotation with GeneMark-ET and AUGUSTUS.;citation_volume=32;citation_year=2016;citation_pmid=26559507;citation_doi=10.1093/bioinformatics/btv661" />
<meta name="citation_reference" content="citation_journal_title=Ann. Appl. Biol.;citation_author=N. W. Hussey;citation_author=I. J. Wyatt;citation_title=Insecticidal control of paedogenetic cecid larvae in mushroom beds.;citation_pages=347-351;citation_volume=48;citation_year=1960;citation_doi=10.1111/j.1744-7348.1960.tb03530.x" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=P. Jones;citation_author=D. Binns;citation_author=H.-Y. Chang;citation_author=M. Fraser;citation_author=W. Li;citation_title=InterProScan 5: genome-scale protein function classification.;citation_volume=30;citation_year=2014;citation_pmid=24451626;citation_doi=10.1093/bioinformatics/btu031" />
<meta name="citation_reference" content="citation_journal_title=Nat. Methods;citation_author=S. Kalyaanamoorthy;citation_author=B. Q. Minh;citation_author=T. K. F. Wong;citation_author=A. von Haeseler;citation_author=L. S. Jermiin;citation_title=ModelFinder: fast model selection for accurate phylogenetic estimates.;citation_pages=587-589;citation_volume=14;citation_year=2017;citation_pmid=28481363;citation_doi=10.1038/nmeth.4285" />
<meta name="citation_reference" content="citation_journal_title=J. Mol. Biol.;citation_author=M. Kanehisa;citation_author=Y. Sato;citation_author=K. Morishima;citation_title=BlastKOALA and GhostKOALA: KEGG Tools for Functional Characterization of Genome and Metagenome Sequences.;citation_pages=726-731;citation_volume=428;citation_year=2016;citation_pmid=26585406;citation_doi=10.1016/j.jmb.2015.11.006" />
<meta name="citation_reference" content="citation_year=2000" />
<meta name="citation_reference" content="citation_journal_title=Exp. Mycol.;citation_author=R. W. Kerrigan;citation_author=M. Imbernon;citation_author=P. Callac;citation_author=C. Billette;citation_author=J.-M. Olivier;citation_title=The Heterothallic Life Cycle of Agaricus bisporus var. burnettii and the Inheritance of Its Tetrasporic Trait.;citation_pages=193-210;citation_volume=18;citation_year=1994;citation_doi=10.1006/emyc.1994.1020" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=S. Koren;citation_author=B. P. Walenz;citation_author=K. Berlin;citation_author=J. R. Miller;citation_author=N. H. Bergman;citation_title=Canu: Scalable and accurate long-read assembly via adaptive κ-mer weighting and repeat separation.;citation_volume=27;citation_year=2017;citation_pmid=28298431;citation_doi=10.1101/gr.215087.116" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=K. Lagesen;citation_author=P. Hallin;citation_author=E. A. Rødland;citation_author=H.-H. Stærfeldt;citation_author=T. Rognes;citation_title=RNAmmer: consistent and rapid annotation of ribosomal RNA genes.;citation_volume=35;citation_year=2007;citation_pmid=17452365;citation_doi=10.1093/nar/gkm160" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=B. Langmead;citation_author=S. L. Salzberg;citation_title=Fast gapped-read alignment with Bowtie 2.;citation_pages=357-359;citation_volume=9;citation_year=2012;citation_issue=4;citation_pmid=22388286;citation_doi=10.1038/nmeth.1923" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=N. Lartillot;citation_author=T. Lepage;citation_author=S. Blanquart;citation_title=PhyloBayes 3: a Bayesian software package for phylogenetic reconstruction and molecular dating.;citation_volume=25;citation_year=2009;citation_pmid=19535536;citation_doi=10.1093/bioinformatics/btp368" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=I. Letunic;citation_author=P. Bork;citation_title=Interactive Tree Of Life (iTOL): An online tool for phylogenetic tree display and annotation.;citation_volume=23;citation_year=2007;citation_pmid=17050570;citation_doi=10.1093/bioinformatics/btl529" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=R. Durbin;citation_title=Fast and accurate long-read alignment with Burrows-Wheeler transform.;citation_volume=26;citation_year=2010;citation_pmid=20080505;citation_doi=10.1093/bioinformatics/btp698" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=V. Lombard;citation_author=H. Golaconda Ramulu;citation_author=E. Drula;citation_author=P. M. Coutinho;citation_author=B. Henrissat;citation_title=The carbohydrate-active enzymes database (CAZy) in 2013.;citation_volume=42;citation_year=2014;citation_pmid=24270786;citation_doi=10.1093/nar/gkt1178" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=T. M. Lowe;citation_author=P. P. Chan;citation_title=tRNAscan-SE On-line: integrating search and context for analysis of transfer RNA genes.;citation_volume=44;citation_year=2016;citation_pmid=27174935;citation_doi=10.1093/nar/gkw413" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=G. Marçais;citation_author=C. Kingsford;citation_title=A fast, lock-free approach for efficient parallel counting of occurrences of k-mers.;citation_volume=27;citation_year=2011;citation_pmid=21217122;citation_doi=10.1093/bioinformatics/btr011" />
<meta name="citation_reference" content="citation_journal_title=EMBnet.journal;citation_author=M. Martin;citation_title=Cutadapt removes adapter sequences from high-throughput sequencing reads;citation_pages=10;citation_volume=17;citation_year=2011;citation_pmid=25452271;citation_doi=10.14806/ej.17.1.200" />
<meta name="citation_reference" content="citation_journal_title=Mycologia;citation_journal_abbrev=Mycologia;citation_author=P. B. Matheny;citation_author=J. M. Curtis;citation_author=V. Hofstetter;citation_author=M. C. Aime;citation_author=J.-M. Moncalvo;citation_title=Major clades of Agaricales: a multilocus phylogenetic overview;citation_pages=982-995;citation_volume=98;citation_year=2006;citation_issue=6;citation_pmid=17486974;citation_doi=10.3852/mycologia.98.6.982" />
<meta name="citation_reference" content="citation_author=C. G. P. McCarthy;citation_author=D. A. Fitzpatrick;citation_year=2019;citation_doi=10.1099/mgen.0.000243" />
<meta name="citation_reference" content="citation_journal_title=Genes (Basel);citation_author=C. G. P. McCarthy;citation_author=D. A. Fitzpatrick;citation_title=Pangloss: A Tool for Pan-Genome Analysis of Microbial Eukaryotes.;citation_pages=521;citation_volume=10;citation_year=2019;citation_doi=10.3390/genes10070521" />
<meta name="citation_reference" content="citation_journal_title=Mycologia;citation_author=R. E. Miller;citation_title=Evidence of Sexuality in the Cultivated Mushroom, Agaricus bisporus.;citation_pages=630-634;citation_volume=63;citation_year=1971;citation_doi=10.1080/00275514.1971.12019142" />
<meta name="citation_reference" content="citation_journal_title=Fungal genetics and biology :  FG &amp; B;citation_journal_abbrev=Fungal Genet Biol;citation_author=F. Moquet;citation_author=C. Desmerger;citation_author=M. Mamoun;citation_author=M. Ramos-Guedes-Lafargue;citation_author=J. M. Olivier;citation_title=A quantitative trait locus of Agaricus bisporus resistance to Pseudomonas tolaasii is closely linked to natural cap color.;citation_pages=34-42;citation_volume=28;citation_year=1999;citation_issue=1;citation_pmid=10512670;citation_doi=10.1006/fgbi.1999.1157" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=E. Morin;citation_author=A. Kohler;citation_author=A. R. Baker;citation_author=M. Foulongne-Oriol;citation_author=V. Lombard;citation_title=Genome sequence of the button mushroom Agaricus bisporus reveals mechanisms governing adaptation to a humic-rich ecological niche.;citation_volume=109;citation_year=2012;citation_pmid=23045686;citation_doi=10.1073/pnas.1206847109" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=L. T. Nguyen;citation_author=H. A. Schmidt;citation_author=A. Von Haeseler;citation_author=B. Q. Minh;citation_title=IQ-TREE: A fast and effective stochastic algorithm for estimating maximum-likelihood phylogenies.;citation_volume=32;citation_year=2015;citation_pmid=25371430;citation_doi=10.1093/molbev/msu300" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=B. P. S. Nieuwenhuis;citation_author=S. Billiard;citation_author=S. Vuilleumier;citation_author=E. Petit;citation_author=M. E. Hood;citation_title=Evolution of uni- and bifactorial sexual compatibility systems in fungi.;citation_pages=445-455;citation_volume=111;citation_year=2013;citation_pmid=23838688;citation_doi=10.1038/hdy.2013.67" />
<meta name="citation_reference" content="citation_journal_title=Can. J. Plant Pathol.;citation_author=L. H. North;citation_author=P. J. Wuest;citation_title=The infection process and symptom expression of verticillium disease of agaricus bisporus.;citation_pages=74-80;citation_volume=15;citation_year=1993;citation_doi=10.1080/07060669309500829" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res;citation_author=H. Ogata;citation_author=S. Goto;citation_author=K. Sato;citation_author=W. Fujibuchi;citation_author=H. Bono;citation_title=KEGG: Kyoto Encyclopedia of Genes and Genomes.;citation_volume=27;citation_year=1999;citation_pmid=9847135;citation_doi=10.1093/nar/27.1.29" />
<meta name="citation_reference" content="citation_title=State of the World’s Fungi. Report. Royal Botanic Gardens, Kew;citation_title=Useful fungi;citation_pages=24-31;citation_year=2018" />
<meta name="citation_reference" content="citation_journal_title=Mycologia;citation_author=C. A. Raper;citation_author=J. R. Raper;citation_author=R. E. Miller;citation_title=Genetic Analysis of the Life Cycle of Agaricus bisporus.;citation_pages=1088-1117;citation_volume=64;citation_year=1972;citation_doi=10.2307/3758075" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=D. A. Rasko;citation_author=G. S. A. Myers;citation_author=J. Ravel;citation_title=Visualization of comparative genomic analyses by BLAST score ratio.;citation_pages=2-2;citation_volume=6;citation_year=2005;citation_pmid=15634352;citation_doi=10.1186/1471-2105-6-2" />
<meta name="citation_reference" content="citation_journal_title=bioRxiv;citation_author=M. Roach;citation_author=S. Schmidt;citation_author=A. Borneman;citation_title=Purge Haplotigs: Synteny Reduction for Third-gen Diploid Genome Assemblies.;citation_pages=460;citation_volume=19;citation_year=2018;citation_doi=10.1186/s12859-018-2485-7" />
<meta name="citation_reference" content="citation_journal_title=Mycologia;citation_author=D. J. Royse;citation_author=B. May;citation_title=Genetic Relatèdness and Its Application in Selective Breeding of Agaricus brunnescens.;citation_pages=569-575;citation_volume=74;citation_year=1982;citation_doi=10.2307/3792744" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=F. A. Simão;citation_author=R. M. Waterhouse;citation_author=P. Ioannidis;citation_author=E. V. Kriventseva;citation_author=E. M. Zdobnov;citation_title=BUSCO: Assessing genome assembly and annotation completeness with single-copy orthologs.;citation_volume=31;citation_year=2015;citation_pmid=26059717;citation_doi=10.1093/bioinformatics/btv351" />
<meta name="citation_reference" content="citation_journal_title=FEMS Microbiol. Rev.;citation_author=C. Soler-Rivas;citation_title=Biochemical and physiological aspects of brown blotch disease of Agaricus bisporus.;citation_volume=23;citation_year=1999;citation_pmid=10525168;citation_doi=10.1111/j.1574-6976.1999.tb00415.x" />
<meta name="citation_reference" content="citation_journal_title=Fungal Genet. Biol.;citation_author=A. S. M. Sonnenberg;citation_author=W. Gao;citation_author=B. Lavrijssen;citation_author=P. Hendrickx;citation_author=N. Sedaghat-Tellgerd;citation_title=A detailed analysis of the recombination landscape of the button mushroom Agaricus bisporus var. bisporus.;citation_pages=35-45;citation_volume=93;citation_year=2016;citation_pmid=27288752;citation_doi=10.1016/j.fgb.2016.06.001" />
<meta name="citation_reference" content="citation_journal_title=Proceedings / . International Conference on Intelligent Systems for Molecular Biology ; ISMB. International Conference on Intelligent Systems for Molecular Biology;citation_journal_abbrev=Proc Int Conf Intell Syst Mol Biol;citation_author=E. L. Sonnhammer;citation_author=G. von Heijne;citation_author=A. Krogh;citation_title=A hidden Markov model for predicting transmembrane helices in protein sequences.;citation_pages=175-182;citation_volume=6;citation_year=1998;citation_pmid=9783223" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=M. Stanke;citation_author=O. Schöffmann;citation_author=B. Morgenstern;citation_author=S. Waack;citation_title=Gene prediction in eukaryotes with a generalized hidden Markov model that uses hints from external sources.;citation_pages=62-62;citation_volume=7;citation_year=2006;citation_pmid=16469098;citation_doi=10.1186/1471-2105-7-62" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=G. W. Vurture;citation_author=F. J. Sedlazeck;citation_author=M. Nattestad;citation_author=C. J. Underwood;citation_author=H. Fang;citation_title=GenomeScope: fast reference-free genome profiling from short reads.;citation_pages=2202-2204;citation_volume=33;citation_year=2017;citation_doi=10.1093/bioinformatics/btx153" />
<meta name="citation_reference" content="citation_journal_title=IMA Fungus;citation_author=A. M. Wilson;citation_author=P. M. Wilken;citation_author=M. A. van der Nest;citation_author=E. T. Steenkamp;citation_author=M. J. Wingfield;citation_title=Homothallism: an umbrella term for describing diverse sexual behaviours.;citation_pages=207-214;citation_volume=6;citation_year=2015;citation_pmid=26203424;citation_doi=10.5598/imafungus.2015.06.01.13" />
<meta name="citation_reference" content="citation_journal_title=Molecular Biology and Evolution;citation_journal_abbrev=Mol Biol Evol;citation_author=Z. Yang;citation_author=R. Nielsen;citation_title=Estimating Synonymous and Nonsynonymous Substitution Rates Under Realistic Evolutionary Models;citation_pages=32-43;citation_volume=17;citation_year=2000;citation_issue=1;citation_pmid=10666704;citation_doi=10.1093/oxfordjournals.molbev.a026236" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=H. Zhang;citation_author=T. Yohe;citation_author=L. Huang;citation_author=S. Entwistle;citation_author=P. Wu;citation_title=DbCAN2: A meta server for automated carbohydrate-active enzyme annotation.;citation_pages=W95-W101;citation_volume=46;citation_year=2018;citation_pmid=29771380;citation_doi=10.1093/nar/gky418" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Whole Genome Sequence of the Commercially Relevant Mushroom Strain Agaricus bisporus var. bisporus ARP23" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10.cover-source.jpg" />
<meta name="twitter:description" content="Agaricus bisporus is an extensively cultivated edible mushroom. Demand for cultivation is continuously growing and difficulties associated with breeding programs now means strains are effectively considered monoculture. While commercial growing practices are highly efficient and tightly controlled, the over-use of a single strain has led to a variety of disease outbreaks from a range of pathogens including bacteria, fungi and viruses. To address this, the Agaricus Resource Program (ARP) was set up to collect wild isolates from diverse geographical locations through a bounty-driven scheme to create a repository of wild Agaricus germplasm. One of the strains collected, Agaricus bisporus var. bisporus ARP23, has been crossed extensively with white commercial varieties leading to the generation of a novel hybrid with a dark brown pileus commonly referred to as ‘Heirloom’. Heirloom has been successfully implemented into commercial mushroom cultivation. In this study the whole genome of Agaricus bisporus var. bisporus ARP23 was sequenced and assembled with Illumina and PacBio sequencing technology. The final genome was found to be 33.49 Mb in length and have significant levels of synteny to other sequenced Agaricus bisporus strains. Overall, 13,030 putative protein coding genes were located and annotated. Relative to the other A. bisporus genomes that are currently available, Agaricus bisporus var. bisporus ARP23 is the largest A. bisporus strain in terms of gene number and genetic content sequenced to date. Comparative genomic analysis shows that the A. bisporus mating loci in unifactorial and unsurprisingly highly conserved between strains. The lignocellulolytic gene content of all A. bisporus strains compared is also very similar. Our results show that the pangenome structure of A. bisporus is quite diverse with between 60–70% of the total protein coding genes per strain considered as being orthologous and syntenically conserved. These analyses and the genome sequence described herein are the starting point for more detailed molecular analyses into the growth and phenotypical responses of Agaricus bisporus var. bisporus ARP23 when challenged with economically important mycoviruses." />
<meta name="og-title" property="og:title" content="Whole Genome Sequence of the Commercially Relevant Mushroom Strain Agaricus bisporus var. bisporus ARP23" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/9/10/3057" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Agaricus bisporus is an extensively cultivated edible mushroom. Demand for cultivation is continuously growing and difficulties associated with breeding programs now means strains are effectively considered monoculture. While commercial growing practices are highly efficient and tightly controlled, the over-use of a single strain has led to a variety of disease outbreaks from a range of pathogens including bacteria, fungi and viruses. To address this, the Agaricus Resource Program (ARP) was set up to collect wild isolates from diverse geographical locations through a bounty-driven scheme to create a repository of wild Agaricus germplasm. One of the strains collected, Agaricus bisporus var. bisporus ARP23, has been crossed extensively with white commercial varieties leading to the generation of a novel hybrid with a dark brown pileus commonly referred to as ‘Heirloom’. Heirloom has been successfully implemented into commercial mushroom cultivation. In this study the whole genome of Agaricus bisporus var. bisporus ARP23 was sequenced and assembled with Illumina and PacBio sequencing technology. The final genome was found to be 33.49 Mb in length and have significant levels of synteny to other sequenced Agaricus bisporus strains. Overall, 13,030 putative protein coding genes were located and annotated. Relative to the other A. bisporus genomes that are currently available, Agaricus bisporus var. bisporus ARP23 is the largest A. bisporus strain in terms of gene number and genetic content sequenced to date. Comparative genomic analysis shows that the A. bisporus mating loci in unifactorial and unsurprisingly highly conserved between strains. The lignocellulolytic gene content of all A. bisporus strains compared is also very similar. Our results show that the pangenome structure of A. bisporus is quite diverse with between 60–70% of the total protein coding genes per strain considered as being orthologous and syntenically conserved. These analyses and the genome sequence described herein are the starting point for more detailed molecular analyses into the growth and phenotypical responses of Agaricus bisporus var. bisporus ARP23 when challenged with economically important mycoviruses." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10.cover-source.jpg" />
    <title>Whole Genome Sequence of the Commercially Relevant Mushroom Strain Agaricus bisporus var. bisporus ARP23 | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"pWPxbkCdhXhpzRJITVbf_wRlzSVIIIA3W99HVLyTg1Q","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"494295","apath":"\/ggg\/9\/10\/3057.atom","pisa":"ggg;9\/10\/3057","ac":{"\/ggg\/9\/10\/3057.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/9\/10\/3057.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;9\/10\/3057"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;9\/10\/3057"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/9\/10\/3057":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022ppy2uZr7Ez6zLMMQYPYDCeZn4GFToG7WmlSnXjP2MAqFiuVugc6tGDKh4O7uIQOOT9lJrMd4RGG5kVe5glsNJ6ZeoRPZWQrN4YmXaG+5ILCyP30FpkYmtyKdOZGbtRqmNo4mI7aHbbzzyQfdTO7lNOeG0nHftxEI3s1i\\\\\\\/L+XqQquWnDhWj4qt2mp3YTzy\\\\\\\/\\\\\\\/K9wXyOu6whikn2lz1dq7cElynGdWltbHJzZczJngIu6Ey10uyAjgYbz2Ajb63q3+Fjm\\\\\\\/rZHq3VjqbsxjlYGP+rgah3\\\\\\\/GPmy0daGL2IBSsHrSo0GSdyiLjexkQC7aiBuGqht4Uon4wMlCvm636b4BxztcHSQpiqcvwlgMYs2vxX5QfjV7MPI\\\\\\\/BoXCKMgyHU1WUpV76xXhAzIfETfQirK\\\\\\\/qrupEvPDQZqIY0BGO1SF87q7qYhDOwpjk0tHwfYoXZcoFFp2runz03vDGAw4dGofTeFPQ0QCXz7e5r0HE9kDOzGseQNFjBGZ57mkOUfjXt1JQ1pP1sxQH5Aj0rWUeZ2BnuYd1ET9lKRdiQVuBwI5HTA5WhfZCJr5Fg2\\\\\\\/V\\\\\\\/WilISy9sMlZbYtvlQr6QmP3STpC64yw77EKjpc8mjzum4fGK40AzWysByiZfEVxC5i7GCH1kubIGw6pPeSkdh9mbr4snjPXA17wduBFPVBxMNeBnWeVfyHbq3YaKkRhOTAEYZ0L8qZHZdhAG03CvHskZ8Rgymzg1KUAoU0lYI5KEqpsmatSfg6rfsgbqVmtkMnoxd4Jo4aVBJmF8HNPlT\\\\\\\/9TP1yOYdLqmJXk35C5clQQ7lW\\\\\\\/m0zaRN4UAg7weqQs10JBeJ3lswSBlMdA18I1oKIBSUuglyBGByzA6dSpXqBl1YcF+6Xkp81shUERbgBjWmRpok1H2eRcdZ+phnDRQBJexBAx9SBI0DHhE96v2b6y4a1MmjY3NIXyguNdZaUu7lEBMBWgVdbVsL+J6u4vvKB6CDA2R7SGZLcSyLfUQliopg5kEpUqYkn1h5M5Ar5edbjBkjk87fkDbWY2u2fw3W8QTamswQ46tVbjn5ZnJ8ZzsX9lM+deZ7Z26R+HUaRFX88ciZDmyRDjzBflsoq0Lb8ZGGHS3sdu4p1Hcu2v6onbbqrE09DYnaYGIeQs89lvDYkQjvcdJZmRjNADi0QuaTCB+im\\\\\\\/o9fLu\\\\\\\/gEEKVja\\\\\\\/Fh2phN1OMsNzXQaFSw2D8EcxsuFi8\\\\\\\/N\\\\\\\/2xgarM9yRJJ5\\\\\\\/kWWyQjaHkCrh9dEhoFJ5DDUQDP4rJpHe7GmXZ3l7V+dMkl5n\\\u0022,\\\u0022iv\\\u0022:\\\u0022za7vcmcmOunpKzECNyuMKQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00226bae9f92c2de8838528d9d1c0312bc99b3efc7a810ae0eaa13e822cbb3c8a405\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022Hyc+Ui+n2FzJ40wRk0jmOLBpQ5lmyrqP6ANUGoExD3s9eNXDmcByUS0o\\\\\\\/Tnc96ueZ6dYDYXMhDH44s56XJVC66TEi8hkX4HgjF5Mx4DiSCoX4DBGjGYdxxqvumOYtFjn1jvDhmdPQ5VHvP7nQFrgPYrKkb1yywG0UCqfLBlTcxqqBIeWDITL1jrSbKHPUV0jvhdUjJptZKjorL9nZCSGDEskFHlYvUtqy7lNrVzniEEQQhvMvhU3TSt0vH4x64qSaBfqPl5iTf3yN+mjuNlNsBLl2yGQaG9xDyg3qlCNYJcEU+IoLy+lDVmCYv2gVinUH51qE6mqx6RHgxCcpAZfAimNWX3cBUKoUEJJnICgiTLEfFFbszvdV3C46b23k4N1qcaoZR7CRrZfbq79\\\\\\\/3Fg+PuGNkrBQOvD2Bz4JXtD7Hui5VPhKQUggIKcid7uSnGqHnKs76mL1QPJVTiJttMClBDn3Bpl\\\\\\\/tIJCX4qNhpHnKNRXiFUmPk6cf\\\\\\\/XbHdGTeAkSNa3C1MNw2\\\\\\\/IcYon7VcFwBzQUPKOMPf0yFkJNLHVaSBrrHhjfWi6dw1sDtUawMM\\\\\\\/woNlS3TT+nt9ZbzXW\\\\\\\/cdpTpRjcb6JI1S6rsqic+2TVY+C6EYERCRyXms87+8EbBKAT03prk8rlfi7TL7VwQFEbC2EYNH+7JmHe1QtkCsfNQ32YOmmPlPe1Vfy3p2mwRLDiP0QJc\\\\\\\/rU5iXw+D5pUjBDXtxwjwpixZdwj3hOZAr1YjD6l4NdcdxLtIYtn\\\\\\\/DF1A7S9qgKZeoWKjoLD8jE7bqcQDSj8skXhj7NwW75W57DlnEM9LBi2Hhg4IqqcJA4la0YJ8RsqcQFyWsMi3LOwP04FFJ80pJGzGIM\\\\\\\/wxO4raqQXkJhXJpMBLvrwFFlBIXmsNWq6\\\\\\\/IMRT+1RAb4\\\\\\\/YXDQSeJs2I0Z0B3BiR0g\\\\\\\/I\\\\\\\/WSOBIVhbu9lSL3BfSvgAB3Rp+6LoqX1Z6JRM7jCcaHdFChxSi\\\\\\\/kBckAl7BbdMlidhZCr5LMURsVsV1VxlSZErTVZWgZcjpQ\\\\\\\/bRh6SWRl1PC+jDtM9OATDH1q9nJTU38CjNx7dlAJ0KtRpBYadgHkuk9tA7283wlqDw2tb6mgmURLkal3OhhWwS\\\\\\\/YqD2avGQ5aZXEzvah60BnnpS2CNPu+keEcYuqlU7QjugcqFUwmCtwX\\\\\\\/ItGuqo\\\\\\\/X\\\\\\\/5DHhv96Ofn602kr72IvJE0vl53ZhWUL5eyz2za4\\\\\\\/0sPnCRQ414aU\\\\\\\/8PTaB6Uvi8r7SKi9Q2YYowXZqX9A\\\\\\\/CFSgqQI8HlQKFde9hLFDThLnzSf\\\\\\\/AI0DpLFNMn6hpC7\\\\\\\/\\\\\\\/Kibb5OdQkqSvhBJnZ+MbgN0rFQSkk0Z2977+TeIcXx0EmKCVYkKN8\\\\\\\/+V2ESJ+mJJXzFHZhP6BxEOx83wwfJUk80Ar9+Kz7yp4KJRDkHW\\\\\\\/RRRKNrQyf\\\\\\\/p8c+6F2xbZDNvsVKrx77zf06FVVQt93o8tVL\\\\\\\/kHjsjoAcl0rc+i+RJmwygMIOuULu5gmDshLrPfdNwXIR4Vois8wqbBUqqLUh8brf1gJI8ac+llO\\\\\\\/ZUxBYKnwciBRK8SOQkVuf6pyn2i63QrSZf+GBLclxGKutqGwGkMSvQM\\\\\\\/e9XEVSHSk3kBexnAMJdRuSnn32yiaLkGsyRKpboXyqho0TWMx9Vd5JQOaOqaC+BJESKlh8ltdg6ejYIBRlDMHvWJYEelaeA+68+Nu89vDIGFlORh1wcu9dUwJGq5TXelOAydZttJrsQdto9m5WfsaBivIQmNC14F1tQeGq5HkUOYswY+Cmu7KrLKs2d5RlbU1zEOpA2I+KNvCqm9Y0WkKgU03weXfcv+9z91Qytw+InoqXhr9+TXrU2R2N3Oh8r0SyD9M2r3p695Vf7uqDekiCHom6eYQcynxgLEe80mud+16TXud9m7YlLZyQW1uNoWSt21aah+D\\\\\\\/U5BWVBCOnRx0DRBGySZilhKsNeeZ+NGEMfcPY43aXfzmRTYVuRigwu2SGQOO6wsGKP4GyA+6pQAewurwD0U3DGrUlrAE2utUfx5buWCtl6C8A+0HYoJed7z7VIzZ27DHfjBqk3FD1jehxHi1nvNQUV1UWyBQlCT9TlrzHR7FG1+uyMUWtsXx5wkorfbw9x2\\\\\\\/heDuSX4WLipSq4wyuUryTaNwyJDQyuEk7vyBBI\\\\\\\/m5nK0hvZ1PkFmsAuKt0HGU47WDWSZ\\\\\\\/4FUoX5iFNPZN+QSIuPjXHRQiDc7Wtfj8xHTnPqSYgnhCGhsL5RDW0wmVwzuYEMu0NTWpVkjwhMS0x4kloFfwynoTbYi8KRr+n3PTV5TM+ZUnEXGMYsGWmq\\\\\\\/qr67FMn672V1MXGGZ962xjmp+xuFsb\\\\\\\/Yl38HxPxlnFyv5e+w+a5tSGR9IH97ZgaaunOGdt+ykGIzt3JrJvzZ6P7oX4vXfWULkc1aLUHNHHKXP\\\\\\\/drI5f3FEfOeuY3hm2XPNtVdrwV2vBSDLap014Lh8mGhp2P8wbXTuP0gmfvafUlKVFWWwcO\\\\\\\/Kfc+YfYDwA7L+qB\\\\\\\/D3OWfUUF29EzsmdKveoMLTu3LXjgO1mfEROJBIA92kWcJ9xiyfz+S6FAsvRIsr0yrzaMIchgn2cU5baBSegfAObYCfZTcJb5mGODbefDF2mNi4UZS+etwPQQF7jA0px1WbA9Sq32hJgKsK+kBr2grT4\\\\\\\/1XUpFmrVIKA8PLWR0Z\\\\\\\/Vf6pi8WHDTTtOrs7+oU\\\\\\\/HQR7Pkplrfa1Dz\\\\\\\/4gCAnuGcziV21rB\\\\\\\/iD4pjSj+mxbImzyOtTWrUNwXNWhMO1y\\\\\\\/FWC1II0AzqekmtAanlbABIEHlSUfgmJEM+3amxG5t7Dq2WjHO+3Zkb8SJUWQ75kb9Qh5nfevJP45PPhJjpip93pwWCIwIkOvNgpcf0XxUPKBcitLSqsJUtQZqpc0n7Np8BkAjANamOFx9LGJ33TyR9LVrndyXYm65ZfSOHtHxjpFtG4+B\\\\\\\/i2IUwjRaXk3bGqOTHuKvoigxXYIshFZmJ\\\\\\\/RhGgMxK+s6Fmktwa5SQD16egp4i1X0rmkjxwtgOYEIxLiZaltYkop6ctnxsqlHilNEr2TkVPE4a+9I0jP0B1dqaQnG6T9L18CBZBjl5ySW0fOYJeV95\\\\\\\/SU3YZ5CzWLWDeS2w909UKv\\\\\\\/URWV9sCAm4FZRbGbpRvzCFx70vs2i8MQna41sQbNYIfibzSakvdZEbmNDx64ZF6AhuzJpCmmWJwu+XxuoerV\\\\\\\/8stnmoIkdNAWW1z9YBeeWG+MVweIvbbTb71ClRBq\\\\\\\/Ktiz7OWNUR\\\\\\\/MOj4X+2tpQTOUZGO5WzwzJa4QFVKxvkZXiX21NdqqEJ8GesZ4sgfjatoGUSIyb8\\\\\\\/HvEMafCpBm6X4ywb\\\\\\\/ie55ebAf6BqFiTieEgXche4QLJNoEAanAd1oFc4a97pEsmJLNSEUAscUaCGSkiCq80vLDhaMSMrP6trA0dNWFALxEKjH8A1d\\\\\\\/nvkWcv30L6HoXmurG795hraV+fqbs25PPnTa2n8JCnCHy6200zIv2HJ+3b1i5H9QmU1gJs2aTNq5SFV5pCQBIyUResM2SiAmFDskb6mf4g5LILsLXfvTocFDSObXPghMEh28DwIZwEM+HFBKbG5UKRfw3iwHO0+jeE\\\\\\\/UvwkA\\\\\\\/eiMlGQP3eNy0c40f4PM8GVDRMNqH2JwTv47Vc0rnKWnVVkHAKusS7oo2KdWbGoyhDZMrzNsXNoKmGiIqaQWzDyH1iyeMmpTTyI6jpdOhXaSW5B\\\\\\\/PYEB+3cPOLDk7GcN1pbj7esdQ83WZeJ0cKIffm+ekyyGGDtzi0e88WONgBie17MvFJV6klqt7vbiiFCCIIOEoPgYF4czBlMy+QZomHk6uE54h+TyqEEaWHr3Hf7fqZ8DaYKybdmkG0VX\\\\\\\/Zt5hq3QBZ4dWH+xYEzcEOt9n0VprBkYzacu33PhBFnh\\\\\\\/S\\\\\\\/L2OGwgljbDYmNRtWTrHncK\\\\\\\/l5QcIfJM8\\\\\\\/ajP2mEekhwq4c93tZfLu+KtoRsa6zoh8bjjYX59cOJsYKn9LdlPHwrhf4mcF7YhsDnQmcB6AYXofOACtE5n\\\\\\\/nSjnBrP1om+uvYwfYgPnoyNO7\\\\\\\/bOuD39W1sC+O1T20J2DiOohALx+tOeBRWEDGvWq46rdIIQvI2eRAF0w5l4T4aaadKFbmiuDzaDQZv+uaynfYimdRnXwpSmxlRNFBvri+f+LOjlVue8hncpnaG+BilnNhLiue+gX2NlAZfCBrbOQ+qREC76IGzx7NB\\\\\\\/6uUCPmjvz8jD24MWes0m\\\\\\\/hH\\\\\\\/lFcIlVkGtVfGQ4SFhKTnZquZG6k9DaV1CEwJldSpCmUhidng+jhc+GXlW9HlfaYtCvKrMPCFWkIlnNBQH+8CRxByR2TfKtP\\\\\\\/NtncNMgZeL569T8pfxWVSxUQrmluOILAp5dzHD021z86yEhx9nSKBwvxDTrBWCHyIIHc3bk5rd6F+wUhVUAd41bB6Vca5zH4jHbs\\\\\\\/TN3bslz723ogXGTHdaGrTMhyao8msavSq+e6+Z23f6gd7wRlni\\\\\\\/85CTpoJGbxCawGZGWk\\\\\\\/LGRljAc9+Zx0EkXvNLVpwcEN1qHlxr97umdzyEYriTjIGFpCy7zd7BldG1x92Yoia4bhjwMfQnRMsqn7jgVfvahxN1MOF1vv4eac7qihgfTWfu2H68qciBKKyp8vUMRm5SbKV4ZRsLwITdQGav+r9MDAP6sxoV9eKrrRl5wTImhtQoN2j8I7RHnXK9jMCMPIW174X\\\\\\\/PZY762YG9EKQAcrdGOUaRGbhdjx8icmy5ZlBwToRTjpnL3vYHhJgSb1uHwcvxnp1qsImkX\\\\\\\/xX4mmivSw79Ac3QSPHUfsnwZPfyOVsHbiRxQ8uJWqQ3B0SbfXAUrPuaifFNHBvHOEmcYGs8g03gcCuJV9qBF7UBS5gtf0aAyu0L3MOkvM22N28791gqYVyzGALNOPLQnCGakA360SV98sB4u\\\\\\\/AF0gRmZM\\\\\\\/SRob\\\\\\\/7bHSPdgTwfZZZg8gh2wV1nXRksQlmfRTcYwkMwjdKkJdROB3\\\\\\\/AWiK+7CsydJqHwdJPj7sh0ywaaPm8N44HQbPsYhFfAgJuFlpB1BqrjPRhwVNQa6tudwgEmuSJirNagwF86bXAMnqpVXfi386HpWzIWUmh0X2RmwcRwHo7Pf6b2I2qQjwvMuFNXkfAOpws0HFFCaRVKt9oCxde7FgxRXVw5OXfWtueNpIa0nmpRfrpkqq2q3P1DTGfxg\\\\\\\/DN6czQ3hC+yQXsDZ9FVMxa\\\\\\\/zcYx3UVJCb4PrIztVvpqfznPhTggLOfDWan5Th299TPUqah\\\\\\\/BVUQvOq\\\\\\\/ngSnpcnBFjLxxdOAM\\\\\\\/XgisrYqVgWX\\\\\\\/PQxE8r294WUpyR7tLaqjOEyhqMftRBGxWqbv3YHwbtaZ1vszDGccCM3xTPDoebsrgktkClPdL5+SmjaX6TLnd1sKaotKh6n9c8YlJsK+ee1JvuAoS10FekQcvZjtcJaH7IiuyFaakgu9axQN7s532JtzF90DxXi6JABQlGrNevMz9IWi+N8T+Erj\\\\\\\/zyJxJ8MadJ8yT+8MERhmiBuwRfK0ffAsKrMKK91C\\\\\\\/bDCZHou9DvwzVELUplWqH9yBOooOUrP\\\\\\\/pss5AGHqA8+D\\\\\\\/G\\\\\\\/D5jNPXFFoNDDomHuAuex0D5koSt0CVST3dYZKrOUsRQYjjewa67Q8\\\\\\\/B9lWkAEYLiqfsxfpQoqVBj1Pu\\\\\\\/cnMgrRln3PEa6xHm2Hv79ICYURdee5yCnR1yrWVNdXMlzC3NlexiPwP1R7bI\\\\\\\/9POoP09tcv5gZViDedMrXKYgCKjnXOKQaW4j1w8dd0zfdkhzZQbTT0Nk0S\\\\\\\/SKR1FYc6N2Oo+ICOxJZheBB9kjGfDQEcK0l6MuEKZ8Ld\\\\\\\/m9m7u+0DnpNZLSn\\\\\\\/QCNDDkh2N47SP\\\\\\\/4Uq+C8WXv5e0TjHb4FEmDR52MF+OrhQWUpvIlnS0mB3TQz0JdBL6Qx4NgceqKzZt5KLoifP9zCHNUzgnJyKFBegPh+Eqkxl7hybqkC7CfpUvjceN2ltgCdrw4V5lJCg0zt1zw8xEpDDHvWsXQlWRz+HgE1fpbbP93hrZq1eeFvJArK6twz6UFnNof9K26A+x7dZ1JfeOFf1ofEAZoINXFnQlUnv\\\\\\\/EjbSupNoV8EyDE9SdRh7U4qddzzUxgyIs8QXbdcNRi55NMapUxNp7pfhUll6TkuSXzsxkwG9ASlvO6VrYITauIncxwEPOaR3c0SD5D96F9Z5GQUqlHelkt62xwUlvO8Agv08XxAeKMtwkE4tsExXsr860EW8VtfLxdknpr\\\\\\\/j6CbQDPyL1KXmJYa\\\\\\\/fjwNDVV7U7fX033SFO6YFJuAg+NkpKBL7G5wzwYwcFFURdFBjYTWgK+ZOj4MUsiXrbe+I09CEs9iOcBqrRtmk19u+OMp3eV52nrXOjGm8rGqPb1mPm7joiqeVHiJoEBxkW4pyni2HGzjyF9WIfqtZMkXZycHs7dXtaXEPkJE9k1Dhao5JyXsyP9xmNR2g2gYBqxN8ocaeoRqvEF4LV6+lQWWbg2\\\\\\\/SsQRclEliR0Goj1hCdJ0ugkh0TsVdVxN5m4Ul6WL1HVHImqf74ylj\\\\\\\/qJmryLR+hdPDZ3xPlwYSxqQIQTg2qdL4+5mvHqbm+YWNO1R\\\\\\\/30pYFpdUOntEmEpCG\\\\\\\/gVQEQJ+HCbwZP1T+fqJ0J1f2Fw\\\\\\\/c1v2cbv+SywQSLwb9yhFzCqDKrYTSYAty8\\\\\\\/aHiGEFoih86n4V99Dh8dBaT+Ygk5U4FLN2ZQAFk8fIi24wGTHkSl2ddMeshB26LEXZ4wRdRTTbBWM0BB1i3FFUfK\\\\\\\/m52v65ERXCyHSzHWYRAXbo\\\\\\\/eSiPxbG\\\\\\\/bP3VC6IUaeqImmYMcVM7WtmIUcHQwConO8\\\\\\\/cOPBcayGgRuZNJ7P7UC2VVHlMBpUfNd2+reeOwENvpvyRxujYgt+DnMr3pAjb7bowvmoA5IDGBwHWa6Fz3u2Ia0mDNsZlA+UfFGh3PbIlB\\\\\\\/TLprWyEJGnDo0dDLymnT1bWe9wibmTPxLjFYOwMr9lISx8sjR7qDJoGfNcJK\\\\\\\/5SsL9ZDRdwKC9L2N0847RjAE1OukD69RdrM7p\\\\\\\/+3maR+zLIDynx4jlLgYhdeaQdzThLuBHPbcnM73q9v+y5CB08u3CUW5Ww0RuOBpoZQ+w7hrTZIAoomNLVA8iynmmVg4m5S1I9LmSJe6Oijrmjlz0lTDdKQ\\\\\\\/cDuuMcYZgOxBR87K2xVX+bqOsrVxaPQHQ1X6v6MT6j5H\\\\\\\/WepqTKNExOo3pbKypOdcfhyzKLwufgI0JdT5N5MrIPcLEbIN5Z039CO9f0c0htbh8Uk9U5QUrPqTiLpUCVAqTkyD+vlqIp+RZ4MJ+8oLiORiI5wGO2Pseqi6lWd0w0AKAnhDUF5KfxS0fVqcuIGZiYi7HyE17nQR\\\\\\\/Q5xjxfqOAP323G9tE8529oPi5oV0uDVRgu5qb+mI62DuIwkJ+7QY0rNEHdtzqp3cH08RnU754Wtxl18vO4OuO+ippYaA\\\\\\\/qeXkA1BKDjhfxziDT5wUA\\\\\\\/bHemiRvlJBybPrWGag8\\\\\\\/plzDJkeVX3Xea7DAxTGdFW15tbXtlIEVI4G82Eoddy5bUQ9nEsdjkJ4PwZGFI4mj2GnTcHdZyTBlNA7RqSwCygWpXfZ3wIaF2vDo84jVVQ66UUnTHR7Jv3rXbRjeMCxk1n+G22us6UH7nI13aXwfvLE4HQiI5bCazRHuOuJHwUUIavmnN39rqcxU8HZvVpklmR4eObhtfXhlrhk1EdcOdBq83luVTCTgVqgcNNo21jgAMMKgffuSwcmHcSsZtvqQICvroM+lwUJvHZ5MzGTq6sc+f1+BhWKT3jSKIRWhwcl+\\\\\\\/WLSA+M5W7\\\\\\\/WC0nwwP7zCClyC61rGSNzAGoH2LGAFeYcDgQgS7Tdk4o0tMcrbsDagwkZlDwcqCC8R7aVejspWUB9vOQZwBIA3H8Hx9q\\\\\\\/aRFW3I0OguyaB4XOFU+MrWDx3MsF7tJRiZn3+37VGWZitdV8cw6YYlu67pq+DM0eTGl7MYFxMJ8zMscXQvgxgrONbWo9NM7dTru49EyoB\\\\\\\/Q9+U9VflfjjIuhiSVMUvY7jc8Y6O9xZiYLg\\\\\\\/Fi4XlTsLxE5hv3PYtPhIIdVXqs9RaFVZWY+\\\\\\\/Hxu4BOZsQfSNZu+61055AbrolDTJorLCP6VImiD\\\\\\\/jth8k9efa1dXBB6n\\\\\\\/uQF9BTedfstg260wPJVliy2ap56deTujpWFixpMLdGN1XEh2jnv6WYh9eDkTm0vGWEIMrVs8whWzpO7gIVMb6A29mz6sWU2+irPmGp3027VQno0wfY5WNWEo6\\\\\\\/e1Ox5N54cWZk\\\\\\\/3BXN9hgmdi0kTEbIlYpqnLdjkkreJWnihqP302TtQKTNYa0csKRdvPZyXJMOr2yFZ5CLxUDDL6bbjZMixp25pV7jwBF1MGUwU4wHzu2f4FSahAnImKtFmeemAusMPsN8qXDduKRdWcQrLHMpl3b60LWfPeIWRkxuq40JjNjrzofdgABq5ReZVNJ7Vermi6wgLSAFm+MjnnbQ4nByBKCReb8ZZlZnlQ3r2AZoSEXjKc0lqCmO7tmPR09tCetD4SDTDY835JZWUUlxt6XDUsNpbr3cIbGOHTVDl698pDiCTO1k3d+jFsol6U9LIhfkh4R+bdFs11Rdm2iOcj1kxMbkP1WlH17+rdY7aVayc0bDcubQUItcBaSsNFT9icGqcWixNJ6lYRXGy\\\\\\\/NuJRIz9pNCBQgVrPgj4B6++xWMPp0\\\\\\\/rLI8ukYbfwc582innI0yr3psLmSlSVY9PIYVikuo4I3NlvBwffS3FmvF2t3p+CXQX6luioeCw\\\\\\\/u7V4XZbpIfJaUF\\\\\\\/bWQ\\\\\\\/GbZ5lDUpZ36GuC4HrLU3uAxjJMEG8qAKSm+rivBlMYXbeMVfFhRtRYrTkUcNDjqPt4ufWKE2KOHm8PomX\\\\\\\/bhXGeaItvEEoguJXULOR3pBdyZAoNdxEOOwRUrbrjzSq0o3Af5uX2o6AF7pNqufv8\\\\\\\/lHrB6\\\\\\\/6weGf+eLNjeu3y9sb\\\\\\\/25htEHD7V7ZZiooHnzeIi4grN9086kKIKAOS9XFVr325AZW565q\\\\\\\/PuS\\\\\\\/eP3ahu0TLjPqD6heNyodNfLYI5E8ZEle9GW3LowH84hwl765YCw9fxiAa8pI\\\\\\\/jaIqYSy1KO1ZdFTfMfB70z8ppVLHgK839QayXxu2aCo642GRvgSfTdrOeNx8nGcjiejsx18FgihM9EBtZ5PCvvBKoq33WGGgb0jKEkVYMqFLILaq6GDQzTsYlT+tSPUiAbkhTn030aXIvPK44XXm4mcrzPQ\\\\\\\/25HAaIuLD1yEVPisUtbTkDJN6yG2nrAvZKbp9xFPuccadej4yAUDyZ3OEoyS+tbEd\\\\\\\/eupO7vpsLC2Sb+HsXrCoYlYI9PDr28ogdgO0DoJwM8hAQst\\\\\\\/zf2hdxigISTPfWp3ZBzy7SNoOEcSOw146pXaMeh7PMKPdNcP9GgyNdi0pTxQVYNQ1JRd2X9oPvO672sdE05Y3tD2sJk3Sw9cRNWJ7Nb0jCAHaOpIO+RhfUK2I\\\\\\\/aVHzGs91LkGJNjGqZR4YuUM6TLTlhZ37lX1+54Q+yYleJd72bagmvA9IKRdvrn397yJbBT1Gv7ocZ3lM0s8GsaMcVUlFcGyKRXZxaT9gevqcQhwVLT0V7EoGvSGP7TjDzCccWdlQ\\\\\\\/ZH2dnQLLeUqApvFTm7Cial5VY0fzJPkhD+1Rwu3FCVVZhY6ZNQV6ss7hFy1Zq+DNDMqESwzMciC6UKBP4Bu2gUrRJCT2EHkC7uGNjsNE4vaBk4yyGfx5nF7cS03+VThBzgyzkuZIGe5I8EVIzBdF\\\\\\\/kkXtL+HR2cMEgFMEJUzvPoPXEMPJzonsShbWJSjZYvz3bC7oyjzk8OCzovOKvlF+Zf+LFSldh6lKk4AN5GSvmC0D2\\\\\\\/AoATUpUf\\\\\\\/gaNf1Od6yTntpcldvAd8DuFAmqUb67pDo+EX4pU+49cs4XPTyOHsZ+mxB\\\\\\\/oeVSiBCN7eykkufMD1m2D3WnMalIe+och4pMRZSBAW2lnMj52LpiF2IOSP1Ho1xGv80dBzLfsu0T2FAVWmzuI89DmyHTjjxNCAFey\\\\\\\/+DOjBUeod3+Nzw2wKhjcKUrrfdcVrdPZ36lkssmYeoyeM35yci\\\\\\\/wHkFfUsBnY90JqNZuprMjYgCEkuTfEogGjQKzfaHKHCQuoWjr6JpqKuANy6wks7AyvmsfooVbuP12ts+Gp48jc\\\\\\\/VbyLvi0hzmuJSwinZtA7q46p5\\\\\\\/olL2+MvXQu7FO5O85wjKnys6sBXy0qo0rJc0IFL0MqThorJhK5dY4LMDSoFIsPxGmogd2eB2aT9roOp4j3xtpgsvJxRMwrsrsIlBZv6Q9jcKS01y3tf+9Ydzmo5Q8WRNLXt+r+HaEU74b3VDesyIFotMz5tnBPWISEcyk8Uo3+5aWCxw7h+U3eTtqBxtUYesu++BX4cx9Mkbt4AV0MoZHezJX3wDYdxZnR9M5wyXnDLPtqMtLQ0yTDggnIt7uqyYkopCd+shY0Ugb8Ld0xkl4wTQDQYZA7vMTeJ5phr\\\\\\\/\\\\\\\/KT9+Im5pCyaEjCZW6e4KnuF+t60Td8OXCdlMYpQYpxL\\\\\\\/Wut2xZY50LwoNAQy1yomco8mwghQKdJQZHRm6Hzb1DVb06wuYCH1xAZWcTSgNBVFTjrcTsfoFP94BYb2ybPlPvKDaQTS\\\\\\\/IQD73wY3OR4aozOIaSdlfS612npDNNyZKSabH17sHVPKaDEOygxD27DDGYNyLm92G97ueueAFj5bbm3VnBJlAzrvQHOkBS1w1OSNN8QqGHbTD16BCReig4iItobcxKiRQZHru2OiwCFK47hWNTg1aDWd59Wx5AZQQ6mvmfAUQobE2THTGvp8VRs7SV8+xobpvnFAgu8YPPajvBWj17G6aJHS+5UCCGpi6+0wCLsfgJ1MUdzjAkSwXmwmgxosg1J\\\\\\\/LhmZfIj11kD61d2T5UW33fQLgjlZz1uIksUK2wEUl9JtOE3BoMS2itiy50UMwAT+USPjLyO8x\\\\\\\/cSgANKoztmY7Q8kkOszvuWVEQrB\\\\\\\/Ulqc3LaQZem4ZeJ7CL6u\\\\\\\/QGoS61NkWWESZmlXs9Kczaj6i3pVqGs3tzOZgjO9uG2T1oGPdUCiuEUbvAr+pp3UQmwzyO\\\\\\\/Ih4kUAP2Xpo0NPAkhlIrifcAeN+GYGCnk9W88FJYINvz2KM2LJmPEiQlqmwQub5aW3QQmv3NB68k2UcrMtCl4YI0nl8Sp8OcuC2vbCAi\\\\\\\/r6Fga1C1NmIkMVRk7KMlBDllWGNSrmx39ZDkuqjmzRmaxYaDUjvY1lLcDP61Gob852J1LE09E5vag7xsS2KPzhvE3IPw3kb1eb2CmFLSNFEF8KOOZIAa2vmTaX7Z1+qReKU8tMBmo1KqgcGxif5Bqs8pPHvE7pIqHPs7skA\\\\\\\/oSiVOVcTJEc2q9Wy31F4vzdXcBpqHl+GdAAus1qmt30wqfy9oiXlaiK50QWkazQovTn3fi5WC0iPoMtD5tMBs3zgg40GFhBaLrqQGnO5Lp7tqZExXCJ9z6YU3Tjs3O5MLAZmYzEFX6EDND9z7xTWl2vh7TtUlcAO8JIHB7FW0lSE5Efej+jO1qZuNNMPKxls21np9Du9cuIqUVxU11SQWkOeQtY5cYP4djDex2x1TIS0qUDcVDBZJwf\\\\\\\/O4Lf55kfkxmOcP8Lr1VMiclj+GuOrCTVXDkHBsPojKpTGxdqCtiEG4X3Ci1co2H22dwzSM\\\\\\\/L8GRldkE0MQS7tJGJXLYfwMh3h5e8rOo67\\\\\\\/fRFkFjbJyTft8xro8P6SK8eJDxNPxOLPfZglSlAT+U2OduK1JegI1XidfpP\\\\\\\/0JINXaJASNQEDHfFpRiLj1itjmZCHjagsknt0c+uxHpdGvj716oOJUXRUE38n+zE\\\\\\\/P+JuXyaWamwxQPCQlZOlMq1LPTCs5f0jaLipdAQTkS2Lw0Ckyc71oIGljQG3tra+dcZ9n39H5tFbl8YKVth0wA3stTvmaFFMgf4tvw98RMg+gbDUWtLBr3YJgOcTe6i14jh8ayZZsYXg8q3A5YC3THdEXjGSFpKyTbf0+wO3j8BCIxG+aW55I\\\\\\\/DtZOP8irfutmyZqbO8Dh4J9skXmyjgzdy5+k5eLAmECh8OOUpkrzH86HT2HgL+f1Cj0ZBQcLOnrONW6hVc\\\\\\\/9Jw8GFLNm7BhRkrccCxlNHBhZFISg1jeCCXD8vq1e49ua\\\\\\\/nYUznhjq5cdfuCfD6XeOAOvdUUnbdT22kp0QLc1Bh5wWXdE2VRNPO2xwvet7n2CX3etEAqfE8Xkoa44F3FeuXIkU3vcBIJ2wO2cyG5P0TZUw5OsPOExZxhFoSoTHm0XFjUE6nYQxTJcWcPnbLQp5ZywivmTiPiKplXu9DYf4VtB73OvkJe3qcDB7+cEQ9eS4s7vfZwLjQltfG0ma90iPvl6SEt6wSj5+vae7+ub1nMM6bDTqdTehjshYQElyWWTAIIHcpF3NRYftmMf\\\\\\\/XvUvIsqNuzEWKRpkea3BZwfEEwebhLKEQaeL1+NubYfOatreTOntSGp3YL6zkWiOCIBQBSuRxP1ShjuGsGkVsQsTCVAF2UL\\\\\\\/IQr6ayivdhKRKjK8JFZzldIAkVD+3pfa0kcWc1tXYMqG8ZF\\\\\\\/qqzaVqCbeN\\\\\\\/2KRnY\\\\\\\/qTve5GtY4n0oFGW23GzcWZPs04wLlNzA3TZJgb2rYBYQ60g51LLZI1z1WwvLRU4\\\\\\\/Hw\\\\\\\/8Uy0YRFYrUQt6yXk+cw0th6+mQZC8Pw3kMd+KQ2AwKA7UH2yQREJHsW1KFjaAMQJqUPAvaimZNzbtnmzViKeRDHlKTYuB9EzGL58kFRGQ\\\\\\\/hYdy7WPToLAGzz3aIaDUVWRG8UVfCBxaoRAwGdl3vEuwBY3aJgdmJ2+FJcK+ilk0+NZTrSvNUe01DaZksKhBZvMJ9Tb4APw7zB289Ev6X0Tv\\\\\\\/dBMSOLmZiB2UJTLf0YQ4ua27zo2sTeKMNxle+UVy2S8wyfSJGJNlwdM4mTK6y5Bd6DYKRlLRH\\\\\\\/rJ64yc4VhLo8K6BTQ3QmLmnxx8TnhK8FHtzttGux4nxEUgTt5FTrnsTFKnv3z4820IeqHxg6Zh5CYD2sdrfp40VMGgy1QzAq4hYFt883bP1xOFYBTNC2NmaVXCSmoTKhyZCnDEE42dIjuS9VnTSbGKUltZLkzDhg1jVtJQBrjxl0meN+opoXEukeH11l27u5kLlrRkKK7gg1nYHSBRKbiSrH00MurDOHv4MjyuuysnvasUZSj8SPpswBdLr+V5yyYDGhw6rYO1Aj6gnFGcNS0Z5B4892mIxUKxhWinF7FWaYPzY2P6K9ddcltCf\\\\\\\/5LpEYn1Y1YvQVfQvNrwau8OQuBlm3UJujjeO028NQTFXrUHXYnNnWzycpWcz+pRfKh9fSaL4TrExin1l2m6ofvulakfKcznphwj7g8DRBh\\\\\\\/0yNGUz5QvCSjlLpGiDEgQeFbNTQ2xNj27DfxtUpmN8HwX5Ik\\\\\\\/urFqUdeM5k\\\\\\\/KOJXTwpP4yJKv5fjhkQjRwlLN\\\\\\\/tQWS0878MdX8O3y\\\\\\\/JZ7U1UX\\\\\\\/Z1K61tv8V2H+EtvVWBcjsZfu8RV5kpnSkT3IDuAYUbmElvNmo++ZJjvr1Wjlw6ICWpJHYCyHb1hBjIR\\\\\\\/7rUDQPyZiiYkwOcndLqT+w+wYdeK3kY927M1hdCOVDoQ4bBSUjXUfUuUDMmYbr5fPfftU5anBvXfXVHbwhvC84onusAkUIFMkbfJadrmvR4SzKiiRfqKWdY8OBDyUgj1v\\\\\\\/kzIp0ziJLz52YURH8F8iF4A6d9T4\\\\\\\/9fDHRQVB69x8Fz6rCjMw9gjA+tlMqvN0jcklH6sMGZg+ozdktkIS35xak\\\\\\\/9AiGcjhlptqvbypO6tnLge6pwA94vXFpgLab2Iczq\\\\\\\/CZ+pyE0AvFsuOC\\\\\\\/4tDy2hYQL9s8eJjahs33raMEob1Eygs2rVQCeh8xKU95HK64Mmgw30\\\\\\\/BHuGMAIhKbuMWAlr3v96BXPSsnlo9en95AbYuPSS1agaalvRs0SgdeTDmxBiZFcfYvUuMXKcmTt3koEiM7PwZWVG6C\\\\\\\/vPI8B0hlbGaL+DQ8z408buJJErFZWu\\\\\\\/vyDHDB1rSRsebZ0Kss1c2otCtaxJ8mNEOgtNnD7K65\\\\\\\/RMusvQ86cbsfROOlcgr5MgJvioRG1uSvjhULE2ZHQ3LeLaPTfeB\\\\\\\/86\\\\\\\/916F6aUZLIJwXnDfyumjBk6sz6QNnnQT4lKLSPaxlXfOpbNplGJvhpQD3I2WjKYZ2d2hYsBIBPXXqswquYiLndlB+YQ5omOKUuUCy+usFFTuKjKHk\\\\\\\/ddCi98W9wGa148RcOFxLvttc5S8WR6WdbjGO0pZnO4uWIYoMIEzcnqf8GuO8aXSn4wls1leXP5xhnQtO2uwKUbnp2+zVyRQQf+akJ9i9SRRcrOlr6LUJoNMchD8XHp5WIWRMs6EajLY1+yYVfrHtsaXe2u36VZPZ0F8qAqmdtH\\\\\\\/OAli3MRwg7q\\\\\\\/0Muwz1X+g\\\\\\\/L2RnGRW9GgVhtoPpv8JNn2xXDxT012R31yf7A\\\\\\\/W\\\\\\\/6Pis6TQqJ+MVJk5eqFzNT9jEwyYkxvHzvhxrDKaYVBASbT48mmWN4VXTRt1ClwocrxrOmaau+53Wg0TW\\\\\\\/4KS8H26mGlu6yIJZ0OvrQzpkLhCnKsfGcpgYfvbbASX3myfYX6abUqra1CGJrjh7bEzsnhmsBK7iq5wKY5fCEsUNOTgM5hhm4sQ\\\\\\\/s3Ewwf6CN4d+ZTyWxljmjeByU9f8qfh5rhgcF+Adbo+2e+Hd8+pEicMNEhqa4vzIViMr0ryWZUbx8WWCFzizZwaCJKZf5JkzDlkNiw+xTrG3WGZhl\\\\\\\/BBu7zAjhNYA8BLFMY2qtGETaJAv8RPeFns2lbit8Cbjx\\\\\\\/ADW9TnWC6CFgOkYJM4etnFrL0Ohd4MJYq1HCDkUrUj75BMmUqoqjmWHI0ZnCtfiIWllGg9sLqgJ8hSGGW8\\\\\\\/fVnpbAjBrT2hHp2jPSkUxFgv9\\\\\\\/LGN0rMWobI13ei5mkdvjJMOv8sTGjA6nwz5OpwXk8j\\\\\\\/9j2f2\\\\\\\/mX3WS2gyg60Vyhlt1+cXibi22Sz9Fo9WQFdWfDFqG8B5SZsRrPxERGiTg\\\\\\\/Tp28zyLW+2i8lFSjSomJs55zPFXLIT1mSd1G59vhmimbn+NXbGIJFPjip9PT981LuZiw3Bwsc1BztSDFZYak4b8eFUfV+V+ZnZ3NlJp79DBt3RDqj5RLVx6Rg8xDf2NCp2ewERIO\\\\\\\/TkkR\\\\\\\/3fV5osqZAhVzHxjvLqgV1URz\\\\\\\/TA4qNX+JyKQHAy1hgXjAsohuQ3I3DS7XQjNA\\\\\\\/L184OAH7McxPPXENl5wb1zoWCgDyl6kHqki0t5oGrXYnmRrxCs\\\\\\\/y7eVs2wweVJ0zGjNbTc23bPvDWbafuY+knfX+oc9ClIz8rj18dgDkeFgn5ABzPz46w5Wd6W9GlNKJuKM8Ldycc+XH6fszr9Jz7dfuGuJCfuLlOAGzPKekAHLMA0SSBFf4nvjla2yUcf5F0QW2QB7igNDSUIgck1EBP4Xga9\\\\\\\/cd3Oq3ZYO1hU5mGFp9b6ndCCyLokQ+jaH+bHZWSpCyJwqxAgxNxwlQ7kw5\\\\\\\/lQtZuP3O++5SeIfwsJr7+I9hNXt2neyz35qXrZDyW+BF3JIyaM+VA4mKJVinaaRLe8sXUL2nwMvewyvdJwh6\\\\\\\/HSSlGpwwaPJcHqF3BmlFH0ZX2Ztps9wsLrTynf6Nx1\\\\\\\/N2X4nM2HrGn\\\\\\\/9LhNXgQl3fyO1G1AjzB42cxxby\\\\\\\/ZNTV1EEdEfLFdgJa3embjMgizTMOISh3i2PYk6BsWWBa9w1zalapX\\\\\\\/snh752G61KDnO4s1Q3H3nIRrnVToU4SjQTgNfZJGm64g\\\\\\\/unShtUOZllnNamXXR32VpLJGr9w6CjmOvxAxBd4Svm4CSVeA2GfE4iEi52SiHjq\\\\\\\/txfXs75gk6LRqfO4uP2Wmb+IyX0uIyPKLV1soeRmYEfcBDL4iAYLUW05qjyHY2LH\\\\\\\/ZBPVlsN3xzAtLtIK0\\\\\\\/IL8zztRsPoSAhy5N8TodvJVKr6RZuHHx\\\\\\\/eQV\\\\\\\/hRGgU7nlwBaR9GwrNiU+RA6NWF7GGyS9zTkGBKGSeGldscQDK9VCuO+iMkumPAAOe7nRKhKVKpXxvCbt44mKMkxeoEsBQ1S3yfGhk0V84wOwu+JTEgHpBxSnzC1m+z6selECk+6KOUWL16QOpei+R37s5FKv3SuL\\\\\\\/AFw9oqIXXqBJ2vXg+DAV4lYCDao03K64NcPq3ADbX\\\\\\\/9ghBDfw9K2Elt4I8hUAtNQIkEDNiteMnxI\\\\\\\/xuDRu1RE7YkHaGyFpZU7GKe28i1dmV+jDnK08sXYPsEEcUg4g3fzcNVNRuBvbdkWMCPkUYz6aR3nphAFkxZv8fHGb05IBqxPzRfadlYX1cy98Jyn9Fw\\\\\\\/Z07Fcheu6lq\\\\\\\/npr6hzqY6CYmik+\\\\\\\/G1DEulImgtDHfuv0Kp+roVb\\\\\\\/AdMUYjivJCNwgMbC4fnGHC7\\\\\\\/Dg3cmuYEOqXW4Qz4\\\\\\\/AT6eMWCVsdMpD9oVRPUS2pOVin0JUm2psEFaOuf4RyjV84pRll46MJiy72y+d2CD79SNgZy5yjJUALeI\\\\\\\/XSr+XuXWfFBH2CWILT+SZHdDlZ3rYkaqOvK9W4fx\\\\\\\/+5C3WVpKrnbY5q6owUJlLzXpmfFN9KWlkuXfQcoYDrOBXzFKDEPXJu5FXTGj+KUqnpJOw4cb1rYbfmtL5APRLfduJ8yRCO14G2Htl50jEeh0cGliFMZ1lCkKrYNiQJvHjVaCfv9yDwdiJ6F5RMsMlkqoRrETXw0cxg\\\\\\\/F4LLeL1Y680DVpxTI4QWcWmLRPu5mN4+3h7nrfFUFqbsNaCs56Tr4TFFmP+y9yxOhOX\\\\\\\/ZyYLAUbisYYr39C+V\\\\\\\/w51zp\\\\\\\/9am3fyHMUhxGzZp\\\\\\\/a\\\\\\\/JKREHbpl1L+mcDqOk+mt8p5EpmMYIusDGmIuyVYbIYzM5nXCTI8dD5JeVZkO4tsz5zea0F6Voy2\\\\\\\/gFqULG5dzw5fYyr80onbIdXnhmazMl\\\\\\\/s6rUSpfmaWZGRyhw6Ro\\\\\\\/m01+9RqKp8SEJxuvVCHKcCvZd2Hpeaxsd+2amOX5dZMaWVlU4Fwgdums7iAjiM\\\\\\\/JpCteth9zKYWZEuD1m14eIEno9ZfTwnXQ2zRUTxge1802Hk+Yf8EtLUeuhP56z\\\\\\\/Dz3MaHoZwl5DN6ug7txRRT8YIalcSEao\\\\\\\/G2RtV9LT\\\\\\\/hX4z+HLxi4Dz7Hf4AaEyKjlNgRUnEI28sR76mVCF3fT8g4IgWi4wHp4c\\\\\\\/wzbG84Ekf8HacoHMn+ypAyU1Nc2aZXMfC7WzAgL05G+FvhAN2gzW9ScEZD6e6QKQqhgzZg2HYrYkQVhPgqLZFeQvxmMwtetOe0wnxM6vxDFgUifj5o77Tq0Et\\\\\\\/\\\\\\\/eE0DL\\\\\\\/SiIiZnov+veOLdx+8O+33sR5zyYidyvWL3Kx1IRg2+bPsiITOsGiXDBEwnJddqHaTaQtA+jDNB04iwafM7E0pv7d94iEjF4lLh9fzT9mp9zPIBvNtipC5m7\\\\\\\/UtPOgaSD5iA0Yd0h92i6teKzBuiSn0RKluhMb\\\\\\\/JB5WRRO8Cns05yQheiuwJdIe8KT7BYvr0+Rzvh8yt+8I0FeV3hDyJAtZk46cF87ey02jTFlo4\\\\\\\/hlNa9XzIIKM0ZH2LHKttPlSplGmvcPEWCaDFZSNW370hBx9xNE55pHKfIltRGkaJsxVeCpWWDFdpakbw5DFEFsbwae915kRJKgBIC5S0xal3KoMfBI\\\\\\\/X5KpXeOn1xTp6NkAVT+dIeAFcpCPT3++XybNxUnXogYouxxAhVyKfjvsVutCCsNQjGDZ3h8td7Pl9eFNXnEanRoXKWQ4uFtbc+3sgQwa0XyukK9TOMnG6DgYyjDcAxGGvKn3tLZd+BZ1lY7ZBOpzijqEC8XzdZxuKvvXqanN0zmoG43OKtwOxbZrU4Y2LCDSMCzS2b4lZT4OC0jf3vrp5JSiOdXSgZ4fUty4EfZ93Zl+1v3mgHSjaFcNBf\\\\\\\/6FSrPUErF\\\\\\\/NWpLV5k\\\\\\\/6C8gIu7aZem5OJNMtcOQBu1WkyZguuO8Z\\\\\\\/iR\\\\\\\/8QvM\\\\\\\/IzaUM\\\\\\\/rabchk5kBYinDsm7QEcUM8HpOr9KZnXi9uK2fiamRwI5FsBmc7vOaloW+2ia2IxIneBl9dj59acHNI9rmkgZdAGcXR2DkN7ru71VAcnKEXX6\\\\\\\/thztXUSUvzdpJ\\\\\\\/vpW6DwlGbpsgghhkJZfEQT+RyZa7KPs9GE\\\\\\\/\\\\\\\/X6R5eGPRRwnnu0bx6LqePk2qgWg7Ux\\\\\\\/KQMkdgPkMUK70rDlIz7pYe6ke6lZl\\\\\\\/UWlFMe7j+ulvksLmIKwBrHvsyq0qymcnkT60zoHgytvSt3mtPuxvIPgYNyBxxP3ojTfb+lYh7uGMTvL6dxG39ekmorAHa13gV4G4qGZ22tEIYZN31OUgjfhuy2TKTHHXw14Sa8E28JTgM0CjWVg4gkIHLD\\\\\\\/RTsw4lTo0PBhx+zsd+xLQTNIpiqbQ1j4s+KBc79w3J5WrlCE8AYyIM\\\\\\\/fD4uHqcnUdEkPUFBccVsHJYf2XCyjtBjfRbkP2N+rex0p7xzKn2Bf3zVGDbG8+L55O57E01ZBNOQ04MzdfhWULR8Pzu2EVLhMoX1YsDAYrjNZT5DfXpKPPWtmGdDbNBll4TDN1Eyk6sVmsT6C\\\\\\\/99bzqyI59mCYN9qIsBtZ9IR3+1fNxp1yDdjQBQPAau5UsigA8jXaQ7eAxjTtQQtlG8LwYBZ6M+tOn1RNlT2k1CmcCPGjho6WWCzIek7IraWXn5oQmrQFjTTfA+UgZuR9suOYPyst5sryY9c0up823jkpdLE\\\\\\\/UfHr1GZde2AC26YEpUh77vftaq5gAtLh8SHGF+ykqkVkd0okHrBgAoFDaXOdtuiPMI9ujTvItUQuP0BzpgeQaoXUdFYFnRnY+PPww1aLU5nnxO8YrQzuWfEBJkmGwsY+iPAKHdwiKjoSc95eLLiSip5UmN\\\\\\\/uMqOiPzPO0owwPoFXhhKtLwsv1LrDgj35GbjTVc67DkPIp1bWiutWhKnbFqjlMczZH6EhM6+k6eQU3sWlfHHZmtsINNh4pWHnwgvqBMn3tysGsKkHc3ofcd0rahuc\\\\\\\/GFPR+PIGljRvXUBtpW7H0ZkoLsTdXY1GWYMu\\\\\\\/9L11k14E1WnUoLPlaRRDeHpMK0Gkaz\\\\\\\/uOprmxdqsEh\\\\\\\/WU9OBO1XxzJniOsKulWi2CKGErU31mt+kCdXviG0dzf\\\\\\\/vFiO+OrXVU6Ii48RxdAClU5CGYciqtUAhhAkFVoPlvai3f82OO6X9nb1MtXQDlKV5AQlY8rb1bi9A6N7gwgrIYw1V9HtQWHf5VSPnXJzFwR0odu\\\\\\\/jkRMboIFyAe5kamQmUzxgKOtNY7aIAS9AfYBLIlAnMs2L\\\\\\\/\\\\\\\/rWlxAdbH6glVyBgj03E\\\\\\\/Phg569F4JYoozuqS\\\\\\\/yq6kaFVoAoJx6F5sC1Ig\\\\\\\/Yw\\\\\\\/m4G2EjbFlbLn86zEKIeb3dytnNYQripEtYt08fIG8urSNpNiCjaxwp00kEQPib9u4w+2ToLOzeYyfpJzq+Nul+rPg5rf9osDlvfic3gJz7ZZR+X0h\\\\\\\/pE1MAWqKRSfn9hQGYOSMx1UH2TxM80qXfnfO11llhojH3STZs9UTahlG1ShnfAr4Zo\\\\\\\/QZAHfcDjIYCnIy0kI\\\\\\\/kDBszvSPBhhOwwv5ZPlqi15tLlK1gdymQApgIvzxuudPZ1v6lOukuU\\\\\\\/JyjW5bRDg5orgqJXB4RzqCSF6Gqtip9FgEl8Aa1B8foFwYUt6Jwb9lu8IgJ6gMv+6ObwFn8fgppDMHxeX5daG0i7EWA8y+61OOfi6lwMyMNk28Lpp2nf+bc9KYZzPZx7MCIMeZW0HmdYJLNKQ\\\\\\\/3OYrduNQl\\\\\\\/uhkwyg4lDuQ7GmcG38UQGrCQ5\\\\\\\/KSGt0+54i30P7iuYt5pGGcPo7Z26IHV4VJGVuW3i30TvGzPRRUUd8XERpgj3FeInd23miEEZ9HuQ56HY9kRMNU6utFttzByOxentnizrVAnT5tRftraBBufKdqVlRgJ+iqmBGX+6CdsmCSmktTDF8tFCd7c0baoEniMLra\\\\\\\/6rJD1cSy8hmuhM44X1HX9RKtuFxlEWG7bleaFTYjj9ZD6kr2czX1x1nC67fWZU2GqlqZUqMYNEa93O0Bgdq\\\\\\\/eOQDwKfydakIEA5OeICQMW5+EwLXOUbephU7Dj1SE8iKoZwB79SIbSvhHiRsYzset2+bgMSm99UpUlUD11TkLdRBMG21IV7TDE8v8vtVh2pzZvfcfoDQNCJoxEnMzvYyUzGOzZ+0G2IxWOhNYevoFyZ41boLNc+0+G8PIHR+UosvdiNSKWiUcdFmTVKZ+w1tnHiUvDEjIf\\\\\\\/yysTRaYbsC6jPzDEeH4BfncBIj+\\\\\\\/tCSMVVH6QPnAp+h5UZVku1\\\\\\\/3S1LLRp4hChY4nvM52ek3fsfZnWOvhgNuugSteNNXtVZpsK384dTDo48sKoP6P\\\\\\\/tRaj\\\\\\\/0mArhYQzRCl327DYrdJDD5R+\\\\\\\/jNHv1SECUw3RxqD96sUebqjGkWuibyIshe\\\\\\\/He0v9qDRupzrl8r\\\\\\\/lORrlh+qEvzUKNz2UULq8OO2COIPbPIQwF8ND32AYomOMK+BKyn1AsfmbysQIh7wHwGwVzhag9\\\\\\\/1Iusqqck9ri3uMrAFvaBEz+d3Eamyhj9VbHq6JvvM6vpIYotAGmqavXh9mBe4W6S1Nv5vK3QDs00uMsJnB3wDDAisxAuqov2JpRwpVDY0ILIuVLrggjAYhZ7fYK1NlnO4ikMU9NLNE+lQF8EWL4+Y5UoseK7tODnizBqiNsyHcpNVLNF2eObhfxlEKRKSYIVKZBX\\\\\\\/9Ntf9waRxz00DDCcAqBhIJiGEJcHaoAo7T2uE6SUVxx+zZwxLowSW9o7oaZbl7vfXNrnf5AFWYLv5F9ff+3tN6vEAL\\\\\\\/f2z\\\\\\\/N+P0IzFAQllT8QJvsejqsGtICMGGpF69aoMKhJQ6Fww0Z4ku24NKc0vU6dz1Szd2LF77f9GA3LbfJiKtRedsSxLefHc8\\\\\\\/flBKu6l2lranX15ZuJ3NpBBwFWER4AaaLXiaO1s8fr2kgfHAoEH0JaMzAsRlr7iFRnaWeRQYlhDz3eAQWVo0GJLM1kNnRqyZlqzEKktTVlvcV9B7K0ppBK2gIldR3gATeqNPOgI7EbFKII7tXDblmVr+j98dG5i9MzNxtuunq+lmVGL573Wbi\\\\\\\/QGXE84uoN7saUZgRUnZ+pNcmZh5TP+37bvEs\\\\\\\/8n0OABemnohCMKpe1lARm3UPZLn3CwAIErs\\\\\\\/6IaryyMVYFtr\\\\\\\/YLB3job2BCvOarijagv8alyE2n8CCxZE4JfGd8bB7Howyggw8HMcCcfPcnfoUpO97X\\\\\\\/2a5iMwqTb2IShHpsqKAi+Xv\\\\\\\/0BssYTO89AQaOc62vH3po3j6PZ0k0QBzbpDacsJo8RE8QrmUcMqkmlmQh443qg8glO7w0mxMvChM4H9b6kMxTnmc0JFoakPfErZ3qIH5Um6I+UxT5S11vY54HaYg2D0gxjRTvjyaSXULFwxojjooWPzRzOaD8dRwMH7y+3b6O2l6+wbaA2DI4k+dQJsOxigYkIwMcDh3r\\\\\\\/oXui3gRj1VZaDMaYJZoDlO7ylyRgyMgvTFbiQ+SYQZ9AjX9fXsbcK1j32RMa4N0cionNBMwVw9ygu2m5rfQ8X7H2R9EgXUI0JDiNhj\\\\\\\/h5FbK9jnTn+PzuEOqXcJxLi3h2thlJa19YcNO9rDYHDY50Omea9xMVdwVvGx9zvFkANVVIOY4YHM5asOYcz\\\\\\\/4Efz+H5iDiPWdclU0L5m\\\\\\\/TUxF673y46UHbCITTYOJ0CRwT8y1GzZfmUrEdPBBCUqiCDy6wcBOy\\\\\\\/XxLbJYiEZU4qpJKRwtMsuG\\\\\\\/XMGYkTRki7zUY4iy5lEW9jAc5ymMQRy2UFLlo3mRqG78fFl5dRjb1lJ7A+jusroiGWCllQeXqV2KA8Q9yO4NqFVUoeWQyOnLP2rcvfRnPBrTNQXS06s9dy0M0iWaFQHssfi1HTTs7dn8FzeQpyXYCHc5y9WUVLRITZxdZ+sf0+BvT0NEi3crQ8c3qUsfoK2pvwnG4F377ADC6I84ORtqWxNIX8Jz10ZCjUEmOZLFpRN1a+i3RWaugtjOCDKo91KJhlrG4dkd3UoQP0+SoqEDUsQ473I+Ek99yYPlusShuSnIIkKIRI\\\\\\\/B\\\\\\\/NbXiZnnuV+BbVWWutgm\\\\\\\/NKlLA2YU\\\\\\\/iXbmhXrFtv0gvsHa0ChLn8F4ssDcT3AgFBYGuB+N+ilfg0zjXk0onsQ9LnBDObl\\\\\\\/G2U4PJUkZfFa6\\\\\\\/kqt75e2VjtvNcuSXDPLmZp\\\\\\\/FexrtdyFnTXq6trkyTQUcfThTPwjZHpjRoXftOuT3OUSvFOaAUVk6dxsxUIssOSvnTxpukvzOj8Frpg+44T74L2x3+iL7mS6mdeDbp9L54Ls\\\\\\\/xm3uiIAUk89zUE354AC3qhMW8qbxaaxF74jeZJ+3GqmbgLUsJyAg0mzbmK02mUvgRaYJoC8v2Nn4exUmdHtLaGTbLuNjwChfMUHWtm0NlSmayUGD1hhbQKPCoKIGYAlAYswHhx6EIl6PITE7s1WYPGWH07f5F0ZoTufzxLd0u7p17d6faP7MKg2jmh9ONaT+BqZcapE2upa68g22swlJbKC01BUuAVA1CYEHjO\\\\\\\/b2o2CHDKu9L1hoPP8Eq9rI3V3C6dcrjcEQMzeZUmm4XiEFMTue3aL44l96lXPbuiZX5m8KgjHaVM+PBEdTKSl1FvM0YXu8FppukDBMJrZJdoKnrMzzDMYsbkw49oN+91KFQDF6B\\\\\\\/7K\\\\\\\/sIq2BovJHuZMDKRdQ+DqtFnfMT+KTkZEpE3gd0b6oRiPFwZB5tTChRfgVMRzdY64tfkm\\\\\\\/SAy6mP+JQ0QT3evMSKR9RBfCboTIasIr415SIicY8xa6hfbl3S2XTkwNLdrZJrW8Pa\\\\\\\/fe2TGrYvexhQs3gyLma0B4xymUNV69yg1rPUKU5sm2\\\\\\\/isuYy6cdWeFZYeFEFyo9wqWnu8E\\\\\\\/bQ6JSOJnGxlcj6hDQjRHktBP5pXPpQmb0JywJiG1cRjelFsC1dUOJ2wb7SzyJq24FF\\\\\\\/XWRE4zWi\\\\\\\/n\\\\\\\/NNEf4C6pTL+UOc3Z7hlg0xwZzGHG4Nh8QcGxh264TrzKarBl1dbOnolpICg24lbKkuQUXJbRNb03Us9O9Rw\\\\\\\/GirGW64V+RT\\\\\\\/cQdsbIVs+2u5tfjWzfrtppbWR4bPcgPsi4YoWkFX7VYYwHDpCWbFraUpOMjuhWjroZ67wRVyH8BsHlJ0c7EunYwHsrOIqBuNOPGA1u29Wzf63+Bxi8zJer4Fk02FU2s77TtmuD3Cz3Ix6zo4nCQ2QASu6QmPcrANonu6xGceFuU8Q9GVUc+JYQ2lGVK8hTewaFFkZinYbdw9sVqecEeMGa5ZtLikqrP\\\\\\\/G3UBRRtLyaUodfdDMxh9YsYaooP3vicJRnTPGmo\\\\\\\/YvLSybvxHKNb9U1ocMTvEXpBBAkxhJ0iKBIU5BR\\\\\\\/G+wSvMKALu4jJbCGGancGBXD77lwAN0t+p9yeIXlKXpSElg9RlndWo49b7OBARkjXRZatnEslwodUCqsz\\\\\\\/3JWCu7mWcXqWlDA0LwWRM0c7kYsbDPiu5MJqDDiWLQ0AqP33bBFQHdv2NIbBHGQE2vXou4\\\\\\\/Lpf4Jx18s3Qs++\\\\\\\/zRQOR9hoXNoUacwdG6cGDRF+6Rql4D\\\\\\\/QYPe4GUY+Ze0jnVtSukvtdr\\\\\\\/40e5aR7Mk4b7SB5vL54ZN3cxZ9rPs9baxnz03ze5T5XLUZP\\\\\\\/+JW70hJK6oSsir+8Wp2SytHcJnsA1rFendi8gXbsJ8SMiUW9ErZ3Pkh8bVjiqJt2uF5R0ARX1BgYNdLFCpIAQlzQWWAtyo3YXOw3fStADB1wtuijvviouQDXBgD3jYvyclGWKQMiyivSg5iY+fVFCpbVpEoMYIJGrcSdyx4eduCGnE1FKyOD2YiEQvrM9+\\\\\\\/4VgkPBiF3RTejlcQxOcoFrPwMy35lMnNdiOYhkuhxlSX1t2NhWCX\\\\\\\/OqUroGhnDyP9NGiBsW5CuR8TMPcVvnHzhpYXzqBucw7STPW4cbAruDHM2scpk9P2RwNY8kMDs+xYX7T3r1oZVv5pBewYtHsI0CjTnj6G8JIJ6UUK2Lo2XbxJiDf3\\\\\\\/jjnrfZ4V\\\\\\\/LW\\\\\\\/tsdhbRN7+MfUJeH2uUNsNkcO3wf55MrYSAnXVTdSzt5ReRvO6H2z7YUW\\\\\\\/LsWG04nsytnR\\\\\\\/pQM0vgqqonGui+dlbsKC\\\\\\\/wrnECE0\\\\\\\/vop5VGV7xKaKLxuAcQs14WkBJZuorpDHPWcGanwOa0CjdQjCJuIXpnT1Jm4kC6rJgcMJPZaQZX9uiTLnLd10xN1yzkWs6dbjqL0\\\\\\\/9Uti2suBHjx2siBwwIVg1499noj4t3iM4kOoDGl5SXjIIqLYeqEycGOaIaZDdJl\\\\\\\/5\\\\\\\/FZ5d1ZZQMzlMw8Vrv7hZiNUZzTRySOHCX+ohYFoOL4b4rtaiecWjMFE1H0g0psq7mjWIykGFIGZVSHQm3AZFUd4EYq8u5xlIQSfeDpUjQJj9lnxlWPzpQD+8OOiIPD8w5V7ZCczXtGNv+Gy7OozK2KAXThj1kkhbqEOr\\\\\\\/elNXypA7fmsXu3QEo+gYWANY8iaMDpsgYzM56nOThealCFMY60S7mB4yWRtT3RUHELPQZpVQU0i3MlD788rGY4uM1FIxn\\\\\\\/\\\\\\\/oki187Og7srgdCQr6zg2p5p0BX69F7mip5R+UazDxqvh6ZaltJ47cAwS7CIQI\\\\\\\/fAZNunYQBBbpbxxuWL1G+cNr39Jo6+\\\\\\\/OB2BU14Uf7bufxWZMIXF6smKaXJPbUpF5iaqBGajQ1dQUzqKcKp9hVyXFf68Xtch384+GEQI4Oypth73wzpPXLUELt1514SCWPZCCg2yokGkYWew2VDRp1JMaZUonaQrc0eSd+Y5rgdHb9SkNNitpT9\\\\\\\/jNSiAH7lWgcGRuG5\\\\\\\/RkSJy2XbVf3dufBlB54GB5DzI3zPxCcqqGtFZWoFBBZNWv6reqCKZuQLLUv\\\\\\\/7tK+08GEYvmrJvaF74qbCmAMkDIBVpnqauzy1SeDAZvy7XAxdKQhJluWEA7Ng5E0C+oSzeqBP3LN8i1waaKJbQQ\\\\\\\/wOZIjJqnzzsumpq7hyI7rU+VmHNAPVxd8JjayggQw9Ujzcj5da1j0\\\\\\\/AZ7r+TYEK18a4OpXlOe80YHebnjCg3KArXrri6OqSTBVEIL4Dm4Im4rCl++nQvOOWpC7iXztcDaxFwNeN42oePE9nWJNfiw9SXqhSUSV8F+t\\\\\\\/EL4rrKIRIvEcCYISRHsj3PpsA7nOv9KVSXhRo74RCHsXVFrFdIl49KO7jJt61EnV1XA\\\\\\\/Ye9nrB4CA7WQfS+t1oXF9Wxt+qVAGdGAD0gS9SmK9NVA44t0KSoMD70b+T8I\\\\\\\/egKsIr0X5CFCloPu8vq0Bq7o\\\\\\\/Fwu5yEvUzdExVeQMGQG2rxXpRcthTQhFWTsbd\\\\\\\/nQ1TXFBRE5LjsJIDkEWiorfbcYRczGOlHfY4aqJeEXmYxGwlLlFpCGb88GwaJvq6nUxInMqEIk8lBpejTisUtF0XnTc00jF8TqAFwI24TkygojghWZUtq0eyJORIIdxRKRrnXVtJvUFI9rtBABeQqkgypJKdCAeWPanpziaxjxCX+YB7DEROnI4OMTGxRI8ZWmqGvPR+YxuFp9YY04Ot+3iqu0ZXP9pYa99lFe9Hrkdrpsnz8dUgjy1KAz2p4P2qtat1SHFMVaybf5Pn2ddRRogCcYBC9ETG+u8Cam69vcvlSgb29p3XbOVWNJFRbId7iMzcRAzHJzFHh2txhiA7kUNXVdBWqPVXoTEfwM76VmXN4mvsSQ0Eo\\\\\\\/T1EnnTjOny0dV0dj01\\\\\\\/DJDeGi8lge36bYuHHl9eY2TZdkaiG+VjjoM+vH95BzXOz46+wfHi+jekqaHxB17rtBlslzhjQ4cPIVXvAx5+l9K+o\\\\\\\/fkhqZVsvuhArfgXMWfvWDvgVNra+HJetR7sUSFJrHhFWxL7JEfen9XJaphC6nMYQ9x9HoUShJQVqto66gT2QYiokeoGwCagaxb+XuUxMYcH50UM5Ddi9Bccng0Pa\\\\\\\/nmkYblkKVR7fk3Ai\\\\\\\/8OnFmG+MCTRVUc9yh3SAOkpspmStfeWsAI0cCd4rjnqtTPvhydiO6330liQ8ktO4mI4KGkhJanHMEDFh0pzCOt9hr6eSP5jC4SozKTdNYdNCWVsq2XT0XSB7YPbaJIaiayCG5FlY9g\\\\\\\/AZ2xwL6yLjia+LZSUpJfpdOyDzeAuv679otDwtkOUa7cNTbEB\\\\\\\/ZgfHYCZu\\\\\\\/okAmBP3qWb7xbN0Z4Ce27cirtnmd\\\\\\\/KrLab2gO\\\\\\\/7QayZTU6ZFQsWtlOLG9z3RnzDPbKFjT\\\\\\\/\\\\\\\/pEtYkjqcpxpq1NDJhygzxp9rjzePpWOnzuu5QHl3Xn6cwHwkVpdQ+QnNSSPtYcvARRrKNU0BxkLfbt5\\\\\\\/PfdgNku4RMu7Q7Hpz81asCx5qI38V2HScOgks35loWpxKk7wpb5U7NfEfYdAwLRJcY\\\\\\\/P3iN1uD7TA6cOaVBdtfsF3mNzyJzWbKjhxMELT9CTC4CL5b0SemykR28O5P1J8pXjDW9xZoiizmo6XQ27LcIL\\\\\\\/FzllC8GQtwpcL3bDptf7SqqvirdPKvIZT+3cZT0u5uao95W\\\\\\\/MqpxmwV8Nom2rTBgtP+F6KbFGw+GXvLsCRXawF1p1VZBGQZ16W4W00BrgTZfB96\\\\\\\/zRwgpBvxy0lmO1uXgDizRfvmYtVs3AFUvpZ5dCDACtaL7LR5s3O0mBqKKyZns7qzwVrqtQHDaGZuhOBQOzA81oNPfnjNeDQAUlosHW5p8jCLqb9ZSolk1gzj3pjIHiPzWzsJvLHFTPnOK8E\\\\\\\/HAulVigYVj+acC3GPwjfIXAXT83Oc5Czcb+TKlnpvjtqCdb04PQWWIKD8o9zkTnKlth\\\\\\\/s19Bb1z4V2d2rknAuBIdDjtcVhWYwLsf4kCnZ12w60tD9z2ta5LZyOXp5Wtzwloa9FwJOc0u+YTkkTCB3EkI5kqYE7y7cTOf9lIGnL5\\\\\\\/ei23YRCS4Hqn6bv1kWx6kFqk2rmiTs4jIszKfsU7tW2pQVc0tn8\\\\\\\/OtX+N5zfUzfyMNTNIBJln4KjucRC1786UvUu4664HMoRx2a8LnqPWIK7mnKBx+8GcH8noXeX\\\\\\\/NWRrKOwzTLy4ohGZu6lkqcg8jH1mXiNK\\\\\\\/ZU2ShKgBW67Mtrt0mU9RQQt8G2CfS4wEARQYeWn5YffaU6wTkVahREkitIPAWfaeChRUq\\\\\\\/\\\\\\\/hZanQIUipLf4SFam2TTN2QX5PiZ4SDkQ2G7SjYsOGjCw0i6pN3Y1RqUKvnNFdAei4siUnUhlNctXPLjBegsMvhvm2Ftaz+jsrUVnpDtKCNI3ds6Bfuf41sy1nbbK8AKoqzyamHhRIym8pwRgmyvxKp4VGhso2FV+C26LLlVMXZcIl9BOrMZ6O\\\\\\\/587ayIjk+dY8tsxvYC3njCnZsYP2W096qlZ6dLYrhYNWqHpDMOoC\\\\\\\/SDrrCx3Izn55ffG27pSw7QHkw1TYPkLf1Fqytuv5dAJJz8HJDQr1AOCgubMyzkEEbCFeA12h7IzExTAsxwVVPnaLw\\\\\\\/HrwstTGDml\\\\\\\/OMlWkRReED1\\\\\\\/MivIUED+ZOf\\\\\\\/gRMwDyujduOktkIdUMNjAe03v0EmgbnWF6RQR7xMd4g2dauuSz9mCFUZhYRZpMvJEFbDueFQY\\\\\\\/4oQAY7mqBYuK09VQcjW\\\\\\\/oIF33zjAkALjzn24L3Ek79JV9gG18PcZDvX54pw6sxqX5oDSrNkPfmiSmi+b2OUa3lQ6km\\\\\\\/4c9bjyPHrTUwF6u0ZGKefY7iuIZ6xtG9s3uATPNxPwfZ13LgJdKx0UAjxoJmYwB2U3aRzHQUiFUm3B2emUm9hgGbyyfD+iIwoSlL03qffmpKfvl5ve4WkCQeZzlH2hJfRoC7TwEo7dpTk9SACfpo8lnQH87Wt\\\\\\\/5uEiyzE8YhIELbNhskLA9gQRmsQxQ3ZVBh2lc32c7mreDXlmGoq9+HS210X3Zu\\\\\\\/f6pHA\\\\\\\/0YN8cEO3RunBbPo0JG1yKOOQcCF\\\\\\\/JqH2IvgVhryBIk84WI9UbYphlHtaZz4Sv0W3+NfmhDvGi4aOU\\\\\\\/7KARUeX2IpVto6a\\\\\\\/pgQkP7LldVJtGB8kBs5KXmWDvIHv8uzsTu9LQffW2zk9WQjtgluGGUF8rvkq7KQfRwfKadsPe5UWX0WvCC+6ACawDlZSQQ==\\\u0022,\\\u0022iv\\\u0022:\\\u00220G7PU9gqwwgEO7Ur1k51Pg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00227cca27968549ad7e1c43be971236d410ff8252c5c5afe0899b1c19c1accfb209\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022GjjrHWU6WU9tM6Cl4hawFJ\\\\\\\/p6RuCaYqgzdtFM7e2mztF1u1IFbdSnnU\\\\\\\/xw6jN0WdCJ7tqJomlNZEI\\\\\\\/A81TIK6tPad\\\\\\\/9SyMfMjlhN0nTIUKEDpChJlQgODk1Mle4v\\\\\\\/MFutFFYtHAnU4VEvKFrkJ07mo7TREHCiShiukI0CxKRddBEAiyB4wYdcfkm6KT0\\\\\\\/tjUCYA4Sd2lVtl8nCEOq+B5PFYn4VKuas4NKoE+J6NQrjGkjXXlX0MaiKp8BPXauleZ27fbwosvtxLKqlrwSk15RykB1JrbaVMTL2+9vs4l+Eg3XoSX5Fd+uINX3xByRoYuNR50Lo+emvHOtfD6NVKM9lBGOxPYuIVxP0F+ftemGkleJnQh+Plwj1Xnb33h8cs2Qftm2uDwq9QQHI3Us1Ld7GvXLRSVZ9Y6teRfitf2xG7EP6gyvgh1QGCGnwG7NButut8YkeOVovw7PVX+g3ziHsTWAI38G6TjZq7ygCeA8cSZJX28FVPpeMKwnVp4xuZHEFIpnnuF7vyvcw5QnWwejzoohcPk5fLbIv7XMGGsvzD+AbUVQ3p9T4aGCIFaqjtaWcohfwiOJb7\\\\\\\/rb+dY0\\\\\\\/wJK0oNtlW0mLfXJfaJk0UeOUR9fX4S6e8vqHllBUJGUHu9BoayWAyvjHbNTRQK75Gbzv40mxBT6Q0\\\\\\\/\\\\\\\/T+F6uPExlcxh8NjUHqEm+Jyi8GnCy+m7rZWAC+w2sm7iEdgYwxIk9lLB2\\\\\\\/CMl7FqBxbkUqXEMdYW2DUvM3p6tcRt2\\\\\\\/x8sHvgFrrAx\\\\\\\/ytL8vgJg2yX+RNmQYgflmjJUmA8eKrQp00BZW4+IaAP3sn\\\\\\\/HK1lCIhza3ZWviiIxocpllnJDDG9QVjEoH2TAbyrnd4w2M0eOINFiBFOw8ayhNfqBrtZq0xnUv1vIOD2KVlA0w\\\\\\\/5uoecJF9AX8wYoIk5+6i468tOyoIKIpCvkkxt+yxDapk9CVYm4DUcJdNDjyhdXAO16H94KlNa6FhrMGrUdUDCJeDYZmSXWXelFSgsKf3bDW4CesjPUCyRc2ItqyZg+S5gGsfFkTQBj1El0XLsSi5Fz6pQEnvxB+rxhYlPZGvT+eF5a7QKxSrzhvhytWBFna4cgBkSRmESrDsl1wJo386GRxXpLeJr1ch6jXKOUhLBVagRh4GmSDbmzoypW4H\\\\\\\/aDr0IIm7hbqfDqMele3N8WaVI9O9qboGoThhg7cVKD8Zb0cmbs6RmF2whEwWY6SpWuka0Nra9Bagx\\\\\\\/NwUPBDRDN7Pm4hl3wZnhr88AGW\\\\\\\/Y71rUYhaYvtZZVIPCRet0pdZoDRyS7Mgnf4IpCd9tFGwTYn9PqxbGgiUXIOnCWdDmfZR0Un1bZTCSZaFGwRYbva5l9MCL+RBL5SNbFFPO8Wa9l1vE6NEIB0s1zETsUYVncqXdcq9+AJxZw\\\\\\\/HynlmRhyg62hlE4RRtNZ3NFxN3ChwV8qyuU98+b4l452lHnDCkaUYgq1bwVoE9k0\\\\\\\/koYlSAQ40aVrnad3e89lbeN0iUueeAnMe\\\\\\\/y8n5ZDkFgwnQVI8XCVG1+mfywfLl5tUn4PK9GSQKZJWijrQXPME9xGn\\\\\\\/hQ4YlJ60E9sx8ksqUa49AZWSoO7jtesC1AZA2YBqXcBd95TVbeZWj+EQtKLWibeX3uAzNw6C847xLFDota9614RVkzOmQGnARUeTCvE+Q+0qZUoV77dd2NYnmV5oNedcMYppZpAKTSGDQODtjA1FnR7XdlUB9t+j17R9nRfQjdnAhZ3pScqb3eeTR6ACPK+kd5Qx57HG+0MYJABk6edeFmxcAgk6LZJpAZQsFyim46Nez4Ov2eHUX+wpwzFAaYjHKhVq3g5jMbGDZ\\\\\\\/qUZZs\\\\\\\/162XzoEokJUMiWD+KHSWjbU202sEjtNYTQZ4DAcllGf\\\\\\\/LZ2X\\\\\\\/l18JvJauD19t2+KX04J7hyf+GlKDWMMuUXQZKf2z\\\\\\\/Sz8YbmcEm3QdH+ZR3PNqnmSq0fk9NEqNjosw6FLqogQl\\\\\\\/EPgLhwBu+Vtx4aINYSNqrUVnHMC8g==\\\u0022,\\\u0022iv\\\u0022:\\\u0022IJw3DNachPG0KrHJwPKn3w==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00228c2389a73339e8893b2d480f6cc627f4db4859996c4117e6ad5164fb48c2a4e6\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-494295 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-genome-report">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/10/3057" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-yyq0wOOY6dSTNdukCPxRZZJuBT2vOl09MrvSE3C1b9o" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/10/3057" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-G-vH3CWTqFDSRNkJPcCFid6ZfDYkEyLf9LHW9_ndL-w" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/9/3045" title="Specific Gene Disruption in the Major Livestock Pests Cochliomyia hominivorax and Lucilia cuprina Using CRISPR/Cas9" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/10/3067" title="Greenlip Abalone (Haliotis laevigata) Genome and Protein Analysis Provides Insights into Maturation and Spawning" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494295" id="node494295" data-pisa="ggg;9/10/3057" data-pisa-master="ggg;g3.119.400563" data-apath="/ggg/9/10/3057.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Whole Genome Sequence of the Commercially Relevant Mushroom Strain <em>Agaricus bisporus</em> var. <em>bisporus</em> ARP23</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-4626-5605" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Eoin O’Connor</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-4240-5175" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jamie McGowan</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-3880-9978" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Charley G. P. McCarthy</span>, <span class="highwire-citation-author" data-delta="3">Aniça Amini</span>, <span class="highwire-citation-author" data-delta="4">Helen Grogan</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0001-7345-6998" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>David A. Fitzpatrick</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3057-3066;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400563 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494295" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Eoin O’Connor </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Genome Evolution Laboratory, Department of Biology, Maynooth University, Maynooth, Co. Kildare, Ireland,</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Teagasc Food Research Centre, Ashtown, Dublin 15, D15 KN3K, Ireland,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Eoin%2BO%E2%80%99Connor%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=O%E2%80%99Connor%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AEoin%2BO%25E2%2580%2599Connor%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-4626-5605" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Eoin O’Connor</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jamie McGowan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Genome Evolution Laboratory, Department of Biology, Maynooth University, Maynooth, Co. Kildare, Ireland,</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Human Health Research Institute, Maynooth University, Maynooth, Co. Kildare, Ireland, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jamie%2BMcGowan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=McGowan%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJamie%2BMcGowan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-4240-5175" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jamie McGowan</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Charley G. P. McCarthy </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Genome Evolution Laboratory, Department of Biology, Maynooth University, Maynooth, Co. Kildare, Ireland,</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Human Health Research Institute, Maynooth University, Maynooth, Co. Kildare, Ireland, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Charley%2BG.%2BP.%2BMcCarthy%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=McCarthy%20CG&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ACharley%2BG.%2BP.%2BMcCarthy%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-3880-9978" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Charley G. P. McCarthy</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Aniça Amini </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Sylvan-Somycel (ESSC - Unité 2), ZI SUD, rue Lavoisier, BP 25, 37130 Langeais, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ani%C3%A7a%2BAmini%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Amini%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAni%25C3%25A7a%2BAmini%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Helen Grogan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Teagasc Food Research Centre, Ashtown, Dublin 15, D15 KN3K, Ireland,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Helen%2BGrogan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Grogan%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHelen%2BGrogan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">David A. Fitzpatrick </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Genome Evolution Laboratory, Department of Biology, Maynooth University, Maynooth, Co. Kildare, Ireland,</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Human Health Research Institute, Maynooth University, Maynooth, Co. Kildare, Ireland, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BA.%2BFitzpatrick%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fitzpatrick%20DA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADavid%2BA.%2BFitzpatrick%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-7345-6998" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for David A. Fitzpatrick</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:david.fitzpatrick@mu.ie" class="" data-icon-position="" data-hide-link-title="0">david.fitzpatrick@mu.ie</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/9/10/3057" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:494295" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:494295/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/10/3057.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:494295" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:494295/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/10/3057.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:494295" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:494295/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/9/10/3057.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:494295" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:494295/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2"><em>Agaricus bisporus</em> is an extensively cultivated edible mushroom. Demand for cultivation is continuously growing and difficulties associated with breeding programs now means strains are effectively considered monoculture. While commercial growing practices are highly efficient and tightly controlled, the over-use of a single strain has led to a variety of disease outbreaks from a range of pathogens including bacteria, fungi and viruses. To address this, the Agaricus Resource Program (ARP) was set up to collect wild isolates from diverse geographical locations through a bounty-driven scheme to create a repository of wild <em>Agaricus</em> germplasm. One of the strains collected, <em>Agaricus bisporus</em> var. <em>bisporus</em> ARP23, has been crossed extensively with white commercial varieties leading to the generation of a novel hybrid with a dark brown pileus commonly referred to as ‘Heirloom’. Heirloom has been successfully implemented into commercial mushroom cultivation. In this study the whole genome of <em>Agaricus bisporus</em> var. <em>bisporus</em> ARP23 was sequenced and assembled with Illumina and PacBio sequencing technology. The final genome was found to be 33.49 Mb in length and have significant levels of synteny to other sequenced <em>Agaricus bisporus</em> strains. Overall, 13,030 putative protein coding genes were located and annotated. Relative to the other <em>A. bisporus</em> genomes that are currently available, <em>Agaricus bisporus</em> var. <em>bisporus</em> ARP23 is the largest <em>A. bisporus</em> strain in terms of gene number and genetic content sequenced to date. Comparative genomic analysis shows that the <em>A. bisporus</em> mating loci in unifactorial and unsurprisingly highly conserved between strains. The lignocellulolytic gene content of all <em>A. bisporus</em> strains compared is also very similar. Our results show that the pangenome structure of <em>A. bisporus</em> is quite diverse with between 60–70% of the total protein coding genes per strain considered as being orthologous and syntenically conserved. These analyses and the genome sequence described herein are the starting point for more detailed molecular analyses into the growth and phenotypical responses of <em>Agaricus bisporus</em> var. <em>bisporus</em> ARP23 when challenged with economically important mycoviruses.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/button-mushroom" class="hw-term hw-article-keyword hw-article-keyword-button-mushroom" rel="nofollow">Button mushroom</a></li><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/agaricus-bisporus" class="hw-term hw-article-keyword hw-article-keyword-agaricus-bisporus" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Agaricus bisporus</em></a></li><li class="kwd"><a href="/keyword/genome-report" class="hw-term hw-article-keyword hw-article-keyword-genome-report" rel="nofollow">Genome report</a></li><li class="kwd"><a href="/keyword/agaricus-resource-program" class="hw-term hw-article-keyword hw-article-keyword-agaricus-resource-program" rel="nofollow">Agaricus Resource Program</a></li><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/agaricus-bisporus-mating-locus" class="hw-term hw-article-keyword hw-article-keyword-agaricus-bisporus-mating-locus" rel="nofollow">Agaricus <em xmlns:default="http://www.w3.org/1999/xhtml">bisporus</em> mating locus</a></li><li class="kwd"><a href="/keyword/agaricus-pangenome" class="hw-term hw-article-keyword hw-article-keyword-agaricus-pangenome" rel="nofollow">Agaricus pangenome</a></li></ul><p id="p-3">The global market for edible mushrooms is estimated to be worth US$42 billion per year (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Prescott <em>et al.</em> 2018</a>). Due to its high yielding potential and appealing morphology the mushroom industry has relied solely on a single white variety of <em>A. bisporus</em>. However, the over-use of a single cultivar has led to a variety of disease outbreaks from a variety of pathogens (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Hussey and Wyatt 1960</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Fletcher 1992</a>; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">North and Wuest 1993</a>; <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Soler-Rivas 1999</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Grogan <em>et al.</em> 2003</a>). One approach to increase disease resistance profiles in important crops is the use of desirable traits from wild germplasms (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Arora <em>et al.</em> 2019</a>). However, breeding of new strains in the mushroom industry has been difficult. Notwithstanding these difficulties the introduction of desirable traits to novel cultivars and the need for genomic biorepositories of wild germplasms of closely related or ancestral agricultural crops remains crucial for the protection of low diversity crops (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Dempewolf <em>et al.</em> 2017</a>).</p><p id="p-4"><em>A. bisporus</em> has effectively been considered a monoculture crop for a considerable amount of time (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Royse and May 1982</a>). To this end, the <em>Agaricus</em> Resource Program (ARP) (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Callac <em>et al.</em> 1996</a>) was set up to collect wild isolates from diverse geographical locations. The focus of this genome report is a wild isolate from the ARP collection referred to as <em>A. bisporus</em> var. <em>bisporus</em> ARP23. Crosses of homokaryons of U1 and old-fashioned brown led to an intermediate hybrid that was subsequently crossed with ARP23 to produce a novel commercially productive hybrid referred to as ‘Heirloom’. Wild strains are also a promising resource for the introduction of disease-resistant traits for common commercial mushroom diseases. It has been shown that the wild tetrasporic <em>A. bisporus</em> var. <em>burnettii</em> has heightened resistance to the pathogen that causes bacterial blotch (<em>Pseudomonas tolaasii</em>) through genetic markers linked to the <em>PPC1</em> allele (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Moquet <em>et al.</em> 1999</a>). Polygenic inheritance of resistance attributes has been described (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Kerrigan 2000</a>) and so the consideration of the introduction of wild <em>A. bisporus</em> germplasm into breeding novel strains must be considered on the basis of careful selection of screened wild strains with distinct mechanisms pertaining to disease resistance (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Foulongne-Oriol <em>et al.</em> 2011</a>).</p><p id="p-5">To date, two genomes of the constituent homokaryons of Horst U1 have been sequenced, H97 (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Morin <em>et al.</em> 2012</a>) and H39 (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Sonnenberg <em>et al.</em> 2016</a>). This represents the genome of the first commercially cultivated white hybrid strain. The manuscript that reported the H97 genome sequence also described the genome of <em>Agarius bisporus</em> var. <em>burnetti</em> (JB137-S8), a strain exclusively native to the Sonoran Desert of California. That study uncovered the genetic and enzymatic mechanisms that favor <em>A. bisporus</em> to a humic-rich environment by primary degradation of plant material. A gene arsenal of compost-induced carbohydrate enzymes (heme-thiolate peroxidase, β-etherases, multicopper oxidase) and CYP450 oxidoreductases for example, together with high protein degradation and nitrogen-scavenging abilities were determined to be crucial to the challenges posed by complex composts (<a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">Morin <em>et al.</em> 2012</a>).</p><p id="p-6">The genome presented herein, represents the first commercially relevant genome for a wild cultivar of <em>Agaricus bisporus</em> var. <em>bisporus</em>, and is an invaluable tool for future efforts in mushroom breeding. Furthermore the genome sequence described will act as the starting point for more detailed OMIC based studies into the growth and phenotypical responses of <em>Agaricus bisporus</em> var. <em>bisporus</em> ARP23 when challenged with economically important mycoviruses.</p><div class="section methods" id="sec-1"><h2 class="">Methods</h2><div id="sec-2" class="subsection"><h3>Strain, culture conditions and homokaryon genotyping</h3><p id="p-7"><em>A. bisporus</em> var. <em>bisporus</em> ARP23 cultures were grown on compost extract agar (aqueous extract of phase II mushroom compost, double-autoclaved for sterility) for three weeks at 25° in the dark. 10 ml of protoplasting medium (50 mM maleic acid, 0.6 M saccharose, 1 M NaOH pH 5.8) containing 10 g Glucanex (Sigma Aldrich cat. no. L1412) (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Kerrigan <em>et al.</em> 1994</a>) was added to established mycelium and incubated for 5 hr at 25° in the dark with occasional shaking. Resulting protoplasts were grown on a saccharose-enriched compost extract medium and incubated for two weeks at 25° in the dark. To assess whether protoclones were homokaryons or heterokaryons, growth tests, hyphal morphology, and nuclear type were conducted. Cultures that had an average diameter of 54 mm (<a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Kerrigan <em>et al.</em> 1994</a>) or less after this time, were isolated as putative-homokaryons. A total of 86 out of 100 isolated protoplasts were assigned as putative homokaryons due to their delayed growth. Hyphae of homokaryons tend to be more ‘thread-like’ with far fewer branching hyphae. To confirm that protoplasts were homokaryons, the presence of a single MAT locus was assessed using the 39Tr 2/5-2/4 primer sets for targeting MAT loci and amplification was carried out as previously described (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Gao <em>et al.</em> 2013</a>). Genotype-validated homokaryons were grown on complete yeast media (2 g proteose peptone, 2 g yeast extract, 20 g glucose, 0.5 MgSO<sub>4</sub>, 0.46 g KH<sub>2</sub>PO<sub>4</sub>, 1 g K<sub>2</sub>HPO<sub>4</sub>, 10 g agar in 500 ml dH<sub>2</sub>0) for 3 weeks in the dark, at 25°. 7 mm agar plugs were excised from the growing hyphal edges of homokaryons of MAT2 genotype (the MAT1 genotype was not recovered) and shaken at 30 Hz for 7 min in 500 µL malt extract and the resulting homogenate added to 50 ml malt extract liquid medium (10 g malt extract in 600 ml of dH<sub>2</sub>O) in a 500 ml Erlenmeyer flask. Liquid cultures were grown for 12 days at 25° in the dark at 150 rpm.</p></div><div id="sec-3" class="subsection"><h3>DNA isolation and libraries</h3><p id="p-8">Fungal mycelium was isolated in Miracloth and washed with sterile PBS. The mycelium was flash frozen and ground in liquid nitrogen using a mortar and pestle. DNA isolation was carried out immediately on ground material with the Wizard Genomic DNA Purification Kit (Promega) following the plant tissue method with minor modifications. Nuclei lysis buffer was supplemented with 0.5 M EDTA and 0.1 mg/mL Proteinase K and cell lysis was carried out at 37° for 30 min. The remainder of the DNA isolation was as per manufacturer’s guidelines. An Illumina paired-end sequencing library with insert size of 270 bp (80 X coverage) and a Pacbio RSII mate pair library of 20 Kb insert size were generated for a hybrid assembly approach. Sequencing on HiSeq 4000 and Pacbio RSII generated 5.90 GB and 57.64 GB of raw sequence data, respectively. DNA library construction and sequencing on the Pacbio (RSII) and Illumina (HiSeq 4000) platforms was carried out by BGI Tech Solutions Co., Ltd. (Hong Kong, China).</p></div><div id="sec-4" class="subsection"><h3>Fruit-body material, RNA isolation and sequencing</h3><p id="p-9">ARP23 mycelia was added to mushroom compost in crates (<em>n</em> = 3) and incubated at 25°, 90–95% relative humidity (spawn-run phase) for 17 days. A layer of peat was added to the surfaces of the colonized compost (case-run phase) and incubated for another 7 days. Temperatures and relative humidity were lowered to 18° and 85–90% and fruit-bodies were harvested after 7 days of development. All cropping procedures were as per standard mushroom growing practices. Fruit-bodies were flash-frozen, freeze-dried and material was crushed in liquid N<sub>2</sub>. RNA was isolated using the RNeasy plant minikit (Qiagen) as per manufacturers guidelines. DNA digestion was done with DNase I (Invitrogen). RNA quantity and quality was assessed with an RNA6000 Nano Assay (Agilent 2100 Bioanalyzer, Agilent Technologies, USA). High-quality RNA was sent to BGI Tech Solutions Co., Ltd. (Hong Kong, China) for RNA sequencing (RNA-seq).</p></div><div id="sec-5" class="subsection"><h3>Genome assembly and gene calling</h3><p id="p-10">Short-read libraries had adaptor removal (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Martin 2011</a>) and quality trimming performed using Trim Galore! (<a href="https://github.com/FelixKrueger/TrimGalore">https://github.com/FelixKrueger/TrimGalore</a>). A minimum phred score cut-off of 25 and a minimum read length of 90 nt was applied to short-read libraries. Long-read libraries were corrected using short reads using Proovread (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Hackl <em>et al.</em> 2014</a>). Corrected long reads were then self-corrected with additional adaptor trimming using Canu (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Koren <em>et al.</em> 2017</a>). Canu was then used for genome assembly with an error rate of 2.5%. Scaffolding of assembled contigs was performed by scaffolding with corrected long reads using SSPACE long-read hybrid assembler (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Boetzer and Pirovano 2014</a>). This primary assembly was then processed using Purge Haplotigs (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Roach <em>et al.</em> 2018</a>) for removal of duplicated haplotypes, with default parameters. These steps were carried out as a precaution to remove potential areas of heterozygosity in the assembly introduced by sequencing data collected from different MAT2 homokaron protoplasts, which were pooled for DNA isolation, due to their excessively slow growth rates in culture. Levels of heterozygosity were not explicitly examined however. Jellyfish v. 1.1.12 (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Marçais and Kingsford 2011</a>) was used to generate histograms of the frequency distribution of kmers from Illumina reads which were fed into GenomeScope (<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Vurture <em>et al.</em> 2017</a>) for estimation of genome size. Synteny between the genome assemblies of ARP23, JB137-S8 and H97 was assessed by performing global whole genome alignments using BWA (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Li and Durbin 2010</a>) and visualized with Jupiter Plot (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Chu 2018</a>). The H39 assembly wasn’t included as it has been shown to have almost complete synteny to H97 (<a id="xref-ref-58-2" class="xref-bibr" href="#ref-58">Sonnenberg <em>et al.</em> 2016</a>).</p><p id="p-11">For gene calling purposes, RNAseq data from fruitbodies with a read length of 100 nt (phred score &gt; 25) were aligned to the final genome assembly using Bowtie 2 (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Langmead and Salzberg 2012</a>). Resulting alignment files were used to train AUGUSTUS using Braker2 (<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Stanke <em>et al.</em> 2006</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Hoff <em>et al.</em> 2016</a>). The completeness of the predicted gene models were assessed using BUSCO with the Basidiomycota BUSCO dataset (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Simão <em>et al.</em> 2015</a>).</p></div><div id="sec-6" class="subsection"><h3>Genome functional annotation and characterization</h3><p id="p-12">Putative open reading frames (ORFs) were assigned protein family (PFam) domains using funannotate (<a href="https://github.com/nextgenusfs/funannotate">https://github.com/nextgenusfs/funannotate</a>) using default settings of Interproscan 5 (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Jones <em>et al.</em> 2014</a>). Gene ontology (GO) IDs (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Ashburner <em>et al.</em> 2000</a>) were assigned where available and a corresponding GO term map was obtained using YeastMine (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Balakrishnan <em>et al.</em> 2012</a>). Information on the pathways associated with different genes were analyzed with KEGG (Kyoto Encyclopedia of Genes and Genomes) (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Ogata <em>et al.</em> 1999</a>) by assigning KO (KEGG ontology) through BlastKOALA (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Kanehisa <em>et al.</em> 2016</a>). A search for repetitive elements was done by identifying tandem repeats (TR) and transposable elements (TE). TRs were identified over the entire assembly with Tandem Repeats Finder (TRF 4.07) (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Benson 1999</a>). Classification of the different categories of TE in the genome were conducted using RepeatMasker 4.06 with the modified version of NCBI Blast for RepeatMasker, RMBLAST (<a href="http://www.repeatmasker.org/RMBlast.html">http://www.repeatmasker.org/RMBlast.html</a>). Simple single repeats were also determined using RepeatMasker with the default settings. tRNAs were identified across all scaffolds using tRNAscan-SE v 2.0 (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Lowe and Chan 2016</a>) and rRNAs were also identified with RNAmmer 1.2 (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Lagesen <em>et al.</em> 2007</a>).</p></div><div id="sec-7" class="subsection"><h3>Carbohydrate-active enzymes</h3><p id="p-13">Translated ORFs were used to search dbCAN2 (<a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Zhang <em>et al.</em> 2018</a>) for presence of Carbohydrate Active Enzymes (CAZys) (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Lombard <em>et al.</em> 2014</a>). For comparative purposes 32 fungal genomes consisting of a variety of Ascomycota and Basidiomycota species (Table S1) were also searched to catalog their CAZy content. As well as including brown rot and white rot fungi, genomes for 7 of the top 10 most highly cultivated mushrooms are also included in this dataset (Table S1).</p></div><div id="sec-8" class="subsection"><h3>Mating locus</h3><p id="p-14">The locus coding for homeodomain proteins typical for the A mating type in <em>Coprinopsis cinerea</em> was used to locate the unifactorial <em>A. bisporus</em> mating-type locus. The homeodomain proteins as well as flanking proteins were individually searched using BLASTP (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Altschul <em>et al.</em> 1997</a>) (evalue 10<sup>−3</sup>) against the predicted proteomes of <em>A. bisporus</em> ARP23, H97 and JB137-S8 respectively. Top hits from these gene sets were then searched back against the <em>C. cinerea</em> gene set to locate reciprocal best BLAST hits which were then considered orthologs.</p></div><div id="sec-9" class="subsection"><h3>Phylogenomic reconstruction</h3><p id="p-15">Orthologous gene families were identified with OrthoFinder2 (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Emms and Kelly 2015</a>), using BLASTp (<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Altschul <em>et al.</em> 1997</a>) as the search algorithm, an inflation value of 2.0 for MCL clustering (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Enright <em>et al.</em> 2002</a>) and the command-line parameter “-msa”. 71 gene families were ubiquitously present and single copy and used for phylogenomic analysis. Each family was individually aligned using MUSCLE (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Edgar 2004</a>) and trimmed using trimAl (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Capella-Gutierrez <em>et al.</em> 2009</a>) with the parameter “-automated1” to remove poorly aligned regions. Trimmed alignments were concatenated together resulting in a final supermatrix alignment of 27,861 amino acids. Phylogenomic analyses were performed using both Maximum likelihood and Bayesian inference. IQ-TREE (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Nguyen <em>et al.</em> 2015</a>) was used to perform maximum likelihood analysis under the LG+F+R5 model, which was the best fit model according to ModelFinder (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Kalyaanamoorthy <em>et al.</em> 2017</a>), and 1,000 ultrafast bootstrap replicates (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Hoang <em>et al.</em> 2018</a>). Bayesian analyses were carried out using PhyloBayes with the CAT model (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Lartillot <em>et al.</em> 2009</a>). Two independent chains were run for 8,000 cycles and convergence was assessed using bpcomp and tracecomp. A consensus Bayesian phylogeny was generated with a burn-in of 10%. Support values represent posterior probabilities. The phylogeny was visualized and annotated using the Interactive Tree of Life (iTOL) (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Letunic and Bork 2007</a>).</p></div><div id="sec-10" class="subsection"><h3>Agaricus bisporus pangenome dataset assembly</h3><p id="p-16">Genome assembly data for three <em>Agaricus bisporus</em> strains (H97, JB137 and H39) were obtained from NCBI. Gene sequence and genomic location datasets were generated for each of the three strains through the pangenome analysis pipeline Pangloss; a gene prediction strategy using a combination of HMM-dependent gene prediction with GeneMark-ES and PWM-dependent long ORF prediction was chosen (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">McCarthy and Fitzpatrick 2019b</a>). Combined with data from <em>A. bisporus</em> ARP23, a total of 42,264 <em>A. bisporus</em> gene sequences and their corresponding genomic locations were predicted. An all-<em>vs.</em>-all BLASTp search was performed on the <em>A. bisporus</em> dataset using an e-value cutoff of 1e<sup>-4</sup> (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Camacho <em>et al.</em> 2009</a>).</p></div><div id="sec-11" class="subsection"><h3>Data availability</h3><p id="p-17">The Bioproject designation for this project is PRJNA544931. This Whole Genome Shotgun project has been deposited at DDBJ/ENA/GenBank under the accession VCNO00000000. The version described in this paper is version VCNO01000000.</p><p id="p-18">Table S1 shows the genomes, taxonomy and download links for the 32 genomes used in the phylogenomic and CAZy studies. Table S2 shows the presence of lignocellulolytic genes in the 32 fungal genomes. Figure S1: Macrosynteny between <em>A. bisporus</em> H97 chromosomes and all <em>A. bisporus</em> ARP23 scaffolds. Only regions larger than 10,000bp are connected with links. Macrosynteny visualized with Jupiter Plot. Supplemental material available at FigShare: <a href="https://doi.org/10.25387/g3.9076991">https://doi.org/10.25387/g3.9076991</a>.</p></div></div><div class="section results" id="sec-12"><h2 class="">Results</h2><div id="sec-13" class="subsection"><h3>Whole-genome assembly</h3><p id="p-19">The genome of the monokaryotic <em>A. bisporus</em> var. <em>bisporus</em> strain ARP23 (ARP23 herein) was sequenced using a hybrid approach of short (Illumina Hiseq 4000) paired-end reads and long-reads (Pacbio RSII). A total of 8,861,726 reads representing a cumulative size of 4.074 GB were generated including 8,424,105 and 437,621 reads from Illumina and PacBio sequencing platforms respectively. Upon trimming adaptors, error-correction and hybrid-assembly of both short and long-read libraries, a 33.49Mb genome with a GC content of 46.33% was generated. The assembly is comprised of 169 contigs, the longest being 1.5Mb with an N50 of 350,711 and an L50 of 26 (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). The average length of contigs is 198,204.6 bp. Kmer-analyses conducted with GenomeScope (<a id="xref-ref-61-2" class="xref-bibr" href="#ref-61">Vurture <em>et al.</em> 2017</a>) suggest a genome size of 34.02Mb indicating that the 169 scaffolds of this assembly cover 98.44% of the entire genome. The completeness of the assembly was quantified by determining the presence/absence of the 1,315 fungal orthologs found in the Basidiomycete BUSCO set. A total of 1,170 (87.6%) complete BUSCO genes were located in the ARP23 assembly, this is comparable to what is observed in A. <em>bisporus</em> H97 (1,172 or 87.8%) and <em>A. bisporus</em> JB137-S8 (1,179 or 88.3%) strains (H97 and JB137-S8 respectively herin) (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). Macrosynteny between the ARP23 assembly and the 13 complete H97 chromosomes was visualized with Jupiter Plot. Overall high levels of synteny are observed with the vast majority of ARP23 scaffolds mapping directly to individual H97 chromosome <strong>(</strong><a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a> &amp; Figure S1<strong>).</strong> There are a number of scaffolds that have hits to multiple chromosomes indicating low levels of genome rearrangements have occurred (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a> &amp; Figure S1). There are no scaffolds in the ARP23 that do not map to the H97 assembly. Scaffold 74 was found to be 137,116 nucleotides in length and contains all 17 mitochondrial genes previously described in the mitochondrion of H97 (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Férandon <em>et al.</em> 2013</a>). Furthermore, it is also of a comparable length to the H97 mitochondrial genome (135,005 bp), therefore scaffold 74 corresponds to the full length ARP23 mitochondrial genome.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/494486/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/494486/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Genome statistics for <em>A. bisporus</em> strains ARP23, H97 and JB137-S8</span><div class="sb-div caption-clear"></div></div></div><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/10/3057/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Macrosynteny between A. bisporus H97 chromosomes and A. bisporus ARP23 scaffolds. Only regions larger than 10,000bp are connected with links. Macrosynteny visualized with Jupiter Plot. For display purposes only the largest scaffolds that correspond to 75% of the ARP23 assembly are incorporated. When all scaffolds are included higher levels of coverage are observed particularly with respect to H97 chromosomes 1 and 13 (Figure S1)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1159653199" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Macrosynteny between &lt;em&gt;A. bisporus&lt;/em&gt; H97 chromosomes and &lt;em&gt;A. bisporus&lt;/em&gt; ARP23 scaffolds. Only regions larger than 10,000bp are connected with links. Macrosynteny visualized with Jupiter Plot. For display purposes only the largest scaffolds that correspond to 75% of the ARP23 assembly are incorporated. When all scaffolds are included higher levels of coverage are observed particularly with respect to H97 chromosomes 1 and 13 (Figure S1).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/10/3057/F1.medium.gif" width="427" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/9/10/3057/F1.medium.gif" width="427" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/10/3057/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/10/3057/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494517" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-21" class="first-child">Macrosynteny between <em>A. bisporus</em> H97 chromosomes and <em>A. bisporus</em> ARP23 scaffolds. Only regions larger than 10,000bp are connected with links. Macrosynteny visualized with Jupiter Plot. For display purposes only the largest scaffolds that correspond to 75% of the ARP23 assembly are incorporated. When all scaffolds are included higher levels of coverage are observed particularly with respect to H97 chromosomes 1 and 13 (Figure S1).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-14" class="subsection"><h3>Genome annotation</h3><p id="p-22">The total length of repetitive elements in <em>A. bisporus</em> strain ARP23 amounted to 264,404 bp (0.79% of the total assembly) this is similar to what is observed for both H97 and JB137-S8 (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>). The number of tandem repeat regions across the 169 scaffolds of the assembled genome was 2,287, with ‘scaffold00003’ containing the greatest number at 321 repeat regions. Simple sequence repeats (SSRs) amounted to 180,995 bp (0.54%) and low complexity regions covered 44,108 bp (0.13%) (<a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>). We searched for the two sub-types of Non-LTR retrotransposons; long interspersed nuclear elements (LINEs) and short interspersed nuclear elements (SINEs). Of the 241 LINE-type elements; 17 belonged to L1, 69 to L2 and 78 to L3. For the 30 regions designated to SINES; 9 belonged to the MIRs and none were classified as ALUs. A single LTR retrotransposon was found for the endogenous retroviruses (ERV) class I. A single hAT-Charlie family DNA transposon and 7 DNA/TcMar-Tiggers were found out of a total of 55 DNA elements. Overall the ARP23 assembly contains 200 tRNAs, 10 rRNAs and 93 sRNAs (<a id="xref-table-wrap-1-5" class="xref-table" href="#T1">Table 1</a>).</p><p id="p-23">In total 13,030 putative protein-coding genes were called for ARP23. Of these 7,725 (59.3%) were annotated with PFam domains. An analysis of the presence of the BUSCO set of orthologous genes for Basidiomycetes in the ARP23 gene set revealed that is missing 40 (3%) of the BUSCO genes. This is comparable to the predicted proteomes of H97 and JB137-S8 which are both missing 39 (2.9%) of the BUSCO genes. ARP23 gene calling incorporated RNAseq data from fruitbodies, the final gene set showed evidence that 775 (5.95%) of protein coding genes are alternatively spliced. Overall the number of genes predicted for ARP23 is larger than that for H97 (10,863) and JB137-S8 (11,289) (<a id="xref-table-wrap-1-6" class="xref-table" href="#T1">Table 1</a>). The average number of introns per ORF for the ARP gene set is 5.4, this number is larger than that observed in H97 (4.6) and JB137-S8 (4.7) respectively (<a id="xref-table-wrap-1-7" class="xref-table" href="#T1">Table 1</a>). High-level functional annotations were assigned for predicted genes using BlastKOALA (<a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Kanehisa <em>et al.</em> 2016</a>). Of the 13,030 predicted ARP23 genes, KO assignments were made for 3,808 (29.22%). General functions and protein families relating to genetic information processing accounted for the majority of KEGG annotation with 1,693 (45.46%) of proteins falling into these categories. Other pathways highly represented were carbohydrate metabolism with 313 (8%) and environmental information processing with 176 (5%). A total of 750 secreted proteins were predicted by assigning signal peptides in SignalP v. 5. Putatively secreted proteins with a transmembrane domain downstream of the N-terminus signal peptide were excluded (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Sonnhammer <em>et al.</em> 1998</a>), with the subsequent prediction of 606 proteins. Secreted proteins involved in hydrolysis of glycoside (<em>n</em> = 54, GO:0004553), oxidation-reduction processes (<em>n</em> = 82, GO:0055114), and fungal hydrophobins (<em>n</em> = 20, PF01185) were highly represented.</p></div><div id="sec-15" class="subsection"><h3>Genome phylogeny</h3><p id="p-24">The availability of whole genomes permits the reconstruction of phylogenomic trees. From our dataset of 32 fungal genomes we located 71 ubiquitously distributed gene families. These were individually aligned and concatenated to give a supermatrix of 27,861 amino acids. Using this supermatrix, phylogenomic reconstruction analyses were performed using both Maximum likelihood and Bayesian inference (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). The resultant phylogeny successfully resolved strongly supported monophyletic clades for the Ascomycota and Basidiomycota phyla. It also resolved monophyletic clades for the Agaricales, Boletales and Polyporales orders within the Basidiomycota clade (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>). Within in the Agaricales order a strongly supported monophyletic Marasmioid clade containing <em>Schizophyllum commune</em>, <em>Moniliophthora roreri</em>, <em>Gymnopus luxurians</em> and <em>Lentinula</em> species is present. A monophyletic Agaricoid clade containing <em>Laccaria bicolor</em>, <em>Coprinopsis cinerea</em> and <em>Agaricus bisporus</em> is also present (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a>). The single Tricholomatoid clade species, <em>Hypsizygus marmoreus</em> is grouped beside the Agaricoid clade in agreement with previous studies (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Matheny <em>et al.</em> 2006</a>). However the two Pluteoid clade species, <em>Pleurotus ostreatus</em> and <em>Volvariella volvacea</em> are not grouped together, however it has been suggested that the Pluteoid clade may not be monophyletic (<a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Matheny <em>et al.</em> 2006</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/10/3057/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Supermatrix phylogeny of 32 fungal species (71 ubiquitous fungal gene families, 27,861 characters). Phylogenomic analyses were performed using both maximum likelihood (IQ-TREE with LG+F+R5 model) and Bayesian inference (PhyloBayes with the CAT model). Both phylogenies were identical except the maximum likelihood phylogeny grouped V. volvacea and H. marmoreus as sister taxa while the Bayesian phylogeny did not. Bayesian posterior probabilities and ultrafast bootstrap supports are indicated at all nodes. For comparative purposes the assembly size and number of gene models for each species are also shown." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1159653199" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Supermatrix phylogeny of 32 fungal species (71 ubiquitous fungal gene families, 27,861 characters). Phylogenomic analyses were performed using both maximum likelihood (IQ-TREE with LG+F+R5 model) and Bayesian inference (PhyloBayes with the CAT model). Both phylogenies were identical except the maximum likelihood phylogeny grouped &lt;em&gt;V. volvacea&lt;/em&gt; and &lt;em&gt;H. marmoreus&lt;/em&gt; as sister taxa while the Bayesian phylogeny did not. Bayesian posterior probabilities and ultrafast bootstrap supports are indicated at all nodes. For comparative purposes the assembly size and number of gene models for each species are also shown.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/10/3057/F2.medium.gif" width="440" height="215"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/9/10/3057/F2.medium.gif" width="440" height="215"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/10/3057/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/10/3057/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494366" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-25" class="first-child">Supermatrix phylogeny of 32 fungal species (71 ubiquitous fungal gene families, 27,861 characters). Phylogenomic analyses were performed using both maximum likelihood (IQ-TREE with LG+F+R5 model) and Bayesian inference (PhyloBayes with the CAT model). Both phylogenies were identical except the maximum likelihood phylogeny grouped <em>V. volvacea</em> and <em>H. marmoreus</em> as sister taxa while the Bayesian phylogeny did not. Bayesian posterior probabilities and ultrafast bootstrap supports are indicated at all nodes. For comparative purposes the assembly size and number of gene models for each species are also shown.</p><div class="sb-div caption-clear"></div></div></div><p id="p-26">With respect to the phylogenetic relationships between the <em>A. bisporus</em> strains, our phylogeny groups all three strains in a monophyletic clade with maximum Bayesian posterior probability (BPP) and bootstrap support (BP). Furthermore, H97 and ARP23 are grouped as sister taxa with relatively strong BPP and BP (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2</a>). This phylogenetic relationship infers that ARP23 is more closely related to H97 that it is to JB137-S8.</p></div><div id="sec-16" class="subsection"><h3>Carbohydrate-Active Enzymes (CAZys)</h3><p id="p-27"><em>A. bisporus</em> is adapted to growth in a humic-rich, leaf-litter environment. The genome of H97 has a carbohydrate-active enzyme gene (CAZyme) (<a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Lombard <em>et al.</em> 2014</a>) repertoire more similar to that of white- and brown-rot basidiomycetes as opposed to closer taxonomically-related species such as <em>C. cinerea</em> and <em>L. bicolor</em> (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2</a>) (<a id="xref-ref-46-3" class="xref-bibr" href="#ref-46">Morin <em>et al.</em> 2012</a>). Similar results were replicated in our analyses of the genomes of <em>A. bisporus</em> H97, JBS137-S8 and ARP23 (Table S2). A total of 411 putative CAZymes were found in the genome of ARP23 including 176 glycoside hydrolases, 60 glycosyl transferases and 14 carbohydrate-binding modules. In terms of lignocellulolytic genes the three strains of <em>A. bisporus</em> have very similar repertoires with ARP23, JB137-S8 and H39 having 149, 142 and 136 genes respectively. Specifically, ARP23 was found to contain 48 cellulases, 19 hemicellulases, 12 pectinases, 17 lignin oxidases and 53 lignocellulolytic auxiliary enzymes (Table S2).</p></div><div id="sec-17" class="subsection"><h3>Mating locus</h3><p id="p-28"><em>Agaricus bisporus</em> has a pseudo-homothallic life cycle with a unifactorial mating system (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Miller 1971</a>; <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Raper <em>et al.</em> 1972</a>). Pseudo-homothallism is a particular system in which automixis is forced, as two haploid nuclei from one meiotic tetrad are packaged together into one spore, therefore self-fertility is the result of the packaging of two independent and opposite mating type nuclei within a single spore (<a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Wilson <em>et al.</em> 2015</a>). This lifestyle not only enables the fungus to reproduce without finding a compatible partner, but also to cross with any compatible mate it may encounter (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Grognet and Silar 2015</a>). To date this type of reproduction in Basidiomycetes has only been observed in Agaricomycetes (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Nieuwenhuis <em>et al.</em> 2013</a>). It is possible that pseudo-homothallics benefit from both homothalism, which allows the possibility to self-cross when no compatible partner is present and heterothallism, which favors the creation of genetic variation through recombination during outbreeding (<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Grognet and Silar 2015</a>).</p><p id="p-29">The locus encoding the homeodomain proteins has previously been located on Chromosome 1 of <em>A. bisporus</em> H97 (<a id="xref-ref-46-4" class="xref-bibr" href="#ref-46">Morin <em>et al.</em> 2012</a>). Our analysis of the mating-type locus of <em>A. bisporus</em> ARP23, H97 and JB137-S8 shows they are all very similar to the A mating-type locus of the model species <em>Coprinopsis cinerea</em> and are located on scaffold 16, chromosome 1 and scaffold 1 respectively. The locus contains a pair of homeodomain transcription factor genes orthologous to b1-2 and a1-2 from <em>C. cinerea</em> (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). The mitochondrial intermediate peptidase (MIP) gene and a Beta-flanking gene which typically accompany the mating A locus are also found in the genomic vicinity (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>). Interestingly, all three <em>A. bisporus</em> strains have an additionally copy of the Beta-flanking gene relative to <em>C. cinerea</em>. Levels of synteny with respect to other flanking genes are also very high (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a>). The JB137-S8 assembly contains six ORFs between the homeodomain proteins, sequence analysis leads us to believe that this is a misassembly artifact, furthermore all ORFs have homologs in H97 and ARP23 but are located on different scaffolds. Relative to the other mating loci of the other two assemblies, H97 has an additional putative ORF (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3</a>). A homolog for this gene is absent for the assemblies of both ARP23 and JB137-SB, furthermore it does not contain any known Pfam domains and a BLASTP search against GenBank retrieves a single significant hit to another Agaricales species (<em>Leucoagaricus</em> sp) therefore it may be dubious.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/10/3057/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Distribution of genes in the mating type locus of A. bisporus H97, ARP23 and JB137-S8. For comparative purposes the mat A locus of the model species C. cinerea is also displayed. Chromosome or scaffold and relative genomic position of loci are indicated. Genes that are colored blue are orthologous in all four genomes and syntenically conserved. Red genes indicate syntenically conserved orthologs in A. bisporus strains but missing from C. cinerea. A. bisporus has a single pair of homeodomain genes (colored black) while C. cinerea has two pairs. Green genes in C. cinerea are absent from A. bisporus strains. A. bisporus H97 has a unique gene (colored yellow). Six ORFs are located between the homoedomain genes of A. bisporus JB137-S8 but are most likely the result of misassembly." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1159653199" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Distribution of genes in the mating type locus of &lt;em&gt;A. bisporus&lt;/em&gt; H97, ARP23 and JB137-S8. For comparative purposes the mat A locus of the model species C. &lt;em&gt;cinerea&lt;/em&gt; is also displayed. Chromosome or scaffold and relative genomic position of loci are indicated. Genes that are colored blue are orthologous in all four genomes and syntenically conserved. Red genes indicate syntenically conserved orthologs in &lt;em&gt;A. bisporus&lt;/em&gt; strains but missing from &lt;em&gt;C. cinerea&lt;/em&gt;. &lt;em&gt;A. bisporus&lt;/em&gt; has a single pair of homeodomain genes (colored black) while &lt;em&gt;C. cinerea&lt;/em&gt; has two pairs. Green genes in &lt;em&gt;C. cinerea&lt;/em&gt; are absent from &lt;em&gt;A. bisporus&lt;/em&gt; strains. &lt;em&gt;A. bisporus&lt;/em&gt; H97 has a unique gene (colored yellow). Six ORFs are located between the homoedomain genes of &lt;em&gt;A. bisporus&lt;/em&gt; JB137-S8 but are most likely the result of misassembly.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/10/3057/F3.medium.gif" width="440" height="245"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/9/10/3057/F3.medium.gif" width="440" height="245"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/10/3057/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/10/3057/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494529" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-30" class="first-child">Distribution of genes in the mating type locus of <em>A. bisporus</em> H97, ARP23 and JB137-S8. For comparative purposes the mat A locus of the model species C. <em>cinerea</em> is also displayed. Chromosome or scaffold and relative genomic position of loci are indicated. Genes that are colored blue are orthologous in all four genomes and syntenically conserved. Red genes indicate syntenically conserved orthologs in <em>A. bisporus</em> strains but missing from <em>C. cinerea</em>. <em>A. bisporus</em> has a single pair of homeodomain genes (colored black) while <em>C. cinerea</em> has two pairs. Green genes in <em>C. cinerea</em> are absent from <em>A. bisporus</em> strains. <em>A. bisporus</em> H97 has a unique gene (colored yellow). Six ORFs are located between the homoedomain genes of <em>A. bisporus</em> JB137-S8 but are most likely the result of misassembly.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-18" class="subsection"><h3>Pangenome analysis of Agaricus bisporus</h3><p id="p-31">Individual reference genomes do not and cannot contain all genetic information for a species due to genetic and genomic variation between individuals within a species. To account for such variation, it has become increasingly common to refer to species with multiple genomes sequenced in terms of their Pangenome, which is defined as the union of all genes observed across all isolates/strains of a species. A species pangenome for <em>A. bisporus</em> was constructed using the synteny-dependent PanOCT method implemented in Pangloss with the default parameters (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Fouts <em>et al.</em> 2012</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">McCarthy and Fitzpatrick 2019a</a>, <a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">2019b</a>). PanOCT clusters homologous sequences into syntenic orthologs clusters (SOCs) based on BLAST score ratio (BSR) assessment of sequence similarity and on proportions of relative synteny (conserved gene neighborhood, CGN) between potential orthologs (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Rasko <em>et al.</em> 2005</a>; <a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Fouts <em>et al.</em> 2012</a>). SOCs with syntenic orthologs from all four <em>A. bisporus</em> strain genomes in our dataset were classified as “core” SOCs, and clusters missing an ortholog from ≥1 strain genome were classified as “accessory” SOCs. After initial construction with PanOCT, the <em>A. bisporus</em> pangenome was refined by merging accessory SOCs based on reciprocal strain best hits between all members of a given pair of accessory SOCs (<a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">McCarthy and Fitzpatrick 2019a</a>, <a id="xref-ref-43-3" class="xref-bibr" href="#ref-43">2019b</a>). In total, we identified 7,732 core SOCs and 8,478 accessory SOCs within our <em>A. bisporus</em> dataset (16,120 in total) (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). The proportion of core SOCs relative to the total number of protein coding genes per genome ranged from a low of ∼60% in ARP23 to a high of 71% in H97. This proportion of core to accessory genes is lower than we have previously observed in a number of model fungal species (<a id="xref-ref-42-3" class="xref-bibr" href="#ref-42">McCarthy and Fitzpatrick 2019a</a>). Analysis of the distribution of SOCs within the <em>A. bisporus</em> accessory genome was performed within Pangloss using UpSetR, which is an R implementation of the UpSet method for visualization of set intersections and occurrences within a dataset using matrix representation. The UpSetR plot in <a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a> shows that singleton SOCs (<em>i.e.</em>, singleton genes) are the most common within the accessory genome, with 2,161 singleton SOCs from ARP23 alone and 6,574 in total (∼78% of all syntenic SOCs in the accessory genome). The distribution of the remaining 1,904 non-singleton SOCs within the <em>A. bisporus</em> accessory genome appears to follow evolutionary history (<a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2</a>), for example H97 and ARP23 share 727 accessory SOCs either exclusively or along with one other strain, while JB137 and ARP23 only share 615 accessory SOCs exclusively or with another strain (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4</a>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/10/3057/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="UpSetR plot of the distribution of syntenic orthologous clusters (SOCs) within the accessory genome of Agaricus bisporus." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1159653199" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;UpSetR plot of the distribution of syntenic orthologous clusters (SOCs) within the accessory genome of &lt;em&gt;Agaricus bisporus&lt;/em&gt;.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/10/3057/F4.medium.gif" width="440" height="439"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/9/10/3057/F4.medium.gif" width="440" height="439"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/10/3057/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/10/3057/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494304" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-32" class="first-child">UpSetR plot of the distribution of syntenic orthologous clusters (SOCs) within the accessory genome of <em>Agaricus bisporus</em>.</p><div class="sb-div caption-clear"></div></div></div><p id="p-33">Selection analysis of the core and accessory genomes was performed using the Yang &amp; Nielsen method as implemented in yn00 with the default parameters for yn00. 680 of 7,732 core SOCs (∼9% of core SOCs) and 172 of 1,904 non-singleton accessory SOCs (∼9% of non-singleton accessory SOCs, ∼2% of all accessory SOCs) showed evidence of at least 1 pairwise alignment under positive selection where d<sub>N</sub>/d<sub>S</sub> ≥ 1 and d<sub>N</sub>/d<sub>S</sub> ≠ ∞ (<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Yang and Nielsen 2000</a>).</p></div><div id="sec-19" class="subsection"><h3>Conclusion</h3><p id="p-34">In this analysis, we have presented the high quality genome sequence of <em>A. bisporus</em> var. <em>bisporus</em> ARP23, a commercially relevant genome. In total the genome was found to be 33.49 Mb in length, have high levels of synteny to H39 and contain 13,030 putative protein coding genes. Relative to the other two <em>A. bisporus</em> genomes that are currently available, ARP23 is the largest <em>A. bisporus</em> strain sequenced to date. Our analyses show that phylogenetically speaking ARP23 is more closely related to H97 than JB137-SB. Furthermore, all three strains have highly conserved mating loci. The lignocellulolytic gene content of all three <em>A. bisporus</em> strains is very similar. The pangenome of <em>A. bisporus</em> is quite diverse with between 60–70% of genes considered as core SOCs depending on isolate under consideration. The above analyses and genome sequence are the starting point for more detailed molecular analyses into the growth and phenotypical responses of ARP23 when challenged with economically important mycoviruses.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-35">EOC is funded by a Teagasc Walsh Fellowship Scheme (grant reference number 10564231). JM and CGPM are funded by postgraduate scholarships from the Irish Research Council, Government of Ireland (grant numbers GOIPG/2016/1112 and GOIPG/2015/2242). We acknowledge the DJEI/DES/SFI/HEA Irish Centre for High-End Computing (ICHEC) for the provision of computational facilities and support.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-2"><p id="p-36">Supplemental material available at FigShare: <a href="https://doi.org/10.25387/g3.9076991">https://doi.org/10.25387/g3.9076991</a>.</p></li><li class="fn" id="fn-3"><p id="p-37"><em>Communicating editor: A. Rokas</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2019-06-11"><span class="received-label">Received </span>June 11, 2019.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2019-07-29"><span class="accepted-label">Accepted </span>July 29, 2019.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2019 OConnor <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3057.1" data-doi="10.1093/nar/25.17.3389"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Altschul</span>,  <span class="cit-name-given-names">S. F.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. L.</span> <span class="cit-name-surname">Madden</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. A.</span> <span class="cit-name-surname">Schäffer</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Zhang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Zhang</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Gapped BLAST and PSI-BLAST: a new generation of protein database search programs.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">3389</span>–<span class="cit-lpage">3402</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/25.17.3389</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F25.17.3389%26rft_id%253Dinfo%253Apmid%252F9254694%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/25.17.3389&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9254694&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3057.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1997XU79300002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3057.2" data-doi="10.1038/s41587-018-0007-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Arora</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Steuernagel</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Gaurav</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Chandramohan</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Long</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">Resistance gene cloning from a wild crop relative by sequence capture and association genetics.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">139</span>–<span class="cit-lpage">143</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/s41587-018-0007-9</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BBiotechnol.%26rft.volume%253D37%26rft.spage%253D139%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41587-018-0007-9%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41587-018-0007-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3057.3" data-doi="10.1038/75556"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ashburner</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. A.</span> <span class="cit-name-surname">Ball</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. A.</span> <span class="cit-name-surname">Blake</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Botstein</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Butler</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Gene Ontology: tool for the unification of biology.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">25</span>–<span class="cit-lpage">29</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/75556</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DAshburner%26rft.auinit1%253DM.%26rft.volume%253D25%26rft.issue%253D1%26rft.spage%253D25%26rft.epage%253D29%26rft.atitle%253DGene%2Bontology%253A%2Btool%2Bfor%2Bthe%2Bunification%2Bof%2Bbiology.%2BThe%2BGene%2BOntology%2BConsortium.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F75556%26rft_id%253Dinfo%253Apmid%252F10802651%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/75556&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10802651&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3057.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000086884000011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3057.4" data-doi="10.1093/database/bar062"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Balakrishnan</span>,  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Park</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Karra</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. C.</span> <span class="cit-name-surname">Hitz</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Binkley</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">YeastMine-An integrated data warehouse for Saccharomyces cerevisiae data as a multipurpose tool-kit.</span> <abbr class="cit-jnl-abbrev">Database (Oxford)</abbr> <span class="cit-vol">2012</span>: <span class="cit-elocation-id">bar062</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/database/bar062</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDatabase%2B%2528Oxford%2529%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fdatabase%252Fbar062%26rft_id%253Dinfo%253Apmid%252F22434830%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/database/bar062&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22434830&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3057.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3057.5" data-doi="10.1093/nar/27.2.573"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Benson</span>,  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Tandem repeats finder: A program to analyze DNA sequences.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">573</span>–<span class="cit-lpage">580</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/27.2.573</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F27.2.573%26rft_id%253Dinfo%253Apmid%252F9862982%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/27.2.573&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9862982&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3057.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000078181500028&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3057.6" data-doi="10.1186/1471-2105-15-211"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Boetzer</span>,  <span class="cit-name-given-names">M.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Pirovano</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">SSPACE-LongRead: scaffolding bacterial draft genomes using long read sequence information.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">211</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2105-15-211</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BBioinformatics%26rft.volume%253D15%26rft.spage%253D211%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-15-211%26rft_id%253Dinfo%253Apmid%252F24950923%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-15-211&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24950923&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3057.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-book" id="cit-9.10.3057.7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Callac</span>,  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Imbernon</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. W.</span> <span class="cit-name-surname">Kerrigan</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.-M.</span> <span class="cit-name-surname">Olivier</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">The Two Life Cycles of Agaricus bisporus</span>, pp. <span class="cit-fpage">57</span>–<span class="cit-lpage">66</span> in <span class="cit-source">Mushroom Biology and Mushroom Products. </span>University Park, Pennsylvania.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3057.8" data-doi="10.1186/1471-2105-10-421"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Camacho</span>,  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Coulouris</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Avagyan</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Ma</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Papadopoulos</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">BLAST+: architecture and applications.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">421</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2105-10-421</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DCamacho%26rft.auinit1%253DC.%26rft.volume%253D10%26rft.spage%253D421%26rft.epage%253D421%26rft.atitle%253DBLAST%252B%253A%2Barchitecture%2Band%2Bapplications.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-10-421%26rft_id%253Dinfo%253Apmid%252F20003500%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-10-421&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20003500&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3057.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3057.9" data-doi="10.1093/bioinformatics/btp348"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Capella-Gutierrez</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">Silla-Martinez</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Gabaldon</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">trimAl: a tool for automated alignment trimming in large-scale phylogenetic analyses.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1972</span>–<span class="cit-lpage">1973</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btp348</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp348%26rft_id%253Dinfo%253Apmid%252F19505945%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp348&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19505945&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3057.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268107100022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3057.10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chu</span>,  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> Jupiter Plot: A Circos-based tool to visualize genome assembly consistency.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3057.11" data-doi="10.2135/cropsci2016.10.0885"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dempewolf</span>,  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Baute</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Anderson</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Kilian</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Smith</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Past and future use of wild relatives in crop breeding.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">57</span>: <span class="cit-fpage">1070</span>–<span class="cit-lpage">1082</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2016.10.0885</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D57%26rft.spage%253D1070%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2016.10.0885%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2016.10.0885&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3057.12" data-doi="10.1093/nar/gkh340"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Edgar</span>,  <span class="cit-name-given-names">R. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">MUSCLE: Multiple sequence alignment with high accuracy and high throughput.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">1792</span>–<span class="cit-lpage">1797</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkh340</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DEdgar%26rft.auinit1%253DR.%2BC.%26rft.volume%253D32%26rft.issue%253D5%26rft.spage%253D1792%26rft.epage%253D1797%26rft.atitle%253DMUSCLE%253A%2Bmultiple%2Bsequence%2Balignment%2Bwith%2Bhigh%2Baccuracy%2Band%2Bhigh%2Bthroughput%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkh340%26rft_id%253Dinfo%253Apmid%252F15034147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkh340&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15034147&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3057.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000220487200025&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3057.13" data-doi="10.1186/s13059-015-0721-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Emms</span>,  <span class="cit-name-given-names">D. M.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Kelly</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">OrthoFinder: solving fundamental biases in whole genome comparisons dramatically improves orthogroup inference accuracy.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">157</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s13059-015-0721-2</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D16%26rft.spage%253D157%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-015-0721-2%26rft_id%253Dinfo%253Apmid%252F26243257%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13059-015-0721-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26243257&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3057.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3057.14" data-doi="10.1093/nar/30.7.1575"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Enright</span>,  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Van Dongen</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. A.</span> <span class="cit-name-surname">Ouzounis</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">An efficient algorithm for large-scale detection of protein families.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">1575</span>–<span class="cit-lpage">1584</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/30.7.1575</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DEnright%26rft.auinit1%253DA.%2BJ.%26rft.volume%253D30%26rft.issue%253D7%26rft.spage%253D1575%26rft.epage%253D1584%26rft.atitle%253DAn%2Befficient%2Balgorithm%2Bfor%2Blarge-scale%2Bdetection%2Bof%2Bprotein%2Bfamilies%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F30.7.1575%26rft_id%253Dinfo%253Apmid%252F11917018%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/30.7.1575&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11917018&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3057.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000174654200013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3057.15" data-doi="10.1016/j.fgb.2013.01.009"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Férandon</span>,  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Xu</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Barroso</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The 135 kbp mitochondrial genome of Agaricus bisporus is the largest known eukaryotic reservoir of group I introns and plasmid-related sequences.</span> <abbr class="cit-jnl-abbrev">Fungal Genet. Biol.</abbr> <span class="cit-vol">55</span>: <span class="cit-fpage">85</span>–<span class="cit-lpage">91</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.fgb.2013.01.009</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFungal%2BGenet.%2BBiol.%26rft.volume%253D55%26rft.spage%253D85%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.fgb.2013.01.009%26rft_id%253Dinfo%253Apmid%252F23428625%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.fgb.2013.01.009&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23428625&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3057.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3057.16" data-doi="10.1007/BF00023910"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fletcher</span>,  <span class="cit-name-given-names">J. T.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">Disease resistance in protected crops and mushrooms.</span> <abbr class="cit-jnl-abbrev">Euphytica</abbr> <span class="cit-vol">63</span>: <span class="cit-fpage">33</span>–<span class="cit-lpage">49</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/BF00023910</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEuphytica%26rft.volume%253D63%26rft.spage%253D33%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00023910%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00023910&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3057.17" data-doi="10.1016/j.funbio.2011.02.017"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Foulongne-Oriol</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Rodier</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Rousseau</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Largeteau</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">Savoie</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Quantitative genetics to dissect the fungal-fungal interaction between Lecanicillium verticillium and the white button mushroom Agaricus bisporus.</span> <abbr class="cit-jnl-abbrev">Fungal Biol.</abbr> <span class="cit-vol">115</span>: <span class="cit-fpage">421</span>–<span class="cit-lpage">431</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.funbio.2011.02.017</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DFungal%2BBiol%26rft.aulast%253DFoulongne-Oriol%26rft.auinit1%253DM.%26rft.volume%253D115%26rft.issue%253D4-5%26rft.spage%253D421%26rft.epage%253D431%26rft.atitle%253DQuantitative%2Bgenetics%2Bto%2Bdissect%2Bthe%2Bfungal-fungal%2Binteraction%2Bbetween%2BLecanicillium%2Bverticillium%2Band%2Bthe%2Bwhite%2Bbutton%2Bmushroom%2BAgaricus%2Bbisporus.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.funbio.2011.02.017%26rft_id%253Dinfo%253Apmid%252F21530924%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.funbio.2011.02.017&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21530924&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3057.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3057.18" data-doi="10.1093/nar/gks757"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fouts</span>,  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Brinkac</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Beck</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Inman</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Sutton</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">PanOCT: Automated clustering of orthologs using conserved gene neighborhood for pan-genomic analysis of bacterial strains and closely related species.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">40</span>: <span class="cit-elocation-id">e172</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gks757</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgks757%26rft_id%253Dinfo%253Apmid%252F22904089%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gks757&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22904089&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3057.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3057.19" data-doi="10.1371/journal.pone.0076826"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gao</span>,  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. J.</span> <span class="cit-name-surname">Baars</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">O.</span> <span class="cit-name-surname">Dolstra</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. G.</span> <span class="cit-name-surname">Visser</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. S.</span> <span class="cit-name-surname">Sonnenberg</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genetic variation and combining ability analysis of bruising sensitivity in agaricus bisporus.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e76826</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0076826</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D8%26rft.spage%253De76826%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0076826%26rft_id%253Dinfo%253Apmid%252F24116171%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0076826&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24116171&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3057.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3057.20" data-doi="10.1017/S0953756203007202"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Grogan</span>,  <span class="cit-name-given-names">H. M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. A. T.</span> <span class="cit-name-surname">Adie</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. H.</span> <span class="cit-name-surname">Gaze</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. P.</span> <span class="cit-name-surname">Challen</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. R.</span> <span class="cit-name-surname">Mills</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Double-stranded RNA elements associated with the MVX disease of Agaricus bisporus.</span> <abbr class="cit-jnl-abbrev">Mycol. Res.</abbr> <span class="cit-vol">107</span>: <span class="cit-fpage">147</span>–<span class="cit-lpage">154</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1017/S0953756203007202</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMycological%2Bresearch%26rft.stitle%253DMycol%2BRes%26rft.aulast%253DGrogan%26rft.auinit1%253DH.%2BM.%26rft.volume%253D107%26rft.issue%253DPt%2B2%26rft.spage%253D147%26rft.epage%253D154%26rft.atitle%253DDouble-stranded%2BRNA%2Belements%2Bassociated%2Bwith%2Bthe%2BMVX%2Bdisease%2Bof%2BAgaricus%2Bbisporus.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0953756203007202%26rft_id%253Dinfo%253Apmid%252F12747325%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0953756203007202&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12747325&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3057.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3057.21" data-doi="10.4161/19420889.2014.994382"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Grognet</span>,  <span class="cit-name-given-names">P.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Silar</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Maintaining heterokaryosis in pseudo-homothallic fungi.</span> <abbr class="cit-jnl-abbrev">Commun. Integr. Biol.</abbr> <span class="cit-vol">8</span>: <span class="cit-elocation-id">e994382</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.4161/19420889.2014.994382</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCommun.%2BIntegr.%2BBiol.%26rft.volume%253D8%26rft.spage%253De994382%26rft_id%253Dinfo%253Adoi%252F10.4161%252F19420889.2014.994382%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.4161/19420889.2014.994382&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3057.22" data-doi="10.1093/bioinformatics/btu392"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hackl</span>,  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Hedrich</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Schultz</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Förster</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Proovread: Large-scale high-accuracy PacBio correction through iterative short read consensus.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">3004</span>–<span class="cit-lpage">3011</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btu392</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu392%26rft_id%253Dinfo%253Apmid%252F25015988%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btu392&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25015988&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3057.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3057.23" data-doi="10.1093/molbev/msx281"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hoang</span>,  <span class="cit-name-given-names">D. T.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">O.</span> <span class="cit-name-surname">Chernomor</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">von Haeseler</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. Q.</span> <span class="cit-name-surname">Minh</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">L. S.</span> <span class="cit-name-surname">Vinh</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">UFBoot2: Improving the Ultrafast Bootstrap Approximation.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">518</span>–<span class="cit-lpage">522</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/msx281</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft.volume%253D35%26rft.spage%253D518%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsx281%26rft_id%253Dinfo%253Apmid%252F29077904%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msx281&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=29077904&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3057.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3057.24" data-doi="10.1093/bioinformatics/btv661"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hoff</span>,  <span class="cit-name-given-names">K. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Lange</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Lomsadze</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Borodovsky</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Stanke</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">BRAKER1: Unsupervised RNA-Seq-based genome annotation with GeneMark-ET and AUGUSTUS.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">767</span>–<span class="cit-lpage">769</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btv661</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtv661%26rft_id%253Dinfo%253Apmid%252F26559507%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btv661&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26559507&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3057.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3057.25" data-doi="10.1111/j.1744-7348.1960.tb03530.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hussey</span>,  <span class="cit-name-given-names">N. W.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">I. J.</span> <span class="cit-name-surname">Wyatt</span></span></li></ol><cite>,<span class="cit-pub-date">1960</span> <span class="cit-article-title">Insecticidal control of paedogenetic cecid larvae in mushroom beds.</span> <abbr class="cit-jnl-abbrev">Ann. Appl. Biol.</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">347</span>–<span class="cit-lpage">351</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1744-7348.1960.tb03530.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BAppl.%2BBiol.%26rft.volume%253D48%26rft.spage%253D347%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1744-7348.1960.tb03530.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1744-7348.1960.tb03530.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3057.26" data-doi="10.1093/bioinformatics/btu031"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>,  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Binns</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.-Y.</span> <span class="cit-name-surname">Chang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Fraser</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Li</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">InterProScan 5: genome-scale protein function classification.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">1236</span>–<span class="cit-lpage">1240</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btu031</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu031%26rft_id%253Dinfo%253Apmid%252F24451626%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btu031&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24451626&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3057.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000336095100007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3057.27" data-doi="10.1038/nmeth.4285"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kalyaanamoorthy</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. Q.</span> <span class="cit-name-surname">Minh</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. K. F.</span> <span class="cit-name-surname">Wong</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">von Haeseler</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">L. S.</span> <span class="cit-name-surname">Jermiin</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">ModelFinder: fast model selection for accurate phylogenetic estimates.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">587</span>–<span class="cit-lpage">589</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nmeth.4285</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BMethods%26rft.volume%253D14%26rft.spage%253D587%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.4285%26rft_id%253Dinfo%253Apmid%252F28481363%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.4285&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28481363&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3057.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3057.28" data-doi="10.1016/j.jmb.2015.11.006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kanehisa</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Sato</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Morishima</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">BlastKOALA and GhostKOALA: KEGG Tools for Functional Characterization of Genome and Metagenome Sequences.</span> <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">428</span>: <span class="cit-fpage">726</span>–<span class="cit-lpage">731</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.jmb.2015.11.006</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BMol.%2BBiol.%26rft.volume%253D428%26rft.spage%253D726%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jmb.2015.11.006%26rft_id%253Dinfo%253Apmid%252F26585406%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jmb.2015.11.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26585406&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3057.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-confproc" id="cit-9.10.3057.29"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kerrigan</span>,  <span class="cit-name-given-names">R. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> A brief history of marker assisted selection in Agaricus bisporus. Sci. Cultiv. edible fungi. Proc. 15th Int. Congr. Sci. Cultiv. Edible Fungi, Maastricht, Netherlands, 15–19 May, 2000. 183–189.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3057.30" data-doi="10.1006/emyc.1994.1020"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kerrigan</span>,  <span class="cit-name-given-names">R. W.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Imbernon</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Callac</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Billette</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.-M.</span> <span class="cit-name-surname">Olivier</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">The Heterothallic Life Cycle of Agaricus bisporus var. burnettii and the Inheritance of Its Tetrasporic Trait.</span> <abbr class="cit-jnl-abbrev">Exp. Mycol.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">193</span>–<span class="cit-lpage">210</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1006/emyc.1994.1020</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DExp.%2BMycol.%26rft.volume%253D18%26rft.spage%253D193%26rft_id%253Dinfo%253Adoi%252F10.1006%252Femyc.1994.1020%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/emyc.1994.1020&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3057.31" data-doi="10.1101/gr.215087.116"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Koren</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. P.</span> <span class="cit-name-surname">Walenz</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Berlin</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. R.</span> <span class="cit-name-surname">Miller</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N. H.</span> <span class="cit-name-surname">Bergman</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Canu: Scalable and accurate long-read assembly via adaptive κ-mer weighting and repeat separation.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">722</span>–<span class="cit-lpage">736</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.215087.116</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.215087.116%26rft_id%253Dinfo%253Apmid%252F28298431%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjI3LzUvNzIyIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMC8zMDU3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3057.32" data-doi="10.1093/nar/gkm160"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lagesen</span>,  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Hallin</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. A.</span> <span class="cit-name-surname">Rødland</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.-H.</span> <span class="cit-name-surname">Stærfeldt</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Rognes</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">RNAmmer: consistent and rapid annotation of ribosomal RNA genes.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">3100</span>–<span class="cit-lpage">3108</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkm160</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkm160%26rft_id%253Dinfo%253Apmid%252F17452365%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkm160&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17452365&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3057.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000247350800027&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3057.33" data-doi="10.1038/nmeth.1923"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Langmead</span>,  <span class="cit-name-given-names">B.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. L.</span> <span class="cit-name-surname">Salzberg</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Fast gapped-read alignment with Bowtie 2.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">357</span>–<span class="cit-lpage">359</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nmeth.1923</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DLangmead%26rft.auinit1%253DB.%26rft.volume%253D9%26rft.issue%253D4%26rft.spage%253D357%26rft.epage%253D359%26rft.atitle%253DFast%2Bgapped-read%2Balignment%2Bwith%2BBowtie%2B2.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1923%26rft_id%253Dinfo%253Apmid%252F22388286%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1923&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22388286&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3057.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000302218500017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3057.34" data-doi="10.1093/bioinformatics/btp368"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lartillot</span>,  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Lepage</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Blanquart</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">PhyloBayes 3: a Bayesian software package for phylogenetic reconstruction and molecular dating.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">2286</span>–<span class="cit-lpage">2288</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btp368</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp368%26rft_id%253Dinfo%253Apmid%252F19535536%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp368&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19535536&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3057.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000269196000023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3057.35" data-doi="10.1093/bioinformatics/btl529"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Letunic</span>,  <span class="cit-name-given-names">I.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Bork</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Interactive Tree Of Life (iTOL): An online tool for phylogenetic tree display and annotation.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">127</span>–<span class="cit-lpage">128</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btl529</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtl529%26rft_id%253Dinfo%253Apmid%252F17050570%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btl529&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17050570&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3057.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000243060300021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3057.36" data-doi="10.1093/bioinformatics/btp698"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">H.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Durbin</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Fast and accurate long-read alignment with Burrows-Wheeler transform.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">589</span>–<span class="cit-lpage">595</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btp698</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp698%26rft_id%253Dinfo%253Apmid%252F20080505%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp698&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20080505&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3057.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000274973800001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3057.37" data-doi="10.1093/nar/gkt1178"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lombard</span>,  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Golaconda Ramulu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Drula</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. M.</span> <span class="cit-name-surname">Coutinho</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Henrissat</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The carbohydrate-active enzymes database (CAZy) in 2013.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">D490</span>–<span class="cit-lpage">D495</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkt1178</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkt1178%26rft_id%253Dinfo%253Apmid%252F24270786%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkt1178&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24270786&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3057.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000331139800073&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3057.38" data-doi="10.1093/nar/gkw413"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lowe</span>,  <span class="cit-name-given-names">T. M.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. P.</span> <span class="cit-name-surname">Chan</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">tRNAscan-SE On-line: integrating search and context for analysis of transfer RNA genes.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">W54</span>–<span class="cit-lpage">W57</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkw413</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkw413%26rft_id%253Dinfo%253Apmid%252F27174935%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkw413&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27174935&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3057.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3057.39" data-doi="10.1093/bioinformatics/btr011"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Marçais</span>,  <span class="cit-name-given-names">G.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Kingsford</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A fast, lock-free approach for efficient parallel counting of occurrences of k-mers.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">764</span>–<span class="cit-lpage">770</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btr011</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtr011%26rft_id%253Dinfo%253Apmid%252F21217122%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btr011&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21217122&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3057.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000288277300004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3057.40" data-doi="10.14806/ej.17.1.200"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Martin</span>,  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Cutadapt removes adapter sequences from high-throughput sequencing reads</span>. <abbr class="cit-jnl-abbrev">EMBnet.journal</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">10</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.14806/ej.17.1.200</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEMBnet.journal%26rft.volume%253D17%26rft.spage%253D10%26rft_id%253Dinfo%253Adoi%252F10.14806%252Fej.17.1.200%26rft_id%253Dinfo%253Apmid%252F25452271%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.14806/ej.17.1.200&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25452271&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3057.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3057.41" data-doi="10.3852/mycologia.98.6.982"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Matheny</span>,  <span class="cit-name-given-names">P. B.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">Curtis</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Hofstetter</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. C.</span> <span class="cit-name-surname">Aime</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.-M.</span> <span class="cit-name-surname">Moncalvo</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Major clades of Agaricales: a multilocus phylogenetic overview.</span> <abbr class="cit-jnl-abbrev">Mycologia</abbr> <span class="cit-vol">98</span>: <span class="cit-fpage">982</span>–<span class="cit-lpage">995</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1080/15572536.2006.11832627</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMycologia%26rft.stitle%253DMycologia%26rft.aulast%253DMatheny%26rft.auinit1%253DP.%2BB.%26rft.volume%253D98%26rft.issue%253D6%26rft.spage%253D982%26rft.epage%253D995%26rft.atitle%253DMajor%2Bclades%2Bof%2BAgaricales%253A%2Ba%2Bmultilocus%2Bphylogenetic%2Boverview%26rft_id%253Dinfo%253Adoi%252F10.3852%252Fmycologia.98.6.982%26rft_id%253Dinfo%253Apmid%252F17486974%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3852/mycologia.98.6.982&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17486974&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3057.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000245858800015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3057.42" data-doi="10.1099/mgen.0.000243"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McCarthy</span>,  <span class="cit-name-given-names">C. G. P.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. A.</span> <span class="cit-name-surname">Fitzpatrick</span></span></li></ol><cite>, <span class="cit-pub-date">2019</span><span class="cit-comment">a</span> Pan-genome analyses of model fungal species. Microb. Genomics 5:.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1099/mgen.0.000243</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft_id%253Dinfo%253Adoi%252F10.1099%252Fmgen.0.000243%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1099/mgen.0.000243&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3057.43" data-doi="10.3390/genes10070521"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McCarthy</span>,  <span class="cit-name-given-names">C. G. P.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. A.</span> <span class="cit-name-surname">Fitzpatrick</span></span></li></ol><cite>, <span class="cit-pub-date">2019</span><span class="cit-comment">b</span> <span class="cit-article-title">Pangloss: A Tool for Pan-Genome Analysis of Microbial Eukaryotes.</span> <abbr class="cit-jnl-abbrev">Genes (Basel)</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">521</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3390/genes10070521</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2528Basel%2529%26rft.volume%253D10%26rft.spage%253D521%26rft_id%253Dinfo%253Adoi%252F10.3390%252Fgenes10070521%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3390/genes10070521&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3057.44" data-doi="10.1080/00275514.1971.12019142"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>,  <span class="cit-name-given-names">R. E.</span></span></li></ol><cite>, <span class="cit-pub-date">1971</span> <span class="cit-article-title">Evidence of Sexuality in the Cultivated Mushroom, Agaricus bisporus.</span> <abbr class="cit-jnl-abbrev">Mycologia</abbr> <span class="cit-vol">63</span>: <span class="cit-fpage">630</span>–<span class="cit-lpage">634</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1080/00275514.1971.12019142</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMycologia%26rft.volume%253D63%26rft.spage%253D630%26rft_id%253Dinfo%253Adoi%252F10.1080%252F00275514.1971.12019142%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/00275514.1971.12019142&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3057.45" data-doi="10.1006/fgbi.1999.1157"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Moquet</span>,  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Desmerger</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Mamoun</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Ramos-Guedes-Lafargue</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">Olivier</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">A quantitative trait locus of Agaricus bisporus resistance to Pseudomonas tolaasii is closely linked to natural cap color.</span> <abbr class="cit-jnl-abbrev">Fungal Genet. Biol.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">34</span>–<span class="cit-lpage">42</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1006/fgbi.1999.1157</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFungal%2Bgenetics%2Band%2Bbiology%2B%253A%2B%2BFG%2B%2526%2BB%26rft.stitle%253DFungal%2BGenet%2BBiol%26rft.aulast%253DMoquet%26rft.auinit1%253DF.%26rft.volume%253D28%26rft.issue%253D1%26rft.spage%253D34%26rft.epage%253D42%26rft.atitle%253DA%2Bquantitative%2Btrait%2Blocus%2Bof%2BAgaricus%2Bbisporus%2Bresistance%2Bto%2BPseudomonas%2Btolaasii%2Bis%2Bclosely%2Blinked%2Bto%2Bnatural%2Bcap%2Bcolor.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Ffgbi.1999.1157%26rft_id%253Dinfo%253Apmid%252F10512670%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/fgbi.1999.1157&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10512670&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3057.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3057.46" data-doi="10.1073/pnas.1206847109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Morin</span>,  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Kohler</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. R.</span> <span class="cit-name-surname">Baker</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Foulongne-Oriol</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Lombard</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genome sequence of the button mushroom Agaricus bisporus reveals mechanisms governing adaptation to a humic-rich ecological niche.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">109</span>: <span class="cit-fpage">17501</span>–<span class="cit-lpage">17506</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.1206847109</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1206847109%26rft_id%253Dinfo%253Apmid%252F23045686%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA5LzQzLzE3NTAxIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMC8zMDU3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3057.47" data-doi="10.1093/molbev/msu300"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nguyen</span>,  <span class="cit-name-given-names">L. T.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H. A.</span> <span class="cit-name-surname">Schmidt</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Von Haeseler</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. Q.</span> <span class="cit-name-surname">Minh</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">IQ-TREE: A fast and effective stochastic algorithm for estimating maximum-likelihood phylogenies.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">268</span>–<span class="cit-lpage">274</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/msu300</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsu300%26rft_id%253Dinfo%253Apmid%252F25371430%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msu300&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25371430&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3057.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3057.48" data-doi="10.1038/hdy.2013.67"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nieuwenhuis</span>,  <span class="cit-name-given-names">B. P. S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Billiard</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Vuilleumier</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Petit</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. E.</span> <span class="cit-name-surname">Hood</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Evolution of uni- and bifactorial sexual compatibility systems in fungi.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">111</span>: <span class="cit-fpage">445</span>–<span class="cit-lpage">455</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/hdy.2013.67</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D111%26rft.spage%253D445%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2013.67%26rft_id%253Dinfo%253Apmid%252F23838688%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2013.67&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23838688&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3057.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3057.49" data-doi="10.1080/07060669309500829"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">North</span>,  <span class="cit-name-given-names">L. H.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. J.</span> <span class="cit-name-surname">Wuest</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">The infection process and symptom expression of verticillium disease of agaricus bisporus.</span> <abbr class="cit-jnl-abbrev">Can. J. Plant Pathol.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">74</span>–<span class="cit-lpage">80</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1080/07060669309500829</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCan.%2BJ.%2BPlant%2BPathol.%26rft.volume%253D15%26rft.spage%253D74%26rft_id%253Dinfo%253Adoi%252F10.1080%252F07060669309500829%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/07060669309500829&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3057.50" data-doi="10.1093/nar/27.1.29"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ogata</span>,  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Goto</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Sato</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Fujibuchi</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Bono</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">KEGG: Kyoto Encyclopedia of Genes and Genomes.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>. <span class="cit-vol">27</span>: <span class="cit-fpage">29</span>–<span class="cit-lpage">34</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F27.1.29%26rft_id%253Dinfo%253Apmid%252F9847135%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/27.1.29&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9847135&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3057.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000077983000006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-book" id="cit-9.10.3057.51"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Prescott</span>,  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Wong</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Panaretou</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Boa</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Bond</span></span> et al.</li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Useful fungi</span>, pp. <span class="cit-fpage">24</span>–<span class="cit-lpage">31</span> in <span class="cit-source"><em>State of the World’s Fungi. Report. Royal Botanic Gardens</em>, <em>Kew</em></span>, edited by <span class="cit-ed"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Willis</span></span>. <span class="cit-comment">Royal Botanical Gardens, Kew</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3057.52" data-doi="10.2307/3758075"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Raper</span>,  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. R.</span> <span class="cit-name-surname">Raper</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. E.</span> <span class="cit-name-surname">Miller</span></span></li></ol><cite>, <span class="cit-pub-date">1972</span> <span class="cit-article-title">Genetic Analysis of the Life Cycle of Agaricus bisporus.</span> <abbr class="cit-jnl-abbrev">Mycologia</abbr> <span class="cit-vol">64</span>: <span class="cit-fpage">1088</span>–<span class="cit-lpage">1117</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1080/00275514.1972.12019354</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMycologia%26rft.volume%253D64%26rft.spage%253D1088%26rft_id%253Dinfo%253Adoi%252F10.2307%252F3758075%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/3758075&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1972N809900006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3057.53" data-doi="10.1186/1471-2105-6-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rasko</span>,  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G. S. A.</span> <span class="cit-name-surname">Myers</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Ravel</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Visualization of comparative genomic analyses by BLAST score ratio.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">2</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2105-6-2</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DRasko%26rft.auinit1%253DD.%2BA.%26rft.volume%253D6%26rft.spage%253D2%26rft.epage%253D2%26rft.atitle%253DVisualization%2Bof%2Bcomparative%2Bgenomic%2Banalyses%2Bby%2BBLAST%2Bscore%2Bratio.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-6-2%26rft_id%253Dinfo%253Apmid%252F15634352%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-6-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15634352&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3057.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3057.54" data-doi="10.1186/s12859-018-2485-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Roach</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Schmidt</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Borneman</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Purge Haplotigs: Synteny Reduction for Third-gen Diploid Genome Assemblies.</span> <abbr class="cit-jnl-abbrev">bioRxiv</abbr>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12859-018-2485-7</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DbioRxiv%26rft.volume%253D19%26rft.spage%253D460%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12859-018-2485-7%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12859-018-2485-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3057.55" data-doi="10.2307/3792744"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Royse</span>,  <span class="cit-name-given-names">D. J.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">May</span></span></li></ol><cite>, <span class="cit-pub-date">1982</span> <span class="cit-article-title">Genetic Relatèdness and Its Application in Selective Breeding of Agaricus brunnescens.</span> <abbr class="cit-jnl-abbrev">Mycologia</abbr> <span class="cit-vol">74</span>: <span class="cit-fpage">569</span>–<span class="cit-lpage">575</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMycologia%26rft.volume%253D74%26rft.spage%253D569%26rft_id%253Dinfo%253Adoi%252F10.2307%252F3792744%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/3792744&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1982NZ29300005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3057.56" data-doi="10.1093/bioinformatics/btv351"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Simão</span>,  <span class="cit-name-given-names">F. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. M.</span> <span class="cit-name-surname">Waterhouse</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Ioannidis</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. V.</span> <span class="cit-name-surname">Kriventseva</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. M.</span> <span class="cit-name-surname">Zdobnov</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">BUSCO: Assessing genome assembly and annotation completeness with single-copy orthologs.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">3210</span>–<span class="cit-lpage">3212</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btv351</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtv351%26rft_id%253Dinfo%253Apmid%252F26059717%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btv351&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26059717&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3057.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3057.57" data-doi="10.1111/j.1574-6976.1999.tb00415.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Soler-Rivas</span>,  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Biochemical and physiological aspects of brown blotch disease of Agaricus bisporus.</span> <abbr class="cit-jnl-abbrev">FEMS Microbiol. Rev.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">591</span>–<span class="cit-lpage">614</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1574-6976.1999.tb00415.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEMS%2BMicrobiol.%2BRev.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1574-6976.1999.tb00415.x%26rft_id%253Dinfo%253Apmid%252F10525168%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1574-6976.1999.tb00415.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10525168&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3057.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000083131800005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3057.58" data-doi="10.1016/j.fgb.2016.06.001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sonnenberg</span>,  <span class="cit-name-given-names">A. S. M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Gao</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Lavrijssen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Hendrickx</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Sedaghat-Tellgerd</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">A detailed analysis of the recombination landscape of the button mushroom Agaricus bisporus var. bisporus.</span> <abbr class="cit-jnl-abbrev">Fungal Genet. Biol.</abbr> <span class="cit-vol">93</span>: <span class="cit-fpage">35</span>–<span class="cit-lpage">45</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.fgb.2016.06.001</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFungal%2BGenet.%2BBiol.%26rft.volume%253D93%26rft.spage%253D35%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.fgb.2016.06.001%26rft_id%253Dinfo%253Apmid%252F27288752%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.fgb.2016.06.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27288752&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3057.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3057.59"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sonnhammer</span>,  <span class="cit-name-given-names">E. L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">von Heijne</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Krogh</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">A hidden Markov model for predicting transmembrane helices in protein sequences.</span> <abbr class="cit-jnl-abbrev">Proc. Int. Conf. Intell. Syst. Mol. Biol.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">175</span>–<span class="cit-lpage">182</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2B%252F%2B...%2BInternational%2BConference%2Bon%2BIntelligent%2BSystems%2Bfor%2BMolecular%2BBiology%2B%253B%2BISMB.%2BInternational%2BConference%2Bon%2BIntelligent%2BSystems%2Bfor%2BMolecular%2BBiology%26rft.stitle%253DProc%2BInt%2BConf%2BIntell%2BSyst%2BMol%2BBiol%26rft.aulast%253DSonnhammer%26rft.auinit1%253DE.%2BL.%26rft.volume%253D6%26rft.spage%253D175%26rft.epage%253D182%26rft.atitle%253DA%2Bhidden%2BMarkov%2Bmodel%2Bfor%2Bpredicting%2Btransmembrane%2Bhelices%2Bin%2Bprotein%2Bsequences.%26rft_id%253Dinfo%253Apmid%252F9783223%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=9783223&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3057.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3057.60" data-doi="10.1186/1471-2105-7-62"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stanke</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">O.</span> <span class="cit-name-surname">Schöffmann</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Morgenstern</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Waack</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Gene prediction in eukaryotes with a generalized hidden Markov model that uses hints from external sources.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">62</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2105-7-62</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DStanke%26rft.auinit1%253DM.%26rft.volume%253D7%26rft.spage%253D62%26rft.epage%253D62%26rft.atitle%253DGene%2Bprediction%2Bin%2Beukaryotes%2Bwith%2Ba%2Bgeneralized%2Bhidden%2BMarkov%2Bmodel%2Bthat%2Buses%2Bhints%2Bfrom%2Bexternal%2Bsources.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-7-62%26rft_id%253Dinfo%253Apmid%252F16469098%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-7-62&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16469098&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3057.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3057.61" data-doi="10.1093/bioinformatics/btx153"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vurture</span>,  <span class="cit-name-given-names">G. W.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F. J.</span> <span class="cit-name-surname">Sedlazeck</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Nattestad</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. J.</span> <span class="cit-name-surname">Underwood</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Fang</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">GenomeScope: fast reference-free genome profiling from short reads.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">2202</span>–<span class="cit-lpage">2204</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.volume%253D33%26rft.spage%253D2202%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtx153%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btx153&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3057.62" data-doi="10.5598/imafungus.2015.06.01.13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wilson</span>,  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. M.</span> <span class="cit-name-surname">Wilken</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. A.</span> <span class="cit-name-surname">van der Nest</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. T.</span> <span class="cit-name-surname">Steenkamp</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. J.</span> <span class="cit-name-surname">Wingfield</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Homothallism: an umbrella term for describing diverse sexual behaviours.</span> <abbr class="cit-jnl-abbrev">IMA Fungus</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">207</span>–<span class="cit-lpage">214</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.5598/imafungus.2015.06.01.13</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DIMA%2BFungus%26rft.volume%253D6%26rft.spage%253D207%26rft_id%253Dinfo%253Adoi%252F10.5598%252Fimafungus.2015.06.01.13%26rft_id%253Dinfo%253Apmid%252F26203424%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.5598/imafungus.2015.06.01.13&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26203424&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3057.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3057.63" data-doi="10.1093/oxfordjournals.molbev.a026236"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>,  <span class="cit-name-given-names">Z.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Nielsen</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Estimating Synonymous and Nonsynonymous Substitution Rates Under Realistic Evolutionary Models.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">32</span>–<span class="cit-lpage">43</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/oxfordjournals.molbev.a026236</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft.stitle%253DMol%2BBiol%2BEvol%26rft.aulast%253DYang%26rft.auinit1%253DZ.%26rft.volume%253D17%26rft.issue%253D1%26rft.spage%253D32%26rft.epage%253D43%26rft.atitle%253DEstimating%2BSynonymous%2Band%2BNonsynonymous%2BSubstitution%2BRates%2BUnder%2BRealistic%2BEvolutionary%2BModels%26rft_id%253Dinfo%253Adoi%252F10.1093%252Foxfordjournals.molbev.a026236%26rft_id%253Dinfo%253Apmid%252F10666704%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/oxfordjournals.molbev.a026236&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10666704&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3057.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000084917200004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3057.64" data-doi="10.1093/nar/gky418"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Yohe</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Huang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Entwistle</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Wu</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">DbCAN2: A meta server for automated carbohydrate-active enzyme annotation.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">W95</span>–<span class="cit-lpage">W101</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gky418</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft.volume%253D46%26rft.spage%253DW95%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgky418%26rft_id%253Dinfo%253Apmid%252F29771380%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gky418&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=29771380&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3057.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/9/10/3057.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/9/3045" title="Specific Gene Disruption in the Major Livestock Pests Cochliomyia hominivorax and Lucilia cuprina Using CRISPR/Cas9" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/10/3067" title="Greenlip Abalone (Haliotis laevigata) Genome and Protein Analysis Provides Insights into Maturation and Spawning" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/9/10"><span>Volume 9 Issue 10, October 2019</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/9/10.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/9/10.cover-source.jpg?itok=yA8kifEd" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 9 (10)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/10/3057" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-B4oMpWBfIosnbkNJTT_l9c0h02QQ_iJKw606GmbxMZs" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/genome-report" class="" data-icon-position="" data-hide-link-title="0">Genome Report</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/9/10/3057" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/9/10/3057" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/9/10/3057" class="active" data-icon-position="" data-hide-link-title="0">Whole Genome Sequence of the Commercially Relevant Mushroom Strain Agaricus bisporus var. bisporus ARP23</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/494295" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-dIXMr91yxhrQMNORzS6pDllOS1RJZQU_vXty4GBs0LI" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/10/3057.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/9/10/3057" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-4gRIlUSGde5eFpGWmudPRiHnNkO0bFH432e5f3cNK8s" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/9/10/3057" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/10/3057.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494295" id="node494295--2" data-pisa="ggg;9/10/3057" data-pisa-master="ggg;g3.119.400563" data-apath="/ggg/9/10/3057.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Whole Genome Sequence of the Commercially Relevant Mushroom Strain <em>Agaricus bisporus</em> var. <em>bisporus</em> ARP23</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-4626-5605" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Eoin O’Connor</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-4240-5175" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jamie McGowan</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-3880-9978" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Charley G. P. McCarthy</span>, <span class="highwire-citation-author" data-delta="3">Aniça Amini</span>, <span class="highwire-citation-author" data-delta="4">Helen Grogan</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0001-7345-6998" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>David A. Fitzpatrick</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3057-3066;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400563 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494295--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Eoin O’Connor </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Genome Evolution Laboratory, Department of Biology, Maynooth University, Maynooth, Co. Kildare, Ireland,</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Teagasc Food Research Centre, Ashtown, Dublin 15, D15 KN3K, Ireland,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Eoin%2BO%E2%80%99Connor%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=O%E2%80%99Connor%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AEoin%2BO%25E2%2580%2599Connor%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-4626-5605" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Eoin O’Connor</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jamie McGowan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Genome Evolution Laboratory, Department of Biology, Maynooth University, Maynooth, Co. Kildare, Ireland,</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Human Health Research Institute, Maynooth University, Maynooth, Co. Kildare, Ireland, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jamie%2BMcGowan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=McGowan%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJamie%2BMcGowan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-4240-5175" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jamie McGowan</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Charley G. P. McCarthy </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Genome Evolution Laboratory, Department of Biology, Maynooth University, Maynooth, Co. Kildare, Ireland,</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Human Health Research Institute, Maynooth University, Maynooth, Co. Kildare, Ireland, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Charley%2BG.%2BP.%2BMcCarthy%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=McCarthy%20CG&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ACharley%2BG.%2BP.%2BMcCarthy%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-3880-9978" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Charley G. P. McCarthy</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Aniça Amini </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Sylvan-Somycel (ESSC - Unité 2), ZI SUD, rue Lavoisier, BP 25, 37130 Langeais, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ani%C3%A7a%2BAmini%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Amini%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAni%25C3%25A7a%2BAmini%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Helen Grogan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Teagasc Food Research Centre, Ashtown, Dublin 15, D15 KN3K, Ireland,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Helen%2BGrogan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Grogan%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHelen%2BGrogan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">David A. Fitzpatrick </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Genome Evolution Laboratory, Department of Biology, Maynooth University, Maynooth, Co. Kildare, Ireland,</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Human Health Research Institute, Maynooth University, Maynooth, Co. Kildare, Ireland, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BA.%2BFitzpatrick%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fitzpatrick%20DA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADavid%2BA.%2BFitzpatrick%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-7345-6998" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for David A. Fitzpatrick</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:david.fitzpatrick@mu.ie" class="" data-icon-position="" data-hide-link-title="0">david.fitzpatrick@mu.ie</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/9/10/3057" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/9/10/3057&amp;title=Whole%20Genome%20Sequence%20of%20the%20Commercially%20Relevant%20Mushroom%20Strain%20Agaricus%20bisporus%20var.%20bisporus%20ARP23" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/9/10/3057&amp;title=Whole%20Genome%20Sequence%20of%20the%20Commercially%20Relevant%20Mushroom%20Strain%20Agaricus%20bisporus%20var.%20bisporus%20ARP23" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/9/10/3057&amp;title=Whole%20Genome%20Sequence%20of%20the%20Commercially%20Relevant%20Mushroom%20Strain%20Agaricus%20bisporus%20var.%20bisporus%20ARP23" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/9/10/3057&amp;text=Whole%20Genome%20Sequence%20of%20the%20Commercially%20Relevant%20Mushroom%20Strain%20Agaricus%20bisporus%20var.%20bisporus%20ARP23" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/9/10/3057&amp;title=Whole%20Genome%20Sequence%20of%20the%20Commercially%20Relevant%20Mushroom%20Strain%20Agaricus%20bisporus%20var.%20bisporus%20ARP23" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/9/10/3057&amp;t=Whole%20Genome%20Sequence%20of%20the%20Commercially%20Relevant%20Mushroom%20Strain%20Agaricus%20bisporus%20var.%20bisporus%20ARP23" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/9/10/3057&amp;title=Whole%20Genome%20Sequence%20of%20the%20Commercially%20Relevant%20Mushroom%20Strain%20Agaricus%20bisporus%20var.%20bisporus%20ARP23" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/9/10/3057&amp;title=Whole%20Genome%20Sequence%20of%20the%20Commercially%20Relevant%20Mushroom%20Strain%20Agaricus%20bisporus%20var.%20bisporus%20ARP23" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/494295/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494295" id="node494295--3" data-pisa="ggg;9/10/3057" data-pisa-master="ggg;g3.119.400563" data-apath="/ggg/9/10/3057.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Whole Genome Sequence of the Commercially Relevant Mushroom Strain <em>Agaricus bisporus</em> var. <em>bisporus</em> ARP23</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-4626-5605" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Eoin O’Connor</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-4240-5175" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jamie McGowan</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-3880-9978" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Charley G. P. McCarthy</span>, <span class="highwire-citation-author" data-delta="3">Aniça Amini</span>, <span class="highwire-citation-author" data-delta="4">Helen Grogan</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0001-7345-6998" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>David A. Fitzpatrick</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3057-3066;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400563 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494295--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Eoin O’Connor </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Genome Evolution Laboratory, Department of Biology, Maynooth University, Maynooth, Co. Kildare, Ireland,</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Teagasc Food Research Centre, Ashtown, Dublin 15, D15 KN3K, Ireland,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Eoin%2BO%E2%80%99Connor%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=O%E2%80%99Connor%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AEoin%2BO%25E2%2580%2599Connor%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-4626-5605" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Eoin O’Connor</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jamie McGowan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Genome Evolution Laboratory, Department of Biology, Maynooth University, Maynooth, Co. Kildare, Ireland,</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Human Health Research Institute, Maynooth University, Maynooth, Co. Kildare, Ireland, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jamie%2BMcGowan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=McGowan%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJamie%2BMcGowan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-4240-5175" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jamie McGowan</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Charley G. P. McCarthy </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Genome Evolution Laboratory, Department of Biology, Maynooth University, Maynooth, Co. Kildare, Ireland,</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Human Health Research Institute, Maynooth University, Maynooth, Co. Kildare, Ireland, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Charley%2BG.%2BP.%2BMcCarthy%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=McCarthy%20CG&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ACharley%2BG.%2BP.%2BMcCarthy%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-3880-9978" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Charley G. P. McCarthy</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Aniça Amini </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Sylvan-Somycel (ESSC - Unité 2), ZI SUD, rue Lavoisier, BP 25, 37130 Langeais, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ani%C3%A7a%2BAmini%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Amini%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAni%25C3%25A7a%2BAmini%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Helen Grogan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Teagasc Food Research Centre, Ashtown, Dublin 15, D15 KN3K, Ireland,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Helen%2BGrogan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Grogan%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHelen%2BGrogan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">David A. Fitzpatrick </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Genome Evolution Laboratory, Department of Biology, Maynooth University, Maynooth, Co. Kildare, Ireland,</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Human Health Research Institute, Maynooth University, Maynooth, Co. Kildare, Ireland, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BA.%2BFitzpatrick%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fitzpatrick%20DA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADavid%2BA.%2BFitzpatrick%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-7345-6998" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for David A. Fitzpatrick</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:david.fitzpatrick@mu.ie" class="" data-icon-position="" data-hide-link-title="0">david.fitzpatrick@mu.ie</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=3057&amp;rft.epage=3066&amp;rft.atitle=Whole%20Genome%20Sequence%20of%20the%20Commercially%20Relevant%20Mushroom%20Strain%20Agaricus%20bisporus%20var.%20bisporus%20ARP23&amp;rft.volume=9&amp;rft.issue=10&amp;rft.date=2019-10-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=O%E2%80%99Connor%2C+Eoin&amp;rft.au=McGowan%2C+Jamie&amp;rft.au=McCarthy%2C+Charley+G.+P.&amp;rft.au=Amini%2C+Ani%C3%A7a&amp;rft.au=Grogan%2C+Helen&amp;rft.au=Fitzpatrick%2C+David+A."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/494295/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/494295/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/494295/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/494295/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/494295/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/494295/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/494295/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/494295/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/494295/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/494295/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/494295/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/494295/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496044" id="node496044" data-pisa="ggg;g3.119.400871v1" data-pisa-master="ggg;g3.119.400871" data-apath="/ggg/early/2020/03/17/g3.119.400871.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/17/g3.119.400871" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title"><em>De Novo</em> Genome Assembly of the <em>indica</em> Rice Variety IR64 Using Linked-Read Sequencing and Nanopore Sequencing</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496040" id="node496040" data-pisa="ggg;g3.119.400893v1" data-pisa-master="ggg;g3.119.400893" data-apath="/ggg/early/2020/03/12/g3.119.400893.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/12/g3.119.400893" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Understanding the Genetic Domestication History of the Jianchang Duck by Genotyping and Sequencing of Genomic Genes Under Selection</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496037" id="node496037" data-pisa="ggg;g3.120.401071v1" data-pisa-master="ggg;g3.120.401071" data-apath="/ggg/early/2020/03/11/g3.120.401071.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/11/g3.120.401071" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title"><em>De Novo</em> Genome Assembly of the Meadow Brown Butterfly, <em>Maniola jurtina</em></span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Genome%20Report" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Genome Report</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/9/10/3057" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Methods</a></li><li class="odd"><a href="#sec-12" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/9/10/3057.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/9/10/3057.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/9/10/3057.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
