<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/9/10/3381" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/9/10/3381.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/9/10/3381.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/9/10/3381.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/9/10/3381/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/9/10/3381.atom" />
<meta name="HW.pisa" content="ggg;9/10/3381" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="A Bayesian Genomic Multi-output Regressor Stacking Model for Predicting Multi-trait Multi-environment Plant Breeding Data" />
<meta name="DC.Identifier" content="10.1534/g3.119.400336" />
<meta name="DC.Date" content="2019-10-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2019 Montesinos-Lopez et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="In this paper we propose a Bayesian multi-output regressor stacking (BMORS) model that is a generalization of the multi-trait regressor stacking method. The proposed BMORS model consists of two stages: in the first stage, a univariate genomic best linear unbiased prediction (GBLUP including genotype × environment interaction GE) model is implemented for each of the L traits under study; then the predictions of all traits are included as covariates in the second stage, by implementing a Ridge regression model. The main objectives of this research were to study alternative models to the existing multi-trait multi-environment (BMTME) model with respect to (1) genomic-enabled prediction accuracy, and (2) potential advantages in terms of computing resources and implementation. We compared the predictions of the BMORS model to those of the univariate GBLUP model using 7 maize and wheat datasets. We found that the proposed BMORS produced similar predictions to the univariate GBLUP model and to the BMTME model in terms of prediction accuracy; however, the best predictions were obtained under the BMTME model. In terms of computing resources, we found that the BMORS is at least 9 times faster than the BMTME method. Based on our empirical findings, the proposed BMORS model is an alternative for predicting multi-trait and multi-environment data, which are very common in genomic-enabled prediction in plant and animal breeding programs." />
<meta name="DC.Contributor" content="Osval A. Montesinos-López" />
<meta name="DC.Contributor" content="Abelardo Montesinos-López" />
<meta name="DC.Contributor" content="José Crossa" />
<meta name="DC.Contributor" content="Jaime Cuevas" />
<meta name="DC.Contributor" content="José C. Montesinos-López" />
<meta name="DC.Contributor" content="Zitlalli Salas Gutiérrez" />
<meta name="DC.Contributor" content="Morten Lillemo" />
<meta name="DC.Contributor" content="Juliana Philomin" />
<meta name="DC.Contributor" content="Ravi Singh" />
<meta name="article:published_time" content="2019-10-01" />
<meta name="article:section" content="Genomic Prediction" />
<meta name="citation_title" content="A Bayesian Genomic Multi-output Regressor Stacking Model for Predicting Multi-trait Multi-environment Plant Breeding Data" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;In this paper we propose a Bayesian multi-output regressor stacking (BMORS) model that is a generalization of the multi-trait regressor stacking method. The proposed BMORS model consists of two stages: in the first stage, a univariate genomic best linear unbiased prediction (GBLUP including genotype × environment interaction GE) model is implemented for each of the L traits under study; then the predictions of all traits are included as covariates in the second stage, by implementing a Ridge regression model. The main objectives of this research were to study alternative models to the existing multi-trait multi-environment (BMTME) model with respect to (1) genomic-enabled prediction accuracy, and (2) potential advantages in terms of computing resources and implementation. We compared the predictions of the BMORS model to those of the univariate GBLUP model using 7 maize and wheat datasets. We found that the proposed BMORS produced similar predictions to the univariate GBLUP model and to the BMTME model in terms of prediction accuracy; however, the best predictions were obtained under the BMTME model. In terms of computing resources, we found that the BMORS is at least 9 times faster than the BMTME method. Based on our empirical findings, the proposed BMORS model is an alternative for predicting multi-trait and multi-environment data, which are very common in genomic-enabled prediction in plant and animal breeding programs.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2019/10/01" />
<meta name="citation_mjid" content="ggg;9/10/3381" />
<meta name="citation_id" content="9/10/3381" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/9/10/3381" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/9/10/3381.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/9/10/3381.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/9/10/3381.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.119.400336" />
<meta name="citation_pmid" content="31427455" />
<meta name="citation_volume" content="9" />
<meta name="citation_issue" content="10" />
<meta name="citation_num_pages" content="13" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Genomic Prediction" />
<meta name="citation_firstpage" content="3381" />
<meta name="citation_lastpage" content="3393" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Osval A. Montesinos-López" />
<meta name="citation_author_institution" content="Facultad de Telemática, Universidad de Colima, Colima, Colima, 28040, México," />
<meta name="citation_author" content="Abelardo Montesinos-López" />
<meta name="citation_author_institution" content="Departamento de Matemáticas, Centro Universitario de Ciencias Exactas e Ingenierías (CUCEI), Universidad de Guadalajara, Guadalajara, Jalisco, 44430, México," />
<meta name="citation_author_email" content="j.crossa@cgiar.org" />
<meta name="citation_author_email" content="aml_uach@hotmail.com" />
<meta name="citation_author" content="José Crossa" />
<meta name="citation_author_institution" content="International Maize and Wheat Improvement Center (CIMMYT), Apdo. Postal 6-641, Ciudad de México, 06600, México," />
<meta name="citation_author_email" content="j.crossa@cgiar.org" />
<meta name="citation_author_email" content="aml_uach@hotmail.com" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-9429-5855" />
<meta name="citation_author" content="Jaime Cuevas" />
<meta name="citation_author_institution" content="Universidad de Quintana Roo, Chetumal, Quintana Roo, México," />
<meta name="citation_author" content="José C. Montesinos-López" />
<meta name="citation_author_institution" content="Departamento de Estadística, Centro de Investigación en Matemáticas, Guanajuato, Guanajuato, 36023, México, and" />
<meta name="citation_author" content="Zitlalli Salas Gutiérrez" />
<meta name="citation_author_institution" content="Departamento de Estadística, Centro de Investigación en Matemáticas, Guanajuato, Guanajuato, 36023, México, and" />
<meta name="citation_author" content="Morten Lillemo" />
<meta name="citation_author_institution" content="Department of Plant Sciences, Norwegian University of Life Sciences, IHA/CIGENE, P.O. Box 5003, NO-1432 Ås, Norway" />
<meta name="citation_author" content="Juliana Philomin" />
<meta name="citation_author_institution" content="International Maize and Wheat Improvement Center (CIMMYT), Apdo. Postal 6-641, Ciudad de México, 06600, México," />
<meta name="citation_author" content="Ravi Singh" />
<meta name="citation_author_institution" content="International Maize and Wheat Improvement Center (CIMMYT), Apdo. Postal 6-641, Ciudad de México, 06600, México," />
<meta name="citation_reference" content="citation_journal_title=WIREs Data Mining Knowl Discov;citation_author=H. Borchani;citation_author=G. Vreanda;citation_author=C. Bielza;citation_author=P. Larrañaga;citation_title=A survey on multi-output regression.;citation_pages=216-233;citation_volume=5;citation_year=2015;citation_doi=10.1002/widm.1157" />
<meta name="citation_reference" content="citation_journal_title=Intell. Data Anal.;citation_author=H. Borchani;citation_author=P. Larrañaga;citation_author=C. Bielza;citation_author=J. Gama;citation_title=Mining multi-dimensional concept-drifting data streams using Bayesian network classifiers.;citation_pages=257-280;citation_volume=20;citation_year=2016;citation_doi=10.3233/IDA-160804" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=P. J. Bradbury;citation_author=Z. Zhang;citation_author=D. E. Kroon;citation_author=T. M. Casstevens;citation_author=Y. Ramdoss;citation_title=TASSEL: Software for association mapping of complex traits in diverse samples.;citation_volume=23;citation_year=2007;citation_pmid=17586829;citation_doi=10.1093/bioinformatics/btm308" />
<meta name="citation_reference" content="citation_journal_title=Mach. Learn.;citation_author=L. Breiman;citation_title=Stacked Regressions.;citation_pages=49-64;citation_volume=24;citation_year=1996;citation_doi=10.1007/BF00117832" />
<meta name="citation_reference" content="citation_journal_title=GSE;citation_author=M. P. Calus;citation_author=R. F. Veerkamp;citation_title=Accuracy of multi-trait genomic selection using different methods. Genetics, Selection, Evolution;citation_pages=26;citation_volume=43;citation_year=2011;citation_doi=10.1186/1297&amp;ndash;9686&amp;ndash;43&amp;ndash;26" />
<meta name="citation_reference" content="citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Trends Plant Sci.;citation_author=J. Crossa;citation_author=P. Pérez-Rodríguez;citation_author=J. Cuevas;citation_author=O. A. Montesinos-López;citation_author=D. Jarquín;citation_title=Genomic Selection in Plant Breeding: Methods, Models, and Perspectives.;citation_pages=961-975;citation_volume=22;citation_year=2017;citation_doi=10.1016/j.tplants.2017.08.011" />
<meta name="citation_reference" content="citation_author=J. Crossa;citation_author=D. Jarquín;citation_author=J. Franco;citation_author=P. Pérez-Rodríguez;citation_author=J. Burgueño;citation_year=2016;citation_pmid=27172218;citation_doi=10.1534/g3.116.029637" />
<meta name="citation_reference" content="citation_year=2014" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=R. J. Elshire;citation_author=J. C. Glaubitz;citation_author=Q. Sun;citation_author=J. A. Poland;citation_author=S. E. Kawamoto;citation_title=A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species.;citation_pages=e19379-e19379;citation_volume=6;citation_year=2011;citation_issue=5;citation_pmid=21573248;citation_doi=10.1371/journal.pone.0019379" />
<meta name="citation_reference" content="citation_journal_title=J. Comput. Syst. Sci.;citation_author=Y. Freund;citation_author=R. E. Schapire;citation_title=A decision-theoretic generalization of on-line learning and an application to boosting.;citation_pages=119-139;citation_volume=55;citation_year=1997;citation_doi=10.1006/jcss.1997.1504" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=D. He;citation_author=D. Kuhn;citation_author=L. Parida;citation_title=Novel applications of multitask learning and multiple output regression to multiple genetic trait prediction.;citation_volume=32;citation_year=2016;citation_pmid=27307640;citation_doi=10.1093/bioinformatics/btw249" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=Y. Jia;citation_author=J.-L. Jannink;citation_title=Multiple-Trait Genomic Selection Methods Increase Genetic Value Prediction Accuracy.;citation_volume=192;citation_year=2012;citation_pmid=23086217;citation_doi=10.1534/genetics.112.144246" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=J. Jiang;citation_author=Q. Zhang;citation_author=L. Ma;citation_author=J. Li;citation_author=Z. Wang;citation_title=Joint prediction of multiple quantitative traits using a Bayesian multivariate antedependence model.;citation_pages=29-36;citation_volume=115;citation_year=2015;citation_pmid=25873147;citation_doi=10.1038/hdy.2015.9" />
<meta name="citation_reference" content="citation_title=Solving Least Squares Problems. Classics in Applied Mathematics;citation_author=C. L. Lawson;citation_author=R. J. Hanson;citation_year=1995;citation_doi=10.1137/1.9781611971217" />
<meta name="citation_reference" content="citation_journal_title=Sci. Rep.;citation_author=H. Li;citation_author=W. Zhang;citation_author=Y. Chen;citation_author=Y. Guo;citation_author=G.-Z. Li;citation_title=A novel multi-target regression framework for time-series prediction of drug efficacy.;citation_pages=40652;citation_volume=7;citation_year=2017;citation_doi=10.1038/srep40652" />
<meta name="citation_reference" content="citation_journal_title=Inf. Sci.;citation_author=G. Melki;citation_author=A. Cano;citation_author=V. Kecman;citation_author=S. Ventura;citation_title=Multi-target support vector regression via correlation regressor chains.;citation_pages=53-69;citation_volume=415–416;citation_year=2017;citation_doi=10.1016/j.ins.2017.06.017" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=T. H. E. Meuwissen;citation_author=B. J. Hayes;citation_author=M. E. Goddard;citation_title=Prediction of total genetic value using genome wide dense marker maps.;citation_pages=1819-1829;citation_volume=157;citation_year=2001;citation_pmid=11290733" />
<meta name="citation_reference" content="citation_author=D. Money;citation_author=K. Gardner;citation_author=Z. Migicovsky;citation_author=H. Schwaninger;citation_author=G. Zhong;citation_author=S. Myles;citation_year=2015;citation_pmid=26377960;citation_doi=10.1534/g3.115.021667" />
<meta name="citation_reference" content="citation_year=2016" />
<meta name="citation_reference" content="citation_author=A. Montesinos-López;citation_author=O. A. Montesinos-López;citation_author=D. Gianola;citation_author=J. Crossa;citation_author=C. M. Hernández-Suárez;citation_year=2018;citation_pmid=30291107;citation_doi=10.1534/g3.118.200740" />
<meta name="citation_reference" content="citation_journal_title=Agron. J.;citation_author=A. Montesinos-López;citation_author=O. A. Montesinos-López;citation_author=D. Gianola;citation_author=J. Crossa;citation_author=C. M. Hernández-Suárez;citation_title=Multivariate Bayesian analysis of on-farm trials with multiple-trait and multiple-environment data.;citation_pages=1-12;citation_volume=111;citation_year=2018;citation_doi=10.2134/agronj2018.06.0362" />
<meta name="citation_reference" content="citation_author=O. A. Montesinos-López;citation_author=A. Montesinos-López;citation_author=J. Crossa;citation_author=D. Gianola;citation_author=C. M. Hernández-Suárez;citation_year=2018;citation_pmid=30291108;citation_doi=10.1534/g3.118.200728" />
<meta name="citation_reference" content="citation_author=O. A. Montesinos-López;citation_author=A. Montesinos-López;citation_author=F. Javier Luna-Vázquez;citation_author=F. H. Toledo;citation_year=2019;citation_pmid=30819822;citation_doi=10.1534/g3.119.400126" />
<meta name="citation_reference" content="citation_year=2015" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=J. A. Poland;citation_author=P. J. Brown;citation_author=M. E. Sorrells;citation_author=J. L. Jannink;citation_title=Development of high-density genetic maps for barley and wheat using a novel two-enzyme genotyping-by-sequencing approach.;citation_pages=e32253-e32253;citation_volume=7;citation_year=2012;citation_issue=2;citation_pmid=22389690;citation_doi=10.1371/journal.pone.0032253" />
<meta name="citation_reference" content="citation_year=2017" />
<meta name="citation_reference" content="citation_year=2016" />
<meta name="citation_reference" content="citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=A. W. Schulthess;citation_author=Y. Zhao;citation_author=C. F. H. Longin;citation_author=J. C. Reif;citation_title=Advantages and limitations of multiple-trait genomic prediction for Fusarium head blight severity in hybrid wheat (Triticum aestivum L.).;citation_pages=685-701;citation_volume=131;citation_year=2017;citation_doi=10.1007/s00122-017-3029-7" />
<meta name="citation_reference" content="citation_year=2012" />
<meta name="citation_reference" content="citation_journal_title=Mach. Learn.;citation_author=E. Spyromitros-Xioufis;citation_author=G. Tsoumakas;citation_author=W. Groves;citation_author=I. Vlahavas;citation_title=Multi-target regression via input space expansion: treating targets as inputs.;citation_pages=55-98;citation_volume=104;citation_year=2016;citation_doi=10.1007/s10994-016-5546-z" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=P. M. VanRaden;citation_title=Efficient Methods to Compute Genomic Predictions;citation_pages=4414-4423;citation_volume=91;citation_year=2008;citation_issue=11;citation_pmid=18946147;citation_doi=10.3168/jds.2007-0980" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="A Bayesian Genomic Multi-output Regressor Stacking Model for Predicting Multi-trait Multi-environment Plant Breeding Data" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10.cover-source.jpg" />
<meta name="twitter:description" content="In this paper we propose a Bayesian multi-output regressor stacking (BMORS) model that is a generalization of the multi-trait regressor stacking method. The proposed BMORS model consists of two stages: in the first stage, a univariate genomic best linear unbiased prediction (GBLUP including genotype × environment interaction GE) model is implemented for each of the L traits under study; then the predictions of all traits are included as covariates in the second stage, by implementing a Ridge regression model. The main objectives of this research were to study alternative models to the existing multi-trait multi-environment (BMTME) model with respect to (1) genomic-enabled prediction accuracy, and (2) potential advantages in terms of computing resources and implementation. We compared the predictions of the BMORS model to those of the univariate GBLUP model using 7 maize and wheat datasets. We found that the proposed BMORS produced similar predictions to the univariate GBLUP model and to the BMTME model in terms of prediction accuracy; however, the best predictions were obtained under the BMTME model. In terms of computing resources, we found that the BMORS is at least 9 times faster than the BMTME method. Based on our empirical findings, the proposed BMORS model is an alternative for predicting multi-trait and multi-environment data, which are very common in genomic-enabled prediction in plant and animal breeding programs." />
<meta name="og-title" property="og:title" content="A Bayesian Genomic Multi-output Regressor Stacking Model for Predicting Multi-trait Multi-environment Plant Breeding Data" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/9/10/3381" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="In this paper we propose a Bayesian multi-output regressor stacking (BMORS) model that is a generalization of the multi-trait regressor stacking method. The proposed BMORS model consists of two stages: in the first stage, a univariate genomic best linear unbiased prediction (GBLUP including genotype × environment interaction GE) model is implemented for each of the L traits under study; then the predictions of all traits are included as covariates in the second stage, by implementing a Ridge regression model. The main objectives of this research were to study alternative models to the existing multi-trait multi-environment (BMTME) model with respect to (1) genomic-enabled prediction accuracy, and (2) potential advantages in terms of computing resources and implementation. We compared the predictions of the BMORS model to those of the univariate GBLUP model using 7 maize and wheat datasets. We found that the proposed BMORS produced similar predictions to the univariate GBLUP model and to the BMTME model in terms of prediction accuracy; however, the best predictions were obtained under the BMTME model. In terms of computing resources, we found that the BMORS is at least 9 times faster than the BMTME method. Based on our empirical findings, the proposed BMORS model is an alternative for predicting multi-trait and multi-environment data, which are very common in genomic-enabled prediction in plant and animal breeding programs." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10.cover-source.jpg" />
    <title>A Bayesian Genomic Multi-output Regressor Stacking Model for Predicting Multi-trait Multi-environment Plant Breeding Data | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__49R9wsCpiKJS6e6ZOiXRsC0989bO7FF1XdNTikSRKGM__k1m2vpL1Nu9n43bJnV24EQ0Z_UWo-Iq0zZ2FkQLL-iU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"XANhDW3tbMOikS-cn6GcNFbXhmA9Xq_8ulHSRcs0Ga0","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"494285","apath":"\/ggg\/9\/10\/3381.atom","pisa":"ggg;9\/10\/3381","ac":{"\/ggg\/9\/10\/3381.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/9\/10\/3381.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;9\/10\/3381"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;9\/10\/3381"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/9\/10\/3381":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022E\\\\\\\/uV1X25VviUpJZztxPWvSORyEzSNmxEB5B+aIVSIGjV+KIT3tviTOnisaIj\\\\\\\/V8kJuZdzAJol3R1QMgWd1IbcxS5vjZGADcNNFJZ1uPM3n+qxmPOREZVdMO7R2r0YFwmTLSf3mdVZbhlR0JRY\\\\\\\/Kx8Sx\\\\\\\/UVdxw0Ihb4LvQ4bnIp7yRurEYINcHXMihGRMQScR2kbXgAkBKBdgYvUYRISGSCX8hVquKwZop784O8rnZGvrLRfPOL88uCQbPGUH4c3ZC3KPL6vh69N8N72tqV3xSpwXt5L7IWraC8o+\\\\\\\/BRu0NWUCAILGF\\\\\\\/q88a0rcp1st1S9mmmmMhPmVJm\\\\\\\/J8LjkIuePU\\\\\\\/bcte2F2Ix27VfY+nnyYpMtrK0DbYQNh6EC08\\\\\\\/N5E1oXJr65nqku7c1+eREDJOqEniA6+6TpcfCI1a5FaoSLPgVVSyXq5\\\\\\\/biN6W53gvccaB6NNqh6Qq2ezSa+pPlzdpTGJcnHRgWR8F2y3t98aOKPe53xP6Mm3jzLoe8m1+0MxkDAeiobbkddvwElZigJ6LaJBOvqkej2Q+ADsIAWZgDvoBSj0X0Jc\\\\\\\/1Crv8+5ct2uhG8rO6IlJHA1R\\\\\\\/BNxmv9TGOcWq52AyP4fIvo3yyi9ATkOeKQyvjbYHIiAP9ZKEapiH0FAFY+qMoyScouaWMmj1ifBtfN9KMndmVriucRLs89pYc29jC0lHgfvehwNCaq7SDmfzauaB9+DUyhK63fM3juxMDsA2mC4Mm6jicIEr1XtYQ\\\\\\\/UbCaipJF0Og0L+zFrebjC8zMp3okyHBv3AUWl2rowIlgE6jRqwvDzNKo\\\\\\\/LUkLNHFyHBP7Jm5W5q9pRetFFD1fh1GJ9MeDK29BE9oyGl2Cglt6Gg6rSHCEh3zjN\\\\\\\/a\\\\\\\/F80G18ccQGf5HUFasd200znDfuDFk7miO8Gn10kU0V+DFVpGiK4mJfh\\\\\\\/B8SGw1Ngf4zAc9rKt5gQlAlijmQZd2R0qDgQjBImWvkbA44Mq4eCBHfkBb2r\\\\\\\/O\\\\\\\/kJ3gWEmNXo+8rswB96cPE\\\\\\\/TBfFFZRIjKjU1NOAHn4A+NWVX2KKzt7xg8f4UUz3dgm8KnrX71MQh3PfAzZ1TbMcl4xVG1GHcTwSSZwPwoNb7H1Eu45kGmfL6M2kS3yepkJr7VtB88N5qYuKijXfZ9bx6tfHQRg6ZO90GslWWXNTPYlus0ERc6RIii0tEnCEx\\\\\\\/jRpJltUCBMBLpDwPuJYIWoyhjro5XCrJ4RQb05c+d+3dVUK2gAfxi6HIvDI+d0QRWWu30KleLgTG\\\\\\\/IQ81FFWOMgGWAO\\\u0022,\\\u0022iv\\\u0022:\\\u0022nkGAj3G+mYpODZr8DsPlGA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022fb89fccc85400e86ae623e06c868083ebf9369526884ac28ff83dd10f921dd03\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022c7jS8tkPDmAC7CL4ANKrbTnGb3RN5W\\\\\\\/JM5G+VzMzGxZL0bUOZvJNsAhZiWNW6MJPQfGJdW7BJYnx7R9GkUOZk4PZTz2upGS0lTw6CYavheivL8uP89Ir96bNnX5tznCaWM5SRN03ussj5F1vwOk+EjfHTN1negI860T5Xrr2bbXtz5r3mJeeKlgFr9nuJ2a+HRWmKWtShHvakXt35CLX3Em8sid14RjguWWLkOtxJvIIwBl3nm36x3b6Upctmoko0uFPG1kFGCQVs31pIk5Pk7zB5VsR7JOJMdw07kCfLpJRzHS9I8TbOPz6hfAbqJyph64ycvKS\\\\\\\/V+kvFI+f+ae7yQ1o3ifANGlamQT\\\\\\\/dQluJBbiAfY8L5IL+1dM4OQ1nmdKULorv7K4ICzEYVZjMV8WkJ0bpJ32u4dxyy0fuvYD65eweA7ZUf3ZT9YOHIie8MBeHLUc7bqIfSA5jPAs0l+7jP7IFWgkCPJuBmJ2OEydQd+LeZYu75MhQotHq\\\\\\\/K+Tms9ADtDUNix4lBvZjhDodcP1jia00IDzWe9xtdDn7867sRCEbUCDKp7FqXduqo8rVpFvgj2EaEU\\\\\\\/1aEBXCsm\\\\\\\/sHr2AF5zUWah70kr4lNzEmFW66u2RvO5fc9r\\\\\\\/AzvJZnS8i0GA+awl2IJEi8P\\\\\\\/HONuSr1GldgmiLaFejXQKgwveJ\\\\\\\/OQtUcIly82K9tscGmf5wlCIitgSDOulYro1MePLQ11CdKG7pvj48js4QIuirvzBRyek+k004LVGUyyMd5xVpP+GF0TsCKrSIiGpgElCmjmB7w6ZQNBcpSsopC95JjMm+SR85YMyc7WxRN+fGabS0zI8QByOclYh20isUpSHe41eNYB02AbovDVuCTNkf8O\\\\\\\/0qK8s5gVT5yA9UQkImzlGW+rg+HurD9SZyMPzcJffZszJo5jc8HHoY4xbb+fEiQAZhlzt2FvWywuQw1dgTUsZ6A4osFZ7ZdAVY5mCS2Q\\\\\\\/AmDwJX5ccSh0GRwvgKcelwYTPX6sjqrP0jhqQ9mXXlwZ7qbIkTquyPsEw3cSv\\\\\\\/FEsJSr1fQ++Wq4FXOxSXbFF1DSLethaACB4SwXnPWFi1rtderD\\\\\\\/HWbO5wJevfCE4VmD6mKRf5hK87Hu5azYYBXMDKN27PcMmWT57G3VUsd\\\\\\\/zJTMkLVhZ2riKsv133fOB\\\\\\\/aTnhg31uf4qCwEbMgc0HixgvD4bNFo4FWJubnVheqVONfdGt6iTMxBhRhmasZfjIVqFG15bytDsa2XHfklGKbIlqLSR\\\\\\\/kioGhlmSnjArQ2V+ZkPC5k6zjZVlJsSUz\\\\\\\/g8p66CbhMJLXxDsPOpHVUxzKBGlrz9Ch8NERYKT5j\\\\\\\/hwvdAeNBYl7DAkrQi0OBPyiBnITcAac2M3VKmIFv69PVHzeNbOwB67Mkm6ixZVdLbjSmfNVtKMXr0NsYSugM8sbYi+2IWB3e8B1QJmLIhhmJPyiZ8DCiM2ljey05otwV4z+N0yGkGPkWAGHBzHaRXGr6vPPluyNSU+tnRkLlcBTjdTffO7g+F2LMCnSZ81koajxOpxNGjmifMyyPRY8CSXi78s81daQpkOpn1IACsZ2TDTIcptcM22Wt0WIS0t7uf7xFzi6ny1PDftytzXxa5DL+dEvGcswIfsu\\\\\\\/P0AaC\\\\\\\/DaKGG02sUIn1UKgaZ0Dz0uV73l86oKORn1cz\\\\\\\/+TvL\\\\\\\/kifewBCLDgcCVRiJGzbrLUe7K+RvXKt+0fvyHTaGTYBmB4\\\\\\\/M3isvjfy2jDIO9+XYYGALJnUL1u8AgMan0lmFVOdQ2bJmvDCoBeexfc7h+rd9AOsW7jh7Z6RsjWb4Wy8EWeaPMCytr\\\\\\\/O3n0ylCGRVEkyUj5OWaVPUd21D8zsHHf11NdoJjxeW76EU5W0jNPGpo38WmfUbaX\\\\\\\/yrz+oYqc4ikvUnjfJiNaxSjU6rhBty7j2gEZDYVssxhlf\\\\\\\/g\\\\\\\/ozOg7kqEqb69BIv4zOt84S\\\\\\\/NbPUc8MO1o1Mg33O3NncwGDAgn\\\\\\\/e4EzObJ9m\\\\\\\/pFmGfx5zSHCe01uydayB8AwBlpHR1DNUhZR6dnjfUxoVzhO1MLLRJEKezb1MOcsFH2Y+gnm2fHd79e8EaxIbRbStRgaXExeKTofGKq8+j\\\\\\\/Vo6nn9Rw80f0lat1nWjJ9xSXAQdqSZ+1tyP00yV2kIOMr\\\\\\\/B6rJ0JcEDuFCxHM1IQ\\\\\\\/1QvLTnLH9p7YprwznwASjpLaf\\\\\\\/Un3InLXSA3OepH1KlXG\\\\\\\/g+GiDWLh\\\\\\\/mvw44L36DF+7CGso5ld0vb3bUdAE4d5B2mMhvuNEmgy6hk2YmQw2TK1s5mVlHsDfQuktuqfTy4cnKCs05xv\\\\\\\/3qgjvp\\\\\\\/CNM9sXFm7qcm1ifQOjcxI09nl4YrHJ+2Ob34otIJPuddohaWqWnW19MP9q8FXfExlSk5SfYUGIyJ1eHeIFnJxRjn5PHOItMNSN9ka2jEHufir0unIOzWAoLZID9nlij8McJLuAMfQRx4Au\\\\\\\/ZCnWfrKH2iHrNbJ9RkRqFsStvTUr3Ul+hP1jvtgFAKJxBdDfHhKVlBX\\\\\\\/nvP6J509omKEpzaDUqSZFQGxV1EKBrfNPYNCtnx4kMyckPe8Z4UcM4QZEmO+YhXyL9zpCe2G8+9YIJkiG\\\\\\\/\\\\\\\/nflRPaD7dimkkU54vIBrsvhzXUnOkOrZ7vwxzLXoHuL3v5HD1r9eykKYJfaXRSUsDwnAR5REyTNNVdcjLhqoeiJrSZw5MpFyiE9v5m4sxHJ2fbjAEkaI\\\\\\\/2UOJPGTXTDMAeayNuVJRTzYR4Jy+LOuELwLOkS9QXMYBArT\\\\\\\/T\\\\\\\/TJSL43zNtVg2\\\\\\\/NOXXvRQt3tGSlmSb1XzjPHBgrlV7wcrSQkIDe+rPPdYTbpKQuVh+LIDfwtqTBUe\\\\\\\/G\\\\\\\/na7L\\\\\\\/uvj5z6sv5TUhlo0fsfBlcfEUZZln8qx\\\\\\\/cOtmE8\\\\\\\/yXJ9gs65IqZw35Xmq3Y85PDvU0TozyAW3nRXHRjjyGR2MC6CzsU8CmIWjeCjuY00HxbiqSUU9tKFaMXBhojyU+LwBZMnMe7OaPwx\\\\\\\/qb4VeJKEb6EmL6j3SDtafmrb701OXa3UKGIBCCSCLXfOH7k5rmmJpalY+bSyjQ0urt1ofWk0ibNzTwWl8OnVon\\\\\\\/If57LHb1qUW79T+DOuMhBnriOsUKi6beQD0J37CqNOGfFfdlf8HNPptAL\\\\\\\/rglm2dXWN2ryVy979oLQXl7dgswHVD0rgZosRNIqi4bOjlrsc1noFmPLzDeM+Q8OB15nFHe6JZAUfpAb3+KIi24uhkCA83PcalSnLktn4MfeuhDNd9zx4hEiG6DWNsJIkeBSo4aI3lPiYoPYYHPNkfagRXlT2M\\\\\\\/8mw4+doEOhB4rBo06XTyGCSQH+M1qnJrA75aKSPMzjEDhGISZhNMCm+6ODbAd04ppIy640cKPF5AcUyHVUSX97LiwEJ83fFqRac63P7jTDMj39zDo+bJWaJBTkwXK0A7wnZ9S09b6pKuMpkkWwNM4U8dgJw2AR\\\\\\\/A7OZNsEqpTCh69r0vCO4Jz5Jz6IgdfdxWtsSORCSdjf8SEdECAQpflbu4MEsqZT+NSK5DYtVN37QH1vRkfsnMCU6DzCkkFS\\\\\\\/9\\\\\\\/t+GiMCzYxD47NgYZnA5hoyJpDhFykz3xQczgu818fK6idUOZ3Wo0YZ+40IjNm\\\\\\\/Q+Jf6TJ2xsjrWot3Tvt2AOXJnkDNX2lbxYEhtq\\\\\\\/wXSLmFkfLGHeBgF8AMuwtiVBOXn8cWrz5Rzp3LtR\\\\\\\/eCxu9IUXKc4ErRcw4NV46wJoMN29xsc61w8MipAqUS4qz9+h0aUBBdqiyRjvc0rniqDZdPXqmoHvNLs6LzFadps8y\\\\\\\/qEjBpsZve06mQnQoRtAvJUjChwjQ75lb0yQi1wM06S8aW3qfROQiTJhO3DvRk9yKqgmLMre1UUu\\\\\\\/G\\\\\\\/Kfri3m1FtwVNIkAyEivPuXq1j0RlCISbjdVFj7ec+z++wJ4\\\\\\\/fhEmtnbc5n304EFhdDc27rd4FAC359gaDHuQHhi03+nUPegkpLEtlCXCJmmTkWczU7jQa\\\\\\\/2+Q0PE1Ta6Tb3ScrWolh2WdQ4VEvTtVurhZCkBYb75J\\\\\\\/Gsy0PLAGrq9uv5CGrYyOlYPxrqZZoAW1Omt1S4dm0oY2zDFYsWhhbjPArei8ndMCYsUMMhaBpegvmYbuHs50qWgNx82sYnlMhQRN8lLwFv7yY48Rjl1U8QFWGBx3vlwYUIcMllG5KVORWXwgkvnbpZ5PdlraVSqcIsyoPKyHu1MHJZrmUjn6waM4r5Ka419e7MhvDa8t1\\\\\\\/mfA7k16ioG+iGjm3chkTiYVC9PwK0hYkbhyRB\\\\\\\/CKR6SPntsyR9IJXyO2HedZXMwAqTW0onvYBDbs0MJ8grCJ9xKJ7gEs5edmJr+wTmDe78iPxvP6gZMQnr2tUHriASrrkvlVD1sab7gGt\\\\\\\/XdLploNj\\\\\\\/zUe9c1\\\\\\\/TyDAKJ6uZjH5gZY1g7IleqkAiN76BamKm\\\\\\\/s3W18oSGdNkctZnUEoIrV1EAB18fc9f6FLRpAqdOLbR8KLYhphPq6ZfpGvkhPT14pGDxEa2mtWgkd0G6EgcbLXeH7xJjv0fxuWrnImvAWqjqMtHJwR59qLKodvEwbGYRJm3vBYQt1z4UeM8OxNYzrWvzzDaWHzgLAmxCHoLjjrx9OLvVXi0s784\\\\\\\/4pLQRky0iE39AJu3HAY32HpohYFOavwYvjUKieDFX9i6\\\\\\\/jtc6vhIbFG5vrzbvSU8E8a7GFNkV6fbAKkQ6SWtbgGZgmJd8CHIcfmLATsWwNjWLrRDM39A8KKp\\\\\\\/NXHg2u2pfqujCQFbiInoc8bUBWB8UhlkU9PvPBDs+mWrrNp+QkuRoax\\\\\\\/dQoic1xv8rHfd\\\\\\\/cvHqMhAoPtsM7Df27WYU8b1j6O0Hib0UqXLhrfQiHnyer7CPxpx0Qd58+ciJNhsh8ov9jtAyAgYB5fbnfWPg7wAcCDC\\\\\\\/\\\\\\\/g24jEh\\\\\\\/vBf6tOL9r1LsSvvcwsYD5xGtr6eqi3puSsfa2hiY3wRvDexI6F\\\\\\\/yzLJaJ72zvHmBfcq33aWANVyX2XvMbAZqKNTM7C\\\\\\\/nO7Fwd5KyAtpTq5W\\\\\\\/VzDoaispEKNRHwdEbqvd1x9O69atrfu+IHv3\\\\\\\/3vTjzaxS\\\\\\\/ff7c1gnJUe2gnTiMRqCGAywHAP\\\\\\\/HaA2YW0PugtAUwm4u2aIxVbtl6kS5r7F6+smr0wc\\\\\\\/ezrCe\\\\\\\/o1OWkHbUX4OtgH2E54NgY59ItbXvfxbaxmU4mUEHTrMv9eNGp9wbLBbv47EJbj3O+xOO\\\\\\\/PH\\\\\\\/wEpPrCzt7HaBRc6BTpSGPwetK1FJG0AbVCxWNxkMD6LaTyBqRaheyVL0d5bDj\\\\\\\/4ZA7KCrAOHKWJJmNSC5S3GN+5jBSj6LF8rjaS6iDw8uZKCEIxgnuSOMGUC+vfCkYC4KKRqUezkiN1wQv8PFLkBsm8MSQb9VEqWNSRwXnVubkzGDH5gXYSoD2L8dUUE62X4cI8FxhegMZTsKzNKPSs1QldonopMIR2\\\\\\\/ifDTZO2T9UTvNvLQ0+kKCgIYJljqbVU6Xq06h2uhRiuCpX5A3zlAztMhSb+qURFFon6FiBbLeUFZHNo7YDkERWqBPcoVyy94+bO6vzqqWHMd3Ipv\\\\\\\/KX5JLfLvhq1+KVao9keMuNW\\\\\\\/ypNsporG8Jc9f0eyUdMJIubqbNnlWCkXNTv78Ixkd3rqbBnciqd17ehQ+1JHXZtT1RLbTCs66Yst6tlzc6OifmwqwF9M7NMM7V7B5bT4fRQwwqBQFJi8AAv304NK6A9eC4sdGykCXx49hZB2vAY3bexQg0JMzv1NEQDohQ4Ye54ujUvLln4S+p1QOXI2CiPcKio2Dvtm0e6oKom52SkF2PsbS\\\\\\\/JKU\\\\\\\/sE7voOJp5vLkfow+EvAEkgRDt7RKpm\\\\\\\/SSM0EP952Fm9Y++1Z530OVLgByzL0xzKXpR8I2jNzdIknK68Xa1PLFPoN5OsI82OyAENPN21XW3lXjJvVRrYabGF4PPBd6oYpI1c2vOJi3tRDOedBI1Xp8mYVJswkZRjxX2\\\\\\\/Or1t7scvSh140KnjRlo0+JkIkSr\\\\\\\/03EQYAiH5b96Tn7PzjnDD86re\\\\\\\/ft+oq2m5Lum72nIuk0hWv2ecM+W\\\\\\\/HzzONqTvDhTATWb5teznVsEUTUi6lFEb0DkijgwwF5ZKRLuGVYxPAUJ+9\\\\\\\/ST5yq4cGKQkVl9DlmlDXyH9LrilX1gZfYW7bcPKXS1v+RAZ1BE8dq9tUwf4OuWkl\\\\\\\/VK\\\\\\\/vpvxCyLTAZ8B0ZsbBFErdTFtIiT0zmZyA1MF4V0Bk0Wb9r5Y0ON8dwErHxcVIoppT2Hi6\\\\\\\/8QZTZRDRxP15Tn\\\\\\\/AqvLQIFPIVd\\\\\\\/rmw83GA1RNxV7Qkph22Ei1i\\\\\\\/dYhXUlyR07p0gxSdBkgggm7fG7kmZeF1dKEU+DhX2tmjFL8bYI3pVVqJo25W+fWEy2ueELKdPZy8XEjiZQ+v+M\\\\\\\/PcZYp\\\\\\\/XYCZ05C86yoAJlqnu1y7G1UZmFWxXlWZc94HSj3JY3B93DEzE07hUlYhTK1v0dTI3q9POCdlydlT7w94oBgMK2JNE01elbMyJIN087KfVTZUsAo3jYniZoH\\\\\\\/Oy7KRNvUCQ5fNtj\\\\\\\/4nSAVv3Q8UQzO\\\\\\\/XNjOSk+7E7W3de48xGpUNIsr4xQAlc2sk782cW8qum0WsCoVdCKoESX71jiP+T0wNGVhVvLpzdf+77875HoOfMscfmKDrcM6N+gTDihKF2iIppYFFMJX5fI63\\\\\\\/aFplK2J3nWfJLowRvJ5sFz7O6gNSGLjcCCKdAGgImGUoUbLDMBtBk8sPLklLe8CDXDbhOuplE85DVHqPdcFAy\\\\\\\/UDLNsrR6uEiUkT3tycMq1aBXN1DQig2+V81+1zJh39t+w6FFX1WNAthuBoLmFWbnyYYMF8sPPlotIV6d7BJ29gd1y1J8ITxlMCWJmbVFMMtmauV+efkSUc1Cdwqzq7Rzvl4P5W73a\\\\\\\/4XPUpCsZ1wLwqJZ3jaImoe86OTwvXhE5hvhEWkQo8XB35cgqtqvseGNX8PYaOVWGVqCtoudG3uHOgyyTztC3gPQncnkJGla2cEqfSdmmw\\\\\\\/ogeabnp1OhCwfv4Ctr5xkyAoILBkUDkKrsXiDgzj9yPPrxVUs9w2ewzBoKU0BVfMo+01em5SXFozIsKTMSc+0uVF1LGEwLVJMZjbwQHGohSxMMzmV+DUpu7zXK16b63BLu1q2Viq46pQbamDw\\\\\\\/HJWQKe6chWsgalnR1sa9fwoW8gMBL08olhWrDhtD18WHnvU7Yyh5oHToyAGI1CEJrMoZBSMD9VIjppZET1sY3QNKTxel\\\\\\\/+PzdMkzTb1OwMAz3JlRGXfUmmLJqmCfk+R\\\\\\\/hRhARKT7q0Uck\\\\\\\/VUDyd+BpOYDmZ84ajP7n6Hd0eyAg1RcGJdhGN2PzKyOjc+\\\\\\\/lqSSmfNQzBA7Nu0AbBnUI+9y6PhZeukwQg3PIymJzUq5wHenF1gQqXFB4vqttZ26Ytqzt1iXCfpEwsUrjSSOUs8AOumW0yZvT8Z65\\\\\\\/S1lmdUOYhtx2Juq+mIMGjy1Hkt\\\\\\\/hRCoQVF\\\\\\\/cJe6U8pzZIIEo3lRtgmY1Db7xg3ee8IMF4EDIbpM7nyZnA3hLqE4yxzorJMRo3EAuOr91YDKcygJzOPXGBC\\\\\\\/kgIeEdrU3VVB8h\\\\\\\/c5A8JtPiLRqwUvw0OwwzgYBN1Pc21UmA\\\\\\\/55ouKSYJ\\\\\\\/P0V61FTi3jq8dZIosc2Fupc2TBH6C7T\\\\\\\/JpVb8TuRGwqZcazwQZnDEP0BpoAw87gmLn24dxY5EVjQ84TbhqG4\\\\\\\/uo0MEkfaxmYITExE7BX1UkGsJS6LniN9bpBZGuAOKm93L3ZhAjDdliThGeBuqWl3NcZVHtDhBenyzGHfN7Dwi0AqthUV+RnUzf+T6IpBTF30\\\\\\\/vdhiG8ePvdjA0ttUTkKfnZ3TPn3B6bYusqM5Qi8fniUvpfJFGVI5ng6XRJ8Ea858lc0+NnvSfY+1rYCzaiALWEgGM8\\\\\\\/dCwnm1HYljcP0nanYOTFxPRUoHJk25RxleltZ\\\\\\\/xPtG7DXO6iQItfXdGNRtCVyeH3iZqLiPgcxjDga656ToiI7yzwM3o7FjRumOzFw7eLVpDnGYvLa5qu9hi4gExIXk03SEU9TEX\\\\\\\/qSPJDOETPoCrMORAeD3ahUuyMrW960V7gWkjkAfZB4sGtvPVvbUPchl8VPa7eH3sehjcZBKn0WiGHlrRItO+6LobKOW8QD9vsAlP8xd670TnPRzklGeIM+Bu1LTdi2MprGngyFfQ6TL8mPa9kumy8WuVM5GfW+lbm9yeXX8kqGOrYLqcFf88mn+5eE66Af6zNumICs3LKyqBqcM79H4gZn7htVUEbhD0mAiYLr0vA5CymeqLwprz9iiFfIjihXhLdvl7HinqAlMRpqpiu6TI5UT0vbUpOkxNgl1EEOa4zT2j1r78PHI6O8IP2GqkHw8A3+PMV9oEwpnTXT5kIQ3F4JQxxv+3t7Cn6r8uy7GTF3Hh9NrudkE1jWf\\\\\\\/IYq2RTqPOeFGkGFC4kP\\\\\\\/juzHKSAr7bXQ8Q8mdANX0XbsJPwAMxspOK6lXIPU7EqY2YWzB1P+s7MOaPDD6aJhBk18XDuSszxeLA+16h72qAwMVzZ\\\\\\\/i8qQDYNDE29\\\\\\\/hNPUpkoUFOkfRT9dqIA3pX0XhKjpNN3mj4n7qmLbgE4hf0WG+Nzmf3K0UJYDdMaqtWpBi4RKLK837C8FNwek0p43OvMCkyQZx9+y0dzdj9HF0lXaYbbCtjoGOL9IidWWOPjufSy1en+891jN7oHV6a6\\\\\\\/7YeJFlfvhWLtPfr2esVke3+pwrwHAE5aK58\\\\\\\/Cxgcbe3\\\\\\\/fAzCXI8VSc\\\\\\\/TMSEY58SWqYYBNDghd1rduwNXcxvoHCpOARGY332Ft+JX6yAKBMT9lA7fYRje+xFdwQDu9UTH9aombpfk79Tr9SDjGVMDhiPMozYe8gE8FdjLmqCed4hJnIWV3QJBFWJfMCn1uE7JOlWx77u9MbRU7gXY2Q+SAn\\\\\\\/HtD18nhJbm9noJPllzlpmk\\\\\\\/\\\\\\\/C2Hfnw+pVu3OT2mSLoI9Vhh7r9nDVN9stQTHJ5nWxDopTGoS09wQK1yMRco0z8gKZ32vzBJH74tQ099\\\\\\\/SgxZPrDLxxIpuWIvCXdLd8vCC\\\\\\\/+0wRFU7aYNwyN2UenwrPDbQe9hWPTpNn+pr0ltz0APL\\\\\\\/5uoD\\\\\\\/ZmasHPoh5Ve8MVqh1witU9LckIzbwWtby0PA1YbProjPbSqUpTDEOm5bb24d8v0ARu1y3i5svXpGF1+HPcZuGSvIpNeCJidQpebah8OY+zOnWwkesJlX5Ajw7qpP4I+kNNPd9ccVQQ15yeQycFd5gS\\\\\\\/Shl19cQ1OlKrgJIjcM3P4sv3wzNm\\\\\\\/NeFdng0x5foxsO3oMEqeJAQzN\\\\\\\/dlhCzlzxp0aMVDDJE8AVIpUMRQYNqQvLTRYA0npZ\\\\\\\/iZOoCKsOl\\\\\\\/whuVMUYSosnL0SuSGJAN9ab71JO7qPjcjzD9SqupYk9mHp2jSg4OQDGx2yxjFBamf6MUDcoThgjxUlNhS8AoSxSuxWmJ5JJrXwJbUEVybe46S9beqygJK\\\\\\\/E+mNbo2hC97Cn7L2NuTkTicZIV7EYnkSQ7kFkEAYF8CtoCpLyEoFcW3Ga\\\\\\\/1XyNrdBstwwI9mz5eHSN0dNwqcnOuz6\\\\\\\/O\\\\\\\/3C73BZOJ7pAL9o7k8HbVDQZqPO0DWia8gdVHyvoT2Y6OD+0GxNUi5cOrp3qfrr75I3tkPNzuRBZL65ie6UakrHiNyWOhZbKKUTP5UQc8jh0SxSAX83a3Yhp59hIqUIjBxmIu\\\\\\\/3j6XOhOAdnP9rYHzoMZubftcVt8mGHJnnux8m\\\\\\\/6tPylfuZLKqrvkHBX6r\\\\\\\/1CztTOGd4vDdiSAizlZbYfypaf2CEQy4uGuYnc2hdwYeMOAycospftJSyNAspJw6giM5gKU1RsL8CsYCA2LD+Xbt04bpA0oms77dVLo1trFF9s+NUK966bvgzuxHsMcHD+Y02ItE\\\\\\\/xwZ2Wx1fGQVh1tRgL2DzhRBsvV2qcuZgeLQocsq+nmKun+i8w4n+JOpFs9mnP+C5f68xWIJ++TAhETrT1f6dzgHgWu0Y9N+h1iHRUBtp9kjeeOnh+cPQK0YuQ9HDPaxAH5oOVHomdXX8zjL3xPr2880uVAlZTy\\\\\\\/eRJwp3Ejsd1hzEpXbIjiOwjzBBEUS3JKk0xSw6dFliMvxjeiLIRGdcGUnSUPOGBmwgnJJC\\\\\\\/Jyx5BkibDSC6daTNpfv1OawBXtg8e3O1gXdAZgjTXEPtUBPNIDWxty08veStkO54\\\\\\\/DLZom+njds4oZW5zAjtd0kuAFioHZlgxqcXwqIAuYxo8PC9n1\\\\\\\/mw5abYE8F6MvQ3fsvoqfq648UKsQ1t0b971WKUBiRVwxyoROsph8091TOc2dc61YecD\\\\\\\/cHjETu0pLt91v9Dt9uU3Pn97ZCiA8s7EdttBOyVnT4irenrvYEnBtq1gJgf8ebszHq4OuGPbfRqqyLfeRJ5XOC\\\\\\\/yEhhVu2ePFFhshTlAoxS3ycL\\\\\\\/whgyBbYQoeaHtzjJMP9iIrRbR4LOrYLr7PKDAp0wlZTBG6UzRwR3Qah9+FEMH6By+csOu\\\\\\\/9LoupIgOQi66\\\\\\\/bRxGV6JFxo5F\\\\\\\/9nN7WfYbeEY8x2ReEJQ36QvyH0BNEhlQ6fWOQUv2PmG+CUgoc\\\\\\\/JViJLmzqObJVOMG4kucGsfcTOx5DnuAhMD+OCwdiqsnbIoaYjDHbHhurIjJcEbTsiqlofrFkGpFpgxBL5\\\\\\\/JjYlmj3x4FBVaLtu5LdI4hCj\\\\\\\/P4fdsho6wsCnxThzOPxyJGA1Coad8R14plxEnUaEy2zXRuzfajNS\\\\\\\/EoGQuJD+zjMNWKEHuRFagEZwMxFwErUSjeACcpkndnqfHn5hQNytjs5AjJoEjXglyf4V3hnpDFhgPdZam5ZciwV0+EDz84Kq3HBGJ\\\\\\\/QYAzpm1F3SNZHATotsW2h2q9RApVqxvxREc\\\\\\\/+O40z8IrFF192Dm\\\\\\\/BxVCk2kgoKAy8bnq1obxoCE6Fu1V3RLt\\\\\\\/pk5YWj\\\\\\\/tGh\\\\\\\/LbzegHZmvOP3cdR+DO4P\\\\\\\/BWi5GSuEWxG0DhLW1PKp0\\\\\\\/ivXkvZxGRGhv31SfUABNf45e7M4ao0qhkiR2s6ck4rg+t+wueVWZ2Eic1\\\\\\\/8B4BIguBZwOoDb4hZLMMPjTbCYjhCxJPNcY\\\\\\\/X\\\\\\\/uCCm59AguzTs4V0sQq5jTIwv+9FrteuV5OI8u\\\\\\\/Im6ILMK\\\\\\\/LYZG7j4Af\\\\\\\/oT12Me0n4IWKtj4kViOmPywlN9dGrYPIF2saPCIVyYTmxcYj\\\\\\\/L5aJE8Fjp7tuGZStW8AHQwMQXQN17L0iWDcX5tcIW6u5Jtm2XnYQsAh2wt4e+s0N5CNCOD497GN6RqtsZRhckUiE2+CPATH6J2bTPySait\\\\\\\/nfPtjFZBzDMuyEhfxXJfrXSsyUIqJel3qQQsWGv+TLWLKRP0UN3umESOi4j\\\\\\\/EMDkri7u4r1T2MedB0dCP1fr6w95ssKERdxkL8gqMA1Qtp+WxYpHrqbqIgGVgEMu0VXoMnMW4OQjnV1nbAIKKLc9LOOyKlxDeSFEVOZNlzyffj1uWRk7rFImsd5OIt6CBIvithGGbp09krvQV6AK3TZktPEcH2bANgKh5X14x7u40yS+18uy9pEaxMAHSi\\\\\\\/aJ9iLNhAEeBUfLlS6pKDB8awgP8kKxGRFh+\\\\\\\/9\\\\\\\/\\\\\\\/p2GR2VbETz3JptdWhEgK\\\\\\\/ElKp2ot8oRSMI6RPWj3mXpQ3rfa0dOiayT4FutX0MPnahM6whmdxctj5OC0cyJawPyfoUDKZrYE9Io1D2Xl0e+uhvC1yyzY49YfVdJvYaMjDpimeBJIMLrXvr3gL5CR3k6o7xj882SprKP8063NYG5CnTgvs0mFYWAe6c0le+XmbG\\\\\\\/EBpP2rmilaXv+1zSsEwWmjmzXy3u7SxFeiHjxdjMC5ds4wNP2bWwxoEopOzpoj26KkThxfXAjDov8OQWEDyhIPBfZA1kZm8Y00lptvmoe55Ag7+Ov8U5FTGEokNQjjExaijHuCDItgKuSAkeXaT3eLYd\\\\\\\/TxlWFqYl\\\\\\\/lnO8S0M2abbPbXavBIbT2LJYkMK8F5qRfKMKvviccEgOWSe1eEA\\\\\\\/9PAzSxcjJkPcM2nCsd3aJ3QN8tCA6b2eUlS2kClWtj27J6oP81ZbooYpYVsDV8J3+VpDYi3FM1mNiEODpC7oZxfJk+B3YOR12eaBBnzQYKyNlU1MXR\\\\\\\/k2r8BRRe0614rSOKQnBvHjqDCVhZ1HwZek\\\\\\\/4\\\\\\\/RXeZ0WJjecWLTY0KyEXJ2G3nRb5CqkHQ5ptPWTUAtUP+QRvU2kqMKAncVNeIZyaLuwkJtKeTInp1Cwv3keCoTf4qzsWnGQUYRBdw0GTWJrxQM2Eniy5SVh41gtB12wnHTvVTlbG+yAQ6M1TfFhIaQ+\\\\\\\/ekfdPO7UdkDUdnwZlhg06ILxmBGxuKMm6YkTOd4VChshIpjJ\\\\\\\/xNbOaIRF6uYP8z0bsiul3T8bffBk59MzIl5OauJRSn5QtyJsTtMPxLy1JNKZr4u3HTCMwOGlr82JHE8+\\\\\\\/tHqA+nfe+R6J+wvAA39McYuvXlIDVX3Z+FVpTLf2gTbCaXGCyHDuS+Kz706xoapkm2NeHCnPbpRSB1Y3PNQhZ9HwEi83PKpi9RRnlfJnyQC81lgcxijs46AGc19UkgdiWtmpTpqcZExiVAX3GcEXhfSQj1Bvk+cXq20alrz\\\\\\\/FjbhlFHh7gfvfHJqaB0yWgorkz7dbA+hcFs79nKCspCux4\\\\\\\/SqsCh98pc04iluwtv0LxGmx2oas9WXyHdvIS\\\\\\\/BAl3psO8lhzezl\\\\\\\/E8uYkoGMZus3AxTs+um\\\\\\\/0o2YUh8SC4S0FAGNvhaQ9wRuO9yagLpzxSAS9cavQhGeB+LoAWTvWrZtXILGudrCgg\\\\\\\/AJ+hJ8gYvxwgHxQCsGhEgcrHCX1GoYWQ12v9Jqp5sW2WLXmE\\\\\\\/3EArZOOeJlTcGiR7tG8sCj6scxyooKsH4bv\\\\\\\/nKXcJqDBOfzto5uWs31Vp0G6ZdoiZ4VrZmdlfIMacTx1iUruP8Fuy1QbB8agjw2GYKY5ed6aQChR4gyvnFZ6DS2XY46DUAa6gfvCBGt\\\\\\\/vS0IUk4FT0M7IigsIEHQmf9U89PrV8VHQDIB\\\\\\\/ay1WFoy4dHw7QGFUFc06P+YT77ejrgPzHh9Ili0hCdaYjtlfoIMJ104DiLUPz+wwJ\\\\\\\/uDbi0c6HS10jbFO7qI0\\\\\\\/Jr7mjlVb2Qx3ERAKf0I7kwDJYmynBloEcdhm4mXRHnhdz+GxcJU5ojJZAlsPf\\\\\\\/duybC5DxK1mq5cqhUeIaFicv84DWEs9ORELynHJ1eFfI9\\\\\\\/whUSgGT0j8k+PzjokSiaZY33IIfV6\\\\\\\/GuX2BCx3wjDWPB7Q\\\\\\\/IAlrK1fQwR+lejvK33v93k7FV\\\\\\\/G7O8y73I+yxSWMIWbjQ6s5aoK+YNCOIjzBuXP9uY+YPiW8GGUNi3mi4Kdk3\\\\\\\/9no4E+OuuidsG5AKiFTl20RcXJpb7btx62Qev0COmtqdx6SgTuDlkszPYn63FsM++4II43FMy0\\\\\\\/oc9NOh\\\\\\\/V\\\\\\\/371ppr4yxpY2q+ubasNVo6Kx48Zq+\\\\\\\/Yqj+ijnFz7yI5xHUUfv\\\\\\\/oMERtDdDZpDjT5k2VRlwfKls7N3qy7eQCd3pSI6fFmf5ZxSlXO3hFbA3dhDM89ruaMryaf4MjtdIonloZh0oRS4EMn70KOpQjHnk9mXgpUzN9ceGuS0n\\\\\\\/tjnzGOTxJpSBeJhMDo4dD\\\\\\\/eXdfqPaQ9SMM1eJ+7FkvJknJ9aSGpRlno\\\\\\\/kXa01Vb\\\\\\\/Zh2h9AtEgupiIsd4nPJirlJyQcbULbG\\\\\\\/miOfdDDTyUDzO7PPyF1w+loq+kg9sxyFKHMqbsxkztnTdOarWIjUia14srlUcYRZPG\\\\\\\/ptmn414DCQmA10sN10S6iBE0YozBkqpC\\\\\\\/WuVwAYPEZE+E6jK2DWqg0xaQPm3fww4SVRm6opUlu9Ugc+ggGX8n4jsAQHC3hizCrgRTrkciuMt\\\\\\\/6bJs1Gos0VuuYgd9rk3qn\\\\\\\/42Iqg2kb14VxNrDakqz9poniTPIzX4DAAxStyHs6F0ueuMrVEQllVhlI4w8oQaxq6bCfhOb\\\\\\\/FQ0nvozDs\\\\\\\/X2g\\\\\\\/W7qU6wUwGQU3P1FHBUulISPQBhYCS0NFNEB+0EoUnD3YlqpdMXmrswhvX9vuCuTnwcyYhmcqrPcQ7sPzKRcsk\\\\\\\/GlbPJSl\\\\\\\/tgpt6X7138bu4U\\\\\\\/VVE\\\\\\\/VvqwpKJkfZBJ0vJGEJqBlL\\\\\\\/ZlGxMCRxqadg7AOLhylay89UUWscfjpkNitCcQ60ghkuJO8S14EM\\\\\\\/vj9AMZs50prS4CfjkA4qP3X2Rz4TJdMthb\\\\\\\/YoO1EdRiqcsR5UCuOmmjspvFIN3x+CIZE6j2+AH3dWtnmW0U06F0cw6d960OAIOfUFYAtABkXDKAO8R20k7uHuRUuHqAK7CRQr+PPGwibhRwHOWNSeHczTNbeVSiRxEs34gnIHCgAAjgpQDwJ2H2pY+pI57vEQOC3SshiT0p1EtMgKxLAFRL3BRTLNk+SVjLAaZhXKKCa2mdwd5CyIejEZ3EA1aUHZt7zpSinfAF4XNRecAbc9bBj5K1aHYgipboKrVOGFqrUHsxpnipDc0CZrUfcSVougytzjhJ0ecfZC1gx5hXk+fj5L1H8QyApqiNfb21HBDOwJ37HxFVgIVgGLJE9HdWEG+h4VsNuUjsx8\\\\\\\/pD2HTGuKkKJuDDY+qs4ZGAkWTGjv2pzlqrbHYuvSCh7AdP0BH9RfyQMKz9CdNLoEs\\\\\\\/0pipKM5pz4ibhwuJdE8DyriUulgTazRAelOMbkZbwkf69TNg+8wOg75IifurA9xrkzEnbWtVPRK5SM0IfkbeZrSO6MRPS7fk4EQwnO9sz2l5+KeTznAtFLmYwGyHVy545ED5303ro7ezlfFTaN4xJHwZnEFLWj+X0KWAmHJJKytYByLUvO7W46zs4l77hFBCGFndxLHBtuxCNzLhB4xjYvJRvSFP41\\\\\\\/3fVyLqyLT5v9l\\\\\\\/BFGN4rfcGXy+4Fp2NYGM+lBXeL+G75Ih5cfk1tmZdqlYjYxVEMnKi1G0Nh819VQ+dkYP55vUN++ANhkkaotTa7jKJzTPQFGRcLRPPZlpVzbx8wxi5UJ6u5s2iM+wEsC0FhOkdq6Ta4qT\\\\\\\/mJgGGu8FWtP2zvaphCT13b1XHfl0qUBTC7aGtxVpVrASHTUTUaPHwemHDSS1xoMMqyeolc2XhGcGdQtyFFKmrGCWRrIc9V80BFSmLvmUANZluth0pC7FRwz10snFBzeVm\\\\\\\/P29PpBOQLTK3isNo8MF51cNgTSuIJPkPtv\\\\\\\/aQcB8AHU\\\\\\\/taxx5Z4N1S4oOcBmTmzVda8ZwurKqBLLTiliPtCUva9E2jxre+BAiWgjBmhCWhU3zDMr18I8+t+TrE7egAhthnP5ZYRsrY9RHVCWaPGwzWwCeyxC5uV0pdRVCqQxK9qELPFaBDMErxwg9GCiJT3UIqiXj1\\\\\\\/+7ohS1NpHeTFETRPsbfP8YXf4j2UCgf\\\\\\\/6P\\\\\\\/2hzyNbMcllfVTko1DzvdhsOyj5vW67iJJ7ZayadVip3i7R1I2hd+3gKQ9Hejn4k67UxODOp48i5OmStLw6qF8JrqrSjd9m6o+vcBuloIDqB2H0T5xjvFzRkVcRfrLSgh7CjuO4h35EufOFGE0crzAiSRbRyC6184VAhiLIj6tg2IpA8SwSQgjuuHomfQl29zH8kAn5bgFXrvW\\\\\\\/pAPsMpQ647yjplwLFYjSHtT3kFmGccr36cGHMJldcYAKnwn1VdcN2qM8S4fYqA5KVGwWdhMgHEXgPmHv1qe2RVEitxL9IZ1LWO+Sa+gsG15ISeQcLRp\\\\\\\/P5JUYjE9fgNV+rIiFSQAx0VcuCuQOwjwBFJw5iDSuyHKEhIwh+KEVQuyu3xMjsJ\\\\\\\/JoQnhw0v\\\\\\\/FcpGprCSNblfAit556u\\\\\\\/3jcHpBeCCWhTT2JnHjAgm4TmFaxcVwoLuXUsY1aFDeiZqe9Gqzc7LN86n3yoan4uPo7ltZ4T696PbxpC4vOSNRSbJqQrKhR6K3bpUWnY6q4yCgVlHVMxr5nY\\\\\\\/0KVRC7DmDG3aiPar67h+1VaqMZTQY3kxcH8AQe90lpJr2ndUwu0esIrClRp\\\\\\\/g\\\\\\\/2ioYNj9L+a5+roZNktGeXiHZAme\\\\\\\/H504Paa\\\\\\\/QuU7YwaAwnDD\\\\\\\/vDW+hhNFWkdJ6p2ksDsd2BucFY2xbs+hGWb9O5mB\\\\\\\/7juBCWAumAhuS9TI2e+YteKx+Iob9\\\\\\\/r4CFEr+6+s30l6CQCAIAT\\\\\\\/7CYaoOPHRBRb8BqKyhupotbhK4Fp02DwPQaLeWdq\\\\\\\/nyESrSk81pvWFnYauo2UI\\\\\\\/PxZiw0EI2\\\\\\\/IsOXgQrCxbqmIOe+xQ0bsMrLa3zTZgwjOhTnGMs2zEXmC7l0aCg8G\\\\\\\/fAXoLuhkkmfinsHgYYjqpSdEPWmbd2RrtHfBVQ7Pt8dba+OtZ34X4fujmZmF6vcjKKHOUCDu5DCc3WJFddECbQAngHgxRz2W8t0OtyBBADh5GTnDJuDOBBVL9\\\\\\\/NKcg0aorxxjOxPKtVIknIgG88BoJ2vd759S9EMilvMLtLtMjKdsfmadWx7RXZmVLoTm3n55n5cK48xjoAC4ZUOAnujie0MkpZGzxzFwd0F+Vc8mFTqA9M2fwpmrKfY4OTl6L6C2UAhJAZRf2eplVA5DkPHGGUDMo2KXl+uN85Q6wLpAejMO4ukb3abT1dRAorBCmLcxkYivYr1pFlo5bghHEZjfxefAlbYDrmrAVdoMkM8ZUVi1JfHQaxLLSHX0z1rKO8bqRrKImfz03G0FC4AHnPRugQPxUD0Q+SnQctIbF8qeVMg6iz5dMWbjp3VFLTlfrLijZ\\\\\\\/ftqFbMwSG3EjAUgaWUr9nlzQMWqlc2whRaSlnCXFVfRx+2IoedP0+MIYzPDyyB+GaJnZC7Q6e1BV1zAZGyrnRE8GDVWMYUIji2dzJeACUt6vumgvcQR8Oe5USK6GreNfM0EZuzw781MJlEhJyGU8ZQKSZe7nX\\\\\\\/Hd+FixrB719eO+8kjcgD0SMJeouFtYxYhitSBfDQ36BE0SUzAB6rpu+4+FlfVkWyyxJqbva34A2v2HQyJZ7Hg9Vf72oJ028q9PmNqmiik+c8zJS4WF9RzQv2DApdXfXb5F3G\\\\\\\/eL0vj\\\\\\\/TA6zcMHfSsx5lpW9NupZeKIE94cZgPoz66PBOBsX8PG82kaK30UbGqL0iAy+wXdJ+SbI3qc7w9JQZ0lurAmWYzo\\\\\\\/xpGc7hojFAPga29\\\\\\\/s1ybqbV9t4AlJ9NfNj8E8Apwx1R01eLKsp6X8FpRRRQGOPShYitHS6XFVeUNOAVnG17HDCevXrKmOT2O4\\\\\\\/HtK9umw4PNbrW\\\\\\\/fPTJXTcOxbM1nEgK6r8NSnxtewPawNToi9533PNFDTEO4sPCmEXEyC7eNVU\\\\\\\/5LXyLR9dqoFoIEUKJ2gjTVGzVcH0NirInhnill4ARZRa1ECfqh90qpKGiM4LC1bK3avnK5HQVY1JICi+MzPXZo4woOIr\\\\\\\/9YckxnNkgFS7D+kkS3xaIVzopm3GnbThq5CCz++rbK3YcFf\\\\\\\/llKwrDGTxHZdF6F\\\\\\\/SKDZEJhmO1xTIh8KpI3hfoiKi1OGhPGCpJGFSZCuXgPH0ZaGlCBAccBuwGx\\\\\\\/bafp75\\\\\\\/HH54x9GqgbB5ZpKBoLTrLu65m+UIPGOra5J15ho+K3qjDGEKt11QTj7SB0hRoxmUjUrHt1SHx0gDLr9l\\\\\\\/sSa9QCNLZudbOvvzrKL+YDq3xYyRKRC35JU10BwLP0i1hzdZD6MzQsVnVtxmcVbnnqun\\\\\\\/ob51BNCeOIDF7kDKviMbr7rMl0A9E0UdBwQJ2zdT9hyA3mscLtx+ImzVaZuI9bQdqSeygcu5\\\\\\\/eL\\\\\\\/38KoevHa63PrM8wgUsGcP+9Ipx+wn19A6KYcnlx60TaS01yGfjBCx1MxO3pcglw5ayKZmD1SHZ\\\\\\\/Wu\\\\\\\/d8\\\\\\\/k3ekN\\\\\\\/gxXJVEiHKO8uhLQuAlnUo4pSQGqLm4AK6gWVIPKO8BIWjaUuNxt5UXvbOvmVJCEHNnEZquJpbYhbq4aEdWTFBkOm+axK2ZpaGnoCFikA1HwdCQ5xTY59YrAvQGaq0MiYPrPoTH9FBLyZ\\\\\\\/LemB8sFhUnXJs8aEaTsl9WlpokMu+BaS4hkrFYtnZChdCXXRx8YDZbOVqB6zodX70Sq+Lvp0fH+C853b5qLd4l+FDV2fOSXemJdScCkWinAap4gDFacnzEpVm9uwSiofhJkvyFJ32Izt9dXBMa2dEYjqfzt26OgD0dWx89itXNV++MyQHxcFyv5KAjvR5lJWsOrXrdDUkYc6ZPgghc6sG\\\\\\\/Y+6ZaBL+mrqRcPVMZACjvs1iulK7SY7xl+QbevossalVGMHEPSxFNKy0cwUDKrwHIb5qg+GNZv64JBCHCof\\\\\\\/Jr4v45kSk1BWyO+KRNCd2db+90IJh\\\\\\\/j5OaaEEoO0OwRU6iOAQ+0VIcr4P\\\\\\\/OdNoqgiBVtR\\\\\\\/GnoEQAx8kyDpapakCWiyMLA5tgg2O\\\\\\\/TFGofNwUdsCg2nZKodjM4KRQooFlVriNcpwwET19Y3LlksS+byF+ZDYK9AAawVfIdb1CeScysvYTcxvkl52O\\\\\\\/yq6z0pUDMxwmtqaQcG5IGAD\\\\\\\/NpDNHAsN0YePVKRMLenu\\\\\\\/b+v6Iesqo5yQIvcn3KU6jDjw6k57iZ3EUQrAZKG8cKXU2363UwaARyp7fnAwSYlt2iiwXM44AcwNfwpV7k1q+tDZqOkyURjqfRT5Mf4bexQx\\\\\\\/YAi9DRrclnF9gBaNHuxqF4rGkKGCBRe5Vy7LpV9aZnpWgBUG764Xu1gZqYqpcYU40z0W4QGtx9z2gEYElzaMqRvNXr4oX2FqoP6\\\\\\\/e70ibpDmM1JQjFsLKj+DA+GpXRUyhLfTwnRKXIAnpifxxJWKOPOXE8Tri+WGG7eeEWWkJOhuMljWdYJcWXQ0IQA5ugkK5fkoZQYk+vLhv9NKjB+rerV42RBi5uQr+w8dWL3RbW5OeZMbgi5\\\\\\\/RURQCQtLrIvDrHe5IH8uHuHTa\\\\\\\/rWdUUHRU4ZIYEmQUO\\\\\\\/zDcSQvvv67jgsH1cw2XlZaKNTxbIsW\\\\\\\/JvrClJ43iOfAu05VM0\\\\\\\/pSyd\\\\\\\/q+i8zLt66KlEtWphy9GjqooDZoNyHnoCiDHEJVyknZ464ubkNE0tABc2hqo5lTm4AYyI6onbOKxtPFrsauoV80DRJMMh24DOycPeqnhR+YjIb8zAlTV2kMI+VBrkErI8WH0XuqdYF9SrjIwwksxF3A7mzsf+F3CZseKZNEa3XltgsHi3WUtEWQZkfDWz7DcT+8bDVXesIuQd+388\\\\\\\/seCYmdshP513nCt6na\\\\\\\/FHYMF6VtBYjxPg5rzuN2unz49ft7+qBwneLEg+z67YGM8mZSKq+srMo0weBymLTdr232l2A5B9U1J0+yOsrSpW40Ww795v3v2xvRB4CC4\\\\\\\/zWOpLAMy9xEoLN+B5UDbcqSFgQGSpM0+VrVga+Bdsui94bq2QADyIwTtF5HonSxdezp3uhFTohrQcNi1CN6\\\\\\\/x+pSsb4eNvWoiumb7vpcRFJkkOqtJ9hLlyy\\\\\\\/6X9e4BKcubWe6arQD\\\\\\\/MQwYr2V1vNc5MuMYCSn5tjkuqvHbeetH0tyWtYqHhKZKu7EprlRHQ2z3Eta\\\\\\\/ToOXIIFv9vjEmVEVrRnz8cqDpXkyxnrZ4y8dVz+gyvoeMOyAZPeoOybn1njIzbuhXaZZ8mnEzstVSt95Lp67U5aHJAAETeybK\\\\\\\/AFpAyADD5y9KnebW9GJzwUa9ahTHUiPxGXgn3ye\\\\\\\/9Oa\\\\\\\/BPGHZ8hThql585rli3L\\\\\\\/1UosrppWZhHtbFkfYOI6XS0mFRNdpvz3uswVbthroIukSByWyZnenFMnC3ZTmGU1JMBvOXz8X3KLwScdFC7OxdKjlgW7mOS5xC9MhNY9i9ewJpl3dDXBl8OQc9YviQa4+hC\\\\\\\/MOBne0DDgSUyJkb3IrCMTkOKYZCUclKGui8eDqROmv3Uh2AwH2jqYnpoeohYzf0D1XufQ3OvLo4EHGps71YHoYVTIHhZQyt4ktyPzZBU33L\\\\\\\/m00WeE0p9TDs0RAwwX17yZn7C9hnPnCAWimpqIgtVFGFek53+oQiPkC6Hy0DoPOUzSYb05NYlBD6Nht0t7FcYvhuG93026s0X63U2hk+Zo4k\\\\\\\/NjRfigXKof5a1\\\\\\\/nFwUR4ptnzDQHpG6MTT\\\\\\\/mJWAT0EtXIZ5sAs0tpLBcgoiOSOkQhYEvsDf+tEt8Mrod4ilsjrT8qmyOneYE1tvKOyfjRmQTnAaW+ABfpmwFK0Y8LArhQUTBLX4bkP1TbbQjo\\\\\\\/z3WrqMq4BsyFgwtmZsv2HeBYFocG79neAfKux6cdsjZcM5SWFHdlh8zRSGNeO8XPXXSqMSURVE10HBOQlE51HT8vnBNTetgNCZjp0lDqjd2qQtmkvNyWpU4fFukww+tC7wm\\\\\\\/ILIw8Ef66J++oMd7T4MdmT+b\\\\\\\/cjOHBQhy4Im5XcW3czScMduzEeJUoMDmC8SKrVtMKNtDdmdu9ifxTwwtjyQTgrIVo9h8ZejqDsNrsSyHVntQ12YrbgwaJqvnCGnrt0oiv\\\\\\\/z9Cy9M7Om9GaLIl5iwOunk1dRpESFD9roYiNeAGFQUXxylpcJ3vHsMKlBr926ZV3oYd488sS\\\\\\\/WLPlXcmgnzd9lgVqF9FH8bBOyx9X87Aj5ZA7Oo9RNJo6UAfwq3B9jS3Xn92KOQh4gLGzgdIbyXJ1hA5t6zf3Vj2aUZB6osXdzCi8tnBcGbkvxyyOfbtgnpMa1F+uUnrda+ARXkuGEk8ulIcgDkfVF1Qf+7fSNj\\\\\\\/Capyu47fRYzeFu88t13O2LkJIJT1UIH8FC5Q4IOMGf5STjm\\\\\\\/iDNHq6BBG+ekJX6fo5dThJchvub+g2YvlCeKyyjoQIX9Agb4XI75QDjVqMc96Tcs4dKsGssZMfBdiXZMxYrEH2akJ3yf\\\\\\\/b8V8PbTphJ9jZA0VzcaS\\\\\\\/gPYP32pqpNuPQ35iRsZQ32sM75kxLQAKTmYS95u287I9jPYw4RiMEf5XPt4m4BSi4\\\\\\\/oIQs\\\\\\\/cOIABVdlfcOPXSlWcbBtfL\\\\\\\/cuiQ6I7ADO4RcQivB4UTe86EXyMd5fLsvIp030OQr3uF7sQH0Hgi3CYM+UTnSurEaKu7EAmmn63nn+h\\\\\\\/grSxU76kuv5LyIS6L8rtmir3z6znsjndVHdtcLp+uKhHC088bB2k6tpvLA\\\\\\\/0puJLHH+lAQkHgp9NSON7zuu8PPJcp7Q7TlbuoWLaqKiLtnvYb8WDomLCYL1QF5zx3S216JCVZfBXdHMs7PlvUTYzVTzkvqdIfGe1wWGXwE+nvuhBJAJ3OWb4O9GtiBvnQIumSAc5paaK4Zhh8pmTKlUni5mHI2mkUa9GjSJHNpQTb0rvAMzKFhmqL1Itly70q8GPmqjs5g8MSTb4MkhGNxRqnVV+Ly+TxVuJfDYzxgzkLTeQUYsVRLNWF9BUrP3oO3109jSUC0HNZItTgGj9P8fxqYPHPiZxz6e654seDHmcINZ9dJhuNXqw\\\\\\\/ECvuC9ECl+gznr8BuVWCdzQnzmL\\\\\\\/R4KXogfnWMFycdk7YxcKOjeW+4RWzO+PCfBMH84EuCmIlmLpRvbMJFycoBjpOuGin3xakef532EVXHwSNaUSgCqiMZBO467LrPM9kSf6gI6MHiepVEA3i\\\\\\\/A+P5u1m8klDdZDCoMio+zB6MMosvHYtVUH\\\\\\\/VgfiwPLXLoN+pprubwDw4wEpkJHNExeo21ZGeS3G0eafISsyUTeUYCPQv6P\\\\\\\/w0vExr6hGaKxv10hhwZkr8V8NObFWfivu1lJ2iLYn3rz1Slf1dC2X83iBqz9iTRj\\\\\\\/Y8Tr9NGxQQOKt+tzWyYrC95vUEc\\\\\\\/5QIC5bNTKhi7pAtGTraxl1e3ubZiC1DgRPAMYMx3Vmwy1g\\\\\\\/WgoIZxuPepOpTOMvEfCUrXE91cSzrhKqsrw5xCPw\\\\\\\/W3J30wmB3aRf4iTGe4aK8BJ0ochrK\\\\\\\/yzdQKg6k8pIGtSCCyijh5gf8MuyBdttXg84AIfkvbqyVZTr0\\\\\\\/MGr3L8vMktU9dkeAmo2eV0sgNsywwXHgUzzEJ39uJlmfppDwXaf0wgqdO3R\\\\\\\/EnotyD7VmcPac4ny4CbI8qRES\\\\\\\/NVb9TZ+nBfDM8Sm+pRmFpRuzYKffWqud4Jv9FokePV5f2h4PhJV5vtM\\\\\\\/eMEgXpXVp34xjwTEvMy+qWB+WeT405IitxLKORAgE0oX60z1YHNh6R5anrpEJQvZxiiOzKW2h+7rinWmqfS8vHqgO1jKEfCF8pcPlvN01EOAEnq4kKmbyki5YU3Nyl\\\\\\\/O+Nm6kqgVjrIWxlSHoac7ujiI9gYUPRP7fiiuHWKA20+ozyJGGdWD\\\\\\\/ihr1CdVQEYGhxP9uP6Yl5Ag1NkBQXDz6JvY8+N4bZbfhWT6bz7YIFtIgUVe+I2m09lyYrOiS3ACcB0clp2CKU32gVFREWF6HCb7wbRiGJQBiX8rXWD8uy9I4FeKq+kWwqTAq6gzJgD6NkbbilSux0oUFXQlnX7nDhcPpKYqUExWN1Ui5lfWiA\\\\\\\/qu8bmmLpQzfjiFdyZY3wjGSsj2cjs\\\\\\\/oslBxMhKsxo4+YF4T8oFOca0QSboIJWqi9DyNND4r0OGkkfMqfNxuYh93m7jAU0z63PfUD\\\\\\\/SwdDd1P2fDHBbT30JCFmPLEdLd2St8XDZH5sd2u8AS3VPtE3\\\\\\\/Em65XFM2IwG3uJoH5b2YPcytSBuO2goWJCkfZl1FFCZGKJPW8cX8bZC+hofbBYYRertoPw7TuSLycPwH2gpf05UfXd0oL\\\\\\\/khZeLYBQLaiGBmozsXjO4+SxWnhPkNlt86+UYW1pJ9\\\\\\\/y7rVoAUUdfm6NykUhmdhmzM4pc7jS87v2yPVWVSBUMPdBOhBJ9yYD+4TlHb4l8+oSCfjRU6m+TF3oNcmRZpsjeH2UEpvqfsPaBMABNBcJgTGGDOYN0GMsRya5WoscmAIJK30mn7efPXM0s16+5ujOfQvcAS6CX\\\\\\\/8pzB+FEA5yG1ILw6Ae1AfrbO\\\\\\\/FkBZfQYEIbxd65GcDny9kBvobhHp32\\\\\\\/RHL\\\\\\\/Hp8HkaizS2WJw66SgUpbQAtv2wwDXU2CJia14rz5XS\\\\\\\/cIcU5eXBYKnN+Tku3sOMfYVr+pWkCuRxku4ZcrZSopLakqS4b8NBH5+UqKBkpQ6cA\\\\\\\/dvPl6maQIg0466gNuOLomKJ84AUofFNZCXTrH0ew7hnHXHg80IjLDVB1Xl6XK8OTnEDP+jS4nuusmEI47mKM8ftW2eNiQ7J50cjjndegQhr0OpC9HgldwwhnhK9M0gun7vKTqcmG7obf0N+04T0kiqRkzPez1FcpsEsXi6b4ngNONIyYcYcpb1VMKul6rOupNpZJz6HjspXxXEc7SSdK8ZFcxbCNv46OsJnXR8Cr4InBFVkj2X8MePPuFYFEVovV96VP4xyYqtXdxQe6QfrNtfynk9yQQcyuyojHM919tdOE78sTdnMwVE+s3aKrhfDtWjeVwY8J6Tf2U9RN1zsL\\\\\\\/YEgdXlnw338Py55yyGgsfW\\\\\\\/DkV0iOVrJEW0d+DnLd3DFxT7eMw4sKySAVvIHyR+PX\\\\\\\/yGoj9lJ6rzhtNzT7SQaUhXikuzu3O\\\\\\\/boLB\\\\\\\/BHEB\\\\\\\/2NrBkaJVSU4qPZ+W25hMDpHXG6jF1DjKSugC6DLBDb5J+rfvrTzVyJf5z6vMZPgxocUnXmApCdAVSGlas8V6EGHc8VA5BlUrooxaytiqJmZz96MyRUJiPd0fZ0129kGJsEoMp3PjB4vLuGp+qv4tjWNfHFQqDZAre3yPpMBQnBjD+OVb1GPlviR0BuaAhxkV9kQ8bhWTBDVP6MViH+cO8LRFb8A1yYr0NLCBPZv4j0d0cq+YkgfMOEkcHNNv3\\\\\\\/0tbJcuDoGoRMs\\\\\\\/OjcHQIipaRE7hymVRMV0cODS8+\\\\\\\/GhlJBx9wCzpmOuE4s81xSqxwnsULSdrgCCYdF8\\\\\\\/IG4bFEK7q976p\\\\\\\/KEXZ1z9KofpF+yKL6v6UdR8r\\\\\\\/7H+1X\\\\\\\/wZWc+kT14KSJVb9vlLlU3RUkoTT42Nh2s38Rtz115EcTov2IIdN1sJbElW8rs0dT6qeRxi9R9PDaXiM+gUcy+lw1vUn0AAM+SRai5zGSCm0VCCJWVyoCksanScyQEI9qH+JtdDs8nwU2ZcgMYF9kLiupRKpcWhjSalzVMLCRdCM2Ssbwhaenu7V6GZxb4bnu9NPfWAqC9uN7NqSy05wovZj\\\\\\\/tgXWiywo9aDNNMZe6+vIomO2aGLvk\\\\\\\/Ccd4q12nyOMnNalobei+tYKCNMl4+eIBaLI9Em3qhUQtK+yWak9py0dXENAQj1C6n50cZhTsBwBf7ajgVuxFPCqgaO2VYgf660QbcRAltPYhKTpNVAC4EldPpMT\\\\\\\/7w8wQjWEB1QgcgrzWcmaeAhVBdOx3wf7\\\\\\\/PSpj4fdC6Guk6+gcr69AV9sdiQhcxpjPaMAcVhwiqaDK8G1D92gQWQAhMTAtXYx6hDsdmDSRZI5rawzZr1Wp2Qx0L5vkVx+2V4zbFV03hFXnjj9GR9X4fIuDAV93KIiSQL0S9MAuz8JJHy4uqogHK6fiRIetHzrEFMfh2yHL4ts9xCcqlwd1xFVJ0vB4DG\\\\\\\/v27+1RPi0pbgoA6limMhDKNVAPWHCy\\\\\\\/lf4tuJySghMqDmFNRfqwU5Mdl3M1\\\\\\\/CnKfu9a2mznB3wPMIvc8baNNJAmN6hfCOat0IxghV7IdRryTrm9gEmYUFQVAwJrm4bymOkrGXVSCtuPvcAWVeZxPDz2p\\\\\\\/UpkzEKnwlZgih4J5yesrHxx9OM7A8HStStE1IHXXoN2qFk5IMGljlQrqyrnrFE4NWzA8OT7zODC9rqPdOMlqNPsAqIP5gRXOJ5vIGfOR7vIPek1DUUf9Hr7ii27CULXGbcTEl2wwzKB1LdMheu+TfFm8q2YPoly1LlCpKWgaHCsYMFclJDVv3KWupNIvteeZCXLqiLAEs1QQALYpq26DvHxT8GNuIjbGO57IckGPzqCP7Attb3f71xdDJv+FQpbPEQD8+kxnaKYtb7q7simCav3FhZOPqlVG02wLXVg2peftGVNVu9IUPc0Vgp3bVQ17NrF3oMzZO0xQisZNWClZAsiUJX6vF0hHY1hqYok4\\\\\\\/mt3Yu2V5fl4ZFTsATcVyExThTWh5vgUoY05LzQZ0RrTo54ZsbTRBHTnG4Rafk3eO8st1hKsuyOp7fwPOz0Ay7EMxI3U1VsbTi7TBFjEsqUeWlvvcRS3xx7n9C4YaeTRraJ4y2BUE7ixRG5flluyTC2cQGQPwe2Ty+DPi0SjaB5jmQORxkNlFVEpaRHne\\\\\\\/KtUxcTkMY0n1xMT901fSBzHxt0CzEb7u\\\\\\\/l75YPQ+p3gfcl8n75Nnjz+Nct8J20axX0wszsBkut\\\\\\\/NhB8302n128AqTaZqlNZQb6TJCP4YnVRpokGFrxQftMv+XyI2THILZPLM8WXhwOvY0gw5jEY2eldSMW\\\\\\\/W+vGmfeeZaIXvLRM2IXg53IwAO3AUFkmURQ4Vf\\\\\\\/XsLLHb3\\\\\\\/u4m9tE3Aa9cWkRgW5Mgomc8ofROf8nasQpcNdMnE+xbS\\\\\\\/O+cima3Tw1qM\\\\\\\/ma6FLm3Du\\\\\\\/x\\\\\\\/Wdp1D2oxDgln0NuIT5X\\\\\\\/kPeUchk7u5973A8nTZXKyWCPdNJDCKHkRSqO0U4YwneEg82nYincH3ZBTaQPLuQwV28C9wKln6SNu7pzfMo6e2pNu7m78nBcjklW8ZPTAMjyYwYRiuKR9lqlmuDwFdamzTZvmV0oo5wATJVVSiAs18hymjdt9y7OMVcZIJY\\\\\\\/B2Fe6IZJb4YPUceHcRRwTE+3jkZjD6ksP\\\\\\\/goclJBxAtnVydzP7HSgYr6KGbTNNYsPt3yBziVarhEiOPGbIxvKhQMDSQXd1p+Bq055UHTT8PW+HdrLqu1RqEtEy5pA4Uem+gt+kfs\\\\\\\/ZJsVcEaaP0Uh8jMeEvxbSJjuJR9i6B2IfCVv0J9NsWjt\\\\\\\/Ichh0YlXdhXRrPa9F6QrZMXhJPBBYyo6j0\\\\\\\/bIPrPbED+IWVlgNnRsTLlGNxoaRUcCr2RVhuCl+n\\\\\\\/eFVkC74iTrj\\\\\\\/5Lv4ZhsqjKvzvDVK450GDqXKtK9YnpZaoc\\\\\\\/k5cjZ0NuJgTqTPLq65Ni4f\\\\\\\/qv56qy6f4OsW5vv24nvhyxcAzrrr64MQNwUgB7skpQqwzT6brUJb71ZvG0eN9R\\\\\\\/sS3YC\\\\\\\/jr7Ruev6hLR2Ig5cRolhDz2YGx58xK9DAP4rYnBBwSudCCPvJtv9vnGBnFZC5ptrvRNezLW9Yn4qKGg7E03UXRlnrdhqRfuS6vYd9OL6d57wUO0djV6HLTJA+l4f7CIyMhVqKmg7r1nRiGCfFMTGCZVJdlr4zPepIECwJgAY5K2e2CpBkAcBgomPZtthtLUV5++sHZzZQxkG\\\\\\\/p55vmtlv72YlUuze5FpkMOLG2SxPwSCbKTDHKXJ2P1hph\\\\\\\/q8STxlfqnby8T7+PFTzQJ4SfaHJ1QhGWC7gPY3s8h89nJy9YafQsVfBs9OeqLig4N3KiF\\\\\\\/NfxjqrXfeeU8ll23U8BTjOnH+SBepjUoD8nt04SGhJhYr7JHOx1KkvbOFQJuaG14QihGF7rtJULYIc40wCodCpvXzWX0GymQDO72uYDj4ojzM0OA6l+gaUKMR9K3Ztk82M+Fa3pPFmNDCZe2XfJ8wxtVO9vMcAoIW0pgnYnI9u6W6\\\\\\\/+UaS\\\\\\\/S9XkzAAb0m1EunSaHgEoCHXpbNmhfBU3mv9YR1OISr9kUZV43O0Lx1x+hinE602dtEKFYR7NhkSWWYgNS4TAkH6xYi9TpgZLnRPmbeovqxZqxjjspQSd2C4TVlTgPFpcTMlr753a9IaeVoUwx9b0CChW0stN6CoQXlFEOPHIbCJalK6pGwo1uFwclf3OmqEXywp+a4OeTO8CiuehlTxre55fQp3gfY3Dhgqo8xjtaVk3MIdxIZQIZa+ZxCtRB+7rd\\\\\\\/yX8xk0Kj0p6HE8D+g3d1VWptSyIOLP0J7xPH2HdOwaMpu9sf8BLQQ5KLd3ycDw7x9hEodwVWRTsA\\\\\\\/k+rPuSew5XPks5PrfY2pLxJ\\\\\\\/MsX2BnKNhnahGlB4d5u3p4x7vnApFmZpqNPuBNGSefoBXf5hvHkw4nnEsBGzgbjBfMdfqBGHBBcGmqohylXV1LqoOmDORJ42wiX\\\\\\\/YuislbaFtSM+6PB5BZw\\\\\\\/Gs1W9gvfePAHGqMdHee6Dp3q0i9xbPMvarIq\\\\\\\/xiJSNcmXSh444L4oj4luxBqcKPHZVXZAP99J\\\\\\\/9kFEK8re8MH3aAh+krpGcbhWas3vIM3GywOdRuiKzvg22Bw0+aSOg9AXKzWUh+T+cGLIiNs63iC9fHffT7oFJxtp2912Zjgj\\\\\\\/mSM1\\\\\\\/Wd74ng4n7cdejpSA5EJ9L0wHG3vqJP1faxkBUT6OALLyYKHFKTzuY0g5y4oGzRe0sswanwtBXqMYD4bWjNr9Me6A6VhknskXSKjq9plGbxD9LX3r0\\\\\\\/jp\\\\\\\/hO6fW4pjrr5xsnRycvnswyavURkPDtKKiqa8d1mEb9dzBcxVxKgFev6ceS65GFpbakLY4wEbZcmwTniv1qTSmOrmSyDoRLOzG8\\\\\\\/yjjb681Hgt+FroMEn4NY69KB1j8seQiSyTlDM9FL3XT8oynJ4oKbuc17oTxqXddSJ77Kxhzej42UYjeOnJN7p2PAeG3HsK\\\\\\\/w1jKMjYaHcy8Mb9NUDqclJYYcZxeu\\\\\\\/tMRrSxeatAFV9YlVJSgdQdduWTPYs4w8q6E83i\\\\\\\/QyVeR6xZK0VIsOsf2BZxF7okfLR8wvhsqHAu03kLlN5z6JKvTlb5Y4oaZWLvcRZ9KJMgaWhB4sGnbzchtWCh0xkDeLBO5T4bxqdwCWtRz5dRlfmwe5OD5ufBz03dA\\\\\\\/0R1fCKwlJpTnIzsbjwH2HtFoBWkcO\\\\\\\/2YRxUFn7hZ0w9+FCwSqxpO1pE84aCZQfcOHZqkJ22UANDDD\\\\\\\/e10YJpzQeXVCOAG41fTZKU4a4ALvo8+Q+vLKTexoE+ycxLY\\\\\\\/teJd7zIG61hzkNKUJoI345DEjt3PRmWe8k7ASGfh3dVPbRATRLznYA1I47DAAYrySnml0\\\\\\\/Bqt5N6PPhtcUTD3bIcN7tS7cNW+VLyJSM2qReajV7sODPETa7J\\\\\\\/fNch0AmZqLyZavTP3bW5FoVmMTqckrRdJJO9z2ECAZXVq2Mt7RapakOmmGs\\\\\\\/NqncxOZ\\\\\\\/7qsmiuaoaMFd+BqFIWA98INGbwWqnlvfyyq3MBI+eFPvPm9Y9VwNz+a9tKRzI8TFik\\\\\\\/iKo7eaBIOPsr2JW8loYwWw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022HIvQQrMFWqo4OGPQNWaxKA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00221c429a4b89168bf56e14861bc05659614d134d5aab8286fe17b6253a2fdb223b\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022T4xss0\\\\\\\/MOjxReC1HFcV405WtQFFgozvx888L9K3FjMoMpgMOKF3mO9luz+UdFLDDp\\\\\\\/JkBf5SRkUWMDf\\\\\\\/ee1mq4zTfcFUsBG43Xs0CvKQFJHK5Q6fxbD0EMR3fmfj4onbi6lV+wghoov5Ah4LAMmAhtG2VcIebXUgOfY+Lz23qJU5hJnyfq53kM2rwovQYjsLL5SW6NlBrl7RTwWy\\\\\\\/PK20q7f6ttHKwgZY3xGhFFrDxB23iSI8VRsRC42aR0kHGQBa\\\\\\\/S5OHoREHnOk+WM6VMdCzJ66adxEOnzV3XWsNhdoSiklkvao4\\\\\\\/PGTP5JjYlKskA+eaDpxHPGFXKwSPlxPUIau2+I5VgiRnYd45gzXyBwsnvzWD9iZvVxRAHV70B2HKh4rqVDMwWZIpqoKg2yq5T4ZTdW0JRDNbrPctXUSMS15zTDLl6hqUdTzGsVm7Z+7lq6uJsByjSYQeZ2OV4xR3O5qAi5enwFiu7agL4E2UxAm6qjIffGG4r06EulmYg0OysUNjhjEMvjz+2vScuqhYFQ3J24JORt1PJnwsVhPNwyCQ2D0VlBJfdZaM0rZ8nRQhK+u1IALG31n5ypcLS9LdrIoRRt7R8qmRLFHsUr7Nx5rpl42SUNXnr1o4tkMqlfa4gx9Y9yrseSh7zRcHel4ahQhVJx+eKp19ZRYHoOSHKu5InbKX\\\\\\\/uiDzeYWJ9Wh2y9ucqQIFaNqPVz3fSEwvJT05GY09VzfM8tTpMoUFHu6YWCwBZWXzUC2TfAbHi+eHbd2M1hJp+sjNZQP\\\\\\\/PWGvHXO5+8YUpn1GwkSCgaYpElL1cOgF3CM571ejr+L7eRBC50f9NrXDbpD9ENERN\\\\\\\/wbPmdSq08T7MSOdd8Sd6wyrshegpX\\\\\\\/kpg65ZEnwqw9i35djS17acOLkRhPbOLWajeR+JfcPrd4612yU1E5cndNq29+woQcpwyfGgQNhY7CQeiRC\\\\\\\/u7SO0JTHGHGOsH4fa5IfcYmXHzxfiXP8y0wSW2BdzT+mCOODgqWYUePPQN1ijgbvBZNBeTk36Uj9wE3jV+8p0ARzFcJgkEBlA1DfCElG\\\\\\\/zRwZEKtjoco6l3ocWXuVUkc9npozh4V1FZCDOw6lwIChf8y9eMgee4F3aQVk8g\\\\\\\/RxUoMmjvhjBAVL+YsqUIKdyy4U2tsMWURczsDP9KFVSByFiv+JGYCggz47\\\\\\\/81jqpZSHtytWzoRcYlmqPPfoQdH6xNNPp580HKigSNNW6HNgfZulQeswi1Cv3tivWmdDiBc3BqpQx90rqHwDw52Wy1lV9BmTiV4FWE4lTmgrtoDt\\\\\\\/dqinDYqY0tHSoSKH6DS6iOiJaz52L3EGnJ78hupOR3ZHpVTEJOBGQyfvu9HlCAAy0F6Tt4wqIZ4Ls9qqZkLrXPmOsIXLSxFx2uNi88MIbbblHStgwwkFV9u2DrBwIv70XPwVuWpRFg1DhUxsPuJ61bxSbdJ8JcIBFO9RWSWsVb3BdyGSxZFcKaczsZWfKGOfoTIhmsmBB71oIwS0ZC1336iOD3ya4rszRA3t2syVrOvFkoz50uURmXYXAxa8FSrGLrW5PUOwJYwj6WM7cPRuk\\\\\\\/xYCgwRFsgZpPPCqH5WYXWqa9zKuEU+oFqd3yMlvxfvhoTciRCHRpf9e1111XDSRg4radiVvHpJarSR9Ygk6UwE128++FCDYE4jZAn6eqVM6X50DG\\\\\\\/UJxRGrvFkxDt05QBHPH6v1nBQ7vlzDuOQ6AWSOsy0eDYeWhsam7MMPcJD0SrvT+T29LoLt+\\\\\\\/DJJWTqTgQq3mNzDXgAgmRx0XZjqaNeU15YTf8XoJ\\\\\\\/5U8qHKSZaL+Of5cJOJ8M42AlO00NDIMZJoYaoZqk90Ft5zT8oIhNp6ZOoAoU5Vjuyi0SgGR2uj830IO0fjoogAo+CsTtz6HVrXQpgXXIV2lJQP1VMKxkl7OVYXiKs9rt8WojGb9lgUBKtKDNm+zaKEnSv\\\\\\\/iOyri1qJE6D8wnccAETfvPgzJ25l8aSBvHHznQ1uj3Vihi+5TA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022wtLMckP6tYd1\\\\\\\/ElZR69y4w==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022b4354d8932a9565a71dd0904ebc9c27db2d8a0531b90e4faabb8e05fb5ef376b\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-494285 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-genomic-prediction">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/10/3381" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-K5H-jhg0z0rP2BXw_hthKOFzZb9DeYyzZJEzd91A_Xg" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/10/3381" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-l5uW-t3vYt-BQxMDurQzVsXJiS4bAdUhzCB1C0IGzBs" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/10/3369" title="Predicting Longitudinal Traits Derived from High-Throughput Phenomics in Contrasting Environments Using Genomic Legendre Polynomials and B-Splines" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/10/3395" title="Evolutionary Dynamics in Structured Populations Under Strong Population Genetic Forces" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494285" id="node494285" data-pisa="ggg;9/10/3381" data-pisa-master="ggg;g3.119.400336" data-apath="/ggg/9/10/3381.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Bayesian Genomic Multi-output Regressor Stacking Model for Predicting Multi-trait Multi-environment Plant Breeding Data</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Osval A. Montesinos-López</span>, <span class="highwire-citation-author" data-delta="1">Abelardo Montesinos-López</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0001-9429-5855" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>José Crossa</span>, <span class="highwire-citation-author" data-delta="3">Jaime Cuevas</span>, <span class="highwire-citation-author" data-delta="4">José C. Montesinos-López</span>, <span class="highwire-citation-author" data-delta="5">Zitlalli Salas Gutiérrez</span>, <span class="highwire-citation-author" data-delta="6">Morten Lillemo</span>, <span class="highwire-citation-author" data-delta="7">Juliana Philomin</span> and <span class="highwire-citation-author" data-delta="8">Ravi Singh</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3381-3393;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400336 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494285" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Osval A. Montesinos-López </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Facultad de Telemática, Universidad de Colima, Colima, Colima, 28040, México,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Osval%2BA.%2BMontesinos-L%C3%B3pez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Montesinos-L%C3%B3pez%20OA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AOsval%2BA.%2BMontesinos-L%25C3%25B3pez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Abelardo Montesinos-López </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Departamento de Matemáticas, Centro Universitario de Ciencias Exactas e Ingenierías (CUCEI), Universidad de Guadalajara, Guadalajara, Jalisco, 44430, México,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Abelardo%2BMontesinos-L%C3%B3pez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Montesinos-L%C3%B3pez%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAbelardo%2BMontesinos-L%25C3%25B3pez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:j.crossa@cgiar.org" class="" data-icon-position="" data-hide-link-title="0">j.crossa@cgiar.org</a>
<a href="mailto:aml_uach@hotmail.com" class="" data-icon-position="" data-hide-link-title="0">aml_uach@hotmail.com</a></span></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">José Crossa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>International Maize and Wheat Improvement Center (CIMMYT), Apdo. Postal 6-641, Ciudad de México, 06600, México,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jos%C3%A9%2BCrossa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Crossa%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJos%25C3%25A9%2BCrossa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-9429-5855" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for José Crossa</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:j.crossa@cgiar.org" class="" data-icon-position="" data-hide-link-title="0">j.crossa@cgiar.org</a>
<a href="mailto:aml_uach@hotmail.com" class="" data-icon-position="" data-hide-link-title="0">aml_uach@hotmail.com</a></span></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Jaime Cuevas </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Universidad de Quintana Roo, Chetumal, Quintana Roo, México,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jaime%2BCuevas%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cuevas%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJaime%2BCuevas%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">José C. Montesinos-López </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Departamento de Estadística, Centro de Investigación en Matemáticas, Guanajuato, Guanajuato, 36023, México, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jos%C3%A9%2BC.%2BMontesinos-L%C3%B3pez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Montesinos-L%C3%B3pez%20JC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJos%25C3%25A9%2BC.%2BMontesinos-L%25C3%25B3pez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Zitlalli Salas Gutiérrez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Departamento de Estadística, Centro de Investigación en Matemáticas, Guanajuato, Guanajuato, 36023, México, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zitlalli%2BSalas%2BGuti%C3%A9rrez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Guti%C3%A9rrez%20ZS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AZitlalli%2BSalas%2BGuti%25C3%25A9rrez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Morten Lillemo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>Department of Plant Sciences, Norwegian University of Life Sciences, IHA/CIGENE, P.O. Box 5003, NO-1432 Ås, Norway</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Morten%2BLillemo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lillemo%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMorten%2BLillemo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Juliana Philomin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>International Maize and Wheat Improvement Center (CIMMYT), Apdo. Postal 6-641, Ciudad de México, 06600, México,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Juliana%2BPhilomin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Philomin%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJuliana%2BPhilomin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Ravi Singh </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>International Maize and Wheat Improvement Center (CIMMYT), Apdo. Postal 6-641, Ciudad de México, 06600, México,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ravi%2BSingh%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Singh%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARavi%2BSingh%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/9/10/3381" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:494285" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:494285/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/10/3381.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:494285" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:494285/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/10/3381.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:494285" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:494285/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/9/10/3381.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:494285" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:494285/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">In this paper we propose a Bayesian multi-output regressor stacking (BMORS) model that is a generalization of the multi-trait regressor stacking method. The proposed BMORS model consists of two stages: in the first stage, a univariate genomic best linear unbiased prediction (GBLUP including genotype × environment interaction GE) model is implemented for each of the L traits under study; then the predictions of all traits are included as covariates in the second stage, by implementing a Ridge regression model. The main objectives of this research were to study alternative models to the existing multi-trait multi-environment (BMTME) model with respect to (1) genomic-enabled prediction accuracy, and (2) potential advantages in terms of computing resources and implementation. We compared the predictions of the BMORS model to those of the univariate GBLUP model using 7 maize and wheat datasets. We found that the proposed BMORS produced similar predictions to the univariate GBLUP model and to the BMTME model in terms of prediction accuracy; however, the best predictions were obtained under the BMTME model. In terms of computing resources, we found that the BMORS is at least 9 times faster than the BMTME method. Based on our empirical findings, the proposed BMORS model is an alternative for predicting multi-trait and multi-environment data, which are very common in genomic-enabled prediction in plant and animal breeding programs.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/bayesian-multi-output-regressor-stacking" class="hw-term hw-article-keyword hw-article-keyword-bayesian-multi-output-regressor-stacking" rel="nofollow">Bayesian multi-output regressor stacking</a></li><li class="kwd"><a href="/keyword/multi-trait" class="hw-term hw-article-keyword hw-article-keyword-multi-trait" rel="nofollow">multi-trait</a></li><li class="kwd"><a href="/keyword/multi-environment" class="hw-term hw-article-keyword hw-article-keyword-multi-environment" rel="nofollow">multi-environment</a></li><li class="kwd"><a href="/keyword/gblup" class="hw-term hw-article-keyword hw-article-keyword-gblup" rel="nofollow">GBLUP</a></li><li class="kwd"><a href="/keyword/genomic-selection" class="hw-term hw-article-keyword hw-article-keyword-genomic-selection" rel="nofollow">genomic selection</a></li><li class="kwd"><a href="/keyword/breeding-programs" class="hw-term hw-article-keyword hw-article-keyword-breeding-programs" rel="nofollow">breeding programs</a></li><li class="kwd"><a href="/keyword/regressor-stacking" class="hw-term hw-article-keyword hw-article-keyword-regressor-stacking" rel="nofollow">regressor stacking</a></li><li class="kwd"><a href="/keyword/genomic-prediction" class="hw-term hw-article-keyword hw-article-keyword-genomic-prediction" rel="nofollow">Genomic Prediction</a></li><li class="kwd"><a href="/keyword/genpred" class="hw-term hw-article-keyword hw-article-keyword-genpred" rel="nofollow">GenPred</a></li><li class="kwd"><a href="/keyword/shared-data-resources" class="hw-term hw-article-keyword hw-article-keyword-shared-data-resources" rel="nofollow">Shared Data Resources</a></li></ul><p id="p-3">Genomic selection (GS), first described by <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Meuwissen <em>et al.</em> (2001)</a>, is a plant and animal breeding methodology that is revolutionizing the selection of superior genotypes because it increases the rate of annual genetic gain by accelerating the breeding cycle and reducing the time and cost of phenotyping. GS assumes that a representative number of markers across the whole genome captures most of the diversity of the genome to estimate breeding values without the need to know where specific genes are located. In addition to markers, GS implementation relies on genomic-enabled prediction (GP) models that combine genomic (or pedigree) information and phenotypic data. Then the prediction equation of the implemented GP model is applied to a set of plants (or animals) for which genotypes (but not phenotypes) are available, to obtain the predicted breeding values (or phenotypes); finally, the best lines (plants or animals) are selected for breeding. For these reasons, it is obvious that GP models are a key element for successfully implementing GS. Considerable research has been done in recent years for improving the prediction accuracy of GP models; however, better prediction models to be able to implement the GS methodology more accurately are still lacking. Most of the research that has been done has been aimed at developing univariate-trait (UT) models and very little at developing multi-trait (MT) models. UT models are trained to predict the value of a single continuous (or categorical) phenotype in a testing dataset, while MT models are trained to predict at least two traits simultaneously.</p><p id="p-4">In general, MT models are preferred over UT models because MT models: 1) represent complex relationships between traits more efficiently, 2) exploit not only the correlation between lines, but also the correlation between traits, 3) improve the selection index because they allow more precise estimates of random effects of lines and genetic correlation between traits, 4) can improve indirect selection because they increase the precision of genetic correlation parameter estimates between traits, and 5) improve the power of hypothesis testing better than UT models. For these reasons, MT models produce more accurate parameter estimates and better predictions than UT models, as documented by <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Montesinos-López <em>et al.</em> (2016</a>, <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">2018a</a>-<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">b)</a>. For example, <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Schulthess <em>et al.</em> (2017)</a> found empirical evidence that MT models improve parameter estimates. <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Calus and Veerkamp (2011)</a>, <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Jia and Jannink (2012)</a>, <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Jiang <em>et al.</em> (2015)</a>, <a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Montesinos-López <em>et al.</em> (2016)</a>, <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">He <em>et al.</em> (2016)</a> and <a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Schulthess <em>et al.</em> (2017)</a> found that MT models outperform UT models in terms of prediction accuracy. However, these authors also found that when the correlation between traits is low using MT models, it is not advantageous (<a id="xref-ref-20-3" class="xref-bibr" href="#ref-20">Montesinos-López <em>et al.</em>, 2016</a>, <a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">2018a-b</a>), since the lower the degree of relatedness between traits, the lower the benefits of MT models (<a id="xref-ref-20-4" class="xref-bibr" href="#ref-20">Montesinos-López <em>et al.</em>, 2016</a>, <a id="xref-ref-21-3" class="xref-bibr" href="#ref-21">2018a</a>-<a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">b</a>). In a recent review of statistical models for genomic-enabled prediction including G×E and for different heritability values, the authors (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Crossa <em>et al.</em>, 2017</a>) did not report very high genomic-enabled prediction accuracies. However, it is recognized that from the computational resources perspective, fitting MT models is much more demanding than fitting UT models.</p><p id="p-5">Multi-trait models are also known as multivariate analyses in statistical literature, and due to their clear advantages over UT models, they have been applied for solving a great diversity of problems in areas like environmental science, education, chemistry, telecommunications, psychology, medicine, communications, engineering and food science, among others. However, the use of MT models is not as popular as the use of UT models due to the following reasons: 1) there is less software available for performing MT analyses, 2) fitting MT models is computationally intensive and much more demanding than fitting UT models, 3) MT models are complex, as traits and variables have different response patterns in different environments and therefore create very complex genotype×environment interactions (G×E), 4) MT results are based on more assumptions than UT results and may be difficult to assess and achieve, and 5) MT models increase the problems of convergence when they are fitted with classic methods like maximum likelihood or restricted maximum likelihood, among others.</p><p id="p-6">Recent literature on MT models has emphasized the use of multi-output regression (also known as multi-target, multi-variate, or multi-response regression) that aims to predict multiple continuous variables using a set of input variables. The output variables may also be binary (multi-label classification) or discrete (multi-dimensional classification) (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Borchani <em>et al.</em>, 2015</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2016</a>). Several real-world applications of multi-output regression have been studied in ecological models for assessing concentrations of species in natural populations, and in chemometric models for inferring concentrations of several substances from multivariate calibration using multivariate spectral data. Obviously, these applications give rise to many challenges such as missing data, the presence of noise that is typical due to the multivariate nature of the phenomenon, and the inevitable compound dependencies between multiple variables. Multi-output regression methods have the advantage of providing realistic methods for modeling multi-output datasets by considering the underlying relationships between the output variables, and thus give a better representation and interpretation of real-world problems (<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Borchani <em>et al.</em>, 2015</a>). A further advantage of multi-output regression approaches is that they may produce simpler models that are computationally very efficient compared to other multi-trait models.</p><p id="p-7"><a id="xref-ref-1-3" class="xref-bibr" href="#ref-1">Borchani <em>et al.</em> (2015</a>; <a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">2016)</a> present the multi-output regression approach as a problem transformation method where the multi-output problem is transformed into independent single-output problems, each of which is solved using a single-output regression algorithm and finally concatenating all the predictions. However, the problem with this method is that the relationships among output variables are ignored and thus predicted independently, and this can certainly affect the prediction accuracy. However, <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Spyromitros-Xioufis <em>et al.</em> (2012</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">2016)</a> recently extended the multi-output regression problem transformation method to target the dependency of the variables by introducing a novel multi-output regression approach called multi-output regressor stacking (MORS) (also called stacked single-target). The MORS method consists of two stages: in the first stage, a certain number of independent single-target models are fitted; however, those values are not used as final predictions but rather included in a second training stage where a second set of meta models is learned. In other words, the multi-output prediction problem is transformed into several single-target problems where any regression model could be used (<em>e.g.</em>, a separate ridge regression for each variable, a regression tree, support vector regression, etc.). Then, according to <a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Spyromitros-Xioufis <em>et al.</em> (2012</a>; <a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">2016)</a>, the second stage includes using as predictor variables the predictions of the target variables obtained from the first-stage model. The second-stage model is expected to correct the predictions of the first-stage model using the information from the first-stage model.</p><p id="p-8">Recently, <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Montesinos-López <em>et al.</em> (2019)</a> presented an R package for analyzing breeding data with multiple traits and multiple environments that is an improved version of the original BMTME of <a id="xref-ref-20-5" class="xref-bibr" href="#ref-20">Montesinos-López <em>et al.</em> (2016)</a>. This improved version of BMTME used the matrix variate normal distribution of <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Montesinos-López <em>et al.</em> (2018c)</a> and the appropriate priors of <a id="xref-ref-21-4" class="xref-bibr" href="#ref-21">Montesinos-López <em>et al.</em> (2018a)</a> and <a id="xref-ref-22-3" class="xref-bibr" href="#ref-22">Montesinos-López <em>et al.</em> (2018b)</a>. Interestingly the R package of <a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Montesinos-López <em>et al.</em> (2019)</a> evaluates the prediction performance of multi-trait multi-environment data in a user-friendly way. Also this R package introduced the Bayesian multi-output regressor stacking (BMORS) functions that are considerably efficient in terms of computational resources. For large datasets, the BME() and BMTME() functions of the BMTME R package are very intense in terms of computing time; however functions BMORS() and BMORS_Env() of BMORS (also included in the BMTME package) are less intensive in terms of computing time and produce similar genome-based prediction accuracies as the BMTME.</p><p id="p-9">Given the previous considerations in terms of model prediction accuracy and the necessary computing resources required to fit different univariate and multivariate models, the main objectives of this research were: (1) to extend the theory of the multi-output regressor stacking model (MORS) to a Bayesian framework in the context of genomic selection; the resulting model is called Bayesian multi-output regressor stacking (BMORS), (2) to apply the BMORS model to seven extensive maize and wheat datasets from plant breeding programs, (3) to compare the prediction accuracy of the BMORS model with the accuracy of the most popular univariate trait (UT) model: the genomic best linear unbiased prediction (GBLUP) model and its multivariate version, called the Bayesian multi-trait multi-environment (BMTME) model, and (4) to compare the computing resources employed by BMORS <em>vs.</em> BMTME and the GBLUP. This comparison will determine the usefulness of the BMORS model compared with other analytical options with much heavier computing time.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Implemented models</h3><div id="sec-3" class="subsection"><h4>Multiple-environment genotype × environment Genomic Best Linear Unbiased Predictor (GBLUP) model:</h4><p id="p-10">This model is the one proposed and described by <a id="xref-ref-21-5" class="xref-bibr" href="#ref-21">Montesinos-López <em>et al.</em> (2018a)</a>. To implement this model, the genomic relationship matrix (GRM) was calculated as suggested by <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">VanRaden (2008)</a>. This model is the conventional GBLUP that includes genotype × environment interaction; it was implemented in the Bayesian Generalized Linear Regression (BGLR) R-package of <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">de los Campos and Pérez-Rodríguez (2014)</a>. Briefly, the model can be described as follows<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-1.gif"/></noscript></span><span class="disp-formula-label">(1)</span></span>where <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-2.gif"/></noscript></span></span> is the response of the j th line in the i th environment (<span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-3.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-4.gif"/></noscript></span></span>. <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-5.gif"/></noscript></span></span>is the fixed effect of the ith environment, <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-6.gif"/></noscript></span></span>denotes the random genomic effect of the jth line, with <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-7.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-8.gif"/></noscript></span></span> is the genomic variance and <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-9.gif"/></noscript></span></span> is the genomic relationship matrix (GRM) and is calculated (<a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">VanRaden 2008</a>) as <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-10.gif"/></noscript></span></span>, where p represents the number of markers and W is the matrix of markers. The <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-11.gif"/></noscript></span></span> matrix is constructed using the observed similarity at the genomic level between lines. Further, <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-12.gif"/></noscript></span></span> is the random interaction term between the genomic effect of the jth line and the ith environment; let <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-13.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-14.gif"/></noscript></span></span> is the interaction variance, and <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-15.gif"/></noscript></span></span> is a random residual associated with the jth line in the ith environment distributed as <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-16.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-17.gif"/></noscript></span></span> is the residual variance.</p><p id="p-11">Since this model was implemented under a Bayesian framework, next we provide the priors used for the parameters. For the beta coefficients of the environments we used a normal distribution with mean 0 and very large variance <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-18.gif"/></noscript></span></span>, that is, <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-19.gif"/></noscript></span></span> For the genomic variance component, <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-20.gif"/></noscript></span></span>, we used a scaled inverse Chi-square with shape and scale parameters, <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-21.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-22.gif"/></noscript></span></span>, respectively, where the proportion of total variance <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-23.gif"/></noscript></span></span> was set to 0.5, and <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-24.gif"/></noscript></span></span> is the phenotypic variance of the response variable of the training set. Also, for the error variance component, we used a scaled inverse Chi-square with shape and scale parameters, <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-25.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-26.gif"/></noscript></span></span> respectively.</p></div><div id="sec-4" class="subsection"><h4>Bayesian multiple-trait multiple-environment (BMTME) model:</h4><p id="p-12">The BMTME model is a multivariate version of the model given in equation (1) defined as follows:<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-27.gif"/></noscript></span><span class="disp-formula-label">(2)</span></span>where Y is of dimension <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-28.gif"/></noscript></span></span>, with L the number of traits and <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-29.gif"/></noscript></span></span>, where J denotes the number of lines (genotypes) and I the number of environments, X is of order <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-30.gif"/></noscript></span></span>, β is of order <span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-31.gif"/></noscript></span></span>, since <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-32.gif"/></noscript></span></span> for <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-33.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-34.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-35.gif"/></noscript></span></span> is of order <span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-36.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-37.gif"/></noscript></span></span> is of order <span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-38.gif"/></noscript></span></span> and contains the genotype×trait interaction term since <span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-39.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-40.gif"/></noscript></span></span> is the effect of genotype × trait interaction term for <span class="inline-formula" id="inline-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-41.gif"/></noscript></span></span> and for <span class="inline-formula" id="inline-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-42.gif"/></noscript></span></span>.<span class="inline-formula" id="inline-formula-41"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-43.gif"/></noscript></span></span> is of order <span class="inline-formula" id="inline-formula-42"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-44.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-45.gif"/></noscript></span></span> is of order <span class="inline-formula" id="inline-formula-44"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-46.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-46.gif"/></noscript></span></span> and contains the genotype×environment×trait interaction, since <span class="inline-formula" id="inline-formula-45"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-47.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-47.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-46"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-48.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-48.gif"/></noscript></span></span> is the effect of genotype×environment×trait interaction for <span class="inline-formula" id="inline-formula-47"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-49.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-49.gif"/></noscript></span></span>, for <span class="inline-formula" id="inline-formula-48"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-50.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-50.gif"/></noscript></span></span> and for <span class="inline-formula" id="inline-formula-49"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-51.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-51.gif"/></noscript></span></span>. Vector <span class="inline-formula" id="inline-formula-50"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-52.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-52.gif"/></noscript></span></span>is distributed under a matrix-variate normal distribution with <span class="inline-formula" id="inline-formula-51"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-53.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-53.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-52"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-54.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-54.gif"/></noscript></span></span> is the unstructured genetic (co)variance matrix of traits of order <span class="inline-formula" id="inline-formula-53"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-55.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-55.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-54"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-56.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-56.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-55"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-57.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-57.gif"/></noscript></span></span>is an unstructured (co)variance matrix of order <span class="inline-formula" id="inline-formula-56"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-58.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-58.gif"/></noscript></span></span>and E is the matrix of residuals of order <span class="inline-formula" id="inline-formula-57"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-59.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-59.gif"/></noscript></span></span> with <span class="inline-formula" id="inline-formula-58"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-60.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-60.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-59"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-61.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-61.gif"/></noscript></span></span> is the unstructured residual (co)variance matrix of traits of order <span class="inline-formula" id="inline-formula-60"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-62.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-62.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-61"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-63.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-63.gif"/></noscript></span></span> is the genomic relationship matrix described above.</p><p id="p-13">The BMTME model resulting from equation (2) was implemented by <a id="xref-ref-20-6" class="xref-bibr" href="#ref-20">Montesinos-López <em>et al.</em> (2016)</a>. It is important to point out that model (2) takes into account the genotype×environment terms in the (<span class="inline-formula" id="inline-formula-62"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-64.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-64.gif"/></noscript></span></span> term, and for comparison purposes, we also ran the model in equation (2) but without the (<span class="inline-formula" id="inline-formula-63"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-65.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-65.gif"/></noscript></span></span> term to study the effect on prediction performance with and without the genotype×environment term. The information of the Gibbs sampler for implementing the BMTME model is found in <a id="xref-ref-22-4" class="xref-bibr" href="#ref-22">Montesinos-López <em>et al.</em> (2018b)</a>, and the priors of this model are given in detail in <a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Montesinos-López <em>et al.</em> (2018c)</a>.</p></div><div id="sec-5" class="subsection"><h4>Bayesian multi-output regressor stacking (BMORS):</h4><p id="p-14">The proposed BMORS is a Bayesian version of the multi-trait regressor stacking method proposed by <a id="xref-ref-31-3" class="xref-bibr" href="#ref-31">Spyromitros-Xioufis <em>et al.</em> (2012</a>; <a id="xref-ref-32-3" class="xref-bibr" href="#ref-32">2016)</a>. The training of BMORS consists of two stages. In the first stage, L single univariate models are implemented using the GBLUP model given in equation (1) of <a id="xref-ref-21-6" class="xref-bibr" href="#ref-21">Montesinos-López <em>et al.</em> (2018a)</a>, but instead of using the resulting predictions directly as the final output, the BMORS includes an additional training stage where a second set of L meta-models are implemented for each of the L traits under study using a Ridge regression model. Each meta-model is implemented with the following model:<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-66.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-66.gif"/></noscript></span><span class="disp-formula-label">(3)</span></span>where the covariates <span class="inline-formula" id="inline-formula-64"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-67.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-67.gif"/></noscript></span></span>, represent the scaled predictions of each trait obtained with the GBLUP model in the first-stage analysis and <span class="inline-formula" id="inline-formula-65"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-68.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-68.gif"/></noscript></span></span>, the corresponding beta coefficients. The scaling of each prediction was performed by subtracting its mean (<span class="inline-formula" id="inline-formula-66"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-69.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-69.gif"/></noscript></span></span>) and dividing by its corresponding standard deviation (<span class="inline-formula" id="inline-formula-67"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-70.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-70.gif"/></noscript></span></span>), that is, <span class="inline-formula" id="inline-formula-68"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-71.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-71.gif"/></noscript></span></span>=<span class="inline-formula" id="inline-formula-69"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-72.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-72.gif"/></noscript></span></span>, for each <span class="inline-formula" id="inline-formula-70"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-73.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-73.gif"/></noscript></span></span>. Therefore, the BMORS model contains as predictor information the scaled predictions of its response variables yielded by the first-stage models. In other words, the BMORS model is based on the idea that a second-stage model is able to correct the predictions of a first-stage model using information about the predictions of other first-stage models (<a id="xref-ref-31-4" class="xref-bibr" href="#ref-31">Spyromitros-Xioufis <em>et al.</em>, 2012</a>; <a id="xref-ref-32-4" class="xref-bibr" href="#ref-32">2016</a>).</p><p id="p-15">This method can be implemented with the BMORS() function in the BMTME package in the R statistical software (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">R Core Team 2017</a>), as shown in <strong>Appendices A</strong> and <strong>B.</strong> The R package BMTME is available at the following link: <a href="https://github.com/frahik/BMTME">https://github.com/frahik/BMTME</a> and it is described in <a id="xref-ref-24-3" class="xref-bibr" href="#ref-24">Montesinos-López <em>et al.</em> (2019)</a>. The priors for implementing this second step were: a normal with mean zero and variance <span class="inline-formula" id="inline-formula-71"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-74.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-74.gif"/></noscript></span></span> for the beta coefficients while for the variance component <span class="inline-formula" id="inline-formula-72"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-75.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-75.gif"/></noscript></span></span> and the error variance (<span class="inline-formula" id="inline-formula-73"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-76.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-76.gif"/></noscript></span></span> we used the scaled inverse Chi-squares with shape and scale parameters, <span class="inline-formula" id="inline-formula-74"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-77.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-77.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-75"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-78.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-78.gif"/></noscript></span></span> respectively, where the proportion of total variance <span class="inline-formula" id="inline-formula-76"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-79.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-79.gif"/></noscript></span></span> was set to 0.5 for the beta coefficients, equal to 0.25 for the error variance component, and <span class="inline-formula" id="inline-formula-77"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-80.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-80.gif"/></noscript></span></span> is the phenotypic variance of the response variable of the training set. Finally, it is important to point out that the underlying method of inference for all the Bayesian methods implemented was based on Markov Chain Monte Carlo.</p></div></div><div id="sec-6" class="subsection"><h3>Experimental datasets</h3><p id="p-16">A total of seven real datasets were analyzed, one dataset comprising maize lines and six datasets comprising wheat lines. All seven datasets include several environments and traits and were previously used in several studies.</p><div id="sec-7" class="subsection"><h4>Maize dataset 1:</h4><p id="p-17">This dataset has 309 maize lines and was used by <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Crossa <em>et al.</em> (2013)</a> and <a id="xref-ref-20-7" class="xref-bibr" href="#ref-20">Montesinos-López <em>et al.</em> (2016)</a>. Traits grain yield (GY), anthesis-silking interval (ASI), and plant height (PH) were evaluated and measured in three environments (E1, E2, and E3). Phenotypes are best linear unbiased estimates (BLUEs) obtained after adjusting for the experimental field design. After filtering for missing values and minor allele frequency, the number of single nucleotide polymorphisms (SNP) was 158,281. Filtering was first done by removing markers that had more than 80% of the maize lines with missing values, and then markers with minor allele frequency lower than or equal to 0.05 were deleted.</p></div><div id="sec-8" class="subsection"><h4>Wheat dataset 2:</h4><p id="p-18">This wheat dataset is composed of 250 wheat lines that were extracted from a large set of 39 yield trials grown during the 2013-2014 crop season in Ciudad Obregon, Sonora, Mexico (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Rutkoski <em>et al.</em>, 2016</a>). The measured traits were plant height (PTHT) recorded in centimeters, and days to heading (DTHD) recorded as the number of days from germination until 50% of spikes emerged in each plot, in the first replicate of each trial. Each trait was measured on 250 lines and three environments. Phenotypes were also BLUEs adjusted by the experimental design. Genomic information was obtained by genotyping-by-sequencing (GBS) and we used a total of 12,083 markers that remained after quality control. Three environments were included: bed planting with 2 irrigation levels (Bed2IR), bed planting with 5 irrigations levels (Bed5IR), and drip irrigation (Drip). Filtering also was done by removing markers that had more than 80% of the wheat lines with missing values, and markers with minor allele frequency lower than or equal to 0.01 also were deleted.</p></div><div id="sec-9" class="subsection"><h4>Wheat Iranian dataset 3:</h4><p id="p-19">This dataset consists of 2374 wheat lines evaluated in a drought environment (D) and a heat environment (H) at the CIMMYT experiment station near Ciudad Obregon, Sonora, Mexico, during the 2010-2011 cycle; it was used in <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Crossa <em>et al.</em> (2016)</a>. Two traits were measured: days to heading (DTHD) and plant height (PTHT). Both traits were measured in the two environments (Env1 and Env2) on the same 2374 lines. Of a total of 40,000 markers after quality control, we used 39,758 markers. In this dataset, markers with more than 80% of missing values were removed and markers with minor allele frequency lower than or equal to 0.05 were deleted.</p></div><div id="sec-10" class="subsection"><h4>Elite wheat yield trial (EYT) datasets 4-7:</h4><p id="p-20">These four datasets were planted at the Norman E. Borlaug Research Station, Ciudad Obregon, Sonora, Mexico, and correspond to elite yield trials (EYT) planted in four different cropping seasons with 4 or 5 environments in each cropping season. The lines that were included each year in each of the environments are the same, but those in different years are different lines. EYT dataset 4 was planted in 2013-2014 and contains 767 lines, EYT dataset 5 was planted in 2014-2015 and contains 775 lines, EYT dataset 6 was planted in 2015-2016 and contains 964 lines, and EYT dataset 7 was planted in 2016-2017 and contains 980 lines. An alpha lattice design was used and the lines were sown in 39 trials, each comprising 28 lines and two checks, with three replications and six blocks. In each dataset, four traits were recorded for each line: days to heading (DTHD, number of days from germination to 50% spike emergence), days to maturity (DTMT, number of days from germination to 50% physiological maturity or the loss of green color in 50% of the spikes), plant height (in centimeters, measured from the ground to the top of the spike), and grain yield (GY).</p><p id="p-21">In EYT datasets 4 and 7, the lines under study were evaluated in 4 environments, while in EYT datasets 5 and 6, the lines were evaluated in 5 environments. For EYT dataset 4, the environments were: bed planting with 5 irrigations (Bed5IR), early heat (EHT), flat planting and 5 irrigations (Flat5IR), and late heat (LHT). For EYT dataset 5, the environments were: bed planting with 2 irrigation levels (Bed2IR), bed planting with 5 irrigations levels (Bed5IR), early heat (EHT), flat planting with 5 irrigation levels (Flat5IR) and late heat (LHT). For EYT dataset 6, the environments were: bed planting with 2 irrigation levels (Bed2IR), bed planting with 5 irrigations levels (Bed5IR), flat planting with 5 irrigation levels (Flat5IR), flat planting with drip irrigation (FlatDrip), and late heat (LHT). Finally, for EYT dataset 7, the four environments were: bed planting with 5 irrigations (Bed5IR), early heat (EHT), flat planting with 5 irrigation levels (Flat5IR) and flat planting with drip irrigation (FlatDrip).</p><p id="p-22">Genome-wide markers for the 4,368 lines in the four datasets were obtained using genotyping-by-sequencing (GBS) (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Elshire <em>et al.</em>, 2011</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Poland <em>et al.</em>, 2012</a>) at Kansas State University using an Illumina HiSeq2500. After filtering, 2,038 markers were obtained. Imputation of missing marker data were done using LinkImpute (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Money <em>et al.</em>, 2015</a>) and implemented in TASSEL (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Bradbury <em>et al.</em>, 2007</a>), version 5. Markers that had more than 50% missing data, less than 5% minor allele frequency, and more than 10% heterozygosity were removed, and 3,485 lines were obtained (767 lines in dataset 1, 775 lines in dataset 2, 964 lines in dataset 3 and 980 lines in dataset 4).</p></div><div id="sec-11" class="subsection"><h4>Evaluation of prediction performance:</h4><p id="p-23">The prediction accuracies of the three models under study (BMORS, BMTME and UT) were evaluated with 10 random cross-validations (CV): the whole dataset was divided into a training (TRN) and a testing (TST) set; 80% (or 60%) of the whole dataset was assigned to TRN and the remaining 20% (or 40%) was assigned to TST. Since we used sampling with replacement, one observation (line) may appear in more than one partition. The CV implemented mimics a prediction problem faced by breeders in incomplete field trials, where some lines may be evaluated in some, but not all, target environments. Since <span class="inline-formula" id="inline-formula-78"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-81.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-81.gif"/></noscript></span></span> denotes the total number of records per each available trait, then to select lines in the TST dataset, we fixed the percentage of data to be used for TST [PTesting = 20% (or 40%)]. Then 0.20 (or 0.4)<span class="inline-formula" id="inline-formula-79"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-82.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-82.gif"/></noscript></span></span> (lines) were chosen at random, and subsequently for each of these lines, one environment was randomly picked from I environments. The cells selected through this algorithm were allocated to the TST dataset, while the cells (<span class="inline-formula" id="inline-formula-80"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-83.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-83.gif"/></noscript></span></span> that were not selected were assigned to the TRN dataset. Lines were sampled with replacement if <span class="inline-formula" id="inline-formula-81"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-84.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-84.gif"/></noscript></span></span>, and without replacement otherwise. The prediction accuracy was evaluated with the average Pearson’s correlation (APC) and mean arctan absolute percentage error (MAAPE) of the testing sets of the 10 random partitions that were generated with the implemented CV. It is important to point out that the first 3 datasets were implemented with 80% and 20% for TRN and TST, respectively, while the last 4 datasets were implemented with 60% and 40% for TRN and TST, respectively. It is important to point out that performance via cross-validation was based on the mean sample from the posterior distribution of predicted values. To make the models comparable in their prediction accuracy as well as on their computing time, exactly the same random cross-validations were used for the three models.</p><p id="p-24">The MAAPE is computed as the arctan of the absolute value of the difference between the observed value minus the predicted value divided by the observed value. Its advantage is that it is defined in radians and therefore scale-free and can include observations with missing values. Another advantage is that it approaches Pi over 2 for dividing by zero.</p></div></div><div id="sec-12" class="subsection"><h3>Data availability</h3><p id="p-25">All seven datasets (Maize dataset 1, Wheat dataset 2, Wheat Iranian dataset 3, EYT datasets 4-7), including phenotypic and genotypic data, can be downloaded from the following link: hdl:11529/10548141 (<a href="http://hdl.handle.net/11529/10548141">http://hdl.handle.net/11529/10548141</a>).</p></div></div><div class="section results" id="sec-13"><h2 class="">Results</h2><p id="p-26">The results of this paper are presented in seven sections, each of which corresponds to a different dataset. Genomic-enabled prediction accuracy is presented in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a> for each model (BMORS, BMTME, and UT), trait, and dataset combination as average Pearson correlations (APC) and mean arctan absolute percentage error (MAAPE) with their corresponding standard deviations. <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a> shows the time in minutes for fitting the three models for each trait and dataset.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/494539/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/494539/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Average Pearson’s correlation (APC), mean arctan absolute percentage error (MAAPE) and their standard deviation (SD) for each trait (grain yield, GY, Plant height, PH, anthesis silking interval, ASI, days to heading, DTHD, plant height, PTHT, days to maturity, DTMT) and each maize and wheat dataset under study for three models, BMTME, BMORS and UT</span><div class="sb-div caption-clear"></div></div></div><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/494510/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/494510/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Time in minutes for fitting models BMTME, BMORS and UT for each dataset. BMTME/BMORS is the ratio of the time for implementing BMTME model <em>vs.</em> BMORS; BMTME/UT is the ratio of the time for implementing BMTME <em>vs.</em> UT model and BMORS/UT is the ratio of the time for implementing BMORS <em>vs.</em> UT model</span><div class="sb-div caption-clear"></div></div></div><div id="sec-14" class="subsection"><h3>Maize dataset 1</h3><p id="p-27">In this dataset we compare the prediction accuracy of the three models (BMORS, BMTME and UT). We did not find large differences in terms of prediction accuracy between the three models (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>) across environments. The predictions in terms of APC for trait GY were 0.3449, 0.3504, and 0.3522 for models BMORS, BMTME, and UT. The APC for ASI was around 042-0.43 for the three models, and for trait PTHT, the range was between 0.4502 and 0.4787 (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). In terms of MAAPE, the range of predictions was between 0.1095 and 0.1142 for trait GY. In summary, we found only slight differences in the APC and MAAPE of the three models for each trait.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/10/3381/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Maize dataset 1. Performance of three models (BMTME, BMORS and UT) under study in terms of average Pearson&#x2019;s correlation (APC) and mean arctan absolute percentage error (MAAPE) for three traits: anthesis-silking interval (ASI), grain yield (GY) and plant height (PH) in three environments, E1, E2, and E3." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1513415800" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Maize dataset 1. Performance of three models (BMTME, BMORS and UT) under study in terms of average Pearson&#x2019;s correlation (APC) and mean arctan absolute percentage error (MAAPE) for three traits: anthesis-silking interval (ASI), grain yield (GY) and plant height (PH) in three environments, E1, E2, and E3.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/10/3381/F1.medium.gif" width="440" height="240"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/9/10/3381/F1.medium.gif" width="440" height="240"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/10/3381/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/10/3381/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494429" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-28" class="first-child">Maize dataset 1. Performance of three models (BMTME, BMORS and UT) under study in terms of average Pearson’s correlation (APC) and mean arctan absolute percentage error (MAAPE) for three traits: anthesis-silking interval (ASI), grain yield (GY) and plant height (PH) in three environments, E1, E2, and E3.</p><div class="sb-div caption-clear"></div></div></div><p id="p-29">For the whole dataset without cross-validation, we also compared the implementation time (computational resources) between the three methods and found that the slowest was the BMTME, while the fastest was the UT implemented in the BGLR package; BMTME was 13.09 and 72.42 times slower than UT and BMORS, respectively, while the BMORS method was only 5.53 times slower than UT (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>).</p></div><div id="sec-15" class="subsection"><h3>Wheat dataset 2</h3><p id="p-30">In terms of APC, we did find some differences between the three models for both traits included in this dataset (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). The BMTME gave the highest APC for trait DTHD (0.8716) and PTHT (0.4782), followed by BMORS and UT with APC around 0.85 for DTHD. The range of predictions in terms of APC for trait DTHD was between 0.8533 and 0.8716, and between 0.4617 and 0.4782 for trait PTHT. However, for MAAPE, we found small differences in prediction accuracy only in trait DTHD, and the best model was the BMTME, which, on average, was better than the BMORS and UT models by 17.63% and 14.28%, respectively. The predictions for trait DTHD ranged between 0.4633 and 0.5625, while for trait PTHT, the predictions ranged between 0.5935 and 0.6023 (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/10/3381/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Wheat dataset 2. Performance of three models (BMTME, BMORS and UT) under study in terms of average Pearson&#x2019;s correlation (APC) and mean arctan absolute percentage error (MAAPE) for two traits: days to heading (DTHD) and plant height (PTHT). Three environments were included: bed planting with two irrigation levels (Bed2IR), bed planting with 5 irrigations levels (Bed5IR), and drip irrigation (Drip)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1513415800" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Wheat dataset 2. Performance of three models (BMTME, BMORS and UT) under study in terms of average Pearson&#x2019;s correlation (APC) and mean arctan absolute percentage error (MAAPE) for two traits: days to heading (DTHD) and plant height (PTHT). Three environments were included: bed planting with two irrigation levels (Bed2IR), bed planting with 5 irrigations levels (Bed5IR), and drip irrigation (Drip).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/10/3381/F2.medium.gif" width="440" height="242"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/9/10/3381/F2.medium.gif" width="440" height="242"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/10/3381/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/10/3381/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494348" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-31" class="first-child">Wheat dataset 2. Performance of three models (BMTME, BMORS and UT) under study in terms of average Pearson’s correlation (APC) and mean arctan absolute percentage error (MAAPE) for two traits: days to heading (DTHD) and plant height (PTHT). Three environments were included: bed planting with two irrigation levels (Bed2IR), bed planting with 5 irrigations levels (Bed5IR), and drip irrigation (Drip).</p><div class="sb-div caption-clear"></div></div></div><p id="p-32">We also compared the implementation time (computational resources) between the three methods for the whole dataset, without cross-validation, and again we found that the slowest was the BMTME, while the fastest was BMORS. BMTME was 9.21 and 56.44 times slower than BMORS and UT, respectively, while the UT method was only 6.12 times slower than BMORS (<a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table2</a>).</p></div><div id="sec-16" class="subsection"><h3>Wheat Iranian dataset 3</h3><p id="p-33">For this dataset we found more differences in APC between the traits for DTMT than for DTHD (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). In trait DTMT the best predictions were observed in model BMTME (0.5182) and the worst in model BMORS (0.4288) (<a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>). The average superiority of BMTME was 17.25% and 6.33% with regard to the BMORS and UT models, respectively. The predictions in terms of APC for trait DTHD ranged between 0.5862 and 0.5918, while for trait DTMT, they ranged between 0.4288 and 0.5182.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/10/3381/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Wheat Iranian dataset 3. Performance of three models (BMTME, BMORS and UT) under study in terms of average Pearson&#x2019;s correlation (APC) and mean arctan absolute percentage error (MAAPE) for two traits: days to heading (DTHD) and days to maturity (DTMT) in two environments (Env1 and Env2)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1513415800" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Wheat Iranian dataset 3. Performance of three models (BMTME, BMORS and UT) under study in terms of average Pearson&#x2019;s correlation (APC) and mean arctan absolute percentage error (MAAPE) for two traits: days to heading (DTHD) and days to maturity (DTMT) in two environments (Env1 and Env2).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/10/3381/F3.medium.gif" width="440" height="241"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/9/10/3381/F3.medium.gif" width="440" height="241"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/10/3381/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/10/3381/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494421" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-34" class="first-child">Wheat Iranian dataset 3. Performance of three models (BMTME, BMORS and UT) under study in terms of average Pearson’s correlation (APC) and mean arctan absolute percentage error (MAAPE) for two traits: days to heading (DTHD) and days to maturity (DTMT) in two environments (Env1 and Env2).</p><div class="sb-div caption-clear"></div></div></div><p id="p-35">In terms of MAAPE, we only found differences between the three models for trait DTMT (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>); the best predictions were observed in models BMTME and UT and the worst in model BMORS, but no significant differences were observed between the BMTME and UT models. The BMTME model was superior by 7.438% to the BMORS model. The predictions in terms of MAAPE for trait DTHD ranged between 0.039 and 0.039, while for trait DTMT, they ranged between 0.056 and 0.061 (<a id="xref-table-wrap-1-5" class="xref-table" href="#T1">Table 1</a>).</p><p id="p-36">Regarding the computer resources used when fitting each model to this dataset, the BMTME was the slowest in terms of implementation time, for it was 10.3 and 58.65 times slower than the BMORS method and UT model, respectively; the UT was 5.5 times faster than the BMORS (<a id="xref-table-wrap-2-4" class="xref-table" href="#T2">Table 2</a>).</p></div><div id="sec-17" class="subsection"><h3>EYT dataset 4</h3><p id="p-37"><a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a> shows that although no large differences were found between the three models in DTHD, DTMT, and Height, the best predictions were observed in model BMTME with gains of 2.559%, 2.571% and 4.767% for traits DTHD, DTMT and Height, respectively (<a id="xref-table-wrap-1-6" class="xref-table" href="#T1">Table 1</a>), compared to the BMORS model, and gains of 2.415%, 3.098% and 5.567%, respectively, compared to the UT model. However, for trait GY, the worst predictions were observed under the BMTME (0.4311), and models BMORS and UT were better than the BMTME model by around 3% and 5%, respectively. Values of APC ranged from 0.8149 to 0.8363 for DTHD, from 0.773 to 0.797 for DTMT, from 0.431 to 0.455 for GY, and from 0.546 and 0.573 for Height.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/10/3381/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="EYT dataset 4. Performance of three models (BMTME, BMORS and UT) under study in terms of average Pearson&#x2019;s correlation (APC) and mean arctan absolute percentage error (MAAPE) for four traits: days to heading (DTHD), days to maturity (DTMT), grain yield (GY) and Height, evaluated in 4 environments: bed planting with 5 irrigations (Bed5IR), early heat (EHT), flat planting and 5 irrigations (Flat5IR), and late heat (LHT)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1513415800" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;EYT dataset 4. Performance of three models (BMTME, BMORS and UT) under study in terms of average Pearson&#x2019;s correlation (APC) and mean arctan absolute percentage error (MAAPE&lt;em&gt;)&lt;/em&gt; for four traits: days to heading (DTHD), days to maturity (DTMT), grain yield (GY) and Height, evaluated in 4 environments: bed planting with 5 irrigations (Bed5IR), early heat (EHT), flat planting and 5 irrigations (Flat5IR), and late heat (LHT).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/10/3381/F4.medium.gif" width="440" height="228"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/9/10/3381/F4.medium.gif" width="440" height="228"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/10/3381/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/10/3381/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494394" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-38" class="first-child">EYT dataset 4. Performance of three models (BMTME, BMORS and UT) under study in terms of average Pearson’s correlation (APC) and mean arctan absolute percentage error (MAAPE<em>)</em> for four traits: days to heading (DTHD), days to maturity (DTMT), grain yield (GY) and Height, evaluated in 4 environments: bed planting with 5 irrigations (Bed5IR), early heat (EHT), flat planting and 5 irrigations (Flat5IR), and late heat (LHT).</p><div class="sb-div caption-clear"></div></div></div><p id="p-39">For the MAAPE criterion, there were differences between the three models in traits DTHD and DTMT, and the best model was the BMTME with gains of 14.888% (DTHD) and 7.826% (DTMT) over the BMORS model and gains of 11.144% (DTHD) and 7.018% (DTMT) for those traits over the UT model (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>). The predictions under MAAPE ranged between 0.0303-0.0356 for DTHD, 0.0212-0.0230 for DTMT, 0.0713-0.0736 for GY, and 0.0374-0.0381 for Height (<a id="xref-table-wrap-1-7" class="xref-table" href="#T1">Table 1</a>).</p><p id="p-40">The implementation time between the three methods was as follows: the slowest was BMTME, the fastest was BMORS and UT was intermediate. BMTME was 11.45 and 60.72 times slower than BMORS and UT, respectively, while the UT method was only 5.30 times faster than BMORS (<a id="xref-table-wrap-2-5" class="xref-table" href="#T2">Table 2</a>).</p></div><div id="sec-18" class="subsection"><h3>EYT dataset 5</h3><p id="p-41">No important differences between the three models in any of the four traits in terms of APC (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>) were found. However, some differences were found for MAAPE in traits DTHD and DTMT with gains of 9.959% (DTHD) and 6.875% (DTMT) compared to the BMORS model, and gains of 8.439% (DTHD) and 6.875% (DTMT) compared to the UT model (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>). The predictions in terms of APC for trait DTHD were around 0.85, between 0.7902 and 0.8112 for trait DTMT, between 0.5486 and 0.5621 for GY, and around 0.55-0.56 for Height (<a id="xref-table-wrap-1-8" class="xref-table" href="#T1">Table 1</a>).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/10/3381/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="EYT dataset 5. Performance of three models (BMTME, BMORS and UT) under study in terms of average Pearson&#x2019;s correlation (APC) and mean arctan absolute percentage error (MAAPE) for four traits (DTHD, DTMT, GY, Height) in five environments: bed planting with 2 irrigations (Bed2IR), bed planting with 5 irrigations (Bed5IR), early heat (EHT), flat planting with 5 irrigations (Flat5IR) and late heat (LHT)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1513415800" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;EYT dataset 5. Performance of three models (BMTME, BMORS and UT) under study in terms of average Pearson&#x2019;s correlation (APC) and mean arctan absolute percentage error (MAAPE) for four traits (DTHD, DTMT, GY, Height) in five environments: bed planting with 2 irrigations (Bed2IR), bed planting with 5 irrigations (Bed5IR), early heat (EHT), flat planting with 5 irrigations (Flat5IR) and late heat (LHT).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/10/3381/F5.medium.gif" width="440" height="233"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/9/10/3381/F5.medium.gif" width="440" height="233"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/10/3381/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/10/3381/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494381" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-42" class="first-child">EYT dataset 5. Performance of three models (BMTME, BMORS and UT) under study in terms of average Pearson’s correlation (APC) and mean arctan absolute percentage error (MAAPE) for four traits (DTHD, DTMT, GY, Height) in five environments: bed planting with 2 irrigations (Bed2IR), bed planting with 5 irrigations (Bed5IR), early heat (EHT), flat planting with 5 irrigations (Flat5IR) and late heat (LHT).</p><div class="sb-div caption-clear"></div></div></div><p id="p-43">In terms of MAAPE, the range for trait DTHD was between 0.02170 and 0.02410, between 0.01490 and 0.01600 for trait DTMT, between 0.0684 and 0.0698 for trait GY and between 0.03460 and 0.03490 for trait Height. For this data set, the BMTME method was the slowest in implementation time, while the fastest was UT. BMTME was 11.61 and 60.43 times slower than BMORS and UT, respectively, while the BMORS method was only 5.2 times slower than UT (<a id="xref-table-wrap-2-6" class="xref-table" href="#T2">Table 2</a>).</p></div><div id="sec-19" class="subsection"><h3>EYT dataset 6</h3><p id="p-44">In terms of APC and MAAPE, we did not find large differences between the three models in any of the three traits (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>). However, although large differences between models in terms of APC were not found for traits DMT and GY, the BMTME model was better than the BMORS and UT models for traits DTMT and GY, whereas for DTHD the best model was BMORS (0.8412) and for Height the best model was UT (0.514) (<a id="xref-table-wrap-1-9" class="xref-table" href="#T1">Table 1</a>). In terms of MAAPE, the BMTME was superior by 6.211% (DTMT) and 3.081% (GY) compared to the BMORS, and by 5.625% (DTMT) and 2.455% (GY) compared to the UT model (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6</a>). For this dataset, the BMTME method was the slowest in implementation time, while the fastest was UT. BMTME was 12.24 and 62.89 times slower than BMORS and UT, respectively, while the BMORS method was only 5.14 times slower than UT (<a id="xref-table-wrap-2-7" class="xref-table" href="#T2">Table 2</a>).</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/10/3381/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="EYT dataset 6. Performance of three models (BMTME, BMORS and UT) under study in terms of average Pearson&#x2019;s correlation (APC) and mean arctan absolute percentage error (MAAPE) for four traits (DTHD, DTMT, GY, Height) in five environments: bed planting with 2 irrigations (Bed2IR), bed planting with 5 irrigations (Bed5IR), flat planting with 5 irrigations (Flat5IR), flat planting with drip irrigation (FlatDrip), and late heat (LHT)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1513415800" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;EYT dataset 6. Performance of three models (BMTME, BMORS and UT) under study in terms of average Pearson&#x2019;s correlation (APC) and mean arctan absolute percentage error (MAAPE) for four traits (DTHD, DTMT, GY, Height) in five environments: bed planting with 2 irrigations (Bed2IR), bed planting with 5 irrigations (Bed5IR), flat planting with 5 irrigations (Flat5IR), flat planting with drip irrigation (FlatDrip), and late heat (LHT).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/10/3381/F6.medium.gif" width="440" height="236"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/9/10/3381/F6.medium.gif" width="440" height="236"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/10/3381/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/10/3381/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494488" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-45" class="first-child">EYT dataset 6. Performance of three models (BMTME, BMORS and UT) under study in terms of average Pearson’s correlation (APC) and mean arctan absolute percentage error (MAAPE) for four traits (DTHD, DTMT, GY, Height) in five environments: bed planting with 2 irrigations (Bed2IR), bed planting with 5 irrigations (Bed5IR), flat planting with 5 irrigations (Flat5IR), flat planting with drip irrigation (FlatDrip), and late heat (LHT).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-20" class="subsection"><h3>EYT dataset 7</h3><p id="p-46"><a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7</a> and <a id="xref-table-wrap-1-10" class="xref-table" href="#T1">Table 1</a> show that the best predictions were observed in model BMTME for traits DTHD (0.8549), DTMT (0.8674), and Height (0.4306). However, for trait GY, the worst predictions were observed under the BMTME model, and models BMORS and UT were better than the BMTME model by around 8%. In terms of MAAPE, we only found differences between the three models for traits DTHD and DTMT, and the best model was the BMTME compared to the BMORS and UT models. However, for trait GY, models BMORS and UT were the best, with a superiority of around 4% compared to the BMTME model, but no relevant difference was observed for this trait between the BMORS and UT models. Finally, concerning computational resources between the three methods for fitting the whole dataset, without cross-validation, we found that the slowest was the BMTME, while the fastest was UT. BMTME was 13.08 and 69.03 times slower than BMORS and UT, respectively, while the UT method was only 5.27 times slower than BMORS (<a id="xref-table-wrap-2-8" class="xref-table" href="#T2">Table 2</a>).</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/10/3381/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="EYT dataset 7. Performance of three models (BMTME, BMORS and UT) under study in terms of average Pearson&#x2019;s correlation (APC) and mean arctan absolute percentage error (MAAPE) for four traits (DTHD, DTMT, GY, Height) in 4 environments: bed planting system with 5 irrigations (Bed5IR), early heat (EHT), flat planting with 5 irrigations (Flat5IR) and flat irrigation with drip irrigation (FlatDr)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1513415800" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;EYT dataset 7. Performance of three models (BMTME, BMORS and UT) under study in terms of average Pearson&#x2019;s correlation (APC) and mean arctan absolute percentage error (MAAPE) for four traits (DTHD, DTMT, GY, Height) in 4 environments: bed planting system with 5 irrigations (Bed5IR), early heat (EHT), flat planting with 5 irrigations (Flat5IR) and flat irrigation with drip irrigation (FlatDr).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/10/3381/F7.medium.gif" width="440" height="226"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/9/10/3381/F7.medium.gif" width="440" height="226"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/10/3381/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/10/3381/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494471" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-47" class="first-child">EYT dataset 7. Performance of three models (BMTME, BMORS and UT) under study in terms of average Pearson’s correlation (APC) and mean arctan absolute percentage error (MAAPE) for four traits (DTHD, DTMT, GY, Height) in 4 environments: bed planting system with 5 irrigations (Bed5IR), early heat (EHT), flat planting with 5 irrigations (Flat5IR) and flat irrigation with drip irrigation (FlatDr).</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-21"><h2 class="">Discussion</h2><p id="p-48">In this study, we propose the BMORS model that basically consists of a two-stage univariate process, where a conventional univariate model is applied to the training set in the first stage, and those values are used in the second stage (for training and testing); here again a univariate model is implemented with the modified training set from which the final predictions are obtained. In the first stage, a GBLUP model was trained with features of the original dataset. Next, GBLUP’s first-stage predictions were used to build a meta-prediction model using the predictions of the first stage as features, but in the second stage, a Ridge regression method was used. This meta-prediction model was employed to obtain the final predictions.</p><p id="p-49">When comparing the prediction ability of the BMORS model to that of UT and multivariate BMTME models, we found in most datasets, the BMTME model was better in terms of prediction accuracy than the UT and BMORS models. The predictions of the proposed BMORS were competitive since in general its prediction performance was similar to those of the BMTME and UT models, which may be due in part to the fact that the proposed BMORS model is a type of feature extraction technique that combines information from multiple predictive models to generate a new model. The main advantage of the proposed BMORS model is that it is considerably faster in terms of implementation time than the BMTME model. For example, in dataset 1, the BMORS method was 13.09 times faster than the BMTME, while in dataset 2, the BMORS method was 9.21 times faster than the BMTME, and in the remaining datasets (3 to 7), the BMORS was 10.3, 11.45, 11.61, 12.24 and 13.08 times faster than the BMTME, respectively, which allows its practical implementation. When comparing the implementation time, the superiority of the BMORS model is clear, since it was at least 9 times faster than the BMTME model. For this reason, although the prediction performance of BMORS is slightly lower than the prediction performance of the BMTME, this is compensated for by the fast implementation time even for moderate and large datasets, which is not possible with the BMTME model. In the proposed method, multiple weak learners can form one learner with (expected) higher prediction performance (strong learner), as opposed to a weak learner that usually performs slightly better than random guessing (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Freund and Schapire 1997</a>). However, the key difference between this technique and conventional ensemble techniques is that the multiple predictions are not from different models for the same trait or response variable, but for different correlated traits predicted with the same model. For this reason, the proposed model is called the featured extraction method, and not an ensemble technique. Also, in the second stage of the stacking process, we only included as predictors the predictions of the first stage, although some authors suggest including the original features plus the predictions of the first stage. It is important to point out that we also tested this option, but we got the worst predictions (results not presented). The proposed BMORS model can outperform the individual models (in the first stage) due to its smoothing nature and its ability to highlight each base model where it performs best and mistrust each base model where it performs poorly.</p><p id="p-50">Our results are not in agreement with those of <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Li <em>et al.</em> (2017)</a>, who used the multi-trait regressor stacking method for time series prediction of drug efficacy and found that this method was considerably superior with regard to univariate analysis in most datasets they evaluated. The results of the present study are in agreement with those of <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Melki <em>et al.</em> (2017)</a>, who evaluated the multi-trait regressor stacking method under the support vector regression framework and found the trick of expanding the predictors in the second stage with the predictions of the target traits in the first stage in order to improve prediction accuracy. These authors also proposed making small modifications to this method in order to produce a performance similar to that of the original multi-trait regressor stacking method. Our results are also in agreement with those reported by <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Santana <em>et al.</em> (2017)</a>, namely, that using the outputs (predictions of the first stage) as additional prediction features in the second stage helps to increase prediction accuracy. However, it is important to point out that under the BMOR model, the increase in prediction accuracy was very modest only in some cases.</p><p id="p-51">It is important to point out that the proposed BMORS method can be used for predicting material that has never been tested in the field, but of course we need to have the complete genotypic information of the lines to be predicted. Under this scenario, the implementation is the same as those described in this paper, but care should be taken when creating the design matrices to avoid problems in its implementation.</p><p id="p-52">An advantage of the proposed BMORS model is that its implementation can be parallelized and implemented using current genomic-enabled prediction software. In this study, we implemented the BMORS model with the BGLR software developed by <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">de los Campos and Pérez-Rodríguez (2014)</a> using a two-stage process. It is important to recall that in the first stage, separate UT models were implemented for each trait with its corresponding training set, and predictions were made for each studied trait for both the training and testing sets, and then all predictions of the traits were used as covariates in a second-stage univariate analysis. With the same training set and with the predictions as covariates (predictions of the traits) of the first stage, the final predictions for each trait were performed. However, to successfully implement the BMORS model, the predictions of all traits used as covariates in the second stage should be standardized by subtracting their mean and dividing by their corresponding standard deviation.</p><p id="p-53">Also it is important to point out that in the first stage, the proposed BMORS method allows parametric and non-parametric approaches, although here in the first stage we used the GBLUP with (co)variance structures of lines and lines<span class="inline-formula" id="inline-formula-82"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-85.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3381/embed/mml-math-85.gif"/></noscript></span></span>environments. However, other covariance structures are allowed that can be built with different types of kernels that can capture in a better way existing non-linearity in the data. Also, the first stage can be built using a pure parametric approach with existing software for genomic selection, like BGLR and others. However, in the second stage we used a Ridge regression approach, but other types of parametric and non-parametric approaches could be used; thus this method can be made more general in a very straightforward manner.</p><p id="p-54">The implementation of the BMORS model is not restricted to a GBLUP in the first stage and Ridge regression in the second stage, which means that we can implement other types of models at each stage. For example, when the BMORS model is implemented in the BGLR statistical software, this allows using Bayesian Ridge regression, Lasso, Bayes A, Bayes B, RKHS, or a mixture of models at each stage. However, if the BMORS model is implemented in conventional restricted maximum likelihood (REML) software used for genomic-enabled prediction, then each stage can only be implemented under a REML framework. This shows that the application of the BMORS model is very flexible and can be implemented using the existing genomic-enabled prediction software. <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Breiman (1996)</a> suggested that in the second stage, the beta (weights) coefficients of the predictor variables should be restricted to positive values; however, we tested this option by implementing the second stage with non-negative least squares using the library nnls (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Lawson and Hanson 1995</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Mullen and van Stokkum 2015</a>), but the prediction accuracy was considerably worse. For this reason, both stages were implemented under a Bayesian framework without restriction on the beta coefficients.</p><p id="p-55">One disadvantage of the proposed BMORS model compared to conventional multivariate analysis, is that it does not allow estimating genetic and residual covariances (and correlations) between traits. This is because the BMORS model is implemented in a two-stage process with separate UT models, which does not allow estimating these covariances. That is, the BMORS model is less interpretable than conventional multivariate analysis, which implies that the interpretation of beta coefficients makes no sense. However, for scientific studies where prediction is the main goal, not measuring the magnitude of the relationship between traits is not critical. It is important to point out that the BMORS method is appropriate for multi-trait analysis and should be preferred over the UT model when the correlation between traits is moderate or large; however, when this correlation is weak or close to zero, the UT model should be preferred because most of the time it will produce better predictions than the BMORS method and in less computing time.</p><p id="p-56">Analyzing MT models in the context of GS is very challenging due to the size and complexity of the underlying datasets, which nowadays is common in many breeding programs; for these reasons, MT models require much more computational effort than UT models. However, due to the advantages of MT models when improving parameter estimates and prediction accuracy, and given the continued and dramatic growth of computational power, MT models play an increasingly important role in data analysis in plant and animal genomic-assisted breeding for selecting the best candidate genotypes early in time. For this reason, in plant and animal breeding the BMORS model is an attractive alternative for selecting candidate genotypes since it produces similar genomic-enabled predictions for multi-trait and multi-environment data as other statistical models but with much less computing time, since across the seven datasets we found that the BMORS model is at least 9 times faster than the BMTME method. Although the three methods evaluated in this article used the Markov Chain Monte Carlo method for Bayesian inference, the differences with regard to the implementation time that we found are mostly due to the model itself, and how the methods were implemented in each library.</p></div><div class="section conclusions" id="sec-22"><h2 class="">Conclusions</h2><p id="p-57">In this research we proposed a Bayesian multiple-output regressor stacking (BMORS) model applied in the context of genomic selection; it is the Bayesian version of the multi-output regressor stacking method proposed by <a id="xref-ref-31-5" class="xref-bibr" href="#ref-31">Spyromitros-Xioufis <em>et al.</em> (2012</a>; <a id="xref-ref-32-5" class="xref-bibr" href="#ref-32">2016)</a>. The BMORS model was implemented using a two-stage process, where in the first stage, a conventional univariate GBLUP model was implemented for its corresponding training set, while in the second stage a Ridge regression model was implemented for the same training set, but using as predictors the predictions in the first stage of the L traits under study. We found that the BMORS model was similar to the UT and BMTME models in terms of prediction accuracy, but in general, the BMTME model was the best. Finally, although more empirical evidence is needed to confirm the prediction performance of the BMORS model, the proposed BMORS model is another alternative for performing multi-trait genomic-enabled prediction in plant and animal breeding programs with significantly less computational time than is required for other genomic-enabled statistical packages.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-90">We thank all scientists, field workers, and lab assistants from National Programs and CIMMYT who collected the data used in this study. We are also thankful for the financial support provided by CIMMYT CRP (maize and wheat), the Bill &amp; Melinda Gates Foundation, as well the USAID projects (Cornell University and Kansas State University) that generated the CIMMYT maize and wheat data analyzed in this study. We acknowledge the financial support provided by the Foundation for Research Levy on Agricultural Products (FFL) and the Agricultural Agreement Research Fund (JA) in Norway through NFR grant 267806.</p></div><div class="section app" id="app-1"><h2 class="">Appendix A</h2><div class="section" id="sec-23"><h3>R code for implementing the BMORS model in the BMTME package</h3><p id="p-58">rm(list=ls())</p><p id="p-59">library(BMTME)</p><p id="p-60">data(“WheatToy”) #########This data set is preloaded in the BMTME package#####</p><p id="p-61">ls()</p><p id="p-62">########Part a) Phenotypic data############################</p><p id="p-63">phenoWheatToy &lt;- phenoWheatToy[order(phenoWheatToy$Env, phenoWheatToy$Gid),]</p><p id="p-64">rownames(phenoWheatToy)=1:nrow(phenoWheatToy)</p><p id="p-65">head(phenoWheatToy)</p><p id="p-66">#########Part b) Design matrix######################################</p><p id="p-67">LG &lt;- cholesky(genoWheatToy)</p><p id="p-68">ZG &lt;- model.matrix(∼0 + as.factor(phenoWheatToy$Gid))</p><p id="p-69">Z.G &lt;- ZG %*% LG</p><p id="p-70">Z.E &lt;- model.matrix(∼0 + as.factor(phenoWheatToy$Env))</p><p id="p-71">ZEG &lt;- model.matrix(∼0 + as.factor(phenoWheatToy$Gid):as.factor(phenoWheatToy$Env))</p><p id="p-72">G2 &lt;- kronecker(diag(length(unique(phenoWheatToy$Env))), data.matrix(genoWheatToy))</p><p id="p-73">LG2 &lt;- cholesky(G2)</p><p id="p-74">Z.EG &lt;- ZEG %*% LG2</p><p id="p-75">Y &lt;- as.matrix(phenoWheatToy[, -c(1, 2)])</p><p id="p-76">head(Y)</p><p id="p-77">############Part c) Cross validation and predictor########################</p><p id="p-78">pheno &lt;- data.frame(GID = phenoWheatToy[, 1], Env = phenoWheatToy[, 2],Response = phenoWheatToy[, 3])</p><p id="p-79">ETA &lt;- list(Env = list(X = Z.E, model = “BRR”), Gen = list(X = Z.G, model = ’BRR’), EnvGen = list(X = Z.EG, model = “BRR”))</p><p id="p-80">CrossValidation &lt;-CV.RandomPart(pheno, NPartitions = 10, PTesting = 0.2,set_seed = 123)</p><p id="p-81">########### Part d) Implementing the BMORS model#############################</p><p id="p-82">pm &lt;-BMORS(Y, ETA = ETA, nIter = 15000, burnIn = 10000, thin = 2, progressBar = TRUE, testingSet = CrossValidation, digits = 4)</p><p id="p-83">summary(pm)</p><p id="p-84">boxplot(pm,select=”MAAPE”,las = 2)</p></div></div><div class="section app" id="app-2"><h2 class="">Appendix B</h2><div class="section" id="sec-24"><h3>Explanation of code of Appendix A and the output of the BMORS model</h3><p id="p-85">To be able to implement the BMORS method, it is necessary to install the BMTME library, which in addition to implementing the BMTME model also allows implementing the BMORS method. It is important to point out that the data used for illustrating the BMORS method is a toy wheat dataset called WheatToy that is preloaded in the BMTME package. For running the code given in Appendix A, you only need to copy and page this code in R and in this way you can run this in a straightforward way.</p><p id="p-86">In Part <strong>a) Phenotypic data</strong>–we are ordering the data first by environment and then in each environment by the GIDs of lines. It is important to have the same order of the design matrices (environments, lines and genotype by environment) with those of the genomic relationship matrix.</p><p id="p-87">In Part <strong>b) Design matrix</strong>–we create the design matrices of environments, lines and of genotypes by environments interaction. Here we also show how to incorporate the GRM into the design matrices of lines and genotype by environment interaction. At the end of this part the matrix of response variables (Y) is selected for which we will be implementing the BMORS method.</p><p id="p-88">In part <strong>c) Cross validation and predictor</strong>—the R code is provided for creating the training-testing sets using the CV.RandomPart, for which an object called pheno is provided that contains information on environments, lines and response variables. In the CV.RandomPart() was specified 10 random partitions, with 20% for testing and 80% for training in each random partition. Then, also in this part of the code the predictor of the model is created as a list, which in this case contains three components corresponding to environments, lines and genotype by environment interactions. Finally, in part d) Implementing the BMORS model— the BMORS() function is used to implement the BMORS method that needs as input the information of the predictor (ETA), the matrix of response variables (Y), the number of iterations (nIter), the burn-in (burnIn), the thin value, and the information of the observations that will be used as testing (testingSet). The number of digits that should be used to print the results is also specified, and a TRUE or FALSE value function is specified in progressBar, to print or not to print the progress of the training process. And finally, with the summary(pm), we printed the output, which in this case is:</p><p id="p-89">&gt; summary(pm)</p><div id="T3" class="table pos-anchor"><div class="table-inline"><table frame="border" id="table-3"><col width="12.5%" span="1" id="column-15"/><col width="12.18%" span="1" id="column-16"/><col width="15.45%" span="1" id="column-17"/><col width="9.66%" span="1" id="column-18"/><col width="15.45%" span="1" id="column-19"/><col width="15.45%" span="1" id="column-20"/><col width="19.31%" span="1" id="column-21"/><thead id="thead-3"><tr id="tr-79"><th valign="top" align="left" scope="col" rowspan="1" colspan="1" id="th-15" class="table-left table-vtop"></th><th valign="top" align="center" scope="col" rowspan="1" colspan="1" id="th-16" class="table-center table-vtop">Environment</th><th valign="top" align="center" scope="col" rowspan="1" colspan="1" id="th-17" class="table-center table-vtop">Trait</th><th valign="top" align="center" scope="col" rowspan="1" colspan="1" id="th-18" class="table-center table-vtop">Pearson</th><th valign="top" align="center" scope="col" rowspan="1" colspan="1" id="th-19" class="table-center table-vtop">SE_Pearson</th><th valign="top" align="center" scope="col" rowspan="1" colspan="1" id="th-20" class="table-center table-vtop">MAAPE</th><th valign="top" align="center" scope="col" rowspan="1" colspan="1" id="th-21" class="table-center table-vtop">SE_MAAPE</th></tr></thead><tbody id="tbody-3"><tr id="tr-80"><td valign="top" align="left" scope="row" rowspan="1" colspan="1" id="td-533" class="table-left table-vtop">1</td><td valign="top" align="left" rowspan="1" colspan="1" id="td-534" class="table-left table-vtop">Bed2IR</td><td valign="top" align="left" rowspan="1" colspan="1" id="td-535" class="table-left table-vtop">DTHD</td><td valign="top" align="char" char="." rowspan="1" colspan="1" id="td-536" class="table-char table-vtop">0.8839</td><td valign="top" align="char" char="." rowspan="1" colspan="1" id="td-537" class="table-char table-vtop">0.0606</td><td valign="top" align="char" char="." rowspan="1" colspan="1" id="td-538" class="table-char table-vtop">0.3720</td><td valign="top" align="char" char="." rowspan="1" colspan="1" id="td-539" class="table-char table-vtop">0.0398</td></tr><tr id="tr-81"><td valign="top" align="left" scope="row" rowspan="1" colspan="1" id="td-540" class="table-left table-vtop">2</td><td valign="top" align="left" rowspan="1" colspan="1" id="td-541" class="table-left table-vtop">Bed2IR</td><td valign="top" align="left" rowspan="1" colspan="1" id="td-542" class="table-left table-vtop">PTHT</td><td valign="top" align="char" char="." rowspan="1" colspan="1" id="td-543" class="table-char table-vtop">0.6065</td><td valign="top" align="char" char="." rowspan="1" colspan="1" id="td-544" class="table-char table-vtop">0.0867</td><td valign="top" align="char" char="." rowspan="1" colspan="1" id="td-545" class="table-char table-vtop">0.7267</td><td valign="top" align="char" char="." rowspan="1" colspan="1" id="td-546" class="table-char table-vtop">0.0444</td></tr><tr id="tr-82"><td valign="top" align="left" scope="row" rowspan="1" colspan="1" id="td-547" class="table-left table-vtop">3</td><td valign="top" align="left" rowspan="1" colspan="1" id="td-548" class="table-left table-vtop">Bed5IR</td><td valign="top" align="left" rowspan="1" colspan="1" id="td-549" class="table-left table-vtop">DTHD</td><td valign="top" align="char" char="." rowspan="1" colspan="1" id="td-550" class="table-char table-vtop">0.9352</td><td valign="top" align="char" char="." rowspan="1" colspan="1" id="td-551" class="table-char table-vtop">0.0197</td><td valign="top" align="char" char="." rowspan="1" colspan="1" id="td-552" class="table-char table-vtop">0.4403</td><td valign="top" align="char" char="." rowspan="1" colspan="1" id="td-553" class="table-char table-vtop">0.0590</td></tr><tr id="tr-83"><td valign="top" align="left" scope="row" rowspan="1" colspan="1" id="td-554" class="table-left table-vtop">4</td><td valign="top" align="left" rowspan="1" colspan="1" id="td-555" class="table-left table-vtop">Bed5IR</td><td valign="top" align="left" rowspan="1" colspan="1" id="td-556" class="table-left table-vtop">PTHT</td><td valign="top" align="char" char="." rowspan="1" colspan="1" id="td-557" class="table-char table-vtop">0.2215</td><td valign="top" align="char" char="." rowspan="1" colspan="1" id="td-558" class="table-char table-vtop">0.1534</td><td valign="top" align="char" char="." rowspan="1" colspan="1" id="td-559" class="table-char table-vtop">0.3790</td><td valign="top" align="char" char="." rowspan="1" colspan="1" id="td-560" class="table-char table-vtop">0.0543</td></tr><tr id="tr-84"><td valign="top" align="left" scope="row" rowspan="1" colspan="1" id="td-561" class="table-left table-vtop">5</td><td valign="top" align="left" rowspan="1" colspan="1" id="td-562" class="table-left table-vtop">Drip</td><td valign="top" align="left" rowspan="1" colspan="1" id="td-563" class="table-left table-vtop">DTHD</td><td valign="top" align="char" char="." rowspan="1" colspan="1" id="td-564" class="table-char table-vtop">0.9341</td><td valign="top" align="char" char="." rowspan="1" colspan="1" id="td-565" class="table-char table-vtop">0.0312</td><td valign="top" align="char" char="." rowspan="1" colspan="1" id="td-566" class="table-char table-vtop">0.5445</td><td valign="top" align="char" char="." rowspan="1" colspan="1" id="td-567" class="table-char table-vtop">0.0548</td></tr><tr id="tr-85"><td valign="top" align="left" scope="row" rowspan="1" colspan="1" id="td-568" class="table-left table-vtop">6</td><td valign="top" align="left" rowspan="1" colspan="1" id="td-569" class="table-left table-vtop">Drip</td><td valign="top" align="left" rowspan="1" colspan="1" id="td-570" class="table-left table-vtop">PTHT</td><td valign="top" align="char" char="." rowspan="1" colspan="1" id="td-571" class="table-char table-vtop">0.6295</td><td valign="top" align="char" char="." rowspan="1" colspan="1" id="td-572" class="table-char table-vtop">0.0733</td><td valign="top" align="char" char="." rowspan="1" colspan="1" id="td-573" class="table-char table-vtop">0.7018</td><td valign="top" align="char" char="." rowspan="1" colspan="1" id="td-574" class="table-char table-vtop">0.0278</td></tr></tbody></table><div class="table-foot"></div></div></div></div></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-91"><em>Communicating editor: E. Huang</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2019-05-09"><span class="received-label">Received </span>May 9, 2019.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2019-08-15"><span class="accepted-label">Accepted </span>August 15, 2019.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2019 Montesinos-Lopez <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3381.1" data-doi="10.1002/widm.1157"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Borchani</span>,  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Vreanda</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Bielza</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Larrañaga</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">A survey on multi-output regression.</span> <abbr class="cit-jnl-abbrev">WIREs Data Mining Knowl Discov</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">216</span>–<span class="cit-lpage">233</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/widm.1157</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DWIREs%2BData%2BMining%2BKnowl%2BDiscov%26rft.volume%253D5%26rft.spage%253D216%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fwidm.1157%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/widm.1157&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3381.2" data-doi="10.3233/IDA-160804"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Borchani</span>,  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Larrañaga</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Bielza</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Gama</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Mining multi-dimensional concept-drifting data streams using Bayesian network classifiers.</span> <abbr class="cit-jnl-abbrev">Intell. Data Anal.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">257</span>–<span class="cit-lpage">280</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3233/IDA-160804</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DIntell.%2BData%2BAnal.%26rft.volume%253D20%26rft.spage%253D257%26rft_id%253Dinfo%253Adoi%252F10.3233%252FIDA-160804%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3233/IDA-160804&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3381.3" data-doi="10.1093/bioinformatics/btm308"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bradbury</span>,  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Zhang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. E.</span> <span class="cit-name-surname">Kroon</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. M.</span> <span class="cit-name-surname">Casstevens</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Ramdoss</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">TASSEL: Software for association mapping of complex traits in diverse samples.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">2633</span>–<span class="cit-lpage">2635</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btm308</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtm308%26rft_id%253Dinfo%253Apmid%252F17586829%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btm308&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17586829&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3381.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000250673800021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3381.4" data-doi="10.1007/BF00117832"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Breiman</span>,  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Stacked Regressions.</span> <abbr class="cit-jnl-abbrev">Mach. Learn.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">49</span>–<span class="cit-lpage">64</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/BF00117832</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMach.%2BLearn.%26rft.volume%253D24%26rft.spage%253D49%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00117832%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00117832&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3381.5" data-doi="10.1186/1297&amp;ndash;9686&amp;ndash;43&amp;ndash;26"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Calus</span>,  <span class="cit-name-given-names">M. P.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. F.</span> <span class="cit-name-surname">Veerkamp</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Accuracy of multi-trait genomic selection using different methods. Genetics, Selection, Evolution</span>. <abbr class="cit-jnl-abbrev">GSE</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">26</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1297–9686–43–26</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGSE%26rft.volume%253D43%26rft.spage%253D26%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297%2526ndash%253B9686%2526ndash%253B43%2526ndash%253B26%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297&amp;ndash;9686&amp;ndash;43&amp;ndash;26&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3381.6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beyene</span>,  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kassa</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pérez-Rodríguez</span>,  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, Hickey <em>et al</em>. <span class="cit-pub-date">2013</span> Genomic prediction in maize breeding populations with genotyping-by-sequencing. G3 (Bethesda) 3: 1903–1926.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3381.7" data-doi="10.1016/j.tplants.2017.08.011"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Pérez-Rodríguez</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Cuevas</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">O. A.</span> <span class="cit-name-surname">Montesinos-López</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Jarquín</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Genomic Selection in Plant Breeding: Methods, Models, and Perspectives.</span> <abbr class="cit-jnl-abbrev">Trends Plant Sci.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">961</span>–<span class="cit-lpage">975</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.tplants.2017.08.011</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BPlant%2BSci.%26rft.volume%253D22%26rft.spage%253D961%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tplants.2017.08.011%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tplants.2017.08.011&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3381.8" data-doi="10.1534/g3.116.029637"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Jarquín</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Franco</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Pérez-Rodríguez</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Burgueño</span></span></li></ol><cite> <span class="cit-pub-date">2016</span> Genomic Prediction of Gene Bank Wheat Landraces. G3 (Bethesda) 6: 1819–1834.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.116.029637</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.116.029637%26rft_id%253Dinfo%253Apmid%252F27172218%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjg6IjYvNy8xODE5IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMC8zMzgxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-web" id="cit-9.10.3381.9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>,  <span class="cit-name-given-names">G.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Pérez-Rodríguez</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span>  Bayesian Generalized Linear Regression. R package version 1.0.4. <a href="http://CRAN.R-project.org/package=BGLR">http://CRAN.R-project.org/package=BGLR</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3381.10" data-doi="10.1371/journal.pone.0019379"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Elshire</span>,  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. C.</span> <span class="cit-name-surname">Glaubitz</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Q.</span> <span class="cit-name-surname">Sun</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. A.</span> <span class="cit-name-surname">Poland</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. E.</span> <span class="cit-name-surname">Kawamoto</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A Robust, Simple Genotyping-by-Sequencing (GBS) Approach for High Diversity Species.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">6</span>: <span class="cit-elocation-id">e19379</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0019379</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DElshire%26rft.auinit1%253DR.%2BJ.%26rft.volume%253D6%26rft.issue%253D5%26rft.spage%253De19379%26rft.epage%253De19379%26rft.atitle%253DA%2Brobust%252C%2Bsimple%2Bgenotyping-by-sequencing%2B%2528GBS%2529%2Bapproach%2Bfor%2Bhigh%2Bdiversity%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0019379%26rft_id%253Dinfo%253Apmid%252F21573248%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0019379&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21573248&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3381.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3381.11" data-doi="10.1006/jcss.1997.1504"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Freund</span>,  <span class="cit-name-given-names">Y.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. E.</span> <span class="cit-name-surname">Schapire</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">A decision-theoretic generalization of on-line learning and an application to boosting.</span> <abbr class="cit-jnl-abbrev">J. Comput. Syst. Sci.</abbr> <span class="cit-vol">55</span>: <span class="cit-fpage">119</span>–<span class="cit-lpage">139</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1006/jcss.1997.1504</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BComput.%2BSyst.%2BSci.%26rft.volume%253D55%26rft.spage%253D119%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fjcss.1997.1504%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/jcss.1997.1504&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3381.12" data-doi="10.1093/bioinformatics/btw249"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">He</span>,  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Kuhn</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Parida</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Novel applications of multitask learning and multiple output regression to multiple genetic trait prediction.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">i37</span>–<span class="cit-lpage">i43</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btw249</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtw249%26rft_id%253Dinfo%253Apmid%252F27307640%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btw249&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27307640&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3381.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3381.13" data-doi="10.1534/genetics.112.144246"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jia</span>,  <span class="cit-name-given-names">Y.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.-L.</span> <span class="cit-name-surname">Jannink</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Multiple-Trait Genomic Selection Methods Increase Genetic Value Prediction Accuracy.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">192</span>: <span class="cit-fpage">1513</span>–<span class="cit-lpage">1522</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.112.144246</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.144246%26rft_id%253Dinfo%253Apmid%252F23086217%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5Mi80LzE1MTMiO3M6NDoiYXRvbSI7czoxOToiL2dnZy85LzEwLzMzODEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3381.14" data-doi="10.1038/hdy.2015.9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Q.</span> <span class="cit-name-surname">Zhang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Ma</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Li</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Wang</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Joint prediction of multiple quantitative traits using a Bayesian multivariate antedependence model.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">115</span>: <span class="cit-fpage">29</span>–<span class="cit-lpage">36</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/hdy.2015.9</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D115%26rft.spage%253D29%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2015.9%26rft_id%253Dinfo%253Apmid%252F25873147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2015.9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25873147&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3381.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-book" id="cit-9.10.3381.15" data-doi="10.1137/1.9781611971217"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lawson</span>,  <span class="cit-name-given-names">C. L.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. J.</span> <span class="cit-name-surname">Hanson</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-source">Solving Least Squares Problems. Classics in Applied Mathematics</span>, <span class="cit-publ-name">SIAM</span>, <span class="cit-publ-loc">Philadelphia</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1137/1.9781611971217</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSolving%2BLeast%2BSquares%2BProblems.%2BClassics%2Bin%2BApplied%2BMathematics%26rft_id%253Dinfo%253Adoi%252F10.1137%252F1.9781611971217%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1137/1.9781611971217&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3381.16" data-doi="10.1038/srep40652"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Zhang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Chen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Guo</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.-Z.</span> <span class="cit-name-surname">Li</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">A novel multi-target regression framework for time-series prediction of drug efficacy.</span> <abbr class="cit-jnl-abbrev">Sci. Rep.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">40652</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/srep40652</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BRep.%26rft.volume%253D7%26rft.spage%253D40652%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsrep40652%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/srep40652&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3381.17" data-doi="10.1016/j.ins.2017.06.017"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Melki</span>,  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Cano</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Kecman</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Ventura</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Multi-target support vector regression via correlation regressor chains.</span> <abbr class="cit-jnl-abbrev">Inf. Sci.</abbr> <span class="cit-vol">415–416</span>: <span class="cit-fpage">53</span>–<span class="cit-lpage">69</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.ins.2017.06.017</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInf.%2BSci.%26rft.volume%253D415416%26rft.spage%253D53%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ins.2017.06.017%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ins.2017.06.017&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3381.18"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>,  <span class="cit-name-given-names">T. H. E.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. J.</span> <span class="cit-name-surname">Hayes</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. E.</span> <span class="cit-name-surname">Goddard</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Prediction of total genetic value using genome wide dense marker maps.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">157</span>: <span class="cit-fpage">1819</span>–<span class="cit-lpage">1829</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D157%26rft.spage%253D1819%26rft_id%253Dinfo%253Apmid%252F11290733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=11290733&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3381.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000168223400036&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3381.19" data-doi="10.1534/g3.115.021667"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Money</span>,  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Gardner</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Migicovsky</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Schwaninger</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Zhong</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Myles</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> LinkImpute: Fast and Accurate Genotype Imputation for Nonmodel Organisms. G3 (Bethesda) 5: 2383–2390.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.115.021667</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.115.021667%26rft_id%253Dinfo%253Apmid%252F26377960%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjk6IjUvMTEvMjM4MyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzkvMTAvMzM4MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3381.20"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Montesinos-López</span>,  <span class="cit-name-given-names">O. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Montesinos-López</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Crossa</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Toledo</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">O.</span> <span class="cit-name-surname">Pérez-Hernández</span></span></li></ol><cite> <span class="cit-etal">et al.</span>, <span class="cit-pub-date">2016</span> A Genomic Bayesian Multi-trait and Multi-environment Model. G3 (Bethesda) 6: 2725–2744.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3381.21" data-doi="10.1534/g3.118.200740"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Montesinos-López</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">O. A.</span> <span class="cit-name-surname">Montesinos-López</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Gianola</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Crossa</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. M.</span> <span class="cit-name-surname">Hernández-Suárez</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span><span class="cit-comment">a</span> Multi-environment genomic prediction of plant traits using deep learners with a dense architecture. G3 (Bethesda) 8: 3813–3828.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.118.200740</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.118.200740%26rft_id%253Dinfo%253Apmid%252F30291107%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjk6IjgvMTIvMzgxMyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzkvMTAvMzM4MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3381.22" data-doi="10.2134/agronj2018.06.0362"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Montesinos-López</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">O. A.</span> <span class="cit-name-surname">Montesinos-López</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Gianola</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Crossa</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. M.</span> <span class="cit-name-surname">Hernández-Suárez</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span><span class="cit-comment">b</span> <span class="cit-article-title">Multivariate Bayesian analysis of on-farm trials with multiple-trait and multiple-environment data.</span> <abbr class="cit-jnl-abbrev">Agron. J.</abbr> <span class="cit-vol">111</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">12</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2134/agronj2018.06.0362</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAgron.%2BJ.%26rft.volume%253D111%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.2134%252Fagronj2018.06.0362%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2134/agronj2018.06.0362&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3381.23" data-doi="10.1534/g3.118.200728"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Montesinos-López</span>,  <span class="cit-name-given-names">O. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Montesinos-López</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Crossa</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Gianola</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. M.</span> <span class="cit-name-surname">Hernández-Suárez</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span><span class="cit-comment">c</span> Multi-trait, multi-environment deep learning modeling for genomic-enabled prediction of plant traits. G3 (Bethesda) 8: 3829–3840.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.118.200728</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.118.200728%26rft_id%253Dinfo%253Apmid%252F30291108%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjk6IjgvMTIvMzgyOSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzkvMTAvMzM4MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3381.24" data-doi="10.1534/g3.119.400126"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Montesinos-López</span>,  <span class="cit-name-given-names">O. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Montesinos-López</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Javier Luna-Vázquez</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F. H.</span> <span class="cit-name-surname">Toledo</span></span></li></ol><cite>, and P. Pérez-Rodríguez, <span class="cit-pub-date">2019</span> An R Package for Bayesian Analysis of Multi-environment and Multi-trait Multi-environment Data for Genome-Based Prediction G3 (Bethesda) 9: 1355–1369.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.119.400126</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.119.400126%26rft_id%253Dinfo%253Apmid%252F30819822%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjg6IjkvNS8xMzU1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMC8zMzgxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3381.25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mullen</span>,  <span class="cit-name-given-names">K. M.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">I. H. M.</span> <span class="cit-name-surname">van Stokkum</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> The Lawson-Hanson algorithm for non-negative least squares (NNLS). R package.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3381.26" data-doi="10.1371/journal.pone.0032253"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>,  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. J.</span> <span class="cit-name-surname">Brown</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. E.</span> <span class="cit-name-surname">Sorrells</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. L.</span> <span class="cit-name-surname">Jannink</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Development of high-density genetic maps for barley and wheat using a novel two-enzyme genotyping-by-sequencing approach.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-elocation-id">e32253</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0032253</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DPoland%26rft.auinit1%253DJ.%2BA.%26rft.volume%253D7%26rft.issue%253D2%26rft.spage%253De32253%26rft.epage%253De32253%26rft.atitle%253DDevelopment%2Bof%2Bhigh-density%2Bgenetic%2Bmaps%2Bfor%2Bbarley%2Band%2Bwheat%2Busing%2Ba%2Bnovel%2Btwo-enzyme%2Bgenotyping-by-sequencing%2Bapproach.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0032253%26rft_id%253Dinfo%253Apmid%252F22389690%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0032253&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22389690&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3381.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-web" id="cit-9.10.3381.27"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">R Core Team</span></li></ol><cite>, <span class="cit-pub-date">2017</span>  R: A language and environment for statistical computing. R Foundation for Statistical Computing. Vienna, Austria. ISBN 3–900051–07–0. URL <a href="http://www.R-project.org/">http://www.R-project.org/</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3381.28"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rutkoski</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mondal</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Autrique</span>,  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reynolds</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Singh</span>,  <span class="cit-name-given-names">R.</span></span></li></ol><cite> <span class="cit-pub-date">2016</span>  Predictor traits from high-throughput phenotyping improve accuracy of pedigree and genomic selection for yield in wheat. G3 (Bethesda) 6: 2799–2808.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-confproc" id="cit-9.10.3381.29"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Santana</span>,  <span class="cit-name-given-names">E. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. M.</span> <span class="cit-name-surname">Mastelini</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Barbon</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span>  Deep Regressor Stacking for Air Ticket Price Prediction. In: XIII Brazilian Symposium on Information Systems: Information Systems for Participatory Digital Governance, Brazilian Computer Society (SBC), pp. 25–31.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3381.30" data-doi="10.1007/s00122-017-3029-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schulthess</span>,  <span class="cit-name-given-names">A. W.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Zhao</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. F. H.</span> <span class="cit-name-surname">Longin</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. C.</span> <span class="cit-name-surname">Reif</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Advantages and limitations of multiple-trait genomic prediction for Fusarium head blight severity in hybrid wheat (<em>Triticum aestivum</em> L.).</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">131</span>: <span class="cit-fpage">685</span>–<span class="cit-lpage">701</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-017-3029-7</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D131%26rft.spage%253D685%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-017-3029-7%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-017-3029-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-web" id="cit-9.10.3381.31"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Spyromitros-Xioufis</span>,  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Groves</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Tsoumakas</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Vlahavas</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> Multi-label classification methods for multi-target regression, arXiv preprint arXiv:1211.65811159–1168. Cornell University Library.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3381.32" data-doi="10.1007/s10994-016-5546-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Spyromitros-Xioufis</span>,  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Tsoumakas</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Groves</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Vlahavas</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Multi-target regression via input space expansion: treating targets as inputs.</span> <abbr class="cit-jnl-abbrev">Mach. Learn.</abbr> <span class="cit-vol">104</span>: <span class="cit-fpage">55</span>–<span class="cit-lpage">98</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s10994-016-5546-z</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMach.%2BLearn.%26rft.volume%253D104%26rft.spage%253D55%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10994-016-5546-z%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10994-016-5546-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3381.33" data-doi="10.3168/jds.2007-0980"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">VanRaden</span>,  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Efficient methods to compute genomic predictions.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">4414</span>–<span class="cit-lpage">4423</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3168/jds.2007-0980</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DVanRaden%26rft.auinit1%253DP.%2BM.%26rft.volume%253D91%26rft.issue%253D11%26rft.spage%253D4414%26rft.epage%253D4423%26rft.atitle%253DEfficient%2BMethods%2Bto%2BCompute%2BGenomic%2BPredictions%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2007-0980%26rft_id%253Dinfo%253Apmid%252F18946147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2007-0980&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18946147&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3381.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000260277200035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/9/10/3381.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/10/3369" title="Predicting Longitudinal Traits Derived from High-Throughput Phenomics in Contrasting Environments Using Genomic Legendre Polynomials and B-Splines" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/10/3395" title="Evolutionary Dynamics in Structured Populations Under Strong Population Genetic Forces" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/9/10"><span>Volume 9 Issue 10, October 2019</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/9/10.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/9/10.cover-source.jpg?itok=yA8kifEd" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 9 (10)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/10/3381" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-sYJFmyl8cLXJyoqV--fW5x8ysAGN4EjoRRjvhRqluUo" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/genomic-prediction" class="" data-icon-position="" data-hide-link-title="0">Genomic Prediction</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/9/10/3381" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/9/10/3381" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/9/10/3381" class="active" data-icon-position="" data-hide-link-title="0">A Bayesian Genomic Multi-output Regressor Stacking Model for Predicting Multi-trait Multi-environment Plant Breeding Data</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/494285" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-XwnkgQY1eSlH4MYh0I97oEKWloJNp_CrdsY_uOEISlA" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/10/3381.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/9/10/3381" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-Wc9katLMtjzZFEGbCOvIOEesrKdPHnhaS4XZiDMZ8vY" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/9/10/3381" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/10/3381.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494285" id="node494285--2" data-pisa="ggg;9/10/3381" data-pisa-master="ggg;g3.119.400336" data-apath="/ggg/9/10/3381.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Bayesian Genomic Multi-output Regressor Stacking Model for Predicting Multi-trait Multi-environment Plant Breeding Data</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Osval A. Montesinos-López</span>, <span class="highwire-citation-author" data-delta="1">Abelardo Montesinos-López</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0001-9429-5855" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>José Crossa</span>, <span class="highwire-citation-author" data-delta="3">Jaime Cuevas</span>, <span class="highwire-citation-author" data-delta="4">José C. Montesinos-López</span>, <span class="highwire-citation-author" data-delta="5">Zitlalli Salas Gutiérrez</span>, <span class="highwire-citation-author" data-delta="6">Morten Lillemo</span>, <span class="highwire-citation-author" data-delta="7">Juliana Philomin</span> and <span class="highwire-citation-author" data-delta="8">Ravi Singh</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3381-3393;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400336 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494285--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Osval A. Montesinos-López </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Facultad de Telemática, Universidad de Colima, Colima, Colima, 28040, México,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Osval%2BA.%2BMontesinos-L%C3%B3pez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Montesinos-L%C3%B3pez%20OA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AOsval%2BA.%2BMontesinos-L%25C3%25B3pez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Abelardo Montesinos-López </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Departamento de Matemáticas, Centro Universitario de Ciencias Exactas e Ingenierías (CUCEI), Universidad de Guadalajara, Guadalajara, Jalisco, 44430, México,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Abelardo%2BMontesinos-L%C3%B3pez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Montesinos-L%C3%B3pez%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAbelardo%2BMontesinos-L%25C3%25B3pez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:j.crossa@cgiar.org" class="" data-icon-position="" data-hide-link-title="0">j.crossa@cgiar.org</a>
<a href="mailto:aml_uach@hotmail.com" class="" data-icon-position="" data-hide-link-title="0">aml_uach@hotmail.com</a></span></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">José Crossa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>International Maize and Wheat Improvement Center (CIMMYT), Apdo. Postal 6-641, Ciudad de México, 06600, México,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jos%C3%A9%2BCrossa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Crossa%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJos%25C3%25A9%2BCrossa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-9429-5855" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for José Crossa</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:j.crossa@cgiar.org" class="" data-icon-position="" data-hide-link-title="0">j.crossa@cgiar.org</a>
<a href="mailto:aml_uach@hotmail.com" class="" data-icon-position="" data-hide-link-title="0">aml_uach@hotmail.com</a></span></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Jaime Cuevas </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Universidad de Quintana Roo, Chetumal, Quintana Roo, México,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jaime%2BCuevas%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cuevas%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJaime%2BCuevas%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">José C. Montesinos-López </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Departamento de Estadística, Centro de Investigación en Matemáticas, Guanajuato, Guanajuato, 36023, México, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jos%C3%A9%2BC.%2BMontesinos-L%C3%B3pez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Montesinos-L%C3%B3pez%20JC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJos%25C3%25A9%2BC.%2BMontesinos-L%25C3%25B3pez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Zitlalli Salas Gutiérrez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Departamento de Estadística, Centro de Investigación en Matemáticas, Guanajuato, Guanajuato, 36023, México, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zitlalli%2BSalas%2BGuti%C3%A9rrez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Guti%C3%A9rrez%20ZS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AZitlalli%2BSalas%2BGuti%25C3%25A9rrez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Morten Lillemo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>Department of Plant Sciences, Norwegian University of Life Sciences, IHA/CIGENE, P.O. Box 5003, NO-1432 Ås, Norway</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Morten%2BLillemo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lillemo%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMorten%2BLillemo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Juliana Philomin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>International Maize and Wheat Improvement Center (CIMMYT), Apdo. Postal 6-641, Ciudad de México, 06600, México,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Juliana%2BPhilomin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Philomin%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJuliana%2BPhilomin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Ravi Singh </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>International Maize and Wheat Improvement Center (CIMMYT), Apdo. Postal 6-641, Ciudad de México, 06600, México,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ravi%2BSingh%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Singh%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARavi%2BSingh%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/9/10/3381" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/9/10/3381&amp;title=A%20Bayesian%20Genomic%20Multi-output%20Regressor%20Stacking%20Model%20for%20Predicting%20Multi-trait%20Multi-environment%20Plant%20Breeding%20Data" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/9/10/3381&amp;title=A%20Bayesian%20Genomic%20Multi-output%20Regressor%20Stacking%20Model%20for%20Predicting%20Multi-trait%20Multi-environment%20Plant%20Breeding%20Data" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/9/10/3381&amp;title=A%20Bayesian%20Genomic%20Multi-output%20Regressor%20Stacking%20Model%20for%20Predicting%20Multi-trait%20Multi-environment%20Plant%20Breeding%20Data" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/9/10/3381&amp;text=A%20Bayesian%20Genomic%20Multi-output%20Regressor%20Stacking%20Model%20for%20Predicting%20Multi-trait%20Multi-environment%20Plant%20Breeding%20Data" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/9/10/3381&amp;title=A%20Bayesian%20Genomic%20Multi-output%20Regressor%20Stacking%20Model%20for%20Predicting%20Multi-trait%20Multi-environment%20Plant%20Breeding%20Data" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/9/10/3381&amp;t=A%20Bayesian%20Genomic%20Multi-output%20Regressor%20Stacking%20Model%20for%20Predicting%20Multi-trait%20Multi-environment%20Plant%20Breeding%20Data" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/9/10/3381&amp;title=A%20Bayesian%20Genomic%20Multi-output%20Regressor%20Stacking%20Model%20for%20Predicting%20Multi-trait%20Multi-environment%20Plant%20Breeding%20Data" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/9/10/3381&amp;title=A%20Bayesian%20Genomic%20Multi-output%20Regressor%20Stacking%20Model%20for%20Predicting%20Multi-trait%20Multi-environment%20Plant%20Breeding%20Data" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/494285/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494285" id="node494285--3" data-pisa="ggg;9/10/3381" data-pisa-master="ggg;g3.119.400336" data-apath="/ggg/9/10/3381.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Bayesian Genomic Multi-output Regressor Stacking Model for Predicting Multi-trait Multi-environment Plant Breeding Data</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Osval A. Montesinos-López</span>, <span class="highwire-citation-author" data-delta="1">Abelardo Montesinos-López</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0001-9429-5855" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>José Crossa</span>, <span class="highwire-citation-author" data-delta="3">Jaime Cuevas</span>, <span class="highwire-citation-author" data-delta="4">José C. Montesinos-López</span>, <span class="highwire-citation-author" data-delta="5">Zitlalli Salas Gutiérrez</span>, <span class="highwire-citation-author" data-delta="6">Morten Lillemo</span>, <span class="highwire-citation-author" data-delta="7">Juliana Philomin</span> and <span class="highwire-citation-author" data-delta="8">Ravi Singh</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3381-3393;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400336 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494285--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Osval A. Montesinos-López </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Facultad de Telemática, Universidad de Colima, Colima, Colima, 28040, México,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Osval%2BA.%2BMontesinos-L%C3%B3pez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Montesinos-L%C3%B3pez%20OA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AOsval%2BA.%2BMontesinos-L%25C3%25B3pez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Abelardo Montesinos-López </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Departamento de Matemáticas, Centro Universitario de Ciencias Exactas e Ingenierías (CUCEI), Universidad de Guadalajara, Guadalajara, Jalisco, 44430, México,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Abelardo%2BMontesinos-L%C3%B3pez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Montesinos-L%C3%B3pez%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAbelardo%2BMontesinos-L%25C3%25B3pez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:j.crossa@cgiar.org" class="" data-icon-position="" data-hide-link-title="0">j.crossa@cgiar.org</a>
<a href="mailto:aml_uach@hotmail.com" class="" data-icon-position="" data-hide-link-title="0">aml_uach@hotmail.com</a></span></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">José Crossa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>International Maize and Wheat Improvement Center (CIMMYT), Apdo. Postal 6-641, Ciudad de México, 06600, México,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jos%C3%A9%2BCrossa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Crossa%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJos%25C3%25A9%2BCrossa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-9429-5855" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for José Crossa</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:j.crossa@cgiar.org" class="" data-icon-position="" data-hide-link-title="0">j.crossa@cgiar.org</a>
<a href="mailto:aml_uach@hotmail.com" class="" data-icon-position="" data-hide-link-title="0">aml_uach@hotmail.com</a></span></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Jaime Cuevas </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Universidad de Quintana Roo, Chetumal, Quintana Roo, México,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jaime%2BCuevas%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cuevas%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJaime%2BCuevas%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">José C. Montesinos-López </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Departamento de Estadística, Centro de Investigación en Matemáticas, Guanajuato, Guanajuato, 36023, México, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jos%C3%A9%2BC.%2BMontesinos-L%C3%B3pez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Montesinos-L%C3%B3pez%20JC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJos%25C3%25A9%2BC.%2BMontesinos-L%25C3%25B3pez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Zitlalli Salas Gutiérrez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Departamento de Estadística, Centro de Investigación en Matemáticas, Guanajuato, Guanajuato, 36023, México, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zitlalli%2BSalas%2BGuti%C3%A9rrez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Guti%C3%A9rrez%20ZS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AZitlalli%2BSalas%2BGuti%25C3%25A9rrez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Morten Lillemo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>Department of Plant Sciences, Norwegian University of Life Sciences, IHA/CIGENE, P.O. Box 5003, NO-1432 Ås, Norway</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Morten%2BLillemo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lillemo%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMorten%2BLillemo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Juliana Philomin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>International Maize and Wheat Improvement Center (CIMMYT), Apdo. Postal 6-641, Ciudad de México, 06600, México,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Juliana%2BPhilomin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Philomin%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJuliana%2BPhilomin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Ravi Singh </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>International Maize and Wheat Improvement Center (CIMMYT), Apdo. Postal 6-641, Ciudad de México, 06600, México,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ravi%2BSingh%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Singh%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARavi%2BSingh%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=3381&amp;rft.epage=3393&amp;rft.atitle=A%20Bayesian%20Genomic%20Multi-output%20Regressor%20Stacking%20Model%20for%20Predicting%20Multi-trait%20Multi-environment%20Plant%20Breeding%20Data&amp;rft.volume=9&amp;rft.issue=10&amp;rft.date=2019-10-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Montesinos-L%C3%B3pez%2C+Osval+A.&amp;rft.au=Montesinos-L%C3%B3pez%2C+Abelardo&amp;rft.au=Crossa%2C+Jos%C3%A9&amp;rft.au=Cuevas%2C+Jaime&amp;rft.au=Montesinos-L%C3%B3pez%2C+Jos%C3%A9+C.&amp;rft.au=Guti%C3%A9rrez%2C+Zitlalli+Salas&amp;rft.au=Lillemo%2C+Morten&amp;rft.au=Philomin%2C+Juliana&amp;rft.au=Singh%2C+Ravi"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/494285/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/494285/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/494285/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/494285/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/494285/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/494285/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/494285/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/494285/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/494285/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/494285/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/494285/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/494285/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495864" id="node495864" data-pisa="ggg;10/3/1113" data-pisa-master="ggg;g3.119.400968" data-apath="/ggg/10/3/1113.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1113" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Multi-trait Genomic Prediction Model Increased the Predictive Ability for Agronomic and Malting Quality Traits in Barley (<em>Hordeum vulgare</em> L.)</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495543" id="node495543" data-pisa="ggg;10/2/769" data-pisa-master="ggg;g3.119.400759" data-apath="/ggg/10/2/769.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/2/769" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Novel Bayesian Networks for Genomic Prediction of Developmental Traits in Biomass Sorghum</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495557" id="node495557" data-pisa="ggg;10/2/539" data-pisa-master="ggg;g3.119.400866" data-apath="/ggg/10/2/539.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/2/539" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Optimized Genetic Testing for Polledness in Multiple Breeds of Cattle</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Genomic%20Prediction" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Genomic Prediction</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/9/10/3381" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-13" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-21" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#sec-22" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Conclusions</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#app-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Appendix A</a></li><li class="even"><a href="#app-2" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Appendix B</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/9/10/3381.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/9/10/3381.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/9/10/3381.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
