<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/9/10/3239" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/9/10/3239.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/9/10/3239.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/9/10/3239.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/9/10/3239/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/9/10/3239.atom" />
<meta name="HW.pisa" content="ggg;9/10/3239" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Exclusion and Genomic Relatedness Methods for Assignment of Parentage Using Genotyping-by-Sequencing Data" />
<meta name="DC.Identifier" content="10.1534/g3.119.400501" />
<meta name="DC.Date" content="2019-10-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2019 Dodds et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Genotypes are often used to assign parentage in agricultural and ecological settings. Sequencing can be used to obtain genotypes but does not provide unambiguous genotype calls, especially when sequencing depth is low in order to reduce costs. In that case, standard parentage analysis methods no longer apply. A strategy for using low-depth sequencing data for parentage assignment is developed here. It entails the use of relatedness estimates along with a metric termed excess mismatch rate which, for parent-offspring pairs or trios, is the difference between the observed mismatch rate and the rate expected under a model of inheritance and allele reads without error. When more than one putative parent has similar statistics, bootstrapping can provide a measure of the relatedness similarity. Putative parent-offspring trios can be further checked for consistency by comparing the offspring’s estimated inbreeding to half the parent relatedness. Suitable thresholds are required for each metric. These methods were applied to a deer breeding operation consisting of two herds of different breeds. Relatedness estimates were more in line with expectation when the herds were analyzed separately than when combined, although this did not alter which parents were the best matches with each offspring. Parentage results were largely consistent with those based on a microsatellite parentage panel with three discordant parent assignments out of 1561. Two models are investigated to allow the parentage metrics to be calculated with non-random selection of alleles. The tools and strategies given here allow parentage to be assigned from low-depth sequencing data." />
<meta name="DC.Contributor" content="Ken G. Dodds" />
<meta name="DC.Contributor" content="John C. McEwan" />
<meta name="DC.Contributor" content="Rudiger Brauning" />
<meta name="DC.Contributor" content="Tracey C. van Stijn" />
<meta name="DC.Contributor" content="Suzanne J. Rowe" />
<meta name="DC.Contributor" content="K. Mary McEwan" />
<meta name="DC.Contributor" content="Shannon M. Clarke" />
<meta name="article:published_time" content="2019-10-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Exclusion and Genomic Relatedness Methods for Assignment of Parentage Using Genotyping-by-Sequencing Data" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Genotypes are often used to assign parentage in agricultural and ecological settings. Sequencing can be used to obtain genotypes but does not provide unambiguous genotype calls, especially when sequencing depth is low in order to reduce costs. In that case, standard parentage analysis methods no longer apply. A strategy for using low-depth sequencing data for parentage assignment is developed here. It entails the use of relatedness estimates along with a metric termed excess mismatch rate which, for parent-offspring pairs or trios, is the difference between the observed mismatch rate and the rate expected under a model of inheritance and allele reads without error. When more than one putative parent has similar statistics, bootstrapping can provide a measure of the relatedness similarity. Putative parent-offspring trios can be further checked for consistency by comparing the offspring’s estimated inbreeding to half the parent relatedness. Suitable thresholds are required for each metric. These methods were applied to a deer breeding operation consisting of two herds of different breeds. Relatedness estimates were more in line with expectation when the herds were analyzed separately than when combined, although this did not alter which parents were the best matches with each offspring. Parentage results were largely consistent with those based on a microsatellite parentage panel with three discordant parent assignments out of 1561. Two models are investigated to allow the parentage metrics to be calculated with non-random selection of alleles. The tools and strategies given here allow parentage to be assigned from low-depth sequencing data.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2019/10/01" />
<meta name="citation_mjid" content="ggg;9/10/3239" />
<meta name="citation_id" content="9/10/3239" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/9/10/3239" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/9/10/3239.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/9/10/3239.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/9/10/3239.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.119.400501" />
<meta name="citation_pmid" content="31383721" />
<meta name="citation_volume" content="9" />
<meta name="citation_issue" content="10" />
<meta name="citation_num_pages" content="9" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="3239" />
<meta name="citation_lastpage" content="3247" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Ken G. Dodds" />
<meta name="citation_author_institution" content="AgResearch, Invermay Agricultural Centre, Private Bag 50034, Mosgiel 9053, New Zealand" />
<meta name="citation_author_email" content="ken.dodds@agresearch.co.nz" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-9347-6379" />
<meta name="citation_author" content="John C. McEwan" />
<meta name="citation_author_institution" content="AgResearch, Invermay Agricultural Centre, Private Bag 50034, Mosgiel 9053, New Zealand" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-4801-6207" />
<meta name="citation_author" content="Rudiger Brauning" />
<meta name="citation_author_institution" content="AgResearch, Invermay Agricultural Centre, Private Bag 50034, Mosgiel 9053, New Zealand" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-5068-9166" />
<meta name="citation_author" content="Tracey C. van Stijn" />
<meta name="citation_author_institution" content="AgResearch, Invermay Agricultural Centre, Private Bag 50034, Mosgiel 9053, New Zealand" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-3714-3542" />
<meta name="citation_author" content="Suzanne J. Rowe" />
<meta name="citation_author_institution" content="AgResearch, Invermay Agricultural Centre, Private Bag 50034, Mosgiel 9053, New Zealand" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-8727-5960" />
<meta name="citation_author" content="K. Mary McEwan" />
<meta name="citation_author_institution" content="AgResearch, Invermay Agricultural Centre, Private Bag 50034, Mosgiel 9053, New Zealand" />
<meta name="citation_author" content="Shannon M. Clarke" />
<meta name="citation_author_institution" content="AgResearch, Invermay Agricultural Centre, Private Bag 50034, Mosgiel 9053, New Zealand" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-4615-8917" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. S. Ackerman;citation_author=P. Johri;citation_author=K. Spitze;citation_author=S. Xu;citation_author=T. G. Doak;citation_title=Estimating seven coefficients of pairwise relatedness using population-genomic data.;citation_volume=206;citation_year=2017;citation_pmid=28341647;citation_doi=10.1534/genetics.116.190660" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=I. Aguilar;citation_author=I. Misztal;citation_author=D. L. Johnson;citation_author=A. Legarra;citation_author=S. Tsuruta;citation_title=Hot topic: a unified approach to utilize phenotypic, full pedigree, and genomic information for genetic evaluation of Holstein final score.;citation_pages=743-752;citation_volume=93;citation_year=2010;citation_issue=2;citation_pmid=20105546;citation_doi=10.3168/jds.2009-2730" />
<meta name="citation_reference" content="citation_journal_title=Nat. Rev. Genet.;citation_author=K. R. Andrews;citation_author=J. M. Good;citation_author=M. R. Miller;citation_author=G. Luikart;citation_author=P. A. Hohenlohe;citation_title=Harnessing the power of RADseq for ecological and evolutionary genomics.;citation_pages=81-92;citation_volume=17;citation_year=2016;citation_pmid=26729255;citation_doi=10.1038/nrg.2015.28" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=N. A. Baird;citation_author=P. D. Etter;citation_author=T. S. Atwood;citation_author=M. C. Currey;citation_author=A. L. Shiver;citation_title=Rapid SNP discovery and genetic mapping using sequenced RAD markers.;citation_pages=e3376-e3376;citation_volume=3;citation_year=2008;citation_issue=10;citation_pmid=18852878;citation_doi=10.1371/journal.pone.0003376" />
<meta name="citation_reference" content="citation_journal_title=Anim. Genet.;citation_author=T. P. Bilton;citation_author=A. J. Chappell;citation_author=S. M. Clarke;citation_author=R. Brauning;citation_author=K. G. Dodds;citation_title=Using genotyping-by-sequencing to predict gender in animals.;citation_pages=307-310;citation_volume=50;citation_year=2019;citation_doi=10.1111/age.12782" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=T. P. Bilton;citation_author=J. C. McEwan;citation_author=S. M. Clarke;citation_author=R. Brauning;citation_author=T. C. Van Stijn;citation_title=Linkage disequilibrium estimation in low coverage high-throughput sequencing data.;citation_volume=209;citation_year=2018;citation_pmid=29588288;citation_doi=10.1534/genetics.118.300831" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=T. P. Bilton;citation_author=M. R. Schofield;citation_author=M. A. Black;citation_author=D. Chagné;citation_author=P. L. Wilcox;citation_title=Accounting for errors in low coverage high-throughput sequencing data when constructing genetic maps using biparental outcrossed populations.;citation_volume=209;citation_year=2018;citation_pmid=29487138;citation_doi=10.1534/genetics.117.300627" />
<meta name="citation_reference" content="citation_journal_title=Gigascience;citation_author=C. C. Chang;citation_author=C. C. Chow;citation_author=L. C. A. M. Tellier;citation_author=S. Vattikuti;citation_author=S. M. Purcell;citation_title=Second-generation PLINK: rising to the challenge of larger and richer datasets.;citation_pages=7;citation_volume=4;citation_year=2015;citation_pmid=25722852;citation_doi=10.1186/s13742-015-0047-8" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=K. G. Dodds;citation_author=J. C. McEwan;citation_author=R. Brauning;citation_author=R. A. Anderson;citation_author=T. C. Van Stijn;citation_title=Construction of relatedness matrices using genotyping-by-sequencing data.;citation_pages=1047;citation_volume=16;citation_year=2015;citation_doi=10.1186/s12864-015-2252-3" />
<meta name="citation_reference" content="citation_journal_title=Aust. J. Exp. Agric.;citation_author=K. G. Dodds;citation_author=J. A. Sise;citation_author=M. L. Tate;citation_title=Practical aspects of a genetic evaluation system using parentage assigned from genetic markers.;citation_pages=935-940;citation_volume=45;citation_year=2005;citation_doi=10.1071/EA05056" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=R. J. Elshire;citation_author=J. C. Glaubitz;citation_author=Q. Sun;citation_author=J. A. Poland;citation_author=K. Kawamoto;citation_title=A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species.;citation_pages=e19379-e19379;citation_volume=6;citation_year=2011;citation_issue=5;citation_pmid=21573248;citation_doi=10.1371/journal.pone.0019379" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=K. E. Grashei;citation_author=J. Ødegård;citation_author=T. H. E. Meuwissen;citation_title=Using genomic relationship likelihood for parentage assignment.;citation_pages=26;citation_volume=50;citation_year=2018;citation_doi=10.1186/s12711-018-0397-7" />
<meta name="citation_reference" content="citation_journal_title=Gigascience;citation_author=K. Hanghøj;citation_author=I. Moltke;citation_author=P. A. Andersen;citation_author=A. Manica;citation_author=T. S. Korneliussen;citation_title=Fast and accurate relatedness estimation from high-throughput sequencing data in the presence of inbreeding.;citation_pages=giz034;citation_volume=8;citation_year=2019;citation_doi=10.1093/gigascience/giz034" />
<meta name="citation_reference" content="citation_journal_title=Plant Sci.;citation_author=C. Kim;citation_author=H. Guo;citation_author=W. Kong;citation_author=R. Chandnani;citation_author=L.-S. Shuang;citation_title=Application of genotyping by sequencing technology to a variety of crop breeding programs.;citation_pages=14-22;citation_volume=242;citation_year=2016;citation_doi=10.1016/j.plantsci.2015.04.016" />
<meta name="citation_reference" content="citation_journal_title=BMC Bioinformatics;citation_author=T. S. Korneliussen;citation_author=A. Albrechtsen;citation_author=R. Nielsen;citation_title=ANGSD: analysis of next generation sequencing data.;citation_pages=356;citation_volume=15;citation_year=2014;citation_pmid=25420514;citation_doi=10.1186/s12859-014-0356-4" />
<meta name="citation_reference" content="citation_journal_title=Livest. Sci.;citation_author=A. Legarra;citation_author=O. F. Christensen;citation_author=I. Aguilar;citation_author=I. Misztal;citation_title=Single Step, a general approach for genomic selection.;citation_pages=54-65;citation_volume=166;citation_year=2014;citation_doi=10.1016/j.livsci.2014.04.029" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=T. C. Marshall;citation_author=J. Slate;citation_author=L. E. B. Kruuk;citation_author=J. M. Pemberton;citation_title=Statistical confidence for likelihood-based paternity inference in natural populations.;citation_pages=639-655;citation_volume=7;citation_year=1998;citation_issue=5;citation_pmid=9633105;citation_doi=10.1046/j.1365-294x.1998.00374.x" />
<meta name="citation_reference" content="citation_journal_title=J. Anim. Sci.;citation_author=K. L. Moore;citation_author=C. Vilela;citation_author=K. Kaseja;citation_author=R. Mrode;citation_author=M. Coffey;citation_title=Forensic use of the genomic relationship matrix to validate and discover livestock pedigrees.;citation_pages=35-42;citation_volume=97;citation_year=2019;citation_doi=10.1093/jas/sky407" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=S. B. Ng;citation_author=K. J. Buckingham;citation_author=C. Lee;citation_author=A. W. Bigham;citation_author=H. K. Tabor;citation_title=Exome sequencing identifies the cause of a mendelian disorder.;citation_pages=30-35;citation_volume=42;citation_year=2009;citation_pmid=19915526;citation_doi=10.1038/ng.499" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=A. Ott;citation_author=S. Liu;citation_author=J. C. Schnable;citation_author=C. Yeh;citation_author=K.-S. Wang;citation_title=tGBS® genotyping-by-sequencing enables reliable genotyping of heterozygous loci.;citation_pages=e178;citation_volume=45;citation_year=2017;citation_doi=10.1093/nar/gkx853" />
<meta name="citation_reference" content="citation_journal_title=Molecular phylogenetics and evolution;citation_journal_abbrev=Mol Phylogenet Evol;citation_author=C. Pitra;citation_author=J. Fickel;citation_author=E. Meijaard;citation_author=C. Groves;citation_title=Evolution and phylogeny of old world deer.;citation_pages=880-895;citation_volume=33;citation_year=2004;citation_issue=3;citation_pmid=15522810;citation_doi=10.1016/j.ympev.2004.07.013" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=J. E. Powell;citation_author=P. M. Visscher;citation_author=M. E. Goddard;citation_title=Reconciling the analysis of IBD and IBS in complex trait studies.;citation_pages=800-805;citation_volume=11;citation_year=2010;citation_issue=11;citation_pmid=20877324;citation_doi=10.1038/nrg2865" />
<meta name="citation_reference" content="citation_journal_title=Ecol. Evol.;citation_author=V. Städele;citation_author=L. Vigilant;citation_title=Strategies for determining kinship in wild populations using genetic data.;citation_pages=6107-6120;citation_volume=6;citation_year=2016;citation_doi=10.1002/ece3.2346" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol. Resour.;citation_author=D. J. Thrasher;citation_author=B. G. Butcher;citation_author=L. Campagna;citation_author=M. S. Webster;citation_author=I. J. Lovette;citation_title=Double-digest RAD sequencing outperforms microsatellite loci at assigning paternity and estimating relatedness: A proof of concept in a highly promiscuous bird.;citation_pages=953-965;citation_volume=18;citation_year=2018;citation_doi=10.1111/1755-0998.12771" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=B. S. Weir;citation_author=J. Goudet;citation_title=A unified characterization of population structure and relatedness.;citation_volume=206;citation_year=2017;citation_pmid=28550018;citation_doi=10.1534/genetics.116.198424" />
<meta name="citation_reference" content="citation_journal_title=J. Anim. Breed. Genet.;citation_author=A. Whalen;citation_author=G. Gorjanc;citation_author=J. M. Hickey;citation_title=Parentage assignment with genotyping-by-sequencing data.;citation_pages=102-112;citation_volume=136;citation_year=2019" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Exclusion and Genomic Relatedness Methods for Assignment of Parentage Using Genotyping-by-Sequencing Data" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10.cover-source.jpg" />
<meta name="twitter:description" content="Genotypes are often used to assign parentage in agricultural and ecological settings. Sequencing can be used to obtain genotypes but does not provide unambiguous genotype calls, especially when sequencing depth is low in order to reduce costs. In that case, standard parentage analysis methods no longer apply. A strategy for using low-depth sequencing data for parentage assignment is developed here. It entails the use of relatedness estimates along with a metric termed excess mismatch rate which, for parent-offspring pairs or trios, is the difference between the observed mismatch rate and the rate expected under a model of inheritance and allele reads without error. When more than one putative parent has similar statistics, bootstrapping can provide a measure of the relatedness similarity. Putative parent-offspring trios can be further checked for consistency by comparing the offspring’s estimated inbreeding to half the parent relatedness. Suitable thresholds are required for each metric. These methods were applied to a deer breeding operation consisting of two herds of different breeds. Relatedness estimates were more in line with expectation when the herds were analyzed separately than when combined, although this did not alter which parents were the best matches with each offspring. Parentage results were largely consistent with those based on a microsatellite parentage panel with three discordant parent assignments out of 1561. Two models are investigated to allow the parentage metrics to be calculated with non-random selection of alleles. The tools and strategies given here allow parentage to be assigned from low-depth sequencing data." />
<meta name="og-title" property="og:title" content="Exclusion and Genomic Relatedness Methods for Assignment of Parentage Using Genotyping-by-Sequencing Data" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/9/10/3239" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Genotypes are often used to assign parentage in agricultural and ecological settings. Sequencing can be used to obtain genotypes but does not provide unambiguous genotype calls, especially when sequencing depth is low in order to reduce costs. In that case, standard parentage analysis methods no longer apply. A strategy for using low-depth sequencing data for parentage assignment is developed here. It entails the use of relatedness estimates along with a metric termed excess mismatch rate which, for parent-offspring pairs or trios, is the difference between the observed mismatch rate and the rate expected under a model of inheritance and allele reads without error. When more than one putative parent has similar statistics, bootstrapping can provide a measure of the relatedness similarity. Putative parent-offspring trios can be further checked for consistency by comparing the offspring’s estimated inbreeding to half the parent relatedness. Suitable thresholds are required for each metric. These methods were applied to a deer breeding operation consisting of two herds of different breeds. Relatedness estimates were more in line with expectation when the herds were analyzed separately than when combined, although this did not alter which parents were the best matches with each offspring. Parentage results were largely consistent with those based on a microsatellite parentage panel with three discordant parent assignments out of 1561. Two models are investigated to allow the parentage metrics to be calculated with non-random selection of alleles. The tools and strategies given here allow parentage to be assigned from low-depth sequencing data." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10.cover-source.jpg" />
    <title>Exclusion and Genomic Relatedness Methods for Assignment of Parentage Using Genotyping-by-Sequencing Data | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"Qy-vZee6FEdZQY8qOr9b4Dmf3sY8TyClvOLvjSV0ozw","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"494272","apath":"\/ggg\/9\/10\/3239.atom","pisa":"ggg;9\/10\/3239","ac":{"\/ggg\/9\/10\/3239.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/9\/10\/3239.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;9\/10\/3239"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;9\/10\/3239"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/9\/10\/3239":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022KSC2ZmpEM5DAM2o6zW+h+AinpV5ovG\\\\\\\/2Jtf5rNvWvUaIcdWU09F4QOu3Ut0W9eM+NvLbDO+QyxxDdRaneTUCQvCtxcba20PY65dZDKmn7E60RcsgN0dU2sgD1wPYTi++o+C\\\\\\\/rE3nSdpe9FhJzyIJ\\\\\\\/iVHjxx0dzM9P2BBhu3\\\\\\\/HwhVp4X+ga3PtmXqCCpwOcEVv3Aq9+wLVn0zqDUa9idswhDBvZzpo\\\\\\\/+XAbg3Q9Ozoh+W2HtnpSCGncX7Z6ldAlOJA\\\\\\\/GImR9utfwVIhCbpKdRyLVLjv5eniGxSshZb18JTwhB\\\\\\\/JlV75qi53T96LCz81chiFNAkGC24oWu\\\\\\\/5ZCWHsvr7105CIKPkLU5usdtzHwzoJxd23s9Q+mSDmFnb9vCu0sDnHaMMBL9WcMDxJte7kOqANGWcl8cwURrsatMAXg\\\\\\\/8JUTLLC7\\\\\\\/VW4gSYqrIKgEjqG0izY25q7AT5rfs8SCGTI1Y8pg\\\\\\\/WTkX3CyVITMPzvs2VNanBElEoGTs0JS9+7OJG1D+IdZmkT590YeA4NXLstaUkZl8b59K8EYIQppE2f\\\\\\\/hNIFWHqncNjfGTK5h83PNK7\\\\\\\/hL0amPfRSAbWWEzK16hCEYgDWFnNOU6TebUQHbP8qIOhSDKSmD9f7i5YWNk0DE0x0uD5tacBjurkeYU59CLF0M25GEiBnxpPl0cG0lkyxQscxh0GKa6\\\\\\\/2VcTNUNrTOsdfkV6qel\\\\\\\/\\\\\\\/WDhbqyG\\\\\\\/+ZvnS5oIFVMvbxe8sgPhsyKPMuCUp2252oJXf++M5mbfEYmpKmlBZWem6UBASO\\\\\\\/CJD1j2TChBEOAJUUeX16NRKCPoAsSl6260l++NXz3+2wpK2rsS\\\\\\\/6kiEJZzaKCNO9qUEmeQJdn6ToK58ddT9\\\\\\\/DPhA1oRQdqZUXkvCwC4ARLTywJsoMy4Q4WabSaGWgQr2w2hCcEERJ83ivANjIcfuTjadc2PJN+34nmo97WF96BT1EkFGHwv8\\\\\\\/d8vEC+os1ZtoRx5FDbEj\\\\\\\/o9FvtPGz5jvlLXDRN0Z15Keb1BWe5ZS+F2sDJtvL\\\\\\\/dnk8cjmeMG8+yEpU5KGLsrIz\\\\\\\/0oABOta0l6ONL5ALRPZbY59zx5s+di2EmzKKBYC4IYYMeAFMnkGpCfj8Zu9NCAvz2tHDK7jbicWvJrtBCcLjhPxOIxBr8ANIp4RfFJ71hsyXtBDVUkBahPrvHA07MQeeR9W\\\\\\\/fzP9NVrdWFrSYapeWQnQonLhDFajqtsKdhEZ46i56JNfVw2I0ByGhL57s8XO3cVunhMIaU36N+Q3FgoqiQv4yvsWaYPHPS\\\u0022,\\\u0022iv\\\u0022:\\\u0022dldvgrfc1oxrwdD91TVuJQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00228814994ab7aeef9b37b6318fcb17eded9aa5a5668f3f303b809081fe296479ce\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022QzBRcmj9oU7q8BLLbhpfy3AeY7TVeQg7tUoBijVr+tZ0Nk2lgah6lnkc3dwdGjgcaEA3oPsVNpkbEV7SA+qxDuhjegyyactkyFctHz05cy7iikfsOyoPDGG6OCWnHIkMqfJkYs0rhWTV3WQCp72ym6BKopQzF6ixyml3t3vTwxz043EkdTjbd+obnvOgJQog7WAys4Uo8CM1UsEyFLHB4RZt35htiHAcX9JDuV+pHcQxu5s2AN9N\\\\\\\/Dg0XHbvlP14AK3GuusEyumsToy\\\\\\\/fFPiSi46XOdvw4wGCYx\\\\\\\/HEI2g\\\\\\\/ALpTvkFJ4NVKDRQnIR9fbzRlYTK7EZP\\\\\\\/5HcK1ZBye\\\\\\\/6zh5dLDA++GMOmJtI67oO\\\\\\\/XcCNXK0rx\\\\\\\/JQyLa4AzEWgbnsM7y4xbUVKhru3SJnB6ckZXkxz6xlQ1tV6ZMkGrWQEyXGR5joYJrVrcCfPzcEb\\\\\\\/zcqQs6yrm+xNk8v86tXXuqdN8y3oc33L6D8Bbq9FZes0WVcW27aKcDIoNreJBeQmBGSW8YrlpNzdRebjyzZWQ6xIG84QAWd03k8pdWhI9\\\\\\\/IEaTQYwGf02ZvMTBmI87XD2MMi293kBc9VhghcZnh97rv5kDYMvkheBkm\\\\\\\/EmEKqU+QGHP8\\\\\\\/BM9C\\\\\\\/p9jfHTi7TXA\\\\\\\/cfnS30ee7LPqNBgVef2rV+5nKG9cSvB1WSvJpMn\\\\\\\/PDa5N4yLUN8moi+TxoWBerCP7ZYwqSEN\\\\\\\/2NZQtpXdzBjKzKDqiIx2Y3mcGx5RGvdAeQGAetlKiFkZDcr2aRFeZIYmt\\\\\\\/uqWTyLoUrHuJNVJCzrK6xTR8CB58mtzjT7wZX\\\\\\\/N32Ui6kOGlnFHb+h+2ZWBsj98M87wZ2csW+CqaS3ZSKaKYj4+B7Rhd4N7G0fS1UngmcP3uIMwEMqmQKeuOlsvx347Z504OA5\\\\\\\/jvrU\\\\\\\/PnwY6U6iHsBfFaicAaRy\\\\\\\/mtjATyP2\\\\\\\/MLtzknATvo72FRkVogHoQ5t2gJ1h\\\\\\\/hofrdXrDmM3zQm4T5NNopDjPen8u9nbrh6JBlmb1iiZsPGsP5G3KAc\\\\\\\/prjz1SKPwHTKFdDS\\\\\\\/POwUg+mrt+OdDS8l9mh+h4PLLPoG5HJos1Zt6x+A\\\\\\\/VSMYZMN7kxpVTRaBaRjDVqjX7DRV8+41lt1eCrWT\\\\\\\/o4iK0S92iEn6lXnaFC7rVbnVVi\\\\\\\/U4CI0Xf1atCbJBuTMcF892oqLVxeBSdPjQLiL977YTwnqBy7o8UeynQSp6ma4WXtYDIW75XoQSxobak3POw2japyJxUy\\\\\\\/Y89E+7s6HO\\\\\\\/dC3jSG5OXPdblqMmLCqhxym3ioL1FUkUUyQXr3O1ShkTFDNxjqVs2d0dN59UTTlV\\\\\\\/Vs3ey3RT83nPLbjALkR1OXs7hj2IFeMPTUZ7IKzIreaLgdL6hcfVFgc8Art\\\\\\\/vUSl+2wu4GJR+DwmR9KQl7u5fdwz8t0YBXvsnaGTPMTRKzlpfuJSmAsqfvChE4p\\\\\\\/hB1YDV8ZCFQEKXpQnxZPTAIN3YQr4tLnzSipiKBJJagH9AMW9ytc0uo6rPzsrutXC4frGG07k960rQ\\\\\\\/7Asx87snSA9x4WFVIAmrtWFPblD+438Onac2S0LeN4IZupggvDovj7LDzoutfel2Zo1DvlxvDLmKo4NjVfDGftTSUq9s\\\\\\\/osdmtOiYwM9GPUnTKsfY5pyL4Hdh8vTy+PP+GWxeOt1H26UDMGcTgC\\\\\\\/dxsyWWwP\\\\\\\/TrLy1nVqnc2rexorj\\\\\\\/nl6CLtrrz82cCLPeZTbd5ZZOcZEhjmKnBLpS7UultwsQg85tYgvzxoWVYwhk8dtMWkluGwS2F\\\\\\\/QHBQdRU\\\\\\\/aZvzJRenwgDgSD7DHN+fE5Un\\\\\\\/68nJeoHmaFkOgeUzQgas5QmDOD\\\\\\\/F+Y4YGxSC17K5pu7+r+hImh+KnYul8sM932ED2DMF4u+GvqyB8DIeydTbrkHaMnsY8AeGCwbVvOqkJBtNKRIKHqZTlt03oJ7M0B1EZOo6i6zLI6psQV1IIs2NH1oZu73fy0cAkBStO0NEWtCsMMdtpzg3Rt8eN9BrhUF\\\\\\\/UMXbtNo6tPriJ7LLppjb81L6zKYiXVID\\\\\\\/VIdDAj5SYMJLiyJfXvEANVFVz94a8Hu\\\\\\\/w+jU5YHtN8SbVqCInyCknk10ECHOCXZPhIf5MmyIMUbBL6eI57VOG9zJInxvjf+8tLQYSMRye8bazZgcfh5YBzJp5It8bL9JzdEhgBSEn9lgAA1hj+dpMop5rDNQssxKF1+VM\\\\\\\/P8PR7BEj1NR1l+wm4OzOAwmLg5nj14QG+cIu0wWhdfL6PuhjOqOshhHvIFUdznXIxkoTjeDbvrIYLBvtOJGuJXu9TPnOvqthE+v9JAwDULu8C3pgHq5+VW7ufIWZtbNBWuxRbzxIEXzefwUW1MzcolaxPapJCP13VWK35FJ90NB3NgcFzeKk3Jw5Cr0YVNmBowjWRyIcFi0etn+9U+FOCwYBaemVpKr3PLkFOcCygrKNIPtl\\\\\\\/HfYTkO9j4F1B7sk8yGQw6GfEFHkSGl1Sj3Y4SnjncGQ70ZvUXhD5BS\\\\\\\/pq4of+MC87jhpJ7gVWPrqWiGNbxKVi\\\\\\\/rUR3c2\\\\\\\/\\\\\\\/9cKJwGfqNrl5Mi\\\\\\\/\\\\\\\/DCQgGhm408bC0o3KZ6kK7lc91kiKEnMYvv\\\\\\\/0KBOn74K52Q6tf+spX9kUHid7jPR2gaeY6y6zFDIWmyvaODvChp2KGT+\\\\\\\/DawUzrjK6+u43AKhARlaDxY+Kl+HkykVYtnIutTcyrxzO3rYQgpoVlB75wykc89qKCdD5Nh8NUk15uJWJ+WWak5xzXD9MBa+\\\\\\\/acpMi0MvoPRpfIQ4I5XNdP+0PjXQmSV+4EqSvKYPO9a9b9jPjvliO6H8z\\\\\\\/S21GYVYUO7sNCBOYg8Eg0qeuBL8T5wAW9tll0edNAsYl5KFy3h4YW4l7dbdW9gIIy2ApLX5ilT4B6tDr+ynU67HNG2Zdc+uxpn5hUIeZtRMzVwKX3IOH1cd6VMFjtyCAQ+tDWwidXT\\\\\\\/RWWBva5UVsgTI6LfXfREntDO2sLx6laREF354WCC8c4QkbdxuhGydSqdgO4i7rNS+S1P\\\\\\\/Y\\\\\\\/ar1+qbqIgImgkpY1MddTKH8uk4as\\\\\\\/1agYLHP6yI9An\\\\\\\/KLc0F\\\\\\\/RByLY77qMXNdqxNv+vfYk2RWOjTNnvyCxv3h4FmG1cMqGjRQ35nrsmBZMnXdJJ9k\\\\\\\/nNtxFeBE2oFHt2\\\\\\\/Xd15MUSxZRLNyeL1pLl1LzCF4ZL+v2QSf0TVPSdjuJltoaqbFKnmT0t4FCXiyqhYntMcasgiLWyLrqEjKVLTAzoN9gMbT4wDVJ1xSJTXoomVDSrDjnTqv\\\\\\\/Cq+v3g6AIKpZmYpnXNFO7uDfdZ8h2KVlDHtCnRAVb0YV7OoDAsP\\\\\\\/4i+02rzGUkN3CjxO\\\\\\\/vXyMHDE4uV9CZbw4bCXkJg9EKqvUwC1yYTLKc6pBCrJ3cPhfH73ewPq49SL1341KWCgF02qnksgJ8wZ1Qrtg9XG8ywJO1bzyKY7qH0T6zgj\\\\\\\/MHgMU27+4qT7omY73Tb8FAGjS2ap2pCuLzAURhB6BvgVf+4MFrtyha3zeAOhghYEl0+sPzFn1pdkldNxSnOzJYTFRemnJo9SGqDu+gDhsYJleJpkHswMxPJFoX7dcHzTzduqbfoSsow\\\\\\\/MlPQ4AG+dLVJ7HPgXbu2tpOqJaYdU9a8fRRre5jsQUbpVQZ2srIvWI6vwDZokBK5RoyYff6Pzmcr4TjnlZvyzSCqcgj47yXO\\\\\\\/b0v2NVN38+tKpT1TnFl0Jz8Q0pFwROJ7584DVP7bE4emcECVrh1d59oXPu49wx1R+yWzj2q0BcO3PL85nIFJv77IOasUEqosEJlNl0OysTX7lXvCVNCcHhWvQHw2ZwliUJR6jCH9lnKkNDr2aWg3okZG96a74obk9Pg1Gep9IelaZC16uu0lZhSEVcto1MC\\\\\\\/mnlIMtx2d3MdXqrln2fuoJHm9eXeXoGVBOR2i\\\\\\\/cTCd6vdr1td0FhSKrDlyvXTBveIKBYxRUYOX9r3JGc5yWxpbPy9SyZJYiCgXDaFeKzAagVtg6L87JO8deRRc1R2QxBk\\\\\\\/j5smOvRuaYTNRxsbbzfjB8qjPXIleeyPgAB09as3WqiZYACedSt5jd8NgxX\\\\\\\/NaOML\\\\\\\/DyQWqyR\\\\\\\/WzUmKEquNnKy8XYVBe8kj9iVx5hybld9dJons9x56oiG2ncPzgWu5NDf\\\\\\\/\\\\\\\/Jc9EX0IwoK3nAMU1c08w2Mj33wFV6IUDHMi+zJJlcnib\\\\\\\/nW6I0R0MeDqD4\\\\\\\/JCxBFGNYnjeXJz647XOANPGIRcbmhZl2LJdUw5H69Hp\\\\\\\/o0hXRuQ5YXwIvyeC8V1Fz7QH2ebItiFXAQjUa\\\\\\\/dfsxaKo4EjeGEil1n3HW95HJB6+qNwplDuNU43oEyDj3buy5Uvqn9NEwVAcvVQXXlcg+gcVCrDRP+Kc1xxQqyRvTKwScU2KBFElLUFhK96oyd+jBBQp8LwLT5n3TYtDTNyzvGq83R5eRRti1vQDFCMT3cMD+rjyDRcegpnIrhs0\\\\\\\/V0cSQDgIsbwv9VDiytosQG0o5L1xLHn0s\\\\\\\/Dlz0AGwxDbbh1OySHJTGl\\\\\\\/7t+ox5Bo9pJlKRsOTGi9tW1\\\\\\\/Jn9i2QRUc2+b1Y86ePNQ5z7RUR30zieH7P6ebWsOniH7oG30Hf10CSKCjOyAwNOsLqfbwWMXUQHXbJa4cVz5d6nIP\\\\\\\/IoSFgFPmLLoo1OcVMa+LUGi45dXgjBFdfdu\\\\\\\/Ag0i0W1FclNmuvBwdvWFgAWHHGYKPvGWycBzMDTRkscUuQ9JIJjB62FEXoa4qkHoNF4enbvzGfl5N+Xkqw3QaVT6DsD42yiVh7bm6hh4LUQw+iZ5UCxiY+YM1+FKwaGnNniKcCOVTHC0iEfg3KFvvN8tw1R8ZbLYp6JHeMqCuK4fDvXgQRZX4spSatg4ZpmxJ+snKxQ+yZ8GMqD3gkpJQYpom4lhysRiFIq7uJ7BINAM6oWcmRxResHEGXq8xnwSzhTFKxprQBH9+KW2b8Ob75ASP3OtRYinhP3vuf872dXPKtLRKJZFApPhdsMZ27Nb3Lksu78lfUF87LugWL1bw4D2ah+sn4klMMj1KU0zxuBUcyas45QNCpI35YYMT9+5vFPm6b3j2bEKIE7sjMCrSZ8WuYHp5G\\\\\\\/cVVgivla8gVv1KcOydbjq6WYR3T\\\\\\\/1GDQ9hIXrB2U+nEbKMY4XJkvehQPXsx3fG4uF+jC69c9EfpkT9cNnc4LS6lQz3vJeMgxFQX5UOmf7epz1OH7h2sSI19tqGNH5iARpW8BOREl9+olsVrIfGsDFMjR0mXKHxwMaeR8+DU+aYbCHnWs\\\\\\\/HsEugvssBzV6Xv0xs+kRb7RLNCCFhiyArz6qTFO\\\\\\\/0AJ1WazXh1TlNx8NJKTEOOU5VVStp9oY7Cjx0TR\\\\\\\/2\\\\\\\/l5D9dHAfGmPq4Jdnpcj4odKqkposbfw\\\\\\\/z7Lz49KfEt8fhBm+XGjRKlsxRxkXnj6wu5Y8sTTpXpHJn3Onuk9CGry5d+L\\\\\\\/86gRjYg2pPQzxhT0+o7fDGnrXTUAqjelUvHcFcPGSmxAjQmuzDUBD4wZLPRM\\\\\\\/AScv8w7WjQN4OZWjm0ulrV3SGgz5gFRA05t010qv0aUZzEvv5p6ptPE9bokAGeCo4m+u1FELsRJ6jpcIlrdBqIz+bQIIpex84C\\\\\\\/tVWbr0bTWrjAxwF2AMfkPl8D4BbdKkVLwPopTneKetkHL6oIERaA2R0+URb9MTzYfYC43cdefsEBUNn6jtPrPSD5GISWUh\\\\\\\/ELesbRVM+9IFcdKQsGyN84kNpT4xjyJYjJuuo2iS5KQ+vWp\\\\\\\/D7T7sflLGTghn3Mm5rE09P6SOG07kLzT\\\\\\\/O5qGihAFmW3Cx5+4AiCchvq+WKtspMoIwMQ8cZgMNd6b0EIHIRSyX2cp4LPflNG8kAzbBy2msirwnymeSg18hECqz7fm58Q8bp21I+ulltsHR4yESZJmqqGDADAPRW+zK7jD4LQVeonRikNEt\\\\\\\/cv3zPuSzDIenOR3NUmeBRJy\\\\\\\/fSPX9BiEDiRMF5q9hDWUcqlGZ+pvfmVvT1MgOchXrdp2vFURedFdhyBA0l6jCuTDVjdUKcyhOebwpKx5q62+XcDmF9bKNMvgKXbpA4MdTZmeXYkMFM5Dx\\\\\\\/EBgh+jnAqst99VJgvlgPMtVWODZ75fcWTx6U33odS02yf7LdfXKUqndCyjrYCddCtK66i4k66Dz4QZA0tuz7m\\\\\\\/PZFyxciz6illm2scZumRXXxkpCedM894JxhZB8qQm1tHBuL9PArzwtnC6awFxdMdNXbEA9H37SqLtoYRFZ1XpV7aRv75pZzbvnUj7weM0jFQDx4j9V++UyW7cCdAv6sj6bSO1Cxt5FfL3+abZsMq48faG0eAZGmFgKDe5jZKJzFTr1h8B2fHwQMRQpjCt6+rm1sJZmKZ8dFWAeZm4i0NSUM+ZvqPuOZORmx+oZmsN1P9iRpSQEARsRZrdHIaj09ClgHZHJIDrZA2JVIAuien7WW7pi3O2wJPeJ4A6JO\\\\\\\/D5C0JiSrh0GHkZVB7CHFMiRQbX8LSlq7YuCpE\\\\\\\/m2eSlaKOXfYhQo4YckYMaSkBMoepbwqHTuTZg9t0YZ8XZwWi2aou+tK30hbIzX1At2tgw1THkjARQ1ad\\\\\\\/uX0H\\\\\\\/+ZB\\\\\\\/DIOwPn3Ei9y0vFWcSa5dT5knPWjjX5cddlF5rPLkwAmbPEJ7W08iX+K5f5+mafDvl7hIdXehyqSwwg3\\\\\\\/N\\\\\\\/KJC761t+bHGKR4N088md9U1tKBpXmtCo6j26yCFsNpgKhzzASty5ics6cRucdMWHmWROc9dvjViIj3a6vMeANSagtZuQRSH8cOwsTudE+Y77zcFHCImc99KscjdP\\\\\\\/rgZd+\\\\\\\/WQ4STb8it7OopCoPHe2uBERQ\\\\\\\/IUYt3eo74vr7nperCTTInVEqacH+597i0346vzMUMQ5vbmWRa\\\\\\\/ynbu+5kVUI0SZRQHO77SQSu4L9cdp7hqoT17EqpzDwH9ilVVdQM8aGvRNOG3sSDUXZaVi1Sokd+amd9NNoivR6SYpi3g74gjGK6raq99HRspYal\\\\\\\/VLNT9TNLVBbhE12\\\\\\\/HYxTmHrcesRtvzl76MCd9U+onf2MLWdOFpu4GRRGypx8zhur4I235ZDbmIJwXWhAjejD6udWOX2VWxXoOmYWki3EV0Gi+Sc\\\\\\\/L7CKaXaXItJZAbEM+mkZhvPgzPeAsXnoRQ4yn5uSCrz3WzVZ06QLOgAB9owma3ePUg9HIU\\\\\\\/26JMWJ+J4yTXtWshdBfo7ZGlQK4i\\\\\\\/THf+5o8yM1OVPTF\\\\\\\/LOTFRHlgOxKgWs+yrYSwBHeTf5NZ+xPhpky+OJ2UwHAlvXkSBB6ndTMbVHa5jt+McKQG6mW21pKa\\\\\\\/PPmohooYG2JSZCpBTxy+wmknRHS\\\\\\\/YAUby1Gm8gp2ByRmEig2d0VtB5LAU4hTNxPcwX+nOkIWsvOdNAaCOVOFiF84Wk8RDa+gw7ZdehRKJ\\\\\\\/TlGY0f2TBwfKW+zmaOqKVrHpfPSoh7UoLRJOgGRLpbMQw9wBkfEO3Vm67udbboLndKHBgkrru7Y3\\\\\\\/t54teOhLqtBvPFt\\\\\\\/OTBB8CCU0OFYSOVHi0PiceRBMREsgZdPiCVhtqC0mhU1jodv95Zis26e1b0lRhfBSp2nPGCuortxfff+V1YbbUcr8CPCbd1JsW5G7TuDIwCjUNDWFMaAUMePdZ2\\\\\\\/\\\\\\\/1WM7\\\\\\\/7pf\\\\\\\/o93z0w3dV1eo8j1zzdWe4fBkIJXYCMfMHH1XPvyR0gCunKK8opsdI+YghbzDE3WXDshxW1FvAUK0PSbpoJY7gAgD3gR7nvw+wuuzbJyq+rxMxwcPSL6CZP\\\\\\\/xwpm\\\\\\\/O\\\\\\\/CvQ0La6trK33oeGRw5+rCd1kAL4qwGFKwE7toBejN4nJ4KUr3+g9J8HuJcFlFJClfBjQ3YavVJs5+2Rq+IH\\\\\\\/dAOaGTnXsCB4o6McyRt\\\\\\\/dShr4zae9LkA2mcbLsz2tG6bd1BlYn9BKJqhG6\\\\\\\/UANlxebO69rOdmVUWzdkGUMy9hhtf0c\\\\\\\/Fx3kdtiHzMWAM786Vc+OWcJc2fb8z3enCuihPyEQ4cQkA8AmI+FTen1GPv61e4EC25b0Ln\\\\\\\/mUbYjChoxTK7uLpO1gYq5vRr3JSUuY1CXtRj8XF5cE\\\\\\\/VC4+OlbDKJMLIked6ibj7tQW\\\\\\\/Q2HJnh7dx8sHXpeSbxn\\\\\\\/V2hscdAvhHHn2wd8UBUFtJHQ31fWsWHUx6ti88Ap5VrJPRUd01uwpNyUW76pJLooN6SzDmxPk8pbcW\\\\\\\/cPHlC0yMpuWOX\\\\\\\/l3lPg3iHRoLooD3mpq\\\\\\\/5VNmoWqo349U+SlaIw+O1GqQnu2gYa0gMaUGqUsl6EQCyAhqbVFVG5tncJ\\\\\\\/t1h9fAi26KBWDoGfIBvG5HWQOwQAX+Uim6kKNLyi\\\\\\\/hda\\\\\\\/mLm\\\\\\\/PYn\\\\\\\/ovRWCqVlQFbia7KMwLuDALpubTQLUk8rU3E9pX2\\\\\\\/tul7hISHq62yNgh4XwrWSA0y5MNv2Z8dzsZX3\\\\\\\/9BmpkWuYiOyWgtQQc\\\\\\\/8XiF4HSR8mGwpbeR8OE1ZI5tLbBvqCrSq35fHRtXGtypzMSvIK8VvWqrwdZu0altDHbETxp8FwqUYAKrDG0ClDCawGVIt57lOjzNAbCzf2Ka4uZKtuc1h\\\\\\\/rxS6V49YAX1CVBwJmlfIRRSSufEtSXDrwfXbyhG5Ydd1aMDxtnxQQg62xn2UC+Pe0ugsZxHWxZmdsignun+9M0JRWJI46M7\\\\\\\/H6wqZjYfJqLmqWUZV9KVk\\\\\\\/4\\\\\\\/ZmMcblIN+3gCzw5MrzyG2HRcf\\\\\\\/93NJYrAJpZ31+LZuIBEbNb+I+38oWJJxipcuxFQvt6EPmPT6QsQCqDMGXwBEjUhOHfg31QvJQS5WgFme+uNrux6jeDC8UmdYTJx2N\\\\\\\/gCN5Vt5nPC+rCUfVUsC6zPEnHDxuMdBBj9njkx\\\\\\\/Lthgfky91R2iC3a7PpPKdmoxXx+tIQr9Ote5fx0IVLb90KPrJWS8xB2UMOyRt9eucQSiJ2NIn0eX\\\\\\\/uAqBg0NTuyymvVcV\\\\\\\/FYskxYX2xKDlBzMPjkmmohEH60MYZiMQ3iZA9zmo7EY9fNM\\\\\\\/\\\\\\\/a9h2mxhMhwKJA7C\\\\\\\/sE\\\\\\\/7AkXpkSZ2rEpyQDhhzVg0YFWM2gJAHCLQaKsmh33wWQNBfK2SoIiLD3IDkflkS6ad1mBJzyQDdYwMJz5c8gIp4of6eR0cmT3de6dh0+KMUlwBBdaSqT8MA7kM1sL\\\\\\\/6vkaSK7\\\\\\\/cc9AXmZSkMn0hXdMNN06yBiW1Pes9hemlUsO6wYlOA7x5gU90SqGxJ9uRS0asFRaB4tNDYTzWMQDFL6JvLgGwBZotI5uKJQLZBYFQUxUBj6k016IUu0jp8v3ydY2ORRnXVwdb6woTkwWznAu7niU+OBg8OT2aXQjRQRKR4uWRQaBvg\\\\\\\/Qs3uYJz83hsYlwekgTgFlcLn1p4thDcsv5jgAiqrK7tLeNVTuKvXmkh6+s1orANwR3hjW6+Q2RsngK2YWmusU71zP5niMW7R74Lm\\\\\\\/eKesXxoHjqwFHwuli6xnNKyf0dLtz0oq1kHAL6slopQuqYLj+EuDfRXepBwePHRN4VOcPvNMNHn9ncmd02Q++BkdR8afPSXNDpmgrgYhyrLFRI1oSphIt3t0vYYzZFrDr72Vmzb\\\\\\\/8vlGN6eL1qKyx5I7l\\\\\\\/Ck8XA4rAXg7ZhwCMxc4qdBnyrPJefrjIdna7qOVXSqelB74WG+EP0eOQYZxPvEf0CLvFdl0xBEfWlvlrUgDcw17ngsEoR2YSfUWrm1ImbC4BJRBSwHIbWXRzvcpt\\\\\\\/FlhnCGpdh2xVnORmfk8sPqxqjzt7XGe168YVQHPmwV+xeGZfWENkoFdOxjhv54\\\\\\\/hL1qhNh6t+Y8g++oQd755hyfOPr5JqvpehfAw\\\\\\\/BaDdFwjd5VzUpb0n7dN3aJhI10VKPAbt6M+Gz\\\\\\\/FNJvVaQx7oUWZpjRr56j6rx9l4ihZ\\\\\\\/qrRKnyMALtTuOEMQ6TGcQrctze8qNgEpE9bA2K9O2Dl05MS7\\\\\\\/\\\\\\\/x+TtNpn\\\\\\\/8Ww9HkKUEkrehRrdBnUd4tzjU+TR8U04W\\\\\\\/eH0wgkT++mW8CTQ3EUs0MlL7e\\\\\\\/gJ\\\\\\\/5X0L9IVXMRhY2IXK\\\\\\\/cSmhBUoN8i7lT1mdD8RomSbCClj4i4xxO\\\\\\\/sKRyglaAQlC5dKFnZVGJjRFpycBjw0+20RD0qoiCr48jCc02PafIbuvRaGfyJQzf9tLuWb9C9i3S2GejBgfUSfyBswYbpdq4MScPU1vxCihnCsD4N+mmj48Agg7go0QrGetIeuWM9H6Fp0ie80GMtbdExNvREEPaXUPxzuS68\\\\\\\/NDYpdsLuqAGdgOP1mkRLG6yoFO9L8z5LfoDlPDnjre\\\\\\\/g8CfWGz8WnXa+odX5gIG\\\\\\\/rwg7yGV9A2FGZK5yldIqPGczVAcjdj\\\\\\\/+YI2B8QWAA5\\\\\\\/VmWQ042k74P+g1IP7zR5ZLHLlmmwYJ84DRFx6sNajH7Ad+D6kyWnrR68h1DxtHXwjxRGcKi3w6QUktjc8YtLaDI4Zu5hr5x+FNhyL7swOH\\\\\\\/UbWsKWdCZoUBQVPZyOqW1qbgaQOwEydWWU\\\\\\\/BRHb2+yM7BM2PHtV0ZN+BRrhCSUNaV\\\\\\\/Al9Yuw9H0TrhrfbVKkKL8kQXhKK1ME3\\\\\\\/RJ4X3dCnInvU1JcUVZDvky+JmDVUArfELuoFrW1f14sg5I81+mwA27Yd0Pr\\\\\\\/wMYrM7px0hJPlgu97LebEZHkcyeNAK3DsCIAigLDIVplwTb2EoxZmlRYE8aGtirvFOxZ5qE1Xr+4fR1J5K5Opatt5\\\\\\\/TGyHoACSGn9uLE46h\\\\\\\/wZ3GDuWWcBE7xEsJ8We6FBIPYpvN5ejfpl1mbCQ8hIjtYJsr4BTz6ICD6mJV\\\\\\\/8cjCLtzrAthEkbssiPAR0hDx1w20YldwoppJwfyM5TrWWyOSG11LKaI6HRjtx9btfPc2Mg3odtZy1myaW\\\\\\\/zNFZZNZ2lN2sP+SCKctqWa\\\\\\\/WGpXfRU9ypaYGusZZDgbAWJllwKmRIFw8uVDQhTUYv249x+MXPDIpaIJYZqlb5\\\\\\\/dgvlAyZ0\\\\\\\/SeeHLefPH2dErD\\\\\\\/Hs0wYX2Cj\\\\\\\/qwZ+PvHIMGCWJgwWAqvnGhEb\\\\\\\/GYLLYYbK5m4+PM\\\\\\\/ZA1+UB0C9bAR8aiKgX7bMwRqWjnTk3M3THcJXJSRXvUoP04TWiMeyHdUmpx1MxdepEBnvE54bXqGfTEs9HyvjRBzhkilnGt4GkWZMoJj89jewk5ouDTM\\\\\\\/Mn0oHXBL8FTK2FJ0fOzPNzfVKrWgTEQ8Ry0j0Mv6jurDP\\\\\\\/oJBJaLOEOpPzdc5lSPZmhdV7Q\\\\\\\/G+Mq94fy3t+q8JOQQu6k7YxV5J983KaQmODNVfals\\\\\\\/0+9+oPA0TCTLWPlpMvz1kfdWCL+mAwITa2FoWKIz9SmrP+o\\\\\\\/Qs6YddiiEJa5iDeqjBydQsiLSypvet\\\\\\\/YWLMOHY4HdgNMOzKzjmuqwVO3jaoKmDa5o6K\\\\\\\/CdPEEnIcwIQfbG9IdYXVIwTylCstBxFMvzIG03vwPyOQoKl63cud6A0G7N140eXhcNzi3kg\\\\\\\/fdn8TnwAeGRzA0UgSL9p6zwRqkL734BnzD75BNPBLnVNuTWJv8FhKGAvl28PQIYtViF126FGY1Kp2R\\\\\\\/ZdKYrr2r9ugtoM5nasV0cM7V7uVgAKFATy331h5QdEWZtqiXjY1\\\\\\\/Pa\\\\\\\/5yx3nvGiiMddQPjSLNVpagzx\\\\\\\/emCTaf6kX24dZeicGXh5oypo011b46N00idemjN0UGSfGpfw6i4RUfn37adS+NDaMfr5g8a4aQFBJMevnB5Gb5W13vh6DiRGhr6iSJJ02jwdxJJ47OiwRof\\\\\\\/KHC1drKem+R2gRf1bLoM69LP\\\\\\\/xPbnn8z53vIweR8FIPN+XGHlnI349qEHXNbfpJGKQh+tczv7L6h9lrJW4I2NWMuRVw3buKNsJVUOnrV9IR\\\\\\\/\\\\\\\/U8KeU+sUBEV9PKba\\\\\\\/d168LeLcfdRHntxPS1d07xeh0zx3DrnvR17ZoVkv01YoPLnoKbCPeYfQHPkSrmqr3WPk0coMjEvyPDuWaWhzGXrP+U26W9JRCcNnmdSEbnSJilZJ2zEuREKPp\\\\\\\/6xRBaricpkIiVyJ8OwPLJpiX+W\\\\\\\/GqCe1YY9MXysCBa3aJCtQ0em3hXjU809AvyLXFp+W5F87ca7+liQRCbR+VImIAYYe28AmJeL+LXM3pTYgJl5ezny4b\\\\\\\/rkYgW6GBMNBgrfd+vKlwwSuWrsf34W4z7KDmwEVkTgkShHSrEK1nT60TfW\\\\\\\/kHAWXgY6Li\\\\\\\/8itzSehAQnItjMUWp6dWQ2Jyi87QWYfHFizGpn8yOD53TGvm\\\\\\\/DfuDsBCt4pzUyqCVSdJiHcAUK6Gpc0EjOACKS+lfkWotNIaRjv\\\\\\\/aYCsoY9Rm5KsQm7uD41dtN\\\\\\\/fs6dn\\\\\\\/bdSQRa55aZAa2aoCiz4\\\\\\\/V4tnfvdqzPs1iVqNIy8XicDsYOMKRswdSMvFW5euwMELSE8KzDkMObowLol+lZiiSyVtDP1JBdXVAEzJIMKGXmTHB\\\\\\\/hQwdsV\\\\\\\/UEEz7ozCeCNtH\\\\\\\/25APgN23BmSFez26XBwCSdTjqAEb5pj6UiwiL6d2hqOwXnBvy5liFR5srkMZL20rBkwbUrMYX0buZsz43cD9IlNjPLPD11k8CWZAVSFfQi5Fe+1mZATeuCJt8L9pSvKf91tuvuQScP9nwqlvSL8WOAjWEnA2NwqSM22FwFNHS+LgHqv\\\\\\\/6XlK0othHmpQZAi8x60xW9c0vEIM+TznJKDCCQTLx0bMIJpqTt1CLBLgWvOQuyyJzKGQ+FEN4oGfraahgI0BWhhVDmJbk8Di6BPA\\\\\\\/eZlMu\\\\\\\/wdfYWwuAWhL0ZmhcnlYwj9+8UsYhqcOusIshEhp8eJ9WRGR8CBQFwyRYRThpMLv6o5sQw0w1vupSuiP7p0GT4NoTDr3sBpoO5qAjSSC5JKlXwHKc30bJ4aCJFQy2WYqyfAAEraR8KyL\\\\\\\/XmMc+5bO1xxsNfj89L7CpMbyCEimHav+HzYZjAC27TbcvUqa9e8jQHpDgJR1MtylBAsN+zn4zij7wB1\\\\\\\/H8CVpXrpLFO6Tk8SU49E4y5UbT3kttOmggz6PtDxmmqAcA9uVN5q8vSlt3xK3N0c4pC47em+nlVkysaBrC8EpJFCoonAZ7cFWufcY3DAlfyqI\\\\\\\/JMK\\\\\\\/HWv8L1yTbpr7LLR8oxnmX+Km09aeMicF2pzzrKqX1qGwh7G0HpxQj6gY59YaTGWXAwZGDcQXzCDN5ZlvArPMy9iHzL3rDRUF6vSNsab7ud\\\\\\\/gDOvvm42Ol3yx7dwK0t\\\\\\\/kxl0OvrAL4\\\\\\\/2r0ZGv7waeSGzuUcXdFRiWoVkBEp5Q+IrucDzluSdnzZGMJCI867d9TsscLEls7H5R5rgT5PYEz\\\\\\\/t6MGu\\\\\\\/rxvARomkJEPnROVwnd3ht9DQgzY2Mk75sdC4JmBAOx7YkkQEzmBxs5XdqYXX4V2FldekLz36R6LKdxlOnCnKd+6gOHXCJaKn28dICj2W3mLj8\\\\\\\/FPprxb7oqLkw8WSjp5+Wy7zaDKaiIm6hAQ5pN\\\\\\\/f3nhaSihv\\\\\\\/8xumKW9dT2a\\\\\\\/6RbVz85oLzoF9NopXPs7dwUYJtDCoRZY+c\\\\\\\/y7fU1i1xDGbnfkh4kul8I9PesXQjCLOen7vM4kPrU7qmsHaIMadyjKqk7U\\\\\\\/QnhfA9MH\\\\\\\/qWYThXtK7RTjP3jsdrH4gLv5gwhzpyH4UUl0vG46TnPiiUSUnQDf\\\\\\\/ogQNGMQQ2TkRhnWxqJ0RqJLqt6jcE51Cy0wQPTlC++RjtEtlfGx6kSF+0T5hsgatxvJNHPQv3Q4Rz\\\\\\\/CZYibKPD+\\\\\\\/NTcrWy9CFpMsueIK5kpLBzVaW+y0t60mdlxxYPQIUnj57jMBlB3BRitcjQL\\\\\\\/+172PvjTbzXx1ODyXmPGk3l5JRixgI7wSP\\\\\\\/YJP0fEq2aFHakk05syx9+xzYQIAx5dIyffp5zD3UxBUQ7d+4jf52KYTgi\\\\\\\/9K26mW8as2SMsKwtae6ZVNut8zL8aLpo9Z98mf4OpOHor\\\\\\\/rp0phwlhhPMl01unjt+LzHPFvOcvDCKua++TG0EOyNnHTYL\\\\\\\/knRmZwAPSBRa5Te8ONZDsoXa+vbHQ9COtL0QVB3YaOmjBYyc4OcYa5VVIkFmQjk\\\\\\\/jtgByFZY7B8M\\\\\\\/ejAqxQerh3mkRa7dzRL2mz\\\\\\\/jJFk5oG+Oe6ySAD3WNW5i5nbVec7arKswzQFiGANxcTZ8Rke2OQnb\\\\\\\/MDkbARMX4\\\\\\\/u1+jwkfq5LvQ8ISusvIdhuPOyRn\\\\\\\/tR\\\\\\\/Kp+kQA2FccPoG40q1bDzeFycg7n\\\\\\\/qaUeaY1iO2q23ZS0PrUX8ame15TRKKBaBcKbWHqvi9hC2FqXugP6CuKgAeLrWa0ZiXWKXFI2fmAOwge0shVBhKF6XiEOni6ORxd1vDliNlkZMVqH38L0CY5arKO\\\\\\\/+jGeoBYK514NCFKByF6C3a18GD+Nr4iAlU1QZX8EjWeQge9KAgtsH8JJg7bq6deUBqzRMA5lS8MfIN2AsACo6lH9V1TOFaSsmeqSbS8Hl7P3APL8oziijFeZIhgY+ZXzWaBs+WykTK6Revr2WKNeMfns05jKbPUzbSb4wQZV1298ulny1ndQywvDgjbJiRvY6STAo1OJOoXM6DD17rsIwUwVcLHe4lRDyPYCKB\\\\\\\/3m8HX8Fghr8z3\\\\\\\/J5VjObrDxiqpiuOgcR2dFW8G\\\\\\\/ucYMeYWeEsMDa99zdI1r0ZRi1dqjpd9VA4HeQ1nWwYc1Oyc3Om7NX3EwbSiS3MmvUbSop3K9kTkHiQNDAQfWy17S1dm04S+Acd9448PwXDZ8VmrtEm\\\\\\\/DASSs36cz3v9Ox4mtpqACc6XYalSdiDam7MmUiNtXO2lxSOarY0pTfIYgaDE0uhZo3jZa\\\\\\\/A2q9Ee21QL2kNDyXgAc7kXb+pmzvY6uTzSPVkKukkz9fmlmDmZpmV2SSHNDl9i9cRngwIvs8ra2xvOoxvo8Gg6KZusVjq2POYBzJbPOJlR0HT5yLtzsr4Hn8RUWaE2X5WcTuKNsH\\\\\\\/PKDZr27I\\\\\\\/ua787G1+CEQxGAZisk3ufzYgRgAZAqWSixbQtxuTL\\\\\\\/a07g7V0T9T1ZW4hJpLmdKT5ON4DLGKF9dSZuNMb\\\\\\\/39al7WkhpZYlgz6+IWEsSsZoI\\\\\\\/e4DmHhRTHBkaQffmt\\\\\\\/vP4RkV41TccydfHX8PkpwFhLiQlJPUboxh+WHUt7i9YTV+3QQMR9OAhzgWLjZGSdMKAZK4JnDkmARXzq+5Et5KWj+DNFdF1\\\\\\\/S0s+dRX0rB7a7zEKMBbmet7El4MiiiDJmOB3pIxhjr5pJ7P3gfAfUzrkyQpMPMLMZxTcgHsVzQU8QrX48J6XJithfYcJo1jOJcCA2ExN1U7E5vxgOr84gn6321tcibAcv8q19zDWQ1Ieol4slTEKWlZoca1MVYNTWAtU+YdTm5vmffL+s5DE+dCMdGNHsHV4UMUGppoixSvzBgMJgRdDmq0rW4HrYuqqIRRjektQY1EQI2oHU6Ob4lMYslGJ9pg56WQOAabBAPp266Q\\\\\\\/X0zXt2QH4pumfbuvLqJPAmeNie01Z6aOz\\\\\\\/E2e50R9ExkScq2L0bzB5oh3h+VoXOQ+F2xnGalQ\\\\\\\/eQJWUJBCHP6SjM4xDSmePdEgH7+5OeP82fNcrO4V6VklGEbl5jT7RKvhhDPcRmaF\\\\\\\/yhdqz0Incp9FKlyYZMz0I7TGG++qId0iqFjfhws\\\\\\\/XQf1qT2ekCXFboXvejIA7ZNrQHmnRtKpB7iyjEEVknZK9X6q2QdkJePcW1woed9iYmD61IsC8uwNwWBz\\\\\\\/bbXPihZ65eKHafzypVoOjIQvIN3j\\\\\\\/KmB52G3PfxOKXhSKxgn9ZF33rDG0TNklm7WvuMcpfecdi1vcMQKRrGSy+CL4RangFm5nmq\\\\\\\/KzMY5arME\\\\\\\/9Fmk\\\\\\\/2bWCYJkUTc4jSPngsfxHQOBw\\\\\\\/Rh2dAK7UmIyx27w+ulZtCIQayAHzmadWcdIuxO9nSLVJtO0CDLLVdz0i838RvTy86gdeTXKL7bwp4Q31KOft9cY0pYz1FvZKpMJBuMQ4oBJdEungOLalhh4WJuJMm7yZQPQdDrgFce\\\\\\\/7zKy3g9hMxm2s4k8TZK\\\\\\\/PUB9hrUe186LT\\\\\\\/TDIX9KJsuU\\\\\\\/brVzkuTrMxH+cfiNBJQfak7MAnmfNFy4Ped0UiBOVswKWHhosr14KeItFtiT6E0IyaBcYNJ3WIwDuPXkfhBJUVnEpr3W2lbkA3NNsfwCamcdgqEf9EO6C2XEQ1CLDXOXTqLM90IkEZvGX4CphkQ2tsEZ9zBctJhufCOofDnYIwV4AGdIMHHSRLnVtaid809D9bXJlKQSYzJDRRUBvNzZDlfpYm6ovXL9oP\\\\\\\/DSL3YYzFso4VDtVm8U4+yGPupmFqEpsT+2DjgPXqFVY4L55kAkOK167diNixNxhL26TmJs5TkeXcaOSCfDUZJmzGHaodqrZiRsyiPsmEfSudD2qfHUZd4BqOKZOu10IvPcLZnW+lEDmFFGNF3sayVfwFdvts6OJQ3eAc3b\\\\\\\/hGYTt+EdRCRLmImDrgggmAKL2+KZQC7K3XzEKp4EKZX5GwG4KV794XJcEg+cgaQAoI+81VXAA4A6ha2IuMbuXwNYOyRMPLqpwgTPKu8ZPdJYt\\\\\\\/ALASdmsb9hhGgLaBM\\\\\\\/8avbUgXgLbxwKdog9x1Lbfl0c\\\\\\\/kVb1o\\\\\\\/jKvxptuEsN0mSiIyLX+a0BRrQhJE8VoaWeuRv96jPUq9kjcM0rX2DiFvwNfq49ORWbAWKrhya2q1aU9JcF4FgfSM\\\\\\\/sJzTq+u\\\\\\\/EpZoTlzSyzWj1yrL8K\\\\\\\/p45vmhz4GGWvr8Dh03bcw352dQyoAFv6RAPoT7jgeOCcbf4ZDTI++oJzcqTqq4E8ld4VE5hHchmDOa6VP+e5mmsmI2ysW7vws8N0IL6OKFxn7Hye45dW4w1Yuk4nELDgspYjUbSVLKCmxIMRKpds4N4rXaNQDP9DbBrg9QxRKZPuyMaSijawA4uGjyW7kWkvGSllE0dKAJwOCzNVW\\\\\\\/x1wjb1K2Nu27v7vzfk9Sd2PlPQlwxFomg0BMW6aY+Lp\\\\\\\/AuBigOICylC2dp8nJeB7sEgoPRn\\\\\\\/xL0jTXaNiZJB1htP3UEnIeOIBKXCp0yFUVzZOLUvLC8sAnq2kVdsTsoNsFvnE3QzEUkQRtYslM0mSlUTYWCR2cGJg+BtSD3IyT0vdWEiKZMPVzy05WuzKwSyjIqtrLeffLSDLePEQYVtr9vB7unyRl9B2+k1SKjUyF9NPfwfDNGfMSEV0E1K9v77mVtTBwWtRWJR2sGtUMJQAe1X1YVb4hEuK3rnxEWNn9wNvOYpsNiPdc4SFWXKTlt1mDD+KmN1Wp2j5dwrMrYEuidW\\\\\\\/VW\\\\\\\/UCvRATIfRug1xrueKRLLrLAR5s38JiIq4Aaoj0sclKMswkNrcyudIjiHWNz4alzqP3q8MvnxOsN1yGM4Q+yQuYeTJEFpd2Ypb119Ewny8dDPCrxPcyivkUDRD\\\\\\\/B1bT\\\\\\\/rhJ6RZgZVnLTTSETkxyaZgnNvrHx7gh1sXe7RK+PA6a33u0060OTjMQAXdaiu9FSKNe\\\\\\\/2esX1Wz5KIbY2cSjzv826UbTBAhzNQ\\\\\\\/sw6Plevuve5RfyC5doLto1Cckddo7tUFkgnOSXF1zp9goT3k1vTDvjIUVa\\\\\\\/6F6gOzyyEBsmgPDTL7gkmrMVEbkz8EcOGtStuSbPSkykfSAIZEzqCFdxt83HtCSZiPzTkHq\\\\\\\/JQX1mm3\\\\\\\/wNKl8QHP9rwY9K4HlDOGhAduSAy2wDZr\\\\\\\/Ggo5ZyO0n0seY+7bJ7+n89Zk8LX0KHBsgflU3inG58Ye5CvaVRh7\\\\\\\/sZuFVDulwGwy4TVV8qLnfgQmnla7VJgqK1jeKaSdBx6xAGof0jb7WvnyPIkRQgRzwqzuoTA9EED2YJneSVpzBa8yIGcw1ptoSjcxsabwlQUNqgd6NyeN2iQMa2kSxwIRW35qCk2U8f6HIJt+rOAqDVT2hSHW6qGhKDjj24AMcveKAY+5n52hCNie9cv3FYNRn2kSZCMmSazDdOAs1421auzGVizuTBB06zq6zU8kz09DDm9\\\\\\\/cFSW2Ff+BCU80RlSffLg17hSeGYVAHxAVrpq9Mudw47FUkNASjgymPH\\\\\\\/DnP9QmHAJ1BD\\\\\\\/etItGcBe1mUQ\\\\\\\/pOFpvjoBoWQCYL+0fr0TtL5gchri\\\\\\\/Rhz9MJW6AuxIA\\\\\\\/QSkZgtfIS5FSOk2KW1D3nIDaOXOFmHNVwqh5Jhd0WAGrq0qpn\\\\\\\/VO3hnJWzSVvXPfN4kjBqOnlFQV\\\\\\\/x3sK412fgYK4KgF9dYtmIQL0e7b5I5Z0EWpxHYrqADV7GvJZ78TNxrnDIOtA+Hz3fRJignvD9pdrECLGB0rP\\\\\\\/yRp902ydMuQ5cJNXt6VarCJboBZ9SvN+3ZAOHrjPQ1AvDRp3RowQiDuaahZ9pKbGs6yjai6P3LGPnCKvymOnCtW9H2FsT2O0e9+nMbDYNv7WrEH8ulha03XqFBpyFBQ4rGnxkLrTwmbInjmZ5NsdttGs9yzUG9CEfnD1lc9UXRQ0zb1aDt5CifqQwshTUsUOtDETfJLMs98GIQBrJ5DZ6lGxn3NLwKvOlOBplaSFcuCpm\\\\\\\/RkBElOJrwb\\\\\\\/8waGlMZgmsNSgDzB08ZF+JkquJyZg+ytS+C8SiqDtTDeg6H0Tf5IWF\\\\\\\/fyasB8vZeIvUedsLoL4NhPvIJAD0eUQLuuyCxzs\\\\\\\/LF7QVAnpFApWFEsg48zGBqYcYxg4rNYoq5XwFOq1HMkzxxKjPAxRDDMd4EJf9RjNtxCS4mJ2e4lNGj8bkTSO+uo9qmrRpSJ+hfBWXdI86Ni6uwLhewXffjuOQp6+ufTtMH1AmylxSaIrrcR6SiP9jrJEEPKsh1YulOImkqwTpP5dC1CaMAtPdeBnQmyEInbdfw1soACMvKmnDcQC3THnHzn8E3tSCXX6on5CWtvEqqGxPxKL2dzD84ZKx0fMSfOa9PPThQrCETYxxQH16LANEFbJj8boFtS5YU\\\\\\\/JWQ+p8Mvj34IoTHTo3Eev\\\\\\\/JijR8AbsIbbQgeWGl5c3Kp6saXEXFuVqIp8CLxxBZ+UCtKBhX7eKhHEbYtT\\\\\\\/i7hZxjbSGeaLEpGPokiisSRBk\\\\\\\/2Yn93l6DD10+Yp+cIsXm6PgVfq9\\\\\\\/Ac\\\\\\\/y4shLvLrXBAhDLf6L9XBoAF8zLL+Hs8VDzRY4GjPckY2D4hklcgaxF+0oyVSbDK3lXF3RcO2bfVQO5vU2+MtDvBnh9yc\\\\\\\/rQZuo6gYECm4m7hlSbv4bn\\\\\\\/SF5N4Kye0OssTeaysMOXGyU5H17HucOSp6Kg46FJrqE2HGFW9ANDZboTr0sMpGjRQFYHwvacNNarNcQShRh4YGCe623fCPlRv5FCwjKN079LwE\\\\\\\/vsvw2zfPmfUmH0+51MQ4NVb56BXrZkTljH3YcrsH9kD2zsN29qkEe6i2qjd2FRdHqGIPJaNsdFVmPa+7yJUPzpxTBxKTr8Kzeh3ahnl\\\\\\\/\\\\\\\/pM8\\\\\\\/RIv4RmoNCVLHqYGtdvkq65DsUfjk+4Vk0mSCuwdDWu+P6xpNy\\\\\\\/+4bKEXyen1JYoByBFcKuLuVdwENVxY19H6Y7DEIu1GgcpbNBBYbbyDR\\\\\\\/y141iVR8zGIDYksTGkBPUjnNGFezt2xB6Dp+aBdVvfmJf08WcK\\\\\\\/EbFCj0ifSMZx3AVGmaAuSc982C4WGdNO97Sc\\\\\\\/EHrgzrF1KMn2ex+U135032eNAEpyvtB3wG9TZ\\\\\\\/R6LgSS4zXNgxulvkPSGEtYbQ4vgmdLpRZpYTQ0yqlvu8sgHPbdjfh6spAl7dBiCp0M1aPKGB2TASs4Iu\\\\\\\/6RqojII1niZIBX2ntvwbThTnSDF49e0Kim4rUyCagEknBToiINCBRE1Ey9eGkxiDWvXtUZ6c2+7MfHeeM1tYS27NxQVMrJVU7kFeAhClS1QWcIQn3dl8YmUHWoXDxBtHrqwhaS0QFiiFwwX7BGecQkZXHPZnCEYQCWuvEGv4Mq+3LNcs400MuqxeJfzns+kfHejhlsoY\\\\\\\/l\\\\\\\/Gh8DiRcovOCH43FLKCgBPXFog9dsafaven\\\\\\\/UnmrLryU9L\\\\\\\/bnbnVrSsYJSEUTyYyPATBsO+zkwAF7B176zty3hm4taZ3W2YEeomLWiJvRkrZs3Kcpk3ezaZ5G04QMcSpWE3NJJlgBfn7Y5i2iGapOfxDfQwNF\\\\\\\/R59WnGGPQCt1y02pZECsnDuw6vjrUlEoq8EFBeb9wTL9rekZN2zW+h5Z5SbzhsHSDYShPdJ7CO1RBS0cIu02mjrY7PlL3hjHxDLFskfwO3HtjBs9d8ezhCqp0nSDbvdaqrSJau7Ad7Czn4Z0Ltt+Cgb2gbfwIJeb4HXq6PqBMuFm96QRffZmkokgNYthVp8joBHeU0ZIhtoFXhoyzSvaIeST\\\\\\\/m8Qmjbzs6npHwwr7aB5+eiijr+jUNvoml4Ar+G2Ye3Lrf9biNFuIpAkEfyxXHhpluGFgOSZSz5GMuniSloXxyWljlBnRrAwlYsrO1v2YRcWJOYRHZlHN8Mm9jU3OfaQolQd9eFjsKuKLXqVl5P6eKvG77zsWHmJYE0UlfmswO9Bu2\\\\\\\/kvd1K57Uj\\\\\\\/O45mQHctbgFo+MJqkkpwnIq\\\\\\\/G+ys9g8HAn78L+sGI6kpWd+16MyxPRWHhlBnj1QSd2K7WYAfRJfQwxqMbTpSNOYEqOaQXNFvtu+BFajmqoCX2DeW6JRtUvP3Nwsjannb9s6IlTdRmzjT\\\\\\\/YdlYC1L0RaY6kGQUy0vF+dzLw56LpCcjWXkOOFEpaj1HpQFNt0HJimMg8C8jmxPl1tSEpXWpiloYRrqJ6SVcc9+kLvySTHqh30Ll4EpUP8NHE2pBXpf5IIF4JFh2o+uy2MTHw7WvWdm5UkbEqqj\\\\\\\/FrLCRQ88bpilnPNZ8ZEg37i17VqxlhBVzIcmTjOVZehe28czGJ6vJjkfGSC2PoOHiZFkas8qO2nILzFJ9PVRNWxjvsVVoc0lbNNseg3CKOntOPMn7KUTgIdND5m653OBZJItgCHk1wLLG+i6LwMW1ljfPdGMUyqOb7MKeAOs+SdWOzGM8zzj7v9MDlNGBCPlKQfj8E8f76OzCTIktGdf6369E6Ajy+Rq+EAbKACsKW1M8hqQCZUxWHEhifzNf4XngJf+R3ks+z2Tj0HF7PU8JBVA2bQlGBv9yjoNFMGnJsAjCSQy7czvEsgy+PpIpxq5d5uO02vkM8qW5lHbZHV0MqYW0xbreIy+AOepM5bzCAxmsGYs1UIEIaxDQl0mbMvsDbibiKuwIIWO1v9wafLryvwM4ND9GJ2\\\\\\\/b\\\\\\\/U+Dmoq7r7Cxq7FW4HRnkc8ERdGoqynwFUyfrsuU6xd6y2OsjnCpn3VH5JOB0rGIyYsLd7zvN1NvjE3NH8c6nGujCnroXYXqF562JUk+\\\\\\\/x3KFf4Amh49geeiuEM8A3KRMZcoks7jbGkUX7x3dgEY56GWqQ2YDHsXx00pFbkJHrxVGDeDwzdABckVPHItJACQeP8sAl630yzI\\\\\\\/2JIZJ70rBY3nYjAnw\\\\\\\/ZbsgmJr9z+ds+5weMrrHQYW0kRpR2HiTcx+7HpblEmssrM6qdp1Nn\\\\\\\/IQiDp\\\\\\\/FvhflVMbGGPbiwiGFWUkPkFAdBJQc51bbYqYnHlH20dEywrpcFI2Y25OTwpxyL5ZdEH3JWdaVgEOPNgcoY90dFKtA0QsMEt2kIquTMoS4L\\\\\\\/eJlgVR3wHAsVr1DvHopEeP2s6Usi+0gNDLUMjJv\\\\\\\/dKZqXUudxmIbnopFLWRY+NtKMypHhSwz+S4Rt2anMx61f5CQHXFQs\\\\\\\/nLjgDxanJ3JKi+7B4I\\\\\\\/IbwpBM3JEDTs7f+YdLPZ+hCxEqglVGSlBk+9sqQGjyc4HWhJhEZ1pZB8bzrsB3h3adJj8JoAVh\\\\\\\/dZjtGjYDIz\\\\\\\/2Ahznd8ED\\\\\\\/kHJeDreyE9QS+fG09n+xkOkx3ZgfGrg0cC+shqgfJg8mkVoEgS\\\\\\\/T2EYfRlPrZ9I0GyCEMD5pIohHyY+msDmBulVCAcFsHK\\\\\\\/prgwPvA2A30dialYSwmQQzzYRXS8qjyDcREmf4X3I8sHD0TmT6KBNxG7n7QnTqC8a\\\\\\\/i49Etpa3ojIIeYrnePKdA4wnG\\\\\\\/hs7vdROg\\\\\\\/3nQ2SxyPBlZW7TIEGC\\\\\\\/lm0XJXbupGcB2z+3TOJ4bLuh5NVykJqlI25TVW6ea2V5yLdsvbX3Fg\\\\\\\/Mo0HvZydZhq7UVrmtma5nG3BM3QWc0XmZ9S+reeBBFpSp+PR195j39+UPx2Uu1PwmiPDrSkqbq\\\\\\\/BLzYILmDdoGpJpdeGV6ie9EEnniKkIH8uUIdoh7\\\\\\\/2S+IbaxU0jcScfENLZJO6LhTrk17FIFUdGf4VfMRulTHsMTBDEh3enbqmAASx+4kzlB7Lv74SKqj+WNuJZXqDEtVldxfrMA4UNkGze77pG3\\\\\\\/gJjgiWpbxnDVSmUCKochtEppw6Ua2hn68F9e5zSFwXNuSv9LNaYi0EzC8jZ2bS57U\\\\\\\/WY8nlkFSB7z4OpCqwoL8PWvM27LYzwsq2WsopEe1E2c5sNE49WD3Nf7VinrSwJL\\\\\\\/39nN5s5SusTihJsDooG\\\\\\\/sFXyTPm+oXBV3zlHG9HbDRMJbxtVyl5BUy02PoHdJiKeqzNG4RvodV\\\\\\\/RnQC+OaN5vBj1x3WwiplQNo3l68VAN\\\\\\\/mtLKRWF7nH3jU4OsvqHKPWPLAoboKbMgPpy3YyZDAKPaom1AnN9xSC0sjp1RZ5YqgEy9IBGLWF150STLF3h90vqL6krnQo2iAvCrbXhFDj8y0Fp+QKRnLHeZSnCkToVSYP95t0VMLA259FoaYJI+f0q1Htp6E8U3YNwGflxCf1O3nHuIQ+FNpJtAD6X7+5lHcLXd2dI37jmSy6xj8ARO6hsetdN1EkJDzN8gyZHRDfh0LFlR4FJcEgOndieDA+cJPwXRukIW4gqqx9WhpiLZNlnX72dSvfyxSAvb0giCrGjUX8TfQOJGNV4FyxcQBtS4xCSo\\\\\\\/f2qG3F8prhJvLbjpGikKBMHBWWs43uwDo98MnUqzTVBwrR3kOADYFnkGVZz9rNPiCqKGwzcOwMCDMYiDaKm5faLCJznZzK0RB\\\\\\\/vrRu8n+eE168vamgGPI92cdglCMwDbksz0NKnZYgor0Y689UqJ7522JktbARidrr3pEFvvL\\\\\\\/mR4\\\\\\\/PEQFuT8HNckJfYdLMwhPbiA0rF16xtVZH0brwswPU1sfmFz6LVh6bWMH+3ONaA2fjmISXeLwi2Ja0\\\\\\\/M4G\\\\\\\/h5YTa4JGAVxXKmtCgNavS0j\\\\\\\/BLxBPU8iTJtHCKNXTKuBEq5JiaisfIc8ATMaLK1ePwlM0NZlURn4WISn3knwT3QkHlcjW6P9vSzU9J12bAGD29aTEUSPJogaRzbXejsFAcAXf62dszmnFXERdrjKe3ejxWu5mootU6EGFidnqJLvnr6CTjEy6\\\\\\\/ImzHqRDwu1mrrce4idwg+s76gV5eGBtPt64HGB2XD0BC9WWuJ5+Sn\\\\\\\/Aath3iVRn9NqRhtGG8zvLZKxeOH3rzMs9\\\\\\\/KSUtzDOdn8n99N8lGyzJjA6Cs\\\\\\\/KUwngQEP6Atjzcvd7Ab+KvcRJdNqnlA1uSOrRvy7\\\\\\\/n1E2wza7aC\\\\\\\/JBvusGuAudD6gLQl\\\\\\\/wzZllpOOI4YOJ3Bt99jP3O5dm3s7UcT+frQG8vC7FodU9+U63ilorImF0CMTN2AnVxdGAd9hQGYXwC2iHtDMhd6XJkMrQksaJFXZ\\\\\\\/YoBkY76+DzFyNTRebXmo\\\\\\\/O7jok8AXwFlpkXKu\\\\\\\/bkeFFEAtpK6QUR6CACtKICcfGfYNdPiIZigygtLItIJKnb4QBjZTnoMK7v8pYE52uhye3X5HpRJGZjfg8Qlrl6S1M3ItV5rRwnFa74obATiIEFmuxeqn0eHk+MpTf4E+eAUyE0G9n4NmRZlJ9g2rc1P0tSRm\\\\\\\/Uce6ttuCtxfKNgcqVa4wi93rakK\\\\\\\/gxoGjWENFUaESSe8d648X5FUofxvTptUBmymZRrCcr0cDlIWa3Euka6EGJ9V71dM2FXHN81biiSQR+hngR7w2r8T5sC2J5Kif8x2jRyjIwA+ZGtTWCWuIbZOJ8rQTu3X88EYfMfKlA7xQa0F8sZNFC0LKFsxs2jkLKT73nq5pQawWr4gIwj9HGWbtK9DLUkP5+WB1i05Ygnwoe6SSc1uaClBBd\\\\\\\/xtNo09isRBMWHMHdXS8NCNLn4CIM2oQH\\\\\\\/1hw0WHBTrYKkLBX8rGNlmWiBdYRWoOmMgCjw9yk+fTKdOTEXHIy\\\\\\\/v3sNW1BhbSDObLlwZkaxkra1MZTdCsslwFd5h2vNCpBp9ZzNTZOJ0XaXooo6sHFtYnFg2FuRHeOK3F6n\\\\\\\/qAoV\\\\\\\/sdXSmk2TRW+CT4O6xqlZ04QqfA1gir9MW0atkAs3+RjvNDa6yGC0\\\\\\\/wOcJZOBg7cizRmE0KvjJ35c+uVyKNqMKnCixI7LcJc6qsTcNShLPOTCvw0BzL9lOfKtAMzts70yYtvECjX5KotomP1HndeMmnyv1Z7Yjmz65WwLviYXcl0Dj0l4STO7nN+CgLp0VlTFEmsePGRvoChfHfgTyLy2AvxjUbgjj0Yd249Mg3HAHz807HYbrYysPe9QRXbHdj4xXBuN3kmYrZZ61w\\\\\\\/kQLrSYIdq4nTcGqBP0YCNyrpHNbBGfAvVRUFDOL27R0Ib6uIywAO\\\\\\\/GK08xmKhqm9PGCdFeSv3GI+y8Zjs8gcO4A2267GV5rS\\\\\\\/JXkmmWfTbadRBjPVqlZUb1ViimP3lsV3fBJZTjlmUFd4vNcd7mJ0VcAfwO5rjfibUDGE2TsF788TEh9U0RU+s6SuKOe5rjGHb\\\\\\\/LjLFFPinDqmB+H+tsn0kYzM2nrp3R2AwUsnUXDgzLtZR6Uo\\\\\\\/crDe2pgqvMlq6rV3a7R5WIKiNnscJM7u8behc\\\\\\\/h5z5\\\\\\\/Gf+II\\\\\\\/FV7d7EZqjQPOXNrtnngdXgWvewdRi+3JUGNtLoM9aobkM\\\\\\\/2skMda6JuMUDLhTOmJAchBaipngf3OiHoJbnUHqo9HIE450J53VL9oAfASL6Z4vMY2KFD+EpvJVlokjWqolL5MTUAEiC1oFFBSRJnTC+0kMFBxZrsR9PnmSl\\\\\\\/ETpQQGbVkdZf\\\\\\\/9H8x6mnF6NliF0mBnc8mV3cj0FFl4ZaRT8CDuE6Y48WcqKt+5eghufSmdF0E4SUD4IvwSWt1ho7+iCO2hrxTn\\\\\\\/bqs00B5XTmkFj8ixaAB7wY+PkeD67lSbNzru9+HNLc4z5qoyH4X2Kdq+qqGzoCRpUsQp1kn4ovobFnXGxULt4QPJU4xKCdKqQgSyMNx2lU\\\\\\\/eD8mB\\\\\\\/LRpa05znD6vW7u1PbEMKh0WbOcLbLqVyvw6ks1GyApetEot5VDxbeBIRjpebPjwPHxV0CJm\\\\\\\/qCP4qUCejkXhJ\\\\\\\/xZNB8VwIAMq4LCWOD6ygNdlnexLOnzYwlJX8C2bkosAtubqiC\\\\\\\/jYeP\\\\\\\/u7LfJ81ncIrh4K6jkHuYarHBBwF5HHPhVv2CbG3CsE9LCWce\\\\\\\/tPJGEU0WQIMOVnAlxqPa3EcdQuFbwrIJR4yl8qJ7S2HBSn8ngvHdPSMO24\\\\\\\/4UYmjzdz3Wz3QJounEIb6\\\\\\\/aB5pT5SVEQBmGGz1LH\\\\\\\/M2ulqBWRClixU6c5edhZcjGCn4+abu9KzK5DCCpL+\\\\\\\/52m\\\\\\\/cLdoydjn+ZsPWcgP5bKSj8cucmy9PDAYDCCpnZNqkKnSF6c8YKV87KEi2\\\\\\\/PNhZvvlA1Y+7qPof+PUz\\\\\\\/i35YJAM+CKlkk9hqJNZm4bNmnpnrsQmPNQR7h9SaqHokdDRGXE6HjRcFplWc9nH1K2angRJjmuTHEqQ6rWHTNMgvCTayrJgf3ZbBa18VYPu7c9drqqEZnZofF\\\\\\\/Cjej5RTYt4aDK6Gu47eDGRnXgS42lksZR97NetBwWJwiicOIN5ZWeh+gu6q9mGQMrKlFItFLHuGT1Uo8EgYQzss3oH8XrmLYlRjNGJ9C2+peJsmgQsBzzqb7XoteGRtN4++05Zslzu0rjM3XA1Xyatkv+5+WHn+Rw1TIX7QeuXLtRwd91zMFfEum8ERAaqKnv3gPv4pn6Pa2prZGixl5m8SPxbqYHXvp4zYKd8jzSUKgN6\\\\\\\/5rdOQDSlxnACdcp2EVdPwSGAJFAk5FbNRcwn6XPwptbvPthD1Shn1e6QN3Fbtn2WJy6nr9Sw+pFlXV1rIlj5bIosZyGz9OkAPhRDYinclmmaeyqAAW2HWlfUOoHQ3Oc4zrB3IZNWzPYlPOqI5WXTsA7lt6Nvm47up4I05u7w5Mm3HcMwNHq8jX8ZpXX8gZ\\\\\\\/0oEu90euDaPpghH\\\\\\\/kr0itCc1XJ90sW2f37MelYL\\\\\\\/Y+qm5+ipyoBClWWyAyufQlsxQpy6SmM\\\\\\\/UgaGPsPKQ9LmmFd+zaL46QRr3CxpcIpaMouIitf12vlTs8x9UjV+yDFhfsDPSnXYBS4MZzh3kPB2dcQYwcghimjeQduFceUATWgZPRyEggx+tx7pDCh+n4+5tLHh71q7f8wpBBGuo54nuyHCzvb8D5JbqX+A4pH6SZoCiMZH\\\\\\\/hifNAjyoC0qwS1rqqnCkPJ2RpXCPJ2gwBTWkcVROIKv\\\\\\\/9JKTo7dpoi2zYvgQ2lDxjRw8sSbVx0nEmIhnttJTz1dYC1CTx0ncJgdzjw7uyDOvKSM6QCFfwyc\\\\\\\/KXPKYgswv09MrCTH0UY403XX1IyHQTFc2hU4muSdwXJ\\\\\\\/LgTiqTFunAX\\\\\\\/Io8iWRRBRVE\\\\\\\/+1YLvkGdCN7ezmyO4tPN+fSdfU9B1nyBexFmjsvJo1qMmdeNR4HwgH4VZ6MA6lLC\\\\\\\/tO9E2JvUoOjmRwP\\\\\\\/UChUHg0aS2ztf4OOht6FxJ3kHdiaJzswhB0PxQHZ3A4M0cdr57duSJB9ESBbxv9er9+FZAZjQSYK7UMB+FUED1\\\\\\\/zXaQRmAOxo9+hDdvZFPxVw6o6Bk+cxX4XgbQeyZCzntSnSK71Q4zNz8bpC9XCralp5LYF9SldkoIGg63lRAjYaJYIg9+DCU6fg1ZmzADtweklak0YAyJnNVDjImPyxIsj5e3Ha7l7Ki8wxFt8V6C\\\\\\\/1h0Pv0FoGBlFo5uFBBghaP+2DjNbQEhU7KX7nMGSsK\\\\\\\/vdNeV8m1IK6obvsWIBM8HEIsRT5IfipstkTtVx7wmDZHpOGizdGFteWA60R89C4WJMpS\\\\\\\/UjCngBfun2ghsSWIu\\\\\\\/Y8oStBtExZtFtn\\\\\\\/cgr2dVG3697LI8KcNT6nec2Oj0+cg25xDMQwCEq7cp2rCuf60TtuSk34HSKZ0cuaj09xLFJTTItQA1g0UwSSCmuUZc8drWajht2hTa6ctprGZHXlrpqr4JlsiUAN1639F9UV2mHVUsvX+Hi8ToG\\\\\\\/+l1M3AOMFfo6i5zxkLBy\\\\\\\/pwI\\\\\\\/n\\\\\\\/qpA5qWhslKdGIMhrlz1yhAj5DBUXwXNI+BHG8658H\\\\\\\/1G0RUY+Gmcl9Tko3TYEPiUs\\\\\\\/XVQPtcmHeCjwT6bznlienQjGV7iWr2dXUOthGjFGS91zptGGjCzK6yKZ7ZfqDecmkbzX2wub+P67sbdz9aWD3tHe1IIlrMYgKIz\\\\\\\/MzPeLiyyCDi+zXqKVn\\\\\\\/fVc1RAWKqJXrl743Jt5dzmpZ2GKLh+876Vl41Y9WPIrCAr+A4tdJH1frJAOHpZboKjBSA7NhqHG8WoVr5\\\\\\\/whUz3Gp37SNpmzLkAGy7SqYYmbH+uECHXdWxl04M+flZDuigh51ZXD82DMJvm0MPyI1tEI0qp8Sqd7af8F1Ev2E70YBeBWFBzc0GAmowh0ooqVR5f7fogwjeRwuVmc0754ybnyI4hi10MH1XCUK\\\\\\\/6CJca5+p+4ghzkNNNMBEtl86uHaaeFe7tZ1s2M1JZxhob+eCfHWpieNEeFv6yHRPpBbW83rJX2LoUUHTI2dmLMaKkECKPW7jra3nbXOgXnQJWIlrcYzmRdzZbp5w4huMcJPxIkW3Gm7Ul506nW9KDnBD3pWsKpN62tC\\\\\\\/sCMtKsEmKbzSTIyUQQIM4lvl7aOi55GhLHyX4mLU8grTKmodt0sPOlWYPrJ874rcslCUZX7BNVPwbdoR+LuBuJU18Q\\\\\\\/wkiIGZo4k0sWMk93ZG9XY1Kig9zSov2WKaOjPbC6YGPtKUNLIhubi3YO65fgepHK83Iyb0Ja7X7CJsiR6o1VuLW7BBT9NgWR9B0fQ5Uos8bYiNkiC+39QXYhLo9VO\\\\\\\/GOuV15gAkvcXBc4x4ISQGCv8I+oeqgyzRPEDYsAU3vHW4YBJukk0FpuK+sMimksWzFV3O3w+KFkTVhFHRlB9lIDylx0GK+GMvau0T6vNHdnrA7VHXFNCj1NBaBGdqOPGBzzvHr0ObeCwGspMt5VOF+hpgAROhYizmuI5Jb98+1oC62Uob9wxmw==\\\u0022,\\\u0022iv\\\u0022:\\\u00229jouE+Wa8UBusYqammlCEw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00221d66c79c0df4834abffd985c2997ac66d9b35eaba0fd4649130cd64d7d61c802\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00228\\\\\\\/mupcl8oH6H1eJNpY+JI+h75+coXm8EKltpK08Rp8x0ineZm+H4x8RyQqA0n0zMk1lCmeHd7cHKmFfpNk1gpMT0vdI\\\\\\\/9D1BiGmVMOE4keoy\\\\\\\/PfOMsykBshp1s\\\\\\\/Pi+6TsY7TAz3iK22cke350UBpEFVOqbXq0RyGEj+htPaLhOEKkdZBvtmDmfEeOXg34VansWFdjb8cqArde4hVLUikFG1SVetXu8VG4GZdOyAc9BF+wqsl1a7o+aRGkd4\\\\\\\/hjU11Zcn5RS7R9skEuyySpCwrPbQ2XJcaW7\\\\\\\/HQ6Dl7kFdNzqMdXC2XxuBHZlSb7Y\\\\\\\/R4Grn71SYwuXHDJI9b22ckUyI4XBFSOO8tAcur6x5pae1vP4EEfs4wXwuv1kIRmofY06xVuCj1qa+EEbqocftsifxXtvAl24x\\\\\\\/dbhkgSpG03d+GjtKJgStxI5bf6CCeqRvyzqkTEtOAAY3xXgZtmnPVih7jo1RO6LcfMqyBYdaH5BccFOLASUm69dxCGWAdjhjhKLmg5vffsY+0dkXRN2hnn7NCT4GtfYxhKzGDlKTsEMw2cfe7OwH7DH58eJvT+Mx0OGjord+IWpdfvP2s0sKhoiIYbkAFyF83ErLvZ2TBsaNOEhscFh7CMDjJUsIqrbKkBH9djznAaedauDsLAutmLCjPeX4nUqW7mep42145sMXM2Pb+sV\\\\\\\/VvAhdpkjbhftE9oAEyHOzeiKNhk\\\\\\\/4iOBXhNEPH9ft8DwzA9r+qBvsaK8xJt2s7+X+UX\\\\\\\/z0jAW0hQamjcTdpoE8dTr9tFqBnVjPNgbpvo7USq+CDTsTlUvzM05kfYwSvHFEFdeJ4oUM5jGPFyU9MqA7vUrba10KYVg\\\\\\\/JW\\\\\\\/gudfVQyilAtiUcV1bbYa+0Si6yxG\\\\\\\/xNlg7bYOS2igQxgISP3dTPKjnfQucrZ+DasfddYDvwgjNuEE9uZgLVEAHVlol0Uqo8PikL22tgpcIq9qIR2vC5cptW6XDpYArhCXyBWaUoT1aOFvu2J\\\\\\\/68wLwYqqxzAhtpE6rpFChmBVvcCAmGyPQVqcjtGfdWvGTM+Ug96OCQ+9vTvNhcHbMMvDkad1hP\\\\\\\/vmUGjycB+2CL19VnaS68sxRfoRny0XYaoQ2YThF1qUKvPILYpautX6RNvOtuF4WciymmAa3NqyRth7E7t1bPRdXq2FN88WWpSz2BvZZxFqRLs8bnskvBd2qn3EHm6oUGFRp\\\\\\\/qPsTa1xtjPAmhzoVMmt3g7Q4BVOM1Uc9\\\\\\\/H6Wz8MJR32Con1R9RnkDn8qG2a+VJAHp3zUMdwrQVidIVWGg86x2O7XwNsqA5g8ODg1C7B8sofo628dZUJr0yf+qJNIkVcXMNMKY56+p3O92+9d876a218iQRLlFpaOsRANo\\\\\\\/diEX3D8p1+TUox466Lk8Ufyb9ssT\\\\\\\/ykC5txqzMB0Rlg9D84yaSWwDmLvScewwXNj54cZHJuaDUH8iBWelz1wSQ8Fb\\\\\\\/4VsjKnCH8hE2zhIjudBIYXutHhvBGm7co\\\\\\\/tWhjGYiTNxnvTZMF5ed2Q2afierW+A6e49WR1b81DRMIleTBNdzhCsbL1Cra1vTVeJwHmd4p35iMU1mLDW1keO+csIUQ+vR8Ch5I+VpRCflPwzBFUN30FKeWeg5XO5lZ07xf8LVyosYW+M2+IZvisVb5j5N667EpiexNpTpUVvLPQpyyKr90DjpN1MEyK13nvHR\\\\\\\/rash1fXwlGhSOr9A9QxoyUG5I9IT7h544XL+bY87JKeZjWDstIDw69AqKeUrc9PegILJUHqhJz5M4AQtFRh+Lce7ZB77KR7HUFjQjG8wJIaYI4xaoS9H31Git8TGa1xL6ARjGmt8ZLVXi5NhMt45YRGmDONY9oHvP0kM8o3jOTThB5jv7AsQa7UpKTBRZWvVdzZXoPcAU3x\\\\\\\/WLc2ITITIQEkis58GWKHqAipek7jYyhlEnXEtAFsH52VEusTKr8jqigXMJimk+LV4O44dFXuFN8qPotaB\\\\\\\/aK6IzS2ifB5sbKvTKQXGk3Cptw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022euaamBeFMhDQCRlFqhiNiA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00226a0a8820f99b1aa5ae08fe988f78efe3b2c91aa8da102e7c945df4179ead1e78\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-494272 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/10/3239" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-iYQYQ3FFQ6tHa43k56-75Ae--8bk0I2U-walFvODw7k" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/10/3239" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-M-zp_qhMwLKawdsAO9o25jyi7R8dzQA6VULdDC0mcHE" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/10/3225" title="Phylogenetic Reclassification of Vertebrate Melatonin Receptors To Include Mel1d" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/10/3249" title="In Silico Karyotyping of Chromosomally Polymorphic Malaria Mosquitoes in the Anopheles gambiae Complex" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494272" id="node494272" data-pisa="ggg;9/10/3239" data-pisa-master="ggg;g3.119.400501" data-apath="/ggg/9/10/3239.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Exclusion and Genomic Relatedness Methods for Assignment of Parentage Using Genotyping-by-Sequencing Data</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-9347-6379" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Ken G. Dodds</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0003-4801-6207" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>John C. McEwan</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0001-5068-9166" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Rudiger Brauning</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0003-3714-3542" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Tracey C. van Stijn</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0001-8727-5960" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Suzanne J. Rowe</span>, <span class="highwire-citation-author" data-delta="5">K. Mary McEwan</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0002-4615-8917" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Shannon M. Clarke</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3239-3247;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400501 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494272" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Ken G. Dodds </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>AgResearch, Invermay Agricultural Centre, Private Bag 50034, Mosgiel 9053, New Zealand</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ken%2BG.%2BDodds%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dodds%20KG&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKen%2BG.%2BDodds%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-9347-6379" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Ken G. Dodds</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:ken.dodds@agresearch.co.nz" class="" data-icon-position="" data-hide-link-title="0">ken.dodds@agresearch.co.nz</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">John C. McEwan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>AgResearch, Invermay Agricultural Centre, Private Bag 50034, Mosgiel 9053, New Zealand</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BC.%2BMcEwan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=McEwan%20JC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJohn%2BC.%2BMcEwan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-4801-6207" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for John C. McEwan</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Rudiger Brauning </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>AgResearch, Invermay Agricultural Centre, Private Bag 50034, Mosgiel 9053, New Zealand</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rudiger%2BBrauning%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Brauning%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARudiger%2BBrauning%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-5068-9166" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Rudiger Brauning</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Tracey C. van Stijn </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>AgResearch, Invermay Agricultural Centre, Private Bag 50034, Mosgiel 9053, New Zealand</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tracey%2BC.%2Bvan%2BStijn%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=van%20Stijn%20TC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ATracey%2BC.%2Bvan%2BStijn%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-3714-3542" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Tracey C. van Stijn</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Suzanne J. Rowe </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>AgResearch, Invermay Agricultural Centre, Private Bag 50034, Mosgiel 9053, New Zealand</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Suzanne%2BJ.%2BRowe%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rowe%20SJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASuzanne%2BJ.%2BRowe%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-8727-5960" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Suzanne J. Rowe</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">K. Mary McEwan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>AgResearch, Invermay Agricultural Centre, Private Bag 50034, Mosgiel 9053, New Zealand</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=K.%2BMary%2BMcEwan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=McEwan%20KM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AK.%2BMary%2BMcEwan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Shannon M. Clarke </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>AgResearch, Invermay Agricultural Centre, Private Bag 50034, Mosgiel 9053, New Zealand</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Shannon%2BM.%2BClarke%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Clarke%20SM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AShannon%2BM.%2BClarke%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-4615-8917" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Shannon M. Clarke</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/9/10/3239" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:494272" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:494272/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/10/3239.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:494272" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:494272/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/10/3239.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:494272" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:494272/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/9/10/3239.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:494272" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:494272/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Genotypes are often used to assign parentage in agricultural and ecological settings. Sequencing can be used to obtain genotypes but does not provide unambiguous genotype calls, especially when sequencing depth is low in order to reduce costs. In that case, standard parentage analysis methods no longer apply. A strategy for using low-depth sequencing data for parentage assignment is developed here. It entails the use of relatedness estimates along with a metric termed excess mismatch rate which, for parent-offspring pairs or trios, is the difference between the observed mismatch rate and the rate expected under a model of inheritance and allele reads without error. When more than one putative parent has similar statistics, bootstrapping can provide a measure of the relatedness similarity. Putative parent-offspring trios can be further checked for consistency by comparing the offspring’s estimated inbreeding to half the parent relatedness. Suitable thresholds are required for each metric. These methods were applied to a deer breeding operation consisting of two herds of different breeds. Relatedness estimates were more in line with expectation when the herds were analyzed separately than when combined, although this did not alter which parents were the best matches with each offspring. Parentage results were largely consistent with those based on a microsatellite parentage panel with three discordant parent assignments out of 1561. Two models are investigated to allow the parentage metrics to be calculated with non-random selection of alleles. The tools and strategies given here allow parentage to be assigned from low-depth sequencing data.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/parentage" class="hw-term hw-article-keyword hw-article-keyword-parentage" rel="nofollow">parentage</a></li><li class="kwd"><a href="/keyword/genotyping-sequencing" class="hw-term hw-article-keyword hw-article-keyword-genotyping-by-sequencing" rel="nofollow">genotyping-by-sequencing</a></li><li class="kwd"><a href="/keyword/mismatch-rate" class="hw-term hw-article-keyword hw-article-keyword-mismatch-rate" rel="nofollow">mismatch rate</a></li><li class="kwd"><a href="/keyword/genomic-relationship-matrix" class="hw-term hw-article-keyword hw-article-keyword-genomic-relationship-matrix" rel="nofollow">genomic relationship matrix</a></li><li class="kwd"><a href="/keyword/exclusion" class="hw-term hw-article-keyword hw-article-keyword-exclusion" rel="nofollow">exclusion</a></li></ul><p id="p-3">As the cost of sequencing declines, it becomes feasible to use this technology to obtain genomic information for research or commercial applications. Often sufficient information is given by sequencing a proportion of the genome, for example by using reduced representational sequencing approaches such as genotyping-by-sequencing (GBS; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Elshire <em>et al.</em> (2011)</a>), restriction-site associated DNA sequencing (RAD-seq; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Baird <em>et al.</em> (2008)</a>) or exon capture sequencing (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Ng <em>et al.</em> 2009</a>). To reduce costs further, sequencing may be undertaken at low depth. However, this increases the chance of not reading both alleles (in diploids) at a locus, which may result in a heterozygous individual being called as homozygous for one of the alleles. An attractive feature of sequencing-based genotyping is that it does not require up-front costs of developing marker panels; SNPs are discovered and genotyped in the same process and this can be done in the absence of a reference genome sequence for the species.</p><p id="p-4">A common use of genotype data is to assign parentage, for example in agricultural or ecological settings (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Grashei <em>et al.</em> 2018</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Städele and Vigilant 2016</a>). Sequencing may provide a cost-competitive option for this task, especially if other genomic information (such as breed assignment or population structure) is also sought and/or if no parentage marker panel is available for the species. Checking recorded pedigrees is useful for quality control of sample assignments. The assignment of dams allows the inference of non-genetic effects such as birth date and litter size (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Dodds <em>et al.</em> 2005</a>) and allows maternal models to be fitted, even if the relationships are estimated genomically. Similarly, if genomic prediction combines information from genotyped and ungenotyped individuals in a ‘single step’ analysis (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Aguilar <em>et al.</em> 2010</a>), some tuning parameters (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Legarra <em>et al.</em> 2014</a>) rely on there being pedigree information on all individuals.</p><p id="p-5">As low-depth sequencing does not provide unambiguous genotype calls, standard parentage analysis methods no longer apply. A common approach to analyzing such data are to filter out the genotype calls with low read depths (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Kim <em>et al.</em> 2016</a>). However, this greatly increases the proportion of missing data and may result in low numbers of SNPs called in both (all) members of a parent-offspring pair (trio). An approach that is used if parentage analysis is the primary objective is to filter to a set of high quality (including high call rate) SNPs (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Thrasher <em>et al.</em> 2018</a>). In both approaches, there is a loss of information in the discarded data. Here we investigate methods for parentage assignment from sequence data that take into account the way in which the genomic information is obtained, <em>i.e.</em>, using a model of random allele reads at a particular SNP.</p><p id="p-6">The methods developed here can be applied to parentage verification (where parentage has been recorded, but genotyping is used to check the parentage) or for parentage assignment (where parents are not recorded but are known to come from a given group and genotyping is used to match to the specific parent(s)). This article will focus on the latter situation as it is more general.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Use of mismatch rates for parentage assignment</h3><p id="p-7">A common approach to parentage assignment is by exclusion: find individuals in the father and mother sets (ideally only one in each set) which have genotypes consistent with parentage, <em>i.e.</em>, no “mismatches”. As (low-depth) GBS data does not always give the true genotype (heterozygous individuals are sometimes observed as homozygotes), there will be mismatches, even with the true parents, with non-zero probability. Here we consider mismatch rates (number of mismatches divided by the number of comparisons) as these will be more stable over differing numbers of comparisons than mismatch counts. One possible approach is to calculate the observed mismatch rate after filtering the genotype calls to a given minimum depth. However, this will reduce the data available for a comparison, especially when trios are considered. Instead we calculate the “excess mismatch rate” (EMM) as the observed mismatch rate minus the expected mismatch rate.</p><p id="p-8">The expected mismatch rate is calculated under the hypothesis of parentage and takes into account the read depths of the individual and the putative parent(s). The derivation is given using the model (and some possible extensions) and notation of <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Dodds <em>et al.</em> (2015)</a>.</p><p id="p-9">Let A and B denote the alleles of a SNP and <em>g</em>* denote the apparent genotype (<em>e.g.</em>, AA* denotes that one or more A alleles and no B alleles are observed). Suppose homozygotes are observed without error, but for heterozygotes<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3239/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3239/embed/mml-math-1.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3239/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3239/embed/mml-math-2.gif"/></noscript></span></span>The binomial model assumes that allele reads are at random. For an AB genotype, each read is equally likely to be an A or a B, leading to <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3239/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3239/embed/mml-math-3.gif"/></noscript></span></span> for a genotype with read depth <em>k</em>. The theory is presented in terms of K for simplicity of presentation and to allow other sampling models (relating <em>K</em> to <em>k</em>) to be easily implemented.</p><p id="p-10">Expected mismatch rates are calculated under the assumption of a randomly mating, non-inbred population in Hardy-Weinberg equilibrium. Let <em>p</em> be the A allele frequency (assumed known). An example of probability calculations for a set of true and observed genotypes for a trio is given in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>. The full table is given in Table S1. The corresponding probabilities for a single parent and offspring genotypes is given in Table S2 and the derivation of the probability of an apparent mismatch, given the offspring apparent genotype, read depth and read depths of the parent(s) is given in the Supplemental material.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/494372/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/494372/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Example of probability calculations for the case where the true genotypes for father, mother and offspring are AA, AB and AA, respectively. <em>K<sub>m</sub></em> is the value of <em>K</em> for the mother</span><div class="sb-div caption-clear"></div></div></div><p id="p-11">Let <em>K<sub>x</sub></em> with <em>x</em> = <em>o,m,f</em> to denote the value of <em>K</em> for the offspring, putative mother and putative father, respectively. The probabilities of an apparent parent-pair-offspring mismatch, given the offspring genotype, are<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3239/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3239/embed/mml-math-4.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3239/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3239/embed/mml-math-5.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3239/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3239/embed/mml-math-6.gif"/></noscript></span></span>The probabilities of an apparent father-offspring mismatch, given the offspring genotype, are<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3239/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3239/embed/mml-math-7.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3239/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3239/embed/mml-math-8.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3239/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3239/embed/mml-math-9.gif"/></noscript></span></span>The equations for mother-offspring mismatch are the same, but with <em>K<sub>f</sub></em> replaced by <em>K<sub>m</sub></em>. For all three situations <em>P</em>(apparent mismatch | BB*) can be obtained from <em>P</em>(apparent mismatch | AA*) by replacing <em>p</em> with 1-<em>p</em>. The expected mismatch rate for an offspring is calculated as the average of the relevant set of these equations over all the SNPs.</p></div><div id="sec-3" class="subsection"><h3>Use of relatedness estimates to assign parentage</h3><p id="p-12">We also consider an approach (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Moore <em>et al.</em> 2019</a>) that uses a relatedness estimator appropriate for the analysis of GBS data (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Dodds <em>et al.</em> 2015</a>). Let <em>r</em> and <em>F</em> denote relatedness and inbreeding, respectively, and subscripts O, F, and M denote the offspring, father and mother, respectively. For non-inbred offspring and parents, <em>r</em><sub>OF</sub> = <em>r</em><sub>OM</sub> = 0.5. Allowing for related and possibly inbred parents, we have, <em>e.g.</em>, <em>r</em><sub>OF</sub> = 0.5 + 0.5<em>F</em><sub>F</sub> + <em>F</em><sub>O</sub>; the relatedness can be greater than 0.5.</p><p id="p-13">Genomic-based estimates give relatedness relative to some (often not well defined) base population (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Powell <em>et al.</em> 2010</a>), and so even a non-inbred individual and a parent may not have a relatedness estimate close to 0.5. A possible solution is to find a transformation (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Weir and Goudet 2017</a>) so that the parents and their non-inbred offspring have estimates near 0.5, for example by using pairs of known (pedigree-based) relatedness or scaling so that minimum estimates are close to zero (if it is reasonable to assume some members of the population are unrelated). In many situations, there will be limited sets of possible male and female parents, and the true parents will be the member of each set that has the highest relatedness with the offspring in question. Any reference to relatedness or inbreeding in the following will be taken to mean estimated values.</p></div><div id="sec-4" class="subsection"><h3>Parentage assignment strategy</h3><p id="p-14">A strategy for parentage assignment, based primarily on either EMM or relatedness metrics, follows. Here “best” is taken to mean either the lowest EMM or the highest related, accordingly.</p><ol class="list-ord " id="list-1"><li id="list-item-1"><p id="p-15">Initially assign the best member of possible fathers as the father.</p></li><li id="list-item-2"><p id="p-16">Initially assign the best member of possible mothers as the mother.</p></li><li id="list-item-3"><p id="p-17">Discard any assignments that fall beyond some threshold (EMM too high or relatedness too low), for example by visually examining the distribution of metrics for the initial assignments.</p></li><li id="list-item-4"><p id="p-18">Discard any initial assignments where the metrics for the best and second best member of the possible parents are deemed too close to accept the initial assignment.</p></li><li id="list-item-5"><p id="p-19">Check that the combined (trio) assignment is consistent (both the trio EMM and <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3239/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3239/embed/mml-math-10.gif"/></noscript></span></span> are not too high).</p></li></ol><p id="p-20">If only one parent gender is being assigned (and the other parent is unknown or not genotyped), then any of the steps above that involve using both parent assignments together will not be relevant. Even though one of the methods (EMM or relatedness) must be chosen to find the best parent, thresholds for both methods can be used jointly to discard dubious assignments.</p><p id="p-21">The methods presented here have been incorporated into R code available at <a href="https://github.com/AgResearch/KGD">https://github.com/AgResearch/KGD</a>. This code is more efficient at calculating relatedness than EMM, and so the relatedness approach would be preferable if the performance of the two approaches is otherwise similar. For the relatedness approach, a bootstrapping method is investigated to aid step 4. The SNP markers are sampled with replacement to obtain a bootstrap sample with the same number of SNPs and the relatedness values recalculated. The proportion of times across many (<em>e.g.</em>, 1000) bootstrap samples that the most related parent has a higher estimated relatedness (with the offspring) than the second most related parent is denoted as the bootstrap support for the assignment.</p><p id="p-22">In step 5 when using relatedness, a low value of <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3239/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3239/embed/mml-math-11.gif"/></noscript></span></span> might also be rejected. In practice, it should suffice to place an upper limit on this value, as the likely errors are when one parent is correctly assigned, and the other parent is incorrectly assigned, due to its high relatedness with the correctly assigned parent.</p></div><div id="sec-5" class="subsection"><h3>Alternative allele sampling models</h3><p id="p-23">There may be models that are better for modeling the data than the binomial model, for example ones where allele reads exhibit clustering such that observed homozygosity is higher (when depth exceeds one) than in the random reads case. The calculations here require <em>K</em>, the probability of all A alleles (no B alleles) for a true AB genotype, and only consider models where this is the same as the probability of all B alleles. In particular, <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3239/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3239/embed/mml-math-12.gif"/></noscript></span></span> when <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3239/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3239/embed/mml-math-13.gif"/></noscript></span></span>.</p><p id="p-24">One possible alternative model is the beta-binomial (BB) model, which is often used to model data that are more dispersed than a binomial, for example, where there is some hidden clustering of the sampling process. The probability of seeing <em>m</em> B alleles with depth <em>k</em> is<span class="disp-formula" id="disp-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3239/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3239/embed/mml-math-14.gif"/></noscript></span></span>where B is the beta function. The mean of the distribution, with <em>k</em> = 1, is α /(α + β) which is equal to <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3239/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3239/embed/mml-math-15.gif"/></noscript></span></span> when β = α. Making this substitution and setting <em>m</em> = 0,<span class="disp-formula" id="disp-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3239/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3239/embed/mml-math-16.gif"/></noscript></span></span>where Γ denotes the gamma function. This model approaches the binomial model as α approaches infinity.</p><p id="p-25">The second alterative model considered assumes that the sampling process for reads for a particular SNP is Markovian – the probability of reading a particular allele depends on which allele was seen for the previous read (of that SNP). We denote the probability of seeing the same allele as was previously read as <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3239/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3239/embed/mml-math-17.gif"/></noscript></span></span>. The binomial model is a special case with <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3239/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3239/embed/mml-math-18.gif"/></noscript></span></span>=0.5. We refer to this as the ‘modified <em>p</em>’ (MP) model. As noted above, the probability of seeing a particular allele for the first read (the only read if <em>k</em> = 1) is 0.5. Because we are only interested in the cases where all the reads are the same, we do not actually need to know the order that the alleles were read. For this model<span class="disp-formula" id="disp-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3239/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3239/embed/mml-math-19.gif"/></noscript></span></span>Examples of the relationship between <em>K</em> and <em>k</em> for these models are shown in Figure S1. Both these models need the value of an extra parameter. Here we estimate the parameter by finding a set of parent-progeny trios that are assumed correct based on the binomial model (which is more conservative for calculating expected mismatch rates). The sum of squares of the deviations of the mismatch rates from the expected mismatch rates for these trios is then minimized (using the <em>optimize</em> function in R) with respect to the unknown parameter.</p><p id="p-26">The alternative sampling models will alter the expected mismatch rates and the self-relatedness (1+<em>F</em>) estimates, but not the relatedness estimates between individuals, as these do not depend on <em>K</em> (<a id="xref-ref-9-3" class="xref-bibr" href="#ref-9">Dodds <em>et al.</em> 2015</a>).</p></div><div id="sec-6" class="subsection"><h3>Animals</h3><p id="p-27">The methods have been applied in two herds of deer, part of Focus Genetics’ breeding program (<a href="http://www.focusgenetics.com">www.focusgenetics.com</a>). One was a Red deer (<em>Cervus elaphus</em>) breeding herd and the other was a Wapiti (also known as Elk; <em>Cervus canadensis</em>) breeding herd. The animals were managed in accordance with the provisions of the New Zealand Animal Welfare Act 1999, and the Codes of Welfare developed under sections 68–79 of the Act. Even though Red deer and Elk are considered different species (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Pitra <em>et al.</em> 2004</a>), they are capable of inter-breeding and produce fertile progeny. We will refer to them as breeds here, as that is how they are considered in New Zealand deer farming. Although the herds are not necessarily purebred, they are denoted here by their predominant breed.</p><p id="p-28">There were 1272 Red deer successfully genotyped by GBS (mean read depth of at least 0.3 over all SNPs called). These consisted of 571 progeny, born in 2015, 34 potential sires (none missing) and 667 potential dams (10 missing, <em>i.e.</em>, unavailable or not successfully genotyped). There were 709 Wapiti successfully genotyped by GBS consisting of 246 progeny, born in 2015, 41 potential sires (none missing) and 422 potential dams (two missing). There may be additional sires and dams that were not sampled. This resource had also been genotyped using a panel of 16 microsatellite markers, a commercial in-house deer parentage panel used by GenomNZ (<a href="http://www.genomnz.co.nz">www.genomnz.co.nz</a>) and parentage assignments made based on those genotypes.</p></div><div id="sec-7" class="subsection"><h3>GBS Genotyping</h3><p id="p-29">Genotyping was undertaken based on the method of <a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Elshire <em>et al.</em> (2011)</a> as described in <a id="xref-ref-9-4" class="xref-bibr" href="#ref-9">Dodds <em>et al.</em> (2015)</a> apart from the following minor differences. DNA extraction was from ear-punch tissue. The GBS-libraries were prepared utilizing the <em>Pst</em>I restriction enzyme. Either 96 or 192 samples were run per lane on an Illumina HiSeq2500 to obtain 100bp single end reads. The genotypes were called without mapping to a reference assembly.</p></div><div id="sec-8" class="subsection"><h3>Analysis</h3><p id="p-30">Allele frequencies were estimated using allele counts across the relevant group of individuals. Relatedness estimates were calculated using the methods of <a id="xref-ref-9-5" class="xref-bibr" href="#ref-9">Dodds <em>et al.</em> (2015)</a>, including the removal of SNPs with Hardy-Weinberg disequilibrium (observed frequency of the reference allele homozygote minus its expected value) below -0.05. Population structure was visualized by plotting the first two components of a principal components analysis of the genomic relatedness matrix (GRM, the matrix of estimated relatedness between all pairs of individuals including self-relatedness). Additional calculations were made and used for assigning parentage as described above. Parentage calculations were made twice, first using all animals in the dataset, using combined allele frequencies and then separately for each breed, using breed specific allele frequencies. SNPs which had a minor allele frequency (MAF) of zero in the breed-specific set were discarded for that breed.</p></div><div id="sec-9" class="subsection"><h3>Data availability</h3><p id="p-31">The GBS data are available in the file HapMap.hmc.txt.gz included in the supplemental material available at FigShare: <a href="https://doi.org/10.25387/g3.9243167">https://doi.org/10.25387/g3.9243167</a>.</p></div></div><div class="section results" id="sec-10"><h2 class="">Results</h2><div id="sec-11" class="subsection"><h3>Combined breed analysis</h3><p id="p-32">Unless stated otherwise, the relatedness has been used to determine best matching parents. The sequencing and SNP calling resulted in 78,042 SNPs for analysis with a mean read depth of 3.6 and call rate of 72%. The fin plot, which shows apparent Hardy-Weinberg disequilibrium plotted against minor allele frequencies (MAFs), is shown in Figure S2. SNPs with Hardy-Weinberg disequilibrium below -0.05 tended to have high depth and near minimum disequilibrium and may represent reads from duplicated regions. These were removed from further analysis, leaving 77,473 SNPs. The distribution of minor allele frequencies (MAFs) of these SNPs is shown in Figure S3 and shows that a high proportion of SNPs had low MAF. The first two principal components of the GRM are shown in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>. The first component, explains 96.9% of the variation, while the second component explains 0.3%.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/10/3239/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Principal components analysis. First two principal components from a principal components analysis of the GRM for the combined breed data." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-80386170" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Principal components analysis. First two principal components from a principal components analysis of the GRM for the combined breed data.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/10/3239/F1.medium.gif" width="440" height="427"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/9/10/3239/F1.medium.gif" width="440" height="427"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/10/3239/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/10/3239/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494434" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-33" class="first-child">Principal components analysis. First two principal components from a principal components analysis of the GRM for the combined breed data.</p><div class="sb-div caption-clear"></div></div></div><p id="p-34"><a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a> shows the relatedness between each progeny and the best matching sire (<em>i.e.</em>, the individual in the sire’s group with the highest relatedness with that progeny). Also plotted is the raw mismatch rate, <em>i.e.</em>, the proportion of SNPs whose raw calls are inconsistent with parentage. A plausible relatedness threshold for declaring parentage is a value near 0.5, perhaps a little less than 0.5 to allow for errors in the estimation process. A threshold of 0.4 is shown in <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a> (and other relevant plots) as an initial threshold. All but 11 of the progeny have a best matching sire relatedness greater than 0.4, while only two were between 0.4 and 0.5. The other values are considerably higher, averaging 0.61 for Red deer and 0.84 for Wapiti. The raw mismatch rates among those above the 0.4 relatedness threshold vary greatly, ranging from 0.011 to 0.065. Raw mismatch rates for the other progeny were all greater than 0.040. <a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2B</a> shows a similar plot, but with excess mismatch rate (EMM) on the vertical axis. Here there is a clear relationship between relatedness and EMM. All progeny with a best relatedness of at least 0.5 have an EMM below 0.007 while the other progeny have an EMM above 0.019. This suggests an EMM threshold between 0.01 and 0.015, and a relatedness threshold of around 0.5 for declaring parentage, although the EMM threshold combined with any lower relatedness threshold does not change the assignments in these data. Here an EMM threshold of 0.01 is shown in the figures and used for assignments.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/10/3239/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Mismatch rates vs. relatedness for combined breed analysis. Raw mismatch rate (A, C) or excess mismatch rate (B, D) vs. estimated relatedness between the best matching sire (A, B) or dam (C, D) and each progeny. The vertical and horizontal gray lines denote the relatedness and excess mismatch rate thresholds, respectively, for declaring parentage. In B and D, points corresponding to cases where another potential parent has offspring-parent relatedness within 0.05 of plotted value are shown with filled symbols in a brighter shade of their breed color; those not reaching the 0.99 bootstrap support threshold are shown as triangles." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-80386170" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Mismatch rates &lt;em&gt;vs.&lt;/em&gt; relatedness for combined breed analysis. Raw mismatch rate (A, C) or excess mismatch rate (B, D) &lt;em&gt;vs.&lt;/em&gt; estimated relatedness between the best matching sire (A, B) or dam (C, D) and each progeny. The vertical and horizontal gray lines denote the relatedness and excess mismatch rate thresholds, respectively, for declaring parentage. In B and D, points corresponding to cases where another potential parent has offspring-parent relatedness within 0.05 of plotted value are shown with filled symbols in a brighter shade of their breed color; those not reaching the 0.99 bootstrap support threshold are shown as triangles.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/10/3239/F2.medium.gif" width="434" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/9/10/3239/F2.medium.gif" width="434" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/10/3239/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/10/3239/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494377" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-35" class="first-child">Mismatch rates <em>vs.</em> relatedness for combined breed analysis. Raw mismatch rate (A, C) or excess mismatch rate (B, D) <em>vs.</em> estimated relatedness between the best matching sire (A, B) or dam (C, D) and each progeny. The vertical and horizontal gray lines denote the relatedness and excess mismatch rate thresholds, respectively, for declaring parentage. In B and D, points corresponding to cases where another potential parent has offspring-parent relatedness within 0.05 of plotted value are shown with filled symbols in a brighter shade of their breed color; those not reaching the 0.99 bootstrap support threshold are shown as triangles.</p><div class="sb-div caption-clear"></div></div></div><p id="p-36">The corresponding relatedness plots for the best matching dams are shown in <a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2C</a> (with raw mismatch rates) and <a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2D</a> (with EMMs). These show a similar pattern to the sire plots. Pairs with an EMM less than 0.009 all had relatedness greater than 0.4 (all but one were greater than 0.49), suggesting relatedness and EMM thresholds of around 0.4 and 0.009, respectively as appropriate threshold for declaring parentage.</p><p id="p-37">One sire assignment and five dam assignments, passing the relatedness (0.4) and EMM (0.01) thresholds, had another potential parent with offspring-parent relatedness within 0.05 of the best matching pair (see Figure S4). If a conservative threshold of 0.99 is used for bootstrap support, three of the dam assignments are rejected, while the other four assignments are retained.</p><p id="p-38">When considering both parent matches together, there were three assignments where the trio EMM exceeded 0.02. It is likely that a higher threshold is appropriate for the trio than pair EMM thresholds, as an error in any of the three individual’s genotypes may generate a mismatch, and so 0.02 is used here as the threshold. For one of these progeny, the second best matching sire and best matching dam gave a low trio EMM (0.006) and this combination passed the other parentage thresholds. Two best sire and dam match trios that passed the relatedness and EMM threshold for each offspring-parent pair had parent relatedness exceeding twice the offspring inbreeding by more than 0.2 (the ‘inbreeding threshold’ being used here to reject a parent pair). One of these was the trio just mentioned (where the second best sire and best dam combination appeared better); the other passed the trio EMM threshold but had a lower EMM with the best sire and second best dam.</p><p id="p-39">There was only one case where the lowest EMM parent was not one of the two highest related parents. This was for a dam of one of the Red calves. The highest related dam was excluded with the EMM threshold. In this case, the microsatellite-based analysis did not give a dam assignment.</p></div><div id="sec-12" class="subsection"><h3>Separate breed analysis</h3><p id="p-40">The combined breed analysis showed quite different relatedness values in the two breeds. In particular, relatedness to the best matching parent for Wapiti is high. The separate breed analysis uses allele frequencies from within each breed, and this may be more appropriate. There were 69,223 SNPs remaining polymorphic within the Wapiti herd and 76,926 in the Red deer herd. Mean SNP depths were similar to the combined analysis (3.8 and 3.3 for Wapiti and Red deer, respectively).</p><p id="p-41"><a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a> shows the excess mismatch rate and relatedness between each progeny and the best matching sire and dam for the separate breeds analyses. It appears that the thresholds of 0.4 for relatedness and 0.01 for EMM (shown on the plots), or slightly more lenient thresholds, are suitable for declaring parentage. For sire assignment, one Wapiti progeny fails this relatedness threshold (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3A</a>), while 13 Red progeny fail both thresholds (none failing only one threshold, (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3C</a>). For dam assignment, these same thresholds excluded four Wapiti progeny (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3B</a>; one failing both thresholds, three failing only the relatedness threshold) and 28 Red progeny (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3D</a>; 24 failing both thresholds, one failing the EMM threshold only and three failing the relatedness threshold only). Most progeny that failed on only one of the thresholds were close to that threshold.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/10/3239/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Excess mismatch rate vs. relatedness for separate breeds analyses. Excess mismatch rate vs. estimated relatedness for the best matching sire (A and C) and dam (B and D) with each progeny for the Wapiti (A and B) and Red deer (C and D) analyses." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-80386170" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Excess mismatch rate &lt;em&gt;vs.&lt;/em&gt; relatedness for separate breeds analyses. Excess mismatch rate &lt;em&gt;vs.&lt;/em&gt; estimated relatedness for the best matching sire (A and C) and dam (B and D) with each progeny for the Wapiti (A and B) and Red deer (C and D) analyses.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/10/3239/F3.medium.gif" width="434" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/9/10/3239/F3.medium.gif" width="434" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/10/3239/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/10/3239/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494532" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-42" class="first-child">Excess mismatch rate <em>vs.</em> relatedness for separate breeds analyses. Excess mismatch rate <em>vs.</em> estimated relatedness for the best matching sire (A and C) and dam (B and D) with each progeny for the Wapiti (A and B) and Red deer (C and D) analyses.</p><div class="sb-div caption-clear"></div></div></div><p id="p-43">All these Wapiti assignments passed the trio EMM threshold, but one failed the inbreeding threshold (parent relatedness near zero, but inbreeding just below -0.1). In the Red herd, there were six trios (passing all the single parent thresholds) that failed the trio EMM threshold. One of these also failed the inbreeding threshold. For this progeny the second best sire along with the best dam formed a trio that passed all the thresholds (as was the case for this progeny in the combined breed analysis).</p><p id="p-44">Using the provisional thresholds discussed here, 13 Red deer did not get a sire assignment and 1 Wapiti and 25 Red deer did not get a dam assignment in the combined analysis while 13 Red deer and 1 Wapiti did not get a sire assignment and 4 Wapiti and 28 Red deer did not get a dam assignment in the separate breed analysis. All progeny that had a parent assigned in either of the combined breed or separate breed analysis had the same best matching parent in the other analysis (even if it did not pass the thresholds) except for one Wapiti mother.</p></div><div id="sec-13" class="subsection"><h3>Comparison with microsatellite-based parentage assignments</h3><p id="p-45">The separate breed analysis gave relatedness values more in line with that expected and the same thresholds appeared suitable for both breeds. Therefore, results from the separate breed analysis are used for comparison with the microsatellite results.</p><p id="p-46">There were 793 progeny with a sire assignment (after applying all criteria) using GBS; two were assigned a different sire and 791 had the same sire assigned using microsatellites. There were 775 progeny with a dam assignment using GBS. Using microsatellites ten of these were unassigned, two were assigned a different dam and 763 had the same dam assigned.</p><p id="p-47">Eight progeny that passed the single parent thresholds were above the trio EMM (with GBS). Of these, one had a different dam assignment and one had a different sire assignment with microsatellites, compared to the best matching parents with GBS. The one with the different sire assignment also failed the inbreeding test, while the trio with the second best matching sire (the one assigned using the microsatellite test) passed the EMM and inbreeding tests. One additional trio failed the inbreeding threshold but passed the EMM threshold; this trio was assigned by the microsatellite test.</p></div><div id="sec-14" class="subsection"><h3>Alternative allele sampling models</h3><p id="p-48"><a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a> compares raw and expected mismatch rates. The raw mismatch rates are above the expected rates for the majority of trios, including those we have accepted as representing parent-progeny trios (shown with ‘Assign code’ Y). For these trios there is a strong relationship (roughly linear) between raw and expected values, suggesting a systematic reason for the difference. To investigate this further, two alternative allele sampling models (BB and MP) were considered.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/10/3239/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Raw vs. expected mismatch rates. Comparison of raw and expected parent-offspring trio mismatch rates for the Wapiti (A) and Red deer (B) analyses. The red lines show where these are equal. The gray lines show the threshold used for excluding a trio from parentage. Assign codes are Y: assign parentage, A: an alternate parentage has lower EMM, I: fails the inbreeding criterion, E: exclude based on trio EMM, F: assign father only, M: assign mother only, N: do not assign either parent." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-80386170" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Raw &lt;em&gt;vs.&lt;/em&gt; expected mismatch rates. Comparison of raw and expected parent-offspring trio mismatch rates for the Wapiti (A) and Red deer (B) analyses. The red lines show where these are equal. The gray lines show the threshold used for excluding a trio from parentage. Assign codes are Y: assign parentage, A: an alternate parentage has lower EMM, I: fails the inbreeding criterion, E: exclude based on trio EMM, F: assign father only, M: assign mother only, N: do not assign either parent.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/10/3239/F4.medium.gif" width="440" height="225"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/9/10/3239/F4.medium.gif" width="440" height="225"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/10/3239/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/10/3239/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494481" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-49" class="first-child">Raw <em>vs.</em> expected mismatch rates. Comparison of raw and expected parent-offspring trio mismatch rates for the Wapiti (A) and Red deer (B) analyses. The red lines show where these are equal. The gray lines show the threshold used for excluding a trio from parentage. Assign codes are Y: assign parentage, A: an alternate parentage has lower EMM, I: fails the inbreeding criterion, E: exclude based on trio EMM, F: assign father only, M: assign mother only, N: do not assign either parent.</p><div class="sb-div caption-clear"></div></div></div><p id="p-50">The estimated values of β (for the BB model) were 3.96 for the Red herd and 4.61 for the Wapiti herd. The estimated values of <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3239/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10/3239/embed/mml-math-20.gif"/></noscript></span></span> (for the MP model) were 0.604 for the Red herd and 0.591 for the Wapiti herd. For both herds the BB model gave a lower sum of squares than the MP model. <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a> shows raw mismatch rates compared to expected rates calculated with the BB model for the Red herd. All four herd x model combinations are given in Figure S5.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/10/3239/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Raw vs. expected mismatch rates using the BB model. Comparison of raw and expected parent-offspring trio mismatch rates for the Red deer analysis using the BB model. The red line shows where raw and expected rates are equal. The gray line shows the threshold used for excluding a trio from parentage. Assign codes shown are from the analysis using the binomial model; see Figure 4 caption for explanation of these symbols." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-80386170" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Raw &lt;em&gt;vs.&lt;/em&gt; expected mismatch rates using the BB model. Comparison of raw and expected parent-offspring trio mismatch rates for the Red deer analysis using the BB model. The red line shows where raw and expected rates are equal. The gray line shows the threshold used for excluding a trio from parentage. Assign codes shown are from the analysis using the binomial model; see Figure 4 caption for explanation of these symbols.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/10/3239/F5.medium.gif" width="440" height="427"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/9/10/3239/F5.medium.gif" width="440" height="427"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/10/3239/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/10/3239/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494475" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-51" class="first-child">Raw <em>vs.</em> expected mismatch rates using the BB model. Comparison of raw and expected parent-offspring trio mismatch rates for the Red deer analysis using the BB model. The red line shows where raw and expected rates are equal. The gray line shows the threshold used for excluding a trio from parentage. Assign codes shown are from the analysis using the binomial model; see <a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a> caption for explanation of these symbols.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-15"><h2 class="">Discussion</h2><p id="p-52">We have investigated methods for assigning parentage based on low depth sequencing data, such as can be produced using GBS. The usual approach to this is to filter genotype calls based on read depth and thereafter assume that genotypes are called with high accuracy. For the example deer data, if the SNPs were filtered as in <a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Thrasher <em>et al.</em> (2018)</a> but without the MAF or Hardy-Weinberg filters (<em>i.e.</em>, read depth of at least 10, SNP call rate on remaining results of at least 95%,) this would leave 131 SNPs. Only nine of these 131 had Hardy Weinberg disequilibrium &gt; -0.05 and those nine all had a MAF below the threshold (0.25) used by <a id="xref-ref-24-3" class="xref-bibr" href="#ref-24">Thrasher <em>et al.</em> (2018)</a>. Some software packages (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Marshall <em>et al.</em> 1998</a>) allow genotype calls with errors, using a supplied error rate that is constant across all genotypes. However here the actual error rate (due to allele sampling) is dependent on the genotype call and its read depth.</p><p id="p-53">Modeling the sequencing process, analogous to that used by <a id="xref-ref-9-6" class="xref-bibr" href="#ref-9">Dodds <em>et al.</em> (2015)</a> for relatedness estimation, <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Bilton <em>et al.</em> (2018b)</a> for linkage analysis and <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Bilton <em>et al.</em> (2018a)</a> for linkage disequilibrium estimation, allows depth-dependent errors. Recently, <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Whalen <em>et al.</em> (2019)</a> have adopted this approach to develop likelihood-based methods for relationship classification (including parentage). Here, we have considered the use of relatedness estimates using the KGD method of <a id="xref-ref-9-7" class="xref-bibr" href="#ref-9">Dodds <em>et al.</em> (2015)</a> and expected mismatch rates. Both these statistics model the probability of observing both alleles for a heterozygote in a depth-dependent way assuming known allele frequencies. Excess mismatch rates (EMM) are then calculated as the difference between raw (assuming the genotype is given by the alleles observed) and expected rates. The EMM is similar to mismatch rates used in exclusion-based parentage assignment methods that allow for some genotyping error. The EMM could be calculated after filtering on read depth, but as it accounts for depth, it allows all SNPs (not filtered out for other reasons) to be used. In practice, it may suffice to calculate the EMM only for the leading candidate parents, based on relatedness estimates. This leads to a computationally efficient approach, as estimated relatedness between pairs can be calculated faster than the EMM (approximately 10x faster using current software). The analysis could be undertaken using some other method to estimate relatedness from low-depth sequencing data, such as those implemented in PLINK (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Chang <em>et al.</em> 2015</a>), mapgd (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Ackerman <em>et al.</em> 2017</a>) or ANGSD (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Hanghøj <em>et al.</em> 2019</a>). In some cases, for example with a small dataset and where inbreeding is unlikely, allele frequency-free methods such as those in <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Hanghøj <em>et al.</em> (2019)</a> may help clarify parent-offspring relationships, but does not include a method for checking the consistency of trios.</p><p id="p-54">The analysis of the deer data showed that the raw mismatch rate was not a reliable indicator of parentage – there was considerable overlap of these values among pairs that were either consistent or not with parentage, based on relatedness estimates. In contrast, it was possible to find relatedness and EMM thresholds for declaring parentage that were almost fully concordant, with only one case where the two methods gave different conclusions. This case did have a low EMM (-0.005) but with a low relatedness (0.27) for the best mother match in the Red deer analysis. The same case was less of an outlier in the combined breed analysis (EMM of -0.009, relatedness of 0.41), showing that the results are dependent to some degree on the allele frequencies used.</p><p id="p-55">The choice of allele frequencies used influences the measures to determine parentage. The theory assumes that allele frequencies for the ‘base population’ are known. The two breeds examined in this study were clearly separated by the 1<sup>st</sup> principal component (a few intermediate Red deer probably had some Wapiti ancestry). For the combined breed analysis, we used estimates based on the total allele counts of all animals genotyped in the study. This resulted in high relatedness estimates within herds (for example, as demonstrated by average relatedness between progeny and best matching sire of 0.84 and 0.60 for the Wapiti and Red herds, respectively). The allele frequencies used may not be a very good representation of the allele frequencies before breed differentiation, due to genetic drift within these two breeds. For example, there are negative estimates of relatedness. For the within herd analysis, the relatedness values dropped, reflecting that the allele frequencies relate to the genetic variation within each herd. The within herd estimates between progeny and best matching parent are more in the range that would be expected for parent-offspring pairs with unrelated parents. Therefore, choosing an appropriate base population for allele frequency estimation may help in choosing thresholds that seem sensible. A within-breed analysis would normally be preferred. Despite the differences in the relatedness estimates using the within breed compared to combined breed allele frequencies, the two analyses were generally consistent in terms of parent assignment. Using a set of provisional thresholds, any time an assignment was made in both analyses it was to the same parent. This would still be the case if the relatedness threshold in the separate breed analyses was dropped to 0.35 (7 more assignments). There was one case of a combined analysis mother assignment which had a different best matching mother in the Wapiti analysis. For this case the EMM was 0.0094, close to the threshold (0.01). A slightly lower EMM threshold may be more appropriate in the combined breed analysis (<a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2</a>). Even among cases not assigned by either analysis, most (31 out of 39) had the same best matching parents. These observations suggest that if appropriate thresholds are chosen, the assignments may be reasonably consistent with respect to the use of different allele frequencies. The method requires appropriate thresholds which can usually be determined when there are sufficient numbers of progeny, similar to what <a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Moore <em>et al.</em> (2019)</a> found using genomic relatedness from SNP chip data.</p><p id="p-56">The estimated inbreeding in the offspring plus half that of the putative parent could be subtracted from the parent-offspring relatedness values to adjust for inbreeding. This could change which possible parent has the highest relatedness match. It may bring the values closer to the value of 0.5, as expected for non-inbreds. However, estimated inbreeding is generally less precise than between individual relatedness estimates (<a id="xref-fig-3-6" class="xref-fig" href="#F3">Figure 3</a>. of <a id="xref-ref-9-8" class="xref-bibr" href="#ref-9">Dodds <em>et al.</em> 2015</a>), especially with low depth sequencing, so the adjusted relatedness would have more variability than the original estimate. In addition, breeding programs usually aim to keep levels of inbreeding low and so there may be little variation in inbreeding relative to the assumed base population. In the combined breed analysis, where the population is genetically diverse and subdivided, using the adjusted relatedness approach gave best matching relatedness values mostly between 0.3 and 0.5 (data not shown). Although the approach reduced the variation in these values, it appears to have over-corrected the relatedness estimates in this case.</p><p id="p-57">The (within breed) GBS-based assignments made here were largely consistent with assignments based on a microsatellite parentage panel. There were four cases of a different parent assignment. Two were the only ones assigned to a particular sire using microsatellites while this sire had no progeny assigned with GBS. Different DNA samples from the sire were used for microsatellites and GBS, so it is possible there was a sample mix-up. The other difference was for a mother assignment, with no clear reason for the difference; a sample mis-identification is possible. The use of trio information (trio EMM and the inbreeding test) was found to be useful in reducing the number of mismatches between the methods, and such information should be used where possible. These tests help prevent false assignments where a true parent is missing, but a close relative of the other parent of the offspring meets the relatedness and single parent EMM thresholds (<em>e.g.</em>, helps avoid assigning a paternal aunt as the mother). A comparison of the GBS-based assignment against recorded parentage, for a sheep example, is given in the Supplementary material where the GBS-based assignment gave one difference to the recorded parentage. This difference was also found using a SNP-chip based parentage check and is a likely parentage recording error.</p><p id="p-58">The EMM tended to be greater than zero, even for cases which appeared to be correct matches. A possible reason for this is that the alleles at a locus are not sampled randomly but there tends to be clustering in the alleles recruited for sequencing. This could be due, for example, to PCR artifacts such as “stacking” (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Andrews <em>et al.</em> 2016</a>) or suboptimal ratios of reagents (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Ott <em>et al.</em> 2017</a>). Two alternative allele sampling models were proposed and applied, the beta-binomial model and a model where the first allele is sampled at random, but subsequently the same allele is more likely to be sequenced than the other allele. Both models improved the EMM values by similar amounts, such that the EMM tended to be close to zero for what appeared to be correct matches, with the beta-binomial model giving a better fit. The use of either of these models would allow tighter EMM thresholds and may lead to more accurate parentage assignments. These models could also be applied in other GBS analyses, such as the estimation of inbreeding (<a id="xref-ref-9-9" class="xref-bibr" href="#ref-9">Dodds <em>et al.</em> 2015</a>), linkage (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Bilton <em>et al.</em> 2018b</a>), linkage disequilibrium (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Bilton <em>et al.</em> 2018a</a>), calculating genotype likelihoods for downstream analyses (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Korneliussen <em>et al.</em> 2014</a>) or predicting gender (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bilton <em>et al.</em> 2019</a>). More work is required to evaluate whether these differences are important, whether the alternate models are significantly better than the random sampling model or whether there are other models (for example, one which allows sequencing error) which are more realistic and provide a better fit.</p><p id="p-59">The developments described here provide methods and guidelines for assigning parentage from sequencing data. These methods have been developed with reduced representational low depth sequencing in mind. This allows sequencing costs to be as low as possible making it a possibility for parentage analysis, perhaps in combination with other genomic analyses. The sequencing approach allows genomic resources to be developed with low start-up costs compared to other approaches. The parentage assignment process requires some choice of appropriate thresholds, but these are usually clear-cut. A possible exception is where the population is genetically diverse with distinct subgroups, and there are different possible approaches to calculating allele frequencies to use in the process. When applied to a deer dataset, the parentage assignment was largely consistent with assignments made using a microsatellite panel; the few differences seen may have been due to sample tracking errors. Therefore, GBS or other sequencing-based methods can be used for parentage assignments, increasing the utility of these sequencing methods.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-60">This project was supported by the Ministry of Business, Innovation and Employment via its funding of the “Genomics for Production &amp; Security in a Biological Economy” program (Contract ID C10X1306) and FarmIQ (Ministry for Primary Industries’ Primary Growth Partnership fund) – FIQ Systems – Plate to Pasture (Reference: PGP06-09020). We thank Focus Genetics (New Zealand) and Pāmu (formerly Landcorp Farming Ltd) (New Zealand) for the use of their animals and genotypes for testing the proposed methods. The animals used in the sheep example were funded by the Pastoral Greenhouse Gas Research Consortium, Sustainable Land Management and Climate Change and the New Zealand Agricultural Greenhouse Gas Research Centre. The animals themselves were part of the Ovita partnership and the related Central Progeny Test: both funded in part or whole by Beef+Lamb New Zealand. We thank Jeanne Jacobs, Timothy Bilton, the <em>G3</em> editorial staff and anonymous reviewers for their helpful comments.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-61">Supplemental material available at FigShare: <a href="https://doi.org/10.25387/g3.9243167">https://doi.org/10.25387/g3.9243167</a>.</p></li><li class="fn" id="fn-2"><p id="p-62"><em>Communicating editor: A. Sethuraman</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2019-07-03"><span class="received-label">Received </span>July 3, 2019.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2019-07-29"><span class="accepted-label">Accepted </span>July 29, 2019.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2019 Dodds <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3239.1" data-doi="10.1534/genetics.116.190660"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ackerman</span>,  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Johri</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Spitze</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Xu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. G.</span> <span class="cit-name-surname">Doak</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Estimating seven coefficients of pairwise relatedness using population-genomic data.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">206</span>: <span class="cit-fpage">105</span>–<span class="cit-lpage">118</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.116.190660</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.116.190660%26rft_id%253Dinfo%253Apmid%252F28341647%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjA2LzEvMTA1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMC8zMjM5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3239.2" data-doi="10.3168/jds.2009-2730"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Aguilar</span>,  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Misztal</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. L.</span> <span class="cit-name-surname">Johnson</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Legarra</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Tsuruta</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Hot topic: A unified approach to utilize phenotypic, full pedigree, and genomic information for genetic evaluation of Holstein final score.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">93</span>: <span class="cit-fpage">743</span>–<span class="cit-lpage">752</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3168/jds.2009-2730</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DAguilar%26rft.auinit1%253DI.%26rft.volume%253D93%26rft.issue%253D2%26rft.spage%253D743%26rft.epage%253D752%26rft.atitle%253DHot%2Btopic%253A%2Ba%2Bunified%2Bapproach%2Bto%2Butilize%2Bphenotypic%252C%2Bfull%2Bpedigree%252C%2Band%2Bgenomic%2Binformation%2Bfor%2Bgenetic%2Bevaluation%2Bof%2BHolstein%2Bfinal%2Bscore.%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2009-2730%26rft_id%253Dinfo%253Apmid%252F20105546%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2009-2730&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20105546&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3239.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000274102500032&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3239.3" data-doi="10.1038/nrg.2015.28"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Andrews</span>,  <span class="cit-name-given-names">K. R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">Good</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. R.</span> <span class="cit-name-surname">Miller</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Luikart</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. A.</span> <span class="cit-name-surname">Hohenlohe</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Harnessing the power of RADseq for ecological and evolutionary genomics.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">81</span>–<span class="cit-lpage">92</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg.2015.28</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BGenet.%26rft.volume%253D17%26rft.spage%253D81%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg.2015.28%26rft_id%253Dinfo%253Apmid%252F26729255%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg.2015.28&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26729255&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3239.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3239.4" data-doi="10.1371/journal.pone.0003376"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Baird</span>,  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. D.</span> <span class="cit-name-surname">Etter</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. S.</span> <span class="cit-name-surname">Atwood</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. C.</span> <span class="cit-name-surname">Currey</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. L.</span> <span class="cit-name-surname">Shiver</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Rapid SNP discovery and genetic mapping using sequenced RAD markers.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">3</span>: <span class="cit-elocation-id">e3376</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0003376</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DBaird%26rft.auinit1%253DN.%2BA.%26rft.volume%253D3%26rft.issue%253D10%26rft.spage%253De3376%26rft.epage%253De3376%26rft.atitle%253DRapid%2BSNP%2Bdiscovery%2Band%2Bgenetic%2Bmapping%2Busing%2Bsequenced%2BRAD%2Bmarkers.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0003376%26rft_id%253Dinfo%253Apmid%252F18852878%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0003376&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18852878&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3239.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3239.5" data-doi="10.1111/age.12782"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bilton</span>,  <span class="cit-name-given-names">T. P.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. J.</span> <span class="cit-name-surname">Chappell</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. M.</span> <span class="cit-name-surname">Clarke</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Brauning</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. G.</span> <span class="cit-name-surname">Dodds</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">Using genotyping-by-sequencing to predict gender in animals.</span> <abbr class="cit-jnl-abbrev">Anim. Genet.</abbr> <span class="cit-vol">50</span>: <span class="cit-fpage">307</span>–<span class="cit-lpage">310</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/age.12782</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnim.%2BGenet.%26rft.volume%253D50%26rft.spage%253D307%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fage.12782%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/age.12782&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3239.6" data-doi="10.1534/genetics.118.300831"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bilton</span>,  <span class="cit-name-given-names">T. P.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. C.</span> <span class="cit-name-surname">McEwan</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. M.</span> <span class="cit-name-surname">Clarke</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Brauning</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. C.</span> <span class="cit-name-surname">Van Stijn</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span><span class="cit-comment">a</span> <span class="cit-article-title">Linkage disequilibrium estimation in low coverage high-throughput sequencing data.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">209</span>: <span class="cit-fpage">389</span>–<span class="cit-lpage">400</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.118.300831</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.118.300831%26rft_id%253Dinfo%253Apmid%252F29588288%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjA5LzIvMzg5IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMC8zMjM5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3239.7" data-doi="10.1534/genetics.117.300627"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bilton</span>,  <span class="cit-name-given-names">T. P.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. R.</span> <span class="cit-name-surname">Schofield</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. A.</span> <span class="cit-name-surname">Black</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Chagné</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. L.</span> <span class="cit-name-surname">Wilcox</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span><span class="cit-comment">b</span> <span class="cit-article-title">Accounting for errors in low coverage high-throughput sequencing data when constructing genetic maps using biparental outcrossed populations.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">209</span>: <span class="cit-fpage">65</span>–<span class="cit-lpage">76</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.117.300627</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.117.300627%26rft_id%253Dinfo%253Apmid%252F29487138%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiMjA5LzEvNjUiO3M6NDoiYXRvbSI7czoxOToiL2dnZy85LzEwLzMyMzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3239.8" data-doi="10.1186/s13742-015-0047-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chang</span>,  <span class="cit-name-given-names">C. C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. C.</span> <span class="cit-name-surname">Chow</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L. C. A. M.</span> <span class="cit-name-surname">Tellier</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Vattikuti</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. M.</span> <span class="cit-name-surname">Purcell</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Second-generation PLINK: rising to the challenge of larger and richer datasets.</span> <abbr class="cit-jnl-abbrev">Gigascience</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">7</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s13742-015-0047-8</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGigascience%26rft.volume%253D4%26rft.spage%253D7%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13742-015-0047-8%26rft_id%253Dinfo%253Apmid%252F25722852%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13742-015-0047-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25722852&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3239.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3239.9" data-doi="10.1186/s12864-015-2252-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dodds</span>,  <span class="cit-name-given-names">K. G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. C.</span> <span class="cit-name-surname">McEwan</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Brauning</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. A.</span> <span class="cit-name-surname">Anderson</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. C.</span> <span class="cit-name-surname">Van Stijn</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Construction of relatedness matrices using genotyping-by-sequencing data.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">1047</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12864-015-2252-3</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D16%26rft.spage%253D1047%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864-015-2252-3%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12864-015-2252-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3239.10" data-doi="10.1071/EA05056"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dodds</span>,  <span class="cit-name-given-names">K. G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. A.</span> <span class="cit-name-surname">Sise</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. L.</span> <span class="cit-name-surname">Tate</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Practical aspects of a genetic evaluation system using parentage assigned from genetic markers.</span> <abbr class="cit-jnl-abbrev">Aust. J. Exp. Agric.</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">935</span>–<span class="cit-lpage">940</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1071/EA05056</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAust.%2BJ.%2BExp.%2BAgric.%26rft.volume%253D45%26rft.spage%253D935%26rft_id%253Dinfo%253Adoi%252F10.1071%252FEA05056%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1071/EA05056&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3239.11" data-doi="10.1371/journal.pone.0019379"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Elshire</span>,  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. C.</span> <span class="cit-name-surname">Glaubitz</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Q.</span> <span class="cit-name-surname">Sun</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. A.</span> <span class="cit-name-surname">Poland</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Kawamoto</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">6</span>: <span class="cit-elocation-id">e19379</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0019379</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DElshire%26rft.auinit1%253DR.%2BJ.%26rft.volume%253D6%26rft.issue%253D5%26rft.spage%253De19379%26rft.epage%253De19379%26rft.atitle%253DA%2Brobust%252C%2Bsimple%2Bgenotyping-by-sequencing%2B%2528GBS%2529%2Bapproach%2Bfor%2Bhigh%2Bdiversity%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0019379%26rft_id%253Dinfo%253Apmid%252F21573248%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0019379&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21573248&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3239.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3239.12" data-doi="10.1186/s12711-018-0397-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Grashei</span>,  <span class="cit-name-given-names">K. E.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Ødegård</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. H. E.</span> <span class="cit-name-surname">Meuwissen</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Using genomic relationship likelihood for parentage assignment.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">50</span>: <span class="cit-fpage">26</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12711-018-0397-7</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D50%26rft.spage%253D26%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12711-018-0397-7%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12711-018-0397-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3239.13" data-doi="10.1093/gigascience/giz034"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hanghøj</span>,  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Moltke</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. A.</span> <span class="cit-name-surname">Andersen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Manica</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. S.</span> <span class="cit-name-surname">Korneliussen</span></span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">Fast and accurate relatedness estimation from high-throughput sequencing data in the presence of inbreeding.</span> <abbr class="cit-jnl-abbrev">Gigascience</abbr> <span class="cit-vol">8</span>: <span class="cit-elocation-id">giz034</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/gigascience/giz034</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGigascience%26rft.volume%253D8%26rft.spage%253Dgiz034%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgigascience%252Fgiz034%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/gigascience/giz034&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3239.14" data-doi="10.1016/j.plantsci.2015.04.016"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>,  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Guo</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Kong</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Chandnani</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.-S.</span> <span class="cit-name-surname">Shuang</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Application of genotyping by sequencing technology to a variety of crop breeding programs.</span> <abbr class="cit-jnl-abbrev">Plant Sci.</abbr> <span class="cit-vol">242</span>: <span class="cit-fpage">14</span>–<span class="cit-lpage">22</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.plantsci.2015.04.016</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BSci.%26rft.volume%253D242%26rft.spage%253D14%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.plantsci.2015.04.016%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.plantsci.2015.04.016&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3239.15" data-doi="10.1186/s12859-014-0356-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Korneliussen</span>,  <span class="cit-name-given-names">T. S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Albrechtsen</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Nielsen</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">ANGSD: analysis of next generation sequencing data.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">356</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12859-014-0356-4</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BBioinformatics%26rft.volume%253D15%26rft.spage%253D356%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12859-014-0356-4%26rft_id%253Dinfo%253Apmid%252F25420514%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12859-014-0356-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25420514&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3239.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3239.16" data-doi="10.1016/j.livsci.2014.04.029"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Legarra</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">O. F.</span> <span class="cit-name-surname">Christensen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Aguilar</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Misztal</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Single Step, a general approach for genomic selection.</span> <abbr class="cit-jnl-abbrev">Livest. Sci.</abbr> <span class="cit-vol">166</span>: <span class="cit-fpage">54</span>–<span class="cit-lpage">65</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.livsci.2014.04.029</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLivest.%2BSci.%26rft.volume%253D166%26rft.spage%253D54%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.livsci.2014.04.029%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.livsci.2014.04.029&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3239.17" data-doi="10.1046/j.1365-294x.1998.00374.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Marshall</span>,  <span class="cit-name-given-names">T. C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Slate</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L. E. B.</span> <span class="cit-name-surname">Kruuk</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">Pemberton</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Statistical confidence for likelihood-based paternity inference in natural populations.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">639</span>–<span class="cit-lpage">655</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1046/j.1365-294x.1998.00374.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DMarshall%26rft.auinit1%253DT.%2BC.%26rft.volume%253D7%26rft.issue%253D5%26rft.spage%253D639%26rft.epage%253D655%26rft.atitle%253DStatistical%2Bconfidence%2Bfor%2Blikelihood-based%2Bpaternity%2Binference%2Bin%2Bnatural%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1365-294x.1998.00374.x%26rft_id%253Dinfo%253Apmid%252F9633105%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1365-294x.1998.00374.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9633105&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3239.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000073810200009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3239.18" data-doi="10.1093/jas/sky407"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Moore</span>,  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Vilela</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Kaseja</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Mrode</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Coffey</span></span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">Forensic use of the genomic relationship matrix to validate and discover livestock pedigrees.</span> <abbr class="cit-jnl-abbrev">J. Anim. Sci.</abbr> <span class="cit-vol">97</span>: <span class="cit-fpage">35</span>–<span class="cit-lpage">42</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/jas/sky407</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAnim.%2BSci.%26rft.volume%253D97%26rft.spage%253D35%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjas%252Fsky407%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/jas/sky407&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3239.19" data-doi="10.1038/ng.499"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ng</span>,  <span class="cit-name-given-names">S. B.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. J.</span> <span class="cit-name-surname">Buckingham</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Lee</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. W.</span> <span class="cit-name-surname">Bigham</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H. K.</span> <span class="cit-name-surname">Tabor</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Exome sequencing identifies the cause of a mendelian disorder.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">30</span>–<span class="cit-lpage">35</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.499</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D42%26rft.spage%253D30%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.499%26rft_id%253Dinfo%253Apmid%252F19915526%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.499&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19915526&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3239.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000273055100013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3239.20" data-doi="10.1093/nar/gkx853"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ott</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Liu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. C.</span> <span class="cit-name-surname">Schnable</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Yeh</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.-S.</span> <span class="cit-name-surname">Wang</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">tGBS® genotyping-by-sequencing enables reliable genotyping of heterozygous loci.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">45</span>: <span class="cit-elocation-id">e178</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkx853</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft.volume%253D45%26rft.spage%253De178%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkx853%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkx853&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3239.21" data-doi="10.1016/j.ympev.2004.07.013"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pitra</span>,  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Fickel</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Meijaard</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Groves</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Evolution and phylogeny of old world deer.</span> <abbr class="cit-jnl-abbrev">Mol. Phylogenet. Evol.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">880</span>–<span class="cit-lpage">895</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.ympev.2004.07.013</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bphylogenetics%2Band%2Bevolution%26rft.stitle%253DMol%2BPhylogenet%2BEvol%26rft.aulast%253DPitra%26rft.auinit1%253DC.%26rft.volume%253D33%26rft.issue%253D3%26rft.spage%253D880%26rft.epage%253D895%26rft.atitle%253DEvolution%2Band%2Bphylogeny%2Bof%2Bold%2Bworld%2Bdeer.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ympev.2004.07.013%26rft_id%253Dinfo%253Apmid%252F15522810%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ympev.2004.07.013&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15522810&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3239.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000225206700026&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3239.22" data-doi="10.1038/nrg2865"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Powell</span>,  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. M.</span> <span class="cit-name-surname">Visscher</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. E.</span> <span class="cit-name-surname">Goddard</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Reconciling the analysis of IBD and IBS in complex trait studies.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">800</span>–<span class="cit-lpage">805</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg2865</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DPowell%26rft.auinit1%253DJ.%2BE.%26rft.volume%253D11%26rft.issue%253D11%26rft.spage%253D800%26rft.epage%253D805%26rft.atitle%253DReconciling%2Bthe%2Banalysis%2Bof%2BIBD%2Band%2BIBS%2Bin%2Bcomplex%2Btrait%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2865%26rft_id%253Dinfo%253Apmid%252F20877324%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2865&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20877324&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3239.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3239.23" data-doi="10.1002/ece3.2346"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Städele</span>,  <span class="cit-name-given-names">V.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Vigilant</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Strategies for determining kinship in wild populations using genetic data.</span> <abbr class="cit-jnl-abbrev">Ecol. Evol.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">6107</span>–<span class="cit-lpage">6120</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/ece3.2346</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcol.%2BEvol.%26rft.volume%253D6%26rft.spage%253D6107%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fece3.2346%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/ece3.2346&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3239.24" data-doi="10.1111/1755-0998.12771"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thrasher</span>,  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. G.</span> <span class="cit-name-surname">Butcher</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Campagna</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. S.</span> <span class="cit-name-surname">Webster</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">I. J.</span> <span class="cit-name-surname">Lovette</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Double-digest RAD sequencing outperforms microsatellite loci at assigning paternity and estimating relatedness: A proof of concept in a highly promiscuous bird.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol. Resour.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">953</span>–<span class="cit-lpage">965</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/1755-0998.12771</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%2BResour.%26rft.volume%253D18%26rft.spage%253D953%26rft_id%253Dinfo%253Adoi%252F10.1111%252F1755-0998.12771%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/1755-0998.12771&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3239.25" data-doi="10.1534/genetics.116.198424"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Weir</span>,  <span class="cit-name-given-names">B. S.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Goudet</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">A unified characterization of population structure and relatedness.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">206</span>: <span class="cit-fpage">2085</span>–<span class="cit-lpage">2103</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.116.198424</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.116.198424%26rft_id%253Dinfo%253Apmid%252F28550018%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjIwNi80LzIwODUiO3M6NDoiYXRvbSI7czoxOToiL2dnZy85LzEwLzMyMzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3239.26"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Whalen</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Gorjanc</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">Hickey</span></span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">Parentage assignment with genotyping-by-sequencing data.</span> <abbr class="cit-jnl-abbrev">J. Anim. Breed. Genet.</abbr> <span class="cit-vol">136</span>: <span class="cit-fpage">102</span>–<span class="cit-lpage">112</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAnim.%2BBreed.%2BGenet.%26rft.volume%253D136%26rft.spage%253D102%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/9/10/3239.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/10/3225" title="Phylogenetic Reclassification of Vertebrate Melatonin Receptors To Include Mel1d" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/10/3249" title="In Silico Karyotyping of Chromosomally Polymorphic Malaria Mosquitoes in the Anopheles gambiae Complex" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/9/10"><span>Volume 9 Issue 10, October 2019</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/9/10.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/9/10.cover-source.jpg?itok=yA8kifEd" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 9 (10)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/10/3239" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-nwQpyKNfXw8kms1TSpVunAxBt708pvUBxjiGvepX-us" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/9/10/3239" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/9/10/3239" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/9/10/3239" class="active" data-icon-position="" data-hide-link-title="0">Exclusion and Genomic Relatedness Methods for Assignment of Parentage Using Genotyping-by-Sequencing Data</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/494272" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-n8ZyZqMmmK2iam6BNDk9I1nN65ZQaAxMc6IO8rke9ig" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/10/3239.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/9/10/3239" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-Nkvj8B8UMxMDeeZvCN-xOqa1EV5w84a1OjtHXMSb-Lk" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/9/10/3239" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/10/3239.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494272" id="node494272--2" data-pisa="ggg;9/10/3239" data-pisa-master="ggg;g3.119.400501" data-apath="/ggg/9/10/3239.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Exclusion and Genomic Relatedness Methods for Assignment of Parentage Using Genotyping-by-Sequencing Data</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-9347-6379" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Ken G. Dodds</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0003-4801-6207" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>John C. McEwan</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0001-5068-9166" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Rudiger Brauning</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0003-3714-3542" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Tracey C. van Stijn</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0001-8727-5960" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Suzanne J. Rowe</span>, <span class="highwire-citation-author" data-delta="5">K. Mary McEwan</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0002-4615-8917" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Shannon M. Clarke</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3239-3247;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400501 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494272--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Ken G. Dodds </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>AgResearch, Invermay Agricultural Centre, Private Bag 50034, Mosgiel 9053, New Zealand</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ken%2BG.%2BDodds%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dodds%20KG&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKen%2BG.%2BDodds%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-9347-6379" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Ken G. Dodds</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:ken.dodds@agresearch.co.nz" class="" data-icon-position="" data-hide-link-title="0">ken.dodds@agresearch.co.nz</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">John C. McEwan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>AgResearch, Invermay Agricultural Centre, Private Bag 50034, Mosgiel 9053, New Zealand</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BC.%2BMcEwan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=McEwan%20JC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJohn%2BC.%2BMcEwan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-4801-6207" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for John C. McEwan</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Rudiger Brauning </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>AgResearch, Invermay Agricultural Centre, Private Bag 50034, Mosgiel 9053, New Zealand</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rudiger%2BBrauning%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Brauning%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARudiger%2BBrauning%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-5068-9166" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Rudiger Brauning</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Tracey C. van Stijn </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>AgResearch, Invermay Agricultural Centre, Private Bag 50034, Mosgiel 9053, New Zealand</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tracey%2BC.%2Bvan%2BStijn%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=van%20Stijn%20TC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ATracey%2BC.%2Bvan%2BStijn%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-3714-3542" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Tracey C. van Stijn</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Suzanne J. Rowe </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>AgResearch, Invermay Agricultural Centre, Private Bag 50034, Mosgiel 9053, New Zealand</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Suzanne%2BJ.%2BRowe%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rowe%20SJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASuzanne%2BJ.%2BRowe%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-8727-5960" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Suzanne J. Rowe</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">K. Mary McEwan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>AgResearch, Invermay Agricultural Centre, Private Bag 50034, Mosgiel 9053, New Zealand</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=K.%2BMary%2BMcEwan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=McEwan%20KM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AK.%2BMary%2BMcEwan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Shannon M. Clarke </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>AgResearch, Invermay Agricultural Centre, Private Bag 50034, Mosgiel 9053, New Zealand</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Shannon%2BM.%2BClarke%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Clarke%20SM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AShannon%2BM.%2BClarke%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-4615-8917" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Shannon M. Clarke</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/9/10/3239" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/9/10/3239&amp;title=Exclusion%20and%20Genomic%20Relatedness%20Methods%20for%20Assignment%20of%20Parentage%20Using%20Genotyping-by-Sequencing%20Data" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/9/10/3239&amp;title=Exclusion%20and%20Genomic%20Relatedness%20Methods%20for%20Assignment%20of%20Parentage%20Using%20Genotyping-by-Sequencing%20Data" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/9/10/3239&amp;title=Exclusion%20and%20Genomic%20Relatedness%20Methods%20for%20Assignment%20of%20Parentage%20Using%20Genotyping-by-Sequencing%20Data" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/9/10/3239&amp;text=Exclusion%20and%20Genomic%20Relatedness%20Methods%20for%20Assignment%20of%20Parentage%20Using%20Genotyping-by-Sequencing%20Data" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/9/10/3239&amp;title=Exclusion%20and%20Genomic%20Relatedness%20Methods%20for%20Assignment%20of%20Parentage%20Using%20Genotyping-by-Sequencing%20Data" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/9/10/3239&amp;t=Exclusion%20and%20Genomic%20Relatedness%20Methods%20for%20Assignment%20of%20Parentage%20Using%20Genotyping-by-Sequencing%20Data" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/9/10/3239&amp;title=Exclusion%20and%20Genomic%20Relatedness%20Methods%20for%20Assignment%20of%20Parentage%20Using%20Genotyping-by-Sequencing%20Data" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/9/10/3239&amp;title=Exclusion%20and%20Genomic%20Relatedness%20Methods%20for%20Assignment%20of%20Parentage%20Using%20Genotyping-by-Sequencing%20Data" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/494272/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494272" id="node494272--3" data-pisa="ggg;9/10/3239" data-pisa-master="ggg;g3.119.400501" data-apath="/ggg/9/10/3239.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Exclusion and Genomic Relatedness Methods for Assignment of Parentage Using Genotyping-by-Sequencing Data</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-9347-6379" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Ken G. Dodds</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0003-4801-6207" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>John C. McEwan</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0001-5068-9166" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Rudiger Brauning</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0003-3714-3542" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Tracey C. van Stijn</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0001-8727-5960" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Suzanne J. Rowe</span>, <span class="highwire-citation-author" data-delta="5">K. Mary McEwan</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0002-4615-8917" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Shannon M. Clarke</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3239-3247;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400501 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494272--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Ken G. Dodds </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>AgResearch, Invermay Agricultural Centre, Private Bag 50034, Mosgiel 9053, New Zealand</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ken%2BG.%2BDodds%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dodds%20KG&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKen%2BG.%2BDodds%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-9347-6379" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Ken G. Dodds</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:ken.dodds@agresearch.co.nz" class="" data-icon-position="" data-hide-link-title="0">ken.dodds@agresearch.co.nz</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">John C. McEwan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>AgResearch, Invermay Agricultural Centre, Private Bag 50034, Mosgiel 9053, New Zealand</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BC.%2BMcEwan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=McEwan%20JC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJohn%2BC.%2BMcEwan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-4801-6207" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for John C. McEwan</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Rudiger Brauning </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>AgResearch, Invermay Agricultural Centre, Private Bag 50034, Mosgiel 9053, New Zealand</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rudiger%2BBrauning%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Brauning%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARudiger%2BBrauning%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-5068-9166" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Rudiger Brauning</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Tracey C. van Stijn </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>AgResearch, Invermay Agricultural Centre, Private Bag 50034, Mosgiel 9053, New Zealand</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tracey%2BC.%2Bvan%2BStijn%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=van%20Stijn%20TC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ATracey%2BC.%2Bvan%2BStijn%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-3714-3542" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Tracey C. van Stijn</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Suzanne J. Rowe </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>AgResearch, Invermay Agricultural Centre, Private Bag 50034, Mosgiel 9053, New Zealand</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Suzanne%2BJ.%2BRowe%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rowe%20SJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASuzanne%2BJ.%2BRowe%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-8727-5960" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Suzanne J. Rowe</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">K. Mary McEwan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>AgResearch, Invermay Agricultural Centre, Private Bag 50034, Mosgiel 9053, New Zealand</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=K.%2BMary%2BMcEwan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=McEwan%20KM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AK.%2BMary%2BMcEwan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Shannon M. Clarke </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>AgResearch, Invermay Agricultural Centre, Private Bag 50034, Mosgiel 9053, New Zealand</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Shannon%2BM.%2BClarke%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Clarke%20SM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AShannon%2BM.%2BClarke%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-4615-8917" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Shannon M. Clarke</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=3239&amp;rft.epage=3247&amp;rft.atitle=Exclusion%20and%20Genomic%20Relatedness%20Methods%20for%20Assignment%20of%20Parentage%20Using%20Genotyping-by-Sequencing%20Data&amp;rft.volume=9&amp;rft.issue=10&amp;rft.date=2019-10-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Dodds%2C+Ken+G.&amp;rft.au=McEwan%2C+John+C.&amp;rft.au=Brauning%2C+Rudiger&amp;rft.au=van+Stijn%2C+Tracey+C.&amp;rft.au=Rowe%2C+Suzanne+J.&amp;rft.au=McEwan%2C+K.+Mary&amp;rft.au=Clarke%2C+Shannon+M."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/494272/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/494272/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/494272/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/494272/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/494272/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/494272/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/494272/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/494272/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/494272/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/494272/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/494272/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/494272/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/9/10/3239" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-10" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-15" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/9/10/3239.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/9/10/3239.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/9/10/3239.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
