<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/9/10/3105" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/9/10/3105.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/9/10/3105.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/9/10/3105.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/9/10/3105.atom" />
<meta name="HW.pisa" content="ggg;9/10/3105" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Programmed Chromosome Deletion in the Ciliate Oxytricha trifallax" />
<meta name="DC.Identifier" content="10.1534/g3.118.200930" />
<meta name="DC.Date" content="2019-10-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2019 Clay et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="The ciliate Oxytricha trifallax contains two nuclei: a germline micronucleus and a somatic macronucleus. These two nuclei diverge significantly in genomic structure. The micronucleus contains approximately 100 chromosomes of megabase scale, while the macronucleus contains 16,000 gene-sized, high ploidy “nanochromosomes.” During its sexual cycle, a copy of the zygotic germline micronucleus develops into a somatic macronucleus via DNA excision and rearrangement. The rearrangement process is guided by multiple RNA-based pathways that program the epigenetic inheritance of sequences in the parental macronucleus of the subsequent generation. Here, we show that the introduction of synthetic DNA molecules homologous to a complete native nanochromosome during the rearrangement process results in either loss or heavy copy number reduction of the targeted nanochromosome in the macronucleus of the subsequent generation. This phenomenon was tested on a variety of nanochromosomes with different micronuclear structures, with deletions resulting in all cases. Deletion of the targeted nanochromosome results in the loss of expression of the targeted genes, including gene knockout phenotypes that were phenocopied using alternative knockdown approaches. Further investigation of the chromosome deletion showed that, although the full length nanochromosome was lost, remnants of the targeted chromosome remain. We were also able to detect the presence of telomeres on these remnants. The chromosome deletions and remnants are epigenetically inherited when backcrossed to wild type strains, suggesting that an undiscovered mechanism programs DNA elimination and cytoplasmically transfers to both daughter cells during conjugation. Programmed deletion of targeted chromosomes provides a novel approach to investigate genome rearrangement and expands the available strategies for gene knockout in Oxytricha trifallax ." />
<meta name="DC.Contributor" content="Derek M. Clay" />
<meta name="DC.Contributor" content="V. Talya Yerlici" />
<meta name="DC.Contributor" content="Danylo J. Villano" />
<meta name="DC.Contributor" content="Laura F. Landweber" />
<meta name="article:published_time" content="2019-10-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Programmed Chromosome Deletion in the Ciliate Oxytricha trifallax" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;The ciliate &lt;i&gt;Oxytricha trifallax&lt;/i&gt; contains two nuclei: a germline micronucleus and a somatic macronucleus. These two nuclei diverge significantly in genomic structure. The micronucleus contains approximately 100 chromosomes of megabase scale, while the macronucleus contains 16,000 gene-sized, high ploidy “nanochromosomes.” During its sexual cycle, a copy of the zygotic germline micronucleus develops into a somatic macronucleus via DNA excision and rearrangement. The rearrangement process is guided by multiple RNA-based pathways that program the epigenetic inheritance of sequences in the parental macronucleus of the subsequent generation. Here, we show that the introduction of synthetic DNA molecules homologous to a complete native nanochromosome during the rearrangement process results in either loss or heavy copy number reduction of the targeted nanochromosome in the macronucleus of the subsequent generation. This phenomenon was tested on a variety of nanochromosomes with different micronuclear structures, with deletions resulting in all cases. Deletion of the targeted nanochromosome results in the loss of expression of the targeted genes, including gene knockout phenotypes that were phenocopied using alternative knockdown approaches. Further investigation of the chromosome deletion showed that, although the full length nanochromosome was lost, remnants of the targeted chromosome remain. We were also able to detect the presence of telomeres on these remnants. The chromosome deletions and remnants are epigenetically inherited when backcrossed to wild type strains, suggesting that an undiscovered mechanism programs DNA elimination and cytoplasmically transfers to both daughter cells during conjugation. Programmed deletion of targeted chromosomes provides a novel approach to investigate genome rearrangement and expands the available strategies for gene knockout in &lt;i&gt;Oxytricha trifallax&lt;/i&gt;.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2019/10/01" />
<meta name="citation_mjid" content="ggg;9/10/3105" />
<meta name="citation_id" content="9/10/3105" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/9/10/3105" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/9/10/3105.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/9/10/3105.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/9/10/3105.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.118.200930" />
<meta name="citation_pmid" content="31506317" />
<meta name="citation_volume" content="9" />
<meta name="citation_issue" content="10" />
<meta name="citation_num_pages" content="14" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="3105" />
<meta name="citation_lastpage" content="3118" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Derek M. Clay" />
<meta name="citation_author_institution" content="Departments of Biochemistry and Molecular Biophysics and Biological Sciences, Columbia University, New York, NY and" />
<meta name="citation_author_institution" content="Department of Molecular Biology, Princeton University, Princeton, NJ" />
<meta name="citation_author" content="V. Talya Yerlici" />
<meta name="citation_author_institution" content="Departments of Biochemistry and Molecular Biophysics and Biological Sciences, Columbia University, New York, NY and" />
<meta name="citation_author_institution" content="Department of Molecular Biology, Princeton University, Princeton, NJ" />
<meta name="citation_author" content="Danylo J. Villano" />
<meta name="citation_author_institution" content="Departments of Biochemistry and Molecular Biophysics and Biological Sciences, Columbia University, New York, NY and" />
<meta name="citation_author" content="Laura F. Landweber" />
<meta name="citation_author_institution" content="Departments of Biochemistry and Molecular Biophysics and Biological Sciences, Columbia University, New York, NY and" />
<meta name="citation_author_email" content="Laura.Landweber@columbia.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-7030-8540" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ther.;citation_author=F. Adikusuma;citation_author=N. Williams;citation_author=F. Grutzner;citation_author=J. Hughes;citation_author=P. Thomas;citation_title=Targeted Deletion of an Entire Chromosome Using CRISPR/Cas9.;citation_pages=1736-1738;citation_volume=25;citation_year=2017;citation_doi=10.1016/j.ymthe.2017.05.021" />
<meta name="citation_reference" content="citation_journal_title=Trends Genet.;citation_author=A. E. Allen;citation_author=M. Nowacki;citation_title=Necessity Is the Mother of Invention: Ciliates, Transposons, and Transgenerational Inheritance.;citation_pages=197-207;citation_volume=33;citation_year=2017;citation_doi=10.1016/j.tig.2017.01.005" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=C. Baudry;citation_author=S. Malinsky;citation_author=M. Restituito;citation_author=A. Kapusta;citation_author=S. Rosa;citation_title=PiggyMac, a domesticated piggyBac transposase involved in programmed genome rearrangements in the ciliate Paramecium tetraurelia;citation_pages=2478-2483;citation_volume=23;citation_year=2009;citation_issue=21;citation_pmid=19884254;citation_doi=10.1101/gad.547309" />
<meta name="citation_reference" content="citation_journal_title=RNA Biol.;citation_author=J. R. Bracht;citation_author=X. Wang;citation_author=K. Shetty;citation_author=X. Chen;citation_author=G. J. Uttarotai;citation_title=Chromosome fusions triggered by noncoding RNA.;citation_pages=620-631;citation_volume=14;citation_year=2017;citation_doi=10.1080/15476286.2016.1195940" />
<meta name="citation_reference" content="citation_journal_title=Protist;citation_author=W. J. Chang;citation_author=N. A. Stover;citation_author=V. M. Addis;citation_author=L. F. Landweber;citation_title=A micronuclear locus containing three protein-coding genes remains linked during macronuclear development in the spirotrichous ciliate Holosticha.;citation_pages=245-255;citation_volume=155;citation_year=2004;citation_pmid=15305799;citation_doi=10.1078/143446104774199628" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_author=X. Chen;citation_author=J. R. Bracht;citation_author=A. D. Goldman;citation_author=E. Dolzhenko;citation_author=D. M. Clay;citation_title=The Architecture of a Scrambled Genome Reveals Massive Levels of Genomic Rearrangement during Development.;citation_pages=1187-1198;citation_volume=158;citation_year=2014;citation_pmid=25171416;citation_doi=10.1016/j.cell.2014.07.034" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_author=W. Fang;citation_author=X. Wang;citation_author=J. R. Bracht;citation_author=M. Nowacki;citation_author=L. F. Landweber;citation_title=Piwi-Interacting RNAs Protect DNA Against Loss During Oxytricha Genome Rearrangement.;citation_pages=1243-1255;citation_volume=151;citation_year=2012;citation_pmid=23217708;citation_doi=10.1016/j.cell.2012.10.045" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=L. Feng;citation_author=G. Wang;citation_author=E. P. Hamilton;citation_author=J. Xiong;citation_author=G. Yan;citation_title=A germline-limited piggyBac transposase gene is required for precise excision in Tetrahymena genome rearrangement.;citation_pages=9481-9502;citation_volume=45;citation_year=2017;citation_doi=10.1093/nar/gkx652" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=A. Galvani;citation_author=L. Sperling;citation_title=Transgene-mediated post-transcriptional gene silencing is inhibited by 3' non-coding sequences in Paramecium;citation_pages=4387-4394;citation_volume=29;citation_year=2001;citation_issue=21;citation_pmid=11691926;citation_doi=10.1093/nar/29.21.4387" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=O. Garnier;citation_author=V. Serrano;citation_author=S. Duharcourt;citation_author=E. Meyer;citation_title=RNA-Mediated Programming of Developmental Genome Rearrangements in Paramecium tetraurelia;citation_pages=7370-7379;citation_volume=24;citation_year=2004;citation_issue=17;citation_pmid=15314149;citation_doi=10.1128/MCB.24.17.7370-7379.2004" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=J. Goecks;citation_author=A. Nekrutenko;citation_author=J. Taylor;citation_title=Galaxy: a comprehensive approach for supporting accessible, reproducible, and transparent computational research in the life sciences.;citation_pages=R86-R86;citation_volume=11;citation_year=2010;citation_issue=8;citation_pmid=20738864;citation_doi=10.1186/gb-2010-11-8-r86" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=U. Götz;citation_author=S. Marker;citation_author=M. Cheaib;citation_author=K. Andresen;citation_author=S. Shrestha;citation_title=Two sets of RNAi components are required for heterochromatin formation in trans triggered by truncated transgenes.;citation_volume=44;citation_year=2016;citation_pmid=27085807;citation_doi=10.1093/nar/gkw267" />
<meta name="citation_reference" content="citation_journal_title=RNA;citation_author=J. S. Khurana;citation_author=D. M. Clay;citation_author=S. Moreira;citation_author=X. Wang;citation_author=L. F. Landweber;citation_title=Small RNA-mediated regulation of DNA dosage in the ciliate Oxytricha.;citation_volume=24;citation_year=2018;citation_pmid=29079634;citation_doi=10.1261/rna.061333.117" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. S. Khurana;citation_author=X. Wang;citation_author=X. Chen;citation_author=D. H. Perlman;citation_author=L. F. Landweber;citation_title=Transcription-Independent Functions of an RNA Polymerase II Subunit, Rpb2, During Genome Rearrangement in the Ciliate, Oxytricha trifallax.;citation_volume=197;citation_year=2014;citation_pmid=24793090;citation_doi=10.1534/genetics.114.163279" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=L. A. Klobutcher;citation_author=G. Herrick;citation_title=Consensus inverted terminal repeat sequence of Paramecium IESs: Resemblance to termini of Tc1-related and Euplotes Tec transposons.;citation_volume=23;citation_year=1995;citation_pmid=7596830;citation_doi=10.1093/nar/23.11.2006" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=R. Durbin;citation_title=Fast and accurate short read alignment with Burrows–Wheeler transform.;citation_volume=25;citation_year=2009;citation_pmid=19451168;citation_doi=10.1093/bioinformatics/btp324" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=B. Handsaker;citation_author=A. Wysoker;citation_author=T. Fennell;citation_author=J. Ruan;citation_title=The Sequence Alignment/Map format and SAMtools.;citation_volume=25;citation_year=2009;citation_pmid=19505943;citation_doi=10.1093/bioinformatics/btp352" />
<meta name="citation_reference" content="citation_journal_abbrev=Cell Stem Cell;citation_author=L. B. Li;citation_author=K.-H. Chang;citation_author=P.-R. Wang;citation_author=R. K. Hirata;citation_author=T. Papayannopoulou;citation_title=Trisomy correction in down syndrome induced pluripotent stem cells.;citation_pages=615-619;citation_volume=11;citation_year=2012;citation_issue=5;citation_pmid=23084023;citation_doi=10.1016/j.stem.2012.08.004" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=C.-Y. G. Lin;citation_author=I.-T. Lin;citation_author=M.-C. Yao;citation_title=Programmed Minichromosome Elimination as a Mechanism for Somatic Genome Reduction in Tetrahymena thermophila.;citation_pages=e1006403;citation_volume=12;citation_year=2016;citation_doi=10.1371/journal.pgen.1006403" />
<meta name="citation_reference" content="citation_journal_title=RNA;citation_author=K. A. Lindblad;citation_author=J. R. Bracht;citation_author=A. E. Williams;citation_author=L. F. Landweber;citation_title=Thousands of RNA-cached copies of whole chromosomes are present in the ciliate Oxytricha during development.;citation_volume=23;citation_year=2017;citation_pmid=28450531;citation_doi=10.1261/rna.058511.116" />
<meta name="citation_reference" content="citation_year=2019" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=H. Matsumura;citation_author=M. Tada;citation_author=T. Otsuji;citation_author=K. Yasuchika;citation_author=N. Nakatsuji;citation_title=Targeted chromosome elimination from ES-somatic hybrid cells.;citation_pages=23-25;citation_volume=4;citation_year=2007;citation_issue=1;citation_pmid=17086180;citation_doi=10.1038/nmeth973" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=E. Meyer;citation_title=Induction of specific macronuclear developmental mutations by microinjection of a cloned telomeric gene in Paramecium primaurelia.;citation_pages=211-222;citation_volume=6;citation_year=1992;citation_issue=2;citation_pmid=1737617;citation_doi=10.1101/gad.6.2.211" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=M. Nowacki;citation_author=J. E. Haye;citation_author=W. Fang;citation_author=V. Vijayan;citation_author=L. F. Landweber;citation_title=RNA-mediated epigenetic regulation of DNA copy number.;citation_volume=107;citation_year=2010;citation_pmid=21078984;citation_doi=10.1073/pnas.1012236107" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=M. Nowacki;citation_author=V. Vijayan;citation_author=Y. Zhou;citation_author=K. Schotanus;citation_author=T. G. Doak;citation_title=RNA-mediated epigenetic programming of a genome-rearrangement pathway.;citation_pages=153-158;citation_volume=451;citation_year=2008;citation_issue=7175;citation_pmid=18046331;citation_doi=10.1038/nature06452" />
<meta name="citation_reference" content="citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Microbiology and Molecular Biology Reviews;citation_journal_abbrev=Microbiol. Mol. Biol. Rev.;citation_author=D. M. Prescott;citation_title=The DNA of ciliated protozoa.;citation_pages=233-267;citation_volume=58;citation_year=1994;citation_issue=2;citation_pmid=8078435" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. R. Quinlan;citation_author=I. M. Hall;citation_title=BEDTools: a flexible suite of utilities for comparing genomic features.;citation_volume=26;citation_year=2010;citation_pmid=20110278;citation_doi=10.1093/bioinformatics/btq033" />
<meta name="citation_reference" content="citation_journal_title=Molecular Biology of the Cell;citation_journal_abbrev=Mol. Biol. Cell;citation_author=F. Ruiz;citation_author=L. Vayssié;citation_author=C. Klotz;citation_author=L. Sperling;citation_author=L. Madeddu;citation_title=Homology-dependent Gene Silencing in Paramecium;citation_pages=931-943;citation_volume=9;citation_year=1998;citation_issue=4;citation_pmid=9529389;citation_doi=10.1091/mbc.9.4.931" />
<meta name="citation_reference" content="citation_journal_title=PLoS Biol.;citation_author=E. C. Swart;citation_author=J. R. Bracht;citation_author=V. Magrini;citation_author=P. Minx;citation_author=X. Chen;citation_title=The Oxytricha trifallax Macronuclear Genome: A Complex Eukaryotic Genome with 16,000 Tiny Chromosomes.;citation_pages=e1001473;citation_volume=11;citation_year=2013;citation_pmid=23382650;citation_doi=10.1371/journal.pbio.1001473" />
<meta name="citation_reference" content="citation_year=2014" />
<meta name="citation_reference" content="citation_year=2019" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=A. M. Zahler;citation_author=Z. T. Neeb;citation_author=A. Lin;citation_author=S. Katzman;citation_title=Mating of the stichotrichous ciliate Oxytricha trifallax induces production of a class of 27 nt small RNAs derived from the parental macronucleus.;citation_pages=e42371-e42371;citation_volume=7;citation_year=2012;citation_issue=8;citation_pmid=22900016;citation_doi=10.1371/journal.pone.0042371" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=E. Zuo;citation_author=X. Huo;citation_author=X. Yao;citation_author=X. Hu;citation_author=Y. Sun;citation_title=CRISPR/Cas9-mediated targeted chromosome elimination.;citation_pages=224;citation_volume=18;citation_year=2017;citation_pmid=29178945;citation_doi=10.1186/s13059-017-1354-4" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Programmed Chromosome Deletion in the Ciliate Oxytricha trifallax" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10.cover-source.jpg" />
<meta name="twitter:description" content="The ciliate Oxytricha trifallax contains two nuclei: a germline micronucleus and a somatic macronucleus. These two nuclei diverge significantly in genomic structure. The micronucleus contains approximately 100 chromosomes of megabase scale, while the macronucleus contains 16,000 gene-sized, high ploidy “nanochromosomes.” During its sexual cycle, a copy of the zygotic germline micronucleus develops into a somatic macronucleus via DNA excision and rearrangement. The rearrangement process is guided by multiple RNA-based pathways that program the epigenetic inheritance of sequences in the parental macronucleus of the subsequent generation. Here, we show that the introduction of synthetic DNA molecules homologous to a complete native nanochromosome during the rearrangement process results in either loss or heavy copy number reduction of the targeted nanochromosome in the macronucleus of the subsequent generation. This phenomenon was tested on a variety of nanochromosomes with different micronuclear structures, with deletions resulting in all cases. Deletion of the targeted nanochromosome results in the loss of expression of the targeted genes, including gene knockout phenotypes that were phenocopied using alternative knockdown approaches. Further investigation of the chromosome deletion showed that, although the full length nanochromosome was lost, remnants of the targeted chromosome remain. We were also able to detect the presence of telomeres on these remnants. The chromosome deletions and remnants are epigenetically inherited when backcrossed to wild type strains, suggesting that an undiscovered mechanism programs DNA elimination and cytoplasmically transfers to both daughter cells during conjugation. Programmed deletion of targeted chromosomes provides a novel approach to investigate genome rearrangement and expands the available strategies for gene knockout in Oxytricha trifallax ." />
<meta name="og-title" property="og:title" content="Programmed Chromosome Deletion in the Ciliate Oxytricha trifallax" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/9/10/3105" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="The ciliate Oxytricha trifallax contains two nuclei: a germline micronucleus and a somatic macronucleus. These two nuclei diverge significantly in genomic structure. The micronucleus contains approximately 100 chromosomes of megabase scale, while the macronucleus contains 16,000 gene-sized, high ploidy “nanochromosomes.” During its sexual cycle, a copy of the zygotic germline micronucleus develops into a somatic macronucleus via DNA excision and rearrangement. The rearrangement process is guided by multiple RNA-based pathways that program the epigenetic inheritance of sequences in the parental macronucleus of the subsequent generation. Here, we show that the introduction of synthetic DNA molecules homologous to a complete native nanochromosome during the rearrangement process results in either loss or heavy copy number reduction of the targeted nanochromosome in the macronucleus of the subsequent generation. This phenomenon was tested on a variety of nanochromosomes with different micronuclear structures, with deletions resulting in all cases. Deletion of the targeted nanochromosome results in the loss of expression of the targeted genes, including gene knockout phenotypes that were phenocopied using alternative knockdown approaches. Further investigation of the chromosome deletion showed that, although the full length nanochromosome was lost, remnants of the targeted chromosome remain. We were also able to detect the presence of telomeres on these remnants. The chromosome deletions and remnants are epigenetically inherited when backcrossed to wild type strains, suggesting that an undiscovered mechanism programs DNA elimination and cytoplasmically transfers to both daughter cells during conjugation. Programmed deletion of targeted chromosomes provides a novel approach to investigate genome rearrangement and expands the available strategies for gene knockout in Oxytricha trifallax ." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/10.cover-source.jpg" />
    <title>Programmed Chromosome Deletion in the Ciliate Oxytricha trifallax | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"-SXOfuPQ0456GXv4GEfGO59boGMudgA17EqDp3uXgJU","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"494273","apath":"\/ggg\/9\/10\/3105.atom","pisa":"ggg;9\/10\/3105","ac":{"\/ggg\/9\/10\/3105.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/9\/10\/3105.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;9\/10\/3105"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;9\/10\/3105"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/9\/10\/3105":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022eOtfuYGMH+ohFxfbm9SDHyAKPhZ17aMU\\\\\\\/Dl8Owrnf8OSPnlg1sRPKPM13V7534ASYT7raOw0qZBA2ZlKc8fP8j5Je8joQeasmegOhd\\\\\\\/kXiaQU4zl+NC96aBxVbzaLNIOXGQ2+neeI288BCLEI7ttF3nz+T3J9B4A34+hGuhNfx+dGja+MHkexYp54\\\\\\\/wMeyT0l2Tc2aikMgbc4GQXiZIdd+jKsfuZyxsu+IBjPMyWktlZrX4XKk54juMM2nV1C0YHOEf9RAW9ahpl+WBjDghqxIDnPqLw5HlHZ17bDfWSTZgRMmHWd0GdjY1nFJ1jl9sL0xzirGW+yfukZxToTFwuPCkr8xlMudMMR4GeORZ31RlkeNRKiYvjEMiAAeEGtfuGHKcFJgblEY0rIDTa5hduzRVSRarhBm4c\\\\\\\/o53tNjuvy+HCF+G+ocMYGbOV37QNpYpGD78PSV+UOI8YEO7qUfI1lgqYhpZkvq0fV\\\\\\\/ARyMnjfyCaf00RFm\\\\\\\/IMiAylCZsR404fOBKtZBuGXZ26gZO9H23LaJ0U8aJMQQmvfq2AATi2M\\\\\\\/kla3qwIfatAbhvYsuumOjoOxTuBFbRlEsIvzoT+vZkfn+mNkLpBP46+6a7daiGgvDc14Z3kEha5JixI\\\\\\\/jc0pctqleL6QHG3ghAwQV8Hp5eTAEYSC8iJ8d9c6Qqm3MnLyM+nQZcnioRWMCwwDxcvNHRRsoghJdPQhdWhApIb3M6JXaYRmEzwkCLZZFAAsKNmabzah4HOGJk4UO3RtpVGUc+TH0ntd0DwaYYPWLXDiX0v3jVs5OdoealO0xDNM7r4SlfhcGLe\\\\\\\/SOOEYVR\\\\\\\/Hx0qJzCgXaXezNTWRdbB3jdAJV\\\\\\\/iye2KntmY0YklgY\\\\\\\/IgID+wd40vjKE7TsOI1ilM+OLH8BTrMPj4LjI+Ne\\\\\\\/COxhNlwxlTLvJ7NLUfeP\\\\\\\/LT4pn2VhWCafcca42Qwgmv\\\\\\\/8\\\\\\\/kcKTbBbVPPId+Ke3zPuh9Pq95egnbZqqVMwEW\\\\\\\/haGa\\\\\\\/AKJFv2YCD3izLEBunUdAutc3i292PEDNyRCs1NDQdhml8N+YIkH0fNYS8KjZhsmQokuni6o0SpVturaOsSbmr+oup+7AuJ6Q0uRVBC0prGZBmvmJZvZhUgiy35tbyGe5XqupRQX6EgKQA3Yq1HDTs7xOH4NwG7QHIhtTXvDkS5ZcFNpzGJJIvjFY5Qsv1ELXHcC3nIUZuepQlEXQKYFWeaYQVlvmko6IUcX4v3EG\\\\\\\/3qLtA8yo2xFaHL67cQNwbGXNlZ\\\\\\\/ZqI\\\\\\\/b6PNjv47nhPfxFo\\\u0022,\\\u0022iv\\\u0022:\\\u0022OAPGhSqTDSD3Ka0yHIb2Xg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00221ef0072c2e6498465f41bc8d26cbc0d7cea3f441af7b96977b0c7167091aaa6f\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022WEmk43Fl+lITRAHlWGnkbK8Vaf3pCjOrgxrW+37YsyIaHiS5LjIhFo3myFMyexSEA1L6TSE34Qqy1c9IssN7waqSzPFR5P+hNNNnD1dbeczQj4zI0\\\\\\\/6r4x8A1yV7Xosu7L8oOVQ0ynMvBqTHZUl+ml4SmB0yt1AbtftvwBQ3ltgoHyjevy16ecNYx47bX2D7lPsG4YBjorq50pU\\\\\\\/WreKTPA\\\\\\\/Oe32RgfcFeWusOCZODDW3q0Byr4rDBD9qMTZPQXzhNI+f3aLobTs+jLLmg4YtLms1RUwuOSSbfAj8NismsrGbYTL04MuSaoDRATPh2enbks1hAXPq8z1oqUNYA026HwZP55aV8PGWPNUq\\\\\\\/sAOigxxX6F7cNrdn5kUCn9CsUjgC37bMODda8PeG3fA5H8nF0LrKVLfebeBOLic4Duv6qkDNlG8WmyUdOH4pI7oLnZKU6QplaHwU3cPNEuBiaa6zM2AK5s0EB8YRbj0b10ygiNnBpmCJuODAev+iTURdgd2fl9Iw4GWjcMCj9Qusm34UqzWHqGt4vytZy9vsfH2xTNxzr7sp9YfLVBFt3IkkmF9IaJ+ew92us0GMLN+P1utVYQxN\\\\\\\/ZhEm8eD31GYoqGFmPqvDbmTfH3zJI8z3hYFnMk7c9pdoizaVtWffHJKW4\\\\\\\/GT7+2KPwB23a+v38oWJAEh2wD9dABGrADX1aDr5R0JWpD36XPFOOE6cVvrqJVFUT00EynQVcKceB2ZIX3fADBbdi81DiVjZfRKOL\\\\\\\/dc5rGSD8tVzKVdio+Z73j8AGlsiIwT8Mu2mgQsUWJMdUuc\\\\\\\/F7rx0BYIlS95hc0lSBRiUoeDHdoVzJpoqBGcul\\\\\\\/33Uuq2M+uy577BD6LkZdcs3\\\\\\\/Zs4cBFoZVys45HUMQo\\\\\\\/GJ2lTt3RLRrQym7u2KcEpEbS\\\\\\\/BkcIvloAZvg0KSba3r3yUutvqRG5aO75XSqWfCiu4KYxPXa7NqrL+Atgl+0LA6W\\\\\\\/1WoikgRkl1S864fJizgZkZhn\\\\\\\/X\\\\\\\/4\\\\\\\/UpSVXFN6M6B+r\\\\\\\/QulCegiz5IjnG178BOBBdeLCazb8XLazcZSIswFQYrMsAHGZQ7XCN9L9k7HAcEcqlWgN2UQ8d7e95b6t1nxDRw8IeM\\\\\\\/Ay\\\\\\\/lnQjJSsQIHZ5Y805XcZ20c0NR5kGT8A+h6WnWGXeM4lI0X9IM1eV3ms4t68UrQbNqr6U076k7eOTqz0pLrI7jolKOII0MshCE29D3z2S8+ClAlbu+tdVoff4wORW1wwOnfyh3qHBEsEikKScf1wxNEjckWvFugNtj46hne+HJk\\\\\\\/G7SczGhEVEkslM6ZCgM91SBJfd5wl9RuFmWcDaKVu8DbKsSLy2LzTiS7JRC3EzIuCYYp4VS+ofqvrNZZWcxpdfCyZ5BGwIr+6vn8S+tAO7QTNalL+yL46B5+44EbaHL5PdFB2KosO1Xxp8c42nYbGJ28VN8miNIEI1bKOkrnSccMQibup08PWhNm9ZD6Sdt+s\\\\\\\/WbdruwCtfJthaKo1jcVlodjBPHgxfXONf3wjL3mmI2yjLU5FwwFXbScgjeWzLVSgxVAjzaHBdz98B9Kfjgsv62UcHMVpFKm6q1LrV8wBqqErVk48RJuxFjSDOTfLXisDg2JVsJFaji5g1+92T3OUW0KfS1Q36wt3Bu1iSBMsMjItFk6cC4WiVV8cOBxDvQcO321snDw2VQX4QRXsYCaI35M8v+2ZuLFHZtCHCxrhRfCgXf+KmRLtg6cG8m8pp9SYu1y+DxelTQGt2T9GLEe7oLLmbVZ9+s7jQhGd5yaHLu9o+wa6pNCxlBLKX8fGkRlfi2NhD1PuXB2rcaysfjSzbpEJsI27WqZgfnTYg4RQadQxQQfwMobGDRDwTgNfvn50uWhEqkA9ereIoTIgoyZj9EUjBzeRZZz+rTdQ6nJHVHFX+nSRav7LkxepKfomCG4wTHpIm0ORTzsnlNKcFLor45z\\\\\\\/JOdy0Lu+GTukyGCrrRIWxPumuL0cLnJ+C\\\\\\\/3QaS20U7g\\\\\\\/VYYpqYCQLb258lgWCsezolcTIBKvOE+iGpHS9lLIsB5i6yc3VLn0\\\\\\\/zDdGZb2oU6Xl+SBG19NqcxGhZ1R9RjklqOFO0rvuvROtHjd8ANLOUUg5JIg3CQFXkfuO7upG1bs1u1AeVktJZaLTQUASJs0KJCW2DA\\\\\\\/vJndX5FyFD4kTWxnqD2+7y+RChq6PuKVqWQSfi29o2Fj9Lb0t760M0qWAx6eRgisSRqClKLWjmTIPn6ndDE5MsFmOh3FtglxfA\\\\\\\/II6FR9c07tPUtXEtZMpPSdJ6WwqZnWreE\\\\\\\/IlvK+52r7uInP\\\\\\\/UZXoI3ox+aD4no6zuQtUM0jpMvv6T4flfRt3UQzZSUW50rremOdhsR2NM1v9pJRt+j0N+Ny5k9h\\\\\\\/bexWdISRPgFulybrwdsnO5i4QVCgTlt6ZyplhDp42r1\\\\\\\/84Cx\\\\\\\/LjvQWKwXOqZqKbslqpWwUiVj3PMYa5pWiXtUZZ1p9XohtLJuAO1KgC0ESyTEdZzYMARRK1woRd+\\\\\\\/F5jLCIvTdexRSubpr43tch2j6j8UdUzLGBjihYTDOA7uGXV23M7yaom1izgcdJVcIp1tbmQbK0EfTPuV5geQtKJZBQM\\\\\\\/WlgHsH64w6zKxkqAoVwsKnv42kiwW5Jj9g6o0VU3GTw4zd8\\\\\\\/A0Ns9m\\\\\\\/Nw2a3+CzWjuJ0AzepIVVSwAyZnFmNIq4VP8yaKIVsxgsw6zPWgcMSwnVycYN8EbtNIu8CtfKolU\\\\\\\/Yi+UvqNnNEPI2La15Fpj9fB8F4xhmYG7vUQrBf+zRE4ViWDlN2ReI942TcVZr5CRvtLkkMJ5fRu1HmdtCau\\\\\\\/yyCFg5lTzOLN38HSr7L9nPXiGks9zNOl4ADwAmPke66msDO0dNUtnPx3QVJusRNLGnrickXD60kopQH2eaavGFCRvAbjkbWYGa2i0\\\\\\\/YvWEwjd+eQhzzkO927bYw7DfVtjKsC9GWv1zqz27bOAczyCD7UPDv1s8NlrLnzZjw4xCrXcsaPAoEv5zSLeRjar1ADzvVQai0+eobBSe4gGHA6nW7FxeoAckaI+2f1AsZ32GNYQKgMPEUk2xtDMCeTjfEG+R6T5hxtleoOjqNCiLwNwepbyItkEyC7luS\\\\\\\/u2PntaIBem2tWZJhnUxi+Sm7zbNsHQKPx1JQYztiEAT0BEekbDOTvua4UKlRW0dCKDA3KeDF3Mdy2n+5uqt8zkvxm8qicQPAQuRT80lckCcqAFXRmScDgQs28URUHbmKx3q90nA6SoJLlE7SxO+ZlyoBbU2WCCoAhyxGXxRhC7K7GJU6y6rRXqbI7GEzivRKYFaXTHMlIc+agAUce\\\\\\\/FN4XrRwgSha\\\\\\\/yOQDJndSXhDnpYzn2rcH3+SplqddSjXVM3tRUQ+HQ4VWnZoIJSx6GN\\\\\\\/4+2Tvv6eyx358Z\\\\\\\/sUIAJDto4ZO4Wbk5Yuuqd4\\\\\\\/ZQRAvwQjvu0BkGLnMHU8m+egI3nj75DM0UCpRb1R9LdDE\\\\\\\/l+N5+HWdpDwNokLpUem4MDAbADnTpda7vFC3z5QK5SJoHH+vow8BoViJjY4Or3GAwgxTuYR6F0kN3ZfBV\\\\\\\/fxaItvZ0FRvjmMEzSVOu7OSBkIs5A3VRfdWrOTOXifYCGPcIG5AIgfo3vsJP1g+4YI65MKNGW9HUYiKnkaoBAMAzxesF9yIblE0gsxA2mFBiPTpWBN3fmg1ajFbQ2Cywak\\\\\\\/CVG\\\\\\\/Hn3Ge4c8hzpfY2W+eQdKfGmMu1UTrzUa9aIvyrP3bLN7m49QlZ9uUQO1KQaftezuLTik7U\\\\\\\/iDukfAPhtqk+2TGquRF5YnRzvygWNZ4ZW0rDjzFyk\\\\\\\/OflVWQ6FXwLoDfapVqh7gi+7lvWDx2hCuEKHOuKzib5NATg4jRZs3OzvNgLJ3pjsvu2j3Q52JUPRJD1zJq8dY392\\\\\\\/T4DfpxbPw3rZMqdHbhzpjbRXrYHzTpATr4Im0qmkdKVAA+aU7TMh+5i6CAFIPTrOZjelhvHs56nAU5stuLTSltnw+ddePP3hz\\\\\\\/oytXEb\\\\\\\/w69RJysv4IOB2k0ZUMHArMwua2qOKdYLICwNrqyTaQUOl8qiwwx2UJTMMc5x5XSD+lz6+\\\\\\\/ARrvXALEvtecJmdPiBbMSouIBw9GMwPB49UhBiKOmsiC+Julujn+VcmRt\\\\\\\/MRN76EL3oxdFemGJRqGgdirefJekR5KtzDRE+jy9BhB1N7hGzmrE3H++\\\\\\\/qGrfVBtr9gsTcclO2dYcw01Ow5zn7FblpUjfvuuc4MmDizJ5dWPOga4iWYkFg2HXqCrvC2V5aSs0Kk\\\\\\\/jv1sTgqLakhP6xI9DQtZDw0V\\\\\\\/XrVBrC8e0S771uPn7mra0koIg2CbChUTzAuktURhCYWvJA+sU8EwBqJe2w86eVHxohd5ePYbKlD3T9amfK1KOVbViJH+FSoYQkLjjv3PKsEAsxdaqUTvykp3tXFZ4neSTZLT8QAf7WCM7TBH7DjzrousfIH48YGSkTQaaODevLH7qyMDOsuP2dFkc3X2e+wvQjhvQ7hoc4EzsBH0TpLzph4whFb2UZSyAjCDsnPUjJmzLJdCMDFlRRsBDsM6awORnsBLkxNiC\\\\\\\/LFCH4nHIX3eiG6R3lFux0ymjhQ7\\\\\\\/b4yWLx+KPdMJZorFGPqLRVErEWTq0DKrP5pc9IDtkFWS9o92g96nOXog1M23mbyasDyfkv65bgBjhPuySldrN\\\\\\\/TelF9aVl7yRkn1CCeGkWubK9SZ7VnR\\\\\\\/amPii7rPh+Te4gGctFstwup\\\\\\\/sjMQ2TmrIWPPaJO+5JoKzg3C14eqOeplOIE1CzUZ\\\\\\\/gfpS7Vcp8FuUSyJoJEwUrD9pSihNUzI3+MSI23wUb9sNes\\\\\\\/NYIRKY081ByH3BmPsYzv8Kt6Rb3KpwzKd\\\\\\\/vH44jQBgAysX9AZzY3J9j9VSAgWZtzb3hSkhglOnkzO83Gy9EnrXCOAQJC9VD\\\\\\\/MjxdmB3zTptBOr6guzLDgiL5RF1RArxhqSB2yeUgf7GJ9FSu8JXYxQ2JBzoMdeKZm\\\\\\\/ip4Ki8AZm7QstgFXT1hn0aRd2afUJG1+MkG0h43\\\\\\\/Pq92\\\\\\\/lV4dEf3tpepaooo\\\\\\\/NQ9\\\\\\\/kp\\\\\\\/\\\\\\\/CRODH5XHhckzj8aMulHwZJAyiFwlfsFugMkaa60EcwdadNP6I93OHfgzj64JCY2diJ2s2Z2zXoKEvRROxm77jKTS8XRETw5Uoba\\\\\\\/NoOJi+xWK9NJA9Yp6++g9rJQgL9EIWEkNIuuUg68jgfiqKhGm5\\\\\\\/L60Z1qkT1Muvu0EvT01fskPO+y5WFwABbgDQfWlAu8Hw27vTvyIZQq0hI\\\\\\\/7H5hUhDdFYd+paLOXCFeTHYOcHMWLwck6J3fzSCtNZ6G1OMWi02nrD2HxMv4GXb\\\\\\\/RQttbr5S6v5KhLaEOn43WJfMb8m8ARqhsyva2KrXwBuM4IbZn65lIfKFH5vasOIIrJgUe09FsETa3K02OCNV+KFqJh2mLXM\\\\\\\/nYgoIZnrD\\\\\\\/L4ozCNtw9XGsORHllFwzmyZo1HyxEzlwEU7l5doDtqPI\\\\\\\/+jvUlCdSkLx6kjnY23fTvV0DwHCC5tnMIZhq5xkkueKFUIuYvmm2avy76pmErf9hC7SliCit8wL7t3JySnOqtk9WzI0YuIXcOnLJYuJnDKrE7JfcooCoFlJfoEj+ODIwoxYyYHlM71J5kCFYOfgd2V91w1VWYokwc0xiXbMFI2AyXY5+Yc3luU9X1tNpIsqBh8VcbKlimOIXGqOaZ6uH\\\\\\\/oF5gbhGThme1FxCjb9m9X0ArVqHyCcC8+3hMYwQSCXCE4BrrJPo7Zx7YTV8R8HygSao8kNCCv6x8MI9U4zH0SouZF\\\\\\\/Ru7hPXLEqsvvcC9uuVetPL8BYQlPJK1Ses9xfNQpUNrkZru\\\\\\\/SlZtZ+DCxcEFLwSA9zXc8vlS7cz4TIyJA0aK9WtisSZW9WXQKpJbfdBvSTsC\\\\\\\/eSWbVQgOeL595nO0Phds3lirJIM4IWWvh3mqnMMgFT78hap\\\\\\\/DZCbNHFLriUQPRN3mLv8aCf04Sb1sLuzUNVHYnRkATmq+C3zlTwOnfMMepYjeXeOZLUJ1dveBThtlUvoYYUrH5sAs3kTpSjk5MXOVhhfONb5sBpTi62hQsf7QJif\\\\\\\/NjLiIwTIbjug4qWYSiKmTAtPG\\\\\\\/7iKcfShQOSV6Qwl+cNr3yz9hCd5UvTh0CKXFwU9DYBe0C2aOiJhj4mCAX2sq1+lWT0g+8XIdl+sho\\\\\\\/Tm6Rqz5yIIWl5y25lAHwfMUsSWy7CRBpZhtS+W\\\\\\\/RlvAvGCGL9P7VvpjKPWsD9rOx2URLLqXpT9EPVIFDori\\\\\\\/KKr7+\\\\\\\/mth1Jd+iJNT0OL54qYfTb2WP3uDhHKvITonouQ1+y0pH3uGVOiWQP6Amma7p+9d\\\\\\\/bXZI5a+04iEHjAk1XxYz5IWDaSd1u63o39QhvumPUWbALqZ3SqwCTYkCvUxnQcJGiUkz3iYV8XKvQOfflTQDxp\\\\\\\/Fw6gUGrTJDeHEL3Xe0LLF8iv7O4fgrcwrAdwSpcCZaVCmc3+FtH1lYJeZs1q01hgah0wtBLsl8jOkVfQpMwhrBHaHKKIV3EgnNO8+SdVTmNgcsXUH3oD7uaoZbAhjOZ4+lNtyT5xZr+LVUUr3FurBYkrdR8XC2q2rUi0gUx5oOylTkG1I+fAmgQLlg1cKaqasCjJY3oEbnbbhVPlQc52wSq8xXJYn67XySj0eZQrQdZvFXZ1r3iB6K7FWMUXTefItc8rf7nO9dff\\\\\\\/bFpwGF0yWQ127nSh\\\\\\\/yVZ0iKdS74JNoULi+FawNw+ULTWvTdv1huLwnlD4CH4chiIM6Ph+TqySDpzZicAYDBnIVpcfEfI47eBfQZenSAcMT884oLtdNDeZsaoXXhWInHXwZ2+p1wrvvSkNVTj\\\\\\\/LkH3l5Rj51GF5rzFE8bmsM4Vo+hxYMlayHhAqFDKd3gOPayHkaAtzlsgmq5u6V4yQTa6vEzPvaZ\\\\\\\/SsHig5N00JJfEtO7BBkZsIR\\\\\\\/MzJMLLunPaJ7mzt5Op1+vFjfirXAxyRbiaAZyqddhLgeli0BPcNpHpHUmbhhzeL0x58Vq8Wy9Go40nbkepXh0B+6shs9huBtEJUBWXUZAZb\\\\\\\/US7mpzreMJvNZAWQV69j0aZbO4kXt9bO3mG8F8U5ZesyFjPU9xxSYo+Nc7eJOVQVFjk93LH86mg2ItS4gpAn1qrKx5F0cIDu9lRs+vgZMI36LVtBdujI+HCh1jG3Qv5xcvETaFg0oobnP5TFn0cP5ApVXpqIpzlt8goj2ipln8ik3uAVIteoBB+SLD5NcA6gERkDwcDLf69Gbu+DFY\\\\\\\/fQBUYXnyIvX8X71gAczzbbi5zMSLK8B8n8hceQQyBbyEJGC201BWyiRn6V8IF4c1nbfXZjZVlsRR6xfg50z\\\\\\\/VYvMSPXnekTVX+kErhVoH46lZKrSY03xMAkIRCLvssYQU7ouB\\\\\\\/EGyee1U2NcFxoESSnX01YzmVf7zkUT8s3bKSKU\\\\\\\/YSdHchfsAmhbBYOovQ7gKg0TyKFwedauOFAtVt7nYzmtm++SAH0IB3ZAhHt5B9ivcEBmwFglJGZn\\\\\\\/P\\\\\\\/Ky4xsXZzqtfqkqW38OeB5yb6xoLYCB1humRi60gAr7EWB0M6DEwc\\\\\\\/rMQeHIskpb4uhq\\\\\\\/2r1gCSvxlv3VN786+N7xRaIvLZqjb355SBEjpFGtAKdMmEha8\\\\\\\/pBwfUjR8q5n9Aid8mPzCbVqKqjWoq3mew91CsCdhmGQp6\\\\\\\/GLg2cfVXPgWnF\\\\\\\/aomsSmYYpTs4X2xNA4TuhCM0E1cxs2KQGMeVg3wRf4ty8xcyGtTDF2NGtZhJuoFehLpbEZ9A9HSCDvJ0e7QWtmL7atJEkwAkJOexqea0nw8x4a1VA3N57i+CE+oTDKBoR70xWWvWKJIZP3stxxsMnaSCl1lmB\\\\\\\/ZMWCt7HJv1DTfklezpm6oONfWHI51H1VXtgKqvewyIBiBNL0Ia8AfbqSbJ0wMlLzzryoxVFavTO6YHybsDpakk3HZOZLHJ2rvZpDt1zqBstMkIVwcZ0HPLztL3jpZmh3R0afF6YgU7XuSiInOOzkwni1R45\\\\\\\/O3C1z3M07M2mqth4fkvcm14brRJVc7MgXroNJzKgXReFlG1\\\\\\\/DVIOoURReft4EWdll7MWAxEEDrORz3XHiIn37Gct6MblZbWZg6zn6iAX2r2kgOPRbe\\\\\\\/rN6HiukODRYDd+iw0KURyTHkbyChWXcw0jEsGDv8THQDiTcdzJuzJGBVtlI8VMYt7V5gyu2tfIzpl2kCRkE\\\\\\\/HuPCdVZF22lsr6fO3U1xEOvItOPkc9q\\\\\\\/giG2uz20L080P7I9bivJbCgly\\\\\\\/moqCjeQaQI8up8uP+dCbH25ijQORDrvQEKDo+TXR4yg+6aXPM1HAMLpSiE\\\\\\\/4GGZbgP8jfmSOO5A\\\\\\\/1x9LzeZqBIv7p94+srD6kDcTRDrvg1Czkq52kFBGu77CUBQ7mU13GKB7BzaKI5B8oFzVl2zzTf+dDvpU1Bx0pYP8uoNUqNtdxQub1X3iVUyIdJGqVkmfHTF7Xd+X9jCYFkA2zcx+Si3PTTwf45XZlztWhkT9pJSLbNJPud4u2+IgCkSlD4frDBwacmGE2rGENZ\\\\\\\/qQfmVASkgbN7pOsqsaU\\\\\\\/O8OeIB4jUBORslxRYT+jhv9+Uh0agdrmc1oUg0DfMUP8U+YMlItqq+lWQgS1kCw+T3M3e4Az1TW6CInr6qES2MQWo0q8ZhLC0BsbCX\\\\\\\/tnPOulFnCB5hlMqPgHeJmbPt\\\\\\\/Wxxhu3m96GZoebYRcaq3hfYqh+n5wq3G\\\\\\\/m1Z3IRjZage6BhQJxpcQ\\\\\\\/jvvr6SFoUB71LP3d2N++2Mqz36doU+XFObUxZs60lzwStE8uikJe7uRUPz6RiHhHVdGbcYVDxZ3ysHkloFbRzCQqwQtDH8Jl\\\\\\\/78lkcthz1BgWLz3TuG3v2EAgwlB5ygI+8WtQ0dLYeaAkeqhiZrv5gOzENvHj7L6LM1p23ps+jr1cXRvD5nzL7kq2BC0Vxe\\\\\\\/lQdhu7kympln+\\\\\\\/sZznvRFj6ZH1k+E6sCBCk3+6hgg0pLzlgJkr1\\\\\\\/WhiHtUHv7IcUyXp2A7WsxpfQjXzQsadCIzF5aWC9AQ09r9En7QuAVw4x\\\\\\\/hoHjkMf6nlv528ypC0KKMqMfBF1mCJnwgI31JY6oQg+15DfHKz9\\\\\\\/z1TkxET1I2hEHzcOwa1qfLSmAu78v0lbBKGAU82V9SEL\\\\\\\/C0WUlm\\\\\\\/xKh3fiRYz00mZ\\\\\\\/CGrqn7+BkGbklge3pvpf583LTF8Ek1wt\\\\\\\/dNogXEmAOsR\\\\\\\/9hwzXEYpC5eQCOh65XtHTzUMwTCF9TcMNxR+H7jH41nmzWyi9wB+bI6DWR9Gw\\\\\\\/5Wf49FWYDlIk9nULWELfRUowSMCdTApUCULHDo9tH5Xrps5P3lBLpgpWSN1tDamZwv3njmXWQNbxa8LuJqdrkzsh7HY25vWCKk6QFfuIXMA9OuId5AHrn9LH3ex8gpftbhoGhCnOKZeuHd7F0LoolLaV4VHVERBv0FEPy1kTHrr\\\\\\\/H\\\\\\\/oIE6NzTxNEZDHNtiQTKXNJwLbaPiymmRv28LSHZz2YaOusR8wuipuzM8knx4+lwv5L2f3PcasTMudOhSjPh4\\\\\\\/zZJNhvxs6j4qOXvGJWWYuE+gODWvtPYaFShVN7DQHKIjR8IowsMz32lwYNtLFF4jeuOj66T\\\\\\\/OM37qJotATuB2HSGk1Tcd+oMPIekPz7m+B1bf39LbdKKohL4QzJfMWLdJMmdAO+BMSohy3vUqd\\\\\\\/YN1BcskECiEXHYTNHgKLgLTJlLHG5XsdSS0DEpsXoUq5Py\\\\\\\/2M8N+eOYhsG\\\\\\\/Q9i5vH8REq9cI5TEcihlryNlnkie6Xc5RcgEstu58G2lSGfog+bo7obBrNCQrxM9HpcXwHhvULI3CC77h06MCj\\\\\\\/9+JNF9k85cfArjRD2KxmdZpCVcZCXcyxIfn\\\\\\\/tzE5ToMHBhALOZNczHhmaqDtf5Bl4Yo0R6uAbhisZWlATxRZkWf7ZZJkflWDzJl1eZRLF5ZlZGjw380+j3\\\\\\\/vIN5fwjl871yeDUP8G\\\\\\\/pcD4sXrnRuIxvTlRQ3qxtZSDEkEJs5UPzJji9sJ47oGZM+BwJMXNsqYA+pAkV5x\\\\\\\/49yxfUWt8ilLrkSIGYHlMZicn6y5GtWhzD8LUi\\\\\\\/gf+d2qZknKQyGwpfuxj9lEthYQ4asrPh2qkB7e4ckFZYs0eBRmd\\\\\\\/ngEIaEWVUBfo1JOJ6g4Fq2a9qv+0xq5Qtor2Ur3LQ0cBJrOrKTxJLNtl8pSIv6V5E\\\\\\\/Jb8BwFwIIWKwQ7N0p4\\\\\\\/mzna4MymgjrMDlAzq2zwqoMPfuIJD5BmSVravMAN9CEVVgw4R+FtI9givZjLK+SQrIgIHbn0UMZtF6rJE7KhWt3F\\\\\\\/eW5YGsbWyym4sKWhri0NcNwKgMJTOR8483RbyEqZjfcwqHx58KyvJBUdKHQtZB9xU1dWhDqYxAUNR49z3rrm721H4eZwKA3fyOE6cp4ldjCMd60PKzBffeO\\\\\\\/+Dxw2fot6V\\\\\\\/Bof3BtO0IiLZcGywA0FMPjtL\\\\\\\/18yUDkXw5RFWIByYhTge5Afr01ZfccI3lw9BEEb94ImXbBiO\\\\\\\/XrIK8FATnH3Xymb+EpqIyy\\\\\\\/ferC\\\\\\\/uqI+u62xzt1UJWPDnNc9Hs2zJzbNVj3DhpHOJBgLz3eFFmqCiBBL5\\\\\\\/VtQZbXcCB5kZqvJHCIA5mnhvZEJc7ut+0cRnmZdKssCIaybamn0TmQ4Bgo21DLFwwL+jZVlJxVdNfOdb2UUCOfi71uyQlpF08n2EDJe0ZxygAQXh1bVfFBhwzPS\\\\\\\/whOn8E7dPqdJOTzokr6xFNxhRh0IHB83lRemZKBMicpY5rnQ2hcyxxfQ7LB++IGS+2xN\\\\\\\/uP9DQ\\\\\\\/8KGARCOeJz9wf\\\\\\\/ezKJOk9U3GolQr1Lk5aP391f9KFT0KLCHt6Ua5FYfLPKtP7snrQu0SAKTAplzRJzM1QazzI0ETynodBoVnhmduzb\\\\\\\/53Si\\\\\\\/aq2RP+CZsEYUe2QzhD1lot7BYc2dC5NuHFsAVr671luO8O58KRdSsdM0H9GQm7KWlZYaV5SZ0CqvB6LkpkWLJgeAeXzNx9waxgn8oFnk+DUFo0\\\\\\\/78SvSNAHZrjwfTgF56nWZXCgDFBEwOH4hjf7JNK5vIzrC\\\\\\\/J28TlLL30\\\\\\\/mCcKMXIivKOfWoVQjNyK1m1CSL5tP7ieO7JYDMzs+ODTHhGrdPcKjrqcGSuLnljdhRuJ6qv4gKYJpYVM25+x\\\\\\\/OZFj8hkbj\\\\\\\/0k56iNO7pIiOTcqXF5KoDpzoOEWsxpEhyW2Lkdp3IglLMkLHUXINw+pdMX2T5ai+E3weVasI8UKS0wVSFtg\\\\\\\/V5a9lvr5lJWOrPYkD6UWf38h+dQB+METsJKPUC0V5JsYDVdGzvCZ3\\\\\\\/\\\\\\\/wkmcWoBLmGpTWg6D8farI6ms+dQcDeYbs5Fzq6brTjbpIgLgdHIZaa6qJylO1\\\\\\\/78PIBmOAu1+0e6DKdouDAULxrVz6DnypaWNNKFgAeEZAL9aGOxW1S9OJ7KNkQsUMezx+9udHdjA4cWjliq7poCdniTjcBLGGvic7N2szA78Nc6OwHcHN+SnTwtbG+KRo9x\\\\\\\/9PxmZRBGTaVXmT+zRfE97llB9MP6xHhxWh2AGpX\\\\\\\/T\\\\\\\/0eheHHdZQ6G6jZ+8dRF7PPsGBDicAGp\\\\\\\/6XWB0uw7yRfkUgYhUFNNuwkuoPKt4b4AZepLbAabJmeOecYqrlklWzrWxmPaEmK0Ro3mD7zJTsVXf0ut8xVB5iPmo1rNddJ4SrYnb9k3\\\\\\\/hkoDCK4x2FoebJrnYHh+94owe0Pk6g4ETmO3fdU7lc9DodEI+CyPjPkJMVh4UlkIorbZvmppmBLj1stL\\\\\\\/3yyHhvIFI3EopI8gvAjB3uvXLYum5WYYGxeMz2GIMHHr9JElUdx3O2po+UojXv2xxHseqyOl1Mh1uOyD9yyw\\\\\\\/iWmPKipYHlCZzMHXhqljO6BnoazG3JJmy6Tob2PEKPfzTWr3AZV2vZUgfOzSjvZgmHGuhlaa8hGjjODsquSSD\\\\\\\/klMqUfcGF8u2pG0SlhtBkM1jq26vxoj6NcY3aHBIG5tfsOgBQz8a77a1CVOhQRb9O6upAUK9O4+z4a8jB2O\\\\\\\/fJ8eUf9eqcPyNyxpvWHDyqW2xo+nxiFO+AaPGzSINSZGPhYq7jPJcVt6sFVCw3xLodRHfHKq7Sv\\\\\\\/6IHqOekGYi3axTKZcHDJKa5FNyWjRIxaJcEjcWpaMFECGWYS3CR7HJ6OJ6l00yxAVks0nRudICAbBBCrVAldIge3Qx04QsHr09mdx9PQ9EkfYYGKw0XI5vH8eiicc666PSEFeuOUVi0j6mqdaeWdBsled92oO9HHFvFPvy9ZiBytgOdwFnxtnNJHxuKOVWzE4kW2QX5I5\\\\\\\/GqMp09\\\\\\\/BquL+BDn4Ieb8n+2QFD7HxhLgnhB9wHFsPAgl3J\\\\\\\/zi+cgUTrIFaXIZKL4L49JVdjISUsvaHQWSMRtkjv5e6G+IAjenLSsHkHAPK2444JCtUA0m3jvT6a41TUWRnk8ZhZJp3Fyj3PDFcxbdRMltAhay0hL9GLcrPuqMb8YyE7n4aYyPHB0OF6czNa9HiY6UGFruO7xPzeRo4uu8T0HnEtHRhubPeg1TKAPnbYThOOxt3sEsaBftGVGLY3gulledF8NitFtUwo7A8tIZ6WeiKrjgfVFrG8b9nt8eSUmHodzIhvjXan\\\\\\\/OaGynsVpw6Pbz7fiTi1Lnz6l6PDsMSX5z7v23wGIsq06NgQun4xt373HEU\\\\\\\/40IlgUCcvpzJGgQvyQg59TYo98fXqInfRNCRzhKs26To64wZysjWchv6+WDYdXEo\\\\\\\/TJAEZ8XzebYGOXOjQg0awGhbUfj8pnZ\\\\\\\/XEXus5VwBe8PLrAXbU2HInKAiLsmqUzu0hkduMRMPniAO0qxq1GNv9bKCp6H3v3CL3AvOn7SHssSNoHd7b\\\\\\\/3hi5djCW2V\\\\\\\/Y\\\\\\\/axk8JDKGBhcXL5DeG4ShHQWB+6HIxq4OpaKK5o25cWCBX+nk5xtNmGb8\\\\\\\/rNqESndIyOilZi+KyBf\\\\\\\/hY73aC83WDSkFz13eX+sAGX\\\\\\\/QIXevPak1\\\\\\\/jExCbnBOGQ8fxXRQZQrcgAfzrguO2ZbPQObcLMFOAzARsOwGs\\\\\\\/268NFEU1ODCAouKWTSYhHEf478S7ZZ5IcKnDNTXeiStQtFhpbDF5o3OvQu3Hk4GdV5FjwgirY4BbER+fwUxECaoW4XDGtGFioVkNI8qTyDt0xSHkS01FGBqzLopUdHS49hvlVn2URBrLEkDNzbEHi5\\\\\\\/viX9C5np2NUBnH+4Q34yFHGszor7W73I4je5pYM9EXFUPECxspAMAVW8cz6lxdANCDJUg5feS+Ep5IUSyOXx8vhA4v2xWZsROQ+VE+WtdAcg+XYazfKGeIZeIaSVnRrZqZTN9XKW0Hr7UeIMoWn6HvV0fcd2c7\\\\\\\/fGy5n1yq\\\\\\\/rdQwDJCt6wiHM8ioV5BxrGLWcLyUdkKK\\\\\\\/DKeRSR030UiU0EmafvPWmZsLSnZDfENjshVl81kDs7zIKGkvwPrtou7Q4EHiFCMAWk2UqiD4v3z7\\\\\\\/lsOFJhJQypmV5PTf95RBPcEuUHWqzG3F5GQCYCOELkWMhioYPY3NvNQ9x6GOml4YsKvopLd8oguhXPNMDGLlOvg3NJNjlxAnCKOFfBzMce3oSzR+vb5\\\\\\\/4DYV8r3G985H5R1Y+0UpkNLmmPPJufqmtpEPZ5SfMPwu2dItAOBnNUwyCFrQs9pro\\\\\\\/+ET1XFtz3eioWcetvo7FuCZHd9Phe2EHGcC3YdqwuuYmu7KFlj\\\\\\\/t81vFyq\\\\\\\/+lI5IMdB+dZQJV3J+JfWmQHWRqsm1PVVYygMKB0AfhohCqbuJRecoYOSqeJloBJaH3SOrXxt20k0oxIqX3JgsG8evcadKkJhX9FLAZr6bspn8WiNVHEEH1Cqe1WoUAKZSC1H7cK8C14\\\\\\\/QFz3PgORExhcTsPVTWKciUWVaT9aObQRHNASHXNPw+yhYJa1mSkIr5jDJiHd1OR5XRIhD943bQW718nJjoM+IXNQpJiokfyjDpwN6JAk6ig94KKSP0TB0eT\\\\\\\/v\\\\\\\/mixgtqt9M5XxVr1VuanvnFBuSrDtPqd3doCtFirv1bj+i\\\\\\\/+yHq8fZpVtlu1tNdHOROKgyp3s3tt8QvF68RcDEcrC9cGaOytbP2qe3P4IlC26M0CWb8yQINX\\\\\\\/anSeY\\\\\\\/y6qhLsVaQP7v1BDawESphJdb\\\\\\\/jP+Zp9BHdek5wqClh5jFA2G\\\\\\\/J17H3MvdCcd01FE2pR7fif54UIIvL\\\\\\\/39goFBDJXw3q1vXfrFS880pz8LOX+GqjD0KuvheE0ZjHkIo75gzyGoBjTY6dUDbBfr\\\\\\\/E8cmIcV8C9WQ8oQLsDIP+IRfu+gXRKzF0sK\\\\\\\/JT7rHCNkUx2WTBBfFHVoUkSKoFY4OJnADjwRpVbhlL7K1prHpIXks3gd1+WujMqlbTlg+9QmcX1LKE0ar8syioR\\\\\\\/wbJ\\\\\\\/v4ej2h1\\\\\\\/E3aUC9Mt2eAItNewRzViucbLfgCc5gkz+gnIO9sxUFN0sYkRe4uv0kgnH7Gfyr0I2YuVt7mSWfg6P6FwvV9Ooot0ljl+J\\\\\\\/O3OLAqbNX6tVtOe8sPgs99IvlU7HzgdRZ0GhchGZyJnw6YxOF+JtfFUENJLpL8Aft2xbiSZ81mSbvvvGHEn4+cNkm6tshTdBagiFYqtG4Q8m+4b2Kqv9V3jPdct2Z3eIafFLXRUmkncXM09PuuiH9eOf2W2RaWpj1RqSbh66j2jK3\\\\\\\/UkGOfc06q1fymQKPEh+wYFKqpm4u6hI4TtF+6SU5wYeMA8rqd8NlMcTy276Q\\\\\\\/h0j920aN4f1QVYlxryJh99+k8h2UVrN7O7rCWE5utC3HB7tAQcI9GDnrJ54gnXPeDfBH39nn\\\\\\\/oHADf+ra0zWtKv\\\\\\\/UIFNWyqOOuLf19uD7h0t82UonTxn2UR8tSMbzV8qUu4PtbYUEDUJvVAKBxz3+G6dBrIkAGb2CWndEQnVek\\\\\\\/BmMKFm+w6EpF7YXxSQuTE+bsTvxhTSfbP1g5LVL1VefiHW8vg2p\\\\\\\/jzKF2os0AHGM8P1urXkAZsd+2pHgtpbe1X8VJD7cgUWoPKlc05MbbPI35XTSy62NBllOjPj5XMvmr4eakMpXvbJi8X\\\\\\\/xeDszdCGEx\\\\\\\/ltb4DdqTv7VAaS1JK3d+WgEdcXji3xl7u8RFor8V6hXrfBpxb89CzwbBFw0ewuHqXpXOmGdDD\\\\\\\/QvVYZVJw8\\\\\\\/jP1PMp49WO+dUjUrYIlr78lfStO4HGQP4pBQqcqyYXqgBUhAruTwPCwfDFI7ESXS1qa8txxoPD5knx7u+G4BEXcn4A3awxAncvaygicb9Ip+uo4rG+nqqimk87Qn5oUrCak4kvEh3ptNqeyFP9xnhyXhjkNH99JgbKx8OgJ61fSudN4gylvMEnCawnq01+ErgbQcutobo3thCHD8qeW8N\\\\\\\/M6sK2lWcuj5oHOd7UgwYYsTyIolkkN2MWQ+L7DYXHY2JOteliMUcw60TLa0mFfnJg5Ac69CzWjnzKdXgf\\\\\\\/U7NJxQ48fdfQ+HQzmCOlIZSj7iIu8QF2PIZSCDNAn+iJJA1SSsXzoK7hHLkEnqY\\\\\\\/p4nKhdyVBrPhj+BG9sdNrcPICYvtHsRzBOPOFQIfMO2z9FkvpSPDSNuchskSE+je235oFKyMK3MRdgnNyVv13EFmCXhkanPGtr+Q6Y18vaVY9dbr5deOV0dwv6c+VKkNpJZ6kFhal6bDsFloQiSrRRddOldMqcbb58XPZw4bQapWyTBYCVk9XUpyapCsif9ltj5jIHSHVONgD7fwP5RGCdyPSg4oTlq4QEi8nyCRTK6uF36jZFz00WrO03mZr8jFgvlk8FGW+7PNrw9SD3oLPeGHdUgwrZu984rJ0uEXpM3WfFOfH0TVD95\\\\\\\/2ntaZk+gCzNLX3usVO9wqb1BodPSwB4+gbsOHGsHQTj0h8eon2MxD+5qV31\\\\\\\/SOrY9B1ii\\\\\\\/YPiFtaZpZN2wZGab1efpCrnjcjEbEWYwo17subbOoZL3obJbOuwv4REBOZo2xxhxhIkon+NWtCFv0sKM117TSj+zO91EhmUTT909hXlSsDnNtLK1hvyHOVW9HSMMb\\\\\\\/i25Ggi2XE\\\\\\\/J\\\\\\\/SUpjbo5MWG+tLgpqNE+AUeZo5xy4UuglM8t25HghBo319e45PBKauXfLWJeUdYwjd+R6o3xttXvS0Ewu48NFyxM1nCPq44L59UhUbi5yjY2t6Kwj30aAbbBz4uAFHIby4C\\\\\\\/cxaMLEdUZzUEFYx0ZWAXv2tWgTHD3AK8PF756+Psmr9TXfgyZ+itbH5d9ziuFf47zhQUcQzxDZ6RedOZVniiFAJW4bGNgjBOzVSMt+C1gxZnoEOGIRJSz1DuB01WwmOdvJGwjbdkA1mbPcm6zWv4HGUI18MrL9K\\\\\\\/ASI2FtiX+tH1DXpwMkMMsMyIUeBl+ts4iepWrE5lRASfgoPknBfGQ72EObKvAd+SZ1FlQSQK77jo9HiOn3AGsBtikiEw8cyOoprTo+mQvDfy8zTnJUu1n3\\\\\\\/0H2ZtE\\\\\\\/8MJnulbxTnHAZqDZiU54U7XiCXRcv+73gX9TEiHTNTA6DbIeY5bkyYDZPxi64Qq9bnpbEwRV3y+S3w653eAL7MrOfbcL0LjNkwpk4VKfsNDr8sWB9wkmU\\\\\\\/QEuHKYJ8HrLegpCsoyJGJABN7pjUK5W4uPk5F384OMMdqf1U0ZTdwjjcBGO6d3UYbq0Xf7wuITGjWzQF9dXL\\\\\\\/mG\\\\\\\/m10dLlkoVtezDenHCaADVCgvwzwrWaz8H6oAW57l6vTGlImxEGQfo8jtkwQulpp0HFS6V0sScQEoeqO1O5xkfMeVytWsOkOHPOTADqSzual3BjmwqhtHm4+D0GluBgrmj6DUzIDsiEaqQlAYrtUM+8YmLqpr84YXpG6EsQi1cYN9SE4PmCMtgG\\\\\\\/LcdmfNWST\\\\\\\/3xRjBUdCyRC3s5V1K2Lv2fUvEdVj6wHUeZ2rfJfWHqgTH12l6FZTdDVjeo5ce\\\\\\\/UCSddohmF8WGP+3G4wsiojo1SeB15Mtxc6egJF1a2dCnZtwwhnPr3nXviMVKoreKYfVzTFA5Ee\\\\\\\/o5SKseLGHFoeEC18pP83l98z7BjlbMQbIc2JWr0QUaByTw4rwa7rD8SCVQ3mWqTy7MrSDYa6EzIMSvvrnLzr4AIewLI\\\\\\\/9uhZ5WVRGdYHCzFXxRL09xTsOfrfjwV54od+9uTYhZ7Mxwkwem3oD3XpGd6zP+\\\\\\\/bxccj6BimkyEU1v0Zhvx7z0jKMXOxzuj\\\\\\\/YgSGSumlCRyj4gUtzvRRlyx\\\\\\\/xZSaXPCIAyJAhshsQ1MY6Kwdn2rgeBpjhWGmINeKIVgIgnKW8qfe5lDHeTLTOVfMuSox1q\\\\\\\/0H2L9k8EVY3szBBOeB4B4ywx4lVkkTT8yuSCnx0EF517YECXdiHHAQN5VkuItMsrxP3lZJacLd9FJfOZdEUd7DAMsiPBoluSGwR6v8I6T3HZ32279ZDau\\\\\\\/41wkJ65G3gt02jyujGQUSiQuqRgCqokn4IB5kfRz0Ngv+VZbCBAs3y6cDY\\\\\\\/FNXCcPdDa55Af9l+lO00N0rArg2wQ5mjU6SqSnxmpICxHftnd571LbLF9JEupPTzi2I+3O9Ahhfsc2Ej+Y4w3BTQct39uo86eH7gmeNFgofSMuERWoiIMQgnP63iGcopeQhH9RkGA\\\\\\\/oM1RyV3skrdpXq99GCtNFC4ggthXcV7lrKbP3wS802BJrqw4S5wvwatJZK9OAdm1IwmY4TZxPXUDlQ4cEQELSYxOnZz9F4KITsxRsnc1zp2KLMJVA+iWJGYgb+kTMuxim\\\\\\\/iLCRwjJzhoqktj4eupXfAUVoTA\\\\\\\/ByU57\\\\\\\/hRav7LNTqP2jovKZKGkJGk0yyqJ6xGAGsElu0OG4tIvahq0GALAiBG8rdTWnNfvfUYqtfzAQGCrIWeG\\\\\\\/lU3qZcj4NVFsk\\\\\\\/mur4XT\\\\\\\/AGJ2u8dk1higpResWlH\\\\\\\/QTuXUaL0zVchaq77xztdoVRSZOGLTX0lndsbaW4jfHV0KX7F\\\\\\\/CqMRVzQUE6D5Zupkhzob9HcrN5HFJ4h7kjnvPSJgGpqZj+Kmio7pA1fLA9j+d+xkW3brcO1JO8q8wO6lRITm0n1Vp7ku9GssYhAjgpxNNOUWcBW\\\\\\\/fc7CW2zZ6\\\\\\\/OOd2kX7DNwjurJJ\\\\\\\/eaoRr8tjbtsbsALLLGnVv68vRXuLCRTJWaaUDTIuwJCiDLeGhABgLeF9BtESG1wN+4r8AB5T2cKEo5seNG827dPA4iaSh9YzWNCxWzKiXVg1pSciyY1blKZr34zcZsGhC+69rfl+XEuR5ftecGGZlJHTWtqBPz2lHeQZGLXNzFKECKNqEcHKCAGOVUnEq0tFySVTDcvNMVDavEc5vzEG10DKuMDzNyzIRHRCTnNU+5H0XyVnwK\\\\\\\/TMCNh4eXWO+kPBSYkklEaVj0GX61LHs+vAU+j58i58JgPp6YUP\\\\\\\/8EoeiSpZzyDr8s43uPkJmZzauftHnGsl8nYIKvObGEsg4uCU\\\\\\\/iC4iGyDifvH5njlim3w76iZ96U+NyQB1R5X5mCZa\\\\\\\/+I94hMUacIJjA\\\\\\\/KZuLDSDTgRhlhi4k5xPvbWns+iOeLRXzaKQIQx\\\\\\\/2O7TCxzb7GbozFtqsdMd3pqtEyRLO5CPva2kjeHOwkgNCdF1IBvl+QGg3jCauVorxW4qp3rI6s6PFVe1wyx\\\\\\\/KZhQSWirFw3U4Qob1X3BBl5KEcNkq6JJlyUo+nr7ZPmMsjrTOe73y1VJUwehTaL0oEVEXPcRCcoqUrpdjA+MDFGvs+WS7Oe8VbtbDTW3sRoG3QJpWvkaYvlG+B1t++w4Rzs+WdE16AFq8xyESFCTjMsukcmKtqau5yAxTIqzNPoXFWtwqbhqE8HAJeXvZSf8sIlWBx1mUNAWjhOb4KFgGOmAgQpDXjYjYmNRZZ3zNlkqnMchGo1Bt4HODGlXuiFyVXP\\\\\\\/E25O94dELTR0CbnOMkQvaTI4ySmKWwUcX7+Qr29mjSw1Z4N3pZUpKtjY5r98RdKkE6DeYnaFOQwZ2ww\\\\\\\/pNNYn1f5bQ164HAqw93FO4oaclw9mLR4jhqoSHGFX6jvCr9372YAIQDlWynB\\\\\\\/NqgpEKFT+vZY2Uozv+dnmb1h+F+GxqW8A4hPBpvEZwpK9xe3OarYOr4pCqK\\\\\\\/LiOP7jFnY6ZNdPccv14DsfFlvBPX+CsvzT01jBLn70VnHPe4x5CtSd3SML+0xDkeYJkaQAjihfUtAvTGmzjEzLJt9tc\\\\\\\/FVGoPB8fSIRQ\\\\\\\/lj5MDW7GOXiJwb7fS4aqiaGRX5bjyiO5caj8u6OmPjiFkeEtMXa2O2o1VEbjuTht0xFJhayNQHX9bDqWlw9+IWCzR\\\\\\\/P\\\\\\\/H8glg9HScExnm4vHIMdeEdh6j4Z5ui4PJCyQkNO+zcjaR26rNsFoKXLKs\\\\\\\/77pf7IuZ4dhWQhJeiFs7C7FxEdo2ES94CyQ0715UtkxOnd\\\\\\\/qh6yQPt\\\\\\\/msF2FoOuHE3uHCM0ABCYfEcT0iDlYp\\\\\\\/TrvTsVwpf9f\\\\\\\/ZznNBpFy1f8u4vOTaSe2sew\\\\\\\/trz4I5YzLMTNaWs1TeKqiefmzev\\\\\\\/qgXNLeW9sMgpLSEJdwt5reoMSxtnewPZL4LXCHvfz4Hbo8n01YlmNWvejD4JimdR\\\\\\\/tBnS8z5lSKmz\\\\\\\/K1ZJDgud8Uv4PGBl5anzQo\\\\\\\/PWEsvhhbH4KMxYpS8mD+zsCH51u\\\\\\\/7VTps4Rybx1lWrPIU0VhT3U4z+AwoQbsP8vd0FvcPjv4i0CeMQqCFy7amlG9eQl9Du6En6kZRY2azCDzDSq8Qqb+QFtahp1aQYZqtqMIvcBlXuxxDnrrZMhPM7O3HCsMd6iw3kLPw4tzTnE6XfQap5eBEiq2cSv6NHsnWdcRVCU44VnF9bnNduhhoU2\\\\\\\/rAQHwjtQKumfU6O3MGJYprjvJTVhKi6ZNpPXicGyp1GTG64mtBbd5Na+kVjkp1zpVFY2mOkTCk+h1ZJKmGUDV7avkJS6U2Kn724CczQ1I4xrTk8Jg+yClAGdOulVUh5v6Tt3DPBpea9IIj0bw0XrTL5pSVx8t6RQGU3WHsiGfIuNurysC1PIcLUrHg1rP7ozVVUxGwfuzVgMg9qCg1UuYCMr8gywwtoLZ7ASEke47dZ1q2jT+u+EEP6eURIG1BjNRY+z0xvm5\\\\\\\/16RsBywM5z7EIxKziLFOVki3TKgAb4AHumW9Mkh3AV8L2wR3iCSHQCXtXovcBIA1GP6CfgQnoyu9Wet6ZQF\\\\\\\/TDxVhJ0rkfNrwbSo7Q\\\\\\\/FOZVmW\\\\\\\/axZpZVQcQ4o00VpoOcpuicVrMxQITg+Hh98FUCDIjjifldpDGpH\\\\\\\/aXzt5Oxx65z5YG5qavvCmvE\\\\\\\/XGoB\\\\\\\/WAuTuNuhImuFVzl5fVY\\\\\\\/mY2YaffR53hjk6fFxV5Jn8gTuwZTGY91DVM0ZBS6H94PRvYaJZWYvck3Dv5lKm9futwhPv4w9VtYKebGIWLdK6PEG0EWhpwo1TBbQDtL1LlN\\\\\\\/x1LKxZdFGTqzU44TGmBRnuw9mh58R0nssIHzn1ZCtV2YoRVIfO2Ti\\\\\\\/q\\\\\\\/DlyfQa0o0oe1+0Pi2W7YxG+hFW6LmxHJ+PYI6u6YKSvIv3ymU0glfIakrrSeIhAIyniBZYB38DUbb0Uck01I+TJuzax7Fsy8cX9CliEFDoTeCZlhNlks9BH1krZAP5vmO6CXE7gyQj4THEvXN\\\\\\\/cePML4dH6f4l3tBYlMQf6pf\\\\\\\/PFHwIQT86fDE0\\\\\\\/0e8X1FOYdbHc2gH0ELXyUDSivG+VgjbssZYVasN9TKqOxXa\\\\\\\/RXINbI6n6DQeXSnG18jVSedrHDj6RXCJty7KVosW\\\\\\\/zU6k4GAgC\\\\\\\/p50QK1qoyiJs\\\\\\\/5fbJoEaQOk2mCW+2NtupH84TH\\\\\\\/FEEG0wEKRnQlw+7ZH1i7dwJmHH2KySsSuBUxHcZFG\\\\\\\/d9RyygQv\\\\\\\/nWFSVwMPITi0mNXR6vrY\\\\\\\/cKb0rnF1NI3vO0IsEbvPypz4MAA6EHm0ZI9b2EdlZTykkH4lheAMj30PI3tuJ4Ki9J\\\\\\\/H7YNW4YUDK3tb4mgwDOHx9aFOa2eLP7\\\\\\\/aQiiV9zd5YzJ5he\\\\\\\/qOfDdPZkQAG4JD4IpFfhueEcAlTi6UkOpvo\\\\\\\/U9FJuwB3dJrhY5Ynjc88k6th5d8QB\\\\\\\/5PIN204Px1sKFq5AqDef4lkbyN9ylBR4LEOLzqpQv6gZpCv4u2raAD87rGIGyrdLpwTwIvqVFPdTzoJ66AaPr3KSXmXm2Nts2zuuerP7Wd4n6yfVEnMrNxaKENXhuOWhT4AfsxvlIqHpTBjOm\\\\\\\/OL1MAqNj+q+RzQ+GHUE+PZiCN5CHtPuUNpbSvp818fsxdm41Q7UuCATo3EvZrGLmopyrXawI920Bji98LGmkYiTAnaG\\\\\\\/YWpZbNoHah7HxSuAkw7cO82h1rZlv2o1JkPsAGSc072GYP92Id1Bg9zmU7kjGoDgir\\\\\\\/dbg8\\\\\\\/CjJG5sakttJeOd0zUQG9O3Yhdeprcd2tuaLOaSOPPnIW+em8nIy+y+WbqVc\\\\\\\/jEx6SPlyxM2AwcWnxasulvJw5U8q0AAKJ2i8O74l2dfQhF5DxUXGcbaM5Y0It56+Hk7KWihS0nubC9nf47e\\\\\\\/HZWIrsC6v9CkcsHQD6832NADnwWi3jh0QPkuSKPKSA+hl1XlsybTmc7zTAS1uCpbgg5l1+8YcpwcJzVZKi7S9AK+OOE87Lz6kDgENTS12luhmryxT7aY0iVVL8wr4s3Vb+LXccrvq0OWxF85tlVl4m22dcJ5mMICfv4TVyOuaFwB8hDBerCoqRrYII601HfPtY9hed2zeKwj3P04BMhQZBdyuIemEdEHur2tV7Aqkhatz+BeegUp9SDHNzaLwbFWpCPxbz5xRqeCDf5NfN5arQkQk219cIQ\\\\\\\/9XWckHXOkGql12\\\\\\\/+xlmpxOOJgbNfaPyNuxURf\\\\\\\/r6aHouqCxlrty7a9SV0QltWwcx6edisnnjGTwZPkmKFigP8bqJtLEVp62B70Ii9l8kngYMMOMHgwEVyi7fE4dBS+z\\\\\\\/mUcgj2ZVH+5sP9aCnt4ItiZl2JzcLg9pbP7Z0Q3fWi0ZJj4fmPkuEAu4MWiZWJJyuMMrcRD7cl77gNngoq+dX8ZnhONVVp1tZC\\\\\\\/6BobTcJCaQF1pzvPwx5k62v62inCBmG2kUzg44TuFiTWTWi2O68fQXvjcKs9T9+8RxDTeRy5+y2BDIX7aCLf\\\\\\\/a0Mi2QdLlmBQSz0Iqiog9NWKA1FBVf116Olscn7u0QHnvJcQMC9uz2eX0tJPYfOXNBBQB8d6bPdFC+zxKaapDvhqHZULFRuh\\\\\\\/1ITGRXRQb+bqorEun7esrjwAgHKJq0wkt5jZsvH9nxmkDJ+p28fjzBWycXcPXwPau2RcH1okwFSzWQReDu8+ZSA8uM8JyWQuIoOuiiNANH8IzdYv\\\\\\\/UYrWsAt29LVUiaakTvuz\\\\\\\/RYVjTpy\\\\\\\/E+9Jwy82P9ja2TFN\\\\\\\/GwaiOBSRXtD0u0USkJhoGpBacqz9D2\\\\\\\/UNQNDXxUjUjGffpIeUKjbRu4BsmVhez7g+B7eArC4N3rOc\\\\\\\/Meyz3X1EnDzK1tcDN\\\\\\\/2f9fuSwqSs5FGXr3Qy7gWSucOcL3bRqdbJVa5l61g9XPX+K58Hcy5fUtHVXWqqlNp7GeOELiOmCYvmm5PbH6+Y\\\\\\\/jFAo81HGQf0xdJTJNsNtzpKVsThNusuCZCVT6MMJ8plzBsyCGWfMWYgHERcxtATjjQrcRhkEkXjCBXjWaqH4BlJHpVNuevgHXNUTbEmKAH88nrLF3LkgDM+JKx0it47DMG+jYTYO73OJEMXbYxAZsLi2nz0oC+62H267CKcBSiMpLmaOrEO0bevtcvfrOIhRpfqG5Ew4d2oR9wXihYiaawNJ91pm3Byz1JT6px8VrBDCRsqdG1RlL3w8uwHCXNoGegXhS149Lo2yb7DcjSPJlTb2rMN0KC9Pphr8JAy+1oi4ES2Yp+0mgvWEC8Qps6wdXxxpqUoDB82Rcvfln13aICfwX2rcsHrjuujbyZM4sMXXhcDHuJDjAcRMMXxNfLjhSPyP8vVgCX6GnoFPuMPLTfUKK0ie4I\\\\\\\/FTmgxnzAfs0lSzgLnXaTBfH3v56KcL3b+jflfP3qBo446XetRP1TsmZOnX35oi3u8Q34rQQWpKhHDtuz4uhF6wQktZmyWUpEVC7O27nBk3AColVHTs5WFhxP6UjEdF5YbKVSrdNnpKOznsgK\\\\\\\/sgtzIfF\\\\\\\/i9gaDSDvVJvCvzSItVzhjCAWPqeIAtKtIlu9G14h+6zbT8Iqhht2GvBozlCGgJ\\\\\\\/u4z2aSy3oQVLLlYJ1y6R+SXwVaBULX\\\\\\\/FXWp\\\\\\\/oJmhj197fEgbdNMvGOJNFT8Lw42AS4vtDEWuM9BayzIFlGS0FVZpFOYm4oSpvfT32djp74x3VTcIDZ4x7ef7dQVX+nbY2+AdtXyKr3+aHnK4EsR4mldxhP+tJwYzMjIcJfudClUej5gEHwoCb9VMDhhiikUrDm8vmHdfswvrLE+eKflq36XtBmmgCDBJPgsv9O0FWVTlvvSj1\\\\\\\/1KwmxaCw0LyJBDhkTJds9o+eesjkmVthgtZGJMEBgJ4+OAp3yp1DkwftL9QPNS5acLG2Lrtpwx+vHeKHkYz76gZI2BFyUthYJHbEaqaqIYRPuDNE2rf5UqrmhvlSnvI\\\\\\\/vru4hspkgmsKiEEnipVaekK\\\\\\\/\\\\\\\/+NmdcDXoEQlnbVOq7bnuO8V9XQYsoYtym7sX7hULKAwcAHpOV7734ubEakHIb7WNbzl1AG4dCcy0y3tP9awGB1pFCKwqIIEh\\\\\\\/DDJ2sHBuvcJq8GEh0K\\\\\\\/SWbYrRcG0Z+6XAzAERihiC17oK\\\\\\\/qsoAibtQOfrPDWpTg8HQ3K+z\\\\\\\/jqojcYeFvcplRRf1Fe5lYnQy\\\\\\\/ap7GdLDp8arXJi8L53+R06VHfdUaEom3\\\\\\\/xnHaBIQXqALA9+\\\\\\\/LVEc+NM5PoTmAd6lDA6fIW6uv7dLdbT6mSy+6ydsH4ZiXHM0x8IhykzbLsJpQCX9tWD2KI0bsbVIjw5ym4yeAjeBI229RexCOhN4jzRwkia72nt+4cGzTmpw0sPgFx4\\\\\\\/ZkeL4chti1cf2NyJNaW4ag0zn6jezzprQY\\\\\\\/+pmvVKmTcWpGF2UZpow0S6bH4N2UlY4CGinaJujA9yXYu4lAeNrskALXCXsvPojt\\\\\\\/\\\\\\\/fYhxQDEANiLhBUiDqs5faz5hvux0yBfOFPi8nQO1QGslKEC70zFzp3SDyYQ8dEWYnGrZAdes0nuE73aa8d6SIdZzZOWbGcSWo1vf3nHMbc4QjOD2xmCgZuvlJU6ZUltMDFkHnsCls7pBbreUd4+u6KyakiO0v83Jdht3agxBO53k13+6zAsZpOFJDtyIkPSchPBYevPTVE2Hs6ihWGO3TkxhTVbJJKEFWcQ34qOE1dBfZnfu0TlIBT0L0Cx2PHXY\\\\\\\/3ue6XwnfBPCtSaGizAgPoKET9HszEqMMzIqXtZx25O+L9HmHJCeD4yXqZgjZwqhrj8WSgndF3jjXGOgFrj4viLo5IO0wX09CWrGiufNGX+eiH0XG+ARyTVkw12jEOW\\\\\\\/PzXSUfHyjUBJiu03vAwcbWjXYhqq+HgkAgcvcwmIwi20Oz1JMfur+hOUk73OUFAm6KEpXNRhYLOTC6TJts+Y06Pnv7vt+4JVgXr3msn5HmFWP5+OMefR0yfWB7pZJT\\\\\\\/nIYun\\\\\\\/\\\\\\\/ZALrb9ysB7nRKhwYkFBQ4SPB1c6wF67+qrMYnu02PKQlbcHHyU2oKrjOTP9O3Vf7Bc9rTjTuyo2AFnj\\\\\\\/hQrhQVS3stjj7u6XHIRlEHCOps0CpxjzBpiaUEIny4YQZOAYdMOyrTMKBCJI635SXhTbRymU0T7XdOeywbOlNELhxzieO4A8vlnj9elKi+Z6sE7aJJepz\\\\\\\/lrlHVy\\\\\\\/2mygXRYODvQxB2HhbSTC333JSTFhfgUesZdudu6i0UFaKqHsmuPmr4z5EQhYkjaVVlFfV33phUPhUF6urBww4n8YMG0JMN3I1GhFArnumRTVr4foF\\\\\\\/CQcLZXxRlrfrq6b2Dw0onW2tu6PGSfovxaKg0ZKQsB99klsMNxsZafLGBE\\\\\\\/y6sCV9Z7KbVyv+En9yI3FQGAS5NfYD9EzLVJKSXpL8D49hi5XP8U8GtfqV56n0z9N6v9A7qhxvqX2xAp7iAaZ+Jg5jTDYpYG7LETKkI1hCeUjsHGk88epPTR1PVcZRmnGtVkhblSti+YRf8fZbL6omRiXFtkym9UtUjTfllNnBesiDGd6XbDQgdEr+ng68CfpWXhvzI5BITg718OpJT07mbDN6\\\\\\\/qRzR25MObkcu8IrGXnYmebuBnf58P7793QlPdHisHCnmys7edYZb2Wz5dGpy7aS7DSzGIAlVTag\\\\\\\/mv84iH7ks\\\\\\\/qZC+O14FwJNDCxlG5eJ85ZRsi4cPW3W1IUPdIZm8+gGSMbjDn2VtI7YJtAN6\\\\\\\/Al8TKzafWDxJeMPT3EWWTB4mi721MQD8fBbk6uikOUssvLrNCJFy5luAkKgvhGUkzIJddJuHzVrWMuMhi1+UWk98\\\\\\\/2EYkrWtGZxRL2WMNNrg\\\\\\\/bzaQd0cjlBkXg783THUkHRiSmPhn5epC7am7SNDQlhMoQSWeqiHfMaynNI4tJvtzFcoQ71dyIRzexjJgmsr0IGIkny6n4MRE7ge65sk7d+xE\\\\\\\/EJg5CnLjVbD4V8XgVzg7ZMaLZpLhSrM5euPf24YsxXNSD9swpE+pDIO\\\\\\\/kF\\\\\\\/hGq1w45lZtAvlN6gHxaj+VjYqNm3MVPbd3Ypnv6fCzUDmwbzn4f4pb4wi6V41rBO4i7OIggU1bCnuPtkK10c2ihQpeZfvaMSS5CE4YLL1xdKEmr18NN\\\\\\\/z1j3Mxe6jLB5Gsmpth50FLq3n4K+qrZk7WrYJEQ1+aQJT+c4dUtt8OSXEIi4Vp7D166YyIp158V3RJuse00r3EyZ4f+CKGJBraPzcUocG6xJfYnABtwIwMaf\\\\\\\/4GE1tk1b\\\\\\\/fZQ9UJ4EcbV9bsXebdCB1c8IrYdcUMq+2KNd7tkzUu3uzRpeBbZheTlmUaZqMmVDjZokpcY6ZBRqdzUMhnQWSWGzJX4BR5HhIdpU3RU8650awtPtzucf+vo3\\\\\\\/M\\\\\\\/4E5InWfunl\\\\\\\/fk4YgOS10+TnupYD1T34h2cOHD94JOir2uzblCkKyL36L14dRvOGX5GduJQJnH9IfezXQ9I\\\\\\\/xw99pyuqAOQKp1C+xVhv0VKLY1T9TvxI06K9q2khDiBOx2A5jiXsFqC+gtQs3ATdiNyJ\\\\\\\/mFjwd3SMSLw4OmHAdACk+gqFp7kD+aljbdR+BMIei\\\\\\\/bQSZVPUXImObODhglWSIb3BvSDk7HM9NE9vaxCmPc588hqIgOVt24GmT2Z+EKNufge7IDcPQmLzJtMkmXfXvV0igUhSa9XhwTmJ72uPIh1+PVtQV0cpcBm+i6sc8Qo0lkf39k5Ok0MoD3wTj2S2VvTIWBq2Un2M8Nk4AruQ+7rFotCrYZQlMCj2sV\\\\\\\/PZCjjkQhgGU8i8g7UJN86+Li94Byt9\\\\\\\/LYcmwH894BJYCVUKtV87dUF4nYtW0a1fRdbttImL1Bhar8HGkYNtshp+bPvRUX7saZfZReSM9y6Gha3GNsnVdES9ONHRbZMvYxGQHlrvXFoYE48PxoU0pygCmZxCGgwEuSChA1Rffcu06XLnHsXF12TBtq\\\\\\\/HK9rSZKREZQZ9m4b3geUhuQVSPnb1+Cv4OBwxkZfsXkVAMnk1wHKUUNqlX4mSZgdAK2KjzeSxEZpsTLv7sCwW75i3BWfw7XWHnli7o4UvK4tdsQod1KJtfvOI2SNM8B1IyFeZd\\\\\\\/zDmtIcIysQCtJ9bECmXxFU8hotsHZX\\\\\\\/tzhjj37nkJWZaN6+XU+a29CR6BLahnBe5j2SADtuItsUKyoUCIacWtjR4P1QjbHYZ\\\\\\\/T1jnmI7GG8TzqnDThGHpEn1H05Z4ipjXSJY7JAzo7RYOjMQZBWXCb\\\\\\\/\\\\\\\/TWy1z2tGg93ChvOwiA+HAbVGVFWCxv4OMobEgfR2RJOBAs5scJVBcYtcMcxTYkSutQKinHvqrAIDEM+3cdlswVhQWEprnZs\\\\\\\/jzU81xrMSGKF3kl5CtDVnqGl5EAUYwKml\\\\\\\/eisCymn\\\\\\\/Dm6V\\\\\\\/Y7KvwulXc3XLD5Sz7OgESyOM5Zew1jEd3ARYQeLYQ\\\\\\\/fxH8M2azmU+1LVt55ri6gFpZJKA41saseCY5twA5Vc+RKd0LveOcLRwrw+Z+Tz65RKfDaq+TwQzLzhIfg9NeWmxDGUgFLuPe9WWw+LojnocBN43RwDC97mwo7Tgy9ZafmPvtMkQOlgCQavBw7ALa3sWGNkFQXCKVA8014Mhkckjq\\\\\\\/icA1PyHj8WdE752cPp9xfts4Q4Z7cR3t+lcgSSiLR\\\\\\\/S2MgiUNpqEDnvUReOXvt1XbgGaLVdGwSVM1xJx2TzBbc2doP+y3fDMH9QOFSLztF1nJBazPS\\\\\\\/6HO1MU1mgR4xcLSDR1ptvXG+sbRj\\\\\\\/RPJ7\\\\\\\/q3kL4hgdD3RIgAWeZJQTQvHQ0WjMzSo18PJCrdwdEg8Uv4WGaZuyPuSaKnYy\\\\\\\/UV3ZvMt4uItBF6jmvnVqRSENx9dD+6B3OdlbKfONpxIhnRjy31HmTwrm9CZy3WltsAFwr\\\\\\\/TQ3ztVhhslsLYYXXapUCjMTo4FwrQO1\\\\\\\/zbNmPe27IFUXkEC3mxYyhPGFBJrg8W0BU7WBUAGFHQeu2UkwbssWNRd7dBMgg2jcnjFBtnYC9oD5eMg8Pll5SKJcsti+KIen8yYAwuGKKHaDAVZ6YxATbsKm9ziEyfH0vyvdxhJ5W\\\\\\\/32CeRPehwb9nKuONB6RMz4AlvHf7H0J2jsnb3zHJQUJfk19Iczfn\\\\\\\/+vFNMr7ilcb8Vo9ZU7hsPksSpAUIpxUzS+tXkpY0iOrxNBmTT5hwAWLWOMtnMb5K7KjYxaBJKYsSzxs8Su8CcZuQZR0lGUuUp97jSvj6osAqFage3h4po7F3ggJO70CpUhz8EvUXtIBrB5HXQ3EPqKvSdyrwmGpywSe8\\\\\\\/gVKyuQd8\\\\\\\/L13SoyVARBVMVhlDPw+chK2AP3yun5MpyadZ7zTB1nEN1Cx4whfWJWamTeXl66waYIL5wzWnEx4I0P8NwGYQRCQVUw6NkZyQUOyAuVoz8B2Tdu9bJLJViGTyqB0bOGk3samo9NGxJJoW4LtoFrgMZqe2lRPGPckZVBd5J0btBfFBwPTthowc7Dt5z+dJSCXoFFcM1KPb7TfJIwUkioeGrCA7TxdoeYjDMwd4JBOjT8A==\\\u0022,\\\u0022iv\\\u0022:\\\u0022FSILKo8McfCSLFIBqNb6fA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022bcbe7ef3d45be120c61b5831b649abaee2fd64deaeeb0add52373d352919fdff\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022e+jZ5Y4sGNHPKRDvSrmSN0g9HHSW2zUra3nzt9isk7rCU0UL3DtaG\\\\\\\/WtbCI23TfdTZJXlbxxmVVTR3w6St3QpiMVZXgN+BpCBlJtkqTmizKkfMtE++hCQkSPX728RoVcJ1F+UQOV7UIN5qY18yUEPasAgKgoqvuWnY2BpeuxB8sYTDY3AubcU1niNWqBS33uac6jGiEG6D6+c6XSBjShuXcFUK\\\\\\\/TDIdwiiUqMmaEVdRVRg22KdlInxjdQMdoYpHntjVL9z57KqXBiCSgoQv4d10gP4fMloF2F0vHzIAbNkCUFZUeydVdQNMH1GO\\\\\\\/l3mTgwdp11R1eYmFhByC4otZ1AfLmy99b29mkCsenVXLVrwWAB46Bhp3gxQcflwW3TBPY9kbFMI8j2OjK7Og7nAnnLOkidx+K8nkvV5lHmhzW8Il1I3gFhqgGs9FZfLIJYNoQdWUV1jmLeDICvFZi9PxJaYAsn\\\\\\\/FRh9WCb76PFwaSR\\\\\\\/EhdteorJhmMQMNnB2CJzIUum1YuPNt09iP2zxsGS2a56wkC9TyhFF2Ed\\\\\\\/sm\\\\\\\/HiR2QYry82AikJNhOrq7hqCP0DPymfdnTQGLvuv8wqU\\\\\\\/H54j33WAskxoMGbcBsVxNccwa\\\\\\\/rAxOr90akXkxqUd7QbuFK37eLlHZRVe7Hl9zWpvHCATFdYlMUYqtuwCWbKUraaoRkPiZBJC41RAnUxjtW7CGbtlIL6LjNYIszlgfPSVBVXmmJtpOj5VYrkonzg0NJggyPMQzQ0JavuUuOQpxvTp5TFJRR1CJDkmuNBn7dDIdGtq2bKT0zqlMPUe2D3vG9mKTruVYwbt3mbJJRr94SjhtsvPVrFVzg\\\\\\\/zNLVRfulqJH+0bwXb5JYa5UjRtKQBLkdrFk56e66L52wvgHeGrUvAZJdVzXdS4SDrJm9Xh2uy7T2C00F8\\\\\\\/X8MqInK19jzI2kqNx\\\\\\\/kw5HMn0u+pKINtbwW\\\\\\\/tx3n+CirBuIewks0S6CwW8ZVeAguA\\\\\\\/jjlN1\\\\\\\/Xh0Gbp0SlPIzN34ObdE7HnNtuOK\\\\\\\/ivUqF4W\\\\\\\/Uh5AQ5Q11ah3qeCEb6SpUt\\\\\\\/saLDyniyWuMLc5tuTKEU3SeV+phCxqplaCdDrxpEFKv2WHcX2G41F4VoJq7fKSbbjOmiuFnJTkc\\\\\\\/tSxinc9uReWqAEmp6h4zgGDuORCmBxa8NdYAn66Agjetv7xU8WUXAv0imq5Dl8M4EclYIWa9yCY0+yQ9LAUoD3uf+DI\\\\\\\/WX9Pd1h6E+FXWqybqSjivTUwIei1eKqowVlhKvTrhRHoas\\\\\\\/0WtdeJl9iq5Kjw0V5zFWUuCuWOXwSQrWRZd6saibV4tKcs+I85XLXo87gjSKReHboMrmV55qE8F4zHeV2jbM2NzrOi8LXU46wD3dXLU9JH+r7cRXnIaKl+kl94W7herStnYizRPLm9PKGWSfo9G4AwjiVPT1w2Gl9uRkuIO6HfnOdnrXnCis9tJbJGWhKuLaxbV1th7FJe1CzU\\\\\\\/RXhE+kEe8J+bDX6+eQdajeQZBRGmz0LtKOyLNwQru0BNopNL+Y+YClCuLqfL23dTlc8Zo0QDEWBxBcGZMvGT+fZjuikwerZ+Esg6EC6eLd5T\\\\\\\/JFMY+Cr7jKZ0zFQxeVxqqZE3U\\\\\\\/LGn5nlppiObFvMrp0zjVpd\\\\\\\/eIYtM3+wUlfq0PHD6EO5nBNTuwhdbng9mPe87Ui8W7lNdexU8Q\\\\\\\/Vnt2Iyl2U0zeSi4JFqKsaT+0MoYXj4UnZplvGa0gQTf4cewVjE4N5kYNtW\\\\\\\/OgD8\\\\\\\/xCf1gGhcSyBEd1vV23vql2IMyog4xGH6rrV0bFX\\\\\\\/YC9Y+eKn7LDfcvGZd5anApB3oK7GSJZkrkzbEezL3eoJJUiT68pcvETupVbCNXMBUQ+eSbYAEnmFeGTcYzwM4ygKgP1JEkPkL7hr5tyWcVKU+IWrPQovTkmbrZHfdT48\\\\\\\/ttbzInNlAzNzdr0CKJ5WI6EU0hNQ9h5Wex5HD\\\\\\\/5bUktbphR9Hb1Wn6HG4om0RrYAjsxzH\\\\\\\/EqbCyg7+RSGkOpGw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022g4057NqcuYOkcbc86UlGZw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00228ff7178f89113ecf9c80192d4e501bfffbd86aea276379661d59f5b4a119d0a2\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-494273 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/10/3105" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-uOwEHExjSstAuC7vtcE11EiQ1JcrNkfzZg_SJycnE3U" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/10/3105" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-tSCs2u58XzPxjuo-4Bi5D2TQBdtYUmBEnmSlKQtNovI" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/10/3101" title="micRocounter: Microsatellite Characterization in Genome Assemblies" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/10/3119" title="Transformation with Artificial Chromosomes in Oxytricha trifallax and Their Applications" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494273" id="node494273" data-pisa="ggg;9/10/3105" data-pisa-master="ggg;g3.118.200930" data-apath="/ggg/9/10/3105.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Programmed Chromosome Deletion in the Ciliate <em>Oxytricha trifallax</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Derek M. Clay</span>, <span class="highwire-citation-author" data-delta="1">V. Talya Yerlici</span>, <span class="highwire-citation-author" data-delta="2">Danylo J. Villano</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-7030-8540" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Laura F. Landweber</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3105-3118;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200930 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494273" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Derek M. Clay </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Departments of Biochemistry and Molecular Biophysics and Biological Sciences, Columbia University, New York, NY and</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Molecular Biology, Princeton University, Princeton, NJ</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Derek%2BM.%2BClay%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Clay%20DM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADerek%2BM.%2BClay%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">V. Talya Yerlici </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Departments of Biochemistry and Molecular Biophysics and Biological Sciences, Columbia University, New York, NY and</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Molecular Biology, Princeton University, Princeton, NJ</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=V.%2BTalya%2BYerlici%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yerlici%20VT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AV.%2BTalya%2BYerlici%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Danylo J. Villano </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Departments of Biochemistry and Molecular Biophysics and Biological Sciences, Columbia University, New York, NY and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Danylo%2BJ.%2BVillano%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Villano%20DJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADanylo%2BJ.%2BVillano%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Laura F. Landweber </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Departments of Biochemistry and Molecular Biophysics and Biological Sciences, Columbia University, New York, NY and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Laura%2BF.%2BLandweber%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Landweber%20LF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALaura%2BF.%2BLandweber%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-7030-8540" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Laura F. Landweber</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:Laura.Landweber@columbia.edu" class="" data-icon-position="" data-hide-link-title="0">Laura.Landweber@columbia.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/9/10/3105" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:494273" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:494273/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/10/3105.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:494273" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:494273/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/10/3105.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:494273" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:494273/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/9/10/3105.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:494273" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:494273/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">The ciliate <em>Oxytricha trifallax</em> contains two nuclei: a germline micronucleus and a somatic macronucleus. These two nuclei diverge significantly in genomic structure. The micronucleus contains approximately 100 chromosomes of megabase scale, while the macronucleus contains 16,000 gene-sized, high ploidy “nanochromosomes.” During its sexual cycle, a copy of the zygotic germline micronucleus develops into a somatic macronucleus via DNA excision and rearrangement. The rearrangement process is guided by multiple RNA-based pathways that program the epigenetic inheritance of sequences in the parental macronucleus of the subsequent generation. Here, we show that the introduction of synthetic DNA molecules homologous to a complete native nanochromosome during the rearrangement process results in either loss or heavy copy number reduction of the targeted nanochromosome in the macronucleus of the subsequent generation. This phenomenon was tested on a variety of nanochromosomes with different micronuclear structures, with deletions resulting in all cases. Deletion of the targeted nanochromosome results in the loss of expression of the targeted genes, including gene knockout phenotypes that were phenocopied using alternative knockdown approaches. Further investigation of the chromosome deletion showed that, although the full length nanochromosome was lost, remnants of the targeted chromosome remain. We were also able to detect the presence of telomeres on these remnants. The chromosome deletions and remnants are epigenetically inherited when backcrossed to wild type strains, suggesting that an undiscovered mechanism programs DNA elimination and cytoplasmically transfers to both daughter cells during conjugation. Programmed deletion of targeted chromosomes provides a novel approach to investigate genome rearrangement and expands the available strategies for gene knockout in <em>Oxytricha trifallax</em>.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/genomic-rearrangement" class="hw-term hw-article-keyword hw-article-keyword-genomic-rearrangement" rel="nofollow">Genomic Rearrangement</a></li><li class="kwd"><a href="/keyword/ciliates" class="hw-term hw-article-keyword hw-article-keyword-ciliates" rel="nofollow">Ciliates</a></li><li class="kwd"><a href="/keyword/epigenetics" class="hw-term hw-article-keyword hw-article-keyword-epigenetics" rel="nofollow">Epigenetics</a></li></ul><p id="p-3">The dual nuclear structure of the ciliate <em>Oxytricha</em> trifallax partitions the germline and somatic functional elements into two distinct nuclei, the micronucleus (MIC) and macronucleus (MAC). The somatic macronucleus is comprised of approximately 16,000 gene-sized chromosomes, while the germline micronucleus contains on the order of 100 megabase-sized chromosomes (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Swart <em>et al.</em> 2013</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Chen et al. 2014</a>). Under normal growth conditions, like other ciliates, <em>Oxytricha</em> reproduces asexually and its macronuclear nanochromosomes are maintained through amitotic division at an average of 2,000 copies per cell (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Prescott 1994</a>). Under stressful conditions, the cell can undergo a non-reproductive sexual cycle. During <em>Oxytricha’s</em> sexual development, cells of different mating types pair and transfer a single post-meiotic haploid micronucleus. The exchanged haploid nuclei fuse with their non-exchanged counterpart to form a new diploid micronucleus. The new macronucleus develops from a copy of the newly formed micronucleus through elimination of more than 90% of the DNA sequence (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Chen <em>et al.</em> 2014</a>) and rearrangement of the remaining sequences generates nanochromosomes in the new macronucleus (reviewed in <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Yerlici and Landweber 2014</a>).</p><p id="p-4">Ciliates have specific mechanisms to identify sequences for retention or deletion from the rearranging nucleus. In the distantly related ciliates <em>Tetrahymena</em> and <em>Paramecium</em>, the differentiation of germline-limited internally eliminated sequences (IESs) from macronuclear destined sequences (MDSs) is programmed by scan RNAs (scnRNA) (reviewed in <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Allen and Nowacki 2017</a>). These small RNAs are produced from the parental micronucleus during genomic rearrangement and are selectively enriched by alignment to macronuclear sequences. Small RNAs that do not match sequences in the parental macronucleus proceed to mark sequences in the developing macronucleus for excision and elimination.</p><p id="p-5">In <em>Paramecium</em> species, there is evidence of an interaction between the scnRNA and RNA interference (RNAi) pathways that can reprogram MDS deletion in subsequent generation. Experimental introduction of truncated genes into the vegetative macronucleus leads to the production of siRNA against the targeted gene and the subsequent silencing of both the native and introduced copies of the gene (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Meyer 1992</a>, <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Ruiz <em>et al.</em> 1998</a>, <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Galvani and Sperling 2001</a>, <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Garnier <em>et al.</em> 2004</a>, <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Götz <em>et al.</em> 2016</a>). Upon undergoing conjugation, this RNAi silencing can trigger, in the subsequent generation, deletion of the genomic regions, flanked by a “TA” dinucleotide, which is the cut site for PiggyMAC transposases (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Klobutcher and Herrick 1995</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Baudry <em>et al.</em> 2009</a>). Further evidence of this interaction came from the observation that injection of dsRNA against specific MDS regions resulted in the deletion of these MDSs from the next generation’s macronucleus (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Garnier <em>et al.</em> 2004</a>).</p><p id="p-6">In contrast to <em>Paramecium</em> and <em>Tetrahymena</em>, the RNA pathways reported so far in <em>Oxytricha</em> mark sequences for retention rather than deletion. Early in <em>Oxytricha</em>’s genomic rearrangement process, the parental macronuclear genome appears to be completely transcribed, generating long RNA templates (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Nowacki <em>et al.</em> 2008</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Lindblad <em>et al.</em> 2017</a>) together with the production of 27 nucleotide piRNAs (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Fang <em>et al.</em> 2012</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Zahler <em>et al.</em> 2012</a>). Collectively, these template RNAs and piRNAs mark retained sequences and program the necessary rearrangements to convert a germline into the somatic nucleus. Experimentally, injection of synthetic piRNAs can result in the retention of normally deleted IESs in the macronucleus. In <em>Oxytricha</em>, there is no simple consensus dinucleotide “TA” cut site. Instead, adjacent MDS ends in the macronucleus contain microhomologous sequences that vary and can be much longer (<a id="xref-ref-6-3" class="xref-bibr" href="#ref-6">Chen <em>et al.</em> 2014</a>). Only one copy of the microhomologous sequence is retained in the macronucleus after MDSs fuse during rearrangement. The general presence of these microhomologous repeats at the ends of MDSs has long suggested that they play an important role in facilitating <em>Oxytricha</em>’s genomic rearrangement (<a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Prescott 1994</a>).</p><p id="p-7">Introduction of artificial DNA or RNA templates early in genomic rearrangement reprograms rearrangement of the next generation’s macronucleus (<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Nowacki <em>et al.</em> 2008</a>, <a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Fang <em>et al.</em> 2012</a>). Here, we show that introduction of synthetic copies of a full length nanochromosome early in genomic rearrangement can reprogram deletion of that chromosome from the subsequent generation’s macronucleus. Furthermore, we exploited this surprising observation to develop a tool for targeted chromosomal deletion to facilitate functional genetic manipulation of <em>Oxytricha trifallax</em>.</p><div class="section methods" id="sec-1"><h2 class="">Methods</h2><div id="sec-2" class="subsection"><h3>Generation of DNA constructs for microinjection</h3><p id="p-8">To generate full length DNA templates including small (30 to 50 bp) deletions, overlap extension PCR was performed using JRB310 genomic DNA as template and Phusion DNA Polymerase (NEB). PCR Primers were synthesized by IDT with standard desalting conditions. Wild type DNA constructs were produced via conventional PCR with JRB310 genomic DNA as template. The PCR products were gel purified using Qiagen MinElute columns according to manufacturer’s instructions. PCR products were singly A-tailed with Taq polymerase (Roche) and TA-TOPO cloned (Invitrogen). TOPO cloned plasmids were transformed into TOP10 One shot chemically competent cells (Invitrogen) according to manufacturer’s instructions. Plasmid DNA was isolated from individual clones using the QIAprep Spin Miniprep kit (Qiagen). TOPO plasmids were verified via Sanger sequencing through Genewiz.</p><p id="p-9">Validated plasmids were used as templates for PCR to generate approximately 100 μg of PCR product. Quality of the PCR products was verified by gel electrophoresis. PCR products were phenol:chloroform extracted and concentrated by ethanol precipitation. DNA pellets were resuspended in nuclease-free water (Ambion) and run through an ultra-free MC column (Millipore) according to manufacturer’s instructions. DNA was brought to a final concentration of 1 to 3 mg/mL for microinjection.</p></div><div id="sec-3" class="subsection"><h3>Oxytricha culturing</h3><p id="p-10"><em>Oxytricha trifallax</em> cells were grown in Pringsheim media (0.11 mM Na<sub>2</sub>HPO<sub>4</sub>, 0.08mM MgSO<sub>4</sub>, 0.85 mM Ca(NO<sub>3</sub>)<sub>2</sub>, 0.35 mM KCl, pH 7.0), fed <em>Chlamydomonas reinhardtii</em> and supplemented with <em>Klebsiella</em> for improved growth. Cells were cleansed of debris and algae by filtering through cheesecloth and concentrated via centrifugation at 80g for 1 min for harvesting (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Khurana <em>et al.</em>, 2014</a>).</p></div><div id="sec-4" class="subsection"><h3>DNA template injections and screening for deletion lines</h3><p id="p-11"><em>Oxytricha</em> cells were mated approximately 1 to 4 weeks post-excystment by mixing 3 mL of each mating type, JRB310 and JRB510, along with 6 mL of fresh 1X Pringsheim. At 3 to 5 hr post mixing, pairs were isolated and placed in Volvic water with 0.2% bovine serum albumin according to previously published methods (<a id="xref-ref-7-3" class="xref-bibr" href="#ref-7">Fang <em>et al.</em> 2012</a>). DNA molecules were injected at 3 to 5 hr post-mixing into the macronuclei of the paired cells as previously described except for the timing (<a id="xref-ref-25-3" class="xref-bibr" href="#ref-25">Nowacki <em>et al.</em> 2008</a>). After injection cells were pooled in Volvic water to improve survival rates. At 60 to 72 hr post mixing, the pooled cells were singled out to grow clonal injected cell lines. As clonal population size grew, lines were transferred to 10 cm petri dishes and grown in 1X Pringsheim media.</p><p id="p-12"><em>Oxytricha</em> clonal lines were screened for deletions using cells or purified genomic DNA in PCR. Genomic DNA was harvested via the NucleoSpin Tissue kit (Macherey-Nagel). Cells were concentrated via centrifugation for 1 min at 80g, and the supernatant was aspirated.</p></div><div id="sec-5" class="subsection"><h3>Mating of deletion lines</h3><p id="p-13">Using the same method for setting up mating as above, pairs were collected at 6 to 12 hr post mixing and pooled. Clonal lines were isolated at 60 to 72 hr post mixing and cultured according to the protocol described above.</p></div><div id="sec-6" class="subsection"><h3>RNA isolation and cDNA synthesis</h3><p id="p-14">RNA was collected from pools of isolated pairs (approximately 50 pairs) at 12 hr post-mixing via the mirVANA miRNA Isolation Kit (Ambion) using the total RNA isolation protocol according to manufacturer’s instructions. RNA was DNAse-treated with Turbo DNAse (Ambion) according to the manufacturer’s instructions. SuperScript III First-Strand Synthesis (Thermofisher Scientific) kit was used according to manufacturer’s instructions to generate cDNA for qPCR analysis.</p></div><div id="sec-7" class="subsection"><h3>Quantitative PCR for genomic DNA and cDNA</h3><p id="p-15">Quantitative PCR was done with Sybr green power mix (ABI) according to the manufacturer’s instructions using either ABI 7900 (ABI) or CFX384 (BioRad) qPCR machines. Standards were generated via conventional PCR and purified by MinElute PCR purification (Qiagen). Purified standards were quantified by Qubit High Sensitivity DNA Assay Kit following the standard protocol (Thermofisher Scientific). Results from qPCR were quantitated via ΔΔCT and standard curves.</p></div><div id="sec-8" class="subsection"><h3>Southern Hybridization</h3><p id="p-16">200 ng of genomic DNA was loaded onto a 1.0% agarose gel and Southern transfer and hybridization were performed according to previously published protocols (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bracht <em>et al.</em> 2017</a>) with minor alterations.</p></div><div id="sec-9" class="subsection"><h3>Illumina sequencing of deletion lines</h3><p id="p-17">Genomic DNA from deletion and wild type cell lines were used for preparing TruSeq Illumina libraries according to manufacturer’s instructions and sequenced in single-end mode on the Illumina HiSeq2500 platform. Raw reads (average length 150nt) were processed using Galaxy on the Princeton University webserver (<a href="http://galaxy.princeton.edu">galaxy.princeton.edu</a>) (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Goecks <em>et al.</em> 2010</a>). Reads were trimmed using Trim Galore version 0.4.3 to remove low quality ends (&lt;Q20) and adapter sequences (maximum allowed error rat: 0.1, minimum read length: 20nt) (Felix Krueger, Babraham Institute <a href="https://www.bioinformatics.babraham.ac.uk/projects/trim_galore/">https://www.bioinformatics.babraham.ac.uk/projects/trim_galore/</a>). Duplicate reads were collapsed using FASTX-toolkit (Assaf Gordon, <a href="http://hannonlab.cshl.edu/fastx_toolkit/">http://hannonlab.cshl.edu/fastx_toolkit/</a>).</p><p id="p-18">Reads were then mapped using BWA-mem with default parameters (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Li and Durbin 2009</a>) onto the <em>Oxytricha</em> JRB310 MAC genome assembly (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Lindblad <em>et al.</em> 2019</a>). SAM files were processed to remove alignments with a low mapping score and non-primary alignments using SAMtools view (parameters: -bSq 5 and -F 256) (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Li <em>et al.</em> 2009</a>). The mapped reads from each library were then subsampled using seqtk (<a href="https://github.com/lh3/seqtk">https://github.com/lh3/seqtk</a>) according to the number of reads mapping to the macronuclear genome to normalize for sequencing depth and converted into bedgraph format using BEDtools genomecov (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Quinlan and Hall 2010</a>). Coverage tracks were generated using Bioconductor software Sushi package (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Phanstiel 2015</a>) (R version 3.4.1).</p></div><div id="sec-10" class="subsection"><h3>Data Availability</h3><p id="p-19">Strains are available on request. Supplemental file 1 contains the nucleotide sequences from the Sanger sequencing. Genomic sequencing data are available in supplemental files 2 through 14. Supplemental material available at FigShare: <a href="https://doi.org/10.25387/g3.7123214">https://doi.org/10.25387/g3.7123214</a>.</p></div></div><div class="section results" id="sec-11"><h2 class="">Results</h2><div id="sec-12" class="subsection"><h3>DNA injection can trigger chromosome deletion</h3><p id="p-20">Previous research demonstrated that injection of synthetic DNA or RNA copies of chromosomes during nuclear development can specifically influence DNA rearrangement in the subsequent generation (<a id="xref-ref-25-4" class="xref-bibr" href="#ref-25">Nowacki <em>et al.</em> 2008</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Nowacki <em>et al.</em> 2010</a>; <a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Bracht <em>et al.</em> 2017</a>). To further investigate the influence of exposure to a synthetic DNA molecule during conjugation, we injected a copy of wild type Contig16116.0, the nanochromosome encoding the Otiwi1 gene, into <em>O. trifallax</em> cells. Surprisingly, some (approximately 10%) of the resulting progeny displayed significant copy number reduction of the endogenous chromosome (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). In addition, we injected a modified version of Contig16116.0, containing a 28 nucleotide deletion flanked by 7 base pair endogenous repeats (see supplementary figure 1), anticipating that this DNA template could program deletion of the 28 nt sequence in the next generation’s somatic genome (<a id="xref-ref-25-5" class="xref-bibr" href="#ref-25">Nowacki <em>et al.</em> 2008</a>). Instead, we observed deletion of the endogenous chromosome, suggesting robustness of this effect to modest deletions in the injected template (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/10/3105/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="DNA injections result in the deletion of nanochromosomes from the subsequent generation. A) PCR amplification from genomic DNA harvested from screened and established lines shows loss of the targeted nanochromosome. The top gel section of each panel shows the deleted chromosome as labeled above, with the name of deletion lines (marked &#x394;) established and assayed in this experiment. The bottom gel section of each panel shows PCR amplification of the TEBP-&#x3B2; gene as input loading control. Deletion line shown in comparison to the parental lines (strains JRB310 and JRB510, marked simply 310 or 510) or the uninjected F1 population derived from mating the two parental lines. In the &#x394;Otiwi1 panels, deletion lines 1 and 2 were generated through injection of a construct containing a 28 bp deletion while line 3 was generated using a full length Otiwi1 chromosome construct. B) Diagram of the nanochromosomes in the PCRs in panel A with the following features labeled: telomeres (red), the locations of genes (green), and the location of the primers used in panel A (pink)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1294374134" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;DNA injections result in the deletion of nanochromosomes from the subsequent generation. A) PCR amplification from genomic DNA harvested from screened and established lines shows loss of the targeted nanochromosome. The top gel section of each panel shows the deleted chromosome as labeled above, with the name of deletion lines (marked &#x394;) established and assayed in this experiment. The bottom gel section of each panel shows PCR amplification of the TEBP-&#x3B2; gene as input loading control. Deletion line shown in comparison to the parental lines (strains JRB310 and JRB510, marked simply 310 or 510) or the uninjected F1 population derived from mating the two parental lines. In the &#x394;Otiwi1 panels, deletion lines 1 and 2 were generated through injection of a construct containing a 28 bp deletion while line 3 was generated using a full length Otiwi1 chromosome construct. B) Diagram of the nanochromosomes in the PCRs in panel A with the following features labeled: telomeres (red), the locations of genes (green), and the location of the primers used in panel A (pink).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/10/3105/F1.medium.gif" width="408" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/9/10/3105/F1.medium.gif" width="408" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/10/3105/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/10/3105/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494409" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-21" class="first-child">DNA injections result in the deletion of nanochromosomes from the subsequent generation. A) PCR amplification from genomic DNA harvested from screened and established lines shows loss of the targeted nanochromosome. The top gel section of each panel shows the deleted chromosome as labeled above, with the name of deletion lines (marked Δ) established and assayed in this experiment. The bottom gel section of each panel shows PCR amplification of the TEBP-β gene as input loading control. Deletion line shown in comparison to the parental lines (strains JRB310 and JRB510, marked simply 310 or 510) or the uninjected F1 population derived from mating the two parental lines. In the ΔOtiwi1 panels, deletion lines 1 and 2 were generated through injection of a construct containing a 28 bp deletion while line 3 was generated using a full length Otiwi1 chromosome construct. B) Diagram of the nanochromosomes in the PCRs in panel A with the following features labeled: telomeres (red), the locations of genes (green), and the location of the primers used in panel A (pink).</p><div class="sb-div caption-clear"></div></div></div><p id="p-22">In order to determine the extent of this phenomenon; five additional chromosomes were tested. Contig20822.0 was also injected as a full-length DNA chromosome containing a 37 nucleotide deletion, whereas the remaining four cases were injected as full-length wild type DNA versions (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a> and supplementary figure 1). qPCR analysis and whole genome sequencing demonstrated severe copy number reduction of the target chromosome within the established cell lines, with some variation in DNA copy level at regions across the chromosomes (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a> and supplementary figure 2).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/10/3105/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Sequence coverage for a deleted chromosome is decreased in deletion lines. Normalized read depth from Illumina whole genome sequencing mapped to targeted chromosomes shows a strong decrease in the deletion lines relative to the uninjected F1 lines derived from JRB310xJRB510 matings. The first deletion line is labeled in red and second line depicted in blue." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1294374134" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Sequence coverage for a deleted chromosome is decreased in deletion lines. Normalized read depth from Illumina whole genome sequencing mapped to targeted chromosomes shows a strong decrease in the deletion lines relative to the uninjected F1 lines derived from JRB310xJRB510 matings. The first deletion line is labeled in red and second line depicted in blue.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/10/3105/F2.medium.gif" width="440" height="424"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/9/10/3105/F2.medium.gif" width="440" height="424"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/10/3105/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/10/3105/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494458" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-23" class="first-child">Sequence coverage for a deleted chromosome is decreased in deletion lines. Normalized read depth from Illumina whole genome sequencing mapped to targeted chromosomes shows a strong decrease in the deletion lines relative to the uninjected F1 lines derived from JRB310xJRB510 matings. The first deletion line is labeled in red and second line depicted in blue.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-13" class="subsection"><h3>Some cases of chromosome deletion result in maintenance of DNA fragments</h3><p id="p-24">To compare copy number variation across different locations on the same chromosome, including the 5′ and 3′ ends of Contig20822.0 that encodes an Alba domain gene (hence known as Alba-like 1 or AL1), a Southern blot was performed on these deletion lines, together with the control parental strains, against a probe for the 3′ end of the chromosome (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3C</a>). This also indicated a significant reduction in the full-length AL1 chromosome levels. Furthermore, in agreement with the qPCR results (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3B</a>), deletion lines 1 and 2 (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a> panels B and C) appear to contain a less abundant, shorter 3′ chromosome fragment. Whole genome sequencing reads were used to search for the presence of short chromosome remnants in other deletion lines.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/10/3105/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Remnants of deleted chromosomes are detected in some of the deletion lines. A) A diagram of the AL1 nanochromosome with various features labeled, including the locations of the telomeres (red), the AL1 gene (green), the MDSs (gold), the qPCR primers (pink, labeled with which qPCR they belong to in panel B, and hybridization region of the Southern probe (blue). B) Quantification of the 3&#x2032; and 5&#x2032; end of the chromosome via qPCR and the relative copy vs. the JRB310xJRB510 F1 genomic DNA. Error bars represent the standard deviation. C) Southern analysis of the first 3 contig20822.0 (AL1) deletion lines (#1, #2, and #3) using a probe against the 3&#x2032; end of the nanochromosome. D) The same membrane was then stripped and re-probed with TEBP-beta for loading control. E) A depiction of the contig14335.0 nanochromosome with MDSs (gold), telomeres (red), and the gene specific primers for the two rounds of telomeric PCR (pink, labeled with which PCR round they are involved in) labeled. F) Second round of telomeric PCR (see methods for description of telomeric PCR) for the 5&#x2032; end of the Contig14335.0 chromosome in contig14335.0 deletion line #2. Telomeric PCR products from contig14335.0 deletion line #2 were Sanger sequenced to identify the exact location of the aberrant telomere addition (see supplementary figure 6)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1294374134" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Remnants of deleted chromosomes are detected in some of the deletion lines. A) A diagram of the AL1 nanochromosome with various features labeled, including the locations of the telomeres (red), the AL1 gene (green), the MDSs (gold), the qPCR primers (pink, labeled with which qPCR they belong to in panel B, and hybridization region of the Southern probe (blue). B) Quantification of the 3&#x2032; and 5&#x2032; end of the chromosome via qPCR and the relative copy &lt;em&gt;vs.&lt;/em&gt; the JRB310xJRB510 F1 genomic DNA. Error bars represent the standard deviation. C) Southern analysis of the first 3 contig20822.0 (AL1) deletion lines (#1, #2, and #3) using a probe against the 3&#x2032; end of the nanochromosome. D) The same membrane was then stripped and re-probed with TEBP-beta for loading control. E) A depiction of the contig14335.0 nanochromosome with MDSs (gold), telomeres (red), and the gene specific primers for the two rounds of telomeric PCR (pink, labeled with which PCR round they are involved in) labeled. F) Second round of telomeric PCR (see methods for description of telomeric PCR) for the 5&#x2032; end of the Contig14335.0 chromosome in contig14335.0 deletion line #2. Telomeric PCR products from contig14335.0 deletion line #2 were Sanger sequenced to identify the exact location of the aberrant telomere addition (see supplementary figure 6).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/10/3105/F3.medium.gif" width="372" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/9/10/3105/F3.medium.gif" width="372" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/10/3105/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/10/3105/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494411" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-25" class="first-child">Remnants of deleted chromosomes are detected in some of the deletion lines. A) A diagram of the AL1 nanochromosome with various features labeled, including the locations of the telomeres (red), the AL1 gene (green), the MDSs (gold), the qPCR primers (pink, labeled with which qPCR they belong to in panel B, and hybridization region of the Southern probe (blue). B) Quantification of the 3′ and 5′ end of the chromosome via qPCR and the relative copy <em>vs.</em> the JRB310xJRB510 F1 genomic DNA. Error bars represent the standard deviation. C) Southern analysis of the first 3 contig20822.0 (AL1) deletion lines (#1, #2, and #3) using a probe against the 3′ end of the nanochromosome. D) The same membrane was then stripped and re-probed with TEBP-beta for loading control. E) A depiction of the contig14335.0 nanochromosome with MDSs (gold), telomeres (red), and the gene specific primers for the two rounds of telomeric PCR (pink, labeled with which PCR round they are involved in) labeled. F) Second round of telomeric PCR (see methods for description of telomeric PCR) for the 5′ end of the Contig14335.0 chromosome in contig14335.0 deletion line #2. Telomeric PCR products from contig14335.0 deletion line #2 were Sanger sequenced to identify the exact location of the aberrant telomere addition (see supplementary figure 6).</p><div class="sb-div caption-clear"></div></div></div><p id="p-26">The most abundant set of remnants was identified in Contig14335.0 deletion line 2, which has high Illumina read coverage exclusively in the center of the nanochromosome, and no sequence coverage at the ends (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>). Moreover, telomeric reads mapping to the central portion of Contig14335.0 were identified in the deletion lines but not the wild type controls. In order to confirm the presence of telomeres on the remnants, PCR using a telomeric primer and a gene specific primer was performed as in (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Chang <em>et al.</em> 2004</a>) on Contig14335.0, deletion line 2. This telomeric PCR confirmed the presence of an internal telomere addition site in the deletion line (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3E</a>). In the case of Contig16116.0, an aberrant product was the dominant product in the conventional PCR screening of deletion line 1 (see supplementary figure 3). Sanger sequencing of this product revealed that it contained incorrect MDS fusion sites at a series of microhomologous repeats. (sequence data provided in supplementary file 1).</p></div><div id="sec-14" class="subsection"><h3>Chromosome deletion eliminates expression of genes on the nanochromosome</h3><p id="p-27">The ostensible loss or severe copy number reduction of the targeted nanochromosomes could lead to a phenotype since the genes present on these nanochromosomes are heavily reduced in DNA copy number or completely absent from the somatic macronucleus. Indeed, we observed the same conjugation specific lethality in the Otiwi1 (Contig16116.0) deletion lines that phenocopy knockdown of the Otiwi1 gene (<a id="xref-ref-7-4" class="xref-bibr" href="#ref-7">Fang <em>et al.</em> 2012</a>). In addition, RT-PCR and qPCR analysis of the AL1 (Contig20822.0) deletion lines showed a massive reduction in the expression of the AL1 gene (supplementary figure 4). Thus, programmed chromosome deletion provides a novel mechanism for generating somatic mutant strains in <em>O. trifallax</em>. Furthermore, deletion of the AL1 gene results in a lethal phenotype when the deletion lines are mated to each other, with no cells surviving beyond 30hrs post-mixing. This suggests the AL1 gene is essential during sexual development, and provides proof of principle for the use of chromosome deletion to test gene essentiality.</p></div><div id="sec-15" class="subsection"><h3>Epigenetics of chromosome deletion</h3><p id="p-28">Given our knowledge that macronuclear chromosome architecture relies on the transfer of parental epigenetic information across sexual generations (<a id="xref-ref-25-6" class="xref-bibr" href="#ref-25">Nowacki <em>et al.</em> 2008</a>; <a id="xref-ref-7-5" class="xref-bibr" href="#ref-7">Fang <em>et al.</em> 2012</a>; <a id="xref-ref-4-3" class="xref-bibr" href="#ref-4">Bracht <em>et al.</em> 2017</a>), we examined the influence across generations of loss or copy number reduction of targeted nanochromosomes. Because the deletion lines for Contigs 17155.0 and 9679.0 target genes are under further investigation elsewhere (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Yerlici <em>et al</em>. 2019</a>) and Contigs 20822.0 and 16116.0 encode genes essential for conjugation, we tested the ability of the deletion lines for Contigs 14335.0 and 18510.0 to generate sexual progeny, since both of these nanochromosomes contain no predicted genes.</p><p id="p-29">Contig18510.0 deletion lines can be successfully backcrossed to the parental strain JRB510. Their progeny, however, do display recovery of the deleted chromosome via conventional PCR (see supplementary figure 5C).</p><p id="p-30">Cell lines containing the deletion of Contig14335.0 were mated to each other or backcrossed to the parental strain JRB510. The Contig14335.0 deletion lines mated to each other had a low survival rate, while the backcrosses showed no apparent survival defect. Both the deletion lines crossed to each other (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a> panels C and D) and Contig14335.0 deletion line 1 backcrossed to JRB510 (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a> panels F and G) showed no recovery of the deleted chromosome. DNA from backcrosses of Contig14335.0 deletion line 2 mated to JRB510 could amplify the full-length version of the deleted chromosome on a population level (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4I</a>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/10/3105/F4/graphic-4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Deletions and remnants are epigenetically inherited A) An illustration of the contig14335.0 with primer locations depicted (purple for the detection PCR shown in panels C and E, pink for the qPCR primers, labeled with the corresponding qPCR they belong to in panels C, E, G). In addition, the locations of the MDSs (gold) and telomeres (red) on the nanochromosome are shown. B) Diagram of the various crosses involved generating the progeny involved in panels C and D. The colors in the diagram correspond to the colors in panel D. C) PCR amplification of genomic DNA harvested from populations of the progeny of ctg14335.0 deletion line 1 mated to deletion line 2, with parental lines and grandparental lines, along with uninjected 310x510 as controls. Lower gel section shows PCR amplification of TEBP-&#x3B2; as loading control. D) Quantitative PCR on the F1 from ctg14335.0 deletion line 1 mated to deletion line 2 surveyed across various regions of the contig14335.0 nanochromosome. E) Diagram of the various crosses involved generating the progeny involved in panels F and G. The colors from the diagram correspond to the colors in the following panel G. F) PCR amplification from genomic DNA harvested from populations of ctg14335.0 deletion line 1 backcrossed to wt strain JRB510 for up to 2 generations (labeled F1 and F2) with parental lines and grandparental lines along with uninjected 310x510 as controls. Lower gel section shows PCR amplification of TEBP-&#x3B2; as loading control. G) Quantitative PCR on genomic DNA used in panel A across multiple locations of ctg14335.0, together with an unrelated locus, TEBP-&#x3B1;. Arrows represent undetectable levels at the respective loci, with arrow color corresponding to the legend. Relative copy levels were determined by setting the levels of JRB310xJRB510 F1 to 1. H) Diagram of the various crosses involved generating the progeny in panels I and J. The colors from the diagram correspond to the colors in the following panel J. I) PCR amplification of genomic DNA harvested from clonal isolates and populations (labeled respectively) of the progeny of ctg14335.0 deletion line 2 backcrossed to JRB510, with parental and grandparental lines, along with uninjected JRB310xJRB510 as controls. Lower gel section shows PCR amplification of TEBP-&#x3B2; as loading control. J) Quantitative PCR on F1 genomic DNA from ctg14335.0 deletion line 2 across various regions of contig14335.0." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1294374134" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Deletions and remnants are epigenetically inherited A) An illustration of the contig14335.0 with primer locations depicted (purple for the detection PCR shown in panels C and E, pink for the qPCR primers, labeled with the corresponding qPCR they belong to in panels C, E, G). In addition, the locations of the MDSs (gold) and telomeres (red) on the nanochromosome are shown. B) Diagram of the various crosses involved generating the progeny involved in panels C and D. The colors in the diagram correspond to the colors in panel D. C) PCR amplification of genomic DNA harvested from populations of the progeny of ctg14335.0 deletion line 1 mated to deletion line 2, with parental lines and grandparental lines, along with uninjected 310x510 as controls. Lower gel section shows PCR amplification of TEBP-&#x3B2; as loading control. D) Quantitative PCR on the F1 from ctg14335.0 deletion line 1 mated to deletion line 2 surveyed across various regions of the contig14335.0 nanochromosome. E) Diagram of the various crosses involved generating the progeny involved in panels F and G. The colors from the diagram correspond to the colors in the following panel G. F) PCR amplification from genomic DNA harvested from populations of ctg14335.0 deletion line 1 backcrossed to wt strain JRB510 for up to 2 generations (labeled F1 and F2) with parental lines and grandparental lines along with uninjected 310x510 as controls. Lower gel section shows PCR amplification of TEBP-&#x3B2; as loading control. G) Quantitative PCR on genomic DNA used in panel A across multiple locations of ctg14335.0, together with an unrelated locus, TEBP-&#x3B1;. Arrows represent undetectable levels at the respective loci, with arrow color corresponding to the legend. Relative copy levels were determined by setting the levels of JRB310xJRB510 F1 to 1. H) Diagram of the various crosses involved generating the progeny in panels I and J. The colors from the diagram correspond to the colors in the following panel J. I) PCR amplification of genomic DNA harvested from clonal isolates and populations (labeled respectively) of the progeny of ctg14335.0 deletion line 2 backcrossed to JRB510, with parental and grandparental lines, along with uninjected JRB310xJRB510 as controls. Lower gel section shows PCR amplification of TEBP-&#x3B2; as loading control. J) Quantitative PCR on F1 genomic DNA from ctg14335.0 deletion line 2 across various regions of contig14335.0.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/10/3105/F4/graphic-4.medium.gif" width="440" height="263"/><noscript><img class="highwire-fragment fragment-image" alt="" src="https://www.g3journal.org/content/ggg/9/10/3105/F4/graphic-4.medium.gif" width="440" height="263"/></noscript></span></a><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/10/3105/F4/graphic-4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/10/3105/F4/graphic-4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494396" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/10/3105/F4/graphic-5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Deletions and remnants are epigenetically inherited A) An illustration of the contig14335.0 with primer locations depicted (purple for the detection PCR shown in panels C and E, pink for the qPCR primers, labeled with the corresponding qPCR they belong to in panels C, E, G). In addition, the locations of the MDSs (gold) and telomeres (red) on the nanochromosome are shown. B) Diagram of the various crosses involved generating the progeny involved in panels C and D. The colors in the diagram correspond to the colors in panel D. C) PCR amplification of genomic DNA harvested from populations of the progeny of ctg14335.0 deletion line 1 mated to deletion line 2, with parental lines and grandparental lines, along with uninjected 310x510 as controls. Lower gel section shows PCR amplification of TEBP-&#x3B2; as loading control. D) Quantitative PCR on the F1 from ctg14335.0 deletion line 1 mated to deletion line 2 surveyed across various regions of the contig14335.0 nanochromosome. E) Diagram of the various crosses involved generating the progeny involved in panels F and G. The colors from the diagram correspond to the colors in the following panel G. F) PCR amplification from genomic DNA harvested from populations of ctg14335.0 deletion line 1 backcrossed to wt strain JRB510 for up to 2 generations (labeled F1 and F2) with parental lines and grandparental lines along with uninjected 310x510 as controls. Lower gel section shows PCR amplification of TEBP-&#x3B2; as loading control. G) Quantitative PCR on genomic DNA used in panel A across multiple locations of ctg14335.0, together with an unrelated locus, TEBP-&#x3B1;. Arrows represent undetectable levels at the respective loci, with arrow color corresponding to the legend. Relative copy levels were determined by setting the levels of JRB310xJRB510 F1 to 1. H) Diagram of the various crosses involved generating the progeny in panels I and J. The colors from the diagram correspond to the colors in the following panel J. I) PCR amplification of genomic DNA harvested from clonal isolates and populations (labeled respectively) of the progeny of ctg14335.0 deletion line 2 backcrossed to JRB510, with parental and grandparental lines, along with uninjected JRB310xJRB510 as controls. Lower gel section shows PCR amplification of TEBP-&#x3B2; as loading control. J) Quantitative PCR on F1 genomic DNA from ctg14335.0 deletion line 2 across various regions of contig14335.0." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1294374134" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Deletions and remnants are epigenetically inherited A) An illustration of the contig14335.0 with primer locations depicted (purple for the detection PCR shown in panels C and E, pink for the qPCR primers, labeled with the corresponding qPCR they belong to in panels C, E, G). In addition, the locations of the MDSs (gold) and telomeres (red) on the nanochromosome are shown. B) Diagram of the various crosses involved generating the progeny involved in panels C and D. The colors in the diagram correspond to the colors in panel D. C) PCR amplification of genomic DNA harvested from populations of the progeny of ctg14335.0 deletion line 1 mated to deletion line 2, with parental lines and grandparental lines, along with uninjected 310x510 as controls. Lower gel section shows PCR amplification of TEBP-&#x3B2; as loading control. D) Quantitative PCR on the F1 from ctg14335.0 deletion line 1 mated to deletion line 2 surveyed across various regions of the contig14335.0 nanochromosome. E) Diagram of the various crosses involved generating the progeny involved in panels F and G. The colors from the diagram correspond to the colors in the following panel G. F) PCR amplification from genomic DNA harvested from populations of ctg14335.0 deletion line 1 backcrossed to wt strain JRB510 for up to 2 generations (labeled F1 and F2) with parental lines and grandparental lines along with uninjected 310x510 as controls. Lower gel section shows PCR amplification of TEBP-&#x3B2; as loading control. G) Quantitative PCR on genomic DNA used in panel A across multiple locations of ctg14335.0, together with an unrelated locus, TEBP-&#x3B1;. Arrows represent undetectable levels at the respective loci, with arrow color corresponding to the legend. Relative copy levels were determined by setting the levels of JRB310xJRB510 F1 to 1. H) Diagram of the various crosses involved generating the progeny in panels I and J. The colors from the diagram correspond to the colors in the following panel J. I) PCR amplification of genomic DNA harvested from clonal isolates and populations (labeled respectively) of the progeny of ctg14335.0 deletion line 2 backcrossed to JRB510, with parental and grandparental lines, along with uninjected JRB310xJRB510 as controls. Lower gel section shows PCR amplification of TEBP-&#x3B2; as loading control. J) Quantitative PCR on F1 genomic DNA from ctg14335.0 deletion line 2 across various regions of contig14335.0.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/10/3105/F4/graphic-5.medium.gif" width="248" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="" src="https://www.g3journal.org/content/ggg/9/10/3105/F4/graphic-5.medium.gif" width="248" height="440"/></noscript></span></a><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/10/3105/F4/graphic-5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/10/3105/F4/graphic-5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494396" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/10/3105/F4/graphic-6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Deletions and remnants are epigenetically inherited A) An illustration of the contig14335.0 with primer locations depicted (purple for the detection PCR shown in panels C and E, pink for the qPCR primers, labeled with the corresponding qPCR they belong to in panels C, E, G). In addition, the locations of the MDSs (gold) and telomeres (red) on the nanochromosome are shown. B) Diagram of the various crosses involved generating the progeny involved in panels C and D. The colors in the diagram correspond to the colors in panel D. C) PCR amplification of genomic DNA harvested from populations of the progeny of ctg14335.0 deletion line 1 mated to deletion line 2, with parental lines and grandparental lines, along with uninjected 310x510 as controls. Lower gel section shows PCR amplification of TEBP-&#x3B2; as loading control. D) Quantitative PCR on the F1 from ctg14335.0 deletion line 1 mated to deletion line 2 surveyed across various regions of the contig14335.0 nanochromosome. E) Diagram of the various crosses involved generating the progeny involved in panels F and G. The colors from the diagram correspond to the colors in the following panel G. F) PCR amplification from genomic DNA harvested from populations of ctg14335.0 deletion line 1 backcrossed to wt strain JRB510 for up to 2 generations (labeled F1 and F2) with parental lines and grandparental lines along with uninjected 310x510 as controls. Lower gel section shows PCR amplification of TEBP-&#x3B2; as loading control. G) Quantitative PCR on genomic DNA used in panel A across multiple locations of ctg14335.0, together with an unrelated locus, TEBP-&#x3B1;. Arrows represent undetectable levels at the respective loci, with arrow color corresponding to the legend. Relative copy levels were determined by setting the levels of JRB310xJRB510 F1 to 1. H) Diagram of the various crosses involved generating the progeny in panels I and J. The colors from the diagram correspond to the colors in the following panel J. I) PCR amplification of genomic DNA harvested from clonal isolates and populations (labeled respectively) of the progeny of ctg14335.0 deletion line 2 backcrossed to JRB510, with parental and grandparental lines, along with uninjected JRB310xJRB510 as controls. Lower gel section shows PCR amplification of TEBP-&#x3B2; as loading control. J) Quantitative PCR on F1 genomic DNA from ctg14335.0 deletion line 2 across various regions of contig14335.0." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1294374134" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Deletions and remnants are epigenetically inherited A) An illustration of the contig14335.0 with primer locations depicted (purple for the detection PCR shown in panels C and E, pink for the qPCR primers, labeled with the corresponding qPCR they belong to in panels C, E, G). In addition, the locations of the MDSs (gold) and telomeres (red) on the nanochromosome are shown. B) Diagram of the various crosses involved generating the progeny involved in panels C and D. The colors in the diagram correspond to the colors in panel D. C) PCR amplification of genomic DNA harvested from populations of the progeny of ctg14335.0 deletion line 1 mated to deletion line 2, with parental lines and grandparental lines, along with uninjected 310x510 as controls. Lower gel section shows PCR amplification of TEBP-&#x3B2; as loading control. D) Quantitative PCR on the F1 from ctg14335.0 deletion line 1 mated to deletion line 2 surveyed across various regions of the contig14335.0 nanochromosome. E) Diagram of the various crosses involved generating the progeny involved in panels F and G. The colors from the diagram correspond to the colors in the following panel G. F) PCR amplification from genomic DNA harvested from populations of ctg14335.0 deletion line 1 backcrossed to wt strain JRB510 for up to 2 generations (labeled F1 and F2) with parental lines and grandparental lines along with uninjected 310x510 as controls. Lower gel section shows PCR amplification of TEBP-&#x3B2; as loading control. G) Quantitative PCR on genomic DNA used in panel A across multiple locations of ctg14335.0, together with an unrelated locus, TEBP-&#x3B1;. Arrows represent undetectable levels at the respective loci, with arrow color corresponding to the legend. Relative copy levels were determined by setting the levels of JRB310xJRB510 F1 to 1. H) Diagram of the various crosses involved generating the progeny in panels I and J. The colors from the diagram correspond to the colors in the following panel J. I) PCR amplification of genomic DNA harvested from clonal isolates and populations (labeled respectively) of the progeny of ctg14335.0 deletion line 2 backcrossed to JRB510, with parental and grandparental lines, along with uninjected JRB310xJRB510 as controls. Lower gel section shows PCR amplification of TEBP-&#x3B2; as loading control. J) Quantitative PCR on F1 genomic DNA from ctg14335.0 deletion line 2 across various regions of contig14335.0.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/10/3105/F4/graphic-6.medium.gif" width="354" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="" src="https://www.g3journal.org/content/ggg/9/10/3105/F4/graphic-6.medium.gif" width="354" height="440"/></noscript></span></a><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/10/3105/F4/graphic-6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/10/3105/F4/graphic-6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494396" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-31" class="first-child">Deletions and remnants are epigenetically inherited A) An illustration of the contig14335.0 with primer locations depicted (purple for the detection PCR shown in panels C and E, pink for the qPCR primers, labeled with the corresponding qPCR they belong to in panels C, E, G). In addition, the locations of the MDSs (gold) and telomeres (red) on the nanochromosome are shown. B) Diagram of the various crosses involved generating the progeny involved in panels C and D. The colors in the diagram correspond to the colors in panel D. C) PCR amplification of genomic DNA harvested from populations of the progeny of ctg14335.0 deletion line 1 mated to deletion line 2, with parental lines and grandparental lines, along with uninjected 310x510 as controls. Lower gel section shows PCR amplification of TEBP-β as loading control. D) Quantitative PCR on the F1 from ctg14335.0 deletion line 1 mated to deletion line 2 surveyed across various regions of the contig14335.0 nanochromosome. E) Diagram of the various crosses involved generating the progeny involved in panels F and G. The colors from the diagram correspond to the colors in the following panel G. F) PCR amplification from genomic DNA harvested from populations of ctg14335.0 deletion line 1 backcrossed to wt strain JRB510 for up to 2 generations (labeled F1 and F2) with parental lines and grandparental lines along with uninjected 310x510 as controls. Lower gel section shows PCR amplification of TEBP-β as loading control. G) Quantitative PCR on genomic DNA used in panel A across multiple locations of ctg14335.0, together with an unrelated locus, TEBP-α. Arrows represent undetectable levels at the respective loci, with arrow color corresponding to the legend. Relative copy levels were determined by setting the levels of JRB310xJRB510 F1 to 1. H) Diagram of the various crosses involved generating the progeny in panels I and J. The colors from the diagram correspond to the colors in the following panel J. I) PCR amplification of genomic DNA harvested from clonal isolates and populations (labeled respectively) of the progeny of ctg14335.0 deletion line 2 backcrossed to JRB510, with parental and grandparental lines, along with uninjected JRB310xJRB510 as controls. Lower gel section shows PCR amplification of TEBP-β as loading control. J) Quantitative PCR on F1 genomic DNA from ctg14335.0 deletion line 2 across various regions of contig14335.0.</p><div class="sb-div caption-clear"></div></div></div><p id="p-32">To assess how common recovery was in the backcrossed progeny of Contig14335.0 deletion line 2 mated to JRB510, clonal lines were singled out. Nine such cell lines (out of 9) displayed no recovery of the full-length chromosome (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4I</a>), suggesting that recovery was infrequent. Quantitative PCR analysis of the backcrossed cells revealed an abundant center fragment derived from the deleted chromosome, with copy number at near wild type levels, as also seen in the parental Contig14335.0 deletion line 2 (<a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4J</a>). To test whether the internal telomere addition site in the parental deletion is inherited in the offspring, telomeric PCR and Sanger sequencing were performed (Supplementary figure 6). This confirmed that the highly abundant remnant in the F1 progeny has the same telomeric addition sites as those identified in the parental line, demonstrating that deletion of Contig14335.0 epigenetically transfers with incomplete penetrance to the subsequent generation, including the inheritance of a truncated telomere-containing chromosome fragment.</p><p id="p-33">Previous experiments involving programmed alterations to macronuclear chromosome structure demonstrated the gradual reversion to the wild type genotype, <em>i.e.</em>, dilution of the programed alteration, after multiple sexual generations (<a id="xref-ref-7-6" class="xref-bibr" href="#ref-7">Fang <em>et al.</em> 2012</a>, <a id="xref-ref-25-7" class="xref-bibr" href="#ref-25">Nowacki <em>et al.</em> 2008</a>). We tested the recovery of the deleted chromosome after an additional sexual generation by mating the progeny of the Contig14335.0 deletion line 1 that had been backcrossed to JRB510 in a second backcross to JRB510. The resulting F2 cells still show no recovery of the targeted chromosome, as assessed by conventional PCR and qPCR (<a id="xref-fig-4-6" class="xref-fig" href="#F4">Figure 4</a> panels F and G). Inheritance of the Contig14335.0 deletion across multiple generations suggests a strong penetrance and dominant influence of the chromosome deletion in <em>Oxytricha</em>.</p></div></div><div class="section discussion" id="sec-16"><h2 class="">Discussion</h2><div id="sec-17" class="subsection"><h3>Injection of synthetic chromosomes into mating cells results in deletion</h3><p id="p-34">Microinjection of a DNA template into <em>Oxytricha</em> cells during genome rearrangement can result in the deletion of the homologous sequence in the next generation macronucleus. This effect has been observed in many cases, implying that it is a general phenomenon. Strong sequence similarity is essential to target chromosome deletion, but the effect is robust to minor sequence variation, such as small deletions. Contig16116.0 and Contig20822.0 chromosome constructs containing small deletions, respectively 28 and 37 nucleotides, were still able to induce chromosome deletion. Surprisingly, whole genome sequencing revealed no significant off-target effects on nanochromosomes containing regions with high similarity to the deleted nanochromosomes (supplementary table 1). This suggests that target recognition occurs at a level above local similarity. We have not determined the precise level of similarity that is sufficient to cause the deletion effect in the subsequent generation.</p><p id="p-35">Feeding <em>Oxytricha</em> cells <em>E. coli</em> expressing dsRNA from a plasmid containing a nanochromosome fragment previously resulted in failed rearrangement of the targeted chromosome (<a id="xref-ref-25-8" class="xref-bibr" href="#ref-25">Nowacki <em>et al.</em> 2008</a>). It is possible that the injected DNA in the current experiments may be acting in the same pathway as the previous RNAi feeding experiments. However, previous experiments that used DNA injection resulted in notably different results. In <a id="xref-ref-25-9" class="xref-bibr" href="#ref-25">Nowacki <em>et al.</em> (2008)</a> and <a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">(2010)</a>, injection at 1 to 3 hr post-pairing (slightly earlier than the 3 to 5 hr injection timepoint in the present study) of an entire synthetic nanochromosome, including telomere overhangs, resulted not in chromosome deletion but instead in <em>increased</em> chromosome copy number in the population of cells examined. The difference in these prior results from the current study may be due to subtleties in the timing of injection, or to differences between average effects on a population of cells (<a id="xref-ref-24-3" class="xref-bibr" href="#ref-24">Nowacki <em>et al.</em> 2010</a>) <em>vs.</em> individually isolated and cloned cells. Hence, the presence of a limited number of cells containing deletions may have been overlooked in <a id="xref-ref-25-10" class="xref-bibr" href="#ref-25">Nowacki <em>et al.</em> (2008</a>, <a id="xref-ref-24-4" class="xref-bibr" href="#ref-24">2010)</a> because we observe deletions at a rate of ∼10% of injected cells, which would be missed in populations, without individual screening.</p></div><div id="sec-18" class="subsection"><h3>Comparison to other DNA deletion phenomena in ciliates</h3><p id="p-36">While programmed DNA deletion is now observed in both <em>Oxytricha</em> and <em>Paramecium</em>, the apparent substrate requirements are different. On a structural level, the deletion in <em>Paramecium</em> depends on the presence of a truncated gene product in order to generate RNAi against the gene. This is not the case in <em>Oxytricha</em> as the injection of a full length nanochromosome results in its deletion from the next generation. Additional evidence supporting the conclusion that <em>Oxytricha</em> deletion is programmed via a separate mechanism is the ability to delete essential, early conjugation genes (Otiwi1 and AL1), given that silencing in injected cells would cause lethality immediately after injection. The apparent mechanistic differences between deletion in <em>Paramecium</em> and <em>Oxytricha</em> implicate an alternative model for deletion in <em>Oxytricha</em>.</p><p id="p-37">In addition, the ciliate <em>Tetrahymena thermophila</em> has recently been reported to have endogenous programmed deletion of macronuclear chromosomes. 50 minichromosomes are transiently maintained and then lost after six generations of vegetative growth post-sexual development (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Lin <em>et al.</em> 2016</a>, <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Feng <em>et al.</em> 2017</a>). Interestingly, these minichromosomes encode post-conjugation development specific genes, suggesting the possibility of programmed elimination from the macronucleus as a mechanism for gene regulation. While the phenomenon of macronuclear chromosome deletion that we report in <em>Oxytricha</em> is induced via the injection of an artificial chromosome, it bears similarity to the observed elimination of whole chromosomes in <em>Tetrahymena</em> as a mechanism to restrict gene expression to early macronuclear development.</p></div><div id="sec-19" class="subsection"><h3>Implications of epigenetic inheritance of the deletions</h3><p id="p-38">The epigenetic inheritance shows different effects in the two cases examined, which seem to correlate with the fold-reduction in the parental deletion. In the case of Contig14335.0, deletion has a dominant effect, with the deletion propagated to the subsequent generation after parental backcross. In case of the Contig18510.0 deletion, the nanochromosome was recovered in the subsequent generation after parental backcross. One of the apparent variables between the two chromosome deletions is efficiency. The established Contig14335.0 deletion lines have strong copy number reduction, based on qPCR and genomic sequencing, whereas the Contig18510.0 deletion lines display only moderate reduction in copy number (∼six-fold decrease) based on qPCR results compared to wild type cells. The dominance of the deletion phenotype could be dependent on the strength in the parental deletion line. This dominant deletion inheritance is similar to the dominant inheritance of IES retention (<a id="xref-ref-7-7" class="xref-bibr" href="#ref-7">Fang <em>et al.</em> 2012</a>), where a normally deleted sequence is retained by progeny even when crossed to wild type cells.</p><p id="p-39">The dominant inheritance of the chromosome deletion is surprising in the case of contig14335.0, based on the current model of early genome rearrangement. In <em>Oxytricha</em>, the piRNAs in association with Otiwi1 guide retention, while RNA templates help program rearrangement. Neither of these systems can easily explain the dominant inheritance of the chromosome deletion of contig14335.0. These results suggest that in addition to these known guiding RNA molecules, there are epigenetic elements that guide deletions. Hence, the deletion lines provide novel insight into the genome rearrangement process and can be used to further investigate the subtractive mechanisms at play.</p></div><div id="sec-20" class="subsection"><h3>Model for the deletion phenomenon</h3><p id="p-40">We show that the injected template has an influence on the subsequent macronuclear generation, likely by interacting with one of the <em>Oxytricha</em> RNA pathways that guide rearrangement, and perhaps transcription of the injected templates could interfere with this pathway. We did not investigate transcription of the injected DNA, however, due to limited amount of material and the relative low efficiency of deletion (∼10% of injected cell lines result in chromosome deletion). As an alternative hypothesis to transcription of the injected product triggering DNA deletion, the injected DNA might inhibit or interfere directly with <em>Oxytricha</em>’s genome rearrangement pathway. In this second view, the injected DNA could inhibit the marking of native sequences for retention in the newly developing macronucleus, perhaps via soaking or sequestering of piRNA or RNA templates similar to the targeted sequence. In this model, the injected DNA would act as a sponge for RNA templates or piRNAs, leaving an inadequate level of RNA to target the native sequence for retention.</p></div><div id="sec-21" class="subsection"><h3>Relevance of chromosome deletion</h3><p id="p-41">Targeted deletion of chromosomes has been previously demonstrated in other model systems through the use of various genetic tools (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Matsumura <em>et al.</em> 2007</a>, <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Li <em>et al.</em> 2012</a>). Recently, targeted chromosome deletion has been generated in mice and human cell lines (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Adikusuma <em>et al.</em> 2017</a>, <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Zuo <em>et al.</em> 2017</a>) through use of CRISPR/Cas9 and induction of targeted double stranded breaks. Chromosome deletion in <em>Oxytricha</em> offers an alternative demonstration of programmed chromosome deletion and emphasizes the diversity of genome rearrangement pathways in <em>Oxytricha</em>.</p><p id="p-42">The heavy reduction in copy number of the targeted chromosome and their encoded gene(s) opens new avenues for generating somatic knockdowns or knockout strains. Moreover, despite the low rate of chromosome loss, it is still a viable method for generating mutant strains, because several dozen injected cells can be screened (even one 24-well plate can be reasonably expected to yield two deletion lines), and the resulting strains can be stably maintained and propagated for use in future experiments. Gene knockdown by IES retention (<a id="xref-ref-7-8" class="xref-bibr" href="#ref-7">Fang <em>et al.</em> 2012</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Khurana <em>et al.</em> 2018</a>) suffers from a variety of sequence constraints, in particular requiring the IES to be near the amino-terminus of the target encoded protein in order to disrupt its open reading frame. The chromosome deletion strategy that we describe here is free of this limitation, thus allowing the functional analysis of a wider range of target genes. In addition, the chromosomal deletion approach offers a simple and more potent way to knock out target genes, compared to programmed IES retention.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-173">We would like to acknowledge and thank the members of the Landweber lab, past and present, at Princeton and Columbia University for their support and advice in the development of this research, and Sam Sternberg for discussion. We would also like to acknowledge and take the opportunity to thank Wei Wang and Jessica Buckles Wiggins from the Sequencing Core Facility at Princeton University for Illumina sequencing. This work was funded by NIH grants GM59708, GM111933, and Human Frontier Science Program RGP004/2014. D.M.C. and L.F.L. developed and designed the experiments. D.J.V. made the DNA libraries. V.T.Y. mapped the reads and performed the bioinformatic analysis. D.M.C. did the rest of the experiments. D.M.C., V.T.Y., and L.F.L. edited the manuscript.</p><p id="p-174"> <em>Note Added in Proof:</em> See <a href="http://www.g3journal.org/content/9/10/3119">3119–3127</a> in this issue for a related work.</p></div><div class="section app" id="app-1"><h2 class="">Appendix</h2><div class="section" id="sec-22"><h3>Primer list:</h3><div id="sec-23" class="subsection"><h4>Chromosome creation primers</h4><div id="sec-24" class="subsection"><h5>AL1:</h5><p id="p-43">5′ end Fw:</p><p id="p-44">ATG AGA GTT TGT GAA AAA TTA AGT TTG TTG AGG TTA CTT TTG ATT GGT TAA TTA G</p><p id="p-45">3′ end Rev:</p><p id="p-46">TAT ATT AAA TAT CAA GAA AAA GTA AAA AGA CAG TAA AAA TAT ATA TAT ATT AAA TGC ATG</p><p id="p-47">Deletion generation overlap extension Fw:</p><p id="p-48">CTA CGC CCG TCC TTG TTC TCA AAA TCT CTC TCA TGG GTG GTC TGT TTT CTC TGT TTA TG</p><p id="p-49">Deletion generation overlap extension Rev:</p><p id="p-50">CAT AAA CAG AGA AAA CAG ACC ACC CAT GAG AGA GAT TTT GAG AAC AAG GAC GGG CG</p><p id="p-51">5′ telomere addition Fw:</p><p id="p-52">CCC CAA AAC CCC AAA ACC CCA TGA GAG TTT GTG AAA AAT TAA GTT TG</p><p id="p-53">3′ telomere addition Rev:</p><p id="p-54">CCC CAA AAC CCC AAA ACC CCT ATA TTA AAT ATC AAG AAA AAG TAA AAA GAC AG</p></div><div id="sec-25" class="subsection"><h5>Otiwi1:</h5><p id="p-55">5′ end Fw:</p><p id="p-56">ATG AGT ATA ATT TGA ATT GTT GTA AAG AGA GTT TCC ATT TGA TTG</p><p id="p-57">3′ end Rev:</p><p id="p-58">GAT TAA TGG ATG GAA GAG ATA GAA GTA AAT TAG AGA TGA TTA ATT TAA TAT AAT TAT AAA</p><p id="p-59">Deletion generation overlap extension Fw:</p><p id="p-60">GGA ATC CGC TTT GAC ACC AAG CTT TCC CAA GGA CAA GAG CTC AAG TTG TTC TCC</p><p id="p-61">Deletion generation overlap extension Rev:</p><p id="p-62">GGA GAA CAA CTT GAG CTC TTG TCC TTG GGA AAG CTT GGT GTC AAA GCG GAT TCC</p><p id="p-63">5′ telomere addition Fw:</p><p id="p-64">CCC CAA AAC CCC AAA ACC CCA TGA GTA TAA TTT GAA TTG TTG TAA AGA GAG TTT CC</p><p id="p-65">3′ telomere addition Rev:</p><p id="p-66">CCC CAA AAC CCC AAA ACC CCA AAA GAT TAA T</p></div><div id="sec-26" class="subsection"><h5>Contig14335.0:</h5><p id="p-67">5′ telomere Fw:</p><p id="p-68">CCC CAA AAC CCC AAA ACC CCG TAT TGG AAT TAT AAA CAA ATA TTA TGA TAG</p><p id="p-69">3′ telomere Rev:</p><p id="p-70">CCC CAA AAC CCC AAA ACC CCT TCC TGT ATA TTT CGA ATT TAA TCA G</p></div><div id="sec-27" class="subsection"><h5>Contig18510.0:</h5><p id="p-71">5′ telomere Fw:</p><p id="p-72">CCC CAA AAC CCC AAA ACC CCA TTA ATT ATA TTG TTT TTG ACA TTG TAA AAT G</p><p id="p-73">3′ telomere Rev:</p><p id="p-74">CCC CAA AAC CCC AAA ACC CCA AGG AGT GTT TCA GAA ATA GAA AAA TTC</p></div><div id="sec-28" class="subsection"><h5>Contig17155.0:</h5><p id="p-75">5′ telomere Fw:</p><p id="p-76">CCC CAA AAC CCC AAA ACC CCA TTA GAG ATA TAA CCA GAA TAT TTT ATG GG</p><p id="p-77">3′ telomere Rev:</p><p id="p-78">CCC CAA AAC CCC AAA ACC CCA TGG TTT GGA ATT TGA AAT CGT GAA GCA ATT AAT AT</p></div><div id="sec-29" class="subsection"><h5>Contig9679.0:</h5><p id="p-79">5′ telomere Fw:</p><p id="p-80">CCC CAA AAC CCC AAA ACC CCA AGA ATT AAC TAT TGA AGT TGA TAA GTA AA</p><p id="p-81">3′ telomere Rev:</p><p id="p-82">CCC CAA AAC CCC AAA ACC CC T AAG TCT ATA TAA AAG TAT TGT TTT AAA AGT A</p></div></div><div id="sec-30" class="subsection"><h4>Detection primers</h4><div id="sec-31" class="subsection"><h5>AL1:</h5><p id="p-83">Fw:</p><p id="p-84">ATG AGA GTT TGT GAA AAA TTA AGT TTG TTG AGG TTA CTT TTG ATT GGT TAA TTA G</p><p id="p-85">Rev:</p><p id="p-86">GAG AGC ACA ACT ACA CTT TGC CTC ACA CTC</p></div><div id="sec-32" class="subsection"><h5>Otiwi1:</h5><p id="p-87">Same as Otiwi1 5′ end Fw and 3′ end Rev primers</p></div><div id="sec-33" class="subsection"><h5>Conttg14335:</h5><p id="p-88">Fw:</p><p id="p-89">GAT AGA ATT GAA AGT AAA TTA GCT GTA AGG</p><p id="p-90">Rev:</p><p id="p-91">GAT CAA GCG ATC TAG CAA GGA G</p></div><div id="sec-34" class="subsection"><h5>Contig18510.0:</h5><p id="p-92">Fw:</p><p id="p-93">TAA CAT AGT CCA GCA TTA CTA AAA TGA TG</p><p id="p-94">Rev:</p><p id="p-95">CTT AAA TGA GTT AAT ATA GAT CCA ACT CG</p></div><div id="sec-35" class="subsection"><h5>Contig17155.0:</h5><p id="p-96">Fw:</p><p id="p-97">AAC TTA GCG GAA AAG TAA AAG CAA TAA TG</p><p id="p-98">Rev:</p><p id="p-99">TGA ACA TTA TCT TTC TTA AGC TGT TAC AAA AC</p></div><div id="sec-36" class="subsection"><h5>Contig9679.0:</h5><p id="p-100">Fw:</p><p id="p-101">ATT GTA GTT ACT GTA TGC TGC TGG</p><p id="p-102">Rev:</p><p id="p-103">TGT CTT CTG TCC ATA CCT TGA AGG</p></div><div id="sec-37" class="subsection"><h5>TEBP-β (Contig22260.0):</h5><p id="p-104">Fw:</p><p id="p-105">GAG CAA ATC ACA ACA AGT TCA ACA ACA AAG CG</p><p id="p-106">Rev:</p><p id="p-107">GCT CAT TTC TTG GTT GAT CTC TTT GAG GCC</p></div><div id="sec-38" class="subsection"><h5>Telomere detection:</h5><p id="p-108">AP12C4A4 (1<sup>st</sup> round):</p><p id="p-109">GTA ATA CGA CTC ACT ATA GGG CAC GCG TGG TCG ACG GCC CGG GCT GGT CCC CAA AAC CCC AAA ACC CCA AAA</p><p id="p-110">Contig14335.0 telomeric fragment Rev (1<sup>st</sup> round):</p><p id="p-111">CAC TTA TGA TAA GCA TAT CAA TCG GGG CTC</p><p id="p-112">AP2 (2<sup>nd</sup> round):</p><p id="p-113">ACT ATA GGG CAC GCG TGG T</p><p id="p-114">Contig14335.0 telomeric fragment nested Rev (2<sup>nd</sup> round):</p><p id="p-115">GGA TTA CTG GTA CTG TAG AAA GAA GGA ATG</p></div></div><div id="sec-39" class="subsection"><h4>Southern primers</h4><div id="sec-40" class="subsection"><h5>AL1 3′ Southern probe:</h5><p id="p-116">Fw:</p><p id="p-117">ATG AGA CCA TCA ACA GAG ACT ATC CCT AAC</p><p id="p-118">Rev:</p><p id="p-119">CTC TTG TCT CTC CTT CTT GAA TCG AGG GTG</p></div><div id="sec-41" class="subsection"><h5>TEBP-β:</h5><p id="p-120">Same as TEBP-β detection chromosome</p></div></div><div id="sec-42" class="subsection"><h4>qPCR primers</h4><div id="sec-43" class="subsection"><h5>Mito Cox1:</h5><p id="p-121">qPCR Fw:</p><p id="p-122">GCC GTG TTT ACG CTT ATT TAC A</p><p id="p-123">qPCR Rev:</p><p id="p-124">CGT CTA GGC ATA CCA GCA TAT C</p></div><div id="sec-44" class="subsection"><h5>AL1 (Contig20822.0):</h5><p id="p-125">5′ qPCR Fw:</p><p id="p-126">AAA GTC TGG TTC TCG TGG C</p><p id="p-127">5′ qPCR Rev:</p><p id="p-128">TCT GAA GCT TGT CCT TGG ATG</p><p id="p-129">3′ qPCR Fw:</p><p id="p-130">GAC AAG AGA CTA GCA GAC ACC</p><p id="p-131">3′ qPCR Rev:</p><p id="p-132">ACT ACA CTT TGC CTC ACA CTC</p></div><div id="sec-45" class="subsection"><h5>Otiwi1 (Contig16116.0):</h5><p id="p-133">5′ qPCR Fw:</p><p id="p-134">TTT AAC AAG AAC AGA AAA GCA TTC AAG</p><p id="p-135">5′ qPCR Rev:</p><p id="p-136">TGG ATG GTG AAT CAA ACT CGA G</p><p id="p-137">Mid qPCR Fw:</p><p id="p-138">CCT GAG ATT TGA GAC CAT CGG</p><p id="p-139">Mid qPCR Rev:</p><p id="p-140">AGT CTA GCA TCA AAT CCA GGC</p><p id="p-141">3′ qPCR Fw:</p><p id="p-142">GAG ACC AAG GAG CCA ACC</p><p id="p-143">3′ qPCR Rev:</p><p id="p-144">TCC AAT TAG TAG TTG CCA TGA GAG</p></div><div id="sec-46" class="subsection"><h5>Contig14335.0:</h5><p id="p-145">5′ qPCR Fw:</p><p id="p-146">TCT CTC ATG AAC GGT TAG AGT TTA G</p><p id="p-147">5′ qPCR Rev:</p><p id="p-148">GGA TGA GTT TAT CGA TTC TGC ATT AG</p><p id="p-149">Mid qPCR Fw:</p><p id="p-150">CCA ATA CTC GAA GTC CTC CTT G</p><p id="p-151">Mid qPCR Rev:</p><p id="p-152">GGT AAT GGA TTA CTG GTA CTG TAG AA</p><p id="p-153">3′ qPCR Fw:</p><p id="p-154">GCT CTC GAG CTC ATC TGA TTC</p><p id="p-155">3′ qPCR Rev:</p><p id="p-156">TCT CCA CTT AAT GAC GAG TAC TTA TAC</p></div><div id="sec-47" class="subsection"><h5>Contig18510.0:</h5><p id="p-157">5′ qPCR Fw:</p><p id="p-158">CTG TCA TAT TAA CAT AGT CCA GCA TTA C</p><p id="p-159">5′ qPCR Rev:</p><p id="p-160">ATC CGT ATT CCT GTC TTC CAT TAG</p><p id="p-161">Mid qPCR Fw:</p><p id="p-162">AGT ACT TCT CTC AAA CAG GAT AGA AC</p><p id="p-163">Mid qPCR Rev:</p><p id="p-164">AGA TAT TGG AGT CTC TCT TTG TGT T</p><p id="p-165">3′ qPCR Fw:</p><p id="p-166">TTC ACA ACA GCC TAA GAA GTA TCT</p><p id="p-167">3′ qPCR Rev:</p><p id="p-168">CAT TTG TAT CAT TCA ATG GCA CTC T</p></div><div id="sec-48" class="subsection"><h5>TEBP-α (Contig22209.0):</h5><p id="p-169">qPCR Fw:</p><p id="p-170">CCA CAG AGC CAC CAT CAG ACT TTA</p><p id="p-171">qPCR Rev:</p><p id="p-172">GAG AAT GGT ACG AGA TCG CTA GTA GC</p></div></div></div></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-175">Supplemental material available at FigShare: <a href="https://doi.org/10.25387/g3.7123214">https://doi.org/10.25387/g3.7123214</a>.</p></li><li class="fn" id="fn-2"><p id="p-176"><em>Communicating editor: T. Hughes</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2018-11-28"><span class="received-label">Received </span>November 28, 2018.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2019-05-28"><span class="accepted-label">Accepted </span>May 28, 2019.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2019 Clay <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3105.1" data-doi="10.1016/j.ymthe.2017.05.021"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Adikusuma</span>,  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Williams</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Grutzner</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Hughes</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Thomas</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Targeted Deletion of an Entire Chromosome Using CRISPR/Cas9.</span> <abbr class="cit-jnl-abbrev">Mol. Ther.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1736</span>–<span class="cit-lpage">1738</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.ymthe.2017.05.021</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BTher.%26rft.volume%253D25%26rft.spage%253D1736%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ymthe.2017.05.021%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ymthe.2017.05.021&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3105.2" data-doi="10.1016/j.tig.2017.01.005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Allen</span>,  <span class="cit-name-given-names">A. E.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Nowacki</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Necessity Is the Mother of Invention: Ciliates, Transposons, and Transgenerational Inheritance.</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">197</span>–<span class="cit-lpage">207</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.tig.2017.01.005</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BGenet.%26rft.volume%253D33%26rft.spage%253D197%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2017.01.005%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tig.2017.01.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3105.3" data-doi="10.1101/gad.547309"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Baudry</span>,  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Malinsky</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Restituito</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Kapusta</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Rosa</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">PiggyMac, a domesticated <em>piggyBac</em> transposase involved in programmed genome rearrangements in the ciliate <em>Paramecium tetraurelia</em>.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">2478</span>–<span class="cit-lpage">2483</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gad.547309</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DBaudry%26rft.auinit1%253DC.%26rft.volume%253D23%26rft.issue%253D21%26rft.spage%253D2478%26rft.epage%253D2483%26rft.atitle%253DPiggyMac%252C%2Ba%2Bdomesticated%2BpiggyBac%2Btransposase%2Binvolved%2Bin%2Bprogrammed%2Bgenome%2Brearrangements%2Bin%2Bthe%2Bciliate%2BParamecium%2Btetraurelia%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.547309%26rft_id%253Dinfo%253Apmid%252F19884254%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6MTA6IjIzLzIxLzI0NzgiO3M6NDoiYXRvbSI7czoxOToiL2dnZy85LzEwLzMxMDUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3105.4" data-doi="10.1080/15476286.2016.1195940"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bracht</span>,  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Wang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Shetty</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Chen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G. J.</span> <span class="cit-name-surname">Uttarotai</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Chromosome fusions triggered by noncoding RNA.</span> <abbr class="cit-jnl-abbrev">RNA Biol.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">620</span>–<span class="cit-lpage">631</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1080/15476286.2016.1195940</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRNA%2BBiol.%26rft.volume%253D14%26rft.spage%253D620%26rft_id%253Dinfo%253Adoi%252F10.1080%252F15476286.2016.1195940%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/15476286.2016.1195940&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3105.5" data-doi="10.1078/143446104774199628"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chang</span>,  <span class="cit-name-given-names">W. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N. A.</span> <span class="cit-name-surname">Stover</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">V. M.</span> <span class="cit-name-surname">Addis</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">L. F.</span> <span class="cit-name-surname">Landweber</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">A micronuclear locus containing three protein-coding genes remains linked during macronuclear development in the spirotrichous ciliate <em>Holosticha</em>.</span> <abbr class="cit-jnl-abbrev">Protist</abbr> <span class="cit-vol">155</span>: <span class="cit-fpage">245</span>–<span class="cit-lpage">255</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1078/143446104774199628</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProtist%26rft.volume%253D155%26rft.spage%253D245%26rft_id%253Dinfo%253Adoi%252F10.1078%252F143446104774199628%26rft_id%253Dinfo%253Apmid%252F15305799%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1078/143446104774199628&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15305799&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3105.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000222400900009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3105.6" data-doi="10.1016/j.cell.2014.07.034"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. R.</span> <span class="cit-name-surname">Bracht</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. D.</span> <span class="cit-name-surname">Goldman</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Dolzhenko</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. M.</span> <span class="cit-name-surname">Clay</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The Architecture of a Scrambled Genome Reveals Massive Levels of Genomic Rearrangement during Development.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">158</span>: <span class="cit-fpage">1187</span>–<span class="cit-lpage">1198</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.cell.2014.07.034</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D158%26rft.spage%253D1187%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2014.07.034%26rft_id%253Dinfo%253Apmid%252F25171416%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2014.07.034&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25171416&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3105.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3105.7" data-doi="10.1016/j.cell.2012.10.045"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fang</span>,  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Wang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. R.</span> <span class="cit-name-surname">Bracht</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Nowacki</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">L. F.</span> <span class="cit-name-surname">Landweber</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Piwi-Interacting RNAs Protect DNA Against Loss During <em>Oxytricha</em> Genome Rearrangement.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">151</span>: <span class="cit-fpage">1243</span>–<span class="cit-lpage">1255</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.cell.2012.10.045</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D151%26rft.spage%253D1243%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2012.10.045%26rft_id%253Dinfo%253Apmid%252F23217708%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2012.10.045&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23217708&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3105.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000311999900014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3105.8" data-doi="10.1093/nar/gkx652"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Feng</span>,  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Wang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. P.</span> <span class="cit-name-surname">Hamilton</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Xiong</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Yan</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">A germline-limited <em>piggyBac</em> transposase gene is required for precise excision in <em>Tetrahymena</em> genome rearrangement.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">9481</span>–<span class="cit-lpage">9502</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkx652</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft.volume%253D45%26rft.spage%253D9481%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkx652%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkx652&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3105.9" data-doi="10.1093/nar/29.21.4387"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Galvani</span>,  <span class="cit-name-given-names">A.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Sperling</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Transgene-mediated post-transcriptional gene silencing is inhibited by 3′ non-coding sequences in <em>Paramecium</em>.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">4387</span>–<span class="cit-lpage">4394</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/29.21.4387</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DGalvani%26rft.auinit1%253DA.%26rft.volume%253D29%26rft.issue%253D21%26rft.spage%253D4387%26rft.epage%253D4394%26rft.atitle%253DTransgene-mediated%2Bpost-transcriptional%2Bgene%2Bsilencing%2Bis%2Binhibited%2Bby%2B3%2527%2Bnon-coding%2Bsequences%2Bin%2BParamecium%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F29.21.4387%26rft_id%253Dinfo%253Apmid%252F11691926%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/29.21.4387&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11691926&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3105.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000171947100020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3105.10" data-doi="10.1128/MCB.24.17.7370-7379.2004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Garnier</span>,  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Serrano</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Duharcourt</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Meyer</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">RNA-mediated programming of developmental genome rearrangements in Paramecium tetraurelia.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">7370</span>–<span class="cit-lpage">7379</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1128/MCB.24.17.7370-7379.2004</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DGarnier%26rft.auinit1%253DO.%26rft.volume%253D24%26rft.issue%253D17%26rft.spage%253D7370%26rft.epage%253D7379%26rft.atitle%253DRNA-Mediated%2BProgramming%2Bof%2BDevelopmental%2BGenome%2BRearrangements%2Bin%2BParamecium%2Btetraurelia%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.24.17.7370-7379.2004%26rft_id%253Dinfo%253Apmid%252F15314149%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjEwOiIyNC8xNy83MzcwIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMC8zMTA1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3105.11" data-doi="10.1186/gb-2010-11-8-r86"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goecks</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Nekrutenko</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Taylor</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Galaxy: a comprehensive approach for supporting accessible, reproducible, and transparent computational research in the life sciences.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">R86</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/gb-2010-11-8-r86</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DGoecks%26rft.auinit1%253DJ.%26rft.volume%253D11%26rft.issue%253D8%26rft.spage%253DR86%26rft.epage%253DR86%26rft.atitle%253DGalaxy%253A%2Ba%2Bcomprehensive%2Bapproach%2Bfor%2Bsupporting%2Baccessible%252C%2Breproducible%252C%2Band%2Btransparent%2Bcomputational%2Bresearch%2Bin%2Bthe%2Blife%2Bsciences.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2010-11-8-r86%26rft_id%253Dinfo%253Apmid%252F20738864%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2010-11-8-r86&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20738864&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3105.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3105.12" data-doi="10.1093/nar/gkw267"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Götz</span>,  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Marker</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Cheaib</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Andresen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Shrestha</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Two sets of RNAi components are required for heterochromatin formation <em>in trans</em> triggered by truncated transgenes.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">5908</span>–<span class="cit-lpage">5923</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkw267</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkw267%26rft_id%253Dinfo%253Apmid%252F27085807%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkw267&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27085807&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3105.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3105.13" data-doi="10.1261/rna.061333.117"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Khurana</span>,  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. M.</span> <span class="cit-name-surname">Clay</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Moreira</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Wang</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">L. F.</span> <span class="cit-name-surname">Landweber</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Small RNA-mediated regulation of DNA dosage in the ciliate Oxytricha.</span> <abbr class="cit-jnl-abbrev">RNA</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">18</span>–<span class="cit-lpage">29</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1261/rna.061333.117</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRNA%26rft_id%253Dinfo%253Adoi%252F10.1261%252Frna.061333.117%26rft_id%253Dinfo%253Apmid%252F29079634%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoicm5hIjtzOjU6InJlc2lkIjtzOjc6IjI0LzEvMTgiO3M6NDoiYXRvbSI7czoxOToiL2dnZy85LzEwLzMxMDUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3105.14" data-doi="10.1534/genetics.114.163279"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Khurana</span>,  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Wang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Chen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. H.</span> <span class="cit-name-surname">Perlman</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">L. F.</span> <span class="cit-name-surname">Landweber</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Transcription-Independent Functions of an RNA Polymerase II Subunit, Rpb2, During Genome Rearrangement in the Ciliate, <em>Oxytricha trifallax</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">197</span>: <span class="cit-fpage">839</span>–<span class="cit-lpage">849</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.114.163279</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.163279%26rft_id%253Dinfo%253Apmid%252F24793090%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk3LzMvODM5IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMC8zMTA1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3105.15" data-doi="10.1093/nar/23.11.2006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Klobutcher</span>,  <span class="cit-name-given-names">L. A.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Herrick</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Consensus inverted terminal repeat sequence of <em>Paramecium</em> IESs: Resemblance to termini of Tc1-related and <em>Euplotes</em> Tec transposons.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">2006</span>–<span class="cit-lpage">2013</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/23.11.2006</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F23.11.2006%26rft_id%253Dinfo%253Apmid%252F7596830%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/23.11.2006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7596830&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3105.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1995RE99600025&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3105.16" data-doi="10.1093/bioinformatics/btp324"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">H.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Durbin</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Fast and accurate short read alignment with Burrows–Wheeler transform.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1754</span>–<span class="cit-lpage">1760</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btp324</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp324%26rft_id%253Dinfo%253Apmid%252F19451168%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp324&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19451168&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3105.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000267665900006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3105.17" data-doi="10.1093/bioinformatics/btp352"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Handsaker</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Wysoker</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Fennell</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Ruan</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The Sequence Alignment/Map format and SAMtools.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">2078</span>–<span class="cit-lpage">2079</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btp352</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp352%26rft_id%253Dinfo%253Apmid%252F19505943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp352&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19505943&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3105.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268808600014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3105.18" data-doi="10.1016/j.stem.2012.08.004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">L. B.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.-H.</span> <span class="cit-name-surname">Chang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.-R.</span> <span class="cit-name-surname">Wang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. K.</span> <span class="cit-name-surname">Hirata</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Papayannopoulou</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Trisomy Correction in Down Syndrome Induced Pluripotent Stem Cells.</span> <abbr class="cit-jnl-abbrev">Cell Stem Cell</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">615</span>–<span class="cit-lpage">619</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.stem.2012.08.004</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DCell%2BStem%2BCell%26rft.aulast%253DLi%26rft.auinit1%253DL.%2BB.%26rft.volume%253D11%26rft.issue%253D5%26rft.spage%253D615%26rft.epage%253D619%26rft.atitle%253DTrisomy%2Bcorrection%2Bin%2Bdown%2Bsyndrome%2Binduced%2Bpluripotent%2Bstem%2Bcells.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.stem.2012.08.004%26rft_id%253Dinfo%253Apmid%252F23084023%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.stem.2012.08.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23084023&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3105.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000311471900008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3105.19" data-doi="10.1371/journal.pgen.1006403"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>,  <span class="cit-name-given-names">C.-Y. G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">I.-T.</span> <span class="cit-name-surname">Lin</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.-C.</span> <span class="cit-name-surname">Yao</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Programmed Minichromosome Elimination as a Mechanism for Somatic Genome Reduction in <em>Tetrahymena thermophila</em>.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">12</span>: <span class="cit-elocation-id">e1006403</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1006403</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D12%26rft.spage%253De1006403%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1006403%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1006403&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3105.20" data-doi="10.1261/rna.058511.116"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lindblad</span>,  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. R.</span> <span class="cit-name-surname">Bracht</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. E.</span> <span class="cit-name-surname">Williams</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">L. F.</span> <span class="cit-name-surname">Landweber</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Thousands of RNA-cached copies of whole chromosomes are present in the ciliate <em>Oxytricha</em> during development.</span> <abbr class="cit-jnl-abbrev">RNA</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">1200</span>–<span class="cit-lpage">1208</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1261/rna.058511.116</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRNA%26rft_id%253Dinfo%253Adoi%252F10.1261%252Frna.058511.116%26rft_id%253Dinfo%253Apmid%252F28450531%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoicm5hIjtzOjU6InJlc2lkIjtzOjk6IjIzLzgvMTIwMCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzkvMTAvMzEwNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3105.21"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lindblad</span>,  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. S.</span> <span class="cit-name-surname">Pathmanathan</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Moreira</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. R.</span> <span class="cit-name-surname">Bracht</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. P.</span> <span class="cit-name-surname">Sebra</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. R.</span> <span class="cit-name-surname">Hutton</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2019</span> Capture of complete ciliate chromosomes in single sequencing reads reveals widespread chromosome isoforms (accepted).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3105.22" data-doi="10.1038/nmeth973"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Matsumura</span>,  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Tada</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Otsuji</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Yasuchika</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Nakatsuji</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Targeted chromosome elimination from ES-somatic hybrid cells.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">23</span>–<span class="cit-lpage">25</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nmeth973</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DMatsumura%26rft.auinit1%253DH.%26rft.volume%253D4%26rft.issue%253D1%26rft.spage%253D23%26rft.epage%253D25%26rft.atitle%253DTargeted%2Bchromosome%2Belimination%2Bfrom%2BES-somatic%2Bhybrid%2Bcells.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth973%26rft_id%253Dinfo%253Apmid%252F17086180%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth973&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17086180&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3105.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000243225600011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3105.23" data-doi="10.1101/gad.6.2.211"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meyer</span>,  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">Induction of specific macronuclear developmental mutations by microinjection of a cloned telomeric gene in <em>Paramecium primaurelia</em>.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">211</span>–<span class="cit-lpage">222</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gad.6.2.211</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DMeyer%26rft.auinit1%253DE%26rft.volume%253D6%26rft.issue%253D2%26rft.spage%253D211%26rft.epage%253D222%26rft.atitle%253DInduction%2Bof%2Bspecific%2Bmacronuclear%2Bdevelopmental%2Bmutations%2Bby%2Bmicroinjection%2Bof%2Ba%2Bcloned%2Btelomeric%2Bgene%2Bin%2BParamecium%2Bprimaurelia.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.6.2.211%26rft_id%253Dinfo%253Apmid%252F1737617%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6NzoiNi8yLzIxMSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzkvMTAvMzEwNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3105.24" data-doi="10.1073/pnas.1012236107"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nowacki</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. E.</span> <span class="cit-name-surname">Haye</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Fang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Vijayan</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">L. F.</span> <span class="cit-name-surname">Landweber</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">RNA-mediated epigenetic regulation of DNA copy number.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">107</span>: <span class="cit-fpage">22140</span>–<span class="cit-lpage">22144</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.1012236107</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1012236107%26rft_id%253Dinfo%253Apmid%252F21078984%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA3LzUxLzIyMTQwIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMC8zMTA1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3105.25" data-doi="10.1038/nature06452"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nowacki</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Vijayan</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Zhou</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Schotanus</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. G.</span> <span class="cit-name-surname">Doak</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">RNA-mediated epigenetic programming of a genome-rearrangement pathway.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">451</span>: <span class="cit-fpage">153</span>–<span class="cit-lpage">158</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature06452</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DNowacki%26rft.auinit1%253DM.%26rft.volume%253D451%26rft.issue%253D7175%26rft.spage%253D153%26rft.epage%253D158%26rft.atitle%253DRNA-mediated%2Bepigenetic%2Bprogramming%2Bof%2Ba%2Bgenome-rearrangement%2Bpathway.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature06452%26rft_id%253Dinfo%253Apmid%252F18046331%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature06452&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18046331&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3105.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-web" id="cit-9.10.3105.26"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Phanstiel</span>,  <span class="cit-name-given-names">D. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <em>Sushi: Tools for visualizing genomics data</em>. R package version 1.16.0.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3105.27"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Prescott</span>,  <span class="cit-name-given-names">D. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">The DNA of ciliated protozoa.</span> <abbr class="cit-jnl-abbrev">Microbiol. Rev.</abbr> <span class="cit-vol">58</span>: <span class="cit-fpage">233</span>–<span class="cit-lpage">267</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMicrobiology%2Band%2BMolecular%2BBiology%2BReviews%26rft.stitle%253DMicrobiol.%2BMol.%2BBiol.%2BRev.%26rft.aulast%253DPrescott%26rft.auinit1%253DD%2BM%26rft.volume%253D58%26rft.issue%253D2%26rft.spage%253D233%26rft.epage%253D267%26rft.atitle%253DThe%2BDNA%2Bof%2Bciliated%2Bprotozoa.%26rft_id%253Dinfo%253Apmid%252F8078435%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoibW1iciI7czo1OiJyZXNpZCI7czo4OiI1OC8yLzIzMyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzkvMTAvMzEwNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3105.28" data-doi="10.1093/bioinformatics/btq033"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Quinlan</span>,  <span class="cit-name-given-names">A. R.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">I. M.</span> <span class="cit-name-surname">Hall</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">BEDTools: a flexible suite of utilities for comparing genomic features.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">841</span>–<span class="cit-lpage">842</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btq033</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq033%26rft_id%253Dinfo%253Apmid%252F20110278%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btq033&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20110278&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3105.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000275243500019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3105.29" data-doi="10.1091/mbc.9.4.931"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ruiz</span>,  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Vayssié</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Klotz</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Sperling</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Madeddu</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Homology-dependent gene silencing in <em>Paramecium</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Cell</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">931</span>–<span class="cit-lpage">943</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1091/mbc.9.4.931</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Bof%2Bthe%2BCell%26rft.stitle%253DMol.%2BBiol.%2BCell%26rft.aulast%253DRuiz%26rft.auinit1%253DF.%26rft.volume%253D9%26rft.issue%253D4%26rft.spage%253D931%26rft.epage%253D943%26rft.atitle%253DHomology-dependent%2BGene%2BSilencing%2Bin%2BParamecium%26rft_id%253Dinfo%253Adoi%252F10.1091%252Fmbc.9.4.931%26rft_id%253Dinfo%253Apmid%252F9529389%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xjZWxsIjtzOjU6InJlc2lkIjtzOjc6IjkvNC85MzEiO3M6NDoiYXRvbSI7czoxOToiL2dnZy85LzEwLzMxMDUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3105.30" data-doi="10.1371/journal.pbio.1001473"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Swart</span>,  <span class="cit-name-given-names">E. C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. R.</span> <span class="cit-name-surname">Bracht</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Magrini</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Minx</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Chen</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The <em>Oxytricha trifallax</em> Macronuclear Genome: A Complex Eukaryotic Genome with 16,000 Tiny Chromosomes.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">11</span>: <span class="cit-elocation-id">e1001473</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pbio.1001473</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BBiol.%26rft.volume%253D11%26rft.spage%253De1001473%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.1001473%26rft_id%253Dinfo%253Apmid%252F23382650%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.1001473&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23382650&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3105.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3105.31"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yerlici</span>,  <span class="cit-name-given-names">V. T.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L. F.</span> <span class="cit-name-surname">Landweber</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> Programmed genome rearrangements in the ciliate <em>Oxytricha. Microbiology Spectrum</em> <strong>2</strong>: <a href="https://doi.org/10.1128/microbiolspec.MDNA300252014">https://doi.org/10.1128/microbiolspec.MDNA3 0025 2014</a></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3105.32"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yerlici</span>,  <span class="cit-name-given-names">V. T.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. W.</span> <span class="cit-name-surname">Lu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. R.</span> <span class="cit-name-surname">Hoge</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. V.</span> <span class="cit-name-surname">Miller</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Neme</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2019</span> Programmed genome rearrangements in Oxytricha produce transcriptionally active extrachromosomal circular DNA. Nucleic Acids Res. pii: gkz725. <a href="https://doi.org/10.1093/nar/gkz725">https://doi.org/10.1093/nar/gkz725</a></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3105.33" data-doi="10.1371/journal.pone.0042371"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zahler</span>,  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z. T.</span> <span class="cit-name-surname">Neeb</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Lin</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Katzman</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Mating of the stichotrichous ciliate <em>Oxytricha trifallax</em> induces production of a class of 27 nt small RNAs derived from the parental macronucleus.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-elocation-id">e42371</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0042371</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DZahler%26rft.auinit1%253DA.%2BM.%26rft.volume%253D7%26rft.issue%253D8%26rft.spage%253De42371%26rft.epage%253De42371%26rft.atitle%253DMating%2Bof%2Bthe%2Bstichotrichous%2Bciliate%2BOxytricha%2Btrifallax%2Binduces%2Bproduction%2Bof%2Ba%2Bclass%2Bof%2B27%2Bnt%2Bsmall%2BRNAs%2Bderived%2Bfrom%2Bthe%2Bparental%2Bmacronucleus.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0042371%26rft_id%253Dinfo%253Apmid%252F22900016%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0042371&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22900016&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3105.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-9.10.3105.34" data-doi="10.1186/s13059-017-1354-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zuo</span>,  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Huo</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Yao</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Hu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Sun</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">CRISPR/Cas9-mediated targeted chromosome elimination.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">224</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s13059-017-1354-4</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D18%26rft.spage%253D224%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-017-1354-4%26rft_id%253Dinfo%253Apmid%252F29178945%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13059-017-1354-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=29178945&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F10%2F3105.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/9/10/3105.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/10/3101" title="micRocounter: Microsatellite Characterization in Genome Assemblies" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/10/3119" title="Transformation with Artificial Chromosomes in Oxytricha trifallax and Their Applications" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/9/10"><span>Volume 9 Issue 10, October 2019</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/9/10.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/9/10.cover-source.jpg?itok=yA8kifEd" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 9 (10)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/10/3105" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-0iMHSjU8pufIroWGrxzA_MWfFUZUYsX7_vieyUsJDHk" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/9/10/3105" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/9/10/3105" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/9/10/3105" class="active" data-icon-position="" data-hide-link-title="0">Programmed Chromosome Deletion in the Ciliate Oxytricha trifallax</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/494273" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-ttaNNEIRCTEkHTucL7NKbs5ZQQ1aFLe-CKyAgJAzQ-E" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/10/3105.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/9/10/3105" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-rY_A50V2SaeSTS7e0iNH4O0ZhmKWRi2FzPElw3-pU7A" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/9/10/3105" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/10/3105.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494273" id="node494273--2" data-pisa="ggg;9/10/3105" data-pisa-master="ggg;g3.118.200930" data-apath="/ggg/9/10/3105.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Programmed Chromosome Deletion in the Ciliate <em>Oxytricha trifallax</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Derek M. Clay</span>, <span class="highwire-citation-author" data-delta="1">V. Talya Yerlici</span>, <span class="highwire-citation-author" data-delta="2">Danylo J. Villano</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-7030-8540" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Laura F. Landweber</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3105-3118;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200930 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494273--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Derek M. Clay </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Departments of Biochemistry and Molecular Biophysics and Biological Sciences, Columbia University, New York, NY and</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Molecular Biology, Princeton University, Princeton, NJ</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Derek%2BM.%2BClay%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Clay%20DM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADerek%2BM.%2BClay%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">V. Talya Yerlici </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Departments of Biochemistry and Molecular Biophysics and Biological Sciences, Columbia University, New York, NY and</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Molecular Biology, Princeton University, Princeton, NJ</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=V.%2BTalya%2BYerlici%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yerlici%20VT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AV.%2BTalya%2BYerlici%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Danylo J. Villano </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Departments of Biochemistry and Molecular Biophysics and Biological Sciences, Columbia University, New York, NY and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Danylo%2BJ.%2BVillano%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Villano%20DJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADanylo%2BJ.%2BVillano%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Laura F. Landweber </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Departments of Biochemistry and Molecular Biophysics and Biological Sciences, Columbia University, New York, NY and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Laura%2BF.%2BLandweber%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Landweber%20LF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALaura%2BF.%2BLandweber%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-7030-8540" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Laura F. Landweber</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:Laura.Landweber@columbia.edu" class="" data-icon-position="" data-hide-link-title="0">Laura.Landweber@columbia.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/9/10/3105" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/9/10/3105&amp;title=Programmed%20Chromosome%20Deletion%20in%20the%20Ciliate%20Oxytricha%20trifallax" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/9/10/3105&amp;title=Programmed%20Chromosome%20Deletion%20in%20the%20Ciliate%20Oxytricha%20trifallax" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/9/10/3105&amp;title=Programmed%20Chromosome%20Deletion%20in%20the%20Ciliate%20Oxytricha%20trifallax" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/9/10/3105&amp;text=Programmed%20Chromosome%20Deletion%20in%20the%20Ciliate%20Oxytricha%20trifallax" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/9/10/3105&amp;title=Programmed%20Chromosome%20Deletion%20in%20the%20Ciliate%20Oxytricha%20trifallax" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/9/10/3105&amp;t=Programmed%20Chromosome%20Deletion%20in%20the%20Ciliate%20Oxytricha%20trifallax" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/9/10/3105&amp;title=Programmed%20Chromosome%20Deletion%20in%20the%20Ciliate%20Oxytricha%20trifallax" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/9/10/3105&amp;title=Programmed%20Chromosome%20Deletion%20in%20the%20Ciliate%20Oxytricha%20trifallax" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/494273/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494273" id="node494273--3" data-pisa="ggg;9/10/3105" data-pisa-master="ggg;g3.118.200930" data-apath="/ggg/9/10/3105.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Programmed Chromosome Deletion in the Ciliate <em>Oxytricha trifallax</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Derek M. Clay</span>, <span class="highwire-citation-author" data-delta="1">V. Talya Yerlici</span>, <span class="highwire-citation-author" data-delta="2">Danylo J. Villano</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-7030-8540" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Laura F. Landweber</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3105-3118;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200930 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494273--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Derek M. Clay </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Departments of Biochemistry and Molecular Biophysics and Biological Sciences, Columbia University, New York, NY and</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Molecular Biology, Princeton University, Princeton, NJ</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Derek%2BM.%2BClay%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Clay%20DM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADerek%2BM.%2BClay%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">V. Talya Yerlici </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Departments of Biochemistry and Molecular Biophysics and Biological Sciences, Columbia University, New York, NY and</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Molecular Biology, Princeton University, Princeton, NJ</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=V.%2BTalya%2BYerlici%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yerlici%20VT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AV.%2BTalya%2BYerlici%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Danylo J. Villano </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Departments of Biochemistry and Molecular Biophysics and Biological Sciences, Columbia University, New York, NY and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Danylo%2BJ.%2BVillano%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Villano%20DJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADanylo%2BJ.%2BVillano%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Laura F. Landweber </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Departments of Biochemistry and Molecular Biophysics and Biological Sciences, Columbia University, New York, NY and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Laura%2BF.%2BLandweber%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Landweber%20LF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALaura%2BF.%2BLandweber%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-7030-8540" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Laura F. Landweber</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:Laura.Landweber@columbia.edu" class="" data-icon-position="" data-hide-link-title="0">Laura.Landweber@columbia.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=3105&amp;rft.epage=3118&amp;rft.atitle=Programmed%20Chromosome%20Deletion%20in%20the%20Ciliate%20Oxytricha%20trifallax&amp;rft.volume=9&amp;rft.issue=10&amp;rft.date=2019-10-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Clay%2C+Derek+M.&amp;rft.au=Yerlici%2C+V.+Talya&amp;rft.au=Villano%2C+Danylo+J.&amp;rft.au=Landweber%2C+Laura+F."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/494273/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/494273/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/494273/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/494273/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/494273/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/494273/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/494273/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/494273/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/494273/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/494273/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/494273/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/494273/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/9/10/3105" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Methods</a></li><li class="odd"><a href="#sec-11" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-16" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#app-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Appendix</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/9/10/3105.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/9/10/3105.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/9/10/3105.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__c-64NG7CB_fwZbyex-B49GkX1AvVESfu7M9yxTNDG_o__Uvi1liJMEKbHtHHv_1fo_FCLvimvbq9jMwPcCPR364Q__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
