<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/9/11/3663" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/9/11/3663.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/9/11/3663.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/9/11/3663.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/9/11/3663.atom" />
<meta name="HW.pisa" content="ggg;9/11/3663" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Genome-Wide Variant Identification and High-Density Genetic Map Construction Using RADseq for Platycladus orientalis (Cupressaceae)" />
<meta name="DC.Identifier" content="10.1534/g3.119.400684" />
<meta name="DC.Date" content="2019-11-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2019 Jin et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Platycladus orientalis is an ecologically important native conifer in Northern China and exotic species in many parts of the world; however, knowledge about the species’ genetics and genome are very limited. The availability of well-developed battery of genetic markers, with large genome coverage, is a prerequisite for the species genetic dissection of adaptive attributes and efficient selective breeding. Here, we present a genome-wide genotyping method with double-digestion restriction site associated DNA sequencing (ddRAD-seq) that is effective in generating large number of Mendelian markers for genome mapping and other genetic applications. Using 139 megagametophytes collected from a single mother tree, we assembled 397,226 loci, of which 108,683 (27.4%) were polymorphic. After stringent filtering for 1:1 segregation ratio and missing rate of &lt;20%, the remaining 23,926 loci (22% of the polymorphic loci) were ordered into 11 linkage groups (LGs) and distributed across 7,559 unique positions, with a total map length of 1,443 cM and an average spacing of 0.2 cM between adjacent unique positions. The 11 LGs correspond to the species’ 11 haploid genome chromosome number. This genetic map is among few high-density maps available for conifers to date, and represents the first genetic map for P. orientalis . The information generated serves as a solid foundation not only for marker-assisted breeding efforts, but also for comparative conifer genomic studies." />
<meta name="DC.Contributor" content="Yuqing Jin" />
<meta name="DC.Contributor" content="Wei Zhao" />
<meta name="DC.Contributor" content="Shuai Nie" />
<meta name="DC.Contributor" content="Si-Si Liu" />
<meta name="DC.Contributor" content="Yousry A. El-Kassaby" />
<meta name="DC.Contributor" content="Xiao-Ru Wang" />
<meta name="DC.Contributor" content="Jian-Feng Mao" />
<meta name="article:published_time" content="2019-11-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Genome-Wide Variant Identification and High-Density Genetic Map Construction Using RADseq for Platycladus orientalis (Cupressaceae)" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;&lt;i&gt;Platycladus orientalis&lt;/i&gt; is an ecologically important native conifer in Northern China and exotic species in many parts of the world; however, knowledge about the species’ genetics and genome are very limited. The availability of well-developed battery of genetic markers, with large genome coverage, is a prerequisite for the species genetic dissection of adaptive attributes and efficient selective breeding. Here, we present a genome-wide genotyping method with double-digestion restriction site associated DNA sequencing (ddRAD-seq) that is effective in generating large number of Mendelian markers for genome mapping and other genetic applications. Using 139 megagametophytes collected from a single mother tree, we assembled 397,226 loci, of which 108,683 (27.4%) were polymorphic. After stringent filtering for 1:1 segregation ratio and missing rate of &amp;lt;20%, the remaining 23,926 loci (22% of the polymorphic loci) were ordered into 11 linkage groups (LGs) and distributed across 7,559 unique positions, with a total map length of 1,443 cM and an average spacing of 0.2 cM between adjacent unique positions. The 11 LGs correspond to the species’ 11 haploid genome chromosome number. This genetic map is among few high-density maps available for conifers to date, and represents the first genetic map for &lt;i&gt;P. orientalis&lt;/i&gt;. The information generated serves as a solid foundation not only for marker-assisted breeding efforts, but also for comparative conifer genomic studies.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2019/11/01" />
<meta name="citation_mjid" content="ggg;9/11/3663" />
<meta name="citation_id" content="9/11/3663" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/9/11/3663" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/9/11/3663.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/9/11/3663.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/9/11/3663.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.119.400684" />
<meta name="citation_pmid" content="31506321" />
<meta name="citation_volume" content="9" />
<meta name="citation_issue" content="11" />
<meta name="citation_num_pages" content="10" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="3663" />
<meta name="citation_lastpage" content="3672" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Yuqing Jin" />
<meta name="citation_author_institution" content="Beijing Advanced Innovation Center for Tree Breeding by Molecular Design, National Engineering Laboratory for Tree Breeding, Key Laboratory of Genetics and Breeding in Forest Trees and Ornamental Plants of Ministry of Education, College of Biological Sciences and Technology, Beijing Forestry University, Beijing, 100083, China," />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-8275-6367" />
<meta name="citation_author" content="Wei Zhao" />
<meta name="citation_author_institution" content="Beijing Advanced Innovation Center for Tree Breeding by Molecular Design, National Engineering Laboratory for Tree Breeding, Key Laboratory of Genetics and Breeding in Forest Trees and Ornamental Plants of Ministry of Education, College of Biological Sciences and Technology, Beijing Forestry University, Beijing, 100083, China," />
<meta name="citation_author_institution" content="Department of Ecology and Environmental Science, UPSC, Umeå University, SE-901 87 Umeå, Sweden, and" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-9437-3198" />
<meta name="citation_author" content="Shuai Nie" />
<meta name="citation_author_institution" content="Beijing Advanced Innovation Center for Tree Breeding by Molecular Design, National Engineering Laboratory for Tree Breeding, Key Laboratory of Genetics and Breeding in Forest Trees and Ornamental Plants of Ministry of Education, College of Biological Sciences and Technology, Beijing Forestry University, Beijing, 100083, China," />
<meta name="citation_author" content="Si-Si Liu" />
<meta name="citation_author_institution" content="Beijing Advanced Innovation Center for Tree Breeding by Molecular Design, National Engineering Laboratory for Tree Breeding, Key Laboratory of Genetics and Breeding in Forest Trees and Ornamental Plants of Ministry of Education, College of Biological Sciences and Technology, Beijing Forestry University, Beijing, 100083, China," />
<meta name="citation_author" content="Yousry A. El-Kassaby" />
<meta name="citation_author_institution" content="Department of Forest and Conservation Sciences, Faculty of Forestry, The University of British Columbia, Vancouver, British Columbia, V6T 1Z4, Canada" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-4887-8977" />
<meta name="citation_author" content="Xiao-Ru Wang" />
<meta name="citation_author_institution" content="Beijing Advanced Innovation Center for Tree Breeding by Molecular Design, National Engineering Laboratory for Tree Breeding, Key Laboratory of Genetics and Breeding in Forest Trees and Ornamental Plants of Ministry of Education, College of Biological Sciences and Technology, Beijing Forestry University, Beijing, 100083, China," />
<meta name="citation_author_institution" content="Department of Ecology and Environmental Science, UPSC, Umeå University, SE-901 87 Umeå, Sweden, and" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-6150-7046" />
<meta name="citation_author" content="Jian-Feng Mao" />
<meta name="citation_author_institution" content="Beijing Advanced Innovation Center for Tree Breeding by Molecular Design, National Engineering Laboratory for Tree Breeding, Key Laboratory of Genetics and Breeding in Forest Trees and Ornamental Plants of Ministry of Education, College of Biological Sciences and Technology, Beijing Forestry University, Beijing, 100083, China," />
<meta name="citation_author_email" content="jianfeng.mao@bjfu.edu.cn" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-9735-8516" />
<meta name="citation_reference" content="citation_year=2010" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=N. A. Baird;citation_author=P. D. Etter;citation_author=T. S. Atwood;citation_author=M. C. Currey;citation_author=A. L. Shiver;citation_title=Rapid SNP discovery and genetic mapping using sequenced RAD markers.;citation_pages=e3376-e3376;citation_volume=3;citation_year=2008;citation_issue=10;citation_pmid=18852878;citation_doi=10.1371/journal.pone.0003376" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=T. M. Beissinger;citation_author=C. N. Hirsch;citation_author=R. S. Sekhon;citation_author=J. M. Foerster;citation_author=J. M. Johnson;citation_title=Marker density and read depth for genotyping populations using genotyping-by-sequencing.;citation_volume=193;citation_year=2013;citation_pmid=23410831;citation_doi=10.1534/genetics.112.147710" />
<meta name="citation_reference" content="Bekaert, M., 2016 Genetic-Mapper: vectorial genetic map drawer. F1000Research. Available at: https://github.com/pseudogene/genetic-mapper." />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=C. Bernhardsson;citation_author=A. Vidalis;citation_author=X. Wang;citation_author=D. G. Scofield;citation_author=B. Schiffthaler;citation_title=An ultra-dense haploid genetic map for evaluating the highly fragmented genome assembly of Norway spruce (Picea abies).;citation_volume=9;citation_year=2019;citation_pmid=30898899;citation_doi=10.1534/g3.118.200840" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=I. Birol;citation_author=A. Raymond;citation_author=S. D. Jackman;citation_author=S. Pleasance;citation_author=R. Coope;citation_title=Assembling the 20 Gb white spruce (Picea glauca) genome from whole-genome shotgun sequencing data.;citation_volume=29;citation_year=2013;citation_pmid=23698863;citation_doi=10.1093/bioinformatics/btt178" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. M. Bolger;citation_author=M. Lohse;citation_author=B. Usadel;citation_title=Trimmomatic: a flexible trimmer for Illumina sequence data.;citation_volume=30;citation_year=2014;citation_pmid=24695404;citation_doi=10.1093/bioinformatics/btu170" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=K. H. Buetow;citation_title=Influence of aberrant observations on high-resolution linkage analysis outcomes.;citation_pages=985-994;citation_volume=49;citation_year=1991;citation_issue=5;citation_pmid=1928104" />
<meta name="citation_reference" content="citation_journal_title=J. Bot.;citation_author=J. G. Burleign;citation_author=W. B. Barbazuk;citation_author=J. M. Davis;citation_author=A. M. Morse;citation_author=P. S. Soltis;citation_title=Exploring diversification and genome size evolution in extant gymnosperms through phylogenetic synthesis.;citation_pages=292857;citation_volume=2012;citation_year=2012" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=M. W. Crepeau;citation_author=C. H. Langley;citation_author=K. A. Stevens;citation_title=From pine cones to read clouds: rescaffolding the megagenome of sugar pine (Pinus lambertiana).;citation_volume=7;citation_year=2017;citation_pmid=28341701;citation_doi=10.1534/g3.117.040055" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=J. Catchen;citation_author=P. A. Hohenlohe;citation_author=S. Bassham;citation_author=A. Amores;citation_author=W. A. Cresko;citation_title=Stacks: an analysis tool set for population genomics.;citation_pages=3124-3140;citation_volume=22;citation_year=2013;citation_pmid=23701397;citation_doi=10.1111/mec.12354" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=J. W. Davey;citation_author=P. A. Hohenlohe;citation_author=P. D. Etter;citation_author=J. Q. Boone;citation_author=J. M. Catchen;citation_title=Genome-wide genetic marker discovery and genotyping using next-generation sequencing.;citation_pages=499-510;citation_volume=12;citation_year=2011;citation_issue=7;citation_pmid=21681211;citation_doi=10.1038/nrg3012" />
<meta name="citation_reference" content="citation_journal_title=Tree Genet. Genomes;citation_author=C. J. Friedline;citation_author=B. M. Lind;citation_author=E. M. Hobson;citation_author=D. E. Harwood;citation_author=A. D. Mix;citation_title=The genetic architecture of local adaptation I: The genomic landscape of foxtail pine (Pinus balfouriana Grev. &amp; Balf.) as revealed from a high-density linkage map.;citation_pages=1-15;citation_volume=11;citation_year=2015;citation_doi=10.1007/s11295-015-0866-x" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=T. R. Gregory;citation_author=J. A. Nicol;citation_author=H. Tamm;citation_author=B. Kullman;citation_author=K. Kullman;citation_title=Eukaryotic genome size databases.;citation_volume=35;citation_year=2007;citation_pmid=17090588;citation_doi=10.1093/nar/gkl828" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_journal_abbrev=Heredity;citation_author=C. A. Hackett;citation_author=L. B. Broadfoot;citation_title=Effects of genotyping errors, missing values and segregation distortion in molecular marker data on the construction of linkage maps.;citation_pages=33-38;citation_volume=90;citation_year=2003;citation_issue=1;citation_pmid=12522423;citation_doi=10.1038/sj.hdy.6800173" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=C. M. Hollenbeck;citation_author=D. S. Portnoy;citation_author=D. Wetzel;citation_author=T. A. Sherwood;citation_author=P. B. Samollow;citation_title=Linkage mapping and comparative genomics of red drum (Sciaenops Ocellatus) using next-generation sequencing.;citation_volume=7;citation_year=2017;citation_pmid=28122951;citation_doi=10.1534/g3.116.036350" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=X. G. Hu;citation_author=H. Liu;citation_author=Y. Jin;citation_author=Y. Q. Sun;citation_author=Y. Li;citation_title=De novo transcriptome assembly and characterization for the widespread and stress-tolerant conifer Platycladus orientalis.;citation_pages=e0148985;citation_volume=11;citation_year=2016;citation_doi=10.1371/journal.pone.0148985" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=X. G. Hu;citation_author=Y. Jin;citation_author=X. R. Wang;citation_author=J. F. Mao;citation_author=Y. Li;citation_title=Predicting impacts of future climate change on the distribution of the widespread conifer Platycladus orientalis.;citation_pages=e0132326;citation_volume=10;citation_year=2015;citation_doi=10.1371/journal.pone.0132326" />
<meta name="citation_reference" content="citation_journal_title=Can. J. For. Res.;citation_author=L. S. Huang;citation_author=J. Song;citation_author=Y. Q. Sun;citation_author=Q. Gao;citation_author=S. Q. Jiao;citation_title=Pollination dynamics in a Platycladus orientalis seed orchard as revealed by partial pedigree reconstruction.;citation_pages=952-957;citation_volume=48;citation_year=2018;citation_doi=10.1139/cjfr-2018-0077" />
<meta name="citation_reference" content="citation_journal_title=DNA Res.;citation_author=J. C. M. Iehisa;citation_author=R. Ohno;citation_author=T. Kimura;citation_author=H. Enoki;citation_author=S. Nishimura;citation_title=A high-density genetic map with array-based markers facilitates structural and quantitative trait locus analyses of the common wheat genome.;citation_volume=21;citation_year=2014;citation_pmid=24972598;citation_doi=10.1093/dnares/dsu020" />
<meta name="citation_reference" content="citation_journal_title=Sci. Rep.;citation_author=Y. Jin;citation_author=Y. Ma;citation_author=S. Wang;citation_author=X. G. Hu;citation_author=L. S. Huang;citation_title=Genetic evaluation of the breeding population of a valuable reforestation conifer Platycladus orientalis (Cupressaceae).;citation_pages=34821;citation_volume=6;citation_year=2016;citation_doi=10.1038/srep34821" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=W. Kai;citation_author=K. Nomura;citation_author=A. Fujiwara;citation_author=Y. Nakamura;citation_author=M. Yasuike;citation_title=A ddRAD-based genetic map and its integration with the genome assembly of Japanese eel (Anguilla japonica) provides insights into genome evolution after the teleost-specific genome duplication.;citation_pages=233;citation_volume=15;citation_year=2014;citation_pmid=24669946;citation_doi=10.1186/1471-2164-15-233" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=W. J. Kent;citation_title=BLAT: The BLAST-like alignment tool.;citation_volume=12;citation_year=2002;citation_pmid=11932250;citation_doi=10.1101/gr.229202. Article published online before March 2002" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=A. Konar;citation_author=O. Choudhury;citation_author=R. Bullis;citation_author=L. Fiedler;citation_author=J. M. Kruser;citation_title=High-quality genetic mapping with ddRADseq in the non-model tree Quercus rubra.;citation_pages=417;citation_volume=18;citation_year=2017;citation_doi=10.1186/s12864-017-3765-8" />
<meta name="citation_reference" content="citation_journal_title=Ann. Eugen.;citation_author=D. D. Kosambi;citation_title=The estimation of map distances from recombination values.;citation_pages=172-175;citation_volume=12;citation_year=1944;citation_doi=10.1111/j.1469-1809.1943.tb02321.x" />
<meta name="citation_reference" content="citation_journal_title=Sci. Rep.;citation_author=G. Li;citation_author=S. Du;citation_author=Z. Wen;citation_title=Mapping the climatic suitable habitat of oriental arborvitae (Platycladus orientalis) for introduction and cultivation at a global scale.;citation_pages=30009;citation_volume=6;citation_year=2016;citation_doi=10.1038/srep30009" />
<meta name="citation_reference" content="citation_journal_title=Acta Botanica Yunnanica;citation_author=L. Li;citation_author=P. Hsu;citation_title=Karyotype analyses in Platycladus orientalis and Fokienia hodginsii.;citation_pages=447-451;citation_volume=6;citation_year=1984" />
<meta name="citation_reference" content="citation_journal_title=Hortic. Plant J.;citation_author=J. Liu;citation_author=B. Liu;citation_author=F. Cheng;citation_author=J. Liang;citation_author=X. Wang;citation_title=A high density linkage map facilitates QTL mapping of flowering time in Brassica rapa.;citation_pages=217-223;citation_volume=2;citation_year=2016;citation_doi=10.1016/j.hpj.2016.11.003" />
<meta name="citation_reference" content="citation_title=Genetics and analysis of quantitative traits;citation_year=1998" />
<meta name="citation_reference" content="citation_journal_title=Nat. Commun.;citation_author=J. I. Meier;citation_author=D. A. Marques;citation_author=S. Mwaiko;citation_author=C. E. Wagner;citation_author=L. Excoffier;citation_title=Ancient hybridization fuels rapid cichlid fish adaptive radiations.;citation_pages=14363;citation_volume=8;citation_year=2017;citation_pmid=28186104;citation_doi=10.1038/ncomms14363" />
<meta name="citation_reference" content="citation_journal_title=Don. Tree Genet. Genome;citation_author=Y. Moriguchi;citation_author=K. Uchiyama;citation_author=S. Ueno;citation_author=T. Ujino-Ihara;citation_author=A. Matsumoto;citation_title=A high-density linkage map with 2560 markers and its application for the localization of the male-sterile genes ms3 and ms4 in Cryptomeria japonica D.;citation_pages=57;citation_volume=12;citation_year=2016;citation_doi=10.1007/s11295-016-1011-1" />
<meta name="citation_reference" content="citation_journal_title=Front. Plant Sci.;citation_author=A. N’Diaye;citation_author=J. K. Haile;citation_author=D. B. Fowler;citation_author=K. Ammar;citation_author=C. J. Pozniak;citation_title=Effect of Co-segregating markers on high-density genetic maps and prediction of map expansion using machine learning algorithms.;citation_pages=1434;citation_volume=8;citation_year=2017;citation_doi=10.3389/fpls.2017.01434" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=D. B. Neale;citation_author=P. E. McGuire;citation_author=N. C. Wheeler;citation_author=K. A. Stevens;citation_author=M. W. Crepeau;citation_title=The Douglas-fir genome sequence reveals specialization of the photosynthetic apparatus in Pinaceae.;citation_volume=7;citation_year=2017;citation_pmid=28751502;citation_doi=10.1534/g3.117.300078" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=D. B. Neale;citation_author=J. L. Wegrzyn;citation_author=K. A. Stevens;citation_author=A. V. Zimin;citation_author=D. Puiu;citation_title=DECODING THE MASSIVE GENOME OF LOBLOLLY PINE USING HAPLOID DNA AND NOVEL ASSEMBLY STRATEGIES.;citation_pages=R59;citation_volume=15;citation_year=2014;citation_pmid=24647006;citation_doi=10.1186/gb-2014-15-3-r59" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=L. G. Neves;citation_author=J. M. Davis;citation_author=W. B. Barbazuk;citation_author=M. Kirst;citation_title=A high-density gene map of loblolly pine (Pinus taeda L.) based on exome sequence capture genotyping.;citation_pages=29-37;citation_volume=4;citation_year=2014;citation_doi=10.1534/g3.113.008714" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=B. Nystedt;citation_author=N. R. Street;citation_author=A. Wetterbom;citation_author=A. Zuccolo;citation_author=Y. C. Lin;citation_title=The Norway spruce genome sequence and conifer genome evolution.;citation_pages=579-584;citation_volume=497;citation_year=2013;citation_pmid=23698360;citation_doi=10.1038/nature12211" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol. Resour.;citation_author=J. Pan;citation_author=B. Wang;citation_author=Z. Y. Pei;citation_author=W. Zhao;citation_author=J. Gao;citation_title=Optimization of the genotyping‐by‐sequencing strategy for population genomic analysis in conifers.;citation_pages=711-722;citation_volume=15;citation_year=2015;citation_doi=10.1111/1755-0998.12342" />
<meta name="citation_reference" content="citation_journal_title=Tree Genet. Genomes;citation_author=T. L. Parchman;citation_author=J. P. Jahner;citation_author=K. A. Uckele;citation_author=L. M. Galland;citation_author=A. J. Eckert;citation_title=RADseq approaches and applications for forest tree genetics.;citation_pages=39;citation_volume=14;citation_year=2018;citation_doi=10.1007/s11295-018-1251-3" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=B. K. Peterson;citation_author=J. N. Weber;citation_author=E. H. Kay;citation_author=H. S. Fisher;citation_author=H. E. Hoekstra;citation_title=Double digest RADseq: an inexpensive method for de novo SNP discovery and genotyping in model and non-model species.;citation_pages=e37135-e37135;citation_volume=7;citation_year=2012;citation_issue=5;citation_pmid=22675423;citation_doi=10.1371/journal.pone.0037135" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=T. D. Petes;citation_title=Meiotic recombination hot spots and cold spots.;citation_pages=360-369;citation_volume=2;citation_year=2001;citation_issue=5;citation_pmid=11331902;citation_doi=10.1038/35072078" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=J. Pittermann;citation_author=S. A. Stuart;citation_author=T. E. Dawson;citation_author=A. Moreau;citation_title=Cenozoic climate change shaped the evolutionary ecophysiology of the Cupressaceae conifers.;citation_volume=109;citation_year=2012;citation_pmid=22628565;citation_doi=10.1073/pnas.1114378109;citation_georef=2013095088" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol. Evol.;citation_author=P. Rastas;citation_author=F. C. Calboli;citation_author=B. Guo;citation_author=T. Shikano;citation_author=J. Merilä;citation_title=Construction of ultradense linkage maps with Lep-MAP2: stickleback F2 recombinant crosses as an example.;citation_volume=8;citation_year=2015;citation_pmid=26668116;citation_doi=10.1093/gbe/evv250" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=Y. Ren;citation_author=H. Zhao;citation_author=Q. Kou;citation_author=J. Jiang;citation_author=S. Guo;citation_title=A high resolution genetic map anchoring scaffolds of the sequenced watermelon genome.;citation_pages=e29453-e29453;citation_volume=7;citation_year=2012;citation_issue=1;citation_pmid=22247776;citation_doi=10.1371/journal.pone.0029453" />
<meta name="citation_reference" content="citation_journal_title=Tree Genet. Genomes;citation_author=S. Sakaguchi;citation_author=T. Sugino;citation_author=Y. Tsumura;citation_author=M. Ito;citation_author=M. D. Crisp;citation_title=High-throughput linkage mapping of Australian white cypress pine (Callitris glaucophylla) and map transferability to related species.;citation_pages=121;citation_volume=11;citation_year=2015;citation_doi=10.1007/s11295-015-0944-0" />
<meta name="citation_reference" content="citation_journal_title=J. Arnold Arbor.;citation_author=K. Sax;citation_author=H. J. Sax;citation_title=J ARNOLD ARBOR;citation_pages=356-375;citation_volume=14;citation_year=1933;citation_doi=10.5962/bhl.part.9959" />
<meta name="citation_reference" content="citation_journal_title=DNA Res.;citation_author=C. Shao;citation_author=Y. Niu;citation_author=P. Rastas;citation_author=Y. Liu;citation_author=Z. Xie;citation_title=Genome-wide SNP identification for the construction of a high-resolution genetic map of Japanese flounder (Paralichthys olivaceus): applications to QTL mapping of Vibrio anguillarum disease resistance and comparative genomic analysis.;citation_volume=22;citation_year=2015;citation_pmid=25762582;citation_doi=10.1093/dnares/dsv001" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=X. Sun;citation_author=D. Liu;citation_author=X. Zhang;citation_author=W. Li;citation_author=H. Liu;citation_title=SLAF-seq: an efficient method of large-scale de novo SNP discovery and genotyping using high-throughput sequencing.;citation_pages=e58700;citation_volume=8;citation_year=2013;citation_pmid=23527008;citation_doi=10.1371/journal.pone.0058700" />
<meta name="citation_reference" content="citation_journal_title=J. Beijing For. Univ.;citation_author=X. Wu;citation_title=Geographical variation of Platycladus orientalis.;citation_pages=1-16;citation_volume=3;citation_year=1986" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=Y. Yu;citation_author=Y. Ouyang;citation_author=W. Yao;citation_title=shinyCircos: an R/Shiny application for interactive creation of Circos plot.;citation_pages=1229-1231;citation_volume=34;citation_year=2017;citation_doi=10.1093/bioinformatics/btx763" />
<meta name="citation_reference" content="citation_journal_title=Atmos. Chem. Phys.;citation_author=X. Zhang;citation_author=T. Huang;citation_author=L. Zhang;citation_author=Y. Shen;citation_author=Y. Zhao;citation_title=Three-North Shelter Forest Program contribution to long-term increasing trends of biogenic isoprene emissions in northern China.;citation_pages=6949-6960;citation_volume=16;citation_year=2016;citation_doi=10.5194/acp-16-6949-2016" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=A. Zimin;citation_author=K. A. Stevens;citation_author=M. W. Crepeau;citation_author=A. Holtz-Morris;citation_author=M. Koriabine;citation_title=Sequencing and Assembly of the 22-Gb Loblolly Pine Genome;citation_pages=875-890;citation_volume=196;citation_year=2014;citation_issue=3;citation_pmid=24653210;citation_doi=10.1534/genetics.113.159715" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Genome-Wide Variant Identification and High-Density Genetic Map Construction Using RADseq for Platycladus orientalis (Cupressaceae)" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11.cover-source.jpg" />
<meta name="twitter:description" content="Platycladus orientalis is an ecologically important native conifer in Northern China and exotic species in many parts of the world; however, knowledge about the species’ genetics and genome are very limited. The availability of well-developed battery of genetic markers, with large genome coverage, is a prerequisite for the species genetic dissection of adaptive attributes and efficient selective breeding. Here, we present a genome-wide genotyping method with double-digestion restriction site associated DNA sequencing (ddRAD-seq) that is effective in generating large number of Mendelian markers for genome mapping and other genetic applications. Using 139 megagametophytes collected from a single mother tree, we assembled 397,226 loci, of which 108,683 (27.4%) were polymorphic. After stringent filtering for 1:1 segregation ratio and missing rate of &lt;20%, the remaining 23,926 loci (22% of the polymorphic loci) were ordered into 11 linkage groups (LGs) and distributed across 7,559 unique positions, with a total map length of 1,443 cM and an average spacing of 0.2 cM between adjacent unique positions. The 11 LGs correspond to the species’ 11 haploid genome chromosome number. This genetic map is among few high-density maps available for conifers to date, and represents the first genetic map for P. orientalis . The information generated serves as a solid foundation not only for marker-assisted breeding efforts, but also for comparative conifer genomic studies." />
<meta name="og-title" property="og:title" content="Genome-Wide Variant Identification and High-Density Genetic Map Construction Using RADseq for Platycladus orientalis (Cupressaceae)" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/9/11/3663" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Platycladus orientalis is an ecologically important native conifer in Northern China and exotic species in many parts of the world; however, knowledge about the species’ genetics and genome are very limited. The availability of well-developed battery of genetic markers, with large genome coverage, is a prerequisite for the species genetic dissection of adaptive attributes and efficient selective breeding. Here, we present a genome-wide genotyping method with double-digestion restriction site associated DNA sequencing (ddRAD-seq) that is effective in generating large number of Mendelian markers for genome mapping and other genetic applications. Using 139 megagametophytes collected from a single mother tree, we assembled 397,226 loci, of which 108,683 (27.4%) were polymorphic. After stringent filtering for 1:1 segregation ratio and missing rate of &lt;20%, the remaining 23,926 loci (22% of the polymorphic loci) were ordered into 11 linkage groups (LGs) and distributed across 7,559 unique positions, with a total map length of 1,443 cM and an average spacing of 0.2 cM between adjacent unique positions. The 11 LGs correspond to the species’ 11 haploid genome chromosome number. This genetic map is among few high-density maps available for conifers to date, and represents the first genetic map for P. orientalis . The information generated serves as a solid foundation not only for marker-assisted breeding efforts, but also for comparative conifer genomic studies." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11.cover-source.jpg" />
    <title>Genome-Wide Variant Identification and High-Density Genetic Map Construction Using RADseq for Platycladus orientalis (Cupressaceae) | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"9qf7kIxQN61246r6WGIg00C4X-qxr7oS-i7bNBjON3w","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"494604","apath":"\/ggg\/9\/11\/3663.atom","pisa":"ggg;9\/11\/3663","ac":{"\/ggg\/9\/11\/3663.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/9\/11\/3663.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;9\/11\/3663"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;9\/11\/3663"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/9\/11\/3663":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022Hnb4CCL5fWxcrd4ukJXiklGkIwYbh7nrLeGvB8VrtaKv7sQLMfLc3vUG\\\\\\\/TsKu50mx2sutalfaMrrxjSK\\\\\\\/dV1TDQRtaK4m4F7Q\\\\\\\/udREkdVL0N7z8cSv7iBED\\\\\\\/VEWSXHXEH48ad1CZhsicYfJaaFQmeDxhDZw3QsghfcPrffOSZHtaDJ4KBnctZdP4MLlaOM87HbnNeR8\\\\\\\/yLrFhkcGM96ZyTTazbFNHaVs9gVrwg0JzrKVEPT\\\\\\\/3LDqOb6gtui+o2QN+GJSOXc7oLja7Pf1E\\\\\\\/\\\\\\\/99iqVe0sQVgUyXElJdgdZaWK2xe4DCF9dpCSHYwLI\\\\\\\/gVtwnhLsVugQDssV4GvpcTUYcPZSsdMEMLBrDw5Ds0NkM80U49HIrHn0ckOCKa4JXmntGTGiH3WlsELTb0iU0UZQE+\\\\\\\/Uw5ZCVew9n6F1k0D\\\\\\\/uJ2cPVZi32S\\\\\\\/aATwr8AmT447RBMNQPpCwBXuShU47UtLbzo+qRx+mduf9Z0lBASTu+rOqn71K4MwIg+fdYU6JKn3w5mpQuBsHz0DzNVMn0y9nd\\\\\\\/jlNT\\\\\\\/GdMrcLwo\\\\\\\/UBxLHvGx+RSaPhF8RQmEzJss8AgcL2SQYkwCTUUfrZ3FZjU2L9EitxsxtqU6P3ItPHBckY0C3A+6dM+zFEeM4wllEfVK2nJ7yHLmQy7UaAlQpo4QmSjs3RYYfMw0kzXXk8A5Scd5XxToSeVoHEuF9HGkZGDEkJD4aLCJl0BI8AO4AJJ08ZM+T\\\\\\\/Zaia5k6RV78smPngJNE0kZzFWD7gC\\\\\\\/fuoBiegjzjZw4zYX8jP+n5j4VizMdW8i6Oc+azru+YLEoIMTreBGrOz6LdguEUzhghLUmEVSngv4C6Lmk4SAhUhu7Os0aPwdexvdGpx6NWEdsgf8Vlx9HgjH56gyPp38rFwXaqG9AEdih+eeA\\\\\\\/nWl3PSM2A\\\\\\\/KI7JUoSCriR\\\\\\\/FXvs\\\\\\\/YU0wlgQSTETDZOb82SC9HlTIwynCGD5NRRb+i8e2tmmOR1paJzP7rQHwZNZ2gS7Ig+oJ\\\\\\\/R1pdlpujeR7H\\\\\\\/2NPCU2CbHdqv0UIK2CIXRxuC3bkvMhmmccf8ookpBgy3YCwBD1aamnza8\\\\\\\/a8QHRDg\\\\\\\/vHXZBYHAdWkaPCM4k420q2R2uDIinVg6OKEncvtLZ1q3gE11l+7vw1M2e\\\\\\\/7KacrJcb2F5YxH9xhL9VSszuKYDbdWB5KKRQNabJvjzlNUVcpNiUdkJmRwGXQ31DLUZcdAn6AWgnkIQq83Za+Cg6AV6a9qI9sQAK23yFCkpEZiM1O7QdhYlCRCYSQXwgWJBZfDsx4eW\\\u0022,\\\u0022iv\\\u0022:\\\u0022D5uM7FBuCyRLPxfBHRBhrQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00223c4a11ac941828fa660d10b40246ae9239b14ce2f8222bfb8a4478b43e53024a\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022EBLQAg76uZavgZdwM6ifJ8xRufDXvzCWx5fRIyiIn\\\\\\\/v1x6VeLhxOK0qEjJaTIcFYccaPWFbh7u\\\\\\\/1ogJT6jcF222bedSgW3gQ+MM8uCL2xIbdbtEMxbGCZ8U57PTdb40mZtbHq1SJXW8Egh9chhNzSPE7FuXQTTZswI6fIZKjr5Ufhvc84rkOQ0DXdv69d64WrBRZj75VsA02w78Y3ovJgVFP7N3IeyN1A\\\\\\\/NO5Vx6SXzp7CQFbC8XkdQ1KPh79R36pY35wX3NHPqsQ6Dq0oq0a1JXGeje5NhgXcm7Euz1ZNTvvza5POwEMja0Q903xuKB4TiJrjxFxqVJgFz+yf0kR2DzP6nI79Qr0qHeaiFR3WQyZfPPwPAAeb2oa26vvhkhsZrS85enwE3Im2AZ\\\\\\\/K0aWwpY7uOq8SjZIEcIEa9VFmK1QFZCI9PdZmQQsSgixtvA4Vv8BJcXzOmeE3lyCJwBhdSp\\\\\\\/9jOdGOZ2P9+tYtBY\\\\\\\/rSs+cXQCORON2KXfi+haWi5bXhpqcy\\\\\\\/g66smtsNIj2dIiNdpLMQePWU3cAzfGsu4uWeaAqvtM79avyk+RvmGz+32azocWpmBA+J1l0TOPa+HkyaPBef15\\\\\\\/IR5aAmJ4ITnJeuf7JD2r9WaxC+fNmJ6R2cCgutOf7Cj1rJO2TVsPhD9ekRZBARE9GMRqLNMGlweyTcTmerpolH0rRmRFCtgwZ8mnT4IZW7R+ehAYYE0H+z\\\\\\\/9Wq7FYIMdhSftSYcNVaVubcTuqZaU4NUmwrEZuq1a3fnc3DGm2xof7v9dOdJ3QIGXqQJhE1UMvryD1IqBJa2zSeuhAKVBOPq+XrYRLh+paOrEavJaOYUMxKuDogPQVmEYD6BPMqX7ANrASF5ShxurgQ\\\\\\\/sfjqJYxzFGKl0ck5thZUIE0U6i3N02gNyeMG9NSdlEAfQeaVJXDuwGdCqcDDsjjSvaiP7UcrgkiwVi6t7+DJkNa1qYBcvWLf9+NNfQmMeg2q24kObyckkCrg8zlezMEJioGatqq20y8vb8iE3KpBaKenuXnGjDtV08fxa+VM4GX0fqAjbwljWJqqSq\\\\\\\/5Gebr37P6D8tmHj1rDZHcDwssnxqrHtUHHoaRb6XJyWRq9S\\\\\\\/5ouSDAM6imMYJb\\\\\\\/wbC4w0DTwI7HPSP3NJcDGz9InI9lDBLevula2RY2+rAg5loW2ILv7a93pquy4ZKwifCN2RYOFHnTEck9Bw84Y5zP3GHwjIOZJlLCO5FRwgRUJhCVXcMgl3JpsPHK\\\\\\\/hq0naVJupUsYzgBIbneSUuyMkcmOVN8fb5T2HPC0+oaNPgvZ6DBRM165qH+Qabjlevlg+13xbadzbFwJ2AJ+dtl\\\\\\\/1BOUpOwL+W9RB6bsp17ZyRGubXByq2GYcpZp7jELTe9OTK7nU7T714tb0cMPWBIn+vZ4LggVfPf+zdeggRrXBnbdHggDFtZj4fGrZaVDbT0Hicx9Oyl+T+joign2zNBW8Q0+oLGXyO37\\\\\\\/Lp5Frz0qG2dlwuD7TXJtFWbp4ItxZBsLyp\\\\\\\/mtNEC204taK8WBPBfcgA\\\\\\\/2gvkJLbecJy\\\\\\\/MPmQ0vxo\\\\\\\/W6Zkf5PnkWCyrXpTOxSaxOBZfzK2kLAL5kG32ceV6OSptOGsFAFPCZ3r4\\\\\\\/aRHQ1RPTPlyvqlToQWnNuGHXV5GxiJpUN+LjpOp0e1i6braEpUEMJkD9tBWC6U1FScQTNf\\\\\\\/tAhQuyKWe5FzdI6oQWf\\\\\\\/Ork7Md1F7+Q\\\\\\\/CkS+EsjbvjQ7gUmPXcahcXNi020N6cPTlpHyfsMibDAbhiv+aEJ6Fz7EetvUP25y9Fmsxytc5YM6pwEJFrZyXET8AnhrEZwAm7hsXi7X8ecga6DGXLarhObluTbSWlysLYrBE6wbol09zE680z6YKHCyP2I194zm12d3FCfKW2Afbosz6GVZcU566cFO4oUd0fAEelrQBsIe708JUcowxbmFGGQG8HaDxtfMniepL5ow1BHG27kiVQ7qmnAMiniNuT9c7hYu5T8QGd\\\\\\\/g0Gl\\\\\\\/fuUZYnhALReaCCD60zyhGs25GFAasnzIZ1Avf0ziy0fH6etaY+E5RIaIojNUM3p2DyQNJL9hEzArtB2MSf3ePPbPyira08P91TN5Az3Se2FBPvk1VRi4ypH7uD6fJt8ahYUmz4sYCbgCpEJZWGe1H51UJgEyp4I7FHqWHhieOcLlN09+kIie8sYdYgWk3tH2VU9dPBSnn+7WzwbF3Ei61JfNawczulzFF7r38Ec\\\\\\\/20QD5q0+XXsyR11ddgn8Brq6XDnvx5UVr7ISvLqVfr\\\\\\\/NSvErAGfncCWhtmRqbZvOVyT6qvY7ws2aE8W8GRxIJw\\\\\\\/2WK8ko\\\\\\\/P2nYIz0v75nRxriTcfXTycUv2D1gGtsTarykyxomzLgjc\\\\\\\/FIio6X3F\\\\\\\/IWtURUN1Y+pKHLPfFrnxPnx6cpAAUlPZEzZ8x8yOy\\\\\\\/k3lXm2cffcuYqDMy3L9ia4QtmTaE6noLSPJj+MP6q3JiXjmXeo7QW+028N4V\\\\\\\/30iLxCUl55HOzt+1birFMO7JkpLomSYswOpuHM\\\\\\\/fibxYupzw7v\\\\\\\/Mv2PM5NVu46XxalOIKF15XimcagAxOGI08PfKxEkiuE9W1KB7p3o4WYEcdB6SiHQlRYYF5xhXC7vLywj+\\\\\\\/bE0kCryMtqZ4GhUU7RYTnR9dQSrqvj3dgUmRjX+HTw4U1YCAfb9ToPNRr4Wcx3X5K\\\\\\\/pp8YdSas7oPzL6JKBlBUwr6q6259ZKJ+CacX2SKgbc8AYrgoi6vCCsg055eAm+wqLf6dEgL1fMybOsfLnLyP2w9G785AE2I8pseCLqC+rksyDpldOVyv+kkRxTSAWkAKFOQf9rjjcBuCq02Wk6xr\\\\\\\/7scyZAVpfXBRPpV9YYPps3KBwqF8DNGP9QY+wNN8oxRvTIzx0Ib1XR5+JAO9g6CZaaN76etQ\\\\\\\/jitVOxSzVj\\\\\\\/u1zQVwMuxGJ25+JJ+58fChB1AHIBi+wC9nzfICqhcmZ2e4LbVi9z887yQV0ToENSG1GE5kVpOjxRpUoFhkANKtYw2b5CJMX06+nmxK0Cf4MbJGxYg2gNstIa0y\\\\\\\/32ZOn99nNbS3ihGMADopw2Sk\\\\\\\/oNvT8GcpxUXy12oVzJL5jnm10+fnXWW\\\\\\\/B6njlJM85s1QzMT4rp5uJkgXitX8jqMGYfTgMq3LB2xRujKdDr1WA9Z0b9WwAMVCWaDLylrdEJSAS\\\\\\\/fXgu9i6ONdibkTq31NOMuMsP4YiAeRh1pCxKzGJPauU2aNKjtnSFB5lF2nVAkcUsXOp\\\\\\\/HVpaJdotXz3IEMzCrBM4j4+a0XAIw6WB0htCtW+Exl8LpldBY9Q7r0H3TCaldGjjFsgB68xDQ73mDUgN\\\\\\\/iQ52P5uBw1IKKJi1Fr41QBYdlLg470GKNLKuypQi2FDcs4COVs1FCKRMnmnFKnEC\\\\\\\/3+GsqbASg619lXXIqlI06+4VpHTft2v\\\\\\\/dFGpWE\\\\\\\/z5zbEN7JnBFOlrbUcCuBVJgQtwTsKw4YDZ\\\\\\\/8n6FACzmMOfrRYMzmzKK89G15Qtkz5o3OAp2wblvmrpbwpyVW+GBT6rm1SjgViZG\\\\\\\/iMs+2ZWlS4npTnCH4xIObgN9+HkJTaEoLOY5tkWxrKN2ZciNxPW4T16I4aQm0fu61sjXvKRYKqm8QpXXvz1S1TJhdwP9OmXb1ESlYzS67AgYSOwc6\\\\\\\/v5tAY63Bvdyi28hPoQcTadzHH5uqxQF2Aw71gDOgkdTODerx6wfJAxpwSYUXjIoP3NjuKWovhwlRq\\\\\\\/Lc1yct1nGKkvRqveO\\\\\\\/56kqAhxRhwBNgBsPIz1Jw5PJU4mz6sNCunRQPkLJijRmswnXOJ0\\\\\\\/0A9yAG+uT6jYMIzhIqvafRcEodlsCt+\\\\\\\/5gqarirNgZ\\\\\\\/54F1ufJaqGK\\\\\\\/q5A8YMbWRcXM99arSyR3e6Dr0RBifM+uUP91bpbhztEJa4cdWTSDB+0BXDQd3fS9q0dsOEvHnnqX8ESFSrcl048qcs02ppL8LHs18nrp4sVKKqFfc7NYOFka5LsxonyhQGzRoXmc+Nh7+tPqVwGvVLbmdVJ3\\\\\\\/7rGXt4NJyVa22J+Tw8SakbwpYp\\\\\\\/\\\\\\\/AzjAKeEe8tyAJpYVmuuVl0gfcc7XuxyocRje2ImvuW69gygh2WngT3ykIhj2N1Yoe9a7LicFMggeRHEwhbjCrTHlQkwkVotwXInC7YPIusBQ89CIwT6eJkEfAyNNAV1sj1zScAWpkz9F9Qc2fcdqUJOpmpyYyalP2\\\\\\\/c87DP4aOCTvPi0e9eP90QO3XYHz84hTo9gY5H0BjgqrYexgIX4oXdJmjN07rhQxdYCzxd4sM4IHLaEHux8yPL6XGlbmWVnbDFUjNGOj0xAiA6Rlwx0MSOQH8Bdyd\\\\\\\/E4p\\\\\\\/w1lfd2FC76plsAe\\\\\\\/Sm2H5ouYyPiLSumoPlm2s1\\\\\\\/blnFoGL3fWatgssLI4GMBNK8ukqr2kaOt5K3KgonDBwo6ney\\\\\\\/7SZVLkBpMvHI9QRl2REWHPvYRCPgs35+YIm2\\\\\\\/HaFKdkz8EdyC102t+4KWF3\\\\\\\/m3Jqk7q2RfPKeUMoLI0dkWRy+LNBT5WvC\\\\\\\/co8YK5uofmQ7ECyTLUClAK6yukJcqUc7yoVknZzouZvhygCVK1pzhjo3C7HA4HB9v0cKFiiz3g8E9kNj7XIyTHUi5JZQ9jhGvTGEpx3HY+unk+G6sUL9+US60ng5xP+R0lUJZLsMiUc0T5s7+tylJcX2h\\\\\\\/iyp\\\\\\\/3GBReytI3QXm4W+\\\\\\\/qLOB\\\\\\\/MW2UrTqVkUdqaxmZp113OYKVz8oTKPOm1k+raaU1nRGFmGZB7lUOYY5wykaHmvx5N+Vq8Kj8iM4pt+OxmF28qBgotPjjzwZyupCCvtzKO6pKOIwS2itrNxDphfElpJw3KN+cSC1u1aJ4hZ+xiyBJktH6MB1+ZfvTs32W13bX3IXdmRe\\\\\\\/ePoUQKf+rnzDLjCh+wgDG5YWy0PCa4N6BYX1fOe1DgIldwye9kIodIjG1PgDOHUbDsZcWVOdS5NjEPPp+B5NxomZObbReV3wi\\\\\\\/v7rK1LFAv8EOoZxW+4RPJxP\\\\\\\/7vZm+aCWGm\\\\\\\/84zerbcm7L73olIPT2ul\\\\\\\/cNDbjwzLqhVncfyKWmh1+\\\\\\\/lp4dFKS766zxgb3OSZBDdfjSq2WbnoD269y1SpZv5EPRMprlcs1uTx3iHweYNeL+3pAznuPNyDarglZFiY877ViZV1DhS0sTEvNR26fWdgY0CWHqRlRPU6iu996GRWrM3wDI1zw3XlkkIY2+egrr38pLlnPWmJlXjEFWTj3aeYniHjIOLRLGrOEawP7hcyvh4RBPnUzFaL04wCIjYMUINzEhwyC0+ivWhOPNTnVBpiZttS3VCPuCeqcnHZtaekS6GBoDsyTNH1jKNLidbUImw4yeLmtIn7YxC\\\\\\\/rfTY7v\\\\\\\/6mAG6v867V13B8mhEw3NEnJeFO19wx9eCZhMyauOJ69vAazqQTpE0ax\\\\\\\/Ooci\\\\\\\/dmsIEfM5Pxowe9etj3k3HoUaaXzRtCKHEN7n8o+UJYIRi8+MK\\\\\\\/1O6e1e1UlEHjDRgBOTS242jOfaVjw9wTlGDDRPOWsA1B4Z8eKIlcmtBfY7+d1FTI78lWk6p0476Sc\\\\\\\/J7EtgdLV4TCYbGnbU8Y9hqNa75mwZCY7ufPeHvrsvBUunxdU1cPFTGh5QKtO0UcGjwogQN8OHW\\\\\\\/FMJEcpglcYTEgwnPKJ6coA9MiZ72ixWxo2tWleTlg81HHc1M\\\\\\\/e6kGd5Zoy0gcFGp7ARi3ZnLbkWj2jg7tXMuVXVIb57yo5tzuzhkbu8KKDNeoG834hwwWPMYor2lP+bGuIoh0BBa3pfrkSLVOsABjdGDXkYHJUar5UK6jqs47mdHJ1\\\\\\\/SwetZfCj99bvJFD\\\\\\\/79Z4PB2RCLuqVut6eFBpYuT0JaNO0YwX7QOfrTQZ7\\\\\\\/0gyqv\\\\\\\/qUG7hx2YpgSDqPhVUgHXSZQdJoWkBFoTAMdRhIltmfEgONgiAQhFgdxr4zG\\\\\\\/75R8f0BIv701IVb1jsaZ9HTIOlgjcwrnOUIA+3wCckmtOyHGXWAaMok559qGe6UG2mPk5KGQeVluiGwHlVYx7tQm42S\\\\\\\/bqx\\\\\\\/ae89TgUa6Y9q3rudIgVgb04V5q4zavDxCTljYhFZMZl1gBhESAR+ahaF4M8uzNXToOR162B2fv5XbPiJGRAehPdLLNsn4cCmJ4PscxKd\\\\\\\/4Ns9ilR7s35PrvDA3SXbgrjaeGatMpWbzHhX7Nri6ohV1NB6hRud04xizQAUCsv0qemYoWciCvJ+a+Unl9YMkDBJZ0OlqGxLpRVu+W8N90nROMIGIBeHlbSbbA8q2gAvksVLxENJuUHqORJLMhrmAthOzlrneEd4\\\\\\\/K34SPqhRih4IesBQzfmcz5xyK4imxpiDhppsSmQoh+OFp\\\\\\\/lZFIpf2XnpKm+AO8D\\\\\\\/LPSJvwRF\\\\\\\/11wU1CaAEHXrbjjOInucLdLB5vffrt7riY9CRUU9swho5L+EEoDa8FVaow9nj2KA572MqvfTNBLMa\\\\\\\/PnotImflwqHPTq9pmhHL9IlLvuwSer7FdSDgbcu2ecHxxUYu56aqBKfuGODve1jqGwPMc4ZF5rCt+WvIVXMlUH1J7Q+AW3yEHmAc74BdCtoJGxpACrXimjf1aIwF16m92h7J4cingAdSiaUj15S1I1+HwlqhZWacvapXEaz9lD58WgMLlfqly4XIku9m1afSk6gBQZB8ssn45fCBmzdqv6obysK6FnCbDZxXvJ1Mu2JVLgpt7eqw6g9B6wP7BqBJKqDK3YBcBnUxH8AUmGR5IGb4j553vZd9mPdasZbI+YiNb1pAwBIB5qmBS0NYYUA5XDOg05KeCq0D9oS13FdvYTbcGp79TcUMyLQXV01KHuV7NKKQLmpSavsYN91zDUkctdcnwJahW0mvWG\\\\\\\/9FvLcFwdOStb1WbSzzYkFcyMr2PPx8bG96QyS63OR785tNBzyng8+XMXuHMVtB7QeTJoxMULzPG8klN4Bk2XJyHXE\\\\\\\/lyrbKubaOP9YdQSOifhq6GH5++iEGlyX5fKFx3YLFwHyR7acdUNAymJ3A\\\\\\\/jxIW7CZNTQFeA56pqjfbgs9A8rkJbyZdvKir8YMcLEkd8eMIz5pFDGz7Gj9Y7iY8QAR6BmU1v8iv+4ax8YJKpVKKqliqAjmYc\\\\\\\/RXrha7sGf1frcoSxgP5nDdP9f73D71brVx53LLrxjuMKVpnsMuyb\\\\\\\/0Ue9oGSQkoJ127N+OzeSXiC9bkMe0bRtEffh702siItgZy9U1+96eiLtSls4Nuf0HZ\\\\\\\/t94avca\\\\\\\/CY3oGsDQWib2f1Tv4QpssfdfaBHl4uMlPS3DKA3OmQ1YYPEPA3+LFE+ItsreH6U1wtHIO61B8Weg3RnKn5BVO3Y2sAwfnQC3i5i2ZzkAZPxBsDRiJWsZ27a5nW5uhCVSRkIQRyvyYHNdR67+lqbBh5TOQir3oLSzubrKCtDTD6fv6Tjf7EwsFZ3vKIWetK5yDI20ub8t1JcksHSFO\\\\\\\/M7OfkV0bt49pwgCJ2FRaQGJtPCvH9Dz4OwuMk8bSJzMQw\\\\\\\/ZTY9J+lvTWO3QCiLekX9Om94hz7wLn6HH9dSE3ahKn0XYtGIGGKdEq\\\\\\\/EZ23tqqEq15YV0o1piHKQatvTAUE6AuydTDr\\\\\\\/A4bC2kGgVr4o0LKW7znPpHDFc9rQy3nijXrD4uPNgUUmkRMZ85aKP2\\\\\\\/vBm6490gXkHWkI689ac4JpsEqZeLAr5Y4LQIF7\\\\\\\/SWp9SkHjX1PQET7kKUia7SEoGgAPK4ZSc8Y43\\\\\\\/Igcg\\\\\\\/cBDfJ0i\\\\\\\/pWVmN7BYrCnSruFK\\\\\\\/Ao5hQ60bvSe86BDJMTKXZoxiGOtwcMAw0zWQVmBnpRDpviCXFtuaQEU8qHGfJBXN3B5q7vlCf6fDj\\\\\\\/ugvyO8dCX1O9YXbFxoCZof1YZ0q99YlMbto\\\\\\\/GXORhlfTUFOwXJ+5lPkGohoCmPOkyiSVEW\\\\\\\/FsJF+clrGVh4acCsCNoDOD0KXE\\\\\\\/PstlePxiZgqwV8jEQRJZ8OjdhyiepVH0WgXJzNoKxm2JD6Ka\\\\\\\/2P+0ofj0mr8HNhn4YehhBG4pDWMeom7iVuxUHZLGNQNZsr3e0JV9m76e12GorlZ9XsfDdSu2GRX8THdbHRCXSlKEez7wQg9f+3Gb+hKQTVg2VYmLHP9A+KD+47PeS59DCAigDwB7PgjQwDZCNHLJoaI\\\\\\\/wXQQ4yYn1\\\\\\\/N\\\\\\\/EdioLuJcbeQ43WPej8aDKlNDnpENwKKB3prF15ASGMWUzyTpt3CW9PryjV9TVg7vNi1chVBZ56ejihwa\\\\\\\/zj63dhajqOrDpf3H7OaROskCw+Zb14EvS039y146S58QKKwTkXjUmEN3eokvwNm0TwozR716LKnR\\\\\\\/aKZ8g9EotwO+L4tlmLiqnmkBRKoUdO9QgmJ2MZbOGYiG431te7Bv5bvRH4F8+iqPuvKqEfSJ1YU6szO6goAqzVDBRlC+eO5Wmpdw5LvEQT0jdRCZ+GL9HdIwyhZyhp5N\\\\\\\/26JWLe9ZZfXMMqRAxUpiRjesMdRJP1yjNEfb9d+QwyYNS1G\\\\\\\/+9CF3jtAOBJ4lUJrWKTOUM4xZ4Qy+0HDGjJMlo+EbCMNhTkCwmBw\\\\\\\/hTzlRHAI2XTBSOy6MSojVe4G4A\\\\\\\/\\\\\\\/hTe4GlxI02uSHk1ilC0nyoEC3rmpQgYWdBdpmpRbobxX7U5N1+xxHvA5G0Xzg29MU2\\\\\\\/l3UCwnkJZbZcluOHdyA5eeN5BNQn9q4LB6DED2WVkzXR0vwYcgGiGX5XSZoDXTx2SWIvp5tsvp6zIXtYLlxJuoX\\\\\\\/B3ML+qeh3nQX+xmDZ45jxeBVm3vYZRAuyZGzl\\\\\\\/UEtJDE2jKYwmwEVG4f1n66DblChV86wsE2kzWEtiqVqgmZuBWACuRlPXhmZLQJJXCYWb9RNzS5n7RC2PmsBVcrmZxhiOc95+MuDOIKfTKZJjUgGMapYpvi3gNsH13BEBMli4l4+0Ba8vzZECJwNchYZ5Pazn0PDOu3pp2xlerblEzmtKACGz3fXhmkjjJY9pmcL8MJn1dLt5YDOWxytT7MWEo4XX5OBZ7ppnhgtfYYFkvQ74iE111kqnGLWBMQxgEpqvdg8IUXgOBi925B8hejY9C3nkku\\\\\\\/dGYaPe3qhRys1C0JoIfhWU1QlfftJfp27YIi30OW5PgJxBpBFCM9UcmeD1rVwob2DSM0lC7R41Qb8IGTIAP31rdcYfpuEDWy00WCT1cSalh4OI6KBHBH4QeCXndsDEKXdqrkp+iXg7o\\\\\\\/QbIY+UHmCK8ijauFRMYVg7ynmHFNHnf\\\\\\\/wTgRqA7hCRfv5EuL419EqBqJyP9yxlGRxenzGhOzPVpKYALDJ6mdjP+wI1DsTw+flkipvDAnx45ypwyd2gjSrt4aitNGuNfOiTRkexIhA1gIU2zBXk4e46YWdX0ZHhtHL2Fg1t3ZpO08hYYMtwB75ekGTg579u9s16anTxQiyFCmAwNEk9HZNWFUTgixspq4LLl7MFWcHUSlVVBd3erBkBrs1TM97B0QPocwlpqvmBUjHQeNDI2t+ec2CaT1WfK0J0ropc22U\\\\\\\/TIpqQBRv6Szr2N4jNLnBBb6MmYpFP8wA87TnjtyhTvv1V7s+NvuNObSktppmvBqWygEvz6KO6qlZUEMrZzbYw8lm4SP6RE1KgTW+u+YrFbHuHXMTK\\\\\\\/X+OTEgJ5RviLwu2NOdDgTHXl3ZzkyS9bCZ+i+hDKuDn+QxWHaiShGtNvSNhqf6RbXIqoJhX33Zns6YC+bm+1eEqKIImjQkk0Yu41Ix\\\\\\\/+OgHteD3Vcc51gj+UuvBMeZfGZ+KZaiP1wRLYtcLh\\\\\\\/p2Nkx405C36Mvi4YytvDQHtx9\\\\\\\/YRz4Vn0+Na+sroN2WujmBa4GPadC874nOZrYSf3vJjqOmziZjLiM0\\\\\\\/X+Y8PpUzqwPi5Pc6b7bmuiVKItz1nRouGXOnfviCBPEcNLmik2PG6Koo2wTmd8XSfR36OkbaSh1+7PaUo2n8eeDNfoRM\\\\\\\/Cdc8qkgqNCXN87hwTbu2cLnXSLvbpCxQnhNPXYr5cT458PsJ9CDdcy+te3+W42HNHyn\\\\\\\/RygC+wkE80h2XZKskx1rKLslHJWPyvHPsxhrSYK8DbDADocX24xJZNCxZnU20LUFW7puquYvYXBDc0g43ulqg\\\\\\\/mkJv9WW8eVARpke9JLKKDhit8ijg6n5r4X8dzONWCk1Z+SEDdVziIw\\\\\\\/1stMn5Mn4\\\\\\\/vYgr10IrpeePKyWYTgeRafVN8N3yZkBMrkzMs5fpCg597Xm4nLXR2ndg0+GIMMzrGHNUqUcb3YCzVKj2j1LhuV6kYIrLfzbwzz5bVIblMvTZCmGnGq+yMwVP27Nnl41dSzRh02j2MeHRxcmhrMEfiUPNai2wQXMYL1OU3kiOCPCieFI5oUogqIMDDH5uy6bk4vXrMo\\\\\\\/ptsYu8BLIlAS2XfzrgBVmjLn54owvBLODKaugB1LCnhOGIIc9hsHQfPO3cA3n4DagLy4hrLodsZPjF+GKZkbtm8LgcQ8amEI34ghfWjsRHTXAIIgKkefW6xQkyhmri5vPnZDl6pIfWqYXoqVdcwPY4gsugHoBBDwvRosuNAYb+vWAoWeMnAqeYxLOPKwCL07Qqw1Fs2fs+KgmuWBLCWdD5x3PK749\\\\\\\/pS0muzVmlcFhq+Uy5CcewRROw+UkvBG0XNvgQRLgTvT1q4TfBtW5b10Hk6YqDN7nuf2HnWk1TE3F\\\\\\\/OSqyZk9ExCuRcyNe5E3Y2KzWazBOjL2Wde\\\\\\\/yvg5\\\\\\\/X76y6kY\\\\\\\/F4vGdQpfJ43BiSzYWSZvFabrOwGiVyggqDLpOtabpzHFzgohEm\\\\\\\/k2y1KOIIPsk3BwJrxIRZWM+Sm4\\\\\\\/Gr6hNXirhjLnplX0be8BGipGzGFOa17yPpWsViqNmq5xRUM+c\\\\\\\/y42\\\\\\\/4lryMaSKXUdnbAEq0JCqlmXpctfyWBY0azcMBDIcRHqHHsNuNzTxY5iJvAVEYUwJcvrvjP4twAtXuoD1xl\\\\\\\/8gzD7vxZmTrCnZFvTwDHqtC5Tb1nucrUFyxK261\\\\\\\/m8qz0WH24CkKnJxCQp1+5iASU9u7ksZVQIKG192HQ220NegcCMnkdNlY\\\\\\\/dPBuhfsJiEeAqdHr6vy5VptI5TzxCHdBz9PMIHnlqO2sRkffkw\\\\\\\/sJ5dx30YH5yReDjQ5UwiIRH2cOQyKdSwHY+sdp5FGu90Nvp2VEnAtRYTxAx+Er30b2QccretuGXVtiVJAikvyeIzHG2Ln9gwqcw524fFkHF6eEzHGttSYkRX26wTbb08QNTZ1kVa1WS2rFSSKPxZhb4cFkU1wvUAgLXjVnYfax56scMnddUBqbj4zg59gDC59gCYBn7nD1kvrtfTKXle9FUMzfnMtcLNMM6znR2O0x0KO8X\\\\\\\/hUTAHBQ\\\\\\\/thNIBLH6W5Ph7WJz13e43A\\\\\\\/XC3QTvt3ubTFCFVVfO8rpKPFU\\\\\\\/MCDo8tsZaRdPkV6gYNizrIjujm40mXLxOmWxsGJQG7R4PtfaKULq9345aRP5zZ\\\\\\\/7upgLJQZoNssh8H22OofJj13+plGl2EUpwm7qIiNhvUthxWaKG5P5olmr7HfQnQ+5bX3Ooh8czqc8EIBy7mz8OJwi3njLVRwDsEAfxHbBKfzt1ygKtyjk6bZxSlaDbjJGSkSnLl9MrUMGPdIXB40\\\\\\\/MBW55kMLsZJTP4mHD53ZGYLAQkILMg2tplkr5CpEigyty9rpi\\\\\\\/xQ864HRZKp2xVY+estAAgR715MleR1MSU5HIiUrRImV4\\\\\\\/Id2leZrfRmY8SaYRwFN9MIWrKw4cxz8\\\\\\\/3Ry9E+a02iw2+hM21EfozAvGHDD3bR2j5i798iuf2wsbAULDdP3gewJTtNjv9\\\\\\\/2NaDbnBb6bAzg6UVXw2wXVa0i11uy0DBIWoPUu0xgU3dQl0EWhPssitTbGgLu1eTbA6SM1dl85Ou8OkX\\\\\\\/JL0EQCnfBh8NhHrs3wTbbTLJwan8d44PSUx8NVcWChPWftnHbG3wpXg2EE0774KbcyEzXyLvowby\\\\\\\/1DMEk8+97BI+8xlg81cThfE+6xlHSSKM4cWGfLrK8A249jhoNWgMEO+vu5IvBuFM9gDZXQan\\\\\\\/wkyeLSzO7EdmeY8HXDv2c\\\\\\\/TI2gpD\\\\\\\/0IFSIzSnwe57Dj9\\\\\\\/nJDolr1JNk4rebWAuEepaodMyumTbhptsv\\\\\\\/\\\\\\\/eXmftpbp2xCexDPdp0F\\\\\\\/5lJpq2zX9uNejtU2x0fDEA+ssfmyUf\\\\\\\/TypDlnoPYcSN9HpVW7CXRHH8lLzRtvSKwuC9qtLQfuiUZLn6dbt0UinNtFs3oL0ws+wlA6xABHAxHQ6j7jDaVqA5lqQ0bjIWW014dP7BNc6QMLaWGzqMTCgvqKuHNhKDwTIA9QU0zfG6SJQM6IqwbLSVWEK2ZTH8lW9GHdmye59gU8etosj5GytMRgWDJ3Q4QWoBFGQFbv+Vb2Z3ViAncw1E+7JpMXIB5tgsDcUemmRN2gSk26tnOSV3NE387HQCHEP0osLuizzhc6UuDq0qk2zARV2v2qqccFQZNgX1BXcQ3UNlWuevR3zm3zFB7gKiBY9OwKLrVZ2HVNABVpt6xCLoqgeszF6AO+VShgTkSd0+tRqj4nIXTuv5JqRVGdVnypLNXgTk+NfrXslRN6RUBCuXtQvI+ePrAaweonw+ihJgA8\\\\\\\/abfeoBYXvdUvc0ZDt4maEcIZTWYwYXugd0A4ooUxcqUqJ\\\\\\\/SKU+oZaAUJmpNiwToNVS7e\\\\\\\/i4kVojy2U3PTD\\\\\\\/AxFqHfXlH8XGRwUz5Lg0iaracDQxCQu+ImbAXAIzqRk97ZeUS7En3s7bXWjyluPKh3X+RwpiFB+SZ+7Ize4AeYdEUQKU8Kh7Sr3c71dUcmVKzCqLEPN75nkyML++LI0TLgKBZOQrqc6sOlTPnJtW3Snsc+ioKDQXhy4YwNOcRCMhflwuUujM8D7P4bMzI20wJ79vKQwo\\\\\\\/oESkgfjaPVsyEODDs9JfYWZgDb\\\\\\\/k717tB8ezc\\\\\\\/39QyHbbP4bZhWnmUbOeE8Ul2waOOHIER17YqWXr4EXTFQtBj1F8sYdBFfffZk65422xY5ukQucjV8+vCLWCKNWj33KNW6nRE8WfshwWNgket+sOgSw68KYUeUh1xcrFqlb5v+prksJpe41HVZ5YuvaSOumZFiGfoHoKNMYB5uADBFYsmcsG6oAZsV6hBT+uakfJyHkXVs2OWUiXm0PMOfgz2qb3I5tb07YqI73+\\\\\\\/oBIYRaDTrl1aT5kjxxEY7ZPuGbiwdcMuuCisYs9yRUFObIPTg1V3Z+5AGRgIh6lokJupGRGFV+waNRPQ2DrgCjfwROg+pVHn4PoZvorrSGk01nhz5gVqdN5hCBFzQee09+6zcHUslJZr1p+vrKo1maClbDAJ\\\\\\\/dKfylQ6M28VYxaa1QJBAAwPMH4uGqf\\\\\\\/JlQXsSwrslGA1FGkwxAdom+m1L9kMM26DX7YonKj4fLOTvhbC7H7TVFsA19ATB7Ox+qOn3M7qaf+WYy50MZXtD5XsVc8VTRr250YH3G4EwdQUc45+oBVWcHJwpSl7uH471IxcSbWiHmflWH9FN1IhcAFTzsg0EigtmSjtlv0CuTNZflQpGZEdn5tQzEFEPFy7owMLddoZFICNUP4ym4a9RKens+UWn8RSn5VgQI+MaUo7hFYl4P\\\\\\\/9Gdy0sWB9OLww4cNAvLgpl+MJjv2sdCKodmZNfZlQp1\\\\\\\/INg4lKgLuyETyTzEwNOQPmOBDppWul1R8rdyfzwOyKhhST1SVhD60rP+NwXMfrHRVLkAq+YCYUiKm7QQNvNMDvP7f+XEBhx1QRUoyESajUotb\\\\\\\/RB76FRCdZhV4m\\\\\\\/0M68uNVQOyOtaFkg\\\\\\\/PRkQs23wQmEYagM\\\\\\\/uU3Y0cyuP16jTuP88WCVO4gNexcvtHEYtslW0uTPVVXpZN2Ao6VoTp0xsKdIYG6jiLgWiRACUwUR1dhJD7llKeE+SY+6k2eFVwZYk0uVc+MbL2BTGUoe94NyAnAGIK\\\\\\\/1xx6UtxHjzbnX5Agn93\\\\\\\/zwDk9FXCHH8DCqpZMq7zfeRDZUnbAlJ6bi5PWWo1U36dETcnTd1PXld68GOQT\\\\\\\/IXDROyLX6JQvYYN\\\\\\\/NZpbmhkUqt9p8GFQOMgk0HBIEAX8tAgbNKIddU2V0IHd+rbc3Y4s\\\\\\\/J8Y8bfGRFlz7Rt4iC26siCgMhaRPq3o6Laa1e4RP2yzRezys7j3EFlOwi6rEJ376se60RJO3RlPmCUpqPQ2lm+ufE1uDzBO6tSHvn2ufJIHbYhy0hksdUrtK3hx2p9G2jnRkKqfR5iVJ7RBR4Tsi8WugzxjJ+itPMFgFBvDIWz1N\\\\\\\/7VI\\\\\\\/zMl+YY3TUo1c6LJ3W1\\\\\\\/HaGNIIWB3tNzxwUIrFhrMGvxaqJH\\\\\\\/qd1NdBK8XmTc8RfVBurHN5UWaCEDMacgLQEwn\\\\\\\/SzB8pIUMprxTaZ5RuJoKxhRNqsVGlhslx5\\\\\\\/yj47P8rSfQ7CsWHIcSwC+CItHvPzFvgRaCuknoiFX4VZ75H1Tf7s2NXOhDZ4cmTJzkBFBwNqEAGE8zxk5OmVxj\\\\\\\/C1zeJCRWl7MdFRFSawQLt82+VSSv3dWTsnqmTpJz0JlY1zuzlDc5\\\\\\\/gNebgABNljNsD7BNXWgww0zfcaos4MNbB\\\\\\\/Ix3J0h6f+ploMB1V\\\\\\\/cbSgQU+l7OysVp1WWvyw9ZHJNGDfNf+ASpJPcHLEVHR8e+gMGiLJTJjZNyjPsk8QYIllIiPAG2pebWSlK7AxcDhZXiSEo42IHcF96kNZFqqiy6KiOOT2hJDBNP626MbokjgPh4rvV6EoT+ZqLPKQxfnPnhMHXGiz66bs9qYigcxCG+UjAvllUH+WF2Dj3m0kvQTpwBGoh8NQfXJmWRxHuou1uTh5ga19ebeels\\\\\\\/K4Qs84CAPKDVaxyaBIfy7VBv6zjw1FvrQqhbjsNDlVAkC4XhEKI6uubrwwrRVm4r5TNZE\\\\\\\/1yoLNirStdAZDGyzi82Khj\\\\\\\/xD7Xt4JtW6h0Y7zAscJkbL9kQWqZEEpiBEYqhTGRR939WEvLjVvoWziWTsBbzqHVDMuH+gj1KkJFz0fXK\\\\\\\/Otokzthk2J2XV+QAZT3bS1KYlDJc6trsUuY7vdyOdwiQGcY7k+ylW6RRf22+YClriiXI8GabJ87zCPwirhskxrkYNmtB0XFI6wl2f6e031Z7pjbyiDbZ5rBzXk9dRAFS9mRKl0kEBxu0Eshbf3bzL4UcJi76V1pW+TjagPP6pK8655cNnS5OHgCazKhrig0tPbP+4IqKYSv7r1tRcKz0tiAOOZ5WQxOtFGq35rUzRMyx9QL71lG9j0361EXCyWx74NSv0q6mKTKwoz4cLJ3jxLsZKeO18kP9HFR\\\\\\\/7sdJuf9RuGwTgd5trvv0R3GXHIwEbG73pocVw9x687gbYFK36VXw54rrNQ497HoDOX6oxTWQ+TTmVudjx6UllXDzs5yoOhgRpto5oTzDWSCw+OCB137XBvkTCvqe0KuxWh5GdU2WR\\\\\\\/5bRXmRafLxzvwV+22hMY2wKxkrJKewsECNfEl8W5rqi8GShNGZNyjxD+W1S20dCkYW0ggM3WO6EFA4joA+Ts45HNc0VGW5nq0ninlVwAqfXRSEedR11TTQh\\\\\\\/yo4OSwgDqTrjKs+s+SGhw\\\\\\\/sFTKgEsd3LXdk+C4XRF9YPCO2tXZXdxFPsgHHtpsNU7Q2o3L9riRsut03A5x7mBXUq\\\\\\\/DFtLj7MhXh+KzrjM0LBrw6FtP2WgckIUKfQRseuTQt7FHF2h183mPhfKwh8xsFZMZSYowXTLlBqz+cVpd\\\\\\\/UZARZdFvvpMeHCGSvxu8VNCGyG8Jo+SKbokeyJ0\\\\\\\/BRDT5RrB35EYLyXwnqxq0NgzSCT7a0lO5qEeYcYSMV1ZmufSMDmCSGwoxKMxrnDGT0QpZYpjdfwxUGHG0EyYVrtOnVOYDDfvBbkzXNAG8Flp2KHM1s8e8lf273pEOki1aA2ZZ9pnAOd7DK4RX9nlMH3zlN0dSWar4ao4w8jNpMhRrisiX8XWIqGa9ZVVE99xp1\\\\\\\/DO7XRU9W188bhVoQ0PLYctm9MyOMGOB\\\\\\\/ZbcwTUUT+y0NdLu5owGdXETflrCOCuNJfxpZdEJZzlNHuUTXgfsI1tlCXfvAS1B2znPxn8yTOxuSOOUHJCDUTmgLO5ccWcOddh4m8cpC4KEzG8wL8FMXNBfHNF6RXlYzLdo8dfe0RTiE3QXGM+sfowQNbHzMYNjErhPCqTiDyjc6PiNtXXvu5zkr3TRK0A+zTp3zeHatyV6J5zbM0Bx0kJG1pko3FcIrcHdFH7rQbkQEP498n45F9p18UdUJIhXYZGIu1vmfY5vueG4h58JzSQ06T0fHc1pn8O7+d\\\\\\\/MJb7WAOrGyL5g6AdH5Tx\\\\\\\/M3hI+KUozNSdPUt7rFDVHKMh\\\\\\\/rX8XT+6tKao6cgnLKd5SX2vP9nrjwsaRvSy1YL4dSkOzeGeDmDAuFJgb8aPqQm\\\\\\\/0qb5Xuxg+YCJaDqellUKchf4GL4scwVnEregm6LRTIJ3mHITm5z0YFxIIrXVAlRfdYFi\\\\\\\/cPB1eGOcp0oJ3ptwjjCy+3HGpzL3UxGe20RyVrYUeVAikUNJoloCuRHjYDTiLBaoqdSCtTace2JoD9GhXio75\\\\\\\/yVeqNPVf8lmHIX5CSakct9NqZtmUM6wlSeKSGU5MwmCdEbN7sLiCK9IVBFWk1T0fakQFQJLitZt1EyAVStZAsdnBC174JduZHNJ0Pk1vPhLvyZHgo0QSWo1OlEPnUuMFjN2yB1lgzzcsko3a7bkXUzQ6iJctWjUjJg3kvyhaF+Rgj0B4Aa1ziZUCYjbptFihxItIdCxdcKvixEVNLfQUlS6EDh6SU8eTsepFHbz990rgRtU0EeZrZGPFFlRa9jG\\\\\\\/kEkviajAg+sPZvpNnk\\\\\\\/AgPAygs+jh3l1y3GnG9G1syQnh89q\\\\\\\/SyyGZqvR+uV4fU74mPO9C4EaNYwc+JPSxD2jSa+1TOHplgQ6qf7jhfsXS3Y0IsuCynVM4MJ5T3uKpufjYmeViHlf87O7xOcj\\\\\\\/F+dFewuBl35YT+7eyjftgDQko7jwW3JrOSNyACIG9fnoTymT\\\\\\\/Nkb8ImLaEM8os2zedwgO8yVSLTifpe8wq6OB6ynZ7rG8at98nFU4lYrVBolSsXCVdpSl04f7LwOYCADpw4kD8PbhWkD5Cc6H\\\\\\\/PqK88sRLjNqi6IUaxBbFgTWJZxkS5NZq4rVTfJyEzJ4R6YVkH5vtszFNp4qa+un26j02gT8EeHgnSJ4q8kY+QGKWcNE9GTvPfzNYm55MQMmzXSCtV9fWzw7F2Ibxu1HsNmD\\\\\\\/hiSjZd0e7I9cdtPs+IQCk3TT4QKy72+ao87bzIDXU4MCSNmmVZ15YlxdAqz+vmrGa3kxjO2J8DrhqAMnBTXhN0CaZY0TxzLMrcPD8TUvUzHig20QtlEI5B2fg0bWH+EamD78uOZu5g4trfhyOqmcH4Jk+OOm1JxZixPdAfTuKbnOHS4jUX6CfA3PWbGq4r0ALyeP2dOzrhp8Q8Il0HsA\\\\\\\/2VKnTIjfLJOd84QWK3SIyLX1TJyJvwUGb9KcFOdk9P697ojzKAoUwvxYdBpfGduX1JsjMez9kRSkmj26h7KjpF95qVvX85SJ9f\\\\\\\/VTd56tKisoZ\\\\\\\/PYFSMfEudr3TqI+Ot+uh5klBBuWGj90w4pXsBRxQNKMnctUtFBxfrrLXC3x8N961h+JLBBT3WDulSBkSxMGaMtswdK8hQQUqzz6n4bPwAE1LuKlaqpUrWfuA\\\\\\\/iYhHlHW7nitam3iGsSUMBiXOS\\\\\\\/0ZzXC0O\\\\\\\/7+xw7VdTzCXUglwJcjT2Lmc7476WGgU\\\\\\\/UhPEmc\\\\\\\/\\\\\\\/PLLd0MZwQ2algdtMAj2HGdOTsCokSuXwCxtZXlnu1dKOkKpB4KGbxWIoLAO0LOPalvCptu6WcyiL7JPiqIFLcH+flyiohCGp4TZivZQB5UDYkJ1elXvg88JnK5o\\\\\\\/de\\\\\\\/v5r65Spm0oC2GuKJQAq9wwqyRW\\\\\\\/WjQiRVtzJNd5HTz84AGfeh5VEYVQYh02p5R7gR9tLAstADtt1etHfwhHXT7k0nPlqtoi+wvuNGhtTvJUO7GA+R9aLc32nZZiaxePjAvNadFiY6WL6IDR2w6vEGvt42DWrl6jUDsX0O039cw+LsvsRoaxPJOVcuuXQlkl5t2M5XNM6+r0nvMh7Qli5+TZGsFXrfN0QFv5kHm0JDfsS5U2IXbj8mgm8umVVsYraU4\\\\\\\/vOMPWaNdktqQK8MJWjK313\\\\\\\/5S5bP9kwDsvIkwfgzXOa6oXPcIM11t4BIOEsXwZHPmgAg7LQqLPjFrEe+USEIEBTEb2UUgpyk\\\\\\\/40+akBggt7hhw5KgdlW+zTVL2ZTwHawR8lUeqMXBt4KPqE\\\\\\\/mHhR2Y\\\\\\\/qoDC57zkWuIyXpJW8GTFov1jxg8AgCARPtOhBRdHooAXE3BhLvpNF2M+EbMj2GUpuyg6FCviwXYf9wsYLIhgt6cqMx4hziHZg+2pGwpFWk4u5IgrRPGRjvqYI27y3e4Y7rS04SBWNU2UX+SBRfsW0Oorj0v5BboZMMv95x0jtHOrr96G\\\\\\\/peeiKSJdqcXOxmsct+qOYAaiLOh3dMGN3+yo9NYqrxjA\\\\\\\/iOdK08R9SpldBbonaf\\\\\\\/3Sg9kOq79EXLHqOFUn67GrsDGZT+Di\\\\\\\/a33wEpWPUpXvHwdrZkBRDfrC61BL4pNKfEQChf0ehVmh+rTTJfi+uxpuV5U9yZCzI9mSBaBEhoXvu2pareeMxEVktUaI8QJ\\\\\\\/+L7osK8NDV532VFBtYiHsEEF7laxbOOK4hM1+XlprQJHYopGCeTK85TrFZImg80zU0jKtQQFZVG8\\\\\\\/eI0Ps9ZKFrySd6nPM5tY\\\\\\\/GuBgFBa9b12Q7DvkJILndxFiZqAlQdjpsdBWcOTTo1E8vR7ct3CJ12zoIT4MESw3xT8tZf0SkBB3+D083g9VLc1rf0ZbEdJ8fQmScvv7mVFpDhgl9bho\\\\\\\/j0cgnYds3h3k6H6vup2kl\\\\\\\/r+H5sUAi0IYtkd4y1HSqydWeGm\\\\\\\/GxXVoQ4uWJ1kHgrqdywQVUxACT3FwUtemD5CMmVVRU+TWJNRlZG3nawVK9sIiV1PnsWZOtC\\\\\\\/r+lCYRsUmJsxGER\\\\\\\/Ql9u5b+avIu2TzjVhnws0aQUF9nXbhd2iLQ4dvsQekn8NxAnX7SWfpfe4v3qPcrzGFzgHoZ3HnzP1k04tbvzLfVsuC\\\\\\\/DaO85H8Uuv1TElOmhsHwpJJazbOYXO1gG9t0O12cG02\\\\\\\/u23M2H6v3\\\\\\\/gs+9m\\\\\\\/j4XLh5XrDeQfR3WZOXon+rvJ97qLUhnUFdd19BlBO9NeA9LWFno3zuc1ddWoZ1MRcjEDM7ZoLTb2tJyVwAC0gI7Fcd3d+6wXDfz3s9tnrO39h\\\\\\\/A\\\\\\\/mf7nN813b1r\\\\\\\/Mb6Qi1jDNs51UZ3PeXrdX33M\\\\\\\/SFET3yG5PwosczhgJ7b1dG+7hts5RneG4bZzXdWcksTVCMGE9WN1\\\\\\\/4XWNyYcEqnUQguSryskORMJd4rQxdGzzBq\\\\\\\/orTmr+LtSde8nOp1FBR7\\\\\\\/Kni5uafTSDzGXRVcViNo2Lcpumns7qH+vqDA1fhaf0dQS6br8fglM22+yyFKIBAaW07VE2yUnpL\\\\\\\/pohSuj7ZJkC63iabvaPTfVMtDjS7WPUYFvC5X\\\\\\\/gVH0ha4\\\\\\\/YlwAMvm6UQQJ5CF\\\\\\\/icjDUxVpDxHmWg6gzGv+4YYNShUxHgpfkN+cQErfhOPdk6wWdD5aqGc7XmrZA2PsbJlMDTpMg8CQq29E22I\\\\\\\/xseEZ7JIeL6kefFuxou4DUQHDll12FzCIewPcGo+iE1UEu6mtchp\\\\\\\/sz\\\\\\\/8Us+aOwguuf2\\\\\\\/cWJ3WdbgQKDTtg8jt\\\\\\\/9jFwf7SZaLKztmI6q0ozVkIJK8rzgCcb\\\\\\\/8Cve3CRAoS8to0JDqrQMwRgLw5QFtKqrd425k\\\\\\\/xJGz7kny4vehAN+Uo\\\\\\\/ZjrM2yGbSHMzPZE++df52HU\\\\\\\/w\\\\\\\/1x+tpgUKMBIDEJc+ZMk29NwlLMto5RkW6jqihfcTviv\\\\\\\/3u7XxKt\\\\\\\/BXyCxVEByQ88415vWnwSxQ94YQ8Q28aEPvHyduIQaQ\\\\\\\/pPyZRshLeTjpUbAAVSI7pnJeG5AsLw9qlZUhWDvYUf2aKGyXSLSufOPzCMy2Ed\\\\\\\/ktrYERYnDIT25ODQ70AEXHPv0Bg6TC184PuzEJx8kY9GkaEehIvGHFCne+R0ofsqT5D6CpCtH1xn7mFzePmmU4JxULM3k52xYSwIXywm\\\\\\\/Knn2mxNb2zkba7948W4rmvY8HggY1iRX\\\\\\\/L7USXojyzhz8dFcxoTnI4xBKbx3cSTXrJVOVuMXAWZIg03D3BdythfvjX9xoZ5SwC5wUvMjQGDUC4thHhb5WtIaF8jEH9n6TOEZWfIYasnpA\\\\\\\/T\\\\\\\/ynCBCd5m+jm3awKwjOnq3D9STHRDaZ\\\\\\\/kh6LuvyyTLe7zWDFYuLb7lSy6\\\\\\\/\\\\\\\/tf7wzKLjLMN5W6ETl0TjHcm4zlp3f2lqtxiFUmc91k5w2ArN74RwOVNK3wd41vtxJQ6akIe+qxbnTwsrNQ8PW97VBJMMXFB1BXXpBAlO+tnd0vIVUk2VQLpjL9c5IaNOeggrbyFCPq0klHIBxNEh\\\\\\\/Tri9J0LXUW\\\\\\\/XdCNoP2lxl3QK4aoQ\\\\\\\/dRdQ5pR+\\\\\\\/OD4\\\\\\\/NiHaBZVO7qN0VDK01uMrM352lZTnr\\\\\\\/+Nu3fgP6YCnZ+roGcIx+uj3EBAnH6cikKveuYcOTOcNHRXgn1A+7FPxQ5HkPWMAoyWl7+9tIS3\\\\\\\/q00L\\\\\\\/PeW5fzkHXaNDSKUJzRUj3cNDG1luVIG5vfC3oBch+6IGTBWkrKd+31hLR1IAUyLcISexToFEiROUxlE9MPNDBROsdGCzWhROQBEg\\\\\\\/7h6Dn7z7DSG1CJMetRk3oupCY\\\\\\\/o+0bljOMSuMdT2wtJGYSMyP3QlTJ9Vul97tL4lfLiz0HCAAWOlgRaDHXuGDF3zviJFC69xVGs4wU702LqK3j3LQCdEkLRkoDXRnT45gAx9KxJti6cujEiSDLj3TZE7pwsbRbf3+cBpXZvxJ4qaXQAb+k6anwsjnO8ot6Zou4dGnp9mpN+txiVg0bs0Oy5ukbZcEKnKAVj6HkUibR8mKgUoBqGFhnirdLvCQpJ6VxKIMcuIiZ+Ycr3lgIhnj5KxprJL85wHVu14\\\\\\\/HILUtsYawndaKivsqb2e47u8XAG0v\\\\\\\/SXtrzT2dJPoBUtp\\\\\\\/49XEN3coLyNsx95Ud4S2wMkDvzHraHynNwL7y9wHUjq\\\\\\\/NEoHd+edAwtVMCqjbQhp8Td2Y9HfoDFeS7fQiA7X+pFLHhfi7BjtVzxCN7cDB4hK9do7hnjiVFmgV\\\\\\\/udmFNgs7KGrfvQ96SGCyGch0nyoy+bhvz8uTsFO0l\\\\\\\/807qh6KLOcga4eF4Kg\\\\\\\/uofJqqlEEmc7oiuG2qNBKJgFNy4hBIBG1XI3DEWwHSmHe4iwyYYvyXwpnIxUH1rhTaijaQfHYp1dDHn6bmj87ok+jkdSmN\\\\\\\/9b5HgEbNPO1jpD286dXfGzgwkln39Jpv+wyMmXDpUlaz7\\\\\\\/HwHqpVaWDaPqg293UI++LJd5HdycEFXRljTVvNpuKAETuiUMF6KmNoeJP+1mqvJzXI9+5HfYivA83C+tq42ZlpbnNWipdsGSpo9z+UQGvb5pDgfEXN5eZxSSJ9TOVIQu687GPyRoacPC2E34zWtgHfJ4zUMFQ397TwSg7pO0DO4co0MjSLbk7+FYgi8SuslGLftyQ+Ind0qrvedcyFSCq0xYtDq+8WWpan+zOMmUlDSuxdVEl3yuzM8LVJB3Ybxg6KeLrSOUduMJTf5hGtCfZXBHkhYp8259FEBYyUTfaARTVh4tv+AAvVG6dPqhkvR8w6iCv0ufU6Odo+lt2gVhtkmlatxCprgk2ZSa460X4UpL7k5s6ZJKD8ELbXozw4fEXjKSFTCvVyM6L3cTb7tAaDIFPVzccOwgqORcmhg6eeBiX0Tmfp\\\\\\\/SRnwhMtmCKydrNAXnV4gixnQfe\\\\\\\/peCV2m6Q5YiCKiiYWBccZRmjo\\\\\\\/mgnIwEZ0Bb1NLUJe9hRZ6\\\\\\\/8YoaS5\\\\\\\/uaQWoRjHJ+FMbzvCUGY+mmRpF4YZxgtMAm2hoV4lQ8o0K2YeKFrMReksS3g9Lww7cvEh369UPJmlc02n\\\\\\\/c9\\\\\\\/Ei0Zr2FRQ6\\\\\\\/gZwpNNz0oiPNtn6bp71+A2M+pVhlWUR+PnhxnOLGP0wncnHg3dKcbxRIlQy1s42JmHuHB+bEL\\\\\\\/UHUl2iEdvAiO5ND9n4rrZUZjeoeUvYW3CBWS4l1W6k4YxR7WijfnmpY7YyzsNJ9yHLTuTa+1xYnvmosZnsoM3XpQwaoWJgcJ06lzoN1FKrt5sMXk16V\\\\\\\/IxAbI1\\\\\\\/0HQxnYtkIO+76mK7eCprJdl6qiV4BNCSUWKY7BQYXMaJbbXuvUZaTRlx2ES6qzLwMSYMzT1MdHdp53oQI4N0JfKPLhhLg4mlCuPGHgieSXBWsMdS6TrMOYcgfDY+t7KxbCDbMzw3NuOM6x3FFpD+b8stwsgPiUc3yojneqyv6iVeve8JexU42I2yBCgGJLeRdb3HIWXxiZCcA52bb2s2V5qoZVcpgHAY9L8tFeouMQLoJyKLiZ5RbxPL8VDwDgee4BBiWgQL3ikwIAKOfh4ZjSS3jxgnaJbp1VgwPyswOLXlYrYpX2ntE7JB7RLp8YzjOrsJpmUN5E3NzaCbjmeKjP6tYA10RjN09uLjMvA3DhE9xoiAG8gnb54ygHWRUtfD8rW35WdkOYKsvuFMUwbsyrHX\\\\\\\/uP6N4+kn\\\\\\\/PLxpcnMJKp165\\\\\\\/vusgmtchSAmxNu1Y6N+wuco5CWhSOi9h1qSpAk0keSxFyiiqLCMoq6FJTtsOGSCYiR3j8WGoTJg3sWVtRlIU24aa5l1wzYeuYekvIQYxIP6ereAud6Y7vXtBpK+9bfTMl8lotKdxOcCtUx9Cn7oF5cMUWC22Upp6TKO8z4y0MadjfRKmsRNpUfCOmcYRy7EQACdBX4CIbJK+KT\\\\\\\/XH8oECKDJt6mRUSNJh0iXw0Wc7fpq2aVOe8aJcd8z0Afgqq1Swx\\\\\\\/Dlzh\\\\\\\/Yq10f+iX1Uz+cP4BFQvDPHSf5jRbRYMrDacThrXp3f0lNnW3VvStSvE2nZwftjzF2fSOLvgBlVMitzBVV85Fk4mvNGPbhNwr4SrSprvKM4sGVSTmBrfHYZQ5Te2sUnVhZCihZPUOUVVhkjHbBMXPvlmF7wP5IQNVF0hr2W\\\\\\\/Tj7X140xbpNWF9\\\\\\\/itwL\\\\\\\/0ZlhVKyoznQniAzK240qbdrqF9SB6LThm2AtNtgdJu01scx80Cinmv1JTVwhsd++iCy1WBPEIawspg8Qo1kPx\\\\\\\/Er8rCLrBfo4oOYYLWFO57A2h1pot04QZQEjN0JehHj7AaGVZcHIjlyicRD1AKGej4xOMJAc7\\\\\\\/sMR632i2kGPuqEVupdbpgVmXp9q78Evzj8gepp4HUst+\\\\\\\/fXQi8QZLTiumecx1q0OegOs2QbJhIG9cHRUK6k83HVbT9Ug1sZQqgObnmUoPOmAQ5mRo83Dem2asanLKtBxcuVywHdFzimiWUmKwv7LuGsg3BVeo5z7u1R4125vRfzI+LYSj6l0AcBPeojPcsvmRcOWeSbb5Znu\\\\\\\/SojoxW5RfN1JwvLdM8gsZjwnxGn76DXdbvvkuFJLEwwxYnCFZBUGa8qBMIXkuyojK9jleig46hIh\\\\\\\/mOnoOjMMktz+acCtshZUnZ2xiSmVPGAteiYlz6bc9PkV2DkkJTg2NHBC6afGBnvNEojPiVF9XEMsX89ezA8XP0bulgDn42MvEcFMoCj99tdcPNTkS6ab9rBeiJfmUhivMGi52kEK2Nu7A0Y3g0xCBWz2PAEyWGsTRQiZ0g2SUPevzvLj1cgPuMUxuPV1ewEqJcEIAt2EPwZVN8srgVbqmEFxxL9YupAhdgjpLvOaW2tRrPJ6MZs6p51\\\\\\\/Pf3NWs8hTtxoJmFrdZhKWg\\\\\\\/g9TC4ss8070kXjpfdUZZvl6HJ4ZZnsZZh2vLQ+8nxN5Ii0Ba9iQkGF73Q9HfIMad88D7Ctb\\\\\\\/N7S7TNmZ4uESB\\\\\\\/izJrVT1+kVxU4nH3A8Tc+zbiBPvkjNRSOqReaWlMumS60T27p0WI1vCWJ6R7PlRaTMNhnNfhN6wVq5+3iqSYhsSZuYseuUGn+6EpIA+7Vizv+HPsk3+dygU4OoHbCaYZn57nLU1fjqS98etMe96nPR\\\\\\\/wIOk4Wxm7GKz2b6+YBgz2Gl25cZDLfJ29uqOnfHCUG25dC1Ki\\\\\\\/eXG5MbeSKXtXs\\\\\\\/JF6\\\\\\\/jEpLBX2Z6bA2oPH3qikEUNTRjzuWgBdLHXEyNWvfUTQkPyPkS39f8GKtWDNCcAIRZTzsfeHgZU1yQUHU8\\\\\\\/F\\\\\\\/XqedqdA67zQxmJJUdxEPTwZyQWQq9w08A6Pfyozh0QiNhTN86J2guTT5J\\\\\\\/VmrlY27vqrVzknOZH6RYYFql2kdUtuDC6JQv2d7D97eYrGiGbyjbAQct+U8yw\\\\\\\/HQ7NKajvMOJwU+W6C5oxZ9WNvakZ\\\\\\\/jNRQQYkBU40mSDwGe+z3A5DQTokzlB0DOunyZ+8ohLJ+uJFnbZ\\\\\\\/o+PsZILNf+PvVYJsFFSUWF0GpwA7\\\\\\\/n2ZAIU4A8eIf6ZxZM1fziAiDZ0e7lCmeYMIXBxbhY5UTk501U5vF\\\\\\\/1mcCVcW+8pq+wUS19g8lCJPx7PI35N1w\\\\\\\/0ckcvzenElbyBqnPHZYh2Gk\\\\\\\/8Bl7ToFHjSiWqun0wti7fdWBDbAAbaV3WkZz7ku8OopgPNQcgTT7MfUq2N+R4uJLtxSqLKXGpVxwxV+o2juzsImZg2xpcGHjlEbE3J9Whv3JdssnPri5BZXPkJpGZAlM3yB2A\\\\\\\/rk2XkV10VD1xhDbPJDoF0tH0uYdMn60CbEunDrdFbTjqFemfDISpR4G8TA9weoR\\\\\\\/+bJ66erk2A2BNRNlDnAYQb23JeOnHD\\\\\\\/hChq2PkE7IN59tO5eB1jcwUxJ9tQj7a1t\\\\\\\/xRX7\\\\\\\/jrjFMK9txentGs5V4ycm2I9wYIe5FOGkOlGX7IGQ\\\\\\\/7NPsiIZ3y23Myw4D5rC4O2DaVOKpAM+HoVlwQWN3JC8dLfmEzEl0FbBSe88Ov38GRBvlFQ1kEeEUy5RvsBg1Yw\\\\\\\/uOfwbmSrwI\\\\\\\/zoFkZUFEq7BuHm5uXXFelBDmUny6a04yJfuR6xGD\\\\\\\/mG6EsWfjyRVbzuMb+yWfaxzOBTXQelOhy2QaiRvriUoeZt+OCFrTldXMTa2\\\\\\\/peX4ZYHyrI4T1i9ILwVjCbdVDhpKTaq7m6opuRi149hse6JMHBIoDS4gnCvzALVRdEkQsVllvCyDBvDDTUHTmDd4j8nSjj8Su+DXi0gdAW6sSIXus9o4hTNwxc6btzGbroy+2J4rB++DIvkdgC8PI41CzyS1KpqrQTW93nC25mj6kStYSFtOLnENvRMlakYCPAmF6NXDcJo\\\\\\\/8QN8TdFCBSx1MctXHGJIb2\\\\\\\/mJIidKpZs2h\\\\\\\/o5Z3VF10WenLsfv+CQSGm6AFO4QazPy0aeQ2c1cJLAylSrUXLL1hCcLL1vmz2vgVhszJqo8SGcLES1yDlglHYksRBg\\\\\\\/rPcaK4339LoCIDXqz10nD58\\\\\\\/c97YaR8hlJGJXKcckgpySSMDAN9iHcaxxAZI68jj0Aa3NwEVJlUJO6g5gRD2wNSwDPZ9Fa7PVX3fZutT7HBEm1Jv3wytf7Z1yIGu6Ifr5lCHgrnHbFoigvYlFkxVFjPr1pKVmVsCvjtaCAXsrby4sla7kpmRJgWWMawdCnUm+iGSOOtXup7p1qO5R8Ur5cwvBs9BykvXaiIyY87TNbVKh4zfiifQ2uuTzY1rNtfMlOcBrBQIDUkmgk4ugarIkVxHkpguIxVfuqdbSr9IOwKicyjj7OcxvbwcPoQe9jh+wKs9QK+f\\\\\\\/6I8DwWjFDFifBgIoMzSr6p4IqGp2sXZL\\\\\\\/LyhX9Kkm+ast41nTgTEGKQNZASxsUiizHPmxEI59FDTqCNXJX41+wJZ6vHO16j9YqZUf1swS+GmVqt26Up\\\\\\\/cPtlcpOLM0MLHltVCRmjfSrCbVPSZglCYk3cRbQU45QoTP0MI9Q2K3TLXrqAPRilGT0OjExL47GCg4Hzr3He9JpGPaqgNl39d50vh\\\\\\\/CthJr+E1HOsqKEsNOPJj6ZVkhPsBwjtF+jNBsDJOI++92NkZqpRuvahv29mmAsnzLirZ6IBkOjNmGi\\\\\\\/tKwEXN66HAjAfFYoZVVsmTxdYfcnBHhwqzHL97aJWgb+GexQNUDyjkfhVjQVMaX27qEFOCfKicGJl+LsdkcvjcCuU+P2GMBgGrcCLjIW1xPVor9npB0sQI++SWMHTqdnBTR6sC\\\\\\\/GRFv7ZM8eYVIZzw1VXj01f7ekkISDkL2VND0I9Hsu5EdY9HYI7RwxfSvDpfZl74Onjk+S2KaWufEjAO+r0SxHlxI0RFl2omiUYyVgf2OEmzmJK2ggo3nSGEZEGQpsoXBbkeNciuHxHdYOMOVDY5+4p8co9KoQwiLhz4VTiaco7+07VdWxY\\\\\\\/dS5I4JJABb4f0P8EuKnJfHioMLqAmx9wokO8GsVgNbn1KUmb0YvbIF9wHjQJaPnHnkGAmkrGcYvHD2MpIQCzNMgV1DyacH3iTQAuDKSeA3ROPp6Wz3WjBQfYKiXSjTr5DwCRCZzjLd24jn205WoTL2SDkYILTD\\\\\\\/VdB7FlqwUzczFky1lhO97QFzqKeqqyPBhLncG66pE0v7an8qxzuJ6b9wmCrD9Uhpsqz6wXXHpr9I\\\\\\\/yEB10ee2kh+oAmrCRW7dRRu6aZfa5\\\\\\\/bn4hfhGKyM455vLfkTlMG2LX8oVa\\\\\\\/NRX1BIgqSGu+aNZBHYncbffIa4Jorkr7R583nIPcReyBd6JNqfGE43busUTBRXZK4NksxSzkjo58QZA1fwVLnes2VLNORfqoExPR66U\\\\\\\/ulhPhS9zt1vQuSQn4m7ryJT5vVCHowpVUeFSKCb5lwp5KILSyhZRdstEfSmLsnih1owE3RBbkP34Ps9gJ7c\\\\\\\/sXvs\\\\\\\/H97j7bmBlIV2t2WlKdFiDfYN2XyxvkTOctyiWesouwfDlH2jZ9er2v2y2kzZdn7iJfHiGhYMRnVeIDyzKMMy3pkg3Vt8Jmr5sTUynQ\\\\\\\/ccRt2KjkxEk0veWq78VhI5TfzwgxzTVcdvL7z+CQz+RkJp5omxj6X12dXQ2rFduRedtfePODCLEH4TjWeaC9A8Ydj7v\\\\\\\/SgUveQj8MTrld5sr4eq7vkJzC75bGIvxYPboWONKt0S9nuGRbxNiQ6ye8eGIV5h0kQ4\\\\\\\/LblNZC6dQBYmE9HWJDvIjYNKJRDamuI7NMzv6+bSXnUlu3OiSta1JioTnC+i1RPQsEvWifu2XrQIZ9fBLVNKedgSce3qQYLKWSxkJkvgshMoVhl9a92OQqj9LRA\\\\\\\/F5JNjpgqOnVZFwBpZLa7QetFu9Xj\\\\\\\/MVe1c2XMDE5JhxS4Qw17YgMNdvjwt7Ht9U7XsG1zaLdI27uyKbzAIz9trQ4549LkkR+HPDH8P4OFop\\\\\\\/RwmUulBPjRfxuq\\\\\\\/sOk4QISYz5b3u81j5o3c1xqGOpIPefeUkTkndP8VTeBgSPVOkDJ8p3NJ2DrK\\\\\\\/u8212s6YQBTzZAX+uMqdBcix2oNQG618clmfkzFU2S+JxBGlS32czf1P0yDwPJX8CU43wjo2RjUE7t8kPdoukqBNtf+tVqwqtnQuwoqbTZtyOnq+04Iucf+Fmed5bp1HYMwAAQqfpmrFeplvYCtcWCs6mg6U7oXpISHApQmSqbVOjhtSADRjasrFWx4ZY04hPoBRCkIml5BR8\\\\\\\/XKvPB0MHv7jpSAKfTXoFeuBrjwD5GQyAX6ZHvQG5R2r9vJSPhB53DO+NZK5WjiH3rmb2A09G+CmBlBdZbogm3rf+lBoEtscWmKlAgucs7d8u5KeBaZiMk7kaRPI4i+FR6uW0pCkT0pQjmNDgYJx9NQyhKH+J9kwv0El0KIPDj\\\\\\\/h8qGyn9OoEhVazJ+dBV6IHHei\\\\\\\/t0\\\\\\\/PcqLgJR\\\\\\\/Ju6NaGJaWIBqH69hqFkU3tt2DE61mnmonfJSEE8zYIY8kXPip+hcC+7Cz37mpwDwlVhGCKAQu48JXLPZdlnS7ZOyIsZgeqNsUMAqHDebzU17piYh4Cebj16+eCYL\\\\\\\/3J6p+ppz1OugfDScIPqxVmZCWE0ozfC9VgdkB9EBZStmuCnOjNoZ5XR3A79tBo9LzGRaHVLkTV5etSeV2wMoze2Cuj\\\\\\\/jhb4+kGryjqfCRU+3hA5JGr1NZBsEwWrFZiDhwmuLSylnmgvgtQYwrHMvC6NNrJ34ArCv3vCf2oFgw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022Wzmm3Cyfpqf3UyfOSLdJjQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022ff48a46ff38dd5191707d9812e18b139c7ba1c00f2a77debe9acfc85bf2153c7\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022sF+h+Ie8YpmUW7NKe676fhZnczS+eXNnkk+Mes7fZH2NReLTViwLYA5kXpvnCauvY6Z2DvtF+A6tIRfz+2XlPoxPq8opfJXuYh5xCZwDZmLOLWDbdmKaS1Qt\\\\\\\/iN65pKq3zgNGaJDUWCXgYjk94DA\\\\\\\/8wzFI+ljWM3pjLH\\\\\\\/4q76sZdRRgepbPEWFJ3yp\\\\\\\/RUQP0v3lm+TCFXVOD+ZVkB4kIxvalmNuWbHkThLbydCnOlC5HDyJOvx7cXq1NPvCx1ZW0Sxnhnros9lRQmt7j9F9gYmVr1zhQm80kdAUbTqJLsH7ETk2EfTNaCR+RUiO0RO8KyMQ6OlbOXNR+MaIUn2x3ge3WwePXJsQKKIJfqLY0yUu\\\\\\\/\\\\\\\/rc+zcFTz\\\\\\\/MO64ihX71e09VjB4621x5Crk2P+fDSto5\\\\\\\/Glsu4hUns0tNGdZrQFXJ1LUTD+eqb0wU97UKiwcrw307vDM\\\\\\\/edRUngqN2jsgvbBN7DYSe9WWLoowzX\\\\\\\/X8mJeD7+grQMU0QqBy5nIfMR7ySmrYLBrmHXdGIOokMebpeIVaMkYC7s7JXKXF5ebdXHjqZDnTQ357n1jJXEZU+f55P1jHXff3ShcuxgT5O4MAe9QqBb6Yi1O5+tO4GrzFSpqGK2JuLAGe\\\\\\\/vFSulXCVykCuac5LxWGGwePxdIaLlYtu28oesYtIJTtP4Vl9HAGY8CN\\\\\\\/VucmxvvR4Irz9c61JSwtOYgzPE\\\\\\\/XAIm11bBdb24dzrGil09sX34qHSZhM8kW1Pz+\\\\\\\/sh9yywNe\\\\\\\/iHOUxkRu3Oif2Vnp1cJ3MZ4SEpG2iBuYAKtcXQt\\\\\\\/wNWnXnECcHWQ6RVytQb\\\\\\\/OQpfY7DegZZWv7e1dxKXmf++5vugKcGyNwriC+hMYUeQxQKxMDHQsDB6eQkggsOsCRWiZXuSRMikX6jUpZs4Nxl\\\\\\\/1AsjyOOn506QBFR8I9wYuwctVfulFkNs3kqTmQp5FimzSA56SaRh3Io\\\\\\\/yzIM+uzl3Efx0i7xqkL0YyF4eDfTRMQdP51tfvpAvNPmAc9KB8v3T6GqkFYAn0FkUCumC3ZAi1tgAgnohx9rQq\\\\\\\/r0DXr04+jzNZDHxkg+uX67Zxyr8bOvXtPSKGy0V0509+fhjJELk0t8PET0Lem805P3K9RImdm9i3OPHIYKvrd5Vc8LpjkG1JyxVxLPSLlXoc8DlaTvlFomaANAmTgXiHufA4CAYMDMrqexfHUKZOsJ15ckq5zJn331VwAzsb8\\\\\\\/yQc04S95tjfLTEz1ww4KGdtsIAnvlWkw0E7kmO25+4AcidMS5MQOPJ79vPnNyQqvnTRdLPdmkPcMpAkhZruOsmv1ZCs3RScHDJyM4auqw8TOiP7\\\\\\\/xDE0zjelbYG\\\\\\\/xjjfRoIQh7cxm4rJ8gLqxinoD+qyHjDs6slmATAzdFubjjLnZy00bk2HfmQ\\\\\\\/HR1daens8dpncx7HxbKTSQioi2dUW8yCn0MTpXf\\\\\\\/s51WOU5d+P4+3TGQtFtRhkpWoSvra0VffUp9SH7SNuuNt+T2kXWfh198499QYrtHR4hOcJvBJbZ03Wl6fRzeO5lQ5GQi9WbTmnu62cOoHZHFKBYnRSTWVQv52FeEySFubZC18XJo4eYrXujjPuGfvPOVOFkkctHgCGUNnY2BMNpEzV0yh7gn6uCeCf3MqsJ5K6UdSguE3O7orZzCPRCMb1xarzei+Uim3ghL2hfsRj3n0Kt5Uc7wWHEdb5o2Tl\\\\\\\/mNUW6wTlCaiGbBbAJSv6+X1\\\\\\\/D0hJH2aUQ7r82xJ3BKdG4wbOpDUeCa+qedBCuebVUD+FUFYvpYqcx7BhIvl6xlx9Cqo63T7+yxeC3f0gWNH2DzjxcttybHu3Pib6mNij0b6UaUUzc1omoSZNXsMykDvWg0ff6xofddWm+uMkMeVPmBzFQZSVEkQSSa6Emo1MN7iBxkJ6o1gN4ASstMROJI1Ev8bt68yjensJENwO9IWaFSqQ2XK+awqBbP990T54820hHCM89QnjfSeC+IePq8+55QzJ+xRlrC7QTMpkPY9k6JGxVw==\\\u0022,\\\u0022iv\\\u0022:\\\u00228ffvhMzFNQf80Nw7Gj0bnA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022f0095637b71dd46f179f28d61256dabdf3967951c04d4064bd188e1b503d139a\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-494604 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/11/3663" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-iqRhV6vn9VuSS_hvGmsJyQEBbbGeJACkpV9DnUKuRZo" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/11/3663" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-w1OMqHpGCZMzpExJVvABZ95PJVsnRnBUBJEPWIYqQfw" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/11/3653" title="Regulation of the Neurospora Circadian Clock by the Spliceosome Component PRP5" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/11/3673" title="Dynamic Patterns of Transcript Abundance of Transposable Element Families in Maize" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494604" id="node494604" data-pisa="ggg;9/11/3663" data-pisa-master="ggg;g3.119.400684" data-apath="/ggg/9/11/3663.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genome-Wide Variant Identification and High-Density Genetic Map Construction Using RADseq for <em>Platycladus orientalis</em> (Cupressaceae)</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-8275-6367" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Yuqing Jin</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-9437-3198" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Wei Zhao</span>, <span class="highwire-citation-author" data-delta="2">Shuai Nie</span>, <span class="highwire-citation-author" data-delta="3">Si-Si Liu</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0002-4887-8977" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Yousry A. El-Kassaby</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0002-6150-7046" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Xiao-Ru Wang</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0001-9735-8516" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jian-Feng Mao</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3663-3672;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400684 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494604" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Yuqing Jin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Beijing Advanced Innovation Center for Tree Breeding by Molecular Design, National Engineering Laboratory for Tree Breeding, Key Laboratory of Genetics and Breeding in Forest Trees and Ornamental Plants of Ministry of Education, College of Biological Sciences and Technology, Beijing Forestry University, Beijing, 100083, China,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yuqing%2BJin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jin%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AYuqing%2BJin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-8275-6367" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Yuqing Jin</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Wei Zhao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Beijing Advanced Innovation Center for Tree Breeding by Molecular Design, National Engineering Laboratory for Tree Breeding, Key Laboratory of Genetics and Breeding in Forest Trees and Ornamental Plants of Ministry of Education, College of Biological Sciences and Technology, Beijing Forestry University, Beijing, 100083, China,</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Ecology and Environmental Science, UPSC, Umeå University, SE-901 87 Umeå, Sweden, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Wei%2BZhao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhao%20W&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AWei%2BZhao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-9437-3198" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Wei Zhao</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Shuai Nie </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Beijing Advanced Innovation Center for Tree Breeding by Molecular Design, National Engineering Laboratory for Tree Breeding, Key Laboratory of Genetics and Breeding in Forest Trees and Ornamental Plants of Ministry of Education, College of Biological Sciences and Technology, Beijing Forestry University, Beijing, 100083, China,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Shuai%2BNie%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nie%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AShuai%2BNie%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Si-Si Liu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Beijing Advanced Innovation Center for Tree Breeding by Molecular Design, National Engineering Laboratory for Tree Breeding, Key Laboratory of Genetics and Breeding in Forest Trees and Ornamental Plants of Ministry of Education, College of Biological Sciences and Technology, Beijing Forestry University, Beijing, 100083, China,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Si-Si%2BLiu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Liu%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASi-Si%2BLiu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Yousry A. El-Kassaby </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Forest and Conservation Sciences, Faculty of Forestry, The University of British Columbia, Vancouver, British Columbia, V6T 1Z4, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yousry%2BA.%2BEl-Kassaby%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=El-Kassaby%20YA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AYousry%2BA.%2BEl-Kassaby%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-4887-8977" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Yousry A. El-Kassaby</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Xiao-Ru Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Beijing Advanced Innovation Center for Tree Breeding by Molecular Design, National Engineering Laboratory for Tree Breeding, Key Laboratory of Genetics and Breeding in Forest Trees and Ornamental Plants of Ministry of Education, College of Biological Sciences and Technology, Beijing Forestry University, Beijing, 100083, China,</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Ecology and Environmental Science, UPSC, Umeå University, SE-901 87 Umeå, Sweden, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xiao-Ru%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AXiao-Ru%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-6150-7046" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Xiao-Ru Wang</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Jian-Feng Mao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Beijing Advanced Innovation Center for Tree Breeding by Molecular Design, National Engineering Laboratory for Tree Breeding, Key Laboratory of Genetics and Breeding in Forest Trees and Ornamental Plants of Ministry of Education, College of Biological Sciences and Technology, Beijing Forestry University, Beijing, 100083, China,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jian-Feng%2BMao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mao%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJian-Feng%2BMao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-9735-8516" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jian-Feng Mao</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jianfeng.mao@bjfu.edu.cn" class="" data-icon-position="" data-hide-link-title="0">jianfeng.mao@bjfu.edu.cn</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/9/11/3663" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:494604" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:494604/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/11/3663.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:494604" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:494604/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/11/3663.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:494604" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:494604/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/9/11/3663.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:494604" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:494604/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3"><em>Platycladus orientalis</em> is an ecologically important native conifer in Northern China and exotic species in many parts of the world; however, knowledge about the species’ genetics and genome are very limited. The availability of well-developed battery of genetic markers, with large genome coverage, is a prerequisite for the species genetic dissection of adaptive attributes and efficient selective breeding. Here, we present a genome-wide genotyping method with double-digestion restriction site associated DNA sequencing (ddRAD-seq) that is effective in generating large number of Mendelian markers for genome mapping and other genetic applications. Using 139 megagametophytes collected from a single mother tree, we assembled 397,226 loci, of which 108,683 (27.4%) were polymorphic. After stringent filtering for 1:1 segregation ratio and missing rate of &lt;20%, the remaining 23,926 loci (22% of the polymorphic loci) were ordered into 11 linkage groups (LGs) and distributed across 7,559 unique positions, with a total map length of 1,443 cM and an average spacing of 0.2 cM between adjacent unique positions. The 11 LGs correspond to the species’ 11 haploid genome chromosome number. This genetic map is among few high-density maps available for conifers to date, and represents the first genetic map for <em>P. orientalis</em>. The information generated serves as a solid foundation not only for marker-assisted breeding efforts, but also for comparative conifer genomic studies.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/radseq" class="hw-term hw-article-keyword hw-article-keyword-radseq" rel="nofollow">RADseq</a></li><li class="kwd"><a href="/keyword/linkage-map" class="hw-term hw-article-keyword hw-article-keyword-linkage-map" rel="nofollow">linkage map</a></li><li class="kwd"><a href="/keyword/marker-distribution" class="hw-term hw-article-keyword hw-article-keyword-marker-distribution" rel="nofollow">marker distribution</a></li><li class="kwd"><a href="/keyword/segregation" class="hw-term hw-article-keyword hw-article-keyword-segregation" rel="nofollow">segregation</a></li><li class="kwd"><a href="/keyword/genome-organization" class="hw-term hw-article-keyword hw-article-keyword-genome-organization" rel="nofollow">genome organization</a></li></ul><p id="p-4">Genomic resources, such as whole genome and RNA sequence data as well as DNA markers, provide thorough understanding of species genetic variation, genome organization, and insights of gene expression and regulation. Forest genomic resources are essential in the translational applications of research findings to tree selective breeding, utilization, and conservation. Conifers are the dominant component of forest ecosystems in the Northern hemisphere and due to their economic and ecological importance, substantial genome sequencing resources have been directed to several representative conifer species (<em>e.g.</em>, <em>Picea abies</em> (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Nystedt <em>et al.</em> 2013</a>), <em>Picea glauca</em> (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Birol <em>et al.</em> 2013</a>), <em>Pinus taeda</em> (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Neale <em>et al.</em> 2014</a>; <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Zimin <em>et al.</em> 2014</a>), <em>Pinus lambertiana</em> (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Crepeau <em>et al.</em> 2017</a>), and <em>Pseudotsuga menziesii</em> (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Neale <em>et al.</em> 2017</a>)). Additional research efforts are expected to be further dedicated to sequencing more tree species spanning several taxonomic groups.</p><p id="p-5">The Cupressaceae family is one of the largest extant conifer lineages in terrestrial ecosystem with substantial horticultural and forestry values. Cupressaceae has 30 genera with 133 species with world-wide distribution and harboring the world’s largest (<em>Sequoiadendron giganteum</em>) and oldest (<em>Fitzroya cupressoides</em>) trees. Members of Cupressaceae family are either monoecious or sub-dioecious and rarely dioecious trees or shrubs, and are remarkable for their morphological diversity and broad habitat adaptation (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Pittermann <em>et al.</em> 2012</a>). Despite their abundance and ecological, economical, and conservation values, our understanding of Cupressaceae genomes is very limited.</p><p id="p-6">Conifer genomes are generally large. The genome size for Pinaceae ranges from 20 – 30 Gb (<a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Pittermann <em>et al.</em> 2012</a>; <a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Nystedt <em>et al.</em> 2013</a>; <a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Birol <em>et al.</em> 2013</a>; <a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Crepeau <em>et al.</em> 2017</a>). For non-Pinaceae species, such as Cupressaceae, Taxaceae and Podocarpaceae the estimated genome sizes are in the range of 9.0 – 12.4 Gb (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Burleign <em>et al.</em> 2012</a>). Even with the advances of next-generation sequencing, the production of full genome sequences and genome assembly for a conifer species is financially and technically prohibitive. An alternative, but not mutually exclusive way to describing the genome of an organism is that of linkage map. Linkage map orders genetic markers and can link phenotypic traits with genomic regions to interpret population and quantitative genetics patterns of variation (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Lynch and Walsh 1998</a>). When available, linkage maps can be complementary to genome assembly in providing local fine-scale genomic information (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bernhardsson <em>et al.</em> 2019</a>). Construction of a single-tree linkage map in conifers is straightforward owing to the presence of the haploid megagametophyte tissue that is readily available from open-pollinated seed (<em>i.e.</em>, no specific crosses are needed). Each megagametophyte sampled from a mother tree represents a distinct haploid meiotic product, allowing simple segregation analysis of maternal alleles.</p><p id="p-7">The construction of a high-density linkage map requires large number of segregating markers. Traditional molecular markers (<em>e.g.</em>, amplified fragment length polymorphism (AFLP) and simple sequence repeats (SSRs)) have been useful in studying forest trees genetic diversity and mating systems. However, their number and genomic distribution is somewhat limited considering the large size of conifer gemones. The developed restriction site associated DNA sequencing (RAD-seq) is an efficient and widely used technique to obtain substantial number of markers data even for species lacking prior genomic information (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Baird <em>et al.</em> 2008</a>), and represents a platform for generating genome-wide polymorphisms for a variety of applications (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Davey <em>et al.</em> 2011</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Parchman <em>et al.</em> 2018</a>). Double digest RAD sequencing (ddRAD-seq) is a modified version of RAD-seq, and possesses greater flexibility and robustness in SNP recovery as it uses two restriction enzymes comprising a rare-cutting one and frequently-cutting one and avoids random shearing of the DNA (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Peterson <em>et al.</em> 2012</a>). Precise and repeatable size selection allows more consistent recovery of shared regions across samples (<a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Peterson <em>et al.</em> 2012</a>). ddRAD-seq has been successfully applied to various organisms for the construction of linkage maps, QTL analysis, comparative genomics and genome assembly (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Kai <em>et al.</em> 2014</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Meier <em>et al.</em> 2017</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Hollenbeck <em>et al.</em> 2017</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Konar <em>et al.</em> 2017</a>).</p><p id="p-8"><em>Platycladus orientalis</em> (2n = 22) is a member of the subfamily Cupressoideae, the largest within Cupressaceae. It is native to Korea, Eastern Russia and Northern China, and is globally introduced to many Asian, European, American, and Oceania countries (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Li <em>et al.</em> 2016</a>). In China, <em>P. orientalis</em> grows in a wide range of environmental conditions and is adapted to diverse and extreme climates, and thus considered a keystone species (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Wu 1986</a>). Its broad adaptation has made <em>P. orientalis</em> the species of choice for major afforestation programs in Northern China (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Zhang <em>et al.</em> 2016</a>). In addition to the role <em>P. orientalis</em> plays in reforestation and environmental protection, the species has substantial economic importance for its extensive use in traditional Chinese medicine and as an ornamental tree. <em>Platycladus orientalis</em> has been the subject of limited genetic investigations using SSR markers, including mating system in seed orchard (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Huang <em>et al.</em> 2018</a>) and genetic diversity in breeding stocks (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Jin <em>et al.</em> 2016</a>), and impact of future climate change on the species’ distribution (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Hu <em>et al.</em> 2015</a>). To date, knowledge about <em>P. orientalis</em> genome is very limited (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Hu <em>et al.</em> 2016</a>). According to <a href="http://data.kew.org/cvalues/">http://data.kew.org/cvalues/</a>, <em>P. orientalis</em> has a C-value of 10.46 pg, which corresponds to a genome size of 10.23 Gb (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Gregory <em>et al.</em> 2007</a>). For such a large and difficult genome, resources for reduced representation are always good to have. The availability of reliable genetic markers with reasonable genome coverage is considered as a prerequisite for conservation, breeding, and utilization of <em>P. orientalis</em>.</p><p id="p-9">The objectives of this study are: 1) to evaluate the applicability of ddRAD-seq for generating a large number of high-quality polymorphic markers in Cupressaceae species, and 2) to construct a high-density genetic map of <em>P. orientalis</em> as a first step toward understanding its genome organization. The significances of such effort are three folds: first providing empirical assessment of the genetic property of ddRAD makers (<em>i.e.</em>, whether the markers segregate in a Mendelian fashion); second, gaining an overview of <em>P. orientalis</em> genome organization and how it compare to other conifer species; and third, generating genomic resources for genetic and breeding applications of <em>P. orientalis</em> and related taxa.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Mapping population and DNA extraction</h3><p id="p-10">Open-pollinated seeds and leaves were collected from an elite tree growing in a seed orchard located in Jiaxian, Henan Province, China. Seeds were soaked in water for 24 hr and the haploid megagametophytes were dissected for DNA isolation. Genomic DNA was extracted from 139 megagametophytes and leaves of the mother tree using a plant genomic DNA kit (TIANGEN Biotech Co., Ltd, Beijing) following manufacturer’s protocol. DNA quality and integrity were assessed using 1% agarose gel electrophoresis, and further measured with the ND-2000 spectrophotometer (NanoDrop Technologies, Wilmington, USA).</p></div><div id="sec-3" class="subsection"><h3>ddRAD-seq library preparation and sequencing</h3><p id="p-11">To select the suitable restriction enzyme combination and fragment size range of digestion products, we performed an <em>in silico</em> analysis using <em>Pinus taeda</em> (<a id="xref-ref-51-2" class="xref-bibr" href="#ref-51">Zimin <em>et al.</em> 2014</a>; <a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Neale <em>et al.</em> 2014</a>) as reference. Two restriction enzymes, <em>Eco</em>RV and <em>Sca</em>I, were selected for ddRAD library construction. <em>Eco</em>RV is blocked by overlapping CpG methylation. Given the heavy methylation of repetitive elements in plants, the use of methylation-sensitive restriction enzyme is expected to reduce repetitive content in RAD-seq libraries (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Pan <em>et al.</em> 2015</a>). The optimal size selection range was set to 414 to 464 bp. This exercise resulted in up to 152,874 fragments in this size range in the pine genome.</p><p id="p-12">A ddRAD-seq library was prepared for the 140 DNA sample (one for the mother tree, 139 from its megagametophytes) following the protocol described by <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Sun <em>et al.</em> (2013)</a> after slight modification. The purpose of including a diploid mother tree sample is to use its genotype as reference to validate the genotype calls of the megagametophytes and improve the accuracy of data filtering. Briefly, 500 ng genomic DNA of each megagametophyte and 2 <em>μ</em>g DNA of the diploid tissue were individually digested with <em>Eco</em>RV and <em>Sca</em>I (New England Biolabs) at 37° for 8 hr followed by a termination step at 65° for 30 min. After digestion, dATP was added to each fragment using a Taq polymerase to produce end-terminal A-overhangs. Next, we performed ligation in each sample with T4 DNA ligase, <em>Eco</em>RV-adapters and <em>Sca</em>I-adapters at room temperature overnight. The digestion and ligation products of the 140 samples, each with a unique barcode, were pooled in equal volume and purified with E.Z.N.A. Cycle Pure Kit (Omega), followed by a PCR amplification. The PCR products were purified using the E.Z.N.A. Cycle Pure Kit (Omega), and quantified by ND-2000 spectrophotometer. Size selection was conducted on a 2% agarose gel, and DNA fragments in the range of 410 – 470 bp were excised and extracted using a gel extraction kit (Qiagen). Finally, after quality control using an Agilent 2100 Bioanalyzer (Agilent Technologies), the library was pair-end sequenced on an Illumina HiSeq 2500 at a read length of 2 × 125 bp.</p></div><div id="sec-4" class="subsection"><h3>Variant discovery</h3><p id="p-13">Sequence reads were first demultiplexed by barcodes with Stacks: process_radtags (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Catchen <em>et al.</em>, 2013</a>). Sequence read quality was checked using FastQC (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Andrews 2010</a>). Low quality reads (average Phred scores &lt;30) and adaptor sequence were removed using Trimmomatic (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Bolger <em>et al.</em> 2014</a>). We trimmed all reads to 100 bp. All clean reads were then clustered by BLAT (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Kent 2002</a>) (-tileSize = 10, stepSize = 5). Sequences with &gt;90% similarity were cataloged into one ddRAD contig/locus. Each ddRAD contig was considered as a bin containing several SNPs. In this study we used the contigs as mapping unit (each regarded as a locus). Due to the haploid nature of megagametophytes, each unique locus should contain one of the possible two alleles of the diploid mother tree genotype. Loci with &gt;2 alleles were identified as repetitive origin and were filtered out. Loci with two segregating alleles in the megagametophyte population were identified as polymorphic. Among the polymorphic loci, we only kept the loci that were in agreement with the mother genotype.</p></div><div id="sec-5" class="subsection"><h3>Genetic map construction and evaluation</h3><p id="p-14">We used the Lep-MAP2 (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Rastas <em>et al.</em> 2015</a>) for marker filtering, grouping and ordering. To be compatible with the input of this program, we set the maternal genotype to 1/2, the paternal genotype to 1/1, and missing genotype to 0/0. The haploid megagametophyte genotype at each locus was converted into a homozygous diploid as input format. The threshold of missing data were set to 20%. Subsequently, markers with significant segregation distortion based on chi-square test (<em>P</em> ≤ 0.05) were discarded. Only markers followed the expected 1:1 segregation ratio were used in map construction.</p><p id="p-15">The linkage map was constructed through three steps: “grouping”, “join singles” and “ordering”. Linkage groups (LGs) were identified by a logarithm of the odds (LOD) score threshold using the ‘SeparateChromosomes’ module, with the minimum LOD score set to 5. After an initial grouping of most of the markers into LGs, remaining loci can be added to the identified LGs using the ‘JoinSingles’ module with a LOD score set from 5 to 10. In the ordering step, markers in each LG were ordered repeatedly for 10 times to select the order with the highest likelihood. The initial recombination probability was set at 0.4. We checked for error prone loci (error rate &gt;0.3) inflating the ends of linkage groups, if present, they should be removed, and marker orders re-evaluated. Kosambi function was used to convert recombination value to map distance (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Kosambi 1944</a>). The length of each LG was summed to calculate the total length of the map. The generated linkage groups were evaluated with the profile of recombination rate and LOD. The heat maps of linkage relationships (LOD) were produced by CheckMatrix (<a href="http://www.atgc.org/XLinkage/MadMapper/">http://www.atgc.org/XLinkage/MadMapper/</a>). Finally, we used Genetic-Mapper (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bekaert 2016</a>) to plot the linkage map. From the full set of segregating markers, we identified the co-segregating markers (<em>i.e.</em>, markers labeled as “duplicates” in the Lep-MAP output file). A set of framework markers were selected by masking the duplicated co-segregating markers. These framework markers were re-ordered to construct the framework map. Synteny between the framework map and full-set markers map was analyzed using shinyCircos (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Yu <em>et al.</em> 2017</a>).</p><p id="p-16">To evaluate whether the mapped markers were randomly distributed, the linkage groups were divided into 10 cM blocks, and the frequency distribution of marker density per block was assessed. The observed frequencies of the number of markers per block were compared with the expected one from Poisson distribution, <em>P(x) = e<sup>μ</sup>μ<sup>x</sup>/x</em>!, where <em>x</em> is the number of markers per block and <em>μ</em> is the average marker density. Average marker density (<em>μ</em>) was used to calculate the expected binomial frequencies for each marker class per block interval for all the linkage groups.</p></div><div id="sec-6" class="subsection"><h3>Data availability</h3><p id="p-17">Raw sequences were deposited in the NCBI Sequence Read Archive under project accession number PRJNA510567. File S1 contains the sequence information of the 23,926 loci on the linkage map. File S2 contains the information of linkage map with 23,926 markers, the marker ID, position and the identified co-segregation markers marked as “duplicates”. Supplemental material available at FigShare: <a href="https://doi.org/10.25387/g3.9754400">https://doi.org/10.25387/g3.9754400</a>.</p></div></div><div class="section results" id="sec-7"><h2 class="">Results</h2><div id="sec-8" class="subsection"><h3>Genome-wide polymorphic marker detection</h3><p id="p-18">A total of 366.5 million pair-end reads were obtained from the 140 samples (139 haploid megagametophytes and one diploid tissue). The high-quality base (Q score &gt;30) ratio was 88.4% and guanine-cytosine (GC) content was 35.0%. Among the 139 haploid megagametophytes, the number of reads recovered varied from a minimum of 1,479,008 to a maximum of 4,170,363, with an average of 2.5 million. The diploid sample obtained 18,058,470 reads. After one-to-one alignment, clustering and correction, 281.57 M high quality reads were assigned to 397,226 contigs (loci), of which 280,762 were detected in the mother tree with an average depth of 50×. The number of loci called per individual megagametophyte ranged from 173,010 to 261,952 with an average of 220,321. The average depth of coverage in the megagametophyte population was 11.3×. Among these loci, 108,683, 288,226, and 317 were polymorphic (27.4% of the total reconstructed loci), monomorphic, and repetitive, respectively. Of the 108,683 polymorphic loci, 45,959 loci matched the genotypes of the mother tree.</p></div><div id="sec-9" class="subsection"><h3>High-density linkage map</h3><p id="p-19">After further filtering away loci with missing rate &gt;20% and segregation distortion (chi-square test, <em>P ≤</em> 0.05), 23,926 of the 45,959 polymorphic loci maintained 1:1 segregation ratio in the megagametophyte population and were also found in the mother genotype. These markers were used in the map construction. In the step of grouping, all markers can be successfully classified into 11 linkage groups (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>), corresponding to the <em>P. orientalis</em> haploid chromosome number, without executing the ‘JoinSingles’ step. In the ordering step, we did not encounter markers with high error rates at the ends of each LGs. Therefore, the number of mapped markers was the same as the 23,926 1:1 segregating markers. We believe that the reason for this result is the rigorous control of missing rate, error rate and LOD in data filtering, which resulted in high quality markers that can be mapped and ordered without much ambiguity. The final map spanned 1,506.75 cM with the number of markers mapped on each linkage group varied from 1,847 (LG11) to 2,902 (LG01) (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). The length of the linkage groups ranged from 108.35 (LG08) to 176.34 cM (LG01), with an average of 136.98 cM. The distance between two adjacent markers on the linkage groups varied from 0 to 16.48 cM (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>; <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). The maximal gap in each group ranged from 4.9 (LG07) to 16 cM (LG10). Among the 23,915 marker intervals on the 11 LGs, 23,012 intervals were &lt;0.2 cM (96.2%) and 406 intervals were larger than 1 cM (1.7%).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/11/3663/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Distribution of full-set and framework markers along the P. orientalis 11 linkage groups. The full-set markers map (left) is composed of 23,926 loci with a total length of 1,506 cM. The framework markers map (right) is composed of 7,559 loci with an average spacing of 0.2 cM and a total length of 1,443 cM. In each linkage group, the black horizontal bars show the positions of the markers. The thickness of bar is proportional to the number of co-segregation markers in the same position." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1951452812" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Distribution of full-set and framework markers along the &lt;em&gt;P. orientalis&lt;/em&gt; 11 linkage groups. The full-set markers map (left) is composed of 23,926 loci with a total length of 1,506 cM. The framework markers map (right) is composed of 7,559 loci with an average spacing of 0.2 cM and a total length of 1,443 cM. In each linkage group, the black horizontal bars show the positions of the markers. The thickness of bar is proportional to the number of co-segregation markers in the same position.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/11/3663/F1.medium.gif" width="440" height="321"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/9/11/3663/F1.medium.gif" width="440" height="321"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/11/3663/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/11/3663/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494638" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-20" class="first-child">Distribution of full-set and framework markers along the <em>P. orientalis</em> 11 linkage groups. The full-set markers map (left) is composed of 23,926 loci with a total length of 1,506 cM. The framework markers map (right) is composed of 7,559 loci with an average spacing of 0.2 cM and a total length of 1,443 cM. In each linkage group, the black horizontal bars show the positions of the markers. The thickness of bar is proportional to the number of co-segregation markers in the same position.</p><div class="sb-div caption-clear"></div></div></div><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/494769/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/494769/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Summary of the <em>P. orientalis</em> linkage map. The full-set of 23,926 segregating markers were ordered into 11 linkage groups. The framework map was constructed using 7,559 markers by masking co-segregating markers</span><div class="sb-div caption-clear"></div></div></div><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/11/3663/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Spatial distribution of marker density along each LG. A) Distribution of the map distance between two adjacent mapped markers. B) distribution of framework and full-set markers along 10 cM sliding windows of each LG. C) Poisson distribution of the observed and expected frequencies of markers distributed at 10 cM interval." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1951452812" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Spatial distribution of marker density along each LG. A) Distribution of the map distance between two adjacent mapped markers. B) distribution of framework and full-set markers along 10 cM sliding windows of each LG. C) Poisson distribution of the observed and expected frequencies of markers distributed at 10 cM interval.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/11/3663/F2.medium.gif" width="440" height="334"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/9/11/3663/F2.medium.gif" width="440" height="334"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/11/3663/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/11/3663/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494683" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-21" class="first-child">Spatial distribution of marker density along each LG. A) Distribution of the map distance between two adjacent mapped markers. B) distribution of framework and full-set markers along 10 cM sliding windows of each LG. C) Poisson distribution of the observed and expected frequencies of markers distributed at 10 cM interval.</p><div class="sb-div caption-clear"></div></div></div><p id="p-22">Among the 23,926 mapped loci, 16,367 were classified as co-segregating loci. Lep-MAP2 identifies “duplicates” based on both map position and consistency in marker order among runs. Markers within a block can be ordered randomly between runs due to small differences in position. Only the consistent marker orders are identified as “duplicates”. We randomly selected one marker from each co-segregating bin and formed a set of 7,559 framework markers. We re-ordered and reconstructed linkage maps using these 7,559 framework markers. No duplicate co-segregation markers were found by the Lep-MAP algorithm in this set of markers. These markers were also grouped into 11 LGs with 416 to 953 loci per group (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>, <a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>). They covered a genome size of 1,442.78 cM, with an average map distance of 0.2 cM between adjacent markers. The sequences of the contigs and detailed information of the markers on each linkage map are available in File S1 and File S2. To compare the synteny between the framework map and the full-set markers’ map, the 7,559 markers in framework map were linked to their positions on the full map. We found highly consistent marker orders between the two maps, as shown in <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>. A near-perfect concordance between the two maps suggests good quality marker selection and map ordering.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/11/3663/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Synteny between the framework marker map and the full-set marker map. Lines connect the same marker on the two maps. Gray blocks on the left represent the 11 linkage groups of the framework map, whereas the 11 blocks on the right represent the full-set marker map." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1951452812" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Synteny between the framework marker map and the full-set marker map. Lines connect the same marker on the two maps. Gray blocks on the left represent the 11 linkage groups of the framework map, whereas the 11 blocks on the right represent the full-set marker map.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/11/3663/F3.medium.gif" width="440" height="436"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/9/11/3663/F3.medium.gif" width="440" height="436"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/11/3663/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/11/3663/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494779" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-23" class="first-child">Synteny between the framework marker map and the full-set marker map. Lines connect the same marker on the two maps. Gray blocks on the left represent the 11 linkage groups of the framework map, whereas the 11 blocks on the right represent the full-set marker map.</p><div class="sb-div caption-clear"></div></div></div><p id="p-24">We examined the distribution of markers in each linkage group and plotted them in 10 cM intervals along the length of each LG. We found a nearly full coverage of markers on each LG, with few regions of low-marker density scattered among the LGs, especially in LG10, which showed a large gap in the region around 40 – 60 cM (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2B</a>). The region with the highest marker density was in the window of 50 – 60 cM on LG06, which contained 525 loci (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2B</a>). The distribution of markers in intervals of 10 cM was in accordance with Poisson distribution, indicating that the markers were randomly distributed in the 11 linkage groups, except for the high end where blocks contained more than 100 markers (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2C</a>).</p><p id="p-25">The linkage relationship among markers along each LG was visualized as heatmaps of LOD (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>), with higher LOD score indicating tighter linkage. The linkage relationship along each LG differed among LGs with more even spread in <em>e.g.</em>, LG01 and LG03, but more clustered in <em>e.g.</em>, LG06, LG08 and LG10. This linkage pattern is largely reflective of the marker density distribution on each LG.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/11/3663/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Heat map of the marker linkage relationships in each LG. The plots show the LOD relationship among loci. X- and Y-axes represent the polymorphic ddRAD loci ordered by the linkage map." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1951452812" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Heat map of the marker linkage relationships in each LG. The plots show the LOD relationship among loci. X- and Y-axes represent the polymorphic ddRAD loci ordered by the linkage map.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/11/3663/F4.medium.gif" width="440" height="332"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/9/11/3663/F4.medium.gif" width="440" height="332"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/11/3663/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/11/3663/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494691" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-26" class="first-child">Heat map of the marker linkage relationships in each LG. The plots show the LOD relationship among loci. X- and Y-axes represent the polymorphic ddRAD loci ordered by the linkage map.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-10"><h2 class="">Discussion</h2><p id="p-27">We developed a high-density linkage map for <em>P. orientalis</em> as a first step toward gaining an insight into the species’ genome organization. The linkage map contains &gt;20K segregating loci covering the entire genome. We demonstrated the efficiency of ddRAD-seq method for genome-wide polymorphism identification in <em>P. orientalis</em>, and its utility and potential for other species lacking reference genome for generating large number of markers for genetic studies. In the present study, we assembled 397,226 loci of which 27.4% (108,683) were polymorphic and 72.6% monomorphic in the studied single-tree megagametophyte population. After strict filtering (<em>e.g.</em>, in agreement with maternal genotype, 1:1 segregation ratio and discarding markers with &gt;20% missing rate), we obtained 23,926 high-quality loci for linkage map construction.</p><p id="p-28">The linkage map for <em>P. orientalis</em> comprised 11 LGs, representing the haploid chromosome number of the species (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Sax and Sax 1933</a>), with 23,926 markers covering 1,506 cM. This nearly saturated linkage map is the first genetic map for <em>P. orientalis</em>, and to our knowledge is among the densest linkage maps available for conifer species. For example, maps with 21,056 markers spanning 3,556 cM for <em>Picea abies</em> (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Bernhardsson <em>et al.</em> 2019</a>), 20,655 markers spanning 1,192 cM for <em>Pinus balfouriana</em> (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Friedline <em>et al.</em> 2015</a>), 2,841 markers spanning 1,637 cM for <em>Pinus taeda</em> (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Neves <em>et al.</em> 2014</a>), 2,560 markers spanning 1,266 cM for <em>Cryptomeria japonica</em> (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Moriguchi <em>et al.</em> 2016</a>), and 4,284 markers spanning 1,033 cM for <em>Callitris glaucophylla</em> (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Sakaguchi <em>et al.</em> 2015</a>). The average marker interval on the current map of <em>P. orientalis</em> is 0.2 cM, which is shorter than most of the maps aforementioned but comparable to the <em>P. abies</em> map (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/11/3663/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The comparison of average map distances among high-density linkage maps in conifer species. A) The average distance among adjacent markers mapped for P. orientalis and two other Cupressaceae species. B) The average distance among adjacent markers mapped for three Pinaceae species." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1951452812" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The comparison of average map distances among high-density linkage maps in conifer species. A) The average distance among adjacent markers mapped for &lt;em&gt;P. orientalis&lt;/em&gt; and two other Cupressaceae species. B) The average distance among adjacent markers mapped for three Pinaceae species.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/11/3663/F5.medium.gif" width="440" height="169"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/9/11/3663/F5.medium.gif" width="440" height="169"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/11/3663/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/11/3663/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494752" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-29" class="first-child">The comparison of average map distances among high-density linkage maps in conifer species. A) The average distance among adjacent markers mapped for <em>P. orientalis</em> and two other Cupressaceae species. B) The average distance among adjacent markers mapped for three Pinaceae species.</p><div class="sb-div caption-clear"></div></div></div><p id="p-30">Among the high-density linkage maps available to date, three maps, <em>P. orientalis</em>, <em>C. glaucophylla</em>, <em>C. japonica</em> are representatives of the Cupressaceae family. The <em>C. glaucophylla</em> map is constructed using a similar technique, RAD-seq, on megagametophytes (<a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Sakaguchi <em>et al.</em> 2015</a>). Comparison of the present <em>P. orientalis</em> map to that of <em>C. glaucophylla</em> showed that we had more than 5× mapped markers, and 45.79% longer map length (1506 <em>vs.</em>. 1033 cM). The large difference in the number of markers recovered between the two studies is likely due to the larger sample size and deeper sequence coverage of <em>P. orientalis</em> that permitted capturing more segregating loci. The map for <em>C. japonica</em> is based on SNP markers from expressed sequence tag (EST) (<a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Moriguchi <em>et al.</em> 2016</a>). A common pattern emerged from the <em>C. japonica</em> and the present study is the presence of increased marker density in the center region of each LG, which corresponds to the centromere of each chromosome (<a id="xref-ref-31-3" class="xref-bibr" href="#ref-31">Moriguchi <em>et al.</em> 2016</a>, <a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2B</a>), suggesting a large number of co-segregating loci in these regions. The strikingly marker-rich regions in linkage groups may be due to the nonrandom digestion of restriction enzymes and the uneven marker polymorphism and recombination rates along the chromosomes (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Petes 2001</a>). Low recombination rate can lead to distorted genetic distances and marker clustering on genetic map (<a id="xref-ref-44-3" class="xref-bibr" href="#ref-44">Sakaguchi <em>et al.</em> 2015</a>). The high degree of marker clustering around the center of each LGs and the detection of this pattern in <em>P. orientalis</em>, <em>C. glaucophylla</em>, and <em>C. japonica</em> genetic maps suggest the presence of symmetrical karyotypes in the Cupressaceae family. Karyotype analysis of <em>P</em>. <em>orientalis</em> established its n = 11 chromosomes, of which two are likely satellite chromosomes, where secondary constrictions are located on their long arms (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Li and Hsu 1984</a>). On the constructed linkage groups, we detected large gaps in LG10 and LG09 (<a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2B</a>), which we postulate that it could be secondary constriction regions. Although this assertion cannot be confirmed without a physical map, either way, these regions could act as landmark references for further studies.</p><p id="p-31">High-density genetic maps inevitably generate many co-segregating markers (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Iehisa <em>et al.</em> 2014</a>; <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Friedline <em>et al.</em> 2015</a>; <a id="xref-ref-44-4" class="xref-bibr" href="#ref-44">Sakaguchi <em>et al.</em> 2015</a>). The effect of co-segregation markers on linkage maps has received less attention. More mapped markers could capture more recombination events. On the other hand, an excess of co-segregation markers could lead to substantial map expansion due to errors in the data. A study on wheat investigated the effect of segregation markers on map length, which showed that with 10% and 80% co-segregation markers, the map length can be inflated by <em>ca</em>. 4% and 12–16%, respectively (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">N’Diaye <em>et al.</em> 2017</a>). In the present study, we generated two maps based on full-set (23,926) and framework markers (7,559), respectively. The map of the full-set markers contained 3× more mapped markers than the framework marker map. The map length, however, is only 4.4% longer than the framework map (1,506 <em>vs.</em>. 1,443 cM, <a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>). This result suggests a near saturation of the framework marker map with 7,559 markers. Framework maps have been used in QTL detection (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Shao <em>et al.</em> 2015</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Liu <em>et al.</em> 2016</a>), and as a reference to calculate the genetic distances between markers (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Ren <em>et al.</em> 2012</a>). Linkage maps can further provide valuable information to maker-based population genetic inferences, such as population structure, diversity and relatedness (<a id="xref-ref-44-5" class="xref-bibr" href="#ref-44">Sakaguchi <em>et al.</em> 2015</a>). In the present study, the map with full-set markers allowed for visualization of uneven marker distribution along each LG and facilitated the detection of centromeres and the genome regions with strong or weak clustering of co-segregation markers. These important features of genome organization cannot be properly detected using framework markers alone.</p><p id="p-32">While the generation of linkage maps from RAD-seq is a complex endeavor due to the inherent stochasticity and error-prone nature of these data, we successfully recovered 108,683 polymorphic loci in the haploid megagametophyte tissues of a single family, and ordered &gt;20K loci into 11 LGs. It should be emphasized that the development of linkage maps based on the relatively low-cost RAD sequencing has unique technical considerations including a substantial amount of missing data as well as a non-uniform distribution of reads over sequenced regions (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Beissinger <em>et al.</em> 2013</a>). Caution is also needed as low frequency of genotyping error (≤3%) appears as double or multiple recombinants tend to inflate the map length and reduce marker order power (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Buetow 1991</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Hackett and Broadfoot 2003</a>). Collectively, all elements such as DNA quality, the choice of restriction enzyme, size selection, and sequencing depth can affect the missing rate. In the present study, we performed <em>in silico</em> digestion analyses to aid the selection of restriction enzymes, and <em>Eco</em>RV and <em>Sca</em>I were selected due to their efficiency and the acceptable read depth. The diploid sample of the mother tree which was used as a genotype reference was sequenced at a high coverage depth (50×) to assist genotype calls in megagametophytes. Our rigorous standard for data filtering (eliminating markers with &gt;20% missing data) and discarding of markers with segregation distortion was effective in reducing the inevitable influence of data error and provided reliable markers for constructing our linkage map. Additionally, the use of a single-tree megagametophyte tissue provided large sample size (n = 139) for correctly detecting polymorphisms; however, we were not able to capitalize on the advantage of using megagametophyte tissue from multiple seed donors which undoubtedly allowed observing markers segregation at other loci.</p><p id="p-33">Cupressaceae genetic and genomic studies are urgently needed owing to their ecological, economic, forestry, and horticulture importance. We believe the approach adopted in the present study is a sensible stop-gap approach for providing high quality markers distributed throughout the genome. These markers will be instrumental in assessing the species genetic diversity, development of conservation strategies, association genetics, genomic selection, and ecological genetics studies. The developed genetic map is expected to assist in scaffolds orienting process of the genome assembly of <em>P. orientalis</em> and its related Cupressaceae species.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-34">Genomic data processing and analyses were performed using resources provided by the Swedish National Infrastructure for Computing (SNIC), through the High Performance Computing Centre North (HPC2N). This study was supported by the National Natural Science Foundation of China (No. 31670664) and by the Fundamental Research Funds for the Central Universities (No. 2018BLCB08).</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-2"><p id="p-35">Supplemental material available at FigShare: <a href="https://doi.org/10.25387/g3.9754400">https://doi.org/10.25387/g3.9754400</a>.</p></li><li class="fn" id="fn-3"><p id="p-36"><em>Communicating editor: D. St.Clair</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2019-04-24"><span class="received-label">Received </span>April 24, 2019.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2019-09-05"><span class="accepted-label">Accepted </span>September 5, 2019.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2019 Jin <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-web" id="cit-9.11.3663.1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Andrews</span>,  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> FastQC: A quality control tool for high throughput sequence data. Available at: <a href="http://www.bioinformatics.babraham.ac.uk/projects/fastqc/">http://www.bioinformatics.babraham.ac.uk/projects/fastqc/</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3663.2" data-doi="10.1371/journal.pone.0003376"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Baird</span>,  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. D.</span> <span class="cit-name-surname">Etter</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. S.</span> <span class="cit-name-surname">Atwood</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. C.</span> <span class="cit-name-surname">Currey</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. L.</span> <span class="cit-name-surname">Shiver</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Rapid SNP discovery and genetic mapping using sequenced RAD markers.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">e3376</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0003376</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DBaird%26rft.auinit1%253DN.%2BA.%26rft.volume%253D3%26rft.issue%253D10%26rft.spage%253De3376%26rft.epage%253De3376%26rft.atitle%253DRapid%2BSNP%2Bdiscovery%2Band%2Bgenetic%2Bmapping%2Busing%2Bsequenced%2BRAD%2Bmarkers.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0003376%26rft_id%253Dinfo%253Apmid%252F18852878%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0003376&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18852878&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3663.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3663.3" data-doi="10.1534/genetics.112.147710"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Beissinger</span>,  <span class="cit-name-given-names">T. M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. N.</span> <span class="cit-name-surname">Hirsch</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. S.</span> <span class="cit-name-surname">Sekhon</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">Foerster</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">Johnson</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Marker density and read depth for genotyping populations using genotyping-by-sequencing.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">193</span>: <span class="cit-fpage">1073</span>–<span class="cit-lpage">1081</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.112.147710</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.147710%26rft_id%253Dinfo%253Apmid%252F23410831%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5My80LzEwNzMiO3M6NDoiYXRvbSI7czoxOToiL2dnZy85LzExLzM2NjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-web" id="cit-9.11.3663.4"><div class="cit-metadata unstructured">Bekaert, M., 2016 Genetic-Mapper: vectorial genetic map drawer. F1000Research. Available at: <a href="https://github.com/pseudogene/genetic-mapper">https://github.com/pseudogene/genetic-mapper</a>.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3663.5" data-doi="10.1534/g3.118.200840"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bernhardsson</span>,  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Vidalis</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Wang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. G.</span> <span class="cit-name-surname">Scofield</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Schiffthaler</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">An ultra-dense haploid genetic map for evaluating the highly fragmented genome assembly of Norway spruce (<em>Picea abies</em>).</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">1623</span>–<span class="cit-lpage">1632</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.118.200840</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.118.200840%26rft_id%253Dinfo%253Apmid%252F30898899%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjg6IjkvNS8xNjIzIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMS8zNjYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3663.6" data-doi="10.1093/bioinformatics/btt178"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Birol</span>,  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Raymond</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. D.</span> <span class="cit-name-surname">Jackman</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Pleasance</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Coope</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Assembling the 20 Gb white spruce (<em>Picea glauca</em>) genome from whole-genome shotgun sequencing data.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">1492</span>–<span class="cit-lpage">1497</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btt178</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtt178%26rft_id%253Dinfo%253Apmid%252F23698863%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btt178&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23698863&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3663.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000320065400003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3663.7" data-doi="10.1093/bioinformatics/btu170"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bolger</span>,  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Lohse</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Usadel</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Trimmomatic: a flexible trimmer for Illumina sequence data.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">2114</span>–<span class="cit-lpage">2120</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btu170</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu170%26rft_id%253Dinfo%253Apmid%252F24695404%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btu170&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24695404&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3663.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000340049100004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3663.8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Buetow</span>,  <span class="cit-name-given-names">K. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">Influence of aberrant observations on high-resolution linkage analysis outcomes.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">985</span>–<span class="cit-lpage">994</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DBuetow%26rft.auinit1%253DK.%2BH.%26rft.volume%253D49%26rft.issue%253D5%26rft.spage%253D985%26rft.epage%253D994%26rft.atitle%253DInfluence%2Bof%2Baberrant%2Bobservations%2Bon%2Bhigh-resolution%2Blinkage%2Banalysis%2Boutcomes.%26rft_id%253Dinfo%253Apmid%252F1928104%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=1928104&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3663.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1991GM87300010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3663.9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Burleign</span>,  <span class="cit-name-given-names">J. G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W. B.</span> <span class="cit-name-surname">Barbazuk</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">Davis</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. M.</span> <span class="cit-name-surname">Morse</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. S.</span> <span class="cit-name-surname">Soltis</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Exploring diversification and genome size evolution in extant gymnosperms through phylogenetic synthesis.</span> <abbr class="cit-jnl-abbrev">J. Bot.</abbr> <span class="cit-vol">2012</span>: <span class="cit-fpage">292857</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBot.%26rft.volume%253D2012%26rft.spage%253D292857%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3663.10" data-doi="10.1534/g3.117.040055"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crepeau</span>,  <span class="cit-name-given-names">M. W.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. H.</span> <span class="cit-name-surname">Langley</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. A.</span> <span class="cit-name-surname">Stevens</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">From pine cones to read clouds: rescaffolding the megagenome of sugar pine (<em>Pinus lambertiana</em>).</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">1563</span>–<span class="cit-lpage">1568</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.117.040055</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.117.040055%26rft_id%253Dinfo%253Apmid%252F28341701%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjg6IjcvNS8xNTYzIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMS8zNjYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3663.11" data-doi="10.1111/mec.12354"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Catchen</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. A.</span> <span class="cit-name-surname">Hohenlohe</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Bassham</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Amores</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">W. A.</span> <span class="cit-name-surname">Cresko</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Stacks: an analysis tool set for population genomics.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">3124</span>–<span class="cit-lpage">3140</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/mec.12354</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D22%26rft.spage%253D3124%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.12354%26rft_id%253Dinfo%253Apmid%252F23701397%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.12354&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23701397&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3663.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000319675300022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3663.12" data-doi="10.1038/nrg3012"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Davey</span>,  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. A.</span> <span class="cit-name-surname">Hohenlohe</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. D.</span> <span class="cit-name-surname">Etter</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. Q.</span> <span class="cit-name-surname">Boone</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">Catchen</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genome-wide genetic marker discovery and genotyping using next-generation sequencing.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">499</span>–<span class="cit-lpage">510</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg3012</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DDavey%26rft.auinit1%253DJ.%2BW.%26rft.volume%253D12%26rft.issue%253D7%26rft.spage%253D499%26rft.epage%253D510%26rft.atitle%253DGenome-wide%2Bgenetic%2Bmarker%2Bdiscovery%2Band%2Bgenotyping%2Busing%2Bnext-generation%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3012%26rft_id%253Dinfo%253Apmid%252F21681211%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3012&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21681211&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3663.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3663.13" data-doi="10.1007/s11295-015-0866-x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Friedline</span>,  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. M.</span> <span class="cit-name-surname">Lind</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. M.</span> <span class="cit-name-surname">Hobson</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. E.</span> <span class="cit-name-surname">Harwood</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. D.</span> <span class="cit-name-surname">Mix</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">The genetic architecture of local adaptation I: The genomic landscape of foxtail pine (<em>Pinus balfouriana</em> Grev. &amp; Balf.) as revealed from a high-density linkage map.</span> <abbr class="cit-jnl-abbrev">Tree Genet. Genomes</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">15</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s11295-015-0866-x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTree%2BGenet.%2BGenomes%26rft.volume%253D11%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11295-015-0866-x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11295-015-0866-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3663.14" data-doi="10.1093/nar/gkl828"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gregory</span>,  <span class="cit-name-given-names">T. R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. A.</span> <span class="cit-name-surname">Nicol</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Tamm</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Kullman</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Kullman</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Eukaryotic genome size databases.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">D332</span>–<span class="cit-lpage">D338</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkl828</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkl828%26rft_id%253Dinfo%253Apmid%252F17090588%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkl828&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17090588&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3663.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000243494600069&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3663.15" data-doi="10.1038/sj.hdy.6800173"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hackett</span>,  <span class="cit-name-given-names">C. A.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">L. B.</span> <span class="cit-name-surname">Broadfoot</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Effects of genotyping errors, missing values and segregation distortion in molecular marker data on the construction of linkage maps.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">90</span>: <span class="cit-fpage">33</span>–<span class="cit-lpage">38</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/sj.hdy.6800173</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.stitle%253DHeredity%26rft.aulast%253DHackett%26rft.auinit1%253DC.%2BA.%26rft.volume%253D90%26rft.issue%253D1%26rft.spage%253D33%26rft.epage%253D38%26rft.atitle%253DEffects%2Bof%2Bgenotyping%2Berrors%252C%2Bmissing%2Bvalues%2Band%2Bsegregation%2Bdistortion%2Bin%2Bmolecular%2Bmarker%2Bdata%2Bon%2Bthe%2Bconstruction%2Bof%2Blinkage%2Bmaps.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsj.hdy.6800173%26rft_id%253Dinfo%253Apmid%252F12522423%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/sj.hdy.6800173&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12522423&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3663.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000181165800007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3663.16" data-doi="10.1534/g3.116.036350"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hollenbeck</span>,  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. S.</span> <span class="cit-name-surname">Portnoy</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Wetzel</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. A.</span> <span class="cit-name-surname">Sherwood</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. B.</span> <span class="cit-name-surname">Samollow</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Linkage mapping and comparative genomics of red drum (<em>Sciaenops Ocellatus</em>) using next-generation sequencing.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">843</span>–<span class="cit-lpage">850</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.116.036350</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.116.036350%26rft_id%253Dinfo%253Apmid%252F28122951%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjc6IjcvMy84NDMiO3M6NDoiYXRvbSI7czoxOToiL2dnZy85LzExLzM2NjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3663.17" data-doi="10.1371/journal.pone.0148985"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>,  <span class="cit-name-given-names">X. G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Liu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Jin</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y. Q.</span> <span class="cit-name-surname">Sun</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Li</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title"><em>De novo</em> transcriptome assembly and characterization for the widespread and stress-tolerant conifer <em>Platycladus orientalis</em>.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">e0148985</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0148985</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D11%26rft.spage%253De0148985%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0148985%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0148985&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3663.18" data-doi="10.1371/journal.pone.0132326"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>,  <span class="cit-name-given-names">X. G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Jin</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X. R.</span> <span class="cit-name-surname">Wang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. F.</span> <span class="cit-name-surname">Mao</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Li</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Predicting impacts of future climate change on the distribution of the widespread conifer <em>Platycladus orientalis</em>.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">e0132326</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0132326</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D10%26rft.spage%253De0132326%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0132326%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0132326&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3663.19" data-doi="10.1139/cjfr-2018-0077"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>,  <span class="cit-name-given-names">L. S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Song</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y. Q.</span> <span class="cit-name-surname">Sun</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Q.</span> <span class="cit-name-surname">Gao</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. Q.</span> <span class="cit-name-surname">Jiao</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Pollination dynamics in a <em>Platycladus orientalis</em> seed orchard as revealed by partial pedigree reconstruction.</span> <abbr class="cit-jnl-abbrev">Can. J. For. Res.</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">952</span>–<span class="cit-lpage">957</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1139/cjfr-2018-0077</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCan.%2BJ.%2BFor.%2BRes.%26rft.volume%253D48%26rft.spage%253D952%26rft_id%253Dinfo%253Adoi%252F10.1139%252Fcjfr-2018-0077%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1139/cjfr-2018-0077&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3663.20" data-doi="10.1093/dnares/dsu020"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Iehisa</span>,  <span class="cit-name-given-names">J. C. M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Ohno</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Kimura</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Enoki</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Nishimura</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A high-density genetic map with array-based markers facilitates structural and quantitative trait locus analyses of the common wheat genome.</span> <abbr class="cit-jnl-abbrev">DNA Res.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">555</span>–<span class="cit-lpage">567</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/dnares/dsu020</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDNA%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fdnares%252Fdsu020%26rft_id%253Dinfo%253Apmid%252F24972598%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/dnares/dsu020&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24972598&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3663.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3663.21" data-doi="10.1038/srep34821"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jin</span>,  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Ma</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Wang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X. G.</span> <span class="cit-name-surname">Hu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L. S.</span> <span class="cit-name-surname">Huang</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genetic evaluation of the breeding population of a valuable reforestation conifer <em>Platycladus orientalis</em> (Cupressaceae).</span> <abbr class="cit-jnl-abbrev">Sci. Rep.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">34821</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/srep34821</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BRep.%26rft.volume%253D6%26rft.spage%253D34821%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsrep34821%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/srep34821&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3663.22" data-doi="10.1186/1471-2164-15-233"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kai</span>,  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Nomura</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Fujiwara</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Nakamura</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Yasuike</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A ddRAD-based genetic map and its integration with the genome assembly of Japanese eel (<em>Anguilla japonica</em>) provides insights into genome evolution after the teleost-specific genome duplication.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">233</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2164-15-233</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D15%26rft.spage%253D233%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-15-233%26rft_id%253Dinfo%253Apmid%252F24669946%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-15-233&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24669946&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3663.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3663.23" data-doi="10.1101/gr.229202. Article published online before March 2002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kent</span>,  <span class="cit-name-given-names">W. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">BLAT: The BLAST-like alignment tool.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">656</span>–<span class="cit-lpage">664</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.229202</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.229202.%2BArticle%2Bpublished%2Bonline%2Bbefore%2BMarch%2B2002%26rft_id%253Dinfo%253Apmid%252F11932250%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjEyLzQvNjU2IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMS8zNjYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3663.24" data-doi="10.1186/s12864-017-3765-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Konar</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">O.</span> <span class="cit-name-surname">Choudhury</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Bullis</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Fiedler</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">Kruser</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">High-quality genetic mapping with ddRADseq in the non-model tree <em>Quercus rubra</em>.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">417</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12864-017-3765-8</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D18%26rft.spage%253D417%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864-017-3765-8%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12864-017-3765-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3663.25" data-doi="10.1111/j.1469-1809.1943.tb02321.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kosambi</span>,  <span class="cit-name-given-names">D. D.</span></span></li></ol><cite>, <span class="cit-pub-date">1944</span> <span class="cit-article-title">The estimation of map distances from recombination values.</span> <abbr class="cit-jnl-abbrev">Ann. Eugen.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">172</span>–<span class="cit-lpage">175</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1469-1809.1943.tb02321.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BEugen.%26rft.volume%253D12%26rft.spage%253D172%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1469-1809.1943.tb02321.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1469-1809.1943.tb02321.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3663.26" data-doi="10.1038/srep30009"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Du</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Wen</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Mapping the climatic suitable habitat of oriental arborvitae (<em>Platycladus orientalis</em>) for introduction and cultivation at a global scale.</span> <abbr class="cit-jnl-abbrev">Sci. Rep.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">30009</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/srep30009</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BRep.%26rft.volume%253D6%26rft.spage%253D30009%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsrep30009%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/srep30009&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3663.27"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">L.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Hsu</span></span></li></ol><cite>, <span class="cit-pub-date">1984</span> <span class="cit-article-title">Karyotype analyses in <em>Platycladus orientalis</em> and <em>Fokienia hodginsii</em>.</span> <abbr class="cit-jnl-abbrev">Acta Botanica Yunnanica</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">447</span>–<span class="cit-lpage">451</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DActa%2BBotanica%2BYunnanica%26rft.volume%253D6%26rft.spage%253D447%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3663.28" data-doi="10.1016/j.hpj.2016.11.003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Liu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Cheng</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Liang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Wang</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">A high density linkage map facilitates QTL mapping of flowering time in <em>Brassica rapa</em>.</span> <abbr class="cit-jnl-abbrev">Hortic. Plant J.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">217</span>–<span class="cit-lpage">223</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.hpj.2016.11.003</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHortic.%2BPlant%2BJ.%26rft.volume%253D2%26rft.spage%253D217%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.hpj.2016.11.003%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.hpj.2016.11.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-book" id="cit-9.11.3663.29"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>,  <span class="cit-name-given-names">M.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Walsh</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-source">Genetics and analysis of quantitative traits</span>, <span class="cit-publ-name">Sinauer Associates</span>, <span class="cit-publ-loc">Sunderland, MA</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3663.30" data-doi="10.1038/ncomms14363"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meier</span>,  <span class="cit-name-given-names">J. I.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. A.</span> <span class="cit-name-surname">Marques</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Mwaiko</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. E.</span> <span class="cit-name-surname">Wagner</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Excoffier</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Ancient hybridization fuels rapid cichlid fish adaptive radiations.</span> <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">14363</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ncomms14363</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun.%26rft.volume%253D8%26rft.spage%253D14363%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms14363%26rft_id%253Dinfo%253Apmid%252F28186104%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms14363&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28186104&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3663.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3663.31" data-doi="10.1007/s11295-016-1011-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Moriguchi</span>,  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Uchiyama</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Ueno</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Ujino-Ihara</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Matsumoto</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">A high-density linkage map with 2560 markers and its application for the localization of the male-sterile genes <em>ms3</em> and <em>ms4</em> in <em>Cryptomeria japonica</em> D.</span> <abbr class="cit-jnl-abbrev">Don. Tree Genet. Genome</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">57</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s11295-016-1011-1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDon.%2BTree%2BGenet.%2BGenome%26rft.volume%253D12%26rft.spage%253D57%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11295-016-1011-1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11295-016-1011-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3663.32" data-doi="10.3389/fpls.2017.01434"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">N’Diaye</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. K.</span> <span class="cit-name-surname">Haile</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. B.</span> <span class="cit-name-surname">Fowler</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Ammar</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. J.</span> <span class="cit-name-surname">Pozniak</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Effect of Co-segregating markers on high-density genetic maps and prediction of map expansion using machine learning algorithms.</span> <abbr class="cit-jnl-abbrev">Front. Plant Sci.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">1434</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fpls.2017.01434</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BPlant%2BSci.%26rft.volume%253D8%26rft.spage%253D1434%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffpls.2017.01434%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fpls.2017.01434&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3663.33" data-doi="10.1534/g3.117.300078"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Neale</span>,  <span class="cit-name-given-names">D. B.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. E.</span> <span class="cit-name-surname">McGuire</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N. C.</span> <span class="cit-name-surname">Wheeler</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. A.</span> <span class="cit-name-surname">Stevens</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. W.</span> <span class="cit-name-surname">Crepeau</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">The Douglas-fir genome sequence reveals specialization of the photosynthetic apparatus in Pinaceae.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">3157</span>–<span class="cit-lpage">3167</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.117.300078</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.117.300078%26rft_id%253Dinfo%253Apmid%252F28751502%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjg6IjcvOS8zMTU3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMS8zNjYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3663.34" data-doi="10.1186/gb-2014-15-3-r59"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Neale</span>,  <span class="cit-name-given-names">D. B.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. L.</span> <span class="cit-name-surname">Wegrzyn</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. A.</span> <span class="cit-name-surname">Stevens</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. V.</span> <span class="cit-name-surname">Zimin</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Puiu</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Decoding the massive genome of loblolly pine using haploid DNA and novel assembly strategies.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">R59</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/gb-2014-15-3-r59</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D15%26rft.spage%253DR59%26rft.atitle%253DDECODING%2BTHE%2BMASSIVE%2BGENOME%2BOF%2BLOBLOLLY%2BPINE%2BUSING%2BHAPLOID%2BDNA%2BAND%2BNOVEL%2BASSEMBLY%2BSTRATEGIES.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2014-15-3-r59%26rft_id%253Dinfo%253Apmid%252F24647006%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2014-15-3-r59&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24647006&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3663.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3663.35" data-doi="10.1534/g3.113.008714"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Neves</span>,  <span class="cit-name-given-names">L. G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">Davis</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W. B.</span> <span class="cit-name-surname">Barbazuk</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Kirst</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A high-density gene map of loblolly pine (<em>Pinus taeda</em> L.) based on exome sequence capture genotyping.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">29</span>–<span class="cit-lpage">37</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.113.008714</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft.volume%253D4%26rft.spage%253D29%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.113.008714%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/g3.113.008714&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3663.36" data-doi="10.1038/nature12211"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nystedt</span>,  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N. R.</span> <span class="cit-name-surname">Street</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Wetterbom</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Zuccolo</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y. C.</span> <span class="cit-name-surname">Lin</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The Norway spruce genome sequence and conifer genome evolution.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">497</span>: <span class="cit-fpage">579</span>–<span class="cit-lpage">584</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature12211</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D497%26rft.spage%253D579%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature12211%26rft_id%253Dinfo%253Apmid%252F23698360%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature12211&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23698360&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3663.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000319556100035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3663.37" data-doi="10.1111/1755-0998.12342"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pan</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Wang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z. Y.</span> <span class="cit-name-surname">Pei</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Zhao</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Gao</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Optimization of the genotyping‐by‐sequencing strategy for population genomic analysis in conifers.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol. Resour.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">711</span>–<span class="cit-lpage">722</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/1755-0998.12342</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%2BResour.%26rft.volume%253D15%26rft.spage%253D711%26rft_id%253Dinfo%253Adoi%252F10.1111%252F1755-0998.12342%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/1755-0998.12342&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3663.38" data-doi="10.1007/s11295-018-1251-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Parchman</span>,  <span class="cit-name-given-names">T. L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. P.</span> <span class="cit-name-surname">Jahner</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. A.</span> <span class="cit-name-surname">Uckele</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L. M.</span> <span class="cit-name-surname">Galland</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. J.</span> <span class="cit-name-surname">Eckert</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">RADseq approaches and applications for forest tree genetics.</span> <abbr class="cit-jnl-abbrev">Tree Genet. Genomes</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">39</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s11295-018-1251-3</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTree%2BGenet.%2BGenomes%26rft.volume%253D14%26rft.spage%253D39%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11295-018-1251-3%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11295-018-1251-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3663.39" data-doi="10.1371/journal.pone.0037135"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Peterson</span>,  <span class="cit-name-given-names">B. K.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. N.</span> <span class="cit-name-surname">Weber</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. H.</span> <span class="cit-name-surname">Kay</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H. S.</span> <span class="cit-name-surname">Fisher</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">H. E.</span> <span class="cit-name-surname">Hoekstra</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Double digest RADseq: an inexpensive method for de novo SNP discovery and genotyping in model and non-model species.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e37135</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0037135</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DPeterson%26rft.auinit1%253DB.%2BK.%26rft.volume%253D7%26rft.issue%253D5%26rft.spage%253De37135%26rft.epage%253De37135%26rft.atitle%253DDouble%2Bdigest%2BRADseq%253A%2Ban%2Binexpensive%2Bmethod%2Bfor%2Bde%2Bnovo%2BSNP%2Bdiscovery%2Band%2Bgenotyping%2Bin%2Bmodel%2Band%2Bnon-model%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0037135%26rft_id%253Dinfo%253Apmid%252F22675423%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0037135&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22675423&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3663.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3663.40" data-doi="10.1038/35072078"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Petes</span>,  <span class="cit-name-given-names">T. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Meiotic recombination hot spots and cold spots.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">360</span>–<span class="cit-lpage">369</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/35072078</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DPetes%26rft.auinit1%253DT.%2BD.%26rft.volume%253D2%26rft.issue%253D5%26rft.spage%253D360%26rft.epage%253D369%26rft.atitle%253DMeiotic%2Brecombination%2Bhot%2Bspots%2Band%2Bcold%2Bspots.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F35072078%26rft_id%253Dinfo%253Apmid%252F11331902%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/35072078&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11331902&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3663.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000168467800013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3663.41" data-doi="10.1073/pnas.1114378109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pittermann</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. A.</span> <span class="cit-name-surname">Stuart</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. E.</span> <span class="cit-name-surname">Dawson</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Moreau</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Cenozoic climate change shaped the evolutionary ecophysiology of the Cupressaceae conifers.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">109</span>: <span class="cit-fpage">9647</span>–<span class="cit-lpage">9652</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.1114378109</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1114378109%26rft_id%253Dinfo%253Apmid%252F22628565%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTA5LzI0Lzk2NDciO3M6NDoiYXRvbSI7czoxOToiL2dnZy85LzExLzM2NjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3663.42" data-doi="10.1093/gbe/evv250"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rastas</span>,  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F. C.</span> <span class="cit-name-surname">Calboli</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Guo</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Shikano</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Merilä</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Construction of ultradense linkage maps with Lep-MAP2: stickleback F<sub>2</sub> recombinant crosses as an example.</span> <abbr class="cit-jnl-abbrev">Genome Biol. Evol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">78</span>–<span class="cit-lpage">93</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/gbe/evv250</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevv250%26rft_id%253Dinfo%253Apmid%252F26668116%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/gbe/evv250&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26668116&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3663.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3663.43" data-doi="10.1371/journal.pone.0029453"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ren</span>,  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Zhao</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Q.</span> <span class="cit-name-surname">Kou</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Jiang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Guo</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A high resolution genetic map anchoring scaffolds of the sequenced watermelon genome.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e29453</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0029453</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DRen%26rft.auinit1%253DY.%26rft.volume%253D7%26rft.issue%253D1%26rft.spage%253De29453%26rft.epage%253De29453%26rft.atitle%253DA%2Bhigh%2Bresolution%2Bgenetic%2Bmap%2Banchoring%2Bscaffolds%2Bof%2Bthe%2Bsequenced%2Bwatermelon%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0029453%26rft_id%253Dinfo%253Apmid%252F22247776%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0029453&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22247776&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3663.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3663.44" data-doi="10.1007/s11295-015-0944-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sakaguchi</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Sugino</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Tsumura</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Ito</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. D.</span> <span class="cit-name-surname">Crisp</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">High-throughput linkage mapping of Australian white cypress pine (<em>Callitris glaucophylla</em>) and map transferability to related species.</span> <abbr class="cit-jnl-abbrev">Tree Genet. Genomes</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">121</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s11295-015-0944-0</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTree%2BGenet.%2BGenomes%26rft.volume%253D11%26rft.spage%253D121%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11295-015-0944-0%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11295-015-0944-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3663.45" data-doi="10.5962/bhl.part.9959"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sax</span>,  <span class="cit-name-given-names">K.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">H. J.</span> <span class="cit-name-surname">Sax</span></span></li></ol><cite>, <span class="cit-pub-date">1933</span> <span class="cit-article-title">Chromosome number and morphology in the conifers.</span> <abbr class="cit-jnl-abbrev">J. Arnold Arbor.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">356</span>–<span class="cit-lpage">375</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.5962/bhl.part.9959</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BArnold%2BArbor.%26rft.volume%253D14%26rft.spage%253D356%26rft.atitle%253DJ%2BARNOLD%2BARBOR%26rft_id%253Dinfo%253Adoi%252F10.5962%252Fbhl.part.9959%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.5962/bhl.part.9959&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3663.46" data-doi="10.1093/dnares/dsv001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shao</span>,  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Niu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Rastas</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Liu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Xie</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genome-wide SNP identification for the construction of a high-resolution genetic map of Japanese flounder (<em>Paralichthys olivaceus</em>): applications to QTL mapping of <em>Vibrio anguillarum</em> disease resistance and comparative genomic analysis.</span> <abbr class="cit-jnl-abbrev">DNA Res.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">161</span>–<span class="cit-lpage">170</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/dnares/dsv001</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDNA%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fdnares%252Fdsv001%26rft_id%253Dinfo%253Apmid%252F25762582%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/dnares/dsv001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25762582&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3663.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3663.47" data-doi="10.1371/journal.pone.0058700"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>,  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Liu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Zhang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Li</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Liu</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">SLAF-seq: an efficient method of large-scale de novo SNP discovery and genotyping using high-throughput sequencing.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e58700</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0058700</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D8%26rft.spage%253De58700%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0058700%26rft_id%253Dinfo%253Apmid%252F23527008%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0058700&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23527008&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3663.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3663.48"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>,  <span class="cit-name-given-names">X.</span></span></li></ol><cite>, <span class="cit-pub-date">1986</span> <span class="cit-article-title">Geographical variation of <em>Platycladus orientalis</em>.</span> <abbr class="cit-jnl-abbrev">J. Beijing For. Univ.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">16</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBeijing%2BFor.%2BUniv.%26rft.volume%253D3%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3663.49" data-doi="10.1093/bioinformatics/btx763"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>,  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Ouyang</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Yao</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">shinyCircos: an R/Shiny application for interactive creation of Circos plot.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">1229</span>–<span class="cit-lpage">1231</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btx763</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.volume%253D34%26rft.spage%253D1229%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtx763%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btx763&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3663.50" data-doi="10.5194/acp-16-6949-2016"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Huang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Zhang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Shen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Zhao</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Three-North Shelter Forest Program contribution to long-term increasing trends of biogenic isoprene emissions in northern China.</span> <abbr class="cit-jnl-abbrev">Atmos. Chem. Phys.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">6949</span>–<span class="cit-lpage">6960</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.5194/acp-16-6949-2016</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAtmos.%2BChem.%2BPhys.%26rft.volume%253D16%26rft.spage%253D6949%26rft_id%253Dinfo%253Adoi%252F10.5194%252Facp-16-6949-2016%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.5194/acp-16-6949-2016&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3663.51" data-doi="10.1534/genetics.113.159715"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zimin</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. A.</span> <span class="cit-name-surname">Stevens</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. W.</span> <span class="cit-name-surname">Crepeau</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Holtz-Morris</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Koriabine</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Sequencing and assembly of the 22-Gb loblolly pine genome.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">196</span>: <span class="cit-fpage">875</span>–<span class="cit-lpage">890</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.113.159715</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DZimin%26rft.auinit1%253DA.%26rft.volume%253D196%26rft.issue%253D3%26rft.spage%253D875%26rft.epage%253D890%26rft.atitle%253DSequencing%2Band%2BAssembly%2Bof%2Bthe%2B22-Gb%2BLoblolly%2BPine%2BGenome%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.159715%26rft_id%253Dinfo%253Apmid%252F24653210%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk2LzMvODc1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMS8zNjYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/9/11/3663.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/11/3653" title="Regulation of the Neurospora Circadian Clock by the Spliceosome Component PRP5" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/11/3673" title="Dynamic Patterns of Transcript Abundance of Transposable Element Families in Maize" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/9/11"><span>Volume 9 Issue 11, November 2019</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/9/11.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/9/11.cover-source.jpg?itok=DjqkLGk9" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 9 (11)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/11/3663" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-piShnFWfmNXBV_OPhFbipPNiQ_SU5Y6wD91Swbhj56s" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/9/11/3663" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/9/11/3663" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/9/11/3663" class="active" data-icon-position="" data-hide-link-title="0">Genome-Wide Variant Identification and High-Density Genetic Map Construction Using RADseq for Platycladus orientalis (Cupressaceae)</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/494604" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-o42L3UpMiFJpsXq59HA9r2o5Q6xo9MxKPY1dUSzcFEE" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/11/3663.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/9/11/3663" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-o7Z0E74CqaHEE8Syp1e-oB35ekDe_Qr3UHmOVEvtHM4" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/9/11/3663" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/11/3663.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494604" id="node494604--2" data-pisa="ggg;9/11/3663" data-pisa-master="ggg;g3.119.400684" data-apath="/ggg/9/11/3663.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genome-Wide Variant Identification and High-Density Genetic Map Construction Using RADseq for <em>Platycladus orientalis</em> (Cupressaceae)</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-8275-6367" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Yuqing Jin</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-9437-3198" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Wei Zhao</span>, <span class="highwire-citation-author" data-delta="2">Shuai Nie</span>, <span class="highwire-citation-author" data-delta="3">Si-Si Liu</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0002-4887-8977" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Yousry A. El-Kassaby</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0002-6150-7046" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Xiao-Ru Wang</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0001-9735-8516" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jian-Feng Mao</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3663-3672;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400684 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494604--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Yuqing Jin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Beijing Advanced Innovation Center for Tree Breeding by Molecular Design, National Engineering Laboratory for Tree Breeding, Key Laboratory of Genetics and Breeding in Forest Trees and Ornamental Plants of Ministry of Education, College of Biological Sciences and Technology, Beijing Forestry University, Beijing, 100083, China,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yuqing%2BJin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jin%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AYuqing%2BJin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-8275-6367" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Yuqing Jin</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Wei Zhao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Beijing Advanced Innovation Center for Tree Breeding by Molecular Design, National Engineering Laboratory for Tree Breeding, Key Laboratory of Genetics and Breeding in Forest Trees and Ornamental Plants of Ministry of Education, College of Biological Sciences and Technology, Beijing Forestry University, Beijing, 100083, China,</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Ecology and Environmental Science, UPSC, Umeå University, SE-901 87 Umeå, Sweden, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Wei%2BZhao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhao%20W&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AWei%2BZhao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-9437-3198" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Wei Zhao</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Shuai Nie </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Beijing Advanced Innovation Center for Tree Breeding by Molecular Design, National Engineering Laboratory for Tree Breeding, Key Laboratory of Genetics and Breeding in Forest Trees and Ornamental Plants of Ministry of Education, College of Biological Sciences and Technology, Beijing Forestry University, Beijing, 100083, China,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Shuai%2BNie%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nie%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AShuai%2BNie%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Si-Si Liu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Beijing Advanced Innovation Center for Tree Breeding by Molecular Design, National Engineering Laboratory for Tree Breeding, Key Laboratory of Genetics and Breeding in Forest Trees and Ornamental Plants of Ministry of Education, College of Biological Sciences and Technology, Beijing Forestry University, Beijing, 100083, China,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Si-Si%2BLiu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Liu%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASi-Si%2BLiu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Yousry A. El-Kassaby </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Forest and Conservation Sciences, Faculty of Forestry, The University of British Columbia, Vancouver, British Columbia, V6T 1Z4, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yousry%2BA.%2BEl-Kassaby%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=El-Kassaby%20YA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AYousry%2BA.%2BEl-Kassaby%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-4887-8977" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Yousry A. El-Kassaby</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Xiao-Ru Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Beijing Advanced Innovation Center for Tree Breeding by Molecular Design, National Engineering Laboratory for Tree Breeding, Key Laboratory of Genetics and Breeding in Forest Trees and Ornamental Plants of Ministry of Education, College of Biological Sciences and Technology, Beijing Forestry University, Beijing, 100083, China,</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Ecology and Environmental Science, UPSC, Umeå University, SE-901 87 Umeå, Sweden, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xiao-Ru%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AXiao-Ru%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-6150-7046" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Xiao-Ru Wang</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Jian-Feng Mao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Beijing Advanced Innovation Center for Tree Breeding by Molecular Design, National Engineering Laboratory for Tree Breeding, Key Laboratory of Genetics and Breeding in Forest Trees and Ornamental Plants of Ministry of Education, College of Biological Sciences and Technology, Beijing Forestry University, Beijing, 100083, China,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jian-Feng%2BMao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mao%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJian-Feng%2BMao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-9735-8516" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jian-Feng Mao</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jianfeng.mao@bjfu.edu.cn" class="" data-icon-position="" data-hide-link-title="0">jianfeng.mao@bjfu.edu.cn</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/9/11/3663" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/9/11/3663&amp;title=Genome-Wide%20Variant%20Identification%20and%20High-Density%20Genetic%20Map%20Construction%20Using%20RADseq%20for%20Platycladus%20orientalis%20%28Cupressaceae%29" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/9/11/3663&amp;title=Genome-Wide%20Variant%20Identification%20and%20High-Density%20Genetic%20Map%20Construction%20Using%20RADseq%20for%20Platycladus%20orientalis%20%28Cupressaceae%29" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/9/11/3663&amp;title=Genome-Wide%20Variant%20Identification%20and%20High-Density%20Genetic%20Map%20Construction%20Using%20RADseq%20for%20Platycladus%20orientalis%20%28Cupressaceae%29" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/9/11/3663&amp;text=Genome-Wide%20Variant%20Identification%20and%20High-Density%20Genetic%20Map%20Construction%20Using%20RADseq%20for%20Platycladus%20orientalis%20%28Cupressaceae%29" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/9/11/3663&amp;title=Genome-Wide%20Variant%20Identification%20and%20High-Density%20Genetic%20Map%20Construction%20Using%20RADseq%20for%20Platycladus%20orientalis%20%28Cupressaceae%29" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/9/11/3663&amp;t=Genome-Wide%20Variant%20Identification%20and%20High-Density%20Genetic%20Map%20Construction%20Using%20RADseq%20for%20Platycladus%20orientalis%20%28Cupressaceae%29" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/9/11/3663&amp;title=Genome-Wide%20Variant%20Identification%20and%20High-Density%20Genetic%20Map%20Construction%20Using%20RADseq%20for%20Platycladus%20orientalis%20%28Cupressaceae%29" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/9/11/3663&amp;title=Genome-Wide%20Variant%20Identification%20and%20High-Density%20Genetic%20Map%20Construction%20Using%20RADseq%20for%20Platycladus%20orientalis%20%28Cupressaceae%29" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/494604/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494604" id="node494604--3" data-pisa="ggg;9/11/3663" data-pisa-master="ggg;g3.119.400684" data-apath="/ggg/9/11/3663.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genome-Wide Variant Identification and High-Density Genetic Map Construction Using RADseq for <em>Platycladus orientalis</em> (Cupressaceae)</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-8275-6367" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Yuqing Jin</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-9437-3198" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Wei Zhao</span>, <span class="highwire-citation-author" data-delta="2">Shuai Nie</span>, <span class="highwire-citation-author" data-delta="3">Si-Si Liu</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0002-4887-8977" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Yousry A. El-Kassaby</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0002-6150-7046" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Xiao-Ru Wang</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0001-9735-8516" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jian-Feng Mao</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3663-3672;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400684 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494604--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Yuqing Jin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Beijing Advanced Innovation Center for Tree Breeding by Molecular Design, National Engineering Laboratory for Tree Breeding, Key Laboratory of Genetics and Breeding in Forest Trees and Ornamental Plants of Ministry of Education, College of Biological Sciences and Technology, Beijing Forestry University, Beijing, 100083, China,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yuqing%2BJin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jin%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AYuqing%2BJin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-8275-6367" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Yuqing Jin</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Wei Zhao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Beijing Advanced Innovation Center for Tree Breeding by Molecular Design, National Engineering Laboratory for Tree Breeding, Key Laboratory of Genetics and Breeding in Forest Trees and Ornamental Plants of Ministry of Education, College of Biological Sciences and Technology, Beijing Forestry University, Beijing, 100083, China,</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Ecology and Environmental Science, UPSC, Umeå University, SE-901 87 Umeå, Sweden, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Wei%2BZhao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhao%20W&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AWei%2BZhao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-9437-3198" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Wei Zhao</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Shuai Nie </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Beijing Advanced Innovation Center for Tree Breeding by Molecular Design, National Engineering Laboratory for Tree Breeding, Key Laboratory of Genetics and Breeding in Forest Trees and Ornamental Plants of Ministry of Education, College of Biological Sciences and Technology, Beijing Forestry University, Beijing, 100083, China,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Shuai%2BNie%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nie%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AShuai%2BNie%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Si-Si Liu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Beijing Advanced Innovation Center for Tree Breeding by Molecular Design, National Engineering Laboratory for Tree Breeding, Key Laboratory of Genetics and Breeding in Forest Trees and Ornamental Plants of Ministry of Education, College of Biological Sciences and Technology, Beijing Forestry University, Beijing, 100083, China,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Si-Si%2BLiu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Liu%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASi-Si%2BLiu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Yousry A. El-Kassaby </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Forest and Conservation Sciences, Faculty of Forestry, The University of British Columbia, Vancouver, British Columbia, V6T 1Z4, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yousry%2BA.%2BEl-Kassaby%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=El-Kassaby%20YA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AYousry%2BA.%2BEl-Kassaby%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-4887-8977" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Yousry A. El-Kassaby</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Xiao-Ru Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Beijing Advanced Innovation Center for Tree Breeding by Molecular Design, National Engineering Laboratory for Tree Breeding, Key Laboratory of Genetics and Breeding in Forest Trees and Ornamental Plants of Ministry of Education, College of Biological Sciences and Technology, Beijing Forestry University, Beijing, 100083, China,</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Ecology and Environmental Science, UPSC, Umeå University, SE-901 87 Umeå, Sweden, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xiao-Ru%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AXiao-Ru%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-6150-7046" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Xiao-Ru Wang</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Jian-Feng Mao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Beijing Advanced Innovation Center for Tree Breeding by Molecular Design, National Engineering Laboratory for Tree Breeding, Key Laboratory of Genetics and Breeding in Forest Trees and Ornamental Plants of Ministry of Education, College of Biological Sciences and Technology, Beijing Forestry University, Beijing, 100083, China,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jian-Feng%2BMao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mao%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJian-Feng%2BMao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-9735-8516" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jian-Feng Mao</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jianfeng.mao@bjfu.edu.cn" class="" data-icon-position="" data-hide-link-title="0">jianfeng.mao@bjfu.edu.cn</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=3663&amp;rft.epage=3672&amp;rft.atitle=Genome-Wide%20Variant%20Identification%20and%20High-Density%20Genetic%20Map%20Construction%20Using%20RADseq%20for%20Platycladus%20orientalis%20%28Cupressaceae%29&amp;rft.volume=9&amp;rft.issue=11&amp;rft.date=2019-11-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Jin%2C+Yuqing&amp;rft.au=Zhao%2C+Wei&amp;rft.au=Nie%2C+Shuai&amp;rft.au=Liu%2C+Si-Si&amp;rft.au=El-Kassaby%2C+Yousry+A.&amp;rft.au=Wang%2C+Xiao-Ru&amp;rft.au=Mao%2C+Jian-Feng"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/494604/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/494604/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/494604/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/494604/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/494604/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/494604/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/494604/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/494604/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/494604/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/494604/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/494604/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/494604/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/9/11/3663" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-7" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-10" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/9/11/3663.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/9/11/3663.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/9/11/3663.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
