<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/9/11/3691" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/9/11/3691.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/9/11/3691.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/9/11/3691.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/9/11/3691.atom" />
<meta name="HW.pisa" content="ggg;9/11/3691" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Benchmarking Parametric and Machine Learning Models for Genomic Prediction of Complex Traits" />
<meta name="DC.Identifier" content="10.1534/g3.119.400498" />
<meta name="DC.Date" content="2019-11-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2019 Azodi et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="The usefulness of genomic prediction in crop and livestock breeding programs has prompted efforts to develop new and improved genomic prediction algorithms, such as artificial neural networks and gradient tree boosting. However, the performance of these algorithms has not been compared in a systematic manner using a wide range of datasets and models. Using data of 18 traits across six plant species with different marker densities and training population sizes, we compared the performance of six linear and six non-linear algorithms. First, we found that hyperparameter selection was necessary for all non-linear algorithms and that feature selection prior to model training was critical for artificial neural networks when the markers greatly outnumbered the number of training lines. Across all species and trait combinations, no one algorithm performed best, however predictions based on a combination of results from multiple algorithms ( i.e. , ensemble predictions) performed consistently well. While linear and non-linear algorithms performed best for a similar number of traits, the performance of non-linear algorithms vary more between traits. Although artificial neural networks did not perform best for any trait, we identified strategies ( i.e. , feature selection, seeded starting weights) that boosted their performance to near the level of other algorithms. Our results highlight the importance of algorithm selection for the prediction of trait values." />
<meta name="DC.Contributor" content="Christina B. Azodi" />
<meta name="DC.Contributor" content="Emily Bolger" />
<meta name="DC.Contributor" content="Andrew McCarren" />
<meta name="DC.Contributor" content="Mark Roantree" />
<meta name="DC.Contributor" content="Gustavo de los Campos" />
<meta name="DC.Contributor" content="Shin-Han Shiu" />
<meta name="article:published_time" content="2019-11-01" />
<meta name="article:section" content="Genomic Prediction" />
<meta name="citation_title" content="Benchmarking Parametric and Machine Learning Models for Genomic Prediction of Complex Traits" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;The usefulness of genomic prediction in crop and livestock breeding programs has prompted efforts to develop new and improved genomic prediction algorithms, such as artificial neural networks and gradient tree boosting. However, the performance of these algorithms has not been compared in a systematic manner using a wide range of datasets and models. Using data of 18 traits across six plant species with different marker densities and training population sizes, we compared the performance of six linear and six non-linear algorithms. First, we found that hyperparameter selection was necessary for all non-linear algorithms and that feature selection prior to model training was critical for artificial neural networks when the markers greatly outnumbered the number of training lines. Across all species and trait combinations, no one algorithm performed best, however predictions based on a combination of results from multiple algorithms (&lt;i&gt;i.e.&lt;/i&gt;, ensemble predictions) performed consistently well. While linear and non-linear algorithms performed best for a similar number of traits, the performance of non-linear algorithms vary more between traits. Although artificial neural networks did not perform best for any trait, we identified strategies (&lt;i&gt;i.e.&lt;/i&gt;, feature selection, seeded starting weights) that boosted their performance to near the level of other algorithms. Our results highlight the importance of algorithm selection for the prediction of trait values.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2019/11/01" />
<meta name="citation_mjid" content="ggg;9/11/3691" />
<meta name="citation_id" content="9/11/3691" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/9/11/3691" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/9/11/3691.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/9/11/3691.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/9/11/3691.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.119.400498" />
<meta name="citation_pmid" content="31533955" />
<meta name="citation_volume" content="9" />
<meta name="citation_issue" content="11" />
<meta name="citation_num_pages" content="12" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Genomic Prediction" />
<meta name="citation_firstpage" content="3691" />
<meta name="citation_lastpage" content="3702" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Christina B. Azodi" />
<meta name="citation_author_institution" content="Department of Plant Biology" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-6097-606X" />
<meta name="citation_author" content="Emily Bolger" />
<meta name="citation_author_institution" content="Department of Mathematics, Moravian College, Bethlehem, PA" />
<meta name="citation_author" content="Andrew McCarren" />
<meta name="citation_author_institution" content="Insight Centre for Data Analytics, School of Computing, Dublin City University, Dublin 9, Ireland" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-7297-0984" />
<meta name="citation_author" content="Mark Roantree" />
<meta name="citation_author_institution" content="Insight Centre for Data Analytics, School of Computing, Dublin City University, Dublin 9, Ireland" />
<meta name="citation_author" content="Gustavo de los Campos" />
<meta name="citation_author_institution" content="Department of Epidemiology &amp; Biostatistics" />
<meta name="citation_author_institution" content="Department of Statistics &amp; Probability" />
<meta name="citation_author_institution" content="Institute for Quantitative Health Science and Engineering, and" />
<meta name="citation_author_email" content="gustavoc@msu.edu" />
<meta name="citation_author_email" content="shius@msu.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-5692-7129" />
<meta name="citation_author" content="Shin-Han Shiu" />
<meta name="citation_author_institution" content="Department of Plant Biology" />
<meta name="citation_author_institution" content="Department of Computational, Mathematics, Science, and Engineering, Michigan State University, East Lansing, MI, 48824" />
<meta name="citation_author_email" content="gustavoc@msu.edu" />
<meta name="citation_author_email" content="shius@msu.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-6470-235X" />
<meta name="citation_reference" content="citation_journal_title=Molecular Systems Biology;citation_journal_abbrev=Mol Syst Biol;citation_author=C. Angermueller;citation_author=T. Pärnamaa;citation_author=L. Parts;citation_author=O. Stegle;citation_title=Deep learning for computational biology;citation_pages=878-878;citation_volume=12;citation_year=2016;citation_issue=7;citation_pmid=27474269;citation_doi=10.15252/msb.20156651" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=J. Beaulieu;citation_author=T. K. Doerksen;citation_author=J. MacKay;citation_author=A. Rainville;citation_author=J. Bousquet;citation_title=Genomic selection accuracies within and between environments and small breeding groups in white spruce.;citation_pages=1048;citation_volume=15;citation_year=2014;citation_pmid=25442968;citation_doi=10.1186/1471-2164-15-1048" />
<meta name="citation_reference" content="citation_year=2018" />
<meta name="citation_reference" content="citation_journal_title=B Stat Methodol;citation_author=Y. Benjamini;citation_author=Y. Hochberg;citation_title=Controlling the False Discovery Rate: A Practical and Powerful Approach to Multiple Testing. J R Stat Soc Ser.;citation_pages=289-300;citation_volume=57;citation_year=1995" />
<meta name="citation_reference" content="Bermingham, M. L., R. Pong-Wong, A. Spiliopoulou, C. Hayward, I. Rudan et al., 2015 Application of high-dimensional feature selection: evaluation for genomic prediction in man. Sci. Rep. 1–12. 10.1038/srep10312" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=M. Blondel;citation_author=A. Onogi;citation_author=H. Iwata;citation_author=N. Ueda;citation_title=A Ranking Approach to Genomic Selection.;citation_pages=e0128570;citation_volume=10;citation_year=2015;citation_doi=10.1371/journal.pone.0128570" />
<meta name="citation_reference" content="citation_journal_title=Mach. Learn.;citation_author=L. Breiman;citation_title=Random Forests.;citation_pages=5-32;citation_volume=45;citation_year=2001;citation_doi=10.1023/A:1010933404324" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=G. de los Campos;citation_author=D. Gianola;citation_author=G. J. Rosa;citation_author=K. A. Weigel;citation_author=J. Crossa;citation_title=Semi-parametric genomic-enabled prediction of genetic values using reproducing kernel Hilbert spaces methods.;citation_pages=295-308;citation_volume=92;citation_year=2010;citation_issue=4;citation_pmid=20943010;citation_doi=10.1017/S0016672310000285" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=G. de los Campos;citation_author=J. M. Hickey;citation_author=R. Pong-Wong;citation_author=H. D. Daetwyler;citation_author=M. P. L. Calus;citation_title=Whole-Genome Regression and Prediction Methods Applied to Plant and Animal Breeding.;citation_volume=193;citation_year=2013;citation_pmid=22745228;citation_doi=10.1534/genetics.112.143313" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=G. de los Campos;citation_author=H. Naya;citation_author=D. Gianola;citation_author=J. Crossa;citation_author=A. Legarra;citation_title=Predicting quantitative traits with regression models for dense molecular markers and pedigree.;citation_volume=182;citation_year=2009;citation_pmid=19293140;citation_doi=10.1534/genetics.109.101501" />
<meta name="citation_reference" content="citation_journal_title=Mach. Learn.;citation_author=O. Chapelle;citation_author=V. Vapnik;citation_author=O. Bousquet;citation_author=S. Mukherjee;citation_title=Choosing Multiple Parameters for Support Vector Machines.;citation_pages=131-159;citation_volume=46;citation_year=2002;citation_doi=10.1023/A:1012450327387" />
<meta name="citation_reference" content="citation_journal_title=Trends Plant Sci.;citation_author=Z. A. Desta;citation_author=R. Ortiz;citation_title=Genomic selection: genome-wide prediction in plant improvement.;citation_pages=592-601;citation_volume=19;citation_year=2014;citation_pmid=24970707;citation_doi=10.1016/j.tplants.2014.05.006" />
<meta name="citation_reference" content="citation_author=T. G. Dietterich;citation_pages=1;citation_volume=1857;citation_year=2000;citation_doi=10.1007/3-540-45014-9_1" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=A. Ehret;citation_author=D. Hochstuhl;citation_author=D. Gianola;citation_author=G. Thaller;citation_title=Application of neural networks with back-propagation to genome-enabled prediction of complex traits in Holstein-Friesian and German Fleckvieh cattle.;citation_pages=22;citation_volume=47;citation_year=2015;citation_doi=10.1186/s12711-015-0097-5" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=J. B. Endelman;citation_title=Ridge regression and other kernels for genomic selection with R package rrBLUP.;citation_pages=250;citation_volume=4;citation_year=2011;citation_doi=10.3835/plantgenome2011.08.0024" />
<meta name="citation_reference" content="citation_journal_title=Plant J.;citation_author=J. Evans;citation_author=E. Crisovan;citation_author=K. Barry;citation_author=C. Daum;citation_author=J. Jenkins;citation_title=Diversity and population structure of northern switchgrass as revealed through exome capture sequencing.;citation_pages=800-815;citation_volume=84;citation_year=2015;citation_pmid=26426343;citation_doi=10.1111/tpj.13041" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=J. Evans;citation_author=M. D. Sanciangco;citation_author=K. H. Lau;citation_author=E. Crisovan;citation_author=K. Barry;citation_title=Extensive Genetic Diversity is Present within North American Switchgrass Germplasm.;citation_volume=11;citation_year=2018;citation_doi=10.3835/plantgenome2017.06.0055" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=S. B. Fernandes;citation_author=K. O. G. Dias;citation_author=D. F. Ferreira;citation_author=P. J. Brown;citation_title=Efficiency of multi-trait, indirect, and trait-assisted genomic selection for improvement of biomass sorghum.;citation_pages=747-755;citation_volume=131;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Ann. Stat.;citation_author=J. H. Friedman;citation_title=Greedy function approximation: A gradient boosting machine.;citation_pages=1189-1232;citation_volume=29;citation_year=2001;citation_doi=10.1214/aos/1013203451" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=D. Gianola;citation_author=R. L. Fernando;citation_author=A. Stella;citation_title=Genomic-assisted prediction of genetic value with semiparametric procedures.;citation_volume=173;citation_year=2006;citation_pmid=16648593;citation_doi=10.1534/genetics.105.049510" />
<meta name="citation_reference" content="Girija, S. S., 2016 Tensorflow: Large-scale machine learning on heterogeneous distributed systems." />
<meta name="citation_reference" content="citation_year=2010" />
<meta name="citation_reference" content="citation_journal_abbrev=Genetica;citation_author=M. Goddard;citation_title=Genomic selection: prediction of accuracy and maximisation of long term response.;citation_pages=245-257;citation_volume=136;citation_year=2009;citation_issue=2;citation_pmid=18704696;citation_doi=10.1007/s10709-008-9308-0" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=J. M. González-Camacho;citation_author=G. de los Campos;citation_author=P. Pérez;citation_author=D. Gianola;citation_author=J. E. Cairns;citation_title=Genome-enabled prediction of genetic values using radial basis function neural networks.;citation_pages=759-771;citation_volume=125;citation_year=2012;citation_issue=4;citation_pmid=22566067;citation_doi=10.1007/s00122-012-1868-9" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=J. M. González-Camacho;citation_author=J. Crossa;citation_author=P. Pérez-Rodríguez;citation_author=L. Ornella;citation_author=D. Gianola;citation_title=Genome-enabled prediction using probabilistic neural network classifiers.;citation_pages=208;citation_volume=17;citation_year=2016;citation_doi=10.1186/s12864-016-2553-1" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=J. M. González-Camacho;citation_author=L. Ornella;citation_author=P. Pérez-Rodríguez;citation_author=D. Gianola;citation_author=S. Dreisigacker;citation_title=Applications of Machine Learning Methods to Genomic Selection in Breeding Wheat for Rust Resistance.;citation_pages=170104;citation_volume=11;citation_year=2018;citation_doi=10.3835/plantgenome2017.11.0104" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=O. González-Recio;citation_author=S. Forni;citation_title=Genome-wide prediction of discrete traits using Bayesian regressions and machine learning.;citation_pages=7-7;citation_volume=43;citation_year=2011;citation_pmid=21329522;citation_doi=10.1186/1297-9686-43-7" />
<meta name="citation_reference" content="citation_journal_title=J. Dairy Sci.;citation_author=O. González-Recio;citation_author=J. A. Jiménez-Montero;citation_author=R. Alenda;citation_title=The gradient boosting algorithm and random boosting for genome-assisted evaluation in large data sets.;citation_pages=614-624;citation_volume=96;citation_year=2013;citation_doi=10.3168/jds.2012-5630" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=D. Habier;citation_author=R. L. Fernando;citation_author=K. Kizilkaya;citation_author=D. J. Garrick;citation_title=Extension of the bayesian alphabet for genomic selection.;citation_pages=186-186;citation_volume=12;citation_year=2011;citation_pmid=21605355;citation_doi=10.1186/1471-2105-12-186" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=C. N. Hansey;citation_author=J. M. Johnson;citation_author=R. S. Sekhon;citation_author=S. M. Kaeppler;citation_author=N. de Leon;citation_title=Genetic diversity of a maize association population with restricted phenology.;citation_pages=704-715;citation_volume=51;citation_year=2011;citation_doi=10.2135/cropsci2010.03.0178" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=E. L. Heffner;citation_author=M. E. Sorrells;citation_author=J.-L. Jannink;citation_title=Genomic Selection for Crop Improvement;citation_pages=1-12;citation_volume=49;citation_year=2009;citation_issue=1;citation_doi=10.2135/cropsci2008.08.0512" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=N. Heslot;citation_author=H.-P. Yang;citation_author=M. E. Sorrells;citation_author=J.-L. Jannink;citation_title=Genomic Selection in Plant Breeding: A Comparison of Models.;citation_pages=146-160;citation_volume=52;citation_year=2012;citation_doi=10.2135/cropsci2011.06.0297" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=W. G. Hill;citation_author=M. E. Goddard;citation_author=P. M. Visscher;citation_title=Data and theory point to mainly additive genetic variance for complex traits.;citation_pages=e1000008-e1000008;citation_volume=4;citation_year=2008;citation_issue=2;citation_pmid=18454194;citation_doi=10.1371/journal.pgen.1000008" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell;citation_author=C. N. Hirsch;citation_author=J. M. Foerster;citation_author=J. M. Johnson;citation_author=R. S. Sekhon;citation_author=G. Muttoni;citation_title=Insights into the maize pan-genome and pan-transcriptome.;citation_volume=26;citation_year=2014;citation_pmid=24488960;citation_doi=10.1105/tpc.113.119982" />
<meta name="citation_reference" content="citation_journal_title=Current opinion in plant biology;citation_journal_abbrev=Curr Opin Plant Biol;citation_author=J. B. Holland;citation_title=Genetic architecture of complex traits in plants.;citation_pages=156-161;citation_volume=10;citation_year=2007;citation_issue=2;citation_pmid=17291822;citation_doi=10.1016/j.pbi.2007.01.003" />
<meta name="citation_reference" content="citation_journal_title=Trends Biotechnol.;citation_author=E. Jonas;citation_author=D.-J. de Koning;citation_title=Does genomic selection have a future in plant breeding?;citation_pages=497-504;citation_volume=31;citation_year=2013;citation_pmid=23870753;citation_doi=10.1016/j.tibtech.2013.06.003" />
<meta name="citation_reference" content="citation_journal_title=Indian J. Anim. Sci.;citation_author=S. A. Kasnavi;citation_author=M. A. Afshar;citation_author=M. M. Shariati;citation_author=N. E. J. Kashan;citation_author=M. Honarvar;citation_title=Performance evaluation of support vector machine (SVM)-based predictors in genomic selection.;citation_pages=1226-1231;citation_volume=87;citation_year=2017" />
<meta name="citation_reference" content="citation_year=2014" />
<meta name="citation_reference" content="citation_title=Applying Neural Networks In: Neural Networks in Finance and Investing: Using Artificial Intelligence to Improve Real World Performance,;citation_pages=64-65" />
<meta name="citation_reference" content="citation_title=Over-Fitting and Model Tuning. Applied Predictive Modeling;citation_pages=61-92;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=IEEE transactions on neural networks / a publication of the IEEE Neural Networks Council;citation_journal_abbrev=IEEE Trans Neural Netw;citation_author=B. LeBaron;citation_author=A. S. Weigend;citation_title=A bootstrap evaluation of the effect of data splitting on financial time series.;citation_pages=213-220;citation_volume=9;citation_year=1998;citation_issue=1;citation_pmid=18252443;citation_doi=10.1109/72.655043" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=A. E. Lipka;citation_author=F. Lu;citation_author=J. H. Cherney;citation_author=E. S. Buckler;citation_author=M. D. Casler;citation_title=Accelerating the Switchgrass (Panicum virgatum L.) Breeding Cycle Using Genomic Selection Approaches.;citation_pages=e112227;citation_volume=9;citation_year=2014;citation_doi=10.1371/journal.pone.0112227" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=N. Long;citation_author=D. Gianola;citation_author=G. J. M. Rosa;citation_author=K. A. Weigel;citation_title=Application of support vector regression to genome-assisted prediction of quantitative traits.;citation_pages=1065-1074;citation_volume=123;citation_year=2011;citation_issue=7;citation_pmid=21739137;citation_doi=10.1007/s00122-011-1648-y" />
<meta name="citation_reference" content="citation_title=Genomic Selection in Plant Breeding: Knowledge and Prospects;citation_author=A. J. Lorenz;citation_author=S. Chao;citation_author=F. G. Asoro;citation_author=E. L. Heffner;citation_author=T. Hayashi;citation_pages=77;citation_volume=110;citation_year=2011;citation_doi=10.1016/B978-0-12-385531-2.00002-5" />
<meta name="citation_reference" content="citation_journal_title=Planta;citation_author=W. Ma;citation_author=Z. Qiu;citation_author=J. Song;citation_author=J. Li;citation_author=Q. Cheng;citation_title=A deep convolutional neural network approach for predicting phenotypes from genotypes.;citation_pages=1307-1318;citation_volume=248;citation_year=2018;citation_doi=10.1007/s00425-018-2976-9" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=T. H. E. Meuwissen;citation_title=Accuracy of breeding values of 'unrelated' individuals predicted by dense SNP genotyping.;citation_pages=35-35;citation_volume=41;citation_year=2009;citation_pmid=19519896;citation_doi=10.1186/1297-9686-41-35" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=T. H. Meuwissen;citation_author=B. J. Hayes;citation_author=M. E. Goddard;citation_title=Prediction of Total Genetic Value Using Genome-Wide Dense Marker Maps;citation_pages=1819-1829;citation_volume=157;citation_year=2001;citation_issue=4;citation_pmid=11290733;citation_doi=10.1093/nar/gkv1344" />
<meta name="citation_reference" content="citation_journal_title=Front. Plant Sci.;citation_author=M. M. Monir;citation_author=J. Zhu;citation_title=Dominance and Epistasis Interactions Revealed as Important Variants for Leaf Traits of Maize NAM Population.;citation_pages=627;citation_volume=9;citation_year=2018;citation_doi=10.3389/fpls.2018.00627" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=G. Moser;citation_author=B. Tier;citation_author=R. E. Crump;citation_author=M. S. Khatkar;citation_author=H. W. Raadsma;citation_title=A comparison of five methods to predict genomic breeding values of dairy bulls from genome-wide SNP markers.;citation_pages=56-56;citation_volume=41;citation_year=2009;citation_pmid=20043835;citation_doi=10.1186/1297-9686-41-56" />
<meta name="citation_reference" content="Neves, H. H., R. Carvalheiro, and S. A. Queiroz, 2012 A comparison of statistical methods for genomic selection in a mice population. 13: 100. https://doi.org/10.1186/1471-2156-13-100" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=A. Norman;citation_author=J. Taylor;citation_author=J. Edwards;citation_author=H. Kuchel;citation_title=Optimising Genomic Selection in Wheat: Effect of Marker Density, Population Size and Population Structure on Prediction Accuracy.;citation_volume=8;citation_year=2018;citation_pmid=29970398;citation_doi=10.1534/g3.118.200311" />
<meta name="citation_reference" content="citation_journal_title=Genet. Res.;citation_author=H. Okut;citation_author=D. Gianola;citation_author=G. J. M. Rosa;citation_author=K. A. Weigel;citation_title=Prediction of body mass index in mice using dense molecular markers and a regularized neural network.;citation_pages=189-201;citation_volume=93;citation_year=2011;citation_doi=10.1017/S0016672310000662" />
<meta name="citation_reference" content="Parker, D. B., 1987 Optimal algorithms for adaptive networks: Second order backpropagation, second order direct backpropagation, and second order hebbing learning. Proceedings of the IEEE First International Conference on Neural Networks. 2: 593–600." />
<meta name="citation_reference" content="citation_journal_title=J. Mach. Learn. Res.;citation_author=F. Pedregosa;citation_author=G. Varoquaux;citation_author=A. Gramfort;citation_author=V. Michel;citation_author=B. Thirion;citation_title=Scikit-learn: Machine Learning in Python.;citation_pages=2825-2830;citation_volume=12;citation_year=2011;citation_pmid=23755062;citation_doi=10.1016/j.cpc.2010.04.018" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=P. Pérez;citation_author=G. de los Campos;citation_title=Genome-wide regression and prediction with the BGLR statistical package.;citation_volume=198;citation_year=2014;citation_pmid=25009151;citation_doi=10.1534/genetics.114.164442" />
<meta name="citation_reference" content="citation_year=2015" />
<meta name="citation_reference" content="citation_title=Neural Networks: Tricks of the Trade;citation_title=Early Stopping - But When?;citation_pages=55-69;citation_year=1998" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=G. P. Ramstein;citation_author=J. Evans;citation_author=S. M. Kaeppler;citation_author=R. B. Mitchell;citation_author=K. P. Vogel;citation_title=Accuracy of Genomic Prediction in Switchgrass (Panicum virgatum L.) Improved by Accounting for Linkage Disequilibrium.;citation_volume=6;citation_year=2016;citation_pmid=26869619;citation_doi=10.1534/g3.115.024950" />
<meta name="citation_reference" content="citation_journal_title=J. Exp. Bot.;citation_author=J.-M. Ribaut;citation_author=M. Ragot;citation_title=Marker-assisted selection to improve drought adaptation in maize: the backcross approach, perspectives, limitations, and alternatives.;citation_volume=58;citation_year=2007;citation_pmid=17158111;citation_doi=10.1093/jxb/erl214" />
<meta name="citation_reference" content="Riedelsheimer, C., F. Technow, and A. E. Melchinger, 2013 Comparison of whole-genome prediction models for traits with contrasting genetic architecture in a diversity panel of maize inbred lines. BMC Genomics 13: 452. https://doi.org/10.1186/1471-2164-13-452" />
<meta name="citation_reference" content="citation_journal_title=Front. Plant Sci.;citation_author=M. Roorkiwal;citation_author=A. Rathore;citation_author=R. R. Das;citation_author=M. K. Singh;citation_author=A. Jain;citation_title=Genome-Enabled Prediction Models for Yield Related Traits in Chickpea.;citation_pages=1666;citation_volume=7;citation_year=2016;citation_doi=10.1016/j.fgb.2010.06.003" />
<meta name="citation_reference" content="Rumelhart, D. E., G. E. Hinton, and R. J. Williams, 1986 Learning internal representation by error propagation, Parallel Distributed Processing, DE Rumelhart and JL McClelland, eds. ISBN: 0-262-68053-X. 318–326." />
<meta name="citation_reference" content="citation_journal_title=Artif. Intell. Med.;citation_author=T. Shaikhina;citation_author=N. A. Khovanova;citation_title=Handling limited datasets with neural networks in medical applications: A small-data approach.;citation_pages=51-63;citation_volume=75;citation_year=2017;citation_doi=10.1016/j.artmed.2016.12.003" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=J. Spindel;citation_author=H. Begum;citation_author=D. Akdemir;citation_author=P. Virk;citation_author=B. Collard;citation_title=Genomic Selection and Association Mapping in Rice (Oryza sativa): Effect of Trait Genetic Architecture, Training Population Composition, Marker Number and Statistical Model on Accuracy of Rice Genomic Selection in Elite, Tropical Rice Breeding Lines.;citation_pages=e1004982;citation_volume=11;citation_year=2015;citation_pmid=25689273;citation_doi=10.1371/journal.pgen.1004982" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=M. G. Usai;citation_author=M. E. Goddard;citation_author=B. J. Hayes;citation_title=LASSO with cross-validation for genomic selection.;citation_pages=427-436;citation_volume=91;citation_year=2009;citation_issue=6;citation_pmid=20122298;citation_doi=10.1017/S0016672309990334" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=A. I. Vazquez;citation_author=G. J. M. Rosa;citation_author=K. A. Weigel;citation_author=G. de los Campos;citation_author=D. Gianola;citation_title=Predictive ability of subsets of single nucleotide polymorphisms with and without parent average in US Holsteins.;citation_pages=5942-5949;citation_volume=93;citation_year=2010;citation_issue=12;citation_pmid=21094768;citation_doi=10.3168/jds.2010-3335" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=S. Webb;citation_title=Deep learning for biology.;citation_pages=555-557;citation_volume=554;citation_year=2018;citation_doi=10.1038/d41586-018-02174-z" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=A. Xavier;citation_author=W. M. Muir;citation_author=K. M. Rainey;citation_title=Assessing Predictive Properties of Genome-Wide Selection in Soybeans.;citation_volume=6;citation_year=2016;citation_pmid=27317786;citation_doi=10.1534/g3.116.032268" />
<meta name="citation_reference" content="citation_journal_title=Rice (N. Y.);citation_author=Y. Xu;citation_author=X. Wang;citation_author=X. Ding;citation_author=X. Zheng;citation_author=Z. Yang;citation_title=Genomic selection of agronomic traits in hybrid rice using an NCII population.;citation_pages=32;citation_volume=11;citation_year=2018;citation_doi=10.1186/s12284-018-0223-4" />
<meta name="citation_reference" content="Zou, H., and T. Hastie, 2005 Regularization and variable selection via the elastic net. J. R. Stat. Soc. Ser. 67: 301–320. http://www.jstor.org/stable/3647580" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Benchmarking Parametric and Machine Learning Models for Genomic Prediction of Complex Traits" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11.cover-source.jpg" />
<meta name="twitter:description" content="The usefulness of genomic prediction in crop and livestock breeding programs has prompted efforts to develop new and improved genomic prediction algorithms, such as artificial neural networks and gradient tree boosting. However, the performance of these algorithms has not been compared in a systematic manner using a wide range of datasets and models. Using data of 18 traits across six plant species with different marker densities and training population sizes, we compared the performance of six linear and six non-linear algorithms. First, we found that hyperparameter selection was necessary for all non-linear algorithms and that feature selection prior to model training was critical for artificial neural networks when the markers greatly outnumbered the number of training lines. Across all species and trait combinations, no one algorithm performed best, however predictions based on a combination of results from multiple algorithms ( i.e. , ensemble predictions) performed consistently well. While linear and non-linear algorithms performed best for a similar number of traits, the performance of non-linear algorithms vary more between traits. Although artificial neural networks did not perform best for any trait, we identified strategies ( i.e. , feature selection, seeded starting weights) that boosted their performance to near the level of other algorithms. Our results highlight the importance of algorithm selection for the prediction of trait values." />
<meta name="og-title" property="og:title" content="Benchmarking Parametric and Machine Learning Models for Genomic Prediction of Complex Traits" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/9/11/3691" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="The usefulness of genomic prediction in crop and livestock breeding programs has prompted efforts to develop new and improved genomic prediction algorithms, such as artificial neural networks and gradient tree boosting. However, the performance of these algorithms has not been compared in a systematic manner using a wide range of datasets and models. Using data of 18 traits across six plant species with different marker densities and training population sizes, we compared the performance of six linear and six non-linear algorithms. First, we found that hyperparameter selection was necessary for all non-linear algorithms and that feature selection prior to model training was critical for artificial neural networks when the markers greatly outnumbered the number of training lines. Across all species and trait combinations, no one algorithm performed best, however predictions based on a combination of results from multiple algorithms ( i.e. , ensemble predictions) performed consistently well. While linear and non-linear algorithms performed best for a similar number of traits, the performance of non-linear algorithms vary more between traits. Although artificial neural networks did not perform best for any trait, we identified strategies ( i.e. , feature selection, seeded starting weights) that boosted their performance to near the level of other algorithms. Our results highlight the importance of algorithm selection for the prediction of trait values." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11.cover-source.jpg" />
    <title>Benchmarking Parametric and Machine Learning Models for Genomic Prediction of Complex Traits | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings, {"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"X_dauZ53tCOUOIDBzMrKNj1796IWzyfoVGYPVZJjomE","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"494609","apath":"\/ggg\/9\/11\/3691.atom","pisa":"ggg;9\/11\/3691","ac":{"\/ggg\/9\/11\/3691.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/9\/11\/3691.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;9\/11\/3691"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;9\/11\/3691"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/9\/11\/3691.abstract":true,"\/content\/9\/11\/3691":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022mtNyhIym5lunXrxR5JIN92Vf3nph1eqVIAcJ980sKFtN2YzRTh8VoRk36UnFU1YGii5rU3rsWOAOA4TTU0uMuxjcN5a5FcPCouWrxWUEv2Ctq3yLuedXJkOE\\\\\\\/Asyv84zLWbfsODB\\\\\\\/vGYncK30kfu++fp+z+0hKGHasZ6isFz3w\\\\\\\/x3u+lz8vcOpVW5LkSTDDR2blpApAYO\\\\\\\/DwLlxCQsu6BJLmo3u8+2JP4iboWgl4qhcACGbB02ZQ0BgUy1Er4nvMr0uL\\\\\\\/UzyvJdJq7XqDldif+i7lTyf7+pEU6VhooH6JF\\\\\\\/dtTiurSIvT9Vqb0YTAB2WBefksMzKuwi8Kmjay9Zi1jNe9XGfA1ASgEF7+5zsMVpexKwR8e7\\\\\\\/cQazyL61h0SAyBBpI3plvrqwb49ZgqumMBpJLilL\\\\\\\/J7njRNigJKOyz+Omtsg51bUCyL+wrxC5k0pDwCKos1yLFPnLmThgh5dAtnJjW8lvYSSL0yvQttSK3vhWD\\\\\\\/0GhRR95sqPe+7A0ErY7FS4PiscLA0LA+EzfbKgM+opZpdhkrv\\\\\\\/8rcUN2m3+r80pz\\\\\\\/EU3Ujo53cUwg5VDBbEu3TvcxR3Edm+zWjNiwfT1zb0Vjn7IQRNgn3MvbtXZTGeYOksP2RKLA0lzKQ60zLm5\\\\\\\/VV6mvWIh4yFO1PeiJiAzfOWEgf3\\\\\\\/CxlmiiKDmgxC8fzdprb9mX2hPaItMMnA83DHqSOlLfTHtNaSfOJG2yXVEEUSXoSEIljY9oEFFt0965QsFVygMblXKuwOaO6vRqZy+hK2E3orDazqojcxx9i2s6QtTiyB8wuA\\\\\\\/c8mO1wYCrA06pbJ2\\\\\\\/j+\\\\\\\/tFeLXE85Bj7POCIsZcLTafIH8ARmEupGTOUJglMsMQUen\\\\\\\/LHyhK4SqZY6xb\\\\\\\/mqALCqyxJeCuodSiJ+s0b1X5hegkyyQGUlxeo9p5zt5hwyxk3N12xN5kIliMY5vxyKGchSB4AoBAfT2QvdOfkom6YuL9epE76hvfUiPtjMndbOkFLUahPM1H9sNAs9dEl7LFwsRfDvaO7Lo25PdEcXnQTckHsTgUyR1pA1QZueeVCScjFkQRTk2Bze1nEvy4FHjm7DyOtnfpRreNWf+UJl\\\\\\\/eDmoBwhSvdsB6lSJM3dIuXO51PcIqymkoSX2yqVcXgfXExHnmqrUvfIOloCY7ROFBinZUPOyhRZWvBnPBrMoUoJeFMDu++0fhKUqbzXCIetfaOHMiiPMpinr+7goPR3AhDn5r6dCeDmZUCLQ7vaDuunST5J8ciI92D2a0BXIrQmjirLQ7+3B\\\u0022,\\\u0022iv\\\u0022:\\\u0022r97K6ppDF9aO761rEaKHBg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022791037fc6ff531b0a8a15e1b7877c4a4af8c89f5dccfecaa45d46fcf09d5a10f\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022iGacVlQNf3b7qEp0Y62kxqJ2vYb4siy\\\\\\\/pWwGrIRGU\\\\\\\/qGwOEuU3CGnC+iuc0ib0srSBMWDz688c2uqDd5JNcsEsRYCxRjjcoo576wAdBLNhk0QvXDomvCReeQVw6o2qaa5PDAn4Y9StgOZBGHMXxu\\\\\\\/o7NW6blKUC+2sJZvL6seymp4DoVVRF8KfpevZ16qlap2rXRVOZ2oJFFt9\\\\\\\/w8++qclb3Ji7QUXfmO0DLgBRecJN6HbE1kimQtkO7u1l1k1oA2nBumm7JHwiZDRBeryYQxVYrdSl9bWUV0O93kiBYksXB5IkkYjQnLvm+CUwSIECFfVXecreazsihDLQ+YYWsBd6n7Tb\\\\\\\/u4mx8x2dcmtDIYgMU0JLWQxq8MpxVBtoDUTz8xMuUQG7tTKMKKm94qf3ySqdIZHqjTmw1hruWuHQmIzifsDZglr6dj\\\\\\\/yWNHsQrsfqkfSf7eWHkyAAu3Wsbd5eZ80hjF1688USCj4WpIDpyxIQ1yf\\\\\\\/jLuX8Nu0Hc0nJDHKRezY4hRHEiZzCPgOxu+Ola\\\\\\\/CDgREAJp96aEYD1q6CnzZ0fBCEsxNWRt3PDuEL9pKKpHstxZQH4ET0TAYFAJlGe4rrb3ntCCrn2XVV4hOZbSlSnD54afRdWwA+pm+TDT6yeVd+JKxthofA4EbT+hhY3tBpv05UHFcSwzwrITLDHEWq2tzJ5nxZytRtuiWvJUhOVcb3IygEnkvmxrQIfqLC6V10THsUa\\\\\\\/DrxkKF40EIQ0IFa1fztCqZ7nIkFsnL5QVbwwUJhS4jQp\\\\\\\/j26SDoyu3ryV69ZJgvOtU\\\\\\\/\\\\\\\/Hu9MWKfOYXQdJZdaMaznpN4\\\\\\\/JUjZAFl8IvsG7Zp\\\\\\\/R5y3d1TkRwvvXj2Q2qUj1zyCvpXxemF9BYAzOV1U4OmJ\\\\\\\/Mv\\\\\\\/jCG6IfaTAyA7OdcQRY9RQE1TYQbmquIpjaoaQ0GNo3zLIgRXxk+MVghLzYhY49Oku2je3UTtj6yWucVitnQJQ6HSwtXh9yx93\\\\\\\/9PUJGfAssrjYRhoayt65k25L9ywrCHbjnHXcDknSasSSAz7MsloNiX4\\\\\\\/m\\\\\\\/WIdyjmchuDn\\\\\\\/kJYwwUK0h1\\\\\\\/xmSoxyO1c\\\\\\\/uRTN2RfWHDjhmO9AEYGQX9dQITmUFI2YE9JJpaAQ5Pu8BqCW3P2NJP4tDl0vG2JPajSAuOrW2rTIbcNtJyhdD2fY1e+NSZjiygpu+zn09BVAFcu\\\\\\\/2RbqsIUj8CaX3noUP5Cu99JRirDGpOQUPX7U+ZPe3vk8prH\\\\\\\/TDfom7+pHEJ8qlYhsVNJKPRta0LSHEm0s2VBz8Y+1DPgpEXJHTGVOTuln2QkAoXrDwRjfLfONQlmvnMXbkYbR5hafPJFR2rON8MXvt8iOtjuGu1Y58tzJF85nTIyI+WQ6qZUoIW9F0QCN3OJtVAIxkJuV4bGsfshfB6AXzlqHpbg4Ex0fN7j4zrpJFSyPElhZZ3Ez6GHeDFSvyk8vqI2gziSBqybPuCqdIpV5DTkZmaXms2TXJzYv9wrEiQC494maKl4PM9FTtpHjFG94iKdN00IwBJMwAH7R7UDbIALOQNjmoHdZvCbsMd670sSklYzIKjaPgyifQhG+76EjcRCNOC0Nh5Hw+7IR81ICVqd9TwH33EcE3ZARs2gSIO7QlVM5+1ANW\\\\\\\/IALYIiTkHV3h8WiNX74ElMUOYGL5ID1bUibaw9pVJB\\\\\\\/89R9y8uplaw+OHqJWjBI9Mhv6aX+kzcBQ2EKOymMzCWDdy4liaub48j5XAPUpRgLCC0\\\\\\\/+3iIp38IKaRv1r9I+RuRahjuaBT2FoEDJA\\\\\\\/\\\\\\\/0r3vpHGh2ojz3QtWqt6ctihBkupH29TBG8jg2zkZM1xUPudasomGV\\\\\\\/D6qKujJDkmaGeahp9rnlhC+wrOZjs1n\\\\\\\/oBzhVzblnLkjOBovHKUQl1fvJyMfXsOryw2Dr\\\\\\\/Qmuda5AoMhR5i7UBHtyBeYTkksLBDNxmO9\\\\\\\/dxxLJLwF++fej59c5Jros90jQcaOIrzc4W43jzKzMaUTQo74Zf88VYheWVh86kmzUKggtvzAIhc0pUqQXP3AIH99ZhSw7zdppI5FUh1CFkfRYqU7TV2dthTAsgzmTuZSP2exEVtE3e5+IXMr3t4r9nBGVFwQgIYlf8QfWJlDNUGygH7KwdwnHV1fwGC5ASsSpxmV04YjoKCptQQlMDcphZoFLZmrX8EtpAY7Lu8RsBxo8mN\\\\\\\/mx\\\\\\\/TkrmojHxHpjYkcAp+kTUlGmz6UsOhuWKu6d9BHp7qLjqjngR\\\\\\\/Qehq\\\\\\\/amHtenqsEUBpb+z6kVjS6j0zYEDVBEX2G\\\\\\\/oyc\\\\\\\/laX1DXPzfVpsTwtrzdcKTZoZp3kPimI0VQ8WtkTGET8AuyShZfTZApzCNxyylE\\\\\\\/tCkg0pMLv4XLmau2lVY4RfV46nowt3oWD\\\\\\\/itX8jTuTjaqYfum53FvaQ46ixI5JFSKftIrXqDvL1tuIcTHEzB1b6d8O4V43SQRgPULqrcYY\\\\\\\/S8KHeVNOyV8B8yHGssiS\\\\\\\/chM5\\\\\\\/6ajD1jbzJLYz3w6VXH800OxotFIZ20zdWEO13gZpOiweD0x6OvxjqtAvee5wV81lxbNxs7i0omZ9NTOPstGUDQDdrxXYAOWJ3sG+6jOieEQdcr7L\\\\\\\/upujsIUXPbacqzY5ewGpnb7s79Kd9wn4P9Ep8Nxq48X0+jK6HSZfzYV2f02UgIdu9EhE2R9iEquGm6vL9\\\\\\\/uvAGadXwl9saKfXabIhDvkCDrwUXNgOWVq9KfdsuWMaql0jPFqiNUWZCF5EWO0IcnWBcpohzx7vgbP9x6aYT7O061nQCA\\\\\\\/YMZu5ZvHucKAywAQWjUFv54nz44DYubrjGebuTIyL7Xri1viuIYizDJZ6x4CF\\\\\\\/YFQlJY3Dlrujf6DXjs83Sgnz9F2s5uW5ypME+3y9+GSTyBTmnTjR2IGyzrlNeKYPAGVT87oVYmfXTX7bc+jRj2zuJvccMsCAPRKIz7kpDy0B2f8uSgI50xlL2TgtX9XZ2cDEeEtQPHpEmcbmNXQ33ou3SDlA41dTnsyHGiuJeWZ8TnakcAUQ2WKZCkY6KLeRJiOc+1n7gG5nhic8tMhNqx1l6QdHudBxmJyKpjjqp15ZAB9h6Epn\\\\\\\/dz15hKlTWpONvNs9ljMLCM6TpY39quCIS3lr1k+y\\\\\\\/XVxsm7a8TzjRbfgZsIXDd9di0RZbIR6YW3Ro96U1VL6KGlHPHvsxOB96Aly5lnTn5WNh4+8YNnywPRoPuy0jvPvQw9LOfH9HeQDKNdG\\\\\\\/s4fbSB1Mx1M+zpojgLSNW\\\\\\\/dklbqjC+b7uEZWv5Y8d\\\\\\\/y0P\\\\\\\/wJVttlSU66He5BDIvfQQWO6+sWmmm48bQ1OqwKGTMVxx2G0L5xT\\\\\\\/NgMwchl9iOplEtFF96RQmNTgcf1eChvHwOsSVEOIp2g+BnxUVfHvH9tO36iYJsKAQKEA4AyTonzSjbz2TkRoukWLzzYRvU+K\\\\\\\/eLbHvd6hNabns4HRMzdjUMAAFN4YjMukLYQNFmjaPcI4Lvabk+wrUGtsFiR6AgEj1WOimgVaVEFs6AnlnTx7lhhd3jbePnsgWvXu64cl8tf33lP4ZS8EKUddja\\\\\\\/a9iUj5b6eeoFXd32OUcq98xtwB\\\\\\\/K9HY5RpVYiZGMXKp2GDbp0sIxHWmFlLBNupdyyEAjs6I4t21zlno1nxLJNI4DbDs8DMw75aNEG4MIi1CIsS\\\\\\\/L+0c30nrOTBQUWckh1xQ2BHjyg113IC9Q\\\\\\\/wWfOkTVlfaISVb\\\\\\\/NTYXv4IZqe8IjiYXc\\\\\\\/nl5koTwE0iPreuyS7IpHvYwwAUE7uK7SlTnc5vQvNXYTBiUABALsVTtcrCTZsCcgoSahiRZUNjxtHSYWOww3V7Dxq3tceBQrATwb+QZLz6g3tXbwOLaJno8KtOXACb\\\\\\\/7aIrQmftzIAiEwpWueZmmw2X6S6WwQoNLREp7ZQ6tHvvWNzP79mwz4ag+r\\\\\\\/W+es0sOi62ANcYtnk0OO4jaWPcCPW5WdWaJlPlgWfYigaghqoj9BeMWnBNOFJ5aHX15yEIsLju3noZ0G2rP8dR3\\\\\\\/RyUbyowd24kMAF9MfUNNb+bzTcPLIE8z5X4gmooDzJxhY9C5uM0Fg55tPYoB4QwwGwWwZD43Do7DmUP27zWAEvpKh25DbPH6CwyuGZlrCsbwNvGoyVeVv3NxwibfJKc5ywA0CEz+kFZzL4E\\\\\\\/z8Yo2YrTyD1O7Q82AKTDc6PydWKo\\\\\\\/GhvpTbGKLpARtAJyUUoGM1CnPNA+2ZTDetzbbQrP6XfKwc8vDRib3+3qWuZ2LOmeGVVztS26T4R9VBgCDxT+5DfA2D6MUNFuY8vyUyDFH2LJpN+y\\\\\\\/pA8S\\\\\\\/9wkZfAbbgOuckKniKM3u5Whu5mxFXkTaz5SzpJfIsQf5jF6+53vjs0KbE9KuC6uDrUJDXTHssdyOUIVEfiAki02af6abVE2wMRsHXOKGewbOEh6lVsMSuMeTK3Ixgs3B4B9wGHV0382hudQGSEJv84FEBeuXyPGS73ePdm9DKIa32KlhE7nSg8S8KmzDMmNxw7gULvhX0GSURtLPj39jou\\\\\\\/cvCcAFygqajYSfiV09HdYoiaL3z+Cf2rAKe408RQsQctZ5wSkcguQpMGKi6fLevoBrlzQiSfee55VWwmjamAVj6pgwQGCc3TFIvYl76uLHH2dBLR8GiB6fDjwywSAOEef9h1XcLhECgWMLrYi8Az5zQw0UmHmAXQ4E6uRleMyvYZOKk5eH7CHWZnVUqa81DU951T0Nc0kV4cBhu+PUEu\\\\\\\/bw\\\\\\\/+alpNrYZkxsFahispitDsUsqO6zwDZSQPtnrUNpOHToIGnrFW+6ytlLpJyKjZXZ43Chul6+Sc3K3Fjpaqi8S0S6psoWJm95KZ4\\\\\\\/xcddHiZ8S\\\\\\\/TVotaz+tknawYbXCW1N5fZnxHAtt6AIDQCf+6DpXFlLONjB9LALNExsj14b4f\\\\\\\/ppeK0uI2xPglBquBVCb5KkyGVwQE78HV29zfCt\\\\\\\/CZ1j4tzFKMEB7++trMUy3mCm1eMFtu\\\\\\\/eK5CHUL6r8QqxrN1aRLGsHHAy4PPlsLYKioeeIwVYkfw7esxI+m\\\\\\\/\\\\\\\/QeHOYGQdNRGL8yA56sznlUTOrzqWyMw4uV5o3Xzrw1XMutPcYnlWq6+CUjEu+69rvyqdRcQhUyWFBYekF51VeTidp7rFpygPxvgZY8xqTJlAfLSkKbBeNoy5pn+zizMuaIzAGnv5nirevJVYYi0Z58310M4k+G1rCjUUE3TIurBLqfp0a4sohrSJbSaooxBYPlFXAAc79LHvNNVkws3sFbnzdA9N1L\\\\\\\/VOhrQMLwLYR1g5XCL7DV3jDG01DgQQ2zyyKjCJ\\\\\\\/EA3yZY2XJY4ZpGGm5P8108TdG7Od7zWLp7Zsz5bMKHtzIbknMBpPeLHHjKzY5Igon6qJyL4GbcF0LsbzD4tl3r03WcvAF5GZ2oN1\\\\\\\/fp1mfaRiZYkbILR+czh9wCRLJeE\\\\\\\/IRRROva4ExT43PXi6JKrytysqUEHa5XTxmN0UBo4VSjr4i5gvhcUF5If7ZDrXzgmCxMYj\\\\\\\/UJZUAHEVVpzKGDShIZ1Dlz5g2wASe+fOn8yuAjQW7QAaaqGhRInMYAh1NACM66pe33OQaVbZS9I7DIm6bVT0fwUl7Xi+nGRbHGUr0qsSBmKPiraqKl\\\\\\\/IWn2W\\\\\\\/3Duu61dMj7KcGc9XTBfHgheCyazDxNniuq2XX2MfNVxnS81basgdGszHA4ZfHehSdQASEfJ5bL8c1Q5OLpRAFpsu1943LqTSDRDzWo\\\\\\\/XXgIAG\\\\\\\/d8IzsLF1Of461yAAq69wUK9lnvGXPStIbQ+\\\\\\\/z2BVS4k5j6jgQhh1RLY3BsnB\\\\\\\/GC60m9CQLPiFNFNQbjUcq8JKu9Hku9AT02ZyDToGN+13TcS4ORDqDnfKRtYwKDBBFTu6f7jCVwceI2etMHZhPqYPmRmeEKlsOTjK4Oo5LW5g\\\\\\\/kIpl7dii6uab+Lr8ozNsZMirZrRC\\\\\\\/m88TAXcxkhi1naljtdHes52UGzuOZ5LVWWycWdEQ0bb7gbNqTc\\\\\\\/a425eA7Tgxpuf\\\\\\\/MrvdTaepV0Np2q4m1Rl46\\\\\\\/fYME31joT0UX5orBWy9MsrOFmP+BldwhiEG5A6yJIsN3tD+y6okTRLXZwt8Vy4se6tPqurK024IXTegEkgZvjGw41w7Yam+rCbEkmsn\\\\\\\/VcDjAKBfpU1A09bdZ0ke7txriu88hANmmY97518+R5gt0F8ImUZWJ2eFGIMAR1QHkPxiErFYeoXlJ+7jTZNGYXtHwdZrzmzLFAWYuOAAzZxg3dqrdpcOYD2s07zF7+iOWAUGK82dXFXKdjCKvaaey3up8Nccio1p7nK4cEkeIfeBfNJJcBzCyuG1JO3uAxAU0n2R3haLipxp5ciTHzAiolbVaLtnUBt3VIAiW0eVBWatY5TNVzB65XUg0C8kXeuaItief9Hi2bj\\\\\\\/Z2vACgEGZGtLAhLUl04jsLYCYLWnX51OXdsRAySWzpHrCeoR+ROo1LWflpsfY6vTNtw3PJxUkMW8YNX9QH9xtORLqIXPHKERyIJ9\\\\\\\/vlMJAxOiIXr9BDyt4YLhXlPnPST3Pmiylqptflb4WUToClqq7yRWWLexn9KI7zwCdrv3JvUFB6t0LWs2vDK4dNy31G+R5sLBwEDBABbrkcX0LNkDOkQ+JBCPBF64NS\\\\\\\/j4J7epSQEadruE+2DIm7ewSKThM7Q\\\\\\\/XTd5aIfdZp3DoFeugvW4KiaOPHOM13A47ZXE5uRlerscVPmg\\\\\\\/uNO8iDjzohSzO2p056fX34vGNVvE9KEOerhnSPr+ZzzWWtssoLAz5o+lSrABA7jD5qE7sSztalG4UATLCTnuE+wzJj0n5lgTWIWSzlkJOxuslz2ctxJJmJIqZjp7Kdz7phtRUVv3LhcmuGBBbQ995iAQ+0ihd529n8EszNE5Yworav7Khbm6ide6qatLkEcJWQOUX\\\\\\\/oMwgJAPvzKvyj9BVJ533UHuKSOs2nojlgcac\\\\\\\/Hy0gk0RlID9uZDoZWSS7\\\\\\\/94DNyr7jXuhGTTHAkKGcGHZdV7KW4KhSZXkJ6sgaQQRsKg1eQCSrYHQqaUAWAF4\\\\\\\/gKmL72gUN4mltMavw8920kottPRePvWg517LMNk6lSp1g5cRejFQjhxQR3jFNXQTVo2W7GQKoNcNigGpGd+t65RxYDB5phqFtwg4w9fcL4Wb7BM5m9Zrq4yP4iCvCx0nBhU9VKIMR9jmmPnk\\\\\\\/DoP9+VQZW\\\\\\\/h545251kjhaqag4lvAzmnKx9cAoJMft4aTgxdVCFgQEXYXIKE5jTuRt3QppqbDJbm2RjVLuKNLI1zj230VTyfBHohHXJSvIHhNyXW+dnSYmzgDxLBYz2\\\\\\\/hzHVjKDvU8qXmPr14oXNvEnkKESBBBB7JnfX4Bd+rYdIoUAHGc9tA4elt5LE4Gl7kViVAXdCpVKOqmO\\\\\\\/w0OD8kYiYCOvhUdv+HcvvafC6LFzPSj0ndQY8q4pWjhNf3W2e8TOpMFB89OT8DVVO9p5bw1lShseNSV7yUT7MljhSH8N1PmUbhkItCeglQKPqz0LynD2dVEq0LyYDM\\\\\\\/Lxn+t5iu81fD+DSl7nOLQcmnbcCXjft+EsWlBy2SND3x\\\\\\\/wGxq\\\\\\\/DgHtx6ukaCAVrjtROwMvxlrg6BCsOxVUc7wDHWcKY+5qGKekA7daoKIChERW1uEY1+knq9\\\\\\\/i3ifJmghCqwetaN8DRI1bDoNEFaw\\\\\\\/Htu7NjWd\\\\\\\/Pk3Ff3JkLbQDhwfsdCOM2dIhxsE48eIuq48Ewl2kpXq3ia8TuUEz\\\\\\\/lDXFLo+pJDvIu1OK2dWAKyyyt\\\\\\\/B4JEToeeo9vepwWX+A0XaimNany9Popf+4y95D8HKyr+n2o\\\\\\\/sTCYoZGWdEyuR7T+FmYpolKzwz3kKQJHigHELBO3p9Y1GI8nJXo7LNbQgLSHNdJBm8NPsTHq6wjKNJvXBBFx9NPrpG9uy3ncSWrUbCj8zsCow+6PbEAyt+9dcKnWq9dy5VnFbGINc9YzKabbK2MyxWQNuA7orkd36wiaFCeGBp2NnXRO0jMtRBZPHZ6KiBy+rcwCB2AubDMCZt5AHOo8vRNwNv1PUqpiuq2Vn4sPnLJxDG9qzPl3yiMFcaN8Qk0\\\\\\\/hZll7hu11BoeEqXtebtA6x+WyERdiUOEpzQqsLxH\\\\\\\/OUJLC9ywNcTzuM+w3zQ9ucy+C5axFDpZ0LkUXz54+4\\\\\\\/RAFh97MyYYoInr+7SYNV8\\\\\\\/zos+3is2oIpSuezgfH5LnPkwn3yexGF0Vc36ZcX86IxmKzP2CpCtZpKtf\\\\\\\/\\\\\\\/AvXSXSfSp4f+pT81sZUEM4N7n3ELiW4EsVQNvOSG0zZgIx9hJFfe8q+oMdTqxdCvntrONs2uJtq3Y4lMhfecj+v0DWwjiekvUQUKt8iPbh5VDqUAvEA0R80bRaH1etr4LAIKMCQ1xysT4u\\\\\\\/THWWc0nwSQ13\\\\\\\/QMFLkgHkqTuFuTjTa0AUF3TOinlTMcPLE4ElvRbr7ZHZ56c\\\\\\\/Z+N7jLO+KgIrS3ueeLDUpIB1CRihWz4F+HY8u2ypYco6x1hN2faxy1fvU4NGdepGCbAhRdeQwMjJCXcCOBkt55\\\\\\\/vj6AytNCWPVKJIZ5Du\\\\\\\/vIKLFqeYiHoKv1nTFeTBmqlLDl9E2oeWGMoZABhiXV952Ehygg51wd7pGs9+4jplmCLxqRYlL5ac8Nobmza0B+L2nZl342wdOBf+fU2ygePXAZ62JbkgmkoqRgbzf3GpjwfAPZqFxbepoNm+IvN4ZpPpqlZNAGDjfkwOc2kyE4XLlgmsAMM1IvSkDXlMobIkmT9bKb8y0Khn7FDSWFSB6N21DjqDJph6feHJ10xa0hJVh6GuHwItTBe3omTw1cBMqjZcSm+M6bAsGj53gnYMqb1wvfx8jVuxHJkbwkmHzQF6+T3QH\\\\\\\/RzLIYAx3ZaY+4OSzJDPHdULjVPQDszhX6DT8QnqynCZv4xKhgeJ0FmL4ffGopvPLdFuU1acWD5PBn6gSt02Mt4SzzpZLWkfWOcyNmZQkot0uNLT\\\\\\\/pMpI5+k\\\\\\\/0kVBTSCgTIvvJZjdLwCAwzCix76Kb2KFwTaIN6CnpFOTdl+6DIrmQvqTxZ\\\\\\\/DQdKN508v\\\\\\\/GXCrDuZxCN9jPtBsDHzUrLRbzRck0QR6HWGVRNXOiLOCFvMuQ0w0tHxxNYo3Gyzeuw9D+HtAy2g+q3AmAUfK0k2i7hWMcxc+pyBuT46bLbfouypU6tv8CIo+FqaCa5f8qVeclJmRmlkqucZjAt3YWcHoVAqjGsiTGKbFDPQGHo+RMLqDJPr2h0KbgIKUh1Va482hrRsoD4EJAapwVEYBgQOwOR1sA6BnVKgmXlC7QzRvCJOtbo1tGjvkf8XFnFyrvpvwF+ukermL5r4tnLmWEgaJvz8uOP3E+NPGs80IO9DmwzUNpooMaTAyq\\\\\\\/QrbC+5iHxUhSeH4jsNgw06Uwt\\\\\\\/ExoPceKEE0EmMUOwBrhpY8ugxn6bse0qpeCtUJN94Rh8+iXJQZSDfBcYMnMOWCbVyNol4XkVFU5jcCQlgVEmPgfA5WAsyxRbDcLdU8abpR3wKMKGQzvxsIgZnrrv\\\\\\\/ujbWtlCVdxTWqJXILoIyFB7GS0GloWdPum+jggOxKZRWoXMenH6olhHvnJcQD0AnBQ138HSxCanVs8g+Rl\\\\\\\/bTVGD8zgUyeuujqF\\\\\\\/7Qvh88XJw8Hm8ZcZSG80t4RL7eyTOnOuqezV2lhqPMFeoVF1sO74tyvayIq9g8XKAs\\\\\\\/lXdwyI1fWOG34DmSkcY\\\\\\\/iLcQWO7FFwccDp7Zje9NhtgOxvGzCPUy6PLl5\\\\\\\/K0VljNKwZfdXc5uMlXqyvtjn\\\\\\\/9x4dAkFi4jnhaSzBHRgKXqdSlaq3Gn12bLB8UCPVar3Rq9Zc29AnVyIiNiCzHDj7fLV6j\\\\\\\/8++qy1Qvocap7j6SG06D5qp+byJr7RkDRP0WlmAwo3Y3wqX90I65WpIKgsSnNkWvlYvY8kOckrSjHFOOCZ\\\\\\\/VPyMY\\\\\\\/HBxGK58N2p\\\\\\\/\\\\\\\/UW9+OOsAP6FDMo5afH\\\\\\\/Y6y4hktUt2nj4+opMKSli99jNWC+S7Fw6CmfHFMOesBJjliwr0MY\\\\\\\/ZPWOCcpOfQw9M+JeozKsr43jWOgx+lPLclnuGIA1gGkYJI7kjIJ09XVNHqyQ\\\\\\\/BFjdzSkjiGukF5yCKKUPTJW+FKSEMtkvxyup+AMtXymtdoD1F8gds7ID1OmKMd\\\\\\\/RHtaMs7Xb+TdSf+lZ3OHli3KP7LZ5B5zHahUIbfLUuqiPOGHl+r44e7Ydq6TlCHFK8vZpAUkhNFprNWvKMIk79OiY1hvrknDcp3XEK2IPlwIUDNHp6vlzvvid77\\\\\\\/rjimHJD8V3arZnqzeWbMSAGIuuiEySEdADV2jOO+Mx9voJLCRoB1aqkThh+ScK4VK665T4+0CUdmPz0YnzZ42CzNwH9gVnswIS33Swnu0oIzbYXdEvhbpMxjCc3hAlhz4X3pZFxX+CdBGryMi4Pu2DUJoT7E9KFdmF0k6ahrEtmGTWZcnd8hwpmGY536FD275FWiJiTfC1RdJlU1zF4sE99w89NWrvIiQ16x6rXKFw12uUNLZKHnQuCa4mV9Hkp1cS68HgkD0jgVihiKvfqXHhtzeEgoStmMZm7z8d9hJbmFVHTzTKwRSvSfGkQiWIR8bLghSFSYDfvzkrZJw6fYTxI5HLF3O2LQ0vmeQl1e70tTUTSb+vj5ZCN5R09yd9BUTOMSRu2vYMU\\\\\\\/rpjLgtjyCojvHf+yxBLVUmsEoSkjlEak1dxK1yDxBM7kmROaMM2sEb3zWUkJBJk+z3T\\\\\\\/M66E5H32fnuw8FRN\\\\\\\/UeJoZdtA+DcE3b4WLKnbhSVd8LpCj2p44MwLmHHoEbTcSihXKObN\\\\\\\/ijyE0PSwydLKN8PvEbnkUWhJlC4rTBLbVZCKpGfXa78JneuO5funys02lJvaUeqPZH5X2kjXcHm1CakpeeTEjpqu9ZZYj4PLZd0Q0XsH0MGGcwKtOBbuM346IVVe7npestk8ijKUIDjdsWfC2TN4p7Ca8+vLYJqomw3gWbAIA53zFtgDAa0kWhLZJsEp+5WZEip35IUyM3Ak+ZcXSkp5obUw2Isx3GAO1cMVN6gj+ynMIb8uEpgWO2hp0XoNAlyivButxKRxM4ClVUsZXXpVGIwtPw9PzJ2KIB0qF5RH1ibUxlXuOUKq5qXE5N5Z4ZUB4UJSl8tAucogFciGTw9GMePGMY7BikqFC4a85aul50RfzLabHaFXfK1l1Y5CmXb+a5wjc8mjIdCzRfUgE0nJqWISQ13Ni0GDuDKOT2guSkCUKL8a8jlFSTySao9HE+K5Pv5q33CUiA1h3\\\\\\\/5jcsQTXO12mVB00qarMd4vTdikqkQGIQNx3qP4rqawwy9RZPIC357REYAqha2rYC+9UV4Eq2382CrkZ3lTcGKszXAx85zo0xsh24YoEaoLV+6Z3w48PYVju9+NUcRPSg\\\\\\\/K0ZYdgR2jk+ysBqy+P1wElN4CgoeNEUex2n6IbwiC\\\\\\\/rGG8VCWEGNwYPzCLDIb1dubSsic6iCJ8s+GmqAI6OkfYB2c18WbrzYcG94R7kj7WHXsJWfmXNy6JPBzWxRbv9Ee6MLItybgm64gZFSYRntb4CQUwCu7PpHVZF8u9kvVFiUXNeZrCg9kUgAt5Z4S7DNAZTS8pubKE4LrBfo7SNZEzUbWVnzmh18fKp\\\\\\\/OBtnaws6okzftnqGXk0Pzt4Bj3TJLZ0KewviqNxTSODTyx2QPXEEpT6UNH+KKhA\\\\\\\/ruTmTnlT7FYaw0\\\\\\\/hV8VMYSLBYmdBcD7eZoNnmt5\\\\\\\/BuXm+DWX7ENFRbeszeS4FCJJCkb4BLn2VBfBDkm17rtFqhDoOCr8P0JYJQrRZHz65QEEa2aKoWrJ\\\\\\\/n+NPM3ucsFvgk\\\\\\\/5coZCltLbrZytL6+gpB6SSwWRaMacqfxYtbsMivoXTf9RmolRY1XIKAYRjw3T\\\\\\\/mdocRWP7Uc4xZDAVL5P3LAkGvjNK7A+IGLvKP50IDkXjlx4yMyj7r0ohvx\\\\\\\/z8UzKFQLKLYzZ\\\\\\\/NqxoaMGoA9nY+dX00sDVjXLexdRx5rigVfFdXfqCUzx1Qf6DKSQmy\\\\\\\/iybbNDlCFNkLyAM7+TEVOnjLHuvmTiryliEloo+K+RlHcbb8xc9L0\\\\\\\/\\\\\\\/Fa3Vw\\\\\\\/W8S7HPHXiAa+MIlu1y8\\\\\\\/EAl0n6AnlPpPINU2OVv62pPyWId5Yl31IDHPVwAsFKmEgwRZ9VoNNzmdLzaZZbaVQ566lwYfUfQcex8DCB1xoWN5FQzEvO7aEpwOVe90Ozq1n8\\\\\\\/VWZL\\\\\\\/IzhSx3nMmfb1JLnonrK4GnDu0bzQyIv8n7wF38JW2WYhz3YyeMnCFUaP1Dk\\\\\\\/6JiuXVS1fqTOL5h7LLQvupUscqzm4ZfpNlV21gQDXRhfFoWWoaf12UvjPJWG+6h9SNw5rUI\\\\\\\/y+xux+n+bkPhJl1Fj8RV9IAKB72G0IAUTE8KqjudsIk7b9DxSSmLKgc7KwycG3ooutR0pbEiP2vi9ZSS9FMh+u4vJGc5fiOXSQsj1QkaFeDy9nz6v+lVfa9l+qrz\\\\\\\/gxTPKk0yngRZBz200NKa25EnHte+GwwW8cQKSEA0gHiX0R33OI6Y0u130Mt5uUAeimWXIn\\\\\\\/kwKSA2EWpCL1qiS6wvffF7eYURDuzkAHqgla0+jPCNIzjPj+gFBEA1fIEnHt5DerdvR2rdMyrTIVAdoq6sLC4fgDSoFhroYFRklnfZm5QYDLTNRlIz+cGgVuebS4yV4RouJ7+084Yq4K+wGHZK9w7ZL4w4pDaLQ1caZjGQ0lOdD2WjWF+J4+nKupCx5LRhK8tvD8iseXJC1auOr97RYMSNkWKS9mAjhTDqjY3Yi3hpOCj73Si2U0fCZxHICR5\\\\\\\/RmzGrVVsvIfeuAKpsvj4roRlpUqDTgv5KwJspFBqxMqwGufI7Qd1v2W9CwJIAEfyB69o9Wat76Xunbe9R6m5nivb+bPQlJIGJurl3vvM9LWtUht2w4Tri5uwIn5vn2b2PxjUlsXSyBBj000g4wsvjL1\\\\\\\/e+ITAaWsHdpnqiDOisBK0oYh0hItXd9RwlSFfkQn+sMMQ+d4RUxBGx+ybo1SV1sqwd1NIjD8zsNqUwLkf5dxpxrYWaTydyB+WP2rfrS+CKCLTVz2QjxNoS3r0V81jzg2gm8ARIWkvJKi0UjfqnaWPeH+wqfhAhnOYr3r4M3aM\\\\\\\/WP40NoXTnyYnDvyXA7sYo3MN1u6JMIgCTj+teUimU3WcIFck+zfhnyCyfpvzCLw0u2NlSSShOmXjVyQlhSYuqAT3y\\\\\\\/Rhc0qurCOJybDZsg8nsCsJwHNbvkT6DcULT7tTu+iv0scd5SgN1AuFqC8tSy+\\\\\\\/XB9TrQuFy7\\\\\\\/ZtDwwgVnLuu8b1xiK8lJzJxQcyDLSx5F2UbfCzxoG23a2u7etNKoq6H91bz5\\\\\\\/U1Nvt7NANGcijouV7LqifUd3msBha8oifl5w2NlumLnavWD\\\\\\\/v7wkEglATluWxLRAwu9Z94+3mcWGDjRXaF+wGSFOzlSeepLwpC5tkA52sGPF8bTcB0fR\\\\\\\/n69uZ\\\\\\\/HaadepPwflgVcpjIi7+u5zzGbO3qeyUR6XRxO775OBLNWKEU6kqdJMvVtHTD8tcoRkI9VALwiyms17k1CcgF8nqQyi\\\\\\\/OJSOP3HWvSS4ystj+tFRJ1Bmf9hqdOEb4\\\\\\\/cMGASFtb\\\\\\\/DrjrxTazEwamLzYbMiuxWanJL5DekZQRqTL69Sq7iFytWqBITUlVM4BpektXRbo48qYTg4paKF7MXXQiRuYHG6FXrff9UVCs4oVLD74Grub3nT8va2iZqeGIxd71GUyqi3PckkG6aRJluVhmbM0HbSYzIBiNMEy8K0kpZYKgps02FnyEkkbgtQoosKp4YJis19GIMoW56E+tI7JD6YUMAH4Cuupz1o5USjxqAGMx6J66FmYSRM0xkgg20bbrbW2+\\\\\\\/kSUi8dwineEeF9Ct33Kan2GbQ8UchNUngwDBqTm7nbv1adX4ZKDGg6wfWKMSTjfbA3SGZcPWniAkma2+cu7yFroJ06WUwr\\\\\\\/p1y\\\\\\\/xwP\\\\\\\/myb5nc5RCYXNoEn\\\\\\\/VqMmBuEjG1l\\\\\\\/SkeiMJu7KBWkESnzzawtXRb2asirSC\\\\\\\/fjtUSMMpNjd9YWsOBu4aJ5ONhFF2UQzG3RJAd6Zt0W\\\\\\\/qPFh0wnH66b8U+0MIqdUJO5Eh7g5ChEuww9K75jKgYAq6s4cTrFX\\\\\\\/QCta9Fh2Ax9haxsB53eriYWb9R+B+Q16Rz\\\\\\\/Qeu6Bdj7EtdjTDJQ0\\\\\\\/Ly85jLLl\\\\\\\/m56\\\\\\\/Sg+5cKWChiw93usRBDQ7bAqmC8GXMgfeD3w6DobolFF\\\\\\\/ABJqPpzucYpJxWQiiqZMwrLCwG60vV++IytsrQwzfDkTNVqVXGIPTcmQ9nR3WiUk6umBNdjZvYRAAnxBnQDiaPYnxzmX4Kxj\\\\\\\/Sh8ETT+2EZ\\\\\\\/cf76y2PfF7ZT5AYJs8RAjZfpYLHdVemHFNK1nV+h0KY5c75Z2zBUjDyh+YiPg+5rIn3ldiYXNN8WOynGZqVGbc5xrVfwdfgr4xEskYnwwaktGdrllSgpx871gTgNwHO8OJzBpxIBIBdMXoWtDhuqAbNSM9PS0RGaZbVH+y5\\\\\\\/6GOs9X3Y1pwGpp0QEt4MXZWCDpcSgMmDYdBg\\\\\\\/C+IPLXA3w5e7V9pYYEPXQEsiC4+QbhJlbnZ+QkQqe7XaqnTrY06ACjmmxPJFyZj1x3r9LcER2nCdauSJ37bi7md2\\\\\\\/oTp0ZVafXuFQaNbV2jKwNQiK83RWrXM+szo\\\\\\\/UZuwMFioEXn54\\\\\\\/rjcn0siS6MspItWwuZPlP5Go4JfpBqIYrdExwXIVLUXB1HmpGvYUPDvMmshdoNXElzXISwp3DDUQNmdltTnDtD3j\\\\\\\/gKgVMG69sfa\\\\\\\/Qbj9xkmTmHSKQdUWMtbKZTuk1DsCCTHOodrGHGCpgpVLQdom7rDCAx1kDo52EF6T0u02LGwBxNh8UwogLt+49tvwLPN1H4py1LwrZVX\\\\\\\/Cl2vzj6+Sx+4vKP6oxwvip4iLofAKhonGa1MD3N\\\\\\\/FH1W8BhazbyYi4wF+IT8grOjehRS0dc7XZFy5rVZMWXm1eIhX9z2BBFwq56pt3nyUA8+6\\\\\\\/CRiSgnQGHSFE2fnqqU22PaoYy7gKrAI1OlJwltAX4C73dE\\\\\\\/AKohIJG\\\\\\\/hfs9WUkClEIAlpbxjS06ORfHNiZCa85Xt7U+ec2TJN+xFvcKQK9aN9HBD4wEI9WfXhvYOm2Zo1Ouuo9UvMlvVzgWPuogf\\\\\\\/P5nNA\\\\\\\/EgQAgl6i36P+qqvpHSUo5dji8oh525uN0NwjUrEjk8xN9rACVGunoBXPWb64uin4e1SFlGag\\\\\\\/vsaT78WS\\\\\\\/g+EXcjURoGU0MsfJxQfJ0NnJQPEI1O4MsTboXi\\\\\\\/ibn42MPs2pWqbXrrM7MDnW9wxmBQXbS4jS+DzOzxJHdq9m5pjp2lJmgmd5HGa1JzjQO+fU8ud+DXC1Nx\\\\\\\/kiVbojqUhMHAgYDtjyPpqEOqGs24gCJZBo5TfKGIuB7zRQFVRBYOyZJT4CZIk0RF7TjCd94jWv6tQmKds3\\\\\\\/fIOmsHVSAt1WIoNeSPBrCOP\\\\\\\/ltl05kdN1TBvf95xeKsGI6IDfTmiEdcW8NWYulmwHRMuhVSgUrIfZLWKaq7e0lYb3rDBy8XovbE39BzeBg+gN3Td3RPIRGBZUg7swu4Ph9HqFsQPVOkrmo7Nvcue8KcCV1\\\\\\\/amhKDsEksFmTXXk44kG\\\\\\\/1kGQxQ86bKHEolmQs\\\\\\\/rV3eE8yELXE+h0wKt2WiT2jZOR8CUvJLi26++kiJBORWF9wERreJ1wlxePUgummIxsrqQT249fpNyK6yT3coBupZ4RaDNTTf3I8uCzp0AgzycDTLI0lVfE+Gh+IRTJ\\\\\\\/UeJ9nMduTKZWKXnQsq3Sd3AWUChL1nE7AlymoVzoMby845ekgpdLhbfclniHdUw2zK251NXMM4cYdLlPwLxDW6O9DbgqoCWnOynGmJcjwkxyX68ke7jE7N78aAy0WwlurPopx7TmL2p104jeY4S16hS4DIcMIgqoTBqVbQlJn6U1+kubvDyK4uyHMgrjWWTwQ2BYvsysBqoG70a6EiCHck+BIGRXamheLNUnyhklUSYtE7LFHj3vg9eoacgP1o18CHSYNaQhE8K3KJhvheGcGCa3PAQ32hFaOGplydWlxvvLUj9yYqnOEd5\\\\\\\/4JPkrY8b+5bMYz2iP\\\\\\\/8cXHV7xf93fHSorHM0NNrfciiG6Qcwa7ceav3dA36TML5V83m4IdTuFb10wa9+owtpm\\\\\\\/d05gSGWTXWzOnndfK7P146FDFRcbLzEFPZDtSCTyi\\\\\\\/7fdjm7XZukS\\\\\\\/lVUgVqGntrm59OtJH3CG8KsfPGpqdkMe\\\\\\\/N8XU2i9I1bcwyoagY9uXUrUhMl6H9rLtuz85eJvd+VwfFog6ze5+pX1NcYU27Msd036vjvewVVEvO\\\\\\\/LsfuY2bZx4BnrL93xYZ+xZqkpHJd2sx+d2Wsn76q8Up+nySgWZeG5wOJuEt\\\\\\\/GmdznxGh7w9vk7dsRINBmdFRsGNS\\\\\\\/6va+G0vndzBwec0chLZdX1bIo95BrJCAKM37J5QvK\\\\\\\/9Zk4ddZxHcMoRSshg3yOrT0hyFhG9AJwFZ0RLEFuK+JREia2tFnJqqrGdQO4ouqMsum8lepxPR2i3BneeLLUpN+s+C2JxKLuPL\\\\\\\/6lPTMLNmZypellLotRbyJ\\\\\\\/H+3WOKKGVHfdTSgrgHgpaA2JwbxPAuO514ZihofKwc5YjSJQL9yy9j8oW1j2vFhvQQL4nQMgeT6BU36Cf3EP2wIFKCr+9ChiT6\\\\\\\/TsdVYu3JhjN3jwfIx8a6APr3cq8tv7\\\\\\\/uJznz5vZ\\\\\\\/lG75KiVEFLwxLGHUecu0E7DFJsUO5owSKKx6ia8osSgOoVvk3GU0q\\\\\\\/9L8kt3BIqPLeOL58S4JFki9puJ9O\\\\\\\/w6HRd3jRN3dE5u+QOILZniHx0zF3qswr7eHGpxbQGaKanjR\\\\\\\/UXTtf7CkNagUiNAOsi03P9IIncxNyLfxwnCeMUgkwt0FkxVfFAtq3r\\\\\\\/b3GbFWsw5VQQmQnNWlBVhd1OEvEPaSzyXsKbqD60YLokeG8vUPR10EzRfttXmZmdbWY2l3\\\\\\\/Fk0taxAjF5Jnyk35JweluunPuIIkftyuth\\\\\\\/JRGG25IxjNIadpbCRcjrSwHjIzZFBH0LjLqKG07O\\\\\\\/nlca1TySx5m64QdgUX2mEm43rUU7PKyMLrqdWFwPn5a3PWtQAvIWYkXwqyQWyTCmk8+v7q2xUR2I5OUPpQb0MjbQmocsPja3R0SOvEz3WdBacbB0M\\\\\\\/rNFJTWe80CN6DKrxMJGuSROWjrMPZ7QfJMTPXqb8uYulgm4meSelYkRjXI6KR2MU7VRCs\\\\\\\/ASeYZBwkOOWVUa0DBgTB4fCnMB56IM2OJtsZcWSL1vheSujugnmLXLdmF767gGRtb7+Cz07z4R3hqVXnErONYZORboiKJpynoaONhXPqMysrp6NtD+WphIr2b7+bxztQcmKt7BaDFzZJh+RlWM6+4k\\\\\\\/ikkkElGaMWPTR5MvBhiNfv1PCLCc8Sb+DFLD\\\\\\\/uMNU3K0jC\\\\\\\/slcO7Hf+OP3BrrQCssakXijNvRItybZA08FYqlQJnho\\\\\\\/clPjUa1E0RTkRBMGHKIqorENtVAPBboUfXLA7VbL8gO2VlJWU6xOmttE0EmOnRzYm4DK3URRA2VAzsTBNMCUZU0frc0f7mtJHKNwu+3yyRZnMPID5jqyoe6YjOIkjm\\\\\\\/z59ZO7LlP4cNhsmEe+Lw\\\\\\\/8dCV5EAgEHyqy+VvX9IqayUFb0nETz4KC0+AmTDYZzSAbjXCn4RKNsa\\\\\\\/eu\\\\\\\/mqv+BIyh+ynYU8JME8fTqei+ChUzx3dzy7uOUrk8XFyi71F5Rt11URVvi27npqc+lk+6AzUiSssMoMVh+aThA+lX2F9527opgv\\\\\\\/DcR2zShD2MDdDIecqlgB1NXT0+4+bGpMfLqkRAMh8da\\\\\\\/YrIgTag6AJXfr5GF\\\\\\\/WM6+ga+zlA36QaRjCQWvoCRVJXZUm5hi26Jq7WM8ZjjBKkbnZukCuPE+JFjA+392BE3eyUyD7bMkvZzco8Eqtre7C5eOBBwm+iCp3WWwFl3K9csgzbUKTd\\\\\\\/dNOqevHIjP5UUBQCPOfu5b8nCEjfAZ+QcSdc8Yy9BduDTPLgQwWoFh\\\\\\\/io8YwIdsfpDxRwPjbwdaWKISMRwa4+NaKs0XdKZ0emNcPyFIfzQExsix7zEPJLS\\\\\\\/rbos06BLpK+t5b8LA+drywYPaH+LVjv12aVmtCu88iZakLVcuDfcqsSDXUQxKBoWIp6\\\\\\\/xvRSp+9SkXVafAdJr1mzNUecDFmoPnCPo9f1THgn8Ps1s\\\\\\\/f+emGVcBSXUd8Seqb97xSWMXVlaADXkxTwx2\\\\\\\/fhPh8xco7K+WtMZjZLR61f9qYLM86gFs+9P+S3TAZx61Y4cRJq2OQNZs71MbrHIQHaYaWdm+\\\\\\\/RNDM1TJieib+mD1MOlezDz+KIB5uVbRcdQ2C4lIp3DIu4T5x\\\\\\\/oQsROVzVhkJGllXvaYIVQ6pct8qGiEKeKtVVjDnL\\\\\\\/X4r3Eh6m7xKOXW28Yg0wkDDPO3rhPhQA44HEiQ9G9KSSl8K7JbaaHs9ps9BaO6rLvOvs1aVlftUWPnaF\\\\\\\/VnZwSQBKFfrM4MZlI\\\\\\\/BXGuWyH1RAKxBPFfz1moOWmpbKpSI9c\\\\\\\/E8nOdqaPr7Q3CynlE09SOFKFAbqILBHTBcdGdMyafS8X155mlFphRTF2kBNd6sLNtFA0Y9b5O9ncy7AL59zVHy1t16s8UVqCXWWCSFa0f0yz7q7cWf4Pdubcki8ibtDqClZTuvlHe+6v97Gwx7mLFQuD6kO\\\\\\\/Z+YDrAo2tvharTXcbl1HfKKayTOzDkOc4V3zzhsFe3C3kMXpcqQW6DoNb8+zcR\\\\\\\/pVoIOzfh+DcQ5fjTujgmVAViIv6QLbZ84NqPN\\\\\\\/5R5dG296YlARJT3u3OlsVp8QhiIrjVsI0YQbVgAMCxBjJEg\\\\\\\/kXB812QSeOeMYKIWZiW0eJ1TZ53UMzWHOra9Bf3VnFbCc4qLpxElj6ey+eJFBbpU3xPEMAxXwLc3UR1rj9g6PqF3vkGF7sjjEcAPvP8wyPP3pNvN0A84V\\\\\\\/+XcvCUc8ewDBXPcHWgtl1KTJhE4jOqLZEdwutgWIUuvNd0\\\\\\\/99i2z3Dt+Em1Zw9WrUtzFsie8\\\\\\\/fL4GWsttFR7NxhCTxzvt1Mi3lgsb8lp8y8B+CpP\\\\\\\/Z1SgwH2wj7Jf+yP3tbXQ1WgjBzb+Nabhend2He\\\\\\\/hyjoq38c20LYGTJjzNhkTf24LE56\\\\\\\/o6VTGRSwwxk1kEIEvL85QUM5qGNM\\\\\\\/1zUBjtPO9RDliJ3HqjBOtdosa5SmCykzf3lVWfvpn+VW5o5y8o+uxaPAGeYryOelfBowO9ckJBD6PDTM\\\\\\\/Y2R\\\\\\\/xxvVp81q0aKOeXkzFKKfMvU+QHQXDrE4QDMPEOuVTv+R7Y342ImTHEOm50o5VgNR+XXe8e0Z8\\\\\\\/LLp9NiWyzn5ziRonpP8ZcdMxH2SGWoYxfUmGj3bK+RSDR7Bzap9x8XXDLc+QmPk1qRBfsuTB6pPjxcJa7U7dwYGxbYfKIDRM+83wrbXAyhOy\\\\\\\/GbDswP7gexPwmDBXR86aGTR5tj9ifaxAB0BQIvFS0kv7N0AamitkfRlDktDGWTrzLEcWb57S4m5Dok+J6FsBv1vaO0MyfWEi28xvkFCcWdgkBuwjPOX0m7zsdeJrNhp2fbUKwnJEqOzYs4SRbgQNwOXmAO4zD6IgPCdj7yhtYPhXKJYQqftNnc+ghHpoyWgWgLbACwQ\\\\\\\/2qo6DekrpGYrE1zvbkkPImT5aTk5KtSSI5rjzevYnQOz6y1zhCZBhv06p7BRF9U5NTLv7467ILSp3UnENUSyhw07oKKq\\\\\\\/N1pxJre\\\\\\\/bXECORK2g1rGBNoV0ynxqQ7ZESuNsCqk1kUEdmlTUw8bVz3U9pQSwkuLxpGvWLTb22EvalAnjLz3+IZBZUQxWlO0r4cXgDyf4lg4nRpUn2ufBRX+GloL\\\\\\\/5H9n2kCN28TGKQhf3UQ2OI+sYWZrabm9K8HVSJ8lpJR4X61SjEhIODWM65Cz1\\\\\\\/xQxvaBMO9EpsFlLRsJ8rqxDKqCKH5iEEoFYfxXHVAk8BjhDNrGBr1EnhtgqVNDqj0lqaLdfrdu77V2XGCLFbZXFWWfDME9gyB\\\\\\\/m8k2yaDBhGJrKUwgtNJ+NarNnwdi0WmXd2ijZWzZFWsZ61mAxiOe1uGYbga4tBVyKt1Lp82M1E1uY1dFZXii72TKwlkVyy7sKxNYoXTrlOs\\\\\\\/dMXfCrQCmGGv8ZwXCUr9VbFhge1e2NubIUl+xq1DCGVxaX4Z0eC6ABAv3k7yVfmjJ\\\\\\\/DOAmnFHSVI5Fpc8Iv\\\\\\\/mnoVl64vba79uKDWSaSOvif+Y+gEOVx\\\\\\\/\\\\\\\/421XjiGghBh157stXl5U2rFBWSal0vW4B+EAocVWcalOK5aDRHLvbGlGWuYhmd8fj5rg7aZl35zZrRPG6Kf+2vF\\\\\\\/vWDSVTqo8IBbAgt1Fh3wTnGXQuBMoOLdfOq7C9OAti33SbV7JZbArLC8zoRrciwCFkprIjAJKYB7go2QUPgMzwxA\\\\\\\/r3CWVIdFrC+x1PbaHoIL3FI5Z9saC9UZu4aI8lQIWZ8SaEqyNAldmMlzjA1Br6c5ns\\\\\\\/IXrMjJkgX2Mg7rjHRRK7+rkkJjN0S1jxiLifLoFmiS55lhlLzrnjpASVud82DYswFfqoaHahOC+5snjyQunGkNkxMhU0MlsuOuiIGbMIRQcqy7AxINEDaW8ECSg0Yn7Vezik5ySu00r44sgf\\\\\\\/EmTFiMiuOyRZU1LIWvm2Oxe60ho9KhtkWYROr4U89czYOqBjwlf6GQVIPRfn536p7FCLsjVV17hxdCxJ53Lwe0b7oG8mYcbVq3V1mOOXjj1udNSDMZi\\\\\\\/q0SutM7VnMEQsdA9eOY8DXaSlpus+yY4amzXlreNxjK\\\\\\\/M2WTPY7zP0hSHVwFSGHeYPbviQFMtQVatAReG9M1M\\\\\\\/FNH0MuXtZfEwprsicNCkyxgIJBIF5g6uqqHQdQTuHM6MdaYB\\\\\\\/iSgjwEubua5g1ZyVpDKj5taZ5zeTvfVBzli\\\\\\\/0TUoM3eLuSrI8+v+kSkIwibe8K8udRrYqUIeLpNNkfors6PIW3gdYKVZf82qvphxN9d\\\\\\\/bmTqPUo17+wIckFSilG+jRY1B46dY3hjRIpYJrcFiSoW+x6HnfG50xrhF7e3s8LtzqI8melWR546kRkIhhGycPgCdzv08W8V481SHEv\\\\\\\/\\\\\\\/crafzQM3nq61Eqw5p+fYS6Z6YRIOiaMayFu6jCUssznlSpiETOMFibk+gmUVLwkW6hGyXXS0kUEyaO2D7q9+rLwPbP+cpBlgRk5rsy4asyz2fvk0rcaV+PAgjx5PI8jtYpFFZICn0LqF+AtLZGyEeDRZ6tkqRDLc5TXd+jDaOFtBIqX8cMaN7ykkxEamb74fjzpBqiYK2i84rqq\\\\\\\/45By0JkHVTMlbERNa7SDaw3hSHr2u2cyf98sx6SOSWhcV\\\\\\\/2o+b+YNCL2aS2JFUt79pvlup4cR3jQ+mfB7ElyMKoazkHUiVL02F+Jrh9hj7ZkvPGuyjT1EI1wF2O8T8G9a7dNcYqx2CtqymyhtbPogsOe76UDGCLGHoo3mw7MPIE0ETROFj0DhSVGrs+kW\\\\\\\/3aAJUZmZWSFCaipHE6SJnziGBEkcNpv0qwqvZoQ1T+HtITlXjQ+4OttN691obqDq0kJJnpmX8HCc0IEbH\\\\\\\/Ez80aFkB+ABteuGsBM2xlBN9QgmWXC+KdHUdbexCFiQZ2RRVt15pbnJ\\\\\\\/\\\\\\\/J\\\\\\\/v6qubPu2gdofGOGmfa4QBl9OiwzVq19vgyebCXwT7Yn\\\\\\\/VdivQfa2ndk1+gQ\\\\\\\/ZJppY\\\\\\\/3UHlts5clWrLO2Lng\\\\\\\/5Xc0\\\\\\\/51vQytXmndmF8THWcUluUGjYYGClUR5Yvdl38joOi6lvvKA0PnQkwPvNuIsJ3AeEkItmti21Og1RMqvDAokyz2es7hYbwV4MDwLxtLfsp4ajxqdaBgaRcWB6OPhPjOKfiPZaFnnm+y++xQjTQgerNZ2P5QoYJFFhKWgtvDCWv4aVf8nOksES9AQnDKACRh2awo1IJw5uM+4Dla3mTqcjhk3t1jjvq+eRnjmoKj\\\\\\\/LWJi54nqvCFhLEFP5XCH8YoF99AnO281RpypRmnzt900Af2JMMyb4Mxqn6iu9mtT9QAnSgpUyYP7SwM58HainKyD\\\\\\\/qhkuGwjPkbZpKwn\\\\\\\/HA7SjE8\\\\\\\/WUGBlhsloC6IH9OdKxzSS1dLUkvm1dX+gOyJDSqIUL+ahXxgGhR9Ml4ztB4XP+toGZAU94Wnbptcsts54wS6FGyLDpHWYcnihkdrbfVnD9t7yEeyLQcHjd4QzoJ5FtW82JsUj3bZg7Sf7HyIzQb7cSRHuZUYexiInNXL+1khSpCGIFgCdbqGH6kWupQuwm6ge4UhESlcxhpobpl7IsBlQi5WDw6Ltjd55CSLgY72B83G9\\\\\\\/Ec44e2hFYUcSbndK0YMkFMrN+xNWpRzvow9DuLTIp1fECFkqzdm8I74Mx9HEKX3Y3V\\\\\\\/0ePJbYVpPkLKZQ0S7qDb2qyVQjsKaE0SA3q8dficWpgFgiNr0mDr\\\\\\\/7lG+cw5s2iizZWeyl1Ytgz0q8n6xDU595Zj68b7ZNqwAt7TwZrJz8ZGrsjUByL7eg1vky8r9d5ZQrwfn+jUf9g5dA9wP1HxwYXxtgglX8F22cp+z+581pUcAxOafUJOesYR6W6Czi4LOUzCv5Abyp\\\\\\\/FhI62xkaJ7XRv79SX667nZQs8WB1LgIDkMUEWSIgvgHaOz1ZzRSIH5YK5bH5deeZrG\\\\\\\/ILVbOZ4q7OPMV9eDi6gTxlNZewra\\\\\\\/BNiP94TPmoSMdvOUSPlJpyJvuZW23zcqNzkuPD2wJwVCFLnwJ9f5OrtlNGiiISXpgBtcMgZjcjwRbbh1fShh6Rd\\\\\\\/jkGRpsNfm13jLemuED4b1DiDZHBpAVJ8Xhu0HNYRGzs+d\\\\\\\/KvYjVPtqitW5BpYSE4qL6RvGm68FfedVv+r8j7L3Ja5n\\\\\\\/XCY9Jg0pY5UyhUQJTCcNiB2ks3B70wwXlo68sGZz2Q4YzpVtuf5e0lF8VrihRPglaX0BNnl\\\\\\\/uy6mvd7N3gy6ookWNh202MFEIMdyYYIivPuU\\\\\\\/ASy310069z0LiDdFpCJtuLBl+9\\\\\\\/A7xzwyrm7VqUiSDeVO2hmLoph1QeAeCMqrv9rF6nAxq36\\\\\\\/zeMjR7KpVJZvR+oCPCIwbvHVmKFHqbKWqL\\\\\\\/YD543PKvJoGGtYq9fv9\\\\\\\/Oz6h7mX\\\\\\\/IEUxmanaPFfzp19SyfRlM1HhmaOHwJ+X\\\\\\\/1Mxo6xv7dO24+Qv+\\\\\\\/IG10DfsDth6IRN6Ih\\\\\\\/R3SqDNf0IOjKckEgmPSZNbaZfYi\\\\\\\/1zSMTPR+Exw6uaIMFdv57HUtluT2pMUg\\\\\\\/xM0OGZvcrxNkcnqjgGKehbkMKAwp9QzYJ96rBqXbRLgC47mvK1ChTeSkKn59CLZGpv7n\\\\\\\/deXNvDiOLOLLO6SgYBBEB9XNvBjYZu96+p7BsMpQtDA84bBbD2rAILJ5CmAJNBnueySP27ciH7Dv26q3Drpf8G0JVJm9P\\\\\\\/vE3gNLsTBuUYY3D0g0HoiAwirwi9a4iHdRbKO9U58ClGA4sWTCYlpR0fm6NdUpdJboXHdBVWU+iswQHUAVj1yKmj6EyRWT5mmPgN7TpvwqnqgVIuIUhYKHDzpVmCsYlUGknvNC7qnP71SB7kcydN8+odxhgmecs52qqRlCqFrb9EY0nnuyJrSr5FGnpvmtiyV6qfOpkovIqSibtGO4fGWZYErNi3M5iEIrsR3cvpX7Hx3zv9epnG742Irwv032dgt9r3fs53eTPJNpGgG02c7wFXKV177HA7YjCnkFmU2fqsB6EWLEPWLqzkmiB9gHH+335ZqqxOhVZS\\\\\\\/30jHIzfgs3HvLE6cDJ59v5d6BvV00EJOfs972aaM5FLbw6fwFUPnxrOJyp1ArWGpUD9nc9vBbeWpfV5QZW4FXM14wbzoXnTsK01Grr3WWO642NORDPbGNyE6ipIQZAeDLCk\\\\\\\/ZRtvDoXLbJlu1WySkEOmAYKmWuUdEU9eJegxDC\\\\\\\/KMJmj3L\\\\\\\/6nzzB6+02b2skcjG1\\\\\\\/m25Sd7IBdD0lC3\\\\\\\/SupLWM8xGT99tKkbmiAsB8ZibhXqTSqyEj4LhcqqK+IzTlKusXJkTTr214ELwov96IymoPzG83d+wjGfTr18U\\\\\\\/xao13jYCX7QSW1g5ww6UEU50f5VHzp41EyO9bI+RiyPx85Eg84kOnGXoCl42DIBM6T\\\\\\\/s9vRewutf4\\\\\\\/B2eQwIWJ9XwXWWE9N8LWGgVHg98bw7CxgDinTV3zlkdRuyPLLu7L5aeQy8wYw5g\\\\\\\/RVD\\\\\\\/y8R1f6IRwlAK6EwthGXTPquV5NyvmtlY0mHeIhuZqF1YYC4JQk6Eco2iIulO6tAwdmG4j+vIh3ZdI+cZLov+zbnR9PqqdD\\\\\\\/4PdQxZ9VU4x5VAE7EQsO\\\\\\\/U2VhvjDwmXSkVm4AtyxK4DhukNtO1SU8e8qVvRQNsL9TU02O0bDS78ar+Uqmboyi45B\\\\\\\/jegPxRV7RERlEU3fTgpjTR7neJjHfRnSYSLBknqLGtmxxbfSKZnV7gBQFVFC3AbAa5mhwyrUVlchgguuv1w+AYf+ZLZQqIxT74U0n9PBlg9kphG5he3t8w1KjLzPDx6N7sNfQbj2P7Slu5Wz8H\\\\\\\/\\\\\\\/bjvcX7i0k3i0OSuwfhjMNMWvHyXRt17dsr9yVD8MS+JVsrEvUploU3qlr8tE1Ke2w0HdPLaBalF6XUIrBsQ2jB+s9wrdEjfIEIq9Vhpgfb3HXowZeTj+1y1fLQYtttZsif0BNZtumkEJUKMPqC1X63qjAR9DrfhAPBCWjvqlIQ47bK70wW4kwmIC\\\\\\\/XQ2xcLOhgXFdofoy7ZAv7g0J2hsr\\\\\\\/kpUwqGACp4FJIUowtkkJfMXyBeuBERcFZDU4KNimPCd7C73kk7rARbmcTsy5u7welNvEnF0GAHXcHzf1Bbn3RaifOjPGIWs\\\\\\\/0hOVXrwlvSpr1sOKytSpIUbvt3FPdmDOVdEZt2tK7U\\\\\\\/721hG4i\\\\\\\/ckUO1\\\\\\\/YDadv1ZVRBrJxF6tIHLdXwO4mdrYzXzzcQlM3\\\\\\\/YhNaEaiIBS0ew8oXejCljCa1vZ2HKMMK1UCO6HgVQ6dP2OO7M7i5mWN384Jj2XQDT3E\\\\\\\/IG44c57jmp7\\\\\\\/XFknQMf3+wYGdMpzrsN6\\\\\\\/fwe9X9zBm3qODlVudVaFv94RN+qHl+GYxX33+B9hhmUU68PySWqoi+bRVkHUMnnPLikSXYd7Jdj1SckYTDuPz7P6Bf4a5VZLl4hgkDgfJVWzp7iK4zZsy3bmcm7RQpcnL0VOCC8w5CyLEz3MTbO63i2rcGPsfzeUbxg4DxgMoYeo6WEIxTj9EWXueQ\\\\\\\/thXpfgfYfVaO55bEj+nYKSxmQHCen0QSoIEDTRmcympKnftyhdPmb7QLUkZfODlcHt4yZp387Q7vQlFD+B233rwpb5C2hVHEDHYPdpnbi\\\\\\\/iSInWX3nkfoTcxMNx6gaRFWsRO4Oy7kVeYghc3Lpa3S\\\\\\\/6QlU3CCMB9i6dkqEqw7IfZHgmJNvR5viASMRF+8xhXzTP4n9xv\\\\\\\/hV7flS6nvd5Lak7z5PbG70R5zS6cviRtUFgdI9CM2TdkZ9yqx5o+SsqB++DLXx59NkuQHmsOELEiLMPQXycnGrWTL4EpNyDbrp9xfDSSil2nKjpkgQ\\\\\\\/iMf9ar+fzdzfimkzD\\\\\\\/gFGe5eesdbJNlQOfgytfK1G5bycZjKFExSFZXicOsZbsSws53tjrgDdvsJAuxoaadQvyzzfY4O3QWTFTk\\\\\\\/DQCCxmOmAdFvzcqhxWyIxigIfOjMdaq3bK7tClo+pdoPpU78WjVYND8o3IcDKqDolY\\\\\\\/W2Dc+kl7QOATKUDtNcIQBTQ+\\\\\\\/LRIYtQNfTLn\\\\\\\/XJ48W+9HDJ+oetbaLvfi6XWRbc0yPeLFQZXzEAZgpOpsmgKJCwbT7IAf54CjbdLtHfTn3Yx6CUuH89UZNHchrFbA3HOvpqCFTef64vYcOmiOSmq3xU4w2NmqbWKYna0hVI0AyFpwxttbconodRGP3NQNViJl9hPkX3cUiJS1nMqLyB9Z7az79uScgSlfAgBHXxxl2z8Y2YZOSzU5DIvC0PnKYKg0RgHjVGtnQD81cMBLAhzZd9MqjLipCeTSVVM1DXDbJfd97RmoBsa7SO9Uem9FT9eUXh4hswyaxx3zwD1nzN51vnBuNMpjqBEysk4aZzOTeonMHGMGlUfWZ1GkxzkcQul2lcjT3AiK952rBUcQWbWp2ftmhic7wj4p1l3MRTEivDLPHgQPBGO8RlqWhYzPbLQ7Gbz7BbCNi06nfJBsN1mL+JJlgdnupRiWNkJonbNEGGFmnVivxvsUpvILXrQy3l6RZwqyzgvc8lp7bTD24ZDYAuXv09tuzNtMRW9OoQ9+YkP9nI+7+Q+ou3AmRJLFY57tSkQc+Tn823zZesZtJ9\\\\\\\/msbWokRFB+KpedVGKGHWKadzqD4MWBK3tC80BMIvyzQi+jm34GphlNCryvbCq6s3c3ngTdxQ9VF1QzEpaOGC\\\\\\\/wfHXQmdAmF5Wmd3pdTwHJ4X3\\\\\\\/6vOdij4hryaKqM\\\\\\\/OyYiQxtvx3MUZTxIC8Kxw1MYpdcoQazO\\\\\\\/Dpz2bz6IuIsXQiyx1AY9kAuWQufKm9+5MOL6G0reXQ34RfkeYofLKmDqYa0tLkIa+mB\\\\\\\/GF+aF5Bb\\\\\\\/NPnYwI84rWiT9CxvXNWC2aWgSl3JZJUYn0ekpsxwwxTsxDrQdsAF2ksMIPC3eTGnQWqYDypuwsIBjat\\\\\\\/5mMTZuOwH6Xo0Uto1dCMwYml1aJMGgUiF3pEGZieM1BjZVJwNWVLcOFFQN0\\\\\\\/AnhJzlsS3wZOqJP5zDcAei2yuX5x0oVRJIN1HNoW0M2IjSnWyxqmfZtE1hSSlOc7Xt4XU26uzvZfWQbVczjoOjitZrC5fEuHofmIvXmAoTcWU6diQOTc6Yxt7fk4xvlutZ9GwCuDKKdF2FqcaJmGLUwRF6M+hYXIU2i0BMAOy2oEfxfJvzAkwYdcne3eD9tXY\\\\\\\/LzZxgFaOMKhJvgnT7NdFid3ZIcpgyf3sGSDcr1XKHQqI8uAFwqBB77h+f7OZBo0Alv2rcAgXfikGqccl0aJ6V5VE6voo2PAjPJ9ZUHQnQQBRJWDE5dZ+DV3XesXm1sxRR0qMxp7JPKFJyD4uKjnN9EzOEyZyPzhhIgdiR8K7mTHWnM+\\\\\\\/NuRvjDbHpvc6C\\\\\\\/aR8qAV2HDP\\\\\\\/ruWcoCW4dTGRu5XcRncfzkjRSYYLj4nh34D0+Ea1n8AXwOE3XQqxm7qpMFCDKOkcGbu\\\\\\\/7cNvz8uaPhfwBxFKPYI5cYhktdb+fUYx4oLlJwIER1Ls2SnoZwXiZnKxaQhpoLO7bh+dvyULImxx4oFSwON40j9XUrsHHZrdDj4uSLGpxEhFAYwkWYzKPb05K6qZOR9zAmdsCT7ka6wXlAHfs+q+GDe6gRwu1V1o6ARbHcmcqyiIV3djiarJjRsI\\\\\\\/dCo1kCAyb52qHnDQ4rcNZGAEQbtZ79DB2we9hJmnvY\\\\\\\/G7FcmQO2AW1QuXo2YLvUqItqpul6ZPpfnSbBLAQD5U1m0FitouCoHQkLiNYxCLZZT8tJr8yp5WIH4uH+hqIp1EP4w0e+fIvQowC2kiGHkp+qjHsDMq1y2KzAwxKfUNGf17rJMA0Q4V+RM4M2R3KlEGPDSido1niQNoOFxAjvHrGTZsk1u9udhHbn\\\\\\\/Mg8tuKgw9Ige1wpYqg9fgOcr4TaAJ2cu6o2hicYmvpI4ymY++NNwqxtzIfnUu1lkbN\\\\\\\/ttVG5xitFJCx125\\\\\\\/ALMdWI\\\\\\\/0TtWnO+NQpF3vMWH6JuOOWiJh+eNCNwoBWnxFjlBN0Nd0DP\\\\\\\/obR3dmRSoE7HVCc1\\\\\\\/pFM7TpMeTrDNrCiHI3J8p0sEMY18CFRvigRXamfLDXbCEFM8nQmG76yHsuYiy7+uPd7UY23nVbH7xh70O\\\\\\\/T5utMHiSojQsiNnG0MUTSGLG4ya9FFjVrg9qDvEJylRaG4GSNqtPUYJ5ttJFW9VFiMycwUP9HU+qg\\\\\\\/lXD9Dez9BgXGqcHovxRJ6ZMAL5KFVVt+6vLmLj0XhAjvaMsclP0mpMRv2+yv7SVaLr\\\\\\\/FsUAdssUjonardMCMyuIlEh2HYjgZNUCv8t1gVSoy9e1J53SzKuXCfRZRy+nJQEmMPFojSKFmUSan\\\\\\\/BPQEyQU2ROk\\\\\\\/1aNDc2onqIPK4YvuC0wKSqRx2JrX2sAJ62AczFWm92Vlj\\\\\\\/A3r25\\\u0022,\\\u0022iv\\\u0022:\\\u0022\\\\\\\/UCdXHJlZF+SM8wpy8mOeQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00225b0c345d0631daf480e3d11f423c3c412c1fef027b768725dcf01957d2c98faf\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022VTrp3sRA01yOIie9kiZbnvzrDZeRsQj5gG7IuIyWtw6HvMaHG4Ak0n2ftdabUEzIezPxh1dN+XtN2gHkXM2x\\\\\\\/XFLQTtQyo2iHPlmMBv1y2gwCT4iwwbNfNzYl1nZARHMIeZXoVETW8w\\\\\\\/cGJbysT+xfazmZPBdzR1sc3QQqlBAJpKpPo6oO8Vzp2o5w1HIW5pnr9xSSjLJyQQ+S7epOOyhGmwbdOpQ6sPy97+NEhpiaFVRTsFvPcp6+EFfhZC+JlS1ByTno44MgTKU7kvrM3k9iJbmLpwcOp4YHCtI5zK\\\\\\\/LzVDC6JekqMNbLLE1ev2MhgOXST0W+xrrVMa488SEHnkIP60Q5D0V6RUoxCsAU4n01LooKDvsQ7ZRBeFnXvyHL62xp0BrBTRNwEmwKWppEAlgeQaSQvQqW08QKphcb\\\\\\\/p5VsItIHpjS0Qu9be93jfMbeDlv\\\\\\\/esy9kiV2GP+7x9jfa+sv9+uYZhbhpjUKtQobOjKu4oXYYLCoFNW4Af2BGOrymwQFD\\\\\\\/w5ZH9JgNW6eSSGpXeeQP3w2Qg07lRX372lrv4ePIgvvO6ngzCmyl\\\\\\\/JCuPCTF\\\\\\\/iQH7dcgC72drYXz1guBOI6JgAhFypr6txtZk6d8O+Oc4kf1Jm9T+nwhZ5ysR5vl6gz9aHeJo3DIbN9xCEzkEPjohk8wA50OQt231NYHHvi1JUhWPlfD3Evb+qIotUtF6nYinmOU503rsPYmzKQTtTF\\\\\\\/Bkc8LbW1LfI+H4re6cBy0KR8ca770XmlMl1etS9UcFhX3qXhFSF9Fd8qOqjRHvCGaG6NxPPYMbE2AbiefYqClh\\\\\\\/P6IwCruORYOtFzpptB8i95h2B84NRIGa961PWFCJoEBmqU1hpZijT2M9x9ODRoXukPCKqYJlFEMYF4z5FmUmev5kUxVCoWXcQlVF7w+heREOJzKjq2vIBB2eUApHjip\\\\\\\/78961jkWD6nuTX6Y3D9VDNRAIx4c9o\\\\\\\/lJpohX4aT5Hm7tbfSC86l0MrL7RLnmvaVYBZ8uuWxeHY\\\\\\\/xedTZJGfy+1hi4fjxhGcVkVW6fwHVI3Q\\\\\\\/WU6b0QxzK6NWGgT9soLx1tQ5HQA1FRyLwZo0tFhcaIo52N1DDOmhumGFVo8sYdRjQ88VC6inuWRnjZzDJhBzppVOF5ZDsXjET3bWyU\\\\\\\/WF5tPFd+3ZsmqYrd0nyNPVcVsuxMALSvZ9ICVheuh+XpCTxORtOTzyVT7VXkFQQVIotJggkCVSmESUcV57PvzD5A6ZDkuetJAyvqi0JIFGLvYv5MkuN47vlqSK55RsHh+Lkp79jLLhUB7YaDQmbinSexKK+GeM\\\\\\\/6okWHiI48rZXfZX3Og31fiLrPW9H8YjSpnS0Ipl0SOV3ZMM0NBHiSdm+AeH2ZMf3+KzRZ7cSQRdH6Qa+Cp7vfBL0c\\\\\\\/83UURHftX6O2vESTxrYaQhTtsd5zY7qLcTEVG17orJghaNGAGKRqBBK1PpcKi1XJaFm+pVm4NPQfbFpE4jM49onT1wDlJ6BUAzRoW91yA8lXBCcvJXZHJLPoqVPLO9pbRHqdUSuSrAjQ2o8HNNsLVM0qQJOlRUgRJgOGgnzP\\\\\\\/gq0WcOROomFmOPz2G11LVcUCWGZrAPOYInMV4veTeNJsOs4NWUE\\\\\\\/1UIltbvakGakPPuMpX8PXmd1g4DuUQBh5h8vdqx5ne4wVxWOrxFye\\\\\\\/oqLyuE5tsQnLxQoAFaVe5xuwtphVOOg12mcyEprwfB3yr0G9pErO03zK9gdvvYFEJnGIUCAbku4xzC3qiuSwZn\\\\\\\/GxKFHSHMpYOUdiEBcDU0HaoT8qMyb3Yfzmji3T3eILG\\\\\\\/HlARrXrEnV6PSoqvxYtFFZVSHKkEz69w6KZ4nVbnxmsvvS\\\\\\\/whr1AU8k7nnXmcwSk+wCLfTmYJv+qvOADK8JeFFiBcd+kXmKQYrGB1SleuC0awBfPbUc+m8NM3RoyTyFrgDuduB4oB2gRcKiMSMiszYkcwGK8ormHznLiR5rLLaaQk6dH0nAhEpSQqBvy6zszUzF61rAKAnncYPQAGpvkBRA=\\\u0022,\\\u0022iv\\\u0022:\\\u0022TnlMppcTiZuc432FVkdt5w==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00223f3f16d5d0a67f4941f06969ba42dc5159841d859674502b81579d64b57d9912\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-494609 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-genomic-prediction">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/11/3691" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-aN1GB5vqCrX5d8EMUTOl1AgmUNjF4swsFU3hA807OB4" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/11/3691" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-r3tot6ST4C6afi2HdlzEdOVwXSDg5qJaNlUVkqXZ3C8" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/11/3683" title="Elucidating the Molecular Determinants of Aβ Aggregation with Deep Mutational Scanning" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/11/3703" title="Activity-Dependent Regulation of the Proapoptotic BH3-Only Gene egl-1 in a Living Neuron Pair in Caenorhabditis elegans" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494609" id="node494609" data-pisa="ggg;9/11/3691" data-pisa-master="ggg;g3.119.400498" data-apath="/ggg/9/11/3691.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Benchmarking Parametric and Machine Learning Models for Genomic Prediction of Complex Traits</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-6097-606X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Christina B. Azodi</span>, <span class="highwire-citation-author" data-delta="1">Emily Bolger</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-7297-0984" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Andrew McCarren</span>, <span class="highwire-citation-author" data-delta="3">Mark Roantree</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0001-5692-7129" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Gustavo de los Campos</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0001-6470-235X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Shin-Han Shiu</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3691-3702;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400498 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494609" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Christina B. Azodi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Biology</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Christina%2BB.%2BAzodi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Azodi%20CB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AChristina%2BB.%2BAzodi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-6097-606X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Christina B. Azodi</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Emily Bolger </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Mathematics, Moravian College, Bethlehem, PA</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Emily%2BBolger%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bolger%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEmily%2BBolger%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Andrew McCarren </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Insight Centre for Data Analytics, School of Computing, Dublin City University, Dublin 9, Ireland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andrew%2BMcCarren%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=McCarren%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAndrew%2BMcCarren%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-7297-0984" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Andrew McCarren</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Mark Roantree </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Insight Centre for Data Analytics, School of Computing, Dublin City University, Dublin 9, Ireland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mark%2BRoantree%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Roantree%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMark%2BRoantree%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Gustavo de los Campos </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Epidemiology & Biostatistics</div><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Statistics & Probability</div><div class='author-affiliation'><span class='nlm-sup'>††</span>Institute for Quantitative Health Science and Engineering, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gustavo%2Bde%2Blos%2BCampos%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=de%20los%20Campos%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGustavo%2Bde%2Blos%2BCampos%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-5692-7129" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Gustavo de los Campos</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:gustavoc@msu.edu" class="" data-icon-position="" data-hide-link-title="0">gustavoc@msu.edu</a>
<a href="mailto:shius@msu.edu" class="" data-icon-position="" data-hide-link-title="0">shius@msu.edu</a></span></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Shin-Han Shiu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Biology</div><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>Department of Computational, Mathematics, Science, and Engineering, Michigan State University, East Lansing, MI, 48824</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Shin-Han%2BShiu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Shiu%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AShin-Han%2BShiu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-6470-235X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Shin-Han Shiu</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:gustavoc@msu.edu" class="" data-icon-position="" data-hide-link-title="0">gustavoc@msu.edu</a>
<a href="mailto:shius@msu.edu" class="" data-icon-position="" data-hide-link-title="0">shius@msu.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/9/11/3691" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:494609" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:494609/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/11/3691.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:494609" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:494609/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/11/3691.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:494609" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:494609/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/9/11/3691.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:494609" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:494609/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">The usefulness of genomic prediction in crop and livestock breeding programs has prompted efforts to develop new and improved genomic prediction algorithms, such as artificial neural networks and gradient tree boosting. However, the performance of these algorithms has not been compared in a systematic manner using a wide range of datasets and models. Using data of 18 traits across six plant species with different marker densities and training population sizes, we compared the performance of six linear and six non-linear algorithms. First, we found that hyperparameter selection was necessary for all non-linear algorithms and that feature selection prior to model training was critical for artificial neural networks when the markers greatly outnumbered the number of training lines. Across all species and trait combinations, no one algorithm performed best, however predictions based on a combination of results from multiple algorithms (<em>i.e.</em>, ensemble predictions) performed consistently well. While linear and non-linear algorithms performed best for a similar number of traits, the performance of non-linear algorithms vary more between traits. Although artificial neural networks did not perform best for any trait, we identified strategies (<em>i.e.</em>, feature selection, seeded starting weights) that boosted their performance to near the level of other algorithms. Our results highlight the importance of algorithm selection for the prediction of trait values.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/genomic-selection" class="hw-term hw-article-keyword hw-article-keyword-genomic-selection" rel="nofollow">Genomic selection</a></li><li class="kwd"><a href="/keyword/artificial-neural-network" class="hw-term hw-article-keyword hw-article-keyword-artificial-neural-network" rel="nofollow">artificial neural network</a></li><li class="kwd"><a href="/keyword/genotype-phenotype-0" class="hw-term hw-article-keyword hw-article-keyword-genotype-to-phenotype" rel="nofollow">genotype-to-phenotype</a></li><li class="kwd"><a href="/keyword/genomic-prediction" class="hw-term hw-article-keyword hw-article-keyword-genomic-prediction" rel="nofollow">Genomic Prediction</a></li><li class="kwd"><a href="/keyword/genpred" class="hw-term hw-article-keyword hw-article-keyword-genpred" rel="nofollow">GenPred</a></li><li class="kwd"><a href="/keyword/shared-data-resources" class="hw-term hw-article-keyword hw-article-keyword-shared-data-resources" rel="nofollow">Shared Data Resources</a></li></ul><p id="p-3">The ability to predict complex traits from genotypes is a grand challenge in biology and is accelerating the speed of crop and livestock breeding (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Heffner <em>et al.</em> 2009</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Lorenz <em>et al.</em> 2011</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Jonas and de Koning 2013</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Desta and Ortiz 2014</a>). Genomic Prediction (GP, aka Genomic Selection), the use of genome-wide genetic markers to predict complex traits, was originally proposed by Meuwissen <em>et al.</em> (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Meuwissen <em>et al.</em> 2001</a>) as a solution to the limitations of Marker-Assisted Selection (MAS) where only a limited number of previously identified markers with the strongest associations are used to select the best lines. GP is particularly well-suited for the prediction of quantitative traits controlled by many small-effect alleles (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Ribaut and Ragot 2007</a>). A major challenge in using GP is estimating the effects of a large number of makers (p) using phenotype information of a comparatively limited number of individuals (n) (<em>i.e.</em>, <em>P</em> &gt; &gt; n) (<a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">Meuwissen <em>et al.</em> 2001</a>). To address this challenge, Meuwissen <em>et al.</em> first presented three statistical methods for GP (<a id="xref-ref-47-3" class="xref-bibr" href="#ref-47">Meuwissen <em>et al.</em> 2001</a>). The first was a linear mixed model called ridge regression Best Linear Unbiased Prediction (rrBLUP), which uniformly shrinks the marker effects. The other two were Bayesian approaches, BayesA (BA) and BayesB (BB), which both differentially shrink the marker effects and with BB also performing variable selection. Since then, additional approaches have been shown to be useful for GP, including Least Absolute Angle and Selection Operator (LASSO) (<a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Usai <em>et al.</em> 2009</a>), Elastic Net (<a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Zou and Hastie 2005</a>), Support Vector Regression with a linear kernel (SVR<sub>lin</sub>) (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Moser <em>et al.</em> 2009</a>; <a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Xu <em>et al.</em> 2018</a>), and additional Bayesian methods including Bayesian LASSO (BL), BayesCπ, and BayesDπ (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">de los Campos <em>et al.</em> 2009</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Habier <em>et al.</em> 2011</a>).</p><p id="p-4">While these approaches perform well when dealing with high dimensional data (<em>i.e.</em>, <em>P</em> &gt; &gt;n), they are all based on a linear mapping from genotype to phenotypes, and therefore may not fully capture non-linear effects (<em>e.g.</em>, epistasis, dominance), which are likely to be important for complex traits (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Holland 2007</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Monir and Zhu 2018</a>). To overcome this limitation, non-linear approaches, including reproducing kernel Hilbert spaces (RKHS) regression (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Gianola <em>et al.</em> 2006</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">de los Campos <em>et al.</em> 2010</a>), Support Vector Regression with non-linear kernels (<em>i.e.</em>, polynomial SVR<sub>poly</sub> and radial basis function SVR<sub>rbf</sub> (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Long <em>et al.</em> 2011</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Kasnavi <em>et al.</em> 2017</a>)), and decision tree based algorithms such as Random Forest (RF) (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">González-Recio and Forni 2011</a>; <a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Spindel <em>et al.</em> 2015</a>) and Gradient Tree Boosting (GTB) (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">González-Recio <em>et al.</em> 2013</a>) have been applied to GP problems. In previous efforts to compare the performance of multiple linear and non-linear approaches (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Heslot <em>et al.</em> 2012</a>; <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Neves <em>et al.</em> 2012</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Blondel <em>et al.</em> 2015</a>; <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Ramstein <em>et al.</em> 2016</a>; <a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Roorkiwal <em>et al.</em> 2016</a>), no single method performs best in all cases. Rather, factors such as the size of the training data set, marker type and number, trait heritability, effective population size, the number of causal loci, as well as genetic architecture (the locus effect size distribution) can all affect algorithm performance (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Meuwissen 2009</a>; <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Riedelsheimer <em>et al.</em> 2013</a>; <a id="xref-ref-64-2" class="xref-bibr" href="#ref-64">Spindel <em>et al.</em> 2015</a>; <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Norman <em>et al.</em> 2018</a>). This highlights the importance of comparing new algorithms across a diverse range of datasets.</p><p id="p-5">With improvements in computing speeds, the development of graphics processing units (GPUs), and breakthroughs in algorithms for backpropagation learning (<a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Rumelhart <em>et al.</em> 1986</a>; <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Parker 1987</a>), there has been a resurgence of research using deep learning (<em>i.e.</em>, artificial neural networks (ANNs)) to model complex biological processes (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Angermueller <em>et al.</em> 2016</a>; <a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Webb 2018</a>). ANNs are a class of machine learning methods that perform layers of transformations on features to create abstraction features, known as hidden layers, which are used for predictions. The first application of ANNs for GP was presented in 2011, when Okut <em>et al.</em> trained fully connected ANNs (<em>i.e.</em>, each node in a layer is connected to all nodes in surrounding layers) containing one hidden layer to predict body mass index in mice (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Okut <em>et al.</em> 2011</a>). Since 2011, more complex ANN architectures have been used for GP including radial basis function neural networks (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">González-Camacho <em>et al.</em> 2012</a>) deep neural networks (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Ehret <em>et al.</em> 2015</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Bellot <em>et al.</em> 2018</a>), deep recurrent neural networks (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Pouladi <em>et al.</em> 2015</a>), probabilistic neural network classifiers (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">González-Camacho <em>et al.</em> 2016</a>, <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">2018</a>), and convolutional neural networks (CNNs) (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Ma <em>et al.</em> 2018</a>). With only one exception (<a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Bellot <em>et al.</em> 2018</a>), these ANNs have been applied to datasets with relatively few genetic markers (&lt;60k), however, as sequencing continues to become less expensive, whole-genome marker datasets are becoming larger with some breeding programs generating data for hundreds of thousands of markers. Because of the internal complexity of ANN models, training an ANN with so many markers can result in sub-optimal solutions (<em>i.e.</em>, underfitting). Therefore, it is especially important to benchmark ANNs against other GP statistical approaches on datasets with high dimensionality where underfitting may occur.</p><p id="p-6">GP has yielded promising results for breeders. However, a comprehensive comparison of GP algorithms, particularly ANNs, on a wide range of GP problems is missing (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>). Here we compared the ability of 12 GP algorithms (see <strong>Methods</strong>, <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1B</a>) to predict a diverse range of physiological traits in six plant species (maize, rice, sorghum, soy, spruce, and switchgrass; <a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1C</a>). These six data sets (referred to as the benchmark data sets) represent a wide range of GP data types, with the size of the training data set ranging from 327 to 5,014 individuals, and 4,000 to 332,000 markers derived from array-based approaches or sequencing. Compared to the linear algorithms included in the study, the non-linear algorithms, especially ANNs, require more pre-modeling tuning (<em>e.g.</em>, hyperparameter selection, feature selection). Therefore, before comparing algorithm performance across all 18 combinations of species and traits, we first focused on predicting plant height in each species in order to establish best practices for model building. Because ANNs are underrepresented in GP comparison studies and our first attempts to use ANNs for GP performed relatively poorly, we focus on methods to improve ANN performance, including reducing model complexity using feature selection and combining relationships learned from linear algorithms into the more complex ANN architectures (<em>i.e.</em>, a seeded ANN approach and convolutional layers (<em>i.e.</em>, CNNs)). Then, using lessons learned from predicting height, we compared the performance of all GP algorithms across all species and traits.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/11/3691/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Algorithms used and compared in past GP studies and algorithms and data included in the GP benchmark. (A) Number of times a GP algorithm was utilized (diagonal) or directly compared to other GP algorithms (lower triangle) out of 91 publications published between 2012-2018 (Table S1). GP algorithms were included if they were utilized in &gt;1 study. (B) A graphical representation of the GP algorithms included in the study and their relationship to each other. Colors designate if the algorithm identifies only linear (orange) or linear and non-linear (green) relationships. The placement of each algorithm on the tree designates (qualitatively) the relationship between different algorithms. The labels at each branch provide more information about how algorithms in that branch differ from others. rrBLUP, ridge regression Best Linear Unbiased Predictor; BRR, Bayesian Ridge Regression; BA, BayesA; BB, BayesB; BL, Bayesian LASSO; SVR, Support Vector Regression (kernel type: lin, linear; poly, polynomial; rbf, radial basis function); RF, Random Forest; GTB, Gradient Tree Boosting; ANN, Artificial Neural Network; CNN, Convolutional Neural Network. (C) Species and traits included in the benchmark with training population types and sizes and marker types and numbers for each dataset. NAM: Nested Association Mapping. DM: partial diallel mating. GBS: genotyping by sequencing. SNP: single nucleotide polymorphism. HT: height. FT: flowering time. YLD: yield. GM: grain moisture. R8: time to R8 developmental stage. DBH: diameter at breast height. DE: wood density. ST: standability." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1024819697" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Algorithms used and compared in past GP studies and algorithms and data included in the GP benchmark. (A) Number of times a GP algorithm was utilized (diagonal) or directly compared to other GP algorithms (lower triangle) out of 91 publications published between 2012-2018 (Table S1). GP algorithms were included if they were utilized in &gt;1 study. (B) A graphical representation of the GP algorithms included in the study and their relationship to each other. Colors designate if the algorithm identifies only linear (orange) or linear and non-linear (green) relationships. The placement of each algorithm on the tree designates (qualitatively) the relationship between different algorithms. The labels at each branch provide more information about how algorithms in that branch differ from others. rrBLUP, ridge regression Best Linear Unbiased Predictor; BRR, Bayesian Ridge Regression; BA, BayesA; BB, BayesB; BL, Bayesian LASSO; SVR, Support Vector Regression (kernel type: lin, linear; poly, polynomial; rbf, radial basis function); RF, Random Forest; GTB, Gradient Tree Boosting; ANN, Artificial Neural Network; CNN, Convolutional Neural Network. (C) Species and traits included in the benchmark with training population types and sizes and marker types and numbers for each dataset. NAM: Nested Association Mapping. DM: partial diallel mating. GBS: genotyping by sequencing. SNP: single nucleotide polymorphism. HT: height. FT: flowering time. YLD: yield. GM: grain moisture. R8: time to R8 developmental stage. DBH: diameter at breast height. DE: wood density. ST: standability.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/11/3691/F1.medium.gif" width="260" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/9/11/3691/F1.medium.gif" width="260" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/11/3691/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/11/3691/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494792" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-7" class="first-child">Algorithms used and compared in past GP studies and algorithms and data included in the GP benchmark. (A) Number of times a GP algorithm was utilized (diagonal) or directly compared to other GP algorithms (lower triangle) out of 91 publications published between 2012-2018 (Table S1). GP algorithms were included if they were utilized in &gt;1 study. (B) A graphical representation of the GP algorithms included in the study and their relationship to each other. Colors designate if the algorithm identifies only linear (orange) or linear and non-linear (green) relationships. The placement of each algorithm on the tree designates (qualitatively) the relationship between different algorithms. The labels at each branch provide more information about how algorithms in that branch differ from others. rrBLUP, ridge regression Best Linear Unbiased Predictor; BRR, Bayesian Ridge Regression; BA, BayesA; BB, BayesB; BL, Bayesian LASSO; SVR, Support Vector Regression (kernel type: lin, linear; poly, polynomial; rbf, radial basis function); RF, Random Forest; GTB, Gradient Tree Boosting; ANN, Artificial Neural Network; CNN, Convolutional Neural Network. (C) Species and traits included in the benchmark with training population types and sizes and marker types and numbers for each dataset. NAM: Nested Association Mapping. DM: partial diallel mating. GBS: genotyping by sequencing. SNP: single nucleotide polymorphism. HT: height. FT: flowering time. YLD: yield. GM: grain moisture. R8: time to R8 developmental stage. DBH: diameter at breast height. DE: wood density. ST: standability.</p><div class="sb-div caption-clear"></div></div></div><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Genotype and phenotype data</h3><p id="p-8">Genotypic data from six plant species were used to predict 3 traits from each species (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1C</a>). The maize phenotypic (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Hansey <em>et al.</em> 2011</a>) and genotypic (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Hirsch <em>et al.</em> 2014</a>) data were from the pan-genome population, maize trait values were averaged over replicate plots. The rice data were from elite breeding lines from the International Rice Research Institute irrigated rice breeding program (<a id="xref-ref-64-3" class="xref-bibr" href="#ref-64">Spindel <em>et al.</em> 2015</a>), and dry season trait data averaged over four years were used. The sorghum data were generated from sorghum lines from the US National Plant Germplasm System grown in Urbana, IL (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Fernandes <em>et al.</em> 2017</a>) and trait values were averaged over two blocks for this study. The soybean data were generated from the SoyNAM population containing recombinant inbred lines (RILs) derived from 40 biparental populations (<a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Xavier <em>et al.</em> 2016</a>). The white spruce data were obtained from the SmartForests project team, using a SNP-chip developed by Quebec Ministry of Forest Wildlife and Parks (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Beaulieu <em>et al.</em> 2014</a>)<sub>.</sub> Switchgrass phenotypic (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Lipka <em>et al.</em> 2014</a>) and genotypic (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Evans <em>et al.</em> 2018</a>) data were generated from the Northern Switchgrass Association Panel (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Evans <em>et al.</em> 2015</a>) which contains clones or genotypes from 66 diverse upland switchgrass populations.</p><p id="p-9">The genotype data were obtained in the form of biallelic SNPs with missing marker data already dropped or imputed by the original authors. Marker calls were converted when necessary to [-1,0,1] corresponding to [aa, Aa, AA] where A was either the reference or the most common allele. Genome locations of maize SNPs were converted from assembly AGPv2 to AGPv4, with AGPv2 SNPs that did not map to AGPv4 being removed, leaving 332,178 markers for the maize analysis. Phenotype values were normalized between 0 and 1. Lines with missing phenotypic value for any of the three traits were removed.</p></div><div id="sec-3" class="subsection"><h3>Genomic selection algorithms</h3><p id="p-10">To assess what statistical approaches are most frequently used for genomic selection, we conducted a literature search of papers applying genomic selection methods to crop or simulated data from January 2012-February 2018. We recorded what statistical approach(es) was(were) applied in each study (Table S1), allowing us to calculate both the total number of times an approach had been applied and how many times any two approaches were directly compared (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1A</a>). Based on the results from this literature search, nine commonly used statistical approaches were included in this study: rrBLUP, Bayes A (BA), Bayes B (BB), Bayesian LASSO, Bayesian-RR, RF, SVR with a linear kernel (SVR<sub>lin</sub>), SVR with polynomial kernel (SVR<sub>poly</sub>), SVR with radial basis function kernel (SVR<sub>rbf</sub>). Three additional machine learning approaches, gradient tree boosting (GTB), artificial neural networks (ANN), and convolutional neural networks (CNN), were also included because of their ability to model non-linear relationships.</p><p id="p-11">Most linear algorithms were implemented in R packages rrBLUP (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Endelman 2011</a>) and BGLR (for Bayesian methods including BRR: Bayesian RR, BA: Bayes A, BB: Bayes B, and BL: Bayesian LASSO) (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Pérez and de los Campos 2014</a>). These algorithms vary in what approach they use to address the <em>P</em> &gt; &gt; n problem (<a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1B</a>), for example rrBLUP performs uniform shrinkage on all marker coefficients to reduce variance of the estimator, while BB performs differential shrinkage of the marker coefficients and variable selection. The differences between these algorithms have been thoroughly reviewed previously (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">de los Campos <em>et al.</em> 2013</a>). Models for Bayesian methods were trained for 12,000 iterations using a burn-in of 2,000.</p><p id="p-12">Non-linear algorithms (SVR<sub>poly</sub>, SVR<sub>rbf</sub>, RF, and GTB) and SVR<sub>lin</sub> were implemented in python using the Scikit-Learn library (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Pedregosa <em>et al.</em> 2011</a>). For SVR algorithms, the marker data are mapped into a new feature space using linear or non-linear kernels (<em>i.e.</em>, poly, rbf) and then linear regression within that feature space is performed with the goal of minimizing error outside of a margin of tolerated error. The RF algorithm works by averaging the predictions from a “forest” of bootstrapped regression trees, where each tree contains a random subset of the lines and of the markers (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Breiman 2001</a>). Related to RF, GTB algorithm uses the principle of boosting (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Friedman 2001</a>) to improve predictions from weak learners (<em>i.e.</em>, regression trees) by iteratively updating the learners to minimize a loss function, therefore generating better weak learners as training progresses.</p><p id="p-13">Artificial Neural Networks (ANNs) were implemented in python using TensorFlow (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Girija 2016</a>). The input layer for the ANNs contained the genetic markers for an individual (<em>x</em>; <a id="xref-fig-1-7" class="xref-fig" href="#F1">Figure 1B</a>), the nodes in the hidden layers were all fully connected to all nodes in the previous and following layers (<em>i.e.</em>, Multilayer Perceptron). A non-linear activation function (selected during the grid search, see below) was applied to each node in the input and hidden layers, except the last hidden layer, which was connected with a linear function to the output layer, the predicted trait value (<em>y</em>). To reduce the likelihood of vanishing gradients, when the error gradient, which controls the degree to which the weights are updated during each iteration of training, becomes so small the weights stop updating thus halting model training, in the ANN, the starting weights (<em>w</em>) were scaled relative to the number of input markers using the Xavier Initializer (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Glorot and Bengio 2010</a>). Weights were then optimized using the Adam Optimizer (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Kingma and Ba 2014</a>) with a learning rate selected by the grid search (described below). To determine the optimal stopping time for training (<em>i.e.</em>, number of epochs), an early stopping approach was used (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Prechelt 1998</a>), where the training set was further divided into training and validation, and early stopping occurred when the change in mean squared error (MSE) for the validation set was &lt; 0.1% for 10 epochs using a 10 epoch burn-in. Occasionally, due to poor random initialization of weights, the early stopping criteria would be reached before the network started to converge and the resulting network would predict the same trait value for every line. When this was observed in the validation set the training process was repeated starting with new initialized weights.</p><p id="p-14">Convolutional Neural Networks (CNNs) were implemented in Python 3.6 using Tensorflow 2.0. The input layer for the CNNs consisted of the genetic markers for an individual one-hot-encoded so that each possible allele at each locus was represented as present or absent. Because of the large size of the possible hyperparameter space (Table S2), a randomized search (using RandomizedSearchCV from Scikit-Learn with 5 folds) was performed on rice for predicting height on one replicate, and the best combination of hyperparameters (lowest average mean squared error) from this one search was used for all other species, traits, and replicates. The input data first passed through a convolutional layer, followed by a maximum pooling layer, a dropout layer, a dense (<em>i.e.</em>, fully connected) layer, a batch normalization layer, and finally to the output layer containing one node with the predicted trait value. The EarlyStopping function in Keras (<a href="https://keras.io/callbacks/#earlystopping">https://keras.io/callbacks/#earlystopping</a>) was used to avoid overfitting (min_delta = 0, patience = 10). To reduce the time and memory requirements, CNN models were trained using a batch size = 100 and run for a maximum of 1,000 epochs. As with ANN models, if the early stopping criteria was reached before the network started to converge, the model would be re-run starting with new initialized weights.</p><p id="p-15">To incorporate predictions from multiple algorithms into one summary prediction, an ensemble approach was used where the ensemble predicted trait value was the mean predicted trait value from 11 algorithms (EN<sub>11</sub>: rrBLUP, BRR, BA, BB, BL, SVR, SVRpoly, SVRrbf, RF, GTB, ANN) or five algorithms (EN<sub>5</sub>: rrBLUP, BL, SVRpoly, RF, ANN). The subset of five consisted of algorithms with differing statistical bases, where rrBLUP represented penalized methods, BL represented the Bayesian approaches, SVRpoly represented non-linear regularized functions, RF represented decision tree based methods, and ANN represented the deep learning approach. This ensemble predicted trait value was then compared to the true trait values to generate performance metrics. A Repeated Measures Analysis of variance (ANOVA) implemented in R was used to compare model performance, where performance of each model on each replicate test set were considered related.</p></div><div id="sec-4" class="subsection"><h3>Hyperparameter grid search using cross-validation</h3><p id="p-16">To obtain the best possible results from each algorithm, a grid search approach was used to determine the combination of hyperparameters that maximized performance for each trait/species combination. No hyperparameter needed to be defined for rrBLUP, BL, or BRR. For rrBLUP, the R package estimates the regularization and kernel parameters from the data. For BL or BRR, parameters for these Bayesian regression methods were also estimated from the data. Between one and five hyperparameters were tested for the remaining algorithms (Table S2).</p><p id="p-17">To avoid biasing our hyperparameter selection, an 80/20 training/testing approach was used, where 20% of the lines were held out from each model as a testing set and the grid search was performed on the remaining 80% of training lines. For RF, SVR<sub>lin</sub>, SVR<sub>poly</sub>, SVR<sub>rbf</sub>, and GTB algorithms, 10 replicates of the grid search were run using the GridSearchCV function from Scikit-Learn with fivefold cross validation. Ten replicates of the grid search were also run for ANN models, where for each replicate 80% of the training data were randomly selected for training the network with each combination of hyperparameters and the remaining 20% used to select the best combination. This whole process (train/test split, grid search) was replicated 10 times, with a different 20% of lines selected as the test set for each replicate. ANOVA implemented in R was used to determine which hyperparameters significantly impacted model performance for each species.</p></div><div id="sec-5" class="subsection"><h3>Assessing predictive performance</h3><p id="p-18">The predictive performance of the models was compared using two metrics. For the grid search analysis, the mean squared error (MSE) between the predicted (Ŷ) and the true (Y) trait value was used. For the model comparisons, Pearson correlation coefficient (<em>r</em>) between the predicted (Ŷ) and the true trait value (Y) was used as it is the standard metric for GP performance (<a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Heffner <em>et al.</em> 2009</a>; <a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Heslot <em>et al.</em> 2012</a>; <a id="xref-ref-60-2" class="xref-bibr" href="#ref-60">Riedelsheimer <em>et al.</em> 2013</a>). It was computed using the cor() function in R for rrBLUP and the Bayesian approaches or the numpy corrcoef() function in Python for the ML and ANN approaches. Only predicted trait values for lines from the test set were considered when calculating <em>r</em>. Summary performance metrics (% of best <em>r</em>, rank, variance) were calculated using the mean predictive performance (<em>r</em>) across all replicates for each GP algorithm for each species/trait combination.</p></div><div id="sec-6" class="subsection"><h3>Feature selection</h3><p id="p-19">The top 10, 50, 100, 250, 500, 1000, 2000, 4000, and 8000 markers were selected using three different feature selection algorithms: Random Forest (RF), Elastic Net (EN), and BayesA (BA). RF and EN feature selection were implemented in Scikit-Learn and BA was implemented in the BGLR package in R. The EN feature selection algorithm requires tuning of the hyperparameter that controls the ratio of the L1- and L2- penalties (<em>e.g.</em>, L1:L2 = 1:10 = 0.1). Because the L1 penalty function performs variable selection by shrinking some coefficients to zero, we started with an initial weight on the L1 penalty of 0.1 and then, if fewer than 8,000 markers remained after variable selection, we reduced it in steps of 0.02 until that criteria was met (a 4,000 marker threshold was used for spruce and soy, which only had 6,932 and 4,240 markers available, respectively).</p><p id="p-20">To avoid bias during feature selection, the 80:20 training/testing approach described above was used, where feature selection was performed on the training data and the ultimate performance of models built using the selected markers was scored on the testing set. This was repeated for all 10 testing sets. A repeat measures ANOVA was conducted to compare feature selection algorithms, the number of features selected, and GP algorithms (<em>i.e.</em>, independent variables) on model performance (<em>i.e.</em>, dependent variable) where replicates were considered repeat measures as they used the same testing set. One-sided, paired Wilcoxon Signed-Rank tests were conducted to determine if model performance (<em>i.e.</em>, dependent variable) increased after feature selection (all <em>vs.</em> top 4,000 for soy and spruce, all <em>vs.</em> top 8,000 for other species) (<em>i.e.</em>, independent variable). Resulting <em>p</em>-values were corrected for multiple testing (<em>q</em>-value) (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Benjamini and Hochberg 1995</a>).</p></div><div id="sec-7" class="subsection"><h3>Initializing ANN starting weights seeded from other GP algorithms</h3><p id="p-21">In addition to building ANNs with randomly initialized starting weights, we tested the usefulness of seeding the starting weights with information from other GP algorithms (<em>i.e.</em>, rrBLUP, BB, BL, or RF). This is an ensemble-like approach in that it utilizes multiple algorithms to make a final prediction. Ensemble approaches often perform better than single algorithm approaches (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Dietterich 2000</a>). First, after the data were divided into training, validation, and testing sets and, for species with large p:n ratios (<em>i.e.</em>, maize, rice, sorghum, switchgrass) the top 8,000 markers were selected, we applied a GP algorithm (rrBLUP, BB, BL, or RF) to the training data. From that model we extracted the coefficients/importance scores assigned to each marker and used those as the starting weights for 25% of the nodes in the first hidden layer. We also tested seeding starting weights for 50% of the nodes to predict height in all 6 species but found this significantly increased the model error (MSE) on the validation set (ANOVA; <em>p</em>-value= 0.04), so only results from seeding 25% were included. Because we still needed to reduce the likelihood of vanishing gradients, described above, we manually adjusted the scale of the coefficients/importance scores to match the distribution of the starting weights assigned the remaining 75% of the nodes in the first hidden layer by Xavier Initialization. Finally, to reduce bias in the ANN, random noise was introduced to the seeded nodes by multiplying each starting weight with a random number from a normal distribution with a mean =0 and the standard deviation equal to the standard deviation of weights from Xavier Initialization.</p><p id="p-22">After the training data were used to determine these seeded starting weights, it was used to train the ANN model, the validation set was used to select the best set of hyperparameters and the early stopping point. Then the final trained model was applied to the testing set and performance metrics were calculated. A repeat measures ANOVA was conducted to test if the seeded or the unseeded ANN models (<em>i.e.</em>, independent variable) differed in the amount of variation (standard deviation) in model performance across replicates (<em>i.e.</em>, dependent variable), with each species acting as a repeat measurement.</p></div><div id="sec-8" class="subsection"><h3>Data availability</h3><p id="p-23">For reproducibility, all six datasets along with training/testing designations are available on Dryad (<a href="https://doi.org/10.5061/dryad.xksn02vb9">https://doi.org/10.5061/dryad.xksn02vb9</a>) and scripts to run all of the algorithms included in this study on GitHub for future benchmarking. All code used in this study is available on GitHub (<a href="https://github.com/ShiuLab/Manuscript_Code/tree/master/2019_GP_Comparison">https://github.com/ShiuLab/Manuscript_Code/tree/master/2019_GP_Comparison</a>). A README file is included, which provides detailed instructions on how to use the code to generate GP models. Supplemental material available at FigShare: <a href="https://doi.org/10.25387/g3.9855590">https://doi.org/10.25387/g3.9855590</a>.</p></div></div><div class="section results" id="sec-9"><h2 class="">Results</h2><div id="sec-10" class="subsection"><h3>Hyperparameter grid search is critical, particularly among non-linear algorithms</h3><p id="p-24">We selected six linear and five non-linear algorithms (note, CNNs are discussed separately) to compare their performance in GP problems (see <strong>Methods</strong>). While some model parameters can be estimated from the data (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">de los Campos <em>et al.</em> 2013</a>), other parameters, referred to as hyperparameters, have to be user-defined (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Chapelle <em>et al.</em> 2002</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Kuhn and Johnson 2013</a>). This was the case for eight of the algorithms in our study: BA, BB, SVR<sub>lin</sub>, SVR<sub>poly</sub>, SVR<sub>rbf</sub>, RF, GTB, and ANN. For these algorithms we conducted a grid search to evaluate the prediction accuracy of models using every possible combination of hyperparameter values (for lists of hyperparameters, see Table S2). To produce unbiased estimates of prediction accuracy the grid search was performed within the training set so that no data from the testing set was used to select hyperparameter values. Then we used the best set of hyperparameters from the grid search to build models using genotype and phenotype data from six plant species. This allowed us to compare the predictive performance of all algorithms included in the benchmark datasets.</p><p id="p-25">To determine which hyperparameters significantly impacted model performance, we tested for changes in model performance (mean squared error; MSE) across the hyperparameter space for each algorithm/species/trait combination using Analysis of Variance (ANOVA). The degrees of freedom hyperparameter for BA and BB, both linear algorithms, that influences the shape of the prior density of marker effects (<a id="xref-ref-9-3" class="xref-bibr" href="#ref-9">de los Campos <em>et al.</em> 2013</a>) had no significant impact on model performance (ANOVA: <em>p</em>-value<em>=</em> 0.41∼1.0; Table S3). Other parameters for the Bayesian algorithms were determined using rules built into the BGLR package that account for factors such as phenotypic variance and the number of markers (p) (<a id="xref-ref-55-2" class="xref-bibr" href="#ref-55">Pérez and de los Campos 2014</a>) and were therefore not considered in our grid search. However, 15 of 16 of the hyperparameters tested for the non-linear algorithms significantly impacted performance in at least one species (Table S3, Figure S1A-C). Using height in maize as an example, we found that SVR<sub>poly</sub> algorithm performed better (<em>i.e.</em>, lower MSE) using 2<sup>nd</sup> degree polynomials compared to using up to 3<sup>rd</sup> degree polynomials (<em>p</em>-value = 1*10<sup>−21</sup>, <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>). For RF-based models, the maximum depth (max depth) of decision trees allowed significantly impacted performance (<em>p</em>-value = 1*10<sup>−3</sup>, Table S3), with shallower trees typically performing better (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2B</a>). This pattern was also observed in RF models predicting height for rice, spruce, and soy (<em>p</em>-value= 1*10<sup>−66</sup>∼5*10<sup>−4</sup>, Table S3, Figure S1B). Because shallower decision trees are less complex, they tend not to overfit, suggesting the best hyperparameters for RF are those that reduce overfitting. The only hyperparameter from the non-linear algorithms that did not impact performance was the rate of dropout (a useful regularization technique to avoid overfitting) for ANN models, where there was no significant change in model performance when two different rates (10% and 50%) were used (<em>p</em>-value= 0.24 ∼0.97, Table S3).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/11/3691/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Grid search results for height in maize and overall GP algorithm performance for predicting height across species. (A) Average of mean squared error (MSE) over hyperparameter space (penalty, C) for Support Vector Regression (SVR) based models predicting height in maize. SVRrbf and SVRpoly results are shown using gamma = 1x10&#x2212;5 and 1x10&#x2212;4, respectively. Poly: polynomial. RBF: Radial Basis Function. (B) Distribution of MSEs across hyperparameter space for Random Forest (RF; left) and Gradient Tree Boosting (GTB; right) as the maximum features available to each tree (Max Features) and maximum tree depth (color) change. GTB results are shown using a learning rate = 0.01. (C) Average MSE across hyperparameter space for ANN models with different network architectures, degrees of regularization (dropout or L2), using either the Rectified Linear Unit (ReLU; left) or Sigmoid (right) activation function. (D) Mean performance (Pearson&#x2019;s Correlation Coefficient: r, text) for predicting height and percent best r (colored box, top algorithm for each species = 100% (red)). White text: the best r values. Violin-plots show the median and distribution of r values for each trait (right) and algorithm (bottom)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1024819697" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Grid search results for height in maize and overall GP algorithm performance for predicting height across species. (A) Average of mean squared error (MSE) over hyperparameter space (penalty, &lt;em&gt;C&lt;/em&gt;) for Support Vector Regression (SVR) based models predicting height in maize. SVR&lt;sub&gt;rbf&lt;/sub&gt; and SVR&lt;sub&gt;poly&lt;/sub&gt; results are shown using gamma = 1x10&lt;sup&gt;&#x2212;5&lt;/sup&gt; and 1x10&lt;sup&gt;&#x2212;4&lt;/sup&gt;, respectively. Poly: polynomial. RBF: Radial Basis Function. (B) Distribution of MSEs across hyperparameter space for Random Forest (RF; left) and Gradient Tree Boosting (GTB; right) as the maximum features available to each tree (Max Features) and maximum tree depth (color) change. GTB results are shown using a learning rate = 0.01. (C) Average MSE across hyperparameter space for ANN models with different network architectures, degrees of regularization (dropout or L2), using either the Rectified Linear Unit (ReLU; left) or Sigmoid (right) activation function. (D) Mean performance (Pearson&#x2019;s Correlation Coefficient: &lt;em&gt;r&lt;/em&gt;, text) for predicting height and percent best &lt;em&gt;r&lt;/em&gt; (colored box, top algorithm for each species = 100% (red)). White text: the best &lt;em&gt;r&lt;/em&gt; values. Violin-plots show the median and distribution of &lt;em&gt;r&lt;/em&gt; values for each trait (right) and algorithm (bottom).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/11/3691/F2.medium.gif" width="440" height="184"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/9/11/3691/F2.medium.gif" width="440" height="184"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/11/3691/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/11/3691/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494688" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-26" class="first-child">Grid search results for height in maize and overall GP algorithm performance for predicting height across species. (A) Average of mean squared error (MSE) over hyperparameter space (penalty, <em>C</em>) for Support Vector Regression (SVR) based models predicting height in maize. SVR<sub>rbf</sub> and SVR<sub>poly</sub> results are shown using gamma = 1x10<sup>−5</sup> and 1x10<sup>−4</sup>, respectively. Poly: polynomial. RBF: Radial Basis Function. (B) Distribution of MSEs across hyperparameter space for Random Forest (RF; left) and Gradient Tree Boosting (GTB; right) as the maximum features available to each tree (Max Features) and maximum tree depth (color) change. GTB results are shown using a learning rate = 0.01. (C) Average MSE across hyperparameter space for ANN models with different network architectures, degrees of regularization (dropout or L2), using either the Rectified Linear Unit (ReLU; left) or Sigmoid (right) activation function. (D) Mean performance (Pearson’s Correlation Coefficient: <em>r</em>, text) for predicting height and percent best <em>r</em> (colored box, top algorithm for each species = 100% (red)). White text: the best <em>r</em> values. Violin-plots show the median and distribution of <em>r</em> values for each trait (right) and algorithm (bottom).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-11" class="subsection"><h3>ANN is the most significantly impacted by hyperparameter choice</h3><p id="p-27">Hyperparameters for SVR<sub>lin</sub>, SVR<sub>poly</sub>, SVR<sub>rbf</sub>, RF, and GTB tended to have moderate effects on MSE, while ANN hyperparameters often caused substantial changes in MSE (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2A-C</a>; Figure S1A-C). Across the six species, the median variance in MSE across the hyperparameter space for ANN was 6*10<sup>6</sup>, but ranged from 3*10<sup>−3</sup>- 0.1 for the other GP algorithms (Figure S1D) For example, for predicting height in maize, SVR<sub>poly</sub> models built using the 2<sup>nd</sup> degree polynomial outperformed those built using the 3<sup>rd</sup> degree polynomial with a decrease in MSE ∼0.05 (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2A</a>), while for ANN models, hyperparameter combinations that performed the best (<em>i.e.</em>, Sigmoid activation function and no L2 regularization) resulted in models with MSEs that were &gt;500 lower than the worst performing model (Rectified Linear Unit (ReLU) activation function, no L2 regularization, and large numbers of hidden nodes; <a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2C</a>). This highlighted that, while hyperparameter selection is necessary for all non-linear algorithms, it is especially critical for building ANNs for GP problems.</p><p id="p-28">Using the best set of hyperparameters for each model, we next compared the predictive performance (Pearson’s correlation coefficient, <em>r</em>, between predicted and true trait values) of each algorithm on plant height. As with past efforts to benchmark GP algorithms (<a id="xref-ref-32-3" class="xref-bibr" href="#ref-32">Heslot <em>et al.</em> 2012</a>; <a id="xref-ref-50-2" class="xref-bibr" href="#ref-50">Neves <em>et al.</em> 2012</a>), no one algorithm always performed the best (white bolded; <a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2D</a>). For example, while rrBLUP performed best for maize, sorghum, and switchgrass, BA performed best for soy, and RF performed best for rice and spruce. Notably, ANNs substantially underperformed compared to other non-linear algorithms, with a median performance at 84% of the best <em>r</em> for each of the six species (<em>i.e.</em>, 16% below the best performing algorithm for that trait/species).</p><p id="p-29">Notably, among the six species, ANN performed the best in soy (<em>r =</em> 0.44) relative to the species best algorithm BA (<em>r =</em> 0.47, <a id="xref-fig-2-7" class="xref-fig" href="#F2">Figure 2D</a>). Soy has the largest number of training lines among the six species (5,014) and has a marker to training line ratio close to one (<a id="xref-fig-1-8" class="xref-fig" href="#F1">Figure 1C</a>). Thus, we hypothesized the poor performance of the ANN models was in part due to our inability to train a network with so many features (markers) and so little training data (lines). During ANN model training, the weights assigned to each connection between nodes in neighboring layers of the network have to be estimated. Because every input marker is connected to every node in the first hidden layer, including more markers in the model will require more weights to be estimated, resulting in a more complex network that is more likely to underfit. In an ideal situation, to account for the complexity in these large networks, five to ten times more instances (lines) than features (markers) would need to be available for training (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Klimasauskas 1993</a>). Alternatively, one can reduce model complexity by only including markers that are most likely to be associated with the trait using feature selection methods.</p></div><div id="sec-12" class="subsection"><h3>Feature selection improves performance of ANN models</h3><p id="p-30">ANNs and sometimes other non-linear algorithms performed poorly compared to linear methods, which could be due to an insufficient number of training lines relative to the number of markers. To address this, we used feature selection to identify and select the markers most associated with trait variation. Because the number of markers associated with a trait is dependent on the genetic architecture of the trait and is not typically known, models were built using a range of numbers of markers (<em>P</em> = 10∼8,000) and were compared to models built using all available markers from each species. Because performing feature selection on the training and testing data can artificially inflate prediction accuracies (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bermingham <em>et al.</em> 2015</a>), feature selection was conducted on the training set only. This was repeated 10 times, using a different subset of lines for testing for each replicate (<strong>see Methods</strong>).</p><p id="p-31">Three feature selection algorithms (RF, BayesA, and Elastic Net (EN)) were compared to predict height in maize, the species with the largest number of markers (p) relative to training lines (n) (p:n = 850, <a id="xref-fig-1-9" class="xref-fig" href="#F1">Figure 1C</a>). While each algorithm selected a largely different subset of markers (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>, Figure S2A), the degree of overlap was significantly greater than random expectation. To demonstrate this, we randomly selected three sets of 8,000 maize markers and counted how many markers were present in all three sets 10,000 times and found that the 99<sup>th</sup> percentile of overlap was equal to 10, however we observed an average of 220 overlapping markers across replicates using these three feature selection approaches. When the different feature selection subsets were used to predict height in maize, there was a significant interaction between the number of available markers (p) and the feature selection method (repeat measures ANOVA: <em>p</em>-value = 1.7*10<sup>−12</sup>). Exploring this interaction further, we found that, while feature selection algorithms performed similarly with large n, RF tended to perform the best when fewer markers were selected for GP (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3B</a>; Figure S2B) and was therefore used to test the impact of feature selection on predicting height in the other five species.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/11/3691/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Impact of feature selection on GP algorithm performance. (A) Average number of overlapping markers in the top 8,000 markers selected by three feature selection algorithms for predicting height in maize across ten replicates. EN: Elastic Net. (B) Change in ANN predictive performance (r) at predicting height in maize as the number of input markers (p) selected by three feature selection algorithms (BayesA: BA, EN, and Random Forest: RF) increases. Dashed line: mean r when all 332,178 maize markers were used. (C) Mean r of rrBLUP, SVRlin, RF, GTB, and ANN models for predicting height using subsets or all (X-axis) markers as features across 10 replicate feature selection and ML runs for each of six species with their ratios of numbers of markers (p) to numbers of lines (n) shown. Data points were jittered horizontally for ease of visualization. (D) The significance (-log10(q-value), paired Wilcoxon Signed-Rank test) of the difference in r between models from different GP algorithms (colored as in Figure 3C) generated using a subset of 4,000 or 8,000 and all markers as input. Dotted line designates significant differences (p-value &lt; 0.05)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1024819697" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Impact of feature selection on GP algorithm performance. (A) Average number of overlapping markers in the top 8,000 markers selected by three feature selection algorithms for predicting height in maize across ten replicates. EN: Elastic Net. (B) Change in ANN predictive performance (&lt;em&gt;r&lt;/em&gt;) at predicting height in maize as the number of input markers (p) selected by three feature selection algorithms (BayesA: BA, EN, and Random Forest: RF) increases. Dashed line: mean &lt;em&gt;r&lt;/em&gt; when all 332,178 maize markers were used. (C) Mean &lt;em&gt;r&lt;/em&gt; of rrBLUP, SVR&lt;sub&gt;lin&lt;/sub&gt;, RF, GTB, and ANN models for predicting height using subsets or all (X-axis) markers as features across 10 replicate feature selection and ML runs for each of six species with their ratios of numbers of markers (p) to numbers of lines (n) shown. Data points were jittered horizontally for ease of visualization. (D) The significance (-log&lt;sub&gt;10&lt;/sub&gt;(&lt;em&gt;q&lt;/em&gt;-value), paired Wilcoxon Signed-Rank test) of the difference in &lt;em&gt;r&lt;/em&gt; between models from different GP algorithms (colored as in Figure 3C) generated using a subset of 4,000 or 8,000 and all markers as input. Dotted line designates significant differences (&lt;em&gt;p&lt;/em&gt;-value &lt; 0.05).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/11/3691/F3.medium.gif" width="240" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/9/11/3691/F3.medium.gif" width="240" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/11/3691/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/11/3691/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494778" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-32" class="first-child">Impact of feature selection on GP algorithm performance. (A) Average number of overlapping markers in the top 8,000 markers selected by three feature selection algorithms for predicting height in maize across ten replicates. EN: Elastic Net. (B) Change in ANN predictive performance (<em>r</em>) at predicting height in maize as the number of input markers (p) selected by three feature selection algorithms (BayesA: BA, EN, and Random Forest: RF) increases. Dashed line: mean <em>r</em> when all 332,178 maize markers were used. (C) Mean <em>r</em> of rrBLUP, SVR<sub>lin</sub>, RF, GTB, and ANN models for predicting height using subsets or all (X-axis) markers as features across 10 replicate feature selection and ML runs for each of six species with their ratios of numbers of markers (p) to numbers of lines (n) shown. Data points were jittered horizontally for ease of visualization. (D) The significance (-log<sub>10</sub>(<em>q</em>-value), paired Wilcoxon Signed-Rank test) of the difference in <em>r</em> between models from different GP algorithms (colored as in <a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3C</a>) generated using a subset of 4,000 or 8,000 and all markers as input. Dotted line designates significant differences (<em>p</em>-value &lt; 0.05).</p><div class="sb-div caption-clear"></div></div></div><p id="p-33">For species with a low p:n ratio (<em>i.e.</em>, soy and spruce), for all GP algorithms tested, as p increased the model performance tended to increase continuously (<em>e.g.</em>, all GP algorithms in sorghum) or, in some cases, the model performance reached a maximum (or a plateau) quickly (<em>e.g.</em>, in soy after 2,500 markers were used) (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3C</a>). For these species, there was no significant improvement in performance after feature selection (all <em>vs.</em> top 4,000) using any GP algorithm (one-sided, paired Wilcoxon Signed-Rank test: <em>q</em>-value = 0.98 ∼0.99; <a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3D</a>). For example, ANNs built using all 6,932 spruce markers performed no better than those built using the top 4,000 markers (<em>p</em>-value<em>=</em> 0.98).</p><p id="p-34">For species with a large p:n ratio (<em>i.e.</em>, maize, rice, sorghum, and switchgrass), a similar pattern was observed for rrBLUP, SVR<sub>lin</sub>, and GTB, where performance increased or reached a plateau as p increased and no significant improvement in performance was found after feature selection (<em>P</em> = 8,000) (<em>q</em>-value = 0.28 ∼0.99; <a id="xref-fig-3-6" class="xref-fig" href="#F3">Figure 3D</a>). However, for these four species, feature selection improved the performance of ANN models (<em>q</em>-value= 0.019 ∼0.047; <a id="xref-fig-3-7" class="xref-fig" href="#F3">Figure 3D</a>). For example, after feature selection prediction of height in maize using ANNs improved from <em>r=</em>0.17 to 0.41, a 141% increase. Ultimately, performing feature selection prior to ANN training for these four datasets with large p:n ratios, improved ANN performance (median <em>r</em> at 89% of the best <em>r</em> for each of the six species) compared to ANNs without feature selection (84% of the best <em>r</em>). Therefore, for the GP benchmark analysis, feature selection was performed prior to model building for additional traits for maize, rice, sorghum, and switchgrass and the top 8,000 markers were used. Because feature selection only improved the performance of RF models in sorghum and switchgrass, we did not perform feature selection before training RF models in the full benchmark study.</p><p id="p-35">While feature selection notably improved ANN performance, ANNs still often underperformed compared to other GP algorithms (<a id="xref-fig-3-8" class="xref-fig" href="#F3">Figure 3C</a>), meaning the they were unable to learn even the linear relationships between markers and traits that were found using the linear-based algorithms. Because ANNs should theoretically at least match the performance of linear algorithms, this suggests that the ANN hyperparameters are not optimal. Furthermore, we found that, even after feature selection, there was greater variation in performance across replicates for ANN models compared to rrBLUP, SVR<sub>lin</sub>, RF, and GB (Figure S2C-D), indicating the ANN models did not always converge on the best solution. One potential reason for the is that the final trained network can be heavily influenced by the initial weights used in ANN, which are selected randomly. In addition, while random weight initialization, a procedure we have used thus far, reduces bias in the network, it can also result in some networks converging on a local, rather than global, optimal solution.</p></div><div id="sec-13" class="subsection"><h3>Non-random initialization of ANN starting weights and convolutional layers improve ANN performance for some species</h3><p id="p-36">To reduce the likelihood of ANNs converging to locally optimal solutions, we developed an approach that allowed the ANNs to utilize the relationship between markers and traits determined by another GP algorithm. In this approach, a GP algorithm was applied to the training lines, and the coefficient or importance score assigned to each marker from this algorithm was used to seed the starting weights (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4A</a>). Four GP algorithms were tested to seed the weights: rrBLUP, BB, BL, and RF (referred to as ANN<sub>rrBLUP</sub>, ANN<sub>BB</sub>, ANN<sub>BL</sub>, and ANN<sub>RF</sub>, respectively). Because this approach could predispose the networks to only learn the relationship already identified by the seed algorithm, two steps were taken to re-introduce randomness into the network (see <strong>Methods</strong>). First, the seeded approach was only used to initialize starting weights for 25% of the nodes in the first hidden layer, while connection weights to the remaining 75% of nodes were initialized randomly as before. Second, noise was infused into the starting weights for the 25% of nodes that were seeded.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/11/3691/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Description and performance results of the seeded ANN approach. (A) An overview of the seeded ANN approach. The network in the top left is an example of a fully connected ANN with 6 input nodes (i.e., 6 markers), two hidden layers, and one output layer (i.e., predicted trait value). The blue node in the first hidden layer represents an example node that will have seeded weights. For this node, the weights (w) connecting each input node to the hidden node will be seeded from the coefficient/importance for each marker as determined by another GP algorithm using the training data. b: bias, which helps control the value at which the activation function will trigger. (B) The distribution of model performance (r) using only all random (None) or 25% seeded (rrBLUP, BayesB, BL, RF) weight initialization, and convolutional neural networks (CNNs). The mean performance of the overall top performing algorithm (i.e., not necessary ANN) shown as dotted red line." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1024819697" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Description and performance results of the seeded ANN approach. (A) An overview of the seeded ANN approach. The network in the top left is an example of a fully connected ANN with 6 input nodes (&lt;em&gt;i.e.&lt;/em&gt;, 6 markers), two hidden layers, and one output layer (&lt;em&gt;i.e.&lt;/em&gt;, predicted trait value). The blue node in the first hidden layer represents an example node that will have seeded weights. For this node, the weights (&lt;em&gt;w&lt;/em&gt;) connecting each input node to the hidden node will be seeded from the coefficient/importance for each marker as determined by another GP algorithm using the training data. &lt;em&gt;b&lt;/em&gt;: bias, which helps control the value at which the activation function will trigger. (B) The distribution of model performance (&lt;em&gt;r&lt;/em&gt;) using only all random (None) or 25% seeded (rrBLUP, BayesB, BL, RF) weight initialization, and convolutional neural networks (CNNs). The mean performance of the overall top performing algorithm (&lt;em&gt;i.e.&lt;/em&gt;, not necessary ANN) shown as dotted red line.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/11/3691/F4.medium.gif" width="440" height="342"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/9/11/3691/F4.medium.gif" width="440" height="342"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/11/3691/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/11/3691/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494737" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-37" class="first-child">Description and performance results of the seeded ANN approach. (A) An overview of the seeded ANN approach. The network in the top left is an example of a fully connected ANN with 6 input nodes (<em>i.e.</em>, 6 markers), two hidden layers, and one output layer (<em>i.e.</em>, predicted trait value). The blue node in the first hidden layer represents an example node that will have seeded weights. For this node, the weights (<em>w</em>) connecting each input node to the hidden node will be seeded from the coefficient/importance for each marker as determined by another GP algorithm using the training data. <em>b</em>: bias, which helps control the value at which the activation function will trigger. (B) The distribution of model performance (<em>r</em>) using only all random (None) or 25% seeded (rrBLUP, BayesB, BL, RF) weight initialization, and convolutional neural networks (CNNs). The mean performance of the overall top performing algorithm (<em>i.e.</em>, not necessary ANN) shown as dotted red line.</p><div class="sb-div caption-clear"></div></div></div><p id="p-38">Applying this approach to predict plant height we found that ANN performance improved for three of six species (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4B</a>). For example, the average performance for rice without seeding (ANN) was <em>r =</em> 0.25 and with seeding from BL (ANN<sub>BL</sub>) was <em>r</em> = 0.32, a 28% improvement, while for sorghum, ANN<sub>BL</sub> had &lt;0.1% improvement over the original ANN methods. Seeding ANN models did not significantly reduce the amount of variation in model performance across replicates (repeated measure ANOVA: <em>p</em>-value= 0.39, Table S4). Ultimately, seeded ANN models had a median performance between 89–90% of the best <em>r</em> for each species (compared to 89% with random initialization, <a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4B</a>). While this represented only a moderate improvement, we included the seeded ANN approach in the benchmark analysis because of how substantial the improvement was for some species (<em>i.e.</em>, rice).</p><p id="p-39">Another deep learning strategy for reducing the complexity of GP problems and consequently decreasing the likelihood of converging on local optimum is to use convolutional and pooling layers to summarize local patterns of genetic markers and learn from these summaries (<a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">Ma <em>et al.</em> 2018</a>). We tested this approach by training Convolutional Neural Networks (CNNs) to predict plant height (Figure S3A). Notably, feature selection (n= 8,000) had either no or a negative impact on CNN performance. For example, the average performance of CNNs at predicting height in maize, the species with the most genetic markers, was <em>r</em> = 0.39, but dropped to <em>r</em> = 0.37 after feature selection. CNNs performed better than ANNs at predicting height in two of six species (yellow; <a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4B</a>), with the biggest improvement in rice where the average performance increased from <em>r</em> = 0.25 using ANNs to <em>r</em> = 0.32 using CNNs, a 32% improvement. While CNN models did not reduce the amount of variation in model performance across replicates (repeated measure ANOVA: <em>p</em>-value = 0.08, Table S4), we included CNNs in the final benchmark analysis because of the promising results in rice and switchgrass.</p></div><div id="sec-14" class="subsection"><h3>No one GP algorithm performs best for all species and traits</h3><p id="p-40">Having established best practices for hyperparameter and feature selection for our datasets, we next compared the performance of all GP algorithms for predicting three traits in each of the six species. For maize, rice, and soy, these traits included height, flowering time, and yield (<a id="xref-fig-1-10" class="xref-fig" href="#F1">Figure 1C</a>). For species where data were not available for one or more of these traits, other traits were used (see the panel labeled “Others”, <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5A</a>). As with past efforts to benchmark GP algorithms (<a id="xref-ref-32-4" class="xref-bibr" href="#ref-32">Heslot <em>et al.</em> 2012</a>; <a id="xref-ref-50-3" class="xref-bibr" href="#ref-50">Neves <em>et al.</em> 2012</a>), different algorithms performed best for different species/trait combinations (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5A</a>; Table S5). Thus, we utilized the predictive power of multiple algorithms to establish an ensemble prediction using all (except CNN: EN<sub>11</sub>) or a subset of five (EN<sub>5</sub>) algorithms (see <strong>Methods</strong>). The ensemble models consistently performed well, with EN<sub>5</sub> or EN<sub>11</sub> being the best (three) or tied for the best (nine) algorithm for 12 of the 18 species/trait combinations included in the benchmark and had a median performance rank of 3 (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5B</a>; Table S6). For the remaining 6 species/trait combinations where EN<sub>5</sub> or EN<sub>11</sub> weren’t among the best performers, they tended to perform only slightly worse (median % of best <em>r</em> = 99.2%, <a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5A</a>). This suggests that ensemble-based predictions are more stable and more likely to result in better trait predictions than a single algorithm.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/11/3691/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparison of algorithms for predicting additional traits. (A) Mean model performance (r; text) for each species/trait combination (y-axis) for each GP algorithm (x-axis). White text: r of the best performing algorithm(s) for a species. Colored boxes: percent of best performance (r) for a species, with the top algorithm for each species = 100% (red). The median % of best performance for each GP algorithm for each type of trait (i.e., height, developmental timing, yield, other) is shown below each heatmap. GM: sorghum grain moisture. DBH and DE: diameter at breast height and wood density, respectively, for spruce. ST: standability for switchgrass. (B) Top left: summary of the number of species/trait combinations that were predicted best by an ensemble (gray) or a non-ensemble model (yellow), or predicted equally well by both (purple). Bottom right: among non-ensemble models that performed or tied for the best, the number of species/trait combinations that were predicted best by a linear (blue) or a non-linear model (green) or predicted equally well by both (orange). (C) Percent of replicates where one GP algorithm (y-axis, winner) outperformed another GP algorithm (x-axis, loser) for predicting height in switchgrass. Orange and cyan texts: linear and non-linear algorithms, respectively. (D) Hierarchical clustering of GP algorithms based on mean predictive performance across all species/trait combinations. Algorithm colored as in (C)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1024819697" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Comparison of algorithms for predicting additional traits. (A) Mean model performance (&lt;em&gt;r&lt;/em&gt;; text) for each species/trait combination (y-axis) for each GP algorithm (x-axis). White text: &lt;em&gt;r&lt;/em&gt; of the best performing algorithm(s) for a species. Colored boxes: percent of best performance (&lt;em&gt;r&lt;/em&gt;) for a species, with the top algorithm for each species = 100% (red). The median % of best performance for each GP algorithm for each type of trait (&lt;em&gt;i.e.&lt;/em&gt;, height, developmental timing, yield, other) is shown below each heatmap. GM: sorghum grain moisture. DBH and DE: diameter at breast height and wood density, respectively, for spruce. ST: standability for switchgrass. (B) Top left: summary of the number of species/trait combinations that were predicted best by an ensemble (gray) or a non-ensemble model (yellow), or predicted equally well by both (purple). Bottom right: among non-ensemble models that performed or tied for the best, the number of species/trait combinations that were predicted best by a linear (blue) or a non-linear model (green) or predicted equally well by both (orange). (C) Percent of replicates where one GP algorithm (y-axis, winner) outperformed another GP algorithm (x-axis, loser) for predicting height in switchgrass. Orange and cyan texts: linear and non-linear algorithms, respectively. (D) Hierarchical clustering of GP algorithms based on mean predictive performance across all species/trait combinations. Algorithm colored as in (C).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/11/3691/F5.medium.gif" width="440" height="359"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/9/11/3691/F5.medium.gif" width="440" height="359"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/11/3691/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/11/3691/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494633" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-41" class="first-child">Comparison of algorithms for predicting additional traits. (A) Mean model performance (<em>r</em>; text) for each species/trait combination (y-axis) for each GP algorithm (x-axis). White text: <em>r</em> of the best performing algorithm(s) for a species. Colored boxes: percent of best performance (<em>r</em>) for a species, with the top algorithm for each species = 100% (red). The median % of best performance for each GP algorithm for each type of trait (<em>i.e.</em>, height, developmental timing, yield, other) is shown below each heatmap. GM: sorghum grain moisture. DBH and DE: diameter at breast height and wood density, respectively, for spruce. ST: standability for switchgrass. (B) Top left: summary of the number of species/trait combinations that were predicted best by an ensemble (gray) or a non-ensemble model (yellow), or predicted equally well by both (purple). Bottom right: among non-ensemble models that performed or tied for the best, the number of species/trait combinations that were predicted best by a linear (blue) or a non-linear model (green) or predicted equally well by both (orange). (C) Percent of replicates where one GP algorithm (y-axis, winner) outperformed another GP algorithm (x-axis, loser) for predicting height in switchgrass. Orange and cyan texts: linear and non-linear algorithms, respectively. (D) Hierarchical clustering of GP algorithms based on mean predictive performance across all species/trait combinations. Algorithm colored as in (C).</p><div class="sb-div caption-clear"></div></div></div><p id="p-42">Focusing on the species/trait combinations where one of the non-ensemble algorithms was or tied for best, we found that a linear algorithm performed best for five of the species/trait combinations, a non-linear algorithm performed best for four species/trait combinations, and both a linear and a non-linear algorithm performed equally well for the remaining six species/trait combinations (<a id="xref-fig-5-5" class="xref-fig" href="#F5">Figure 5B</a>). This finding suggests that linear and non-linear algorithms are equally well suited for GP. The linear algorithms BRR and BA performed best overall, being among the top performers for 9 and 8 traits, respectively, and with the top two median ranks of five and 4.5, respectively (Table S6). The top performing non-linear algorithm was SVR<sub>poly</sub>, which was among the top performers for 8 traits and had a median rank of 6. There was notably greater performance variation across species/traits for non-linear algorithms (mean variance = 1.03%) compared linear algorithms (mean variance = 0.65%) (Table S6). For example, SVR<sub>rbf</sub> performed poorly at predicting developmental timing traits (median 83% of the best <em>r</em>), however it had or was tied for the best prediction for three of the four “other” traits (median 100% of the best <em>r</em>) (<a id="xref-fig-5-6" class="xref-fig" href="#F5">Figure 5A</a>). Results from ANN models using randomly initialized (ANN) and BB seeded (ANN<sub>BB</sub>) weights are shown because ANN<sub>BB</sub> had the best performance of the seeded ANN models (see Table S5, S6 for results from other seeded ANNs). Notably, none of the randomly initialized ANN (median rank = 13.5), the ANN<sub>BB</sub> (median rank = 13), or the CNN (median rank = 15.5) models performed best for any trait (Table S6).</p><p id="p-43">One limitation of comparing the mean score or performance rank is that small but consistent differences in model performance could be missed. To account for this, we also calculated the number of times an algorithm outperformed another algorithm for each trait across the replicates. Using this metric, we were able to identify algorithms that consistently outperformed others for a given trait/species combination (<a id="xref-fig-5-7" class="xref-fig" href="#F5">Figure 5C</a>, Figure S4<strong>)</strong>. We frequently observed that linear algorithms had higher win percentages than nonlinear algorithms, this was the case for all three traits in maize and soybean for example (Figure S4). However, there were plenty of exceptions. RF and SVR<sub>rbf</sub> had higher win percentages than linear algorithms for predicting height and diameter at breast height (DBH) in spruce and ANN<sub>BB</sub> had a higher win percentage than all algorithms except BA and BB for predicting flowering time in rice (Figure S3). In a few cases, assessing win percentages allowed us to identify winners when mean predictive performance (<em>r</em>) was tied. For example, for predicting height in switchgrass. SVR<sub>poly</sub> had the same average performance (<em>r</em> = 0.61) as multiple of the linear algorithms (<em>i.e.</em>, rrBLUP, BA, etc.), however, it outperformed those algorithms in 70–80% of replicates (<a id="xref-fig-5-8" class="xref-fig" href="#F5">Figure 5C</a>).</p><p id="p-44">In order to determine which algorithms perform similarly, we performed hierarchical clustering of the algorithms based on their performance across the 18 species/trait combinations (from <a id="xref-fig-5-9" class="xref-fig" href="#F5">Figure 5A</a>). Interestingly, linear and non-linear algorithms did not clearly separate from each other (<a id="xref-fig-5-10" class="xref-fig" href="#F5">Figure 5D</a>). For example, rrBLUP and SVR<sub>lin</sub> were more similar to the neural network based models (<em>i.e.</em>, CNN and ANN<sub>BB</sub>), than they were to the linear Bayesian algorithms (<em>i.e.</em>, BA, BB, BL, and BRR). Notably, while the Bayesian algorithms tended to cluster together closely performance-wise, the non-linear algorithms tended to have a greater distance between them. Finally, in order to identify if algorithm performance was similar for specific types of traits (<em>e.g.</em>, whether similar algorithms perform well at predicting traits related to developmental timing) or across species/population composition (<em>e.g.</em>, whether similar algorithms perform well on diversity panels), we performed hierarchical clustering of each species/trait based on performance of all 14 algorithms (from <a id="xref-fig-5-11" class="xref-fig" href="#F5">Figure 5A</a>). Surprisingly, species/trait combinations with similar patterns of algorithm performance were often not the same species, trait, or population type (Figure S5), suggesting that we cannot generalize easily the differences in performance based on species, trait, or population type.</p></div></div><div class="section discussion" id="sec-15"><h2 class="">Discussion</h2><p id="p-45">We conducted a benchmarking comparison of GP algorithms on 18 species/trait combinations that differ in the type and size of the training data set and of the marker data available. Similar to previous GP algorithm benchmark studies conducted on smaller datasets (<a id="xref-ref-32-5" class="xref-bibr" href="#ref-32">Heslot <em>et al.</em> 2012</a>; <a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Blondel <em>et al.</em> 2015</a>), a key result from this analysis is that no one model performs best for all species and all traits. We further demonstrate that, while similar algorithms perform similarly across the 18 species/trait combinations, algorithm performance was not clearly related to the trait type or population composition. With that said, linear algorithms tend to perform consistently well, while the performance of non-linear algorithms varied widely by trait. Studies of gene networks have shown that non-additive interactions (<em>e.g.</em>, epistasis, dominance) are important for development and regulation of complex traits (<a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Holland 2007</a>; <a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">Monir and Zhu 2018</a>). One may expect approaches that can consider non-linear combinations would therefore be better suited for modeling complex trait. This was not the case and we found the inconsistency of non-linear algorithms surprising.</p><p id="p-46">We have three, non-mutually exclusive, explanations for why linear algorithms often outperform non-linear algorithms. First, the traits included in this study vary in their genetic architecture (<em>i.e.</em>, the number and distribution of allele effects), therefore we may be observing that linear algorithms outperform non-linear algorithms when the trait has a predominantly additive genetic basis. Second, there is evidence that even highly complex biological systems generate allelic patterns that are consistent with a linear, additive genetic model because of the discrete nature of DNA variation and the fact that many markers have extreme allele frequencies (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Hill <em>et al.</em> 2008</a>). The proportion of dominance and epistatic variance that can be captured by an additive (<em>i.e.</em>, linear) model increases when allele frequencies are extreme (<a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Hill <em>et al.</em> 2008</a>). This phenomenon is even more important with inbred lines (<em>e.g.</em>, soy and rice); where, at each locus there are only 2 possible variants (<em>e.g.</em>, AA and TT); thus, the additive model fully captures the single-locus genetic variance. However, the fraction of epistatic variance that can be captured by an additive model depends on how many multi-locus genotypes are present in the data and this depends on allele frequencies. Thus, the distribution of allele frequency (which due to mutation, selection, and drift is often enriched at extreme values) is one of the reasons why additive models often capture and perform very well at predicting traits that at the biological level are affected by complex epistatic networks. Finally, a third explanation is that the amount of training data available for most GP problems was insufficient for learning non-linear interactions between large numbers of markers, therefore the linear models, which focus on modeling linear relationships, outperform the non-linear models.</p><p id="p-47">Three findings from our study suggest that limited training data plays a role. First, we found that non-linear algorithms performed better at predicting traits in species with a small marker number to population size (p:n) ratio. For example, RF, SVR<sub>poly</sub>, and SVR<sub>rbf</sub> performed best at predicting traits in spruce and ANN models tended to perform better at predicting traits in soy, the species with the second smallest and smallest p:n, respectively. Second, the ANN models significantly improved after feature selection. This was not the case for other algorithms in our study or with previous efforts to use feature selection for GP (<a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Vazquez <em>et al.</em> 2010</a>; <a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Bermingham <em>et al.</em> 2015</a>). For example, for predicting traits in Holstein cattle, the top 2,000 markers had only 95% of the predictive ability of all the markers using BL (<a id="xref-ref-66-2" class="xref-bibr" href="#ref-66">Vazquez <em>et al.</em> 2010</a>). With a fixed training data size, prediction accuracy is a function of how much genetic variation is captured by markers in linkage disequilibrium with quantitative trait loci and the accuracy of the estimated effects (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Goddard 2009</a>). Because feature selection removes markers from the model, such decreases in performance after feature selection for non-ANN models are likely due to the reduction in the amount of genetic variation captured without a subsequent increase in the accuracy of the estimated effects. However, we hypothesize that feature selection significantly improved performance for ANNs because it improved the accuracy of the estimated effects (<em>i.e.</em>, the connection weights) more than it reduced the amount of genetic variation captured. Third, ANNs that have been trained on small datasets often have unstable performance likely because ANNs are sensitive to the initialized weight values when they do not have enough training data to learn from (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">LeBaron and Weigend 1998</a>; <a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Shaikhina and Khovanova 2017</a>). We observed greater instability in performance across replicates for ANNs compared to other algorithms (Figure S2C-D), suggesting that our ANN models may have benefitted from additional training data.</p><p id="p-48">However, a recent study involving large sample size (n∼80,000) in humans compared linear models with two types of ANN algorithms, multilayer perceptron and convolutional neural networks, and did not find any clear superiority of the ANN methods relative to linear models, if anything the linear model offered higher predictive power than the ANNs (<a id="xref-ref-3-3" class="xref-bibr" href="#ref-3">Bellot <em>et al.</em> 2018</a>). While they also found that feature selection improved the performance of their ANN models, using the top 10k of the 50k markers, these models still did not outperform the linear models (<a id="xref-ref-3-4" class="xref-bibr" href="#ref-3">Bellot <em>et al.</em> 2018</a>). Given that these results are from a single study in humans, we believe it will be informative to benchmark ANNs on a larger crop dataset in the future.</p><p id="p-49">While there is a great deal of excitement about the uses of deep learning in the field of genetics, there is still much work to be done to improve performance of deep learning-based models. In this study we identified dimensionality as a major limitation to training ANNs for GP. Additional areas of deep learning research also need to be further explored. For example, in this study we limited the ANN hyperparameter space searched because the grid search method was too computationally intensive to be more thorough. Because changes in hyperparameters had a large impact on model performance, further hyperparameter tuning could lead to better performing models. For example, we limited our search to include nine possible network architectures with between one and three hidden layers each containing between 5-100 nodes (Table S2), but it is possible that ANNs with different network architectures, such as more hidden layers, or different combinations of layer sizes, could have performed better. Similarly, given that the hyperparameter space for CNN models was only tested for one species and trait (height in rice), it is likely that model-specific hyperparameter selection could improve the performance of CNN models beyond what we were able to achieve here.</p><p id="p-50">In summary, we provided a thorough comparison of 12 GP algorithms and two ensembles for predicting diverse traits in six plant species with a range of marker types and numbers and population types and sizes. We found that no GP algorithm was best for all species/trait combinations and that trait type or population type were not closely associated with which algorithms worked best. While neural network approaches did not tend to outperform linear or other non-linear models, strategies to tailor neural networks for GP problems (<em>e.g.</em>, non-random initialization of stating weights, convolutional and pooling layers) show promise. Unlike previous GP algorithm benchmark studies (<a id="xref-ref-32-6" class="xref-bibr" href="#ref-32">Heslot <em>et al.</em> 2012</a>), we found that the performance of ensemble models, generated by combining predictions from multiple individual GP algorithms, consistently tied with or exceeded the performance of the best individual algorithm. Taken together, these finds lead us to recommend that breeders test the performance of multiple algorithms on their training population to identify which algorithm or combination of algorithms performs best for traits important to their breeding program.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-51">We thank Peipei Wang and John Lloyd from the Shiu lab, Gabriel Rovere from the MSU QuantGen group, and Fouad Bahrpeyma from the Insight Center for their valuable suggestions to our project. This work was supported by the National Science Foundation (NSF) Graduate Research Fellowship [Fellow ID: 2015196719], Graduate Research Opportunities Abroad (GROW) Fellowship to C.B.A.; NSF PlantGenomics Research Experiences for Undergraduate to E.B.; the U.S. Department of Energy Great Lakes Bioenergy Research Center [BER DE-SC0018409] and National Science Foundation [IOS-1546617, DEB-1655386] to S.-H.S.; and the National Institutes of Health [R01GM099992, R01FM101219] to G.D.L.C.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-52">Supplemental material available at FigShare: <a href="https://doi.org/10.25387/g3.9855590">https://doi.org/10.25387/g3.9855590</a>.</p></li><li class="fn" id="fn-2"><p id="p-53"><em>Communicating editor: J. Birchler</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2019-07-03"><span class="received-label">Received </span>July 3, 2019.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2019-09-09"><span class="accepted-label">Accepted </span>September 9, 2019.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2019 Azodi <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3691.1" data-doi="10.15252/msb.20156651"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Angermueller</span>,  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Pärnamaa</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Parts</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">O.</span> <span class="cit-name-surname">Stegle</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Deep learning for computational biology.</span> <abbr class="cit-jnl-abbrev">Mol. Syst. Biol.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">878</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.15252/msb.20156651</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BSystems%2BBiology%26rft.stitle%253DMol%2BSyst%2BBiol%26rft.aulast%253DAngermueller%26rft.auinit1%253DC.%26rft.volume%253D12%26rft.issue%253D7%26rft.spage%253D878%26rft.epage%253D878%26rft.atitle%253DDeep%2Blearning%2Bfor%2Bcomputational%2Bbiology%26rft_id%253Dinfo%253Adoi%252F10.15252%252Fmsb.20156651%26rft_id%253Dinfo%253Apmid%252F27474269%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibXNiIjtzOjU6InJlc2lkIjtzOjg6IjEyLzcvODc4IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMS8zNjkxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3691.2" data-doi="10.1186/1471-2164-15-1048"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Beaulieu</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. K.</span> <span class="cit-name-surname">Doerksen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">MacKay</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Rainville</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Bousquet</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genomic selection accuracies within and between environments and small breeding groups in white spruce.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">1048</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2164-15-1048</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D15%26rft.spage%253D1048%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-15-1048%26rft_id%253Dinfo%253Apmid%252F25442968%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-15-1048&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25442968&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3691.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3691.3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bellot</span>,  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">de los Campos</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Pérez-Enciso</span></span></li></ol><cite>, 2018 Can Deep Learning Improve Genomic Prediction of Complex Human Traits? Genetics genetics.<span class="cit-pub-date">2018</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3691.4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Benjamini</span>,  <span class="cit-name-given-names">Y.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Hochberg</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Controlling the False Discovery Rate: A Practical and Powerful Approach to Multiple Testing. J R Stat Soc Ser.</span> <abbr class="cit-jnl-abbrev">B Stat Methodol</abbr> <span class="cit-vol">57</span>: <span class="cit-fpage">289</span>–<span class="cit-lpage">300</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DB%2BStat%2BMethodol%26rft.volume%253D57%26rft.spage%253D289%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3691.5" data-doi="10.1038/srep10312"><div class="cit-metadata"><cite>Bermingham, M. L., R. Pong-Wong, A. Spiliopoulou, C. Hayward, I. Rudan <em>et al.</em>, 2015 Application of high-dimensional feature selection: evaluation for genomic prediction in man. Sci. Rep. 1–12.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/srep10312</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSCIENTIFIC%2BREPORTS%26rft.volume%253D5%26rft.spage%253D10312%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsrep10312%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/srep10312&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3691.6" data-doi="10.1371/journal.pone.0128570"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Blondel</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Onogi</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Iwata</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Ueda</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">A Ranking Approach to Genomic Selection.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">e0128570</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0128570</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D10%26rft.spage%253De0128570%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0128570%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0128570&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3691.7" data-doi="10.1023/A:1010933404324"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Breiman</span>,  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Random Forests.</span> <abbr class="cit-jnl-abbrev">Mach. Learn.</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">5</span>–<span class="cit-lpage">32</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1023/A:1010933404324</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMach.%2BLearn.%26rft.volume%253D45%26rft.spage%253D5%26rft_id%253Dinfo%253Adoi%252F10.1023%252FA%253A1010933404324%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1023/A:1010933404324&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000170489900001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3691.8" data-doi="10.1017/S0016672310000285"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>,  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Gianola</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G. J.</span> <span class="cit-name-surname">Rosa</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. A.</span> <span class="cit-name-surname">Weigel</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Crossa</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Semi-parametric genomic-enabled prediction of genetic values using reproducing kernel Hilbert spaces methods.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">92</span>: <span class="cit-fpage">295</span>–<span class="cit-lpage">308</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1017/S0016672310000285</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DDE%2BLos%2BCampos%26rft.auinit1%253DG.%26rft.volume%253D92%26rft.issue%253D4%26rft.spage%253D295%26rft.epage%253D308%26rft.atitle%253DSemi-parametric%2Bgenomic-enabled%2Bprediction%2Bof%2Bgenetic%2Bvalues%2Busing%2Breproducing%2Bkernel%2BHilbert%2Bspaces%2Bmethods.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672310000285%26rft_id%253Dinfo%253Apmid%252F20943010%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672310000285&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20943010&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3691.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3691.9" data-doi="10.1534/genetics.112.143313"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>,  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">Hickey</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Pong-Wong</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H. D.</span> <span class="cit-name-surname">Daetwyler</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. P. L.</span> <span class="cit-name-surname">Calus</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Whole-Genome Regression and Prediction Methods Applied to Plant and Animal Breeding.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">193</span>: <span class="cit-fpage">327</span>–<span class="cit-lpage">345</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.112.143313</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.143313%26rft_id%253Dinfo%253Apmid%252F22745228%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkzLzIvMzI3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMS8zNjkxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3691.10" data-doi="10.1534/genetics.109.101501"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>,  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Naya</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Gianola</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Crossa</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Legarra</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Predicting quantitative traits with regression models for dense molecular markers and pedigree.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">182</span>: <span class="cit-fpage">375</span>–<span class="cit-lpage">385</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.109.101501</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.109.101501%26rft_id%253Dinfo%253Apmid%252F19293140%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTgyLzEvMzc1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMS8zNjkxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3691.11" data-doi="10.1023/A:1012450327387"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chapelle</span>,  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Vapnik</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">O.</span> <span class="cit-name-surname">Bousquet</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Mukherjee</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Choosing Multiple Parameters for Support Vector Machines.</span> <abbr class="cit-jnl-abbrev">Mach. Learn.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">131</span>–<span class="cit-lpage">159</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1023/A:1012450327387</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMach.%2BLearn.%26rft.volume%253D46%26rft.spage%253D131%26rft_id%253Dinfo%253Adoi%252F10.1023%252FA%253A1012450327387%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1023/A:1012450327387&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3691.12" data-doi="10.1016/j.tplants.2014.05.006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Desta</span>,  <span class="cit-name-given-names">Z. A.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Ortiz</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genomic selection: genome-wide prediction in plant improvement.</span> <abbr class="cit-jnl-abbrev">Trends Plant Sci.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">592</span>–<span class="cit-lpage">601</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.tplants.2014.05.006</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BPlant%2BSci.%26rft.volume%253D19%26rft.spage%253D592%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tplants.2014.05.006%26rft_id%253Dinfo%253Apmid%252F24970707%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tplants.2014.05.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24970707&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3691.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-confproc" id="cit-9.11.3691.13" data-doi="10.1007/3-540-45014-9_1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dietterich</span>,  <span class="cit-name-given-names">T. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> Ensemble methods in machine learning. Int. Workshop Mult. Classif. Syst.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/3-540-45014-9_1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.volume%253D1857%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1007%252F3-540-45014-9_1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/3-540-45014-9_1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000165404700001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3691.14" data-doi="10.1186/s12711-015-0097-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ehret</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Hochstuhl</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Gianola</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Thaller</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Application of neural networks with back-propagation to genome-enabled prediction of complex traits in Holstein-Friesian and German Fleckvieh cattle.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">22</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12711-015-0097-5</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D47%26rft.spage%253D22%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12711-015-0097-5%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12711-015-0097-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3691.15" data-doi="10.3835/plantgenome2011.08.0024"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Endelman</span>,  <span class="cit-name-given-names">J. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Ridge regression and other kernels for genomic selection with R package rrBLUP.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">250</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2011.08.0024</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D4%26rft.spage%253D250%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2011.08.0024%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2011.08.0024&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3691.16" data-doi="10.1111/tpj.13041"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Evans</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Crisovan</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Barry</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Daum</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Jenkins</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Diversity and population structure of northern switchgrass as revealed through exome capture sequencing.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">84</span>: <span class="cit-fpage">800</span>–<span class="cit-lpage">815</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/tpj.13041</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BJ.%26rft.volume%253D84%26rft.spage%253D800%26rft_id%253Dinfo%253Adoi%252F10.1111%252Ftpj.13041%26rft_id%253Dinfo%253Apmid%252F26426343%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/tpj.13041&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26426343&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3691.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3691.17" data-doi="10.3835/plantgenome2017.06.0055"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Evans</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. D.</span> <span class="cit-name-surname">Sanciangco</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. H.</span> <span class="cit-name-surname">Lau</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Crisovan</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Barry</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Extensive Genetic Diversity is Present within North American Switchgrass Germplasm.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr>. <span class="cit-vol">11</span><span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2017.06.0055</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2017.06.0055%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2017.06.0055&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3691.18"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fernandes</span>,  <span class="cit-name-given-names">S. B.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. O. G.</span> <span class="cit-name-surname">Dias</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. F.</span> <span class="cit-name-surname">Ferreira</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. J.</span> <span class="cit-name-surname">Brown</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Efficiency of multi-trait, indirect, and trait-assisted genomic selection for improvement of biomass sorghum.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">131</span>: <span class="cit-fpage">747</span>–<span class="cit-lpage">755</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D131%26rft.spage%253D747%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3691.19" data-doi="10.1214/aos/1013203451"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Friedman</span>,  <span class="cit-name-given-names">J. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Greedy function approximation: A gradient boosting machine.</span> <abbr class="cit-jnl-abbrev">Ann. Stat.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">1189</span>–<span class="cit-lpage">1232</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1214/aos/1013203451</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BStat.%26rft.volume%253D29%26rft.spage%253D1189%26rft_id%253Dinfo%253Adoi%252F10.1214%252Faos%252F1013203451%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1214/aos/1013203451&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3691.20" data-doi="10.1534/genetics.105.049510"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>,  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. L.</span> <span class="cit-name-surname">Fernando</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Stella</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Genomic-assisted prediction of genetic value with semiparametric procedures.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">173</span>: <span class="cit-fpage">1761</span>–<span class="cit-lpage">1776</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.105.049510</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.105.049510%26rft_id%253Dinfo%253Apmid%252F16648593%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3My8zLzE3NjEiO3M6NDoiYXRvbSI7czoxOToiL2dnZy85LzExLzM2OTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3691.21"><div class="cit-metadata unstructured">Girija, S. S., 2016 Tensorflow: Large-scale machine learning on heterogeneous distributed systems.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3691.22"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Glorot</span>,  <span class="cit-name-given-names">X.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Bengio</span></span></li></ol><cite> Understanding the difficulty of training deep feedforward neural networks. <span class="cit-pub-date">2010</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3691.23" data-doi="10.1007/s10709-008-9308-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>,  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Genomic selection: prediction of accuracy and maximisation of long term response.</span> <abbr class="cit-jnl-abbrev">Genetica</abbr> <span class="cit-vol">136</span>: <span class="cit-fpage">245</span>–<span class="cit-lpage">257</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s10709-008-9308-0</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DGenetica%26rft.aulast%253DGoddard%26rft.auinit1%253DM.%26rft.volume%253D136%26rft.issue%253D2%26rft.spage%253D245%26rft.epage%253D257%26rft.atitle%253DGenomic%2Bselection%253A%2Bprediction%2Bof%2Baccuracy%2Band%2Bmaximisation%2Bof%2Blong%2Bterm%2Bresponse.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10709-008-9308-0%26rft_id%253Dinfo%253Apmid%252F18704696%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10709-008-9308-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18704696&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3691.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000265818900005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3691.24" data-doi="10.1007/s00122-012-1868-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">González-Camacho</span>,  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">de los Campos</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Pérez</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Gianola</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. E.</span> <span class="cit-name-surname">Cairns</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genome-enabled prediction of genetic values using radial basis function neural networks.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">125</span>: <span class="cit-fpage">759</span>–<span class="cit-lpage">771</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-012-1868-9</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DGonzalez-Camacho%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D125%26rft.issue%253D4%26rft.spage%253D759%26rft.epage%253D771%26rft.atitle%253DGenome-enabled%2Bprediction%2Bof%2Bgenetic%2Bvalues%2Busing%2Bradial%2Bbasis%2Bfunction%2Bneural%2Bnetworks.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-012-1868-9%26rft_id%253Dinfo%253Apmid%252F22566067%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-012-1868-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22566067&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3691.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3691.25" data-doi="10.1186/s12864-016-2553-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">González-Camacho</span>,  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Crossa</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Pérez-Rodríguez</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Ornella</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Gianola</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genome-enabled prediction using probabilistic neural network classifiers.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">208</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12864-016-2553-1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D17%26rft.spage%253D208%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864-016-2553-1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12864-016-2553-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3691.26" data-doi="10.3835/plantgenome2017.11.0104"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">González-Camacho</span>,  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Ornella</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Pérez-Rodríguez</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Gianola</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Dreisigacker</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Applications of Machine Learning Methods to Genomic Selection in Breeding Wheat for Rust Resistance.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">170104</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2017.11.0104</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D11%26rft.spage%253D170104%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2017.11.0104%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2017.11.0104&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3691.27" data-doi="10.1186/1297-9686-43-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">González-Recio</span>,  <span class="cit-name-given-names">O.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Forni</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genome-wide prediction of discrete traits using Bayesian regressions and machine learning.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">7</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1297-9686-43-7</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DGonzalez-Recio%26rft.auinit1%253DO.%26rft.volume%253D43%26rft.spage%253D7%26rft.epage%253D7%26rft.atitle%253DGenome-wide%2Bprediction%2Bof%2Bdiscrete%2Btraits%2Busing%2BBayesian%2Bregressions%2Band%2Bmachine%2Blearning.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-43-7%26rft_id%253Dinfo%253Apmid%252F21329522%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-43-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21329522&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3691.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3691.28" data-doi="10.3168/jds.2012-5630"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">González-Recio</span>,  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. A.</span> <span class="cit-name-surname">Jiménez-Montero</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Alenda</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The gradient boosting algorithm and random boosting for genome-assisted evaluation in large data sets.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">96</span>: <span class="cit-fpage">614</span>–<span class="cit-lpage">624</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3168/jds.2012-5630</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BDairy%2BSci.%26rft.volume%253D96%26rft.spage%253D614%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2012-5630%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2012-5630&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3691.29" data-doi="10.1186/1471-2105-12-186"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Habier</span>,  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. L.</span> <span class="cit-name-surname">Fernando</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Kizilkaya</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. J.</span> <span class="cit-name-surname">Garrick</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Extension of the bayesian alphabet for genomic selection.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">186</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2105-12-186</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DHabier%26rft.auinit1%253DD.%26rft.volume%253D12%26rft.spage%253D186%26rft.epage%253D186%26rft.atitle%253DExtension%2Bof%2Bthe%2Bbayesian%2Balphabet%2Bfor%2Bgenomic%2Bselection.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-12-186%26rft_id%253Dinfo%253Apmid%252F21605355%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-12-186&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21605355&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3691.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3691.30" data-doi="10.2135/cropsci2010.03.0178"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hansey</span>,  <span class="cit-name-given-names">C. N.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">Johnson</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. S.</span> <span class="cit-name-surname">Sekhon</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. M.</span> <span class="cit-name-surname">Kaeppler</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">de Leon</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genetic diversity of a maize association population with restricted phenology.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">51</span>: <span class="cit-fpage">704</span>–<span class="cit-lpage">715</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2010.03.0178</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D51%26rft.spage%253D704%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2010.03.0178%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2010.03.0178&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000287480900034&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3691.31" data-doi="10.2135/cropsci2008.08.0512"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heffner</span>,  <span class="cit-name-given-names">E. L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. E.</span> <span class="cit-name-surname">Sorrells</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.-L.</span> <span class="cit-name-surname">Jannink</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Genomic Selection for Crop Improvement.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">12</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2008.08.0512</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DHeffner%26rft.auinit1%253DE.%2BL.%26rft.volume%253D49%26rft.issue%253D1%26rft.spage%253D1%26rft.epage%253D12%26rft.atitle%253DGenomic%2BSelection%2Bfor%2BCrop%2BImprovement%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2008.08.0512%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2008.08.0512&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000263344300001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3691.32" data-doi="10.2135/cropsci2011.06.0297"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heslot</span>,  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.-P.</span> <span class="cit-name-surname">Yang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. E.</span> <span class="cit-name-surname">Sorrells</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.-L.</span> <span class="cit-name-surname">Jannink</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genomic Selection in Plant Breeding: A Comparison of Models.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">146</span>–<span class="cit-lpage">160</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2011.06.0297</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D52%26rft.spage%253D146%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2011.06.0297%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2011.06.0297&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3691.33" data-doi="10.1371/journal.pgen.1000008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hill</span>,  <span class="cit-name-given-names">W. G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. E.</span> <span class="cit-name-surname">Goddard</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. M.</span> <span class="cit-name-surname">Visscher</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Data and theory point to mainly additive genetic variance for complex traits.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e1000008</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1000008</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DHill%26rft.auinit1%253DW.%2BG.%26rft.volume%253D4%26rft.issue%253D2%26rft.spage%253De1000008%26rft.epage%253De1000008%26rft.atitle%253DData%2Band%2Btheory%2Bpoint%2Bto%2Bmainly%2Badditive%2Bgenetic%2Bvariance%2Bfor%2Bcomplex%2Btraits.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000008%26rft_id%253Dinfo%253Apmid%252F18454194%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18454194&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3691.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3691.34" data-doi="10.1105/tpc.113.119982"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hirsch</span>,  <span class="cit-name-given-names">C. N.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">Foerster</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">Johnson</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. S.</span> <span class="cit-name-surname">Sekhon</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Muttoni</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Insights into the maize pan-genome and pan-transcriptome.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">121</span>–<span class="cit-lpage">135</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1105/tpc.113.119982</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.113.119982%26rft_id%253Dinfo%253Apmid%252F24488960%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjg6IjI2LzEvMTIxIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMS8zNjkxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3691.35" data-doi="10.1016/j.pbi.2007.01.003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Holland</span>,  <span class="cit-name-given-names">J. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Genetic architecture of complex traits in plants.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Plant Biol.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">156</span>–<span class="cit-lpage">161</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.pbi.2007.01.003</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Bplant%2Bbiology%26rft.stitle%253DCurr%2BOpin%2BPlant%2BBiol%26rft.aulast%253DHolland%26rft.auinit1%253DJ.%2BB.%26rft.volume%253D10%26rft.issue%253D2%26rft.spage%253D156%26rft.epage%253D161%26rft.atitle%253DGenetic%2Barchitecture%2Bof%2Bcomplex%2Btraits%2Bin%2Bplants.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.pbi.2007.01.003%26rft_id%253Dinfo%253Apmid%252F17291822%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.pbi.2007.01.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17291822&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3691.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000245573100009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3691.36" data-doi="10.1016/j.tibtech.2013.06.003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jonas</span>,  <span class="cit-name-given-names">E.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.-J.</span> <span class="cit-name-surname">de Koning</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Does genomic selection have a future in plant breeding?</span> <abbr class="cit-jnl-abbrev">Trends Biotechnol.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">497</span>–<span class="cit-lpage">504</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.tibtech.2013.06.003</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BBiotechnol.%26rft.volume%253D31%26rft.spage%253D497%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tibtech.2013.06.003%26rft_id%253Dinfo%253Apmid%252F23870753%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tibtech.2013.06.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23870753&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3691.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3691.37"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kasnavi</span>,  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. A.</span> <span class="cit-name-surname">Afshar</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. M.</span> <span class="cit-name-surname">Shariati</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N. E. J.</span> <span class="cit-name-surname">Kashan</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Honarvar</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Performance evaluation of support vector machine (SVM)-based predictors in genomic selection.</span> <abbr class="cit-jnl-abbrev">Indian J. Anim. Sci.</abbr> <span class="cit-vol">87</span>: <span class="cit-fpage">1226</span>–<span class="cit-lpage">1231</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DIndian%2BJ.%2BAnim.%2BSci.%26rft.volume%253D87%26rft.spage%253D1226%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3691.38"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kingma</span>,  <span class="cit-name-given-names">D. P.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Ba</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> Adam: A Method for Stochastic Optimization. arXiv. <a href="https://arxiv.org/abs/1412.6980">https://arxiv.org/abs/1412.6980</a></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3691.39"><div class="cit-metadata"><cite>Klimasauskas, C. C., 1993 <span class="cit-article-title">Applying Neural Networks In: Neural Networks in Finance and Investing: Using Artificial Intelligence to Improve Real World Performance,</span> pp. <span class="cit-fpage">64</span>–<span class="cit-lpage">65</span>. <span class="cit-ed"> <span class="cit-name-given-names">R. R.</span> <span class="cit-name-surname">Trippi</span></span> and <span class="cit-ed"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Turban</span></span>, <span class="cit-comment">editors,</span> <span class="cit-publ-loc">Probus, Chicago</span>. ISBN: 1557384525.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3691.40"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kuhn</span>,  <span class="cit-name-given-names">M.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Johnson</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Over-Fitting and Model Tuning. Applied Predictive Modeling</span>. <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">New York, NY</span>. <span class="cit-fpage">61</span>–<span class="cit-lpage">92</span>. <a href="https://doi.org/10.1007/978-1-4614-6849-3_4">https://doi.org/10.1007/978-1-4614-6849-3_4</a></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3691.41" data-doi="10.1109/72.655043"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">LeBaron</span>,  <span class="cit-name-given-names">B.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. S.</span> <span class="cit-name-surname">Weigend</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">A bootstrap evaluation of the effect of data splitting on financial time series.</span> <abbr class="cit-jnl-abbrev">IEEE Trans. Neural Netw.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">213</span>–<span class="cit-lpage">220</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1109/72.655043</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DIEEE%2Btransactions%2Bon%2Bneural%2Bnetworks%2B%252F%2Ba%2Bpublication%2Bof%2Bthe%2BIEEE%2BNeural%2BNetworks%2BCouncil%26rft.stitle%253DIEEE%2BTrans%2BNeural%2BNetw%26rft.aulast%253DLebaron%26rft.auinit1%253DB.%26rft.volume%253D9%26rft.issue%253D1%26rft.spage%253D213%26rft.epage%253D220%26rft.atitle%253DA%2Bbootstrap%2Bevaluation%2Bof%2Bthe%2Beffect%2Bof%2Bdata%2Bsplitting%2Bon%2Bfinancial%2Btime%2Bseries.%26rft_id%253Dinfo%253Adoi%252F10.1109%252F72.655043%26rft_id%253Dinfo%253Apmid%252F18252443%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1109/72.655043&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18252443&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3691.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3691.42" data-doi="10.1371/journal.pone.0112227"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lipka</span>,  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Lu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. H.</span> <span class="cit-name-surname">Cherney</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. S.</span> <span class="cit-name-surname">Buckler</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. D.</span> <span class="cit-name-surname">Casler</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Accelerating the Switchgrass (Panicum virgatum L.) Breeding Cycle Using Genomic Selection Approaches.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e112227</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0112227</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De112227%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0112227%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0112227&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3691.43" data-doi="10.1007/s00122-011-1648-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Long</span>,  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Gianola</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G. J. M.</span> <span class="cit-name-surname">Rosa</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. A.</span> <span class="cit-name-surname">Weigel</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Application of support vector regression to genome-assisted prediction of quantitative traits.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">123</span>: <span class="cit-fpage">1065</span>–<span class="cit-lpage">1074</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-011-1648-y</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DLong%26rft.auinit1%253DN.%26rft.volume%253D123%26rft.issue%253D7%26rft.spage%253D1065%26rft.epage%253D1074%26rft.atitle%253DApplication%2Bof%2Bsupport%2Bvector%2Bregression%2Bto%2Bgenome-assisted%2Bprediction%2Bof%2Bquantitative%2Btraits.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-011-1648-y%26rft_id%253Dinfo%253Apmid%252F21739137%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-011-1648-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21739137&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3691.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-book" id="cit-9.11.3691.44" data-doi="10.1016/B978-0-12-385531-2.00002-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lorenz</span>,  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Chao</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F. G.</span> <span class="cit-name-surname">Asoro</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. L.</span> <span class="cit-name-surname">Heffner</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Hayashi</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-source">Genomic Selection in Plant Breeding: Knowledge and Prospects</span>, chap 2. <span class="cit-publ-name">Elsevier</span>, <span class="cit-publ-loc">Amsterdam, Netherlands</span><span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/B978-0-12-385531-2.00002-5</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenomic%2BSelection%2Bin%2BPlant%2BBreeding%253A%2BKnowledge%2Band%2BProspects%26rft.volume%253D110%26rft.spage%253D77%26rft_id%253Dinfo%253Adoi%252F10.1016%252FB978-0-12-385531-2.00002-5%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/B978-0-12-385531-2.00002-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3691.45" data-doi="10.1007/s00425-018-2976-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>,  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Qiu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Song</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Li</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Q.</span> <span class="cit-name-surname">Cheng</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">A deep convolutional neural network approach for predicting phenotypes from genotypes.</span> <abbr class="cit-jnl-abbrev">Planta</abbr> <span class="cit-vol">248</span>: <span class="cit-fpage">1307</span>–<span class="cit-lpage">1318</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00425-018-2976-9</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlanta%26rft.volume%253D248%26rft.spage%253D1307%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00425-018-2976-9%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00425-018-2976-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3691.46" data-doi="10.1186/1297-9686-41-35"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>,  <span class="cit-name-given-names">T. H. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Accuracy of breeding values of’unrelated’individuals predicted by dense SNP genotyping.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">35</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1297-9686-41-35</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DMeuwissen%26rft.auinit1%253DT.%2BH.%26rft.volume%253D41%26rft.spage%253D35%26rft.epage%253D35%26rft.atitle%253DAccuracy%2Bof%2Bbreeding%2Bvalues%2Bof%2B%2527unrelated%2527%2Bindividuals%2Bpredicted%2Bby%2Bdense%2BSNP%2Bgenotyping.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-41-35%26rft_id%253Dinfo%253Apmid%252F19519896%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-41-35&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19519896&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3691.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3691.47" data-doi="10.1093/nar/gkv1344"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>,  <span class="cit-name-given-names">T. H..</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. J.</span> <span class="cit-name-surname">Hayes</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. E.</span> <span class="cit-name-surname">Goddard</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Prediction of Total Genetic Value Using Genome-Wide Dense Marker Maps.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">157</span>: <span class="cit-fpage">1819</span>–<span class="cit-lpage">1829</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMeuwissen%26rft.auinit1%253DT.%2BH.%2BE.%26rft.volume%253D157%26rft.issue%253D4%26rft.spage%253D1819%26rft.epage%253D1829%26rft.atitle%253DPrediction%2Bof%2BTotal%2BGenetic%2BValue%2BUsing%2BGenome-Wide%2BDense%2BMarker%2BMaps%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkv1344%26rft_id%253Dinfo%253Apmid%252F11290733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1Ny80LzE4MTkiO3M6NDoiYXRvbSI7czoxOToiL2dnZy85LzExLzM2OTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3691.48" data-doi="10.3389/fpls.2018.00627"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Monir</span>,  <span class="cit-name-given-names">M. M.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Zhu</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Dominance and Epistasis Interactions Revealed as Important Variants for Leaf Traits of Maize NAM Population.</span> <abbr class="cit-jnl-abbrev">Front. Plant Sci.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">627</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fpls.2018.00627</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BPlant%2BSci.%26rft.volume%253D9%26rft.spage%253D627%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffpls.2018.00627%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fpls.2018.00627&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3691.49" data-doi="10.1186/1297-9686-41-56"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Moser</span>,  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Tier</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. E.</span> <span class="cit-name-surname">Crump</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. S.</span> <span class="cit-name-surname">Khatkar</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">H. W.</span> <span class="cit-name-surname">Raadsma</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A comparison of five methods to predict genomic breeding values of dairy bulls from genome-wide SNP markers.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">56</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1297-9686-41-56</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DMoser%26rft.auinit1%253DG.%26rft.volume%253D41%26rft.spage%253D56%26rft.epage%253D56%26rft.atitle%253DA%2Bcomparison%2Bof%2Bfive%2Bmethods%2Bto%2Bpredict%2Bgenomic%2Bbreeding%2Bvalues%2Bof%2Bdairy%2Bbulls%2Bfrom%2Bgenome-wide%2BSNP%2Bmarkers.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-41-56%26rft_id%253Dinfo%253Apmid%252F20043835%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-41-56&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20043835&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3691.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3691.50"><div class="cit-metadata"><cite>Neves, H. H., R. Carvalheiro, and S. A. Queiroz, 2012 A comparison of statistical methods for genomic selection in a mice population. <span class="cit-vol">13</span>: <span class="cit-fpage">100</span>. <a href="https://doi.org/10.1186/1471-2156-13-100">https://doi.org/10.1186/1471-2156-13-100</a></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3691.51" data-doi="10.1534/g3.118.200311"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Norman</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Taylor</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Edwards</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Kuchel</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Optimising Genomic Selection in Wheat: Effect of Marker Density, Population Size and Population Structure on Prediction Accuracy.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">2889</span>–<span class="cit-lpage">2899</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.118.200311</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.118.200311%26rft_id%253Dinfo%253Apmid%252F29970398%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjg6IjgvOS8yODg5IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMS8zNjkxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3691.52" data-doi="10.1017/S0016672310000662"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Okut</span>,  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Gianola</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G. J. M.</span> <span class="cit-name-surname">Rosa</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. A.</span> <span class="cit-name-surname">Weigel</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Prediction of body mass index in mice using dense molecular markers and a regularized neural network.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">93</span>: <span class="cit-fpage">189</span>–<span class="cit-lpage">201</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1017/S0016672310000662</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BRes.%26rft.volume%253D93%26rft.spage%253D189%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672310000662%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672310000662&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3691.53"><div class="cit-metadata"><cite>Parker, D. B., 1987 Optimal algorithms for adaptive networks: Second order backpropagation, second order direct backpropagation, and second order hebbing learning. Proceedings of the IEEE First International Conference on Neural Networks. <span class="cit-vol">2</span>: <span class="cit-fpage">593</span>–<span class="cit-lpage">600</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3691.54" data-doi="10.1016/j.cpc.2010.04.018"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pedregosa</span>,  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Varoquaux</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Gramfort</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Michel</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Thirion</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Scikit-learn: Machine Learning in Python.</span> <abbr class="cit-jnl-abbrev">J. Mach. Learn. Res.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">2825</span>–<span class="cit-lpage">2830</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BMach.%2BLearn.%2BRes.%26rft.volume%253D12%26rft.spage%253D2825%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cpc.2010.04.018%26rft_id%253Dinfo%253Apmid%252F23755062%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cpc.2010.04.018&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23755062&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3691.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3691.55" data-doi="10.1534/genetics.114.164442"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pérez</span>,  <span class="cit-name-given-names">P.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">de los Campos</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genome-wide regression and prediction with the BGLR statistical package.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">198</span>: <span class="cit-fpage">483</span>–<span class="cit-lpage">495</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.114.164442</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.164442%26rft_id%253Dinfo%253Apmid%252F25009151%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk4LzIvNDgzIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMS8zNjkxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3691.56"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pouladi</span>,  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Salehinejad</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. M.</span> <span class="cit-name-surname">Gilani</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> Deep Recurrent Neural Networks for Sequential Phenotype Prediction in Genomics. arXiv:1511.02554</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-book" id="cit-9.11.3691.57"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Prechelt</span>,  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Early Stopping - But When?</span> pp. <span class="cit-fpage">55</span>–<span class="cit-lpage">69</span> in <span class="cit-source">Neural Networks: Tricks of the Trade</span>, edited by <span class="cit-ed"> <span class="cit-name-given-names">G. B.</span> <span class="cit-name-surname">Orr</span></span> and <span class="cit-ed"> <span class="cit-name-given-names">K.-R.</span> <span class="cit-name-surname">Müller</span></span>. <span class="cit-publ-name">Springer Berlin Heidelberg</span>, <span class="cit-publ-loc">Berlin, Heidelberg</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3691.58" data-doi="10.1534/g3.115.024950"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ramstein</span>,  <span class="cit-name-given-names">G. P.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Evans</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. M.</span> <span class="cit-name-surname">Kaeppler</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. B.</span> <span class="cit-name-surname">Mitchell</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. P.</span> <span class="cit-name-surname">Vogel</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Accuracy of Genomic Prediction in Switchgrass (Panicum virgatum L.) Improved by Accounting for Linkage Disequilibrium.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">1049</span>–<span class="cit-lpage">1062</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.115.024950</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.115.024950%26rft_id%253Dinfo%253Apmid%252F26869619%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjg6IjYvNC8xMDQ5IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMS8zNjkxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3691.59" data-doi="10.1093/jxb/erl214"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ribaut</span>,  <span class="cit-name-given-names">J.-M.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Ragot</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Marker-assisted selection to improve drought adaptation in maize: the backcross approach, perspectives, limitations, and alternatives.</span> <abbr class="cit-jnl-abbrev">J. Exp. Bot.</abbr> <span class="cit-vol">58</span>: <span class="cit-fpage">351</span>–<span class="cit-lpage">360</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/jxb/erl214</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BExp.%2BBot.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjxb%252Ferl214%26rft_id%253Dinfo%253Apmid%252F17158111%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/jxb/erl214&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17158111&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3691.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000243992800024&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3691.60"><div class="cit-metadata"><cite>Riedelsheimer, C., F. Technow, and A. E. Melchinger, 2013 Comparison of whole-genome prediction models for traits with contrasting genetic architecture in a diversity panel of maize inbred lines. <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">452</span>. <a href="https://doi.org/10.1186/1471-2164-13-452">https://doi.org/10.1186/1471-2164-13-452</a></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3691.61" data-doi="10.1016/j.fgb.2010.06.003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Roorkiwal</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Rathore</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. R.</span> <span class="cit-name-surname">Das</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. K.</span> <span class="cit-name-surname">Singh</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Jain</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genome-Enabled Prediction Models for Yield Related Traits in Chickpea.</span> <abbr class="cit-jnl-abbrev">Front. Plant Sci.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">1666</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fpls.2016.01666</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BPlant%2BSci.%26rft.volume%253D7%26rft.spage%253D1666%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.fgb.2010.06.003%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.fgb.2010.06.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3691.62"><div class="cit-metadata"><cite>Rumelhart, D. E., G. E. Hinton, and R. J. Williams, 1986 Learning internal representation by error propagation, Parallel Distributed Processing, DE Rumelhart and JL McClelland, eds. ISBN: 0-262-68053-X. <span class="cit-fpage">318</span>–<span class="cit-lpage">326</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3691.63" data-doi="10.1016/j.artmed.2016.12.003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shaikhina</span>,  <span class="cit-name-given-names">T.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">N. A.</span> <span class="cit-name-surname">Khovanova</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Handling limited datasets with neural networks in medical applications: A small-data approach.</span> <abbr class="cit-jnl-abbrev">Artif. Intell. Med.</abbr> <span class="cit-vol">75</span>: <span class="cit-fpage">51</span>–<span class="cit-lpage">63</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.artmed.2016.12.003</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DArtif.%2BIntell.%2BMed.%26rft.volume%253D75%26rft.spage%253D51%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.artmed.2016.12.003%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.artmed.2016.12.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3691.64" data-doi="10.1371/journal.pgen.1004982"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Spindel</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Begum</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Akdemir</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Virk</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Collard</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genomic Selection and Association Mapping in Rice (Oryza sativa): Effect of Trait Genetic Architecture, Training Population Composition, Marker Number and Statistical Model on Accuracy of Rice Genomic Selection in Elite, Tropical Rice Breeding Lines.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">e1004982</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1004982</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D11%26rft.spage%253De1004982%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1004982%26rft_id%253Dinfo%253Apmid%252F25689273%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1004982&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25689273&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3691.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3691.65" data-doi="10.1017/S0016672309990334"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Usai</span>,  <span class="cit-name-given-names">M. G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. E.</span> <span class="cit-name-surname">Goddard</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. J.</span> <span class="cit-name-surname">Hayes</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">LASSO with cross-validation for genomic selection.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">427</span>–<span class="cit-lpage">436</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1017/S0016672309990334</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DUsai%26rft.auinit1%253DM.%2BG.%26rft.volume%253D91%26rft.issue%253D6%26rft.spage%253D427%26rft.epage%253D436%26rft.atitle%253DLASSO%2Bwith%2Bcross-validation%2Bfor%2Bgenomic%2Bselection.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672309990334%26rft_id%253Dinfo%253Apmid%252F20122298%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672309990334&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20122298&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3691.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3691.66" data-doi="10.3168/jds.2010-3335"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vazquez</span>,  <span class="cit-name-given-names">A. I.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G. J. M.</span> <span class="cit-name-surname">Rosa</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. A.</span> <span class="cit-name-surname">Weigel</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">de los Campos</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Gianola</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Predictive ability of subsets of single nucleotide polymorphisms with and without parent average in US Holsteins.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">93</span>: <span class="cit-fpage">5942</span>–<span class="cit-lpage">5949</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3168/jds.2010-3335</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DVazquez%26rft.auinit1%253DA.%2BI.%26rft.volume%253D93%26rft.issue%253D12%26rft.spage%253D5942%26rft.epage%253D5949%26rft.atitle%253DPredictive%2Bability%2Bof%2Bsubsets%2Bof%2Bsingle%2Bnucleotide%2Bpolymorphisms%2Bwith%2Band%2Bwithout%2Bparent%2Baverage%2Bin%2BUS%2BHolsteins.%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2010-3335%26rft_id%253Dinfo%253Apmid%252F21094768%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2010-3335&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21094768&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3691.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3691.67" data-doi="10.1038/d41586-018-02174-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Webb</span>,  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Deep learning for biology.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">554</span>: <span class="cit-fpage">555</span>–<span class="cit-lpage">557</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/d41586-018-02174-z</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D554%26rft.spage%253D555%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fd41586-018-02174-z%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/d41586-018-02174-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3691.68" data-doi="10.1534/g3.116.032268"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xavier</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W. M.</span> <span class="cit-name-surname">Muir</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. M.</span> <span class="cit-name-surname">Rainey</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Assessing Predictive Properties of Genome-Wide Selection in Soybeans.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">2611</span>–<span class="cit-lpage">2616</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.116.032268</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.116.032268%26rft_id%253Dinfo%253Apmid%252F27317786%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjg6IjYvOC8yNjExIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMS8zNjkxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3691.69" data-doi="10.1186/s12284-018-0223-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>,  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Wang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Ding</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Zheng</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Yang</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Genomic selection of agronomic traits in hybrid rice using an NCII population.</span> <abbr class="cit-jnl-abbrev">Rice (N. Y.)</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">32</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12284-018-0223-4</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRice%2B%2528N.%2BY.%2529%26rft.volume%253D11%26rft.spage%253D32%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12284-018-0223-4%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12284-018-0223-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3691.70"><div class="cit-metadata"><cite>Zou, H., and T. Hastie, 2005 Regularization and variable selection via the elastic net. J. R. Stat. Soc. Ser. <span class="cit-vol">67</span>: <span class="cit-fpage">301</span>–<span class="cit-lpage">320</span>. <a href="http://www.jstor.org/stable/3647580">http://www.jstor.org/stable/3647580</a></cite></div><div class="cit-extra"></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/9/11/3691.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/11/3683" title="Elucidating the Molecular Determinants of Aβ Aggregation with Deep Mutational Scanning" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/11/3703" title="Activity-Dependent Regulation of the Proapoptotic BH3-Only Gene egl-1 in a Living Neuron Pair in Caenorhabditis elegans" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/9/11"><span>Volume 9 Issue 11, November 2019</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/9/11.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/9/11.cover-source.jpg?itok=DjqkLGk9" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 9 (11)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/11/3691.abstract" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-tCA8tXvH1S0DiJ_kpdsRMRY9pDOIjVRcToAasBzSW8g" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/genomic-prediction" class="" data-icon-position="" data-hide-link-title="0">Genomic Prediction</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/9/11/3691" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/9/11/3691" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/9/11/3691" class="active" data-icon-position="" data-hide-link-title="0">Benchmarking Parametric and Machine Learning Models for Genomic Prediction of Complex Traits</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/494609" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-nrIYmgpaNF8wisytU1vLHw46Zx3g0M7NPNa_s5oM5y0" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/11/3691.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/9/11/3691" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-Y7vFc_h0YuhxDDXd-OwBgGtaDnZ0Gf4DkOVpYHj_ISU" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/9/11/3691" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/11/3691.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494609" id="node494609--2" data-pisa="ggg;9/11/3691" data-pisa-master="ggg;g3.119.400498" data-apath="/ggg/9/11/3691.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Benchmarking Parametric and Machine Learning Models for Genomic Prediction of Complex Traits</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-6097-606X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Christina B. Azodi</span>, <span class="highwire-citation-author" data-delta="1">Emily Bolger</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-7297-0984" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Andrew McCarren</span>, <span class="highwire-citation-author" data-delta="3">Mark Roantree</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0001-5692-7129" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Gustavo de los Campos</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0001-6470-235X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Shin-Han Shiu</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3691-3702;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400498 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494609--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Christina B. Azodi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Biology</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Christina%2BB.%2BAzodi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Azodi%20CB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AChristina%2BB.%2BAzodi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-6097-606X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Christina B. Azodi</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Emily Bolger </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Mathematics, Moravian College, Bethlehem, PA</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Emily%2BBolger%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bolger%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEmily%2BBolger%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Andrew McCarren </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Insight Centre for Data Analytics, School of Computing, Dublin City University, Dublin 9, Ireland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andrew%2BMcCarren%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=McCarren%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAndrew%2BMcCarren%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-7297-0984" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Andrew McCarren</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Mark Roantree </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Insight Centre for Data Analytics, School of Computing, Dublin City University, Dublin 9, Ireland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mark%2BRoantree%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Roantree%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMark%2BRoantree%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Gustavo de los Campos </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Epidemiology & Biostatistics</div><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Statistics & Probability</div><div class='author-affiliation'><span class='nlm-sup'>††</span>Institute for Quantitative Health Science and Engineering, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gustavo%2Bde%2Blos%2BCampos%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=de%20los%20Campos%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGustavo%2Bde%2Blos%2BCampos%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-5692-7129" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Gustavo de los Campos</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:gustavoc@msu.edu" class="" data-icon-position="" data-hide-link-title="0">gustavoc@msu.edu</a>
<a href="mailto:shius@msu.edu" class="" data-icon-position="" data-hide-link-title="0">shius@msu.edu</a></span></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Shin-Han Shiu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Biology</div><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>Department of Computational, Mathematics, Science, and Engineering, Michigan State University, East Lansing, MI, 48824</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Shin-Han%2BShiu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Shiu%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AShin-Han%2BShiu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-6470-235X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Shin-Han Shiu</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:gustavoc@msu.edu" class="" data-icon-position="" data-hide-link-title="0">gustavoc@msu.edu</a>
<a href="mailto:shius@msu.edu" class="" data-icon-position="" data-hide-link-title="0">shius@msu.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/9/11/3691" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/9/11/3691&amp;title=Benchmarking%20Parametric%20and%20Machine%20Learning%20Models%20for%20Genomic%20Prediction%20of%20Complex%20Traits" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/9/11/3691&amp;title=Benchmarking%20Parametric%20and%20Machine%20Learning%20Models%20for%20Genomic%20Prediction%20of%20Complex%20Traits" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/9/11/3691&amp;title=Benchmarking%20Parametric%20and%20Machine%20Learning%20Models%20for%20Genomic%20Prediction%20of%20Complex%20Traits" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/9/11/3691&amp;text=Benchmarking%20Parametric%20and%20Machine%20Learning%20Models%20for%20Genomic%20Prediction%20of%20Complex%20Traits" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/9/11/3691&amp;title=Benchmarking%20Parametric%20and%20Machine%20Learning%20Models%20for%20Genomic%20Prediction%20of%20Complex%20Traits" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/9/11/3691&amp;t=Benchmarking%20Parametric%20and%20Machine%20Learning%20Models%20for%20Genomic%20Prediction%20of%20Complex%20Traits" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/9/11/3691&amp;title=Benchmarking%20Parametric%20and%20Machine%20Learning%20Models%20for%20Genomic%20Prediction%20of%20Complex%20Traits" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/9/11/3691&amp;title=Benchmarking%20Parametric%20and%20Machine%20Learning%20Models%20for%20Genomic%20Prediction%20of%20Complex%20Traits" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/494609/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494609" id="node494609--3" data-pisa="ggg;9/11/3691" data-pisa-master="ggg;g3.119.400498" data-apath="/ggg/9/11/3691.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Benchmarking Parametric and Machine Learning Models for Genomic Prediction of Complex Traits</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-6097-606X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Christina B. Azodi</span>, <span class="highwire-citation-author" data-delta="1">Emily Bolger</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-7297-0984" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Andrew McCarren</span>, <span class="highwire-citation-author" data-delta="3">Mark Roantree</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0001-5692-7129" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Gustavo de los Campos</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0001-6470-235X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Shin-Han Shiu</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3691-3702;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400498 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494609--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Christina B. Azodi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Biology</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Christina%2BB.%2BAzodi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Azodi%20CB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AChristina%2BB.%2BAzodi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-6097-606X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Christina B. Azodi</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Emily Bolger </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Mathematics, Moravian College, Bethlehem, PA</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Emily%2BBolger%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bolger%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEmily%2BBolger%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Andrew McCarren </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Insight Centre for Data Analytics, School of Computing, Dublin City University, Dublin 9, Ireland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andrew%2BMcCarren%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=McCarren%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAndrew%2BMcCarren%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-7297-0984" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Andrew McCarren</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Mark Roantree </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Insight Centre for Data Analytics, School of Computing, Dublin City University, Dublin 9, Ireland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mark%2BRoantree%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Roantree%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMark%2BRoantree%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Gustavo de los Campos </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Epidemiology & Biostatistics</div><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Statistics & Probability</div><div class='author-affiliation'><span class='nlm-sup'>††</span>Institute for Quantitative Health Science and Engineering, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gustavo%2Bde%2Blos%2BCampos%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=de%20los%20Campos%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGustavo%2Bde%2Blos%2BCampos%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-5692-7129" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Gustavo de los Campos</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:gustavoc@msu.edu" class="" data-icon-position="" data-hide-link-title="0">gustavoc@msu.edu</a>
<a href="mailto:shius@msu.edu" class="" data-icon-position="" data-hide-link-title="0">shius@msu.edu</a></span></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Shin-Han Shiu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Biology</div><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>Department of Computational, Mathematics, Science, and Engineering, Michigan State University, East Lansing, MI, 48824</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Shin-Han%2BShiu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Shiu%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AShin-Han%2BShiu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-6470-235X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Shin-Han Shiu</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:gustavoc@msu.edu" class="" data-icon-position="" data-hide-link-title="0">gustavoc@msu.edu</a>
<a href="mailto:shius@msu.edu" class="" data-icon-position="" data-hide-link-title="0">shius@msu.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=3691&amp;rft.epage=3702&amp;rft.atitle=Benchmarking%20Parametric%20and%20Machine%20Learning%20Models%20for%20Genomic%20Prediction%20of%20Complex%20Traits&amp;rft.volume=9&amp;rft.issue=11&amp;rft.date=2019-11-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Azodi%2C+Christina+B.&amp;rft.au=Bolger%2C+Emily&amp;rft.au=McCarren%2C+Andrew&amp;rft.au=Roantree%2C+Mark&amp;rft.au=de+los+Campos%2C+Gustavo&amp;rft.au=Shiu%2C+Shin-Han"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/494609/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/494609/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/494609/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/494609/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/494609/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/494609/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/494609/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/494609/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/494609/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/494609/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/494609/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/494609/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495864" id="node495864" data-pisa="ggg;10/3/1113" data-pisa-master="ggg;g3.119.400968" data-apath="/ggg/10/3/1113.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1113" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Multi-trait Genomic Prediction Model Increased the Predictive Ability for Agronomic and Malting Quality Traits in Barley (<em>Hordeum vulgare</em> L.)</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495543" id="node495543" data-pisa="ggg;10/2/769" data-pisa-master="ggg;g3.119.400759" data-apath="/ggg/10/2/769.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/2/769" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Novel Bayesian Networks for Genomic Prediction of Developmental Traits in Biomass Sorghum</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495557" id="node495557" data-pisa="ggg;10/2/539" data-pisa-master="ggg;g3.119.400866" data-apath="/ggg/10/2/539.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/2/539" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Optimized Genetic Testing for Polledness in Multiple Breeds of Cattle</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Genomic%20Prediction" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Genomic Prediction</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/9/11/3691" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-9" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-15" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/9/11/3691.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/9/11/3691.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/9/11/3691.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__c-64NG7CB_fwZbyex-B49GkX1AvVESfu7M9yxTNDG_o__Uvi1liJMEKbHtHHv_1fo_FCLvimvbq9jMwPcCPR364Q__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
