<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/9/11/3547" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/9/11/3547.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/9/11/3547.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/9/11/3547.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/9/11/3547/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/9/11/3547.atom" />
<meta name="HW.pisa" content="ggg;9/11/3547" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Phased Diploid Genome Assemblies for Three Strains of Candida albicans from Oak Trees" />
<meta name="DC.Identifier" content="10.1534/g3.119.400486" />
<meta name="DC.Date" content="2019-11-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2019 Hamlin et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Although normally a harmless commensal, Candida albicans , it is also one of the most common causes of bloodstream infections in the U.S. Candida albicans has long been considered an obligate commensal, however, recent studies suggest it can live outside animal hosts. Here, we have generated PacBio sequences and phased genome assemblies for three C. albicans strains from oak trees (NCYC 4144, NCYC 4145, and NCYC 4146). PacBio datasets are high depth (over 400 fold coverage) and more than half of the sequencing data are contained in reads longer than 15 kb. Primary assemblies showed high contiguity with several chromosomes for each strain recovered as single contigs, and greater than half of the alternative haplotype sequence was assembled in haplotigs at least 174 kb long. Using these assemblies we were able to identify structural polymorphisms, including a polymorphic inversion over 100 kb in length. These results show that phased de novo diploid assemblies for C. albicans can enable the study of genomic variation within and among strains of an important fungal pathogen." />
<meta name="DC.Contributor" content="Jennafer A. P. Hamlin" />
<meta name="DC.Contributor" content="Guilherme B. Dias" />
<meta name="DC.Contributor" content="Casey M. Bergman" />
<meta name="DC.Contributor" content="Douda Bensasson" />
<meta name="article:published_time" content="2019-11-01" />
<meta name="article:section" content="Genome Report" />
<meta name="citation_title" content="Phased Diploid Genome Assemblies for Three Strains of Candida albicans from Oak Trees" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Although normally a harmless commensal, &lt;i&gt;Candida albicans&lt;/i&gt;, it is also one of the most common causes of bloodstream infections in the U.S. &lt;i&gt;Candida albicans&lt;/i&gt; has long been considered an obligate commensal, however, recent studies suggest it can live outside animal hosts. Here, we have generated PacBio sequences and phased genome assemblies for three &lt;i&gt;C. albicans&lt;/i&gt; strains from oak trees (NCYC 4144, NCYC 4145, and NCYC 4146). PacBio datasets are high depth (over 400 fold coverage) and more than half of the sequencing data are contained in reads longer than 15 kb. Primary assemblies showed high contiguity with several chromosomes for each strain recovered as single contigs, and greater than half of the alternative haplotype sequence was assembled in haplotigs at least 174 kb long. Using these assemblies we were able to identify structural polymorphisms, including a polymorphic inversion over 100 kb in length. These results show that phased &lt;i&gt;de novo&lt;/i&gt; diploid assemblies for &lt;i&gt;C. albicans&lt;/i&gt; can enable the study of genomic variation within and among strains of an important fungal pathogen.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2019/11/01" />
<meta name="citation_mjid" content="ggg;9/11/3547" />
<meta name="citation_id" content="9/11/3547" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/9/11/3547" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/9/11/3547.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/9/11/3547.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/9/11/3547.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.119.400486" />
<meta name="citation_pmid" content="31540974" />
<meta name="citation_volume" content="9" />
<meta name="citation_issue" content="11" />
<meta name="citation_num_pages" content="8" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Genome Report" />
<meta name="citation_firstpage" content="3547" />
<meta name="citation_lastpage" content="3554" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Jennafer A. P. Hamlin" />
<meta name="citation_author_institution" content="Department of Plant Biology and Institute of Bioinformatics, University of Georgia, Athens, GA, and" />
<meta name="citation_author" content="Guilherme B. Dias" />
<meta name="citation_author_institution" content="Department of Genetics and Institute of Bioinformatics, University of Georgia, Athens, GA, United States of America" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-1459-3148" />
<meta name="citation_author" content="Casey M. Bergman" />
<meta name="citation_author_institution" content="Department of Genetics and Institute of Bioinformatics, University of Georgia, Athens, GA, United States of America" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-5462-9854" />
<meta name="citation_author" content="Douda Bensasson" />
<meta name="citation_author_institution" content="Department of Plant Biology and Institute of Bioinformatics, University of Georgia, Athens, GA, and" />
<meta name="citation_author_email" content="dbensasson@uga.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-9015-2707" />
<meta name="citation_reference" content="citation_journal_title=bioRxiv;citation_author=M. Alonge;citation_author=S. Soyk;citation_author=S. Ramakrishnan;citation_author=X. Wang;citation_author=S. Goodwin;citation_title=Fast and accurate reference-guided scaffolding of draft genomes.;citation_year=2019;citation_doi=10.1101/519637" />
<meta name="citation_reference" content="citation_journal_title=Yeast (Chichester, England);citation_journal_abbrev=Yeast;citation_author=J. A. Barnett;citation_title=A history of research on yeasts 12: medical yeasts part 1, Candida albicans.;citation_pages=385-417;citation_volume=25;citation_year=2008;citation_issue=6;citation_pmid=18509848;citation_doi=10.1002/yea.1595" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=D. Bensasson;citation_author=J. Dicks;citation_author=J. M. Ludwig;citation_author=C. J. Bond;citation_author=A. Elliston;citation_title=Diverse Lineages of Candida albicans Live on Old Oaks.;citation_volume=211;citation_year=2019;citation_pmid=30463870;citation_doi=10.1534/genetics.118.301482" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=G. Benson;citation_title=Tandem repeats finder: a program to analyze dna sequences.;citation_volume=27;citation_year=1999;citation_pmid=9862982;citation_doi=10.1093/nar/27.2.573" />
<meta name="citation_reference" content="citation_journal_title=Trends in genetics :  TIG;citation_journal_abbrev=Trends Genet;citation_author=J. Berman;citation_author=L. Hadany;citation_title=Does stress induce (para)sex? Implications for Candida albicans evolution.;citation_pages=197-203;citation_volume=28;citation_year=2012;citation_issue=5;citation_pmid=22364928;citation_doi=10.1016/j.tig.2012.01.004" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=C. W. Birky;citation_title=Heterozygosity, Heteromorphy, and Phylogenetic Trees in Asexual Eukaryotes;citation_pages=427-437;citation_volume=144;citation_year=1996;citation_issue=1;citation_pmid=8878706" />
<meta name="citation_reference" content="citation_journal_title=Fungal genetics and biology :  FG &amp; B;citation_journal_abbrev=Fungal Genet Biol;citation_author=M.-E. Bougnoux;citation_author=C. Pujol;citation_author=D. Diogo;citation_author=C. Bouchier;citation_author=D. R. Soll;citation_title=Mating is rare within as well as between clades of the human pathogen Candida albicans.;citation_pages=221-231;citation_volume=45;citation_year=2008;citation_issue=3;citation_pmid=18063395;citation_doi=10.1016/j.fgb.2007.10.008" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=B. R. Braun;citation_author=M. van Het Hoog;citation_author=C. d’Enfert;citation_author=M. Martchenko;citation_author=J. Dungan;citation_title=A human-curated annotation of the Candida albicans genome.;citation_pages=36-57;citation_volume=1;citation_year=2005;citation_issue=1;citation_pmid=16103911;citation_doi=10.1371/journal.pgen.0010036" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=G. Butler;citation_author=M. D. Rasmussen;citation_author=M. F. Lin;citation_author=M. A. S. Santos;citation_author=S. Sakthikumar;citation_title=Evolution of pathogenicity and sexual reproduction in eight Candida genomes.;citation_pages=657-662;citation_volume=459;citation_year=2009;citation_issue=7247;citation_pmid=19465905;citation_doi=10.1038/nature08064" />
<meta name="citation_reference" content="citation_journal_title=Nat. Methods;citation_author=C.-S. Chin;citation_author=P. Peluso;citation_author=F. J. Sedlazeck;citation_author=M. Nattestad;citation_author=G. T. Concepcion;citation_title=Phased diploid genome assembly with single-molecule real-time sequencing.;citation_pages=1050-1054;citation_volume=13;citation_year=2016;citation_pmid=27749838;citation_doi=10.1038/nmeth.4035" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=J. Eid;citation_author=A. Fehr;citation_author=J. Gray;citation_author=K. Luong;citation_author=J. Lyle;citation_title=Real-time dna sequencing from single polymerase molecules.;citation_volume=323;citation_year=2009;citation_pmid=19023044;citation_doi=10.1126/science.1162986" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=B. Gel;citation_author=E. Serra;citation_title=karyoploter: an r/bioconductor package to plot customizable genomes displaying arbitrary data.;citation_pages=3088-3090;citation_volume=33;citation_year=2017;citation_pmid=28575171;citation_doi=10.1093/bioinformatics/btx346" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. Gurevich;citation_author=V. Saveliev;citation_author=N. Vyahhi;citation_author=G. Tesler;citation_title=Quast: quality assessment tool for genome assemblies.;citation_volume=29;citation_year=2013;citation_pmid=23422339;citation_doi=10.1093/bioinformatics/btt086" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=M. A. Hickman;citation_author=G. Zeng;citation_author=A. Forche;citation_author=M. P. Hirakawa;citation_author=D. Abbey;citation_title=The ‘obligate diploid’ Candida albicans forms mating-competent haploids.;citation_pages=55-59;citation_volume=494;citation_year=2013;citation_pmid=23364695;citation_doi=10.1038/nature11865" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=M. P. Hirakawa;citation_author=D. A. Martinez;citation_author=S. Sakthikumar;citation_author=M. Z. Anderson;citation_author=A. Berlin;citation_title=Genetic and phenotypic intra-species variation in Candida albicans.;citation_volume=25;citation_year=2015;citation_pmid=25504520;citation_doi=10.1101/gr.174623.114" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=T. Jones;citation_author=N. A. Federspiel;citation_author=H. Chibana;citation_author=J. Dungan;citation_author=S. Kalman;citation_title=The diploid genome sequence of Candida albicans.;citation_volume=101;citation_year=2004;citation_pmid=15123810;citation_doi=10.1073/pnas.0401648101" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=P. A. Kitts;citation_author=D. M. Church;citation_author=F. Thibaud-Nissen;citation_author=J. Choi;citation_author=V. Hem;citation_title=Assembly: a resource for assembled genomes at NCBI.;citation_volume=44;citation_year=2016;citation_pmid=26578580;citation_doi=10.1093/nar/gkv1226" />
<meta name="citation_reference" content="citation_journal_title=Gigascience;citation_author=J. Korlach;citation_author=G. Gedman;citation_author=S. B. Kingan;citation_author=C.-S. Chin;citation_author=J. T. Howard;citation_title=De novo pacbio long-read and phased avian genome assemblies correct and add to reference genes generated with intermediate and short reads.;citation_pages=gix085;citation_volume=6;citation_year=2017;citation_doi=10.1093/gigascience/gix085" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=M. Legrand;citation_author=A. Forche;citation_author=A. Selmecki;citation_author=C. Chan;citation_author=D. T. Kirkpatrick;citation_title=Haplotype mapping of a diploid non-meiotic organism using existing and induced aneuploidies.;citation_pages=e1-e1;citation_volume=4;citation_year=2008;citation_issue=1;citation_pmid=18179283;citation_doi=10.1371/journal.pgen.0040001" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_title=A statistical framework for SNP calling, mutation discovery, association mapping and population genetical parameter estimation from sequencing data.;citation_volume=27;citation_year=2011;citation_pmid=21903627;citation_doi=10.1093/bioinformatics/btr509" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_title=Minimap2: pairwise alignment for nucleotide sequences.;citation_pages=3094-3100;citation_volume=34;citation_year=2018;citation_pmid=29750242;citation_doi=10.1093/bioinformatics/bty191" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=R. Durbin;citation_title=Fast and accurate short read alignment with Burrows-Wheeler transform.;citation_volume=25;citation_year=2009;citation_pmid=19451168;citation_doi=10.1093/bioinformatics/btp324" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=B. Handsaker;citation_author=A. Wysoker;citation_author=T. Fennell;citation_author=J. Ruan;citation_title=The Sequence Alignment/Map format and SAMtools.;citation_volume=25;citation_year=2009;citation_pmid=19505943;citation_doi=10.1093/bioinformatics/btp352" />
<meta name="citation_reference" content="citation_journal_title=N. Engl. J. Med.;citation_author=S. S. Magill;citation_author=J. R. Edwards;citation_author=W. Bamberg;citation_author=Z. G. Beldavs;citation_author=G. Dumyati;citation_title=Multistate Point-Prevalence Survey of Health Care–Associated Infections.;citation_pages=1198-1208;citation_volume=370;citation_year=2014;citation_pmid=24670166;citation_doi=10.1056/NEJMoa1306801" />
<meta name="citation_reference" content="citation_journal_title=PLOS Comput. Biol.;citation_author=G. Marçais;citation_author=A. L. Delcher;citation_author=A. M. Phillippy;citation_author=R. Coston;citation_author=S. L. Salzberg;citation_title=Mummer4: a fast and versatile genome alignment system.;citation_pages=e1005944;citation_volume=14;citation_year=2018;citation_doi=10.1371/journal.pcbi.1005944" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=M. J. McEachern;citation_author=J. B. Hicks;citation_title=Unusually large telomeric repeats in the yeast Candida albicans.;citation_pages=551-560;citation_volume=13;citation_year=1993;citation_issue=1;citation_pmid=8417351;citation_doi=10.1128/MCB.13.1.551" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=D. Muzzey;citation_author=K. Schwartz;citation_author=J. S. Weissman;citation_author=G. Sherlock;citation_title=Assembly of a phased diploid Candida albicans genome facilitates allele-specific measurements and provides a simple model for repeat and indel structure.;citation_pages=R97;citation_volume=14;citation_year=2013;citation_pmid=24025428;citation_doi=10.1186/gb-2013-14-9-r97" />
<meta name="citation_reference" content="citation_journal_title=Fungal genetics and biology :  FG &amp; B;citation_journal_abbrev=Fungal Genet Biol;citation_author=A. Nantel;citation_title=The long hard road to a completed Candida albicans genome.;citation_pages=311-315;citation_volume=43;citation_year=2006;citation_issue=5;citation_pmid=16517185;citation_doi=10.1016/j.fgb.2006.01.002" />
<meta name="citation_reference" content="citation_journal_title=Eukaryot. Cell;citation_author=F. C. Odds;citation_author=M.-E. Bougnoux;citation_author=D. J. Shaw;citation_author=J. M. Bain;citation_author=A. D. Davidson;citation_title=Molecular Phylogenetics of Candida albicans.;citation_volume=6;citation_year=2007;citation_pmid=17416899;citation_doi=10.1128/EC.00041-07" />
<meta name="citation_reference" content="citation_journal_title=FEMS Yeast Res.;citation_author=D. A. Opulente;citation_author=Q. K. Langdon;citation_author=K. V. Buh;citation_author=M. A. B. Haase;citation_author=K. Sylvester;citation_title=Pathogenic budding yeasts isolated outside of clinical settings.;citation_volume=19;citation_year=2019;citation_doi=10.1093/femsyr/foz032" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol. Evol.;citation_author=S. Panthee;citation_author=H. Hamamoto;citation_author=S. A. Ishijima;citation_author=A. Paudel;citation_author=K. Sekimizu;citation_title=Utilization of hybrid assembly approach to determine the genome of an opportunistic pathogenic fungus, Candida albicans timm 1768.;citation_pages=2017-2022;citation_volume=10;citation_year=2018;citation_doi=10.1093/gbe/evy166" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. R. Quinlan;citation_author=I. M. Hall;citation_title=BEDTools: a flexible suite of utilities for comparing genomic features.;citation_volume=26;citation_year=2010;citation_pmid=20110278;citation_doi=10.1093/bioinformatics/btq033" />
<meta name="citation_reference" content="citation_journal_title=Ecol. Evol.;citation_author=H. A. Robinson;citation_author=A. Pinharanda;citation_author=D. Bensasson;citation_title=Summer temperature can predict the distribution of wild yeast populations.;citation_pages=1236-1250;citation_volume=6;citation_year=2016;citation_doi=10.1002/ece3.1919" />
<meta name="citation_reference" content="citation_journal_title=Nat. Commun.;citation_author=J. Ropars;citation_author=C. Maufrais;citation_author=D. Diogo;citation_author=M. Marcet-Houben;citation_author=A. Perin;citation_title=Gene flow contributes to diversification of the major fungal pathogen Candida albicans.;citation_pages=2253;citation_volume=9;citation_year=2018;citation_doi=10.1038/s41467-018-04787-4" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=K. Sanyal;citation_author=M. Baum;citation_author=J. Carbon;citation_title=Centromeric dna sequences in the pathogenic yeast Candida albicans are all different and unique.;citation_volume=101;citation_year=2004;citation_pmid=15272074;citation_doi=10.1073/pnas.0404318101" />
<meta name="citation_reference" content="citation_journal_title=Nat. Rev. Genet.;citation_author=F. J. Sedlazeck;citation_author=H. Lee;citation_author=C. A. Darby;citation_author=M. C. Schatz;citation_title=Piercing the dark matter: bioinformatics of long-range sequencing and mapping.;citation_volume=19;citation_year=2018;citation_doi=10.1038/s41576-018-0003-4" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=F. A. Simão;citation_author=R. M. Waterhouse;citation_author=P. Ioannidis;citation_author=E. V. Kriventseva;citation_author=E. M. Zdobnov;citation_title=BUSCO: assessing genome assembly and annotation completeness with single-copy orthologs.;citation_volume=31;citation_year=2015;citation_pmid=26059717;citation_doi=10.1093/bioinformatics/btv351" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=R. Tewhey;citation_author=V. Bansal;citation_author=A. Torkamani;citation_author=E. J. Topol;citation_author=N. J. Schork;citation_title=The importance of phase information for human genomics.;citation_pages=215-223;citation_volume=12;citation_year=2011;citation_issue=3;citation_pmid=21301473;citation_doi=10.1038/nrg2950" />
<meta name="citation_reference" content="citation_journal_title=eLife;citation_author=R. T. Todd;citation_author=T. D. Wikoff;citation_author=A. Forche;citation_author=A. Selmecki;citation_title=Genome plasticity in Candida albicans is driven by long repeat sequences.;citation_volume=8;citation_year=2019;citation_doi=10.7554/eLife.45954" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=M. van het Hoog;citation_author=T. J. Rast;citation_author=M. Martchenko;citation_author=S. Grindle;citation_author=D. Dignard;citation_title=Assembly of the Candida albicans genome into sixteen supercontigs aligned on the eight chromosomes.;citation_pages=R52-R52;citation_volume=8;citation_year=2007;citation_issue=4;citation_pmid=17419877;citation_doi=10.1186/gb-2007-8-4-r52" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=E. M. Zdobnov;citation_author=F. Tegenfeldt;citation_author=D. Kuznetsov;citation_author=R. M. Waterhouse;citation_author=F. A. Simao;citation_title=Orthodb v9. 1: cataloging evolutionary and functional annotations for animal, fungal, plant, archaeal, bacterial and viral underorthologs.;citation_volume=45;citation_year=2016;citation_pmid=27899580;citation_doi=10.1093/nar/gkw1119" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=B. Zhou;citation_author=S. S. Ho;citation_author=S. U. Greer;citation_author=X. Zhu;citation_author=J. M. Bell;citation_title=Comprehensive, integrated, and phased whole-genome analysis of the primary encode cell line k562.;citation_volume=29;citation_year=2019;citation_pmid=30737237;citation_doi=10.1101/gr.234948.118" />
<meta name="citation_reference" content="citation_journal_title=bioRxiv;citation_title=Structural variants, hemizygosity and clonal propagation in grapevines.;citation_year=2019" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Phased Diploid Genome Assemblies for Three Strains of Candida albicans from Oak Trees" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11.cover-source.jpg" />
<meta name="twitter:description" content="Although normally a harmless commensal, Candida albicans , it is also one of the most common causes of bloodstream infections in the U.S. Candida albicans has long been considered an obligate commensal, however, recent studies suggest it can live outside animal hosts. Here, we have generated PacBio sequences and phased genome assemblies for three C. albicans strains from oak trees (NCYC 4144, NCYC 4145, and NCYC 4146). PacBio datasets are high depth (over 400 fold coverage) and more than half of the sequencing data are contained in reads longer than 15 kb. Primary assemblies showed high contiguity with several chromosomes for each strain recovered as single contigs, and greater than half of the alternative haplotype sequence was assembled in haplotigs at least 174 kb long. Using these assemblies we were able to identify structural polymorphisms, including a polymorphic inversion over 100 kb in length. These results show that phased de novo diploid assemblies for C. albicans can enable the study of genomic variation within and among strains of an important fungal pathogen." />
<meta name="og-title" property="og:title" content="Phased Diploid Genome Assemblies for Three Strains of Candida albicans from Oak Trees" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/9/11/3547" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Although normally a harmless commensal, Candida albicans , it is also one of the most common causes of bloodstream infections in the U.S. Candida albicans has long been considered an obligate commensal, however, recent studies suggest it can live outside animal hosts. Here, we have generated PacBio sequences and phased genome assemblies for three C. albicans strains from oak trees (NCYC 4144, NCYC 4145, and NCYC 4146). PacBio datasets are high depth (over 400 fold coverage) and more than half of the sequencing data are contained in reads longer than 15 kb. Primary assemblies showed high contiguity with several chromosomes for each strain recovered as single contigs, and greater than half of the alternative haplotype sequence was assembled in haplotigs at least 174 kb long. Using these assemblies we were able to identify structural polymorphisms, including a polymorphic inversion over 100 kb in length. These results show that phased de novo diploid assemblies for C. albicans can enable the study of genomic variation within and among strains of an important fungal pathogen." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11.cover-source.jpg" />
    <title>Phased Diploid Genome Assemblies for Three Strains of Candida albicans from Oak Trees | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"rP-fWt36b_0lj8DlNjTX0s5iCfUSkMZ68L3nCOV-h-I","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"494610","apath":"\/ggg\/9\/11\/3547.atom","pisa":"ggg;9\/11\/3547","ac":{"\/ggg\/9\/11\/3547.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/9\/11\/3547.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;9\/11\/3547"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;9\/11\/3547"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/9\/11\/3547":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022Wk3tO4UnUFj+mnLaLLV\\\\\\\/GLpoDILtrCaniP+f+GEUJQf+JnDfQG8GbfgkyTJv7iTsmbx+M2qGHokr4ODiP1sA1jvVyrfZk\\\\\\\/CFe9NSVqrMlx0wD3qbIdOaBFZKRbrBdLRFLmAUSvKynU6Noft5q18bqdtdbKL7iIq3iCkrBPL4IKIoX7s8T1S5c\\\\\\\/RB8tmvLGurz3AHdP2qDEsGuxZmzSLw8wPDpYkzbd9YydRKW4nke+fUD6ykw3JTYgQL8hPSv26mSzzCRjjLrM4x\\\\\\\/M4ma625EXp08jUzJR4oJ7egsBLPwgGPSKnBtiEw5jOY5Jnt5anrUv82kowx2iNJBL8lEXGtn5ni2WQc+azNrVNXZi0pkocFIMOWcAf\\\\\\\/dNczQ22UHazpv7D+msfV\\\\\\\/vLTZgm3Y9yJC+MuPE7JUVEgQtQEjkkinW4eJvydoACnpcM8d39LibCNKMTyMobibQAIV6Gsdf7cRAztmw4D1OOD53e6jadYx2fnJ8gfDfZnWY\\\\\\\/8fqaJ\\\\\\\/m34wE543XFrtAl42yeF4+tDgPAVz9Q99d3REMhl6kGXnmLoJJQHOG6qpA+wDGs5DJ+MWcRHNzgzzYarYrGuyV9ZoR6IZEjTBBToRNvna9As79ube90bLREFAj\\\\\\\/kOK+GBOYcSJ8GKYpY\\\\\\\/1CoreRh4N7qTd8YBde9jq8447lkfqLRfJC3D0IFnhCTeQRSLRXdfcj8vmhQQBhc1xvWbMhwwSZBJOB6M0GzQ2H0zSR1LSXyAYrg7iBbI0Y5YgnG0kQJa79uLb8Eo0P5Eg9lrBAIWcdgE5oh0Pmm6rCn8oWMY9vD+4hR5OLrNX91AbSYJ\\\\\\\/vW6F0csZ4UwsGBK63vlgGQ03WTkLPZpiv3kq3UIWl6EkV5GYuAe30lZom+VpVZ\\\\\\\/kG6qyXGWD9gPkXP\\\\\\\/fGo8HgXS42Q6dbMt89HQfnMFBFvjdL04+\\\\\\\/\\\\\\\/XI0K04xgYGQo4AJ8ABOE+lfZIyLTkMqLTvLqamw6\\\\\\\/Vt\\\\\\\/ejYMT8gKNiZJhzEFS1YdJQaGftbHbiF+N9qWNWLOa1wQ7rAY8eg1tt3KxMc4jJ5q2DTgpC6oE7fbn7P4ljENxP1bacIqtwok8tUhJTO+UuLfcHC6UUrB9VhWKyf2OXmfzogBBasim4ERxkUZCfNBwQYo5FFF2bY3vJ1vc4eTPCIaYzylbultxr55pRYqbdHeDJa6CvipaJ\\\\\\\/Nen01+CsnaD0FdPN\\\\\\\/FGhgHDwFb93rZK\\\\\\\/B4z\\\\\\\/bpFDwsA71W4tshaOiHo\\\\\\\/4JtIhJXHesFBP4dxbQTtlyGVv07b0zaU4SNvj\\\u0022,\\\u0022iv\\\u0022:\\\u0022Quv7of10ApC3lotnRKclNg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00221cf35d2712bba853a19f026b55c399e35395478bbf645a127b2436a7ba72559b\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022HrlRuGkGmWxRsDGN05AlK+5bQ\\\\\\\/gktZ\\\\\\\/\\\\\\\/rp8vWcuDjBdBI0uBELwLtQa+RWn9w0Qsp\\\\\\\/pDSTKrBoytb+AMtCeo+nrSy0LOj3fcqjakdMO0+O0NMQL61SeofnHSLd4ngOtUmzjjHgVfPnyGsmNSa9DccjX1x3FN7jJ8DeyrxxanPW4yJAdrxGeKxjskFf3wB8RJKzVDKbElAzows5H+4IZtGyp5eOqFQeVsBX5uRvfQmdtwhcxajlwD8OST4dxucPGqV1p5ElTSz9dgThJ41PlEkBAwT2h0kwubj6WNPOY1BAYpHq4yoILYpyhhzakfUSHHjfLsA7YQr5nADSrzI9qgiegpuR+xVii3VBvBQrc8GnFEyQw700ASQH+1yDOhKM1j0DpDe+MkNWE5Q5zmMj7BViJYf18niGJUe1uoJJy3cGvq3AFycLRNTN4MJlgzxrXKuqe5yOQjf3wJjMtt7PJvqPIK6FasQWmgPsEnova7m2VZfXO+Odn51H0djjgOuj2R+VvzNKprtgW9v+IH+r\\\\\\\/TKZrK5iZ7YyeuGyGcTrefkbwMEdsrjdxh6uPb602IEu4cjZkZhkyy+8PgIE9WoMt5rQ1eUmzj6fFvx8mq7mb4o6TJJoANiRpmI9fVFQlpMcR1+VOEdEzjbcT4WWEM7xiNCsHEs33f78IVIewQqLzI4n0Y4HePaOPtq+6\\\\\\\/eoCL2xkaS8UmpAe9YV\\\\\\\/vpw1JWHXoBvPUn\\\\\\\/mCbrr2d91\\\\\\\/Lp2STQGoZzlukydnrpnc1JlMheaiCuLYhFVL\\\\\\\/rkD45yeI2oMR3EAucKgncgEr8Apwk+qAWhKyOjqb7IiyZkfLbCZanMY1WvuQaS3KDIcjjPKqMeJi6h9lYXFQyLD9GDYcE7+p\\\\\\\/L6cMXp1v5C6UScWhp3874B14YKKGhMzHpH\\\\\\\/VbFWHouKiPIOrdpFNFxIMOknwgifW+KiFxDTC7O4yAU3l6StonOE+VK2SCPhQwcCJyxsmSshYQw+PQ5cLjR2UrGSsmTQxkVw33YOvqSpPEFKexGgEgJb+H1eIXWQz\\\\\\\/ZpXR+WLRQ82fnIweTRckPj1\\\\\\\/\\\\\\\/7IbNxEK5dVJ7kFaGXBL8P52aql5IJ5C1\\\\\\\/6No4ta6hqP2g8FxrXRhFkthLfrQRhRPavAEEtIFeJfsdxyNrogyLaI1R+5TQGnRMVzYcWzUhjnGxXL8fZhfnfnA+KwTtB5nUUvG3\\\\\\\/UeI+7Cwir8vCOoUu42XX5yFXU0H9YaLCMEq08OeRGUoSy+QRsx85dJZI4sJHhrlarNlWUJoO9LYSj43sgz4gDWxekTv\\\\\\\/ELTye0yPVd5TlwbNGqp462f4Fg7ka9vFR7tt0sen5fNRR2EgkSmscKlvW0YhjMA9N\\\\\\\/J\\\\\\\/ThJptPgoVqWKFsnDlwLLE81aLeG3OtEnc2isdp9S+qv\\\\\\\/DRkvGSOUKAYLI0jvu1sFni88GAiwCtaD0YWcjNGP7s+wwKil4eLlgWRTVzYXr2OrZxeYit7MmhZB8vJi5l4CcByFlDW1p0X9ltA8wf0RcLuCwbw1v51vfecMciylrbwEqcfvi8Y1gW+s+ULGeFenzBCYehjQTnjlkCQE1ynI76pyznbnnXodKlyL5vhicStX28OUp1lvMOyBGVaqgpruf2Shsqvk3\\\\\\\/3tddrT6HmZWtJU0s6Tk\\\\\\\/xY4aNXFoJHvcVPdDL0Sm7Bh6qK0hiHSDH1rlXlhkllvLkXARneZ4TT2QyrD9rJooWL+IhQm35hsu9XDgno\\\\\\\/Zz5VLf7GaqpXOWyoH91PKt9Z+VyI\\\\\\\/3PbZK7UrocMs6Lhn\\\\\\\/g8uJ6aEmUcBhf+ORAqB0uwyAOis5lh9BBp86Px8RDCK6b1ZsEsDBB9GLmIhYmnQyU0DeNr+TyZzB+X9n8jUE7Z49FCE7B2250Ey0pT7i8OxEJAygA52A1gvnpRf6cMrilj4ROnP8RP6EiPlHyacuUy9BmuOA3C9cXBA8tNOGkt5R7QDIiz2F6lJDylmXBdsC5+vmyYsThAfepPpXxYXo1ZzLV6pQRSXNydKLc5a6NFP7NFY7usMo8+5Elb4+jEyCW0bIl8LQYn+4TY8sVQQ5lFIOu1ZVvipGPr5GWMIVzA9NLBmILi3CSx2r9aLdwWTWWSRcvwj9nqH0cQVhbBhK\\\\\\\/aUq3UTtdSYcR8cZwVBI4weoVeCOHzsSo+1YnE7e5s5D4lFpuVkVQhTB\\\\\\\/0PGB0rrTjSn6c1mqxPN6Oieg3wJAZ\\\\\\\/cDvqv4GiMZ06aGxqAn2GpKL1a993cAKoiXnZzf5yXjI+jW1hFpBbDTaSl2WAsZrJpUl\\\\\\\/N9Q3AaRfUGG4kM2+WqUpmq3DPUTOqxeFOKZTf1DZhfNnyq2EE4Q2ngFtWfyqg4GnASKw7F8ifPwfbWG6Yw9O\\\\\\\/XuyOHXR19mF\\\\\\\/aHVcZwWFG0CjQII5Fh4s2vDqf1WjzVpxrFiAyHwepazgJMCId7fI4beRrj1okeV3aMe+7HsO7Jhl9ekI4GbxHHHpUvwNv\\\\\\\/dFr65oY4jmvUvo\\\\\\\/zuVQaiksCEdsp+keVEDfN1orKFOhXGWbs6GfjSe684+o7auIMt4Nqmnjubi6xImifQGKJsAalatOwR2ZGN9cJNWsvL6qxy1BtqHfst+2HaTkGPPykSZUsYAOFNHDEeQvzxUGpM+U0FpVsEnYRKOXcfowZmIPsedGev9ob1+coDyw\\\\\\\/KBQEPtZdzo5Cm3jQejcRFt+P1rYvlQaa8vVIIGTIZQnmz5aD+aCBmgmzYDbuLTOa10EOmySJMuDE8Gs+mL6cLXhFrP0\\\\\\\/8sSpw0OXSz6voB+y+m+ik3XhYDDTlEV70sY8jQDOc31k3eIWxd3z9CL+8iuvqeM\\\\\\\/QoldhL4PZVv6rYwfofSw\\\\\\\/5nHWJ4DpM2T4guerfT3reQi6GkCrmpucAJe1t3u1XsKuOMl5QL0ULSqHa306Flk8BDlcU3b9jpo4Tu844si3ovg4N5dSt5Wgy\\\\\\\/Md+QhStLyVOq\\\\\\\/2kzbBPjrJIGfByrqdA+26QThN6qsTp9E4DIDL0yhtedA\\\\\\\/wxilQcpBFufxUouHmyLxTvYx4GmcqyVgcyJgPAN4BjlsmmaQf+q0jex1HHim0\\\\\\\/175YRDmBm9GU++FvpbHVGB6UG07DRaypUCehuNOZmn4UfFBs\\\\\\\/5VwyhI94oYmME1473nhwsFJU+J1NGHs9MS35SYXeQ5iIjP9GXcwsEpbI3EjU1LfOPO\\\\\\\/qDPOQ94EGBelTRZSdb2BBFmJsGPPF1HHQITuNuitSl+eNybr\\\\\\\/ViLvYKhsDyoffCvQSGNb0xQA1GNh5JXMRIMqOaKTQ7JvKyngOIUE4Rvy30a0aDo9RUF+hQHxn9fX2HH70\\\\\\\/logL9viPmKRs9S3n+dli9ZTNTcefN39mDv2Qz2H6nk1A82MTNeCgFM3Abaez1EPuQOC0zTZgnahnI0h0r1kJJlaDdrMiXSigpuNXWPtfzw4HIKspGOrZsfegcYnTYWDy0ZfRtb4Q4XVb+pAmuHWN+JT\\\\\\\/P4ikgOmmrdbDv+y+2UT2qIAKR7hIBb2YcJqAp\\\\\\\/8X+yvuTzLMihiXRff5ivFUy1fxvzKM0ifacyNYli3dzODAXRa+TQLH1kodMM9QP1yszOs1Azm19L1ask2UAqDkhT+Z11tM07kh3AO8Dn7KbNiouZpuQENKBf2KDP2QZd5AOb+MGNuPxyz+vm7cbNKmQUo+bQKsIHCXddMMqg6u7id2DtWTG\\\\\\\/IgYKDHFDzeo6\\\\\\\/Z7JP0veWV12aKBE1blprHqie004ACLIc9eVttV6GSIj+jVOXqBvt6QKEkigWE\\\\\\\/mHnBuGcKutmmejeCA1lyyA0nseKCV22cD69N+t2Yy+qMzeXwESwQYHd48oSom4TpN6Pq7WToCif96TCOmkQdBJVWxXMxW\\\\\\\/cCJMTb1mR8oRtc++yiMjXyMrPDfJ8dzBx4ezc2D+KVs5A6sLtzPgkcVsZu5rXvLUATveFLy9VTV\\\\\\\/WJt03\\\\\\\/eqK6kYIhC5lBg7eRaynF70PoJd93S0wkpwf9RblN2dGRGuMysIzM8RTr5vr0skI53FfBy9d+i2rkyORm\\\\\\\/rhr45wUavCtzu9wvh+FEbqqqjMpjHRUie+UXZ+ELvVb70UDZGsJuO5Hzwaf7e+QQgWy5LhJ7+U5eGkXcHYRNV3zQ+ckhEvc2eiLrMXNd94dCxsUEwydt2HuCnDxTOHGZjpU+Bjim7NxFaIUCD74FlUms3w9rbR328OhPy+3j6B2uBRUO6N2orUmeEXLmVafl3fLCjT6uH1VdVkZp4\\\\\\\/1Lun0mV1IVUsSFlpTL6RcP689KYlTrYMkQhnJSBsghSK32zDHnqwUfEKXefYETvlX3hu20AqjQNs6ek8u9FbszS2xVSxgbr\\\\\\\/8uUaJZa70OI\\\\\\\/jZ1K6xHQehHwf9lM+XRZ2AZXEWlfYXOJ7L5TeBudDQIF8yvuMbk0e7lquhkgda0+to0\\\\\\\/uIfqJF5mZYk+PFl94DJfqp+N8d9lMOvWUPOJj4bzelYKBp2t1tGmRRxsCB3++77UlVXE3eEaWXq2MvScjmvqPL0yjnwHVd4YBcaM+zwUvFAag2ldLXG\\\\\\\/8rpJc18XFLw0jtmsHIWdVfz8bwUBCl92zGOpq\\\\\\\/p0npCN13pcpkAnPAKdMovFxEs1GtUrFl3zYteGvXUQQ9\\\\\\\/3FkwIXRByIvULZPPef093oo8szr9YdmYc+Kjsjrz4wCIlcVD3y86pXBvaL1di34hQnWdkwMWjeilvysuTyj9yfRW0N0gWyDdimya3RePWVzmHXhMUrijkCDUfqKTEEL2kg7amMsxUhJjiSsF970db241k3Hjwmfu35u4XRn12+LAobgmxI93bi3Qweiw08yqXlGotwFd1ApclSqTVfvi7EnPU\\\\\\\/HC5H5d5AkGAiPzUIWs5Fe0IvBYtpDBxjv0yrNo5flSX6XBuH1czJjLHds\\\\\\\/+eGRH2Yl1kKmsJAH1viU1amWfvW0\\\\\\\/TUFMn\\\\\\\/dxCkYQUTJii7Jr\\\\\\\/e0gp6UePwaxgTnJJl7DrKL3NciNY6iyWjpI10SBhzvGxSzW8gv4fviVLDua4LWhnXDb3YmDvr1RpyLX3OTc93C8mJebExhJqNKjnND1gdA6fvaoMViomDvgd3gCSm6mxfQk0GSHryAOcbZWpJJuQiy8ItckgMbz780J6L4p\\\\\\\/f386dtnURNKavXUDqIhxArPv3ENyrqmMFgTUGmtzDVjPF0UJcM84rANCdMt9ud80H9JHMyoDDEx838WaDRdpaWUWFCGzKfeyln8WPT3gJ10GQ1UEHspVtgQWtprBkbY0wvK2MGUPE24RKngUMnoYgCyb+6eII5edbk7yDfZ\\\\\\\/fdNArlu9gOyLJiTNRibRb+qUvr+D6CkD1PlmvUUDWZMXwpVrKys8VTRj7rANBlt8W8FU6EV6GuWGzv6CZi2FhqLZ0Ik1d11jygntC9FOHbUhj8wPlsrkGQHYyojycieQ+VtuV5zeqERAI087juYRAudfAg2+0uPHY9V\\\\\\\/nl2gOemqKUqv9bYnb3cA0py7fBEQ6rvPidtBqqhiPOsaX7WkTdsAikHfkgb7r\\\\\\\/v9GVHKEtJd9WBUE0Gf0ywOBDHLkNfcGw45Rx19Y7jLVxdoQZOPDe\\\\\\\/bTWnzjt47fWYd1sxfi0EdfiX56g6MBHzO+YMIPVYupXBCOH2\\\\\\\/wlqhCJcE\\\\\\\/OvhDLy+Zb7CIRPFl+VDHESabik\\\\\\\/EqfB\\\\\\\/zeM3vPLX0HezxKb9WgS6\\\\\\\/BY6VMhet3UpKaARVlCIYSYdp\\\\\\\/BA4tpxXfnT8uG7OmRnuq2JAy5g9w4eOHZ0qgDxMUrO6568q2qbrXPhGVq5i+p6bhy2Ib2VjKznG3Mq9bmL1Zro9E7\\\\\\\/T0hDI+koSe95ds0u8TcSMzLX0r6pZOiE04wi+qATP1hFS13lCwfhCXyhJEp9JWNT4rrHJDqhqDh8C\\\\\\\/xB+7uDlJi2\\\\\\\/44sqE2FqiOIxT21ef9ifz5OSj1SFDm\\\\\\\/EZ1qYt5\\\\\\\/1shcx5esOUw5X19r8R6MOgZuKf5rITNBsnYL0ZQEKvYZNoOukcMwlceesz4x\\\\\\\/P93xKpjaXgvqoRst5IxS0UMO0YBTpzQfd\\\\\\\/GAph8Frasdq2y\\\\\\\/tSAfC9YwXLA9Y2+uV9096ybXIpRiUu\\\\\\\/qiKv8yIU1fhwZjxl4hRRZbr6W9+K\\\\\\\/TiwZUTksePybjaUqJIVcYlr1FfCFXnEzQBNnE5CIUUzvKqPpsREeb7RTSePdAASDrI9LLNrLfuPzxwOLt5xXE0Al5X7uj36ASxaAdImoSu\\\\\\\/go7Mok5Acbb4zYuunFukqQEITWBTgWSQcjOBv\\\\\\\/dDOcRUaEcJoOGgEnNk4\\\\\\\/8xi3I1hUmNc3SES8iBBp6VUAzGR5fpjvV5lqegEubVsiSC1+qvEWrwEU5lTAYE8Jx0vQcB5FUS1uW\\\\\\\/iyc5z6ahxeqpXOLet54a7wsKC\\\\\\\/zN5Pi0vwML8znBEJJbZnAV28pHyv5TL15KhCZ0QUpAxo8YXsIF8TUvRIWmN596kVW9KeRlic45fBtRGP4Kw+YE7qD5P7grjzpvGC835Mu2H3ATcTKHjRvdf0mgyIFw\\\\\\\/11CjgOJhpfKjnT06z08mbZ5q5ii7vYrWvh5WouBpmsBAyn4i8ejFKTDzdL\\\\\\\/QgdV3peAfMUWvxx9mIgNQx2s05i+HVEb0F9FJO\\\\\\\/TUOw4SZMXzwrTiJZnZ6gbIIO8FF2F85w9hVGx9s8VlqrAjY9bKlgHZr5+DDZn1r6s2aGOYM2Wr1obnejRfrOuBTLHeP9qnr\\\\\\\/PwzH\\\\\\\/lUI2s5RGRGlGX18M4KMgmgUBz7v77H\\\\\\\/NbljFJwNUr4JajrkYxcWplf9HNn\\\\\\\/v3XgxcWFxXgH2DQz+vaou9PeUDRFOS0cJnTtA+m3dQiCaKvokrczHM3qW+WQt5lvgRD3SmLtUQd2klVEF76i5lNCCAYkDdeIaXxNw62Ce0LyybscGL7DHBkuPPfGdyZvjwVCTKoV0RCd+21PP3NPp0eLycFTQuO1XXN4HsbAa76NyflO\\\\\\\/fvwJ68qscvn1j0XMveHBfKStZIHygW+6FkAcB0RF+t\\\\\\\/\\\\\\\/MCGsGdSveqOeNP5t1vAcOgTDrL95sOQgxiuLXYVEm7dXEB9DZIrU1EKegjmypXuaYRQfqhW6OaT25TwhFxW68ztMz7qmHebijpssuBeLAIUO+m5SwdaSU9LJ3UNVu3yDEsyDgXgTFliXKEuTIJWc2lsVTpS14QAneXFBi15dRSxfSqQ+OACnkVYwRyYBnR9J9eHTAJ81odtWWeEpOQe+\\\\\\\/ZXZc+IWdQCzkfBXO7KdXT+ct\\\\\\\/NGtb1u4lXzSqjWh2aKJ4MguiAj5p03bBkwxqvmETKFJwfOc01mNLVN42beFRGP7xfoQSP3pRuO0282kEoVbbTAU1mFYT5QPYgwFbqTnUkvxTauomH4ExYPXxM1Ki2VJLsa0KHlNqsEGBK2t6QaT6GLSCXxqKIA53OPzlpS3lGFU4EF+mdHlOXB7\\\\\\\/l30UZaRZuPpc5GeroDdHxLj\\\\\\\/+U4PORH\\\\\\\/FpVX5bfKAXXWAxVhjVyo8n2Cbeq5Sj7IJ5LEFbopgvnjv5yGqVDkO8WU0wEP4GlQjpngSqbI\\\\\\\/Ry7OWzWszcn58qjBplkz\\\\\\\/XQNN6dG5BCfyxtCVDDHSwW9uz7BvRPP1sEyBZRk4aejX5JLC8REjMfc+LO4wlwHOlUoWSUz9c9TD5qNXpMXKfD7a3miO0Nrl7viKD\\\\\\\/7AwqPaTVPKN87+ajPK2pAf6cr2a3PdQXz1i4gUEA2zg9tT2Fvp2x6rHxZ4bTebiEuiy60Z61lGdPpHG3oNjVCF2JyV+m+PpgfdQdoK+orOp9hctTVaVfzx3bTlIpDvzuYzFLdnT7NhwWGv8M0a57HCXDdoL+QuYRiKtRufHfvWIxUz7ovJozy8l\\\\\\\/t8l4m65JO9YXJvBGi9XI9QnzwUfhegoGqtaqtHlX8XCtHfoRS\\\\\\\/N6NMQ0Yk1XtNCbtbcZ\\\\\\\/eoy5nwxptoSq8cm8XqUJQMIHNkzhi\\\\\\\/MFpGqG7qqF6JHWZPXZNslGdZRjRNzkKUuvLU46VUbCgss84LjANQnjZx1aR0vyqk5zKQ8Gl8M3BOMeYjuYjQRgOb6Iwn3okn8dAFZ3o3CyU6dGUBdXpab9MRGi7cX\\\\\\\/v\\\\\\\/xwbM8DVETKe02TZ6ZDGqL+vN+YFuFpRH0C6+kjcVBd9TQHZih0B67WZnIickPUCBGPNItwLUmfdOwvlKExUvZkO3UJGVmjYLSB2y1LDxWXhnyjxeHWQXEil\\\\\\\/MUzrJVHzu3xfEanLLAtatlnOG4E64w8Bsf04PTcA4X2CgTUZSpL4oY8QybvCTgPW7BQqXGkukL9FdrmxmsXtDBfGn1Xe1y5TyliNqGR0Im3f\\\\\\\/LwH3kxDGy\\\\\\\/0Efx+5dsOqwWWcWa2fMrK4g\\\\\\\/RSlKoWGVBoNRyDa6+tiszFlNuv8usMH+tHSQ2RWEdoX1p2upmwhAPWQG\\\\\\\/CXBBouGWHdjdz1W+TTTX25ZkjFveWKSSlH5VfpOZlk74KpWuMRZl7O5VVMlTNp2BErZGoWLK4HQLBv6tgZCsjvmDYI82\\\\\\\/OJMegoxPAhDQN0eiAQwvDKQSGNRA+OHv7MLouMCg3K2vzu8gUTsDkowkplVf4U52wlquuNthto6K7zYkjwuRLtLRI6gCii1FGDEmhxm0Q1tK5k+yUpVgj0wccD0d1\\\\\\\/KL5aEkGwOlakseZFooxLxp7hiV+l+3TqquA41yWtTFm3fKo+i10rOI5qk9\\\\\\\/PQU3WZfujbFxz1PBr49dUa12mUHiBqQ9h57KrMckVs8nE5u6navwPqavWEEns3pk8gyr\\\\\\\/vQzRm3grVLex0g1F7mJshW+dQNd4fvScLEbtxg4QDIxRtxw1iXWG\\\\\\\/2ETuHnZKu\\\\\\\/rVfY1X4XKFwQF21b51FC2IaXQe+bQOWGiuWDQkwvKnm99krpL5u1WPZdiWCvTjjUvonlb91oXTo0wDCrHuTfS63dJDeMxc6mUBXs9w4nMhvrJpWXPOCc5JekpxFdCmg7LGX0Kig9df9V2La2eux4WdemnRiUZnwgAVoS3BqnHBfuSLs9KEX1B2b6x4mdGA8qHT065eVfFkApnNYDgONxSar2gb0LHEcupK646KnaxvRLdm0cMmdVpevCfNrKJwB4VribCPCq9IgqvvNQ\\\\\\\/Pz1xxGU54wF92Kd7vY9CwoSKcXPhotwVIIm5B\\\\\\\/gmvpWzmf8wku3ro3no+NzxFvWlyQTWdKKC4uRUq30M6cqzP2ZLehCl84Bju14LX8KUOXIxjXet8nbcYvu1+Lg9mVCMwKKPzo9CZXw3GjhPw3SjfAytKM4DMMwZFssbJSa885QVgYoFIlg2qoRxA2s1jZ5NMmoR9MTSqD9nCmVbAV+pJVWIgrA4fnpKxQsOQXGe83zCfhLIpkMNwStl93rQQGkOO4+q09Sx8PCjcZTTdDpJOZIg+Nq\\\\\\\/4GMXK8iX1hi1EzRrk1jywclkmxL5Zdf7f6XGTBALKdSJoh9xVgw\\\\\\\/jWhGuKEjYgHQ1IWlBl9geCkW3xouAxssrTCcKj2J3jPyp6Ga+kxa99asObDlK81NEljJUypzBy14mjr4ft3YdSmrgjl7aJZMLFOHQHebpTzeLp9zQ+cw1PkqzBUPeG5\\\\\\\/lEXoJZ9tcjzeAVCvJiDxvTVnW7g8lhD91qd5kkQQ8vxW6KbxY0bULyhDwCP00UsjEly56VD65t3zJTQca+uirwAkSt5yJD\\\\\\\/nipss5kDYafz5v\\\\\\\/NwojBMcXqmMZA8dNpKrG3KJDUFPkAEUmr4V1anV1SVEhjri0btPB9GI0CVfdo3740plvqyoq261BOTGqTzKoJ\\\\\\\/t5bj7ghT+XacAYD3oswffLNw4FPuCp7hUkKe3EJ08ONB46Cl38SHvgMBx0NtCCL8qlSRvLtonZ2g59o1ePAZ6HoldqJTVqcT6fH5GA6kuQ2Z2kvvgdafweL3Ixs+ct35LSPyD6QAbKANRLQwti0utpwKBEpqyqEFPx1ZCM59aiRexTYuYzixlgfiuMs\\\\\\\/Dai21jt7jHuau0\\\\\\\/OCFIJPhBDC1Ew5vSoZOmb15ScbPFmxAHZP3Yz9oY3JmHaVKGDuSqLsDKvwX8Gj3y7H2ErpQQ+whSAGy2p3b\\\\\\\/ieGq0zDzpvF+ejqczNNH0Q0sKfEsH3LrKJl9BS164Y91qjjYvgu86t79tjtOEvwwKLLpz8W+WoMcif2aFJHR59DhCpfGu+qi3ojVg1hvw9d95UNw1n1IFOZmkByeMiv\\\\\\\/DrqXh9\\\\\\\/QaDe\\\\\\\/mSgATEutsYYxtnp3xRH8z2e4KqvoT3o1dB3e172A\\\\\\\/DshaRcBiXyT\\\\\\\/RUax+oTFAPenqFjQZg6GMN3vE0LMQdDBogG7m60nUGUoBiPkfvSHu+XPzlv1bykxG8f1OnY8ixLklRI\\\\\\\/+2mxMnaK9FBJU7hPrFOFieSwhgK\\\\\\\/hJ7WXdA6QCxZuojhJitlRPWYOW\\\\\\\/WFzSXdRshoBMJgtKKezc7qRR1imlUNxlNq7pNzwgpsQdolJIzle0BAsAUuqIzcL7gHb+aZ3XShe7DiOhKIweoqH362OpnX0HGOG0YoEE6MVUbDgHVPefAZNdhTltkIx+od7AIzSRITyzfBA2w+XafJdIKO9PAfQo7BymrvQFBvNYvoNVYiQV1+cgLcN6VIKy5Dlv8U1oXE4AhY+Yzzfz2vSoqCw5mFEPQ6Hdo0U51TW3kLFX4tKEcmJnE92gYyGiVP2ik7XelwPtluGpjncyLydY9hcmKfe4\\\\\\\/R1ndRBuNM8Dalv8RcPWaTQq1tjJt1N49eCuaTUMFS2AetOXWu\\\\\\\/zVGP2mgogIMqFnFSd0PEx6MHNoTAeJpgcMCJNKghkYTQcbfNygrklZ6HLXWE76uc1duqHAt9mi3R6BJee65Z1ecle+cLEsxo1M\\\\\\\/ExSoENTWqMKrRachJzw1Zkg5FAYZdjcPVk1iTJrAvucyYQ6cq+wYOCM80Ouuy+sc4aHwvDEIXSVQBHo2mYSdprQFVAHPbwDcDhrUrEIVIWCP\\\\\\\/OqEsJtCzeG\\\\\\\/zW6CnSB3iqFDmiOb2DvFgZFGltn0z8E7d9IhMHtpLQqtD26wmri0cmjZD01ATyL2OUvtfM4NRUeHspmGfM3VHXljJQHFWdWLKUQTzN6vDCjG06wVTCWwP1biOJnVPy\\\\\\\/66TmmGjNKaPMRtMb9KNkbNZtFQEWAufmPHKRmC9OLwsCb4OHJJaeF9SipGtPcZan\\\\\\\/V3xS22fg4b8MIic+Gwn6UV5Z0onwK53wLQ6igpNG\\\\\\\/eIia\\\\\\\/fAfJT3uHQk63G7AHQlrugU+msCz0Cq9cdCuQ4N2HMRGSqXk12JIEt8Ro7Q598fIiKkzcH99FpVm7\\\\\\\/9LvhzarGHyZUusgk9NhV\\\\\\\/hpdn\\\\\\\/bRW\\\\\\\/KwGvcDDV3ndD52nq0gv8YH3p1tPh5vfnTmVxKqMKwQKvPAt+SUA0i1fzlr03r7VuGiVVuRRsqrrghURjBPIyf6uWLvHN++GjquEsqzuNYdCpjgH4RUIgQWVmo09CfQDy\\\\\\\/2uGtL0kw+fp8NJ6hZecSalot\\\\\\\/QheGOs6jwOCSIwOhCvTDVkcguu+EhYlQ+jsRz3EHwE17GPUPhciJxhDr\\\\\\\/t8z9CGBuLVUdCFifBfmmqgTN\\\\\\\/HWedcNmLlbXx1KDfoLjM11lb6vJsHPvwprfVX2jPNSuNQk4aHlcfCdBesjrgMTwcqsxYohbIk0Wqa6tlNU9j3K2ZEDkHNQlmVr9jAY40aih7WI0uw6gF9qZ0\\\\\\\/bXxKjhE4Y6BylEYgH6qnm4ksdyZLv440PEPCQ9sCN1clKD34HFBw21wpEBRs4B7Dp+jF1O8X5uoJgVDR52osUZhuNq5kLFUGLKmNzQFxPRYyCIAdLD1rogcv5ZLxcsMO18cWI+DRE53vs6d0bHN8BGGH09jI1G\\\\\\\/Dc0EfSNLJ+tVm4tdR1mN4VEnVGeNJNiLOxd5eb1Ftx35n+46ChajIC2GDg3YQZjVSfB8CVHfRATQTnEbvLs9oBWIw3XDsp4VIG+HFLBQbTmTWgno0lmUASgTp2o05x+cgoMjM3se5oaWojeGcyZ867aUNPBTB94UOaHEJqLL2NP9FvsrcSAHXpXh8EPNV2FrEvJ2qoiOMheaLQh\\\\\\\/pP+ViIPPzSK7OGkJHhNsH7v1bPll3Ki8rJ0Z2\\\\\\\/+COAKaBH65pn5fwHjzJmS\\\\\\\/CbwWizT8m312mKz3Tdr7uff\\\\\\\/D7qN4T3k2mt3A21NIeCphIvifMsBAtT6kWNnV7bSC6zDBxIt6l9r5ODkV91P1SyUH9D+5vGbBTKfR4hCAA4P2DKOdwUIS7I6Bymy7PJtNKXdjc5UAUphrD37k5h83nFDnM1ajo\\\\\\\/76GcwEFd+WFWTfQ6tCne062m\\\\\\\/eCkft9izEGh4nETVfa1Y85aktxyUOFO9hPErnjbw\\\\\\\/ooatxWfB2r7EsYZINWHsgkrtZz0TYGrFZnf\\\\\\\/JVER8291La4\\\\\\\/UXB3aHRZoE\\\\\\\/09nmW4QsFifg+QtMsLSK2v5nNiLSLuOqGYpaf8lrz12OoVKNCfEF88aVFqQOLCEDLlhw8ru1Ag1\\\\\\\/9+ymjRlmblNFVWqgMfLstuvcEHVCRzbONT1Qct6s+5cpI3RM5l5CmEiVT6e1Xl0INwgBRMRAOOaMy8Rtju8\\\\\\\/dNdFFfu4zf8PpWKjKNkdEX7iqP4gxLTY\\\\\\\/tkJgkdXvw9NFjCFny9WQXpDuW2C0kNWLYwMUB5t32JLe25qZmqUS4L3GM6vbkKaBT4kYQRPhVkUFMORtk95mrznBiDSImGEN6RRmqrnTuGwBT7uvddiCl9729sg7F8VX+b6aS+MJBYJnbATzqo2Yw9KX34W5L7bPso+N2fRCXPgkdG6zwsYYsV+1IEzK7PgytiCLr8Skba8Lvs5a2oov5mzsjbzUezxRTCJrz3dMtXOgSlBiVNL3GA+DvcRyUSKiVS7UVkNuXt9UPTbtMkWqJe8qP\\\\\\\/e8BNqZQ4i1+vWeX1+bwvx\\\\\\\/Y+8eZQDKH\\\\\\\/VQMrEQgOCnT6WqZsi80FBLQ7zmulNWfTxw5XIdQFt7URgmTuvPTav03Owl2B67sBRvjdFliYw4AuctW2tZQ4WNV7XLuhvXJmKPieXf3EyeBQ5TtshVTYxXqp253ILyajMhqKp1IeJSXvKrryr5juVaz5T6\\\\\\\/qeNAzvHTGRzVCBexfbCYkU8VtjphMCYcWlT33o2rShqxGVFjo6wZ65leI61Rs1I1virqjte07HMOhBsKhH+9XHUKri9JWH1TcE4uwqW+iBsHv+9wyQ\\\\\\\/pbcEKGuzGqx2BXfBFlULI+VaDsfEoixnLaLbTgTKP6TOMl\\\\\\\/FoSSjmkqTrjvjCd7NZ27aqyAYeoVebDxPjUrmZ\\\\\\\/KSHpsMTraAqYw2jlx4JwD2kY1iyt7PoBSYJuk276drrvI+HBhRc2lLXm17wPRPeo29EtAHO57rVcQnJYdNa0CR3NMCua02IvGh8kZQ4af9D0AheyN18dljHBY21SMyeY0DcHU36UggWIIHjy59m7gnhpDP4BDmWpsvgZBV2IGRXyMfhL4Q9vZqTC1tIVhDl4fbczw2jWPWr3Vt3rDGwLqTd\\\\\\\/4rwMF08sbBv0w5w8gAS\\\\\\\/ZEvh4KwyXNq+sKb3hjxKOviFERU+gWSnwhP3XhKa3b0wyNdT7O+VXcRSoA0nLC+Tu9JqvGKrmQE9xochIxFl3VhYzp3du9OXmDCKTDlA2ESpTgiGVbHqljhY3u\\\\\\\/XteVVSgPpaIAR2pGm0PIJzmRkEN3WK\\\\\\\/Gs0v4UB4\\\\\\\/5W2zYrq2SlP\\\\\\\/2ZE7DzTDqr6Jy2bi\\\\\\\/+OJf8j6VEUpOyeFFWrKoibokgdD2qUBnsfSR7O\\\\\\\/6llYGEppGzcLZ8hXMTbo9hRHuDw09qqq4NxlXnmY9p2nSaWnUrnVQZLFb5hhD0fmHNEf117Q10SuKay\\\\\\\/edWRr\\\\\\\/R5qFfikQw8oFNZwqGuU2rMCqydpds25DlLP5qUpgI6AsumDq7QM0wSebRqVwxNsRWtCq14YPrWNEJZDp3cgmUdy7ac0iO3KSKBDUx6Chs2X6zyjp8tAEmrBp4nm6foPlIJplSXAK2TYE9LBbfPeRIw2nuhZkaGQ34CbplHs73sj3tQhEQpUTeMU2r8gI7R2AsZ7PA6NrvKJM1jM\\\\\\\/cfvmCYzb4z4gpCufS1UpmRdI\\\\\\\/KABm9gnClGn4jLhUTsbX204wErvV3B5qD5mM\\\\\\\/M49TZRHr0b\\\\\\\/USuVU6XTBwGmkncPiunbbvrbKzAkh3PjNOTvSHdzfSyCLTUchjs0gFkY8MT+24xfpnbPSiYdrrE5BOceU5Uf2dV7q\\\\\\\/8YHo6GywsVGUGz9XRuUzYvkocnGFPo5AXwEac\\\\\\\/xTHacvrPPK1sL2vu6vB+fVjfGmLmls2d8Ki+PsmD4wCAAJYyYk+lPs1JzwI75KEdWOBAl9dFSU9YslEqBRrK3+lRpUpJdrDMKPoyEDHcGYrlpqSXHPMD+mC9ehHIJ4ZhNd+Ps\\\\\\\/0sP70dHxCsVgTr7YMO5GUKSJ6zFFM+wiCTyI053zz93QoYj2HduFPXUAYRkl636vKL2NBQuJNHcvJjwkCTjVE8OmFlI1OIaoz3WPLO+OOPpcXo1yfrRnEZSAjQ50X7Ind7nEehuXSOM+YK4gWa94WEY5hig7sybg1VIE+Ive8HVRsbZeETyLtfLI4WBDnVguSmQKs1\\\\\\\/4B4UBTdXmIy+JsogGU7rY5Mwmv+uaq23xA7HO+2ZpIdiGnAEARuuBy3v8zscTzDXw4cvu4\\\\\\\/8UJgasLUCUyHd9EMEGmOWzlHpgA0pQMaMQKvN4f36+Zhm8yWu0XfLe7wujdDvASntPXuPIdqBN6v7GwSYgGe1ymyxbj2m0vXOQbdzxI1Apyg4gH018vWeULquWuJ\\\\\\\/2jTfiXV97GtWEkjaVkfboXJQ3O3+RLhEi7z+1GPwE7HRDz18fDJU\\\\\\\/hpPkqCixJj9w07WGaCGfhR1viVbXkyvMO9jSzWCEjvWr4HwxCTxF7rNDMjuJE0ka3s4TMEUkGoE\\\\\\\/gGEFWwKn1SUthkceFpvGVxUbt4Q1qIGylG8tvaFE3dCUiKiWou5UkTMi1xRsSV4rq82m\\\\\\\/Nc+CI2Ddlwx7jIoIPgfZCJtuBy1fo34xRFqJClC2BT4J3+30UHchh3qgWvRrRMhWcfmuFeoKRlaoDCt1SWey1hUXv4+MjJfkSqXK6yppOiq3Wu+ljyI8ujLcIu\\\\\\\/AfzG6sMr+yzS8KnsRVR\\\\\\\/3MbPa5lSOzVB0ac94maLfv187EVXUUd2jKRQ3baaU\\\\\\\/IBKA6ibfpPtoXxeLmqgn+TLtRIvU1pjoZ6P6KBGuB8wlKqoqNPeMZYtnwfivFgwFrDZdNezeE69OArH1QnIeemwRFeqim24r3l\\\\\\\/1Xb5e+PYdEmyNo4glikuD5LhAsapJ3LJ1TiazoFQW5VDs+98ywRPLF9HHVyl2HWf0kPSW1nPjW636eyR5XQNV2l6IwESLI0T2A39r+ywl3nb8nCx3acQMorrTQC9C0kQ5NAXJsAGlO3bAbfaXwKs46nz0jXMExTxayzutMkZuVF2Bvke0R+aMS3zeAFJgCH5aXQK4SIWllidYVHF+UdqzAS3aWDocuFV+R6cm8A3Iig1l11Vl6QABF4pATudJqTutUO2ge20xIz9SiMx9ReriqqX2Ed5cGC+P90TVLCQ68rgiu9YEe3O8Nr0AfH1Z4N03d+OSAxVCQI\\\\\\\/rzWkuRI25REZfq\\\\\\\/kK7wq6jZaCJAPujzKUG04ChUw2tKmg3fg0LK4ln9PRLU2ryvsBJvfZi\\\\\\\/Q1MMuj7kb1VJfwEyrKH3w0G58SQZCyQDoQGZYNIF3VU2pqI22DEWuUUlFWdQJKJtPiUCtvBsMfbD7DiDCvE827MHTYx7omJSkTF\\\\\\\/V1I3nRBsaktGpkv3qNLhBFXc\\\\\\\/WaGvZpNCF+N4T99O0GXVGS4ATsbKR5d3NlZaIEAZx+HNhT063uiuQ2pH13WP+OSliGysuCBUfkQLF+YK\\\\\\\/eM0OK5xYHJ8ibFgIiclMhmOfCB0OtwMvuFQ9kcTHhDELpl3yNxpNleAxwagzKYIVjMw0a0Zy3PhYMshjKUGHURXq1jBy8wUvtrEAtvgnU7DXtIPPaCc4GwfoEkaAEkZBbTMTeuXynFdwF1OiaO8CAqAAT5nWqkkSa\\\\\\\/ZzIv9SYfpLpIuHiOkTR2gjwBGy0KEJkYM8TvCtK\\\\\\\/IqQWcnf8CWtlIZFpcqsLzZt4aXja2A14mXVFjDt6Ksg6mhwGKjDA+EprHoE8v+ByJpVEsqJ6V13ouuQTvJskVhwRdWoY0R+W9zcvkTRKJklBI1BJfuU+kzYdDPnJDaBWe79sTuqvJcNZ4pRZ+93K59nchMqQq68ridOVy35NOnypjFtlxggfn\\\\\\\/rNn1CzaL1CZBB9PuFhaA16zYL1es6va7eQ\\\\\\\/3bJ70QKL1ikuwCtDv\\\\\\\/JTyGYnZDYQ7zcTfmXMwSHmZN9tV6FfsFKlpHJmlAiki\\\\\\\/UtnsE4Ip8xq0HC\\\\\\\/AE8B\\\\\\\/EcykTMF3bnTLixt4qTD97vMgKBawAQsbSF165V7rWNSczipqqwx+N2uZBbxUTZbwUBa2yRhN0QWGhRvS\\\\\\\/vNyIJNkelwVMSfyfqrsuiQNM9LYMKfyZH0R7XFVVbV27u0acSCi0g1tvvvOuFhISQQLxs4tp20v9J8yC3S0TQgJ17tWnULb0MTfqGDU4LnAv0wdVGuGYGJ9RkQt0kSUsyDE7z2r4K1ae7mrKAPu3hZC8o9ltwB5Ijh4XEwkpJdZr2TIcN5XueFtOFiWA0W6YYd+BGz+JJudOpVcXDcfz+orPg2Zs5OrBUicjLKDOZR7TMybp4N+MSRIHJVupOW1wk8eTV0+TwlXuk6Ih4ei2dQzzyBQhi7WThGU9+5gcNjhEQZcdS6PAYCKf3z6D5GQQvpsh0jnqKQdvz1OWmLqW5BUiNBuZ9suZ9DHjslLjp63Bkz+SZOVWnKsgUU1XxdvfX0NhfzEnDk4WB\\\\\\\/EIFgueA8PdCJfYd8vxX\\\\\\\/qBkOqx\\\\\\\/C0q1DZHpqAHRn8o70zrHi4q9O2+OBubeu8IP3NBPlTLaW5F\\\\\\\/T8kdeYYjjh7II6C8eR5nNqhsXp9j0awhPlnNDC0JdaDHFkGa1fqJw\\\\\\\/MG0J8FDMdwNbDkK7DQOxl3WmgOVaMUoQ\\\\\\\/rfSYvS5B70qbD1V\\\\\\\/RcyIA4yVR9zpyFQDGDYwPtL\\\\\\\/Tta2R1iXGTN\\\\\\\/YU3CU8BuhwI4IGMd5RSuZOpZmwsK79ROOEB1A535KayOQuBOitI0SWWZbAlPaJQoqQdGbSAgkXXCHL7bV7Dr390d9jgLgVS3ZrupzCzOsh8mPckHQv+CN0ODEpUSTEpgLE1kvy8bEhapLoLs5KUFGR3NWR2Q7Z4WH3e7XcdLF8SJJZbNrlgI0lJuySzijcWbd27CGjuV8vsDzhppWaxUxJz9Cey+36DZyggq1RKJY6nSaqNrhoyfk6iYps7zfCA5xustHsEguP3P1TjaxSBIRj\\\\\\\/dEmCAFmYaT+qoU1s9NmFJjwaRK4UsIGeeUbj6FPv7sacIMiindVxYumHlFEdnwe137iPNunAYlI03rn6mhxDnrDi2U4HyPR7WahBI2Aa+qnH+BWoe4Zwz5qffxAKtm66dI1\\\\\\\/uyJ8D63nRJ3rZAzfxW7u9tgxnR2ntu\\\\\\\/Q2+8aMqYKMidc8k59wzbg27QSDNdMVini2BsIhfAGBq0Mhk+hKZBQKmGZNOQ\\\\\\\/wpnt93K05wB3+iYhT++HeMkvdI\\\\\\\/QwAXHcdsohbseDJVhHZ7dLuWcHQkI2oFBNTu8gl6vgq\\\\\\\/HkLFq9BmZPheHhtPE1GZguL+nnXFSStBI6gfWWfs25lQpkjQ3v34N\\\\\\\/3lGPYLdkC9ZfWRHiiFKfjEe3GTaPLpHk3ZAoMWGuVH7WF68ZBFulAb9C8N7RG5z0mrKvGOwSyyJ5HkCAjc6PCK7lZvsMP1bkULrXsMZtr4FWBUqZ3AIaKuHCMvfuY+8xTZZqPknXxqMWpLrrgUcjBJ\\\\\\\/Fbwbpb0hAfNtLNT7oKXq4oKSgYWsR1TZcLLAZpXwEZFqdhEiO7OrgrIjoU9vHVI4F0u7SsAs4qPw7BrV\\\\\\\/X2pKwDW4zVVHEY\\\\\\\/UsGGaGeDeXNCEuT5M1NclyWzLPvvlnTumjY\\\\\\\/AdCBpvPIe+ZFDTVQDFjpHxblIVluWte+0gMRvqOBT7LDBjMUkvFPYL5+8xwXFEevmjF8gaBTrc7YeifgVYd1MzjsKGSxmaaV+agZ68Do3ojEvVQvE1hwvKPgDsyxMv5eQGwoRzZKHt176UjJgq0zWhoIfZSC3rpiYVOZJ6KJbZxMAi5jL7pACKMkmFihJ5lJShoXGfTb3e51uam1vE1ZPpG5dn1hjdBA62Xo2iZOWIaiBujFt+zhWbmCzUanSdsmRIdjEMN7llwTI7bn3nWE9anu\\\\\\\/0pJ+cNM1sW9LQkxQdtXo2jcE6NLo4iDbr3mfPKi\\\\\\\/0fFz83R+wChvS4GZk55boN55RfywiW7RhvQHY1SbvnZOiEBb61z1sChip6vIl3lgGHYOSFvsscWAPm4HKfXlA6B4cnLm1Roc5r1aNwjsxtqSjtlP6b0U6HTwWni9Af9CLHSXvoJ+o+Kf7e+zkXWZCLLH1Cshz6RAKFPg2MaUChEQw9Sgm9fz91BUy9FNWnyUMTBl8\\\\\\\/g9h68IwJLkBxs8jYMNlVAYoRT\\\\\\\/gz\\\\\\\/ECeluVZESdSJeTvBKhp35R9y0\\\\\\\/gCyop14FRzj5J0o9CdA\\\\\\\/7q\\\\\\\/ns5mkEpz+M+s1BaQb6dA21yZf7Imz8gE7AD3fzFF2lfS92Zqmi4fIkh2FyFRuvWomNrw9\\\\\\\/D4Oh7YZDK8OQKe7J0vY2regbYXzmgiFb18t0tCuwlAZdwuLTPs27qLLTnzCVT7ibjOJyzjzxhjtOHtExlH6hGUR95E7o3Z0lfsAeXozDNVWkTTxMjNaghkH5QNiOgbuQrRihqAKaEodhCrxXvW1W107SsBsxav3cNFJIARDRf02QO2zSXhy9nM686PK88LuPwcybgJWxhPG0qItF\\\\\\\/K3iut5X6LR59zKiGJ3on+f1b6lIJ4m\\\\\\\/KlvQ9BfZV0qS\\\\\\\/cOqPO8Yqr9z5ivdvnJee9Ja4rJ27L9zqscWSfuaNuENb\\\\\\\/pxaBDJCq091Ph03pbZK7edqpUaqopvIokxRGGKMApGRYBxKUoGp6tL\\\\\\\/VGIEF4DNK5lg3XBokVv6AttAiqhCNbtUxO1mZRqY98vDWJ6\\\\\\\/qmP\\\\\\\/cIqeejgqRt1jFm+ojhjLeod0LnQAwiWaLzU24ta5mDD02hsRYEkgp6ZGZ388BJyohhcY5efI\\\\\\\/oGywrMVfhVMvbEAuKqbzrLKkCh6kHJnh5tBRcudcL3jvwgbfT286A3ee2zPlWum5OsIaNA6WIPeKUwWq+Xxy14AFl\\\\\\\/l1Iexh+jFzEvck6YzmpxJvtKKOXUCUGlLgaHVykG+a0JIeEG\\\\\\\/FfyPgjk1h5ku9YljjFT3OtUaYI0op7veCtwRi63y2JZXQnHD6q9NZNwJ6r+Tjxalzpq5cgelYCWOgfG3ON9qL\\\\\\\/tgh0j87GHkyOed+s+qycrZLA0hx+1LBlyAx\\\\\\\/DU01hmabI7or215h2\\\\\\\/VZJkZpvL6fqQ\\\\\\\/vqeNreCRLSTAA+1G5nnTAfv5fcS4X9E7WaKU+dN0G8dSaU70MxbpB\\\\\\\/\\\\\\\/lJJ\\\\\\\/j5Edrc1GyzsXrkIwc1OvyCWASzj\\\\\\\/Adabcvjn8YRESxqWSwnRW1ayFe9Mylp687tQ8p4VZkBqeh01Q8KfnMtVJouSJpRrk40JfuF8w9S1CA34TGM94MzUEzJJuzu2oVOLtppbcO49D3hl\\\\\\\/DeTU1wmKbKygx5x2xcFjS69ezopgdNF4C6BWa00ESaQodWcyaGYggKVS+bEutFqBoQWDzLzcziWJfJddxTKh7bDc4rKS0Ocf+ZrQjcB0WcJRGt6Pya5Vo1OPKJ6gK1y5Ml0BkfeQhgT41Ij9oRtTlvjb5+59RS9gtinVQqB+HVFs7N524lJcCdq9iOsUpaGiS1AwYxr7EtuCHCPvlYwCAcfIcmY0y8NPEXTcb\\\\\\\/DJ4yJNykdo6PJdq23NdNLL2uAxa0n9xnQzkMc3wEMGnDLd18ZHsA\\\\\\\/JK2ujLqz0ulX3QNSHoBcU2lQVdyjE9FqdaOkC2MyZ6S\\\\\\\/Xwga8rcxCYEUEyQjOYZYnVlJm\\\\\\\/nEYyoKxoAL7u+Xme8qZ14O43w8yJe6TVN\\\\\\\/AcrCnkzFLoUD2bxDMNC+1MxKwqkOlb\\\\\\\/UnrNR8nehneHyk2+AvST8uecD0DdD\\\\\\\/h+Twba26ay\\\\\\\/GLvOcqVJDVe0AO3Lb2qIVIoxABqFDU5OOeIdbYOVs0kSwbpRtw8ry3tnsmhZ2WNU3Ac5qDhKApNTcmxty3p1OhsFWmOMqMRXtEwYMbCxURCOJW01xV7vPQAFCWCIWdCzVBsfOuweQEKv88CDlYA+aL9efnpnk1lRwX4OI5cS6vr104tMyV7Yu1NtesO9Kh4akyqyIYBnDWKAw47lA36OOIj9PQEfhUiGyB+jg\\\\\\\/S0jBsKj3B3wxs7WaokZFYz9P7+Oas78H\\\\\\\/yTYLXRMSJ0jKzP6WF+DGZLE\\\\\\\/F2EUxZscnCq4t7UXydXMi+A4xGY+ZyKTg6k4HqWNousaN7nN2mQe9GpLuZJ8wvPuL116DLqTvRucAgX11R\\\\\\\/UsKBhS0USzHHBeGoLp7hN5\\\\\\\/G4bOwxHi1gSteROD+HLwtCA0lIDELzd2Quf+fuXn6pt3y3JFJZTJWErfZBz2Q8rlLF07xp8Hi\\\\\\\/92JJA3AUL3Tx0ig0yColi3McAl2wOHOpRNUGufNpN3UgobbVfGTB+JVo66Xvo+93RcfFuf5xZO1RSFwL5S4HwIPZa51NKU1+vwdkh9qRHNgGF\\\\\\\/NtsLWVthln2hUPPInrIHcCNzv4yo1QeoyqEUGSuKd0dCeGKN2GdNxoDbLQBLnw\\\\\\\/kiXQjn2KUAz9pTZQT\\\\\\\/XZ0DQ0ELyi4jnZP9kraRdJj1UUWt2mnUQx8X+Pc1TRD+qDiI1xqpOrz1cJfe51KB642beVFGKo6W3h1ZyqFUBxHxbAp6kveLveH9VIIpftFgasRro8IHc\\\\\\\/GxBaRFevl\\\\\\\/row6BfpI9jQlzucPII1\\\\\\\/kHM6rNmNOeeoesn8bpyUHRnAc3PH\\\\\\\/3H3BW1+cvUNSXxYd5G4cfnvIb+CbtPkHVfFmrrcbVblMOSLdZfoReCEENzlBnpT2tUSd0rUM3KXCh4sJw6cV91R3UC8KQHt3JLkrqseafIexB+dDcg8I547peeTsKxqXyiJAAu5fhDCMXQibZ2FROfwMUm5OWEfhdbEC86DT5MnT+VSO0MEcGnaH\\\\\\\/LPXwF\\\\\\\/iXq2Z33Roebdrrbt7RzHpMwy4n+Ag0f40wYkn2Lo3OWStoipC36bkKtaSLa3WBY22JaVDK\\\\\\\/9jiIDUTWKB5yba\\\\\\\/rpeTfRLeGEsHcey6fT1s51T4+X9JON1OWXUbHxF0HYpd++IWyW8ygv8XLdBRjjBPN52tWnpb5MAuz+vXFmC6TXMxGLf+QC9z82nbyykhL3VkLPy1AJmk+fUPRrJoWoYZfgfFBdt6fkW2xmMPy5Q3KkJFvyf1eOQ7dcvCFXH1oO2Si\\\\\\\/UJ\\\\\\\/b4twyQnywOf+QbQHRHfQy2qt4zvIyRe042cUH84t2of4aGQn3VjaZEU6dHxUepdIdOpC9Jg7wFaPM8K7yrd+wY3UkfNkVyuRlms4mh0h8zoCemVqmmAu4eb2m4dVLEYhH8bNtJrqZfXY4Vr3f8jgHQkJPnDQvJyrz1\\\\\\\/pPRAkwOmCvfaVfnfdTI2Ee9by0pMWHYw6h+h3mo25M3xbjSM2FhYMRc4YmXjSH+AX0HW87EEWNpKf9hf5rVGC3oaJSepFgh3C2zpHRbZ7fbKbH9jFIAhGrQpTKVao\\\\\\\/Iumvj2gQ1xb33mqVwWhrEDo5ZZ0IBHhs50L4h+tPzspb5SQxwGWC464oUhO92h2vOzO\\\\\\\/RXwbvsoDYVWMs9O1rdbZu6YnpuMGFXZnJT3bdL396ZvMVsbLoXMzOEKpDNLpSVjHiDCAWJAB+Uim03NEhoZPLagfm5bkiAQI3doMfMYfrRqE0Zl4aQAji8\\\\\\\/1eP\\\\\\\/tZb59NrS5oMX2WkXlLCcaH7PShfVPNOumPgY8JR+UmZT+gmGjhatLCvBq\\\\\\\/WaQfrG9LRCcxHZJpHkcdoVlwlSv8Si11g9OVrE4DH4oAiU4z0XdbVC\\\\\\\/k7nDmRlzyoC8DjKTdO0IrAFXXgqRbLvWzFsQi5Aycpt6P37n4wTKX1+4gtXffpB9o0KtTU2Yd40dhBs4DPb3q2NMyaptrqPV6iGOjYAkAtdi+9KMTOSBfF\\\\\\\/Gx1Si3EIUOCXKNcp0nrh+aJogBlFf9vy4kUrmbI2l3LKb0euokj7+lWueXy5PNzo39LHoa1RpZfr2x8pJHSawHEnopHEfivc6Grq\\\\\\\/Wa7W6ulL0+NbwerGEu7o0921Eg5o5ltreDpqG2yA5EGsx7WCzI\\\\\\\/zUqIQNzbY92xeI9QKFYCXDdnP8DOaqN7gGyGHQcBRQO8Q+Qt4W2X4JKdNBq76j7sgXCxKK8vA0qtQuttuNJx1iaKS1ct7ZJM2bZenetwuLaiG57L9nhJqh0hN9\\\\\\\/W4Tk+cq\\\\\\\/1nJeNxesc6KGeWr7dF+2wNXsxpTKJYIaRR5E+bDBNvwdqHKZsEl4\\\\\\\/DtM3eg\\\\\\\/iqdkz32TGKaNz99nFlGIiLo3a+JCKcbHvbOfYY5cQhyUBviGbg+OpoYBFdcsj3zV3j\\\\\\\/2IgT7xC9b8gyvsdKd+DhsF9jaFmoGeIbc1ZRUjYc9maJybAN4Bf\\\\\\\/AcKJYnoQZ3kSGWkKrmO2PFLejAHzcot7ZL1W2oiG1N6y8PNMwoDpKXsdbdRYzPwi6mfe49NhP4pnRTelGY9nclL0BixNaCIHfkOzMSbwM481S2AsIw3vNTGeDssKG\\\\\\\/H+EsCEWpOoDq14SqNsAYj3H8N6wszof93YMTcLjMqMN7YnnSr51N8j\\\\\\\/U1BwELbzASGMoxvDFe\\\\\\\/VrkXmqV3XnoCQOTMzXwWrcjD6ZTu7WkYnaDgqK1twd9Wrai6XyWqueLG9OTIOhql4CV0hjDB0oxMRLLopvjevQsoFHyaet\\\\\\\/WpfgZ2ezdeV4Rl8YvKxqWtiQCKzt034SnkMojiZqM\\\\\\\/IoQiFX6I8wgFql+dA1KCyAQGek\\\\\\\/6MBzTW4aFROvyCmzQh4oQb4GPzUoxOo63hYJjwsmsFDRwRrmeJV8bLn2\\\\\\\/5T\\\\\\\/DQ8NsH7Kcs9cID63ONMd6fXy28XXlEIY1HGhk1h17H+6qmuqG7VmN4Eopo\\\\\\\/SN+9FA3m1Hvw7Sa97iOcs8gy0QFhr\\\\\\\/VVRwzejLNxkxMrj4hAeRp\\\\\\\/662dT6mjWhxaceGOIIKbJNvEV17hUTzQ0w0he7YzjqHjv0hgH3AjZMrlyP9e2r08kNRVITGo3VNLxceEpKgZqAyIq8uhKez\\\\\\\/C8w84b2ROrQ6GLPtqM\\\\\\\/GCqTBwVUnIyfJlvJwG+DQmVCSAXingWYh50rGlTb4w3+JEy2w6z4gU5vLYR8HFg1KEQR9Yu\\\\\\\/VP4QPTohMcOj\\\\\\\/5DKjSBRBSqKEe1V2lfFMJ5OO2XJ0ehTXO+oZfuo8YdiS2c\\\\\\\/YeBt9HSgfS+3U\\\\\\\/3gHDZX8e\\\\\\\/92F41kRdzl\\\\\\\/FoI9F0+ktRemaLPvstnQVm+e9CN8nQkJvezy87709WM8nIJynfmteUWu2tfBwv2wkhNTMLXd4EB5hnHHv+Jjk4RHlrgqvT0SIGPhu\\\\\\\/+e3xvKd\\\\\\\/10rBsLYCN6nLUtsKDz0\\\\\\\/5yAM15jecjpEpZzElskrn89Bw6x2h50vQp8zBbCGJshVtyazGCFckrSFON+xjkOrCu16Tp\\\\\\\/rGkwJYKmS5Y2PmrZ+J\\\\\\\/AAqbR+u98NNkxJPfra71N3ria\\\\\\\/lzna71A2PuG4hjT9QoXspZZFkktwbtgQxyCtmB8MoisfEq5O2EL76l6lmsUEvEh+uZ44kn7fFetic9j16Zm9jXdN8KfkEafltgzUurPK2YvkAKF2Oi5MjsGlVrfPxMv4shEmwPgfgLOsAlR9iMtaZYygReftkBZ5v511v0L0orsS9RQeHOG9FCgBLPHsvYzSPhZ3ubdh+4V6l1c2IxhyrRQ+ot9dIg8QaU0Y\\\\\\\/6hkyeeQtVcbiEHp6\\\\\\\/onbzNVIQ2vBJAhrFN8G2gSuvYwaE7p4AEEIUsp6LsDdMHO7o4\\\\\\\/Kj1Mc4BPz6JcLPLRDI2zlNcl7DhhBF2QzRwK19zbF10cytkLyeNwTDyOaKRSV7TtI5xY65ZDy5pSlsXje2ZIZA2ILuQyBL6UzgMFnCDNv9VgJY+st6polRN0GrSnohAfF1NiXvCbDTlv73FjtmypUkZ5tRNkrITGJ9O2jAOK0sc+gFOHHwS6ws6\\\\\\\/66MZMzbMXA6eeWCwtEgL0BH3YsId55esFN+f50LwfLl0aQVbELlIeSRv4KRKaXfKyi0jyzPMDdLr9oDZKHmUGTCm5F8Ottw2G647N6gDvNUVPO9MFstzrM6chhGIfSlldRP8P5nASznGVAUSQ+RQ4h1qy7whfFgLMWMQgo1iRV6ocOFvXUOjicFcwR0Gsd\\\\\\\/lEtJfwf0x96gTi71wvhgowCHHlLKqHeJOinQrABulN\\\\\\\/jdd\\\\\\\/Q4IHKSJe7cuh0KLgWryknchypSY3Ex+gFqYW6YFxgmyBfcJVRlZkHv\\\\\\\/kIRIW8ACXjlggfQcQ6ONGmX56d+F0+XABFDHIeZmNUS7JP16DICunbmXJRHPLaW\\\\\\\/TZBWFUYxlGxEw1qH6S9oAznaqU9pwRP6xF0x5y4HkT23Ytbd6a7TC\\\\\\\/wk3uq9pwvSRhWV9xIQdfEEDthV7ZPrpFpEWpHskqB1EcopjaYe3XvEqh3OptPGmO5zuIbdmFO8DtSqKfL14DdjrtAfd+PNQXGC0oMmZYJmzkrkrk7ZGfkhCsiihP\\\\\\\/gY1kieFPFOrnxPXll62My+sdglQYB2e\\\\\\\/VKAnATLBL5PKqa2MdExnfq3nkh8g0Fp9CKB0qcfivLJWTTWmfJgyu3Dh\\\\\\\/3wDxumw6vhI3himW2nYV\\\\\\\/HFWvnoOgpgj78A+tLnvetmWXF5AqSZp7lExuBbgKlXW756bhkIuw+BxV7Sb3rvfrClmoOPTogzEYAXLfWNIqCckZWregxRsQlW9YVrIF9RAtz2M3RKgW41UnWVVuyUiFZGM+GJr\\\\\\\/pV2eFirDsaUWhEVfqWdCgh6t7ArWKmuVdUIJE8CKvyVCjlE31MBsCmT8R6mS\\\\\\\/FmA5q5SLp\\\\\\\/MaUoyhy\\\\\\\/w2tE2DqMPGOyWPx158yYW3Z4Z9OuZL48jV9tm7RJbXgV3HbV6QmF\\\\\\\/tg2mEwyRtTVkK3loura7+\\\\\\\/cdDHdDUqjoOXTIuBpFRQIRmSfzgaTklasAUG594OAZ\\\\\\\/GU3gupaF1Fc9ZbSNo\\\\\\\/WrwVzxFYc5fDiZDD\\\\\\\/HMFPDgpmj9M9uQ3Lw3gjlAhjs+mVH7BgCPw0Z352a9J+79P3w1pOWoGXeyIWRY71zbsmEIguXb1rEHntRLLg9Im2wXFBNnzpoFCbFg7H1ljK0l2eFr2BxEJhY9DRyIohU5uXNwcIh\\\\\\\/IrqhGdbIA32B1AGg8RVHB\\\\\\\/ErV\\\\\\\/IJHqkvwz9nKIU8SLpR0bSJ64OGiudoeSbZx1jWTlopVwxeBlcNpaXI8DJ3v+TWhsPer3EGMQtQ6hb\\\\\\\/2cAvCrrUq\\\\\\\/biN6qMZJo5w2GXKDdbVtc3UR83JAYVkP0DP91ivS69e0ExXFQJ2lzDPrxoFEw7GGcQGDxe5XvqBvn4ZTLUBlEF9DvWtoT97JXBcoG8biFx6tymi089OG06OrBivig6qcg2j67jBcXJVS0kE67uf5rGmo3KqBm\\\\\\\/UlMJwsa2W3a6AP1UFp\\\\\\\/Lj+b7NWmvYF7T9fE3K4cu5uIbGcp5dh9M4PSNT2U2DWUYwXIv9EOdz+b+kYKDuBZnD7eiDXpWir3Sp51qYMl9ZAnW0saUGWFtdEuZi8ucrhndkTU3M31VW3fdDVFkVcCNs+ShhvWMj+UhFD+h7p0YpYYRfvzg13v7ff8B2lA+JW6VTU8KGZhAN3jCTxjxszLBxMMZfKRmOtHdeV40M91\\\\\\\/NdRonrPEsjWtM3ysvvKdyCUeZ9EEine6C31VqM0IT40W0tQi6fDJs0mDFmncfOIMb7W4x8XKVXQ8UZSgaPQZFsk3jF0kpv+nEFIy2arz2lqcBn3SzFxCgahBMUbuayIdjU4vTFOfqbezC2tzd7Sm+rTX3TUHCvYYOVnE8g8Exrn3JA6FOzqSQoMFTXxHTNxMV3CS2jOFSzQAYvzWmJeTVErv8rBqIfOJDPTt\\\\\\\/YCCGYA+YXCJ5AtN\\\\\\\/l\\\\\\\/nADImTuBzT51IQuXaizKNg+MNLXY0d5klA08iaVI3fr0bzgsz7QwheY3EeqvbkuTwItSs6NX+5N9c8Z9FwgOIsg4XNJUBgNU7hkeJhVLAScvEmaLZrd0e9jk2wdqslk6BP9bcSm0U2YZCPp2zlYiG4P0RDVkEUYjVksn5MZaU\\\\\\\/qUWcVTHFb9uCZ5AdPibU2NPHWnB7KGZYuBNES9fRgu\\\\\\\/maBcV+wzEDYS8tiVnSsM2UDLUxGuDOD6PcaJyaZ42I8JDw25ckxoG6ilOQVf9HR+ldL+jhJxYohatmVtr24DlSyG1vTBfQB30G\\\\\\\/rPRYJ6CNIycOBbDvml0JomvRnbCOPjgq5Rezf2AEMkb6c50htYusXpgu2tdkvVXObno7ApOiwUs7cJjzT7K3Y7BqppUTSzmmlRfxgBjeG6pHD5av96x48HpohJxbWiikWk4aPDNpS5MUJhp8+uFYwm1UDVIbf1ScCeHM7gQzWNeoTzv+ZWLuSzi8JjXhyFWaxVeRYr+Z34YNJeMKfmZkajTFL\\\\\\\/sbDuq3+QMjiHh0lWA9AOxBbj1gf0vC3snJXHyGg3+HSd1OjiddzN6UIQkxqLDtNO3gRmieRkdbD24D7iAs91M6tIEXOXIMGxFsRJsv9ckI2CMfKcT0c043thams8EbRUNqgA30gYC1CpDbwtyBV649oX+9p8HDJgV93nF+YGe7ykRExqbcIgbkxBHgWCSYmmenSvWn14PHj\\\\\\\/2FT\\\\\\\/Fi\\\\\\\/acsO16wvNXbnd7ur7Yw035BVP+oR4+P2TdtMY4pPOQ1Rg\\\\\\\/Yt1meWjXywC5uX\\\\\\\/LQJyY2F03L\\\\\\\/xgD\\\\\\\/PN88A9NpXx6VH3R1Y\\\\\\\/bY0o4I2dFkD6wB5EEMZ1YwQ1M6CnSU8RyLB5fm3UGz4oh0WDarVT4i6A2oA4jkX0n1SxJaHgALBt4LWi+FE8ORa8MRIcsyZANgCAEry9xvHIe0yf4a3GcdJovskErshm36I+d4zP1UoUOIgsKykmfF8fO8o6++ZrqcNPDRFNx+RaFGDxVUkoEFRVr2gLlmL8RNrroApqjyXtLMowaOmtt6tArrSTvk4LS0kgu3XDdiZkHMGBm80+Ra0aglBLBWT9nxDx08e2TdyuVV7gqHbcRhxgz5LbtbHDBrdEmVfEm\\\\\\\/6Ha4uoZahM01iquSWvnWIpd+LVVRYi3WLb9Sot9yLt5DgSLjqx+H7Eql03g0wIJ6qc8+umL\\\\\\\/Me1+gdPHRJ6zlWKv9ePFUJgLO1B54qoxotlbQz5iBDv17dtc3k+S0CZW9Fpc87MXOZhnilnmqfi8L5HYwxU3Me6bVwCWOhC+xFsV4X6SqQfQaLhyZu\\\\\\\/N1eQf9DP5Y5\\\\\\\/vJq7njLRpOY8jrj7G6eJu5wuRDd0NxsD0CqeKrnJw06M+Sl6Nb+BJGQQe6LJjBX19H7KDKhLzaca8jhPzPvmLsIP0eRleUjsYmSbyLhA6scPlbqNIVwYDQEaEJ8j6iQbQI5RqjbZzlynLpXpdVvtS7w4UIUlh6UO+IjJ8m3MX\\\\\\\/sCY2K6dkMYSB8Q88HyhZYpap4KGJvNB5K3u0gGVE4FEAbio2MPTOZSTKS4kBNVbEhm747\\\\\\\/cKuZmczDIvPHkrUeMrlPaxvj2tSDDboyZmoH2bCGmh\\\\\\\/sAxYeB9P5N+N9Ae2SYSj89+BEkLvc4yY0\\\\\\\/Wt9tRszQs5DMn3by\\\\\\\/d\\\\\\\/yn1o0REU9tafq8mbZCat3jUC+DkvNjRrjqQTg6p0ZsE\\\\\\\/mzcpCpvIJuz3ZFH1Yb9sLv3wxhEfDTC0SrqLlcJ5aP5XcJmF8N9GeHwtiINrFIcEPS1vpzPNJKrjmD0PlmRuM4dG8jEBrek\\\\\\\/ogxQ3j8bMqqn41ROPCAR7QQ2SZpwFBwTftZRYCBNfanJj0XYdJv66qB+JIMta31SypzJx+FCAJnBzf7kvZTiSeRquqg3gY18xvDrW4KyClO52JCyR1vi\\\\\\\/ccYrqfWMxNK007DyoLC3aZ5DhMD9+6cMHUnimnhuUPq8gODEFGyr4oRTS8QQEDWCxdl1zLlg4EYhqSEE4TLS4eIIIl1d+Jn02nMJ0M6ZOiRSA23MZvxQxczKyRq8ed9ukIT43is6ep1h9fljtvJVh4H2gUSVilhAJw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022cZdge+KSg00Zkt5VBnvmQQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00228452a9e03937c0c000bfb510bb4c93a298f62815e8a1a9e8e39931851feca49e\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022MXS1AiScO+0tla6KsKmnEshtb84hXPaxw46kqG6Gs9zL4PtYsLHqFWsS3c+kBRV1BV\\\\\\\/Wfx1DJ5kbpF+K9bHG6jXvgo5JcLN70T5\\\\\\\/OwiKOEJpbIK8jw\\\\\\\/lm3VSxT4diPIyzi3rAOlCAEOxJo2YZrlpmwfzxoAUGavKnHxylCNgIOD9BkKNAuAQEIffOwA+nfRctj63+gE9Zsr3sfWZbTItVxzZKUL\\\\\\\/C4U+q+LnAcBEntPw4mTdy3dXPdeYPl1aFbOnUDUnxI1+oC61lNGLnowFV4MnkK0qW8putnPvlZxBruL1ITWmgPr4PuidvgKQ8pCL\\\\\\\/0qjbfZvzP69ifrV\\\\\\\/L0bINBnNlZP0uP\\\\\\\/ZSz6Uf1hF+W20M29WrQKjDET37pb\\\\\\\/2qfEKdmgucEWQMaO\\\\\\\/csuM9iL2DQBBGeZnCKP4J+syjeEpn3+qNBy221WhbkEpoRSto2SdD+qeiH8EULSkHDdQzztJLyPGwd+AzTnfe+MrVw+w09Sf2twus68HbeNIQzeo7fipAAVPS5sVRfbkgS7B1pqyzz0PIExIlxNEWWukzoHDmtBzf3d5bELOMbPCLEJsAYWqDofZcWZZOSIin+M7sbbo7+gZHz9sDFNkqoNZGLlU+D43CBjNGkwvqhIy8b5n8UwR+5wtwneRiLxPThSoFmnuuoo+xAXlM3mHyc9do04ZTWAiuH3zgaPKDZBI0yMeGSB+3YejHDYU0Dkgj\\\\\\\/GauQJ7rICjInfyHsblnahZR2W0QiFBVf6wZ22\\\\\\\/SAssdiYCEn\\\\\\\/J5Z21OkbQoqcsZGtoT\\\\\\\/r9Ptpx0fCYQDvsEubGn8sTawaSOq1R\\\\\\\/uT4pr34UVZCiKL5ziMRt7+pXu\\\\\\\/9ylSrPS6uoXF08i22zxGmtdUc3mMGXpKwx9L17jhpDijVF0pWnUy+e8HY1wteNA\\\\\\\/wjutiTYKEU0KpiFucfXdXwOXP8eqpAUWWqnCtFe1DaZqLdjElvtI\\\\\\\/c8B44Oz0yyjLA8vga4aR7B7ymv5uBBVVfqL6kTjVmpSyY9UBjaNzPwU\\\\\\\/Q06yB3qxwZlGU2R1ChjJdyvUGCt2fB2CCxoaWuIW1mnB4hPKGqJECEh9STIseFeHQsTzO\\\\\\\/8MZNc1wUiHBI4385PQNGpPaHx\\\\\\\/VP63WgYCZiuWWsGkaDelh4lO0F6E\\\\\\\/zQyWMxqVpM8nP7QF6BwZczpX2KfftLtKPK84UTfHx\\\\\\\/zN3FNn2ibEKUbyl0WFSIk7UlRCmuTkxZ3mCabb2VM\\\\\\\/+AhqcQ9h3NGGqc3UWsyqhBxRvnSQniJMjrB\\\\\\\/znUN64V\\\\\\\/sJHI6Y8c3LS\\\\\\\/oywe3OF9hz2UPFJtHuEW2sfj39ZgAa3cBdOP2mVV+7gFCDzIjgIMMH9\\\\\\\/VhCOKSqHnz\\\\\\\/oQ7+Zq7g4EBcpKpcIFhCmfrDrkiC9ivilNWMuxgpMVvHHpkEof+h3vRODTjW6YiaeBbXRemGTwvgEXDnYmhkC28PYQmpTQ\\\\\\\/1At3I\\\\\\\/3diyZrycFrPpTADVBF2ZRFOVlX6crtUbIcmi8Os2dhWgSvnVQ9FbvnAWf+01\\\\\\\/Em1bf+dWaOmJRbz1qVbzYvHFdzbtn+zuEbYPbE+NeBfOq8+bwoFauycvOoH4LSy54wCJfVMgvXV5QmIg+Ntl8HEY9ppwrbf4fni8NDehtp4AtiUJeCBarYwCK\\\\\\\/GBgNPt1UXshwjQWdAUB45W6BT8JSQ3+i7jnsOFJ\\\\\\\/\\\\\\\/s8L3IaedKSGcz9y5eYAxHXYyQ5Rq3rMu6tcxbLPCWtRloPDThs2n\\\\\\\/Znwy6EHu93qjcnYZ\\\\\\\/uUaX3pQoPd\\\\\\\/aDawZzAPTM\\\\\\\/qruejuZcIz04L9Hh\\\\\\\/Kla27B6MOQybbFhN8O0Aw2Y+p9\\\\\\\/X1T5FQ1d66Mso0bR0k4XpBFA7loBqNyp8\\\\\\\/FOcnbeZ8olDTXGHrGXNXxHxuZEaLcLQQo9bkeh5CaP5WTk2xlpKrCqfa6wd3VMVleNaJ4fA\\\\\\\/YwEFWpEXFKSkf6nRvvwBo4sSA8eU\\\\\\\/DvTAmZ85dv9QhXfLDE1BFzYG+An2yf4FOfhzqTx7Id8OE4\\\\\\\/JZdYB7EMDqWtw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022ddPafutB0qxdxm6SH9zGBA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022ce93d474822f5b607d3dbffa252089603e254a74db30d13d755db830bd2aabf6\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-494610 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-genome-report">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/11/3547" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-G-8ULEDpo0CfGNORzQ0-HmRND7KGgr6htzd70HExpIs" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/11/3547" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-cRVuFEAFGpFqIrRNj0Wz4f-iqdRcVRhD1RMPkdFviMk" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/11/3537" title="Construction of High-Resolution RAD-Seq Based Linkage Map, Anchoring Reference Genome, and QTL Mapping of the Sex Chromosome in the Marine Medaka Oryzias melastigma" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/11/3555" title="Pharmaco-Genetic Screen To Uncover Actin Regulators Targeted by Prostaglandins During Drosophila Oogenesis" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494610" id="node494610" data-pisa="ggg;9/11/3547" data-pisa-master="ggg;g3.119.400486" data-apath="/ggg/9/11/3547.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Phased Diploid Genome Assemblies for Three Strains of <em>Candida albicans</em> from Oak Trees</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Jennafer A. P. Hamlin</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-1459-3148" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Guilherme B. Dias</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-5462-9854" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Casey M. Bergman</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-9015-2707" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Douda Bensasson</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3547-3554;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400486 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494610" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jennafer A. P. Hamlin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Biology and Institute of Bioinformatics, University of Georgia, Athens, GA, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jennafer%2BA.%2BP.%2BHamlin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hamlin%20JA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJennafer%2BA.%2BP.%2BHamlin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Guilherme B. Dias </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics and Institute of Bioinformatics, University of Georgia, Athens, GA, United States of America</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Guilherme%2BB.%2BDias%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dias%20GB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGuilherme%2BB.%2BDias%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-1459-3148" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Guilherme B. Dias</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Casey M. Bergman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics and Institute of Bioinformatics, University of Georgia, Athens, GA, United States of America</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Casey%2BM.%2BBergman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bergman%20CM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ACasey%2BM.%2BBergman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-5462-9854" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Casey M. Bergman</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Douda Bensasson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Biology and Institute of Bioinformatics, University of Georgia, Athens, GA, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Douda%2BBensasson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bensasson%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADouda%2BBensasson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-9015-2707" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Douda Bensasson</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:dbensasson@uga.edu" class="" data-icon-position="" data-hide-link-title="0">dbensasson@uga.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/9/11/3547" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:494610" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:494610/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/11/3547.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:494610" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:494610/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/11/3547.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:494610" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:494610/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/9/11/3547.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:494610" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:494610/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">Although normally a harmless commensal, <em>Candida albicans</em>, it is also one of the most common causes of bloodstream infections in the U.S. <em>Candida albicans</em> has long been considered an obligate commensal, however, recent studies suggest it can live outside animal hosts. Here, we have generated PacBio sequences and phased genome assemblies for three <em>C. albicans</em> strains from oak trees (NCYC 4144, NCYC 4145, and NCYC 4146). PacBio datasets are high depth (over 400 fold coverage) and more than half of the sequencing data are contained in reads longer than 15 kb. Primary assemblies showed high contiguity with several chromosomes for each strain recovered as single contigs, and greater than half of the alternative haplotype sequence was assembled in haplotigs at least 174 kb long. Using these assemblies we were able to identify structural polymorphisms, including a polymorphic inversion over 100 kb in length. These results show that phased <em>de novo</em> diploid assemblies for <em>C. albicans</em> can enable the study of genomic variation within and among strains of an important fungal pathogen.</p></div><ul class="kwd-group"><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/candida-albicans-yeast" class="hw-term hw-article-keyword hw-article-keyword-candida-albicans-yeast" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Candida albicans</em> yeast</a></li><li class="kwd"><a href="/keyword/haplotype-phasing" class="hw-term hw-article-keyword hw-article-keyword-haplotype-phasing" rel="nofollow">haplotype phasing</a></li><li class="kwd"><a href="/keyword/smrt-sequencing" class="hw-term hw-article-keyword hw-article-keyword-smrt-sequencing" rel="nofollow">SMRT sequencing</a></li></ul><p id="p-4">The fungus <em>Candida albicans</em> is found in the healthy human gut flora as a lifelong, harmless commensal (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Odds <em>et al.</em> 2007</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Barnett 2008</a>). However, under certain circumstances, <em>C. albicans</em> can cause human yeast infections ranging from superficial infections of the skin to life-threatening systemic diseases (<a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Odds <em>et al.</em> 2007</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Magill <em>et al.</em> 2014</a>) and is one of the most common causes of blood stream infections in the U.S. (<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Magill <em>et al.</em> 2014</a>). While typically considered an obligate commensal, recent work has shown that <em>C. albicans</em> can be isolated from oak trees in the United Kingdom (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bensasson <em>et al.</em> 2019</a>) and fruits, soil, and plant matter in the USA (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Opulente <em>et al.</em> 2019</a>), providing evidence that <em>C. albicans</em> can also live outside animal hosts. <em>Candida albicans</em> strains from oak trees are closely related to pathogenic strains, and comparison of strains from these different environments may be useful for identifying the genetic determinants of pathogenesis (<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Bensasson <em>et al.</em> 2019</a>).</p><p id="p-5">Genetic and genomic studies have confirmed that the <em>C. albicans</em> genome typically exists in a diploid heterozygous state (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Barnett 2008</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Hickman <em>et al.</em> 2013</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Hirakawa <em>et al.</em> 2015</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Ropars <em>et al.</em> 2018</a>). The heterozygous nature of the <em>C. albicans</em> genome makes it particularly hard to assemble with current methods that assume a haploid or highly inbred genome (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Jones <em>et al.</em> 2004</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Nantel 2006</a>). The initial effort to assemble the genome of the diploid <em>C. albicans</em> reference strain SC5314 performed a haploid assembly of Sanger shotgun sequences followed by an effort to separate (or “phase”) haplotypes in the subset of the genome where homologous chromosomes had sufficient divergence to be assembled as distinct contigs (<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Jones <em>et al.</em> 2004</a>). Subsequently, <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">van het Hoog <em>et al.</em> (2007)</a> integrated contigs from <a id="xref-ref-17-3" class="xref-bibr" href="#ref-17">Jones <em>et al.</em> (2004)</a> with optical maps and other data to generate a haploid representation of all eight <em>C. albicans</em> chromosomes. <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Muzzey <em>et al.</em> (2013)</a> then mapped short read sequences from SC5314 and a panel of diploid strains with loss of heterozygosity (LOH) for particular chromosomes (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Legrand <em>et al.</em> 2008</a>) to the haploid assembly from <a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">van het Hoog <em>et al.</em> (2007)</a> to phase SNPs and short indel variants that distinguish the two haplotypes in SC5314. More recently, long-read sequencing has been used to generate a <em>de novo</em> haploid assembly of a pathogenic strain of <em>C. albicans</em> (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Panthee <em>et al.</em> 2018</a>). However, no phased <em>de novo</em> diploid genome assembly has currently been reported for pathogenic or environmental strains of <em>C. albicans</em>.</p><p id="p-6">Phasing haplotypes is important for understanding the relationship between DNA sequence and phenotype in diploid organisms (reviewed in <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Tewhey <em>et al.</em> 2011</a>). Phased genomes can also expose misassemblies in reference genomes (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Korlach <em>et al.</em> 2017</a>), help identify point mutations and large structural variation between haplotypes (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Zhou <em>et al.</em> 2019b</a>), and allow detection of allele-specific expression and epigenetic modifications (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Zhou <em>et al.</em> 2019a</a>). In a predominantly asexual species such as <em>C. albicans</em> (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Birky 1996</a>; <a id="xref-ref-2-3" class="xref-bibr" href="#ref-2">Barnett 2008</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Bougnoux <em>et al.</em> 2008</a>; <a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Hirakawa <em>et al.</em> 2015</a>), haplotype phasing of alleles is needed for correct inference of phylogenetic relationships (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Birky 1996</a>), and for detecting rare mating events that have potential implications for understanding adaptation to antifungal drugs (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Berman and Hadany 2012</a>) and disease emergence (<a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Ropars <em>et al.</em> 2018</a>).</p><p id="p-7">Despite their utility, phased diploid representations of genomes are not commonly obtained during <em>de novo</em> genome assembly since phasing parental haplotypes is technically challenging. As of May 2019, there were only ∼120 assemblies with some level of haplotype resolution in the NCBI Assembly database, a small number compared to well over 300,000 haploid assemblies (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Kitts <em>et al.</em> 2016</a>). Since for many organisms it is often not feasible or practical to construct strains with single haplotypes to circumvent the challenges of diploid assembly, different computational strategies have been developed to directly assemble diploid genomes. One such method is called FALCON-Unzip, which involves using heterozygous variation captured by PacBio Single-Molecule Real-Time (SMRT) long-read sequencing to generate a <em>de novo</em> diploid assembly with haplotypes phased in heterozygous regions (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Chin <em>et al.</em> 2016</a>).</p><p id="p-8">Here, we generated whole genome shotgun sequences using the PacBio SMRT technology (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Eid <em>et al.</em> 2009</a>) and applied the FALCON/FALCON-Unzip (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Chin <em>et al.</em> 2016</a>) assembly and phasing pipeline to three strains of <em>C. albicans</em> collected from oak trees in the UK (<a id="xref-ref-4-3" class="xref-bibr" href="#ref-4">Bensasson <em>et al.</em> 2019</a>). Our results demonstrate the feasibility of generating essentially-complete <em>de novo</em> diploid assemblies of <em>C. albicans</em> genomes using long-read sequencing and provide resources for future analysis of genome evolution and function in this important human fungal pathogen.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Yeast strains</h3><p id="p-9">DNA was extracted from three strains of <em>C. albicans</em>, NCYC 4144 (FRI10b.1), NCYC 4145 (FRI11a.1) and NCYC 4146 (FRI5d.SM), which were isolated from oak tree bark (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Robinson <em>et al.</em> 2016</a>). The strains analyzed here were previously characterized using Illumina short-read sequencing, are all phylogenetically distinct and show unusually high heterozygosity (<a id="xref-ref-4-4" class="xref-bibr" href="#ref-4">Bensasson <em>et al.</em> 2019</a>). More specifically, comparison between the genome-wide SNPs of these oak strains and representatives from the known <em>C. albicans</em> clades and singletons described in <a id="xref-ref-35-3" class="xref-bibr" href="#ref-35">Ropars <em>et al.</em> (2018)</a> showed that NCYC 4146 belongs to clade 4, NCYC 4144 to clade 18 and NCYC 4145 does not resemble any known genome (<a id="xref-ref-4-5" class="xref-bibr" href="#ref-4">Bensasson <em>et al.</em> 2019</a>).</p></div><div id="sec-3" class="subsection"><h3>Extraction of high molecular weight genomic DNA</h3><p id="p-10">For extraction of high molecular weight DNA, we used the Promega Wizard Genomic DNA Purification Kit (A1125), and modified it to include an extra RNA digestion step, ethanol precipitation, longer centrifugation steps at higher centrifugal force and more reactions per sample (general recommendations are detailed in <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Bensasson 2018</a>). For each strain, a single colony was used to inoculate 15 ml of YPD media and grown for 20-24 hr at 3∘C. Cells were pelleted in two minutes at a relative centrifugal force of 16,162 g in 1.4 ml volumes in thirteen (NCYC 4145), fourteen (NCYC 4146) or eleven (NCYC 4144) 1.5 ml tubes. Cell walls were digested overnight at 37∘C using 100 units lyticase (Sigma, L2524-50KU). Cells in each tube were pelleted (2 min at 16,128 g), resuspended in 300 μl Promega Nuclei Lysis solution and 100 μl Promega Protein Precipitation solution, cooled on ice for 5-30 min and cell debris was pelleted at 16,162 g for 10 min. The DNA in the supernatant was precipitated in 300 μl isopropanol and pelleted at 16,162 g for 10 min, washed with 300 μl of 70% ethanol at room temperature, pelleted at 16,162 g for 5 min, and air dried for 15 min. Pellets were resuspended in Promega DNA rehydration solution (1x TE buffer). RNA was digested using 1.5 μl (5.25 units) Promega RNase solution at 37∘C for 1-2 hr, then at room temperature overnight. For each tube, 1 μl was visualized alongside a high molecular weight ladder on an agarose gel in 1x TAE buffer, and extracts showing clear bands were pooled into 4 tubes per strain. DNA extracts were cooled on ice, then digested for a second time with 21 units Promega RNase solution at 37∘C for 1 hr then cooled on ice. DNA was precipitated using 0.5 volumes Promega Protein Precipitation Solution and 2 volumes of cold 96–100% ethanol on ice for 15 min, then pelleted at 16,162 g for 10 min. Pellets were washed with 3 volumes 70% ethanol, repelleted at 16,162 g for 3 min, and air dried for 15 min. DNA was resuspended in Promega DNA rehydration solution by incubation at 37∘C for 1-1.5 hr. DNA extracts were spun at 16,128 g for 10 min, and supernatants from multiple tubes for the same strain were pooled to produce 15-30μg genomic DNA per strain. The quality and quantity of DNA was assessed by agarose gel electrophoresis (0.8% agarose in 1x TAE buffer) alongside a high molecular weight ladder (<em>e.g.</em>, GeneRuler High Range DNA Ladder, SM1351), by NanoDrop, Qubit fluorometer and Fragment Analyzer Automated CE System, and showed average DNA fragment lengths over 43 kb for NCYC 4144, and over 34 kb for NCYC 4145 and NCYC 4146.</p></div><div id="sec-4" class="subsection"><h3>Long-read genome sequencing</h3><p id="p-11">Long-read genome sequencing was performed on the PacBio Sequel sequencing platform by the Georgia Genomics and Bioinformatics Core facility at the University of Georgia. A large insert library was constructed for each strain using the SMRTbell Template Prep Kit following the PacBio’s instructions for &gt;20 kb Template Preparation using BluePippin Size-Selection System for Sequel Systems. Fragment Analyzer analyses showed the final size of the insert libraries was 37 kb for NCYC 4144, and 22-24 kb for NCYC 4145 and NCYC 4146. Primers were annealed to the templates, then templates were bound to polymerase using the Sequel Binding Kit 2.0. The resultant polymerase bound complexes were purified using SMRTbell Clean Up Columns for Sequel, and were then bound to MagBeads for loading. Genomic DNA for each strain was loaded onto a single SMRTcell v1 and run on the PacBio Sequel System with a movie time of 10 hr.</p></div><div id="sec-5" class="subsection"><h3>De novo genome assembly and scaffolding</h3><p id="p-12">We performed <em>de novo</em> assembly with FALCON (v1.2.3) and phasing with FALCON-Unzip (v1.1.3; <a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">Chin <em>et al.</em> 2016</a>). The assembly/phasing/polishing software were downloaded as part of the pb-assembly metapackage (v0.0.1) from bioconda (<a href="https://github.com/PacificBiosciences/pb-assembly">https://github.com/PacificBiosciences/pb-assembly</a>). FALCON assembly was performed with a genome size parameter of 14.2 Mb, seed coverage of 30x, and a length cutoff for corrected reads of 1,000 bp. Consensus sequences were obtained after phasing using the Arrow polishing algorithm (v2.3.2) in FALCON-Unzip. Minimum coverage required for Arrow polishing was set to 5 and the positions of regions that could not be polished were summarized using a script (fastaLC2n.pl; <a href="https://github.com/bensassonlab/scripts">https://github.com/bensassonlab/scripts</a>) and are provided in Files S1 to S6. Software versions and parameters used for assembly are provided in Files S7 to S9.</p><p id="p-13">Reference-based scaffolding of primary contigs was performed using RaGOO (v1.01; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Alonge <em>et al.</em> 2019</a>) aligned to the SC5314 reference genome (Assembly 22, A haplotype, GCF_000182965.3). RaGOO scaffolding was only used to assign primary contigs to chromosomes, and subsequent QC analysis was performed in the raw, non-scaffolded assemblies. Haplotig placement relative to the primary assembly of each strain was done by aligning haplotigs to primary contigs with minimap2 (v2.16-r922; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Li 2018</a>) using the ’-x asm5′ alignment preset. The longest aligned segment for each haplotig was used to determine its placement in the primary assembly, allowing unaligned sequence from either end of the haplotig to be reported as alignment “tails”.</p></div><div id="sec-6" class="subsection"><h3>Diploid assembly quality assessment</h3><p id="p-14">To assess their overall quality, we compared the resulting assemblies to the reference genome for <em>C. albicans</em> strain SC5314 (Assembly 22, A haplotype, GCF_000182965.3) and collected statistics using QUAST v5.0.2 (<a href="https://github.com/ablab/quast">https://github.com/ablab/quast</a>, commit 67a1136, <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Gurevich <em>et al.</em> 2013</a>). Completeness of the assemblies was assessed by searching for highly conserved single-copy orthologs using BUSCO (v3.0.2, <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Simão <em>et al.</em> 2015</a>) with the Saccharomycetales ortholog gene set from OrthoDB v9 (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Zdobnov <em>et al.</em> 2016</a>).</p><p id="p-15">To obtain a visual summary of long-range assembly contiguity and correctness we performed whole genome alignments between each diploid assembly and the reference genome using minimap2 (v2.16-r922; <a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Li 2018</a>) with the assembly alignment preset ‘-x asm10’, and visualized them with dotPlotly (<a href="https://github.com/tpoorten/dotPlotly/">https://github.com/tpoorten/dotPlotly/</a>). Only alignments at least 20 kb in length and with a mapping quality of 60 were visualized. For detailed analysis of structural variants in chromosome 3, we performed alignments of the diploid assemblies to the reference genome using nucmer from the MUMmer package (v4.0.0beta2, <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Marçais <em>et al.</em> 2018</a>) with the settings: –maxmatch -l 80 -c 100, then visualized nucmer alignments of at least 1,000 bp using dotPlotly.</p><p id="p-16">We assessed the phasing status across the genome by aligning contigs from diploid assemblies to the haploid reference and analyzing the depth of contig coverage. Alignments were obtained as described above, converted to BED format, and the depth of contig coverage along the genome was calculated using BEDTools genomecov (v2.28.0; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Quinlan and Hall 2010</a>). A visual summary of depth of contig coverage across the genome was obtained with the kpPlotCoverage function from the karyoploteR package (v1.10.2; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Gel and Serra 2017</a>). For regions of the genome where phasing was successful, we expect a depth of contig coverage of two, whereas for regions where the haplotypes were not phased, we expect a depth of coverage of one. Since phasing by FALCON-Unzip can only be done in regions were heterozygous variants are present, we also contrasted the extent of unphased regions in our assemblies (depth of contig coverage = 1) to the LOH regions detected by <a id="xref-ref-4-6" class="xref-bibr" href="#ref-4">Bensasson <em>et al.</em> (2019)</a> based on Illumina short-read sequencing.</p><p id="p-17">For independent confirmation of phased base calls and LOH regions in each strain, we mapped Illumina HiSeq short-read data (PRJEB27862, <a id="xref-ref-4-7" class="xref-bibr" href="#ref-4">Bensasson <em>et al.</em> 2019</a>) to primary assemblies using bwa mem (v0.7.17; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Li and Durbin 2009</a>), produced a sorted bam file using SAMtools (v1.9; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Li <em>et al.</em> 2009</a>), then called variants using bcftools mpileup (-d 10,000) and bcftools call (-c) (v1.9; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Li 2011</a>). Because of the very high depth of Illumina sequencing for NCYC 4146, we only used data from one run (ERR2708456) for variant calling in this strain. We used vcf2allelePlot.pl (<a id="xref-ref-4-8" class="xref-bibr" href="#ref-4">Bensasson <em>et al.</em> 2019</a>) running R (version 3.5.0) to generate allele ratio plots for high quality base calls (phred-scaled quality &gt;40) in order to annotate LOH regions, identify regions within the primary assemblies that had unexpectedly low heterozygosity (below 0.1% using 100 kb non-overlapping sliding windows), and summarize levels of genome-wide heterozygosity for each strain. Sites were counted as heterozygous if allele ratios were ∼0.5; more specifically between 0.2 and 0.8.</p></div><div id="sec-7" class="subsection"><h3>Annotation of telomeres and centromeres</h3><p id="p-18">In order to check whether the primary genome assemblies extend into telomeres, we extracted 5 kb from the termini of every chromosome in the primary assemblies and identified repeats using Tandem Repeats Finder (TRF, v.4.09, <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Benson 1999</a>) with the following command: ‘trf &lt;fasta&gt; 2 5 7 80 10 50 500 -d -m -h'. We then searched the TRF results for the presence of terminal arrays of the 23 bp telomeric tandem repeats described for <em>C. albicans</em> (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">McEachern and Hicks 1993</a>). We also obtained the centromeric sequences for each chromosome from the SC5314 reference genome (Assembly 22, haplotype A) and used them as queries against the primary contigs of each of our assemblies. Alignments were performed with minimap2 (v2.16-r922, <a id="xref-ref-22-3" class="xref-bibr" href="#ref-22">Li 2018</a>) on default settings.</p></div><div id="sec-8" class="subsection"><h3>Data availability</h3><p id="p-19">Raw PacBio reads for the three <em>Candida</em> strains are available at the NCBI short-read archive (SRA) under BioProject PRJNA533645. The phased diploid assemblies for the three oak strains are associated with the overall BioProject PRJNA543321. Individual GenBank accession numbers for primary contigs and haplotigs, respectively, for each strain are as follows: NCYC 4144: GCA_005890765.1 and GCA_005890695.1; NCYC 4145: GCA_005890775.1 and GCA_005890685.1; NCYC 4146: GCA_005890745.1 and GCA_005890705.1. Coordinates of haplotigs relative to their respective primary assembly are available in Files S10, S11, and S12. Annotations of positions of centromeres, telomeric repeats, confirmed LOH regions, assembly gaps, uncertain regions with unexpectedly low heterozygosity, and regions that were not polished by FALCON-Unzip are available for primary assemblies in Files S1, S2, and S3. Annotations of unpolished regions for alternative haplotig assemblies are provided in Files S4, S5 and S6. A full description of software version numbers for phased assembly is provided in File S7 and the configuration files used to run FALCON and FALCON-Unzip are provided in Files S8 and S9. Supplemental material available at FigShare: <a href="https://doi.org/10.25387/g3.9835208">https://doi.org/10.25387/g3.9835208</a>.</p></div></div><div class="section results-discussion" id="sec-9"><h2 class="">Results and Discussion</h2><div id="sec-10" class="subsection"><h3>High coverage long-read datasets for C. albicans oak strains</h3><p id="p-20">We generated PacBio DNA sequencing data for three strains of <em>C. albicans</em> isolated from oak trees. Each sample was independently sequenced in a single SMRT cell on the PacBio Sequel instrument. A total of 703,252, 689,752, and 863,237 subreads with average lengths of 9,082, 8,899, and 9,576 bp were obtained for strains NCYC 4144, NCYC 4145 and NCYC 4146, respectively. Theoretical coverage ranged from 415 to 558x assuming haploid coverage for a 14.8 Mb genome size (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>; <a id="xref-ref-41-3" class="xref-bibr" href="#ref-41">van het Hoog <em>et al.</em> 2007</a>). For all strains, half of the data were present in reads of approximately 16 kb or longer (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>) with the longest reads obtained being 69 kb (NCYC 4144), 68 kb (NCYC 4145), and 101 kb (NCYC 4146; <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/494775/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/494775/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">PacBio sequencing statistics for <em>C. albicans</em> oak strains</span><div class="sb-div caption-clear"></div></div></div><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/11/3547/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Read length distribution for the PacBio data sets generated for three C. albicans oak strains. Vertical lines indicate read N50 length." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2043233161" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Read length distribution for the PacBio data sets generated for three &lt;em&gt;C. albicans&lt;/em&gt; oak strains. Vertical lines indicate read N50 length.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/11/3547/F1.medium.gif" width="310" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/9/11/3547/F1.medium.gif" width="310" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/11/3547/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/11/3547/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494855" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-22" class="first-child">Read length distribution for the PacBio data sets generated for three <em>C. albicans</em> oak strains. Vertical lines indicate read N50 length.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-11" class="subsection"><h3>Long reads allow Near-complete primary assemblies and extensive haplotype phasing in C. albicans</h3><p id="p-23">We used the FALCON/FALCON-Unzip assembly and phasing pipeline (<a id="xref-ref-11-4" class="xref-bibr" href="#ref-11">Chin <em>et al.</em> 2016</a>) to obtain phased diploid assemblies for the three <em>C. albicans</em> oak strains. For each strain, the assembly is composed of two sets of contigs; a primary contigs set, and an alternate haplotigs set. The primary contigs provide a pseudo-haploid representation of the genome, (<em>i.e.</em>, they may include haplotype switches), and the haplotigs represent alternate haplotypes in the regions for which phasing was achieved. Phasing interruption and haplotype switches may occur when FALCON-Unzip is no longer able to establish linkage between variants (<em>e.g.</em>, in LOH regions). We note that FALCON-Unzip removes non-linear contigs with high copy number, and as a consequence there is no mtDNA represented in the assemblies reported here.</p><p id="p-24">Primary assembly size varied from 14.7 Mb in NCYC 4144 to 15.5 Mb in NCYC 4145 and NCYC 4146 (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). The current reference assembly for strain SC5314 spans ∼14.2 Mb, whereas the predicted genome size for <em>C. albicans</em> obtained from physical and optical maps is ∼14.8-14.9 Mb (<a id="xref-ref-17-4" class="xref-bibr" href="#ref-17">Jones <em>et al.</em> 2004</a>; <a id="xref-ref-41-5" class="xref-bibr" href="#ref-41">van het Hoog <em>et al.</em> 2007</a>). Thus, the primary assembly for NCYC 4144 is very close to the predicted haploid reference genome size for <em>C. albicans</em>, while NCYC 4145 and NCYC 4146 have an additional ∼500 kb of sequence in their primary assemblies. Haplotig assembly size varied from 12.5-13.7 Mb (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>), indicating that the vast majority (84–92%) of the genome was phased in each assembly.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/494735/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/494735/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title"><em>De novo</em> genome assembly statistics</span><div class="sb-div caption-clear"></div></div></div><p id="p-25">All primary assemblies achieved high contiguity, with primary contig N50 ranging from 1.2 Mb in NCYC 4146, to 1.6 Mb in NCYC 4145, and 1.9 Mb in NCYC 4144 (<a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>). Several chromosomes were assembled as contiguous gapless sequences spanning the entire chromosome, reflecting the low fragmentation of these assemblies. For strain NCYC 4144, chromosomes 1 through 7 were recovered as single contigs, while chromosome R was split in two contigs (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>). In strain NCYC 4145, chromosomes 1, 3, 4, and 6 were recovered as single contigs whereas the remaining chromosomes were split into either 3, 5, or 9 contigs (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2B</a>). In strain NCYC 4146, chromosomes 3, 4, and 5 were recovered as single contigs whereas the remaining chromosomes were split into either 2, 4, or 5 contigs (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2C</a>). These highly contiguous and near complete primary assemblies are a consequence of the long length and high depth of the PacBio reads used for the assembly. Most read lengths (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>) exceed the length of known repeats in <em>C. albicans</em>, such as Major Repeat Sequences, tandemly arrayed telomeric repeat units and retroelements (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Todd <em>et al.</em> 2019</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/11/3547/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Diploid assemblies of C. albicans generated in this study (y-axis) aligned to the reference strain SC5314 (x-axis). For each chromosome that is labeled on the x-axis the primary contigs and haplotigs are stacked, with the primary contigs always placed first in the lower portion of each plot. Darker vertical grid lines demarcate chromosome limits, and horizontal lines show gaps between contigs. Note that the primary assemblies (lower diagonals for each chromosome) are nearly gapless. These dot plots display only alignments that are at least 20 kb in length and aligned segments are colored by percent identity with SC5314." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2043233161" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Diploid assemblies of &lt;em&gt;C. albicans&lt;/em&gt; generated in this study (y-axis) aligned to the reference strain SC5314 (x-axis). For each chromosome that is labeled on the x-axis the primary contigs and haplotigs are stacked, with the primary contigs always placed first in the lower portion of each plot. Darker vertical grid lines demarcate chromosome limits, and horizontal lines show gaps between contigs. Note that the primary assemblies (lower diagonals for each chromosome) are nearly gapless. These dot plots display only alignments that are at least 20 kb in length and aligned segments are colored by percent identity with SC5314.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/11/3547/F2.medium.gif" width="201" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/9/11/3547/F2.medium.gif" width="201" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/11/3547/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/11/3547/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494764" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-26" class="first-child">Diploid assemblies of <em>C. albicans</em> generated in this study (y-axis) aligned to the reference strain SC5314 (x-axis). For each chromosome that is labeled on the x-axis the primary contigs and haplotigs are stacked, with the primary contigs always placed first in the lower portion of each plot. Darker vertical grid lines demarcate chromosome limits, and horizontal lines show gaps between contigs. Note that the primary assemblies (lower diagonals for each chromosome) are nearly gapless. These dot plots display only alignments that are at least 20 kb in length and aligned segments are colored by percent identity with SC5314.</p><div class="sb-div caption-clear"></div></div></div><p id="p-27">Whole-genome alignments and dot plots indicate large-scale genome colinearity between the reference strain SC5314 and both primary contigs or haplotigs for the oak strains (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2</a>), suggesting no major missassemblies in our assemblies or the SC5314 reference genome. <em>C. albicans</em> centromeres are composed of discrete loci with unique sequences on each chromosome (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Sanyal <em>et al.</em> 2004</a>). Mapping the centromeric sequences annotated in the SC5314 reference genome (Assembly 22) to our primary assemblies revealed the presence of a unique match on each chromosome at the expected positions. The only exception was centromere 6 of strain NCYC 4146, which maps to two chromosome 6 locations on either side of a sequencing gap, indicating either a duplication or potential misassembly (Files S1-S3). Likewise, the majority of chromosome ends in primary assemblies (38 out of 48) contain an array of the 23 bp telomeric repeats described for <em>C. albicans</em> (<a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">McEachern and Hicks 1993</a>). The presence of telomeric repeats at the ends of most chromosome sequences (Files S1-S3) indicate that the primary assemblies often reached the chromosome termini and are thus nearly complete.</p><p id="p-28">Haplotig N50 (the N50 length of phased blocks) ranged from 174 kb in NCYC 4145 to 254 kb in NCYC 4144 (<a id="xref-table-wrap-2-4" class="xref-table" href="#T2">Table 2</a>). Although FALCON-Unzip produces many haplotigs per chromosome instead of fully phased chromosomes, the phase block length achieved in these assemblies is more than two orders of magnitude larger than the average gene length in <em>C. albicans</em> (1,439 bp, <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Braun <em>et al.</em> 2005</a>). BUSCO analysis of highly conserved single-copy genes in the primary contig sets revealed the presence of 97.1%, 96.2%, and 96.9% complete BUSCOs for NCYC 4144, NCYC 4145, and NCYC 4146, respectively (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>). We observed higher levels of completely duplicated BUSCOs in the primary assemblies for NCYC 4145 and NCYC 4146 relative to NCYC 4144, which suggests that the extra sequence in primary assemblies for these strains may be associated with duplications in the assembly. Analysis of the same single-copy genes in the haplotig sets revealed that 79.2%, 88.2%, and 84.3% of their alleles are complete in the alternative haplotype of NCYC 4144, NCYC 4145, and NCYC 4146, respectively. These results indicate that our primary assemblies are near complete in terms of gene content, and that both alleles for the majority of single copy genes are phased.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/494659/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/494659/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">BUSCO scores for the primary contigs of <em>C. albicans de novo</em> assemblies. Values represent percentages and numbers inside parentheses indicate absolute number of genes for each category</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-12" class="subsection"><h3>Assembled PacBio genome sequences have high quality base calls</h3><p id="p-30">We verified the quality of base calls in the primary assemblies by mapping independently-generated Illumina sequence (<a id="xref-ref-4-9" class="xref-bibr" href="#ref-4">Bensasson <em>et al.</em> 2019</a>) to the corresponding primary PacBio assembly for each strain. This resulted in high quality Illumina base calls for over 14 million invariant nucleotide sites in each strain where base calls from Illumina data matched the primary PacBio assembly. Furthermore, we observed over 70,000 sites with high quality SNP variants for each strain, which almost exclusively had the allele ratio of ∼0.5 that is expected when mapping a heterozygous diploid strain to one of its parental haplotypes. All strains had fewer than 270 high quality variants in Illumina data with allele ratios of &gt;0.95; we expect such allele ratios of ∼1 if the base call in the primary PacBio assembly is incorrect. The rate of these probable errors is lower than one every 50 kb or one every 300 SNPs and confirms that the base quality of the majority of sites in the assemblies is high. For the strain with the most complete primary assembly (NCYC 4144), an unexpectedly large proportion of these errors (66 out of 169 errors; 39%) occur in the repetitive fraction of the primary assembly (21%) that NCBI present in lower case in the assemblies (Binomial exact test, p-value = <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3547/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3547/embed/mml-math-1.gif"/></noscript></span></span>). For the other strains, the proportion of errors in repetitive regions is close to expectations; 69 out of 269 (26%) for NCYC 4145 and 24 out of 143 for NCYC 4146 (17%). For these other two strains, a larger proportion (&gt;29%) of errors (allele ratio ∼1) occur in the fraction of the primary assembly (1%) that could not be polished using FALCON-Unzip (Binomial exact test, p-value <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3547/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3547/embed/mml-math-2.gif"/></noscript></span></span>). The positions of the unpolished regions in all primary and alternate assemblies are provided in Files S1 to S6.</p><p id="p-31">Heterozygous SNPs identified by mapping Illumina reads to primary PacBio assemblies also allowed us to confirm previously-identified LOH regions and estimates of genome-wide heterozygosity for each strain (<a id="xref-ref-4-10" class="xref-bibr" href="#ref-4">Bensasson <em>et al.</em> 2019</a>). However, patterns of SNP variation also revealed that the primary assemblies for strains NCYC 4145 and NCYC 4146 contained six regions each that showed unexpectedly low heterozygosity at (or near) breaks in the assembly not previously identified as LOH regions in <a id="xref-ref-4-11" class="xref-bibr" href="#ref-4">Bensasson <em>et al.</em> (2019)</a>. These “uncertain” regions could represent poorly resolved structural variants or assembly errors and may explain the excess sequence and increased levels of duplication observed in NCYC 4145 and NCYC 4146, including the extra copy of centromere 6 seen for NCYC 4146 (File S6). The DNA for these two strains was of lower molecular weight, yielded insert libraries with shorter inserts (22-24 kb) than for NCYC 4144 (37 kb) as well as shorter reads (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>). Further long read data for these two strains would show whether the extra sequence in their assemblies are errors or genuine duplications. Uncertain regions span ∼1 Mb in the primary assemblies of NCYC 4145 and NCYC 4146 and are described more fully in the annotations for their primary assemblies (Files S2 and S3) along with the positions of centromeres, telomeres, confirmed LOH regions, gaps and unpolished regions for all three strains (Files S1-S3).</p></div><div id="sec-13" class="subsection"><h3>Phasing efficiency and loss of heterozygosity</h3><p id="p-32">Whole genome alignments indicate that some regions of the genome for each strain are not phased, such as chromosome 5 in strain NCYC 4144 (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2A</a>). The phasing efficiency of FALCON-Unzip depends on the presence of heterozygous variants captured by long reads. Accordingly, variants separated by long tracts of homozygosity cannot be phased by this approach (<a id="xref-ref-11-5" class="xref-bibr" href="#ref-11">Chin <em>et al.</em> 2016</a>). Since the strains analyzed here were previously shown to contain segmental and whole-chromosome loss of heterozygosity (<a id="xref-ref-4-12" class="xref-bibr" href="#ref-4">Bensasson <em>et al.</em> 2019</a>), FALCON-Unzip is not expected to phase the entirety of these genomes.</p><p id="p-33">Approximately 16.1%, 4.9% and 7.7% of the genomes of NCYC 4144, NCYC 4145, and NCYC 4146, respectively, were defined as having LOH based on variants detected in short read data relative to the SC5314 reference genome (<a id="xref-ref-4-13" class="xref-bibr" href="#ref-4">Bensasson <em>et al.</em> 2019</a>). As expected if LOH regions impact FALCON-Unzip phasing, the strain with the highest amount of LOH would have the lowest number of genes phased in haplotigs, and <em>vice versa</em>, which is demonstrated by the complete BUSCO score for haplotigs in NCYC 4144 compared to NCYC 4145 and NCYC 4146 (<a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>). To more directly analyze the correspondence between the LOH regions defined previously and the phasing efficiency of FALCON-Unzip, we mapped all contigs (primary and haplotigs) from diploid assemblies to the haploid SC5314 reference genome and analyzed the resulting depth of contig coverage profiles. While the average depth of contig coverage is approximately 2 in the non-LOH regions for the three strains, this value drops to an average of 1.31, 1.58, and 1.39 in LOH regions of NCYC 4144, NCYC 4145, and NCYC 4146, respectively (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). The difference in average depth of coverage between LOH and non-LOH regions is significant at a 99% confidence level (Welch two sample <em>t</em>-test, p-value &lt; 0.001). These results confirm the expectation that homozygous regions disrupt phasing by FALCON-Unzip.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/11/3547/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Depth of contig coverage for three diploid C. albicans assemblies mapped to the haploid reference genome (strain SC5314). Black boxes below each coverage plot highlight the regions defined as loss of heterozygosity in Bensasson et al. (2019) and generally correspond to regions of contig depth of coverage = 1." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2043233161" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Depth of contig coverage for three diploid &lt;em&gt;C. albicans&lt;/em&gt; assemblies mapped to the haploid reference genome (strain SC5314). Black boxes below each coverage plot highlight the regions defined as loss of heterozygosity in Bensasson &lt;em&gt;et al.&lt;/em&gt; (2019) and generally correspond to regions of contig depth of coverage = 1.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/11/3547/F3.medium.gif" width="440" height="314"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/9/11/3547/F3.medium.gif" width="440" height="314"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/11/3547/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/11/3547/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494799" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-34" class="first-child">Depth of contig coverage for three diploid <em>C. albicans</em> assemblies mapped to the haploid reference genome (strain SC5314). Black boxes below each coverage plot highlight the regions defined as loss of heterozygosity in <a id="xref-ref-4-14" class="xref-bibr" href="#ref-4">Bensasson <em>et al.</em> (2019)</a> and generally correspond to regions of contig depth of coverage = 1.</p><div class="sb-div caption-clear"></div></div></div><p id="p-35">Although largely matching the previously known LOH regions, contig coverage profiles sometimes reveal windows for which the observed depth is &gt;2, or where the depth is not as expected given the presence or absence of LOH (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>). Since contig coverage is calculated after aligning the <em>de novo</em> assemblies to the haploid reference, duplications in either haplotype that align to the same location in the reference genome could result in contig depth &gt;2 (see subsection below). Additionally, regions defined as LOH by short-read variants that have the expected diploid contig coverage could represent regions where haplotype divergence at the nucleotide level is low, but more substantial at the structural level. Indeed, FALCON-Unzip also uses heterozygous structural variants to phase assemblies, which could potentially be used even in the absence of significant nucleotide divergence (<a id="xref-ref-11-6" class="xref-bibr" href="#ref-11">Chin <em>et al.</em> 2016</a>).</p></div><div id="sec-14" class="subsection"><h3>Phased diploid assemblies reveal structural variants</h3><p id="p-36">The evolution of <em>C. albicans</em> is characterized by frequent genomic rearrangements often involving open reading frames (ORFs) that are flanked by inverted repeats (<a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Todd <em>et al.</em> 2019</a>). For example, there are two groups of ORFs in neighboring regions of chromosome 3 which appear to have undergone duplication followed by inversion, generating inverted repeats (<a id="xref-ref-41-6" class="xref-bibr" href="#ref-41">van het Hoog <em>et al.</em> 2007</a>; <a id="xref-ref-40-3" class="xref-bibr" href="#ref-40">Todd <em>et al.</em> 2019</a>), and these can lead to further large-scale rearrangements (<a id="xref-ref-40-4" class="xref-bibr" href="#ref-40">Todd <em>et al.</em> 2019</a>). Close inspection of whole-genome alignment dot plots between the oak strains’ diploid assemblies relative to the SC5314 reference genome in this region of chromosome 3 reveals that oak strains have experienced inversions of the sequence flanked by these inverted repeats (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4A</a>). The inversions are polymorphic among the three strains, with each strain displaying a unique genotype. Strains NCYC 4146 and NCYC 4144 are heterozygous for the first and second inversions, respectively, whereas strain NCYC 4145 is homozygous for both inversions. These results highlight the utility of long-read sequencing in detecting structural variants, and indicate that this approach should prove useful in gauging how much of <em>C. albicans</em> heterozygosity comes from structural variants as opposed to single nucleotide polymorphisms (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Sedlazeck <em>et al.</em> 2018</a>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/11/3547/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Structural polymorphisms in the diploid assemblies of chromosome 3 aligned to the haploid reference genome (strain SC5314) at a previously known locus (van het Hoog et al. 2007; Todd et al. 2019). (A) Dot plots of both primary contigs and haplotigs showing a small homozygous inversion in strain NCYC 4144 followed by a large heterozygous inversion. In strain NCYC 4146, the small inversion is heterozygous and the large inversion is homozygous, while NCYC 4145 is homozygous for both inversions. Both the small and the large inversion are flanked by inverted repeats, which are more easily recognized on the dot plots when the intervening inversions are absent (the NCYC 4144 or NCYC 4146 haplotigs). (B) Contig alignment depth over the same region. The inverted repeats on both sides of the two polymorphic inversions show the increased contig alignment depth expected for repeats." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2043233161" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Structural polymorphisms in the diploid assemblies of chromosome 3 aligned to the haploid reference genome (strain SC5314) at a previously known locus (van het Hoog &lt;em&gt;et al.&lt;/em&gt; 2007; Todd &lt;em&gt;et al.&lt;/em&gt; 2019). (A) Dot plots of both primary contigs and haplotigs showing a small homozygous inversion in strain NCYC 4144 followed by a large heterozygous inversion. In strain NCYC 4146, the small inversion is heterozygous and the large inversion is homozygous, while NCYC 4145 is homozygous for both inversions. Both the small and the large inversion are flanked by inverted repeats, which are more easily recognized on the dot plots when the intervening inversions are absent (the NCYC 4144 or NCYC 4146 haplotigs). (B) Contig alignment depth over the same region. The inverted repeats on both sides of the two polymorphic inversions show the increased contig alignment depth expected for repeats.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/11/3547/F4.medium.gif" width="440" height="376"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/9/11/3547/F4.medium.gif" width="440" height="376"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/11/3547/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/11/3547/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494783" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-37" class="first-child">Structural polymorphisms in the diploid assemblies of chromosome 3 aligned to the haploid reference genome (strain SC5314) at a previously known locus (<a id="xref-ref-41-7" class="xref-bibr" href="#ref-41">van het Hoog <em>et al.</em> 2007</a>; <a id="xref-ref-40-5" class="xref-bibr" href="#ref-40">Todd <em>et al.</em> 2019</a>). (A) Dot plots of both primary contigs and haplotigs showing a small homozygous inversion in strain NCYC 4144 followed by a large heterozygous inversion. In strain NCYC 4146, the small inversion is heterozygous and the large inversion is homozygous, while NCYC 4145 is homozygous for both inversions. Both the small and the large inversion are flanked by inverted repeats, which are more easily recognized on the dot plots when the intervening inversions are absent (the NCYC 4144 or NCYC 4146 haplotigs). (B) Contig alignment depth over the same region. The inverted repeats on both sides of the two polymorphic inversions show the increased contig alignment depth expected for repeats.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section conclusions" id="sec-15"><h2 class="">Conclusions</h2><p id="p-38">Here we show that PacBio SMRT sequencing is an affordable and efficient approach to obtain phased diploid assemblies for the model yeast species <em>C. albicans</em>. Our results suggest that SMRT sequencing and FALCON-Unzip assembly of the SC5314 reference strain is now merited and could provide an end to the “long hard road” to generate a complete diploid <em>C. albicans</em> reference genome (<a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Nantel 2006</a>). Our work also demonstrates that phased diploid assemblies generated using PacBio long-read data can provide detailed insights into genome structure and evolution in <em>C. albicans</em> that are not possible to obtain using short read sequencing. This advance is especially important for an asexual diploid species such as <em>C. albicans</em> with unusually high rates of structural rearrangements (<a id="xref-ref-7-3" class="xref-bibr" href="#ref-7">Birky 1996</a>; <a id="xref-ref-41-8" class="xref-bibr" href="#ref-41">van het Hoog <em>et al.</em> 2007</a>; <a id="xref-ref-40-6" class="xref-bibr" href="#ref-40">Todd <em>et al.</em> 2019</a>) that are associated with gene family evolution and pathogenicity (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Butler <em>et al.</em> 2009</a>; <a id="xref-ref-40-7" class="xref-bibr" href="#ref-40">Todd <em>et al.</em> 2019</a>).</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-39">This study was supported in part by resources and technical expertise from the Georgia Advanced Computing Resource Center. We thank Katherine Sandlin, Roger Nilsen and Magdy Alabady at the Georgia Genomics and Bioinformatics Core for performing some of the DNA extractions and for providing the PacBio sequencing service. This work was funded by the University of Georgia.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1"><a class="rev-xref" href="#xref-fn-1-1">↵</a><span class="fn-label">1</span> Equally contributing authors.</p></li><li class="fn" id="fn-4"><p id="p-40">Supplemental material available at FigShare: <a href="https://doi.org/10.25387/g3.9835208">https://doi.org/10.25387/g3.9835208</a>.</p></li><li class="fn" id="fn-5"><p id="p-41"><em>Communicating editor: D. Baltrus</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2019-06-30"><span class="received-label">Received </span>June 30, 2019.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2019-09-18"><span class="accepted-label">Accepted </span>September 18, 2019.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2019 Hamlin <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3547.1" data-doi="10.1101/519637"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Alonge</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Soyk</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Ramakrishnan</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Wang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Goodwin</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">Fast and accurate reference-guided scaffolding of draft genomes.</span> <abbr class="cit-jnl-abbrev">bioRxiv</abbr><span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/519637</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DbioRxiv%26rft_id%253Dinfo%253Adoi%252F10.1101%252F519637%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvcnhpdiI7czo1OiJyZXNpZCI7czo4OiI1MTk2Mzd2MSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzkvMTEvMzU0Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3547.2" data-doi="10.1002/yea.1595"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Barnett</span>,  <span class="cit-name-given-names">J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">A history of research on yeasts 12: medical yeasts part 1, <em>Candida albicans</em>.</span> <abbr class="cit-jnl-abbrev">Yeast</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">385</span>–<span class="cit-lpage">417</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/yea.1595</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DYeast%2B%2528Chichester%252C%2BEngland%2529%26rft.stitle%253DYeast%26rft.aulast%253DBarnett%26rft.auinit1%253DJ.%2BA.%26rft.volume%253D25%26rft.issue%253D6%26rft.spage%253D385%26rft.epage%253D417%26rft.atitle%253DA%2Bhistory%2Bof%2Bresearch%2Bon%2Byeasts%2B12%253A%2Bmedical%2Byeasts%2Bpart%2B1%252C%2BCandida%2Balbicans.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fyea.1595%26rft_id%253Dinfo%253Apmid%252F18509848%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/yea.1595&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18509848&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3547.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3547.3"><div class="cit-metadata"><cite>Bensasson, D., 2018 DNA extraction protocol for yeast PacBio sequencing. protocols.io.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.17504/protocols.io.rved63e</span></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3547.4" data-doi="10.1534/genetics.118.301482"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bensasson</span>,  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Dicks</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">Ludwig</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. J.</span> <span class="cit-name-surname">Bond</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Elliston</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">Diverse Lineages of <em>Candida albicans</em> Live on Old Oaks.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">211</span>: <span class="cit-fpage">277</span>–<span class="cit-lpage">288</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.118.301482</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.118.301482%26rft_id%253Dinfo%253Apmid%252F30463870%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjExLzEvMjc3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMS8zNTQ3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3547.5" data-doi="10.1093/nar/27.2.573"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Benson</span>,  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Tandem repeats finder: a program to analyze dna sequences.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">573</span>–<span class="cit-lpage">580</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/27.2.573</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F27.2.573%26rft_id%253Dinfo%253Apmid%252F9862982%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/27.2.573&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9862982&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3547.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000078181500028&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3547.6" data-doi="10.1016/j.tig.2012.01.004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Berman</span>,  <span class="cit-name-given-names">J.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Hadany</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Does stress induce (para)sex? Implications for <em>Candida albicans</em> evolution.</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">197</span>–<span class="cit-lpage">203</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.tig.2012.01.004</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DBerman%26rft.auinit1%253DJ.%26rft.volume%253D28%26rft.issue%253D5%26rft.spage%253D197%26rft.epage%253D203%26rft.atitle%253DDoes%2Bstress%2Binduce%2B%2528para%2529sex%253F%2BImplications%2Bfor%2BCandida%2Balbicans%2Bevolution.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2012.01.004%26rft_id%253Dinfo%253Apmid%252F22364928%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tig.2012.01.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22364928&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3547.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000303634800001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3547.7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Birky</span>,  <span class="cit-name-given-names">C. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Heterozygosity, Heteromorphy, and Phylogenetic Trees in Asexual Eukaryotes.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">144</span>: <span class="cit-fpage">427</span>–<span class="cit-lpage">437</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DBirky%26rft.auinit1%253DC.%2BW.%26rft.volume%253D144%26rft.issue%253D1%26rft.spage%253D427%26rft.epage%253D437%26rft.atitle%253DHeterozygosity%252C%2BHeteromorphy%252C%2Band%2BPhylogenetic%2BTrees%2Bin%2BAsexual%2BEukaryotes%26rft_id%253Dinfo%253Apmid%252F8878706%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTQ0LzEvNDI3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMS8zNTQ3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3547.8" data-doi="10.1016/j.fgb.2007.10.008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bougnoux</span>,  <span class="cit-name-given-names">M.-E.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Pujol</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Diogo</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Bouchier</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. R.</span> <span class="cit-name-surname">Soll</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Mating is rare within as well as between clades of the human pathogen <em>Candida albicans</em>.</span> <abbr class="cit-jnl-abbrev">Fungal Genet. Biol.</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">221</span>–<span class="cit-lpage">231</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.fgb.2007.10.008</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFungal%2Bgenetics%2Band%2Bbiology%2B%253A%2B%2BFG%2B%2526%2BB%26rft.stitle%253DFungal%2BGenet%2BBiol%26rft.aulast%253DBougnoux%26rft.auinit1%253DM.%2BE.%26rft.volume%253D45%26rft.issue%253D3%26rft.spage%253D221%26rft.epage%253D231%26rft.atitle%253DMating%2Bis%2Brare%2Bwithin%2Bas%2Bwell%2Bas%2Bbetween%2Bclades%2Bof%2Bthe%2Bhuman%2Bpathogen%2BCandida%2Balbicans.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.fgb.2007.10.008%26rft_id%253Dinfo%253Apmid%252F18063395%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.fgb.2007.10.008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18063395&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3547.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3547.9" data-doi="10.1371/journal.pgen.0010036"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Braun</span>,  <span class="cit-name-given-names">B. R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">van Het Hoog</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">d’Enfert</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Martchenko</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Dungan</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">A human-curated annotation of the <em>Candida albicans</em> genome.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">36</span>–<span class="cit-lpage">57</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.0010001</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DBraun%26rft.auinit1%253DB.%2BR.%26rft.volume%253D1%26rft.issue%253D1%26rft.spage%253D36%26rft.epage%253D57%26rft.atitle%253DA%2Bhuman-curated%2Bannotation%2Bof%2Bthe%2BCandida%2Balbicans%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.0010036%26rft_id%253Dinfo%253Apmid%252F16103911%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.0010036&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16103911&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3547.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000234295900006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3547.10" data-doi="10.1038/nature08064"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Butler</span>,  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. D.</span> <span class="cit-name-surname">Rasmussen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. F.</span> <span class="cit-name-surname">Lin</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. A. S.</span> <span class="cit-name-surname">Santos</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Sakthikumar</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Evolution of pathogenicity and sexual reproduction in eight <em>Candida</em> genomes.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">459</span>: <span class="cit-fpage">657</span>–<span class="cit-lpage">662</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature08064</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DButler%26rft.auinit1%253DG.%26rft.volume%253D459%26rft.issue%253D7247%26rft.spage%253D657%26rft.epage%253D662%26rft.atitle%253DEvolution%2Bof%2Bpathogenicity%2Band%2Bsexual%2Breproduction%2Bin%2Beight%2BCandida%2Bgenomes.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature08064%26rft_id%253Dinfo%253Apmid%252F19465905%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature08064&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19465905&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3547.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000266608600034&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3547.11" data-doi="10.1038/nmeth.4035"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chin</span>,  <span class="cit-name-given-names">C.-S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Peluso</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F. J.</span> <span class="cit-name-surname">Sedlazeck</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Nattestad</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G. T.</span> <span class="cit-name-surname">Concepcion</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Phased diploid genome assembly with single-molecule real-time sequencing.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">1050</span>–<span class="cit-lpage">1054</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nmeth.4035</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BMethods%26rft.volume%253D13%26rft.spage%253D1050%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.4035%26rft_id%253Dinfo%253Apmid%252F27749838%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.4035&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27749838&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3547.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3547.12" data-doi="10.1126/science.1162986"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eid</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Fehr</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Gray</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Luong</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Lyle</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Real-time dna sequencing from single polymerase molecules.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">323</span>: <span class="cit-fpage">133</span>–<span class="cit-lpage">138</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.1162986</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1162986%26rft_id%253Dinfo%253Apmid%252F19023044%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjMvNTkxMC8xMzMiO3M6NDoiYXRvbSI7czoxOToiL2dnZy85LzExLzM1NDcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3547.13" data-doi="10.1093/bioinformatics/btx346"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gel</span>,  <span class="cit-name-given-names">B.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Serra</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">karyoploter: an r/bioconductor package to plot customizable genomes displaying arbitrary data.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">3088</span>–<span class="cit-lpage">3090</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btx346</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.volume%253D33%26rft.spage%253D3088%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtx346%26rft_id%253Dinfo%253Apmid%252F28575171%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btx346&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28575171&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3547.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3547.14" data-doi="10.1093/bioinformatics/btt086"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gurevich</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Saveliev</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Vyahhi</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Tesler</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Quast: quality assessment tool for genome assemblies.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">1072</span>–<span class="cit-lpage">1075</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btt086</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtt086%26rft_id%253Dinfo%253Apmid%252F23422339%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btt086&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23422339&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3547.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000318109300015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3547.15" data-doi="10.1038/nature11865"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hickman</span>,  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Zeng</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Forche</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. P.</span> <span class="cit-name-surname">Hirakawa</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Abbey</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The ‘obligate diploid’ <em>Candida albicans</em> forms mating-competent haploids.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">494</span>: <span class="cit-fpage">55</span>–<span class="cit-lpage">59</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature11865</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D494%26rft.spage%253D55%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature11865%26rft_id%253Dinfo%253Apmid%252F23364695%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature11865&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23364695&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3547.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000314741200032&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3547.16" data-doi="10.1101/gr.174623.114"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hirakawa</span>,  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. A.</span> <span class="cit-name-surname">Martinez</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Sakthikumar</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. Z.</span> <span class="cit-name-surname">Anderson</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Berlin</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genetic and phenotypic intra-species variation in <em>Candida albicans</em>.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">413</span>–<span class="cit-lpage">425</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.174623.114</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.174623.114%26rft_id%253Dinfo%253Apmid%252F25504520%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjI1LzMvNDEzIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMS8zNTQ3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3547.17" data-doi="10.1073/pnas.0401648101"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>,  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N. A.</span> <span class="cit-name-surname">Federspiel</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Chibana</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Dungan</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Kalman</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">The diploid genome sequence of Candida albicans.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">101</span>: <span class="cit-fpage">7329</span>–<span class="cit-lpage">7334</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.0401648101</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0401648101%26rft_id%253Dinfo%253Apmid%252F15123810%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTAxLzE5LzczMjkiO3M6NDoiYXRvbSI7czoxOToiL2dnZy85LzExLzM1NDcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3547.18" data-doi="10.1093/nar/gkv1226"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kitts</span>,  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. M.</span> <span class="cit-name-surname">Church</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Thibaud-Nissen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Choi</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Hem</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Assembly: a resource for assembled genomes at NCBI.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">D73</span>–<span class="cit-lpage">D80</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkv1226</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkv1226%26rft_id%253Dinfo%253Apmid%252F26578580%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkv1226&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26578580&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3547.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3547.19" data-doi="10.1093/gigascience/gix085"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Korlach</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Gedman</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. B.</span> <span class="cit-name-surname">Kingan</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.-S.</span> <span class="cit-name-surname">Chin</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. T.</span> <span class="cit-name-surname">Howard</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title"><em>De novo</em> pacbio long-read and phased avian genome assemblies correct and add to reference genes generated with intermediate and short reads.</span> <abbr class="cit-jnl-abbrev">Gigascience</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">gix085</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/gigascience/gix085</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGigascience%26rft.volume%253D6%26rft.spage%253Dgix085%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgigascience%252Fgix085%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/gigascience/gix085&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3547.20" data-doi="10.1371/journal.pgen.0040001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Legrand</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Forche</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Selmecki</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Chan</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. T.</span> <span class="cit-name-surname">Kirkpatrick</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Haplotype Mapping of a Diploid Non-Meiotic Organism Using Existing and Induced Aneuploidies.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e1</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.0040001</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DLegrand%26rft.auinit1%253DM.%26rft.volume%253D4%26rft.issue%253D1%26rft.spage%253De1%26rft.epage%253De1%26rft.atitle%253DHaplotype%2Bmapping%2Bof%2Ba%2Bdiploid%2Bnon-meiotic%2Borganism%2Busing%2Bexisting%2Band%2Binduced%2Baneuploidies.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.0040001%26rft_id%253Dinfo%253Apmid%252F18179283%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.0040001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18179283&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3547.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3547.21" data-doi="10.1093/bioinformatics/btr509"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A statistical framework for SNP calling, mutation discovery, association mapping and population genetical parameter estimation from sequencing data.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">2987</span>–<span class="cit-lpage">2993</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btr509</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtr509%26rft_id%253Dinfo%253Apmid%252F21903627%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btr509&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21903627&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3547.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000296099300009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3547.22" data-doi="10.1093/bioinformatics/bty191"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Minimap2: pairwise alignment for nucleotide sequences.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">3094</span>–<span class="cit-lpage">3100</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/bty191</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.volume%253D34%26rft.spage%253D3094%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbty191%26rft_id%253Dinfo%253Apmid%252F29750242%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bty191&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=29750242&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3547.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3547.23" data-doi="10.1093/bioinformatics/btp324"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">H.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Durbin</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Fast and accurate short read alignment with Burrows-Wheeler transform.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1754</span>–<span class="cit-lpage">1760</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btp324</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp324%26rft_id%253Dinfo%253Apmid%252F19451168%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp324&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19451168&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3547.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000267665900006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3547.24" data-doi="10.1093/bioinformatics/btp352"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Handsaker</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Wysoker</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Fennell</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Ruan</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The Sequence Alignment/Map format and SAMtools.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">2078</span>–<span class="cit-lpage">2079</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btp352</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp352%26rft_id%253Dinfo%253Apmid%252F19505943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp352&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19505943&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3547.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268808600014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3547.25" data-doi="10.1056/NEJMoa1306801"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Magill</span>,  <span class="cit-name-given-names">S. S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. R.</span> <span class="cit-name-surname">Edwards</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Bamberg</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z. G.</span> <span class="cit-name-surname">Beldavs</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Dumyati</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Multistate Point-Prevalence Survey of Health Care–Associated Infections.</span> <abbr class="cit-jnl-abbrev">N. Engl. J. Med.</abbr> <span class="cit-vol">370</span>: <span class="cit-fpage">1198</span>–<span class="cit-lpage">1208</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1056/NEJMoa1306801</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DN.%2BEngl.%2BJ.%2BMed.%26rft.volume%253D370%26rft.spage%253D1198%26rft_id%253Dinfo%253Adoi%252F10.1056%252FNEJMoa1306801%26rft_id%253Dinfo%253Apmid%252F24670166%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1056/NEJMoa1306801&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24670166&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3547.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000333426000007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3547.26" data-doi="10.1371/journal.pcbi.1005944"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Marçais</span>,  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. L.</span> <span class="cit-name-surname">Delcher</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. M.</span> <span class="cit-name-surname">Phillippy</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Coston</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. L.</span> <span class="cit-name-surname">Salzberg</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Mummer4: a fast and versatile genome alignment system.</span> <abbr class="cit-jnl-abbrev">PLOS Comput. Biol.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">e1005944</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pcbi.1005944</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLOS%2BComput.%2BBiol.%26rft.volume%253D14%26rft.spage%253De1005944%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.1005944%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.1005944&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3547.27" data-doi="10.1128/MCB.13.1.551"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McEachern</span>,  <span class="cit-name-given-names">M. J.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. B.</span> <span class="cit-name-surname">Hicks</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Unusually large telomeric repeats in the yeast <em>Candida albicans</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">551</span>–<span class="cit-lpage">560</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1128/MCB.13.1.551</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DMcEachern%26rft.auinit1%253DM%2BJ%26rft.volume%253D13%26rft.issue%253D1%26rft.spage%253D551%26rft.epage%253D560%26rft.atitle%253DUnusually%2Blarge%2Btelomeric%2Brepeats%2Bin%2Bthe%2Byeast%2BCandida%2Balbicans.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.13.1.551%26rft_id%253Dinfo%253Apmid%252F8417351%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjg6IjEzLzEvNTUxIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMS8zNTQ3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3547.28" data-doi="10.1186/gb-2013-14-9-r97"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Muzzey</span>,  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Schwartz</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. S.</span> <span class="cit-name-surname">Weissman</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Sherlock</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Assembly of a phased diploid <em>Candida albicans</em> genome facilitates allele-specific measurements and provides a simple model for repeat and indel structure.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">R97</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/gb-2013-14-9-r97</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D14%26rft.spage%253DR97%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2013-14-9-r97%26rft_id%253Dinfo%253Apmid%252F24025428%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2013-14-9-r97&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24025428&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3547.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3547.29" data-doi="10.1016/j.fgb.2006.01.002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nantel</span>,  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">The long hard road to a completed <em>Candida albicans</em> genome.</span> <abbr class="cit-jnl-abbrev">Fungal Genet. Biol.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">311</span>–<span class="cit-lpage">315</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.fgb.2006.01.002</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFungal%2Bgenetics%2Band%2Bbiology%2B%253A%2B%2BFG%2B%2526%2BB%26rft.stitle%253DFungal%2BGenet%2BBiol%26rft.aulast%253DNantel%26rft.auinit1%253DA.%26rft.volume%253D43%26rft.issue%253D5%26rft.spage%253D311%26rft.epage%253D315%26rft.atitle%253DThe%2Blong%2Bhard%2Broad%2Bto%2Ba%2Bcompleted%2BCandida%2Balbicans%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.fgb.2006.01.002%26rft_id%253Dinfo%253Apmid%252F16517185%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.fgb.2006.01.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16517185&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3547.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000237619200001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3547.30" data-doi="10.1128/EC.00041-07"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Odds</span>,  <span class="cit-name-given-names">F. C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.-E.</span> <span class="cit-name-surname">Bougnoux</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. J.</span> <span class="cit-name-surname">Shaw</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">Bain</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. D.</span> <span class="cit-name-surname">Davidson</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Molecular Phylogenetics of <em>Candida albicans</em>.</span> <abbr class="cit-jnl-abbrev">Eukaryot. Cell</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">1041</span>–<span class="cit-lpage">1052</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1128/EC.00041-07</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEukaryot.%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1128%252FEC.00041-07%26rft_id%253Dinfo%253Apmid%252F17416899%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZXVrY2VsbCI7czo1OiJyZXNpZCI7czo4OiI2LzYvMTA0MSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzkvMTEvMzU0Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3547.31" data-doi="10.1093/femsyr/foz032"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Opulente</span>,  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Q. K.</span> <span class="cit-name-surname">Langdon</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. V.</span> <span class="cit-name-surname">Buh</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. A. B.</span> <span class="cit-name-surname">Haase</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Sylvester</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">Pathogenic budding yeasts isolated outside of clinical settings.</span> <abbr class="cit-jnl-abbrev">FEMS Yeast Res.</abbr> <span class="cit-vol">19</span> foz032<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/femsyr/foz032</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEMS%2BYeast%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Ffemsyr%252Ffoz032%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/femsyr/foz032&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3547.32" data-doi="10.1093/gbe/evy166"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Panthee</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Hamamoto</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. A.</span> <span class="cit-name-surname">Ishijima</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Paudel</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Sekimizu</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Utilization of hybrid assembly approach to determine the genome of an opportunistic pathogenic fungus, <em>Candida albicans</em> timm 1768.</span> <abbr class="cit-jnl-abbrev">Genome Biol. Evol.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">2017</span>–<span class="cit-lpage">2022</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/gbe/evy166</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%2BEvol.%26rft.volume%253D10%26rft.spage%253D2017%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevy166%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/gbe/evy166&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3547.33" data-doi="10.1093/bioinformatics/btq033"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Quinlan</span>,  <span class="cit-name-given-names">A. R.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">I. M.</span> <span class="cit-name-surname">Hall</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">BEDTools: a flexible suite of utilities for comparing genomic features.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">841</span>–<span class="cit-lpage">842</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btq033</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq033%26rft_id%253Dinfo%253Apmid%252F20110278%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btq033&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20110278&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3547.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000275243500019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3547.34" data-doi="10.1002/ece3.1919"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Robinson</span>,  <span class="cit-name-given-names">H. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Pinharanda</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Bensasson</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Summer temperature can predict the distribution of wild yeast populations.</span> <abbr class="cit-jnl-abbrev">Ecol. Evol.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">1236</span>–<span class="cit-lpage">1250</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/ece3.1919</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcol.%2BEvol.%26rft.volume%253D6%26rft.spage%253D1236%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fece3.1919%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/ece3.1919&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3547.35" data-doi="10.1038/s41467-018-04787-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ropars</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Maufrais</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Diogo</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Marcet-Houben</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Perin</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Gene flow contributes to diversification of the major fungal pathogen <em>Candida albicans</em>.</span> <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">2253</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/s41467-018-04787-4</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun.%26rft.volume%253D9%26rft.spage%253D2253%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41467-018-04787-4%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41467-018-04787-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3547.36" data-doi="10.1073/pnas.0404318101"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sanyal</span>,  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Baum</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Carbon</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Centromeric dna sequences in the pathogenic yeast <em>Candida albicans</em> are all different and unique.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">101</span>: <span class="cit-fpage">11374</span>–<span class="cit-lpage">11379</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.0404318101</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0404318101%26rft_id%253Dinfo%253Apmid%252F15272074%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTAxLzMxLzExMzc0IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMS8zNTQ3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3547.37" data-doi="10.1038/s41576-018-0003-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sedlazeck</span>,  <span class="cit-name-given-names">F. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Lee</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. A.</span> <span class="cit-name-surname">Darby</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. C.</span> <span class="cit-name-surname">Schatz</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Piercing the dark matter: bioinformatics of long-range sequencing and mapping.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">329</span>–<span class="cit-lpage">346</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/s41576-018-0003-4</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BGenet.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41576-018-0003-4%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41576-018-0003-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3547.38" data-doi="10.1093/bioinformatics/btv351"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Simão</span>,  <span class="cit-name-given-names">F. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. M.</span> <span class="cit-name-surname">Waterhouse</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Ioannidis</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. V.</span> <span class="cit-name-surname">Kriventseva</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. M.</span> <span class="cit-name-surname">Zdobnov</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">BUSCO: assessing genome assembly and annotation completeness with single-copy orthologs.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">3210</span>–<span class="cit-lpage">3212</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btv351</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtv351%26rft_id%253Dinfo%253Apmid%252F26059717%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btv351&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26059717&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3547.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3547.39" data-doi="10.1038/nrg2950"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tewhey</span>,  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Bansal</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Torkamani</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. J.</span> <span class="cit-name-surname">Topol</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">N. J.</span> <span class="cit-name-surname">Schork</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The importance of phase information for human genomics.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">215</span>–<span class="cit-lpage">223</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg2950</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DTewhey%26rft.auinit1%253DR.%26rft.volume%253D12%26rft.issue%253D3%26rft.spage%253D215%26rft.epage%253D223%26rft.atitle%253DThe%2Bimportance%2Bof%2Bphase%2Binformation%2Bfor%2Bhuman%2Bgenomics.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2950%26rft_id%253Dinfo%253Apmid%252F21301473%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2950&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21301473&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3547.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3547.40" data-doi="10.7554/eLife.45954"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Todd</span>,  <span class="cit-name-given-names">R. T.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. D.</span> <span class="cit-name-surname">Wikoff</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Forche</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Selmecki</span></span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">Genome plasticity in <em>Candida albicans</em> is driven by long repeat sequences.</span> <abbr class="cit-jnl-abbrev">eLife</abbr> <span class="cit-vol">8</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.7554/eLife.45954</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DeLife%26rft_id%253Dinfo%253Adoi%252F10.7554%252FeLife.45954%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.7554/eLife.45954&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3547.41" data-doi="10.1186/gb-2007-8-4-r52"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">van het Hoog</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. J.</span> <span class="cit-name-surname">Rast</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Martchenko</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Grindle</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Dignard</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Assembly of the <em>Candida albicans genome</em> into sixteen supercontigs aligned on the eight chromosomes.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">R52</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/gb-2007-8-4-r52</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253Dvan%2BHet%2BHoog%26rft.auinit1%253DM.%26rft.volume%253D8%26rft.issue%253D4%26rft.spage%253DR52%26rft.epage%253DR52%26rft.atitle%253DAssembly%2Bof%2Bthe%2BCandida%2Balbicans%2Bgenome%2Binto%2Bsixteen%2Bsupercontigs%2Baligned%2Bon%2Bthe%2Beight%2Bchromosomes.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2007-8-4-r52%26rft_id%253Dinfo%253Apmid%252F17419877%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2007-8-4-r52&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17419877&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3547.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3547.42" data-doi="10.1093/nar/gkw1119"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zdobnov</span>,  <span class="cit-name-given-names">E. M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Tegenfeldt</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Kuznetsov</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. M.</span> <span class="cit-name-surname">Waterhouse</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F. A.</span> <span class="cit-name-surname">Simao</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Orthodb v9. 1: cataloging evolutionary and functional annotations for animal, fungal, plant, archaeal, bacterial and viral underorthologs.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">D744</span>–<span class="cit-lpage">D749</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkw1119</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkw1119%26rft_id%253Dinfo%253Apmid%252F27899580%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkw1119&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27899580&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3547.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3547.43" data-doi="10.1101/gr.234948.118"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>,  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. S.</span> <span class="cit-name-surname">Ho</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. U.</span> <span class="cit-name-surname">Greer</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Zhu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">Bell</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2019</span><span class="cit-comment">a</span> <span class="cit-article-title">Comprehensive, integrated, and phased whole-genome analysis of the primary encode cell line k562.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">472</span>–<span class="cit-lpage">484</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.234948.118</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.234948.118%26rft_id%253Dinfo%253Apmid%252F30737237%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjI5LzMvNDcyIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMS8zNTQ3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3547.44"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>,  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Minio</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Massonnet</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Solares</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Lv</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2019</span><span class="cit-comment">b</span> <span class="cit-article-title">Structural variants, hemizygosity and clonal propagation in grapevines.</span> <abbr class="cit-jnl-abbrev">bioRxiv</abbr> <span class="cit-comment">doi: 10.1101/508119</span></cite></div><div class="cit-extra"></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/9/11/3547.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/11/3537" title="Construction of High-Resolution RAD-Seq Based Linkage Map, Anchoring Reference Genome, and QTL Mapping of the Sex Chromosome in the Marine Medaka Oryzias melastigma" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/11/3555" title="Pharmaco-Genetic Screen To Uncover Actin Regulators Targeted by Prostaglandins During Drosophila Oogenesis" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/9/11"><span>Volume 9 Issue 11, November 2019</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/9/11.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/9/11.cover-source.jpg?itok=DjqkLGk9" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 9 (11)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/11/3547" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-d-W7t8BC1T1PavAa-YIZwKO6F8s5liJNDiY_7nkECHg" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/genome-report" class="" data-icon-position="" data-hide-link-title="0">Genome Report</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/9/11/3547" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/9/11/3547" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/9/11/3547" class="active" data-icon-position="" data-hide-link-title="0">Phased Diploid Genome Assemblies for Three Strains of Candida albicans from Oak Trees</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/494610" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-xgSSKsR2v8hch4mhWUoMm374VW-EvElcYkEF5X5K96Q" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/11/3547.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/9/11/3547" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-4y42EelwPy92qWKObhi4JKX0AkvD7dR6Je70W9nCq9E" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/9/11/3547" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/11/3547.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494610" id="node494610--2" data-pisa="ggg;9/11/3547" data-pisa-master="ggg;g3.119.400486" data-apath="/ggg/9/11/3547.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Phased Diploid Genome Assemblies for Three Strains of <em>Candida albicans</em> from Oak Trees</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Jennafer A. P. Hamlin</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-1459-3148" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Guilherme B. Dias</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-5462-9854" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Casey M. Bergman</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-9015-2707" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Douda Bensasson</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3547-3554;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400486 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494610--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jennafer A. P. Hamlin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Biology and Institute of Bioinformatics, University of Georgia, Athens, GA, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jennafer%2BA.%2BP.%2BHamlin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hamlin%20JA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJennafer%2BA.%2BP.%2BHamlin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Guilherme B. Dias </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics and Institute of Bioinformatics, University of Georgia, Athens, GA, United States of America</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Guilherme%2BB.%2BDias%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dias%20GB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGuilherme%2BB.%2BDias%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-1459-3148" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Guilherme B. Dias</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Casey M. Bergman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics and Institute of Bioinformatics, University of Georgia, Athens, GA, United States of America</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Casey%2BM.%2BBergman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bergman%20CM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ACasey%2BM.%2BBergman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-5462-9854" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Casey M. Bergman</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Douda Bensasson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Biology and Institute of Bioinformatics, University of Georgia, Athens, GA, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Douda%2BBensasson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bensasson%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADouda%2BBensasson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-9015-2707" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Douda Bensasson</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:dbensasson@uga.edu" class="" data-icon-position="" data-hide-link-title="0">dbensasson@uga.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/9/11/3547" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/9/11/3547&amp;title=Phased%20Diploid%20Genome%20Assemblies%20for%20Three%20Strains%20of%20Candida%20albicans%20from%20Oak%20Trees" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/9/11/3547&amp;title=Phased%20Diploid%20Genome%20Assemblies%20for%20Three%20Strains%20of%20Candida%20albicans%20from%20Oak%20Trees" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/9/11/3547&amp;title=Phased%20Diploid%20Genome%20Assemblies%20for%20Three%20Strains%20of%20Candida%20albicans%20from%20Oak%20Trees" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/9/11/3547&amp;text=Phased%20Diploid%20Genome%20Assemblies%20for%20Three%20Strains%20of%20Candida%20albicans%20from%20Oak%20Trees" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/9/11/3547&amp;title=Phased%20Diploid%20Genome%20Assemblies%20for%20Three%20Strains%20of%20Candida%20albicans%20from%20Oak%20Trees" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/9/11/3547&amp;t=Phased%20Diploid%20Genome%20Assemblies%20for%20Three%20Strains%20of%20Candida%20albicans%20from%20Oak%20Trees" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/9/11/3547&amp;title=Phased%20Diploid%20Genome%20Assemblies%20for%20Three%20Strains%20of%20Candida%20albicans%20from%20Oak%20Trees" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/9/11/3547&amp;title=Phased%20Diploid%20Genome%20Assemblies%20for%20Three%20Strains%20of%20Candida%20albicans%20from%20Oak%20Trees" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/494610/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494610" id="node494610--3" data-pisa="ggg;9/11/3547" data-pisa-master="ggg;g3.119.400486" data-apath="/ggg/9/11/3547.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Phased Diploid Genome Assemblies for Three Strains of <em>Candida albicans</em> from Oak Trees</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Jennafer A. P. Hamlin</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-1459-3148" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Guilherme B. Dias</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-5462-9854" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Casey M. Bergman</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-9015-2707" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Douda Bensasson</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3547-3554;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400486 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494610--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jennafer A. P. Hamlin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Biology and Institute of Bioinformatics, University of Georgia, Athens, GA, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jennafer%2BA.%2BP.%2BHamlin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hamlin%20JA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJennafer%2BA.%2BP.%2BHamlin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Guilherme B. Dias </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics and Institute of Bioinformatics, University of Georgia, Athens, GA, United States of America</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Guilherme%2BB.%2BDias%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dias%20GB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGuilherme%2BB.%2BDias%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-1459-3148" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Guilherme B. Dias</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Casey M. Bergman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics and Institute of Bioinformatics, University of Georgia, Athens, GA, United States of America</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Casey%2BM.%2BBergman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bergman%20CM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ACasey%2BM.%2BBergman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-5462-9854" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Casey M. Bergman</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Douda Bensasson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Biology and Institute of Bioinformatics, University of Georgia, Athens, GA, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Douda%2BBensasson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bensasson%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADouda%2BBensasson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-9015-2707" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Douda Bensasson</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:dbensasson@uga.edu" class="" data-icon-position="" data-hide-link-title="0">dbensasson@uga.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=3547&amp;rft.epage=3554&amp;rft.atitle=Phased%20Diploid%20Genome%20Assemblies%20for%20Three%20Strains%20of%20Candida%20albicans%20from%20Oak%20Trees&amp;rft.volume=9&amp;rft.issue=11&amp;rft.date=2019-11-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Hamlin%2C+Jennafer+A.+P.&amp;rft.au=Dias%2C+Guilherme+B.&amp;rft.au=Bergman%2C+Casey+M.&amp;rft.au=Bensasson%2C+Douda"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/494610/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/494610/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/494610/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/494610/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/494610/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/494610/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/494610/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/494610/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/494610/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/494610/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/494610/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/494610/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496044" id="node496044" data-pisa="ggg;g3.119.400871v1" data-pisa-master="ggg;g3.119.400871" data-apath="/ggg/early/2020/03/17/g3.119.400871.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/17/g3.119.400871" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title"><em>De Novo</em> Genome Assembly of the <em>indica</em> Rice Variety IR64 Using Linked-Read Sequencing and Nanopore Sequencing</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496040" id="node496040" data-pisa="ggg;g3.119.400893v1" data-pisa-master="ggg;g3.119.400893" data-apath="/ggg/early/2020/03/12/g3.119.400893.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/12/g3.119.400893" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Understanding the Genetic Domestication History of the Jianchang Duck by Genotyping and Sequencing of Genomic Genes Under Selection</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496037" id="node496037" data-pisa="ggg;g3.120.401071v1" data-pisa-master="ggg;g3.120.401071" data-apath="/ggg/early/2020/03/11/g3.120.401071.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/11/g3.120.401071" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title"><em>De Novo</em> Genome Assembly of the Meadow Brown Butterfly, <em>Maniola jurtina</em></span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Genome%20Report" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Genome Report</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/9/11/3547" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-9" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results and Discussion</a></li><li class="even"><a href="#sec-15" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Conclusions</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/9/11/3547.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/9/11/3547.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/9/11/3547.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
