<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/9/11/3813" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/9/11/3813.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/9/11/3813.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/9/11/3813.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/9/11/3813/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/9/11/3813.atom" />
<meta name="HW.pisa" content="ggg;9/11/3813" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Limits to Genomic Divergence Under Sexually Antagonistic Selection" />
<meta name="DC.Identifier" content="10.1534/g3.119.400711" />
<meta name="DC.Date" content="2019-11-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2019 Kasimatis et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Since the autosomal genome is shared between the sexes, sex-specific fitness optima present an evolutionary challenge. While sexually antagonistic selection might favor different alleles within females and males, segregation randomly reassorts alleles at autosomal loci between sexes each generation. This process of homogenization during transmission thus prevents between-sex allelic divergence generated by sexually antagonistic selection from accumulating across multiple generations. However, recent empirical studies have reported high male-female FST statistics. Here, we use a population genetic model to evaluate whether these observations could plausibly be produced by sexually antagonistic selection. To do this, we use both a single-locus model with nonrandom mate choice, and individual-based simulations to study the relationship between strength of selection, degree of between-sex divergence, and the associated genetic load. We show that selection must be exceptionally strong to create measurable divergence between the sexes and that the decrease in population fitness due to this process is correspondingly high. Individual-based simulations with selection genome-wide recapitulate these patterns and indicate that small sample sizes and sampling variance can easily generate substantial male-female divergence. We therefore conclude that caution should be taken when interpreting autosomal allelic differentiation between the sexes." />
<meta name="DC.Contributor" content="Katja R. Kasimatis" />
<meta name="DC.Contributor" content="Peter L. Ralph" />
<meta name="DC.Contributor" content="Patrick C. Phillips" />
<meta name="article:published_time" content="2019-11-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Limits to Genomic Divergence Under Sexually Antagonistic Selection" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Since the autosomal genome is shared between the sexes, sex-specific fitness optima present an evolutionary challenge. While sexually antagonistic selection might favor different alleles within females and males, segregation randomly reassorts alleles at autosomal loci between sexes each generation. This process of homogenization during transmission thus prevents between-sex allelic divergence generated by sexually antagonistic selection from accumulating across multiple generations. However, recent empirical studies have reported high male-female &lt;i&gt;F&lt;sub&gt;ST&lt;/sub&gt;&lt;/i&gt; statistics. Here, we use a population genetic model to evaluate whether these observations could plausibly be produced by sexually antagonistic selection. To do this, we use both a single-locus model with nonrandom mate choice, and individual-based simulations to study the relationship between strength of selection, degree of between-sex divergence, and the associated genetic load. We show that selection must be exceptionally strong to create measurable divergence between the sexes and that the decrease in population fitness due to this process is correspondingly high. Individual-based simulations with selection genome-wide recapitulate these patterns and indicate that small sample sizes and sampling variance can easily generate substantial male-female divergence. We therefore conclude that caution should be taken when interpreting autosomal allelic differentiation between the sexes.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2019/11/01" />
<meta name="citation_mjid" content="ggg;9/11/3813" />
<meta name="citation_id" content="9/11/3813" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/9/11/3813" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/9/11/3813.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/9/11/3813.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/9/11/3813.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.119.400711" />
<meta name="citation_pmid" content="31530636" />
<meta name="citation_volume" content="9" />
<meta name="citation_issue" content="11" />
<meta name="citation_num_pages" content="12" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="3813" />
<meta name="citation_lastpage" content="3824" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Katja R. Kasimatis" />
<meta name="citation_author_institution" content="Institute of Ecology and Evolution and" />
<meta name="citation_author_email" content="k.kasimatis@utoronto.ca" />
<meta name="citation_author" content="Peter L. Ralph" />
<meta name="citation_author_institution" content="Institute of Ecology and Evolution and" />
<meta name="citation_author_institution" content="Department of Mathematics, University of Oregon, Eugene, Oregon 97403" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-9459-6866" />
<meta name="citation_author" content="Patrick C. Phillips" />
<meta name="citation_author_institution" content="Institute of Ecology and Evolution and" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-7271-342X" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_journal_abbrev=Evolution;citation_author=G. Arnqvist;citation_title=Assortative mating by fitness and sexually antagonistic genetic variation.;citation_pages=2111-2116;citation_volume=65;citation_year=2011;citation_issue=7;citation_pmid=21729064;citation_doi=10.1111/j.1558-5646.2011.01270.x" />
<meta name="citation_reference" content="citation_title=Sexual conflict;citation_author=G. Arnqvist;citation_author=L. Rowe;citation_year=2005;citation_doi=10.1515/9781400850600" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=G. Bhatia;citation_author=N. Patterson;citation_author=S. Sankararaman;citation_author=A. L. Price;citation_title=Estimating and interpreting FST: the impact of rare variants.;citation_volume=23;citation_year=2013;citation_pmid=23861382;citation_doi=10.1101/gr.154831.113" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=W. F. Bodmer;citation_title=DIFFERENTIAL FERTILITY IN POPULATION GENETICS MODELS;citation_pages=411-424;citation_volume=51;citation_year=1965;citation_issue=3;citation_pmid=17248243" />
<meta name="citation_reference" content="citation_journal_title=Trends in Ecology &amp; Evolution;citation_journal_abbrev=Trends in Ecology &amp; Evolution;citation_author=R. Bonduriansky;citation_author=S. F. Chenoweth;citation_title=Intralocus sexual conflict.;citation_pages=280-288;citation_volume=24;citation_year=2009;citation_issue=5;citation_pmid=19307043;citation_doi=10.1016/j.tree.2008.12.005" />
<meta name="citation_reference" content="citation_title=Elements of evolutionary genetics;citation_year=2010" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=D. Charlesworth;citation_author=B. Charlesworth;citation_title=Sex differences in fitness and selection for centric fusions between sex-chromosomes and autosomes.;citation_pages=205-214;citation_volume=35;citation_year=1980;citation_issue=2;citation_pmid=6930353;citation_doi=10.1017/S0016672300014051" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=C. Cheng;citation_author=M. Kirkpatrick;citation_title=Sex-specific selection and sex-biased gene expression in humans and flies.;citation_pages=1006170e;citation_volume=12;citation_year=2016;citation_doi=10.1371/journal.pgen.1006170" />
<meta name="citation_reference" content="citation_journal_title=PLoS Biol.;citation_author=T. Connallon;citation_author=S. F. Chenoweth;citation_title=Dominance reversals and the maintenance of genetic variation for fitness.;citation_pages=e3000118;citation_volume=17;citation_year=2019;citation_doi=10.1371/journal.pbio.3000118" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=T. Connallon;citation_author=A. G. Clark;citation_title=The resolution of sexual antagonism by gene duplication.;citation_volume=187;citation_year=2011;citation_pmid=21220356;citation_doi=10.1534/genetics.110.123729" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_journal_abbrev=Evolution;citation_author=T. Connallon;citation_author=R. M. Cox;citation_author=R. Calsbeek;citation_title=Fitness consequences of sex-specific selection.;citation_pages=1671-1682;citation_volume=64;citation_year=2010;citation_issue=6;citation_pmid=20050912;citation_doi=10.1111/j.1558-5646.2009.00934.x" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=L. Dutoit;citation_author=C. F. Mugal;citation_author=P. Bolívar;citation_author=M. Wang;citation_author=K. Nadachowska-Brzyska;citation_title=Sex-biased gene expression, sexual antagonism and levels of genetic diversity in the collared flycatcher (ficedula albicollis) genome.;citation_pages=3572-3581;citation_volume=27;citation_year=2018" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_author=S. P. Flanagan;citation_author=A. G. Jones;citation_title=Genome-wide selection components analysis in a fish with male pregnancy.;citation_pages=1096-1105;citation_volume=71;citation_year=2017;citation_doi=10.1111/evo.13173" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_journal_abbrev=Evolution;citation_author=J. D. Fry;citation_title=The genomic location of sexually antagonistic variation: some cautionary comments.;citation_pages=1510-1516;citation_volume=64;citation_year=2010;citation_issue=5;citation_pmid=19922443;citation_doi=10.1111/j.1558-5646.2009.00898.x" />
<meta name="citation_reference" content="citation_journal_title=Am. Nat.;citation_author=J. B. S. Haldane;citation_title=The effect of variation of fitness.;citation_pages=337-349;citation_volume=71;citation_year=1937;citation_doi=10.1086/280722" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=B. C. Haller;citation_author=P. W. Messer;citation_title=SLiM 3: forward genetic simulations beyond the Wright-Fisher model.;citation_volume=36;citation_year=2019;citation_doi=10.1093/molbev/msy228" />
<meta name="citation_reference" content="citation_journal_title=Am. Nat.;citation_author=S. Immler;citation_author=S. P. Otto;citation_title=The evolutionary consequences of selection at the haploid gametic stage.;citation_pages=241-249;citation_volume=192;citation_year=2018;citation_doi=10.1086/698483" />
<meta name="citation_reference" content="citation_journal_title=Annual Review of Ecology and Systematics;citation_journal_abbrev=Annu. Rev. Ecol. Syst.;citation_author=J. Jaenike;citation_title=SEX CHROMOSOME MEIOTIC DRIVE;citation_pages=25-49;citation_volume=32;citation_year=2001;citation_issue=1;citation_doi=10.1146/annurev.ecolsys.32.081501.113958" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. Jakobsson;citation_author=M. D. Edge;citation_author=N. A. Rosenberg;citation_title=The relationship between FST and the frequency of the most frequent allele.;citation_volume=193;citation_year=2013;citation_pmid=23172852;citation_doi=10.1534/genetics.112.144758" />
<meta name="citation_reference" content="citation_journal_title=Trends Ecol. Evol.;citation_author=S. Joseph;citation_author=M. Kirkpatrick;citation_title=Haploid selection in animals.;citation_pages=592-597;citation_volume=19;citation_year=2004;citation_doi=10.1016/j.tree.2004.08.004" />
<meta name="citation_reference" content="citation_journal_title=J. Hered.;citation_author=K. R. Kasimatis;citation_author=T. C. Nelson;citation_author=P. C. Phillips;citation_title=Genomic signatures of sexual conflict.;citation_pages=780-790;citation_volume=108;citation_year=2017;citation_doi=10.1093/jhered/esx080" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. F. Kidwell;citation_author=M. T. Clegg;citation_author=F. M. Stewart;citation_author=T. Prout;citation_title=REGIONS OF STABLE EQUILIBRIA FOR MODELS OF DIFFERENTIAL SELECTION IN THE TWO SEXES UNDER RANDOM MATING;citation_pages=171-183;citation_volume=85;citation_year=1977;citation_issue=1;citation_pmid=838269" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_journal_abbrev=Heredity;citation_author=M. Kozielska;citation_author=F. J. Weissing;citation_author=L. W. Beukeboom;citation_author=I. Pen;citation_title=Segregation distortion and the evolution of sex-determining mechanisms.;citation_pages=100-112;citation_volume=104;citation_year=2010;citation_issue=1;citation_pmid=19672280;citation_doi=10.1038/hdy.2009.104" />
<meta name="citation_reference" content="citation_title=The genetic basis of evolutionary change;citation_year=1974" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol. Evol.;citation_author=E. A. Lucotte;citation_author=R. Laurent;citation_author=E. Heyer;citation_author=L. Ségurel;citation_author=B. Toupance;citation_title=Detection of allelic frequency differences between the sexes in humans: a signature of sexually antagonistic selection.;citation_volume=8;citation_year=2016;citation_pmid=27189992;citation_doi=10.1093/gbe/evw090" />
<meta name="citation_reference" content="citation_journal_title=Nat. Rev. Genet.;citation_author=J. E. Mank;citation_title=Population genetics of sexual conflict in the genomic era.;citation_pages=1-10;citation_volume=7;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Nat. Ecol. Evol.;citation_author=J. E. Mank;citation_title=The transcriptional architecture of phenotypic dimorphism.;citation_pages=6;citation_volume=1;citation_year=2017;citation_doi=10.1038/s41559-016-0006" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_journal_abbrev=Evolution;citation_author=M. M. Patten;citation_author=D. Haig;citation_title=Maintenance or loss of genetic variation under sexual and parental antagonism at a sex-linked locus.;citation_pages=2888-2895;citation_volume=63;citation_year=2009;citation_issue=11;citation_pmid=19573084;citation_doi=10.1111/j.1558-5646.2009.00764.x" />
<meta name="citation_reference" content="citation_title=R: A Language and Environment for Statistical Computing;citation_year=2018" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_author=W. R. Rice;citation_title=Sex chromosomes and the evolution of sexual dimorphism.;citation_pages=735-742;citation_volume=38;citation_year=1984;citation_doi=10.2307/2408385" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_author=W. R. Rice;citation_title=The accumulation of sexually antagonistic genes as a selective agent promoting the evolution of reduced recombination between primitive sex chromosomes.;citation_pages=911-914;citation_volume=41;citation_year=1987;citation_doi=10.2307/2408899" />
<meta name="citation_reference" content="citation_journal_title=Behav. Ecol. Sociobiol.;citation_author=W. R. Rice;citation_author=B. Holland;citation_title=The enemies within: intergenomic conflict, interlocus contest evolution (ICE), and the intraspecific Red Queen.;citation_pages=1-10;citation_volume=41;citation_year=1997;citation_doi=10.1007/s002650050357" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_author=A. Singh;citation_author=D. Punzalan;citation_title=The strength of sex-specific selection in the wild.;citation_pages=2818-2824;citation_volume=72;citation_year=2018;citation_doi=10.1111/evo.13625" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=J. M. Smith;citation_author=R. Hoekstra;citation_title=Polymorphism in a varied environment: how robust are the models?;citation_pages=45-57;citation_volume=35;citation_year=1980;citation_issue=1;citation_pmid=7450498;citation_doi=10.1017/S0016672300013926" />
<meta name="citation_reference" content="citation_journal_title=Genes (Basel);citation_author=K. L. Tsai;citation_author=J. M. Evans;citation_author=R. E. Noorai;citation_author=A. N. Starr-Moss;citation_author=L. A. Clark;citation_title=Novel Y Chromosome Retrocopies in Canids Revealed through a Genome-Wide Association Study for Sex.;citation_pages=1-11;citation_volume=10;citation_year=2019;citation_doi=10.3390/genes10040320" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_author=B. S. Weir;citation_author=C. C. Cockerham;citation_title=Estimating F-statistics for the analysis of population structure.;citation_pages=1358-1370;citation_volume=38;citation_year=1984;citation_pmid=28563791;citation_doi=10.2307/2408641" />
<meta name="citation_reference" content="citation_title=Sex and evolution;citation_year=1975" />
<meta name="citation_reference" content="citation_journal_title=Evol. Letters;citation_author=A. E. Wright;citation_author=M. Fumagalli;citation_author=C. R. Cooney;citation_author=N. I. Bloch;citation_author=F. G. Vieira;citation_title=Male-biased gene expression resolves sexual conflict through the evolution of sex-specific genetic architecture.;citation_pages=403-410;citation_volume=215;citation_year=2018" />
<meta name="citation_reference" content="citation_journal_title=Ann. Eugen.;citation_author=S. Wright;citation_title=The genetical structure of populations.;citation_pages=323-354;citation_volume=15;citation_year=1951;citation_doi=10.1111/j.1469-1809.1949.tb02451.x" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Limits to Genomic Divergence Under Sexually Antagonistic Selection" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11.cover-source.jpg" />
<meta name="twitter:description" content="Since the autosomal genome is shared between the sexes, sex-specific fitness optima present an evolutionary challenge. While sexually antagonistic selection might favor different alleles within females and males, segregation randomly reassorts alleles at autosomal loci between sexes each generation. This process of homogenization during transmission thus prevents between-sex allelic divergence generated by sexually antagonistic selection from accumulating across multiple generations. However, recent empirical studies have reported high male-female FST statistics. Here, we use a population genetic model to evaluate whether these observations could plausibly be produced by sexually antagonistic selection. To do this, we use both a single-locus model with nonrandom mate choice, and individual-based simulations to study the relationship between strength of selection, degree of between-sex divergence, and the associated genetic load. We show that selection must be exceptionally strong to create measurable divergence between the sexes and that the decrease in population fitness due to this process is correspondingly high. Individual-based simulations with selection genome-wide recapitulate these patterns and indicate that small sample sizes and sampling variance can easily generate substantial male-female divergence. We therefore conclude that caution should be taken when interpreting autosomal allelic differentiation between the sexes." />
<meta name="og-title" property="og:title" content="Limits to Genomic Divergence Under Sexually Antagonistic Selection" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/9/11/3813" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Since the autosomal genome is shared between the sexes, sex-specific fitness optima present an evolutionary challenge. While sexually antagonistic selection might favor different alleles within females and males, segregation randomly reassorts alleles at autosomal loci between sexes each generation. This process of homogenization during transmission thus prevents between-sex allelic divergence generated by sexually antagonistic selection from accumulating across multiple generations. However, recent empirical studies have reported high male-female FST statistics. Here, we use a population genetic model to evaluate whether these observations could plausibly be produced by sexually antagonistic selection. To do this, we use both a single-locus model with nonrandom mate choice, and individual-based simulations to study the relationship between strength of selection, degree of between-sex divergence, and the associated genetic load. We show that selection must be exceptionally strong to create measurable divergence between the sexes and that the decrease in population fitness due to this process is correspondingly high. Individual-based simulations with selection genome-wide recapitulate these patterns and indicate that small sample sizes and sampling variance can easily generate substantial male-female divergence. We therefore conclude that caution should be taken when interpreting autosomal allelic differentiation between the sexes." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11.cover-source.jpg" />
    <title>Limits to Genomic Divergence Under Sexually Antagonistic Selection | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings, {"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"u5eFXm8QNLZocuml50cW5gRl01-RNbffWK0NTUFUWgI","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"494589","apath":"\/ggg\/9\/11\/3813.atom","pisa":"ggg;9\/11\/3813","ac":{"\/ggg\/9\/11\/3813.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/9\/11\/3813.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;9\/11\/3813"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;9\/11\/3813"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/9\/11\/3813.abstract":true,"\/content\/9\/11\/3813":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022APt0jl6iRB8Rl0trEJFEIXRO03AFjl3cb8l+NBiC6ZqUiXk8RWYhrA9vEx5bYPQp\\\\\\\/3BRxRPoeT0ahF7ZxuFZUnlKiO8f3DLix\\\\\\\/n9K5UV0EynzrjZCdFSFhlYlZhQz3krH9HGQT9kgAKg2GZdqcaOy07G\\\\\\\/7Lgn0gnWO1f2wx+A9ebx8u9nqPggPRR\\\\\\\/Q5VwnGRU3kwQzZH\\\\\\\/95d+Ytappzfk\\\\\\\/zNGAZpxs2dlcwLi1tFtL6Ljn4Si6ICELw9oqwKL5m5mvGk9sppZxOosIG9vK4n\\\\\\\/Ejhh\\\\\\\/4SGo0nmgYPVUL94cyjv\\\\\\\/L2z3TE8U0Z0dyQ6FM5Kzs9eI\\\\\\\/D\\\\\\\/cXIhio6lM\\\\\\\/k1ySAYLePhjFs0NZuH+WgyT1hZG558VqEMcw+v4zjYbZvkckfsEN0yZiBUDZ5Mu+0sqaqdgxfJGCYJ4AsETqywbG2fVuV9387G8oXl0+saFWApXr1dNUveq0IPU09vSJAkHQuWBgHEfcNDmVYFrZyn+BLZf4QIfa2lqdzUaLLMRdVER1z+cz9YJl\\\\\\\/hutzBIJfejQGgOa3ixqcU2Lnrrh1MwT76oX2pQajP6CdOcEVxVRIiL7mBacvUMaSsp6hmRDQjIWepj\\\\\\\/Zxa9J8ZglVoQdfjSIG8kJeymfeYV7winhgJJYabrLZROrfvVvf1oepze3JLpZTdptQo3g8VQYxCuP+p6yZAkYdIo+w6x6fXjb28a8Et2bbLugcb1vEOlCk6IYHkkwq9FwkRjc+J3P1yiI4zPtU8FChgPoFfOW99EzN9jR3VLQ4raWLPfpOa7\\\\\\\/1NKSgKuXyjXDcdM9EbEdAnx2MQlh6rvNkCMsenRNr6buh9xTNdmbz\\\\\\\/s2OlsvcYkWGXnnObB9rx3XYPByFPs1FIXi9iVEArSNvQgRIBJGXj6iO1i8lCrBf6mKF992I694AaDxkrUj4nNBGgm6ULWq8bPSaZ65vc0Dqr3nVp4JIRS5m5Rswkw0P82ou+18ihVoNdqJTt8ZtpucY9SyHuJ15sA0R8xe1n+MJmySMwHenQ8uORwArFH\\\\\\\/6S0WFy2yABdYd3JGX2twKNAL+0D89tdmm8\\\\\\\/qMmd5pV2qk9+Zt\\\\\\\/Q6gUJDTQ24eC1jn4H+u0w4JSoBfdPzMw\\\\\\\/Odw0a6xcElL30PvKnTIreKGGjfzO5C5wntIlcFU7MlUZ+ZT6BjMfxcXRyUJqD+BLcObZVolqi4kD5buq6ps8qJc3DbSuk8lbzm3jgS+cuRt8mUONTqLK2FYDLUkX0ItY\\\\\\\/gN7tGocFEKWlOGLMiSiGIkHXWs4VFsAIdUOIoqWW\\\u0022,\\\u0022iv\\\u0022:\\\u0022Izf\\\\\\\/p6GqQFCHZ2bL3w8dMw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022adafd446294ee81aa75060ba2bd9d7e3165595c1b35caf6196221ad5f6f80af1\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00222153o5g40kT\\\\\\\/XLvVNJEpuPkW\\\\\\\/2E0GIzopp7kC3C2RSSAnWAsXqdldSWCAOSg9+nlovQ+HnqlvnxlqGXu+0I1gcZ4I\\\\\\\/ZDjSD9xH7zh9TobNXViY3IfDJKnJ\\\\\\\/\\\\\\\/qTSEGOAwwuysELy2j+leXiceFOlxb\\\\\\\/FZfqkHV2HATcKXrKI\\\\\\\/xAuVkggC4CAUUYV55tQBjCcR4h69YTCziPzfE4DkZF\\\\\\\/j82bAqX68UrIYFutrwSn3t5EhMiKGnIT2OAfyCiCktnzZ8u3DnKNHE5Ukcoy+NsOWk5DBBj7z\\\\\\\/MR8Cks6uKRgrHiqKT87fvB04sCVs+K3LH2SNF7cE929zEEUONWRn2Klt5X7OBTaM8+Sk8OS+x28vaN+aZttCmhHynrGNBLKsvG1sNGLcV8lV3W3BAyfmvT0xzE5\\\\\\\/96dsmLvgryP6VA\\\\\\\/Kx8kP+X71mEY+LbaYrqVpUqgG5nvPr4c1X9yktjXvCu8DgNAUkKhoXaTE\\\\\\\/EXPg6z73cgW+X3VvLv8pOvT+gfP3QMWp2\\\\\\\/go\\\\\\\/7t7aHzYJ\\\\\\\/f74noBotaxY2PBwUtuPfeNsYdhRnvbZ0GYpfrUhaL\\\\\\\/y2M7KuSs0\\\\\\\/W85aXLfRclULDJ72hsL7IPqDLj3CFIotvaM0rbEDWc2jES0Tp9ricE4tmcx4ZKzbZQmT62zs5s22TOo\\\\\\\/towYYsVrscHewkgeVXtvFmq3dLCbJXHKGd9Hs4\\\\\\\/HUtxM+vcnFj37HEik\\\\\\\/1sI63YeUpY9e1oLZ87+2QV+K8WHgNEQvRzWmcdq2SMiarXJuy6FeZycYz90gsclzgbYuWa9hfUNf9qQ5FffeQV4Pc5muHoDjSgHN0dy1zvGFt4OeB3NMHtA6swWxHoVF2sZ0SKOiIQgqMAkdgZh8mapFNlrjSqe9fpX69aokvqJPVzEtmDrEykuRDhpe3MIavxM\\\\\\\/Xz97vrMiQhdB1JHVWzi+u9fskDogAnyWwpSFtn9\\\\\\\/CI9TkF+MDeWMYxpyy4IFcDr8thGcxPVn4JaUhATLQ4xNrVS8kERvDDEOx+6S+Ajc9K3n3hKxd8n\\\\\\\/Ke8I2gJEmWJSFHCf\\\\\\\/mFTNWvPzjreJ0AqcOcuOMHTjPBPBEr6PfvCFLesUj0Fv7Ky0DcgWKpHTA30qP6DWWphiJG3WE6IQ\\\\\\\/uGKhzKC5Dk1THwtrUVodUeY3jE0bfIrxopK\\\\\\\/Z1gSuHxVbjpDSLlB+VoqHEFKXLP0YW6GcU+dHyHbP0JIEAp+VjaYrvd1fb9lcRCq4mq65q1ztAd2skoWCF9rw9jLZrLfC0hLDFBmnRobVD9GtLaQ5eLcJt4I6GWYeFOGMrshN7dfGI6hzluQ7SPNNKcMvG6XxKTSioCAcZQNTxmCqCo\\\\\\\/4nxtfUbDT3OHvPzbreIWXiS09le2HKQngBa8cSx3hcISNcAIpaALyu9Qg+s8u3suUtySmQ7P8wr9BN1ZizLlccjbrbvKjFWEjlNM47R6+DDMA+9Jsq3sPnXfW8EvCj6RaCoyWgTx86mLXdxTbwJXCzXZ6PcTCfQfyOzBV8KAQXBBfHRDSWJ2+0y6egH8Zu9oskEEfe9aoaSmsI\\\\\\\/C+57AdCmgJmF+ISAyEwOBUcPBLgN8RG53cJ\\\\\\\/g67kSuwEruaDWn9QSfLC\\\\\\\/opB5izQdRRbMIIKkQS0c+tCpnnniu7JrengIlxdo63OudKZyh6ISEMbUPh+PpoDVnagyjfZxIZpM569\\\\\\\/5s0ebtPbRtQFSFqO5GDUTn4ypd6wy4r7K6D1TU+GGImtWw9Zj78vEPDto3Nu0TDM+OQ6wsxHRPyTsckf\\\\\\\/L3vjvTPJpCBVjkjSVQb9Jm2V39fdid2E50rPf4Jp+\\\\\\\/ueq+2HF0XSv4WRaDCmrq1XVUL0Im6NAXF3\\\\\\\/Tr+ZlerEqV2N\\\\\\\/FH\\\\\\\/3qeMRbxYvgraaY6RGceYp\\\\\\\/rz8qTSe9zkpylIlLG1C1T3B7DmzhXCQO4dXcBcA6LGsJndQJeMjJT1Q+hNSSEYznMEK1lBQgoYu1CDQaodPI2FOMdvw17ktXts7PoktWmoW6sZBxSpfyOXfcyawdU0TwotdEiE30zvEDvnWOuhkx9tEcB7h4Ynr2wKIUAR7cSaK6kFEPHM9aWWUOzqeTfE4uG+nZ2HQjYIGKMPi5BX8N4Kp8xBbDJphcCjTQnc7qCr9wx9a5bDChaosFrY3+9HB6EE2oqI5Y\\\\\\\/vGvCNIealGxfRP7SdHaG5B9srAw4+a1N4F+i8UgGGnAHB2KWY2Kf9PzxdAasIJ\\\\\\\/xK73xwXbFU90AJp5D7vRi04fB4SVU4QtoAzFFeILUodG2BFDcPA1tVq7d3uUa4L8Pa6+BWfcv1Y8Hx2n6g6WJJbzXzjS2qxx+A+IWoh7r+5Hm5hBNd8nVEjOcEoEm7sUegWKgXKJL02g75m5oUh6YjhVMx9iuOOMZckPKpU\\\\\\\/qmH9Hreqfw\\\\\\\/ulkRBZD0MafhwGiiotP8LPUItVG9Zv29V5wlTQ6\\\\\\\/+DgxYOJn2QmZ3CGH83ncrkMtmidRbaUIJqroHN1tKE1AZj8Gq\\\\\\\/VKCuXLRHb7XHz8V\\\\\\\/xMqFmcVqMLz8WYSi\\\\\\\/XbjDuoY4J3uoVa8Iv8WdZ7ULx0TNMmgS63nYhiFUZo2Swjz\\\\\\\/zz+U\\\\\\\/W5mVE\\\\\\\/PYqKg2tMwa4RiIT4Ugb\\\\\\\/jQSbHBQkkB3s7eJQUo89yZisZpMhaWQcRHSMMD\\\\\\\/gm2gyCs9w1sFOF\\\\\\\/\\\\\\\/h9dPGb5fjWmjcUukY\\\\\\\/2eemS5b7bEZShpmtffkca+LzUocBi5AOYQem9i6zhaHo0uDRvwrW3PDms21y18XmBX2EFxaei3P1fIanYbIkTjC3lT8upS6D\\\\\\\/acLkk6rSCts9jqbu+\\\\\\\/K2Ca6wYERAkrU5cOg1yyibhhkV3TtWz7vTfI6orYk\\\\\\\/bvJibmDGV\\\\\\\/eXGfjdpzZkWRr8CPICzDsCoSmcSsCgoMNAQ43px+0uf2iQrpnpUNmzoDMIFJ8qLv4yAEXs7MvDYUP4fLpZVTz5rMiv37+jk7QJfPBZFNvvFzg\\\\\\\/EuOHz0TDusvNJRo31+0jP6pwFBZp0B1HXZI7uH8Nriaspph5Q+PFAiBs376RAlVyn8S6cU1\\\\\\\/pU6l6wWQ4fpfEHZhAQqBdgQXT+DLApIvnf3LO6ce\\\\\\\/M7IrtKS14N6d8zhv1fyjvSxN0WyIAjWgCYpQLTwSvzRW+\\\\\\\/1OF2+oD\\\\\\\/c00p\\\\\\\/oNYPvleWki33aiNxgqm30NUh5iJsBhwS7RUylkwyHOOArvqxkvNcX1Wu7D37V9tJIM85nr9uXWReY9J50QdzB7dPlV7aCIaYF60oZ11m5YTC8v8yj0eit0VEtvnR7yool7EWxD+kvGSmRMlNuu40jlYJT8kpjLufagqMaq8vOApSrQ0Z87\\\\\\\/dXN3U488MHLOppRXEEUlzO5ZJM2MsT0C+KW2wISq0uRpVM1uGdfHNOLhuvQrIgSM1UCTtwEbnndQGy57xiG3bMWkm0AEh+yE3abbRFza4EABBfH9WXFfgBR8iQo2lTcDrPZ0dhNoBrKI7KPPe1+TivNhZCoaRBbSp2mgBvuO7FcaTjbdC2\\\\\\\/H+qeYCM0ksxLLbZBa5T2BtieyLdEAPmgwjWeXrwoTXP\\\\\\\/UqTQJAySPb1prNfC40KpPaD2RdMJFTJoHlBeW6XUYpfAFYAGErpDAwjkouf5dIfT\\\\\\\/Xq3F\\\\\\\/RE+rC0g83GkqY1lWqCFuuIjzLuw9q8D3cdtrIV\\\\\\\/w7zFbHJdAgphAr2HhHLfJQk\\\\\\\/1bN5YPoE504+tkTSo\\\\\\\/QCJrmbt2yzqJzITPO4QM+pAvx5RSAAtbukBhtMymIqrSqM3\\\\\\\/bqupQHTW4K3Pn+qvDnNEVPtNvst+oIw068I5sB+ZT0\\\\\\\/n8EAUeJk1H2zdRcslioySMpHAWW+fAiPMMOzFFy97JnT4rPtY3nOO0SN8IX+Dpw+gQ++tGlxuUFctX3tLsx+KWwYoWNPfouK6MyT9437Fi1eH6hQVpTeGvE9fpKWnlzWaPTBDnHTIkVrJfxen4ScRkzCa7unUH0Wh0odtH1Kro45Bmcgo9omERhxLifVeKqCK1fx2WYKQ0JKXanhLYRCYdrtuWOJdVaNczbqgncIbbr\\\\\\\/QParU+uq5N+li7ncW3zswrPL\\\\\\\/rrk9QT+Zez9AJT3M3ZVMIVaEIo7aA1O1iZvtJkaPFMAWNcK1hr\\\\\\\/rqvb5DnB859qeassBhlVFzHG7w\\\\\\\/EF5fYtEwo2yB6p1VlzcxUQ5jTIM\\\\\\\/L5vgyhT385k5l6sOcTH864U5pw7lEwtyqKPwUgvwFYwcnSuVytr0VFt3U99bS\\\\\\\/JOby+Fz2pvqO3Nb0wivqMytAwBSJ5g157xXrPsiFRGYA29QlGledxAbLlqgE1YmcET4bdWYTb5n4S1ATLQvtczz1pCE6nu0N78LQqzudeRyBqujBZVuGJhkM9eXHzhUe8rmH\\\\\\\/r7TfcOOSSxDZvSuJ\\\\\\\/q1q\\\\\\\/cpk94LPZSK5L\\\\\\\/hGo1zRJIMiV91DWueZ9C1Mt9OSJSnxVGAjgGAiYt9bGit7vzcRQS93EdvlzEBKo3BCVanES43FSjPfSTzvmuB5LZN1d1tPsMO3hr+Q9olhkz53JovMhhHraARw44EstOJl+Ke0Qrbf4UREsEqlSY5zRZWBGhOjOPK5p9cDq7Omg48W4sMheFYcoxZ2tfnRbbwgrvDVqYkZxskuInhszffJgHiHmNZFGQVBokDUVbG7uVygB6JDEUjWQDNXefJPN\\\\\\\/RGTbPZuNjkjsyx\\\\\\\/+LXnCiYGBpG7lafIlGxWCEnE\\\\\\\/m5JHq4ZcsDG7X6AJ4bl+wwFaD1LQKna7JzNkTI\\\\\\\/XMuZzViuQkZiTpe8BQ6HME+Z8A45vrTQoP7GcElch+Hv7mbL\\\\\\\/Q+JIdV\\\\\\\/lYvrbtRNCXK9nbYbt8D2VDkK6aLah\\\\\\\/p92U6vZxFvYAtUvFpAWwroZqt+jhuoL2Y00REAdocwk06K8zxL0tvcgriMxlbXYK3oZjdWV\\\\\\\/LzV2PFoQTwit5CATI1ZY9vf0LKv4vGcIf2jQUkB+74YZ7JDrEadGj0unu9eYf2ttQq4n4cHWpjbW3nJFANNJoKcq2msSZ5kFHDMaAO99++x6Cqzq6whZXNJb\\\\\\\/azWyrgt0RJ834+F6AWxN6CO8ed2IJtEw1ZkveJ5iH0iV8yqXREgh8QH24IJgV\\\\\\\/\\\\\\\/Cf6epFd2jdwQaCI5YycsqFcPMqWGNrqDQ20ZE\\\\\\\/2Gi4YoAt0Kd\\\\\\\/Tv78\\\\\\\/KAOCvE+4pPA7W3StkLcQ9S5A\\\\\\\/9\\\\\\\/PiSIjlkpLmg6RciIVPHOPX20dP9aGNCfWzg5vuMZ8f7w7fcjFqrwB3Rs5h50Bhp6SGkY4N9si8lgjvZwZkDcK+52jiqxlbMZeMIGcMByB+BEkJee9l3KNcqqCjADGIKiytIBjh\\\\\\\/8+1wbaVHPDmMLCOmNVHgWvbU+jvpcGW4dQff0j1n1RWSU+LiuneqKUwk748+\\\\\\\/YTXXRlyvNoHO\\\\\\\/nsjQsiuUEyJoaNyWCt0HElrhx1JQmwA+eAVOoS4++fo84wCBbbvpJYqfMKgPVIErDJ60vZ\\\\\\\/AcYbUnmUw6OtFlbNhlG2jb4grh032w80EfhvZ4\\\\\\\/Eb1ULhAnq5D63On4bL7nmFNotd5KLnXDf37K+id357FwbP2JBR1NaOEllP7Q4y\\\\\\\/ylFPtuSJQbzsMW5RVIoEyMShe6WCQyAJ9mRZdnj3UqbjhLEmaexbtxowNBhrog08bF5xbtfbwafA9NdNA4ZemQKwPWgSwB6pMVEVuHFdLPgH\\\\\\\/H4yCPB4SFysc2FePqPl+flrXmJJl53ov11XkoLipKxZQY9NNtlSNRN4pEB0s0Gb5sNtbQ9iJUJlcaX7GJS1yYZuo1\\\\\\\/PDpKO9DXc8ahC4EXVz2v3W\\\\\\\/J7lDyrrgIkofp0fAj36woAIND7UhNFE+4V9DfrvJH8JNAoq9uAccOrrW0SRps3pLKPzwURuqlCSCzCy6+J4IYDVNbhLflfyoC2x9WyS5mWa2jVDL8wSKJY9HJnlijMEQCpQ17x0TlqMMPLh32cWuOgd6J14pEyj7o+qfqXP0BRZzoaBFQXb4wIjKCojcXmAHZI2d3zPKFHS7cLoxu81wbb8r3e\\\\\\\/3MwROgSOOlVce7a4CXUl201aDANGQQzavAi3P9ThhfqD4tiJ81OXJ9S+frvkcvL+7XKkz2thN4G2emSUKMccANJHBWVxklVUSNCSDISRwJ4vePpy46ble2ry\\\\\\\/Jo0EZkEgywsgHeNd184eeaBVbeplJlV9zGFyYkF1LA1cXKjEn7nDkdV+2llN1sklvq3qvWelLbCH7mxWfJ5cPsYBvhZEX7j8+kFZHUyiNz+XlG4keM4Ca2Di4m3Rftvfi2e94NKH\\\\\\\/YXLLXpjRqe5C5ygCnbaE0JflA8dc++pXMcSfk5yLLl2kmScDoqbHafgKWrpSwCbCWKDwh4eucQ7xubhvDaC4K2Ez6UQd5vwh\\\\\\\/PxMfiWbMEUlx0hlYudYjm3FLuQm5aGtnqopuLE+ak5tfSJEITNM+DJdNQeMt1+ePcPKa5YJa61edoSgbzA9g\\\\\\\/dKshGvOZly2PUGL\\\\\\\/VPs6iVVNFRuvG63OOyuhSzBFul2ZbnuR0UqSmFZpnYbdEeKLfMzIK15Bu9YycxltvU58DQygeLkFuxoMvlKkhmrlkqaywTNwYx01dK1eHDyf4rmsernBFQuw7sOVfbbNBNkVLNN4ZNEgVzGSe9S9hOUmveJoE14tF6tgUOwLftmK6pc8Z247AN+1JNg1mzX3jjTL5TVI5OVBieqWe3QSJBEhSgDRMtuRkQAoRkxESPjcK4PQRPEpJpnehJYOlMLM38rj9zEJUCOkSUxqM57QvY4Va6cUfczHGFQINFxyfbnYFeIjF+KT1NruqaGguPdyKegWgzHgVQblYMsed4bvvhnmzGZ+mE79\\\\\\\/tRbUW+1m5cBQWKhMN9CDy5drXhz4fHrw2n7sugFebwVxM\\\\\\\/oWsqIvcFxPhJ\\\\\\\/a\\\\\\\/uVZFHzG3tvYMbln0eMF3OEyHhta8JNaJnSihhRzKsSE1Xzvy4N+xxUJGT8zhPFycRqTna+SxXG+hfL9eS8lSBwa+H5LqL5L+QVV0ErlPY6CRLIfvlTDrvEnUrVoFHc3wOlta8fSn58kkEZcyaiyX4616J523TvYfLszU4pKqQJsRRAdfa9VypNE4QYINP+CJWXFn3zNPf0APv1TAApfmCfYNQwZTicL2ZGmSKCU5OTWygxIoZvEBKeEIlYu+ZNJfENbJTUjJqvi+2Rlzg8CL4EyhO7FCKYJxrEp4sSBFoERvNFn63lczTc+Oe+td7sRLTq6STiLLMXwGsWV2KGm0D1L9eM5q9FP1MQ8gkebyqeFBPPHQqU6NhPoprZxnqqyYbLmh\\\\\\\/BJvipmYdR2yHvWJFmAHEdjTBQXr9SiXfxMMY62sno8sHMAtsmFz8+uobArJaXSzNpGZEwCZFOb7HX9INyi4CeC6WGtba9A14v\\\\\\\/s9qABGSWzT6eO1OQdUvDpFWs3LvKVcJVRWGQqj6nz7dUeINd40WtrpxMkIWtbfrhTadmPE5koQN8Xn1KyV3jV53SFIKZe2JVHTqrvAVpKwAknNNjYHZqM2ii8XcVR+7ScsopMWwwLdxQvE0GEtGh7h55XZWPUtrK0+kwDPSMvicReODYQWuYGgqp0s48xaHD5oe9JCiJoyvh4byJNqP+q4UWSfE0o\\\\\\\/KT2dkt01\\\\\\\/wQdCIJpqG6RgkXlHO\\\\\\\/Eil9WxLK9gMochnDK+lh7halIO3kd0qqEmVn0pHxpVsywjBDJOibiICN3CkZ4lZ730JZ8qMqBHZJS41yLGbQ7zNOtGKm1LNe8xXTw4RjM00bpovSLRSbe7RZ2H98kwlryj6OReWlFHP\\\\\\\/1TcE+hEWxsBOAmqK3V+MTIIiLyysgYMAG386ZCH7dRDIViRdhz+MK\\\\\\\/pjbTWRrsiBKiou4tci10JSc3TpBZWbXYAYC\\\\\\\/AoxJW21uwIEltjwSv7DJ8nUvy\\\\\\\/JXH6WPhS26hqzt2\\\\\\\/35kGMGvLsAAiiJoxAKlipMpjTobWQwU3qdUAJjvNDFZWbQKsevs0\\\\\\\/SS7XU4cUPeuWidf27Zg2Mixbjsd1H5I7W+qKfkUTRjbgSeQ1uei5cHzDAmfDfWDp7UAkeNTSBJXdHe23\\\\\\\/xUoDECmBFLRZ80ozAzHpS+YfhXgghOsQOXKsixdSm0tW+MK2bzvk4t8C4fIFasTEtfO8ZOkkauUhyFUNguALatAAIJ7PbhdrEqlP0VlrhNE4l0lPM+EMglEBlFo5pF9v9yruYYuDtqckJBRDsDO10N4YIk9StKduVvL4yYo38ZRmNeM4P8nJWENLDPtN9S9al3PyMyyLMCtP1FR0mjbLpQDsK9lESFbHVVVnFgrIFCnEuHUG+DjMvhBMzt2jHn1jREIhF\\\\\\\/UHf2jRyAh83ztaHuiOOh49eVmwVYiNkWV3ZfDzBMp55jl8Jx7A9gtCX5XNxA4edQpA47iqcPPZG8kYuTye1otdE1cj9I1FBRsLXZlHK8MKq0jPyFcrdKKoFmrrJGmyHzEJ1an4LW5AYmbcV5NCPVeykYdTiVq2p94eDEY4FIIqGIACmOX1WH8+ARbC6fQ\\\\\\\/uGUTxUcmG3KBubkDAQaYIhbT+ZbIJqE7DxFAwaWZSO0Z3aWkNP9jqNtMdGD2mwIJ747uj9i\\\\\\\/LstwEGqDI13FOfosfqkzuvtyHtvrozRPS3iQMdzaICj8bSW7\\\\\\\/eDTJ\\\\\\\/FpmRXx48odXVLYJsuASoIbONSs0MEkFmrn1F5MdUa5Fiqj\\\\\\\/\\\\\\\/9yXf5xAZBkaI5605kosrq0TSR8rJla4Y9bxoRZ\\\\\\\/5Q9AgeG9B7f\\\\\\\/NiAzD36Ta4IvY87VOGnTccFb4SzPS9lZGTrx0wH2u4DUF2OjyQYrtkUrt\\\\\\\/f4COfR2aFkZ3DrzkjbxyUZxuHqPf\\\\\\\/PqzL8mOX0LoZUUdDI4VqfzoCU6O7z5Dj+UXo8yjN6\\\\\\\/oLnO4CC5hZjJWvzgviPPn4a2zDGeKjySzOzTZoPJgtWmdQwxAvc89VAdx0OJ4wsBcH5ugaPoRBckPGiC2Pc60zg\\\\\\\/P1Cz2euwhdib+hgY+ixOzmUnuy+yP0zovpbk7N6j2hUddtSwL1OJ1OWSWFPkjYdHbfE3bavOeZSjCImvAQGdfAC4d6kRq\\\\\\\/iRKUyOrwcCgi+MTjzfAUInj451LrATPGbP8ioHjcHMzZNlnNGYYoOYkc+k3xmV5wbfX\\\\\\\/K4bBvUnL\\\\\\\/j7R9M0QtuoICIlSOp231ulqX2OEa1DikaKGSb\\\\\\\/lb2d3xvV442BDf16w6BqU8w\\\\\\\/wpJoZ4X1ZjjpNUcWQ2PKSZkO3PwwsXRNFZ8b8+q1S9v6CMs8aJTGD8XFmUiJHisy503U5dDT0ulCqEnHXRDKBVSY3p\\\\\\\/daezXpm2j71Q2vKzkrnVagbFn4ysOr71\\\\\\\/nKzlkSii6ge1vCP1nvrUCZ37dgfDsi95k3nJqBGQWERrfWBeyMFUxqARityPclt6WiB3FaKvZtr+DBw2m3+sS\\\\\\\/5bqO17NYIen5z4HLciX71BRxs6YQ8Zh1gJxtgYiXk0MDSkebVI4KJRu8CykfVPqVsD2S2zwvKPA71xAjjkw\\\\\\\/Egtt5Me65V6XcN0FBFh59mEDJ2XrMGsglH\\\\\\\/uZnpOlmow7sHKXMzSBHuPRkJUzHChf9LS93cAht7vM+psd9BXDZBSu3fDhBV0NwTn7hBhwMOq87sbk+84bBjXvtd7GcSytugRs\\\\\\\/cLRTF7djaxDvc+DNWnnNv\\\\\\\/AIjWN1x1MwD+ySN9kz88YgUB\\\\\\\/qYYnDl0z4LXWgoJMMplFe5KT8+dPOLVCTT6E498AheeGNKp4lkzgzh7sQJENknAqRryOXp+WrSBbM0bpyKHy9GPuvIce+\\\\\\\/t2O0Q4Gsv1TfGka8zv\\\\\\\/oOqG1F\\\\\\\/rKRkHGB\\\\\\\/CcUKcjtjqvIDnYUm90Jlr8BJph\\\\\\\/uooyynvF2U9TKXcr\\\\\\\/qSE\\\\\\\/+Qcb3Mfe+DZjSPwxW7OO3gNU7ioakTqbQDofr663nbjUfW+Irq4zV0MNsd168fRxhv696aMRnfQMcUTVGY3Vss1Brxppv4T6Ye+yzY85rjL6kH3pIrwwyFBlSfRTfG6YTTVzw6QmOj5HLSB+fjDH0QtB0tkbDjdKwgReuqy6dgH7fa70x34jLFCjKAdm\\\\\\\/PawczPq35H2NYuQBCmCqosy2fPsO2M49kemh\\\\\\\/Z1RquZvRo2BQdv2qhkrLTkYTnq91VkM3QywhyddslmiSuc7o4OJQjV\\\\\\\/d78c+vx+iwXcRkoQ2IrU2cTnZaXXXD+GVIJtiABTI0qf3lUnIp4wQNnSU8b2PdsApT9\\\\\\\/Ns8UwJbm3d7\\\\\\\/mHHNLNS\\\\\\\/tFlimWi8z1dNfp+7mX+nh6FQRRkLgKxZWD++\\\\\\\/fT0Vb4UFvhk\\\\\\\/6Vcu1it9KAGGR5XWGv47b6GGc+p9tarKMFUdOygm\\\\\\\/q1bqlw\\\\\\\/j73e5MVLJ5VePIrB3YNOsRZnqDFU+3Y\\\\\\\/o7Ox+ci0yk5SaXZ37dlUUKmiQ+3J+2ihhCOVSuyJaPaCp+ZSEsI39ZRSr6frP6tuARG3jdR938EwQioRH5FQO16ndN5TEiA95gPoI0rZYxr3GwR0KVYK9TFnDcuZgO1VkWJed6bxD4qfR1tTZQ3ujeIcAAJd\\\\\\\/6ALJReV3g+xZmlpBCaQ\\\\\\\/8LrhLxpTjGQmmnCPTzCd59VbE6j4elEBcBLvficelskW08sW\\\\\\\/wwvAo2eYe4DwtfxRM\\\\\\\/oils6GCDqD7VJYWodqvM0M+9jvA3fAVFFawFbm45gyIa2\\\\\\\/MZo65yHjuHtXpp6ONcrVXN9PzN3nEurrWTHQ\\\\\\\/dABwz2F+ThiGbOkM\\\\\\\/KUNyZ2ejOJ1svXSqtL0gdvL+NSHk8Gsk7jH0t03ldjNgQMghyqZFJyd8y8jfR0qxm\\\\\\\/BIhcw0ORecwmorNVp2s8XBXZPFp+xWEhJvmIASW72+9RMNZhdFGiMXNGWntrPeg\\\\\\\/LdMZX\\\\\\\/wMgF7E4U010gORo1w046osV461tafNmASJL4skRe4bhf6EJjwjSyAF6Wf071VtvgqRjHj3DhC+rD8WSHaIk7qoblRvOZ8Ek4ke5T\\\\\\\/hbRG0ICYw8VcrgFDIOuTTH+GpU6Iyp5KVFQbIpbm9XQleXzYTBGpD2F5Z\\\\\\\/vm9EM\\\\\\\/AXD0eApWT4hehsXIHeJQOXOaaswLig0HFk8EK2BvMnrZfwCHFFK\\\\\\\/5gTzzjha3ShjVYjhGVF8\\\\\\\/HsB2OmqK0KfCfvgJEl9YnHSQNjOxRxrfkVBJ2q8xygDc7HmkCDSX82grMfE50aNrqodovk9wD79wvv\\\\\\\/p9ioEZRaT3QvCxcNSO7YpTvVoHH+bEStvMXfuiKqEGVOvbuYpdxgstXBqUx0SLty5EAa7wP0VFrfJ3kJ\\\\\\\/gzVevl9u\\\\\\\/\\\\\\\/FFRgDPtLFDqZJaEw39TtwL7eKhwXBYJpcDhSYy6GL+X5sLJI81Eda50Mqg8S7zsy8HvO91LsAjrFB6Q0p0FOk\\\\\\\/PcZQd3DU+2KaC+VX679GOfbn8jS8xPd251W0cvLaetPNBUo7mEfT0wWi1MTnHghofekZTRgHkVg78RhlLo1eNYNXJkjixyVDMxoDIRDVEGrgUw8udPcOKljQibdOUfzBaliAP\\\\\\\/sq8cI0MNNUyib3kRYmTxYPCr8guZJGOsZ+2kzZ1UmJVnF07k5\\\\\\\/n2seuixgT2LTT3gWczRSc9TAl5E3zGMIEZ10f6Lo8XARh+8ZezuL264iim3LJmOqiuh6qcHPnyHbWyVpZz\\\\\\\/zD\\\\\\\/7KDhv6za3MAoWuoLwofjah+bBtKPUsJ6nXiz7nbbsuZwWwrg4zHFRznpTU\\\\\\\/+gbLWDff57Lrn6u37KTdluWuTCM\\\\\\\/BcTY6+Nh2GZVgjg9d+HxdEcSmWooKWHFolUCtGpAQXMnfgjEujp\\\\\\\/63SdsHickjHQdJ+G+OHdBDz1lEYE9rOhgzmPVEO444ADXx6ySjSDWlAETnqspie9z4EJAL9Fw80eO1Ye8FJ+YBpT16VGi3xssLCw1OfoaJzwKNfkvHaYV3tmDDYu5HdcP0wPvvM9ml88KrkIwo9uDrN3+qeo1xR\\\\\\\/Qe97O9nmMQB2K4+EjYRdwyn3gEdVl7BDqkGWByJ+GH7ri+zOUdWhysMu4QuxtDFM5eYPvrVn\\\\\\\/oUzaGNMGfyNFEk39GQACTl4h\\\\\\\/4Yb3xY2DngEwR3MnKCaGgJ5T1yhXtnnPpZULPBZtEMt01ZcwseYlCekhwIdW+hGFVHSi3X9EfrLIE8Pr9E7cO4BQFOOXDSmuLteItJJylhUXbjtoT3YCdAgFdDcjQRRRpBhJIx0Xzl3pFN9TxgjTWSBcE5UZA\\\\\\\/UNbtw7KNR0rtQCMA2PnvhprZzcaeKegkTjgbncnD9uODwBUaLTHZJmfkASDa8yIQgIrgy7eOIgeotT9a9qlaWNd9fHfygpu4lPnDhiJywVSl3E2E9Kj5DFaf77dk8rksJZIWg7fZYGON5xWqdtmRU5MzP\\\\\\\/juyn+F79uBYwsia50RFQnUnm54eNQ58cvBbs8NjE75hQder7GNH2yBJh6GCIfVoMpYc49HTzqRp4k\\\\\\\/eRYHeqQh\\\\\\\/3DCx0BHWJtibeVP2dPJYYe6y+Uz\\\\\\\/zo2pMgh6yFpjnVyxUpVohS2jvGRKehYXQ679FeMCJIHF6I4is8AWVu4+fAC6MZTFf1nRWR\\\\\\\/\\\\\\\/OMG8IOavxh7+59HAyF8t2bdzu5pe31ZZX4ZZnycCVKz4tsgt\\\\\\\/kBB0zZG2N70P0DdJi6aJk\\\\\\\/mb1wviW242AQJaVrcw\\\\\\\/tK9KQUhcBLithxpwxLJ3NhT0NOfdujzojUJqcSyZYuGk31WaZRyhy5AjM9cAvH9FJBtsIYRsQJgJmmSCJPbCitBr7DTp1RqOZrBSaK0lSi+Uy8jlOB\\\\\\\/0GTiB1BI0Vww7kmfx7QphnbIYUsURqix7hVG0XzA0VGzcKMQWMyQ1pPIKKPfSypCQ7JfNhiRPysA6RIoX53f03S4jv9Ll0u018iV6dvzTuV549LlbzuEniwjU\\\\\\\/BUn9oK69Z1vtAgf6c5AVH6T9nrORXK4zduxdDKiPz0TzMHp4h3Jg6B89KTgtj0XdsIXvMAo5X+0nJ1wIwh9GJquJw6zrh5\\\\\\\/rUtOyFCBvltWlH8JJf22Dx5ltCmEpSuGo5IB9e8RiL9vxrIgNqVGIkAIXUIK1It2xxpCX0\\\\\\\/MrZFw\\\\\\\/END6y5mP\\\\\\\/LoMUuS9eeir+tT6ViRZN591WoXflOA\\\\\\\/hOrV14gRfKHWdZ77RCXHHh8xWeXE3Cgq14Y8JJOyzdjsjuI1+jyfcn52imvQhoH9MfoOS4VSqv+XLZtcIQkgDyhxV2UtAxgk1Z7g7\\\\\\\/JcHEd9\\\\\\\/mzF+essS1AobwmosMW89Jitt6Ot4n6DyiYhjoqKy66TujcMwPw1BzRKM\\\\\\\/uB48zWVsdg0EoK5DdxHd\\\\\\\/Hp2p8f+behk2irTZ+mbQSRm2scawi7FC0jCGFjnXKSatpN8XDSxUqYMi1PeHK+KdXHU946AnzS88ycbuRoH\\\\\\\/uethhP3KWizpraJAWZeGTSnYnGdm9GvtWk5AZAhZ7hP7LALJp0oJqks4Ff1pdjX\\\\\\\/D20s490hCKiKULZHR+wacF0YmCd+6Mfuip7gTVmYbTTlwrA5cSZnSM9A3TBWhUOtfYBui1mOXDRJZhqXr0o0lTxgVO9XD5e8Q9p5TE5i0vcrzIzeQyEmqKS7X\\\\\\\/7ENFSLXr9u6wkK7Z6zaTHwEc2adJJhE2eD3AgQuyEH3OAL0bNOiAHAWyohxxLr6z86CjYhyhN2tYUITvJWzlgmKltexaiSVw+FbkUk5GFL3Vp6VdSxrCWrtdjA6GreTR+rD4ief5LD3qMQyjo89YOv1dwkvHP7RLRnQ8ab9uNLYKXdAR7pfUtZOJb\\\\\\\/6wdjDvxOHmoC8UfrLIL0EOLBJjj+5xFhUHyBVLxg2s6\\\\\\\/+Zh+j7HCwKtci\\\\\\\/OTL9v3SZ07q\\\\\\\/ZqqFtb6DqRRyE0\\\\\\\/It1SBk0ZrZfbfXXwG6vsRbvrSwCywsWKv5lbHIC5550lBx0uTVs5mFXwsFnh7+C5jVKUqgAXVp4CLBtKigax0qXVOLEBJl0rZGlKuzV8NfemhO1\\\\\\\/kN61qVcMWIwhJK1Vyn0+f82wlqnuOpresz9tjb8+oI0jm56pLeWj86MMdSG+OBxU0xq5uM8v5EW++4iHo0mxqoNj\\\\\\\/o7+70Lk0EEFWi5HUtMs+Pp5Qz+rpezM50kSqsw2dg6rIKzvhaFRZJaYRcvzjt9bkIxaZYg4rhQG6cLqqgr9u9kj7eVPRPdd\\\\\\\/YAsDzYUSDmS\\\\\\\/e2ohC5SWAxwxyNCQC3tIl6m+lYqgg0+v0h5bPiu4ejdfmJdLm1x2U635CBsXGpOtzuh7ZQP9wru88CQMODG9SEfnYmYoD6wkFvPKbXaU8uPH1HdA4xcljbkEH7hYpTQYduI2oAtw+buLNZe0B2Fw+wYqXx1xusTZM6Wh6dYhgrDEoaUZ6k0FAL1NynwNw1q8ObZmDE1Sya5ze47AL4y8Mb4VFCZTuaXOc+7JgX5s5xAQt8y7T7kjaEb0Pp3CaRDhsD57Zo3LretCTTzX2bIRGQyMBRqaYPfY8dylKo\\\\\\\/KfQDArVsMVfCu3SUntTCfRYMtpjDSiQd+SxCB6lPLJWklrO2Np4J6CROvIgt46spJDJbrZJaSdk88lRm+Pqy7H2GDLiItfbjlNZcFx5KM5kvkFf0ce3Cn\\\\\\\/Ue8uJi4mBrU9fThPZ17jP6KpT\\\\\\\/YMr1Fxe6wtZw3Zg2MgqEPrOaNsI3p2fgFUBEgmcMnsCemv6OITZtNHamWwm3k87tUlVjaxSOYiuiBGTLzWUM+feaip35NSzdJgGdupik8pypcY6uu7bWzbdLsZ0VlhXs\\\\\\\/w953oXt6VF+5s3BsFaN90zKW6sM1xG695+2w4oFt5+285kzQdjXAXItJ+dQVp4SvodwJXrAD8WtLJ1ZtAPprfQAEATydWr93xR7vrk1Ar7pL8UNJ+VxyjfVDm0KZXJNrKWF6+o7JQemAGPhTE2+xcAUqgYipq6b2\\\\\\\/1qqmtHMytR0HbqJRYkA8ZQWTPY\\\\\\\/LDpK9qRLWLFav9oEbuMaXQAvSeWiE2hNZ1P6JSwrwmFZpHmbaErc7n0Ga708Q9xJBVROaLtMLrgbPyGQbyYl9OIb9Kzy8\\\\\\\/Cg04o8bQGdrom+Ns0+vTfy+PZuOSXWshqkElQybd+EP307mRGeD2TxhqLtRuxl3W1\\\\\\\/ACPABYeNIjsbbjdGulMhAX1vl3Itfg72OESB4YNZ0IPiwj762lvZNG09GJB+6ucDZC791F1Na1cVubld6txRwvlGctqwXeSM5NySoIjqU0xbsnNyffGY1DLIAnVZ4S+Aq\\\\\\\/EsKFeT4nDO+jkvSpzwDIroTY5el0ZwJay5xNPlLuvp15X6rWhoAJlJw3XFCnwJxL9e+Lg62QRCQgkGbsprn7iG7URtMPCphrJVy7XaAfvoFc0LfRfhqqmjXceBnZ0O9coVyP1WYAT29F0V+Y4YbXZV\\\\\\\/2vVYYVtA7U6cQQ1wCC0VzIHqCABuLQzhEIkwLvW9bSlt8Ek0reWHISMKUCvEuNS8kpYEzstYA4Nb6+9Ad8TpiMm93C4Y7ozvNk5I0NV6jzQSEsW409hdCCr8YXbaunXdcU\\\\\\\/ws+ES3Ca2PsuVrwNNB+eaAa5Hcmjz1b32Pk3r5xk8Cn\\\\\\\/efmjD275aQy4+krkwiTWrO7CM0HztgbqjrzNCuZNctiHPULjc\\\\\\\/4Id85uYufyRsy96NLRCWHRGKVOl0fvYs\\\\\\\/XsezAqw28vgaY9L7RhVGrRSC\\\\\\\/G32tAAooPDly3r0ELVfexKAxh4i9DRZs9cgs+0dyUJh7+Ejj7H1IHbkz1ZWqXq\\\\\\\/pW4TBwSOVChMbNz9dnJhwWGGpri0hraAa6S4YyX1MdXiGtB+3OnCpT0YAld9B3TRCxRGh0W3OzMJfqSyyA+j1v7Jkkkbju0Q+YRMz7YA5zL6dHwlNk\\\\\\\/nB8bwza0je9yZnc6+iQWc1Sof+i\\\\\\\/LneoajGMmBwNOCluGeBvpnKO5z3TIaF7WwS\\\\\\\/FPGgboDxb6aEBnihZrIld3OVMtYM4IYWdcv2rvlQ9PufMlQTMyGMn82+8L4aRH1Ds9Bn4Cd4\\\\\\\/kT+4si0\\\\\\\/EaIKacjYdFX4OBHxGG9JFJknjS5Kf26mhQ\\\\\\\/vOmRx3pfbh56JD\\\\\\\/MMdC8O1y4bazVcmjpZk0k5lINmu+d5h5IPpRitNGe9y4JIsbU3MAz1ACzJonOACunVgY5FeY\\\\\\\/rOKhEBZbbkbJNfxr+jhYr0vAgac7ip4I5PYgstIAb\\\\\\\/TyqcbC9PYUwbBIKl1mmzauSeShhfXbKUIeaQiHsMnZJwYHanROYaD6ErPZsuqhFd+i0qUbsN+gA0eGdF\\\\\\\/4a09AULKYos3N+TloaDZ2lX5HzhL18Zp7a01F\\\\\\\/eZ4tH2npSUKycRmVP834dCa+bdPXZjsR2J2bGtvO8ca22Uwfvcy\\\\\\\/EPpvKj9b3laEUtIiJkyF+qTVivZn13wnbXPrl8KUZdaWFKsdjjbyLVOG6PywNr+7YfUFbDfmRTAi2AXjn8p6EdyK+yc9DyqrnuHbSzUaY9cTzLy+4oCx3NpuJD1B9Cx+NdQGa18kLK+Tok6nfwTchVeY9XxquXFE4z50ecDrRCK9YLdZbz1P7Nk\\\\\\\/q15p51Ry3oflqLqYCEHFRcX8CQ18+QMnIQV4aXW1fWzBJ2Q23Z0+ZAkEVnf\\\\\\\/2IOxKqFKyh4QklKUdAzY\\\\\\\/t6+UPWuVeCBxgPE6Mx\\\\\\\/LEdHGMbDYrkLLD4dyCSn5ZOYqLNDPzxzyxLfUG\\\\\\\/A+xABnT8eG7Geq5Wf1BehDOHmBiJG7\\\\\\\/93dHN9SRtZrt8iXzjRjSa5C6zZsAcsrj41Pegg2sBV5ErLKq0rq2fPAAFNDf0ejiNhSc344eYckWwFVmolUIuyzHYYxmLasYY\\\\\\\/JeevrZk7IMH0orjcx9knpN0pn\\\\\\\/GDMW\\\\\\\/cLSxq91Fei1SwQzmG2IDzCK+eMwxjdpISmy5Lj8p3bMuSMqxCQUlFaaWjQlKuIuUnA57YRvJgAT\\\\\\\/xR5bfvkujQ1mMXOD3O1dxxsWqQdgODwrTJ10X53YrUlsKw32vYw1yV+vMSo8Arh68m4t9c7W+yHF8q5eUWRcXNsYseQUnGuYyDi6BiIRfJHDMRbUJul4sSYgkIlQhdBs9QkqZVgskru4cURVVXFuu3jKFMfQZlimXZiLepU7H8SFvBy4zYJJY3LiVeJOfIvDh7n9N3NyoX\\\\\\\/YDj7bzeaPnjA52+LsIKJl97eMMCJ5sKuVZJfbgtp02FZ8+yzKw93a+kW9c1D5DJEs\\\\\\\/7krdi7PZtwqxkX09yJsBG6KK3fe0LzQu3DiQBUbVoSb3kZeM0ZQzOOtB4dEVDFtQyeeL88tUCjGNL9UVRR2YPx3A\\\\\\\/XhLfhAwR5RoR+NqcFsn2sTxsiiTl++QVahzY3QuaQDJv\\\\\\\/1ME6V8ANpSIgGKUhyuyom5pAeLOp3Ihafpl5X\\\\\\\/D0pcbwyIRMQxb7RcqPtVB94uY6mpizBuF41RHApbnHgyEUPxrIynxjBHy+wsEwGxOa4mqdo+\\\\\\\/HI2ETZA8QsSLQ7iYKUG9L2KaAkBlU8Z26pO21TmRKykocJXFTRwyQ\\\\\\\/BYrsJCUpyrBPQ9nrz6muO8C+7QPnwQdmCJ+Myg4bzUR+aIhV0Anr5jDXAXXTlQQaoXx8Q4fHeuGvEfRK+4JJX6Z53TSO0mXRR4TKAYs2kfl02p2u\\\\\\\/O9xXdI\\\\\\\/cnGb8Kna9JfCTfI84dL6xwIFKJxWZ86w1kX3EfbXE9ir77bUAZK\\\\\\\/KUO\\\\\\\/YaAMacDm10Q7kvuxjUOX5v9Vadcnsr41NVyCxWJNZmfTIyjD8zi62l20El6tl\\\\\\\/+s0v3LH65Z3gV9MIrD7sfALxLCTx4Nxr\\\\\\\/8XKj7fUG\\\\\\\/dlSMhe6c2tNOkop85XAP9xXQwZVensWNvI4nRFfHU2u4zZTeswbOVGOoccnzqOaS6S97z0wWxxsso4Wu63Bc42NzNiA4G1vxvBOqwxs3yrGwOfip4FTv64e8iDdUg5efMNdwYOuibS5pLh87HnrU90nhn\\\\\\\/4ejmwlFmMrTaeEpAZVe2HaxgGkh6aTiQ9AZi3GJHZEc2Z6+7hNY1Oi8vCWzhgwUCJ8BhBdGm2ZuvoeKhbtNuOqCofWRNLa3y0O7gAhgD5QOzMWU4\\\\\\\/Ijs9xxbP6\\\\\\\/6HjRN2lzYw242LxbpcpNj0Hq+hFQRP10hYIv3rlVnNfunnMpUzH3SYCvhfiv\\\\\\\/xYK4EDBt+a9VZts\\\\\\\/1b8CuHxM4TDHkEo37sIJB1nt7ovlasFMl\\\\\\\/6hCSTcvRUr6DQdgIFeYbNlikVmv\\\\\\\/QS\\\\\\\/N9qFfimiShQ2ha+DsCCpVs0UogQYSy6TMYNgbnIL9fceAqo6+LMhrZlTU\\\\\\\/Ix\\\\\\\/ts4ryAJ6AIn7FI0985VZ2xBinmKRX86\\\\\\\/oapE4vrfG4mOoeBYDFgpYfnHjaumcApCYvWIhd6HdNlGTpRqHv2m9XA8jEZQg5DjmFCOoU4Fp\\\\\\\/T6zEm5aM+YR3Wz8rPdDUxBflGKp69LsCstIR1pcimxOFUxdeLJEQf0q1\\\\\\\/mdTnz8K7PWBm\\\\\\\/P0Eu7p8bWTN5n3xNBEhv1sx+5bqF7m6xX9eGWsMeYGhg+YdszlKWL9dUwX54B6ycgKeJ1j8Iok+992eb0vAVRPhSt9WET2Ff7VaLL6ydFsl\\\\\\\/lWBOQl7NaGABlmHXc3zPdkeRhduwJ8IrcbKFsbNfhqi\\\\\\\/ZepxSobNfmRehL6TwXnK+B5mh2VY1tslj9l0xsjsx6GLC27vQV7Fr0wUcUu2wfKwwE8+QrWYZgTZEFFhtbB2yiLnA3hW1fWiTKfuQ2seyWVwE7ChqSh0tkWMsSLfA2Qlen+q2ff9bhR0DqbQ\\\\\\\/JjFGB\\\\\\\/FYaxMAkmsPWm8Psr2Hugje7VkB\\\\\\\/S9cmPORutT8YvEHh05cZzuJ499l0UX9Cb60j90wjnMxnFr9QUuh62dyhvIeNgmjlKzt1y3M8BNroS5NfclzDzWR6NqhaJwRp5MVDuPyK8UQeFwrfFmGuvv23R1rxCr62PTXrUGArAQl4ZCHP3ylBNSat9LiJmdNp7k6iwWZ43bKshEr653hS4iJxoubbTG15RCaQRV7ySSs1k+JKU5yByO5x0JGCMJjFmPFoz+pqpxqOMbjp1Am8QGIBDX+YbdYuEh73dab+ygNTbdAJ9wFe8a4ubi5Aa9pU69tSQZFbwkhMzwcS3T3AdXD4v4cZVAuU5L+dDyBC94qlbA20pmK51x52lCPbaQHb8wR0rfIu7Augnv5qm16J4FY7NQkB\\\\\\\/PVZkdXAod9s5cLB1GAVGsFTRGNKSxAX1JMTP2GwU3gdMjlDSECHbCTdVYwLIhO3UsWVfHhNH7Mu5g\\\\\\\/PYkKAiJiD+xydb1Rj0yjUWJpfl76KMZty\\\\\\\/GmrH3SVuk9aVByeqRnopl8qLOZMXpC5hQYIjEDN0iLAaum5AQriYUehr4RfvrmkKIIcl7RlIcCqsNRiwKY2JT0DsLGo7q4A2yYqGWq4EnRYPyXaJ6HF5HDOQC3e7apFoQL9qY61guHIrwJu+uGEMVs\\\\\\\/A3t8uHAw9uSSG\\\\\\\/SK+9Ikc4UGbO13+VPjfLdblBaabaVEIm1Ftx47MKWi6KomtkgGrD3tmi8Vl3WsDPNNPZl\\\\\\\/opj7UZBr4TX8nrtqIY2r7pgxuINo2GEDtKTkydVBOlSVCENC8CKTa5aVVntQ+EywqK9AbPXVmnpZuPpSEJeyjPwns3lzzLc79dpoDwouFm83QwrJXYHTFtXsSTo\\\\\\\/woB3I\\\\\\\/YghxL4x5GnYOGzJWClWnxPLiwFVCKRM1hJfacrdeZ7lTkYE\\\\\\\/LIzZDY6xJGsIurHELHw\\\\\\\/6CctjoEDRaM3s9kY9Ji1vagPeB0xj7iEMH5ZslVuOzO6f8uJ4FrRygc2eQLKQeCsSzaOqd+MoTAXaGZJ2VCaucEpvWAPsLWVlgC5sRSlHZHjj+TlAlaZ0zjBYs2jPKwL9SGH03nNwHZmZwUBI7KXX9FbQG1v11mrCuPPS8iJwjf\\\\\\\/hQAgOIzroOsf9+OIPxy031bVSyJ4MFo6rgXslBd5s83WQVFz5KdpNDBo+CwFCAp9DgKq2kh2ZUri\\\\\\\/Ih7E1b\\\\\\\/vEr9L\\\\\\\/kcJ0UoMhe99rSHwDhU8BcUwz+4aOkJQLMNEz2fbERqiSQg0d\\\\\\\/5vBQSDz3iXRZ5X+tHZVXvU2pwjqcZlBSfGhqlBdDsV1BuOYEFXcAXmjsPavbSaC+8Rji8qNNIVLbtSPdjPh3F0nuygmMhxqH713zSPvXrZIA1ZpUUOfoai5EkDv7L9uZY09dCttdQgt8YsoVW5FExyEgVgp23x81w\\\\\\\/Fm5QIqY714nAbebKuSxAkXYT6W2QhuG18clG14292hZ8K1AQZeGtGmZ8BeCyU20SSpVypPTWBCVNCYW9kpqQ66PQsKGeLBbxQoVXSm2wXFj\\\\\\\/QAz\\\\\\\/amBVSLH1725pA6XWQXxYcRBkzSS9GOa6rfOgmQj3FMQrm0RedyIXhfsyBvWZEIG63O0qUOCoZ8QLiVCWgKFgSGtnrRxWe5ZpaoZjDGJrPb1PyUS8vNPMckehaexmEzCGrfTJ99npv+K4nRZc248Ux6ewP4sDKuRpZdhvdJPlcmYwCuZ39xkxwsytvcOjzRBdj4aeMmlVIAm0lS6MUAq+2Mxdn8yFSlwBV2dCqVYw9tyQ+aa28+QnWYWldwE83H4nY5juGf7zeYi6y0SbPP1fIclQRE\\\\\\\/sCcZKsK\\\\\\\/D6rJOe5Dqozc+0uD9F4iGME3L4usOJLLth7T9wW3DPwDY\\\\\\\/An9v1buIMUVBjZVTITHYjD\\\\\\\/OjZLqvbe48iNc7RZOib7H3rUWEIk8sk+Lzu6W0V72utlVqJLEbL5gstBCIY8ey7deGc6WC0MXGlBPA0a3pPabx6plfw+lYn7XBHsSwxQYylAge3zfuMNEg\\\\\\\/VDeGSdsC9Lynt9VzGs77xrkR1ON7EyWCDoYS8r7YUthSNbwqOMmvuwZDHyNDWStEYT+KGApENUDyFtWsnR8q3lgSFQiMXyFBLIhWK31xTfEH2rX5mdFXH1a5yhqB67BnyBgLwf7OkoZ5z2vodqSHTdTCGSJ2lHOAh8B9uvWuypKxQjP439nrlU9rNwr3ROFAQo34SBQ8bxRgsAigI9LZYT9IJVWGSjRdBs5e0JBIKZ1\\\\\\\/ITZSpXFQNi0mCddvHv3SWxoW5nnQlSFKwGHrX+mYr2dZfu91Zj1GQOFMYNbDBre5siV8RAKy4hs03W28w2fCgUtQicz1qXo10G32GZ7VRW5M41kToBs1IAsXV2I5ewn7Pk8ut44c5\\\\\\\/p+7I1t6dambQS\\\\\\\/n6qG75aqfDu1O4TqLmdMxPnT0IYBCXA3VpzNU1CXltK2Q081qbhDH4Trwl3EoUhCVJgp117+hiCpWDMA9aIQv57cAoHeMav+g\\\\\\\/HikT0C0eBsl2dpXMsPXBvnINaGBRmq+o5q6N1NKrKt8ROvnHCq+xDshfx49fbMzPYgEefl\\\\\\\/gL6B3R2qK2z3wAAtMuLz8HNRT40n7iCqG7IhTZGeid79hi1ILHvD2CFrNtipjMiZlekyAYxrO+jVg2V\\\\\\\/gN8LzHubEvAiR+WueAjj2\\\\\\\/k9bKRvFrRC7Vqu\\\\\\\/WunSU1t+12rxLg9T\\\\\\\/3R4Nbrlo7L5PejWDGfdhkPYe1a5XENdNwIOraZaRpaHfepI\\\\\\\/grtOzrXzJB08CLAYey9CrCSp4+l7gupnIXHb1iSXIm6yFpcymL3PVC2hacvkGChaJl\\\\\\\/6vpCaDxng7z11gSMZb7rz8gNcaHCF7pcSGY2Q2\\\\\\\/Y7MtlouDZ2I3nTDCSZiqxY0xwodnvoNSBVlQk8RgHrbGHp5x+zUP8vW2giYxN4NdhPFiknnlCnrysfv1qCfTbJvgLU1OpJbzKDn8vj88B83yOVHq5mXOtdJ+8HFYUsjQPLqsnugcK30ohC8uA5HW53fqMCXZoWGQxHegc4GbKvM6EGNvfw2dTQBMAA60y1yyCW+P1qBdyX3phnKlImmH+QQPzE5o8qjjf1kz3JXRh2\\\\\\\/Ur9lGa6n81OXPfiB3mrRk9YG4X78BjtQBmv8YVmOaYQVr5AWtK4bcLYqW7EBIoYo\\\\\\\/TEn8k+wocrk4cjL82iKs+D\\\\\\\/S2nWaeLApkTbC9rPKOsSSaZ96QU2a8fplqgoCL40dQ4QJA0paFFS5Kn9OpTVoXHgBYJNqom0DcGgBVXFssc0vNCTT8vQZIxrqsO1VP1vVjN0oYfHO8eJHcAB6JEnwGBDl9E7Fzl6eFd04Tr8502So4BpgbbPw7rbmyPkN1VTU+DrYRr16z41ZkNbiVbfqEglZGa6Q7LALCRIc65lKYdortxG3drG9fUcLLEofc4KMZp6ssFrLABLwCcGKc7Mp\\\\\\\/\\\\\\\/b559F2HBwtWNwwlJs7md9UEmfXVSHjdvsdWhVc6ZGRXIzhykBTWnVYsOE6uQ4RRLM2c4k+TB3DXan8DYGKsjkPkOpVbQ6IJCD56tf8IAeTC8\\\\\\\/mhCc9sZw5Rju4kZr0og64Q3GNK9e6ReV9e9e1PsI1yoHfNbh70Mn4miJw+D6R8wfpoQ7KjX\\\\\\\/QoqSA9DZHL7fInX+dAeTh\\\\\\\/d3blV35iHrTRHe7Zccompd1Qs9v+8v1pmUSB5A23bEKJ3ifZfUDuXifCUEzmkcWym8EfIjsES\\\\\\\/\\\\\\\/01Cs0xTRUCrbqehhnULXUf1WylOzE2Vx6I3vGAZhuzeSbW\\\\\\\/WcT7s1Mzde1\\\\\\\/Hn6ku5IPjnDDMiyKdrmLCk1FoRsZbh2mP83HuFMwy5crB46IWhtbU7CN0kfEW5TQFCfyvQpq\\\\\\\/WJxxmGZU5dMThwqNWw1tu5tSjj5YwTThAb6gueaxcwpeoVmM9DIopLeLGT58gYrOC\\\\\\\/pfir0opnz1apRR5FHMxs6sWbAeHvEOg1vTjmXFMl08yLeDf4AkgL+e2gLIk1cEyLpAbY5qm0xo74euJmrwIwwvD0p1Gsir08+Tb2QeL0+F\\\\\\\/0SL0WC4rIDetfJ8\\\\\\\/LEJum2grROlajtsyN0ztiMluyi0hABiSdPk1APPUGGP7Sx24AZv6QYmFCUVSSHWP8ywHtjnSgOpvYnoit1k+Ps+wiUiU4fxsNhgOcaW3ywRS8bCVIRC7wfGK3bIhb2wgscMnJqHFisvuYnx\\\\\\\/2DS7ibDXZ6mIEx3a\\\\\\\/\\\\\\\/7fvjrawblGSEUWukPxu7eNAFZrRk\\\\\\\/WK+r6TMlrrDZb0YrCB6K0\\\\\\\/tBACz\\\\\\\/8pBQjtpfPSWTwo6x25LvdnGpz1n93pcxFM6fHmJd4pfK8N2zAzK6webI1GZfOgQEyNPB61Ow3UOoeoPiKc56Ok6gVcFlcQbj8xb9yQ\\\\\\\/QbkASycKol1CuFplwYCepPUGEXj1gOrMwtxxVOrtvKjIwL3dfESxtP5YFxlQntclLIuHv4ZZSpPbF8306Ht\\\\\\\/oOP3pFFPsei1qGVJmIVqRfZNo+gRQuRabHKBLaoD8vg\\\\\\\/IMyUyyG7GdPlXgyxljd\\\\\\\/keWz6WiGaPTvT8uvMwJ+m\\\\\\\/C2aANlucohB9yGUqa8CK+SHkDD4FdrUjV0Joh66+PN5FV57FmXKzNb0I9Acc5vLHxlQnU57FaYPz47w1+elHwGJohdxsfCbnemkJjrTiy1b\\\\\\\/N2N289heM5j3a9kt+7le0VxK+a2fs1jx2LbmmfvQ2\\\\\\\/EIgwAgEE4NE+jeM9IPMTOqxhrRsjpfiP2fxyjPephnwX78\\\\\\\/QMUQkwba4634bhzXmyjsjdBrrGLCpLjGISrYIEWWHrWwx9ykPFAGzomMxDz5vErsyyTNU1sqhwD7RdOmtMl5lTImQdQ\\\\\\\/eJpvFEfaOaIDHMxRtWdpCJ1oyP4dfnvIkPh87difuC8L5w4YX5ng02zkkghIuTYghDmmQ5m2n3Y7GytuDwPM7wBI5SQbL5VNzxQI498iCtm3u8+eXP3i5bPR1FTZLNatgJAM6NiqsqteoN950heD5mmQbTo345I4XJel0AEKvr8WHcW+Dd\\\\\\\/unrXQXFX2cdWlr9vUQYEJ6ru94cpb3zoKV\\\\\\\/nXQLZsBmLStXBWdpc9Nc2kjdAwXecH0wbcNIpVW4VHz\\\\\\\/mfo5A44feeDjQ1\\\\\\\/izCkSayr3dzgvgsKpEZaAbbxajsfd+RMG6aXUbSAFXwuN\\\\\\\/IfJJrKw6sh2flvI\\\\\\\/ZW5AmfN687qGTCIXhZDSHt\\\\\\\/UzC0+sJhK+f7CeS34lnPa03jjQxxR1U+5PLrbW5pRW\\\\\\\/5Z+EGeL2kiCbhzSDFwakRJdlUGjczpedg4JrgnlKgtND+jm2UNlIlmdqm8+yAsyn2ZMTXXz4uv8bV3rMPF5GARWDpUrqgE37TR0Vx6wM1KZHqtrBmX6Jh9kFSAxPUg7yINyY8OkB1MLTecZARm0KlRi23QnIBd+KhZMrpuwMheCb\\\\\\\/i2jaB5OWJEmGj26zx8MO+Ag5aSvOjl7hJHboWcGhp7KfGTKkSdjh0\\\\\\\/gyBPgPwTpd0W8ZLyRgVI+2QCiUAUHs+4TKw\\\\\\\/u9on2o6CrHdkgfU8wNqlI3\\\\\\\/QBfwvhjPE+SvdLLUhecJh6G1zSR7urfcYqWbQlzsjY6WNN9jDVz2cs4GhZOaoniWQXVzAA71aKxVdFcnhN9efESBA734MZFsKeNsDg2YAg+kFvFasIaT\\\\\\\/AICDIdV1XYjnFHkEkwpc9rOFeTOYNFFkAg+Ax1ZjBKzQFWyYQZkwOUxWYzdKKh4MKjdM4e8PcklXoRk2dHmfufZzBB4XQUYpIDZdYDskeGnZl9XPAhrdJkGTdWfaeb\\\\\\\/vHvUMJhB\\\\\\\/Muq2hokTq4k8\\\\\\\/Ey5wkAtaxF+ES3kC7nk\\\\\\\/Q+h8gNL973O31tM1MTAziw4Ce1R0aPfIh4ozdFdx45Z9hn9vt5RgSEC76e3qWaP7jilP3MhkRiFvaWi00tHa71tfIMSmHurK4iP8VBHzzmf+Uh9eaNQpWfHnSsg\\\\\\\/qPrz+moDPE8WZ2\\\\\\\/GjPB7PaaeN9iAH84jp2cVRBBh9i6PQ6PYHkeLgeU7n00kjxjrFa55Tf++jWVmcHRWStDnfssDExxNev+mM+1rWzvg4xSimSe9UzWRC93oTIngOHBfyqwsQOCSAG7PLj4TAQPJOjx7lKAssi9ibuXz0\\\\\\\/2RpcXbS\\\\\\\/zejksVU2X\\\\\\\/fxYq437vnbCi\\\\\\\/No3kn4p7eLmbt2nlD9DRoy0ljsKfWZAWvAdyvewfGy\\\\\\\/Rkcx\\\\\\\/47hM12vMbFVq+zM0bm2XYhwWfS4qlPuiA4q0voAp0HxXp2K1oBfr0xJgJQztWMzMsfCM53R5KnU0cFLI0+BFHhxhSHVfLAEVlx589ujQ8Mtt43Oti7n8YGtZQGtGONgMAXKWxb3dc9gi7sW0YwRjLTGsjV\\\\\\\/JwtN1BnanA5GW5jNzJVXQ3pT\\\\\\\/cOMTybaTojcBUifP1lIEVY9RvgW7LDwNU4U3c3QfspcTKjNiyjVU+lfRCNFbFLFjpSh1WeWer5QTI0XemCAa9gNHVxOlDp07XmIHqew6FCEavCbwTG3NhzD+lW\\\\\\\/veFUh+GEi8cZ53jdSAUPPlvpsVmBBq8yrrww5sDxsu9U\\\\\\\/AffgMScDq6YhCotsjslUlceiVDDnO1fNonyj9z6KQJTfJRUOSqPDd0l+gcdrAHLy9Ca\\\\\\\/1xTbi2\\\\\\\/+HwcvX\\\\\\\/3xii\\\\\\\/6ygTotMUEgaoO\\\\\\\/Pucu005R+q02LnBqEAeUirzH8RdseU4jTxpVq0jQut4XeAhC23TXuyymEAcgcn1eABlbplkm4q0HDWHCNCVlDti3jg9U45A3xvD7+4TF3aePbu9lVB0pLGgH+u+fKpIh4nyhsdTX1QcTIknyyRTXlNMnKQODWFR+DwOQkCjUEoZ7GCMyTVOHqcTEweU6yeG1Lt1WbgyZ5ngCGy3YOaud8hmmNMcsPgP9aH0hjEN89ZgJQrZZSG1eAHEr7QODACHgaIpYXBtbkJgzcjnkZKl9\\\\\\\/Ccox\\\\\\\/M6rUprY5u9Qg9fUJLfX+NyDDGaZj9PGAkvSLhWhAF8RXAnt+1NewrcsCO+naAyw+2Dak95xYKSvNnfD19v\\\\\\\/a2yTu4tDz2HtzgKO2O1YAgoRdJwR79TlaoeDfdQKB3cP\\\\\\\/Igne\\\\\\\/LqAkQYoJv+4hXOOJ5VmpQYWtGDDQNy6HxCvXGhcb2CTdxrWJqEpNB9INtWbgRVIgbpUUC2Hg8TjIZ8xKqPs0flsORoLO\\\\\\\/pWZ8UBV0EInbT\\\\\\\/NUxY5NEbYSTjmsc9yOE8RJA\\\\\\\/8KT3BMMDd0NnjDB8D4PCLmyyzUloaIOh\\\\\\\/ke4\\\\\\\/9uQDFZf385QZNH8ZseV3hqu8sTbrJYaT6emc4Idqp7L9Tn0KHctcziFHYoEuawnzg0bHfDeSsidhv358j2vBp9nr8614SUzk9BConMLzSdRKKtIaNIGKI54icNjjasJ4I5Pu6LohdynKtuctar1jxoSmnQT0pS+DbRgLx2+6yDvS8m2RB7EmqoWDWHNHxQJ3dMdZ+BgvXJegEO9Hfe9JqjvjSiO6v0SYg1XzevrmfX0yV1smoMcjD+5TihFAaAVPW3UGIjs377DoOoHWWDWTs8+QZM3fgUWBfQeSA3V4EsnllxTVm9gnQ4H3Cu+Zd0gWUPsk81nJlV1NGZdDPGzP5KdZP9L6LG6y9fb8gO25OmUjjwtyKj5Qe9APJjnO8TqXdNfbkrN5gWmqDVaAG+gyz3pZqlV\\\\\\\/NdfB7N2hm\\\\\\\/3fRT3xKMSkeQmpeaEfdF3WiJS1DyghX9pfgCAZwl36VTvHXeLp8c8JngojQ19nYVXgyqgGY\\\\\\\/pT8YsXWXfEWhra+uYKwDNqOlnPp8ivYvwJVORcuQ8bPAw8zWCAfgu5SyOI6+CWihkglXIYB\\\\\\\/pzb\\\\\\\/Et7g5lvraz5BRyLRKwA6R\\\\\\\/jnscux8BHxA4KYUYoMWx\\\\\\\/plcjxBHcQ\\\\\\\/esmD\\\\\\\/sj0Ql7qUR2tpqIwqjlPu8tgH9qAMV2qcPgyTXfCizswODMYBtYV7XZkkHoPqySru0YHOmZuzOKVaZkBqLE0hXFjeKz9nRbTa7Rj0K5+IiBI0tBpUzMf5pOnClpuB2iDtOlMLZtMJVqFCRuOWhMr0bpSoXLcaaiiN50tSCk\\\\\\\/cePEKcomr8hL2R82oliKNJ4Ky9k9RP+Jj1vAnFsNJAeilGCWa+13iznUwNATJTyV76q7o53B2CODnaQiTMqwvl5\\\\\\\/MWw7BWtDOFxFSnlcpJrG4FdysCf3H\\\\\\\/osdbp97BdGskzJ0LxlLyTWqheHoZB\\\\\\\/T3tcxWkeOnzq1c+fW0fIM4vhwW7JuCH6G6SF+5WZZ4XLlNMMjo+PYnNCiLcs1Do9hjrXxPYWbIkaPO\\\\\\\/ugHyE0KyODN825XC9GjpEkYk35SKLrcC64tyaZVuuDertnw7Bbrv9NvxJutNC9Zj\\\\\\\/WsN1Gagf+exSl3lz6a0FOfbn\\\\\\\/Z6MV2SDLHIVfzC\\\\\\\/sWYDkJVs\\\\\\\/fLwNye3OLz\\\\\\\/LLNCP6IYw8s+BVkply8Nb9GFskSTlGds4Pvcj82BqDocyGv4gsO4EhkUcHJjVChAP+xyWEGppCCJTf3dTPBesqThx7xBG85l3SifgXVAfJghz+eeU5CJ0sz6V6mLQ6U9C8gI1CX8NJia13iDdvbGjOViMLzh40bASQ5p3HBUK5KslWOwj00A9BuCi4UV9pRHcXDIsrUGvdcRJcfyf2\\\\\\\/fDUadoCO2xpdCbX5L8hNEflPXgv8XvFcPAtQTVYCx+XEz1g9D\\\\\\\/VYdCw0pWEdLGZcvQOwVmSocTF2Pp08OoshT67NzOucKyhD89RouZjYUsF4DuRjAF1MhXqilBnW9HbouwTeyF56C8NzHmvX3JEPrGD2ZA2YamfKQ99ewidM89JPJMvKHZFnAZM0xitJShluRD7EVkKZnrP67rtAD1LgcQrqTnGrckgxeo7WNHyCZggEc9fGmkRdH6Uk+7Kc0f+VrDSl5TZuxdBBTNblndC1eZUJ8yMivkzaZ4NRTePSMGwuaRm5pqnyNJEWJjbMyA8GpLmlz\\\\\\\/B6OgWC7vylfD8cdsf3mZAIbO\\\\\\\/QFjlK4FjJmmhZ2iaGeAHthmUzlL3Mve2cT4X1\\\\\\\/IeNxTfGC7fw05lumORtWiUlJ05OUO8aRZwrxSlMV3YNX74lm6hVXuYA7Ia\\\\\\\/r7bowu54OHESEfdhMsQAnD63FlDlWTGdVNMCgjDzb1q5xdtCWePX0glYms6VGvjnj2T3K\\\\\\\/L4RRYA2qbCjVpUucQ81n84TslXMY69f7gpaA0GN96SfSdrAUUw99eujCwvkP8xua8cEbg\\\\\\\/snE7WpZHuW46+Y\\\\\\\/r+BTmCEVUtFHGuWx3OZjWvBtoUw5urOpqc8h0CtPL9T8EGovGAmVA0SmqAPS5qhR3+7t2HMN2Wj1g9I4sGi5qo4UbXHvmeG7Qozbwxr0Qqz9XU26R\\\\\\\/b5Euz5JEgnBO\\\\\\\/n2f5WrkGAZEenqjf1HaKpr+f2lG5xzBb2ljlTMi23Vyvd+9n8FaX7h5J+\\\\\\\/FqSUZGa8pOk3zLvPFx\\\\\\\/YipQaXz7kbtvhzz8U5CyAR9eTXVi3Mp97d9P\\\\\\\/A2Qd1r5zfvpDnXpKZ7w4YfpDR2o0w==\\\u0022,\\\u0022iv\\\u0022:\\\u0022GiqMNJ2tY+UfEN0KbjlYaQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022f5bbbd73c0087b283029a09dbabe4451abfb247927ca3f5ceaaa0188378e8c59\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022FKadQQthMQ5n2oJU1i2x0\\\\\\\/BkJrCyxv5a12lMzbKT6wYD7Q9iEgPtLsCYlU8jZJVfdq9tdpzaJ40bU1PvRShmF\\\\\\\/aUqLDAh1+fGteHgZqW4LWS3pfw3HoNrTSajbnG5MTosE7yIxDyja2lyp\\\\\\\/1pAbPFWgs7YvDwy0ey1r9f6UkIhdiSM08gEbjHpyZjqt3kk1l2JNDuqaZtG+nQ2LStqwqeNF4z0oifHa0nuJtA\\\\\\\/XmIS1DYgn9Tb8KYBvCfv4D9lQKrm5e1oXBk+4mUyWPtwNmXSqGKe9VGmHTX1Iq22oF3AEXny\\\\\\\/C2BdcZ+sPRFPfC\\\\\\\/Iwt+EvGY7zeY+RvEpEkA1yXWj0EmDTN8fprgFkJwPaVKMLtM5P3179rkS\\\\\\\/5j23LJdSwJ0TLtueBqJ633Nvb6vWFMwWYpoIXM\\\\\\\/WUROfi3RVi9KvKOgwY2yIfeoyvBNxe\\\\\\\/7bibbwoRBvePSZeVmh1j2lEqFKEUaXkPwH8enEy2whzHsGdRttej5T9Xjpr3fU631ZmEFJyW8CQNO8UH9GQ7bmEQEbD6YKDUQFMnOZznQjtQK5MshnCzAtfb8pCni+DNNL6aozz9exc+kfJMUnnik9kn7A6gGwALpyZ3G3d4ZAp11d2lrebRJzorh2vGQkz427rCWeL3MTZn9myda5OwBdcEaUTzLn9re992g6467vqgWGXg1Bf9whe7Sxqh11eywTUp5l+xKb77uCccX4VYNqiOLCSteKSsDNGINImm5s25XpcgutUDjm\\\\\\\/PfC\\\\\\\/NjXDX6JvBOqbUjVk8zWVQxjj7RVSwyNq7jGi7TIyM+qT1SKrfz3jOufFEPq7aT9Yb8R\\\\\\\/mlHCiYsfNgYWH4rwrb8gFkiBi1YB5832fEBpYHBKWNdfR6sOFYZnUzCmqa\\\\\\\/k8ieBGKyIlx\\\\\\\/jMAqK3Aj4L6OxY1nXiimp1qP89fJHv4rcKrlbEFhR01LBZj7HZ44chARHjXh+lUII7iV3xr+641vB6XwNlJY18t0luKGb6J6f4G6stgkdRTvYaT5Wi9Rczf7hU38QBj0eiYV3bMD1mW8wyt\\\\\\\/+HqS4ZLJ3hro+Q8YxgzYZfFHkf5hQ1uYuJUINUUIEZTM+6cz2\\\\\\\/Ovv3mFKdyqUJNh06e0PetTMGVaZ4ewJDizFB7yegEH8Tjhgw0hVMagPJh1bI3vYmLFI3\\\\\\\/1u2mRDmBkRkt8FKDIH2A94bWthD9tWJB3GrA1A2rzWek97qrZqAe8XeM0kWaJHZ1SoLv92Pti+oXEl3vEvdK62t3mp7ti+mRVZeOlPFFead863CyqlJMS6td1JIVhRGf1F1fhKXV3ILxDibLsU9otpx+ZbQT6Dvwui1W+9RHdUQT8oMLsVty1tDUfQaCgidxaNWBY5j4oh6erj47yfOVHIhIhT9SJ+LSMLFgLouZnaUdNG11PnzlCugVBenTxwGfQUm08jWvZHFUSDOCDpFBVEB2kdDxEGweNt1V\\\\\\\/eMF13pTNhLsMYjlyYBWofvqJ\\\\\\\/XRiU7kUhrZxZR7f03nVeG7EdbMNghno0oYrhcpp6yjj5lIBNR4HMFLpdZXMLGtSRXsgCn4kBkK57SS1kfbj\\\\\\\/34VFjBGCWJvU2az0oNXLkWCIuuCfbT3MbL085G4L36uoyaUgCsTJle5HHybiNw+Me77\\\\\\\/vSO4hzQo6y+mDxy0J3aZnaXXTTYVQ\\\\\\\/mRSYeRA2J7hy0n7nKRZz00tVYDUk0GK5KRAUGHLaDvQTztoA4XoIGOysioRoi0f4\\\\\\\/tqVaHeqW3C1suCqukdcKfablVlgdHlPYkFo+DZWka9OpNMAqlUeEAdI5+J7scem+l3rF+RiiQCdkM7W4ZJABfCQIYdWS63empEas+OWCON1u98JW54bjSIIRNWazkzGnL1mJ+TlEwSgZGm1ybnibXTI1hk9rmbTOsb37bSih8FPto5JVz2+lRlrYSn8l9oOZwAF\\\\\\\/MlBMExCb6Sgw+VX+kVVv3lQ4miLc0v6olczYr6\\\\\\\/UgRq3fUTWYFCT7mVzTgNaAiR3R1gVoXCTclmk+ndo0Q==\\\u0022,\\\u0022iv\\\u0022:\\\u0022EzCpcN\\\\\\\/Hz0uAxXl01rfk6w==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022ede88444472bf8d6c273f7aba8429c26f3c6421841d3005cfa018f8eacca4e21\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-494589 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/11/3813" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-oCw6ENiS3Vth_t_Q5RzH-nBFTETHxrM1wEWohL4oQug" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/11/3813" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-DWZ3XirVzb5SrHg2z7W81pruPWj5xIDpfIqTR_ddwfc" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/11/3801" title="Genetic Dissection Reveals the Role of Ash1 Domains in Counteracting Polycomb Repression" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/11/3825" title="Visualization and Quantification of Transposon Activity in Caenorhabditis elegans RNAi Pathway Mutants" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494589" id="node494589" data-pisa="ggg;9/11/3813" data-pisa-master="ggg;g3.119.400711" data-apath="/ggg/9/11/3813.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Limits to Genomic Divergence Under Sexually Antagonistic Selection</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Katja R. Kasimatis</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-9459-6866" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Peter L. Ralph</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0001-7271-342X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Patrick C. Phillips</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3813-3824;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400711 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494589" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Katja R. Kasimatis </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Ecology and Evolution and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Katja%2BR.%2BKasimatis%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kasimatis%20KR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKatja%2BR.%2BKasimatis%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:k.kasimatis@utoronto.ca" class="" data-icon-position="" data-hide-link-title="0">k.kasimatis@utoronto.ca</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Peter L. Ralph </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Ecology and Evolution and</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Mathematics, University of Oregon, Eugene, Oregon 97403</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Peter%2BL.%2BRalph%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ralph%20PL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APeter%2BL.%2BRalph%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-9459-6866" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Peter L. Ralph</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Patrick C. Phillips </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Ecology and Evolution and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Patrick%2BC.%2BPhillips%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Phillips%20PC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APatrick%2BC.%2BPhillips%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-7271-342X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Patrick C. Phillips</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/9/11/3813" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:494589" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:494589/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/11/3813.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:494589" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:494589/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/11/3813.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:494589" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:494589/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/9/11/3813.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:494589" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:494589/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Since the autosomal genome is shared between the sexes, sex-specific fitness optima present an evolutionary challenge. While sexually antagonistic selection might favor different alleles within females and males, segregation randomly reassorts alleles at autosomal loci between sexes each generation. This process of homogenization during transmission thus prevents between-sex allelic divergence generated by sexually antagonistic selection from accumulating across multiple generations. However, recent empirical studies have reported high male-female <em>F<sub>ST</sub></em> statistics. Here, we use a population genetic model to evaluate whether these observations could plausibly be produced by sexually antagonistic selection. To do this, we use both a single-locus model with nonrandom mate choice, and individual-based simulations to study the relationship between strength of selection, degree of between-sex divergence, and the associated genetic load. We show that selection must be exceptionally strong to create measurable divergence between the sexes and that the decrease in population fitness due to this process is correspondingly high. Individual-based simulations with selection genome-wide recapitulate these patterns and indicate that small sample sizes and sampling variance can easily generate substantial male-female divergence. We therefore conclude that caution should be taken when interpreting autosomal allelic differentiation between the sexes.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/antagonistic-selection" class="hw-term hw-article-keyword hw-article-keyword-antagonistic-selection" rel="nofollow">antagonistic selection</a></li><li class="kwd"><a href="/keyword/sexual-conflict" class="hw-term hw-article-keyword hw-article-keyword-sexual-conflict" rel="nofollow">sexual conflict</a></li><li class="kwd"><a href="/keyword/male-female-divergence" class="hw-term hw-article-keyword hw-article-keyword-male-female-divergence" rel="nofollow">male-female divergence</a></li><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/intersexual-fst" class="hw-term hw-article-keyword hw-article-keyword-intersexual-fst" rel="nofollow">intersexual <em xmlns:default="http://www.w3.org/1999/xhtml">F<sub>ST</sub></em></a></li><li class="kwd"><a href="/keyword/genetic-load" class="hw-term hw-article-keyword hw-article-keyword-genetic-load" rel="nofollow">genetic load</a></li></ul><p id="p-3">Females and males use largely the same genome to produce distinct phenotypes and behaviors. This ubiquitous phenomenon requires an association between dimorphic phenotypes and their sexual environment (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Kasimatis <em>et al.</em> 2017</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Mank 2017b</a>). Genes residing on a sex chromosome have a physical link to sex determination. Particularly, on heteromorphic sex chromosomes, the lack of recombination allows for selection to act in a sex-specific manner to optimize beneficial genes within each sex (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Rice 1984</a>, <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">1987</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Charlesworth and Charlesworth 1980</a>). Conversely, the shared genetic basis of autosomal genes constrains such sex-specific optimization of fitness. When autosomal-based traits have different optimal fitness values in each sex, then selection acts in a sexually antagonistic manner to push females and males in opposing directions in phenotype space (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Rice and Holland 1997</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bonduriansky and Chenoweth 2009</a>). However, recombination and meiotic segregation uncouple beneficial alleles from their sexual environment every generation, constraining the resolution of antagonism via the creation of separate female and male genomic pools. This homogenization process tethers together the evolutionary responses of the sexes and creates an inherent intersexual genomic conflict (reviewed in <a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Bonduriansky and Chenoweth 2009</a>; <a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Kasimatis <em>et al.</em> 2017</a>).</p><p id="p-4">Identifying sexually antagonistic loci—particularly using reverse genomics approaches—has proved challenging. Initial studies calculated differentiation between females and males using Wright’s fixation index (<span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-1.gif"/></noscript></span></span>), and interpreted high values as evidence of sexually antagonistic selection. Empirical data from multiple taxonomic groups (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Lucotte <em>et al.</em> 2016</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Flanagan and Jones 2017</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Wright <em>et al.</em> 2018</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Dutoit <em>et al.</em> 2018</a>) suggest that hundreds to thousands of SNPs have elevated male-female autosomal differentiation with outliers exceeding <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-2.gif"/></noscript></span></span> (<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Lucotte <em>et al.</em> 2016</a>; <a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Wright <em>et al.</em> 2018</a>) and even approaching <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-3.gif"/></noscript></span></span> (<a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Flanagan and Jones 2017</a>). (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Cheng and Kirkpatrick 2016</a>) interpreted correlations of male–female <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-4.gif"/></noscript></span></span> with bias in gene expression to mean that many genes actively affect sex-specific viability in both humans and <em>Drosophila melanogaster</em>. Taken at face value, both the number of sexually antagonistic alleles and the degree of differentiation are striking. However, these results are difficult to evaluate as they suggest that there must be quite a large amount of selective mortality within each sex to create such high divergence within a generation (as noted by <a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Cheng and Kirkpatrick 2016</a>).</p><p id="p-5">Several different processes could in principle generate divergence (or apparent divergence) between the sexes. First, sex biases in chromosome segregation through associations with the sex determining region could distort allele frequencies between the sexes. Over time, this segregation distortion can contribute to the generation of neo-sex chromosomes (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Jaenike 2001</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Kozielska <em>et al.</em> 2010</a>) – particularly heteromorphic sex chromosomes – leading to sex-specific differentiation in the trivial sense that the locus is completely absent in one sex. Second, gametic selection resulting in a sex-specific fertilization bias could also distort allele frequencies (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Joseph and Kirkpatrick 2004</a>). Both of these processes occur during the gametic phase of the lifecycle and have long been recognized for their potential ability to distort segregation ratios within the sexes (reviewed in <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Immler and Otto 2018</a>). In contrast, sexually antagonistic viability selection that occurs post-fertilization is a fundamentally different mechanism because there is no direct co-segregation of sex with the alleles under selection. Previous work on sexually antagonistic viability selection has largely focused on its potential role in maintaining genetic variation due to the sex-specific pleiotropic effects of the locus. In particular, <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Kidwell <em>et al.</em> (1977)</a> laid out a framework for analyzing sexual antagonism that has widely been used in the field (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Arnqvist 2011</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Connallon <em>et al.</em> 2010</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Connallon and Clark 2011</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Patten and Haig 2009</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Fry 2010</a>). A little appreciated feature of the Kidwell model is that it tracks allele frequencies (rather than diploid genotype frequencies) in adults from each generation to the next. Although the model incorporates diploid selection, this sampling paradigm is sufficient because the “random union of gametes” model of mating only requires allele frequencies to generate diploid genotype frequencies in the next generation. However, this model simplification prevents the inclusion of other models of mating, such as assortative mating among genotypes.</p><p id="p-6">In this paper, we will first build a model of sexually antagonistic viability selection, segregation, and transmission, extending the model of <a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Kidwell <em>et al.</em> (1977)</a> to include assortative mating. We use this model to evaluate how much between-sex differentiation is produced across a range of selection, dominance, and assortative mating parameters. Second, we use these results to evaluate the claims that the observed between-sex allelic differentiation is caused by sexually antagonistic viability selection. We then use simulation to test the conclusions of our deterministic model, as well as the role of sampling variance in generating loci with high between-sex differentiation. Both our single locus model and individual-based simulations with antagonistic loci distributed genome-wide indicate that antagonistic selection must be remarkably strong to produce non-negligible divergence between the sexes. Instead, simulations indicate that sampling variance is much more likely to account for extreme between-sex divergence and must therefore be explicitly included in any analyses of putative signatures of male-female divergence.</p><div class="section methods" id="sec-1"><h2 class="">Methods</h2><div id="sec-2" class="subsection"><h3>Model</h3><p id="p-7">Consider an autosomal locus in which are found two alleles: one female-beneficial (<span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-5.gif"/></noscript></span></span>) and one male-beneficial (<span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-6.gif"/></noscript></span></span>). Sexual antagonism results in a fitness cost to individuals carrying the allele favored in the other sex (<a id="xref-ref-22-3" class="xref-bibr" href="#ref-22">Kidwell <em>et al.</em> 1977</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bodmer 1965</a>). The life cycle is shown in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>. Each generation begins with zygotic frequencies equal in each sex, but then genotype-dependent survival results in different genotype frequencies in each sex at time of mating. The relative fitnesses of genotypes <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-7.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-8.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-9.gif"/></noscript></span></span> in females are <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-10.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-11.gif"/></noscript></span></span> is the cost of a female having the male-favorable allele and <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-12.gif"/></noscript></span></span> is the dominance coefficient in females. (This is a model of viability selection, so here and below “fitness” refers to viability.) Writing the frequencies of the three genotypes in zygotes as <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-13.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-14.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-15.gif"/></noscript></span></span> at the start of generation <em>t</em>, the genotype frequencies in females after selection will then be proportional to <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-16.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-17.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-18.gif"/></noscript></span></span>, respectively. Similarly, the relative fitnesses of the genotypes <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-19.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-20.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-21.gif"/></noscript></span></span> in males are <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-22.gif"/></noscript></span></span>, and the genotype frequencies in males after selection are proportional to <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-23.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-24.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-25.gif"/></noscript></span></span>, respectively.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/11/3813/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Lifecycle of the model. Zygotes are subject to sexually antagonistic viability selection ( and ), perturbing allele frequencies in adults in a sex-specific manner. Sex-specific adult allele frequencies are given in Equations 1 and 2, where  and . Surviving adults produce gametes of each allele type in frequencies corresponding to Equations 1 and 2. At this time meiotic segregation breaks the association between the locus and sex. Females and males mate with frequencies proportional to the mate choice matrix (M) to produce the zygote pool in the next generation. Kidwell et al. (1977) gives the recursion for the allele frequencies in gametes (, ), under the assumption of random, genotype-independent mating." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-244047068" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Lifecycle of the model. Zygotes are subject to sexually antagonistic viability selection (&lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-26&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; and &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-27&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;), perturbing allele frequencies in adults in a sex-specific manner. Sex-specific adult allele frequencies are given in Equations 1 and 2, where &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-28&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; and &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-29&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;. Surviving adults produce gametes of each allele type in frequencies corresponding to Equations 1 and 2. At this time meiotic segregation breaks the association between the locus and sex. Females and males mate with frequencies proportional to the mate choice matrix (M) to produce the zygote pool in the next generation. Kidwell &lt;em&gt;et al.&lt;/em&gt; (1977) gives the recursion for the allele frequencies in gametes (&lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-30&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;, &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-31&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;), under the assumption of random, genotype-independent mating.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/11/3813/F1.medium.gif" width="440" height="271"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/9/11/3813/F1.medium.gif" width="440" height="271"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/11/3813/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/11/3813/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494836" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-8" class="first-child">Lifecycle of the model. Zygotes are subject to sexually antagonistic viability selection (<span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/F1/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/F1/embed/mml-math-26.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/F1/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/F1/embed/mml-math-27.gif"/></noscript></span></span>), perturbing allele frequencies in adults in a sex-specific manner. Sex-specific adult allele frequencies are given in Equations 1 and 2, where <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/F1/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/F1/embed/mml-math-28.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/F1/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/F1/embed/mml-math-29.gif"/></noscript></span></span>. Surviving adults produce gametes of each allele type in frequencies corresponding to Equations 1 and 2. At this time meiotic segregation breaks the association between the locus and sex. Females and males mate with frequencies proportional to the mate choice matrix (M) to produce the zygote pool in the next generation. <a id="xref-ref-22-4" class="xref-bibr" href="#ref-22">Kidwell <em>et al.</em> (1977)</a> gives the recursion for the allele frequencies in gametes (<span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/F1/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/F1/embed/mml-math-30.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/F1/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/F1/embed/mml-math-31.gif"/></noscript></span></span>), under the assumption of random, genotype-independent mating.</p><div class="sb-div caption-clear"></div></div></div><p id="p-9">Therefore, the frequency of the female-beneficial allele in females post-selection, which we denote <span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-32.gif"/></noscript></span></span>, is<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-33.gif"/></noscript></span><span class="disp-formula-label">(1)</span></span>The same quantity for males is:<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-34.gif"/></noscript></span><span class="disp-formula-label">(2)</span></span>In a deterministic model of non-overlapping generations without gamete-specific selection, the genotype frequencies in the next generation are determined by the frequency of gametes joining from each of the nine possible mating combinations weighted according to mate choice. We parameterize mate choice using a matrix whose rows are indexed by male genotypes and columns by female genotypes, such that <span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-35.gif"/></noscript></span></span> is the frequency of pairings of male genotype <em>i</em> with female genotype <em>j</em> relative to that expected under random mating. We focus on three common mating scenarios by structuring the mate choice matrix as:<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-36.gif"/></noscript></span></span>Under random mating, each pairing occurs with equal likelihood (<span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-37.gif"/></noscript></span></span>). Positive assortative mating by genotype occurs when females and males with the same genotype mate more frequently than those with different genotypes (<span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-38.gif"/></noscript></span></span>). Conversely, disassortative mating by genotype – or positive assortative mating by fitness – occurs when <span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-39.gif"/></noscript></span></span> individuals mate with <span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-40.gif"/></noscript></span></span> individuals (<span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-41.gif"/></noscript></span></span>).</p><p id="p-10">The genotype frequencies in the next generation can be concisely calculated with some matrix algebra. Let <span class="inline-formula" id="inline-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-42.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-43.gif"/></noscript></span></span> be the vectors of relative fitnesses in females and males respectively. Then, define the <span class="inline-formula" id="inline-formula-41"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-44.gif"/></noscript></span></span> matrix of fitness-weighted mate pairings, F, so that for each pair of genotypes <em>a</em> and <em>b</em>, the entry <span class="inline-formula" id="inline-formula-42"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-45.gif"/></noscript></span></span>. In other words, <span class="inline-formula" id="inline-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-46.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-46.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-44"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-47.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-47.gif"/></noscript></span></span> denotes the matrix with <span class="inline-formula" id="inline-formula-45"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-48.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-48.gif"/></noscript></span></span> on the diagonal and zeros elsewhere. Finally, define <span class="inline-formula" id="inline-formula-46"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-49.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-49.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-47"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-50.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-50.gif"/></noscript></span></span>. Then, the vector of frequencies of each genotype among zygotes (before selection) in the next generation can be calculated using the current frequencies as a weighted sum over possible mating pairs:<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-51.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-51.gif"/></noscript></span><span class="disp-formula-label">(3)</span></span>Here, <span class="inline-formula" id="inline-formula-48"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-52.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-52.gif"/></noscript></span></span> is the column vector of genotype frequencies and <span class="inline-formula" id="inline-formula-49"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-53.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-53.gif"/></noscript></span></span> is its transpose. This set of equations can be derived by noting that the relative frequencies of <span class="inline-formula" id="inline-formula-50"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-54.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-54.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-51"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-55.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-55.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-52"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-56.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-56.gif"/></noscript></span></span> genotypes produced in the next generation are <span class="inline-formula" id="inline-formula-53"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-57.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-57.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-54"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-58.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-58.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-55"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-59.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-59.gif"/></noscript></span></span>, respectively; since <span class="inline-formula" id="inline-formula-56"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-60.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-60.gif"/></noscript></span></span>, the identity matrix, these sum to <span class="inline-formula" id="inline-formula-57"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-61.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-61.gif"/></noscript></span></span>, the denominator in equations (3).</p><p id="p-11">We used Mathematica v11.1.1.0 (Wolfram Research, Inc.) to find the equilibria of this system and determine stability of those equilibria. The complete notebook is provided in File S1.</p></div><div id="sec-3" class="subsection"><h3>Within-generation statistics</h3><p id="p-12">Sex-specific viability selection creates differences in allele frequencies between the sexes each generation. We can therefore quantify the effects of sexually antagonistic selection using the male-female <span class="inline-formula" id="inline-formula-58"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-62.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-62.gif"/></noscript></span></span> statistic, which we calculate as the squared difference in allele frequencies between sexes, normalized by the total heterozygosity across sexes (<a id="xref-ref-8-3" class="xref-bibr" href="#ref-8">Cheng and Kirkpatrick 2016</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Wright 1951</a>):<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-63.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-63.gif"/></noscript></span><span class="disp-formula-label">(4)</span></span>Sex-specific selection creates divergence between the sexes by increasing the frequency of the beneficial allele in each sex. Therefore, at the population level, this opposing action of section skews genotype frequencies away from Hardy-Weinberg equilibrium. The deviation from Hardy-Weinberg equilibrium within the population due to sex-specific effects can be quantified using Wright’s <span class="inline-formula" id="inline-formula-59"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-64.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-64.gif"/></noscript></span></span> statistic (<a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Wright 1951</a>):<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-65.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-65.gif"/></noscript></span></span>A population fitness cost due to sexual antagonism (<em>i.e.</em>, genetic load), is generated each generation. Within each sex, the genetic load is the difference between the maximum possible fitness and the mean fitness (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Haldane 1937</a>). The population’s average genetic load (<em>L</em>) is the average of the loads for each sex (assuming an equal sex ratio), which is given by:<span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-66.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-66.gif"/></noscript></span><span class="disp-formula-label">(5)</span></span>where <span class="inline-formula" id="inline-formula-60"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-67.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-67.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-61"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-68.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-68.gif"/></noscript></span></span>.</p></div><div id="sec-4" class="subsection"><h3>Simulations</h3><p id="p-13">We used R (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">R Core Team 2018</a>) (File S2) to simulate allele frequency dynamics at a single locus in a population subject to selection and drift. During viability selection each generation, each individual survived with probability equal to their (sex- and genotype-dependent) fitness. Then, the genotype frequencies within each sex were multiplied to give the matrix of relative frequencies of possible mating pairs, which was further weighted by the mate choice matrix. To generate the next generation, a fixed number of mating pairs are sampled from this distribution, and offspring are produced by random choice of parental alleles.</p><p id="p-14">We also implemented simulations with sexually antagonistic selection acting at many loci, genome-wide with SLiM v3.1, an evolution simulation framework (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Haller and Messer 2019</a>) (recipes in File S3). Individuals each had a genome of 100 Mb, a uniform recombination rate of <span class="inline-formula" id="inline-formula-62"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-69.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-69.gif"/></noscript></span></span> per nucleotide, and a mutation rate of <span class="inline-formula" id="inline-formula-63"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-70.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-70.gif"/></noscript></span></span>. All mutations are sexually antagonistic (we do not simulate neutral variation): each new mutation was beneficial in a randomly chosen sex and detrimental in the other, with selection coefficients drawn independently for each sex from a Gaussian distribution with mean zero and standard deviation 0.01. Each mutation also had dominance coefficients drawn independently for each sex from a uniform distribution between 0 and 1. The model had overlapping generations: each time step, first viability selection occurred (with probability of survival equal to fitness), followed by reproduction by random mating. The number of new offspring was chosen so that the population size fluctuated around 10,000 diploids, and simulations were run for 1,000 time steps. For a neutral comparison, we also simulated from the same scenario but with no fitness effects. We ran 5 independent simulations of each scenario (<em>i.e.</em>, neutral and sexually antagonistic).</p><p id="p-15">After the final generation, genetic load and male-female <span class="inline-formula" id="inline-formula-64"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-71.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-71.gif"/></noscript></span></span> at each locus were calculated. <span class="inline-formula" id="inline-formula-65"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-72.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-72.gif"/></noscript></span></span> values were calculated both using all individuals within the population as well as using smaller subsamples of 100 individuals and 50 individuals with equal numbers of each sex. Subsample sizes were chosen to reflect sample sizes currently used in the literature (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Dutoit <em>et al.</em> 2018</a>; <a id="xref-ref-13-3" class="xref-bibr" href="#ref-13">Flanagan and Jones 2017</a>). Male-female <span class="inline-formula" id="inline-formula-66"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-73.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-73.gif"/></noscript></span></span> values within the subsamples were calculated both with equation 4 and Weir and Cockerham’s <span class="inline-formula" id="inline-formula-67"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-74.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-74.gif"/></noscript></span></span> estimator (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Weir and Cockerham 1984</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Bhatia <em>et al.</em> 2013</a>) to examine the impact of the statistic used on the distribution of <span class="inline-formula" id="inline-formula-68"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-75.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-75.gif"/></noscript></span></span> values. (With equal female and male subsampling, Weir and Cockerham’s <span class="inline-formula" id="inline-formula-69"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-76.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-76.gif"/></noscript></span></span> is equivalent to Hudson’s <span class="inline-formula" id="inline-formula-70"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-77.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-77.gif"/></noscript></span></span> (<a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Bhatia <em>et al.</em> 2013</a>).)</p></div><div id="sec-5" class="subsection"><h3>Data availability</h3><p id="p-16">The model equations, equilibrium analyses, and stability analyses are given in the Mathematica notebook in File S1. The single locus simulations and SLiM statistical analyses are given in File S2. The SLiM code and simulation data are available in Files S3-S5. Supplemental material available at FigShare: <a href="https://doi.org/10.25387/g3.9785951">https://doi.org/10.25387/g3.9785951</a>.</p></div></div><div class="section results" id="sec-6"><h2 class="">Results</h2><p id="p-17">We first examine the conditions under which our model supports a stable polymorphism, and then examine the degree of between-sex divergence and genetic load expected under both equilibrium and non-equilibrium (selective sweep) conditions. Our single locus results expand upon previous the results of previous studies (<a id="xref-ref-22-5" class="xref-bibr" href="#ref-22">Kidwell <em>et al.</em> 1977</a>; <a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Arnqvist 2011</a>; <a id="xref-ref-8-4" class="xref-bibr" href="#ref-8">Cheng and Kirkpatrick 2016</a>; <a id="xref-ref-21-3" class="xref-bibr" href="#ref-21">Kasimatis <em>et al.</em> 2017</a>). We then verify these results using simulations, which also provide an opportunity to explore the effects of statistical sampling on inferences of sex-specific differentiation from genomic samples.</p><div id="sec-7" class="subsection"><h3>Transmission dynamics at a sexually antagonistic locus</h3><div id="sec-8" class="subsection"><h4>Maintenance of polymorphism usually requires symmetric selection between the sexes under random mating:</h4><p id="p-18">We will quantify the strength and degree of asymmetry between the sex-specific allelic effects using the overall strength (<em>s</em>) and the ratio of selection coefficients (<em>α</em>), so that <span class="inline-formula" id="inline-formula-71"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-78.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-78.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-72"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-79.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-79.gif"/></noscript></span></span>. The full solution for the maintenance of polymorphism under arbitrary patterns of dominance can be solved by setting <span class="inline-formula" id="inline-formula-73"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-80.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-80.gif"/></noscript></span></span> in the recursion equations above (Equations (3); File S1). Under general conditions, this system yields a fifth-order polynomial that does not readily generate a closed form solution in symbolic form, although the equilibria can be easily found numerically. Symbolic solutions are possible under some specific conditions.</p><p id="p-19">Assuming random mating and additivity of allelic effects (<span class="inline-formula" id="inline-formula-74"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-81.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-81.gif"/></noscript></span></span>), the frequency of the <span class="inline-formula" id="inline-formula-75"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-82.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-82.gif"/></noscript></span></span> allele at equilibrium (denoted <span class="inline-formula" id="inline-formula-76"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-83.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-83.gif"/></noscript></span></span>) can be expressed in terms of the strength of selection and asymmetry in selection:<span class="disp-formula" id="disp-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-84.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-84.gif"/></noscript></span><span class="disp-formula-label">(6)</span></span>When the strength of selection is equal between the sexes (<span class="inline-formula" id="inline-formula-77"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-85.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-85.gif"/></noscript></span></span>), an equilibrium frequency of <span class="inline-formula" id="inline-formula-78"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-86.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-86.gif"/></noscript></span></span> is always predicted. This theoretical solution is well supported by the stochastic simulations as well (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A-B</a>). The bounds on the non-trivial equilibrium frequency can be found by setting <span class="inline-formula" id="inline-formula-79"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-87.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-87.gif"/></noscript></span></span> to zero or one. By solving these equations for <em>α</em> in terms of the strength of selection (<em>s</em>), we find that for the equilibrium to be stable, <em>α</em> and <em>s</em> must satisfy the condition:<span class="disp-formula" id="disp-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-88.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-88.gif"/></noscript></span><span class="disp-formula-label">(7)</span></span>These bounds can also be found by calculating the Jacobian matrix for the full set of transition equations (File S1) and agree with those identified by <a id="xref-ref-22-6" class="xref-bibr" href="#ref-22">Kidwell <em>et al.</em> (1977)</a>. In general, the equilibrium conditions describe an expanding envelope in parameter space that allows more asymmetry in the pattern of antagonistic selection as the absolute strength of selection increases (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A &amp; B</a>). To a first order approximation in <em>s</em>, equation (7) shows that the equilibrium is stable only if asymmetry is not larger than the strength of selection, such that <span class="inline-formula" id="inline-formula-80"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-89.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-89.gif"/></noscript></span></span>, as shown in <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2A</a>. Thus, when selection is weak or moderate, the maintenance of a polymorphism requires approximately equal selection between the sexes. However, the permissible degree of asymmetry increases with the strength of selection (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2B</a>). For example, when <span class="inline-formula" id="inline-formula-81"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-90.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-90.gif"/></noscript></span></span> a stable polymorphism can be maintained so long as the asymmetry in fitness (<span class="inline-formula" id="inline-formula-82"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-91.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-91.gif"/></noscript></span></span>) is less than 50%. Selection coefficients of this magnitude mean mortality rates of 40% or higher each generation due to a single incorrect sexually antagonistic allele, which seems biologically implausible. Therefore, under additivity, any stable antagonistic polymorphisms in natural populations must have approximately equal fitness effects in the two sexes, while less balanced antagonistic loci will quickly be fixed or lost.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/11/3813/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The equilibrium space for the  allele under differing selection and dominance conditions. A) The equilibrium space at an additive locus (, Equation 10), when selection is weak and related between the sexes by the ratio &#x3B1;. Here the equilibrium space is symmetric around  and confined to approximately equal selection between the sexes. The solid black line represent the permissible bounds on &#x3B1; (7) and the dashed gray line represents the first order Taylor series approximation. B) The equilibrium space at an additive locus increases as the strength of selection increases. The solid black line represents the bounds on &#x3B1; and the dashed gray line represents the second order Taylor series approximation. C) The equilibrium space across all dominance conditions when selection is equal between the sexes (). When the dominance coefficients between the sexes sum to no greater than one (), then the equilibrium is stable. However, when the sum is greater than one the equilibrium is unstable. D) Strong, asymmetric selection () narrows the equilibrium space and range of stable conditions ()." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-244047068" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;The equilibrium space for the &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-92&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; allele under differing selection and dominance conditions. A) The equilibrium space at an additive locus (&lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-93&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;, Equation 10), when selection is weak and related between the sexes by the ratio &lt;em&gt;&#x3B1;&lt;/em&gt;. Here the equilibrium space is symmetric around &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-94&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; and confined to approximately equal selection between the sexes. The solid black line represent the permissible bounds on &lt;em&gt;&#x3B1;&lt;/em&gt; (7) and the dashed gray line represents the first order Taylor series approximation. B) The equilibrium space at an additive locus increases as the strength of selection increases. The solid black line represents the bounds on &lt;em&gt;&#x3B1;&lt;/em&gt; and the dashed gray line represents the second order Taylor series approximation. C) The equilibrium space across all dominance conditions when selection is equal between the sexes (&lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-95&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;). When the dominance coefficients between the sexes sum to no greater than one (&lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-96&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;), then the equilibrium is stable. However, when the sum is greater than one the equilibrium is unstable. D) Strong, asymmetric selection (&lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-97&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;) narrows the equilibrium space and range of stable conditions (&lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-98&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/11/3813/F2.medium.gif" width="429" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/9/11/3813/F2.medium.gif" width="429" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/11/3813/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/11/3813/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494665" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-20" class="first-child">The equilibrium space for the <span class="inline-formula" id="inline-formula-83"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/F2/embed/mml-math-92.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/F2/embed/mml-math-92.gif"/></noscript></span></span> allele under differing selection and dominance conditions. A) The equilibrium space at an additive locus (<span class="inline-formula" id="inline-formula-84"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/F2/embed/mml-math-93.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/F2/embed/mml-math-93.gif"/></noscript></span></span>, Equation 10), when selection is weak and related between the sexes by the ratio <em>α</em>. Here the equilibrium space is symmetric around <span class="inline-formula" id="inline-formula-85"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/F2/embed/mml-math-94.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/F2/embed/mml-math-94.gif"/></noscript></span></span> and confined to approximately equal selection between the sexes. The solid black line represent the permissible bounds on <em>α</em> (7) and the dashed gray line represents the first order Taylor series approximation. B) The equilibrium space at an additive locus increases as the strength of selection increases. The solid black line represents the bounds on <em>α</em> and the dashed gray line represents the second order Taylor series approximation. C) The equilibrium space across all dominance conditions when selection is equal between the sexes (<span class="inline-formula" id="inline-formula-86"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/F2/embed/mml-math-95.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/F2/embed/mml-math-95.gif"/></noscript></span></span>). When the dominance coefficients between the sexes sum to no greater than one (<span class="inline-formula" id="inline-formula-87"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/F2/embed/mml-math-96.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/F2/embed/mml-math-96.gif"/></noscript></span></span>), then the equilibrium is stable. However, when the sum is greater than one the equilibrium is unstable. D) Strong, asymmetric selection (<span class="inline-formula" id="inline-formula-88"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/F2/embed/mml-math-97.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/F2/embed/mml-math-97.gif"/></noscript></span></span>) narrows the equilibrium space and range of stable conditions (<span class="inline-formula" id="inline-formula-89"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/F2/embed/mml-math-98.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/F2/embed/mml-math-98.gif"/></noscript></span></span>).</p><div class="sb-div caption-clear"></div></div></div><p id="p-21">On the other hand, if dominance is allowed to vary between the sexes but selection is equally antagonistic across the sexes (<span class="inline-formula" id="inline-formula-90"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-99.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-99.gif"/></noscript></span></span>), there is always a single real, non-trivial equilibrium (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2C</a>), whose stability depends on the sum of the dominance coefficients between the sexes (<a id="xref-ref-22-7" class="xref-bibr" href="#ref-22">Kidwell <em>et al.</em> 1977</a>). When <span class="inline-formula" id="inline-formula-91"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-100.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-100.gif"/></noscript></span></span> the equilibrium is stable (File S1). This stability boundary makes sense as the mean fitness of homozygous individuals is lower than that of heterozygous individuals (assuming equal sex ratios):<span class="disp-formula" id="disp-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-101.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-101.gif"/></noscript></span></span>In other words, the equilibrium remains stable if the deleterious effects of dominance in one sex do not outweigh the benefits in the other sex. Interestingly, weak selection at a locus with sex-beneficial dominance (<span class="inline-formula" id="inline-formula-92"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-102.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-102.gif"/></noscript></span></span>) can maintain a stable polymorphism despite greater asymmetry in selection than can an additive model (File S1). This expansion of the region of stability is likely a result of heterozygotes being shielded from antagonistic selection and suggests that modifying dominance can act to maintain sexual antagonism at a locus (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Connallon and Chenoweth 2019</a>). Conversely, when <span class="inline-formula" id="inline-formula-93"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-103.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-103.gif"/></noscript></span></span>, dominance favors the deleterious allele in each sex, pushing the population to an unstable state and leading to the fixation of the less costly allele. Allowing for asymmetry in the strength of selection narrows the equilibrium space and reduces the range of dominance coefficients resulting in stability (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2D</a>).</p></div><div id="sec-9" class="subsection"><h4>Assortative mating by fitness expands the polymorphism space:</h4><p id="p-22">Under positive assortative mating by fitness, high fitness matings occur between disparate genotypes and therefore produce an excess of heterozygotes each generation. In this situation (<em>i.e.</em>, <span class="inline-formula" id="inline-formula-94"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-104.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-104.gif"/></noscript></span></span>), up to three real non-trivial equilibria can exist depending on the selection and dominance parameters (File S1). However, as with random mating, at most one equilibrium is stable. When selection is symmetrically antagonistic across the sexes (<span class="inline-formula" id="inline-formula-95"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-105.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-105.gif"/></noscript></span></span>), an <span class="inline-formula" id="inline-formula-96"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-106.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-106.gif"/></noscript></span></span> allele frequency of approximately 0.5 is always predicted, regardless of dominance. This prediction is borne out by the single locus simulation results, which further show that assortative mating by fitness tends to make the stable equilibrium more robust to the effects of genetic drift (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3C</a>). Increasing the asymmetry of selection can introduce an additional unstable equilibrium, and increasing the strength of sex-deleterious dominance (toward <span class="inline-formula" id="inline-formula-97"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-107.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-107.gif"/></noscript></span></span>) can introduce a second unstable equilibrium (File S1). These theoretical predictions agree with previous simulations of assortative mating (<a id="xref-ref-1-3" class="xref-bibr" href="#ref-1">Arnqvist 2011</a>). As with random mating, the relationship between the strength and asymmetry in selection is the critical factor in determining when equilibria are stable. Specifically, when the asymmetry in selection is sufficiently large, fixation of the more favored allele is expected. Fixation only tends to occur under unrealistically large viability costs, however, and so the predominant outcome of assortative mating by fitness is the maintenance of heterozygotes and an expansion of the equilibrium space relative to random mating.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/11/3813/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The change in the frequency of a newly derived sexually antagonistic allele () over time. The black line represents the predicted allele frequency from the recursion equation. The overlaid pink and blue lines represent the simulated population (N = 20,000) of females and males, respectively. The strength of selection (s), ratio of selection between the sexes (&#x3B1;), dominance relationship ( and ), and mate choice coefficients (, , and ) are given in each panel. A) Random mating with additive dominance and symmetric selection between the sexes maintains a stable polymorphism. B) Random mating with complete male dominance and symmetric selection between the sexes maintains a stable polymorphism. C) Assortative mating by fitness with additive dominance maintain a stable polymorphism. D) Assortative mating by genotype with additive dominance has an unstable equilibrium. Multiple simulated populations show how drift will quickly lead to fixation or loss of the  allele." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-244047068" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;The change in the frequency of a newly derived sexually antagonistic allele (&lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-108&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;) over time. The black line represents the predicted allele frequency from the recursion equation. The overlaid pink and blue lines represent the simulated population (N = 20,000) of females and males, respectively. The strength of selection (&lt;em&gt;s&lt;/em&gt;), ratio of selection between the sexes (&lt;em&gt;&#x3B1;&lt;/em&gt;), dominance relationship (&lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-109&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; and &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-110&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;), and mate choice coefficients (&lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-111&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;, &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-112&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;, and &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-113&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;) are given in each panel. A) Random mating with additive dominance and symmetric selection between the sexes maintains a stable polymorphism. B) Random mating with complete male dominance and symmetric selection between the sexes maintains a stable polymorphism. C) Assortative mating by fitness with additive dominance maintain a stable polymorphism. D) Assortative mating by genotype with additive dominance has an unstable equilibrium. Multiple simulated populations show how drift will quickly lead to fixation or loss of the &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-114&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; allele.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/11/3813/F3.medium.gif" width="336" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/9/11/3813/F3.medium.gif" width="336" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/11/3813/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/11/3813/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494701" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-23" class="first-child">The change in the frequency of a newly derived sexually antagonistic allele (<span class="inline-formula" id="inline-formula-98"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/F3/embed/mml-math-108.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/F3/embed/mml-math-108.gif"/></noscript></span></span>) over time. The black line represents the predicted allele frequency from the recursion equation. The overlaid pink and blue lines represent the simulated population (N = 20,000) of females and males, respectively. The strength of selection (<em>s</em>), ratio of selection between the sexes (<em>α</em>), dominance relationship (<span class="inline-formula" id="inline-formula-99"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/F3/embed/mml-math-109.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/F3/embed/mml-math-109.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-100"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/F3/embed/mml-math-110.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/F3/embed/mml-math-110.gif"/></noscript></span></span>), and mate choice coefficients (<span class="inline-formula" id="inline-formula-101"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/F3/embed/mml-math-111.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/F3/embed/mml-math-111.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-102"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/F3/embed/mml-math-112.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/F3/embed/mml-math-112.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-103"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/F3/embed/mml-math-113.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/F3/embed/mml-math-113.gif"/></noscript></span></span>) are given in each panel. A) Random mating with additive dominance and symmetric selection between the sexes maintains a stable polymorphism. B) Random mating with complete male dominance and symmetric selection between the sexes maintains a stable polymorphism. C) Assortative mating by fitness with additive dominance maintain a stable polymorphism. D) Assortative mating by genotype with additive dominance has an unstable equilibrium. Multiple simulated populations show how drift will quickly lead to fixation or loss of the <span class="inline-formula" id="inline-formula-104"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/F3/embed/mml-math-114.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/F3/embed/mml-math-114.gif"/></noscript></span></span> allele.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-10" class="subsection"><h4>Assortative mating by genotype leads to fixation:</h4><p id="p-24">In contrast to assortative mating by fitness, if assortative mating is by genotype (<span class="inline-formula" id="inline-formula-105"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-115.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-115.gif"/></noscript></span></span>), there is only a single non-trivial equilibrium (File S1). This equilibrium is always unstable, regardless of dominance, as shown by the leading eigenvalue of the Jacobian matrix. <a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3D</a> shows allele frequency trajectories that start at this unstable equilibrium rapidly go to loss or fixation (with the choice determined by random genetic drift). Thus, these mating dynamics shrink the parameter space for maintaining a stable polymorphism and lead to the loss of the weaker antagonistic allele.</p></div></div><div id="sec-11" class="subsection"><h3>Male-female divergence is exceptionally low</h3><p id="p-25">A number of studies have observed high mean male-female divergences (measured by <span class="inline-formula" id="inline-formula-106"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-116.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-116.gif"/></noscript></span></span>). For instance, <a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">Dutoit <em>et al.</em> (2018)</a> found a mean male-female <span class="inline-formula" id="inline-formula-107"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-117.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-117.gif"/></noscript></span></span> across genes with male-biased expression in a sample of 43 flycatchers of each sex. <a id="xref-ref-38-3" class="xref-bibr" href="#ref-38">Wright <em>et al.</em> (2018)</a> found a larger average male-female <span class="inline-formula" id="inline-formula-108"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-118.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-118.gif"/></noscript></span></span> value of 0.03 across sex-biased genes in transcriptomes of 11 male and four female Trinidadian guppies. Similarly, <a id="xref-ref-13-4" class="xref-bibr" href="#ref-13">Flanagan and Jones (2017)</a> identified 473 genome-wide outliers having male-female <span class="inline-formula" id="inline-formula-109"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-119.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-119.gif"/></noscript></span></span> values above roughly 0.05 in a RADseq study of 171 male and 57 female gulf pipefish. Finally, <a id="xref-ref-25-3" class="xref-bibr" href="#ref-25">Lucotte <em>et al.</em> (2016)</a> found an average male-female <span class="inline-formula" id="inline-formula-110"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-120.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-120.gif"/></noscript></span></span> of 0.067 across autosomal SNPs in the human HAPMAP data that showed significant nonzero male-female <span class="inline-formula" id="inline-formula-111"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-121.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-121.gif"/></noscript></span></span> in all 11 populations (with around 100 samples of both sexes per population). Previous work showed that selection within a single generation at an additive locus must be strong to generate substantial male-female <span class="inline-formula" id="inline-formula-112"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-122.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-122.gif"/></noscript></span></span> values (<a id="xref-ref-8-5" class="xref-bibr" href="#ref-8">Cheng and Kirkpatrick 2016</a>; <a id="xref-ref-21-4" class="xref-bibr" href="#ref-21">Kasimatis <em>et al.</em> 2017</a>). The model we study here allows us to estimate the strength of antagonistic selection required to produce male-female <span class="inline-formula" id="inline-formula-113"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-123.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-123.gif"/></noscript></span></span> values as large as these, both at stably polymorphic loci and at loci undergoing a selective sweep.</p><p id="p-26">When selection and dominance coefficients are chosen such that a stable equilibrium is maintained, divergence between the sexes tends to be exceptionally low (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4A</a>). For example, a 10% viability cost (<span class="inline-formula" id="inline-formula-114"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-124.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-124.gif"/></noscript></span></span>) results in a between-sex <span class="inline-formula" id="inline-formula-115"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-125.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-125.gif"/></noscript></span></span> value of 0.0007 at equilibrium (assuming an additive locus and random mating). An equilibrium male-female <span class="inline-formula" id="inline-formula-116"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-126.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-126.gif"/></noscript></span></span> value of 0.0016 (as in flycatchers) requires at least a 15% viability cost within each sex (<span class="inline-formula" id="inline-formula-117"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-127.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-127.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-118"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-128.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-128.gif"/></noscript></span></span>). To produce equilibrium <span class="inline-formula" id="inline-formula-119"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-129.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-129.gif"/></noscript></span></span> values an order of magnitude larger (as reported for the largest loci in the other taxa) requires a 30–65% viability cost (<span class="inline-formula" id="inline-formula-120"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-130.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-130.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-121"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-131.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-131.gif"/></noscript></span></span>). For these values to be a product of viability selection, the field would need to have overlooked as much as 50% genotype-dependant mortality (or infertility) for each sex every generation, which seems implausible in these taxa.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/11/3813/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Divergence between the sexes due to a single generation of sexually antagonistic selection. A) Male-female  for an additive locus () at equilibrium, where the strength of selection between the sexes is related by the ratio &#x3B1;. B) Male-female  as a function of selection for three dominance regimes: sex-specific beneficial (), additive (), and deleterious (). The sex-specific deleterious dominance curve is at an unstable equilibrium, while the additive and beneficial dominance curves are at a stable equilibrium (for all curves  and the equilibrium frequency is 0.5). Sex-specific beneficial dominance always results in the lowest divergence between the sexes. The inset graph highlights the similarly low divergence values generated under weak and moderately weak selection. C) Male-female  at a sex-beneficial locus (: the polymorphic equilibrium is stable only if ) as a function of  allele frequency for varying degrees of asymmetry in selection () with a fixed mean selection coefficient ()." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-244047068" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Divergence between the sexes due to a single generation of sexually antagonistic selection. A) Male-female &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-132&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; for an additive locus (&lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-133&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;) at equilibrium, where the strength of selection between the sexes is related by the ratio &lt;em&gt;&#x3B1;&lt;/em&gt;. B) Male-female &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-134&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; as a function of selection for three dominance regimes: sex-specific beneficial (&lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-135&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;), additive (&lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-136&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;), and deleterious (&lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-137&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;). The sex-specific deleterious dominance curve is at an unstable equilibrium, while the additive and beneficial dominance curves are at a stable equilibrium (for all curves &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-138&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; and the equilibrium frequency is 0.5). Sex-specific beneficial dominance always results in the lowest divergence between the sexes. The inset graph highlights the similarly low divergence values generated under weak and moderately weak selection. C) Male-female &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-139&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; at a sex-beneficial locus (&lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-140&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;: the polymorphic equilibrium is stable only if &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-141&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;) as a function of &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-142&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; allele frequency for varying degrees of asymmetry in selection (&lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-143&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;) with a fixed mean selection coefficient (&lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-144&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/11/3813/F4.medium.gif" width="414" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/9/11/3813/F4.medium.gif" width="414" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/11/3813/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/11/3813/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494707" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-27" class="first-child">Divergence between the sexes due to a single generation of sexually antagonistic selection. A) Male-female <span class="inline-formula" id="inline-formula-122"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/F4/embed/mml-math-132.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/F4/embed/mml-math-132.gif"/></noscript></span></span> for an additive locus (<span class="inline-formula" id="inline-formula-123"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/F4/embed/mml-math-133.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/F4/embed/mml-math-133.gif"/></noscript></span></span>) at equilibrium, where the strength of selection between the sexes is related by the ratio <em>α</em>. B) Male-female <span class="inline-formula" id="inline-formula-124"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/F4/embed/mml-math-134.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/F4/embed/mml-math-134.gif"/></noscript></span></span> as a function of selection for three dominance regimes: sex-specific beneficial (<span class="inline-formula" id="inline-formula-125"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/F4/embed/mml-math-135.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/F4/embed/mml-math-135.gif"/></noscript></span></span>), additive (<span class="inline-formula" id="inline-formula-126"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/F4/embed/mml-math-136.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/F4/embed/mml-math-136.gif"/></noscript></span></span>), and deleterious (<span class="inline-formula" id="inline-formula-127"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/F4/embed/mml-math-137.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/F4/embed/mml-math-137.gif"/></noscript></span></span>). The sex-specific deleterious dominance curve is at an unstable equilibrium, while the additive and beneficial dominance curves are at a stable equilibrium (for all curves <span class="inline-formula" id="inline-formula-128"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/F4/embed/mml-math-138.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/F4/embed/mml-math-138.gif"/></noscript></span></span> and the equilibrium frequency is 0.5). Sex-specific beneficial dominance always results in the lowest divergence between the sexes. The inset graph highlights the similarly low divergence values generated under weak and moderately weak selection. C) Male-female <span class="inline-formula" id="inline-formula-129"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/F4/embed/mml-math-139.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/F4/embed/mml-math-139.gif"/></noscript></span></span> at a sex-beneficial locus (<span class="inline-formula" id="inline-formula-130"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/F4/embed/mml-math-140.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/F4/embed/mml-math-140.gif"/></noscript></span></span>: the polymorphic equilibrium is stable only if <span class="inline-formula" id="inline-formula-131"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/F4/embed/mml-math-141.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/F4/embed/mml-math-141.gif"/></noscript></span></span>) as a function of <span class="inline-formula" id="inline-formula-132"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/F4/embed/mml-math-142.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/F4/embed/mml-math-142.gif"/></noscript></span></span> allele frequency for varying degrees of asymmetry in selection (<span class="inline-formula" id="inline-formula-133"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/F4/embed/mml-math-143.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/F4/embed/mml-math-143.gif"/></noscript></span></span>) with a fixed mean selection coefficient (<span class="inline-formula" id="inline-formula-134"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/F4/embed/mml-math-144.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/F4/embed/mml-math-144.gif"/></noscript></span></span>).</p><div class="sb-div caption-clear"></div></div></div><p id="p-28">Greater divergence can be generated across a broader range of selection values when an antagonistic locus transiently sweeps through a population. Here a viability cost of 10% produces higher divergence than at equilibrium, although divergence is still low in absolute terms (<span class="inline-formula" id="inline-formula-135"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-145.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-145.gif"/></noscript></span></span> across dominance values, under random mating). Again, at least a 30% viability cost would be required to produce <span class="inline-formula" id="inline-formula-136"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-146.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-146.gif"/></noscript></span></span> values above 0.05. Sex-specific beneficial dominance (<span class="inline-formula" id="inline-formula-137"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-147.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-147.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-138"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-148.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-148.gif"/></noscript></span></span>) is expected to generate the lowest levels of between-sex divergence, while sex-specific deleterious dominance (<span class="inline-formula" id="inline-formula-139"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-149.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-149.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-140"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-150.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-150.gif"/></noscript></span></span>) yields the greatest levels of divergence, though such a scenario seems biologically unstable (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4B</a>). Importantly, under weak selection dominance has only a negligible effect on divergence. In fact, varying dominance does not generate quantitative changes in <span class="inline-formula" id="inline-formula-141"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-151.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-151.gif"/></noscript></span></span> unless selection is remarkably strong (<span class="inline-formula" id="inline-formula-142"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-152.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-152.gif"/></noscript></span></span>). Rather, asymmetry in selection seems a more important driver of divergence in non-equilibrium populations, as this asymmetry is precisely the factor that moves populations away from equilibrium conditions to a state in which the least costly allele sweeps to fixation. Across the range of <em>α</em> values with a fixed mean strength of selection between the sexes, divergence slightly increases as asymmetry between the sexes increases (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4C</a>). However, the male-female <span class="inline-formula" id="inline-formula-143"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-153.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-153.gif"/></noscript></span></span> values are of the same magnitude despite strong asymmetry when the mean strength of selection in confined in this manner. When the strength of selection varies independently between the sexes, increasing <em>α</em> yields much greater divergence, though this result is confounded by overall stronger selection in one sex. Overall, substantial divergence between the sexes still requires strong selection in non-equilibrium populations.</p></div><div id="sec-12" class="subsection"><h3>Sexual antagonism generates a substantial genetic load</h3><p id="p-29">Since the mean relative fitness for females and the mean relative fitness for males are each maximized under fixation for different alleles at an antagonistic locus, sexually antagonistic selection generates a genetic load within the population at both a polymorphic equilibrium and during a selective sweep. At equilibrium under random mating, the load is maximized if the strengths of selection in each sex are equal (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5A</a>), and dominance has little to no effect. Importantly, across strengths of selection up to <span class="inline-formula" id="inline-formula-144"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-154.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-154.gif"/></noscript></span></span>, the load generated at equilibrium exceeds <span class="inline-formula" id="inline-formula-145"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-155.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-155.gif"/></noscript></span></span> between males and females by nearly a factor of 10 (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5B</a>). For example, a 10% viability cost (<span class="inline-formula" id="inline-formula-146"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-156.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-156.gif"/></noscript></span></span>) results in a reduction of population fitness up to 5%, with a maximum <span class="inline-formula" id="inline-formula-147"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-157.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-157.gif"/></noscript></span></span> value of 0.0007. The load produced by a single antagonistic locus with <span class="inline-formula" id="inline-formula-148"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-158.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-158.gif"/></noscript></span></span> equal to the <strong>mean</strong> male-female <span class="inline-formula" id="inline-formula-149"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-159.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-159.gif"/></noscript></span></span> reported in human HAPMAP data (<a id="xref-ref-25-4" class="xref-bibr" href="#ref-25">Lucotte <em>et al.</em> 2016</a>) would exceed 20% (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5B</a>). This relationship indicates that even weak selection driving low—and probably undetectable—levels of divergence can generate a substantial fitness reduction due to the sex-specific nature of selection.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/11/3813/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The genetic load created by sexually antagonistic selection. A) The genetic load generated at equilibrium for an additive locus across strengths (s) and asymmetries (&#x3B1;) of selection. B) A comparison of male-female divergence and genetic load for a locus at equilibrium across a gradient of selection coefficients with varying asymmetry. The load generated at a locus exceeds the degree of divergence between the sexes. Each curve corresponds to a different fixed strength of selection from  to  and each point along the curves corresponds to a different value of &#x3B1; from 0.6 to 2. C) The population inbreeding coefficient  for an additive locus () at equilibrium. The excess of heterozygous individuals in the population represents the departures from Hardy-Weinberg equilibrium due to sex-specific selection." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-244047068" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;The genetic load created by sexually antagonistic selection. A) The genetic load generated at equilibrium for an additive locus across strengths (&lt;em&gt;s&lt;/em&gt;) and asymmetries (&lt;em&gt;&#x3B1;&lt;/em&gt;) of selection. B) A comparison of male-female divergence and genetic load for a locus at equilibrium across a gradient of selection coefficients with varying asymmetry. The load generated at a locus exceeds the degree of divergence between the sexes. Each curve corresponds to a different fixed strength of selection from &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-160&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; to &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-161&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; and each point along the curves corresponds to a different value of &lt;em&gt;&#x3B1;&lt;/em&gt; from 0.6 to 2. C) The population inbreeding coefficient &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-162&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; for an additive locus (&lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-163&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;) at equilibrium. The excess of heterozygous individuals in the population represents the departures from Hardy-Weinberg equilibrium due to sex-specific selection.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/11/3813/F5.medium.gif" width="440" height="196"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/9/11/3813/F5.medium.gif" width="440" height="196"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/11/3813/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/11/3813/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494846" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-30" class="first-child">The genetic load created by sexually antagonistic selection. A) The genetic load generated at equilibrium for an additive locus across strengths (<em>s</em>) and asymmetries (<em>α</em>) of selection. B) A comparison of male-female divergence and genetic load for a locus at equilibrium across a gradient of selection coefficients with varying asymmetry. The load generated at a locus exceeds the degree of divergence between the sexes. Each curve corresponds to a different fixed strength of selection from <span class="inline-formula" id="inline-formula-150"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/F5/embed/mml-math-160.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/F5/embed/mml-math-160.gif"/></noscript></span></span> to <span class="inline-formula" id="inline-formula-151"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/F5/embed/mml-math-161.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/F5/embed/mml-math-161.gif"/></noscript></span></span> and each point along the curves corresponds to a different value of <em>α</em> from 0.6 to 2. C) The population inbreeding coefficient <span class="inline-formula" id="inline-formula-152"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/F5/embed/mml-math-162.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/F5/embed/mml-math-162.gif"/></noscript></span></span> for an additive locus (<span class="inline-formula" id="inline-formula-153"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/F5/embed/mml-math-163.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/F5/embed/mml-math-163.gif"/></noscript></span></span>) at equilibrium. The excess of heterozygous individuals in the population represents the departures from Hardy-Weinberg equilibrium due to sex-specific selection.</p><div class="sb-div caption-clear"></div></div></div><p id="p-31">An alternative way to examine load is by quantifying the excess of heterozygosity due to sexually antagonistic selection, using the <span class="inline-formula" id="inline-formula-154"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-164.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-164.gif"/></noscript></span></span> statistic (the inbreeding coefficient). Here sex-specific selection creates homozygous pools of each sex within a generation, which leads to an excess of heterozygotes at the start of the next generation. Under weak selection, these departures from Hardy-Weinberg equilibrium are of similar magnitude as male-female <span class="inline-formula" id="inline-formula-155"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-165.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-165.gif"/></noscript></span></span> (<a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5C</a>). However, under strong sexual antagonism—such as that required to generate the empirically observed divergence values—<span class="inline-formula" id="inline-formula-156"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-166.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-166.gif"/></noscript></span></span> can approach 10%.</p><p id="p-32">Antagonistic loci that do not have a polymorphic equilibrium tend to produce even greater load while sweeping than a locus at stable equilibrium under similar selective conditions (File S1). Unless selection was very weak (<span class="inline-formula" id="inline-formula-157"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-167.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-167.gif"/></noscript></span></span>), load tended to exceed 10%. Under strong, asymmetric selection load can approach 70% during a sweep. Additionally, the fitness cost of sexual antagonism remains after an allele fixes. The load generated during a sweep is affected by dominance, with additive loci generating loads that are intermediate to the other dominance scenarios. Beneficial dominance within each sex can apparently resolve some of the underlying antagonism by shielding selection on heterozygotes and therefore reducing the load. In contrast, sex-specific deleterious dominance generated the greatest load.</p></div><div id="sec-13" class="subsection"><h3>Genome-wide antagonistic selection also produces low divergence</h3><p id="p-33">Our analytical results are based on a single-locus model, yet empirical studies report averages across large numbers of loci. To complement the single-locus theory, we quantified the effects of sexually antagonistic selection throughout the genome using individual-based simulations in SLiM <a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Haller and Messer (2019)</a>. First, we analyzed male-female <span class="inline-formula" id="inline-formula-158"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-168.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-168.gif"/></noscript></span></span> values calculated using true population-wide allele frequencies (<em>i.e.</em>, whole-population <span class="inline-formula" id="inline-formula-159"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-169.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-169.gif"/></noscript></span></span>). Simulations in which every new mutation was sexually antagonistic in a population of 10,000 individuals resulted in a mean male-female <span class="inline-formula" id="inline-formula-160"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-170.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-170.gif"/></noscript></span></span> of 0.00005 and a between-replicate standard deviation of 0.0001, consistent with the single-locus theory (since <em>s</em> was around 0.01). However, entirely neutral simulations (equal mutation rates but no selection) resulted in the same mean and SD of male-female <span class="inline-formula" id="inline-formula-161"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-171.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-171.gif"/></noscript></span></span> values. Both the sexually antagonistic and neutral simulations averaged around 1,400 SNPs after 1,000 generations of evolution. Although qualitatively similar, the distribution of whole-population male-female <span class="inline-formula" id="inline-formula-162"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-172.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-172.gif"/></noscript></span></span> values across loci was statistically significantly different between the neutral and sexually antagonistic simulations (Kolmogorov-Smirnov test: <span class="inline-formula" id="inline-formula-163"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-173.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-173.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-164"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-174.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-174.gif"/></noscript></span></span>; <a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6A</a>). However, this difference in distributions was driven by the larger number of intermediate frequency alleles in the sexually antagonistic simulations. In particular, neutral simulations across all five replicates had only two SNPs with a frequency above 10%, while the sexually antagonistic simulations had over 200 SNPs with a frequency above 10%. Despite there being true differences between the neutral and sexually antagonistic simulations, the male-female divergences observed were still exceptionally low. In fact, neither model had any loci with male-female <span class="inline-formula" id="inline-formula-165"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-175.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-175.gif"/></noscript></span></span> greater than 0.001 (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6A</a>). Sexually antagonistic simulations had an average 21% decrease in population fitness (<span class="inline-formula" id="inline-formula-166"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-176.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-176.gif"/></noscript></span></span>) after 1000 generations of evolution, again consistent with single-locus calculations. Even the minimum load observed under the sexually antagonistic scenario corresponded to a 18% decrease in population fitness.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/11/3813/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The distribution of per locus  values generated from simulated populations after 1000 generations of evolution. A) The density of male-female  values for a population of 10,000 individuals is centered around  = 0.0005. The neutral (gray) and sexually antagonistic (teal) simulations were similar but statistically significantly different. B) The distribution of male-female  values when subsampling the full populations to either 100 individuals or 50 individuals with equal sex ratios. Divergence values were calculated using the theoretical derivation (Equation 4) and Weir and Cockerham&#x2019;s . Subsampling increases the tail of the distribution substantially. The sexually antagonistic simulations are significantly different from the neutral simulations due to an increased sampling variance in the sexually antagonistic scenario. C) Cumulative distribution curves of per-locus male-female  values, both between random samples from the two sexes (solid lines) and between sets of individuals chosen randomly independently of sex (dotted lines). Male-female  distributions differed between neutral (gray) and antagonistic (teal) simulations but were not higher for between-sex comparisons, showing that higher  values in the antagonistic simulation was not directly due to selection." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-244047068" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;The distribution of per locus &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-177&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; values generated from simulated populations after 1000 generations of evolution. A) The density of male-female &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-178&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; values for a population of 10,000 individuals is centered around &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-179&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; = 0.0005. The neutral (gray) and sexually antagonistic (teal) simulations were similar but statistically significantly different. B) The distribution of male-female &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-180&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; values when subsampling the full populations to either 100 individuals or 50 individuals with equal sex ratios. Divergence values were calculated using the theoretical derivation (Equation 4) and Weir and Cockerham&#x2019;s &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-181&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;. Subsampling increases the tail of the distribution substantially. The sexually antagonistic simulations are significantly different from the neutral simulations due to an increased sampling variance in the sexually antagonistic scenario. C) Cumulative distribution curves of per-locus male-female &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-182&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; values, both between random samples from the two sexes (solid lines) and between sets of individuals chosen randomly independently of sex (dotted lines). Male-female &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-183&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; distributions differed between neutral (gray) and antagonistic (teal) simulations but were not higher for between-sex comparisons, showing that higher &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-184&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; values in the antagonistic simulation was not directly due to selection.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/11/3813/F6.medium.gif" width="440" height="394"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/9/11/3813/F6.medium.gif" width="440" height="394"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/11/3813/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/11/3813/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494849" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-34" class="first-child">The distribution of per locus <span class="inline-formula" id="inline-formula-167"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/F6/embed/mml-math-177.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/F6/embed/mml-math-177.gif"/></noscript></span></span> values generated from simulated populations after 1000 generations of evolution. A) The density of male-female <span class="inline-formula" id="inline-formula-168"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/F6/embed/mml-math-178.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/F6/embed/mml-math-178.gif"/></noscript></span></span> values for a population of 10,000 individuals is centered around <span class="inline-formula" id="inline-formula-169"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/F6/embed/mml-math-179.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/F6/embed/mml-math-179.gif"/></noscript></span></span> = 0.0005. The neutral (gray) and sexually antagonistic (teal) simulations were similar but statistically significantly different. B) The distribution of male-female <span class="inline-formula" id="inline-formula-170"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/F6/embed/mml-math-180.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/F6/embed/mml-math-180.gif"/></noscript></span></span> values when subsampling the full populations to either 100 individuals or 50 individuals with equal sex ratios. Divergence values were calculated using the theoretical derivation (Equation 4) and Weir and Cockerham’s <span class="inline-formula" id="inline-formula-171"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/F6/embed/mml-math-181.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/F6/embed/mml-math-181.gif"/></noscript></span></span>. Subsampling increases the tail of the distribution substantially. The sexually antagonistic simulations are significantly different from the neutral simulations due to an increased sampling variance in the sexually antagonistic scenario. C) Cumulative distribution curves of per-locus male-female <span class="inline-formula" id="inline-formula-172"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/F6/embed/mml-math-182.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/F6/embed/mml-math-182.gif"/></noscript></span></span> values, both between random samples from the two sexes (solid lines) and between sets of individuals chosen randomly independently of sex (dotted lines). Male-female <span class="inline-formula" id="inline-formula-173"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/F6/embed/mml-math-183.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/F6/embed/mml-math-183.gif"/></noscript></span></span> distributions differed between neutral (gray) and antagonistic (teal) simulations but were not higher for between-sex comparisons, showing that higher <span class="inline-formula" id="inline-formula-174"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/F6/embed/mml-math-184.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/F6/embed/mml-math-184.gif"/></noscript></span></span> values in the antagonistic simulation was not directly due to selection.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-14" class="subsection"><h3>Sampling variance can generate spurious signals of male-female divergence</h3><p id="p-35">Both the single locus model and genome-wide simulations indicate that, while theoretically possible, we would need strong sexually antagonistic viability selection to maintain high divergence between the sexes. Alternatively, the large observed <span class="inline-formula" id="inline-formula-175"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-185.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-185.gif"/></noscript></span></span> statistics might be due to sampling variance. The empirical studies we cite have relatively small sample sizes (between 15 and 200). The male-female <span class="inline-formula" id="inline-formula-176"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-186.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-186.gif"/></noscript></span></span> values we reported above from simulation were calculated from the <em>entire</em> population. To evaluate the effect of sampling, we calculated male-female <span class="inline-formula" id="inline-formula-177"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-187.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-187.gif"/></noscript></span></span> values from random samples of individuals in our SLiM simulations of two sizes: 100 individuals (50 females and 50 males) and 50 individuals (25 females and 25 males). This subsampling produced dramatically higher male-female <span class="inline-formula" id="inline-formula-178"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-188.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-188.gif"/></noscript></span></span> values under both the neutral (100 individuals, mean ± standard deviation across replicates: <span class="inline-formula" id="inline-formula-179"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-189.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-189.gif"/></noscript></span></span>; 50 individuals: <span class="inline-formula" id="inline-formula-180"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-190.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-190.gif"/></noscript></span></span>) and sexually antagonistic (100 individuals: <span class="inline-formula" id="inline-formula-181"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-191.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-191.gif"/></noscript></span></span>; 50 individuals: <span class="inline-formula" id="inline-formula-182"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-192.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-192.gif"/></noscript></span></span>) simulations (<a id="xref-fig-6-3" class="xref-fig" href="#F6">Figure 6B</a>). Additionally, the tail of the distribution was lengthened under both neutral and sexually antagonistic simulation, including in both cases estimated values of <span class="inline-formula" id="inline-formula-183"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-193.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-193.gif"/></noscript></span></span> several orders of magnitude larger than the values calculated from actual population allele frequencies. In other words, ignoring the noisiness of the <span class="inline-formula" id="inline-formula-184"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-194.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-194.gif"/></noscript></span></span> estimator would lead us to conclude that between-sex divergence is much higher than it actually is. There was a significant difference in the distribution of male-female <span class="inline-formula" id="inline-formula-185"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-195.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-195.gif"/></noscript></span></span> values between the neutral and sexually antagonistic simulations both when subsampling at 100 individuals (Kolmogorov-Smirnov test: <span class="inline-formula" id="inline-formula-186"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-196.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-196.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-187"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-197.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-197.gif"/></noscript></span></span>) and 50 individuals (<span class="inline-formula" id="inline-formula-188"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-198.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-198.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-189"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-199.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-199.gif"/></noscript></span></span>). However, there was no correlation between the <span class="inline-formula" id="inline-formula-190"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-200.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-200.gif"/></noscript></span></span> values calculated from the full population and those obtained from samples of either 100 individuals (<span class="inline-formula" id="inline-formula-191"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-201.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-201.gif"/></noscript></span></span>) or the 50 individual subset (<span class="inline-formula" id="inline-formula-192"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-202.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-202.gif"/></noscript></span></span>). This lack of correlation also holds true for the neutral model (100 individuals: <span class="inline-formula" id="inline-formula-193"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-203.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-203.gif"/></noscript></span></span>; 50 individuals: <span class="inline-formula" id="inline-formula-194"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-204.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-204.gif"/></noscript></span></span>).</p><p id="p-36">Empirical studies often use estimators of <span class="inline-formula" id="inline-formula-195"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-205.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-205.gif"/></noscript></span></span>, such as Weir and Cockerham’s <span class="inline-formula" id="inline-formula-196"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-206.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-206.gif"/></noscript></span></span> to account for population size and sample allele frequencies, rather than population allele frequencies. Therefore, in addition to using equation 4, we calculated male-female <span class="inline-formula" id="inline-formula-197"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-207.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-207.gif"/></noscript></span></span> values in subsamples using Weir and Cockerham’s method (<a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Weir and Cockerham 1984</a>; <a id="xref-ref-3-3" class="xref-bibr" href="#ref-3">Bhatia <em>et al.</em> 2013</a>). However, this did not qualitatively affect the tail of the distribution of <span class="inline-formula" id="inline-formula-198"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-208.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-208.gif"/></noscript></span></span> (<a id="xref-fig-6-4" class="xref-fig" href="#F6">Figure 6B</a>). This is not surprising, because Weir and Cockerham’s method is designed to obtain <em>unbiased</em> estimates of <span class="inline-formula" id="inline-formula-199"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-209.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-209.gif"/></noscript></span></span> – and so re-centers the mean of the distribution – but does not reduce sampling variance.</p><p id="p-37">Although there were more high male-female <span class="inline-formula" id="inline-formula-200"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-210.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-210.gif"/></noscript></span></span> sites in samples from the sexually antagonistic simulations (<a id="xref-fig-6-5" class="xref-fig" href="#F6">Figure 6B</a>), this did not seem to be a direct result of selection, but was due to the fact that there are many more intermediate frequency alleles in the sexually antagonistic simulations because of balanced polymorphisms. (If the larger number of high <span class="inline-formula" id="inline-formula-201"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-211.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-211.gif"/></noscript></span></span> sites were due to selection on those sites, then we’d expect to see a correlation between estimated <span class="inline-formula" id="inline-formula-202"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-212.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-212.gif"/></noscript></span></span> and population <span class="inline-formula" id="inline-formula-203"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-213.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-213.gif"/></noscript></span></span>, which, as noted above, we do not.) To test this hypothesis, we calculated <span class="inline-formula" id="inline-formula-204"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-214.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-214.gif"/></noscript></span></span> between two random samples of size 50 drawn from each simulation <em>independently</em> of sex, and also between random samples of size 25. If the enrichment of high between-sex <span class="inline-formula" id="inline-formula-205"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-215.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-215.gif"/></noscript></span></span> sites in the antagonistic simulations are in fact due to the difference in allele frequency distribution rather than the direct result of selection, then the enrichment should persist even in these samples drawn after randomizing sex. Indeed this enrichment persists, as shown in <a id="xref-fig-6-6" class="xref-fig" href="#F6">Figure 6C</a>. Thus, the higher number of intermediate frequency sites in the sexually antagonistic model creates a higher sampling variance of <span class="inline-formula" id="inline-formula-206"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-216.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-216.gif"/></noscript></span></span>, as expected based on theory (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Jakobsson <em>et al.</em> 2013</a>). In particular, the tail of the <span class="inline-formula" id="inline-formula-207"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-217.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-217.gif"/></noscript></span></span> distributions show many higher values, such that an increase by two orders of magnitude relative to the full population was observed (<a id="xref-fig-6-7" class="xref-fig" href="#F6">Figure 6B</a>). These results suggest that separating signals of weak antagonistic selection from sampling noise will be extremely difficult.</p></div></div><div class="section discussion" id="sec-15"><h2 class="">Discussion</h2><p id="p-38">Sexually antagonistic viability selection creates allelic divergence between the sexes because the proportions of each genotype that die before reproduction differs between the sexes. This between-sex divergence for non-sex-linked elements is created anew each generation because chromosomal segregation re-assorts autosomal associations across the sexes during sexual reproduction. An emerging trend in sexual antagonism research is the use of male-female genomic comparisons to identify sexually antagonistic loci. These recent studies identified hundreds of sexually divergent autosomal loci with mean divergence between the sexes in the range of 2–7% (<a id="xref-ref-25-5" class="xref-bibr" href="#ref-25">Lucotte <em>et al.</em> 2016</a>; <a id="xref-ref-13-5" class="xref-bibr" href="#ref-13">Flanagan and Jones 2017</a>; <a id="xref-ref-38-4" class="xref-bibr" href="#ref-38">Wright <em>et al.</em> 2018</a>). Taken as reported, these studies suggest the extent and strength of sexually antagonistic selection is far greater than might be anticipated. To assess these claims, we used a population genetic model to determine the magnitude of divergence generated by sexually antagonistic viability selection, the strength of selection required to drive such divergence, and the population fitness costs generated by this process.</p><p id="p-39">Although sexual antagonism has been a topic of particular interest over the last few decades (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Arnqvist and Rowe 2005</a>), some of the early investigations of sex-specific selection were largely motivated as part of a general attempt to elucidate all possible means by which the large amounts of segregating polymorphisms observed within natural populations could be maintained (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Lewontin 1974</a>). In this context, <a id="xref-ref-22-8" class="xref-bibr" href="#ref-22">Kidwell <em>et al.</em> (1977)</a> focused on how sex-specific selection, could maintain a polymorphism at an autosomal locus when alleles had opposing effects in the sexes. Our analysis agrees with <a id="xref-ref-22-9" class="xref-bibr" href="#ref-22">Kidwell <em>et al.</em> (1977)</a>, though we highlight that maintenance of such polymorphisms would create a substantial genetic load. Additionally, with weaker selection, the parameter space allowing a stable polymorphism becomes quite narrow. These results highlight the necessity for considering biologically relevant conditions – as discussed by <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Smith and Hoekstra (1980)</a> – particularly when theory is is informing signatures of selection within the genome. Our analysis also allows for non-random mate choice, a potentially important underlying component of sexual conflict (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Arnqvist and Rowe 2005</a>). We build on <a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Bodmer (1965)</a> to generate a fully generalizable mate choice matrix and find that assortative mating can indeed have a large impact on the conditions for the maintenance of polymorphism. Supporting previous simulations (<a id="xref-ref-1-4" class="xref-bibr" href="#ref-1">Arnqvist 2011</a>), we found positive assortative mating by fitness maintained polymorphisms. In particular, we show that the combination of asymmetrical selection between the sexes and deleterious dominance conditions expanded the equilibrium space relative to random mating. However, such deleterious sex-specific dominance would likely be selected against, suggesting that the strength of selection is the more relevant parameter in natural populations.</p><p id="p-40">While the maintenance of polymorphism may have been a primary motivation for previous work, a goal of modern genomics is to use specific signals of genomic differentiation to identify the loci underlying sexually antagonistic genetic effects (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Mank 2017a</a>). Building on our previous work (<a id="xref-ref-21-5" class="xref-bibr" href="#ref-21">Kasimatis <em>et al.</em> 2017</a>) allowed us to consider the expected degree of between-sex divergence both when an antagonistic polymorphism is maintained at equilibrium in the population, and when no such stable equilibrium exists, so one of the two alleles sweeps toward fixation to the detriment of one sex. Our model and accompanying simulations highlight several potential limitations of detecting sex-specific differentiation in empirical studies.</p><p id="p-41">First, detectable quantitative divergence between the sexes requires exceptionally strong sexually antagonistic selection. Previous work indicates that <span class="inline-formula" id="inline-formula-208"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-218.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-218.gif"/></noscript></span></span> values between populations is of order <span class="inline-formula" id="inline-formula-209"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-219.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-219.gif"/></noscript></span></span> (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Charlesworth and Charlesworth 2010</a>; <a id="xref-ref-8-6" class="xref-bibr" href="#ref-8">Cheng and Kirkpatrick 2016</a>), but we find that substantially lower values of <span class="inline-formula" id="inline-formula-210"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-220.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-220.gif"/></noscript></span></span> are often obtained in practice. Even a 10% viability cost in each sex resulted in between-sex <span class="inline-formula" id="inline-formula-211"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-221.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-221.gif"/></noscript></span></span> values of less than 0.001 (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4</a>), a signal that is unlikely to be distinguishable from noise without sampling many thousands of individuals within each sex. Critically, to achieve divergence values greater than 0.03 – such as seen in estimates from human data – would require a 30–60% viability cost in each sex under our model. These remarkably high sex-specific mortality rates are, to the best of our knowledge, not observed in nature (see <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Singh and Punzalan 2018</a>). (However, exceptionally high fecundity animals might withstand such high sex-specific mortality (see <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Williams 1975</a>).) <a id="xref-ref-8-7" class="xref-bibr" href="#ref-8">Cheng and Kirkpatrick (2016)</a> also pointed out that even small male-female <span class="inline-formula" id="inline-formula-212"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-222.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-222.gif"/></noscript></span></span> values would require unrealistic amounts of genetic load, but still argue in favor of ongoing sex-specific selection at many genes. Strong sex-specific gametic selection is perhaps more plausible than viability selection on adults, but such high levels of genotype-dependent gamete “mortality” in these organisms still do not seem consistent with empirical observations. Additionally, gametic selection requires an epistatic association between the autosomal locus and the sex determining region, which seems highly implausible across so many loci.</p><p id="p-42">Second, asymmetry in the strength of selection between the sexes is critical in determining the degree of divergence generated. When the strength of selection is weak and approximately the same between the sexes, polymorphisms may be stably maintained, but between-sex divergence is small. However, there is no <em>a priori</em> reason to expect that antagonistic mutations should be perfectly symmetrical in their effects and therefore that polymorphic loci should be stable over time. Alleles with more asymmetric effects will often sweep, producing larger but transient between-sex divergences, although again only under moderate to strong selection. Of course, other types of selection (<em>e.g.</em>, spatially varying selection) could contribute to maintenance of more asymmetric polymorphisms, but the observations about genetic load should still hold. Here, we found that dominance has little quantitative effect on male-female divergence, particularly when selection is weak. In general, understanding what the distribution of sex-specific effects underlying antagonistic selection looks like will provide important information on the potential for sexually antagonistic loci to contribute to genetic variation and genome evolution.</p><p id="p-43">Thus, the theoretical predictions from our single locus model seem at odds with the empirical patterns reported to date. Taken as true measurements of sexually antagonistic selection, the empirical data could be described by two, non-exclusive genomic patterns. Divergent loci could either be stable polymorphisms or could be arising and sweeping to fixation through a constant genomic churn of antagonistic interactions. Either of these explanations require an exceptionally high genetic load. Again, there is currently no indication that mortality occurs in such a high, sex-specific manner, particularly in some of the vertebrate species that have been examined.</p><p id="p-44">Individual-based simulations with many linked selected loci genome-wide recapitulate the predictions of the single-locus model, finding again that even in this more complex situation, weak selection can only produce very low levels of divergence. Most importantly, however, we found that estimating male-female <span class="inline-formula" id="inline-formula-213"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-223.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-223.gif"/></noscript></span></span> from samples of the sizes used in the literature (hundreds or less) produced distributions with larger means and longer tails, even in the complete absence of antagonistic selection. Even in simulations with antagonistic selection, any high divergence values were a result of random sampling noise, and did not correlate with the true divergence values or strength of selection. These simulations highlight the sensitivity of <span class="inline-formula" id="inline-formula-214"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-224.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-224.gif"/></noscript></span></span> statistics to sampling variance, which is a major obstacle for identification of antagonistic loci from sex-specific differentiation. Most existing empirical studies have not taken these effects fully into account. Our simulations are not intended to be comprehensive, but demonstrate that sampling variance can be more important than selection itself in driving high estimates of divergence, and highlight the need for proper sampling theory.</p><p id="p-45">At the very least, studies analyzing male-female <span class="inline-formula" id="inline-formula-215"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-225.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-225.gif"/></noscript></span></span> values should compare values to empirical distributions found by random permutation of sex labels, as done by <a id="xref-ref-12-4" class="xref-bibr" href="#ref-12">Dutoit <em>et al.</em> (2018)</a>. Connecting significant SNPs to a phenotype such as sex-biased expression (<a id="xref-ref-8-8" class="xref-bibr" href="#ref-8">Cheng and Kirkpatrick 2016</a>; <a id="xref-ref-12-5" class="xref-bibr" href="#ref-12">Dutoit <em>et al.</em> 2018</a>) can provide additional evidence for selection, but it is difficult to control for all confounding factors (<em>e.g.</em>, overall expression level). Furthermore, population substructure remains a concern even when comparing to permuted data if there is a cryptic correlation of sampling with sex. For instance, suppose that the sampled population is composed of a mixture of two diverged subpopulations, and that the sex and admixture coefficients of the sampled individuals are correlated. As noted by <a id="xref-ref-8-9" class="xref-bibr" href="#ref-8">Cheng and Kirkpatrick (2016)</a>, this will create spurious male-female <span class="inline-formula" id="inline-formula-216"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-226.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-226.gif"/></noscript></span></span>. (The samples in <a id="xref-ref-12-6" class="xref-bibr" href="#ref-12">Dutoit <em>et al.</em> (2018)</a> were composed of mating pairs from a single island, so this seems unlikely to explain their results, unless one sex is much more likely to disperse between islands than the other.) Other issues beyond sampling variance may well play a role in the large observed male-female <span class="inline-formula" id="inline-formula-217"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-227.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-227.gif"/></noscript></span></span> values. Reads from the sex chromosome that are wrongly aligned to an autosome, particularly in the heterogametic sex, have the potential to generate spurious <span class="inline-formula" id="inline-formula-218"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-228.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-228.gif"/></noscript></span></span> peaks (see <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Tsai <em>et al.</em> (2019)</a>), an issue that may affect some classes of genes – such as those with sex-biased expression – more than others.</p><p id="p-46">Furthermore, some studies report large numbers of loci with high average <span class="inline-formula" id="inline-formula-219"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-229.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-229.gif"/></noscript></span></span>. Should we interpret this as evidence of antagonistic selection across many loci simultaneously, or at just a few loci that affect others through linkage? This is not clear, because each generation’s sex-specific selection on a single antagonistic allele will also cause between-sex frequency differences at other loci to the extent they are in linkage disequilibrium with the locus under selection. More work is needed to quantify this effect.</p><p id="p-47">More generally, our investigation calls into question whether it is even possible, at any sample size, to identify the action of realistically strong sex-specific antagonistic selection using male-female <span class="inline-formula" id="inline-formula-220"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-230.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-230.gif"/></noscript></span></span>. Many of the loci previously suggested as contributing to sex-specific antagonistic selection seem likely to be spurious signals resulting from poor statistical inference. While we believe sexually antagonistic selection does contribute to genomic evolution, we strongly caution against the use and over-interpretation of male-female <span class="inline-formula" id="inline-formula-221"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-231.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3813/embed/mml-math-231.gif"/></noscript></span></span> statistics, especially with small sample sizes, and until the potential bioinformatic confounders are better understood.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-48">We thank Göran Arnqvist and Thomas Nelson for their helpful discussion and two anonymous reviewers for their comments. This work was supported by the National Institutes of Health (training grant T32GM007413 to KRK and R01GM102511 to PCP) and the ARCS Oregon Chapter (KRK).</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-49">Supplemental material available at FigShare: <a href="https://doi.org/10.25387/g3.9785951">https://doi.org/10.25387/g3.9785951</a>.</p></li><li class="fn" id="fn-2"><p id="p-50"><em>Communicating editor: K. Thornton</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2019-07-23"><span class="received-label">Received </span>July 23, 2019.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2019-09-16"><span class="accepted-label">Accepted </span>September 16, 2019.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2019 Kasimatis <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3813.1" data-doi="10.1111/j.1558-5646.2011.01270.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Arnqvist</span>,  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Assortative mating by fitness and sexually antagonistic genetic variation.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">65</span>: <span class="cit-fpage">2111</span>–<span class="cit-lpage">2116</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1558-5646.2011.01270.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.stitle%253DEvolution%26rft.aulast%253DArnqvist%26rft.auinit1%253DG.%26rft.volume%253D65%26rft.issue%253D7%26rft.spage%253D2111%26rft.epage%253D2116%26rft.atitle%253DAssortative%2Bmating%2Bby%2Bfitness%2Band%2Bsexually%2Bantagonistic%2Bgenetic%2Bvariation.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1558-5646.2011.01270.x%26rft_id%253Dinfo%253Apmid%252F21729064%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1558-5646.2011.01270.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21729064&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3813.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000292447000024&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-book" id="cit-9.11.3813.2" data-doi="10.1515/9781400850600"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Arnqvist</span>,  <span class="cit-name-given-names">G.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Rowe</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-source">Sexual conflict</span>, <span class="cit-publ-name">Princeton University Press</span>, <span class="cit-publ-loc">Princeton, NJ</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1515/9781400850600</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSexual%2Bconflict%26rft_id%253Dinfo%253Adoi%252F10.1515%252F9781400850600%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1515/9781400850600&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3813.3" data-doi="10.1101/gr.154831.113"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bhatia</span>,  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Patterson</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Sankararaman</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. L.</span> <span class="cit-name-surname">Price</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Estimating and interpreting FST: the impact of rare variants.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">1514</span>–<span class="cit-lpage">1521</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.154831.113</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.154831.113%26rft_id%253Dinfo%253Apmid%252F23861382%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjIzLzkvMTUxNCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzkvMTEvMzgxMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3813.4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bodmer</span>,  <span class="cit-name-given-names">W. F.</span></span></li></ol><cite>, <span class="cit-pub-date">1965</span> <span class="cit-article-title">Differential fertility in population genetics models.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">51</span>: <span class="cit-fpage">411</span>–<span class="cit-lpage">424</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DBodmer%26rft.auinit1%253DW.%2BF.%26rft.volume%253D51%26rft.issue%253D3%26rft.spage%253D411%26rft.epage%253D424%26rft.atitle%253DDIFFERENTIAL%2BFERTILITY%2BIN%2BPOPULATION%2BGENETICS%2BMODELS%26rft_id%253Dinfo%253Apmid%252F17248243%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo4OiJnZW5ldGljcyI7czo1OiJyZXNpZCI7czo4OiI1MS8zLzQxMSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzkvMTEvMzgxMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3813.5" data-doi="10.1016/j.tree.2008.12.005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bonduriansky</span>,  <span class="cit-name-given-names">R.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. F.</span> <span class="cit-name-surname">Chenoweth</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Intralocus sexual conflict.</span> <abbr class="cit-jnl-abbrev">Trends Ecol. Evol.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">280</span>–<span class="cit-lpage">288</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.tree.2008.12.005</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2BEcology%2B%2526%2BEvolution%26rft.stitle%253DTrends%2Bin%2BEcology%2B%2526%2BEvolution%26rft.aulast%253DBonduriansky%26rft.auinit1%253DR.%26rft.volume%253D24%26rft.issue%253D5%26rft.spage%253D280%26rft.epage%253D288%26rft.atitle%253DIntralocus%2Bsexual%2Bconflict.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tree.2008.12.005%26rft_id%253Dinfo%253Apmid%252F19307043%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tree.2008.12.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19307043&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3813.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000266188100009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-book" id="cit-9.11.3813.6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>,  <span class="cit-name-given-names">B.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Charlesworth</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-source">Elements of evolutionary genetics</span>, <span class="cit-publ-name">Roberts &amp; Company</span>, <span class="cit-publ-loc">Engelwood, CO</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3813.7" data-doi="10.1017/S0016672300014051"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>,  <span class="cit-name-given-names">D.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Charlesworth</span></span></li></ol><cite>, <span class="cit-pub-date">1980</span> <span class="cit-article-title">Sex differences in fitness and selection for centric fusions between sex-chromosomes and autosomes.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">205</span>–<span class="cit-lpage">214</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1017/S0016672300014051</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DCharlesworth%26rft.auinit1%253DD.%26rft.volume%253D35%26rft.issue%253D2%26rft.spage%253D205%26rft.epage%253D214%26rft.atitle%253DSex%2Bdifferences%2Bin%2Bfitness%2Band%2Bselection%2Bfor%2Bcentric%2Bfusions%2Bbetween%2Bsex-chromosomes%2Band%2Bautosomes.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672300014051%26rft_id%253Dinfo%253Apmid%252F6930353%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672300014051&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=6930353&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3813.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1980JX94100007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3813.8" data-doi="10.1371/journal.pgen.1006170"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cheng</span>,  <span class="cit-name-given-names">C.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Kirkpatrick</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Sex-specific selection and sex-biased gene expression in humans and flies.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">e1006170</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1006170</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D12%26rft.spage%253D1006170e%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1006170%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1006170&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3813.9" data-doi="10.1371/journal.pbio.3000118"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Connallon</span>,  <span class="cit-name-given-names">T.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. F.</span> <span class="cit-name-surname">Chenoweth</span></span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">Dominance reversals and the maintenance of genetic variation for fitness.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">e3000118</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pbio.3000118</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BBiol.%26rft.volume%253D17%26rft.spage%253De3000118%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.3000118%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.3000118&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3813.10" data-doi="10.1534/genetics.110.123729"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Connallon</span>,  <span class="cit-name-given-names">T.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. G.</span> <span class="cit-name-surname">Clark</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The resolution of sexual antagonism by gene duplication.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">187</span>: <span class="cit-fpage">919</span>–<span class="cit-lpage">937</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.110.123729</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.123729%26rft_id%253Dinfo%253Apmid%252F21220356%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg3LzMvOTE5IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMS8zODEzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3813.11" data-doi="10.1111/j.1558-5646.2009.00934.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Connallon</span>,  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. M.</span> <span class="cit-name-surname">Cox</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Calsbeek</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Fitness consequences of sex-specific selection.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">64</span>: <span class="cit-fpage">1671</span>–<span class="cit-lpage">1682</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1558-5646.2009.00934.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.stitle%253DEvolution%26rft.aulast%253DConnallon%26rft.auinit1%253DT.%26rft.volume%253D64%26rft.issue%253D6%26rft.spage%253D1671%26rft.epage%253D1682%26rft.atitle%253DFitness%2Bconsequences%2Bof%2Bsex-specific%2Bselection.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1558-5646.2009.00934.x%26rft_id%253Dinfo%253Apmid%252F20050912%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1558-5646.2009.00934.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20050912&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3813.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000278343200012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3813.12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dutoit</span>,  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. F.</span> <span class="cit-name-surname">Mugal</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Bolívar</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Wang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Nadachowska-Brzyska</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Sex-biased gene expression, sexual antagonism and levels of genetic diversity in the collared flycatcher (ficedula albicollis) genome.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">3572</span>–<span class="cit-lpage">3581</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D27%26rft.spage%253D3572%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3813.13" data-doi="10.1111/evo.13173"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Flanagan</span>,  <span class="cit-name-given-names">S. P.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. G.</span> <span class="cit-name-surname">Jones</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Genome-wide selection components analysis in a fish with male pregnancy.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">71</span>: <span class="cit-fpage">1096</span>–<span class="cit-lpage">1105</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/evo.13173</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.volume%253D71%26rft.spage%253D1096%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fevo.13173%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/evo.13173&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3813.14" data-doi="10.1111/j.1558-5646.2009.00898.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fry</span>,  <span class="cit-name-given-names">J. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The genomic location of sexually antagonistic variation: some cautionary comments.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">64</span>: <span class="cit-fpage">1510</span>–<span class="cit-lpage">1516</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.stitle%253DEvolution%26rft.aulast%253DFry%26rft.auinit1%253DJ.%2BD.%26rft.volume%253D64%26rft.issue%253D5%26rft.spage%253D1510%26rft.epage%253D1516%26rft.atitle%253DThe%2Bgenomic%2Blocation%2Bof%2Bsexually%2Bantagonistic%2Bvariation%253A%2Bsome%2Bcautionary%2Bcomments.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1558-5646.2009.00898.x%26rft_id%253Dinfo%253Apmid%252F19922443%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1558-5646.2009.00898.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19922443&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3813.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000277096400027&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3813.15" data-doi="10.1086/280722"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Haldane</span>,  <span class="cit-name-given-names">J. B. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1937</span> <span class="cit-article-title">The effect of variation of fitness.</span> <abbr class="cit-jnl-abbrev">Am. Nat.</abbr> <span class="cit-vol">71</span>: <span class="cit-fpage">337</span>–<span class="cit-lpage">349</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1086/280722</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BNat.%26rft.volume%253D71%26rft.spage%253D337%26rft_id%253Dinfo%253Adoi%252F10.1086%252F280722%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/280722&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000200907300029&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3813.16" data-doi="10.1093/molbev/msy228"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Haller</span>,  <span class="cit-name-given-names">B. C.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. W.</span> <span class="cit-name-surname">Messer</span></span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">SLiM 3: forward genetic simulations beyond the Wright-Fisher model.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">632</span>–<span class="cit-lpage">637</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/msy228</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsy228%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msy228&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3813.17" data-doi="10.1086/698483"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Immler</span>,  <span class="cit-name-given-names">S.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. P.</span> <span class="cit-name-surname">Otto</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">The evolutionary consequences of selection at the haploid gametic stage.</span> <abbr class="cit-jnl-abbrev">Am. Nat.</abbr> <span class="cit-vol">192</span>: <span class="cit-fpage">241</span>–<span class="cit-lpage">249</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1086/698483</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BNat.%26rft.volume%253D192%26rft.spage%253D241%26rft_id%253Dinfo%253Adoi%252F10.1086%252F698483%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/698483&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3813.18" data-doi="10.1146/annurev.ecolsys.32.081501.113958"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jaenike</span>,  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Sex chromosome meiotic drive.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Ecol. Syst.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">25</span>–<span class="cit-lpage">49</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1146/annurev.ecolsys.32.081501.113958</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2BReview%2Bof%2BEcology%2Band%2BSystematics%26rft.stitle%253DAnnu.%2BRev.%2BEcol.%2BSyst.%26rft.aulast%253DJaenike%26rft.auinit1%253DJ.%26rft.volume%253D32%26rft.issue%253D1%26rft.spage%253D25%26rft.epage%253D49%26rft.atitle%253DSEX%2BCHROMOSOME%2BMEIOTIC%2BDRIVE%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.ecolsys.32.081501.113958%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.ecolsys.32.081501.113958&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000172908800002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3813.19" data-doi="10.1534/genetics.112.144758"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jakobsson</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. D.</span> <span class="cit-name-surname">Edge</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">N. A.</span> <span class="cit-name-surname">Rosenberg</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The relationship between FST and the frequency of the most frequent allele.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">193</span>: <span class="cit-fpage">515</span>–<span class="cit-lpage">528</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.112.144758</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.144758%26rft_id%253Dinfo%253Apmid%252F23172852%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkzLzIvNTE1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMS8zODEzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3813.20" data-doi="10.1016/j.tree.2004.08.004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Joseph</span>,  <span class="cit-name-given-names">S.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Kirkpatrick</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Haploid selection in animals.</span> <abbr class="cit-jnl-abbrev">Trends Ecol. Evol.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">592</span>–<span class="cit-lpage">597</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.tree.2004.08.004</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BEcol.%2BEvol.%26rft.volume%253D19%26rft.spage%253D592%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tree.2004.08.004%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tree.2004.08.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000225033100008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3813.21" data-doi="10.1093/jhered/esx080"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kasimatis</span>,  <span class="cit-name-given-names">K. R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. C.</span> <span class="cit-name-surname">Nelson</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. C.</span> <span class="cit-name-surname">Phillips</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Genomic signatures of sexual conflict.</span> <abbr class="cit-jnl-abbrev">J. Hered.</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">780</span>–<span class="cit-lpage">790</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/jhered/esx080</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BHered.%26rft.volume%253D108%26rft.spage%253D780%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjhered%252Fesx080%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/jhered/esx080&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3813.22"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kidwell</span>,  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. T.</span> <span class="cit-name-surname">Clegg</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F. M.</span> <span class="cit-name-surname">Stewart</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Prout</span></span></li></ol><cite>, <span class="cit-pub-date">1977</span> <span class="cit-article-title">Regions of stable equilibria for models of differential selection in the two sexes under random mating.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">85</span>: <span class="cit-fpage">171</span>–<span class="cit-lpage">183</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DKidwell%26rft.auinit1%253DJ.%2BF.%26rft.volume%253D85%26rft.issue%253D1%26rft.spage%253D171%26rft.epage%253D183%26rft.atitle%253DREGIONS%2BOF%2BSTABLE%2BEQUILIBRIA%2BFOR%2BMODELS%2BOF%2BDIFFERENTIAL%2BSELECTION%2BIN%2BTHE%2BTWO%2BSEXES%2BUNDER%2BRANDOM%2BMATING%26rft_id%253Dinfo%253Apmid%252F838269%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiODUvMS8xNzEiO3M6NDoiYXRvbSI7czoxOToiL2dnZy85LzExLzM4MTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3813.23" data-doi="10.1038/hdy.2009.104"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kozielska</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F. J.</span> <span class="cit-name-surname">Weissing</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L. W.</span> <span class="cit-name-surname">Beukeboom</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Pen</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Segregation distortion and the evolution of sex-determining mechanisms.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">104</span>: <span class="cit-fpage">100</span>–<span class="cit-lpage">112</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/hdy.2009.104</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.stitle%253DHeredity%26rft.aulast%253DKozielska%26rft.auinit1%253DM.%26rft.volume%253D104%26rft.issue%253D1%26rft.spage%253D100%26rft.epage%253D112%26rft.atitle%253DSegregation%2Bdistortion%2Band%2Bthe%2Bevolution%2Bof%2Bsex-determining%2Bmechanisms.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2009.104%26rft_id%253Dinfo%253Apmid%252F19672280%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2009.104&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19672280&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3813.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000272794600012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-book" id="cit-9.11.3813.24"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lewontin</span>,  <span class="cit-name-given-names">R. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1974</span> <span class="cit-source">The genetic basis of evolutionary change</span>, <span class="cit-publ-name">Columbia University Press</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3813.25" data-doi="10.1093/gbe/evw090"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lucotte</span>,  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Laurent</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Heyer</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Ségurel</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Toupance</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Detection of allelic frequency differences between the sexes in humans: a signature of sexually antagonistic selection.</span> <abbr class="cit-jnl-abbrev">Genome Biol. Evol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">1489</span>–<span class="cit-lpage">1500</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/gbe/evw090</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevw090%26rft_id%253Dinfo%253Apmid%252F27189992%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/gbe/evw090&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27189992&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3813.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3813.26"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mank</span>,  <span class="cit-name-given-names">J. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span><span class="cit-comment">a</span> <span class="cit-article-title">Population genetics of sexual conflict in the genomic era.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">10</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BGenet.%26rft.volume%253D7%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3813.27" data-doi="10.1038/s41559-016-0006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mank</span>,  <span class="cit-name-given-names">J. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span><span class="cit-comment">b</span> <span class="cit-article-title">The transcriptional architecture of phenotypic dimorphism.</span> <abbr class="cit-jnl-abbrev">Nat. Ecol. Evol.</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">6</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/s41559-016-0006</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BEcol.%2BEvol.%26rft.volume%253D1%26rft.spage%253D6%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41559-016-0006%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41559-016-0006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3813.28" data-doi="10.1111/j.1558-5646.2009.00764.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Patten</span>,  <span class="cit-name-given-names">M. M.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Haig</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Maintenance or loss of genetic variation under sexual and parental antagonism at a sex-linked locus.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">63</span>: <span class="cit-fpage">2888</span>–<span class="cit-lpage">2895</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1558-5646.2009.00764.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.stitle%253DEvolution%26rft.aulast%253DPatten%26rft.auinit1%253DM.%2BM.%26rft.volume%253D63%26rft.issue%253D11%26rft.spage%253D2888%26rft.epage%253D2895%26rft.atitle%253DMaintenance%2Bor%2Bloss%2Bof%2Bgenetic%2Bvariation%2Bunder%2Bsexual%2Band%2Bparental%2Bantagonism%2Bat%2Ba%2Bsex-linked%2Blocus.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1558-5646.2009.00764.x%26rft_id%253Dinfo%253Apmid%252F19573084%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1558-5646.2009.00764.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19573084&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3813.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000271031900010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-book" id="cit-9.11.3813.29"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">R Core Team</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-source">R: A Language and Environment for Statistical Computing</span>, <span class="cit-publ-name">R Foundation for Statistical Computing</span>, <span class="cit-publ-loc">Vienna, Austria</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3813.30" data-doi="10.2307/2408385"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rice</span>,  <span class="cit-name-given-names">W. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1984</span> <span class="cit-article-title">Sex chromosomes and the evolution of sexual dimorphism.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">735</span>–<span class="cit-lpage">742</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1558-5646.1984.tb00346.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.volume%253D38%26rft.spage%253D735%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2408385%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2408385&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1984TJ95300003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3813.31" data-doi="10.2307/2408899"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rice</span>,  <span class="cit-name-given-names">W. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span> <span class="cit-article-title">The accumulation of sexually antagonistic genes as a selective agent promoting the evolution of reduced recombination between primitive sex chromosomes.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">911</span>–<span class="cit-lpage">914</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1558-5646.1987.tb05864.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.volume%253D41%26rft.spage%253D911%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2408899%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2408899&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1987J007200019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3813.32" data-doi="10.1007/s002650050357"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rice</span>,  <span class="cit-name-given-names">W. R.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Holland</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">The enemies within: intergenomic conflict, interlocus contest evolution (ICE), and the intraspecific Red Queen.</span> <abbr class="cit-jnl-abbrev">Behav. Ecol. Sociobiol.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">10</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBehav.%2BEcol.%2BSociobiol.%26rft.volume%253D41%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs002650050357%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s002650050357&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1997XL53900001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3813.33" data-doi="10.1111/evo.13625"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Singh</span>,  <span class="cit-name-given-names">A.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Punzalan</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">The strength of sex-specific selection in the wild.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">72</span>: <span class="cit-fpage">2818</span>–<span class="cit-lpage">2824</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/evo.13625</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.volume%253D72%26rft.spage%253D2818%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fevo.13625%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/evo.13625&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3813.34" data-doi="10.1017/S0016672300013926"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>,  <span class="cit-name-given-names">J. M.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Hoekstra</span></span></li></ol><cite>, <span class="cit-pub-date">1980</span> <span class="cit-article-title">Polymorphism in a varied environment: how robust are the models?</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">45</span>–<span class="cit-lpage">57</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1017/S0016672300013926</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DSmith%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D35%26rft.issue%253D1%26rft.spage%253D45%26rft.epage%253D57%26rft.atitle%253DPolymorphism%2Bin%2Ba%2Bvaried%2Benvironment%253A%2Bhow%2Brobust%2Bare%2Bthe%2Bmodels%253F%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672300013926%26rft_id%253Dinfo%253Apmid%252F7450498%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672300013926&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7450498&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3813.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1980JN14700004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3813.35" data-doi="10.3390/genes10040320"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tsai</span>,  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">Evans</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. E.</span> <span class="cit-name-surname">Noorai</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. N.</span> <span class="cit-name-surname">Starr-Moss</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">L. A.</span> <span class="cit-name-surname">Clark</span></span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">Novel Y Chromosome Retrocopies in Canids Revealed through a Genome-Wide Association Study for Sex.</span> <abbr class="cit-jnl-abbrev">Genes (Basel)</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">11</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3390/genes10040320</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2528Basel%2529%26rft.volume%253D10%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.3390%252Fgenes10040320%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3390/genes10040320&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3813.36" data-doi="10.2307/2408641"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Weir</span>,  <span class="cit-name-given-names">B. S.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. C.</span> <span class="cit-name-surname">Cockerham</span></span></li></ol><cite>, <span class="cit-pub-date">1984</span> <span class="cit-article-title">Estimating F-statistics for the analysis of population structure.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">1358</span>–<span class="cit-lpage">1370</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.volume%253D38%26rft.spage%253D1358%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2408641%26rft_id%253Dinfo%253Apmid%252F28563791%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2408641&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28563791&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3813.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1984TY40400017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-book" id="cit-9.11.3813.37"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Williams</span>,  <span class="cit-name-given-names">G. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1975</span> <span class="cit-source">Sex and evolution</span>, <span class="cit-publ-name">Princeton University Press</span>, <span class="cit-publ-loc">Princeton, NJ</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3813.38"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>,  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Fumagalli</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. R.</span> <span class="cit-name-surname">Cooney</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N. I.</span> <span class="cit-name-surname">Bloch</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F. G.</span> <span class="cit-name-surname">Vieira</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Male-biased gene expression resolves sexual conflict through the evolution of sex-specific genetic architecture.</span> <abbr class="cit-jnl-abbrev">Evol. Letters</abbr> <span class="cit-vol">215</span>: <span class="cit-fpage">403</span>–<span class="cit-lpage">410</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvol.%2BLetters%26rft.volume%253D215%26rft.spage%253D403%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3813.39" data-doi="10.1111/j.1469-1809.1949.tb02451.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>,  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1951</span> <span class="cit-article-title">The genetical structure of populations.</span> <abbr class="cit-jnl-abbrev">Ann. Eugen.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">323</span>–<span class="cit-lpage">354</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1469-1809.1949.tb02451.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BEugen.%26rft.volume%253D15%26rft.spage%253D323%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1469-1809.1949.tb02451.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1469-1809.1949.tb02451.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/9/11/3813.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/11/3801" title="Genetic Dissection Reveals the Role of Ash1 Domains in Counteracting Polycomb Repression" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/11/3825" title="Visualization and Quantification of Transposon Activity in Caenorhabditis elegans RNAi Pathway Mutants" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/9/11"><span>Volume 9 Issue 11, November 2019</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/9/11.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/9/11.cover-source.jpg?itok=DjqkLGk9" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 9 (11)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/11/3813.abstract" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-nfytH-TidkbMBVbDEL56qLIcDRbqn6GhL30fEZErEVE" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/9/11/3813" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/9/11/3813" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/9/11/3813" class="active" data-icon-position="" data-hide-link-title="0">Limits to Genomic Divergence Under Sexually Antagonistic Selection</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/494589" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-E2DErDpo2xZBQuaGMNq09SjI93p_qWDiJFW6Xh9hi4M" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/11/3813.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/9/11/3813" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-ovzl6kYwMrsONqC2by2lhBI6X-_YAUc7LGik8wGceHQ" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/9/11/3813" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/11/3813.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494589" id="node494589--2" data-pisa="ggg;9/11/3813" data-pisa-master="ggg;g3.119.400711" data-apath="/ggg/9/11/3813.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Limits to Genomic Divergence Under Sexually Antagonistic Selection</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Katja R. Kasimatis</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-9459-6866" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Peter L. Ralph</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0001-7271-342X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Patrick C. Phillips</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3813-3824;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400711 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494589--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Katja R. Kasimatis </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Ecology and Evolution and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Katja%2BR.%2BKasimatis%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kasimatis%20KR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKatja%2BR.%2BKasimatis%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:k.kasimatis@utoronto.ca" class="" data-icon-position="" data-hide-link-title="0">k.kasimatis@utoronto.ca</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Peter L. Ralph </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Ecology and Evolution and</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Mathematics, University of Oregon, Eugene, Oregon 97403</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Peter%2BL.%2BRalph%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ralph%20PL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APeter%2BL.%2BRalph%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-9459-6866" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Peter L. Ralph</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Patrick C. Phillips </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Ecology and Evolution and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Patrick%2BC.%2BPhillips%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Phillips%20PC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APatrick%2BC.%2BPhillips%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-7271-342X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Patrick C. Phillips</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/9/11/3813" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/9/11/3813&amp;title=Limits%20to%20Genomic%20Divergence%20Under%20Sexually%20Antagonistic%20Selection" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/9/11/3813&amp;title=Limits%20to%20Genomic%20Divergence%20Under%20Sexually%20Antagonistic%20Selection" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/9/11/3813&amp;title=Limits%20to%20Genomic%20Divergence%20Under%20Sexually%20Antagonistic%20Selection" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/9/11/3813&amp;text=Limits%20to%20Genomic%20Divergence%20Under%20Sexually%20Antagonistic%20Selection" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/9/11/3813&amp;title=Limits%20to%20Genomic%20Divergence%20Under%20Sexually%20Antagonistic%20Selection" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/9/11/3813&amp;t=Limits%20to%20Genomic%20Divergence%20Under%20Sexually%20Antagonistic%20Selection" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/9/11/3813&amp;title=Limits%20to%20Genomic%20Divergence%20Under%20Sexually%20Antagonistic%20Selection" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/9/11/3813&amp;title=Limits%20to%20Genomic%20Divergence%20Under%20Sexually%20Antagonistic%20Selection" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/494589/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494589" id="node494589--3" data-pisa="ggg;9/11/3813" data-pisa-master="ggg;g3.119.400711" data-apath="/ggg/9/11/3813.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Limits to Genomic Divergence Under Sexually Antagonistic Selection</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Katja R. Kasimatis</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-9459-6866" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Peter L. Ralph</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0001-7271-342X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Patrick C. Phillips</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3813-3824;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400711 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494589--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Katja R. Kasimatis </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Ecology and Evolution and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Katja%2BR.%2BKasimatis%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kasimatis%20KR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKatja%2BR.%2BKasimatis%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:k.kasimatis@utoronto.ca" class="" data-icon-position="" data-hide-link-title="0">k.kasimatis@utoronto.ca</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Peter L. Ralph </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Ecology and Evolution and</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Mathematics, University of Oregon, Eugene, Oregon 97403</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Peter%2BL.%2BRalph%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ralph%20PL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APeter%2BL.%2BRalph%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-9459-6866" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Peter L. Ralph</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Patrick C. Phillips </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Ecology and Evolution and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Patrick%2BC.%2BPhillips%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Phillips%20PC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APatrick%2BC.%2BPhillips%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-7271-342X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Patrick C. Phillips</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=3813&amp;rft.epage=3824&amp;rft.atitle=Limits%20to%20Genomic%20Divergence%20Under%20Sexually%20Antagonistic%20Selection&amp;rft.volume=9&amp;rft.issue=11&amp;rft.date=2019-11-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Kasimatis%2C+Katja+R.&amp;rft.au=Ralph%2C+Peter+L.&amp;rft.au=Phillips%2C+Patrick+C."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/494589/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/494589/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/494589/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/494589/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/494589/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/494589/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/494589/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/494589/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/494589/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/494589/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/494589/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/494589/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/9/11/3813" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Methods</a></li><li class="odd"><a href="#sec-6" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-15" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/9/11/3813.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/9/11/3813.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/9/11/3813.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__c-64NG7CB_fwZbyex-B49GkX1AvVESfu7M9yxTNDG_o__Uvi1liJMEKbHtHHv_1fo_FCLvimvbq9jMwPcCPR364Q__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
