<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/9/11/3715" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/9/11/3715.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/9/11/3715.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/9/11/3715.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/9/11/3715.atom" />
<meta name="HW.pisa" content="ggg;9/11/3715" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Three Quantitative Trait Loci Explain More than 60% of Variation for Chill Coma Recovery Time in a Natural Population of Drosophila ananassae" />
<meta name="DC.Identifier" content="10.1534/g3.119.400453" />
<meta name="DC.Date" content="2019-11-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2019 Koniger et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Ectothermic species such as insects are particularly vulnerable to climatic fluctuations. Nevertheless, many insects that evolved and diversified in the tropics have successfully colonized temperate regions all over the globe. To shed light on the genetic basis of cold tolerance in such species, we conducted a quantitative trait locus (QTL) mapping experiment for chill coma recovery time (CCRT) in Drosophila ananassae , a cosmopolitan species that has expanded its range from tropical to temperate regions. We created a mapping population of recombinant inbred advanced intercross lines (RIAILs) from two founder strains with diverging CCRT phenotypes. The RIAILs were phenotyped for their CCRT and, together with the founder strains, genotyped for polymorphic markers with double-digest restriction site-associated DNA (ddRAD) sequencing. Using a hierarchical mapping approach that combined standard interval mapping and a multiple-QTL model, we mapped three QTL which altogether explained 64% of the phenotypic variance. For two of the identified QTL, we found evidence of epistasis. To narrow down the list of cold tolerance candidate genes, we cross-referenced the QTL intervals with genes that we previously identified as differentially expressed in response to cold in D. ananassae , and with thermotolerance candidate genes of D. melanogaster . Among the 58 differentially expressed genes that were contained within the QTL, GF15058 showed a significant interaction of the CCRT phenotype and gene expression. Further, we identified the orthologs of four D. melanogaster thermotolerance candidate genes, [ MtnA ][1], [ klarsicht ][2], [ CG5246 ][3] ( D.ana / GF17132 ) and [ CG10383 ][4] ( D.ana / GF14829 ) as candidates for cold tolerance in D. ananassae .

 [1]: https://identifiers.org/bioentitylink/FB:FBgn0002868?doi=10.1534/g3.119.400453
 [2]: https://identifiers.org/bioentitylink/FB:FBgn0001316?doi=10.1534/g3.119.400453
 [3]: https://identifiers.org/bioentitylink/FB:FBgn0038484?doi=10.1534/g3.119.400453
 [4]: https://identifiers.org/bioentitylink/FB:FBgn0032699?doi=10.1534/g3.119.400453" />
<meta name="DC.Contributor" content="Annabella Königer" />
<meta name="DC.Contributor" content="Saad Arif" />
<meta name="DC.Contributor" content="Sonja Grath" />
<meta name="article:published_time" content="2019-11-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Three Quantitative Trait Loci Explain More than 60% of Variation for Chill Coma Recovery Time in a Natural Population of Drosophila ananassae" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Ectothermic species such as insects are particularly vulnerable to climatic fluctuations. Nevertheless, many insects that evolved and diversified in the tropics have successfully colonized temperate regions all over the globe. To shed light on the genetic basis of cold tolerance in such species, we conducted a quantitative trait locus (QTL) mapping experiment for chill coma recovery time (CCRT) in &lt;i&gt;Drosophila ananassae&lt;/i&gt;, a cosmopolitan species that has expanded its range from tropical to temperate regions. We created a mapping population of recombinant inbred advanced intercross lines (RIAILs) from two founder strains with diverging CCRT phenotypes. The RIAILs were phenotyped for their CCRT and, together with the founder strains, genotyped for polymorphic markers with double-digest restriction site-associated DNA (ddRAD) sequencing. Using a hierarchical mapping approach that combined standard interval mapping and a multiple-QTL model, we mapped three QTL which altogether explained 64% of the phenotypic variance. For two of the identified QTL, we found evidence of epistasis. To narrow down the list of cold tolerance candidate genes, we cross-referenced the QTL intervals with genes that we previously identified as differentially expressed in response to cold in &lt;i&gt;D. ananassae&lt;/i&gt;, and with thermotolerance candidate genes of &lt;i&gt;D. melanogaster&lt;/i&gt;. Among the 58 differentially expressed genes that were contained within the QTL, &lt;i&gt;GF15058&lt;/i&gt; showed a significant interaction of the CCRT phenotype and gene expression. Further, we identified the orthologs of four &lt;i&gt;D. melanogaster&lt;/i&gt; thermotolerance candidate genes, &lt;i&gt;MtnA&lt;/i&gt;, &lt;i&gt;klarsicht&lt;/i&gt;, &lt;i&gt;CG5246&lt;/i&gt; (&lt;i&gt;D.ana&lt;/i&gt;/&lt;i&gt;GF17132&lt;/i&gt;) and &lt;i&gt;CG10383&lt;/i&gt; (&lt;i&gt;D.ana&lt;/i&gt;/&lt;i&gt;GF14829&lt;/i&gt;) as candidates for cold tolerance in &lt;i&gt;D. ananassae&lt;/i&gt;.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2019/11/01" />
<meta name="citation_mjid" content="ggg;9/11/3715" />
<meta name="citation_id" content="9/11/3715" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/9/11/3715" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/9/11/3715.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/9/11/3715.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/9/11/3715.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.119.400453" />
<meta name="citation_pmid" content="31690597" />
<meta name="citation_volume" content="9" />
<meta name="citation_issue" content="11" />
<meta name="citation_num_pages" content="11" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="3715" />
<meta name="citation_lastpage" content="3725" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Annabella Königer" />
<meta name="citation_author_institution" content="Division of Evolutionary Biology, Ludwig-Maximilians-Universität (LMU) München, 82152 Planegg-Martinsried, Germany and" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-4554-9390" />
<meta name="citation_author" content="Saad Arif" />
<meta name="citation_author_institution" content="Centre for Functional Genomics, Department of Biological and Medical Sciences, Oxford Brookes University, Oxford, OX3 0BP, United Kingdom" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-0811-8604" />
<meta name="citation_author" content="Sonja Grath" />
<meta name="citation_author_institution" content="Division of Evolutionary Biology, Ludwig-Maximilians-Universität (LMU) München, 82152 Planegg-Martinsried, Germany and" />
<meta name="citation_author_email" content="grath@bio.lmu.de" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-3621-736X" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=D. Arends;citation_author=P. Prins;citation_author=R. C. Jansen;citation_author=K. W. Broman;citation_title=R/qtl: high-throughput multiple QTL mapping.;citation_volume=26;citation_year=2010;citation_pmid=20966004;citation_doi=10.1093/bioinformatics/btq565" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=H. Attrill;citation_author=K. Falls;citation_author=J. L. Goodman;citation_author=G. H. Millburn;citation_author=G. Antonazzo;citation_title=FlyBase: establishing a Gene Group resource for Drosophila melanogaster.;citation_volume=44;citation_year=2016;citation_pmid=26467478;citation_doi=10.1093/nar/gkv1046" />
<meta name="citation_reference" content="citation_journal_title=J. Stat. Softw.;citation_author=D. Bates;citation_author=M. Mächler;citation_author=B. Bolker;citation_author=S. Walker;citation_title=Fitting Linear Mixed-Effects Models Using lme4.;citation_pages=1-48;citation_volume=67;citation_year=2015;citation_doi=10.18637/jss.v067.i01" />
<meta name="citation_reference" content="citation_journal_title=Soc. Ser. B Methodol.;citation_author=Y. Benjamini;citation_author=Y. Hochberg;citation_title=Controlling the False Discovery Rate: A Practical and Powerful Approach to Multiple Testing. J. R. Stat.;citation_pages=289-300;citation_volume=57;citation_year=1995" />
<meta name="citation_reference" content="citation_journal_title=Biochem. Pharmacol.;citation_author=W. Bock;citation_title=The UDP-glycosyltransferase (UGT) superfamily expressed in humans, insects and plants: Animal-plant arms-race and co-evolution.;citation_pages=11-17;citation_volume=99;citation_year=2016;citation_doi=10.1016/j.bcp.2015.10.001" />
<meta name="citation_reference" content="citation_title=A Guide to QTL Mapping with R/qtl;citation_author=K. Broman;citation_author=S. Sen;citation_year=2009;citation_doi=10.1007/978-0-387-92125-9" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_journal_abbrev=Bioinformatics;citation_author=K. W. Broman;citation_author=H. Wu;citation_author=S. Sen;citation_author=G. A. Churchill;citation_title=R/qtl: QTL mapping in experimental crosses;citation_pages=889-890;citation_volume=19;citation_year=2003;citation_issue=7;citation_pmid=12724300;citation_doi=10.1093/bioinformatics/btg112" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=A. Catalán;citation_author=A. Glaser-Schmitt;citation_author=E. Argyridou;citation_author=P. Duchen;citation_author=J. Parsch;citation_title=An Indel Polymorphism in the MtnA 3′ Untranslated Region Is Associated with Gene Expression Variation and Local Adaptation in Drosophila melanogaster.;citation_pages=e1005987;citation_volume=12;citation_year=2016;citation_doi=10.1371/journal.pgen.1005987" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. M. Catchen;citation_author=A. Amores;citation_author=P. Hohenlohe;citation_author=W. Cresko;citation_author=J. H. Postlethwait;citation_title=Stacks: Building and Genotyping Loci De Novo From Short-Read Sequences. G3: Genes, Genomes;citation_pages=171-182;citation_volume=1;citation_year=2011;citation_doi=10.1534/g3.111.000240" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol. Resour.;citation_author=J. M. Catchen;citation_author=P. A. Hohenlohe;citation_author=L. Bernatchez;citation_author=W. C. Funk;citation_author=K. R. Andrews;citation_title=Unbroken: RADseq remains a powerful tool for understanding the genetics of adaptation in natural populations.;citation_volume=17;citation_year=2017;citation_doi=10.1111/1755-0998.12669" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=A. G. Clark;citation_author=M. B. Eisen;citation_author=D. R. Smith;citation_author=C. M. Bergman;citation_author=B. Oliver;citation_title=Evolution of genes and genomes on the Drosophila phylogeny.;citation_pages=203-218;citation_volume=450;citation_year=2007;citation_issue=7167;citation_pmid=17994087;citation_doi=10.1038/nature06341" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=D. J. Cutler;citation_author=J. D. Jensen;citation_title=To Pool, or Not to Pool?;citation_pages=41-43;citation_volume=186;citation_year=2010;citation_issue=1;citation_pmid=20855575;citation_doi=10.1534/genetics.110.121012" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=A. Das;citation_author=S. Mohanty;citation_author=W. Stephan;citation_title=Inferring the Population Structure and Demography of Drosophila ananassae From Multilocus Data;citation_pages=1975-1985;citation_volume=168;citation_year=2004;citation_issue=4;citation_pmid=15611168;citation_doi=10.1534/genetics.104.031567" />
<meta name="citation_reference" content="citation_journal_title=Prof. Anim. Sci.;citation_author=D. P. Dasmohapatra;citation_author=N. K. Tripathy;citation_author=C. C. Das;citation_title=Temperature-related chromosome polymorphism in Drosophila ananassae.;citation_pages=243-247;citation_volume=91;citation_year=1982;citation_doi=10.1007/BF03185015" />
<meta name="citation_reference" content="citation_journal_title=Trends in genetics :  TIG;citation_journal_abbrev=Trends Genet;citation_author=J. R. David;citation_author=P. Capy;citation_title=Genetic variation of Drosophila melanogaster natural populations.;citation_pages=106-111;citation_volume=4;citation_year=1988;citation_issue=4;citation_pmid=3149056;citation_doi=10.1016/0168-9525(88)90098-4" />
<meta name="citation_reference" content="citation_journal_title=J. Therm. Biol.;citation_author=J. R. David;citation_author=P. Gibert;citation_author=E. Pla;citation_author=G. Petavy;citation_author=D. Karan;citation_title=Cold stress tolerance in Drosophila: Analysis of chill coma recovery in D. Melanogaster.;citation_pages=291-299;citation_volume=23;citation_year=1998;citation_doi=10.1016/S0306-4565(98)00020-5" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. R. David;citation_author=F. Lemeunier;citation_author=L. Tsacas;citation_author=A. Yassin;citation_title=The Historical Discovery of the Nine Species in the Drosophila melanogaster Species Subgroup;citation_pages=1969-1973;citation_volume=177;citation_year=2007;citation_issue=4;citation_pmid=18073416;citation_doi=10.1534/genetics.104.84756" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=E. De Gregorio;citation_author=P. T. Spellman;citation_author=G. M. Rubin;citation_author=B. Lemaitre;citation_title=Genome-wide analysis of the Drosophila immune response by using oligonucleotide microarrays.;citation_volume=98;citation_year=2001;citation_pmid=11606746;citation_doi=10.1073/pnas.221458698" />
<meta name="citation_reference" content="citation_journal_title=JCB;citation_author=H. Elhanany-Tamir;citation_author=Y. V. Yu;citation_author=M. Shnayder;citation_author=A. Jain;citation_author=M. Welte;citation_title=Organelle positioning in muscles requires cooperation between two KASH proteins and microtubules.;citation_volume=198;citation_year=2012;citation_pmid=22927463;citation_doi=10.1083/jcb.201204102" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. Futschik;citation_author=C. Schlötterer;citation_title=The Next Generation of Molecular Markers From Massively Parallel Sequencing of Pooled DNA Samples.;citation_volume=186;citation_year=2010;citation_pmid=20457880;citation_doi=10.1534/genetics.110.114397" />
<meta name="citation_reference" content="Falconer, D. S., and T. F. C. Mackay, 1996 Introduction to Quantitative Genetics. Oliver &amp; Boyd, Edinburgh, London." />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Protoc;citation_author=D. W. Huang;citation_author=B. T. Sherman;citation_author=R. A. Lempicki;citation_title=Systematic and integrative analysis of large gene lists using DAVID bioinformatics resources.;citation_pages=44-57;citation_volume=4;citation_year=2009;citation_issue=1;citation_pmid=19131956;citation_doi=10.1038/nprot.2008.211" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. Kirkpatrick;citation_author=N. Barton;citation_title=Chromosome Inversions, Local Adaptation and Speciation.;citation_volume=173;citation_year=2006;citation_pmid=16204214;citation_doi=10.1534/genetics.105.047985" />
<meta name="citation_reference" content="citation_journal_title=Genes (Basel);citation_author=A. Königer;citation_author=S. Grath;citation_title=Transcriptome Analysis Reveals Candidate Genes for Cold Tolerance in Drosophila ananassae.;citation_pages=624;citation_volume=9;citation_year=2018;citation_doi=10.3390/genes9120624" />
<meta name="citation_reference" content="citation_title=Evolutionary Biology;citation_author=D. Lachaise;citation_author=M.-L. Cariou;citation_author=J. R. David;citation_author=F. Lemeunier;citation_author=L. Tsacas;citation_title=EVOLUTIONARY BIOLOGY;citation_pages=159-225;citation_volume=22;citation_year=1988;citation_doi=10.1007/978-1-4613-0931-4_4" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=B. Handsaker;citation_author=A. Wysoker;citation_author=T. Fennell;citation_author=J. Ruan;citation_title=The Sequence Alignment/Map format and SAMtools.;citation_volume=25;citation_year=2009;citation_pmid=19505943;citation_doi=10.1093/bioinformatics/btp352" />
<meta name="citation_reference" content="citation_journal_title=Sci. Rep.;citation_author=H. A. MacMillan;citation_author=J. M. Knee;citation_author=A. B. Dennis;citation_author=H. Udaka;citation_author=K. E. Marshall;citation_title=Cold acclimation wholly reorganizes the Drosophila melanogaster transcriptome and metabolome.;citation_pages=28999;citation_volume=6;citation_year=2016;citation_doi=10.1038/srep28999" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=A. Marchler-Bauer;citation_author=M. K. Derbyshire;citation_author=N. R. Gonzales;citation_author=S. Lu;citation_author=F. Chitsaz;citation_title=CDD: NCBI’s conserved domain database.;citation_volume=43;citation_year=2015;citation_pmid=25414356;citation_doi=10.1093/nar/gku1221" />
<meta name="citation_reference" content="citation_journal_title=Comptes Rendus Hebdomadaires des Seances de l'Academie des Sciences. Serie B: Sciences Physiques;citation_journal_abbrev=Comptes Rendus Hebdomadaires des Seances de l'Academie des Sciences. Serie B: Sciences Physiques;citation_author=B. Moreteau;citation_author=J. P. Morin;citation_author=P. Gibert;citation_author=G. Pétavy;citation_author=E. Pla;citation_title=Evolutionary changes of nonlinear reaction norms according to thermal adaptation: a comparison of two Drosophila species.;citation_pages=833-841;citation_volume=320;citation_year=1997;citation_issue=10;citation_pmid=9436536;citation_doi=10.1016/S0764-4469(97)85020-2" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_journal_abbrev=Heredity;citation_author=T. J. Morgan;citation_author=T. F. C. Mackay;citation_title=Quantitative trait loci for thermotolerance phenotypes in Drosophila melanogaster.;citation_pages=232-242;citation_volume=96;citation_year=2006;citation_issue=3;citation_pmid=16404413;citation_doi=10.1038/sj.hdy.6800786" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_author=J. P. Morin;citation_author=B. Moreteau;citation_author=G. Pétavy;citation_author=R. Parkash;citation_author=J. R. David;citation_title=Reaction norms of morphological traits in drosophila: adaptive shape changes in a stenotherm circumtropical species?;citation_pages=1140-1148;citation_volume=51;citation_year=1997;citation_doi=10.2307/2411043" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=F. M. Norry;citation_author=A. C. Scannapieco;citation_author=P. Sambucetti;citation_author=C. I. Bertoli;citation_author=V. Loeschcke;citation_title=QTL for the thermotolerance effect of heat hardening, knockdown resistance to heat and chill-coma recovery in an intercontinental set of recombinant inbred lines of Drosophila melanogaster.;citation_pages=4570-4581;citation_volume=17;citation_year=2008;citation_issue=20;citation_pmid=18986501;citation_doi=10.1111/j.1365-294X.2008.03945.x" />
<meta name="citation_reference" content="citation_journal_title=Mechanisms of ageing and development;citation_journal_abbrev=Mech Ageing Dev;citation_author=D. Paik;citation_author=Y. G. Jang;citation_author=Y. E. Lee;citation_author=Y. N. Lee;citation_author=R. Yamamoto;citation_title=Misexpression screen delineates novel genes controlling Drosophila lifespan.;citation_pages=234-245;citation_volume=133;citation_year=2012;citation_issue=5;citation_pmid=22366109;citation_doi=10.1016/j.mad.2012.02.001" />
<meta name="citation_reference" content="citation_journal_title=J. Insect Physiol.;citation_author=C. Plantamp;citation_author=K. Salort;citation_author=P. Gibert;citation_author=A. Dumet;citation_author=G. Mialdea;citation_title=All or nothing: Survival, reproduction and oxidative balance in Spotted Wing Drosophila (Drosophila suzukii) in response to cold.;citation_pages=28-36;citation_volume=89;citation_year=2016;citation_doi=10.1016/j.jinsphys.2016.03.009" />
<meta name="citation_reference" content="citation_journal_title=Methods in molecular biology (Clifton, N.J.);citation_journal_abbrev=Methods Mol Biol;citation_author=D. A. Pollard;citation_title=Design and construction of recombinant inbred lines.;citation_pages=31-39;citation_volume=871;citation_year=2012;citation_pmid=22565831;citation_doi=10.1007/978-1-61779-785-9_3" />
<meta name="citation_reference" content="R Core Team, 2018 R: a language and environment for statistical computing, Vienna, Austria. Available at http://www.R-project.org/" />
<meta name="citation_reference" content="citation_journal_title=Genes (Basel);citation_author=T. J. S. Ramnarine;citation_author=A. Glaser-Schmitt;citation_author=A. Catalán;citation_author=J. Parsch;citation_title=Population Genetic and Functional Analysis of a cis-Regulatory Polymorphism in the Drosophila melanogaster Metallothionein A gene.;citation_pages=147;citation_volume=10;citation_year=2019;citation_doi=10.3390/genes10020147" />
<meta name="citation_reference" content="citation_journal_title=Gene;citation_journal_abbrev=Gene;citation_author=J. Ross;citation_author=H. Jiang;citation_author=M. R. Kanost;citation_author=Y. Wang;citation_title=Serine proteases and their homologs in the Drosophila melanogaster genome: an initial analysis of sequence conservation and phylogenetic relationships.;citation_pages=117-131;citation_volume=304;citation_year=2003;citation_pmid=12568721;citation_doi=10.1016/S0378-1119(02)01187-3" />
<meta name="citation_reference" content="citation_journal_title=Int. J. Mol. Sci.;citation_author=B. Ruttkay-Nedecky;citation_author=L. Nejdl;citation_author=J. Gumulec;citation_author=O. Zitka;citation_author=M. Masarik;citation_title=The Role of Metallothionein in Oxidative Stress.;citation_pages=6044-6066;citation_volume=14;citation_year=2013;citation_pmid=23502468;citation_doi=10.3390/ijms14036044" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. D. Schug;citation_author=S. G. Smith;citation_author=A. Tozier-Pearce;citation_author=S. F. McEvey;citation_title=The Genetic Structure of Drosophila ananassae Populations From Asia, Australia and Samoa.;citation_volume=175;citation_year=2007;citation_pmid=17237518;citation_doi=10.1534/genetics.106.066613" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=F. J. Sedlazeck;citation_author=P. Rescheneder;citation_author=A. von Haeseler;citation_title=NextGenMap: fast and accurate read mapping in highly polymorphic genomes.;citation_volume=29;citation_year=2013;citation_pmid=23975764;citation_doi=10.1093/bioinformatics/btt468" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=S. Sen;citation_author=G. A. Churchill;citation_title=A Statistical Framework for Quantitative Trait Mapping;citation_pages=371-387;citation_volume=159;citation_year=2001;citation_issue=1;citation_pmid=11560912" />
<meta name="citation_reference" content="citation_journal_title=Insect molecular biology;citation_journal_abbrev=Insect Mol Biol;citation_author=B. J. Sinclair;citation_author=A. G. Gibbs;citation_author=S. P. Roberts;citation_title=Gene transcription during exposure to, and recovery from, cold and desiccation stress in Drosophila melanogaster.;citation_pages=435-443;citation_volume=16;citation_year=2007;citation_issue=4;citation_pmid=17506850;citation_doi=10.1111/j.1365-2583.2007.00739.x" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=P. Singh;citation_author=B. N. Singh;citation_title=Population genetics of Drosophila ananassae: genetic differentiation among Indian natural populations at the level of inversion polymorphism.;citation_pages=191-199;citation_volume=89;citation_year=2007;citation_issue=4;citation_pmid=18208624;citation_doi=10.1017/S0016672307008890" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_journal_abbrev=Heredity;citation_author=W. Stephan;citation_author=H. Li;citation_title=The recent demographic and adaptive history of Drosophila melanogaster.;citation_pages=65-68;citation_volume=98;citation_year=2007;citation_issue=2;citation_pmid=17006533;citation_doi=10.1038/sj.hdy.6800901" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=W. Stephan;citation_author=V. S. Rodriguez;citation_author=B. Zhou;citation_author=J. Parsch;citation_title=Molecular evolution of the metallothionein gene Mtn in the melanogaster species group: results from Drosophila ananassae.;citation_pages=135-143;citation_volume=138;citation_year=1994;citation_issue=1;citation_pmid=8001781" />
<meta name="citation_reference" content="citation_journal_title=Journal of Insect Physiology;citation_journal_abbrev=Journal of Insect Physiology;citation_author=M. Telonis-Scott;citation_author=R. Hallas;citation_author=S. W. McKechnie;citation_author=C. W. Wee;citation_author=A. A. Hoffmann;citation_title=Selection for cold resistance alters gene transcript levels in Drosophila melanogaster.;citation_pages=549-555;citation_volume=55;citation_year=2009;citation_issue=6;citation_pmid=19232407;citation_doi=10.1016/j.jinsphys.2009.01.010" />
<meta name="citation_reference" content="citation_title=Drosophila ananassae: genetical and biological aspects;citation_year=1993" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_author=N. Tucić;citation_title=Genetic Capacity for Adaptation to Cold Resistance at Different Developmental Stages of Drosophila melanogaster.;citation_pages=350-358;citation_volume=33;citation_year=1979;citation_doi=10.2307/2407625" />
<meta name="citation_reference" content="citation_journal_title=Journal of Insect Physiology;citation_journal_abbrev=Journal of Insect Physiology;citation_author=H. Udaka;citation_author=C. Ueda;citation_author=S. G. Goto;citation_title=Survival rate and expression of Heat-shock protein 70 and Frost genes after temperature stress in Drosophila melanogaster lines that are selected for recovery time from temperature coma.;citation_pages=1889-1894;citation_volume=56;citation_year=2010;citation_issue=12;citation_pmid=20713057;citation_doi=10.1016/j.jinsphys.2010.08.008" />
<meta name="citation_reference" content="citation_journal_title=J. Evol. Biol.;citation_author=C. J. Vermeulen;citation_author=P. Sørensen;citation_author=K. K. Gagalova;citation_author=V. Loeschcke;citation_title=Transcriptomic analysis of inbreeding depression in cold-sensitive Drosophila melanogaster shows upregulation of the immune response.;citation_pages=1890-1902;citation_volume=26;citation_year=2013;citation_pmid=23944235;citation_doi=10.1111/jeb.12183" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=K. von Heckel;citation_author=W. Stephan;citation_author=S. Hutter;citation_title=Canalization of gene expression is a major signature of regulatory cold adaptation in temperate Drosophila melanogaster.;citation_pages=574;citation_volume=17;citation_year=2016;citation_doi=10.1186/s12864-016-2866-0" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_author=C. Williams;citation_author=M. Watanabe;citation_author=M. Guarracino;citation_author=M. Ferraro;citation_author=A. Edison;citation_title=Cold adaptation shapes the robustness of metabolic networks in Drosophila melanogaster.;citation_pages=3505-3523;citation_volume=68;citation_year=2014;citation_pmid=25308124;citation_doi=10.1111/evo.12541" />
<meta name="citation_reference" content="citation_journal_title=Journal of Experimental Biology;citation_journal_abbrev=J. Exp. Biol.;citation_author=J. Zhang;citation_author=K. E. Marshall;citation_author=J. T. Westwood;citation_author=M. S. Clark;citation_author=B. J. Sinclair;citation_title=Divergent transcriptomic responses to repeated and single cold exposures in Drosophila melanogaster;citation_pages=4021-4029;citation_volume=214;citation_year=2011;citation_issue=23;citation_pmid=22071194;citation_doi=10.1242/jeb.059535" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Three Quantitative Trait Loci Explain More than 60% of Variation for Chill Coma Recovery Time in a Natural Population of Drosophila ananassae" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11.cover-source.jpg" />
<meta name="twitter:description" content="Ectothermic species such as insects are particularly vulnerable to climatic fluctuations. Nevertheless, many insects that evolved and diversified in the tropics have successfully colonized temperate regions all over the globe. To shed light on the genetic basis of cold tolerance in such species, we conducted a quantitative trait locus (QTL) mapping experiment for chill coma recovery time (CCRT) in Drosophila ananassae , a cosmopolitan species that has expanded its range from tropical to temperate regions. We created a mapping population of recombinant inbred advanced intercross lines (RIAILs) from two founder strains with diverging CCRT phenotypes. The RIAILs were phenotyped for their CCRT and, together with the founder strains, genotyped for polymorphic markers with double-digest restriction site-associated DNA (ddRAD) sequencing. Using a hierarchical mapping approach that combined standard interval mapping and a multiple-QTL model, we mapped three QTL which altogether explained 64% of the phenotypic variance. For two of the identified QTL, we found evidence of epistasis. To narrow down the list of cold tolerance candidate genes, we cross-referenced the QTL intervals with genes that we previously identified as differentially expressed in response to cold in D. ananassae , and with thermotolerance candidate genes of D. melanogaster . Among the 58 differentially expressed genes that were contained within the QTL, GF15058 showed a significant interaction of the CCRT phenotype and gene expression. Further, we identified the orthologs of four D. melanogaster thermotolerance candidate genes, [ MtnA ][1], [ klarsicht ][2], [ CG5246 ][3] ( D.ana / GF17132 ) and [ CG10383 ][4] ( D.ana / GF14829 ) as candidates for cold tolerance in D. ananassae .

 [1]: https://identifiers.org/bioentitylink/FB:FBgn0002868?doi=10.1534/g3.119.400453
 [2]: https://identifiers.org/bioentitylink/FB:FBgn0001316?doi=10.1534/g3.119.400453
 [3]: https://identifiers.org/bioentitylink/FB:FBgn0038484?doi=10.1534/g3.119.400453
 [4]: https://identifiers.org/bioentitylink/FB:FBgn0032699?doi=10.1534/g3.119.400453" />
<meta name="og-title" property="og:title" content="Three Quantitative Trait Loci Explain More than 60% of Variation for Chill Coma Recovery Time in a Natural Population of Drosophila ananassae" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/9/11/3715" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Ectothermic species such as insects are particularly vulnerable to climatic fluctuations. Nevertheless, many insects that evolved and diversified in the tropics have successfully colonized temperate regions all over the globe. To shed light on the genetic basis of cold tolerance in such species, we conducted a quantitative trait locus (QTL) mapping experiment for chill coma recovery time (CCRT) in Drosophila ananassae , a cosmopolitan species that has expanded its range from tropical to temperate regions. We created a mapping population of recombinant inbred advanced intercross lines (RIAILs) from two founder strains with diverging CCRT phenotypes. The RIAILs were phenotyped for their CCRT and, together with the founder strains, genotyped for polymorphic markers with double-digest restriction site-associated DNA (ddRAD) sequencing. Using a hierarchical mapping approach that combined standard interval mapping and a multiple-QTL model, we mapped three QTL which altogether explained 64% of the phenotypic variance. For two of the identified QTL, we found evidence of epistasis. To narrow down the list of cold tolerance candidate genes, we cross-referenced the QTL intervals with genes that we previously identified as differentially expressed in response to cold in D. ananassae , and with thermotolerance candidate genes of D. melanogaster . Among the 58 differentially expressed genes that were contained within the QTL, GF15058 showed a significant interaction of the CCRT phenotype and gene expression. Further, we identified the orthologs of four D. melanogaster thermotolerance candidate genes, [ MtnA ][1], [ klarsicht ][2], [ CG5246 ][3] ( D.ana / GF17132 ) and [ CG10383 ][4] ( D.ana / GF14829 ) as candidates for cold tolerance in D. ananassae .

 [1]: https://identifiers.org/bioentitylink/FB:FBgn0002868?doi=10.1534/g3.119.400453
 [2]: https://identifiers.org/bioentitylink/FB:FBgn0001316?doi=10.1534/g3.119.400453
 [3]: https://identifiers.org/bioentitylink/FB:FBgn0038484?doi=10.1534/g3.119.400453
 [4]: https://identifiers.org/bioentitylink/FB:FBgn0032699?doi=10.1534/g3.119.400453" />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11.cover-source.jpg" />
    <title>Three Quantitative Trait Loci Explain More than 60% of Variation for Chill Coma Recovery Time in a Natural Population of Drosophila ananassae | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"CXNVnCKonf7pDiC9DQ9YpY0doD7aqn8EhubrJdVJ114","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"494618","apath":"\/ggg\/9\/11\/3715.atom","pisa":"ggg;9\/11\/3715","ac":{"\/ggg\/9\/11\/3715.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/9\/11\/3715.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;9\/11\/3715"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;9\/11\/3715"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/9\/11\/3715.figures-only":true,"\/content\/9\/11\/3715":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022UefE0o+ZYpAn62uxA4n8jkvnNXnxu0ZatWwqCMPGTQof+P3XjZ5ZwvlI0j+YNI1ZwZ0DVn7bNiAuS6diDUt3DDrOxieS8fjYAB58XZUESn2g\\\\\\\/PJvVT4\\\\\\\/CLpndllEQoADjrFXgPSXJ0Motzynn0W+P\\\\\\\/jlCxg2+TesF7dq3NyCWs8Vs3ierTlKjpf\\\\\\\/6MH5Gcxyu0fkcfr7\\\\\\\/kEj9fIAsrSQvGhCeXETp\\\\\\\/LLBQbjCRVpM4UG+3hT5\\\\\\\/WzAKI0KhmEAbizG1eykHGRhQeJ24Y\\\\\\\/rRLtIvRq3O40n80GQFbO2evKljuo67JuNOlYcgGJUG3j4BGElZZF6NOhxJDu\\\\\\\/ESi8Lah55lKOeKU5mlJQtPe1cm2ytuLBdtZlSx9EdJ8pdgVnXOSagOa\\\\\\\/QTT\\\\\\\/KotHBSpgaYRm4CpdzVXxGiBQ3RQc7vWzzETEmDO8d\\\\\\\/AGo5G7liwqjjlwqr3ZDBLEUY9dpzQD2P\\\\\\\/L2i6v\\\\\\\/rVBXku+hqn0MYSp1N6QKYEsSMYKM81ZXGZG+neX+QbPUu58RnQGFfNnn3nIwCDR9eOCFRXo798rHaQuOhInRaF2jKEIjs6h1V2O5EJalWdGUffzF\\\\\\\/mRxDSGRdjrYo+YcNZC6ylor1tKjGGSjjZCxHEExVvXmcHquBpe+uBujjBMYux6UuyF9RlEmHNuGCiNDC9X04RdZj4sC+Pkv6jUW9OWqPBeLchC6KRdAxyY3\\\\\\\/+zrHzmUmmLO8Ot2ojTETbt5lTN7cGZ\\\\\\\/hddh4CgJTJuYOyB940RMVsJdu1uM7vl14si9oQboEGG9HQ6Yo3BVO+4ia\\\\\\\/r1wf1GjQh2QLBu35gLAbLrMChezDz5OvFq1inLfJBfYsZGqUJuxpMZpIHX+JEQBYEJ\\\\\\\/wUZ1En0AjnvOs9UA73cznNWcwkJJJ0RuL\\\\\\\/MtCtjd6jkpvManPPO94j8xHnUKLX9PaGTSOzUAEJPyYLtAK5EOyagZg9iIjlqGJ4DvKuqWJWfTIt72ByGSvGDWhrQLLdhPBDquLXH8jDu6tip7NttYgsudQTyumTt+vMzP8o3Wx7rjxRb0K7mPKWtBPIXLHMyGhiFemgdwK\\\\\\\/KINoE52m9IXrKD6GOdKdTKUjPtk5enLBA8GkJiVn0TyrdjO02JkT\\\\\\\/5mLViZqOqR\\\\\\\/kKZX1O2i4vyOJWO72duj8tzRDCZv2Q1iypFCunIQJSJXAiF75yoEat2YOKOxQktzbMtRPzcveqyWOB\\\\\\\/TYj2lU4b77oPdW2GMh+E1214ogmuAkum9sHtHzkoSEGoJ1A+vs\\\\\\\/dEhA+zo85Zp2poY7JJ8xI\\\u0022,\\\u0022iv\\\u0022:\\\u0022OsoxHldB14JrVA7ZJVlt1w==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00224520f26ef750378a715bdec0a73449346f57b8976225308d6b045fd477035660\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022NGh7FuT3\\\\\\\/IyjKZ18fKlVMM9\\\\\\\/JHpBODMOtYf5m+pm79cDxJRjmRBpr06j5sY7pgYgqoI1XOojtowW4t+9tIHDR9dtW9C\\\\\\\/Gw4TCFQilRelxK\\\\\\\/+jopiR+A7ni6skwOsOiv8qH3pLgzIonKewfBDMBy1nMMy7oXLrilQE72EiobneVMjlTbsAJ5BlN0KWucbfdCdrPSMW0OMDVcRGjyCqbqWH+DnP70FNxx95MYNJuZ\\\\\\\/CNufGGLBlir8ClEggKtnEVDVHAQTHGkAKG8dsrNdwr61c4BxmnSTMDNJtChR+8C70Yn6ORhYKbf5XWbs\\\\\\\/tbffsTyOoMYVzxTCguvQ9dki76fktntXNENIj8hAi69SCiRPWABYNCVNF2pIviki3bWM2rs5TNe6alwxdGF9J6yvmonSqXu9Z0LAVRN7+O77QSWBXPZ4B76yByOh2In51lJJGY5aoboaO\\\\\\\/7LQ6YRoE56seL1nYAM8H0SL9RN6KvriRxT4i0gFfSYEF7tNS91+nEfcjMoXpc8+CaxWwCXz8MopScUln+ugYpBvr3U+ktCBsQ8UKTDht8Mz0vFtkwK\\\\\\\/XZkhVvT6cH0FY6JcarkVI3+b+CNOCfjZnMq1pi7GwnRN3IcQ\\\\\\\/Y64rEtUryUJw3YOrxbJJewqyqKyGF5YEZmw6CYLpZjRW5IBPWnN3uF2P8nV0G3Evrg7AckgaeRTthgma\\\\\\\/ronLV8Xe2w827i85tAsXe2UKbi\\\\\\\/vNoPvTDJ4WsimaId3iIL0MvImwgwH\\\\\\\/fmQly2j6RXEBQTWk\\\\\\\/tjRtAMFYHnD00u67+VV+Ppqq9bdfx+dfJpe0HgunnwfQ2xtekTFwIMEn\\\\\\\/+Sdsf+OY5PoG\\\\\\\/olOMkUXevh+4wbZLcWfwy\\\\\\\/lisRdxpg\\\\\\\/rGxmnlG+KZWOKve0cS+O9mUSgOq\\\\\\\/wBvxGT0OWu7CjTbXI7UFgZiZ3bR0lGqmjNtk+\\\\\\\/yU9Yn9RU+qHPJ8AkoxbsIoarbw\\\\\\\/KT1uB0FGrcL3uW1an3klumhJISPOZ0px0fgF9ilH\\\\\\\/c+vf27bPousrhy1u4BL1DvEO6Hifa59nDZPsKrFu7XOnDpTCvLr+iXxES1\\\\\\\/x8g8pjz6PAo69nNd8yIwdpW4FdEoVFT1wXnYZRlgPwABXQlbjrGoSLd90tphZt7l12bGb4QephU5DmBew+nKbj15mHDabxcsxE2B2eZZrarMOtO89s1CLwKlG1XWaWcmBgCyfP7Rds2ATvdW\\\\\\\/fmmh1V3CG6EdBH\\\\\\\/3v5qHlQR\\\\\\\/hSf2cddRJt5eLeddTlhBZWYlZrLzERmgvCCFlw+zIR+\\\\\\\/6\\\\\\\/Edd2iHYWR3evFAodhI2XxN+ajqvx7RO2ONFR\\\\\\\/0eAPniU\\\\\\\/Y9zioysdwQcZluRNqFKLsPF7ONCpjrM\\\\\\\/EZBsOt3iQPuZpxBMWEo4nJYCDxWRwQGuq6wjxauTSgjWfVbICfT5tJI3usZkwxU0TlUcsQYXgyMiaqdzZUWA+Zu1tAKIaxpiQMeLNhFDZbL+gtEt3AbsqEj8l2yoWGzwM0w7iPFFAP8eQKX8xxP7eGxfc7hta+wMEUWKvtJ\\\\\\\/DMl8qAeYYWjIbHs9P5hcOBikMyohPwlkC039XTabkuqsZBoAAW4x7Ja\\\\\\\/DkfH6dh8Bf\\\\\\\/BUhCV8mydEx+beNY5+NzaB4N5cAelC5EYA8alw\\\\\\\/SE56L114ChAeDuQUinB9yz3nMzjv4ZN+go4pyHCpEuVMi0AOZ4WUwY1xipG5ftWm881lAGjX1LuYtwZUuQVe+6pEpRqGr0m25ZMvaQJEzDhFOLag2rhq9xBXBSWj5c6RSr6eFNimdDSnt23YNxzIY\\\\\\\/yeFTHZ5\\\\\\\/7QGuof403HP1vXfwv0JFyd6hESHrO7um6y\\\\\\\/2LcrwDqMJfB7H1pzfm+RuZ4xycSl3y\\\\\\\/1XespwfcKoraQN5AFNyvdBS6NL5VKkB2biIXa3+z\\\\\\\/WIvWI2xil0h\\\\\\\/JjkD0M1+ZP26Ja0Td+sM5Q+nWnNVPwOpLZ0QOJnB1WjaDtnzAZkPLURL8+GdHtCzX6HFtgbAeuRfPoVSroIOB1mA4cZjGXN6j9dJ4eg08yKTewZtRH90VN4Cqcv08no2lQqRc7RpwcBEPravA9stuoGNmjAJJ4T4gkD3Rfa8xnQYfBKzo9vNxR22LaKvTnUYVdmNQQJvTMSW541S8xcFxizbEJO7EYIi487ZaWpIfnUlfqNqQc7MQndGqAx8UFWePloil5qLQStfWYCutAQzI1t0TQJ\\\\\\\/Z6Y3qUCZBJQc20m\\\\\\\/O+8CKZKu1gPVkwImzHlE9JrYv4fO9ffTEpTcsWljRMRVDzUaC5eIL7nqzEEjnleQsL05F+Zlh67QrnY\\\\\\\/CyTYQZf2u52hSJgeO3NaCEoMQnTA9U+0u3HjDY7mnqQXdUcp\\\\\\\/9IPF5Ra3\\\\\\\/agj7eBRxyl7UoabdtkPo9ua77YVOH6y6Bj6agyEu0RmvojWFni7WU2ClB2l3sSJB7CbirwMHCpZMK7q6M0WVSGNSLEm8MuUOe+8J4UBpfQ2YVJLXgaSTxQ7nGoKE74tZdD\\\\\\\/s7RQe+BEMat+Z8AERwIsKb1RDyJOrdeiQTZteOu\\\\\\\/OMFaInmgwoLl02Gr5TmZhHMvrbrA5H55RtKasb8Yq0wwyMGIavlbfMU0IT5M65KFV6itxgygJOYnTdIKQCLBjpqdKYl7I1h3aqhcr1HtlR8KWlpYAfzf\\\\\\\/5K2yLamb4rD7cOVS\\\\\\\/KrYhncU6gYxQt3hh8HaXsvkT5T8+8Ob0Slg6e4aH5O+t2Y1xJ09f4j9IsvC2an1XXNvfQ8lv\\\\\\\/0siwzhSLse\\\\\\\/PoUVcmAU7rI8fY0mPkuYiPnARV\\\\\\\/8tzw9YrTvHQLLMD+yb1fVPOKYAOHOkdHLr1oF+ZP9rKuUl6rlibvcgsuOATlCO48B4CxsOUzrSAOznptV197DtCbag4TtOaAivJ7nhWwGLWR5pZEXaFUN7CMaFk2DkpxfEVlynaqlKA+vpCMGfW019usZtnOsMVLF1nITTuMQEMWb6xSSX4rZIATjP1l8zqGb1SdCnZ\\\\\\\/f1RWAMsXwYuAjD7awZhp3EMi6kkgJa2gyWskT9YL9lTQW+hq0p20L0y3rHaKE3hspMopAFa7O6nYLw2GOz08opvsjc9yhpOrg0CQ7ZpgLyaa8aKnGPOXVepEJgKmX8TZpYq8vgn6zwtMQbHOYh8fw3BAkXP1S6on\\\\\\\/jtoElb\\\\\\\/hDoEY6TKFcvZe5pVae+lFD8ETUesqbqO4RxGgH1JlBmFuUAJGPZQLRdUzCOEFVV9moTzjyhCNqVFNeozo4fopwJqjpwxigabM1yio7cZdi9i5tfAPhe3nKom3YOYmeeBfX4hgGXTM2yQArsE5EFSxAU2Biu\\\\\\\/k684fVfoTXySibtUDkJqVr4JBAq4PlyTmbv1F4abv4DZ0Z5KLwsRO9whEkdqQxds+IQtiyVSF7abeq8t5+oRCaoMLAEVfMwNq1Dh66inLGvORkCBdo+Ib2KqeOG5UjTuyMEB7gYva0pfpYHeTqqJjBHH+JfPkteDvfr5MZ6w9yG4tkDtjUYjarCsV429\\\\\\\/qpNa9KRBzDChxLiZEWQiKPJBKxLAI0RVU3YeCuZ8UmEzfyco4CYNDsNF6MBaxgz9Vt3Jxn1NWYb0xUSMI03IWxd\\\\\\\/3uZsQBTuha4rxoe\\\\\\\/UuHQ6GptFhlXg3\\\\\\\/0KiavxPs47C3DmH\\\\\\\/A14ypQlU7i6LxX\\\\\\\/cpcF88Mtcgm+K1QCbgivN52aUxGAN49vbg6IyHXPOPwIri2TBF\\\\\\\/ikgssyWdFzyf462vYtIuQujBzAQuk4w8YVUEeMGhQOqfYF7WW3NZwDy0Fxdc4xU3zLyARY89I1t3PG8qkjoLSbiHYTkyPubjEL8B4zpf\\\\\\\/NvwMxa8noeIZvwzBLncNraebnFOhM7tpVhFzCwonopuMVT39OWxDsN3J+uwmuYWkwCDb2hiFDEgzUiyGBFkIJBIbUJI1SpCS0AgQXkRS2GCxysXy93pwV2rzNcMEZvH\\\\\\\/KLWPknqgbf9XNea2IWw\\\\\\\/hwk68ERYaIF+z23zzA+55OpBx7toSiiMYGLgAoQz6xHznLAu35B6q0VOsKhVLjzl\\\\\\\/CxzyCsaURKXMv8nOKwEKQowyG4PqfP+agKI8y\\\\\\\/hp6ToP08fc7Pr+7EzAwqQE\\\\\\\/OxvNJibtoP2cZCaQOIXweH4ZYj3h\\\\\\\/KUszIlibXe+KgUpbCMDgAL3QWay4M4cnHVUO6nNNyL\\\\\\\/daplDXXNmxjQ4qMPqFRkA4iyqZTixinn9aDg8gBQIoYXCZx+FGFZ1xeRoo\\\\\\\/kHDRDyWpceDQl6GHc4DA0Y3GKTZdUejFjrEZmjr0BjNRKsH5DT2VgFbcPj4uRfUaFucS1ScCA+\\\\\\\/55ajTZQoj2btk9bPGHM4+AjzBmYN5\\\\\\\/2dIks\\\\\\\/YSXZ4Br2ghEjPlZ9dYr0LM4+zSbEkpOIC6x\\\\\\\/mFxunRLS2U7uFwET7D5gcE0yh\\\\\\\/GKY6NkA8IM7UKoWxq5XL8kDA28mgFX3f974C8nWwqX2ORF5HeE5OrZJ3V0ckWWfxkeZ0zsv700Xf6K5g09VtuhHFAquRbtbZ3ND7X0YWGtdSLVkZlQJZ3PIuN8+7XsfecRyCNsuAa8E4rvKBmhvYns0ZoWN1fRxFUzL\\\\\\\/V\\\\\\\/iDXI\\\\\\\/bYNOAcwl\\\\\\\/svCnHXOZ5pP5cB0y1GYuTj0aLyzCEQIakCsbPGJtPaKTR4hN6bL6rVdUTZoaKQKlg8pCDqUv3LXSR0tAkyydAZGN4CPG9agL6sTM651DZe6w7T96JbYhDFwrGrPjlmy6XthxhGPgAfGUPKIUYmvu6ao3KiBkg5P\\\\\\\/Z9+VdNEfIlhP3PZ\\\\\\\/5j0HYwEuV\\\\\\\/IIVDi6o9HNfiqSqjOuiiHBgma2o3Gs9uf0zXP4S1h66Eh3M3pTH9qrxLgPvpJHaecJU1jYVZynGVONXJern7OyUCn9\\\\\\\/w6LPTw+EmuRttObah2mwZWv5iF+Zi4f8AvTKerDIT4u19MWiu1CaqZtP+weIQ8ERcTArzIgYCMdBr\\\\\\\/LaKgPHfH7coaY\\\\\\\/WxvKIsD5yM2Ri5cnFnD7JeZbvEZDBTkTIOO2h\\\\\\\/FOO6BSFLEQ0ZnMpZEsXD2jLwpwZVDRKlHI95XhWtb1zY32bGeJsN2nfBbjrw0DL0CyhMuPvW9CDggEdvRhusy5M8h47d8I2a4BkfTrP4vPS67Jklpq11e+1gUY2uoq0YtRvRDdWr7uC41IORI1K9Z6l5IvQO0sYRaGmWsdp3GiTgZkUHLeMzYA\\\\\\\/59m4Jy4oM4FjUMYVnOanB\\\\\\\/vK9QLPU9TFvl3M69IN+WD0rE4g\\\\\\\/Ae1Ou2vmaS5J0TSpsYOjiqU+FlEDR44voSo1xIWHqSqP9dOeWrSEmRpO3QgCSVQa7S5eoYZSXhF0ADYN\\\\\\\/KOYOFd9bQlS8q\\\\\\\/3wxd5E2a+qnAojAuFUaYB3ViHuQWeuhGLYlgdwTX1Of7CpODBjxPZR5WYRNlf56vFbmtD6Nl\\\\\\\/F9ezfWyb6Xyrdk3il8RaUSs9o46cKke5oBh2iIbuonC+SYx0whyFkjcFe657fIbyXKNIpebzY8Bv\\\\\\\/4S1okKBR3ROEOFXT9OaBxi0OCP2m5Dzvjrk8YGXYX5LkSnizl144FgNh\\\\\\\/EjkvPUpJ\\\\\\\/4+5h24DptgyeLmbJy4EKvOxyJtIT5Eh2T5cwU5Ljv1TjjCQpTpnCdUmzEH3fnVYQSdNC2aqD6ikAwnxIA0ghdQZO1bRbDvRtbNrCl534ANWckXrMr\\\\\\\/sUg\\\\\\\/teoaOixp0fqJ5tu9agyEHC6hrx3YjCOeBb7yiQQEMJm79sA1T8VbSdSnz9QvRKUn18T5SfrfUwsj+x4UMy56k5Ys8g323vEucY\\\\\\\/cPYunlyQua82DXx2B5bqoSXuJi4N\\\\\\\/f\\\\\\\/VaUC4j0ehSCC4s1X76EFB3YFVJ7WugU1VFaZ\\\\\\\/uFjeBvqaBZc\\\\\\\/KQAvfmLB0zN9QDic0qWQhGpi53ZXkL+4eA7z8oAcbmLUD5lkyGkNKb0vG6jBaFrXxDafHQ\\\\\\\/a2RpC4D2frGXOKw7Ur9innkHJID5sBJBaHIvkwspRFj8MZq+MVwVdYoHoT6parALdD5+0Sm1epQA6VhEtHAmdROG+oh2oC5fN\\\\\\\/AeVwfNTg9jzvG2HDqhUZmsDdTjmtL36Uapx+A50T08UtaJFwF+qhgAT\\\\\\\/I7WjrdZX9HZUh4ekaegoTjB3NbgDBfhsEv8FjdMs+AKiqK6KZ\\\\\\\/ZLOpZdFuNG0ahsF4erkgtgt248Lw4HeBB3DN8VYU87cSeVSxE7LpN7LNNWyzHYFwDXMJfTW8gaMfUrGvbhQ1mp5IxcY9AjLwNHPd2YTznJAGjYlhjf7l1x7Az0x8EJ4UC97OL+dpmZ2IvjwAUPLCHYS7uKer5NXTKjz\\\\\\\/FKUmbNPAW7mKF9WMbMdcW0NmYbJMFR5AF+jWwgix641wGN7EbJwUmycxqoeTEG9nVBWMNmwfD2AgSd60lymUYqMA5xsgCksoslea8zF9FgFWx6E2MNM7oUhpVYoOrknULP+QBsFHcw+8ded8PebhkMySbuedyg1BJP4sOGm2Y1XxKPbbUkNnKI+CG+HQevhPgOQgMz6nj+YJgTV5lgFUyv26afF48M13b9S6QxazZUAw57\\\\\\\/38subbM+dDM9hiAbH2XM1agGYkZmYZFQs0+b+eZniBBevM\\\\\\\/MOvw8u2Wb2zSq18LpJCkkgnVTegRANQgZzaLrYmO7mT6o7VXQcK24RY0P0uUamgW5KPNxjQjxxyJ6G4N2WkZYgHq4pDn\\\\\\\/VnloKi6LrXfZX66bjy9BkrjKDiEidrDSEhPlNW\\\\\\\/34VeDvDJtqiaqAQvvtBsaq1CjAh3\\\\\\\/ZKrmcV1IVUZezm9GbBVdwat7PoIBrBZALjgKmD2KGYiyzoC+EkxeqYErnv1sN2jRQILc1fmeT5nMFRyRwCJYjYmmNEp1Lc96f6taKZrgAsRagZmW68\\\\\\\/+HzpCWU3cteGqFW1SkGIs6Vs2I+9rfXEJXSGNjvQkfKm6fPycwnVLJIii2DbJuszKIwVNjvVvjXgk6LqmTGu6aRF+9aSiP2VV5UZCbGEHKJJAubqecHukxKbAZKBXFPuaE96aEQHQzs2oNNpEWuVxho5PvKrIi4PZCGTSxjFnEAh2UqkA3JT3zWvOoi4d+1A0H+e\\\\\\\/EL0yibBBbqHNElRbD0OQoEyY0rjjjr4Fsl9CADMr8PDaMZDvL2Z7519Gand\\\\\\\/wK3x+h6AvDVYXV2BFr6xMT1Ar+ouXzpV\\\\\\\/BUEYwgKxLlvyvCBga4tBOu5dEJVjVmqpiGw0SGEQNY4g28idE6z+jomXX3lW\\\\\\\/IQ53nIQ7Yah9v1gDMu2HfT+Y+pJrOaG2etrHrq6XB9K4c0Wh1WUscU3EY\\\\\\\/\\\\\\\/66pMQwYNc3y6brGiHp81FCU0ywLqx6TOkdo9DgN0R7ZtOG9+SnrzHojsNQ1jHZ79g\\\\\\\/uRO2ehMnHTCCPpcORPIKOun6V\\\\\\\/yWdrlcccR185V5\\\\\\\/c0A3zpLVQaDRkwr7FG5vTKPzGmJuAG6nmGR+TJOpIiH1lOgi6sZS5kf9G8kgqJsiOu4S7b\\\\\\\/1\\\\\\\/AbBK1NP4p5dutOHBxbXU8JfYnPVWzDdQmpVkn0jR11g1CkFvEWhfCFFh046Y2\\\\\\\/qaRIxN7FRhVylmntjiYfK\\\\\\\/6w2meZGqlbVY3ItY7deD0i23NuS3ipVyfFzCxaPzIPo0ky8YvUerc4gXz1fo6+pISJQOzVInrd\\\\\\\/POJOIIUKUzv0r59CHAPJXV0wxDvhLX9qhhMf4zf+SFz5bGgPWx74yZQ4mVH6imDhqxjZlRybbXT6ZzA\\\\\\\/GDhK7PE2QvmvUEbR6PRPcYjq+H+bEp6R1dJ4LPklHIM\\\\\\\/ZxCUBcrhAey19rhLQvSQjsqkRJKRz4jhwlF25MDlWvXFen65x3ijbeUPnJd76LP\\\\\\\/tKoCqneLd4bF\\\\\\\/hsBV71uhWS2vL0LAuNQRfxuXPPuHsBqj2Hs29dmpN+woMCGLAr1SxN+o9LgFAvaOlznjH7U9DKDYEYSOnCfueJhW0+0DWpQZvK+TAkor\\\\\\\/TMXN4mNwXQh7KkXMap9UBm5DJ5pf4dvWEtOVkT\\\\\\\/1Q8FoiqNK4ghphBBVFWFyetqFHNEjKoW+du1\\\\\\\/wfEAqrdLrVK4nwUIpouaFt8CNcWW7rXs\\\\\\\/\\\\\\\/urHvtFiOQT6UPIawdQNNfbBxlPt9Q71kvQkMloXYH2xRm7xEDybu6TtBEfX+0di3scnkTLUXflIzlgIBQ0\\\\\\\/NNz0ju3T6jGlloFpCYjJ5jjV+jMdk3WEzeLdIMOtdu8fVtOckcBwm6KnscHg0RAX2K0+l+nLhPqIDEM3l2Lm58MlPfkhiZgKkd5rj5hAULQviO9s4MHZdBAhmVOYIVkWrYK6BcpipfOFisVkdRKvcG0Y54QRGhH1SJC\\\\\\\/494kA7cSBEwWtdI\\\\\\\/RNzG0geAQvU4K6LWR+eCXdwkfRuZZCsLXAcnBJDphs3\\\\\\\/xRUU4yyo\\\\\\\/FreOuL1r5WEyA+AjE\\\\\\\/kN0LooL7P35PUbKqE8p8HKH\\\\\\\/5OI2FHvCo85giqpncEDMww5PHI\\\\\\\/55R1f16FxUFkphDiatYjJb33ZI8k23GF05B7LS+UcNBFI7L\\\\\\\/iqw6KfWOgGmEdf0qsWLhsAOzWQvYvqMYS94AxZZ6z1c5CyYQUNIKVMAtDUiwNLe+X6h7MOz5uZPMpOWOiJDiXqlfesElLIV5ObY9PTQ0vY2t+Qa0Z6rsJOd36Vvq+MVro8Tv3XAj7hUJof0GsbsfMRxVwxT6IQikuGIxSBspnkpaRA9is5b7cK\\\\\\\/MIzjx0u6RiwCnKO\\\\\\\/8nqiRfnaL8mt8oGOc6XYOajphGNVt+KVlwdUkbqayAC19tBIVrVhtDHbvHDXDMgG1uxrisfUEZRHvG9J1u\\\\\\\/XRn4jvbnoBaT52RcWGDVAk0eMYgwA2y+ikphMYzYdtChEALOI21FATE6JPJZGlOLPE6rLGHnlcqPR0fLZF9Gdubjm8Cq9srU51xCmBrBQ1oJNmE9qWiV7hVYFoHOtX\\\\\\\/zmmH4VXnhYnkTeLiX6Uwbs0OAGGLt9\\\\\\\/Tu7pwyDTYlrKZSAn6CRQcUeudfrFCG5Q3ZGYi345bxLLH3MaPC46ZMYJBfvP6fbQvLa2Ie4iYP\\\\\\\/7Lr5PYLEQ+YzGo50HEVVsJqW7cv0Cj9VZtwkhsbioLpNtiF\\\\\\\/2khLtUPnzIh0it+J9QTQZrl1QRHO\\\\\\\/FzH0wtiwwdTsCNtzocRJJj4\\\\\\\/8LOhcMHBOgpqNuRo+UwSi1WnGJ+4teR+hzzzPMCRReQI9iXVXAQC1CAcQiTRTE0lU6r09TQQ3FhjhZjiFyIV4iQ2lKix8eUH9n60ZIb61ivIZcTbEHskGXmpmtgpLutgk70Rxjh+im6NPNOeOHyMMjdzd2jtxfDFnGhVn4e8S406kWjUN9iK9TQHemJ4Rq71tAqU1mRmJT9PCsPoKH7\\\\\\\/qCoEzDCe+HTn15wSjS8WLhdPRLUVPmjiaYpSjjE1NZucPMWuU+mc+VJ2Njux6ysNTT4CGGV310g6XgTilfBucba+Hod1+5oQhVZr+9VFt++LukeXALBVNhbrVZSPdoisS9bQb3dZnIw2ffroUcSzqZ9otYZGjtlpbYhLhc4R8kNWTLjReY0Q3BPPC2H+9chua39P\\\\\\\/Wri2KNqakxfSa4ZCN6yUOc2ipoKtCxBDiYWLE3Ij6OPKjU3xgRhe6xf2sQPBrGhdLCsbUoSyJyJSl04HcC3PCJEtzyuMkJaggESAd1E9N07NplfiCHisT+q+qBGkQ7tcRxTJ07MmkUtjj1QDvxaXxC+VBoLlgJGpmPUHqZtN6Df95Qn1FqxQ+bIsB1Gc3fAjXuGCjU74FLp5cKEg2LZgC3otbLcBfg0eMwr0sgLYhx64hBhTk4hS0YAvkGclXf1SkjbQlK65ul2J5+I2kD6tgShCsFaP7Rc2UNvKURMJdW8eEwqcWRmYtXBaJ7N2KPM9ZeTXGegA1kdG52DjcnGF6fkMBamMIqLZl32dojmSn+vGkeGoqn1Q2VwMPu5WswqcSJl3kxndobR9Z526aA4DDXTHBXC4UlbyRJhuZG7j5\\\\\\\/kjG+RK3ZfzuHzhXwYRCunLv15xaKTLntgJb\\\\\\\/obUUgCHeG5XPJCd0RZ+SX8Sf\\\\\\\/G0X\\\\\\\/uzAhyfTD7w+hFTEySU8vq3DL754ie\\\\\\\/F4OknJu3D04kXrhsiFrT+crAOT7L3j52wcAyHqJTURyVZiDUvS8w3oci+046R8kTNZN\\\\\\\/h18VtbN05K+4qQCYriXo1Dv4KHsl9hGLG0tIWGWmulDafkrkjmfCEZeS02JRu9E3UNtv2ebsnLHkQR\\\\\\\/jM+pa\\\\\\\/uSGJ70PGy8DgsG6vu5f9vB2Rs9HOUbf2Y+0HqnuNXUvtsN0dm74E52B3Py3EATUArINP4xEPLudTsLtrQDAtTqNtL6da36uMo9OcRM8WKge3MlGtdHd9QeIiGx+kHhZqhg1GGDOZUW23TqFbn9yH+7oVQ2j8QPJyKnBDUC0UVJjcBcJlLlpvPlThC2t3qayLGdXC2Dix0Oqd+ny4ef1UrbtBfuXj\\\\\\\/vpkMFZLXUzutfM7EPuhMIJ\\\\\\\/8Co9zoca1W2GLTG0nXN3nOMxr6dZtT7BDI2QlCTrz63MIh\\\\\\\/cfpUmpcCMaHjrG5eBBWSdYw\\\\\\\/\\\\\\\/XtQes9ahigovIHFX7yXcVl1HAgP4YB7Giq171egXdkObQbfmQdkk5VKS7rHsA1zSuV6NcZx8+rpPTjTG1uzKcqa52kncZRXKJW8JnEtk23Im5EgEHuWiMM8xOXm7mtOtRAGYWJCAW9UUSK+36q9bTomq515WKvdQhSZDG4E4xA0+J\\\\\\\/RLlriPeNQNG\\\\\\\/mXtSHOARWSOumRQaZ7m5dvSLioXATeouerU0QtYWMET7S7ISvk1PK6PPIiptzK6kn9jd6eG1fV00yf5f9DvV+MOssBBW8qaYLcpnCTq3PMq8DtpW8GO5s3IJO2d2M7Ce2AwwmVhrI2Gw1GbfoyDIP904hC3QDrTLiaCF+2pFM8BB2JPDwbNPgdXV6o26TClCNLk7PpprIJKXU8PmS0j\\\\\\\/85EpgCeaVBc0dpCMZhzK3fZBmCZ2\\\\\\\/xewz9R2iaxI0efVRy3KIcyQLWTmPIMswl+c5uA9Zz\\\\\\\/3PKi8sI9rAjsk2aRb62prkTjfVk3HCMgcIYd4HK6nWFWk9ochoju4U+MWeptTEsXU3bAwueC6l+p\\\\\\\/8EDQmERVtxX3DVMlyUUZGe5arzjaatcpmJv29o4qxXNkMLmlBvEMj\\\\\\\/o0F11ZhpomRJDpVsW5m78GcZ6iv8yLtulguDEOmgzJQkAEQxY9V6F3W4MNxgaM5AtIXSFOKnlNlYCyfDHI9FWf7pu6tdzpP70z5cddmhdnGq3mexY6bZc7ujiDBQM03edk7C+4Sv7dxB0eSyZFTRrUMJdkUQ2lTRqRzvaCHr3HDdgpcy0YT36\\\\\\\/Oc02yIMht5wkcFzZiYVFnyzRMea5bPEHv4vArN\\\\\\\/lqUzY+Bjaz4FD\\\\\\\/+9Yj78Nx6afK8gcurhK6HZZrJI3NrWsUjSCBdYpyZUmeOuIj1BPsShKx+cu1qV8TyXnvnb58+XOU9wGK6WSJWP6zg+zFc19RWa0EUVEU6z8nluPJIc2RzE5BzPTTfIZt1OCYan5cf8hYavDbkJYjlSSL\\\\\\\/xcxQ9FXCqllcFMgAiBEa1j9FmcO+Q\\\\\\\/wS7WihM3LgyOVj0ApFIA4nNVzhwGPde0RIyHBNPJslAlpxWGZMqFSr3ZCwuizNLgjupV61qHfACbdVZGhb6pBxqDyEvUbgtWacBMs8MxLpgoGd2kz3NNnsAG2937uL5QLafi4oeeKAPuQZNBY6VWUByNy5C4hjuR7DhEGIBgILUvfF01B85RZtHtAmmMgbcrPApvhc3xogoutwiodoXarGtxs4i+vWtDse9mbe+zOwBEB2LsQySUMPpHNgn4Rk+CZF4pkI652Jezvtq3NVuAbAx725SVY9cXe6zojtSt3rDhIQvzh1gkaF\\\\\\\/wNLAR69UDETAZlEKeR+uRe+8yf0P\\\\\\\/VKdhY5NbI4dhs0PT28VFPA20pyIzkFtxjV\\\\\\\/Ug3FpHYxitc0ioi3RACMzsUMHw33un6RxqT1q2w1zbCGVoGBzFFGQZWen2t0OP7\\\\\\\/M2cbmVfO5RUngT07J3gtf85CMYbWVTH9U8cNXNLNXXL9Qbasr\\\\\\\/AsKVND2JQPtUqY7XuO97oLHontfyL9Eor1vXWZr1R5Hjlz6fQcPIKZkpCf2txxDRoraIWkeHLgJz7F46O\\\\\\\/DbXewrdAu5rfzkTPsTJtw5NzNPcjow5iGcq+r3wBihblBE4DHxsSv7oka3R3W6wIf63LBT64MvmhQDUqxlYLziXCXIHugpwANq2oEVsmDfFcvThupbkAN9DzZbkvpF2ob0OS9BRAVnE\\\\\\\/mJg2rEGyX1QYjVeBG7hoWHSX07ZifCqMRAYC7ExHmJ4jH\\\\\\\/MhIfzNib\\\\\\\/QclY0DtXQpqllOzZcdYEe4hOGAE+6nT9jB6epg9Z\\\\\\\/7H\\\\\\\/+p2sgepJLkTghE7eSt4GwifYumgwVGu2mBC\\\\\\\/6iq9IWvIsOg0Zc5qodJ34XBWbjNvqtBoqHp84TqBb9ZP3lDp+YjZoWLvNMH7xqML1JBNwsp084mXhWSsDnamIZL2u2JYQOuhh90BaOt8UyNMBW7QO2lLsiIcVQEG81CIXqO3j7UgXYByX2R94CE4hvZD+kLvGiYlnD+yD8mFAb5UQdUO6OiLzGKaOAV\\\\\\\/BiA4MHj1DeJfUBPc1InXCA9+ywGB2CVatHx0RyCCVIvd8+rM1NaytonydLyX9955R758YnMpdmJdYfyg42kDfrO7ftbyhvKaawD5KT7pHA1DMv7uJS\\\\\\\/5QBRt7UT2HyJ\\\\\\\/jM\\\\\\\/876vzUwFHQOKgg6CZhv9ZRsvd+US7\\\\\\\/7X5EKhpNjqjyLBmEFUr+vi4urdI71ta1tpBrDOmXF4AR90u5tqDVmVD0aL1HGEpfymPd1Zmhc1MMRfoRRVWQNUVNmLDQRLQesaUBspqgUtUAWeYIeJP743Kgk2f14qSAHBfchQ6iEBrBmkr18v6U0sk2S7nWIImNfZxOxwYvHEyKirZ3zW6Gr7dPylLKQrNn8eu1SbAN3ud5uSPL0N\\\\\\\/3Rctdcbgzd7h7GEMaAvmvltx5S5PMSenn+zxZHlIpydnZ8HuF2q1VMcKMAvU6FlBzURCECWetFtBv3XthBErdOPeBwqPJP2pYTdcYl9L3+SIAKh34jWo7L9JFTbmLoEN9K+40+5+L\\\\\\\/fp2PGWH+mBxTNLMS5F8Bg6ouJtNBZZC\\\\\\\/m9d4XazYMizGFF3JXXlpwziKGWKVl\\\\\\\/CAluhIC8BrX3VOkFgvXBB9c++9dh++yd+3ufchoVYIXLPS8YWsKwOR0tNtMWAY76XUUxKVgngzJAzdhAi8UytZjSVPFPTRl+MlgmXH6xDo1L5aBcIdbCcoZOXrf7wk15gGkn14nrcZ4gTZ0xFGkP60IVSBPcEM7VUw07L0SXPfrpj3zc+Um6LhN4JLMrusxOei6zf0lNdOPXBnFjVG0lkBsJHtY\\\\\\\/9+7gQTT3VUtUJJzLioiMCSFBFO5P8BEP9LrDGwWD3IRX+iAxcljhtVnhX5CvKKUGRBbHpMbDQmYGyN2LstY5TAAhVs6mh11cWY7Yo\\\\\\\/dnT1FT7bIOBG+z5OR4taze1hcMKP0bgfVGEYi8sBmimA3bUGz5spfGGLcsHDXYehOag072wMVIImFCkmr95g7AEdPflaaUz+qYHCH+aF2ho5nU8RiQiHXy2vtzQ9\\\\\\\/8oMlvToCNSwFVCe1fGQp21PZjrJLTAtzr9yoo3ui+9UqlvThWStImGmfeZe6KzT52p0f6uf5+qQJ+WzUpzYgCbI3IY+DHTolvAKyyWz4Qkt+BFkcPynmJkgxcPMplfR+GAuaAWtIvDeAd5bPaPdbK\\\\\\\/tIUck8Y8R7a1R3QjROv+P5px0ICstu9oc+ztCZcG7mYhCYrQtarE2B5ru\\\\\\\/nuLV8qdxWd4EZugyv+iiQAeIsF\\\\\\\/mCKDSyTUk0jVUHgLdPlHeqwlmIlFct7N4wVEEi3P\\\\\\\/sxaHYFMGaZ7ob6Oc9SFrYJiwveojMXMfhZiwdPuLTQwd562LHTk19SN1OQFVZ2tQP9pQ6d97gcIqzMPSNjhIdtW8I1xdYW0VVjWbLpBzJuyVE9vBlAQtTjyG2nMJ9GMT+EyMcDhwso0zKmdtQ+91\\\\\\\/4OBEzY+cbvDFRB3hRlQZsmx\\\\\\\/fsSiljjhj3r58peVkZt0iq4lyq\\\\\\\/xAxbGVijgk89Sl8Jj2yxhDwU9VXm+66Yvx3D45hGyZuU3AUhFccLzTuuUs\\\\\\\/KQxaX7y9HrOX1z4M3wSN0pCzsQLrCpUnMDhuWVbUKt+weInQMgwzPVBBjd8rUj0e5TjiVM1iBuuVI1EEIZv\\\\\\\/wU1IaACNvBT74e+x2zFBk86RcBQREoMdkkJ\\\\\\\/d0mO1dVpf\\\\\\\/XDifTrZe5SrAcwOsqunboD6h4fnpSjMOTguzEu7MhAvwA8YHDPfvndhcIfXYeH+6o9hxZRyARJ6MEIpRJ\\\\\\\/pS6kvSvTv8NJKgIt8fMB9BDfHxckh6GW36VWUAuecBm12e\\\\\\\/sRcvDLkXt0666ujSP1SyrX0cP4sc1Zu5382In2TF1iWUuS9h+sRgCHREu71sJadXaWSqNkgoPekwr51e\\\\\\\/WrMlSZL+FKJnM0Z2vgYO\\\\\\\/s9+pHAUpxqgh+IFn5xcj6nmFkoUW2VYaBoSQuKeNqeZT2oqlKt4F6qTdvlicQgH154Vn7VdCGfFFs3aoBQ5TSdTzJn0T+kriYPcn0ZRJyI9UsUAw+8ZL62TEeaZh+xDaiQ\\\\\\\/Y8\\\\\\\/jk9pEt\\\\\\\/T6cNTKjJbQWl6ZgIeDgApk2e1sHhcMk7QnQKbCXmhjLJ1Wm3oc7aO\\\\\\\/WcsNeEMl9gPTXac2JYQp+cuKUF+PIilCYpUzilSUB+Qt6wec7Rz4oOoYDctWUbrciXwPU40UxcwLOqmdpurv\\\\\\\/t8\\\\\\\/Y29+EgVZeQtC3s2YZY8HLQB9nVy1JP65rVwdItEs+crrVFQbEvE+QYvdUKMGDSA9zpeLA8sGcqi6G3CaJUk6y\\\\\\\/ORP6s7IZltEQ4+xH0ppoKWTr0V59nFPYB6zeGFl5J7fYdZbttUYs8FunhfVNVjvZM4E6yl+OamiJURxQM5ZzFmPLveKiS5wwIaE9IWoc5UCiaPbHvnwMjphWqQhvbNA3im42EfeKQaeXu72v5iuHPS0Q7I2cUPT2o1Dqiv2+QquFr5pQpRQ1QCGjNNqhWIFUQKc95fuVDdxIbtzl\\\\\\\/sZ2DjJ6ovXfsyJ03JURzKL0boRk4yQGPgcQDbK6cm6ovVE0dzsBzJ4x\\\\\\\/JedVWiAvDKEG3Xp3QRrAosHqVLDzW\\\\\\\/Vj+XeiSIvEoGhm3R2zeBRQpSkSa0HaqEPmuYhGawBbPM1DSiH2wg5KwsFWwrhsvhqIbs\\\\\\\/0H36kVh7tCbjrEIjFnwVpbdKOXGxnTbwCFCLsoTauoQLHkDqknHHL5Kd7+BRyxbCfgzVpv8x\\\\\\\/LAaISQix4lN+\\\\\\\/4PyATM0dkARxHGB6m30DMWslyiKajObADUatATUoPFSpEyVrUD5ZQGudgvgHFUhs\\\\\\\/9hYZXVgsQ+gFAQpjDnicymdMoPgnAAknxNbM4VcbSUfoeSOGKw28DKwQTqz50Vwg6JOcKyAgQvSaSipYSYnjZ+DysH2SscWy+9DVZDnSx2tqSm6vpGsB\\\\\\\/GuXZoNEZ4NK65n6mWoCNtl3RPH4r\\\\\\\/5juMKtv3DnIu9hrS85Gi00C7U8H+n+emfZUc9o9heFmUaUZqd2AXl18cIZhrq+vpGidcYBBquJNZfzTw\\\\\\\/InhG6jQ782LOGgaMU1xiGnF6m2X2cr2M+8Vq13dMtWFIkuml5CXyAKHJ+mdH1ury3JUcFoEk4KTJE+VqJ8XqApPHptEeuNch6h8NCYBXym1Axf0R1YybWFjnUlc9121RKBmfww6ykik162BPizLDgCcfPIXay\\\\\\\/mx7Ig29vWS5IRnJLpRwiQuSHVExOueKu3jmCxZrGk5Jwsh3DSUQNqTkr3d7VV27hPlSMRMEb2MdWkCVjEtJH52HhF2Wk02qLEndtAgzmDIqYBRvvzL8jrujJsn7Szu+KvM39h3GEGq6rgzLfy2UOf7MesyAmsv6ug1WuCKh6j1\\\\\\\/97EHdDcFHhl2pwgSGXAMvtyaHAd5c83pTBH\\\\\\\/gU2XD7x7wjnD2gUPJ1XpEvMc9nAHSf9N3BZug7+16meuERDJ2fYm6JHL8X5yxjq52VrRR8df0Qxmdln5W8UCfbv8PmDkMN+Gqxa+GxocZjmP2kzjIpCbporwXBdZdNMTjDAI\\\\\\\/TF7o28sSu1wCKIc8gXLKPFXghbnIvZQ\\\\\\\/9fjrBJNdGqeOfkJ06t1lAl9IumzypdphxmRlzwQ47pfcut+6bszXxMe5rrcZcPh0VoluSXTuj+etOMT962gdWippNM1LYeFdl1BIGArvsUt1+I9ixFNQ0fL5e4PARLEHbtA3V8fOwZVOAMR1wdvgmhDTCp+Kx1Ug0DkxpIlUxIFaJ5udb\\\\\\\/fSnUVA+UCu58nTFNfAzLGGM0R2gul6ohmOdS0pWU6Y6oZkjbqJINzasMzixWiLGwBBUrpQqb3BxrMIkZA2LOrf49ceQD4PC\\\\\\\/kk8mMIg7o1c1lVYvgh0avWVIhIvf7u3Rc\\\\\\\/+bNjxWf5yw+J1vYU7685RlT4TXP+I9RXHSBGUkC1FU4+91OBfoAK8HZnV7JBRAxyZhdOyAlPOQFJJGKHwI1Yt7ZYd\\\\\\\/m4glB2ur99EE6ghTW3+rwrHIn0r+3V7i\\\\\\\/6iM8o5MrS59YP0kYTsPbPbd9a336XdADVzYl7fFzVgA+6ojMYD4FPaYiJS0gvuXxRT6NT4j5ByaxDCDN8ersvBnXhbakIKypYRosStz5OsAX4MxwsxT5y2KWJbvQvLMw7O244JWhdUUda2TJopX8\\\\\\\/TEjf9Ib3kJ6f+kO0CVqv8ohXRtQOKzdLouSG\\\\\\\/DJAszcUSZ+DnkSAD81+p39Gi+EethV044m86xLUjEkFYfp8A0p7PBHFk0UDvrr2zGqmaGvdoF4\\\\\\\/1Ga+JENxlC1Aqq\\\\\\\/4W9LkezDO5yDu3STUTqv\\\\\\\/Gi4Mcij5zVvrWc4+umOyGPqo0UjDo5AH\\\\\\\/zaOM1aiX+LAPnoyenpPA0LAh1DDaxCRBwq66Jt7bXZaRxUaAob45GubNC8ExUJzEPY3humTqvg3Ugf7USplUABhZgfC2aSguK0oxIG4XlOnXn0EqsuPsSZoYOGHJSuaYYbT7lQ5dDLp3KEwaVe+eBTSZzMjSHNKwr6ihZekYfWFkhAVVjfjjKi8mbXpKhlY+l08ntWb80QkNFqeK8snLOFAdwFhzyJQeeAdDolguhvWFeeBXNZ01bWs0XoiohbnACC+bVMIH8UjDu\\\\\\\/AD2GBa9xQn2sJ9BIsTtmuGoJAMGa00hWUT2YXHX0IBJUpxg4rgLJMnZqyIRrQJKKJ4ZJ+HBo\\\\\\\/v1IGnMlvnJuwBqoOAOhs+s5o5okat0I5gXiVspL37dQcDwDHjzX8twkU5ppb0nDZq7Mro2NH7kSOZz9rcYWjHgkf6sEz4hioLFnFKzdot1Fh9lg\\\\\\\/nGoUL0yT9NoQHW7a5g\\\\\\\/hMnpvTdvQtI902UD2+AQL8pulMAE29P7pI1wNP7i2JpBxqmD5brQ+cVpUqOYo8LKM7x0lPWXaR2so66h18EmlL8YETrISp49g+w012XuviN5rh4N0BbMoVuaHzyO+9+u\\\\\\\/rX+r4GvT4RQFfXW5X2doCf9c4rFa0f8WDJ+ytE9YwR1LrOjyK33fXX9u4k+DPRriLiCa6MI4kFPgdNbfcdQA2JbhsMVCYMEIeoDJxNQRd5AUZFUYk6je7BgvQQOKDknB4pbf\\\\\\\/M9tzLNU61VP7FvuUA\\\\\\\/IoDTAc1+FBQfiHJN3EwEm9V59BnvobWKj++R1m3LiD8iEzP6XzoVaLYUVSQgZjeFVcXsNfzl8xcK5AeVZxtF5pR8zioaWzs9\\\\\\\/f+BxMtyxKZzxQSulscgogRKsxBSPbHrAz5xuG1KzTygBS0nAUjAwKgz68ffbzPkYn+2udTcyYbIHe+MXx0UzBIs3M80B22xy623nAdxw+7mdW+2chBb4+2Q8SYNysS9\\\\\\\/bQYXmZglylYNGE3f2t2z5QbcaBCHsqKZtR9Bb6bLsUIGh2bNEyXVWdP5kvQSmiYzBTP+lqYn93ZtEanzp\\\\\\\/jbZkIMZzpdOWuTdoz04iS1wbiHkFSOjndha7XMOzWZOfngyLrTymCdDAZmA2RazuJRMUjLOP1Lclzt69r1njm11lmAFEwXz6BXXyDzpKoTJIWFM8Ww8+56EvwpBtG0WBodUT6w6r58Zeo8NTKj6J\\\\\\\/UJgUg3nu7Q3KAZE4xYwNCQIaQ4EfTU3C\\\\\\\/JPSKkqlGUBLKVHWsbS61WCRUHAi89EPifM07gmdCBQzj3VQd0LZmBC4xnZvhk+mpKN+RRs35ReHdBj3lSY+jTV5yS5fZao7+Q4hk7M3EhkA9Dy2l\\\\\\\/PQGSmSsEa2uoXsjBUTXvBFVwGBBjd1Kqvw8lJh2XiRhr5oDe1dD\\\\\\\/304I9MsfZqUPepb+jEwyh3TvF0TxCeKbqkP+oBpAisNgsqXeRQynMpCgJqPJKVH3d7dg\\\\\\\/zXOVSwCrh2U6wTkr4iFL+JDSaQxkdchGPNz7u8TjEBcb0BcqWyWqOgE3M1FiPEsV\\\\\\\/z0UGUmITOo4HFlSatnyuZmiPVZovURh0uEAVB8wFcVVw0NPzudweeu5iJPU7NQmb8mwa3v\\\\\\\/\\\\\\\/BdbfO\\\\\\\/WbZROzQ6OpvpyXLR\\\\\\\/oy2T65iaHyiGfFqfFhhgiUiwwNidevndWGZOldzjVWRwkWX1ipESV3A5hZOW8FZGZ6DAui7RYsFhSOANoTapOHUqU3bRYbD9kX9zkPaTzEJ7QKfS9SHVPD8CVN\\\\\\\/KRbJj5UizA4Wle1PIGBkNwJyVq2YFguVetUsJ\\\\\\\/Y7a0+8D1N\\\\\\\/cmCZbJjyX\\\\\\\/wLUbUrQEgpeQ2RB38bxr0m0Ck4SvoePjlgB\\\\\\\/6TDPg\\\\\\\/mAqERu1DLf2xlWvBwMoWoexx6j2GsuV+ugW7dKb0HQn3E7gVZhwLAb5KP\\\\\\\/LPVI816cClvT2c4dl5zhpB\\\\\\\/7ZzA6Ax6R\\\\\\\/d3ENhNOKAlWj+kJgEHEAFJLh20WQy8NQg0VP4T2fynyQ4ZKHJaeBz\\\\\\\/zz3yx00hWKO7n0GE9pCW\\\\\\\/LT1FW32WVEWDCgOtBTnknM6WqSUhrfe1GVTEI8D\\\\\\\/KOOb+VC3U1qPLuKsclB8bOjQI+LV3+USFN+JNcNUvBSP7uStygto1kYnLKeYS2wryzdYQ+VlIhzQbdgwpnLgN+W36jjiJVUijsRxPafKpnEM\\\\\\\/7dRSvgXG+zeAlgeXSKBfAFr0oNrlADkzJVtkKr3IGqKy6R6Z1mF0iGia9vnwENXejL6pXcVfL4HCDLURz65YGLspLL1oTz7PwcR3sSWiB7Qls00LdSCU1AvEHCtP27omrR\\\\\\\/CC0BTe2f3v3MU6Qn\\\\\\\/5slFLm6KcNfZKsSCemZVJiu\\\\\\\/vpooSC2AUE\\\\\\\/6JDIhg6t1mlC3JgPC+gX9dB0vrzrUQ8EM0p6NbQB7xWjgm1Ayn15O7dJqwv+ZDRYz0XdG4z8sBq+wgBk6dtGIY5dX8Aka0wj+yBbam9OCY\\\\\\\/gE46Qakcl0FGIcHTjE7JQ9NmSn39wslaOlp2HWXkfuFQRuBQGaHJrgNcVof7fXfwDAhYBF3rHk6CnTc2mwaPSwmbdxk1oMmTo8KJ5gK9qF801ODK9lN4JcEYpehPJWJhEzzZHjXUTTcxSeU9JxwJPCr2JThITi9yAlKgIr3Qp4dtPuPRTYV6WWdi5cgT7BhqXXyl5aCk1JE6MzNvTz9+GyNl4LxS2Z\\\\\\\/cuuGjUPwCiCXqTKQmpp1z0B\\\\\\\/Zl8k2QGDfh5yB9HZR3P5ElgMeWVtCP7b8MbIRon328XFK0Mrgh\\\\\\\/I56SR+se3q+Ewwg97rqFfcEeXoBvupISUUxnEeXbbkkJP8BVvu1bBK5ut7vXA3NkXi6NO+5Qs39K+Jv74+BIn76zRJFzcevjAOdv\\\\\\\/yTPuqL3+3Q2yFKXTx7tET4+Yv0y\\\\\\\/mf+7R1Ro82P+iC5HBe\\\\\\\/J1PmKdAauci9w\\\\\\\/jy5x4CytAiRWaRmfC7RjmuNNCRrPq1ZznKrkfoeEH0WNnVbzwmqf0LjTJhe8EEX+uGwAz1k1YWidb93NjNTd\\\\\\\/kDA1bN4TlYPvQam7iitlouIwTggCQnxLkc8FXEgFOHeal4YNR0FA4uIarpmq3rE7kETHrJFvRt5qJfja9T\\\\\\\/g5NQiatIpfjo\\\\\\\/e9eTOXV6WhgUBYQ4pbdJHFVYzAf\\\\\\\/A3O1lmWP1fkBom5rlHXxJ4AkpvNy0gIHMPVrwtkDlmld9Oi9AmeAgb19Qh+SFt5bXjERRsPHdUCswi\\\\\\\/uGZpkye3tamIsEFm2bV7R2ecD4x\\\\\\\/DEOGeN7awbHOqFY+PPuZaazyH4semYz6T1O1013OmQvOrig8\\\\\\\/F+ZdUyPVzFD7zJTE8l9iiNutwhO8q0HCGZnP7+1BICkhwu5y2VcXlN49EiiuvA5PqoZrnsVzpxNJ7Suiaw+ceoJUd7MD7HhwFblRyFka6au6dVJkOfoRTb8ekr+X9WK\\\\\\\/ZdCCjBQUWKz2ZTf221lsj7JOyAPC0ik59Fdj9OUSGa7USXjkRlFS5zx\\\\\\\/mrX4CIAuZpE9alhOCwghLveIt8HXsKTZkEP7HYkFZKot26pZJ2bs4gmMha5KBIaklLk1qCv6psAKLGe7bfVG\\\\\\\/YKAyu9CGmrfhfsN5YgtstyZLNNxtHwF8cQBkW39wgYuWvmmdshl6yi22y5sUXjk5DI2Ur\\\\\\\/mJo8k8Mz\\\\\\\/czz9F84AKng6cctiCU\\\\\\\/TOsen84Ubmpo05bKPLkR39ga7imqLm1o2+bwoezbQzyMsGPt0ZJaOFn7FpGfY0YUwzJFyVqyhRis3EJ9RMhIUdYVZvu+\\\\\\\/5FSWrPl1h63+rGXYTM8vDooLSxf7MlcL2HRoEIthZo2e3dKe\\\\\\\/hmWiMtn1mAwwlOulhwMxRRaVTXFu4Z5p56Rpy29VF+xz4sfqK\\\\\\\/TlOMuajEm\\\\\\\/xhI0M5sDn0CqPMW\\\\\\\/yC2YVoSphlsjaKplH+tlkF4KFoX8\\\\\\\/DC\\\\\\\/EwCE7hCxoYs4Gt\\\\\\\/Jp3J1ranK3vyhkIlmnlbJjp1nkVYzxT3pVNScyt+jv+LSAJGOMXG7U1GEdHCMHjs6wWLjS8UrMd45D5D9etPmZXVFrKoWfiz0\\\\\\\/lC90mBMfxfa3lvP5JBd+3hlNXBhAMVe5G0P0RBHz4GQq2H+Tn5nni8hHUmwerwgnZCbCRGFf0NUZZDGsWdFyEQNArmyJM6yTFW9QE\\\\\\\/+FZT+flaDsNPsCD0LMITPSg+hNfe02bfhtcRtVpMs01EJXXKi2TSb4cjmEKVXoS5M\\\\\\\/43qpq5O8hJa3emVAVxDDNTlXzMBlHOeFiFMOek71AQFqOy7BBIDNDHPyi\\\\\\\/SPLB29y7EAT+xMs4dNo1K7mzOoj6RJGoD2asfV8cDcLA+UC2pci4Cm7lXugZjCBPS11zhOe04yjbwPwk3gLH8KnX3UdI0wQrQL98ZmMiqRuba8\\\\\\\/9VfiS8lOM3PW+vfid0AK4JsZeUzeufHyAUuUGnj9zsyQjUsPvtDCxqGaZ1FQop4zhmvpo1lvsGCn6B\\\\\\\/HRs5GM8hrpqIDv1JHEAErA7l5LjM24K0ZatG5t\\\\\\\/G7LkGs7laBD79bGwK+VHkTwrnsNPJBJL1gi0+xN+jGJPcDHmGU9T4jL7kCN8wRR\\\\\\\/mqiL9HrLMJFt9IhghR3MFr3q\\\\\\\/vAoUx1g0DjPWjfPrVvDO63Kjf3m4T8TC+DtbkxSLcUV1F25+TmBtGFb6CH+CZMzQsboLiQ0Shf6zJQ11x3i9EH9JoG8lbnhoKPzisnAuH976ie8fcuTszJp4GdxzvQS5zUdyhRlLA4i5lQoAMI2VrGFxWsEgTdPkae+3Jkp3kOlMr8HFafYmMdw8ArhqEh9HfX4c6O3HcEfBZjlenEZIIyBGRdOr6iF2dtaLOsTu9to0mUnhDaecJn8wncr8mcZ57ya97XPafc32HPvz6uM7O+f5dcQKQGUwxmvw66Lm9R5Q4fDGoYW80EYFsERK0fD7v88x8\\\\\\\/yFkyTvaQtKPxItnj7T1dBzilYPxyonhQgtGxtaUUr5jXXFC3D+ScQvhApVQJyVcNB63GKc3NyZBsn9mR\\\\\\\/Wj64WebkMlP541I4vaL27sMK77OvooHp6qCWUaaRCP8skbOHBbVL6oCGvdMgxEpSkYv3NGyLhOQAX+\\\\\\\/cBF2HpJQ+QVPNftkn+3HECjLI\\\\\\\/\\\\\\\/W2EM1JVCdow1GFP4EXf\\\\\\\/Wya3YnTz7CZF6Ehz5ImN8ubIz3d3rhqqtZtHMMhB+NoNlbSaikIZYIfLGhGeWWeXEigBIPFkN6lC7yPtGuWnePeuovK5aRZ8YBl+nSk7iVLT0lRAItlQVWwyPl0xOwhy+G30aFPgHoKNfFmaubHYW2K2Y1en999Pm0XnStGdSCaAFAlCHFezubYVJCoIb+i8qTlF9ZieMKar7mw6SX9wK6n8m40xDuKc7m68DNCYwooQLMT\\\\\\\/5bvu\\\\\\\/JfhM536QArJ0BTruF47AdqFGEhk09lwLNVe0roGEV8S3snR4JhBjIvIWx9hHFAY586C6yDcDv+3Bv+2c57+iWXUqHAIdUmeoyUfhCvB9Uudx0x+\\\\\\\/Llx\\\\\\\/HZVkVKsC36fNNM\\\\\\\/a1wi8IHwSP+tLhFh3N1MPDpVGBjCanp5XW3Fr3zB\\\\\\\/dhVdOjQgSl53aJH\\\\\\\/TzvIUIBR+12ZezHqLeGi3byL1+wH4vgMAzvK1UQJzC57JRHOv0D7dHkkcn27QQm3WsHXWtHbVJhRn2elukUU1VsFTpSVx5Y3MxU5gPim02z60bacgXnmsk44u2H6Vndxl62VHZPnIxqBl7gcOHDxgh3TnLE4BRARQEN\\\\\\\/rWtrZgpNR5jG5Bmpd2t+96kCRFaR9VivsyD8Cgz2y7Fs8Phndt+FRSWEaKrh4+vUA74UeG+G8vbhIM\\\\\\\/my\\\\\\\/7Yg1WoPPhmjWnAy0\\\\\\\/RsGkArlNjePXzRQrCyhA9Pxf+GRXqhUjZlZRq6odOXO5dJxJp2HR2\\\\\\\/rRv4rF\\\\\\\/hzYuoYzsJL1Qny0coB8jNnuW6cMXTJuL6BHJwtdWgNaU56tVXyHziEbPO1RIk2My5Me8lWeVGziNnOnD2yOk7jns\\\\\\\/K2Jdhg0TvLgkWFr1xc9OwH\\\\\\\/p6Gk8gQCMHsR6BkCXO6WXyh16m+qjGdLQNGsReVT9q8s+5Fma+1npM52oo\\\\\\\/+iU7qNW47uogOzdctBweGgU+XbHRkiJ\\\\\\\/x8yoBIfqDugt2cvBfMn5fa0NFl6Tg0VZBqkzGTuc7VwuL8LVg64onU5WzqWs1882qs+rAlAxJOVZAOhRINqC02xmENaisrCXeo21\\\\\\\/ndwGwFMEzczkMheNlv14X+sx9b9Vjht6GzyNzryqtL5s9gHZvQ1vBzMJw1lw6NpmhYgS5rFs\\\\\\\/QrzvDiaTDDzg\\\\\\\/tXVvaKWJBfpAvrt9CJ35GHZ3PJ4u648a05OcU\\\\\\\/NauNs4KMHDvYmrq8dAUUd+YaXRjEFbEXHfezbYUJZfiLm2d0T+0Ej0LrC4xUEnzmY9JDPDFlHlHj\\\\\\\/j2mmrdGHIgeVhc4S7a1aGK+623lB7Hr2OuwF9GKZCN4a5GaU\\\\\\\/7u8z1psK1\\\\\\\/fKDz0pgZwrBvNP3wmQDaONWpOQn3F5fOl+qP6KT8ZJWTx14+\\\\\\\/mPDiQ2BG1NPfLQVmDbTSRj7BPYOwHBib\\\\\\\/DKt2+P3DXe+6WKUAXlKIJLknfKp6RyWXW5PpR0I0F8hZalj+xdB02FGSzb4ivKElPCnE6Vq1z8KuYwKSG0VG\\\\\\\/vbY7M2NmNWbyujJ0YpxBnD4ZEAw8NnVT00tuHsMzNnyZz1PPrsnxVHpEnaEYoLcuD31pEU0iZsQ1JQbhLxZ31XNypBrn57TIDhesrGKAQPFDL7zoTO0jxnQtcR7Bcr12dodNCL9xwpV5KKe+pOH8UYGFlxnVMZxFyUkugWbD4so96Dn0FvTE9xMQsVli8TdZtctWH2g33JEQxG\\\\\\\/e08CPn7vdBGggdoKril3qQwbbEllwYtup9ZtcJx3Z4pY4tBTYCYc7tERzvm1KIJZvo1KN5yEIIgKlXDxe5eMz9awN57LL\\\\\\\/D1vwyngxIqdzMeyA3U465NjzHWTwtun+nRL9dXjiASzUVUFo0j9Lh\\\\\\\/UJoIpWnPc8GxFyNomgTARZoNEsZFPZWefK8IppuHD7AFu1eIji0Bo4iFy+Z73Za7HrrzM1VoMLICZhh1uoY7ierA9IXTHHT4xjlflN2m4fUuWeLubjGuyYoITTqvuoOuRY0ETuThZoM3LEgIFCfSXx8jRz4s4yvmuAjBwcooPnArKJG98EBWbfNujymGHiVj1doz+yk6DuQzQiQTGt1EyWcMD4nO8vEdO6nTxSHyupAp7VEf9kg0ScQkeQiSLZwd+6czq0nZ3DKzweaEQHfCCEQapK6sbITnQkuaKSfd4mnLbpGI6F1qFT33R5i7mptwdKoJc+AMWz15xmkG3GJfCX1ZdilfrpsUR2YDMNPHwt0Rzft6EwalQmI+9epLHyUagXIrhqepoxYD45cm2EI+Dmvn5aPpBMRC\\\\\\\/ymtMKh6OeVZ5AR6jAgVyxWCNDT9EGBQCJs0iubC8Bu3lJTjKHY8SafhNzv4FQrTkTSaXXLi6D5\\\\\\\/bTlWv1rtFdS4Ag8+RjOTcPLQU6dDBnf\\\\\\\/madj1jxYdly6g1tW1egd0vZ7mtQwgT5nlvVkLlwxZWhiExYH5uQU1hTygl5cgvI5BkN2cjwvW62LBZZp4JnPFI4dVpI+kWN1wOx0IWjoXnFWo+kwa1KhmwOiDnEqQ7FtkvGyre+bDPhPCfewIEHlSk1lw8lzbk0DIIv2f7aiLpLIbEtlQt45eedPExw7BxIVZFC2SlY1peSx\\\\\\\/y3F6HTi5p1WUsPDZBmoE0UVf4no3jCqxTh3GvnjmiNqKmGe7BcCor0O4bXsnuH6Xe3Etej1Qo4+8o3+ERtpALH6na351Ul7TZxVHqs\\\\\\\/KMRTtdh3evIKjt5mZ2GdYMNotLissoaDPVlCg3N6jdd9JVrtqg2U07oSsIAB2EqJ3I0JM6K6\\\\\\\/FvqaR4qCjh0xG5TOxYKw\\\\\\\/xCV\\\\\\\/0qOtcqgmCxk4Txj46Kyw4qPKcexW1FWWp6qNVPfwLsT5rbTLb\\\\\\\/0+gPeic0CpVF1F5hD+atpTjK1JlZ+wbWmgNR6h4gePC0P+8SRQd8FMSwCQlv97w6OtRdy6axbB1zwBjwcaahfuy5shxZsJ6JYxKHcgJVYTpdeACXNV3\\\\\\\/xURVABP2tmAU7YknMhgz9eRZfQoQaGhoBBT9zt4mgVHib54vsy3OeeFxRHcT+EpXU0PNoxqWuE3DS9ym3\\\\\\\/KsJxcuyfwwo1wVvzGE3F2mAAJz00EDptezDAum53gIuK72ECXTZK9\\\\\\\/6nSYKbyIxFv1zRRdhhxo\\\\\\\/L\\\\\\\/qrzn1++TIB6yL6TdvHNYhENqo9DOEQm\\\\\\\/WytyLCtP33BGJQ1bY6sjZEb2qYowoeLR7Mt+\\\\\\\/ETJmENuvqsMSyQDGVoq2AudtjpZPhceXUQXyZNH\\\\\\\/uqYA+9Ma8EVyApYo8bNz6yA1P+yWKDVyfklXqFiIvhEcz8muP0oVYG\\\\\\\/OlXTsHxNyr6klWCBiFXf0BWCbMWS6soQa\\\\\\\/v5PAKkSjOWFjH2Hcmb1ktY9ur2fZ7zYoy4SUO0gy2hZ\\\\\\\/MHz5TrDnkYuJRtKyNXvHPTILEAUfE54k7eF\\\\\\\/\\\\\\\/lVrDtm\\\\\\\/Go7pO8FzcyGUBQS1Dn7bd\\\\\\\/szGwINFMrpYg+Obzeq1CFFNCBNjvMnTCKcVC9EK6NNLsrETdmVn3xa3Xs1LaPnzAhoiGydxSd8xedVAfULfjf1GOhmrOQjuz254OsPYVIVnSIiaQyzBHfayO3wIkunTEXIFdAz8wwQ+4FxT0ySqPvtTt7Cr1lum5uW2C5EYURH8VgBAP9XMm1PLyNrdFKp1HrFaTX9PIoK7raX6\\\\\\\/kD1LoIFzOEln82Rwtfor2pdBktn6xits1qnAX7cUjRIQv7WIbfaZ3OY9dWvyVhZR+k+RiiZ3PDk0XKo1q9Maeet0\\\\\\\/qBWuvhLxKX\\\\\\\/VrODy9+SB8glA5eFET7p8Kq7O1M8WWV0Oe2lryjALCLayfQF\\\\\\\/MYm\\\\\\\/YaZhbc4RUJJjqZF36H4j9jX0rbmOC1mqrOQ\\\\\\\/ieo06vAH4ssYx6Ppkipacs7Tx7aplEHKI\\\\\\\/IIL27uX+qQPIubw2LbDXXl7E5E2c\\\\\\\/cfp4wLk7qLM5qVxZNYU6ZmKjcKATf+0irCXf+o86nZrmOSfKVIO8az6OrJzWMvEV\\\\\\\/lWc6S7BsJhjEd0CpMGQ2X0yi0PVpIc3YlbKv8ZIvDHYSu383MMlM\\\\\\\/JeM2b+qNfPQAZw0O6YJAr1rAcGDmi\\\\\\\/MoUZsc91XwJbeMbrI9GI8gMXjZjpGxEB91TdrVwa3jd91yfuYmbtMuXbX5Htv3UErQQZ4UxSlJydlh5ztybdYovOYSCwSeLkYBfiQoPPGvQNM6y3oQG9mKjJ5h+Wt0mzoa\\\\\\\/W89YdiuIWgJ\\\\\\\/R7J0zVg\\\\\\\/FlHkH50g0HDwYErtAeQdEia0sf6GTnVGZrRdPZJHNwSUx9m7BkIdqnzeA2rI79xkKzxpIsFvFEzRy9g2WsBJw0\\\\\\\/C2uOw297jCT3xwKizdTYMApUgqdQFwawwvHEomUR9Eh\\\\\\\/SLwTvt48zBOaQlXapYdO8wh6GnC4l5vwSuJh4Om0reKhI8TodUgfaNYHrRfHcVg6uQe6Ac1VildE+09DFBUgH\\\\\\\/LIJbeOLqFILCJKrbrvUJxzqVBOePRU\\\\\\\/CEycMQznbFj6xZ8lZvC9BW2hTUgwVO4rcaNFUdbquhH2whdwcWE6TU0PrmMrFkHEQ\\\\\\\/Ent+Ev0dY19AvtLlyZjV+f\\\\\\\/1Fcu6mqe0euYjBNLXY2dn38S3MU1tBMbQApDO58WhjKt2ojG2D6H0S5RlnDhpJeddz0og2mvRnhg7Rvy11kiiFWePHPABfNYJ+\\\\\\\/Kc49pjNZCtvhVFjszU4JvNLjkvRH\\\\\\\/H4DNQ8h9Jk5bgRZAtlQXbaLCJpZ1GVz4EO9lo7CX3lWZxa28XBH3dz4oVijAqoW3MwBscVlX7+E0DNTr0R\\\\\\\/dxKdtgfTHz+q5hgJrnS0JuOZiv8BPKaA6qy9NW1Vulcu9+bRPDfEwESBJZJMfZ377s\\\\\\\/Z2PH91+9Z44Htzx9zndf64J8cqjyKziW1sdakeJ+trNRSld4KsL9fIt8CaB8y6BbkGjlmatbXSehBIneedAWRo\\\\\\\/Ok+d9bMvJkneVuUny5cauEmuYai3+zlyvp0DZW8KXJBOrtxE1NdqRcXh5r2O8NyMtVkrV3GPulubLiY5s4BEDtWeZ\\\\\\\/F+ybHOQvLlz5HyGnRuo\\\\\\\/jSkbE6F45fKAwbPV3SLMcRXMhrTuYjPfVCJZ3hY4LJoh9dn\\\\\\\/5h3kiyvGemjURXmF0DjAshCAE\\\\\\\/EuZnnPk4McWmGu09oLLbGXNS+68z9HLjK+AckuFiA0xgmaiBZ+lBTD8Il4GoCAjq\\\\\\\/4HmYzuZb6A0xtfhIPev9Id0fs+7R+wLFZRE21yIUXuB2iVNZXvMGHK7WoKSOGu8xgsw63pLNYUfeEvpAM\\\\\\\/IxuCYLhqla9qaoQvgRCgppi15I67cP5kAjIczbNRnrnpTXiNednr9\\\\\\\/skujfZ\\\\\\\/a84qgsekgN1bACxvzb7O12cKcfYobnXLORaVJvuY4\\\\\\\/dP2O6kw+lWT2zeTs0Y2bpgA5a1alLuppRO4D\\\\\\\/vinnNbKLlIRx6VWYIOmxcldkm6AtG24VlWvmyLaonOs4xC6LTwtFtVN31sjHZn\\\\\\\/LIepFlgdjGt25U9bEvkMIQzQvA5eITbhJenTSjfO8DMgrdZWurJLbcVGtkbrhYq9iF8u\\\\\\\/kgblmfMKDrNnm2O44gKJ0JWoJCwEM0eduaXgkUaPv4Du3NpYFWh7vkk31YC5tW9bCJnARagzTg0o0JRjjETW0kblmbIvd5w7wCbsMQvbGlJ7by01pGJRNW5abNI6hA7JxVbtbXBadXbELI192I\\\\\\\/K9UZW6f5QyBOA0JmZ+f2913tRcGs16NqlRfLu7PI45qqsVF5kdLh6wWoh0ogHRXbue\\\\\\\/kFbJOIP0H99JGa12fROj0de9qvrYKSPPX968HObB3j0D6U+Ndztn2imz6YgpVv6Pj\\\\\\\/nQlid3jfV7\\\\\\\/mHrpvDLQu77g6nX2IH9WZGOw00ag0Lr+yKveStjT17i3BeNe7+V2fPhEIK2NDE+u7C8y5VhfiVZWkvNlx+2ZwT2\\\\\\\/MxTGku90QhKBCnnv+LI5n9gsAxOhwFaCoBn\\\\\\\/PSgv\\\\\\\/qP5Iziu4Vfo9dk6kp6OxqD2KwFJhq8pzpSFgga31RGK3jBnZFCK9tGB01dfjwIcK1KU7RfirfnlVBzp8nCCFH244WCCRZGvW0GAloMKqpLt1znwHVEk+vG1Qu2TrCyUzGrvT5Hu\\\\\\\/yEH\\\\\\\/1wMu2\\\u0022,\\\u0022iv\\\u0022:\\\u0022YoGxVKu0D+UT912hlWrVsA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002293d3f6c669996de2c845819322db23bf9961a2197f9062b42598f252f9fe86a0\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022O6Di9+PkwyIzbopnarhf+csyiZwLafLGAse86t1kIWs9mFBlqsNnLxM64LGhizlmtK4MsR6V4U\\\\\\\/cKj0\\\\\\\/7w5t67eF\\\\\\\/WyODfxbj0P1qCNXwv0uUK4C7aAl4Hv2tO3eCzrCYIQtFrWbreL2WuVzV8L41dH6MTs8n72QzaX11Tln6bj3WVWS8rCH0806Ufg7nvxo85vqLeyYS8pJ8cPLeH2T\\\\\\\/3h3I3CnHj7h8og\\\\\\\/0SCafuMlqT7ngmpc\\\\\\\/QKiKNUKdiX7fLuXiqmwwdQtjg6KVtB2YydVtPuI9QTXve8NoLwX718Dnhv0OW1MVqkuUXkdqT1+6EY3bIymQCZBdzT1+4inYdrxk4IJwo\\\\\\\/lzqOn9sn+qMWMWNAIepUjAZ5kKdmKOtUuXNSeiUo236MltQdheyi2Y6dmawnMQzVxECOOb6lRuEAZaPxrqi0zsTknMS5VPSyavi1sZSeaR099YEstTxL1uPOoLpySwSBsQo5lIdV2JLirxxWn5oIOAbMrJVJ7b0LOVX+sWrO+WL9ZSiIbfPDDSSizXOO3iBG7kltgwQ0nHsbWd08cxKv\\\\\\\/IZj3cyy9H08DEWGAUTjgkkHuN8lLsQizo1RdaWjOQB2a4kOHYghUd+JDL0XJp8imkfLoSe+I1S02aobK3kcEp73+uWgL1eavnxN4cPDp3uFbBfUMJoG1+bbGT57qamS5nbt8q+rK1gOdl0oUgRdN3lJ2Zzn1xOiblfpd+reDF2zkA4TBlrKrJF9jcOhkyYAP9zyfursI\\\\\\\/BY2qkqSA3cubQzziA1Ll8PHUL4h5W9CGz7zTz7kpDzFaUM+DzB3n\\\\\\\/cz70N83u4WqbnBE+uF+GIMKl61krJBdTG0ePWwlvVK7aAYEfDi2XgadO7EvsevmMZUxylmb5fanRVzY2ygWgcg2LJ+ssumqaylPqQkixga7qHgxay7khUdM\\\\\\\/5RBwQs4M0y7FfWpGktsTGLtrhEj56tEynIiVILH0DcvBraH0bRVlJO87dp8F5HfbhtUpCiB\\\\\\\/Pb240MbKPgmV9bw7OnTyV+NRaiTNX2r0lm1VVGNudeihqp+8MET89j18UaYOZfKNotywPoGjmWsmdRPfsT4fwr3QGyi+vleQ6g5DTdcVo\\\\\\\/AgPnlRy2sNxmY7Tgci7Xg4dy+MBZP4phgMP2HaCvB+hd5sdN6cZsQKR3H2683WUS6psl0wP8LEFSuN+NZNoov8Q\\\\\\\/KaInzdkzBDLWCikkVwabAIAtatIVthv6F36gtVMVn4N+QBNIeBi1C9rY+oMGqZYt+GW2GSJGhYDRF6dA4HxsKKBQ+6N006d5awpNfCZdAfVZhXmvMQcsJtsemjIG0zmp5nGT0uHRQPykzPzjbpUT552ahQjz1GyFKAgUFyPxvBcnkkOJJYw8oldmyptXUsD4dlBtDBDYZ1nctrwPSWBvomQ5GRyQ0OawgkF4rnEZXWAN3xf2SyC\\\\\\\/wWkEufIAH7qTAzlCiUfyDnzHPRH2ke0HH7diszCIzcVgq2XffAROFcjjQGgP2bWvyPzv107vHZwLIea1bPFxzI8s8H6tUnNpY2yvZYww7izkokqY5\\\\\\\/GDRRS22WtvJnTq1BrMS\\\\\\\/MFMbnr7psldbudxrQsSalfX04s7aXzbDPxfkZ2Iph6tuMvevE6F68lsMGN6tY2\\\\\\\/2jlma5BcA1T8SUYasmkuSgxCHueFEc9qRCUVpQNbL9ZGxaJ19Bx1VPVkR50p0sWB2WALHQhqaAnrO0XGxtphDYaznHAK2UAKuV2K7KufKwJjNY87RBaac++hpWMxD7kUL20dIDzrrG\\\\\\\/hyuGPLx8J+PulUSEdLCw6920N7nv7MT9JCrI9Hbj5rgJzSm\\\\\\\/xi0zMPilbY7tW750oPv1Z1fxwHSVIAEM\\\\\\\/5jIEAydRewfctyEknTuluFfulXCMt6refiimtKXgTXxakabAoNzzH+ScqWymiP34L99aC0fJK1LsUiwcEP1wQWJ9qChyMzTFz9p41k9Z60mLnwAy2j84t7D7Zb1RyfhBPcx\\\\\\\/\\\\\\\/KRUqmyWCm1zJ6OCqT1q\\\\\\\/H2FvS4pw5MGoWd0xlk7LEqOrbdqqMo\\\u0022,\\\u0022iv\\\u0022:\\\u0022aVS2a7QDWn0JYgF+ulEXqg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00222d59f02dc0351bedffc828f3a945f390366089840216e693f858d302ed85c170\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-494618 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/11/3715" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-60KIiG3XTIPvSz08dnryqkxQ5joVhN7KtK6sc6bJ2WA" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/11/3715" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-EwtcPnaKK4FKWXYaeHQt9KCsxDFZqvRxYjXXkcWwkRs" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/11/3703" title="Activity-Dependent Regulation of the Proapoptotic BH3-Only Gene egl-1 in a Living Neuron Pair in Caenorhabditis elegans" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/11/3727" title="Pitfalls and Remedies for Cross Validation with Multi-trait Genomic Prediction Methods" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494618" id="node494618" data-pisa="ggg;9/11/3715" data-pisa-master="ggg;g3.119.400453" data-apath="/ggg/9/11/3715.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Three Quantitative Trait Loci Explain More than 60% of Variation for Chill Coma Recovery Time in a Natural Population of <em>Drosophila ananassae</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-4554-9390" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Annabella Königer</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0003-0811-8604" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Saad Arif</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0003-3621-736X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Sonja Grath</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3715-3725;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400453 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494618" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Annabella Königer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Division of Evolutionary Biology, Ludwig-Maximilians-Universität (LMU) München, 82152 Planegg-Martinsried, Germany and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Annabella%2BK%C3%B6niger%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=K%C3%B6niger%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAnnabella%2BK%25C3%25B6niger%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-4554-9390" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Annabella Königer</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Saad Arif </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Centre for Functional Genomics, Department of Biological and Medical Sciences, Oxford Brookes University, Oxford, OX3 0BP, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Saad%2BArif%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Arif%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASaad%2BArif%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-0811-8604" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Saad Arif</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Sonja Grath </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Division of Evolutionary Biology, Ludwig-Maximilians-Universität (LMU) München, 82152 Planegg-Martinsried, Germany and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sonja%2BGrath%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Grath%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASonja%2BGrath%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-3621-736X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Sonja Grath</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:grath@bio.lmu.de" class="" data-icon-position="" data-hide-link-title="0">grath@bio.lmu.de</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/9/11/3715" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:494618" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:494618/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/11/3715.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:494618" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:494618/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/11/3715.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:494618" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:494618/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/9/11/3715.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:494618" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:494618/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Ectothermic species such as insects are particularly vulnerable to climatic fluctuations. Nevertheless, many insects that evolved and diversified in the tropics have successfully colonized temperate regions all over the globe. To shed light on the genetic basis of cold tolerance in such species, we conducted a quantitative trait locus (QTL) mapping experiment for chill coma recovery time (CCRT) in <em>Drosophila ananassae</em>, a cosmopolitan species that has expanded its range from tropical to temperate regions. We created a mapping population of recombinant inbred advanced intercross lines (RIAILs) from two founder strains with diverging CCRT phenotypes. The RIAILs were phenotyped for their CCRT and, together with the founder strains, genotyped for polymorphic markers with double-digest restriction site-associated DNA (ddRAD) sequencing. Using a hierarchical mapping approach that combined standard interval mapping and a multiple-QTL model, we mapped three QTL which altogether explained 64% of the phenotypic variance. For two of the identified QTL, we found evidence of epistasis. To narrow down the list of cold tolerance candidate genes, we cross-referenced the QTL intervals with genes that we previously identified as differentially expressed in response to cold in <em>D. ananassae</em>, and with thermotolerance candidate genes of <em>D. melanogaster</em>. Among the 58 differentially expressed genes that were contained within the QTL, <em>GF15058</em> showed a significant interaction of the CCRT phenotype and gene expression. Further, we identified the orthologs of four <em>D. melanogaster</em> thermotolerance candidate genes, <a href="https://identifiers.org/bioentitylink/FB:FBgn0002868?doi=10.1534/g3.119.400453"><em>MtnA</em></a>, <a href="https://identifiers.org/bioentitylink/FB:FBgn0001316?doi=10.1534/g3.119.400453"><em>klarsicht</em></a>, <a href="https://identifiers.org/bioentitylink/FB:FBgn0038484?doi=10.1534/g3.119.400453"><em>CG5246</em></a> (<em>D.ana</em>/<em>GF17132</em>) and <a href="https://identifiers.org/bioentitylink/FB:FBgn0032699?doi=10.1534/g3.119.400453"><em>CG10383</em></a> (<em>D.ana</em>/<em>GF14829</em>) as candidates for cold tolerance in <em>D. ananassae</em>.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/qtl-mapping" class="hw-term hw-article-keyword hw-article-keyword-qtl-mapping" rel="nofollow">QTL mapping</a></li><li class="kwd"><a href="/keyword/rad-sequencing-0" class="hw-term hw-article-keyword hw-article-keyword-rad-sequencing" rel="nofollow">RAD-sequencing</a></li><li class="kwd"><a href="/keyword/cold-tolerance" class="hw-term hw-article-keyword hw-article-keyword-cold-tolerance" rel="nofollow">cold tolerance</a></li><li class="kwd"><a href="/keyword/chill-coma-recovery-time" class="hw-term hw-article-keyword hw-article-keyword-chill-coma-recovery-time" rel="nofollow">chill coma recovery time</a></li><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/drosophila-ananassae" class="hw-term hw-article-keyword hw-article-keyword-drosophila-ananassae" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Drosophila ananassae</em></a></li></ul><p id="p-3">Temperature is one of the major factors that influence the geographical distribution and abundance of ectothermic species. Physiological mechanisms to regulate body temperature are usually limited in ectotherms and resilience towards temperature extremes often determines the species fate upon climate change or range expansion. <em>Drosophila</em> spp. have successfully mastered such thermal challenges as they colonized temperate regions all over the globe and are now present on all of the earth’s continents except Antarctica (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Lachaise <em>et al.</em>, 1988</a>). By far the most prominent example of the genus is <em>Drosophila melanogaster</em>, which originated in sub-Saharan Africa, colonized temperate regions after the last glaciation about 15,000 years ago and nowadays has a worldwide distribution (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">David and Capy 1988</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Stephan and Li 2007</a>). In <em>D. melanogaster</em>, multiple candidate genes for cold tolerance have been identified (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Sinclair <em>et al.</em>, 2007</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Telonis-Scott <em>et al.</em>, 2009</a>; <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Zhang <em>et al.</em>, 2011</a>). Nevertheless, it remains challenging to link genotype and phenotype (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">von Heckel <em>et al.</em>, 2016</a>; <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Udaka <em>et al.</em>, 2010</a>) and it has been suggested that adaptation to local temperatures may have required simultaneous selection at multiple loci (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Norry <em>et al.</em>, 2008</a>; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Tucić 1979</a>).</p><p id="p-4">Previously, we examined the cold tolerance of <em>Drosophila ananassae</em> (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Königer and Grath 2018</a>), a species of tropical origin that belongs to the Melanogaster species group (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Clark <em>et al.</em>, 2007</a>). The home range of <em>D. ananassae</em> was dated back to an expansion of the South-East-Asian continental shelf (’Sundaland’) during the late Pleistocene about 18,000 years ago (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Das <em>et al.</em>, 2004</a>). Since then, the species expanded to temperate regions and has nowadays a quasi-cosmopolitan distribution (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Tobari 1993</a>).</p><p id="p-5">We measured cold tolerance by means of a test for chill coma recovery time (CCRT), which is defined as the time the flies need to stand on their legs after a cold-induced coma (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">David <em>et al.</em>, 1998</a>). There was substantial variation in CCRT among fly strains that were derived from a population originating from the ancestral species range in Bangkok, Thailand. Most strikingly, there were two groups of strains within this population: cold-sensitive and cold-tolerant strains. While the cold-tolerant strains recovered as fast as strains from a temperate population (Kathmandu, Nepal) originating from the derived species range, the cold-sensitive strains had a significantly longer CCRT (<a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Königer and Grath 2018</a>). The difference between these two phenotypic groups was large if compared to within-population variation in <em>D. melanogaster</em> (<a id="xref-ref-52-2" class="xref-bibr" href="#ref-52">von Heckel <em>et al.</em>, 2016</a>). Thus, the phenotypes of the Bangkok strains could suggest that there is standing genetic variation for cold tolerance within this population. In a previous transcriptome analysis of the cold-sensitive and cold-tolerant strains from Bangkok, 15% of all protein-coding genes responded to the cold shock by either up- or downregulation. However, only two genes with unknown function, <em>GF15058</em> and <em>GF14647</em>, changed their transcript levels in a phenotype-specific manner after the cold shock, <em>i.e.</em>, their expression regulation differed significantly between the cold-tolerant and cold-sensitive phenotype (<a id="xref-ref-24-3" class="xref-bibr" href="#ref-24">Königer and Grath 2018</a>). <em>D. ananassae</em> displays high levels of population structure (<a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Das <em>et al.</em>, 2004</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Schug <em>et al.</em>, 2007</a>) and the two distinct phenotypes indicate that there is genetic sub-structure within the Bangkok population as well. Further, the presence of genetic variation for cold tolerance in a tropical climate suggests pleiotropy that was maintained by balancing selection.</p><p id="p-6">Hence, the present study was undertaken to gain better insight into the genetic architecture of cold tolerance in the Bangkok population of <em>D. ananassae</em>. We performed a genome-wide scan for quantitative trait loci (QTL) influencing CCRT in a mapping population of recombinant inbred advanced intercross lines (RIAILs) which was generated from the most cold-tolerant strain and the most cold-sensitive strain of the Bangkok population. By combining double-digest restriction site-associated DNA sequencing (ddRAD) markers and a hierarchical mapping approach, we identified three QTL of large effect which altogether explain 64% of the phenotypic variance in the mapping population. We further combined the present results with lists of genes that are differentially expressed in response to the cold shock in <em>D. ananassae</em> (<a id="xref-ref-24-4" class="xref-bibr" href="#ref-24">Königer and Grath 2018</a>) and <em>D. melanogaster</em> (<a id="xref-ref-52-3" class="xref-bibr" href="#ref-52">von Heckel <em>et al.</em>, 2016</a>).</p><p id="p-7">Both species belong to the Melanogaster group and shared a common ancestor around 15-20 million years ago (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Clark <em>et al.</em>, 2007</a>). Our approach allowed us to narrow down the list of candidate genes for cold tolerance in <em>D. ananassae</em> and to point out potential commonalities between the two species.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Mapping population</h3><p id="p-8">All flies used in this study were raised on standard cornmeal molasses medium, at constant room temperature (22 ± 1°) and at a 14:10 h light:dark cycle (details of the food recipe can be found in <a id="xref-ref-24-5" class="xref-bibr" href="#ref-24">Königer and Grath 2018</a>). The two fly strains (Fast and Slow) that were used as founders for the mapping population were collected in 2002 in Bangkok, Thailand, and established as isofemale strains (<a id="xref-ref-13-3" class="xref-bibr" href="#ref-13">Das <em>et al.</em>, 2004</a>). The strains were not selected for cold tolerance and maintained at low density at a population size of approximately 500 individuals. The phenotypic difference in chill coma recovery time (CCRT) among the Fast and Slow founder strains was maintained for at least eight years (see results section and Supplementary file 1: Table S1 and Table S2). Recombinant Inbred Advanced Intercross Lines (RIAILs) were generated as follows (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>): two initial crosses between the two parental strains were set up (Fast males x Slow females and Slow males x Fast females). Individuals from both F1 generations were mixed and allowed to mate freely with each other. Up to generation F4, intercrossing was continued in the form of mass breedings. In generation F4, 360 mating pairs were set up in separate vials to allow for one more generation of intercrossing and to initiate the inbred strains. From generation F5, full-sibling inbreeding was carried out by mating brother-sister pairs for five subsequent generations. Throughout all generations (P - F10), the parents were removed before the offspring hatched to avoid back-crosses. From generation F10 on, RIAILs were kept at low density in 50 ml vials.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/11/3715/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Crossing scheme for the generation of the RIAIL mapping population. Drawings of single chromosome pairs were used as representatives for the full genome. An initial, reciprocal cross between the cold-sensitive founder strain Slow (shown in red) and the cold-tolerant founder strain Fast (shown in blue) was set up to generate the heterozygous F1 generation. Intercrosses were continued in the form of massbreedings until generation F4, where single mating pairs were picked to allow for one more generation of intercrossing and to initiate inbreeding. From generation F5, full-sibling inbreeding was carried out for five subsequent generations. Throughout all generations (P - F10), the parents were removed before the offspring hatched to avoid back-crosses." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2000374605" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Crossing scheme for the generation of the RIAIL mapping population. Drawings of single chromosome pairs were used as representatives for the full genome. An initial, reciprocal cross between the cold-sensitive founder strain Slow (shown in red) and the cold-tolerant founder strain Fast (shown in blue) was set up to generate the heterozygous F1 generation. Intercrosses were continued in the form of massbreedings until generation F4, where single mating pairs were picked to allow for one more generation of intercrossing and to initiate inbreeding. From generation F5, full-sibling inbreeding was carried out for five subsequent generations. Throughout all generations (P - F10), the parents were removed before the offspring hatched to avoid back-crosses.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/11/3715/F1.medium.gif" width="434" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/9/11/3715/F1.medium.gif" width="434" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/11/3715/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/11/3715/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494746" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-9" class="first-child">Crossing scheme for the generation of the RIAIL mapping population. Drawings of single chromosome pairs were used as representatives for the full genome. An initial, reciprocal cross between the cold-sensitive founder strain Slow (shown in red) and the cold-tolerant founder strain Fast (shown in blue) was set up to generate the heterozygous F1 generation. Intercrosses were continued in the form of massbreedings until generation F4, where single mating pairs were picked to allow for one more generation of intercrossing and to initiate inbreeding. From generation F5, full-sibling inbreeding was carried out for five subsequent generations. Throughout all generations (P - F10), the parents were removed before the offspring hatched to avoid back-crosses.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-3" class="subsection"><h3>Test for chill coma recovery time (CCRT)</h3><p id="p-10">CCRT was measured for flies of 4 – 6 days of age as described previously (<a id="xref-ref-24-6" class="xref-bibr" href="#ref-24">Königer and Grath 2018</a>). For the two founder strains Fast and Slow, CCRT was measured for males and females separately. For the RIAILs, only female flies were phenotyped. All female flies were collected and phenotyped as virgins. In brief, collection and sex-separation were carried out under light CO<sub>2</sub>-anesthesia, whereby ten flies from the same sex and strain were collected into a 50 ml vial containing 10 ml of cornmeal molasses medium. At the age of 4–6 days, the flies were transferred without anesthesia into new vials without food. For the cold shock, the vials were placed in an ice water bath (0 ± 0.5°) for exactly 3 h. Back at room temperature (22 ± 1°), CCRT was monitored in 2 min intervals for the duration of 90 min. Flies that were still not standing after 90 min were assigned a recovery time of 92 min. Flies that died during the experiment (&lt; 1%) were excluded from the analysis. On average, we tested 40 female individuals per RIAIL and 100 individuals per founder strain and sex. In order to analyze the relationship between fly strain and CCRT in the two founder strains, we fitted a generalized linear mixed effects model on square root transformed measures for CCRT with lme4 (version 1.1-21) (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Bates <em>et al.</em>, 2015</a>) in R (version 3.6.1) (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">R Core Team 2018</a>). Fly strain and sex were added as fixed effects and the intercept of each technical replicate was added as random effect to the model, while allowing for heterogeneous variances among the two fly lines. We then fitted two reduced models lacking either the sex or the line fixed effect. Significance of fixed effects was determined by likelihood ratio tests comparing the full model to each reduced model.</p></div><div id="sec-4" class="subsection"><h3>DNA extraction and sequencing</h3><p id="p-11">DNA was extracted from 94 RIAILs and the two parental strains with the DNeasy Blood &amp; Tissue Kit (QIAGEN, Hilden, Germany). For each strain, 10 virgin female individuals were pooled. DNA concentration and purity were assessed with a spectrophotometer (NanoDrop ND 1000, VWR International, Radnor, PA, USA). Library preparation and double-digest restriction site-associated DNA sequencing (ddRAD-seq) was carried out by an external sequencing service (ecogenics GmbH, Balgach, Switzerland) in the following way: DNA was double-digested with EcoRI and MseI and ligated to respective adapters comprising EcoRI and MseI restriction overhangs. Molecular identifier tags were added by polymerase chain reaction. The individual sample libraries were pooled, and the resulting library pools were size-selected for fragments between 500-600 bp with gel electrophoresis and extraction of the respective size range. The resulting size selected library pools were sequenced on a NextSeqTM 500 Sequencing System (Illumina, San Diego, CA), producing single-ended reads of 75bp length. Demultiplexing and trimming from Illumina adapter residuals was also carried out by the external service.</p></div><div id="sec-5" class="subsection"><h3>Marker catalog construction and data curation</h3><p id="p-12">The software pipeline Stacks (version 1.45) (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Catchen <em>et al.</em>, 2011</a>) was used to analyze the sequence data and to identify markers. First, to examine the quality of the sequence reads, the <em>process_radtags</em> program was run in Stacks, applying a sliding window size of 50% of the read length (<em>-w 0.5</em>) to filter out reads which drop below a 99% probability of being correct (Phred score &lt; 20) (<em>-s 20</em>). Second, the processed reads of each sample were mapped to the <em>D. ananassae</em> reference genome (FlyBase release 1.05 (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Attrill <em>et al.</em>, 2016</a>) with NextGenMap (version 0.5.0) (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Sedlazeck <em>et al.</em>, 2013</a>). Third, the mapped reads were converted to bam format, sorted and indexed with samtools (version 0.1.18) (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Li <em>et al.</em>, 2009</a>). Fourth, the <em>ref_map.pl</em> wrapper program was run in Stacks, which executes the Stacks core pipeline by running each of the Stacks components individually. In brief, <em>pstacks</em> assembled RAD loci for each sample, <em>cstacks</em> created a catalog of RAD loci from the two parental samples to create a set of all possible alleles expected in the mapping population and <em>sstacks</em> matched all RIAIL samples against the catalog. The <em>genotypes</em> program was executed last, applying automated corrections to the data (<em>-c</em>) to correct for false-negative heterozygote alleles. Only those loci which were present in at least 80% of the samples were exported (<em>-r 75</em>). Fifth, we applied additional corrections to the catalog by running the <em>rxstacks</em> program with the following filtering settings: non-biological haplotypes unlikely to occur in the population were pruned out (<em>–prune_haplo</em>), SNPs were recalled once sequencing errors were removed using the bounded SNP model (<em>–model_type_bounded</em>) with an error rate of 10% <em>(–bound_high 0.1</em>), and catalog loci with an average log likelihood less than –200 were removed (<em>–lnl_lim -200.00</em>). Sixth, <em>cstacks</em> and <em>sstacks</em> and <em>genotypes</em> (<em>-r 75</em>) were rerun to rebuild, match and export a new catalog with the filtered SNP calls. <em>Load_radtags.pl</em> and <em>index_radtags.pl</em> were used to upload and index the new catalog to a MySQL database. Seventh, a custom R script was used to remove markers with extreme values of residual heterozygosity within RIAILs, using cutoffs based on our inbreeding scheme (&gt; 15% and &lt; 35%) (Falconer and Mackay 1996) and to remove markers with an allele frequency drift &lt; 10% from further analysis. Eighth, the MySQL database was used to manually check the markers for errors. Additionally, we investigated the proportion of residual heterozygosity within the two founder strains by counting the occurrence of heterozygous sites in the haplotype models. A total of 1,400 markers were included in the downstream analysis.</p></div><div id="sec-6" class="subsection"><h3>Genetic map construction</h3><p id="p-13">Genetic map construction was conducted with R/qtl (version 1.42) (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Broman <em>et al.</em>, 2003</a>). The function <em>countXO</em> was used to remove seven RIAILs with &gt; 200 crossover events. One more RIAIL was removed due to a low number of genotyped markers (&lt; 700). The downstream analysis included 1,400 markers and 86 RIAIL-samples (Supplementary file 1: Table S4). Markers were partitioned into linkage groups based on a logarithm of the odds (LOD) score threshold of 8 and a maximum recombination frequency (rf) of 0.35, assuming a sequencing error rate of 1%. Map distances were calculated using the Haldane map function. As a sanity check, the functions <em>plotRF</em> and <em>checkAlleles</em> were used to test for potentially switched alleles and linkage groups were visually validated (based on rf and LOD scores).</p></div><div id="sec-7" class="subsection"><h3>Analysis of quantitative trait loci (QTL)</h3><p id="p-14">QTL mapping was conducted with R/qtl (version 1.42) (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Broman <em>et al.</em>, 2003</a>). Prior to mapping, the genotype probabilities between marker positions were calculated with the function <em>calc.genoprob</em> on a maximum grid size of 1 cM. To identify major QTL, standard interval mapping was performed using the Expectation Maximization (EM) algorithm as implemented with the <em>scanone</em> function. The results are expressed as a LOD score (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Sen and Churchill 2001</a>). Significance thresholds were calculated with 1,000 genome-wide permutations. The initial single-QTL scan was extended with a more complex, two-dimensional scan using Haley-Knott-Regression as implemented with the <em>scantwo</em> function. Significance thresholds were again calculated with 1,000 genome-wide permutations.</p><p id="p-15">To screen for additional QTL, estimate QTL effects and refine QTL positions, multiple-QTL mapping (MQM) was performed (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Arends <em>et al.</em>, 2010</a>). Here, missing genotypes were simulated from the joint distribution using a Hidden Markov model with 1,000 simulation replicates and an assumed error rate of 1% as implemented with the <em>sim.geno</em> function. The MQM model was identified with a forward selection/backward elimination search algorithm as implemented with the <em>stepwise</em> function, with the model choice criterion being penalized LOD scores. The penalties were derived on the basis of the significance permutations from the two-dimensional genome scan.</p><p id="p-16">To estimate the support interval for each identified QTL, an approximate 95% Bayesian credible interval was calculated as implemented by the <em>bayesint</em> function. Gene annotations for QTL intervals were downloaded from FlyBase (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Attrill <em>et al.</em>, 2016</a>) and screened for enriched GO terms and KEGG pathways with DAVID (version 6.8) (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Huang <em>et al.</em>, 2009</a>). Enrichment was calculated against the background of all annotated genes (<a id="xref-ref-2-3" class="xref-bibr" href="#ref-2">Attrill <em>et al.</em>, 2016</a>) using default settings (EASE-score of 0.1 after multiple testing correction according to Benjamini-Hochberg (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Benjamini and Hochberg 1995</a>)).</p><p id="p-17">In addition, we cross-referenced the QTL gene lists with lists of differentially expressed genes from a previously conducted transcriptome analysis, where we compared gene expression among cold-tolerant and cold-sensitive fly strains from the Bangkok population (including the two parental founder strains used in this study) in response to the 3 h cold shock at 0° (<a id="xref-ref-24-7" class="xref-bibr" href="#ref-24">Königer and Grath 2018</a>). Moreover, the transcriptome analysis also comprises lists of differentially expressed genes of cold-tolerant and cold-sensitive fly strains of <em>Drosophila melanogaster</em> in response to a cold shock (<a id="xref-ref-52-4" class="xref-bibr" href="#ref-52">von Heckel <em>et al.</em>, 2016</a>), allowing us to compare expression regulation of orthologous genes within the QTL regions among these two <em>Drosophila</em> species.</p></div><div id="sec-8" class="subsection"><h3>Data Availability</h3><p id="p-18">File S1 contains phenotyping data, DNA concentrations and information on the QTL model. File S2 contains detailed information on all three identified QTL. Sequence data have been deposited in NCBI’s Sequence Read Archive and are accessible through series accession number PRJNA544044. Code used to perform the data analysis is available upon request. Supplemental material available at FigShare: <a href="https://doi.org/10.25387/g3.8288591">https://doi.org/10.25387/g3.8288591</a>.</p></div></div><div class="section results" id="sec-9"><h2 class="">Results</h2><div id="sec-10" class="subsection"><h3>Chill coma recovery time (CCRT) Phenotype</h3><p id="p-19">The average CCRT of the cold-tolerant founder strain (Fast) was 29.29 min (standard deviation, SD, 9.90) for females and 30.13 min (SD 8.65) for males. CCRT of the cold-sensitive founder strain (Slow) was 63.70 min (SD 20.70) for females and 53.92 min (SD 18.40) for males (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>, Supplementary file 1: Table S1). The effect of the founder strain on CCRT was significant (Likelihood ratio test, <em>P</em>-value &lt; 0.0001). The effect of sex on CCRT was not significant (Likelihood ratio test, <em>P</em>-value = 0.2522) (see Supplementary file 1: Table S3). The average CCRT of the RIAILs ranged from 27.60 min to 83.03 min (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>, Supplementary file 1: Table S4).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/11/3715/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Chill coma recovery time (CCRT) of 4-6 day old flies of two strains of D. ananassae from Bangkok (Thailand) that were used as founder strains for the mapping population. The effect of the founder strain on CCRT was significant (Likelihood ratio test, P-value &lt; 0.0001). The effect of sex on CCRT was not significant (Likelihood ratio test, P-value = 0.2522)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2000374605" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Chill coma recovery time (CCRT) of 4-6 day old flies of two strains of &lt;em&gt;D. ananassae&lt;/em&gt; from Bangkok (Thailand) that were used as founder strains for the mapping population. The effect of the founder strain on CCRT was significant (Likelihood ratio test, &lt;em&gt;P&lt;/em&gt;-value &lt; 0.0001). The effect of sex on CCRT was not significant (Likelihood ratio test, &lt;em&gt;P&lt;/em&gt;-value = 0.2522).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/11/3715/F2.medium.gif" width="287" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/9/11/3715/F2.medium.gif" width="287" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/11/3715/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/11/3715/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494832" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-20" class="first-child">Chill coma recovery time (CCRT) of 4-6 day old flies of two strains of <em>D. ananassae</em> from Bangkok (Thailand) that were used as founder strains for the mapping population. The effect of the founder strain on CCRT was significant (Likelihood ratio test, <em>P</em>-value &lt; 0.0001). The effect of sex on CCRT was not significant (Likelihood ratio test, <em>P</em>-value = 0.2522).</p><div class="sb-div caption-clear"></div></div></div><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/11/3715/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Chill coma recovery time (CCRT) of 4 - 6 day old virgin female flies of the 94 recombinant inbred advanced intercross lines (RIAILs) and the two founder strains. CCRT of the RIAILs is displayed with white bars. CCRT of the two founder strains is displayed in blue (Fast founder) and pink (Slow founder) (see also Figure 2). The RIAILs were numbered in ascending order according to their average CCRT. The fact that recovery times of 19 RIAILs exceeded the recovery time of the Slow founder strain suggests non-additive effects at causal loci." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2000374605" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Chill coma recovery time (CCRT) of 4 - 6 day old virgin female flies of the 94 recombinant inbred advanced intercross lines (RIAILs) and the two founder strains. CCRT of the RIAILs is displayed with white bars. CCRT of the two founder strains is displayed in blue (Fast founder) and pink (Slow founder) (see also Figure 2). The RIAILs were numbered in ascending order according to their average CCRT. The fact that recovery times of 19 RIAILs exceeded the recovery time of the Slow founder strain suggests non-additive effects at causal loci.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/11/3715/F3.medium.gif" width="440" height="315"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/9/11/3715/F3.medium.gif" width="440" height="315"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/11/3715/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/11/3715/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494758" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-21" class="first-child">Chill coma recovery time (CCRT) of 4 - 6 day old virgin female flies of the 94 recombinant inbred advanced intercross lines (RIAILs) and the two founder strains. CCRT of the RIAILs is displayed with white bars. CCRT of the two founder strains is displayed in blue (Fast founder) and pink (Slow founder) (see also <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>). The RIAILs were numbered in ascending order according to their average CCRT. The fact that recovery times of 19 RIAILs exceeded the recovery time of the Slow founder strain suggests non-additive effects at causal loci.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-11" class="subsection"><h3>Sequencing and genetic map</h3><p id="p-22">In total, we obtained 331,867,133 sequence reads with an average of 3,281,450 reads per sample. 0.6% of the total reads (2,074,057) failed the Stacks <em>process_radtags</em> quality check and were excluded from the analysis. In each of the samples, &gt; 94% of all reads mapped to the <em>D. ananassae</em> reference genome. The amount of residual heterozygosity within the founder strain sequences was &lt;0.5% (Supplementary file 1: Table S9). The Stacks core pipeline matched 5,468 markers to the initial catalog. After additional corrections with the <em>rxstacks</em> program, 3,092 markers remained. From this new catalog, 691 markers were excluded due to extreme values of heterozygosity (&gt;35% or &lt; 15%), 735 markers were excluded due to high levels of allele frequency drift (&gt; 10%) and 266 markers were excluded that fell into both categories. Thus, after all filtering steps, a total of 1,400 markers and 86 RIAILs were used for genetic map construction. The markers were partitioned into eight linkage groups (Supplementary file 1: Table S6). The total map length was 962.0 cM, with an average marker spacing of 0.7 cM and a maximum marker spacing of 55.5 cM (Figure 5). Across all samples, 91.6% of the genotypes were available of which 37.4% were homozygous for the cold-tolerant (Fast) allele (FF), 27.9% were heterozygous (FS) and 34.7% were homozygous for the cold-sensitive (Slow) allele (SS).</p></div><div id="sec-12" class="subsection"><h3>One- and two-dimensional genome scans</h3><p id="p-23">Interval mapping in the context of a single-QTL model revealed two major areas with LOD peaks which exceeded the permuted 5% significance level (LOD 3.53), one on scaffold 13337 (QTL1) and one on scaffold 13340 (QTL2) (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). The highest peak on scaffold 13337 was at 6.08 cM (LOD 5.80) and the highest peak on scaffold 13340 was at 80.05 cM (LOD 4.08).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/11/3715/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="LOD-curves obtained with standard interval mapping reveal two significant QTL, QTL1 on scaffold 13337 and QTL2 on scaffold 13340. Significance thresholds (dotted lines) were calculated with 1,000 genome-wide permutations. The short vertical lines on the X-axis correspond to the marker positions." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2000374605" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;LOD-curves obtained with standard interval mapping reveal two significant QTL, QTL1 on scaffold 13337 and QTL2 on scaffold 13340. Significance thresholds (dotted lines) were calculated with 1,000 genome-wide permutations. The short vertical lines on the X-axis correspond to the marker positions.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/11/3715/F4.medium.gif" width="440" height="288"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/9/11/3715/F4.medium.gif" width="440" height="288"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/11/3715/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/11/3715/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494742" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-24" class="first-child">LOD-curves obtained with standard interval mapping reveal two significant QTL, QTL1 on scaffold 13337 and QTL2 on scaffold 13340. Significance thresholds (dotted lines) were calculated with 1,000 genome-wide permutations. The short vertical lines on the X-axis correspond to the marker positions.</p><div class="sb-div caption-clear"></div></div></div><p id="p-25">The next step was to extend the initial, single-QTL scan with a two-dimensional scan, where we compared two possible models: the full (epistatic) model (H<sub>f1</sub>) which allowed for the possibility of a second QTL and interactions among QTL was compared to the additive model (H<sub>a1</sub>) which allowed for the possibility of a second QTL without interaction. Both the full and the additive model reached maximum LOD scores at the same positions, 7.08 cM on scaffold 13337 and 30.1 cM on scaffold 13340 (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). In comparison to the single-QTL model, we found supporting evidence for the presence of a second QTL under the additive model (lodd.av1 <em>P</em>-value = 0.006), but not under the full model (lod.fv1 <em>P</em>-value = 0.668). There is no evidence for interaction among the two loci (lod.int <em>P</em>-value = 1).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/494628/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/494628/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Results of the two-dimensional genome scan</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-13" class="subsection"><h3>Multilpe-QTL model</h3><p id="p-32">In order to identify possible additional QTL of moderate effect, refine QTL positions, separate linked loci and to estimate QTL effects, we applied a forward selection/backward elimination algorithm with penalized LOD scores and identified a model with three main terms and one interaction term. The overall fit of the model had a LOD score of 19.26 and explained 64.34% of the phenotypic variance (Figure S2, Supplementary file 1: Table S7). In comparison to the one- and two-dimensional genome-scans, there was an additional locus on scaffold 12916 at position 16.7 cM (QTL3) which interacted with one of the previously identified loci, on scaffold 13340 (QTL2) (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>, <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figures 5</a>, <a id="xref-fig-6-1" class="xref-fig" href="#F6">6</a> and Figure S3).</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/494843/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/494843/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">QTL confidence intervals and estimated effects</span><div class="sb-div caption-clear"></div></div></div><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/11/3715/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genetic map with QTL positions as refined with the multiple-QTL model. X-axis = genomic scaffolds. Y-axis = genetic distances in centiMorgan (cM) for markers (short horizontal lines)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2000374605" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Genetic map with QTL positions as refined with the multiple-QTL model. X-axis = genomic scaffolds. Y-axis = genetic distances in centiMorgan (cM) for markers (short horizontal lines).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/11/3715/F5.medium.gif" width="440" height="282"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/9/11/3715/F5.medium.gif" width="440" height="282"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/11/3715/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/11/3715/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494698" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-35" class="first-child">Genetic map with QTL positions as refined with the multiple-QTL model. X-axis = genomic scaffolds. Y-axis = genetic distances in centiMorgan (cM) for markers (short horizontal lines).</p><div class="sb-div caption-clear"></div></div></div><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/11/3715/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Interaction of QTL2 on scaffold 13340 and QTL3 on scaffold 12916. X-axis = genotypes for QTL2. The genotypes for QTL3 are represented by lines in different colors. Error bars are plotted at +/&#x2212; 1 SE. F = cold-tolerant parental allele (fast CCR), S = cold-sensitive parental allele (slow CCR)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2000374605" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Interaction of QTL2 on scaffold 13340 and QTL3 on scaffold 12916. X-axis = genotypes for QTL2. The genotypes for QTL3 are represented by lines in different colors. Error bars are plotted at +/&#x2212; 1 SE. F = cold-tolerant parental allele (fast CCR), S = cold-sensitive parental allele (slow CCR).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/11/3715/F6.medium.gif" width="440" height="284"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/9/11/3715/F6.medium.gif" width="440" height="284"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/11/3715/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/11/3715/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494782" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-36" class="first-child">Interaction of QTL2 on scaffold 13340 and QTL3 on scaffold 12916. X-axis = genotypes for QTL2. The genotypes for QTL3 are represented by lines in different colors. Error bars are plotted at +/− 1 SE. F = cold-tolerant parental allele (fast CCR), S = cold-sensitive parental allele (slow CCR).</p><div class="sb-div caption-clear"></div></div></div><p id="p-37">QTL effects were estimated for additivity ((SS-FF)/2) and deviation from dominance ((FS-(FF+SS)/2), where F denotes the cold-tolerant Fast allele and S denotes the cold-sensitive Slow allele (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>, Supplementary file 1: Table S8). QTL3 on scaffold 12916 was a transgressive QTL, as the cold-tolerant allele was associated with having a more cold-sensitive phenotype (longer CCRT), resulting in a negative effect size (Figure S1C). For QTL1, the estimated additive effect was positive while the estimated dominance effect was negative. RIAILs homozygous for the cold-tolerant allele had the most cold-tolerant phenotype, RIAILs homozygous for the cold-sensitive allele had the least cold-tolerant phenotype and heterozygote RIAILs had an intermediate phenotype (Figure S1A). The effect estimates for QTL2 went in the same direction as for QTL1. Here, however, the heterozygous phenotype was associated with the most cold-tolerant phenotype (Figure S1B). The more complex relationships of additive and dominance effects for the interaction of QTL3 and QTL2 can be understood best by plotting the interaction of the phenotype and the genotype at both marker positions (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6</a>).</p><p id="p-38">As revealed by the interaction plot (<a id="xref-fig-6-3" class="xref-fig" href="#F6">Figure 6</a>), RIAILs homozygous for the cold-sensitive (S) allele at both QTL also had the most cold-sensitive phenotype. The most cold-tolerant phenotype, was reached by those RIAILs which were homozygous for the cold-tolerant allele at QTL2 but homozygous for the cold-sensitive allele at (the transgressive) QTL3. Interestingly, cold tolerance of RIAILs which were heterozygous at QTL3 seemed to be independent from their genotype at QTL2.</p><p id="p-39">The results of a drop-one-term at a time ANOVA indicated strong evidence for all three loci and the interaction of QTL2 and QTL3. For each QTL, the model with the QTL of interest at that particular position was compared to the model with the QTL of interest omitted, while all other QTL positions were fixed at their maximum likelihood estimates (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>, Figure S3).</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/494798/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/494798/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Summary table for the drop one term ANOVA</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-14" class="subsection"><h3>Candidate gene meta analysis</h3><p id="p-41">All three QTL together contained 259 protein-coding genes (<a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>, Supplementary file 2: Tables S1, S2, S5). Among them were 58 differentially expressed genes (<a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>), which was significantly more than we would expect by chance (Fisher’s exact test, <em>P</em> = 0.02867).</p><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/494759/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/494759/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">Cold tolerance candidate genes within QTL regions</span><div class="sb-div caption-clear"></div></div></div><p id="p-44">QTL1 spanned 140 kb and contained eleven protein coding genes (Supplementary file 2: Table S1). There was no enrichment of KEGG pathways or GO terms. However, three of the eleven genes were previously identified to be differentially expressed in response to a cold shock (Supplementary file 2: Table S1, (<a id="xref-ref-24-9" class="xref-bibr" href="#ref-24">Königer and Grath 2018</a>). Two of them, <em>GF24884</em> (ortholog of <a href="https://identifiers.org/bioentitylink/FB:FBgn0035101?doi=10.1534/g3.119.400453"><em>p130CAS</em></a>) and <em>GF24880</em> (ortholog of <em>Phosphoinositide-dependent kinase 1</em>), were upregulated in both phenotypes after the cold shock and one of them, <em>GF24896</em> (ortholog of <a href="https://identifiers.org/bioentitylink/FB:FBgn0001316?doi=10.1534/g3.119.400453"><em>klarsicht</em></a>), was exclusively upregulated in the cold-tolerant phenotype only. Klarsicht was previously reported as upregulated in cold-acclimated flies of <em>D. melanogaster</em> (<a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">MacMillan <em>et al.</em>, 2016</a>).</p><p id="p-45">QTL2 spanned 1.0 Mb and contained 138 protein-coding genes which were enriched in one molecular function, “serine-type endopeptidase activity” (GO:0004252) and one biological process, “intracellular cholesterol transport” (GO:0032367) (Supplementary file 2: Table S3). Out of the 138 genes, 26 were previously identified as differentially expressed in response to a cold shock. Among them, nine genes were upregulated and five genes were downregulated in both phenotypes (see Supplementary file 2: Table S2, and (<a id="xref-ref-24-10" class="xref-bibr" href="#ref-24">Königer and Grath 2018</a>)). In the cold-tolerant phenotype, one gene, <em>GF17809</em> (ortholog of <a href="https://identifiers.org/bioentitylink/FB:FBgn0038893?doi=10.1534/g3.119.400453"><em>Archease</em></a>), was exclusively upregulated and one gene, <em>GF17856</em> (ortholog of <em>Niemann-Pick type C-2c</em>), was exclusively downregulated. In the cold-sensitive phenotype, one gene, <em>GF17176</em> (ortholog of <em>aluminum tubes</em>), was exclusively upregulated and nine genes were exclusively downregulated (see Supplementary file 2: Table S2, and (<a id="xref-ref-24-11" class="xref-bibr" href="#ref-24">Königer and Grath 2018</a>).</p><p id="p-46">Nine genes drove the enrichment in the GO category “serine-type endopeptidase activity” (see Supplementary file 2: Table S3). All of them were located in the downstream region of QTL2 at 6,515,565 - 6,527,729 bp and adjacent to one another (<a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7</a>). Seven of these genes were also differentially expressed in response to cold shock. Among them was <em>GF17132</em>, which was upregulated in both phenotypes and its ortholog in <em>D. melanogaster</em> showed a significant interaction of phenotype and cold shock (Supplementary file 2: Table S2, <a id="xref-ref-52-6" class="xref-bibr" href="#ref-52">von Heckel <em>et al.</em>, 2016</a>). QTL2 also contained the gene <em>Metallothionein A</em> (<a href="https://identifiers.org/bioentitylink/FB:FBgn0002868?doi=10.1534/g3.119.400453"><em>MtnA</em></a>) which caught our attention because it is involved in metal ion homeostasis and in its <em>D. melanogaster</em> ortholog, an InDel polymorphism is associated with local adaptation to oxidative stress upon migration out of Sub-Saharan Africa into Europe (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Catalán <em>et al.</em>, 2016</a>). <a href="https://identifiers.org/bioentitylink/FB:FBgn0002868?doi=10.1534/g3.119.400453"><em>MtnA</em></a> was downregulated in response to cold in <em>D. melanogaster</em> (<a id="xref-ref-52-7" class="xref-bibr" href="#ref-52">von Heckel <em>et al.</em>, 2016</a>) but not in <em>D. ananassae</em> (<a id="xref-ref-24-12" class="xref-bibr" href="#ref-24">Königer and Grath 2018</a>).</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/11/3715/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Schematic illustration of a genomic region within QTL2 that contains nine genes of the enriched GO category &#x201C;serine-type endopeptidase activity&#x201D; (see also Supplementary file 2: Table S2 and S3). Genes were differentially expressed in response to the cold shock in either the cold-sensitive (slow) phenotype alone (genes shown in pink color) or in both phenotypes, cold-sensitive and cold-tolerant (fast) (genes shown in blue and pink color). Genes that were not differentially expressed are shown in gray. Gene lengths and distances between genes are not drawn to scale." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2000374605" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Schematic illustration of a genomic region within QTL2 that contains nine genes of the enriched GO category &#x201C;serine-type endopeptidase activity&#x201D; (see also Supplementary file 2: Table S2 and S3). Genes were differentially expressed in response to the cold shock in either the cold-sensitive (slow) phenotype alone (genes shown in pink color) or in both phenotypes, cold-sensitive and cold-tolerant (fast) (genes shown in blue and pink color). Genes that were not differentially expressed are shown in gray. Gene lengths and distances between genes are not drawn to scale.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/11/3715/F7.medium.gif" width="440" height="103"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/9/11/3715/F7.medium.gif" width="440" height="103"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/11/3715/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/11/3715/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494649" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-47" class="first-child">Schematic illustration of a genomic region within QTL2 that contains nine genes of the enriched GO category “serine-type endopeptidase activity” (see also Supplementary file 2: Table S2 and S3). Genes were differentially expressed in response to the cold shock in either the cold-sensitive (slow) phenotype alone (genes shown in pink color) or in both phenotypes, cold-sensitive and cold-tolerant (fast) (genes shown in blue and pink color). Genes that were not differentially expressed are shown in gray. Gene lengths and distances between genes are not drawn to scale.</p><div class="sb-div caption-clear"></div></div></div><p id="p-48">QTL3 spanned 1.2 Mb and contained 110 protein coding genes which were enriched in three molecular functions: “sequence-specific DNA binding” (GO:0043565), “ATPase activity” (GO:0016887) and “phosphotransferase activity, alcohol group as acceptor” (GO:0016773) and one KEGG pathway: “Hippo signaling pathway – fly” (dan04391) (Supplementary file 2: Table S6). Out of the 110 genes, 29 were previously identified as differentially expressed in response to a cold shock (<a id="xref-table-wrap-4-2" class="xref-table" href="#T4">Table 4</a>). Among them, 12 genes were upregulated and seven genes were downregulated in both phenotypic groups, cold-tolerant and cold-sensitive (see Supplementary file 2: Table S5, (<a id="xref-ref-24-13" class="xref-bibr" href="#ref-24">Königer and Grath 2018</a>)). In the cold-tolerant phenotype, two genes, <em>GF15043</em> (ortholog of <a href="https://identifiers.org/bioentitylink/FB:FBgn0051974?doi=10.1534/g3.119.400453"><em>CG31974</em></a>) and <em>GF14846</em> (ortholog of <a href="https://identifiers.org/bioentitylink/FB:FBgn0000166?doi=10.1534/g3.119.400453"><em>bicoid</em></a><em> stability factor</em>), were exclusively upregulated and two genes, <em>GF15020</em> (ortholog of <em>ABC transporter expressed in trachea</em>) and <em>GF14865</em> (ortholog of <a href="https://identifiers.org/bioentitylink/FB:FBgn0031224?doi=10.1534/g3.119.400453"><em>CG11454</em></a>), were exclusively downregulated. In the cold-sensitive phenotype, five genes were exclusively downregulated but there were no exclusively upregulated genes. One of the five downregulated genes was <em>GF15058</em> (ortholog of <a href="https://identifiers.org/bioentitylink/FB:FBgn0032684?doi=10.1534/g3.119.400453"><em>CG10178</em></a>), which was one out of two genes with a significant interaction of phenotype and cold shock. The function of <em>GF15058</em> is unknown but it is predicted to have UDP-glycosyl-transferase-activity (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Marchler-Bauer <em>et al.</em>, 2015</a>).</p></div></div><div class="section discussion" id="sec-15"><h2 class="">Discussion</h2><p id="p-49">We used a panel of 86 recombinant inbred advanced intercross lines (RIAILs) and 1,400 ddRAD markers to map QTL that underlie natural variation in cold tolerance among two fly strains of <em>D. ananassae</em> from a population in Bangkok, Thailand. The recovery time segregated significantly between the two founder strains. CCRT in the cold-sensitive strain was about twice as high as in the cold-tolerant strain. The three identified QTL for CCRT explain as much as 64% of the variance in the phenotype. This proportion is equal to a previous mapping experiment for CCRT in <em>D. melanogaster</em>, in which three QTL explained 64% of the variance for CCRT in an intercontinental set of recombinant inbred lines (<a id="xref-ref-32-3" class="xref-bibr" href="#ref-32">Norry <em>et al.</em>, 2008</a>). The founder strains for this mapping population were sampled from Denmark and Australia and thus from two geographically different thermal environments. Another study (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Morgan and Mackay 2006</a>) also identified three QTL for CCRT in <em>D. melanogaster</em> in a set of recombinant inbred lines derived from two laboratory strains that differed significantly for the phenotype. In this mapping population, the three loci explained 25% of the phenotypic variance for CCRT. While two of the reported QTL for CCRT in <em>D. melanogaster</em> co-localized across these two studies, none of the reported candidate genes co-localize with the QTL intervals in <em>D. ananassae</em> (this study).</p><p id="p-50">Compared to sequencing of pooled samples (Pool-sequencing), RAD-based approaches come at the cost of marker density, especially in crossing designs with low genetic differentiation between the founder strains and low levels of linkage disequilibrium (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Futschik and Schlötterer 2010</a>). Thus, to increase the mapping resolution and to expand the genetic map, we generated a mapping population in which five generations of intercrosses allowed for a sufficient number of crossover events (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Pollard 2012</a>). Subsequently, we used stringent filtering cutoffs to account for potential sequencing errors. We thereby excluded a relatively high number of markers which did not meet our criteria for expected levels of hererozygosity and allele frequency drift. While this step certainly increased the robustness of the identified loci, estimates for deviations from dominance may be biased by segregation distortion or inbreeding effects. Eventually, excluding these markers came at the cost of chromosomal coverage, as many smaller genomic scaffolds were excluded from the analysis at this step. It is therefore possible that our results do not cover all potential QTL. However, the reduction of genome complexity that results from RAD-sequencing has two major benefits. First, it is more cost-effective than whole-genome sequencing of individual samples, allowing for a larger number of samples to be analyzed and consequently for greater statistical power to detect QTL. Second, it is more accurate than whole-genome Pool-sequencing (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Catchen <em>et al.</em>, 2017</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Cutler and Jensen 2010</a>).</p><p id="p-51">Populations of <em>D. ananassae</em> segregate for several polymorphic inversions, the frequencies of which follow latitudinal clines (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Singh and Singh 2007</a>). Moreover, chromosomal inversions appear to be temperature-sensitive in <em>D. ananassae</em> (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Dasmohapatra <em>et al.</em>, 1982</a>), which has also been demonstrated for <em>D. melanogaster</em> (<a id="xref-ref-32-4" class="xref-bibr" href="#ref-32">Norry <em>et al.</em>, 2008</a>). A cold tolerance factor that is located within an inversion could rise quickly in frequency upon seasonal fluctuations in temperature or upon local adaptation to temperate habitats by suppressed recombination in heterozygous karyotypes (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Kirkpatrick and Barton 2006</a>) - and thus confound QTL mapping. However, the CCRT phenotypes of the mapping population were distributed on a continuum (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>), which speaks against the influence of a major inversion. Furthermore, there was no apparent excess of residual heterozygosity within the three QTL scaffolds in the founder genomes. Generally, detecting inversions from short-read ddRAD-sequences only is difficult. Therefore, we can neither confirm nor deny the presence of inversions in the tested strains.</p><p id="p-52">Additionally, it needs to be noted that, in general, QTL confidence intervals should be considered as support regions rather than absolute boundaries (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Broman and Sen 2009</a>) and the causal genetic variants may be located anywhere within these intervals. However, combining the identified intervals with a previous transcriptome analysis in <em>D. ananassae</em> (<a id="xref-ref-24-14" class="xref-bibr" href="#ref-24">Königer and Grath 2018</a>) allowed us to narrow down the list of potentially causal genes.</p><p id="p-53">The identification of candidate genes for cold tolerance has been extensively addressed in <em>D. melanogaster</em>. Like <em>D. ananassae</em>, <em>D. melanogaster</em> is of tropical origin (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">David <em>et al.</em>, 2007</a>), but its thermal range (12-32°) (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Moreteau <em>et al.</em>, 1997</a>) is broader than the thermal range of <em>D. ananassae</em> (16-32°) (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Morin <em>et al.</em>, 1997</a>). Since both species adapted independently to temperate regions, it is unclear if the same genes that underly cold tolerance in <em>D. melanogaster</em> are also of importance in <em>D. ananassae</em>. Hence, we screened the QTL intervals in <em>D. ananassae</em> for cold tolerance candidate genes in <em>D. melanogaster</em> (<a id="xref-ref-52-8" class="xref-bibr" href="#ref-52">von Heckel <em>et al.</em>, 2016</a>; <a id="xref-ref-27-3" class="xref-bibr" href="#ref-27">MacMillan <em>et al.</em>, 2016</a>; <a id="xref-ref-32-5" class="xref-bibr" href="#ref-32">Norry <em>et al.</em>, 2008</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Ramnarine <em>et al.</em>, 2019</a>) to highlight commonalities and differences among the two species.</p><p>From the combined data, we identified three types of candidates:</p><ol class="list-romanupper " id="list-1"><li id="list-item-1"><p id="p-55">The expression profile of <em>GF15058</em> (<em>D.mel</em>/<a href="https://identifiers.org/bioentitylink/FB:FBgn0032684?doi=10.1534/g3.119.400453"><em>CG10178</em></a>) in QTL3 is directly associated with a difference in the CCRT phenotype in <em>D. ananassae</em>. <em>GF15058</em> was one out of two genes that responded to the cold shock in a phenotype-specific way (<a id="xref-ref-24-15" class="xref-bibr" href="#ref-24">Königer and Grath 2018</a>). Its function was inferred from electronic annotation to be uridine diphosphate (UDP) glycosyltransferase activity. UDP-glycosyltransferases (UGTs) are membrane-bound enzymes that are located in the endoplasmatic reticulum and catalyze the addition of a glycosyl group from a uridine triphosphate (UTP) sugar to a small hydrophobic molecule. Therefore, UGTs play an essential role in maintaining homeostatic function and detoxification and are known as major members of phase II drug metabolizing enzymes (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bock 2016</a>). The cold shock led to a downregulation of <em>GF15058</em> in the Slow strains but not in the Fast strains. However, the Fast genotype at QTL3 is transgressive, <em>i.e.</em>, it increases CCRT. Thus, if <em>GF15058</em> was indeed one of the causal factors, our results suggest that keeping transcript abundance at a constant level after the cold shock is so costly for the organism that it slows down recovery.</p></li><li id="list-item-2"><p id="p-56">The expression profile of <em>GF17132</em> (<em>D.mel</em>/<a href="https://identifiers.org/bioentitylink/FB:FBgn0038484?doi=10.1534/g3.119.400453"><em>CG5246</em></a>) in QTL2 is directly associated with a difference in the CCRT phenotype in <em>D. melanogaster</em>, where it showed a significant interaction of phenotype and cold shock (<a id="xref-ref-52-9" class="xref-bibr" href="#ref-52">von Heckel <em>et al.</em>, 2016</a>). It was also differentially expressed in response to the cold shock in <em>D. ananassae</em> (<a id="xref-ref-24-16" class="xref-bibr" href="#ref-24">Königer and Grath 2018</a>). Moreover, <em>GF17132</em> belongs to a cluster of genes that code for serine peptidases in QTL2 (<a id="xref-fig-7-2" class="xref-fig" href="#F7">Figure 7</a>). Serine peptidases are involved in proteolysis, <em>i.e.</em>, they catalyze the hydrolysis of peptide bonds (<a id="xref-ref-2-4" class="xref-bibr" href="#ref-2">Attrill <em>et al.</em>, 2016</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Ross <em>et al.</em>, 2003</a>). This process plays a central role in the immune response of insects (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">De Gregorio <em>et al.</em>, 2001</a>) and serine proteases were suggested previously to be involved in the cold stress response as well (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Vermeulen <em>et al.</em>, 2013</a>).</p></li><li id="list-item-3"><p id="p-57">We identified three more genes that have been associated with thermotolerance in experiments other than the transcriptome analyses: <a href="https://identifiers.org/bioentitylink/FB:FBgn0002868?doi=10.1534/g3.119.400453"><em>MtnA</em></a> (<em>D.mel</em>/<a href="https://identifiers.org/bioentitylink/FB:FBgn0002868?doi=10.1534/g3.119.400453"><em>MtnA</em></a>), <em>GF24896</em> (<em>D.mel/klarsicht)</em> and <em>GF14829</em> (<em>D.mel/</em><a href="https://identifiers.org/bioentitylink/FB:FBgn0032699?doi=10.1534/g3.119.400453"><em>CG10383</em></a>).</p></li></ol><p>The gene <a href="https://identifiers.org/bioentitylink/FB:FBgn0002868?doi=10.1534/g3.119.400453"><em>MtnA</em></a> in QTL2 codes for metallothionein A which promotes resistance to oxidative stress. It binds heavy metals and neutralizes reactive oxygen and nitrogen species (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Ruttkay-Nedecky <em>et al.</em>, 2013</a>). Exposure to cold leads to an increased abundance of free radicals, thereby inducing oxidative stress (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Williams <em>et al.</em>, 2014</a>). In <em>D. melanogaster</em>, a 49 bp deletion in the 3′UTR of <a href="https://identifiers.org/bioentitylink/FB:FBgn0002868?doi=10.1534/g3.119.400453"><em>MtnA</em></a> is associated with its transcriptional upregulation and with increased tolerance to oxidative stress (<a id="xref-ref-8-3" class="xref-bibr" href="#ref-8">Catalán <em>et al.</em>, 2016</a>). The frequency of this polymorphism in natural populations follows latitudinal clines, suggesting that upregulation of <a href="https://identifiers.org/bioentitylink/FB:FBgn0002868?doi=10.1534/g3.119.400453"><em>MtnA</em></a> is favored in temperate environments (<a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Ramnarine <em>et al.</em>, 2019</a>). However, a direct link between cold stress and oxidative stress is yet to be established in drosophilids (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Plantamp <em>et al.</em>, 2016</a>). <a href="https://identifiers.org/bioentitylink/FB:FBgn0002868?doi=10.1534/g3.119.400453"><em>MtnA</em></a> was downregulated after the cold shock in both phenotypes of <em>D. melanogaster</em> (<a id="xref-ref-52-10" class="xref-bibr" href="#ref-52">von Heckel <em>et al.</em>, 2016</a>) and not differentially expressed in <em>D. ananassae</em>. Moreover, a previous sequence analysis of <a href="https://identifiers.org/bioentitylink/FB:FBgn0002868?doi=10.1534/g3.119.400453"><em>MtnA</em></a> in <em>D. ananassae</em> reported the 3′UTR deletion polymorphism as absent in 110 strains that were sampled in tropical and temperate regions around the world (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Stephan <em>et al.</em>, 1994</a>).</p><p id="p-58">The gene <em>GF24896</em> (<em>D.mel/</em><a href="https://identifiers.org/bioentitylink/FB:FBgn0001316?doi=10.1534/g3.119.400453"><em>klarsicht</em></a>) in QTL1 is expressed in a wide range of tissues, where it interacts with microtubules and promotes evenly spaced positioning of nuclei. Knock-out of <a href="https://identifiers.org/bioentitylink/FB:FBgn0001316?doi=10.1534/g3.119.400453"><em>klarsicht</em></a> in muscle cells impairs locomotion and flight (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Elhanany-Tamir <em>et al.</em>, 2012</a>) – functions that are also disabled during chill coma. The gene was reported previously to be upregulated with cold-acclimation in <em>D. melanogaster</em> (<a id="xref-ref-27-4" class="xref-bibr" href="#ref-27">MacMillan <em>et al.</em>, 2016</a>). In <em>D. ananassae</em>, <em>GF24896</em> is upregulated after the cold shock in fast strains but not in slow strains (<a id="xref-ref-24-17" class="xref-bibr" href="#ref-24">Königer and Grath 2018</a>), suggesting a potential contribution of this gene to faster recovery from cold exposure.</p><p id="p-59">Lastly, the gene <em>GF14829</em> (<em>D.mel</em>/<a href="https://identifiers.org/bioentitylink/FB:FBgn0032699?doi=10.1534/g3.119.400453"><em>CG10383</em></a>) in QTL3 is involved in the regulation of glycosylphosphatidylinositol metabolism. After the cold shock, it is upregulated in fast and slow strains of <em>D. ananassae</em> and in Slow strains of <em>D. melanogaster</em>. Interestingly, over-expression of <a href="https://identifiers.org/bioentitylink/FB:FBgn0032699?doi=10.1534/g3.119.400453"><em>CG10383</em></a> increases lifespan in <em>D. melanogaster</em> (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Paik <em>et al.</em>, 2012</a>). It was also the only gene within all three QTL for CCRT in <em>D. ananassae</em> that mapped to a heat-tolerance QTL in <em>D. melanogaster</em> (<a id="xref-ref-32-6" class="xref-bibr" href="#ref-32">Norry <em>et al.</em>, 2008</a>). In the face of the transgressive nature of QTL3, potential allelic effects resulting in trade-offs between CCRT, heat-resistance and lifespan should be investigated in both species.</p><p id="p-60">In conclusion, we identified three large-effect QTL for recovery from cold exposure in <em>D. ananassae</em>. Combining the present results with previous results obtained from <em>D. melanogaster</em> allowed us to shed light on commonalities and differences in the genetic basis of cold tolerance between these two species. The combined data point at the five above mentioned genes as candidates for recovery from cold exposure. Building on the present results, re-sequencing of the QTL intervals in cold-tolerant and cold-sensitive strains from Bangkok and additional strains from temperate populations may uncover signatures of selection. Moreover, the identified genes serve as the groundwork for more detailed analyses such as loss-of-function experiments to establish a link between genotype and phenotype in both species.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-61">We thank E. Argyridou, A. Glaser-Schmitt, E. Katab, H. Lainer, D. Metzler, J. Parsch, and the members of the LMU evolutionary biology group for their help in the lab and/or their constructive feedback during the course of this project. This work was supported by the Deutsche Forschungsgemeinschaft grant GR 4495/2-1.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-12"><p id="p-62">Supplemental material available at FigShare: <a href="https://doi.org/10.25387/g3.8288591">https://doi.org/10.25387/g3.8288591</a>.</p></li><li class="fn" id="fn-13"><p id="p-63"><em>Communicating editor: J. Comeron</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2019-06-18"><span class="received-label">Received </span>June 18, 2019.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2019-09-09"><span class="accepted-label">Accepted </span>September 9, 2019.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2019 Koniger <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3715.1" data-doi="10.1093/bioinformatics/btq565"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Arends</span>,  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Prins</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. C.</span> <span class="cit-name-surname">Jansen</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. W.</span> <span class="cit-name-surname">Broman</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">R/qtl: high-throughput multiple QTL mapping.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">2990</span>–<span class="cit-lpage">2992</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btq565</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq565%26rft_id%253Dinfo%253Apmid%252F20966004%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btq565&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20966004&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3715.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000284430900014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3715.2" data-doi="10.1093/nar/gkv1046"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Attrill</span>,  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Falls</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. L.</span> <span class="cit-name-surname">Goodman</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G. H.</span> <span class="cit-name-surname">Millburn</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Antonazzo</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">FlyBase: establishing a Gene Group resource for <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">D786</span>–<span class="cit-lpage">D792</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkv1046</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkv1046%26rft_id%253Dinfo%253Apmid%252F26467478%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkv1046&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26467478&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3715.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3715.3" data-doi="10.18637/jss.v067.i01"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bates</span>,  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Mächler</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Bolker</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Walker</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Fitting Linear Mixed-Effects Models Using lme4.</span> <abbr class="cit-jnl-abbrev">J. Stat. Softw.</abbr> <span class="cit-vol">67</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">48</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.18637/jss.v067.i01</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BStat.%2BSoftw.%26rft.volume%253D67%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.18637%252Fjss.v067.i01%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.18637/jss.v067.i01&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3715.4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Benjamini</span>,  <span class="cit-name-given-names">Y.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Hochberg</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Controlling the False Discovery Rate: A Practical and Powerful Approach to Multiple Testing. J. R. Stat.</span> <abbr class="cit-jnl-abbrev">Soc. Ser. B Methodol.</abbr> <span class="cit-vol">57</span>: <span class="cit-fpage">289</span>–<span class="cit-lpage">300</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSoc.%2BSer.%2BB%2BMethodol.%26rft.volume%253D57%26rft.spage%253D289%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3715.5" data-doi="10.1016/j.bcp.2015.10.001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bock</span>,  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">The UDP-glycosyltransferase (UGT) superfamily expressed in humans, insects and plants: Animal-plant arms-race and co-evolution.</span> <abbr class="cit-jnl-abbrev">Biochem. Pharmacol.</abbr> <span class="cit-vol">99</span>: <span class="cit-fpage">11</span>–<span class="cit-lpage">17</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.bcp.2015.10.001</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochem.%2BPharmacol.%26rft.volume%253D99%26rft.spage%253D11%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.bcp.2015.10.001%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.bcp.2015.10.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-book" id="cit-9.11.3715.6" data-doi="10.1007/978-0-387-92125-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Broman</span>,  <span class="cit-name-given-names">K.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Sen</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-source">A Guide to QTL Mapping with R/qtl</span>, <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">New York</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/978-0-387-92125-9</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DA%2BGuide%2Bto%2BQTL%2BMapping%2Bwith%2BR%252Fqtl%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-0-387-92125-9%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-0-387-92125-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3715.7" data-doi="10.1093/bioinformatics/btg112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Broman</span>,  <span class="cit-name-given-names">K. W.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Wu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Sen</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">G. A.</span> <span class="cit-name-surname">Churchill</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">R/qtl: QTL mapping in experimental crosses.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">889</span>–<span class="cit-lpage">890</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btg112</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DBroman%26rft.auinit1%253DK.%2BW.%26rft.volume%253D19%26rft.issue%253D7%26rft.spage%253D889%26rft.epage%253D890%26rft.atitle%253DR%252Fqtl%253A%2BQTL%2Bmapping%2Bin%2Bexperimental%2Bcrosses%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtg112%26rft_id%253Dinfo%253Apmid%252F12724300%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btg112&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12724300&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3715.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000182634800014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-2" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3715.8" data-doi="10.1371/journal.pgen.1005987"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Catalán</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Glaser-Schmitt</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Argyridou</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Duchen</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Parsch</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">An Indel Polymorphism in the MtnA 3′ Untranslated Region Is Associated with Gene Expression Variation and Local Adaptation in <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">e1005987</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1005987</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D12%26rft.spage%253De1005987%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1005987%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1005987&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3715.9" data-doi="10.1534/g3.111.000240"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Catchen</span>,  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Amores</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Hohenlohe</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Cresko</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. H.</span> <span class="cit-name-surname">Postlethwait</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Stacks: Building and Genotyping Loci De Novo From Short-Read Sequences. G3: Genes, Genomes</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">171</span>–<span class="cit-lpage">182</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D1%26rft.spage%253D171%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.111.000240%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/g3.111.000240&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3715.10" data-doi="10.1111/1755-0998.12669"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Catchen</span>,  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. A.</span> <span class="cit-name-surname">Hohenlohe</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Bernatchez</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W. C.</span> <span class="cit-name-surname">Funk</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. R.</span> <span class="cit-name-surname">Andrews</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Unbroken: RADseq remains a powerful tool for understanding the genetics of adaptation in natural populations.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol. Resour.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">362</span>–<span class="cit-lpage">365</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/1755-0998.12669</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%2BResour.%26rft_id%253Dinfo%253Adoi%252F10.1111%252F1755-0998.12669%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/1755-0998.12669&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3715.11" data-doi="10.1038/nature06341"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Clark</span>,  <span class="cit-name-given-names">A. G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. B.</span> <span class="cit-name-surname">Eisen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. R.</span> <span class="cit-name-surname">Smith</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. M.</span> <span class="cit-name-surname">Bergman</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Oliver</span></span> <span class="cit-etal">et al.</span></li></ol><cite> <span class="cit-pub-date">2007</span> <span class="cit-article-title">Evolution of genes and genomes on the Drosophila phylogeny.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">450</span>: <span class="cit-fpage">203</span>–<span class="cit-lpage">218</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DClark%26rft.auinit1%253DA.%2BG.%26rft.volume%253D450%26rft.issue%253D7167%26rft.spage%253D203%26rft.epage%253D218%26rft.atitle%253DEvolution%2Bof%2Bgenes%2Band%2Bgenomes%2Bon%2Bthe%2BDrosophila%2Bphylogeny.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature06341%26rft_id%253Dinfo%253Apmid%252F17994087%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature06341&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17994087&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3715.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000250746200042&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3715.12" data-doi="10.1534/genetics.110.121012"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cutler</span>,  <span class="cit-name-given-names">D. J.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. D.</span> <span class="cit-name-surname">Jensen</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">To Pool, or Not to Pool?</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">186</span>: <span class="cit-fpage">41</span>–<span class="cit-lpage">43</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.110.121012</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DCutler%26rft.auinit1%253DD.%2BJ.%26rft.volume%253D186%26rft.issue%253D1%26rft.spage%253D41%26rft.epage%253D43%26rft.atitle%253DTo%2BPool%252C%2Bor%2BNot%2Bto%2BPool%253F%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.121012%26rft_id%253Dinfo%253Apmid%252F20855575%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiMTg2LzEvNDEiO3M6NDoiYXRvbSI7czoxOToiL2dnZy85LzExLzM3MTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3715.13" data-doi="10.1534/genetics.104.031567"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Das</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Mohanty</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Stephan</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Inferring the Population Structure and Demography of <em>Drosophila ananassae</em> From Multilocus Data.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">168</span>: <span class="cit-fpage">1975</span>–<span class="cit-lpage">1985</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.104.031567</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DDas%26rft.auinit1%253DA.%26rft.volume%253D168%26rft.issue%253D4%26rft.spage%253D1975%26rft.epage%253D1985%26rft.atitle%253DInferring%2Bthe%2BPopulation%2BStructure%2Band%2BDemography%2Bof%2BDrosophila%2Bananassae%2BFrom%2BMultilocus%2BData%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.104.031567%26rft_id%253Dinfo%253Apmid%252F15611168%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2OC80LzE5NzUiO3M6NDoiYXRvbSI7czoxOToiL2dnZy85LzExLzM3MTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3715.14" data-doi="10.1007/BF03185015"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dasmohapatra</span>,  <span class="cit-name-given-names">D. P.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N. K.</span> <span class="cit-name-surname">Tripathy</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. C.</span> <span class="cit-name-surname">Das</span></span></li></ol><cite>, <span class="cit-pub-date">1982</span> <span class="cit-article-title">Temperature-related chromosome polymorphism in <em>Drosophila ananassae</em>.</span> <abbr class="cit-jnl-abbrev">Prof. Anim. Sci.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">243</span>–<span class="cit-lpage">247</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/BF03185015</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProf.%2BAnim.%2BSci.%26rft.volume%253D91%26rft.spage%253D243%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF03185015%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF03185015&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3715.15" data-doi="10.1016/0168-9525(88)90098-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">David</span>,  <span class="cit-name-given-names">J. R.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Capy</span></span></li></ol><cite>, <span class="cit-pub-date">1988</span> <span class="cit-article-title">Genetic variation of <em>Drosophila melanogaster</em> natural populations.</span> <abbr class="cit-jnl-abbrev">Trends Genet. TIG</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">106</span>–<span class="cit-lpage">111</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/0168-9525(88)90098-4</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DDavid%26rft.auinit1%253DJ.%2BR.%26rft.volume%253D4%26rft.issue%253D4%26rft.spage%253D106%26rft.epage%253D111%26rft.atitle%253DGenetic%2Bvariation%2Bof%2BDrosophila%2Bmelanogaster%2Bnatural%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0168-9525%252888%252990098-4%26rft_id%253Dinfo%253Apmid%252F3149056%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0168-9525(88)90098-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=3149056&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3715.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1988N173500008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3715.16" data-doi="10.1016/S0306-4565(98)00020-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">David</span>,  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Gibert</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Pla</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Petavy</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Karan</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Cold stress tolerance in Drosophila: Analysis of chill coma recovery in <em>D. Melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">J. Therm. Biol.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">291</span>–<span class="cit-lpage">299</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/S0306-4565(98)00020-5</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BTherm.%2BBiol.%26rft.volume%253D23%26rft.spage%253D291%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0306-4565%252898%252900020-5%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0306-4565(98)00020-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000076801000005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3715.17" data-doi="10.1534/genetics.104.84756"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">David</span>,  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Lemeunier</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Tsacas</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Yassin</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">The Historical Discovery of the Nine Species in the <em>Drosophila melanogaster</em> Species Subgroup.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">177</span>: <span class="cit-fpage">1969</span>–<span class="cit-lpage">1973</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.104.84756</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DDavid%26rft.auinit1%253DJ.%2BR.%26rft.volume%253D177%26rft.issue%253D4%26rft.spage%253D1969%26rft.epage%253D1973%26rft.atitle%253DThe%2BHistorical%2BDiscovery%2Bof%2Bthe%2BNine%2BSpecies%2Bin%2Bthe%2BDrosophila%2Bmelanogaster%2BSpecies%2BSubgroup%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.104.84756%26rft_id%253Dinfo%253Apmid%252F18073416%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3Ny80LzE5NjkiO3M6NDoiYXRvbSI7czoxOToiL2dnZy85LzExLzM3MTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3715.18" data-doi="10.1073/pnas.221458698"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">De Gregorio</span>,  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. T.</span> <span class="cit-name-surname">Spellman</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G. M.</span> <span class="cit-name-surname">Rubin</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Lemaitre</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Genome-wide analysis of the <em>Drosophila</em> immune response by using oligonucleotide microarrays.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">98</span>: <span class="cit-fpage">12590</span>–<span class="cit-lpage">12595</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.221458698</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.221458698%26rft_id%253Dinfo%253Apmid%252F11606746%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiOTgvMjIvMTI1OTAiO3M6NDoiYXRvbSI7czoxOToiL2dnZy85LzExLzM3MTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3715.19" data-doi="10.1083/jcb.201204102"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Elhanany-Tamir</span>,  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y. V.</span> <span class="cit-name-surname">Yu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Shnayder</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Jain</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Welte</span></span></li></ol><cite> <em>et al.</em>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Organelle positioning in muscles requires cooperation between two KASH proteins and microtubules.</span> <abbr class="cit-jnl-abbrev">JCB</abbr> <span class="cit-vol">198</span>: <span class="cit-fpage">833</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1083/jcb.201204102</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJCB%26rft_id%253Dinfo%253Adoi%252F10.1083%252Fjcb.201204102%26rft_id%253Dinfo%253Apmid%252F22927463%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamNiIjtzOjU6InJlc2lkIjtzOjk6IjE5OC81LzgzMyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzkvMTEvMzcxNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3715.20" data-doi="10.1534/genetics.110.114397"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Futschik</span>,  <span class="cit-name-given-names">A.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Schlötterer</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The Next Generation of Molecular Markers From Massively Parallel Sequencing of Pooled DNA Samples.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">186</span>: <span class="cit-fpage">207</span>–<span class="cit-lpage">218</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.110.114397</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.114397%26rft_id%253Dinfo%253Apmid%252F20457880%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg2LzEvMjA3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMS8zNzE1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-book no-rev-xref" id="cit-9.11.3715.21"><div class="cit-metadata unstructured">Falconer, D. S., and T. F. C. Mackay, 1996 <em>Introduction to Quantitative Genetics</em>. Oliver &amp; Boyd, Edinburgh, London.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3715.22" data-doi="10.1038/nprot.2008.211"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>,  <span class="cit-name-given-names">D. W.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. T.</span> <span class="cit-name-surname">Sherman</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. A.</span> <span class="cit-name-surname">Lempicki</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Systematic and integrative analysis of large gene lists using DAVID bioinformatics resources.</span> <abbr class="cit-jnl-abbrev">Nat. Protoc.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">44</span>–<span class="cit-lpage">57</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nprot.2008.211</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BProtoc%26rft.aulast%253DHuang%26rft.auinit1%253Dd.%2Ba.%2B%2B.%2BW.%26rft.volume%253D4%26rft.issue%253D1%26rft.spage%253D44%26rft.epage%253D57%26rft.atitle%253DSystematic%2Band%2Bintegrative%2Banalysis%2Bof%2Blarge%2Bgene%2Blists%2Busing%2BDAVID%2Bbioinformatics%2Bresources.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnprot.2008.211%26rft_id%253Dinfo%253Apmid%252F19131956%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nprot.2008.211&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19131956&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3715.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000265781800006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3715.23" data-doi="10.1534/genetics.105.047985"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kirkpatrick</span>,  <span class="cit-name-given-names">M.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Barton</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Chromosome Inversions, Local Adaptation and Speciation.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">173</span>: <span class="cit-fpage">419</span>–<span class="cit-lpage">434</span> (erratum: Genetics 208: 433).<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.105.047985</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.105.047985%26rft_id%253Dinfo%253Apmid%252F16204214%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTczLzEvNDE5IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMS8zNzE1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3715.24" data-doi="10.3390/genes9120624"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Königer</span>,  <span class="cit-name-given-names">A.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Grath</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Transcriptome Analysis Reveals Candidate Genes for Cold Tolerance in <em>Drosophila ananassae</em>.</span> <abbr class="cit-jnl-abbrev">Genes (Basel)</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">624</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3390/genes9120624</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2528Basel%2529%26rft.volume%253D9%26rft.spage%253D624%26rft_id%253Dinfo%253Adoi%252F10.3390%252Fgenes9120624%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3390/genes9120624&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-book" id="cit-9.11.3715.25" data-doi="10.1007/978-1-4613-0931-4_4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lachaise</span>,  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.-L.</span> <span class="cit-name-surname">Cariou</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. R.</span> <span class="cit-name-surname">David</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Lemeunier</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Tsacas</span></span></li></ol><cite> <em>et al.</em>, <span class="cit-pub-date">1988</span> <span class="cit-article-title">Historical Biogeography of the <em>Drosophila melanogaster</em> Species Subgroup</span>, pp. <span class="cit-fpage">159</span>–<span class="cit-lpage">225</span> in <span class="cit-source">Evolutionary Biology</span>, edited by <span class="cit-ed"> <span class="cit-name-given-names">M. K.</span> <span class="cit-name-surname">Hecht</span></span>, <span class="cit-ed"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Wallace</span></span>, and <span class="cit-ed"> <span class="cit-name-given-names">G. T.</span> <span class="cit-name-surname">Prance</span></span>. <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">Boston</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/978-1-4613-0931-4_4</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolutionary%2BBiology%26rft.volume%253D22%26rft.spage%253D159%26rft.atitle%253DEVOLUTIONARY%2BBIOLOGY%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-1-4613-0931-4_4%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-1-4613-0931-4_4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3715.26" data-doi="10.1093/bioinformatics/btp352"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Handsaker</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Wysoker</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Fennell</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Ruan</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The Sequence Alignment/Map format and SAMtools.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">2078</span>–<span class="cit-lpage">2079</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btp352</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp352%26rft_id%253Dinfo%253Apmid%252F19505943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp352&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19505943&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3715.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268808600014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-2" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3715.27" data-doi="10.1038/srep28999"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">MacMillan</span>,  <span class="cit-name-given-names">H. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">Knee</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. B.</span> <span class="cit-name-surname">Dennis</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Udaka</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. E.</span> <span class="cit-name-surname">Marshall</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Cold acclimation wholly reorganizes the <em>Drosophila melanogaster</em> transcriptome and metabolome.</span> <abbr class="cit-jnl-abbrev">Sci. Rep.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">28999</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/srep28999</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BRep.%26rft.volume%253D6%26rft.spage%253D28999%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsrep28999%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/srep28999&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3715.28" data-doi="10.1093/nar/gku1221"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Marchler-Bauer</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. K.</span> <span class="cit-name-surname">Derbyshire</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N. R.</span> <span class="cit-name-surname">Gonzales</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Lu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Chitsaz</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">CDD: NCBI’s conserved domain database.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">D222</span>–<span class="cit-lpage">D226</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gku1221</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgku1221%26rft_id%253Dinfo%253Apmid%252F25414356%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gku1221&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25414356&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3715.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3715.29" data-doi="10.1016/S0764-4469(97)85020-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Moreteau</span>,  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. P.</span> <span class="cit-name-surname">Morin</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Gibert</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Pétavy</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Pla</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Evolutionary changes of nonlinear reaction norms according to thermal adaptation: a comparison of two <em>Drosophila</em> species.</span> <abbr class="cit-jnl-abbrev">C. R. Acad. Sci. III</abbr> <span class="cit-vol">320</span>: <span class="cit-fpage">833</span>–<span class="cit-lpage">841</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/S0764-4469(97)85020-2</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DComptes%2BRendus%2BHebdomadaires%2Bdes%2BSeances%2Bde%2Bl%2527Academie%2Bdes%2BSciences.%2BSerie%2BB%253A%2BSciences%2BPhysiques%26rft.stitle%253DComptes%2BRendus%2BHebdomadaires%2Bdes%2BSeances%2Bde%2Bl%2527Academie%2Bdes%2BSciences.%2BSerie%2BB%253A%2BSciences%2BPhysiques%26rft.aulast%253DMoreteau%26rft.auinit1%253DB.%26rft.volume%253D320%26rft.issue%253D10%26rft.spage%253D833%26rft.epage%253D841%26rft.atitle%253DEvolutionary%2Bchanges%2Bof%2Bnonlinear%2Breaction%2Bnorms%2Baccording%2Bto%2Bthermal%2Badaptation%253A%2Ba%2Bcomparison%2Bof%2Btwo%2BDrosophila%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0764-4469%252897%252985020-2%26rft_id%253Dinfo%253Apmid%252F9436536%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0764-4469(97)85020-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9436536&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3715.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3715.30" data-doi="10.1038/sj.hdy.6800786"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Morgan</span>,  <span class="cit-name-given-names">T. J.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. F. C.</span> <span class="cit-name-surname">Mackay</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Quantitative trait loci for thermotolerance phenotypes in <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">96</span>: <span class="cit-fpage">232</span>–<span class="cit-lpage">242</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/sj.hdy.6800786</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.stitle%253DHeredity%26rft.aulast%253DMorgan%26rft.auinit1%253DT.%2BJ.%26rft.volume%253D96%26rft.issue%253D3%26rft.spage%253D232%26rft.epage%253D242%26rft.atitle%253DQuantitative%2Btrait%2Bloci%2Bfor%2Bthermotolerance%2Bphenotypes%2Bin%2BDrosophila%2Bmelanogaster.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsj.hdy.6800786%26rft_id%253Dinfo%253Apmid%252F16404413%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/sj.hdy.6800786&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16404413&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3715.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000235499000007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3715.31" data-doi="10.2307/2411043"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Morin</span>,  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Moreteau</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Pétavy</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Parkash</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. R.</span> <span class="cit-name-surname">David</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Reaction norms of morphological traits in <em>drosophila</em>: adaptive shape changes in a stenotherm circumtropical species?</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">51</span>: <span class="cit-fpage">1140</span>–<span class="cit-lpage">1148</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.volume%253D51%26rft.spage%253D1140%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2411043%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2411043&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3715.32" data-doi="10.1111/j.1365-294X.2008.03945.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Norry</span>,  <span class="cit-name-given-names">F. M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. C.</span> <span class="cit-name-surname">Scannapieco</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Sambucetti</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. I.</span> <span class="cit-name-surname">Bertoli</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Loeschcke</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">QTL for the thermotolerance effect of heat hardening, knockdown resistance to heat and chill-coma recovery in an intercontinental set of recombinant inbred lines of <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">4570</span>–<span class="cit-lpage">4581</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1365-294X.2008.03945.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DNorry%26rft.auinit1%253DF.%2BM.%26rft.volume%253D17%26rft.issue%253D20%26rft.spage%253D4570%26rft.epage%253D4581%26rft.atitle%253DQTL%2Bfor%2Bthe%2Bthermotolerance%2Beffect%2Bof%2Bheat%2Bhardening%252C%2Bknockdown%2Bresistance%2Bto%2Bheat%2Band%2Bchill-coma%2Brecovery%2Bin%2Ban%2Bintercontinental%2Bset%2Bof%2Brecombinant%2Binbred%2Blines%2Bof%2BDrosophila%2Bmelanogaster.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2008.03945.x%26rft_id%253Dinfo%253Apmid%252F18986501%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2008.03945.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18986501&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3715.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3715.33" data-doi="10.1016/j.mad.2012.02.001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Paik</span>,  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y. G.</span> <span class="cit-name-surname">Jang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y. E.</span> <span class="cit-name-surname">Lee</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y. N.</span> <span class="cit-name-surname">Lee</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Yamamoto</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Misexpression screen delineates novel genes controlling <em>Drosophila</em> lifespan.</span> <abbr class="cit-jnl-abbrev">Mech. Ageing Dev.</abbr> <span class="cit-vol">133</span>: <span class="cit-fpage">234</span>–<span class="cit-lpage">245</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.mad.2012.02.001</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMechanisms%2Bof%2Bageing%2Band%2Bdevelopment%26rft.stitle%253DMech%2BAgeing%2BDev%26rft.aulast%253DPaik%26rft.auinit1%253DD.%26rft.volume%253D133%26rft.issue%253D5%26rft.spage%253D234%26rft.epage%253D245%26rft.atitle%253DMisexpression%2Bscreen%2Bdelineates%2Bnovel%2Bgenes%2Bcontrolling%2BDrosophila%2Blifespan.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.mad.2012.02.001%26rft_id%253Dinfo%253Apmid%252F22366109%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.mad.2012.02.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22366109&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3715.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3715.34" data-doi="10.1016/j.jinsphys.2016.03.009"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Plantamp</span>,  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Salort</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Gibert</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Dumet</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Mialdea</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">All or nothing: Survival, reproduction and oxidative balance in Spotted Wing <em>Drosophila</em> (<em>Drosophila suzukii</em>) in response to cold.</span> <abbr class="cit-jnl-abbrev">J. Insect Physiol.</abbr> <span class="cit-vol">89</span>: <span class="cit-fpage">28</span>–<span class="cit-lpage">36</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.jinsphys.2016.03.009</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BInsect%2BPhysiol.%26rft.volume%253D89%26rft.spage%253D28%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jinsphys.2016.03.009%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jinsphys.2016.03.009&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3715.35" data-doi="10.1007/978-1-61779-785-9_3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pollard</span>,  <span class="cit-name-given-names">D. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Design and Construction of Recombinant Inbred Lines.</span> <abbr class="cit-jnl-abbrev">Methods Mol Biol</abbr> <span class="cit-vol">871</span>: <span class="cit-fpage">31</span>–<span class="cit-lpage">39</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/978-1-61779-785-9_3</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2Bin%2Bmolecular%2Bbiology%2B%2528Clifton%252C%2BN.J.%2529%26rft.stitle%253DMethods%2BMol%2BBiol%26rft.aulast%253DPollard%26rft.auinit1%253DD.%2BA.%26rft.volume%253D871%26rft.spage%253D31%26rft.epage%253D39%26rft.atitle%253DDesign%2Band%2Bconstruction%2Bof%2Brecombinant%2Binbred%2Blines.%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-1-61779-785-9_3%26rft_id%253Dinfo%253Apmid%252F22565831%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-1-61779-785-9_3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22565831&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3715.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-web" id="cit-9.11.3715.36"><div class="cit-metadata unstructured">R Core Team, 2018 R: a language and environment for statistical computing, Vienna, Austria. Available at <a href="http://www.R-project.org/">http://www.R-project.org/</a></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3715.37" data-doi="10.3390/genes10020147"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ramnarine</span>,  <span class="cit-name-given-names">T. J. S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Glaser-Schmitt</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Catalán</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Parsch</span></span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">Population Genetic and Functional Analysis of a cis-Regulatory Polymorphism in the <em>Drosophila melanogaster</em> Metallothionein A gene.</span> <abbr class="cit-jnl-abbrev">Genes (Basel)</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">147</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3390/genes10020147</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2528Basel%2529%26rft.volume%253D10%26rft.spage%253D147%26rft_id%253Dinfo%253Adoi%252F10.3390%252Fgenes10020147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3390/genes10020147&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3715.38" data-doi="10.1016/S0378-1119(02)01187-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ross</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Jiang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. R.</span> <span class="cit-name-surname">Kanost</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Wang</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Serine proteases and their homologs in the <em>Drosophila melanogaster</em> genome: an initial analysis of sequence conservation and phylogenetic relationships.</span> <abbr class="cit-jnl-abbrev">Gene</abbr> <span class="cit-vol">304</span>: <span class="cit-fpage">117</span>–<span class="cit-lpage">131</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/S0378-1119(02)01187-3</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGene%26rft.stitle%253DGene%26rft.aulast%253DRoss%26rft.auinit1%253DJ.%26rft.volume%253D304%26rft.spage%253D117%26rft.epage%253D131%26rft.atitle%253DSerine%2Bproteases%2Band%2Btheir%2Bhomologs%2Bin%2Bthe%2BDrosophila%2Bmelanogaster%2Bgenome%253A%2Ban%2Binitial%2Banalysis%2Bof%2Bsequence%2Bconservation%2Band%2Bphylogenetic%2Brelationships.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0378-1119%252802%252901187-3%26rft_id%253Dinfo%253Apmid%252F12568721%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0378-1119(02)01187-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12568721&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3715.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000181124500012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3715.39" data-doi="10.3390/ijms14036044"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ruttkay-Nedecky</span>,  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Nejdl</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Gumulec</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">O.</span> <span class="cit-name-surname">Zitka</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Masarik</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The Role of Metallothionein in Oxidative Stress.</span> <abbr class="cit-jnl-abbrev">Int. J. Mol. Sci.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">6044</span>–<span class="cit-lpage">6066</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3390/ijms14036044</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInt.%2BJ.%2BMol.%2BSci.%26rft.volume%253D14%26rft.spage%253D6044%26rft_id%253Dinfo%253Adoi%252F10.3390%252Fijms14036044%26rft_id%253Dinfo%253Apmid%252F23502468%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3390/ijms14036044&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23502468&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3715.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3715.40" data-doi="10.1534/genetics.106.066613"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schug</span>,  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. G.</span> <span class="cit-name-surname">Smith</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Tozier-Pearce</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. F.</span> <span class="cit-name-surname">McEvey</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">The Genetic Structure of Drosophila ananassae Populations From Asia, Australia and Samoa.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">175</span>: <span class="cit-fpage">1429</span>–<span class="cit-lpage">1440</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.106.066613</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.106.066613%26rft_id%253Dinfo%253Apmid%252F17237518%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3NS8zLzE0MjkiO3M6NDoiYXRvbSI7czoxOToiL2dnZy85LzExLzM3MTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3715.41" data-doi="10.1093/bioinformatics/btt468"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sedlazeck</span>,  <span class="cit-name-given-names">F. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Rescheneder</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">von Haeseler</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">NextGenMap: fast and accurate read mapping in highly polymorphic genomes.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">2790</span>–<span class="cit-lpage">2791</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btt468</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtt468%26rft_id%253Dinfo%253Apmid%252F23975764%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btt468&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23975764&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3715.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000325997500018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3715.42"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sen</span>,  <span class="cit-name-given-names">S.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">G. A.</span> <span class="cit-name-surname">Churchill</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">A statistical framework for quantitative trait mapping.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">159</span>: <span class="cit-fpage">371</span>–<span class="cit-lpage">387</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DSen%26rft.auinit1%253DS.%26rft.volume%253D159%26rft.issue%253D1%26rft.spage%253D371%26rft.epage%253D387%26rft.atitle%253DA%2BStatistical%2BFramework%2Bfor%2BQuantitative%2BTrait%2BMapping%26rft_id%253Dinfo%253Apmid%252F11560912%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTU5LzEvMzcxIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMS8zNzE1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3715.43" data-doi="10.1111/j.1365-2583.2007.00739.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sinclair</span>,  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. G.</span> <span class="cit-name-surname">Gibbs</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. P.</span> <span class="cit-name-surname">Roberts</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Gene transcription during exposure to, and recovery from, cold and desiccation stress in <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">Insect Mol. Biol.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">435</span>–<span class="cit-lpage">443</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1365-2583.2007.00739.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInsect%2Bmolecular%2Bbiology%26rft.stitle%253DInsect%2BMol%2BBiol%26rft.aulast%253DSinclair%26rft.auinit1%253DB.%2BJ.%26rft.volume%253D16%26rft.issue%253D4%26rft.spage%253D435%26rft.epage%253D443%26rft.atitle%253DGene%2Btranscription%2Bduring%2Bexposure%2Bto%252C%2Band%2Brecovery%2Bfrom%252C%2Bcold%2Band%2Bdesiccation%2Bstress%2Bin%2BDrosophila%2Bmelanogaster.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2583.2007.00739.x%26rft_id%253Dinfo%253Apmid%252F17506850%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-2583.2007.00739.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17506850&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3715.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000248858000006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3715.44" data-doi="10.1017/S0016672307008890"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Singh</span>,  <span class="cit-name-given-names">P.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. N.</span> <span class="cit-name-surname">Singh</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Population genetics of <em>Drosophila ananassae</em>: genetic differentiation among Indian natural populations at the level of inversion polymorphism.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">89</span>: <span class="cit-fpage">191</span>–<span class="cit-lpage">199</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1017/S0016672307008890</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DSingh%26rft.auinit1%253DP.%26rft.volume%253D89%26rft.issue%253D4%26rft.spage%253D191%26rft.epage%253D199%26rft.atitle%253DPopulation%2Bgenetics%2Bof%2BDrosophila%2Bananassae%253A%2Bgenetic%2Bdifferentiation%2Bamong%2BIndian%2Bnatural%2Bpopulations%2Bat%2Bthe%2Blevel%2Bof%2Binversion%2Bpolymorphism.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672307008890%26rft_id%253Dinfo%253Apmid%252F18208624%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672307008890&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18208624&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3715.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3715.45" data-doi="10.1038/sj.hdy.6800901"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stephan</span>,  <span class="cit-name-given-names">W.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Li</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">The recent demographic and adaptive history of <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">98</span>: <span class="cit-fpage">65</span>–<span class="cit-lpage">68</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/sj.hdy.6800901</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.stitle%253DHeredity%26rft.aulast%253DStephan%26rft.auinit1%253DW.%26rft.volume%253D98%26rft.issue%253D2%26rft.spage%253D65%26rft.epage%253D68%26rft.atitle%253DThe%2Brecent%2Bdemographic%2Band%2Badaptive%2Bhistory%2Bof%2BDrosophila%2Bmelanogaster.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsj.hdy.6800901%26rft_id%253Dinfo%253Apmid%252F17006533%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/sj.hdy.6800901&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17006533&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3715.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000243730100003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3715.46"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stephan</span>,  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">V. S.</span> <span class="cit-name-surname">Rodriguez</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Zhou</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Parsch</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Molecular evolution of the metallothionein gene Mtn in the melanogaster species group: results from <em>Drosophila ananassae</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">138</span>: <span class="cit-fpage">135</span>–<span class="cit-lpage">143</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DStephan%26rft.auinit1%253DW.%26rft.volume%253D138%26rft.issue%253D1%26rft.spage%253D135%26rft.epage%253D143%26rft.atitle%253DMolecular%2Bevolution%2Bof%2Bthe%2Bmetallothionein%2Bgene%2BMtn%2Bin%2Bthe%2Bmelanogaster%2Bspecies%2Bgroup%253A%2Bresults%2Bfrom%2BDrosophila%2Bananassae.%26rft_id%253Dinfo%253Apmid%252F8001781%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTM4LzEvMTM1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMS8zNzE1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3715.47" data-doi="10.1016/j.jinsphys.2009.01.010"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Telonis-Scott</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Hallas</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. W.</span> <span class="cit-name-surname">McKechnie</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. W.</span> <span class="cit-name-surname">Wee</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. A.</span> <span class="cit-name-surname">Hoffmann</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Selection for cold resistance alters gene transcript levels in <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">J. Insect Physiol.</abbr> <span class="cit-vol">55</span>: <span class="cit-fpage">549</span>–<span class="cit-lpage">555</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.jinsphys.2009.01.010</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BInsect%2BPhysiology%26rft.stitle%253DJournal%2Bof%2BInsect%2BPhysiology%26rft.aulast%253DTelonis-Scott%26rft.auinit1%253DM.%26rft.volume%253D55%26rft.issue%253D6%26rft.spage%253D549%26rft.epage%253D555%26rft.atitle%253DSelection%2Bfor%2Bcold%2Bresistance%2Balters%2Bgene%2Btranscript%2Blevels%2Bin%2BDrosophila%2Bmelanogaster.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jinsphys.2009.01.010%26rft_id%253Dinfo%253Apmid%252F19232407%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jinsphys.2009.01.010&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19232407&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3715.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000266417000008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-book" id="cit-9.11.3715.48"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tobari</span>,  <span class="cit-name-given-names">Y. N.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-source">Drosophila ananassae: genetical and biological aspects</span>, <span class="cit-publ-name">Japan Scientific Societies Press Karger</span>, <span class="cit-publ-loc">Tokyo, Basel, New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3715.49" data-doi="10.2307/2407625"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tucić</span>,  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">1979</span> <span class="cit-article-title">Genetic Capacity for Adaptation to Cold Resistance at Different Developmental Stages of <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">350</span>–<span class="cit-lpage">358</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1558-5646.1979.tb04688.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.volume%253D33%26rft.spage%253D350%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2407625%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2407625&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1979GV98300009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3715.50" data-doi="10.1016/j.jinsphys.2010.08.008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Udaka</span>,  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Ueda</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. G.</span> <span class="cit-name-surname">Goto</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Survival rate and expression of Heat-shock protein 70 and Frost genes after temperature stress in <em>Drosophila melanogaster</em> lines that are selected for recovery time from temperature coma.</span> <abbr class="cit-jnl-abbrev">J. Insect Physiol.</abbr> <span class="cit-vol">56</span>: <span class="cit-fpage">1889</span>–<span class="cit-lpage">1894</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.jinsphys.2010.08.008</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BInsect%2BPhysiology%26rft.stitle%253DJournal%2Bof%2BInsect%2BPhysiology%26rft.aulast%253DUdaka%26rft.auinit1%253DH.%26rft.volume%253D56%26rft.issue%253D12%26rft.spage%253D1889%26rft.epage%253D1894%26rft.atitle%253DSurvival%2Brate%2Band%2Bexpression%2Bof%2BHeat-shock%2Bprotein%2B70%2Band%2BFrost%2Bgenes%2Bafter%2Btemperature%2Bstress%2Bin%2BDrosophila%2Bmelanogaster%2Blines%2Bthat%2Bare%2Bselected%2Bfor%2Brecovery%2Btime%2Bfrom%2Btemperature%2Bcoma.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jinsphys.2010.08.008%26rft_id%253Dinfo%253Apmid%252F20713057%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jinsphys.2010.08.008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20713057&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3715.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3715.51" data-doi="10.1111/jeb.12183"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vermeulen</span>,  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Sørensen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. K.</span> <span class="cit-name-surname">Gagalova</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Loeschcke</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Transcriptomic analysis of inbreeding depression in cold-sensitive <em>Drosophila melanogaster</em> shows upregulation of the immune response.</span> <abbr class="cit-jnl-abbrev">J. Evol. Biol.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">1890</span>–<span class="cit-lpage">1902</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/jeb.12183</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BEvol.%2BBiol.%26rft.volume%253D26%26rft.spage%253D1890%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fjeb.12183%26rft_id%253Dinfo%253Apmid%252F23944235%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/jeb.12183&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23944235&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3715.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3715.52" data-doi="10.1186/s12864-016-2866-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">von Heckel</span>,  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Stephan</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Hutter</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Canalization of gene expression is a major signature of regulatory cold adaptation in temperate <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">574</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12864-016-2866-0</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D17%26rft.spage%253D574%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864-016-2866-0%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12864-016-2866-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3715.53" data-doi="10.1111/evo.12541"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Williams</span>,  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Watanabe</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Guarracino</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Ferraro</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Edison</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Cold adaptation shapes the robustness of metabolic networks in <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">68</span>: <span class="cit-fpage">3505</span>–<span class="cit-lpage">3523</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/evo.12541</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.volume%253D68%26rft.spage%253D3505%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fevo.12541%26rft_id%253Dinfo%253Apmid%252F25308124%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/evo.12541&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25308124&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3715.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3715.54" data-doi="10.1242/jeb.059535"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. E.</span> <span class="cit-name-surname">Marshall</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. T.</span> <span class="cit-name-surname">Westwood</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. S.</span> <span class="cit-name-surname">Clark</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. J.</span> <span class="cit-name-surname">Sinclair</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Divergent transcriptomic responses to repeated and single cold exposures in <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">J. Exp. Biol.</abbr> <span class="cit-vol">214</span>: <span class="cit-fpage">4021</span>–<span class="cit-lpage">4029</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1242/jeb.059535</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BExperimental%2BBiology%26rft.stitle%253DJ.%2BExp.%2BBiol.%26rft.aulast%253DZhang%26rft.auinit1%253DJ.%26rft.volume%253D214%26rft.issue%253D23%26rft.spage%253D4021%26rft.epage%253D4029%26rft.atitle%253DDivergent%2Btranscriptomic%2Bresponses%2Bto%2Brepeated%2Band%2Bsingle%2Bcold%2Bexposures%2Bin%2BDrosophila%2Bmelanogaster%26rft_id%253Dinfo%253Adoi%252F10.1242%252Fjeb.059535%26rft_id%253Dinfo%253Apmid%252F22071194%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiamV4YmlvIjtzOjU6InJlc2lkIjtzOjExOiIyMTQvMjMvNDAyMSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzkvMTEvMzcxNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/9/11/3715.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/11/3703" title="Activity-Dependent Regulation of the Proapoptotic BH3-Only Gene egl-1 in a Living Neuron Pair in Caenorhabditis elegans" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/11/3727" title="Pitfalls and Remedies for Cross Validation with Multi-trait Genomic Prediction Methods" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/9/11"><span>Volume 9 Issue 11, November 2019</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/9/11.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/9/11.cover-source.jpg?itok=DjqkLGk9" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 9 (11)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/11/3715.figures-only" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-srJS7PlcWSk-dsS4pQsXxjt2LoLJvAKQYB4difgHKKE" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/9/11/3715" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/9/11/3715" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/9/11/3715" class="active" data-icon-position="" data-hide-link-title="0">Three Quantitative Trait Loci Explain More than 60% of Variation for Chill Coma Recovery Time in a Natural Population of Drosophila ananassae</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/494618" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-Q1smv1ekzB4bXcAM5bqRFjV-WpYNavawGbmhW_xfUEo" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/11/3715.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/9/11/3715" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-QMWoDcjPk3XCeRJIFvrSzAps1yeV3tCe7QQFmHN-ZDk" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/9/11/3715" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/11/3715.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494618" id="node494618--2" data-pisa="ggg;9/11/3715" data-pisa-master="ggg;g3.119.400453" data-apath="/ggg/9/11/3715.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Three Quantitative Trait Loci Explain More than 60% of Variation for Chill Coma Recovery Time in a Natural Population of <em>Drosophila ananassae</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-4554-9390" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Annabella Königer</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0003-0811-8604" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Saad Arif</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0003-3621-736X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Sonja Grath</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3715-3725;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400453 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494618--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Annabella Königer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Division of Evolutionary Biology, Ludwig-Maximilians-Universität (LMU) München, 82152 Planegg-Martinsried, Germany and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Annabella%2BK%C3%B6niger%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=K%C3%B6niger%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAnnabella%2BK%25C3%25B6niger%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-4554-9390" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Annabella Königer</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Saad Arif </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Centre for Functional Genomics, Department of Biological and Medical Sciences, Oxford Brookes University, Oxford, OX3 0BP, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Saad%2BArif%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Arif%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASaad%2BArif%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-0811-8604" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Saad Arif</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Sonja Grath </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Division of Evolutionary Biology, Ludwig-Maximilians-Universität (LMU) München, 82152 Planegg-Martinsried, Germany and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sonja%2BGrath%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Grath%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASonja%2BGrath%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-3621-736X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Sonja Grath</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:grath@bio.lmu.de" class="" data-icon-position="" data-hide-link-title="0">grath@bio.lmu.de</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/9/11/3715" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/9/11/3715&amp;title=Three%20Quantitative%20Trait%20Loci%20Explain%20More%20than%2060%25%20of%20Variation%20for%20Chill%20Coma%20Recovery%20Time%20in%20a%20Natural%20Population%20of%20Drosophila%20ananassae" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/9/11/3715&amp;title=Three%20Quantitative%20Trait%20Loci%20Explain%20More%20than%2060%25%20of%20Variation%20for%20Chill%20Coma%20Recovery%20Time%20in%20a%20Natural%20Population%20of%20Drosophila%20ananassae" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/9/11/3715&amp;title=Three%20Quantitative%20Trait%20Loci%20Explain%20More%20than%2060%25%20of%20Variation%20for%20Chill%20Coma%20Recovery%20Time%20in%20a%20Natural%20Population%20of%20Drosophila%20ananassae" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/9/11/3715&amp;text=Three%20Quantitative%20Trait%20Loci%20Explain%20More%20than%2060%25%20of%20Variation%20for%20Chill%20Coma%20Recovery%20Time%20in%20a%20Natural%20Population%20of%20Drosophila%20ananassae" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/9/11/3715&amp;title=Three%20Quantitative%20Trait%20Loci%20Explain%20More%20than%2060%25%20of%20Variation%20for%20Chill%20Coma%20Recovery%20Time%20in%20a%20Natural%20Population%20of%20Drosophila%20ananassae" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/9/11/3715&amp;t=Three%20Quantitative%20Trait%20Loci%20Explain%20More%20than%2060%25%20of%20Variation%20for%20Chill%20Coma%20Recovery%20Time%20in%20a%20Natural%20Population%20of%20Drosophila%20ananassae" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/9/11/3715&amp;title=Three%20Quantitative%20Trait%20Loci%20Explain%20More%20than%2060%25%20of%20Variation%20for%20Chill%20Coma%20Recovery%20Time%20in%20a%20Natural%20Population%20of%20Drosophila%20ananassae" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/9/11/3715&amp;title=Three%20Quantitative%20Trait%20Loci%20Explain%20More%20than%2060%25%20of%20Variation%20for%20Chill%20Coma%20Recovery%20Time%20in%20a%20Natural%20Population%20of%20Drosophila%20ananassae" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/494618/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494618" id="node494618--3" data-pisa="ggg;9/11/3715" data-pisa-master="ggg;g3.119.400453" data-apath="/ggg/9/11/3715.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Three Quantitative Trait Loci Explain More than 60% of Variation for Chill Coma Recovery Time in a Natural Population of <em>Drosophila ananassae</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-4554-9390" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Annabella Königer</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0003-0811-8604" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Saad Arif</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0003-3621-736X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Sonja Grath</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3715-3725;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400453 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494618--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Annabella Königer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Division of Evolutionary Biology, Ludwig-Maximilians-Universität (LMU) München, 82152 Planegg-Martinsried, Germany and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Annabella%2BK%C3%B6niger%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=K%C3%B6niger%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAnnabella%2BK%25C3%25B6niger%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-4554-9390" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Annabella Königer</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Saad Arif </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Centre for Functional Genomics, Department of Biological and Medical Sciences, Oxford Brookes University, Oxford, OX3 0BP, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Saad%2BArif%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Arif%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASaad%2BArif%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-0811-8604" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Saad Arif</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Sonja Grath </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Division of Evolutionary Biology, Ludwig-Maximilians-Universität (LMU) München, 82152 Planegg-Martinsried, Germany and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sonja%2BGrath%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Grath%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASonja%2BGrath%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-3621-736X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Sonja Grath</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:grath@bio.lmu.de" class="" data-icon-position="" data-hide-link-title="0">grath@bio.lmu.de</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=3715&amp;rft.epage=3725&amp;rft.atitle=Three%20Quantitative%20Trait%20Loci%20Explain%20More%20than%2060%25%20of%20Variation%20for%20Chill%20Coma%20Recovery%20Time%20in%20a%20Natural%20Population%20of%20Drosophila%20ananassae&amp;rft.volume=9&amp;rft.issue=11&amp;rft.date=2019-11-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=K%C3%B6niger%2C+Annabella&amp;rft.au=Arif%2C+Saad&amp;rft.au=Grath%2C+Sonja"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/494618/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/494618/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/494618/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/494618/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/494618/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/494618/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/494618/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/494618/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/494618/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/494618/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/494618/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/494618/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/9/11/3715" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-9" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-15" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/9/11/3715.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/9/11/3715.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/9/11/3715.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
