<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/9/11/3521" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/9/11/3521.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/9/11/3521.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/9/11/3521.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/9/11/3521.atom" />
<meta name="HW.pisa" content="ggg;9/11/3521" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Genome Improvement and Genetic Map Construction for Aethionema arabicum, the First Divergent Branch in the Brassicaceae Family" />
<meta name="DC.Identifier" content="10.1534/g3.119.400657" />
<meta name="DC.Date" content="2019-11-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2019 Nguyen et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="The genus Aethionema is a sister-group to the core-group of the Brassicaceae family that includes Arabidopsis thaliana and the Brassica crops. Thus, Aethionema is phylogenetically well-placed for the investigation and understanding of genome and trait evolution across the family. We aimed to improve the quality of the reference genome draft version of the annual species Aethionema arabicum . Second, we constructed the first Ae. arabicum genetic map. The improved reference genome and genetic map enabled the development of each other. We started with the initially published genome (version 2.5). PacBio and MinION sequencing together with genetic map v2.5 were incorporated to produce the new reference genome v3.0. The improved genome contains 203 MB of sequence, with approximately 94% of the assembly made up of called (non-gap) bases, assembled into 2,883 scaffolds (with only 6% of the genome made up of non-called bases (Ns)). The N50 (10.3 MB) represents an 80-fold increase over the initial genome release. We generated a Recombinant Inbred Line (RIL) population that was derived from two ecotypes: Cyprus and Turkey (the reference genotype. Using a Genotyping by Sequencing (GBS) approach, we generated a high-density genetic map with 749 (v2.5) and then 632 SNPs (v3.0) was generated. The genetic map and reference genome were integrated, thus greatly improving the scaffolding of the reference genome into 11 linkage groups. We show that long-read sequencing data and genetics are complementary, resulting in an improved genome assembly in Ae. arabicum . They will facilitate comparative genetic mapping work for the Brassicaceae family and are also valuable resources to investigate wide range of life history traits in Aethionema ." />
<meta name="DC.Contributor" content="Thu-Phuong Nguyen" />
<meta name="DC.Contributor" content="Cornelia Mühlich" />
<meta name="DC.Contributor" content="Setareh Mohammadin" />
<meta name="DC.Contributor" content="Erik van den Bergh" />
<meta name="DC.Contributor" content="Adrian E. Platts" />
<meta name="DC.Contributor" content="Fabian B. Haas" />
<meta name="DC.Contributor" content="Stefan A. Rensing" />
<meta name="DC.Contributor" content="M. Eric Schranz" />
<meta name="article:published_time" content="2019-11-01" />
<meta name="article:section" content="Genome Report" />
<meta name="citation_title" content="Genome Improvement and Genetic Map Construction for Aethionema arabicum, the First Divergent Branch in the Brassicaceae Family" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;The genus &lt;i&gt;Aethionema&lt;/i&gt; is a sister-group to the core-group of the Brassicaceae family that includes &lt;i&gt;Arabidopsis thaliana&lt;/i&gt; and the Brassica crops. Thus, &lt;i&gt;Aethionema&lt;/i&gt; is phylogenetically well-placed for the investigation and understanding of genome and trait evolution across the family. We aimed to improve the quality of the reference genome draft version of the annual species &lt;i&gt;Aethionema arabicum&lt;/i&gt;. Second, we constructed the first &lt;i&gt;Ae. arabicum&lt;/i&gt; genetic map. The improved reference genome and genetic map enabled the development of each other. We started with the initially published genome (version 2.5). PacBio and MinION sequencing together with genetic map v2.5 were incorporated to produce the new reference genome v3.0. The improved genome contains 203 MB of sequence, with approximately 94% of the assembly made up of called (non-gap) bases, assembled into 2,883 scaffolds (with only 6% of the genome made up of non-called bases (Ns)). The N&lt;sub&gt;50&lt;/sub&gt; (10.3 MB) represents an 80-fold increase over the initial genome release. We generated a Recombinant Inbred Line (RIL) population that was derived from two ecotypes: Cyprus and Turkey (the reference genotype. Using a Genotyping by Sequencing (GBS) approach, we generated a high-density genetic map with 749 (v2.5) and then 632 SNPs (v3.0) was generated. The genetic map and reference genome were integrated, thus greatly improving the scaffolding of the reference genome into 11 linkage groups. We show that long-read sequencing data and genetics are complementary, resulting in an improved genome assembly in &lt;i&gt;Ae. arabicum&lt;/i&gt;. They will facilitate comparative genetic mapping work for the Brassicaceae family and are also valuable resources to investigate wide range of life history traits in &lt;i&gt;Aethionema&lt;/i&gt;.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2019/11/01" />
<meta name="citation_mjid" content="ggg;9/11/3521" />
<meta name="citation_id" content="9/11/3521" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/9/11/3521" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/9/11/3521.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/9/11/3521.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/9/11/3521.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.119.400657" />
<meta name="citation_pmid" content="31554715" />
<meta name="citation_volume" content="9" />
<meta name="citation_issue" content="11" />
<meta name="citation_num_pages" content="10" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Genome Report" />
<meta name="citation_firstpage" content="3521" />
<meta name="citation_lastpage" content="3530" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Thu-Phuong Nguyen" />
<meta name="citation_author_institution" content="Biosystematics Group, Wageningen University, Droevendaalsesteeg 1, 6708 PB Wageningen, The Netherlands," />
<meta name="citation_author" content="Cornelia Mühlich" />
<meta name="citation_author_institution" content="Faculty of Biology, Plant Cell Biology, University of Marburg, Karl-von-Frisch-Str. 8, 35043 Marburg, Germany," />
<meta name="citation_author" content="Setareh Mohammadin" />
<meta name="citation_author_institution" content="Biosystematics Group, Wageningen University, Droevendaalsesteeg 1, 6708 PB Wageningen, The Netherlands," />
<meta name="citation_author" content="Erik van den Bergh" />
<meta name="citation_author_institution" content="Biosystematics Group, Wageningen University, Droevendaalsesteeg 1, 6708 PB Wageningen, The Netherlands," />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-9865-574X" />
<meta name="citation_author" content="Adrian E. Platts" />
<meta name="citation_author_institution" content="Center for Genomics and Systems Biology, Department of Biology, New York University, New York, and" />
<meta name="citation_author" content="Fabian B. Haas" />
<meta name="citation_author_institution" content="Faculty of Biology, Plant Cell Biology, University of Marburg, Karl-von-Frisch-Str. 8, 35043 Marburg, Germany," />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-7711-5282" />
<meta name="citation_author" content="Stefan A. Rensing" />
<meta name="citation_author_institution" content="Faculty of Biology, Plant Cell Biology, University of Marburg, Karl-von-Frisch-Str. 8, 35043 Marburg, Germany," />
<meta name="citation_author_institution" content="LOEWE Center for Synthetic Microbiology (SYNMIKRO), Philipps University of Marburg, Germany" />
<meta name="citation_author_email" content="stefan.rensing@biologie.uni-marburg.de" />
<meta name="citation_author_email" content="eric.schranz@wur.nl" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-0225-873X" />
<meta name="citation_author" content="M. Eric Schranz" />
<meta name="citation_author_institution" content="Biosystematics Group, Wageningen University, Droevendaalsesteeg 1, 6708 PB Wageningen, The Netherlands," />
<meta name="citation_author_email" content="stefan.rensing@biologie.uni-marburg.de" />
<meta name="citation_author_email" content="eric.schranz@wur.nl" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-6777-6565" />
<meta name="citation_reference" content="citation_journal_title=Plant Syst. Evol.;citation_author=I. A. Al-Shehbaz;citation_author=M. A. Beilstein;citation_author=E. A. Kellogg;citation_title=Systematics and phylogeny of the Brassicaceae (Cruciferae): an overview.;citation_pages=89-120;citation_volume=259;citation_year=2006;citation_doi=10.1007/s00606-006-0415-z" />
<meta name="citation_reference" content="citation_journal_title=Botany;citation_author=W. Arshad;citation_author=F. Marone;citation_author=M. E. Collinson;citation_author=G. Leubner-Metzger;citation_author=T. Steinbrecher;citation_title=Fracture of the dimorphic fruits of Aethionema arabicum (Brassicaceae).;citation_year=2019;citation_doi=10.1139/cjb-2019-0014" />
<meta name="citation_reference" content="citation_journal_title=American Journal of Botany;citation_journal_abbrev=Am. J. Bot.;citation_author=M. A. Beilstein;citation_author=I. A. Al‐Shehbaz;citation_author=S. Mathews;citation_author=E. A. Kellogg;citation_title=Brassicaceae phylogeny inferred from phytochrome A and ndhF sequence data: tribes and trichomes revisited;citation_pages=1307-1327;citation_volume=95;citation_year=2008;citation_issue=10;citation_pmid=21632335;citation_doi=10.3732/ajb.0800065" />
<meta name="citation_reference" content="Bibalani, G. H., 2012 Investigation on flowering phenology of Brassicaceae in the Shanjan region Shabestar district, NW Iran (usage for honeybees)." />
<meta name="citation_reference" content="citation_journal_title=Journal of computational biology :  a journal of computational molecular cell biology;citation_journal_abbrev=J Comput Biol;citation_author=S. Boisvert;citation_author=F. Laviolette;citation_author=J. Corbeil;citation_title=Ray: simultaneous assembly of reads from a mix of high-throughput sequencing technologies.;citation_pages=1519-1533;citation_volume=17;citation_year=2010;citation_issue=11;citation_pmid=20958248;citation_doi=10.1089/cmb.2009.0238" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. M. Bolger;citation_author=M. Lohse;citation_author=B. Usadel;citation_title=Trimmomatic: a flexible trimmer for Illumina sequence data.;citation_volume=30;citation_year=2014;citation_pmid=24695404;citation_doi=10.1093/bioinformatics/btu170" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=M. J. Chaisson;citation_author=G. Tesler;citation_title=Mapping single molecule sequencing reads using Basic Local Alignment with Successive Refinement (BLASR): Theory and Application.;citation_pages=238-238;citation_volume=13;citation_year=2012;citation_issue=1;citation_pmid=22988817;citation_doi=10.1186/1471-2105-13-238" />
<meta name="citation_reference" content="citation_journal_title=Plant Mol. Biol. Report.;citation_author=S. L. Dellaporta;citation_author=J. Wood;citation_author=J. B. Hicks;citation_title=A plant DNA minipreparation: Version II.;citation_pages=19-21;citation_volume=1;citation_year=1983;citation_doi=10.1007/BF02712670" />
<meta name="citation_reference" content="citation_title=Molecular Techniques in Taxonomy;citation_title=DNA Protocols for Plants;citation_pages=283-293;citation_year=1991" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=R. J. Elshire;citation_author=J. C. Glaubitz;citation_author=Q. Sun;citation_author=J. A. Poland;citation_author=K. Kawamoto;citation_title=A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species.;citation_pages=e19379-e19379;citation_volume=6;citation_year=2011;citation_issue=5;citation_pmid=21573248;citation_doi=10.1371/journal.pone.0019379" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=A. C. English;citation_author=S. Richards;citation_author=Y. Han;citation_author=M. Wang;citation_author=V. Vee;citation_title=Mind the Gap: Upgrading Genomes with Pacific Biosciences RS Long-Read Sequencing Technology.;citation_pages=e47768-e47768;citation_volume=7;citation_year=2012;citation_issue=11;citation_pmid=23185243;citation_doi=10.1371/journal.pone.0047768" />
<meta name="citation_reference" content="citation_journal_title=Trends in plant science;citation_journal_abbrev=Trends Plant Sci;citation_author=A. Franzke;citation_author=M. A. Lysak;citation_author=I. A. Al-Shehbaz;citation_author=M. A. Koch;citation_author=K. Mummenhoff;citation_title=Cabbage family affairs: the evolutionary history of Brassicaceae.;citation_pages=108-116;citation_volume=16;citation_year=2011;citation_issue=2;citation_pmid=21177137;citation_doi=10.1016/j.tplants.2010.11.005" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=J. C. Glaubitz;citation_author=T. M. Casstevens;citation_author=F. Lu;citation_author=J. Harriman;citation_author=R. J. Elshire;citation_title=TASSEL-GBS: A High Capacity Genotyping by Sequencing Analysis Pipeline.;citation_pages=e90346;citation_volume=9;citation_year=2014;citation_pmid=24587335;citation_doi=10.1371/journal.pone.0090346" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=S. Gnerre;citation_author=I. MacCallum;citation_author=D. Przybylski;citation_author=F. J. Ribeiro;citation_author=J. N. Burton;citation_title=High-quality draft assemblies of mammalian genomes from massively parallel sequence data.;citation_volume=108;citation_year=2011;citation_pmid=21187386;citation_doi=10.1073/pnas.1017351108" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=X. Guo;citation_author=J. Liu;citation_author=G. Hao;citation_author=L. Zhang;citation_author=K. Mao;citation_title=Plastome phylogeny and early diversification of Brassicaceae.;citation_pages=176;citation_volume=18;citation_year=2017;citation_doi=10.1186/s12864-017-3555-3" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=A. Haudry;citation_author=A. E. Platts;citation_author=E. Vello;citation_author=D. R. Hoen;citation_author=M. Leclercq;citation_title=An atlas of over 90,000 conserved noncoding sequences provides insight into crucifer regulatory regions.;citation_pages=891-898;citation_volume=45;citation_year=2013;citation_pmid=23817568;citation_doi=10.1038/ng.2684" />
<meta name="citation_reference" content="citation_journal_title=Plant J.;citation_author=M. Hiss;citation_author=R. Meyberg;citation_author=J. Westermann;citation_author=F. B. Haas;citation_author=L. Schneider;citation_title=Sexual reproduction, sporophyte development and molecular variation in the model moss Physcomitrella patens: introducing the ecotype Reute.;citation_pages=606-620;citation_volume=90;citation_year=2017;citation_doi=10.1111/tpj.13501" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol. Evol.;citation_author=J. A. Hofberger;citation_author=E. Lyons;citation_author=P. P. Edger;citation_author=J. Chris Pires;citation_author=M. Eric Schranz;citation_title=Whole genome and tandem duplicate retention facilitated glucosinolate pathway diversification in the mustard family.;citation_volume=5;citation_year=2013;citation_pmid=24171911;citation_doi=10.1093/gbe/evt162" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=C.-H. Huang;citation_author=R. Sun;citation_author=Y. Hu;citation_author=L. Zeng;citation_author=N. Zhang;citation_title=Resolution of Brassicaceae Phylogeny Using Nuclear Genes Uncovers Nested Radiations and Supports Convergent Morphological Evolution.;citation_volume=33;citation_year=2016;citation_pmid=26516094;citation_doi=10.1093/molbev/msv226" />
<meta name="citation_reference" content="citation_journal_title=PLOS Comput. Biol.;citation_author=D. H. Huson;citation_author=S. Beier;citation_author=I. Flade;citation_author=A. Górska;citation_author=M. El-Hadidi;citation_title=MEGAN Community Edition - Interactive Exploration and Analysis of Large-Scale Microbiome Sequencing Data.;citation_pages=e1004957;citation_volume=12;citation_year=2016;citation_doi=10.1371/journal.pcbi.1004957" />
<meta name="citation_reference" content="citation_journal_title=Perspect. Plant Ecol. Evol. Syst.;citation_author=E. Imbert;citation_title=Ecological consequences and ontogeny of seed heteromorphism.;citation_pages=13-36;citation_volume=5;citation_year=2002;citation_doi=10.1078/1433-8319-00021" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=J. Keilwagen;citation_author=M. Wenk;citation_author=J. L. Erickson;citation_author=M. H. Schattat;citation_author=J. Grau;citation_title=Using intron position conservation for homology-based gene prediction.;citation_volume=44;citation_year=2016;citation_pmid=26893356;citation_doi=10.1093/nar/gkw092" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=W. J. Kent;citation_author=C. W. Sugnet;citation_author=T. S. Furey;citation_author=K. M. Roskin;citation_author=T. H. Pringle;citation_title=The Human Genome Browser at UCSC.;citation_volume=12;citation_year=2002;citation_pmid=12045153;citation_doi=10.1101/gr.229102. Article published online before print in May 2002" />
<meta name="citation_reference" content="citation_journal_title=Plant J.;citation_author=D. Lang;citation_author=K. K. Ullrich;citation_author=F. Murat;citation_author=J. Fuchs;citation_author=J. Jenkins;citation_title=The Physcomitrella patens chromosome-scale assembly reveals moss genome structure and evolution.;citation_pages=515-533;citation_volume=93;citation_year=2018;citation_doi=10.1111/tpj.13801" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=M. A. Larkin;citation_author=G. Blackshields;citation_author=N. P. Brown;citation_author=R. Chenna;citation_author=P. A. McGettigan;citation_title=Clustal W and Clustal X version 2.0.;citation_volume=23;citation_year=2007;citation_pmid=17846036;citation_doi=10.1093/bioinformatics/btm404" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiol.;citation_author=T. Lenser;citation_author=K. Graeber;citation_author=Ö. S. Cevik;citation_author=N. Adigüzel;citation_author=A. A. Dönmez;citation_title=Developmental Control and Plasticity of Fruit and Seed Dimorphism in Aethionema arabicum.;citation_volume=172;citation_year=2016;citation_pmid=27702842;citation_doi=10.1104/pp.16.00838" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=R. Durbin;citation_title=Fast and accurate long-read alignment with Burrows–Wheeler transform.;citation_volume=26;citation_year=2010;citation_pmid=20080505;citation_doi=10.1093/bioinformatics/btp698" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=B. Handsaker;citation_author=A. Wysoker;citation_author=T. Fennell;citation_author=J. Ruan;citation_title=The Sequence Alignment/Map format and SAMtools.;citation_volume=25;citation_year=2009;citation_pmid=19505943;citation_doi=10.1093/bioinformatics/btp352" />
<meta name="citation_reference" content="citation_journal_title=Gigascience;citation_author=R. Luo;citation_author=B. Liu;citation_author=Y. Xie;citation_author=Z. Li;citation_author=W. Huang;citation_title=SOAPdenovo2: an empirically improved memory-efficient short-read de novo assembler.;citation_pages=18;citation_volume=1;citation_year=2012;citation_pmid=23587118;citation_doi=10.1186/2047-217X-1-18" />
<meta name="citation_reference" content="citation_journal_title=J. Exp. Bot.;citation_author=Z. Mérai;citation_author=K. Graeber;citation_author=P. Wilhelmsson;citation_author=K. K. Ullrich;citation_author=W. Arshad;citation_title=Aethionema arabicum: a novel model plant to study the light control of seed germination.;citation_pages=3313-3328;citation_volume=70;citation_year=2019;citation_doi=10.1093/jxb/erz146" />
<meta name="citation_reference" content="citation_journal_title=Am. J. Bot.;citation_author=S. Mohammadin;citation_author=K. Peterse;citation_author=S. J. van de Kerke;citation_author=L. W. Chatrou;citation_author=A. A. Dönmez;citation_title=Anatolian origins and diversification of Aethionema, the sister lineage of the core Brassicaceae.;citation_volume=104;citation_year=2017;citation_pmid=28743759;citation_doi=10.3732/ajb.1700091" />
<meta name="citation_reference" content="citation_journal_title=Plant Syst. Evol.;citation_author=S. Mohammadin;citation_author=W. Wang;citation_author=T. Liu;citation_author=H. Moazzeni;citation_author=K. Ertugrul;citation_title=Genome-wide nucleotide diversity and associations with geography, ploidy level and glucosinolate profiles in Aethionema arabicum (Brassicaceae).;citation_pages=619-630;citation_volume=304;citation_year=2018;citation_doi=10.1007/s00606-018-1494-3" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_title=Database resources of the National Center for Biotechnology Information.;citation_volume=44;citation_year=2016;citation_pmid=26615191;citation_doi=10.1093/nar/gkv1290" />
<meta name="citation_reference" content="citation_journal_title=Nat. Sci.;citation_author=Y. Ronin;citation_author=D. Mester;citation_author=D. Minkov;citation_author=A. Korol;citation_title=Building reliable genetic maps: different mapping strategies may result in different maps.;citation_pages=576;citation_volume=02;citation_year=2010" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=B. A. Rowan;citation_author=V. Patel;citation_author=D. Weigel;citation_author=K. Schneeberger;citation_title=Rapid and Inexpensive Whole-Genome Genotyping-by-Sequencing for Crossover Localization and Fine-Scale Genetic Mapping. G3: Genes, Genomes;citation_pages=385-398;citation_volume=5;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Plant J.;citation_author=P. Stam;citation_title=Construction of integrated genetic linkage maps by means of a new computer package: Join Map.;citation_pages=739-744;citation_volume=3;citation_year=1993;citation_doi=10.1111/j.1365-313X.1993.00739.x" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=H. Tang;citation_author=X. Zhang;citation_author=C. Miao;citation_author=J. Zhang;citation_author=R. Ming;citation_title=ALLMAPS: robust scaffold ordering based on multiple maps.;citation_pages=3;citation_volume=16;citation_year=2015;citation_pmid=25583564;citation_doi=10.1186/s13059-014-0573-1" />
<meta name="citation_reference" content="van Ooijen, J. W., 2006 JoinMap 4. Software for the calculation of genetic linkage maps in experimental populations." />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=M. Watson;citation_author=M. Thomson;citation_author=J. Risse;citation_author=R. Talbot;citation_author=J. Santoyo-Lopez;citation_title=poRe: an R package for the visualization and analysis of nanopore sequencing data.;citation_volume=31;citation_year=2015;citation_pmid=25173419;citation_doi=10.1093/bioinformatics/btu590" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=P. K. I. Wilhelmsson;citation_author=J. O. Chandler;citation_author=N. Fernandez-Pozo;citation_author=K. Graeber;citation_author=K. K. Ullrich;citation_title=Usability of reference-free transcriptome assemblies for detection of differential expression: a case study on Aethionema arabicum dimorphic seeds.;citation_pages=95;citation_volume=20;citation_year=2019;citation_doi=10.1186/s12864-019-5452-4" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=T. D. Wu;citation_author=S. Nacu;citation_title=Fast and SNP-tolerant detection of complex variants and splicing in short reads.;citation_volume=26;citation_year=2010;citation_pmid=20147302;citation_doi=10.1093/bioinformatics/btq057" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Genome Improvement and Genetic Map Construction for Aethionema arabicum, the First Divergent Branch in the Brassicaceae Family" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11.cover-source.jpg" />
<meta name="twitter:description" content="The genus Aethionema is a sister-group to the core-group of the Brassicaceae family that includes Arabidopsis thaliana and the Brassica crops. Thus, Aethionema is phylogenetically well-placed for the investigation and understanding of genome and trait evolution across the family. We aimed to improve the quality of the reference genome draft version of the annual species Aethionema arabicum . Second, we constructed the first Ae. arabicum genetic map. The improved reference genome and genetic map enabled the development of each other. We started with the initially published genome (version 2.5). PacBio and MinION sequencing together with genetic map v2.5 were incorporated to produce the new reference genome v3.0. The improved genome contains 203 MB of sequence, with approximately 94% of the assembly made up of called (non-gap) bases, assembled into 2,883 scaffolds (with only 6% of the genome made up of non-called bases (Ns)). The N50 (10.3 MB) represents an 80-fold increase over the initial genome release. We generated a Recombinant Inbred Line (RIL) population that was derived from two ecotypes: Cyprus and Turkey (the reference genotype. Using a Genotyping by Sequencing (GBS) approach, we generated a high-density genetic map with 749 (v2.5) and then 632 SNPs (v3.0) was generated. The genetic map and reference genome were integrated, thus greatly improving the scaffolding of the reference genome into 11 linkage groups. We show that long-read sequencing data and genetics are complementary, resulting in an improved genome assembly in Ae. arabicum . They will facilitate comparative genetic mapping work for the Brassicaceae family and are also valuable resources to investigate wide range of life history traits in Aethionema ." />
<meta name="og-title" property="og:title" content="Genome Improvement and Genetic Map Construction for Aethionema arabicum, the First Divergent Branch in the Brassicaceae Family" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/9/11/3521" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="The genus Aethionema is a sister-group to the core-group of the Brassicaceae family that includes Arabidopsis thaliana and the Brassica crops. Thus, Aethionema is phylogenetically well-placed for the investigation and understanding of genome and trait evolution across the family. We aimed to improve the quality of the reference genome draft version of the annual species Aethionema arabicum . Second, we constructed the first Ae. arabicum genetic map. The improved reference genome and genetic map enabled the development of each other. We started with the initially published genome (version 2.5). PacBio and MinION sequencing together with genetic map v2.5 were incorporated to produce the new reference genome v3.0. The improved genome contains 203 MB of sequence, with approximately 94% of the assembly made up of called (non-gap) bases, assembled into 2,883 scaffolds (with only 6% of the genome made up of non-called bases (Ns)). The N50 (10.3 MB) represents an 80-fold increase over the initial genome release. We generated a Recombinant Inbred Line (RIL) population that was derived from two ecotypes: Cyprus and Turkey (the reference genotype. Using a Genotyping by Sequencing (GBS) approach, we generated a high-density genetic map with 749 (v2.5) and then 632 SNPs (v3.0) was generated. The genetic map and reference genome were integrated, thus greatly improving the scaffolding of the reference genome into 11 linkage groups. We show that long-read sequencing data and genetics are complementary, resulting in an improved genome assembly in Ae. arabicum . They will facilitate comparative genetic mapping work for the Brassicaceae family and are also valuable resources to investigate wide range of life history traits in Aethionema ." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11.cover-source.jpg" />
    <title>Genome Improvement and Genetic Map Construction for Aethionema arabicum, the First Divergent Branch in the Brassicaceae Family | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"lMujUTpVwFmoYLmu42FDxtDG-jmHcbS3F9F9f6ogKv0","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"494623","apath":"\/ggg\/9\/11\/3521.atom","pisa":"ggg;9\/11\/3521","ac":{"\/ggg\/9\/11\/3521.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/9\/11\/3521.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;9\/11\/3521"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;9\/11\/3521"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/9\/11\/3521":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022\\\\\\\/OukovGqBGvCTpUFlA4tFAIf+4MnHtrA1nrR06ZQm\\\\\\\/en51Qu4bJAY6SpJW1CPu2CvCjJZvEdK09lxnQaMeved1BmTjYP1byJlOUd\\\\\\\/h6xE8P218KI59MYoDbp6qHhB02hs3vaHT+MjEwtxXb1aEqBX4D7RDFH+UHbeIZ+bogbt5FM33dAXcPNC+2iZCp13J4ZcEaPsnMsJnnXWhubmhT5HZmoM3e3czLw6YtDPG+JzYsQrZYE57THf0qr9hPEBTumgYp3WOXhpX1vDsZk+x1CdtPD1LkNMop9UUpOYeU\\\\\\\/wa5UTh5w8\\\\\\\/jU1x8mzEyszTj6X5qwRG+tbx\\\\\\\/6+ISy793bGXbCzzaepEtz2R2ZqKbZ\\\\\\\/FsFM5NV+kU1V\\\\\\\/6D9eVlASDYzHC9wdRkyTkcC9S8\\\\\\\/vEsUg6AfCto\\\\\\\/Ng6NoGTFk1ofd8qdqfL164JHUJO7H7nb8QhRcDpo2Bftc31nHsfr7o1vK1k5w0eYJOxvJxfae75PIG1BxpvgRSOyabo3IvkcfyQlEqd1DwmoHFkJxgrTOg\\\\\\\/6SSoTg7o9sSg7nQe5hs18E9ROWCNG7qp6Ac4cYOMrNAhzkTDO\\\\\\\/tovuIdpQ74i2GdzNEoehU1VsUtZwstP1rqOjZV6cJZ3o\\\\\\\/nomnxW5yHehgL9RPdObMfVarl3lWytgrShP0dZDGLG2lMfQ+ljp1yQjf8oc4tQoe+8m9PIyzC5V626DCDKiXK8HZnSFeDDBnK8p8FkQzoTkgkta8z\\\\\\\/N90Bf7sjiACMinJMHksivNvixpOGLsfILufBE3Ik9QfjB0CCKJGmvEy9OqkpCvqFPlBvsoyyDbFbFwmwvZY35wKKIkSAgrLNwii1jRNDqZKCcg5XLpaOO+1r2pdlZuAlRAOWxU\\\\\\\/8jbI0HOeevOM6HFtR1330BTnqvqJq4kDbaRWs3\\\\\\\/IPOvCMRMFsx3SU60rWLz13znqaO6Fr+rGL0eXe2CcIAWyL\\\\\\\/8Eh7\\\\\\\/8W6ILOvtZ5\\\\\\\/pQsPDTBIVpyr4XLqL+r7cPDgxekTzAWKeL8I6P2G+kTOuME2S3S7v7tOe6geAnAVmLy\\\\\\\/sKhTVDSFqawJiCT7eJsBZlUlXngRaoKKi\\\\\\\/aYD+57hxYLLiTy75h4sxruj7zquOixJou8GAMNBCnFMShuqigZoTtqGQTVLyAzdWn+OsI8nty9LAyvo9xZTiZjcbzFSoZnvrSFpyOkGSeK990cwxzeD5nt3YbmuM5XySTGfEpfi1cqjoHpiGqQ9ZQ1sdKHfaJ9+Vci2qcocrHJZ5KBBWH5ny1UQahjhZjAqcPbmNAjSu\\\u0022,\\\u0022iv\\\u0022:\\\u0022lBMwepSMhpDEii3plSoEbQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00228d63d6fb408fcce4ade50e4580169f5440b78fc9e2ce49c3523e726071bce5ff\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022wzs1KCwL7joZtIfT1rHSRK\\\\\\\/h3RPqp+VGQRdawjbo0rFd9QLafPktUxBVFY5U+LPPbn4+a0x29eTH27gf3PEe+db1UZOMQVKm4q9EnTCd664jyPotpVIHbEWuuZupU9fc08ZpqZsNl1kTRgvOiXQ\\\\\\\/OZZS430DOsfKhZPOIhEbtoVaCC\\\\\\\/PhURNO79nd6ErX7O2NERJQbymDEkESruDnn6BxLEGM4J413+V9HpDvqkGlu4ctFzahDDJiuYdDlY\\\\\\\/qv9+RbvikeNNOM53140+hrrN\\\\\\\/2LSg0hXeFgEnWX+QcgeyuiHNIQuHGunwsZ3fHYfswEa7vi6eZntkDK7IKAW5dzPBLUm7vsbHYR7OpN7KT35IC+h5R9pA8rA\\\\\\\/1yRnSAZ1RJN7m7+jTIurzC7sq3zNNLxqtfQutBEyqYaQj9bQf\\\\\\\/6xmIn0zc+dXWcLgx+XCxhl4ArLOufZonkz7eMq11rUTNAZXvyZJVREYzgmqUHuruHSsysn410UR+5rRW0GZrKTdFPUG1WN62V02YZxMybcH1KgHOXDtKW+IIDppJRUJy5cJ7iYcnbyBjAg1XREYDmAWJvr\\\\\\\/QrI3C5uStHy4r0aaowWEF5Spgcc8ETo0TTs7rQ1VUcO0BvzwbT0vO6evu7tpURuv8LbMHy4Xm548Ms5N9FWfcy19m0Gjy8ajn0RWanYec4vzgiyQwTjsW1HD7WYDjqnE2AlfUmviY4PfBmBLAB5efFfV5di5xEVZllMvIHtDrr\\\\\\\/xaVIwHEkb+z3Hd0GIh\\\\\\\/YhRUtjkDEPC7NqQLWPx0VGLroHt8X72bjJQkXeegV3TcnHGPklSDICat6EzrS6kfkkj2WY0MShBf6S94b719X+Tu0U1dJHo6sOJhcNx6flK+r1GRP2YcgiBUzYHtHTki30n0wPgzExQ6agy6QB758ZfWk6vaXSEI2cxfX2\\\\\\\/0X2qtT+l01X1niyDfJI5v2IJ1XhfaDmWuy8vvobhky3c0D+HrTY27kTPQBUaf5VMx3a3cDoe4PltPsSR3kQ8xeu3VgoNFGJwRu9sUXOpyFWJAR1z6PTYgMzAOisjVudKEO9S7o2JpjZfuz2Zttf+1S6DEavBefzgtBJKYZ+P2QfVP51\\\\\\\/fGqaPtUojAOvZSWvl7M\\\\\\\/p+lTVwIG+dbqtOpScKQAbVHUKUmdhZjc6B5BzoPAbIMbFoq0iLw2EgZcHMQVJyRtqSFh7xe70o\\\\\\\/EMS+fCuI8AQuZy4AIKaL4BX2F\\\\\\\/XCRkS15Hwr3lTn75WjCznDwE8\\\\\\\/lVSuk92r7tRvFk9mQ0SSaJgw5nFzdNPFVBSc9C7o3qqF84YLPkusZonFwzxxCLg5JUDLW0IihWcupAk6BNVV3\\\\\\\/h0z0NgxbgV8QQM0bCOQG4gMWWfMwVRVSv+DP\\\\\\\/JBjX9DmyOOqgpDULHqaFZiQK+Ca0m7yS9tN8sZwzHxeyNSKGwGQq5mWJikl\\\\\\\/NHOAZWNq5nGjnzoYfH9jYgFjEvfHfUocpB0S7JnNro+VxX2CpZdsx1yTnkS1jkvubgxxqxL5+o6OyQyzDxN7HVNPghQC+nik2Fyx1C1XfYi5hVXZgshUDSVHuTgpNmDjYdlAx85OcK9aaTk0tGoYbU\\\\\\\/Dlx369bfySqCsgevICC4OfIRvaTQjFmL0g4UWFN5bi0sdAgpssa6AKZ6bDm7LtZJi30mbiU6Qmk29SMXMUH+yaRJkIQe2kiN7hI19sbOwhnp1X\\\\\\\/w8nN9rc1gXll3q9hDvhZx8e8IfsiVfsuNyXk46gwmFJiEqGoRRYbf6cbXWdPT9DTVczcOI4NBYW5P33WbQrrmv2JjkPekfS3d1RGO4pgPx0Eq8VXyaQZDOM\\\\\\\/gdBuJVlyVqT\\\\\\\/K0SkBQYkJs0jGiD97ujeDXRwEXFUFbrbihha4BfgSYVkT6RHwdGlhS4G5dWb4sluQLY7DbPDnvWXkzK9AxCGFrO5yTu0pPKlIfnv6bXB7L1I2U7J0RZ6SH3D6hLHjpvuPauFmlKHNcjIYd7x8GqnVpcpluP3q9g1\\\\\\\/Q\\\\\\\/NCf4OJ1htmlcEaYmR8WIqq664o49XhPJDLHAZtvxfyFoLioBY5SloJHdttrm4gaeS6tBwjRVKREkUof8otTrlJl+wuX5Q9lIZ58AJlPuJ8\\\\\\\/5Bcg8N4kHgrHqDMxfZS90J003S9Km93Zi5m2yhWS3POyz7Av0u\\\\\\\/qEh2\\\\\\\/ux9\\\\\\\/eiOZ+0kNZ65jpWrc5LGk8xZXUeALjyo1vcMUt35Ck1BG3votHi35SfCv6Uf5ZBLAG5BjdW6RfPq5VRGdWEPsqFozOxfYgN\\\\\\\/T51uXIx+ig3HSTkTAHH5rBM4ic+z1zaGolUqOXntI\\\\\\\/Nc0FfAFeBeT+FjiNcH3RLR3VfLJ5H+YrkV1zKjDUZFwH1W4EJlyisSG1unnWTuG2aFOV8C+NhMCiRFnm5q4axUJCE2GltvYtRpBromEezKC7hH9w5d\\\\\\\/fuSj2P9GId9QS\\\\\\\/WeFAGe\\\\\\\/oElfRANQluCXvk5vKfBIOEF2Vswj7yads23AckZJ+yZFiD7Vu2T\\\\\\\/SAdpGnusAYC9tKAwAKQxm6mjL1ZXK8okt6Va1hr1Dqi6lwcuJjx8\\\\\\\/vEtEB6KZmIe7ooFPOqzUKO\\\\\\\/G4F747zKJmLB9gGsY2B583X0z3AzwlLfgY2mPQz1xA8JkusWsotqVuNp3G7YGWKwgRqy72be3HEZOENWfOgSQZufYlXRlPeFbJa8mMyAo\\\\\\\/Hi3t50Agwo868J6rx6IvIc\\\\\\\/37Id5aG20vNig0nUd4T6EjidPuBDabyKRnWkOpFkMePxyOfvcQgbkOWGTEU+Kix8abmTrfHOy3Pww\\\\\\\/VkuihYqliHkzd\\\\\\\/UPktH\\\\\\\/t79ab7SJBZ1dVlzRYzv0cCb\\\\\\\/MPS0MjJdlpoHBVD7LN5Idvzv1LAA4ICWxbQozaVeqrr\\\\\\\/kf+Xy9G0JQj3YU+ha++KPIG2qkRaLf8LRpawf2g87ituXHWdzbl8H8Jy0TQMEPahk31Y6aVvP1AvJ1U4\\\\\\\/t+2pBn28PKknJqI59TY1eCbPo\\\\\\\/Xws0aW+kUHdPhEPToDiJgTx7wrxXCVfV+Mlqb5d2X465jmsMD5RUsgIQOOMyRNQjyg59hTRkdrtOXnTk9r2nC+4g3yPMBUEUjRTCOln+tQR2rLFQWwkzjOHTkWHTlah3Mo3Rflf\\\\\\\/UT1bh\\\\\\\/zdo8PfwrpQdoIphFvdMfMFFMS01TIeHvndrg5J1gwy1siYjoZj+OJGIkfPSg5YxBpF4F4aLSxQvxFG2TDOM7fox7ffUjagb+Ql1YEzL3MsbMNHMCBLmVryHncJxsWvruIVHc\\\\\\\/Ol3wadTgyA0XOhGeJnx6R7TVLlCO0RUXBYGvdi0Ry2Ffr5JT8MvJbFD9QEZHCPBW9VENqRiwG4Uz2YUBoQLm8PvpjzNBeWPfKmb\\\\\\\/fiUcUgk5gBqd4sP0BNliD4ost8oqMsse7lJxH7uZssd+FqRye5Imtgxz7e4NpdHdckzBI9CLFlFUE5VitKAOTPCAWsplSRC3ShUPuH52spPg2tFxgkqE6T8\\\\\\\/tLFoRKxrqySutRKfLXmtzTopHULMsHjPVzQ6SQNA5Zr1CKaRbnC13sTbHK04VvSDKW94Hpgo+v1jyBZa0wTxpXFdXscmPCGKSFMjbG\\\\\\\/3qyQzDuBrYnYI0xujfyyX2xtD\\\\\\\/liLB9VmS0qvari5IA2G7yaF70p1BSg24WeBMbb+4SliJw+7vzkKAmHdG5zG6FkCQJqTGRO9vU0gAL3DpW1k3x28Wtif+CdsAl82f3fQnZevAWlH71s+AarhujnS6cxorKPiO+TmuALU0JXByaGrKiN7cPP40rQLHddti8g5FJi2ri+3vcpF24aRSez+jXPiB0qWlkAE0fpQAlqLpscJif5HqvbAZrcZ7Ku66GhVaO8\\\\\\\/grPPoEqEhtYjEMFg+gwY3vwIdBMu2hz6ikmY8pUr7fd9xXwNqLh83Hewgyb0mc8ceJvZm+Kel2Hl4+XsE3kWiDKGRCS3V\\\\\\\/SBjLRAyxdodXLx6W4gUXjHxdx2H6eh1L533v7r7NFJ6ai7wuz6F1YvStgI37rs0QCKtj0ueJBO6GgbbM9+0YUtbXb4UpmKcoBZ1zw9QzDs8jo+yDTM7gGzYRmoPdyPVFYZKCiBAG6uBbksPKYC45ZVeQ0nGbzGU0Tl3W5sW2BkB6d0jY+K6maxcrr7iunEJWaqXg2T3v0VTkatqmQbrqnRy+8Vi9ANgcvMbtoIEWovj\\\\\\\/7vop6z9EOOzTuAMD5W2kwd6DFdbWPuNqCT+PhdlzuRtmHaxmb6G0AGOmyxkrknvRG7jg1Dno6hg48rEs\\\\\\\/23VyVdq63BZoUR7MHK9aBwlfU6AqrcvkAFBSEitUm49tbM2ulc1yXmSjhdkJzURfeTiRhayzhEsQRiV9UnJNuUdGCS\\\\\\\/U2+5VbiCsAT0bRtpduTY59kr832EnQp8mA4d0WcWKRN0LuyoAgSWVlFZPMnmUJFCQS9iWstWHRLrQqQItZ6+pARFvIiropEGzwKtAvkta1WdVx8AXw9Ve4Y4B1RZOvYW2TlbDNskYrcjYfLEfJBCQebmkm8Cd79x8JJpYVVjVz4285E4y81qjzJQsVeUWIvK0\\\\\\\/h0dquNqZ+f6vRSTXUkWCi8ydjQo708IyS7pj8aXRR5zcuTPqtRoAke+bgBudBV86fNV1lqD5ZBMOADeY3Yjl8n6BNcQEs0ueMPODwUO1ZYtFdr+1THD8RViJ5XM70asxWafT08d4BY1wLacF1RE5plrhB\\\\\\\/zYu0RslctMnwN+Vly9DtERZGAiO2RVQEC7rOfwWYbRltd823nr8xqaG0Ym+fCenR\\\\\\\/C8BXRaXlAX5HTObFvt6RHL9m4wYadmkEneTvuRInilQPfMSFrbPJcTGhoAevxAR8zTfkgMzlzgmuGPStsnFGLR\\\\\\\/bqAW7oiEygw1FpvuFbRS+DmkReY2G4HlCDJahmW2xQqAkqxUYZAr6fvpwuNiKy80E1ULAj4yoBG9uCMNsQEfn2DXsFnszq+cxp\\\\\\\/eNXbBsv+mg15zX5oO2q+nuCMxakw78RQeJgPUXX+NOhicYawS869a7L2as3juk46Rv5ijF5wH0F1JSCOQC1NLgThOIaGuE+mqrWNSL8xdqsX4QwRGY6OYI5YJF6\\\\\\\/III+hhR4kejkQekuUso\\\\\\\/neY5f\\\\\\\/rlDIWS1NSfXrURhy8PE4tKZezR0Vlyc7Zx+D9dzp7AZ\\\\\\\/tGpthxAfEjIHmbqLCXgrOAYWTZqMmzkhSv+L4S+j6WACdvYlo8Pre0SWy\\\\\\\/UrdGFKVCX86xXfqeATU8k8R\\\\\\\/RL7DlbfQ7YP3vPk6VMqBb56kYAmlAsmnDfEiCNU+Dqj0uB0VNbn5u3c8Zu15zgXhM9hP3M0BK9OmueDQLAM2pr7IniX+uz0zDxdlE53\\\\\\\/c+EmPqaQnliXroPBxX4KiuakUZs\\\\\\\/x3cPyq33u2y+Z2RMc\\\\\\\/kxRzgi7aqhCQktLKU8kcsNhcmfAwvlve\\\\\\\/a6FXjUHSSX07upBEgt9ktCOLlWeVU3VOY7Ktojth5k7sUjwciu5M6J1oVYM1JDnB\\\\\\\/vhtMn6gYt3Tyon77+o9OcWd5aI1FwGdAIq4O7SrQpixbxPkjI5bDR\\\\\\\/tJ32xs7zt74MeYttuvwqCovTD5PFRseG8byvqZgFRWA78fC59ouV6Oz6VhK2P8CkwE12mgb3YGPiYgjcSfVPyhH8Ly+\\\\\\\/ZA2Yy370T9WcN9Tkmul1wizLJ2hXGprqC5HgsPvFmh7QjIJYyqViwOXSgSBr\\\\\\\/MFaRBt1IdW9dz1FaIiXzUlEplcoOmoocI74HdIgs8JqLnK+RKfe5m11JX8XLvz8exy7MwWBY\\\\\\\/rK8ln+ahauL3gGFP3bOpxE7lM8MPn0yCNQKCRV7XlVPTnJySg9MIO\\\\\\\/HEie2bPPRJI6vHuqnRazXo+poi+5NEbYuRYSApc7OeU9Qm0MsRfSMf6JPbPuHz3swSeDJfOXGcYivCOCVvvdoVuiL\\\\\\\/YyCBGpq3p8a9B3sdwyABHUXnVN1Sn6oKhe2o9YS+bXNA0yrDrG\\\\\\\/Tg+sPCLBQJly1CS8yiWHCfU2oLuAOLygsrlZwBL4K7YuXMv9a98TPSxSqGWSP1AvJPZNgf3OI3YY8NHk2ZpT9k+2fcYutF3U8M3yz1yF0M9IkWOE6VssLBxTwRUWJzWHfZ6aXpI9dXuV17SChiEkiCq7BIsrExYV1h9MmUY\\\\\\\/sPr9rmczLZiaZbxGzKXRT3+GO2umYscS2uHK\\\\\\\/HYCOnj0PdFDiyuqxReWnTD+k\\\\\\\/J+2O\\\\\\\/y+\\\\\\\/Nb350pLBMVZG7\\\\\\\/jyRe3mX6EarxpsQgZR+Ke4LUvlEwO6QMhuHGqFUliRQv9GzERsQbt0sdv7r\\\\\\\/QdWNx\\\\\\\/\\\\\\\/VXOCY9nmf6UeeGAx4YryvlBBYpU7FkZ\\\\\\\/esV+rckeRC6J+es\\\\\\\/g8grkmQEHTC\\\\\\\/xHWNVtzIzLonfubwZzwdZVbkosLZL+XZGLmrDakwkWeVfBYUs+4JEaMJUP2z2ciB1rc1LMT6xpwdcQtAOMuISnBuJoaB\\\\\\\/5+1PHj4SsGoBpXDW43r\\\\\\\/yyHf945+Tv2G6tN8DX7iLtIy\\\\\\\/8EPobb8sSJxLRuT9dqj9xyM4qNZI4IL7IO2osQ83DgOnckGmeQ0dKgryUb24AdCh4KhLumLurKW0ffZ9P4Cc0G1e9IE3MiJZL\\\\\\\/8jx\\\\\\\/Wjib7wjv6Dv9j3RVWZ9qdFFMJmigZB7MUKpl7aa80CyFHrucNVjUz4oWJEPAjfs3jiuZftyIKDF2gBczGCkC49c7WJMLCZs9mIwIH8kD812oKfyBY9nQjb0cGt+iKtsiN3nLmM\\\\\\\/u\\\\\\\/MpOGy59LAnG2MGxUi0+KtK5FHHPylYAVAzkCJi7R4HTOn0uq3mYdmQ\\\\\\\/FWqsnfwrrDUnY1pFJLFoaPodi4D3AQNd6UWRSP\\\\\\\/LSfQ3uoSovt7jpkDqK2JdFxR31TXfQzbCk2idousxPK1M5zNwNmijv09WB4WeEYzL8V3QXgmJ7mUTSwEn49jH900qd1zto6Sm\\\\\\\/nvF4box6cREVf9pdpZc4it1trI3Z63MV6N+N58GaIlhOxVk8pr2tLfqZNv2ERU7QQub7lFT\\\\\\\/jpfTiNP2MZex8KzLGUyXtH3L0uzlXp3ahvP4iRBKHO3bK14Yq3\\\\\\\/H0df5\\\\\\\/Wn8QtQCrCNzjytuj7ltOjlwu41t6zeeHYAfHZZpV\\\\\\\/hNnwIc03nQt0CEt7RLsOO3NoG\\\\\\\/MkviV0Iywk5z5Kcw5K29sZo\\\\\\\/8Scasz9ui8iMcbPIWMYwzHkdqbKwRpHMiFnuk2B3yEdC9v4rDognKqbrCVW2GcJEWKRWqWEQ9N\\\\\\\/sVuTWq9FznnjTMbOSilMoKaaZDJeVa76eMimOdvBpf86vDtB6ZQ+gjg8LPS\\\\\\\/KHlax0DnihJw0NCbGCqoE+YqnjWAaCiC8K7y7XYp9TIvMqg38bqVRo4mrQm9Zf5vJqAb\\\\\\\/8OVC8e6nHDr7\\\\\\\/9EmmvuStZ7HqAKWfXkbAlV9mmu5KS\\\\\\\/HM4V2UnbuHKGVbIfs6zUXqXpsCENX+FvQzMSFnAWVG4FzhAX0Tf17laXDVd39l8J\\\\\\\/MSR1Ndfk+DevP4vXcuJ\\\\\\\/9XFQOWFfkc2GTgG8kM9Z3IVQAcQpBoRgB8h9NMzev557AZrXPehSkZTC3k97s+MpFALqUrlGiO8UsomsVTHVi5PNLSHs8OCCwrCkQXMJBcg4JlI\\\\\\\/dUw\\\\\\\/rrjgv+HC7iflC\\\\\\\/qw8zWoZQHgQJ7\\\\\\\/VCaeHg3OPiUl9vYhES1Yc64yVc20DMvSLbHHuf58NBiuV1\\\\\\\/tNhtpMpn\\\\\\\/mwa30Jn7H0njfaRTnSS8rHN7nRWXGN4\\\\\\\/JvRRrikxBtMeyttfYh7vt\\\\\\\/8ZgfYUE4Pgi+G0oXSyDxgllaMssfNDNFAkjqg6kGwd\\\\\\\/RNmBOWYxmMYjNicOVaqYLgozGtNUCiIzrscyCLrm7AKu0+0TytG6H4Ge3XZE6eSQJ4sSkraF5\\\\\\\/fCC2umq988RBtvzFXLbSJh0xg2jqk2HD3Qkz8TsgiLVOJ5494IQOksYLclEVyEUMNUz9yrxxIRuCKdgaEDmL5uts2\\\\\\\/DRoyewftgw44WT1a6\\\\\\\/hDh\\\\\\\/gZ+wRcn+Mj\\\\\\\/sbTKssYAew+8OwaV9pCBC4gANsmw0VfXLfC5Gj6KqO4Np8z6efR5B75cbwsq7ksKRca4vqz1AqkpZm5z2c7qOfLV9JIr6l6Fh89tvk1uKE7NVdsYbZEvNKlMh3Mz60Xpv\\\\\\\/Be\\\\\\\/u5BbIdkL9mv1st1kc1uwxDYYhmAOlJ3HxxgHRmoQXIGuhIxXkenc2fhPWFr2UYTZlrzbvBsLHB+Z7iXVWlHRqWO4HyaHBhdNV8xggtR2r9UHlKTGvlAGp0mL0MKBdw0oqDeeoE8nwOXP9rz6cqcF6rTSbne6DQ1rRbLlAZIi6UCYQzcBXeZC7kJrk9OaPqf+W075qgwpFuqXG2Hwf0s1XRQeuuKG4a1hrqs1E83d9p1Vbm+CHIpyMPLKIEXZr21fwPS18nO5dVfly5fE29Bhtg03Z9gwgOoCs0MHvknMlYjZ198jk7tGr8At0m+\\\\\\\/yaFSfggAi404Qp19cnV2sbC\\\\\\\/frdOrmWLBONcV0136SBHNFPKgSRGfE7KtdXfUppwBpTbxaTYyINtJ12cMojcXtt75uAqEB2LGWWEhOdPAOqgiKvubtnOwdsMIkamvvdFWbsyGM3UJTV3pt84W9gLJRMAZNLLxqkgwXCVvAOwmu0htc6Rq61DYFyCvjyKawsCrjfD9IeB83wzYFt0VmJQ8KqSjEIZKb6NNAaPUd1h20MGCQ0pA0U8vqqkKKPFy+CRHdESF1wBZl\\\\\\\/5QcxsIPBQQjR7Q9qjN7Ta057XVYcmV\\\\\\\/h8K\\\\\\\/plEG9Fnn9aVHDkFz7w1OsAwvNfuWFLy+Ke+mGdgWyZ0V4hXTRsyb2zcSbxFFlzoQE6YZeJJozr1CbxxrdhYFBwzYaFFkQbpbj9ZbbnJCOtqv2YppQZFZVWarqZeiuchnbu0XEcd+0w0Too2Sg4fn653BzMn0s+f5kv77cCgZ9QOUKwtSvDFAlbwHLQ2r41ZHeAPwKmD3ICikwDEDBgwHLThRQiL7HFPU3LWSYO0ilK+gIGou+lOJ3qga0g8wx+WlLnyecEkRPOUkMuXtMuGzdGbmXTvZSpjGvsgHzAJgOAsElqJpjUFwS\\\\\\\/jwBbHB6YkWQNhIVUOW4rFgSNCWxrAXpc1BydbiUuL\\\\\\\/2ra63ZYLAOwGzV95uI0R85Y2+sp2VGrqrlOfyX2FERt4P8MHXKWQWV8uGT6doLTpb+UpOw6iVIAY7pIaoq4W2HnfFf4kuxICCcTTSkCxSCH2vlM0neT51oYoIbZhmhcEu40YTOeaEE\\\\\\\/b2oG9KmiUbk3imQlYO1cDhwH2Fr4fIA0SHlp76IGOxsDXSA8x402d68yq4WMOj7qZfH1XcoN7+39\\\\\\\/MFflEbvIkoviWs6pR4hyXrAIAFNh+v9AoSa+2o\\\\\\\/OdlzUtkYBggpdU+qvef0u5\\\\\\\/toBB+yzLnA55Zwtj2CF75mWBs2rp4v2Z+asD75y\\\\\\\/Zo6D\\\\\\\/D9lTH7BCNChK2ZA6Uy2HZpT754gk4viBuqBOaG1KPqkAXBBsRbA21eguR578DhiuG6YZhW8H5hCSXRNTMUP6v8BWP2gF+uOfjA3wiYBWQGliOZIhUM\\\\\\\/dDmMcoNO+KP4QbU618Fk\\\\\\\/GwiJI8\\\\\\\/ou2+fcs6ZdTLCcWkslzCdN0uZKJhTAFG0fIS9s0SzKAwkV\\\\\\\/4m99MePt1UL\\\\\\\/7uHY+rnEgQhTdGJA9oz\\\\\\\/WxgEt3y+4W\\\\\\\/MXxdMTzUY0Gs2ta6gWjEwqsQiosR3J+uZ2eXFQFGlr7X6c8KrZo5tz69z3thpx1tSWS5OtrMO0hCrZMVcmVzNRGCpHl5DCu03drIG5Z0KT12Q1v6ndtIxFoAygy\\\\\\\/V4dbqRVmFH3y109Qi23FtslqIRo6S3AGsDtzsXzKmvhzwwSU2CsYl4U9FaI8a5mQUGvNJ61Ss565EcbCmsiXEC9foXLIMyKNkdQ5\\\\\\\/+MLb61C1fm\\\\\\\/Q4h1R0vCzgy9Bfux8Ua97wqEaUX2Zsp+intpdqxoMlFCCn0Ta1G0eUWqhV7yKUhVMxFxFAkrb4NOrlSaDMCIYM\\\\\\\/7BcV4hcAKG35ytQ8i0+481KOqGqMlxGvjRsXV27xsXR8sG77518Lh3suLfRdvoXgcT7uENiXXwx5uw3N1tg2NEtbJQyeL5J\\\\\\\/rfQ8JPxZ6IZ8GJIo3mGx5qOZUAcbYBEIazb7LYMfZdeYw5NCg6NV85\\\\\\\/6DpK2O9SRJHxFgAjvl2z8rF43bcE\\\\\\\/9eSkMQxyZDTLTdoQu33WLBNPvnM3ee10G7cq9HMgN4Z981IgKBeHuXGzfNTCHEJbHTKlI8hFxJhdN2ZKvbK7LWA9XGbT7hHF7kAFJiUbAHbFgE0p0vfxshYPBIdmiAgOnLlI5OvDFWvFVw2FlmTJxPp\\\\\\\/K4XdYf8g2cOJG1i0PY3NZjv3gqWoFJbACFMsGnIr00hJ5DUZREyPI\\\\\\\/Fm7McydgQsnOK4TudndpiFdfyXFAsp6G7S483vwWv2j5LnlcplAsRRzrBi3lw\\\\\\\/iwB6iG6wHh2BmVwQXwgyF+md7OibWEE3XthLJBImkYwAgxj6p4pOLTmtkYycurmQk\\\\\\\/U49NuQ5L4J81qECfWkeV6115J3+qsYmlfZmV7uIfliQV5yBqhpZsxnQToPkfU3msyR9UVxUrpQjqCzbF2Or4N9bZH5VJrDZhbcgDh+B6UQ7TG941nuk\\\\\\\/iOdlvQNQ+9iMOKAVcCFrHQDxdxkrLk6semDtN3JJ8jwdo0F1fUF1GUenmrrgzJhm+xLFIbi3OODSKrNbIUh0tUJeA4R9SjQjN\\\\\\\/gJXIp8\\\\\\\/3X5bmJUnIRk2C4gJCgP1i4fCGdehgXylqp1Wse6YlFsjqHU09MaSrZTAt0kMiG6i\\\\\\\/Eweg\\\\\\\/z4h6+jn3Qbn\\\\\\\/nAP3lOHZBHEZ0+a24RnikeeMTsN7aJ8IFj95YKUsFrnkJJD9rlQgqgZd00MtO6RCrosPxNfbMSzdUMCZFIGI12Ey2WrhFFKgfA0u6xC1TFsKfLQZiXv3jMZIoNvOAd+Tnw5u4MRI9s1xbud1+\\\\\\\/WkhIxXJTIPUJPcuM4yzepvzDmxM5UFcZy7ynC1q0qIgXyfbaG+2NQS9cDOlr5c1NdIiBYbioIWGFR5YZd5jrCVNLJNIg1Yv52Bz+BZVoj22ByiKCSpaXrv8Y3CYwNQcJqPzUjfj\\\\\\\/7J0NdMpmK9MbZZyeXLGRyrrWou3nuX4iJa9Epc8KmlNOh+4Jt3MEuK0+SoVTJGtQSvHMVPBYLdFpcUxn\\\\\\\/OVpJIEaU4J5WfTbf7G\\\\\\\/AjNCqgZAotknKwHKABZm2BTyeHX67ALNHLIrT\\\\\\\/SpfKhOOpejfshz2HcH53M1Rm+WvoRL94wO2vAu9oJsyLATlEiIu1diHPlWJDCfIcQVbSL1+UEQ0xiva64fPZlAMMywtlkuQWtLonrPV3enQ62pMO1hAmZagv0+8kFW4E+azyAShgm\\\\\\\/LLIf1XwaT+G2JrqgqSM1mWCtYYlTwqd0jFBs9R0TvUkTEmp3beCHFUtPUi1la4a2ZG6YeYty\\\\\\\/rPKNvkAJE\\\\\\\/zoD65BfA5ixhHt2L\\\\\\\/m9XNUXs8dpBUIu4jBHdbe\\\\\\\/BfOkgGFAlwJIheGclJ76OPpnULP3mIR4ACQ\\\\\\\/uKra\\\\\\\/EPLAAOmejGtuNo3WqLZevZSyZy5wA8qg\\\\\\\/oWUwT1IYc5yB9eRxGXNSO3pK1q4zO3xuNyMmKaSBXmlWKByqFgXtrwWpDB0FLbe5AMcLGoVdgsmaPNiSSQ7lgZdj26od9yL3IGMpeCNvBGKoGG0zoeUWkfWtWspAwm5KsB58I7cN7RtjPIGFHJXRLT\\\\\\\/XN3Pnk\\\\\\\/7wWm8SxBr7R0hdMFI9RQSk\\\\\\\/LI8bvxvkJKUC\\\\\\\/YaMsA1ZIVev2FbmbM6vVkYaDbP+USwDTTuVxYwHLKSsHUha3DC0Lq79N45KNG7MHz8U5eL0UZf6jAeIAZAv759FXoRfgkKL4zGMIWiZyB6uxvTsNn6XjfZuQODMzt06\\\\\\\/2\\\\\\\/9l6sf6GKZDasceNTBKTicNe4jonEAFxY2ukL180VaYUX2FB9rabtzgZ6l\\\\\\\/6wo1qU+oV5\\\\\\\/iq0jdYv5huNtac6pKr5QgezMNoEdZWk4uumN2iB213cg8CmlxGZZabSQEIdWQImW\\\\\\\/dQuAFgx3VCU20bm\\\\\\\/0JVB4KSsql39+cEd\\\\\\\/z3YjPqGjAxeDdZcDQ43tf2j2LCWNC5O+nGRvy7okKc7yvp2mN6OLITWmn0kWSIkh4hdz72mtAuKE\\\\\\\/iBYjZweY8ptiEVdDt\\\\\\\/0N5uJcxS0Fpw+OjUiO2p1emWHaKK5i8Do60g+yqGCWpb1Q6Jml4MzB5Rahrda80UCT1Pky\\\\\\\/AvRTerJm\\\\\\\/t4IAFRiAnV+e\\\\\\\/vBNIYxUD8Z4nHDfGYr8de0EBg5Zc9xlg3u2dCT3dSUpeWTn\\\\\\\/Br6++iQ6ZJTEG8Epl6aPbqD1hBbI8phANHVVIsA3Lz5ANhb6SraKIzgciXGzq9zWalxfHm8TtEJ7+naRyaqLv+LjIStViidQwzYPS7OS82My0uVaVoM88UAtYp7Rxqz2fn2+IgIAv3c3wbQA2zzfBlhUJxrf99IugX697BAm9hc7LkeECw14K5xsx2F4hgwtnmPsn\\\\\\\/AsDHaE2YJ+krwm9bh6ue9ydhTd\\\\\\\/LStJXN4s8K0XWnHv5vnq0GyZVQJjTxI54AYqtp4thx7Alud49+QbcoZ4jYGxbT4jwxitDaQuP0zfZ+PgF6OuQArnl+v62tcosn74bJVsJqn6bmlOlru1pz\\\\\\\/E\\\\\\\/17J3bGSQInuy56ifwIUxsoKncaxMOSkF14MJZbKDa5t7fWpreZEqk+cJ+QCtNd9+0nynWden+VamMDBjaiLUJy20Aja85w9RJUjKn16698HMLHvuf2wDvNu87V\\\\\\\/LNRCg0sgVauQZvxQ2PRX8xSWzUNEUTJHAFCr7\\\\\\\/qVz6snM+Fo\\\\\\\/++Sz0sl163WPe2N5CBmavIvh8j7y\\\\\\\/bFYUAhy2inuN4j2KUPVwOoOb7aisp\\\\\\\/6CG58o5Ugansycoo+unWw2iRk1BFWaW9XwgEVwp+wDyhnMHCfVuhxePBQmssq1nm9B+8MBLOBKSrfVFUjJWWHDUn3u9lbJ5bLtArUcdp60Sox6iUG4oUL1FrCCNkGlT9D0j4oVt0aoNJY9hOmzD7EeAUGib9xJzNbc42ZbgaV3IIYklXfgtcsbz4euxPEsctSVKtSK1+Biy36Mazuz6+Ou0uozpyBy7wmnXbs9S+yCBzyF9DQOGf+LqGEs\\\\\\\/Bo22AJsPh7a39shM0q9sx3EH3s1cZAnouHFNSd1bIpYm6NTzqfrsEyEyMvFOq6mnju+mhcDg7hSN8Um3M1dT0szXFu0mct+kBaou3eoMsAJk7sjIgTSIC8UzzYp9R2qN5+ywiDDkgN6UgfbEhau9k0gVCWBpb6F+Qg6MIVlm4Yr3lS8i8ukIVvnE3DjMScIhum6GWgFACP6\\\\\\\/jbuz0RptVRar7dkXtRckxUG4tWiMzKKP1o5di+90wVJ3xOoMHHjgQluxdz2MJ9rQsuw\\\\\\\/hLZFMKQ5A2mvLBOjl\\\\\\\/4oQ3lxXX6LoUh8Nq0Gq9x5UCkXhaKU+4\\\\\\\/u9MgTn9rAfVLw5Uniy3hlhOnu3Nt6Er9mM6BQMKNXb0gWpHZYs5VEYuF4WbXCL3spSH6Le47d6XE7vNyJg\\\\\\\/L7\\\\\\\/I5oPVeMLLUiI9QtfQha\\\\\\\/ICdOYTZf50AZfKcwSEO9xOynKgZVs4yQjKpr6ErgrqJOJ3AWZTMHWlBQAd+F6ioevw73EvoksXnlFvcZLPJxyQwYsHePo4h522VxpOBU2LjGC59ZHejbysTdAomOLUr135k90QomTZxlHjjUaOM51Umqo5zmbF1tEv++dJqrBVmctPfYlBOP5y7+jxA4ZnlQlHnCT693lD9laU6TUu0KvdFKomwbkHAEtZngiZ9qCHAAJTehz+Ctzi7rmcHsgjCb8\\\\\\\/DBu5RPdWrhk4GhlJ06UO92u9s70T3riOdZGkeeoLhBA1AdedSUShGYJNlwRjNI2c+E7BLFeI6vpPzHiFjwfHmKnhyKN+vXvKDBjQe3Bd1seRY9\\\\\\\/WRStEqs2gKhj8IKy6mnj6EL0ys11QHUeCMIlqzIM4BQIEoQuIqLO9ZTexhLOUD85oS5S6qs5QJbsmCIq9h44Prp31vuHP0hFaK8H8xjZqPn9tJiPWdHCux\\\\\\\/eF1OSmqw4MKEpgaa5tdwVHN7o1E+xN91uhChdRXN0L\\\\\\\/rRTIehRuXizVAQhawjEXqb0MZkLuUo0lSvB9s3cOo56peVWcQt\\\\\\\/rcrcX7Cdba9hSVYpfi5lzaTUpfjIg2Pcpv0xBQtI+\\\\\\\/W2EgV5Ft+wpLABC+fXxfEsT7FooZp+AmPuPHfZVXGyRXooQ3GfGY\\\\\\\/mQ1dTTO6oYw3LK\\\\\\\/gO2tWQvVDYijkqfnry+Uh5\\\\\\\/LBWlCjSCZ\\\\\\\/77hdbQ+sja19qBK48ZRBtpR87XFFP92W4ZwYUzXHOa0aXlaqXvBXq7VH2OP7V7tLoNBB\\\\\\\/H5oF+mFkf1imeBbQfbZYdHkVR3wbB0ZrbbLTFNm4shqDNwRUp6qcP9QjTUvDbqDwu4Sr40zRZWVHhwzNoM2eXehb4qo6Iy9+w9gxbIqPZ0bvQcDcnFncuQQ1\\\\\\\/8Niwga++J2GxnYe8+n2G+S8W8xxFGiQ3ocW0m3ZFLyCW7Xy2wdK96g8U0Z9paLo21cQLUReAkpu+z7V9G+g4mcZrSZY9zGK00Rjmn4jK3KJEMZ1+Z6cduEojn\\\\\\\/geuHSuXfAvpWazkfQPV\\\\\\\/X9bS26Yzt5KDj1GDDYmYv06EE3UGZ\\\\\\\/TWisJ7HwgdZ6Y3fyYw+1Rx0ISlJ9FtL+Yo1lkT+fJNFAqmo1i7s\\\\\\\/uvy1Uuo7GKyOMQD89Geaq2Ui3vcRixRudsB5xr8dlpgQK+HvECtAd8YjDhmm5ZT1RnrfXjxxjVrrWEZSqvndpM+dfXKWrlUf+kGR\\\\\\\/E9cA0uJGYV6tZDgukaW2k62fIbgN3V9egzyb9aeUcV6HjQ2B5bbT9EfwcBv68ncJUCWw7lcbmT7j4RcH88ibNngoskmKTypwtjSnI1jkc3xEuM37o4F2ueX7DeJO2\\\\\\\/feKfc957OgajJ59AJyfpyS4nLsBC5byJLrcSMz3HxhCAo+80MhrGEWQaNQyo+sQeBFqF54WSENsntdBQ7ilnhbn1df46Q9P\\\\\\\/NsQFi7ivQw4RLJQ\\\\\\\/jNlVDqmn44gxp594YWY1\\\\\\\/\\\\\\\/hJ1jterEm5VtJgTkgv4xSAymLAenrlgax8bS\\\\\\\/as876kplMvM5QSzh5rMqBRMNL\\\\\\\/A0bLAqV2gtbvHQsUCXicvItsBBFoVZKMZili1bQebEa0IjKdqY\\\\\\\/X7n8rZGMfChwEF5f2KoPc3n0xrOnkAUgbJzDWDFXwiiE7xQIMfGTytTUf3OihRum4lPmxBG7QjEjfLbBjCe3IvLrYuddS9ztMMmhafEAxTP6+7uOMBsGpl7RuUZBEsXJYd4cPTRc2klt4MiKR+S9aDYb5NrZnoipyNf8vatTYuN7n9cy\\\\\\\/S\\\\\\\/TDO9F8XmDExxiUtYBg8GbqoqeueioK8DWY1DwxfiqoQ2\\\\\\\/KrJTiRCaEPOgapL3vHo\\\\\\\/dB50aDWhzk02VTOwYnoh55tH2GDmNd4UURMiUWdPBT9D77SSn4Wik+bbS4XY5noUSEhnDMYOjlJsHrueezIYi5Mcegl6mLZNALGgzCFVyqwztTmi+p690ji\\\\\\\/l4IzZnqm9cpK54pvSRcXLlCgg0n+Wyc978k3OpX5Bu2UY1bail3RfstSxQz3oKXse3ZwCDzOHDrcKA7QwI4Z1qaB9XaxeX8OO9JnJv4XG2DvO0cPYlyuwtF0n6c5I6yoF6ppNjZFuXLmAAO33q+LvPz9YlIvrrg3vgXF\\\\\\\/wBekX+v0O+agD2N1vRwB9Cshk9+ho8Se6oqBvoZX\\\\\\\/8Kcz2hi2\\\\\\\/d8EF8nCe151oBWRBlyoBZ3opvFHdZ+VJj+33N0CncWaugMg47UADvdGGbrKSujXXiCYQNEIDXXh1jjasUYIT+ewiAp1b27JAJaIW\\\\\\\/nuUX0PUCL5uF3EDbdPh62unIXmvhPtsXWGLyvDeV3Cy3kbbQap2h2CeA+4zaxltHaPADBwKwVIcS8n9HfYGjc5F6QYpBwjL+465VTiVAu8OqcNxgLIbjJODSxLqBQGNc0k0Nk0PuqVmN7jl5JYysGeazX6iuEFSJTx3z6QLj6iMHw6BiqgpSc2M7\\\\\\\/BlFvdr6NEiEPwQiQDXGZvewVBQHKs\\\\\\\/yzW4gTRIFVta7KQ6lLH42msx8xzyNK5H256GhA5cCe2RSWER3FwujNGCECUSv8P2bqmFY3hbGKryMKqGxqGa\\\\\\\/it36nQfnNUxC4aFUk5cbBF9Fgv\\\\\\\/C9xE6Z8wSgTQo4UG4ATEvGnZTzesaVCRpBMQA3EddP8SN9+456k8GTF8ucKaVzx4cIo793m5baLst2L+\\\\\\\/of94b62Eg0mQPfztMxsQG94NNpWdkKYXQ9Ohzrx5juRkBc+CTRmpAV2qlRQXdL+TbXuG6BmJO6gfla5V5u6tBTiotgI5h62qIQ4fxRpw\\\\\\\/0868ESJMXVZuOKh92BdvgsOX6YN+MAlz8qiJkvM2wTvlRPiqru08bEjjljp2VMZm6argJET99+n8K0Vm7tIJzjYbcOlpg7QfDvpnL+MwzdJf2XZPJ4MFZUWLA5sjlBTREB8fD1rAYJ71XcoGOf4k9\\\\\\\/zVMK0jPecJDBMgAMmfziYCw+xvS8dFTGoXHUQmBdZEYlZxVQEO8FZI0Q3N7fdCjDQ3FfAkP2CiBt9bcO7bYJCzLies4OOH5JPxAx8kQ3vAQ1eHlRWUAoE02muxuBjXOBIrMuWXINQyQkJO6URmApUtOxqVBtbDN+iYn0ojfAnDRt9bxQMVeRgDV0ICm4W5TCz0NRgxlD+4GuTqZc8WoVKX9GNFn6rG15r\\\\\\\/p5668r8ELEuKsj6ha19Y5PMwhHOt7yr57MYOZoii+qSQr\\\\\\\/H5LyCEk88LsII9GGtpsbRNn9h4pMWfgXh9YFuzfD27\\\\\\\/3ygw+e63kZdSwbG2qwChRg4XxjXRNPz1VsrZfuQluwWq6QTi8TfHDPih5e+kZaNmKarAFUrhMGBw04+ukpo4HCcH59OY\\\\\\\/1GmbfXAegzEb\\\\\\\/qx36enp+ms3nMGvoGND1N+0BvX1TZXBSZDf6nNGIT8imZu6Z5jK3A0\\\\\\\/uf54x\\\\\\\/FHxYEizwatE95EI11gVwfB7F2+xOaZM4cDwzEwf4DV\\\\\\\/IamodfNvuw9xf\\\\\\\/X60vmI\\\\\\\/CLPL7o5wilFJk\\\\\\\/1boj1nYeqiXfVV1Q0eGHmGEzjWRiDVFcaT7XW4COOlZPd2tObo\\\\\\\/JXpo2zDcR15zdzhZ1l2N1VOFSaSr8S6TGjm8ZFr+uRGnGFSg5dZFW\\\\\\\/9okpoVSNJoS\\\\\\\/CLqwi2i\\\\\\\/gfJSJvB+MwHn9AVOYJP7xhDSz\\\\\\\/DK+4h1xFoP1BPQFn3PjCivvWZZV7Lb6s\\\\\\\/cs4kxTHWCP5xUspGtTQEtgu69gY5JJnBQ0C4FD3dmmJpqkBbVl+YiTCzwebDV0cGB7AGB4yTwRxQMmg417Nk4bMyxv+U8zVExVyd57Aa+z2LZoU\\\\\\\/VauaQkiswADUZ5hnpqwc5C28aNqJOfW+YxMEycWl67xjesLg0qwzy0SjebhR6N0FBLEqJWaja2VhyPb6kZ1iWG\\\\\\\/mAbk00RSR5J5JXzJTwrg93wUdtB\\\\\\\/z7czElFW4rb4ZEY\\\\\\\/nxqo9lnhakDV7gr417TBOYYJzJYE\\\\\\\/8393Ye4bs4TwjCggUynKR\\\\\\\/oPd05OQPIgr9gr\\\\\\\/rb6bB9RA17lDbU2lGOIgMoADLV8RHi0EHwKtYpsOOWlW6jTPrPFnLxJJhU7jQ88JZhgn5HwY2UMDMAYrW7BLrzjfrkKdIq\\\\\\\/4lSK3cS682HqEgiLjO+JWFDfqXaMwe9On3TKqyLGDb8hdvq+g1cSKRsbMNuAYUqATOQE5om35Y4+4dbj0tayz8rikpPHydF8uegJap7sRiwWhKfyVDIXj4XPT7tuK9x12Iq5KfgVNwocjj9q\\\\\\\/G+Fw+bfnHq7vAABkrVKSddy8OIXGG8JR1K718fhMieMayRW0ib6KRf\\\\\\\/gY8L6Y6jiYazxGbA5zA+rGc0gRzC\\\\\\\/4g2M3YVmWC5kh6dEn5EsE5yctfIUyiOp28wMCa184VN0BkIiClRmtGAIQ3zcTQL0gokICip7ixIoTFdOcl1V8I\\\\\\\/z7QrMgDFS8g0JSZednje+fa4RWotSXY0YciffzA1j+hAxqSe9XK+1sjn5f\\\\\\\/C+cj76L5WYaZOMk0xrt8iBrmp\\\\\\\/1RmaqRGhPygFHn6UATYfd79y6Ul7Ge+JJiQPpNZa8GoW0GCgwqMKoGk5uXTXfxrqRq4jWOhHB6igLGwIyrL+VnX\\\\\\\/aQ+TNwU2GZfBKm\\\\\\\/y5RRoeQhgrFb4dBD2k0vgLJPwAj6rhJmcH6SeFVbMme6dvGBYI7HFSvABGJksTJOnFtmathi9WHzf+f1Ly65jpV9US5cIms75Pk2GFxukYih6J1rggx4sXvIw9BYpU+IaOPM4hJr0n7Mr92b\\\\\\\/FZh9h35l0ywMbsM8+b1zYsn\\\\\\\/Hlx8cKHe5VVRiTUcyClo9R2LhOxVsnfsi0oAkY8QqhngfYRalnho26oDq6k4c5+gujsFC\\\\\\\/Qyju7n5DxBrSfHd6n5xWpi2ZmutZ0mgqovE6r4TSxdDkuBqK7YabOAzHVrym6MF\\\\\\\/5nyqZDpZz8UTIhhDezhzUhoq1AYnRbWzMMnnH2dir7KWJl2rDxGA7edRApVOMkYJk7YrL3oqCrwD2wDBtbTjwA3Npe\\\\\\\/2gc35Yv+ghYYJ3QW0U0yo6DQQTVhYQOJolwh9\\\\\\\/L12LekIuOtCyvVETq2sonBEY3K\\\\\\\/VrrGq65G5KDn09Ibh6DdaYRwZbPwTnTL2OtaeoZudL3HNyXpFrXbUPBWE6U\\\\\\\/RPCmvkGV3d+jnXADbiWWsJ21huWnWZKU+8gPbx3NNaQOwusTVJfvKhAJ1aAfiGEU2XdJRT\\\\\\\/cSr3S1kuI2tlWxfnz1i6ogHQB+\\\\\\\/cstjhMxvQmJNAKwauEtxPRz43hzgUKJFAiGb7NCd4MxlUZ+63klAbT9hIskrGBHb8O9Gx9njg54eDffX+w3LY6aYR54aP09z3bMaiMpyfuxN6eJRiWe\\\\\\\/37lLGcUjWST7VYslgXMqpAT1GGjEbJcdndtqaGQiikDwesmJiEWGxHf\\\\\\\/2iSy5IPYeXAu0CtXF+uTGL6Hqr43aFPEOGG5IQP2OXnmPWGI8tKEv66xySuxtjagJohj0jjMdgZgxPrkve3VzWWY2H9EVoZIgHYto+U+zKrkruuUBareutQn6HjDCZ+aCDr62D2qBr1ndfxV+diQ7wCffxoWJk40Wd2TI1FOL1sOU98Omg3N6aEb3Yn+ft2k5YvMnhvSZe0le\\\\\\\/05pfrCkJGiZmBlBUQetZmWYmtNIAA7w2z4ULCG+PiLTIh6hhwdHW3p1BKBjuKDQX9U1BPUrW\\\\\\\/fyyjbldvfEUddxxv1mktBK4QBu0FJTHD2vJGGRlpkKjsvKg+YspAcban6Ahls4PCDZLn43lbDwgfXVb7FEyiZIaggBrWNamAQOysB\\\\\\\/skO\\\\\\\/cepAyrsq+IzEwoEJ\\\\\\\/DHZgmlYbXrr2K3q\\\\\\\/retJaTpeIB\\\\\\\/hrwHplZWGdvWKRowTMO8Oy2mQ2arIBoPlujKytVfpTWcOIAvo2Czwe2Cgf7czi6LLQH8S9ysWfREvfDkZ\\\\\\\/4w+7lbGBT4r\\\\\\\/Vtb+OrO6jCrgKffWOGFSTE4WAP857ATlTs0asfcbpmZSMR5OlKyWx9Rj\\\\\\\/oyOwusP2bSQ7dZPSD0lm610rz4Inlo09KbgjMDSWwjoOZo41WU0uWVMxv8\\\\\\\/x5ArXhFe1a43ddRdSveIBvA8vOmtCEQ5YXOGcC8mZAl7Y92f9tdofB3X7\\\\\\\/YZseRISAQv1eFTUtq\\\\\\\/zfbdWP5jIwrvae4aAG9bwoam\\\\\\\/hJvfKQb4eRrycjjb7VnTiLGM\\\\\\\/jAsjXzRUXw8pmrtHay6i5y0apf83hGVryip6SQB\\\\\\\/oOF\\\\\\\/oOM8ZpBhgNNxhVZtAWSJdix99nudwltWnhJrz4C8ELiRpAcTvMx9PJiLW3wyNfNA49z40lSRMJO+qufAlyHu9r6RuBPEsLu5191gNgGoCSk\\\\\\\/YCyqweMRqABl3DMUIeFeUKoRvMU6Iz3\\\\\\\/0Y3GbffydElIe14dKwcAU\\\\\\\/rD1StH3yfHqWrbl3Iie+2A9TpCVxsSVo\\\\\\\/DIe5ghWvGVg6q5cud5WNkf2XOj22h5Lg6LNCs+EkE5xS7p0i9gczLqDAJPMlhZoxT1f3mjy4SGRa2Gj1fiLcjkUBK\\\\\\\/JCtvwRXZV+Y6NfYOYDcClglXFQZ5q1nucCbmaqLYeZ2BbHGg+KqLXjWItCYR\\\\\\\/2KI05w6TcSUB6bU6H\\\\\\\/Z2hmeUA6tY9U3UrCnlmCTVlqyQigOf\\\\\\\/lBpoy\\\\\\\/lA3wsFJyE5GsiNfH7Va+HZgCQmajWybc+T+AQmJ8vK0iEdeADwUYdDL3Ns9QY3dMz7o1pCu8gxZ1WmidmLfw8snDlXNpnxxzSnJMkOATY0gSI7AEz5+ywBAlhp1QMAYzfUypzlfMHerXOnK\\\\\\\/4IKhcS\\\\\\\/4MqGX5AnxeFWXefT+Suc9mxtOgjOLV1NojsnLU2oK1cUGkyYxUkinsd8DZHV9jkoZWR1lXUwnvTGHQ\\\\\\\/mzl1RYt3cQ1sdOwdtgKciZ\\\\\\\/GLZziwIHs+4y6sjf6frSASZ5rSDlOJo3v+S8OmyiAos7t9FUBe+dqtJhNUEcrfbdGo0xPnxeROAeaw5sJdPQKsCenbMQFKThuLUj81qpcDt9SESgqZzC74dKL1Sk29q3IA2\\\\\\\/OPaEyjnfbRnZ6PAYS57a2pcvBIv8fpMJkIueX3lHSGzOTp5DewDt3UaJHKNB+L+HpTQS6EUDx+ICKSIN5\\\\\\\/3QBLQ+l\\\\\\\/g0NJRp6AR8vkUTW0nwEjR5PE7Ahy664sJkw6Njg23sKIOvD8TzWFPX1xgZnuq8W8NU6MDYu07uec+ttx0JPcbhGhtFxbCtxKQcJ55GZPre82QoMEIVmxXgqgk4YEBx\\\\\\\/bDOuYvL+9gyIcFEClSF0xG5vS7sWKXGkI2nShcKnavOMtFb1S2DsW1Y6XT9kJ+pxX2CtiHdVqKHo15\\\\\\\/dOqYz6L4vv+KVfG8ByViBQaf+cs9l0ojkMQIOsnvObGJkFMalhdrjU5r7U5BbRMnlqwusCi7ZIn\\\\\\\/6tRpiDe2qWLxLtwZgSWrvUOBHLpd9i6NnwTqnEXTRdVjqvLNfBuAr2Q6rtx6FWBlzSExXZrwZBUPPieu51c1+DLHHNokVdvL\\\\\\\/hRaTazBiHbqmIja08KbXS4yjOJf6TZ1kfRLny\\\\\\\/k1loFgchfumF9alBO6p4xYLwcEb0xahIYuLDuDYykD1uVe2kw9Yhb9bMOBdtmDFX0M4eewWZUZavVyeAs5tURpLpsegpMlqJrB12zuYZS+QptBuA4QZweojten3cCOYFXqaYiqqyjxGHfP1pRl6vyyCPhU389XpTlVRMfw9GtI1arI9Id+yhtRAq73Bb8pRgshjDLDEExBSIezR0VciMK65OzwhqwmIhVDI7hIebsZmzKQXk+Yb4\\\\\\\/LCgfqcUsDqsJ8vMzOXhxfB6EpaTcb3jEDnebmLoNar12I837jyfJI+dXAkO0PSovoxaJtVleCS8RfLHTOIaSrFfA+OfsrdnKXsgb6CgNfq\\\\\\\/yEjxlU1Qc9sfxRlGQ+LyAko+O6gAcamQu6II0E8ntPbxB6TgERLGtQK9VaZMGyfIXX2Yed4ku5ocCAoBFC8Xc5dpK48hTtwunn4hjRLSMapfmKpk\\\\\\\/KVP1RIZ8ZR0pJz\\\\\\\/bj\\\\\\\/6bsB25pKEmtykoKOAxRxR4g8ZDZhRv9H71Tlphf61OSP0Eld3fM06wSS2Dir5S4iZ9ytdfPX9fumHawp43GD1YjrYIw2tpMOfDkJtGiVBPrEyMNcLPdw2ZpoHQWZQlNxPrNwsgFpOmBq2jdQUzyTgbDLGeBHTt6g417eTfwOrmdoGmye+wLU6NlhoWKCzAXt57qjjX3Me1HgXpSaFcMcwO\\\\\\\/QqLjSQyIGeN6YtXVqLXpyG7NyfySskDU9sWkd+hL1luVOZPj5vCCQmBujrkoQmGwtt\\\\\\\/CalsQJQntLb9RS6vVQW1P5rQc1aN3pknqap3761Dp5pHM1w2z6Jh60eOD2nw\\\\\\\/UifF4Gy0qsxl3641LFNOTeO3leIb+9whw2TMBzyffF0XHbPgcmbAQULC8SM88bgS3fLDveyl9iNRcSUKsLUVqpyMmir3iDPNUulCgODU9zbZCgRbADK7xW\\\\\\\/FoBYm9rpH+mM5gciuCPxzbItU4+kR\\\\\\\/R6dqTM4YQ6vTzc4cJgSSWByIEpV9FFk\\\\\\\/lIWP4hVUTXqrt46f9wi6bnwOS3jm2UA\\\\\\\/9wcfsmc8YsglrG7okUYak7ztshxYMB49vHAjUh04Pk94GKXV5E5ZJxWQmdDy++g5UlZhpfxBA2zRHUtz2Gb4jOBsqEab06JPj2cp6HMv3K6m409s2PeZnbtGPSidXiXTKs59750VxvPBHA9dtDj7essLXXOxt5bqQtNwT2HUwOh\\\\\\\/XYpTsvS06xMAXr3XzNnt0vljmjsnwjw09\\\\\\\/KBqpY7TGvkmEVR+swMkT9EcBWi7KUOJwn3\\\\\\\/WQNwckK4FIqPU6EnYD7\\\\\\\/Kh8jWjPhTSzGhPph2B4J798tH4PfHkKoVb3eYPi4EP4AHa9tb+jHGl0niD8vwJhR\\\\\\\/mqePe02EoPf8jKrtcTQ\\\\\\\/AFfmeXqL+kzbw9hgma4\\\\\\\/tGmZaf8Q6Rb0P495keHGWunhztvPtjlrNMZoCDeoz3IrWZqdthA7JfkgG6lsLQC\\\\\\\/ScrIGvUKdGMmAJMqhpDxVBOTlTn+FQPCz5j8AAF2cTC8PgpeOAXOzwgt+SsNQoZ1\\\\\\\/1Ru\\\\\\\/x8SoRxhMoW7xZNydnraF3Y0CCXWTlCTLPHJNQwj6Ok6545pRdGNlaEl5W1Fe1mYT1Uqe9IuS79C5a1zjTKNIdcePldtJlnRfbxXJ4t03A05c6nFgwSgqtgTUao0NZ4vmQ0U74eYsdvF4ScMjw0WvL3pd\\\\\\\/w82NlkjrgrwDkjWaJw\\\\\\\/k5o0M+350FEYA1gsb8WnVIAcx+L6Bm0d25eWVKfs30QJkSVyEXCpt9D3biQt\\\\\\\/iHgzJeoFuCOdBMSKBPscRI0MbkBMEgS7HpXLfuVTQNuDWaMFfl9SOctbotmZWGOZZDhUMBDVEghR9YF9WlADxsbwy28dG7CGhQiDmAhl+eF\\\\\\\/U4HExQipqWpOCSaK7AmhDWH1x+8rL+9WGvJToajoyCwW3iVUxmpN+xiMVPBL7ZzOg8zl0Jq0pEbbHjOJoUG2afcsCNfBX\\\\\\\/de6rREzXY0v2plQK2yFPOV1F0WCqoduqgu769uI0uwdu4A80YkSdkqBfkK6\\\\\\\/iiFeWiYmi+n\\\\\\\/zYA8C1tOqaM583JKCZiI7ENg\\\\\\\/Mg\\\\\\\/yDNh9YJtcNbiRpjz04wSllFnLLPVpzxHh1nFt+r8tP+kUtHb4huE5iGG0v7zehW9UHCtpVLZ1iYUJQcSBKkq20gHksLs8nOKdrJyDDy8uS0UHqvarpWdERSJLIkDuAnLXoA\\\\\\\/jva86HMQRX8jx71UNVxT3w4dyOoVqq3EYb\\\\\\\/+AAy6KBhYLrGdTlPj9QaGmpvuadmjMt11NLbC9WKdpLhCOmIqXcMS341x4EkXIwEPwEnHbH9e++yDCjYmSCDFBH9MyYgRzOKVNcEXtFeDZJOVvGGI\\\\\\\/9j9oDnz4G3jIcBcEoMVhup2CffCKmg0P4U82QWZXz5nNsy5Sjzbawox9fN7vryZrtLtdlkGsBqDkaM52DqJFFI8iINPkcwunv4tAb9gUcZ6WJc+yy8wR9TdyLFqT30+dQe+6Fk6gKY2EPNul15ALP1RK7VM\\\\\\\/AAiNOq28WXzMQhK6CIHep\\\\\\\/d7W1QIjvezRyIJUgEl3oFZam85cjsBfDf+HnuirZuSSl+DcCAyjyZ3mpDD409IiOADFn2pbqJ33EDTZ+XFSKgT3NByohMueAvwNPRvPBU1i5A\\\\\\\/IOGiFrxd4MpgQ4YTmmgE3f0r9rAuc87Ac1jfp9SPar+O1Xc90AGB3KTKg8qbYDB0KXyAg4HnQFLHgKwRTR2EhGSelikm9mhrb5RmT96doyHeKGBi6og3RVtsqODhHAqXPfeSSo6Z7ZPayaHCkjuWxtcdYCKpbjoUyrrtwTSf8rwxoeK4p5Sn7QNBSzqNCpAIx9pb3NVLyNhVGdv4kNKSlou5tZoF\\\\\\\/K\\\\\\\/2e0QaBFroH8kjCKx+xvnIuoteAbgG8c\\\\\\\/kET0pKx5\\\\\\\/i\\\\\\\/gyArQYY4VAI282PjIKTR+eZ\\\\\\\/WvGWgvEYdtm3yp+jO2kOYNvrxd8Gz6dDOeTSrdURlSlBR\\\\\\\/2zbOSrfYY2XNjzCSUvGggm765zqq+Ty7\\\\\\\/LQXxA\\\\\\\/dDCO7OLoariKikz7BGuvoUDeOg\\\\\\\/KWB4+gIsuFMvj8ZfkiG82ekJCMJ61IXyoCbY4zTpTezyuTeFgLB5\\\\\\\/41f7pq2bgP4dwhoomklJj89Iv\\\\\\\/+uBcMgpXZpquW8ojdkVV2awA+6E7BNBfFcc17E+GzYjGvsDSjNIMmnVKYvGud7mDkwN4QZdwWWRAxBZSNwtyLoWd9Hq6XkCvCxYUCvEBnVdcR\\\\\\\/AfBWzu4U2bleQPypSunb2uty41smvO2v2LGK4KqZAUWWUNuF5t86Ri6C0Kk\\\\\\\/5Mfiqot9ZUnAAHBCjlCbkiGOq\\\\\\\/wStApw6bA1djoajaHWdd0klmJITnESSclDWdSH6HwHypAvmZZoGIt\\\\\\\/KYo23WbyxlfJoMzl\\\\\\\/ogOBruk5gi+bpvejX\\\\\\\/dzpY9cYE3VLD7gHx2IJ7dXwNzLWaUrmGC5+DG7OqgdJpIfrvRMwGaqwAVChEUvIJKbyWV5zCL54b1MYIU5fg3NfLmj9xutLRX5lZ9ABbRa7rQ7vDOsZZFmPPGeb5jCSdPWRB\\\\\\\/W2n2FbOc0F+gfmMXmYZxZpY4vmoE3xpFgMRADApqksbhIov3aWSupkGwT3MIZ6VXPL8SoZ53TsBU28c60NzX\\\\\\\/FSJfKcHqPs0G3aYB0od55+N\\\\\\\/Ub6wKZPro6Px0csUoTP4lwjwlXdMVZGryr2BZpHPoKtM+CzOWvqWwcfChC2YkrMTNEEL1P7OlZGbsz7TDC9DEkkDQJh1nDhEoZBJjDp2HZsE1Glh80cJBI+aXhguEo5gyPRNQXnXPmd1NeJymi+dI3jnjX4dZDqW3WWhR6HbbDTE1e57IRBVU8sscu1yCP7VvlpWFb64q95UxaALvsbI3iMOIKGeyPaMJzE1zB816lei0odk1\\\\\\\/Q75Ig1xVL7hubFUGlk\\\\\\\/59FSzQJcS0EF+ehZNBZ1jPaKs4N7VnhuGVTpZDff+TXI8O4sQr8hn033QQz4e8YTBDrj+KQHTn4rZdZAs3PjjYxG1cwvb9RvS6WpD4CAWnqLAsQQzyz\\\\\\\/bcvgvQbnbpKsltfnDyxDM6zhWko+Qq+FCfIifVkJAPXSA8fdTSBPSh1D2oYsKh6wBtakz4xynBur3ue898xm5z63Ti6ffexkL+f0yhXwGg0t\\\\\\\/Hah5HTZ8wGR70BYbrUXPa4Ef2t\\\\\\\/KjuzkudOVC9TVJ\\\\\\\/O10L0Oh73HbgAQ\\\\\\\/f6a\\\\\\\/YRSkiMZEr2r5Gmi4Y0hh2JmlvzFVxjhFZw3mUcuixBUx8uRln2dY0HmC8pmPVswH97BCxYohXFYA80roBo\\\\\\\/F\\\\\\\/gZaOjx0jWAJ58205sEERZf8ZidtdmwkUVR0o1WfxRUPGicHUNEIDKhbhxZtc9xKmjAXWVBtVry5xS95zErofXiqYYT4sV6laR7f4ThvwtzRi6odhd55yEvyCjp6WUV9aweof\\\\\\\/eolW6TqcAInYIzCdKSZ0DWieY3EPCJ77E3vKV6+EYo8zEHKvSHtXbUgJqCF573A99FJQG9Rv99r44n70uyrFtfKju2bRoqBpo774Jp1v2qKj33fdf9lchYRw6GBYaFMSxffTrkg5nmhbKNXtu5H\\\\\\\/ixJIPQ8FHLOMhdJaVuydWkjqqFDK8pqJaUimW3WlvP00KjHNxDLsDA+Hk3WVKP8ZTuvBhk+VOJ\\\\\\\/YXU09flrYqQYE\\\\\\\/7302u2KGJ4gWAkGBo\\\\\\\/QzKomuRG8Pa566o6l9FJH8UD1cUMn+\\\\\\\/9Yort0BYwjm1m\\\\\\\/wPu2xqRdAt8Z71eJYJWU1zLSzU60n7csGgpAX9TZIxdl0BbC3Bpxr+iE7wq6Q16RJYj2+P3czYcJCGtEWrOupT\\\\\\\/qN9dpSj+QNMBg6YArwDNRRZAegBvwM+8RbeYiJPk34\\\\\\\/kKzDSm9CMYsyFoOipjROQX4Kwo9ivSrDss7yWHgEGrusfAQw2xAJS4Nyw1tiyBDR0LbW9Akb9c8xLLbE22Iq5oLv0LcwyYJcZaAiMFmKGJU+XHidlAEuQJXpcV50ALuhMePBE97c\\\\\\\/Qer46Hi7XNk5KX3Wd1kr\\\\\\\/zFowTNuU6SEbVnlqYew8dCiQrYLlCCLruR1XWFupiCha02giL6xfGw59Y0I6aSBjDlhSMtq4\\\\\\\/VgvPhJItgZJvDMBV0urWMJ0b5SuN8Ujul5aFKy90RsgFiDJxNYVCYFILYSR0SpGNTCW1MQXi0AxrCdhDSAN\\\\\\\/Cl\\\\\\\/LIyNo8HRnQbA\\\\\\\/gk77uv7q1+jhlr07aAwgBxZssKgakhol2CQJfnu5XHsIrQcjlM16tn\\\\\\\/d7qyeyeigOVlShmr\\\\\\\/ALUnOgRGiQ2Y4PYxTv+XbOgsBLKZsEZhbD1kYiXKifqlzPRyEW43iVJ+8fV4zOT3dr89SL\\\\\\\/Jb85EAryl8iNhsCDIYE2dB+j4T9rFk9t+3EX0bjf\\\\\\\/p6esANZBgSxA7CY7kc6EzrqimiJ9bxJUTDLE2TI1v41c497hsDlw0LJNwFog0Xiq44lqCDuzVWt5bO8b9CAT0DqbShc8+TagkYFzm1\\\\\\\/vLwz3kTrTEHIR4cHXPchyIY5Nd2os\\\\\\\/VYMGpYnQU1fcRrxUpmPAUUymDGE\\\\\\\/+PAH2y1fWeJ27O2dLubvqYTlF4cIzj76bHLH40uAP\\\\\\\/lpIY34\\\\\\\/NC8kwFE02TLGpq2SCsPN8R0jtC0F1arHoSOyuHAE2ZOJT7ysvvKRWOYExWt8pB+6+K2WAskFTMVYk3RJnVvMDYD5PE0WQz2fIa8wqzD9Ct467k+fo5c0CW11KUP6Xli8\\\\\\\/oaHkyuRdE\\\\\\\/jtPfj\\\\\\\/XUf4PhdaWW2Y31mpBswaKZ+ZihcG7CErjHHpbfyLIJ59ZAZzUNbLiFXhS2UNGnP+hb0SJ\\\\\\\/fMiHNjlUtVWCQbcobPqtuOj236GEPwQhNSBvtbTE6UNI5W2n3PmixqEsrf0KdVH4tX+ObkYc4ecdSsXttJReXpRWAqwOzPtyJL6lxaloSH2x9mNWke0NF\\\\\\\/h+piW7skkbUITuTClmi+AnYO0bGJAdmPWlCiF42\\\\\\\/2qKIubw0TwIbQ6dypFq1d1w0UfIAexy1w1CBOWLi7765CNNd6OybJApjguT0u7APCl7i\\\\\\\/J\\\\\\\/UtUf\\\\\\\/qCCV\\\\\\\/gyoM31E5zeAY4NnmYh01XFThJcmU\\\\\\\/zXZCK3\\\\\\\/ge2Fb44uhHh\\\\\\\/ZleRD\\\\\\\/HGq8MV+Wwkj3SnVukGVS6\\\\\\\/IAAtp2aMP6BjNd4GXw8cSGsjZm2Azba5GwyqdwiszYnrU9pamhveDVGCrcLYF36guNW3\\\\\\\/BjcoB2EPqskRvibm\\\\\\\/KFa2mksT1+VU7UWAWYzdIstU74qPpwR6FDeJAiQZH4316pDL2wbRkk4t3bm5Y\\\\\\\/NBoLPnLgo6XiAjpccfndbd1NAKegb0IOEbfw9oJ0yrEzE67RoJi3j2CHww0rcnCFMtwSI\\\\\\\/g5fFC9e93EQfrNEfS9QZcW5hxJkl0jPtK+hT34dMLczEIfPpzyfOQz7Iq7fS+\\\\\\\/jJL2eORNzksv3O9pKKrd3V5RPhnHdN9ZtUA1Uw1DCVQN5Qh7sKa7Afcy+din0EA93Q+f+2\\\\\\\/PNOCde\\\\\\\/1kMClYAOUmQU03E63JTgfhMYNb1tXalxgHy+EuW\\\\\\\/r2Sjuid8\\\\\\\/HFq0ICweBMdLscJtcomb4iuQlQhKtFL7fiQzfBvm6t95+2FPWrInz\\\\\\\/KLIApz4ghN0QLWYev1WXCwaW\\\\\\\/YZNt3W90gb36R\\\\\\\/Ovl03TW4Rn0tmQ5be8B+pN0HQZN4OM\\\\\\\/FNop4YwIkCETRPmsztRqysvTzuCdcK6OfyFTNkJlpJWxD5Od8ovhBONeHJM\\\\\\\/dHF9io1qLOi7ziJtwWGS5IowMYrOdXNIJbgUTdph2xzk865uHMjx92g2KXWpOm5GWzSJQXd7amsWYOysJHZ303ewKsk3f87GoT14WsEtGGddAUrQdZ5Slyh7adzKd1OV+y7r9NIW0ZiSQe1QNvLkhyaWXpWlRRUGy+dYR5xtZzxOAbQeRyE3dgzIea01m7R3vuKXdTrCh+ybj5KXuhTgw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022pFxYSAXlHiucEar7WluEtQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022929dfff8b0867fda7c761f247111b39cfebfdf26ab44a758d10a9102a476dfea\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022TJqcG23UziXftEDAEFlvXtBtP+iTVtShD58aEOYizGYAe1TkdimYTS6V3E9yH2XILXqjGoq0ajQ2O+ylHs6WzPtvkcTiex7oMmSeod71dYdsvOVHO0tWNhi96wsWGuPK1SJxPhO4UCI9L4ONy8x8hYsgDTWziOF3Lloac6AlBQ183Qi7VeZQXUg0FgMdJeKJ1UTEB4Uzh1K9xjvboZrv33FlMpPHC9RDyhkN+GpQW2qjfxSfQ4CyUcZOTwqT+zDQEqdd55r+x184JbWYPXv+sH+\\\\\\\/N6EmE03zEsWd6DR0iEsovCbH3lTfnxpRIpLjgQkUzmx6zaByfUrV1NzaLO7R72okkCjR4MpvGFMkSPFdD4BDxrup2r6+2Ri6XCbIlbaM+TtgByrbLRb38pzmw6QjzIE6JH2DQpk2FRT9Hd1UKXcgQpEz3yMIgxNXyP8MgLNTG5pCiHZXYWNOZYpczBDloD6vOZbbcwkDJxw7WJQmclHc+ZNYMPcsm\\\\\\\/DvUCtMa9y2r7lfdKPxnDP0APMhq2g8547dwSIRNz5zds+l9CPOYpCbBdqKLA2rQl19t2ofdWLgbB8mcIe+Q4Nx8QZh5NKJsXbgqIj7Q0eut0ybitBiN+UTMqkVxj1bnXodGreUvhC1+SWWWwCtr0URcclG5MAqt+6t8eeLh8lvOjMwEuw2IVtxyRWxYPNC0mvO64OhXP4weuZh3IEmKqJjOTt4QmPLU2r+lJe3lW6ErxVb6qzB0uObi7vJgf0iNlA3p3iIO\\\\\\\/QbO4dtdNrq7x6dHvFdieC4inO18eSiT3EMwyZOvzNSBxw6pkxU11d8SMJvbLOGzjvOf7GNhcy3kyETEZ+qt6DujkFrQunRyYIiNFvZ9ZOPqX53s281s4MW252HZMvGJ9jl2CggnPhFUec\\\\\\\/Wgz\\\\\\\/hH8UjwL34K5UG6Ar7KKq4+ET1GuEs355aX6WR\\\\\\\/Rmrm5P9nVqKi7\\\\\\\/t2\\\\\\\/GeEfxxcrLS2D7dEwbqknioMbD12idK96XT5S8AJOqgunk9c+uMqUTGB569Pjlox1NFYj9PRyJ8yMITqBu2ZNaOBaRZbwmakFjG\\\\\\\/fFrJQWwZrplDLJ90kF4EKm6E2KkhuOZP\\\\\\\/KY6+YO9Zt1KAn\\\\\\\/zNS9EGJO0MHSKTwh66GPdgJ\\\\\\\/urGCg6P7c1xttE1\\\\\\\/i9LNpEU35k0vQNYnIOBmybAYDafo23DF1gguFr9hkKYvA2Dv8DdNT8aozxYwrSgd8i\\\\\\\/qQ\\\\\\\/SIJWvK5K4mWr9xbdKZt1iPmx7DNOGU\\\\\\\/WEkSE9VWFAMHSJwo7nVoifK7ZR+qTtRBhvUVEnnN+JqQlNtveXJdlOtBV5glr3WcQqyur0+OtMIBxWgG0eMfhn5xeqRqX09\\\\\\\/s9OxlxV83DRkzWnOJ7WWGJgCA4HoX\\\\\\\/+kx5DaIzVyo57xJg7BMDPL03TmH+ICW9KyBrS+rzHB\\\\\\\/Y0XjTb24N1cJGMmazv\\\\\\\/of5nXC\\\\\\\/ePQ5zmmFqWWyCJk3nHIfW0aSQfs0q4enk+KbbgRYptu1t+Kf1LlOVRhqnZSNo5z6IQbYfAm8lqDbWPuTg3e\\\\\\\/jr4eD8szJ3GieqwM17Sy8dPQBJqQ1SQ99GfAkAhVt0V0KLnjIoQUKvbH5w1xtJXR\\\\\\\/cGRBBDFSpmJSD0nY2+dZPYi64+BAY7vlJjR+qv01sLysleHWiRsoZM95k6j7Nu6RtxJj0bSEz8HboDddcUhBdcaKDgD3oqFxFLmWW4RvcWT1SfJhm0FRs2Zby59HnGsZDk3cWI8gEwr3754bTL59L37jSi07bIzZKZoh22GH\\\\\\\/KdWR3kEvdnx+jLaM8LDydnHYz3ll2GL2N+h9btA9KLOOfnOrQEaYyy2foVjqHkEjZOH3N2YX+1l7NzPMH6hd2ACYocN88i\\\\\\\/TJFl0OmCjsipl8GCvHwU7cpXse3aP0BkuArCA+5PLpLyvfQR+BUtAsELs9T+0DAqScMlT9WKVjjmHmjqxPb\\\\\\\/AUWp04f4QGkVrv8clkCz1iBuYtiWAtLj+hb2sVF+XIQGseuJEKxh6WwFUYgQ==\\\u0022,\\\u0022iv\\\u0022:\\\u00222CKbwo9Jb7cex7wJ+Qv2BA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00225ea8d8be64ecf930fef80506af7b56198575b4ec39b5cf4f5b91740479f41b1d\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-494623 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-genome-report">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/11/3521" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-X-Yelh78wvPvrJrFXYi-mIWZnAnWFHhkm5gX2Xw-kE4" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/11/3521" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-qu2wETtB8CeKL-XHiCGJpvzd3APhjSwVbPV1BmPUOP4" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/11/3513" title="Updated Genome Assembly and Annotation for Metrosideros polymorpha, an Emerging Model Tree Species of Ecological Divergence" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/11/3531" title="De Novo Assembly and Annotation from Parental and F1 Puma Genomes of the Florida Panther Genetic Restoration Program" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494623" id="node494623" data-pisa="ggg;9/11/3521" data-pisa-master="ggg;g3.119.400657" data-apath="/ggg/9/11/3521.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genome Improvement and Genetic Map Construction for <em>Aethionema arabicum</em>, the First Divergent Branch in the Brassicaceae Family</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Thu-Phuong Nguyen</span>, <span class="highwire-citation-author" data-delta="1">Cornelia Mühlich</span>, <span class="highwire-citation-author" data-delta="2">Setareh Mohammadin</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0001-9865-574X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Erik van den Bergh</span>, <span class="highwire-citation-author" data-delta="4">Adrian E. Platts</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0002-7711-5282" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Fabian B. Haas</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0002-0225-873X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Stefan A. Rensing</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="7"><a href="http://orcid.org/0000-0001-6777-6565" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>M. Eric Schranz</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3521-3530;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400657 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494623" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Thu-Phuong Nguyen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Biosystematics Group, Wageningen University, Droevendaalsesteeg 1, 6708 PB Wageningen, The Netherlands,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Thu-Phuong%2BNguyen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nguyen%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AThu-Phuong%2BNguyen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Cornelia Mühlich </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Faculty of Biology, Plant Cell Biology, University of Marburg, Karl-von-Frisch-Str. 8, 35043 Marburg, Germany,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Cornelia%2BM%C3%BChlich%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=M%C3%BChlich%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACornelia%2BM%25C3%25BChlich%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Setareh Mohammadin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Biosystematics Group, Wageningen University, Droevendaalsesteeg 1, 6708 PB Wageningen, The Netherlands,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Setareh%2BMohammadin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mohammadin%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASetareh%2BMohammadin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Erik van den Bergh </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Biosystematics Group, Wageningen University, Droevendaalsesteeg 1, 6708 PB Wageningen, The Netherlands,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Erik%2Bvan%2Bden%2BBergh%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=van%20den%20Bergh%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AErik%2Bvan%2Bden%2BBergh%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-9865-574X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Erik van den Bergh</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Adrian E. Platts </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Center for Genomics and Systems Biology, Department of Biology, New York University, New York, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Adrian%2BE.%2BPlatts%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Platts%20AE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAdrian%2BE.%2BPlatts%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Fabian B. Haas </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Faculty of Biology, Plant Cell Biology, University of Marburg, Karl-von-Frisch-Str. 8, 35043 Marburg, Germany,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Fabian%2BB.%2BHaas%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Haas%20FB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AFabian%2BB.%2BHaas%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-7711-5282" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Fabian B. Haas</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Stefan A. Rensing </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Faculty of Biology, Plant Cell Biology, University of Marburg, Karl-von-Frisch-Str. 8, 35043 Marburg, Germany,</div><div class='author-affiliation'><span class='nlm-sup'>§</span>LOEWE Center for Synthetic Microbiology (SYNMIKRO), Philipps University of Marburg, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Stefan%2BA.%2BRensing%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rensing%20SA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AStefan%2BA.%2BRensing%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-0225-873X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Stefan A. Rensing</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:stefan.rensing@biologie.uni-marburg.de" class="" data-icon-position="" data-hide-link-title="0">stefan.rensing@biologie.uni-marburg.de</a>
<a href="mailto:eric.schranz@wur.nl" class="" data-icon-position="" data-hide-link-title="0">eric.schranz@wur.nl</a></span></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">M. Eric Schranz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Biosystematics Group, Wageningen University, Droevendaalsesteeg 1, 6708 PB Wageningen, The Netherlands,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=M.%2BEric%2BSchranz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Schranz%20ME&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AM.%2BEric%2BSchranz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-6777-6565" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for M. Eric Schranz</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:stefan.rensing@biologie.uni-marburg.de" class="" data-icon-position="" data-hide-link-title="0">stefan.rensing@biologie.uni-marburg.de</a>
<a href="mailto:eric.schranz@wur.nl" class="" data-icon-position="" data-hide-link-title="0">eric.schranz@wur.nl</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/9/11/3521" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:494623" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:494623/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/11/3521.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:494623" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:494623/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/11/3521.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:494623" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:494623/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/9/11/3521.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:494623" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:494623/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">The genus <em>Aethionema</em> is a sister-group to the core-group of the Brassicaceae family that includes <em>Arabidopsis thaliana</em> and the Brassica crops. Thus, <em>Aethionema</em> is phylogenetically well-placed for the investigation and understanding of genome and trait evolution across the family. We aimed to improve the quality of the reference genome draft version of the annual species <em>Aethionema arabicum</em>. Second, we constructed the first <em>Ae. arabicum</em> genetic map. The improved reference genome and genetic map enabled the development of each other. We started with the initially published genome (version 2.5). PacBio and MinION sequencing together with genetic map v2.5 were incorporated to produce the new reference genome v3.0. The improved genome contains 203 MB of sequence, with approximately 94% of the assembly made up of called (non-gap) bases, assembled into 2,883 scaffolds (with only 6% of the genome made up of non-called bases (Ns)). The N<sub>50</sub> (10.3 MB) represents an 80-fold increase over the initial genome release. We generated a Recombinant Inbred Line (RIL) population that was derived from two ecotypes: Cyprus and Turkey (the reference genotype. Using a Genotyping by Sequencing (GBS) approach, we generated a high-density genetic map with 749 (v2.5) and then 632 SNPs (v3.0) was generated. The genetic map and reference genome were integrated, thus greatly improving the scaffolding of the reference genome into 11 linkage groups. We show that long-read sequencing data and genetics are complementary, resulting in an improved genome assembly in <em>Ae. arabicum</em>. They will facilitate comparative genetic mapping work for the Brassicaceae family and are also valuable resources to investigate wide range of life history traits in <em>Aethionema</em>.</p></div><ul class="kwd-group"><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/aethionema-arabicum" class="hw-term hw-article-keyword hw-article-keyword-aethionema-arabicum" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Aethionema arabicum</em></a></li><li class="kwd"><a href="/keyword/brassicaceae" class="hw-term hw-article-keyword hw-article-keyword-brassicaceae" rel="nofollow">Brassicaceae</a></li><li class="kwd"><a href="/keyword/genome-improvement" class="hw-term hw-article-keyword hw-article-keyword-genome-improvement" rel="nofollow">genome improvement</a></li><li class="kwd"><a href="/keyword/genetic-map" class="hw-term hw-article-keyword hw-article-keyword-genetic-map" rel="nofollow">genetic map</a></li><li class="kwd"><a href="/keyword/pacbio" class="hw-term hw-article-keyword hw-article-keyword-pacbio" rel="nofollow">PacBio</a></li><li class="kwd"><a href="/keyword/minion" class="hw-term hw-article-keyword hw-article-keyword-minion" rel="nofollow">MinION</a></li><li class="kwd"><a href="/keyword/genotyping-sequencing-0" class="hw-term hw-article-keyword hw-article-keyword-genotyping-by-sequencing" rel="nofollow">Genotyping by Sequencing</a></li></ul><div class="section other1" id="sec-1"><h2 class="">Background</h2><p id="p-3">The genus <em>Aethionema</em> belongs to the important plant family Brassicaceae. The crucifers contain many species of interest, such as the Brassica crop plants (<em>e.g.</em>, <em>B. rapa</em>, <em>B. oleracea</em> and <em>B. napus</em>), ornamental plants (such as the genera <em>Aubrieta</em>, <em>Iberis</em>, <em>Lunaria</em> and <em>Draba</em>) and research model plant species (including <em>Arabidopsis thaliana</em>, <em>A. lyrata</em>, <em>Capsella rubella</em> and <em>Arabis alpina</em>). Phylogenetic studies have established <em>Aethionema</em> as the sister-group of the core-group in the family (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Beilstein <em>et al.</em> 2008</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Huang <em>et al.</em> 2016</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Guo <em>et al.</em> 2017</a>). Thus, <em>Aethionema</em> holds an essential phylogenetic position for studies on genome and trait evolution across the Brassicaceae family.</p><p id="p-4">The monogeneric tribe <em>Aethionemeae</em> consists of 57 species and is distributed mainly in the Irano-Turanian region, a hot spot for species radiation and speciation (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Al-Shehbaz <em>et al.</em> 2006</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Franzke <em>et al.</em> 2011</a>). This tribe displays various interesting morphological and ecological characteristics, especially fruit and seed heteromorphism. Heteromorphism is defined as the production of two or more distinct fruit or seed morphs on the same individual (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Imbert 2002</a>), which includes morphological size, shape and color; physiological dormancy and germination of fruits and seeds. <em>Aethionema arabicum</em> is one of the seven reported heteromorphic species of <em>Aethionema</em> (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Lenser <em>et al.</em> 2016</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Mohammadin <em>et al.</em> 2017</a>). <em>Aethionema arabicum</em> is a small diploid annual, with a short life cycle starting from seed germination to the end of the vegetative development in spring, followed by reproduction and the end of life cycle in summer (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bibalani 2012</a>). Both annual life history and heteromorphism probably evolved as adaptive responses to unpredictable environments, especially dry arid habitats, indicating a wide range of natural variation for ecologically adaptive traits in <em>Ae. arabicum</em>.</p><p id="p-5">Owing to its unique phylogenetic position and interesting characteristics, <em>Ae. arabicum</em> is an ideal sister-group model for research. Therefore, <em>Aethionema</em> genome and genetic resources are desirable. The initially published <em>Ae. arabicum</em> draft genome (v1.0) contains 59,101 scaffolds with an N50 of 115,195 bp while the genome was predicted to be 320 Mbp in size with n = 11 (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Haudry <em>et al.</em> 2013</a>). Here we first aimed to (i) improve the quality of the reference genome and (ii) to construct the first <em>Ae. arabicum</em> genetic map. A higher quality version of the genome assembled by the VEGI consortium was later released as version 2.5, which is used as the starting point of our analyses.</p><p id="p-6">High throughput sequencing using Pacific Biosciences (PacBio) and Oxford Nanopore MinION (MinION) technology followed and resolved many uncalled gaps in the v2.5 genome and supported further super-scaffolding, which resulted in genome v3.0.</p><p id="p-7">The genetic map was constructed using Genotyping by Sequencing (GBS) on a Recombinant Inbred Line (RIL) population. The 216 RILs were derived from a cross between Turkey (reference ecotype) and Cyprus ecotypes. The first version of genetic map v2.5 was obtained based on genome v2.5 with 746 Single Nucleotide Polymorphism (SNP) markers. The later genetic map v3.0 was built with 626 SNPs generated based on genome v3.0.</p><p id="p-8">Here we show that the long-read genome assembly and the genetic map of <em>Ae. arabicum</em> supported the development of each other. They will serve as a substantial resource for further research on <em>Aethionema</em> as well as the Brassicaceae family.</p></div><div class="section methods" id="sec-2"><h2 class="">Data description and Methods</h2><div id="sec-3" class="subsection"><h3>Overview of the workflow</h3><p id="p-9">An overview of the improvements of the genome of <em>Aethionema arabicum</em> and the generation and improvement of its genetic map are depicted in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>. The genome draft version 1.0 was first improved by Ray (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Boisvert <em>et al.</em> 2010</a>) and AllPathsLGs (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Gnerre <em>et al.</em> 2011</a>) and led to the release of genome v2.5 (available on <a href="http://genomevolution.org">genomevolution.org</a>). Genome v2.5 was used as a basis for SNP calling after GBS of the RILs. This generated SNP markers used to construct the genetic map v2.5. Scaffolds were ordered with AllMaps (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Tang <em>et al.</em> 2015</a>) based on the maximum co-linearity to genome v2.5 and genetic map v2.5. This resulted in genome vAM. Gap filling and super-scaffolding improvement for genome vAM was obtained by PacBio sequencing leading to genome v2.6. PBjelly2 (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">English <em>et al.</em> 2012</a>) run using the MinION reads further improved genome v2.6 to v3.0. We revisited the genetic map v2.5 by recalling SNPs according to genome v3.0 and constructed a genetic map v3.0 with the newly called SNP markers. Below we describe the workflow in detail in the three following sections: (i) the initial genome assembly, (ii) genetic map construction and (iii) genome improvement.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/11/3521/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Overview of the analyses performed in this study. In filled boxes are data sets, approaches and companying tools are in open boxes." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1711713484" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Overview of the analyses performed in this study. In filled boxes are data sets, approaches and companying tools are in open boxes.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/11/3521/F1.medium.gif" width="440" height="340"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/9/11/3521/F1.medium.gif" width="440" height="340"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/11/3521/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/11/3521/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494795" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-10" class="first-child">Overview of the analyses performed in this study. In filled boxes are data sets, approaches and companying tools are in open boxes.</p><div class="sb-div caption-clear"></div></div></div><div id="sec-4" class="subsection"><h4>The initial genome (v2.5): The starting point:</h4><div id="sec-5" class="subsection"><h5>Genome re-assembly using AllPathsLG</h5><p id="p-11">The version 1.0 assembly generated by the Ray assembler (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Boisvert <em>et al.</em> 2010</a>) was fragmented <em>in silico</em> into a set of artificial overlapping reads, combined with additional paired end and mate pair data (described in (<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Haudry <em>et al.</em> 2013</a>)), and re-assembled using the AllPathsLG assembler (<a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Gnerre <em>et al.</em> 2011</a>). This iterative assembly process leveraged additional short read data and AllPathsLG’s extensive error correction to minimize the potential for errors in the first assembly to contribute to the second round of assembly. Gap closing was then performed using GapCloser, part of the SOAPdenovo2 package (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Luo <em>et al.</em> 2012</a>). Gene annotations were lifted over from assembly version 1.0 to version 2.5 using the LiftOver tool from the UCSC Genome Browser tools package (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Kent <em>et al.</em> 2002</a>).</p><p id="p-12">Genome version 2.5 contains 3,166 scaffolds, has an N50 of 564,741 bp and was published as version 2.5 on <a href="https://genomevolution.org/coge/">https://genomevolution.org/coge/</a>.</p></div></div><div id="sec-6" class="subsection"><h4>Genetic map construction:</h4><div id="sec-7" class="subsection"><h5>Plant material</h5><p id="p-13">Two <em>Aethionema arabicum</em> ecotypes were used, Turkey (TUR) and Cyprus (CYP). The TUR accession comes from the living plant collections at the Botanical Garden in Jena, Germany (Botanischer Garten Jena). The seeds for this genotype were derived from a plant in the Botanical Garden in Nancy, France. The CYP ecotype was collected in 2010 near Kato Moni (coordinates UTM WGS 84: 508374 - 3879403) at an altitude 410 m on pillow lava by Charalambos S Christodoulou (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Mohammadin <em>et al.</em> 2018</a>).</p><p id="p-14">These two ecotypes were used as parents for the development of the recombinant inbred line population, where TUR was the father and CYP the mother. Seeds from initial F<sub>1</sub> plants were used to generate an initial F<sub>2</sub> population. For each of the 216 segregating F<sub>2</sub> plants, a single seed was randomly chosen to further grow and reproduce the next generation. The procedure was repeated until F<sub>8</sub>, when the experiment was performed with 216 RILs.</p><p id="p-15">To grow the plants for the GBS, F<sub>8</sub> seeds of 216 RILs were placed on filter paper, wetted with distilled water, in petri dishes. Imbibed seeds were incubated at 4° in dark for 3 days, followed by germination in the light at 20° for 2 days. Seedlings were transferred to soil pots (10.5 cm diameter 10 cm height) in November 2014. Plants were grown in greenhouse (Wageningen University and Research, the Netherlands) in partially controlled conditions, long day (16 h light and 8 h dark) and at 20°.</p></div></div></div></div><div class="section other2" id="sec-8"><h2 class="">Genotyping by Sequencing (GBS)</h2><div id="sec-9" class="subsection"><h3>DNA isolation</h3><p id="p-16">Young tissues from leaves and flower buds were collected from each F9 plant for DNA isolation. The DNA isolation was done according to a modified CTAB protocol (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Doyle 1991</a>). In brief, plant material was frozen with liquid nitrogen and ground into powder. Each sample was incubated with 500 µl of CTAB buffer at 65° in the water bath for 30 min. After 30 min cooling at room temperature, equal volume (500 µl) of chloroform:isoamylalcohol (24:1 v/v) was added, and vigorously hand-mixed for a min. 400 µl of supernatant was recovered after centrifuging at maximum speed for 5 min. The supernatant was cleaned again with a chloroform:isoamylalcohol step. DNA precipitation was performed by adding an equal volume of cold isopropanol with 30 min incubation on ice and centrifugation at maximum speed for 15 min. The DNA pellet was cleaned twice with 1 ml of 70% ethanol and centrifugation at maximum speed for 5 min. Dry DNA pellet was dissolved in Milli-Q water.</p></div><div id="sec-10" class="subsection"><h3>Constructing GBS libraries</h3><p id="p-17">DNA was treated with RNAse overnight at 37° with RNAse one by Promega. Quality was checked on a 1% agarose gel and DNA quantity was checked with Pico Green. Based on this, DNA was diluted down to 20 ng/µl with MQ water and used in further analysis. GBS was performed in general by following the procedure described in (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Elshire <em>et al.</em> 2011</a>). Oligonucleotides for creation of common as well as 96 barcoded ApeKI adapters were obtained from Integrated DNA Technologies and diluted to 200 µM. For each barcoded and common adapter, top and bottom strand oligos were combined to a 50 µM annealing molarity in TE to 100 µl total volume. Adapter annealing was carried out in a thermocycler (Applied Biosystems) at 95° for 2 min, ramp to 25° by 0.1 degree per second, hold at 25° for 30 min and 4° forever. Annealed adapters were further diluted to a 0.6 ng/µl concentrated working stock of combined barcoded and common adapter in 96 well microtiter plate and dried using a vacuum oven. For each genomic DNA sample 100 ng (10 ng/µl) was used and added to lyophilized adapter mix and dried down again using a vacuum oven.</p><p id="p-18">Adapter DNA mixtures were digested using 2.5 Units ApeKI (New England Biolabs) for 2 hr at 75° in a 20 µl volume. Digested DNA and Adapters were used in subsequent ligation by 1.6 µl (400 Units/µl) T4DNA Ligase in a 50 µl reaction volume at 22° for one hour followed by heat inactivation at 65° for 30 min. Sets of 96 digested DNA samples, each with a different barcode adapter, were combined (10µl each) and purified using a Qiaquick PCR Purification columns (Qiagen). Purified pooled DNA samples were eluted in a final volume of 10µl. DNA Fragments were amplified in 50 µl volume reactions containing 2 µl pooled DNA, 25 µl KAPA HiFi HotStart Master Mix (Kapa Biosystems), and 2 µl of both PCR primers (12.5 µM). PCR cycling consisted of 98° for 30 sec, followed by 18 cycles of 98° for 30 sec, 65° for 30 sec, 72° for 30 sec with a final extension for 5 min and kept at 4°. Amplified libraries were purified as above but eluted in 30 µl. Of the amplified libraries 1 µl was loaded onto a Bioanalyzer High Sensitivity DNA Chip (Agilent technologies) for evaluation of fragment sizes and 1 µl was used for quantification using Qubit (Life Technologies). Amplified library products were used for extra size selection using 2% agarose gel cassette on a blue pippin system (Sage Science) to remove fragments smaller than 300 bp. Eluted size selected libraries were purified by AmpureXP beads (Agencourt). Final libraries were used for clustering on five lanes of an illumina Paired End flowcell using a cBot. Sequencing used an illumina HiSeq2000 instrument using 2*100 nt Paired End reads.</p></div><div id="sec-11" class="subsection"><h3>Sequencing and processing raw GBS data</h3><p id="p-19">Raw sequencing data were processed using the TASSEL software package (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Glaubitz <em>et al.</em> 2014</a>) version 5.2.37 using the GBSv2 pipeline. For quality filtering and barcode trimming, the GBSSeqToTagDBPlugin was run with the following parameters: kmerLength: 64, minKmerL: 20, mnQs: 20, mxKmerNum 100000000. Tags were dumped from the produced database using TagExportToFastqPlugin and mapped to the reference genome using the bwa software package (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Li and Durbin 2010</a>) in single-ended mode (samse). Positional information from aligned SAM files was stored in the TASSEL database using the SAMToGBSdbPlugin. The DiscoverySNPCallerPlugin was run using the following parameters: mnLCov: 0.1, mnMAF: 0.01. Found SNPs were scored for quality using SNPQualityProfilerPlugin and the Average taxon read depth at SNP was used as a quality score for filtering in the next step (minPosQS parameter), these scores were written to the TASSEL database using UpdateSNPPositionQualityPlugin. Finally, the ProductionSNPCallerPluginV2 was run with the following parameters: Avg Seq Error Rate: 0.002, minPosQS: 10, mnQS: 20.</p></div><div id="sec-12" class="subsection"><h3>Genetic map calculation</h3><p id="p-20">We used JoinMap v4.1 for the genetic map construction (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Stam 1993</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">van Ooijen 2006</a>). The genetic map v2.5 was built with 749 SNPs generated by GBS based on genome v2.5 (unprocessed and processed data available as S1 and S2). A set of 632 SNPs called according to genome 3.0 was used for the genetic map v3.0 (unprocessed and processed data available as S3 and S4). Regression and Maximum likelihood mapping were used to calculate these maps (the linkage group information for both 2.5 and 3.0 genetic maps are available as S5).</p><div id="sec-13" class="subsection"><h4>Genome improvement:</h4><div id="sec-14" class="subsection"><h5>Genome version vAM: AllMaps</h5><p id="p-21">We ran AllMaps (<a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Tang <em>et al.</em> 2015</a>) with default setting to combine genetic map v2.5 and physical map genome v2.5. This step resulted in genome vAM, in which scaffolds were ordered and oriented to reconstruct chromosomes.</p></div></div></div></div><div class="section other3" id="sec-15"><h2 class="">Contamination removal</h2><p id="p-22">The <em>Ae. arabicum</em> scaffolds v2.5 were checked for contaminations. The genome scaffolds were split into 197,702 1 kbp fragments and blasted against the NCBI nt database (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">NCBI Resource Coordinators 2016</a>) using Tera-BLAST (TimeLogic Tera-BLAST algorithm, Active Motif Inc., Carlsbad, CA). The output was then analyzed by MEGAN 6 (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Huson <em>et al.</em> 2016</a>). All scaffolds for which more than 50% of their entire length was found in bacteria and with no hit in Viridiplantae were marked as putatively contaminated. A hit was counted with a minimum bit score of 50. Additionally, Bisulfite sequencing (BS-seq) CpG and Chromatin ImmunoPrecipitation DNA-Sequencing (ChIP-seq) H3 data were checked to identify contaminants (Aethionema_contamination.xlsx). We ensured that these scaffolds were not combined with another scaffold by PBjelly2. After screening, three v2.5 scaffolds were removed: Scaffold_2406, Scaffold_2454 and Scaffold_2594. They had a total length of 1,758 bp without any annotated genes. A summary of the contamination screen is available as S6.</p></div><div class="section other4" id="sec-16"><h2 class="">Long read generation for genome improvement</h2><div id="sec-17" class="subsection"><h3>PacBio reads</h3><p id="p-23">Genomic DNA (gDNA) for <em>Ae. arabicum</em> was obtained from leaves of the Cyprus and Turkey ecotypes. DNA was extracted using a modified protocol (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Hiss <em>et al.</em> 2017</a>) based on (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Dellaporta <em>et al.</em> 1983</a>). For the Turkey ecotype 35.70 µg and for the Cyprus ecotype 21.45 µg high molecular weight DNA were sent to the Max Planck-Genome-Centre, Cologne, and sequenced using the PacBio RS II machine (library insert size was 10-15 kbp gDNA). Four flow cells for Cyprus and six for Turkey were sequenced. <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a> summarizes the statistics of the reads. The CG content of the pooled reads was 38%.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/494704/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/494704/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">(Mérai <em>et al.</em> 2019): Overview of the <em>Aethionema arabicum</em> PacBio reads</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-18" class="subsection"><h3>MinION reads</h3><p id="p-25">To obtain MinION reads, gDNA was extracted from the Turkey ecotype (four leaf samples, 100 mg each) as outlined above. After pooling the samples, the gDNA concentration was measured using Hoechst 33258 DNA dye and resulted in 73.85 ng/µl. The library preparation was done using the Oxford Nanopore SQK-NSK007 protocol and R9.4 chemistry to design an 8 kbp 2D library. The sequencing run was carried out using Oxford Nanopores MinION technology. The flow cell sequenced 30,935 reads (122,362,072 nt) at -205 mV and 24 hr of runtime. After base calling with the MinKNOW 1.6 software (Oxford Nanopore Technologies Ltd.) the read length ranged from 5 to 63,441 nt with an average length of 3,955 nt. The average phred quality score was 11 and the GC content 41%, reads were not filtered or trimmed. The initial sequence format FAST5 was converted to FASTQ format by using the R package poRe version 0.21 (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Watson <em>et al.</em> 2015</a>). Because the MinION flow cell had previously been used for <em>Physcomitrella patens</em> DNA in the same run, the 30,935 reads were filtered for putative <em>P. patens</em> contamination. The reads were mapped with the long read mapper GMAP version 2017-08-15 (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Wu and Nacu 2010</a>) against the <em>P. patens</em> genome V3 (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Lang <em>et al.</em> 2018</a>). All settings were kept at default. 1,447 reads were characterized as putative <em>P. patens</em> reads and therefore removed.</p></div><div id="sec-19" class="subsection"><h3>Genome improvement using long reads</h3><p id="p-26">To perform super-scaffolding and gap filling, the program PBjelly 2 version 15.8.24 was used (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">English <em>et al.</em> 2012</a>). It internally uses BLASR v5.1 (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Chaisson and Tesler 2012</a>) for mapping reads to a reference. PBjelly setup was used with a minimal gap size (–minGap) of 10 and called BLASR with -minMatch 12 (based on the observation that lower values yield less mappings), -bestn 1 (mapping reads uniquely) and –noSplitSubreads.</p></div><div id="sec-20" class="subsection"><h3>Genome version 2.6: PacBio sequencing incorporation</h3><p id="p-27">We ran PBjelly2 with 381,022 (152,398 CYP, 228,624 TUR) PacBio reads which where head-cropped with 20 (due to suspicious per base sequence content suggestion presence of adapters) using Trimmomatic version 0.36 (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Bolger <em>et al.</em> 2014</a>).</p><p id="p-28">PBjelly2 was used to improve genome v2.5 and vAM. Comparing the results, we found some scaffold connections which were made by PBjelly2 (v2.5) were no longer possible for vAM (these scaffolds were already connected). Five connections formed for v2.5 scaffolds were already introduced by the genetic map approach (see above). Twelve connections which could be established in v2.5 were not formed in the PBjelly2 output for improving vAM, because the scaffolds were already connected with other scaffolds. Since PBjelly2 only fills gaps with reads and is not able to place whole scaffolds in gaps, it was necessary to split the vAM genome at certain points to be able to obtain the twelve connections which were not present in the PBjelly2 output for vAM (visualized in <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). Split scaffolds were reconnected again after running PBjelly2, using N-stretches of length 100 to keep all improvements introduced in vAM if they were not formed by PBjelly2 (scaffolds in the vAM genome were combined using stretches of 100 Ns to denote a gap of unknown length). Since it is possible that PBjelly2 only fills a gap partially, we had to identify the positions of the gaps introduced by AllMaps in the new genome version and checked if they were filled completely or not. If the gap length was reduced, it was extended to have a length of 100 again. This approach produced genome v2.6.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/11/3521/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Problem arising from applying PBjelly2 on vAM. Scaffold borders are visualized in blue and extensions of scaffolds introduced by PBjelly2 are shown in brown. Assuming the true order of the scaffolds is shown on top of the figure, but scaffold X and scaffold Z were already combined in the vAM assembly (second bar from top) this could lead to a partial filling of the N-stretch and maybe an extension of scaffold Y. However, PBjelly2 would not be able to place scaffold Y between the two other scaffolds (middle bar). If the scaffolds were thus split again (second bar from bottom), it is possible that the connections are made correctly applying PBjelly2 on the split version (bottom bar). This only visualizes a theoretical case, in this work it appeared every time that scaffold X and Y were connected by PBjelly2 and scaffold Z had to be reconnected afterward." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1711713484" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Problem arising from applying PBjelly2 on vAM. Scaffold borders are visualized in blue and extensions of scaffolds introduced by PBjelly2 are shown in brown. Assuming the true order of the scaffolds is shown on top of the figure, but scaffold X and scaffold Z were already combined in the vAM assembly (second bar from top) this could lead to a partial filling of the N-stretch and maybe an extension of scaffold Y. However, PBjelly2 would not be able to place scaffold Y between the two other scaffolds (middle bar). If the scaffolds were thus split again (second bar from bottom), it is possible that the connections are made correctly applying PBjelly2 on the split version (bottom bar). This only visualizes a theoretical case, in this work it appeared every time that scaffold X and Y were connected by PBjelly2 and scaffold Z had to be reconnected afterward.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/11/3521/F2.medium.gif" width="440" height="335"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/9/11/3521/F2.medium.gif" width="440" height="335"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/11/3521/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/11/3521/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494770" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-29" class="first-child">Problem arising from applying PBjelly2 on vAM. Scaffold borders are visualized in blue and extensions of scaffolds introduced by PBjelly2 are shown in brown. Assuming the true order of the scaffolds is shown on top of the figure, but scaffold X and scaffold Z were already combined in the vAM assembly (second bar from top) this could lead to a partial filling of the N-stretch and maybe an extension of scaffold Y. However, PBjelly2 would not be able to place scaffold Y between the two other scaffolds (middle bar). If the scaffolds were thus split again (second bar from bottom), it is possible that the connections are made correctly applying PBjelly2 on the split version (bottom bar). This only visualizes a theoretical case, in this work it appeared every time that scaffold X and Y were connected by PBjelly2 and scaffold Z had to be reconnected afterward.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-21" class="subsection"><h3>Genome version 3.0: MinION sequencing incorporation</h3><p id="p-30">After improving the genome to v2.6 using the PacBio reads, the same approach was applied for 30,935 MinION TUR reads to obtain the <em>Ae. arabicum</em> genome v3.0. The MinION reads were also checked for contamination. The genome version 3.0 is available at <a href="https://genomevolution.org/coge/GenomeView.pl?gid=36061">https://genomevolution.org/coge/GenomeView.pl?gid=36061</a>.</p></div><div id="sec-22" class="subsection"><h3>Name convention of Ae. arabicum v3.0 genome scaffolds</h3><p id="p-31">Scaffolds of genome v3.0 were named and ordered according to their length from long to short. The longest eleven scaffolds were named linkage group (LG) based on the genetic map. Scaffolds which were combined are named csc for concatenated scaffold and the other ones are named sc (scaffold). The v3.0 scaffold names therefore follow the scheme type-number˽v2.5 scaffold[.v2.5 scaffold…]. I.e., the scaffold type (LG, CSC, SC), followed by a minus and the number of the scaffold, separated by a blank, followed a list of scaffolds denoting the v2.5 scaffolds or the v3.0 scaffolds. This naming system resulted in a shift in LG order between v2.5 and v3.0 (Supplementary file linkage_group_map.xlsx).</p></div><div id="sec-23" class="subsection"><h3>Migration of proteins to new genome version</h3><p id="p-32">To perform the lift over of the gene models from v2.5 to v3.0, a combination of Gene Model Mapper (GeMoMa) v1.4 (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Keilwagen <em>et al.</em> 2016</a>) and flo (flo - same species annotations lift over pipeline, <a href="https://github.com/wurmlab/flo">https://github.com/wurmlab/flo</a>) were used. The results of both programs were concatenated. flo results were preferred over GeMoMa results if the results of the two programs differed, because flo works with alignments on nucleotide level while GeMoMa works with blasting proteins on amino acid level. If a protein could not be lifted completely, it is marked as partial in the resulting GFF (v3.0). A total of 34 genes had to be lifted manually, because they were either not lifted at all or only partially. If an intron could not be lifted, it was added by hand. If an exon or CDS could not be lifted, the new location was deriving from neighboring features which could be migrated to the new genome version. The location was then used to extract the nucleotide sequence from the genome using samtools v1.4 (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Li <em>et al.</em> 2009</a>). Only if the sequence was identical to the original sequence extracted from v2.5, the feature was migrated. This check was performed with ClustalW v2.1 (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Larkin <em>et al.</em> 2007</a>). After the migration step, the GFF file was checked and corrected. Genes which did not contain a start or a stop, contained internal stops or whose CDS sequence had a length not dividable by 3 were marked as potential pseudogenes with “pseudo=true”. To check if a gene contains internal stops each of its CDS features was checked individually for having at least one frame which results in no stop codons. Genes which were identical to other genes (start and stop position are equal) or were contained in other genes were removed. If the 3′ CDS of a gene did not contain a stop codon but could be added by extending the CDS by three nucleotides, the CDS was corrected. The lifted genes were classified as shown in <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/494844/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/494844/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Overview of gene liftover: GFF migration statistics</span><div class="sb-div caption-clear"></div></div></div><p id="p-33">Most genes could be lifted by flo and GeMoMa. The reason why flo was able to lift more proteins is that GeMoMa works with protein sequences and the program was not able to generate proteins for 20,056 CDS features, either because a gene did not possess a CDS or because of faulty CDS sequences.</p></div><div id="sec-24" class="subsection"><h3>Name convention of v3.0 genes</h3><p id="p-34">Old gene IDs were kept in the note attribute of the genes in the GFF and the linkage group numbers of the genetic map are also noted. The names of the genes were changed into Aa3typeNumberGenenumber: Aa for <em>Aethionema arabicum</em>, indicator genome version 3, followed by the type of scaffold, its number and the number of the gene (starting with 1 at the 5′ end), <em>e.g.</em>, Aa3LG1G2 or Aa3SC2601G1). For transcript isoforms (splice variants) this locus nomenclature can be extended by the number of the isoform (.X). Version 3.0 of the genome and all gene models are available at <a href="https://genomevolution.org/coge/GenomeView.pl?gid=36061">https://genomevolution.org/coge/GenomeView.pl?gid=36061</a>.</p></div><div id="sec-25" class="subsection"><h3>Data availability</h3><p id="p-35">The genome version 2.5 is available at: <a href="https://genomevolution.org/coge/GenomeInfo.pl?gid=23428">https://genomevolution.org/coge/GenomeInfo.pl?gid=23428</a>. The genome version 3.0 is available at: <a href="https://genomevolution.org/coge/GenomeView.pl?gid=36061">https://genomevolution.org/coge/GenomeView.pl?gid=36061</a>. The GBS unprocessed and processed reads for genome mapping 2.5 and 3.0 are available as supplmental files S1-S4. The linkage group information for both 2.5 and 3.0 genetic maps are available as S5. A summary of the contamination screen is available as S6. MinION and PacBio single end reads are available from NCBI SRA (BioProject PRJNA558876). Supplemental material available at FigShare: <a href="https://doi.org/10.25387/g3.8233055">https://doi.org/10.25387/g3.8233055</a>.</p></div></div><div class="section results-discussion" id="sec-26"><h2 class="">Results and Discussion</h2><div id="sec-27" class="subsection"><h3>Reference genome improvement</h3><p id="p-36">The published draft version of the <em>Ae. arabicum</em> genome utilized the Ray assembler and contained 59,101 scaffolds with an N50 of 115,195bp (<a id="xref-ref-16-3" class="xref-bibr" href="#ref-16">Haudry <em>et al.</em> 2013</a>). Reassembly using the the AllPathsLG assembler and gap-closing using the SOAPdenovo GapCloser tool were used as a starting point for super-scaffolding. This resulted in a reassembly with 3,166 scaffolds, and a scaffold N<sub>50</sub> of 564,741bp labeled and published as version 2.5 on <a href="https://genomevolution.org/coge/">https://genomevolution.org/coge/</a>. The subsequent genome versions (vAM, v2.6 and v3.0) were obtained using linkage map and long read correction. The quality improvement of the genome is presented as the increase in total number of bases, reduced number of scaffolds and number of gaps, as well as bigger N<sub>50</sub> and smaller L<sub>50</sub> parameters (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>). In comparison with the starting genome v2.5, the final genome v3.0 has 9% less scaffolds (from 3,166 to 2,883). The overall length of genome v3.0 was extended from 196,005,095 to 203,449,326 bases (17% more) and the number of uncalled based was reduced from 25,768,296 to 13,790,434 (from 13.2 to 6.8%) (<a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>).</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/494766/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/494766/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Statistic overview of <em>Aethionema arabicum</em> genome versions</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-28" class="subsection"><h3>Genome improvement using long reads</h3><div id="sec-29" class="subsection"><h4>Read mapping efficiency:</h4><p id="p-38">The results of the mapping of the reads to the genome using PBjelly2 are summarized in <a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>. Almost the same number of PBjelly2 reads were mapped to genome v2.5 and vAM. However, it was important to apply PBjelly2 on both genomes in order to find scaffold connections which were not possible due to a combination of certain scaffolds in vAM (see supplementary file combination_comparison_pbjelly_for_v2.5_vs_pbjelly_for_vGM.ods for details). The genome v2.6 resulted by improving the split vAM genome using PBjelly2, reconnecting scaffolds and resizing gaps if needed. We also compared the results for improving v2.5 with vAM, but there were no new scaffold connections which were missed by improving the v2.6 version, so we did not perform a split step for improving the genome using the MinION reads. The mapping efficiency for the MinION reads is lower than for the PacBio reads due to a contamination of the reads (see Methods for details). There are 5.9% more MinION reads which were mapped to v2.6 than v2.5, demonstrating that the changes done to the genome are supported by the very long reads.</p><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/494678/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/494678/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">Mapping efficiency of PBjelly2’s mapping step. The percentages in brackets give the percentage of the total number of reads (CYP, TUR or CYP + TUR). The line “# covered scaffolds” gives the number of scaffolds in which at least one read was mapped. Here, the number in brackets gives the percentage of the total number of scaffolds</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-30" class="subsection"><h4>The effect of PBjelly2 runs applied to the different assembly versions:</h4><p id="p-39">In comparing the improvement approaches for v2.5, the genetic map approach was able to combine the highest number of scaffolds, resulting in the L<sub>50</sub> value lowered from 56 to 9 and the N<sub>50</sub> value increased 20-fold. The increase of the N<sub>50</sub> value in case of the PBjelly2 result (using the PacBio reads for improving scaffolds) results mainly from improvements of the shorter scaffolds. Comparing the PBjelly2 result for applying the PacBio reads to v2.5 and vAM shows that the reduction of scaffold number and increase of number of bases in the genome is similar (<a id="xref-table-wrap-5-1" class="xref-table" href="#T5">Table 5</a>). MinION reads could also be used for v2.5 assembly improvement, however the results were not as good as for using PacBio reads, due to a much smaller number of reads. Improving the genome v2.6 with the MinION reads is also possible, but the improvement is not as good as for v2.5. This demonstrates that connections done using the PacBio reads are also supported by MinION reads.</p><div id="T5" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/494845/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/494845/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 5</span> <span class="caption-title">Overview of the PBjelly2 result statistics for the different setups</span><div class="sb-div caption-clear"></div></div></div><p id="p-40">Comparison of values for the different genome versions with the values for the PBjelly2 output is shown. The PBjelly2 outputs are denoted in the form “read type” <em>vs.</em> “genome version” to show which reads were used to improve which version of the genome. The result for PacBio <em>vs.</em> vAM was the basis for v2.6 and MinION <em>vs.</em> v2.6 was the basis for v3.0.</p><p id="p-41">While PBjelly2 does not do as good a job as the genetic map approach in connecting scaffolds, its power is revealed by the gap filling. In genome v2.5 a total of 1,910 scaffolds contained uncalled bases. This number was reduced to 1,711 scaffolds (by 7.3%; <a id="xref-table-wrap-6-1" class="xref-table" href="#T6">Table 6</a>) using PBjelly2 with PacBio reads. The exact number of uncalled bases in the v2.5 <em>A. arabicum</em> genome was 25,768,296 (<a id="xref-table-wrap-6-2" class="xref-table" href="#T6">Table 6</a>). In the PBjelly2 result only 13,940,203 Ns (<a id="xref-table-wrap-6-3" class="xref-table" href="#T6">Table 6</a>) are left, a reduction by 45.9%. Comparing this with the PBjelly2 result for the improvement of the vAM genome using PacBio reads (<a id="xref-table-wrap-6-4" class="xref-table" href="#T6">Table 6</a>), more gaps were removed from the connected genome. The number of scaffolds containing Ns was reduced by 10.8% and the overall number of Ns was reduced by 50.0%, while the overall percentage of Ns in the genome remained the same in the two results. Due to the small number of MinION reads, the improvement of the assembly is less pronounced.</p><div id="T6" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/494703/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/494703/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 6</span> <span class="caption-title">Gap/N analysis of different genome versions</span><div class="sb-div caption-clear"></div></div></div><p id="p-42">This table gives an overview of the number of Ns in the different genome versions and the PBjelly2 results. For the number of scaffolds containing Ns, the percentage is given relative to the total number of scaffolds is given in brackets. For the number of Ns, the percentage is relative to the total number of bases in the respective assembly.</p></div></div><div id="sec-31" class="subsection"><h3>Migration of proteins to new genome version</h3><p id="p-43">The genome v2.5 harbors 23,594 annotated protein coding genes. Eight of them could not be lifted because they were located next to a gap in the genome. Since it is possible that PBjelly2 changes the sequences around gaps, the sequences of the genes were not identical anymore and the programs were therefore not able to migrate some genes from one assembly version to another. We checked the expression of the genes which could not be lifted using Illumina RNA-seq data representing several developmental stages (data not shown) and found that all of them had almost no expression, as a result they were not lifted manually. In addition to some unlifted genes, there were 17 genes which could be lifted only partially due to the same reason. All the other 23,569 genes could be lifted. A set of 579 genes were removed due to being identical with other genes, and 140 genes were removed because they were located completely in another gene. A total of 1,202 genes have no starting methionine, 2,055 have no stop, 132 genes contain internal stops and for 1,019 genes the length of the CDSs is not dividable by three. In the end 19,363 genes were lifted which were not marked as potential pseudogene or partial.</p><p id="p-44">We find that the starting point (assembly version) for improvement is not relevant. PBjelly2 is able to make more improvements using the PacBio data than with the MinION data due to the higher number of PacBio reads. The number of added bases per read is higher for PacBio than MinION reads (18.71 <em>vs.</em> 9.67) and also the number of removed Ns is higher (31.07 <em>vs.</em> 5.06) while the MinION reads lead to more scaffold connections per read (2.49 × 10<sup>−4</sup> <em>vs.</em> 3.88 × 10<sup>−4</sup> connected scaffolds). Using the MinION reads for improvement makes only a few changes, but they show that they support the changes which were made to the genome using the PacBio data. Since the <em>Ae. arabicum</em> genome was almost not contaminated at all, only three small scaffolds had to be removed. Gene models were filtered for multiple genes and genes contained in other genes. If a problem with a gene was found, it was marked in the resulting GFF file. We note that there are gene models which are probably not correct and need to be fixed in future studies.</p><p id="p-45">The combination of genetic mapping and long reads significantly improved the structure of the assembly, reducing the total scaffold number and decreasing the number of gaps.</p></div><div id="sec-32" class="subsection"><h3>Genetic map of Aethionema arabicum</h3><div id="sec-33" class="subsection"><h4>Genetic map v2.5:</h4><div id="sec-34" class="subsection"><h5>SNP calling</h5><p id="p-46">A GBS approach (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Rowan <em>et al.</em> 2015</a>) was used to generate genetic variation data for genetic mapping. Illumina sequencing of the parental lines and the RILs resulted in 442,101,405 raw reads after quality filtering. Using the TASSEL package (<a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Glaubitz <em>et al.</em> 2014</a>) to match sequence tags to markers, 160,379 SNPs could be called based on genome v2.5. SNPs identified through the GBS method often take the form of many SNP ‘islands’, where a multitude of SNPs are present over only a few kbp of sequence with the same states across individuals. This makes genetic mapping difficult as it results in a very large number of markers that are mostly redundant. This is often referred to as the “large P, small n” problem where p is the number of markers and n is the number of individuals (or recombination events) being used for mapping (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Ronin <em>et al.</em> 2010</a>). This leads to a situation where the number of markers greatly exceeds the resolution of recombination for the population used; thus, only a fraction of the markers can accurately be ordered. Another factor is that many markers cluster (co-segregate) to the same position in the map or are on the same genomic scaffold for example in our study. When there are many closely-linked markers, then any amount of genotyping errors leads to compounding and increasing estimates of recombination (map inflation). We reduced this SNP amount using a sliding window approach collapsing a group of SNPs that all have the same states across individuals into one single marker over windows of 10 kbp, thus the bigger the scaffold the more selected SNPs. This, together with filtering non-informative SNPs (missing data in more than 30% many individuals) resulted in a core group of markers of 5,428 SNPs.</p></div><div id="sec-35" class="subsection"><h5>Genetic map calculation</h5><p id="p-47">We used JoinMap 4.1 to calculate the genetic linkage map for the <em>Ae. arabicum</em> RIL population. For map v2.5, we first checked the marker similarity among the initial set of 5,428 SNPs that were obtained from GBS based on genome v2.5 by a pairwise comparison. SNPs that were highly similar (higher than 90%) were represented by one marker, which refined the number of markers to 1,818. Grouping was selected at a LOD threshold of 9.0, which led to the grouping of the expected 11 Linkage Groups (LGs) (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). We further optimized each LG to avoid inflation of the map distance due to saturating SNPs using a Maximum likelihood model. Markers that were not more than 0.1 cM away were also eliminated. As a result, a final set of 746 SNP markers was used on 11LGs. Out of 11 LGs, there are three LGs (4, 7 and 11) containing cluster of segregation distorted SNPs (more than 50% of SNPs, supplemental file linkage_group_map.xlsx).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/11/3521/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Aethionema arabicum genetic map v2.5. Genetic map version 2.5 consists of eleven linkage groups. On each linkage group, genetic distance in cM is present on the left and SNP markers on the right." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1711713484" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;em&gt;Aethionema arabicum&lt;/em&gt; genetic map v2.5. Genetic map version 2.5 consists of eleven linkage groups. On each linkage group, genetic distance in cM is present on the left and SNP markers on the right.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/11/3521/F3.medium.gif" width="440" height="276"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/9/11/3521/F3.medium.gif" width="440" height="276"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/11/3521/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/11/3521/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494856" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-48" class="first-child"><em>Aethionema arabicum</em> genetic map v2.5. Genetic map version 2.5 consists of eleven linkage groups. On each linkage group, genetic distance in cM is present on the left and SNP markers on the right.</p><div class="sb-div caption-clear"></div></div></div><p id="p-49">The <em>Ae. arabicum</em> genetic map v2.5 consists of 11 LGs with average distance size of 162.5 cM, covered by 746 SNP markers with average of 67 markers per LG. The average marker spacing was 2.4 cM, equivalent to approximately 169 kbp. The centromere is suggested by the high density of SNP markers within a small genetic distance (<em>e.g.</em>, a low recombination frequency). These markers typically belong to relatively small scaffolds, consistent with a high-repetitive DNA content, where only a few SNPs were called. LG4 centromere is located at the end of the linkage, suggesting an assembly problem or that LG4 is a telocentric chromosome.</p><p id="p-50">Overall the markers are distributed relatively dense and even in v2.5, with the biggest gap smaller than 18 cM. SNPs that reside in the same scaffold were in agreement among each other on the direction of their scaffold.</p></div></div><div id="sec-36" class="subsection"><h4>Genetic map v3.0:</h4><p id="p-51">The procedure to build genetic map v3.0 was similar to v2.5. SNP calling was performed based on genome v3.0 instead of v2.5 resulting in a raw set of 141,914 SNPs. After similar quality control strategy as for v2.5, we construct v3.0 with a core set of 632 SNPs (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). The 11 LGs were maintained with the total size of∼1945 cM, average marker distance of 3.1 cM. This inflation of genetic distance in genetic map v3.0 compared with v2.5 can be explain by the newly retrieved SNPs due to resolved Ns in the genome. These new SNPs are mainly located in the centromeric regions. In general, SNP order and orientation in LGs are in agreement with map v2.5 (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>). We have made adjusted linkage group order in map v3.0 compared with map v2.5 (linkage_group_map.xlsx). Based on the size of the group, the biggest one is LG 1, and the smallest is LG 11 (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/11/3521/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Aethionema arabicum genetic map v3.0. Genetic map version 3.0 consists of eleven linkage groups. On each linkage group, genetic distance in cM is present on the left and SNP markers on the right." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1711713484" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;em&gt;Aethionema arabicum&lt;/em&gt; genetic map v3.0. Genetic map version 3.0 consists of eleven linkage groups. On each linkage group, genetic distance in cM is present on the left and SNP markers on the right.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/11/3521/F4.medium.gif" width="440" height="307"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/9/11/3521/F4.medium.gif" width="440" height="307"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/11/3521/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/11/3521/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494712" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-52" class="first-child"><em>Aethionema arabicum</em> genetic map v3.0. Genetic map version 3.0 consists of eleven linkage groups. On each linkage group, genetic distance in cM is present on the left and SNP markers on the right.</p><div class="sb-div caption-clear"></div></div></div><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/11/3521/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The alignment of genetic map v2.5, v3.0 and physical map. The alignment of the genetic map v2.5 and v3.0 were based on relative SNPs. The left ruler indicates genetic distance in cM and the right indicates physical distance in bp according to genome v3.0." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1711713484" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;The alignment of genetic map v2.5, v3.0 and physical map. The alignment of the genetic map v2.5 and v3.0 were based on relative SNPs. The left ruler indicates genetic distance in cM and the right indicates physical distance in bp according to genome v3.0.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/11/3521/F5.medium.gif" width="440" height="215"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/9/11/3521/F5.medium.gif" width="440" height="215"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/11/3521/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/11/3521/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494644" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-53" class="first-child">The alignment of genetic map v2.5, v3.0 and physical map. The alignment of the genetic map v2.5 and v3.0 were based on relative SNPs. The left ruler indicates genetic distance in cM and the right indicates physical distance in bp according to genome v3.0.</p><div class="sb-div caption-clear"></div></div></div><p id="p-54">However, there is a significant difference between v3.0 and v2.5 at three LGs that harbor clusters of segregation distorted SNPs, LG 5, 6, and 9 (equivalent LG 4, 11 and 7 in v2.5, respectively): the reduced number of markers as well as the increased marker distance (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a> and <a id="xref-fig-4-3" class="xref-fig" href="#F4">4</a>). In order to maintain 11 LGs and certain degree of newly called SNP incorporation, we had to reduced number of distorted markers in those LGs in v3.0, as a result the dearth of markers was observed (see supplemental file. linkage_group_map.xlsx).</p></div></div></div><div class="section conclusions" id="sec-37"><h2 class="">Conclusions and potential implications</h2><p id="p-55"><em>Aethionema</em> is becoming an outgroup model for the rest of the core Brassicaceae. Studies on its genome, relevant life-history traits and their evolution rely on high-quality genomic and genetic resources. Thus, this work helps pave the way for future comparative genomic and trait research of the entire Brassicaceae family. We have provided an advanced version of the <em>Aethionema arabicum</em> genome and its first genetic map, which allows for pseudochromosome construction needed for analysis of genome evolution. It should be noted that due to the liftover procedure no previously undiscovered gene models were added; future work will need to add and refine gene models. Finally, quantitative trait loci (QTL) mapping for the wide range of traits in <em>Aethionema</em> (<em>e.g.</em>, flowering time, chemical defenses (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Hofberger <em>et al.</em> 2013</a>; <a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Mohammadin <em>et al.</em> 2018</a>), fruit heteromorphism (<a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Lenser <em>et al.</em> 2016</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Wilhelmsson <em>et al.</em> 2019</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Arshad <em>et al.</em> 2019</a>) and light-control of germination (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Mérai <em>et al.</em> 2019</a>) will be greatly enabled by the genetic map and improved genome assembly.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-56">We thank Dr. Elio Schijlen for technical support on Genotyping by Sequencing technology, Dr. Lidija Berke for her help on running AllMaps and Marco Göttig, Rabea Meyberg, Christopher Grosche and Manuel Hiss for their help with long read sequencing. We also thank the other members of the SeedAdapt Consortium and Dr. Laurie Grandont for fruitful discussions about the work.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-3"><p id="p-57">Supplemental material available at FigShare: <a href="https://doi.org/10.25387/g3.8233055">https://doi.org/10.25387/g3.8233055</a>.</p></li><li class="fn" id="fn-4"><p id="p-58"><em>Communicating editor: J. Ross-Ibarra</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2019-03-29"><span class="received-label">Received </span>March 29, 2019.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2019-09-11"><span class="accepted-label">Accepted </span>September 11, 2019.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2019 Nguyen <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3521.1" data-doi="10.1007/s00606-006-0415-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Al-Shehbaz</span>,  <span class="cit-name-given-names">I. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. A.</span> <span class="cit-name-surname">Beilstein</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. A.</span> <span class="cit-name-surname">Kellogg</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Systematics and phylogeny of the Brassicaceae (Cruciferae): an overview.</span> <abbr class="cit-jnl-abbrev">Plant Syst. Evol.</abbr> <span class="cit-vol">259</span>: <span class="cit-fpage">89</span>–<span class="cit-lpage">120</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00606-006-0415-z</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BSyst.%2BEvol.%26rft.volume%253D259%26rft.spage%253D89%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00606-006-0415-z%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00606-006-0415-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000238998800003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3521.2" data-doi="10.1139/cjb-2019-0014"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Arshad</span>,  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Marone</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. E.</span> <span class="cit-name-surname">Collinson</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Leubner-Metzger</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Steinbrecher</span></span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">Fracture of the dimorphic fruits of Aethionema arabicum (Brassicaceae).</span> <abbr class="cit-jnl-abbrev">Botany</abbr> <span class="cit-fpage">1</span>–<span class="cit-lpage">11</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1139/cjb-2019-0014</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBotany%26rft_id%253Dinfo%253Adoi%252F10.1139%252Fcjb-2019-0014%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1139/cjb-2019-0014&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3521.3" data-doi="10.3732/ajb.0800065"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Beilstein</span>,  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">I. A.</span> <span class="cit-name-surname">Al‐Shehbaz</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Mathews</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. A.</span> <span class="cit-name-surname">Kellogg</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Brassicaceae phylogeny inferred from phytochrome A and ndhF sequence data: tribes and trichomes revisited.</span> <abbr class="cit-jnl-abbrev">Am. J. Bot.</abbr> <span class="cit-vol">95</span>: <span class="cit-fpage">1307</span>–<span class="cit-lpage">1327</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3732/ajb.0800065</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2BJournal%2Bof%2BBotany%26rft.stitle%253DAm.%2BJ.%2BBot.%26rft.aulast%253DBeilstein%26rft.auinit1%253DM.%2BA.%26rft.volume%253D95%26rft.issue%253D10%26rft.spage%253D1307%26rft.epage%253D1327%26rft.atitle%253DBrassicaceae%2Bphylogeny%2Binferred%2Bfrom%2Bphytochrome%2BA%2Band%2BndhF%2Bsequence%2Bdata%253A%2Btribes%2Band%2Btrichomes%2Brevisited%26rft_id%253Dinfo%253Adoi%252F10.3732%252Fajb.0800065%26rft_id%253Dinfo%253Apmid%252F21632335%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiYW1qYm90IjtzOjU6InJlc2lkIjtzOjEwOiI5NS8xMC8xMzA3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMS8zNTIxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3521.4"><div class="cit-metadata unstructured">Bibalani, G. H., 2012 Investigation on flowering phenology of Brassicaceae in the Shanjan region Shabestar district, NW Iran (usage for honeybees).</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3521.5" data-doi="10.1089/cmb.2009.0238"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Boisvert</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Laviolette</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Corbeil</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Ray: Simultaneous Assembly of Reads from a Mix of High-Throughput Sequencing Technologies.</span> <abbr class="cit-jnl-abbrev">J. Comput. Biol.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">1519</span>–<span class="cit-lpage">1533</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1089/cmb.2009.0238</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bcomputational%2Bbiology%2B%253A%2B%2Ba%2Bjournal%2Bof%2Bcomputational%2Bmolecular%2Bcell%2Bbiology%26rft.stitle%253DJ%2BComput%2BBiol%26rft.aulast%253DBoisvert%26rft.auinit1%253DS.%26rft.volume%253D17%26rft.issue%253D11%26rft.spage%253D1519%26rft.epage%253D1533%26rft.atitle%253DRay%253A%2Bsimultaneous%2Bassembly%2Bof%2Breads%2Bfrom%2Ba%2Bmix%2Bof%2Bhigh-throughput%2Bsequencing%2Btechnologies.%26rft_id%253Dinfo%253Adoi%252F10.1089%252Fcmb.2009.0238%26rft_id%253Dinfo%253Apmid%252F20958248%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1089/cmb.2009.0238&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20958248&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3521.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3521.6" data-doi="10.1093/bioinformatics/btu170"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bolger</span>,  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Lohse</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Usadel</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Trimmomatic: a flexible trimmer for Illumina sequence data.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">2114</span>–<span class="cit-lpage">2120</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btu170</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu170%26rft_id%253Dinfo%253Apmid%252F24695404%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btu170&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24695404&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3521.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000340049100004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3521.7" data-doi="10.1186/1471-2105-13-238"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chaisson</span>,  <span class="cit-name-given-names">M. J.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Tesler</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Mapping single molecule sequencing reads using basic local alignment with successive refinement (BLASR): application and theory.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">238</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2105-13-238</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DChaisson%26rft.auinit1%253DM.%2BJ.%26rft.volume%253D13%26rft.issue%253D1%26rft.spage%253D238%26rft.epage%253D238%26rft.atitle%253DMapping%2Bsingle%2Bmolecule%2Bsequencing%2Breads%2Busing%2BBasic%2BLocal%2BAlignment%2Bwith%2BSuccessive%2BRefinement%2B%2528BLASR%2529%253A%2BTheory%2Band%2BApplication.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-13-238%26rft_id%253Dinfo%253Apmid%252F22988817%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-13-238&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22988817&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3521.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3521.8" data-doi="10.1007/BF02712670"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dellaporta</span>,  <span class="cit-name-given-names">S. L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Wood</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. B.</span> <span class="cit-name-surname">Hicks</span></span></li></ol><cite>, <span class="cit-pub-date">1983</span> <span class="cit-article-title">A plant DNA minipreparation: Version II.</span> <abbr class="cit-jnl-abbrev">Plant Mol. Biol. Report.</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">19</span>–<span class="cit-lpage">21</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/BF02712670</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BMol.%2BBiol.%2BReport.%26rft.volume%253D1%26rft.spage%253D19%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF02712670%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF02712670&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-book" id="cit-9.11.3521.9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Doyle</span>,  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">DNA Protocols for Plants</span>, pp. <span class="cit-fpage">283</span>–<span class="cit-lpage">293</span> in <span class="cit-source">Molecular Techniques in Taxonomy</span>, edited by <span class="cit-ed"> <span class="cit-name-given-names">G. M.</span> <span class="cit-name-surname">Hewitt</span></span>, <span class="cit-ed"> <span class="cit-name-given-names">A. W. B.</span> <span class="cit-name-surname">Johnston</span></span>, and <span class="cit-ed"> <span class="cit-name-given-names">J. P. W.</span> <span class="cit-name-surname">Young</span></span>. NATO ASI Series, <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">Berlin Heidelberg</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3521.10" data-doi="10.1371/journal.pone.0019379"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Elshire</span>,  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. C.</span> <span class="cit-name-surname">Glaubitz</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Q.</span> <span class="cit-name-surname">Sun</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. A.</span> <span class="cit-name-surname">Poland</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Kawamoto</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A Robust, Simple Genotyping-by-Sequencing (GBS) Approach for High Diversity Species.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e19379</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0019379</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DElshire%26rft.auinit1%253DR.%2BJ.%26rft.volume%253D6%26rft.issue%253D5%26rft.spage%253De19379%26rft.epage%253De19379%26rft.atitle%253DA%2Brobust%252C%2Bsimple%2Bgenotyping-by-sequencing%2B%2528GBS%2529%2Bapproach%2Bfor%2Bhigh%2Bdiversity%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0019379%26rft_id%253Dinfo%253Apmid%252F21573248%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0019379&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21573248&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3521.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3521.11" data-doi="10.1371/journal.pone.0047768"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">English</span>,  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Richards</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Han</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Wang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Vee</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Mind the gap: upgrading genomes with Pacific Biosciences RS long-read sequencing technology.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e47768</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0047768</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DEnglish%26rft.auinit1%253DA.%2BC.%26rft.volume%253D7%26rft.issue%253D11%26rft.spage%253De47768%26rft.epage%253De47768%26rft.atitle%253DMind%2Bthe%2BGap%253A%2BUpgrading%2BGenomes%2Bwith%2BPacific%2BBiosciences%2BRS%2BLong-Read%2BSequencing%2BTechnology.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0047768%26rft_id%253Dinfo%253Apmid%252F23185243%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0047768&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23185243&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3521.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3521.12" data-doi="10.1016/j.tplants.2010.11.005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Franzke</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. A.</span> <span class="cit-name-surname">Lysak</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">I. A.</span> <span class="cit-name-surname">Al-Shehbaz</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. A.</span> <span class="cit-name-surname">Koch</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Mummenhoff</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Cabbage family affairs: the evolutionary history of Brassicaceae.</span> <abbr class="cit-jnl-abbrev">Trends Plant Sci.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">108</span>–<span class="cit-lpage">116</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.tplants.2010.11.005</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bplant%2Bscience%26rft.stitle%253DTrends%2BPlant%2BSci%26rft.aulast%253DFranzke%26rft.auinit1%253DA.%26rft.volume%253D16%26rft.issue%253D2%26rft.spage%253D108%26rft.epage%253D116%26rft.atitle%253DCabbage%2Bfamily%2Baffairs%253A%2Bthe%2Bevolutionary%2Bhistory%2Bof%2BBrassicaceae.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tplants.2010.11.005%26rft_id%253Dinfo%253Apmid%252F21177137%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tplants.2010.11.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21177137&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3521.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000288342200006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3521.13" data-doi="10.1371/journal.pone.0090346"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Glaubitz</span>,  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. M.</span> <span class="cit-name-surname">Casstevens</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Lu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Harriman</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. J.</span> <span class="cit-name-surname">Elshire</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">TASSEL-GBS: A High Capacity Genotyping by Sequencing Analysis Pipeline.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e90346</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0090346</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De90346%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0090346%26rft_id%253Dinfo%253Apmid%252F24587335%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0090346&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24587335&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3521.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3521.14" data-doi="10.1073/pnas.1017351108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gnerre</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">MacCallum</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Przybylski</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F. J.</span> <span class="cit-name-surname">Ribeiro</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. N.</span> <span class="cit-name-surname">Burton</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">High-quality draft assemblies of mammalian genomes from massively parallel sequence data.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">1513</span>–<span class="cit-lpage">1518</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.1017351108</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1017351108%26rft_id%253Dinfo%253Apmid%252F21187386%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTA4LzQvMTUxMyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzkvMTEvMzUyMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3521.15" data-doi="10.1186/s12864-017-3555-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>,  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Liu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Hao</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Zhang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Mao</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Plastome phylogeny and early diversification of Brassicaceae.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">176</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12864-017-3555-3</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D18%26rft.spage%253D176%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864-017-3555-3%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12864-017-3555-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3521.16" data-doi="10.1038/ng.2684"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Haudry</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. E.</span> <span class="cit-name-surname">Platts</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Vello</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. R.</span> <span class="cit-name-surname">Hoen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Leclercq</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">An atlas of over 90,000 conserved noncoding sequences provides insight into crucifer regulatory regions.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">891</span>–<span class="cit-lpage">898</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.2684</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D45%26rft.spage%253D891%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2684%26rft_id%253Dinfo%253Apmid%252F23817568%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2684&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23817568&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3521.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3521.17" data-doi="10.1111/tpj.13501"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hiss</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Meyberg</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Westermann</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F. B.</span> <span class="cit-name-surname">Haas</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Schneider</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Sexual reproduction, sporophyte development and molecular variation in the model moss Physcomitrella patens: introducing the ecotype Reute.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">90</span>: <span class="cit-fpage">606</span>–<span class="cit-lpage">620</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/tpj.13501</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BJ.%26rft.volume%253D90%26rft.spage%253D606%26rft_id%253Dinfo%253Adoi%252F10.1111%252Ftpj.13501%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/tpj.13501&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3521.18" data-doi="10.1093/gbe/evt162"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hofberger</span>,  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Lyons</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. P.</span> <span class="cit-name-surname">Edger</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Chris Pires</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Eric Schranz</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Whole genome and tandem duplicate retention facilitated glucosinolate pathway diversification in the mustard family.</span> <abbr class="cit-jnl-abbrev">Genome Biol. Evol.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">2155</span>–<span class="cit-lpage">2173</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/gbe/evt162</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevt162%26rft_id%253Dinfo%253Apmid%252F24171911%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/gbe/evt162&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24171911&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3521.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3521.19" data-doi="10.1093/molbev/msv226"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>,  <span class="cit-name-given-names">C.-H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Sun</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Hu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Zeng</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Zhang</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Resolution of Brassicaceae Phylogeny Using Nuclear Genes Uncovers Nested Radiations and Supports Convergent Morphological Evolution.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">394</span>–<span class="cit-lpage">412</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/msv226</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsv226%26rft_id%253Dinfo%253Apmid%252F26516094%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msv226&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26516094&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3521.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3521.20" data-doi="10.1371/journal.pcbi.1004957"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huson</span>,  <span class="cit-name-given-names">D. H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Beier</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Flade</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Górska</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">El-Hadidi</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">MEGAN Community Edition - Interactive Exploration and Analysis of Large-Scale Microbiome Sequencing Data.</span> <abbr class="cit-jnl-abbrev">PLOS Comput. Biol.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">e1004957</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pcbi.1004957</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLOS%2BComput.%2BBiol.%26rft.volume%253D12%26rft.spage%253De1004957%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.1004957%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.1004957&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3521.21" data-doi="10.1078/1433-8319-00021"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Imbert</span>,  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Ecological consequences and ontogeny of seed heteromorphism.</span> <abbr class="cit-jnl-abbrev">Perspect. Plant Ecol. Evol. Syst.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">13</span>–<span class="cit-lpage">36</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1078/1433-8319-00021</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPerspect.%2BPlant%2BEcol.%2BEvol.%2BSyst.%26rft.volume%253D5%26rft.spage%253D13%26rft_id%253Dinfo%253Adoi%252F10.1078%252F1433-8319-00021%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1078/1433-8319-00021&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3521.22" data-doi="10.1093/nar/gkw092"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Keilwagen</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Wenk</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. L.</span> <span class="cit-name-surname">Erickson</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. H.</span> <span class="cit-name-surname">Schattat</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Grau</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Using intron position conservation for homology-based gene prediction.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">e89</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkw092</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkw092%26rft_id%253Dinfo%253Apmid%252F26893356%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkw092&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26893356&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3521.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3521.23" data-doi="10.1101/gr.229102. Article published online before print in May 2002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kent</span>,  <span class="cit-name-given-names">W. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. W.</span> <span class="cit-name-surname">Sugnet</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. S.</span> <span class="cit-name-surname">Furey</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. M.</span> <span class="cit-name-surname">Roskin</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. H.</span> <span class="cit-name-surname">Pringle</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">The Human Genome Browser at UCSC.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">996</span>–<span class="cit-lpage">1006</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.229102</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.229102.%2BArticle%2Bpublished%2Bonline%2Bbefore%2Bprint%2Bin%2BMay%2B2002%26rft_id%253Dinfo%253Apmid%252F12045153%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjEyLzYvOTk2IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMS8zNTIxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3521.24" data-doi="10.1111/tpj.13801"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lang</span>,  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. K.</span> <span class="cit-name-surname">Ullrich</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Murat</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Fuchs</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Jenkins</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">The Physcomitrella patens chromosome-scale assembly reveals moss genome structure and evolution.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">93</span>: <span class="cit-fpage">515</span>–<span class="cit-lpage">533</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/tpj.13801</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BJ.%26rft.volume%253D93%26rft.spage%253D515%26rft_id%253Dinfo%253Adoi%252F10.1111%252Ftpj.13801%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/tpj.13801&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3521.25" data-doi="10.1093/bioinformatics/btm404"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Larkin</span>,  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Blackshields</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N. P.</span> <span class="cit-name-surname">Brown</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Chenna</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. A.</span> <span class="cit-name-surname">McGettigan</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Clustal W and Clustal X version 2.0.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">2947</span>–<span class="cit-lpage">2948</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btm404</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtm404%26rft_id%253Dinfo%253Apmid%252F17846036%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btm404&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17846036&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3521.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000251197700021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3521.26" data-doi="10.1104/pp.16.00838"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lenser</span>,  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Graeber</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Ö. S.</span> <span class="cit-name-surname">Cevik</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Adigüzel</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. A.</span> <span class="cit-name-surname">Dönmez</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Developmental Control and Plasticity of Fruit and Seed Dimorphism in Aethionema arabicum.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">172</span>: <span class="cit-fpage">1691</span>–<span class="cit-lpage">1707</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1104/pp.16.00838</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.16.00838%26rft_id%253Dinfo%253Apmid%252F27702842%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czoxMDoiMTcyLzMvMTY5MSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzkvMTEvMzUyMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3521.27" data-doi="10.1093/bioinformatics/btp698"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">H.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Durbin</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Fast and accurate long-read alignment with Burrows–Wheeler transform.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">589</span>–<span class="cit-lpage">595</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btp698</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp698%26rft_id%253Dinfo%253Apmid%252F20080505%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp698&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20080505&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3521.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000274973800001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3521.28" data-doi="10.1093/bioinformatics/btp352"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Handsaker</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Wysoker</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Fennell</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Ruan</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The Sequence Alignment/Map format and SAMtools.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">2078</span>–<span class="cit-lpage">2079</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btp352</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp352%26rft_id%253Dinfo%253Apmid%252F19505943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp352&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19505943&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3521.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268808600014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3521.29" data-doi="10.1186/2047-217X-1-18"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Luo</span>,  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Liu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Xie</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Li</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Huang</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">SOAPdenovo2: an empirically improved memory-efficient short-read de novo assembler.</span> <abbr class="cit-jnl-abbrev">Gigascience</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">18</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/2047-217X-1-18</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGigascience%26rft.volume%253D1%26rft.spage%253D18%26rft_id%253Dinfo%253Adoi%252F10.1186%252F2047-217X-1-18%26rft_id%253Dinfo%253Apmid%252F23587118%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/2047-217X-1-18&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23587118&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3521.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3521.30" data-doi="10.1093/jxb/erz146"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mérai</span>,  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Graeber</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Wilhelmsson</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. K.</span> <span class="cit-name-surname">Ullrich</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Arshad</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">Aethionema arabicum: a novel model plant to study the light control of seed germination.</span> <abbr class="cit-jnl-abbrev">J. Exp. Bot.</abbr> <span class="cit-vol">70</span>: <span class="cit-fpage">3313</span>–<span class="cit-lpage">3328</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/jxb/erz146</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BExp.%2BBot.%26rft.volume%253D70%26rft.spage%253D3313%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjxb%252Ferz146%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/jxb/erz146&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3521.31" data-doi="10.3732/ajb.1700091"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mohammadin</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Peterse</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. J.</span> <span class="cit-name-surname">van de Kerke</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L. W.</span> <span class="cit-name-surname">Chatrou</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. A.</span> <span class="cit-name-surname">Dönmez</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Anatolian origins and diversification of Aethionema, the sister lineage of the core Brassicaceae.</span> <abbr class="cit-jnl-abbrev">Am. J. Bot.</abbr> <span class="cit-vol">104</span>: <span class="cit-fpage">1042</span>–<span class="cit-lpage">1054</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3732/ajb.1700091</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BJ.%2BBot.%26rft_id%253Dinfo%253Adoi%252F10.3732%252Fajb.1700091%26rft_id%253Dinfo%253Apmid%252F28743759%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiYW1qYm90IjtzOjU6InJlc2lkIjtzOjEwOiIxMDQvNy8xMDQyIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMS8zNTIxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3521.32" data-doi="10.1007/s00606-018-1494-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mohammadin</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Wang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Liu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Moazzeni</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Ertugrul</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Genome-wide nucleotide diversity and associations with geography, ploidy level and glucosinolate profiles in Aethionema arabicum (Brassicaceae).</span> <abbr class="cit-jnl-abbrev">Plant Syst. Evol.</abbr> <span class="cit-vol">304</span>: <span class="cit-fpage">619</span>–<span class="cit-lpage">630</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00606-018-1494-3</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BSyst.%2BEvol.%26rft.volume%253D304%26rft.spage%253D619%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00606-018-1494-3%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00606-018-1494-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3521.33" data-doi="10.1093/nar/gkv1290"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">NCBI Resource Coordinators</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Database resources of the National Center for Biotechnology Information.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">D7</span>–<span class="cit-lpage">D19</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkv1290</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkv1290%26rft_id%253Dinfo%253Apmid%252F26615191%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkv1290&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26615191&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3521.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3521.34"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ronin</span>,  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Mester</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Minkov</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Korol</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Building reliable genetic maps: different mapping strategies may result in different maps.</span> <abbr class="cit-jnl-abbrev">Nat. Sci.</abbr> <span class="cit-vol">02</span>: <span class="cit-fpage">576</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BSci.%26rft.volume%253D02%26rft.spage%253D576%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3521.35"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rowan</span>,  <span class="cit-name-given-names">B. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Patel</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Weigel</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Schneeberger</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Rapid and Inexpensive Whole-Genome Genotyping-by-Sequencing for Crossover Localization and Fine-Scale Genetic Mapping. G3: Genes, Genomes</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">385</span>–<span class="cit-lpage">398</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D5%26rft.spage%253D385%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3521.36" data-doi="10.1111/j.1365-313X.1993.00739.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stam</span>,  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Construction of integrated genetic linkage maps by means of a new computer package: Join Map.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">739</span>–<span class="cit-lpage">744</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1365-313X.1993.00739.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BJ.%26rft.volume%253D3%26rft.spage%253D739%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-313X.1993.00739.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-313X.1993.00739.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1993LC75800012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3521.37" data-doi="10.1186/s13059-014-0573-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tang</span>,  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Zhang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Miao</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Zhang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Ming</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">ALLMAPS: robust scaffold ordering based on multiple maps.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">3</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s13059-014-0573-1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D16%26rft.spage%253D3%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-014-0573-1%26rft_id%253Dinfo%253Apmid%252F25583564%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13059-014-0573-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25583564&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3521.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-web" id="cit-9.11.3521.38"><div class="cit-metadata unstructured">van Ooijen, J. W., 2006 JoinMap 4. Software for the calculation of genetic linkage maps in experimental populations.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3521.39" data-doi="10.1093/bioinformatics/btu590"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Watson</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Thomson</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Risse</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Talbot</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Santoyo-Lopez</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">poRe: an R package for the visualization and analysis of nanopore sequencing data.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">114</span>–<span class="cit-lpage">115</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btu590</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu590%26rft_id%253Dinfo%253Apmid%252F25173419%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btu590&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25173419&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3521.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3521.40" data-doi="10.1186/s12864-019-5452-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wilhelmsson</span>,  <span class="cit-name-given-names">P. K. I.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. O.</span> <span class="cit-name-surname">Chandler</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Fernandez-Pozo</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Graeber</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. K.</span> <span class="cit-name-surname">Ullrich</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">Usability of reference-free transcriptome assemblies for detection of differential expression: a case study on Aethionema arabicum dimorphic seeds.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">95</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12864-019-5452-4</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D20%26rft.spage%253D95%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864-019-5452-4%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12864-019-5452-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3521.41" data-doi="10.1093/bioinformatics/btq057"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>,  <span class="cit-name-given-names">T. D.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Nacu</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Fast and SNP-tolerant detection of complex variants and splicing in short reads.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">873</span>–<span class="cit-lpage">881</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btq057</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq057%26rft_id%253Dinfo%253Apmid%252F20147302%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btq057&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20147302&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3521.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000276045800004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/9/11/3521.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/11/3513" title="Updated Genome Assembly and Annotation for Metrosideros polymorpha, an Emerging Model Tree Species of Ecological Divergence" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/11/3531" title="De Novo Assembly and Annotation from Parental and F1 Puma Genomes of the Florida Panther Genetic Restoration Program" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/9/11"><span>Volume 9 Issue 11, November 2019</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/9/11.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/9/11.cover-source.jpg?itok=DjqkLGk9" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 9 (11)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/11/3521" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-wh2YziXApGLJM2au1UTfoFGIhVxJoGCHamhI4GnRM_4" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/genome-report" class="" data-icon-position="" data-hide-link-title="0">Genome Report</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/9/11/3521" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/9/11/3521" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/9/11/3521" class="active" data-icon-position="" data-hide-link-title="0">Genome Improvement and Genetic Map Construction for Aethionema arabicum, the First Divergent Branch in the Brassicaceae Family</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/494623" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-3RxnaQNiDO_DkC0IgOyMqMpXJkTeBBKPHfKOLuENtH4" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/11/3521.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/9/11/3521" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-lrTdhvxQJ5zvaK8W49U9BA-NTSpAbt24Y2UN8sdIA5g" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/9/11/3521" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/11/3521.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494623" id="node494623--2" data-pisa="ggg;9/11/3521" data-pisa-master="ggg;g3.119.400657" data-apath="/ggg/9/11/3521.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genome Improvement and Genetic Map Construction for <em>Aethionema arabicum</em>, the First Divergent Branch in the Brassicaceae Family</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Thu-Phuong Nguyen</span>, <span class="highwire-citation-author" data-delta="1">Cornelia Mühlich</span>, <span class="highwire-citation-author" data-delta="2">Setareh Mohammadin</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0001-9865-574X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Erik van den Bergh</span>, <span class="highwire-citation-author" data-delta="4">Adrian E. Platts</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0002-7711-5282" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Fabian B. Haas</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0002-0225-873X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Stefan A. Rensing</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="7"><a href="http://orcid.org/0000-0001-6777-6565" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>M. Eric Schranz</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3521-3530;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400657 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494623--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Thu-Phuong Nguyen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Biosystematics Group, Wageningen University, Droevendaalsesteeg 1, 6708 PB Wageningen, The Netherlands,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Thu-Phuong%2BNguyen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nguyen%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AThu-Phuong%2BNguyen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Cornelia Mühlich </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Faculty of Biology, Plant Cell Biology, University of Marburg, Karl-von-Frisch-Str. 8, 35043 Marburg, Germany,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Cornelia%2BM%C3%BChlich%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=M%C3%BChlich%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACornelia%2BM%25C3%25BChlich%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Setareh Mohammadin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Biosystematics Group, Wageningen University, Droevendaalsesteeg 1, 6708 PB Wageningen, The Netherlands,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Setareh%2BMohammadin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mohammadin%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASetareh%2BMohammadin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Erik van den Bergh </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Biosystematics Group, Wageningen University, Droevendaalsesteeg 1, 6708 PB Wageningen, The Netherlands,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Erik%2Bvan%2Bden%2BBergh%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=van%20den%20Bergh%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AErik%2Bvan%2Bden%2BBergh%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-9865-574X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Erik van den Bergh</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Adrian E. Platts </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Center for Genomics and Systems Biology, Department of Biology, New York University, New York, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Adrian%2BE.%2BPlatts%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Platts%20AE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAdrian%2BE.%2BPlatts%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Fabian B. Haas </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Faculty of Biology, Plant Cell Biology, University of Marburg, Karl-von-Frisch-Str. 8, 35043 Marburg, Germany,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Fabian%2BB.%2BHaas%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Haas%20FB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AFabian%2BB.%2BHaas%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-7711-5282" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Fabian B. Haas</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Stefan A. Rensing </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Faculty of Biology, Plant Cell Biology, University of Marburg, Karl-von-Frisch-Str. 8, 35043 Marburg, Germany,</div><div class='author-affiliation'><span class='nlm-sup'>§</span>LOEWE Center for Synthetic Microbiology (SYNMIKRO), Philipps University of Marburg, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Stefan%2BA.%2BRensing%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rensing%20SA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AStefan%2BA.%2BRensing%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-0225-873X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Stefan A. Rensing</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:stefan.rensing@biologie.uni-marburg.de" class="" data-icon-position="" data-hide-link-title="0">stefan.rensing@biologie.uni-marburg.de</a>
<a href="mailto:eric.schranz@wur.nl" class="" data-icon-position="" data-hide-link-title="0">eric.schranz@wur.nl</a></span></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">M. Eric Schranz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Biosystematics Group, Wageningen University, Droevendaalsesteeg 1, 6708 PB Wageningen, The Netherlands,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=M.%2BEric%2BSchranz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Schranz%20ME&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AM.%2BEric%2BSchranz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-6777-6565" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for M. Eric Schranz</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:stefan.rensing@biologie.uni-marburg.de" class="" data-icon-position="" data-hide-link-title="0">stefan.rensing@biologie.uni-marburg.de</a>
<a href="mailto:eric.schranz@wur.nl" class="" data-icon-position="" data-hide-link-title="0">eric.schranz@wur.nl</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/9/11/3521" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/9/11/3521&amp;title=Genome%20Improvement%20and%20Genetic%20Map%20Construction%20for%20Aethionema%20arabicum%2C%20the%20First%20Divergent%20Branch%20in%20the%20Brassicaceae%20Family" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/9/11/3521&amp;title=Genome%20Improvement%20and%20Genetic%20Map%20Construction%20for%20Aethionema%20arabicum%2C%20the%20First%20Divergent%20Branch%20in%20the%20Brassicaceae%20Family" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/9/11/3521&amp;title=Genome%20Improvement%20and%20Genetic%20Map%20Construction%20for%20Aethionema%20arabicum%2C%20the%20First%20Divergent%20Branch%20in%20the%20Brassicaceae%20Family" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/9/11/3521&amp;text=Genome%20Improvement%20and%20Genetic%20Map%20Construction%20for%20Aethionema%20arabicum%2C%20the%20First%20Divergent%20Branch%20in%20the%20Brassicaceae%20Family" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/9/11/3521&amp;title=Genome%20Improvement%20and%20Genetic%20Map%20Construction%20for%20Aethionema%20arabicum%2C%20the%20First%20Divergent%20Branch%20in%20the%20Brassicaceae%20Family" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/9/11/3521&amp;t=Genome%20Improvement%20and%20Genetic%20Map%20Construction%20for%20Aethionema%20arabicum%2C%20the%20First%20Divergent%20Branch%20in%20the%20Brassicaceae%20Family" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/9/11/3521&amp;title=Genome%20Improvement%20and%20Genetic%20Map%20Construction%20for%20Aethionema%20arabicum%2C%20the%20First%20Divergent%20Branch%20in%20the%20Brassicaceae%20Family" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/9/11/3521&amp;title=Genome%20Improvement%20and%20Genetic%20Map%20Construction%20for%20Aethionema%20arabicum%2C%20the%20First%20Divergent%20Branch%20in%20the%20Brassicaceae%20Family" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/494623/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494623" id="node494623--3" data-pisa="ggg;9/11/3521" data-pisa-master="ggg;g3.119.400657" data-apath="/ggg/9/11/3521.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genome Improvement and Genetic Map Construction for <em>Aethionema arabicum</em>, the First Divergent Branch in the Brassicaceae Family</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Thu-Phuong Nguyen</span>, <span class="highwire-citation-author" data-delta="1">Cornelia Mühlich</span>, <span class="highwire-citation-author" data-delta="2">Setareh Mohammadin</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0001-9865-574X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Erik van den Bergh</span>, <span class="highwire-citation-author" data-delta="4">Adrian E. Platts</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0002-7711-5282" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Fabian B. Haas</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0002-0225-873X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Stefan A. Rensing</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="7"><a href="http://orcid.org/0000-0001-6777-6565" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>M. Eric Schranz</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3521-3530;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400657 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494623--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Thu-Phuong Nguyen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Biosystematics Group, Wageningen University, Droevendaalsesteeg 1, 6708 PB Wageningen, The Netherlands,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Thu-Phuong%2BNguyen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nguyen%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AThu-Phuong%2BNguyen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Cornelia Mühlich </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Faculty of Biology, Plant Cell Biology, University of Marburg, Karl-von-Frisch-Str. 8, 35043 Marburg, Germany,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Cornelia%2BM%C3%BChlich%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=M%C3%BChlich%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACornelia%2BM%25C3%25BChlich%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Setareh Mohammadin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Biosystematics Group, Wageningen University, Droevendaalsesteeg 1, 6708 PB Wageningen, The Netherlands,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Setareh%2BMohammadin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mohammadin%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASetareh%2BMohammadin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Erik van den Bergh </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Biosystematics Group, Wageningen University, Droevendaalsesteeg 1, 6708 PB Wageningen, The Netherlands,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Erik%2Bvan%2Bden%2BBergh%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=van%20den%20Bergh%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AErik%2Bvan%2Bden%2BBergh%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-9865-574X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Erik van den Bergh</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Adrian E. Platts </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Center for Genomics and Systems Biology, Department of Biology, New York University, New York, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Adrian%2BE.%2BPlatts%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Platts%20AE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAdrian%2BE.%2BPlatts%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Fabian B. Haas </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Faculty of Biology, Plant Cell Biology, University of Marburg, Karl-von-Frisch-Str. 8, 35043 Marburg, Germany,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Fabian%2BB.%2BHaas%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Haas%20FB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AFabian%2BB.%2BHaas%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-7711-5282" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Fabian B. Haas</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Stefan A. Rensing </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Faculty of Biology, Plant Cell Biology, University of Marburg, Karl-von-Frisch-Str. 8, 35043 Marburg, Germany,</div><div class='author-affiliation'><span class='nlm-sup'>§</span>LOEWE Center for Synthetic Microbiology (SYNMIKRO), Philipps University of Marburg, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Stefan%2BA.%2BRensing%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rensing%20SA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AStefan%2BA.%2BRensing%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-0225-873X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Stefan A. Rensing</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:stefan.rensing@biologie.uni-marburg.de" class="" data-icon-position="" data-hide-link-title="0">stefan.rensing@biologie.uni-marburg.de</a>
<a href="mailto:eric.schranz@wur.nl" class="" data-icon-position="" data-hide-link-title="0">eric.schranz@wur.nl</a></span></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">M. Eric Schranz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Biosystematics Group, Wageningen University, Droevendaalsesteeg 1, 6708 PB Wageningen, The Netherlands,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=M.%2BEric%2BSchranz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Schranz%20ME&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AM.%2BEric%2BSchranz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-6777-6565" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for M. Eric Schranz</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:stefan.rensing@biologie.uni-marburg.de" class="" data-icon-position="" data-hide-link-title="0">stefan.rensing@biologie.uni-marburg.de</a>
<a href="mailto:eric.schranz@wur.nl" class="" data-icon-position="" data-hide-link-title="0">eric.schranz@wur.nl</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=3521&amp;rft.epage=3530&amp;rft.atitle=Genome%20Improvement%20and%20Genetic%20Map%20Construction%20for%20Aethionema%20arabicum%2C%20the%20First%20Divergent%20Branch%20in%20the%20Brassicaceae%20Family&amp;rft.volume=9&amp;rft.issue=11&amp;rft.date=2019-11-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Nguyen%2C+Thu-Phuong&amp;rft.au=M%C3%BChlich%2C+Cornelia&amp;rft.au=Mohammadin%2C+Setareh&amp;rft.au=van+den+Bergh%2C+Erik&amp;rft.au=Platts%2C+Adrian+E.&amp;rft.au=Haas%2C+Fabian+B.&amp;rft.au=Rensing%2C+Stefan+A.&amp;rft.au=Schranz%2C+M.+Eric"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/494623/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/494623/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/494623/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/494623/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/494623/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/494623/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/494623/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/494623/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/494623/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/494623/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/494623/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/494623/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496044" id="node496044" data-pisa="ggg;g3.119.400871v1" data-pisa-master="ggg;g3.119.400871" data-apath="/ggg/early/2020/03/17/g3.119.400871.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/17/g3.119.400871" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title"><em>De Novo</em> Genome Assembly of the <em>indica</em> Rice Variety IR64 Using Linked-Read Sequencing and Nanopore Sequencing</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496040" id="node496040" data-pisa="ggg;g3.119.400893v1" data-pisa-master="ggg;g3.119.400893" data-apath="/ggg/early/2020/03/12/g3.119.400893.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/12/g3.119.400893" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Understanding the Genetic Domestication History of the Jianchang Duck by Genotyping and Sequencing of Genomic Genes Under Selection</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496037" id="node496037" data-pisa="ggg;g3.120.401071v1" data-pisa-master="ggg;g3.120.401071" data-apath="/ggg/early/2020/03/11/g3.120.401071.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/11/g3.120.401071" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title"><em>De Novo</em> Genome Assembly of the Meadow Brown Butterfly, <em>Maniola jurtina</em></span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Genome%20Report" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Genome Report</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/9/11/3521" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Background</a></li><li class="odd"><a href="#sec-2" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Data description and Methods</a></li><li class="even"><a href="#sec-8" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Genotyping by Sequencing (GBS)</a></li><li class="odd"><a href="#sec-15" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Contamination removal</a></li><li class="even"><a href="#sec-16" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Long read generation for genome improvement</a></li><li class="odd"><a href="#sec-26" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results and Discussion</a></li><li class="even"><a href="#sec-37" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Conclusions and potential implications</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/9/11/3521.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/9/11/3521.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/9/11/3521.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
