<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/9/11/3575" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/9/11/3575.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/9/11/3575.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/9/11/3575.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/9/11/3575.atom" />
<meta name="HW.pisa" content="ggg;9/11/3575" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="A Deep Learning Approach for Detecting Copy Number Variation in Next-Generation Sequencing Data" />
<meta name="DC.Identifier" content="10.1534/g3.119.400596" />
<meta name="DC.Date" content="2019-11-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2019 Hill, Unckless. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Copy number variants (CNV) are associated with phenotypic variation in several species. However, properly detecting changes in copy numbers of sequences remains a difficult problem, especially in lower quality or lower coverage next-generation sequencing data. Here, inspired by recent applications of machine learning in genomics, we describe a method to detect duplications and deletions in short-read sequencing data. In low coverage data, machine learning appears to be more powerful in the detection of CNVs than the gold-standard methods of coverage estimation alone, and of equal power in high coverage data. We also demonstrate how replicating training sets allows a more precise detection of CNVs, even identifying novel CNVs in two genomes previously surveyed thoroughly for CNVs using long read data." />
<meta name="DC.Contributor" content="Tom Hill" />
<meta name="DC.Contributor" content="Robert L. Unckless" />
<meta name="article:published_time" content="2019-11-01" />
<meta name="article:section" content="Software and Data Resources" />
<meta name="citation_title" content="A Deep Learning Approach for Detecting Copy Number Variation in Next-Generation Sequencing Data" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Copy number variants (CNV) are associated with phenotypic variation in several species. However, properly detecting changes in copy numbers of sequences remains a difficult problem, especially in lower quality or lower coverage next-generation sequencing data. Here, inspired by recent applications of machine learning in genomics, we describe a method to detect duplications and deletions in short-read sequencing data. In low coverage data, machine learning appears to be more powerful in the detection of CNVs than the gold-standard methods of coverage estimation alone, and of equal power in high coverage data. We also demonstrate how replicating training sets allows a more precise detection of CNVs, even identifying novel CNVs in two genomes previously surveyed thoroughly for CNVs using long read data.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2019/11/01" />
<meta name="citation_mjid" content="ggg;9/11/3575" />
<meta name="citation_id" content="9/11/3575" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/9/11/3575" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/9/11/3575.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/9/11/3575.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/9/11/3575.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.119.400596" />
<meta name="citation_pmid" content="31455677" />
<meta name="citation_volume" content="9" />
<meta name="citation_issue" content="11" />
<meta name="citation_num_pages" content="8" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Software and Data Resources" />
<meta name="citation_firstpage" content="3575" />
<meta name="citation_lastpage" content="3582" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Tom Hill" />
<meta name="citation_author_institution" content="4055 Haworth Hall, The Department of Molecular Biosciences, University of Kansas, 1200 Sunnyside Avenue, Lawrence, KS 66045." />
<meta name="citation_author_email" content="tom.hill@ku.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-4661-6391" />
<meta name="citation_author" content="Robert L. Unckless" />
<meta name="citation_author_institution" content="4055 Haworth Hall, The Department of Molecular Biosciences, University of Kansas, 1200 Sunnyside Avenue, Lawrence, KS 66045." />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-8586-7137" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=A. Abyzov;citation_author=A. E. Urban;citation_author=M. Snyder;citation_author=M. Gerstein;citation_title=CNVnator: an approach to discover, genotype, and characterize typical and atypical CNVs from family and population genome sequencing.;citation_volume=21;citation_year=2011;citation_pmid=21324876;citation_doi=10.1101/gr.114876.110" />
<meta name="citation_reference" content="citation_journal_title=Annu. Rev. Ecol. Evol. Syst.;citation_author=M. A. Beaumont;citation_title=Approximate Bayesian Computation in Evolution and Ecology.;citation_pages=379-406;citation_volume=41;citation_year=2010;citation_doi=10.1146/annurev-ecolsys-102209-144621" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=M. A. Beaumont;citation_author=W. Zhang;citation_author=D. J. Balding;citation_title=Approximate Bayesian Computation in Population Genetics;citation_pages=2025-2035;citation_volume=162;citation_year=2002;citation_issue=4;citation_pmid=12524368" />
<meta name="citation_reference" content="citation_journal_title=PLOS Comput. Biol.;citation_author=R. Bouckaert;citation_author=J. Heled;citation_author=D. Kühnert;citation_author=T. Vaughan;citation_author=C. H. Wu;citation_title=BEAST 2: A Software Platform for Bayesian Evolutionary Analysis.;citation_pages=1003537e;citation_volume=10;citation_year=2014;citation_pmid=24722319;citation_doi=10.1371/journal.pcbi.1003537" />
<meta name="citation_reference" content="citation_journal_title=Chemom. Intell. Lab. Syst.;citation_author=C. D. Brown;citation_author=H. T. Davis;citation_title=RECEIVER OPERATING CHARACTERISTICS CURVES AND RELATED DECISION MEASURES: A TUTORIAL;citation_pages=24-38;citation_volume=80;citation_year=2006;citation_doi=10.1016/j.chemolab.2005.05.004" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=M. Cardoso-Moreira;citation_author=J. R. Arguello;citation_author=S. Gottipati;citation_author=L. G. Harshman;citation_author=J. K. Grenier;citation_title=Evidence for the fixation of gene duplications by positive selection in Drosophila.;citation_volume=26;citation_year=2016;citation_pmid=27197209;citation_doi=10.1101/gr.199323.115" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=M. Chakraborty;citation_author=N. W. VanKuren;citation_author=R. Zhao;citation_author=X. Zhang;citation_author=S. Kalsow;citation_title=Hidden genetic variation shapes the structure of functional elements in Drosophila.;citation_pages=20-25;citation_volume=50;citation_year=2018;citation_doi=10.1038/s41588-017-0010-y" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=X. Chen;citation_author=O. Schulz-Trieglaff;citation_author=R. Shaw;citation_author=B. Barnes;citation_author=F. Schlesinger;citation_title=Manta: Rapid detection of structural variants and indels for germline and cancer sequencing applications.;citation_volume=32;citation_year=2016;citation_pmid=26647377;citation_doi=10.1093/bioinformatics/btv710" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=P. Dharanipragada;citation_author=S. Vogeti;citation_author=N. Parekh;citation_title=iCopyDAV: Integrated platform for copy number variations-Detection, annotation and visualization.;citation_pages=e0195334;citation_volume=13;citation_year=2018;citation_doi=10.1371/journal.pone.0195334" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=G. dos Santos;citation_author=A. J. Schroeder;citation_author=J. L. Goodman;citation_author=V. B. Strelets;citation_author=M. A. Crosby;citation_title=FlyBase: Introduction of the Drosophila melanogaster Release 6 reference genome assembly and large-scale migration of genome annotations.;citation_volume=43;citation_year=2015;citation_pmid=25398896;citation_doi=10.1093/nar/gku1099" />
<meta name="citation_reference" content="citation_journal_title=Infect. Genet. Evol.;citation_author=T. Hill;citation_author=R. L. Unckless;citation_title=The dynamic evolution of Drosophila innubila Nudivirus.;citation_pages=151-157;citation_volume=57;citation_year=2018;citation_doi=10.1016/j.meegid.2017.11.013" />
<meta name="citation_reference" content="citation_journal_title=Annu. Rev. Genomics Hum. Genet.;citation_author=K. Inoue;citation_author=J. R. Lupski;citation_title=Molecular Mechanisms for Genomic Disorders.;citation_volume=3;citation_year=2002;citation_pmid=12142364;citation_doi=10.1146/annurev.genom.3.032802.120023" />
<meta name="citation_reference" content="citation_journal_title=Genome Res. ;citation_author=J. L. Freeman;citation_author=G. H. Perry;citation_author=L. Feuk;citation_author=R. Redon;citation_author=S. A. McCarroll;citation_title=Copy Number Variation: New Insights in Genome Diversity.;citation_volume=16;citation_year=2006;citation_pmid=16809666;citation_doi=10.1101/gr.3677206" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=J. D. Jensen;citation_author=K. R. Thornton;citation_author=P. Andolfatto;citation_title=An approximate bayesian estimator suggests strong, recurrent selective sweeps in Drosophila.;citation_pages=e1000198-e1000198;citation_volume=4;citation_year=2008;citation_issue=9;citation_pmid=18802463;citation_doi=10.1371/journal.pgen.1000198" />
<meta name="citation_reference" content="citation_year=2018" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=J. O. Korbel;citation_author=A. E. Urban;citation_author=J. P. Affourtit;citation_author=B. Godwin;citation_author=F. Grubert;citation_title=Paired-End Mapping Reveals Extensive Structural Variation in the Human Genome.;citation_volume=318;citation_year=2007;citation_pmid=17901297;citation_doi=10.1126/science.1149504" />
<meta name="citation_reference" content="citation_journal_title=Brief. Bioinform.;citation_author=P. Larrañaga;citation_author=B. Calvo;citation_author=R. Santana;citation_author=C. Bielza;citation_author=J. Galdiano;citation_title=Machine learning in bioinformatics.;citation_volume=7;citation_year=2006;citation_pmid=16761367;citation_doi=10.1093/bib/bbk007" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=R. M. Layer;citation_author=C. Chiang;citation_author=A. R. Quinlan;citation_author=I. M. Hall;citation_title=LUMPY: a probabilistic framework for structural variant discovery.;citation_pages=R84;citation_volume=15;citation_year=2014;citation_pmid=24970577;citation_doi=10.1186/gb-2014-15-6-r84" />
<meta name="citation_reference" content="citation_title=WGsim - Read simulator for next generation sequencing;citation_year=2011" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=R. Durbin;citation_title=Fast and accurate short read alignment with Burrows-Wheeler transform.;citation_volume=25;citation_year=2009;citation_pmid=19451168;citation_doi=10.1093/bioinformatics/btp324" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=B. Handsaker;citation_author=A. Wysoker;citation_author=T. Fennell;citation_author=J. Ruan;citation_title=The sequence alignment/map format and SAMtools.;citation_volume=25;citation_year=2009;citation_pmid=19505943;citation_doi=10.1093/bioinformatics/btp352" />
<meta name="citation_reference" content="citation_journal_title=R News;citation_author=A. Liaw;citation_author=M. Wiener;citation_title=Classification and Regression by Random Forest.;citation_pages=18-22;citation_volume=2;citation_year=2002;citation_doi=10.1177/154405910408300516" />
<meta name="citation_reference" content="citation_journal_title=Nat. Rev. Genet.;citation_author=M. W. Libbrecht;citation_author=W. S. Noble;citation_title=Machine learning applications in genetics and genomics.;citation_pages=321-332;citation_volume=16;citation_year=2015;citation_pmid=25948244;citation_doi=10.1038/nrg3920" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=M. G. Nelson;citation_author=R. S. Linheiro;citation_author=C. M. Bergman;citation_title=McClintock: An Integrated Pipeline for Detecting Transposable Element Insertions in Whole-Genome Shotgun Sequencing Data.;citation_volume=7;citation_year=2017;citation_pmid=28637810;citation_doi=10.1534/g3.117.043893" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol. Resour.;citation_author=N. Palmieri;citation_author=V. Nolte;citation_author=J. Chen;citation_author=C. Schlötterer;citation_title=Genome assembly and annotation of a Drosophila simulans strain from Madagascar.;citation_pages=372-381;citation_volume=15;citation_year=2014;citation_pmid=24961367;citation_doi=10.1111/1755-0998.12297" />
<meta name="citation_reference" content="citation_journal_title=Machine Learning in Python.;citation_author=F. Pedregosa;citation_author=R. Weiss;citation_author=M. Brucher;citation_title=Scikit-learn;citation_pages=2825-2830;citation_volume=12;citation_year=2011" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. R. Quinlan;citation_author=I. M. Hall;citation_title=BEDTools: a flexible suite of utilities for comparing genomic features.;citation_volume=26;citation_year=2010;citation_pmid=20110278;citation_doi=10.1093/bioinformatics/btq033" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=R. Rahman;citation_author=G.-W. Chirn;citation_author=A. Kanodia;citation_author=Y. A. Sytnikova;citation_author=B. Brembs;citation_title=Unique transposon landscapes are pervasive across Drosophila melanogaster genomes.;citation_volume=43;citation_year=2015;citation_pmid=26578579;citation_doi=10.1093/nar/gkv1193" />
<meta name="citation_reference" content="citation_journal_title=BMC evolutionary biology [electronic resource];citation_journal_abbrev=BMC Evol Biol;citation_author=S. Rastogi;citation_author=D. Liberles;citation_title=Subfunctionalization of duplicated genes as a transition state to neofunctionalization.;citation_pages=28-28;citation_volume=5;citation_year=2005;citation_pmid=15831095;citation_doi=10.1186/1471-2148-5-28" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=T. Rausch;citation_author=T. Zichner;citation_author=A. Schlattl;citation_author=A. M. Stutz;citation_author=V. Benes;citation_title=DELLY: structural variant discovery by integrated paired-end and split-read analysis.;citation_volume=28;citation_year=2012;citation_pmid=22962449;citation_doi=10.1093/bioinformatics/bts378" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=R. Redon;citation_author=S. Ishikawa;citation_author=K. R. Fitch;citation_author=L. Feuk;citation_author=G. H. Perry;citation_title=Global variation in copy number in the human genome.;citation_pages=444-454;citation_volume=444;citation_year=2006;citation_issue=7118;citation_pmid=17122850;citation_doi=10.1038/nature05329" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=J. T. Robinson;citation_author=H. Thorvaldsdottir;citation_author=W. W. Inckler;citation_author=M. Guttman;citation_author=E. S. Lander;citation_title=Integrative genomics viewer.;citation_pages=24-26;citation_volume=29;citation_year=2011;citation_doi=10.1038/nbt0111-24" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=R. L. Rogers;citation_author=J. M. Cridland;citation_author=L. Shao;citation_author=T. T. Hu;citation_author=P. Andolfatto;citation_title=Landscape of Standing Variation for Tandem Duplications in Drosophila yakuba and Drosophila simulans.;citation_volume=31;citation_year=2014;citation_pmid=24710518;citation_doi=10.1093/molbev/msu124" />
<meta name="citation_reference" content="citation_journal_title=Nat. Rev. Genet.;citation_author=C. Schlötterer;citation_author=R. Tobler;citation_author=R. Kofler;citation_author=V. Nolte;citation_title=Sequencing pools of individuals - mining genome-wide polymorphism data without big funding.;citation_pages=749-763;citation_volume=15;citation_year=2014;citation_pmid=25246196;citation_doi=10.1038/nrg3803" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=D. R. Schrider;citation_author=J. Ayroles;citation_author=D. R. Matute;citation_author=A. D. Kern;citation_title=Supervised machine learning reveals introgressed loci in the genomes of Drosophila simulans and D. sechellia.;citation_pages=e1007341;citation_volume=14;citation_year=2018;citation_doi=10.1371/journal.pgen.1007341" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=D. R. Schrider;citation_author=D. Houle;citation_author=M. Lynch;citation_author=M. W. Hahn;citation_title=Rates and genomic consequences of spontaneous mutational events in Drosophila melanogaster.;citation_volume=194;citation_year=2013;citation_pmid=23733788;citation_doi=10.1534/genetics.113.151670" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=D. R. Schrider;citation_author=A. D. Kern;citation_title=S/HIC: Robust Identification of Soft and Hard Sweeps Using Machine Learning.;citation_pages=e1005928;citation_volume=12;citation_year=2016;citation_pmid=26977894;citation_doi=10.1371/journal.pgen.1005928" />
<meta name="citation_reference" content="citation_journal_title=Trends Genet.;citation_author=D. R. Schrider;citation_author=A. D. Kern;citation_title=Supervised Machine Learning for Population Genetics: A New Paradigm.;citation_pages=301-312;citation_volume=34;citation_year=2018;citation_doi=10.1016/j.tig.2017.12.005" />
<meta name="citation_reference" content="citation_journal_title=PLOS Comput. Biol.;citation_author=S. Sheehan;citation_author=Y. S. Song;citation_title=Deep Learning for Population Genetic Inference.;citation_pages=e1004845;citation_volume=12;citation_year=2016;citation_doi=10.1371/journal.pcbi.1004845" />
<meta name="citation_reference" content="Smit, A. F. A., and R. Hubley, 2013–2015 RepeatMasker Open-4.0, pp. RepeatMasker." />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=A. H. Sturtevant;citation_title=The Bar Gene, a Duplication.;citation_pages=210;citation_volume=83;citation_year=1937" />
<meta name="citation_reference" content="citation_title=R: A Language and Environment for Statistical Computing, pp;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Curr. Biol.;citation_author=R. L. Unckless;citation_author=V. M. Howick;citation_author=B. P. Lazzaro;citation_title=Convergent Balancing Selection on an Antimicrobial Peptide in Drosophila.;citation_pages=257-262;citation_volume=26;citation_year=2016;citation_pmid=26776733;citation_doi=10.1016/j.cub.2015.11.063" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=K. Ye;citation_author=M. H. Schulz;citation_author=Q. Long;citation_author=R. Apweiler;citation_author=Z. Ning;citation_title=Pindel: a pattern growth approach to detect break points of large deletions and medium sized insertions from paired-end short reads.;citation_volume=25;citation_year=2009;citation_pmid=19561018;citation_doi=10.1093/bioinformatics/btp394" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=T. Zichner;citation_author=D. A. Garfield;citation_author=T. Rausch;citation_author=A. M. Stutz;citation_author=E. Cannavo;citation_title=Impact of genomic structural variation in Drosophila melanogaster based on population-scale sequencing.;citation_volume=23;citation_year=2013;citation_pmid=23222910;citation_doi=10.1101/gr.142646.112" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="A Deep Learning Approach for Detecting Copy Number Variation in Next-Generation Sequencing Data" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11.cover-source.jpg" />
<meta name="twitter:description" content="Copy number variants (CNV) are associated with phenotypic variation in several species. However, properly detecting changes in copy numbers of sequences remains a difficult problem, especially in lower quality or lower coverage next-generation sequencing data. Here, inspired by recent applications of machine learning in genomics, we describe a method to detect duplications and deletions in short-read sequencing data. In low coverage data, machine learning appears to be more powerful in the detection of CNVs than the gold-standard methods of coverage estimation alone, and of equal power in high coverage data. We also demonstrate how replicating training sets allows a more precise detection of CNVs, even identifying novel CNVs in two genomes previously surveyed thoroughly for CNVs using long read data." />
<meta name="og-title" property="og:title" content="A Deep Learning Approach for Detecting Copy Number Variation in Next-Generation Sequencing Data" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/9/11/3575" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Copy number variants (CNV) are associated with phenotypic variation in several species. However, properly detecting changes in copy numbers of sequences remains a difficult problem, especially in lower quality or lower coverage next-generation sequencing data. Here, inspired by recent applications of machine learning in genomics, we describe a method to detect duplications and deletions in short-read sequencing data. In low coverage data, machine learning appears to be more powerful in the detection of CNVs than the gold-standard methods of coverage estimation alone, and of equal power in high coverage data. We also demonstrate how replicating training sets allows a more precise detection of CNVs, even identifying novel CNVs in two genomes previously surveyed thoroughly for CNVs using long read data." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11.cover-source.jpg" />
    <title>A Deep Learning Approach for Detecting Copy Number Variation in Next-Generation Sequencing Data | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"taLbEwYWqmS3v_r9d342CAtlMeOyyLNbiA1at0MIUxs","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"494621","apath":"\/ggg\/9\/11\/3575.atom","pisa":"ggg;9\/11\/3575","ac":{"\/ggg\/9\/11\/3575.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/9\/11\/3575.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;9\/11\/3575"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;9\/11\/3575"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/9\/11\/3575":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022RHDFm8UC5qyrotq83cUS28SrR4PXYX\\\\\\\/++v8d4g\\\\\\\/e1to\\\\\\\/PNB1AK5cvYGh6QHWaeRkeo5qgitueirvzbKAb0YW+ADPjaWekAQOOF8ZoLYqx+nBNhnHutbk9lhGZsIycEot8TLIbdmCEIA2dVVKt0fgHyj4P2C5n3H8JJ\\\\\\\/sZXQa5mvq+dIGVA22gXub8Ru\\\\\\\/73Z9C6s776bgdchLUepFkjmP81hXeWQdwcFSO8yPXVqGbPSSMTbwM5uaYUKO7ILPsYkTcoB5aPpWUfbFlMvrrhDwvSgZqbNp4fyRwRDRPJ+ubikRXO9\\\\\\\/gYyaNGCdWaVch3cqwQeUTuHsHuSvfDFaScSIta3daMZB5PsdLyQ2T4aG\\\\\\\/7qR2vpA6Bwt3bkbLO\\\\\\\/JFSIAZ\\\\\\\/c75Gq9lZrAh6Q3LPjw52blWiSQiX4UKVQIh2W2U\\\\\\\/fbkpZuWR6UcSyUw0Hge7lxPXCq4j7zDAnlG6ivPkragJOoei45JPAZyfUdFAZNbu4PzAIGuB1seZGQlYFsHuWi0soelz67I7adEsdLmYc4uYZMj89dcCBOMIHXAya\\\\\\\/9RzVYvMqLE1OxgqY8rZjaBy+I+KvDnYKzNgn7NDfz0meC\\\\\\\/CSzOcjLK1MiJD\\\\\\\/\\\\\\\/7lJ\\\\\\\/zF\\\\\\\/IJafVNEXngY7qWvlgDdjp6DHPLUN\\\\\\\/JDFKiUhMCM20tjgW5mV3VKw44dFRH9LyLPSlaTVXLEp\\\\\\\/wujF0U4XK7Qkm8gmg38TPmgukEKWefIEEEYJpPE76SumgSS3e2R6DS9sfw6ZpyHF1r6rD5A3M4TfijHYeiTrYRyF66S6jsigoWInJqGH27SpLSjJApXXoKejIoujZf3wfYY0HPx+BF8uVwEK0nOEav4xYQL+ze0GnLRnOQ0d2Qx7njsToI0wsytynMvo1L9hjXu5x\\\\\\\/pdDO5Fb7l0yZi52jnGGNOyYnn4KyQYdZRq6GSktcGqgBCfVD07KRHSDnvgh5cemE+hMRzKR52p4fgUfGlQldRweSpdqPuSvyq8fXSYZcPWS4rm5WPGeQkjvMBpYBVepjvH4lDJe8GBDM3Nt1j4h9ldGBAXU0\\\\\\\/fMDs0kq4IUNK3c6TmtqY3fnzjWeaotBHD5NqcJ60edrPIhP9258EHYkI+W5G\\\\\\\/YHkbtN52iat4pgcgCCnJttgp1uWw5wWay\\\\\\\/YLNtWypVFtNjXuhRuQlB5xJSqMqYqm1u0Rvnd0K1yzMtwPxEhpTRfY95ElS5k991\\\\\\\/QoNq55C+2jKRh6iGAy44EkTbrYo7eaiGEHyYWzFUrV6ZWeNIA6\\\\\\\/uGNWAvrU7RKdZtSd1IQkC\\\u0022,\\\u0022iv\\\u0022:\\\u0022lvd5JUHiLylFjI02DCjqsw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022d94654e52e4106874ea82db4a948590913b0566c76d3e542b593485fce2bb2d3\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022LVtT+HsYH36\\\\\\\/lFRrNH69qp+qc6hJweUreecDd+giqMLl+3hP6\\\\\\\/c6pC0h3elqBgtOAh7+T\\\\\\\/qZ1pXjA4foS+ADdRfnCeRllP0n0HGRAYzt6t5EVHld5XiippBRzWkMIzUB15iQLZeoo\\\\\\\/x7YGD1tSTLIb3SVvUsF2Xt1XrLNHjU8fX9EQIZZPWJgjJ+m9fcrrFbQ6\\\\\\\/5eTMXSDSAILAksn56R0pf7dYc\\\\\\\/l9Jj0rwd5EfakTNw8LMcRCpa6Pr+ZVfHLSvo6roqm0+pTTaFTfgEHi8QvYZ\\\\\\\/\\\\\\\/gg6E6sOkN8NNLRME2bNOXogT\\\\\\\/L3KoRFmVuL7hqIt1sumOfd\\\\\\\/gLFfNC38IIJ5lOySSs1KZaidCTEn3Vzh8BCS2mstkUxTV6ccvRz90URQho8ghAPWoAGPGQympVibgzUT806\\\\\\\/anWkkZ4MqQURlcFxFw1TbQukHxH6+fzIlGklslNExGD320fWo3FXP+GvUhYYJbKk0baOuoS+W1IDlgLKnuUTVFNtss1KKjoOaM1QLl5ZVrgJ1lAaQYU8x9Fr6GVCNpnG5ViXYa85AvP4OKDQVbZ7nLeTLxjrnLNX\\\\\\\/7gGhV+yvVl2FolBXOWuKmGFSERMcYFFadxaiUI\\\\\\\/QnHrF4BI8wnXy0ps2AzGNLUKIcpsd6+240U+VBZb6xBNoJSFZI8GOD4tHrO2Zc7faxQ1IEItGvZ9C14aPnKdvsPrzSImp7bsljNT44XH1sJuu8cDX7\\\\\\\/p\\\\\\\/6sy95bi8UI79FZGBlTh2UtVBWYwTLDRQmTwP46FgAC7H3+9HdOOJPOMudFc9BB8WDZTehWpUKhpXCQDGWELDcjkHnUWljOFuPR5gWyuD\\\\\\\/4f37RKjp0K9DPcJ9pF7XggbVB3evsGPoq5\\\\\\\/B4s+Se7phS5i8gaQ2iePTHsfZbNjhuDNf9GFAgyqe97XzXynekvd5PkIuH5xWIaSxxiMlkNkAk\\\\\\\/7QPcnFKbP\\\\\\\/h\\\\\\\/BghWB1Hlf1LG+xV6CsXlw7UbeedYr9MDecU2qXRFSU435V7PNpnNm41pg9d+3jS9mrBXEMYOZeiRsq7ZVDpte2+dfWwqN0XW8jUWdX1Z8UW\\\\\\\/jfj9OXwAS4+KtpHLTaGrHCfp3KYvvSMusoyEhQxBIgYT4NSJEikneiRtAyMWBZnCAhH+hBs6BUsffGN1VcnomN2uwVF5IqtPLpz77aEsTXqgfcl2gBrN6tTMGY1\\\\\\\/3QHwNN+D1cGrR9MX4Boa2VBfypVeVTAKMhJwN4Z9whWnhFbccf5XEBGAX9giWXi4Vrew1fIYPl1a6fjvzG3Jk6ljXTzIiauuuY+lzFY0XOs5t1BvXyKVy6iUdvr2rHwTTEkH0atHQkociULJKX+7fQ89UvlRhbUt+tNhWV4bI4SRgq2gQEmQ56+siosVRouBjR6pWtKbDZoMU1U8ouT8km3Or+BPDPUmh5a+RieXzMtphty3upnxH2DLy+Zqsu5xajncj\\\\\\\/Thu3HBZp8x6jfvOfhwjhr5HcwhLq3PeD5EMcsSs0Keo+G96ov\\\\\\\/dwncAGzEzdq0p980UqrPiYKGCfpA1hoXxq+4k0hK2qeZ3BCFl+djfh4hlhA3UrDUr1eyELgMTPEPbWwEWum3VF\\\\\\\/M8t\\\\\\\/oOhui8uXp4GgVdRK8qXCtuBcuZatiJePRqrgbJF7tm+CwUOO8WwRkvSJFnqJ5pLcuks3D6xHFdO8yCuPGjCxgIMpYiYPtXUSRyu33H5EyL0T1r6HQRIHKeqjvhZ6lWLFCSTAPGYgMDJAcD34MqPalfkvl02d+ZhiX4QR88KxzC6mmKPL\\\\\\\/5M\\\\\\\/VvjaPz0pFLkgahCX+8WTzLJysaSphhQZR2++79gi09lTRRAvuKdJFFVr0l1\\\\\\\/cXkdBJICIBmIRpcArxDCKYPOXdFw75axA1Q56c6YCHBsRhOFQkTeVGo3weGQtpn5vb\\\\\\\/r3\\\\\\\/dORoTFTSBCeQfW67P4grKbaK46hbJIj0UaXh4GSf+zPa3gD2vQ2CR47HYMwj3xFg0HB9NSMRZ+SoWjRZCsJL7brNGHE+unFM+dXSi9Rjx4se+e6WJ+NYRomnhf1SlThSztH1\\\\\\\/NEMNZPJQZB38fJqLhJGNUGlO\\\\\\\/gTo80Lrhb7aaggkP8rGPyM9baUaAOzsgUb+CX\\\\\\\/TM1WhTWigJFHWr5OovM1tCtj73SIdGEczidsO7yV+CuQ+TisP++Ex0hUBw+G9tmXAXq4K4ahtAyY1kRuXQmuAVAfX+ge7CsHDzjLNimykS09LCLD1NdGjoJerO0d9vVXGJWlwYkim21H5AJ9i+Qf7OZkXF5v5ZiZ55SRMyzgd9DTCnIhrOejC+7nHFxZD\\\\\\\/IQ3r+wFjcORw1Fj5DV409UEIjjgthMAF18jrbUX5Qbn+3vuXRxnPIEaAPb9uf9Hym2E9+sgCvSVogYut4pXC7hOZeEnhFttJNWetI1n3Y0ISXkqYEtSOgRsnX0fukNrZdUr3vV9nsidSec3yEDZvv9yraGl7Xdy\\\\\\\/HT8nAMm3QAA7WRUvs5Hx+ZHfQPdAuK1DOdNaj8XFSzGenxPQBFw9anvUn\\\\\\\/M9+GYXDCKBl9JcnnMZtwr9AYVSqfTuO\\\\\\\/rsjzEpaeB1gBYwGS3umriz8cXJpTureo+yhW3PUM6H58XDVJPVYFBjFnBsv4RhtiFyi9ba5fWMwMqBupvLsHfPbs1ao+3hR5BBMS5dusY9ZM2UWEmip+TYVzB16DkLxvQH12ub5B1ogc39mDKcbE9utbel4wUkQutq0lIoGvpJzVuXuSilexGvffhDBrFM9U7GpDlPaE+IjAUDxRlqWK3iecBK0761NeDr3WY6Q1fNILu7Ylfd3BG9WBuy5ltXj5+TgZYRnMokdWupTluU9Nj9Xe4taOwl0v4emK1jffERP8VPa9jhL8IC67kUdeMo\\\\\\\/EvAtVu2xaL5s5wRilHu3IgdIkFQc\\\\\\\/zMshXZeliWfbNWZKgGnOuV5KyR5HzVmaK54RNL5fWNWRUhBSfDgSiEwCQCeWVHQiYIr2niBTABX\\\\\\\/UCBHm0+wWohBYTGKRXN+WaCsD14VcqZ9qWAqdsW3yj4OFR51IDeTm1wf5YwD5p70WZkRKFxJpbQy+3C0kAS3mVHspPP99b48EmIImBXWR7fZJu6bVJpIGrdMmyBRzW41Q7ZCyr8p9x\\\\\\\/OAvKazAGrPBJtMkzAOEFCA5XQ3QqOQ5fpo8OfKXlcLjX0PiTPNIIioN8Y5uK16EUSAPct1lJDoJlnZZCIZDAkECStQ8klqWXxCJCgETUX6YJan7oVr3fXM17kGOlEEaMPucb+DenULJYn3jxz00Tnx7Hhz8Kuxjep1cghDbZ3gBNK4tG38MG3tAelFK+Xbbi+LWIHVaylb2vQuW1LcdJp90EbU2A9gry5yaCj16mqyZterQhb3xM+Rf8Kd0Z+AEKCqyNePF3zfpw4VUFRJ5GrBUXQLo9pSELhFhcIj68oxAcozqU9OiTKPPRLq4cBXLsFHSIi9s0in21KRzJAOuET46xhKhUvlCbTft1FSpD+WOtTva\\\\\\\/jmy2ZXfZ+BlS5xSdEmH0+xMVwWATVRnGSgIxrGNQJTklp\\\\\\\/8NrK2M+03UcYkRq+1YDTnOeFf4PhP01M9\\\\\\\/Ao7dWj1yLen4SWHtI\\\\\\\/yNjAPLCfEG\\\\\\\/hj+uWH8z8I+CSHGIFKvkuvnl1p0JHh7HvbPSDHeJRB26\\\\\\\/2IqoqtMN67RoEQ88qCpw1XG3KFM9VQP7LpsOS\\\\\\\/gbFqVF1n7UkKbsqUdtOFFCQ2+odIwl+m+iPmeShx96oMM8g1CUzDRjrv9U6IZyZTrTgcOXcrf9tPcNV0qaV89K\\\\\\\/qNAdcF6DO8MBHfcdslpZWqbJuj4y1N+TTs2srEJw8LcEUUL38YS6GhOunzJXJ43UJdJYQEAEzNC57A89\\\\\\\/rjSdnw8ayGvuWXT61uaP3qbTDeeWYJlr3p5GFTNmJvhnvawq6gk1p7AlomYlxidvyiWZS4+in+fhs73pKTtN0FBu8gLsHAv3o8obomCSKFNbsh9VW7Ue\\\\\\\/VlPzyRby4hmyiQ8z2stwg5uusA1o9sR0glTeHv4bVCHvwVP\\\\\\\/919yBwm0jKXWLnp0A2TgYARZkQ8+ss6I8QQGdatxwHL\\\\\\\/gF1hp3Kuyj\\\\\\\/tKPtf+EkYfXipZUFKR\\\\\\\/5EVoIfOGBQ1KgDg5LCZveE91PoeYjSYHOSiwTwImkgULss6vC4PJRp\\\\\\\/Dnspy695KZClLENhjzHhhSq42YsZcCU9yi9h+7gKwRnz36ZlGOzrk426txBKramKV\\\\\\\/hBfjj6s608\\\\\\\/L8G5nd8e0yUHZVXYhuinwr6VUl1hi7tQrowfmMQypOPUddpzq\\\\\\\/nqfFxPOFk7AgPa0EUfR3r9wHLJc5f385Ngv11tKGU9+3OdbXIfxWAN2IvLwmHBa1J\\\\\\\/tZz9tGwmA6iNdryePfKsi4xKVt40NGcgzBNG2Dc76EoUL9ZbJ\\\\\\\/G3\\\\\\\/cZvU+\\\\\\\/64+pAm5HkSa7xZJZmDzwF7+Yr4N5SDQ6BArNUIGzoRgukM7CekI\\\\\\\/uBv0bsU9hpIxWa+fzHPyxXO0DKeCGo+C3PEHLkdC8dipqbgXgQRSv5O2bw1trfa\\\\\\\/N4qusPYIHvI3ggKGR3CBm5MdHUaKYGGHb2p2pzFK7vsZVrXWDySCuHuMTLaDlvTvxQbF3EAoMfeW4yD+IGu5XC0o97oMhE8B35ydiIr+iHUkNobA96A6bzx39FuEN5SEAK2yUgAC+X181WM9q33zciQ0+6DXOIeKfEOrwNUs8SRzfCC7knFC4Ji2QH2Cnt9qJ6OUwWKGqsyAyitg1FEKW6EPgJJeEpqcXFod\\\\\\\/U1cq3Ir7d2UJaNSw1SUnZsjJnd10LcomA6Ck0ZiduC7Qq8ThkO1+8cXiGfg5\\\\\\\/JLY1tO+B2r67vgsl6qj6IAZgKys8vCjxFeEeZZqG62ceTgYIG1k+o1WBOvDmSX50Rgmj+5QH\\\\\\\/qhfKpFflZtB0lAo1RoK\\\\\\\/D90+PyhyYMPa3ryii+DJh7J8vv5wHfgxoWvrilnQorEJuBxbpzNJ7Xe27Bn98bIb0nLeTpIBm63Rw\\\\\\\/dJhYaHLukYSi5M+coKlgI3jxzk945PbY8uvet281QZafQIj9TpaTwnPZulCtlFuki7zZ4FW4j11p8EQ8YFpl16eQtagbPY4oKURvgxuFHYWguMzHvhcY3Yzwm7DrNMJof3iL2M5IYTStL5qENqmUgN03TBY+33a3l3mCcACKyrGXtUPe202B72DbOdcRf7pWo0n56nK3z+PS92F\\\\\\\/4rmi66QmJOqpP4xxdk9JaqrkhwOqiHPGHTnuccnhlrOhmbgaVTcHcKTxXmjLzTWJkQ1aXS83+MCKQYQCMDXL+3tgS\\\\\\\/udHo1Lygg681NMqep61USXrfL1dGVicBnjGt43p956pWh\\\\\\\/ZCLd15mf14gIJn2ki40TqeGg5v3BZwIP94cSjceqnHw5ZKuIVBaynCs2VpECo5ICIjmzWzbYUme7lQkHIRQllqr+nhasufyc45GuvTVF+kirNPqotAcBWbAbW3UnRk8qW8BKKQSlWtTDHnICYAGuNpSROGt\\\\\\\/zRt6n\\\\\\\/5xehOlvZawW9EkqXNE+nIq+J\\\\\\\/3V94CcdXGYsVlpl9xePUOJRQpK2oIevgrqEEALDLHKAUbLDtbhFLKeZwGdXjjqEJqw94Xq3lXtU8e8d7hnPSTkXxmIlLi3ZaOvcfJsINJvf9h67HG5jFR3n9o7GsKAHWQbg36lYt62K6XogBDexGYFioU4lPyU3JWdjN4TxZH+5JcTltnscueFvZUbxAIgGHmclwl9xOiDBK+w30W511e7IvaPhUM5p3ojwKFbLskF3gfbptLDZSCuuzpPq2OrPCEuKhxc8oJcPOup72FpUGBFx+ofkMfi0b09PYX9ZTx7m9KJcYIypmBUe4jUIwxs9yiQkQiWjXnljrW0mqmbumrceuxrSkmS2oH+BONvB7RAGMXwKhluQuhJG6n0IGK7cyFRSO5TZl5ljRFNmiAOnLD5zn5f3IpEiimuOJC3Oe3sEmGp8NANSvpPG3RRXS+m9n\\\\\\\/9DgUbcGDICrl9ZI07eXrAyTZCXSPeivas8Xt4fUct9fVT17SJL3IgAc8Gj\\\\\\\/Zj5FtzY1etuHhIiT7nwzN4S5mXi8iWcSR8G+E0YIm5JYAABIkCGFKsNbMV2Gby8NItU2oqFrbosZ2+jpxd1U3PVzv3FvJ7VyPoKjZTzOPFpPkKXvO2qdNJyt4DCSoA57ne9ri4+DGZ3QF3Z4VUlPCeAVp566Le0q63UngCEksGfdMOks9fJCrK1M4+LU5TuqJM6TEYC6UncCnGkYCHw97xBZN+dSyHVShPNPPQchYVhx4yOURH+TV8\\\\\\\/3kPTaKZQPqNoGDCLyMrltbrK\\\\\\\/rBVR4KtGQdXwVvmUt+ZzgIaVc8xvywM1JAO3tVRgQJZHpZydSJkV9PVc+dAOpjKvbTC8\\\\\\\/38KZSxQb+jpw0E24EsN7dJ7I\\\\\\\/DisPxEteOReCGi7E\\\\\\\/LeKT8WxbntxsWrqF2bS8j9X4sCSeh\\\\\\\/DbqIw1qEW4Z44UYMUJEqk17GnZ5EJlf3VWmo0VWP9HPJogHDRzqVv8\\\\\\\/N+iywcKtwhHnZSf423lVFOHrmD9a9Lyipc\\\\\\\/bjKw3mHqjva7O+PRZTYPOlqVH28Dj9zXxGiG91nViqeARmsFsX1j7+W6qvx27qTxB0GcQfvZ5KlmkLs5oRYWxQl2XQyZHNZKOn06rtVxxiuLbcobWmU25bLu+0T9E91UYpCEqSBG\\\\\\\/gbqkIh7DbIMXKtmI\\\\\\\/Ibeg3UZHAeAYZcVNDeB22KIrKBfnZoM3gZzdll5wB7WYOcLw6o1PMbzYo+\\\\\\\/\\\\\\\/q40R0m2OmRkCelLpYLSGWtA38v2xRMrt5hFpP3MPixNnDaeUQuz1ZZzPu0bRaatK8FW8bRWdzOYKyyF3SD\\\\\\\/rF9iiH3Ea+Dzvha5VC57sgE+uUl4i9W9D+FKXX5jzQwuDlvUBMlARSvG\\\\\\\/u2MtnkMqZ9ecFnQA3MiwiL5uZcV5MZ3bBKR1FAr\\\\\\\/kkLYyBmgW353A6cLwRsblGol4jYoIQIAghSZdMCOgH2hOyMiy6CNUEO9lUP5zjxGxVJ062Fu\\\\\\\/flBAomHZNmXzOkuolT58UxNEgIxvim2Nfx8VCHy1IsicvWYbAI6Vu7WiHv7sDzdukgPjmgDCCHEELSGCjeKqHex\\\\\\\/mt9m5QgqyQY18PfOHkT0BDxzfRgJEuWjiBNYDIxhfuGK33JgIylJezVpjJFwmXmATM5R8TAzOYaQfrXD+xTN\\\\\\\/CcMV3yz0+hcEcmqalwWJAKZriSqHHC9dhL3woisH3gQaCYQND+wH74MB5lZMJqlw\\\\\\\/6eJ4HMkqSmDKjdZECJYTvFV8cKcWOMimsoJPx52pW+h\\\\\\\/KQadXebZRgkfCQJSel+Tq+nBJHNkoa+HGcxoGD8PQVgR5ZqoWGI+4jkESpY2kq604YFwCIN8b9iTp\\\\\\\/LBTfbgdbrqtDp3O73mGvD7p3V21yUQZZHtlWLlujJHyzv969O2vBOvnRv9OdV82S5rrhgXImzOl976Hiv11PMg2\\\\\\\/xsT3zkByxEwBp9zTUZfGhgtrb8pMAxB6Lmb8mADJfrYvAqhZ56qQIiV7xuwTgHd5MT5kJ1bZdNst5EtFfTaVtAuXWcMRjYSEjI45FF+Pofxz2U9LWqzE71l3HiyWDFtC3\\\\\\\/l00wZBo9+8E0yHEDjIb1x1a5ZsWS4RDPiPzUgqmjn2xWmDvnrMTVksXxKXk8DHyD921hgPGWLogMvdnVCOSprGvB7dRnzMH7tzPykONL+rTDt8muJUWDoKulpjpfyjh1vKFL3VRQEMXd781M7yjZbjxr1RXfT7z0TLLX+dtYYuwQbMPyH8HJtI0ymUeA\\\\\\\/P9U\\\\\\\/vCwjm2IRJhgyNulN4yANcSiHeOP2gi\\\\\\\/sIditKNoyWcXzMMFyglcW4RrJALl2K8UcqxJErTNCbYZFnUNhDPonH+R0Zq0GuuTfDnNKV0v+Loh+ZBIAXpU+bhyrw+FXaZVnsvdUyRYvWUunXADVh1JSTs5Z5zEuKgQQRkPslH7NNfnnyYJe2kCMxmxhEGech8wdOx8k7ZwAep5GtaNjLexrRiJ9ag4wyrubTnUuG8ohGdO90llHt9Y6SCWD7wCahCOpcXN8VtrmPZa9O5BFnGLwAAfzygWF3mZoo8Tgt1zoniEqfGNbr38h6nUwWXTL\\\\\\\/mply1+DeWZAWDcAkQxW3CHWm4kSoiKVf8Gn6PsYdfUNxBgJ230aGgrw8WlWZ+IqlAiPJ\\\\\\\/iFtJ4u1rREa2QUcV1iNURdXxXPHpSJy9MKKH6PZ+qJkS+N2vrRmqGSgVZiwEEfr+AmyMMok55sF27GNQiM45sCbzPy7aW6YLmpZ9VatSIFi3owjRL17my8ZZCDBwPE7MapS6aWW\\\\\\\/thuGLffyox4vtQsCtPCaU4ggQPjlOISZIJ3C+N7AZzbcSxggNUbmp+SqunXZm0WGqtvhEIQ+t4eCnCYBePx5V0YvTUz\\\\\\\/CibpXhcVHrPMJF9UlIBzJDBh3rpUq2xx4Z8PWpz9sMFlbGX3rTvohFnX1RQ3sHvorSlwWYt3yCr1dHD\\\\\\\/RjJQ\\\\\\\/xL6DXfqOGi7ywtvbACn5C97MYvYTcMG2t5SJd1mSlr+WssebBG34xSsCuUoeOs0PNU8otfONcr3UjgUFluQJP7uIQ5r66AoiHD5r4q49wOKJ8HXY9d5B4jBJt2OzjOgdY0KlchhrhVinB4\\\\\\\/2KuoSyN8D7Hw7nyW92wqEFmsmF5ZJ2GQkZ1MC6Ypw+6Mid5L+4wbqZhb+MKOhU7LO\\\\\\\/miopH5Xx7YAc2gZ1ZgIUAehnd\\\\\\\/SfAj\\\\\\\/pa7kyBpjxMXb4nxZUdaOEXILsTAMhH0XWp2AZI8ft2XTZU2n6SEFkmovnT31owaAyNVhm3fC5R+fPKgWJcyYFVQ6mD9YTbOf9FSUaVr4MoUuMATb75a+XiDy63Xcvf3bPeHyK7butRhAeqV4INEFYX5iAtj3rajUrEcoYlIqGRhl9EcsRx+AJ1jVCpQsQjXxKlXIecD2ZQx\\\\\\\/ARcjfAIe9Xh1i\\\\\\\/JmUSNN\\\\\\\/7G5si7vwv1s25mA1OmzGNLHrpIp+zqO53OTI4IrXuuFJrUF6o0+ZQm3MrHYMbsZQOhY2DH5oPXvD2+5C96BEsYQ4\\\\\\\/trkki3rcdWV3hXMo5xxii\\\\\\\/pYeTcsMX6p\\\\\\\/P76mQjQekxxn8vIdi0R\\\\\\\/mxow5MycxzG1HxGshi8HA9OfJJNrSyP0DM5FfGP2BExU7wlwgbCZLQmCel3laEZRLYb7DPWhXPqwUdZu86G6eOJWN6GMg69ZdiepTF7OsGt0K5crLaYpcJqS6ac70bPVEPy2Hior5P6bqw4U+ROdB0sPZiZgbkKH9nf1VtfYvd4lhI6et3ZPl9aUssJlhqClk+9HJpBPH5PnjL1AsjkbXZNi\\\\\\\/HF5aPOetQaQ+GwCv+NwqkpMf7qWsy7km+\\\\\\\/HIUSrjE8PSajxKbKed+\\\\\\\/b0vv5ubNSMBQt0L9LhNtJ4zLWmNlYgIl69yh6xnGRFCtQAjBUO+3TWu4p0Q+GXAYugv+mLlvDoz4X0rNSFPXEhESTp9LlViyX5cK5u7yImsvh2yqCiyRLCpj7+V0NLB6fPjDsb8guhfAiSEq3RJYc4Rg93xB9pkeQF4jXJx7LsCTN1Umg2bSn251YZMnPj0psiSPGNUXWmXyslv\\\\\\\/fA9Zc+j6rQn89lm8eHDLVUxUqF332IU0aFsAyR7AseiAwElAzrB8enuEj8p8mqHhm7km47r633Ub1yQeq81mrRQIJpx69dQcOLq9Jqzx2xhHIi3OXwDC1NyVDDgWHFQZd2GJcCuMC2smDwL3zrAidLMrspZansf6buNc\\\\\\\/gg1rywPw\\\\\\\/Vkit0pUk+eP+9pG1P6Mkbh4dJfMLqSYwocOYSy3tfKuf2UTo65cJXnS2dg\\\\\\\/NIVARBWW6idtb8dJZ9pJKugRVvUWQZyn63iHAkVOpAlR0TMV3GPSR7xNT4zipYw5vUea57u6GyNkQ9OLLzRV1mAw\\\\\\\/VkUl9oP+nVyrFs\\\\\\\/+vFANXtoPmjLc3pLuBjoMoDynzHTF5xi3NJIOh\\\\\\\/yfYK71gL+5ruawasnlFhtNRJh\\\\\\\/qKp97FzTRXYssuqxe4MSsLdlQaKhRQblZDLQyguJd6C2zCqMsfyyM4IQzxfkBOy97fBhYsKj9RZ3eMGKFkvG9e3fe4gBh8vqHNwOwpSdPzdoZCIIdp77qAIiDK5t4INlhhaJgEhU+jqW4luFOzEAus1iNJ+Qnl2Zw1JV12Ze0EizaCCiJ1sS6osGBQw+RYGgtUWAN+S2Okd+VvgY6CbC02giPx66rASCqUluzJiRX1F0wQnQfR64z2vuIRAIVGKxrcQ+WBDowZ8E4vAxpRYiSwsbCX87zs8bj+HHGmfb0wkUsLKgjxJ\\\\\\\/pLyC8N6G0mnlOJki7c4+qJhSM9FojbDPNS1YHeQYv+Z4R+yN3aH3V8TWzqxkhV8zL3ZVkHnyziP99NK69FdL31QZqMgKEcHRyeXiKZ\\\\\\\/SxA4aCZgeRnb0EOaCJW2tQDeqOqmmtxjf63hmzrngyLEFSgUUnNJLp3ZPqPvBowdUEh0k702D9RVbMM\\\\\\\/cFRqQM1T+67Ln\\\\\\\/IRDzyYQNYGqtUnwj1jTq0XJwmFn6VPHEW\\\\\\\/WqIcgznCqtHRDc1fcj5oCdDfetAjYnI9qQRdWqOrGX1OQ53VGWV3nURyzZfeh\\\\\\\/swqVEBV2VzjMgVYDcJcaKVfUBlJHpuHaAhIkYpiqYSe1URUvDh9oEr0P7wJUDS7n0O+0H98nhIXmYQapNI9J+YTmna3nmzDXbsyJr+p5ZHqWV3\\\\\\\/wm8fDxi9ZjrEyd65v76U8+gtDpzak1xlXSH5JyGrIGEQtqDxcfgDMboFO+M0wzM+hd5HUi\\\\\\\/t4z06gA7V0DqKe0QKCBEMBjXtGWhLBf2\\\\\\\/j0c\\\\\\\/BS0X88D6y2z0zyAu0sS2mUkHXSCJA6qvW8PBV8Vev6c0Kju\\\\\\\/4rfKsCOJi2QO9mtc2BVyDEWu2LqOCwjJjwcwIZy7z4rg7WVhKnT5BS\\\\\\\/l6YNg80e7cIi3CFXIZENvSnM+i6iipe\\\\\\\/b2CEo8ZKTKqY5keZC4BJNRxh7KhmUBTnlUnqYwBe8UBMOjAdGXOF5VJh3dxA0i47GAux1IQM2Xt+7eRdrTnadVe2hIU6Ah3JlkwntbWQZ3L7ZF\\\\\\\/2WoPHAOQcVJRf2bR1nfXMbdxsmKhwGxwptLFafo4sfvDDFVVzq3HoYQxUB5Cm9skfSeXPDUSYnjgXs7+nF7Hv3tlMFBzJES+tk91rWEDY+IOK2ga3s1DDGP2uyFEXCmlD2Gf8fw\\\\\\\/ME0MjliEz4jkvYl51mwhjN5Pigi+7wP2NcxKE51DBPgLmRYsr2FG\\\\\\\/\\\\\\\/x3NHglRgOXquvyvO3ztB87Vry1z1x37j1q\\\\\\\/3y7gQKVN5Z3SShEkDzY\\\\\\\/4HKYICFqD+JEjP7SS9QgfcRPIZAE1m0VgUz+rv33J1MASH6GpZW8QgQ1baqazKCPMLzCE2x5jqRsKT+9BY2sSIbRFMYeqBpFoIXwbtzCZDNC45jt1Fu\\\\\\\/4G9d\\\\\\\/kIl+a6FS88oNOPnIPCG3XiFCZsCYhNP7WU83kmI2OI6aCPGbLEBSXC8vR2Z15k77pBHlhqxP+4MsrYcZsRCSjoQNrNXsUEguKWPwNnDL6hNGbsucSsyTmAomSVjCOykxHbDnsr+HnTdbFnvausbtxuIAPC9qAadogh9eaQGo0U4sT5XtqgBVScZZvOdlwUcYkFDdl7YAk8\\\\\\\/4TuXTup2\\\\\\\/SaERai9v2rOdQvcB7cnaAHfEtF6s7HrloLhDg9kn4U3QYhtXJgRorpY0\\\\\\\/hDJJ2iAUFEpqtJZRV9vNjQBtM+d35QQz2Nkz30fx9\\\\\\\/9G5eOWzZem\\\\\\\/mi\\\\\\\/Ql2YKVv6krPI9plevPJgxfnUlNh9pvDHelzIJnipos0AS9eoKhtcM87NGNUi71O21q1pgRj+cuiFtZtUatV5az+eV+bCtow32aIeEL29bEMTrZvqVjJs13GIbNyPRdHF7YnjNXwEmJGFlFo9dz2gWikZP++UJeMHew5Gsq974TC7v9wJMpQu44\\\\\\\/ZlBK2XHtLYiW0Aoyi6ChCmXvKFJnZ67vbXj7lLCnHfP+VIaHnpZoXHX51SFqKy4jLvByfBiRvIl5FXw8mqBYep8uLIQRuylfrkcgf9ShSmBrY+oX44T3dg0VveOsioH+IZ5oa6sA8cmsT8Jnmr3O8jn5ifTPy1aISKkFJsiTvjgxeXW7Bxqi9bnnnKKbkD4IY7CXpDQG\\\\\\\/qyKqtNRt0wYZ7ySz5BtrVUEcEQeuZQEGSZT0Do3IS48tXGL7LIEb5lkxsOVErqsFlahsyqdAq1blfcLfrt7R0Y99U0dAHJPicfLUaMXGDoC6+9\\\\\\\/BkHcneVHkskuGxBM7M9osDqYLRy1pfzVqgx2UPOiJV3tuQgyAGDPFper0Sh1h1AnifCnAAjjApSKBrjfBNZi9hPxyRsAWD5VLmyX3MLjwfpWr+l\\\\\\\/KKPud0zXFEhKYA3ZFUjxJ1W4SUGw+F7wN2fNQEbTUuh4z6RghYA40mVvHQ3X+bqMPucovj9jJ5C6QEbaqk2RaJqTZHGSEXmgC0cGUrVjkzPjEeIt+8qxV31i62EkB+wizZwGzVz3wpdC26xGTASvDqJLTr\\\\\\\/sX+z8toLaep3llOZn5D4AMqC0bFkfp2MXoL0\\\\\\\/DzrCfxnfQrbgJVLLvg+3OqJo+2MMDlnGyQzvbFWyDXyHmmqP6d7oB6fD6KYEstMTpRX56l\\\\\\\/K+OVaKqoWKG8ftovdBBI9DTAWXbAA3IQD6gtIFscKnJAp8uNM\\\\\\\/5LtN7zlZXPKYzeKGtWTwHDE1cFuXf\\\\\\\/WXLKtU9H8t2xdArrpmWWF8jFjiXPR5A0j2awGEwPEEDikKAqQXndGkXEGrmSWT+R0HV27F7lv53s\\\\\\\/mbPeuvHGj8I18eh4kbSfPyIS\\\\\\\/6DIQYu9H2SKdgVeDNN6ubNkEo+6NjWYF\\\\\\\/s52UxflXvNk2HESCMcMhsGSeAXg6ILZr4\\\\\\\/cJPPpaVnP1XrupHlWlxrDhfqCymRaO1EI\\\\\\\/ESM\\\\\\\/F82youD08MQccIGBqsYP5iR5wgR0nvgYS9fIH3jdaSboogArb69QhKUqdXbu+F3DUEyg1dNkg+KTaxUFOCmHj7NYXVcFuQxBr+hPVEGWwbpaVIHGEtjFDJ2oaOzyaJ6JZbXs\\\\\\\/8f4SreD4m65wdrVjvw87QeW+XDI0r2LG4gZ\\\\\\\/E58yY27VlorKfQMXbZFuczU1eJnfFt7gKwZK5TOY1fm36fzGp79KAK4NbGVKSUYavr3NfNhf2WrlIYDYzwbBsMTQ+KYxvEfDHt1ttu\\\\\\\/0glPYCvv\\\\\\\/KZbBrNzIpXKsVvIH3iAg45mi3t0nh\\\\\\\/5nqDH3Jy\\\\\\\/6E4RUHNsrUWEj32Njgza6r1Bxyq6GNqj6CC9CbjCkNmIPDAKUmWsAtDRxm0tcL\\\\\\\/4cOis4nO3UqpMjB5VoUbJ+7arKEJbE8UsUB\\\\\\\/Fvkp\\\\\\\/q5HV7u1p6bsA8dr21mjeTNddiXI\\\\\\\/Zx4vP0SOQzty2jjKcZz89AuUXt7ItQTdkvsB714WK2cMd+fU+tDR+YUmIxdQWXydB1rQ3IPlloSEczESNPdjFDDxoLoT5apR\\\\\\\/xFHXjbQtEML62X\\\\\\\/2F7uKNZRkg8kDtho4O9UFoJbkgEfT9i4ZCcFV7vx4m8LdZfcRPX5Dqu5SqVsRLwMcDRs+OsjxS1r70OsuEnCQJ+Lf92d+cYDochCMStGY891f0\\\\\\\/ukx+vL0+Y30aXr0ubyvB9KAo4FLZajps5kDhGlCKEo2cc6f7YjB\\\\\\\/MatNZR\\\\\\\/HulDMBZ5Nn5O1vze\\\\\\\/4ORdL35UBWj+q1N7bOp36fG8E3RzSzhOny3IO9u4T4b0MKmjItjDrVKo\\\\\\\/8XkUYS5F5WpO4zWwtxxIGYdn78R8iissyeAWpFWJMCzJ3h8hmZjlETcWI4FwNoDv019+YMsmVHhH7EPCik48vg6XOAbxFK6HECQar6PxzWIFH78VlxItXLnMK0EWwwe\\\\\\\/VtPC5\\\\\\\/kqCtqJPfOZXgmSoS3cdilYi6JAPCv8MapAjIuLTvUUGH8kcNd6vNpQ\\\\\\\/c1g4Qbd0qFjCstWVhCqphfNREdvH4IGNjmG3ClF7jQ8d0LgyVsxF2Hk+IjpCLXhcd8kbBrrZPMdatxKoRxP29THerH3qf97dxn8EEJJM3pKZ5xGBcC1ypcVjaNi0AQTEeRN70aYOvAF0E+DZjIkpXCm3+midR6q7i+yCPWZmnng7wsWVHYX3MQ891EHL9V7qTESGfFKrNrxtLQxPZaWlDPEyuuhxPEQawNYGxh812MdtKDX0RqukdT9A\\\\\\\/NeSzpy0HIFetZyt7SP06V8\\\\\\\/QafDRNvV2A10I2cgoc3MMWsFgHffze8RQUh9t55zwMqv07IOJt5c6SKaiF1uO1bUj3L9eQAm6HOUyqr6ihVVHFN7gOQtOvawrI5nZRxEPkKGwQ19GO+TzSMIpGphJlYNfRP+rDLLROSdJukv91pAdD94JLYSwJOsoTdWfrl1Xd0PZB\\\\\\\/Kdl3jhiRFKOdKYK\\\\\\\/iW4obri2CJpbcUYt\\\\\\\/\\\\\\\/iZJp2JqhAuYIs3irhkJuMhouQyQeRdX+AzgQqYF892260+KeS8+RPD1j0XclDAo+ducvzgecLLg+k2Jw2ucNGI0AlIlKOYBtrsnOy6o37DTTn9\\\\\\\/MQZfISIDw2CgIvqueoYW+3UJnJfLS7WrwjCVsUyCRCNPRYsrdOKs1o6tYNQRKE5UZY74Z55jZZIbWhb4AQmbApzf+1bUv6LapGBZW3rStt9YLWDFXvuR+O5l02vG7cWy6FPla+mPAGOYjV\\\\\\\/gtEw31GSw06PsN4kO\\\\\\\/m5Wq4cfx72nT2oXHiF1pkrFUUsStIZYTxrQOOe5zGESk8FPX4JjpykP3TDxD0wDCmSc\\\\\\\/olGYDxTCQhj7hIpYhicIG8jN5JO7vlA04es9jb4BUjn3N+0\\\\\\\/z3r7F\\\\\\\/OOZuFOjSe4lq2acBTxuFs7pnFOJaxbomj6umlMMzdBTv97i23gea0d9rB09kLwytGWhF27IavDZxpAMzhLJlrs8uPJutHdJYuQG10DyVzpCHCm26TCF7IlMv2BVmthBWr9GlrMswo\\\\\\\/eMHGJ2Mt9RbAUFmhw\\\\\\\/fOqdmQiDcxwI3\\\\\\\/Wntmk7N4r6B+gi2HrkJYautN6trhwRpUyLO9CenGwjbj2e8Tp\\\\\\\/T9vwQk\\\\\\\/\\\\\\\/XuM22+H4q9Evl8hMt4hHvro8q2LA5dsa6O+RKmj9Eor12p7b6Art\\\\\\\/SbNW1ehwGKnxUrhXOVaPlOYgLjBuONBB6I8BnCAwghODYRl77w6BhhQq2WWMvmjfe14c314gBZCtFLBSgbwafqDjEM+Vsnhdg7EgN69N7ufeFYJOqAGrjOsmVfzIB\\\\\\\/9MBPl6tltp7NZ+St3CI4lkvHl+BinciZhZajldxBH5n\\\\\\\/cX\\\\\\\/UopB0wE0T2ifQcjohCsRDWzmxdVZ\\\\\\\/nyYL3sfK9KTSvAZSlFYYEOGg9Ft455TQhnBvFlGZcZ7zCf0QHm9Ta+4jmrJzXlHB2G+vHbBZdoWYRODFhgVRAr7zNpyqq+BkSWZJQ\\\\\\\/VDMqrY\\\\\\\/JTtSvaTPlCZuNzuOw75U5cVkitkt9+DyKTNwUYIXyyUxgwqGzoOkO5km9NvOFFbQTQjzbTyAzA4urjxsPeU9WnKC7BsORYPtMoJeWsVK\\\\\\\/\\\\\\\/mX+ODkO0QzRzfI8E68CC5FGA2AmuYpjScRcpfQl7a6aN7C1cAv2ECGHOXjYMSABhXLI3WeZgkhQ0H2NHje7OLgMr+QkAGE7en1vdVAYfqxx9O1HnXCdMC\\\\\\\/Pj4OxZoIIX8NpDbzuQdTWc3R1AU+vR\\\\\\\/CyamTLLlPFSkB6kEB0O5H81kRp50jrHfLABY4xjVjOs\\\\\\\/XyKtTjNDX3yNmFdySO5xJ8N6qdoJ55VLO6ArYPi\\\\\\\/PCBUJ5m2KWjuaeifbzPqeQnv1BrMaGHBO4oFcahfkOQ2V5ubh6qjzIayOFMjsMssxiY+tFo8x3+l6IGdXS3ZrYHkH0jw1jVNULHVUjQ4vmze6lq3yZZR9qitZGN+tmTIL3D3jXq30nqQQ+4HIgcge1\\\\\\\/qKVOeTXA2JCS\\\\\\\/zMtcc0KNEpQvCBk2Yah3VUNnQBh2DteJsP9gg9qJqixmeW9JC9uaZJmPa9qmR\\\\\\\/yrewn3amap7SHR6mJTYX7bXxFS41HOofQgiqgckQ5KhiX7PefZi95mRo9yOm2qtAST6H7AtXC6EIWhmhCORIsFCr9sZdLb1qINZF5tB3N9MmdvhNmq4eVoSLmyGYlpe8gnys4GRPCJNtsGiqbZVjN+BQS5qa2SORhnGg0ot5BK8daZEg1jfb6UOzTKLQWubT5PpnfkytejqfAnE3+suD3A3Cd9xrCmtgJtPC6u8x8qbA3M4l46o80wRTUrveGpU+2dneqf73YMCIfGpG+JzzMUPXwxe7P5Wz1tG0+T\\\\\\\/6jMVrXeMxfjlXW1nUQA+WVyBiyfF+sO934YYndx6uoND457Yj1+G42tVix1qWfehvTQjqVFm9K+M8AhnpYjEBbfUYjmaaSSc6WFgYDJPlPdLz61tJEXx9s5ma98n1oZYOSVvL6u\\\\\\\/2K+sJJc8gpVKjVE7h2KZaoKp4Pjtaae0MqGt2LjJqN2ZFn\\\\\\\/yYE1epIomLfVoNfts9XDiYHV2yb2RdxBX324D2YHeJJp0R7L\\\\\\\/f2NC4Yu01NOr2j0e9Y2eqZhL4FjoPa\\\\\\\/jaLfEMZggn0Z4XNKP+v3Ayss9XnfzAnq+hFoMl6Cz3BGBsVWEjbzi+ByLfjNA7Ub219GE5vjqXvDwh4KaiI31I9AbuZPTGcPcmQW+7dcj39hYTcj8xifTOZ5jNT\\\\\\\/mpj+FpYVOltLf2VPge9u0zgtjAG2+DPjRP3nO+vzXrEH5XeF2wFhWsP8CIp9DlGJeYaHUfkzKJqG+mtzY2QSNp4X9F5VJF3nvc8jbwyjKejKztsmLH5DAobLpDB4BNLVNy0lG9ZTi+M2F6qVkVUk2uXmjMxE7GQ7pK99EHqGcOXv4mwuFJVACQ6Aue\\\\\\\/4HrYgW8u1uz\\\\\\\/RB2DYd4eCvzwm7cSDIGa1wbmeJ+bxOFXra6Czymoe3KJT0M7BzFEnziePY6WsiHT5lfzd8GVksLrdylmPoF2HG1nmTtTP1hlu+IePecKYb+M1mNLGyYC1NBYYL9cUd80BXAGTBp9a2Pv57fs9fXjdHBys89qNgwqJK6IemQVFEGZoyY6Ahvyn3u\\\\\\\/oCeSI4UdDUrNR+\\\\\\\/JZ\\\\\\\/p1jAQRzNx1CkN9RRqipbedXc1v0ISYv31w6WCOH53OnxuCbUsI2WhujIJrR8mk6vEfKiLWhEzz9hvSvOWv7SaLvAMLcVLaLjPGU3wMeuc99jFOni5cjL4r9dxqH4njC\\\\\\\/6m+xlLjgAJyy0Wp7Ylh161S8mOr1o0NyK3V1lvIOmJ3lFHjZwznJOeolAgIbV5WZY27VhqtLHLdDUD7tH5QmeMJKvruueV0zhAzBodvbZtwpig+W7F2MSbSocMSy3OpVLmtkfKwBQlmqwPGGiqyuWZDhn5s+VBZdL9kgcqxfF85m9zCfuwfKt5UVnsSBA7+TGPYra6WF80wl8ajS5SWj+sKyTIJhzQucEYvAsD2rKw1k5EH1rIlSGPm5PQPrQ8vEQanbHVyZpCyZrF1cWulRIk99\\\\\\\/kY25e\\\\\\\/3TMvpWWuyi8np36jhPG3pdOOxeFRjQlOff0VYx4BFEpeyw3Jgw\\\\\\\/UlHiFwfM7yOW2K+6deY3neJCXI3hbzXeUvvpwg0bwouHmYtFdtLtWc1RsMZQamfwd5k7bMcrNYZ+lC4NlGlUjM38LdqrJ4SMmbDywzRif3hMoyDnii+V8QlfaaQusZish9sUaBfx0QRfMN1i4rsQb8RiIHN8CBWn5aRM1xBpP1w5v\\\\\\\/3lRLO+h\\\\\\\/ESkaQGAnrkjNsKuGy4jXwL9ABzObK+E1Tc63Sck3lVn1C4cOs1vgNsGCTMA5L9YncFwmA5ktEi\\\\\\\/ZA\\\\\\\/UTzhcdymKxJy9vppXZyulR9a57EkrczLziYGwsDVm8mU0LucklZGMluxa418o3kQ5VsyWD7qX\\\\\\\/X+MAAeISHeMZU2j5LhjXsLodbtFAdLB8coJ7CLAL4KzGJ7x0JIBP1qPZ741IokkwX44nI4ittMCGiY4TTpuOGybQwFaDAuR6i91KGK+oaRDwqteeTaLtBBaiOzJqOPYTnmQUh3WBA6AUiM1kvl8zMxDbBtef2CeR6SEVIk3kgxUt2DAGJRnhnV85SHQAeQ9opXYZxnmIZ9YLnY4mojg7UjSlZdsVaUp2E1WMAzp\\\\\\\/utz1SgGZB45a4+3HoaqrcZV5uTr6RfO+xiYQmqU4Hy\\\\\\\/AV0+aw4Q5U1g8m12v6KpuiWy\\\\\\\/7JH+6ig4TaS46qWrKfHSyXavU6HE+Gj9wHh3Q\\\\\\\/c674jnWovZjrJk6JaduqL746Z2D\\\\\\\/2QCbgNklX63U4LgdYOLuQnkXZenEeSJZxIOeU\\\\\\\/ohoXpOv\\\\\\\/SvdnnsaaqhM1NizbBTo1n7VqDjKU+MgwNLmah7JndGQ6nstQWkrZBcS8ms5pxfHTuxf0+T\\\\\\\/CoVdBGlc0cQOfl7xYR4zjxTxIOF2XLLJAZM0ugUPxsdx\\\\\\\/xsqzMlYhkAS8leiK58Ocp9rc5GiccmKNggnonyr4At5kpw1eZ\\\\\\\/lr+TQ6RNHmMakrRcXaq+KtQ7J+RS7oBqOlEAdd7URSFLaKR6drpf7ouwhajTMpCy+po+6YWiRyCGnO9F9lvjy1Wb5vLoFl+kSiUW4cpj3VqdFo5dNildLHzT1V6\\\\\\\/bMlcwFIFewgtDyaY1xja+Qhec9J0x1dCt0lzlspSSbh1GavjDvABa8g2u3H+Baw4oVfVuEmNAUQs7FAOFzK8FPo2vHSIPWmFFJPQ6ehc5j92iCiBh+9THlh8iFx8G4xdloWbLWXKK6obDnRIMHSE3ZE20GbOm5ZJ0d+HnZ5tmLqL+WMCVmfUVAKebIBfZ8JVXCKVp6kvtcYn3fV3KuI7A66FWk2NDVaawchlH\\\\\\\/3IvJKUL0hpbawFcFUDLd0C+cLZAiiLgkeSE8\\\\\\\/YWodEZjU4xD1tPF2JCvkQbeYbbzi\\\\\\\/Ebi4ilssIOovI7ESCrZmRmx0Vx3gJfXV7F57ZW81EqXXvGq+10X1Gyil9Jp5CviZz3OYs0uhbCJq5bpGR45yq9uHl6f8VtXeR3dGBudvqpz5shev36ee\\\\\\\/miHA3xap+uol4K0z2Cnrg4cqIK5OjSF\\\\\\\/s86j4oWIKTnSYTTAZLp6Y3DmrWLwlAQV8lZQaOFo4X8L5JKVf98DH0W40osZs+Rpa5mRvrfxTC+17nmiv2r0cnyX7ksy0n9AjAf1JH\\\\\\\/6EyvQxvx2dBIRFqF0ocaQNlXVNYDdaK3IPPB01vp5Yi50tZ0ImzApA+XzgTtTYtQggf4lSCBfcS4Tlad59CAmoUiG6t0hmw59xjibgxk0y9M\\\\\\\/5nBzSIsbBrPWx43tzr7L1hW5jAgdjXfZOTvoloq\\\\\\\/kKDQwqQFOhLys1pqwA8tJY9U5JK6293kI\\\\\\\/l18KB5C3OXex3N0viJjduZUksf9eAdg8qHY2sMn7eT6hdXNKihE6BseHACAm5sU7HqMGTpdGFuLLYSJxv9ApCakIkgFk53suzl21tEqFxCxiAod124o7z4PsAK1zb9RO+\\\\\\\/JOhu6b1BsL9ILVAXVwMxya6SJHfs2b2HdTj1K1+\\\\\\\/6TqPEQYB5nJmP+eK3lXSsR01XfCwuDrVlnERsp3S2sHptq6xjL2ekiUWR7gV4maKrNuqjA80lCtliAPJ1nPWilzTuMIqiTgg5r4KI2DNJNHX\\\\\\\/CDADFsJW\\\\\\\/I9sZcvyTpuNj3WfVuZutVEZCCjVvyFnZ+1+koJKwjN7LKQr+bQe51R3Fddpra+OpQGoW\\\\\\\/lLnQipKrM1cLZtaOoidFg3GSw0jP8ufiFTSKWJjemvpdar6SSG0\\\\\\\/4ATgQzeKlAMrulkZDfAgYS8HwVmsuNA\\\\\\\/JNt8ns4AsOEWRVH6C9\\\\\\\/jaEGdOs\\\\\\\/5mDKj8lVmMYj6x1bcP19RQEoJdaluV6FDMQpnqyHt5cJMcOhb7NyulqeGg3FNwnJPqbIz4hSTBl\\\\\\\/9M3yGxKLlBun1GXtNgw3FKImntTaiih\\\\\\\/1UiQtYLKkRQmlPt7zWXHJAKxuItQdks0q65KYU579BZqTK49Qs0GM9H0JkABHGmBOiw8P\\\\\\\/9Mdo2FjRGi7L\\\\\\\/mWGUUD+1PglyDUqwvSB+qpXeDMXmExGPu37i66eDVDhYmGRLsnz6Qu6W\\\\\\\/qmSyRNySL74AlkB7CajHia9HuQ3MTp1ZOXKnnKtWp6E8qSjJrAcvCBbqELuGKQ4tc9nMY\\\\\\\/2qKt+8pgjjOohVDTrZIaGyS+tTH9W25j6B+tcBwafdea1iE81\\\\\\\/FzU+6tCc+ZqQxbtyGGB4t+JR+uCsUbDlfdPigtwOXUBzkyrnpzKVjfHVLhFWnuWbwn9Lwsgul7Ev1y0hCwJ02O6+sczCdp9+Bt7+r+urpUZlGK+CcDbzo3fXsngTzHtH7HR7tjGxScBQ59d7crNaIsZE7+M8SMcxs2FxWglVDYjCa\\\\\\\/xBKDVJaWuhJOA6aFTEhP7PgYY5a\\\\\\\/4HYCJ6qMp7d3AJnhdWF4tBOicaDmqLhtcV1yHV9XG8NdVEYtjIcYYP681Hcbf\\\\\\\/EaABCki\\\\\\\/p58jGmM67WdZWrziSILMqW6uEGGGvZoSWg+dFYWrwLh5Y0JCCDlgDx4sEskTE9p57Gk22Sq\\\\\\\/sYE4DpsWb+Uh9Vvl1eZThwOeTanht5bAIRr2G4Tb3muCxHJk4Ydf3dLSEqySx+kEh3ZUKlu73JWiGWxj\\\\\\\/AB6zHdX8vzc4wnYfJbUonxDPa6ThZPqxK9dyJCLMn0lE8\\\\\\\/Lvw5iiUB+n\\\\\\\/5dtpHMXLUJE+zGF\\\\\\\/8i5E1UXyGL9RHHkEeHtBa1+k01gGZzKzNRhTwojZ3ysJiauJ9Kv3u\\\\\\\/qRUemgNrzIly3tCKTVUprocWhVWD0jiBEN6birRNLPSvMdNPoz25eYLo1WlehPIuvDX0h\\\\\\\/kRWCtNf4OU07+\\\\\\\/dL87cg7S+GRddIa\\\\\\\/06P8MrFOEDXJDzmePiEc0Ox+ZHriZuFs+LGwEOUWD3XYVkLG9nrAoX0bcedomwvoGhC+CY\\\\\\\/jyKFLI\\\\\\\/usW3tCCP+FGNoNFEY1pApj22xtbSyNl2T6mk0rAKYQLte6TQyEIa9cixBRaO15R4Sgdj7QALNnk91aDhJpF7oxGE+T7d8iNOyVMyKyL9GspOIHj5GnYSwvTRPx0VC0afAGd9866l2K5SqNd1hxFufwhX6MqB6tBwiACd4xv6QmGokNKTmxETl5KVIqAgUP9nxBfDVticMU1jBQndpIsCeZN025hrkW+r2ZW+07HTjfcN8c5Tk1FRYtFci8FsJaUx8O+FzO9vXoPzIgnrpYR+0LfEEZ7acpu+S1tY6gmwZ+ULnJo9W8+GXO231V\\\\\\\/p2Z3nVYw0fDh0W6gNjsg1eQj5YKDlxx2TF1P5Ja2w9tnftS5irnLojI504i6LhUWaQWlL011ZFkZrIwdbFuqqK1qeHfDR5wUlpU62pBAlbqamiDCiudPi\\\\\\\/OO83UUN4pYOSGhs+aeJOpZPBFC0rxAtCKYkNC03m44pnh3Zl7H9I710dONxPZyelM2CTNk1pTGGcgjktj3Ex912cGDnPV6seDOdkqeDzhwE8dIrM+g7\\\\\\\/f+u1kbniFzs82KB2fTbQVCha1bjrB82eF\\\\\\\/hOwI6SQXq\\\\\\\/DaxuAeCFuawCuIcC\\\\\\\/0GMaDS1t2h8aHGRZ0+hWxDUFHEbdB\\\\\\\/y9aIyJnBHr3jbZ0SS6\\\\\\\/jAtKxicNkTcsKQTMGnxhl8DTHIv0mxMUsrWROxGE9fQpKplePGBlI30csdCWdPzxSziN\\\\\\\/7JBtu4tFteCzPAaanevTsTjL8DnsMxLMcvQ4SvJj9BiMRmFWA+8zBnLMgtNydx93zfZRDG1AqmmOdWCMAbP2nYC8sAhm2wCaxsp1rnC\\\\\\\/7slw2ny\\\\\\\/2XKuYy+1bUW6Iof0P76fjp9ZhvPcxhWMmLN9rpod45FcE3k4wrRAR9aRDG0OPkJtOW1DjCe+Lk7TTQ8l5XcPYf6KCwba0gfJtNBKWBX5zzfJkfDTa119O1VxeCCrJyACkOUdaTMJyHjHpcoseK8\\\\\\\/8PGSPPs9zufbEMpnJcBly5DNUmPhdY2hjVFnZXs6LkrgeWp9vSdmHjpfMVTvNrMTUaRJTtXxZpMd29NGdIKFGirn32b+63LKH+\\\\\\\/X5VDvzSShvSHSnP9LJ8KUYQBCGtJ96CYIn+X5LV6U6iMks3nRSZ+5pJmun+0Q00ewLdeizaoAfolUb3chnUFgzrFOwv49fC1s7O7RjJ8mMDKWi\\\\\\\/LubxCh0+PYPkPENctu+6AfOZ6ww5dd\\\\\\\/y8XmIe67psvH9wp6eFJjTLTeM9ofYihxHYKGxywGuM6HvzQn9fGYf9BiwNyqp1EfJmn2qNYxVCdFLr1i\\\\\\\/OYC9x9+BsQatnnBFuwHSa9pHD4i0y6BQg\\\\\\\/o9Ud1ub3RwurwbyAs+My5fINeEQ\\\\\\\/nk89taGoRkTtVZBIcMrkXPqCGG1q8pOlDG3a6upLPqtnftb+weQYR3LXPEN1i7Ea5oGr5luq0inQCavi3a5NvpoK1JaLqDRUAv3jRg+69NV0QduZDKWA\\\\\\\/aVM5lPPFicafAVhVCjFQUGTCST4qhN5YPb+0eMs1vy2+u77jmupG9NpDeD0I3i5DYqGL\\\\\\\/ckl\\\\\\\/fb9rOFqRQJkQOuJyeP9V+n0VJ3P3DzkRmzffzTWlF1brKANGYn\\\\\\\/KHHMa0LX8yEXue4h7HkZxLsznExrrkqCjOWndTshLeSVdZ5x1DiBNQfg5Sbak8y3XGLx1HFTVEARERZbV8wJ\\\\\\\/xbRQLU7L4V\\\\\\\/s7dVWQQlcC5bk+Mb9UYvjr5h\\\\\\\/l5UFAZW41G3wgYDYi5BwE0ctAjAu4alo2+TeflsRbg85\\\\\\\/0wk9hCaeNwR6X486iL7QEC0YQXAlHUflLkXoCo7Nw8m4Eoim\\\\\\\/971kdt4tkHUv5HZp0T0ryCFkAH+hfJStglZPaJ\\\\\\\/Px58yw6YWlAflPbxBvopu5S3wkvhkzBuEm6svxCFAUYSYm0+ZUz5hsLRza6MRVbga1ysAriyAdaQvalaTPA+NPwq474kTH0Y\\\\\\\/ePq3csiG9+sLt2lmvw3\\\\\\\/PUn0vwl2r5P+5p6TsyJlB5fV7vI9GkXV5r7HWaK8YpwJgtaBxCuw4\\\\\\\/oJ6m16p16R6XTlMvXxJBsj8HXwK8jMnxxGvK5XaV\\\\\\\/7B4\\\\\\\/V2QfzflkfuXVIu4OPAK1TpIIJwQZVUqO6TG41uqiyC+vT0r412jZjUddLBr2avjhoEttvlxH9Y18BI+4fcVleGjv9zf6JQ7d3FDdu8T4Oefdm+1x7SYR4EE0Jg94KTSsP7CNEWiglYMtJyqKihCwFPPlJUStWu1TKHWQ8j+7Y8JUFkmBbmjWkgUAb7SB5cSsKHrpWd7dqSqYyYpnY1nYIWI5eBBgdTX5Rg4zKOaEZCaO53EtTQWR+foGgMFZAhnfF9kQl4h\\\\\\\/0dfcmoK\\\\\\\/CESAPuzKnEOwDS5s9CJjp2xT6Ouf9Z13Cx8PM5oP8whORosyTFbB2L2hH9QqroC+mT+kmUz4344dtNTQGhV7F6yZII8+zC\\\\\\\/jJbtCs03d431iGLAl4jk3G\\\\\\\/rTlsJln11EPkD2YV8\\\\\\\/lnxs7wis7qdXWGX8Rijej3idvaxWBoVX\\\\\\\/YFTciFNiWnhwM\\\\\\\/f5SRoRu2Ypm\\\\\\\/RZbTtthj8QK2rH5TR46qBa+Eh5dN1f\\\\\\\/hxksRYm2HIQi9A\\\\\\\/11d8CppOtMcWJrCatcQswPwokQjdtHHgFqTSwyl4QIehDloLNsLsmRbY9baCjpVPEQ31hVRCpvhcBsoHaGpFY7keGg5D15193dsKt5X7qrL7CX8lBpYDW2UzmLmfr356MNBy5Vm1u3J603MnAdg8NAYSUj4wggzaN\\\\\\\/YipvmTIyZYwQdjVPTri7tNWVUwE0DrGJVpm4\\\\\\\/V8h+AzLOqJPn5eUnFZ8mP5wyVh5n9TGWVLMQSVzzK7jTh7fjIHOd1SGraK91W\\\\\\\/M5U5BcIbFp3i6Cmq\\\\\\\/TalipxpbVhIHXgY+Fo1TxtGouCSEtEBJbvAKS7ZEh2Ha1vOuPcXEnMsHBppnq6Wgn\\\\\\\/ht9thDEk1wRZjOLh7WGKaQTAM+9zJTgTraXA5ZOZmkmvnKX\\\\\\\/ZvgaHpCGrlPl6vjBffEz4M0QvoF3+rjsi6GusCaoDQEvUExYhWrTCzo91vHBoq8lxFZQRw\\\\\\\/d11scjIMZsdqAmTw6Mjh5g35TCyI5YYWcvn7B8az3D+UvvJ21WfmoC6LC2L1cbW6yfrK28ys3aXFp18eSfTM0U5dUVKvTQhaGTAqHPhKpzHV95Cg2sU2h88odz2CLrek62FqHYbl0+z3HQm6cpbTyZZW+T8Q\\\\\\\/r6zYF2LxG05pOmIsmYCdvqb2WdilEnB+RGqYhW8xQtSWCytK7Ngz4PPxqSZahlAscl8\\\\\\\/lBUa5b5tFoEvW0ws+VlU\\\\\\\/0DiTyJm01WQSx8oHl7Fi3XaiFW75nyHyM5d4v3DdZERRpPagYzWycz1m\\\\\\\/aRt\\\\\\\/wk0O73TR8+IlimQAn0bdOOg01JrLrrMbJH+EmEeBW0vCBnh61GCxX6gvWoH9VIy5xxu5vcn3LmqXD1kO89tobOhG3TU0W5HjWQDxh+PalcmtdBasdwiD1f\\\\\\\/wQVdznXK8fdgfskI1+gZTelEFf7Ay30yTFljiesHQ+UYYQYaPPjozSzMXeCmRji1uTlTTIcoRoI2mPQdcJJXtFGtXAhz6GK9xTJL0S\\\\\\\/Va0u2ebYWVZ65aMAs0cywFSNCtsyfuRvqwSQ16UcgduBCWdwjH0O6UE6E9yoSKV06OWFqkKhLypwThezfCO7PUOoC5bPk8oo5a2uIlYBXjuLsb7Rpb50x1v0YWw8YJoJ9FMEViQ5msl94FgNKnO4JbwWGBGW\\\\\\\/9I0eIlqErmX2dSqaVd4AQlSuXNy5VeDFIXVKwEcTLGPbKU07RIFJ3babJJSkwZptG34cFRejxXBU8mVeWEzuohP1dZYIg0eFi5QROdNNu2EPN3XkQpzDZmGU1vTjN00sRVCXdA1ZQKRW\\\\\\\/ygw8pDQLfkKoiiotsy1ZzlTscYPCPnsTMQqKJ4YiFpYbF9TuH4N8zUM+AWsRrq1IJDw6VV9s0QC0D0mfs97Gby7V+MhEiEjHbWSzMeQfLQHpka1C6FS7mjai\\\\\\\/L6ZV6qUNDwiTxGO3JEWqWrzZR9PaHOy3pGpiJ51XYo\\\\\\\/t7e2R7Fxw6uZmgtsG+d11Pvz9Ngj95r4TvLrgYciwGxPyvj9jNBlXVU59bHx\\\\\\\/J6CagqVxgp1udeNjFHR5roiJcg\\\\\\\/2oOgm\\\\\\\/CMSrVqH2ReY6qbePUQ3UG6hoGMPdpIjp6bEqRrZNsRbBqle5AIKfcy5dnKVAUzDKHY+5tKTzopdwvCqZ\\\\\\\/MA5qKMfYVvn\\\\\\\/4oe8Ceu3vn2mFKodF0nGnxoGcoxMLnT6UBl9xeiz4TSMqBChagF1+HWfE4PTgEuylZFjG2ZYrypiGF3BPiH9EM8Cfq+GzAek58y\\\\\\\/U7Rv8\\\\\\\/QxeWyDccF2JDh2FrdOm5LWki1Qmqf8rnrhuexdnWAnAbZDfvgavEl5w1LL+ca6OD\\\\\\\/9kc\\\\\\\/RAp7HzbjXpuEuh4ssV1O1l9uksgGtDakn2\\\\\\\/WNgGUhLzWpQur8f4sKHJUyrKWPeTRu\\\\\\\/qRqh5CKjyAc1Vux+AePz\\\\\\\/Vq08H1pSJhTrrfQkMpcOdw15I47AYpiLna7J7Mi4krkzWauWounLMeu2vsaQCV9rksRwdeXPF5Kw7vFhrDLfmJUn9f1MLiULKgjKm9KLMoV5cxdDBXxl1mkN2HIza2xmGiZsB1xtC8AEbt3HmyqY7sF0n9BOIfW16uvw6kQB+aKtNzbtaD8qm+GuIjg8oKK8FEPLoKE49bgqqJ\\\\\\\/FYR07aTBNM1my5XerEoT2tRlnmJo4VhHFjbLG+jG2KA18JE4IfX8SPnlkfDH4gGsNnZt5st1jTTQ06qmuh7YaqJOInbB4gE++Xp3ZmOgJgWRA+zI0DePTlsmuRn5iQMZdi9pUhhhL\\\\\\\/Tu1AzunDToMPz4NYjBuX2e4wigsjzn3bTyh46OWY3T5\\\\\\\/NC0q8c+bpj\\\\\\\/JlrqQqq+2D1XSi4d7KKWsoE7fFW8cpak1y+6ZDSV1EB\\\\\\\/um1qT4n0pZTDI08FvMiqqKrtIG9hYcGIxa1PurMKfhKB13Ot0ed0U1+GBed9g0a88lx8pAjRUnDX3uG5d5XtSga6UU3tljjsBsYXHqbMlwXBw2mfY7BaqvODEgepqPfGN3AnsxsFDD+MveugH26uBIIkSkyX53rIFvMKO+PNDdVKX4997vONHspD8T+dNNhIxUL4aIJ1pfmoMxo1KmvM5ELgCRK+U42XskXDlN\\\\\\\/X23FNbDNjKjrQQOP2574UusCQLMx6OdnXNYE\\\\\\\/xqyszG\\\\\\\/Au89PXKHGQxx7w9a2Y96S+I+RGkSfd7pqBX+R8T1z8FzxoPqtSICtpZOeHyc0UtsVR+wdyIEo7YPbEtKV2YGDrC4Btk74ObDoYjHoRk2jrztfqy1ZUp6ylAE4Zg7E+j3+970rP+1LsiFdwspVrytDg9fUjihIy8u+iHt4RZebtetGKpoFjeXa+gp7OoM0EG+2An7bR8ApvbA\\\\\\\/y\\\\\\\/LXTIv8dPLKaq9R+JeiRfX6jwwZmIdYq2XVUbi8n+K6T37sSabR1Sk5FHbedBATwmJ+GZAQoJEbJp5iyitZHkU\\\\\\\/dy4QEx31YIv8qtQxe7PMb1B0jKrBxtzmq37EbX6Jv\\\\\\\/\\\\\\\/zbSNwjStGrR1RCtyV4ocj\\\\\\\/sRMItoeuOzsc9gLwIihCLsU7R8ebowZUInsLPsrvLL0\\\\\\\/ppMaqQWo46QZO8QUQjoMaxNTXjP001t4QTE9xSpqYZV\\\\\\\/6OaqoI4HlosH\\\\\\\/TCsqo2gdO2ob\\\\\\\/x6b47fNrpqNC6eRApMs490X77zuyMRffeB+poZMsVhvJfyCaHxXODNKlUP2J4EcIriHbzSugZXsKvvMfJd1m+biJTux1glaJywMsuPiHRZQgB1dbGwx9G+MfsR7MnFRNfGEayVosl4zAXZxuNme3yMGJW9JSHw\\\\\\\/n\\\\\\\/\\\\\\\/cEVviio3uAr61MwtRHTLh\\\\\\\/Sh9adsqWX8BG02H4mMnWEBzaNAQ3kRJvMZzezqS622q\\\\\\\/DgcDuMxebsmL3r4u2uJVk23vuTQR7CDOCiBL1vhaDtMGQ02S1zOLrVDO4kuk6A+mKvXP5j3EiFShX4EpArBE20QOyYoopsyHq3FNT5Sf1ADnmVZNmPOLLSOJWpcfhYFaJfjZQfNZjBVOHnvSN9C0Z5jjPjZe4gL4CnMhEFwle6mW\\\\\\\/+uUWojRLfsYe8\\\\\\\/EvwIY5sREi9cZzTt69lW27FcRdgkyisEkOXs0efnZRp\\\\\\\/yOoECfr4PoQn1T0yY5X0vUbjoirOvJZbgbGXgAk62GmDyAPVjgHOERh1AyoeJ38z9+snAxEQPL\\\\\\\/QCOhunqsdU2lWKO9AMcJikydnqN6+YXmqziklTTKaqN0kPc1vnAlf9t8yqARKfKxRl\\\\\\\/hKMI42oeS9eFSsWVDLSBJIEzQI0ZxLc55JanzfVfOa4HoBpADFFtUrtG9LLUbw01qLrwvFeRH16MqISQkgRjaChQK4f3QacexQF5I8gwP9HkcO\\\\\\\/zcB1vC4WC7RwyCOdp41LzRURWRi8ifcJOE9IxMYRepaaV2SavCqaXpbzbUzp5bOE\\\\\\\/vlkuiSYfMR177ONP\\\\\\\/nIBrXoB7ku3lCCnB9kLI3ofj2\\\\\\\/\\\\\\\/DbmHe22dFjDgoRUX6rAh6MP5VNlz8I0LhuXoGyEcYQZ6I5\\\\\\\/uBWWQ607gt1F6zx5otIy59zqoiNDJuO6luDvvHJCQrNroGI7yUAOLHJxK0QBegUAFnO4oYUfiwtzaM6ap4RwyxNZQmzNxba8ki1VoQmu+ozjX+2JDt56oxguIBE4ysKzolJSEA44UAfo4J6eqVp3VHcANfc2tK3Cix85izOzl7b1fAvnw5Efez5\\\\\\\/0z7xLy0XY\\\\\\\/f0YMI7nYPTVR3sAfJmo+o2geeh2piuW+jiQwdABg8M5zkvnn+U7gEBKo9hx\\\\\\\/9ipDbs\\\\\\\/FTGc\\\\\\\/A1L7yXOCNgNi0VXT+N8N8ViGkS5JnGX4imzPMq8QZefUDX22LSBJ7LfoF2h0wYERQicX1zD8kCoTRIM6A2LRB1yanK3e3LWjlTpCovEbl\\\\\\\/Oo7g1HxANUar2p6YIBhNLy\\\\\\\/S0jYLET+z8Ghf97XPAWNXt5Nm4jmaXYHZC8PZTVyMBjzJ82SeVYIm\\\\\\\/cAN+bavZyOzClkxaFMyhQft0qaQh0ettXyMgTCNjFEPKKDNLcnx4pBtmHF8sYaVLU3ZWbnU+kDJRfj3Jb\\\\\\\/bJWez7a1EP6JUXSx52MXJCRzPG2w==\\\u0022,\\\u0022iv\\\u0022:\\\u0022SE9lAiF+kgdZenhCjKNJSA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002281294aeb38d42e6ee53059c4e95c5e8198394c2d82cf71839bf928347b429661\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00222PBcqvH1HEoOchrILwfoDWe6Qcb19yulHXjSxfsubrNYGrnnskjziHljAD6XgJgkJtajBjvbr9+9j\\\\\\\/MOTVzVRzIr\\\\\\\/R2Tr4UurNtTtR5Fu8V9BaXMJypy59EqQq9GDaAvRVtH8lbvW9LdXfZIQHmBd0c6TQa5nxYJwR+l9ArxMnZjI2Q7bTBkHwfx2CBHyTISA+8oKWaci7AKavQEUzWMrlhERk3CgULsUu9\\\\\\\/lrF1QXyivmu+2FZhs4j+G3ClsW\\\\\\\/D9r7gTCLvDJbnBawKejgDkc6xS+bn+axRdiSc\\\\\\\/mLNFCnxhrqNSRgHzshNnLzXXiJpCW3PEkQcIl6\\\\\\\/od\\\\\\\/TeJpdg8\\\\\\\/UYBsBFHmKrIrnHfJzNtsiizf1uIzTtx+J3V+XynIZiGKQDwXTuGWXXCx0C1H554qPBNFP3PW6UeA7tTmEv1Zrz66iacslrZ3GBQP3ZVzTm0IhechQnqF6sb4YzMKp2ofyug5D\\\\\\\/gRTRIXXllcuv6Yo8wNZ8IeMBnTjQWmOpqpuKNmxjMdQr6QQ9n9HcLSZVDvOolWwFNHQXsejWsVY+QStRKy\\\\\\\/LjO5INp7EPV3SC8mGZ8XyuHQnIKRtZGCBCfu5XXQB3b2dxvd5g5is3ZtKuXj3dLY4BM6\\\\\\\/QkLJcB9\\\\\\\/nMTStgtV4GurkXaX5Q6vNK7PnZU8e3xQspPzFrX6xTtvPhTVxnw3rQH8a1I95kNIyf481qubcguRigHutGLSQMAv+nBbXBXiStn32HdWFzCtBuWO0sAdBLa5S0V6vj08M\\\\\\\/dHU6NwBPhPDI2BzNCIci6AyxOL8kp1jsP6QzuxDSWQqKVdNN62uJ9KurGnekKUyhbi+cgJ7F6iaXi88rUJZM7\\\\\\\/A4rB0stqjlF1JpcDmwvan4Gya8Nxzh0qW4i5mdVKX04U4EyRJpCXC0EJ9Pkveb79VCJOZMmRJ4dJ0b3+empmX5tRtK7pG2JazJ3q4onxEJQG8T8LJhadVYtMkhD87GXrXFs6l+yrd8kAmEjjh098dxxsCgEKbmYMcQmapSxrXkC8r+KfYxSsZ41d3yn2zjRsOX0rCOc1VpNoK4cTE4DQiAwUiU3W414Kzp4tvSAUUA5wwe8+kd43xJF6E4bB21ku6\\\\\\\/XguQM2EmEzQZy9CvbzQ+EYtqA3Sz3a6htaDnWNXbWBl9oEZez\\\\\\\/yu40kKU+YF6q0nA3HaehexbpwBq4sWmus4YKHsw+elLKO3r6oi9qaPpm968A4T+V+Lw61L8z9txhTRt99IhnW0iv3Fy4MaxUKaBoAoy7ZCyrfuq+T7Y9QjVxximRT4r00Sy2e6qvL2Or6gsqsCcgUcUm3qmrKZlbyiydch\\\\\\\/Izujb7wjguriYPi0e3H8RxIe3b+Q9aZ3pmHti1P2O5WWzHiI1FTpXmx7idRGYLFutYK14la+DG4MhLLfmIfUQQxSnJFdgkj+b1qOi0ww6EFDWEk4AeFUdZnkUzLe1pM7ppc2Kh3ZgwjoCwKbWdWOce5gxQnDnyWiy0gMcrYo5u6OYduny3+7M\\\\\\\/urW9t9djUTv2Bw6l3+BQBMEuuaHFsWtGu0FsqGTdQrzq1eFsgShe0nlT8y1rA7nvCTLloBJhUzi9x2DNuNryBKk+8TUfCf6aSTVy9JO1OBaESvt8kVFixdcnL8MhfQUcwJVNfXGs8YlHgA9eYcUVotjGYMT3WbWLb7lUfO\\\\\\\/Lbx7\\\\\\\/Dlk0jcUwq\\\\\\\/nKGxZKM0nOKmW6uXxa0nYsUVcxcUV23qVjEDxhn31o0QxeDkUh1L2VxCDdY0TZS0rZhnH7thCIpqKzPCW6uz+gm6SK0Fm8lmrOEu2ibHQUwDyRY7\\\\\\\/4hN5qQjy77VJJGWldcVgwKQu02wWrw\\\\\\\/AcGoec3oi9WRHM5wcoeh6oFJa+dmHcRNmmDfOsXTLXdOIBuLndO9aWvnYs0ZnRiXOEqRHL6VrSk625XK9ZYxX7T1iL8JVU0PtX3PvxMK6D0LwEWwUe8xowh\\\\\\\/KRI5FAA9RQ4Ns\\\\\\\/kLwTapkC9LTLN33PTuS37Q2zhWPUNByOmfYA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022msAu3hULM5yxoa3C1JgwjA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002208444a31c7a5075ee1448fbb63932022c9e9aa9011a3b2725e3ee23a675e449c\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-494621 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-software-and-data-resources">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/11/3575" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-LfvG0eqnVVOb5JwUuvfM6Jj6deYttB-MABU41qab8ng" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/11/3575" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-aiGBOnwbJR8L_ETw_kin_s0_BkBKW4FQpzs4q7M2Fn8" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/11/3567" title="A Screen of the Conserved Kinome for Negative Regulators of LIN-12 Negative Regulatory Region (“NRR”)-Missense Activity in Caenorhabditis elegans" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/11/3583" title="Multiple Functions of the Essential Gene PpV in Drosophila Early Development" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494621" id="node494621" data-pisa="ggg;9/11/3575" data-pisa-master="ggg;g3.119.400596" data-apath="/ggg/9/11/3575.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Deep Learning Approach for Detecting Copy Number Variation in Next-Generation Sequencing Data</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-4661-6391" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Tom Hill</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-8586-7137" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Robert L. Unckless</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3575-3582;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400596 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494621" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Tom Hill </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>4055 Haworth Hall, The Department of Molecular Biosciences, University of Kansas, 1200 Sunnyside Avenue, Lawrence, KS 66045.</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tom%2BHill%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hill%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ATom%2BHill%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-4661-6391" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Tom Hill</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:tom.hill@ku.edu" class="" data-icon-position="" data-hide-link-title="0">tom.hill@ku.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Robert L. Unckless </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>4055 Haworth Hall, The Department of Molecular Biosciences, University of Kansas, 1200 Sunnyside Avenue, Lawrence, KS 66045.</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Robert%2BL.%2BUnckless%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Unckless%20RL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARobert%2BL.%2BUnckless%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-8586-7137" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Robert L. Unckless</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/9/11/3575" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:494621" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:494621/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/11/3575.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:494621" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:494621/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/11/3575.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:494621" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:494621/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/9/11/3575.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:494621" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:494621/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Copy number variants (CNV) are associated with phenotypic variation in several species. However, properly detecting changes in copy numbers of sequences remains a difficult problem, especially in lower quality or lower coverage next-generation sequencing data. Here, inspired by recent applications of machine learning in genomics, we describe a method to detect duplications and deletions in short-read sequencing data. In low coverage data, machine learning appears to be more powerful in the detection of CNVs than the gold-standard methods of coverage estimation alone, and of equal power in high coverage data. We also demonstrate how replicating training sets allows a more precise detection of CNVs, even identifying novel CNVs in two genomes previously surveyed thoroughly for CNVs using long read data.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/coverage" class="hw-term hw-article-keyword hw-article-keyword-coverage" rel="nofollow">coverage</a></li><li class="kwd"><a href="/keyword/deletion" class="hw-term hw-article-keyword hw-article-keyword-deletion" rel="nofollow">deletion</a></li><li class="kwd"><a href="/keyword/duplication" class="hw-term hw-article-keyword hw-article-keyword-duplication" rel="nofollow">duplication</a></li><li class="kwd"><a href="/keyword/machine-learning-0" class="hw-term hw-article-keyword hw-article-keyword-machine-learning" rel="nofollow">machine-learning</a></li><li class="kwd"><a href="/keyword/next-generation-sequencing-0" class="hw-term hw-article-keyword hw-article-keyword-next-generation-sequencing" rel="nofollow">next-generation sequencing</a></li></ul><p id="p-3">Copy number variation (CNV) of DNA sequences is responsible for functional phenotypic variation in many organisms, particularly when it comes to causing or fighting diseases (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Sturtevant 1937</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Inoue and Lupski 2002</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Rastogi and Liberles 2005</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Freeman <em>et al.</em> 2006</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Redon <em>et al.</em> 2006</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Schrider <em>et al.</em> 2013</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Unckless <em>et al.</em> 2016</a>). Additionally, recent studies have found that duplications and deletions are an important type of mutations with functional consequences for evolution to act upon (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Korbel <em>et al.</em> 2007</a>; <a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Schrider <em>et al.</em> 2013</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Zichner <em>et al.</em> 2013</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Rogers <em>et al.</em> 2014</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Cardoso-Moreira <em>et al.</em> 2016</a>). Despite its importance, properly detecting copy number variants is difficult and so the extent that CNVs contribute to phenotypic variation has yet to be fully ascertained (<a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Redon <em>et al.</em> 2006</a>; <a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">Zichner <em>et al.</em> 2013</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Chakraborty <em>et al.</em> 2018</a>). This detection difficulty is due to challenges in aligning CNVs, with similar copies of duplications being combined in assembling Sanger-sequencing genomes (<a id="xref-ref-31-3" class="xref-bibr" href="#ref-31">Redon <em>et al.</em> 2006</a>; <a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Chakraborty <em>et al.</em> 2018</a>). Additionally, mapping short-read NGS data to a reference genome lacking the duplication will combine the two gene copies as a single copy (<a id="xref-ref-31-4" class="xref-bibr" href="#ref-31">Redon <em>et al.</em> 2006</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Ye <em>et al.</em> 2009</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Abyzov <em>et al.</em> 2011</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Rausch <em>et al.</em> 2012</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Layer <em>et al.</em> 2014</a>). Several tools have been developed to detect these CNVs in next-generation sequencing (NGS) data, but for proper accuracy, they require high coverages of samples (for the detection of split-mapped reads, or better estimations of relative coverage), long-reads (able to span CNVs) or computationally intensive methods (<a id="xref-ref-31-5" class="xref-bibr" href="#ref-31">Redon <em>et al.</em> 2006</a>; <a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Ye <em>et al.</em> 2009</a>; <a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Abyzov <em>et al.</em> 2011</a>; <a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Rausch <em>et al.</em> 2012</a>; <a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Layer <em>et al.</em> 2014</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Chen <em>et al.</em> 2016</a>; <a id="xref-ref-7-3" class="xref-bibr" href="#ref-7">Chakraborty <em>et al.</em> 2018</a>). This limits the ability to detect CNVs between samples sequenced to relatively low coverages, with short reads, or on lower quality genomes.</p><p id="p-4">The recent development of numerous machine learning techniques in several aspects of genomics suggests a role for machine learning in the detection of copy number variants (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Larrañaga <em>et al.</em> 2006</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Libbrecht and Noble 2015</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Sheehan and Song 2016</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Schrider <em>et al.</em> 2018</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Schrider and Kern 2018</a>). Contemporary machine learning methods can classify windows across the genome associated with selective sweeps with surprising accuracy, even using lower quality data (<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Larrañaga <em>et al.</em> 2006</a>; <a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Libbrecht and Noble 2015</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Kern and Schrider 2018</a>). Additionally, supervised machine learning techniques are generally less computationally intensive than other unsupervised machine learning methods such as Approximate Bayesian Computation, because the user provides a training set for the supervised detection of classes (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Beaumont <em>et al.</em> 2002</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Jensen <em>et al.</em> 2008</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Beaumont 2010</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bouckaert <em>et al.</em> 2014</a>; <a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Schrider and Kern 2018</a>).</p><p id="p-5">Here we introduce a novel deep-learning based method for detecting copy number variation (primarily duplications and deletions), named ‘<strong>du</strong>plication and <strong>de</strong>letion Classifier using <strong>M</strong>achine <strong>L</strong>earning’ (dudeML). We outline our rationale for the statistics used to detect CNVs and the method employed, in which we calculate relative coverage changes across a genomic window (divided into sub-windows) which allows for the classification of window copy number using different machine learning classifiers. Using both simulated and known copy number variants, we show how dudeML can correctly detect copy number variants, outperforming basic coverage estimates alone and other CNV predictors when read depth is low.</p><div class="section methods" id="sec-1"><h2 class="">Methods and Model</h2><div id="sec-2" class="subsection"><h3>Machine learning method</h3><p id="p-6">Inspired by recent progress in machine learning for population genomics (<a id="xref-ref-17-3" class="xref-bibr" href="#ref-17">Larrañaga <em>et al.</em> 2006</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Pedregosa <em>et al.</em> 2011</a>; <a id="xref-ref-23-3" class="xref-bibr" href="#ref-23">Libbrecht and Noble 2015</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Schrider and Kern 2016</a>; <a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Sheehan and Song 2016</a>; <a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Kern and Schrider 2018</a>; <a id="xref-ref-38-3" class="xref-bibr" href="#ref-38">Schrider and Kern 2018</a>), we sought to develop a method to accurately and quickly classify the presence or absence of copy number variants in genomic windows using a supervised machine learning classifier. Based on previous software and methods for copy number detection (<a id="xref-ref-44-3" class="xref-bibr" href="#ref-44">Ye <em>et al.</em> 2009</a>; <a id="xref-ref-1-3" class="xref-bibr" href="#ref-1">Abyzov <em>et al.</em> 2011</a>; <a id="xref-ref-30-3" class="xref-bibr" href="#ref-30">Rausch <em>et al.</em> 2012</a>; <a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Chen <em>et al.</em> 2016</a>), we identified a number of values that may help determine if a duplication or deletion is present in a particular genomic window. We reasoned that both standardized median and mean coverage should indicate if a window is an outlier from the average coverage of a scaffold (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>, black), and that the interquartile range and standard deviation in standardized coverage of a window will increase in regions with higher coverage, decrease in regions with lower coverage and increase dramatically at copy number variant (CNV) edges due to rapid shifts in coverage (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>, gray). Here we define standardization as dividing the coverage by the mean or median of the coverage of all bases on the contig, so the standardized coverage is distributed around 1, with a minimum of 0 and no limit to the maximum. Another component of some CNV detection algorithms are unidirectional split mapped reads and the mapping of improper pairs surrounding or within a CNV which also indicate the breakpoint of a structural variant such as a deletion or tandem duplication (expected at the red/blue borders in <a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>) (<a id="xref-ref-44-4" class="xref-bibr" href="#ref-44">Ye <em>et al.</em> 2009</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Palmieri <em>et al.</em> 2014</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/11/3575/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Schematic demonstrating the rationale behind each statistic used to initially determine the presence/absence of each copy number variant. We expect the standardized median and mean coverage (black line) to increase in duplications (red) and decrease in deletions (blue). We expect the standard deviation and interquartile range of the standardized coverage to greatly increase at the edges of CNVs (gray line). At the borders of CNVs we also expect an increase in reads with supplementary alignments and improperly mapped read pairs, specifically across the edges of deletions (dark blue) or within a tandemly duplicated region (dark red)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1840640188" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Schematic demonstrating the rationale behind each statistic used to initially determine the presence/absence of each copy number variant. We expect the standardized median and mean coverage (black line) to increase in duplications (red) and decrease in deletions (blue). We expect the standard deviation and interquartile range of the standardized coverage to greatly increase at the edges of CNVs (gray line). At the borders of CNVs we also expect an increase in reads with supplementary alignments and improperly mapped read pairs, specifically across the edges of deletions (dark blue) or within a tandemly duplicated region (dark red).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/11/3575/F1.medium.gif" width="440" height="261"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/9/11/3575/F1.medium.gif" width="440" height="261"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/11/3575/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/11/3575/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494643" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-7" class="first-child">Schematic demonstrating the rationale behind each statistic used to initially determine the presence/absence of each copy number variant. We expect the standardized median and mean coverage (black line) to increase in duplications (red) and decrease in deletions (blue). We expect the standard deviation and interquartile range of the standardized coverage to greatly increase at the edges of CNVs (gray line). At the borders of CNVs we also expect an increase in reads with supplementary alignments and improperly mapped read pairs, specifically across the edges of deletions (dark blue) or within a tandemly duplicated region (dark red).</p><div class="sb-div caption-clear"></div></div></div><p id="p-8">We used these measures across a set of sub-windows within a window to define the copy number and CNV class of the focal sub-window at the center, using a classifier (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1</a>). Initially, we sought to identify which of the measures (and in what windows) are most useful for determining the presence or absence of a copy number variant, relative to a reference genome. To do this, we simulated tandem duplications and deletions (100-5000bp) across the <em>Drosophila melanogaster</em> reference chromosome 2L. We then simulated 100bp paired-end reads for this chromosome using WGsim (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Li 2011</a>) (parameters: -e 0.05 -d 500 -1 100 -2 100 -r 0.01 -R 0.15) and mapped these to the standard reference 2L using BWA and SAMtools (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Li and Durbin 2009</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Li <em>et al.</em> 2009</a>) in two forms, either ignoring repetitive windows based on calls from RepeatMasker (Smit and Hubley 2013-2015) (parameters: -s -gff -gccalc -lib [custom] -no_is -e ncbi -norrna -pa 5) or without masking and including all windows. We also simulated a second set of CNVs and related short read data as a test set. We considered samples mapped to the repeat masked genome and excluding masked windows to be ‘no repeats’, while samples which included the repetitive regions were labeled ‘with repeats’.</p><p id="p-9">To identify candidate CNVs, we calculated the values described above (standardized mean and median, standard deviation and interquartile range of the standardized coverage, and number of improper pairs) in sub-windows multiple times, each time for a different sub-window size (ranging from 10bp to 1000bp, sliding the same distance). We reformatted the data to vectors including the statistics for a focal sub-window and 10 sub-windows upstream and downstream of the focal sub-window, creating a set of statistics describing the 20 sub-windows around a focal sub-window, for every window in the chromosome. We then assigned each window a class, based on the known copy number and known class (deletion-containing, duplication-containing or normal) for the focal sub-window. We trained a random forest classifier with 100 estimators (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Liaw and Wiener 2002</a>; <a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Pedregosa <em>et al.</em> 2011</a>) to extract the features necessary to classify the presence or absence of a CNV in the central sub-window. We also repeated this process to identify important statistics as coverage of the training set decreases. The scripts and a tutorial for this process are available at <a href="https://github.com/tomh1lll/dudeml">https://github.com/tomh1lll/dudeml</a>, including the tool for detecting CNVs.</p></div><div id="sec-3" class="subsection"><h3>Classifier optimization</h3><p id="p-10">We sought to determine how window size (10 - 1000bp), number of windows (1 - 41), mean coverage of data (0.2 - 40) the number of individuals in a sequenced pool (1 - 40), and how the machine learning algorithm (Random Forest 100 estimators and 500 estimators, Extra Trees 100 and 500 estimators, Decision Tree, and Convolutional Neural Network classifiers) (<a id="xref-ref-26-3" class="xref-bibr" href="#ref-26">Pedregosa <em>et al.</em> 2011</a>) affects the ability to correctly classify a CNV in simulated data. In all cases using the same 2-D vector of values as input (<a id="xref-ref-26-4" class="xref-bibr" href="#ref-26">Pedregosa <em>et al.</em> 2011</a>). In each case we changed only one variable, otherwise coverage was set at 20-fold, window-size was set at 50bp, the number of sub-windows each side was set to 5 and the algorithm was set as Random Forest (100 estimators). For all comparisons (coverages, window sizes, number of windows or algorithm comparisons) we counted the number of True and False positive CNVs and estimated a receiver operating characteristic curve (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Brown and Davis 2006</a>).</p><p id="p-11">We used bedtools (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Quinlan and Hall 2010</a>) and RepeatMasker (Smit and Hubley 2013-2015) to identify regions on chromosome 2L without high levels of repetitive content. We then simulated 2000 duplications and 2000 deletions across these regions, varying in size between 100bp and 5000bp. To assess a machine learning classifier’s ability to detect CNVs across pooled data, we created a further subset of CNVs present at different frequencies in pools of chromosomes. We employed pools of 2 (the equivalent of sequencing an outbred diploid individual), 5, 10, 15, 20, 30 and 40 chromosomes, allowing the CNV to vary in frequency between 2.5% and 100% across samples, based on the number of chromosomes simulated (<em>e.g.</em>, a 50% minimum in a pool of 2 chromosomes, equivalent to a heterozygous CNV, and a 5% minimum in a pool of 20, equivalent to a singleton CNV in a pool of 20 chromosomes). This process was repeated twice to create independent test and training sets, both with known CNVs.</p><p id="p-12">We generated chromosomes containing simulated CNVs and simulated reads for these chromosomes using WGsim (<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Li 2011</a>) (parameters: -e 0.05 -d 500 -1 100 -2 100 -r 0.01 -R 0.15). We simulated reads to multiple median depths of coverage per base, between 0.2 to 40. We then combined all reads for each pool set and mapped these reads to the <em>D. melanogaster</em> Iso-1 reference 2L (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">dos Santos <em>et al.</em> 2015</a>) using BWA and SAMtools (<a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Li and Durbin 2009</a>; <a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Li <em>et al.</em> 2009</a>).</p><p id="p-13">For each data set, of varying window sizes, coverages and pool sizes, we then reformatted each window as described above to give the statistics for the focal window and 10 windows up and downstream, unless otherwise stated. For each training set, we gave each vector a class label, noting the presence of a duplication, deletion or neither. We also gave each vector a second-class label of copy number relative to the chromosome average: 0 for a fixed deletion, 0.5 for a deletion found in 50% of chromosomes, 1.75 for a duplication found in 75% chromosomes, 3 for a fixed duplication with 3 copies etc. We then used SKlearn to train a classifier based on the vectors assigned to each class (<a id="xref-ref-26-5" class="xref-bibr" href="#ref-26">Pedregosa <em>et al.</em> 2011</a>). The classifiers were then used to assign classes to windows in the test sets, which were then compared to their known designations to identify the true positive detection rate of each set.</p><p id="p-14">Finally, we compared our classifier to several other CNV callers: Delly (<a id="xref-ref-30-4" class="xref-bibr" href="#ref-30">Rausch <em>et al.</em> 2012</a>), Pindel (<a id="xref-ref-44-5" class="xref-bibr" href="#ref-44">Ye <em>et al.</em> 2009</a>), iCopyDAV (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Dharanipragada <em>et al.</em> 2018</a>), CNVnator (<a id="xref-ref-1-4" class="xref-bibr" href="#ref-1">Abyzov <em>et al.</em> 2011</a>), using parameters best fitting for each given data set) as well as pure coverage for samples of decreasing coverage or increasing poolsize. We were unable to get iCopyDAV to function for samples with coverage less than an average of onefold and so did not include samples below this coverage.</p></div><div id="sec-4" class="subsection"><h3>Examining the copy number estimation of dudeML <em>vs.</em> coverage-based estimates</h3><p id="p-15">We sought to compare the estimation of copy number variation of sequences using dudeML. For this we downloaded the genome of the large DNA virus Drosophila innubila Nudivirus (DiNV) from NCBI genome database: NC040699.1 (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Hill and Unckless 2018</a>) and created test and training sets for multiple copy numbers of DiNV (ranging from 0 to 50), relative to the coverage of 2L in <em>Drosophila melanogaster</em>. We then estimated copy number for each version of DiNV using dudeML and based on median coverage of DiNV divided by the median coverage of 2L, we also predicted copy number by fitting a loess regression on coverage <em>vs.</em> copy number in R (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Team 2013</a>). For these methods we then compared the rate of correct copy number estimation as copy number increases, and the mean difference in predicted and actual copy number as copy number increases.</p></div><div id="sec-5" class="subsection"><h3>Testing the classifier on real data with known CNVs</h3><p id="p-16">To test the classifier using known variants from long-read sequencing data, we downloaded the <em>D. melanogaster</em> Iso-1 and A4 reference genomes (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">dos Santos <em>et al.</em> 2015</a>; <a id="xref-ref-7-4" class="xref-bibr" href="#ref-7">Chakraborty <em>et al.</em> 2018</a>). Then, based on (<a id="xref-ref-7-5" class="xref-bibr" href="#ref-7">Chakraborty <em>et al.</em> 2018</a>), we extracted windows with known duplications and deletions relative to each other, for example a tandem duplication present in one genome but not the other would appear as a deletion of one copy of the sequence when reads for the non-duplicated strain are mapped to the genome containing the duplication. Mapping the corresponding short-read data for the duplicated strain to the non-duplicated strain genome would increase the coverage of the duplicated region. We downloaded short reads for each <em>D. melanogaster</em> genome (Iso-1: SRA ERR701706-11, A4: <a href="http://wfitch.bio.uci.edu/%E2%88%BCdspr/Data/index.html">http://wfitch.bio.uci.edu/∼dspr/Data/index.html</a>, ADL6/b3852 within SRA051316.tar) and mapped them to both the Iso-1 and A4 genomes separately using BWA MEM and SAMtools (<a id="xref-ref-20-3" class="xref-bibr" href="#ref-20">Li and Durbin 2009</a>; <a id="xref-ref-21-3" class="xref-bibr" href="#ref-21">Li <em>et al.</em> 2009</a>). Using the previously described methods, we calculated the statistics for each sub-window of each genome using bedtools and custom python scripts contained within dudeML. Using the training set described previously, we then classified each window of Iso-1 and A4 mapped to alternative reference and compared called CNVs to the previously detected CNVs.</p><p id="p-17">For each dataset, we also simulated 100 independent training sets, which we used to test the effectiveness of bootstrapping the random forest classifier. Each window was reclassified for each bootstrap training set, which are then used to calculate the consensus state for each window and the proportion of bootstrap replicates supporting that states.</p><p id="p-18">Finally, to validate any apparent ‘false-positive’ CNVs identified with our machine learning classifier, we downloaded pacific bioscience long read data for both Iso-1 and A4 (A4 PacBio SRA: SRR7874295 - SRR7874304, Iso-1 PacBio SRA: SRR1204085 - SRR1204696), and mapped this data to the opposite reference genome. For each high confidence (greater than 95% of bootstraps) ‘false-positive’ CNV, we manually visualized the PacBio data in the integrative genomics viewer (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Robinson <em>et al.</em> 2011</a>), looking for changes in coverage and split-mapped reads. For a randomly chosen group of these CNVs, we designed primers and confirmed CNVs using PCR (Data S1 &amp; S2). We designed primer pairs around each CNV to assess product size differences between strains, as well as inside the CNV for strain specific amplification for deletions or laddering in the case of duplications. PCR products from primer sets in both Iso-1 and A4 were then run on a 2% gel using gel electrophoresis (Figure S9).</p></div><div id="sec-6" class="subsection"><h3>Data availability</h3><p id="p-19">All data used in this manuscript are freely available and published online. dudeML and the simulated data used in this manuscript is available at <a href="https://github.com/tomh1lll/dudeml">https://github.com/tomh1lll/dudeml</a>. The DiNV genome can be found on the NCBI genome repository under the accession GCA_004132165.1. The latest releases of both the Iso-1 and A4 <em>Drosophila melanogaster</em> genomes can be found on the NCBI genome database at: <a href="https://www.ncbi.nlm.nih.gov/genome/genomes/47">https://www.ncbi.nlm.nih.gov/genome/genomes/47</a>. Short reads for each <em>D. melanogaster</em> genome can be accessed either through the NCBI SRA or on the DSPR online repository - Iso-1: SRA ERR701706-11, A4: <a href="http://wfitch.bio.uci.edu/%E2%88%BCdspr/Data/index.html">http://wfitch.bio.uci.edu/∼dspr/Data/index.html</a>, ADL6/b3852 within SRA051316.tar. Finally, long read data for both Iso-1 and A4 can be found on the NCBI SRA: A4 PacBio SRA: SRR7874295 - SRR7874304, Iso-1 PacBio SRA: SRR1204085 - SRR1204696. Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.9735890">https://doi.org/10.25387/g3.9735890</a>.</p></div></div><div class="section results-discussion" id="sec-7"><h2 class="">Results and Discussion</h2><div id="sec-8" class="subsection"><h3>A machine learning classifier can detect CNVs with high accuracy</h3><p id="p-20">We sought to develop a rapid, simple and accurate classifier of copy number variants in next-generation sequencing data (<a id="xref-ref-26-6" class="xref-bibr" href="#ref-26">Pedregosa <em>et al.</em> 2011</a>; <a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Schrider and Kern 2016</a>; <a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Schrider <em>et al.</em> 2018</a>). First, we assessed how useful multiple statistics are in the detection of non-reference duplications and deletions in short-read next-generation sequencing data (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1</a>). We simulated short read data for a chromosome containing multiple insertions and deletions relative to a reference genome and mapped these reads to the original reference chromosome. We divided each genomic window into a set of sub-windows, centered on a focal sub-window of interest. For each sub-window we calculated standardized median and mean coverage, the standard deviation and interquartile range of the standardized coverage within each sub-window, and the standardized number of reads with supplementary alignments (<em>e.g.</em>, split mapped) and reads in improper pairs (with too large or small insert sizes or mapped to different chromosomes) across the sub-window. We reasoned that each of these statistics can signal the increase or decrease of copy number of a sequence relative to a reference genome (<a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1</a>, see Materials and Methods). We created a vector of these statistics for the focal sub-window and each sub-window surrounding it, giving a set of values summarizing the window. These vectors of statistics for windows with known copy number variants (CNVs) are then fed into a machine learning classifier, which identifies the values most important to the correct classification of the presence of a CNV and the correct estimation of copy number. For simplicity we will refer to this classifier as the <strong>Du</strong>plication and <strong>De</strong>letion Classifier using <strong>M</strong>achine <strong>L</strong>earning (dudeML) moving forward. The tool developed as a wrapper for the pipeline, instructions for installation, specifics of the pipeline for detecting copy number variants, and links to test data used in this manuscript are available at <a href="https://github.com/tomh1lll/dudeml">https://github.com/tomh1lll/dudeml</a>. Using dudeML on high coverage (&gt;20-fold), simulated copy number variants, we find that standardized median coverage and interquartile range, and standardized mean coverage and standard deviation across windows (per individual, across all sites in a sub-window) are important for classifying the focal sub-window (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>). Surprisingly, the number of reads with supplementary alignments (reads where two ends map to different regions of the genome) and improperly paired reads are relatively unimportant for finding CNVs for smaller sub-window sizes (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2A</a>, below 1000bp), but increases in importance as sub-window size increases (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2A</a>). Though the breadth of a distribution will vary depending on the window-size and mean size of the CNV, the most important sub-windows for classifying a CNV appear to be the focal sub-window and up to 5 windows up and downstream of the focal sub-window (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2A</a>). We also find different statistics have different contributions across different window sizes and different coverages. For example, larger windows are more likely to include the edges of the CNV so standard deviation and improper read pairs are more important for CNV classification in larger windows (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2A</a>). However, larger windows appear to have lower true-positive rates, again due to the increased chance of overlapping with repeat content (Figure S1, true-positive rate ∼ window size: GLM t-value = -2.968, p-value = 0.00303), which can be accommodated to some degree by including repeat content (Figure S4). Similarly, as coverage increases, the classifier relies less on standard deviation and standardized mean of the sub-windows surrounding the focal sub-window, and more on median, improper pairs and interquartile range of the focal sub-window (Figure S2 &amp; S3).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/11/3575/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="A. Relative contribution of each statistic to the classification of copy number variants, across windows in increasing distance from the focal sub-window (dashed lined), separated by sub-window size. B. Comparison of detection of copy number variants between other CNV detectors, pure coverage estimations and using dudeML for varying parameters. Detection rate decreases across all methods with decreasing coverage, with increasing pool sizes and increasing repetitive content of the focal region. Note that iCopyDAV requires a certain level of coverage to function correctly, so was not used below onefold coverage." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1840640188" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;A. Relative contribution of each statistic to the classification of copy number variants, across windows in increasing distance from the focal sub-window (dashed lined), separated by sub-window size. B. Comparison of detection of copy number variants between other CNV detectors, pure coverage estimations and using dudeML for varying parameters. Detection rate decreases across all methods with decreasing coverage, with increasing pool sizes and increasing repetitive content of the focal region. Note that iCopyDAV requires a certain level of coverage to function correctly, so was not used below onefold coverage.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/11/3575/F2.medium.gif" width="440" height="412"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/9/11/3575/F2.medium.gif" width="440" height="412"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/11/3575/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/11/3575/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494794" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-21" class="first-child">A. Relative contribution of each statistic to the classification of copy number variants, across windows in increasing distance from the focal sub-window (dashed lined), separated by sub-window size. B. Comparison of detection of copy number variants between other CNV detectors, pure coverage estimations and using dudeML for varying parameters. Detection rate decreases across all methods with decreasing coverage, with increasing pool sizes and increasing repetitive content of the focal region. Note that iCopyDAV requires a certain level of coverage to function correctly, so was not used below onefold coverage.</p><div class="sb-div caption-clear"></div></div></div><p id="p-22">We also compared different supervisor machine learning classifiers and found no significant differences between most classifiers (Figure S4), though the most successful classifier on simulated data were a Random Forest Classifier (true-positive rate ∼ classifier GLM t-value = 0.758, p-value = 0.765), with no difference in results between 100 and 500 estimators (<a id="xref-ref-26-7" class="xref-bibr" href="#ref-26">Pedregosa <em>et al.</em> 2011</a>). We incorporated all classifiers into dudeML as they may be more successful in conditions not tested here.</p></div><div id="sec-9" class="subsection"><h3>CNV machine learning classifiers are relatively agnostic to coverage and can detect CNVs in pooled data with high accuracy</h3><p id="p-23">For simulated data of varying coverages containing known CNVs, we compared dudeML to the prediction of a CNV based on copy number alone (rounding the median coverage of the sub-window to the nearest whole value, divided by the average coverage of the chromosome), or other CNV detection software: Delly (<a id="xref-ref-30-5" class="xref-bibr" href="#ref-30">Rausch <em>et al.</em> 2012</a>), CNVnator (<a id="xref-ref-1-5" class="xref-bibr" href="#ref-1">Abyzov <em>et al.</em> 2011</a>), iCopyDAV (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Dharanipragada <em>et al.</em> 2018</a>) and Pindel (<a id="xref-ref-44-6" class="xref-bibr" href="#ref-44">Ye <em>et al.</em> 2009</a>). All methods appear to have decreasing rates of CNV detection as coverage decreases (<a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2B</a>, true-positive rate ∼ coverage: GLM t-value = 3.090, p-value = 0.0029), with dudeML having a high rate of success (<a id="xref-fig-2-7" class="xref-fig" href="#F2">Figure 2B</a>), decreasing at a similar rate to Delly and CNVnator (<a id="xref-fig-2-8" class="xref-fig" href="#F2">Figure 2B</a>, true-positive rate ∼ coverage * dudeML: GLM t-value = 1.001, p-value = 0.329), but having a high rate of success at drastically low coverage (0.1x). As most CNV detection programs rely on split-mapped reads of certain mapping orientations to detect copy number variants, low coverage data likely lacks a sufficient abundance of these reads for the correct detection of CNVs (<a id="xref-ref-44-7" class="xref-bibr" href="#ref-44">Ye <em>et al.</em> 2009</a>; <a id="xref-ref-1-6" class="xref-bibr" href="#ref-1">Abyzov <em>et al.</em> 2011</a>; <a id="xref-ref-30-6" class="xref-bibr" href="#ref-30">Rausch <em>et al.</em> 2012</a>). Similarly, the spurious nature of data at low coverages prevents pure relative coverage comparisons from being useful. With machine learning however, if the training data are like the sampled data, the classifier relies on thousands of similar examples in each state to more reliably predict the presence or absence of a CNV. In fact, correctly predicting a CNV in data of decreasing coverage with a poorly optimized training set has a similar success rate as pure-coverage alone (Figure S3), highlighting the importance of a training set as similar to the true data as possible.</p><p id="p-24">We also investigated the detection of CNVs in genomic regions of increasing repetitive content using each method (using data at 20-fold coverage). CNVs found in highly repetitive regions are less likely to be correctly detected across all software formats (<a id="xref-fig-2-9" class="xref-fig" href="#F2">Figure 2B</a>, true-positive rate ∼repetitive portion of window: GLM t-value = -0.4657, p-value = 1.54e-05), though dudeML factoring in repetitive content appears to have relatively (though not significantly) higher success than most other methods (true-positive rate ∼repetitive portion of window * dudeML: GLM t-value = 0.1328, p-value = 0.18639).</p><p id="p-25">Often, populations are sequenced as pools of individuals instead of individually prepared samples because the approach reduces cost while still providing relatively high power for population genetic inference (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Schlötterer <em>et al.</em> 2014</a>). To asses dudeML’s ability to detect the correct number of copies of a gene in a population, we simulated CNVs at varying frequencies throughout pools of chromosomes. We generated simulated pools as both test data and training sets of 1 (haploid or inbred), 2 (diploid, 50% coverage), 5, 10, 20 and 40 chromosomes (pools at onefold coverage for each chromosome), again, we compared this to each other tools ability to detect the CNV and relative coverage estimates. As expected, as pool size increased the true positive rate decreased in all methods (<a id="xref-fig-2-10" class="xref-fig" href="#F2">Figure 2</a>, true-positive rate ∼pool size: GLM t-value = -4.883, p-value = 1.20e-05). All other forms of detection decreased in detection rate at a much faster rate than dudeML in both forms (true-positive rate ∼pool size * dudeML: GLM t-value &gt; 2.414, p-value &lt; 0.019660).</p></div><div id="sec-10" class="subsection"><h3>CNV machine learning classifiers can be optimized to drastically improve the quality of CNV detection, and should consider repetitive content and sample coverage</h3><p id="p-26">We next tested the extent that changing different parameters affected dudeML’s ability to correctly detect CNVs. We examined the effects of decreasing coverage, increasing sub-window size, increasing pool size and increasing the number of sub-windows on correctly classifying CNVs with dudeML, when including repetitive regions or ignoring them (Figures S4 &amp; S5). As dudeML classifies windows as containing a CNV or not, while other classifiers call CNVs, we found both the true-positive rate for windows and for correct CNV calls.</p><p id="p-27">As sub-window-size increases, the false-negative rate increases in the repeat masked sample, due to more windows overlapping with repetitive regions, excluding these windows from the analysis. In the repetitive sample, the false-positive rate increases as sub-window-size decreases due to more spurious coverage in the focal window affecting the detection of CNVs. Similar effects are seen when increasing the number of sub-windows in the repeat-masked sample, as smaller sub-window numbers with stochastic coverage can lead to more false-positives, while larger sub-window numbers have more windows ignored by the classifier due to overlaps with repeat content (Figures S4 &amp; S5). We also tested classifiers of different coverages and sub-window sizes against samples with an incorrect/poorly optimized training set. We find that as training sets diverge in similarity from the actual data, the true-positive rate decreases dramatically (Figure S3, true-positive rate ∼ absolute(log10(classifier coverage) – log10(sample coverage)): GLM t-value = -14.147, p-value = 7.36e-32), giving the classifier a success rate like pure-coverage alone (<a id="xref-fig-2-11" class="xref-fig" href="#F2">Figure 2</a>, Figure S1), highlighting the importance of a training set as like the true data as possible.</p></div><div id="sec-11" class="subsection"><h3>dudeML also accurately predicts copy number of a gene target region compared to pure coverage, for copy number estimation of genes or viruses in a sample</h3><p id="p-28">We sought to examine further applications of dudeML. Alongside predicting if a window contains a CNV or not, dudeML also predicts the copy number of a window. We examined how well dudeML predicts copy numbers compared to copy number estimation from coverage alone (window coverage/median coverage) and compared to the predictions from a loess regression. As copy number increases, the success rate of all methods decreases (Figure S6, true copy number proportion ∼ copy number: GLM t-value = -10.639 p-value = 4.87e-12). At all copy numbers, dudeML has much higher true-positive rate and accuracy in predicting copy number (Figure S6, true copy number proportion ∼ copy number + dudeML: GLM t-value = 6.146 p-value = 7.12e-07), with success rate and accuracy decreasing much more slowly as copy number increases (Figure S6, true copy number proportion ∼ copy number * dudeML: GLM t-value = 2.616, p-value = 0.0134). In fact, even when incorrect, the copy number estimate of dudeML is much closer to the actual copy number than other methods (Figure S6). The success of dudeML in predicting copy number suggests it could be used to predict the copy number of genes with high variation in copy number, viral genome copies relative to host genome copy number, or copy number of RNA viruses relative to a housekeeping gene. Additionally, though better tools are available (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Rahman <em>et al.</em> 2015</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Nelson <em>et al.</em> 2017</a>) to detect transposable element copy numbers, dudeML could be used for this purpose. dudeML could even be used for B chromosome copy number estimation, though may have issues depending on the window-size and the size of the usually poorly assembled B-linked contigs.</p></div><div id="sec-12" class="subsection"><h3>Resampling increases CNV machine learning classifier accuracy</h3><p id="p-29">To further tune the accuracy of our classifier, we tested its effectiveness on the detection of copy number variants in real data, as opposed to simulated copy number variants in simulated reads (though with a classifier still using simulated CNVs and simulated data for training). We therefore downloaded two <em>Drosophila melanogaster</em> reference genomes – both assembled with long-read data – with identified duplications and deletions relative to each other (A4 and Iso-1) (<a id="xref-ref-7-6" class="xref-bibr" href="#ref-7">Chakraborty <em>et al.</em> 2018</a>). When data from one reference is mapped to the other, regions with copy number variants show signatures of changes in standardized coverage and standard deviation as seen in simulated data (<a id="xref-fig-1-7" class="xref-fig" href="#F1">Figure 1</a>, Figure S7, Data S1). These real datasets slightly differ in their important features from the simulated data, in that the standardized mean and median hold slightly less importance, while standard deviation has slightly more importance in CNV prediction (Figure S7). However, this is not significantly different from the training sets generated with simulations (feature importance ∼ feature * real: GLM t-value = 0.331, p-value = 0.652).</p><p id="p-30">As before we trained the classifier based on mean and median coverage, interquartile range and standard deviation of coverage and the number of improper pairs, for windows containing simulated CNVs and standard regions. We performed this analysis with and without repetitive regions in the simulated training set and the real data. We then predicted windows with duplications or deletions using a random forest approach (<a id="xref-ref-26-8" class="xref-bibr" href="#ref-26">Pedregosa <em>et al.</em> 2011</a>).</p><p id="p-31">Strangely, and unseen in simulated examples, the proportion of false-positives was extremely high, with orders of magnitudes more false-positives compared to true-positives (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). This number of false-positives is reduced by filtering out CNVs with low predicted probabilities (less than 95% probability), but still produced a similar number of false-positives to true-positives (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>, Data S3).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/494661/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/494661/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">The number of predicted copy number variants in each strain (relative to the alternate strain), compared to previously identified copy-number variants (Chakraborty <em>et al.</em> 2018), across differing numbers of bootstraps and probability cutoffs, including the true-positive rate (TPR) for each category. The total count of previously identified CNVs is also included in the table. Table contains CNVs called in repeat masked samples with repeat masked training set, therefore CNVs overlapping with TEs have been removed, all other permutations of CNV calls, with and without repeats and with differing numbers of bootstraps and filtering, can be found in Data S3</span><div class="sb-div caption-clear"></div></div></div><p id="p-32">We suspected that artifacts and false CNVs may be caused by real structural variants and areas with inconsistent mapping rates that went undetected in the original simulated training set, consistent with this, twice as many false-positive CNVs are called when repetitive regions are included in both the samples and training sets (true-positive rate ∼ TE: GLM t-value = -0.279, p-value = 0.781, <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>, Data S3). We attempted to control for this by resampling across multiple training sets with independently generated CNVs. We generated 100 independent training sets across both the Iso-1 and A4 reference genomes to create 100 independent classifiers. Following this we performed a bootstrapping-like approach, predicting the copy number of each window 100 times using the 100 independent classifiers and taking the consensus of these calls. As the number of replicates increased, the false-positive rate drops dramatically with little effect on the true-positive rate (FPR ∼ replicates: GLM t-value = -5.309, p-value = 4.40e-06, <a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>, <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>, Data S3). As the probability value for CNV increases, the false-positive rate also decreases (FPR ∼ replicates: GLM t-value = -8.506, p-value = 1.65e-10). This did however remove some real duplications, so provides a conservative set of CNVs (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a>) (<a id="xref-ref-7-7" class="xref-bibr" href="#ref-7">Chakraborty <em>et al.</em> 2018</a>). This suggests that multiple independent training sets can remove any artifacts found in a single training set which may lead to false calls (<a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>, <a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/11/3575/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Number of CNVs detected in Drosophila melanogaster strains with known CNVs relative to each other after 100 bootstraps, for both samples including and excluding repeats. Note that the sample without repeats also include CNVs within masked regions (which we did not survey) as false-negatives. CNVs are grouped by their previously known detection in these strains (&#x2018;True-Positive&#x2019;), their lack of knowledge in these strains (&#x2018;False-Positive&#x2019;) and if known CNVs were missed (&#x2018;False-Negative&#x2019;). CNVs are labeled based on the confidence in the calls, the proportion of bootstraps confirming them." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1840640188" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Number of CNVs detected in &lt;em&gt;Drosophila melanogaster&lt;/em&gt; strains with known CNVs relative to each other after 100 bootstraps, for both samples including and excluding repeats. Note that the sample without repeats also include CNVs within masked regions (which we did not survey) as false-negatives. CNVs are grouped by their previously known detection in these strains (&#x2018;True-Positive&#x2019;), their lack of knowledge in these strains (&#x2018;False-Positive&#x2019;) and if known CNVs were missed (&#x2018;False-Negative&#x2019;). CNVs are labeled based on the confidence in the calls, the proportion of bootstraps confirming them.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/11/3575/F3.medium.gif" width="440" height="331"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/9/11/3575/F3.medium.gif" width="440" height="331"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/11/3575/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/11/3575/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494715" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-33" class="first-child">Number of CNVs detected in <em>Drosophila melanogaster</em> strains with known CNVs relative to each other after 100 bootstraps, for both samples including and excluding repeats. Note that the sample without repeats also include CNVs within masked regions (which we did not survey) as false-negatives. CNVs are grouped by their previously known detection in these strains (‘True-Positive’), their lack of knowledge in these strains (‘False-Positive’) and if known CNVs were missed (‘False-Negative’). CNVs are labeled based on the confidence in the calls, the proportion of bootstraps confirming them.</p><div class="sb-div caption-clear"></div></div></div><p id="p-34">Interestingly, including repetitive regions in the training set increases the detection rate of high confidence false-positives, and decreases the confidence in true-positive CNV calls (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3</a>), suggesting that even if repetitive regions are screened using dudeML, repetitive regions included in the training set results in decreased confidence in CNV calls (Figure S8).</p><p id="p-35">As so many false-positives are found with high confidence across both samples, we next visually inspected the regions of the genome called as false-positive CNVs in at least 95 of 100 bootstraps (Figure S8): 99 duplications (23% of duplications called) and 59 deletions (15% of deletions called) across both strains). We extracted long reads (&gt; 250bp) from PacBio data for both strains and mapped these to the opposite strain’s genome, which we then visualized in the integrative genomics viewer (<a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Robinson <em>et al.</em> 2011</a>). All false-positive CNVs examined show similar signatures to true-positive copy numbers (<em>e.g.</em>, split-mapped reads across regions of 0 coverage for deletions, and supplementary alignments of reads in regions of high coverage for duplications), suggesting that they are in fact real CNVs and not false-positives (18 examples given in Data S1). We further PCR validated 12 of these CNVs, chosen at random (Figure S8 &amp; S9, Data S2). While we could validate all deletions, we found no length variation in PCR product for putative duplications for primers designed outside the duplication, which suggests if these duplications exist they may not be tandem duplications (which would produce a longer or laddered PCR product) and instead are <em>trans</em> duplications. Logically this would fit with the absence of these CNVs in the previous survey which searched for tandem duplications specifically (<a id="xref-ref-7-8" class="xref-bibr" href="#ref-7">Chakraborty <em>et al.</em> 2018</a>), while dudeML identifies duplications primarily based on coverage-based statistics and so is agnostic to <em>cis vs. trans</em> duplications. However, the lack of PCR verification could support these duplications as segregating within the originally sequenced line or are false-positives.</p><p id="p-36">Based on these results, bootstrapping (and filtering low probability calls) appears to average over random effects of simulated training sets to improve the accuracy of CNV calls, removing a majority of false-positive CNVs called while not majorly affecting the true-positives (in fact improving their calls in some cases), allowing a more conservative assessment of the copy number variants found throughout an assessed strain. Most high confidence false-positives also appear to be actual CNVs, suggesting that dudeML can detect CNVs other tools miss – even using long read data.</p></div><div id="sec-13" class="subsection"><h3>Conclusion</h3><p id="p-37">In summary, we have shown that machine learning classifiers, such as dudeML, perform quite well at detecting copy number variants in comparison to other methods, particularly in samples with reduced coverage or in pools, using statistics easily derived from the sample. These tools are not computationally intensive and can be used across many datasets to detect duplications and deletions for numerous purposes. We expect machine learning to provide powerful tools for bioinformatic use in the future.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-38">We thank Justin Blumenstiel, Joanne Chapman, Mark Holder, John Kelly, Maria Orive, Daniel Schrider and Carolyn Wessinger for their input in designing the tool, for discussion on machine learning methods and for comments on the manuscript. We also thank two anonymous reviewers for their input and helpful comments on honing and improving the tool. This work was supported by a K-INBRE postdoctoral grant to TH (NIH Grant P20 GM103418) and by NIH Grants R00 GM114714 and R01 AI139154 to RLU.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-39">Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.9735890">https://doi.org/10.25387/g3.9735890</a>.</p></li><li class="fn" id="fn-2"><p id="p-40"><em>Communicating editor: A. Kern</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2019-05-31"><span class="received-label">Received </span>May 31, 2019.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2019-08-16"><span class="accepted-label">Accepted </span>August 16, 2019.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2019 Hill, Unckless</li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3575.1" data-doi="10.1101/gr.114876.110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Abyzov</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. E.</span> <span class="cit-name-surname">Urban</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Snyder</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Gerstein</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">CNVnator: an approach to discover, genotype, and characterize typical and atypical CNVs from family and population genome sequencing.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">974</span>–<span class="cit-lpage">984</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.114876.110</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.114876.110%26rft_id%253Dinfo%253Apmid%252F21324876%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjIxLzYvOTc0IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMS8zNTc1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3575.2" data-doi="10.1146/annurev-ecolsys-102209-144621"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Beaumont</span>,  <span class="cit-name-given-names">M. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Approximate Bayesian Computation in Evolution and Ecology.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Ecol. Evol. Syst.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">379</span>–<span class="cit-lpage">406</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1146/annurev-ecolsys-102209-144621</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnu.%2BRev.%2BEcol.%2BEvol.%2BSyst.%26rft.volume%253D41%26rft.spage%253D379%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-ecolsys-102209-144621%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev-ecolsys-102209-144621&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000284978200018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3575.3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Beaumont</span>,  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Zhang</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. J.</span> <span class="cit-name-surname">Balding</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Approximate Bayesian Computation in Population Genetics.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">162</span>: <span class="cit-fpage">2025</span>–<span class="cit-lpage">2035</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DBeaumont%26rft.auinit1%253DM.%2BA.%26rft.volume%253D162%26rft.issue%253D4%26rft.spage%253D2025%26rft.epage%253D2035%26rft.atitle%253DApproximate%2BBayesian%2BComputation%2Bin%2BPopulation%2BGenetics%26rft_id%253Dinfo%253Apmid%252F12524368%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2Mi80LzIwMjUiO3M6NDoiYXRvbSI7czoxOToiL2dnZy85LzExLzM1NzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3575.4" data-doi="10.1371/journal.pcbi.1003537"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bouckaert</span>,  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Heled</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Kühnert</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Vaughan</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. H.</span> <span class="cit-name-surname">Wu</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">BEAST 2: A Software Platform for Bayesian Evolutionary Analysis.</span> <abbr class="cit-jnl-abbrev">PLOS Comput. Biol.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">e1003537</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pcbi.1003537</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLOS%2BComput.%2BBiol.%26rft.volume%253D10%26rft.spage%253D1003537e%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.1003537%26rft_id%253Dinfo%253Apmid%252F24722319%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.1003537&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24722319&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3575.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3575.5" data-doi="10.1016/j.chemolab.2005.05.004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>,  <span class="cit-name-given-names">C. D.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">H. T.</span> <span class="cit-name-surname">Davis</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Receiver operating characteristics curves and related decision measures: A tutorial.</span> <abbr class="cit-jnl-abbrev">Chemom. Intell. Lab. Syst.</abbr> <span class="cit-vol">80</span>: <span class="cit-fpage">24</span>–<span class="cit-lpage">38</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.chemolab.2005.05.004</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DChemom.%2BIntell.%2BLab.%2BSyst.%26rft.volume%253D80%26rft.spage%253D24%26rft.atitle%253DRECEIVER%2BOPERATING%2BCHARACTERISTICS%2BCURVES%2BAND%2BRELATED%2BDECISION%2BMEASURES%253A%2BA%2BTUTORIAL%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.chemolab.2005.05.004%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.chemolab.2005.05.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3575.6" data-doi="10.1101/gr.199323.115"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cardoso-Moreira</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. R.</span> <span class="cit-name-surname">Arguello</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Gottipati</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L. G.</span> <span class="cit-name-surname">Harshman</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. K.</span> <span class="cit-name-surname">Grenier</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Evidence for the fixation of gene duplications by positive selection in <em>Drosophila</em>.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">787</span>–<span class="cit-lpage">798</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.199323.115</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.199323.115%26rft_id%253Dinfo%253Apmid%252F27197209%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjI2LzYvNzg3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMS8zNTc1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3575.7" data-doi="10.1038/s41588-017-0010-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chakraborty</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N. W.</span> <span class="cit-name-surname">VanKuren</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Zhao</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Zhang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Kalsow</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Hidden genetic variation shapes the structure of functional elements in <em>Drosophila</em>.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">50</span>: <span class="cit-fpage">20</span>–<span class="cit-lpage">25</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/s41588-017-0010-y</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D50%26rft.spage%253D20%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41588-017-0010-y%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41588-017-0010-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3575.8" data-doi="10.1093/bioinformatics/btv710"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">O.</span> <span class="cit-name-surname">Schulz-Trieglaff</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Shaw</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Barnes</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Schlesinger</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Manta: Rapid detection of structural variants and indels for germline and cancer sequencing applications.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">1220</span>–<span class="cit-lpage">1222</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btv710</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtv710%26rft_id%253Dinfo%253Apmid%252F26647377%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btv710&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26647377&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3575.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3575.9" data-doi="10.1371/journal.pone.0195334"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dharanipragada</span>,  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Vogeti</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Parekh</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">iCopyDAV: Integrated platform for copy number variations-Detection, annotation and visualization.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">e0195334</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0195334</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D13%26rft.spage%253De0195334%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0195334%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0195334&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3575.10" data-doi="10.1093/nar/gku1099"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">dos Santos</span>,  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. J.</span> <span class="cit-name-surname">Schroeder</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. L.</span> <span class="cit-name-surname">Goodman</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">V. B.</span> <span class="cit-name-surname">Strelets</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. A.</span> <span class="cit-name-surname">Crosby</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">FlyBase: Introduction of the <em>Drosophila melanogaster</em> Release 6 reference genome assembly and large-scale migration of genome annotations.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">D690</span>–<span class="cit-lpage">D697</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gku1099</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgku1099%26rft_id%253Dinfo%253Apmid%252F25398896%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gku1099&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25398896&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3575.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3575.11" data-doi="10.1016/j.meegid.2017.11.013"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hill</span>,  <span class="cit-name-given-names">T.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. L.</span> <span class="cit-name-surname">Unckless</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">The dynamic evolution of Drosophila innubila Nudivirus.</span> <abbr class="cit-jnl-abbrev">Infect. Genet. Evol.</abbr> <span class="cit-vol">57</span>: <span class="cit-fpage">151</span>–<span class="cit-lpage">157</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.meegid.2017.11.013</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInfect.%2BGenet.%2BEvol.%26rft.volume%253D57%26rft.spage%253D151%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.meegid.2017.11.013%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.meegid.2017.11.013&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3575.12" data-doi="10.1146/annurev.genom.3.032802.120023"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Inoue</span>,  <span class="cit-name-given-names">K.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. R.</span> <span class="cit-name-surname">Lupski</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Molecular Mechanisms for Genomic Disorders.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Genomics Hum. Genet.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">199</span>–<span class="cit-lpage">242</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1146/annurev.genom.3.032802.120023</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnu.%2BRev.%2BGenomics%2BHum.%2BGenet.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.genom.3.032802.120023%26rft_id%253Dinfo%253Apmid%252F12142364%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.genom.3.032802.120023&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12142364&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3575.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000178678800009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3575.13" data-doi="10.1101/gr.3677206"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Freeman</span>,  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G. H.</span> <span class="cit-name-surname">Perry</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Feuk</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Redon</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. A.</span> <span class="cit-name-surname">McCarroll</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Copy Number Variation: New Insights in Genome Diversity.</span> <abbr class="cit-jnl-abbrev">Genome Res. </abbr><span class="cit-vol">16</span>: <span class="cit-fpage">949</span>–<span class="cit-lpage">961</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.3677206</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%2B%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.3677206%26rft_id%253Dinfo%253Apmid%252F16809666%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE2LzgvOTQ5IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMS8zNTc1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3575.14" data-doi="10.1371/journal.pgen.1000198"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jensen</span>,  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. R.</span> <span class="cit-name-surname">Thornton</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Andolfatto</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">An approximate bayesian estimator suggests strong, recurrent selective sweeps in <em>Drosophila</em>.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e1000198</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1000198</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DJensen%26rft.auinit1%253DJ.%2BD.%26rft.volume%253D4%26rft.issue%253D9%26rft.spage%253De1000198%26rft.epage%253De1000198%26rft.atitle%253DAn%2Bapproximate%2Bbayesian%2Bestimator%2Bsuggests%2Bstrong%252C%2Brecurrent%2Bselective%2Bsweeps%2Bin%2BDrosophila.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000198%26rft_id%253Dinfo%253Apmid%252F18802463%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000198&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18802463&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3575.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3575.15"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kern</span>,  <span class="cit-name-given-names">A. D.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. R.</span> <span class="cit-name-surname">Schrider</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> diploS/HIC: An Updated Approach to Classifying Selective Sweeps. G3: (Bethesda)8<strong>:</strong> 1959–1970.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3575.16" data-doi="10.1126/science.1149504"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Korbel</span>,  <span class="cit-name-given-names">J. O.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. E.</span> <span class="cit-name-surname">Urban</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. P.</span> <span class="cit-name-surname">Affourtit</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Godwin</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Grubert</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Paired-End Mapping Reveals Extensive Structural Variation in the Human Genome.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">318</span>: <span class="cit-fpage">420</span>–<span class="cit-lpage">426</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.1149504</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1149504%26rft_id%253Dinfo%253Apmid%252F17901297%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMTgvNTg0OS80MjAiO3M6NDoiYXRvbSI7czoxOToiL2dnZy85LzExLzM1NzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3575.17" data-doi="10.1093/bib/bbk007"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Larrañaga</span>,  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Calvo</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Santana</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Bielza</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Galdiano</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Machine learning in bioinformatics.</span> <abbr class="cit-jnl-abbrev">Brief. Bioinform.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">86</span>–<span class="cit-lpage">112</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bib/bbk007</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBrief.%2BBioinform.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbib%252Fbbk007%26rft_id%253Dinfo%253Apmid%252F16761367%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bib/bbk007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16761367&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3575.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000240930700007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3575.18" data-doi="10.1186/gb-2014-15-6-r84"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Layer</span>,  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Chiang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. R.</span> <span class="cit-name-surname">Quinlan</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">I. M.</span> <span class="cit-name-surname">Hall</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">LUMPY: a probabilistic framework for structural variant discovery.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">R84</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/gb-2014-15-6-r84</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D15%26rft.spage%253DR84%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2014-15-6-r84%26rft_id%253Dinfo%253Apmid%252F24970577%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2014-15-6-r84&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24970577&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3575.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-book" id="cit-9.11.3575.19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-source">WGsim - Read simulator for next generation sequencing</span>, <span class="cit-publ-name">GitHub Repository</span>. <a href="http://github.com/lh3/wgsim">http://github.com/lh3/wgsim</a></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3575.20" data-doi="10.1093/bioinformatics/btp324"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">H.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Durbin</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Fast and accurate short read alignment with Burrows-Wheeler transform.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1754</span>–<span class="cit-lpage">1760</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btp324</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp324%26rft_id%253Dinfo%253Apmid%252F19451168%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp324&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19451168&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3575.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000267665900006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3575.21" data-doi="10.1093/bioinformatics/btp352"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Handsaker</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Wysoker</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Fennell</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Ruan</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The sequence alignment/map format and SAMtools.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">2078</span>–<span class="cit-lpage">2079</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btp352</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp352%26rft_id%253Dinfo%253Apmid%252F19505943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp352&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19505943&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3575.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268808600014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3575.22" data-doi="10.1177/154405910408300516"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liaw</span>,  <span class="cit-name-given-names">A.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Wiener</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Classification and Regression by Random Forest.</span> <abbr class="cit-jnl-abbrev">R News</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">18</span>–<span class="cit-lpage">22</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DR%2BNews%26rft.volume%253D2%26rft.spage%253D18%26rft_id%253Dinfo%253Adoi%252F10.1177%252F154405910408300516%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1177/154405910408300516&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3575.23" data-doi="10.1038/nrg3920"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Libbrecht</span>,  <span class="cit-name-given-names">M. W.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">W. S.</span> <span class="cit-name-surname">Noble</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Machine learning applications in genetics and genomics.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">321</span>–<span class="cit-lpage">332</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg3920</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BGenet.%26rft.volume%253D16%26rft.spage%253D321%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3920%26rft_id%253Dinfo%253Apmid%252F25948244%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3920&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25948244&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3575.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3575.24" data-doi="10.1534/g3.117.043893"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nelson</span>,  <span class="cit-name-given-names">M. G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. S.</span> <span class="cit-name-surname">Linheiro</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. M.</span> <span class="cit-name-surname">Bergman</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">McClintock: An Integrated Pipeline for Detecting Transposable Element Insertions in Whole-Genome Shotgun Sequencing Data.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">2763</span>–<span class="cit-lpage">2778</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.117.043893</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.117.043893%26rft_id%253Dinfo%253Apmid%252F28637810%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjg6IjcvOC8yNzYzIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMS8zNTc1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3575.25" data-doi="10.1111/1755-0998.12297"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Palmieri</span>,  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Nolte</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Chen</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Schlötterer</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genome assembly and annotation of a <em>Drosophila simulans</em> strain from Madagascar.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol. Resour.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">372</span>–<span class="cit-lpage">381</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/1755-0998.12297</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%2BResour.%26rft.volume%253D15%26rft.spage%253D372%26rft_id%253Dinfo%253Adoi%252F10.1111%252F1755-0998.12297%26rft_id%253Dinfo%253Apmid%252F24961367%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/1755-0998.12297&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24961367&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3575.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3575.26"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pedregosa</span>,  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Weiss</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Brucher</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Scikit-learn</span>. <abbr class="cit-jnl-abbrev">Machine Learning in Python.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">2825</span>–<span class="cit-lpage">2830</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMachine%2BLearning%2Bin%2BPython.%26rft.volume%253D12%26rft.spage%253D2825%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3575.27" data-doi="10.1093/bioinformatics/btq033"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Quinlan</span>,  <span class="cit-name-given-names">A. R.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">I. M.</span> <span class="cit-name-surname">Hall</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">BEDTools: a flexible suite of utilities for comparing genomic features.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">841</span>–<span class="cit-lpage">842</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btq033</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq033%26rft_id%253Dinfo%253Apmid%252F20110278%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btq033&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20110278&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3575.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000275243500019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3575.28" data-doi="10.1093/nar/gkv1193"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rahman</span>,  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.-W.</span> <span class="cit-name-surname">Chirn</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Kanodia</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y. A.</span> <span class="cit-name-surname">Sytnikova</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Brembs</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Unique transposon landscapes are pervasive across <em>Drosophila melanogaster</em> genomes.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">10655</span>–<span class="cit-lpage">10672</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkv1193</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkv1193%26rft_id%253Dinfo%253Apmid%252F26578579%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkv1193&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26578579&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3575.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3575.29" data-doi="10.1186/1471-2148-5-28"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rastogi</span>,  <span class="cit-name-given-names">S.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Liberles</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Subfunctionalization of duplicated genes as a transition state to neofunctionalization.</span> <abbr class="cit-jnl-abbrev">BMC Evol. Biol.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">28</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2148-5-28</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bevolutionary%2Bbiology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BEvol%2BBiol%26rft.aulast%253DRastogi%26rft.auinit1%253DS.%26rft.volume%253D5%26rft.spage%253D28%26rft.epage%253D28%26rft.atitle%253DSubfunctionalization%2Bof%2Bduplicated%2Bgenes%2Bas%2Ba%2Btransition%2Bstate%2Bto%2Bneofunctionalization.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2148-5-28%26rft_id%253Dinfo%253Apmid%252F15831095%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2148-5-28&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15831095&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3575.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3575.30" data-doi="10.1093/bioinformatics/bts378"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rausch</span>,  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Zichner</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Schlattl</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. M.</span> <span class="cit-name-surname">Stutz</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Benes</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">DELLY: structural variant discovery by integrated paired-end and split-read analysis.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">i333</span>–<span class="cit-lpage">i339</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/bts378</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbts378%26rft_id%253Dinfo%253Apmid%252F22962449%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bts378&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22962449&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3575.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000308532300005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3575.31" data-doi="10.1038/nature05329"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Redon</span>,  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Ishikawa</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. R.</span> <span class="cit-name-surname">Fitch</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Feuk</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G. H.</span> <span class="cit-name-surname">Perry</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Global variation in copy number in the human genome.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">444</span>: <span class="cit-fpage">444</span>–<span class="cit-lpage">454</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature05329</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DRedon%26rft.auinit1%253DR.%26rft.volume%253D444%26rft.issue%253D7118%26rft.spage%253D444%26rft.epage%253D454%26rft.atitle%253DGlobal%2Bvariation%2Bin%2Bcopy%2Bnumber%2Bin%2Bthe%2Bhuman%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature05329%26rft_id%253Dinfo%253Apmid%252F17122850%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature05329&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17122850&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3575.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000242215700038&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3575.32" data-doi="10.1038/nbt0111-24"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Robinson</span>,  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Thorvaldsdottir</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W. W.</span> <span class="cit-name-surname">Inckler</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Guttman</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. S.</span> <span class="cit-name-surname">Lander</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Integrative genomics viewer.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">24</span>–<span class="cit-lpage">26</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D29%26rft.spage%253D24%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt0111-24%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt0111-24&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000286048900013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3575.33" data-doi="10.1093/molbev/msu124"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rogers</span>,  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">Cridland</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Shao</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. T.</span> <span class="cit-name-surname">Hu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Andolfatto</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Landscape of Standing Variation for Tandem Duplications in <em>Drosophila yakuba</em> and <em>Drosophila simulans</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">1750</span>–<span class="cit-lpage">1766</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/msu124</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsu124%26rft_id%253Dinfo%253Apmid%252F24710518%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msu124&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24710518&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3575.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000339423800011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3575.34" data-doi="10.1038/nrg3803"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schlötterer</span>,  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Tobler</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Kofler</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Nolte</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Sequencing pools of individuals - mining genome-wide polymorphism data without big funding.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">749</span>–<span class="cit-lpage">763</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg3803</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BGenet.%26rft.volume%253D15%26rft.spage%253D749%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3803%26rft_id%253Dinfo%253Apmid%252F25246196%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3803&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25246196&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3575.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3575.35" data-doi="10.1371/journal.pgen.1007341"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schrider</span>,  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Ayroles</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. R.</span> <span class="cit-name-surname">Matute</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. D.</span> <span class="cit-name-surname">Kern</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Supervised machine learning reveals introgressed loci in the genomes of <em>Drosophila simulans</em> and <em>D. sechellia</em>.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">e1007341</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1007341</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D14%26rft.spage%253De1007341%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1007341%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1007341&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3575.36" data-doi="10.1534/genetics.113.151670"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schrider</span>,  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Houle</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Lynch</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. W.</span> <span class="cit-name-surname">Hahn</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Rates and genomic consequences of spontaneous mutational events in <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">194</span>: <span class="cit-fpage">937</span>–<span class="cit-lpage">954</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.113.151670</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.151670%26rft_id%253Dinfo%253Apmid%252F23733788%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk0LzQvOTM3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMS8zNTc1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3575.37" data-doi="10.1371/journal.pgen.1005928"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schrider</span>,  <span class="cit-name-given-names">D. R.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. D.</span> <span class="cit-name-surname">Kern</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">S/HIC: Robust Identification of Soft and Hard Sweeps Using Machine Learning.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">e1005928</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1005928</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D12%26rft.spage%253De1005928%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1005928%26rft_id%253Dinfo%253Apmid%252F26977894%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1005928&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26977894&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3575.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3575.38" data-doi="10.1016/j.tig.2017.12.005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schrider</span>,  <span class="cit-name-given-names">D. R.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. D.</span> <span class="cit-name-surname">Kern</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Supervised Machine Learning for Population Genetics: A New Paradigm.</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">301</span>–<span class="cit-lpage">312</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.tig.2017.12.005</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BGenet.%26rft.volume%253D34%26rft.spage%253D301%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2017.12.005%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tig.2017.12.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3575.39" data-doi="10.1371/journal.pcbi.1004845"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sheehan</span>,  <span class="cit-name-given-names">S.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y. S.</span> <span class="cit-name-surname">Song</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Deep Learning for Population Genetic Inference.</span> <abbr class="cit-jnl-abbrev">PLOS Comput. Biol.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">e1004845</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pcbi.1004845</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLOS%2BComput.%2BBiol.%26rft.volume%253D12%26rft.spage%253De1004845%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.1004845%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.1004845&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-9.11.3575.40"><div class="cit-metadata unstructured">Smit, A. F. A., and R. Hubley, 2013–2015 RepeatMasker Open-4.0, pp. RepeatMasker.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3575.41"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sturtevant</span>,  <span class="cit-name-given-names">A. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1937</span> <span class="cit-article-title">The Bar Gene, a Duplication.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">83</span>: <span class="cit-fpage">210</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.volume%253D83%26rft.spage%253D210%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-book" id="cit-9.11.3575.42"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Team</span>,  <span class="cit-name-given-names">R. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-source"><em>R: A Language and Environment for Statistical Computing</em>, <em>pp</em></span>, <span class="cit-publ-name">R Foundation for Statistical Computing</span>, <span class="cit-publ-loc">Vienna, Austria</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3575.43" data-doi="10.1016/j.cub.2015.11.063"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Unckless</span>,  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">V. M.</span> <span class="cit-name-surname">Howick</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. P.</span> <span class="cit-name-surname">Lazzaro</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Convergent Balancing Selection on an Antimicrobial Peptide in <em>Drosophila</em>.</span> <abbr class="cit-jnl-abbrev">Curr. Biol.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">257</span>–<span class="cit-lpage">262</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.cub.2015.11.063</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BBiol.%26rft.volume%253D26%26rft.spage%253D257%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cub.2015.11.063%26rft_id%253Dinfo%253Apmid%252F26776733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cub.2015.11.063&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26776733&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3575.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3575.44" data-doi="10.1093/bioinformatics/btp394"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ye</span>,  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. H.</span> <span class="cit-name-surname">Schulz</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Q.</span> <span class="cit-name-surname">Long</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Apweiler</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Ning</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Pindel: a pattern growth approach to detect break points of large deletions and medium sized insertions from paired-end short reads.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">2865</span>–<span class="cit-lpage">2871</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btp394</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp394%26rft_id%253Dinfo%253Apmid%252F19561018%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp394&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19561018&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3575.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000271106800026&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3575.45" data-doi="10.1101/gr.142646.112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zichner</span>,  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. A.</span> <span class="cit-name-surname">Garfield</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Rausch</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. M.</span> <span class="cit-name-surname">Stutz</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Cannavo</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Impact of genomic structural variation in <em>Drosophila melanogaster</em> based on population-scale sequencing.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">568</span>–<span class="cit-lpage">579</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.142646.112</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.142646.112%26rft_id%253Dinfo%253Apmid%252F23222910%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjIzLzMvNTY4IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMS8zNTc1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/9/11/3575.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/11/3567" title="A Screen of the Conserved Kinome for Negative Regulators of LIN-12 Negative Regulatory Region (“NRR”)-Missense Activity in Caenorhabditis elegans" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/11/3583" title="Multiple Functions of the Essential Gene PpV in Drosophila Early Development" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/9/11"><span>Volume 9 Issue 11, November 2019</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/9/11.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/9/11.cover-source.jpg?itok=DjqkLGk9" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 9 (11)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/11/3575" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-1oEyq8LM9YukFfeEahD99CoFqFi7Y_gyfqQgS5Gfzc8" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/software-and-data-resources" class="" data-icon-position="" data-hide-link-title="0">Software and Data Resources</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/9/11/3575" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/9/11/3575" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/9/11/3575" class="active" data-icon-position="" data-hide-link-title="0">A Deep Learning Approach for Detecting Copy Number Variation in Next-Generation Sequencing Data</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/494621" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-zK2hk87va31BuwFZCkm6GGLLYzL7ss63a-7L9xSLGO8" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/11/3575.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/9/11/3575" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-qqmNQj6v5qHLlqXJA8NkFDabxUN6GcVfRHO_yxYZdyY" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/9/11/3575" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/11/3575.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494621" id="node494621--2" data-pisa="ggg;9/11/3575" data-pisa-master="ggg;g3.119.400596" data-apath="/ggg/9/11/3575.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Deep Learning Approach for Detecting Copy Number Variation in Next-Generation Sequencing Data</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-4661-6391" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Tom Hill</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-8586-7137" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Robert L. Unckless</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3575-3582;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400596 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494621--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Tom Hill </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>4055 Haworth Hall, The Department of Molecular Biosciences, University of Kansas, 1200 Sunnyside Avenue, Lawrence, KS 66045.</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tom%2BHill%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hill%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ATom%2BHill%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-4661-6391" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Tom Hill</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:tom.hill@ku.edu" class="" data-icon-position="" data-hide-link-title="0">tom.hill@ku.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Robert L. Unckless </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>4055 Haworth Hall, The Department of Molecular Biosciences, University of Kansas, 1200 Sunnyside Avenue, Lawrence, KS 66045.</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Robert%2BL.%2BUnckless%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Unckless%20RL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARobert%2BL.%2BUnckless%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-8586-7137" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Robert L. Unckless</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/9/11/3575" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/9/11/3575&amp;title=A%20Deep%20Learning%20Approach%20for%20Detecting%20Copy%20Number%20Variation%20in%20Next-Generation%20Sequencing%20Data" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/9/11/3575&amp;title=A%20Deep%20Learning%20Approach%20for%20Detecting%20Copy%20Number%20Variation%20in%20Next-Generation%20Sequencing%20Data" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/9/11/3575&amp;title=A%20Deep%20Learning%20Approach%20for%20Detecting%20Copy%20Number%20Variation%20in%20Next-Generation%20Sequencing%20Data" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/9/11/3575&amp;text=A%20Deep%20Learning%20Approach%20for%20Detecting%20Copy%20Number%20Variation%20in%20Next-Generation%20Sequencing%20Data" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/9/11/3575&amp;title=A%20Deep%20Learning%20Approach%20for%20Detecting%20Copy%20Number%20Variation%20in%20Next-Generation%20Sequencing%20Data" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/9/11/3575&amp;t=A%20Deep%20Learning%20Approach%20for%20Detecting%20Copy%20Number%20Variation%20in%20Next-Generation%20Sequencing%20Data" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/9/11/3575&amp;title=A%20Deep%20Learning%20Approach%20for%20Detecting%20Copy%20Number%20Variation%20in%20Next-Generation%20Sequencing%20Data" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/9/11/3575&amp;title=A%20Deep%20Learning%20Approach%20for%20Detecting%20Copy%20Number%20Variation%20in%20Next-Generation%20Sequencing%20Data" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/494621/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494621" id="node494621--3" data-pisa="ggg;9/11/3575" data-pisa-master="ggg;g3.119.400596" data-apath="/ggg/9/11/3575.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Deep Learning Approach for Detecting Copy Number Variation in Next-Generation Sequencing Data</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-4661-6391" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Tom Hill</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-8586-7137" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Robert L. Unckless</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3575-3582;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400596 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494621--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Tom Hill </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>4055 Haworth Hall, The Department of Molecular Biosciences, University of Kansas, 1200 Sunnyside Avenue, Lawrence, KS 66045.</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tom%2BHill%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hill%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ATom%2BHill%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-4661-6391" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Tom Hill</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:tom.hill@ku.edu" class="" data-icon-position="" data-hide-link-title="0">tom.hill@ku.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Robert L. Unckless </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>4055 Haworth Hall, The Department of Molecular Biosciences, University of Kansas, 1200 Sunnyside Avenue, Lawrence, KS 66045.</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Robert%2BL.%2BUnckless%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Unckless%20RL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARobert%2BL.%2BUnckless%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-8586-7137" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Robert L. Unckless</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=3575&amp;rft.epage=3582&amp;rft.atitle=A%20Deep%20Learning%20Approach%20for%20Detecting%20Copy%20Number%20Variation%20in%20Next-Generation%20Sequencing%20Data&amp;rft.volume=9&amp;rft.issue=11&amp;rft.date=2019-11-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Hill%2C+Tom&amp;rft.au=Unckless%2C+Robert+L."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/494621/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/494621/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/494621/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/494621/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/494621/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/494621/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/494621/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/494621/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/494621/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/494621/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/494621/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/494621/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496031" id="node496031" data-pisa="ggg;g3.120.401086v1" data-pisa-master="ggg;g3.120.401086" data-apath="/ggg/early/2020/03/09/g3.120.401086.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/09/g3.120.401086" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The UCSF Mouse Inventory Database Application, an Open Source Web App for Sharing Mutant Mice within a Research Community</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495860" id="node495860" data-pisa="ggg;10/3/925" data-pisa-master="ggg;g3.119.400983" data-apath="/ggg/10/3/925.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/925" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">An Automated Method To Predict Mouse Gene and Protein Sequences Using Variant Data</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495845" id="node495845" data-pisa="ggg;10/3/945" data-pisa-master="ggg;g3.119.400945" data-apath="/ggg/10/3/945.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/945" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">CGSEA: A Flexible Tool for Evaluating the Associations of Chemicals with Complex Diseases</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Software%20and%20Data%20Resources" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Software and Data Resources</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/9/11/3575" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Methods and Model</a></li><li class="odd"><a href="#sec-7" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results and Discussion</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/9/11/3575.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/9/11/3575.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/9/11/3575.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
