<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/9/11/3855" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/9/11/3855.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/9/11/3855.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/9/11/3855.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/9/11/3855/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/9/11/3855.atom" />
<meta name="HW.pisa" content="ggg;9/11/3855" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Efficient Estimation of Marker Effects in Plant Breeding" />
<meta name="DC.Identifier" content="10.1534/g3.119.400728" />
<meta name="DC.Date" content="2019-11-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2019 Xavier. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="The evaluation of prediction machines is an important step for a successful implementation of genomic-enabled selection in plant breeding. Computation time and predictive ability constitute key metrics to determine the methodology utilized for the consolidation of genomic prediction pipeline. This study introduces two methods designed to couple high prediction accuracy with efficient computational performance: 1) a non-MCMC method to estimate marker effects with a Laplace prior; and 2) an iterative framework that allows solving whole-genome regression within mixed models with replicated observations in a single-stage. The investigation provides insights on predictive ability and marker effect estimates. Various genomic prediction techniques are compared based on cross-validation, assessing predictions across and within family. Properties of quantitative trait loci detection and single-stage method were evaluated on simulated plot-level data from unbalanced data structures. Estimation of marker effects by the new model is compared to a genome-wide association analysis and whole-genome regression methods. The single-stage approach is compared to a GBLUP fitted via restricted maximum likelihood, and a two-stages approaches where genetic values fit a whole-genome regression. The proposed framework provided high computational efficiency, robust prediction across datasets, and accurate estimation of marker effects." />
<meta name="DC.Contributor" content="Alencar Xavier" />
<meta name="article:published_time" content="2019-11-01" />
<meta name="article:section" content="Genomic Prediction" />
<meta name="citation_title" content="Efficient Estimation of Marker Effects in Plant Breeding" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;The evaluation of prediction machines is an important step for a successful implementation of genomic-enabled selection in plant breeding. Computation time and predictive ability constitute key metrics to determine the methodology utilized for the consolidation of genomic prediction pipeline. This study introduces two methods designed to couple high prediction accuracy with efficient computational performance: 1) a non-MCMC method to estimate marker effects with a Laplace prior; and 2) an iterative framework that allows solving whole-genome regression within mixed models with replicated observations in a single-stage. The investigation provides insights on predictive ability and marker effect estimates. Various genomic prediction techniques are compared based on cross-validation, assessing predictions across and within family. Properties of quantitative trait loci detection and single-stage method were evaluated on simulated plot-level data from unbalanced data structures. Estimation of marker effects by the new model is compared to a genome-wide association analysis and whole-genome regression methods. The single-stage approach is compared to a GBLUP fitted via restricted maximum likelihood, and a two-stages approaches where genetic values fit a whole-genome regression. The proposed framework provided high computational efficiency, robust prediction across datasets, and accurate estimation of marker effects.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2019/11/01" />
<meta name="citation_mjid" content="ggg;9/11/3855" />
<meta name="citation_id" content="9/11/3855" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/9/11/3855" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/9/11/3855.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/9/11/3855.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/9/11/3855.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.119.400728" />
<meta name="citation_pmid" content="31690600" />
<meta name="citation_volume" content="9" />
<meta name="citation_issue" content="11" />
<meta name="citation_num_pages" content="12" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Genomic Prediction" />
<meta name="citation_firstpage" content="3855" />
<meta name="citation_lastpage" content="3866" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Alencar Xavier" />
<meta name="citation_author_institution" content="Corteva Agrisciences, 8305 NW 62nd Ave. Johnston IA, and" />
<meta name="citation_author_institution" content="Purdue University, 915 W State St. West Lafayette IN" />
<meta name="citation_author_email" content="alencar.xavier@corteva.com" />
<meta name="citation_author_email" content="xaviera@purdue.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-5034-9954" />
<meta name="citation_reference" content="citation_journal_title=J. Stat. Softw.;citation_author=D. Bates;citation_author=M. Mächler;citation_author=B. Bolker;citation_author=S. Walker;citation_title=Fitting linear mixed-effects models using lme4.;citation_pages=1-48;citation_volume=67;citation_year=2015;citation_doi=10.18637/jss.v067.i01" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=X. Cai;citation_author=A. Huang;citation_author=S. Xu;citation_title=Fast empirical Bayesian LASSO for multiple quantitative trait locus mapping.;citation_pages=211-211;citation_volume=12;citation_year=2011;citation_pmid=21615941;citation_doi=10.1186/1471-2105-12-211" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=Y. Da;citation_author=C. Wang;citation_author=S. Wang;citation_author=G. Hu;citation_title=Mixed model methods for genomic prediction and variance component estimation of additive and dominance effects using snp markers.;citation_pages=e87666;citation_volume=9;citation_year=2014;citation_pmid=24498162;citation_doi=10.1371/journal.pone.0087666" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=H. D. Daetwyler;citation_author=M. P. Calus;citation_author=R. Pong-Wong;citation_author=G. de los Campos;citation_author=J. M. Hickey;citation_title=Genomic prediction in animals and plants: simulation of data, validation, reporting, and benchmarking.;citation_volume=193;citation_year=2013;citation_pmid=23222650;citation_doi=10.1534/genetics.112.147983" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=H. D. Daetwyler;citation_author=M. J. Hayden;citation_author=G. C. Spangenberg;citation_author=B. J. Hayes;citation_title=Selection on optimal haploid value increases genetic gain and preserves more genetic diversity relative to genomic selection.;citation_volume=200;citation_year=2015;citation_pmid=26092719;citation_doi=10.1534/genetics.115.178038" />
<meta name="citation_reference" content="citation_journal_title=Agron. J.;citation_author=T. M. Damesa;citation_author=J. Möhring;citation_author=M. Worku;citation_author=H.-P. Piepho;citation_title=One step at a time: stage-wise analysis of a series of experiments.;citation_pages=845-857;citation_volume=109;citation_year=2017;citation_doi=10.2134/agronj2016.07.0395" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=G. de los Campos;citation_author=D. Gianola;citation_author=G. J. Rosa;citation_author=K. A. Weigel;citation_author=J. Crossa;citation_title=Semi-parametric genomic-enabled prediction of genetic values using reproducing kernel Hilbert spaces methods.;citation_pages=295-308;citation_volume=92;citation_year=2010;citation_issue=4;citation_pmid=20943010;citation_doi=10.1017/S0016672310000285" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=G. de los Campos;citation_author=J. M. Hickey;citation_author=R. Pong-Wong;citation_author=H. D. Daetwyler;citation_author=M. P. Calus;citation_title=Whole-genome regression and prediction methods applied to plant and animal breeding.;citation_volume=193;citation_year=2013;citation_pmid=22745228;citation_doi=10.1534/genetics.112.143313" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=G. de los Campos;citation_author=H. Naya;citation_author=D. Gianola;citation_author=J. Crossa;citation_author=A. Legarra;citation_title=Predicting quantitative traits with regression models for dense molecular markers and pedigrees.;citation_volume=182;citation_year=2009;citation_pmid=19293140;citation_doi=10.1534/genetics.109.101501" />
<meta name="citation_reference" content="citation_journal_title=Trends Plant Sci.;citation_author=Z. A. Desta;citation_author=R. Ortiz;citation_title=Genomic selection: genome-wide prediction in plant improvement.;citation_pages=592-601;citation_volume=19;citation_year=2014;citation_pmid=24970707;citation_doi=10.1016/j.tplants.2014.05.006" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=B. W. Diers;citation_author=J. Specht;citation_author=K. M. Rainey;citation_author=P. Cregan;citation_author=Q. Song;citation_title=Genetic architecture of soybean yield and agronomic traits. G3: Genes, Genomes;citation_pages=3367-3375;citation_volume=8;citation_year=2018" />
<meta name="citation_reference" content="citation_journal_abbrev=J Stat Softw;citation_author=D. Eddelbuettel;citation_author=R. François;citation_author=J. Allaire;citation_author=K. Ushey;citation_author=Q. Kou;citation_title=Using the R Package crlmm for Genotyping and Copy Number Estimation.;citation_pages=1-32;citation_volume=40;citation_year=2011;citation_issue=12;citation_pmid=22523482;citation_doi=10.18637/jss.v040.i01" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=J. B. Endelman;citation_title=Ridge regression and other kernels for genomic selection with r package rrblup.;citation_pages=250-255;citation_volume=4;citation_year=2011;citation_doi=10.3835/plantgenome2011.08.0024" />
<meta name="citation_reference" content="citation_journal_title=J. Agric. Biol. Environ. Stat.;citation_author=R. Fernando;citation_author=A. Toosi;citation_author=A. Wolc;citation_author=D. Garrick;citation_author=J. Dekkers;citation_title=Application of whole-genome prediction methods for genome-wide association studies: a bayesian approach.;citation_pages=172-193;citation_volume=22;citation_year=2017;citation_doi=10.1007/s13253-017-0277-6" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=R. L. Fernando;citation_author=H. Cheng;citation_author=B. L. Golden;citation_author=D. J. Garrick;citation_title=Computational strategies for alternative single-step bayesian regression models with large numbers of genotyped and non-genotyped animals.;citation_volume=48;citation_year=2016;citation_doi=10.1186/s12711-016-0273-2" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=R. L. Fernando;citation_author=J. C. Dekkers;citation_author=D. J. Garrick;citation_title=A class of bayesian methods to combine large numbers of genotyped and non-genotyped animals for whole-genome analyses.;citation_pages=50;citation_volume=46;citation_year=2014;citation_pmid=25253441;citation_doi=10.1186/1297-9686-46-50" />
<meta name="citation_reference" content="citation_title=Genome-wide association studies and genomic prediction;citation_author=R. L. Fernando;citation_author=D. Garrick;citation_title=Bayesian methods applied to gwas;citation_year=2013;citation_doi=10.1007/978-1-62703-447-0_10" />
<meta name="citation_reference" content="citation_journal_abbrev=J Stat Softw;citation_author=J. Friedman;citation_author=T. Hastie;citation_author=R. Tibshirani;citation_title=Regularization Paths for Generalized Linear Models via Coordinate Descent.;citation_pages=1-22;citation_volume=33;citation_year=2010;citation_issue=1;citation_pmid=20808728;citation_doi=10.1111/j.1467-9868.2005.00503.x" />
<meta name="citation_reference" content="citation_journal_title=J. Anim. Sci.;citation_author=D. Garrick;citation_title=Equivalent mixed model equations for genomic selection.;citation_pages=376;citation_volume=85;citation_year=2007" />
<meta name="citation_reference" content="citation_journal_title=Livest. Sci.;citation_author=D. Garrick;citation_author=J. Dekkers;citation_author=R. Fernando;citation_title=The evolution of methodologies for genomic prediction.;citation_pages=10-18;citation_volume=166;citation_year=2014;citation_doi=10.1016/j.livsci.2014.05.031" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=D. J. Garrick;citation_author=J. F. Taylor;citation_author=R. L. Fernando;citation_title=Deregressing estimated breeding values and weighting information for genomic regression analyses.;citation_pages=55-55;citation_volume=41;citation_year=2009;citation_pmid=20043827;citation_doi=10.1186/1297-9686-41-55" />
<meta name="citation_reference" content="citation_journal_title=Nat. Rev. Genet.;citation_author=M. Georges;citation_author=C. Charlier;citation_author=B. Hayes;citation_title=Harnessing genomic information for livestock improvement.;citation_volume=20;citation_year=2019;citation_doi=10.1038/s41576-018-0082-2" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=D. Gianola;citation_author=R. L. Fernando;citation_author=A. Stella;citation_title=Genomic assisted prediction of genetic value with semi-parametric procedures.;citation_volume=173;citation_year=2006;citation_pmid=16648593;citation_doi=10.1534/genetics.105.049510" />
<meta name="citation_reference" content="citation_title=Asreml user guide release 3.0;citation_year=2008" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. Goiffon;citation_author=A. Kusmec;citation_author=L. Wang;citation_author=G. Hu;citation_author=P. Schnable;citation_title=Improving response in genomic selection with a population-based selection strategy: optimal population value selection.;citation_volume=206;citation_year=2017;citation_pmid=28526698;citation_doi=10.1534/genetics.116.197103" />
<meta name="citation_reference" content="citation_journal_title=Journal of Animal Science;citation_journal_abbrev=J ANIM SCI;citation_author=H.-U. Graser;citation_author=S. Smith;citation_author=B. Tier;citation_title=A Derivative-Free Approach for Estimating Variance Components in Animal Models by Restricted Maximum Likelihood;citation_pages=1362-1370;citation_volume=64;citation_year=1987;citation_issue=5;citation_doi=10.2527/jas1987.6451362x" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=D. Habier;citation_author=R. Fernando;citation_author=J. C. Dekkers;citation_title=The Impact of Genetic Relationship Information on Genome-Assisted Breeding Values;citation_pages=2389-2397;citation_volume=177;citation_year=2007;citation_issue=4;citation_pmid=18073436;citation_doi=10.1534/genetics.107.081190" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=D. Habier;citation_author=R. L. Fernando;citation_author=K. Kizilkaya;citation_author=D. J. Garrick;citation_title=Extension of the bayesian alphabet for genomic selection.;citation_pages=186-186;citation_volume=12;citation_year=2011;citation_pmid=21605355;citation_doi=10.1186/1471-2105-12-186" />
<meta name="citation_reference" content="citation_journal_title=J. Am. Stat. Assoc.;citation_author=D. A. Harville;citation_title=Maximum likelihood approaches to variance component estimation and to related problems.;citation_volume=72;citation_year=1977;citation_doi=10.1080/01621459.1977.10480998" />
<meta name="citation_reference" content="Hastie, T., J. Friedman, and R. Tibshirani, 2009 The elements of statistical learning, volume 2. Springer series in statistics New York. 10.1007/978-0-387-84858-7" />
<meta name="citation_reference" content="citation_journal_title=BMC genetics [electronic resource];citation_journal_abbrev=BMC Genet;citation_author=T. Hayashi;citation_author=H. Iwata;citation_title=EM algorithm for Bayesian estimation of genomic breeding values.;citation_pages=3-3;citation_volume=11;citation_year=2010;citation_pmid=20092655;citation_doi=10.1186/1471-2156-11-3" />
<meta name="citation_reference" content="citation_title=Genome-wide association studies and genomic prediction;citation_author=B. Hayes;citation_title=Overview of statistical methods for genome-wide association studies (gwas);citation_pages=149-169;citation_volume=1019;citation_year=2013;citation_pmid=23756890;citation_doi=10.1007/978-1-62703-447-0_6" />
<meta name="citation_reference" content="citation_journal_title=Livest. Sci.;citation_author=M. Henryon;citation_author=P. Berg;citation_author=A. C. Sørensen;citation_title=Animal-breeding schemes using genomic information need breeding plans designed to maximise long-term genetic gains.;citation_pages=38-47;citation_volume=166;citation_year=2014;citation_doi=10.1016/j.livsci.2014.06.016" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=N. Heslot;citation_author=J.-L. Jannink;citation_author=M. E. Sorrells;citation_title=Perspectives for genomic selection applications and research in plants.;citation_pages=1-12;citation_volume=55;citation_year=2015;citation_doi=10.2135/cropsci2014.03.0249" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=N. Heslot;citation_author=H.-P. Yang;citation_author=M. E. Sorrells;citation_author=J.-L. Jannink;citation_title=Genomic selection in plant breeding: a comparison of models.;citation_pages=146-160;citation_volume=52;citation_year=2012;citation_doi=10.2135/cropsci2011.06.0297" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=J. M. Hickey;citation_author=T. Chiurugwi;citation_author=I. Mackay;citation_author=W. Powell;citation_author=A. Eggen;citation_title=Genomic prediction unifies animal and plant breeding programs to form platforms for biological discovery.;citation_pages=1297-1303;citation_volume=49;citation_year=2017;citation_pmid=28854179;citation_doi=10.1038/ng.3920" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=R. Howard;citation_author=A. L. Carriquiry;citation_author=W. D. Beavis;citation_title=Parametric and nonparametric statistical methods for genomic selection of traits with additive and epistatic genetic architectures. G3: Genes, Genomes;citation_pages=1027-1046;citation_volume=4;citation_year=2014" />
<meta name="citation_reference" content="Huang, A. and D. Liu, 2016 EBglmnet: a comprehensive R package for sparse generalized linear regression models. Bioinformatics pp. 1–3, btw143. 10.1093/bioinformatics/btw143" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=A. Jacobson;citation_author=L. Lian;citation_author=S. Zhong;citation_author=R. Bernardo;citation_title=General combining ability model for genomewide selection in a biparental cross.;citation_pages=895-905;citation_volume=54;citation_year=2014;citation_doi=10.2135/cropsci2013.11.0774" />
<meta name="citation_reference" content="citation_journal_title=J. Stat. Softw.;citation_author=A. Karatzoglou;citation_author=A. Smola;citation_author=K. Hornik;citation_author=A. Zeileis;citation_title=kernlab-an s4 package for kernel methods in r.;citation_pages=1-20;citation_volume=11;citation_year=2004;citation_doi=10.18637/jss.v011.i09" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=H. P. Kärkkäinen;citation_author=M. J. Sillanpää;citation_title=Back to basics for bayesian model building in genomic selection.;citation_volume=191;citation_year=2012;citation_pmid=22554888;citation_doi=10.1534/genetics.112.139014" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=A. Legarra;citation_author=P. Croiseau;citation_author=M. P. Sanchez;citation_author=S. Teyssèdre;citation_author=G. Sallè;citation_title=A comparison of methods for whole-genome qtl mapping using dense markers in four livestock species.;citation_pages=6;citation_volume=47;citation_year=2015;citation_doi=10.1186/s12711-015-0087-7" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=A. Legarra;citation_author=I. Misztal;citation_title=Technical Note: Computing Strategies in Genome-Wide Selection;citation_pages=360-366;citation_volume=91;citation_year=2008;citation_issue=1;citation_pmid=18096959;citation_doi=10.3168/jds.2007-0403" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=A. Legarra;citation_author=C. Robert-Granié;citation_author=P. Croiseau;citation_author=F. Guillaume;citation_author=S. Fritz;citation_title=Improved Lasso for genomic selection.;citation_pages=77-87;citation_volume=93;citation_year=2011;citation_issue=1;citation_pmid=21144129;citation_doi=10.1017/S0016672310000534" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. Legarra;citation_author=C. Robert-Granié;citation_author=E. Manfredi;citation_author=J.-M. Elsen;citation_title=Performance of genomic selection in mice.;citation_volume=180;citation_year=2008;citation_pmid=18757934;citation_doi=10.1534/genetics.108.088575" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=C. Lehermeier;citation_author=N. Krämer;citation_author=E. Bauer;citation_author=C. Bauland;citation_author=C. Camisan;citation_title=Usefulness of Multiparental Populations of Maize (Zea mays L.) for Genome-Based Prediction;citation_pages=3-16;citation_volume=198;citation_year=2014;citation_issue=1;citation_pmid=25236445;citation_doi=10.1534/genetics.114.161943" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=L. Lian;citation_author=A. Jacobson;citation_author=S. Zhong;citation_author=R. Bernardo;citation_title=Genomewide prediction accuracy within 969 maize biparental populations.;citation_pages=1514-1522;citation_volume=54;citation_year=2014;citation_doi=10.2135/cropsci2013.12.0856" />
<meta name="citation_reference" content="citation_journal_title=Biometrics;citation_author=C.-S. Lin;citation_author=G. Poushinsky;citation_title=A modified augmented design for an early stage of plant selection involving a large number of test lines without replication.;citation_pages=553-561;citation_volume=39;citation_year=1983;citation_doi=10.2307/2531083" />
<meta name="citation_reference" content="citation_journal_title=J. Dairy Sci.;citation_author=Z. Liu;citation_author=M. Goddard;citation_author=F. Reinhardt;citation_author=R. Reents;citation_title=A single-step genomic model with direct estimation of marker effects.;citation_pages=5833-5850;citation_volume=97;citation_year=2014;citation_pmid=25022678;citation_doi=10.3168/jds.2014-7924" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=T. H. E. Meuwissen;citation_author=B. J. Hayes;citation_author=M. E. Goddard;citation_title=Prediction of Total Genetic Value Using Genome-Wide Dense Marker Maps;citation_pages=1819-1829;citation_volume=157;citation_year=2001;citation_issue=4;citation_pmid=11290733;citation_doi=10.1093/nar/gkv1344" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=T. H. E. Meuwissen;citation_author=T. R. Solberg;citation_author=R. Shepherd;citation_author=J. A. Woolliams;citation_title=A fast algorithm for BayesB type of prediction of genome-wide estimates of genetic value.;citation_pages=2-2;citation_volume=41;citation_year=2009;citation_pmid=19284681;citation_doi=10.1186/1297-9686-41-2" />
<meta name="citation_reference" content="citation_journal_title=J. Stat. Softw.;citation_author=B.-H. Mevik;citation_author=R. Wehrens;citation_title=The pls package: principal component and partial least squares regression in r.;citation_pages=1-23;citation_volume=18;citation_year=2007;citation_pmid=15238266;citation_doi=10.1360/jos180001" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=I. Misztal;citation_title=Inexpensive computation of the inverse of the genomic relationship matrix in populations with small effective population size.;citation_volume=202;citation_year=2016;citation_pmid=26584903;citation_doi=10.1534/genetics.115.182089" />
<meta name="citation_reference" content="citation_title=A unified approach to utilize phenotypic, full pedigree and genomic information for a genetic evaluation of holstein final score;citation_pages=240-244;citation_year=2009" />
<meta name="citation_reference" content="Misztal, I. and A. Legarra, 2017 Invited review: efficient computation strategies in genomic selection. animal 11: 731–736." />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=J. Möhring;citation_author=H.-P. Piepho;citation_title=Comparison of Weighting in Two-Stage Analysis of Plant Breeding Trials;citation_pages=1977-1988;citation_volume=49;citation_year=2009;citation_issue=6;citation_doi=10.2135/cropsci2009.02.0083" />
<meta name="citation_reference" content="citation_journal_title=Bayesian Anal.;citation_author=R. B. O’Hara;citation_author=M. J. Sillanpää;citation_title=A review of bayesian variable selection methods: what, how and which.;citation_pages=85-117;citation_volume=4;citation_year=2009;citation_doi=10.1214/09-BA403" />
<meta name="citation_reference" content="citation_journal_title=J. Open Res. Softw.;citation_author=A. Onogi;citation_author=H. Iwata;citation_title=Vigor: variational bayesian inference for genome-wide regression.;citation_pages=1-11;citation_volume=4;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=Z. Ou;citation_author=R. J. Tempelman;citation_author=J. P. Steibel;citation_author=C. W. Ernst;citation_author=R. O. Bates;citation_title=Genomic prediction accounting for residual heteroskedasticity. G3: Genes, Genomes;citation_pages=1-13;citation_volume=6;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=J. Am. Stat. Assoc.;citation_author=T. Park;citation_author=G. Casella;citation_title=The bayesian lasso.;citation_pages=681-686;citation_volume=103;citation_year=2008;citation_doi=10.1198/016214508000000337" />
<meta name="citation_reference" content="citation_journal_title=Biometrika;citation_journal_abbrev=Biometrika;citation_author=H. D. Patterson;citation_author=R. Thompson;citation_title=Recovery of inter-block information when block sizes are unequal;citation_pages=545-554;citation_volume=58;citation_year=1971;citation_issue=3;citation_doi=10.1093/biomet/58.3.545" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=P. Pèrez;citation_author=G. de Los Campos;citation_title=Genome-wide regression &amp; prediction with the bglr statistical package.;citation_volume=198;citation_year=2014;citation_pmid=25009151;citation_doi=10.1534/genetics.114.164442" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=P. Pérez-Rodríguez;citation_author=D. Gianola;citation_author=J. M. González-Camacho;citation_author=J. Crossa;citation_author=Y. Manès;citation_title=Comparison between linear and non-parametric regression models for genome-enabled prediction in wheat. G3: Genes, Genomes;citation_pages=1595-1605;citation_volume=2;citation_year=2012" />
<meta name="citation_reference" content="citation_journal_abbrev=Biom J;citation_author=H.-P. Piepho;citation_author=J. Moehring;citation_author=T. Schulz-Streeck;citation_author=J. O. Ogutu;citation_title=A stage-wise approach for the analysis of multi-environment trials.;citation_pages=844-860;citation_volume=54;citation_year=2012;citation_issue=6;citation_pmid=23007738;citation_doi=10.1002/bimj.201100219" />
<meta name="citation_reference" content="citation_journal_title=Update;citation_author=G. Ridgeway;citation_title=Generalized boosted models: A guide to the gbm package.;citation_pages=2007;citation_volume=1;citation_year=2007" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=L. Schaeffer;citation_title=Pseudo Expectation Approach to Variance Component Estimation;citation_pages=2884-2889;citation_volume=69;citation_year=1986;citation_issue=11;citation_doi=10.3168/jds.S0022-0302(86)80743-3" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=T. Schulz-Streeck;citation_author=J. O. Ogutu;citation_author=H.-P. Piepho;citation_title=Comparisons of single-stage and two-stage approaches to genomic selection.;citation_pages=69-82;citation_volume=126;citation_year=2013;citation_pmid=22903736;citation_doi=10.1007/s00122-012-1960-1" />
<meta name="citation_reference" content="citation_title=Variance Components;citation_author=S. R. Searle;citation_author=G. Casella;citation_author=C. E. McCulloch;citation_title=Prediction of random variables;citation_year=1992;citation_doi=10.1002/9780470316856.ch10" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=S. Sebastian;citation_author=L. Streit;citation_author=P. Stephens;citation_author=J. Thompson;citation_author=B. Hedges;citation_title=Context-Specific Marker-Assisted Selection for Improved Grain Yield in Elite Soybean Populations;citation_pages=1196-1206;citation_volume=50;citation_year=2010;citation_issue=4;citation_doi=10.2135/cropsci2009.02.0078" />
<meta name="citation_reference" content="citation_journal_title=Aust. N. Z. J. Stat.;citation_author=A. Smith;citation_author=B. Cullis;citation_author=A. Gilmour;citation_title=Applications: the analysis of crop variety evaluation data in australia.;citation_pages=129-145;citation_volume=43;citation_year=2001;citation_doi=10.1111/1467-842X.00163" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=X. Sun;citation_author=L. Qu;citation_author=D. J. Garrick;citation_author=J. C. Dekkers;citation_author=R. L. Fernando;citation_title=A Fast EM Algorithm for BayesA-Like Prediction of Genomic Breeding Values.;citation_pages=e49157-e49157;citation_volume=7;citation_year=2012;citation_issue=11;citation_pmid=23152868;citation_doi=10.1371/journal.pone.0049157" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=M. Taskinen;citation_author=E. A. Mäntysaari;citation_author=I. Strandèn;citation_title=Single-step snp-blup with on-the-fly imputed genotypes and residual polygenic effects.;citation_pages=36;citation_volume=49;citation_year=2017;citation_doi=10.1186/s12711-017-0310-9" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=E. Thompson;citation_author=R. Shaw;citation_title=Estimating polygenic models for multivariate data on large pedigrees.;citation_pages=971-978;citation_volume=131;citation_year=1992;citation_issue=4;citation_pmid=1516823" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=P. M. VanRaden;citation_title=Efficient Methods to Compute Genomic Predictions;citation_pages=4414-4423;citation_volume=91;citation_year=2008;citation_issue=11;citation_pmid=18946147;citation_doi=10.3168/jds.2007-0980" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=T. Wang;citation_author=Y.-P. P. Chen;citation_author=M. E. Goddard;citation_author=T. H. Meuwissen;citation_author=K. E. Kemper;citation_title=A computationally efficient algorithm for genomic prediction using a bayesian model.;citation_pages=34;citation_volume=47;citation_year=2015;citation_doi=10.1186/s12711-014-0082-4" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=V. Wimmer;citation_author=C. Lehermeier;citation_author=T. Albrecht;citation_author=H.-J. Auinger;citation_author=Y. Wang;citation_title=Genome-wide prediction of traits with different genetic architecture through efficient variable selection.;citation_volume=195;citation_year=2013;citation_pmid=23934883;citation_doi=10.1534/genetics.113.150078" />
<meta name="citation_reference" content="citation_journal_title=J. Anim. Sci. Biotechnol.;citation_author=A. Wolc;citation_author=J. Arango;citation_author=P. Settar;citation_author=J. E. Fulton;citation_author=N. P. O’Sullivan;citation_title=Mixture models detect large effect qtl better than gblup and result in more accurate and persistent predictions.;citation_pages=7;citation_volume=7;citation_year=2016;citation_doi=10.1186/s40104-016-0066-z" />
<meta name="citation_reference" content="Wright, M. N., and A. Ziegler, 2015 Ranger: a fast implementation of random forests for high dimensional data in c++ and r. arXiv preprint arXiv:1508.04409." />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. Xavier;citation_author=D. Jarquin;citation_author=R. Howard;citation_author=V. Ramasubramanian;citation_author=J. E. Specht;citation_title=Genome-wide analysis of grain yield stability and environmental interactions in a multiparental soybean population. G3: Genes, Genomes;citation_pages=519-529;citation_volume=8;citation_year=2018" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. Xavier;citation_author=W. M. Muir;citation_author=K. M. Rainey;citation_title=Assessing predictive properties of genome-wide selection in soybeans. G3: Genes, Genomes;citation_pages=2611-2616;citation_volume=6;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. Xavier;citation_author=S. Xu;citation_author=W. M. Muir;citation_author=K. M. Rainey;citation_title=Nam: association studies in multiple populations.;citation_volume=31;citation_year=2015;citation_pmid=26243017;citation_doi=10.1093/bioinformatics/btv448" />
<meta name="citation_reference" content="citation_journal_title=Biometrics;citation_journal_abbrev=Biometrics;citation_author=S. Xu;citation_title=An empirical Bayes method for estimating epistatic effects of quantitative trait loci.;citation_pages=513-521;citation_volume=63;citation_year=2007;citation_issue=2;citation_pmid=17688503;citation_doi=10.1111/j.1541-0420.2006.00711.x" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_journal_abbrev=Heredity;citation_author=S. Xu;citation_title=An expectation-maximization algorithm for the Lasso estimation of quantitative trait locus effects.;citation_pages=483-494;citation_volume=105;citation_year=2010;citation_issue=5;citation_pmid=20051978;citation_doi=10.1038/hdy.2009.180" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. Xu;citation_title=Mapping quantitative trait loci by controlling polygenic background effects.;citation_volume=195;citation_year=2013;citation_pmid=24077303;citation_doi=10.1534/genetics.113.157032" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=Z. Yang;citation_author=N. A. Zaitlen;citation_author=M. E. Goddard;citation_author=P. M. Visscher;citation_author=A. L. Price;citation_title=Advantages and pitfalls in the application of mixed-model association methods.;citation_pages=100-106;citation_volume=46;citation_year=2014;citation_pmid=24473328;citation_doi=10.1038/ng.2876" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=N. Yi;citation_author=S. Xu;citation_title=Bayesian lasso for quantitative trait loci mapping.;citation_volume=179;citation_year=2008;citation_pmid=18505874;citation_doi=10.1534/genetics.107.085589" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=Z.-B. Zeng;citation_author=T. Wang;citation_author=W. Zou;citation_title=Modeling quantitative trait loci and interpretation of models.;citation_volume=169;citation_year=2005;citation_pmid=15654105;citation_doi=10.1534/genetics.104.035857" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=Z. Zhang;citation_author=E. Ersoz;citation_author=C.-Q. Lai;citation_author=R. J. Todhunter;citation_author=H. K. Tiwari;citation_title=Mixed linear model approach adapted for genome-wide association studies.;citation_pages=355-360;citation_volume=42;citation_year=2010;citation_issue=4;citation_pmid=20208535;citation_doi=10.1038/ng.546" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=L. Zhou;citation_author=R. Mrode;citation_author=S. Zhang;citation_author=Q. Zhang;citation_author=B. Li;citation_title=Factors affecting gebv accuracy with single-step bayesian models.;citation_pages=100-109;citation_volume=120;citation_year=2018;citation_doi=10.1038/s41437-017-0010-9" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Efficient Estimation of Marker Effects in Plant Breeding" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11.cover-source.jpg" />
<meta name="twitter:description" content="The evaluation of prediction machines is an important step for a successful implementation of genomic-enabled selection in plant breeding. Computation time and predictive ability constitute key metrics to determine the methodology utilized for the consolidation of genomic prediction pipeline. This study introduces two methods designed to couple high prediction accuracy with efficient computational performance: 1) a non-MCMC method to estimate marker effects with a Laplace prior; and 2) an iterative framework that allows solving whole-genome regression within mixed models with replicated observations in a single-stage. The investigation provides insights on predictive ability and marker effect estimates. Various genomic prediction techniques are compared based on cross-validation, assessing predictions across and within family. Properties of quantitative trait loci detection and single-stage method were evaluated on simulated plot-level data from unbalanced data structures. Estimation of marker effects by the new model is compared to a genome-wide association analysis and whole-genome regression methods. The single-stage approach is compared to a GBLUP fitted via restricted maximum likelihood, and a two-stages approaches where genetic values fit a whole-genome regression. The proposed framework provided high computational efficiency, robust prediction across datasets, and accurate estimation of marker effects." />
<meta name="og-title" property="og:title" content="Efficient Estimation of Marker Effects in Plant Breeding" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/9/11/3855" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="The evaluation of prediction machines is an important step for a successful implementation of genomic-enabled selection in plant breeding. Computation time and predictive ability constitute key metrics to determine the methodology utilized for the consolidation of genomic prediction pipeline. This study introduces two methods designed to couple high prediction accuracy with efficient computational performance: 1) a non-MCMC method to estimate marker effects with a Laplace prior; and 2) an iterative framework that allows solving whole-genome regression within mixed models with replicated observations in a single-stage. The investigation provides insights on predictive ability and marker effect estimates. Various genomic prediction techniques are compared based on cross-validation, assessing predictions across and within family. Properties of quantitative trait loci detection and single-stage method were evaluated on simulated plot-level data from unbalanced data structures. Estimation of marker effects by the new model is compared to a genome-wide association analysis and whole-genome regression methods. The single-stage approach is compared to a GBLUP fitted via restricted maximum likelihood, and a two-stages approaches where genetic values fit a whole-genome regression. The proposed framework provided high computational efficiency, robust prediction across datasets, and accurate estimation of marker effects." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11.cover-source.jpg" />
    <title>Efficient Estimation of Marker Effects in Plant Breeding | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"z1YzuLnr4c468IfAQnGIGdTDodWJHSaokToKJJm7Lio","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"494613","apath":"\/ggg\/9\/11\/3855.atom","pisa":"ggg;9\/11\/3855","ac":{"\/ggg\/9\/11\/3855.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/9\/11\/3855.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;9\/11\/3855"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;9\/11\/3855"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/9\/11\/3855.figures-only":true,"\/content\/9\/11\/3855":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022bRD4f4ReOK8+6kMPZRmQ00dCfxJd4b27T2U4upDVy+PtJSYiiG\\\\\\\/0jz\\\\\\\/ut282l\\\\\\\/pABUkrHLLZz6gb7gSSe04Dlw9LSF7eNiAEBIZyADcpzWPfoB1sJe3bEdxg\\\\\\\/7qe\\\\\\\/ofqWY0q77YUIjF6aAZZYPuNQ1ArmHCAE8B6g5qpb+9l5dh1m1iqpohmCwxdnj4J7fu1SH+Z4wSKdwXx76yLmr341nuVCPlidytRqnGSAOeReCgqKqpLyyU1NRa0vusw0dtDkgvML1roZq7pa4qvSn7wo5LZb6AHI3ERCrR2LuLdbENCe215FV8jMlMEZgQMFTkQUPwmmyYiA8ujka74ScVhEX20hGWsRCr3WSjez0PAKFtmjErSzHLyTiq05bUvIZzKq95jNlx+7B\\\\\\\/+bXmh7QYJrx3J+Ifllav9OgaWrQpsGtGjLbU\\\\\\\/47mGuE7Z2GLd40fIKjyv4zHu8R7bmIAQ8KPFMiwrXN2C9Fw96Mb6yk6QON0ZWBOpyxpXmaMwR7qspajtUUmT79k2akhI6SlzVcW3DloaQCVcvPONZkkK8z0iHLs88qpzWlIIZdXe23FvSJ6oge1aa0GnPVc07mdZh49WeD29oNrBdfJveKNHe67AigcW7QURhgIXSS3Pb4XqWmSE7qgxD5\\\\\\\/XoOxIfzW9zd\\\\\\\/lCpxzUM9H9yZMy9Lyqikmo3F2WQPWQwuC2CzqZgK6zMHuU\\\\\\\/S7Xg8K4xNCKzDdCqFF\\\\\\\/jNUzxiIZD7rwXpLd3OKYsqI7hWDGo7skrxKn3jEUJD6tl0snODl\\\\\\\/sxxjf0iL7C7\\\\\\\/FiUq8WKt4qubd3S7UIKuutmbq54j\\\\\\\/qXeARb7g8C3ieyRYt8ndmRlcCLUP2OulyVNkyoMUTvOIT5YUlq3HEsJfJW1iXYt79X1W3M1wRHeSmMtwvdj\\\\\\\/klrwkUcqjiYpcmmuWginf\\\\\\\/AYYCQ1DThwyDBcwivRBF2MCCIcJkzW0h+9Fk2ui3TFTVsQPYUNjQHkG+Sd5fSej8iSN5eMtxhdQGVfwqpPKRH8K\\\\\\\/v\\\\\\\/+VPwmO\\\\\\\/7zco7JsnnecdTzonmtVWPhTGJDQxTDMp5+R1wZB7N3V5V3nL81H2L0RhtwJBdtJGqV5eTGYlfFsZURzMpPdwuB7Q7VN7TxCmqKxmTQVOceMbXoKjLl7zkgBbBaFdNwI0COQWTuZgeCAKAj68YFsuI9cVbr0wz0ve6igc8\\\\\\\/B6p1VWj7ynmMQ5rKDszhfjiDf5GUNaufWXm71FtJou7RSMNyD7\\\\\\\/wZxJIw1N90Hj1Fg8E3v7XsXjsiwZ+38nt6jcyBgGtC\\\u0022,\\\u0022iv\\\u0022:\\\u002256Rb9OsAthPR0CmwKQinUg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00222f1152d50b01fbe866503a2fa9311ff62adaa9234175af3f0f7787a98091d889\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00225A0aV9F89KazOY2WVsN4BFqdHsjGYItrWZ+sjuFpa5mLm48ualeOLXgMReg\\\\\\\/\\\\\\\/0KnWttdB9js+qO9XCM5cNWOA9QKXbW47nJ3tQOCWZrHZRlID23tTB3zM4SjvhKiPVJqEl7jRpBoIxV19brFZShvxkj6AyJw+m64aP2FueZQtQZne6C2gn\\\\\\\/YXluN\\\\\\\/IQvsuF7M2mSVtJKpQpYWFWpKez0N7HZ8AGq7jtnOpFJoWpNia7agNuMbQkhyEUDn0giCzPDHDqnXuR33Ffl4SQiRJz6Y0CS1qwxkm6exWka9rMNCUjr5TeM1HxQ8L1eOd8Y56leAiz0DXw8QwdGiZAAdYs\\\\\\\/eOOAMF6aoqNw1FB2W3bnKKDCtYVfQkKCQtlQzhyYNP7cdcJfpt9hIAjUnHC7G76Wy9JNmprZEyiEeOsxuE8wz\\\\\\\/oiYZBk+mlrm5apiugSjp9GdW\\\\\\\/dco6PX0rygXx1WxB+ZebSn32heFY76FNLT7ixMpdyB4f6KJUCuUS0p4TD4C\\\\\\\/SuL3hw\\\\\\\/jEZPV2ponf8ZJH6o+Wd7hWoyiqKXBadPq0E6\\\\\\\/nDkYpK3RhhneQZiw+JXJRiUy\\\\\\\/S5lhcP+gqQa2wEPL7pimfQDYdUkQlYczo56STh6+UUK9NHBh\\\\\\\/lr2HoT0FmA5kvxrZA4oU+naqHth4h44QIiuUEYM19cB8wjj4CmyHjSDtiwm8ru3ZtxOBbg3fPfYvhEPc7wmAPbRwP1C3dqjwxpgybYCqFPo+3tMkLOix\\\\\\\/3zo5CFvUhsA3woz449DPs2LdFNFXoQxDb0li92LBL0BV57gnryrmbH15r+7NZErO3uKwLis1WM3\\\\\\\/esI0HsQFnIoAj8XPfKgyvX9eUknXaraKcYSONdI7eNhVqk3qTFDwEbSLDsv29sBGCIAB3yTbWy0BPJhUlx3DNE1zmPCTj2JMxzwZBXbpeIpWpX4LcZPVUOMnRD3wHxDqrKMCn8lKf7+nPiawJAkgVgxwoVIplYdLfAkq2jXsETY3lQbBQ1EMAtV9fgMmdNAPj0dEF8i1BIAsc6j3++j8Z4U9kFxcZbUfytbttbET0zW7uvZIHzW3yto6GD+YFWzLfhEhD883O\\\\\\\/kSPff99sWVzmQBQ7BnBAqhWmMJGCwbn7fnY9PxTFiJ\\\\\\\/QTR\\\\\\\/BNC4ytE3uaD+eH0YIF8TXVFCLob+MiMkdSIPETL+JtCjqY7FUsqeAS6oxcgm5PlpQQBI3QoJcyNBCE8b7UVdg2WO5DLqTTLGepM0+5D2Nm0uEhlCUFliSnQJbyYNREMTQizafnOnvFFJuaWuVl8bR+7\\\\\\\/kSNrnhSPnXsGCNxjuwFJYx4aHBpfhk5e5fDYlxwiVK2s0gt+Uzm3w6Mfu1c\\\\\\\/aIBrIsQC4XyIUKMQFvHFLJ19DWkwc50Gm74VA8QHKjs8U0\\\\\\\/BI5rgEhKU3ExVbyZ141oH8OiEOcUUDMF+hL84oZw4LMsjnEcws\\\\\\\/WVmQoxCGmQ7KdBp2yW\\\\\\\/MOv\\\\\\\/NOraeAH+vavEApiaDt9H1l9brmgp1lnvumd3pAS9ybyQ4e+Ykg79xNfEZHvnwPY05qnaTBfJzhWftof8fwdVzUFb5\\\\\\\/IA9o\\\\\\\/9ZpqR2L1HeOja\\\\\\\/fmT+6lESekiHj0zF2QNjvfUH7tK4so\\\\\\\/NbSC1qB7DOSiz6ppgIVZm0IgYE5qhdUDFXMwuk34eIPbxAZW0QuqMEK8Tn1xJwfopeaJnpdK36xmz8V2tqcUrfliUFjMvJGegGLRyvgNSUP3xlV7ZuCynR8\\\\\\\/eFvzSjaonLpduestRu\\\\\\\/t8nRCI4RJ\\\\\\\/FIVFLzdALJ1Xp\\\\\\\/rznIFxPXZsjBXJLVkFtWUKTctyf0nWwsefS6hl4ak2miZibN+eSIJdES0B\\\\\\\/txr7O61ZnLpK\\\\\\\/fqpa5EisWcUQXKlZgcSjor+VscztH6eRJLA4mBqW+4tzs1eu5ctk6oeztQMaoxX202g0cUHNsaEYQIqcCii6Xu5vO72shs2YHeJBXOmEJLrVt1pPzLNiXnyRxuB4np6HRo\\\\\\\/daHhjtsZ0u5Tdv6Ing\\\\\\\/rldc+jlZrPDG2OdQq1RsK+SedWSaGLF\\\\\\\/VyWAw1cTGx\\\\\\\/wIAynvA+2eHUII9w8l+qpJyaj8H5ECL8BS7+R5qlZ2dAdACoSxu7RCE3JCt8deW8OfgOSiIrM878WvBeIVO2J1JU+DM51wmbega5\\\\\\\/oq4zZjcnRWirJkLUFcQ9hdqq7FpfJQKWGYveqbfGZzIVFQAVGkbmJ4iOUewsKmDydH5+SzMA0j+BzYoPtPzqq17+NfSePuCmoQcRqTsly\\\\\\\/AZixjm4jOhzLvpKT5fn320sdTGKlI9iLLv4giuRYXtQwYU2xZUUQ8CGLxBeaj0A5yhSPRA8FYNxQYVQK7ELEO\\\\\\\/OEEfzZC8YpI5wqGPNvaFOPwDRY\\\\\\\/e1xCQZHJDzXn+bOIjTKN8HDnhRNgskvznWGMvdnIM8i1LOaPipDU8Pzk5epaNkjZsmpAbc3QyW5jtVE0oI0vmaTOtR5g9ZSX\\\\\\\/XwCoToiZWkNuAsnZ7KVQZdsL8SLOwNIf763Bkcx3V7WzQGckXwdLe362CeRCGi992IFoHmRoTvvZmRZehCzKhN1xlA8bCbF2RijX7x7NEjQMxyc\\\\\\\/VvKaRki+ocPQsMFNj8nc2eBZvX4l\\\\\\\/EiZ0qPMFOUlAiRJ0YipxPM+jwO9+by3F9cEUn6ealI2SRYN0euVCB3xMBH5tfCslxR3rm6z3G\\\\\\\/3ZICTXyrHDYnzirF1WVHjtbCrfAG+5R\\\\\\\/a3mi6UAD+lIPKHgdJVoDOa7Ah9d9eeEK4HN4tWb\\\\\\\/b8dJIn7krIRhalWWcSO+gknKV0Nzei4sS7x93PHqp4YQR3rLP8wPq2gBVzq08ee1\\\\\\\/FI953oNFHZfYSq1f1KTyKEFYr9EXtvV48jACj5rgNpw1qZPeBEqbXMrsK91R9uSVws6RBi9gCe14sPB0K3OtdEj4mJUBvIDEsh+dOopIJ1dbarNsVBSjwXOU3GBGk0kSB3k6mFOtgW4vXr+EeQOwUtiV3nqmq0j5bNExypLlnsQlZwtStMIaAAl5FaZ+DWsrbVKgfdY1QuBxfxAIaQN2yKPNiIlkDq1cJiOQiwO021CMdwJUimpO7seoCVB\\\\\\\/NDYfFi8ZCVgJBMxLZvLnZXl9pj22SxaNXCv5ztfPtTGlWVbU6MNn1EAhCDvXhUHWDz09LoQgGJEIzEqpxCJJiXhTe6q7jiZlUtzeEL60ZgNpeQPVFibXURh8lYXCcMFjgmCNbzim182girsneerc+LcgT3cekUDdwACG0IfW6rYUbDj4+yxs8U96DGZmmiGgxQoXgSEmyupZaas4vQmaZ3\\\\\\\/HYyVUJKGp9n\\\\\\\/hqXWqn02LE2rbZsS2Tzezf\\\\\\\/zWIWF6oyNhCe59fVUb1tTNf7uRILt0tMSi2aekZxkRxEJ5fZp2y\\\\\\\/YH9zBjnsjB4vIHPFmx8zLD8lgKzNxld+PIRXgbZKtoeS++2e8kz34WxhXZGzLO6Nbr5wJFtRvQd+7HWu0lntfYg7t74wDKbLcH2usyiI1JfnSrnlF+KwuGWakdevgzFp1FFeaen1R660BpiyTzVWoAf7J++beRtC3vs1CaGL41Pfi\\\\\\\/DGTkf+fbe4Q84tzz9Zbkpn\\\\\\\/f4Fr+kE8BGsBHrh26+sBbaG0IT87FJ8qxrNYn788uS5BIsMhf1\\\\\\\/iUN1SUjpmpfjkJDUQ1W9QZICICGsIccRTnrUoznjtJWzp2Wygfxkhk6CIpJEOQMJTqpsjZksS9BAKdMX9cljsJwreIgJyR9Xa6QXBfrRRUXSE2jys9HmJSkhqknml72+riN9gSlC+cmliqztoHUiLqgJ+qrqwiAJjaZFiA0dWx1xlXlf\\\\\\\/s5H5xC9em5nwzMNQF1ClN4arEVCObukp3DYUwWTsUGPSWAm31B1JqLdMXL2uKvvohPmaCstSKIClbHzGh0TzLDhWJEkuRSbpuuHnkBGxdHIWtE+icRR735kieTBHOwmCXtblAKTDtflpUtZMVsSnurapd3lwpAYXwoCC7\\\\\\\/V4VdMIL\\\\\\\/imrGybC\\\\\\\/3IQzclYKj3ced32Mh\\\\\\\/t2Pt2JFW92kle2SphyHY\\\\\\\/KhXeziShTyErS+DZBV\\\\\\\/UfyeT8mgX\\\\\\\/d9AzBcgT3MHKmAfRHLMN8plcqxvb\\\\\\\/va\\\\\\\/+6nuDzjtmULhBwJzZk1iApvt\\\\\\\/IJt5NxGF72Kb2TDspSiaudUF7\\\\\\\/qTuYHVusE1iWL8BTVo0mPL2iaFA5dJsknR3YVGF8mLF6Qx9uimbcv+V3p+gcKuVnzuG2WOHx5AGnOE4xcpBjrY7y7pYM4es4X8ar6iZ2TLIN8X4s6F+0NZAxLvJicPhPJEaQH399R60zyWgZ89WXaT8N+IE7G6G5iaTUujJ7mwlR6ZeMUNMelJksRAJx0QN4KeQynLh9VouFXxKJ0YdWvG5zhroS0L8xj2T0pEs0QHj44HyYIF873q\\\\\\\/MB2pEzF2AzF4ZYJQumiY5OltRD35+knoBKwpPUtZWoGrc55eFtJstYLgRLcAMIQ4cbN5mLlVsdrVOXDHDP1P8visorW7U0dZTvohXl8EKWd\\\\\\\/zG+ZCaMdbPplsZvIiSJOGSZ0iVUScYWWeQ\\\\\\\/iRIwq2YlgdQ++Z5URG77bB3jY0pHI921BQYugnUlEZ2l2W1OCVbf\\\\\\\/+FGkvNrjKS7EmRYrx\\\\\\\/Is6EJApKgidugxpizCFi4IuCKI\\\\\\\/N\\\\\\\/CF0rPaFwuCMYPLymk2LZHZiFm42s+1aUJrwnZAjXyn+ISAVq8zw5h4OYQBM3XCHGfg15xqwP1PUZIK0Q53kVSBcaFIg\\\\\\\/64f\\\\\\\/0C7d1ywEdRDLQvYaFqyoTBKk0Eg+VuMJY2tA5juOhncMT9qC1iY6IEFl92HxZCi5+gBRt9q3dUK7t2JKzAuQG2opmrKmSZMBGeYXMNzW6X2ijBgp4xOrxPSuNaq4GteNpg+Hpit0A20acdLHOLf81hi5f4+VDd2h+AKlvuImJ5EbdEpi64OUM3J9a0FU8oQCrKNSxTW9cc\\\\\\\/zlZ2VplYn99rvYEA9vrs\\\\\\\/EXhQvw+UAPSMgh0qif4mpEej35tjeDx6L7LUIWsfakp8+fPlkBWyEkhDAeqcWcHpdt9ihP30BzOwQfsRJ7SI4oAv7iHWixp\\\\\\\/7sfNnokSQZqVGiAMKmLaWfj\\\\\\\/7pWzf7SUAd3IEyk5BgZzQaWXoY7oE3fVJ3nTiXQKmD53J1lMtVva9FLIzUs5Sut7kn36t8W9RYfj+Tv8E7P1ap7LWRMJ5Yr0brDThhihs9oSVGBI43aGSqYezreWB8dha6c\\\\\\\/WKuCa4efdqea+8VoJAvyyAbAU+iIQyC3k9BiHdPoOuGNImVzNrVpA5g0BoFquWutiQ8TXTw2T9e4h9GMyFnpBQ+pQ9h9lPT5AdYEASuP6\\\\\\\/EzBu17KkiF4eKhJGXuIXhnTUxy1AWGJ9\\\\\\\/PFKTN5m1Cl3mnJZvr5ea10Y9\\\\\\\/wb+Zjx7ckhoUrbxyG61SZJq86Wc5EnbhdZZf5V4tk\\\\\\\/I4jptBgpCI8mG9pS21gJx6zwk72FcN\\\\\\\/KHtaadELjsm+nGBfClJ4uky9NF54qZkTrnBMeFVkPe4ir4AvGpuQ4uBQcRVbsqHXQ5fv+1HL5KtuUtGvL0r6+mjZfPgYjeuGa2bqFcCg\\\\\\\/WoIkfQCwQDKhI4BP7o3cweQeyNVgvAXxIN+zpU\\\\\\\/SSO6Q+9xxx\\\\\\\/cQzRTolEiBIBxFjlYrjpaX\\\\\\\/Yz7RyuH6v0VEN7uwXX0iZ8Utte7Az5XHapt8LINVOc3S7yfc+A6gcEI7omyFIRUY9vSYAvfgXVHu2M551g44u3ApB6LXfJ529266e9\\\\\\\/dB+BCdqdaqgBjYv0uTum8Fvim6HAo\\\\\\\/lZ3aA+I1jlX\\\\\\\/gLE0x+UkwIq\\\\\\\/a+j7hvoFDHb5usDULSmejBUvucSCq0ZkCWIIyAk9tI5eSBbmCa6uk7dHGcsIV7KSqXciFMWy2IB\\\\\\\/yZWX+n5hD0J27PedrOO3nWyTfsleSN0IEnnzBYwI56Cm9LBGBDH2D6EHoaUk+YiOcXz\\\\\\\/UCyjvM9+Ub\\\\\\\/ki6u9Koz++F4qWTCMELC+VfIuC3ynu+xH348xIanusilhxp59Hvwn7wMLr+3CcD0uDdyfnENomwMWbzjBPbmlJLnBWu52pyrQcgy1Q3BJi\\\\\\\/0l34n14HLv3V+n4TLx\\\\\\\/l3XA7o08LjVksDTU2tPyZOIWbfSO+Wq7l8+TGflFZU8urXuqMGmeDStNQYgAt8xyMYpnGl\\\\\\\/YdYz1efW3gQnReWzW31p5LxDlL4merz4M9WhRZ4HFBfIm+w0qudhzOeKdHljmKXnFbjr4fTyi5MiL+BVeZHdW2DmTrOsNFSw1AHhRofo6ugrF0u\\\\\\\/Zc+2LK564AXZ3fA8IxN0HLnA\\\\\\\/slqDI8ePE\\\\\\\/aLKu1KJ0JK3M6ud0UtI+IGegG8J8eF89JmovL71ddnbhHIaH46Q+1ur0ZG1CyIoH6ea7iVuw45CVBG0NSHd0wRFULtk0LWwl701FpEmJTHqoPzxyV\\\\\\\/N3VHmuVMqfqsihF+7d7pFdhQ\\\\\\\/5YY9AC6Pg8l8P0lD0zXsJJ6FwN57Pl2\\\\\\\/V06M1Va36oH5bYzqnyIRQ6M9lvVePseLIy9Q0Maqcenv1X5l+E7Ak2TV6a3TTmoUMVvtzwoGxtlqpiBI4nIqHwXHVDbmhD06STmM3CAQu0vZt4zzY7A1u7utRPDiIT5\\\\\\\/2lV\\\\\\\/7IIi3LMw8\\\\\\\/+aOFgxFgZ7iTeEObZawhNrRhp9erNGFbOs8k1JUP83OLy8yXQOTvb\\\\\\\/7YDs3xkiu9VkpZTADfwfbkegPPGxJJnL+PuV9cJ+J4wsGjDcUe7L3URQ5T3G3UiYgTg5x96MZV9NWFwd53arCBmjD6BPLflB3WcaYZUwBt6byPhLDzi2AZTQXX4KjQti1lseS1F+WzEoLaR9eh0wnAycNVj3IDiDig0QhCgeXuRSIxS7x15RmiyQUcmp8YSUBy4tePpLqq7URNgkpjA9na1AFCnVmXiRU0mCwOC7rhfwWQ2cjcF2EupiBBXo7nWEJt7udynG9prXmkezQmjnzEc4r0gkeZa1AxhiqcmpWK1lV6Q9E42KUW7AToTc8El9DR2uzSFI3c5pusYLHZzYZziZYOq8n8CepFrXhiJz0ZGaYoGDgpzHVe8y6cM1KSiN2dClOovOMiwT2Qu+gje\\\\\\\/zUOdG\\\\\\\/JdbaDluasobqogv8i8J59fq8Q9F7Wthj9l+InclwdNfUABC\\\\\\\/oU22M6bT6btMOp4a5oQUJzjrdRYmSx5GyEhG5B7f6leWT42ffFUzOyR+5nYjMZhKJwC6w5AsBN5deJevUiEmRpGPAX0OlNmF+icDHxhKmzFBcjxF7AQCzKep06glSmivkPXbsinmQaM\\\\\\\/V8+G+TAXU6ZI99bT+p+PV8ZK7ovrtbR81DUTizCDkj8vXi3SKcN8eMouFsNPT4SJCdXaytswa2dn+0NJvCUwV4T4jNE3O\\\\\\\/PqHwffx\\\\\\\/X6JBZNkD6RDaiyLdQ640JAJfFcHdOtyMilFtHuVYHp5i4bqZPhtIQ6tLOw18yRhFWAWZmkKjGdXcfcsNmijXYy0z9jU2TLilWxz8ZNwctdWCn7S\\\\\\\/Fm1z8RyWU9fOR6+rtLcn1pq3dAhHBZZCKMi2dg4Of\\\\\\\/ikAdmjSeY2nFTR3CVmFs4504DsYPC5DpXmdnSBTViDUTRTF7bYGbe6ftN15S2ABpKyCMdq1IiLg3\\\\\\\/TwfR1JsvSkgLSQosfWgx8zp3bSm1zkL9KLl0wdsjoDK9JzD3LyRcvOOnW+\\\\\\\/3LFfLaK0un\\\\\\\/v4jUgrKL5m5zguKfdYQBd+G\\\\\\\/N8\\\\\\\/2oCDcFJ9iXKNjWCl8t0HVPCHhMPqvL\\\\\\\/Ps9XWZXE\\\\\\\/QyN6Fb0qalqMUCV2LOMVn5uIUh\\\\\\\/Vewd5yZqK9YtMh47dNu\\\\\\\/BYMSC4pro6JP3vLzngeNY3NuaRuzJrbtJRiO54ugdHZjOfbCOfsHn27Yla2z1NG7cA7rbomOKcVvzlRvaSlFmdHAalAFoTGbhwx4VTVjl1ROtt6GJNyayn+iWoT\\\\\\\/3NUkIhBkkbDEmqAikIP4w3dEiyz5XgoiL\\\\\\\/VwFYR4GicSHDN9gb7npIbvA38BTbEZ2TyBjikNrZT2o6XiSf8CUfqQ1XQs63t3VnBbOph16kB+1ESPGjR1oSN\\\\\\\/qO0sQ00UjQ1h1m9AOIUDe7U\\\\\\\/GNPhfvTir1XRqgwORwP1ZJh3VfswjB\\\\\\\/QzEdrcwR9ENbQx9ucdEe+id8HKQh1ViRP46njHehcZkrGsGZHodB9XfRVNF4kz\\\\\\\/DKJLMaHzQwsLEizYtjXPE1Y9JuaYT6gKKzKslA8BfbXN4sxAH0LpQ+\\\\\\\/K3neUOuzaOeT8\\\\\\\/V0fUQwZiOPNLHH3K2KHR3F4aU2UZ1p\\\\\\\/XAn9CXov8KS2oiki5535upvvVfuE0Scx3YEZy+fx64ptebc0d20X\\\\\\\/NK676kY3VnK68xNEOhjY9CuqhOpbIxVLhpVCRsutgP90IJBmXSRI6R0gIQFELTtU7C34iF80hd12oRpqQ+kXL7kziQhP3\\\\\\\/J4ohFAmY7XyhHcNAYWNKsuajEv3xqIUW7rUJtQRjj97\\\\\\\/KNLCT\\\\\\\/hzTJGfiwx2l9XAMg4jU\\\\\\\/nwVK9wIoOCzCjCHYS3VRTTIaMz+YHRhRspVo98lO1p5Jb11SVFX6tLkRGIfRxo41osel8x0u1QbEUc+PhLDJpVLchAJ44OL4gSTtwlHEPTmQDFwnYetCi0+fi5ReW3tPdt8KktSFZuOsfn49lt0JYIU0FpnwNvkbuxGhtsw4TxZmbHvNngYPa3YQsmWiEcaass2A7vWF8G5KpgiXdhAxECFVgb7ZYgzWZT9DXm2kcl6o7S+AGpSNvgSkOGeFYzuPuEUzEhVUqiUh8uDK3Z9o7wW74XTONW\\\\\\\/n6YXUqL4NBmwtUamWh4qnks8mOs5lQg6px5qn8wkpDflpDLYAbAgd3r8iuaJeUB8CVXEVr6KYNhJpxxKreE45crIYQ2iUyqKA6ChZOtcJWITx4VqIBMDGU3v\\\\\\\/K5WajiNT9INQK0+Fg7CRoU+fMeUb5YCahkDjvDdtPkms8YakB0ZZxWBMjtldm8KmHkelOfpEOM8ievn7Yz7muLzVj0Ufla+t2YNEeTy5bEvBd9XCSoFxA7CxIzqBgRi6Vll95cxaQFzxM41NvycW6WKMquqS9HVRPHI76PLOqSSuoMF\\\\\\\/4hwB9\\\\\\\/vqMHJukKRQBD5qz4BQ8KkEHS5sxJkjMVL5DgOpn3myaqlX3F1xhOXSSmXT5nOcDmi6jKE\\\\\\\/s5D6bwAxeGTTZgmNhi2OrP3Ufb0X0qNJmhPjQw4caM1B+cC3iAbCwrvQuMTiDTRvgWQpKPlY0Ay25brnbhFMoqRnztNM+rB5tU0TbW1H\\\\\\\/4I1pmpSuuCVPhD\\\\\\\/UR++o1wFUSpBcwY6TNFbfaYfi1cHkwrxBkRaehvzd+U3nl\\\\\\\/AlqyY71eBRVMqS8mgHmpmagAjhxQFdJKZHcN3UrZHikJYYN5ZiUzAjovKy6+egtKvB0CVmrkmOEAsKwVBZA5MuFd5c\\\\\\\/GBTGSGh4kC\\\\\\\/lbUBiOx9Q+VKqJoGqcSuTIqBx7a7o8d+zLG\\\\\\\/2v1tAWp3k3LxSwWysdrTkn4ndwJbbs2iJPdaHkYH\\\\\\\/8ldYObF7NE\\\\\\\/K46nZ+D0UVCKplD7bN\\\\\\\/tiHS8iJk0eyWT74NcQTKyDTBvGrNzidtVLz8kEv\\\\\\\/dIlGqtOCN6tbrlnq3Yg4JByvmfuiCcB4W4ecSN89rX5qvn4T08gZHQQk4oDXa8lqXkJZRgXdCqwajq4\\\\\\\/yoAHu7l+MrdYXW4rxNoZ9JrPMYB0vNTYFLuhTRkxc5J4lsvu3Wvtc29VN\\\\\\\/C1OwRfolqxwvYQHMRZHFsvyoYiMLDEZIVoD5\\\\\\\/BLkQKYuw4LO2IbykUQk9i+RVTwMFxrGaumhEV\\\\\\\/MT2+FJEja19KD1QXCMupDQGLEWO2NvqILxG0kKrvPuUKBsn4xwN14F7jabAfPDIRsNSYTSJLm8UWzeBFGtPaYDXFuHp1wz\\\\\\\/OiRnNFn3YufK+AAJh6vUZGyXYc7XTeu4Dh9BHjiwz9BvgdLVGCZi7WidDQmDv4RRsMtSwc3Rco3v+ZWtpMVBKW9Av5Tc6RbZz+p5V5PRgygukhBRyk1TIL8dy04Glx5mNVM\\\\\\\/OAkO44jPs8PKMog8f4MsrS6buHjn8M0yN+gb4PY3irXygxPs8ue\\\\\\\/kvbTAXkUPk9g5VrN1PsOMeBnd9VXZKKv8mL+FqRPZ6nuYgKRZV4IY6vSDmSGSsIl00Tq6AUJHkgz4DqsRwg9wqc3XTKold1uF87ncKidmnN8aPadkheGkAFnQofB2KpqncsTzUB570T3JyCAU+AwEnObEJcg3nDKlBAZEACfq5iE8pBo2An7QIgcRdBJBe4\\\\\\\/SgUiM4S7\\\\\\\/O73P4tfQHFs\\\\\\\/CC59FTW5A\\\\\\\/8tIJGxDVmh32iYPfOvdXeA2x58TrqlRLRbLltJJ0\\\\\\\/xFhVXk1UaKWi\\\\\\\/z96loK3nJ5EJzLcPFHFUxzjx75BSNJDiqiUC0ZeH42aCTGGTmIRSyYE7Dw\\\\\\\/LWsKN1fexgunqkaaZFLLbHma95YkwVSENYes0UoV4WNNgLogwRk95fmXrdG7E20cKsv\\\\\\\/5ofinyYNPVxcz4vUBhbI72gBmKwTwBrwcV1G34vrwf+21wXMPoXJNmmMsEPPwkZ3LnLy8l+uVoQfzdDwn1FaVEMAAhdQWGbp9mOsWI7oeS9qWiVE56lWjqXIqMiWSEAiTEs4uIEni1nsonfHUjwpQj\\\\\\\/xX6Tv6jVkKldvp20TxRf8i4ISGMjlbDsktAEg6VLA2\\\\\\\/ZQOhgAch2s88TxsiV8koMoIWsKYTjMP8aBFtgNl5E4cdwina4M4CJV+J9wrHtR2j2f8kfMAh\\\\\\\/IXumB6kP9iWPkb9aavHUklv4bzCxFi8XkzAK7+0gQMi0yGC7vpe+F5kq\\\\\\\/Pfq3Aw3NB8v0tE8s1d9hrn3dr6kzRo2TlK4FLLcmLKBEn1K9qC\\\\\\\/BWMt8f2Xlrzv56xMdwkSOgaRiVG4\\\\\\\/m9t49+ygdGddntyD0frl12RQtUXd3J4wp6cID3L\\\\\\\/iok6K0Zo\\\\\\\/GkekFghsA0mxH8JTXHL1qZjGctjUIbnzdeElopRn1PCnMeMCasJgrmlrCOjjhrmr5aGVP6ZcsiTUgpvvayKZporWbrNHUqlEgg8m5H4QBk5Wvv0wz4cSaZ7pKQ7csVXXAvrXnxMbui9drOsjnoQvbkEKsBk\\\\\\\/oRC4DUO8T2wwoygfewkLRvHUjpkf8eBhIig\\\\\\\/P3atIPqIneUO7ZUKEJOHGdhYTbVJM+dkRXIO8JHlKFSSVzrODzMjDlsUCCIi6eVehOBMF56\\\\\\\/ahPrgeV7zHabqgbmkq861T65oiDx2jghqSBEKedhB7kspG0RVb5+nzbHrRmWUNkrjUAA5BEFonTplyyG8Yh6TReaIwImjeP1\\\\\\\/V+VFND6Y23nCDp3mYA7toB+LtOQKObNtAb52JbbC5TJWQs4VxMfoLPAb7gUxI4ScCVEBWGuc\\\\\\\/dtd9mDr385kF6J4zJCOpoY\\\\\\\/+q3uzwobNs3MYyQwELmc91zVeRcBUTIG8tqTBGLd56vB3tWLmmRFIDH5trX7SUxRgmE+MZkeCQNM6h7r17MjsNHmic90+bbtX6lpZ0i\\\\\\\/JXd+ii2lM0Pxx2bMkv2Y5YGeC282YqJvjXI9MdysP80xrneAFOXV06sqfpmkDUN7eGBpTeBAYC0htslbdafB8LAk3iNaq70Xq7gKexzxlcEYxnXfQTRfHh+OS2IrXBgjDDTjPB6PH+kI0MAu23\\\\\\\/zRjNZRUVl0bqrY\\\\\\\/wGMky1Ob6pxrVvOSkyWROsR19UnOK2hZglFmn857v9swZiMDzpyX30YVvo2btHzN6QeS7EP6tQStDcRIPcvH0G9VNiB5FzOZvmE9VMv6no3c\\\\\\\/HRhu6\\\\\\\/4hdy3LsSu1fDIkRzj2MYThD9MqVmrGh+QwF0aQSGq3IYpfzekp8WtJ+KCAMvqXkhvrPNDfMzZW4URh3F94rVVYEz9Ww0OeQ4u91Ijlqu+0zh5maxdzAylXZ+FEjRtyuFQnIvfcgyvsvAP8qu+elBySj9sHXoEzjv2BPaPYRXdVchtGXXjw7s1cD4vCPttIcgXbPGUmXby2bEamJWNEt6E1e0atpy004\\\\\\\/uPbdv7FChLjSNiX589vNpi7gsoa0VUEHg1\\\\\\\/YOZN6JBIsBE0\\\\\\\/9cQbK0\\\\\\\/dPh\\\\\\\/XK7NbA5uAaU6L5FLQlvaXyuin1dLuwAGoja\\\\\\\/dBGlRoZXYTQ1M9DvYN\\\\\\\/pjwEj7ao0ABRchzfBTj2UcTf4PsDeM8QJsePxCRMsvYnXAacGBDaifdnsBEqpX91zwYNyNi0dRE84w+Z24E8j7lo+eYk2HIHAruq4aLqq5t87CYhp+ydL29jOsqxtYvbdRuhsHRGShjbOBlIAWLND9ME2F+bGDyaN9Ranj4+mLd7fwa0VFe9ubwYd\\\\\\\/QWKpN3QwOjvZXJEbr8sGbXqcSUiPXs49yWuD54grd30eVgwV1g4JDfPPjz2YmhCFsahubSzBCvDkFAZtETBgrUOL\\\\\\\/hOKraMvNF30YR2yQpZcDjumtHK2tYi2DKBvLI\\\\\\\/QozQ5ZhzLwFyD4iwT6+kKQxekR6f\\\\\\\/r1zgmIDOqkstMKtAPBy11zPJgHu89AW+nVzHpdjWRmTOzEY7DPKrviTIf1ThfqG3gcG\\\\\\\/CXxk3LbUzb\\\\\\\/ZHw7ery3uI3jwfUIz23Gem38JYqou\\\\\\\/SWJiJiK14T\\\\\\\/ychiDlvQ8uE8sJjZ9W9BF5UJldCJi3vUBs9O5sWMPno8XvBHB39AfTwP3XWhZisa8Xki+MsqHzbfx5Lk01P\\\\\\\/Y28ijNE7eNZ3foc77JsXrpMSGKSCpnKCq1XZPIHMo+glGhnvNfDwAJcSXNc9byu8p4krWlg\\\\\\\/tY+2Om2S7HW8JENhpmOWcIGZwsDQ4dG3ZhhTU95Tz6UEKG4h+oQ66ZAe5\\\\\\\/J\\\\\\\/WApqnFzppNTn84XNB7yxbvqJW0gf5e+aTwJBvB4\\\\\\\/lnSc3b9Gkg\\\\\\\/9pAMx5UtJJTsix0PtCypWwHI4pCe5QucHVR5AMYCcLF8e4Dlh8vlnKkIIdPesEqG1dLEw3J+ujY1LeifyeiKs3kIYuc2EErxbfuQv7NJsteoFlJTUA9CvNFHcARt3T1AbpWtb9Ch0w0imds8z\\\\\\\/YZ6Ox8GYm4Tuc1l+nYDyXA9jGT1e46\\\\\\\/BKchIOK5JKFzF8hnS41B4yg\\\\\\\/czsPcWOQefewqTSjpuWVpjnHKJbq34Gwb4msjMob5C9fxJVEYsocOoRXVkD3IY9eJ3mgp+c+tl7SWTo7uDUk\\\\\\\/k7L1nJSKWGmz29NfSLLoonloBBcRlOH4YlYT1A5niETj09gRNpyB9MXyPgcQH6KeqHx8SY2MGSOBUJvQH7XPezQE0y4oBcHuRay1ooMzU+KAU0Gp769wJjnfO6Q0skD0PDxldGehEiWe7XjPGFCPA4Ar0rowVxlLWCRYqzaXyz0qjhH4VH9cPs59RdLfZPVH7cfdhMRhQotqqURvFKwLDroN6SsTKRIoIKvOAA570jmSyZDYkRor5PsFWBBO0XF7lrUM+lRXrq3qEzSyjjes1wn+tVkt7S+uZgK1HhATCnIPe7MdzM8sIsPasS9p47FquSAGXw4ek1GM6MlzHJH7amFPLnhkTwGBMtbHooa17kUaevxjVi4XQljsB1x9c7Zb5FidcavNG8rNfM8j0\\\\\\\/UjaG5dphDkwvXHl51UhJkGGXoD\\\\\\\/RFsG6YoIO\\\\\\\/dZjRTSYXv5h3br0RdeIYbpPb3tlDZZX3mJ1lH+yyZ+j\\\\\\\/z2usl0U33gxBPMP4yBUQu\\\\\\\/2K4FFvqZl55cuOI7XCqQ8XpZy3Lj\\\\\\\/xYM76YYFt2K1oiaHe0nLkSmuZFPbtQwSDE8Lp9MIk0syRDio1JF2Unu2r0XI6c87DYWXDJl6Fp7zIka3E9dgwH2FMU6owCFQoiWFtNyZ3dBl0kQ5KgMMlqjdZTafErvvmNtIy5CYtQWq9WpKMNryWNSAfxdYp4ZSzItVBqXnRuiIHps1txpuZhxuplOVX3XyYOEK9DsAeBEZia81fVS07qs9fZuvt0OmBHeeTDmGNZ9IbS0LJZKe7fMr8bgJKfUwmh2GqNGRW34\\\\\\\/5ndQENbie8spIDFt4HCbswthsQrG546h0QoYWqQIfIzDPBgQ6DPIiEG+JdTB0CsoQyXxbmQf5eroflSVAs0SzeZL\\\\\\\/vwarh+jxBcRBOJEyBwznktqBkUTOWeebeUVdB3esvj+bjNSLJftOP+PaVNgd1j4KSh9WAzD8rXwEZ8IaY1+PsCpfLT8NbJsX1ajE11hfLU3pg7IAX8rLMtQaAAX8yy+pSiQ9t1rXyNWnHZj1AgS1LOktFUo\\\\\\\/0qx585tHk1cAHUmbTkf5+zjQTA0tEgr4gZGucOTZfzFR7+eviiEt6zr2fyfSSY8DnFhusSyg2zC52XXDHOzrlyHBL9KXocNd5tw5\\\\\\\/vlz\\\\\\\/I95SGVRQ2gVVS63CtmDIbOTKQZZnCLJK\\\\\\\/NF6Beb9IjeUY6Wbv23xaf\\\\\\\/jpHUJE6j3tK7vGt7OkWEkg+gJwrjvvfwaBW+Xy+FraIVs0fnCq2xZiIbie6y4HVhdFGz3WBySTXW9Auh34xcKdCdZgf2MprozG4291Z\\\\\\\/PICjchTheISp5aXmQaWVX3T77Eg7iA9aMzumaD55WQ8QmuFCf6BDGjdDxQ3Z9PhXNWfFQ2dcb8sIEASuIAtepHxWkKLfMEKgDHqyfBJq7hhoE+lNb9zHBqKMuTZYqdkFD4MlrazvITmGo6v55+sVAG3KuC0JefAuvNjWAzWIRxzdysoNBu+vKmhBJXSogy1TuDIcjFDoykIr24SLDlqPiyZ6yxIvM8NmB84Yy12wbSR51cfb21AdHRtBjKcf1UvsZ4mtExlto1KK44zViFlAD7kqjnISA9Ek\\\\\\\/enMA4oawhMFDFGRxJaf1x4Xjv\\\\\\\/jQlxQ4DyG2Oxb3BMI+pU7bzvubH3WwFyvlLtwZV5Wsl5Fu4XtN\\\\\\\/BFJuGqaBa1Psre7T9XprDlWtKnyMxnegoI0ngjq4JXzWcfvHu0MCcgRNBe3A8kfuRlfFPrq1R4gyxgIc7Jc+nUb9x1LQIq8p5D7EFQ1+TEiWSB5lC94IUR4k+9AOVY7H05EsCMlNszTxKB8WfGlymnMT7Sxa1HEcSxwB2Wuaa+Tu2sKIa6ALOD1FVjJYsjqUZceUbParWqoSPHa1hZ2+5\\\\\\\/XGxUrvCx3AYmMzKA7TVC81dV141A+wNCfrirKDap8jeffpLZpWAWjb1VzoJEMrW0cjEBzzL+EwMDJv8tfZlmHc5MsMLnOtm9nBJvospecsFqPLSVj2h81XjLzS7wEC1smDNC5Y\\\\\\\/WtS3KKoZ\\\\\\\/Nl+1Va737bAQRspHPxXwQLbPdZLFYlUYMVecSeBXP+W3llLzhGHh7P6B1emYrcd5Vl97jwPD8TCSJone1C17ejdS5aA4ALbzVHxtE2lbIzwjqD9kx2NEwmqXAOAEAwUAdzJhctWnpnPiJkg49CLWOeiC3rptWZc88LZYUVX60+qbnSA0rsdHGOYgqaJ\\\\\\\/v9PVUPjvw1LWveB12GpvheYF0i\\\\\\\/XNh7VP3EY5ZlZV+l9sN\\\\\\\/IFqBk3wfx0gISen2Ms1szSrce+PDuz30WDAtiyCeJh2ZhJe9\\\\\\\/WmT25gbA3UVjSVxviMVor8CPr0pvUh3fgBM4i8rUImAzMV1prJRt\\\\\\\/wydhfJjyt22kTL3Kk3vKC9YmC7WLIvANpaKzsY6qoapW60Uy7xF6EM03OUYmZe71ByV5LZe9+XIeKlmqcfCGsGoMKqamZ247FJ7L5xUELVEZq6W0zm40c0Hf0IOAA3dw0ZSPlvsVDqeRAXCyU02FNQLDTPjqwwWU\\\\\\\/en1152GWgCPQILF4uXgkhVsqYkjuWyBksdLK7iivwwV9SF49O7dMJJd3NEJNtE63abEsfDWpcAD3Uro5Ux+oCCttXE4yoaxi+ExRlpJZprmdNGkGxjBmaEShjXmmGUvzF9Ywy6wrJP\\\\\\\/YZTyxP4i3CLTbHm+u+gtzvy4Ty4AARw85EvtsSP+WobpsWboeSGaMXswopjqsotUdZviy\\\\\\\/kBZ2bnP4bJIr6kNCRvZfWKMT4irQnFoNhALAqkm2Y7oGa3Q9v98GN\\\\\\\/0vjXqUB4IM1+8Iffkc57y1wq\\\\\\\/CZJJsLP48XRTDREs76CH7p\\\\\\\/+RhikQqmK\\\\\\\/TSo5dSWzBxz0MtJ\\\\\\\/uvbJe4iSHNSOhl\\\\\\\/pL3KHM\\\\\\\/8z\\\\\\\/JVte3wnq2ZGQSYXe2cISOevqBo+bSB0ACBcIfbe5A3NhzLsaa6VPRVtBVmt6a1f31UwBTYBqjzt5kM+ywExTc1bYuyRR7QWm6cZH9aoPGRhlbgpyKn+2M1zKe8lm81xEHS4aCUtTUfrIe5nyxtUPA5QL9SAtCp\\\\\\\/IrqbnyA+Tl9Tw9C7VxExAE88EGPwUZRzXkc45hQ4kqv4uLmU9kPHCzz65ubqAzubtN7X6JeOzeovBD00rq1ku95uElSPnHPsEDvr+8rm+T+P53zL8mzWjPHh2eiN+hdkrjVbxMHrjM6xH3sPWSKUKrExf4ZGxZ6qW6fprNkMgKTp6ZBsmsxP4bAoWaM2Zk3mjZIig7MgQVVsYcm9Wh2QsJ6dF2v2J+pWFtOAbW1Okrr6AjAvoCnk3nGli3UDQA6PDyN9AN6HtViKklq16BYePm1boxjYJNLXkAAD\\\\\\\/A\\\\\\\/saL+UsdFoTUb1U2Q1HpzWXSF9xmUFkbbrjmmL5hRhpDg1qOFS+nR491wBRUnICcJnuRxa\\\\\\\/hdhTBrylp+jle9\\\\\\\/PCtLPvGr1cCx+oEFL1\\\\\\\/g5rV2HezgjDzh\\\\\\\/ZgksiNAMm\\\\\\\/NpRItzv3L0YPzW7IQTZfhr9ZJWusLAUh0WW5LCPu+qMFuelXu6jNt0pMKeqF+w9XyaHlKtK+AbiZ93E9rSjYWH2ucwqfdMjp8idht+bmgNQFlKgwG2oIEWgE1aLI40PO5gsPw\\\\\\\/SI2mxRayLJoRfUMJ\\\\\\\/tZ5m7wK9JfIjlSMUXhACyLn5SC5w8gXlQGl3bQecIhOIxGNvoMD9ARL8b7CEr81WxEbGMZ8AR0GdhZs4MSDzqven+nY98XotxfZ3zy+pGeBx6yoqSsKIpp4sZqgRe+QADWek+3wBuODCMJFxtN04oeKJbfLoV1MxTZ0uuZV82ZIWKEklCAGWy\\\\\\\/M\\\\\\\/7P6IX97LHdVgkVEXVzQ7Pu9PtA2pjKtT7oupyJRMsRV8nFB9wTx6R6jama8Mnj\\\\\\\/ylQXWhzNTOBOARlmY7i8c26M5\\\\\\\/vVIe3XZvtoUdtPRg2wzqvpS+Q870bUAA7ko+hkpahqpmW6byW\\\\\\\/NCJRfhiJqxvAcw69BkyRqCrJ073pVsSi4Weycxt6Bqb\\\\\\\/WiuJyBUOMPkHV2NMw9DrVmt3GoletG+PdhWKrH3k4SS66ycgnJop3sk7G1TL5+T4O8MF0f2EqQbsQomp2Sh1cL2McJxbQD5WqO+X362eIUnV7IktnblY4W+xmW7CObiWf38\\\\\\\/3Un1brjNRYuRTO5ncyQNt5zNDFC0YhtOCdu6IbTJDToMpmXyVRvd5dZR4tJdSK2RI2FmSdxfdA8rjtQtn9zL5FeA2QHzSTQ9nBnpHbKWgTuxYw3IULq9QbGTaQmLer3Ko48VMMbDMZ5k\\\\\\\/Jn1i66FmNeEgesHLB\\\\\\\/EIKmSxcV3x7+\\\\\\\/v9Q9xSNqT23UT39MKN27bWTLZ+oUbGr5xQcb7aIGEjkr2XJlWpCnCLuIyB\\\\\\\/HqF+WN0rOhIwA8O91mW4+q+EEUhPHwbcKRsrAINeKpzeBtV4mTqPRG9tkAJHZ0W3eBkycZVAGL9Z9otTRm\\\\\\\/Zvc0Gy+m6QHxooVRXJV1\\\\\\\/UQkWzJ3SaLu51ryvXLscwZ\\\\\\\/37985uu7IlS3oA6b87DGs8u4hn3uJvIV5s+Y8vnxNrYUk4+wlmAvRh\\\\\\\/friyygaIU+L9uyC\\\\\\\/W3sMGqSHXmzXhvJSc\\\\\\\/1U5SBbdDMJMILuIprhRhs2Se\\\\\\\/4UGnQY0W63jt0oohIjnPXTS7Z1Px8aRZgGwBT728Oe4TWZ8Sc5ZChJeHLLKBaZdjW7lg\\\\\\\/UGDxJSEWI7Zymktv9v47uIUqVPY2Rf7MTPmOQSNwmBHNkvo8Aj36VdKQ0aGkfL\\\\\\\/v1l6vVdJjmmDsvVJjaqH7PDgX7JXxCRe0Zr04wjh7CqU5A5T3AUEuwr1XM1rcallDPvQxwB2Zb1KM6m+q4FtjghSaqfWHrEQldoQ5e+ldBRnarFZX+ZZMe2qFUEWIXZgwoYhaxzZoX3tpi0v9o18xDmwt8RGAy9cSW\\\\\\\/tznCQQRuoIVyYcyEG34o3Oe08zb8luX+WcZ+DU6uzUgb+ccgT9hZczJjaP65tCb\\\\\\\/WxQsmtNmE+m4NH8MqTikcrqFc1L5G8K0q51O47UhUa\\\\\\\/dntPM5vd8ZoB4K6vQ\\\\\\\/Bn1HyzH4jAyOMUjGwCLZs4yjMULdBmNwKdNCRY7tQiswveRNVZs+ONsHyJWNEoWwcchdYeibXb9x0GJFxlGFFSRM7QlIyRMUqQiyGEN4QpljKaex6IC1jkQBXFs1EM\\\\\\\/U1VxQbESx+9JnHOTl2+20klrxz7AHNQlLyhWjW1BhPDJsDj4hxFzJyFQMQBha2keITfE0riNFuvAvbTTzTz+ci2hCKjMhw8e\\\\\\\/4DHQ9vZZ5CnXedlkIQjEiWsVdx3eBbeuA1xU90EMoj2fmngRXUm7w0lEIm6+A5y+Fex7Y3i8y45ba19zGjqyJ+HwsuSgbHkIQjhzmhJ0+Z8OpB\\\\\\\/G4dPf927PW9SkuSFjI0TdkftcJ763MNx3UcYAzFm\\\\\\\/XjObfopTgZkK0O3+9xcYpTsjJuzTPoLCZ095BupsZML6luWTAzEpxALpWJrDOIdiDasL8JrxuyPjI0VNlc5p1DGATAQDuR8U9zj0dXRp5voPCFOHynrV76UvoWocj2caE4NLnUU2OUMQSAvDY6NUPJdHErQBZHMJcMIoYYgDouYMsbsF1KH+bzWZwdIaBu1WAuTn+s3RLetFltuCeO8e8LD\\\\\\\/fvgdxjdydB6SZHb24C0fG7WP\\\\\\\/4IUsV4CrFUsb+OR1jMkz1FpJctnqCF3iIww4qY4AswWNfCcKyafjxKs0ObdjfywL1Fv5dajIekg0PP1LvCANQdO+Qla4EqraQ2Ro1JV+YbDfjJju6O+Eu3lHk8et9unGfBjjE2okWS5gvpKNKDF90pXHULCMpVpVm+LTQkzJmL1hfoG\\\\\\\/RFpszKgOLEWYYWECx7RiTkJe8elE4laiNSexlBUaGJYtRNz9SYtB2RfnDiIDbqUJIXTeoODdy2X0NJ1yxlNm73lgFe0z6+\\\\\\\/9Lo0Gg0j+4vxyRvjHZemmAZZJWwRUTExkoenEabLw6Ps1rrG25Au3FW2iebgp64VrFzc3yQihxiAUcZOvZyO2bu4PGk5\\\\\\\/WIq\\\\\\\/Jj942Eu\\\\\\\/SxacHHCxB+EOVXWXccONwVLOYFnH8A8ZvLzOYj\\\\\\\/XqEgDeAbwYbGLQYq2RJsCSixW2TSmPQ6dY7PD\\\\\\\/MlxxJvVgr8fP59aTrrdGp2sohJdM9AVboe3Dw++cDTTDQT0BYsZEWScJoADjPwUc0YiWmf\\\\\\\/ATKG4tgxsSAdiMWsNkFgUafQ3tYikqsSYrzSKAgpVlyAWIOBA+B5iXdDZYBPrhAciyaTBChVKJtRJ9Zw+nbvpYyyZX2T9XQemd+9wCqZ\\\\\\\/HOQXPChADMUcNJOjKtqKbV\\\\\\\/SpkSN84EHbUpg7G6QGuLBLDT8rq1J3SHVAS\\\\\\\/7wthnhtmubQUD7xiigtCoiTt23ImzocKnBtAptuSG4RyhzbFudhvNPLZ+2D9j6YlEEQUOIGB+p0qFOfzsW9C3DH0DcCH4nv1EAVkn+P+SOre5hkdlOCbU1No99SKMHy00GCKakSUjWbbA0V3cz+UtUq0nSaw7Huhww7EIfxwZ1vPvZqu5nsaj3BOc0ifafBZLM9qnt+SowSDSGxof3peLgi+WhDGu\\\\\\\/c4NTzwKVvOIh5Yc49ddgxyNExRpJ43yUBwXOfisszf9DMbL7H+rUijd64YrTnOTfl0dpBLd3ILVG4jkTpV71dFiFSXYZGqfPkA1Arb0lkRxy72pX7g67X9ZBYt1AVm7uH0qYSwZOAU\\\\\\\/B+W5OOqMeHum+JqChtZYLFs4i+VQN9YRU3ec1ZJZiSKzqsyU6vwRO7YbnugJrirqNz6INxv4hI9ZE7N5pBqnOZJ41Uv8MmAJjumwoPzA0ApqnlYDmbwdz1lmCLlO6GxLwUn2Xgt7HdCzSmrCPLOBh1X5hIxdLvvIFSBs7fDok7MzHX2YWjawJaBsEDCqeOiiDeO6JoCFCp1kUUaq2RSk4iaeBNeHkor3iI7955iZdN9lyBVoxFPwM1VsGDJ5Kdlc0Z4jODpybUKDvzLTyw2Ae9Pnt1dR8e\\\\\\\/fvjSiXcRWnMmOc7IBuEwm+Yws\\\\\\\/tnqn7WXjd0FdvLW7UlX8XXM0mH0NZIEzfqIW4SS3XpPLXMaL0Qzd6XS\\\\\\\/yJyNuUUrhaA08dLS\\\\\\\/H00aTnMFg0NhNBQSBWLyYhest3B1zD6ZqqP+8X8VPf5AhX8CTq1Rp+0GbowzR5dvOaM6JLaAcrrJ7DRKFOcCCfpm3ta9+V9xdcm8Ix3OB24l2by+gF9fqnTqMBO9MZuaw4viChwP7MfIS9M1IfaecjeDdZUQDBjVn5lWQUr7PqGYgzkJwFPwfo3VYCE1hteRYb+HrgGdcjiGhi7ggvvH+GaJFkk9XpjGhI3gE3itBkQVE2asSDh+kie09kJhU410nVD8QjzEzmsRSIj9Uzf9hILq11\\\\\\\/UEXcMJBT7yHpKTNNd6ICsLM44+gfSi9o8PQ7g4v850ebbViXZ51psvwuut1KZR+FfyVc+aBfBmdvYwmuRXMOfcE2fbE8MFoU1KstlXLUy4UdJRGiiuyC+qqV3Kq1xIOVLBb3G8Kzt\\\\\\\/+\\\\\\\/IVVljsJ\\\\\\\/Z7qwX2PhINlknYGUS9Jc7raWj7cpRIm632S33EUqM1V0Q13N3+7dfvuY2XJPZtdpamwCXY71PRuvHkjd5cFpRLfun5\\\\\\\/Qe3aNOKQMQoRBVVe\\\\\\\/Qo0pKAuyGPfyNazIPM6yKWFCoUIUu7CwmSaQ2f1g8zuiI+Q587Rov\\\\\\\/qWuAg6Mlt1A4v2XW20MF9mFP8y2bLE8D7j+V9B9uuQnCTgHF98ohoqvboToe03uGaQDQzsE9Zqg363DvShaiSBmJlT\\\\\\\/AvS5UM015qq2ueC3ZsIsN6qScRWVRxXzEsghB2ocjwVCXIcTbt0fC59564K4RlvCKP6h4b0aKXzaRKv8aMMojlSgUrIyLPlTb07EkYBEVvWD79+jWlXyonMXQ4SqiWPaWb7QcQHHEnuANlipZe7ZN2bmbJLO9XwAr8y7t56lvDJVsnNFGthJ3K2VtCU1voIlLS4h0z+4rfURpkZAcT3ofaULAd9ffKSkqEW79l+hTe85ztk9hRPnDjB1wu6ezvsbhYQN6YyGB091MUeBeHvwfHvmivO\\\\\\\/kVKk4ZwnfP424zNjGpJLWo9vIZFraU92dzaC9JtnQdE3YFos7rAMqyO1bJwYA+2xr6bGccqnIs0L1FLEWRbCGqZ5ecPGHc7WUDN+lcP5y+DE5CRzJjR\\\\\\\/9oKPn3XIAwBlTQZ7RIklFaZT4MGs6xhJ+qqCF2REE8vgD8e1FkwgvSHcrp4sPx4lONSQeMU46J\\\\\\\/W0n7sgLGJSNWcbCdKGN\\\\\\\/S5RwfhY4bPXR\\\\\\\/f7+RaI65ow9fG6x3LhDfCbM200q77Owc999kZomDVQDgb1jmcTvnoTHVBvaAw0x5Sog2\\\\\\\/wZ0edJx3Q1HaWgH3dp6RyrzEA9m5wRRS1zWmpb9Pmbf99EbRpOpcgqWZmMLH\\\\\\\/wU9RDF23OilcP8fbSyp\\\\\\\/x\\\\\\\/XIBk3cp8xCFk10xF8h9Og44lvBCk+E4SlUYJrO1k7DodxFFlcKeuF5DpwOu9dtUYngVbrVrGlJtnXwu\\\\\\\/KisMoynnvqVdeJnMs1omBThnLrIVpd3+R8BG4P5DYuKITnMGqolSqfLXHPOAuQTba2e3smg46pmWX5ED987HIFUUTbvokZag3VSlb6evBVklPWC2TN86sMi5nrM0Bd8cEUWT6EeTDRQhoHSd7xREsjKrVCKMDppZbnaNApYe1OZVRl9eCrbxRLh0g+szm2FCOdscq0\\\\\\\/354D7MywaGLLdNTBki4+DjeOdAe5o0jdvASCW0F5bcNRLw3pbLXwvrZZlwADpWP9aqDqsNB+wOM0q5GZJHE1Q+MEDl2fV7Iqt4NMm6ZYsQCJXts5myZK9ZY+nvtkcMFyiK7NEe92nX+nL8+56uqQkVxVNm4pqFpk+pjft1XArs40sE51s5k06rFCgq9L5dIfw6yEXqkcN+rS5TE0Bs9p6o\\\\\\\/qdVJUoURh0baNj5LiPlQSZO3nL5poqNzP3ftTruC+GtaRRQpfT4zgkLASbj\\\\\\\/1vKKDmMpavjTH8dq8ZuAISGYlHKKrqCcU7pTUvzE8tZIpl3i+tIY9\\\\\\\/97T\\\\\\\/9gvrRv037SWme4iOvUjHDPXqFFpX\\\\\\\/IVme+fNckVLmi++DVQz6SBajlu4gVzlncFt9vnQhfRXeBWawxiV8RkGyIx\\\\\\\/j+nagot5cv42rQRY7VIlheXDXAKdJJVieSvUMvcMGKhfDSGNEzHGc2I0zuYeI8D1PcpMNas2ZE7XYyG1QN3o27CDyNuQfLy8EOSGk2lUkZAiNFm8VcBjaOMMnFqemsinF9TpAl+r9tgMeKfPELl8KpMUwDTG2\\\\\\\/KASE7hOMuRt88Fd17tAHHExyW+BspS\\\\\\\/jFabxH5XfehS+aTSeD8LKfYYm2nvOhTBCAeFF86qwMl8IPyWHFhEKPjSwpbu6c3N8VSLDH429mWAHKZ9MLFfH7drY4dYflOvfbnmYsIqviddPCpQ73tG\\\\\\\/7MLbjFuWISHOYjzYI5v0ETyUDz0\\\\\\\/MCnVf7xYeEHtgPSzsYSG0ZmDbxRVBpRFefq+A3BaFb8x0J8rwITxCkoLgaJ1v2LV3R8ykhlyxj3V8\\\\\\\/CxsDc5n35Vw5xnahKLwPtQwtqz627Oc8jAwfWy556KV9Xg4MczeVMFQbHMfYYO1TFeb9nwTjqxBJSeRHt22gbU77vB2tKfElAxskBc8p9+Qoi9K0762emSo+CqSxYVpMrMui1cUQgUrU7GGsAwHzUefqD5UEXFx1LkReEWWUgPmoiQnvWRBQJEuX4MFpyxWVQrbeLE2dJeZ3bY33Rt8V8WYBmOcOTTUf3co\\\\\\\/XzL3cddM6hRpLVm5+gi2v9iAMqKcBCYuvOmu+BC5u9QpL7g7iQOUe7oygtPP1\\\\\\\/okCmyhLhC73YLD3ITEsXGO2zBUgqWz+FYGoQPNKLodNn0UKDRW7m6GOcQ2MDVhjW3Q\\\\\\\/I+oXL6m8iZQMBSt9UNDwzOT+Kx3Fk5Ywm9uDXbUPSh0ZPjM7bcvh0PNIcy4AZyoC43bQoFa5mR8dqOOZBD6bl0YQSkOVi00hA8cVSVzAZCEhY30JFKaqygP8S+aE6k3\\\\\\\/DFGN7ky8hH2h0GIDrD3YV2ZyNZX65m7B1jBx0CwPVK\\\\\\\/O9OU14C61yRtJrkWT87GVRtDhjf4zkgLrXRghSKbn4EsY0ajoFvOuhBK0vDnYfKtfHiu0LEaJxRd3R+GapRJV+vHa01eNUsZnzDZXl16Juo8F7OaNIsQb5v56SdXUgDkgvZDtWY6Q\\\\\\\/DtM8LTGbXG78PLhIlxtNZDpDllq4wC\\\\\\\/tqLqjmPlwe31RldRzTDC3NZtzw7SAQ+ZTNSmVG8U7DjvhL+n58daYsCPhr0mgFc2+kDMOkA4YUIlT1ymXL0MLPybXjLTPsgIqe\\\\\\\/uP8J94XIVfSf36ixrgMU3bKcz88gLAjGhHNB4mQD2i9U5nhU\\\\\\\/veQE3fnZQ5gJJIlkvpOeXvMFjdRUorGM+Cfd4VE2imHEmutQzPbOHxTZyQS4NzAYNkPjV\\\\\\\/t3fhsYgPhUnzll8TMHE5+ewFfzg4hm0aW7w5hS+hblKBoHsWk8\\\\\\\/JRs7t0uTwvk2kebNQf8iqWoaPKhQYaUUJzE7nFhbpO5IpkmrK1cJOXAlhOUJl9jQoLPiRtbLllgJzB50rw1xpa6OX1Rd8KzSQfGu223iAHZKTkyajU7XVGcgB7IN9ur3SKwzaXXsgHsUNiw0cwiYV9gsjWiZCpaXoxfRp6lDM4\\\\\\\/UUbc8IPeQiHm4+IHEcDBf21x3mLbXY5cVSlRpMY+sUn175sNxbDWSMU+uE+HxUBdObncCNWM2QkZkP72nmtvdSfkc+v3FR1bckvJTpCGOlAcVlEjCu6T8SHNK77iengD1suYXD+QzcZjy9yofW5LNMcL9ixvfnhJ7xoCyehekfXTX\\\\\\\/GF7h0qI\\\\\\\/8l8sjQO8ktAXA06mCooSL5SzC\\\\\\\/LkZ2hCm+1GRiHetGCByUtAxjJ9MiRyVujd8GkmCDxya+cu+pzu5sF32Cq0x+bBHYfn+u5Wv0c5sgi+QtdUDcZHyrqk3djg4muCJyQet\\\\\\\/ZRD\\\\\\\/zUknzlAvcpEwUrHc5ypC177HUQOMYOlccIpMxVMa1de+mLPuGGiDcZGQ6GdI3Uq9y3o1MRF3d3WtYpbV9mRh+h4RSVT04\\\\\\\/JHEOjjrnJ+q8\\\\\\\/Px5TLqIcN90CY1yuxDog8qDxL2u0UUSbxLe+GFyymtjHgMTLipJDYqJUnkuKvfndgj1zwZMKdREZJX+GljOm6b3o34RKOkCaXUh\\\\\\\/j+XxPdmbooVFdH\\\\\\\/kPKKaXDjd54BVPw4gBIn8ZHj7I29qJu\\\\\\\/x3GMKJmmKWap5MPjmNDykyKIZZ+JBo3vM\\\\\\\/mrCfhXGl6N5ClcVL81H0dUixSfejsZDgo4HLjz9\\\\\\\/kE6vZ0A8TlOmmH1d80UCGHJebDgyYgNRctWtbHi08X+\\\\\\\/xEZcSIeqCNqNRPihkD79N8AaWmKdOpz\\\\\\\/8SAOGA5qN1iqKNh5TPEZD2YCQcGscpZTBAAKA8YPF8XLAAijpgBGhyiX39mQueDi+cu0I5zXblgPr9kUfBBV5mUm6IWNihIo3tskF9qfuEbHuucVZJALkthEDg1nJx5t\\\\\\\/ora2f5Jkn9vFaZZYq1Slxs4OHDGVbO6z4Bft4gl0yEKwvPCaYRjpuB7KC1HtGxXnGfJq7AXmCvrY9Dw0RaZ2oPsCH7igqjJoOAtjsI5PDggkA6NKBWe2ahvRAKBbr8epTgvvUMtxE9vi1jS1BvO3nFpqScNMJTzmVNoN+ui\\\\\\\/pzhIIex6jNmE887X7EYnuTFzpjoFA5tcPvjvLXweivhsMXT9Tzpqyg4zTY4Dc\\\\\\\/iN4IT5HpgRltLWT9RNjWK86KjlhXwOnuziZu2xwX7svTziIVC8NhnB7ebqKvbxG\\\\\\\/ZEtx3xArwEtVW5aGvgCIraBW8g996K\\\\\\\/ZeOZJOUf7CexyfT6TMy9GdhLwXC+Z1AiT+T+cVmwggv6ngbwE1ZEByDyK+ENR9Wz6jA6fmKckbExzGS28UqpeCPgoa1umXnSKCcieaLfqMSd2xZ6bu98MVZ8Yv7AotZ0e2FN3NgLx6lryuWjUoVBr7fQVpueuv+Is71XJLOGhVBqwIMABIl6wKza2R\\\\\\\/i+HzuvfNDYRqyACgNI\\\\\\\/YVagWNs1dZSxwPVcbN2qpS7bwrWLkIaic8FXGJRdsnROclz9YV89wBkIDRy2ioNfey2nroY+tsPn5rEMcsnyTiX1MdIdrI8yh2LluqxMMOuRfxscHq2GtXk+g4S5xNmDX2P+n9nu3x8lAjyzT7NCcKD+ig4ZNdxx8EfRzSXexG8RGlqD\\\\\\\/ykaFWAxk4uQDgV9TIOiJi26tBA6W+FNcc7H8ujtE3rDlZdIc0MhfLSCF8bAIZQxUSae8dS9NQmjRBn4ZtNoU7iBC3JmzOt3GNhW32pn4xzTUMVlsKEHxLkzNFist5x+xkTzBmdbf0hycpk59SunppYG3EMKfYzqcp6xPIPMMRKFuUGN4OsBNfme7BKUQF3A0fkEhPYiM2y+Y\\\\\\\/\\\\\\\/McyO7VMi7oP57O7NBb+ld1wnJlt1SnyBZr75pNDnMFEDhCqV9lTc4MZqPnGS++eH76fqKfbF5Vbz7hhGFnilKRGV80WgIKfrI2KHWZ3wlZJlX9gSJJeaHVWksEL4dsjkwlnxLh3iFQEjLbUQ5p52IBdewRchGprZhQGIoWhXRk9KqdrBC5f\\\\\\\/WUBFixmvlpVifp9DK2n3aXW034W0gCcQjfjRTJE\\\\\\\/73W6I5Yqtdp6mF9uOw6RoAwNX7B9FS06Vp3nGYdudHNuyHnnNq0OxEpuRAGa6lW4m7rN\\\\\\\/dX6neutmlUQDjHC9i95c0\\\\\\\/HKFAYwdp7iJEl2FlFHg3racgPA5lvHr7unDhWkMKObxBMXbjM79VYePl+uoP5bsEfen4HJdhJbOEK1K4F+VTGFRbXnw1l\\\\\\\/nwRNxZ3hWa6UJ\\\\\\\/dY5mq4682KNoT\\\\\\\/bvIuu3SUE+VnoYVx5P79AwUrpa0YWGalo6sVG6Y2rXgVC3PF4NSvJhGWvLxTcy0jeITyFufbyyqBF0NbuHA+VFMuTtZ5uxRS1tqmO7WrPdJyalBHIVmUtZZFNxKzSL8AY4lD2yPUVy14dTw0sFnHnLVInCkr0undunV65UDUso4hyuCPbgRqdZymAoI2j4yi0mXzH4cFLw6HIJbPiNyFHgxYGKjGnPGr7jyt5GRvXHd5lqSkIH4JQl+u2uWHTmLcZRLlCc6D5EN1hbsqcMBmshLMZeBWdBjkMPKRCKKgivhByb44nwrjAO1VKcpVbqX3eat7kLnagv3znwIV7p8NQNFDHKmPnmdm8Iy2rCn2JBB08f997htVFajm5O2sKYv1xjro6BVzZ1Sxozwj+9bFaPBBrjnCGLxtI+GZdy94PaYXXiTGyoB8jApVbCbcxwnssro7mwjIDm5WF2h8aA5yD\\\\\\\/QF6G5cN0X7V9haN8XRaqKlO5LXq9QqVC+J5Tl\\\\\\\/wM8QM+GMzHyN1t2fdsUNjhTv9a4FPvctUV1KxqTa\\\\\\\/brMZ+h9351Qo1B2vAwFpbaoX+8a8+z2KGHQQJxR05pZdzjuYXu7paezjW8ADE4DEvY6VzfNTkJUjubPRzBpLGlYS2mzv8b0yJ384aDcec3ksPQSiK0i4Ejd4rXbF2dNTaelkhi9Zwyk78l5I66KYEJF7jEZrhNSulVZobjkW7ZWU1xMM+gLGz807DOgedBZCrs6oXJvgH36tHQ6lup1qTRb0WxVP0Ot+4QwZtO8Z5en7Wp1438ri1Z8Cnw\\\\\\\/OyZJtio0h3vuj2AGiRq8Gjw7VqGa+Wxq+XTHlKc8Z1C0DhVe28CnUJuKpCd6oYMU3APYGwpU9Pv8W\\\\\\\/ljPdsbORgO45aZIfJzJnsqoq\\\\\\\/AVq19kWTSWlk3BPVpqBVR1mDZ1X7gfF40YSW0C3\\\\\\\/GYjG35q8bas6aB2XaFqfdcX6YrVq+ek7aODQFmFKEoKQe3ehCFo2y9csUXCe6CXsY\\\\\\\/qruBL2xLQIs0ST8W1hr5JJhVD2teoqRBkeBXJYW8qgarRLk3\\\\\\\/8rabtNmGo3uPuMP70HuRHxNy0ZPQDyR61aSfZlMNzRz666mFvILcoXV3oujjCvPLaalnrtoxuRVDuVwdnmIkJ0BzWm5XG4nroioc0DgO8nbaensoYsc0l+\\\\\\\/8muG\\\\\\\/nnb0gX0Mz27mEg8Ot6TJOMk5uIJQ2NgENxRtqrD65hTiGrL8BKZb8eiJwyBoD+81\\\\\\\/9xv+F\\\\\\\/LtFwjwvGdpQLuli1D4uJ7jdyDXTyqk6jrr2Lo\\\\\\\/wkciGVjGcrQrrEH+nIUWnBbnrzqe5Eqgi0a7ilCUQeVoOZZTdVKimT0pDIOk3Czw0tSCeKx3V74LXuvDHmCNNPf8OycQc5+84cNTFcua\\\\\\\/2i0St4cayqU7+sxm58hI2774eP+ZD\\\\\\\/T6fgUnFZBZUoGQHnWEuhlXkVuyKORRkgF0qtgkl\\\\\\\/XLT0hvGOp4DnQhVQhCz60XoKP2LtLEi1Gxq5dVLtz3dNYMYhKgHRWykjFZ4ojTL0FxVg8X423gJ4qzId3iPfQr1ZJoOC\\\\\\\/+mkEes\\\\\\\/Xh1bgQTlnpMnVc4j5AUlStp9spV5pR2LbakxPOi\\\\\\\/rfb3rTyZwX+sYzI1wHP1voqOBNi\\\\\\\/aBFF1bc6Kpp2dCsV81yQFOWUpXsusRgFh4EK4qRVn6wflccwrCb+2dYz5mEisUWCizmLD\\\\\\\/JPWbDYdrRgkqQ5xr8mRxni8q6bPQWS\\\\\\\/h4z2hebShb\\\\\\\/dfH27fAvR39SNWrme\\\\\\\/BZQ7aMKXcRoaLdsrkNAAiniZGRul\\\\\\\/6F\\\\\\\/hkkmo2B0gE\\\\\\\/uQYqNp+a1+rx0Dv84nSJ6PlQTOeJw6frEb02rYPP5yMCsL52zi7W9H2oygRK+3YwJq19ycErQShxnryEtnoxOvF7zwnELHKHv52h2ppv5Vn4HR2bNAs0x7iVym2cV8AAxsjPyWr6M2dCprZ3sxPyEow7Ryz3xjxLR4SbcoSdhIA7mGhO\\\\\\\/YQjRHXUbBtLCP2hyZGLwRkUk+Vs+e3VWH\\\u0022,\\\u0022iv\\\u0022:\\\u0022DFhR3dVwlumb1ubJOsSq\\\\\\\/w==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022c63f9497b7bac106a4b222b0c8e589b08fe19c524669dc4141f80efb81928506\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022ex0ujdvXhBfRKhI68uO2+ancKPZakc0XwE75a2cllYxkrE5\\\\\\\/extQG3fLWgc5vZTEh1tQKtz5PMAxJ4swtpqaLgCc+Q95j0plnDQdoOeNsyVTgfVcyL6N\\\\\\\/c0k6LC7XBa0V2R8uuiAEb3rQJ3gDkSIVOYnNZf8Cuo5KJK0f8BV0ZEHy3qaBYbY3e0P07\\\\\\\/czDF4OxMCLTa9LxrNIGR7ddK9nowch5xzrvTBjvG4pZAVSNeLpb0S3OTNx\\\\\\\/XL4h3a9xO4ukv\\\\\\\/DxOAI3NSIi4dOujQlgwglcCO36U4JgZr1HBnOmjdT8U+nZ4Uk\\\\\\\/JUEyk7sHjmzvbccyibXX01Nu2R2R\\\\\\\/fjqrHJm4jfoYR7gZGH0UZRMB1UmATdu\\\\\\\/WClC5HUN4hpOBG6BkkAsDHYhx0JLNxRmWsZ9B6uzP+RheA35pvrkz0kW05C09qXkaQDQa+pIT+iNnx2QT\\\\\\\/9ytz0t4l3uOTxaYgJOOJKabvyrx9LwsGDfCHjXQnyZo+tNdm+AKweHdetKUUnOod0SmlbeQOshvxMq4yPfHGK8kEtCUWOBBdYvJMVs6qAl5k3DDlduGf+MfdpiwxeOWHkjhG0Vh54dvKlQHciEAq5kKpV+gUojz31c+LEjLJwAuE\\\\\\\/FRdKk6JVZqPqGtHiGOv1SICsa9vzzJ781QDAYmVJe97SE2LCFSR3eY8fcYxvMEq903n4Qz6p4a5st0o77TcP94S9CHC2Eny6s4jFRrBDI+hWuyw4P2Hts+LHjCPIamf7uqnpwFSy7FFIpGIYPLNVJACMEDqlSb2aW8mdKCTXV9GwC2l\\\\\\\/NYgxH+yL+LE4dt30ioQzLWEDnRM8sDWGCXiDThE6b1kmVXju4UCpb1ktUsKDoWOMJeQsiiqSoP1N\\\\\\\/zM8v7xUU4N0g3a2Mpt69hotlXJH0P+zB794F9g7vmgePN+9wYVNknyghOCzEOf9K3VUgZcNvZELLC5t6Li8Q4MFJtCpdzS1odmSq8kF87yLXXNM916gVAosDTi3qDe9LKkBogRVocflUI1eQEoe1+cfVR\\\\\\\/iEX5BBA6WhKxAtMeiZCoQBfF2beQ1ZuVRHQ\\\\\\\/bdnrXxKkodeXImKfjNWlPMsWxGGOPTuOqRJtvZJr\\\\\\\/8pyolimGUfZRHup82RcpxpadaWARVnojQrn6YvS6zS\\\\\\\/Yf\\\\\\\/P2nV\\\\\\\/h2GWXUl+WxhiIj3syoF4WCGRCV3BpdMqYY\\\\\\\/6\\\\\\\/+FUTi7wDiRJFKlsFfYuZV61YlSkKmlZDcqixPj7WV3EGru3s5eoRe2rLqi7FzzCEEZZtYtHmo+G1P0zxtzwl9UbO1aKq+aXeSNWcsuXqfbVTueggUxNXhQebjIyUkCKIwkP4aMUVxjxC1DS62TCfna+36YA4Fng1k+n8ADx7pfqEvA+PkTGxS93hHonM0RLYmKHg5tf77\\\\\\\/EohyJ4CIS0Yaw3VojH2yFDOs26EsU\\\\\\\/QcdWFlPmymZqkCSLD7U7N7l8O6MrG6fTvVS0OF0sX3SsWUXd9Mu99ME8agzA7yD2XkNBk1VZluxnCKrm4ABytkM9uJJmF31blz372b3WF3mIMUr6lYxR8gB4uI1qSnZak+RUvrAct7XTsFHEgrI4eeVSuXIidDZjNgb1u\\\\\\\/vgkLri9DMRIFMi9p9k+w8KZwHERf7LBSbznDhT75f5hG8CupQwIAnHQMB0N3AQUm18K4HyQU9qgyQS09PvOKl6cOKEPIBq0qDMFrhu8z50FjkMVgHGxcnIL+JpV0RGZ3dFQfy+O8iw24lwMnpFl0Vf5iIK9D0VB7z3t57myqWd\\\\\\\/K0XA35+Pk3Pi8qdhXaia06vW2Dsv7O86U9oXtsQe3D21jkst2zA0ZNjZpaWuSSX24hZXog\\\\\\\/OB58nBnSVT8HJfTkqnBSGp0sYV5+pCnpbVTLkgoGFFT64xHmzuNOukeIXf5EdVYcwGGgrOPoJ4O0tk6qZqHT+hyjn1IJ2VXH2hqTjYn6wTL5gRnYKkO0kqn62FzFvSbuX+A+53N7RPqtD0kolzZ2ScfkvivJrbURIVE5FLVy1QEm8LsRNc1NYADl4+EKMZfSYcNnS+bQ0Y+rVZNJau\\\u0022,\\\u0022iv\\\u0022:\\\u0022S\\\\\\\/Vk2XgsKsXIoPIs7KcCEw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00220bae76b08a53f08a7be0d21a128b9c1dd740a17cf62dab2bebb702bda90e5835\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-494613 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-genomic-prediction">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/11/3855" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-VUJQUzOYfIm9gUtZHQbina1pEYDDUoRvzHsj78WLb1Y" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/11/3855" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-LbFbaSaemWefYL_ePwhaJIpzd8lTYhis24CJRiGwUkA" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/11/3843" title="Genome-Wide Analysis of Cell Cycle-Regulating Genes in the Symbiotic Dinoflagellate Breviolum minutum" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/11/3867" title="Mapping of the Sex Determining Region on Linkage Group 12 of Guppy (Poecilia reticulata)" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494613" id="node494613" data-pisa="ggg;9/11/3855" data-pisa-master="ggg;g3.119.400728" data-apath="/ggg/9/11/3855.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Efficient Estimation of Marker Effects in Plant Breeding</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-5034-9954" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Alencar Xavier</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3855-3866;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400728 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494613" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Alencar Xavier </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Corteva Agrisciences, 8305 NW 62nd Ave. Johnston IA, and</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Purdue University, 915 W State St. West Lafayette IN</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alencar%2BXavier%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Xavier%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAlencar%2BXavier%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-5034-9954" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Alencar Xavier</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:alencar.xavier@corteva.com" class="" data-icon-position="" data-hide-link-title="0">alencar.xavier@corteva.com</a>
<a href="mailto:xaviera@purdue.edu" class="" data-icon-position="" data-hide-link-title="0">xaviera@purdue.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/9/11/3855" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:494613" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:494613/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/11/3855.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:494613" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:494613/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/11/3855.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:494613" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:494613/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/9/11/3855.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:494613" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:494613/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">The evaluation of prediction machines is an important step for a successful implementation of genomic-enabled selection in plant breeding. Computation time and predictive ability constitute key metrics to determine the methodology utilized for the consolidation of genomic prediction pipeline. This study introduces two methods designed to couple high prediction accuracy with efficient computational performance: 1) a non-MCMC method to estimate marker effects with a Laplace prior; and 2) an iterative framework that allows solving whole-genome regression within mixed models with replicated observations in a single-stage. The investigation provides insights on predictive ability and marker effect estimates. Various genomic prediction techniques are compared based on cross-validation, assessing predictions across and within family. Properties of quantitative trait loci detection and single-stage method were evaluated on simulated plot-level data from unbalanced data structures. Estimation of marker effects by the new model is compared to a genome-wide association analysis and whole-genome regression methods. The single-stage approach is compared to a GBLUP fitted via restricted maximum likelihood, and a two-stages approaches where genetic values fit a whole-genome regression. The proposed framework provided high computational efficiency, robust prediction across datasets, and accurate estimation of marker effects.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/mixed-model" class="hw-term hw-article-keyword hw-article-keyword-mixed-model" rel="nofollow">Mixed model</a></li><li class="kwd"><a href="/keyword/laplace-prior" class="hw-term hw-article-keyword hw-article-keyword-laplace-prior" rel="nofollow">Laplace prior</a></li><li class="kwd"><a href="/keyword/single-stage" class="hw-term hw-article-keyword hw-article-keyword-single-stage" rel="nofollow">Single-stage</a></li><li class="kwd"><a href="/keyword/gauss-seidel" class="hw-term hw-article-keyword hw-article-keyword-gauss-seidel" rel="nofollow">Gauss-Seidel</a></li><li class="kwd"><a href="/keyword/predictability" class="hw-term hw-article-keyword hw-article-keyword-predictability" rel="nofollow">Predictability</a></li><li class="kwd"><a href="/keyword/elapsed-time" class="hw-term hw-article-keyword hw-article-keyword-elapsed-time" rel="nofollow">Elapsed time</a></li><li class="kwd"><a href="/keyword/genomic-prediction" class="hw-term hw-article-keyword hw-article-keyword-genomic-prediction" rel="nofollow">Genomic Prediction</a></li><li class="kwd"><a href="/keyword/genpred" class="hw-term hw-article-keyword hw-article-keyword-genpred" rel="nofollow">GenPred</a></li><li class="kwd"><a href="/keyword/shared-data-resources" class="hw-term hw-article-keyword hw-article-keyword-shared-data-resources" rel="nofollow">Shared Data Resources</a></li></ul><p id="p-3">Genome-wide markers are utilized in plant and animal breeding to capture quantitative trait loci (QTL) and relationship among individuals for prediction and selection (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Meuwissen <em>et al.</em> 2001</a>, <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Habier <em>et al.</em> 2007</a>, <a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">VanRaden 2008</a>). Most individuals in the plant breeding pipeline are genotyped, whereas in animal breeding genomic information enhances the pedigree-based relationship (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Henryon <em>et al.</em> 2014</a>). With the ever increasing volume of genotypic and phenotypic data, various statistical methods have been developed to handle large datasets, enabling better use of genomic information for more accurate selection and better allocation of resources (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Heslot <em>et al.</em> 2012</a>).</p><p id="p-4">Evaluating the predictive performance of these various methodologies has become an important step for a successful implementation of genomic-enabled selection (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">de los Campos <em>et al.</em> 2013</a>, <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Heslot <em>et al.</em> 2015</a>), since the prediction method utilized to generate breeding values may have major impact on the short-term genetic gain, as well as long-term changes on the germplasm (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Daetwyler <em>et al.</em> 2015</a>, <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Hickey <em>et al.</em> 2017</a>).</p><p id="p-5">Genomic predictions models are used to estimate breeding values of observed individuals and to predict breeding values of unobserved individuals in early-generations. Accuracy is the most important criterion to define which technique will be used to generate the breeding values. Besides accuracy, the computational efficiency has also become a key component of prediction pipelines due to the growing number of genotyped individuals, observations per individuals, traits, and genotyping density (Georges <em>et al.</em> 2018). Hence the method of choice must have two desirable features: computational feasibility and accurate prediction across various scenarios (<a id="xref-ref-74-2" class="xref-bibr" href="#ref-74">VanRaden 2008</a>, <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Misztal and Legarra 2017</a>).</p><p id="p-6">In plant breeding, the calibration of such models are typically done in two steps: 1) Estimate the genetic values from phenotypes of replicated trials; 2) Calibrate marker effects upon the genetic values to estimate the breeding values and enable prediction. This approach is referred to as a ”two-stage” approach. However, single-stage analysis can benefit genomic evaluation by jointly modeling genotypes and replicated phenotypes (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Liu <em>et al.</em> 2014</a>).</p><p id="p-7">Few studies have attempted to estimate marker effects directly from replicated trials. <a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">Taskinen <em>et al.</em> (2017)</a> proposed using pedigree information of ungenotyped individuals for imputation and subsequent estimation of marker effects. <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Da <em>et al.</em> (2014)</a> provided two frameworks to fit genomic models to estimate variance components and marker effects, one approach suitable for large number of observations and another for large number of markers, but not for both. However, such methods often translate into poor computational performance or convergence issues (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Misztal 2016</a>).</p><p id="p-8"><a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Fernando <em>et al.</em> (2014</a>, <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">2016</a>) provided a framework where marker effects can be estimated from whole-genome regression (WGR) methods via Markov chain Monte Carlo (MCMC), enabling a broader range of prior assumptions for the distribution of marker effects that can provide predictive advantages in single-stage approaches (<a id="xref-ref-89-1" class="xref-bibr" href="#ref-89">Zhou <em>et al.</em> 2018</a>).</p><p id="p-9">Flexible models that enable the estimation of marker effects among other parameters are commonly based on MCMC method (<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Fernando <em>et al.</em> 2014</a>), but these techniques can be computationally prohibitive at times (<a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">Wang <em>et al.</em> 2015</a>) and must be replaced by Gauss–Seidel iterations (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Garrick <em>et al.</em> 2014</a>).</p><p id="p-10">This study proposes an efficient non-MCMC solver for WGR and mixed models based on conditioning and iterative updates. The idea is to develop a single-stage solver by jointly iterating the steps of the multi-stage analysis. Predictive ability and computing time of the proposed framework are evaluated through simulations and cross-validation on real data, comparing it to other methods.</p><div class="section" id="sec-1"><div id="sec-2" class="subsection"><h3>Statistical models</h3><p id="p-11">Iterative conditional modeling enables solving complex models without the computationally demanding operations (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Graser <em>et al.</em> 1987</a>, <a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">Thompson and Shaw 1992</a>, <a id="xref-ref-55-2" class="xref-bibr" href="#ref-55">Misztal and Legarra 2017</a>). In these methods, conditional expectations are used to efficiently estimate variance components, fixed effects, breeding values, and marker effects (<a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Da <em>et al.</em> 2014</a>, <a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">Liu <em>et al.</em> 2014</a>, <a id="xref-ref-16-3" class="xref-bibr" href="#ref-16">Fernando <em>et al.</em> 2014</a>, <a id="xref-ref-72-2" class="xref-bibr" href="#ref-72">Taskinen <em>et al.</em> 2017</a>). Two statistical approaches are introduced in this section. First, an iterative algorithm for WGR that speeds up the marker calibration. Second, a framework to enables solving WGR into a model with replicated observations using a specific type of conditioning.</p></div><div id="sec-3" class="subsection"><h3>Whole-genome model</h3><p id="p-12">This section describes the implementation of the fast Laplace model (FLM), an iterative method to fit a WGR using a Laplace prior. Laplace priors are popular in genetic analysis for QTL detection and genomic prediction (<a id="xref-ref-82-1" class="xref-bibr" href="#ref-82">Xu 2007</a>, <a id="xref-ref-83-1" class="xref-bibr" href="#ref-83">Xu 2010</a>, <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Cai <em>et al.</em> 2011</a>, <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Legarra <em>et al.</em> 2011</a>).</p><p id="p-13">The implementation below is based on iterative conditional expectation (ICE) estimates of regression coefficients alongside their associated parameters, updating one parameter at a time (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Meuwissen <em>et al.</em> 2009</a>). This type of algorithm is commonly referred to as coordinate descent (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Friedman <em>et al.</em> 2010</a>).</p><p id="p-14">Consider the following univariate linear model fitting phenotypes as a function of an intercept and genotypic information:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-1.gif"/></noscript></span><span class="disp-formula-label">(1)</span></span>where <em>y</em> corresponds to a vector of phenotypes, <em>μ</em> is the intercept, <em>M</em> is a matrix of parameters where each <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-2.gif"/></noscript></span></span> cell corresponds to <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-3.gif"/></noscript></span></span> locus of the <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-4.gif"/></noscript></span></span> individual coding <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-5.gif"/></noscript></span></span> as <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-6.gif"/></noscript></span></span>, <em>β</em> refers to the vector of marker effects, <em>ϵ</em> represent the vector of residuals.</p><p id="p-15">The first operation in each iteration is the intercept update as:<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-7.gif"/></noscript></span><span class="disp-formula-label">(2)</span></span>Marker effects and regularization parameters are updated one at a time until convergence. Conditioning the response to all but the <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-8.gif"/></noscript></span></span> marker (<span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-9.gif"/></noscript></span></span>) provides a simple probabilistic structure:<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-10.gif"/></noscript></span><span class="disp-formula-label">(3)</span></span><span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-11.gif"/></noscript></span><span class="disp-formula-label">(4)</span></span>where <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-12.gif"/></noscript></span></span> is a vector containing the information of the <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-13.gif"/></noscript></span></span> marker, <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-14.gif"/></noscript></span></span> is the parameter that regularizes <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-15.gif"/></noscript></span></span>, as the marker effect associated with the <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-16.gif"/></noscript></span></span> marker is estimated as:<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-17.gif"/></noscript></span><span class="disp-formula-label">(5)</span></span>Each marker has an independent regularization. The regularization parameter <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-18.gif"/></noscript></span></span>, which shapes the marker effects collectively into a Laplace distribution, is derived from an inverse-Gaussian density with expectation (<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Park and Casella 2008</a>):<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-19.gif"/></noscript></span><span class="disp-formula-label">(6)</span></span>The scale parameter <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-20.gif"/></noscript></span></span> was adapted from <a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Legarra <em>et al.</em> (2011)</a>, as the sum of marker variances:<span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-21.gif"/></noscript></span><span class="disp-formula-label">(7)</span></span>Residual variance and full-conditional marker variance are estimated by maximum likelihood (<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Patterson and Thompson 1971</a>, <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Harville 1977</a>, <a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Searle <em>et al.</em> 1992</a>):<span class="disp-formula" id="disp-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-22.gif"/></noscript></span><span class="disp-formula-label">(8)</span></span><span class="disp-formula" id="disp-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-23.gif"/></noscript></span><span class="disp-formula-label">(9)</span></span>where <em>n</em> corresponds to the total number of observations, <em>q</em> is the number of parameters (<span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-24.gif"/></noscript></span></span>), <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-25.gif"/></noscript></span></span> represents the inverse of the full-conditional left-hand side, <em>P</em> is the projection matrix of the whole model (<span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-26.gif"/></noscript></span></span>), and <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-27.gif"/></noscript></span></span> represents the rank of the design matrix of fixed effects (<span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-28.gif"/></noscript></span></span>).</p><p id="p-16">The optimization path consists of iteratively updating <em>μ</em>, <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-29.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-30.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-31.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-32.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-33.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-34.gif"/></noscript></span></span>, … and <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-35.gif"/></noscript></span></span>. The pseudo-code for the implementation is provided below (Algorithm 1) and an implementation for R is provided in the appendix. In this study, the convergence criteria was set as <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-36.gif"/></noscript></span></span> for marker effects or a maximum of 300 iterations.</p></div><div id="sec-4" class="subsection"><h3>Iterative single-stage method</h3><p id="p-17">The previous section presented how the algorithm for FLM works in the case where each individual has a single phenotypic value. Now consider the scenario of replicated trials, where genotyped individuals are replicated across multiple environments. This approach is here referred to as fast Laplace model in single-stage (FLM-SS). The term ”single-stage” has been used to define the joint modeling of replicated observations at the phenotypic level (<a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Piepho <em>et al.</em> 2012</a>) or with genomic information (<a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Schulz-Streeck <em>et al.</em> 2013</a>), which is not to be confused with the ”single-step” that elsewhere defines models that combine pedigree and genomic information (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Misztal <em>et al.</em> 2009</a>).</p><p id="p-18"><strong>Algorithm 1</strong> Fast Laplace model</p><p id="p-19"><strong>Setup initial values:</strong></p><ul class="list-simple " id="list-1"><li id="list-item-1"><p id="p-20">1. Compute <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-37.gif"/></noscript></span></span> for each marker</p></li><li id="list-item-2"><p id="p-21">2. Compute <span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-38.gif"/></noscript></span></span></p></li><li id="list-item-3"><p id="p-22">3. Set <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-39.gif"/></noscript></span></span> as initial value for all <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-40.gif"/></noscript></span></span></p></li></ul><p><strong>Repeat until convergence:</strong></p><ul class="list-simple " id="list-2"><li id="list-item-4"><p id="p-24">1. Update intercept<span class="disp-formula" id="disp-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-41.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-42.gif"/></noscript></span></span></p></li><li id="list-item-5"><p id="p-25">2. Loop for <span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-43.gif"/></noscript></span></span> marker in <span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-44.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-45.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-46.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-46.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-47.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-47.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-48.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-48.gif"/></noscript></span></span></p></li><li id="list-item-6"><p id="p-26">3. Update residual variance<span class="disp-formula" id="disp-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-49.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-49.gif"/></noscript></span></span></p></li></ul><p>Most iterative methods provide an alternative way of fitting mixed model without changing its statistical properties. Efficient algorithms to solve complex models with marker information use conditional expectations, going back and forth with the estimation of fixed and random effect coefficients, without altering the outcome (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Garrick 2007</a>, <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Legarra and Misztal 2008</a>, <a id="xref-ref-74-3" class="xref-bibr" href="#ref-74">VanRaden 2008</a>). The following model can illustrate the single-stage procedure:<span class="disp-formula" id="disp-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-50.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-50.gif"/></noscript></span><span class="disp-formula-label">(10)</span></span>where <em>y</em> is the vector of phenotypes, <em>X</em> and <em>b</em> represent the design matrix and fixed effect coefficients used to capture nuisance parameters, such as environmental sources of variation. The random terms <em>Z</em> and <em>a</em> correspond to the incidence matrix of individuals and additive genetic effects, hereby estimated from the WGR (<span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-51.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-51.gif"/></noscript></span></span>). Residuals (<span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-52.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-52.gif"/></noscript></span></span>) are assumed to be normally distributed as <span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-53.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-53.gif"/></noscript></span></span>, such that residuals can be autocorrelated (<span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-54.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-54.gif"/></noscript></span></span>) or not (<span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-55.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-55.gif"/></noscript></span></span>).</p><p id="p-27">Fixed effect coefficients are solved via least squares, conditioning the response variable to all terms but the fixed effect. This conditioning works by reshaping the linear model into:<span class="disp-formula" id="disp-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-56.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-56.gif"/></noscript></span><span class="disp-formula-label">(11)</span></span>Providing the following solution of coefficients:<span class="disp-formula" id="disp-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-57.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-57.gif"/></noscript></span><span class="disp-formula-label">(12)</span></span>To avoid building large and dense design matrix of marker effects (<span class="inline-formula" id="inline-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-58.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-58.gif"/></noscript></span></span>), the random effect coefficients are updated using a link function in two steps (<span class="inline-formula" id="inline-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-59.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-59.gif"/></noscript></span></span>). First, estimate the least-squared genetic values (<span class="inline-formula" id="inline-formula-41"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-60.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-60.gif"/></noscript></span></span>) as follows:<span class="disp-formula" id="disp-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-61.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-61.gif"/></noscript></span><span class="disp-formula-label">(13)</span></span>The coefficients are solved as:<span class="disp-formula" id="disp-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-62.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-62.gif"/></noscript></span><span class="disp-formula-label">(14)</span></span>Note the values of <span class="inline-formula" id="inline-formula-42"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-63.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-63.gif"/></noscript></span></span> are not the final random coefficients, but intermediary outcomes serving as input to WGR. The shrinkage of genetic values occurs when <span class="inline-formula" id="inline-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-64.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-64.gif"/></noscript></span></span> is updated into <em>a</em>.</p><p id="p-28">Subsequently, the WGR algorithm introduced in the previous section takes place, solving the following equation to estimate marker effects and breeding values:<span class="disp-formula" id="disp-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-65.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-65.gif"/></noscript></span><span class="disp-formula-label">(15)</span></span>In this case, the vector of residuals (<em><strong>ϵ</strong></em>) represents genetic signal not captured by the markers. Genotypes may have variable number of observations and weights can be assigned to the genotypes to account for the unbalancedness. The WGR step can be solved assuming unweighted observations for computational convenience, or weighted according to the number of observations of each genotype, thus <span class="inline-formula" id="inline-formula-44"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-66.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-66.gif"/></noscript></span></span>. The last step of the iteration regards updating the vector of breeding values as:<span class="disp-formula" id="disp-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-67.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-67.gif"/></noscript></span><span class="disp-formula-label">(16)</span></span>In summary, this single-stage algorithm works through the iterative update of <em>b</em>, <em>a</em>, and <em>β</em> until convergence (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). Using Gauss-Seidel (<a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Legarra and Misztal 2008</a>) to update coefficients, this system of equations mitigates the computational burden of building and inverting large matrices. It must be acknowledged that the proposed algorithm is a computational trick and it may not replicate the same exact outcome of modeling the <span class="inline-formula" id="inline-formula-45"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-68.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-68.gif"/></noscript></span></span> matrix.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/11/3855/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Approaches for modeling breeding values (a) and marker effects (&#x3B2;) from two-stages and different single-stage models." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1992299307" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Approaches for modeling breeding values (&lt;em&gt;a&lt;/em&gt;) and marker effects (&lt;em&gt;&#x3B2;&lt;/em&gt;) from two-stages and different single-stage models.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/11/3855/F1.medium.gif" width="440" height="353"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/9/11/3855/F1.medium.gif" width="440" height="353"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/11/3855/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/11/3855/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494850" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-29" class="first-child">Approaches for modeling breeding values (<em>a</em>) and marker effects (<em>β</em>) from two-stages and different single-stage models.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-5" class="subsection"><h3>Additional random effects</h3><p id="p-30">The study has focused on simple mixed models with fixed effects and a single random effect to model genetics. However, it often necessary to include random terms to capture genotype-environment interactions and spatial trends. This section describes how additional random effects can be included into the single-stage approach through conditioning.</p><p id="p-31">Consider a model with one additional random effect:<span class="disp-formula" id="disp-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-69.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-69.gif"/></noscript></span><span class="disp-formula-label">(17)</span></span>Conditioning the response variable to all effects but the additional random effect (<span class="inline-formula" id="inline-formula-46"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-70.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-70.gif"/></noscript></span></span>), yields:<span class="disp-formula" id="disp-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-71.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-71.gif"/></noscript></span><span class="disp-formula-label">(18)</span></span>Assuming <span class="inline-formula" id="inline-formula-47"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-72.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-72.gif"/></noscript></span></span>, the solution for the the random effect coefficients is given by:<span class="disp-formula" id="disp-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-73.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-73.gif"/></noscript></span><span class="disp-formula-label">(19)</span></span>where <span class="inline-formula" id="inline-formula-48"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-74.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-74.gif"/></noscript></span></span>. The solution for the residual variance is provided in equation (9) replacing <em>ϵ</em> by <em>e</em>.</p><p id="p-32">Conditional to other model terms, the variance component associated to this random effect is estimated as (<a id="xref-ref-61-2" class="xref-bibr" href="#ref-61">Patterson and Thompson 1971</a>, <a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Harville 1977</a>):<span class="disp-formula" id="disp-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-75.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-75.gif"/></noscript></span><span class="disp-formula-label">(20)</span></span>where <span class="inline-formula" id="inline-formula-49"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-76.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-76.gif"/></noscript></span></span> is the number of columns of <em>W</em>, and <span class="inline-formula" id="inline-formula-50"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-77.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-77.gif"/></noscript></span></span> is the full-conditional left hand-side equation. For random effects with non-orthogonal design matrices, such as adjacent matrices to model spatial auto-correlation, the variance component can be efficiently approximated as (<a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Schaeffer 1986</a>):<span class="disp-formula" id="disp-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-78.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-78.gif"/></noscript></span><span class="disp-formula-label">(21)</span></span>With more random terms in the model, all coefficients (<em>b</em>, <em>a</em>, and <em>β</em>) must be updated conditional to <em>g</em>.</p></div></div><div class="section materials-methods" id="sec-6"><h2 class="">Materials and Methods</h2><div id="sec-7" class="subsection"><h3>Dataset</h3><p id="p-33">The soybean dataset described below was utilized to assess genomic prediction methodologies. The cross-validation analyses were performed on the genetic merit the genotypes estimated beforehand. These data were not used for the evaluation of the single-step procedure.</p><p id="p-34">The soybean dataset contains 40 bi-parental families that share a common parent. Phenotypes and genotypes were obtained using the function ’BLUP’ from the R package SoyNAM. Each family contains approximately 140 individuals genotyped with 4320 markers, and the number of polymorphic markers ranged from 547 to 1262 within family. The soybean trait under evaluation was the best linear unbiased predictors (BLUP) of grain yield collected in as many as 18 environments under a modified augmented design (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Lin and Poushinsky 1983</a>). BLUPs were generated by modeling grain yield as a function of environment (random effect), genetic merit (random effect) and local block effects estimated from the checks (fixed effect). The population and experimental settings are described in details by <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Diers <em>et al.</em> (2018)</a> and <a id="xref-ref-79-1" class="xref-bibr" href="#ref-79">Xavier <em>et al.</em> (2018)</a>.</p><p id="p-35">The BLUPs are not optimal for conducting second stage analyses (<a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Smith <em>et al.</em> 2001</a>), but were used for demonstration here, as raw plot level data were not available. BLUPs may create a heteroscedastic scenario that may or may not affect the prediction accuracy if not accounted for (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">de los Campos <em>et al.</em> 2013</a>, <a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Ou <em>et al.</em> 2016</a>). This pitfall can be addressed by the deregression of coefficients (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Garrick <em>et al.</em> 2009</a>), where the genetic merit of individuals is unshrunken based on the individual’s reliability.</p></div><div id="sec-8" class="subsection"><h3>Cross-validation</h3><p id="p-36">The cross-validation focused on two criteria: 1) the predictive ability measured as the correlation between predicted and observed genetic values, computed within-family from fivefold cross-validations with individuals were sampled at random (80% calibration, 20% validation) in each round and this procedure was repeated 20 times, and leave-family-out cross-validation by using 39 families to predict the family left out, and repeating this procedure for all 40 families; and 2) the elapsed time for calibrating the model using the whole data.</p></div><div id="sec-9" class="subsection"><h3>Prediction methods</h3><p id="p-37">FLM was compared to a set of methods designed for high dimensional problems that are implemented and freely available in R. Including: Bayesian alphabet (A, B, C, RR, L) and reproducing kernel Hilbert spaces (RKHS) implemented in BGLR (<a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Pérez and de Los Campos 2014</a>); BayesC<em>π</em> and BayesD<em>π</em> implemented in the bWGR; GBLUP with REML variance components implemented in rrBLUP (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Endelman 2011</a>); boosting implemented in gbm (<a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Ridgeway 2007</a>); <span class="inline-formula" id="inline-formula-51"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-79.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-79.gif"/></noscript></span></span> machines - ridge regression, elastic-net and LASSO implemented in glmnet (<a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Friedman <em>et al.</em> 2010</a>); partial least square (PLS) implemented in pls (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Mevik and Wehrens 2007</a>); random forest implemented in ranger (<a id="xref-ref-78-1" class="xref-bibr" href="#ref-78">Wright and Ziegler 2015</a>); <em>ν</em> and <em>ϵ</em> support vector machines (SVM) implemented in kernlab (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Karatzoglou <em>et al.</em> 2004</a>); the empirical Bayesian LASSO (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Cai <em>et al.</em> 2011</a>) implemented in EBglmnet (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Huang and Liu 2016</a>); and the extended Bayesian LASSO (<a id="xref-ref-44-3" class="xref-bibr" href="#ref-44">Legarra <em>et al.</em> 2011</a>) implemented in VIGoR (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Onogi and Iwata 2016</a>). The latter two methods are efficient implementations with Laplace prior.</p><p id="p-38">Methods above were deployed with default settings. Tuning parameters for ridge, LASSO and elastic-net were computed through 10-fold cross validation in the training set. To mitigate the computational burden necessary to tune parameters, PLS used 5 components and the empirical Bayes Lasso hyperparameters a-b were set to 0.5. The Gaussian kernel employed for RKHS was computed as <span class="inline-formula" id="inline-formula-52"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-80.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-80.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-53"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-81.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-81.gif"/></noscript></span></span> is the squared Euclidean distance matrix computed from the marker information and <em>δ</em> is the average value of <span class="inline-formula" id="inline-formula-54"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-82.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-82.gif"/></noscript></span></span>. The GBLUP model utilized the genomic relationship matrix described by <a id="xref-ref-74-4" class="xref-bibr" href="#ref-74">VanRaden (2008)</a>.</p></div><div id="sec-10" class="subsection"><h3>Detection of QTL</h3><p id="p-39">An experimental population was generated through simulation to evaluate FLM estimates of large effect parameters. From an F2 bi-parental cross with 1000 individuals, 250 individuals were randomly selected and randomly mated to generate a new population of 1000 individuals. This bottle-necking with subsequent random mating was repeated 5 times. The resulting allele frequency ranged from 0.32 to 0.63. The simulated genome had 10 chromosomes of length 100 cM. The genotyping density was 0.5 marker/cM. A causative marker was assigned to the center of each chromosome with alternating values of positive and negative one.</p><p id="p-40">The response variable was evaluated under heritability of 0.25 and 0.50. The ability of FLM to detect major genes was compared to the Bayesian ridge regression and Bayesian LASSO implemented in the R package BGLR (<a id="xref-ref-62-2" class="xref-bibr" href="#ref-62">Pérez and de Los Campos 2014</a>), and a mixed model association based on P3D algorithm (<a id="xref-ref-88-1" class="xref-bibr" href="#ref-88">Zhang <em>et al.</em> 2010</a>) implemented in the R package NAM (<a id="xref-ref-81-1" class="xref-bibr" href="#ref-81">Xavier <em>et al.</em> 2015</a>). Three population sizes were evaluated to estimate the allele effects: 250, 500 and 1000 individuals.</p></div><div id="sec-11" class="subsection"><h3>Evaluation of single-stage method</h3><p id="p-41">Breeding data are inherently unbalanced. Genotypes are often unreplicated or not equally distributed across environments, and observations from different environments present a variable degree of noise. The single-stage approach was evaluated on simulated datasets that recreated such condition.</p><div id="sec-12" class="subsection"><h4>Simulated dataset:</h4><p id="p-42">The simulations were based on assigning the simulated individuals described in the previous section, a genetic pool with 1000 genotypes, to a random set of environments. Each simulated scenario was performed with a combination of number of observations across trials (n = 250, 500, 1000, 2500, 5000, 10000) and genetic architectures (10, 50 and 100 QTL). The number of environments for each simulation was sampled from a uniform distribution between 4 and 10 with effect <span class="inline-formula" id="inline-formula-55"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-83.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-83.gif"/></noscript></span></span>. To simulate heteroscedasticity, each environment had a different heritability sampled from a uniform distribution between 0.25 to 0.75. Individuals were sampled with replacement, such that each environment had an unequal number of entries. Each scenario (combination of size and genetic architecture) was repeated 20x with different seeds to sample the individuals, number of locations, and heritability of the locations.</p><p id="p-43">For the simulated scenarios with less than 1000 observations, most genotypes were unreplicated since the observed individuals were sampled from a pool of 1000 genotypes. Selection across unreplicated trials are not unusual when genomic prediction is deployed (<a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Sebastian <em>et al.</em> 2010</a>) since genotypes are connected through the relationship information captured by markers (<a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Habier <em>et al.</em> 2007</a>). Phenotypic values were generated by adding an environmental effect and random noise to the true breeding values. For simplicity, genotype-by-environment interactions, non-additive genetics, and spatial noise were not considered.</p></div><div id="sec-13" class="subsection"><h4>Prediction methods:</h4><p id="p-44">Three methods were evaluated. 1) Iterative single-stage with Laplace (FLM-SS) and Gaussian prior (ridge regression, RR-SS), implemented in RcppEigen (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Eddelbuettel <em>et al.</em> 2011</a>). 2) Two-stage approach described by <a id="xref-ref-67-2" class="xref-bibr" href="#ref-67">Schulz-Streeck <em>et al.</em> (2013)</a> based on fitting the best linear unbiased estimators (BLUE) of genetic values without using genomic information (first step), treating environment as a random effect, and subsequently fitting a WGR (second step) to estimate breeding values. The first-stage BLUEs were computed with the lme4 package (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Bates <em>et al.</em> 2015</a>), and markers were fitted with the Bayesian LASSO implemented the BGLR package (<a id="xref-ref-62-3" class="xref-bibr" href="#ref-62">Pérez and de Los Campos 2014</a>), carrying over the covariances from the first-stage to account for the environmental heteroscedasticity, assuming the second-stage residual covariances to be inherited from the first-stage. 3) GBLUP fitted with the commercial software ASReml (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Gilmour <em>et al.</em> 2008</a>) using a genomic additive relationship matrix (<a id="xref-ref-87-1" class="xref-bibr" href="#ref-87">Zeng <em>et al.</em> 2005</a>, <a id="xref-ref-84-1" class="xref-bibr" href="#ref-84">Xu 2013</a>). GBLUP is also a single-stage procedure to generate breeding values (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>), however marker effects are not explicitly computed for the prediction on new individuals.</p></div><div id="sec-14" class="subsection"><h4>Evaluation criteria:</h4><p id="p-45">The criteria for comparison was the computation time necessary to fit the model as the elapsed time, and the prediction accuracy as the correlation between estimated breeding values and true breeding values.</p></div><div id="sec-15" class="subsection"><h4>Statistical models:</h4><p id="p-46">The evaluated models aim to estimate breeding values (<span class="inline-formula" id="inline-formula-56"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-84.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-84.gif"/></noscript></span></span>) from phenotypes (<em>y</em>). GBLUP and Single-Stage fit environment (<span class="inline-formula" id="inline-formula-57"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-85.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-85.gif"/></noscript></span></span>) as fixed effect and genetics as random effect, as <span class="inline-formula" id="inline-formula-58"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-86.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-86.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-59"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-87.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-87.gif"/></noscript></span></span>, respectively. The two-stages fits environment (<span class="inline-formula" id="inline-formula-60"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-88.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-88.gif"/></noscript></span></span>) as random and genetic merit as fixed effect (<span class="inline-formula" id="inline-formula-61"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-89.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-89.gif"/></noscript></span></span>) in the first stage, followed by modeling the genetic merit (<em>u</em>) as function of intercept (<em>μ</em>) and marker effects (<span class="inline-formula" id="inline-formula-62"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-90.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-90.gif"/></noscript></span></span>), weighting observations with the coefficient variance, <span class="inline-formula" id="inline-formula-63"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-91.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-91.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-64"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-92.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-92.gif"/></noscript></span></span>, which translates into observations with weights <span class="inline-formula" id="inline-formula-65"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-93.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-93.gif"/></noscript></span></span>.</p><p id="p-47">The three models can be summarized as follows:</p><ol class="list-ord " id="list-3"><li id="list-item-7"><p id="p-48">Iterative single-stage (FLM-SS, RR-SS):<span class="disp-formula" id="disp-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-94.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-94.gif"/></noscript></span><span class="disp-formula-label">(22)</span></span></p></li><li id="list-item-8"><p id="p-49">Two-stage approach:<span class="disp-formula" id="disp-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-95.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-95.gif"/></noscript></span><span class="disp-formula-label">(23)</span></span></p></li><li id="list-item-9"><p id="p-50">GBLUP:<span class="disp-formula" id="disp-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-96.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-96.gif"/></noscript></span><span class="disp-formula-label">(24)</span></span></p></li></ol></div></div><div id="sec-16" class="subsection"><h3>Data availability</h3><p id="p-51">The soybean data are available in the R package SoyNAM. The R code for FLM is in the appendix. Cross-validation scripts are available on GitHub (<em><a href="http://github.com/alenxav/FLM">github.com/alenxav/FLM</a></em>). The FLM-SS code can be made available for research purposes.</p></div></div><div class="section results" id="sec-17"><h2 class="">Results</h2><div id="sec-18" class="subsection"><h3>Genomic prediction analysis</h3><p id="p-52">The summary of prediction statistics from cross-validation is presented in <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>. FLM was the most predictive methodology within-family and the second most predictive under leave-family-out cross-validation. Most methods provided satisfying predictive ability. Considering computation time (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>), PLS and the three non-MCMC implementations of the Laplace prior had the lowest computational cost. Kernel methods had high computational cost.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/11/3855/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Box-and-whiskers plot displaying the predictive ability across 40 bi-parental soybeans families. Prediction within-family (top), leave-family-out (center) and computation time (bottom)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1992299307" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Box-and-whiskers plot displaying the predictive ability across 40 bi-parental soybeans families. Prediction within-family (top), leave-family-out (center) and computation time (bottom).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/11/3855/F2.medium.gif" width="284" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/9/11/3855/F2.medium.gif" width="284" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/11/3855/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/11/3855/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494690" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-53" class="first-child">Box-and-whiskers plot displaying the predictive ability across 40 bi-parental soybeans families. Prediction within-family (top), leave-family-out (center) and computation time (bottom).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-19" class="subsection"><h3>Learning properties</h3><p id="p-54">The ability of different approaches to correctly estimate major effects through simulation is presented in <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>. Marker effects estimated from genome-wide association analysis were the closest to the true simulated values, however GWA resulted in an abundance of false signals across the genome as the marker effects are not estimated conditional to the neighbor markers.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/11/3855/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Simulation-based evaluation of marker effect estimation (y-axis) across the genome (x-axis) with varying the heritability and number of individuals, testing: Fast Laplace model (FLM), Bayesian ridge regression (BRR), Bayesian LASSO (BL), genome-wide associations (GWA) analysis, and the true value (True). Effects were plotted larger at the QTL positions and smaller in every other locus." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1992299307" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Simulation-based evaluation of marker effect estimation (y-axis) across the genome (x-axis) with varying the heritability and number of individuals, testing: Fast Laplace model (FLM), Bayesian ridge regression (BRR), Bayesian LASSO (BL), genome-wide associations (GWA) analysis, and the true value (True). Effects were plotted larger at the QTL positions and smaller in every other locus.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/11/3855/F3.medium.gif" width="440" height="297"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/9/11/3855/F3.medium.gif" width="440" height="297"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/11/3855/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/11/3855/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494731" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-55" class="first-child">Simulation-based evaluation of marker effect estimation (y-axis) across the genome (x-axis) with varying the heritability and number of individuals, testing: Fast Laplace model (FLM), Bayesian ridge regression (BRR), Bayesian LASSO (BL), genome-wide associations (GWA) analysis, and the true value (True). Effects were plotted larger at the QTL positions and smaller in every other locus.</p><div class="sb-div caption-clear"></div></div></div><p id="p-56">The allele effect estimated by FLM was closer to the true value than its MCMC counterpart, the Bayesian LASSO, and this difference was more evident in the low heritability scenario (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a> bdf). Bayesian ridge regression captured the large effects reasonably well in scenarios with where the heritability was 0.5, but the estimates were not close to the real values in any situation. In general, more realistic values were achieved by all methods as the population size and heritability increased.</p></div><div id="sec-20" class="subsection"><h3>Single-stage efficiency</h3><p id="p-57">The comparison of accuracy and speed among GBLUP, two-stages approach, and iterative single-stage (FLM-SS, RR-SS) is presented in <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>. As the number of observations increased, all methods converged to the simulated true values. GBLUP was slightly better than its iterative single-stage counterpart, RR-SS, indicating that the proposed algorithm provides a comparable, however not identical, predictive outcome. This small difference is likely due to how well methods handle information unbalancedness.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/11/3855/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Simulation-based comparison of methods: GBLUP from replicated trials (GBLUP, red); two-stages approach where whole-genome regression fitted on genetic values (Two-stage, green); and an iterative single-stage approach, FLM single-stage (FLM-SS, blue) and Ridge Regression single-stage (RR-SS, yellow). Accuracy under 10 QTL (a), 50 (b) and 100 (c) QTL and elapsed time (d) to fit the model." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1992299307" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Simulation-based comparison of methods: GBLUP from replicated trials (GBLUP, red); two-stages approach where whole-genome regression fitted on genetic values (Two-stage, green); and an iterative single-stage approach, FLM single-stage (FLM-SS, blue) and Ridge Regression single-stage (RR-SS, yellow). Accuracy under 10 QTL (a), 50 (b) and 100 (c) QTL and elapsed time (d) to fit the model.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/11/3855/F4.medium.gif" width="440" height="210"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/9/11/3855/F4.medium.gif" width="440" height="210"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/11/3855/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/11/3855/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494767" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-58" class="first-child">Simulation-based comparison of methods: GBLUP from replicated trials (GBLUP, red); two-stages approach where whole-genome regression fitted on genetic values (Two-stage, green); and an iterative single-stage approach, FLM single-stage (FLM-SS, blue) and Ridge Regression single-stage (RR-SS, yellow). Accuracy under 10 QTL (a), 50 (b) and 100 (c) QTL and elapsed time (d) to fit the model.</p><div class="sb-div caption-clear"></div></div></div><p id="p-59">The accuracy of Gaussian prior models (GBLUP and RR-SS) was sensitive to the number of observations when the trait was controlled by a small number of QTL. As the number of QTL increased, the predictive advantage of FLM-SS and two-stages over GBLUP and RR-SS decreased, and GBLUP outperformed the other methods under the scenario with the lowest number of observations.</p><p id="p-60">The computation time of GBLUP was considerably larger than iterative single-stage, and two-stage approach yielded intermediate computational performance. The discrepancy in computation time between iterative single-stage and two-stages can be attributed to the MCMC sampling in the second step and the estimation of variance components in the first step.</p></div></div><div class="section discussion" id="sec-21"><h2 class="">Discussion</h2><p id="p-61">The discussion section frames FLM as a potential method of choice for genomic prediction in plant breeding. The proposed methodology provided accurate prediction across datasets, as well as computational efficiency. Besides the predictive and computational performance, the FLM is an easy-to-implement regression method (Algorithm 1) without the need for tuning or matrix inversion.</p><div id="sec-22" class="subsection"><h3>Predictive ability</h3><p id="p-62">Most methods provide comparable predictive performance (<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Pérez-Rodríguez <em>et al.</em> 2012</a>, <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Howard <em>et al.</em> 2014</a>, <a id="xref-ref-80-1" class="xref-bibr" href="#ref-80">Xavier <em>et al.</em> 2016</a>). This study compared prediction methods within-family and family-out predictions, with predictive ability around 0.3 and 0.5, respectively, consistent with literature (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Legarra <em>et al.</em> 2008</a>, <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Lian <em>et al.</em> 2014</a>, <a id="xref-ref-80-2" class="xref-bibr" href="#ref-80">Xavier <em>et al.</em> 2016</a>). Within-family predictions rely on modeling the Mendelian segregation between markers and QTL, whereas across-family predictions are based on capturing the relationship among families (<a id="xref-ref-27-3" class="xref-bibr" href="#ref-27">Habier <em>et al.</em> 2007</a>, <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Daetwyler <em>et al.</em> 2013</a>, <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Lehermeier <em>et al.</em> 2014</a>). FLM provided competitive values of predictive ability for both validation methods. However, the predictive performance of models may vary according to genetic architecture, marker density, trait heritability, and the size of the training set (<a id="xref-ref-8-3" class="xref-bibr" href="#ref-8">de los Campos <em>et al.</em> 2013</a>, <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Legarra <em>et al.</em> 2015</a>).</p><p id="p-63">Feature selection is a desirable statistical property known to improve the parsimony and predictive ability of WGR models (<a id="xref-ref-76-1" class="xref-bibr" href="#ref-76">Wimmer <em>et al.</em> 2013</a>). FLM deploys the so-called Laplacian variable selection (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">O’Hara and Sillanpää 2009</a>), which imposes strong shrinkage without eliminating the parameters from the model. Markers not linked to QTL often play an important role on prediction by capturing relationship among individuals (<a id="xref-ref-27-4" class="xref-bibr" href="#ref-27">Habier <em>et al.</em> 2007</a>). In addition, when regression coefficients have priors shaped by heavy tailed distribution, such as Laplace and Student’s t, models are suited to capturing QTL because these priors relax the shrinkage of markers with large effect (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">de los Campos <em>et al.</em> 2009</a>, <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Kärkkäinen and Sillanpää 2012</a>). Other models with similar properties include BayesA, BayesB, BayesC and the Bayesian LASSO (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">de los Campos <em>et al.</em> 2009</a>, <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Habier <em>et al.</em> 2011</a>, <a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Heslot <em>et al.</em> 2012</a>, <a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Kärkkäinen and Sillanpää 2012</a>, <a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Legarra <em>et al.</em> 2015</a>).</p><p id="p-64">From the signal detection perspective, models able to capture relationship and accurately detect QTL are deployed for association studies and haplotype analysis (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Fernando and Garrick 2013</a>, <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Hayes 2013</a>, <a id="xref-ref-85-1" class="xref-bibr" href="#ref-85">Yang <em>et al.</em> 2014</a>, <a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Daetwyler <em>et al.</em> 2015</a>, <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Fernando <em>et al.</em> 2017</a>, <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Goiffon <em>et al.</em> 2017</a>). For the scenarios under evaluation, FLM provided a more accurate marker effects estimation than the Bayesian LASSO and ridge regression, with less spurious association than GWA (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>). Both FLM and Bayesian LASSO have a Laplace prior, but with substantial algorithmic differences. Empirical priors have been reported to improve the predictive properties of Laplace models (<a id="xref-ref-82-2" class="xref-bibr" href="#ref-82">Xu 2007</a>, <a id="xref-ref-86-1" class="xref-bibr" href="#ref-86">Yi and Xu 2008</a>, <a id="xref-ref-83-2" class="xref-bibr" href="#ref-83">Xu 2010</a>, <a id="xref-ref-2-3" class="xref-bibr" href="#ref-2">Cai <em>et al.</em> 2011</a>), thus FLM likely benefits from regularization free of hyperparameters. Moreover, iterative algorithms often outperform their MCMC counterpart in terms of accuracy (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Hayashi and Iwata 2010</a>, <a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">Sun <em>et al.</em> 2012</a>, <a id="xref-ref-75-2" class="xref-bibr" href="#ref-75">Wang <em>et al.</em> 2015</a>). The resulting improvement in signal detection translates into higher predictive ability in scenarios where capturing linkage disequilibrium is more important than the relationship among individuals, as depicted in within-family predictions (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a>).</p><p id="p-65">The genetic signal captured by WGR methods is solely additive, which is desirable to estimate breeding values but sub-optimal for the prediction of phenotypes. Unlike additive models, semi-parametric methods can capture non-linear relationship patterns and different levels of epistasis. For this reason, additive models are frequently outperformed by semi-parametric methods, such as RKHS, SVR, random forest and neural networks (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Gianola <em>et al.</em> 2006</a>, <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">De Los Campos <em>et al.</em> 2010</a>, <a id="xref-ref-63-2" class="xref-bibr" href="#ref-63">Pérez-Rodríguez <em>et al.</em> 2012</a>, <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Desta and Ortiz 2014</a>, <a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Howard <em>et al.</em> 2014</a>). For the datasets under evaluation, linear models were as predictive as semi-parametric methods, which suggests that most genetic signal was due to additive genetics.</p><p id="p-66">Both RKHS and <em>ϵ</em>-SVR are kernel methods that utilize a Gaussian kernel, but these methods differ with regards to their loss-functions. Whereas RKHS follows a <span class="inline-formula" id="inline-formula-66"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-97.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11/3855/embed/mml-math-97.gif"/></noscript></span></span> loss that penalizes square error and coefficients, <em>ϵ</em>-SVR only penalizes the error greater than <em>ϵ</em> (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Hastie <em>et al.</em> 2009</a>). Interestingly, <em>ν</em>-SVR did not provide the same degree of predictive ability, despite sharing the same kernel as RKHS and <em>ϵ</em>-SVR.</p></div><div id="sec-23" class="subsection"><h3>Computational performance</h3><p id="p-67">The time required to calibrate a prediction machine is an important factor to chose a methodology when genomic prediction is utilized for various traits, with often model re-calibration (<a id="xref-ref-51-2" class="xref-bibr" href="#ref-51">Meuwissen <em>et al.</em> 2009</a>, <a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Hayashi and Iwata 2010</a>, <a id="xref-ref-71-2" class="xref-bibr" href="#ref-71">Sun <em>et al.</em> 2012</a>, <a id="xref-ref-75-3" class="xref-bibr" href="#ref-75">Wang <em>et al.</em> 2015</a>). Results indicate a clear discrepancy across methods with regards to the computing time required to fit the prediction models. <a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2</a> shows the most computationally efficient methods were PLS and the non-MCMC implementations. Other regression-type methods provided intermediate efficiency and kernel-type methods were computationally expensive.</p><p id="p-68">Most prediction methods display some computation burden: tuning parameters in machine learning methods; MCMC iterations in Bayesian methods; variance components in GBLUP; and matrix inversion or decomposition in kernel methods. FLM estimates full-conditional variance components, dismissing expensive matrix operations, cross-validation for tuning parameters, or MCMC. The other two prediction methods that efficiently implement Laplace prior, the empirical Bayesian LASSO and the extended Bayesian LASSO, did not provide satisfactory predictive ability. Besides FLM, our results indicate that BayesB is also a cost-effective regression method by providing reasonable computational cost with hist predictive ability across datasets.</p><p id="p-69">It is important to point out that kernel methods can be a suitable alternative in high dimensional models, since these rely on the number of individuals rather than the number of parameters. Kernel methods are computationally demanding for two other reasons: it is necessary to 1) build the kernel and 2) compute its inversion or Eigendecomposition. The time needed to build the kernel depends on the number of both individuals and parameters. Many kernels require the computation of distance matrices, which is more computationally demanding.</p><p id="p-70">For the prediction of new observations, kernels must be augmented with the genotypes of observed and unobserved individuals, making the inversion or spectral decomposition more challenging. This is particularly cumbersome in plant breeding where the size of the offspring being predicted and selected can be much larger than the training set, whereas the parameter estimates required for prediction from regression and tree models can be stored and easily employed for prediction of new observations.</p></div><div id="sec-24" class="subsection"><h3>Iterative single-stage modeling</h3><p id="p-71">The two-stage and iterative single-stage approaches were faster than GBLUP by an order of magnitude. Such difference can be attributed to the sparse nature of the algorithm and the complexity associated to the estimation of variance components. GBLUP was fit using AI-REML, a general-purpose algorithm, whereas FLM-SS was specifically designed to provide efficient computation of breeding models. The two-stages model provided an intermediate outcome.</p><p id="p-72">The lower accuracy of the GBLUP and RR-SS in scenarios with few QTL can be attributed to the statistical nature based an infinitesimal model. Gaussian priors work by capturing the relationship among individuals (<a id="xref-ref-27-5" class="xref-bibr" href="#ref-27">Habier <em>et al.</em> 2007</a>), whereas FLM-SS and two-steps models enable fitting priors that are suitable to capture both relationship and QTL (<a id="xref-ref-77-1" class="xref-bibr" href="#ref-77">Wolc <em>et al.</em> 2016</a>). Similar results were reported by <a id="xref-ref-89-2" class="xref-bibr" href="#ref-89">Zhou <em>et al.</em> (2018)</a>, where one-step BayesA and BayesB consistently outperformed the one-step GBLUP under various simulated scenarios. This advantage is also depicted in within-family prediction (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2</a>) where the prediction power comes from detecting LD between markers and QTL, as well as in <a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a>, where it takes a larger number of observations to BRR (counterpart of GBLUP) to identify large effect markers (<a id="xref-ref-8-4" class="xref-bibr" href="#ref-8">de los Campos <em>et al.</em> 2013</a>, <a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Henryon <em>et al.</em> 2014</a>, <a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Hickey <em>et al.</em> 2017</a>).</p><p id="p-73">Frameworks where marker effects are estimate alongside all other parameters are not new, but rarely utilized (<a id="xref-ref-16-4" class="xref-bibr" href="#ref-16">Fernando <em>et al.</em> 2014</a>, <a id="xref-ref-49-3" class="xref-bibr" href="#ref-49">Liu <em>et al.</em> 2014</a>, <a id="xref-ref-72-3" class="xref-bibr" href="#ref-72">Taskinen <em>et al.</em> 2017</a>). Methods and implementations of genomic prediction have been incorporated from animal breeding into plant breeding without much consideration about the large differences in data flow and other statistical properties (<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Heslot <em>et al.</em> 2015</a>, <a id="xref-ref-36-3" class="xref-bibr" href="#ref-36">Hickey <em>et al.</em> 2017</a>). Two of the major factors that differentiate plant and animal breeding are replicated trials and offspring size. The single-stage framework proposed in this study was design for genomic prediction following the plant breeding data structure, being beneficial from the computational and predictive standpoint.</p></div><div id="sec-25" class="subsection"><h3>Two-stages and iterative single-stage</h3><p id="p-74">Multi-stage procedures have been proposed to address the computational burden of single-stage without compromising in the quality of the results (<a id="xref-ref-70-2" class="xref-bibr" href="#ref-70">Smith <em>et al.</em> 2001</a>). There are various methods to propagate weights from one stage to the next (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Möhring and Piepho 2009</a>). However, multi-stage analysis cannot recreate the exact results of single-stage unless the whole covariance matrix is carried over stages (<a id="xref-ref-64-2" class="xref-bibr" href="#ref-64">Piepho <em>et al.</em> 2012</a>). Even so, the results of multi-stage only reproduce single-stage if the variance components of the first-stage analysis are identical to the variance components estimated from single-stage, and if the nuisance parameters (non-target fixed effects) are estimated free of error. In this case, sufficient statistics (means and covariance) may fully reproduce single-stage. Multi-stage approaches commonly provide results comparable to those from single-stage analysis (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Damesa <em>et al.</em> 2017</a>). However, if statistical terms introduced in the second-stage analysis would potentially affect the variance component estimates of first-stage terms, then the single-stage results cannot be exactly reproduced by two-stages. That is not necessarily the case for iterative methods where all coefficients and the variance components are estimated conditional to each other, and the information of all terms is propagated across iterations.</p><p id="p-75">Future directions for this research include the comparison of breeding values estimated from GBLUP, multi-stage and RR-SS to understand how the proposed algorithm reproduces standard procedures on real data. An evaluation of the estimated variance components is also a desirable target. Further studies must contrast scenarios with genotype-environment interaction, spatial trends, dominance and epistasis.</p></div><div id="sec-26" class="subsection"><h3>CONCLUSION</h3><p id="p-76">A robust prediction methodology is a key component for a successful genomic-assisted breeding pipeline. This study introduced a fast and accurate algorithm for solving a WGR with Laplace prior, alongside a single-stage methodology that allows to connect WGR into mixed models with replicated observations.</p><p id="p-77">The proposed framework provided more accurate predictions and higher computational efficiency than other methods based on a cross-validation evaluation on real datasets. With a simulated dataset, it was shown that the fast Laplace model provided reasonably accurate estimation of QTL effects, being less biased than Bayesian LASSO and ridge regression, and proving less spurious signals than genome-wide association analysis. The algorithm extension to single-stage also presented promising properties, benefiting both computation and prediction.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-79">David Habier is acknowledged for revision and useful discussions. Makram Geha contributed to the overall goal the research. Shizhong Xu gave an insight on the coordinate descent methodology. James Holland provided valuable editorial feedback.</p></div><div class="section app" id="app-1"><h2 class="">Appendix</h2><div class="section" id="sec-27"><h3>RCPP CODE TO IMPLEMENT FLM IN R</h3><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/11/3855/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1992299307" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/11/3855/F5.medium.gif" width="789" height="1688"/><noscript><img class="highwire-fragment fragment-image" alt="Figure5" src="https://www.g3journal.org/content/ggg/9/11/3855/F5.medium.gif" width="789" height="1688"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/11/3855/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/11/3855/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494727" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><p id="p-78"></p></div></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-80"><em>Communicating editor: J. Holland</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2019-08-21"><span class="received-label">Received </span>August 21, 2019.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2019-09-18"><span class="accepted-label">Accepted </span>September 18, 2019.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2019 Xavier</li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3855.1" data-doi="10.18637/jss.v067.i01"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bates</span>,  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Mächler</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Bolker</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Walker</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Fitting linear mixed-effects models using lme4.</span> <abbr class="cit-jnl-abbrev">J. Stat. Softw.</abbr> <span class="cit-vol">67</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">48</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.18637/jss.v067.i01</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BStat.%2BSoftw.%26rft.volume%253D67%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.18637%252Fjss.v067.i01%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.18637/jss.v067.i01&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3855.2" data-doi="10.1186/1471-2105-12-211"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cai</span>,  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Huang</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Xu</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Fast empirical bayesian lasso for multiple quantitative trait locus mapping.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">211</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2105-12-211</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DCai%26rft.auinit1%253DX.%26rft.volume%253D12%26rft.spage%253D211%26rft.epage%253D211%26rft.atitle%253DFast%2Bempirical%2BBayesian%2BLASSO%2Bfor%2Bmultiple%2Bquantitative%2Btrait%2Blocus%2Bmapping.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-12-211%26rft_id%253Dinfo%253Apmid%252F21615941%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-12-211&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21615941&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3855.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3855.3" data-doi="10.1371/journal.pone.0087666"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Da</span>,  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Wang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Wang</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Hu</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Mixed model methods for genomic prediction and variance component estimation of additive and dominance effects using snp markers.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e87666</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0087666</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De87666%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0087666%26rft_id%253Dinfo%253Apmid%252F24498162%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0087666&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24498162&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3855.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3855.4" data-doi="10.1534/genetics.112.147983"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Daetwyler</span>,  <span class="cit-name-given-names">H. D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. P.</span> <span class="cit-name-surname">Calus</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Pong-Wong</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">de los Campos</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">Hickey</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genomic prediction in animals and plants: simulation of data, validation, reporting, and benchmarking.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">193</span>: <span class="cit-fpage">347</span>–<span class="cit-lpage">365</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.112.147983</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.147983%26rft_id%253Dinfo%253Apmid%252F23222650%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkzLzIvMzQ3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMS8zODU1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3855.5" data-doi="10.1534/genetics.115.178038"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Daetwyler</span>,  <span class="cit-name-given-names">H. D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. J.</span> <span class="cit-name-surname">Hayden</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G. C.</span> <span class="cit-name-surname">Spangenberg</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. J.</span> <span class="cit-name-surname">Hayes</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Selection on optimal haploid value increases genetic gain and preserves more genetic diversity relative to genomic selection.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">200</span>: <span class="cit-fpage">1341</span>–<span class="cit-lpage">1348</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.115.178038</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.115.178038%26rft_id%253Dinfo%253Apmid%252F26092719%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjIwMC80LzEzNDEiO3M6NDoiYXRvbSI7czoxOToiL2dnZy85LzExLzM4NTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3855.6" data-doi="10.2134/agronj2016.07.0395"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Damesa</span>,  <span class="cit-name-given-names">T. M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Möhring</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Worku</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.-P.</span> <span class="cit-name-surname">Piepho</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">One step at a time: stage-wise analysis of a series of experiments.</span> <abbr class="cit-jnl-abbrev">Agron. J.</abbr> <span class="cit-vol">109</span>: <span class="cit-fpage">845</span>–<span class="cit-lpage">857</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2134/agronj2016.07.0395</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAgron.%2BJ.%26rft.volume%253D109%26rft.spage%253D845%26rft_id%253Dinfo%253Adoi%252F10.2134%252Fagronj2016.07.0395%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2134/agronj2016.07.0395&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3855.7" data-doi="10.1017/S0016672310000285"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>,  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Gianola</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G. J.</span> <span class="cit-name-surname">Rosa</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. A.</span> <span class="cit-name-surname">Weigel</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Crossa</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Semi-parametric genomic-enabled prediction of genetic values using reproducing kernel hilbert spaces methods.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">92</span>: <span class="cit-fpage">295</span>–<span class="cit-lpage">308</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1017/S0016672310000285</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DDE%2BLos%2BCampos%26rft.auinit1%253DG.%26rft.volume%253D92%26rft.issue%253D4%26rft.spage%253D295%26rft.epage%253D308%26rft.atitle%253DSemi-parametric%2Bgenomic-enabled%2Bprediction%2Bof%2Bgenetic%2Bvalues%2Busing%2Breproducing%2Bkernel%2BHilbert%2Bspaces%2Bmethods.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672310000285%26rft_id%253Dinfo%253Apmid%252F20943010%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672310000285&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20943010&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3855.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3855.8" data-doi="10.1534/genetics.112.143313"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>,  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">Hickey</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Pong-Wong</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H. D.</span> <span class="cit-name-surname">Daetwyler</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. P.</span> <span class="cit-name-surname">Calus</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Whole-genome regression and prediction methods applied to plant and animal breeding.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">193</span>: <span class="cit-fpage">327</span>–<span class="cit-lpage">345</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.112.143313</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.143313%26rft_id%253Dinfo%253Apmid%252F22745228%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkzLzIvMzI3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMS8zODU1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3855.9" data-doi="10.1534/genetics.109.101501"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>,  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Naya</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Gianola</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Crossa</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Legarra</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Predicting quantitative traits with regression models for dense molecular markers and pedigrees.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">182</span>: <span class="cit-fpage">375</span>–<span class="cit-lpage">385</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.109.101501</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.109.101501%26rft_id%253Dinfo%253Apmid%252F19293140%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTgyLzEvMzc1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMS8zODU1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3855.10" data-doi="10.1016/j.tplants.2014.05.006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Desta</span>,  <span class="cit-name-given-names">Z. A.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Ortiz</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genomic selection: genome-wide prediction in plant improvement.</span> <abbr class="cit-jnl-abbrev">Trends Plant Sci.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">592</span>–<span class="cit-lpage">601</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.tplants.2014.05.006</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BPlant%2BSci.%26rft.volume%253D19%26rft.spage%253D592%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tplants.2014.05.006%26rft_id%253Dinfo%253Apmid%252F24970707%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tplants.2014.05.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24970707&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3855.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3855.11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Diers</span>,  <span class="cit-name-given-names">B. W.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Specht</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. M.</span> <span class="cit-name-surname">Rainey</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Cregan</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Q.</span> <span class="cit-name-surname">Song</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Genetic architecture of soybean yield and agronomic traits. G3: Genes, Genomes</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">3367</span>–<span class="cit-lpage">3375</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D8%26rft.spage%253D3367%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3855.12" data-doi="10.18637/jss.v040.i01"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eddelbuettel</span>,  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">François</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Allaire</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Ushey</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Q.</span> <span class="cit-name-surname">Kou</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Rcpp: Seamless r and c++ integration.</span> <abbr class="cit-jnl-abbrev">J. Stat. Softw.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">18</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.18637/jss.v040.i08</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DJ%2BStat%2BSoftw%26rft.aulast%253DJara%26rft.auinit1%253DA.%26rft.volume%253D40%26rft.issue%253D12%26rft.spage%253D1%26rft.epage%253D32%26rft.atitle%253DUsing%2Bthe%2BR%2BPackage%2Bcrlmm%2Bfor%2BGenotyping%2Band%2BCopy%2BNumber%2BEstimation.%26rft_id%253Dinfo%253Adoi%252F10.18637%252Fjss.v040.i01%26rft_id%253Dinfo%253Apmid%252F22523482%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.18637/jss.v040.i01&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22523482&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3855.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3855.13" data-doi="10.3835/plantgenome2011.08.0024"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Endelman</span>,  <span class="cit-name-given-names">J. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Ridge regression and other kernels for genomic selection with r package rrblup.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">250</span>–<span class="cit-lpage">255</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2011.08.0024</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D4%26rft.spage%253D250%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2011.08.0024%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2011.08.0024&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3855.14" data-doi="10.1007/s13253-017-0277-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>,  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Toosi</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Wolc</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Garrick</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Dekkers</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Application of whole-genome prediction methods for genome-wide association studies: a bayesian approach.</span> <abbr class="cit-jnl-abbrev">J. Agric. Biol. Environ. Stat.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">172</span>–<span class="cit-lpage">193</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s13253-017-0277-6</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAgric.%2BBiol.%2BEnviron.%2BStat.%26rft.volume%253D22%26rft.spage%253D172%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs13253-017-0277-6%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s13253-017-0277-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3855.15" data-doi="10.1186/s12711-016-0273-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>,  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Cheng</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. L.</span> <span class="cit-name-surname">Golden</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. J.</span> <span class="cit-name-surname">Garrick</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Computational strategies for alternative single-step bayesian regression models with large numbers of genotyped and non-genotyped animals.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">48</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12711-016-0273-2</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12711-016-0273-2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12711-016-0273-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3855.16" data-doi="10.1186/1297-9686-46-50"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>,  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. C.</span> <span class="cit-name-surname">Dekkers</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. J.</span> <span class="cit-name-surname">Garrick</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A class of bayesian methods to combine large numbers of genotyped and non-genotyped animals for whole-genome analyses.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">50</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1297-9686-46-50</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D46%26rft.spage%253D50%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-46-50%26rft_id%253Dinfo%253Apmid%252F25253441%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-46-50&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25253441&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3855.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-book" id="cit-9.11.3855.17" data-doi="10.1007/978-1-62703-447-0_10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>,  <span class="cit-name-given-names">R. L.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Garrick</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Bayesian methods applied to gwas</span>, pp. <span class="cit-fpage">237</span>–<span class="cit-lpage">274</span> in <span class="cit-source">Genome-wide association studies and genomic prediction</span>, edited by <span class="cit-ed"><span class="cit-name-surname">van der Werf Gondro</span>,  <span class="cit-name-given-names">C.</span></span>, and <span class="cit-ed"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Hayes</span></span>. <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">Berlin, Germany</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/978-1-62703-447-0_10</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome-wide%2Bassociation%2Bstudies%2Band%2Bgenomic%2Bprediction%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-1-62703-447-0_10%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-1-62703-447-0_10&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3855.18" data-doi="10.1111/j.1467-9868.2005.00503.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Friedman</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Hastie</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Tibshirani</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Regularization paths for generalized linear models via coordinate descent.</span> <abbr class="cit-jnl-abbrev">J. Stat. Softw.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">22</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.18637/jss.v033.i01</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DJ%2BStat%2BSoftw%26rft.aulast%253DFriedman%26rft.auinit1%253DJ.%26rft.volume%253D33%26rft.issue%253D1%26rft.spage%253D1%26rft.epage%253D22%26rft.atitle%253DRegularization%2BPaths%2Bfor%2BGeneralized%2BLinear%2BModels%2Bvia%2BCoordinate%2BDescent.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1467-9868.2005.00503.x%26rft_id%253Dinfo%253Apmid%252F20808728%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1467-9868.2005.00503.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20808728&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3855.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000275203200001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3855.19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Garrick</span>,  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Equivalent mixed model equations for genomic selection.</span> <abbr class="cit-jnl-abbrev">J. Anim. Sci.</abbr> <span class="cit-vol">85</span>: <span class="cit-fpage">376</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAnim.%2BSci.%26rft.volume%253D85%26rft.spage%253D376%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3855.20" data-doi="10.1016/j.livsci.2014.05.031"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Garrick</span>,  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Dekkers</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Fernando</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The evolution of methodologies for genomic prediction.</span> <abbr class="cit-jnl-abbrev">Livest. Sci.</abbr> <span class="cit-vol">166</span>: <span class="cit-fpage">10</span>–<span class="cit-lpage">18</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.livsci.2014.05.031</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLivest.%2BSci.%26rft.volume%253D166%26rft.spage%253D10%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.livsci.2014.05.031%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.livsci.2014.05.031&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3855.21" data-doi="10.1186/1297-9686-41-55"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Garrick</span>,  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. F.</span> <span class="cit-name-surname">Taylor</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. L.</span> <span class="cit-name-surname">Fernando</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Deregressing estimated breeding values and weighting information for genomic regression analyses.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">55</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1297-9686-41-55</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DGarrick%26rft.auinit1%253DD.%2BJ.%26rft.volume%253D41%26rft.spage%253D55%26rft.epage%253D55%26rft.atitle%253DDeregressing%2Bestimated%2Bbreeding%2Bvalues%2Band%2Bweighting%2Binformation%2Bfor%2Bgenomic%2Bregression%2Banalyses.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-41-55%26rft_id%253Dinfo%253Apmid%252F20043827%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-41-55&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20043827&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3855.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-9.11.3855.22" data-doi="10.1038/s41576-018-0082-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Georges</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Charlier</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Hayes</span></span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">Harnessing genomic information for livestock improvement.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">135</span>–<span class="cit-lpage">156</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/s41576-018-0082-2</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BGenet.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41576-018-0082-2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41576-018-0082-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3855.23" data-doi="10.1534/genetics.105.049510"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>,  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. L.</span> <span class="cit-name-surname">Fernando</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Stella</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Genomic assisted prediction of genetic value with semi-parametric procedures.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">173</span>: <span class="cit-fpage">1761</span>–<span class="cit-lpage">1776</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.105.049510</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.105.049510%26rft_id%253Dinfo%253Apmid%252F16648593%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3My8zLzE3NjEiO3M6NDoiYXRvbSI7czoxOToiL2dnZy85LzExLzM4NTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-book" id="cit-9.11.3855.24"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gilmour</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Gogel</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Cullis</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Thompson</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Butler</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-source">Asreml user guide release 3.0</span>, <span class="cit-publ-name">VSN Int Ltd</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3855.25" data-doi="10.1534/genetics.116.197103"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goiffon</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Kusmec</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Wang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Hu</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Schnable</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Improving response in genomic selection with a population-based selection strategy: optimal population value selection.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">206</span>: <span class="cit-fpage">1675</span>–<span class="cit-lpage">1682</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.116.197103</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.116.197103%26rft_id%253Dinfo%253Apmid%252F28526698%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjIwNi8zLzE2NzUiO3M6NDoiYXRvbSI7czoxOToiL2dnZy85LzExLzM4NTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3855.26" data-doi="10.2527/jas1987.6451362x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Graser</span>,  <span class="cit-name-given-names">H.-U.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Smith</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Tier</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span> <span class="cit-article-title">A derivative-free approach for estimating variance components in animal models by restricted maximum likelihood 1.</span> <abbr class="cit-jnl-abbrev">J. Anim. Sci.</abbr> <span class="cit-vol">64</span>: <span class="cit-fpage">1362</span>–<span class="cit-lpage">1370</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2527/jas1987.6451362x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BAnimal%2BScience%26rft.stitle%253DJ%2BANIM%2BSCI%26rft.aulast%253DGraser%26rft.auinit1%253DH.-U.%26rft.volume%253D64%26rft.issue%253D5%26rft.spage%253D1362%26rft.epage%253D1370%26rft.atitle%253DA%2BDerivative-Free%2BApproach%2Bfor%2BEstimating%2BVariance%2BComponents%2Bin%2BAnimal%2BModels%2Bby%2BRestricted%2BMaximum%2BLikelihood%26rft_id%253Dinfo%253Adoi%252F10.2527%252Fjas1987.6451362x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2527/jas1987.6451362x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1987J700900011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3855.27" data-doi="10.1534/genetics.107.081190"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Habier</span>,  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Fernando</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. C.</span> <span class="cit-name-surname">Dekkers</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">The impact of genetic relationship information on genome-assisted breeding values.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">177</span>: <span class="cit-fpage">2389</span>–<span class="cit-lpage">2397</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.107.081190</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DHabier%26rft.auinit1%253DD.%26rft.volume%253D177%26rft.issue%253D4%26rft.spage%253D2389%26rft.epage%253D2397%26rft.atitle%253DThe%2BImpact%2Bof%2BGenetic%2BRelationship%2BInformation%2Bon%2BGenome-Assisted%2BBreeding%2BValues%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.081190%26rft_id%253Dinfo%253Apmid%252F18073436%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3Ny80LzIzODkiO3M6NDoiYXRvbSI7czoxOToiL2dnZy85LzExLzM4NTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3855.28" data-doi="10.1186/1471-2105-12-186"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Habier</span>,  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. L.</span> <span class="cit-name-surname">Fernando</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Kizilkaya</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. J.</span> <span class="cit-name-surname">Garrick</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Extension of the bayesian alphabet for genomic selection.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">186</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2105-12-186</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DHabier%26rft.auinit1%253DD.%26rft.volume%253D12%26rft.spage%253D186%26rft.epage%253D186%26rft.atitle%253DExtension%2Bof%2Bthe%2Bbayesian%2Balphabet%2Bfor%2Bgenomic%2Bselection.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-12-186%26rft_id%253Dinfo%253Apmid%252F21605355%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-12-186&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21605355&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3855.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3855.29" data-doi="10.1080/01621459.1977.10480998"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Harville</span>,  <span class="cit-name-given-names">D. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1977</span> <span class="cit-article-title">Maximum likelihood approaches to variance component estimation and to related problems.</span> <abbr class="cit-jnl-abbrev">J. Am. Stat. Assoc.</abbr> <span class="cit-vol">72</span>: <span class="cit-fpage">320</span>–<span class="cit-lpage">338</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1080/01621459.1977.10480998</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAm.%2BStat.%2BAssoc.%26rft_id%253Dinfo%253Adoi%252F10.1080%252F01621459.1977.10480998%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/01621459.1977.10480998&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3855.30" data-doi="10.1007/978-0-387-84858-7"><div class="cit-metadata"><cite>Hastie, T., J. Friedman, and R. Tibshirani, 2009 <em>The elements of statistical learning</em>, volume 2. Springer series in statistics New York.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/978-0-387-84858-7</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft_id%253Dinfo%253Adoi%252F10.1007%252F978-0-387-84858-7%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-0-387-84858-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3855.31" data-doi="10.1186/1471-2156-11-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hayashi</span>,  <span class="cit-name-given-names">T.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Iwata</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Em algorithm for bayesian estimation of genomic breeding values.</span> <abbr class="cit-jnl-abbrev">BMC Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">3</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2156-11-3</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenetics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenet%26rft.aulast%253DHayashi%26rft.auinit1%253DT.%26rft.volume%253D11%26rft.spage%253D3%26rft.epage%253D3%26rft.atitle%253DEM%2Balgorithm%2Bfor%2BBayesian%2Bestimation%2Bof%2Bgenomic%2Bbreeding%2Bvalues.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2156-11-3%26rft_id%253Dinfo%253Apmid%252F20092655%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2156-11-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20092655&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3855.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-book" id="cit-9.11.3855.32" data-doi="10.1007/978-1-62703-447-0_6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>,  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Overview of statistical methods for genome-wide association studies (gwas)</span>, pp. <span class="cit-fpage">149</span>–<span class="cit-lpage">169</span> in <span class="cit-source">Genome-wide association studies and genomic prediction</span>, edited by <span class="cit-ed"><span class="cit-name-surname">van der Werf Gondro</span>,  <span class="cit-name-given-names">C.</span></span>, and <span class="cit-ed"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Hayes</span></span>. <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">Berlin, Germany</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/978-1-62703-447-0_6</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome-wide%2Bassociation%2Bstudies%2Band%2Bgenomic%2Bprediction%26rft.volume%253D1019%26rft.spage%253D149%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-1-62703-447-0_6%26rft_id%253Dinfo%253Apmid%252F23756890%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-1-62703-447-0_6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23756890&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3855.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3855.33" data-doi="10.1016/j.livsci.2014.06.016"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Henryon</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Berg</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. C.</span> <span class="cit-name-surname">Sørensen</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Animal-breeding schemes using genomic information need breeding plans designed to maximise long-term genetic gains.</span> <abbr class="cit-jnl-abbrev">Livest. Sci.</abbr> <span class="cit-vol">166</span>: <span class="cit-fpage">38</span>–<span class="cit-lpage">47</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.livsci.2014.06.016</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLivest.%2BSci.%26rft.volume%253D166%26rft.spage%253D38%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.livsci.2014.06.016%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.livsci.2014.06.016&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3855.34" data-doi="10.2135/cropsci2014.03.0249"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heslot</span>,  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.-L.</span> <span class="cit-name-surname">Jannink</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. E.</span> <span class="cit-name-surname">Sorrells</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Perspectives for genomic selection applications and research in plants.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">55</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">12</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2014.03.0249</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D55%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2014.03.0249%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2014.03.0249&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3855.35" data-doi="10.2135/cropsci2011.06.0297"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heslot</span>,  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.-P.</span> <span class="cit-name-surname">Yang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. E.</span> <span class="cit-name-surname">Sorrells</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.-L.</span> <span class="cit-name-surname">Jannink</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genomic selection in plant breeding: a comparison of models.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">146</span>–<span class="cit-lpage">160</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2011.06.0297</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D52%26rft.spage%253D146%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2011.06.0297%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2011.06.0297&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3855.36" data-doi="10.1038/ng.3920"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hickey</span>,  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Chiurugwi</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Mackay</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Powell</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Eggen</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Genomic prediction unifies animal and plant breeding programs to form platforms for biological discovery.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">1297</span>–<span class="cit-lpage">1303</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.3920</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D49%26rft.spage%253D1297%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.3920%26rft_id%253Dinfo%253Apmid%252F28854179%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.3920&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28854179&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3855.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3855.37"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Howard</span>,  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. L.</span> <span class="cit-name-surname">Carriquiry</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">W. D.</span> <span class="cit-name-surname">Beavis</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Parametric and nonparametric statistical methods for genomic selection of traits with additive and epistatic genetic architectures. G3: Genes, Genomes</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">1027</span>–<span class="cit-lpage">1046</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D4%26rft.spage%253D1027%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3855.38" data-doi="10.1093/bioinformatics/btw143"><div class="cit-metadata"><cite>Huang, A. and D. Liu, 2016 EBglmnet: a comprehensive R package for sparse generalized linear regression models. Bioinformatics pp. 1–3, btw143.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btw143</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtw143%26rft_id%253Dinfo%253Apmid%252F27153585%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btw143&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27153585&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3855.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-9.11.3855.39" data-doi="10.2135/cropsci2013.11.0774"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jacobson</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Lian</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Zhong</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Bernardo</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">General combining ability model for genomewide selection in a biparental cross.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">54</span>: <span class="cit-fpage">895</span>–<span class="cit-lpage">905</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2013.11.0774</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D54%26rft.spage%253D895%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2013.11.0774%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2013.11.0774&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3855.40" data-doi="10.18637/jss.v011.i09"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Karatzoglou</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Smola</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Hornik</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Zeileis</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">kernlab-an s4 package for kernel methods in r.</span> <abbr class="cit-jnl-abbrev">J. Stat. Softw.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">20</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.18637/jss.v011.i09</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BStat.%2BSoftw.%26rft.volume%253D11%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.18637%252Fjss.v011.i09%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.18637/jss.v011.i09&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3855.41" data-doi="10.1534/genetics.112.139014"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kärkkäinen</span>,  <span class="cit-name-given-names">H. P.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. J.</span> <span class="cit-name-surname">Sillanpää</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Back to basics for bayesian model building in genomic selection.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">191</span>: <span class="cit-fpage">969</span>–<span class="cit-lpage">987</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.112.139014</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.139014%26rft_id%253Dinfo%253Apmid%252F22554888%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkxLzMvOTY5IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMS8zODU1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3855.42" data-doi="10.1186/s12711-015-0087-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Legarra</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Croiseau</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. P.</span> <span class="cit-name-surname">Sanchez</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Teyssèdre</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Sallè</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">A comparison of methods for whole-genome qtl mapping using dense markers in four livestock species.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">6</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12711-015-0087-7</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D47%26rft.spage%253D6%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12711-015-0087-7%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12711-015-0087-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3855.43" data-doi="10.3168/jds.2007-0403"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Legarra</span>,  <span class="cit-name-given-names">A.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Misztal</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Computing strategies in genome-wide selection.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">360</span>–<span class="cit-lpage">366</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3168/jds.2007-0403</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DLegarra%26rft.auinit1%253DA.%26rft.volume%253D91%26rft.issue%253D1%26rft.spage%253D360%26rft.epage%253D366%26rft.atitle%253DTechnical%2BNote%253A%2BComputing%2BStrategies%2Bin%2BGenome-Wide%2BSelection%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2007-0403%26rft_id%253Dinfo%253Apmid%252F18096959%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2007-0403&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18096959&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3855.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000251800200041&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3855.44" data-doi="10.1017/S0016672310000534"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Legarra</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Robert-Granié</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Croiseau</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Guillaume</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Fritz</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Improved lasso for genomic selection.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">93</span>: <span class="cit-fpage">77</span>–<span class="cit-lpage">87</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1017/S0016672310000534</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DLegarra%26rft.auinit1%253DA.%26rft.volume%253D93%26rft.issue%253D1%26rft.spage%253D77%26rft.epage%253D87%26rft.atitle%253DImproved%2BLasso%2Bfor%2Bgenomic%2Bselection.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672310000534%26rft_id%253Dinfo%253Apmid%252F21144129%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672310000534&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21144129&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3855.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3855.45" data-doi="10.1534/genetics.108.088575"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Legarra</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Robert-Granié</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Manfredi</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.-M.</span> <span class="cit-name-surname">Elsen</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Performance of genomic selection in mice.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">180</span>: <span class="cit-fpage">611</span>–<span class="cit-lpage">618</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.108.088575</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.108.088575%26rft_id%253Dinfo%253Apmid%252F18757934%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTgwLzEvNjExIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMS8zODU1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3855.46" data-doi="10.1534/genetics.114.161943"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lehermeier</span>,  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Krämer</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Bauer</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Bauland</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Camisan</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Usefulness of multiparental populations of maize (zea mays l.) for genome-based prediction.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">198</span>: <span class="cit-fpage">3</span>–<span class="cit-lpage">16</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.114.161943</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DLehermeier%26rft.auinit1%253DC.%26rft.volume%253D198%26rft.issue%253D1%26rft.spage%253D3%26rft.epage%253D16%26rft.atitle%253DUsefulness%2Bof%2BMultiparental%2BPopulations%2Bof%2BMaize%2B%2528Zea%2Bmays%2BL.%2529%2Bfor%2BGenome-Based%2BPrediction%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.161943%26rft_id%253Dinfo%253Apmid%252F25236445%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6NzoiMTk4LzEvMyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzkvMTEvMzg1NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3855.47" data-doi="10.2135/cropsci2013.12.0856"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lian</span>,  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Jacobson</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Zhong</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Bernardo</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genomewide prediction accuracy within 969 maize biparental populations.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">54</span>: <span class="cit-fpage">1514</span>–<span class="cit-lpage">1522</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2013.12.0856</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D54%26rft.spage%253D1514%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2013.12.0856%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2013.12.0856&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3855.48" data-doi="10.2307/2531083"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>,  <span class="cit-name-given-names">C.-S.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Poushinsky</span></span></li></ol><cite>, <span class="cit-pub-date">1983</span> <span class="cit-article-title">A modified augmented design for an early stage of plant selection involving a large number of test lines without replication.</span> <abbr class="cit-jnl-abbrev">Biometrics</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">553</span>–<span class="cit-lpage">561</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2307/2531083</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiometrics%26rft.volume%253D39%26rft.spage%253D553%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2531083%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2531083&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3855.49" data-doi="10.3168/jds.2014-7924"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>,  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Goddard</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Reinhardt</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Reents</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A single-step genomic model with direct estimation of marker effects.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">97</span>: <span class="cit-fpage">5833</span>–<span class="cit-lpage">5850</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3168/jds.2014-7924</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BDairy%2BSci.%26rft.volume%253D97%26rft.spage%253D5833%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2014-7924%26rft_id%253Dinfo%253Apmid%252F25022678%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2014-7924&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25022678&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3855.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3855.50" data-doi="10.1093/nar/gkv1344"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>,  <span class="cit-name-given-names">T. H. E.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. J.</span> <span class="cit-name-surname">Hayes</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. E.</span> <span class="cit-name-surname">Goddard</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Prediction of total genetic value using genome-wide dense marker maps.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">157</span>: <span class="cit-fpage">1819</span>–<span class="cit-lpage">1829</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMeuwissen%26rft.auinit1%253DT.%2BH.%2BE.%26rft.volume%253D157%26rft.issue%253D4%26rft.spage%253D1819%26rft.epage%253D1829%26rft.atitle%253DPrediction%2Bof%2BTotal%2BGenetic%2BValue%2BUsing%2BGenome-Wide%2BDense%2BMarker%2BMaps%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkv1344%26rft_id%253Dinfo%253Apmid%252F11290733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1Ny80LzE4MTkiO3M6NDoiYXRvbSI7czoxOToiL2dnZy85LzExLzM4NTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3855.51" data-doi="10.1186/1297-9686-41-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>,  <span class="cit-name-given-names">T. H. E.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. R.</span> <span class="cit-name-surname">Solberg</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Shepherd</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. A.</span> <span class="cit-name-surname">Woolliams</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A fast algorithm for bayesb type of prediction of genome-wide estimates of genetic value.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">2</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1297-9686-41-2</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DMeuwissen%26rft.auinit1%253DT.%2BH.%26rft.volume%253D41%26rft.spage%253D2%26rft.epage%253D2%26rft.atitle%253DA%2Bfast%2Balgorithm%2Bfor%2BBayesB%2Btype%2Bof%2Bprediction%2Bof%2Bgenome-wide%2Bestimates%2Bof%2Bgenetic%2Bvalue.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-41-2%26rft_id%253Dinfo%253Apmid%252F19284681%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-41-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19284681&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3855.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3855.52" data-doi="10.1360/jos180001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mevik</span>,  <span class="cit-name-given-names">B.-H.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Wehrens</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">The pls package: principal component and partial least squares regression in r.</span> <abbr class="cit-jnl-abbrev">J. Stat. Softw.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">23</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.18637/jss.v018.i02</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BStat.%2BSoftw.%26rft.volume%253D18%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1360%252Fjos180001%26rft_id%253Dinfo%253Apmid%252F15238266%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1360/jos180001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15238266&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3855.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3855.53" data-doi="10.1534/genetics.115.182089"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Misztal</span>,  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Inexpensive computation of the inverse of the genomic relationship matrix in populations with small effective population size.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">202</span>: <span class="cit-fpage">401</span>–<span class="cit-lpage">409</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.115.182089</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.115.182089%26rft_id%253Dinfo%253Apmid%252F26584903%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjAyLzIvNDAxIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMS8zODU1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-book" id="cit-9.11.3855.54"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Misztal</span>,  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Aguilar</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Johnson</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Legarra</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Tsuruta</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> pp. <span class="cit-fpage">240</span>–<span class="cit-lpage">244</span> in <span class="cit-source"><em>A unified approach to utilize phenotypic</em>, <em>full pedigree and genomic information for a genetic evaluation of holstein final score</em></span>, <span class="cit-publ-name">Interbull Bulletin</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3855.55"><div class="cit-metadata unstructured">Misztal, I. and A. Legarra, 2017 Invited review: efficient computation strategies in genomic selection. animal <strong>11</strong>: 731–736.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3855.56" data-doi="10.2135/cropsci2009.02.0083"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Möhring</span>,  <span class="cit-name-given-names">J.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.-P.</span> <span class="cit-name-surname">Piepho</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Comparison of weighting in two-stage analysis of plant breeding trials.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">1977</span>–<span class="cit-lpage">1988</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2009.02.0083</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DMohring%26rft.auinit1%253DJ.%26rft.volume%253D49%26rft.issue%253D6%26rft.spage%253D1977%26rft.epage%253D1988%26rft.atitle%253DComparison%2Bof%2BWeighting%2Bin%2BTwo-Stage%2BAnalysis%2Bof%2BPlant%2BBreeding%2BTrials%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2009.02.0083%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2009.02.0083&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000271607300004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3855.57" data-doi="10.1214/09-BA403"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">O’Hara</span>,  <span class="cit-name-given-names">R. B.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. J.</span> <span class="cit-name-surname">Sillanpää</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A review of bayesian variable selection methods: what, how and which.</span> <abbr class="cit-jnl-abbrev">Bayesian Anal.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">85</span>–<span class="cit-lpage">117</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1214/09-BA403</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBayesian%2BAnal.%26rft.volume%253D4%26rft.spage%253D85%26rft_id%253Dinfo%253Adoi%252F10.1214%252F09-BA403%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1214/09-BA403&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000273483200007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3855.58"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Onogi</span>,  <span class="cit-name-given-names">A.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Iwata</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Vigor: variational bayesian inference for genome-wide regression.</span> <abbr class="cit-jnl-abbrev">J. Open Res. Softw.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">11</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BOpen%2BRes.%2BSoftw.%26rft.volume%253D4%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3855.59"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ou</span>,  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. J.</span> <span class="cit-name-surname">Tempelman</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. P.</span> <span class="cit-name-surname">Steibel</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. W.</span> <span class="cit-name-surname">Ernst</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. O.</span> <span class="cit-name-surname">Bates</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genomic prediction accounting for residual heteroskedasticity. G3: Genes, Genomes</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">13</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D6%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3855.60" data-doi="10.1198/016214508000000337"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Park</span>,  <span class="cit-name-given-names">T.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Casella</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">The bayesian lasso.</span> <abbr class="cit-jnl-abbrev">J. Am. Stat. Assoc.</abbr> <span class="cit-vol">103</span>: <span class="cit-fpage">681</span>–<span class="cit-lpage">686</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1198/016214508000000337</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAm.%2BStat.%2BAssoc.%26rft.volume%253D103%26rft.spage%253D681%26rft_id%253Dinfo%253Adoi%252F10.1198%252F016214508000000337%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1198/016214508000000337&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000257897500025&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3855.61" data-doi="10.1093/biomet/58.3.545"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Patterson</span>,  <span class="cit-name-given-names">H. D.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Thompson</span></span></li></ol><cite>, <span class="cit-pub-date">1971</span> <span class="cit-article-title">Recovery of inter-block information when block sizes are unequal.</span> <abbr class="cit-jnl-abbrev">Biometrika</abbr> <span class="cit-vol">58</span>: <span class="cit-fpage">545</span>–<span class="cit-lpage">554</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/biomet/58.3.545</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiometrika%26rft.stitle%253DBiometrika%26rft.aulast%253DPATTERSON%26rft.auinit1%253DH.%2BD.%26rft.volume%253D58%26rft.issue%253D3%26rft.spage%253D545%26rft.epage%253D554%26rft.atitle%253DRecovery%2Bof%2Binter-block%2Binformation%2Bwhen%2Bblock%2Bsizes%2Bare%2Bunequal%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbiomet%252F58.3.545%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/biomet/58.3.545&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1971L054700013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3855.62" data-doi="10.1534/genetics.114.164442"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pèrez</span>,  <span class="cit-name-given-names">P.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">de Los Campos</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genome-wide regression &amp; prediction with the bglr statistical package.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">198</span>: <span class="cit-fpage">483</span>–<span class="cit-lpage">495</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.114.164442</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.164442%26rft_id%253Dinfo%253Apmid%252F25009151%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk4LzIvNDgzIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMS8zODU1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3855.63"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pérez-Rodríguez</span>,  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Gianola</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">González-Camacho</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Crossa</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Manès</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Comparison between linear and non-parametric regression models for genome-enabled prediction in wheat. G3: Genes, Genomes</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">1595</span>–<span class="cit-lpage">1605</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D2%26rft.spage%253D1595%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3855.64" data-doi="10.1002/bimj.201100219"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Piepho</span>,  <span class="cit-name-given-names">H.-P.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Moehring</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Schulz-Streeck</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. O.</span> <span class="cit-name-surname">Ogutu</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A stage-wise approach for the analysis of multi-environment trials.</span> <abbr class="cit-jnl-abbrev">Biom. J.</abbr> <span class="cit-vol">54</span>: <span class="cit-fpage">844</span>–<span class="cit-lpage">860</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/bimj.201100219</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DBiom%2BJ%26rft.aulast%253DPiepho%26rft.auinit1%253DH.%2BP.%26rft.volume%253D54%26rft.issue%253D6%26rft.spage%253D844%26rft.epage%253D860%26rft.atitle%253DA%2Bstage-wise%2Bapproach%2Bfor%2Bthe%2Banalysis%2Bof%2Bmulti-environment%2Btrials.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fbimj.201100219%26rft_id%253Dinfo%253Apmid%252F23007738%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/bimj.201100219&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23007738&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3855.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3855.65"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ridgeway</span>,  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Generalized boosted models: A guide to the gbm package.</span> <abbr class="cit-jnl-abbrev">Update</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">2007</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DUpdate%26rft.volume%253D1%26rft.spage%253D2007%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3855.66" data-doi="10.3168/jds.S0022-0302(86)80743-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schaeffer</span>,  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">1986</span> <span class="cit-article-title">Pseudo expectation approach to variance component estimation.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">69</span>: <span class="cit-fpage">2884</span>–<span class="cit-lpage">2889</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3168/jds.S0022-0302(86)80743-3</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DSchaeffer%26rft.auinit1%253DL.%2BR.%26rft.volume%253D69%26rft.issue%253D11%26rft.spage%253D2884%26rft.epage%253D2889%26rft.atitle%253DPseudo%2BExpectation%2BApproach%2Bto%2BVariance%2BComponent%2BEstimation%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.S0022-0302%252886%252980743-3%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.S0022-0302(86)80743-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3855.67" data-doi="10.1007/s00122-012-1960-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schulz-Streeck</span>,  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. O.</span> <span class="cit-name-surname">Ogutu</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.-P.</span> <span class="cit-name-surname">Piepho</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Comparisons of single-stage and two-stage approaches to genomic selection.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">126</span>: <span class="cit-fpage">69</span>–<span class="cit-lpage">82</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-012-1960-1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D126%26rft.spage%253D69%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-012-1960-1%26rft_id%253Dinfo%253Apmid%252F22903736%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-012-1960-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22903736&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3855.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-book" id="cit-9.11.3855.68" data-doi="10.1002/9780470316856.ch10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Searle</span>,  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Casella</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. E.</span> <span class="cit-name-surname">McCulloch</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">Prediction of random variables</span>, pp. <span class="cit-fpage">367</span>–<span class="cit-lpage">377</span> in <span class="cit-source">Variance Components</span>, <span class="cit-publ-name">John Wiley and Sons, Inc.</span>, <span class="cit-publ-loc">Hoboken, NJ</span><span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/9780470316856.ch10</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVariance%2BComponents%26rft_id%253Dinfo%253Adoi%252F10.1002%252F9780470316856.ch10%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/9780470316856.ch10&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3855.69" data-doi="10.2135/cropsci2009.02.0078"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sebastian</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Streit</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Stephens</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Thompson</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Hedges</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Context-specific marker-assisted selection for improved grain yield in elite soybean populations.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">50</span>: <span class="cit-fpage">1196</span>–<span class="cit-lpage">1206</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2009.02.0078</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DSebastian%26rft.auinit1%253DS.%2BA.%26rft.volume%253D50%26rft.issue%253D4%26rft.spage%253D1196%26rft.epage%253D1206%26rft.atitle%253DContext-Specific%2BMarker-Assisted%2BSelection%2Bfor%2BImproved%2BGrain%2BYield%2Bin%2BElite%2BSoybean%2BPopulations%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2009.02.0078%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2009.02.0078&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3855.70" data-doi="10.1111/1467-842X.00163"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Cullis</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Gilmour</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Applications: the analysis of crop variety evaluation data in australia.</span> <abbr class="cit-jnl-abbrev">Aust. N. Z. J. Stat.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">129</span>–<span class="cit-lpage">145</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/1467-842X.00163</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAust.%2BN.%2BZ.%2BJ.%2BStat.%26rft.volume%253D43%26rft.spage%253D129%26rft_id%253Dinfo%253Adoi%252F10.1111%252F1467-842X.00163%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/1467-842X.00163&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3855.71" data-doi="10.1371/journal.pone.0049157"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>,  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Qu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. J.</span> <span class="cit-name-surname">Garrick</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. C.</span> <span class="cit-name-surname">Dekkers</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. L.</span> <span class="cit-name-surname">Fernando</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A fast em algorithm for bayesa-like prediction of genomic breeding values.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e49157</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0049157</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DSun%26rft.auinit1%253DX.%26rft.volume%253D7%26rft.issue%253D11%26rft.spage%253De49157%26rft.epage%253De49157%26rft.atitle%253DA%2BFast%2BEM%2BAlgorithm%2Bfor%2BBayesA-Like%2BPrediction%2Bof%2BGenomic%2BBreeding%2BValues.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0049157%26rft_id%253Dinfo%253Apmid%252F23152868%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0049157&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23152868&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3855.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference  in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3855.72" data-doi="10.1186/s12711-017-0310-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Taskinen</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. A.</span> <span class="cit-name-surname">Mäntysaari</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Strandèn</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Single-step snp-blup with on-the-fly imputed genotypes and residual polygenic effects.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">36</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12711-017-0310-9</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D49%26rft.spage%253D36%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12711-017-0310-9%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12711-017-0310-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference  in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3855.73"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>,  <span class="cit-name-given-names">E.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Shaw</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">Estimating polygenic models for multivariate data on large pedigrees.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">131</span>: <span class="cit-fpage">971</span>–<span class="cit-lpage">978</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DThompson%26rft.auinit1%253DE.%2BA.%26rft.volume%253D131%26rft.issue%253D4%26rft.spage%253D971%26rft.epage%253D978%26rft.atitle%253DEstimating%2Bpolygenic%2Bmodels%2Bfor%2Bmultivariate%2Bdata%2Bon%2Blarge%2Bpedigrees.%26rft_id%253Dinfo%253Apmid%252F1516823%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTMxLzQvOTcxIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMS8zODU1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference  in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3855.74" data-doi="10.3168/jds.2007-0980"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">VanRaden</span>,  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Efficient methods to compute genomic predictions.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">4414</span>–<span class="cit-lpage">4423</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3168/jds.2007-0980</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DVanRaden%26rft.auinit1%253DP.%2BM.%26rft.volume%253D91%26rft.issue%253D11%26rft.spage%253D4414%26rft.epage%253D4423%26rft.atitle%253DEfficient%2BMethods%2Bto%2BCompute%2BGenomic%2BPredictions%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2007-0980%26rft_id%253Dinfo%253Apmid%252F18946147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2007-0980&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18946147&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3855.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000260277200035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference  in text" id="ref-75">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3855.75" data-doi="10.1186/s12711-014-0082-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.-P. P.</span> <span class="cit-name-surname">Chen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. E.</span> <span class="cit-name-surname">Goddard</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. H.</span> <span class="cit-name-surname">Meuwissen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. E.</span> <span class="cit-name-surname">Kemper</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">A computationally efficient algorithm for genomic prediction using a bayesian model.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">34</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12711-014-0082-4</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D47%26rft.spage%253D34%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12711-014-0082-4%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12711-014-0082-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-76-1" title="View reference  in text" id="ref-76">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3855.76" data-doi="10.1534/genetics.113.150078"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wimmer</span>,  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Lehermeier</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Albrecht</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.-J.</span> <span class="cit-name-surname">Auinger</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Wang</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genome-wide prediction of traits with different genetic architecture through efficient variable selection.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">195</span>: <span class="cit-fpage">573</span>–<span class="cit-lpage">587</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.113.150078</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.150078%26rft_id%253Dinfo%253Apmid%252F23934883%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk1LzIvNTczIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMS8zODU1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-77-1" title="View reference  in text" id="ref-77">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3855.77" data-doi="10.1186/s40104-016-0066-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wolc</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Arango</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Settar</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. E.</span> <span class="cit-name-surname">Fulton</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N. P.</span> <span class="cit-name-surname">O’Sullivan</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Mixture models detect large effect qtl better than gblup and result in more accurate and persistent predictions.</span> <abbr class="cit-jnl-abbrev">J. Anim. Sci. Biotechnol.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">7</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s40104-016-0066-z</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAnim.%2BSci.%2BBiotechnol.%26rft.volume%253D7%26rft.spage%253D7%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs40104-016-0066-z%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s40104-016-0066-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-78-1" title="View reference  in text" id="ref-78">↵</a><div class="cit ref-cit ref-web" id="cit-9.11.3855.78"><div class="cit-metadata unstructured">Wright, M. N., and A. Ziegler, 2015 Ranger: a fast implementation of random forests for high dimensional data in c++ and r. arXiv preprint arXiv:1508.04409.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-79-1" title="View reference  in text" id="ref-79">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3855.79"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xavier</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Jarquin</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Howard</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Ramasubramanian</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. E.</span> <span class="cit-name-surname">Specht</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Genome-wide analysis of grain yield stability and environmental interactions in a multiparental soybean population. G3: Genes, Genomes</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">519</span>–<span class="cit-lpage">529</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D8%26rft.spage%253D519%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-80-1" title="View reference  in text" id="ref-80">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3855.80"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xavier</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W. M.</span> <span class="cit-name-surname">Muir</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. M.</span> <span class="cit-name-surname">Rainey</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Assessing predictive properties of genome-wide selection in soybeans. G3: Genes, Genomes</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">2611</span>–<span class="cit-lpage">2616</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D6%26rft.spage%253D2611%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-81-1" title="View reference  in text" id="ref-81">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3855.81" data-doi="10.1093/bioinformatics/btv448"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xavier</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Xu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W. M.</span> <span class="cit-name-surname">Muir</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. M.</span> <span class="cit-name-surname">Rainey</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Nam: association studies in multiple populations.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">3862</span>–<span class="cit-lpage">3864</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtv448%26rft_id%253Dinfo%253Apmid%252F26243017%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btv448&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26243017&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3855.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-82-1" title="View reference  in text" id="ref-82">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3855.82" data-doi="10.1111/j.1541-0420.2006.00711.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>,  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">An empirical bayes method for estimating epistatic effects of quantitative trait loci.</span> <abbr class="cit-jnl-abbrev">Biometrics</abbr> <span class="cit-vol">63</span>: <span class="cit-fpage">513</span>–<span class="cit-lpage">521</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1541-0420.2006.00711.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiometrics%26rft.stitle%253DBiometrics%26rft.aulast%253DXu%26rft.auinit1%253DS.%26rft.volume%253D63%26rft.issue%253D2%26rft.spage%253D513%26rft.epage%253D521%26rft.atitle%253DAn%2Bempirical%2BBayes%2Bmethod%2Bfor%2Bestimating%2Bepistatic%2Beffects%2Bof%2Bquantitative%2Btrait%2Bloci.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1541-0420.2006.00711.x%26rft_id%253Dinfo%253Apmid%252F17688503%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1541-0420.2006.00711.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17688503&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3855.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000246843900023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-83-1" title="View reference  in text" id="ref-83">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3855.83" data-doi="10.1038/hdy.2009.180"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>,  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">An expectation–maximization algorithm for the lasso estimation of quantitative trait locus effects.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">105</span>: <span class="cit-fpage">483</span>–<span class="cit-lpage">494</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/hdy.2009.180</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.stitle%253DHeredity%26rft.aulast%253DXu%26rft.auinit1%253DS.%26rft.volume%253D105%26rft.issue%253D5%26rft.spage%253D483%26rft.epage%253D494%26rft.atitle%253DAn%2Bexpectation-maximization%2Balgorithm%2Bfor%2Bthe%2BLasso%2Bestimation%2Bof%2Bquantitative%2Btrait%2Blocus%2Beffects.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2009.180%26rft_id%253Dinfo%253Apmid%252F20051978%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2009.180&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20051978&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3855.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000283168600009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-84-1" title="View reference  in text" id="ref-84">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3855.84" data-doi="10.1534/genetics.113.157032"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>,  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Mapping quantitative trait loci by controlling polygenic background effects.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">195</span>: <span class="cit-fpage">1209</span>–<span class="cit-lpage">1222</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.113.157032</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.157032%26rft_id%253Dinfo%253Apmid%252F24077303%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5NS80LzEyMDkiO3M6NDoiYXRvbSI7czoxOToiL2dnZy85LzExLzM4NTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-85-1" title="View reference  in text" id="ref-85">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3855.85" data-doi="10.1038/ng.2876"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>,  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N. A.</span> <span class="cit-name-surname">Zaitlen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. E.</span> <span class="cit-name-surname">Goddard</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. M.</span> <span class="cit-name-surname">Visscher</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. L.</span> <span class="cit-name-surname">Price</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Advantages and pitfalls in the application of mixed-model association methods.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">100</span>–<span class="cit-lpage">106</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.2876</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D46%26rft.spage%253D100%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2876%26rft_id%253Dinfo%253Apmid%252F24473328%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2876&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24473328&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3855.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-86-1" title="View reference  in text" id="ref-86">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3855.86" data-doi="10.1534/genetics.107.085589"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yi</span>,  <span class="cit-name-given-names">N.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Xu</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Bayesian lasso for quantitative trait loci mapping.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">179</span>: <span class="cit-fpage">1045</span>–<span class="cit-lpage">1055</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.107.085589</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.085589%26rft_id%253Dinfo%253Apmid%252F18505874%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3OS8yLzEwNDUiO3M6NDoiYXRvbSI7czoxOToiL2dnZy85LzExLzM4NTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-87-1" title="View reference  in text" id="ref-87">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3855.87" data-doi="10.1534/genetics.104.035857"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zeng</span>,  <span class="cit-name-given-names">Z.-B.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Wang</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Zou</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Modeling quantitative trait loci and interpretation of models.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">169</span>: <span class="cit-fpage">1711</span>–<span class="cit-lpage">1725</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.104.035857</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.104.035857%26rft_id%253Dinfo%253Apmid%252F15654105%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2OS8zLzE3MTEiO3M6NDoiYXRvbSI7czoxOToiL2dnZy85LzExLzM4NTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-88-1" title="View reference  in text" id="ref-88">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3855.88" data-doi="10.1038/ng.546"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Ersoz</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.-Q.</span> <span class="cit-name-surname">Lai</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. J.</span> <span class="cit-name-surname">Todhunter</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H. K.</span> <span class="cit-name-surname">Tiwari</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Mixed linear model approach adapted for genome-wide association studies.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">355</span>–<span class="cit-lpage">360</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.546</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DZhang%26rft.auinit1%253DZ.%26rft.volume%253D42%26rft.issue%253D4%26rft.spage%253D355%26rft.epage%253D360%26rft.atitle%253DMixed%2Blinear%2Bmodel%2Bapproach%2Badapted%2Bfor%2Bgenome-wide%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.546%26rft_id%253Dinfo%253Apmid%252F20208535%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.546&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20208535&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3855.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000276150500017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-89-1" title="View reference  in text" id="ref-89">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3855.89" data-doi="10.1038/s41437-017-0010-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>,  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Mrode</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Zhang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Q.</span> <span class="cit-name-surname">Zhang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Li</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Factors affecting gebv accuracy with single-step bayesian models.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">120</span>: <span class="cit-fpage">100</span>–<span class="cit-lpage">109</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/s41437-017-0010-9</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D120%26rft.spage%253D100%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41437-017-0010-9%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41437-017-0010-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/9/11/3855.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/11/3843" title="Genome-Wide Analysis of Cell Cycle-Regulating Genes in the Symbiotic Dinoflagellate Breviolum minutum" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/11/3867" title="Mapping of the Sex Determining Region on Linkage Group 12 of Guppy (Poecilia reticulata)" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/9/11"><span>Volume 9 Issue 11, November 2019</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/9/11.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/9/11.cover-source.jpg?itok=DjqkLGk9" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 9 (11)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/11/3855.figures-only" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-xopFLhIHEoYg6ypgVwyqF2z6DGvAMn6xaMfWjUuVOn0" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/genomic-prediction" class="" data-icon-position="" data-hide-link-title="0">Genomic Prediction</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/9/11/3855" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/9/11/3855" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/9/11/3855" class="active" data-icon-position="" data-hide-link-title="0">Efficient Estimation of Marker Effects in Plant Breeding</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/494613" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-3nN3PcIrjGzXfOiCb2f-2NmYIm-keASUYPa9wLVy2Lg" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/11/3855.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/9/11/3855" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-KT2cxLjHe67p1A-UoX8pZpWRffb8ykLmXEOEt0EDXQQ" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/9/11/3855" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/11/3855.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494613" id="node494613--2" data-pisa="ggg;9/11/3855" data-pisa-master="ggg;g3.119.400728" data-apath="/ggg/9/11/3855.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Efficient Estimation of Marker Effects in Plant Breeding</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-5034-9954" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Alencar Xavier</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3855-3866;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400728 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494613--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Alencar Xavier </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Corteva Agrisciences, 8305 NW 62nd Ave. Johnston IA, and</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Purdue University, 915 W State St. West Lafayette IN</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alencar%2BXavier%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Xavier%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAlencar%2BXavier%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-5034-9954" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Alencar Xavier</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:alencar.xavier@corteva.com" class="" data-icon-position="" data-hide-link-title="0">alencar.xavier@corteva.com</a>
<a href="mailto:xaviera@purdue.edu" class="" data-icon-position="" data-hide-link-title="0">xaviera@purdue.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/9/11/3855" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/9/11/3855&amp;title=Efficient%20Estimation%20of%20Marker%20Effects%20in%20Plant%20Breeding" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/9/11/3855&amp;title=Efficient%20Estimation%20of%20Marker%20Effects%20in%20Plant%20Breeding" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/9/11/3855&amp;title=Efficient%20Estimation%20of%20Marker%20Effects%20in%20Plant%20Breeding" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/9/11/3855&amp;text=Efficient%20Estimation%20of%20Marker%20Effects%20in%20Plant%20Breeding" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/9/11/3855&amp;title=Efficient%20Estimation%20of%20Marker%20Effects%20in%20Plant%20Breeding" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/9/11/3855&amp;t=Efficient%20Estimation%20of%20Marker%20Effects%20in%20Plant%20Breeding" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/9/11/3855&amp;title=Efficient%20Estimation%20of%20Marker%20Effects%20in%20Plant%20Breeding" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/9/11/3855&amp;title=Efficient%20Estimation%20of%20Marker%20Effects%20in%20Plant%20Breeding" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/494613/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494613" id="node494613--3" data-pisa="ggg;9/11/3855" data-pisa-master="ggg;g3.119.400728" data-apath="/ggg/9/11/3855.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Efficient Estimation of Marker Effects in Plant Breeding</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-5034-9954" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Alencar Xavier</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3855-3866;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400728 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494613--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Alencar Xavier </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Corteva Agrisciences, 8305 NW 62nd Ave. Johnston IA, and</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Purdue University, 915 W State St. West Lafayette IN</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alencar%2BXavier%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Xavier%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAlencar%2BXavier%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-5034-9954" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Alencar Xavier</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:alencar.xavier@corteva.com" class="" data-icon-position="" data-hide-link-title="0">alencar.xavier@corteva.com</a>
<a href="mailto:xaviera@purdue.edu" class="" data-icon-position="" data-hide-link-title="0">xaviera@purdue.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=3855&amp;rft.epage=3866&amp;rft.atitle=Efficient%20Estimation%20of%20Marker%20Effects%20in%20Plant%20Breeding&amp;rft.volume=9&amp;rft.issue=11&amp;rft.date=2019-11-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Xavier%2C+Alencar"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/494613/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/494613/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/494613/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/494613/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/494613/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/494613/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/494613/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/494613/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/494613/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/494613/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/494613/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/494613/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495864" id="node495864" data-pisa="ggg;10/3/1113" data-pisa-master="ggg;g3.119.400968" data-apath="/ggg/10/3/1113.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1113" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Multi-trait Genomic Prediction Model Increased the Predictive Ability for Agronomic and Malting Quality Traits in Barley (<em>Hordeum vulgare</em> L.)</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495543" id="node495543" data-pisa="ggg;10/2/769" data-pisa-master="ggg;g3.119.400759" data-apath="/ggg/10/2/769.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/2/769" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Novel Bayesian Networks for Genomic Prediction of Developmental Traits in Biomass Sorghum</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495557" id="node495557" data-pisa="ggg;10/2/539" data-pisa-master="ggg;g3.119.400866" data-apath="/ggg/10/2/539.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/2/539" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Optimized Genetic Testing for Polledness in Multiple Breeds of Cattle</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Genomic%20Prediction" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Genomic Prediction</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/9/11/3855" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-6" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-17" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-21" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#app-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Appendix</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/9/11/3855.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/9/11/3855.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/9/11/3855.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__c-64NG7CB_fwZbyex-B49GkX1AvVESfu7M9yxTNDG_o__Uvi1liJMEKbHtHHv_1fo_FCLvimvbq9jMwPcCPR364Q__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
