<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/9/11/3537" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/9/11/3537.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/9/11/3537.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/9/11/3537.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/9/11/3537.atom" />
<meta name="HW.pisa" content="ggg;9/11/3537" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Construction of High-Resolution RAD-Seq Based Linkage Map, Anchoring Reference Genome, and QTL Mapping of the Sex Chromosome in the Marine Medaka Oryzias melastigma" />
<meta name="DC.Identifier" content="10.1534/g3.119.400708" />
<meta name="DC.Date" content="2019-11-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2019 Lee et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Medaka ( Oryzias sp.) is an important fish species in ecotoxicology and considered as a model species due to its biological features including small body size and short generation time. Since Japanese medaka Oryzias latipes is a freshwater species with access to an excellent genome resource, the marine medaka Oryzias melastigma is also applicable for the marine ecotoxicology. In genome era, a high-density genetic linkage map is a very useful resource in genomic research, providing a means for comparative genomic analysis and verification of de novo genome assembly. In this study, we developed a high-density genetic linkage map for O. melastigma using restriction-site associated DNA sequencing (RAD-seq). The genetic map consisted of 24 linkage groups with 2,481 single nucleotide polymorphism (SNP) markers. The total map length was 1,784 cM with an average marker space of 0.72 cM. The genetic map was integrated with the reference-assisted chromosome assembly (RACA) of O. melastigma , which anchored 90.7% of the assembled sequence onto the linkage map. The values of complete Benchmarking Universal Single-Copy Orthologs were similar to RACA assembly but N50 (23.74 Mb; total genome length 779.4 Mb; gap 5.29%) increased to 29.99 Mb (total genome length 778.7 Mb; gap 5.2%). Using MapQTL analysis with SNP markers, we identified a major quantitative trait locus for sex traits on the Om10. The integration of the genetic map with the reference genome of marine medaka will serve as a good resource for studies in molecular toxicology, genomics, CRISPR/Cas9, and epigenetics." />
<meta name="DC.Contributor" content="Bo-Young Lee" />
<meta name="DC.Contributor" content="Min-Sub Kim" />
<meta name="DC.Contributor" content="Beom-Soon Choi" />
<meta name="DC.Contributor" content="Atsushi J. Nagano" />
<meta name="DC.Contributor" content="Doris Wai Ting Au" />
<meta name="DC.Contributor" content="Rudolf Shiu Sun Wu" />
<meta name="DC.Contributor" content="Yusuke Takehana" />
<meta name="DC.Contributor" content="Jae-Seong Lee" />
<meta name="article:published_time" content="2019-11-01" />
<meta name="article:section" content="Genome Report" />
<meta name="citation_title" content="Construction of High-Resolution RAD-Seq Based Linkage Map, Anchoring Reference Genome, and QTL Mapping of the Sex Chromosome in the Marine Medaka Oryzias melastigma" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Medaka (&lt;i&gt;Oryzias&lt;/i&gt; sp.) is an important fish species in ecotoxicology and considered as a model species due to its biological features including small body size and short generation time. Since Japanese medaka &lt;i&gt;Oryzias latipes&lt;/i&gt; is a freshwater species with access to an excellent genome resource, the marine medaka &lt;i&gt;Oryzias melastigma&lt;/i&gt; is also applicable for the marine ecotoxicology. In genome era, a high-density genetic linkage map is a very useful resource in genomic research, providing a means for comparative genomic analysis and verification of &lt;i&gt;de novo&lt;/i&gt; genome assembly. In this study, we developed a high-density genetic linkage map for &lt;i&gt;O. melastigma&lt;/i&gt; using restriction-site associated DNA sequencing (RAD-seq). The genetic map consisted of 24 linkage groups with 2,481 single nucleotide polymorphism (SNP) markers. The total map length was 1,784 cM with an average marker space of 0.72 cM. The genetic map was integrated with the reference-assisted chromosome assembly (RACA) of &lt;i&gt;O. melastigma&lt;/i&gt;, which anchored 90.7% of the assembled sequence onto the linkage map. The values of complete Benchmarking Universal Single-Copy Orthologs were similar to RACA assembly but N50 (23.74 Mb; total genome length 779.4 Mb; gap 5.29%) increased to 29.99 Mb (total genome length 778.7 Mb; gap 5.2%). Using MapQTL analysis with SNP markers, we identified a major quantitative trait locus for sex traits on the Om10. The integration of the genetic map with the reference genome of marine medaka will serve as a good resource for studies in molecular toxicology, genomics, CRISPR/Cas9, and epigenetics.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2019/11/01" />
<meta name="citation_mjid" content="ggg;9/11/3537" />
<meta name="citation_id" content="9/11/3537" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/9/11/3537" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/9/11/3537.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/9/11/3537.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/9/11/3537.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.119.400708" />
<meta name="citation_pmid" content="31530635" />
<meta name="citation_volume" content="9" />
<meta name="citation_issue" content="11" />
<meta name="citation_num_pages" content="9" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Genome Report" />
<meta name="citation_firstpage" content="3537" />
<meta name="citation_lastpage" content="3545" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Bo-Young Lee" />
<meta name="citation_author_institution" content="Department of Biological Science, College of Science, Sungkyunkwan University, Suwon 16419, South Korea," />
<meta name="citation_author" content="Min-Sub Kim" />
<meta name="citation_author_institution" content="Department of Biological Science, College of Science, Sungkyunkwan University, Suwon 16419, South Korea," />
<meta name="citation_author" content="Beom-Soon Choi" />
<meta name="citation_author_institution" content="Phyzen Genomics Institute, Seongnam 13558, South Korea," />
<meta name="citation_author" content="Atsushi J. Nagano" />
<meta name="citation_author_institution" content="Faculty of Agriculture, Ryukoku University, Kyoto 612-8577, Japan," />
<meta name="citation_author" content="Doris Wai Ting Au" />
<meta name="citation_author_institution" content="Department of Chemistry and" />
<meta name="citation_author_institution" content="State Key Laboratory of Marine Pollution, The City University of Hong Kong, Hong Kong SAR, China," />
<meta name="citation_author" content="Rudolf Shiu Sun Wu" />
<meta name="citation_author_institution" content="Department of Science and Environmental Studies, The Education University of Hong Kong, Hong Kong SAR, China, and" />
<meta name="citation_author" content="Yusuke Takehana" />
<meta name="citation_author_institution" content="Department of Animal Bioscience, Nagahama Institute of Bio-Science and Technology, Nagahama, Shiga 526-0829, Japan" />
<meta name="citation_author_email" content="jslee2@skku.edu" />
<meta name="citation_author_email" content="y_takehana@nagahama-i-bio.ac.jp" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-9965-8499" />
<meta name="citation_author" content="Jae-Seong Lee" />
<meta name="citation_author_institution" content="Department of Biological Science, College of Science, Sungkyunkwan University, Suwon 16419, South Korea," />
<meta name="citation_author_email" content="jslee2@skku.edu" />
<meta name="citation_author_email" content="y_takehana@nagahama-i-bio.ac.jp" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-0944-5172" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. Amores;citation_author=J. Catchen;citation_author=A. Ferrara;citation_author=Q. Fontenot;citation_author=J. H. Postlethwait;citation_title=Genome evolution and meiotic maps by massively parallel DNA sequencing: Spotted Gar, an outgroup for the teleost genome duplication.;citation_volume=188;citation_year=2011;citation_pmid=21828280;citation_doi=10.1534/genetics.111.127324" />
<meta name="citation_reference" content="citation_journal_title=Rev. Fish. Sci.;citation_author=O. Arellano-Aguilar;citation_author=R. M. Montoya;citation_author=C. M. Garcia;citation_title=Endogenous functions and expression of cytochrome P450 enzymes in teleost fish: a review.;citation_pages=541-556;citation_volume=17;citation_year=2009;citation_doi=10.1080/10641260903243487" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=N. A. Baird;citation_author=P. D. Etter;citation_author=T. S. Atwood;citation_author=M. C. Currey;citation_author=A. L. Shiver;citation_title=Rapid SNP discovery and genetic mapping using sequenced RAD markers.;citation_pages=e3376-e3376;citation_volume=3;citation_year=2008;citation_issue=10;citation_pmid=18852878;citation_doi=10.1371/journal.pone.0003376" />
<meta name="citation_reference" content="citation_author=A. M. Bolger;citation_author=M. Lohse;citation_author=B. Usadel;citation_year=2014;citation_pmid=24695404;citation_doi=10.1093/bioinformatics/btu170" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=J. M. Catchen;citation_author=A. Amores;citation_author=P. Hohenlohe;citation_author=W. Cresko;citation_author=J. H. Postlethwait;citation_title=Stacks: Building and genotyping loci de novo from short-read sequences.;citation_pages=171-182;citation_volume=1;citation_year=2011;citation_doi=doi:10.1534/g3.111.000240" />
<meta name="citation_reference" content="citation_journal_title=Marine Pollution Bulletin;citation_journal_abbrev=Marine Pollution Bulletin;citation_author=X. Chen;citation_author=L. Li;citation_author=J. Cheng;citation_author=L. L. Chan;citation_author=D. Z. Wang;citation_title=Molecular staging of marine medaka: a model organism for marine ecotoxicity study.;citation_pages=309-317;citation_volume=63;citation_year=2011;citation_issue=5-12;citation_pmid=21708389;citation_doi=10.1016/j.marpolbul.2011.03.042" />
<meta name="citation_reference" content="citation_journal_title=Brief. Funct. Genomics;citation_author=J. W. Davey;citation_author=M. L. Blaxter;citation_title=RADseq: next-generation population genetics.;citation_volume=9;citation_year=2010;citation_pmid=21266344;citation_doi=10.1093/bfgp/elq031" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=J. W. Davey;citation_author=P. A. Hohenlohe;citation_author=P. D. Etter;citation_author=J. Q. Boone;citation_author=J. M. Catchen;citation_title=Genome-wide genetic marker discovery and genotyping using next-generation sequencing.;citation_pages=499-510;citation_volume=12;citation_year=2011;citation_issue=7;citation_pmid=21681211;citation_doi=10.1038/nrg3012" />
<meta name="citation_reference" content="citation_journal_title=BioMed Res. Int.;citation_author=S. Dong;citation_author=M. Kang;citation_author=X. Wu;citation_author=T. Ye;citation_title=Development of a promising fish model (Oryzias melastigma) for assessing multiple responses to stresses in the marine environment.;citation_pages=563131;citation_volume=2014;citation_year=2014;citation_doi=10.1155/2014/563131" />
<meta name="citation_reference" content="citation_journal_title=BMC Genet.;citation_author=M. Dukić;citation_author=D. Berner;citation_author=M. Roesti;citation_author=C. R. Haag;citation_author=D. Ebert;citation_title=A high-density genetic map reveals variation in recombination rate across the genome of Daphina magna.;citation_pages=137;citation_volume=17;citation_year=2016;citation_doi=10.1186/s12863-016-0445-7" />
<meta name="citation_reference" content="citation_journal_title=Methods in molecular biology (Clifton, N.J.);citation_journal_abbrev=Methods Mol Biol;citation_author=P. D. Etter;citation_author=S. Bassham;citation_author=P. A. Hohenlohe;citation_author=E. A. Johnson;citation_author=W. A. Cresko;citation_title=SNP discovery and genotyping for evolutionary genetics using RAD sequencing.;citation_pages=157-178;citation_volume=772;citation_year=2011;citation_pmid=22065437;citation_doi=10.1007/978-1-61779-228-1_9" />
<meta name="citation_reference" content="citation_journal_title=Chromosoma;citation_author=X. Feng;citation_author=X. Wang;citation_author=X. Yu;citation_author=X. Zhang;citation_author=C. Lu;citation_title=Microsatellite-centromere mapping in common carp through half-tetrad analysis in diploid meiogynogenetic families.;citation_pages=67-79;citation_volume=124;citation_year=2015;citation_doi=10.1007/s00412-014-0485-6" />
<meta name="citation_reference" content="citation_journal_title=Aquaculture;citation_author=W. J. Guo;citation_author=J. G. Tong;citation_author=X. M. Yu;citation_author=C. K. Zhu;citation_author=X. Feng;citation_title=A second generation genetic linkage map for silver carp (Hypophthalmichehys molitrix) using microsatellite markers.;citation_pages=97-106;citation_volume=412–413;citation_year=2013;citation_doi=10.1016/j.aquaculture.2013.06.027" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=C. Holt;citation_author=M. Yandell;citation_title=MAKER2: an annotation pipeline and genome-database management tool for second-generation genome projects.;citation_pages=491-491;citation_volume=12;citation_year=2011;citation_pmid=22192575;citation_doi=10.1186/1471-2105-12-491" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=S. Huang;citation_author=M. Kang;citation_author=A. Xu;citation_title=HaploMerger2: rebuilding both haploid sub-assemblies from high-heterozygosity diploid genome assembly.;citation_pages=2577-2579;citation_volume=33;citation_year=2017;citation_pmid=28407147;citation_doi=10.1093/bioinformatics/btx220" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=R. Kajitani;citation_author=K. Toshioto;citation_author=H. Noguchi;citation_author=A. Toyoda;citation_author=Y. Ogura;citation_title=Efficient de novo assembly of highly heterozygous genomes from whole-genome shotgun short reads.;citation_volume=24;citation_year=2014;citation_pmid=24755901;citation_doi=10.1101/gr.170720.113" />
<meta name="citation_reference" content="citation_author=A. Kanamori;citation_author=Y. Sugita;citation_author=Y. Yuasa;citation_author=T. Suzuki;citation_author=K. Kawamuyra;citation_year=2016;citation_pmid=26865699;citation_doi=10.1534/g3.115.022699" />
<meta name="citation_reference" content="citation_journal_title=Mar. Environ. Res.;citation_author=B.-M. Kim;citation_author=J. Kim;citation_author=I.-Y. Choi;citation_author=S. Raisuddin;citation_author=D. W. T. Au;citation_title=Omics of the marine medaka (Oryzias melastigma) and its relevance to marine environmental research.;citation_pages=141-152;citation_volume=113;citation_year=2016;citation_doi=10.1016/j.marenvres.2015.12.004" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=J. Kim;citation_author=D. M. Larkin;citation_author=Q. Cai;citation_author=Y. Asan;citation_author=Y. Zhang;citation_title=Reference-assisted chromosome assembly.;citation_volume=110;citation_year=2013;citation_pmid=23307812;citation_doi=10.1073/pnas.1220349110" />
<meta name="citation_reference" content="citation_journal_title=Current biology :  CB;citation_journal_abbrev=Curr Biol;citation_author=M. Kondo;citation_author=I. Nanda;citation_author=U. Hornung;citation_author=S. Asakawa;citation_author=N. Shimizu;citation_title=Absence of the candidate male sex-determining gene dmrt1b(Y) of medaka from other fish species.;citation_pages=416-420;citation_volume=13;citation_year=2003;citation_issue=5;citation_pmid=12620191;citation_doi=10.1016/S0960-9822(03)00125-8" />
<meta name="citation_reference" content="citation_journal_title=Aquatic Toxicology;citation_journal_abbrev=Aquatic Toxicology;citation_author=R. Y. C. Kong;citation_author=J. P. Giesy;citation_author=R. S. S. Wu;citation_author=E. X. Chen;citation_author=M. W. Chiang;citation_title=Development of a marine fish model for studying in vivo molecular responses in ecotoxicology.;citation_pages=131-141;citation_volume=86;citation_year=2008;citation_issue=2;citation_pmid=18055030;citation_doi=10.1016/j.aquatox.2007.10.011" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=B.-Y. Lee;citation_author=W.-J. Lee;citation_author=J. T. Streelman;citation_author=K. L. Carleton;citation_author=A. E. Howe;citation_title=A second-generation genetic linkage map of Tilapia (Oreochromis spp.).;citation_volume=170;citation_year=2005;citation_pmid=15716505;citation_doi=10.1534/genetics.104.035022" />
<meta name="citation_reference" content="citation_journal_title=Sci. Rep.;citation_author=J.-S. Rhee;citation_author=B.-S. Choi;citation_author=J. Kim;citation_author=B.-M. Kim;citation_author=Y.-M. Lee;citation_title=Diversity, distribution, and significance of transposable elements in the genome of the only selfing hermaphrodictic vertebrate Kryptolebias marmoratus.;citation_pages=40121;citation_volume=7;citation_year=2017;citation_doi=10.1038/srep40121" />
<meta name="citation_reference" content="citation_journal_title=DNA Res.;citation_author=Y. Li;citation_author=S. Liu;citation_author=Z. Qin;citation_author=G. Waldbieser;citation_author=R. Wang;citation_title=Construction of a high-density, high-resolution genetic map and its integration with BAC-based physical map in Channel catfish.;citation_volume=22;citation_year=2015;citation_pmid=25428894;citation_doi=10.1093/dnares/dsu038" />
<meta name="citation_reference" content="citation_journal_title=Cytogenetics and cell genetics;citation_journal_abbrev=Cytogenet Cell Genet;citation_author=M. Matsuda;citation_author=C. Matsuda;citation_author=S. Hamaguchi;citation_author=M. Sakaizumi;citation_title=Identification of the sex chromosomes of the medaka, Oryzias latipes, by fluorescence in situ hybridization.;citation_pages=257-262;citation_volume=82;citation_year=1998;citation_issue=3-4;citation_pmid=9858830;citation_doi=10.1159/000015113" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=M. Matsuda;citation_author=Y. Nagahama;citation_author=A. Shinomiya;citation_author=T. Sato;citation_author=C. Matsuda;citation_title=DMY is a Y-specific DM-domain gene required for male development in the medaka fish.;citation_pages=559-563;citation_volume=417;citation_year=2002;citation_issue=6888;citation_pmid=12037570;citation_doi=10.1038/nature751" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=T. Myosho;citation_author=Y. Takehana;citation_author=S. Hamaguchi;citation_author=M. Sakaizumi;citation_title=Turnover of sex chromosomes in celebensis group medaka fishes.;citation_volume=5;citation_year=2015;citation_pmid=26497145;citation_doi=10.1534/g3.115.021543" />
<meta name="citation_reference" content="citation_journal_title=Current opinion in cell biology;citation_journal_abbrev=Curr Opin Cell Biol;citation_author=A. Meyer;citation_author=M. Schartl;citation_title=Gene and genome duplications in vertebrates: the one-to-four (-to-eight in fish) rule and the evolution of novel gene functions.;citation_pages=699-704;citation_volume=11;citation_year=1999;citation_issue=6;citation_pmid=10600714;citation_doi=10.1016/S0955-0674(99)00039-3" />
<meta name="citation_reference" content="citation_journal_title=BioEssays :  news and reviews in molecular, cellular and developmental biology;citation_journal_abbrev=Bioessays;citation_author=A. Meyer;citation_author=Y. Van de Peer;citation_title=From 2R to 3R: evidence for a fish-specific genome duplication (FSGD).;citation_pages=937-945;citation_volume=27;citation_year=2005;citation_issue=9;citation_pmid=16108068;citation_doi=10.1002/bies.20293" />
<meta name="citation_reference" content="citation_journal_title=Fish Biol. J. MEDAKA;citation_author=K. Naruse;citation_title=Classification and phylogeny of fishes of the genus Oryzias and its relatives.;citation_pages=1-9;citation_volume=8;citation_year=1996;citation_doi=10.18999/fisbjm.8.1" />
<meta name="citation_reference" content="citation_journal_title=Tree Genet. Genomes;citation_author=S. Sakaguchi;citation_author=T. Sugino;citation_author=Y. Tsumura;citation_author=M. Ito;citation_author=M. D. Crisp;citation_title=High-throughput linkage mapping of Australian white cypress pine (Callitris glaucophylla) and map transferability to related species.;citation_pages=121;citation_volume=11;citation_year=2015;citation_doi=10.1007/s11295-015-0944-0" />
<meta name="citation_reference" content="citation_journal_title=Mechanisms of development;citation_journal_abbrev=Mech Dev;citation_author=M. Schartl;citation_title=A comparative view on sex determination in medaka.;citation_pages=639-645;citation_volume=121;citation_year=2004;citation_issue=7-8;citation_pmid=15210173;citation_doi=10.1016/j.mod.2004.03.001" />
<meta name="citation_reference" content="citation_journal_title=DNA Res.;citation_author=C. Shao;citation_author=Y. Niu;citation_author=P. Rastas;citation_author=Y. Liu;citation_author=Z. Xie;citation_title=Genome-wide SNP identification for the construction of a high-resolution genetic map of Japanese flounder (Paralichthys olivaceus): applications to QTL mapping of Vibrio anguillarum disease resistance and comparative genomic analysis.;citation_volume=22;citation_year=2015;citation_pmid=25762582;citation_doi=10.1093/dnares/dsv001" />
<meta name="citation_reference" content="citation_journal_abbrev=Chemosphere;citation_author=M. Shen;citation_author=J. Xu;citation_author=T. Y. Tsang;citation_author=D. W. T. Au;citation_title=Toxicity comparison between Chattonella marina and Karenia brevis using marine medaka (Oryzias melastigma): Evidence against the suspected ichthyotoxins of Chattonella marina.;citation_pages=585-591;citation_volume=80;citation_year=2010;citation_issue=5;citation_pmid=20444485;citation_doi=10.1016/j.chemosphere.2010.03.051" />
<meta name="citation_reference" content="citation_journal_title=Toxicology and applied pharmacology;citation_journal_abbrev=Toxicol Appl Pharmacol;citation_author=X. Shi;citation_author=Y. Du;citation_author=P. K. Lam;citation_author=R. S. Wu;citation_author=B. Zhou;citation_title=Developmental toxicity and alteration of gene expression in zebrafish embryos exposed to PFOS.;citation_pages=23-32;citation_volume=230;citation_year=2008;citation_issue=1;citation_pmid=18407306;citation_doi=10.1016/j.taap.2008.01.043" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=F. A. Simão;citation_author=R. M. Waterhouse;citation_author=P. Ioannidis;citation_author=E. V. Kriventseva;citation_author=E. M. Zdobnov;citation_title=BUSCO: Assessing genome assembly and annotation completeness with single-copy orthologs.;citation_volume=31;citation_year=2015;citation_pmid=26059717;citation_doi=10.1093/bioinformatics/btv351" />
<meta name="citation_reference" content="citation_journal_title=PLoS biology;citation_journal_abbrev=PLoS Biol;citation_author=S. Shifman;citation_author=J. T. Bell;citation_author=R. R. Copley;citation_author=M. S. Taylor;citation_author=R. W. Williams;citation_title=A high-resolution single nucleotide polymorphism genetic map of the mouse genome.;citation_pages=e395-e395;citation_volume=4;citation_year=2006;citation_issue=12;citation_pmid=17105354;citation_doi=10.1371/journal.pbio.0040395" />
<meta name="citation_reference" content="citation_journal_title=Nat Commun;citation_author=Y. Takehana;citation_author=M. Matsuda;citation_author=T. Myosho;citation_author=M. L. Suster;citation_author=K. Kawakami;citation_title=Co-option of Sox3 as the male-determining factor on the Y chromosome in the fish Oryzias dancena.;citation_pages=4157;citation_volume=5;citation_year=2014;citation_pmid=24948391;citation_doi=10.1038/ncomms5157" />
<meta name="citation_reference" content="citation_journal_title=Molecular phylogenetics and evolution;citation_journal_abbrev=Mol Phylogenet Evol;citation_author=Y. Takehana;citation_author=K. Naruse;citation_author=M. Sakaizumi;citation_title=Molecular phylogeny of the medaka fishes genus Oryzias (Beloniformes: Adrianichthyidae) based on nuclear and mitochondrial DNA sequences.;citation_pages=417-428;citation_volume=36;citation_year=2005;citation_issue=2;citation_pmid=15955519;citation_doi=10.1016/j.ympev.2005.01.016" />
<meta name="citation_reference" content="citation_journal_title=Chromosome research :  an international journal on the molecular, supramolecular and evolutionary aspects of chromosome biology;citation_journal_abbrev=Chromosome Res;citation_author=Y. Takehana;citation_author=S. Hamaguchi;citation_author=M. Sakaizumi;citation_title=Different origins of ZZ/ZW sex chromosomes in closely related medaka fishes, Oryzias javanicus and O. hubbsi.;citation_pages=801-811;citation_volume=16;citation_year=2008;citation_issue=5;citation_pmid=18607761;citation_doi=10.1007/s10577-008-1227-5" />
<meta name="citation_reference" content="citation_journal_title=Nat. Commun.;citation_author=M. Tine;citation_author=H. Kuhl;citation_author=P.-A. Gagaire;citation_author=B. Louro;citation_author=E. Desmarais;citation_title=European sea bass genome and its variation provide insights into adaptation to euryhalinity and speciation.;citation_pages=5770;citation_volume=5;citation_year=2014;citation_pmid=25534655;citation_doi=10.1038/ncomms6770" />
<meta name="citation_reference" content="citation_journal_title=Proc. Japan Acad. B.;citation_author=H. Uwa;citation_author=T. Iwamatsu;citation_author=O. P. Szxena;citation_title=Karyotype and cellular DNA content of the Indian ricefish, Oryzias melastigma.;citation_pages=43-47;citation_volume=59;citation_year=1983;citation_doi=10.2183/pjab.59.43" />
<meta name="citation_reference" content="citation_journal_title=Wageningen, Netherlands, Kyazma B.V.;citation_title=MapQTL6, Software for the mapping of quantitative trait loci in experimental population of diploid species.;citation_year=2009" />
<meta name="citation_reference" content="citation_journal_title=Kyazma B.V., Wageningen, Netherlands.;citation_title=JoinMap ® 5, Software for the calculation of genetic linkage maps in experimental populations of diploid species.;citation_year=2018" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_journal_abbrev=Heredity;citation_author=J.-N. Volff;citation_title=Genome evolution and biodiversity in teleost fish.;citation_pages=280-294;citation_volume=94;citation_year=2005;citation_issue=3;citation_pmid=15674378;citation_doi=10.1038/sj.hdy.6800635" />
<meta name="citation_reference" content="citation_journal_title=Journal of Heredity;citation_journal_abbrev=J Hered;citation_author=R. E. Voorrips;citation_title=MapChart: Software for the Graphical Presentation of Linkage Maps and QTLs;citation_pages=77-78;citation_volume=93;citation_year=2002;citation_issue=1;citation_pmid=12011185;citation_doi=10.1093/jhered/93.1.77" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=L. Wang;citation_author=B. Bai;citation_author=P. Liu;citation_author=S. Q. Juang;citation_author=Z. Y. Wan;citation_title=Construction of high-resolution recombination maps in Asian seabass.;citation_pages=63;citation_volume=18;citation_year=2017;citation_doi=10.1186/s12864-016-3462-z" />
<meta name="citation_reference" content="citation_journal_title=Sci. Rep.;citation_author=L. Wang;citation_author=Z. Y. Wan;citation_author=B. Bai;citation_author=S. Q. Huang;citation_author=E. Chua;citation_title=Construction of a high-density linkage map and fine mapping of QTL for growth in Asian seabass.;citation_pages=16358;citation_volume=5;citation_year=2015;citation_pmid=26553309;citation_doi=10.1038/srep16358" />
<meta name="citation_reference" content="citation_journal_title=Aquatic Toxicology;citation_journal_abbrev=Aquatic Toxicology;citation_author=M. Wang;citation_author=Y. Wang;citation_author=J. Wang;citation_author=L. Lin;citation_author=H. Hong;citation_title=Proteome profiles in medaka (Oryzias melastigma) liver and brain experimentally exposed to acute inorganic mercury.;citation_pages=129-139;citation_volume=103;citation_year=2011;citation_issue=3-4;citation_pmid=21458406;citation_doi=10.1016/j.aquatox.2011.02.020" />
<meta name="citation_reference" content="citation_journal_title=Environmental Toxicology and Chemistry;citation_journal_abbrev=Environmental Toxicology and Chemistry;citation_author=J. R. Wheeler;citation_author=K. M. Y. Leung;citation_author=D. Morritt;citation_author=N. Sorokin;citation_author=H. Rogers;citation_title=Freshwater to saltwater toxicity extrapolation using species sensitivity distributions.;citation_pages=2459-2467;citation_volume=21;citation_year=2002;citation_issue=11;citation_pmid=12389927;citation_doi=10.1002/etc.5620211127" />
<meta name="citation_reference" content="citation_journal_abbrev=Chemosphere;citation_author=X. Wu;citation_author=Q. Huang;citation_author=C. Fang;citation_author=T. Ye;citation_author=L. Qiu;citation_title=PFOS induced precocious hatching of Oryzias melastigma--from molecular level to individual level.;citation_pages=703-708;citation_volume=87;citation_year=2012;citation_issue=7;citation_pmid=22273185;citation_doi=10.1016/j.chemosphere.2011.12.060" />
<meta name="citation_reference" content="citation_journal_title=Sci. Rep.;citation_author=S. Xiao;citation_author=P. Wang;citation_author=Y. Zhang;citation_author=L. Fang;citation_author=Y. Liu;citation_title=Gene map of large yellow croaker (Larimichthys crocea) provides insights into teleost genome evolution and conserved regions associated with growth.;citation_pages=18661;citation_volume=5;citation_year=2015;citation_doi=10.1038/srep18661" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=C. Zhu;citation_author=Y. Sun;citation_author=X. Yu;citation_author=J. Tong;citation_title=Centromere localization for bighead carp (Aristichthys nobilis) through half-tetrad analysis in diploid gynogenetic families.;citation_pages=e82950;citation_volume=8;citation_year=2013;citation_pmid=24376614;citation_doi=10.1371/journal.pone.0082950" />
<meta name="citation_reference" content="citation_journal_title=Anim. Genet.;citation_author=C. Zhu;citation_author=J. Tong;citation_author=X. Yu;citation_author=W. Guo;citation_author=X. Wang;citation_title=A second-generation genetic linkage map for bighead carp (Aristichthys nobilis) based on microsatellite markers.;citation_pages=699-708;citation_volume=45;citation_year=2014;citation_doi=10.1111/age.12194" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Construction of High-Resolution RAD-Seq Based Linkage Map, Anchoring Reference Genome, and QTL Mapping of the Sex Chromosome in the Marine Medaka Oryzias melastigma" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11.cover-source.jpg" />
<meta name="twitter:description" content="Medaka ( Oryzias sp.) is an important fish species in ecotoxicology and considered as a model species due to its biological features including small body size and short generation time. Since Japanese medaka Oryzias latipes is a freshwater species with access to an excellent genome resource, the marine medaka Oryzias melastigma is also applicable for the marine ecotoxicology. In genome era, a high-density genetic linkage map is a very useful resource in genomic research, providing a means for comparative genomic analysis and verification of de novo genome assembly. In this study, we developed a high-density genetic linkage map for O. melastigma using restriction-site associated DNA sequencing (RAD-seq). The genetic map consisted of 24 linkage groups with 2,481 single nucleotide polymorphism (SNP) markers. The total map length was 1,784 cM with an average marker space of 0.72 cM. The genetic map was integrated with the reference-assisted chromosome assembly (RACA) of O. melastigma , which anchored 90.7% of the assembled sequence onto the linkage map. The values of complete Benchmarking Universal Single-Copy Orthologs were similar to RACA assembly but N50 (23.74 Mb; total genome length 779.4 Mb; gap 5.29%) increased to 29.99 Mb (total genome length 778.7 Mb; gap 5.2%). Using MapQTL analysis with SNP markers, we identified a major quantitative trait locus for sex traits on the Om10. The integration of the genetic map with the reference genome of marine medaka will serve as a good resource for studies in molecular toxicology, genomics, CRISPR/Cas9, and epigenetics." />
<meta name="og-title" property="og:title" content="Construction of High-Resolution RAD-Seq Based Linkage Map, Anchoring Reference Genome, and QTL Mapping of the Sex Chromosome in the Marine Medaka Oryzias melastigma" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/9/11/3537" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Medaka ( Oryzias sp.) is an important fish species in ecotoxicology and considered as a model species due to its biological features including small body size and short generation time. Since Japanese medaka Oryzias latipes is a freshwater species with access to an excellent genome resource, the marine medaka Oryzias melastigma is also applicable for the marine ecotoxicology. In genome era, a high-density genetic linkage map is a very useful resource in genomic research, providing a means for comparative genomic analysis and verification of de novo genome assembly. In this study, we developed a high-density genetic linkage map for O. melastigma using restriction-site associated DNA sequencing (RAD-seq). The genetic map consisted of 24 linkage groups with 2,481 single nucleotide polymorphism (SNP) markers. The total map length was 1,784 cM with an average marker space of 0.72 cM. The genetic map was integrated with the reference-assisted chromosome assembly (RACA) of O. melastigma , which anchored 90.7% of the assembled sequence onto the linkage map. The values of complete Benchmarking Universal Single-Copy Orthologs were similar to RACA assembly but N50 (23.74 Mb; total genome length 779.4 Mb; gap 5.29%) increased to 29.99 Mb (total genome length 778.7 Mb; gap 5.2%). Using MapQTL analysis with SNP markers, we identified a major quantitative trait locus for sex traits on the Om10. The integration of the genetic map with the reference genome of marine medaka will serve as a good resource for studies in molecular toxicology, genomics, CRISPR/Cas9, and epigenetics." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/11.cover-source.jpg" />
    <title>Construction of High-Resolution RAD-Seq Based Linkage Map, Anchoring Reference Genome, and QTL Mapping of the Sex Chromosome in the Marine Medaka Oryzias melastigma | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"CNvyaTCKk3tnG3SjVfQiqhxwlmat1D3HeiEfJmqb3BY","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"494592","apath":"\/ggg\/9\/11\/3537.atom","pisa":"ggg;9\/11\/3537","ac":{"\/ggg\/9\/11\/3537.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/9\/11\/3537.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;9\/11\/3537"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;9\/11\/3537"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/9\/11\/3537":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022jX9BccPGWQWVt79rzbAD3ladpy\\\\\\\/gr6aUXxkOw33Pv9vGDSkgaVN2iP4Dg1c35XtTdaH2Gkc\\\\\\\/syZi9QKu9y3T4z51z2xrmVgEh76kgM1WVI6GXLYUr4e3U9GGir+g5DbJIb6niSLkuj4ajdNL432QRd9z60qfWAe3\\\\\\\/RK+58M36F9w5T71tibh\\\\\\\/he4Q0cxx4LZsAmu+qX58XiVD4ju2FyqjKJoPR0R\\\\\\\/pTczM+4aCwlmghWHz38CkkWOeIy+EGU3Im2via+j38++xg1ohnM0tQO5wDXI2CjzTypCPEdJR5PFUW1LSVAisiDCoVzDmPGviM4M2SYon6gZOs7fCw8sKUMYs4HS\\\\\\\/sLtvUqLMb9btPSx\\\\\\\/DZibmqfwGAKTqM+HXRFXUapPCfGPKcFCcyyAocL+Kuf0b0FAosAV74d+kXKce2ZjLF+7Wp6e7+VDAoe3FfYsA1kp8FkzoLAXMzIfsSZe188LTlFfFTAFBNE4AmBw6DkxRNUBPyLkqPfSXsu8wIQiyfqK0p\\\\\\\/N44MI7SF+jnzEbgqx5W12CX+02fLQ+S7CdZKh\\\\\\\/kA0PkxOnwi7var56YVtzCWT6PLZtd8aC78TO\\\\\\\/3yCKXNrufRB4IC3PeTfM56J+IP+hKyIMPkB+MG01AVDVzKbOzDH93tJXgKnTFlGq9G3mcCzm88tAP2peTpA\\\\\\\/Hf1UD01xC\\\\\\\/dtKcntXSk9koxzIUeTsH+b1dKDiG62j3kjvVGmFzzd2+FoXk0wB6UIZ7\\\\\\\/Flwpa3J6bnUXo6dAhBcsG7YoSpNMFyiqU0imkagvo6EKvEBTP\\\\\\\/AS3+gtfu0ZgMG6qGK0gRNcPXuQChoHsYsCr5dvlOIkOI8KuxUIQh5fTFdBuSxAZpfI6AaDJmbwO0xT6+l2T4ADE7yBfG\\\\\\\/333aDirS9dsTQ\\\\\\\/kMaFkyEFIvK76eYbxgd+uIVw6IfPI6ewdaPFmFqkpM0bO1Ayl84+KBxhnafj6+B8SWaNN0ay2\\\\\\\/W38bjdIvEINdFfjLeQM8AAUuUPlwm5TBYFKc+lzNUXN2LPUCrmraxbtmwxtKgL\\\\\\\/fiB0DsmELoJK5vcQpgr+GPumkbxDOC2eWl4wq9ffPPaR6qTR12KsojSnOnhp3S3bMAZkhg2KMIeSyf3kaTojiy9n\\\\\\\/NCgPUTn0xM6wllkXmzBU\\\\\\\/HBdvH0kOibxh43jp6f32ruYwUzgEi2tipvgjN3abNkCVorbjM6iIfq7I7iakvcbiDP2cRUMtavZcEGhi71kZ45AQtbOD8A+h6MrYY8ENWlGay6hYWrh80IxNNX9bWTWtX4bi4\\\u0022,\\\u0022iv\\\u0022:\\\u0022N\\\\\\\/o5wvwS40JUbl1j1T91Cw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022c4871c2e1d7cc8a6eed0849bc1989a2498e845bdd04f1190f5792b838195a001\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022MgbEL7SPi5yw8owl6Jeg3qNCjlBIXy905GJw+z6byVlOG9BH\\\\\\\/zNBlHylSF4T2EdAKVVbV6Pn7Ekm0r40XLpfUmtchRPVn2nctUK+7wjjbUz6SJ2QgJzjB1nkKp3xkv+rMhveo5bCdlzY5+8WjIg5vK6Ja8mt5I5MMDFMuXW2acRd8IExeJrO7pKNlRzujNmKGpvrU5qnEEQRgJjikBVXWiNN8IHdLXOnq5TzU7+Zp426W015WhqU+gfuPxDsy1uZIGoUFB\\\\\\\/HuzaT7hgukRYfi72X3OLrRg+CO44rruDlUNn1p5tTpa+4+D0dKpiuLCFsOcfWnbA8lYjd03VkmmH3Tv0FQJd+VhMC1wwjur+h2oP1TvIRs+s6WhhIANBwJ7QfoCEGbd8NU+RIgd+eTC5RrWV9By5bB1RVwQMtKqAmYQBue5zTxpVSqurM+L1z4KiyPWyXOJpiasJaE7J958Zx+7hezcqleh6GV+xLUFWOT+J84j7LrhKVxOczjBSz0Zr2FSk344r7x6yzHzNhh9axIm7wFs2xHyqRnUOctZ9yRtOcmIoW\\\\\\\/yDyN63Sw0OS7z4BJVDTbFTz0W8JZNVLuKm9+yl135gMZfyfurtWEDEST8OlxgnfK2MS+J+mi8+MSkiZGa2hdoZ+e0pUxEnzq2Gb4fS8X2VRruo5fp\\\\\\\/gh5IddPZkOHxCLwqR6uxVe2\\\\\\\/4oa2IydOAJWn\\\\\\\/C9HoSMX6Rj4EzqJJOkGTFu\\\\\\\/yfd13fZCfGci5Jjbcc8E00jfImrpYHMqFo21XsE100zEl2vH1k\\\\\\\/UpxoyFzISROiHRc8SHTww3ASrGM1tWM9ob+bBSgwdkK7TuW8kZgSr29r2Ygsf6uIAcIAXI5XjUD33ArZRh10HGgS8BhmvZNzPlvx1bJn1PurHQBO43Pg8LBo+yhU5x7cxhnGQavAiS1JVVaakwkfDP1PPgM\\\\\\\/t\\\\\\\/hn71rGI93KAsEAH5mLx8pvZv0dp7B16HVo9BXiIqzTm\\\\\\\/9lAi0OP9uwAvsgvRIJiSWywLV+8u+e2zoD8Kcj7t8RV\\\\\\\/pWeXtbGg8Mglg\\\\\\\/VDfTBIeFExVMEtaG5qrO4c3BgbD4h65FfTW1tHprX4u4oMmTCMUf1+sVHc\\\\\\\/KA+RX2JtJOxIKqP\\\\\\\/DPZg0e7zzSU+PsEVcQbN\\\\\\\/6OoRFwz1mk3TJJIV5TN44mRFedv8Irk7i\\\\\\\/c1BgYro84Jbuv0fmSeOpPu4AUfLG7c2A\\\\\\\/Dm\\\\\\\/\\\\\\\/iefCC6Dha4tn9V44meDe7E3ifweGzWm+pe69XlmEElld7Ov2Acy9YZZ7dZC31RoUBU6Cn2OOY+3zZvou9CakdSh50im6CF9xypMTFXFCwtTbIrovOuKxHyQt5H41QAQVVvICEjzji5erk2UO3D7pBhSD6BkjBw8hhHXoGs9Uoexh93kavmkXa13ctafnFnsz4vxQBGv7aK9g7ielJmHngE3F9k0xbWpxD9r3O\\\\\\\/OVRRkPfzFoUi0A0D3MN1wheJeC+RLXyWxPiboKVVmVU2PXCmx\\\\\\\/lV16qpdgUXZvLY39BzD50gduEvOYdGM0zKLb\\\\\\\/vL+I\\\\\\\/CL4YIg1LvOj\\\\\\\/uVtlyyuozyLedsK3fWfysu8ZfTi0sAA02X4f7h8PyWQA6aW7ZIK5I60ToiUpmN8TrLhCy1uEbYp5tElL9fLg3UGWT27ICd5dUtqqkcaIXZq2kPUYWJ1XS3gZPgbzcFXo3r0\\\\\\\/LDhMLHqTt4ND1hN4AMtyaBchxr9tyN7aY\\\\\\\/iCuHJQ8b7E8lRq89DlO+kMjtDwdKIzgmLZrnbCHkWvMi1p6fzK6YhHOtUZ0uCO4b7mr4kMScNqAEX83FRzq\\\\\\\/ZWDnyh8q\\\\\\\/ybs+TKnecRQ02B8SEMOeHRf+BSpmF\\\\\\\/9GF2ZLUUUKU7tUUjHp1u87Em07NdKKSAbDHtWLJeqSVIBkymkj8zvldKqOgYXAQPzFim2z6A5njI4DEhcg+KOfuy4ZeHtvnKL\\\\\\\/vOyWUEYlM6i6\\\\\\\/8uXd9oBxvHCNypUYgo7lUbsAcQeRUM1IzGayavkKZogJHZC+3Rg3VQJNwbvfTlPu7zhwVRhRFVosnKcs0bHf9rg\\\\\\\/lfneIt4ebzYnkOtWVFtkdXYbdImkBS7OQajD7tvR+VmzV\\\\\\\/u2WB5h\\\\\\\/pNkTePr4x5bqej400UZRdFAY1mx1ftKk8\\\\\\\/FTkbYUAzdE9Gc1ZFG8wsLcPx3cn89G42KWJgdWYMautRH0HzDVBqk+5g67NBgJbTiFQNSr95O7XjZ4STjyJ0xDPCeqtDRnLpkJvD39L292tgKjR\\\\\\\/W6IUgOFIBlQ88yNnGQo7ZEqT6EyrH08I8Zz5ZPfDXhHvUg+\\\\\\\/289HCozEyR8Rjj\\\\\\\/xtfV6zEHygiU9QNCmLrEJ2djX7cgHdPeaqSe6DwvK4h4Y49TO3\\\\\\\/IeXAq+RSNQeAx0QVF1kAQPE4A1AFMSujd0Qo9oNpooQezPkhLe3UY2nW3HY6+wqBx\\\\\\\/dsjfIbqt\\\\\\\/6pqOViEJFGdjxaz5w+DRtlKVsSWeKAfxO5c3NFDUc8UnxZCSugRAu6NeirQYdNQBzHg7HTQpPllFQ5MOzabBQF3oGw9YsoWEVvX6I4BDkfK2OlfwgsTdo8ZhBxKnHmj5fAyRGe2KidrZdqQKhSoh9BJm3fXs52rM29vA6+4bsf71k4+JQUd+k4BrwQvfMq3meD3JgXLIBMZNLEuFrjgXFUBGA8pitis1lkmKTwKf4Xnz9ddfJWWNQYgXyOxt3PB\\\\\\\/ZqqFOov2rjX9UDJT4fzFmPB5YJAdlsJd8GqynaVtpzk8yFrAfjX9ToBDXzp9hrf\\\\\\\/hgrAVVtRsCwNKsyn8JzUnDPAJJIA1emIGcHR3QKLw9BpfN6+xNMcQCPmrfGOf6zQLWoRUOqZw749YBtTNfe1xREmHEif2YCdZ9mnQX6aa4LRzI4KxnfwkUENTRN3ebiwPyZup3KAvzQRLTcEf3SX7J0vsa0fM+vv4CAZfw0zLcTi4uuKx+EmWCYVywhynrx4K0jlTS7WUgUrZtQFv0fcEIXYcF\\\\\\\/7DWzRQnvx7gigw1tJYNrNswxMOvAoScKNrgbgrdclnN1S3LwNO9pvrMyD9t0KfZkrPR7fz9yR9Ibb1zV2NtJkTl0jHK+d8bX6BCyfvQzG2HkTtf2mFGZVwJsTDdKLnEqi2GtKxDVjnsBZ78ZXOOrKYRHOwg0gOeXTM3yt3ZU3dc8csDevdxLi2UmotMFSR3EuHfVZiTMhcvhpFrCA7Eo6ohbq7Tl1dlIMyINuHP+G2HWwuHHH\\\\\\\/HU5ZfF74S3bKVaGBdSxenRtR5zoKZqBzx+ihYveDzxisyXvgnjKLDah7uYsgXPtjDbSZGBaddHtmqMy6W7EwHxcrYFtog5plPbUW1BfwSVM4nBUod4\\\\\\\/18JIvEo7XQnwhj5FK9Kt9+83XSJYMa4w8nX\\\\\\\/sSsHWo8Z8v5iWIR7b1stM3PVF8A0Lr\\\\\\\/aGDkR4zvzIDhAPfykDB3ynVbQa9WYJvfz4F\\\\\\\/76y5cEI74YVILYSn3AVYxyEMkRtM8RJikSvX2tSpVKE1O3qiDXO8bLmiZh8wrAJDD7pkqUX2ODt0huHIAlAtUCO6gFh6r7hdh\\\\\\\/atAhU7RZLPVkHWbhu3SFWh30w+Hmcs+pJVVErw5MYPDJYoG5fVvPsaTpdvOFu0pj6Y+RPHSPsch6ioYkptS8tFdtnNskJwkyLfPIb\\\\\\\/Rce3YMOOMWln7dkm2A3OO3FynY46JdKgikI6lwiOUj9smxjGcYKd46PR3xpEo3MwzecFFHdWIrvj2DQYM0fn0mfOKUuTaXUXJuGhn6qK4NYEv\\\\\\\/XbykMiPnzPbNQK0swNXnCEsafSYglAvS\\\\\\\/TNGW0mH+TgXH2trRiS4bZ2hsj7Ch71RzCtH0hxw\\\\\\\/DCSnIKajySzLVX7mTi0bd6ykmGEUc6wlgzmi7N8zSoKtn4lgFyAiBn\\\\\\\/CcHi4a9MAUWMR8VzZONfwg88CUN23QEq7EOr0t4iml\\\\\\\/Gh35J6h5T9xB\\\\\\\/97WFNiRCKcX7UBC+KJ\\\\\\\/EvtZu61CJrOr8RaXQnGCswCIPGMIG1T26yA866PXJQtILhAPCDLHsKqx18oNXGD9R7W48Ai4lyX08xneQTElz1egNoyscFSY5tHdNpU5IaJPosYCDWeMD1r9iBHm31hbxh7MPcnCpaYTcH5vt6Ooh57M3hzak9mvFLe1A2nKO27++pFHhE6hwDjNPyub0YmLtpnZSNPmckEo7+uwDxcx5H6L65youTmSf\\\\\\\/HMaryCsb82ejtK7ov8KZVp9U+dY+S6RUckTl\\\\\\\/ylXQHJkFRsA8Vu5F2AqfJXk7Ma5XuK4fnWQ1HaXW8z4HJ2C1NaWH1A+IJmeff8dDhIIMCt\\\\\\\/LvBL9qOeKgxRmprGwolyQU1ZXoCKN2LzePckG8\\\\\\\/iPpz84re5gl7PShiB1saOp06JFId02H3ADvpLafMg8sZtg9tBsQeB2G\\\\\\\/cjRf8dkiNKOx3S8+mFNszD8oTxz7kvOmaUd6n2gQlvR0sWAUQomJ137pnHb\\\\\\\/SMa5JIBKooPOIPEfp2KK\\\\\\\/TRAFFYgqrmf+V58lEEThwdlahWGtVzqDmO1aw8le1VC3dOTHTQDj7pSh8SPx78hyn6nDsxbhA+LaWyh33cCp6C2RIFnba5b+wV8L+5GrWWNGsQ2XqajE5\\\\\\\/f0oRSbf09vIFFPCdNK8qXFuIu6Qmj5klcfrcRROilO\\\\\\\/ohFV6fq5kyKym5aPl62l7xZ32NGMgJ53R6Xsmvn0\\\\\\\/dE1dOSjrz9feExGXuDtXVi3jcZhsGvpkorDBfgFVDfCWlLK5YOZKGiIaLD7Db4j+982erlHp1rwlpYY6Om9xLAD\\\\\\\/+vmM9QLx9iYB29F2ZeVseXyGyiiUHi1p414YAEQ5gO2oaWZGRE98ZRUkLDdebFH9wo7v\\\\\\\/QygtAzNRQiZlyIPFXvTVcQMeXXYv1N2H1qL5I8BIFO\\\\\\\/ZVdcjd2E4bt4caaexPXDHImq1xkt1GKyaloxOpwBW23WSfrCzRmSjfVAQF0ssOZ5ZIL+0L2TxRpYV485b2wGsw2z3HBkRTGTG0e8cpd\\\\\\\/1wHy2nUVoG7Sn9dr7tBn+2ekjuvFozhxN6G\\\\\\\/r4p\\\\\\\/xdV\\\\\\\/lZexnOXr8nJjq2FRGtruabC8fGNxo9wwS7PKwlWH\\\\\\\/rplY9sCy\\\\\\\/YI8J93rc7rpq2RFtV7M6lxSn1LE4IwN0\\\\\\\/\\\\\\\/mCv5pdiXIcdxplGhlbFq4ctX9U4zkpciSCqCqspiB8bapjgVOOQRH6147e5puGef8ToDil5dt7CCwCY+TvjRV0tkf9Y5tmWP1GF+cFrpu8czSL3eAHNKOvFObYAT5G5jiqguVEOODXQrrjClt9H94Ycne0WXLBbOSa8+NIdXbbLriN+dqxCBWB8+GBksG\\\\\\\/sVx0lgOdg\\\\\\\/qhw8x\\\\\\\/rO9rAV668PB3k8LuqHjtrOp5Pao98vExR+7btWFVxY+pMIfpj6wJeo2yW8Xa+Bh0tgeRWahWhmmWKR\\\\\\\/KBZDIg4PDskLyMCNH1J+WuEY5Moj1uW8Gb6b9e4v8653Na1xyMJZ6C\\\\\\\/RwHA6RXvBqsFpgzTFngdAm3EyzdMi04x520IGux7vxU9z\\\\\\\/P5e11xJi6kVSBgU6QB6xpIK4JWNZDbUmTH8LR54yfRrhJNfFKGP7lBIZKfWc4JCaPb9bMnNyumb9bEI2Zmz42O1rIJB3Q3mafIXZMWjw4NKfZyPvaAwuURH\\\\\\\/xigwmgiAF4TIdKIWwKXfQfh1l86S7q2Q5WO8fikNWOI1KdvO4nT\\\\\\\/AbhNvf+T9J3YE94TtEhfiKqV2bbKGPwBRXk7de2IuvBHZjMS0u5GTa1v9BsUfn1dRnH6YdR4VMQUiweiarPK4gAeO3ok3Hn5XhOIq0iC1LhY1MKcyfn7H02hdGiehobOt1PVjnDfB203G7dGTo0OJgRle\\\\\\\/4CphCxN1k+ZYEshg8MLQimkhyeOKALyfCfz6WiarXGP3Hlhzg4U0nYWIGxA2PPT0Fhjf6qUEVRB0dNZI4SLLNlernn7N5yyNwRrmwtzk4bSsIFnFPpOredAaIhodOdJuLBMb2QDT52kX8a58RlorqdMyyG2ZpXRF9XXyApJoBWSgzT4Cy\\\\\\\/MNBZ3bQ5j75xiHwvSNEf+gokkylBBEBtx9ZJ+0wuhaQTRWq\\\\\\\/r\\\\\\\/0Bfxx8f3qWdmHX2t+yfNsXuooZKhIfbzJkdtLT299mtc\\\\\\\/fBLFju9VLGoCon23AZQE5RlW6deBq9EEKR8kl25poVPxoEUZaSs67a4MElqDRKJwC1lEIkrKGvUpXiv2Xu1eqLfpDagZjFETvBRj291ceBIhYWsUrFWu1uZFFwYW8WmuHxDvyIoAtIkLlMhAeVRwXe6UpUDbs9k9v\\\\\\\/UcUv\\\\\\\/HvhPKQFNUYAeXowqJCHqdZbWr3yYzwpX6pmJhbZMtvsi4fR2UeEgEbzdOGZSfo19aNuCfkGMYP62tQVL8hcCs3HzOeL6fuk7cFj53UcJ61qhuGRgdGhDoeL\\\\\\\/smLnnaKoRgTj79bNQaoMIK+NUZsSuI0orom4iIwVPsCaN1YXsNXEEfD2nkDq85vr1nXmALrDVs6R+ihtGJVikfyKzS1Ylm8vEsHW33PseCxnrVjT\\\\\\\/7Wilqtqz+\\\\\\\/y338sghK8cV2ouNJCUk7JiT6ck5TkfuWx2rs5h58p5FpNofQgWcRVVKGL9YzWCp9\\\\\\\/tPNknvhIJp\\\\\\\/V7SMe0yywf9gtXXpJIjc5nCXJSLqqBfoRrszm+wRGJ4hcmi8IEn56I\\\\\\\/SAR4wzubF1FJuUgJ24zI\\\\\\\/OkK6J\\\\\\\/UkIlQoCpkqT+rQ+HshNWsD7duu6e0BTXtp4znFXjCI8xCC48jozS4u8YoDfMO58NWMbHY41l7\\\\\\\/Lg4eVCNGhdQdB1zGgFmna\\\\\\\/A2Q1xP1\\\\\\\/ItYk3YjfqShmjUaUu2HQYedHZQmA5CTQK612anIFbHUtG3Jj+gWmD4yN1g8eBZWkRXAPSgOmt1ToW9jNo8cQkateKjxgsyHFbGts554MKUUYZptYTNfOx1GybaYSppBwwEgez80oanX1Hir+0a34r23oG6okMotGTO6tJzXpOsiJ+NBJAIXyl3qRXfyZxDqLhOBImdS0uTdcwVO0+M8pIUhjKXcjzfaQw5Usew5RZnxTAI6PcVDtqiZGZDK+5dLOlNkXl2SizIRYnTZxW6eosJX93NSocUA9XNaIIAUZienQ8QMwaOTXQ+9p4bS89Ao31jidZpdGspHDN3FZfF6uGf1pgbYLWkQCZMDgtuKWTBYvB7c1\\\\\\\/moX5g1djK1g0auNjwT+mn2xy16z8jxPgONi2ARLcYaczFiTsF50XX6aEzaq61pmHB0\\\\\\\/yvovjI2IbTpvQTU2x2AEdBKtwCGJHf0LHMtWAgkx3CSo\\\\\\\/JRn\\\\\\\/ITIaH5zCe\\\\\\\/dNSWT1SWLWFTeKzp\\\\\\\/wJiIfPJUQ3sLwU2+1UczZKoyXGbcKKLb13zjOkQPHVlZ3gXK4IUWOxMEI2p+RCe8wGghtU3B7FtnvRSSZtSGOCl5zIomZYDtRCZ4tihs46UhWIGWcI4rzW8mnaV0j1LY5jHggkbiU7JeUMvcUyRF377bAnBYVjbeLaTN\\\\\\\/cvZ3DbDxe4XrIyPoZiGVKI3Cb\\\\\\\/Gi\\\\\\\/HottcFImHu\\\\\\\/Oto\\\\\\\/IbRZIi3U8xt16+h7wHc59mg8iRnAn6zecc9ByTMjHRL0p+dewUcgh7nidglGVWAEmAEWJgBFnqOu+7ZGB+bjiPov33v+7X5QQDmOJHHzCdPgqNyYzyp0fdlGhfAICCNUw4VmUktQf3qBGkZpHkqMhSb7Tf7\\\\\\\/x6MUg17N6Vc1XCxdbaFnQHyqs4WiRqyR1HOloeccVNumhfsPxoZkln0rZl\\\\\\\/is0CkPxymtSEtoU0Jnv0q3JhV5NPApluweE61tDu5DHPK0cxW9xZeWjAl6TbH0n0m4yCeoWMMx94hXheSu\\\\\\\/gHzy0U15VxTSJ4MHJn24CX30jWjDV0MIvIVde9Q2MqanR8WKQ8YFAf2DakxRIq7t8UodBLzi32Twlt0UNSKky3s3gEniUT1qNY7PF+H\\\\\\\/aWNsknzoMEHM3YO6rLV1rr4XrsuxH6DHAlfsdoU3n5M+RA1XO2nbdeEYdaRi+Xc6ph3Cw0JHJeWMIkwr3L9l2d6rohBoEms9uLauQqUtvETzewA8ODLTlXCgpX5JIFM\\\\\\\/U1\\\\\\\/cUiv3FgiMNRM0HRBniAiJxU0CT+tpqZpseVQITyeGBCjKj6n++goDH3t2AzhPSTfiIsx9gfXCeHfNWzS7QdqdPQ\\\\\\\/znYUcfqhwBdVdBVXqu1e84AC+zN2ylC1+1D0fkF\\\\\\\/9SVMc9hFI5znVeOsrbqkyhWPF2U8kzqgmMLVgMpEpkO\\\\\\\/9WDD4XPy1v+BXIn7mO2A5Eo3\\\\\\\/jtP77bJRkN13YzsFlZyb8M5oaj8ykmq\\\\\\\/DDTnhVHDRtrjYVXLO1zLZ6rkHN8k6nATuv7NAEi0vC3I\\\\\\\/yFOfTQ3a13dvrrBhp9TTp9elhK2pnf5fg4mNdnSUHw0T1nEK7SldTLrK0HIA65bnNp8D67ihCqjdxWKUUXIKFT8fM9UBhTJnI+XAt+AS3gz2Fd8lBwHS4jO9wwhqqOlVaseBEkmS9thSHYklfazMmE65OUSp90s6uDW6jOj4SqhL+oedccUOtUWCmxki6lR4Tq0uqHQICsKjBqslNbmT8x6YRdgU8bUvjmQqm+OFe+lUL67Sb+RvVncQewjRYkXI3Js2az40jUjOkfAE0rRiVrxNoUyMmbx1UJXtzfU0YmQoN2EcrqbbINKMQpndwXXB5mI4hAMTrTd4SZl8eO8zaixwhBYfSgqeOi1eNFuNlWN\\\\\\\/BmMJpMe9TByv2YrwP3cJOjsJPMTclIWVEwNi5edNsmsKdrgaLbV5J5f3VJAmyeduK11r2nBuevnkrOAXJCEmg57ZBZN55+Zt1RbfF0BDg2wuYHLoqc\\\\\\\/rg8+CvlfCYymnWB8QE4kkFvjUmtB6tgrmWKTHdoJ5kcV4K9h+v8hThldgURsw8CGZpIi9zbe96LWMLfRZKegbbTWUb3INb94vVnRf9RCnOLKQBs7O\\\\\\\/5zshxCuWrEA6bqr6UCIQw1YH2CohI06J938fvDuxvD6HAiR+O4Ah3Fa\\\\\\\/ooUuvXqUTevPbM15HUcag1uINbNq3MLRDBVs\\\\\\\/qU4xUAZ4l+eAtEI7QnXCg+9divhilFsG+LgJDm0L125J9AWw6w2uJdZNrOf50M+KcyUBLgXKwt\\\\\\\/q2FpDRR0T14ZcAXw9bcB1+\\\\\\\/pjaFi\\\\\\\/pQ9yH6pYRdAWhC0vZFaDC3Mm\\\\\\\/q43SdSe\\\\\\\/TrLSYWQR1X9EtsbZH9R6OuJWx2ZKwYClCKmfGo4UmLmptlQanB6w\\\\\\\/kvMiItcnbPysHE9FXRhD2gHeWxpDPpF2\\\\\\\/E25MVnJSQb9Yb0Uf\\\\\\\/YXJ3OsHoLOON32qKLtpFwGbJXD+kxoJKOhFyLAiIWVNF\\\\\\\/7nX0vMOgL8tRSo2qL6GZIsY95DUywAR9K7+83Ktrd2l3Kjp08HE19zMlOoZ\\\\\\\/dZ5Y2dQ6mmZyxJEg0o0ED32ANYlHROTKaiCLIQqgLzIC7i5Q7Ub4jaz6NKpkMSo+VySc9X\\\\\\\/34k7W\\\\\\\/rgTayBzeVNt68TK77Dfx1XGtVWzr7+yZFAGmZgpHhZXRrQIYBlwvYhmO4UWkExREHhEJajFOJzHbaqeXckrQM72ctbvGPH+tPLx8soEzUdNOJTIInsbaFUU7QEFOxuAT\\\\\\\/hQ96yczgrD289T\\\\\\\/3Hb+fsBaJn0k0szPyNPrBQpuIrrJ7Gi7L5V3O6qnmeiGPjl6REe3eaCo8J8QU2GOrMNnCbq+JmGBK2jtkz4i4PaNacRXtexBLMdZGF4gG6AuqsVDdHY3yQ9WDrTP\\\\\\\/EWSN2yxnvlZ6Yu0W2eNFHcpsyFHxadew+VlUeZNN6vbjPzpnXEirV5Yg5DvF0TKF0CyR7W9f4fQiyVxedyKTjLzq6CxSU1kxIC0wfSNr60GkP0gru9vhdRi\\\\\\\/gHsuNmT+ZjzHIZ9qcbIb2RjyAxghGsNJGeBQQCwhd8uWrJo7\\\\\\\/iQWFigPn+pZDOe2RtkY9kof9Xd4sy4yJIZ\\\\\\\/cu3M2pB61E2O5MRXVG05Z78wlGzo9\\\\\\\/pdbMlsoRu9aJRHsm8yf61wGsDazdEMHRs3atwnk8H1AB9pgF9kQmDaVmLn5k4wntv0QbhLTYl352D5KkBdF364bITT4d7yn7vOoKx\\\\\\\/XEFKvSCNc7KPm2roeZY2fGjFJ+QkSaezuddEF7w5i+sYA8vmbTQXlLebhbaG5e0b36I8K+xqgBK\\\\\\\/06NC3vXWS6OjDFEbi4st8I2zzAzLHZcO3u4gImLF50C358z\\\\\\\/DhW6FvjPvLqRzmF\\\\\\\/niMBQvANOyk2BsyTlSfZnGOaCw6gr6UaAQ2b8azuhiTvIW9RrhrWx6AqLipXB3zcap3kjke7fI\\\\\\\/dtIZZtbWsFhLRuYFCEx2+WSVt5qej21uDq+IKLVuUZAwvPyUf+Al0og1iEPQ6cTNV+5\\\\\\\/2vhz4gUQWsclGQnH9GSp1mhX1vIQI4PgS9FL9IpPaYNEUZFDM\\\\\\\/RMLBr2KEnIYDOPhpYyfOMB3lUkPqqjW4rpQVBqj6\\\\\\\/1tIx\\\\\\\/1xeuYG67S+W6L60Xr9WgRK9ti40Vdn3KljWO1xCoCHmipy\\\\\\\/l5qYgl82MNLQ8XIM5huOcNvEx4We+DYalaHJMVnPcK2xNmc7++cxU6+K7WuDlWQoDvvViYb94UdxIsaip8miMgaFjA8sprqnXxFr1SssuFoZQtviWCLoiYOMgBPUOgcfQPE4l6+uaICtv6rVl5DDuz\\\\\\\/8Au37FHZUVj7POCjzitRGEDe5oyDNuoYHzU1QTZqVNM67UbaAKTQggdAgrRyklSDcTqNt6tnsTZwllweqKBg17ZZ9Arbl2VsUd8oH9stHPkoAlrNv9cpqi8Z6hXT5BrQI5P1p6iD2kxHzmGWIOJs+V4gaAwuQXk4yv2waacTwHvd\\\\\\\/WSx8IS+ua29eRK2ZGQnJGQU9DteoZWh\\\\\\\/RWhjr3E7vJDRFwUepPDM\\\\\\\/uTnVmts6hhnnjaYG7er48NKwFzCNXxhBhx6jrQGw5cFKNjbCm6o+i7ZqxwxuXFp2IU+hHSkisBGC3tzEz1dQ5GS5Wrlkbf2vO8IVrCSJrQIawQPAAF104VrCRVnOURpXM8hTnRqMeHonbFh4IcN2c10Xwi2lHoESwUbFcELw1\\\\\\\/wxF4dY5SMBkgMA7iY6+EaHao77r4dmRNmXVXSOOtN6wiql+i1Gva2DHh2PVAw4HuS3rc5QK8juPwfiKwWhSXu0KtWNJv9DazbJJYN6H17kYutloQ7WacCOaq4uBElpbD5YssbyBsElv0VeO83koaDrohFFmav\\\\\\\/dcYYzrmAbGEjTn10YdckMr3hY\\\\\\\/\\\\\\\/2A2K3wRQ7CUHndvmfpnLffqT9PTfKMSNWSwPph0W5vPZ79gdMrxVfgQO6jM6vJ1cxuWNsxVW5qCWdKN\\\\\\\/wN\\\\\\\/FI0ImJA4A0MMNNniID6Lct0eE\\\\\\\/yQYY0QmQqWrnQQNP8iU8EIJjMpwQuQftfxBfVLip5dFp4TBIbZamNYY+3ftpXXU4LS14rPJ84w\\\\\\\/Jc54g1HiypMS0Dm9JYKi855dgVXagkIX9rpd3DHRYu\\\\\\\/ew8ucpASgqSGfMY1DaI+Dj8SNEVmhA\\\\\\\/bzdHImgporg\\\\\\\/oe7KaPTlQFZobIPO3CWw7Vb8WKFUmJI52\\\\\\\/brozKY\\\\\\\/iKV6iNRNFtY5rKyIBQplOWO4SqLk6Zrd5nSlKTku4vLy2MC1BjJ09GW3L+kvM8yp9gYwFD+YY\\\\\\\/MKchZLLp7yifZ1G\\\\\\\/shNG3BD8pdLz8ihCDmmAKvBntJfIBNOuboMQEAAXXp4qtOiPsRN07fPdeyqKqNi\\\\\\\/iEwim6AgZ81bZaWaZawHWlwVNQk1viJ718KmSWNnbl+kjVNTrD9J+0XGtsgbNVJztfdhqAe3Wdt6+CQWisF54EMncYAnxNU\\\\\\\/i6wM+gnv6I8D0FBGcocgs1LdcTog\\\\\\\/wsM87THf04yBCc7PrlCtqb0wwQ0bpsXy\\\\\\\/02P9SER4RRcRObDgAyJGtt1ADPQdl5VQZ4myjE2PPwKFTEusQp7Y7bvHEBNyTl2Bll+3M285gLASM4Y8U4xDmyMfJa4pndLAPBHt+XsbFQdkneDaX4MHe5u2mhWTxRT8+QE0f7G0SNeSEoLykeylpu3CH3A0qN83m1TM6dKoDJ7pk5ubv1OHcW5Y0UCKilUuDi4Om8cV\\\\\\\/KCLU+2WFkKL70yAgT3ZxYoSDS+Pa1n0zhBbdcP8obVLqOf\\\\\\\/eBLLH6WvR\\\\\\\/Cc+AQkhX0qaNeaC3HPN4+QfibgXbLn31rV0toQ3cbMt\\\\\\\/chIbXfVVJGUAjEzuler8NJD9MiYqPqZvw+1ujd5sWEz8FJ\\\\\\\/gfyGDJWWYQjwK5mShQDhFZ\\\\\\\/NGHnn6ycyqfhDlxfZ11BKuqtnQQAjBKrqvTExmcskQRkF7clIQIhb\\\\\\\/QTGrVt2dT7IXX\\\\\\\/DN1N2NCFF6861J0R4Vx53hB3oDiol++C\\\\\\\/x\\\\\\\/3NPu622DwhlcfQ+fOS1HvSQjTAoLQ06e+zVAPBTPsHBzGf2zXEaWxNwvPcuhjppFJLPlkujo1Tij\\\\\\\/epHc\\\\\\\/g8YUpi+RvteNUXrAMWb3cedd+A3jhWkAQ+7rXmYpPNlHyZiFH4+HEZmBga6R+00fePM6XFJhRwd5bzhbfJYrobzDiUVlnQq6uZJsKZXOJ35NLqNCp29mSH5k\\\\\\\/6Xq\\\\\\\/W2X\\\\\\\/4k2FCUButPACZSI2RPRFtMWboBR9X2FSARZlQmAC+KUiM\\\\\\\/dZ1Km95nVRQttjkKcEmDvF8KnmN55QiGEFxureX\\\\\\\/OPSyVzJZQH2FplyQ\\\\\\\/TR2bnr5Ifc\\\\\\\/TrznVN\\\\\\\/G5fsMVcJDHombRPZSfcXMSOJWNbWZ\\\\\\\/i8hQtKDYIQXpZw0aHvvAZyEkmnGn+KG8lsvAtjk1VRvc2XPIkSKDYHDengQKguDBTbQJy54qz0omLu86S2mlyqFe6ZkOZ7Zr8rEK7zcKX5tRjhQ7YnBg20LeyNpVH4g\\\\\\\/Zv4W+X1B4F0Na8soNe11tsuubQD76i5FrsKqKkZiNsPUx9Y7q+6oYPXiMbQNktFlGnPGz\\\\\\\/COMyVIFS6ArPB8eMp2bOlFC7hwISA8BVlE2NHi8mS0wUGnZcrh1xglqGDWWoy+g+YjWC\\\\\\\/uMhJKBnP8WqHaFDsEAccS6mQccr9jGSdE\\\\\\\/WOvFxAXYxEzNoMLAoZRKKEDwflyIXlviJls4jqviAMlm81U28nQHJmVilBdRVrw3WCTftfzgTroH8utddRZohdLgGbVUE4PyggUW5wEvL5znny1r\\\\\\\/ED+oz3GkztpxThQj2Dn8brCZGy886QxtMfswshz5wrtpz0dY9oMuztWlVf7DaZZCk5EES9dhJwmAKcJHRWilUgtV5N4kSbaMrd368uadgjj72y\\\\\\\/r+\\\\\\\/ixt\\\\\\\/Cdsgl6LZkYC\\\\\\\/eV+oGxZ7vpcRToXE0ncbjhPJxU6JuqmPHTLSvTI7mVG61uqCvsnZ50rmuRDLtwRi08KYV\\\\\\\/LCfhvNOU0owpLY52WBAbdRA9qAkpElZiUycFVOvQkBNXBLKLel8bdXy71xzpoYAE\\\\\\\/hrNIgj4RioUBTUYejmL7PWliqBmrM45\\\\\\\/hoESSITQdXu3BnLd0nbMmOcY58yoDoWzT3nFeBPfIL+FNu6t+23Ps5RuCPA4qzzfh7KziFFIt30BmXyT7iIo8EQ\\\\\\\/4L7dLqczUpQ1wGC1RkXtKVfmira17hTkYLV6V8uQNdsUkUqZBdURgL4bLrD4KYT6AAKekGW3nu4BnfMbEV+tTpp3Vrf7iu\\\\\\\/vaxTR7BPRTjvo3wSmwvcgf0nHohB3vqOWcU7rUXVrqO+0cD+8IsrWfLR+sNxOhDjsW+3zsQFuOs+OnyrHJ3Jq\\\\\\\/oVXOdKVs1K\\\\\\\/xNLDM6fxpHvhVr9+ejjPlaSak211hbJ313MduAHU\\\\\\\/qUvIXxni7czbqRmJTdMBauP1mnSqV51o7C4FtLygaFwxWzVv35Hx2WkdA5LIR\\\\\\\/T72EQSlf+eZUPi13fksBR9gkFEltCX+\\\\\\\/0gQN4Mqn5bueDNE4bRCQQPJfMewIC46du4a+GY+LGrtD9I3eQd5xU71yqKcC9pxKWvyrUWSVwHIw1WBVefChvL3sdtDdxeDZpIWvuXCXavtfiMWhDuWGfZH01qMkX4T8wjNPnJ+0noMWup55BUB9n4VgL2kCUxWbh4rM7h5OC5TsE2j58EjfpxzBPG+6vCxGH8la8AmE8sfNp\\\\\\\/kr2ybbL1QSHUioz9eK3OjD0ESFyAANdO4PxrLZ0qlKG5XLmUhPFMaKEZPcNZ3B8OAeKyI2e28gRBcGC4zATX3doXnpUlbMG8xt3GAyW94yQZ8eGGCZqrbyAEMAUrYY+LX8mixlhyTJzHehIKg9b5WcnF6hDPjJENg8U7Ibaqnd\\\\\\\/ciNmqbwn\\\\\\\/8Ni6zcFJB0ERoa9GJf05J3+XSYenai+WmCL5yz\\\\\\\/3ep4PzW+sVmqoGciBuM4b6P6PHZDvgXPymtmORBOV4Su\\\\\\\/pqO2rwuLYdhLtVA8E1sefJWRlO32Eyur8\\\\\\\/EPMiwnYyExoKS0EuWrQXfxtW31Y+ehJ7\\\\\\\/TAPwuhSNgG\\\\\\\/1PjRXTl4M0Iu8SkB152t9sGnVmBkz\\\\\\\/SBxAzzlv9chTyMraFrwOzi5cTKc9gOzPUuD7opryYy0TeHFlaG8xOKvtXEkGhvTvgp1OmJyWVZ3RKvWPS37VJGmzhC\\\\\\\/zqW7YpwL94fPAjxxznTkNADVFwh8Rq0eqI3Lu67nVJYh\\\\\\\/eQyO3fujuZwH+kReIsyqYT3Sl+NBu5x6jo2yJRnH721tGt2DeB78bBU0SEQxtD+XB2o7uID\\\\\\\/WFgRY67B6hFDNxYGI1CnoV4RMRGGaHFZ4FLE849dIH+LkXl6uOSfoqEOvl\\\\\\\/0NROn\\\\\\\/tmsmEV301JZYFTDl9cBzYDK\\\\\\\/B2CIQ3HVIaWA9+T2Syvwkh44\\\\\\\/gW0dko3jixFe3vvgGOAc\\\\\\\/6rse5JzSDBi7gdXhorHUuv1lRGOHvS9p5OHsmE5dziG7HnEkds+f2hnBHdVgjXZosZR89R4aeXDOiQT8skXyn6vRO3y\\\\\\\/OtF15qUvBYnOSGIc5XX5Dzh6R0BkziR9hxSlZ5mWdKzNHCgC1FYvqNu3m+TawzNAV5fGIiN0\\\\\\\/gMJBbXVaASMAlPDd6XE\\\\\\\/CQqHeGsMIeZkBbdZD4w3QPOlBoPM\\\\\\\/onB9qm7uEUKc+uwjCX69yEvUB4SpFq7Pp+kr9bS89BG5vBT4a+Ohj\\\\\\\/zxJHdxYM6kmj5FtjJygx\\\\\\\/MDgTbAJBcQj9BX3AC\\\\\\\/lhKjOsUfS7WI58JanDJJQ+PPxJfpu+fYcwOIFqDi0g\\\\\\\/lwV94+hwwUmnyMP\\\\\\\/GbPKvP3A\\\\\\\/UUpTa0uP2JBDfjS09F9vDjmAF810p0LdLRW\\\\\\\/eTMByXMWaJABj6RFCJeS4MxgxlXrvwFvocQ\\\\\\\/07jbOUA\\\\\\\/1cC8bXscJT2EQQWkR+glMwED4v\\\\\\\/KVd+zho9xuCY5\\\\\\\/PA8gFc9HFN1yatB3mtpMKgVdtQ4IbiJmyJyVjHG1WuAV5XOqnLgTxskmQSeeSYTHdqeh1X344aIcFgPFUwGlswybTA+7JuQb\\\\\\\/UMylJ3vgSZoSyGmUwQIfbzS776yfKJbNCbs2X4wLRQsq+eO2wtrinYOot89BtxIwUuNsa+vX5Y9QPGEE162ZjYy\\\\\\\/F3hM\\\\\\\/aiD0j9YjTamfBSfdJ07o0me+djRtGNEX29hudaX3SHcGsxptwQayKHolMCXsLZfhrDbiC7yasvlc+jcjhn28MGLXnG+4rPgDP7PimckJAgtVb\\\\\\\/ViAELuRCd7uOXv020Sr0hdFP4tD2Ue9PVa1Nd\\\\\\\/zcDHIWtlQ2AebbKJJzGRD+BEXpYTDqyker5URd8Ty6Hv8L9SFM4oE1yOV4xMjaIx+RDCJc3rO+k2muVVKcp7m9eisb1cqPFADuglfjk09UPxYwCKBZ0Z3vuIL\\\\\\\/ompozJMD0EDhHRhQzzPUE+cHti\\\\\\\/cagAXF5LAMhs6nc72XuW1iC7HJhdwxDTaAKVWJ1DOWRNCE9bGEekVusrR4yR3EOvEJQwA7oThLd46T4zc2ulnI6aReK72anLpLK9oGFvpZ+1Rs8jLIkM5hFKNQ4sLla4DVAEP8DM9HAh8bGUojFrA3IZWcabrh9KD3wFeF4geH5Iv+SYCKV46\\\\\\\/\\\\\\\/IOleNtd7jjpwIf3mvcko1Sqt+FfgB6+E9fLArZRvgdKmAjOU+RD84ja3g16GLFwNfLKcxvb\\\\\\\/Fp1UxE+JUD\\\\\\\/iU61p2P2ZsHb7hEcCfQW\\\\\\\/lqVGcuGFrwm3Vl2TNDJRYbx1vXpVZSmYLxP4VC3LQk5s58qu58yQ4Dnv708GWWI7blIFGSSKC7g6CIP+nx6xQyrOc7QRZu4lpkCaO1iluW8HtIGHocZfpEetug7DWqS+QIjkG33ihrp5KQTCqZszzqgs0SX8veP0D0rPOdVVi9bLtN10YFn9I+0mH6NDH\\\\\\\/98b679wcVwnMrqE3IE5FJPGf\\\\\\\/BELA028Bq9YMi2\\\\\\\/a1W+6kHjaWBtU49zPXVYMbgm5ycSqdHAcE15rd7lMb5H8ZWQbaqIY8d29ZffKvLOTya9ekRZSzS2thVeF\\\\\\\/Xbo+wAy24+ZoVfEsWDxH0s0j\\\\\\\/02Uf\\\\\\\/ncM9MlOa+CJJWqnF7wI3wT3AAjFDD9Bq6SjMz\\\\\\\/JMtU7BmY1whIAkseiUakI1YTLKAE5S5A6tOh4RAZ8fzq0+KUGolXBzhEIEl1RxRHC25EeAbsCmgPnbC3SysOOZR1yfwQ1gHReWIEzmKke+Fqj5GA+D56C6f0SszrxlassjoDoLSwjJ7j0oSzDVKFqVra6dt8RN6BLNcxZ+P1aITQerWn+FfPYho4AmiGeuoMgM9y4o1\\\\\\\/TFrTmt4u84ZWyQX2FEHb7zlIud3yjmqJ4PLc8+QvRn3uul5LW0WD7fvGM4r\\\\\\\/Ll+27W17q3kWoY4J2Ff7LRWg0XAfSpZdcLsp7fnQkx\\\\\\\/6kVuJf6YMvPYR8D81diPdEUXSBurFJwYWhWtvm\\\\\\\/XncYWsxW9Qh+glOzX4tG2sPMVbPehrDXcmBRrtojX6W2DHxz3it+Q4BbJOyXs37O\\\\\\\/ekb73cghPLCmMNs8Wp0niziEGIk0fHAqKqMSciv\\\\\\\/UCB\\\\\\\/Pe\\\\\\\/sa\\\\\\\/GB+7jYkeDTlWoHDtlF+a\\\\\\\/twoYAme+BbTLcFTUbqbTfFJr7gZERl3kMT5SfpzG3DtC5SLZSQ1neS9JBxOqA6Tg8Ev3Aw6TZQ5hI\\\\\\\/3aeyqfRifBYlUOdUugjy41VJdLPdGjWy3SVgWhY5+K+cRmkfOl9iTMYFh7EJmYAE3mVF2VCbl6YzQlAIrssMQu7SeWILNGdPemKSOa0479lowbBLxm4fMhUNB\\\\\\\/+zih4PZUBMKYaGRZkmiFtPH1qA2Ts30iWQppdLR8uEQjwu1XINJl1lrTmNS8YmI5GgYJm3Rmmm6H2yRg9taMrWdTLN+Pp2TwEuX+\\\\\\\/49HYsVnPNxPk9J+0bK\\\\\\\/kx6SbIOMe70aVlylENeFutKoBcqp5l5zh0qYJX5K3Z6JMR3H2bEIsPpF1BTrrK0UYDcJYIIi09zqgR3ReyICAFY00kfjL6tyX123O4rqIwWs3xVfrjRAgF814AoH0Gol7By9MiIeddzyGtbnkqw8Mbd2+4E\\\\\\\/d1fbvsRQSqpNuGcwLuHH+7y+4RSaWNWo3PuHj\\\\\\\/uuJsCu1drnHvXkWo1kmck8FoI5ZckW3doKWWhVT+XRdHX0bQYEDu1mwwPeWOm6HVVbWQBWsCuGTbGlYqu9duk5RHTF83FUJhI7tODDBKuw5T2199caTsRi6UJ9YkrUs2FpXO4rFBtvk49D2ZXTd97rtG3YsdjFt+tOkn3POGYbUHDSgxdCHd3v2pHxRq+bbv75FuH5H1pJc1VVmk\\\\\\\/40c5JAkUOrnjwsMMMaxASirXutbOzkF9w+oMq7D9TZkWybTPGhu4qV8+4wbhRjaiZNb7upxq+cYfsnKxVNO5FOgC+EfzGXCtN5zK+8g5glHnM3IhYYLHGiHCZEW4lPQiFk5RvsPjzSdSGuk44uQmyOUTE9SbU9bfeIh5HbB+OpAKAQMait1\\\\\\\/p84ydwntOUPzFYIJaIhYnfQztzbmVN+oV56g\\\\\\\/YsuPJiMOvhnBEZURkl8h3MoaBvE31OezCCNjHU9xhGynBefWYNJzPZAFhK\\\\\\\/SLKOS7L3cnNoIOHzkT2\\\\\\\/Tro9X6B2ngj6SYeRgc6CkxKU1vGcVYfa6r87JgnsRmqMutsSgdbrKJDuSp\\\\\\\/s1hhkF8uAh3P2YzSWq3gS4PDlgzlsxauQAtywCmOSDkfxBEc9egUVyyvDm4vV9CF13gwnmUDAz7xOQxnagk0ynm9ir1sfaNGYLI9u\\\\\\\/TYbrwDfGjyjo+qm3ZNhoc6iQblqRii35YGexDj2qd\\\\\\\/MjwHOS1Uea6XlSYudvb2+RDpYSmyp7xPBy2ZVHX\\\\\\\/rY+bZQFPBTTbJM1ekv97y7oYnEWTDpGdB\\\\\\\/9D93MWI\\\\\\\/I2Dume2MkfBt\\\\\\\/dOc7HSUyZyCpVNucrSO0\\\\\\\/EmPOofZoNDlWTzkyEC\\\\\\\/oQjmKw\\\\\\\/i4ByBpjtmifk6Fg4nDLYcLMBXbK3EYfMj2GgEDi0MIiLkK5ishx\\\\\\\/nv88xh1amehprWJNIHxXF661VOLPahhyj6\\\\\\\/kBdQeto7Rqr6mORuOXZzyrHHoSUUlZoQrEpZqLBGFwqu5gwLSgi8KZp9r1gHwQe2tlTigu2ZQ1D6lXjteqmUoQuZq8v2cXGfYJPQ\\\\\\\/hDIbFiybJjnNAw2Sv6YdP3u5pU8i5tzbZlSdfRYGNI8k9enzARhUM2yh5zzh5QNMNuk8c6ZvEzJTO5afh4y8j+X+bieV1bZk6f5xGVjHjvI\\\\\\\/sybO1w2SmLOSengXBnk\\\\\\\/0z59i5p7PBbYohM6CyYD\\\\\\\/JlT4xj\\\\\\\/D0giEm2W3HfDU2Acu0TMxJBIu2PvJmebhlJY\\\\\\\/\\\\\\\/nyblfxSwYhSDxSULdvFrP0XoVVgctJm3pdlwBlPuWoTAAXxu7v0rMFe5dOaHJU9JjXg2v8izUtpDJBGTGUJPxME+DVG\\\\\\\/z6LEdKoCCXxoW+QHHbT87Rq\\\\\\\/D8RxBpZ8TzaHQs6PWGEyrZgN7jycJ3rBeUdbgs11Z9jbmNKtnhnVWiZmmUfaI5N91RXF9ydI32aVu1P9t8nSpqEHH3aHZ9JWMSrliQ5F623rQaGfWmsqpiTEp8ofTxkLjl0OTAziXDCe2UzD7+FxmvRPd9wxR76m7N3aekKjj1aCS\\\\\\\/KNUh0op1b6DZECnLVaKpfUh5i9L4Hp0sGcRiSqkW22fQpgQyCDFkwcN7j1+XZmNjCwdNC2UPnwwcFywr\\\\\\\/noWP461z9eChnWfAonx78ypC3T9BqFZmQHdC1r110ZsDf5EDUGtdpSXjmA6CNNvKhBqYy\\\\\\\/zMwl6sP+NFwe0dzmiL81+\\\\\\\/Mm9gCGgvOXJ\\\\\\\/0JjHwlEcKcO7Qu7YL48Tr\\\\\\\/VNArosVyEurmSW9LfO7efHIDk7bJsTvRKt+Xhy42+ZGc7RLRDpUj9UMdME8AOg05gCkjbaCpwUq22WVgx41RmQMnZhQZIG+mftLuYm0D0UNr01LRP5d32iM3Hu57zhpbak5wnEzTY4DZhFECH1Ypvx\\\\\\\/iSHU7UE9LMmLhnhKqQ0nVTuUN21f\\\\\\\/09VH\\\\\\\/PX\\\\\\\/zLs+iBHB9B2naBMGSbx05D0391FjAEXrNaDRpUVgbF2Ir3+7704351BPUMSAJSVfmEqR7iL75wurfzDR0oHDWXVQAWdWkVURFW8d7Oo+SgFA\\\\\\\/A786O9hLG6mi\\\\\\\/YAdMnKSqefTU2RHynuRJ2t3p3LJ\\\\\\\/1i5WTGpvheRsOlw1wsw7Xr4ppO3oVrZnC76ImC8s742bSss7IYLGTpIcyR17nPS7gIHzHLql8dIEbt3BWOs38a6pxkLEqXbk+FsL3wxBf6GuTUBMFruQTYlfvOWYY60Hs9bdMeNC95lIcwrDPq26dmrWcZ9J1a9c2RRVO00Pl0bYecwhATFYq7nwLxOucqZa6td8XqJvvxk0Y+RbwoX0q2tU3Dxp7j59bpRWDYcyS+yqAStKVaTDNnPbrnH6Ck9Dxb0r6jghiveDMNhe6Wh9gbgY54wZK1AyxmWN9BjgKUzKNgMGSqem8lsp2yZR\\\\\\\/lhKxLslq1AUyeGX+oFJJ\\\\\\\/hjc1cP2FkMGm9Ej3C7+ClZrKe9hYdYi1HOtGFG1Jzg6GDGIFXRpWaFim0BWuqb8+WrOUJU7FEhKY1gZiVs\\\\\\\/ewul7D7bbO9zJbOLzTnyPe5qOOlOP5Z0jPhyBIgHzhweXk\\\\\\\/BQ24n6hYOfnKwO7D0ZegyrbGZ93150nwpAJgnyh0Lh6Biug4tl1St1OwjTMDnDHn8YcMHdE+RSkqEDfmJAOJUS6bbhOZTds0CR5nYTprwwQbLxapZFAB17pH\\\\\\\/zf0rA32hqQX+66MziLTlDKgrrZva1adOGPI5M7HI8Rs+eqVvTDCmhATHl0\\\\\\\/spZMUYY63hRMbeQd73L0WPGPanVxU0bl\\\\\\\/UxdmpYIsXgWvXZmfnLbHtPcVYc6DALz1squTGKYHk\\\\\\\/iXTP8mQUg4mAXZfp57wDTlluGsP6Gggg9fCbSiNWjr\\\\\\\/VNTQ8KiPv1aeWYh8v5RbVYHJTI09azDipYB9AUkyjE4MsGmv+Zi54JkQCWoaLM\\\\\\\/QlJ7s0Gz9Q7ifl7N\\\\\\\/U3Kfe5wGs1ey\\\\\\\/rdlJei+gzWKRmOrne2zGEZt\\\\\\\/tn6x4VYJ\\\\\\\/dC6+aWYoZGz1CPouzGfzxwNRncMa\\\\\\\/9uXIfe\\\\\\\/+BW3zt5LVCrqdLPW8Chnl2iwDDZFfI2qtXF4YF4pWkKXDWeEpOP+ivk7Jn2rcigpxW\\\\\\\/I4LSgwcFhnca0zxaue+Aj8V\\\\\\\/Ivw\\\\\\\/uDxyo7VEXcfb8RvOjj5RS6gPoTGw7lyeVe6cV5j2ZlKELiNw1abW3OOmdEROy6XQ+SQhADZOkNucTDXo0toYjEXRQprXcHhyzbzy7sP4ZQl\\\\\\\/TFQHMmJSZrR\\\\\\\/33e30hLCVBo94sjVe64a9mdib0LADZCXuZYeyQYVkysgwfb0n8mwGsiDbNu7BFOxKptSxQqybdQgTYS7z3E21aip4Uwpp7Eg48aF+kH4+op1\\\\\\\/5ut2kvw6unLrUoZ5ILOgOz4WLvO+S7+\\\\\\\/f9qufTOP36tF06V0C5TnN\\\\\\\/jGhI7BZVoY+5dJOEyfMgiiCCFYizug81gZuGKuIYIR3+R9evaFf4JFvdqFRxJ5LNgNtn3oi\\\\\\\/IXAms1uwkBEN7FiX\\\\\\\/I\\\\\\\/HsbD9uhZKghwf81RuEWTlO61noekXnmIpYLLpSn3PLJiS6McdqYiJgEKBlM\\\\\\\/gAaKhR7\\\\\\\/aeYFPH\\\\\\\/GrsvoKi+ekoqRbi2n13XGBEmtRzMR94PP5quf8a2t2CQl1RPQ0nE3DphjLJw9IIiymRlfi+liUfJsBbKl9aCOGaKn+wn7OgnZkkhCW5CI0HG4HUSp\\\\\\\/cyEFU66+i4YPT2lZwj6rD1NwuelIKQU+Me7H8EtQXse\\\\\\\/p2o7mOvzNS\\\\\\\/KnWQrBiYdZJWX44\\\\\\\/XOlPd6wiqHSBZK9VNL6LyLyhOgZYS6EqDF5FzaFfGT\\\\\\\/zxGSNgTckKxStxCxgT7Lm+WKNmRkadjxbCJ61wjFRvItGqGC72HY0NZdJ8exQNs7E8PBN6ll48P3Em8syGyxI85D2c8M2laerBFZeWrpmX9FfEymHLk8duHluuU2D7YbKtnMDDh\\\\\\\/3\\\\\\\/CtYRjE3fdwLR5wfaB\\\\\\\/iw7kxiI9Y3um8FppB+qrQysdd13VBBTuspec+mUW6zWf9XGL0bjBNhkSJ07bJbwF1fpdIzuja23WeK9BR\\\\\\\/R\\\\\\\/+r7xMEK7rwFrD5cJIPL3bACGWerCIGw1J4krFvSt2HPpxtmxZTcUpKybacKpynSNKTQptx6oyOSf8G5wt7fITwyDvHwqfAX6TpFHtCDS8q2++BPCIvViz\\\\\\\/OVsby4sOinGnDqbld9c6aFq8y5oVrbpe\\\\\\\/oylvgqwTgXkhUwq0nhhbCFR\\\\\\\/P7zNIt9xrUMN52PYG\\\\\\\/pMErG5i\\\\\\\/DlvqZE+ugZptE8yYp4jDmPuCngoA6VA5cy5ACCvhKIL8KrPFlesIp9cfYR7NJByhLq6RKRYiFP9MOMP6hELPCchr+89zBhI+fkMhCrGpnpEksjRErvPSZ2MnsHvUJ7bn9jODvmGrkzkWljmFjwFjo\\\\\\\/FUBK9Xdm6UxNj1xFQtM\\\\\\\/YzMl1jGY4KREIh4WcPJfRTgBK2aTawurCg6VHtTgFEGw3CEL5t5\\\\\\\/GF\\\\\\\/OM3jLoTGO+PqHVdzVfnn8\\\\\\\/0IprriUyWkiJ06CDWrnwjaaKu61pVf9lsvXGTTDzMOH+tyJo1n6rcSHV5UCGQ\\\\\\\/6wnzsW9D+IiCKmMyFmfPLhiHMoZA8wAdx5sGL41\\\\\\\/C69sVMim6WEU\\\\\\\/n\\\\\\\/pEG3Ovsdmp5m1kYalKwaVY8Tk9uadlyAPD7u+ADpx2ysmeKicF1\\\\\\\/80fU2e8h5sYDLOdbfLjEtpK1XQjOZLe7lPkhoIh5AZGFeLgSGtU2hwSv4+z8P5j1SRGDuH2P76bmiXTRJzauwxhehrsBtloH8rUqECVz3B68fqD1vhaQXrhjpNNLxYoP8oT7lSWIoBieYSABO8++\\\\\\\/dwSVZlBXlhevHGByWZO+fI2qeNHkrGaWPB0GGEaeY2sOwUiG4jQ+Ze2i73Gdoxcv9oxz4ZYdYu1PGETyKCgA+LTyLnNIzzCj8GogsT4+lX2oEDBTDtSCov\\\\\\\/\\\\\\\/6EjQKJ+t9mlCt6PRybd3K7JXwjmuqgo0QeC2+9Xv60t1YdMRO\\\\\\\/APFD\\\\\\\/h9+f3oLeprGsIb6JNM8kUYRg8JrFUrOq+hwdaY3YnCbYnhel+6gdKUYrI6Y4JapXKDHOx+OwUemobreU66p169Ns6SUWCy6dPJ2wrBM7uS2HZV1U\\\\\\\/5mlYkLujTfR1cEnRK5GAwR7kRHirFIs\\\\\\\/L1vi7BoieVuoUWSTqNz5UCZcBRi6PAWG1ATicsUkzzQg\\\\\\\/cv8nl6XIMEUaTkHKLD79VXxY8dEmJFHBRlI\\\\\\\/G19QfnrpaIl7HhTbt86G7vn2lzsbC9AZgeJRnTXhrPfmrpS4qycGMNYLF2yDnuFCpeDj+P7mUGBiAAH+I2xkIWo9UwNp1W35jtw1xCzL3qIDbkHS2CXWon3577l8YdISOatDqPDN8nwgcH2TxuKHGG\\\\\\\/0XAyhZDdYVGvvCJ452Y78RFW8EFr9h0p8\\\\\\\/m7SPnIuezMs7HOP+LHserFv9HvWNZ3SYAI+bVYl3gp46YgIkKvTswVLO\\\\\\\/Fy9D7iYlZ1PT2rSfS5SaSLoPynWWYJsl9hBJvlL2J1jnWu2XZfJGPLehBPHvf+\\\\\\\/VMXBHWPWgmI2XxcyJsHh++0qfgOeiVgDB+DFtnHgM7wPZah6D8O4DguxjjLlpXQssSCVXX3vlnzwKlQY8Wb7Et+ycgGiQnO5PF1\\\\\\\/lKAEKbFrpc9SqTYfrGJn1jIsDqdkv9+7\\\\\\\/7iFM7fKUG6OC2RgfURHq\\\\\\\/o2GOGaWcoJQpE+HipgWjGjY9b6Fl7NwhgUB5lfKsUdK7Us1WC1lmrg+v+9ATaViFKd30zkZtgCoHEG2OY0x0uvkCBI0pMD8Ei+6wQInMlygkTBTqk0SWElDDHzlgPmVWHG59DS0Ute2OwjLXMBXFeAuvkzzGswqBA9eVzsAW6eIaRKGkKL42wNF9jlQiten0aO96iA1U8bXC87wfQjm842G+A0MYreXc\\\\\\\/\\\\\\\/a5izazV7NJuOniOdVhEo6JGtBGTmPXwIx5LNsDswkpN0Xyy5qXkZ\\\\\\\/kcAcCeHYslhjD6DP1paFNamJ6YOxO8GSQS7k7FmfaZxhDtJOm4aIeMDQIGODmdja\\\\\\\/5X0IL5uieCj3qnYoLQGFQQudl3wdLS6LNCa6VvFbZfnQEef+a5lo8\\\\\\\/JnCsFlR42xVFp\\\\\\\/lAy6bUj5Vhqp3Q2g3AzWhu3lRJv+UnGZGx3DnzsYcgmFhLU5vedjZx3+s28IZmmV2NaEJDAaJIEfZZlutZoWrS8aAUSU82eaa2\\\\\\\/3B\\\\\\\/9Kn91h8Yto1H5BLSyUrP9z3nFYr8Ufgw5xmzwndjdntBSV6KaXxG2HMIqZvHG8vwNZNJKTecBEYLBAmJDnmMmXgY1xU8cSbvrhiqUvV2EsTunp43kF6A3hVy2Dtq+ii3cut7zeuw1bhNiQC2MdziTRnpMH1mD3\\\\\\\/QnuEBm69HGspUpQJqDOI+KIoWFNuIcjvQhZ6zxETlNsgoLMLUgWtuVSg6nOahbOepnUQ2cqwjDKsHfh2VDwmf2bN3kbRGLap2NnPYxHMUzPJlH1ySm8j619UlcZAp+ZP9EU+sKyd5peq8FRrxDRh1PWUktHdeAM95Vuqp0hbCuNW8unlYw53zZAAxUGjwVDTFmbmD6U3efHVNCP5a8CtUj+1ROQuKh26VK5sMc\\\\\\\/8noFFC2gheOerIf40vginKLa22Pj9YzvfmZ7yXWYKl74RCEb2iBF4UPSwspxOtE1V9\\\\\\\/4++h1m9amdsdwEaOvZgdmFqrncTO4Vg4JHL4CZ3rgw4Osnc\\\\\\\/rw4vbnlzYaotY00hnIa29xKhCAh1uQFB5mVe5XgujnzesG1E\\\\\\\/0KN1B6ydLCokfZCZxKBHb0tLnjAyQ5B8VjaxSacWYtK21jqKDvdc\\\\\\\/bxp8pIf9lbK9LwY84U6WYobW\\\\\\\/tLxXp4w0kg8z2mMGwV8yh6E1LednrssRoOZSSFjsgy7wIcX8Mmtqg\\\\\\\/HAashH4znz8+zDuuBEm79uVuVt31ocKkKpkqwrL\\\\\\\/6MnL7EJj3uwvTRJxtHAWBPWThCsNTd0ljy2MezEDGv+0wzjfyt1tNS03Xb0T5KP0BUXyd9OjSbHG2EyQSxBXESUyxIvuM3bwDkgAZB\\\\\\\/SUNG3pO3ggcQXWtHLcy96pPtFC\\\\\\\/c7R6VcLwaj2hECpWCU6QTpVwdOlM3XtzwBBHvlf\\\\\\\/AVruUMxlsNw4JTq0Lluxc+q+1PPKTk1xxJvdPbWH0e\\\\\\\/SPNUEARAGAjxyDti1nbKXTS0lh63MFQZlOpHDFYQVbKPEWBCz66zejOYexnpGkQ8u330Y2JvW5KsfHHVZXwWc3vpawJ1l4G+DGzXT5TqWB569QK\\\\\\\/76O6sCfGq+\\\\\\\/Vq9M2CgXKBrbF6KuH6Fru5SsOJsGpkOtHY6JQko6\\\\\\\/odcxFevA\\\\\\\/H+dJYx4lwIqJupDPO02AifHMllrhZq0E6X+SBvRLTTQa3BSk3Q281g9QRurLj9PJtbp+Qsjm7OlgzkF9nuVEDHecCemIIw8lTI1PTvwrz+1bqi\\\\\\\/DMMY57qzaHnZwhh2WbLgcjA3SwdlNFEaJNf3SO6XZI9RlCXvcxPoeJ5+GB6OULKxAvwDlz\\\\\\\/TO1Luv97Z+7TJkRzHoDuCpYwEcALujozA\\\\\\\/5dT57J1NNS6Sbf2\\\\\\\/Bkr5XPZTat3BfmNbBqW62QH3nIv8f7tLTvbry3L0KSDiKaWP+yQKX2pzx51GZNwWtKbMyhZXcJL1pVisWDFmUe0YJW9Gh6S8B1cQBJ8BaBvYSUXUq9v8LzxBBCeSwhDYV+JmnliVsjahSI9zxFwLKIeCUGbCB7\\\\\\\/loEjJDef58jCPcL80dEAutCzNZ8oK2g18yKdlISehT1Gnd4wl+LDbtQLegPpH378AV3WhjCxwahB1rVOnR\\\\\\\/YHHQn9YH7KXL4\\\\\\\/2L4XeYeUr7aaszChK\\\\\\\/\\\\\\\/r3RGUlUvWwC+LKlSPQhUVS1q70u8UTRxoruztWDwcmcQuZxXnMMoVmkkEpdqK4cw+v4O+QB5aJZgTXCe6TXxmN0ExCBT5ud\\\\\\\/T0ObvdtuQI2cXgJ1yx9frcOJt7YNRAuyqhRzb2Ia9ktia4J4bWy1k7ARPvZpqR448Llw\\\\\\\/ryHVOLHP4pXo3xtYnfWkjDa4MwquyAa+Ip+wwhocQFaQqAdrezBKHoTqfMSzrWCHsQxXUJ0fRxKd0nnccunjpMZiaIIFT\\\\\\\/1sCv+kwKSIJLoH2S4ZVf0MwNjC8o2NXsDLHl\\\\\\\/LllwQn1dk9pthpdrUZ6UHFCkkub14BsGmzJC1xaF2ajq6g87w6W1XneBojPSvg1lMC9zSpzogX9PEgSTVDrP4TXAScNXAZGVHqntwT10HtX4dnNl402k2YPSy\\\\\\\/Kvm2dDhreIb421vRIL+6SuciL2L+4jFT3hmn+EN7Te5NeYvXweMyqmJTBehOwQIPbHtBmom9X8sCsE9JrUZqUVnnKX8wYoks5kJO9pRtjkwEupmGHaCP7hNTnx2kSuJgl8pwyFJ0cC2OOu90PfzM6xntVoSm63tsXRCVKoertSZqH+PeIp1sIpQyui+ayAhTGTL+pL0HHMyv7iYr0uTVWQkFUrAdwW\\\\\\\/5e\\\\\\\/qelImqKh1unt\\\\\\\/UM5oGfnNrD0izPWa28nC\\\\\\\/P7MfFHddT0rTJz4H5gamrDs8cbQ8\\\\\\\/xKu0rPs5OIfY80wJcwdytdspaxg5WWoW\\\\\\\/wjMGZWGQh2gX1NyNOy6trMslgypfQzRSARx4pTsoHC+kn8H62Uxu3W2gEL8T7jk68UjzeLYu7vq+CT7\\\\\\\/VFgaJOsFx44jSBTTJf3YbfYd3hma9pZ4SgKNYb7JAkChGF0zB3UFODBxxp0Ll7AeqcmwZV+u4Bru72lM\\\\\\\/eK6QZERakZeHKAAc9ixaLhOo7Wz6N\\\\\\\/gJlHG9fVNz6KWhDI4bDS+8mt8fuZvVSUXoTTmrPvUN2\\\\\\\/d7yhBVSSY1OLxGgYNizwBlwdoouS+Do2Rs2OnHtYCfqpNospv9yJdC+qh0mgY+DD8oM\\\\\\\/qX9FpnodsTDMRerfseIJqwvhFinzLswhGESqaVn9Kh+Y4h1Mj9VZrdYASpRmkUnaK3OmqMrItffwbCB8FGStSRyRAUEZWM6buFTd48lqSUtazvqUI92JYNSBp49a3Ex4Dl0Qbw3B\\\\\\\/AJ\\\\\\\/tT4rdiwb9E2excjI6N39hS6IytIJbG7LHxg29entTbirkYr+W\\\\\\\/WcnnhWx41odZkgVTlQs\\\\\\\/Y7Hy44MlsAh4K+m1n3syZfyRTybkBm\\\\\\\/xOtNOSbZ02wedmHAWoJvTsWIHSdj3s2VzkArCfw4iVX2vuBExIo\\\\\\\/ZAVIOmdeODWfJDPrJwpyCHbjE2vzBMv+IdWkJJ0yDD2fnf6TrfNXV1h8VuK\\\\\\\/yd\\\\\\\/ssD7JBM63oWfEOFI7MDgbea+NenNIpEqy4IBB848oRmFgqsv3U9a1ArpkUUMBoTTStmlq3sJbGUsbEOQXOAvKWOeOvuag8xomHjA13+t1Mr9HDHlX+L3hZYKfies2XHmCW0StXiRX5sz14HOkvSiG\\\\\\\/NPAs1VtTeSk5PwDUeoYyFVbCbPiL1BNuk5UlUCTmWPlTkNCWVaj5v\\\\\\\/5NgkiP42Q9fHG2WtSvs8heN0rVet6UAKlyDW1QmJVY47qhKehfB4RjD3xuox0Nzbf6kje\\\\\\\/\\\\\\\/RCfqUNTKcpATWwExEly\\\\\\\/LwvGWtDdjTI1Y18kdShGxLVnOQicPUXFblEf4ENj1fnNP\\\\\\\/LDkFpiY9p2g6\\\\\\\/ifKHEGbgeVV\\\\\\\/AHddxKWWlJFeeP2F5oN8X9KKSBQFRZmPqOCTkwfBH79odZE6fOx0UsTLl6BBfy9fuBVetHPHmcQW\\\\\\\/Fxvv0\\\\\\\/9+PwV5L8\\\\\\\/um2VdovnO3Vm+aHpXwCG58gSikV5pTgpOgwHqQMVwo2O9I8+H1AHpEgJmza2QCOgCuuDOzpdLEDz7kb3nMKoUvdtY90gXNyt600KOuJ\\\\\\\/sg+\\\\\\\/uFE8eKYNaPC821utrOeQ8xyj+ehSazILNuLsr4yyk4h+OSfZwevXHYeNJ+6LqI7Bo6Xlzdizaf72qEOZuWaXPev8qndfFYrDhkVDNJwCieGqeRoG8JkyiwyYB+qhhtuu0TQ9BB6m2P+tWD\\\\\\\/oIFNeziMtkp\\\\\\\/+dRz\\\\\\\/LNwbAhESc5rFXz9FUU7FNa34cv8wAnfMBqny\\\\\\\/DUmPzsRsbuY+GJAbNvKsd5FSnFMikAIRjJ6rTVwv7erU51yh5SCFQaV2X\\\\\\\/\\\\\\\/w95ZOa7kVfMtIArwCraGpGkoLhMg3owPiKvXEBcue4bEU5QMa40WbDELT3cJaKnKl0PO4hb2qktYHn+7BKSNsVYbK8BU5F3o8Ji7wDIMEyBPgGbI2efhdiPCwbSjw6ttt6x82drUxPwqPuz7imU1vzhDVNWq3p9dJk6LOsEuL8VKkkS2hbRAMubX9yOst9f\\\\\\\/jXbfb85oEmBXCju+nNuVhD\\\\\\\/AVgHa5nWK6NqOKiOhEZM\\\\\\\/VXu1BAwqQyzqCb6OAorSv9G8SZFLEsbeQeVv4rH9+8nryb0FLnAmszm45o2qTefN5nKieTBwmnDwAGkVoGrKcD6pRJyMfHKt3r3Vl1A5LRqUr6TLAYFiNVSBJK8mQqnGVLPLtC1R98\\\\\\\/bym+BNPjXjpCXLLRIZwLOa\\\\\\\/hfsWa0KR7qSE3dIvCjMgcfpO1g+6\\\\\\\/5SwuhPR9qTvp9JRm\\\\\\\/NZ2\\\\\\\/nvWNWUONKNIWJun2IAMc2BuXHp9mJLZTR\\\\\\\/mJASDXNvS4jqwx+ofMEIi2qA9zO6nKI0b0q19gWAtMc9WlJfVvC9KWUWQXw83EKGW2dh7jbfiWD+ulMfVQ0hfmnj9uEJwqF7L9PRykhn9GrqA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022uGrytBTqffREQuN3YOE8Bg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002270e34d6a639dce8e84bf1f1661240093f24268928816802e5b1f86047e924837\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022nPaYfYr4UZpLDa53ggw2TaQ3umvAjZrilJQZT5riAmTaXtTinkjut1RpL\\\\\\\/NYLMfutBjbabV0fhVr1GMW\\\\\\\/bQZS4PLxg4dBOdTGgpVOEU7xHrjS47ON9hZ4delUpK5SnXqYgmfd8ChbVmpfgKn\\\\\\\/ZhIMVm4H9IVAh\\\\\\\/SYL49409PkGlDxZ+iROViVVa\\\\\\\/5ECghwGBRDm1SGUZgIzSx75qH5s9uog7WnB8XbCvzutLbswb0Wcn3BCf\\\\\\\/EbH5WZ6cxtT7F5DSZv3\\\\\\\/9RAoU0b325ITcriKweiF3mTAIa62DyekxNzUlJbHuPDubLvW67VWyi1O7Q9ONaZmxV\\\\\\\/tT8BZ4ZFg80ZlHC0nOIHOUg+h9piU81S1VdvwpO8oHDDWQIZnPcCpnW4dizBbloyYvos9ErZVDaQZ\\\\\\\/P6DQJqggchqMX7+mfejOGK7Pn0Cu98vIM4vT4mTyBXlX7K7DQgxDK+0\\\\\\\/4dAtNKQ+0K6Myw27H7I3C5fAtEYQZqrQCBltHc+JmDNWFn249no+fSqPgiWK5YWrVzSLOlVEolYEkITivOBAq2vQArIMBKnWk5jjibstVbH7l82tIh69ymS+9LoSVntxD5WVQkJJFl1ehqmW8BnDqAfZyF3C5gd0T+5BxQ\\\\\\\/pn0e8eCL8vcKhN7p6llPUaS4vPpCJKAz+4ozeuq2frdOQ79oPkB4fZQlejUxoY6krp2xTbDOqCV9c+d8zyUCPixEmKUbsS2F1X8SMQ1vNmxs8MXy4gQjFdK+wHAospu7ZpnWO7Q5z3e5Qqj7P5j\\\\\\\/vFOPI5ErMqR33dg1pYvfztk2dgG2sitbSh9P1WosF9F32hmapXnpJEEGf0m88mx2hArCVq25xy+xRX\\\\\\\/b\\\\\\\/CIFdcB\\\\\\\/f4dWyKY0JBZRfY6Gep9+yl+s\\\\\\\/MYvXO\\\\\\\/LuCC19Sxj3anANZRGLcyUI\\\\\\\/C8OG3j3W0qN\\\\\\\/qnqruqSuDeP+707IsrutTHHYDCDi7u24Z19OiYaz39Z+ucVQqQAlmxs9FAQJ\\\\\\\/qF35MpBQAVamyDrE6FGvo+Esm7KMs9pP9ClgMDXPgZWnhtNxkdcDlPxb64d8MN4JPanTK5ceslOQ4yXViA0UQtDlPOxrbUZkD1JMtwzZ9sfEF+f45HupHJ6+8Jp2lvD8VWcRFulDap3VEGybZdwcfsIq8nJM7QQcqWqJu0B9y1GkzrS2p5ClYseGbAVf8QkJdqVqJFeV+krQHAeyiiApEGQXicqfls6cgM77Z4L5EjZYdGMnmt\\\\\\\/ngRm\\\\\\\/YdueC5Qzomz7tn1UHUXjZzBJCXEbKBPhGkBm5blufR5VgsA7MaMZgLMiw2JfFcUFawXsOPQ8j3ZxtwFCXaWPFAwVSxNgvK6L3kzcD14saQeYaJxJmv1eWy1Jw8+W0sQLyQ7fQOXo9HY59Bdo36CpdQUBeph526g5dvl4kv6wd2AaM9uEe1bec2lQXLqVQDsXAv\\\\\\\/Zipd92oZzVIv9X8K9LtW+oUdTqOqStRtio1pC3FYec6es6SAm2iSK8ocKp3qkeGs1eogXeCEXcYA+TkL0qDEEFM9BjH1zo5vkZaszVnvVO0AYplU5uHD8goaPIXEkCqUIuMCh54i9eqmLuZwAjmIzBff46MoJmFC8WFrWSMbZEsmNQD0pgOkn2AaMhaWMU+qXorHiLjbBm8a7HPsBPaHWHUU5i5HAG1LN5z5he3IKW6iw458Tir9fVNO0cGJBb5sh64DKWKK9cIYUljMb4eRvpyCY\\\\\\\/VfshNjf3m2Oe+bpR2rTrRuEUbrtnN5t4518BBLEmgROhEeeex+RZxlKhsjhhSCXCoQOcDrUtndAP3GII\\\\\\\/Wh06RG1YsdYL0YxKQAC0cQpfMoSUuBzFvw6eJ9eJwMX\\\\\\\/IcZPXZRfgOHJ6DC3EKYMPDmIHztwx12lpWjZqJzlU8ohngNE9ywzSQyMTtwFE+UN53TbyTVqQeg4OSeoqrZpj5SRJ7vbvfCIKXxB2Ib6yFwPCMs+VX3t23Bg\\\\\\\/B66Zb0WWcF05IDJ2QIM+elEMsvsEsrWNjc1DT4gwMdQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022qYcAR34bkZGIsUC+ailWWg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022f355803224faff4e3b0792f8c7fe7a45360288d66155f171f19c5f47cf7890d7\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-494592 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-genome-report">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/11/3537" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-pTZvtorUKO0QM1sHr5dqe5LuJp3MgIvY9JTUD7GiTXs" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/11/3537" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-oS-FpPUE0Fym4tpjCTJ2hXJK8mIMdME4zEgTz5Os1Gc" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/11/3531" title="De Novo Assembly and Annotation from Parental and F1 Puma Genomes of the Florida Panther Genetic Restoration Program" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/11/3547" title="Phased Diploid Genome Assemblies for Three Strains of Candida albicans from Oak Trees" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494592" id="node494592" data-pisa="ggg;9/11/3537" data-pisa-master="ggg;g3.119.400708" data-apath="/ggg/9/11/3537.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Construction of High-Resolution RAD-Seq Based Linkage Map, Anchoring Reference Genome, and QTL Mapping of the Sex Chromosome in the Marine Medaka <em>Oryzias melastigma</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Bo-Young Lee</span>, <span class="highwire-citation-author" data-delta="1">Min-Sub Kim</span>, <span class="highwire-citation-author" data-delta="2">Beom-Soon Choi</span>, <span class="highwire-citation-author" data-delta="3">Atsushi J. Nagano</span>, <span class="highwire-citation-author" data-delta="4">Doris Wai Ting Au</span>, <span class="highwire-citation-author" data-delta="5">Rudolf Shiu Sun Wu</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0002-9965-8499" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Yusuke Takehana</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="7"><a href="http://orcid.org/0000-0003-0944-5172" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jae-Seong Lee</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3537-3545;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400708 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494592" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Bo-Young Lee </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biological Science, College of Science, Sungkyunkwan University, Suwon 16419, South Korea,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bo-Young%2BLee%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lee%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABo-Young%2BLee%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Min-Sub Kim </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biological Science, College of Science, Sungkyunkwan University, Suwon 16419, South Korea,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Min-Sub%2BKim%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kim%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMin-Sub%2BKim%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Beom-Soon Choi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Phyzen Genomics Institute, Seongnam 13558, South Korea,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Beom-Soon%2BChoi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Choi%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABeom-Soon%2BChoi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Atsushi J. Nagano </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Faculty of Agriculture, Ryukoku University, Kyoto 612-8577, Japan,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Atsushi%2BJ.%2BNagano%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nagano%20AJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAtsushi%2BJ.%2BNagano%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Doris Wai Ting Au </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Chemistry and</div><div class='author-affiliation'><span class='nlm-sup'>**</span>State Key Laboratory of Marine Pollution, The City University of Hong Kong, Hong Kong SAR, China,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Doris%2BWai%2BTing%2BAu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Au%20DW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADoris%2BWai%2BTing%2BAu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Rudolf Shiu Sun Wu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>Department of Science and Environmental Studies, The Education University of Hong Kong, Hong Kong SAR, China, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rudolf%2BShiu%2BSun%2BWu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wu%20RS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARudolf%2BShiu%2BSun%2BWu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Yusuke Takehana </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>Department of Animal Bioscience, Nagahama Institute of Bio-Science and Technology, Nagahama, Shiga 526-0829, Japan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yusuke%2BTakehana%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Takehana%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AYusuke%2BTakehana%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-9965-8499" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Yusuke Takehana</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jslee2@skku.edu" class="" data-icon-position="" data-hide-link-title="0">jslee2@skku.edu</a>
<a href="mailto:y_takehana@nagahama-i-bio.ac.jp" class="" data-icon-position="" data-hide-link-title="0">y_takehana@nagahama-i-bio.ac.jp</a></span></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Jae-Seong Lee </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biological Science, College of Science, Sungkyunkwan University, Suwon 16419, South Korea,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jae-Seong%2BLee%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lee%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJae-Seong%2BLee%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-0944-5172" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jae-Seong Lee</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jslee2@skku.edu" class="" data-icon-position="" data-hide-link-title="0">jslee2@skku.edu</a>
<a href="mailto:y_takehana@nagahama-i-bio.ac.jp" class="" data-icon-position="" data-hide-link-title="0">y_takehana@nagahama-i-bio.ac.jp</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/9/11/3537" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:494592" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:494592/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/11/3537.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:494592" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:494592/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/11/3537.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:494592" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:494592/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/9/11/3537.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:494592" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:494592/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Medaka (<em>Oryzias</em> sp.) is an important fish species in ecotoxicology and considered as a model species due to its biological features including small body size and short generation time. Since Japanese medaka <em>Oryzias latipes</em> is a freshwater species with access to an excellent genome resource, the marine medaka <em>Oryzias melastigma</em> is also applicable for the marine ecotoxicology. In genome era, a high-density genetic linkage map is a very useful resource in genomic research, providing a means for comparative genomic analysis and verification of <em>de novo</em> genome assembly. In this study, we developed a high-density genetic linkage map for <em>O. melastigma</em> using restriction-site associated DNA sequencing (RAD-seq). The genetic map consisted of 24 linkage groups with 2,481 single nucleotide polymorphism (SNP) markers. The total map length was 1,784 cM with an average marker space of 0.72 cM. The genetic map was integrated with the reference-assisted chromosome assembly (RACA) of <em>O. melastigma</em>, which anchored 90.7% of the assembled sequence onto the linkage map. The values of complete Benchmarking Universal Single-Copy Orthologs were similar to RACA assembly but N50 (23.74 Mb; total genome length 779.4 Mb; gap 5.29%) increased to 29.99 Mb (total genome length 778.7 Mb; gap 5.2%). Using MapQTL analysis with SNP markers, we identified a major quantitative trait locus for sex traits on the Om10. The integration of the genetic map with the reference genome of marine medaka will serve as a good resource for studies in molecular toxicology, genomics, CRISPR/Cas9, and epigenetics.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/marine-medaka" class="hw-term hw-article-keyword hw-article-keyword-marine-medaka" rel="nofollow">marine medaka</a></li><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/oryzias-melastigma" class="hw-term hw-article-keyword hw-article-keyword-oryzias-melastigma" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Oryzias melastigma</em></a></li><li class="kwd"><a href="/keyword/linkage-map" class="hw-term hw-article-keyword hw-article-keyword-linkage-map" rel="nofollow">linkage map</a></li><li class="kwd"><a href="/keyword/reference-genome" class="hw-term hw-article-keyword hw-article-keyword-reference-genome" rel="nofollow">reference genome</a></li><li class="kwd"><a href="/keyword/sex" class="hw-term hw-article-keyword hw-article-keyword-sex" rel="nofollow">sex</a></li><li class="kwd"><a href="/keyword/quantitative-trait-locus" class="hw-term hw-article-keyword hw-article-keyword-quantitative-trait-locus" rel="nofollow">quantitative trait locus</a></li></ul><p id="p-3">Many fish species are useful for ecotoxicological research, as they indicate an early warning of environmental contamination caused by various aquatic pollutants (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Arellano-Aguilar <em>et al.</em> 2009</a>). Recently, an increase in the contamination levels of estuaries and coastal water due to anthropogenic pollutants emphasizes the need for marine sentinel model fish species. Medaka (<em>Oryzias</em> sp.) is an important fish species in ecotoxicology and considered a model species, as its biological features include certain advantages such as small body size and short generation time (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Kim <em>et al.</em> 2016</a>). Japanese medaka <em>Oryzias latipes</em> is important and widely used model species for the studies of genetics, evolution, and ecotoxicology, with abundant genomic resources. Since <em>O. latipes</em> is a freshwater species, their responses to environmental toxicants can be different in those of marine fish (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Shi <em>et al.</em> 2008</a>; <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Wu <em>et al.</em> 2012</a>; <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Wheeler <em>et al.</em> 2002</a>). Marine medaka <em>Oryzias melastigma</em> inhabits brackish water in Asian regions including Pakistan, India, Burma, and Thailand (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Naruse 1996</a>). <em>O. melastigma</em> has been acknowledged as a potential model fish for marine ecotoxicological studies and is useful for the evaluation of acute and/or chronic toxicity, and embryo toxicity testing (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Chen <em>et al.</em> 2011</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Dong <em>et al.</em> 2014</a>; <a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Kim <em>et al.</em> 2016</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Kong <em>et al.</em> 2008</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Shen <em>et al.</em> 2010</a>; <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Wang <em>et al.</em> 2011</a>; <a id="xref-ref-18-3" class="xref-bibr" href="#ref-18">Kim <em>et al.</em> 2016</a>). The genus <em>Oryzias</em> has been divided into three major monophyletic species groups; the <em>latipes</em>, the <em>javanicus</em>, and the <em>celebenesis</em> groups, while <em>O. dancena</em>, a closely related species of <em>O. melastigma</em>, has been phylogenetically placed in the <em>javanicus</em> group (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Takehana <em>et al.</em> 2005</a>). Phenotypic distinction between male and female in medaka is distinguished by a number of secondary sex characters including shape and size of dorsal and anal fins due to morphologically indistinguishable sex chromosomes (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Schartl 2004</a>). Moreover, <em>Oryzias</em> species shows the both XY and ZW sex-determining systems. <em>O. latipes</em>, <em>O. dancena</em>, and <em>O. minutillus</em> have an XX/XY sex-determining system and <em>O. hubbsi</em> has a ZZ/ZW (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Matsuda <em>et al.</em> 1998</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Takehana <em>et al.</em> 2008</a>). For example, sex-determining gene <em>dmrt1bY</em> has been identified in a few species of the <em>O. latipes</em> group and <em>sox3<sup>Y</sup></em> in <em>O. dancena</em> and some species of the <em>celebenesis</em> group (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Kondo <em>et al.</em> 2003</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Matsuda <em>et al.</em> 2002</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Myosho <em>et al.</em> 2015</a>). There is a lack of studies on sex-determining genes in <em>O. melastigma</em>. Therefore, more genomic resources specifically devoted for the marine medaka <em>O. melastigma</em> are required.</p><p id="p-4">A genetic linkage map is a very useful tool to understand genetic architecture such as chromosome structure, segregation distortion regions, recombination rate, and recombination hotspots (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Meyer and Van de Peer 2005</a>; <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Zhu <em>et al.</em> 2014</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Guo <em>et al.</em> 2013</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Shifman <em>et al.</em> 2006</a>). Furthermore, it provides a framework for mapping the chromosomal location of single-gene traits and quantitative traits of interest, and helps to facilitate candidate gene cloning, and comparative genomic analysis with some genome information together (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Lee <em>et al.</em> 2005</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Amores <em>et al.</em> 2011</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Feng <em>et al.</em> 2015</a>; <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Zhu <em>et al.</em> 2013</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Li <em>et al.</em> 2015</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Shao <em>et al.</em> 2015</a>; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Wang <em>et al.</em> 2015</a>; <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Xiao <em>et al.</em> 2015</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Kanamori <em>et al.</em> 2016</a>). A high-density genetic map also plays an important role in assembling whole genome sequences by examining the accuracy of <em>de novo</em> genome assembly (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Dukić <em>et al.</em> 2016</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Rhee <em>et al.</em> 2017</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Wang <em>et al.</em> 2017</a>). Indeed, the importance of a high-density genetic map has been demonstrated during the <em>de novo</em> genome assembly in teleost fish, as it validated the presence of additional genome duplication (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Meyer and Schartl 1999</a>; <a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Meyer and Van de Peer 2005</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Volff 2005</a>). In addition, restriction-site associated DNA (RAD) sequencing based on next-generation sequencing (NGS) enables the rapid discovery of genome-wide genetic markers and high-throughput single nucleotide polymorphism (SNP) genotyping in mapping families and facilitates the construction of high-density genetic linkage maps in both model and non-model organisms (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Baird <em>et al.</em> 2008</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Davey and Blaxter 2010</a>; <a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Amores <em>et al.</em> 2011</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Davey <em>et al.</em> 2011</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Etter <em>et al.</em> 2011</a>).</p><p id="p-5">We have recently published a reference genome assembly (total genome length 779.4 Mb) of <em>O. melastigma</em> as a model species in environmental toxicology (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Kim <em>et al.</em> 2018</a>). In this study, we constructed a high-density genetic linkage map of <em>O. melastigma</em> using an F1 full-sib family. Using the genetic linkage map, sex QTL has been mapped in the genome of <em>O. melastigma</em> and the previous genome assembly was anchored onto the linkage map to improve the contiguity of the assembly. The development of a high-density genetic map is imperative to facilitate both genetic and genomic studies in <em>O. melastigma</em>. The present study will assist in a better understanding of genome-based research in molecular toxicology, genomics, CRISPR/Cas9, and epigenetics.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and methods</h2><div id="sec-2" class="subsection"><h3>Mapping cross</h3><p id="p-6">The marine medaka <em>O. melastigma</em> used in this study were obtained from the City University of Hong Kong (kindly provided by Dr. Doris W.T. Au) and maintained at Nagahama Institute of Bio-Science and Technology, Nagahama, Japan. A male and a female fish were bred to produce F1 progenies. In total, 58 F1 individuals were used to create a linkage group (LG).</p></div><div id="sec-3" class="subsection"><h3>RAD sequencing</h3><p id="p-7">Genomic DNA was extracted from muscle tissue using DNeasy Blood &amp; Tissue Kit (Qiagen, Hilden, Germany) according to the manufacturer’s instructions. The size and quality of DNA isolated was checked on 1% agarose gel by electrophoresis and the concentration was measured using Qubit florometer (Thermo Fisher Scientific, Waltham, MA, USA). Genomic DNA (40 ng) of each sample was digested with <em>Bgl</em>II (5 Unit) and <em>Eco</em>RI (5 Unit), ligated with a Y-shaped adaptor, amplified by polymerase chain reaction (PCR) with KAPA HiFi HS ReadyMix (Kapa Biosystems, Wilmington, MA, USA), and fragments were selected with E-Gel Size Select (Life Technologies, Carlsbad, CA, USA). The mean size of the selected fragments was 333 bp (CV 16.4%). Further details of the library preparation method are described in a previous study by <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Sakaguchi <em>et al.</em> (2015)</a>. RAD sequencing (RAD-seq) was performed using 58 F1 individuals and both parents with HiSeq2500 (Illumina, San Diego, CA, USA) with eight cycles for index read and 51 cycles for the reads of interest. For each parental sample, the same amounts were aliquoted in four different reaction tubes and sequencing of each reaction was carried out to reduce PCR amplification bias. All procedures related to RAD-seq including the library construction were performed by Clockmics Inc. (Osaka, Japan).</p></div><div id="sec-4" class="subsection"><h3>Extracting RAD-tags and SNP genotyping by Stacks</h3><p id="p-8">Quality filtration of sequence reads was performed using Trimmomatic v.0.33 (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bolger <em>et al.</em> 2014</a>) with parameter options of -0.33.jar SE -phred33 TOPHRED33 ILLUMINACLIP TruSeq3-PE-2.fa:2:30:10 LEADING:19 TRAILING:19 SLIDINGWINDOW:30:20 AVGQUAL:20 MINLEN:51. RAD-tag extraction and genotyping were performed with Stacks v.1.47 software (<a href="http://creskolab.uoregon.edu/stacks/">http://creskolab.uoregon.edu/stacks/</a>) (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Catchen <em>et al.</em> 2011</a>). The sequence reads were aligned to the available reference genome <strong>(GCF_002922805.1,</strong> <a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Kim <em>et al.</em> 2018</a><strong>)</strong> using GSnap (<a href="https://github.com/juliangehring/GMAP-GSNAP">https://github.com/juliangehring/GMAP-GSNAP</a>) with default parameters (-t 30 –n 1 –m 5 –i 2), which converted to BAM files. All RAD-tags catalog from the parental samples were extracted by Stacks using the <em>ref_map.pl</em> pipeline with the parameters –m10 and -P 3 and genotyping was called by the parameters of minimum number of 5 reads to call a homozygous genotype, a minimum minor allele frequency of 0.1 to call a heterozygote, and a maximum minor allele frequency of 0.05 to call a homozygote. Among RAD-tags, single nucleotide polymorphism (SNP) markers with maximum likelihood of 0 were selected for mapping and the SNP markers with genotypes of at least 53 F1 offsprings (&gt; 90%) were collected for map construction using the command <em>genotypes</em> –r 53 of Stacks v.1.47. Data of raw sequences were deposited in the Sequence Read Archive (SRA) (<a href="http://www.ncbi.nlm.nih.gov/sra">http://www.ncbi.nlm.nih.gov/sra</a>) under the accession number <strong>PRJNA514812</strong>.</p></div><div id="sec-5" class="subsection"><h3>Linkage map construction</h3><p id="p-9">Linkage analysis of genetic markers (SNPs) was performed using JoinMap 5.0 (Wageningen, Netherlands, <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Van Ooijen 2018</a>). The SNP markers with a significant segregation distortion (χ<sup>2</sup> test, <em>P</em> &lt; 0.01) were removed from the analysis of linkage map construction. Linkage groups were identified by the grouping parameters of independent Likelihood Odds Ratio (LOD) threshold of 5 provided in JoinMap 5.0. Map distances were calculated by the Kosambi’s mapping function and mapping algorithm used for building linkage map was regression mapping based on the default parameters. The regression mapping adds loci one by one starting from the most informative pair of loci. The best position of each added locus is searched by comparing the goodness-of-fit of the calculated map for each tested position. JoinMap performed three rounds of marker positioning with a jump threshold of 5 and we took second round of map as a final map as recommended by the manual. The linkage map was visualized using MapCart 2.32 (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Voorrips 2002</a>). The name of the linkage groups was matched with the homologous chromosomes of Japanese medaka.</p></div><div id="sec-6" class="subsection"><h3>Anchoring the reference genome on to linkage map</h3><p id="p-10">Genetic markers in the linkage map were anchored to the reference genome <strong>(GCF_002922805.1)</strong> of the marine medaka <em>O. melastigma</em> using Chromonomer v. 1.08 (<a href="http://catchenlab.life.illinois.edu/chromonomer/">http://catchenlab.life.illinois.edu/chromonomer/</a>). The integrated genome assembly based on the genetic map was re-assessed with benchmarking universal single-copy orthologs (BUSCO) v.3.0 (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Simão <em>et al.</em> 2015</a>) using the vertebrate database (OrthoDB v.9.0; <a href="https://www.orthodb.org/?page=filelist">https://www.orthodb.org/?page=filelist</a>). The gene annotation of the final assembly was carried out using MAKER v.2.31.8 pipeline with manual curation (Suppl. Fig. S1) (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Holt and Yandell 2011</a>).</p></div><div id="sec-7" class="subsection"><h3>Comparative analysis of two medaka genomes</h3><p id="p-11">The final linkage map based genome assembly of marine medaka was compared with the genome of Japanese medaka (Hd-rR strain) to compare the similarity between two medaka genomes using Mummer v.3.0 (<a href="http://mummer.sourceforge.net/manual/#coords">http://mummer.sourceforge.net/manual/#coords</a>).</p></div><div id="sec-8" class="subsection"><h3>Sex linkage analysis</h3><p id="p-12">The mapping panel consisted of 27 males and 31 females. In order to examine sex determining mechanism in <em>O. melastigma</em> by linkage analysis, two completely linked sex markers (sex_xy and sex_zw) were additionally added in the genotype files. For sex_xy and sex_zw genotypes, male individuals were coded as heterozygotes and homozygotes, respectively, while female individuals were coded as homozygotes and heterozygotes, respectively. The linkage analysis of sex markers was performed with JoinMap 5.0 (Wageningen, Netherlands, <a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">Van Ooijen 2018</a>). Since the sex trait is qualitative, the phenotype for sex was converted into binary code; male 1 and female 0. For QTL analysis, standard interval mapping was performed and significance was determined by permutation test (n = 1000) using MapQTL 6.0 (van Ooijen <em>et al.</em> 2002).</p></div><div id="sec-9" class="subsection"><h3>Data availability</h3><p id="p-13">Sequenced species <em>O. melastigma</em> is available upon request. Suppl. Data contains all supplemental tables and figures. Suppl. File S1 contains the names of 2481 SNP, genotype, map positions, and tag sequences in Map info tab and phenotypes for sex of all individuals were in Phenotype tab. Suppl. File S2 contains the re-scaffolding process and results by Chromonomer. Suppl. File S3 includes the gene list annotated in the sex-determining region. Raw data of RAD seq were deposited to GenBank under the accession number <strong>PRJNA514812</strong>. The genome sequence data are available in GenBank with accession number <strong>PRJNA401159</strong>. The genome JBrowse is available at <a href="http://rotifer.skku.edu:8080/Om2">http://rotifer.skku.edu:8080/Om2</a>. Supplemental material available at FigShare: <a href="https://doi.org/10.25387/g3.7811978">https://doi.org/10.25387/g3.7811978</a>.</p></div></div><div class="section results" id="sec-10"><h2 class="">Results</h2><div id="sec-11" class="subsection"><h3>Constructing genetic linkage map of the marine medaka Oryzias melastigma</h3><p id="p-14">Stacks software extracted 113,367 RAD-tags from <em>O. melastigma</em> genome. Among them, the number of putative SNP markers was 34,040. To distinguish polymorphisms from sequencing errors, we collected 24,441 SNP markers with Likelihood Ratio of 0, including 8,518 SNP markers that have been located in the same RAD-tags more than twice. Among the remaining 15,922 loci, 4,497 SNP loci were successfully genotyped in at least 53 F1 individuals (&gt;90%). After removing the markers that showed a segregation distortion (<em>P</em> &lt; 0.01), 3,732 were finally used for building a genetic map. The 3,730 markers were grouped into 24 LGs with a LOD ≥ 5.0 by independence LOD and two remaining markers were not linked with any of those groups. The regression mapping function by JoinMap positioned 2,481 SNPs in the second rounds of mapping by comparing the goodness-of-fit (Suppl. File S1). The 24 LGs were consistent with the number of chromosomes (n = 24) in <em>O. melastigma</em> (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Uwa <em>et al.</em> 1983</a>). The total map length was 1,784 cM and each LG included 57-173 markers with an average marker interval of 0.72 cM (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a> and <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/11/3537/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="A linkage map of the marine medaka Oryzias melastigma. The map consists of 24 linkage groups and the bars on each linkage group represents single nucleotide polymorphism (SNP) markers. Colors of bars indicate the reference-assisted chromosome assembly scaffolds that SNP was extracted. (Name, sequences, and position of SNP are included in the Suppl. File 1.)" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-883619400" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;A linkage map of the marine medaka &lt;em&gt;Oryzias melastigma&lt;/em&gt;. The map consists of 24 linkage groups and the bars on each linkage group represents single nucleotide polymorphism (SNP) markers. Colors of bars indicate the reference-assisted chromosome assembly scaffolds that SNP was extracted. (Name, sequences, and position of SNP are included in the Suppl. File 1.)&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/11/3537/F1.medium.gif" width="440" height="202"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/9/11/3537/F1.medium.gif" width="440" height="202"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/11/3537/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/11/3537/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494713" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-15" class="first-child">A linkage map of the marine medaka <em>Oryzias melastigma</em>. The map consists of 24 linkage groups and the bars on each linkage group represents single nucleotide polymorphism (SNP) markers. Colors of bars indicate the reference-assisted chromosome assembly scaffolds that SNP was extracted. (Name, sequences, and position of SNP are included in the Suppl. File 1.)</p><div class="sb-div caption-clear"></div></div></div><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/494676/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/494676/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Summary of the genetic linkage map of <em>Oryzias melastigma</em></span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-12" class="subsection"><h3>Re-scaffolding of the reference genome with the genetic map</h3><p id="p-16">Using the SNP marker information in the high-density genetic linkage map, 810 markers were anchored to 134 scaffolds and among them, 35 were split into 1 to 9 positions, producing a total of 260 integrated scaffolds (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Tables 2</a>, Suppl. Table S2, and Suppl. Fig. S2). After integration, the length of the genome scaffolds aligned on the map was 712,537,413 bp (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>). Out of the 260 integrated scaffolds, the orientation was determined in 160 scaffolds spanning 670,530,120 bp, which accounted for 94% of the total scaffold length in the linkage map (<a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>). Among 40 reference-assisted chromosome assembly (RACA) scaffolds previously published (<a id="xref-ref-19-3" class="xref-bibr" href="#ref-19">Kim <em>et al.</em> 2018</a>), 20 RACA scaffolds (RACA3, 5, 6, 7, 8, 9, 11, 12, 14, 15, 18, 21, 22, 23, 25, 26, 27, 29, 30, 40) were aligned to the 13 linkage groups (Om23, Om21, Om20, Om19, Om17, Om16, Om14, Om 12, Om11, Om10, Om09, Om08, Om01) without any modification (Suppl. Table S1). Other RACA scaffolds showed the major alignment of one linkage group with the partial alignment of another linkage groups (Suppl. Table S1). Among them, RACA 31 showed the most frequent rearrangement during the anchoring process, which was a major alignment with Om 7 and partial alignments with another 4 linkage groups (Om20, Om18, Om17, and Om02) (Suppl. Table S1). Four linkage groups (Om03, Om05, Om15, and Om22) were completely aligned by each of 4 RACA scaffolds (RACA4, RACA16, RACA33, and RACA36) respectively (<a id="xref-table-wrap-2-4" class="xref-table" href="#T2">Table 2</a>), although some parts of sequences aligned in the linkage groups were inserted from small scaffolds (Suppl. File S2) and the parts of RACA scaffolds were located in parts of another linkage groups (Supp. Table S1). Overall, the final genetic map based genome assembly consisted of 8,493 scaffolds; 24 linkage map-based scaffolds (90.7%) and 8,469 unanchored scaffolds (9.3%). The total genome length was 779 Mb with an N50 value of 29,978,720 bp (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>). BUSCO analysis indicated that the final genome assembly of <em>O. melastigma</em> represented 96.8% of the complete copy in the vertebrate gene model (<a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>). The genome annotation pipeline in the final assembly was defined as 24,507 genes (<a href="http://rotifer.skku.edu:8080/Om2">http://rotifer.skku.edu:8080/Om2</a>), ranging from 661 to 1,216 genes per LG (<a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a> and Suppl. Table S3).</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/494786/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/494786/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Physical lengths of linkage map anchored with the reference genome assembly in <em>Oryzias melastigma</em><sup>#</sup></span><div class="sb-div caption-clear"></div></div></div><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/494646/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/494646/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Statistics of the final genome assembly before and after anchoring in <em>Oryzias melastigma</em></span><div class="sb-div caption-clear"></div></div></div><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/494634/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/494634/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">Assessment of LG-based assembly completeness</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-13" class="subsection"><h3>Comparative genomic analysis of two medaka genomes</h3><p id="p-18">The final genome assembly integrated with a genetic map provides an efficient resource for comparative genomic analysis with other medaka genomes such as a Japanese medaka (<em>O. latipes</em>). The 24 genetic map-based scaffolds showed good homology in gene contents and sequences similarity with chromosomes in <em>O. latipes</em>, Most LG-based scaffolds of marine medaka showed collinear relationships completely or in a majority with the counterpart chromosomes of <em>O. latipes</em> (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a> and Suppl. Fig. S3). Other LG/chromosomes in <em>O. melastigma</em> showed disrupted collinearity due to intrachromosomal rearrangements or possible errors in linkage map. Reversely matched parts in the collinearly related scaffolds were most likely caused by the undetermined orientation of scaffolds involved in the anchoring process. In addition, some LG-based scaffolds showed in/del regions (Om05, Om08, Om11, Om14, Om15, Om16, and Om17) compared to Japanese medaka chromosomes.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/11/3537/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genome-wide comparison of the genomic sequences between Oryzias melastigma and Oryzias latipes using Mummer. Red and blue dots represent forward and reverse match, respectively." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-883619400" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Genome-wide comparison of the genomic sequences between &lt;em&gt;Oryzias melastigma&lt;/em&gt; and &lt;em&gt;Oryzias latipes&lt;/em&gt; using Mummer. Red and blue dots represent forward and reverse match, respectively.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/11/3537/F2.medium.gif" width="440" height="315"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/9/11/3537/F2.medium.gif" width="440" height="315"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/11/3537/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/11/3537/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494719" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-19" class="first-child">Genome-wide comparison of the genomic sequences between <em>Oryzias melastigma</em> and <em>Oryzias latipes</em> using Mummer. Red and blue dots represent forward and reverse match, respectively.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-14" class="subsection"><h3>Mapping of sex-determining regions</h3><p id="p-20">The linkage analysis of two sex markers (sex_xy and sex_zw) showed that both markers were more or less linked to markers in the Om10. Sex_xy showed the highest LOD scores with 17018 (LOD = 17.06) and strong linkages with other markers in this linkage group (Suppl. Table S4). Although sex_zw showed the highest LOD value (7.42) with 16660, which also had the same LOD with sex_xy, the LOD scores of sex_zw with other markers indicated weak or suspect linkage (Suppl. Table S4). This investigation suggested that the <em>O. melastigma</em> has XY sex-determining system.</p><p id="p-21">A significant QTL for sex was detected in Om10 with the LOD significance threshold of 5.3 based on the permutation test (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>). Among 2,481 markers mapped in the Om linkage groups, 58 markers showed the significant association with sex (Suppl. Table S5), most of which were markers from the RACA27. The most significant two markers associated with sex were 17040 (24.3 cM) and 17018 (24.4 cM) with LOD of 35 (Suppl. Table S5). The genotypes of those markers were completely linked with the sex phenotype of all individuals, except for one animal (d004), which could be caused by wrong phenotyping or sex-reversal. When the animal was removed from the QTL analysis, LOD scores of 17018 and 17040 increased to 881.12 and 299.4, respectively (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3B</a>). Physical location of sex-determining region was mapped between 10,353,256 bp and 10,538,878 bp in Om10, spanning approximately 186 kb, and 10 genes were located in the regions without any obvious candidate gene (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3C</a>). However, the sox3 was located on the 10,625,885 bp, which was 87 kb apart from the mapped region (Suppl. Table S3; <a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3C</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/11/3537/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Quantitative trait locus mapping for sex traits on the Om10 scaffold of Oryzias melastigma. (A) Standard interval mapping. Scales on the left represents the map position (cM) of linkage group Om10; Scales on the top of the graph represents the value of the Likelihood Odds Ratio (LOD) scores. The red dotted line indicates the threshold of significance (LOD = 5.3). (B) Standard interval mapping without d004 individual. Scales on the left represents the physical (Mbp) of Om10 scaffold; Scales on the top of the graph represents the value of the LOD scores. The red dotted line indicates the threshold of significance (LOD = 5.3). (C) Gene lists in the sex-determining region of Om10 scaffold." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-883619400" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Quantitative trait locus mapping for sex traits on the Om10 scaffold of &lt;em&gt;Oryzias melastigma&lt;/em&gt;. (A) Standard interval mapping. Scales on the left represents the map position (cM) of linkage group Om10; Scales on the top of the graph represents the value of the Likelihood Odds Ratio (LOD) scores. The red dotted line indicates the threshold of significance (LOD = 5.3). (B) Standard interval mapping without d004 individual. Scales on the left represents the physical (Mbp) of Om10 scaffold; Scales on the top of the graph represents the value of the LOD scores. The red dotted line indicates the threshold of significance (LOD = 5.3). (C) Gene lists in the sex-determining region of Om10 scaffold.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/11/3537/F3.medium.gif" width="440" height="388"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/9/11/3537/F3.medium.gif" width="440" height="388"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/11/3537/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/11/3537/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494805" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-22" class="first-child">Quantitative trait locus mapping for sex traits on the Om10 scaffold of <em>Oryzias melastigma</em>. (A) Standard interval mapping. Scales on the left represents the map position (cM) of linkage group Om10; Scales on the top of the graph represents the value of the Likelihood Odds Ratio (LOD) scores. The red dotted line indicates the threshold of significance (LOD = 5.3). (B) Standard interval mapping without d004 individual. Scales on the left represents the physical (Mbp) of Om10 scaffold; Scales on the top of the graph represents the value of the LOD scores. The red dotted line indicates the threshold of significance (LOD = 5.3). (C) Gene lists in the sex-determining region of Om10 scaffold.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-15"><h2 class="">Discussion</h2><p id="p-23">A high-resolution genetic map is very useful in diverse genomic research and has been applied to many fish species (<a id="xref-ref-1-3" class="xref-bibr" href="#ref-1">Amores <em>et al.</em> 2011</a>; <a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Li <em>et al.</em> 2015</a>; <a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Shao <em>et al.</em> 2015</a>; <a id="xref-ref-53-2" class="xref-bibr" href="#ref-53">Xiao <em>et al.</em> 2015</a>; <a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">Wang <em>et al.</em> 2015</a>; <a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Kanamori <em>et al.</em> 2016</a>). In this study, a high-density genetic map was constructed in <em>O. melastigma</em> using RAD sequencing and was used for verifying the previously published marine medaka reference genome and for aligning the scaffolds at the chromosomal level. The genetic map of <em>O. melastigma</em> consists of 24 LGs with 2,481 SNP markers, which cover 24 chromosomes (<a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Uwa <em>et al.</em> 1983</a>). The 810 genetic markers were anchored to the genetic map (<a id="xref-table-wrap-2-5" class="xref-table" href="#T2">Table 2</a> and Suppl. Figure 2), which mapped 90.7% (713 Mb) of the reference genome assembly sequence onto the genetic map (<a id="xref-table-wrap-3-3" class="xref-table" href="#T3">Table 3</a>). Although all SNP markers were extracted by the alignment of RAD sequences against the reference genome of marine medaka (<a id="xref-ref-19-4" class="xref-bibr" href="#ref-19">Kim <em>et al.</em> 2018</a>), the number of anchored markers were lower than anticipated. Many markers in the LGs were excluded from the integrating process due to inconsistency between the genetic map and reference assembly, indicating possible errors in marker order and/or <em>de novo</em> assembly. Indeed, in most cases the markers tended to be highly concentrated in the narrow regions, suggesting that more recombinant individuals will be required to obtain a more precise marker order. Also, it is likely that the size of the mapping family was not big enough for the marker concentrated regions, compared with the number of markers analyzed, resulting in relatively low number of anchored markers. Despite this shortage, the marine medaka genetic map was still integrated with the reference assembly, considering other fish species had a similar or slightly lower level of scaffolds mapping onto the genetic map (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Tine <em>et al.</em> 2014</a>; <a id="xref-ref-25-3" class="xref-bibr" href="#ref-25">Li <em>et al.</em> 2015</a>; <a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">Wang <em>et al.</em> 2017</a>).</p><p id="p-24">Previously, we have developed a reference genome for the marine medaka <em>O. melastigma</em> in several steps (<a id="xref-ref-19-5" class="xref-bibr" href="#ref-19">Kim <em>et al.</em> 2018</a>). <em>De novo</em> assembly was performed using the combination of Platanus and Haplomerger2 assemblers (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Kajitani <em>et al.</em> 2014</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Huang <em>et al.</em> 2017</a>). The contiguity of <em>de novo</em> genome assembly was further increased by RACA (Kim <em>et al.</em> 2013), which assisted in the construction of highly ordered and oriented scaffolds of <em>de novo</em> assembly and reassembled scaffolds into longer chromosomal fragments using the comparative genome information of closely related species. RACA assembly of <em>O. melastigma</em> generated 40 gigantic scaffolds (Suppl. Table 1), which account for 674 Mb (86.5%) of the total genome sequence (<a id="xref-ref-19-6" class="xref-bibr" href="#ref-19">Kim <em>et al.</em> 2018</a>). Taken together, the genetic map-integrated assembly improved the reference sequences by mapping 90.7% of genome sequences onto 24 LGs with 94% determined orientation of mapped scaffolds (<a id="xref-table-wrap-2-6" class="xref-table" href="#T2">Table 2</a>). The BUSCO and N50 demonstrated a better quality of integrated final genome assembly with values increased to 96.8% and 29,987,720 bp, respectively (<a id="xref-table-wrap-3-4" class="xref-table" href="#T3">Tables 3</a> and <a id="xref-table-wrap-4-2" class="xref-table" href="#T4">4</a>).</p><p id="p-25">Quantitative trait locus (QTL) analysis identified that Om10 was strongly associated with sex in <em>O. melastigma</em> and two markers, 17040 and 10718, showed the most significant association for sex, without any recombinant between genotype and phenotype. LOD scores of 17040 (LOD = 34.99) is little higher than that of 10718 (LOD = 34.98), which is likely associated with the missing genotype in one individual for 10718 (Suppl. Table S5). Thus, sex determining locus should be located between 17018 and 17040. Although no obvious candidate gene for sex was found in the mapped sex-determining regions, we noticed <em>sox3</em> outside the mapped region (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3C</a>). It was intriguing because a <em>cis</em>-regulatory element in the downstream region of <em>sox3</em> on Y chromosome plays a key signal to sex determination in <em>O</em>. <em>dancena</em> (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Takehana <em>et al.</em> 2014</a>), suggesting that the same <em>cis</em>-regulatory element might be present in the mapped regions of <em>O. melastigma</em>. For further analysis, fine mapping with additional recombinants and the functional analysis of candidate genes or regulatory elements need to be performed in the mapped sex-determining region.</p><p id="p-26">In summary, a high-density genetic map is a very useful resource to determine the accuracy of <em>de novo</em> genome assembly, especially with massively parallel short sequencing reads. In addition, the integration of a genetic map with reference genomes will be a useful resource in various genomic studies including comparative genomic analyses, fine mapping of QTL, positional cloning of candidate genes, and CRISPR/Cas9 studies.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-27">This study was supported by a grant from the National Research Foundation (2017R1D1A1B03036026) to Bo-Young Lee.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-2"><p id="p-28">Supplemental material available at FigShare: <a href="https://doi.org/10.25387/g3.7811978">https://doi.org/10.25387/g3.7811978</a>.</p></li><li class="fn" id="fn-3"><p id="p-29"><em>Communicating editor: D. J. de Koning</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2019-07-04"><span class="received-label">Received </span>July 4, 2019.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2019-09-13"><span class="accepted-label">Accepted </span>September 13, 2019.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2019 Lee <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3537.1" data-doi="10.1534/genetics.111.127324"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Amores</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Catchen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Ferrara</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Q.</span> <span class="cit-name-surname">Fontenot</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. H.</span> <span class="cit-name-surname">Postlethwait</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genome evolution and meiotic maps by massively parallel DNA sequencing: Spotted Gar, an outgroup for the teleost genome duplication.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">188</span>: <span class="cit-fpage">799</span>–<span class="cit-lpage">808</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.111.127324</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.127324%26rft_id%253Dinfo%253Apmid%252F21828280%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg4LzQvNzk5IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMS8zNTM3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3537.2" data-doi="10.1080/10641260903243487"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Arellano-Aguilar</span>,  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. M.</span> <span class="cit-name-surname">Montoya</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. M.</span> <span class="cit-name-surname">Garcia</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Endogenous functions and expression of cytochrome P450 enzymes in teleost fish: a review.</span> <abbr class="cit-jnl-abbrev">Rev. Fish. Sci.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">541</span>–<span class="cit-lpage">556</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1080/10641260903243487</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRev.%2BFish.%2BSci.%26rft.volume%253D17%26rft.spage%253D541%26rft_id%253Dinfo%253Adoi%252F10.1080%252F10641260903243487%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/10641260903243487&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3537.3" data-doi="10.1371/journal.pone.0003376"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Baird</span>,  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. D.</span> <span class="cit-name-surname">Etter</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. S.</span> <span class="cit-name-surname">Atwood</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. C.</span> <span class="cit-name-surname">Currey</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. L.</span> <span class="cit-name-surname">Shiver</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Rapid SNP discovery and genetic mapping using sequenced RAD markers.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">e3376</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0003376</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DBaird%26rft.auinit1%253DN.%2BA.%26rft.volume%253D3%26rft.issue%253D10%26rft.spage%253De3376%26rft.epage%253De3376%26rft.atitle%253DRapid%2BSNP%2Bdiscovery%2Band%2Bgenetic%2Bmapping%2Busing%2Bsequenced%2BRAD%2Bmarkers.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0003376%26rft_id%253Dinfo%253Apmid%252F18852878%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0003376&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18852878&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3537.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3537.4" data-doi="10.1093/bioinformatics/btu170"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bolger</span>,  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Lohse</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Usadel</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> Trimmomatic: a flexible trimmer for Illumina sequence data. Bioinformatics 30: 2114–2120.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btu170</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu170%26rft_id%253Dinfo%253Apmid%252F24695404%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btu170&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24695404&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3537.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000340049100004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3537.5" data-doi="doi:10.1534/g3.111.000240"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Catchen</span>,  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Amores</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Hohenlohe</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Cresko</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. H.</span> <span class="cit-name-surname">Postlethwait</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Stacks: Building and genotyping loci <em>de novo</em> from short-read sequences.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">171</span>–<span class="cit-lpage">182</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.111.000240</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft.volume%253D1%26rft.spage%253D171%26rft_id%253Dinfo%253Adoi%252Fdoi%253A10.1534%252Fg3.111.000240%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=doi:10.1534/g3.111.000240&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3537.6" data-doi="10.1016/j.marpolbul.2011.03.042"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Li</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Cheng</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L. L.</span> <span class="cit-name-surname">Chan</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. Z.</span> <span class="cit-name-surname">Wang</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Molecular staging of marine medaka: a model organism for marine ecotoxicity study.</span> <abbr class="cit-jnl-abbrev">Mar. Pollut. Bull.</abbr> <span class="cit-vol">63</span>: <span class="cit-fpage">309</span>–<span class="cit-lpage">317</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.marpolbul.2011.03.042</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMarine%2BPollution%2BBulletin%26rft.stitle%253DMarine%2BPollution%2BBulletin%26rft.aulast%253DChen%26rft.auinit1%253DX.%26rft.volume%253D63%26rft.issue%253D5-12%26rft.spage%253D309%26rft.epage%253D317%26rft.atitle%253DMolecular%2Bstaging%2Bof%2Bmarine%2Bmedaka%253A%2Ba%2Bmodel%2Borganism%2Bfor%2Bmarine%2Becotoxicity%2Bstudy.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.marpolbul.2011.03.042%26rft_id%253Dinfo%253Apmid%252F21708389%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.marpolbul.2011.03.042&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21708389&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3537.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3537.7" data-doi="10.1093/bfgp/elq031"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Davey</span>,  <span class="cit-name-given-names">J. W.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. L.</span> <span class="cit-name-surname">Blaxter</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">RADseq: next-generation population genetics.</span> <abbr class="cit-jnl-abbrev">Brief. Funct. Genomics</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">416</span>–<span class="cit-lpage">423</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bfgp/elq031</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBrief.%2BFunct.%2BGenomics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbfgp%252Felq031%26rft_id%253Dinfo%253Apmid%252F21266344%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bfgp/elq031&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21266344&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3537.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000286675100007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3537.8" data-doi="10.1038/nrg3012"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Davey</span>,  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. A.</span> <span class="cit-name-surname">Hohenlohe</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. D.</span> <span class="cit-name-surname">Etter</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. Q.</span> <span class="cit-name-surname">Boone</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">Catchen</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genome-wide genetic marker discovery and genotyping using next-generation sequencing.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">499</span>–<span class="cit-lpage">510</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg3012</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DDavey%26rft.auinit1%253DJ.%2BW.%26rft.volume%253D12%26rft.issue%253D7%26rft.spage%253D499%26rft.epage%253D510%26rft.atitle%253DGenome-wide%2Bgenetic%2Bmarker%2Bdiscovery%2Band%2Bgenotyping%2Busing%2Bnext-generation%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3012%26rft_id%253Dinfo%253Apmid%252F21681211%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3012&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21681211&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3537.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3537.9" data-doi="10.1155/2014/563131"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dong</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Kang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Wu</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Ye</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Development of a promising fish model (<em>Oryzias melastigma</em>) for assessing multiple responses to stresses in the marine environment.</span> <abbr class="cit-jnl-abbrev">BioMed Res. Int.</abbr> <span class="cit-vol">2014</span>: <span class="cit-fpage">563131</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1155/2014/563131</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioMed%2BRes.%2BInt.%26rft.volume%253D2014%26rft.spage%253D563131%26rft_id%253Dinfo%253Adoi%252F10.1155%252F2014%252F563131%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1155/2014/563131&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3537.10" data-doi="10.1186/s12863-016-0445-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dukić</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Berner</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Roesti</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. R.</span> <span class="cit-name-surname">Haag</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Ebert</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">A high-density genetic map reveals variation in recombination rate across the genome of <em>Daphina magna</em>.</span> <abbr class="cit-jnl-abbrev">BMC Genet.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">137</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12863-016-0445-7</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenet.%26rft.volume%253D17%26rft.spage%253D137%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12863-016-0445-7%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12863-016-0445-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3537.11" data-doi="10.1007/978-1-61779-228-1_9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Etter</span>,  <span class="cit-name-given-names">P. D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Bassham</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. A.</span> <span class="cit-name-surname">Hohenlohe</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. A.</span> <span class="cit-name-surname">Johnson</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">W. A.</span> <span class="cit-name-surname">Cresko</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">SNP discovery and genotyping for evolutionary genetics using RAD sequencing.</span> <abbr class="cit-jnl-abbrev">Methods Mol. Biol.</abbr> <span class="cit-vol">772</span>: <span class="cit-fpage">157</span>–<span class="cit-lpage">178</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/978-1-61779-228-1_9</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2Bin%2Bmolecular%2Bbiology%2B%2528Clifton%252C%2BN.J.%2529%26rft.stitle%253DMethods%2BMol%2BBiol%26rft.aulast%253DEtter%26rft.auinit1%253DP.%2BD.%26rft.volume%253D772%26rft.spage%253D157%26rft.epage%253D178%26rft.atitle%253DSNP%2Bdiscovery%2Band%2Bgenotyping%2Bfor%2Bevolutionary%2Bgenetics%2Busing%2BRAD%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-1-61779-228-1_9%26rft_id%253Dinfo%253Apmid%252F22065437%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-1-61779-228-1_9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22065437&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3537.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000321876000010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3537.12" data-doi="10.1007/s00412-014-0485-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Feng</span>,  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Wang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Yu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Zhang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Lu</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Microsatellite-centromere mapping in common carp through half-tetrad analysis in diploid meiogynogenetic families.</span> <abbr class="cit-jnl-abbrev">Chromosoma</abbr> <span class="cit-vol">124</span>: <span class="cit-fpage">67</span>–<span class="cit-lpage">79</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00412-014-0485-6</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DChromosoma%26rft.volume%253D124%26rft.spage%253D67%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00412-014-0485-6%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00412-014-0485-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3537.13" data-doi="10.1016/j.aquaculture.2013.06.027"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>,  <span class="cit-name-given-names">W. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. G.</span> <span class="cit-name-surname">Tong</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X. M.</span> <span class="cit-name-surname">Yu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. K.</span> <span class="cit-name-surname">Zhu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Feng</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">A second generation genetic linkage map for silver carp (<em>Hypophthalmichehys molitrix</em>) using microsatellite markers.</span> <abbr class="cit-jnl-abbrev">Aquaculture</abbr> <span class="cit-vol">412–413</span>: <span class="cit-fpage">97</span>–<span class="cit-lpage">106</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.aquaculture.2013.06.027</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAquaculture%26rft.volume%253D412413%26rft.spage%253D97%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.aquaculture.2013.06.027%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.aquaculture.2013.06.027&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3537.14" data-doi="10.1186/1471-2105-12-491"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Holt</span>,  <span class="cit-name-given-names">C.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Yandell</span></span>, </li></ol><cite><span class="cit-pub-date">2011</span> <span class="cit-article-title">MAKER2: an annotation pipeline and genome-database management tool for second-generation genome projects.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">491</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DHolt%26rft.auinit1%253DC.%26rft.volume%253D12%26rft.spage%253D491%26rft.epage%253D491%26rft.atitle%253DMAKER2%253A%2Ban%2Bannotation%2Bpipeline%2Band%2Bgenome-database%2Bmanagement%2Btool%2Bfor%2Bsecond-generation%2Bgenome%2Bprojects.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-12-491%26rft_id%253Dinfo%253Apmid%252F22192575%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-12-491&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22192575&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3537.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3537.15" data-doi="10.1093/bioinformatics/btx220"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Kang</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Xu</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">HaploMerger2: rebuilding both haploid sub-assemblies from high-heterozygosity diploid genome assembly.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">2577</span>–<span class="cit-lpage">2579</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btx220</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.volume%253D33%26rft.spage%253D2577%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtx220%26rft_id%253Dinfo%253Apmid%252F28407147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btx220&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28407147&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3537.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3537.16" data-doi="10.1101/gr.170720.113"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kajitani</span>,  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Toshioto</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Noguchi</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Toyoda</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Ogura</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Efficient <em>de novo</em> assembly of highly heterozygous genomes from whole-genome shotgun short reads.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">1384</span>–<span class="cit-lpage">1395</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.170720.113</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.170720.113%26rft_id%253Dinfo%253Apmid%252F24755901%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjI0LzgvMTM4NCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzkvMTEvMzUzNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3537.17" data-doi="10.1534/g3.115.022699"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kanamori</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Sugita</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Yuasa</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Suzuki</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Kawamuyra</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> A genetic map for the only self-fertilizing vertebrate. G3 6: 1095–1106.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.115.022699</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.115.022699%26rft_id%253Dinfo%253Apmid%252F26865699%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjg6IjYvNC8xMDk1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMS8zNTM3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3537.18" data-doi="10.1016/j.marenvres.2015.12.004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>,  <span class="cit-name-given-names">B.-M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Kim</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">I.-Y.</span> <span class="cit-name-surname">Choi</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Raisuddin</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. W. T.</span> <span class="cit-name-surname">Au</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Omics of the marine medaka (<em>Oryzias melastigma</em>) and its relevance to marine environmental research.</span> <abbr class="cit-jnl-abbrev">Mar. Environ. Res.</abbr> <span class="cit-vol">113</span>: <span class="cit-fpage">141</span>–<span class="cit-lpage">152</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.marenvres.2015.12.004</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMar.%2BEnviron.%2BRes.%26rft.volume%253D113%26rft.spage%253D141%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.marenvres.2015.12.004%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.marenvres.2015.12.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3537.19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>,  <span class="cit-name-given-names">H.-S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.-Y.</span> <span class="cit-name-surname">Lee</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Han</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.-B.</span> <span class="cit-name-surname">Jeong</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.-S.</span> <span class="cit-name-surname">Hwang</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> The genome of the marine medaka <em>Oryzias melastigma</em>. Mol. Ecol. Resour. 18: 656–665.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/1755–0998.12769</span></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-9.11.3537.20" data-doi="10.1073/pnas.1220349110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. M.</span> <span class="cit-name-surname">Larkin</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Q.</span> <span class="cit-name-surname">Cai</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Asan</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Zhang</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Reference-assisted chromosome assembly.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">110</span>: <span class="cit-fpage">1785</span>–<span class="cit-lpage">1790</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.1220349110</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1220349110%26rft_id%253Dinfo%253Apmid%252F23307812%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTEwLzUvMTc4NSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzkvMTEvMzUzNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3537.21" data-doi="10.1016/S0960-9822(03)00125-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kondo</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Nanda</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">U.</span> <span class="cit-name-surname">Hornung</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Asakawa</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Shimizu</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Absence of the candidate male sex-determining gene <em>dmrt1b(Y)</em> of medaka from other fish species.</span> <abbr class="cit-jnl-abbrev">Curr. Biol.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">416</span>–<span class="cit-lpage">420</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/S0960-9822(03)00125-8</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bbiology%2B%253A%2B%2BCB%26rft.stitle%253DCurr%2BBiol%26rft.aulast%253DKondo%26rft.auinit1%253DM.%26rft.volume%253D13%26rft.issue%253D5%26rft.spage%253D416%26rft.epage%253D420%26rft.atitle%253DAbsence%2Bof%2Bthe%2Bcandidate%2Bmale%2Bsex-determining%2Bgene%2Bdmrt1b%2528Y%2529%2Bof%2Bmedaka%2Bfrom%2Bother%2Bfish%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0960-9822%252803%252900125-8%26rft_id%253Dinfo%253Apmid%252F12620191%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0960-9822(03)00125-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12620191&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3537.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000181388100024&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3537.22" data-doi="10.1016/j.aquatox.2007.10.011"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kong</span>,  <span class="cit-name-given-names">R. Y. C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. P.</span> <span class="cit-name-surname">Giesy</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. S. S.</span> <span class="cit-name-surname">Wu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. X.</span> <span class="cit-name-surname">Chen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. W.</span> <span class="cit-name-surname">Chiang</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Development of a marine fish model for studying <em>in vivo</em> molecular responses in ecotoxicology.</span> <abbr class="cit-jnl-abbrev">Aquat. Toxicol.</abbr> <span class="cit-vol">86</span>: <span class="cit-fpage">131</span>–<span class="cit-lpage">141</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.aquatox.2007.10.011</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAquatic%2BToxicology%26rft.stitle%253DAquatic%2BToxicology%26rft.aulast%253DKong%26rft.auinit1%253DR.%2BY.%26rft.volume%253D86%26rft.issue%253D2%26rft.spage%253D131%26rft.epage%253D141%26rft.atitle%253DDevelopment%2Bof%2Ba%2Bmarine%2Bfish%2Bmodel%2Bfor%2Bstudying%2Bin%2Bvivo%2Bmolecular%2Bresponses%2Bin%2Becotoxicology.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.aquatox.2007.10.011%26rft_id%253Dinfo%253Apmid%252F18055030%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.aquatox.2007.10.011&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18055030&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3537.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000253696200002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3537.23" data-doi="10.1534/genetics.104.035022"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>,  <span class="cit-name-given-names">B.-Y.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.-J.</span> <span class="cit-name-surname">Lee</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. T.</span> <span class="cit-name-surname">Streelman</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. L.</span> <span class="cit-name-surname">Carleton</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. E.</span> <span class="cit-name-surname">Howe</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">A second-generation genetic linkage map of Tilapia (<em>Oreochromis</em> spp.).</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">170</span>: <span class="cit-fpage">237</span>–<span class="cit-lpage">244</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.104.035022</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.104.035022%26rft_id%253Dinfo%253Apmid%252F15716505%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTcwLzEvMjM3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMS8zNTM3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3537.24" data-doi="10.1038/srep40121"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rhee</span>,  <span class="cit-name-given-names">J.-S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.-S.</span> <span class="cit-name-surname">Choi</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Kim</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.-M.</span> <span class="cit-name-surname">Kim</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.-M.</span> <span class="cit-name-surname">Lee</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Diversity, distribution, and significance of transposable elements in the genome of the only selfing hermaphrodictic vertebrate <em>Kryptolebias marmoratus</em>.</span> <abbr class="cit-jnl-abbrev">Sci. Rep.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">40121</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/srep40121</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BRep.%26rft.volume%253D7%26rft.spage%253D40121%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsrep40121%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/srep40121&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3537.25" data-doi="10.1093/dnares/dsu038"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Liu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Qin</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Waldbieser</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Wang</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Construction of a high-density, high-resolution genetic map and its integration with BAC-based physical map in Channel catfish.</span> <abbr class="cit-jnl-abbrev">DNA Res.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">39</span>–<span class="cit-lpage">52</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/dnares/dsu038</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDNA%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fdnares%252Fdsu038%26rft_id%253Dinfo%253Apmid%252F25428894%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/dnares/dsu038&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25428894&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3537.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3537.26" data-doi="10.1159/000015113"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Matsuda</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Matsuda</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Hamaguchi</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Sakaizumi</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Identification of the sex chromosomes of the medaka, <em>Oryzias latipes</em>, by fluorescence in situ hybridization.</span> <abbr class="cit-jnl-abbrev">Cytogenet. Cell Genet.</abbr> <span class="cit-vol">82</span>: <span class="cit-fpage">257</span>–<span class="cit-lpage">262</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1159/000015113</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCytogenetics%2Band%2Bcell%2Bgenetics%26rft.stitle%253DCytogenet%2BCell%2BGenet%26rft.aulast%253DMatsuda%26rft.auinit1%253DM.%26rft.volume%253D82%26rft.issue%253D3-4%26rft.spage%253D257%26rft.epage%253D262%26rft.atitle%253DIdentification%2Bof%2Bthe%2Bsex%2Bchromosomes%2Bof%2Bthe%2Bmedaka%252C%2BOryzias%2Blatipes%252C%2Bby%2Bfluorescence%2Bin%2Bsitu%2Bhybridization.%26rft_id%253Dinfo%253Adoi%252F10.1159%252F000015113%26rft_id%253Dinfo%253Apmid%252F9858830%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1159/000015113&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9858830&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3537.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000077796000026&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3537.27" data-doi="10.1038/nature751"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Matsuda</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Nagahama</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Shinomiya</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Sato</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Matsuda</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title"><em>DMY</em> is a Y-specific DM-domain gene required for male development in the medaka fish.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">417</span>: <span class="cit-fpage">559</span>–<span class="cit-lpage">563</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature751</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DMatsuda%26rft.auinit1%253DM.%26rft.volume%253D417%26rft.issue%253D6888%26rft.spage%253D559%26rft.epage%253D563%26rft.atitle%253DDMY%2Bis%2Ba%2BY-specific%2BDM-domain%2Bgene%2Brequired%2Bfor%2Bmale%2Bdevelopment%2Bin%2Bthe%2Bmedaka%2Bfish.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature751%26rft_id%253Dinfo%253Apmid%252F12037570%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature751&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12037570&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3537.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000175860300045&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3537.28" data-doi="10.1534/g3.115.021543"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Myosho</span>,  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Takehana</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Hamaguchi</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Sakaizumi</span></span></li></ol><cite> <span class="cit-pub-date">2015</span> <span class="cit-article-title">Turnover of sex chromosomes in celebensis group medaka fishes.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">2685</span>–<span class="cit-lpage">2691</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.115.021543</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.115.021543%26rft_id%253Dinfo%253Apmid%252F26497145%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjk6IjUvMTIvMjY4NSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzkvMTEvMzUzNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3537.29" data-doi="10.1016/S0955-0674(99)00039-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meyer</span>,  <span class="cit-name-given-names">A.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Schartl</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Gene and genome duplication invertebrates: the one-to-four (-to eight in fish) rule and the evolution of novel gene functions.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Cell Biol.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">699</span>–<span class="cit-lpage">704</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/S0955-0674(99)00039-3</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Bcell%2Bbiology%26rft.stitle%253DCurr%2BOpin%2BCell%2BBiol%26rft.aulast%253DMeyer%26rft.auinit1%253DA.%26rft.volume%253D11%26rft.issue%253D6%26rft.spage%253D699%26rft.epage%253D704%26rft.atitle%253DGene%2Band%2Bgenome%2Bduplications%2Bin%2Bvertebrates%253A%2Bthe%2Bone-to-four%2B%2528-to-eight%2Bin%2Bfish%2529%2Brule%2Band%2Bthe%2Bevolution%2Bof%2Bnovel%2Bgene%2Bfunctions.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0955-0674%252899%252900039-3%26rft_id%253Dinfo%253Apmid%252F10600714%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0955-0674(99)00039-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10600714&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3537.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000084010000009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3537.30" data-doi="10.1002/bies.20293"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meyer</span>,  <span class="cit-name-given-names">A.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Van de Peer</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">From 2R to 3R: evidence for a fish‐specific genome duplication (FSGD).</span> <abbr class="cit-jnl-abbrev">BioEssays</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">937</span>–<span class="cit-lpage">945</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/bies.20293</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioEssays%2B%253A%2B%2Bnews%2Band%2Breviews%2Bin%2Bmolecular%252C%2Bcellular%2Band%2Bdevelopmental%2Bbiology%26rft.stitle%253DBioessays%26rft.aulast%253DMeyer%26rft.auinit1%253DA.%26rft.volume%253D27%26rft.issue%253D9%26rft.spage%253D937%26rft.epage%253D945%26rft.atitle%253DFrom%2B2R%2Bto%2B3R%253A%2Bevidence%2Bfor%2Ba%2Bfish-specific%2Bgenome%2Bduplication%2B%2528FSGD%2529.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fbies.20293%26rft_id%253Dinfo%253Apmid%252F16108068%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/bies.20293&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16108068&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3537.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000231603900009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3537.31" data-doi="10.18999/fisbjm.8.1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Naruse</span>,  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Classification and phylogeny of fishes of the genus <em>Oryzias</em> and its relatives.</span> <abbr class="cit-jnl-abbrev">Fish Biol. J. MEDAKA</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">9</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.18999/fisbjm.8.1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFish%2BBiol.%2BJ.%2BMEDAKA%26rft.volume%253D8%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.18999%252Ffisbjm.8.1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.18999/fisbjm.8.1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3537.32" data-doi="10.1007/s11295-015-0944-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sakaguchi</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Sugino</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Tsumura</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Ito</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. D.</span> <span class="cit-name-surname">Crisp</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">High-throughput linkage mapping of Australian white cypress pine (<em>Callitris glaucophylla</em>) and map transferability to related species.</span> <abbr class="cit-jnl-abbrev">Tree Genet. Genomes</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">121</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s11295-015-0944-0</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTree%2BGenet.%2BGenomes%26rft.volume%253D11%26rft.spage%253D121%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11295-015-0944-0%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11295-015-0944-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3537.33" data-doi="10.1016/j.mod.2004.03.001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schartl</span>,  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">A comparative view on sex determination in medaka.</span> <abbr class="cit-jnl-abbrev">Mech. Dev.</abbr> <span class="cit-vol">121</span>: <span class="cit-fpage">639</span>–<span class="cit-lpage">645</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.mod.2004.03.001</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMechanisms%2Bof%2Bdevelopment%26rft.stitle%253DMech%2BDev%26rft.aulast%253DSchartl%26rft.auinit1%253DM.%26rft.volume%253D121%26rft.issue%253D7-8%26rft.spage%253D639%26rft.epage%253D645%26rft.atitle%253DA%2Bcomparative%2Bview%2Bon%2Bsex%2Bdetermination%2Bin%2Bmedaka.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.mod.2004.03.001%26rft_id%253Dinfo%253Apmid%252F15210173%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.mod.2004.03.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15210173&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3537.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000222571200006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3537.34" data-doi="10.1093/dnares/dsv001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shao</span>,  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Niu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Rastas</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Liu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Xie</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genome-wide SNP identification for the construction of a high-resolution genetic map of Japanese flounder (<em>Paralichthys olivaceus</em>): applications to QTL mapping of <em>Vibrio anguillarum</em> disease resistance and comparative genomic analysis.</span> <abbr class="cit-jnl-abbrev">DNA Res.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">161</span>–<span class="cit-lpage">170</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/dnares/dsv001</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDNA%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fdnares%252Fdsv001%26rft_id%253Dinfo%253Apmid%252F25762582%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/dnares/dsv001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25762582&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3537.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3537.35" data-doi="10.1016/j.chemosphere.2010.03.051"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shen</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Xu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. Y.</span> <span class="cit-name-surname">Tsang</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. W. T.</span> <span class="cit-name-surname">Au</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Toxicity comparison between <em>Chattonella marina</em> and <em>Karenia brevis</em> using marine medaka (<em>Oryzias melastigma</em>): Evidence against the suspected ichthyotoxins of <em>Chattonella marina</em>.</span> <abbr class="cit-jnl-abbrev">Chemosphere</abbr> <span class="cit-vol">80</span>: <span class="cit-fpage">585</span>–<span class="cit-lpage">591</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.chemosphere.2010.03.051</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DChemosphere%26rft.aulast%253DShen%26rft.auinit1%253DM.%26rft.volume%253D80%26rft.issue%253D5%26rft.spage%253D585%26rft.epage%253D591%26rft.atitle%253DToxicity%2Bcomparison%2Bbetween%2BChattonella%2Bmarina%2Band%2BKarenia%2Bbrevis%2Busing%2Bmarine%2Bmedaka%2B%2528Oryzias%2Bmelastigma%2529%253A%2BEvidence%2Bagainst%2Bthe%2Bsuspected%2Bichthyotoxins%2Bof%2BChattonella%2Bmarina.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.chemosphere.2010.03.051%26rft_id%253Dinfo%253Apmid%252F20444485%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.chemosphere.2010.03.051&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20444485&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3537.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3537.36" data-doi="10.1016/j.taap.2008.01.043"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shi</span>,  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Du</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. K.</span> <span class="cit-name-surname">Lam</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. S.</span> <span class="cit-name-surname">Wu</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Zhou</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Developmental toxicity and alteration of gene expression in zebrafish embryos exposed to PFOS.</span> <abbr class="cit-jnl-abbrev">Toxicol. Appl. Pharmacol.</abbr> <span class="cit-vol">230</span>: <span class="cit-fpage">23</span>–<span class="cit-lpage">32</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.taap.2008.01.043</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DToxicology%2Band%2Bapplied%2Bpharmacology%26rft.stitle%253DToxicol%2BAppl%2BPharmacol%26rft.aulast%253DShi%26rft.auinit1%253DX.%26rft.volume%253D230%26rft.issue%253D1%26rft.spage%253D23%26rft.epage%253D32%26rft.atitle%253DDevelopmental%2Btoxicity%2Band%2Balteration%2Bof%2Bgene%2Bexpression%2Bin%2Bzebrafish%2Bembryos%2Bexposed%2Bto%2BPFOS.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.taap.2008.01.043%26rft_id%253Dinfo%253Apmid%252F18407306%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.taap.2008.01.043&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18407306&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3537.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000257098300004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3537.37" data-doi="10.1093/bioinformatics/btv351"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Simão</span>,  <span class="cit-name-given-names">F. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. M.</span> <span class="cit-name-surname">Waterhouse</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Ioannidis</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. V.</span> <span class="cit-name-surname">Kriventseva</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. M.</span> <span class="cit-name-surname">Zdobnov</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">BUSCO: Assessing genome assembly and annotation completeness with single-copy orthologs.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">3210</span>–<span class="cit-lpage">3212</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btv351</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtv351%26rft_id%253Dinfo%253Apmid%252F26059717%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btv351&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26059717&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3537.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3537.38" data-doi="10.1371/journal.pbio.0040395"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shifman</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. T.</span> <span class="cit-name-surname">Bell</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. R.</span> <span class="cit-name-surname">Copley</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. S.</span> <span class="cit-name-surname">Taylor</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. W.</span> <span class="cit-name-surname">Williams</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">A high-resolution single nucleotide polymorphism genetic map of the mouse genome.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e395</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pbio.0040395</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DShifman%26rft.auinit1%253DS.%26rft.volume%253D4%26rft.issue%253D12%26rft.spage%253De395%26rft.epage%253De395%26rft.atitle%253DA%2Bhigh-resolution%2Bsingle%2Bnucleotide%2Bpolymorphism%2Bgenetic%2Bmap%2Bof%2Bthe%2Bmouse%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.0040395%26rft_id%253Dinfo%253Apmid%252F17105354%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.0040395&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17105354&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3537.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3537.39" data-doi="10.1038/ncomms5157"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Takehana</span>,  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Matsuda</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Myosho</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. L.</span> <span class="cit-name-surname">Suster</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Kawakami</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Co-option of Sox3 as the male-determining factor on the Y chromosome in the fish Oryzias dancena.</span> <abbr class="cit-jnl-abbrev">Nat Commun</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">4157</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ncomms5157</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat%2BCommun%26rft.volume%253D5%26rft.spage%253D4157%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms5157%26rft_id%253Dinfo%253Apmid%252F24948391%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms5157&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24948391&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3537.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3537.40" data-doi="10.1016/j.ympev.2005.01.016"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Takehana</span>,  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Naruse</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Sakaizumi</span></span></li></ol><cite>, 2005 Molecular phylogeny of the medaka fishes genus <em>Oryzias</em> (Beloniformes: Adrianichthyidae) based on nuclear and mitochondrial DNA sequences. Mol. Phylogenet. Evol. 36: 417–428.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.ympev.2005.01.016</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bphylogenetics%2Band%2Bevolution%26rft.stitle%253DMol%2BPhylogenet%2BEvol%26rft.aulast%253DTakehana%26rft.auinit1%253DY.%26rft.volume%253D36%26rft.issue%253D2%26rft.spage%253D417%26rft.epage%253D428%26rft.atitle%253DMolecular%2Bphylogeny%2Bof%2Bthe%2Bmedaka%2Bfishes%2Bgenus%2BOryzias%2B%2528Beloniformes%253A%2BAdrianichthyidae%2529%2Bbased%2Bon%2Bnuclear%2Band%2Bmitochondrial%2BDNA%2Bsequences.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ympev.2005.01.016%26rft_id%253Dinfo%253Apmid%252F15955519%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ympev.2005.01.016&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15955519&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3537.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000230265900016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3537.41" data-doi="10.1007/s10577-008-1227-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Takehana</span>,  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Hamaguchi</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Sakaizumi</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Different origins of ZZ/ZW sex chromosomes in closely related medaka fishes, Oryzias javanicus and O. hubbsi.</span> <abbr class="cit-jnl-abbrev">Chromosome Res.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">801</span>–<span class="cit-lpage">811</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s10577-008-1227-5</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DChromosome%2Bresearch%2B%253A%2B%2Ban%2Binternational%2Bjournal%2Bon%2Bthe%2Bmolecular%252C%2Bsupramolecular%2Band%2Bevolutionary%2Baspects%2Bof%2Bchromosome%2Bbiology%26rft.stitle%253DChromosome%2BRes%26rft.aulast%253DTakehana%26rft.auinit1%253DY.%26rft.volume%253D16%26rft.issue%253D5%26rft.spage%253D801%26rft.epage%253D811%26rft.atitle%253DDifferent%2Borigins%2Bof%2BZZ%252FZW%2Bsex%2Bchromosomes%2Bin%2Bclosely%2Brelated%2Bmedaka%2Bfishes%252C%2BOryzias%2Bjavanicus%2Band%2BO.%2Bhubbsi.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10577-008-1227-5%26rft_id%253Dinfo%253Apmid%252F18607761%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10577-008-1227-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18607761&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3537.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000258315200010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3537.42" data-doi="10.1038/ncomms6770"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tine</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Kuhl</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.-A.</span> <span class="cit-name-surname">Gagaire</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Louro</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Desmarais</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">European sea bass genome and its variation provide insights into adaptation to euryhalinity and speciation.</span> <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">5770</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ncomms6770</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun.%26rft.volume%253D5%26rft.spage%253D5770%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms6770%26rft_id%253Dinfo%253Apmid%252F25534655%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms6770&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25534655&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3537.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3537.43" data-doi="10.2183/pjab.59.43"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Uwa</span>,  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Iwamatsu</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">O. P.</span> <span class="cit-name-surname">Szxena</span></span></li></ol><cite>, <span class="cit-pub-date">1983</span> <span class="cit-article-title">Karyotype and cellular DNA content of the Indian ricefish, <em>Oryzias melastigma</em>.</span> <abbr class="cit-jnl-abbrev">Proc. Japan Acad. B.</abbr> <span class="cit-vol">59</span>: <span class="cit-fpage">43</span>–<span class="cit-lpage">47</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2183/pjab.59.43</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BJapan%2BAcad.%2BB.%26rft.volume%253D59%26rft.spage%253D43%26rft_id%253Dinfo%253Adoi%252F10.2183%252Fpjab.59.43%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2183/pjab.59.43&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-9.11.3537.44"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Van Ooijen</span>,  <span class="cit-name-given-names">J. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">MapQTL6, Software for the mapping of quantitative trait loci in experimental population of diploid species.</span> <abbr class="cit-jnl-abbrev">Wageningen, Netherlands, Kyazma B.V.</abbr></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3537.45"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Van Ooijen</span>,  <span class="cit-name-given-names">J. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">JoinMap <sup>®</sup> 5, Software for the calculation of genetic linkage maps in experimental populations of diploid species.</span> <abbr class="cit-jnl-abbrev">Kyazma B.V., Wageningen, Netherlands.</abbr></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3537.46" data-doi="10.1038/sj.hdy.6800635"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Volff</span>,  <span class="cit-name-given-names">J.-N.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Genome evolution and biodiversity in teleost fish.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">94</span>: <span class="cit-fpage">280</span>–<span class="cit-lpage">294</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/sj.hdy.6800635</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.stitle%253DHeredity%26rft.aulast%253DVolff%26rft.auinit1%253DJ.%2BN.%26rft.volume%253D94%26rft.issue%253D3%26rft.spage%253D280%26rft.epage%253D294%26rft.atitle%253DGenome%2Bevolution%2Band%2Bbiodiversity%2Bin%2Bteleost%2Bfish.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsj.hdy.6800635%26rft_id%253Dinfo%253Apmid%252F15674378%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/sj.hdy.6800635&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15674378&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3537.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000227208900005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3537.47" data-doi="10.1093/jhered/93.1.77"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Voorrips</span>,  <span class="cit-name-given-names">R. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">MapChart: software for the graphical presentation of linkage maps and QTLs.</span> <abbr class="cit-jnl-abbrev">J. Hered.</abbr> <span class="cit-vol">93</span>: <span class="cit-fpage">77</span>–<span class="cit-lpage">78</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/jhered/93.1.77</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BHeredity%26rft.stitle%253DJ%2BHered%26rft.aulast%253DVoorrips%26rft.auinit1%253DR.%2BE.%26rft.volume%253D93%26rft.issue%253D1%26rft.spage%253D77%26rft.epage%253D78%26rft.atitle%253DMapChart%253A%2BSoftware%2Bfor%2Bthe%2BGraphical%2BPresentation%2Bof%2BLinkage%2BMaps%2Band%2BQTLs%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjhered%252F93.1.77%26rft_id%253Dinfo%253Apmid%252F12011185%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/jhered/93.1.77&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12011185&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3537.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000175627300018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3537.48" data-doi="10.1186/s12864-016-3462-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Bai</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Liu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. Q.</span> <span class="cit-name-surname">Juang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z. Y.</span> <span class="cit-name-surname">Wan</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Construction of high-resolution recombination maps in Asian seabass.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">63</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12864-016-3462-z</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D18%26rft.spage%253D63%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864-016-3462-z%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12864-016-3462-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3537.49" data-doi="10.1038/srep16358"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z. Y.</span> <span class="cit-name-surname">Wan</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Bai</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. Q.</span> <span class="cit-name-surname">Huang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Chua</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Construction of a high-density linkage map and fine mapping of QTL for growth in Asian seabass.</span> <abbr class="cit-jnl-abbrev">Sci. Rep.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">16358</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/srep16358</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BRep.%26rft.volume%253D5%26rft.spage%253D16358%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsrep16358%26rft_id%253Dinfo%253Apmid%252F26553309%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/srep16358&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26553309&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3537.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3537.50" data-doi="10.1016/j.aquatox.2011.02.020"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Wang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Wang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Lin</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Hong</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Proteome profiles in medaka (<em>Oryzias melastigma</em>) liver and brain experimentally exposed to acute inorganic mercury.</span> <abbr class="cit-jnl-abbrev">Aquat. Toxicol.</abbr> <span class="cit-vol">103</span>: <span class="cit-fpage">129</span>–<span class="cit-lpage">139</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.aquatox.2011.02.020</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAquatic%2BToxicology%26rft.stitle%253DAquatic%2BToxicology%26rft.aulast%253DWang%26rft.auinit1%253DM.%26rft.volume%253D103%26rft.issue%253D3-4%26rft.spage%253D129%26rft.epage%253D139%26rft.atitle%253DProteome%2Bprofiles%2Bin%2Bmedaka%2B%2528Oryzias%2Bmelastigma%2529%2Bliver%2Band%2Bbrain%2Bexperimentally%2Bexposed%2Bto%2Bacute%2Binorganic%2Bmercury.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.aquatox.2011.02.020%26rft_id%253Dinfo%253Apmid%252F21458406%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.aquatox.2011.02.020&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21458406&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3537.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3537.51" data-doi="10.1002/etc.5620211127"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wheeler</span>,  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. M. Y.</span> <span class="cit-name-surname">Leung</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Morritt</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Sorokin</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Rogers</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Freshwater to saltwater toxicity extrapolation using species sensitivity distributions.</span> <abbr class="cit-jnl-abbrev">Environ. Toxicol. Chem.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">2459</span>–<span class="cit-lpage">2467</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/etc.5620211127</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEnvironmental%2BToxicology%2Band%2BChemistry%26rft.stitle%253DEnvironmental%2BToxicology%2Band%2BChemistry%26rft.aulast%253DWheeler%26rft.auinit1%253DJ.%2BR.%26rft.volume%253D21%26rft.issue%253D11%26rft.spage%253D2459%26rft.epage%253D2467%26rft.atitle%253DFreshwater%2Bto%2Bsaltwater%2Btoxicity%2Bextrapolation%2Busing%2Bspecies%2Bsensitivity%2Bdistributions.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fetc.5620211127%26rft_id%253Dinfo%253Apmid%252F12389927%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/etc.5620211127&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12389927&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3537.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3537.52" data-doi="10.1016/j.chemosphere.2011.12.060"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>,  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Q.</span> <span class="cit-name-surname">Huang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Fang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Ye</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Qiu</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">PFOS induced precocious hatching of <em>Oryzias melastigma</em>–from molecular level to individual level.</span> <abbr class="cit-jnl-abbrev">Chemosphere</abbr> <span class="cit-vol">87</span>: <span class="cit-fpage">703</span>–<span class="cit-lpage">708</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.chemosphere.2011.12.060</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DChemosphere%26rft.aulast%253DWu%26rft.auinit1%253DX.%26rft.volume%253D87%26rft.issue%253D7%26rft.spage%253D703%26rft.epage%253D708%26rft.atitle%253DPFOS%2Binduced%2Bprecocious%2Bhatching%2Bof%2BOryzias%2Bmelastigma--from%2Bmolecular%2Blevel%2Bto%2Bindividual%2Blevel.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.chemosphere.2011.12.060%26rft_id%253Dinfo%253Apmid%252F22273185%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.chemosphere.2011.12.060&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22273185&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3537.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3537.53" data-doi="10.1038/srep18661"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xiao</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Wang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Zhang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Fang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Liu</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Gene map of large yellow croaker (<em>Larimichthys crocea</em>) provides insights into teleost genome evolution and conserved regions associated with growth.</span> <abbr class="cit-jnl-abbrev">Sci. Rep.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">18661</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/srep18661</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BRep.%26rft.volume%253D5%26rft.spage%253D18661%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsrep18661%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/srep18661&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3537.54" data-doi="10.1371/journal.pone.0082950"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>,  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Sun</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Yu</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Tong</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Centromere localization for bighead carp (<em>Aristichthys nobilis</em>) through half-tetrad analysis in diploid gynogenetic families.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e82950</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0082950</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D8%26rft.spage%253De82950%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0082950%26rft_id%253Dinfo%253Apmid%252F24376614%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0082950&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24376614&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F11%2F3537.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-9.11.3537.55" data-doi="10.1111/age.12194"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>,  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Tong</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Yu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Guo</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Wang</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A second-generation genetic linkage map for bighead carp (<em>Aristichthys nobilis</em>) based on microsatellite markers.</span> <abbr class="cit-jnl-abbrev">Anim. Genet.</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">699</span>–<span class="cit-lpage">708</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/age.12194</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnim.%2BGenet.%26rft.volume%253D45%26rft.spage%253D699%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fage.12194%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/age.12194&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/9/11/3537.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/11/3531" title="De Novo Assembly and Annotation from Parental and F1 Puma Genomes of the Florida Panther Genetic Restoration Program" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/11/3547" title="Phased Diploid Genome Assemblies for Three Strains of Candida albicans from Oak Trees" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/9/11"><span>Volume 9 Issue 11, November 2019</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/9/11.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/9/11.cover-source.jpg?itok=DjqkLGk9" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 9 (11)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/11/3537" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-BjgMcg4sUb-3XFinmgdGIlOi_zM9or-P46N4kQPkTjA" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/genome-report" class="" data-icon-position="" data-hide-link-title="0">Genome Report</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/9/11/3537" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/9/11/3537" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/9/11/3537" class="active" data-icon-position="" data-hide-link-title="0">Construction of High-Resolution RAD-Seq Based Linkage Map, Anchoring Reference Genome, and QTL Mapping of the Sex Chromosome in the Marine Medaka Oryzias melastigma</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/494592" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-2C5by9HV_ieEPgUQ4d8njc6i2E2j5PnNjvK_56JVnr8" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/11/3537.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/9/11/3537" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-rq0N24aPNd79aDjdKovinBI-k4KFOGYU3KwbndE7vH8" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/9/11/3537" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/11/3537.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494592" id="node494592--2" data-pisa="ggg;9/11/3537" data-pisa-master="ggg;g3.119.400708" data-apath="/ggg/9/11/3537.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Construction of High-Resolution RAD-Seq Based Linkage Map, Anchoring Reference Genome, and QTL Mapping of the Sex Chromosome in the Marine Medaka <em>Oryzias melastigma</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Bo-Young Lee</span>, <span class="highwire-citation-author" data-delta="1">Min-Sub Kim</span>, <span class="highwire-citation-author" data-delta="2">Beom-Soon Choi</span>, <span class="highwire-citation-author" data-delta="3">Atsushi J. Nagano</span>, <span class="highwire-citation-author" data-delta="4">Doris Wai Ting Au</span>, <span class="highwire-citation-author" data-delta="5">Rudolf Shiu Sun Wu</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0002-9965-8499" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Yusuke Takehana</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="7"><a href="http://orcid.org/0000-0003-0944-5172" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jae-Seong Lee</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3537-3545;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400708 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494592--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Bo-Young Lee </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biological Science, College of Science, Sungkyunkwan University, Suwon 16419, South Korea,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bo-Young%2BLee%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lee%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABo-Young%2BLee%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Min-Sub Kim </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biological Science, College of Science, Sungkyunkwan University, Suwon 16419, South Korea,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Min-Sub%2BKim%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kim%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMin-Sub%2BKim%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Beom-Soon Choi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Phyzen Genomics Institute, Seongnam 13558, South Korea,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Beom-Soon%2BChoi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Choi%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABeom-Soon%2BChoi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Atsushi J. Nagano </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Faculty of Agriculture, Ryukoku University, Kyoto 612-8577, Japan,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Atsushi%2BJ.%2BNagano%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nagano%20AJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAtsushi%2BJ.%2BNagano%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Doris Wai Ting Au </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Chemistry and</div><div class='author-affiliation'><span class='nlm-sup'>**</span>State Key Laboratory of Marine Pollution, The City University of Hong Kong, Hong Kong SAR, China,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Doris%2BWai%2BTing%2BAu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Au%20DW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADoris%2BWai%2BTing%2BAu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Rudolf Shiu Sun Wu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>Department of Science and Environmental Studies, The Education University of Hong Kong, Hong Kong SAR, China, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rudolf%2BShiu%2BSun%2BWu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wu%20RS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARudolf%2BShiu%2BSun%2BWu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Yusuke Takehana </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>Department of Animal Bioscience, Nagahama Institute of Bio-Science and Technology, Nagahama, Shiga 526-0829, Japan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yusuke%2BTakehana%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Takehana%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AYusuke%2BTakehana%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-9965-8499" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Yusuke Takehana</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jslee2@skku.edu" class="" data-icon-position="" data-hide-link-title="0">jslee2@skku.edu</a>
<a href="mailto:y_takehana@nagahama-i-bio.ac.jp" class="" data-icon-position="" data-hide-link-title="0">y_takehana@nagahama-i-bio.ac.jp</a></span></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Jae-Seong Lee </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biological Science, College of Science, Sungkyunkwan University, Suwon 16419, South Korea,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jae-Seong%2BLee%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lee%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJae-Seong%2BLee%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-0944-5172" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jae-Seong Lee</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jslee2@skku.edu" class="" data-icon-position="" data-hide-link-title="0">jslee2@skku.edu</a>
<a href="mailto:y_takehana@nagahama-i-bio.ac.jp" class="" data-icon-position="" data-hide-link-title="0">y_takehana@nagahama-i-bio.ac.jp</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/9/11/3537" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/9/11/3537&amp;title=Construction%20of%20High-Resolution%20RAD-Seq%20Based%20Linkage%20Map%2C%20Anchoring%20Reference%20Genome%2C%20and%20QTL%20Mapping%20of%20the%20Sex%20Chromosome%20in%20the%20Marine%20Medaka%20Oryzias%20melastigma" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/9/11/3537&amp;title=Construction%20of%20High-Resolution%20RAD-Seq%20Based%20Linkage%20Map%2C%20Anchoring%20Reference%20Genome%2C%20and%20QTL%20Mapping%20of%20the%20Sex%20Chromosome%20in%20the%20Marine%20Medaka%20Oryzias%20melastigma" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/9/11/3537&amp;title=Construction%20of%20High-Resolution%20RAD-Seq%20Based%20Linkage%20Map%2C%20Anchoring%20Reference%20Genome%2C%20and%20QTL%20Mapping%20of%20the%20Sex%20Chromosome%20in%20the%20Marine%20Medaka%20Oryzias%20melastigma" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/9/11/3537&amp;text=Construction%20of%20High-Resolution%20RAD-Seq%20Based%20Linkage%20Map%2C%20Anchoring%20Reference%20Genome%2C%20and%20QTL%20Mapping%20of%20the%20Sex%20Chromosome%20in%20the%20Marine%20Medaka%20Oryzias%20melastigma" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/9/11/3537&amp;title=Construction%20of%20High-Resolution%20RAD-Seq%20Based%20Linkage%20Map%2C%20Anchoring%20Reference%20Genome%2C%20and%20QTL%20Mapping%20of%20the%20Sex%20Chromosome%20in%20the%20Marine%20Medaka%20Oryzias%20melastigma" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/9/11/3537&amp;t=Construction%20of%20High-Resolution%20RAD-Seq%20Based%20Linkage%20Map%2C%20Anchoring%20Reference%20Genome%2C%20and%20QTL%20Mapping%20of%20the%20Sex%20Chromosome%20in%20the%20Marine%20Medaka%20Oryzias%20melastigma" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/9/11/3537&amp;title=Construction%20of%20High-Resolution%20RAD-Seq%20Based%20Linkage%20Map%2C%20Anchoring%20Reference%20Genome%2C%20and%20QTL%20Mapping%20of%20the%20Sex%20Chromosome%20in%20the%20Marine%20Medaka%20Oryzias%20melastigma" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/9/11/3537&amp;title=Construction%20of%20High-Resolution%20RAD-Seq%20Based%20Linkage%20Map%2C%20Anchoring%20Reference%20Genome%2C%20and%20QTL%20Mapping%20of%20the%20Sex%20Chromosome%20in%20the%20Marine%20Medaka%20Oryzias%20melastigma" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/494592/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494592" id="node494592--3" data-pisa="ggg;9/11/3537" data-pisa-master="ggg;g3.119.400708" data-apath="/ggg/9/11/3537.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Construction of High-Resolution RAD-Seq Based Linkage Map, Anchoring Reference Genome, and QTL Mapping of the Sex Chromosome in the Marine Medaka <em>Oryzias melastigma</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Bo-Young Lee</span>, <span class="highwire-citation-author" data-delta="1">Min-Sub Kim</span>, <span class="highwire-citation-author" data-delta="2">Beom-Soon Choi</span>, <span class="highwire-citation-author" data-delta="3">Atsushi J. Nagano</span>, <span class="highwire-citation-author" data-delta="4">Doris Wai Ting Au</span>, <span class="highwire-citation-author" data-delta="5">Rudolf Shiu Sun Wu</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0002-9965-8499" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Yusuke Takehana</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="7"><a href="http://orcid.org/0000-0003-0944-5172" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jae-Seong Lee</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3537-3545;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400708 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494592--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Bo-Young Lee </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biological Science, College of Science, Sungkyunkwan University, Suwon 16419, South Korea,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bo-Young%2BLee%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lee%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABo-Young%2BLee%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Min-Sub Kim </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biological Science, College of Science, Sungkyunkwan University, Suwon 16419, South Korea,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Min-Sub%2BKim%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kim%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMin-Sub%2BKim%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Beom-Soon Choi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Phyzen Genomics Institute, Seongnam 13558, South Korea,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Beom-Soon%2BChoi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Choi%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABeom-Soon%2BChoi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Atsushi J. Nagano </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Faculty of Agriculture, Ryukoku University, Kyoto 612-8577, Japan,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Atsushi%2BJ.%2BNagano%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nagano%20AJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAtsushi%2BJ.%2BNagano%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Doris Wai Ting Au </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Chemistry and</div><div class='author-affiliation'><span class='nlm-sup'>**</span>State Key Laboratory of Marine Pollution, The City University of Hong Kong, Hong Kong SAR, China,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Doris%2BWai%2BTing%2BAu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Au%20DW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADoris%2BWai%2BTing%2BAu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Rudolf Shiu Sun Wu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>Department of Science and Environmental Studies, The Education University of Hong Kong, Hong Kong SAR, China, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rudolf%2BShiu%2BSun%2BWu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wu%20RS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARudolf%2BShiu%2BSun%2BWu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Yusuke Takehana </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>Department of Animal Bioscience, Nagahama Institute of Bio-Science and Technology, Nagahama, Shiga 526-0829, Japan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yusuke%2BTakehana%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Takehana%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AYusuke%2BTakehana%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-9965-8499" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Yusuke Takehana</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jslee2@skku.edu" class="" data-icon-position="" data-hide-link-title="0">jslee2@skku.edu</a>
<a href="mailto:y_takehana@nagahama-i-bio.ac.jp" class="" data-icon-position="" data-hide-link-title="0">y_takehana@nagahama-i-bio.ac.jp</a></span></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Jae-Seong Lee </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biological Science, College of Science, Sungkyunkwan University, Suwon 16419, South Korea,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jae-Seong%2BLee%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lee%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJae-Seong%2BLee%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-0944-5172" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jae-Seong Lee</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jslee2@skku.edu" class="" data-icon-position="" data-hide-link-title="0">jslee2@skku.edu</a>
<a href="mailto:y_takehana@nagahama-i-bio.ac.jp" class="" data-icon-position="" data-hide-link-title="0">y_takehana@nagahama-i-bio.ac.jp</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=3537&amp;rft.epage=3545&amp;rft.atitle=Construction%20of%20High-Resolution%20RAD-Seq%20Based%20Linkage%20Map%2C%20Anchoring%20Reference%20Genome%2C%20and%20QTL%20Mapping%20of%20the%20Sex%20Chromosome%20in%20the%20Marine%20Medaka%20Oryzias%20melastigma&amp;rft.volume=9&amp;rft.issue=11&amp;rft.date=2019-11-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Lee%2C+Bo-Young&amp;rft.au=Kim%2C+Min-Sub&amp;rft.au=Choi%2C+Beom-Soon&amp;rft.au=Nagano%2C+Atsushi+J.&amp;rft.au=Au%2C+Doris+Wai+Ting&amp;rft.au=Wu%2C+Rudolf+Shiu+Sun&amp;rft.au=Takehana%2C+Yusuke&amp;rft.au=Lee%2C+Jae-Seong"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/494592/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/494592/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/494592/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/494592/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/494592/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/494592/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/494592/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/494592/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/494592/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/494592/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/494592/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/494592/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496052" id="node496052" data-pisa="ggg;g3.120.401151v1" data-pisa-master="ggg;g3.120.401151" data-apath="/ggg/early/2020/03/26/g3.120.401151.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/26/g3.120.401151" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Long-Read Genome Sequencing and Assembly of <em>Leptopilina</em> <em>boulardi</em>: A Specialist <em>Drosophila</em> Parasitoid</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496044" id="node496044" data-pisa="ggg;g3.119.400871v1" data-pisa-master="ggg;g3.119.400871" data-apath="/ggg/early/2020/03/17/g3.119.400871.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/17/g3.119.400871" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title"><em>De Novo</em> Genome Assembly of the <em>indica</em> Rice Variety IR64 Using Linked-Read Sequencing and Nanopore Sequencing</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496040" id="node496040" data-pisa="ggg;g3.119.400893v1" data-pisa-master="ggg;g3.119.400893" data-apath="/ggg/early/2020/03/12/g3.119.400893.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/12/g3.119.400893" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Understanding the Genetic Domestication History of the Jianchang Duck by Genotyping and Sequencing of Genomic Genes Under Selection</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Genome%20Report" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Genome Report</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/9/11/3537" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and methods</a></li><li class="odd"><a href="#sec-10" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-15" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/9/11/3537.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/9/11/3537.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/9/11/3537.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
