<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/2/10/1279" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/2/10/1279.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/2/10/1279.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/2/10/1279.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/2/10.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/2/10/1279.atom" />
<meta name="HW.pisa" content="ggg;2/10/1279" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Functional Analysis With a Barcoder Yeast Gene Overexpression System" />
<meta name="DC.Identifier" content="10.1534/g3.112.003400" />
<meta name="DC.Date" content="2012-10-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2012 A. C. Douglas et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Systematic analysis of gene overexpression phenotypes provides an insight into gene function, enzyme targets, and biological pathways. Here, we describe a novel functional genomics platform that enables a highly parallel and systematic assessment of overexpression phenotypes in pooled cultures. First, we constructed a genome-level collection of ~5100 yeast barcoder strains, each of which carries a unique barcode, enabling pooled fitness assays with a barcode microarray or sequencing readout. Second, we constructed a yeast open reading frame (ORF) galactose-induced overexpression array by generating a genome-wide set of yeast transformants, each of which carries an individual plasmid-born and sequence-verified ORF derived from the Saccharomyces cerevisiae full-length EXpression-ready (FLEX) collection. We combined these collections genetically using synthetic genetic array methodology, generating ~5100 strains, each of which is barcoded and overexpresses a specific ORF, a set we termed “barFLEX.” Additional synthetic genetic array allows the barFLEX collection to be moved into different genetic backgrounds. As a proof-of-principle, we describe the properties of the barFLEX overexpression collection and its application in synthetic dosage lethality studies under different environmental conditions." />
<meta name="DC.Contributor" content="Alison C. Douglas" />
<meta name="DC.Contributor" content="Andrew M. Smith" />
<meta name="DC.Contributor" content="Sara Sharifpoor" />
<meta name="DC.Contributor" content="Zhun Yan" />
<meta name="DC.Contributor" content="Tanja Durbic" />
<meta name="DC.Contributor" content="Lawrence E. Heisler" />
<meta name="DC.Contributor" content="Anna Y. Lee" />
<meta name="DC.Contributor" content="Owen Ryan" />
<meta name="DC.Contributor" content="Hendrikje Göttert" />
<meta name="DC.Contributor" content="Anu Surendra" />
<meta name="DC.Contributor" content="Dewald van Dyk" />
<meta name="DC.Contributor" content="Guri Giaever" />
<meta name="DC.Contributor" content="Charles Boone" />
<meta name="DC.Contributor" content="Corey Nislow" />
<meta name="DC.Contributor" content="Brenda J. Andrews" />
<meta name="article:published_time" content="2012-10-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Functional Analysis With a Barcoder Yeast Gene Overexpression System" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Systematic analysis of gene overexpression phenotypes provides an insight into gene function, enzyme targets, and biological pathways. Here, we describe a novel functional genomics platform that enables a highly parallel and systematic assessment of overexpression phenotypes in pooled cultures. First, we constructed a genome-level collection of ~5100 yeast barcoder strains, each of which carries a unique barcode, enabling pooled fitness assays with a barcode microarray or sequencing readout. Second, we constructed a yeast open reading frame (ORF) galactose-induced overexpression array by generating a genome-wide set of yeast transformants, each of which carries an individual plasmid-born and sequence-verified ORF derived from the &lt;i&gt;Saccharomyces cerevisiae&lt;/i&gt; full-length EXpression-ready (FLEX) collection. We combined these collections genetically using synthetic genetic array methodology, generating ~5100 strains, each of which is barcoded and overexpresses a specific ORF, a set we termed “barFLEX.” Additional synthetic genetic array allows the barFLEX collection to be moved into different genetic backgrounds. As a proof-of-principle, we describe the properties of the barFLEX overexpression collection and its application in synthetic dosage lethality studies under different environmental conditions.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2012/10/01" />
<meta name="citation_mjid" content="ggg;2/10/1279" />
<meta name="citation_id" content="2/10/1279" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/2/10/1279" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/2/10/1279.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/2/10/1279.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/2/10/1279.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.112.003400" />
<meta name="citation_pmid" content="23050238" />
<meta name="citation_volume" content="2" />
<meta name="citation_issue" content="10" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="1279" />
<meta name="citation_lastpage" content="1289" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Alison C. Douglas" />
<meta name="citation_author_institution" content="Department of Molecular Genetics, University of Toronto, Ontario M5S 1A8, Canada" />
<meta name="citation_author_institution" content="Banting and Best Department of Medical Research, University of Toronto, Ontario M5G 1L6, Canada" />
<meta name="citation_author_institution" content="Donnelly Centre, University of Toronto, Ontario M5S 3E1, Canada, and" />
<meta name="citation_author" content="Andrew M. Smith" />
<meta name="citation_author_institution" content="Department of Molecular Genetics, University of Toronto, Ontario M5S 1A8, Canada" />
<meta name="citation_author_institution" content="Banting and Best Department of Medical Research, University of Toronto, Ontario M5G 1L6, Canada" />
<meta name="citation_author_institution" content="Donnelly Centre, University of Toronto, Ontario M5S 3E1, Canada, and" />
<meta name="citation_author" content="Sara Sharifpoor" />
<meta name="citation_author_institution" content="Donnelly Centre, University of Toronto, Ontario M5S 3E1, Canada, and" />
<meta name="citation_author" content="Zhun Yan" />
<meta name="citation_author_institution" content="Donnelly Centre, University of Toronto, Ontario M5S 3E1, Canada, and" />
<meta name="citation_author_institution" content="Department of Pharmaceutical Sciences, University of Toronto, Ontario M5S 3M2, Canada" />
<meta name="citation_author" content="Tanja Durbic" />
<meta name="citation_author_institution" content="Donnelly Centre, University of Toronto, Ontario M5S 3E1, Canada, and" />
<meta name="citation_author" content="Lawrence E. Heisler" />
<meta name="citation_author_institution" content="Donnelly Centre, University of Toronto, Ontario M5S 3E1, Canada, and" />
<meta name="citation_author_institution" content="Department of Pharmaceutical Sciences, University of Toronto, Ontario M5S 3M2, Canada" />
<meta name="citation_author" content="Anna Y. Lee" />
<meta name="citation_author_institution" content="Donnelly Centre, University of Toronto, Ontario M5S 3E1, Canada, and" />
<meta name="citation_author_institution" content="Department of Pharmaceutical Sciences, University of Toronto, Ontario M5S 3M2, Canada" />
<meta name="citation_author" content="Owen Ryan" />
<meta name="citation_author_institution" content="Department of Molecular Genetics, University of Toronto, Ontario M5S 1A8, Canada" />
<meta name="citation_author_institution" content="Banting and Best Department of Medical Research, University of Toronto, Ontario M5G 1L6, Canada" />
<meta name="citation_author_institution" content="Donnelly Centre, University of Toronto, Ontario M5S 3E1, Canada, and" />
<meta name="citation_author" content="Hendrikje Göttert" />
<meta name="citation_author_institution" content="Department of Molecular Genetics, University of Toronto, Ontario M5S 1A8, Canada" />
<meta name="citation_author_institution" content="Banting and Best Department of Medical Research, University of Toronto, Ontario M5G 1L6, Canada" />
<meta name="citation_author_institution" content="Donnelly Centre, University of Toronto, Ontario M5S 3E1, Canada, and" />
<meta name="citation_author" content="Anu Surendra" />
<meta name="citation_author_institution" content="Donnelly Centre, University of Toronto, Ontario M5S 3E1, Canada, and" />
<meta name="citation_author" content="Dewald van Dyk" />
<meta name="citation_author_institution" content="Banting and Best Department of Medical Research, University of Toronto, Ontario M5G 1L6, Canada" />
<meta name="citation_author_institution" content="Donnelly Centre, University of Toronto, Ontario M5S 3E1, Canada, and" />
<meta name="citation_author" content="Guri Giaever" />
<meta name="citation_author_institution" content="Department of Molecular Genetics, University of Toronto, Ontario M5S 1A8, Canada" />
<meta name="citation_author_institution" content="Donnelly Centre, University of Toronto, Ontario M5S 3E1, Canada, and" />
<meta name="citation_author_institution" content="Department of Pharmaceutical Sciences, University of Toronto, Ontario M5S 3M2, Canada" />
<meta name="citation_author" content="Charles Boone" />
<meta name="citation_author_institution" content="Department of Molecular Genetics, University of Toronto, Ontario M5S 1A8, Canada" />
<meta name="citation_author_institution" content="Banting and Best Department of Medical Research, University of Toronto, Ontario M5G 1L6, Canada" />
<meta name="citation_author_institution" content="Donnelly Centre, University of Toronto, Ontario M5S 3E1, Canada, and" />
<meta name="citation_author_email" content="brenda.andrews@utoronto.ca" />
<meta name="citation_author_email" content="corey.nislow@utoronto.ca" />
<meta name="citation_author_email" content="charlie.boone@utoronto.ca" />
<meta name="citation_author" content="Corey Nislow" />
<meta name="citation_author_institution" content="Department of Molecular Genetics, University of Toronto, Ontario M5S 1A8, Canada" />
<meta name="citation_author_institution" content="Banting and Best Department of Medical Research, University of Toronto, Ontario M5G 1L6, Canada" />
<meta name="citation_author_institution" content="Donnelly Centre, University of Toronto, Ontario M5S 3E1, Canada, and" />
<meta name="citation_author_email" content="brenda.andrews@utoronto.ca" />
<meta name="citation_author_email" content="corey.nislow@utoronto.ca" />
<meta name="citation_author_email" content="charlie.boone@utoronto.ca" />
<meta name="citation_author" content="Brenda J. Andrews" />
<meta name="citation_author_institution" content="Department of Molecular Genetics, University of Toronto, Ontario M5S 1A8, Canada" />
<meta name="citation_author_institution" content="Banting and Best Department of Medical Research, University of Toronto, Ontario M5G 1L6, Canada" />
<meta name="citation_author_institution" content="Donnelly Centre, University of Toronto, Ontario M5S 3E1, Canada, and" />
<meta name="citation_author_email" content="brenda.andrews@utoronto.ca" />
<meta name="citation_author_email" content="corey.nislow@utoronto.ca" />
<meta name="citation_author_email" content="charlie.boone@utoronto.ca" />
<meta name="citation_reference" content="citation_title=Mol. Cell Proteomics;citation_author=G. D. Amoutzias;citation_author=Y. He;citation_author=K. S. Lilley;citation_author=Y. Van de Peer;citation_author=S. G. Oliver;citation_title=EVALUATION AND PROPERTIES OF THE BUDDING YEAST PHOSPHOPROTEOME;citation_pages=M111 009555;citation_volume=11;citation_year=2012" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=A. Baryshnikova;citation_author=M. Costanzo;citation_author=Y. Kim;citation_author=H. Ding;citation_author=J. Koh;citation_title=Quantitative analysis of fitness and genetic interactions in yeast on a genome scale.;citation_pages=1017-1024;citation_volume=7;citation_year=2010;citation_issue=12;citation_pmid=21076421;citation_doi=10.1038/nmeth.1534" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=V. I. Bashkirov;citation_author=E. V. Bashkirova;citation_author=E. Haghnazari;citation_author=W. D. Heyer;citation_title=Direct Kinase-to-Kinase Signaling Mediated by the FHA Phosphoprotein Recognition Domain of the Dun1 DNA Damage Checkpoint Kinase;citation_pages=1441-1452;citation_volume=23;citation_year=2003;citation_issue=4;citation_pmid=12556502;citation_doi=10.1128/MCB.23.4.1441-1452.2003" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=A. Breitkreutz;citation_author=H. Choi;citation_author=J. R. Sharom;citation_author=L. Boucher;citation_author=V. Neduva;citation_title=A Global Protein Kinase and Phosphatase Interaction Network in Yeast;citation_pages=1043-1046;citation_volume=328;citation_year=2010;citation_issue=5981;citation_pmid=20489023;citation_doi=10.1126/science.1176495" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=G. Chua;citation_author=Q. D. Morris;citation_author=R. Sopko;citation_author=M. D. Robinson;citation_author=O. Ryan;citation_title=Identifying transcription factor functions and targets by phenotypic activation.;citation_volume=103;citation_year=2006;citation_pmid=16880382;citation_doi=10.1073/pnas.0605140103" />
<meta name="citation_reference" content="citation_journal_title=Current opinion in microbiology;citation_journal_abbrev=Curr Opin Microbiol;citation_author=G. Chua;citation_author=M. D. Robinson;citation_author=Q. Morris;citation_author=T. R. Hughes;citation_title=Transcriptional networks: reverse-engineering gene regulation on a global scale.;citation_pages=638-646;citation_volume=7;citation_year=2004;citation_issue=6;citation_pmid=15556037;citation_doi=10.1016/j.mib.2004.10.009" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=M. Costanzo;citation_author=A. Baryshnikova;citation_author=J. Bellay;citation_author=Y. Kim;citation_author=E. D. Spear;citation_title=The Genetic Landscape of a Cell;citation_pages=425-431;citation_volume=327;citation_year=2010;citation_issue=5964;citation_pmid=20093466;citation_doi=10.1126/science.1180823" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=S. J. Dixon;citation_author=Y. Fedyshyn;citation_author=J. L. Koh;citation_author=T. S. Prasad;citation_author=C. Chahwan;citation_title=Significant conservation of synthetic lethal genetic interaction networks between distantly related eukaryotes.;citation_volume=105;citation_year=2008;citation_pmid=18931302;citation_doi=10.1073/pnas.0806261105" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=R. D. Dowell;citation_author=O. Ryan;citation_author=A. Jansen;citation_author=D. Cheung;citation_author=S. Agarwala;citation_title=Genotype to Phenotype: A Complex Problem;citation_pages=469-469;citation_volume=328;citation_year=2010;citation_issue=5977;citation_pmid=20413493;citation_doi=10.1126/science.1189015" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=J. A. Downs;citation_author=N. F. Lowndes;citation_author=S. P. Jackson;citation_title=A role for Saccharomyces cerevisiae histone H2A in DNA repair.;citation_pages=1001-1004;citation_volume=408;citation_year=2000;citation_issue=6815;citation_pmid=11140636;citation_doi=10.1038/35050000" />
<meta name="citation_reference" content="citation_journal_title=BioEssays :  news and reviews in molecular, cellular and developmental biology;citation_journal_abbrev=Bioessays;citation_author=S. J. Elledge;citation_author=Z. Zhou;citation_author=J. B. Allen;citation_author=T. A. Navas;citation_title=DNA damage and cell cycle regulation of ribonucleotide reductase.;citation_pages=333-339;citation_volume=15;citation_year=1993;citation_issue=5;citation_pmid=8343143;citation_doi=10.1002/bies.950150507" />
<meta name="citation_reference" content="citation_journal_title=Methods in enzymology;citation_journal_abbrev=Methods Enzymol;citation_author=E. Ericson;citation_author=S. Hoon;citation_author=R. P. St Onge;citation_author=G. Giaever;citation_author=C. Nislow;citation_title=Exploring gene function and drug action using chemogenomic dosage assays.;citation_pages=233-255;citation_volume=470;citation_year=2010;citation_pmid=20946813;citation_doi=10.1016/S0076-6879(10)70010-0" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=D. M. Gelperin;citation_author=M. A. White;citation_author=M. L. Wilkinson;citation_author=Y. Kon;citation_author=L. A. Kung;citation_title=Biochemical and genetic analysis of the yeast proteome with a movable ORF collection;citation_pages=2816-2826;citation_volume=19;citation_year=2005;citation_issue=23;citation_pmid=16322557;citation_doi=10.1101/gad.1362105" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=G. Giaever;citation_author=A. M. Chu;citation_author=L. Ni;citation_author=C. Connelly;citation_author=L. Riles;citation_title=Functional profiling of the Saccharomyces cerevisiae genome.;citation_pages=387-391;citation_volume=418;citation_year=2002;citation_issue=6896;citation_pmid=12140549;citation_doi=10.1038/nature00935" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Protoc;citation_author=R. D. Gietz;citation_author=R. H. Schiestl;citation_title=Large-scale high-efficiency yeast transformation using the LiAc/SS carrier DNA/PEG method.;citation_pages=38-41;citation_volume=2;citation_year=2007;citation_issue=1;citation_pmid=17401336;citation_doi=10.1038/nprot.2007.15" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=M. E. Hillenmeyer;citation_author=E. Fung;citation_author=J. Wildenhain;citation_author=S. E. Pierce;citation_author=S. Hoon;citation_title=The Chemical Genomic Portrait of Yeast: Uncovering a Phenotype for All Genes;citation_pages=362-365;citation_volume=320;citation_year=2008;citation_issue=5874;citation_pmid=18420932;citation_doi=10.1126/science.1150021" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=Y. Ho;citation_author=A. Gruhler;citation_author=A. Heilbut;citation_author=G. D. Bader;citation_author=L. Moore;citation_title=Systematic identification of protein complexes in Saccharomyces cerevisiae by mass spectrometry.;citation_pages=180-183;citation_volume=415;citation_year=2002;citation_issue=6868;citation_pmid=11805837;citation_doi=10.1038/415180a" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=C. H. Ho;citation_author=L. Magtanong;citation_author=S. L. Barker;citation_author=D. Gresham;citation_author=S. Nishimura;citation_title=A molecular barcoded yeast ORF library enables mode-of-action analysis of bioactive compounds.;citation_pages=369-377;citation_volume=27;citation_year=2009;citation_issue=4;citation_pmid=19349972;citation_doi=10.1038/nbt.1534" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=Y. Hu;citation_author=A. Rolfs;citation_author=B. Bhullar;citation_author=T. V. Murthy;citation_author=C. Zhu;citation_title=Approaching a complete repository of sequence-verified protein-encoding clones for Saccharomyces cerevisiae.;citation_volume=17;citation_year=2007;citation_pmid=17322287;citation_doi=10.1101/gr.6037607" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=M. Huang;citation_author=Z. Zhou;citation_author=S. J. Elledge;citation_title=The DNA replication and damage checkpoint pathways induce transcription by inhibition of the Crt1 repressor.;citation_pages=595-605;citation_volume=94;citation_year=1998;citation_issue=5;citation_pmid=9741624;citation_doi=10.1016/S0092-8674(00)81601-3" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=S. Kaluarachchi;citation_author=H. Friesen;citation_author=A. Baryshnikova;citation_author=J. P. Lambert;citation_author=Y. T. Chong;citation_title=Exploring the yeast acetylome using functional genomics.;citation_pages=936-948;citation_volume=149;citation_year=2012;citation_issue=4;citation_pmid=22579291;citation_doi=10.1016/j.cell.2012.02.064" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=E. S. Kroll;citation_author=K. M. Hyland;citation_author=P. Hieter;citation_author=J. J. Li;citation_title=Establishing genetic interactions by a synthetic dosage lethality phenotype.;citation_pages=95-102;citation_volume=143;citation_year=1996;citation_pmid=8722765" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=W. Lee;citation_author=R. P. St Onge;citation_author=M. Proctor;citation_author=P. Flaherty;citation_author=M. I. Jordan;citation_title=Genome-wide requirements for resistance to functionally distinct DNA-damaging agents.;citation_pages=e24-e24;citation_volume=1;citation_year=2005;citation_issue=2;citation_pmid=16121259;citation_doi=10.1371/journal.pgen.0010024" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=J. Y. Masson;citation_author=D. Ramotar;citation_title=The Saccharomyces cerevisiae IMP2 gene encodes a transcriptional activator that mediates protection against DNA damage caused by bleomycin and other oxidants;citation_pages=2091-2100;citation_volume=16;citation_year=1996;citation_issue=5;citation_pmid=8628275" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Cell;citation_author=R. S. McIsaac;citation_author=S. J. Silverman;citation_author=M. N. McClean;citation_author=P. A. Gibney;citation_author=J. Macinskas;citation_title=Fast-acting and nearly gratuitous induction of gene expression and protein depletion in Saccharomyces cerevisiae.;citation_volume=22;citation_year=2011;citation_pmid=21965290;citation_doi=10.1091/mbc.E11-05-0466" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=V. Measday;citation_author=K. Baetz;citation_author=J. Guzzo;citation_author=K. Yuen;citation_author=T. Kwok;citation_title=Systematic yeast synthetic lethal and synthetic dosage lethal screens identify genes required for chromosome segregation.;citation_volume=102;citation_year=2005;citation_pmid=16172405;citation_doi=10.1073/pnas.0503504102" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=W. Niu;citation_author=Z. Li;citation_author=W. Zhan;citation_author=V. R. Iyer;citation_author=E. M. Marcotte;citation_title=Mechanisms of cell cycle control revealed by a systematic and quantitative overexpression screen in S. cerevisiae.;citation_pages=e1000120-e1000120;citation_volume=4;citation_year=2008;citation_issue=7;citation_pmid=18617996;citation_doi=10.1371/journal.pgen.1000120" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=S. E. Pierce;citation_author=E. L. Fung;citation_author=D. F. Jaramillo;citation_author=A. M. Chu;citation_author=R. W. Davis;citation_title=A unique and universal molecular barcode array.;citation_pages=601-603;citation_volume=3;citation_year=2006;citation_issue=8;citation_pmid=16862133;citation_doi=10.1038/nmeth905" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Protoc;citation_author=S. E. Pierce;citation_author=R. W. Davis;citation_author=C. Nislow;citation_author=G. Giaever;citation_title=Genome-wide analysis of barcoded Saccharomyces cerevisiae gene-deletion mutants in pooled cultures.;citation_pages=2958-2974;citation_volume=2;citation_year=2007;citation_issue=11;citation_pmid=18007632;citation_doi=10.1038/nprot.2007.427" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=M. D. Robinson;citation_author=J. Grigull;citation_author=N. Mohammad;citation_author=T. R. Hughes;citation_title=FunSpec: a web-based cluster interpreter for yeast.;citation_pages=35-35;citation_volume=3;citation_year=2002;citation_pmid=12431279;citation_doi=10.1186/1471-2105-3-35" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=S. Sharifpoor;citation_author=D. van Dyk;citation_author=M. Costanzo;citation_author=A. Baryshnikova;citation_author=H. Friesen;citation_title=Functional wiring of the yeast kinome revealed by global analysis of genetic network motifs.;citation_volume=22;citation_year=2012;citation_pmid=22282571;citation_doi=10.1101/gr.129213.111" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=A. M. Smith;citation_author=L. E. Heisler;citation_author=J. Mellor;citation_author=F. Kaper;citation_author=M. J. Thompson;citation_title=Quantitative phenotyping via deep barcode sequencing.;citation_volume=19;citation_year=2009;citation_pmid=19622793;citation_doi=10.1101/gr.093955.109" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=A. M. Smith;citation_author=L. E. Heisler;citation_author=R. P. St Onge;citation_author=E. Farias-Hesson;citation_author=I. M. Wallace;citation_title=Highly-multiplexed barcode sequencing: an efficient method for parallel analysis of pooled samples.;citation_volume=38;citation_year=2010;citation_pmid=20460461;citation_doi=10.1093/nar/gkq368" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=R. Sopko;citation_author=D. Huang;citation_author=N. Preston;citation_author=G. Chua;citation_author=B. Papp;citation_title=Mapping pathways and phenotypes by systematic gene overexpression.;citation_pages=319-330;citation_volume=21;citation_year=2006;citation_issue=3;citation_pmid=16455487;citation_doi=10.1016/j.molcel.2005.12.011" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=R. P. St Onge;citation_author=R. Mani;citation_author=J. Oh;citation_author=M. Proctor;citation_author=E. Fung;citation_title=Systematic pathway analysis using high-resolution fitness profiling of combinatorial gene deletions.;citation_pages=199-206;citation_volume=39;citation_year=2007;citation_issue=2;citation_pmid=17206143;citation_doi=10.1038/ng1948" />
<meta name="citation_reference" content="citation_journal_title=Methods in molecular biology (Clifton, N.J.);citation_journal_abbrev=Methods Mol Biol;citation_author=A. H. Tong;citation_author=C. Boone;citation_title=Synthetic genetic array analysis in Saccharomyces cerevisiae.;citation_pages=171-192;citation_volume=313;citation_year=2006;citation_pmid=16118434;citation_doi=10.1385/1-59259-958-3:171" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=A. H. Tong;citation_author=M. Evangelista;citation_author=A. B. Parsons;citation_author=H. Xu;citation_author=G. D. Bader;citation_title=Systematic Genetic Analysis with Ordered Arrays of Yeast Deletion Mutants;citation_pages=2364-2368;citation_volume=294;citation_year=2001;citation_issue=5550;citation_pmid=11743205;citation_doi=10.1126/science.1065810" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=A. H. Tong;citation_author=G. Lesage;citation_author=G. D. Bader;citation_author=H. Ding;citation_author=H. Xu;citation_title=Global Mapping of the Yeast Genetic Interaction Network;citation_pages=808-813;citation_volume=303;citation_year=2004;citation_issue=5659;citation_pmid=14764870;citation_doi=10.1126/science.1091317" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=Z. Yan;citation_author=M. Costanzo;citation_author=L. E. Heisler;citation_author=J. Paw;citation_author=F. Kaper;citation_title=Yeast Barcoders: a chemogenomic application of a universal donor-strain collection carrying bar-code identifiers.;citation_pages=719-725;citation_volume=5;citation_year=2008;citation_issue=8;citation_pmid=18622398;citation_doi=10.1038/nmeth.1231" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=X. Zhao;citation_author=R. Rothstein;citation_title=The Dun1 checkpoint kinase phosphorylates and regulates the ribonucleotide reductase inhibitor Sml1;citation_pages=3746-3751;citation_volume=99;citation_year=2002;citation_issue=6;citation_pmid=11904430;citation_doi=10.1073/pnas.062502299" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=Z. Zhou;citation_author=S. J. Elledge;citation_title=DUN1 encodes a protein kinase that controls the DNA damage response in yeast.;citation_pages=1119-1127;citation_volume=75;citation_year=1993;citation_issue=6;citation_pmid=8261511;citation_doi=10.1016/0092-8674(93)90321-G" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=H. Zhu;citation_author=J. F. Klemic;citation_author=S. Chang;citation_author=P. Bertone;citation_author=A. Casamayor;citation_title=Analysis of yeast protein kinases using protein chips.;citation_pages=283-289;citation_volume=26;citation_year=2000;citation_issue=3;citation_pmid=11062466;citation_doi=10.1038/81576" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Functional Analysis With a Barcoder Yeast Gene Overexpression System" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/2/10.cover-source.jpg" />
<meta name="twitter:description" content="Systematic analysis of gene overexpression phenotypes provides an insight into gene function, enzyme targets, and biological pathways. Here, we describe a novel functional genomics platform that enables a highly parallel and systematic assessment of overexpression phenotypes in pooled cultures. First, we constructed a genome-level collection of ~5100 yeast barcoder strains, each of which carries a unique barcode, enabling pooled fitness assays with a barcode microarray or sequencing readout. Second, we constructed a yeast open reading frame (ORF) galactose-induced overexpression array by generating a genome-wide set of yeast transformants, each of which carries an individual plasmid-born and sequence-verified ORF derived from the Saccharomyces cerevisiae full-length EXpression-ready (FLEX) collection. We combined these collections genetically using synthetic genetic array methodology, generating ~5100 strains, each of which is barcoded and overexpresses a specific ORF, a set we termed “barFLEX.” Additional synthetic genetic array allows the barFLEX collection to be moved into different genetic backgrounds. As a proof-of-principle, we describe the properties of the barFLEX overexpression collection and its application in synthetic dosage lethality studies under different environmental conditions." />
<meta name="og-title" property="og:title" content="Functional Analysis With a Barcoder Yeast Gene Overexpression System" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/2/10/1279" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Systematic analysis of gene overexpression phenotypes provides an insight into gene function, enzyme targets, and biological pathways. Here, we describe a novel functional genomics platform that enables a highly parallel and systematic assessment of overexpression phenotypes in pooled cultures. First, we constructed a genome-level collection of ~5100 yeast barcoder strains, each of which carries a unique barcode, enabling pooled fitness assays with a barcode microarray or sequencing readout. Second, we constructed a yeast open reading frame (ORF) galactose-induced overexpression array by generating a genome-wide set of yeast transformants, each of which carries an individual plasmid-born and sequence-verified ORF derived from the Saccharomyces cerevisiae full-length EXpression-ready (FLEX) collection. We combined these collections genetically using synthetic genetic array methodology, generating ~5100 strains, each of which is barcoded and overexpresses a specific ORF, a set we termed “barFLEX.” Additional synthetic genetic array allows the barFLEX collection to be moved into different genetic backgrounds. As a proof-of-principle, we describe the properties of the barFLEX overexpression collection and its application in synthetic dosage lethality studies under different environmental conditions." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/2/10.cover-source.jpg" />
    <title>Functional Analysis With a Barcoder Yeast Gene Overexpression System | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"FZ-1kPUIcIwfZxmwmPiioGaBq5dMczhoS4pnnjt6ULo","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459341","apath":"\/ggg\/2\/10\/1279.atom","pisa":"ggg;2\/10\/1279","ac":{"\/ggg\/2\/10\/1279.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/2\/10\/1279.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;2\/10\/1279"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;2\/10\/1279"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/2\/10\/1279":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022OcZ9V6K5yGGwnQNN8WJQRxR28oGGjmsUy21U5vRgsB6EqkDQFCUu\\\\\\\/AdJEwo19vT7HhLoHEybeoQEaTupFiP2xZtZAiTknL0ebPlTR1sRn4icLb0VeyA+J12Iu2\\\\\\\/4AhzRqgDoJibB6su9838QQGGANHslQuiZbDxuwWeiOdFpfS1xE9VSpVg0K9D0nY\\\\\\\/HRCTpYctLJQkR78onSeGiJU+OCkwgRXjqr6DO9fvu+PiQpgSJ9gQa0yoAk1FzZAFofJYWLSJpGQ+tjdbETAJPYuV\\\\\\\/fjqgn7HP7kodbgwaPNrgbt8ZN1mLOSu7t6XfWwUGGcjVFym7qXYnebmlmrs9FDJpkpNPRo73qUjqAE6xuANbpTd8F7qcJMLdvc3mA4PTtaO+h\\\\\\\/Cpv7uOCNEeH9HycBkNS\\\\\\\/7CwtuAPphm2HOOiA5hbOqlMWD3aN7oyUKTCMVH2DoLxQ++hNr0xpFUTLwoAF\\\\\\\/c+pxt+Urlw5iMLD1AwQoK6VaJNLHUV79WI9eaAFiZz8tB\\\\\\\/lK9ZIzwNFNejPzLuNp81QHSdyjfnCxLr2jo4oi6twJS9srnDVyG\\\\\\\/VT+aDLRLyoloExyz3qdjKD2xu+JTiwTiBSpXcKFzKjlPBGytPBPXEBR4iMd9pCedg1j05ODoQoY28mmzDnFXaGiv3Hs\\\\\\\/vcNaln3UexWyu8bxt6twJGZ+s6+J7kc0+lxdXDrnT0T6jn\\\\\\\/W+3jL1bAouhFGjVgDCrTZZA1qmq3YH4PW2bPGwv6bf070vS9jxYFeoWw\\\\\\\/8u1EkgBflFdzexaVRuvws2JeVqhTVEv0aBgmwUotHhtmoyu9ymAmMRAnpANPhvzTiraZCAG1KAO++qnF\\\\\\\/gFtz4a3anBUf\\\\\\\/u1l2UAOzUf8uK5e7rg3AyKw378FpUsx5mr76TQIvsQ+N2G3qxWLV7Vdq7g9a\\\\\\\/yTBETLDABXW7jClO2Nnmu+sx3mBI4ukXKBdm3b579O4fRAVP\\\\\\\/QXGsj3auYxO8\\\\\\\/WD\\\\\\\/psAl2D27jyNolkKHYBPsC\\\\\\\/ep6DXh8jPTTQbSWaBpbn2tO\\\\\\\/jmDbJ\\\\\\\/lJcPoDjsogHxuAyc4uyLLn8TkKQmFSed+puHikmuPYgvBK5njLu31y2OTaOZQdNhBNGmjimg4zaf+uGlRbTs9SKGN19NSGqHEtJ27eIHuOsuthHKQAQEcWts+AvZtFyDr9MmKKyeAk1lREWb8eCgJ1QgOJn0SwJbo2DLglRyFB4GddSFQ7oN9CmNpoJNYhzVPcb9g6l526Tqs9s12euAI4GOb72QXaE\\\\\\\/for4RlIzhOaOq3xq6utwuKc\\\u0022,\\\u0022iv\\\u0022:\\\u0022pHOw8YZjNz762328mp4CDg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00224cc74e92372756c8c675446177720677126677cf0341d05fc679d0d493c8436d\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022LuAnDNr17T\\\\\\\/4xny\\\\\\\/rAIqL95gxuOXEU9s47jifB4NKkRCIOuO6hqwii3UZPpDin4FygJBjNlHWfWnrPW8UsyWneoxNN43yqxo25J4yIbeh339MkAYUjUBlYD3AeScXE9PqQgCfHUVP6nvU3gKKnam3aEnlZOHZzXZtbsjy8c34QUVXszrSYqzQuTJsRuz3WNy\\\\\\\/wUximEdQQ50H4ZV07BrvvL03Z9c0orabet0t2akKytT6IRIHUT8iDT2xyx4TNXP\\\\\\\/Qd0ocMg6LCLwPtuWrcT8WKjbdGv9adcYd6Pr8s0UJ0Xr6YtdsWdi8am9ot1SHV+OkR5vhYCfU70r5kPr6uX8H4lga1ImIaEpiGDXImxoCJzNcnUg61Op5qEUtWISapJSJsdeRSwjyinjKiMBYQ+68EerM3DfMtqzY9OXiAdn4p3VYqLh0t5TID+M\\\\\\\/WCUlYfer23aR404tGLU1OG2A2TgjrGDSeeD+zrcBVV7bC8YRzROmRiuNlI8PtfrQ0esCJOzdZdQbJ+cnv8bMvjrgRBDF7Fr4nPIe1ck5RIGoZHEzXOnwadasftVPsvNOut0kHG0RwyoMH9mLytZRlHBrinTS5j+4b4sR+wDgPY3QmvsXUIKmsVA6lK0Js+iyYHKMnN05fhli15vx2DKOVJ+vCoI759r+wjHdcmROKyjd\\\\\\\/K1lTwpK\\\\\\\/bmzyUWSzPLTjU\\\\\\\/LHtP+41DxkieNf07wexpGhAHglRiRLRooyvE+KUziUHxkb8utagWfzgPjWRAs4Fy3HmVsygf7HP0JZPOspXKMjRotmYyHdRfbd64dQO8GoauNJAr4lg6hX7mrNUoLbww5LaL1fYtvNeGSvqgfPwRPOM7oqBZFdW6ZadEW7GKBirLLVMl8jkhOlVUvANSxAioQgoP72qq0HujaeGxoqZnLWXlpz9dDS8Yutz7cdv4QUhIl++KQHhXSpkFIZuNR6Z2oc5WruonrL9c\\\\\\\/DQ1HnkmIqSoXcB3S7\\\\\\\/qUOpLQLMsXx9gxsGJ0UNC9nZKC\\\\\\\/RalwpAIJzWJRLAOYeE33Kr22\\\\\\\/G6Dw8rdAfhuTE5ekaZv2WlznWWcx4IPk3cuuJEgi2boue59yXfGV6J+2mN9LinbZs3UfiY6R69mpCWdY7I+\\\\\\\/y1ol9f8khZEZDBlN9Kl0GpnROgO9QdePFYcvwh7DR6HV8PdBVKSQHqgl6\\\\\\\/zaeTLMHfZEPQ5vTO6iM115iFp6qk06+Vl1Cbkfd9B\\\\\\\/AwgoJLKydkaweBOOXvMP1Lo1rpmf9w5jCEvNi9YuZFJ+eqvD220hiNykbYIKSgNB3Malj0RFgxxumUTMJ3H6jNGkfaBsoZFvfOHquyqJhrP8qIsrkpo74PUzWaB74fp4xUqP7\\\\\\\/rPNpoKbcz39eOEHjps8M1ZADkIhKG3skjIiOTTXtGIGf3tKfyhhbiKCkpfEvv8AC\\\\\\\/TIIL\\\\\\\/By5QPAGBxk30K1j4JsRQzhqzjRaUE\\\\\\\/prbmZWcgxTytzICRzIyqxSB3\\\\\\\/pjwtuzhJM29j8d2miCTsyZFteC7\\\\\\\/sf4sWPCT1oo4o74bVob7Zdc2NovIGgfHjvt3JD92PlJB7B4dy0P+2rGbz+as13HvteI6Z3Eh71nOnCX10aRAihw01ZGa3oEp5Y+J4ZOoV1mqgZDLpE2O5\\\\\\\/Uz+1MTNw\\\\\\\/P9GCIi+ciRdj1L0bw0dRQ5CRjiTdmSqsw6fadmhbK5+X303hPQR3\\\\\\\/goGOzCqjdr7PneqKqOgedKTfrMsRO0uBuqR8oREfyvc3WZMxELdanE9BZ3FJWw3uNVF+8xYxScCPzTP2pcFohzprbrX1qqxSdgT7tWwZJPbbH5RBjzdX0jtuREn3OjeRSHf2eyArI2tcPoE0saNIoDV9fij\\\\\\\/Jj4QhadlXyp2\\\\\\\/bo0yd\\\\\\\/ydo5i59n9Dcfww202MMHv1FYXBUWR2NDLuAT4nq7xzFQFv+4FTSNl2zYBOf4IGRg\\\\\\\/F73lXhz\\\\\\\/p9IhjklrFWgYrtf1D7ZT1mG3yA6vX1RtolnA2vAoJ2CpDd0xK81bkrUx8v7QOK\\\\\\\/h5kPdA96rwxzedDSRYRiZRIfSQi80TIftU66or5eRwW6Xi9s2+h2ePEY4bClN21OI0ePTjr3JTJRzlYUF8XFwxruqXP6nTZrbPsutNCnmoCF5znaHGTwT7pw\\\\\\\/7mFHnYIlsrPbRqADiiueL+WRCGO5tMjugGxHkh\\\\\\\/LqzgvSWOLTQtQyWJkkxOoPI8Wm4qgVdM5qcBlRKKoifIzSmAfeXK1tn0jlBLxXGUJDtDXjbmVPnMuxhV6kxg9kBFqerbQfB7bSh8cJZFBQfrbh9IGhC9m57JQv9GqcEEXhRwcFkcTQSzg0QQzoW\\\\\\\/LqF44Y1s0jz1Wy8tbp+Ul6\\\\\\\/qJl7pRucWsJ0hNAYUHO2uZlgbMxSw589O\\\\\\\/v0DCHaAS7CVZb5d0eIwINsKaP1boC\\\\\\\/+TVk80foMGOwIplxe56s\\\\\\\/JjjZV8IxgVrI9EyOuLEyC4CIOqPHiey0dfLrFky3RDL3QSKthfRJyJn5tjFLoSSdE8RLxbvx3ddrLdtmItp3spXCkStWmcmjeiWnm2JV8QPGWEDsc4KxlfZBcb4JE68KE3CJbTkBmCbEhfRvFCeT5kFZQ5qciSam7FJU2IjAtBE5hoNiqkThZzv9wWxV7ojx2nWGL3NdhwdFgtMnYVIkfIz6Hx6ZfxDJ\\\\\\\/FWIgxbRWF\\\\\\\/Upfiiw5ASr5uXSq0ZkNQSOFXOssEuiEjADJht7vH2o3nt\\\\\\\/xyI3ZzIC4kQ45F03nbhodDijBTihYxfbQf06InO3DBCfvKUgX9PuLGweLDBMCufESi27GseOGd21X878wl794xYJWYwyP3YCH6LhK5xIWES1t\\\\\\\/bEgDxckci6zZl9a2Mskvd1faZJLqFhfWpCVXu\\\\\\\/VpijU1QwWIKPvnhAAkJzkr7NA\\\\\\\/WUohB+dMka59dYwlkTADU88YCHrkk2hlbCNhFGrN1k2cvtGtSfoil2t7OBFsrRTwb1h7V+r9V3FiBDyambqg2Q67l5qtHPPbkFQlTtUz8ZMYi\\\\\\\/RplTKT1xcp4VLfhK0metHCTEr+0jB7JL9ax6iDigJNbQRuhYRSjjLvW2oAuOA1S3VSbS9888cixIpnkGFzQ1J5uMK0WR5aIFgVZSiShgncB\\\\\\\/Bb7sRqcgx9ZnDxss\\\\\\\/OC5IA985iQOzzT3p4OTDmAU3QVLvYttxpoqgHF75JZCbzK4IdB5Pddv5uUWZpm8d3jOZp4ew\\\\\\\/xY4qHnF7BTVQAdcDjfcX8ptC0NJ\\\\\\\/O6tnnNTflCt\\\\\\\/ZV+u9hQ00+W\\\\\\\/cM2i1gZeHBwzcHWeEKuQnTdlH6j9ys+2yUXyRNYp+U3Gf0cyt3aYLyxTjdIqA2avzjsipJ6HGrFD5+ye+jawkyIVSsRhSq2cIqtWLVuPWfFcoVUK4TGGnHYLwa9+MqCeV8tpBM17n\\\\\\\/nKYN8hQie2ajMiUXZUmbjOPw1ENjf3fjyFou78hf1tEAf6cOQX8EdgxAuA0ZvauPnP5EQop2olHhZ7sdYnOMu5bt22dpNU5kw9fDbZoeXKLp4jPS\\\\\\\/OlsU5N\\\\\\\/rxV1xatw2zadnhd0xBNsIqzvF3c5aQKbud+88gZ6Tgx1F92Vva8Ge3n5nxkqyszh56+Vg4J3lK0HPLEiiOv3a8sGQg4DnV316+Z6+ZwuL83UEN\\\\\\\/2yZP5cuuUXq75XUmg53njVVsgmO5fSXYEgNiHwqyiEk4z5ALBSbJtQEY5rpzl36FtvlXNN+WCOfdJWzFw2VnCstwes76jbq1NU3RJvrXNYXO8C8yJzp6f6yUqrGfgOi6vMJ7Vy\\\\\\\/kDJ1VzrfVdT\\\\\\\/+S7qltG08wucdFG1QxbIW8pdxhCIP4oqB8hnaiPeKdz+p73shq1I+z4w2BT+ga8K9+NOZgn+1q2AWt4R5ZIHONuoyt1npZKFrUnASPcKEjNPijX9dkolwlfzQ5VT6ifKToJknoxEvSkW3yNyZY6zYuIEyqJYdhVkpbv4yqRP1ffHpStyxlITxQb9QhIDhkWhi4INHnpY\\\\\\\/ocq7HsH8lJmIZd21FRGouM3zuZFecrhpd7xL44AQ1tnuOxw30R38v6zs5Fi1nWN5OVMw6MAOIQtjx69lVYKIANl7B3c3NfZeAdCTpZEi4WVFLR7RhWrFWWbhSxnCbcSL59vzuQFvlOLG0dt1ZwpIrQ3Ff1KaIlbHyC3+XKq48md+\\\\\\\/+tL0MZ6QHWwCa8kle288plEf3VTKifS18cwjEFrX6vdvh9e1o07HuuhX3NsryWOXfM7\\\\\\\/CgTHOJU87c2P5thVJY894p3PJ8UA7un1fWNSETeqy7+ycdRwa+teAl2Dv01ZyPVNNTPSNb8ZIDJqBvfXGctkjhyNoZwYsHEmieLg5BWLG2OVHz2Ua60qutUKP28edCQdhEABBVeneqo\\\\\\\/ewH6XycNzaDZF0nIdxUKC37IhO5TukAbEaAvXek0XnE98Q3NoxVjMvlf7WPg6gwWNW8WAjTwayiIHeS2JW3imWoIxozkrVDvS5GqCCJqbi4wz3DOR3Yyv6500\\\\\\\/dbSG3NSn+X7hIXp9vo0gWiIpTy9CRlx51lBX2HtGzbVmQK2hbbvuFrwLWlJyShtUANwrq\\\\\\\/BFom7oFzb4PO4VKvJIY\\\\\\\/ZOiKaRHGQjpL2P9UTOBMBGC5\\\\\\\/63jscW6dif+53wXCQq\\\\\\\/fMgJM4zbpe9yTIgEpmdl0kq2KtDrct+BHFK42TrH9P9hOo1yAJaJVCSBq4RMQlkNgKssNDbO4Fk3bxThXKX2x7wnyDiHB+m5vB4wUJ3wNAaZiszQVcTL5Fp3\\\\\\\/HbjIDD1BkXwpMEzLTmuCKkXsl2ne2pjOe2vZV0ecNGrbH7JccZfaMNiq0MZv+PVt8gMl6htE0DEE22PGKe3EU8czeSh7HE8lEkWaNqdHuCmUizX3HW8WJ6PaYPDHOZm9U5Bqk7b1p\\\\\\\/mZkG+ODkYFThgEI2bhGh8qMMYSyTvYZV2tWqCtSYq\\\\\\\/zRPlcppTKx3tXfVAbTvfzADPS0zz1cvKkZUZMwSJFwcnLcyn5xw4yG7\\\\\\\/amS\\\\\\\/TPuDaa\\\\\\\/osuNvcPJUf4OLhaXTpkXFV7\\\\\\\/eMsb36yJ8Ge0RIhy\\\\\\\/zc5QMgb2R5NXA8toUGr9tdB7K8RJ3fHApr2hlG3YUy9uyw84XrliupzJkQzkadbYm3Fe2dczbqMmNA+HgaiDiuMBGQ6L6FH6Yo3boplflb\\\\\\\/lsK8Ruk6rZeogmAdVzRhUDnKXqjNxIAZ0sLn1jHqBc\\\\\\\/LA9AExDoo9kMfeG4xyI0vXhJd2KNbFGbKE4EsL1gbngdJOiWWGGNPJvQTIKSkLggFhFlmZo\\\\\\\/k5m88ZEXMn9\\\\\\\/tLtrxnLop+Tk5ADIlXnKuJyuE1EbNTNrnE4EBr6skwMTy\\\\\\\/k1uAJ8JF74gTUOheiQIUTymloTh\\\\\\\/u8EEHgiO7ZBqOrP9QoIdvuvVVsVxO7AHSxp1485+\\\\\\\/QDc38XV\\\\\\\/63Cb7a5Pp\\\\\\\/M5u5sBBDQvK1r+d+OUhEBoWauspxYQ+D5mTiAEJf9ElNNVfaiO75hotUoW4InMpVcGfY9LKgsSadDISdpkHPyegm5iDD\\\\\\\/cWraFTytjKaKAQ6phhNX61fLlamHcgZA1mUWXo\\\\\\\/6sEOIGGMkTlMd71MtzAis+2+9AY2BoeDVvXQ7hcGcASB7dZRS6SqaYC7VGIsOnkXtWdury2LcVgHmyfdyQ1b4HZvT4N85W4R4CsCUtWR1rVfY6\\\\\\\/3bWLoXblTRQZAzwMgHm7gRNj2beFAgB+DoWkgbPcB3PhpU923OYdkdC6u8C7+zbKsYf4F287q3OM36E2Ce32qVKa0W0k3D0nXbfV79Nm7iilvELQItgll43+ZWzcmP0TsqQ5q2BENLQ6zJGwwOncq+uURBQGW0j+Pa4rcNPER14APkyyR2kR1tfcn1khTVqLtwVDxEowa70QdFG7FZd+LjzakJmRI2rUncn+OwUC4lbf5BTLgAkUJLrxb6Jq3bZ559jjrbos5mVB3Ydh0hMDIrH11JfI1eJT\\\\\\\/OTe1izdRdhkSe26x\\\\\\\/XK2+\\\\\\\/0k6EEDiL4AfzyUAIYOIBt39AzOmD1QBml2Ijab1F1CHg2E8pThVUNy9+rkz4dfNwriSkhc7WctBnXkbTEIxYmo4HqEkCFznCCm4474GOa4POFYfLHKZ1Su43gThSLfxCen\\\\\\\/UC+G2IClGfym+OxlrAhtHBSJ4sLv+h4bnAkKed4mlXlic2DQBgU3MqXkwQ0IaAij2bXjDzmXXie08fG\\\\\\\/GJWB7dgiuZIJJj1uSizGHPoEJGJ31cQHz9vV9X8rphn3tBQ2Xrsjc+QNomTKuf71CNxtnUn1xmojleuD09QpmVG1GCe9UyZXnk8ig+g1lARcRfNh3p1vQgZo26k6+yejhkB3mRXeh4aGerhn8tT2Bza4j8ygJFtcP0crZdoZGd+xFoywjcPVjY18Hs\\\\\\\/HltEMnZiKYBUqwflLbyq+\\\\\\\/f5dgunovA5LyL67CAKAYVgqi3vznblPUGMlhM2mAf83IXU1jj1Xf3+n\\\\\\\/iRwuiDAB\\\\\\\/jm+\\\\\\\/KyBkr2H8D\\\\\\\/ojPbnO1vx4rgnGtvotrVBgb2DnazNc+WcqBGYLs5C744UYSHf1PqBrm3EuamJiY\\\\\\\/02Dpb9W2zcp2QrPx79e6joloioWCG\\\\\\\/s6ahpiUC9NErKeRplXudbfAlUW5jl3+xoQd2GdCAWIQxQFthxwEg5q8iowniGRF4RHo8INqtlsmjN9GK1khkRz4spXmOnHGCIa2zic515YStMnsGowi18Bz9wzOfd6W2bRzqz2bkzw7ZhHdnBiVno+CuumR149LRv3HubDAPzd84zblsWAjB2zZnL4MQ7Y6tlCPNSizGGKPmglnLeuazprwebwKLoXmaovpYhlI5lznXEgsy\\\\\\\/9FL63qRYguEccTj2y8SbIhSM7J3Ht2UPnbR1BXWChFiRuqIFmfiKQ56fW8SkQrVsND8kD4MivHHL0xGtudSRas\\\\\\\/X178iZ5MmaHjeJnuM7pFvfyoGG6w34JKnJ2amUxDNawMLpS4F5aNe8RKUWTNxNERJsGHXIZ84zQiq0D1Sr21U+J\\\\\\\/xaQWBSq94hCV33tJPWS2N3ZFKlFIAWNQyZ5pmeTQcU6EQDbKI7nX9CR5KuFaA2kFTQz3j4o2Gu1bkZgDgot+4XvJrhpQkGFdeqMn\\\\\\\/gEHH183jCQpUaQi3jME4gWhhl9xXJulDWiuDAJ438mZdrm8Y+WTTYdozA9dptNdq6T\\\\\\\/Cc4WotLtvW1uUEPCWcb9Pv2HSrrrZ0Fqk4FoNWMqcxPAsNpSH5jK1MqHNiVLWkF\\\\\\\/reipbaI7QAsr4VP3YqlqFTbRp9SdOCqAnAvfvG6gbAhuFgOWkpfN9vCnH\\\\\\\/+8BmvswbHnVt55A9w3citxoyTsTUHAYyhUCpgkmKDcQZzBihLmnTPC0nhxcavaNLPyDz54OKTqqjuh75vxnEIWv2Mc0c6x6PnFjK11NRb8UykfI0QR1cbUmd7nDSqUhFiwjWaiv9mkv0YvlbJiHOxa\\\\\\\/Pp1l6NO6uZPuE9jScI5kB\\\\\\\/JA6ODbUDJIIyKIYt3TaAl5WE+v4cYJ5L7lh6l0SRkBgySSLZg4IjXbK01HQZXxcxVB1PkS5qlRkq1W\\\\\\\/Q7KB2vt+F825FhC+j2\\\\\\\/KXxrtrYLHof1tOgHl0jX7UmdptPxRxlWwFzynn0SPR9ezHk9cUSs3aSgekmU4KchmcJEqjd\\\\\\\/BuF4RCbwgYp+f4hkRXJUBPGfTGzxNWmE\\\\\\\/ZMT8AZEzXpjF+PgaGnV84HC5j4ssjUjIv93iREcu9AbHFNQIIph6Fcb+YUqU13jIAYlQIMVrLQqX4pk9IBESrTca8fuagipmdFaZSkhkf8yNrw53pL0zllZYvr8FqL6GSF+\\\\\\\/ZqWicIzkGux\\\\\\\/OL51bYMn\\\\\\\/ApoXwNvx4ItcEOOK6E9DH2aiGH1+RXTooHbH3n+NgN4lwF3E\\\\\\\/ZSYZJYNAuEXY8NeVrh8FDTyYNausLFY9+cpXhqYFNu\\\\\\\/y2BCj1CFbr9heQoHhyt5Q7wSrPovO02sul8076sRSVnwgyjkEAuSibZhISfKDSoR05zJSWk+NjX9sOoNiNEOnEyNPPYe375PUOr7QfMRuLTtaLOvC2Cz\\\\\\\/sgoVCG+iHVQiH\\\\\\\/3cQl4MHKZc3bEtl\\\\\\\/KxeW0g66AfWZ2fkSQ1TKZuTka6CKIWk+gxTKaqu1sqEnNd2jU1ae+Nh6CgjVAKxAzCpHgHQbeUkRiurpFGZFSZtaa9Sk44bhkaNGOId8+qffwjXigLUrdLJFhQQCUvUtWk+D89NJP3o82WuGP0jpHrgO8zeI0aPmQkploACv9pRBRKakRHdrEza0YqNxkPZ63uprnCnUuhVg57kFVi3cGa6i8lbWgpg1POFepg\\\\\\\/hdb\\\\\\\/v0vQwrW8EowQYkU\\\\\\\/0D2DdyadY6StsBUPXs9X7Sxz9Lz0HoV8AzFazHc+bANsLEzV9qmkiNA60o3Qjke64Qw9k5+HrAIdCuFSajUpiwRkhsdUYBJ6PXz+DLB3pv22zr0pySO0TTOrql4Hj\\\\\\\/LK1qQd+i6kz5YP7qa9fcQLrTY0PAmfL9sjCjGcR7GdMlv1NLliqHaxgugHD73ZAzeTOJCqR4Phnd9U6r7WCAY3RstiVyLbXU33jYelRxenJphTANhXi8yo2xqLl8hNGc8uiQSqwPGlySIPL9gPKFTIm39AfTklMhmwURx3Xd65mpzcCLaQ+\\\\\\\/Uv8WGJuCTD2TK0qqoxXBh1AoP0wFXQk4493SuuXh7TJwm4X5VUZWMJeuk4Ols\\\\\\\/ztRfNqbZ\\\\\\\/2\\\\\\\/bKSp7If99jSKS5Gle5ZClgyJEVfyw1FfTZwOboCPNGXrTiT8D\\\\\\\/InuatYMezTvOGHkM\\\\\\\/k1FKYmXXXYhAfpXyCQ2V7siOBEnz7TgiXJ+Dc9AyyE5V0iJjvn7iCSq4EnAzhBr1Nqci0iAyQkM0KArlisNPIEkRYyBd9FGN4ZUJNQhRT3hOk95E+3i85NQvEeW6hQr7hKvlgrT6CSfU3r1xZ88ofn8Y\\\\\\\/rd4i9\\\\\\\/w2q21Penn8JdlC7lTu0JuIFUL1VD6MA3y4B7HP0Nn8AZ\\\\\\\/s61xBxydcG0jyCOn+Vj5HRURaohgMU1+OmDIX6UcHO\\\\\\\/ruIqGrXMXPjezpJ6Ovg9YY8KUOeSk9sYo7faT80s\\\\\\\/FW3KquO6CJgN2DZ24bChrjVfWkA6JHcYpkqydSHcwrqtDXzojLzMSH8f3GGtJTT2HIAXDpdHZesasCF2W0m+pB+GkoGSgB56ff5RZF6I6R9KV3\\\\\\\/6BN7ZhfnNwiBBdjKJLQh8vdXd54JiJ+dasNue9caAuS1iNJDv0IG6+mxR0e9AASodmYAQO78W\\\\\\\/lxpAiCXJRzKQIdpCZmPsczV7MbV3yXpVpJPCXUq\\\\\\\/6GqmVRkM5iTHWdDBtPlCCfzT4F2B\\\\\\\/WVI2wjzQwGUEumb755Almmpr2UIzI5IXaqzFDhIxK3ssGmIQ4iDieK3UeaeW69LJwMuFwfGr9FR25+ubk+JuGKS\\\\\\\/m8MIiZsSDPod953mYeiykWap5IFLG2BRRKDKycmLHzVEXv8EUVLTE9TNbGanOC+DItOjVrCx0CMYOuQapehdKGTjJVR0O820Mcsz+CSwdAhysEmZ2xSvO\\\\\\\/mldJmEuaK9n5p8HpEEVDFAlr\\\\\\\/4hP\\\\\\\/Zsy5JXiOWzgGRvKILgsE8ADgMY5e4+tP9BLKJHOA+LUV6br0ELs2Pac2C1FZlT3PPJnJ3lDMs\\\\\\\/WqlBRE30vvtaZgsH904nze+nZFrrRlbSjOxqvwfktKENoN5t9Z\\\\\\\/dcyTD0Po7OO0jqCV51p0coeUXKMv5nd\\\\\\\/JdJdlJvZNN\\\\\\\/TbCNq1XqwccoUNn27fVW8YxKVmwoM1WoCAgip9BUeLhLdcBhUltPBu2Cqb6m8adXa2zveCjA7t8YPtQSge+1v2Yn3qhBMep1zJyuMCoAGD0dvE+Za\\\\\\\/BK+CKVQKY\\\\\\\/cSBASYAZ0uF3CvglHn48K+gvx0w\\\\\\\/YSWerem1Xf88\\\\\\\/LMv2okduHDzaNPq8XzbdvPa1IGs65UG+4eO1LRrz9eOJOIn323tBqrsEMDjMVRZ09ayaX51IHskcWHKzNx98UTNgYGdIOfb6qdnV5dL76hsgEZylQE2867lQGtx15Blnnv1uCOVZd9MlfKRU4KVfkwtobPKDsRxRyTcIHojl1ls\\\\\\\/9ApRGjEcs+xqrNqws\\\\\\\/CbVvVFxKFkE5rqOFaDQGPAcPKMQiCNHg9mLfHVP6k+NmXPDEyCMK3DtUxqxASQqvqRJlLvBi2PlCPCAd3T+nuP4ZIyBYN3Gmun3kennMGCgRpyLWSbbGzOUDZwFabqRZhcI\\\\\\\/0cOtbG9zQIW\\\\\\\/EfYerhxpMP48\\\\\\\/x6Ovw41ivPDJIKgR4jtd6Sbk5HsCMcrq3dUdWhwtoa59xutGx2Jjlv6djJYBONucEhO6Gx\\\\\\\/Zo4DKt0duOsBczcWAKqzpe1wrqWVgUPWChgwJ5Sl1BosnsZp4xSB9vnMDjJBYv2TiC\\\\\\\/3sTcvcH0zemVV\\\\\\\/f9w3bpwquiLwXS8sVOOOPXf4jzkKkfqx8\\\\\\\/jnOliCZA8O9XzlPjsImjFTgR+A8UuhZTxl+FJpnDiWwy7OsAM2\\\\\\\/HEX31aoSf7Qx3eqQGM4qzgfpVX37FvVc5ztH4aVMJwcWoaP0kdePbCTmAdlOQzMG\\\\\\\/cERHyELq\\\\\\\/CL9WTnZaSNQTC3SYNgkz+\\\\\\\/IWuGQbju5mhaAaseqk4ghiLBcoO0++tLGF0FJUyxyPiGr3+WE80KQ8IIym1XUBsOzltOK+3NkK6BaCUUS4Nfy4Di0XEhiD3JUEh0E4SAesuhaOK6KFmukihMa\\\\\\\/uKO9qTaelLsZNi6xCR0lGBeuMqt56s0Dp8Fj3XakhJtc3zipBsHrIS0BIOpMtrVTG6kv5Wc8ZhXKxjnYLUI2mxCilFqZTSIij7CCtyHBeh1ZwgALJTWvJ9Xslp1ne7O+QpJmQKf+FPouXbLdfgcYUrspOQ5QixH2VrDp0J5Z5lVE7mu0VwL5xplpbHuA1we34YSUNbF7SBol5lWR8Eo4MX3aBkG4GPTMGm6p\\\\\\\/tF\\\\\\\/VYLL\\\\\\\/zCZg3oP6GcIMYR97sbQ9WPKKPJpvM2RNPM8PDq9nU8A6gNFBTr2+GsqsIYoS5r8u8GpyLkSGVDZn5H8xtTifZV65Cme3gEPnlaHgs5GsmxEIgA+ArpbE7jJmzSFg+rhuH8X+Wv9xiv7ks1grxz01sKto9wm7tW4CCt55xC82zFpkruhWQ\\\\\\\/lEDVho8S+sbSLwDhPsB7FSZNIbjaBHmLK8qSp1rcs797yT8Q6h6o05U6Q0kv2h6D59MyDJiGrwcYnSkJnjFV+ggtLFn9CHSywN8k1K872LaLFVc\\\\\\\/dvaf1dk8qeYlj7Ig1gT7TwQXHMnfjXsUoR9Mygw5eRLGDMQstYoOBrAQgT6SZRYoU1lOc4fSn3ynmaRu1drv5uzFRS5+ac2Bnu8g9lgsAeKXL9JpiWlXGMpwj7zmNZ1oqmLiD4Ritq6YGioTYa8HM2xqftOlXP4t+2qV0v8zgLbK9gIbILv3nn7oNd\\\\\\\/bE6jIDN2XjIlX+Hdv\\\\\\\/X1JRt4DSuVb7hFC7hb1IfNUOSFPswD2SYblWuO9rjdCuYYKACAbj497eWJJIZZWhKfuk49+ZtB4yc1UWXZm3YqAzy5jMGyGa7sjpWjMwQT9CHq1xfkGMTKTVewX2R0eyXVMojrpI62+q3qsF2GbTMlSZACAt\\\\\\\/Ww1tqkZfNx73srh7QjRm4U0ssWu\\\\\\\/eXqta+Rnm2WCYellWYxQH9UOBZwT48rxjKh+25zPnqIWWFlLtApcpYHTQ2SgnEw\\\\\\\/\\\\\\\/J7nOvSuq7dC9+zV7N\\\\\\\/a\\\\\\\/LbPhBCzEnde8DyN8xEk6rsm05RxjCI\\\\\\\/ppUPUPOEdLKM1BHdfYRUwt0tRZ3oIqFCbQhnfbjiDrbWPOFxKQuwGPixZGG2\\\\\\\/C7eyt7Whs+FtKUfrrSCz5mbCweq1ZGO7bXD\\\\\\\/j+yGbprzwQHn8ZdR7Lt9mzkraOe3o81PW0u9Z82BSMGrdVZBLyD+4I2aUyF1tKbP4tK8Wt\\\\\\\/X19zvbIMlHzMQV6wpPAjvoLG\\\\\\\/ft+x+Mvxob3RSxw9NDFxLM5h6\\\\\\\/vYTW60LYhruhr6XMXV3JaRSOf96DybRslWABXdhDc8iXYwmh5MzxPlqkzaqkWDT2p0Dz6kkDAfGi\\\\\\\/rsQTvyZW5hOsW9Jm3CS5XSfMbaCn9SY58mz755Mx8mjh1kdiLejZ8ibC1QrV8803vPdrMz+W+Fn3vdCCkVJunX8UzRtNAaq3Rw1uY26RBAfDEaAaU4v39yaIO6azG5N7hYe1mE+KpiU7gQoEnonbwQtlJMXm4jasCKGUYJjisk+vQBoO+lhwtKe7GoMhybujGnDFDGBUKhsxxz1RMooSn65ST0iF9jATay6Cuz5dawpQHtzYIoYsmerGA8\\\\\\\/QSuOc\\\\\\\/3tZZG3R6pDIB3PEAY7wm9\\\\\\\/L2RCEcmiAD2kg6gC4wg0B2IfzSWstxlM0SUaEi4LZAdlHuTPFmV1mfZHtut3CCZg4NEapy9U8fDh25qkmMvn\\\\\\\/pdOIxTmS2Z7VozpW7aZhS1ji3GjPJJDj3e\\\\\\\/VpWpA7Rln3pmONkVwSa5lsRzY+Q97dDM4YzZhti9z7He7JCdcUsECjbCagCUiejcuH5zLWM5tkz3OKUc75Be1YI1djK+45sNzAjRi4FNsequJ4nyonvkA1qnQUI4YhbUlZhcl6UrXY6tHcuUBJOvMhKOCap+iZcY6kfVs+knnjhFNrXmFASKJrguNmhm1fpSASvqrsfdWOsmvMzzxSrHeFYB+9jHm9E5Xbi+Ion6z2jP8gTFsO3FgEbWAtiVrtJqJT04OxvxJTNbjtGpoqqif2EEuZarWUju8ZvOo7P3t88ear98JlmkFhin45jFaU27t\\\\\\\/Bvy+KC9tSF9OBdMJIsliWNUhAkjMNblqW0AaAYOZITLrScha1fOa5MSuIEIlh+kdyOiBDtRO29V7CZY0ir93hS04nF0fQ9kwToSXKxJAdSMTawvlRUFWHszfN6GAnrVuTnUj2hRORhO3XNkgctRc+hVtFVfbqmQsQIPbPuCa0mDqIi77nJ5UQLE9Sj\\\\\\\/t6kwI5fOZKTPLqmHCw6xC3iGyNp1QeRpRHEIM+dX0uhGcoAVgKBUaBVCs29mW0Is1cOLjN6M7knyt1ZLp4GNJck3RGKQmrc7VHIEih8CfmjSnMU2LJRVwK+uzyNCZirVvFUQiJchk53CK2eZA1N0ngqhlLpH7on3kX9tbjzvcwBPCF3+XYFIqqRuygWeDxEhP5hFQCtWkjwjfSQOWnQbsGsVCcnQQJzMQVfGFIlg+0m3Ol3vPvlEquPrzEf2sHyqHYn6t1F6BRNe+FyHv0YcZcD59fJIsw4qBFOqS9n2E3ni0R+XMBPQKjqPjGP270jNWPIyFr4a3YuEs7zJkCasp2ntNfNmppJaYm7r1r87zuryiQ4P7yRgSbNhHxAMs56idHhJKe83v7Bu38q3X+G7e8Aykr1gpR60WYp3OKTuo5RVgjInr9T9vVb4J7hGS3Y6b91\\\\\\\/lxLHbn5FwrKUtoDmUA8NRPtSeoRkmp\\\\\\\/m3W56ElSwHPFWyCVM0JjIb9pYsQAilsU7Nmye7ZWlABGkl0dZpAJJjOV1NhMAXhJV+NXR+YWNQQnlCxsp7qUxSOPG\\\\\\\/AGS80TkkrkrBJ9XmNy2vqIWyld8zbUH0EtDO0NGWDUF+y45t49p\\\\\\\/4G870HHRwNzXKRBKeMGidbrBx4I0Oo9BdNEX34M9O38UjCXkaYNkfgHeJVcxLwJMl6OFNwbNsr5ZQJe9Jim1zlbiDJF2k\\\\\\\/LSPBFhozTR1DcFGjwsI68YsOP0pwO\\\\\\\/cdfHaSwgO6Ai0+a0O1LI8EAoG08kpgt\\\\\\\/eyf\\\\\\\/h+jP+oJ3yQuKZX7OO9Ldu7qs18Lz35Xm2jWq0X8LgOS2aTj18c7YaXUNFr13fGjT4sM1nLN8XEDq3bRcNlLAlFmSAJ6lA+yhxcGIfCWuIUqZqANvIlo8sr3gXZePGVtBvPotgf6jSQqTcv7VQI1ORKjrzZQl8+Yhu4hXGgGa86Y+VT68ep6vd4mWaCC2TKMT1ewZWgOjiauf238efS+FbDq1oewG3g+6Synkwml2N86WbyUdYHAKMLxvbol9LbEuGFh1cKg0SME7DK8JhL+kZLpXh0GdhiKsCIMphqYzO27MMiLXbPVb\\\\\\\/0jl+VbGHgP103THaSzCiaFcE58mWF2kul9NtPENv8UKqzhVhcbD3qEzBgAlzzeVA+WaRA\\\\\\\/EsIlaDt5y8\\\\\\\/ls6VoDXRgq0LZ0muiIW1OxB6+8KUsxHe\\\\\\\/hTX5qw6i7VFBreEGU7dOiHGgJj5dl65Jjnqp8EyVgBNjZPyez\\\\\\\/Sp2DBvFtndpSmP6eqoWFESDkQeBeyyWexBMiCPwEc+OMUdMvdYV6lYCR0MkxlzUenyoaBP4TUxxmhW9AryQnvHWCelh6prAdaxX7FH\\\\\\\/9+gLlGhCl5MNXCWh9Ywh57V7hD+K1bQI4uju1ftV1Lqmo8Z3W8ARTfCTHuTkQDSome\\\\\\\/xJ8fsQ3uWYG9zZGMtKfXs0mNwwLMcVUBpLP\\\\\\\/T5tLAlwsgjr+TYE5EsIU0QD8TjRz4wQSvSOmCrduLQljmK\\\\\\\/FwJxcn+lYIMUKOw2TkRkBsdIG66O+I98ArWF+kM+xdlzP1L0FfwNNqoyk2vpK4Hp\\\\\\\/FIwX3rCELec82djmMGEMtAAsDwC\\\\\\\/AllIsurOTr8XHdT59fA+Yu2sY1n03HDeMM6Un2yagVtGVf\\\\\\\/\\\\\\\/2AOBXcHx31UiI0b0uHqGzB+NMxE8BP4ppeHKB22+kp4MxsC0wW1IupJ1BftKdNrbmOel3+R7MeLEiwP7j14k07bG0yV3kgMpjdQHowAnVyIEFxZW0GgWgQZtRSBSE5149h2vWD+pxN8gIC9EO14bc8PZIPDnPRGGxQb0s1HG5iF0uZG55f3Cswh2855j7DICmjj9WrgbWKkJ2OnUWxIBVAWCJ\\\\\\\/BW+6liu2qsSMChhUm9TEhmNu4dUqYUMrx9RjN6VXeD5b+Fr5Gpx1gcvBhk\\\\\\\/\\\\\\\/IgD292z1ptzTtZf8RvCYFIlcZQGqNZOXOP9XH+NLhq6N0HDRzxbrBko31DQfCKtFgyADsRsn7aK1FfoneqeKrdPK+bpIIlQzvdHgsqS423sb9VIeP0hC3TLiLCY5hVrwaePAktXCnIDtksc6rRtrqVmPvX\\\\\\\/eg85B41TXePjHuHBE6duAA5oInQwebOOIlYhSfKPe2\\\\\\\/wUKz8MCxHNk1GCvzn6+dcLfw4HIZMhn3m+rPxmMfAuwnEJoSDTT8c8YsKWysmMDwiyQpODhu01bngO4zYDDhRXkRu+B98Qv3RxzcmvcsRQg7w9c8ayc0s0dpp9MNQQGFQAT3umeVkylmMhGkmQ41y0EbjY8zy8u59g8Gp0JNLNsWD1hYxNhVbQ6URH4D9WNJduKu+7gN6gnGPSTbWm\\\\\\\/YulQMkZIQm0UdF6Yu+6CnFDef5aygc3mwe7UKBNT9E2viYQmX8Hd0qbIdK8DZ7o8YyO5917GT0F9Rfij06c3uRJYHMjwx1v4uzRT0OnUOPSJL7iFr\\\\\\\/5h28yHinimxvjVdbZSwbsJRTpslClJ3fxdceO7T6RUtXqNCVfUW04hFCYW7yqn9n73Sll1C\\\\\\\/fSaMm074eXmcU9UAXg6JUoMLaXquBgYLvErBLuou+UffgcpBjV7WftyDjj+l9g4mXrZyjPxz0aQknc8e1QDbowQhEGMSGXW6xNAAaw\\\\\\\/7cWVxYXYR8ufpRuiH\\\\\\\/YP4Yfk9XmB2OLyMBjOpszQePOhXdA9xFyvTnnNBTbay98eIBfJ8Ck9PlalguUfxiyiyOjPscckOxlxo5RkG5\\\\\\\/ndSLqGRBE6XzrTtqgU6nhhPuHWnv6qDAmfH4CIfSDyKR5ODnR6G54B0YerWEGp+upXW0\\\\\\\/6W\\\\\\\/oPNpRQLsMZ2bkf13Vv8jhJ4dYtE0RPGIL8cHriIVsTLvSQVcxJvkXnwp4aRVFs6l6A0OneE8N0DZg2qf91o8v5Nbqqn8inSh+kiEkzegux\\\\\\\/4pK+X\\\\\\\/xqVI2nS1ezMgspAB75oGC4bmt1JJUqAu0o5vfyjVXyUKVcJTP8EypiZyrZmHnx5ZCT\\\\\\\/IeMJVFfWOiBGIsMdDCeVxlbXX+PGBLDeAfykJ8L3UxrzxWJQMZak9TCXi62aezAlZ6AjDVHXRXov4fXJquiB\\\\\\\/8c2kq7bwyh3Zsxq\\\\\\\/5JUG3E1H4oin7jLsH6q\\\\\\\/Nwg6XfmrnJwjG3ZZ6W4ekLFEh3RGoJDCSjZ8gh2Ar4+ahcIiLxkltwWO1qOg08jFClI0FbkbBIRNIhd2CPmqGAwefBNeg1dkMIT+GBtqkAUrUVjkLuza+Hp7Ydn3FmcI2R60gYoXwLM78mPQFOtyTrYg4VtxmxUwT27WOKrO6aFElbcanT15hWgtgEdB20P3FkDo2cD6IrxgJU1GTzkmfRVgVax7XHbZUEiokgq6d5NsNxl0q2f6\\\\\\\/k+KOHAlnMPPPuZbaJT5Vwh+SDVSEQ44B2l6H3tRCPSXzd8xGJRX80OqDkAM0lIDDkPJRhc9MgF5ksNiIugtBtVjenBw1iB\\\\\\\/UhB3NqNHBGB+TpKcYV5R4tAI66x7J1xUPq1Xv9uqmU2zPJMSZyikXX5+a476zqCcx\\\\\\\/AxNIKb8plRa5zoaNgAa6Ke86QbEimEvnHLNzw0zFCXxjt0IzGI067nceNvUA8kkwTq4UWJzTSjDQYaQRSnhkSTeECM4gMlgAVzgraoZNlr1HpML+Csvjj90Hih8Kdh5LR0RgE\\\\\\\/IVtq5SFkNBEwaASm2JGins7omOpR0SQ0oWsjbFpcvechVPYwo9\\\\\\\/nGktWbsfEdKOIGfVgH1jkYefUWJQx3WcFxw5C2Uh\\\\\\\/uA696mE5J+FXI94qY8tQPERhFSnE5ah1oYUjikkHEbwr7i5jjubToHYXxwKaSu5yYIq8T7nO\\\\\\\/rRh+XzzWpuJA7n2yHYVtZgU7U8nQyMjvQ8NAnmk+VvZAyp7fu9\\\\\\\/rifG1V\\\\\\\/yoy2IAOgcRSkombhl29LjNn9F1izhVrtyBOZnOyW92IaXHfyHRWZyWd8TUd+XUnetbCnvmHE+O90XT5zTWQQqlf3FzJJA+\\\\\\\/VfUHrDLXSYa\\\\\\\/KhsJERONVVJ2RSWmhV6LdPxtIdKmh9JLI+kqZ7Q5kpUkvSl2odBO63phYwSVP4LO3b3PtC00IksAen3dpkZ0SmLXrgzWzImbMrksWZG97Uuf+Z+CdJUb4T1RGdej96gIcrSK\\\\\\\/1PZ9dvjYRsRxJRf9s02pw3V1HjaQYhDhrmva9hpmgohPsRZ2aFm4fsZO3+UzM9rLtfBjo\\\\\\\/201fpgz3L1DDZR6hm8D29YYokpauWom04Lfe1iy1713e9xq+bGTJE2MvgLQ0OXj6mOfeZMAPsmm1aY2mgRKsW3pa4XPQ2Alj+svmftRpED6nHTDMBuJ7A+95\\\\\\\/1nwVupKl5SgYsWsLD\\\\\\\/mJ5\\\\\\\/+iLkl7Tr3H8lEW0sMCcln8u+XA60kaoQtNZ2\\\\\\\/UsTf3WZB6q5qO71UfukvNkqmaN5n6u+NxBA\\\\\\\/kGdOtv29XM04toEefWe4oIVYPQ9HnKh4IZCPN5zvqctMFhnPY3I4CDET5qeb6u4xZ5gPQvBCXl0oLSI9FOWC1qpNWMq6cnrYuj+AqFFUhh2cKeo6mijMsshcHP0uhImb1LSPFYJhPWdCtk8Wl4j4QidAI80K+WTBY5HxAFi69oFasQGN4a\\\\\\\/cad\\\\\\\/56QEA30tyiTXLHXj+0pBzPl7D2pcZooLHHGxaQhpsSPv4bwdl\\\\\\\/dzx2vc9e3n+ePhoZyFr2gloWjLy+uPv+nfoRJCoFgbIBlOOaqS59fHYx8MdOeHXaGyWAZrHlbgHTI18ov\\\\\\\/tYlxWek6i9IB+GE33C\\\\\\\/OrzQoM4WV+noN5XL\\\\\\\/\\\\\\\/p+fb3xXKz0OVP516deMg4a5q\\\\\\\/ZkUup6cBAYDp5YHrL4zmiXZYm04uP8BMItbTxnay\\\\\\\/Fg1+EO6E2zmq1mzqbjRn9WsBI7nvhPb5k6YEu5vCFNbN7l1UUM3DT3yJ5NfT9q+qNG\\\\\\\/XVZ06ffbltcGZhqCIAciZNg74idgQSkeBobGo068Jf5XbHBk0+gAl6zXdnAW9VxAuNWS08xH7UeVA99bX5EO77rjWl+VQcZHv++CHmpZKiHmlEyQzW36AZiI3CBhiql703sAJfd01lBDGVCQcCOp\\\\\\\/K5m4ctxnxoQXNPsGHzJDTMvESBNmCuQpX267bXTvScoZI+MZiQmRTWHtCokNRWg2fpUS3syNtc\\\\\\\/qLmsX0cyA9TS+qspHpAfZ70rtV4UEjQ8ibpH024KxfkxZ31kZ5XgmZBcfU4mRPTWDeR4Kw+o+dfmt3OUP3WTUh2oJ6oRdOXXq+UkRGgLJW4vpZ+w1pw2k5nKhlHFamp2q50VqbQRcKNJ+auV3k+0ROR8cYQ3sqHeV0ulO4EUAdV2Kdsl7dMtiOrRMv+jSpJvI7lLSEV5XhIfThVSc\\\\\\\/kBe3ySxvyta0tzI52msUqbeAoYZ5iKgdKyMJTWNvJoVznlpiJcHDgi+2ng5schfxmI2gTmeaz3uuaR6CDLB4eDFT14YyF4r7YsLz+Q9Wc5xDCmzDlokjKNsf9cIZcMDqCzvslL0g4i7N9BE2e6TEkrYeOkn6excscgL7Q1ymh0dNheX8Cmg5uH5V4AaTdjS14Kv6ahOHEIvlTFuK+qATit+S+pDDTRsYW0hVXKl1iZid2kcqssOFx5lhPpM9SQ8BH8Iu7RB11CnQbMFR61RaTxlmIpzTEwwm5IjNnndicojj4qZx548Qg20LFOkrcat6tFAQnkyfPja1v8IGgvzy3NZsLXOe+PSD97y\\\\\\\/2QciWw58fOgsTxTREErmVnD7NXS3OxHqYg+7Of9OiwiiSy3rQB8h7vg7sSgnQsovRlD5o6+cQaAWszPPKgs0Am15FltNMj8f9F0lp9N9ps3df6s3+Gh0TtfCK5cW27Q8VGQss3spyf8mJ0M+KoOrX\\\\\\\/+gMYNyHNb1OEMZr1VIcRjw49Bxvvv3nHHhnr6oYuzm81IzRjaqYc31afI4UdvqoXRmZ\\\\\\\/\\\\\\\/CXDqrOWO0MYmEvjtLX9mGOUyOew6aTirMOYJ6rzH+kgO9oAlCa\\\\\\\/GKAHcGKIOKohj1gq9ASkn7hObn9Uf5IreTTJDaDhXkQmLQdwrxf\\\\\\\/M4aKe0+hFMqU1RpshVsbQ8HYIjBndu4D0+544zSVNDYhrhqQnLQqGRmGSb0dVReKpCi5gt2qJhIfd8WFkRLDLnKpa+qaim4Agw7WNf5hABSBmfKEYLadT7gTREz2\\\\\\\/+pOm9h1wVYp5uOmg9+M3F\\\\\\\/fVtD5Gy1LVT20OV3RAo6js7dg7CJ+U9unZBJDa4k\\\\\\\/TQHm02BYJLEylo9BC2kU8yHjXKFC2Osnqb9VWkUC6XpP+jnXc2pUItBN9X3dMMf\\\\\\\/Gcw3Bc+NKUNsZsroF+xIqzPKZGe4zdB\\\\\\\/5cdbhgI1vQ5HK0w8hsT\\\\\\\/kkv9O\\\\\\\/XShlko4yfZs9wIWfc\\\\\\\/YnruSE\\\\\\\/lgjeGXzK+ZXI4N+pMemqNwO7AzUQ6+50cgukTXBz0yj3Z5Y2fmKKrrRxKsJE2dwDkm3jB8vLCvLmyRyvlUFxQEnS3BnMRxCGQ\\\\\\\/7zF6\\\\\\\/VyZPa5AXhm7VdinYg\\\\\\\/5wd+SxaErVGFSRxdidVo+mRVEY87+dHemcHRQVJplWvzZXwp50ERDiu0HzEIyDzISj+OX+D\\\\\\\/Fr0QzeYWCBNWmZJafTv7Uw6KsQ4SSDbZ5\\\\\\\/7tuP7X5nYe3n3EFMbrgjeE8mMcNulFo9H2yOnfdHvH\\\\\\\/LgjymavLbiQsHUhyAou7XDALNHK7AulfFwRIh6aAomizslJYJ5jfaZDTnx3GBqrQAS61ooh0\\\\\\\/m\\\\\\\/gGLyBeh1KSLcvhPBr+7lFloDJ5kwUrlpTtaqxJ6H5XxtwEyRPGmrVmJEO9xZSZDr769vwB07VZnjom7GwqXTIWFbRrsGLU4rbSivrpIJqt8tCLJNVjfrfFWq4ZU7VVwObj6uOBVaTM1dB1LfjryA\\\\\\\/iTLmRwFUjGPFCvpzhFt3AhsIqO1BuM9nj27gE6uvSWAz7Tb7nIGwq0U05A9OcTqH1Z0j5YDTHW8JSku9Dc9ojLpebjpXtSEcmFk08RnbkRrxWs4NxiStw263MEmrDDPBte7k8JJsoZ++IIp7GGeL0Moib8iSruMmPg0uJbmSzcZU1bDfbBOf\\\\\\\/QMClUS1IC7q3Fr4kLo8tmENJroSp8gWM+FYATkvAabl88UCgiG7KMnFvgQOWKE6EJzYV6bPd\\\\\\\/KA2wmjKXTQlDcPKAFlE+UxtGGzr8a8kXvxHAzXEpU6xi+17uNQHRNqPpNaGLC6TTUOsM1FJSOzYB\\\\\\\/niSXKz8L3ePCpw6yZ\\\\\\\/JfTaRvnbNLEiRjwLHjwnYIJuOa\\\\\\\/4Udk04gsCSM8Z\\\\\\\/q6dr888zbyS1Ij\\\\\\\/UlXFicDFD7i5LQFL5+O+YK5us6TnDnTw3uqNmbEyGMEt7FyiEyaGNlewSHpvZ\\\\\\\/rVEGew1qk6orIiwCAFMxLTmfxTizxrm6tqMeCwQcHGH0k5U8s5M61bt6xh4CKWCILX\\\\\\\/\\\\\\\/2mFWNsZ3dkkjdFIxN9iPwEuDYAfR6fZVEBRVBWTYgBAkVs3ArRGEkSIsv1GYBUy3MCTEgzoIrrejbPlwJZOM6lh95ysSebIu0sk8tQuNGqcwP2NG8uMF6GT40pejGoPNTss2Xsgx7\\\\\\\/mCHvhDyRhprewpydBTQtY3BAvZ9ZHdIBJeP1b1JMYHxN\\\\\\\/sDcIReNKWrM2ndO8MB9nbfKmOahMm8lBr1QsDEMJUFp8OCxKYUKNiEIEFGCKuC1mFgceV5ry4Vi3cmRsDvITWl\\\\\\\/oenItCjItHDn3v88yRfgddh9D6N0uR0NhzrkRhgeUVd6LVhL0clVtebCf1DyzNpIuJOpX2K9TqAViVEa2XxsXo08nV56t4lGrmYjR3c\\\\\\\/kCqyHIYDNanxKrzZlu\\\\\\\/Xwqtoe31I3L1H5BpNNceOFy\\\\\\\/w9wwUAL8vG5dPhgwabmdP7sWVfa9yDXD91YVaLj4J44m1A\\\\\\\/MgM5WNsrc0Nm9KIzrQp3QlYawBdgOx9VabweVVGtQZeGEx5kFTWHcRcd+G28ONuetPyCjG0T4odV0MHOMHhQVsWBekNw+h4tWHKhWCRY5Lu\\\\\\\/0f\\\\\\\/B33tGxwvR5j4WHz+p3FOMuJkN2dEGSxvVfn6n+menZR1BvTe8Ya\\\\\\\/qozW29Q8l3VFPpgnCckFlYP+plkkTppQko\\\\\\\/BKZkF5KujVhms0v4aq2dsw95G4ycRxSwyimuPj0ywOR3zlCuousxo4TavNhdjUIk47Xs8mQ2v555NKlMxPYqXz97BgHfhaojVIjcBLqq3j+PiTsDBGnP3j86TmtbA5UwXsW4HxLi41HPNMw1e1OfuXZ0ZLH7o35+nSZki6i6mIQFf3RcEdDmeWMAajeUGTQaHHAbItH6nnHa6BcK0TW63BaxOlCq9acxyqf1sw11AN\\\\\\\/ORgek4ts+hpJxoqExzEhgylhliyDR31E8hMBQnGEVQ\\\\\\\/LzaMzFLcCbvovavQmKPlvaBs8qrfr\\\\\\\/vkT7Duwotsib+hOBUcdrSIJ\\\\\\\/j6RxcBN\\\\\\\/Ir+sbmNtam7IYI\\\\\\\/pk2K6Lvbxs5QiWdVsmpqKbveMgRL7kOiCAqlKkTJmDs+JVfbYkBHeEQMNREkPK4X1fUhDyfjd7RpVQLa\\\\\\\/WGR0Js3Kt1jL3dNL5ShJSpWo5L8oMU8dejYSY0EdukxpiQGd69vBQL20or87owHfwF4pxhHgEhTZ\\\\\\\/dGRv3jhYeUR9O2ejQRSWfSs3VM+K+qxptOQUx0pI+uz+QNAYF6Bm5iWU3EA7TZg9QuGIl+Y4QTsljTdZhg8ERiL56um0UnyMFN\\\\\\\/2FrhH6KX91U6Xb0ojXx20gk+fesi\\\\\\\/s915beVZ75oPVCGi877GPVhhW6nnjXeXlGtsy4EM7Pg0QsNw4SjEv1wUmmiTK69sqSzpOQjdyxZjTfefJhmWb3lx0EFBqYKV2M7yAvxuPegkoKUMtcQv6tz9DvdAjz3Qia91dbtPCOyuWEXBE1Q64wP0b4TceEmuyDGZhFEBmpoDNBVKpLBouKQGAVgMEulYuPl3JEiFpzabsAdMI0j4h8PVNMFI++lcIWtwHVB5l+KwkStuTtCoaMMx\\\\\\\/YaEnuY+9vVw7Ljwt0FPbSm6KhO\\\\\\\/bLnxqTSB8UpzGeiSRfXeeLB2Wl2e2nxYTNLWmtvBh+hO+VpdFoRQ5xyYovYXyYStMHUNsKWmFNiqrxOhopxQe89ladXyG+Y0P3WEaDlhnLg215LGuJ8kKXSiVCUCND2xuH065MLHNfS6\\\\\\\/Ulj6b1c8KbLoaOaT\\\\\\\/9SAXQmOwMMe6UBWBsefMWK2g05BUx\\\\\\\/Y\\\\\\\/1ZvtR\\\\\\\/MR4SA3z9K1+hMCUUzESn9lKcgTeIihAhFNj7g+p8D3bVnRU\\\\\\\/BZkl+6ioqzjsW05FFcvVLWpowI\\\\\\\/A80dCRJHrzLWy1lC\\\\\\\/ElmgCn480qSV6kdFgvyWIc3XqndV4nGJ6Cj0WEFkkmMZdl9mKK9dxdQS6GKgUbLt5DIdzueM\\\\\\\/mD0mnsRJT2UC3USdGGLbQuO06SP\\\\\\\/lTxaJlhrQ598VLAmjphGkizWPqZhNqUZxbQGOf\\\\\\\/M5yNGXzqUUyM6Z9nBwoyymWZJ\\\\\\\/HnCLnS4nxHH79ej7JAbEdaYcZiwMAlQDOi2qC31IfnU5erC5KJePn7LjYvNjWmcg1wUSFZPGXynB5QilrhMgfkEtcUC7UybvxzHOdWSdS\\\\\\\/YyejXy9rUl1rR\\\\\\\/WF6wCcEbtlpKd0AtAWYB2ngrlNFO83fcLOqnPHUhx+jxvF9vJNuJDXTGHJjpRi6KqvjVhRCP1wlhiP2skVo6pA2TmWeNdpZyraVxqyhJ9MJIezHhHL6E9ej7cFYCuOikuQ7gAMU4fTyUAlkVEyHtMrb8hm0HLLojxWFpIQ3t8mUlt0b74nFlA0KcE3mJhYwaxUJlfhRDBQoO1ELr6VXd3hX0Ydnt1oEDPa5a85y2K1k4RboTROBK+0qeNxvE\\\\\\\/mVrKGGxWf\\\\\\\/KLrAOTWy2iKoihxKrcXd0LxTgiGhdf5mX2\\\\\\\/k52yMYriYTBr67yioMhpE9GFQevpa\\\\\\\/o0GglOZEaRj45p3QJDCmuG6D40ZwoCVHg1Be88qEQIE+zuEgRpgC4Bf28W76AxGC5d8F+Im\\\\\\\/FK+hnwimFj4O5QKX7EtrSmvJwTvsSnx0y6sXB2EAQnhlVlcxvQQGl3vURO0HInYg+MdeQOulMW+R+Z7BIBWN5CPFddJewcvOHXJhZ8JtbiqBupbGxmLQrzbcZ+e3\\\\\\\/mAaU873ApQCAjdd4C1yObsLjIw\\\\\\\/KWGBvxyLwEx5WrhCXJSC0rBL1OjTPrNSS7O2IGoVz4e\\\\\\\/sPcaa8ML+pVi7Cn+QG94Zjesb1AcFCAqJ5XAzw4KYSSlNkpxcGAoU5yRDGybDrUX9eVHHO2FqetnLLdzo5iMd7rkXe3utDkFwW3R+S2yY4tQwaEp9vZEj59XN3H7G1Tu6tPsCdzL2JLUxsvfI\\\\\\\/BP9aHzz2X0mY+Q5x9G04\\\\\\\/dWWK4+6m7Vpl2Y4bsOdkE9Hmyh1bvT+zlNzCfytVAt2GooD6fOFcyBWCUtZdUgqAwjxK5vvL4QMOmiOrhWMn9CqtPIjsMyjHOWDQQ76Aq6oQwK5RvJM++DlFQMNBu9\\\\\\\/hPp5G\\\\\\\/3rfsrLt8Anxp+MIRAkseBymw18vL25w76EVEy0nYeH98nxwq6qcoOK6V2vfqGajU5sc44thI+XaPPAc2xAsLT+ZByG\\\\\\\/5tR+1ObizDYzWRiC3HubhnlzIxZvjzkdjQNI7wpxy0+XerYTKNnB0O4IbdK5K3SAuFi8A4WMorxPsuYXWgzJ9W2f+Ll85wJM81upNawLuPluh9CqEBPYSzpw1Nk1gszt\\\\\\\/QmAuM6J+1dsFl3yA8eeeoLo+Z61Gqn0KVQU2cyreS2BXPYxu7f3J\\\\\\\/F3vjuEEzYT962Su88LrfZF0pnmt47KhHIpK965joxWT1lk3a0ClQ75n1cha0wJpiNdIxJNqEGFQoSh+OqRMmJEqjqyVS1x0MshWmpqhMF0\\\\\\\/5SxD39dw9dkKfBQJHIF5zp+7xOz4McRuh0HO4YSvBx91joV96PRgq\\\\\\\/SZNka63rrIfQxjbn6FvvvP0iZZMYlPYDDO+sVfwyMmP6aYl6ISH7S8AL69Non6UQedc4y7uzVtVnXhn6enAbJAvVBl0hKDAvqxNtKJuNe0Iu7k5uChcObGw7AH\\\\\\\/lZ6bi9lJOasKVXL89Ue5zxVZJ5FNeYoJtSrcZf8yi\\\\\\\/nUEFbf\\\\\\\/ONwWH58L4GACLZViSjPTIkmK7IsN\\\\\\\/5fp+a7M7acJ835lSJfUGku8G2iOt7XuUNzuidhN3rMUcR\\\\\\\/up6jqsneydu7fYP3QNfraiyn45EerrjJmP\\\\\\\/7Aygodr8Ho9Kt46lAejhLwQMJFTGXSmomyPndx8HAtK92FDpuD9rbO2zNZDzZNc2Eu+QW2W48fVVg8FB0tERL19fQrSIfHI8PShIx1cRaELpxuHqruZ12oZcTeecwqwpfG9OYaQ7LST12f0vY4Vqlot9giW6kgArc7\\\\\\\/ukt62SL1wwB8fvv2r3rt+feAQytiUnI8llvSWDandP3VBV1CH9SBtvL35haJmLIiw0uehnM\\\\\\\/SKaP4QZd0eptyw8AikQqrl6DQikgHdv9HOXR+5H1qhb+SPF0b\\\\\\\/7YuvK8GN2WSG0JAw\\\\\\\/DpLV8zEeFcPJlngSzoBtoLaB1DerqkT94oG0a9A79Q57UH7aRLoU+ENSCOHiFya4ELuA4aHjdi4AhOZuzay7+TxqqPmBot2MbaKJZZtFpd2wxzUnM0\\\\\\\/sqhHBMwvfFbpfvAh6aElu8f3Vmretc6R5qUH3B9y2X0hR\\\\\\\/GyxeHUh8iw0yseqPvKEH10vbqaMqrCsGtO5ae1vX0A9ZdxpXr06CkgYHJSun+knkIc4Ng1dxDk4O1yZjc4JNgbtychb3LOnSHn3aU3eH6IJ4O\\\\\\\/3utlsXgJz4BrGK1ENsncU\\\\\\\/nSfUeFpv7YHi3A55HbrrBJsaondqQBZLbMpY+s1YulyjWYfvaT\\\\\\\/qoIPA4L7clStmmF1JLQ\\\\\\\/zXP4q8WqJYg2uqDbhFQgUpyHJirBHw83MKI23J4r4\\\\\\\/DbcRAjJ1WP2lsrJIV84aysjT5Ei3sM61WIvTioVi1HzY1yS5UXuXWpUxsm4XFIHMLo1NRLDxz7rLvL6+hPosoo4UQ6x4e9\\\\\\\/UJ5x4aJicHMoRqEKvaHmK+Ay808o4L1T7rirnYiFUhaeGqzZuRwiwZgEQeNiGTwPB8nHCJbgn0trV1VaRLMlyOPX8AU53R5Dom+ueCzy6fWgsIzQ+W9LDINj4Uxj08h9nyp29fR5ALVoMAKHAm+np\\\\\\\/tQWQAT7u3UM1S+I8F+s1WaMMWeyt3coIZpUDntAnFa+OyHGDYNqz1G6f0WABvasDTEOVZr5XxIuAFJfxOOVzyITju5IN\\\\\\\/QFI0M0300mAhXhbwDL+zU\\\\\\\/XPx+MHrhTFYB+z5\\\\\\\/P8\\\\\\\/w9Z2mz38h4X\\\\\\\/2mwu+RificT9SyDzJ\\\\\\\/s4l7Xq\\\\\\\/MqnoNcU7v34yN7QKgTW12nClT\\\\\\\/iVy4sd7ZSAAVSFvETQcRRzaWVroku5ACBJTZ8ra6GhIP1rZhzK1riXv8CfeoD+d0NESMbVuVX\\\\\\\/cnmXNC3aIZRs8UslVOdqxQS+yKduZY9LRovrLzZCSgiGQUAnHhjtyacIMWQszaenKSliudL8lv45C6bo6wo92eugmqEoyREmqbapXMK8LzJTNtnTLw4tH5CNOh8XcE7b8ntnPkI48Ek0alFEhd2u3Qqm9N8yT20Z0uRDCSppFyL7dtBv\\\\\\\/BFa4rpiN5WZ8vaYHLyEk9WEKQSRTg4NHSj04EwamO5hn09AHOLJghazN2XvbKCQQKnqGbYE5s4ymHOcZqQupl7jeUFRwFb\\\\\\\/4L311+j3VBODJjg5lgQAtXP7ad3ovyBOJXf8Tmk3Tk4kJ3iWYA7RnCon\\\\\\\/UH48MHy3AILNwcWK6Otv\\\\\\\/HA77Oq6JuqHG2uvLEoqBSgOXXGBpzVSDRSYlpe5l42oncKRC3FJ5nK6WodagN030Yiy7U0kVYliKw3tBzTMOhZn8h0Fl2m1qnX51p0X\\\\\\\/xyAspPBizDbH9SaLga0tPUTm+EcGnCY1MSPKZcPPmAtH94er8gd2X0gdw49O\\\\\\\/058VGm3aOcWjdEWY+zNS4Pz5g9rhA12gLTv\\\\\\\/yIxF\\\\\\\/rKJh5\\\\\\\/sg2NYrTYPBlAsd6jceJfY0ozvNnLfE3r7ct33\\\\\\\/oxIdt3kQ18kNLj7\\\\\\\/f3QMsYvRh3RQFl4D+BcndxnfGgnyHJ0Lcedmy5nmblLJK1kCQ6ZLauC77tAANzZAntNVN1TfQa3zwHUPcqeBqOvZwwSoKxqfWYscnnI9lhsz\\\\\\\/LUfReKKVq28nWZSbJ5lYx3enmMSxfh5RU8NQ7HzAZImff15i3nUBjf+dQ4Cly1cIuN\\\\\\\/yiQjlmtiUiiXyXmmSXWoXmd+yIgaUWM8icMXmO0rzUyXODur+qD\\\\\\\/pMgaDpHmd\\\\\\\/Dx8ijv52i9AWcnkyS4hOQnugHAmkTwfSSxC6IHELIVKiaHRxYu+gOlmB1XKOuDdgaZjMFOdhmABSIwQN+o1C+Ukjn9TVF5CM8vObO26JXIGLGKdSesx\\\\\\\/GfrKdfbScIotVuprj2\\\\\\\/OpjLMnuWGIVyDYV8VMOxbpzWR3E+0SsXRsmSolYKJXXvWkBrpqUess2AdiswIHTKgU20Omzn+yX6\\\\\\\/4HTsQzs4EIhKezXogbJrKxFJOr5sgu1ao1jE9rwsjsDycjUc373oW6wmPYadK1UONLyEtvNA8NxDn5GpF0m8an9myLurhjrdg1FDk8zOY39h1rzELCOMTGb\\\\\\\/1mA1xrfdFfM046TsQQ96rkVs+dmn5apu2jXGZmMCz8pAobIuRWPpB1rQlNqIipFsDSYwkvoMDFRcC8382RN2lrIcXNpsWLpD3lEZr2U\\\\\\\/E9K5I\\\\\\\/8FEmuuZ3ohCn51Lj9FOiWV1PZ+\\\\\\\/sEZ9OG8HE4kTugJvPS6+u9II2FB0ONmfIDfPtiDJbgrVAi\\\\\\\/K2G38l19eSWD\\\\\\\/M3zL9hT9mXLXPJnn23PyQcNtbw0oKVqqZzlpd20Q65mfbIggqQHM9SxTrIG2wMVAALn8rCXclP12mhAmeRuD5RwtbJgW9bCYvKFlj2Y7H0q6cB9x+24i9HcZu4ebb3f3jqTPL6KqX0Nq91qVw3HHFcdGLykINo1O82a08Oz0lO3MJ4idFp2IK8fr1EQlU1eQdA9zp9x8wDU56l2fa1UUUNDiDpq6jTODaUZusf9MlB9q3OVgFcuIMMznogDX+A6MMGquQ\\\\\\\/ceeQkDEFwdvQN7Tnfy9d9HADFZeDu+dD\\\\\\\/BLQHH52TL40hoRDPKnyHrFa4gb9pxBdzzO6SPS2QzfDvQap5jbQ95n8Ko4LcaTntFyGKTZ\\\\\\\/Mhdum54jS5Yof9xB18gelem6oaRvfDPXdA2LFQICP2UNjsj4VGn3pWAUlnmAdMlK1RNUGBPKmIwvN\\\\\\\/Q1EAoC8g5IezbPaEgovEv6H+d2Rh3qVa1mVKSzZ+WlPAbFW4jSLSZ7xgn32oWUiLgoWMOGuWWERifQyFhJ3oAKKpHgQZ4oMkrrsGM3EnVATPdlX4eDIjc+zgRY1PCMqGEZmIGM5cbN8rECyqEQ35XOQeb+8Ds7tOmmoPKL7RUohs0WG3vroyxpXBCBhFvgKIGNRWJziyTDDAmfz4vNr3rIvdwbW2GSWNQ8iupwCahQ6Eiw4QRnDadaCjjpVY\\\\\\\/m\\\\\\\/IeoCADq4Ws8FwlTeBHOveXVRT5B6WbVbeER5j9PhSmOsVg0Ybra2oJLxgw+G31fmmzi6cxyNjIQCzslY2jzf2Q9sFM6gh+bMlFVCX6ze7WFwC5cf191KLwi5AXm+8WcMkmBtHe\\\\\\\/bhHZbR1vjvlTtGzTLgCFmhUolGEab1WTXNU9iPQrNFVDXsQnFB8uyqbxRcIYo18vBXTJD0mM85\\\\\\\/C+xBZl3I0xVvX8tHtLsQWo5UXhcraeju\\\\\\\/VJ+yvtOsGJDAulpe2YW9X4Rs1aQXB3A3eQvooWZ23\\\\\\\/DiQqdRYfdf7ZSn8nwCm34Af6cbwzzDWzg8vDcfHID9SC2LnlpTGmXGHCJb5xB10NHXGIyN07xi5IT3QOH4n21I4StuShTPkd0MGpomVHvbJKCtZ\\\\\\\/fTlTrHsMAows+YA8SdHR0XPtCnED1GC0KQCE2sIYQNb7QYn8EBcu7OccaSVvvU7MPyXFXwTGp\\\\\\\/VK6b+Yh\\\\\\\/PfrMN\\\\\\\/w9gU\\\\\\\/9XHvITmS4fsvSoh00Yz8HT6pOkVLgKp5VQjV1I+oo\\\\\\\/7eTIGN6b5bbE\\\\\\\/3tUl3hIo\\\\\\\/8v4sZIe2J908EZAQ3I9NbCT9rbzuBSXPI6zsPLcY6orW+qz3KacjA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022Gf3ogik9ysaU+ncCEVjbxg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022163681e23d98f5bf750c8db5c227d79fd09e4528d2d752ebfb151a40c92aa024\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022KS3DddPJi9xI\\\\\\\/AECe7VpsDX9hLnhgrUL7WgQYJI9sLt1Do54eln4KUOs7Foj3+BYSTO+sjg96Pkt4s+TD4hcjvqpDXCwA1VY4m27Z8WS2s+33BkL5+g7gaBDQDgTPutzTK\\\\\\\/sQl3baP5EYRLL6Hbo12UbzIbnyUozSDh5wTutUdbFpdc2oN9a3athia1jflW5FYp6u+wS\\\\\\\/ug\\\\\\\/6SsPFIisKQZZmVMBGympLf1danBarqAVBJVKChQ6YgETQCNlJk8zHmjJ9AyX\\\\\\\/vtbNzSaH\\\\\\\/2QTYHovbwo1h63DYMNkwK2GqRbgZSI1ErOhxUmZ44TGzKghc8rNEkZZdRP2ohxMcokqqYUz0zpMH3O0g6oOw5gZ3+G7TolJE+e+\\\\\\\/7wqPNk0MsNm7zaiNOLln65G23FO2Xe4wB1A2U5W97nqnORaz7dcMbi+Fq1sz6Y3CmRhGaoqj94ElwRdNSgXw1TBsFAupbHkr\\\\\\\/29EOBKZDtIlu1AMk8AMBI5DCQIgK4aecL4gGmnPf0AXKZI88oX+78WPhaqtvvTpg3\\\\\\\/D3DpqegdIGTN94WdZN+0PTBsQCN6uTpbeNhUUGYV89+3NMa6cJbRzFxDyoOXV7rDUcPtaX12tyANZvDIgEzgvMcow1pZJwVTrBpCjj7+Ayzg0PAFMU1CxtFt2E0vJdVd2S3Ph9mnscBsEZhcpCJbV1ngdw979zo70qjcwUB+riohEdAkZ2Mf8dM\\\\\\\/wPZlGCW02r+cZkU9\\\\\\\/VBxIZK0Bf\\\\\\\/Ku+pWhGBvhqtGWTzTmoICxL9BpYuudW49MdGpi0ZO9HJQzPLYBZPc2qyIy7vzQsxTHwkLOQqe20m5r6kNAQeerV2mD6Ecm7B36rXgKPd79t93JUke07iRDFh3U5eRKu9l0sSsqFgNUnhdF8luHziAf85T8qBFV+ma\\\\\\\/ZthcUCxU++sY91JBU+8aM9xKGZ2cRT+6C2TQ7OsLDq0TOBOhebRTFCu98isZoEjfgPGwBtsOWLaMdgYAITe3Ff+tiacOxf\\\\\\\/K5IXJA08chHZcoexVV6j9Wt2CG56jf8XzqtYKl+UMClJ+IQxcrMGRd5Hx6m+nbRjzknhx0HhxlVIbIyqt8RaivwoLRH5Cy64tRp7QL1C0yDp9llGzNefs\\\\\\\/7Tsj4TsNP3QEWGS1FM8iU8CuZqqy8Sd1xRzkRShbqAf5yzThJp7tBGDB7o8wZF\\\\\\\/mRaENJGqu3Oi1eznmJECSOwJSeWVztaHbvhWyFWxSbXi\\\\\\\/G7TRrJzV9i199wrTjIVkVquTMxTzHr8ucPMK5381l28C61JovWqXKioomCXbkj1y7Uf\\\\\\\/4QVENZDdb2K3\\\\\\\/yqBroF5d9LmSi\\\\\\\/r0rKPmKCHMES2nsCb1flbG4Ce6WK2DZ7eZx7dBK475MovPSmEGRGOMEXmcL9SloLmoTw7O3chbDxlfX+YGC7YWALL41ZEZzsdGBTGI3fzd8bi+c6I4hhNiqiYHrkiq2WuGwBFz\\\\\\\/VtarzPmuKX01B1MMd7gaVQgLvPn7+w3pSeL8L40QeraofdcqZksXLx0QHO4Unm3B5cDgquYpFlc5Ale2+rzRDLQAL3Gq9J7MVHzqUtOfx17zAe7MYsMsoLsoYkLnod3jXAJ++\\\\\\\/4d5dzXX9j4qjCgxnl9PcrYR1am4Tnpoz8VVDEftmLtGvn8BtAnqcNgk741NzswKJwF06tk75+Zi95581VZ\\\\\\\/OED8+2D7SpN1eMWfZn4NlUUNHYSnOCF02UNpfvQ8xd7NzekfV921hXyOtnxVRQUbCao+CcbQd2ms6tko\\\\\\\/OabpuyjBzAN1OPqyVSVvJlfZK9lJ9CS4SCfPsJQHIzR27x7toE4vzedlRtp5BthdDfwKfyLUWmRZ9x9RV33MhOHGhqA+sew\\\\\\\/ctl7RPaV\\\\\\\/GA2012NnVbmyG7MecWDCtKNmIeja3fxI7qUhngTduDSxSfVZoFPCkiF86xGRhuCWJnttjVsIrH6S4SlJCHaKEw5HcFlQCA3\\\\\\\/6kp8Ozj0cwC2gF1IT6wMgAPvh3Dp6VStuyzh5G8SGIYJXg==\\\u0022,\\\u0022iv\\\u0022:\\\u0022jWveVwuYCL76Y+akHUo6IQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002296063c6db162fb092d22f3d60946e4b4d67f918a3e55f4b6c6cdf657a45a2433\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459341 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/2/10/1279" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-JyQb00Q-QHwgpiEX0V3oKa1TZB2SZfacbH4KSsgg_9w" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/2/10/1279" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-KEhM3mPuaOSafvU6wPtO0f5jixYqTzxllJOItyWoQKk" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/2/10/1269" title="SlWUS1; An X-linked Gene Having No Homologous Y-Linked Copy in Silene latifolia" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/2/11/1291" title="Mapping the Genetic Basis of Symbiotic Variation in Legume-Rhizobium Interactions in Medicago truncatula" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459341" id="node459341" data-pisa="ggg;2/10/1279" data-pisa-master="ggg;2/10/1279" data-apath="/ggg/2/10/1279.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Functional Analysis With a Barcoder Yeast Gene Overexpression System</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Alison C. Douglas</span>, <span class="highwire-citation-author" data-delta="1">Andrew M. Smith</span>, <span class="highwire-citation-author" data-delta="2">Sara Sharifpoor</span>, <span class="highwire-citation-author" data-delta="3">Zhun Yan</span>, <span class="highwire-citation-author" data-delta="4">Tanja Durbic</span>, <span class="highwire-citation-author" data-delta="5">Lawrence E. Heisler</span>, <span class="highwire-citation-author" data-delta="6">Anna Y. Lee</span>, <span class="highwire-citation-author" data-delta="7">Owen Ryan</span>, <span class="highwire-citation-author" data-delta="8">Hendrikje Göttert</span>, <span class="highwire-citation-author" data-delta="9">Anu Surendra</span>, <span class="highwire-citation-author" data-delta="10">Dewald van Dyk</span>, <span class="highwire-citation-author" data-delta="11">Guri Giaever</span>, <span class="highwire-citation-author" data-delta="12">Charles Boone</span>, <span class="highwire-citation-author" data-delta="13">Corey Nislow</span> and <span class="highwire-citation-author" data-delta="14">Brenda J. Andrews</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2012 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 2 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1279-1289;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.003400 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459341" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Alison C. Douglas </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular Genetics, University of Toronto, Ontario M5S 1A8, Canada</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Banting and Best Department of Medical Research, University of Toronto, Ontario M5G 1L6, Canada</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Donnelly Centre, University of Toronto, Ontario M5S 3E1, Canada, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alison%2BC.%2BDouglas%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Douglas%20AC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlison%2BC.%2BDouglas%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Andrew M. Smith </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular Genetics, University of Toronto, Ontario M5S 1A8, Canada</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Banting and Best Department of Medical Research, University of Toronto, Ontario M5G 1L6, Canada</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Donnelly Centre, University of Toronto, Ontario M5S 3E1, Canada, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andrew%2BM.%2BSmith%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Smith%20AM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAndrew%2BM.%2BSmith%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Sara Sharifpoor </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Donnelly Centre, University of Toronto, Ontario M5S 3E1, Canada, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sara%2BSharifpoor%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sharifpoor%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASara%2BSharifpoor%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Zhun Yan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Donnelly Centre, University of Toronto, Ontario M5S 3E1, Canada, and</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Pharmaceutical Sciences, University of Toronto, Ontario M5S 3M2, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zhun%2BYan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yan%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AZhun%2BYan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Tanja Durbic </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Donnelly Centre, University of Toronto, Ontario M5S 3E1, Canada, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tanja%2BDurbic%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Durbic%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATanja%2BDurbic%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Lawrence E. Heisler </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Donnelly Centre, University of Toronto, Ontario M5S 3E1, Canada, and</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Pharmaceutical Sciences, University of Toronto, Ontario M5S 3M2, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lawrence%2BE.%2BHeisler%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Heisler%20LE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALawrence%2BE.%2BHeisler%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Anna Y. Lee </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Donnelly Centre, University of Toronto, Ontario M5S 3E1, Canada, and</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Pharmaceutical Sciences, University of Toronto, Ontario M5S 3M2, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anna%2BY.%2BLee%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lee%20AY&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAnna%2BY.%2BLee%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Owen Ryan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular Genetics, University of Toronto, Ontario M5S 1A8, Canada</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Banting and Best Department of Medical Research, University of Toronto, Ontario M5G 1L6, Canada</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Donnelly Centre, University of Toronto, Ontario M5S 3E1, Canada, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Owen%2BRyan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ryan%20O&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AOwen%2BRyan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Hendrikje Göttert </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular Genetics, University of Toronto, Ontario M5S 1A8, Canada</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Banting and Best Department of Medical Research, University of Toronto, Ontario M5G 1L6, Canada</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Donnelly Centre, University of Toronto, Ontario M5S 3E1, Canada, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hendrikje%2BG%C3%B6ttert%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=G%C3%B6ttert%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHendrikje%2BG%25C3%25B6ttert%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Anu Surendra </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Donnelly Centre, University of Toronto, Ontario M5S 3E1, Canada, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anu%2BSurendra%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Surendra%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAnu%2BSurendra%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Dewald van Dyk </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Banting and Best Department of Medical Research, University of Toronto, Ontario M5G 1L6, Canada</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Donnelly Centre, University of Toronto, Ontario M5S 3E1, Canada, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dewald%2Bvan%2BDyk%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=van%20Dyk%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADewald%2Bvan%2BDyk%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Guri Giaever </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular Genetics, University of Toronto, Ontario M5S 1A8, Canada</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Donnelly Centre, University of Toronto, Ontario M5S 3E1, Canada, and</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Pharmaceutical Sciences, University of Toronto, Ontario M5S 3M2, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Guri%2BGiaever%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Giaever%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGuri%2BGiaever%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-12"><div class="author-tooltip-name">Charles Boone </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular Genetics, University of Toronto, Ontario M5S 1A8, Canada</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Banting and Best Department of Medical Research, University of Toronto, Ontario M5G 1L6, Canada</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Donnelly Centre, University of Toronto, Ontario M5S 3E1, Canada, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Charles%2BBoone%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Boone%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ACharles%2BBoone%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:brenda.andrews@utoronto.ca" class="" data-icon-position="" data-hide-link-title="0">brenda.andrews@utoronto.ca</a>
<a href="mailto:corey.nislow@utoronto.ca" class="" data-icon-position="" data-hide-link-title="0">corey.nislow@utoronto.ca</a>
<a href="mailto:charlie.boone@utoronto.ca" class="" data-icon-position="" data-hide-link-title="0">charlie.boone@utoronto.ca</a></span></li></ul></div><div class="author-tooltip-13"><div class="author-tooltip-name">Corey Nislow </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular Genetics, University of Toronto, Ontario M5S 1A8, Canada</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Banting and Best Department of Medical Research, University of Toronto, Ontario M5G 1L6, Canada</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Donnelly Centre, University of Toronto, Ontario M5S 3E1, Canada, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Corey%2BNislow%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nislow%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ACorey%2BNislow%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:brenda.andrews@utoronto.ca" class="" data-icon-position="" data-hide-link-title="0">brenda.andrews@utoronto.ca</a>
<a href="mailto:corey.nislow@utoronto.ca" class="" data-icon-position="" data-hide-link-title="0">corey.nislow@utoronto.ca</a>
<a href="mailto:charlie.boone@utoronto.ca" class="" data-icon-position="" data-hide-link-title="0">charlie.boone@utoronto.ca</a></span></li></ul></div><div class="author-tooltip-14"><div class="author-tooltip-name">Brenda J. Andrews </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular Genetics, University of Toronto, Ontario M5S 1A8, Canada</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Banting and Best Department of Medical Research, University of Toronto, Ontario M5G 1L6, Canada</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Donnelly Centre, University of Toronto, Ontario M5S 3E1, Canada, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Brenda%2BJ.%2BAndrews%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Andrews%20BJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABrenda%2BJ.%2BAndrews%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:brenda.andrews@utoronto.ca" class="" data-icon-position="" data-hide-link-title="0">brenda.andrews@utoronto.ca</a>
<a href="mailto:corey.nislow@utoronto.ca" class="" data-icon-position="" data-hide-link-title="0">corey.nislow@utoronto.ca</a>
<a href="mailto:charlie.boone@utoronto.ca" class="" data-icon-position="" data-hide-link-title="0">charlie.boone@utoronto.ca</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/2/10/1279" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459341" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459341/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/2/10/1279.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459341" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459341/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/2/10/1279.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459341" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459341/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/2/10/1279.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459341" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459341/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/2/10/1279.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459341" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459341/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-5">Systematic analysis of gene overexpression phenotypes provides an insight into gene function, enzyme targets, and biological pathways. Here, we describe a novel functional genomics platform that enables a highly parallel and systematic assessment of overexpression phenotypes in pooled cultures. First, we constructed a genome-level collection of ~5100 yeast barcoder strains, each of which carries a unique barcode, enabling pooled fitness assays with a barcode microarray or sequencing readout. Second, we constructed a yeast open reading frame (ORF) galactose-induced overexpression array by generating a genome-wide set of yeast transformants, each of which carries an individual plasmid-born and sequence-verified ORF derived from the <em>Saccharomyces cerevisiae</em> full-length EXpression-ready (FLEX) collection. We combined these collections genetically using synthetic genetic array methodology, generating ~5100 strains, each of which is barcoded and overexpresses a specific ORF, a set we termed “barFLEX.” Additional synthetic genetic array allows the barFLEX collection to be moved into different genetic backgrounds. As a proof-of-principle, we describe the properties of the barFLEX overexpression collection and its application in synthetic dosage lethality studies under different environmental conditions.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/barflex-array" class="hw-term hw-article-keyword hw-article-keyword-barflex-array" rel="nofollow">barFLEX array</a></li><li class="kwd"><a href="/keyword/gene-overexpression" class="hw-term hw-article-keyword hw-article-keyword-gene-overexpression" rel="nofollow">gene overexpression</a></li><li class="kwd"><a href="/keyword/barcoders" class="hw-term hw-article-keyword hw-article-keyword-barcoders" rel="nofollow">barcoders</a></li><li class="kwd"><a href="/keyword/synthetic-dosage-lethality" class="hw-term hw-article-keyword hw-article-keyword-synthetic-dosage-lethality" rel="nofollow">synthetic dosage lethality</a></li></ul><p id="p-6">Deletion or overexpression of most yeast genes has little effect on cell fitness (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Giaever <em>et al.</em> 2002</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Sopko <em>et al.</em> 2006</a>); however, a phenotype associated with the perturbation of a particular query gene often can be revealed in specific genetic backgrounds, such as those defective for functionally related genes (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Kroll <em>et al.</em> 1996</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Measday <em>et al.</em> 2005</a>; <a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Sopko <em>et al.</em> 2006</a>). To systematically address genetic buffering, we developed methods for global mapping of genetic interaction networks. In particular, the synthetic genetic array (SGA) method automates the analysis of yeast genetic interactions, enabling the systematic exploration of gene function through genetic network analysis (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Dixon <em>et al.</em> 2008</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Tong <em>et al.</em> 2001</a>, <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">2004</a>). SGA has been used extensively to map digenic interactions among deletion alleles of the ~5000 nonessential yeast genes (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Costanzo <em>et al.</em> 2010</a>; <a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Tong <em>et al.</em> 2004</a>). Double mutants with a more severe fitness defect than expected (based on a model for the combined fitness of the individual single mutants) represent a negative genetic interaction, with synthetic lethality as the most extreme case. SGA methodology also has been adapted to map synthetic dosage lethal (SDL) interactions quantitatively (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Kaluarachchi <em>et al.</em> 2012</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Sharifpoor <em>et al.</em> 2012</a>), which occur when gene overexpression is of little consequence in a wild-type (WT) cell but causes a severe phenotype (<em>e.g.</em>, lethality) in a specific mutant background [(<a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Kroll <em>et al.</em> 1996</a>) <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003400/-/DC1/003400SI.pdf">supporting information</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003400/-/DC1/FigureS1.pdf">Figure S1</a>].</p><p id="p-7">Genome-wide SDL screening of yeast kinase mutants has identified new targets and regulators of kinases (<a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Sharifpoor <em>et al.</em> 2012</a>; <a id="xref-ref-34-3" class="xref-bibr" href="#ref-34">Sopko <em>et al.</em> 2006</a>). The kinome SDL screens revealed that most kinase deletion mutants are resistant to gene overexpression but that biologically meaningful genetic interactions could be discovered when screens were performed under conditions in which the kinase is active. For example, SDL screening of the high osmolarity responsive kinase, <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004103">Hog1</a>, revealed no interactions in standard growth conditions but identified 74 SDL interactions when assessed in the presence of 0.2 M sodium chloride (<a id="xref-ref-31-3" class="xref-bibr" href="#ref-31">Sharifpoor <em>et al.</em> 2012</a>). This finding highlights the need for developing efficient methods for parallel analysis of gene overexpression phenotypes in diverse conditions.</p><p id="p-8">In budding yeast, a number of functional genomic resources are available that enable systematic analysis of SDL phenotypes through conditional induction of gene overexpression. These collections include: (1) a set of yeast strains each carrying a unique galactose-inducible N-terminal glutatione <em>S</em>-transferase (GST)-tagged open reading frame [ORF (<a id="xref-ref-34-4" class="xref-bibr" href="#ref-34">Sopko <em>et al.</em> 2006</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Zhu <em>et al.</em> 2000</a>)]; (2) the movable ORF (mORF) strain collection, each carrying a unique galactose-inducible C-terminal HA- and protein A-tagged ORF (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Gelperin <em>et al.</em> 2005</a>); and (3) a set of yeast strains each carrying a unique galactose-inducible Flag epitope-tagged ORF [(<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Breitkreutz <em>et al.</em> 2010</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Ho <em>et al.</em> 2002</a>) <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>]. Although each of these collections represent powerful resources for SDL analysis, they all contain tags that may compromise gene function, none of them carry fully sequenced-verified ORFs, and none of the strains are barcoded for highly parallel analysis in pooled cultures.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/461851/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/461851/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Summary of overexpression resources available for <em>Saccharomyces cerevisiae</em></span><div class="sb-div caption-clear"></div></div></div><p id="p-10">To circumvent the aforementioned limitations with existing yeast arrays, we combined the sequence-verified Full-Length EXpression-ready (FLEX) plasmid library of galactose-inducible untagged ORFs (<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Hu <em>et al.</em> 2007</a>) with a set of Barcoder yeast strains, which are compatible with parallel competitive growth analysis. The strains in our resultant barcoded FLEX (barFLEX) array carry a unique oligonucleotide identifier integrated at a neutral locus (<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002386">ho</a></em>) and a specific <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000224">GAL1</a>-ORF</em> plasmid. In a proof-of-principle analysis, we describe a robust protocol for using the barFLEX collection to explore fitness defects caused by gene overexpression in pooled cultures under different genetic or environmental perturbations.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Yeast strains used in this study</h3><p id="p-11">For a summary of the yeast strains used, see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003400/-/DC1/TableS1.xlsx">Table S1</a>.</p></div><div id="sec-3" class="subsection"><h3>Assembly of the yeast FLEX array</h3><p id="p-12">The plasmids from the FLEX collection (<a id="xref-ref-19-3" class="xref-bibr" href="#ref-19">Hu <em>et al.</em> 2007</a>) were mini-prepped in 96-well format from bacterial stocks using the Nucleospin Multi-96 Plus Plasmid Kit (Macherey-Nagel; cat. no. 740625.24). For quality control, we used capillary sequencing to assess the identity of the plasmid stocks. We found 3 of 76 plasmids were incorrect, giving an inherent error rate of 4% in the stocks we prepared from the FLEX collection strains (<a href="http://www.hip.harvard.edu/">http://www.hip.harvard.edu/</a>). We also tested 55 random strains from the SGA-FLEX array by polymerase chain reaction (PCR) amplification and restriction digestion of the ORFs to ensure that each ORF was represented on the correct location on the array. Almost all amplified ORFs (96%) appeared correct by one or both assays (data available on request).</p><p id="p-13">Samples (2 μL) of each plasmid were transformed into Y6897 using standard lithium acetate and polyethylene glycol protocol (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Gietz and Schiestl 2007</a>). The transformants were selected on synthetic minimal media supplemented with glucose but lacking uracil (SD-URA) and then patched onto rectangular agar plates (OmniTray; Nunc International). The strain background for this collection was chosen such that the collection can be mated to the Barcoders and carried through the SGA protocol. The resulting diploids were sporulated and selected to yield <em>MAT</em>α haploid strains. The media used throughout SGA were prepared as described previously (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Tong and Boone 2006</a>). The transformants were patched in 96-well format with empty spaces left at the edges to allow for adding negative control “border” strains (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Baryshnikova <em>et al.</em> 2010</a>).</p></div><div id="sec-4" class="subsection"><h3>Expansion of the Barcoder collection</h3><p id="p-14">To expand the original set of 1141 Barcoders (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Yan <em>et al.</em> 2008</a>), 3974 new Barcoder strains were constructed to accommodate all 5336 ORFs in the yeast SGA-FLEX array. PCR products carrying 2 unique barcodes flanking the <em>kanMX</em> resistance cassette were transformed into the BY4741 strain background as described previously (<a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Yan <em>et al.</em> 2008</a>). The PCR products were derived from 2 sources. A total of 3661 PCR products had been amplified from the nonessential yeast deletion collection using a common priming sequence (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Dowell <em>et al.</em> 2010</a>). To supplement these cassettes, 313 new pairs of primers were used to provide 313 additional amplicons for transformation and Barcoder generation. All 3974 PCR products were reamplified using the forward and reverse primers described previously (<a id="xref-ref-39-3" class="xref-bibr" href="#ref-39">Yan <em>et al.</em> 2008</a>) to direct all of the products to the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002386">ho</a></em> locus in BY4741. Specifically, we used the following UPTAG and DOWNTAG primer pairs to amplify the kanamycin resistance cassette with 20-mer barcodes, whereby U2 and D1 are homologous to the <em>kanMX</em> cassette, and U1 and D2 are homologous to the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002386">ho</a></em> locus (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003400/-/DC1/FigureS2.pdf">Figure S2</a>):</p><ul class="list-simple " id="list-1"><li id="list-item-1"><p id="p-15">Uptag: U1-GATGTCCACGAGGTCTCT; U2-CGTACGCTGCAGGTCGAC;</p></li><li id="list-item-2"><p id="p-16">Downtag: D1-CGGTGTCGGTCTCGTAG; D2-ATCGATGAATTCGAGCTCG</p></li></ul><p id="p-17">PCRs were performed using Platinum PCR Supermix High Fidelity (Invitrogen). The transformants were selected by replica-plating onto standard yeast peptone dextrose (YPD) + G418 (200 mg/L Geneticin; Invitrogen) and patched onto agar plates in a 96-colony format. The Barcoder array was designed to pair perfectly with the FLEX array except for a small number of negative controls that were eliminated through SGA.</p></div><div id="sec-5" class="subsection"><h3>Validating the barcode sequences in the Barcoder collection</h3><p id="p-18">Genomic DNA from the yeast Barcoder strains was isolated from a pool of the Barcoder collection. UPtag and DNtag PCR products were amplified separately; purified, mixed in equal molar ratios; and sequenced on an Illumina GAIIx, as described previously (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Smith <em>et al.</em> 2009</a>, <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">2010</a>). In brief, barcode sequences were compared against the dataset generated previously (<a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Smith <em>et al.</em> 2009</a>) to determine whether additional annotation of barcode sequence was required. During sequencing, instead of using the standard Illumina sequencing primer, we used the yeast common primers that were adjacent to the Illumina sequencing primer. We used one primer for the UPtags (U1) and one primer for the DNtags (D2), each at a 10 μM concentration (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003400/-/DC1/FigureS3.pdf">Figure S3</a>).</p></div><div id="sec-6" class="subsection"><h3>Assembly of the barcoded FLEX collection (barFLEX)</h3><p id="p-19">The FLEX collection was crossed to the new Barcoder collection by robotic replica pinning on a Virtek Colony Arrayer (Bio-Rad Laboratories) in 384 format. The array was mated on YPD omni-trays by pinning once from each collection to the same plate. SGA was performed as described previously (<a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Tong and Boone 2006</a>) with the following selection media: two diploid selections (SD-URA+G418), selection on sporulation media, three haploid selections (SD-LEU-URA-ARG-LYS+Canavanine+Thialysine+G418). Haploid strains with the mating type <em>MAT</em>α were selected by excluding leucine from the medium (only <em>MAT</em>α strains will grow due to activation of the α-specific promoter fusion, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001661">STE3</a></em>pr-<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000523">LEU2</a></em>). At this step, <em>MAT</em><strong>a</strong> strains also could be selected by changing the media to SD-HIS-URA-ARG-LYS+Canavanine+Thialysine+G418 (only <em>MAT</em><strong>a</strong> strains grow due to activation of the <strong>a</strong>-specific promoter fusion, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001868">STE2</a></em>pr-<em>S.p.<a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001378">his5</a></em>). The barFLEX collection was maintained in the <em>MAT</em>α state to allow for subsequent rounds of SGA with <em>MAT</em><strong>a</strong> query deletion strains.</p></div><div id="sec-7" class="subsection"><h3>Pooled growth experiments</h3><p id="p-20">The pooled yeast cultures were created according to previously described methods (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Pierce <em>et al.</em> 2007</a>). Pools were allowed to recover from −80° freezer stocks in SD-URA-LEU liquid medium for 5 hr before the liquid growth experiment was performed. Cultures were then diluted to OD 0.06 in either SD-URA-LEU or SG-URA-LEU liquid and grown for 20 generations. The cultures were grown for five generations and then diluted into new media a total of three times. Cell aliquots were saved every five generations on a cold plate and then pelleted and frozen for genomic DNA extraction. After PCR amplification of the barcodes using a set of common primers, the PCR product was hybridized to the TAG4 microarray (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Pierce <em>et al.</em> 2006</a>). The signal intensity of each barcode was quantified to determine the fitness of each strain. Only one of the two barcodes for each strain was used for analysis based on which tag gave the best signal intensity and number of reads. The signal from the UPTAGs and DNTAGs was normalized separately using quantile normalization. The signal from the growth in glucose was compared with the growth in galactose, and the resulting Log<sub>2</sub> ratios (treatment/control) were used for analysis. Therefore, strains with decreased abundance in the pool show negative ratios. Pooled experiments in the presence of methyl methanesulfonate (MMS; Sigma-Aldrich) were performed by including either 0.001% or 0.0001% MMS in the growth medium.</p></div><div id="sec-8" class="subsection"><h3>Kinase deletion screens</h3><p id="p-21">Two deletion alleles (<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000747">ura3</a>Δ</em> and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002259">dun1</a>Δ</em>) were created in the BY4741 background to cross to the barFLEX array. The deletion alleles were marked with <em>natMX</em> providing resistance to nourseothricin (NAT). The <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000747">ura3</a>Δ</em>::<em>natMX</em> and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002259">dun1</a>Δ</em>::<em>natMX</em> deletion strains were crossed to the barFLEX array using robotic pinning and selection through SGA as follows: 2X diploid selection SD-URA+G418 +NAT, 1X sporulation media, 2X haploid selection SD-LEU-URA-ARG-LYS+Canavanine+Thialysine+G418+NAT. Colonies from the final haploid selection plates were pooled and frozen at −80° before testing.</p></div><div id="sec-9" class="subsection"><h3>Yeast serial spot dilutions</h3><p id="p-22">Confirmations of genetic interactions were performed using a serial spot dilution assay. Yeast cultures were grown to saturation overnight and then diluted 10-fold, five times. Five microliters of the dilutions was plated on glucose and galactose using a Biotek Precision 2000 robot and grown at 30° for 2 and 3 d, respectively, before imaging.</p></div><div id="sec-10" class="subsection"><h3>Liquid growth curve assays</h3><p id="p-23">SDL interactions identified in the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002259">dun1</a>Δ</em>::<em>natMX</em> screen were confirmed using high-resolution growth curves in a TECAN spectrophotometer. Cultures were grown in minimal glucose media to saturation then diluted with a 96-well pin tool in minimal galacose media and grown at 30°. Optical density measurements were recorded every 15 min, while shaking for 2 d. Cultures were grown in triplicate and the average G measurements were recorded. The average G measures the rate of growth as the slope of a growth curve of a particular strain during log phase. The raw fitness defect caused by overexpression was measured by normalizing average G values to the negative control [<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002259">dun1</a>Δ</em>::<em>natMX</em> overexpressing <em>FLEX-<a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005728">HIS3</a></em> plasmid (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">St Onge <em>et al.</em> 2007</a>)]. Strains with a fitness defect greater than 10% and twice the standard deviation were considered SDL with <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002259">dun1</a>Δ</em>::<em>natMX</em>.</p></div></div><div class="section results-discussion" id="sec-11"><h2 class="">Results and Discussion</h2><div id="sec-12" class="subsection"><h3>Construction of the yeast FLEX array (SGA-FLEX)</h3><p id="p-24">To construct a yeast gene overexpression array in a genetic background that is compatible with SGA technology, we made use of the Yeast FLEX array, a unique collection of Gateway-compatible plasmids carrying sequence-verified yeast ORFs (<a id="xref-ref-19-4" class="xref-bibr" href="#ref-19">Hu <em>et al.</em> 2007</a>). We used a version of the FLEX collection in which each of 5192 unique ORFs (5532 plasmids in total) is expressed from the galactose-inducible <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000224">GAL1</a>/10</em> promoter on the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000747">URA3</a></em>-based CEN plasmid pBY011 (<a id="xref-ref-19-5" class="xref-bibr" href="#ref-19">Hu <em>et al.</em> 2007</a>). We purified plasmid DNA from the bacterial transformants harboring the FLEX plasmids and used them to transform yeast strain Y6897 [<em>MAT</em>α <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005728">his3</a>Δ1 <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000523">leu2</a>Δ0 <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000747">ura3</a>Δ0 <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004294">met15</a>Δ0 <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000789">can1</a>Δ</em>::<em>STE2pr-Sp_his5 <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005212">lyp1</a>Δ</em>::<em>STE3pr-<a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000523">LEU2</a></em> (<a id="xref-ref-36-3" class="xref-bibr" href="#ref-36">Tong and Boone 2006</a>)], which is compatible with both SGA technology and the Barcoders collection (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>) (<a id="xref-ref-39-4" class="xref-bibr" href="#ref-39">Yan <em>et al.</em> 2008</a>). The transformed strains were arrayed in 96-well format plates (n = 72) and then consolidated into 18 different 384-well plates using robotic pinning to create the SGA-FLEX array (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003400/-/DC1/TableS2.xlsx">Table S2</a>).</p></div><div id="sec-13" class="subsection"><h3>Expanding the Barcoder collection</h3><p id="p-25">The use of pooled cultures of differentially barcoded mutant strains for genetic screens has a wide range of applications and offers particular advantages for chemical genetics, where small amounts of expensive and scarce chemicals can be screened for effects on the fitness of mutant strains (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Hillenmeyer <em>et al.</em> 2008</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Ho <em>et al.</em> 2009</a>; <a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Pierce <em>et al.</em> 2007</a>; <a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">St Onge <em>et al.</em> 2007</a>). Parallel analysis of pooled cultures also enables rapid analysis of cellular fitness in multiple environmental conditions. Previously, we expanded the tool-kit of barcoded reagents available in yeast to include a genomic collection of 1141 arrayed strains (Barcoders) with unique integrated barcodes at the same neutral locus, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002386">ho</a></em> (<a id="xref-ref-39-5" class="xref-bibr" href="#ref-39">Yan <em>et al.</em> 2008</a>). We expanded the number of strains in the Barcoder collection to permit genome-wide assays and to allow full coverage of the SGA-FLEX collection. Our protocol for constructing Barcoder strains involved PCR-amplification of barcodes from the original YKO (yeast knockout) array, which contains two barcodes linked to each mutant locus (<a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Giaever <em>et al.</em> 2002</a>; <a id="xref-ref-39-6" class="xref-bibr" href="#ref-39">Yan <em>et al.</em> 2008</a>). To generate the initial collection of Barcoders, we selected the 2282 unassigned barcodes that were present on the commercially available TAG4 microarray (<a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Pierce <em>et al.</em> 2006</a>) and incorporated them in pairs into 1141 tagged Barcoder strains. To augment the collection, we selected an additional 7948 barcodes present on the TAG4 microarray. Most of these additional barcodes have been used successfully in studies with the yeast deletion mutant collection (<a id="xref-ref-28-3" class="xref-bibr" href="#ref-28">Pierce <em>et al.</em> 2006</a>). We used primers that recognize the flanking sequence of the barcoded cassettes as well as homology to the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002386">ho</a></em> locus to produce PCR products for directed gene replacement into the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002386">ho</a></em> locus in the <em>MAT</em><strong>a</strong> WT strain BY4741. The new Barcoder strains (3974) were rearrayed together with the 1141 original strains to create a final array of 5115 Barcoder strains in a format compatible with the SGA-FLEX array (Barcoder v2; <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003400/-/DC1/TableS3.xlsx">Table S3</a>).</p></div><div id="sec-14" class="subsection"><h3>Generating the barFLEX collection</h3><p id="p-26">We next used the Barcoder v2 collection to efficiently add barcode identifiers to strains in the SGA-FLEX array (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>). In brief, we used the SGA method to select for haploid strains carrying one overexpression plasmid and an integrated barcode as described previously (<a id="xref-ref-39-7" class="xref-bibr" href="#ref-39">Yan <em>et al.</em> 2008</a>). The Barcoder concept is based on the principle that a barcode and any marked genetic element of interest (in this case an overexpression plasmid) do not need to be genetically linked but must only appear in the same strain in order to track that strain’s behavior (<em>e.g.</em>, fitness) in a pool. The Barcoder v2 collection can be thus used to “barcode” any SGA-compatible collection of yeast strains. We validated our barFLEX collection by sequencing 54 randomly selected ORFs and 25 uptags. In this test, 96% of barcodes and 96% of ORFs were in the correct strain and in the expected location on the strain array (Lists of ORFs and barcodes tested available on request; <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003400/-/DC1/TableS4.xlsx">Table S4</a>)</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/10/1279/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Construction of the barFLEX array. Step1: SGA using robotic pinning pairs each FLEXGene overexpression plasmid (pGAL-ORF) with a unique barcode-carrying strain. The FLEXGene plasmids are marked with a URA3 selectable marker and the Barcoder strains are marked with a kanMX resistance marker. This selection assembles an inducible barcoded overexpression array after only one round of SGA." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1505736795" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Construction of the barFLEX array. Step1: SGA using robotic pinning pairs each FLEXGene overexpression plasmid (pGAL-ORF) with a unique barcode-carrying strain. The FLEXGene plasmids are marked with a &lt;em&gt;URA3&lt;/em&gt; selectable marker and the Barcoder strains are marked with a &lt;em&gt;kanMX&lt;/em&gt; resistance marker. This selection assembles an inducible barcoded overexpression array after only one round of SGA.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/10/1279/F1.medium.gif" width="440" height="431"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/2/10/1279/F1.medium.gif" width="440" height="431"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/10/1279/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/10/1279/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461905" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-27" class="first-child">Construction of the barFLEX array. Step1: SGA using robotic pinning pairs each FLEXGene overexpression plasmid (pGAL-ORF) with a unique barcode-carrying strain. The FLEXGene plasmids are marked with a <em>URA3</em> selectable marker and the Barcoder strains are marked with a <em>kanMX</em> resistance marker. This selection assembles an inducible barcoded overexpression array after only one round of SGA.</p><div class="sb-div caption-clear"></div></div></div><p id="p-28">In summary, our barFLEX collection contains 5111 strains (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003400/-/DC1/TableS5.xlsx">Table S5</a>). The collection can be manipulated to generate either <em>MAT</em><strong>a</strong> or <em>MAT</em>α backgrounds, depending on the applied final haploid selection. The barFLEX strains are compatible with the SGA screening platform to introduce marked alleles of any genes of interest for combinatorial overexpression genetics.</p></div><div id="sec-15" class="subsection"><h3>Characterization of the barFLEX array</h3><p id="p-29">To assess the performance of our barFLEX collection in pooled competitive growth assays, we generated <em>MATα</em> barFLEX strains and then pooled them as described previously (<a id="xref-ref-29-3" class="xref-bibr" href="#ref-29">Pierce <em>et al.</em> 2007</a>). As an initial test of the quality of the barcodes, we grew the pool in either 2% glucose or 2% galactose for 20 generations. We isolated genomic DNA at the end of the growth period, amplified the barcodes, and then hybridized the amplicons to the TAG4 microarray (<a id="xref-ref-29-4" class="xref-bibr" href="#ref-29">Pierce <em>et al.</em> 2007</a>). In total, ~80% of the Barcoder strains had signal intensities for both barcodes that were three times greater than the background hybridization intensity levels, consistent with our previously published experiments using the TAG4 array (<a id="xref-ref-28-4" class="xref-bibr" href="#ref-28">Pierce <em>et al.</em> 2006</a>; <a id="xref-ref-39-8" class="xref-bibr" href="#ref-39">Yan <em>et al.</em> 2008</a>). When we asked what fraction of Barcoder strains had at least 1 barcode that satisfied our detection criteria, this fraction increased to ~90% (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003400/-/DC1/TableS5.xlsx">Table S5</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003400/-/DC1/TableS6.xlsx">Table S6</a>). The undetected barcodes below background (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003400/-/DC1/TableS7.xlsx">Table S7</a>) reflect those that contain errors in either their common priming sequences or barcodes (<a id="xref-ref-32-3" class="xref-bibr" href="#ref-32">Smith <em>et al.</em> 2009</a>).</p><p id="p-30">Due to the ever-decreasing costs of next-generation sequencing, we previously compared barcode microarray data to “bar-seq” data and found them to be generally equivalent (<a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Smith <em>et al.</em> 2010</a>). We confirmed the sequences for all the barcodes within the Barcoder collection by sequence analysis as follows (sequence data can be accessed at <a href="http://chemogenomics.med.utoronto.ca/supplemental/barflex/">http://chemogenomics.med.utoronto.ca/supplemental/barflex/</a>). We assessed the presence of barcodes in the pool by next generation sequencing. The reads generated by the sequencing of DNA prepared from the pool in a single Illumina flow cell were counted and assigned to the barcodes expected to be in the pool. Global sequence alignments were generated for each of the unidentified barcodes to all unclassified reads. Reads were associated with barcodes for which alignment scores exceeded 80 (+5 match, −1 mm/indel; <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003400/-/DC1/TableS8.xlsx">Table S8</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003400/-/DC1/TableS9.xlsx">Table S9</a>)</p><p id="p-31">Of the 5106 upstream barcodes, 2518 (49%) were identified as a perfect match and another 1774 (35%) were matched by alignment. For the 5038 downstream barcodes the numbers of perfect matches and aligned reads were 2435 (48%) and 1796 (36%), respectively. Considering both barcodes for the 5106 individual barcoder strains, 1777 (35%) had direct matches for both barcodes and 3176 (62%) for at least one barcode. At least one barcode was identified by alignment in 1506 (22%) strains. For 807 strains (8%), neither barcode was clearly seen in the sequence data generated from this particular sequenced sample. Because these percentages are derived from a single sample, we expect that additional strains will be detectable with further algorithm development (<em>e.g.</em>, which accounts for PCR-induced variation as well as flow cell geography) and the supporting website will be updated to include these additional strains. We conclude that the barcodes of the barFLEX collection are suitable for analysis using either next generation sequencing or microarrays. We have generated a list of high scoring alignments of all unique reads in our sequence data (<a href="http://chemogenomics.med.utoronto.ca/supplemental/barflex/">http://chemogenomics.med.utoronto.ca/supplemental/barflex/</a>).</p></div><div id="sec-16" class="subsection"><h3>Detecting overexpression toxicity using pooled cultures derived from barFLEX</h3><p id="p-32">Several groups have surveyed the yeast genome for genes that cause an obvious fitness defect when overexpressed (<a id="xref-ref-13-3" class="xref-bibr" href="#ref-13">Gelperin <em>et al.</em> 2005</a>; <a id="xref-ref-34-6" class="xref-bibr" href="#ref-34">Sopko <em>et al.</em> 2006</a>). All previous tests of overexpression toxicity have been performed by assaying growth on solid medium. We used our barFLEX collection to explore overexpression toxicity by manipulating pool aliquots in liquid medium as described previously (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Ericson <em>et al.</em> 2010</a>). We grew samples of the barFLEX pool in medium containing glucose (repressing) or galactose (inducing) and then collected samples every five generations for measurement of relative strain abundance. After five generations of growth in galactose-containing medium, very few strains decreased in abundance in the pool (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>). However, over the next 10 to 15 generations of growth, a number of strains, including those carrying deletion alleles of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000553">SPS22</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003184">KIP3</a></em>, and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001088">INM1</a></em>, decreased in abundance and the changes were identified clearly by microarray analysis (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2, B and C</a>). At 20 generations of growth (which is roughly equivalent to a medium-sized individual colony on an agar plate), more than 360 genes exhibited decreased abundance within the pool (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2D</a>; <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003400/-/DC1/TableS10.xlsx">Table S10</a>). Replicates of this experiment correlated well (r = 0.7897) and the variance further decreased when only the highly toxic genes (defined as those with Log<sub>2</sub> ratio &lt; −1) were considered (r = 0.847).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/10/1279/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparison of toxicity in pooled cultures after 5 (A), 10 (B), 15 (C), and 20 (D) generations of growth. Scatterplots depict all barcode Log2 ratios of signal intensities that correspond to the fitness of each strain on the barFLEX array. Strains containing plasmids bearing toxic genes have a lower negative Log2 ratio. The greatest number of toxic genes are detected at 20 generations of growth. Genes are ordered on the X-axis according to their position on the barFLEX array. Genes referred to in the text are highlighted on the plots. Red line shows the cut off at Log2 ratio of &#x2212;1." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1505736795" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Comparison of toxicity in pooled cultures after 5 (A), 10 (B), 15 (C), and 20 (D) generations of growth. Scatterplots depict all barcode Log&lt;sub&gt;2&lt;/sub&gt; ratios of signal intensities that correspond to the fitness of each strain on the barFLEX array. Strains containing plasmids bearing toxic genes have a lower negative Log&lt;sub&gt;2&lt;/sub&gt; ratio. The greatest number of toxic genes are detected at 20 generations of growth. Genes are ordered on the X-axis according to their position on the barFLEX array. Genes referred to in the text are highlighted on the plots. Red line shows the cut off at Log2 ratio of &#x2212;1.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/10/1279/F2.medium.gif" width="440" height="313"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/2/10/1279/F2.medium.gif" width="440" height="313"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/10/1279/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/10/1279/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461994" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-33" class="first-child">Comparison of toxicity in pooled cultures after 5 (A), 10 (B), 15 (C), and 20 (D) generations of growth. Scatterplots depict all barcode Log<sub>2</sub> ratios of signal intensities that correspond to the fitness of each strain on the barFLEX array. Strains containing plasmids bearing toxic genes have a lower negative Log<sub>2</sub> ratio. The greatest number of toxic genes are detected at 20 generations of growth. Genes are ordered on the X-axis according to their position on the barFLEX array. Genes referred to in the text are highlighted on the plots. Red line shows the cut off at Log2 ratio of −1.</p><div class="sb-div caption-clear"></div></div></div><p id="p-34">We compared the growth of the barFLEX collection in pooled liquid culture to growth of the array on plates. We examined the collection on solid medium for defects in growth by measuring colony size differences on glucose- or galactose-containing medium. The barFLEX array was pinned in a 1536 strain format onto glucose- and galactose-containing plates and colony size was assessed using an automated colony scoring method (<a id="xref-ref-38-3" class="xref-bibr" href="#ref-38">Tong <em>et al.</em> 2004</a>). This analysis was undertaken to: (1) assess the quality of the array, that is, whether barcodes and plasmids were assembled correctly; (2) validate the array for use in a pooled culture format and; (3) compare our results with previous measurements of gene toxicity on solid medium. In total, 411 genes in the barFLEX array appeared to cause a slow-growth phenotype using colony size as a proxy for fitness (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003400/-/DC1/TableS11.xlsx">Table S11</a>). Despite the obvious technical differences between the two experiments, the competitive growth assay of overexpression toxicity correlated significantly with the measurements of toxicity using growth on solid medium (R = 0.68, R<sup>2</sup> = 0.46; <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>). Only 6 of the 65 genes that were considered toxic on solid medium did not meet the cutoff that we used to identify a toxic gene in our pooled liquid growth assay (Log<sub>2</sub> ratio &lt; −1). Also, only 4 of the 95 genes that caused a significant growth defect when overexpressed in liquid medium (Log<sub>2</sub> &lt; −3) were not obviously toxic in our plate-based assay. Genes that were highly toxic upon overexpression as assessed in our competitive liquid growth assay (Log<sub>2</sub> &lt; −4, representing a 16-fold difference in abundance) were also identified in the plate assay, with a false-positive rate of 2% (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003400/-/DC1/TableS12.xlsx">Table S12</a>). As expected, the false-positive rate increased as the toxicity decreased, presumably due to the increased experimental variability associated with assessing a more subtle phenotype.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/10/1279/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Characterization of the barFLEX toxic genes. (A) Comparison of barFLEX toxic lists collected using pooled growth vs. growth in solid medium. Log2 ratio of barcode intensities are plotted against colony fitness scores. (B) Venn Diagram of overlap between toxic lists in barFLEX, GST, and mORF collections. Three different genomic yeast collections were tested for genes that cause slow growth phenotypes when overexpressed. The FLEX and GST collections were tested on solid growth media + galactose to induce overexpression. The mORF collection was tested on solid media + galactose + glycerol + ethanol. (C) Overlap of GST and barFLEX by degree of toxicity. The overlap between only the toxic genes in the GST-ORF and barFLEX collection is shown. The GST-ORF collection was scored for toxic genes by serial spot dilution. A scale from 1 to 4 was used to qualitatively describe the fitness defect caused by overexpressing each gene (Sopko et al. 2006)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1505736795" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Characterization of the barFLEX toxic genes. (A) Comparison of barFLEX toxic lists collected using pooled growth &lt;em&gt;vs.&lt;/em&gt; growth in solid medium. Log&lt;sub&gt;2&lt;/sub&gt; ratio of barcode intensities are plotted against colony fitness scores. (B) Venn Diagram of overlap between toxic lists in barFLEX, GST, and mORF collections. Three different genomic yeast collections were tested for genes that cause slow growth phenotypes when overexpressed. The FLEX and GST collections were tested on solid growth media + galactose to induce overexpression. The mORF collection was tested on solid media + galactose + glycerol + ethanol. (C) Overlap of GST and barFLEX by degree of toxicity. The overlap between only the toxic genes in the GST-ORF and barFLEX collection is shown. The GST-ORF collection was scored for toxic genes by serial spot dilution. A scale from 1 to 4 was used to qualitatively describe the fitness defect caused by overexpressing each gene (Sopko &lt;em&gt;et al.&lt;/em&gt; 2006).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/10/1279/F3.medium.gif" width="440" height="341"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/2/10/1279/F3.medium.gif" width="440" height="341"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/10/1279/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/10/1279/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461895" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-35" class="first-child">Characterization of the barFLEX toxic genes. (A) Comparison of barFLEX toxic lists collected using pooled growth <em>vs.</em> growth in solid medium. Log<sub>2</sub> ratio of barcode intensities are plotted against colony fitness scores. (B) Venn Diagram of overlap between toxic lists in barFLEX, GST, and mORF collections. Three different genomic yeast collections were tested for genes that cause slow growth phenotypes when overexpressed. The FLEX and GST collections were tested on solid growth media + galactose to induce overexpression. The mORF collection was tested on solid media + galactose + glycerol + ethanol. (C) Overlap of GST and barFLEX by degree of toxicity. The overlap between only the toxic genes in the GST-ORF and barFLEX collection is shown. The GST-ORF collection was scored for toxic genes by serial spot dilution. A scale from 1 to 4 was used to qualitatively describe the fitness defect caused by overexpressing each gene (<a id="xref-ref-34-7" class="xref-bibr" href="#ref-34">Sopko <em>et al.</em> 2006</a>).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-17" class="subsection"><h3>Comparing barFLEX toxic set to other datasets of toxic genes</h3><p id="p-36">As noted previously our analysis of the barFLEX collection revealed 411 genes that caused a sick or lethal phenotype when overexpressed, using colony-size as a measure for fitness (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003400/-/DC1/TableS11.xlsx">Table S11</a>). Previous assessment of the GST-ORF collection (<a id="xref-ref-34-8" class="xref-bibr" href="#ref-34">Sopko <em>et al.</em> 2006</a>) revealed a toxic set of 769 genes, and another study of the mORF GAL-ORF overexpression collection (<a id="xref-ref-13-4" class="xref-bibr" href="#ref-13">Gelperin <em>et al.</em> 2005</a>) identified 370 toxic genes. Comparing these results, we found that the toxic genes sets associated with any two collections overlaps by ~25% (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3B</a>), differences that presumably reflect the unique features of each collection and the unique assays used for assessing toxicity. Key array features to consider include: (1) the low-copy centromeric plasmid backbone (CEN) of the FLEX collection <em>vs</em>. the high-copy backbone (2 μ) of the GST collection; (2) the quality of each ORF in each collection (sequenced verified in the FLEX collection but not in others); (3) the particular strain background used and; (4) differences in the assays used to measure gene toxicity. For example, the mORF collection was scored for overexpression toxicity by analyzing the size of the colony resulting from spotting a single 3μL culture onto solid medium (<a id="xref-ref-13-5" class="xref-bibr" href="#ref-13">Gelperin <em>et al.</em> 2005</a>).</p><p id="p-37">In contrast, the GST-ORF toxic set was identified by serial spot dilution assays and genes were assigned a qualitative score ranging from 1 (lethal) to 4 (least sick) (<a id="xref-ref-34-9" class="xref-bibr" href="#ref-34">Sopko <em>et al.</em> 2006</a>) while we measured growth of the barFLEX collection by pinning the array and measuring colony size using an automated scoring method. The degree of overlap between toxic gene lists correlated with the degree of toxicity as determined by serial spot dilution of the GST collection, with close to 40% overlap in the most toxic genes, down to ∼10% overlap in the less toxic genes (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3C</a>). This result is expected if the most toxic genes (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003400/-/DC1/TableS10.xlsx">Table S10</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003400/-/DC1/TableS12.xlsx">Table S12</a>) are more easily detected using different assays (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003400/-/DC1/TableS13.xlsx">Table S13</a>).</p><p id="p-38">Previous analysis of genes that cause fitness defects when overexpressed revealed a significant functional enrichment for genes with annotated roles in cell cycle regulation and mitosis (<a id="xref-ref-13-6" class="xref-bibr" href="#ref-13">Gelperin <em>et al.</em> 2005</a>; <a id="xref-ref-34-10" class="xref-bibr" href="#ref-34">Sopko <em>et al.</em> 2006</a>). The barFLEX toxic gene set was similarly enriched for cell cycle genes [<em>e.g.</em>, mitosis (<em>P</em> = 1.232 × 10<sup>−8</sup>) and genes involved in cell cycle regulation [<em>P</em> = 2.986 × 10<sup>−8</sup> (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Robinson <em>et al.</em> 2002</a>)]. Genes that were uniquely toxic in the GST-ORF collection were also enriched for genes with roles in intracellular protein transport (<em>P</em> = 6.91 × 10<sup>−5</sup>), which we did not observe with the barFLEX collection, suggesting the GST-ORF enrichment may reflect an effect of the GST tag on protein transport that is less likely to be seen with untagged genes expressed in the barFLEX collection.</p></div><div id="sec-18" class="subsection"><h3>Using the barFLEX collection for SDL screening in a pooled format</h3><p id="p-39">As noted previously, overexpression of most genes in WT cells is phenotypically benign, but sensitivity to gene dosage can be revealed in strains mutated for an interacting protein or pathway component, a so-called SDL interaction. To develop protocols for using the barFLEX collection for identifying SDL interactions, we performed an SDL screen using a query strain deleted for the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002259">DUN1</a></em> kinase. We chose the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002259">dun1</a>Δ</em> strain for our test screens since it is conditionally activated by DNA damage and has been previously screened using colony size measurements on solid medium and the GST-tagged overexpression array (<a id="xref-ref-31-4" class="xref-bibr" href="#ref-31">Sharifpoor <em>et al.</em> 2012</a>). <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002259">Dun1</a> responds to DNA damage downstream of the <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006074">Rad53</a> and <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000340">Mec1</a> kinases and phosphorylates <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002483">Rad55</a>, a protein involved in double strand break repair (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Bashkirov <em>et al.</em> 2003</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Zhao and Rothstein 2002</a>). We generated a <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002259">dun1</a>Δ</em> query strain marked with a <em>natMX</em> antibiotic resistance cassette in a <em>MAT</em><strong>a</strong> BY4741 background (lacking SGA mating type selectable markers). We then used SGA to introduce the kinase deletion allele into the barFLEX collection and selected for haploids carrying the overexpression plasmid, the integrated barcode, and the deletion allele (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003400/-/DC1/FigureS3.pdf">Figure S3</a>). After the final selection, we pooled the strains and cultured them alongside a negative control strain (<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000747">ura3</a>Δ</em>::<em>natMX</em>) in glucose- or galactose-containing medium for a total of 20 generations (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003400/-/DC1/FigureS3.pdf">Figure S3</a>).</p><p id="p-40">To identify SDL interactions, we compared the pooled growth ratio of each barcoded strain that also carried our deletion of interest in glucose/galactose after 20 generations (cutoff Log<sub>2</sub> ratio &lt; −1), eliminating any genes that were also toxic in the negative control screen (difference of Log<sub>2</sub> ratio &lt; −1). Using this statistical cut-off, the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002259">dun1</a>Δ</em> screen revealed a list of 52 interactions (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4A</a>; <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003400/-/DC1/TableS14.xlsx">Table S14</a>), 25 of which have at least one phosphorylated residue (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Amoutzias <em>et al.</em> 2012</a>). The SDL interactions included genes encoding two known downstream proteins <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006222">RLF2</a></em>, which encodes a member of the CAF-1 chromatin assembly complex, and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005899">NDD1</a></em>, which encodes a transcriptional activator of S-phase genes, as the top hits. We also identified genes known to be involved in DNA repair, such as <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001441">MPH1</a></em>, a 3′-5′ DNA helicase involved in error-free bypass of DNA lesions, and previously uncharacterized genes, <em>e.g.</em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006219">YPR015C</a></em>. We confirmed all of the SDL interactions identified from the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002259">dun1</a></em>Δ pooled screens using either serial spot dilution as previously described (<a id="xref-ref-31-5" class="xref-bibr" href="#ref-31">Sharifpoor <em>et al.</em> 2012</a>) or using automated liquid growth curve assays (see <em>Materials and Methods</em>). We used stringent criteria for identifying an SDL interaction in our secondary assays; we demanded an obvious growth difference that was clear over two 10-fold serial spot dilutions or a &gt;10% difference in growth fitness scores, with the fitness defect larger than twice the standard deviation between the control and the kinase deletion strain (<a id="xref-ref-35-3" class="xref-bibr" href="#ref-35">St Onge <em>et al.</em> 2007</a>). Using these assays, we confirmed 20 of the interactions for <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002259">dun1</a></em>Δ using either spot dilutions or automated liquid growth curves (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>; <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003400/-/DC1/TableS14.xlsx">Table S14</a>), corresponding to a confirmation rate of ~38%. This confirmation rate is less than that seen in large-scale SDL screens of the kinome [46% (<a id="xref-ref-31-6" class="xref-bibr" href="#ref-31">Sharifpoor <em>et al.</em> 2012</a>)] using a colony-based readout and likely reflects a combination of factors, including our stringent cut-off, which affects sensitivity, and innate differences in the assay conditions.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/10/1279/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Assessment of SDL interactions in a dun1 deletion using the barFLEX collection in pooled cultures under standard growth conditions or in the presence of 0.001% and 0.0001% MMS. Scatterplots depict all SDL interactions identified using dun1&#x394; in standard growth conditions (A) or in the presence of 0.001% MMS (B) and 0.0001% MMS (C). Log2 ratios of barcode signal intensities corresponding to fitness of each strain when overexpressed in the dun1&#x394; are plotted. Genes are ordered in the X-axis according to their position on the barFLEX array. Genes referred to in the text are highlighted on the plots." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1505736795" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Assessment of SDL interactions in a &lt;em&gt;dun1&lt;/em&gt; deletion using the barFLEX collection in pooled cultures under standard growth conditions or in the presence of 0.001% and 0.0001% MMS. Scatterplots depict all SDL interactions identified using &lt;em&gt;dun1&lt;/em&gt;&#x394; in standard growth conditions (A) or in the presence of 0.001% MMS (B) and 0.0001% MMS (C). Log&lt;sub&gt;2&lt;/sub&gt; ratios of barcode signal intensities corresponding to fitness of each strain when overexpressed in the &lt;em&gt;dun1&lt;/em&gt;&#x394; are plotted. Genes are ordered in the X-axis according to their position on the barFLEX array. Genes referred to in the text are highlighted on the plots.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/10/1279/F4.medium.gif" width="180" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/2/10/1279/F4.medium.gif" width="180" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/10/1279/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/10/1279/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461970" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-41" class="first-child">Assessment of SDL interactions in a <em>dun1</em> deletion using the barFLEX collection in pooled cultures under standard growth conditions or in the presence of 0.001% and 0.0001% MMS. Scatterplots depict all SDL interactions identified using <em>dun1</em>Δ in standard growth conditions (A) or in the presence of 0.001% MMS (B) and 0.0001% MMS (C). Log<sub>2</sub> ratios of barcode signal intensities corresponding to fitness of each strain when overexpressed in the <em>dun1</em>Δ are plotted. Genes are ordered in the X-axis according to their position on the barFLEX array. Genes referred to in the text are highlighted on the plots.</p><div class="sb-div caption-clear"></div></div></div><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/10/1279/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Confirmed SDL interactions for dun1 using serial spot dilutions. Eight confirmed SDL interactions with DUN1 are shown. Cells were diluted 15-fold at each step and spotted on either glucose (left) or galactose (right) to induce overexpression. Corresponding Log2 ratios of barcode intensities in both control (black) and dun1&#x394; (red) strains are shown." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1505736795" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Confirmed SDL interactions for &lt;em&gt;dun1&lt;/em&gt; using serial spot dilutions. Eight confirmed SDL interactions with &lt;em&gt;DUN1&lt;/em&gt; are shown. Cells were diluted 15-fold at each step and spotted on either glucose (left) or galactose (right) to induce overexpression. Corresponding Log&lt;sub&gt;2&lt;/sub&gt; ratios of barcode intensities in both control (black) and &lt;em&gt;dun1&lt;/em&gt;&#x394; (red) strains are shown.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/10/1279/F5.medium.gif" width="440" height="428"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/2/10/1279/F5.medium.gif" width="440" height="428"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/10/1279/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/10/1279/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461936" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-42" class="first-child">Confirmed SDL interactions for <em>dun1</em> using serial spot dilutions. Eight confirmed SDL interactions with <em>DUN1</em> are shown. Cells were diluted 15-fold at each step and spotted on either glucose (left) or galactose (right) to induce overexpression. Corresponding Log<sub>2</sub> ratios of barcode intensities in both control (black) and <em>dun1</em>Δ (red) strains are shown.</p><div class="sb-div caption-clear"></div></div></div><p id="p-43">Screening the kinome using traditional SDL screening approaches revealed that many kinase deletion alleles show few or no SDL interactions (<a id="xref-ref-31-7" class="xref-bibr" href="#ref-31">Sharifpoor <em>et al.</em> 2012</a>). To test the utility of the barFLEX array for revealing condition-specific SDL interactions, we next treated the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002259">dun1</a></em>Δ-barFLEX pool with a sublethal dose of the DNA methylating agent MMS (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4, B and C</a>). We examined both a control (<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000747">ura3</a></em>Δ-barFLEX<em>)</em> and the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002259">dun1</a></em>Δ-barFLEX collections throughout 20 generations of growth in three different conditions: (1) galactose (to induce overexpression) for 20 generations; (2) galactose for five generations of growth followed by addition of 0.0001% MMS for 15 generations; (3) galactose for five generations followed by addition of 0.001% MMS for five generations to a total of 20 generations. Glucose (noninduced) controls were performed in parallel. The dose of MMS used in the experiment was selected to induce only a slight fitness defect in the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002259">dun1</a>Δ</em> strain. In the presence of 0.001% MMS, the WT (<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000747">ura3</a>Δ</em>) screen identified 197 additional toxic genes (Log<sub>2</sub> &lt; −1; <a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4B</a>; <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003400/-/DC1/TableS15.xlsx">Table S15</a>) including known repair genes <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003143">SAE2</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001441">MPH1</a></em>, and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004745">PSO2</a></em>, which is consistent with published chemogenomic data (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Lee <em>et al.</em> 2005</a>). At 0.0001% MMS, 135 additional toxic genes were identified (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4C</a>; <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003400/-/DC1/TableS16.xlsx">Table S16</a>), including <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001441">MPH1</a></em>, as well as other DNA repair genes such as <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005366">TOP1</a></em> and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001416">IMP2</a></em> (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Downs <em>et al.</em> 2000</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Masson and Ramotar 1996</a>). The <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002259">dun1</a>Δ</em> screen in the presence of either concentration of MMS identified a number of condition-specific SDL interactions (77 additional genes for 0.001% MMS and 45 additional genes for 0.0001% MMS), including interactions with the known checkpoint gene <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002907">LCD1</a></em> and the repair gene <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003796">RAD26</a></em> (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003400/-/DC1/TableS17.xlsx">Table S17</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003400/-/DC1/TableS18.xlsx">Table S18</a>). The strain overexpressing the known target of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002259">Dun1</a>, <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004523">Sml1</a>, decreased in strain abundance in all three of the conditions compared with WT but was below the stringent statistical cut-off that we chose for our experiments.</p><p id="p-44">We were intrigued by our discovery of a previously uncharacterized gene, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006219">YPR015C</a></em>, which encodes a zinc finger transcription factor, as an SDL interaction with <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002259">DUN1</a></em> deletion in both standard growth and in DNA damage-inducing conditions. In WT cells, <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002259">Dun1</a> regulates the levels of ribonucleotide reductase gene expression (<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003563">RNR2</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001328">RNR3</a></em>, and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003412">RNR4</a></em>) in both standard growth conditions and in the presence of DNA damage through degradation of the ribonucleotide reductase inhibitor <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004523">Sml1</a> (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Elledge <em>et al.</em> 1993</a>; <a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Zhao and Rothstein 2002</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Zhou and Elledge 1993</a>). <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002259">Dun1</a> also acts with <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000340">Mec1</a> and <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006074">Rad53</a> to phosphorylate a repressor of <em>RNR</em> transcription, <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004166">Crt1</a> (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Huang <em>et al.</em> 1998</a>). Phosphorylation of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004166">Crt1</a> leads to derepression of <em>RNR</em> gene expression as well as expression of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006219">YPR015C</a></em>. Overexpression of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006219">YPR015C</a></em> causes up-regulation of <em>RNR</em> genes (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Chua <em>et al.</em> 2006</a>), suggesting that <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006219">YPR015C</a></em> may be involved in the DNA damage checkpoint. Finally, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006219">YPR015C</a></em> overexpression causes a cell cycle delay in G2 (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Niu <em>et al.</em> 2008</a>), as well as a mildly toxic phenotype in the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004523">SML1</a></em> deletion strain (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>). To ask whether <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006219">YPR015C</a></em> acts in the same pathway as <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002259">Dun1</a>-<a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004523">Sml1</a>, we constructed an <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004523">sml1</a></em>Δ<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002259">dun1</a></em>Δ double-mutant strain carrying a plasmid expressing <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006219">YPR015C</a></em> from the inducible GAL promoter. Consistent with the known suppression of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002259">DUN1</a></em> mutant phenotypes by deletion of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004523">SML1</a></em>, we saw that deletion of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004523">SML1</a></em> also partially suppressed the SDL phenotype caused by overexpression of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006219">YPR015C</a></em> in the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002259">dun1</a></em>Δ mutant (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6</a>). This result is consistent with our finding that <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006219">YPR015C</a></em> is less toxic when overexpressed in <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004523">sml1</a></em>Δ, compared with the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002259">dun1</a></em>Δ background (<a id="xref-fig-6-3" class="xref-fig" href="#F6">Figure 6</a>). Together, these results suggest that the toxicity caused by the overexpression of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006219">YPR015C</a></em> in the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002259">dun1</a>Δ</em> is likely due to increased expression of the <em>RNR</em> genes.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/10/1279/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Deletion of SML1 suppresses the toxicity caused by the overexpression of YPR015C in a dun1&#x394; background. Serial 15-fold dilutions of each strain were spotted on (right) glucose- and (left) galactose-containing plates to induce overexpression." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1505736795" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Deletion of &lt;em&gt;SML1&lt;/em&gt; suppresses the toxicity caused by the overexpression of &lt;em&gt;YPR015C&lt;/em&gt; in a &lt;em&gt;dun1&lt;/em&gt;&#x394; background. Serial 15-fold dilutions of each strain were spotted on (right) glucose- and (left) galactose-containing plates to induce overexpression.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/10/1279/F6.medium.gif" width="440" height="241"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/2/10/1279/F6.medium.gif" width="440" height="241"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/10/1279/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/10/1279/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461950" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-45" class="first-child">Deletion of <em>SML1</em> suppresses the toxicity caused by the overexpression of <em>YPR015C</em> in a <em>dun1</em>Δ background. Serial 15-fold dilutions of each strain were spotted on (right) glucose- and (left) galactose-containing plates to induce overexpression.</p><div class="sb-div caption-clear"></div></div></div><p id="p-46">To summarize, induction of gene overexpression using nutrient-inducible promoters (<em>e.g.</em>, galactose-inducible promoters) has enabled functional assessment of genetic perturbations for gain-of-function alleles (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Chua <em>et al.</em> 2004</a>; <a id="xref-ref-13-7" class="xref-bibr" href="#ref-13">Gelperin <em>et al.</em> 2005</a>; <a id="xref-ref-34-11" class="xref-bibr" href="#ref-34">Sopko <em>et al.</em> 2006</a>). A modification of this approach uses a chimeric transcriptional activator that fuses the DNA-binding domain of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006169">Gal4</a> to the estrogen receptor and the VP16 activator, allowing rapid and specific induction of individual genes using the hormone, β-estradiol (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">McIsaac <em>et al.</em> 2011</a>). Such fast-acting and graded promoters enable dynamic assessment of overexpression phenotypes resulting from mis-regulation of complex and presumably highly buffered regulatory networks. Here, we describe development of the new barFLEX array that allows for assessment of overexpression phenotypes in pooled liquid cultures. We validated the barFLEX collection for facile detection of genes exhibiting conditional toxicity and for SDL screening in liquid cultures. Although simple in design, testing WT backgrounds in different conditions promises to reveal new gene functions. In addition, the full Barcoder reagent set we describe is amenable to diverse applications and can be used to barcode other plasmids or mutant collections for high throughput experiments, including those involving other overexpression systems (<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">McIsaac <em>et al.</em> 2011</a>).</p><p id="p-47">In addition to screening for fitness defects caused by gene overexpression in WT cells, we also validated the barFLEX collection for detection of genes exhibiting conditional SDL using kinases as a test case. By perturbing a barFLEX pool carrying a deletion of the kinase gene <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002259">DUN1</a></em> with the DNA damaging agent MMS, we uncovered novel genetic interactions that were not apparent in standard conditions. This general approach will be extremely valuable for studying kinases that are known to be conditionally active and other regulators that are required under particular conditions. Conditions that might be useful for SDL and other overexpression screens could be identified by surveying data from systematic phenotypic analysis of the deletion collection (<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Hillenmeyer <em>et al.</em> 2008</a>) or from large genetic interaction datasets that may indicate a genetic background suitable for screening (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Costanzo <em>et al.</em> 2010</a>).</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-48">The described project was supported by grants from the Canadian Cancer Society (020380), Canadian Institute for Health Research (MOP-81340), and the National Human Genome Research Institute (R01 HG003317-08) to G.G. and C.N. and the Canadian Institute for Health Research (MOP-102629 and MOP-114929) and the National Institutes of Health (R01 HG005853-02) to B.J.A. and C.B. A.M.S. was supported by the University of Toronto Open Fellowship.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003400/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003400/-/DC1</a>.</p></li><li class="fn" id="fn-5"><p id="p-49">Communicating editor: R. B. Brem</p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2012-06-13"><span class="received-label">Received </span>June 13, 2012.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2012-08-22"><span class="accepted-label">Accepted </span>August 22, 2012.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2012 A. C. Douglas <em>et al</em>.</li></ul><div class="license" id="license-1"><p id="p-4">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-other" id="cit-2.10.1279.1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Amoutzias</span>  <span class="cit-name-given-names">G. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">He</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lilley</span>  <span class="cit-name-given-names">K. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van de Peer</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oliver</span>  <span class="cit-name-given-names">S. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Evaluation and properties of the budding yeast phosphoproteome</span>. <span class="cit-source">Mol. Cell Proteomics</span> <span class="cit-vol">11</span>: <span class="cit-fpage">M111 009555</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BCell%2BProteomics%26rft.volume%253D11%26rft.spage%253DM111%2B009555%26rft.atitle%253DEVALUATION%2BAND%2BPROPERTIES%2BOF%2BTHE%2BBUDDING%2BYEAST%2BPHOSPHOPROTEOME%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-2.10.1279.2" data-doi="10.1038/nmeth.1534"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Baryshnikova</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Costanzo</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ding</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koh</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Quantitative analysis of fitness and genetic interactions in yeast on a genome scale.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">1017</span>–<span class="cit-lpage">1024</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DBaryshnikova%26rft.auinit1%253DA.%26rft.volume%253D7%26rft.issue%253D12%26rft.spage%253D1017%26rft.epage%253D1024%26rft.atitle%253DQuantitative%2Banalysis%2Bof%2Bfitness%2Band%2Bgenetic%2Binteractions%2Bin%2Byeast%2Bon%2Ba%2Bgenome%2Bscale.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1534%26rft_id%253Dinfo%253Apmid%252F21076421%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1534&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21076421&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F10%2F1279.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000284686300018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-2.10.1279.3" data-doi="10.1128/MCB.23.4.1441-1452.2003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bashkirov</span>  <span class="cit-name-given-names">V. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bashkirova</span>  <span class="cit-name-given-names">E. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haghnazari</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heyer</span>  <span class="cit-name-given-names">W. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Direct kinase-to-kinase signaling mediated by the FHA phosphoprotein recognition domain of the Dun1 DNA damage checkpoint kinase.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">1441</span>–<span class="cit-lpage">1452</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DBashkirov%26rft.auinit1%253DV.%2BI.%26rft.volume%253D23%26rft.issue%253D4%26rft.spage%253D1441%26rft.epage%253D1452%26rft.atitle%253DDirect%2BKinase-to-Kinase%2BSignaling%2BMediated%2Bby%2Bthe%2BFHA%2BPhosphoprotein%2BRecognition%2BDomain%2Bof%2Bthe%2BDun1%2BDNA%2BDamage%2BCheckpoint%2BKinase%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.23.4.1441-1452.2003%26rft_id%253Dinfo%253Apmid%252F12556502%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjk6IjIzLzQvMTQ0MSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzIvMTAvMTI3OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-2.10.1279.4" data-doi="10.1126/science.1176495"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Breitkreutz</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Choi</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sharom</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boucher</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Neduva</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">A global protein kinase and phosphatase interaction network in yeast.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">328</span>: <span class="cit-fpage">1043</span>–<span class="cit-lpage">1046</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DBreitkreutz%26rft.auinit1%253DA.%26rft.volume%253D328%26rft.issue%253D5981%26rft.spage%253D1043%26rft.epage%253D1046%26rft.atitle%253DA%2BGlobal%2BProtein%2BKinase%2Band%2BPhosphatase%2BInteraction%2BNetwork%2Bin%2BYeast%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1176495%26rft_id%253Dinfo%253Apmid%252F20489023%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMjgvNTk4MS8xMDQzIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMi8xMC8xMjc5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-2.10.1279.5" data-doi="10.1073/pnas.0605140103"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chua</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morris</span>  <span class="cit-name-given-names">Q. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sopko</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robinson</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ryan</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Identifying transcription factor functions and targets by phenotypic activation.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">103</span>: <span class="cit-fpage">12045</span>–<span class="cit-lpage">12050</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0605140103%26rft_id%253Dinfo%253Apmid%252F16880382%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTAzLzMyLzEyMDQ1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMi8xMC8xMjc5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-2.10.1279.6" data-doi="10.1016/j.mib.2004.10.009"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chua</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robinson</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morris</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hughes</span>  <span class="cit-name-given-names">T. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Transcriptional networks: reverse-engineering gene regulation on a global scale.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Microbiol.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">638</span>–<span class="cit-lpage">646</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Bmicrobiology%26rft.stitle%253DCurr%2BOpin%2BMicrobiol%26rft.aulast%253DChua%26rft.auinit1%253DG.%26rft.volume%253D7%26rft.issue%253D6%26rft.spage%253D638%26rft.epage%253D646%26rft.atitle%253DTranscriptional%2Bnetworks%253A%2Breverse-engineering%2Bgene%2Bregulation%2Bon%2Ba%2Bglobal%2Bscale.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.mib.2004.10.009%26rft_id%253Dinfo%253Apmid%252F15556037%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.mib.2004.10.009&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15556037&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F10%2F1279.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000225782400012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-2.10.1279.7" data-doi="10.1126/science.1180823"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Costanzo</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baryshnikova</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bellay</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spear</span>  <span class="cit-name-given-names">E. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The genetic landscape of a cell.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">327</span>: <span class="cit-fpage">425</span>–<span class="cit-lpage">431</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DCostanzo%26rft.auinit1%253DM.%26rft.volume%253D327%26rft.issue%253D5964%26rft.spage%253D425%26rft.epage%253D431%26rft.atitle%253DThe%2BGenetic%2BLandscape%2Bof%2Ba%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1180823%26rft_id%253Dinfo%253Apmid%252F20093466%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjcvNTk2NC80MjUiO3M6NDoiYXRvbSI7czoxOToiL2dnZy8yLzEwLzEyNzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-2.10.1279.8" data-doi="10.1073/pnas.0806261105"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dixon</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fedyshyn</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koh</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Prasad</span>  <span class="cit-name-given-names">T. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chahwan</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Significant conservation of synthetic lethal genetic interaction networks between distantly related eukaryotes.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">105</span>: <span class="cit-fpage">16653</span>–<span class="cit-lpage">16658</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0806261105%26rft_id%253Dinfo%253Apmid%252F18931302%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA1LzQzLzE2NjUzIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMi8xMC8xMjc5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-2.10.1279.9" data-doi="10.1126/science.1189015"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dowell</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ryan</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jansen</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheung</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Agarwala</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genotype to phenotype: a complex problem.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">328</span>: <span class="cit-fpage">469</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DDowell%26rft.auinit1%253DR.%2BD.%26rft.volume%253D328%26rft.issue%253D5977%26rft.spage%253D469%26rft.epage%253D469%26rft.atitle%253DGenotype%2Bto%2BPhenotype%253A%2BA%2BComplex%2BProblem%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1189015%26rft_id%253Dinfo%253Apmid%252F20413493%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjgvNTk3Ny80NjkiO3M6NDoiYXRvbSI7czoxOToiL2dnZy8yLzEwLzEyNzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-2.10.1279.10" data-doi="10.1038/35050000"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Downs</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lowndes</span>  <span class="cit-name-given-names">N. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jackson</span>  <span class="cit-name-given-names">S. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">A role for <em>Saccharomyces cerevisiae</em> histone H2A in DNA repair.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">408</span>: <span class="cit-fpage">1001</span>–<span class="cit-lpage">1004</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DDowns%26rft.auinit1%253DJ.%2BA.%26rft.volume%253D408%26rft.issue%253D6815%26rft.spage%253D1001%26rft.epage%253D1004%26rft.atitle%253DA%2Brole%2Bfor%2BSaccharomyces%2Bcerevisiae%2Bhistone%2BH2A%2Bin%2BDNA%2Brepair.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F35050000%26rft_id%253Dinfo%253Apmid%252F11140636%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/35050000&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11140636&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F10%2F1279.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000165951100057&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-2.10.1279.11" data-doi="10.1002/bies.950150507"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Elledge</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Allen</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Navas</span>  <span class="cit-name-given-names">T. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">DNA damage and cell cycle regulation of ribonucleotide reductase.</span> <abbr class="cit-jnl-abbrev">Bioessays</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">333</span>–<span class="cit-lpage">339</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioEssays%2B%253A%2B%2Bnews%2Band%2Breviews%2Bin%2Bmolecular%252C%2Bcellular%2Band%2Bdevelopmental%2Bbiology%26rft.stitle%253DBioessays%26rft.aulast%253DElledge%26rft.auinit1%253DS.%2BJ.%26rft.volume%253D15%26rft.issue%253D5%26rft.spage%253D333%26rft.epage%253D339%26rft.atitle%253DDNA%2Bdamage%2Band%2Bcell%2Bcycle%2Bregulation%2Bof%2Bribonucleotide%2Breductase.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fbies.950150507%26rft_id%253Dinfo%253Apmid%252F8343143%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/bies.950150507&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8343143&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F10%2F1279.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1993LF19300006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-2.10.1279.12" data-doi="10.1016/S0076-6879(10)70010-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ericson</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoon</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">St Onge</span>  <span class="cit-name-given-names">R. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Giaever</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nislow</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Exploring gene function and drug action using chemogenomic dosage assays.</span> <abbr class="cit-jnl-abbrev">Methods Enzymol.</abbr> <span class="cit-vol">470</span>: <span class="cit-fpage">233</span>–<span class="cit-lpage">255</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2Bin%2Benzymology%26rft.stitle%253DMethods%2BEnzymol%26rft.aulast%253DEricson%26rft.auinit1%253DE.%26rft.volume%253D470%26rft.spage%253D233%26rft.epage%253D255%26rft.atitle%253DExploring%2Bgene%2Bfunction%2Band%2Bdrug%2Baction%2Busing%2Bchemogenomic%2Bdosage%2Bassays.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0076-6879%252810%252970010-0%26rft_id%253Dinfo%253Apmid%252F20946813%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0076-6879(10)70010-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20946813&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F10%2F1279.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-2.10.1279.13" data-doi="10.1101/gad.1362105"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gelperin</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">White</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilkinson</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kon</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kung</span>  <span class="cit-name-given-names">L. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Biochemical and genetic analysis of the yeast proteome with a movable ORF collection.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">2816</span>–<span class="cit-lpage">2826</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DGelperin%26rft.auinit1%253DD.%2BM.%26rft.volume%253D19%26rft.issue%253D23%26rft.spage%253D2816%26rft.epage%253D2826%26rft.atitle%253DBiochemical%2Band%2Bgenetic%2Banalysis%2Bof%2Bthe%2Byeast%2Bproteome%2Bwith%2Ba%2Bmovable%2BORF%2Bcollection%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.1362105%26rft_id%253Dinfo%253Apmid%252F16322557%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6MTA6IjE5LzIzLzI4MTYiO3M6NDoiYXRvbSI7czoxOToiL2dnZy8yLzEwLzEyNzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-2.10.1279.14" data-doi="10.1038/nature00935"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Giaever</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chu</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ni</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Connelly</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Riles</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Functional profiling of the Saccharomyces cerevisiae genome.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">418</span>: <span class="cit-fpage">387</span>–<span class="cit-lpage">391</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DGiaever%26rft.auinit1%253DG.%26rft.volume%253D418%26rft.issue%253D6896%26rft.spage%253D387%26rft.epage%253D391%26rft.atitle%253DFunctional%2Bprofiling%2Bof%2Bthe%2BSaccharomyces%2Bcerevisiae%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature00935%26rft_id%253Dinfo%253Apmid%252F12140549%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature00935&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12140549&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F10%2F1279.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000177009700029&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-2.10.1279.15" data-doi="10.1038/nprot.2007.15"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gietz</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schiestl</span>  <span class="cit-name-given-names">R. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Large-scale high-efficiency yeast transformation using the LiAc/SS carrier DNA/PEG method.</span> <abbr class="cit-jnl-abbrev">Nat. Protoc.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">38</span>–<span class="cit-lpage">41</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BProtoc%26rft.aulast%253DGietz%26rft.auinit1%253DR.%2BD.%26rft.volume%253D2%26rft.issue%253D1%26rft.spage%253D38%26rft.epage%253D41%26rft.atitle%253DLarge-scale%2Bhigh-efficiency%2Byeast%2Btransformation%2Busing%2Bthe%2BLiAc%252FSS%2Bcarrier%2BDNA%252FPEG%2Bmethod.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnprot.2007.15%26rft_id%253Dinfo%253Apmid%252F17401336%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nprot.2007.15&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17401336&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F10%2F1279.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000253138000007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-2.10.1279.16" data-doi="10.1126/science.1150021"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hillenmeyer</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fung</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wildenhain</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pierce</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoon</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">The chemical genomic portrait of yeast: uncovering a phenotype for all genes.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">320</span>: <span class="cit-fpage">362</span>–<span class="cit-lpage">365</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DHillenmeyer%26rft.auinit1%253DM.%2BE.%26rft.volume%253D320%26rft.issue%253D5874%26rft.spage%253D362%26rft.epage%253D365%26rft.atitle%253DThe%2BChemical%2BGenomic%2BPortrait%2Bof%2BYeast%253A%2BUncovering%2Ba%2BPhenotype%2Bfor%2BAll%2BGenes%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1150021%26rft_id%253Dinfo%253Apmid%252F18420932%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjAvNTg3NC8zNjIiO3M6NDoiYXRvbSI7czoxOToiL2dnZy8yLzEwLzEyNzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-2.10.1279.17" data-doi="10.1038/415180a"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ho</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gruhler</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heilbut</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bader</span>  <span class="cit-name-given-names">G. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moore</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Systematic identification of protein complexes in <em>Saccharomyces cerevisiae</em> by mass spectrometry.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">415</span>: <span class="cit-fpage">180</span>–<span class="cit-lpage">183</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DHo%26rft.auinit1%253DY.%26rft.volume%253D415%26rft.issue%253D6868%26rft.spage%253D180%26rft.epage%253D183%26rft.atitle%253DSystematic%2Bidentification%2Bof%2Bprotein%2Bcomplexes%2Bin%2BSaccharomyces%2Bcerevisiae%2Bby%2Bmass%2Bspectrometry.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F415180a%26rft_id%253Dinfo%253Apmid%252F11805837%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/415180a&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11805837&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F10%2F1279.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000173159300044&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-2.10.1279.18" data-doi="10.1038/nbt.1534"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ho</span>  <span class="cit-name-given-names">C. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Magtanong</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barker</span>  <span class="cit-name-given-names">S. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gresham</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nishimura</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A molecular barcoded yeast ORF library enables mode-of-action analysis of bioactive compounds.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">369</span>–<span class="cit-lpage">377</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DHo%26rft.auinit1%253DC.%2BH.%26rft.volume%253D27%26rft.issue%253D4%26rft.spage%253D369%26rft.epage%253D377%26rft.atitle%253DA%2Bmolecular%2Bbarcoded%2Byeast%2BORF%2Blibrary%2Benables%2Bmode-of-action%2Banalysis%2Bof%2Bbioactive%2Bcompounds.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.1534%26rft_id%253Dinfo%253Apmid%252F19349972%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.1534&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19349972&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F10%2F1279.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000264971800023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-2" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-2.10.1279.19" data-doi="10.1101/gr.6037607"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rolfs</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bhullar</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Murthy</span>  <span class="cit-name-given-names">T. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Approaching a complete repository of sequence-verified protein-encoding clones for <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">536</span>–<span class="cit-lpage">543</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.6037607%26rft_id%253Dinfo%253Apmid%252F17322287%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE3LzQvNTM2IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMi8xMC8xMjc5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-2.10.1279.20" data-doi="10.1016/S0092-8674(00)81601-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Elledge</span>  <span class="cit-name-given-names">S. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">The DNA replication and damage checkpoint pathways induce transcription by inhibition of the Crt1 repressor.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">94</span>: <span class="cit-fpage">595</span>–<span class="cit-lpage">605</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DHuang%26rft.auinit1%253DM.%26rft.volume%253D94%26rft.issue%253D5%26rft.spage%253D595%26rft.epage%253D605%26rft.atitle%253DThe%2BDNA%2Breplication%2Band%2Bdamage%2Bcheckpoint%2Bpathways%2Binduce%2Btranscription%2Bby%2Binhibition%2Bof%2Bthe%2BCrt1%2Brepressor.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0092-8674%252800%252981601-3%26rft_id%253Dinfo%253Apmid%252F9741624%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0092-8674(00)81601-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9741624&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F10%2F1279.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000075851900007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-2.10.1279.21" data-doi="10.1016/j.cell.2012.02.064"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kaluarachchi</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Friesen</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baryshnikova</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lambert</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chong</span>  <span class="cit-name-given-names">Y. T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Exploring the yeast acetylome using functional genomics.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">149</span>: <span class="cit-fpage">936</span>–<span class="cit-lpage">948</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DKaluarachchi%2BDuffy%26rft.auinit1%253DS.%26rft.volume%253D149%26rft.issue%253D4%26rft.spage%253D936%26rft.epage%253D948%26rft.atitle%253DExploring%2Bthe%2Byeast%2Bacetylome%2Busing%2Bfunctional%2Bgenomics.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2012.02.064%26rft_id%253Dinfo%253Apmid%252F22579291%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2012.02.064&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22579291&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F10%2F1279.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000303934700024&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-2.10.1279.22"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kroll</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hyland</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hieter</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">J. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Establishing genetic interactions by a synthetic dosage lethality phenotype.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">143</span>: <span class="cit-fpage">95</span>–<span class="cit-lpage">102</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D143%26rft.spage%253D95%26rft_id%253Dinfo%253Apmid%252F8722765%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=8722765&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F10%2F1279.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-2.10.1279.23" data-doi="10.1371/journal.pgen.0010024"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">St Onge</span>  <span class="cit-name-given-names">R. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Proctor</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Flaherty</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jordan</span>  <span class="cit-name-given-names">M. I.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Genome-Wide requirements for resistance to functionally distinct DNA-damaging agents.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">e24</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DLee%26rft.auinit1%253DW.%26rft.volume%253D1%26rft.issue%253D2%26rft.spage%253De24%26rft.epage%253De24%26rft.atitle%253DGenome-wide%2Brequirements%2Bfor%2Bresistance%2Bto%2Bfunctionally%2Bdistinct%2BDNA-damaging%2Bagents.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.0010024%26rft_id%253Dinfo%253Apmid%252F16121259%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.0010024&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16121259&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F10%2F1279.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-2.10.1279.24"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Masson</span>  <span class="cit-name-given-names">J. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ramotar</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">The <em>Saccharomyces cerevisiae</em> IMP2 gene encodes a transcriptional activator that mediates protection against DNA damage caused by bleomycin and other oxidants.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">2091</span>–<span class="cit-lpage">2100</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DMasson%26rft.auinit1%253DJ.%2BY.%26rft.volume%253D16%26rft.issue%253D5%26rft.spage%253D2091%26rft.epage%253D2100%26rft.atitle%253DThe%2BSaccharomyces%2Bcerevisiae%2BIMP2%2Bgene%2Bencodes%2Ba%2Btranscriptional%2Bactivator%2Bthat%2Bmediates%2Bprotection%2Bagainst%2BDNA%2Bdamage%2Bcaused%2Bby%2Bbleomycin%2Band%2Bother%2Boxidants%26rft_id%253Dinfo%253Apmid%252F8628275%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjk6IjE2LzUvMjA5MSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzIvMTAvMTI3OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-2.10.1279.25" data-doi="10.1091/mbc.E11-05-0466"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McIsaac</span>  <span class="cit-name-given-names">R. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Silverman</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McClean</span>  <span class="cit-name-given-names">M. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gibney</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Macinskas</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Fast-acting and nearly gratuitous induction of gene expression and protein depletion in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Cell</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">4447</span>–<span class="cit-lpage">4459</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1091%252Fmbc.E11-05-0466%26rft_id%253Dinfo%253Apmid%252F21965290%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xjZWxsIjtzOjU6InJlc2lkIjtzOjEwOiIyMi8yMi80NDQ3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMi8xMC8xMjc5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-2.10.1279.26" data-doi="10.1073/pnas.0503504102"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Measday</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baetz</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guzzo</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yuen</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kwok</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Systematic yeast synthetic lethal and synthetic dosage lethal screens identify genes required for chromosome segregation.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">102</span>: <span class="cit-fpage">13956</span>–<span class="cit-lpage">13961</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0503504102%26rft_id%253Dinfo%253Apmid%252F16172405%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTAyLzM5LzEzOTU2IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMi8xMC8xMjc5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-2.10.1279.27" data-doi="10.1371/journal.pgen.1000120"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Niu</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhan</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Iyer</span>  <span class="cit-name-given-names">V. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marcotte</span>  <span class="cit-name-given-names">E. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Mechanisms of cell cycle control revealed by a systematic and quantitative overexpression screen in <em>S. cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e1000120</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DNiu%26rft.auinit1%253DW.%26rft.volume%253D4%26rft.issue%253D7%26rft.spage%253De1000120%26rft.epage%253De1000120%26rft.atitle%253DMechanisms%2Bof%2Bcell%2Bcycle%2Bcontrol%2Brevealed%2Bby%2Ba%2Bsystematic%2Band%2Bquantitative%2Boverexpression%2Bscreen%2Bin%2BS.%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000120%26rft_id%253Dinfo%253Apmid%252F18617996%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000120&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18617996&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F10%2F1279.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-2.10.1279.28" data-doi="10.1038/nmeth905"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pierce</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fung</span>  <span class="cit-name-given-names">E. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jaramillo</span>  <span class="cit-name-given-names">D. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chu</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">R. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">A unique and universal molecular barcode array.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">601</span>–<span class="cit-lpage">603</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DPierce%26rft.auinit1%253DS.%2BE.%26rft.volume%253D3%26rft.issue%253D8%26rft.spage%253D601%26rft.epage%253D603%26rft.atitle%253DA%2Bunique%2Band%2Buniversal%2Bmolecular%2Bbarcode%2Barray.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth905%26rft_id%253Dinfo%253Apmid%252F16862133%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth905&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16862133&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F10%2F1279.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000239598600009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-2.10.1279.29" data-doi="10.1038/nprot.2007.427"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pierce</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">R. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nislow</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Giaever</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Genome-wide analysis of barcoded Saccharomyces cerevisiae gene-deletion mutants in pooled cultures.</span> <abbr class="cit-jnl-abbrev">Nat. Protoc.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">2958</span>–<span class="cit-lpage">2974</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BProtoc%26rft.aulast%253DPierce%26rft.auinit1%253DS.%2BE.%26rft.volume%253D2%26rft.issue%253D11%26rft.spage%253D2958%26rft.epage%253D2974%26rft.atitle%253DGenome-wide%2Banalysis%2Bof%2Bbarcoded%2BSaccharomyces%2Bcerevisiae%2Bgene-deletion%2Bmutants%2Bin%2Bpooled%2Bcultures.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnprot.2007.427%26rft_id%253Dinfo%253Apmid%252F18007632%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nprot.2007.427&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18007632&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F10%2F1279.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000253140000035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-2.10.1279.30" data-doi="10.1186/1471-2105-3-35"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Robinson</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grigull</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mohammad</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hughes</span>  <span class="cit-name-given-names">T. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">FunSpec: a web-based cluster interpreter for yeast.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">35</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DRobinson%26rft.auinit1%253DM.%2BD.%26rft.volume%253D3%26rft.spage%253D35%26rft.epage%253D35%26rft.atitle%253DFunSpec%253A%2Ba%2Bweb-based%2Bcluster%2Binterpreter%2Bfor%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-3-35%26rft_id%253Dinfo%253Apmid%252F12431279%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-3-35&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12431279&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F10%2F1279.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-2.10.1279.31" data-doi="10.1101/gr.129213.111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sharifpoor</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Dyk</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Costanzo</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baryshnikova</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Friesen</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Functional wiring of the yeast kinome revealed by global analysis of genetic network motifs.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">791</span>–<span class="cit-lpage">801</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.129213.111%26rft_id%253Dinfo%253Apmid%252F22282571%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjIyLzQvNzkxIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMi8xMC8xMjc5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-2.10.1279.32" data-doi="10.1101/gr.093955.109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heisler</span>  <span class="cit-name-given-names">L. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mellor</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kaper</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Quantitative phenotyping via deep barcode sequencing.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">1836</span>–<span class="cit-lpage">1842</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.093955.109%26rft_id%253Dinfo%253Apmid%252F19622793%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIxOS8xMC8xODM2IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMi8xMC8xMjc5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-2.10.1279.33" data-doi="10.1093/nar/gkq368"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heisler</span>  <span class="cit-name-given-names">L. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">St Onge</span>  <span class="cit-name-given-names">R. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Farias-Hesson</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wallace</span>  <span class="cit-name-given-names">I. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Highly-multiplexed barcode sequencing: an efficient method for parallel analysis of pooled samples.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">e142</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkq368%26rft_id%253Dinfo%253Apmid%252F20460461%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiIzOC8xMy9lMTQyIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMi8xMC8xMjc5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-2.10.1279.34" data-doi="10.1016/j.molcel.2005.12.011"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sopko</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Preston</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chua</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Papp</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Mapping pathways and phenotypes by systematic gene overexpression.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">319</span>–<span class="cit-lpage">330</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DSopko%26rft.auinit1%253DR.%26rft.volume%253D21%26rft.issue%253D3%26rft.spage%253D319%26rft.epage%253D330%26rft.atitle%253DMapping%2Bpathways%2Band%2Bphenotypes%2Bby%2Bsystematic%2Bgene%2Boverexpression.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2005.12.011%26rft_id%253Dinfo%253Apmid%252F16455487%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2005.12.011&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16455487&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F10%2F1279.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000235436100003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-2.10.1279.35" data-doi="10.1038/ng1948"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">St Onge</span>  <span class="cit-name-given-names">R. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mani</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oh</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Proctor</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fung</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Systematic pathway analysis using high-resolution fitness profiling of combinatorial gene deletions.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">199</span>–<span class="cit-lpage">206</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DOnge%26rft.auinit1%253DR.%2BP.%26rft.volume%253D39%26rft.issue%253D2%26rft.spage%253D199%26rft.epage%253D206%26rft.atitle%253DSystematic%2Bpathway%2Banalysis%2Busing%2Bhigh-resolution%2Bfitness%2Bprofiling%2Bof%2Bcombinatorial%2Bgene%2Bdeletions.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1948%26rft_id%253Dinfo%253Apmid%252F17206143%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1948&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17206143&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F10%2F1279.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-2.10.1279.36" data-doi="10.1385/1-59259-958-3:171"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tong</span>  <span class="cit-name-given-names">A. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boone</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Synthetic genetic array analysis in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Methods Mol. Biol.</abbr> <span class="cit-vol">313</span>: <span class="cit-fpage">171</span>–<span class="cit-lpage">192</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2Bin%2Bmolecular%2Bbiology%2B%2528Clifton%252C%2BN.J.%2529%26rft.stitle%253DMethods%2BMol%2BBiol%26rft.aulast%253DTong%26rft.auinit1%253DA.%2BH.%26rft.volume%253D313%26rft.spage%253D171%26rft.epage%253D192%26rft.atitle%253DSynthetic%2Bgenetic%2Barray%2Banalysis%2Bin%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1385%252F1-59259-958-3%253A171%26rft_id%253Dinfo%253Apmid%252F16118434%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1385/1-59259-958-3:171&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16118434&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F10%2F1279.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-2.10.1279.37" data-doi="10.1126/science.1065810"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tong</span>  <span class="cit-name-given-names">A. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Evangelista</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parsons</span>  <span class="cit-name-given-names">A. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bader</span>  <span class="cit-name-given-names">G. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Systematic genetic analysis with ordered arrays of yeast deletion mutants.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">294</span>: <span class="cit-fpage">2364</span>–<span class="cit-lpage">2368</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DTong%26rft.auinit1%253DA.%2BH.%2BY.%26rft.volume%253D294%26rft.issue%253D5550%26rft.spage%253D2364%26rft.epage%253D2368%26rft.atitle%253DSystematic%2BGenetic%2BAnalysis%2Bwith%2BOrdered%2BArrays%2Bof%2BYeast%2BDeletion%2BMutants%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1065810%26rft_id%253Dinfo%253Apmid%252F11743205%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyOTQvNTU1MC8yMzY0IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMi8xMC8xMjc5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-2.10.1279.38" data-doi="10.1126/science.1091317"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tong</span>  <span class="cit-name-given-names">A. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lesage</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bader</span>  <span class="cit-name-given-names">G. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ding</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Global mapping of the yeast genetic interaction network.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">303</span>: <span class="cit-fpage">808</span>–<span class="cit-lpage">813</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DTong%26rft.auinit1%253DA.%2BH.%2BY.%26rft.volume%253D303%26rft.issue%253D5659%26rft.spage%253D808%26rft.epage%253D813%26rft.atitle%253DGlobal%2BMapping%2Bof%2Bthe%2BYeast%2BGenetic%2BInteraction%2BNetwork%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1091317%26rft_id%253Dinfo%253Apmid%252F14764870%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMDMvNTY1OS84MDgiO3M6NDoiYXRvbSI7czoxOToiL2dnZy8yLzEwLzEyNzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-2.10.1279.39" data-doi="10.1038/nmeth.1231"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yan</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Costanzo</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heisler</span>  <span class="cit-name-given-names">L. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paw</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kaper</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Yeast Barcoders: a chemogenomic application of a universal donor-strain collection carrying bar-code identifiers.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">719</span>–<span class="cit-lpage">725</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DYan%26rft.auinit1%253DZ.%26rft.volume%253D5%26rft.issue%253D8%26rft.spage%253D719%26rft.epage%253D725%26rft.atitle%253DYeast%2BBarcoders%253A%2Ba%2Bchemogenomic%2Bapplication%2Bof%2Ba%2Buniversal%2Bdonor-strain%2Bcollection%2Bcarrying%2Bbar-code%2Bidentifiers.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1231%26rft_id%253Dinfo%253Apmid%252F18622398%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1231&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18622398&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F10%2F1279.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000258077400016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-2.10.1279.40" data-doi="10.1073/pnas.062502299"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rothstein</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">The Dun1 checkpoint kinase phosphorylates and regulates the ribonucleotide reductase inhibitor Sml1.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">99</span>: <span class="cit-fpage">3746</span>–<span class="cit-lpage">3751</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DZhao%26rft.auinit1%253DX.%26rft.volume%253D99%26rft.issue%253D6%26rft.spage%253D3746%26rft.epage%253D3751%26rft.atitle%253DThe%2BDun1%2Bcheckpoint%2Bkinase%2Bphosphorylates%2Band%2Bregulates%2Bthe%2Bribonucleotide%2Breductase%2Binhibitor%2BSml1%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.062502299%26rft_id%253Dinfo%253Apmid%252F11904430%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiI5OS82LzM3NDYiO3M6NDoiYXRvbSI7czoxOToiL2dnZy8yLzEwLzEyNzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-2.10.1279.41" data-doi="10.1016/0092-8674(93)90321-G"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Elledge</span>  <span class="cit-name-given-names">S. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">DUN1 encodes a protein kinase that controls the DNA damage response in yeast.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">75</span>: <span class="cit-fpage">1119</span>–<span class="cit-lpage">1127</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DZhou%26rft.auinit1%253DZ.%26rft.volume%253D75%26rft.issue%253D6%26rft.spage%253D1119%26rft.epage%253D1127%26rft.atitle%253DDUN1%2Bencodes%2Ba%2Bprotein%2Bkinase%2Bthat%2Bcontrols%2Bthe%2BDNA%2Bdamage%2Bresponse%2Bin%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0092-8674%252893%252990321-G%26rft_id%253Dinfo%253Apmid%252F8261511%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0092-8674(93)90321-G&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8261511&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F10%2F1279.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1993MM89300010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-2.10.1279.42" data-doi="10.1038/81576"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klemic</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chang</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bertone</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Casamayor</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Analysis of yeast protein kinases using protein chips.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">283</span>–<span class="cit-lpage">289</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DZhu%26rft.auinit1%253DH.%26rft.volume%253D26%26rft.issue%253D3%26rft.spage%253D283%26rft.epage%253D289%26rft.atitle%253DAnalysis%2Bof%2Byeast%2Bprotein%2Bkinases%2Busing%2Bprotein%2Bchips.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F81576%26rft_id%253Dinfo%253Apmid%252F11062466%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/81576&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11062466&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F10%2F1279.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000165176500015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/2/10/1279.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/2/10/1269" title="SlWUS1; An X-linked Gene Having No Homologous Y-Linked Copy in Silene latifolia" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/2/11/1291" title="Mapping the Genetic Basis of Symbiotic Variation in Legume-Rhizobium Interactions in Medicago truncatula" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/2/10"><span>Volume 2 Issue 10, October 2012</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/2/10.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/2/10.cover-source.jpg?itok=qNC6A40M" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 2 (10)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/2/10/1279" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-INJbcWrDHl7HTnvKillrwa05BBl78c7v4mfDJtYgcbU" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/2/10/1279" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/2/10/1279" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/2/10/1279" class="active" data-icon-position="" data-hide-link-title="0">Functional Analysis With a Barcoder Yeast Gene Overexpression System</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459341" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-jVaQEiCuRIzlip7tvfbVJW91SDOCoZktIHLl_4is1H4" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/2/10/1279.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/2/10/1279" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-z7VzLgomtndZNujp4MaSCSGofAGXP4FIaSLsll7zn04" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/2/10/1279" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/2/10/1279.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459341" id="node459341--2" data-pisa="ggg;2/10/1279" data-pisa-master="ggg;2/10/1279" data-apath="/ggg/2/10/1279.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Functional Analysis With a Barcoder Yeast Gene Overexpression System</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Alison C. Douglas</span>, <span class="highwire-citation-author" data-delta="1">Andrew M. Smith</span>, <span class="highwire-citation-author" data-delta="2">Sara Sharifpoor</span>, <span class="highwire-citation-author" data-delta="3">Zhun Yan</span>, <span class="highwire-citation-author" data-delta="4">Tanja Durbic</span>, <span class="highwire-citation-author" data-delta="5">Lawrence E. Heisler</span>, <span class="highwire-citation-author" data-delta="6">Anna Y. Lee</span>, <span class="highwire-citation-author" data-delta="7">Owen Ryan</span>, <span class="highwire-citation-author" data-delta="8">Hendrikje Göttert</span>, <span class="highwire-citation-author" data-delta="9">Anu Surendra</span>, <span class="highwire-citation-author" data-delta="10">Dewald van Dyk</span>, <span class="highwire-citation-author" data-delta="11">Guri Giaever</span>, <span class="highwire-citation-author" data-delta="12">Charles Boone</span>, <span class="highwire-citation-author" data-delta="13">Corey Nislow</span> and <span class="highwire-citation-author" data-delta="14">Brenda J. Andrews</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2012 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 2 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1279-1289;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.003400 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459341--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Alison C. Douglas </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular Genetics, University of Toronto, Ontario M5S 1A8, Canada</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Banting and Best Department of Medical Research, University of Toronto, Ontario M5G 1L6, Canada</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Donnelly Centre, University of Toronto, Ontario M5S 3E1, Canada, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alison%2BC.%2BDouglas%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Douglas%20AC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlison%2BC.%2BDouglas%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Andrew M. Smith </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular Genetics, University of Toronto, Ontario M5S 1A8, Canada</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Banting and Best Department of Medical Research, University of Toronto, Ontario M5G 1L6, Canada</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Donnelly Centre, University of Toronto, Ontario M5S 3E1, Canada, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andrew%2BM.%2BSmith%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Smith%20AM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAndrew%2BM.%2BSmith%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Sara Sharifpoor </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Donnelly Centre, University of Toronto, Ontario M5S 3E1, Canada, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sara%2BSharifpoor%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sharifpoor%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASara%2BSharifpoor%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Zhun Yan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Donnelly Centre, University of Toronto, Ontario M5S 3E1, Canada, and</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Pharmaceutical Sciences, University of Toronto, Ontario M5S 3M2, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zhun%2BYan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yan%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AZhun%2BYan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Tanja Durbic </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Donnelly Centre, University of Toronto, Ontario M5S 3E1, Canada, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tanja%2BDurbic%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Durbic%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATanja%2BDurbic%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Lawrence E. Heisler </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Donnelly Centre, University of Toronto, Ontario M5S 3E1, Canada, and</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Pharmaceutical Sciences, University of Toronto, Ontario M5S 3M2, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lawrence%2BE.%2BHeisler%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Heisler%20LE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALawrence%2BE.%2BHeisler%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Anna Y. Lee </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Donnelly Centre, University of Toronto, Ontario M5S 3E1, Canada, and</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Pharmaceutical Sciences, University of Toronto, Ontario M5S 3M2, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anna%2BY.%2BLee%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lee%20AY&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAnna%2BY.%2BLee%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Owen Ryan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular Genetics, University of Toronto, Ontario M5S 1A8, Canada</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Banting and Best Department of Medical Research, University of Toronto, Ontario M5G 1L6, Canada</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Donnelly Centre, University of Toronto, Ontario M5S 3E1, Canada, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Owen%2BRyan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ryan%20O&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AOwen%2BRyan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Hendrikje Göttert </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular Genetics, University of Toronto, Ontario M5S 1A8, Canada</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Banting and Best Department of Medical Research, University of Toronto, Ontario M5G 1L6, Canada</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Donnelly Centre, University of Toronto, Ontario M5S 3E1, Canada, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hendrikje%2BG%C3%B6ttert%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=G%C3%B6ttert%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHendrikje%2BG%25C3%25B6ttert%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Anu Surendra </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Donnelly Centre, University of Toronto, Ontario M5S 3E1, Canada, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anu%2BSurendra%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Surendra%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAnu%2BSurendra%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Dewald van Dyk </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Banting and Best Department of Medical Research, University of Toronto, Ontario M5G 1L6, Canada</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Donnelly Centre, University of Toronto, Ontario M5S 3E1, Canada, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dewald%2Bvan%2BDyk%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=van%20Dyk%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADewald%2Bvan%2BDyk%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Guri Giaever </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular Genetics, University of Toronto, Ontario M5S 1A8, Canada</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Donnelly Centre, University of Toronto, Ontario M5S 3E1, Canada, and</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Pharmaceutical Sciences, University of Toronto, Ontario M5S 3M2, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Guri%2BGiaever%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Giaever%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGuri%2BGiaever%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-12"><div class="author-tooltip-name">Charles Boone </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular Genetics, University of Toronto, Ontario M5S 1A8, Canada</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Banting and Best Department of Medical Research, University of Toronto, Ontario M5G 1L6, Canada</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Donnelly Centre, University of Toronto, Ontario M5S 3E1, Canada, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Charles%2BBoone%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Boone%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ACharles%2BBoone%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:brenda.andrews@utoronto.ca" class="" data-icon-position="" data-hide-link-title="0">brenda.andrews@utoronto.ca</a>
<a href="mailto:corey.nislow@utoronto.ca" class="" data-icon-position="" data-hide-link-title="0">corey.nislow@utoronto.ca</a>
<a href="mailto:charlie.boone@utoronto.ca" class="" data-icon-position="" data-hide-link-title="0">charlie.boone@utoronto.ca</a></span></li></ul></div><div class="author-tooltip-13"><div class="author-tooltip-name">Corey Nislow </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular Genetics, University of Toronto, Ontario M5S 1A8, Canada</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Banting and Best Department of Medical Research, University of Toronto, Ontario M5G 1L6, Canada</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Donnelly Centre, University of Toronto, Ontario M5S 3E1, Canada, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Corey%2BNislow%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nislow%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ACorey%2BNislow%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:brenda.andrews@utoronto.ca" class="" data-icon-position="" data-hide-link-title="0">brenda.andrews@utoronto.ca</a>
<a href="mailto:corey.nislow@utoronto.ca" class="" data-icon-position="" data-hide-link-title="0">corey.nislow@utoronto.ca</a>
<a href="mailto:charlie.boone@utoronto.ca" class="" data-icon-position="" data-hide-link-title="0">charlie.boone@utoronto.ca</a></span></li></ul></div><div class="author-tooltip-14"><div class="author-tooltip-name">Brenda J. Andrews </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular Genetics, University of Toronto, Ontario M5S 1A8, Canada</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Banting and Best Department of Medical Research, University of Toronto, Ontario M5G 1L6, Canada</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Donnelly Centre, University of Toronto, Ontario M5S 3E1, Canada, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Brenda%2BJ.%2BAndrews%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Andrews%20BJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABrenda%2BJ.%2BAndrews%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:brenda.andrews@utoronto.ca" class="" data-icon-position="" data-hide-link-title="0">brenda.andrews@utoronto.ca</a>
<a href="mailto:corey.nislow@utoronto.ca" class="" data-icon-position="" data-hide-link-title="0">corey.nislow@utoronto.ca</a>
<a href="mailto:charlie.boone@utoronto.ca" class="" data-icon-position="" data-hide-link-title="0">charlie.boone@utoronto.ca</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/2/10/1279" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/2/10/1279&amp;title=Functional%20Analysis%20With%20a%20Barcoder%20Yeast%20Gene%20Overexpression%20System" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/2/10/1279&amp;title=Functional%20Analysis%20With%20a%20Barcoder%20Yeast%20Gene%20Overexpression%20System" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/2/10/1279&amp;title=Functional%20Analysis%20With%20a%20Barcoder%20Yeast%20Gene%20Overexpression%20System" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/2/10/1279&amp;text=Functional%20Analysis%20With%20a%20Barcoder%20Yeast%20Gene%20Overexpression%20System" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/2/10/1279&amp;title=Functional%20Analysis%20With%20a%20Barcoder%20Yeast%20Gene%20Overexpression%20System" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/2/10/1279&amp;t=Functional%20Analysis%20With%20a%20Barcoder%20Yeast%20Gene%20Overexpression%20System" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/2/10/1279&amp;title=Functional%20Analysis%20With%20a%20Barcoder%20Yeast%20Gene%20Overexpression%20System" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/2/10/1279&amp;title=Functional%20Analysis%20With%20a%20Barcoder%20Yeast%20Gene%20Overexpression%20System" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459341/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459341" id="node459341--3" data-pisa="ggg;2/10/1279" data-pisa-master="ggg;2/10/1279" data-apath="/ggg/2/10/1279.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Functional Analysis With a Barcoder Yeast Gene Overexpression System</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Alison C. Douglas</span>, <span class="highwire-citation-author" data-delta="1">Andrew M. Smith</span>, <span class="highwire-citation-author" data-delta="2">Sara Sharifpoor</span>, <span class="highwire-citation-author" data-delta="3">Zhun Yan</span>, <span class="highwire-citation-author" data-delta="4">Tanja Durbic</span>, <span class="highwire-citation-author" data-delta="5">Lawrence E. Heisler</span>, <span class="highwire-citation-author" data-delta="6">Anna Y. Lee</span>, <span class="highwire-citation-author" data-delta="7">Owen Ryan</span>, <span class="highwire-citation-author" data-delta="8">Hendrikje Göttert</span>, <span class="highwire-citation-author" data-delta="9">Anu Surendra</span>, <span class="highwire-citation-author" data-delta="10">Dewald van Dyk</span>, <span class="highwire-citation-author" data-delta="11">Guri Giaever</span>, <span class="highwire-citation-author" data-delta="12">Charles Boone</span>, <span class="highwire-citation-author" data-delta="13">Corey Nislow</span> and <span class="highwire-citation-author" data-delta="14">Brenda J. Andrews</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2012 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 2 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1279-1289;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.003400 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459341--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Alison C. Douglas </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular Genetics, University of Toronto, Ontario M5S 1A8, Canada</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Banting and Best Department of Medical Research, University of Toronto, Ontario M5G 1L6, Canada</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Donnelly Centre, University of Toronto, Ontario M5S 3E1, Canada, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alison%2BC.%2BDouglas%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Douglas%20AC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlison%2BC.%2BDouglas%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Andrew M. Smith </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular Genetics, University of Toronto, Ontario M5S 1A8, Canada</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Banting and Best Department of Medical Research, University of Toronto, Ontario M5G 1L6, Canada</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Donnelly Centre, University of Toronto, Ontario M5S 3E1, Canada, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andrew%2BM.%2BSmith%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Smith%20AM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAndrew%2BM.%2BSmith%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Sara Sharifpoor </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Donnelly Centre, University of Toronto, Ontario M5S 3E1, Canada, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sara%2BSharifpoor%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sharifpoor%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASara%2BSharifpoor%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Zhun Yan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Donnelly Centre, University of Toronto, Ontario M5S 3E1, Canada, and</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Pharmaceutical Sciences, University of Toronto, Ontario M5S 3M2, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zhun%2BYan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yan%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AZhun%2BYan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Tanja Durbic </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Donnelly Centre, University of Toronto, Ontario M5S 3E1, Canada, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tanja%2BDurbic%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Durbic%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATanja%2BDurbic%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Lawrence E. Heisler </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Donnelly Centre, University of Toronto, Ontario M5S 3E1, Canada, and</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Pharmaceutical Sciences, University of Toronto, Ontario M5S 3M2, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lawrence%2BE.%2BHeisler%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Heisler%20LE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALawrence%2BE.%2BHeisler%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Anna Y. Lee </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Donnelly Centre, University of Toronto, Ontario M5S 3E1, Canada, and</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Pharmaceutical Sciences, University of Toronto, Ontario M5S 3M2, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anna%2BY.%2BLee%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lee%20AY&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAnna%2BY.%2BLee%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Owen Ryan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular Genetics, University of Toronto, Ontario M5S 1A8, Canada</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Banting and Best Department of Medical Research, University of Toronto, Ontario M5G 1L6, Canada</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Donnelly Centre, University of Toronto, Ontario M5S 3E1, Canada, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Owen%2BRyan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ryan%20O&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AOwen%2BRyan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Hendrikje Göttert </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular Genetics, University of Toronto, Ontario M5S 1A8, Canada</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Banting and Best Department of Medical Research, University of Toronto, Ontario M5G 1L6, Canada</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Donnelly Centre, University of Toronto, Ontario M5S 3E1, Canada, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hendrikje%2BG%C3%B6ttert%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=G%C3%B6ttert%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHendrikje%2BG%25C3%25B6ttert%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Anu Surendra </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Donnelly Centre, University of Toronto, Ontario M5S 3E1, Canada, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anu%2BSurendra%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Surendra%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAnu%2BSurendra%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Dewald van Dyk </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Banting and Best Department of Medical Research, University of Toronto, Ontario M5G 1L6, Canada</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Donnelly Centre, University of Toronto, Ontario M5S 3E1, Canada, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dewald%2Bvan%2BDyk%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=van%20Dyk%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADewald%2Bvan%2BDyk%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Guri Giaever </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular Genetics, University of Toronto, Ontario M5S 1A8, Canada</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Donnelly Centre, University of Toronto, Ontario M5S 3E1, Canada, and</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Pharmaceutical Sciences, University of Toronto, Ontario M5S 3M2, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Guri%2BGiaever%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Giaever%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGuri%2BGiaever%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-12"><div class="author-tooltip-name">Charles Boone </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular Genetics, University of Toronto, Ontario M5S 1A8, Canada</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Banting and Best Department of Medical Research, University of Toronto, Ontario M5G 1L6, Canada</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Donnelly Centre, University of Toronto, Ontario M5S 3E1, Canada, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Charles%2BBoone%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Boone%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ACharles%2BBoone%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:brenda.andrews@utoronto.ca" class="" data-icon-position="" data-hide-link-title="0">brenda.andrews@utoronto.ca</a>
<a href="mailto:corey.nislow@utoronto.ca" class="" data-icon-position="" data-hide-link-title="0">corey.nislow@utoronto.ca</a>
<a href="mailto:charlie.boone@utoronto.ca" class="" data-icon-position="" data-hide-link-title="0">charlie.boone@utoronto.ca</a></span></li></ul></div><div class="author-tooltip-13"><div class="author-tooltip-name">Corey Nislow </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular Genetics, University of Toronto, Ontario M5S 1A8, Canada</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Banting and Best Department of Medical Research, University of Toronto, Ontario M5G 1L6, Canada</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Donnelly Centre, University of Toronto, Ontario M5S 3E1, Canada, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Corey%2BNislow%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nislow%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ACorey%2BNislow%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:brenda.andrews@utoronto.ca" class="" data-icon-position="" data-hide-link-title="0">brenda.andrews@utoronto.ca</a>
<a href="mailto:corey.nislow@utoronto.ca" class="" data-icon-position="" data-hide-link-title="0">corey.nislow@utoronto.ca</a>
<a href="mailto:charlie.boone@utoronto.ca" class="" data-icon-position="" data-hide-link-title="0">charlie.boone@utoronto.ca</a></span></li></ul></div><div class="author-tooltip-14"><div class="author-tooltip-name">Brenda J. Andrews </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular Genetics, University of Toronto, Ontario M5S 1A8, Canada</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Banting and Best Department of Medical Research, University of Toronto, Ontario M5G 1L6, Canada</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Donnelly Centre, University of Toronto, Ontario M5S 3E1, Canada, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Brenda%2BJ.%2BAndrews%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Andrews%20BJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABrenda%2BJ.%2BAndrews%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:brenda.andrews@utoronto.ca" class="" data-icon-position="" data-hide-link-title="0">brenda.andrews@utoronto.ca</a>
<a href="mailto:corey.nislow@utoronto.ca" class="" data-icon-position="" data-hide-link-title="0">corey.nislow@utoronto.ca</a>
<a href="mailto:charlie.boone@utoronto.ca" class="" data-icon-position="" data-hide-link-title="0">charlie.boone@utoronto.ca</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1279&amp;rft.epage=1289&amp;rft.atitle=Functional%20Analysis%20With%20a%20Barcoder%20Yeast%20Gene%20Overexpression%20System&amp;rft.volume=2&amp;rft.issue=10&amp;rft.date=2012-10-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Douglas%2C+Alison+C.&amp;rft.au=Smith%2C+Andrew+M.&amp;rft.au=Sharifpoor%2C+Sara&amp;rft.au=Yan%2C+Zhun&amp;rft.au=Durbic%2C+Tanja&amp;rft.au=Heisler%2C+Lawrence+E.&amp;rft.au=Lee%2C+Anna+Y.&amp;rft.au=Ryan%2C+Owen&amp;rft.au=G%C3%B6ttert%2C+Hendrikje&amp;rft.au=Surendra%2C+Anu&amp;rft.au=van+Dyk%2C+Dewald&amp;rft.au=Giaever%2C+Guri&amp;rft.au=Boone%2C+Charles&amp;rft.au=Nislow%2C+Corey&amp;rft.au=Andrews%2C+Brenda+J."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459341/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459341/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459341/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459341/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459341/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459341/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459341/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459341/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459341/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459341/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459341/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459341/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/2/10/1279" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-11" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results and Discussion</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/2/10/1279.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/2/10/1279.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/2/10/1279.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/2/10/1279.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
