<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/2/10/1243" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/2/10/1243.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/2/10/1243.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/2/10/1243.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/2/10.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/2/10/1243/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/2/10/1243.atom" />
<meta name="HW.pisa" content="ggg;2/10/1243" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Overlapping ETS and CRE Motifs (G/CCGGAAGTGACGTCA) Preferentially Bound by GABPα and CREB Proteins" />
<meta name="DC.Identifier" content="10.1534/g3.112.004002" />
<meta name="DC.Date" content="2012-10-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2012 Chatterjee et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Previously, we identified 8-bps long DNA sequences (8-mers) that localize in human proximal promoters and grouped them into known transcription factor binding sites (TFBS). We now examine split 8-mers consisting of two 4-mers separated by 1-bp to 30-bps (X4-N1-30-X4) to identify pairs of TFBS that localize in proximal promoters at a precise distance. These include two overlapping TFBS: the ETS⇔ETS motif (C/GCCGGAA G CGGAA) and the ETS⇔CRE motif (C/GCGGAA GTG ACGTCAC). The nucleotides in bold are part of both TFBS. Molecular modeling shows that the ETS⇔CRE motif can be bound simultaneously by both the ETS and the B-ZIP domains without protein-protein clashes. The electrophoretic mobility shift assay (EMSA) shows that the ETS protein GABPα and the B-ZIP protein CREB preferentially bind to the ETS⇔CRE motif only when the two TFBS overlap precisely. In contrast, the ETS domain of ETV5 and CREB interfere with each other for binding the ETS⇔CRE. The 11-mer (CGGAA GTG ACG), the conserved part of the ETS⇔CRE motif, occurs 226 times in the human genome and 83% are in known regulatory regions. In vivo GABPα and CREB ChIP-seq peaks identified the ETS⇔CRE as the most enriched motif occurring in promoters of genes involved in mRNA processing, cellular catabolic processes, and stress response, suggesting that a specific class of genes is regulated by this composite motif." />
<meta name="DC.Contributor" content="Raghunath Chatterjee" />
<meta name="DC.Contributor" content="Jianfei Zhao" />
<meta name="DC.Contributor" content="Ximiao He" />
<meta name="DC.Contributor" content="Andrey Shlyakhtenko" />
<meta name="DC.Contributor" content="Ishminder Mann" />
<meta name="DC.Contributor" content="Joshua J. Waterfall" />
<meta name="DC.Contributor" content="Paul Meltzer" />
<meta name="DC.Contributor" content="B. K. Sathyanarayana" />
<meta name="DC.Contributor" content="Peter C. FitzGerald" />
<meta name="DC.Contributor" content="Charles Vinson" />
<meta name="article:published_time" content="2012-10-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Overlapping ETS and CRE Motifs (G/CCGGAAGTGACGTCA) Preferentially Bound by GABPα and CREB Proteins" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Previously, we identified 8-bps long DNA sequences (8-mers) that localize in human proximal promoters and grouped them into known transcription factor binding sites (TFBS). We now examine split 8-mers consisting of two 4-mers separated by 1-bp to 30-bps (X&lt;sub&gt;4&lt;/sub&gt;-N&lt;sub&gt;1-30&lt;/sub&gt;-X&lt;sub&gt;4&lt;/sub&gt;) to identify pairs of TFBS that localize in proximal promoters at a precise distance. These include two overlapping TFBS: the ETS⇔ETS motif (&lt;sup&gt;C&lt;/sup&gt;/&lt;sub&gt;G&lt;/sub&gt;CCGGAA&lt;b&gt;G&lt;/b&gt;CGGAA) and the ETS⇔CRE motif (&lt;sup&gt;C&lt;/sup&gt;/&lt;sub&gt;G&lt;/sub&gt;CGGAA&lt;b&gt;GTG&lt;/b&gt;ACGTCAC). The nucleotides in bold are part of both TFBS. Molecular modeling shows that the ETS⇔CRE motif can be bound simultaneously by both the ETS and the B-ZIP domains without protein-protein clashes. The electrophoretic mobility shift assay (EMSA) shows that the ETS protein GABPα and the B-ZIP protein CREB preferentially bind to the ETS⇔CRE motif only when the two TFBS overlap precisely. In contrast, the ETS domain of ETV5 and CREB interfere with each other for binding the ETS⇔CRE. The 11-mer (CGGAA&lt;b&gt;GTG&lt;/b&gt;ACG), the conserved part of the ETS⇔CRE motif, occurs 226 times in the human genome and 83% are in known regulatory regions. &lt;i&gt;In vivo&lt;/i&gt; GABPα and CREB ChIP-seq peaks identified the ETS⇔CRE as the most enriched motif occurring in promoters of genes involved in mRNA processing, cellular catabolic processes, and stress response, suggesting that a specific class of genes is regulated by this composite motif.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2012/10/01" />
<meta name="citation_mjid" content="ggg;2/10/1243" />
<meta name="citation_id" content="2/10/1243" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/2/10/1243" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/2/10/1243.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/2/10/1243.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/2/10/1243.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.112.004002" />
<meta name="citation_pmid" content="23050235" />
<meta name="citation_volume" content="2" />
<meta name="citation_issue" content="10" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="1243" />
<meta name="citation_lastpage" content="1256" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Raghunath Chatterjee" />
<meta name="citation_author_institution" content="Laboratory of Metabolism, National Cancer Institute, National Institutes of Health, Bethesda, Maryland 20892" />
<meta name="citation_author" content="Jianfei Zhao" />
<meta name="citation_author_institution" content="Laboratory of Metabolism, National Cancer Institute, National Institutes of Health, Bethesda, Maryland 20892" />
<meta name="citation_author" content="Ximiao He" />
<meta name="citation_author_institution" content="Laboratory of Metabolism, National Cancer Institute, National Institutes of Health, Bethesda, Maryland 20892" />
<meta name="citation_author" content="Andrey Shlyakhtenko" />
<meta name="citation_author_institution" content="Laboratory of Metabolism, National Cancer Institute, National Institutes of Health, Bethesda, Maryland 20892" />
<meta name="citation_author" content="Ishminder Mann" />
<meta name="citation_author_institution" content="Laboratory of Metabolism, National Cancer Institute, National Institutes of Health, Bethesda, Maryland 20892" />
<meta name="citation_author" content="Joshua J. Waterfall" />
<meta name="citation_author_institution" content="Genetics Branch, National Cancer Institute, National Institutes of Health, Bethesda, Maryland 20892" />
<meta name="citation_author" content="Paul Meltzer" />
<meta name="citation_author_institution" content="Genetics Branch, National Cancer Institute, National Institutes of Health, Bethesda, Maryland 20892" />
<meta name="citation_author" content="B. K. Sathyanarayana" />
<meta name="citation_author_institution" content="Laboratory of Molecular Biology, National Cancer Institute, National Institutes of Health, Bethesda, Maryland 20892" />
<meta name="citation_author" content="Peter C. FitzGerald" />
<meta name="citation_author_institution" content="Genome Analysis Unit, National Cancer Institute, National Institutes of Health, Bethesda, Maryland 20892" />
<meta name="citation_author" content="Charles Vinson" />
<meta name="citation_author_institution" content="Laboratory of Metabolism, National Cancer Institute, National Institutes of Health, Bethesda, Maryland 20892" />
<meta name="citation_author_email" content="Vinsonc@mail.nih.gov" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=S. Ahn;citation_author=M. Olive;citation_author=S. Aggarwal;citation_author=D. Krylov;citation_author=D. D. Ginty;citation_title=A Dominant-Negative Inhibitor of CREB Reveals that It Is a General Mediator of Stimulus-Dependent Transcription of c-fos;citation_pages=967-977;citation_volume=18;citation_year=1998;citation_issue=2;citation_pmid=9447994;citation_doi=10.1128/MCB.18.2.967" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=G. Badis;citation_author=M. F. Berger;citation_author=A. A. Philippakis;citation_author=S. Talukder;citation_author=A. R. Gehrke;citation_title=Diversity and complexity in DNA recognition by transcription factors;citation_volume=324;citation_year=2009;citation_pmid=19443739;citation_doi=10.1126/science.1162327" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=A. H. Batchelor;citation_author=D. E. Piper;citation_author=F. C. de la Brousse;citation_author=S. L. McKnight;citation_author=C. Wolberger;citation_title=The Structure of GABP{alpha}/: An ETS Domain- Ankyrin Repeat Heterodimer Bound to DNA;citation_pages=1037-1041;citation_volume=279;citation_year=1998;citation_issue=5353;citation_pmid=9461436;citation_doi=10.1126/science.279.5353.1037" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=S. C. Biddie;citation_author=S. John;citation_author=P. J. Sabo;citation_author=R. E. Thurman;citation_author=T. A. Johnson;citation_title=Transcription factor AP1 potentiates chromatin accessibility and glucocorticoid receptor binding.;citation_pages=145-155;citation_volume=43;citation_year=2011;citation_issue=1;citation_pmid=21726817;citation_doi=10.1016/j.molcel.2011.06.016" />
<meta name="citation_reference" content="citation_journal_title=Genomics;citation_journal_abbrev=Genomics;citation_author=M. Bina;citation_author=P. Wyss;citation_author=W. Ren;citation_author=W. Szpankowski;citation_author=E. Thomas;citation_title=Exploring the characteristics of sequence elements in proximal promoters of human genes.;citation_pages=929-940;citation_volume=84;citation_year=2004;citation_issue=6;citation_pmid=15533710;citation_doi=10.1016/j.ygeno.2004.08.013" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=A. Bird;citation_title=The dinucleotide CG as a genomic signalling module.;citation_pages=47-53;citation_volume=409;citation_year=2011;citation_issue=1;citation_pmid=21295585;citation_doi=10.1016/j.jmb.2011.01.056" />
<meta name="citation_reference" content="citation_journal_title=European journal of immunology;citation_journal_abbrev=Eur J Immunol;citation_author=S. Boehlk;citation_author=S. Fessele;citation_author=A. Mojaat;citation_author=N. G. Miyamoto;citation_author=T. Werner;citation_title=ATF and Jun transcription factors, acting through an Ets/CRE promoter module, mediate lipopolysaccharide inducibility of the chemokine RANTES in monocytic Mono Mac 6 cells.;citation_pages=1102-1112;citation_volume=30;citation_year=2000;citation_issue=4;citation_pmid=10760799;citation_doi=10.1002/(SICI)1521-4141(200004)30:4&lt;1102::AID-IMMU1102&gt;3.0.CO;2-X" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=P. Carninci;citation_author=A. Sandelin;citation_author=B. Lenhard;citation_author=S. Katayama;citation_author=K. Shimokawa;citation_title=Genome-wide analysis of mammalian promoter architecture and evolution.;citation_pages=626-635;citation_volume=38;citation_year=2006;citation_issue=6;citation_pmid=16645617;citation_doi=10.1038/ng1789" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=S. De Val;citation_author=N. C. Chi;citation_author=S. M. Meadows;citation_author=S. Minovitsky;citation_author=J. P. Anderson;citation_title=Combinatorial regulation of endothelial gene expression by ets and forkhead transcription factors.;citation_pages=1053-1064;citation_volume=135;citation_year=2008;citation_issue=6;citation_pmid=19070576;citation_doi=10.1016/j.cell.2008.10.049" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=P. J. Farnham;citation_title=Insights from genomic profiling of transcription factors.;citation_pages=605-616;citation_volume=10;citation_year=2009;citation_issue=9;citation_pmid=19668247;citation_doi=10.1038/nrg2636" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=P. C. FitzGerald;citation_author=A. Shlyakhtenko;citation_author=A. A. Mir;citation_author=C. Vinson;citation_title=Clustering of DNA sequences in human promoters;citation_volume=14;citation_year=2004;citation_pmid=15256515;citation_doi=10.1101/gr.1953904" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=P. C. FitzGerald;citation_author=D. Sturgill;citation_author=A. Shyakhtenko;citation_author=B. Oliver;citation_author=C. Vinson;citation_title=Comparative genomics of Drosophila and human core promoters.;citation_pages=R53-R53;citation_volume=7;citation_year=2006;citation_issue=7;citation_pmid=16827941;citation_doi=10.1186/gb-2006-7-7-r53" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=M. C. Frith;citation_author=J. L. Spouge;citation_author=U. Hansen;citation_author=Z. Weng;citation_title=Statistical significance of clusters of motifs represented by position specific scoring matrices in nucleotide sequences;citation_pages=3214-3224;citation_volume=30;citation_year=2002;citation_issue=14;citation_pmid=12136103;citation_doi=10.1093/nar/gkf438" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=C. W. Garvie;citation_author=J. Hagman;citation_author=C. Wolberger;citation_title=Structural studies of Ets-1/Pax5 complex formation on DNA.;citation_pages=1267-1276;citation_volume=8;citation_year=2001;citation_issue=6;citation_pmid=11779502;citation_doi=10.1016/S1097-2765(01)00410-5" />
<meta name="citation_reference" content="citation_journal_title=Advances in cancer research;citation_journal_abbrev=Adv Cancer Res;citation_author=B. J. Graves;citation_author=J. M. Petersen;citation_title=Specificity within the ets family of transcription factors.;citation_pages=1-55;citation_volume=75;citation_year=1998;citation_pmid=9709806;citation_doi=10.1016/S0065-230X(08)60738-1" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=P. C. Hollenhorst;citation_author=M. W. Ferris;citation_author=M. A. Hull;citation_author=H. Chae;citation_author=S. Kim;citation_title=Oncogenic ETS proteins mimic activated RAS/MAPK signaling in prostate cells;citation_pages=2147-2157;citation_volume=25;citation_year=2011;citation_issue=20;citation_pmid=22012618;citation_doi=10.1101/gad.17546311" />
<meta name="citation_reference" content="citation_journal_title=Annual review of biochemistry;citation_journal_abbrev=Annu Rev Biochem;citation_author=P. C. Hollenhorst;citation_author=L. P. McIntosh;citation_author=B. J. Graves;citation_title=Genomic and biochemical insights into the specificity of ETS transcription factors.;citation_pages=437-471;citation_volume=80;citation_year=2011;citation_pmid=21548782;citation_doi=10.1146/annurev.biochem.79.081507.103945" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=S. M. Iguchi-Ariga;citation_author=W. Schaffner;citation_title=CpG methylation of the cAMP-responsive enhancer/promoter sequence TGACGTCA abolishes specific factor binding as well as transcriptional activation.;citation_pages=612-619;citation_volume=3;citation_year=1989;citation_issue=5;citation_pmid=2545524;citation_doi=10.1101/gad.3.5.612" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=H. Ji;citation_author=H. Jiang;citation_author=W. Ma;citation_author=D. S. Johnson;citation_author=R. M. Myers;citation_title=An integrated software system for analyzing ChIP-chip and ChIP-seq data.;citation_pages=1293-1300;citation_volume=26;citation_year=2008;citation_issue=11;citation_pmid=18978777;citation_doi=10.1038/nbt.1505" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=D. S. Johnson;citation_author=A. Mortazavi;citation_author=R. M. Myers;citation_author=B. Wold;citation_title=Genome-wide mapping of in vivo protein-DNA interactions;citation_volume=316;citation_year=2007;citation_pmid=17540862;citation_doi=10.1126/science.1141319" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=P. F. Johnson;citation_title=Identification of C/EBP basic region residues involved in DNA sequence recognition and half-site spacing preference.;citation_pages=6919-6930;citation_volume=13;citation_year=1993;citation_issue=11;citation_pmid=8413284;citation_doi=10.1128/MCB.13.11.6919" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=N. Kaplan;citation_author=T. R. Hughes;citation_author=J. D. Lieb;citation_author=J. Widom;citation_author=E. Segal;citation_title=Contribution of histone sequence preferences to nucleosome organization: proposed definitions and methodology.;citation_pages=140-140;citation_volume=11;citation_year=2010;citation_issue=11;citation_pmid=21118582;citation_doi=10.1186/gb-2010-11-11-140" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=A. E. Kel;citation_author=E. Gossling;citation_author=I. Reuter;citation_author=E. Cheremushkin;citation_author=O. V. Kel-Margoulis;citation_title=MATCHTM: a tool for searching transcription factor binding sites in DNA sequences;citation_pages=3576-3579;citation_volume=31;citation_year=2003;citation_issue=13;citation_pmid=12824369;citation_doi=10.1093/nar/gkg585" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=P. V. Kharchenko;citation_author=M. Y. Tolstorukov;citation_author=P. J. Park;citation_title=Design and analysis of ChIP-seq experiments for DNA-binding proteins.;citation_pages=1351-1359;citation_volume=26;citation_year=2008;citation_issue=12;citation_pmid=19029915;citation_doi=10.1038/nbt.1508" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=T. Lagrange;citation_author=A. N. Kapanidis;citation_author=H. Tang;citation_author=D. Reinberg;citation_author=R. H. Ebright;citation_title=New core promoter element in RNA polymerase II-dependent transcription: sequence-specific DNA binding by transcription factor IIB;citation_pages=34-44;citation_volume=12;citation_year=1998;citation_issue=1;citation_pmid=9420329;citation_doi=10.1101/gad.12.1.34" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=P. Machanick;citation_author=T. L. Bailey;citation_title=MEME-ChIP: motif analysis of large DNA datasets;citation_volume=27;citation_year=2011;citation_pmid=21486936;citation_doi=10.1093/bioinformatics/btr189" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=L. Marino-Ramirez;citation_author=J. L. Spouge;citation_author=G. C. Kanga;citation_author=D. Landsman;citation_title=Statistical analysis of over-represented words in human promoter sequences;citation_pages=949-958;citation_volume=32;citation_year=2004;citation_issue=3;citation_pmid=14963262;citation_doi=10.1093/nar/gkh246" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=I. Martianov;citation_author=M. A. Choukrallah;citation_author=A. Krebs;citation_author=T. Ye;citation_author=S. Legras;citation_title=Cell-specific occupancy of an extended repertoire of CREM and CREB binding loci in male germ cells.;citation_pages=530-530;citation_volume=11;citation_year=2010;citation_pmid=20920259;citation_doi=10.1186/1471-2164-11-530" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=V. Matys;citation_author=O. V. Kel-Margoulis;citation_author=E. Fricke;citation_author=I. Liebich;citation_author=S. Land;citation_title=TRANSFAC(R) and its module TRANSCompel(R): transcriptional gene regulation in eukaryotes;citation_pages=D108-D110;citation_volume=34;citation_year=2006;citation_issue=suppl_1;citation_pmid=16381825;citation_doi=10.1093/nar/gkj143" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=L. A. Mirny;citation_title=Nucleosome-mediated cooperativity between transcription factors;citation_volume=107;citation_year=2010;citation_pmid=21149679;citation_doi=10.1073/pnas.0913805107" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=Y. M. Oh;citation_author=J. K. Kim;citation_author=S. Choi;citation_author=J. Y. Yoo;citation_title=Identification of co-occurring transcription factor binding sites from DNA sequence using clustered position weight matrices;citation_pages=e38;citation_volume=40;citation_year=2011;citation_pmid=22187154" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_title=Computational analysis of core promoters in the Drosophila genome;citation_author=U. Ohler;citation_author=H. Liao;citation_author=. Niemann;citation_author=G. M. Rubin;citation_volume=3;citation_year=2002" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=M. Pachkov;citation_author=I. Erb;citation_author=N. Molina;citation_author=E. van Nimwegen;citation_title=SwissRegulon: a database of genome-wide annotations of regulatory sites;citation_volume=35;citation_year=2007;citation_pmid=17130146;citation_doi=10.1093/nar/gkl857" />
<meta name="citation_reference" content="citation_journal_title=Current opinion in structural biology;citation_journal_abbrev=Curr Opin Struct Biol;citation_author=D. Panne;citation_title=The enhanceosome.;citation_pages=236-242;citation_volume=18;citation_year=2008;citation_issue=2;citation_pmid=18206362;citation_doi=10.1016/j.sbi.2007.12.002" />
<meta name="citation_reference" content="citation_journal_title=EMBO J.;citation_author=D. Panne;citation_author=T. Maniatis;citation_author=S. C. Harrison;citation_title=Crystal structure of ATF-2/c-Jun and IRF-3 bound to the interferon-beta enhancer;citation_volume=23;citation_year=2004;citation_pmid=15510218;citation_doi=10.1038/sj.emboj.7600453" />
<meta name="citation_reference" content="citation_journal_title=J. Biol. Chem.;citation_author=D. Panne;citation_author=S. M. McWhirter;citation_author=T. Maniatis;citation_author=S. C. Harrison;citation_title=Interferon regulatory factor 3 is regulated by a dual phosphorylation-dependent switch;citation_volume=282;citation_year=2007;citation_pmid=17526488;citation_doi=10.1074/jbc.M703019200" />
<meta name="citation_reference" content="citation_journal_title=Journal of computational chemistry;citation_journal_abbrev=J Comput Chem;citation_author=E. F. Pettersen;citation_author=T. D. Goddard;citation_author=C. C. Huang;citation_author=G. S. Couch;citation_author=D. M. Greenblatt;citation_title=UCSF Chimera--a visualization system for exploratory research and analysis.;citation_pages=1605-1612;citation_volume=25;citation_year=2004;citation_issue=13;citation_pmid=15264254;citation_doi=10.1002/jcc.20084" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=K. J. Polach;citation_author=J. Widom;citation_title=A model for the cooperative binding of eukaryotic regulatory proteins to nucleosomal target sites.;citation_pages=800-812;citation_volume=258;citation_year=1996;citation_issue=5;citation_pmid=8637011;citation_doi=10.1006/jmbi.1996.0288" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=K. S. Pollard;citation_author=M. J. Hubisz;citation_author=K. R. Rosenbloom;citation_author=A. Siepel;citation_title=Detection of nonneutral substitution rates on mammalian phylogenies;citation_volume=20;citation_year=2010;citation_pmid=19858363;citation_doi=10.1101/gr.097857.109" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=E. Portales-Casamar;citation_author=S. Thongjuea;citation_author=A. T. Kwon;citation_author=D. Arenillas;citation_author=X. Zhao;citation_title=JASPAR 2010: the greatly expanded open-access database of transcription factor binding profiles;citation_volume=38;citation_year=2010;citation_pmid=19906716;citation_doi=10.1093/nar/gkp950" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=V. Rishi;citation_author=P. Bhattacharya;citation_author=R. Chatterjee;citation_author=J. Rozenberg;citation_author=J. Zhao;citation_title=CpG methylation of half-CRE sequences creates C/EBPalpha binding sites that activate some tissue-specific genes;citation_volume=107;citation_year=2010;citation_pmid=21059933;citation_doi=10.1073/pnas.1008688107" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=J. M. Rozenberg;citation_author=A. Shlyakhtenko;citation_author=K. Glass;citation_author=V. Rishi;citation_author=M. V. Myakishev;citation_title=All and only CpG containing sequences are enriched in promoters abundantly bound by RNA polymerase II in multiple tissues.;citation_pages=67-67;citation_volume=9;citation_year=2008;citation_pmid=18252004;citation_doi=10.1186/1471-2164-9-67" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=P. J. Sabo;citation_author=R. Humbert;citation_author=M. Hawrylycz;citation_author=J. C. Wallace;citation_author=M. O. Dorschner;citation_title=Genome-wide identification of DNaseI hypersensitive sites using active chromatin sequence libraries;citation_volume=101;citation_year=2004;citation_pmid=15070753;citation_doi=10.1073/pnas.0400678101" />
<meta name="citation_reference" content="citation_journal_title=Journal of Biological Chemistry;citation_journal_abbrev=J Biol Chem;citation_author=J. Sawada;citation_author=N. Simizu;citation_author=F. Suzuki;citation_author=C. Sawa;citation_author=M. Goto;citation_title=Synergistic Transcriptional Activation by hGABP and Select Members of the Activation Transcription Factor/cAMP Response Element-binding Protein Family;citation_pages=35475-35482;citation_volume=274;citation_year=1999;citation_issue=50;citation_pmid=10585419;citation_doi=10.1074/jbc.274.50.35475" />
<meta name="citation_reference" content="citation_journal_title=J. Biol. Chem.;citation_author=M. A. Schumacher;citation_author=R. H. Goodman;citation_author=R. G. Brennan;citation_title=The structure of a CREB bZIP.somatostatin CRE complex reveals the basis for selective dimerization and divalent cation-enhanced DNA binding;citation_volume=275;citation_year=2000;citation_pmid=10952992;citation_doi=10.1074/jbc.M007293200" />
<meta name="citation_reference" content="citation_journal_title=Annual review of biochemistry;citation_journal_abbrev=Annu Rev Biochem;citation_author=S. T. Smale;citation_author=J. T. Kadonaga;citation_title=The RNA polymerase II core promoter;citation_pages=449-479;citation_volume=72;citation_year=2003;citation_pmid=12651739;citation_doi=10.1146/annurev.biochem.72.121801.161520" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=M. Thomas-Chollier;citation_author=C. Herrmann;citation_author=M. Defrance;citation_author=O. Sand;citation_author=D. Thieffry;citation_title=RSAT peak-motifs: motif analysis in full-size ChIP-seq datasets;citation_pages=e31;citation_volume=40;citation_year=2011;citation_pmid=22156162" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=A. Umezawa;citation_author=H. Yamamoto;citation_author=K. Rhodes;citation_author=M. J. Klemsz;citation_author=R. A. Maki;citation_title=Methylation of an ETS site in the intron enhancer of the keratin 18 gene participates in tissue-specific repression;citation_pages=4885-4894;citation_volume=17;citation_year=1997;citation_issue=9;citation_pmid=9271368;citation_doi=10.1128/MCB.17.9.4885" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=A. Valouev;citation_author=D. S. Johnson;citation_author=A. Sundquist;citation_author=C. Medina;citation_author=E. Anton;citation_title=Genome-wide analysis of transcription factor binding sites based on ChIP-Seq data.;citation_pages=829-834;citation_volume=5;citation_year=2008;citation_issue=9;citation_pmid=19160518;citation_doi=10.1038/nmeth.1246" />
<meta name="citation_reference" content="citation_journal_title=Sub-cellular biochemistry;citation_journal_abbrev=Subcell Biochem;citation_author=C. Vinson;citation_author=R. Chatterjee;citation_author=P. Fitzgerald;citation_title=Transcription factor binding sites and other features in human and Drosophila proximal promoters.;citation_pages=205-222;citation_volume=52;citation_year=2011;citation_pmid=21557085;citation_doi=10.1007/978-90-481-9069-0_10" />
<meta name="citation_reference" content="citation_journal_title=The EMBO Journal;citation_journal_abbrev=EMBO Journal;citation_author=G. H. Wei;citation_author=G. Badis;citation_author=M. F. Berger;citation_author=T. Kivioja;citation_author=K. Palin;citation_title=Genome-wide analysis of ETS-family DNA-binding in vitro and in vivo.;citation_pages=2147-2160;citation_volume=29;citation_year=2010;citation_issue=13;citation_pmid=20517297;citation_doi=10.1038/emboj.2010.106" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=T. Whitington;citation_author=M. C. Frith;citation_author=J. Johnson;citation_author=T. L. Bailey;citation_title=Inferring transcription factor complexes from ChIP-seq data;citation_volume=39;citation_year=2011;citation_pmid=21602262;citation_doi=10.1093/nar/gkr341" />
<meta name="citation_reference" content="citation_journal_title=Trends in genetics :  TIG;citation_journal_abbrev=Trends Genet;citation_author=Z. Wunderlich;citation_author=L. A. Mirny;citation_title=Different gene regulation strategies revealed by analysis of binding motifs.;citation_pages=434-440;citation_volume=25;citation_year=2009;citation_issue=10;citation_pmid=19815308;citation_doi=10.1016/j.tig.2009.08.003" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=X. Xie;citation_author=J. Lu;citation_author=E. J. Kulbokas;citation_author=T. R. Golub;citation_author=V. Mootha;citation_title=Systematic discovery of regulatory motifs in human promoters and 3' UTRs by comparison of several mammals.;citation_pages=338-345;citation_volume=434;citation_year=2005;citation_issue=7031;citation_pmid=15735639;citation_doi=10.1038/nature03441" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=X. Zhang;citation_author=D. T. Odom;citation_author=S. H. Koo;citation_author=M. D. Conkright;citation_author=G. Canettieri;citation_title=Genome-wide analysis of cAMP-response element binding protein occupancy, phosphorylation, and target gene activation in human tissues;citation_volume=102;citation_year=2005;citation_pmid=15753290;citation_doi=10.1073/pnas.0501076102" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Overlapping ETS and CRE Motifs (G/CCGGAAGTGACGTCA) Preferentially Bound by GABPα and CREB Proteins" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/2/10.cover-source.jpg" />
<meta name="twitter:description" content="Previously, we identified 8-bps long DNA sequences (8-mers) that localize in human proximal promoters and grouped them into known transcription factor binding sites (TFBS). We now examine split 8-mers consisting of two 4-mers separated by 1-bp to 30-bps (X4-N1-30-X4) to identify pairs of TFBS that localize in proximal promoters at a precise distance. These include two overlapping TFBS: the ETS⇔ETS motif (C/GCCGGAA G CGGAA) and the ETS⇔CRE motif (C/GCGGAA GTG ACGTCAC). The nucleotides in bold are part of both TFBS. Molecular modeling shows that the ETS⇔CRE motif can be bound simultaneously by both the ETS and the B-ZIP domains without protein-protein clashes. The electrophoretic mobility shift assay (EMSA) shows that the ETS protein GABPα and the B-ZIP protein CREB preferentially bind to the ETS⇔CRE motif only when the two TFBS overlap precisely. In contrast, the ETS domain of ETV5 and CREB interfere with each other for binding the ETS⇔CRE. The 11-mer (CGGAA GTG ACG), the conserved part of the ETS⇔CRE motif, occurs 226 times in the human genome and 83% are in known regulatory regions. In vivo GABPα and CREB ChIP-seq peaks identified the ETS⇔CRE as the most enriched motif occurring in promoters of genes involved in mRNA processing, cellular catabolic processes, and stress response, suggesting that a specific class of genes is regulated by this composite motif." />
<meta name="og-title" property="og:title" content="Overlapping ETS and CRE Motifs (G/CCGGAAGTGACGTCA) Preferentially Bound by GABPα and CREB Proteins" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/2/10/1243" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Previously, we identified 8-bps long DNA sequences (8-mers) that localize in human proximal promoters and grouped them into known transcription factor binding sites (TFBS). We now examine split 8-mers consisting of two 4-mers separated by 1-bp to 30-bps (X4-N1-30-X4) to identify pairs of TFBS that localize in proximal promoters at a precise distance. These include two overlapping TFBS: the ETS⇔ETS motif (C/GCCGGAA G CGGAA) and the ETS⇔CRE motif (C/GCGGAA GTG ACGTCAC). The nucleotides in bold are part of both TFBS. Molecular modeling shows that the ETS⇔CRE motif can be bound simultaneously by both the ETS and the B-ZIP domains without protein-protein clashes. The electrophoretic mobility shift assay (EMSA) shows that the ETS protein GABPα and the B-ZIP protein CREB preferentially bind to the ETS⇔CRE motif only when the two TFBS overlap precisely. In contrast, the ETS domain of ETV5 and CREB interfere with each other for binding the ETS⇔CRE. The 11-mer (CGGAA GTG ACG), the conserved part of the ETS⇔CRE motif, occurs 226 times in the human genome and 83% are in known regulatory regions. In vivo GABPα and CREB ChIP-seq peaks identified the ETS⇔CRE as the most enriched motif occurring in promoters of genes involved in mRNA processing, cellular catabolic processes, and stress response, suggesting that a specific class of genes is regulated by this composite motif." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/2/10.cover-source.jpg" />
    <title>Overlapping ETS and CRE Motifs (G/CCGGAAGTGACGTCA) Preferentially Bound by GABPα and CREB Proteins | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"4vsuVbMVQXsH49CxB_ixj0z7o-0Ii9hKOiJDyETddtU","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459331","apath":"\/ggg\/2\/10\/1243.atom","pisa":"ggg;2\/10\/1243","ac":{"\/ggg\/2\/10\/1243.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/2\/10\/1243.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;2\/10\/1243"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;2\/10\/1243"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/2\/10\/1243":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022ac6MrmE28Z\\\\\\\/IO1pWJgCOUOLYZFCV8KBUyg9DaBAdZsApvjj9oK8BebaYORSDbDffM9yCo5UlSQMsVDv4da3tJLx+4huiEIaMBOkyqkLkp+gKV7ov1IAp5oplvQlrrAsnbhDTJZJPoIQ\\\\\\\/uxRJuZ\\\\\\\/C1olM\\\\\\\/H9sXC5\\\\\\\/4UyX\\\\\\\/lK8m5ATTk1KY9\\\\\\\/ohn6f0n4rHpt2\\\\\\\/C3mIpPnwlQSjYKEoo4+ifSIj3xSRNUkx75f1jNZyCR\\\\\\\/S0WqNy\\\\\\\/50Hwup3e1QeOVfNf2SWEHMgvmSu3BqducQAv9JuXzHu49g79O96tXkcDzRyq1sAf71KLqhG6DGML+q3fTSWd013tJRVnd6E9tcP9UHDXsBwfucLkzYlV15zY7\\\\\\\/GwaN0aV24o\\\\\\\/QnLh+JFchMoe\\\\\\\/QFZAcYGR6drKSAV52z8xJghmgsol2\\\\\\\/YG0Z6HjYWJBjYy2LWom0ZxWDgNrqvaxigz61\\\\\\\/XgzPP21w6OSxYGLL1h8LspoSSiGquTgAaE30tnZWJpFqEramObhLeSQmZD5GSAt13RkoWkUXh8\\\\\\\/41MXmw++YN5\\\\\\\/Q1vizPZ6U7vQAPK5JfrIptWY7+AIPEdopXgMSlpN7krU0rDex1L6mTp5GVlr\\\\\\\/s+Z\\\\\\\/9+pA9m81hP+5X7SNZOOcgUrGKZIbzuw+NJ+MlpbltQbYaAxTlfZDlzha2dZPcrMoCdhWCxZFkX\\\\\\\/n5TKI5xeflySvgx2X4W1TKUnZ97wNhyVYutlgDzH6PRe7V2gLzWTnFUmG9MuPcnhZnRWlOuG4TEJCF8XNvazKrulqUvPz3JdfYJgC0QAu9CNQursYVYWSaRPSynlc0bMeTPh0j61FCn5ScKlMCTXfi8xMyTvwRLOBtlZaRSpNWnRz3M4EjeidYw89C+d3F4+wd36ZEjbhRTWJgiHzEgXsSSLlmSpwltZWeBXxCpCFbGKdxV+iJ2I\\\\\\\/ejL1+c9\\\\\\\/VH0jyfVejbGhnfd4N\\\\\\\/Cl2aUCB+TfZngYuyHE3c1DRihD0DM1++b53gGrR+UnFo0xs2e5+hbj82byQ9Z3Xyu0YimibmnOLud2d2Bm0rtUzVmV61GqFn5iDd\\\\\\\/\\\\\\\/OtJDqEUblU1aR+13ponMT6lguFS1XnYi1UoKJqz1Eldf5qoNaisqm4DMvykzJKOLOlYeNtJBCpJJVt6mkBlXwLQBtmhUwdHn1OE7LHKFKdEOotJZO\\\\\\\/7OAg0s1KlRl9DdXihm1zjpERQ2qm+if3\\\\\\\/ME7KBG9Ou3Tt0\\\\\\\/7TZITU7Pyc9daI88AIzrXlAA+yFwzNcbrJ9tuz5KHSQmpsiO8Xy7svtQBAvTNtK\\\u0022,\\\u0022iv\\\u0022:\\\u0022gBDzWXbVnj8qN9LEQhOLRQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00220227cbb50ae7b466eb8605963003b9b8d830877d9a3e8b1c4c57bec9aa31d3de\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022cJ4wAaqY6EkVzAe8qEhx05gXkpYx9Lr\\\\\\\/sdKUtKN9rI0KI5LQ62PCXRRXNTaQ9\\\\\\\/067J1L\\\\\\\/oMiAVolxvgtMrh6UG4K7+JqlPUMKI+S\\\\\\\/qi2hZgbvCgN3EVl360g+ExH9Wl40eb0vaYWqex7zuSl0HHF+Pfe8a9IWvY11YZyyZsWSPzPWsdFhUi2eoMXmfd65ToMuoSfl82C+4RTDcIhxXDVNo42sRdKm+CLVIYmz\\\\\\\/ES9JX\\\\\\\/Z\\\\\\\/TgoZORjYsjyQj5XG+zLlcXYhZJWs61VRDRx0oYQgKEOnrNLcEquI6ndw5ddDJ54nDnkI3pomv\\\\\\\/uND5+V4xZyVjDcRSgLKv6T4i4iPcXMIeDx6YC7t1Qwfs2aa9wlLFAI1XGkwMuLEPVs\\\\\\\/X4Ds8rP0k\\\\\\\/uQmmmgBKCbRs3HeiT0lOVXQYUyeQ4j6GijauYuimO2U+ndA1HnmcYVEIz8SJypM3ypfYymxsgcifbUCtqY\\\\\\\/LMsGE9Lt3tTRthnKd9kVKDiYeHYjRip+x4OiC92gdsCeAK0BFbiqtjWO2Ra\\\\\\\/mo0Hc2uZZOfyKm6AdlG4kv1gP4VwI3PM80xsN0PJWrgUKgFvE71VDD4bZZIzronBU5TT16QDt1x6elCi\\\\\\\/14ayk2EsIjng0ElR1yzwNQNOJNAO5d50PmCi3quYP\\\\\\\/WrE3qcd0Qud7WNrBJcUg8rjIFMQO2a4C9LpLTecnsDNGyKvOrwK8KbNkNay4ZEAvzm4IBX5\\\\\\\/WDVGnktGnOfNg8kePnZfXeIOapa6P\\\\\\\/6JWsSJu5CAgwhB+D51wAzJewTO9qaI1jjgHY6PVIx9fNKtVQcx1Ijbd4rlt82poJEJ\\\\\\\/lhxOGGhgg3Jvus4l6TrTFmMG0xE1wQxtbtGG3bdXhiketpZYzOp1WvEWsdD6ZVWPO2KpqS0HX65KhYjsb7xDZNxilORtNdGHVdD9BdKQAEvf+qechZoFyMEDNBnOQiWfyPxbNI2rK7EE+vr1+C\\\\\\\/54wjj2WqpgEhZNWTJwFyBDU1xLlRud396TBIXLNzJFQCQYTn6rvN4ewkA+O93prUvcWX05N5i\\\\\\\/CSgw4ikzS856OJOijz2eNSHYqsdcPPyVlsvSlNWy1naHzbsbXSuXK3\\\\\\\/+CLWAJO\\\\\\\/itulJSZaAiFvSePTlmxzGGjcL3K3rXG2GbKZSzUZvDlkfBuCfWmqwLJ8UXNTDF7cWj\\\\\\\/AprNPOal9jdGmrsROLNwDNKNU8iiAgOYO38ZLLn8jQVu+eYbWQTTboSQdklH+0VC3atEVWk7ow3xkV+4e1i7ZEu6R5sEncRjaWI52LpoxYUPumNtpHvu01ocNuj9iIWGyz6hPHM\\\\\\\/UWuINh8H3AGl0e1jhVfYuB8E5EzWPF2wCUaSlHkIdh4sftaIxD9AoH3CauMZviTp356\\\\\\\/4dCv47ZxkYJ6NIuDMuKSvmQTbFkx5CxRiG4USyGasDa+kJKjroSkR719\\\\\\\/izavZMan2f3hQEoTBT8kLU0cgH8rUTUo61vWK3p5iLz1MLR3ZivmitI3SHdPMQpXs0KVaZYLmG4zlI4ZVTYNj\\\\\\\/5lyos3OW\\\\\\\/cRIkbu1crTEfu0sm8aXSfBBgFSok8a\\\\\\\/NTKiUpT4Z1mvauG9dR5zT8Cyy\\\\\\\/gSDKXS3qgJeO4s18ouueNGkjiC3C765JCRwfr8kVlfD17VXE\\\\\\\/sCFeA6jPjeQEXVsgSvllZcs9oHw690hs+dQbM2eki9piG7s1cftMCdDaD\\\\\\\/gtuyAT8fdkVMhe851UQZTI2ytOLYLLg3pz6oZSJvpqfWRKhMuFyZ2S7wP09XlL7OCgL9jt3lExVdsA\\\\\\\/2imgkrDdWVE19btvDqCVM5mzTGROzGFPj+aaDOK4TYDOPEfFF9vluDYCsLgDK5SKOqFytBSA7MZeBpMTJ18obhxWfvFG2mxePL2HOjEICh4vzOHg9JINzmzXOE7iTo1tgKYABL+4QsZobClc6tjDrc9E9LSTc2PL+M+pF\\\\\\\/FthACjc1jM\\\\\\\/8JH0sJgHIQAYBB+Zy2zNJiu6X6Cfne0KjIlMkkQdAkproifYcr72n8mPXB0vGge6GRQMouFEny0a4u4AegKoDy6PYhnN7j8dVw7y96jn4m2mQ\\\\\\\/eTpxhnG0\\\\\\\/Sec+cm638DG3GUxCZMp62EX763hOkM2e9PA09i\\\\\\\/aiNMa597Zyd9zY3tFRKsdmTyGJyGFAcomD8i2FqQgmR3GnYEKhgUWekPuiAc8FpZtIQGVu\\\\\\\/bkod3yXutQne5PIjIh4X16Sn\\\\\\\/SRavKOH174Wpm\\\\\\\/eoY9iqoAtah9lY1zPraAYLp+okcNjUjh7kNBedHG\\\\\\\/jLve9i5xK6UQqu\\\\\\\/7FkKJ4au08ETgztGcPwgNeiqXRTfhAP2YxL49zsM\\\\\\\/PtVqG\\\\\\\/IIXS0G\\\\\\\/j2QNAnI9OB\\\\\\\/Mtke9fA8A7wmY1nDHHmyoaXR1BIFpM+N3b7zx4U6oSZhmd3kgZZNUKqdFUKsyAIFZo6vl3CdTPLLMqToBPutUfb7WPOS2TyBg813FQNfGODXyDTaPcgAZ5deDGio1MweBpHBXvYInaHbEAbEgwRPsqqgREbjFppCC0K0R5zFMJcSufCDb3CBtKH7FM1qOq7g2zFO4xcquARq5+F+tbuxG3meq7NPsmsh3+WBWf34ttD7G26dirUpdFgHv3uD8\\\\\\\/yz5GMc9+3v865fdUgiHMW6NKmZkncGNDLH5RyUaMNT6q7ksYxAWR5a5wiGsFuY1JJtopsABT3yzt9I0JqyCicADpoPDspQfE\\\\\\\/eI6rbRt5Gk8zzTWDELZYtYDtE7B6fOed1VMjEjYpvpF7lCAshfSHvURKqpiKXh5ozyWPdXqaFjPFfbiGTpWqfUryMtMzRoPHXCThjrtxCONNzMpq1+QSPzw44zeLJbiZHouPEDA1y\\\\\\\/z13Ob3DHHHEfIOm\\\\\\\/l76+YS5gW\\\\\\\/j+9UmZYUXSTiYaKj6g2FjCaZ+rNMJp0ygPI+3oq1g6EE4y1GnB695BBlp67zR9m8QbC5+UHrhRJSCpRcWm0PeQ7zxEJkywtaHA6LhTCczzTJzx5Yk8V2Pvx51EZd5p+V22rWZgKUTFuLSeYdz\\\\\\\/oBKEXVrCmmTOrurDnxxJwaWmHJd1Jmf7CiVz2yqXzW6rP1VZDTv7vjXxfxMbM7a8lJ1CXhse7hqNrbox23vkdyM6\\\\\\\/oyU80gXAeBZwVNyLtOR8JwLDEcXNbsq2RbxHXsjWNd9arrOvoCsUw+88\\\\\\\/OXzExXdeh13osHQhrMTgvo8J\\\\\\\/ofFogYB48c3\\\\\\\/gBMZpLkbyPdj3lTExPnrASTmV\\\\\\\/iIA1pzCXHX4+hl5fT\\\\\\\/0Kkgww6AhbIhmOMOMzzCsC1YqEVwYFalGo4tbVVyYfJOjmiQ3+WIBcGCuy7psEiI\\\\\\\/0y8wmObH78KWSnnbXKs92ZuMb3CGaSiO+CCU0q0fVDt0jMqRLJHLiQrIVddKJ2Q4o7LeqRb1vC8xy2mTXP\\\\\\\/obWSuZcFMYNf5wUJ1yVtIYmiIR3liWAa66cq9g+YPejLBGeDGnnlsuOU5zDOZvJbpTyLV3jNVHbKjVlSjMCVb0\\\\\\\/PwHukheEpQUyFRxHYGrY6YXsXrRGCDf4LA2ZqCgb765xkdY7eo8l89N2xFFgnMQf1KrW48wcbrY7E8WjFzmFcCy3u6QjxcWcq806ae0f\\\\\\\/bwNvoZrJHGe52X+mJZf5d\\\\\\\/yKz1acitz0nk9KyEUC6Yr8pq25mYjYTMTv32B5\\\\\\\/tm35cDsklgzFW7N4E6Mn3bzkolQ1VQhiO7aP6N0BWJTIG1RQL65YmV5eXm0hpxAdnHzuB8W6dYmFClQmGmuYOQ3KrOE5jX5+BkK4PMNTF4a0VVGwC3OGu42pp1q1i+Ig5tOcI91UgRo8V+jwNia8bQwKHFEhafRmqvL6fL4DX3TBd5uToMbZeUr50pZn9Pf0s72vxk+zTScK3IEPr5G6WWtshP1h\\\\\\\/NUlsUg0OAjU54tJVumGxfc5Hh\\\\\\\/UWyGqwzbu5o1jGy6DoNrqSH8OZtpaZKLCKqeEGJSFmIT7o4RwlSkvnsO7QxaDY2WME8d8\\\\\\\/NXtfK7gOPr0VHPR59Kfl0+l5EplrECdwTJSfFXu+JMOINl2ryZ4Jt2y8Q1nT7cz7WyLHrXBaNqYnewEPScCqmTr8EANX3dSZWQzQgsQGDxjPzhuGk6EgXHIgTBPDcebmmMlen0QsWV1fDqC+PlfP3crjdaC0WeG1LTGa07UUc283fKYcZ1dEdnbQvgjTMX7k4iD\\\\\\\/eWdTRlXap\\\\\\\/q12iT9IBc\\\\\\\/zsMnfFYNKu+SzmqT890RWzCOfiVXyTO9NZ7UfINzAlE6P42\\\\\\\/ZSnBYIZSURM7wYDU9Ow7VX6FeBxKoIx720KwPCUzLIl5C5X8ztedg\\\\\\\/rvk3EV2t1XVAWEZvFY20oGdY8ZcaxWXA0i4ML1OcaTQKGTxPa5ozywFKeRoxY87M7mc\\\\\\\/3nmllrMio7z8J3yvFKYr+MjRWXjDQKel1uus2Om5t4g14+iDM2bMD1WUqoQXcRE3xQlmHr6GnzYH3jx6LbUtFK0C+VCy\\\\\\\/Hv3COw+CWQ1tC2j1J+N3RKxyUyngGEL\\\\\\\/0UgUHd2zGk4usRVyyln9A16IzJWsoaWSnCD7yqx5sCMADMJHVxreY8bBVMgvREbFJAXk9NTThG5hilPPHAATN2A3oNXSh68xdOfJBED8m3dmxPjAE3qiQn6VsC24r\\\\\\\/Xf5Wcht+2fC0NuV3Bbz7csH+zZqwC8pWVLMd1ZBsWfEMY\\\\\\\/nqIB0imNFktqts3iFDfCByiyfgHI6knb1x9qtNsd7lj0H0CTZPktefievFp9qyWdvS50KgUC658vDEMAfDi37oS0r0B9DcvwhecgBVAUW0u77wb6ZaSXE\\\\\\\/HS1AfHvXLQpdmKMCfQFYVUt8mpy7cLnH4jfOcUR6RA64lwLVbNTKncal2fxh9eWs\\\\\\\/EQvm\\\\\\\/bQiUdSRihUcBGROFccrXZ4su7bb+kW+SJAsty\\\\\\\/Ay6yf4u2MCVwTESGntQb0dG0Jsghd3dIqBo9wpC5S8ezBumPskDFkjzPr5EF\\\\\\\/qBvl\\\\\\\/tDNmajn9MKKVoG5HkGw6MKNyEYF+v\\\\\\\/0pSg4lXuO4MqtKhAZF5cCfK8GZudAlY0fRZ3GI+XfgpERMwS6JjrqijQwET47M7dmHdrWNNsEKMR\\\\\\\/9pTWVEL9RiEKii5UhGskzpGYShOOq2VehlVBXxJRQxMKfPjMueNAJxCv1qOKt9j\\\\\\\/0qXyQuN1OvkwayP6TwJqicrXT6JQ0i196QCrl0WaeXE38bNxPmKhxjEK1fVhQFYRw5F3AhiTXQRUlHlHfSmheJuWM\\\\\\\/Vq0muXvaN0maHvJnoL8VsoHjBM8ECyNyFtPPjwVFzk73J\\\\\\\/FWzYf0l\\\\\\\/IyNUpfmjBfz5aJFalPs1LbwpKmDJ4b2VUqXcPLEJQetf8dIHejcalNkN1ZPF+OQop9jrN7SrWfZ5cySOWXUfpDqEa0ul\\\\\\\/6E2F3qr992AvG01UNFZV+33khAkv+LpoMJ9\\\\\\\/Ih1yfDXDtNN\\\\\\\/KmU5RkS2KSeU26iZTdnbSclTh1yBjJDkpAecw0V+UMysmliV3K8OEReH6t0WBRcs7Df\\\\\\\/f7LG5XXSqn+4cRnJseW+1cXb8fq4ewIZiEyAU2RspSeanP2JTyntHPHTch+uk1rIlPi\\\\\\\/rrorw2hEPIMAdn\\\\\\\/usVlGm4T+nLa3FDLkwccpQLU1oIkn1ottFlQ7LC\\\\\\\/sFAkjT0l8GKc7a1DymzMkwuJ7c6+yC9PV3xfUluIB62FGZk70d73fdjLfDT40675WTtOWc7pYUr3n\\\\\\\/PYsTOfnl4W327DdZEWYczZSUKk95KQEP3gQGjW79YXMf0mN+0eclUAg6mlMbhaTxveGZV1R5WJ2AhF7A9CMVFi7bStpoXPLZPsxRpcSK+rZUmXXedbuqiKTvNWW6M8nHha+uqM7Uu9QFQ\\\\\\\/kKOZIP9ImDSzDWZB7IR6V6F9TcsroOg48s9klTGXQYe8nnEomo8oPLiC4YSDG7eeCcfSA4hgUcVPbNdtNYwE4VbLlZw8fkREgnqMBpuxyrM80tzffq5E6Y\\\\\\\/PEEjxOHkss74e+7tqY7wFdcdrN1\\\\\\\/1ruz66fQGfPzLNpVLqaEcdlFGXLqtavjuIq4TofCtxJxp7+oNIscpdZWXp1iT7cAeLjTcQ+VzTWn1ftynDeOCS0t8awcEZOl2wbZfIUcbduMqAmXx3nI+M71EsP6tMruUNVhrVEOC92kWZKZOkv0Iivvl6+2ZNJaWpxfXIWLpTqHSeTiqlfHSwV5AtTvlooZL1XZ90XeQXe4sHFZpPwjj0PFEuyot1HOJWHO\\\\\\\/Z3CNyd1p0dCWNoIbsG22HLkzvgnRsWzxjrrznHpjyXmLe3S2h\\\\\\\/N672eQJufGW7Q1KSEWEQoTPIeXkLlcraCL6J+jZewhLLt65hyQeGCwj8mcPD13hY2YTpl6nVYrek0a+eZdob1Os9mafiaa0L9tvVZBtgSpu5Dt8+chu1zii6R4MZuPng\\\\\\\/4gt5mCHOJSBHMreiaBhJk8nyBCj0PgokR+X5jdaE6iMbRAMeZzoM7Qr6ix8LPSBQf1tuPtdOBmCfIukSlyiaStajEDK8QOtYNUCteQ0UBe6GJ+5tiDGAsil2plFjeVECzkIAD5fudXZQEegcGuwyba2lCtca1CPJOMJoAyYPk3UPMvoouznD6AgAbWweyHeEchTZSmf3mR67l4iCTUJoj3BKTatsHFVqry4UXP1EA8G+kVqJSzSiHsm7PSB2Zxt1rju357VvpDD4rzKJqfGJ7CjmSVYKGvGVDaR75by5Mz7HZEcHoAjxSW4C+95YiPZV2p+u1fbud8cgBiBK9BSlG0TLIXTYf9BHvgvbWzKNpcsLY02uoDGIe9cK3XndRw6dQIMzhNFpCPx9hUckbHlU6LJs7ypxbZ4eQEgHreRrb6nhhcDlouDsPoSsJXxCM8mI2g3sL0hLLCVx9ycEaKyXbtAXveGl+SKrk3+IypovLBb+zSb85f\\\\\\\/B0SrZbW8a4G6P9guLhRNRCpIYWqWY\\\\\\\/h0JZO4ofx92yIdkBnb68o2FcR0HACJxKG5Aic18fkEEGlQ3U\\\\\\\/hn0\\\\\\\/5TppBaGfPLKwYaq\\\\\\\/Zv1Dvjlfw5D+OfrIgwvexB38s82dhKUzYlhKvExDPlDCrgELZkUW7kVSc4\\\\\\\/c58q4Qr+Gnny\\\\\\\/QmIVb0RDQBTaQMjI7iKVdqU60XWSCTeC9iaveX+iSSN2uo\\\\\\\/lJeC1Hpy9uYsI7WQf6iVRNamNUCF3MPemZldTkfzCO1timseO4sv65s5FQjU8ckXhgW53+v1XD0RghEoZKd9j543YIVxNbkn4f8AYN2vz63dtT303xsKz+OwnFbGxU\\\\\\\/Vzo5IGQwjSQD7YEUUaG8pk7iA4lvHDOExOnDLW5FvtQbcAFcN0B97uclj1YY\\\\\\\/JV5XBpMHj0E+U9ExyO+kIk3rmimDoAeMEGDiDE4awkond5kBJbUimO5K4RfNc4sr7L13qrdAPKFVBqCIUi5H7dqB+xsbfSTjlR\\\\\\\/Ds1MOCpgrwH7LxzDQFLIjWMOjAy2okwmfpjNuCcwFmtMfWCJ38tDlSyla2+81gFHheTBiRgFceCkOVuvj+pz5t7dT65+gWYqUAS3Ly37z4Vi82iwWIvj9c3QL+Uuix3lsIaGtChMBiwzIXokxAo\\\\\\\/rJ5A5snI\\\\\\\/iPax451cBK48yfdN61T2ocDxReRRCSqo9\\\\\\\/SDIc5dKUtLG\\\\\\\/q+XigZ2g4NuPNSvrVL+C2o9\\\\\\\/7Rmdu+GsQBxazh7EU7uS7DS+NOANTNMb6IwEg5s7qVafeqgVuHbFApAvdi3T1o26QUjeje32lcFA+yoUScIj2rTczqyoRsG6s0VORGSKz38xYKW3fodvyps4GLajUxpSR0L87sTkv7o3I1FhFej\\\\\\\/VHho3KhZp4T2VRkltBNIQojGH\\\\\\\/Nvc7yGI9Yw09DNl8L7RUiq3PAOZhbNCltNfYPa+XBdtzQeNc2Q5LnjW9SnAPbsGx7ZH0mTm1hZu2c1bn0U9SRBwnJJjkglH1E+v9m7T\\\\\\\/szP4oUgkiO7jukum6Z6CilbY6xa3DhPHbwj9bPapBnyndDg483lzZkEisuS020FBI+4wPzpUrCJbFA0sLowycp6u2oxOz0tnQ2VZGwib+KaC6jNlNRmGeLi0NIIX6BI3L+iGkaWqTs5NC70lSTdQR1dNI0B8iuLD9UuMNGwL1bqhQqxsm2QRTHXVI+lmIqHVRS9iJIQcroLRSqGibauRaCUFjda1\\\\\\\/XOqTvR4NDzOy\\\\\\\/eS5jOE9+aguQRdh1o94h1VAgX\\\\\\\/p5xEmdRNzz6iI58XzvLWe4nyURH1EkmcmVZEogUMhucHkykc+LFp4IEKmqy07CDHwHS1kDV3PGZ3pM31RmcMsaEAItLr2MkXgz4nW+c\\\\\\\/dnwXhVczsy3FpcL7RPqpFF7SRyDFpxVcKQPcE2BOWmit1Q27mBVCBuh6pqvJgs7RHTmoLu9Xe3aIR\\\\\\\/VNru62cV1oy9TK5QEms4da9U7wXWAH+kUAiOhWzoEEa1i4Um547mTBNiW7LqguOP8ClJQD55rZP\\\\\\\/tjO+PSBIUrSRfJYMz8ZxqIrXiEhVRQyxaIdj\\\\\\\/+PYsBeTNNTbVPMyGm+PErhlJ8PpKHwxRPBIrKxNll9J04MW\\\\\\\/FaCwsfnPhW0YrgMoSTCBMbSl3aej3\\\\\\\/UVVQf9Ca8DsDBV4zP2hl+kFl1D4yvTMBPllVwXDGDjTD6I9mJ9PwAVEkSjQZ9r2S0BL+qQ5QCdF3zP5tCr\\\\\\\/HaxueTZ5UiNKpNQne3pAVm3FXcpgk77CTERz3RxEywbwlidOfba77GKf1rudNZDMp1e5Z9a69oDWzcfxt\\\\\\\/a\\\\\\\/gkMOAsVPbRhCrJYaqcRihxHhfr2Nn6DHkGQZZZ7baRflpZGqPA\\\\\\\/6HprjPyRHkjlr9R3wdj8GW8q\\\\\\\/nplJJEZBi6SskFG1imXS2uLxSbUOpp5M\\\\\\\/TBsgN\\\\\\\/9yRP0p6Py06RwR4itfXy3N0siwJD\\\\\\\/24nBjMrH9wiCWi3m3IT7+ysvrH4ToCRZIxCQIl3+x77EV9kNK4AWg0pWDIeo9W8Lymn4BbupRaTwMPvKcmZ2K387CL5XbLspHgclfryQnl8CzZoZFSFEFT3+lrBeW7DWYjjUXNbL4Zj7C80Eqg\\\\\\\/XdAh4HwaJ05JZ0VVmGf2FCdtvKAmbxcJ\\\\\\\/5juRfqr0a4gAN5Q6wKDA98qpQEAuT1QbvM3G1jm+Q8ldlCb9WU4jgo7kwlSkDnGAwo+11+r3vJD9bV\\\\\\\/2JLaoygl8rH5hNGFxWDpNGVswO0RMcu1q9uXsu4Rjzm5XReT0W8XgLOfBBYCJk7p7et+2DrMmX\\\\\\\/KY195PdyrBVDoEP4hm3Y8JnhJ3P8KPvGlmmY+i4lxxx\\\\\\\/dbKRa8mXH3reZhu+AQ6E6bWZq1Fc4lC6UAcG44+II5NrE3Sp0tgT7NkA7emwy7czp5hR8djoZxW1ZsYr1tR\\\\\\\/YfG8T6atFL7fNxC\\\\\\\/die0NtmilLAWBiht3hh2Noq0fD2VzYg8RHRa6\\\\\\\/3ig1hbdTK5pFXba7B64lwYeMlPyKKE8zbF8FUvQo\\\\\\\/t8OOl8tsij3rCSOMDjxdKMqmjt99dPrB7qxB2jy62+vGxMt\\\\\\\/oloTEZEdWNfDVVZpSu36wEbRDPECXFUnQ108aSGx9e\\\\\\\/8Lbi3t9Iyu0My3n\\\\\\\/L0RHxGoIWW0AgLSmdt1XYGgMXuCUqMi5vf3CgDXeEqtAcAtBRrJT975LIM4jmCseQBXkruForIHGCDIGRv3ddBvufRusX4MJh3kHuEvhuhImsvqlJErOrPj0Nt8E8SoAEWCrfegB5x2b\\\\\\\/Q1NcBYxlh2mM04ujyFPGR8Ry1aRgPGLafrld\\\\\\\/3nZGDtsKDL8XZ2RsYPpt0OMsWNXvLx3J7NoRoIwCHUG9XWYlC5KIIelq2xLWRzmVFZIkpq3n6IHMLeJaYE7PO0zSHrYO7iLgOdLMJov7LrgVAxda0vzCAwg9AON6RuWTi0PiWiB3\\\\\\\/tvYzuU0GsQAhdr6\\\\\\\/8USHuZvCLHVJ5eb+hkLuDfR8qFSAcwOgo4XGnCfKYmhFLTyCnS+GYkioa3BBcEaryqvwx+6hlSEmM+C3DxQXW14ca12hCYdapnoiWTixFyFaoyfGr0Gpz\\\\\\\/jkQsKrQ4BY4O6sbYB+dagMjHswzLFS61EGhG+yFWYum2leGjk4mHhenAxNHRsf0H0\\\\\\\/f+BAmXy8R5mKESwYst3m1MNTI7QpZQOxyOtPU8nanWD7CI2c+X62\\\\\\\/ngPTV\\\\\\\/XjvSwUM\\\\\\\/Two7TFp9GFOdevVO5\\\\\\\/+1htW7tgyQ7EjuF7pmrAsWAs8zE0YR3FkpURDiXtZgc0fLBgsggcHQbV\\\\\\\/vLh7YI8QnCo\\\\\\\/TfPXWg5WT2lmsh89xoOKFV1BggcgC9DVHn0EV1oPYPn4\\\\\\\/\\\\\\\/Ik9URXUpXvCOSeMpHfp1fvipCHNpL3It0v8iW805yjhv7VJK5KrDAreSi9jjSrKqG5Rb4cUmxpdIMEqDp2JgdMuV7SZ2VP9sYZyj2XyXLqVSwVEd6VRn+wRwhXzrPx7qWnRkdRlRLlfax7oNFgOb+PrwDLgJRaMaVgCe9lm+Hjx39xizA4bRYAe+WLtIjDLX9cdqkBw5pIU+NOJPBT2JRLDz6JaFPFhwmZ2aNHjJwvg3FfiiFZvRlzLETV9nBNgJ1Gc5Sa9oOg8Q\\\\\\\/DAhqTgBwfYr0bLGSBlpWxueVHrxXHSKJT9fNbeznKjlwMC9pJvZ83BSMy3H0ggA20G4I7avl6ilKLnANYTK2h+08OpCre5qnTjlB6LlficBHcUz7ceNM7bWSc9jAcr\\\\\\\/CyqFRBNkXe5ou2Gg3Elv7YeAc7Cmq1z1vs9RwETtLrhVDldUyMynTmCY6Hy87SLemkmM57dFgJ0FnHnlBSAijXjV8JlHfXdhyFjNbJfGr\\\\\\\/XZaLrsxYgPYRzzoZVvXidy00n3Ar50QQ+eMNrEUw6UBL00\\\\\\\/BTJtFziNDz+1rSBqHeG0arH4fUra6c9JPScEMD\\\\\\\/WzfoywHxyI8u7FyAdnMw8HILXsQsZAJaw9yz56xf1rBCeTiT8C90EJXwzYC23M5QzmA4RLnFTSkiwzxHJbO994BNFn8I18I81DSZOIlDIwwHLi+7KOL0ZHjN2Najjr6XOme8jUlLP8PfFCZnF46X39uInDLoXlIQAHo\\\\\\\/Q78uezb5no4bLcswmPRAuFo0U6Mxu74NNjjoMLXZOxaBRTZOUOmjioWhoVWdcq+17jxjhTQWBXdCQBs2f0W3iRNHNy2MDIfELBf21najAuNRea44ElY+s\\\\\\\/hcYVXljwOn0jo4gtIkNRKABpW9TUojDy\\\\\\\/hd+Q3fUWtLT0s+m+3X3UyKbMSeWDxaMvx5OiLXXyIS08hRphYuxAUCuRV94q8++Pe1rMVdYCvbREmE0+Y6eqvS3fPTG0X0ZZV4ltIpGhW6t3k6nhjzNHM1ZRV0qWSxfH8xtPEIQEyYCsJnZx1mABeDTdMthKr3nMfh2bSRNmBIocdcHGFIVogDmnlp5VAQgF8OHYVgsD\\\\\\\/8hJPzthPzMflfWO3f\\\\\\\/Hwc1QSgU8M31Y8jIvwfzIcCRmyXrJ9B6j4\\\\\\\/RnA5E3QEVx003QRmb2vPKTAMMcwU1tNFGqVMY0e3YTHt9PDxWDlJ0enUbX595xqzf8qUhyL3V3gv6M1qtVCTbxo2fbzHAxCdPo1\\\\\\\/OWbpzUXgyeSmUYTEZ2Ie9S5bLr65cc35+RHdvOD5wlgP0KYP7OaDitt+UUSrslVkMBo48a7n25hJs1Z3gTqdGT25D7jTsEq74yqgnDJQhQfCwTPA4UcOiXIGCMUi2jU+GgFiJJfad2pTyAvjrljSRPMmv1mhKi\\\\\\\/0WJPXtOSf16a+HDwwCKKUvfr8X0fw5ck7th75PY\\\\\\\/7d1VOCOVhQKGWEOnTc9gizAjxhhjCZ5Ccj4OigFuPC0rbxkFSvTCGgxPFy1Q1os4qMT+\\\\\\\/IUr1n++7sevhWQCAk\\\\\\\/LBTekxvinGwx+zY5K2GqkgqZbtwKmpGwim828coS98jOdYoJlwap7BBoT3V18PWrmlljJnomvkrfYw4\\\\\\\/6ZbNmH7sQ86ykfy\\\\\\\/EyVA+v249aUrhBj0veKlz8j1R3pHOxf5tVFO6U5b0Kv0FkPet8FjkZj9omWMtbifRhFM0u5CyoTv7NHDg2NiTr6qwo9dE9tQIFrlDS3v1fWs5AnSdizABcZXBBBnEEIyo2+KFuZZyDnHLb44JHa18n8egIBZuWkXeaYfRliQqK9fj9Bl\\\\\\\/qzWykUxSfs+Q9gZy392u2W9EUplA0zY2W5W7+Yiz6GYwDnZmyPAo2EDPNqWywXff721s3qf1Ju\\\\\\\/GGLGOgLV0abYOmGnDGVwPz6AuPMGr5xFzGNT0FdQWbEZoAm5iWKCI5vpan2DphOjX3Zs9vfFCcI\\\\\\\/5lbxrh+P2Ujzob9Z3dIDVaqAx3XA387cOtJlayOSEVbMLGRbEedT+R5V7m89HfJViA0cYXInmLSp7l3bqn0TblYPxzVf7NbdiO58dC9dwEuzWJEDnOoJaaQI5e4mmtCrHV8rUMnqGvvuj7XqV6JGk589aPBivOwNbilvKsnICA7zeT61Tk04afH385SoF30S4xiX7VQ2ZrJbco\\\\\\\/AuAJ601iBCTk5TxqWoB0\\\\\\\/imADJ4IDDHa0q\\\\\\\/qx3QiQld1osJn507M6mJJ+fokqJ5kA5+9EViKbCGOO5DSL8UkWuO+mcBx7UGZUU4N9P7ShoLjX2l0O69a7SgiCsaVnR1Uy7cyBHeLmzzQrqq4G\\\\\\\/JuVyNGLe\\\\\\\/aSV79sBNgGl7JzlWGBMp0r6G0+KTekl\\\\\\\/lY5Hy2rKu9wz8I93cs9MFlw6YSZ3moOAVMj0Mt9DJ4kJbbxLfHEKM6wN5tN33qhoJdr7v8JmS7M+YiAJvesWVJU7OKPJQ6caLmbnzyMreXrgbwwwUuknwxKTMOzxYkGRZdFClJpifIVMyY9m0Iimmxh7aiQIGQD5089ZLBl6G6mH90SnREJp2kJl2XGBvuV\\\\\\\/5YpGwbacxrC8NsUMSSM4UvW3yxAqwYPmZG\\\\\\\/g0ZRPdse2HFwmkiYKqXZQ8zHT2KYa0793dAhntjxlhQHasjco0rdqVwSqSfrrKNXF6d485xWk9K+rdPqpw7beGpR\\\\\\\/EcmARvqWS3LkgL9oxLy2wjAh44IxWZu8s7cnBmPUYLqGeowXer5mEtfG9q\\\\\\\/MSf1gjl7eYvHB4\\\\\\\/MrwRT5f4xKDYUJ2Fw\\\\\\\/8ewOlore0LqnCg6tZv3sfpud5sVabByp7rK0Nyx+bPd4c7kOM1ibobehm3f7v+T1YJW1tap1CGmwQocDeXeVGHqoikcpAzXNTa\\\\\\\/5o2A80FS0VEwB6QO2PXouUykbc8fnOCnDzGKuruQvvL2PvDxzv8C2QS8MnZb25+zwLQBocenCw5uHXo8tMgRgjr44upbLeSEBwkZjbb779xMEBWr4e5inkfTMdnBbwryQ2utG3gL+DflmW+mBGFTr2I80a6OETXk+uPjvGxfrMR\\\\\\\/ysJubClQ1hP9nVanEnFFfXBUhYn15RtDxpU1yC3zz0+H7DK3byGVKI+jjESmekt6wAUDFNIQh9ZGhrtACmHK8o4ezhXYleOs0km3ftnnyl2+a+rBk9sLuhl6iJpy09vfmOByzmVJxNehakMCW9M+R791wvBfjFWtWfuq4HxvY9aVxjL0QSM94HJtV\\\\\\\/mNU5jfUuMR+bXGmCxHgmvx1ephzPzPeVYxR3iB4seriwtJdITHbaVnkDLs88uCSrzmspRg8g6qGHkj4epFg6ZMW5EmZjxmUi87SdQkgnznK35wvC26wAKRzOQd0U0KPBqoZgSpTAefXp6jtj9q5Qc5OPaEL5Jq7yxvG9IGtYOx\\\\\\\/PwWE2JbtEbBEERwplgqwuwxfy7H0cBDT2LzMMxSgDt8lXsmBq8beO5jL3eST67ggvX\\\\\\\/6Bktf1ND4t7tygx+RxqAYNmfEJGS8faZ1EKrslhvgTtVpF+rsi5itaWAcIESmWxQV8AmHcU7Sr0lMhIpYJaY2hWrzANHLwro5PQmM1ZeAPnvu8WmhjojDT95ucrvTuX2A8oFbUWvH0PQ015COWd1SinohFx89eGGEfeYHBKX0PRE365ugOJKXWZA8YSJZEJYCPy3Ikyozy\\\\\\\/399lF6gYGLu+Wyb1d2xlFdVXK3Ub+3AV9D0nik1DHbBKm4D5xBh7mtK7BA7wFDoz1A438fL1VW3\\\\\\\/pPEwN0sYpd3dUdOTzZtuZUzbtEHLR2FZG5IQgpunfXHIJf5+OJ0fAxBxAF+huY6qsYSiW5977G60mvS+rzKmmfFpC\\\\\\\/hm7qgKYUawP0LQb0WcEnWrCPD+fN9LxBaghYfzqFpio6LR4rJIK7DXKRdZ2btz2rSIYiWaqaT+r7sfJ2lhv3ZraMkPJpj3qz\\\\\\\/hSTfldbGxCRa2P7uJF\\\\\\\/\\\\\\\/EYMD1q4H0kJuVgFpWybtsnjDJ8La8vyxYt45ZTJetIQugIjqiWTzP1XG2uimmGttqWJ83Fvxts6pF2UpVegzBDXmKxW5aOlag8Qov9MkpLbpKsFf+QhKagnUSRmZwZA9vUDtHAAtasz6oCwoRiB74cHyQ66IeQ902X5X0GD0N0wTrv4RAbIyeT7oe+EONeZ1vK88SNLForCuSCUnzi8jC9DRW72y7gPFpX0PjEgt5rajYZT2alfAw2zUhjOCpSGMbynn\\\\\\\/BpG8vcshMgyha+sj2AZhCakPAyWgpbD7qQtD6NxiB\\\\\\\/67Qj6rUgp+rZEsIV3q4n4jc3wXmEOpHbvKFbJMXgs50L7xN4QBpRvkOTkUqEB\\\\\\\/Z7u7P7S7tRYdZCqB2LNUheLsgxz+Rnp9W7ecsYAfvzBoOntyIJVvLNtg6Qf7FkcNg2peIeSb\\\\\\\/cQfL8NZZdrg24YyjRTbvaKmiOpoe9ZTgNpt2tuZ0\\\\\\\/h7Te4zgj4Y38KxmsqpgBCqgNyVU7CDQvPaFOfFBk9azRqkIXqvFHe8QY33r4R720tEj82eNP6dR\\\\\\\/9RRkSayM6YVU1EiFjkVM0J93BMd6GvYPoQuM6nXxRttIeoy2nLaFDDNXaHgv17duIvrDwzGXCAxocFfguJs4EA8CVr63WL+ogdTP3P6ScPLOIbuVdj4pq9VvR8KSqB7\\\\\\\/KHb3he00ay0D8rVo7YK1mCUqIXXPsTJhwcfBl\\\\\\\/gCfi3h1rNSzao6wRJDtV3qQcSixGmD1A20FMxV6Mg5Agh9IlPYXJRUTxLVPVEd\\\\\\\/1wDDRfrpqo\\\\\\\/GLdjj7LuoKy6prVRCozMFvveo3wNHRw768nruKmZUfUGPsRSTVshpz4Kw8WolNhsp7YoCC3ZW2nPc95SpIhQyYnaVCxxelOjpZ8XIRJdGQIahrkb8cvo+nS7AOWhzEXAlKDoClpIdE1be1wIW3jylwbts5EQ7JydBkny0Bp\\\\\\\/QgKV6CnKZNx1buyMYO+b6cuXT5BFv2uemFwdgz5fm+XEShoJFos72XoXzppLA6L+A2V\\\\\\\/9f+AtyZv9ZfLBXxwiSKEUb59l3ZBRj05P+guPaijGpLN95\\\\\\\/jwxq0bWolfhd2mUFk5ejBAEKrptqc7Me2j0SYUWK1vQ7jDMC8iTZzJKdLmvwYF29tVVpW9PM8gk21rcbem8ZDPW+63vBzjTIsW78+XnHMlHwp0JDgCySiGnyKGM+gsZMd0wKQPNZs1QGnFeE7aFiOnOHA5iEZlDtS3Qg5CTs4L1h8yZ5Ggv++\\\\\\\/FjGEgS\\\\\\\/5lNmBLaWmHolCrq40sQv5qslitk2gKfezf3bSRPQgXWRaf2JuJ58JO1NRkYFbyzmygdI42e6cfaGLyEB8paa1dDqGJDCg3V1WH5D5KjsjP0L3uBu4tScFB+CAARU5HheNKGC17P9Rt4VeDvdi766BZdagcKT2BkqOqKn3xgPCa78R2k\\\\\\\/Zu8PWMudr\\\\\\\/bXVTviZwj\\\\\\\/IuP5cgm428PPIPezLotyxHul8h8DUJm20LfLw4JDLRwxrbsERVFchiaIPrPCKX9BVlDq3LyPjPxAEVfBWCjJEDjyvXX6v7jHvRaUMrWeGE4yHPOgHhKyUnnBSr\\\\\\\/7R7qprg\\\\\\\/pvZkTbk1Gv3MfiWckry9dSDdT+KVZaNtozbngi5H5IBdtUYgaOccM00+svy9X\\\\\\\/9pyEkyBkaP3nhq1smJJvEZCMtMjF+MRnWrqm3yU\\\\\\\/CAKnoE\\\\\\\/vXAQRzuIf9W1vl\\\\\\\/LMaJpv5mtTXdMUrebbmqK\\\\\\\/5yJVNkVgok8o9mywtQmrzo96N33OxVizs1wLsfOXS0r0aSZL+CFEAhDmFPPEQQS3Alt0xoRQnArIfsGOHRcVri11HciB62MdpJ18NNBzVD+eYpYaMrRG07BRzmAPmyDZ5UtQrrnws0RCuDVAw\\\\\\\/wHogMdC8z41GKn8DlKEKqZyoc180FIXBjHvbPGGKSBV1g3yQ+lkdoYfcgiAW2l20esAP0ChYRIy35HHq9kO2ZvMxl3cKCdnmBAWRbI4LGV6R\\\\\\\/pqKfcFhuXf4gS32Jc0f+3A7t65ok7vMtpTewhAk+LxW\\\\\\\/FL882xt1SoVbGhK+NGH1plWiOhm2kSm75G6KjpjgKNzfyvOgAb1PGL2NQH4D3ZFn18fShZdhh+NTLOx4dfyvx9D5\\\\\\\/la\\\\\\\/LdsdF1wWAbIl2M8fa+gLQa5Xs4X4K3gIVRGH\\\\\\\/7Yj3ASyfymNnnTkAPAUCmCDIB8iTIbt0fFpNAD1b2E+qAzi8c8vjvO5dudwURe07t2tssB2GEgDgEKJUnuA1V6x7mOpF6W8\\\\\\\/twMaNt5D7fQuY1pTkA90lSZsiPEtPbfc7ls3e5H9\\\\\\\/ADv6cqBT6rGTLUojhkEj9E5SxCDnC3wDrQxBzFrtyS0irXXoSAQY8SaYOc483wBOlCPs6OEkybuwed7YIbLhuKKqkgnzO4MHR8V17hjsbFkLzwtgOJ20ZsNrk6n7r9o20tvz2DdtcLaWKcsQwzQwYi3pYMxnAPC0tzQ4+G1oo1ba1GXmvK90Yq5g7Egq8uVy6w7SqQVAL6QE2wEqEwOn8GkRrxFLsh+QpnLEloqsE083hQsqwoffejRSdFu4sxd0QU3TFUTHbGBj+CVKeY3GtoheU4LNoyQ2qqAsORGGz1bhH6+UfVOpPPwMXlik3LRK8g7GWZ5lVRwLUd7uxmja0N8VMrwiZQCSgVEXyKfC9UJCTaSwKGgiufWs80Mu9gnF2HVuqVklUXC5P6z+PeORChnau\\\\\\\/rk6oTdSMpYjzTULTLtabOi8EjpM8YKwZ3cJogJ7QLlc36hLnOROnwDgCBz4dWCr+D0Xd9EQoMeyqGOSYhclmkqOaoqsspnJYg88mpFelGl3j\\\\\\\/GvQHWFKOyoAmUKiy5Ijo12bsyiKISsFLjbrT4C2T4Wh7oK582cbA4vxP6jz8m35oLn8TWVYOfxzYyYcdj1aT\\\\\\\/mk0w7mJgE0DOO0UVbtN1wFDuYt4Pl3SMQOvicSETZc9yrk\\\\\\\/w7SV2ygUYNQIAd+Fns6jN\\\\\\\/0CmB7MaqSra9mzLpyvMLA+LzzCHS6ps2njQr5FThHxrPglCvWjPVAXFWVtSI4rUH6XMKGSyrbpa+HV0zuCataYeA2IlhoEKYAC+G3n95U8+DsESKCsBf4gpORbsCoIfw\\\\\\\/pNAaHwHAzJA5N0k5oG8lMOinBjV7fR\\\\\\\/X2NJ46RaYnK3lXy6W6+0vwwL+Kf7QSLamPtn5t67O6tjZTnsWoykx2rqBWRQCAy8bbVRh4nrvlMuF+JOKtTcc15d2V6v0jEhCcjcDw9Ht32DrfV3eBbG9bo5+Tk5H3Rnoi14Ibfpbv26zsvVsUwFo++oTQfPy8Vavb6Mc4y1TGi4ZYJCjLXgagqt6TduZx+BAp\\\\\\\/VMDBl0JPB2KrLva2knn15F\\\\\\\/BxU57m+yIBmzGPD0pWySWNnXnOdDK+Sa\\\\\\\/itVjKF5vCySVG0xBEa03j7EFHABqM1yHPlZsjmW1f6UwhaHB7isbYmRmeGSO0Ucc5v0bGDf7NnmYDDh5IMeKTiNXjWwN09IDDxg48vcVQi0kTnOsFc87uWdmAmfLdp+eb2zgmk0ykdaAqWE4uqQu4eJMZ7f5VHOnMBomzF+8nrECq8yxE7sXN3fhIlGd2zW\\\\\\\/k+wspo4EX+e7vHgQfjaym0a8k7rVLDsh\\\\\\\/FGj+9fWPZqMlt2pp5Pm3CuS95aOSc7JgUyPylhuEE8RgM4cyNnutmQFQYaoPmVuLrg5iuczLzVuKJeellNdLReE5VFsVbmxhMPkH95vxO2G18B63YplgqSCQgrE2+2+xG4cBm0545zeJc+jbEZMTrk4VxIVe+OHXFn2EvdglGY44JL4wi5RDoEhzYGtx8D4yp46CsHISPq6j8kPcnO1ITxlcqOj2hW9bIigBFT\\\\\\\/UbY2qloSqlhqFhwOBv7M5a97DRhZSc+siS\\\\\\\/A6\\\\\\\/DPs44db+RU8Fw84qef+4ABRANagRuTYgsIBkgkNJX6eyhw4jaz4XB4yBUflef2VzKVupuMLnSZacmZNizIjm+dLR4CA0slgwmg1eq0sN\\\\\\\/DHt2DmlI8DNgrW2WAE6+rdDbdlSx0yhX2Ud4\\\\\\\/0x2KNaxP2CH7sreXzCKF+rbM\\\\\\\/Zz0hUtLXDa+Tv3znlf1wFsUnpf7GaHPgKiggjOHis8C3NO5dpaIxxEZ2gye7yDdwMA31ufnZUv3\\\\\\\/yUfpYk6VvuPz9gFH9UcwMVFfO7WaE2m4UgDrLgBpKCH4DFeFx6PjvM8840fvGf0rWotIkqYmwVTU9dwD7a60HU3RUW8GXohOZErzXKePG3ZyexCYM5B32WyLB3A17CgH72ZA1RLkpBDGKye9TejV\\\\\\\/WhviA+clJQiS4+XHvEfVtT1UrnOJIRYHa5BYOUvyxAzHeVyH\\\\\\\/5HqfdXMbaQd7OhRMBFzxTVTyxttc2YviUjtXjXC+xmvF4ZZ56BCqUsK7Dv\\\\\\\/C3KBadJjdNX4R5jSai21dbzOAN8VolLm08Qh7GfwqOe+fLy4sMPZuYNhNIXDDYqB6L421i\\\\\\\/RYFmxWDVCGkJHnfflOxjuCPWl5eVdA+dc8JbuQ\\\\\\\/BI2cIYO7Yxih1dTwcAwZELL3HwQycZpkey8PTPOYllWK9e037TxXIpjW95WhkxA7antPR\\\\\\\/mtjP8aToGAkmmR\\\\\\\/QE90mqu8i6IIfffiHl\\\\\\\/M+CLqbcuLI7\\\\\\\/n79jBMpcVQg4yPqs81HhjZucq57JKGzBH3BfwqTedBZJgXRyo9eKTjKBKHCeMq53QxxggfzONrY9hxfDUaL5IEFgQqiajspGCWOX47PYg+VCg3X5+cGmVdCex2HBdEL6YFKiZT+T2Ywt7DFu9k9Oz3xuT6Ua2JuMVcqLO9rOA5+NwUr61IqRjQIPmJylgCPYHOyGIwSjhMvms5qKm8KTgTHSFs97CLG6kFkkkTg68L4AvdFH5tNJemFlT1PL\\\\\\\/+T32FYoP2miSQz6JCY84IJgXZZmQ2AUEqw0cjK6K\\\\\\\/MZ+3Hr9Qhu3puDYqUUu77H5UiW1bnr0CfIG1gEzlzntnRp+tGTevFlnjDUsc8UHbUOkJ91TcBOiE20NJPXB48e2eB+AdMfRs32jat7lKPUpH8+qmkj4nvou\\\\\\\/GikZeNMJzF9Suw0bO22AsJbkbxuF3ZUpKmzkaJKOrAbJpkgsQz22\\\\\\\/oL7mczl2uzR2ug3AY+7qOzExnCsBqq7Tdd4zDFZZxTWXds0PE3LJ9ZmV0wQpf0z77iNwUfcd6aHnaHhkP4dotaD7q3u2fquaMDJBJOY7pvmm0R0oxYNCkTpn8tlzZ24XKHitKErKcZIu+N0LfLbBbtQPPGpbOxmhVZmQZHuDLrXjpBVv2y0AmW2xa9pv+lizaaSU7u2gC7Hm+Gztz9wYDH22XCXG83m+cnPhfVgQ9n6k5ggrlUTWcLqD95D0sb8tY45N1Ex6FY5W3qp0Z+NPy94PI4TXm3feDZFoDq714cRkKo5yDgr4RRuMzzbVTxWPobJNFChlLML4E9Xu5YT43bhMAfTc55CXbdZo6uKxTRYUC62ZkXweEHKALF4OLmQn\\\\\\\/ESozOHZdALujTrnLEh7E90eXUw05dik\\\\\\\/BWwD6lmJI+tEtPO0rDC7mCvPAxjevZ3zHiPywtC9smtjrWA2w\\\\\\\/GT97GGK3A13F5+v3aud1pDRMgFTLTqVTMv1iMgIRL8GjQY2V4HPkDlwj6JSlgyUkJG5qyWTT\\\\\\\/NCGzsHV4roJfrzCCS1zY37n9WebMFRDx\\\\\\\/wd\\\\\\\/CiCPnJhU8rJR5fg6vw25W+5Y\\\\\\\/zA4zNGFFLfMEnQ2Q7p5MsLD3h40s8whVAm\\\\\\\/1eDlop0Nxpu0SMbsVYJ9ZyA\\\\\\\/caO12Loybfmx7T164OQ4Pw7MXpGP732iBcgeODu7CJZMHzA4K1R+vb\\\\\\\/fvP+IGwO+z+lf8SFPzoU5vI22c+MC606ZX3qPq2a+usrDtPFLLvq+l2P+0+xpKD1edTU8C6\\\\\\\/3zUbDdquCjlrYHuJWz1vqHLkChq9AbZWDSpf4qM4v2it3EItoFKgeaGaVDouvkiw1zYCScyGRho0hHqeNb8mdTY2MYmQpeNu1qyCqryl6qR83k2HHMARwPwdcokl3YI42Q+ZmA5axVJXNNu40FVIdukgTDqfEk3jeWHtg0SqnB1fv3LOwJ\\\\\\\/1yIJ2A1gjgHj+YlxxIG3RLvlvuJK2VchTIW7HrQhLa79o8aQNQsadIOIFnGdPWPjzzEwis6ibr6ie+RFNhTEEm4xDwY5gzdheELyI6IqT5sXDVv+PSX\\\\\\\/s75\\\\\\\/DMBgYpdQ7q5e10mZ2lj8PrfIH9U7oipInRgxh4lNPbR13GZpwjpAQPjTfnTJZwpc+MAo\\\\\\\/cqTjuc69+1OOpFnTpHUlWBv\\\\\\\/DO\\\\\\\/hOeqQ\\\\\\\/SeJGXhmNOsKIpZxmEnw\\\\\\\/uE43UGFjlvIg2fT8wBRkbwDaojsm4TjbFK9CQKzz7srD+NUk0GlzxFj6wd8mUaQyyPpux+y2Pd9\\\\\\\/uGB31PFy5lTJzrXOk5HGAjyT4CS690jI0cKySm6D4UQVa+8mluQ9rhufgXDQOD1wH5eCYtA4Z4mRbYQgvqK5SRzkJst60avEhIjTOmZyFp+HF\\\\\\\/wfFLCp56wEDa37Enr9DLF6J3oq1QyOnAsLLZK2wXvXn9fqubkzYxsrPj31bsb1agrsN9DtwlUZ6RR05aggfiFPPrS0Gq4+wyh0n5CawK5qsPLfUbo0nxu5bKZJSvA4lAUFV1jfItbnpcDi7bXGbOW00uWX4hmbuzzas0f0+rd+UElGhW7HBXK1IBsHhXwHK\\\\\\\/ZJ7GpmPgj+kElBXU6PuCgagHzHkvjxw2Ec5cayacDnqspuw6fD6macoBNy4bPnPvNKFPtSbHm4QbhSX\\\\\\\/QyCSS6PVXTdxLzI8cWe44iPEdteDjfkmYNIZFu3Dh6BPDdYdqDWIlfn3W02qR+JjoTzU9untLTqODSIK+4yUdUpPPVI2Jn3aRH2I1CA3L90AhA4EIpL4ZQQUm\\\\\\\/Zu9nB+1joLnfa0NN2JELe8BkXzElrj99h\\\\\\\/UDh3fQ\\\\\\\/6bGqO3rACoTzP9Y5448V3P4pcgOY8i\\\\\\\/zPgqmgeK2FZc1hSMtn5wxxCkm\\\\\\\/Je3cXNWpv2yl+vAg9n6oRh8Zm3A0xN14ScgphqPRTWXzHXdhUfrjlRBmAzN\\\\\\\/9GwjexieUuExQAS9q3FgZoPr5UpRuh7\\\\\\\/pM00qSXF20GR5JgeO\\\\\\\/O+lqCoIX\\\\\\\/boNBQjUBoYWbHHDs3hOnUqbsBVXBJF4CZEKOHUfAsKAeV6gTM24YzjR3H8A3WefA\\\\\\\/k+3Ylfx5VLChLKW60x0DnBmdleH67BL3\\\\\\\/HYzuYeclNNZ+Xkow6kpOmJapEW7o1MhyJVgf27sPqo+\\\\\\\/NwNHFBoyQ6aME6pxIt9YeXYgiZHfjmCiNH3SME6VinvTnjPLAvJXodoSXByhfsEOt2EgzR7Cok5997Y\\\\\\\/8I8g0qPcvo7evLiNpMwTMOcbjx13AkzPkepgTD\\\\\\\/F+j0p7\\\\\\\/nFF6oFlC4+3yG6iMns9yTcVLxRVO0WU5ykA44RhzQyn5qqOrAB0Dy+pmKho2PbKwdT5RerPYW6yKlKoXc6A\\\\\\\/5vEbRVYdp6s1sLEZaizVpJ+X\\\\\\\/1DZnZeNwbRkPR9gC0fpkA6\\\\\\\/CJREhEBugHqaVHlVdK9IGfsfZJikAbmvfQdKTaflVZ8YVhEPxU37dXobhWG+2Q5HgaMvqxCtHfDCKzhPFuk6ExrKB7ynRqFUb4G8yVxFhT0pG7YNt4ODnxKHx18p5y4lFnUd79LAgbL1mX0cDB4TTFzn1LukoMEyK77TrP5uE9sDL8RjuYqg8NPAekm\\\\\\\/uJwMZhiFglfBITf\\\\\\\/JPbI3mi6e9U5THOan1Qs1QfhYfcI+KyeAHGrnoHu60E7aSXrpY8VlEQH0hwfYEkUiUK0V5zbOZcB8gMZj9vLmH3\\\\\\\/iCcnJTyeQCaUWKYQOLPBTiamcSw8cEQD5TEFVyethAo41bTzN806aXY9vAQQS6razsC7ufY0LVDT6b+Pw9QpupsaRrGp\\\\\\\/Q6pxQ0tuHSNI3DUgfDWplm89X1UAJfQtUcDh+ZTgNN\\\\\\\/fDcOqb1szLOvF+pRNk4CtAXtjeRM0p\\\\\\\/GaqY95c56YB2WHxQDjTe7p\\\\\\\/K8DsPjR8XaFRzTNeqmKCW91P9fV4QekKnhAvre0CzBtv6nAwywAEvAaWpP\\\\\\\/RkOoDdQVKxChUCyA2xLPWrmSYULdF4j+Tg9YcZmSmnEvKn9zDijtAGo0l2jiC8MdybarQDZpni1C8YSyhMYMmfdEJ9iS8bd8hAOsqo1Qx++LP+4Yzx17JSs+tbAcNfRyjIdQDjQg4YhsuT2bFDI33fHUJbbC\\\\\\\/yETBRbq\\\\\\\/BRScBMDBRByxkLhRsUaNmj2Rn2MpHt43YnXJbyftwF7XZW49YVKnNoKbymHhCAc7VkM2n4ga7zMcfsGo7EyKKX\\\\\\\/xtCom4h8qGylAV9zcEy4xX2nheohNZkpV7mYrG7vs45tvnt3HM8iV7AsTYaBYoDeIUIR63otiY+T6r4PWVk1smbMkCaoluxL\\\\\\\/7T72r2dCALuaPsvE4t881fGXaDjs7jAaUfCvSNN9O3VdaNhBe5qt8JgkJxsWXFtEPWUtXFT\\\\\\\/cuskBLFTUDYyNXdEHZxatWD8DYyP5EVkwYfrHovd0ObY2PKhbC5ADpkbEVHaEhjxlUxYPJQm3U6566SkDTy9EIXuKVIdmBjCaCQuDmhZlgspoMChm8ENrWnnCBh37R7oiiW3PjupkZSMqTY2FxlTYqRw\\\\\\\/Gff3U+Zrq2SPYmZJ5TLes5YaZ05biUhibZC0EPbKqH9Xvh9B5Rnhr4brNWxByEeOjNbV7iqj2agM3tK\\\\\\\/hotXKBSS+QPt6twl8Fg2y7jQwA3Q6Ff\\\\\\\/+O4ZEpV\\\\\\\/osv+iUz0+DRlqHWbt81GY2DAtgPpuPdsqsLopxm87rXfj3wf1scVr9vmwpi05xu0h4mubYId0f+ZUpZ8cExZRDUKLLX3UIodz9YPNabZ7fpB0DRYObkflzGfh5tgFRjpw5d7FGI+oePf1X1azuFbRWF3ns3t5JLRQIRSFKZnCnceMBCktRigBNltC2GQvAP1guWVIbXuDyWb2Na0\\\\\\\/ELz9Hvp2GOjQY8O3KKjDYXGaulJlxYXVRq5QV06b4ek7Nt9mA7hEvlO0OFDB5sKA2nKoObyWOt4c26aSb5HV9eOUqmdUuQrPWZdEQYSsU0C5yxe8EzCyoumSYDEUbZWH\\\\\\\/U9N\\\\\\\/wn6dQ2hYWgVHU\\\\\\\/MinrmMNiOC4WnP4pHYaWJ1\\\\\\\/Qe13vYTeVGLXllikENhnZWPLr3zWYEWUr+OoNeFKG2slrZ\\\\\\\/bVEV4q0VZStn3tVZIPFIZHU7Bk4a4d6g1kP5GeEEOfkWykfA9qzDRc3uYHzK8dLflUW\\\\\\\/pFqKeftosAQpi70+dE9gstG\\\\\\\/ZhcTYf5uP9zfrAe5yImF0zbwkD1xt6I7tmmD29Pq4WMOd8q8Q66YJ17ftbZ3f3DCUqwrJdbtrD1dT1s6P6eqyWWKqXMmSTwR91i8F59Ej2RFRx7eY5BWpncTxZ8YQwgM4A1Mrpl9t7C23L+enkgLz+NVSH01CmgraunZiIPiXtnfQT++enI8BEOYzbTaDRDvvv75jMojDxpRN1Mqvs4AILpm415Yv3R2+p0iiEZIVOX2y1a4zvaBueBl69KbIIYqTLZrwiszrscQQj7FMLoVb88lyT7MyhauwtUBRu8R+V7oq8ENC71tqSWWIgVYLwU\\\\\\\/oEXRvH1W263e6GuvoN+wn5c7x50Jm7k2YrXQV9OaxwDm6auRbWBMKpGOMB59Q8b17YigfXCu1rrqapGVXxOw+lFjMZVnc8k3tCLS5ptD3Cec1ya0SG+yG6HXBbK8l0l9ifOGAF6Yo4ag58J\\\\\\\/4R1kPastAebSF7CPkl7z1y\\\\\\\/pHwPhnMg2vJhSStWBv0hkro+nSY\\\\\\\/e7FXWPW6qB2bXMyhxC+47g6Uz8FPp05vD8fZjKKpz07Zsl2mSrSxmkNcKVeRPbT0iL0jWGFrRDNBM1kpJQvNxBJ\\\\\\\/vbOjzg1ptURhhFcytEGnVLvBS6OrMq5FAnQOkrJY0qzryxDGWlvlGbF2IYhhQhXwaoHe4s1lREDzjRdHP5wnYr\\\\\\\/Gevt0QXxl7A5mC\\\\\\\/OxXYzsuBEvS3RjR3+KlyLpJUd9M0\\\\\\\/zEuVpE\\\\\\\/5DPjpJVJs942EBjDzi9JbcUtBssqJnlIjJaClc9iIUkCTY8XZXoUkYY01xJyyFaNiiZFJCLqXkcHdo+qEZLPD3wJ5MThw51t\\\\\\\/86BWX0MrA9cQAGGW9cj6uiy0uG0VfrdqG2nCOrqjAdykBH0bU20KJplgea6FdHrHYWaLAdCVIsPE3yxLg1Jy6\\\\\\\/ci2zN8MG4lNwYnuuM+5eMknerSBk+mn9CznIDFmgGuJU5LK5SeOJmBr5vLpJtOcuZ61\\\\\\\/MBmbXomTMG8cxkrGkU3MHSDbvK\\\\\\\/wnphlwLGQdr6Cbm1tZYHvUpNklaPSF4A4TCT3eSdifoFivYWnUUdVDJQCqNWY3xemTwgablYxLtIzgrCR5BNAyDAG6Tqw6Iy0tVlDo\\\\\\\/YvWGgVwJ64dKfDzkyrLnII6vFVBhTLVmn7CCvAZHewyCqA6ZyUm8blVKi7OSiujkU10JFUN97Mz9Ttwx7u04CVLq3Z+AtDBPXAG+DlagQCplceCUyn7FOwBPwMYP4ZpGbX43g85KVH\\\\\\\/FyMjrV\\\\\\\/7UmdEnSq74bclzEcowt65VFh3NNdERECmZevRgzWgelSEwqZpU4e7XiSZGWZgg+gMoPUarJFwMS97RkeyUWPeUAfYz+hTOEiXOeDtE74IAUGaUhMd+FPeTGUjJbIKAqoSzwuR2dnPLi1FWB4crtNZtQ6uFc0\\\\\\\/74ggV9jUuK+BSRdjkWmuH7W4YG1g6STa237O1ovxpIMpww7uTIO1UVt2havKxC\\\\\\\/Ov3e304JX\\\\\\\/cE4A8KoF8gaKdJih+J13dD4EUU9wXLaIulW+fKS0CozUtyCkt8MegYqbAu6hZEhAheYX1IEwZaqfFAKwkm1LG+ZFP64nnWiAyPDQchYbU8b0nALNMirZOa2mHxPExHFEnEs7mNY1Pvng\\\\\\\/+Yump8g1j1kRzMHweCagd71v\\\\\\\/Z2wEwYYv5Pp7S0MYKhOBkEgSaLEKVBQY2se0mJ22iwv6\\\\\\\/phGoIF5yBTvSQt9O26v9qc6hc6sTPzVkmZzrOJzhlSAP8DZEsVGVKJQpUpr6qe0FD7kld7eamI61vmEm+iUINKgcjQVSRKGtAyrDZlsZs6\\\\\\\/EbfJogZZe5kVf8RpPB8vbxRru95SUq6glYIqo2\\\\\\\/WfwwiEDJOh3lZVM5egk\\\\\\\/my0NBVMPruql4fldVz932oziksdUDU6ADEyUM\\\\\\\/28vAJnZOllVbN0mvmdQ3bPJ0J8+jSdU+iQRf7RYSs2H\\\\\\\/6FYv+er58Xev2HtE+FXljyMZ1jGAfRDX3VNp3JBDXga7sh0YqRtvKJNTaoNy5LJTB85\\\\\\\/t2GK1vWZOlID\\\\\\\/kLyo0qcoUACo0haXCskHEA7JZhJNwy9O4zVwLJ9V2eNgiG7AQLCjXBboeWGGOGg6iB8BTgdTOlBUlbhkow60DAfQGCQ2qKM6\\\\\\\/Z7PwImcpIcPjXxC7\\\\\\\/kWqTg\\\\\\\/q6c99q46G1eCqM\\\\\\\/QrZrlZxTwBlH79z5cz3gq1LxD9IbssQ8MWhHz912illF6zNclan6FhEjVEShb0oBpniB0l4eseKJ\\\\\\\/Q082DqcUBY4PBHFrOpZZg7e+SJydkdZ0+J0x9D9lUk9O15uuL9YgMzYAj37ElZhPYuKv2AslfVDGjZnIgKWh5PBvAZxlM7xFy0f8d76triakZM6kpwS5M0\\\\\\\/VVRHZdLxkoRZahWvqbrCWZY7vuI8aTPwYSF+0jn9nslbl8mDiSU3fCQCzbutuxb\\\\\\\/ztjD3hWzup6VX1Q4x4qmF9sAH6pvgnbb5l1rouErsyv3bG0VgSpqmd7CM3ZWgkZd3ya3R57lfKwySD6+HNko9DDF0OqY74Rd6QhUbwxUP0UQ8amFp8Ib2X3GQYDptlyEWoiofWz3xlAYS++SeGONuXVd\\\\\\\/c5+cjxw\\\\\\\/Pc9ETaEpuHEOd6sfTwhzT0GVH9zR1lcGYkDyIe5F0HrSScZOkp\\\\\\\/FhK8ljAZeQD9rtqhVQtz8Ig3sFDPUb4qHbugXiZCsQgxXEdhR2K1mAkxkD98LjZ2QVdXrEfUuV\\\\\\\/gd0t6cmjIXnhF0dEHTM1jc38DcuHIN3r5ABoSaZZfpgyYQFEjellNcAVqDxB6a\\\\\\\/BKnV1qWspqkeXadHk68N24HZnA6Dk\\\\\\\/ZpY3s4NryGbeu1lb27iu8Yyq9KLDMgDilQM+\\\\\\\/A2g2gTDXrjkwFzvCfpEadaX8UXZ6Fy0MoefoyRGa2WFV+W8fQBnUdycU0b8T6f7896iPrmj4ypHmm8AaCpU+7I6eV3TRRoSXUo\\\\\\\/Ok8IBOlPY8Ht9n+KH\\\\\\\/nXBjpCdmIw1LchyGuFNXzwAZnJstl08QTp3SHiGnyLssDFY79MvhndrrkrAIbjkzaeZrrg8AMjO5LLQmn5R4eqg5yhBDs0Zz1br+h2x9WjbL9\\\\\\\/XfCBeOV7mDO6Oza0+IFGDre5QqgVNMqOzf\\\\\\\/\\\\\\\/exkk399Yoe0mKOMSP9D7DXQzFaKSgx4a\\\\\\\/wNudfPrX7RFcMBbxDdK4\\\\\\\/IT9P+pqE\\\\\\\/r668XDZRnYONFLl\\\\\\\/lcAfXzGdd2aacwfDDgUb7BwnA0f68NIMMNQdtmBLHAluon\\\\\\\/w4uPRBZelBIyQxptIk1WceIMTJytqvlgetugEKcZ+vMabYTqKeer2dDLy27cA05blysUR7fG0Wy\\\\\\\/Iu5vioG9T\\\\\\\/IBPmTNBKnpjRsLBEskO67LRCnl8vXBqB2VZjGH8PPFIUI6E5fbC1dbjbzn3Eg+L7IiQDyiSkXkg4xy1oE8xSD4E+qSkBGmqKawOq3TRWJKZmi2yG6lsN7P\\\\\\\/4LnHHqXvB+qvdN0I0oJibrPi37RyDsWkur9oWxb0RoU0sDytCgnRweZ1a8mxF7Jb+OlpMtzMFCCvs0orftALttvFXdFbUFI0HXij5pEhUdae50AA2nUrpjgPm7x85q0p0mgtP6hvu0Lm\\\\\\\/AYmALXGmBysoTvTrK5Ic15Q9iV48vzCYgQ8lSEwOPrVAHLEhjt63dBiHpcOjpkVqCDRPaSWdG2n7jKI63IYH4okSs3rZ\\\\\\\/OL3AfeKz0jyk6LOODI2CFHZK7brDszLerbGhw5ENx5SmaHZqs12qY9I7DF+vahjpuCW5prFq33uhuyKZoeo8tIrcG\\\\\\\/MIbkx1aq13gumcVeDf9HzEKwgj9ocEZlf7blisdWatyJj37lYlDTbw6y4445cjPs32l9VjyUWcmCqvz7YIRfI2y41+jkm0Hx6lRARa61stfWG1pdfdlUjvlEQlZCkXZXdWZXleSk\\\\\\\/Kce7viic1v4v5laI92GdKCjEFC0b5YxXE1s7WDrN9lXrTsnPdTkr+XXxK1Q4onI3jeV4HOmfgyW0DbbkMwfj73cVRzs8BrKkLdu7YgRzGp+D4olk183XBPQOjxI1d\\\\\\\/WaRLZL06vrynAKlGuk4ffi28FletRYG+BY9URCy5kAD8uCyHeCzjFAFAuVp1MCGZzr9Sn7pCO2jmnS748kh7J9BX20EeQxnMxSgJ3p1oHVrU31n7Y8C8FaU4Jq\\\\\\\/tpPlQGbSFByMumGqE44LWbAjgHgyBy+DIwVv80qrkms3s4V+izK+O7gvb9ZIoCywQPqQelC7dHg\\\\\\\/0NCJ7ca2PNYQCCrYB9rgp7QMKFV6tBZT7YGLm\\\\\\\/ZACuoS1VGo+TK67Q+isl1ycmwipGb0uPLYOnS1oWLyYlY0Io+ytjas9F\\\\\\\/5LGkD0YN1oaRPkb7d+cDOgccG56lhwPRI2stoI3iKuz87FTL9LCmkRk6oSIpW6MRWrp8m1DemvRuJOjkpqfGiOeulPVgglU0+ftftyT8DZ8qyI8dFyy1YsoYf2rPZ1WuUDrSj\\\\\\\/XKnYIF7q1XPHZ0IUHP+3RHNkrHj7gQe+U7I7qBQTWyuA==\\\u0022,\\\u0022iv\\\u0022:\\\u00227X\\\\\\\/YYThRIxSh5+kOx+Tiig==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022f2ab71fe8e1a81b2adb9b628d48b022aab1ad1ea998fd517d14267315e453281\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022QBe57vLL6r\\\\\\\/TGkSba8sOygshF5t07gkEe8qeKPr2faBDObEMNf5eF0JyulW0\\\\\\\/EDGR2ChQXzIpQJq+cku7Yr55iQSP4nnJOYlxNx49y4+YI\\\\\\\/WHR6KSXqWmOfCR75Wx6wtHdca3x7NQORDG9qEt3KmlXvbvOTPaaTwHcd5QMDgtPjPaJhSYkT8+MAJsPgXr00XPqEyEBD9lNa7du34C4z1Uya\\\\\\\/Q4X1IVGtsz9WXvuWOpuxP4DgVM3Tts2COfVQzPcdtikWFiWSOte8\\\\\\\/IT0S7JXEygWxhljG9hYn6eMZ4RtRNgTDWuxdxskHRIhBet3Ps2frUmMiFtBE41zsZJOPxXHJ0VUVkEdlltK59LgWnr6mYjqPrUEdOve9kRgyhxEN1vMm3wOMd3NwH4uP2MiisIRPJC69zZ1HoZ3y1EzWhFVyx02Zyoi7VNQf+1av+SJkjnU2kl9XCqfuw9QEEWJ+jEf1gmkFo7Q302QGHtwK3G1VQVEjuuVuZArLJPC9iiG3A1Ff9Yx0kBfaay+RIIOIVP9UJ6hQWirzY22ls5BsGQdv44t2GKSie+iDT3lykVGiDjNdgV22oYYj6l7lIHehSqwrqQSuVAo9Gz9FKnbjVJWdXO8U\\\\\\\/iZGVWcXRUn4TpYkWfz2XDO\\\\\\\/ZBpQWfSmpafMYBCSIsl+6y9ZnrA8XyFsx7GKJ8eNcRmcp+8aIauGq1ArDl85pU9d5qUnbD+goR9GQd89t1ikMEvWfOdS0a6QXcfuW38QgOiOyu9Q3liYA89GvkNRjYXKgaw2I44PPOYT61xdeX+EnpfihbL04cioIDK0a7D+Dz8Yum\\\\\\\/mUH0fwuM\\\\\\\/pveDjEW0ihazJWSIOkDvtlIj0RIM8264KA5zzHdA1RzO0VZ1np5y9BE3NRES7YU9hcdXNpwg0Y+xS6jO4Noq2g7YfNhRVOiAaXt6wuGO\\\\\\\/eNa\\\\\\\/XQjyyx4m\\\\\\\/PhVpqlJEdKJ5a0wN4Rg6Sq\\\\\\\/t\\\\\\\/Cievu84ytDpis87fLjPGtUYZj3oRLDzK4idxCiwPn9h82xr1lZ5OYqPmavKJKyp3iYuTTwcphU5njaSonRFuGSYVEp4gSRSbjfX\\\\\\\/ExdRwYG+WM+MAwpU1AOaNV0QVLbsz7e85HaXMkxouTqgMXhwymH0eai0euucz9ggIot1bAV05Cc9RyrI9qmVt1fPF+131hSscMtb4KCjFdNnhyo7y2eWcckRbcSVjC6JumXPB0TgVZUxSxzFFBVlQNOxtd+I526vcbztTBQnNWc3Y3ku+NNka5VTDdR3juVeZyCJTzR6E23kQPZIuSKpWnM2B+4nykeeGDoWLaowFbENK6f5V7DFjpkQ7UEFuWCyb5GMfbXqXASb\\\\\\\/C4dtSq6J3SdA58\\\\\\\/RmpvSrzwJaY1ZcsoapLsUKmoiJ8W632GZuM7r2CdD0rsYsIbh3RRj35t7hpmd4vbauwIVK76oM7dK8E+mqN3Ahp4d2I\\\\\\\/X\\\\\\\/zXnMrsNrCPiVomDoVYIlvToOk1L4uayIRO9bJzZsGF+4GUIv4KbzypY1yzL3lcgB32CEBVf6pik5XK24Sr0pMhTiasHD9t0zoIPin3MJ5T+WMGgCwhgf78MkTPSegyK3J2oLgLYCPEyCdMb2gpMS\\\\\\\/qLFCksVxjdncp1FeLVuAa+H91E1j\\\\\\\/p2nCQ1qV9TlU\\\\\\\/y3Obp9Dq6tnpIl14fdFGuj5lrAkagdmgOxdUeOvl6ktndNRYzMW2tc9MFKBCP9zievOKgd4xZZiDeLj8BUyrwvp3zJf\\\\\\\/62iYP6oswQ4erPLjVAq+0zujOI4\\\\\\\/lbRKUAuDK7SV39UpXQROEB6A7jxC2qpPrZ1s3rSOAxhrfQhOCzFjVBEca1+dFyv21KzynbQ0Nd9gQW+W\\\\\\\/jJ\\\\\\\/0CpOyTYYCOdq0f39ZmLDdVp0ugrqAJ6kt0YAszMkvBq1QGR9Ey0\\\\\\\/MLPcUpxCvb\\\\\\\/cEdvBl3i3G1JzUbHDSnd7fFE\\\\\\\/Fkd1A8hooaMejtDxdwj2OItXEXg25lhaB4MjF0GDx0BqVkYYYqsKHNf65B2+S6rlCnJQN8t3w==\\\u0022,\\\u0022iv\\\u0022:\\\u00223VkByPh42x9u2dXMKjzz+A==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002289e58b32b8e9ec43b074f72a2abceeb3c15e99ddc668d75c3867495bf8cfe5a5\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459331 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/2/10/1243" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-PQ8c7WKElcHlJo7emnalVFHWTdKSmcgZuXE3PTPW-3A" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/2/10/1243" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-_7E80BrDIzEuAOZe3yrSD9qEEjJzqaxFQtHfqXSW6-A" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/2/10/1233" title="Second-Generation Genetic Linkage Map of Catfish and Its Integration with the BAC-Based Physical Map" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/2/10/1257" title="Exploration of the Genetic Organization of Morphological Modularity on the Mouse Mandible Using a Set of Interspecific Recombinant Congenic Strains Between C57BL/6 and Mice of the Mus spretus Species" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459331" id="node459331" data-pisa="ggg;2/10/1243" data-pisa-master="ggg;2/10/1243" data-apath="/ggg/2/10/1243.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Overlapping ETS and CRE Motifs (<sup>G</sup>/<sub>C</sub>CGGAAGTGACGTCA) Preferentially Bound by GABPα and CREB Proteins</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Raghunath Chatterjee</span>, <span class="highwire-citation-author" data-delta="1">Jianfei Zhao</span>, <span class="highwire-citation-author" data-delta="2">Ximiao He</span>, <span class="highwire-citation-author" data-delta="3">Andrey Shlyakhtenko</span>, <span class="highwire-citation-author" data-delta="4">Ishminder Mann</span>, <span class="highwire-citation-author" data-delta="5">Joshua J. Waterfall</span>, <span class="highwire-citation-author" data-delta="6">Paul Meltzer</span>, <span class="highwire-citation-author" data-delta="7">B. K. Sathyanarayana</span>, <span class="highwire-citation-author" data-delta="8">Peter C. FitzGerald</span> and <span class="highwire-citation-author" data-delta="9">Charles Vinson</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2012 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 2 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1243-1256;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.004002 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459331" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Raghunath Chatterjee </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Laboratory of Metabolism, National Cancer Institute, National Institutes of Health, Bethesda, Maryland 20892</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Raghunath%2BChatterjee%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chatterjee%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARaghunath%2BChatterjee%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jianfei Zhao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Laboratory of Metabolism, National Cancer Institute, National Institutes of Health, Bethesda, Maryland 20892</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jianfei%2BZhao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhao%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJianfei%2BZhao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Ximiao He </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Laboratory of Metabolism, National Cancer Institute, National Institutes of Health, Bethesda, Maryland 20892</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ximiao%2BHe%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=He%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AXimiao%2BHe%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Andrey Shlyakhtenko </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Laboratory of Metabolism, National Cancer Institute, National Institutes of Health, Bethesda, Maryland 20892</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andrey%2BShlyakhtenko%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Shlyakhtenko%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAndrey%2BShlyakhtenko%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Ishminder Mann </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Laboratory of Metabolism, National Cancer Institute, National Institutes of Health, Bethesda, Maryland 20892</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ishminder%2BMann%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mann%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AIshminder%2BMann%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Joshua J. Waterfall </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Genetics Branch, National Cancer Institute, National Institutes of Health, Bethesda, Maryland 20892</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joshua%2BJ.%2BWaterfall%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Waterfall%20JJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJoshua%2BJ.%2BWaterfall%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Paul Meltzer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Genetics Branch, National Cancer Institute, National Institutes of Health, Bethesda, Maryland 20892</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Paul%2BMeltzer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Meltzer%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APaul%2BMeltzer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">B. K. Sathyanarayana </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Laboratory of Molecular Biology, National Cancer Institute, National Institutes of Health, Bethesda, Maryland 20892</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=B.%2BK.%2BSathyanarayana%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sathyanarayana%20BK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AB.%2BK.%2BSathyanarayana%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Peter C. FitzGerald </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Genome Analysis Unit, National Cancer Institute, National Institutes of Health, Bethesda, Maryland 20892</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Peter%2BC.%2BFitzGerald%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=FitzGerald%20PC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APeter%2BC.%2BFitzGerald%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Charles Vinson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Laboratory of Metabolism, National Cancer Institute, National Institutes of Health, Bethesda, Maryland 20892</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Charles%2BVinson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Vinson%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ACharles%2BVinson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:Vinsonc@mail.nih.gov" class="" data-icon-position="" data-hide-link-title="0">Vinsonc@mail.nih.gov</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/2/10/1243" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459331" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459331/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/2/10/1243.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459331" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459331/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/2/10/1243.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459331" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459331/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/2/10/1243.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459331" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459331/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/2/10/1243.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459331" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459331/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">Previously, we identified 8-bps long DNA sequences (8-mers) that localize in human proximal promoters and grouped them into known transcription factor binding sites (TFBS). We now examine split 8-mers consisting of two 4-mers separated by 1-bp to 30-bps (X<sub>4</sub>-N<sub>1-30</sub>-X<sub>4</sub>) to identify pairs of TFBS that localize in proximal promoters at a precise distance. These include two overlapping TFBS: the ETS⇔ETS motif (<sup>C</sup>/<sub>G</sub>CCGGAA<strong>G</strong>CGGAA) and the ETS⇔CRE motif (<sup>C</sup>/<sub>G</sub>CGGAA<strong>GTG</strong>ACGTCAC). The nucleotides in bold are part of both TFBS. Molecular modeling shows that the ETS⇔CRE motif can be bound simultaneously by both the ETS and the B-ZIP domains without protein-protein clashes. The electrophoretic mobility shift assay (EMSA) shows that the ETS protein GABPα and the B-ZIP protein CREB preferentially bind to the ETS⇔CRE motif only when the two TFBS overlap precisely. In contrast, the ETS domain of ETV5 and CREB interfere with each other for binding the ETS⇔CRE. The 11-mer (CGGAA<strong>GTG</strong>ACG), the conserved part of the ETS⇔CRE motif, occurs 226 times in the human genome and 83% are in known regulatory regions. <em>In vivo</em> GABPα and CREB ChIP-seq peaks identified the ETS⇔CRE as the most enriched motif occurring in promoters of genes involved in mRNA processing, cellular catabolic processes, and stress response, suggesting that a specific class of genes is regulated by this composite motif.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/proximal-promoters" class="hw-term hw-article-keyword hw-article-keyword-proximal-promoters" rel="nofollow">proximal promoters</a></li><li class="kwd"><a href="/keyword/transcription-factor-binding-sites" class="hw-term hw-article-keyword hw-article-keyword-transcription-factor-binding-sites" rel="nofollow">transcription factor binding sites</a></li><li class="kwd"><a href="/keyword/co-localization" class="hw-term hw-article-keyword hw-article-keyword-co-localization" rel="nofollow">co-localization</a></li><li class="kwd"><a href="/keyword/transcriptional-start-site" class="hw-term hw-article-keyword hw-article-keyword-transcriptional-start-site" rel="nofollow">transcriptional start site</a></li><li class="kwd"><a href="/keyword/emsa" class="hw-term hw-article-keyword hw-article-keyword-emsa" rel="nofollow">EMSA</a></li></ul><p id="p-4">Gene expression is controlled by many genetic and epigenetic elements in a highly coordinated manner, but the DNA sequence of the genome is the ultimate arbiter. Specific DNA sequences in both proximal promoters and more distant regions are bound by sequence-specific DNA binding proteins that regulate gene expression (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Smale &amp; Kadonaga 2003</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Farnham 2009</a>). Additionally, CpG islands (regions of 300-bps to 3000-bps containing a high frequency of the CG dinucleotide) are frequently located at or near mammalian promoters (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Bird 2011</a>). Many experimental (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Carninci <em>et al.</em> 2006</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Johnson <em>et al.</em> 2007</a>) and computational methods have been employed to identify biologically relevant transcription factor binding sites (TFBS). The computational methods typically examine DNA sequence enrichment near a biologically defined regulatory region like the transcriptional start site (TSS) (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Frith <em>et al.</em> 2002</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Ohler <em>et al.</em> 2002</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Kel <em>et al.</em> 2003</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bina <em>et al.</em> 2004</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">FitzGerald <em>et al.</em> 2004</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Marino-Ramirez <em>et al.</em> 2004</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Matys <em>et al.</em> 2006</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Pachkov <em>et al.</em> 2007</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Ji <em>et al.</em> 2008</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Kharchenko <em>et al.</em> 2008</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Portales-Casamar <em>et al.</em> 2010</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Oh <em>et al.</em> 2011</a>; <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Vinson <em>et al.</em> 2011</a>). Examination of related mammals has also identified many DNA motifs in promoters that are conserved, suggesting that they may be TFBS, while the 3′UTR have conserved sequences thought to be microRNAs (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Xie <em>et al.</em> 2005</a>).</p><p id="p-5">In an earlier study, we identified 8-bps long DNA sequences (8-mers) that are localized in human proximal promoters (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">FitzGerald <em>et al.</em> 2004</a>) and Drosophila promoters (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">FitzGerald <em>et al.</em> 2006</a>), and we presented evidence that motifs near the TSS are biologically functional. In human promoters, these sequences were grouped into known TFBS, including SP1, CCAAT, ETS, E-Box, CRE, Box A, NRF1, and TATA. Analyses of promoters with the conservation of DNA sequences among the related mammals greatly enhanced the identification of regulatory motifs (<a id="xref-ref-54-2" class="xref-bibr" href="#ref-54">Xie <em>et al.</em> 2005</a>).</p><p id="p-6">To identify additional biologically important DNA sequences in human proximal promoters, we analyzed the distribution of discontinuous 8-mers, also called split 8-mers (<a id="xref-ref-50-2" class="xref-bibr" href="#ref-50">Vinson <em>et al.</em> 2011</a>). Each split 8-mer is composed of two 4-mers separated by 1-bp to 30-bps. If each 4-mer represents a part of a TFBS, this calculation would identify pairs of TFBS that co-occur in the same proximal promoter as observed in other mammalian promoters (<a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">FitzGerald <em>et al.</em> 2004</a>). Split 8-mer enrichment in promoters declines with increasing distance between the two 4-mers. In contrast, Drosophila contains many split 8-mers in which the 4-mers are separated by 20-bps to 30-bps that localize in promoters (<a id="xref-ref-50-3" class="xref-bibr" href="#ref-50">Vinson <em>et al.</em> 2011</a>).</p><p id="p-7">This article examines the split 8-mers that localize in human promoters. We extended our previous work with split 8-mers in human promoters (<a id="xref-ref-50-4" class="xref-bibr" href="#ref-50">Vinson <em>et al.</em> 2011</a>) by evaluating whether the split 8-mers that localize in promoters have a preferred distance between the two 4-mers. This analysis identified an ETS motif overlapping with a CRE motif (ETS⇔CRE) that localizes in proximal promoters. DNA binding experiments show that GABPα and CREB preferentially bind the two TFBS when they overlap and produce the ETS⇔CRE motif enriched in proximal promoters.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Dataset generation</h3><p id="p-8">From University of California Santa Cruz Genome Bioinformatics website (<a href="http://genome.ucsc.edu/">http://genome.ucsc.edu/</a>), we obtained the DNA sequence data for RefSeq genes in the Golden Path Human Genome Assembly with annotated TSS, representing sequences from –1,000 bp to +500 bp relative to the TSS. The initial dataset contained 26,431 promoters. The set was further processed to improved relevance and the validity of the analysis using the following criteria. First, for promoters with 100% identical sequences, only one copy of them was kept (5483 promoters were removed). Second, promoters containing unknown nucleotides (N) of at least 150 bps were removed (8 promoters). Third, promoters with duplicated RefSeq numbers were removed (411 promoters). Fourth, of the remaining 20,529 promoters, 18,451 were determined to have unique sequences, whereas 2078 promoters had duplicated sequences shared among themselves. Among these 2078 promoter sequences, 68 had more than 10 overlapping duplicated regions of at least 250 bps with other promoter sequences and were deleted from the analysis. One thousand five hundred thirty-five (1535) promoter sequences contained closely identical sequences among themselves, and they comprised 701 unique groups (pairs in most cases); only 701 “representative” promoters were kept for the analysis. An additional 475 promoters were kept for the analysis, although they did have some mixed overlapping sequencing. This allowed us to retain only 1176 out of these 2078 promoters. Fifth, two thousand four hundred eighty-four (2484) promoters had start of the coding sequences (translational start sites) within 30-bps of the TSS, and these promoters were excluded from the following analysis. Finally, a set of 17,143 promoters (18,451 + 1,176 − 2,484) was obtained and considered for the analysis.</p></div><div id="sec-3" class="subsection"><h3>Analysis of split 8-mers distributions</h3><p id="p-9">There are 4<sup>8</sup> discontinuous non-degenerative 8-mers (X<sub>4</sub>-N<em><sub>k</sub></em>-X<sub>4</sub>; N denotes any arbitrary nucleotides and <em>k</em> denotes spacing between two 4-mers), and of these, <em>ξ</em>4<sup>4</sup> are palindromes and <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/10/1243/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/10/1243/embed/mml-math-1.gif"/></noscript></span></span> are non-palindromes, where each sequence and its complement is represented and <em>ξ</em> = 1 if <em>k</em> is even and 0 if odd. Thus, the number of 8-mers can be reduced to<span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/10/1243/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/10/1243/embed/mml-math-2.gif"/></noscript></span></span>. Those 32,896 or 32,768 8-mers were automatically generated by a custom-made program. The promoter set was searched against them, and final distributions were generated. To analyze the data, we divided 1500-bps into 75 bins each containing 20-bps, numbering bin 1 [–1000 bp; –981 bp] to bin 75 [+481 bp; +500 bp]. We determined the number of times the first nucleotide of a studied DNA sequence (or the last of its complement) occurred within each 20-bps bin. To detect and quantify non-uniform distributions (localization) and the probability of non-uniformity of split 8-mers, we determined localization factor (LF) and <em>P</em>-value as described previously (<a id="xref-ref-11-4" class="xref-bibr" href="#ref-11">FitzGerald <em>et al.</em> 2004</a>; <a id="xref-ref-50-5" class="xref-bibr" href="#ref-50">Vinson <em>et al.</em> 2011</a>).</p></div><div id="sec-4" class="subsection"><h3>Molecular modeling</h3><p id="p-10">The molecular model of the ETS and CREB dimer interacting with a single chain DNA with a specific base pair sequence of CCGGAAGTGACGTCA was built by using two PDB structures, the ETS-1 protein bound to an ETS site (PDB ID: 1K79) (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Garvie <em>et al.</em> 2001</a>) and the CREB dimer bound to the CRE (PDB ID: 1DH3) (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Schumacher <em>et al.</em> 2000</a>). The 10 nucleotides (shown underlined) of the E chain of the DNA (TAGTG<span class="underline">CCGGAA</span><strong><span class="underline">ATG</span></strong><span class="underline">T</span>) of 1K79 were aligned to the 10 nucleotides (shown underlined) in the B chain of the DNA (<span class="underline">CCTTGG</span><strong><span class="underline">CTG</span></strong><span class="underline">A</span>CGTCAGCCAAG) of 1DH3, using Chimera visualization software (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Pettersen <em>et al.</em> 2004</a>). This alignment also results in the nucleotides ATG (shown in bold) of 1K79 aligning with the nucleotides CTG (shown in bold) of 1DH3. The ETS-1 protein and the complementary strand (F chain) of DNA of 1K79 were carried along with the E chain of its DNA during this alignment. From this aligned structures, the first 10 nucleotides (CCTTGGCTGA) and their base pairs in the complimentary chain in the 1DH3 structure were deleted. The remaining chains containing the nucleotides TAGTGCCGGAAATGT of 1K79 and the nucleotides CGTCAGCCAAG of 1DH3 were covalently linked to one another using Chimera software to form one long chain of DNA with the sequence TAGTG<span class="underline">CCGGAA</span><strong><span class="underline">A</span></strong><span class="underline">TG</span><strong><span class="underline">T</span></strong><span class="underline">CGTCA</span>GCCAAG. Similarly, its complimentary DNA chain was also built. The 12<sup>th</sup> and 15<sup>th</sup> bases in this long chain (shown in bold) were mutated to G and A bases, respectively, and the final complex containing this long DNA and the ETS and CRE was subjected to an energy minimization using the Discovery Studio (Accelrys Software) molecular modeling software.</p></div><div id="sec-5" class="subsection"><h3>Electrophoretic mobility shift assay (EMSA)</h3><p id="p-11">EMSA was performed similarly as described previously (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Rishi <em>et al.</em> 2010</a>). GABPα and CREB proteins were <em>in vitro</em> translated using PURExpress <em>In Vitro</em> Protein Synthesis Kit (New England Biolabs, USA) according to manufacturer instructions. The T7 expression plasmids containing the DNA binding domain of GABPα (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Badis <em>et al.</em> 2009</a>) or the B-ZIP domain of CREB (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Ahn <em>et al.</em> 1998</a>) was used as the template DNA. GABPα has a GST-tag at the N-terminus. The protein concentrations were estimated by Western blot using purified GST-CREB or CREB with known concentrations as concentration standards. <em>In vitro</em> translated proteins were mixed with 7 pM <sup>32</sup>P end-labeled double-stranded oligonucleotides containing variants of ETS and CREB binding sites in the gel shift buffer (0.5 mg/ml BSA, 10% glycerol, 2.5 mM DTT, 12.5 mM K<sub>2</sub>HPO<sub>4</sub>-KH<sub>2</sub>PO<sub>4</sub>, pH 7.4, 0.25 mM EDTA). The final volume of the reaction was adjusted to 20 µl. For regular EMSA, the reactions were incubated at 37° for 20 min, followed by cooling at room temperature for 5 min before loading. For supershift experiments, the reactions were first incubated at 37° for 20 min without antibodies. Antibodies (catalog # sc-186, sc-459, or sc-2027, Santa Cruz Biotechnology, USA) were then added, and the reactions were incubated on ice for 30 min, followed by incubation at room temperature for 15 min before loading. 10 µl samples were resolved on 7.5% PAGE at 150 V for 1.5 hr in the 1x TBE buffer (25 mM Tris-boric acid, 0.5 mM EDTA). Sequences of oligonucleotides used for EMSA experiments are listed in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>. For EMSA using ETV5 and CREB, we used purified proteins containing the DNA binding domain of ETV5 or the B-ZIP domain of CREB.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/461857/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/461857/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">DNA probe sequences for EMSA (binding sites underlined)</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-6" class="subsection"><h3>Motif enrichment using ChIP-seq peaks</h3><p id="p-12">For motif analysis, we used published 6442 GABPα ChIP-seq peaks from human Jurket cell line (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Valouev <em>et al.</em> 2008</a>) and 3998 CREB ChIP-seq peaks from mouse in GC1 cells (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Martianov <em>et al.</em> 2010</a>). For motif detection, we used MEME (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Machanick &amp; Bailey 2011</a>) and the peak-motifs package of the Regulatory Sequence Analysis Tools (RSAT) (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Thomas-Chollier <em>et al.</em> 2011</a>). Two thousand eight hundred thirty-four (2834) CREB binding promoters, which were obtained from the ChIP-chip data on human HEK293T cells in three time points (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Zhang <em>et al.</em> 2005</a>), were mapped to human (hg18), which successfully resulted in 2384 promoters bound by CREB. For <em>de novo</em> motif prediction, we used 1463 common binding regions of human CREB ChIP-chip and GABPα ChIP-seq data.</p></div><div id="sec-7" class="subsection"><h3>PhyloP conservation</h3><p id="p-13">Base by base PhyloP score or the <em>P</em>-values for conservation or acceleration <em>P</em>-values based on an alignment and a model of neutral evolution among the 36 mammalian genomes were (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Pollard <em>et al.</em> 2010</a>) downloaded from UCSC database (<a href="http://genome.ucsc.edu/">http://genome.ucsc.edu/</a>). PhyloP scores for each nucleotide in the motif, including 15-bps upstream and 15-bps downstream of each occurrence in the genome, were averaged for all occurrences of each motif.</p></div><div id="sec-8" class="subsection"><h3>Gene Ontology analysis</h3><p id="p-14">Gene Ontology (GO) analysis was performed using DAVID (<a href="http://david.abcc.ncifcrf.gov/">http://david.abcc.ncifcrf.gov/</a>). Go terms with <em>P</em>-values &lt; 0.01 were considered as significantly enriched GO terms. Additionally, Benjamini-Hochberg corrected <em>P</em>-values &lt; 0.01 were considered for the analysis with <em>in vivo</em> ChIP data.</p></div></div><div class="section results" id="sec-9"><h2 class="">Results</h2><div id="sec-10" class="subsection"><h3>Split 8-mers that localize in human proximal promoters</h3><p id="p-15">We aligned human promoters relative to the TSS and determined the distribution of split 8-mers in the promoter region. The split 8-mers consist of two 4-mers separated by 1-bp to 30-bps (X<sub>4</sub>-N<sub>1-30</sub>-X<sub>4</sub>). We considered the promoter region from −1000-bps to +500-bps relative to the TSS and divided the 1500-bp region into 75 bins of 20-bps each. We used a human DNA promoter sequence set obtained from UCSC and removed promoters containing repetitive sequences, resulting in a set of 17,143 promoter sequences (see <em>Materials and Methods</em>). The distribution of each split 8-mer in promoters was determined and a measure of non-uniform distribution termed “localization factor” (LF) was calculated (<a id="xref-ref-50-6" class="xref-bibr" href="#ref-50">Vinson <em>et al.</em> 2011</a>). The statistical significance of the non-random distribution of LF was determined by calculating a probability value (<em>P</em>-value) for each split 8-mer.</p><p id="p-16">Many continuous 8-mers (X<sub>4</sub>-N<sub>0</sub>-X<sub>4</sub>) are enriched in proximal promoters (−120-bps to the TSS) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.004002/-/DC1/004002SI.pdf">supporting information</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.004002/-/DC1/FigureS1.pdf">Figure S1</a>, A and B, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.004002/-/DC1/TableS1.pdf">Table S1</a>) (<a id="xref-ref-11-5" class="xref-bibr" href="#ref-11">FitzGerald <em>et al.</em> 2004</a>, <a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">2006</a>; <a id="xref-ref-54-3" class="xref-bibr" href="#ref-54">Xie <em>et al.</em> 2005</a>; <a id="xref-ref-50-7" class="xref-bibr" href="#ref-50">Vinson <em>et al.</em> 2011</a>). In contrast, fewer split 8-mers with an insert length of 4-bps (X<sub>4</sub>-N<sub>4</sub>-X<sub>4</sub>) localize in proximal promoters (<a id="xref-ref-11-6" class="xref-bibr" href="#ref-11">FitzGerald <em>et al.</em> 2004</a>; <a id="xref-ref-50-8" class="xref-bibr" href="#ref-50">Vinson <em>et al.</em> 2011</a>) (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1, A and B</a>). As insert length increases, preferential localization of split 8-mers in the proximal promoter decreases for both CG- and non-CG split 8-mers and is much more pronounced for the non-CG 8-mers (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1, C and D</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/10/1243/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="(A and B) LF and probability for split 8-mers with a 4-bp insert (X4-N4-X4). (C and D) For each 8-mer (X4-N0-30-X4), we determine which insert length produced the largest LF and plot that value in the column representing that insert length. (C) LF for the 12,547 continuous 8-mers and 10,951 split 8-mers containing the CG dinucleotide. We plot that &#x2013;log P-value at the insert length with the highest LF. (D) Same as (C) but for all non-CG containing 8-mers, the 20,349 continuous 8-mers, and 21,945 split 8-mers with insert length from 1-bp to 30-bps." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-685147227" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;(A and B) LF and probability for split 8-mers with a 4-bp insert (X&lt;sub&gt;4&lt;/sub&gt;-N&lt;sub&gt;4&lt;/sub&gt;-X&lt;sub&gt;4&lt;/sub&gt;). (C and D) For each 8-mer (X&lt;sub&gt;4&lt;/sub&gt;-N&lt;sub&gt;0-30&lt;/sub&gt;-X&lt;sub&gt;4&lt;/sub&gt;), we determine which insert length produced the largest LF and plot that value in the column representing that insert length. (C) LF for the 12,547 continuous 8-mers and 10,951 split 8-mers containing the CG dinucleotide. We plot that &#x2013;log &lt;em&gt;P&lt;/em&gt;-value at the insert length with the highest LF. (D) Same as (C) but for all non-CG containing 8-mers, the 20,349 continuous 8-mers, and 21,945 split 8-mers with insert length from 1-bp to 30-bps.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/10/1243/F1.medium.gif" width="440" height="328"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1&#x2002;" src="https://www.g3journal.org/content/ggg/2/10/1243/F1.medium.gif" width="440" height="328"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/10/1243/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/10/1243/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461964" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1 </span> <p id="p-17" class="first-child">(A and B) LF and probability for split 8-mers with a 4-bp insert (X<sub>4</sub>-N<sub>4</sub>-X<sub>4</sub>). (C and D) For each 8-mer (X<sub>4</sub>-N<sub>0-30</sub>-X<sub>4</sub>), we determine which insert length produced the largest LF and plot that value in the column representing that insert length. (C) LF for the 12,547 continuous 8-mers and 10,951 split 8-mers containing the CG dinucleotide. We plot that –log <em>P</em>-value at the insert length with the highest LF. (D) Same as (C) but for all non-CG containing 8-mers, the 20,349 continuous 8-mers, and 21,945 split 8-mers with insert length from 1-bp to 30-bps.</p><div class="sb-div caption-clear"></div></div></div><p id="p-18">The most localizing split 8-mer sequences with an insert length of 1-bps and 2-bps both represent the CRE motif (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1D</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.004002/-/DC1/TableS1.pdf">Table S1</a>), suggesting that the CRE is 10-bps long (GTGACGTCAC). The most localizing sequence with both a 3-bps and 4-bps insert are a CG-rich 4-mer followed by TATA (CCGG-N<sub>3</sub>-TATA and GCCG-N<sub>4</sub>-TATA), sequences previously identified that function in proximal promoters (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Lagrange <em>et al.</em> 1998</a>). These split 4-mers are not strand specific, indicating that the CG-rich 4-mer can be either before or after the strand-specific TATAA (<a id="xref-ref-11-7" class="xref-bibr" href="#ref-11">FitzGerald <em>et al.</em> 2004</a>). Virtually all the localizing split 8-mers with an insert length of 5-bps or more contain the CG dinucleotide (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1, C and D</a>). The 20 most localizing split 8-mers with insert length of 0-bps, 2-bps, 4-bps, and 5-bps to 30-bps are presented in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.004002/-/DC1/TableS1.pdf">Table S1</a>.</p></div><div id="sec-11" class="subsection"><h3>Split 8-mers that localize in promoters at a unique insert length</h3><p id="p-19">The split 8-mers that localize in proximal promoters were grouped into three classes (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.004002/-/DC1/TableS1.pdf">Table S1</a>): (i) split 8-mers with a short insert length of 1-bps or 2-bps representing a single TFBS (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.004002/-/DC1/FigureS2.pdf">Figure S2</a>, A–D); (ii) split 8-mers that localize in proximal promoters at many insert lengths representing co-localizing TFBS, each represented by a single 4-mer (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.004002/-/DC1/FigureS2.pdf">Figure S2</a>, E–H); and (iii) split 8-mers that localize in proximal promoters at a specific insert length. These include CGGA-N<sub>4</sub>-ACGT, which represents an ETS motif and a CRE motif, and unidentified sequences; <em>e.g.</em> GGGA-N<sub>2</sub>-TGTA (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.004002/-/DC1/FigureS2.pdf">Figure S2</a>, I and J).</p><p id="p-20">To identify split 8-mers that localize in proximal promoters at only a precise insert length, the max LF for all split 8-mers with insert lengths from 0-bps to 30-bps (X<sub>4</sub>-N<sub>1-30</sub>-X<sub>4</sub>) were determined and compared with the ratio of max LF to the second highest LF (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2, A and B</a>). A close to 1 ratio of max LF to the second highest LF indicates localization of split 8-mers at various insert lengths, whereas a ratio with higher values is indicative of split 8-mers that are localized at a precise insert length. Both kinds of sequences are observed for 8-mers with a high LF. To identify the insert length that produces the precisely positioned pairs of 4-mers, we examined each insert length. Continuous 8-mers (X<sub>4</sub>-N<sub>0</sub>-X<sub>4</sub>) have many sequences with a high LF and large ratio (LF(MAX)/LF(MAX-1). These sequences are the TFBS previously described that localize in proximal promoters (<a id="xref-ref-11-8" class="xref-bibr" href="#ref-11">FitzGerald <em>et al.</em> 2004</a>). The two 4-mers (TGAC and GTCA) that create the CRE (TGACGTCA) motif preferentially localize in promoters when the insert length is 0-bps (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2D</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.004002/-/DC1/FigureS2.pdf">Figure S2</a>, A and B). Similar results were obtained for the ETS motif (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.004002/-/DC1/FigureS2.pdf">Figure S2</a>, C and D). When we examined split 8-mers with an insert length of 2-bps, fewer 8-mers had both a high LF and ratio (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2, E and F</a>). These include GTGA-N<sub>2</sub>-TCAC, representing the CRE; CGGA-N<sub>2</sub>-TGAC, representing overlapping ETS and CRE TFBS (ETS⇔CRE) (CGGA<em>AG</em>TGAC); and GGAA-N<sub>2</sub>-GGAA, representing an ETS motif overlapping with a second ETS motif (ETS⇔ETS) (GGAA<em>GC</em>GGAA) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.004002/-/DC1/TableS1.pdf">Table S1</a> and <a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2</a>). A systematic analysis of the human promoters using comparative genomics for the detection of regulatory motifs also identified an unannotated motif GGAANCGGAANY (<a id="xref-ref-54-4" class="xref-bibr" href="#ref-54">Xie <em>et al.</em> 2005</a>), which is essentially the ETS⇔ETS motif. Insert length of 4-bps produced even fewer sequences that are precisely localized (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2, G and H</a>). Insert length of 5-bps to 30-bps identified many 8-mers with a high LF but a low ratio, indicating that they are co-occurring in promoters at many insert lengths (<a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2, I and J</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/10/1243/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Identification of split 8-mers that localize in promoters only at a unique insert length. (A) The maximum LF for all 8-mers (X4-N0-30-X4) is plotted on the horizontal axis vs. maximum LF for 8-mers with an insert length from 0-bps to 30-bps [LF(Max)] divided by the second highest LF [LF(Max-1)]. The points at the top right of the plot represent 8-mers that localize in promoters only at one inset length. (B) Probability (P) (P = 10&#x2013;x) of the LF being non-random. To identify the insert length that produces unique localization in promoters, the horizontal axis shows the probability of LF for split 8-mers for specific insert lengths. (C and D) Localization of continuous 8-mers (X4-N0-X4) in proximal promoters only when the inset length is 0-bps. (E and F) Localization of split 8-mers with insert length of 2-bps (X4-N2-X4) in proximal promoters only when the inset length is 2-bps. (G and H) Localization of split 8-mers with insert length of 4-bps (X4-N4-X4) in proximal promoters only when the inset length is 4-bps. (I and J) Unique localization of split 8-mers with insert length ranging from 5-bps to 30-bps (X4-N5-30-X4)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-685147227" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Identification of split 8-mers that localize in promoters only at a unique insert length. (A) The maximum LF for all 8-mers (X&lt;sub&gt;4&lt;/sub&gt;-N&lt;sub&gt;0-30&lt;/sub&gt;-X&lt;sub&gt;4&lt;/sub&gt;) is plotted on the horizontal axis &lt;em&gt;vs.&lt;/em&gt; maximum LF for 8-mers with an insert length from 0-bps to 30-bps [LF(Max)] divided by the second highest LF [LF(Max-1)]. The points at the top right of the plot represent 8-mers that localize in promoters only at one inset length. (B) Probability (P) (&lt;em&gt;P&lt;/em&gt; = 10&lt;sup&gt;&#x2013;x&lt;/sup&gt;) of the LF being non-random. To identify the insert length that produces unique localization in promoters, the horizontal axis shows the probability of LF for split 8-mers for specific insert lengths. (C and D) Localization of continuous 8-mers (X&lt;sub&gt;4&lt;/sub&gt;-N&lt;sub&gt;0&lt;/sub&gt;-X&lt;sub&gt;4&lt;/sub&gt;) in proximal promoters only when the inset length is 0-bps. (E and F) Localization of split 8-mers with insert length of 2-bps (X&lt;sub&gt;4&lt;/sub&gt;-N&lt;sub&gt;2&lt;/sub&gt;-X&lt;sub&gt;4&lt;/sub&gt;) in proximal promoters only when the inset length is 2-bps. (G and H) Localization of split 8-mers with insert length of 4-bps (X&lt;sub&gt;4&lt;/sub&gt;-N&lt;sub&gt;4&lt;/sub&gt;-X&lt;sub&gt;4&lt;/sub&gt;) in proximal promoters only when the inset length is 4-bps. (I and J) Unique localization of split 8-mers with insert length ranging from 5-bps to 30-bps (X&lt;sub&gt;4&lt;/sub&gt;-N&lt;sub&gt;5-30&lt;/sub&gt;-X&lt;sub&gt;4&lt;/sub&gt;).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/10/1243/F2.medium.gif" width="274" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2&#x2002;" src="https://www.g3journal.org/content/ggg/2/10/1243/F2.medium.gif" width="274" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/10/1243/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/10/1243/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461955" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2 </span> <p id="p-21" class="first-child">Identification of split 8-mers that localize in promoters only at a unique insert length. (A) The maximum LF for all 8-mers (X<sub>4</sub>-N<sub>0-30</sub>-X<sub>4</sub>) is plotted on the horizontal axis <em>vs.</em> maximum LF for 8-mers with an insert length from 0-bps to 30-bps [LF(Max)] divided by the second highest LF [LF(Max-1)]. The points at the top right of the plot represent 8-mers that localize in promoters only at one inset length. (B) Probability (P) (<em>P</em> = 10<sup>–x</sup>) of the LF being non-random. To identify the insert length that produces unique localization in promoters, the horizontal axis shows the probability of LF for split 8-mers for specific insert lengths. (C and D) Localization of continuous 8-mers (X<sub>4</sub>-N<sub>0</sub>-X<sub>4</sub>) in proximal promoters only when the inset length is 0-bps. (E and F) Localization of split 8-mers with insert length of 2-bps (X<sub>4</sub>-N<sub>2</sub>-X<sub>4</sub>) in proximal promoters only when the inset length is 2-bps. (G and H) Localization of split 8-mers with insert length of 4-bps (X<sub>4</sub>-N<sub>4</sub>-X<sub>4</sub>) in proximal promoters only when the inset length is 4-bps. (I and J) Unique localization of split 8-mers with insert length ranging from 5-bps to 30-bps (X<sub>4</sub>-N<sub>5-30</sub>-X<sub>4</sub>).</p><div class="sb-div caption-clear"></div></div></div><p id="p-22">This analysis identified many split 8-mers with distinctive distributions; we focused our analysis on the overlapping ETS and CRE motifs. The distribution of the ETS⇔CRE motif split 8-mer CGGA-N<sub>4</sub>-ACGT shows localization in proximal promoters (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>). The split 8-mer CGGA-N<sub>0-30</sub>-ACGT preferentially localizes in proximal promoters when separated by 4-bps, with the continuous 12-mer CGGAA<strong>GTG</strong>ACGT being the most localizing and abundant (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3, A and B</a>). More modest localization is observed at 20-bps and 22-bps, which has not been evaluated. This sequence contains both the ETS motif (CGGAA<strong>GTG)</strong> and the CRE motif (<strong>GTG</strong>ACGT). The <strong>GTG</strong> trinucleotide is common to both the ETS and CRE motifs. These TFBS overlap to produce the ETS⇔CRE motif. The full ETS⇔CRE motif would be the two 16-mers <sup>C</sup>/<sub>G</sub>CGGAA<strong>GTG</strong>ACGTCAC that occur five times in the human genome (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). There are more than 4×10<sup>9</sup> 16-mers, and thus, each 16-mer would be expected to occur by chance only about once in a vertebrate genome of ∼3×10<sup>9</sup> bps.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/10/1243/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The ETS&#x21D4;CRE motif. (A) Distribution of the split 8-mer CGGA-N4-ACGT and the 12-mer CGGAAGTGACGT in human promoters. (B) LF for CGGA-N4-ACGT from insert size of 0-bps to 30-bps. (C) Space-filling model of ETS and CREB proteins binding to ETS&#x21D4;CRE (GCGGAAGTGACGTCA). Note the 3-bp overlap of the two TFBS. (D and E) Ribbon presentation of ETS and CREB proteins binding to ETS&#x21D4;CRE motif from the side and top relative to DNA." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-685147227" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The ETS&#x21D4;CRE motif. (A) Distribution of the split 8-mer CGGA-N&lt;sub&gt;4&lt;/sub&gt;-ACGT and the 12-mer CGGAA&lt;strong&gt;GTG&lt;/strong&gt;ACGT in human promoters. (B) LF for CGGA-N&lt;sub&gt;4&lt;/sub&gt;-ACGT from insert size of 0-bps to 30-bps. (C) Space-filling model of ETS and CREB proteins binding to ETS&#x21D4;CRE (GCGGAA&lt;strong&gt;GTG&lt;/strong&gt;ACGTCA). Note the 3-bp overlap of the two TFBS. (D and E) Ribbon presentation of ETS and CREB proteins binding to ETS&#x21D4;CRE motif from the side and top relative to DNA.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/10/1243/F3.medium.gif" width="440" height="314"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3&#x2002;" src="https://www.g3journal.org/content/ggg/2/10/1243/F3.medium.gif" width="440" height="314"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/10/1243/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/10/1243/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461966" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3 </span> <p id="p-23" class="first-child">The ETS⇔CRE motif. (A) Distribution of the split 8-mer CGGA-N<sub>4</sub>-ACGT and the 12-mer CGGAA<strong>GTG</strong>ACGT in human promoters. (B) LF for CGGA-N<sub>4</sub>-ACGT from insert size of 0-bps to 30-bps. (C) Space-filling model of ETS and CREB proteins binding to ETS⇔CRE (GCGGAA<strong>GTG</strong>ACGTCA). Note the 3-bp overlap of the two TFBS. (D and E) Ribbon presentation of ETS and CREB proteins binding to ETS⇔CRE motif from the side and top relative to DNA.</p><div class="sb-div caption-clear"></div></div></div><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/461982/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/461982/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Occurrence of specific motifs in human genome, promoters, CpG islands, and housekeeping DHS regions</span><div class="sb-div caption-clear"></div></div></div><p id="p-24">Two versions of the ETS motif that localize in proximal promoters differ only in the first nucleotide, the more common CCGGAA and the rarer GCGGAA (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.004002/-/DC1/FigureS3.pdf">Figure S3</a>A) (<a id="xref-ref-11-9" class="xref-bibr" href="#ref-11">FitzGerald <em>et al.</em> 2004</a>). DNA binding specificities of the 27 human ETS family members identify three proteins (SPI1, SPIB, and SPIC) that preferentially bind the rarer ETS motif (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Kaplan <em>et al.</em> 2010</a>). The rarer GCGGAA ETS motif is enriched compared with the CCGGAA motif in the ETS⇔CRE motif (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.004002/-/DC1/FigureS3.pdf">Figure S3</a>B).</p></div><div id="sec-12" class="subsection"><h3>Molecular model of ETS⇔CRE motif bound by DNA</h3><p id="p-25">To evaluate the potential for simultaneous binding of three proteins (ETS monomer and CREB dimer) to the ETS⇔CRE motif, we built a molecular model using PDB files of the ETS1 protein bound to an ETS site (PDB ID: 1K79) (<a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Garvie <em>et al.</em> 2001</a>) and the CREB dimer bound to the CRE (PDB ID: 1DH3) (<a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">Schumacher <em>et al.</em> 2000</a>). The two structures were aligned computationally after superimposing 10 DNA bases on each strand of DNA. The combined structure did not produce protein clashes, suggesting that both proteins could potentially bind the ETS⇔CRE motif simultaneously (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3, C–E</a>). The <strong>GTG</strong> trinucleotide, which is common to both the ETS and CRE motifs, interacts with both proteins in the model. The ETS domain, a winged helix-turn-helix protein fold, interacts with the major groove using an α-helix to bind the core GGAA of the motif. It also crosses the phosphate backbone and interacts with the minor groove of the <strong>GTG</strong> trinucleotide (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Hollenhorst <em>et al.</em> 2011b</a>). The CREB dimer interacts with the <strong>GTG</strong> trinucleotide in the major groove and never crosses the DNA backbone.</p></div><div id="sec-13" class="subsection"><h3>The ETS protein GABPα and the B-ZIP protein CREB preferentially bind to ETS⇔CRE</h3><p id="p-26">EMSA was used to investigate whether ETS and B-ZIP proteins could simultaneously bind the ETS⇔CRE motif (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). In the EMSA experiments, we used the B-ZIP protein CREB to bind the CRE motif and the ETS proteins GABPα or ETV5 to bind the ETS motif (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). Eight DNA probes were examined. Three DNA probes contained mutations in either or both motifs that abolished protein binding to the expected TFBS (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4A</a>). Five DNA probes examined the spacing between the two motifs; one probe has a deletion of 1-bp and three DNA probes have an insert of 1-bps, 2-bps, or 3-bps between the ETS and CRE motifs. CREB bound well at 10 nM (<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Ahn <em>et al.</em> 1998</a>), whereas GABPα binding was weaker, being detectable at 200 nM. When GABPα and CREB were mixed, GABPα binding was enhanced only on the DNA probe containing the ETS⇔CRE motif (compare lane 17 with lane 9 of <a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4A</a>). None of the deletion or insertion probes form the CREB|GABPα|DNA complex (lanes 18–24, <a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4A</a>). Supershift experiments demonstrated that both GABPα and CREB proteins were present in the complex formed only on the ETS⇔CRE motif containing DNA probe (<a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4A</a>), suggesting that this specific overlap of three base pairs between ETS and CRE motifs is important for binding by both GABPα and CREB. Importantly, the ETV5 member of the ETS family formed neither the CREB|ETV5|DNA complexes nor the CREB|DNA or ETV5|DNA complex forms (<a id="xref-fig-4-6" class="xref-fig" href="#F4">Figure 4B</a>). A dose-response EMSA showed that binding of one protein precludes the binding of another protein. Even when we saturated the probes with higher concentrations of ETV5 or CREB proteins, no CREB|ETV5|DNA complex was observed.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/10/1243/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="EMSA showing preferential DNA binding of the ETS protein GABP&#x3B1; and B-ZIP protein CREB to the ETS&#x21D4;CRE sequence (GCGGAAGTGACGTCA). (A) Left panel: The DNA binding domain of GABP&#x3B1; with N-terminal GST tag and the B-ZIP domain of CREB were in vitro translated alone or together, and subjected to EMSA with eight DNA probes (Table 1). Lanes 1&#x2013;8, 3 nM CREB; lanes 9&#x2013;16, 200 nM GABP&#x3B1;; lanes 17&#x2013;24, 3 nM CREB and 200 nM GABP&#x3B1;. Right panel: Supershift experiment demonstrates that the indicated CREB-GABP&#x3B1;-DNA complex contains both CREB and GABP&#x3B1;. Lanes 1, 4, and 5, in vitro translated 3 nM CREB and 200 nM GABP&#x3B1;; lane 2, no protein; lane 3, in vitro translation without protein-encoding DNA.*GABP&#x3B1;-DNA complex; &#x25CA;CREB-GABP&#x3B1;-DNA complex. (B) A dose response EMSA of the ETS protein ETV5 and B-ZIP protein CREB binding to the ETS&#x21D4;CRE sequence (GCGGAAGTGACGTCA). Increasing concentrations of ETV5 (1.3, 4, 12.5, 40, and 125 nM) or CREB (1, 3, 10, 30, and 100 nM) alone shows dose-responsive binding (lanes 2&#x2013;6 and lanes 7&#x2013;11) to the ETS&#x21D4;CRE motif. Increasing concentrations of ETV5 with fixed concentrations of CREB shows that both proteins cannot simultaneously bind to the ETS&#x21D4;CRE motif. (C) Enriched motifs generated using the peak-motifs package of Regulatory Sequence Analysis Tools (RSAT). For de novo motif detection, we used all 6442 human GABP&#x3B1; ChIP-seq peaks (Valouev et al. 2008) and all 3998 mouse CREB ChIP-seq peaks (Martianov et al. 2010) as input sequences. In CREB ChIP-seq peaks, the most enriched motif is the canonical CRE, and ETS&#x21D4;CRE motif is among the other significantly enriched motifs. In GABP&#x3B1; ChIP-seq peaks, ETS motif is the primary enriched motif, and ETS&#x21D4;ETS is among the other enriched motifs. De novo motif detection using all 2953 ETS motif&#x2013;containing regions predicted ETS&#x21D4;CRE as the best-enriched motif. De novo motif detection using 1453 commonly bound region by CREB and GABP&#x3B1; predicted ETS&#x21D4;CRE as the best-enriched motif. ETS&#x21D4;CRE&#x21D4;ETS is one of the other enriched motifs in these regions. The number of sites below each motif indicates the number of peaks that have at least one predicted motif." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-685147227" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;EMSA showing preferential DNA binding of the ETS protein GABP&#x3B1; and B-ZIP protein CREB to the ETS&#x21D4;CRE sequence (GCGGAA&lt;strong&gt;GTG&lt;/strong&gt;ACGTCA). (A) Left panel: The DNA binding domain of GABP&#x3B1; with N-terminal GST tag and the B-ZIP domain of CREB were &lt;em&gt;in vitro&lt;/em&gt; translated alone or together, and subjected to EMSA with eight DNA probes (Table 1). Lanes 1&#x2013;8, 3 nM CREB; lanes 9&#x2013;16, 200 nM GABP&#x3B1;; lanes 17&#x2013;24, 3 nM CREB and 200 nM GABP&#x3B1;. Right panel: Supershift experiment demonstrates that the indicated CREB-GABP&#x3B1;-DNA complex contains both CREB and GABP&#x3B1;. Lanes 1, 4, and 5, &lt;em&gt;in vitro&lt;/em&gt; translated 3 nM CREB and 200 nM GABP&#x3B1;; lane 2, no protein; lane 3, &lt;em&gt;in vitro&lt;/em&gt; translation without protein-encoding DNA.*GABP&#x3B1;-DNA complex; &lt;sup&gt;&#x25CA;&lt;/sup&gt;CREB-GABP&#x3B1;-DNA complex. (B) A dose response EMSA of the ETS protein ETV5 and B-ZIP protein CREB binding to the ETS&#x21D4;CRE sequence (GCGGAA&lt;strong&gt;GTG&lt;/strong&gt;ACGTCA). Increasing concentrations of ETV5 (1.3, 4, 12.5, 40, and 125 nM) or CREB (1, 3, 10, 30, and 100 nM) alone shows dose-responsive binding (lanes 2&#x2013;6 and lanes 7&#x2013;11) to the ETS&#x21D4;CRE motif. Increasing concentrations of ETV5 with fixed concentrations of CREB shows that both proteins cannot simultaneously bind to the ETS&#x21D4;CRE motif. (C) Enriched motifs generated using the peak-motifs package of Regulatory Sequence Analysis Tools (RSAT). For &lt;em&gt;de novo&lt;/em&gt; motif detection, we used all 6442 human GABP&#x3B1; ChIP-seq peaks (Valouev &lt;em&gt;et al.&lt;/em&gt; 2008) and all 3998 mouse CREB ChIP-seq peaks (Martianov &lt;em&gt;et al.&lt;/em&gt; 2010) as input sequences. In CREB ChIP-seq peaks, the most enriched motif is the canonical CRE, and ETS&#x21D4;CRE motif is among the other significantly enriched motifs. In GABP&#x3B1; ChIP-seq peaks, ETS motif is the primary enriched motif, and ETS&#x21D4;ETS is among the other enriched motifs. &lt;em&gt;De novo&lt;/em&gt; motif detection using all 2953 ETS motif&#x2013;containing regions predicted ETS&#x21D4;CRE as the best-enriched motif. &lt;em&gt;De novo&lt;/em&gt; motif detection using 1453 commonly bound region by CREB and GABP&#x3B1; predicted ETS&#x21D4;CRE as the best-enriched motif. ETS&#x21D4;CRE&#x21D4;ETS is one of the other enriched motifs in these regions. The number of sites below each motif indicates the number of peaks that have at least one predicted motif.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/10/1243/F4.medium.gif" width="331" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4&#x2002;" src="https://www.g3journal.org/content/ggg/2/10/1243/F4.medium.gif" width="331" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/10/1243/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/10/1243/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462023" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4 </span> <p id="p-27" class="first-child">EMSA showing preferential DNA binding of the ETS protein GABPα and B-ZIP protein CREB to the ETS⇔CRE sequence (GCGGAA<strong>GTG</strong>ACGTCA). (A) Left panel: The DNA binding domain of GABPα with N-terminal GST tag and the B-ZIP domain of CREB were <em>in vitro</em> translated alone or together, and subjected to EMSA with eight DNA probes (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>). Lanes 1–8, 3 nM CREB; lanes 9–16, 200 nM GABPα; lanes 17–24, 3 nM CREB and 200 nM GABPα. Right panel: Supershift experiment demonstrates that the indicated CREB-GABPα-DNA complex contains both CREB and GABPα. Lanes 1, 4, and 5, <em>in vitro</em> translated 3 nM CREB and 200 nM GABPα; lane 2, no protein; lane 3, <em>in vitro</em> translation without protein-encoding DNA.*GABPα-DNA complex; <sup>◊</sup>CREB-GABPα-DNA complex. (B) A dose response EMSA of the ETS protein ETV5 and B-ZIP protein CREB binding to the ETS⇔CRE sequence (GCGGAA<strong>GTG</strong>ACGTCA). Increasing concentrations of ETV5 (1.3, 4, 12.5, 40, and 125 nM) or CREB (1, 3, 10, 30, and 100 nM) alone shows dose-responsive binding (lanes 2–6 and lanes 7–11) to the ETS⇔CRE motif. Increasing concentrations of ETV5 with fixed concentrations of CREB shows that both proteins cannot simultaneously bind to the ETS⇔CRE motif. (C) Enriched motifs generated using the peak-motifs package of Regulatory Sequence Analysis Tools (RSAT). For <em>de novo</em> motif detection, we used all 6442 human GABPα ChIP-seq peaks (<a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">Valouev <em>et al.</em> 2008</a>) and all 3998 mouse CREB ChIP-seq peaks (<a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Martianov <em>et al.</em> 2010</a>) as input sequences. In CREB ChIP-seq peaks, the most enriched motif is the canonical CRE, and ETS⇔CRE motif is among the other significantly enriched motifs. In GABPα ChIP-seq peaks, ETS motif is the primary enriched motif, and ETS⇔ETS is among the other enriched motifs. <em>De novo</em> motif detection using all 2953 ETS motif–containing regions predicted ETS⇔CRE as the best-enriched motif. <em>De novo</em> motif detection using 1453 commonly bound region by CREB and GABPα predicted ETS⇔CRE as the best-enriched motif. ETS⇔CRE⇔ETS is one of the other enriched motifs in these regions. The number of sites below each motif indicates the number of peaks that have at least one predicted motif.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-14" class="subsection"><h3>Motif detection in CREB and GABPα ChIP-seq peaks</h3><p id="p-28">We examined published ChIP-seq data sets for GABPα (<a id="xref-ref-49-3" class="xref-bibr" href="#ref-49">Valouev <em>et al.</em> 2008</a>) in humans and CREB in mouse (<a id="xref-ref-28-3" class="xref-bibr" href="#ref-28">Martianov <em>et al.</em> 2010</a>) to determine whether the ETS⇔CRE motif is enriched in the ChIP-seq peaks. The peak-motif package (<a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">Thomas-Chollier <em>et al.</em> 2011</a>) of RSAT was used for evaluating the enriched motifs in these ChIP-seq regions. Using all CREB peak regions, the peak-motif identified the overlapping ETS⇔CRE motif, which is more enriched than the canonical CRE motif (<a id="xref-fig-4-7" class="xref-fig" href="#F4">Figure 4C</a> and <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>). When we used only the GABPα ChIP-seq peaks for <em>de novo</em> motif detection, we identified the canonical ETS and the ETS⇔ETS motif, but not the ETS⇔CRE motif. However, when we examined the 2953 peaks that contain the canonical ETS motif, we detected that the ETS⇔CRE motif is the best-enriched motif (<a id="xref-fig-4-8" class="xref-fig" href="#F4">Figure 4C</a>).</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/461928/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/461928/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Enrichment of ETS, CRE and ETS⇔CRE motifs in CREB and GABPα ChIP-seq peaks</span><div class="sb-div caption-clear"></div></div></div><p id="p-29">An additional analysis used the GABPα ChIP-seq data already described from human Jurkat cell line and CREB ChIP-chip data from human HEK293T cells (<a id="xref-ref-55-2" class="xref-bibr" href="#ref-55">Zhang <em>et al.</em> 2005</a>). One thousand four hundred sixty-three (1463) peaks are common between CREB and GABPα binding sites. <em>De novo</em> motif detection using these regions by peak-motif detected ETS⇔CRE motif as the best-enriched motif (<a id="xref-fig-4-9" class="xref-fig" href="#F4">Figure 4C</a>). Interestingly, among the other enriched motifs, we observed a palindromic ETS⇔CRE⇔ETS motif, in which the second ETS canonical motif is in the opposite strand (<a id="xref-fig-4-10" class="xref-fig" href="#F4">Figure 4C</a>), suggesting the biological significance of the coordinated regulation of ETS and CREB in regulating the gene expression. The promoters with ETS⇔CRE, obtained from the commonly bound regions by CREB and GABPα, are significantly enriched for the GO terms of proteolysis involved in macromolecule catabolic process, RNA processing, and cellular response to stress (<a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>). However, the MEME-ChIP package (<a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Machanick &amp; Bailey 2011</a>) of the MEME Suite failed to detect the ETS⇔CRE motif as an enriched motif in any data set.</p><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/461900/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/461900/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">Enriched GO terms of genes commonly bound by CREB and GABPα with ETS⇔CRE motifs</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-15" class="subsection"><h3>Length of ETS⇔CRE motif</h3><p id="p-30">Two strategies were used to evaluate the length of the ETS⇔CRE motif: (i) enrichment in 8000 housekeeping DNase I hypersensitive sites (DHS) (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Sabo <em>et al.</em> 2004</a>) and (ii) conservation in mammalian genomes.</p><p id="p-31">We extended the ETS motif 8-mer CGGAAGTG toward the CRE (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5A</a>) and counted the occurrences in the genome and known regulatory regions, including annotated promoters, proximal promoters, CpG islands, housekeeping DHS, and all DHS identified in 45 cell types (<a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Sabo <em>et al.</em> 2004</a>) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.004002/-/DC1/TableS2.pdf">Table S2</a>). The housekeeping DHS are defined as the DNase hypersensitive regions that are present in all 45 cell types (<a id="xref-ref-43-3" class="xref-bibr" href="#ref-43">Sabo <em>et al.</em> 2004</a>). The ETS 8-mer CGGAAGTG occurs 16,846 times in the genome and 6% of them are in housekeeping DHS. Similar results were observed when the motif is extended to the 9-mer (CGGAAGTGA) and 10-mer (CGGAAGTGAC). A transition occurs with the 11-mer (CGGAAGTGACG), with 60% occurring in housekeeping DHS and 83% occurring in known regulatory regions (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.004002/-/DC1/TableS2.pdf">Table S2</a>). The 11-mer contains two CG dinucleotides, which are rare outside of regulatory regions.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/10/1243/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="(A) Preferential localization in housekeeping DHS compared with the genome for different length of ETS&#x21D4;CRE sequences. The ETS (CGGAAGTG) and CRE (AAGTGACG) 8-mers were lengthened toward the indicated arrows, and for each bp extension, preferential localization in housekeeping DHS are calculated. A jump in localization of ETS (CGGAAGTG) occurs when the second CG dinucleotide is included, which creates the 11-mer CGGAAGTGACG. The ETS 8-mer CGGAAGTG occurs 16,846 times in the genome and 1073 times in housekeeping DHS, a ratio of &#x223C;8%. The ratio in housekeeping DHS of 8-mers (CGGAAGTN) with a different final nucleotide are shown as a colored dot (G = yellow, A = green, T = red, C = blue). The ETS 9-mer CGGAAGTGA occurs 343 times in housekeeping DHS with a similar enrichment in housekeeping DHS as the 8-mer. When the sequence is extended to the 11-mer CGGAAGTGACG, enrichment in housekeeping DHS jumps to 60%. If the final G in the 11-mer is changed to the three other nucleotides, enrichment in housekeeping DHS is only 10%. When the ETS&#x21D4;CRE motif is extended to a 12-mer and beyond, enrichment in housekeeping DHS remains constant. When the ETS&#x21D4;CRE motif is extended from the CRE side toward the ETS side, a jump in localization in housekeeping DHS occurs when the AAGTGACG 8-mer is extended to the CGGAAGTGACG 11-mer. (B) Conservation or phyloP score in 30 mammals for the CRE 8-mer. (C) phyloP score for the ETS 8-mer. (D) phyloP score for the ETS&#x21D4;CRE 11-mer." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-685147227" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;(A) Preferential localization in housekeeping DHS compared with the genome for different length of ETS&#x21D4;CRE sequences. The ETS (CGGAAGTG) and CRE (AAGTGACG) 8-mers were lengthened toward the indicated arrows, and for each bp extension, preferential localization in housekeeping DHS are calculated. A jump in localization of ETS (CGGAAGTG) occurs when the second CG dinucleotide is included, which creates the 11-mer CGGAAGTGACG. The ETS 8-mer CGGAAGTG occurs 16,846 times in the genome and 1073 times in housekeeping DHS, a ratio of &#x223C;8%. The ratio in housekeeping DHS of 8-mers (CGGAAGTN) with a different final nucleotide are shown as a colored dot (G = yellow, A = green, T = red, C = blue). The ETS 9-mer CGGAAGTGA occurs 343 times in housekeeping DHS with a similar enrichment in housekeeping DHS as the 8-mer. When the sequence is extended to the 11-mer CGGAAGTGACG, enrichment in housekeeping DHS jumps to 60%. If the final G in the 11-mer is changed to the three other nucleotides, enrichment in housekeeping DHS is only 10%. When the ETS&#x21D4;CRE motif is extended to a 12-mer and beyond, enrichment in housekeeping DHS remains constant. When the ETS&#x21D4;CRE motif is extended from the CRE side toward the ETS side, a jump in localization in housekeeping DHS occurs when the AAGTGACG 8-mer is extended to the CGGAAGTGACG 11-mer. (B) Conservation or phyloP score in 30 mammals for the CRE 8-mer. (C) phyloP score for the ETS 8-mer. (D) phyloP score for the ETS&#x21D4;CRE 11-mer.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/10/1243/F5.medium.gif" width="389" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5&#x2002;" src="https://www.g3journal.org/content/ggg/2/10/1243/F5.medium.gif" width="389" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/10/1243/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/10/1243/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461999" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5 </span> <p id="p-32" class="first-child">(A) Preferential localization in housekeeping DHS compared with the genome for different length of ETS⇔CRE sequences. The ETS (CGGAAGTG) and CRE (AAGTGACG) 8-mers were lengthened toward the indicated arrows, and for each bp extension, preferential localization in housekeeping DHS are calculated. A jump in localization of ETS (CGGAAGTG) occurs when the second CG dinucleotide is included, which creates the 11-mer CGGAAGTGACG. The ETS 8-mer CGGAAGTG occurs 16,846 times in the genome and 1073 times in housekeeping DHS, a ratio of ∼8%. The ratio in housekeeping DHS of 8-mers (CGGAAGTN) with a different final nucleotide are shown as a colored dot (G = yellow, A = green, T = red, C = blue). The ETS 9-mer CGGAAGTGA occurs 343 times in housekeeping DHS with a similar enrichment in housekeeping DHS as the 8-mer. When the sequence is extended to the 11-mer CGGAAGTGACG, enrichment in housekeeping DHS jumps to 60%. If the final G in the 11-mer is changed to the three other nucleotides, enrichment in housekeeping DHS is only 10%. When the ETS⇔CRE motif is extended to a 12-mer and beyond, enrichment in housekeeping DHS remains constant. When the ETS⇔CRE motif is extended from the CRE side toward the ETS side, a jump in localization in housekeeping DHS occurs when the AAGTGACG 8-mer is extended to the CGGAAGTGACG 11-mer. (B) Conservation or phyloP score in 30 mammals for the CRE 8-mer. (C) phyloP score for the ETS 8-mer. (D) phyloP score for the ETS⇔CRE 11-mer.</p><div class="sb-div caption-clear"></div></div></div><p id="p-33">It is important to note that the 11-mer CGGAAGTGACT can represent the overlapping of an ETS motif and an AP1 motif (TGA<sup>C</sup>/<sub>G</sub>TCA) to create the ETS⇔AP1. The ETS⇔AP1 motif may be cooperatively bound by an ETS protein and B-ZIP proteins that bind the AP1 motif. This sequence does not occur in housekeeping DHS, but it is enriched in tissue-specific DHS (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>) as observed previously (<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Hollenhorst <em>et al.</em> 2011b</a>). When the motif is extended to a 12-mer, localization in housekeeping DHS does not increase but the occurrence decreases, indicating that the 11-mer is the core of longer and diverse ETS⇔CRE motifs (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5A</a>).</p><p id="p-34">When the motif is extended from the CRE side toward the ETS motif, we again observe that localization in housekeeping DHS jumps to its maximal value when the motif is extended to the second CG and forms the 11-mer CGGAA<strong>GTG</strong>ACG. This suggests that the 226 ETS⇔CRE 11-mers in the genome contain different versions of the longer ETS⇔CRE 16-mers that may have distinct functions when they are bound by different combinations of ETS and B-ZIP family members.</p></div><div id="sec-16" class="subsection"><h3>Conservation of the ETS⇔CRE motif in mammals</h3><p id="p-35">The conservation of the ETS⇔CRE motif was examined in 36 mammalian genomes (<a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Pollard <em>et al.</em> 2010</a>). Initially, we examined the PhyloP signature for the ETS (CGGAAGTG) and CRE (TGACGTCA) 8-mers. Both PhyloP signatures show conservation (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5, B and C</a>), except for the CG that has negative PhyloP values. We presume this simply reflects the chemical deamination of the C in the CG dinucleotide when it is methylated, a well-known hypermutable process that is not directly modeled in PhyloP. In contrast, in the ETS⇔CRE 11-mer (GGAA<strong>GTG</strong>ACG), all nucleotides, including both CG, are “highly” conserved, having scores four times larger than either the ETS or CRE motifs (<a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5D</a>). Conservation extends 1-bp beyond the CG on the ETS (5′) side of the motif to either a C or G, which is known to affect DNA binding of ETS family members (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Wei <em>et al.</em> 2010</a>). Beyond the CG on the CRE (3′) side to the ETS⇔CRE motif, the 4-bps (TCAC) region, which is the second half of the CRE motif, is not conserved. Provocatively, these nucleotides actually have negative PhyloP values and as here it does not have deamination effect of CG dinucleotides, it suggests that the sequences bound by the second monomer of the B-ZIP dimer in this context are evolving faster than neutral (<a id="xref-ref-39-3" class="xref-bibr" href="#ref-39">Pollard <em>et al.</em> 2010</a>).</p></div><div id="sec-17" class="subsection"><h3>1-bp variants of the ETS⇔CRE 11-mer</h3><p id="p-36">We examined whether 1-bp variants of the ETS⇔CRE 11-mer are also enriched in housekeeping DHS (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6, A–D</a>). Of the 147 occurrences, 51 (35%) of the most abundant 1-bp variant (CGGAAGTG<em>G</em>CG) are in housekeeping DHS. Two additional variants (CGGA<em>C</em>GTGACG and CGGAAGTG<em>C</em>CG) are abundant and enriched in housekeeping promoters, suggesting that they may also be functional. The GGA in the core of the ETS motif is critical for the sequence-specific binding (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Graves &amp; Petersen 1998</a>) and shows very little variability in housekeeping DHS, suggesting that there are virtually no occurrences of the crippled ETS⇔CRE motif in regulatory regions. In the genome, all 1-bp variants that do not disrupt the CG are less abundant than the ETS⇔CRE 11-mer. In contrast, 1-bp variants that do disrupt either of the two CG are typically more abundant than the ETS⇔CRE, highlighting the profound effect of the CG dinucleotide on the occurrence of a DNA sequence in the genome. A molecular model of the ETS⇔CRE 16-mer bound by ETS and CREB is color-coded to visualize each nucleotide (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6E</a>). Potentially, the abundant 1-bp nucleotide variants of the ETS⇔CRE motif in housekeeping promoters are bound by different combinations of ETS and B-ZIP proteins.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/10/1243/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="(A) Occurrences of the ETS&#x21D4;CRE 11-mer CGGAAGTGACG and all 1-bp variants in housekeeping DHS vs. enrichment in housekeeping DHS compared with the genome. (B) Histogram showing abundance of the ETS&#x21D4;CRE 11-mer CGGAAGTGACG and 1-bp variants in housekeeping DHS. (C) Table showing occurrences of 1-bp variances of the ETS&#x21D4;CRE 11-mer CGGAAGTGACG in the genome and housekeeping DHS. The numbers highlighted in yellow are the consensus ETS&#x21D4;CRE 11-mer. (D) Graphical presentation of the 1-bp variants of ETS&#x21D4;CRE 11-mer in housekeeping DHS. (E) Molecular model of the ETS&#x21D4;CRE motif with the nucleotides in color to highlight which parts of the structure are conserved and variable." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-685147227" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;(A) Occurrences of the ETS&#x21D4;CRE 11-mer CGGAA&lt;strong&gt;GTG&lt;/strong&gt;ACG and all 1-bp variants in housekeeping DHS &lt;em&gt;vs.&lt;/em&gt; enrichment in housekeeping DHS compared with the genome. (B) Histogram showing abundance of the ETS&#x21D4;CRE 11-mer CGGAA&lt;strong&gt;GTG&lt;/strong&gt;ACG and 1-bp variants in housekeeping DHS. (C) Table showing occurrences of 1-bp variances of the ETS&#x21D4;CRE 11-mer CGGAAGTGACG in the genome and housekeeping DHS. The numbers highlighted in yellow are the consensus ETS&#x21D4;CRE 11-mer. (D) Graphical presentation of the 1-bp variants of ETS&#x21D4;CRE 11-mer in housekeeping DHS. (E) Molecular model of the ETS&#x21D4;CRE motif with the nucleotides in color to highlight which parts of the structure are conserved and variable.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/10/1243/F6.medium.gif" width="240" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6&#x2002;" src="https://www.g3journal.org/content/ggg/2/10/1243/F6.medium.gif" width="240" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/10/1243/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/10/1243/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462007" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6 </span> <p id="p-37" class="first-child">(A) Occurrences of the ETS⇔CRE 11-mer CGGAA<strong>GTG</strong>ACG and all 1-bp variants in housekeeping DHS <em>vs.</em> enrichment in housekeeping DHS compared with the genome. (B) Histogram showing abundance of the ETS⇔CRE 11-mer CGGAA<strong>GTG</strong>ACG and 1-bp variants in housekeeping DHS. (C) Table showing occurrences of 1-bp variances of the ETS⇔CRE 11-mer CGGAAGTGACG in the genome and housekeeping DHS. The numbers highlighted in yellow are the consensus ETS⇔CRE 11-mer. (D) Graphical presentation of the 1-bp variants of ETS⇔CRE 11-mer in housekeeping DHS. (E) Molecular model of the ETS⇔CRE motif with the nucleotides in color to highlight which parts of the structure are conserved and variable.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-18" class="subsection"><h3>Four abundant ETS⇔CRE 13-mers <strong>(<sup>C</sup>/<sub>G</sub>CGGAA</strong>GTG<strong>ACG<sup>T</sup>/<sub>C</sub>)</strong></h3><p id="p-38">The abundance of longer versions of the ETS⇔CRE 11-mer in the genome and regulatory regions was evaluated (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.004002/-/DC1/FigureS3.pdf">Figure S3</a>C). We initially focused on 16-mers, the potential length of the ETS⇔CRE motif. Of the 226 11-mers in the genome, 171 different 16-mers occur, and the most abundant 16-mer (CCGGAAGTGACGCGAG) occurs seven times. The canonical motif CCGGAA<strong>GTG</strong>ACGTCAC occurs three times in the genome. The alignment of ETS⇔CRE 11-mers, including surrounding DNA sequences, identified four abundant ETS⇔CRE 13-mers (<sup>C</sup>/<sub>G</sub>CGGAA<strong>GTG</strong>ACG<sup>T</sup>/<sub>C</sub>) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.004002/-/DC1/FigureS3.pdf">Figure S3</a>C), representing 70% of all ETS⇔CRE 11-mers (<a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7, A and B</a>). Each 13-mer correlated with different GO terms, suggesting distinct functions (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.004002/-/DC1/TableS3.pdf">Table S3</a>). The nucleotide before the CG in the ETS motif is either G or C, and these are known to be bound by different ETS family members (<a id="xref-ref-51-2" class="xref-bibr" href="#ref-51">Wei <em>et al.</em> 2010</a>). The nucleotide after the central CG in the CRE is typically a pyrimidine, T and C. They are 5-fold more abundant than the purines G and A (<a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>). The T and C in this position are optimal for binding the B-ZIP proteins CREB and C/EBP, respectively (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Johnson 1993</a>). Each of the four ETS⇔CRE 13-mers is expected to be optimally bound by a specific combination of ETS monomers and B-ZIP dimers.</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/10/1243/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="(A) Abundance of 4 ETS&#x21D4;CRE 13-mers (C/GCGGAAGTGACGT/C) and 1-bp variants in housekeeping DHS vs. percentage of occurrences in housekeeping DHS compared with the genome. All N-CG-N7-CG-13N-mers are shown. The four abundant ETS&#x21D4;CRE 13-mers (C/GCGGAAGTGACGT/C) are shown in red. (B) Histogram of occurrences of the ETS&#x21D4;CRE 13-mers C/GCGGAAGTGACGT/C and all 1-bp variances in housekeeping DHS. (C) Pie chart representation of the occurrence of the dinucleotides at the end of the CRE motif TGACGTNN that occurs 2046 times in proximal promoters (&#x2212;200-bps to +60-bps). (D) Pie chart representation of the occurrence of the dinucleotides at the end of the ETS&#x21D4;CRE motif CGGAAGTGACGTNN. (E) Preferential occurrence in promoters compared to the genome for all pairs of CG separated by 0-bps to 9-bps [CG-(0-9)-CG]. (F) Same as (E), but sequences with an internal CG are excluded. The one sequence that is abundant primarily in promoters is the ETS&#x21D4;CRE motif." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-685147227" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;(A) Abundance of 4 ETS&#x21D4;CRE 13-mers (&lt;sup&gt;C&lt;/sup&gt;/&lt;sub&gt;G&lt;/sub&gt;CGGAA&lt;strong&gt;GTG&lt;/strong&gt;ACG&lt;sup&gt;T&lt;/sup&gt;/&lt;sub&gt;C&lt;/sub&gt;) and 1-bp variants in housekeeping DHS &lt;em&gt;vs.&lt;/em&gt; percentage of occurrences in housekeeping DHS compared with the genome. All N-CG-N&lt;sub&gt;7&lt;/sub&gt;-CG-&lt;sup&gt;13&lt;/sup&gt;N-mers are shown. The four abundant ETS&#x21D4;CRE 13-mers (&lt;sup&gt;C&lt;/sup&gt;/&lt;sub&gt;G&lt;/sub&gt;CGGAA&lt;strong&gt;GTG&lt;/strong&gt;ACG&lt;sup&gt;T&lt;/sup&gt;/&lt;sub&gt;C&lt;/sub&gt;) are shown in red. (B) Histogram of occurrences of the ETS&#x21D4;CRE 13-mers &lt;sup&gt;C&lt;/sup&gt;/&lt;sub&gt;G&lt;/sub&gt;CGGAA&lt;strong&gt;GTG&lt;/strong&gt;ACG&lt;sup&gt;T&lt;/sup&gt;/&lt;sub&gt;C&lt;/sub&gt; and all 1-bp variances in housekeeping DHS. (C) Pie chart representation of the occurrence of the dinucleotides at the end of the CRE motif TGACGTNN that occurs 2046 times in proximal promoters (&#x2212;200-bps to +60-bps). (D) Pie chart representation of the occurrence of the dinucleotides at the end of the ETS&#x21D4;CRE motif CGGAAGTGACGTNN. (E) Preferential occurrence in promoters compared to the genome for all pairs of CG separated by 0-bps to 9-bps [CG-&lt;sub&gt;(0-9)&lt;/sub&gt;-CG]. (F) Same as (E), but sequences with an internal CG are excluded. The one sequence that is abundant primarily in promoters is the ETS&#x21D4;CRE motif.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/10/1243/F7.medium.gif" width="322" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7&#x2002;" src="https://www.g3journal.org/content/ggg/2/10/1243/F7.medium.gif" width="322" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/10/1243/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/10/1243/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461886" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7 </span> <p id="p-39" class="first-child">(A) Abundance of 4 ETS⇔CRE 13-mers (<sup>C</sup>/<sub>G</sub>CGGAA<strong>GTG</strong>ACG<sup>T</sup>/<sub>C</sub>) and 1-bp variants in housekeeping DHS <em>vs.</em> percentage of occurrences in housekeeping DHS compared with the genome. All N-CG-N<sub>7</sub>-CG-<sup>13</sup>N-mers are shown. The four abundant ETS⇔CRE 13-mers (<sup>C</sup>/<sub>G</sub>CGGAA<strong>GTG</strong>ACG<sup>T</sup>/<sub>C</sub>) are shown in red. (B) Histogram of occurrences of the ETS⇔CRE 13-mers <sup>C</sup>/<sub>G</sub>CGGAA<strong>GTG</strong>ACG<sup>T</sup>/<sub>C</sub> and all 1-bp variances in housekeeping DHS. (C) Pie chart representation of the occurrence of the dinucleotides at the end of the CRE motif TGACGTNN that occurs 2046 times in proximal promoters (−200-bps to +60-bps). (D) Pie chart representation of the occurrence of the dinucleotides at the end of the ETS⇔CRE motif CGGAAGTGACGTNN. (E) Preferential occurrence in promoters compared to the genome for all pairs of CG separated by 0-bps to 9-bps [CG-<sub>(0-9)</sub>-CG]. (F) Same as (E), but sequences with an internal CG are excluded. The one sequence that is abundant primarily in promoters is the ETS⇔CRE motif.</p><div class="sb-div caption-clear"></div></div></div><p id="p-40">The dinucleotides following the CRE 6-mer TGACGT-N<sub>2</sub> in proximal promoters are enriched only for CA dinucleotide, which produces the canonical CRE 8-mer TGACGTCA (<a id="xref-fig-7-2" class="xref-fig" href="#F7">Figure 7C</a>). In contrast, the dinucleotides following the ETS⇔CRE 12-mer CGGAAGTGACGT are also enriched for AN dinucleotides, suggesting that the CRE and the ETS⇔CRE motifs in promoters are bound by different B-ZIP proteins (<a id="xref-fig-7-3" class="xref-fig" href="#F7">Figure 7D</a>).</p></div><div id="sec-19" class="subsection"><h3>Localization of pairs of CG in DHS</h3><p id="p-41">In the ETS⇔CRE motif, the two CG are separated by 7-bps (CG-N<sub>7</sub>-CG). To identify whether additional pairs of CG preferentially occur in promoters, we counted in the whole genome the occurrence of sequences containing a pair of CG separated by 0-bps to 9-bps (CG-N<sub>0-9</sub>-CG) and determined what fraction are in housekeeping DHS. The ETS⇔CRE motif stands out among all other sequences containing pairs of CG, being abundant and primarily in promoters (<a id="xref-fig-7-4" class="xref-fig" href="#F7">Figure 7, E and F</a>).</p></div><div id="sec-20" class="subsection"><h3>CG methylation status of the ETS⇔CRE motif in two mouse primary cells</h3><p id="p-42">Methylation of the CG dinucleotide in canonical ETS and CRE motifs inhibits binding of both ETS and CREB proteins (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Iguchi-Ariga &amp; Schaffner 1989</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Umezawa <em>et al.</em> 1997</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Rozenberg <em>et al.</em> 2008</a>). An important feature of the ETS⇔CRE motif is the presence of two CG that can be methylated. We used two mouse methylomes at 100X coverage for newborn mouse dermal fibroblasts and 45X coverage for primary keratinocytes. The four ETS⇔CRE 13-mers have different methylation properties (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.004002/-/DC1/TableS4.pdf">Table S4</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.004002/-/DC1/FigureS4.pdf">Figure S4</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.004002/-/DC1/FigureS5.pdf">Figure S5</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.004002/-/DC1/FigureS6.pdf">Figure S6</a>). All 21 occurrences of the GCGGAAGTGACGT 13-mer are unmethylated on both CG dinucleotides in dermal fibroblasts and keratinocytes, suggesting that they are in functional regions of the genome. Of the 45 occurrences of the more abundant 13-mer CCGGAAGTGACGT, 33 are unmethylated in both cells (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.004002/-/DC1/FigureS4.pdf">Figure S4</a>C, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.004002/-/DC1/FigureS5.pdf">Figure S5</a>A, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.004002/-/DC1/FigureS6.pdf">Figure S6</a>A).</p><p id="p-43">Not all 13-mers with two CG dinucleotides separated by 7-bp are unmethylated. Only 10% of CACGCACACACCG is unmethylated (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.004002/-/DC1/FigureS4.pdf">Figure S4</a>G, <a id="xref-fig-5-5" class="xref-fig" href="#F5">Figures 5E</a> and <a id="xref-fig-6-3" class="xref-fig" href="#F6">6E</a>). Comparing two methylome data for these motifs shows that unmethylated 13-mer motifs are common and generally unmethylated in both cell types (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.004002/-/DC1/FigureS6.pdf">Figure S6</a>, A–D) and that these unmethylated ETS⇔CRE sequences are mainly enriched in promoters (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.004002/-/DC1/TableS4.pdf">Table S4</a>), lending support to the suggestion that every occurrence of an unmethylated version of the ETS⇔CRE motif is biologically important.</p></div></div><div class="section discussion" id="sec-21"><h2 class="">Discussion</h2><p id="p-44">We determined the distribution in human promoters of split DNA 8-mers consisting of a pair of 4-mers separated by 1-bp to 30-bps. A striking result is that few split 8-mers with insert length of 5-bps or greater (X<sub>4</sub>-N<sub>5-30</sub>-X<sub>4</sub>) localize in proximal promoters. This is in sharp contrast to Drosophila promoters, in which many split 8-mers with a 20-bp to 30-bp insert length (X<sub>4</sub>-N<sub>20-30</sub>-X<sub>4</sub>) localize in proximal promoters (<a id="xref-ref-50-9" class="xref-bibr" href="#ref-50">Vinson <em>et al.</em> 2011</a>). We examined split 8-mers in human promoters and identified pairs of 4-mers that localized at a specific insert length and not others. This article focused on the ETS motif (<sup>C</sup>/<sub>G</sub>CGGAA<strong>GTG)</strong> precisely overlapping with a CRE motif (<strong>GTG</strong>ACGTCAC) to create a composite site, the ETS⇔CRE motif (<sup>C</sup>/<sub>G</sub>CGGAA<strong>GTG</strong>ACGTCAC). The trinucleotide <strong>GTG</strong> is common in the two TFBS, being the 3′ end of the ETS motif and 5′ end of the palindromic CRE motif. Molecular modeling using X-ray structures of ETS and B-ZIP proteins binding the ETS⇔CRE motif suggests that the ETS monomer and B-ZIP dimer can bind the overlapping TFBS without any protein-protein clashes. Instead of ETS and B-ZIP proteins competing for binding the ETS⇔CRE motif, the ETS protein GABPα and the B-ZIP protein CREB preferentially bind the ETS⇔CRE motif only when the <strong>GTG</strong> trinucleotide overlaps. In contrast, the ETS protein ETV5 competes with CREB to bind the ETS⇔CRE motif. <em>De novo</em> enriched motif detection using the <em>in vivo</em> CREB and GABPα ChIP-seq binding regions identified the ETS⇔CRE motif along with the canonical CRE and ETS motifs, suggesting an <em>in vivo</em> function for the motif. Additionally, the conservation of the ETS⇔CRE motif is signifying its biological function (<a id="xref-ref-54-5" class="xref-bibr" href="#ref-54">Xie <em>et al.</em> 2005</a>; <a id="xref-ref-39-4" class="xref-bibr" href="#ref-39">Pollard <em>et al.</em> 2010</a>).</p><p id="p-45">The ETS domain has been shown to interact with several different DNA binding proteins to bind sequences containing chimeric aspects of each TFBS (<a id="xref-ref-17-3" class="xref-bibr" href="#ref-17">Hollenhorst <em>et al.</em> 2011b</a>). The ETS protein GABPα initially was observed interacting with GABPβ to bind a chimeric sequence (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Batchelor <em>et al.</em> 1998</a>). ETS was subsequently shown to interact with additional proteins. The forkhead proteins interact at the 5′ end of the ETS motif (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">De Val <em>et al.</em> 2008</a>), whereas SRF, PAX, and potentially CREB interact at the 3′ end of the ETS motif (<a id="xref-ref-17-4" class="xref-bibr" href="#ref-17">Hollenhorst <em>et al.</em> 2011b</a>). Several of these interactions have been identified by examining tissue-specific enhancer sequences (<a id="xref-ref-17-5" class="xref-bibr" href="#ref-17">Hollenhorst <em>et al.</em> 2011b</a>). The cytokine, RANTES (regulated upon activation, normal T cell expressed) is induced by LPS through binding in promoters by ATF and Jun proteins to a composite site containing non-overlapping ETS and CRE motifs (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Boehlk <em>et al.</em> 2000</a>).</p><p id="p-46">ETS and CRE motifs co-occur in proximal promoters (<a id="xref-ref-11-10" class="xref-bibr" href="#ref-11">FitzGerald <em>et al.</em> 2004</a>). Cooperative DNA binding by GABPα and CREB to adjacent ETS and CRE sites separated by various distances up to 15-bps has been reported (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Sawada <em>et al.</em> 1999</a>). The cooperative binding is mapped to the non-DNA binding region of GABPα, suggesting that cooperativity is via protein-protein interactions. These investigators did not observe that the two motifs needed to be precisely aligned relative to each other for cooperative binding. These results are in sharp contrast to what we observed; the precise overlap produces enhanced GABPα and CREB binding, suggesting that the cooperative binding we observed between the ETS and CREB DNA binding domains is distinct from the cooperative binding observed when full-length proteins are examined. The ETS and CRE motifs at different spacing than the observed ETS⇔CRE motif may be preferentially bound by different combinations of ETS and B-ZIP proteins and may have specific functions in regulating gene expression. Oncogenic ETS family members in prostate cancer localize at ETS⇔AP1 motifs that have the same overlap (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Hollenhorst <em>et al.</em> 2011a</a>) observed in the ETS⇔CRE motif. The AP1 or TRE 7-mer (TGA<sup>C</sup>/<sub>G</sub>TCA) is a 1-bp deletion at the center of the CRE, disrupting the CG dinucleotide. Recently, the ETS and CRE motifs were observed to co-occur in ChIP-seq data sets with a spacing of 1-bp to 2-bp (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Whitington <em>et al.</em> 2011</a>), whereas we highlight the ETS⇔CRE motif at a precise spacing with unique biochemical properties.</p><p id="p-47">Overlapping protein binding is observed in the enhanceosome where the ATF-2/c-Jun heterodimer binds to the same DNA base pairs as the IRF-3 protein. Again, there are no protein-protein interactions (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Panne <em>et al.</em> 2004</a>, <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">2007</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Panne 2008</a>); instead, it appears that the cooperative binding of these three polypeptides is via allosteric changes to the DNA. This is similar to what may occur when GABPα and CREB preferentially bind the ETS⇔CRE motif.</p><p id="p-48">Recently, it was suggested that a fundamental difference between prokaryotic and eukaryotic systems is that eukaryotic systems have short TFBS that proteins do not recognize with sufficient specificity to bind to cognate sites exclusively (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Wunderlich &amp; Mirny 2009</a>) and need to cooperate with other TF to displace a nucleosome and become functional (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Polach &amp; Widom 1996</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Mirny 2010</a>). The overlap of two TFBS as observed in the ETS⇔CRE motif creates a long DNA sequences that are generally rare in mammalian genomes and could thus function like a prokaryotic system in which each occurrence is functional.</p><p id="p-49">An alternative method to create specificity in vertebrate genomes is to have two TFBS that only need to be within 150-bps of each other and function together because they compete with nucleosomes for binding (<a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Polach &amp; Widom 1996</a>; <a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Mirny 2010</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Biddie <em>et al.</em> 2011</a>). It appears that both mechanisms operate in mammalian genomes. An advantage of the overlapping TFBS is that it allows for cooperative binding between specific members of each TF family, thus increasing specificity. This is absent in the model of two TF independently binding to DNA to displace a nucleosome. The nucleosome displacement mechanism allows different TF to act cooperatively, and it allows selection of which family member is functioning.</p><p id="p-50">We have taken a DNA-centric perspective to evaluate which DNA sequences are important, eschewing the common practice embodied in the use of position weight matrices (PWM), of averaging two or more DNA sequences to create a logo or hybrid sequence. An inherent issue with the DNA-centric perspective is to know the length of the DNA sequence. An upper bound to the length of a DNA sequence is when it becomes unique in the genome, instead of having thousands of occurrences in which only a subset is functional. Vertebrate genomes are not big enough to accommodate all possible 16-mers. The ETS⇔CRE 16-mer is long enough so that random occurrences are not expected. Here, we have taken the approach that different sequences should not be averaged because this could obscure details concerning longer sequences having a distinct function. For example, the ETS⇔CRE 13-mers GCGGAAGTGACGT and CCGGAAGTGACGT enrich for distinct GO terms in addition to having distinct methylation properties. Closer examination of proximal promoters may identify additional examples of pairs of DNA sequences that are constrained relative to each other as we observed for the ETS⇔CRE motif. The identification of these sequences will be essential as we deconvolute the genome into functional units.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-51">We thank B. K. Lee of National Cancer Institute, National Institutes of Health (NIH) for his thoughtful comments, and we thank the high-performance computational capabilities of the Helix &amp; Biowulf Systems at the NIH, Bethesda, MD (<a href="http://helix.nih.gov">http://helix.nih.gov</a>).</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.004002/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.004002/-/DC1</a></p></li><li class="fn" id="fn-2"><p id="p-52"><em>Communicating editor: T. R. Hughes</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2012-06-07"><span class="received-label">Received </span>June 7, 2012.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2012-08-19"><span class="accepted-label">Accepted </span>August 19, 2012.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2012 Chatterjee <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-2.10.1243.1" data-doi="10.1128/MCB.18.2.967"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ahn</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Olive</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aggarwal</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krylov</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ginty</span>  <span class="cit-name-given-names">D. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">A dominant-negative inhibitor of CREB reveals that it is a general mediator of stimulus-dependent transcription of c-fos</span>. <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">967</span>–<span class="cit-lpage">977</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DAhn%26rft.auinit1%253DS.%26rft.volume%253D18%26rft.issue%253D2%26rft.spage%253D967%26rft.epage%253D977%26rft.atitle%253DA%2BDominant-Negative%2BInhibitor%2Bof%2BCREB%2BReveals%2Bthat%2BIt%2BIs%2Ba%2BGeneral%2BMediator%2Bof%2BStimulus-Dependent%2BTranscription%2Bof%2Bc-fos%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.18.2.967%26rft_id%253Dinfo%253Apmid%252F9447994%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjg6IjE4LzIvOTY3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMi8xMC8xMjQzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-2.10.1243.2" data-doi="10.1126/science.1162327"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Badis</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berger</span>  <span class="cit-name-given-names">M. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Philippakis</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Talukder</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gehrke</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Diversity and complexity in DNA recognition by transcription factors</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">324</span>: <span class="cit-fpage">1720</span>–<span class="cit-lpage">1723</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1162327%26rft_id%253Dinfo%253Apmid%252F19443739%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMjQvNTkzNS8xNzIwIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMi8xMC8xMjQzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-2.10.1243.3" data-doi="10.1126/science.279.5353.1037"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Batchelor</span>  <span class="cit-name-given-names">A. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Piper</span>  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de la Brousse</span>  <span class="cit-name-given-names">F. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McKnight</span>  <span class="cit-name-given-names">S. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wolberger</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">The structure of GABPalpha/beta: an ETS domain- ankyrin repeat heterodimer bound to DNA</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">279</span>: <span class="cit-fpage">1037</span>–<span class="cit-lpage">1041</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DBatchelor%26rft.auinit1%253DA.%2BH.%26rft.volume%253D279%26rft.issue%253D5353%26rft.spage%253D1037%26rft.epage%253D1041%26rft.atitle%253DThe%2BStructure%2Bof%2BGABP%257Balpha%257D%252F%253A%2BAn%2BETS%2BDomain-%2BAnkyrin%2BRepeat%2BHeterodimer%2BBound%2Bto%2BDNA%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.279.5353.1037%26rft_id%253Dinfo%253Apmid%252F9461436%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyNzkvNTM1My8xMDM3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMi8xMC8xMjQzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-2.10.1243.4" data-doi="10.1016/j.molcel.2011.06.016"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Biddie</span>  <span class="cit-name-given-names">S. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">John</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sabo</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thurman</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Transcription factor AP1 potentiates chromatin accessibility and glucocorticoid receptor binding</span>. <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">145</span>–<span class="cit-lpage">155</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DBiddie%26rft.auinit1%253DS.%2BC.%26rft.volume%253D43%26rft.issue%253D1%26rft.spage%253D145%26rft.epage%253D155%26rft.atitle%253DTranscription%2Bfactor%2BAP1%2Bpotentiates%2Bchromatin%2Baccessibility%2Band%2Bglucocorticoid%2Breceptor%2Bbinding.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2011.06.016%26rft_id%253Dinfo%253Apmid%252F21726817%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2011.06.016&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21726817&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F10%2F1243.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000292723400015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-2.10.1243.5" data-doi="10.1016/j.ygeno.2004.08.013"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bina</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wyss</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ren</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Szpankowski</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thomas</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Exploring the characteristics of sequence elements in proximal promoters of human genes</span>. <abbr class="cit-jnl-abbrev">Genomics</abbr> <span class="cit-vol">84</span>: <span class="cit-fpage">929</span>–<span class="cit-lpage">940</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenomics%26rft.stitle%253DGenomics%26rft.aulast%253DBina%26rft.auinit1%253DM.%26rft.volume%253D84%26rft.issue%253D6%26rft.spage%253D929%26rft.epage%253D940%26rft.atitle%253DExploring%2Bthe%2Bcharacteristics%2Bof%2Bsequence%2Belements%2Bin%2Bproximal%2Bpromoters%2Bof%2Bhuman%2Bgenes.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ygeno.2004.08.013%26rft_id%253Dinfo%253Apmid%252F15533710%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ygeno.2004.08.013&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15533710&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F10%2F1243.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000225306600004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-2.10.1243.6" data-doi="10.1016/j.jmb.2011.01.056"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bird</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The dinucleotide CG as a genomic signalling module</span>. <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">409</span>: <span class="cit-fpage">47</span>–<span class="cit-lpage">53</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DBird%26rft.auinit1%253DA.%26rft.volume%253D409%26rft.issue%253D1%26rft.spage%253D47%26rft.epage%253D53%26rft.atitle%253DThe%2Bdinucleotide%2BCG%2Bas%2Ba%2Bgenomic%2Bsignalling%2Bmodule.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jmb.2011.01.056%26rft_id%253Dinfo%253Apmid%252F21295585%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jmb.2011.01.056&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21295585&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F10%2F1243.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-2.10.1243.7" data-doi="10.1002/(SICI)1521-4141(200004)30:4&lt;1102::AID-IMMU1102&gt;3.0.CO;2-X"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Boehlk</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fessele</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mojaat</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miyamoto</span>  <span class="cit-name-given-names">N. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Werner</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">ATF and Jun transcription factors, acting through an Ets/CRE promoter module, mediate lipopolysaccharide inducibility of the chemokine RANTES in monocytic Mono Mac 6 cells</span>. <abbr class="cit-jnl-abbrev">Eur. J. Immunol.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">1102</span>–<span class="cit-lpage">1112</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEuropean%2Bjournal%2Bof%2Bimmunology%26rft.stitle%253DEur%2BJ%2BImmunol%26rft.aulast%253DBoehlk%26rft.auinit1%253DS.%26rft.volume%253D30%26rft.issue%253D4%26rft.spage%253D1102%26rft.epage%253D1112%26rft.atitle%253DATF%2Band%2BJun%2Btranscription%2Bfactors%252C%2Bacting%2Bthrough%2Ban%2BEts%252FCRE%2Bpromoter%2Bmodule%252C%2Bmediate%2Blipopolysaccharide%2Binducibility%2Bof%2Bthe%2Bchemokine%2BRANTES%2Bin%2Bmonocytic%2BMono%2BMac%2B6%2Bcells.%26rft_id%253Dinfo%253Adoi%252F10.1002%252F%2528SICI%25291521-4141%2528200004%252930%253A4%253C1102%253A%253AAID-IMMU1102%253E3.0.CO%253B2-X%26rft_id%253Dinfo%253Apmid%252F10760799%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/(SICI)1521-4141(200004)30:4&lt;1102::AID-IMMU1102&gt;3.0.CO;2-X&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10760799&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F10%2F1243.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000086543700016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-2.10.1243.8" data-doi="10.1038/ng1789"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Carninci</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sandelin</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lenhard</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Katayama</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shimokawa</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Genome-wide analysis of mammalian promoter architecture and evolution</span>. <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">626</span>–<span class="cit-lpage">635</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DCarninci%26rft.auinit1%253DP.%26rft.volume%253D38%26rft.issue%253D6%26rft.spage%253D626%26rft.epage%253D635%26rft.atitle%253DGenome-wide%2Banalysis%2Bof%2Bmammalian%2Bpromoter%2Barchitecture%2Band%2Bevolution.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1789%26rft_id%253Dinfo%253Apmid%252F16645617%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1789&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16645617&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F10%2F1243.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000237954800015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-2.10.1243.9" data-doi="10.1016/j.cell.2008.10.049"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">De Val</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chi</span>  <span class="cit-name-given-names">N. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meadows</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Minovitsky</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Anderson</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Combinatorial regulation of endothelial gene expression by ets and forkhead transcription factors</span>. <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">135</span>: <span class="cit-fpage">1053</span>–<span class="cit-lpage">1064</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DDe%2BVal%26rft.auinit1%253DS.%26rft.volume%253D135%26rft.issue%253D6%26rft.spage%253D1053%26rft.epage%253D1064%26rft.atitle%253DCombinatorial%2Bregulation%2Bof%2Bendothelial%2Bgene%2Bexpression%2Bby%2Bets%2Band%2Bforkhead%2Btranscription%2Bfactors.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2008.10.049%26rft_id%253Dinfo%253Apmid%252F19070576%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2008.10.049&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19070576&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F10%2F1243.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000261642800021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-2.10.1243.10" data-doi="10.1038/nrg2636"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Farnham</span>  <span class="cit-name-given-names">P. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Insights from genomic profiling of transcription factors</span>. <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">605</span>–<span class="cit-lpage">616</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DFarnham%26rft.auinit1%253DP.%2BJ.%26rft.volume%253D10%26rft.issue%253D9%26rft.spage%253D605%26rft.epage%253D616%26rft.atitle%253DInsights%2Bfrom%2Bgenomic%2Bprofiling%2Bof%2Btranscription%2Bfactors.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2636%26rft_id%253Dinfo%253Apmid%252F19668247%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2636&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19668247&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F10%2F1243.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000269063200010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-2.10.1243.11" data-doi="10.1101/gr.1953904"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">FitzGerald</span>  <span class="cit-name-given-names">P. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shlyakhtenko</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mir</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vinson</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Clustering of DNA sequences in human promoters</span>. <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">1562</span>–<span class="cit-lpage">1574</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.1953904%26rft_id%253Dinfo%253Apmid%252F15256515%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjE0LzgvMTU2MiI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzIvMTAvMTI0My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-2.10.1243.12" data-doi="10.1186/gb-2006-7-7-r53"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">FitzGerald</span>  <span class="cit-name-given-names">P. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sturgill</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shyakhtenko</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oliver</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vinson</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Comparative genomics of Drosophila and human core promoters</span>. <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">R53</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DFitzgerald%26rft.auinit1%253DP.%2BC.%26rft.volume%253D7%26rft.issue%253D7%26rft.spage%253DR53%26rft.epage%253DR53%26rft.atitle%253DComparative%2Bgenomics%2Bof%2BDrosophila%2Band%2Bhuman%2Bcore%2Bpromoters.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2006-7-7-r53%26rft_id%253Dinfo%253Apmid%252F16827941%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2006-7-7-r53&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16827941&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F10%2F1243.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-2.10.1243.13" data-doi="10.1093/nar/gkf438"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Frith</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spouge</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hansen</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weng</span>  <span class="cit-name-given-names">Z.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Statistical significance of clusters of motifs represented by position specific scoring matrices in nucleotide sequences</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">3214</span>–<span class="cit-lpage">3224</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DFrith%26rft.auinit1%253DM.%2BC.%26rft.volume%253D30%26rft.issue%253D14%26rft.spage%253D3214%26rft.epage%253D3224%26rft.atitle%253DStatistical%2Bsignificance%2Bof%2Bclusters%2Bof%2Bmotifs%2Brepresented%2Bby%2Bposition%2Bspecific%2Bscoring%2Bmatrices%2Bin%2Bnucleotide%2Bsequences%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkf438%26rft_id%253Dinfo%253Apmid%252F12136103%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiIzMC8xNC8zMjE0IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMi8xMC8xMjQzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-2.10.1243.14" data-doi="10.1016/S1097-2765(01)00410-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Garvie</span>  <span class="cit-name-given-names">C. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hagman</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wolberger</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Structural studies of Ets-1/Pax5 complex formation on DNA</span>. <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">1267</span>–<span class="cit-lpage">1276</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DGarvie%26rft.auinit1%253DC.%2BW.%26rft.volume%253D8%26rft.issue%253D6%26rft.spage%253D1267%26rft.epage%253D1276%26rft.atitle%253DStructural%2Bstudies%2Bof%2BEts-1%252FPax5%2Bcomplex%2Bformation%2Bon%2BDNA.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1097-2765%252801%252900410-5%26rft_id%253Dinfo%253Apmid%252F11779502%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S1097-2765(01)00410-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11779502&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F10%2F1243.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000172907800014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-2.10.1243.15" data-doi="10.1016/S0065-230X(08)60738-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Graves</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Petersen</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Specificity within the ets family of transcription factors</span>. <abbr class="cit-jnl-abbrev">Adv. Cancer Res.</abbr> <span class="cit-vol">75</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">55</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAdvances%2Bin%2Bcancer%2Bresearch%26rft.stitle%253DAdv%2BCancer%2BRes%26rft.aulast%253DGraves%26rft.auinit1%253DB.%2BJ.%26rft.volume%253D75%26rft.spage%253D1%26rft.epage%253D55%26rft.atitle%253DSpecificity%2Bwithin%2Bthe%2Bets%2Bfamily%2Bof%2Btranscription%2Bfactors.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0065-230X%252808%252960738-1%26rft_id%253Dinfo%253Apmid%252F9709806%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0065-230X(08)60738-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9709806&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F10%2F1243.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000079074200001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-2.10.1243.16" data-doi="10.1101/gad.17546311"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hollenhorst</span>  <span class="cit-name-given-names">P. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferris</span>  <span class="cit-name-given-names">M. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hull</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chae</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span><span class="cit-comment">a</span> <span class="cit-article-title">Oncogenic ETS proteins mimic activated RAS/MAPK signaling in prostate cells</span>. <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">2147</span>–<span class="cit-lpage">2157</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DHollenhorst%26rft.auinit1%253DP.%2BC.%26rft.volume%253D25%26rft.issue%253D20%26rft.spage%253D2147%26rft.epage%253D2157%26rft.atitle%253DOncogenic%2BETS%2Bproteins%2Bmimic%2Bactivated%2BRAS%252FMAPK%2Bsignaling%2Bin%2Bprostate%2Bcells%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.17546311%26rft_id%253Dinfo%253Apmid%252F22012618%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6MTA6IjI1LzIwLzIxNDciO3M6NDoiYXRvbSI7czoxOToiL2dnZy8yLzEwLzEyNDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-2.10.1243.17" data-doi="10.1146/annurev.biochem.79.081507.103945"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hollenhorst</span>  <span class="cit-name-given-names">P. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McIntosh</span>  <span class="cit-name-given-names">L. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Graves</span>  <span class="cit-name-given-names">B. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span><span class="cit-comment">b</span> <span class="cit-article-title">Genomic and biochemical insights into the specificity of ETS transcription factors</span>. <abbr class="cit-jnl-abbrev">Annu. Rev. Biochem.</abbr> <span class="cit-vol">80</span>: <span class="cit-fpage">437</span>–<span class="cit-lpage">471</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bbiochemistry%26rft.stitle%253DAnnu%2BRev%2BBiochem%26rft.aulast%253DHollenhorst%26rft.auinit1%253DP.%2BC.%26rft.volume%253D80%26rft.spage%253D437%26rft.epage%253D471%26rft.atitle%253DGenomic%2Band%2Bbiochemical%2Binsights%2Binto%2Bthe%2Bspecificity%2Bof%2BETS%2Btranscription%2Bfactors.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.biochem.79.081507.103945%26rft_id%253Dinfo%253Apmid%252F21548782%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.biochem.79.081507.103945&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21548782&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F10%2F1243.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000292895600020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-2.10.1243.18" data-doi="10.1101/gad.3.5.612"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Iguchi-Ariga</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schaffner</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">CpG methylation of the cAMP-responsive enhancer/promoter sequence TGACGTCA abolishes specific factor binding as well as transcriptional activation</span>. <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">612</span>–<span class="cit-lpage">619</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DIguchi-Ariga%26rft.auinit1%253DS%2BM%26rft.volume%253D3%26rft.issue%253D5%26rft.spage%253D612%26rft.epage%253D619%26rft.atitle%253DCpG%2Bmethylation%2Bof%2Bthe%2BcAMP-responsive%2Benhancer%252Fpromoter%2Bsequence%2BTGACGTCA%2Babolishes%2Bspecific%2Bfactor%2Bbinding%2Bas%2Bwell%2Bas%2Btranscriptional%2Bactivation.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.3.5.612%26rft_id%253Dinfo%253Apmid%252F2545524%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6NzoiMy81LzYxMiI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzIvMTAvMTI0My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-2.10.1243.19" data-doi="10.1038/nbt.1505"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ji</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">D. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Myers</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">An integrated software system for analyzing ChIP-chip and ChIP-seq data</span>. <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">1293</span>–<span class="cit-lpage">1300</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DJi%26rft.auinit1%253DH.%26rft.volume%253D26%26rft.issue%253D11%26rft.spage%253D1293%26rft.epage%253D1300%26rft.atitle%253DAn%2Bintegrated%2Bsoftware%2Bsystem%2Bfor%2Banalyzing%2BChIP-chip%2Band%2BChIP-seq%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.1505%26rft_id%253Dinfo%253Apmid%252F18978777%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.1505&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18978777&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F10%2F1243.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000260832200025&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-2.10.1243.20" data-doi="10.1126/science.1141319"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">D. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mortazavi</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Myers</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wold</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Genome-wide mapping of in vivo protein-DNA interactions</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">316</span>: <span class="cit-fpage">1497</span>–<span class="cit-lpage">1502</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1141319%26rft_id%253Dinfo%253Apmid%252F17540862%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMTYvNTgzMC8xNDk3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMi8xMC8xMjQzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-2.10.1243.21" data-doi="10.1128/MCB.13.11.6919"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">P. F.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Identification of C/EBP basic region residues involved in DNA sequence recognition and half-site spacing preference</span>. <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">6919</span>–<span class="cit-lpage">6930</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DJohnson%26rft.auinit1%253DP%2BF%26rft.volume%253D13%26rft.issue%253D11%26rft.spage%253D6919%26rft.epage%253D6930%26rft.atitle%253DIdentification%2Bof%2BC%252FEBP%2Bbasic%2Bregion%2Bresidues%2Binvolved%2Bin%2BDNA%2Bsequence%2Brecognition%2Band%2Bhalf-site%2Bspacing%2Bpreference.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.13.11.6919%26rft_id%253Dinfo%253Apmid%252F8413284%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjEwOiIxMy8xMS82OTE5IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMi8xMC8xMjQzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-2.10.1243.22" data-doi="10.1186/gb-2010-11-11-140"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kaplan</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hughes</span>  <span class="cit-name-given-names">T. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lieb</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Widom</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Segal</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Contribution of histone sequence preferences to nucleosome organization: proposed definitions and methodology</span>. <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">140</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DKaplan%26rft.auinit1%253DN.%26rft.volume%253D11%26rft.issue%253D11%26rft.spage%253D140%26rft.epage%253D140%26rft.atitle%253DContribution%2Bof%2Bhistone%2Bsequence%2Bpreferences%2Bto%2Bnucleosome%2Borganization%253A%2Bproposed%2Bdefinitions%2Band%2Bmethodology.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2010-11-11-140%26rft_id%253Dinfo%253Apmid%252F21118582%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2010-11-11-140&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21118582&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F10%2F1243.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-2.10.1243.23" data-doi="10.1093/nar/gkg585"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kel</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gossling</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reuter</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheremushkin</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kel-Margoulis</span>  <span class="cit-name-given-names">O. V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">MATCH: a tool for searching transcription factor binding sites in DNA sequences</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">3576</span>–<span class="cit-lpage">3579</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DKel%26rft.auinit1%253DA.E.%26rft.volume%253D31%26rft.issue%253D13%26rft.spage%253D3576%26rft.epage%253D3579%26rft.atitle%253DMATCHTM%253A%2Ba%2Btool%2Bfor%2Bsearching%2Btranscription%2Bfactor%2Bbinding%2Bsites%2Bin%2BDNA%2Bsequences%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkg585%26rft_id%253Dinfo%253Apmid%252F12824369%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiIzMS8xMy8zNTc2IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMi8xMC8xMjQzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-2.10.1243.24" data-doi="10.1038/nbt.1508"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kharchenko</span>  <span class="cit-name-given-names">P. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tolstorukov</span>  <span class="cit-name-given-names">M. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Park</span>  <span class="cit-name-given-names">P. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Design and analysis of ChIP-seq experiments for DNA-binding proteins</span>. <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">1351</span>–<span class="cit-lpage">1359</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DKharchenko%26rft.auinit1%253DP.%2BV.%26rft.volume%253D26%26rft.issue%253D12%26rft.spage%253D1351%26rft.epage%253D1359%26rft.atitle%253DDesign%2Band%2Banalysis%2Bof%2BChIP-seq%2Bexperiments%2Bfor%2BDNA-binding%2Bproteins.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.1508%26rft_id%253Dinfo%253Apmid%252F19029915%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.1508&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19029915&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F10%2F1243.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000261591300019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-2.10.1243.25" data-doi="10.1101/gad.12.1.34"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lagrange</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kapanidis</span>  <span class="cit-name-given-names">A. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reinberg</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ebright</span>  <span class="cit-name-given-names">R. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">New core promoter element in RNA polymerase II-dependent transcription: sequence-specific DNA binding by transcription factor IIB</span>. <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">34</span>–<span class="cit-lpage">44</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DLagrange%26rft.auinit1%253DT.%26rft.volume%253D12%26rft.issue%253D1%26rft.spage%253D34%26rft.epage%253D44%26rft.atitle%253DNew%2Bcore%2Bpromoter%2Belement%2Bin%2BRNA%2Bpolymerase%2BII-dependent%2Btranscription%253A%2Bsequence-specific%2BDNA%2Bbinding%2Bby%2Btranscription%2Bfactor%25A0IIB%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.12.1.34%26rft_id%253Dinfo%253Apmid%252F9420329%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6NzoiMTIvMS8zNCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzIvMTAvMTI0My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-2.10.1243.26" data-doi="10.1093/bioinformatics/btr189"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Machanick</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bailey</span>  <span class="cit-name-given-names">T. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">MEME-ChIP: motif analysis of large DNA datasets</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">1696</span>–<span class="cit-lpage">1697</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtr189%26rft_id%253Dinfo%253Apmid%252F21486936%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjcvMTIvMTY5NiI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzIvMTAvMTI0My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-2.10.1243.27" data-doi="10.1093/nar/gkh246"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Marino-Ramirez</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spouge</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kanga</span>  <span class="cit-name-given-names">G. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Landsman</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Statistical analysis of over-represented words in human promoter sequences</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">949</span>–<span class="cit-lpage">958</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DMarino-Ramirez%26rft.auinit1%253DL.%26rft.volume%253D32%26rft.issue%253D3%26rft.spage%253D949%26rft.epage%253D958%26rft.atitle%253DStatistical%2Banalysis%2Bof%2Bover-represented%2Bwords%2Bin%2Bhuman%2Bpromoter%2Bsequences%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkh246%26rft_id%253Dinfo%253Apmid%252F14963262%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjg6IjMyLzMvOTQ5IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMi8xMC8xMjQzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-2.10.1243.28" data-doi="10.1186/1471-2164-11-530"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Martianov</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Choukrallah</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krebs</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ye</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Legras</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Cell-specific occupancy of an extended repertoire of CREM and CREB binding loci in male germ cells</span>. <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">530</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DMartianov%26rft.auinit1%253DI.%26rft.volume%253D11%26rft.spage%253D530%26rft.epage%253D530%26rft.atitle%253DCell-specific%2Boccupancy%2Bof%2Ban%2Bextended%2Brepertoire%2Bof%2BCREM%2Band%2BCREB%2Bbinding%2Bloci%2Bin%2Bmale%2Bgerm%2Bcells.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-11-530%26rft_id%253Dinfo%253Apmid%252F20920259%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-11-530&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20920259&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F10%2F1243.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-2.10.1243.29" data-doi="10.1093/nar/gkj143"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Matys</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kel-Margoulis</span>  <span class="cit-name-given-names">O. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fricke</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liebich</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Land</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">TRANSFAC and its module TRANSCompel: transcriptional gene regulation in eukaryotes</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">D108</span>–<span class="cit-lpage">D110</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DMatys%26rft.auinit1%253DV.%26rft.volume%253D34%26rft.issue%253Dsuppl_1%26rft.spage%253DD108%26rft.epage%253DD110%26rft.atitle%253DTRANSFAC%2528R%2529%2Band%2Bits%2Bmodule%2BTRANSCompel%2528R%2529%253A%2Btranscriptional%2Bgene%2Bregulation%2Bin%2Beukaryotes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkj143%26rft_id%253Dinfo%253Apmid%252F16381825%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjE1OiIzNC9zdXBwbF8xL0QxMDgiO3M6NDoiYXRvbSI7czoxOToiL2dnZy8yLzEwLzEyNDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-2.10.1243.30" data-doi="10.1073/pnas.0913805107"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mirny</span>  <span class="cit-name-given-names">L. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Nucleosome-mediated cooperativity between transcription factors</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">107</span>: <span class="cit-fpage">22534</span>–<span class="cit-lpage">22539</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0913805107%26rft_id%253Dinfo%253Apmid%252F21149679%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA3LzUyLzIyNTM0IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMi8xMC8xMjQzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-2.10.1243.31"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Oh</span>  <span class="cit-name-given-names">Y. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Choi</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yoo</span>  <span class="cit-name-given-names">J. Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Identification of co-occurring transcription factor binding sites from DNA sequence using clustered position weight matrices</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">e38</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft.volume%253D40%26rft.spage%253De38%26rft_id%253Dinfo%253Apmid%252F22187154%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=22187154&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F10%2F1243.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-2.10.1243.32"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ohler</span>  <span class="cit-name-given-names">U.</span></span></li></ol><cite>, G. C<span class="cit-auth"><span class="cit-name-surname">Liao</span>  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Niemann</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rubin</span>  <span class="cit-name-given-names">G. M.</span></span>. <span class="cit-pub-date">2002</span> <span class="cit-article-title">Computational analysis of core promoters in the Drosophila genome</span>. <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">3</span>: RESEARCH0087.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-2.10.1243.33" data-doi="10.1093/nar/gkl857"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pachkov</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Erb</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Molina</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Nimwegen</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">SwissRegulon: a database of genome-wide annotations of regulatory sites</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">D127</span>–<span class="cit-lpage">D131</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkl857%26rft_id%253Dinfo%253Apmid%252F17130146%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjE1OiIzNS9zdXBwbF8xL0QxMjciO3M6NDoiYXRvbSI7czoxOToiL2dnZy8yLzEwLzEyNDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-2.10.1243.34" data-doi="10.1016/j.sbi.2007.12.002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Panne</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">The enhanceosome</span>. <abbr class="cit-jnl-abbrev">Curr. Opin. Struct. Biol.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">236</span>–<span class="cit-lpage">242</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Bstructural%2Bbiology%26rft.stitle%253DCurr%2BOpin%2BStruct%2BBiol%26rft.aulast%253DPanne%26rft.auinit1%253DD.%26rft.volume%253D18%26rft.issue%253D2%26rft.spage%253D236%26rft.epage%253D242%26rft.atitle%253DThe%2Benhanceosome.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.sbi.2007.12.002%26rft_id%253Dinfo%253Apmid%252F18206362%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.sbi.2007.12.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18206362&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F10%2F1243.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000255800200015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-2.10.1243.35" data-doi="10.1038/sj.emboj.7600453"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Panne</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maniatis</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harrison</span>  <span class="cit-name-given-names">S. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Crystal structure of ATF-2/c-Jun and IRF-3 bound to the interferon-beta enhancer</span>. <abbr class="cit-jnl-abbrev">EMBO J.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">4384</span>–<span class="cit-lpage">4393</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEMBO%2BJ.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsj.emboj.7600453%26rft_id%253Dinfo%253Apmid%252F15510218%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZW1ib2pubCI7czo1OiJyZXNpZCI7czoxMDoiMjMvMjIvNDM4NCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzIvMTAvMTI0My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-2.10.1243.36" data-doi="10.1074/jbc.M703019200"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Panne</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McWhirter</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maniatis</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harrison</span>  <span class="cit-name-given-names">S. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Interferon regulatory factor 3 is regulated by a dual phosphorylation-dependent switch</span>. <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">282</span>: <span class="cit-fpage">22816</span>–<span class="cit-lpage">22822</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M703019200%26rft_id%253Dinfo%253Apmid%252F17526488%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyODIvMzEvMjI4MTYiO3M6NDoiYXRvbSI7czoxOToiL2dnZy8yLzEwLzEyNDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-2.10.1243.37" data-doi="10.1002/jcc.20084"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pettersen</span>  <span class="cit-name-given-names">E. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">T. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">C. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Couch</span>  <span class="cit-name-given-names">G. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Greenblatt</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">UCSF Chimera–a visualization system for exploratory research and analysis</span>. <abbr class="cit-jnl-abbrev">J. Comput. Chem.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1605</span>–<span class="cit-lpage">1612</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bcomputational%2Bchemistry%26rft.stitle%253DJ%2BComput%2BChem%26rft.aulast%253DPettersen%26rft.auinit1%253DE.%2BF.%26rft.volume%253D25%26rft.issue%253D13%26rft.spage%253D1605%26rft.epage%253D1612%26rft.atitle%253DUCSF%2BChimera--a%2Bvisualization%2Bsystem%2Bfor%2Bexploratory%2Bresearch%2Band%2Banalysis.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fjcc.20084%26rft_id%253Dinfo%253Apmid%252F15264254%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/jcc.20084&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15264254&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F10%2F1243.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000223379100005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-2.10.1243.38" data-doi="10.1006/jmbi.1996.0288"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Polach</span>  <span class="cit-name-given-names">K. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Widom</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">A model for the cooperative binding of eukaryotic regulatory proteins to nucleosomal target sites</span>. <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">258</span>: <span class="cit-fpage">800</span>–<span class="cit-lpage">812</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DPolach%26rft.auinit1%253DK.%2BJ.%26rft.volume%253D258%26rft.issue%253D5%26rft.spage%253D800%26rft.epage%253D812%26rft.atitle%253DA%2Bmodel%2Bfor%2Bthe%2Bcooperative%2Bbinding%2Bof%2Beukaryotic%2Bregulatory%2Bproteins%2Bto%2Bnucleosomal%2Btarget%2Bsites.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fjmbi.1996.0288%26rft_id%253Dinfo%253Apmid%252F8637011%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/jmbi.1996.0288&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8637011&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F10%2F1243.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1996UL05700009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-2.10.1243.39" data-doi="10.1101/gr.097857.109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pollard</span>  <span class="cit-name-given-names">K. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hubisz</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosenbloom</span>  <span class="cit-name-given-names">K. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Siepel</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Detection of nonneutral substitution rates on mammalian phylogenies</span>. <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">110</span>–<span class="cit-lpage">121</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.097857.109%26rft_id%253Dinfo%253Apmid%252F19858363%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjIwLzEvMTEwIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMi8xMC8xMjQzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-2.10.1243.40" data-doi="10.1093/nar/gkp950"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Portales-Casamar</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thongjuea</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kwon</span>  <span class="cit-name-given-names">A. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arenillas</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">JASPAR 2010: the greatly expanded open-access database of transcription factor binding profiles</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">D105</span>–<span class="cit-lpage">D110</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkp950%26rft_id%253Dinfo%253Apmid%252F19906716%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjE1OiIzOC9zdXBwbF8xL0QxMDUiO3M6NDoiYXRvbSI7czoxOToiL2dnZy8yLzEwLzEyNDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-2.10.1243.41" data-doi="10.1073/pnas.1008688107"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rishi</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bhattacharya</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chatterjee</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rozenberg</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">CpG methylation of half-CRE sequences creates C/EBPalpha binding sites that activate some tissue-specific genes</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">107</span>: <span class="cit-fpage">20311</span>–<span class="cit-lpage">20316</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1008688107%26rft_id%253Dinfo%253Apmid%252F21059933%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA3LzQ3LzIwMzExIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMi8xMC8xMjQzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-2.10.1243.42" data-doi="10.1186/1471-2164-9-67"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rozenberg</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shlyakhtenko</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glass</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rishi</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Myakishev</span>  <span class="cit-name-given-names">M. V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">All and only CpG containing sequences are enriched in promoters abundantly bound by RNA polymerase II in multiple tissues</span>. <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">67</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DRozenberg%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D9%26rft.spage%253D67%26rft.epage%253D67%26rft.atitle%253DAll%2Band%2Bonly%2BCpG%2Bcontaining%2Bsequences%2Bare%2Benriched%2Bin%2Bpromoters%2Babundantly%2Bbound%2Bby%2BRNA%2Bpolymerase%2BII%2Bin%2Bmultiple%2Btissues.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-9-67%26rft_id%253Dinfo%253Apmid%252F18252004%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-9-67&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18252004&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F10%2F1243.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-2.10.1243.43" data-doi="10.1073/pnas.0400678101"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sabo</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Humbert</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hawrylycz</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wallace</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dorschner</span>  <span class="cit-name-given-names">M. O.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Genome-wide identification of DNaseI hypersensitive sites using active chromatin sequence libraries</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">101</span>: <span class="cit-fpage">4537</span>–<span class="cit-lpage">4542</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0400678101%26rft_id%253Dinfo%253Apmid%252F15070753%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTAxLzEzLzQ1MzciO3M6NDoiYXRvbSI7czoxOToiL2dnZy8yLzEwLzEyNDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-2.10.1243.44" data-doi="10.1074/jbc.274.50.35475"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sawada</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Simizu</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Suzuki</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sawa</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goto</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Synergistic transcriptional activation by hGABP and select members of the activation transcription factor/cAMP response element-binding protein family</span>. <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">274</span>: <span class="cit-fpage">35475</span>–<span class="cit-lpage">35482</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBiological%2BChemistry%26rft.stitle%253DJ%2BBiol%2BChem%26rft.aulast%253DSawada%26rft.auinit1%253DJ.-i.%26rft.volume%253D274%26rft.issue%253D50%26rft.spage%253D35475%26rft.epage%253D35482%26rft.atitle%253DSynergistic%2BTranscriptional%2BActivation%2Bby%2BhGABP%2Band%2BSelect%2BMembers%2Bof%2Bthe%2BActivation%2BTranscription%2BFactor%252FcAMP%2BResponse%2BElement-binding%2BProtein%2BFamily%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.274.50.35475%26rft_id%253Dinfo%253Apmid%252F10585419%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyNzQvNTAvMzU0NzUiO3M6NDoiYXRvbSI7czoxOToiL2dnZy8yLzEwLzEyNDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-2.10.1243.45" data-doi="10.1074/jbc.M007293200"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schumacher</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goodman</span>  <span class="cit-name-given-names">R. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brennan</span>  <span class="cit-name-given-names">R. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">The structure of a CREB bZIP.somatostatin CRE complex reveals the basis for selective dimerization and divalent cation-enhanced DNA binding</span>. <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">275</span>: <span class="cit-fpage">35242</span>–<span class="cit-lpage">35247</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M007293200%26rft_id%253Dinfo%253Apmid%252F10952992%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyNzUvNDUvMzUyNDIiO3M6NDoiYXRvbSI7czoxOToiL2dnZy8yLzEwLzEyNDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-2.10.1243.46" data-doi="10.1146/annurev.biochem.72.121801.161520"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Smale</span>  <span class="cit-name-given-names">S. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kadonaga</span>  <span class="cit-name-given-names">J. T.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">The RNA polymerase II core promoter</span>. <abbr class="cit-jnl-abbrev">Annu. Rev. Biochem.</abbr> <span class="cit-vol">72</span>: <span class="cit-fpage">449</span>–<span class="cit-lpage">479</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bbiochemistry%26rft.stitle%253DAnnu%2BRev%2BBiochem%26rft.volume%253D72%26rft.spage%253D449%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.biochem.72.121801.161520%26rft_id%253Dinfo%253Apmid%252F12651739%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.biochem.72.121801.161520&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12651739&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F10%2F1243.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000185092500015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-2.10.1243.47"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thomas-Chollier</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Herrmann</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Defrance</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sand</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thieffry</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">RSAT peak-motifs: motif analysis in full-size ChIP-seq datasets</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">e31</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft.volume%253D40%26rft.spage%253De31%26rft_id%253Dinfo%253Apmid%252F22156162%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=22156162&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F10%2F1243.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-2.10.1243.48" data-doi="10.1128/MCB.17.9.4885"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Umezawa</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yamamoto</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rhodes</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klemsz</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maki</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Methylation of an ETS site in the intron enhancer of the keratin 18 gene participates in tissue-specific repression</span>. <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">4885</span>–<span class="cit-lpage">4894</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DUmezawa%26rft.auinit1%253DA.%26rft.volume%253D17%26rft.issue%253D9%26rft.spage%253D4885%26rft.epage%253D4894%26rft.atitle%253DMethylation%2Bof%2Ban%2BETS%2Bsite%2Bin%2Bthe%2Bintron%2Benhancer%2Bof%2Bthe%2Bkeratin%2B18%2Bgene%2Bparticipates%2Bin%2Btissue-specific%2Brepression%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.17.9.4885%26rft_id%253Dinfo%253Apmid%252F9271368%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjk6IjE3LzkvNDg4NSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzIvMTAvMTI0My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-2.10.1243.49" data-doi="10.1038/nmeth.1246"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Valouev</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">D. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sundquist</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Medina</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Anton</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Genome-wide analysis of transcription factor binding sites based on ChIP-Seq data</span>. <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">829</span>–<span class="cit-lpage">834</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DValouev%26rft.auinit1%253DA.%26rft.volume%253D5%26rft.issue%253D9%26rft.spage%253D829%26rft.epage%253D834%26rft.atitle%253DGenome-wide%2Banalysis%2Bof%2Btranscription%2Bfactor%2Bbinding%2Bsites%2Bbased%2Bon%2BChIP-Seq%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1246%26rft_id%253Dinfo%253Apmid%252F19160518%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1246&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19160518&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F10%2F1243.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000258912700017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-2.10.1243.50" data-doi="10.1007/978-90-481-9069-0_10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vinson</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chatterjee</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fitzgerald</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Transcription factor binding sites and other features in human and Drosophila proximal promoters</span>. <abbr class="cit-jnl-abbrev">Subcell. Biochem.</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">205</span>–<span class="cit-lpage">222</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSub-cellular%2Bbiochemistry%26rft.stitle%253DSubcell%2BBiochem%26rft.aulast%253DVinson%26rft.auinit1%253DC.%26rft.volume%253D52%26rft.spage%253D205%26rft.epage%253D222%26rft.atitle%253DTranscription%2Bfactor%2Bbinding%2Bsites%2Band%2Bother%2Bfeatures%2Bin%2Bhuman%2Band%2BDrosophila%2Bproximal%2Bpromoters.%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-90-481-9069-0_10%26rft_id%253Dinfo%253Apmid%252F21557085%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-90-481-9069-0_10&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21557085&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F10%2F1243.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-2.10.1243.51" data-doi="10.1038/emboj.2010.106"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wei</span>  <span class="cit-name-given-names">G. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Badis</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berger</span>  <span class="cit-name-given-names">M. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kivioja</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Palin</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genome-wide analysis of ETS-family DNA-binding in vitro and in vivo</span>. <abbr class="cit-jnl-abbrev">EMBO J.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">2147</span>–<span class="cit-lpage">2160</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BEMBO%2BJournal%26rft.stitle%253DEMBO%2BJournal%26rft.aulast%253DWei%26rft.auinit1%253DG.%2BH.%26rft.volume%253D29%26rft.issue%253D13%26rft.spage%253D2147%26rft.epage%253D2160%26rft.atitle%253DGenome-wide%2Banalysis%2Bof%2BETS-family%2BDNA-binding%2Bin%2Bvitro%2Band%2Bin%2Bvivo.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Femboj.2010.106%26rft_id%253Dinfo%253Apmid%252F20517297%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/emboj.2010.106&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20517297&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F10%2F1243.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000279630600007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-2.10.1243.52" data-doi="10.1093/nar/gkr341"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Whitington</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Frith</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bailey</span>  <span class="cit-name-given-names">T. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Inferring transcription factor complexes from ChIP-seq data</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">e98</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkr341%26rft_id%253Dinfo%253Apmid%252F21602262%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjk6IjM5LzE1L2U5OCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzIvMTAvMTI0My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-2.10.1243.53" data-doi="10.1016/j.tig.2009.08.003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wunderlich</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mirny</span>  <span class="cit-name-given-names">L. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Different gene regulation strategies revealed by analysis of binding motifs</span>. <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">434</span>–<span class="cit-lpage">440</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DWunderlich%26rft.auinit1%253DZ.%26rft.volume%253D25%26rft.issue%253D10%26rft.spage%253D434%26rft.epage%253D440%26rft.atitle%253DDifferent%2Bgene%2Bregulation%2Bstrategies%2Brevealed%2Bby%2Banalysis%2Bof%2Bbinding%2Bmotifs.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2009.08.003%26rft_id%253Dinfo%253Apmid%252F19815308%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tig.2009.08.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19815308&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F10%2F1243.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000271573100004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-2.10.1243.54" data-doi="10.1038/nature03441"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xie</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kulbokas</span>  <span class="cit-name-given-names">E. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Golub</span>  <span class="cit-name-given-names">T. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mootha</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Systematic discovery of regulatory motifs in human promoters and 3′ UTRs by comparison of several mammals</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">434</span>: <span class="cit-fpage">338</span>–<span class="cit-lpage">345</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DXie%26rft.auinit1%253DX.%26rft.volume%253D434%26rft.issue%253D7031%26rft.spage%253D338%26rft.epage%253D345%26rft.atitle%253DSystematic%2Bdiscovery%2Bof%2Bregulatory%2Bmotifs%2Bin%2Bhuman%2Bpromoters%2Band%2B3%2527%2BUTRs%2Bby%2Bcomparison%2Bof%2Bseveral%2Bmammals.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature03441%26rft_id%253Dinfo%253Apmid%252F15735639%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature03441&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15735639&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F10%2F1243.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000227715100041&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-2.10.1243.55" data-doi="10.1073/pnas.0501076102"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Odom</span>  <span class="cit-name-given-names">D. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koo</span>  <span class="cit-name-given-names">S. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Conkright</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Canettieri</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Genome-wide analysis of cAMP-response element binding protein occupancy, phosphorylation, and target gene activation in human tissues</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">102</span>: <span class="cit-fpage">4459</span>–<span class="cit-lpage">4464</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0501076102%26rft_id%253Dinfo%253Apmid%252F15753290%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTAyLzEyLzQ0NTkiO3M6NDoiYXRvbSI7czoxOToiL2dnZy8yLzEwLzEyNDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/2/10/1243.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/2/10/1233" title="Second-Generation Genetic Linkage Map of Catfish and Its Integration with the BAC-Based Physical Map" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/2/10/1257" title="Exploration of the Genetic Organization of Morphological Modularity on the Mouse Mandible Using a Set of Interspecific Recombinant Congenic Strains Between C57BL/6 and Mice of the Mus spretus Species" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/2/10"><span>Volume 2 Issue 10, October 2012</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/2/10.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/2/10.cover-source.jpg?itok=qNC6A40M" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 2 (10)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/2/10/1243" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-71IApaUqtRfUusmjcv8M-K_sx_8MYlZ2_n7GSopuSlk" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/2/10/1243" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/2/10/1243" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/2/10/1243" class="active" data-icon-position="" data-hide-link-title="0">Overlapping ETS and CRE Motifs (G/CCGGAAGTGACGTCA) Preferentially Bound by GABPα and CREB Proteins</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459331" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-0kth3NBjpH0Wzd30TWrf76Kxk9mJyhISp2pHHFSVQys" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/2/10/1243.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/2/10/1243" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-sAZlMP75OlSO1vDGSDD3IW0dYG3Uk6ENsxuAdT19Gb4" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/2/10/1243" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/2/10/1243.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459331" id="node459331--2" data-pisa="ggg;2/10/1243" data-pisa-master="ggg;2/10/1243" data-apath="/ggg/2/10/1243.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Overlapping ETS and CRE Motifs (<sup>G</sup>/<sub>C</sub>CGGAAGTGACGTCA) Preferentially Bound by GABPα and CREB Proteins</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Raghunath Chatterjee</span>, <span class="highwire-citation-author" data-delta="1">Jianfei Zhao</span>, <span class="highwire-citation-author" data-delta="2">Ximiao He</span>, <span class="highwire-citation-author" data-delta="3">Andrey Shlyakhtenko</span>, <span class="highwire-citation-author" data-delta="4">Ishminder Mann</span>, <span class="highwire-citation-author" data-delta="5">Joshua J. Waterfall</span>, <span class="highwire-citation-author" data-delta="6">Paul Meltzer</span>, <span class="highwire-citation-author" data-delta="7">B. K. Sathyanarayana</span>, <span class="highwire-citation-author" data-delta="8">Peter C. FitzGerald</span> and <span class="highwire-citation-author" data-delta="9">Charles Vinson</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2012 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 2 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1243-1256;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.004002 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459331--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Raghunath Chatterjee </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Laboratory of Metabolism, National Cancer Institute, National Institutes of Health, Bethesda, Maryland 20892</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Raghunath%2BChatterjee%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chatterjee%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARaghunath%2BChatterjee%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jianfei Zhao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Laboratory of Metabolism, National Cancer Institute, National Institutes of Health, Bethesda, Maryland 20892</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jianfei%2BZhao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhao%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJianfei%2BZhao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Ximiao He </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Laboratory of Metabolism, National Cancer Institute, National Institutes of Health, Bethesda, Maryland 20892</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ximiao%2BHe%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=He%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AXimiao%2BHe%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Andrey Shlyakhtenko </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Laboratory of Metabolism, National Cancer Institute, National Institutes of Health, Bethesda, Maryland 20892</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andrey%2BShlyakhtenko%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Shlyakhtenko%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAndrey%2BShlyakhtenko%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Ishminder Mann </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Laboratory of Metabolism, National Cancer Institute, National Institutes of Health, Bethesda, Maryland 20892</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ishminder%2BMann%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mann%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AIshminder%2BMann%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Joshua J. Waterfall </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Genetics Branch, National Cancer Institute, National Institutes of Health, Bethesda, Maryland 20892</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joshua%2BJ.%2BWaterfall%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Waterfall%20JJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJoshua%2BJ.%2BWaterfall%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Paul Meltzer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Genetics Branch, National Cancer Institute, National Institutes of Health, Bethesda, Maryland 20892</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Paul%2BMeltzer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Meltzer%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APaul%2BMeltzer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">B. K. Sathyanarayana </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Laboratory of Molecular Biology, National Cancer Institute, National Institutes of Health, Bethesda, Maryland 20892</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=B.%2BK.%2BSathyanarayana%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sathyanarayana%20BK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AB.%2BK.%2BSathyanarayana%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Peter C. FitzGerald </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Genome Analysis Unit, National Cancer Institute, National Institutes of Health, Bethesda, Maryland 20892</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Peter%2BC.%2BFitzGerald%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=FitzGerald%20PC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APeter%2BC.%2BFitzGerald%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Charles Vinson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Laboratory of Metabolism, National Cancer Institute, National Institutes of Health, Bethesda, Maryland 20892</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Charles%2BVinson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Vinson%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ACharles%2BVinson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:Vinsonc@mail.nih.gov" class="" data-icon-position="" data-hide-link-title="0">Vinsonc@mail.nih.gov</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/2/10/1243" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/2/10/1243&amp;title=Overlapping%20ETS%20and%20CRE%20Motifs%20%28G/CCGGAAGTGACGTCA%29%20Preferentially%20Bound%20by%20GABP%CE%B1%20and%20CREB%20Proteins" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/2/10/1243&amp;title=Overlapping%20ETS%20and%20CRE%20Motifs%20%28G/CCGGAAGTGACGTCA%29%20Preferentially%20Bound%20by%20GABP%CE%B1%20and%20CREB%20Proteins" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/2/10/1243&amp;title=Overlapping%20ETS%20and%20CRE%20Motifs%20%28G/CCGGAAGTGACGTCA%29%20Preferentially%20Bound%20by%20GABP%CE%B1%20and%20CREB%20Proteins" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/2/10/1243&amp;text=Overlapping%20ETS%20and%20CRE%20Motifs%20%28G/CCGGAAGTGACGTCA%29%20Preferentially%20Bound%20by%20GABP%CE%B1%20and%20CREB%20Proteins" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/2/10/1243&amp;title=Overlapping%20ETS%20and%20CRE%20Motifs%20%28G/CCGGAAGTGACGTCA%29%20Preferentially%20Bound%20by%20GABP%CE%B1%20and%20CREB%20Proteins" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/2/10/1243&amp;t=Overlapping%20ETS%20and%20CRE%20Motifs%20%28G/CCGGAAGTGACGTCA%29%20Preferentially%20Bound%20by%20GABP%CE%B1%20and%20CREB%20Proteins" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/2/10/1243&amp;title=Overlapping%20ETS%20and%20CRE%20Motifs%20%28G/CCGGAAGTGACGTCA%29%20Preferentially%20Bound%20by%20GABP%CE%B1%20and%20CREB%20Proteins" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/2/10/1243&amp;title=Overlapping%20ETS%20and%20CRE%20Motifs%20%28G/CCGGAAGTGACGTCA%29%20Preferentially%20Bound%20by%20GABP%CE%B1%20and%20CREB%20Proteins" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459331/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459331" id="node459331--3" data-pisa="ggg;2/10/1243" data-pisa-master="ggg;2/10/1243" data-apath="/ggg/2/10/1243.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Overlapping ETS and CRE Motifs (<sup>G</sup>/<sub>C</sub>CGGAAGTGACGTCA) Preferentially Bound by GABPα and CREB Proteins</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Raghunath Chatterjee</span>, <span class="highwire-citation-author" data-delta="1">Jianfei Zhao</span>, <span class="highwire-citation-author" data-delta="2">Ximiao He</span>, <span class="highwire-citation-author" data-delta="3">Andrey Shlyakhtenko</span>, <span class="highwire-citation-author" data-delta="4">Ishminder Mann</span>, <span class="highwire-citation-author" data-delta="5">Joshua J. Waterfall</span>, <span class="highwire-citation-author" data-delta="6">Paul Meltzer</span>, <span class="highwire-citation-author" data-delta="7">B. K. Sathyanarayana</span>, <span class="highwire-citation-author" data-delta="8">Peter C. FitzGerald</span> and <span class="highwire-citation-author" data-delta="9">Charles Vinson</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2012 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 2 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1243-1256;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.004002 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459331--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Raghunath Chatterjee </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Laboratory of Metabolism, National Cancer Institute, National Institutes of Health, Bethesda, Maryland 20892</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Raghunath%2BChatterjee%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chatterjee%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARaghunath%2BChatterjee%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jianfei Zhao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Laboratory of Metabolism, National Cancer Institute, National Institutes of Health, Bethesda, Maryland 20892</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jianfei%2BZhao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhao%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJianfei%2BZhao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Ximiao He </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Laboratory of Metabolism, National Cancer Institute, National Institutes of Health, Bethesda, Maryland 20892</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ximiao%2BHe%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=He%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AXimiao%2BHe%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Andrey Shlyakhtenko </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Laboratory of Metabolism, National Cancer Institute, National Institutes of Health, Bethesda, Maryland 20892</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andrey%2BShlyakhtenko%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Shlyakhtenko%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAndrey%2BShlyakhtenko%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Ishminder Mann </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Laboratory of Metabolism, National Cancer Institute, National Institutes of Health, Bethesda, Maryland 20892</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ishminder%2BMann%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mann%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AIshminder%2BMann%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Joshua J. Waterfall </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Genetics Branch, National Cancer Institute, National Institutes of Health, Bethesda, Maryland 20892</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joshua%2BJ.%2BWaterfall%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Waterfall%20JJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJoshua%2BJ.%2BWaterfall%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Paul Meltzer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Genetics Branch, National Cancer Institute, National Institutes of Health, Bethesda, Maryland 20892</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Paul%2BMeltzer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Meltzer%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APaul%2BMeltzer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">B. K. Sathyanarayana </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Laboratory of Molecular Biology, National Cancer Institute, National Institutes of Health, Bethesda, Maryland 20892</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=B.%2BK.%2BSathyanarayana%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sathyanarayana%20BK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AB.%2BK.%2BSathyanarayana%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Peter C. FitzGerald </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Genome Analysis Unit, National Cancer Institute, National Institutes of Health, Bethesda, Maryland 20892</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Peter%2BC.%2BFitzGerald%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=FitzGerald%20PC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APeter%2BC.%2BFitzGerald%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Charles Vinson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Laboratory of Metabolism, National Cancer Institute, National Institutes of Health, Bethesda, Maryland 20892</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Charles%2BVinson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Vinson%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ACharles%2BVinson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:Vinsonc@mail.nih.gov" class="" data-icon-position="" data-hide-link-title="0">Vinsonc@mail.nih.gov</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1243&amp;rft.epage=1256&amp;rft.atitle=Overlapping%20ETS%20and%20CRE%20Motifs%20%28G/CCGGAAGTGACGTCA%29%20Preferentially%20Bound%20by%20GABP%CE%B1%20and%20CREB%20Proteins&amp;rft.volume=2&amp;rft.issue=10&amp;rft.date=2012-10-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Chatterjee%2C+Raghunath&amp;rft.au=Zhao%2C+Jianfei&amp;rft.au=He%2C+Ximiao&amp;rft.au=Shlyakhtenko%2C+Andrey&amp;rft.au=Mann%2C+Ishminder&amp;rft.au=Waterfall%2C+Joshua+J.&amp;rft.au=Meltzer%2C+Paul&amp;rft.au=Sathyanarayana%2C+B.+K.&amp;rft.au=FitzGerald%2C+Peter+C.&amp;rft.au=Vinson%2C+Charles"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459331/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459331/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459331/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459331/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459331/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459331/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459331/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459331/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459331/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459331/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459331/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459331/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/2/10/1243" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-9" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-21" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/2/10/1243.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/2/10/1243.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/2/10/1243.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/2/10/1243.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
