<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/2/10/1179.full.txt" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/2/10/1179.full.pdf" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/2/10.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/2/10/1179/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/2/10/1179.atom" />
<meta name="HW.pisa" content="ggg;2/10/1179" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="A Fast and Efficient Approach for Genomic Selection with High-Density Markers" />
<meta name="DC.Identifier" content="10.1534/g3.112.003822" />
<meta name="DC.Date" content="2012-10-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright Â© 2012 Pungpapong et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Recent advances in high-throughput genotyping have motivated genomic selection using high-density markers. However, an increasingly large number of markers brings up both statistical and computational issues and makes it difficult to estimate the breeding values. We propose to apply the penalized orthogonal-components regression (POCRE) method to estimate breeding values. As a supervised dimension reduction method, POCRE sequentially constructs linear combinations of markers, i.e. orthogonal components, such that these components are most closely correlated to the phenotype. Such a dimension reduction is able to group highly correlated predictors and allows for collinear or nearly collinear markers. Different from BayesB, which predetermines hyperparameters, POCRE uses an empirical Bayes thresholding method to obtain data-driven optimal hyperparameters and effectively select important markers when constructing each component. Demonstrated through simulation studies, POCRE greatly reduces the computing time compared with BayesB. On the other hand, unlike fBayesB which slightly sacrifices prediction accuracy for fast computation, POCRE provides similar or even better accuracy of predicting breeding values than BayesB in both simulation studies and real data analyses." />
<meta name="DC.Contributor" content="Vitara Pungpapong" />
<meta name="DC.Contributor" content="William M. Muir" />
<meta name="DC.Contributor" content="Xianran Li" />
<meta name="DC.Contributor" content="Dabao Zhang" />
<meta name="DC.Contributor" content="Min Zhang" />
<meta name="article:published_time" content="2012-10-01" />
<meta name="article:section" content="Genomic Selection" />
<meta name="citation_title" content="A Fast and Efficient Approach for Genomic Selection with High-Density Markers" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Recent advances in high-throughput genotyping have motivated genomic selection using high-density markers. However, an increasingly large number of markers brings up both statistical and computational issues and makes it difficult to estimate the breeding values. We propose to apply the penalized orthogonal-components regression (POCRE) method to estimate breeding values. As a supervised dimension reduction method, POCRE sequentially constructs linear combinations of markers, &lt;i&gt;i.e.&lt;/i&gt; orthogonal components, such that these components are most closely correlated to the phenotype. Such a dimension reduction is able to group highly correlated predictors and allows for collinear or nearly collinear markers. Different from BayesB, which predetermines hyperparameters, POCRE uses an empirical Bayes thresholding method to obtain data-driven optimal hyperparameters and effectively select important markers when constructing each component. Demonstrated through simulation studies, POCRE greatly reduces the computing time compared with BayesB. On the other hand, unlike fBayesB which slightly sacrifices prediction accuracy for fast computation, POCRE provides similar or even better accuracy of predicting breeding values than BayesB in both simulation studies and real data analyses.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2012/10/01" />
<meta name="citation_mjid" content="ggg;2/10/1179" />
<meta name="citation_id" content="2/10/1179" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/2/10/1179" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/2/10/1179.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/2/10/1179.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/2/10/1179.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.112.003822" />
<meta name="citation_pmid" content="23050228" />
<meta name="citation_volume" content="2" />
<meta name="citation_issue" content="10" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Genomic Selection" />
<meta name="citation_firstpage" content="1179" />
<meta name="citation_lastpage" content="1184" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Vitara Pungpapong" />
<meta name="citation_author_institution" content="Department of Statistics, Purdue University, West Lafayette, Indiana 47907" />
<meta name="citation_author_institution" content="Department of Statistics, Chulalongkorn Business School, Bangkok, Thailand 10330" />
<meta name="citation_author" content="William M. Muir" />
<meta name="citation_author_institution" content="Department of Animal Science, Purdue University, West Lafayette, Indiana 47907" />
<meta name="citation_author" content="Xianran Li" />
<meta name="citation_author_institution" content="Department of Agronomy, Kansas State University, Manhattan, Kansas 66506, and" />
<meta name="citation_author" content="Dabao Zhang" />
<meta name="citation_author_institution" content="Department of Statistics, Purdue University, West Lafayette, Indiana 47907" />
<meta name="citation_author_institution" content="School of Biomedical Engineering, Capital Medical University, Beijing, China 100069" />
<meta name="citation_author_email" content="zhangdb@stat.purdue.edu" />
<meta name="citation_author_email" content="minzhang@stat.purdue.edu" />
<meta name="citation_author" content="Min Zhang" />
<meta name="citation_author_institution" content="Department of Statistics, Purdue University, West Lafayette, Indiana 47907" />
<meta name="citation_author_institution" content="School of Biomedical Engineering, Capital Medical University, Beijing, China 100069" />
<meta name="citation_author_email" content="zhangdb@stat.purdue.edu" />
<meta name="citation_author_email" content="minzhang@stat.purdue.edu" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=R. Bernardo;citation_author=J. Yu;citation_title=Prospects for Genomewide Selection for Quantitative Traits in Maize;citation_pages=1082-1090;citation_volume=47;citation_year=2007;citation_issue=3;citation_doi=10.2135/cropsci2006.11.0690" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=E. S. Buckler;citation_author=J. B. Holland;citation_author=P. J. Bradbury;citation_author=C. B. Acharya;citation_author=P. J. Brown;citation_title=The Genetic Architecture of Maize Flowering Time;citation_pages=714-718;citation_volume=325;citation_year=2009;citation_issue=5941;citation_pmid=19661422;citation_doi=10.1126/science.1174276" />
<meta name="citation_reference" content="citation_journal_abbrev=J Anim Breed Genet;citation_author=M. P. L. Calus;citation_author=R. F. Veerkamp;citation_title=Accuracy of breeding values when using and ignoring the polygenic effect in genomic breeding value estimation with a marker density of one SNP per cM.;citation_pages=362-368;citation_volume=124;citation_year=2007;citation_issue=6;citation_pmid=18076473" />
<meta name="citation_reference" content="citation_title=Boosting and Naive Bayesian Learning;citation_author=C. Elkan;citation_year=1997" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=M. A. Gore;citation_author=J. M. Chia;citation_author=R. J. Elshire;citation_author=Q. Sun;citation_author=E. S. Ersoz;citation_title=A First-Generation Haplotype Map of Maize;citation_pages=1115-1117;citation_volume=326;citation_year=2009;citation_issue=5956;citation_pmid=19965431;citation_doi=10.1126/science.1177837" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=B. J. Hayes;citation_author=P. J. Bowman;citation_author=A. J. Chamberlain;citation_author=M. E. Goddard;citation_title=Invited review: Genomic selection in dairy cattle: Progress and challenges;citation_pages=433-443;citation_volume=92;citation_year=2009;citation_issue=2;citation_pmid=19164653;citation_doi=10.3168/jds.2008-1646" />
<meta name="citation_reference" content="citation_journal_title=Briefings in Functional Genomics;citation_author=J.-L. Jannink;citation_author=A. J. Lorenz;citation_author=H. Iwata;citation_title=Genomic selection in plant breeding: from theory to practics;citation_volume=9;citation_year=2010;citation_pmid=20156985;citation_doi=10.1093/bfgp/elq001" />
<meta name="citation_reference" content="citation_journal_title=Ann. Stat.;citation_author=I. M. Johnstone;citation_author=B. W. Silverman;citation_title=Needles and straw in haystacks: empirical Bayes estimates of possibly sparse sequences;citation_pages=1594-1649;citation_volume=32;citation_year=2004;citation_doi=10.1214/009053604000000030" />
<meta name="citation_reference" content="citation_journal_abbrev=J Anim Breed Genet;citation_author=N. Long;citation_author=D. Gianola;citation_author=G. J. M. Rose;citation_author=K. A. Weigel;citation_author=S. Avendano;citation_title=Machine learning classification procedure for selecting SNPs in genomic selection: application to early mortality in broilers.;citation_pages=377-389;citation_volume=124;citation_year=2007;citation_issue=6;citation_pmid=18076475" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=M. D. McMullen;citation_author=S. Kresovich;citation_author=H. S. Villeda;citation_author=P. Bradbury;citation_author=H. Li;citation_title=Genetic Properties of the Maize Nested Association Mapping Population;citation_pages=737-740;citation_volume=325;citation_year=2009;citation_issue=5941;citation_pmid=19661427;citation_doi=10.1126/science.1174320" />
<meta name="citation_reference" content="citation_journal_abbrev=J Anim Breed Genet;citation_author=T. H. E. Meuwissen;citation_title=Genomic selection: marker assisted selection on a genome wide scale.;citation_pages=321-322;citation_volume=124;citation_year=2007;citation_issue=6;citation_pmid=18076468" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=T. H. E. Meuwissen;citation_author=B. J. Hayes;citation_author=M. E. Goddard;citation_title=Prediction of Total Genetic Value Using Genome-Wide Dense Marker Maps;citation_pages=1819-1829;citation_volume=157;citation_year=2001;citation_issue=4;citation_pmid=11290733" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=T. H. E. Meuwissen;citation_author=T. R. Solberg;citation_author=R. Shepherd;citation_author=J. A. Woolliams;citation_title=A fast algorithm for BayesB type of prediction of genome-wide estimates of genetic value.;citation_pages=2-2;citation_volume=41;citation_year=2009;citation_pmid=19284681;citation_doi=10.1186/1297-9686-41-2" />
<meta name="citation_reference" content="citation_title=Machine Learning;citation_author=T. M. Mitchell;citation_year=1997" />
<meta name="citation_reference" content="citation_journal_abbrev=J Anim Breed Genet;citation_author=W. M. Muir;citation_title=Comparison of genomic and traditional BLUP-estimated breeding value accuracy and selection response under alternative trait and genomic parameters.;citation_pages=342-355;citation_volume=124;citation_year=2007;citation_issue=6;citation_pmid=18076471;citation_doi=10.1111/j.1439-0388.2007.00700.x" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. F. R. Resende;citation_author=P. Munoz;citation_author=M. D. V. Resende;citation_author=D. J. Garrick;citation_author=R. L. Fernando;citation_title=Accuracy of genomic selection methods in a standard dataset of Lobolly Pine (Pinus taeda L.);citation_volume=190;citation_year=2012;citation_pmid=22271763;citation_doi=10.1534/genetics.111.137026" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=P. Scheet;citation_author=M. Stephens;citation_title=A fast and flexible statistical model for large-scale population genotype data: applications to inferring missing genotypes and haplotypic phase.;citation_pages=629-644;citation_volume=78;citation_year=2006;citation_issue=4;citation_pmid=16532393;citation_doi=10.1086/502802" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=T. R. Solberg;citation_author=A. K. Sonesson;citation_author=J. A. Woolliams;citation_author=T. H. E. Meuwissen;citation_title=Reducing dimensionality for prediction of genome-wide breeding values.;citation_pages=29-29;citation_volume=41;citation_year=2009;citation_pmid=19296851;citation_doi=10.1186/1297-9686-41-29" />
<meta name="citation_reference" content="citation_journal_title=J. R. Stat. Soc., B;citation_author=R. J. Tibshirani;citation_title=Regression shrinkage and selection via the lasso;citation_pages=267-288;citation_volume=58;citation_year=1996" />
<meta name="citation_reference" content="citation_journal_title=J. R. Stat. Soc., B;citation_author=R. J. Tibshirani;citation_author=S. Rosset;citation_author=J. Zhu;citation_author=K. Knight;citation_title=Sparsity and smoothness via the fused lasso;citation_pages=91-108;citation_volume=67;citation_year=2005;citation_doi=10.1111/j.1467-9868.2005.00490.x" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=M. G. Usai;citation_author=M. E. Goddard;citation_author=B. J. Hayes;citation_title=LASSO with cross-validation for genomic selection.;citation_pages=427-436;citation_volume=91;citation_year=2009;citation_issue=6;citation_pmid=20122298;citation_doi=10.1017/S0016672309990334" />
<meta name="citation_reference" content="citation_journal_title=Biometrika;citation_author=H. Wang;citation_author=R. Li;citation_author=C. Tsai;citation_title=Tuning parameter selectors for the smoothly clipped absolute deviation method;citation_volume=94;citation_year=2007;citation_pmid=19343105;citation_doi=10.1093/biomet/asm053" />
<meta name="citation_reference" content="citation_title=Perspectives in Probability and Statistics, Papers in Honour of M. S. Bartlett;citation_title=Soft modelling by latent variables: the nonlinear iterative partial least squares approach, pp. 117â142;citation_author=H. Wold;citation_year=1975" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. Yu;citation_author=J. B. Holland;citation_author=M. D. McMullen;citation_author=E. S. Buckler;citation_title=Genetic design and statistical power of nested association mapping in maize;citation_pages=539-551;citation_volume=178;citation_year=2008;citation_pmid=18202393;citation_doi=10.1534/genetics.107.074245" />
<meta name="citation_reference" content="citation_journal_title=J. R. Stat. Soc., B;citation_author=M. Yuan;citation_author=Y. Lin;citation_title=Model selection and estimation in regression with grouped varaible;citation_pages=49-67;citation_volume=68;citation_year=2006;citation_doi=10.1111/j.1467-9868.2005.00532.x" />
<meta name="citation_reference" content="citation_journal_title=Electronic Journal of Statistics;citation_author=D. Zhang;citation_author=Y. Lin;citation_author=M. Zhang;citation_title=Penalized orthogonal-components regression for large p small n data;citation_pages=781-796;citation_volume=3;citation_year=2009;citation_doi=10.1214/09-EJS354" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. Zhong;citation_author=J. C. M. Dekkers;citation_author=R. L. Fernando;citation_author=J.-L. Jannink;citation_title=Factors affecting accuracy from genomic selection in populaitons derived from multiple inbred lines: a barley case study;citation_pages=355-364;citation_volume=182;citation_year=2009;citation_pmid=19299342;citation_doi=10.1534/genetics.108.098277" />
<meta name="citation_reference" content="citation_journal_title=J. R. Stat. Soc., B;citation_author=H. Zou;citation_author=T. Hastie;citation_title=Regularization and variable selection via the elastic net;citation_pages=301-320;citation_volume=67;citation_year=2005;citation_doi=10.1111/j.1467-9868.2005.00503.x" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="A Fast and Efficient Approach for Genomic Selection with High-Density Markers" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/2/10.cover-source.jpg" />
<meta name="twitter:description" content="Recent advances in high-throughput genotyping have motivated genomic selection using high-density markers. However, an increasingly large number of markers brings up both statistical and computational issues and makes it difficult to estimate the breeding values. We propose to apply the penalized orthogonal-components regression (POCRE) method to estimate breeding values. As a supervised dimension reduction method, POCRE sequentially constructs linear combinations of markers, i.e. orthogonal components, such that these components are most closely correlated to the phenotype. Such a dimension reduction is able to group highly correlated predictors and allows for collinear or nearly collinear markers. Different from BayesB, which predetermines hyperparameters, POCRE uses an empirical Bayes thresholding method to obtain data-driven optimal hyperparameters and effectively select important markers when constructing each component. Demonstrated through simulation studies, POCRE greatly reduces the computing time compared with BayesB. On the other hand, unlike fBayesB which slightly sacrifices prediction accuracy for fast computation, POCRE provides similar or even better accuracy of predicting breeding values than BayesB in both simulation studies and real data analyses." />
<meta name="og-title" property="og:title" content="A Fast and Efficient Approach for Genomic Selection with High-Density Markers" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/2/10/1179" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Recent advances in high-throughput genotyping have motivated genomic selection using high-density markers. However, an increasingly large number of markers brings up both statistical and computational issues and makes it difficult to estimate the breeding values. We propose to apply the penalized orthogonal-components regression (POCRE) method to estimate breeding values. As a supervised dimension reduction method, POCRE sequentially constructs linear combinations of markers, i.e. orthogonal components, such that these components are most closely correlated to the phenotype. Such a dimension reduction is able to group highly correlated predictors and allows for collinear or nearly collinear markers. Different from BayesB, which predetermines hyperparameters, POCRE uses an empirical Bayes thresholding method to obtain data-driven optimal hyperparameters and effectively select important markers when constructing each component. Demonstrated through simulation studies, POCRE greatly reduces the computing time compared with BayesB. On the other hand, unlike fBayesB which slightly sacrifices prediction accuracy for fast computation, POCRE provides similar or even better accuracy of predicting breeding values than BayesB in both simulation studies and real data analyses." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/2/10.cover-source.jpg" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.g3journal.org/content/2/10/1179" />
    <title>A Fast and Efficient Approach for Genomic Selection with High-Density Markers | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings, {"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"3f9WKwZwvNXwpFKIwEfqLfEOBfNF75RYpy1tCDKg_Gc","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459336","apath":"\/ggg\/2\/10\/1179.atom","pisa":"ggg;2\/10\/1179","ac":{"\/ggg\/2\/10\/1179.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/2\/10\/1179.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;2\/10\/1179"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;2\/10\/1179"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/2\/10\/1179":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022Hz1+bXmtBvdxqNeZhHfDUygpGC1VlmBwFtG1hIqOtz8aywWLKACELAWRxzd9Fz4drnI8HLzbhg8vtsGk1eeTY2SRZKjnKfDw10O1NQeQ\\\\\\\/hGAl8FP1yeoHzvf9oALTKXdGeGjuCIV7EVwgLEb5WMIRnSEPtLe2cFK6Ek3sAPTxcvv7DBNxMMevxPx9Xob1zYJukBtA4Ale1yCuEOvyekzUKm2ifiORVadStgZei3UHnrVUVM58Du9BgAvNqviSi07LjeCk2E5DEWBHaAjaR1pnhYw9Y3k5fEXYq5LtH0XZUPg7r65JqKIgQMN8uAqV6FVs+P4Pe\\\\\\\/JX5vE8SrkqrH8Dx\\\\\\\/WHDjMxOohlExlwyNlx6Sul1l1faPwUomutQaS5GMNBdWzWRZ0\\\\\\\/ILcj5SnkIcjcTSsy2DxPvFRSk06Zu7baZABKuBdhXyJao2BXnyB3lHqGxGTZqLBvIfpL2q2CqBHn2n88+bSJeZWeKzjJBQBgds+prA3JIK6kwgnyjqds3KUTeDf2e0XXqjNxuVkK0H1NBp0z7EJOOXYf4P4FQU1h2WmIpGYoDflSHQbzbxqf8NiOcD2PMY0MDXlhWWyvEtsX6TrCo2hmfSkznHgotrelYYh74A4QeiOdTV2M87\\\\\\\/v1BgpjTpxAInVQBmI78al+hy1hMJSrL4bbujLa3RVwHwYix2F5tbr1tg+sBK+DOkUGg1sa9XzPU6Q2GFCSywdMkzQkWVfQRzxRncWo+YrZAvXMOO0cLQ2VHvTT5h4JeFUQUxhBCa7jwLyRLXyWplS6sMm\\\\\\\/5goqUSbszXUbQx8W6mjYXTNitG44k2aMB+Z1Up0x3535SsE1kvvchcB\\\\\\\/FZG30EFWdzygPZm52TqAF\\\\\\\/7qcAcRhZpBDN9NBVu0jNYmqTtkFJ9dW0rrcpYi57Eu97CnChJ\\\\\\\/9aZsVSAgsjarMi4WTMX8DxVUPmSE1YUrLMB6\\\\\\\/Q7qB5npIzNl01w76JT\\\\\\\/9K7gJpS1X4z1klD8HY44of1nxJeTdo4Y5liqzCtfxhJaMab7qAv9GQ0aMU+IwdStuiTvahz4SYZMDW3ktCJTRHchmMLPKnOM79OiBHdEysGTSqvTKdDnP2iFLlXJG8LzjCWEnQatcdliVX2\\\\\\\/md6radCqze716Kp51F1E69JS8ghfr14npNkByNad7oi7GKA4jMaGKk+71Uoi7tzrc7ELAoS0KGgYjfwPncGYRlnPuRd39CcMHVhyAvAf+sR1Rbe1Kk7xnmwayGi\\\\\\\/McRuHH16Bfr+sVj84FqKh4E4iHqsBuXg9tHiyX\\\u0022,\\\u0022iv\\\u0022:\\\u0022mW3B8jMcuHKfS6ORsYmfDw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002226ecef20edf56363c664a5df91b6c64c359d3877345fd4f3285c9a9edd94b4b9\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u002260qo9jD4h5oUani9Iy7WplIADQMIqosHxO2NaJQBp81wg7AheMFYT0RZxFJnZAlDnAKLBZewL2LbOSEqM6jmN\\\\\\\/+JQOEXrxKMt8SeY1I\\\\\\\/fbQkny7mbEEjkY0kTO7VBEKPAbAksTZYzTmyufT21rvMHEeRe8dhYiYO1ZAcSAPYtV9mX7NExhXFwJRsh2tLZJ0es1qx65gTEtEQzcujTUIywEV+9xyKRUn8B\\\\\\\/htXSs9FRrZDO54G8ps6zLLyqJjIY90b3ln0V4Y9ljGEpvExGb97x+w\\\\\\\/j+5lCwCGGIWKdc\\\\\\\/MJOhIOzAE0nLaLWk+NsCpmisgf98USP2AHXjRBGZo81vc+Zmehzlb6u7ocUzf2zqNG1hU56NzxeBtT4eTJXaPyhu7nKyoUDvrmKDSpcMN0QRK7OUphc6xOL3ufAHm+l\\\\\\\/PYz88c8Q36OTmzaXBOX02KetEeQXz\\\\\\\/apkBQaN9ZseokmMz5c9K+6A6jJU8gaRR0ehvTOim40vVYXWxws7\\\\\\\/XYw42\\\\\\\/3o5Hq9\\\\\\\/8UN16nPMcocgGVSMkQdq7nZMgRL6D65Upnh0svGZZ\\\\\\\/Z0gJMXOTwGi\\\\\\\/FkNgdZCl0P6SXZMnH3k8QukoOLQOJSshZO+mkzU7kABJPGOWuDJYThkGtclHfurNtP9MMFwfEcwIf1yr\\\\\\\/FJja\\\\\\\/C\\\\\\\/XAQ5dJL08ULRwda23YAnxm\\\\\\\/TaHm8zzEbB5ar6zro1LRZu9D8JNp8yTmI4ycDDJxw9ZL1q7E9gSikNJ0pWdilnSeBdBqwmB0k3JobiT2G8QDGd\\\\\\\/NrWSMqLKTmYOdzZXFV\\\\\\\/pYW0lV0FFA3IZQZe9hFXQ1xdEnwEj3dh5jN7L3bkS1YUWM3RqPrbCea2MYOWvkaaMN\\\\\\\/tuo3jlq7N5vNMgopcim9HJDasLMfyhE+f\\\\\\\/\\\\\\\/ru29OQPAntHLB\\\\\\\/wgoUSVDBxhtw8K2VXzcli62CyTMgTqziwLyF4TpZjm89eXEabWKJ9sfUjIWQ4ljq1julrEs1kB9k589wPmjMwo7FOHAxOWrEaOZcRnRL43B97Fbo\\\\\\\/lNMcSZ2rymC37zj4VSQbO6RdI\\\\\\\/Yl+xKpPaOgUQOyIl7ljrsm0PPPYf8MeJ4xh4fw4Y76u0Vq2nQVuQBBMTUJUJaIwCNeh4TICpyYdMHqIp+u7oizBd3Y0lSVBGPYDbJwsJaaC2F5KVpWzVTFWCmaCfGcyLwcFYFpzwj2id5s67RPrSFxyBm1It7IzpGEHLeWS8TFoCkqnZPnIp\\\\\\\/nbz1CyT78L6SwnxIegPc8q7AG9XSsm+i5M7eqAdoGF+iTWMJhyJfSxMM6f6z13rs09IlfCkrHiX6LxWgF5e49lNOpzr5rzXFC\\\\\\\/sWFyKbnx8Plv2DQsh7\\\\\\\/clePOz8xcy3E9xBwI9LSFIBYmAMn1W1kedFeZBQQs3668b7YouX1u2TEoZ31Ocb5w9NEKzm\\\\\\\/\\\\\\\/4l3yEPVlzXtCFAu7UtKk\\\\\\\/SvLsNOWOuYR+x5yqIQKwDbH2Bbgdam7kK8v\\\\\\\/uMo1h7ASDp4iUEZTWhrK57QlV4piI1o2JJJ8\\\\\\\/5553649QFOTwdu1DCvY3mCWYnngmM5slkKK7MtbpyHZm7rk91xHZkC1LBVG8uTPeLSImUnCwzHd6pOjUE56aiTKwiZpOpuRcJ9Ez\\\\\\\/ZmSLf9IfEZNtC+zl+Ap4tbN4lOsgheW8wFkLXlkCd\\\\\\\/ALloHYdBZQvfMY8ztGHQ+ScK2i\\\\\\\/lI336zp0STIb2bZKa2OXhx56zoMm\\\\\\\/neM9UpUBuTUTPZ6Pv+RJgW2Ccj2Ex1d+1l7E+\\\\\\\/9mau22zEIEkfBH+UUgT95fVnU4CyARReTdOj3s4NdSGW6YwV8TXbhcPbjn6f18WN6MWyQZ3oOLJ37qV8kR+OfTrkGtNEp6rwrWxPfZtFqL9iT2Q6qvbMae9o8SCcZtpXk7LNcB8IfCXcbLDRVCfzJEQJdcp3Es5DjZo23VGZG0IHrr5iJf\\\\\\\/a02pxLLrBRhE5Id\\\\\\\/uIJtmgeAfK3ALcRnkjmesmCvjR9F1fgQaonhfJ2zJFSpBfwyNdR5ZgMRkz5qEmCX1adh2ZFsn1KMWOTugMfgDQr4Zt1fcDuLzxCGTbv2WHkQPuZGbkediq\\\\\\\/DYafOpmHTiciZmFd+2K6gCMjSKYwhySRVpMszXYP0gXbq2OzxQYJY0uuqrpw5kOZT9Z3DExUkuxl75ThIgTB\\\\\\\/SY9j4Kys9C07iXe6l2DueYRKQDGOHFsr4xc\\\\\\\/sxYiZsiqRyPebnSiam27LjavvZ0VVSEhpjKRP5+Vi1Lsq+EA3X5vrwCPw7cmkRJxmlS4VYHHG7K\\\\\\\/vj4c8hW2WKQ2VmxpCPRZ+MULLWwEGiZGdbzlFAYs9vRg1UE8x9\\\\\\\/ptnf8OjlXrSotuV5UzKD+4PIDYXdO6EZSsqKpc6KsrITad4\\\\\\\/8kEG4bLWPb3wnDPNxoVQVO5xu1PKvlIhoUX8ktCFElNHDDRnhQWeJrohq4UYcMJf7sDlLBTQheH+mwnTyV8Nj1hMmKq5q57MR0WWfg3qfLeSYSN1eQTBp6o9Mvqg\\\\\\\/BVuX23YUGQGAxVVyDZrlEpCxuSQq\\\\\\\/jueB8NnZjA\\\\\\\/Pb69ibEsFhjk0zeNunxD4PCb2OJXH6+tZYxKyYybzqFDrHurZ77E7IgwSnpo7BPf8jxY0JockdKXTLkDnXxB3yAaSXrA5X9iW1TvGlsl89CsOZOYP1aZhdnHka5gnXUa5qFaq\\\\\\\/r5+iZkhuXUwQ2s9FVo66bNHRjH5ayhalvBbqwXotvQJzCdqozEFDizE\\\\\\\/vroyB\\\\\\\/sB4Z70Pi5EX1NEpr87Ag6FqP3COgXtnRqf1zq8rkadGz813s5xZs41\\\\\\\/zOaOc++2J0Le71zZFjefwyNsxZwz0YYHRklxTonsK0RRMFu2NyF8qd+lhZIOpILU0Lm2WfMcbCOBzRgnxJ2exstr1JVBAwjgx2wpVxDnszKL0fMch0RBA2xY7L0otOHpG0YepJrP6XHiTZD8QKmOHh8b+Q\\\\\\\/A4OhCwVR8\\\\\\\/Vj1SdVUJJ9n4q5MSqMWWE2Qf4eUCoEs1fVbCPYktLkzh0ynEIXEgVan+XULfrCyyXAYetSCKlinR04j4ILFe5EWOXPUonGvU8YnVh7QUvA6EecL8YcdEZ9NQ2hSZW2ptgTsjxJa10kOBjkAT8NTGzGkMBipaaHkHTPmcrP8UtzkzH4mW4KJsIplCiehbb9dEzv8RsCgH6LEU78gM+o7ASjf6hgvziZrqYSC8tNb1ntYV\\\\\\\/bl804Iyf85uztPeQ4Z6Eq8eSZbSVPoZPJR9wP89Wf8FamxTVtuU\\\\\\\/P7laH4jeTkuUrekkdOzJRPSi7ou21+HwuAYPWqUPJIZKESfUN8QLfdy9y86cxwzZQ\\\\\\\/lwvmVselVNx7UlQq5TQ3cZQqv1J7ymDp2Bz+Udl1iywCK7s6Pq5IKmxC\\\\\\\/zu+hHgvcW3r7U5E4EvHfu0VVICJOUr1pzVVbJhmyqcKGkQd9Urp2FqzMo05x1PBvBWejVy+e8snKI17n2Jp3lxoYFExrnqZidRY5kLJMsLI5VEELvz5+B8R7t7VoOexar61PVfjgmAZRr3JAnlfC2qfBZ9FO6zL3K71oSowbC+H+Vx7mj6F8W8QyPBR93jwS2OipUhqO36uHjW3UXlQ4gkNPAajPZIfuMDUsmwGgvRigAqW2vqawdLBEgQ8R2A6tjW9ooowhh5iIwqQo53o34haRY48lAozpVOKxlhqvZFkJ9yW29WegN1UJbQ7lUPbCjJb3UUL8pdCDe0k2igdgEburEr5OmK+f73f6I+xYih6G0GUKY6bOHB4Ib8k\\\\\\\/SYZzdp5wwpZhK7nPY0YVSqBVFAth\\\\\\\/y1\\\\\\\/7S2zKdlagBK8P1Nd6O6Tm3R4nJ+xB6TxlQvWfbJnTRPqOizg1B+\\\\\\\/\\\\\\\/frnNVw0dq5PN4QSqC5xOfL8bPHOrY+gCYiYn1spYeySoQhF6cyaVSY\\\\\\\/BW5kMc\\\\\\\/tcOsCe+zvhQXaXQOGSYJnOTWpLbVQYaI1\\\\\\\/LqRq3qb37sHvE0fhrUR\\\\\\\/1Vgfe1h32UQF8o5Iu9D5xVTGUwt4\\\\\\\/5qU724iUzW+9RyD1IJUrXlTvepMpOr8hYal6uCYDe25ga28pkIaxBYtVsWxJ7BWxI5TaCDNNMtVpxo5bD4TIbdpT2Ea9Qisl3mjEuGAYDrQZs5TEr0+sk01hFbobfJBULrKDNfhQRAmMGBrBrA6G6W+r+V3YTlXyQHQUrZYiYOvz0oqhU7bH5ihQERf57c2czC7EKvNgHqY7Azu+VmTtmPsFfr+LS0QokF9vkEo7Zt17rQRpfEPu4ncY48KnaKadxi7LaWcXfxIJj\\\\\\\/ErjPI+TrIj8eMKgZ0c\\\\\\\/u9AemenuMCnlC+k9JHuQNxs5z5v7Jo24eZwDismBuTqST6lGKZNx2KXdWzF5o0HdwXjjrtQq+sK1FhS2D\\\\\\\/ev3De86NhF22v2rTtKmdya2CzIxdc3gHlahPZUBqkBw7\\\\\\\/p99mZrm3uMxQbB4nQzScWl9lexubbYt7\\\\\\\/moTr+AOaliDAFsJxQAXNEvuB3koKIFbMQku7bnavaa6BXapwtQDVHOVa6Mrr\\\\\\\/hKq\\\\\\\/61kcR9Q+coMam6m2ojbkU8RRmliRa6eSKPSOrvIo3kXRY1JTEhjXeqTuvGZIGO0mcMyRC+u1orM06TVOhw\\\\\\\/U6FpxONLci2vSm3uKkbaC0PIq\\\\\\\/EjAsfY2vfcbwNNrgGrC5V2uTfdG6GlNpK0+NYuT6grq4+WUcQFwb\\\\\\\/yWWjyzFMdAfDZlibsuu6w5kbvOx9kS6m\\\\\\\/YMuMMGh9dG3FPte+CcKsjsD2CHkfVqYh\\\\\\\/h2eN8hIhSoSlnNS+a6KRBeuK6gH6qKnqUuodp5kjDAxJs+TFs84Zpn4rRogLD1UzUDZA0DWMfPU2L2gYIVqnG7tp32x4qbKVOU4ERvxFENQ+6sokPSsD9aWy654KEmymRjekByWfY9MXBf1x0MMitmzIcmlvqDmWz6XJgt6sphxLR1LAbO3RF9IlX3hCZP16S5M67mVD9qO85ThyPjY3nQAn+dWQJPrDjLI036s+q6twhWWtXdwi6MXqUGouZHqFxXb4bJYzrVG8RhOThsbBlxO03GEl+8t5qs5pz59yGjTd\\\\\\\/Ev4ukwIaDYaU\\\\\\\/aXwcHUZczwOgxt+9wQn1wKvrQqO8YcJcShSfd1CIKDTK20YVl204bOjtG2+7fTa8k5dr4K6TTy+K\\\\\\\/DMCJNGGTYRf0NH\\\\\\\/jOPxC70uSj3nf82hAQEJ7H5gWaKR2XTsjDmvnfiEqBM344hLcd7\\\\\\\/WzqbyhnUm+s1TbK5ss04eqLIAhRMtFi16VzLQGWbdvqbi9or\\\\\\\/pgIARvVgyGg8\\\\\\\/H9rbk2hCEqezhZRQjFosuJAJovjucIBuQ2n2WEKeDLRQKM4sfZ2nBLcqCTfnO1I6NS6YCY8\\\\\\\/efXU2jDVQ0rho27An8rMJLGlUlEerfl3cBOYcWGKJk9o17etYlbtJIzRxW+jNbN7w0A9kNHTrruwTVoMgJfEOGlcwf6XDCdB0h+EDjFhSIUkVQUU\\\\\\\/\\\\\\\/BmZtezM\\\\\\\/bC0\\\\\\\/vypeagOJdac1p3xZKj5v2EXBVHQsqUFV0oU7sTrZWSLe69qzzhPysyPW7wt9CU93S3EJYPrjJjYWEnOcDmoMhJvw6IoHBOrkxwy1e\\\\\\\/4gzRFidqaUuj5J6x4gt3FeGSp6ODKQoJ6x6eebEebPZtAgiusVDnEeBvMwIIoHXT597O\\\\\\\/iiEpAah+3XM84h5a4OQTuHGEZLNTxg+1Aey2E7HzlvHNz7qzjZZ6PG49gSUtg9cex4mMaYaYwAr1l0kFv8hy6RLjwge0OElRAe+CkVRm7dFFzkAOxrUJwkpQ8WAnxgRtlgKPU1H00Hup\\\\\\\/Nr8asbWCmWu1V9Uwfz8WQJtWh\\\\\\\/T7+TKlMvb4OANAZTgGPrehcDGQ0KwkOVYVlRjc9wDbEKBRS969ETWkQOxM5+MDyEf9WxYuRHLs4sK+JF6vcNlUbDh\\\\\\\/hdw2MTRauPfb0G9vJXOIDg9rC9XY\\\\\\\/dclr8iuJgWx\\\\\\\/54uczPCSJoiXaMupEgn21c4Z2TK9HlN\\\\\\\/iih3gg4yFO9CYLpAezAJSn4YqFROL45Mx7sbWcoVtBJts6WpO5lD\\\\\\\/pT6A7oI7lk\\\\\\\/akq7oJKHv9VxUp18ABx87Yz3yPIziZCEZAEbVciRpoMFi3S6TaUlOH+O3fhHh49JXaXCUVeNrwvlj57HVIgMWwit6smtRi6yeswKmzASkG1RJCZYz\\\\\\\/2p1IbVf1KB3oBYYtsfDu\\\\\\\/TN+E4fIcXzwwYtSLYajriwszg8BcyXIUvxoY1xnOLjeq1P1Dho9aK33QCAn0TiJ5xN+rB\\\\\\\/EYujfq\\\\\\\/mp8AgYE4kIeSlqPnb1yUC8IApTfsyloq0iaLJbaLJicNcv8GJFjHZe+wTd7j8QjNLNe7\\\\\\\/FznbAiA7kza7FZb5tj02powfaUUoDHbVvnJITzzUvkSUVJf+GksvKdcdHZhcG25V68zisXP4PjXT3zniVl8AingxvDsD+Qpug8H03lJExCM40JuRgML3nI645D0WvFQf9toSAtB1qgA6WuS9MWluaAMI5b8WNpwLBl4rVyGof9aiDySwb1qw5gz63IdEo+XwPaIAso4+jZRvYk+TbHoxNC18+y9v+rF8s+JkbKcfEsgsfvLNhdu7APn3DKjFfAURDe0qAyWDeZm1UL1h2hfZDLR\\\\\\\/zyMGs64595L5Spmvo5cmbaELCUW6PBxZ5r4QgmsxQyDL\\\\\\\/45xhtASLo4zZE7iF8F75OhBCUvq4mkWIo0bU0I6OdbEJyqis+N3MbetgWJDh2AuSBSCyar8YfI7DqLa2TwSf5J6GWDFLO22JMnb2azCFaaHRR\\\\\\\/q7vDPs8B+LZaFYkOfVFwx+HLxfJw0aY75gtLVx5PWJSJB5aGWiE+LbmjpVCcmG0s1us80HtMpQxrTiL99\\\\\\\/5ILlV1Ei+LJjd1Myeq9zCaphQOM1c+6ImQnl2NVSkQriSBkLahiVyhQZkAc5pa0nh\\\\\\\/P\\\\\\\/N9NFhWhFcfhGzJLFbsnQtAesGR3AP6imu2m7O3hplOEsiYxWL2SibBEy9wX\\\\\\\/jyOI93odvIG5VjDocNWEK14qluaIDqfwTcH6bPFbZ+rmUmSPVpAkKWn33CSAfoYrOW656MZnpO\\\\\\\/GionT2V+amuda6sjL29o1\\\\\\\/snwtovRvCKbZZaIZpDLndkOUInYdjoIRyka\\\\\\\/dXtdTHZR95ZNzB9uOgiWk72SWbewnk9CJHqnk1qqkSAOUGvnnmXlqLoyVg5MwAKaRBRC3BUuQn5DtimbET2A5lA0Thw9\\\\\\\/OkIIsiPWdnAYgFkF6RSpJWb\\\\\\\/HUtnUMuVRYTW19IBpj50dMgIT+9NHxSWCmtUY3Qv1vbL7hVAfaecLa3F+5RL8BuCpnLAyT9c5AojaZ6WyIr1yOnuxdZgEVfuGb460DZYtJm2+cMZpKmFLqNCXoeAlSDUZcRKgm2ILxjVrj+4IPUrJik86fcp6SlTI7zkN++AgHtoyFZFh3kqDoPAuHvjR+zF3\\\\\\\/rOAy2D1Kj4ANNryzOGTvdGObjsBETmEUrgOrV8Zs9uNv1mjTbDZaOfe1IW4oUP57c4+xKeGUx+Kw4syfqzLuZrJU\\\\\\\/JWIz+RkuDe1zlpWEHLhYmCiorTWgVyrzo7irNYAlQaQ0spxw1Zsdr3ivLNRYM9SdB9YDkEDBoiPws6+F+4I6DRuh5pqVXaEPXtj7Jobh1CmbWaYB9qXXchPz1pi5bLKANKoPJTcLFgu6ACx8JoJ839MCMQL4AEyDv2ZGFs+3MAdinmBq6x+iB9QkjFuRt6mUCJU9zMz8tFsF8IejD9VOAuxDx+pZ9\\\\\\\/R\\\\\\\/TSNdpFnLb29GD5O34QtPrhNn1O7qGFsaUZqTt65zTu6iSCvyIPrSZBhF3OcAEXoLV+Ya3CjhejceGG1LB2SoeDYryzah29pKycY6w\\\\\\\/Lt3PuzL3fioyPB7mSD7JBqvofLWSJgH+ga1KDgBGpsr\\\\\\\/1+dQYjRDj4VQY7RRj2tG5xlqRhzSnwMXObYZN1pHvQsEBOQS6cTje+I5c2l7Egr5kUMSFA8uEeaXwy3oOhGlP8ARcHo7E97x20qyuD3WSVyI2zFXPlx5QJsTzNDHEZ\\\\\\\/Gxx3t+zc3l39IpcbCsCnoZHQBcMwT7OzuOHLCT+sucsZ4CnPf+ZW34BWTGyFdRgOYmAJ8RnTIOZMQBRdYiHw78tQ8YiFPo27lDFbsfgB6N3xacP6bJ7d1ZFKJ1QlxfF6o3l3zG5qwXxMw4BrYPP2z527TE75pdHH7yqnIJQf3gbUF5PsBASikN+OYSPY8gYNRUlhx\\\\\\\/VsOnL9l7I8hblfxxD21oupWTDZs\\\\\\\/sLrbrb6TOdw7SAztI9Qwtxbc+LF3MsBOyf8DBRc26Prqi\\\\\\\/8aJWWfHVMKCkEUkkT9ozJO6LYBv81oibk+ChZl8JUBiT2AmW7Nqbsy12oa5+7FN6CJmpW+HZAO1\\\\\\\/q9wDl73zQuGxDdyT+lMBtQMHW3BhFQfbJJib06+ibnUM14mL+FoykwKSXhP87sgddWg3Z2pI\\\\\\\/yW+r\\\\\\\/conR9i6eqtpKoRG9JPom4HfzeLrUrvVGi8pQqmSchb0ONRVMCwLT1QD7g1QkQifoS8FNRe\\\\\\\/McR3k9iz7LZxpFPj3j\\\\\\\/NetRYWVtBF1wOYBDf6w648ZJqCu7+mheHjsYniZBigdYf7xhcfNZLhFRs5RvIG1XvQ+Ah5WVMm+p3ah6HO4u9Z+\\\\\\\/Y0kaEdlI56O3oEKleq1nINS\\\\\\\/0tRGjblC6EwkOtCp4pJAwRJrOhW6WY0kAYtGDFO70UTp960EtexS26KbZ+OVFWN7QJU5gYh+Yaph3Kcn608JPFGC5+qfaXaN\\\\\\\/SQIvSalbty14dwgqTcpiTwqnFdNCYiHC5OgjJYpm9XVOiwXlAcbF\\\\\\\/drBgcGsvfdq4N\\\\\\\/bVasTB0hxd23q2esv70j\\\\\\\/3fyeHiJ7PaGcA0BRmioJVkuxRKiBKaTWM9sBn7nUKJVR0KCIHy7j1uUEldIuZiv5pmHJO8OkILNS1CNzrw5F48QTRXUELZnh82TUVW6iQDxw\\\\\\\/6d\\\\\\\/uvNxut8Z5FEpcqD+wrG0n39ECriiqzw2AS+Vn4YcufsCv7KbvCVyxZPzIUOMjsSf3n++tx01sXl+TAN\\\\\\\/vRPhvSNUipMs\\\\\\\/ZnmbecTq0u7O2nOVY3FnkCc4SvHjvC35S4\\\\\\\/D35KIjNVNH858jD0n6g0mc+Cwi8lYZZJnd4J9ZNom0uTAgIyGYlXs1IdaX9foHAd5n8WgvBXf6JsjP49xx9dbaE6\\\\\\\/\\\\\\\/2wcQpy\\\\\\\/Lck5yd6mu5yGu7UfYSIAv3wtMXUY1D8nMY8T3G5snZ+8iMvHtQxqeBwOPRFocPETTPtnQsNaLflwaCdA+PIaE8RekYg9R1aCAf1BaEbggSbxnjA1WOgaVxxx8rrsgJg4y4+AKHlM6ScKarE7ibhLCPDrswmFtAdRBU7SJb9dm6W8TzXo0qZP9SRMX2P292T+6L3B1hQ0\\\\\\\/Ox8XiHYQSqIkbhbU2h1jqr65k1lBDNGo\\\\\\\/Do0VyAqjWlYyuyw4GQ0usGU0k7nYpUL1BbkXZeD73Big0z\\\\\\\/K3Ce62JWYcbHbOYguHKJxKzKVODlzD+RvytCY+x3bf4NhnsUWqafHhF8USrbXE8v16OJefEeYFh35OXzw4IUkWZ1NT7Zii6asJhfKi0Je6pnVllyYtjGggKCSJoTg9hz78r4aKK26biYLDC8E5UcFG30jMWJNmilQeURVgdpUTjbtFEtHJl11aWEKuzzTDDTx5IkwvwSdcz2zF50VcCN5ZrmF0GNBc\\\\\\\/6dbEB6hEdDKChPO3FLsXramIOd5q6F8x0XDATeC1t3Vz3Z36Z8lew8VhxaryuQhEDG+4WYA++KjmX0Uxnyh7RHDtwd9bu517NrLR2NA7S4NsiPQFXvBOgk+yHYphwEI7br1q9jFm01RnemFjaAb8kg0Bs0ksRTtdOWIrVCjWBf1umb\\\\\\\/4+mDsHosRVNN\\\\\\\/MJtgWdPxpVVb5DlPeQTuixkOGkhbd7pcZxIJFzED\\\\\\\/EMBn0hSqJgEs7Y4uyHa\\\\\\\/3HbuAGeJmgzcghAt5bIOZGdvyoxMRstptPmRRh\\\\\\\/wrQbt8lSii0JgseXJ9rzd9OfbL1B2cc11RGBg0naGfIx1hsKon2G9mO3Fg7h95AoNvgSTC+mFNPy7ldgj6YVsdUj2gehb2ZsUgFcK2SfQ6Hm+MdIpIpy2HATBr1tuWYwr2t8qRpFwhc5UjZgVV1bBBxhL6MjetFr39nLnConC3YPGl70n0iy2buewN+RLPb7TjENZ++iE0Wp2BKvzCW6uA3R7wpQHnYzMaCHULDsvlqGUmWo6qwcPdCmKM2uwsWWFt9GTAVJW2Spn6GCUV+PJH4OE40BARNhachktaXq+CSU9dZdTpyrcNvX1wPbTPqRgyTP1GIINivoaXkaXc58Qpj\\\\\\\/zFcWq\\\\\\\/PfddWo+4cTJsK0ZDSYHVDP6MLBoEqYyO05m1FThulQunlu71OU\\\\\\\/p7Y9x6HyfysfvVFmeYw8jEvnQN0\\\\\\\/cjxhTazv4HRMbnaoglgl+tZbmnvxY9D5kSLiPLhHFEARTzWfUKUJS8ANaWaDtSvdEVNCqFJH6bZtA7dlLVPTINYRMrvwd4z+GowuEYIgrs4bl1LUnkY7imQy80B+DRc+REdg1ofDgtv8tnerh6mvuyCQt8rWDCUtBj9PrsP5AVOiZ3D\\\\\\\/DbKCuDgu2Qb5SKNsSdJAd20vy+61iG9k+MQLInhDoKPjCrkJUy\\\\\\\/y24ZkyRyEI3LoIit0zmnipUA\\\\\\\/Y1aUzdIWxcsku1uGvmLSdA\\\\\\\/Mj2qkFQrL8tFD13REtcweTMECSrguaIfwM1AcGBvwvKJpDatEOEbHbvJphRSZkYNVIVioqOKFxUbmxVtKha9dzRJvQMazngXTJ\\\\\\\/PPT4vjTSOitqp8rqM7nJmXP2uO8lTf4ZGiYwq751VNR5Ga2CJlHpwg0OK8crQ9anuG\\\\\\\/ag+FnKkTI4Pg\\\\\\\/GxulFOTmjZHG8MX+v\\\\\\\/dI5LGzzxyNlaPBkk5+f5YZfluF3dzLIMLZw6Jq2oeK7GAlhL0ZB4Z0K1z0TJLlLAz6oTvWczJcLJF56NsT2xVeB6sAZXutNCa7s1O7Oyu8MhZHWJZdIWY6LcazrdLLmoTx3zYVjus5MB7fkAfZQidJ76Gr9N7LSgq\\\\\\\/MqhBCGZex3JGgyvwm0fJWfZauPE5Jc5xUtBfJJOJaYzJ+YvcdMUBC0eGXLZI0+WXQbwg4+C1cE4PwyLd\\\\\\\/rwzvhLd8JfFZS\\\\\\\/FNG3CxnvXz23J3+hMVOM5RqV8UysNOPDlYOUUH9CseuThjRjA11rBXX5Y38A344jfLzidKVBdhORXDKjyRDD+Z5vTQKfIoAGCRngtcurKJYNyBJnQrVqBbYr5ZZHNVBnif8dav385YuqZ6WN435hRextX89yGy\\\\\\\/CRBtDGzKC7YORIo2RV7XuDM8noZoPb3g46RYhNXr9a\\\\\\\/NkDkC0Z+Y0Yn6221vPH864vv5OlttIlMPEYDg+o7TU6djuxCajzVNCSuxk\\\\\\\/MGt9qND8pN953pPqedpc8UFTKXW7gsWVOx1WGqFOAy3P0Z7yIXeCNfiGg4KrVrXovQkdDg19GK+VlhSOot+uL46RuRAg4XQaXXpxmkyy626xh7T9B5gxHSIhz2UI1\\\\\\\/taMAvKV0b\\\\\\\/8WHMpORZQ2oWSVmdoziu5TUz0YKcvZy6SGKgvErzviy\\\\\\\/UeCV0Y11yuH9ebTWQ5v+aLjO\\\\\\\/092hlXoot0K1ErUG1tgTJqUhks+K1E7v3rYWeYsJVtWE8qS27vs538vPnYeqCoHbpotJQ5GzavmIU9890bE2hnvqgQEcaK3NcV+zJrOLR9yw14glpxvzZpw1P4ZkbgogOsKSdOKQRfGLAjMGjmQb5BV7J1hXvoBSRNft1lE4Rwj3KKYpHBEWmL5vxoFemDDNMjdtQCKhgZRCCweg\\\\\\\/9R4LLZBVZ1PdbtWUo38rOkJePP4xBME0gof8RSEZexf94eainrNEiJhWbTysLjLkN96kxJXi0YJ+C5QRtnNbbUVO96rHTe\\\\\\\/b\\\\\\\/vMfFQ0G\\\\\\\/3voM4SuOSaW08YQuxxfOEOmgMDxCUanbA+l4cxU7Kfpg0+VvTMRticjVOf899WsYWx3ohWg9sUTFzTv9vcFAa2zXkniHZZ6YsgqJ9esWCuZ0yHCkx7Lxzknaz2f5IZ0IJACa8Qsgkmn13pvQ2c4bPKvK64tEHph2iR7NAQHDk4nZdFkalwtVwG0AnzVOY94KBQfzluyoxB\\\\\\\/1PM0hydWRuH8jNdx\\\\\\\/6Dj+9RomUNHiKGHywIFBjvC32jt2z05J4\\\\\\\/DCzs8BUITeUtlDgDj3JkpErLWw63jsc3fihmBb3vgDWCscMIfsXiisQvoQExz\\\\\\\/N0kk9dCKtc6QBD5rK6kcYZcN+6dpl+UKiqMrx8Pu\\\\\\\/Zhg0BvwvjPsLitQu4REwZNNUytAGX7ywbXbBaRCA\\\\\\\/6GKsr1ZJpF+jjjpditF+\\\\\\\/5fI6qjKhMQC6fJyzgHdqYnBhcpuoLD9FNfcvNgH0rr1Dua5iUaVIJ7bzpqrvpqJ7DwzLY5uzS5sb8anozIDQ2EAgZheK51hkMz0K6ifiCxUBllP6jyUJ6v6dW26erWXB7xuRcHE8vN7\\\\\\\/wKhCp5TrkmxSxcJ1jXqYBW2weYp8x362LJRKbJVzJVu5IeQV7muFylinOx7jDDHoSyswuFpiUoXqm6cJYI\\\\\\\/ViWDPy2Qv0Mhj\\\\\\\/BlhuOE6YGrhXM8SSDsax3L8adpA971EbIutXFugw5+dDn7VJRyGEiqsYPF3q+vWc7ue7yU7VjnvqVo2canJB0Ksnk+3m+mbzUP6CkgiLaKHLNuNX8CZ03teFQpD2Y2UmjAYYI\\\\\\\/0uWRwkXvdDJqpIGvHULTsnMld3nBG5o06shWqodm+Ox596MXeg9MPJUFwoC\\\\\\\/NCzJ9\\\\\\\/EHWhECf3k+cnxVPCXz2SmRy9lBpTUGwZsEbNKVYAQWoMCAldzyypJ4HZXV1gRZIyS51wRyWtdtLqlB2C9E5FZtY5tiQQNJQiJqKj\\\\\\\/KX\\\\\\\/xPUFH16DFNGH7UsdNbPrynvfcaFWbQk9SMxSkplmGb0Hsi8XnKr6\\\\\\\/SBAL9ZFj0X2k6I80aT7lFZ3SDenmc+G2p4RKMBR\\\\\\\/XiBqMASTozehQWUT8twrIp2hHU\\\\\\\/J0x9npRQS1en5AesM1pIha7HmB8YwAUNKNwtFaLQYfaGuaOWvs8LPl8yv8UVh\\\\\\\/ctIMfSjksbjA4TZJZ8HD8JwapkYuP73G23wJfAMteeka6Rwn\\\\\\\/N3eVQENwUCPYL0jS0Ldcxo7sFDa\\\\\\\/rAjOJ5UNx2O4WuADWv1krrWZ2XqKwfsivJ2UztasUWNCJH9J1vPykyrTZOY0U\\\\\\\/SdPBYiP6ijj8OxZ\\\\\\\/SE+WkOONgclCn0\\\\\\\/kWCqlllcEzGJYarbfGN0VMjdlE6j9a4oB5AjVYpPEWXm\\\\\\\/EWAJGcjM6lLP56hFe+wv77ChClq6yPt5RqujPDOdo+MOkMsE0sHUatHFv1mEDi64K8YFP6wWLveM8xE2UQVrtAtH2gasTr2m\\\\\\\/+lfMxPMbqcNZv1AR4S6T2jJMeQZtmrEfTSZKFRlUH0fBeKVyIBb0oZyabpvTM\\\\\\\/qQy72SkvpY\\\\\\\/mNsDyExhG1oeRrmS\\\\\\\/xxDYHMBrf3TCE8jpa2oCOPX+770mrpJwRWDK\\\\\\\/BUTgYUC5tLHYR6a76Repxf2VZ9C1lqX5LsTHqg5e37+jNookdL0mCgMuO3TCDhdk3f3uzbsdjKlOnmwciNWubckwxm01qrz+XLcPeThhw9c9yrHUvZ6QHZI+PBBgsSav14QlQibXa7GuESRjiPh7KJGCaGiGbRkayJkC8Ugsq2MuCEF4oqacSxNNuYuBfl+\\\\\\\/jlN1vKnqxGo7uW7qJG5y6c6OA0gcIJhUiKWQsNvVHb7HZzJYQgH7TKvv+QPhZOJt+bhhN1i6XBGbyiuhIv4kKdxGOGj8SUWaaHHDhu+qg9sAqzE8\\\\\\\/DkCA1PcATLzBEREtMJ8zULuZOwFs5Xv2Qd22k+60vpeDFl5gBg1If4ReCDlHeIRLqRb0olqD+B8ZUHJwFALE3KbbzEXynSzARl4GrRYTwgAQe8wDdHziGp4kpqnqWv8gjmL5jKpksZQSPdj+B4lBk1UBKYzdCDnV5pEyx27UqgC2mgU2pcCcxZxajTYcWB7w5K9U48Xz\\\\\\\/C+yzt7iBHEPjneGFfzHnZMx3Yl0CSYlZ+LesGlwPqLxg+e6132zO471FFPQEqHJsYCpkyRqLAUqS5CFOweNb1QxnDZnWJoPcUR7VGM5GH1bHKumRKyyJMqHO9VeIQS6L\\\\\\\/eEBS8+RaFNsqRiTElXrn3tAefRPHriHajyU3DNWYd0D7+MqG7QVr8CwOwoPzwJHXxUU8p+ZF+Hkfs7TWk34Ga2cgRSqGrw2GgQoWvmYyg6XBvyVNfwXYEqAmovOf5g3twn040A3Ovj17qPgASMSdiTTKPnKNmLS608SUbrfysxxllIFfglBOf6bKjR2xAE5KlQ4epK++jFfFfbCbF0plUoYRWFTsI83kWFTsKMJ1sJojE9CVsIfvCEt\\\\\\\/S7QSe9uauxpD4iZzcVpNCmMPQYLThaGdZOyHtnOXt1F1kr+d08Xsa92d00zG02pZY4Ux4D6BErPo689nVm296AKbfwA7R1M6G1u0zuPIykS9WleKNBP6ZxIpZSSTn16ETBAxHV0KM3XceTDY\\\\\\\/Fvpe7HHGlgPMzEaWFDWgAwh0avmDLtyumKXUINc12qw0lIgy1rMPYxgQ7bvmBRiyXEg99HGHKEay6SRjZwnTucKoQBctZVyiBrLLYcV+RlyilqqN8lWCO0QrWrkFiN0VNO29qRvUvPZgT+6J6hZsR8P67JCg8tRHe9OeFsK4\\\\\\\/aUB\\\\\\\/XtM7RHUJvlRYVWw5IR6Aj6VzMqe\\\\\\\/LrWs1SW1znmCLefZiAy0cE53KFAsW409l6PCyaU2gc8m9QRwKs19A77Z397cb9b+nXILos4Y1b9Sktk3PRM1J83Out53yiDxrk06lSX\\\\\\\/IRscgsTPybeuOCV5uP\\\\\\\/G7oUtzshQ2Efwq43LJyOu9DAbKDer6em2nQH0RZJLMM0\\\\\\\/\\\\\\\/Ij1QvqXxSQm9e4rfK8E53Aa8K8OOfc5RQtxEbd\\\\\\\/YyBM9EWTxIARpD+3jb4xcDZTqLKqnZL2ouARF8WqWzPsqFQZZAXIJ+ZcvomRlRnMy2NzLn+9u6RU+ZnxLnhApoziSZ9iSxSOqC\\\\\\\/RhWDC34tVHMkmXAQtAPMRvAKqrNexhmrOgx7Pf\\\\\\\/YtJhf3oqwN9aud289m\\\\\\\/CCzDSFt5vjjOG19oneFSpNf6YhszY7eFL38Z5hgoraxtC\\\\\\\/nyimJQrJiXOU8UE0ZJ2\\\\\\\/RsuqZRaoWuPf3BdyTb4swiQsjVac0\\\\\\\/sTSBqYGeCFMZR26mcDRGvbp569Lx6Xyfy49iw9pR3tJXYAejm6q6Q00odL9BmyL92rBHY6YHyMKbO3jXp6FgdlKMtnwaQeNKaC8P9NgwgsJwaLusf+M8s7Fm1Ti7RLSwUKaCwQ+8n0251l9FGjj2HR2g\\\\\\\/\\\\\\\/U1DxFDbMCaNmPFMEF7uT6lIFrZZvWlL0mCiRIcIiPkmgYf5BwDKCNg2+na3P7pYMZEXgFoWZ9zfAf0Cf422SUGFmyKlWUma24NdKvBwmMCSI6RqFxpVKgk3yfXy7MGNREBP03ZjfnliCWtlfcVLE3tPjYy3lyvzA8IiERNtYYsCvxJlOCGHfVsE4VUmddpCEeGwhr6B1ydd+CosR5skqi5ZVNq6I7e1U7EyniFDURJy\\\\\\\/hOauW\\\\\\\/pf80khZLVQshQ\\\\\\\/++NpMRGeY+wIay3ypCysmF7YHTj3ZVGDYRuyh+TZLb1Z7dg\\\\\\\/\\\\\\\/0bKvJGSQ7SCPOZfoCCfOMahCi634WXGDMU1Gwkgb5nEOIieKpRzpsmOLhaPefCWhDanJtbHGTPjakyt7xXx71Z9AOqs349qz5OXtJ3Q0saLMoK5ZiMgRBVuzzYCX2R7DF9KKeZ7aruTtCLqEcnfXqIVEkAElVhIXNqOSBaPOGVwatuwdrQAQyYNXOKhA0B2PkIqpZbVek+oNVYJ+01NxBOVY63pHSrOfdnxsvTdo3FsGSCka9XPlmCwfCnkBq65E2KlqmZYVA1j7AlBZMkw7ixDaS7rztnUnqgHq129Hw2i7t619kCahrI3gZSKWLspx62kHZlLCU2FhxSrIUvB5mSGaIlJJ3L5o4nbO9he\\\\\\\/AaakWZAHBXUmpwtMt7\\\\\\\/k+mLR934KOdrsSgz6IKfzOma5LpicgNtpgdB2eDMmhy7F9\\\\\\\/nirhjduCdtGD5ZoGrkywFISlpxyluhLOYKp4O5eEwMbCYXRKZxrs\\\\\\\/jP4SC4XRoorb+h8ZuMHDp+ndKm35sKNts+n8v+6tRISOt0UptmsPZoYybT1lcUbA7K+7XneW6p68nQcy3wrT\\\\\\\/qYiFlAaVWp1MtDa2Igy2LFYN5PI74932LEQ44vXiAGzPzLrSFubPyItfsY3HNICj9tK8M9eZ6t3KJVW3ry7V5jTZr8pBErp0uCDKOKBNYFFLy4SbXcEWdbg\\\\\\\/wFgSPFx2b5YB\\\\\\\/P9iVprrStaSdiOs5jj9EvtDm5NG1nC7vegL3TDNyyaM8TG245rkqKXCpCfJfpnAxe5wb5ADRES9O5dw7CiudZJEeYFmeycSMawxapgcEVy4j5gfqVtqNeySUG3eA87lF8AhLQuWwksA9qdxYxT0qNQ9eZ5BsE3P4WJKQ5JHHkO4cwwQIzlCjHT1u4ppcRVR8m5639ZkPS2JuRNQYYf5af3lNRB5h0yyjTMb62APHlpYLr+nACIKAilP88sOaN4F6p7X970CoqlxJlvQO8iFdATNBexSjt1ICqpAtkw\\\\\\\/5bw6lhi9XSTSBfT9MVMDeAJ\\\\\\\/un5JrrFU\\\\\\\/8iFK50CLajmzUWSyomypLt8nIUO9zPw+bZcQcBZ0X++SFFSQVGvxQqcgaf5uQmQ5sKcuj1JHNbUSdGSHONXTekYSikrF5scfOAh1+9\\\\\\\/quBaFCQvmRWSXTEOvNGFB6BoLoA5cS4VYVrLRDyzcK88tfGQ2KV8kHOJC8C5MrTcvRkh9CI69ppNnXElUbOcDoZjSYjtPoSVbMI9IifiX+zOiscUommKwwOhMfeLhehRCkkEx5dZAD5wfuE2U7OEXL8tvus9tgzzkwgbLAa57MlGA4UXX6MoJhRVYi8Z2fcva5\\\\\\\/ei30BwgIRbUXlBDLxUt+XsryWnSQSVw9MJ0eewgX48bTiIPh43plUM69ia1qG49shzCyUg4nYRvMzskjvpt83y\\\\\\\/b+htGZJXo1gszyxylWczE7llV6LPrbK3ZMNq4dxDDiIJXhPufHipd8rZAcVp87O8Ur9W0KjyGotuq\\\\\\\/yt4tBVtILhiP5xxjYrUWWNNQzc8E6CZgHKbCfBSoeFnVB0s6AWypnjUbmYdsfXbGkbQMlQQ6UHvSpuKm74VcIvsJFK02uCFP+CNCj\\\\\\\/2KTlQiywRUHtT0zzS\\\\\\\/AL0pvzHxWoC8bi7gI0Fp7vHrhgFSnYAU\\\\\\\/zpT2\\\\\\\/GrE0N4Acp6mY06ntbh64ia2CiMC0FZEtY66dlBu0Yh2SRegX+RqC2wkh08I0SrAr62KJ0zQ+G50QEsQ6W45cSN0ELk+OMWuYXi1rw9jeXEIT1lqnEt2we2j5kWqyLKEHJPhYTYUwVDdFvr9CLFog+VNLkUC1+wTgNq125CVNILAuT27D3bzuWD5amssdkraaXAp6usXEWLSkIjk5S1thS4sEfvWtUSFnI2rdWEzGhg3xsKJ9Yty7StaN\\\\\\\/EiKDsDeRBbyTRAfw78Xj\\\\\\\/v2OXsXPQwf3D5otXCVNvO2bUNf7FsywKSB+A6p85PnBZZH2y8piSj9w7AHY5Hq8VQfZBeA+NeZfa8+wExXGXYU6YnoFt9Equwz\\\\\\\/Q+Dqmza+f06qPR5Z7mwra4rXnBDHcNAC6bfPGVU\\\\\\\/PbsyPypIeWNim1j4FGfw+FvOf\\\\\\\/0Itz\\\\\\\/VQWXMFGkgWWblIK38klgfK1w6b9N17nKHI\\\\\\\/pxxIvK\\\\\\\/u9hvb9AQqg34cbjb5WQ8xG6S8PwI2tkjjV\\\\\\\/6Y9EeDhU+wO3Z6XkB7LX0ZZBkPBuq6+L37J8VUkhsihp1N850ml1\\\\\\\/xOONI12oem\\\\\\\/CLk6tBJq8vGoMvQkBJO7ngQSPVYnUMbE+Ec1UkBgmL+n4XOaFM6AWQjVKxGI02sR6Pzt4RARz5xO257haIrdtpHCWE7Tu6rRVHmaMlbni8kVScncr6gVkjnZuesmIu8VQe7L21frQcV5cSq5rSpQGT\\\\\\\/+imWFPwdlPNCFSoVcRBIjfXjxJ7UHzTb2iOaAbdDAPs6zrV68hIkmQEKvetrIOLubDOsopsGD6PTfX3wzFeC6Oi7HCoHklSZbQWMvZXZGyLCUFIsvNbSPAQLzQDJG9Kodu1vkYtYxv3Fi8IQfdYj+Hi5hGrXGvp1FkJQ+JDLwxqrMCm5bWxe1\\\\\\\/Hb7tAl3A9gkCJaLRMuigfKLlyHGxJACtcSgBv9UjB7aEJxyzL5EqDTE4us7grquQbujLAlxaIfD9PpeOLdbCcMsd18I9VbL9BKcaxzGuacFZ2azwVL47EKJbVbL5cYFIQZu51iumPWCx9+eZBpz5EfMMUWqty80PNbDSYS8bxtXTuU3ba9MdztsHwtN1OoeSZrWosGsShUqNuNaMnpa\\\\\\\/ANxE6wSWwdiZ1HIjBIF4SaIUpYCAXRipKbF4etMyA2WqnnAEz67ngG+hdpgT442rzEEE89eoXOY9UXSAxm0og0vGBzdclLi9bAclcv9dGHHqRpY7E0BXeocb0Qefw1z4dWLZSKbQk1M6HNCi70oDAf7YerwICmIf\\\\\\\/jBciuWZ7Opy5iCBbZax21lOSkdqo9QYKSN0go4MiDcCnqREkQH6u0X33+gMOXbq+Exkey83yDidDcilLkOY87Nlz2Soh\\\\\\\/\\\\\\\/rlZPfUerWb\\\\\\\/A8OZIxFuOCIa4XrYOSNOJzzPQg6ObRNYDAMVk+QyHuUYJKOdoJFkUtLMi9vem9b\\\\\\\/vqEm\\\\\\\/jmvsKj17azylRudb71qfxTF88psMVCw2AiV5lMod+ZOQ9LWswuYamPqodwQT3kbKr\\\\\\\/unBzeFBw1ccPlEAfcfCknOshDhFj\\\\\\\/E64J4W44USjSfE9euv7WjzlNd8V5X9w3oSfIgy32xHVpWt9bPIm\\\\\\\/Gef1dw1iaoZZkee4yegxGlS\\\\\\\/x6dF6okUNRdn0NzIHAK0NaGmuhN3esepmsNP1Mas2mIITaszVi9iRPAEvrg7QsvLa1ZrU\\\\\\\/QBGZxAivQ3JiZmByD0G8ndJVJrghlBjluSxkMrf2e35cAMMkq6EnnKuo+AV+Hx9KvaghL3jXcqATGOTvLbQdzyTRUrCytSAe+WiLysN8SgKMZqY0XrxT\\\\\\\/zReA9gSM1bVFL1lWbxCCe59ZWNGAql5SqGUggglhVlBau+mnrQ5ZOZ5oqfxxFfvM1PNglYU4oskMqMEctQYGGcVsnyL66Z7QGyrUPKdh3tm2Cmt4yPPfkkStLs5EMcL\\\\\\\/ShtcCLSd5lnGRhflzCjQWttXijPtHdSWILZN\\\\\\\/ZTVmwXEj57oaKfXa5tnr0TcaAQMpIYmCaxZscHOhwomzldrrZlLIFYd9X9UvQp2WUjivlln7Qr4TGub5HkqRl7X5DRshZbaYsqq\\\\\\\/NSMbn1APyqCWipNaUqoxueULiwZ68W1QEa01k1\\\\\\\/2adUP0U6kZPx15Zy7ZAGEc4dICgc1XM7bzsQUNuxnO5lM8ziGTVXUjM5DAmQDRaZuhnk958+AKQZUqu3EXc5Oj4crNmnyTGfp+SumC6ZHcRElO2E3fHmkMxwbUFwjdjPaFokV54QNMOe66Gk+IWSxs+L344wA6XP+YLWzkyMT5xQlLEw2izmntWCZG5f1\\\\\\\/iVgGPj3JXdps2L7HCaUZPsGK42q4w0YAEIPn6K4Lcf7Armxpc78dc8uhq5m0g8ZiJBPUS5I+UrPrGysGy+ozNfyeSXiEb\\\\\\\/YTTyaQCGlWDTcQ2MPsZv+Y6RuA3+sykgoloTt9aT\\\\\\\/SjHiZAcQEMifZH9jmdWp2x3+hDTNcn5Vi6VxwY+Oa\\\\\\\/kN\\\\\\\/yDbYr15O8mrHiuJdXz8JfmmLKLcedhtHY91W5Pc60+smJbRcnzyMJnJ6CTiRQiNQjn9peK0dWvq0Sbo6fk4SEf7PNs79wBW51PDR\\\\\\\/LPVaX\\\\\\\/GhSpb7pmreA1ZCzP4oI\\\\\\\/WomxGC+E5vKKfND5hRc87TSUpW4tfc9Ff2Jz29Qhb5EnPxaZcq4IRlwZYoyasqCRO+sVgyyYwA3Anzca\\\\\\\/VX3oe9jopwZ+QJIcQlUGJWdaMh5GKNg0g3UnT1cZFKdZWNHnxzuz6xks7eONHoIvKSQw2fL7vWyuSm9RY5qS1gcwHaqM3NG79oMqZkxoGOTPkwtxI0sppLrO+ul7mXZiqLG9JoI8RoPAF+qewOhmMbEEfUjkx2tecjpJrc9LlE6DjYhYuNyZCW190QQBC9d0dCUY+bOER\\\\\\\/efgnAmGiMEH1uw0YmVdjA4k7x40gIfCQyTduj\\\\\\\/ICqE7JXFHJPJYACAhxhOwUizMvcytEGRE5t+8\\\\\\\/UediBpJvNUmgel4RxVLYh6nc0QhmWeAje9tid4vuD+QQkeq+3y3U46KJxBhoAIrdXzDCyUKRGaSilbKV+0hVAQuQ2kDqwzCA54g\\\\\\\/AMjYkQwi5JlG8Ek2B3sK9oUPOIBGdR6ou1jlTEonoZw6gjcZJdqLhTSZps1JF7CdkO1fFD2q88k9WmbTooTXay1d0mCt9Exvrye+9A4jfvlc2pXC+Cp8ubTg9Z\\\\\\\/yyqdNc00gupW2osJNswN9MyugSLZpjOi5U18pDYS3+CwmJ+86BjBqFiWpCvuREAyuFggDJSlFaTEcSZVbO8hpA6sKr8CH77ksGjvVr5EIG+BblPSigT3tS5ND5RDm4wmDzT2Nr1yHuNp4i2g4GiusM9+BZ9TY8yQRwNdPR8L3YvlGhop+ziiC7CsB2gUZUs9qF2tVDkKFqO0Ib+3lj7eBPN2Ib8QHw1ejVqmUIjfFbs0MvECahMh2zXYBRmG3slSgGr8e9rveNXUDxod5K2rV6YaPKaGYO7nOC5UWOikTKfwnMs3NjllQ\\\\\\\/ePClCobuwpZvhn1MsuHX7lL65\\\\\\\/bEZivuBd3vS5010g0LA2NirpZx8AyQyy6X4fZkuKxQDlcTe2QVKwPS0acqK0U2nM2jCiFKkPhUXMWCsRKyVDxavVZVSqxk5c1rXLco4zv+wN+Su97bdncR5btpSS8Ei6oURyfChIaT7l9713oxmRPi1iebyoPqx\\\\\\\/9jao7DcyvmFPGuzup+6Ou27G8KnXAbyfVoJDdOsVJ5kUdo0Gt4CtAn8V9U5kC6ZkI5zVtGi4wIaWOybCkFrkgv1naY4gchjV3wGOWuuNoDYKHbqzHwMbytfLqfozuZeWaaX+SDJS8e+96jeaBf9aRUZ8BW09Ap0lQQ7MGLnIIRHdBHnzKl+CJE\\\\\\\/TbkR6DQki3IDG3GLPlTrgU5eWX+XqV\\\\\\\/jLd43Cl+RyXrsZrijjoJI8Bf0mhtjGQIqsJ4U6c3d\\\\\\\/pD+sLmUD6xOz7oAR\\\\\\\/JPeNW86o80DFtSjzWpVH77kE3QJtlorPDl62NJJtPoNd2o6p689+vtt4U56PTi7kKbPxc9V5Uy83ZuI7uHGDWHYXJaGn9pwzRW21+kCiK8bIQCcvFAC9icVnt9UTOhk3nYFJmOmNbxESlEgOchz0Jbzpo3m0fIrnQCnXT5D88ZADQ2nO2YNH0s0j+Ig0NI+0Va2Q3P2di6n5gGlbcFySfOvNozNL1D1QBj97svaJAxoZPqMuUKO8cSujzCLKltCRRKh\\\\\\\/PXTC07PEuH9DqETWWsN1GCAtuDsZgf4fQS88RkXpfOtcb7L+cyS7ipRQPSlvZzzV5QKpapqF1yL0s6xVKraTcaEXR6OROugKyN2kP+3vNFD4Nkza4nVpQw4rmUf3FkvQUnlcBVP1vol1+3xeoglY\\\\\\\/5TTT19iEEIQFXv5uAy7p8qF7E7fdmGryMRIy1PaDVhRGAMBs2Mx24Eo8wbXsFFQM25UEJq0jrHQKQLEknkahTKNxH6uaSxz6p0iwYafYbjcjMsKXxk7HQmeF6RetTJiWvZjzlizqkQe2unJu4ZE06W4ivmx65cDgi4InhKCtYO3Jyh88S\\\\\\\/r1eJjNYEO3jnQnURd9y4kXlEiay0W3dMjkvovFDIbo9vUne1bRZW+WLzXoIRCLD3Z0snGxYsT1+22JaHeoPN9cjfxldLSm+a+VHoYedOZC07KrsJCKUv\\\\\\\/BCqXTib16OhvfWFgHuVXDVEpgnpktsxb6Fzr\\\\\\\/kmGv0WQQFwraMCpU8m3oP2rZTYmdZGDyyQztOx+PpCgtoHeJX5Ke43BtG074bTLTI5Ymcldm0CbqQGStEgHtH2kHKxMHGN1HZXgyBVdAGKwjkfTzxbAi+4smpwMbfWDCia7zoBJ76yr4Cfsr3DXWKR0PblM5OfHFgtaFYRKciiX3Fn8Kgc6ocsfcMkprwZQoHoxfFZyPPoZYjxmLmzRBNonBOLraXFpYJsHhvYN+b2I9xrUnN8g37tdFJ5GHbLcfwkfihvoWVouFxT2Ht\\\\\\\/Zc6vzja21D4dtiG1to9lkInYSpp4J0NWo+hV8h1Qhi9KXYSu6wncOVpSBTBhU2LkopBYZJUSto3CHOsf5dWMfYuK+L1Prata4tdDGeTMs3OgJYQepC4BcN1KpUh\\\\\\\/lt+ewoXdEqUA418XS1QG6Uonc9fjv2Eu4NpZhVkU7gHutSQ1Ra70i3zSM0PoLwK63KNR8qJrexM2w\\\\\\\/1CB7ZCf2RO03Nv7k6961bJVSlzH0FdAnDcFl1o+xBoS9INnCiiRVGzP3rRSrXrp+gZOonUtAkKgxDyxSknaDP5ubp34c1wD6QOFGI+91tPZtuxOC8PiURSxWBPmuoT+BGVhlKo2+c+kUqptS7YuEiBbbgMrxa+JIHQRq1V1pogyByZ0NypybGLLPk3x26pjOMPsmH66oupZNXgPVlCHJXeUOcCbQ4l7QJlQTx8NKM1QWaYlVFLOKgtqctTCXdy7BXK2aKyhj4JatgdAJCzR\\\\\\\/h82xuEVPY9owvzgt80SFUp+52mA8aZqZRuQfTgLMP\\\\\\\/CFwWnRSInotC3ssy39ncp2BAUddkVIug7bck952zRo7ljKrU2uyjsTWPISDlNRdXpAWWiQRDSaVfVk8Fo1Ov0Zaj9Dtc+vqZ\\\\\\\/JodQYMzTEEBE5c8XO0oWtimcibGbV\\\\\\\/KTPai9ModD0yUrt1mYExEBwymP8doy3+oebpGX6D\\\\\\\/w8NNvjeIFYbFn3QCeG7MrAz+il\\\\\\\/hPw6wvSqiCKQ80G6nAwxWTQIkKT7IlOz554qn4IPsnluuYUEyCeyh4VqY9i6hK7TQlgiKFViXqWjxpmGkNTHh1OJAA+3c3m3N7d8IVsE4IL2OLoFYfSGu\\\\\\\/U+ZjxFNO6UiUE3dVWUHhuzrDfrZaGDWtaut+M8kK\\\\\\\/1lSCA4S96gXrxT99oPKSodVpWMMu6PxGtMnERGWhpPCKjcGfGUyLFEhf4V5LauuSoE6\\\\\\\/BbagFrKXXTiNczGqWjqJ1BKIfTyPpJBbAh3zRieAlYLO87xIsnVKNcT6mCM2068epuGZSZEBnAP2QlPgZvXG\\\\\\\/GC1TLp+t5yaKUWfKsXGZCJFpZfzO3Op8RQoGIpKiPAhqOyts63QqmcQWTFe\\\\\\\/eb6D8qLj6Nf3385wFWwYXITjv4BiymFi0zb6q0TzROzpRjeH\\\\\\\/MsMV5p\\\\\\\/K0hs0yVVH9Kh\\\\\\\/mohVwlnhrxaGCmLby+LznFPm1NLAD+KvTPyPXH0lg4fkubGztJd0IAHc9OgF7UunGhAJT1qsOapku3Ckdj8Y+sab+KJHxkpOub2VxvW5PlHU5tGH9UetTH1oUHIzxgR3ZU4wLaQvJk6rryye+fmKfimzmd2LzcQEfkzDLuvohvzSJSNrF\\\\\\\/9Q\\\\\\\/fElgncvp8I9S9JFFUaIVEu58OfVLbBPwG+RsQjAjRukkIenSoUelByVesjFwfLHSykly4uBwWA7oDsj\\\\\\\/80tJUKEb3KNZpTlxyORXCWwONrw\\\\\\\/KwijVPc281E77IdWEnNScqPIWqeDQKA+ZBivBlRNTz89Ibw06CP3V0neKLwerx+XFhIB+K3I2SMQhxP\\\\\\\/BhixhXy2ldicAgf\\\\\\\/0yAweL5KCI\\\\\\\/bXeziMXX6aRfTUUaciPYkvGHkqbFytWaJhLQNX0k+jXHshM4OJPODFQ3H5pn5Fh34WZqMG8AHEuqXG3hYI7dSPf7Xm6zgOPm2sJBPoPXxi1hJJ3gf1+dGH+YyWyD4m02Si0vORdSEjaA0wp0ndsck+TYNMt+f\\\\\\\/RCoWxLKKC31B3C6GNwI7ZJrBIS5290aT5vpcMuk7f5ley4xP5+MXO4aZ\\\\\\\/TOFAFGj39MXqejL4AbovwE1dB7iZeDdtq8WyvtD1zCNW6DoQmh5ORzlQ97QVV6vOlEANYIbm4TqtfxoDOTAVAUhRg5\\\\\\\/LZZ+UfM9NhDnUc3r8d\\\\\\\/QUCtNr0QIpfowzYX8qOHMv2dkLRuZzA+JuYrWMf808Jaam6hSR4pTU0bGJ0SmyQrWSd8Sy8Kgsi9F2cw12pWwv3kRAvGy64dn4NJkOU9lKx1W3K7Up6JiqIjMldDq9qvQFTn4lIGsbjYTfpKewQVji7FxEbCulAV7NwRLExgUnLQ9in6h4lG3bRnox5dN3hB2iKmffN3KvdSp5TzhuDV2p1FBsfeF34D1JAE37dYxGJUWYCRngOJ6xQWk0GsWScuq3N8WFiWQC0vdQShBMUJqLYthVynUdX8mHq5dTcQtDf1fN4JIRE+7h\\\\\\\/5C8SIJzPDFIYyjGxG424Vyqs\\\\\\\/jJ4YM1k+nGMgR8OxONtJcZNBPQrOdWJPkCeC30djpUwM6\\\\\\\/AoVE4pNh\\\\\\\/qY2\\\\\\\/1A+Yuv6y4jyqwlgy\\\\\\\/QiDfLz3NUfRqkcMNeOW7yFIC89cbMdHnl2487dPZNMUxS4PIHyvvikIaGo2ZzXhyyxhYtgyb2BQbLD5vjJTfPIFGBOw3hENW3nrJMp1MygvxMX8JL09JpOC71VMg4o1BnZg5VWCqRq7gs5JLxmRR+yDtNUJh8B0\\\\\\\/KE0d1SYpzvmctYILK2\\\\\\\/lN\\\\\\\/TCdi9DY6UViC\\\\\\\/4tQGNkmgYl3FsjzIo3ZA4Qpr2gFVvqvqcd3hLCib5L5HNVHLgcma+xbx9cg+FDuVixt4fx2aSVQdHta7bYw5ZJjlhUIU3rjgnKKp1GqqFBAQACTGPng45xBakRBNG5Wr\\\\\\\/YW8FcD1Y92vcuWLhBJUp\\\\\\\/xpj\\\\\\\/rJbvBPn9pkwG6kglhFY457HychEC\\\\\\\/a1l2g3IfmBS9elveRrrpAkbK\\\\\\\/5AiwkZDQMW38llUndq\\\\\\\/Tv8diavRAqLhZFY2RcYPT6TH+tOxO1jaHAapMIvktjJMnqcMCSasSZLGEqrL2AFbFm7zT4ZBYGEepw72rKAbpyerby4fvLIbRfGVkYOO6v5YfbsCAALv6YJDlqtxBIAoAXevsmuzo5xY5Wbge6RhcFLXDQR6E0MFjHgPd\\\\\\\/IQ1M9C8qjQi6qE6Y1Ye630hR2orfBVd1HSW5vFQjLgxYuXL0w7lMS5JNBma5OzgFNDqxE1PkQsIxqWXbP0yWDfXu5Z5B9+bLwequnkPqP\\\\\\\/tV2\\\\\\\/s8VoN9c46GO1x9dHKtzgEfdCR7gTQ8xMSvvvE9nD\\\\\\\/yuhIamT8769v7xLzagwgXylSJO4ZHLtleENONh+wBC+xhjb6M+6Z6AvLvnVjayhfs4i1zW0mczRlIIgan8OWqNixlHt8PLy3lX2ENi80Amv7eEh+Gc9tQvV\\\\\\\/Ktvg6nS\\\\\\\/PWQrmSAi0I4yIbBXZe4BAz\\\\\\\/W\\\\\\\/RYGcfmiCGZUJXKOCyq5blMaA7cq4MElYLpBr1im9w9w\\\\\\\/dHpy4KEIMkDXH5yG2rGuIZ\\\\\\\/vZD2PZwfBkfUa1sVMdqj\\\\\\\/YC1VXtOnPFLifNPPHf4cf0U4HjmrzLEqI09ZyCsb5t3iw17WP\\\\\\\/RthS\\\\\\\/hPgeye5CHoJMvrE47Qtn6D1bztZ9bToq8rAcdXEfX7M5Rp4G8O\\\\\\\/y9OK9QjzIW5tVcIkgoiS7LJuesefv6ZkykxziqCKSeiyplSgpFwFAxqDCNJDYjWv6s2bhn5NrEqO1RzyCp7VgN05+v0ugAZVF68isMtIiLsKEr1hXsmmzWAKWEFzqwXQcmeLf0mFdCJnKvCQoBinAicg0S04H9Iashw2iwA0MF3aDlzIVUyvCPjY2M8RJbadIA9MDM39VIyxLW2bVhSLaPKiQ7zzhozdw8c0nhdux77OPKRr4K7toozTr2RMicTCCImIJC8SjhAwmkAOkFhlNRI4yW0jUcZVNvS7BwHVkFmca9cU9zRVkmtdCjU8HVSvxyrzafI2WbIIVCsc9+XXMFsamzg1loOYpcWP7Xqrd6EY7BcBenFOjehM\\\\\\\/lwyubp4Kz8TrVmzoGaRWURIDjezKo3RjrUJtRft3CIQTFGfJ+asUhhyP79GnWHS9CyKaM6vYmZPIb1CXrle88aO2PtRs+rTWzgtLbxT8KZD9x6qGbDQHiVSO15AZFkyM6+BsId4uTE6KbmEfxYRcnpV+LOiG5tzgWxL1EZ824tv5nbI0EGVRWI0OU1Dl8nKbFf802SaGr82RNhWQZD63\\\\\\\/S3AzwedB2Q+M30bSAHNwm5WTBwKzl5KpETJkeLR7rPOqQ0Tk2VsMChyzfjn2p9+WqPhCR6C5IUjNOpRvL2ajtNOZFr7zAi+gOBwkObapJC6\\\\\\\/0SLiGYuEzcMmGIyks3U+ev8XXAvn3U2fjYjiLKfyRavEMitkg2YBKzjRTryvgPbVzMMyse3XH+eQrRyAZu0Oskw6oJ0cGMlkzhFPHB\\\\\\\/AUx92EOu\\\\\\\/T15M77jJy39dzPSWU2WQ611SsiXi1P75wvxYLXv2AYiAPOd4G\\\\\\\/0Upwn13odlHPyjMNRjnw+L8bbNKtBqdDT7mXAs5bx9KKAVO0T0\\\\\\\/zf33GV8lJwpbiblnoMseYlLlGoXQl+ExjGmtvPjBvk7HSR15u4PUvDPA747oA9Wy6ZHf5BXO0yAuHAR29lA9dREjotdJEt8+8VigtndRW8e8y+tz8I\\\\\\\/rYv\\\\\\\/CeVyXla7UwPykpwlJ9msT4tqXpsPXKzqpHuJ0S35efX4LfT3UWR7llFNr+H8zZ+jElaclUr\\\\\\\/cKa5HOgSTMvRi5TUr4WB70V1uvckMBwmO225GAfntGqsX39BxvzBrefNf80JGRdO+B5VlU8goLeynRVxefaHySA\\\\\\\/7qftOYId4ydSptIXq089SEUpn2CLLjgyIxjsuxN5Dge6fG\\\\\\\/WN8KlJJ4akCtm8V6Bd2fK9wB1phssOv+5DJ3j5Hz8m8fYDJ8mZuFmWS5ABMVX3hXI49EzCAGfSQzx\\\\\\\/kqLAJQ2KkrIYpP3krBvu0ri4AQcAby7MK\\\\\\\/mnFIErSq5HeAkwg\\\\\\\/3zGrosvFsy18DDg6ChKRH3qACBzskE2OiIyOQnCQDRcpWOo\\\\\\\/TE0pY\\\\\\\/Nxad+F2DcxI87X+n8Vj9bO8PxVWRvAa8GxINNZdqAZOihb7EZ8YbXkhMfi1a4twoGLCdjMVu2h47DQqf3j7kS8scnWXo1ZX2JGj8YKxgkLYHBMdx5VMj5ghSRSkWCEGUKkgR10i6qAPwvGWsqkHL8W+PgxNrXwfe+fl9P7CwOuIS2LTLYq4IugGBT826n+8GGdO3KGsFVGlmeEUZ1yNM0DC6ifd6TYR1JNnYEXFpp1kQ8qOF7V5w4uYDkciWUBbge9r\\\\\\\/I0n4R70PjJASReSqE8xmW+b4QDX78WHs5d7OigxjqHgbTB8mHh5Hv05MFxCKtrkzxtmGaSKxZN3b3xn8aF1Ep03qT715TUVRo+Cu0OpDCqolB0Z1m1CacJyWITjEVOzJLdampyB1vmW1FNDZpqW92VRbgsZEqnDLKxMvhYlWDfiraz\\\\\\\/rz3CL4gS+AkX8tuBT0xeGYQ+VXKS38m6cA3a\\\\\\\/C+PqNndiN2yvKIQ0pY5aMjwprlgMdbRB7rkY10Osl9HWRzGxRSVbNodftjM\\\\\\\/0dJ\\\\\\\/WsyoWJ9+m35Dv7I\\\\\\\/jtI+gQ9RWYTmKqZTrrZ64mcf\\\\\\\/Qxh\\\\\\\/Yba2sDlZbFbCxU8p1e0I7FBgEhodvcOD9v8Tcw93sUOoRw5vqz2\\\\\\\/JYNRkjDoALk9R1dfOBgsMUuYj36LuHH\\\\\\\/qK35FhQ9KE171Ela0oZv7SSNrlAi5TdwhettnXZ++6InQJoNZsFy9Ca6I1rYI3P8Ifh8SvujmqSHoHgdZbwH9hXhlDHKrgyMedXW1\\\\\\\/6vgrs1T+EZnDnKSRw9GCqZxFXn\\\\\\\/u0wybP4gLZMXDPh4tx2XLQ\\\\\\\/odky4DI27HA1x5oPaDJbofoB7TC6SxX3VkQ\\\\\\\/tWTveXIwXGzALbDmWEiP0KRBIk7VtfXg==\\\u0022,\\\u0022iv\\\u0022:\\\u0022uLwtKI2hs\\\\\\\/vDl1rV\\\\\\\/LyODA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002230e45835e3b2e0a160ec9430179a5336a937409b5551064fef131ce466e98868\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022ovjVaheYRbzAGkSONem94WwR+wtv2fcRa+sod\\\\\\\/J+KJoJ4\\\\\\\/9D69kjm9eCXu9p28SunWUWj4YSsEEN88ihRAidreRMZ9XsG+GzsGjyFgUDfC4HIYyMngHzWsc+dxSXXoceY6M6VMZzVRtM+56PjAYY1XH8qlP6n3Xa5kTu\\\\\\\/5XjUt72xvrT6NC\\\\\\\/cbiT32sAM6PfQLD5Dqg\\\\\\\/JsHzatLQxU9ACDv+vuhN04JKmCbS7Z2GZUQiGCRD0BKVeKv8cyUrc\\\\\\\/neVDrlhD+Yx0E1D+BJr1MpT0bv3cf5GT4t351Czyq9bErWZ6yDG4bNqpN+Gfyrzak4vGSWm8tAP3hubmFN\\\\\\\/uiTwfPeuHCjPExHespcy0Epx7+7N+NPcys3X5xm2d8po0oVs+qMlwY9yh\\\\\\\/BlZAFJQNGdpJzzpxWls5iyXJ1hrdMX6uXsq4bnPB8D6x7SlvA13mK8549Y2Qe5LSQeybaaksGnDntYltPHPQiCBA+JPweMRw2tb\\\\\\\/bQSM3uU0YNFxSmANNIeYvFPUIIbOqOEgZb6bdCRUnqnM9O40fp6eBXWoEJNfqWxRSflySKY+kLerErShk1ylJ7shYX8VjgUEUgDb3GsResMd2302ifKY+XZO3MboKhFTMbU0Lo+214xADPXv8s8d9T1nswgNarG0psybgyGBb5iuMgxvDUlFTb7tSKmvFIbFyanFRYDjSCk\\\\\\\/dYUc8zSLA9Sz1orntqWUqqQGDFRm1Vwpeh\\\\\\\/ZLrHdLhITZ9hmzPcpjjhe1KPIf\\\\\\\/fNkX6gdwdRMyXt4rY9W\\\\\\\/fVAkaIngNi8iFVktH4ApS2INpKdb93fJr0oPv00Hxx10IbILhgS\\\\\\\/5Cr6XOnoJF40ujsVJVX0OplptKTU6yGfSPgRQnKRjlMDuBh1hNPUrNab3kp4Oi68l+4rtEN91cjtR\\\\\\\/pZ1\\\\\\\/Au76EEQZy4VG2eADfpw3ZLrbsUYkbAVf2ZDSIlUbZ3LobLad\\\\\\\/rg\\\\\\\/lzwYbhO7IWY74R929wN8eFGaN8hEX4KefTUsVvhUJ8kvCWSLnJC74Y0HclnX1RivDPW7KbZM3QLwmHzY\\\\\\\/tocPth7PhoCETpMDwcbHOusFuUlBct3yRZEVpbTj6x9YCMVfgtNbLjUcDnt1jLY3xmU71HNteLC1H8Qpi6s7KxLF6GV+kGtB5OZzavTAMg0bUN2\\\\\\\/1HE\\\\\\\/26vVJKYCPIZO\\\\\\\/zKTzgZhNAztrVUz\\\\\\\/7p2Jh2xxTrqNi8VRZNk2toNrmULD7EQX3OESjS+2qjid8NqJzP+8gumdkFL+sCDv0E5h3t+cCddHzASXnw3kBOAEuTjXSFvr3fy2cBtPHg+I7GW2zUKRk72BxPiOh8b54ZZwTQb+I7u\\\\\\\/o2xRcj1xd9Kp9+TYtpnpTf6zIgUREQ\\\\\\\/Ko2+hioiz8Qxyh\\\\\\\/0croCXeBTnMMTYGJyhgqN1+nhgenJN3atCM6G8wVemV0pUxk8vB49jsFtZOPnrvK1xjWt68wBq7V6ru\\\\\\\/pevC06bP4V5Z\\\\\\\/AQ\\\\\\\/7l8bNtSYqMfo5XwpOHqcdYsRrXWw6z\\\\\\\/VWxmeyHbtsAcCPxAWLtFeEOMA+2NnzoKveFeSC\\\\\\\/fwewnXIXcId+rnet0bt9hVplNpKs2t3tDe6429fv+Z7V3zb8ssvlQn7XkNHX5H\\\\\\\/Fb1sUhq3PmThomhXgg\\\\\\\/LfQ7AvQdoDHuW0akd4+XPV6zUOGUCvAS5aMYklTZhWOMvg5JP9VvYLQRc+f\\\\\\\/X6XoE6D11pSAh\\\\\\\/zFKDJywn\\\\\\\/WeNTQT9xQm8IZ4i67PJK\\\\\\\/hjMkx4EWXJq5OCTlsIH3zDSsUgFh26cCyATtzU845qTchTReTIuOeAgJ7MCaw0+hhZsbNUJw+NeMe+2dvtnYXMFbLR86CfXdzcNJz3wbdnVo18bxVw4gXHu4DAfy1KMAfV5WGvQfVRRcOqxy3KAM7RhOeUCdqKSGJtb7pdC02p+vUoTTMTBISf5kmgHoP0f6BQjvuYjvDLVMg9kXgXr21j+lLPuYpHZKJ3s657o9JReRlZHNYxHFNM9av+l51fZYN8zsk+qNKNnlsb1UwyQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022KBt3AtCfL9na4ISLyuLQ9A==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00220dd83f2333fbd58b90c9aeffcc5765dc16bddf768a763670d9638428ce2820ef\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459336 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-genomic-selection">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/2/10/1179" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-D9OC6YEPf0DwLJwTqITNkRWg0RvH8Gyxo4fkQw44E38" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/2/10/1179" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-M4usyxsHNO5U1BJuuR4WRdu82032LeD85piCIfX692o" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/2/10/1169" title="Identification of Genes Underlying Hypoxia Tolerance in Drosophila by a P-element Screen" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/2/10/1185" title="Multiple Pathways Regulate Minisatellite Stability During Stationary Phase in Yeast" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459336" id="node459336" data-pisa="ggg;2/10/1179" data-pisa-master="ggg;2/10/1179" data-apath="/ggg/2/10/1179.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Fast and Efficient Approach for Genomic Selection with High-Density Markers</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Vitara Pungpapong</span>, <span class="highwire-citation-author" data-delta="1">William M. Muir</span>, <span class="highwire-citation-author" data-delta="2">Xianran Li</span>, <span class="highwire-citation-author" data-delta="3">Dabao Zhang</span> and <span class="highwire-citation-author" data-delta="4">Min Zhang</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2012 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 2 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1179-1184;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.003822 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459336" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Vitara Pungpapong </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Statistics, Purdue University, West Lafayette, Indiana 47907</div><div class='author-affiliation'><span class='nlm-sup'>â </span>Department of Statistics, Chulalongkorn Business School, Bangkok, Thailand 10330</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Vitara%2BPungpapong%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pungpapong%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AVitara%2BPungpapong%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">William M. Muir </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â¡</span>Department of Animal Science, Purdue University, West Lafayette, Indiana 47907</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=William%2BM.%2BMuir%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Muir%20WM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AWilliam%2BM.%2BMuir%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Xianran Li </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>Â§</span>Department of Agronomy, Kansas State University, Manhattan, Kansas 66506, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xianran%2BLi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Li%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AXianran%2BLi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Dabao Zhang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Statistics, Purdue University, West Lafayette, Indiana 47907</div><div class='author-affiliation'><span class='nlm-sup'>**</span>School of Biomedical Engineering, Capital Medical University, Beijing, China 100069</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dabao%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADabao%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:zhangdb@stat.purdue.edu" class="" data-icon-position="" data-hide-link-title="0">zhangdb@stat.purdue.edu</a>
<a href="mailto:minzhang@stat.purdue.edu" class="" data-icon-position="" data-hide-link-title="0">minzhang@stat.purdue.edu</a></span></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Min Zhang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Statistics, Purdue University, West Lafayette, Indiana 47907</div><div class='author-affiliation'><span class='nlm-sup'>**</span>School of Biomedical Engineering, Capital Medical University, Beijing, China 100069</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Min%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMin%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:zhangdb@stat.purdue.edu" class="" data-icon-position="" data-hide-link-title="0">zhangdb@stat.purdue.edu</a>
<a href="mailto:minzhang@stat.purdue.edu" class="" data-icon-position="" data-hide-link-title="0">minzhang@stat.purdue.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/2/10/1179" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459336" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459336/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/2/10/1179.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459336" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459336/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/2/10/1179.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459336" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459336/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/2/10/1179.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459336" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459336/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/2/10/1179.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459336" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459336/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">Recent advances in high-throughput genotyping have motivated genomic selection using high-density markers. However, an increasingly large number of markers brings up both statistical and computational issues and makes it difficult to estimate the breeding values. We propose to apply the penalized orthogonal-components regression (POCRE) method to estimate breeding values. As a supervised dimension reduction method, POCRE sequentially constructs linear combinations of markers, <em>i.e.</em> orthogonal components, such that these components are most closely correlated to the phenotype. Such a dimension reduction is able to group highly correlated predictors and allows for collinear or nearly collinear markers. Different from BayesB, which predetermines hyperparameters, POCRE uses an empirical Bayes thresholding method to obtain data-driven optimal hyperparameters and effectively select important markers when constructing each component. Demonstrated through simulation studies, POCRE greatly reduces the computing time compared with BayesB. On the other hand, unlike fBayesB which slightly sacrifices prediction accuracy for fast computation, POCRE provides similar or even better accuracy of predicting breeding values than BayesB in both simulation studies and real data analyses.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/genotypic-estimate-breeding-values-gebv" class="hw-term hw-article-keyword hw-article-keyword-genotypic-estimate-of-breeding-values-gebv" rel="nofollow">genotypic estimate of breeding values (GEBV)</a></li><li class="kwd"><a href="/keyword/genomic-selection" class="hw-term hw-article-keyword hw-article-keyword-genomic-selection" rel="nofollow">genomic selection</a></li><li class="kwd"><a href="/keyword/penalized-orthogonal-components-regression-pocre" class="hw-term hw-article-keyword hw-article-keyword-penalized-orthogonal-components-regression-pocre" rel="nofollow">penalized orthogonal-components regression (POCRE)</a></li><li class="kwd"><a href="/keyword/phenotypic-estimate-breeding-values-pebv" class="hw-term hw-article-keyword hw-article-keyword-phenotypic-estimate-of-breeding-values-pebv" rel="nofollow">phenotypic estimate of breeding values (PEBV)</a></li><li class="kwd"><a href="/keyword/genpred" class="hw-term hw-article-keyword hw-article-keyword-genpred" rel="nofollow">GenPred</a></li><li class="kwd"><a href="/keyword/shared-data-resources" class="hw-term hw-article-keyword hw-article-keyword-shared-data-resources" rel="nofollow">Shared data resources</a></li></ul><p id="p-4">The advantages of using genome-wide genetic markers to estimate the breeding values have been widely recognized (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Meuwissen <em>et al.</em> 2001</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Calus and Veerkamp 2007</a>). Recent developments in high-throughput genotyping technology make it feasible and may further popularize the genomic selection in both plant and animal breeding programs (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Meuwissen 2007</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Zhong <em>et al.</em> 2009</a>). However, such genomic selection practice is statistically challenging by analyzing a large number of genetic markers and efficiently estimating the breeding values. Moreover, high correlation between genetic markers and a relatively small number of individuals makes it difficult to directly employ classical statistical methods to estimate the breeding values.</p><p id="p-5">A variety of methods have been proposed to estimate the total breeding value, for example, least squares (LS) approach, best linear unbiased prediction (BLUP), and Bayesian methods (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Meuwissen <em>et al.</em> 2001</a>, <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">2009</a>). With a large number of genetic markers, the least squares approach relies on a suboptimal forward stepwise procedure to select significant genetic markers, and then uses them to predict the genetic values. The BLUP method is usually practiced using a single variance parameter to model the variation of all genes. While BLUP may be extended to allow for heterogeneous variances across loci, <a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">Meuwissen <em>et al.</em> (2001)</a> proposed Bayesian models, which allow for heterogeneous variation across genes, and implemented BayesB using a Markov chain Monte Carlo algorithm. Unlike BLUP, which assumes genetic effects drawn from a normal distribution, BayesB implies a Student t-distribution that has heavier tails than the normal distribution and therefore can capture extreme genetic effects. To address the intensive computation required by BayesB, <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Meuwissen <em>et al.</em> (2009)</a> employed an iterative conditional expectation (ICE) method to speed up the algorithm, hence named fBayesB, but slightly killed the accuracy of predictions.</p><p id="p-6"><a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Long <em>et al.</em> (2007)</a> proposed a two-step machine learning classification procedure to select SNPs for genomic selection with binary traits. As a preprocessing method, an information gain-based (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Mitchell 1997</a>) filter was employed as the first step to select a small arbitrary number of genetic markers out of a huge number of candidates. At the second step, a naive Bayesian classifier (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Elkan 1997</a>) was adopted to optimize performance of the selected genetic markers. Although losing efficiency, dichotomizing phenotypic values was suggested to apply the above procedure for genomic selection with continuous traits (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Long <em>et al.</em> 2007</a>). <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Solberg <em>et al.</em> (2009)</a> applied dimension reduction methods, such as the partial least squares regression (PLSR) and principal component regression (PCR) to genomic selection and showed that their performance was inferior to BayesB, although both methods were computationally faster and simpler.</p><p id="p-7">The fact that an <em>L</em><sub>1</sub> penalty function can enable variable selection has recently motivated many penalized least squares methods to fit high-dimensional linear regression models, such as lasso proposed by <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Tibshirani (1996)</a>. <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Usai <em>et al.</em> (2009)</a> applied the lasso method to estimate marker effects for genomic selection and concluded that it provided more accurate estimates of breeding values than BLUP and BayesA (<a id="xref-ref-12-4" class="xref-bibr" href="#ref-12">Meuwissen <em>et al.</em> 2001</a>). However, lasso lacks a grouping property, that is, it tends to select one marker from a group of highly correlated markers (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Zou and Hastie 2005</a>). Such a grouping property may play an important role in genomic selection, as markers from the same quantitative trait locus or genetic region are usually highly correlated and therefore may be preferred to be simultaneously included in or excluded from the prediction model. Many lasso variants have been proposed to take advantage of the grouped markers either implicitly or explicitly, for example, <a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Zou and Hastie (2005)</a>, <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Tibshirani <em>et al.</em> (2005)</a>, and <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Yuan and Lin (2006)</a>.</p><p id="p-8">Here we propose to apply the penalized orthogonal-components regression (POCRE) to estimate/predict breeding values and therefore enforce genomic selection. POCRE was proposed by <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Zhang <em>et al.</em> (2009)</a> to select variables from high-dimensional, low-sample-size data. POCRE takes advantage of both Bayesian inference and supervised dimension reduction. With supervised dimension reduction, POCRE first constructs properly penalized orthogonal components, which are, upon standardization, most closely correlated to the response variables. A new penalization framework, implemented via empirical Bayes thresholding, is presented to effectively identify sparse markers for each component. POCRE is computationally efficient owing to its sequential construction of sparse orthogonal components. In addition, such construction offers other properties such as grouping highly correlated markers and allowing for collinear or nearly collinear markers.</p><p id="p-9">On the basis of simulation studies, <a id="xref-ref-12-5" class="xref-bibr" href="#ref-12">Meuwissen <em>et al.</em> (2001)</a> compared four methods, including LS, BLUP, BayesA, and BayesB, and concluded that BayesB had the best overall performance. Therefore, we demonstrated the utility of POCRE in genomic selection by comparing it with BayesB using similar extensive simulation studies, and the performance was evaluated in terms of the prediction accuracy and computation time. We also applied both BayesB and POCRE to a pine dataset and a maize dataset to estimate the genetic values. With the data split into training subsets and test subsets, we compared the estimation accuracy.</p><div class="section methods" id="sec-1"><h2 class="">Methods</h2><div id="sec-2" class="subsection"><h3>Penalized orthogonal-components regression (POCRE)</h3><p id="p-10">With <em>p</em> markers genotyped from a total of <em>n</em> individuals, the classical regression model can be written as<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/10/1179/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/10/1179/embed/mml-math-1.gif"/></noscript></span><span class="disp-formula-label">(1)</span></span>where <strong>Y</strong><em><sub>n</sub></em><sub>Ã1</sub> collects the phenotypic value of all individuals, <strong>X</strong><em><sub>n</sub></em><sub>Ã</sub><em><sub>p</sub></em> represents the genotype information of all individuals, <em>Î¼</em> is an overall mean of polygenic effects, <em>Î²<sub>p</sub></em><sub>Ã1</sub> corresponds to the effect of each marker, and Îµ is the error term. Because typical genomic selection presents many more markers than individuals, in other words, <em>p</em> â« <em>n</em>, special algorithms are needed to estimate <em>Î²</em>. <a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Zhang <em>et al.</em> (2009)</a> recently described a penalized orthogonal-components regression (POCRE) approach that sequentially constructs sparsely loaded orthogonal components with proper regularization. They demonstrated that this approach works well when fitting regression models with <em>p</em> â« <em>n</em> data.</p><p id="p-11">Assume that both <strong>Y</strong> and <strong>X</strong> are centralized, and accordingly, assume <em>Î¼</em> = 0 in <a id="xref-disp-formula-1-1" class="xref-disp-formula" href="#disp-formula-1">Model 1</a>. POCRE sequentially constructs orthogonal components <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/10/1179/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/10/1179/embed/mml-math-2.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/10/1179/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/10/1179/embed/mml-math-3.gif"/></noscript></span></span>, â¦, where <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/10/1179/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/10/1179/embed/mml-math-4.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/10/1179/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/10/1179/embed/mml-math-5.gif"/></noscript></span></span>, <em>k</em> â¥ 2, are iteratively built to be orthogonal to <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/10/1179/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/10/1179/embed/mml-math-6.gif"/></noscript></span></span>. The loading <em>Ï<sub>k</sub></em>, <em>k</em> â¥ 1, is obtained as <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/10/1179/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/10/1179/embed/mml-math-7.gif"/></noscript></span></span> which together with <em>Î±</em> minimizes<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/10/1179/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/10/1179/embed/mml-math-8.gif"/></noscript></span><span class="disp-formula-label">(2)</span></span>where <em>g<sub>Î»</sub></em>(<em>Î³</em>) is a penalty function defined by a proper regularization on <em>Î³</em> with tuning parameter <em>Î»</em>.</p><p id="p-12">When <em>g<sub>Î»</sub></em>(<em>Î³</em>) â¡ 0, the optimal <em>Î³</em> solving <a id="xref-disp-formula-2-1" class="xref-disp-formula" href="#disp-formula-2">Equation 2</a> is proportional to the leading eigenvector of <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/10/1179/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/10/1179/embed/mml-math-9.gif"/></noscript></span></span>, that is, <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/10/1179/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/10/1179/embed/mml-math-10.gif"/></noscript></span></span> when a single phenotype is considered. <a id="xref-ref-26-3" class="xref-bibr" href="#ref-26">Zhang <em>et al.</em> (2009)</a> employed the empirical Bayes thresholding method proposed by <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Johnstone and Silverman (2004)</a> to introduce proper penalty <em>g<sub>Î»</sub></em>(<em>Î³</em>). While POCRE<sub>0</sub> directly applies the empirical Bayes thresholding method, POCRE applies it to the statistics from Fisherâs z-transformation. Such penalty helps estimating covariance between phenotype and genotypes, and it provides adaptively sparse loadings of orthogonal components. The empirical Bayes implementation is also computationally efficient. The tuning parameter <em>Î»</em> can be appropriately set to account for possible dependence structure among different markers.</p><p id="p-13">The sequential construction of the orthogonal components stops when the optimal <em>Î³</em> solving <a id="xref-disp-formula-2-2" class="xref-disp-formula" href="#disp-formula-2">Equation 2</a> is zero, which implies that <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/10/1179/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/10/1179/embed/mml-math-11.gif"/></noscript></span></span> is almost uncorrelated to <strong>Y</strong>. Then regressing <strong>Y</strong> on the orthogonal components, <em>i.e.</em> <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/10/1179/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/10/1179/embed/mml-math-12.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/10/1179/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/10/1179/embed/mml-math-13.gif"/></noscript></span></span>, â¦, provides estimates of <em>Î²</em><sub>1</sub>,â¦, <em>Î²<sub>p</sub></em> in Model 1. Since non-zero loadings in <em>Ï<sub>j</sub></em>, <em>j</em> = 1, 2,â¦, are sparse, most of the estimated <em>Î²</em><sub>1</sub>,â¦, <em>Î²<sub>p</sub></em> are therefore zero, reflecting the fact that most markers do not (significantly) contribute to predicting phenotypic values, as discussed in the literature, <em>e.g.</em> <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Hayes <em>et al.</em> (2009)</a>. Note that, when the above sparsity argument is violated, it is preferable to set <em>g<sub>Î»</sub></em>(<em>Î³</em>) â¡ 0 in <a id="xref-disp-formula-2-3" class="xref-disp-formula" href="#disp-formula-2">Equation 2</a>, which essentially leads to the partial least squares algorithm (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Wold 1975</a>) by selecting an optimal number of components.</p></div><div id="sec-3" class="subsection"><h3>Choice of the tuning parameter <em>Î»</em></h3><p id="p-14">While the tuning parameter <em>Î»</em> is used to account for possible dependence structure among different markers, it can be chosen on the basis of previous experience with similar studies. In the case of no experience with similar studies, a simulation study may be designed to utilize the available genotypic values and to reveal an appropriate choice of the tuning parameter. We will discuss several strategies on the basis of cross-validation (CV) to elicit a data-driven optimal value for the tuning parameter in the simulation studies.</p></div><div id="sec-4" class="subsection"><h3>Simulation studies</h3><p id="p-15">A gene-level simulation program was utilized as described by <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Muir (2007)</a>. For this set of simulations, the conditions and assumptions were as similar to <a id="xref-ref-12-6" class="xref-bibr" href="#ref-12">Meuwissen <em>et al.</em> (2001)</a> as possible. Specifically, the genetic architecture included 1000 SNP loci and 1000 QTL, both biallelic. There were a total of 10 chromosomes, and the length of each chromosome was 100 cM, which implies one marker per centimorgan. The 1000 SNP loci were equally spaced, whereas the 1000 QTL were randomly generated from a uniform distribution across the genome. The allelic effects were additive and drawn from a gamma distribution with a shape parameter of 0.4 and scale parameter of 1.66, meaning there will be few alleles with large effects. The mutation rate was set at 2.5 Ã 10<sup>â5</sup> and 2.5 Ã 10<sup>â3</sup> for QTL and marker SNP, respectively, per locus per generation. On the basis of the genetic variance, environmental effects were generated from a Gaussian distribution with mean zero and variance calculated to meet the required heritability, which is 0.5 in the simulation. To establish mutation drift equilibrium, a population of 100 (50 males and 50 females) were allowed to randomly mate for 1000 generations. Each family produced 2 offspring. The population was expanded by twice in the next generation (<em>i.e.</em> 200 individuals at generation 1001) and then by a factor of 10 in the following generations (<em>i.e.</em> 2000 individuals for each generation after generation 1001). The 2200 individuals at generations 1001 and 1002 were genotyped and phenotyped, respectively (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Muir 2007</a>), and served as the training data. The remaining generations, serving as the test data, were only genotyped but not phenotyped, although the true breeding values were generated for testing the accuracy of models. A total of 50 replicates were simulated.</p><p id="p-16">As shown above, genomic selection can proceed on a population consisting of several generations. For instance, the data in simulation studies include two generations: 200 individuals from generation 1001, and 2000 individuals from generation 1002. With different ways to split the data, we consider three implementations of 10-fold CV in analyzing these datasets. The first strategy, termed as CV0, is to ignore the mixed generations and to randomly split 2200 individuals in each sample into 10 different folds, that is, each with 220 individuals. The second strategy, termed as CV1, always includes all 200 individuals from generation 1001 in the training data and only splits 2000 individuals from generation 1002 into 10 different folds. Therefore, each test data will only include 200 individuals from generation 1002 but none from generation 1001. The third strategy, termed as CV2, separately splits the 200 individuals from generations 1001 and the 2000 individuals from generation 1002 into 10 folds, such that each fold includes 20 individuals from generation 1001 and 200 individuals from generation 1002.</p><p id="p-17">POCRE and POCRE<sub>0</sub> were employed to analyze all simulated datasets with the tuning parameter elicited via cross-validation. All three cross-validation techniques (CV0, CV1, and CV2) were used in the analyses. Here we applied BayesB and compared it with POCRE and POCRE<sub>0</sub> in terms of the computation time and the prediction accuracy, which is defined as the correlation between the estimated and true breeding values. For BayesB, the probability <em>Ï</em> that a SNP has zero effect is assessed empirically from the data, and the other hyperparameters were chosen following <a id="xref-ref-12-7" class="xref-bibr" href="#ref-12">Meuwissen <em>et al.</em> (2001)</a>.</p></div></div><div class="section results" id="sec-5"><h2 class="">Results</h2><div id="sec-6" class="subsection"><h3>Prediction accuracy</h3><p id="p-18">A striking nature of BayesB is that it employs a mixture prior for each regression coefficient in <a id="xref-disp-formula-1-2" class="xref-disp-formula" href="#disp-formula-1">Equation 1</a> and allows it to take zero value with positive probability. Each coefficient in BayesB was estimated from the posterior mean, and a property of this estimator is that all means are non-zero. Though fBayesB has computational advantages over BayesB, <a id="xref-ref-13-3" class="xref-bibr" href="#ref-13">Meuwissen <em>et al.</em> (2009)</a> demonstrated that fBayesB is slightly less accurate than BayesB in terms of correlation between estimated breeding values (EBV) and true breeding values (TBV). Throughout this article, we shall focus on comparing accuracy between POCRE and BayesB only.</p><p id="p-19">As shown in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>, all POCRE and POCRE<sub>0</sub> methods provided better estimates of breeding values for generations 1001 and 1002 than BayesB. For POCRE and POCRE<sub>0</sub> methods, their predictions of breeding values for generation 1003 were not only more accurate but also more robust than those of BayesB. Indeed, except for generation 1004 with POCRE<sub>0</sub> (CV0), all POCRE implementations provided more robust predictions of breeding values than BayesB.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/461898/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/461898/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Prediction accuracy of BayesB and POCRE</span><div class="sb-div caption-clear"></div></div></div><p id="p-21">When comparing the three different strategies of cross-validation for both POCRE and POCRE<sub>0</sub>, we observe that CV0 was the worst strategy as it always provided the lowest accuracy in predicting breeding values among all three strategies for both POCRE and POCRE<sub>0</sub>. For POCRE, CV1 performed the best by providing better prediction accuracy than the other two. However, for POCRE<sub>0</sub>, CV2 provided the best prediction accuracy in generations 1003â1005, whereas CV1 provided the best prediction accuracy in generations 1006 and 1007. In general, prediction accuracy between CV1 and CV2 was only slightly different for both POCRE and POCRE<sub>0</sub>. All cross-validation strategies were similarly robust in predicting breeding values. Therefore, we suggest using either CV1 or CV2 scheme for cross-validation to choose the tuning parameters for both POCRE and POCRE<sub>0</sub>.</p><p id="p-22">It is well known that the coefficient in a non-intercept simple regression of TBV against EBV, denoted as <em>Î²</em>(<em>TBV</em>, <em>EBV</em>), can be used to evaluate the bias of genomic selection methods. With <em>Î²</em>(<em>TBV</em>, <em>EBV</em>) = 1, the method provides unbiased prediction of the breeding values; <em>Î²</em>(<em>TBV</em>, <em>EBV</em>) &lt; 1 implies an overestimated breeding value, and <em>Î²</em>(<em>TBV</em>, <em>EBV</em>) &gt; 1 implies an underestimated breeding value. <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a> shows the estimated <em>Î²</em>(<em>TBV</em>, <em>EBV</em>) of all methods across different generations. The predictions of all methods alternated between overestimate and underestimate from generations 1003 to 1007. That is, these methods all overestimated/overpredicted the breeding values in generations 1001/1002, 1004, and 1006, but they underpredicted the breeding values in generations 1003, 1005, and 1007. In general, all methods performed quite well in predicting the breeding values in terms of bias.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/461849/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/461849/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Comparison of the estimated <em>Î²</em>(<em>TBV</em>, <em>EBV</em>) using BayesB and POCRE</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-7" class="subsection"><h3>Computing time</h3><p id="p-24">As one of the primary goals of this study was to propose a fast approach for genomic selection, we compared the computing time of BayesB with POCRE. All methods were programmed in MATLAB, and all simulated data were analyzed on a Linux server with dual Intel Xeon Quad-Core 5410 2.33 GHz processors without taking advantage of parallel computation. We witnessed that the computing times for POCRE and POCRE<sub>0</sub> were similar and both took much less time than BayesB. Specifically, BayesB took 14,685.12 sec for one dataset based on 5,000 burn-ins and another 5000 iterations for inference.</p><p id="p-25">When the tuning parameters of POCRE and POCRE<sub>0</sub> need to be elicited through cross-validation, the computation time relies on the number of folds for cross-validation and the number of candidate values searched for the tuning parameter. For example, when a 10-fold cross-validation is utilized to search the tuning parameter from the pool {0.7, 0.71,â¦, 0.8}, 101 datasets need to be analyzed by either POCRE or POCRE<sub>0</sub> for choosing the tuning parameter; therefore, it took about 1010 sec to search for the optimal tuning parameter and another 10 sec to fit the model. Hence, the computation time is about 1/14 of that of BayesB. To further reduce the computation time of POCRE and POCRE<sub>0</sub>, some strategies (discussed later) can be designed to search for a much smaller number of candidate tuning parameter values during cross-validation, and the number of folds can be reduced when a large number of individuals is available. Indeed, available multiple processors in many computers imply that parallel computation can greatly shorten the computing time of POCRE and POCRE<sub>0</sub>.</p></div><div id="sec-8" class="subsection"><h3>Real data analyses</h3><p id="p-26">Although genomic selection emerged from dairy cattle breeding programs, it is also useful to plant breeding programs (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Jannink <em>et al.</em> 2010</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Bernardo and Yu 2007</a>). To further demonstrate the utility of the proposed method, we applied the methods to two datasets: pine and maize.</p><div id="sec-9" class="subsection"><h4>Pine data:</h4><p id="p-27">The pine dataset consists of 850 individuals and 4698 SNPs. Missing genotypes were imputed by sampling alleles from a Bernoulli distribution with variance equal to locus allele frequency. Individuals and SNPs having more than 20% missing values were removed from the data before the analysis. Diameter at breast height (DBH) and height (HT) were collected for pine trees in Nassau that were used in our analysis. Details of this data are referred to in <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Resende <em>et al.</em> (2012)</a>.</p><p id="p-28">Here we applied POCRE and POCRE<sub>0</sub> to evaluate their performance with 10-fold cross-validation because independent validation population is not available. As shown in <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>, POCRE and POCRE<sub>0</sub> yielded similar accuracies. Indeed, POCRE provided more robust accuracies for both DBH and HT traits. Accuracies of estimated breeding values for POCRE were also better than POCRE<sub>0</sub>, except in the training data of HT trait. The results of POCRE and POCRE<sub>0</sub> were similar in terms of the bias as shown in <a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/461911/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/461911/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Results of POCRE and POCRE<sub>0</sub> for analyzing pine dataset</span><div class="sb-div caption-clear"></div></div></div><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/461926/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/461926/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">Results of POCRE and POCRE<sub>0</sub> for analyzing pine dataset</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-10" class="subsection"><h4>Maize data:</h4><p id="p-31">This data were collected from a maize study on flowering time (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Buckler <em>et al.</em> 2009</a>). The maize nested association mapping (NAM) population, consisting of 5000 recombinant inbred lines (RIL), was created by crossing 25 diverse lines to a common parent, B73 (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Yu <em>et al.</em> 2008</a>). The phenotype data for days to anthesis (DA, male flowering) collected for 5000 RILs were used in this analysis; this trait has an estimated heritability around 94% (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Buckler <em>et al.</em> 2009</a>). The genotype data are SNPs with 5% or more minor allele frequency across 10 chromosomes, a total of 284,025 SNPs. Briefly, all 5000 NAM RILs and 26 parents were genotyped with 1106 B73-specific tag SNPs (tSNP) across the genome, and additional SNPs were obtained through sequencing of the 26 parents (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Gore <em>et al.</em> 2009</a>). Imputations were performed with fastPHASE (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Scheet and Stephens 2006</a>) to fill the data if a SNP was missing in less than 5 parents, and the resulting SNP data were then projected to 5000 RILs based on tSNPs (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">McMullen <em>et al.</em> 2009</a>; <a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Yu <em>et al.</em> 2008</a>).</p><p id="p-32">We randomly split the data into training and test sets of sizes 3500 and 1392, respectively, to evaluate the performance of our method. Five replicates were carried out to test the accuracy, which results in five pairs of training and test sets. Without the true breeding values (TBV), we can estimate them using either the original phenotypic values (termed PEBV hereafter) or genotypic values (termed GEBV hereafter) with BayesB or POCRE. The correlation between PEBV and GEBV, that is, <em>corr</em>(<em>PEBV</em>, <em>GEBV</em>), was estimated to assess the accuracy of genomic selection. Indeed, <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/10/1179/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/10/1179/embed/mml-math-14.gif"/></noscript></span></span> with <em>h</em><sup>2</sup> referring to the heritability.</p><p id="p-33">With the large number of SNPs included in this example, it was computationally intensive to run the Markov chain Monte Carlo for BayesB. Therefore, we only applied POCRE and POCRE<sub>0</sub> to analyze the whole-genome dataset, with 5-fold cross-validation to elicit the tuning parameters. <a id="xref-table-wrap-5-1" class="xref-table" href="#T5">Table 5</a> shows the results based on the five replicates. For the training datasets, we observed that POCRE provided estimated <em>corr</em>(<em>PEBV</em>, <em>GEBV</em>) averaged at 0.9381, and POCRE<sub>0</sub> at 0.9426; both are very close to the estimated heritability 94%. Both methods are robust in estimating the breeding values of the training datasets. For the test datasets, both methods provided robust estimates of breeding values, with estimated standard deviations 0.0045 and 0.0039 for POCRE and POCRE<sub>0</sub>, respectively, without the first replicate. <a id="xref-table-wrap-6-1" class="xref-table" href="#T6">Table 6</a> shows that the bias of the estimation is minimal.</p><div id="T5" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/462025/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/462025/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 5</span> <span class="caption-title">Results of analyzing maize flowering time data using POCRE and POCRE<sub>0</sub></span><div class="sb-div caption-clear"></div></div></div><div id="T6" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/462032/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/462032/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 6</span> <span class="caption-title">Results of the estimated <em>Î²</em>(<em>TBV</em>, <em>EBV</em>) in analyzing maize flowering time data using POCRE and POCRE<sub>0</sub></span><div class="sb-div caption-clear"></div></div></div><p id="p-35">To compare BayesB, POCRE, and POCRE<sub>0</sub>, we chose 2000 SNPs based on univariate tests performed for each SNP separately. From the SNPs with <em>P</em>-values less than 0.0001, we randomly selected 200 SNPs, and we chose 1800 SNPs from those with <em>P</em>-values greater than 0.0001. With the selected 2000 SNPs, we applied all methods to re-analyze the five replicates. Five-fold cross-validation was employed to elicit the tuning parameter for POCRE and POCRE<sub>0</sub>. The results are summarized in <a id="xref-table-wrap-7-1" class="xref-table" href="#T7">Tables 7</a> and <a id="xref-table-wrap-8-1" class="xref-table" href="#T8">8</a>. Both POCRE and POCRE<sub>0</sub> provided the larger estimate of <em>corr</em>(<em>PEBV</em>, <em>GEBV</em>) for the training datasets, and they predicted the breeding values with better accuracy and more robustly than BayesB. Note that with only 2000 SNPs, <em>corr</em>(<em>PEBV</em>, <em>GEBV</em>) estimated from the test datasets are smaller than those presented in <a id="xref-table-wrap-5-2" class="xref-table" href="#T5">Table 5</a>.</p><div id="T7" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/461883/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/461883/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 7</span> <span class="caption-title">Comparison of BayesB and POCRE for analyzing the maize flowering time data using 2000 SNPs</span><div class="sb-div caption-clear"></div></div></div><div id="T8" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/461980/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/461980/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 8</span> <span class="caption-title">Comparison of BayesB and POCRE for analyzing the maize flowering time data using 2000 SNPs</span><div class="sb-div caption-clear"></div></div></div></div></div></div><div class="section discussion" id="sec-11"><h2 class="">Discussion</h2><p id="p-38">Here we present alternative approaches for genomic selection that provide competitive prediction of breeding values and also significantly reduce the computing time. First, note that the prediction accuracy of both POCRE and POCRE<sub>0</sub> are comparable to that of BayesB in simulation studies, and their performance is much better than BayesB in analyzing maize flowering time data. Second, POCRE and POCRE<sub>0</sub> enjoy enormous computational advantage over BayesB. Finally, POCRE and POCRE<sub>0</sub> make it feasible to analyze data with a huge number of markers. For example, both POCRE and POCRE<sub>0</sub> (with tuning parameters elicited via 5-fold cross-validation) took about 36 hr to analyze the maize flowering time data, which includes a total of 284,025 SNPs for each of the 5000 genetic lines. This is promising because current high-throughput biotechnologies routinely genotype millions of markers. Our proposed method has the potential to take advantage of such high-dimensional data that may challenge the genomic selection.</p><p id="p-39">Results from both simulation studies and real data analysis showed the importance of the tuning parameter for the performance of POCRE and POCRE<sub>0</sub>. While cross-validation is a popular method to select the tuning parameter, it may not be optimum as shown in our simulation studies and others (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Wang <em>et al.</em> 2007</a>). Because the squared error loss was used as a criterion in cross-validation to select the optimal tuning parameter with training data, it suffers from overfitting the model, even with a large sample size. To explore various choices of tuning parameter, wider ranges can be used. Optimal tuning parameters in different genomic selection studies may differ considerably, and this issue is still under investigation.</p><p id="p-40">As evidenced in the simulation studies, selecting the tuning parameter using cross-validation should also consider the possible population structure in the data. The time usage for cross-validation relies on the number of folds and number of potential tuning parameter values under investigation. Parallel computation can significantly reduce the computation time of cross-validation. As genomic selection datasets usually include a large number of individuals, we can choose the number of folds as small as two. We have considered a grid search of the optimal tuning parameter in the simulation studies and real data analysis. For a strategic thorough search, we may first take a coarse grid to search and then look into a finer interval suggested from the initial coarse grid search. On the other hand, some targeted simulation studies may be designed to shed light on the optimal tuning parameter for a specific study.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-41">This project was partially supported by the National Science Foundation CAREER IIS-0844945, National Institutes of Health U01-CA1-28535, Discovery Park at Purdue University, National Research Initiative Competitive Grant 2008-35205-04723, and Agriculture and Food Research Initiative Competitive Grant 2009-65205-05665 from the U.S. Department of Agriculture National Institute of Food and Agriculture.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003822/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003822/-/DC1</a>.</p></li><li class="fn" id="fn-9"><p id="p-42"><em>Communicating editor: D. J. de Koning</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2012-04-29"><span class="received-label">Received </span>April 29, 2012.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2012-07-25"><span class="accepted-label">Accepted </span>July 25, 2012.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright Â© 2012 Pungpapong <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">âµ</a><div class="cit ref-cit ref-journal" id="cit-2.10.1179.1" data-doi="10.2135/cropsci2006.11.0690"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bernardo</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span>â<span class="cit-article-title">Prospects for genomewide selection for quantitative traits in maize</span>. <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">1082</span>â<span class="cit-lpage">1090</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DBernardo%26rft.auinit1%253DR.%26rft.volume%253D47%26rft.issue%253D3%26rft.spage%253D1082%26rft.epage%253D1090%26rft.atitle%253DProspects%2Bfor%2BGenomewide%2BSelection%2Bfor%2BQuantitative%2BTraits%2Bin%2BMaize%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2006.11.0690%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2006.11.0690&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000247424200020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">âµ</a><div class="cit ref-cit ref-journal" id="cit-2.10.1179.2" data-doi="10.1126/science.1174276"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Buckler</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holland</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bradbury</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Acharya</span>  <span class="cit-name-given-names">C. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">The genetic architecture of maize flowering time</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">325</span>: <span class="cit-fpage">714</span>â<span class="cit-lpage">718</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DBuckler%26rft.auinit1%253DE.%2BS.%26rft.volume%253D325%26rft.issue%253D5941%26rft.spage%253D714%26rft.epage%253D718%26rft.atitle%253DThe%2BGenetic%2BArchitecture%2Bof%2BMaize%2BFlowering%2BTime%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1174276%26rft_id%253Dinfo%253Apmid%252F19661422%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjUvNTk0MS83MTQiO3M6NDoiYXRvbSI7czoxOToiL2dnZy8yLzEwLzExNzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">âµ</a><div class="cit ref-cit ref-journal" id="cit-2.10.1179.3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Calus</span>  <span class="cit-name-given-names">M. P. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Veerkamp</span>  <span class="cit-name-given-names">R. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span>â<span class="cit-article-title">Accuracy of breeding values when using and ignoring the polygenic effects in genomic breeding value estimation wiht a marker density of one SNP per cM</span>. <abbr class="cit-jnl-abbrev">J. Anim. Breed. Genet.</abbr> <span class="cit-vol">124</span>: <span class="cit-fpage">362</span>â<span class="cit-lpage">368</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DJ%2BAnim%2BBreed%2BGenet%26rft.aulast%253DCalus%26rft.auinit1%253DM.%2BP.%26rft.volume%253D124%26rft.issue%253D6%26rft.spage%253D362%26rft.epage%253D368%26rft.atitle%253DAccuracy%2Bof%2Bbreeding%2Bvalues%2Bwhen%2Busing%2Band%2Bignoring%2Bthe%2Bpolygenic%2Beffect%2Bin%2Bgenomic%2Bbreeding%2Bvalue%2Bestimation%2Bwith%2Ba%2Bmarker%2Bdensity%2Bof%2Bone%2BSNP%2Bper%2BcM.%26rft_id%253Dinfo%253Apmid%252F18076473%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=18076473&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F10%2F1179.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000251432800006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">âµ</a><div class="cit ref-cit ref-book" id="cit-2.10.1179.4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Elkan</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span>â<span class="cit-source">Boosting and Naive Bayesian Learning</span>. <span class="cit-comment">Technical Report No. CS97-557. Department of Computer Science and Engineering, University of California, San Diego</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">âµ</a><div class="cit ref-cit ref-journal" id="cit-2.10.1179.5" data-doi="10.1126/science.1177837"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gore</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chia</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Elshire</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ersoz</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">A first-generation haplotype map of maize</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">326</span>: <span class="cit-fpage">1115</span>â<span class="cit-lpage">1117</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DGore%26rft.auinit1%253DM.%2BA.%26rft.volume%253D326%26rft.issue%253D5956%26rft.spage%253D1115%26rft.epage%253D1117%26rft.atitle%253DA%2BFirst-Generation%2BHaplotype%2BMap%2Bof%2BMaize%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1177837%26rft_id%253Dinfo%253Apmid%252F19965431%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMjYvNTk1Ni8xMTE1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMi8xMC8xMTc5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">âµ</a><div class="cit ref-cit ref-journal" id="cit-2.10.1179.6" data-doi="10.3168/jds.2008-1646"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bowman</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chamberlain</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">Invited review: Genomic selection in dairy cattle: progress and challengess</span>. <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">92</span>: <span class="cit-fpage">433</span>â<span class="cit-lpage">443</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DHayes%26rft.auinit1%253DB.%2BJ.%26rft.volume%253D92%26rft.issue%253D2%26rft.spage%253D433%26rft.epage%253D443%26rft.atitle%253DInvited%2Breview%253A%2BGenomic%2Bselection%2Bin%2Bdairy%2Bcattle%253A%2BProgress%2Band%2Bchallenges%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2008-1646%26rft_id%253Dinfo%253Apmid%252F19164653%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2008-1646&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19164653&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F10%2F1179.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000262654900001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">âµ</a><div class="cit ref-cit ref-journal" id="cit-2.10.1179.7" data-doi="10.1093/bfgp/elq001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lorenz</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Iwata</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title">Genomic selection in plant breeding: from theory to practics</span>. <abbr class="cit-jnl-abbrev">Briefings in Functional Genomics</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">166</span>â<span class="cit-lpage">177</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBriefings%2Bin%2BFunctional%2BGenomics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbfgp%252Felq001%26rft_id%253Dinfo%253Apmid%252F20156985%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoiYmZncCI7czo1OiJyZXNpZCI7czo3OiI5LzIvMTY2IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMi8xMC8xMTc5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">âµ</a><div class="cit ref-cit ref-journal" id="cit-2.10.1179.8" data-doi="10.1214/009053604000000030"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Johnstone</span>  <span class="cit-name-given-names">I. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Silverman</span>  <span class="cit-name-given-names">B. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span>â<span class="cit-article-title">Needles and straw in haystacks: empirical Bayes estimates of possibly sparse sequences</span>. <abbr class="cit-jnl-abbrev">Ann. Stat.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">1594</span>â<span class="cit-lpage">1649</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BStat.%26rft.volume%253D32%26rft.spage%253D1594%26rft_id%253Dinfo%253Adoi%252F10.1214%252F009053604000000030%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1214/009053604000000030&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000223519100010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">âµ</a><div class="cit ref-cit ref-journal" id="cit-2.10.1179.9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Long</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rose</span>  <span class="cit-name-given-names">G. J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weigel</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Avendano</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span>â<span class="cit-article-title">Machine learning classification procedure for selection SNPs in genomic selection: application to early mortality in broilers</span>. <abbr class="cit-jnl-abbrev">J. Anim. Breed. Genet.</abbr> <span class="cit-vol">124</span>: <span class="cit-fpage">377</span>â<span class="cit-lpage">389</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DJ%2BAnim%2BBreed%2BGenet%26rft.aulast%253DLong%26rft.auinit1%253DN.%26rft.volume%253D124%26rft.issue%253D6%26rft.spage%253D377%26rft.epage%253D389%26rft.atitle%253DMachine%2Blearning%2Bclassification%2Bprocedure%2Bfor%2Bselecting%2BSNPs%2Bin%2Bgenomic%2Bselection%253A%2Bapplication%2Bto%2Bearly%2Bmortality%2Bin%2Bbroilers.%26rft_id%253Dinfo%253Apmid%252F18076475%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=18076475&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F10%2F1179.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000251432800008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">âµ</a><div class="cit ref-cit ref-journal" id="cit-2.10.1179.10" data-doi="10.1126/science.1174320"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McMullen</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kresovich</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Villeda</span>  <span class="cit-name-given-names">H. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bradbury</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">Genetic properties of the maize nested association mapping population</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">325</span>: <span class="cit-fpage">737</span>â<span class="cit-lpage">740</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DMcMullen%26rft.auinit1%253DM.%2BD.%26rft.volume%253D325%26rft.issue%253D5941%26rft.spage%253D737%26rft.epage%253D740%26rft.atitle%253DGenetic%2BProperties%2Bof%2Bthe%2BMaize%2BNested%2BAssociation%2BMapping%2BPopulation%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1174320%26rft_id%253Dinfo%253Apmid%252F19661427%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjUvNTk0MS83MzciO3M6NDoiYXRvbSI7czoxOToiL2dnZy8yLzEwLzExNzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">âµ</a><div class="cit ref-cit ref-journal" id="cit-2.10.1179.11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>  <span class="cit-name-given-names">T. H. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span>â<span class="cit-article-title">Genomic selection: marker assisted selection on a genome wide scale</span>. <abbr class="cit-jnl-abbrev">J. Anim. Breed. Genet.</abbr> <span class="cit-vol">124</span>: <span class="cit-fpage">321</span>â<span class="cit-lpage">322</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DJ%2BAnim%2BBreed%2BGenet%26rft.aulast%253DMeuwissen%26rft.auinit1%253DT.%26rft.volume%253D124%26rft.issue%253D6%26rft.spage%253D321%26rft.epage%253D322%26rft.atitle%253DGenomic%2Bselection%253A%2Bmarker%2Bassisted%2Bselection%2Bon%2Ba%2Bgenome%2Bwide%2Bscale.%26rft_id%253Dinfo%253Apmid%252F18076468%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=18076468&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F10%2F1179.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">âµ</a><div class="cit ref-cit ref-journal" id="cit-2.10.1179.12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>  <span class="cit-name-given-names">T. H. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span>â<span class="cit-article-title">Prediction of total genetic value using genome-wide dense marker maps</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">157</span>: <span class="cit-fpage">1819</span>â<span class="cit-lpage">1829</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMeuwissen%26rft.auinit1%253DT.%2BH.%2BE.%26rft.volume%253D157%26rft.issue%253D4%26rft.spage%253D1819%26rft.epage%253D1829%26rft.atitle%253DPrediction%2Bof%2BTotal%2BGenetic%2BValue%2BUsing%2BGenome-Wide%2BDense%2BMarker%2BMaps%26rft_id%253Dinfo%253Apmid%252F11290733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1Ny80LzE4MTkiO3M6NDoiYXRvbSI7czoxOToiL2dnZy8yLzEwLzExNzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">âµ</a><div class="cit ref-cit ref-journal" id="cit-2.10.1179.13" data-doi="10.1186/1297-9686-41-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>  <span class="cit-name-given-names">T. H. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Solberg</span>  <span class="cit-name-given-names">T. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shepherd</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Woolliams</span>  <span class="cit-name-given-names">J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">A fast algorithm for BayesB type of prediction of genome-wide estimates of genetic value</span>. <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">2</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DMeuwissen%26rft.auinit1%253DT.%2BH.%26rft.volume%253D41%26rft.spage%253D2%26rft.epage%253D2%26rft.atitle%253DA%2Bfast%2Balgorithm%2Bfor%2BBayesB%2Btype%2Bof%2Bprediction%2Bof%2Bgenome-wide%2Bestimates%2Bof%2Bgenetic%2Bvalue.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-41-2%26rft_id%253Dinfo%253Apmid%252F19284681%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-41-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19284681&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F10%2F1179.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">âµ</a><div class="cit ref-cit ref-book" id="cit-2.10.1179.14"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mitchell</span>  <span class="cit-name-given-names">T. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span>â<span class="cit-source">Machine Learning</span>, <span class="cit-publ-name">McGraw-Hill</span>, <span class="cit-publ-loc">Hightstown, NJ</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">âµ</a><div class="cit ref-cit ref-journal" id="cit-2.10.1179.15" data-doi="10.1111/j.1439-0388.2007.00700.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Muir</span>  <span class="cit-name-given-names">W. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span>â<span class="cit-article-title">Comparison of genomic and traditional BLUP-estimated breeding value accuracy and selection response under alternative trait and genomic parameters</span>. <abbr class="cit-jnl-abbrev">J. Anim. Breed. Genet.</abbr> <span class="cit-vol">124</span>: <span class="cit-fpage">342</span>â<span class="cit-lpage">355</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DJ%2BAnim%2BBreed%2BGenet%26rft.aulast%253DMuir%26rft.auinit1%253DW.%2BM.%26rft.volume%253D124%26rft.issue%253D6%26rft.spage%253D342%26rft.epage%253D355%26rft.atitle%253DComparison%2Bof%2Bgenomic%2Band%2Btraditional%2BBLUP-estimated%2Bbreeding%2Bvalue%2Baccuracy%2Band%2Bselection%2Bresponse%2Bunder%2Balternative%2Btrait%2Band%2Bgenomic%2Bparameters.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1439-0388.2007.00700.x%26rft_id%253Dinfo%253Apmid%252F18076471%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1439-0388.2007.00700.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18076471&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F10%2F1179.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000251432800004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">âµ</a><div class="cit ref-cit ref-journal" id="cit-2.10.1179.16" data-doi="10.1534/genetics.111.137026"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Resende</span>  <span class="cit-name-given-names">M. F. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Munoz</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Resende</span>  <span class="cit-name-given-names">M. D. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garrick</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span>â<span class="cit-article-title">Accuracy of genomic selection methods in a standard dataset of Lobolly Pine (Pinus taeda L.)</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">190</span>: <span class="cit-fpage">1503</span>â<span class="cit-lpage">1510</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.137026%26rft_id%253Dinfo%253Apmid%252F22271763%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5MC80LzE1MDMiO3M6NDoiYXRvbSI7czoxOToiL2dnZy8yLzEwLzExNzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">âµ</a><div class="cit ref-cit ref-journal" id="cit-2.10.1179.17" data-doi="10.1086/502802"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Scheet</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span>â<span class="cit-article-title">A fast and flexible statistical model for large-scale population genotype data: applications to inferring missing genotypes and haplotypic phase</span>. <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">78</span>: <span class="cit-fpage">629</span>â<span class="cit-lpage">644</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DScheet%26rft.auinit1%253DP.%26rft.volume%253D78%26rft.issue%253D4%26rft.spage%253D629%26rft.epage%253D644%26rft.atitle%253DA%2Bfast%2Band%2Bflexible%2Bstatistical%2Bmodel%2Bfor%2Blarge-scale%2Bpopulation%2Bgenotype%2Bdata%253A%2Bapplications%2Bto%2Binferring%2Bmissing%2Bgenotypes%2Band%2Bhaplotypic%2Bphase.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F502802%26rft_id%253Dinfo%253Apmid%252F16532393%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/502802&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16532393&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F10%2F1179.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000236755900009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">âµ</a><div class="cit ref-cit ref-journal" id="cit-2.10.1179.18" data-doi="10.1186/1297-9686-41-29"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Solberg</span>  <span class="cit-name-given-names">T. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sonesson</span>  <span class="cit-name-given-names">A. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Woolliams</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>  <span class="cit-name-given-names">T. H. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">Reducing dimensionality for prediction of genome-wide breeding values</span>. <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">29</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DSolberg%26rft.auinit1%253DT.%2BR.%26rft.volume%253D41%26rft.spage%253D29%26rft.epage%253D29%26rft.atitle%253DReducing%2Bdimensionality%2Bfor%2Bprediction%2Bof%2Bgenome-wide%2Bbreeding%2Bvalues.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-41-29%26rft_id%253Dinfo%253Apmid%252F19296851%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-41-29&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19296851&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F10%2F1179.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">âµ</a><div class="cit ref-cit ref-journal" id="cit-2.10.1179.19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tibshirani</span>  <span class="cit-name-given-names">R. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span>â<span class="cit-article-title">Regression shrinkage and selection via the lasso</span>. <abbr class="cit-jnl-abbrev">J. R. Stat. Soc., B</abbr> <span class="cit-vol">58</span>: <span class="cit-fpage">267</span>â<span class="cit-lpage">288</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BR.%2BStat.%2BSoc.%252C%2BB%26rft.volume%253D58%26rft.spage%253D267%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">âµ</a><div class="cit ref-cit ref-journal" id="cit-2.10.1179.20" data-doi="10.1111/j.1467-9868.2005.00490.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tibshirani</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosset</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Knight</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span>â<span class="cit-article-title">Sparsity and smoothness via the fused lasso</span>. <abbr class="cit-jnl-abbrev">J. R. Stat. Soc., B</abbr> <span class="cit-vol">67</span>: <span class="cit-fpage">91</span>â<span class="cit-lpage">108</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BR.%2BStat.%2BSoc.%252C%2BB%26rft.volume%253D67%26rft.spage%253D91%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1467-9868.2005.00490.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1467-9868.2005.00490.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">âµ</a><div class="cit ref-cit ref-journal" id="cit-2.10.1179.21" data-doi="10.1017/S0016672309990334"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Usai</span>  <span class="cit-name-given-names">M. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">LASSO with cross-validation for genomic selection</span>. <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">427</span>â<span class="cit-lpage">436</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DUsai%26rft.auinit1%253DM.%2BG.%26rft.volume%253D91%26rft.issue%253D6%26rft.spage%253D427%26rft.epage%253D436%26rft.atitle%253DLASSO%2Bwith%2Bcross-validation%2Bfor%2Bgenomic%2Bselection.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672309990334%26rft_id%253Dinfo%253Apmid%252F20122298%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672309990334&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20122298&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F10%2F1179.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">âµ</a><div class="cit ref-cit ref-journal" id="cit-2.10.1179.22" data-doi="10.1093/biomet/asm053"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tsai</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span>â<span class="cit-article-title">Tuning parameter selectors for the smoothly clipped absolute deviation method</span>. <abbr class="cit-jnl-abbrev">Biometrika</abbr> <span class="cit-vol">94</span>: <span class="cit-fpage">553</span>â<span class="cit-lpage">568</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiometrika%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbiomet%252Fasm053%26rft_id%253Dinfo%253Apmid%252F19343105%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiYmlvbWV0IjtzOjU6InJlc2lkIjtzOjg6Ijk0LzMvNTUzIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMi8xMC8xMTc5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">âµ</a><div class="cit ref-cit ref-book" id="cit-2.10.1179.23"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wold</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">1975</span>â<span class="cit-article-title">Soft modelling by latent variables: the nonlinear iterative partial least squares approach, pp. 117â142</span> <span class="cit-comment">in</span> <span class="cit-source">Perspectives in Probability and Statistics, Papers in Honour of M. S. Bartlett</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Gani</span>  <span class="cit-name-given-names">J.</span></span>. <span class="cit-publ-name">Academic Press</span>, <span class="cit-publ-loc">London</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">âµ</a><div class="cit ref-cit ref-journal" id="cit-2.10.1179.24" data-doi="10.1534/genetics.107.074245"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holland</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McMullen</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buckler</span>  <span class="cit-name-given-names">E. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span>â<span class="cit-article-title">Genetic design and statistical power of nested association mapping in maize</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">178</span>: <span class="cit-fpage">539</span>â<span class="cit-lpage">551</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D178%26rft.spage%253D539%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.074245%26rft_id%253Dinfo%253Apmid%252F18202393%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/genetics.107.074245&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18202393&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F10%2F1179.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000252742200042&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">âµ</a><div class="cit ref-cit ref-journal" id="cit-2.10.1179.25" data-doi="10.1111/j.1467-9868.2005.00532.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yuan</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span>â<span class="cit-article-title">Model selection and estimation in regression with grouped varaible</span>. <abbr class="cit-jnl-abbrev">J. R. Stat. Soc., B</abbr> <span class="cit-vol">68</span>: <span class="cit-fpage">49</span>â<span class="cit-lpage">67</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BR.%2BStat.%2BSoc.%252C%2BB%26rft.volume%253D68%26rft.spage%253D49%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1467-9868.2005.00532.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1467-9868.2005.00532.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">âµ</a><div class="cit ref-cit ref-journal" id="cit-2.10.1179.26" data-doi="10.1214/09-EJS354"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">Penalized orthogonal-components regression for large p small n data</span>. <abbr class="cit-jnl-abbrev">Electronic Journal of Statistics</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">781</span>â<span class="cit-lpage">796</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DElectronic%2BJournal%2Bof%2BStatistics%26rft.volume%253D3%26rft.spage%253D781%26rft_id%253Dinfo%253Adoi%252F10.1214%252F09-EJS354%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1214/09-EJS354&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">âµ</a><div class="cit ref-cit ref-journal" id="cit-2.10.1179.27" data-doi="10.1534/genetics.108.098277"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhong</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dekkers</span>  <span class="cit-name-given-names">J. C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">Factors affecting accuracy from genomic selection in populaitons derived from multiple inbred lines: a barley case study</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">182</span>: <span class="cit-fpage">355</span>â<span class="cit-lpage">364</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D182%26rft.spage%253D355%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.108.098277%26rft_id%253Dinfo%253Apmid%252F19299342%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/genetics.108.098277&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19299342&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F10%2F1179.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000270213800030&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">âµ</a><div class="cit ref-cit ref-journal" id="cit-2.10.1179.28" data-doi="10.1111/j.1467-9868.2005.00503.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zou</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hastie</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span>â<span class="cit-article-title">Regularization and variable selection via the elastic net</span>. <abbr class="cit-jnl-abbrev">J. R. Stat. Soc., B</abbr> <span class="cit-vol">67</span>: <span class="cit-fpage">301</span>â<span class="cit-lpage">320</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BR.%2BStat.%2BSoc.%252C%2BB%26rft.volume%253D67%26rft.spage%253D301%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1467-9868.2005.00503.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1467-9868.2005.00503.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/2/10/1179.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/2/10/1169" title="Identification of Genes Underlying Hypoxia Tolerance in Drosophila by a P-element Screen" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/2/10/1185" title="Multiple Pathways Regulate Minisatellite Stability During Stationary Phase in Yeast" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/2/10"><span>Volume 2 Issue 10, October 2012</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/2/10.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/2/10.cover-source.jpg?itok=qNC6A40M" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 2 (10)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/2/10/1179" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Rl0VfGPFk5Jkc4HYy8gTn_cHyu5UyUvGLS4IJ5Fzr-o" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/genomic-selection" class="" data-icon-position="" data-hide-link-title="0">Genomic Selection</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/2/10/1179" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/2/10/1179" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/2/10/1179" class="active" data-icon-position="" data-hide-link-title="0">A Fast and Efficient Approach for Genomic Selection with High-Density Markers</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459336" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-M25EPwfBWYewLDVe6Tw6AGT_f57XPedVznA2pi8PlZs" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/2/10/1179.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/2/10/1179" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-ywgYn0BBfQD070JmQrQfe2jhnNarikvOSg0bECyZyos" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/2/10/1179" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/2/10/1179.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459336" id="node459336--2" data-pisa="ggg;2/10/1179" data-pisa-master="ggg;2/10/1179" data-apath="/ggg/2/10/1179.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Fast and Efficient Approach for Genomic Selection with High-Density Markers</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Vitara Pungpapong</span>, <span class="highwire-citation-author" data-delta="1">William M. Muir</span>, <span class="highwire-citation-author" data-delta="2">Xianran Li</span>, <span class="highwire-citation-author" data-delta="3">Dabao Zhang</span> and <span class="highwire-citation-author" data-delta="4">Min Zhang</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2012 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 2 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1179-1184;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.003822 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459336--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Vitara Pungpapong </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Statistics, Purdue University, West Lafayette, Indiana 47907</div><div class='author-affiliation'><span class='nlm-sup'>â </span>Department of Statistics, Chulalongkorn Business School, Bangkok, Thailand 10330</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Vitara%2BPungpapong%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pungpapong%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AVitara%2BPungpapong%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">William M. Muir </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â¡</span>Department of Animal Science, Purdue University, West Lafayette, Indiana 47907</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=William%2BM.%2BMuir%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Muir%20WM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AWilliam%2BM.%2BMuir%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Xianran Li </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>Â§</span>Department of Agronomy, Kansas State University, Manhattan, Kansas 66506, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xianran%2BLi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Li%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AXianran%2BLi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Dabao Zhang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Statistics, Purdue University, West Lafayette, Indiana 47907</div><div class='author-affiliation'><span class='nlm-sup'>**</span>School of Biomedical Engineering, Capital Medical University, Beijing, China 100069</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dabao%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADabao%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:zhangdb@stat.purdue.edu" class="" data-icon-position="" data-hide-link-title="0">zhangdb@stat.purdue.edu</a>
<a href="mailto:minzhang@stat.purdue.edu" class="" data-icon-position="" data-hide-link-title="0">minzhang@stat.purdue.edu</a></span></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Min Zhang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Statistics, Purdue University, West Lafayette, Indiana 47907</div><div class='author-affiliation'><span class='nlm-sup'>**</span>School of Biomedical Engineering, Capital Medical University, Beijing, China 100069</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Min%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMin%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:zhangdb@stat.purdue.edu" class="" data-icon-position="" data-hide-link-title="0">zhangdb@stat.purdue.edu</a>
<a href="mailto:minzhang@stat.purdue.edu" class="" data-icon-position="" data-hide-link-title="0">minzhang@stat.purdue.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/2/10/1179" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/2/10/1179&amp;title=A%20Fast%20and%20Efficient%20Approach%20for%20Genomic%20Selection%20with%20High-Density%20Markers" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/2/10/1179&amp;title=A%20Fast%20and%20Efficient%20Approach%20for%20Genomic%20Selection%20with%20High-Density%20Markers" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/2/10/1179&amp;title=A%20Fast%20and%20Efficient%20Approach%20for%20Genomic%20Selection%20with%20High-Density%20Markers" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/2/10/1179&amp;text=A%20Fast%20and%20Efficient%20Approach%20for%20Genomic%20Selection%20with%20High-Density%20Markers" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/2/10/1179&amp;title=A%20Fast%20and%20Efficient%20Approach%20for%20Genomic%20Selection%20with%20High-Density%20Markers" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/2/10/1179&amp;t=A%20Fast%20and%20Efficient%20Approach%20for%20Genomic%20Selection%20with%20High-Density%20Markers" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/2/10/1179&amp;title=A%20Fast%20and%20Efficient%20Approach%20for%20Genomic%20Selection%20with%20High-Density%20Markers" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/2/10/1179&amp;title=A%20Fast%20and%20Efficient%20Approach%20for%20Genomic%20Selection%20with%20High-Density%20Markers" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459336/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459336" id="node459336--3" data-pisa="ggg;2/10/1179" data-pisa-master="ggg;2/10/1179" data-apath="/ggg/2/10/1179.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Fast and Efficient Approach for Genomic Selection with High-Density Markers</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Vitara Pungpapong</span>, <span class="highwire-citation-author" data-delta="1">William M. Muir</span>, <span class="highwire-citation-author" data-delta="2">Xianran Li</span>, <span class="highwire-citation-author" data-delta="3">Dabao Zhang</span> and <span class="highwire-citation-author" data-delta="4">Min Zhang</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2012 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 2 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1179-1184;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.003822 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459336--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Vitara Pungpapong </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Statistics, Purdue University, West Lafayette, Indiana 47907</div><div class='author-affiliation'><span class='nlm-sup'>â </span>Department of Statistics, Chulalongkorn Business School, Bangkok, Thailand 10330</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Vitara%2BPungpapong%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pungpapong%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AVitara%2BPungpapong%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">William M. Muir </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â¡</span>Department of Animal Science, Purdue University, West Lafayette, Indiana 47907</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=William%2BM.%2BMuir%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Muir%20WM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AWilliam%2BM.%2BMuir%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Xianran Li </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>Â§</span>Department of Agronomy, Kansas State University, Manhattan, Kansas 66506, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xianran%2BLi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Li%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AXianran%2BLi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Dabao Zhang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Statistics, Purdue University, West Lafayette, Indiana 47907</div><div class='author-affiliation'><span class='nlm-sup'>**</span>School of Biomedical Engineering, Capital Medical University, Beijing, China 100069</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dabao%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADabao%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:zhangdb@stat.purdue.edu" class="" data-icon-position="" data-hide-link-title="0">zhangdb@stat.purdue.edu</a>
<a href="mailto:minzhang@stat.purdue.edu" class="" data-icon-position="" data-hide-link-title="0">minzhang@stat.purdue.edu</a></span></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Min Zhang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Statistics, Purdue University, West Lafayette, Indiana 47907</div><div class='author-affiliation'><span class='nlm-sup'>**</span>School of Biomedical Engineering, Capital Medical University, Beijing, China 100069</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Min%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMin%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:zhangdb@stat.purdue.edu" class="" data-icon-position="" data-hide-link-title="0">zhangdb@stat.purdue.edu</a>
<a href="mailto:minzhang@stat.purdue.edu" class="" data-icon-position="" data-hide-link-title="0">minzhang@stat.purdue.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1179&amp;rft.epage=1184&amp;rft.atitle=A%20Fast%20and%20Efficient%20Approach%20for%20Genomic%20Selection%20with%20High-Density%20Markers&amp;rft.volume=2&amp;rft.issue=10&amp;rft.date=2012-10-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Pungpapong%2C+Vitara&amp;rft.au=Muir%2C+William+M.&amp;rft.au=Li%2C+Xianran&amp;rft.au=Zhang%2C+Dabao&amp;rft.au=Zhang%2C+Min"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459336/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459336/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459336/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459336/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459336/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459336/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459336/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459336/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459336/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459336/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459336/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459336/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490720" id="node490720" data-pisa="ggg;8/9/3019" data-pisa-master="ggg;g3.118.200430" data-apath="/ggg/8/9/3019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/3019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">A Bayesian Decision Theory Approach for Genomic Selection</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490724" id="node490724" data-pisa="ggg;8/9/2889" data-pisa-master="ggg;g3.118.200311" data-apath="/ggg/8/9/2889.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/2889" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Optimising Genomic Selection in Wheat: Effect of Marker Density, Population Size and Population Structure on Prediction Accuracy</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490727" id="node490727" data-pisa="ggg;8/9/3039" data-pisa-master="ggg;g3.118.200435" data-apath="/ggg/8/9/3039.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/3039" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">BGGE: A New Package for Genomic-Enabled Prediction Incorporating Genotype Ã Environment Interaction Models</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Genomic%20Selection" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Genomic Selection</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/2/10/1179" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Methods</a></li><li class="odd"><a href="#sec-5" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-11" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/2/10/1179.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/2/10/1179.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/2/10/1179.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/2/10/1179.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__CK8qNg7zHnCPfipcSU0qjCyIiw2pe-WKGx2V0TgizGw__wprDSLLH9CuacxJkxX2DuWt0EaauZA-f_LGjovfY3aw__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
