<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/8/1/219" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/8/1/219.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/8/1/219.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/8/1/219.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/8/1/219.atom" />
<meta name="HW.pisa" content="ggg;8/1/219" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Comparison of ChIP-Seq Data and a Reference Motif Set for Human KRAB C2H2 Zinc Finger Proteins" />
<meta name="DC.Identifier" content="10.1534/g3.117.300296" />
<meta name="DC.Date" content="2018-01-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2018 Barazandeh et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="KRAB C2H2 zinc finger proteins (KZNFs) are the largest and most diverse family of human transcription factors, likely due to diversifying selection driven by novel endogenous retroelements (EREs), but the vast majority lack binding motifs or functional data. Two recent studies analyzed a majority of the human KZNFs using either ChIP-seq (60 proteins) or ChIP-exo (221 proteins) in the same cell type (HEK293). The ChIP-exo paper did not describe binding motifs, however. Thirty-nine proteins are represented in both studies, enabling the systematic comparison of the data sets presented here. Typically, only a minority of peaks overlap, but the two studies nonetheless display significant similarity in ERE binding for 32/39, and yield highly similar DNA binding motifs for 23 and related motifs for 34 (MoSBAT similarity score &gt;0.5 and &gt;0.2, respectively). Thus, there is overall (albeit imperfect) agreement between the two studies. For the 242 proteins represented in at least one study, we selected a highest-confidence motif for each protein, utilizing several motif-derivation approaches, and evaluating motifs within and across data sets. Peaks for the majority (158) are enriched (96% with AUC &gt;0.6 predicting peak vs. nonpeak) for a motif that is supported by the C2H2 “recognition code,” consistent with intrinsic sequence specificity driving DNA binding in cells. An additional 63 yield motifs enriched in peaks, but not supported by the recognition code, which could reflect indirect binding. Altogether, these analyses validate both data sets, and provide a reference motif set with associated quality metrics." />
<meta name="DC.Contributor" content="Marjan Barazandeh" />
<meta name="DC.Contributor" content="Samuel A. Lambert" />
<meta name="DC.Contributor" content="Mihai Albu" />
<meta name="DC.Contributor" content="Timothy R. Hughes" />
<meta name="article:published_time" content="2018-01-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Comparison of ChIP-Seq Data and a Reference Motif Set for Human KRAB C2H2 Zinc Finger Proteins" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;KRAB C2H2 zinc finger proteins (KZNFs) are the largest and most diverse family of human transcription factors, likely due to diversifying selection driven by novel endogenous retroelements (EREs), but the vast majority lack binding motifs or functional data. Two recent studies analyzed a majority of the human KZNFs using either ChIP-seq (60 proteins) or ChIP-exo (221 proteins) in the same cell type (HEK293). The ChIP-exo paper did not describe binding motifs, however. Thirty-nine proteins are represented in both studies, enabling the systematic comparison of the data sets presented here. Typically, only a minority of peaks overlap, but the two studies nonetheless display significant similarity in ERE binding for 32/39, and yield highly similar DNA binding motifs for 23 and related motifs for 34 (MoSBAT similarity score &amp;gt;0.5 and &amp;gt;0.2, respectively). Thus, there is overall (albeit imperfect) agreement between the two studies. For the 242 proteins represented in at least one study, we selected a highest-confidence motif for each protein, utilizing several motif-derivation approaches, and evaluating motifs within and across data sets. Peaks for the majority (158) are enriched (96% with AUC &amp;gt;0.6 predicting peak &lt;i&gt;vs.&lt;/i&gt; nonpeak) for a motif that is supported by the C2H2 “recognition code,” consistent with intrinsic sequence specificity driving DNA binding in cells. An additional 63 yield motifs enriched in peaks, but not supported by the recognition code, which could reflect indirect binding. Altogether, these analyses validate both data sets, and provide a reference motif set with associated quality metrics.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2018/01/01" />
<meta name="citation_mjid" content="ggg;8/1/219" />
<meta name="citation_id" content="8/1/219" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/8/1/219" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/8/1/219.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/8/1/219.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/8/1/219.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.117.300296" />
<meta name="citation_pmid" content="29146583" />
<meta name="citation_volume" content="8" />
<meta name="citation_issue" content="1" />
<meta name="citation_num_pages" content="11" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="219" />
<meta name="citation_lastpage" content="229" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Marjan Barazandeh" />
<meta name="citation_author_institution" content="Terrence Donnelly Centre for Cellular and Biomolecular Research, University of Toronto, Ontario M5S 1A8, Canada" />
<meta name="citation_author" content="Samuel A. Lambert" />
<meta name="citation_author_institution" content="Department of Molecular Genetics, University of Toronto, Ontario M5S 1A8, Canada" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-8222-008X" />
<meta name="citation_author" content="Mihai Albu" />
<meta name="citation_author_institution" content="Terrence Donnelly Centre for Cellular and Biomolecular Research, University of Toronto, Ontario M5S 1A8, Canada" />
<meta name="citation_author" content="Timothy R. Hughes" />
<meta name="citation_author_institution" content="Terrence Donnelly Centre for Cellular and Biomolecular Research, University of Toronto, Ontario M5S 1A8, Canada" />
<meta name="citation_author_institution" content="Department of Molecular Genetics, University of Toronto, Ontario M5S 1A8, Canada" />
<meta name="citation_author_institution" content="Canadian Institutes for Advanced Research, Toronto, Ontario M5G 1M1, Canada" />
<meta name="citation_author_email" content="t.hughes@utoronto.ca" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-8721-4719" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=G. Badis;citation_author=M. F. Berger;citation_author=A. A. Philippakis;citation_author=S. Talukder;citation_author=A. R. Gehrke;citation_title=Diversity and complexity in DNA recognition by transcription factors.;citation_volume=324;citation_year=2009;citation_pmid=19443739;citation_doi=10.1126/science.1162327" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=T. L. Bailey;citation_author=M. Boden;citation_author=F. A. Buske;citation_author=M. Frith;citation_author=C. E. Grant;citation_title=MEME SUITE: tools for motif discovery and searching.;citation_volume=37;citation_year=2009;citation_pmid=19458158;citation_doi=10.1093/nar/gkp335" />
<meta name="citation_reference" content="citation_journal_title=Cell biochemistry and biophysics;citation_journal_abbrev=Cell Biochem Biophys;citation_author=K. J. Brayer;citation_author=D. J. Segal;citation_title=Keep your fingers off my DNA: protein-protein interactions mediated by C2H2 zinc finger domains.;citation_pages=111-131;citation_volume=50;citation_year=2008;citation_issue=3;citation_pmid=18253864;citation_doi=10.1007/s12013-008-9008-5" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=D. S. Day;citation_author=L. J. Luquette;citation_author=P. J. Park;citation_author=P. V. Kharchenko;citation_title=Estimating enrichment of repetitive elements from high-throughput sequence data.;citation_pages=R69-R69;citation_volume=11;citation_year=2010;citation_issue=6;citation_pmid=20584328;citation_doi=10.1186/gb-2010-11-6-r69" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_author=B. Deplancke;citation_author=D. Alpern;citation_author=V. Gardeux;citation_title=The genetics of transcription factor DNA binding variation.;citation_pages=538-554;citation_volume=166;citation_year=2016;citation_pmid=27471964;citation_doi=10.1016/j.cell.2016.07.012" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=R. O. Emerson;citation_author=J. H. Thomas;citation_title=Adaptive evolution in zinc finger transcription factors.;citation_pages=e1000325-e1000325;citation_volume=5;citation_year=2009;citation_issue=1;citation_pmid=19119423;citation_doi=10.1371/journal.pgen.1000325" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_title=An integrated encyclopedia of DNA elements in the human genome.;citation_pages=57-74;citation_volume=489;citation_year=2012;citation_issue=7414;citation_pmid=22955616;citation_doi=10.1038/nature11247" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=M. Imbeault;citation_author=P. Y. Helleboid;citation_author=D. Trono;citation_title=KRAB zinc-finger proteins contribute to the evolution of gene regulatory networks.;citation_pages=550-554;citation_volume=543;citation_year=2017;citation_pmid=28273063;citation_doi=10.1038/nature21683" />
<meta name="citation_reference" content="citation_journal_title=Nat. Methods;citation_author=A. Isakova;citation_author=R. Groux;citation_author=M. Imbeault;citation_author=P. Rainer;citation_author=D. Alpern;citation_title=SMiLE-seq identifies binding motifs of single and dimeric transcription factors.;citation_pages=316-322;citation_volume=14;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=F. M. J. Jacobs;citation_author=D. Greenberg;citation_author=N. Nguyen;citation_author=M. Haeussler;citation_author=A. D. Ewing;citation_title=An evolutionary arms race between KRAB zinc-finger genes ZNF91/93 and SVA/L1 retrotransposons.;citation_pages=242-245;citation_volume=516;citation_year=2014;citation_pmid=25274305;citation_doi=10.1038/nature13760" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_author=A. Jolma;citation_author=J. Yan;citation_author=T. Whitington;citation_author=J. Toivonen;citation_author=K. R. Nitta;citation_title=DNA-binding specificities of human transcription factors.;citation_pages=327-339;citation_volume=152;citation_year=2013;citation_pmid=23332764;citation_doi=10.1016/j.cell.2012.12.009" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=A. Jolma;citation_author=Y. Yin;citation_author=K. R. Nitta;citation_author=K. Dave;citation_author=A. Popov;citation_title=DNA-dependent formation of transcription factor pairs alters their binding specificity.;citation_pages=384-388;citation_volume=527;citation_year=2015;citation_pmid=26550823;citation_doi=10.1038/nature15518" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=D. Karolchik;citation_author=A. S. Hinrichs;citation_author=T. S. Furey;citation_author=K. M. Roskin;citation_author=C. W. Sugnet;citation_title=The UCSC Table Browser data retrieval tool;citation_pages=D493-496;citation_volume=32;citation_year=2004;citation_issue=90001;citation_pmid=14681465;citation_doi=10.1093/nar/gkh103" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=P. V. Kharchenko;citation_author=M. Y. Tolstorukov;citation_author=P. J. Park;citation_title=Design and analysis of ChIP-seq experiments for DNA-binding proteins.;citation_pages=1351-1359;citation_volume=26;citation_year=2008;citation_issue=12;citation_pmid=19029915;citation_doi=10.1038/nbt.1508" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=I. V. Kulakovskiy;citation_author=Y. A. Medvedeva;citation_author=U. Schaefer;citation_author=A. S. Kasianov;citation_author=I. E. Vorontsov;citation_title=HOCOMOCO: a comprehensive collection of human transcription factor binding sites models.;citation_volume=41;citation_year=2013;citation_pmid=23175603;citation_doi=10.1093/nar/gks1089" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=S. A. Lambert;citation_author=M. Albu;citation_author=T. R. Hughes;citation_author=H. S. Najafabadi;citation_title=Motif comparison based on similarity of binding affinity profiles.;citation_volume=32;citation_year=2016;citation_pmid=27466627;citation_doi=10.1093/bioinformatics/btw489" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=S. G. Landt;citation_author=G. K. Marinov;citation_author=A. Kundaje;citation_author=P. Kheradpour;citation_author=F. Pauli;citation_title=ChIP-seq guidelines and practices of the ENCODE and modENCODE consortia;citation_pages=1813-1831;citation_volume=22;citation_year=2012;citation_issue=9;citation_pmid=22955991;citation_doi=10.1101/gr.136184.111" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=B. Langmead;citation_author=S. L. Salzberg;citation_title=Fast gapped-read alignment with Bowtie 2.;citation_pages=357-359;citation_volume=9;citation_year=2012;citation_issue=4;citation_pmid=22388286;citation_doi=10.1038/nmeth.1923" />
<meta name="citation_reference" content="citation_journal_title=PLOS Comput. Biol.;citation_author=A. Mathelier;citation_author=W. W. Wasserman;citation_title=The next generation of transcription factor binding site prediction.;citation_pages=e1003214;citation_volume=9;citation_year=2013;citation_pmid=24039567;citation_doi=10.1371/journal.pcbi.1003214" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=A. Mathelier;citation_author=O. Fornes;citation_author=D. J. Arenillas;citation_author=C. Y. Chen;citation_author=G. Denay;citation_title=JASPAR 2016: a major expansion and update of the open-access database of transcription factor binding profiles.;citation_volume=44;citation_year=2016;citation_pmid=26531826;citation_doi=10.1093/nar/gkv1176" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=T. Matsui;citation_author=D. Leung;citation_author=H. Miyashita;citation_author=I. A. Maksakova;citation_author=H. Miyachi;citation_title=Proviral silencing in embryonic stem cells requires the histone methyltransferase ESET.;citation_pages=927-931;citation_volume=464;citation_year=2010;citation_issue=7290;citation_pmid=20164836;citation_doi=10.1038/nature08858" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=V. Matys;citation_author=O. V. Kel-Margoulis;citation_author=E. Fricke;citation_author=I. Liebich;citation_author=S. Land;citation_title=TRANSFAC(R) and its module TRANSCompel(R): transcriptional gene regulation in eukaryotes;citation_pages=D108-D110;citation_volume=34;citation_year=2006;citation_issue=suppl_1;citation_pmid=16381825;citation_doi=10.1093/nar/gkj143" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. S. Najafabadi;citation_author=M. Albu;citation_author=T. R. Hughes;citation_title=Identification of C2H2-ZF binding preferences from ChIP-seq data using RCADE.;citation_volume=31;citation_year=2015;citation_pmid=25953800;citation_doi=10.1093/bioinformatics/btv284" />
<meta name="citation_reference" content="citation_journal_title=Nat. Biotechnol.;citation_author=H. S. Najafabadi;citation_author=S. Mnaimneh;citation_author=F. W. Schmitges;citation_author=M. Garton;citation_author=K. N. Lam;citation_title=C2H2 zinc finger proteins greatly expand the human regulatory lexicon.;citation_pages=555-562;citation_volume=33;citation_year=2015;citation_pmid=25690854;citation_doi=10.1038/nbt.3128" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=M. B. Noyes;citation_author=X. Meng;citation_author=A. Wakabayashi;citation_author=S. Sinha;citation_author=M. H. Brodsky;citation_title=A systematic characterization of factors that regulate Drosophila segmentation via a bacterial one-hybrid system.;citation_volume=36;citation_year=2008;citation_pmid=18332042;citation_doi=10.1093/nar/gkn048" />
<meta name="citation_reference" content="citation_journal_title=Virology;citation_author=H. M. Rowe;citation_author=D. Trono;citation_title=Dynamic control of endogenous retroviruses during development.;citation_pages=273-287;citation_volume=411;citation_year=2011;citation_pmid=21251689;citation_doi=10.1016/j.virol.2010.12.007" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=F. W. Schmitges;citation_author=E. Radovani;citation_author=H. S. Najafabadi;citation_author=M. Barazandeh;citation_author=L. F. Campitelli;citation_title=Multiparameter functional diversity of human C2H2 zinc finger proteins.;citation_volume=26;citation_year=2016;citation_pmid=27852650;citation_doi=10.1101/gr.209643.116" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=D. C. Schultz;citation_author=K. Ayyanathan;citation_author=D. Negorev;citation_author=G. G. Maul;citation_author=F. J. Rauscher;citation_title=SETDB1: a novel KAP-1-associated histone H3, lysine 9-specific methyltransferase that contributes to HP1-mediated silencing of euchromatic genes by KRAB zinc-finger proteins;citation_pages=919-932;citation_volume=16;citation_year=2002;citation_issue=8;citation_pmid=11959841;citation_doi=10.1101/gad.973302" />
<meta name="citation_reference" content="citation_journal_title=BMC Bioinformatics;citation_author=J. Wang;citation_title=Quality vs. accuracy: result of a reanalysis of protein-binding microarrays from the DREAM5 challenge by using BayesPI2 including dinucleotide interdependence.;citation_pages=289;citation_volume=15;citation_year=2014;citation_pmid=25158938;citation_doi=10.1186/1471-2105-15-289" />
<meta name="citation_reference" content="citation_journal_title=Nat. Biotechnol.;citation_author=M. T. Weirauch;citation_author=A. Cote;citation_author=R. Norel;citation_author=M. Annala;citation_author=Y. Zhao;citation_title=Evaluation of methods for modeling transcription factor sequence specificity.;citation_pages=126-134;citation_volume=31;citation_year=2013;citation_pmid=23354101;citation_doi=10.1038/nbt.2486" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_author=M. T. Weirauch;citation_author=A. Yang;citation_author=M. Albu;citation_author=A. G. Cote;citation_author=A. Montenegro-Montero;citation_title=Determination and inference of eukaryotic transcription factor sequence specificity.;citation_pages=1431-1443;citation_volume=158;citation_year=2014;citation_pmid=25215497;citation_doi=10.1016/j.cell.2014.08.009" />
<meta name="citation_reference" content="citation_journal_title=Annual review of biophysics and biomolecular structure;citation_journal_abbrev=Annu Rev Biophys Biomol Struct;citation_author=S. A. Wolfe;citation_author=L. Nekludova;citation_author=C. O. Pabo;citation_title=DNA recognition by Cys2His2 zinc finger proteins.;citation_pages=183-212;citation_volume=29;citation_year=2000;citation_pmid=10940247;citation_doi=10.1146/annurev.biophys.29.1.183" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_title=Impact of cytosine methylation on DNA binding specificities of human transcription factors.;citation_author=Y. Yin;citation_author=E. Morgunova;citation_author=A. Jolma;citation_author=E. Kaasinen;citation_author=B. Sahu;citation_volume=356;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=Y. Zhang;citation_author=T. Liu;citation_author=C. A. Meyer;citation_author=J. Eeckhoute;citation_author=D. S. Johnson;citation_title=Model-based analysis of ChIP-Seq (MACS).;citation_pages=R137-R137;citation_volume=9;citation_year=2008;citation_issue=9;citation_pmid=18798982;citation_doi=10.1186/gb-2008-9-9-r137" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Comput Biol;citation_author=Y. Zhao;citation_author=D. Granas;citation_author=G. D. Stormo;citation_title=Inferring binding energies from selected binding sites.;citation_pages=e1000590-e1000590;citation_volume=5;citation_year=2009;citation_issue=12;citation_pmid=19997485;citation_doi=10.1371/journal.pcbi.1000590" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Comparison of ChIP-Seq Data and a Reference Motif Set for Human KRAB C2H2 Zinc Finger Proteins" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1.cover-source.jpg" />
<meta name="twitter:description" content="KRAB C2H2 zinc finger proteins (KZNFs) are the largest and most diverse family of human transcription factors, likely due to diversifying selection driven by novel endogenous retroelements (EREs), but the vast majority lack binding motifs or functional data. Two recent studies analyzed a majority of the human KZNFs using either ChIP-seq (60 proteins) or ChIP-exo (221 proteins) in the same cell type (HEK293). The ChIP-exo paper did not describe binding motifs, however. Thirty-nine proteins are represented in both studies, enabling the systematic comparison of the data sets presented here. Typically, only a minority of peaks overlap, but the two studies nonetheless display significant similarity in ERE binding for 32/39, and yield highly similar DNA binding motifs for 23 and related motifs for 34 (MoSBAT similarity score &gt;0.5 and &gt;0.2, respectively). Thus, there is overall (albeit imperfect) agreement between the two studies. For the 242 proteins represented in at least one study, we selected a highest-confidence motif for each protein, utilizing several motif-derivation approaches, and evaluating motifs within and across data sets. Peaks for the majority (158) are enriched (96% with AUC &gt;0.6 predicting peak vs. nonpeak) for a motif that is supported by the C2H2 “recognition code,” consistent with intrinsic sequence specificity driving DNA binding in cells. An additional 63 yield motifs enriched in peaks, but not supported by the recognition code, which could reflect indirect binding. Altogether, these analyses validate both data sets, and provide a reference motif set with associated quality metrics." />
<meta name="og-title" property="og:title" content="Comparison of ChIP-Seq Data and a Reference Motif Set for Human KRAB C2H2 Zinc Finger Proteins" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/8/1/219" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="KRAB C2H2 zinc finger proteins (KZNFs) are the largest and most diverse family of human transcription factors, likely due to diversifying selection driven by novel endogenous retroelements (EREs), but the vast majority lack binding motifs or functional data. Two recent studies analyzed a majority of the human KZNFs using either ChIP-seq (60 proteins) or ChIP-exo (221 proteins) in the same cell type (HEK293). The ChIP-exo paper did not describe binding motifs, however. Thirty-nine proteins are represented in both studies, enabling the systematic comparison of the data sets presented here. Typically, only a minority of peaks overlap, but the two studies nonetheless display significant similarity in ERE binding for 32/39, and yield highly similar DNA binding motifs for 23 and related motifs for 34 (MoSBAT similarity score &gt;0.5 and &gt;0.2, respectively). Thus, there is overall (albeit imperfect) agreement between the two studies. For the 242 proteins represented in at least one study, we selected a highest-confidence motif for each protein, utilizing several motif-derivation approaches, and evaluating motifs within and across data sets. Peaks for the majority (158) are enriched (96% with AUC &gt;0.6 predicting peak vs. nonpeak) for a motif that is supported by the C2H2 “recognition code,” consistent with intrinsic sequence specificity driving DNA binding in cells. An additional 63 yield motifs enriched in peaks, but not supported by the recognition code, which could reflect indirect binding. Altogether, these analyses validate both data sets, and provide a reference motif set with associated quality metrics." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1.cover-source.jpg" />
    <title>Comparison of ChIP-Seq Data and a Reference Motif Set for Human KRAB C2H2 Zinc Finger Proteins | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"9PY8cHV6e9zWkm1MBH1pQvnDYzkFS6jTi2vhcokLOV0","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"487830","apath":"\/ggg\/8\/1\/219.atom","pisa":"ggg;8\/1\/219","ac":{"\/ggg\/8\/1\/219.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/8\/1\/219.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;8\/1\/219"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;8\/1\/219"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/8\/1\/219":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022U\\\\\\\/8Nto\\\\\\\/AHoMVZXTbMx8KAIthbjMskth7BrWcY6NjX6iSAPy4hQWkkPpJeofdzKDHsLHcvLktUBRwYi\\\\\\\/DHKGUmWToCjqVfuABHIMfiUrpKYkZUYeKEvxii8SSykcAMaowKL3KR7j\\\\\\\/zseUkdP65qNWlZ+tAFmEeY8RKp13cnNtP4xaMzr4gT3WUl+Frv+p7yKTGfo\\\\\\\/QgCvYbzyWacgZz\\\\\\\/zgMRpMcb02UqgzhFbB9\\\\\\\/RSHb3p+vch4eP2qQnZ7kcnmaHVm0ymG8tQm8nvvKxALqd2yl68QjBbNPQq36qSrqslSiVfv9RgoiK38YZxxMaCYezdmSTeb0NHnmRCl0g\\\\\\\/TDqE05KmCKItYYpjFGVHuMoxeyI16\\\\\\\/oOZq8P5CxJ\\\\\\\/rkqZxKpAicLrcDQ9swKShr9NN+iWF9\\\\\\\/fibIEVSEOAm2U+9DLc3QbxrM1beIUOjCZWlL2fUOLlfz0tV1sAW4PwISfmxbVV9bDRpnPDf3aArU4e3qx33rrt65gtp\\\\\\\/pQfScLJm5vNOOlTfclTDw2\\\\\\\/paJXyJxI6N6H4ls8hHUwYCJ0XSNg7IEUzTBsvkLpqADJo9Nspg+OOUirPzeAJAMJiB6RvAg4oXyNAI7\\\\\\\/xcxsvPfpohKCADbImOUp3dhBi2YffD4a3Nq9OPBU8Y7D3GuvCJ9aNdkkvZjuKTY8BQvbEqvLpcUt2zUinzj+n03Jfke6T\\\\\\\/8Y6tli8xLrJyLW2EA6CPijVR8d0hh1eDozNsXIwyElh+DVdqrArxx1rZQ\\\\\\\/I6w2\\\\\\\/5yVYqtvx2n3xG1vYMPWBtR8jky42i\\\\\\\/pLWc\\\\\\\/ZsE9\\\\\\\/hUC+q6692c+P4pvKfEHdRde8P3mNfrU6gnjMQpm0N4AO8+3vA0pJ00eqUmsH5T8gr5uawXDM8d9HhKr93Va9HkiiAmiKImjLrKg+vUE0kji8aLPFW+I2Y81DuQja2JocIavn0WFAllUP0bsRLWoqBei9zrIEyBWAvH2nf7QzE+JYnwD2TNXE82HD+E4D75nnQ2cenwQIZkq1f3zViPnhp8Cto9x+xyoPQh\\\\\\\/Kh5Pvodm34ieTTPPOnXAXX8i0GkvTw0HxyCpWmHLMJMD6Q2x7jzW1JFfi4+HIuC1QSDPaLosBeulsOw8bQg2iMp9s9kMSLS4juQT7ngeYF6zMwixwtejBgRhQZc2kpGaEXMcEOznUAefmBuuAQRIkDWABpSMLKjS0+sxyfCMxMdT4+3VQX0LqhneKRS66utLbEsOW54h\\\\\\\/b0+XXcBdF6cVFEhroeYq+77iHnoavj\\\\\\\/oCPHLajl+pGgJ0zBI1DH\\\u0022,\\\u0022iv\\\u0022:\\\u0022R\\\\\\\/lvsssyYf\\\\\\\/mZe\\\\\\\/O8S+rTg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022ddbe290746d4948056f1f758afe6f044745022bef093faa47ef6793e7ded49c8\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022gAYcMxb+Z8CTO0\\\\\\\/V\\\\\\\/Bm91vG6nJc+sGB1SDbwpCt8zU65biQ4yAnK4Y9Dsn7Rx0Whj5WgF4ukbWj4ntKm0d5Ea0UewQyi+AW01bN8TaQthkicouey6Br7ciJZEdxMXvsZAOEv5AzIbShpVH04dkS8j12wjAlIsPlxQ3G1tJioLAE6My9VPpBQErdjNS9MTXR6GZ0kKQc3tyPgwbAwv7KcXJ\\\\\\\/CvyVLFKQ8V2Rq74IgOxaKXEkDaVImx5hqFs1QXA21+3nZRldZnIhdYogosoZy8fG2YpQlpLcJ5x5NyY3nPCMg2MkKlOXHHYT7RZ6vzJI0YWw+VwTOoBxgsRpyNWtPSvT8YHrE9xgt9tqrBvzBsGR5R8PdFK+BzGnLV8khDFGRhHgS+rxyl9oY2vluWv\\\\\\\/KFPJnQkr1aLhrXSLXvHmrR8ONyVAiGZNIWLo988xWDpBs9tcQS1ZwQAHNNeQCS5EUwZ+taNwvtQBcN1Yy+u4SmDSHMIApRzZUStOMXW3vm5HiOnkdhPXFoFvNhL0d79SrsyT5GNbjrKPWPlsLbBa4OcY0XZb8b+deIwUBTsfFmGI9r\\\\\\\/crnujI\\\\\\\/IQ81Yv03CI9KEe\\\\\\\/Y7lrDRiCCSfFvWa4RB0jzbE4+OhoqOQujOzR31GcK73xVawp30XNSeGYjK0pIBAIadHCdbbKnhCp32dI29sVyRS8ymoXG7guHVofYQA42PF7\\\\\\\/8X2Mir7OhYY806Www2UvFpVhZxWxtJr5zAxyxYxxp\\\\\\\/RRPwybbM6nq3u0xPdl4eySLDgqU9X4Qbv29g9Fi2L5tWPzoy7FWRC\\\\\\\/rS\\\\\\\/XhKC+01HLfk1uej1vt+k1APWOUXnpGw4bqW7pF0qr7PvdsWNZLfjTpkhjYq7Yit1iDuuAsNjmFUYjzUZW9HSIedsEhhtIy8E5E+0RJjiYqE9R6NjYBE4Cb+PGoZ80bMmYHwh\\\\\\\/PbsFkmJMgHceHmgA\\\\\\\/ifCOOwdpxYoHcXladvHnIBhMIV25K76D4FPLxgiR2AKkCNjF2oEoehUSX2fgYay6LYU5VMXtsLtvvsKfdKon4Lh\\\\\\\/spSPC9goN+IkqLhL1lfBIq8oMY3EmvJNX1wXpO5CoQatFdg8IHkoUSoKAGW8HnOLfmVrRM5wDGBq\\\\\\\/XStdVj6BITTGsKhj6LAgKultf\\\\\\\/q3EYSmSKtLIEP++jJDOwHKq\\\\\\\/nrpxCWbCAf\\\\\\\/UVmipGAvekCt4Rdg\\\\\\\/TPQn+XCWXgWHe7U22k4RYQFHaEbInxqnkP8HyO+qoMGDsmpQ4tupSuBcXu76YPBM2VZuxo+hLQpWtltDZXrG0+vPWZwuAAQRa8Cx+wCazg9taKOnSQ8jnZc6LXzCvX7nIz6rcr15Y+ROV3qGybaycWT9O8esxxIXFWSBIdWog2TTgWtyL0o4+sw7MeBkaKufGtTBEkbqxMcGExJuehpx979HhJCYmNzFgdulfa4TEogtFCvyeJim1bxQX6BfyfG8tA0QlsDfSbVHrwckkXKse7Y5xLH04lkzGOOIfpmcC13QvZLhEvwpP0RPXH3qZdQVHdTTIWI6OAmHM23PQmgXl7xvsGXltdoKYsvfS3Y0fG9e46EBLVytoOwenpQrVz+m4rbEQYKqVBvUrCVCC2DRF\\\\\\\/1Ozl+69Xt1dJAOYmbLxR1GP7NlkANOrySmHzVT\\\\\\\/dbCQgqn9eXe8+Z47RHTrbIaE7YoEm10SG5H1cldKZjZBU0E8L1TAeTOH86avdz0QglDsWjNzJhQNyP5kX2EPuidnE52H1kBQ\\\\\\\/6Q3pwo29vTI4VgjGThyGQE3Vqp1gUOXeovcrNQbfM3k5p7gXYrSCd\\\\\\\/9V2bRh5jaH\\\\\\\/MIljxmR9v0xdaHIgTUwNlkdIFigw2ytwazD1ud6ZF3yxwrYjJ2CQ0ODXjffeuklJMtWQozQvDTGy6yPRlhXc5QW5mCWQafW6jZPwgUuUH+64WLLDIAY4zFFcqTxmZyJ67DqMnw0qpWurwTXY1wfCOnssYnEs2UAnoLZFXgXQMAbJ2nd4cvNn3S3Ujk4AlZ3Y+IukjsPutgrconcBonQ+\\\\\\\/Fx\\\\\\\/sE34puD+s50W5V5lKEufH2J25IZRuRGA\\\\\\\/m5jsOZD8v7joutmIacQxl5N+I\\\\\\\/W+F3JmMygYgNzgElsJAHMftvN6XFlf\\\\\\\/ArapieF\\\\\\\/fVmnt5CI+mg53T\\\\\\\/V12OQtG1WBzUBlHJIk3XIzIU5B2w8It6\\\\\\\/m15nR+AvLwwwFWaJGhVxOi3WFm98imuRmx0AK\\\\\\\/0fCGHkutUL4Q0NhMHUW6AMvEBsmNOQ8QAgMXHb05Ud0EQPmN5gjA4y\\\\\\\/5Rzgv294ZQMmoA9WlbCbBwaG7cyoAYqBC8UZ2rMJX6xD+nJa5YBc6DBV35DnzUvQwQroU+42QLRrFAg3YQOcYbPAbbOgvxM8n1EQ8hlB9Fz6giBVS5Oj5l1WNBzbtdcWdFUrzhr0ft9Ly1MZHJH+csWVL87oBb3A30bOhrHjzlWWRIH\\\\\\\/+fpTvzX6BuxAA+4wQeDOXQf1xb5Mm+yVM0csEjKh\\\\\\\/CtD90BdPaZ7BtmmMOt21bphtAlytBOEEBv7dXqGqtiligb+qm6sTTcj7nyGazqhCHBuCGGHh6V6RGCuqylQwiMuum042JUDZhHPJ5TkoK0EQgUodu8bx4qnMk+1Lqv4QQ0ZMjX3Rk5BfWb20FwX\\\\\\\/WjDS0iVPdtMVF4qOyQJBfsgO0ZdfVTu9aUkBTjF03StR2Uy8E31JMftoAmG3Kc+d45M\\\\\\\/yZo8TAROvaEAjZrEuDV\\\\\\\/wFKk4n8j4DACLrtTAnsef4x1yC5vn\\\\\\\/2Jhijlxe3ydKxQZuyK1TqLBs0PDnFKeFGtD3JLE5TF+M3DANDhnBPvXF+jW6Cmo8hWGV+YZrar1I9nYBCPSbqQ03aEnFOkvBAGUuxHlfT0Z7bOiq5dRqjoZoFLfwfrD6hYVWMyowiDFqdYFIUehu9iYHRa94Gkp84g340B2KulNpkH9lyzYyp0BkDiPgbtxmG5vCM2ru2TVjNpERuNfqLHvSO1N9Hu2w1IbYxYkRdA1zI0bDJmTsA2XWWXPJ3xHozuBLs3ZH0A1bCzTxpJuAFm4jnaD0zSwEuZZD6cQJ\\\\\\\/Uclf7v7QHtdwHMHv\\\\\\\/YmJ8rHCWliHxIvIq0NvBt1gy62\\\\\\\/Sn4Fc37Hy5BBIUgVEjY4myvAuut\\\\\\\/kzy4VlzbEQ\\\\\\\/JGthIBNGulP3\\\\\\\/+TBlNUSH6tVFVX5tyGQjvbFnJXvUb5C0WNxf8Lq5tYqYX7fTT5lAS9C+kQ9UQbnI38vZjhUXsNG00XZlRLtaHuknikLMPRoLpLNqWTuxFRh\\\\\\\/0zZ3bhTxILEIBU+9mJpZ2perkTYjziRFjH72oMVa4yOzWtMhuZIrhjo7GvhkHwq3sDdvO4G3WNf\\\\\\\/rtWGYpYhbgAr9SYSYoAjhu+hnz\\\\\\\/Ed1fRek6xs5gZVxBB15xhreD7MgSvlV4I7xMdWobMb1xft27yhCyYRKRbAge7q05RttYVWQi1Y76eMur2UiSmU0MGGefpnM8CD7fn9G58B9JgSTZFSW548Bg0JSOpE8GLp1UuYZOlXJUFlWzoAES9TklACXNJotvIC+rm62CTN2GqIeBMwwshaZFAybwccRPWAbvDpRfDexI1d0HYiGLfekf8LMe+yLfxqThiSdDKAJJ3p9OzGQMuRsFSC3JP96nvXTrVQmIqiHRJLrKUFVg4FrR1\\\\\\\/OvQqRorj4XpKI83Pnr4BkWvE8183pKtaZWV7kzNoPjAIV0MmMbfsxXfOkcEqrG1sCp6ZtuEAlBkPlzjFrqgPT05Zwowc80cuBmIif+fjmoQhwF6kPqkca7leIpyUI6RWUc1rTTGv6+yy8zse+IXdWAuWIvCQ\\\\\\\/K0X24+vOv3cPoB603vJM5RY7Jmz3T8fjTmFq4Llg2JItx9ooeNnCDcqZxrcAGiKU+ztV9kWTqhdcWOxS\\\\\\\/7Hu7cpMFVO0GjgknpkQw1B6SonKseP9HhC+4H7okJJuLRljj3MEQCu9IphJ+7IReKwnCmd\\\\\\\/TUYJbkXfwSoAo+c00nAoB+T0\\\\\\\/zI8JypeO9BypUyzOf9IKHkyFuWehbElB\\\\\\\/PV+\\\\\\\/qiy9LcxbaxiSI7njlWq\\\\\\\/VNG19NHfhoowdKKut2umTZ6ZUnj1pBlXjxs\\\\\\\/5YUYIepMajAdo+ut4IbnAyYNGoNeFAiMLej628oWIrQxSTf24+PijfmE0PPHcisNrct0WQ8pX4zEN7Ok8fCxYexix2\\\\\\\/0EWaS+iiGT1Y5YvLnYo\\\\\\\/B98xjh45FHEcG83XvDTY1rWD9e41V5iaLdChCqBV1Dpx3\\\\\\\/YroqZFo535uSmCbYJGW7T+iZGqotxgBklfm2XeHbjfge6I+d5VA36rEiNf3oxP\\\\\\\/LtPhgwx1PfdQPTc9ayhdPxH4HWPhyAXm27TKcBz+YcMcwCvXLuzhEGyNqzsi60Bs4Qe\\\\\\\/bouuISB8cs2K+amcikipOUolpoopNHRfxdNUQEkzcfzwTSwJYzX5MYC7owaOoEQY+Cp2RtL4fdam\\\\\\\/rXh9CKAaUkYYEyKuGtxS\\\\\\\/GDUUcItYyAxpZT7b\\\\\\\/C4TNOSoKgJaUWVZKUkLpQXA7Iwkm5VWxrkGuBgJS2Y9Fd8FZafFTTftYV4NrS25WKoFqU\\\\\\\/5nzMZNfQID3\\\\\\\/50EcFxBmkOToqyG2\\\\\\\/uQerXu6LWXW34Y5NxOm+Vdo1x+ux7aQJCI4wvgy+WgUee3\\\\\\\/iNOs+j7tufUx4MkKU8HboBTN9L0PmxXe4\\\\\\\/4p38N7KP7ZqpTFCoLjV8s8BNrYSaLDzhyguKbq3UekxkHT+EJZpE++lXl+SOnZwBkAjwknJgwjSa7WiedCTwOzPz90P2sv4PgkS4c6CO1NSQDHnE00rnZRlP6dYy08jgdJ6UKlrLSeiV9fx9BNCskQFrWv3oq4U3FOmU+\\\\\\\/biX0yCPNa2Nl9YIHaB5KVP2Ni+nsIRZDt+AUrfFw9ErA8HHCqWaZhd8q8btDAWwqgZ1g2VNyMd1cCM+uQE6cc+2e6ELT9zVDg+wS\\\\\\\/GQDZBDhRzXLGCqRZ2CsUK3twXPs\\\\\\\/B81Z1GSSnhCdm5p2xKiD7LtQ59QIdhuT4N8hFwvbV3yKchF52F476XP0yweWkLy5k8Je0iRR+Buqmcn+roLRtDAmazFWp4Gowsgjeomqcjh2FlqZ6CkJMXPO04y\\\\\\\/rOY4nMnbn0n8+jBXcnamCOMjbRNPjYAR0a0bomoaimwpjTRqRXgFTyYCHYraADCMcq9qbbZwGrajr0Maj4zoGDPCjVsN5hDiN8qVzTB5HM8CXWY2qoqq\\\\\\\/4bNC7Ei3XOUIVWwFYFijAL+FdlGYu36+AHaX5b6rlN10sl9JmWwqsA\\\\\\\/YfOKXQNmhIwJ5DQPQ6d4lKcXWHTu\\\\\\\/urRCgWCYV2DFwFTDmOYqzhN5f4eeFpVQrt9Y8CR7dY6h\\\\\\\/NrgqaSXz905803Gkem9SZj9l9YQWBhxFnGP6BWoOVqeUQ+ct8S79qnBWH7yNEIwuwwtdQvq9XCa5nu1dL5OUKCFZB+6t0Ow+L\\\\\\\/q0\\\\\\\/OqS18Z9owvwZKvIzB4c8d9UxQ4f6g5JtPzge6IHvocbClvnP6xqHDMPmQLY1R\\\\\\\/tZhA4Mb+C0GgYH46\\\\\\\/E97hVRtXAoD0Oz8MJeH2eO7fxVyx38J2TcRA4S3sx9jx5lWLsQk8J5B+6w1IFUPZfr2RiKagHvR01XfA4fr9GSB0R1oQZDDyxRIOCgcRNUxbnhxryIF2pt3Cclx8FRuW9ACowGonEkOZk2tTnRTVIwdKTduYsZ7osZYUBK8GElh9aHVnOFkDmJIuSzC\\\\\\\/dabn2K6GrIaWTpMOPFtLadTgGlhEwfOdUSxUggNIDVGZShS7OxEDDoceddiaNILvgzuS4esa5WvPnuF8KxIuaeeaQhcrZ6O0DF4aFScIYw8lESJkxLCrO3WOlYNgigHZmR3Nt6Ug8\\\\\\\/S6GkgDO+zWUZaH5nE9NscPmtvlkhHMaxCWg9vka1FZx4rTZjgTlsWFnm1zgPfjd0t5wrk0HR\\\\\\\/gW\\\\\\\/hwdDavIAubrro\\\\\\\/BJEFnTzIh4a9InAdmrsLQ0jmgAIDHtsAmJqCU9F6VinZ4mAqwAHxk0xmTQOIpVvl58uxzn55OT3SlXf3Q37EobDo1c+99urQh+BmERflOHimn4tKWsRKzgtFuIKWLyLKUgujQKAfhAKuD966I2ntB9szqSYNjl3MGe84uv9pR6EsqIa52U4GL3stzKsS6tH\\\\\\\/8BsakoAKZrOWLp0lbdnOpyK0iiRs9AM3ZCJjAf7fLob60gAPvVAVhtWcmip7gQyO8Ve9TNbzoIlAxc5epZjsLrfgiE4HICTdfqp7s3r5IkUizFFsyvWhVMah\\\\\\\/d+cpv3vKF5GVQw8M1c\\\\\\\/irCsPd7V9aqY0l4P6GhujarQulEdU2hHYCtcfSKCrIKT8K5OqdQKPrx7Sl9ZTdXb\\\\\\\/C\\\\\\\/iIJxw0HQyYmcQwqMV5D8yqL0g+hEB3zkBcorsHuKjB2O2RQoFueIfwt0AxJ6Y3LrXOVB+yuE2EsKFPxZHC+y6FKGaBlIpBYTlvC96rzA6ZblLzkv8Fb2Vi9XuWWJQLoqC2EvHBOe34ayJY4Pq2ph8V7llRGmpOG+NZ9ZXdfbwjXMoPMk4jvAWZVzZT+F4K3Ex3R3W\\\\\\\/itti7P9IxdT9JLHEWMAtLKcYeGYqUPVnHPm1da9KTHo5RxdGWfNqQlMHDWjfqzyllndLYDJWPTGF8VP0+RoeYWkTrPdE6Q\\\\\\\/g9\\\\\\\/JXiZPVw6sjxOEkuiP7OM52lgTQNtSve37urnBYbXp+Sp8vuVcUFy3bsKLK1EfemuYfJUX8IRPN5f5khqefZkwMNRJ9seCQ6hGTFu\\\\\\\/TB5LYdJnkMzLkI9AviA1WLi8MO9GBJLPdw4s9tUGZw7h8S2usegfaEp24l4EFkQuFUQcNCqDFRWlnKwIHY9R5pRsWCfdqpkd+pqo76DmVVEVlidLg4BvmRnovbjmmDxrLBT6UuCq43OB9ey2T0dGbmUBQA0ZTfiJnPgelD46A+rCYco5SBWxUca1ETnrNF7Z9JPebe0gulASD2n+a+XswzkYAgTVIq9qYJsoYmDFoD0AU5tWsgX6EMUxkU+z2c4GxutzkIRrjF4Vb\\\\\\\/T9zRpt0Yn4cB6UAX\\\\\\\/tYdnNv\\\\\\\/r6oYyl\\\\\\\/dNpJwIiyCIa7RnnpTj+NzGoL4Z2q6ah24AvI6Mxyki3R1rIFDMfSAIdc\\\\\\\/tLb1xV5jOi30ccFdtL5zWmzD6HqZTsxIpgrWJHzrhELl2t1LCjAeXvkPFzibwAYDixKj3Xsi8\\\\\\\/dmsq\\\\\\\/+RHKUc4FJhmlVxm5wMfEOD3ov+PeerkJKOLicNxmP\\\\\\\/5jhK6teFjaLTgWqEiK60+MtR+Ph4+rpA4jbqDDf7wSlgWqTQ+N3s7lbNyUHUwYJf0wS7Vd4SUn63Aa9lbBI57T4EsUNHicvIVu\\\\\\\/KrLK9NU\\\\\\\/C4ew4rZ+bUdKsd+osfDWErBdH59nnc4psjPNBmgQz0prpXB7fvhqEAIDXssQbrAJVu39Jev4SZHzIiPKncxGCTviMdH9VYSjkLhNDKORoD\\\\\\\/SFpx9qrORcdm9Xz+tY83DqBFZ0n8kvxBkft4R89Q\\\\\\\/neLREKc5MrUy6RJAyi1tSpvjMM5LqQtwYPNUclmpz3nn3CagDQrD6lhIysqSjPPPIBGtZTSGzv4k3Z48eNUzX4Ko40Kcnci5YjXwhmbIhkq3uFf1DNKm+l81+1c2QWGhDTJqGDOO8tpO7yBjMdc8vr0FIDzcv4szx16Y\\\\\\\/\\\\\\\/gbdFGiQ4UKBjB7alDu2lhK5EJx2FGkgclDpS8Ykl9Rioxr1zfb7poIwSBTuIEaL1PH2gTQfNSmcxYVLnLpJ29vlyrurbzeqIOcovcXFWK0n93+PsCKuTNHjCnidRQdraHlb9eFFN5sPWyLLdbkcfGjLYF4c1x+C223Ro990H\\\\\\\/690AOrhdPxXUBba+mO8V1oxwKskJTCtrjaLpBnAvmDSzJvMThcRE+cZU4HvmRtEnlmZQC4dfmIxwEmNUNDuPIIwci0VdDRS91pVveYnh84Mn3ZiwrczYanhoeHhM8FytFTRcpXGvpAssGOOJIB1uFlnhQZShz3UDnjbZ3\\\\\\\/icFUxqAXUj1CRcBz7hffSnNlgDMXy9IPWYHlOKl1t+BjwY8TXh0lrZSlt2V+r14vVITsMXK9gkcctWqragqbTttZZG5dyKNDfgewbwYg5PxleBoCUBOvJHXFY\\\\\\\/ursUUVcaHtmRr2d8hm7eF9tXjL5kROj\\\\\\\/UX3OErLkAOftssCpzBrXKrjfv\\\\\\\/MsMwz79e6c+MSrcaEyjkbe4bIxxnrq998X0g65mVKa0hB8XSF1pA1skBav\\\\\\\/aM3iZKbtq5p+AumMK+O58vZqEDbl66lIyenCZhziRqpsqJIjiBAZcwkBTqeJWQQDXaQYy6NLiSmEdsCZMYAv9ssYx558tqQX0nvl2D0WvNcqSiyeClfR0eSlP9oSGRM1RmfwOsO2BMyrqiLnwjd+mEB13CXsF3VO96FH+OBRcE75S5FeZsUAJ00S7GWOQZt3F0O5alFDeZzlQb+f4HI1RVHsuSzU4VZqRGqprhdcvdCZ6QzbfTX7HQS3i5YngrXuE+aQpGqPdEbQGtPBW+sKTHjLkfMSF74XqR7bgNFPsWX9bL1KOfuVVplsWAAz1z2\\\\\\\/GgG6Zf\\\\\\\/gFOdS2+gi8hhgMLIJ3aMHF8e7xEBkoQcF2iFubid6Y48MR4GhWPZHBcF5aYtEgdpPpDt+\\\\\\\/YeLL4dK8qtrXdqNPq5uL3YBuEMr0XpKMTvjS766rB2R959McR2IT+dCWR53qa8kxO9hFIvUwkUse4u+WL8ijgzlEF+0aDb9A9NRDLLF0tumdHDK1HpFqF8nlt8rSV+bjmTpWNYgui2DaNjF7Ly+nxua7eYZ3j3X\\\\\\\/EicuLwWhQGQ+80GYA1CgnTRSr1kt6c8lusXADZCxLOA6WV1uS+4h1sKwSKAeIEaZY+OHoGhawo0SdLDU+NtXLInYVlMWYldIEmO+f6bgYiB4s4kYafLH63Xkr1KX1XrPVV68XlMfBHBDd8dAQ+8og5U22ZvZspBMiIvrEW3K1D+EXIncQyVtK3TCM\\\\\\\/7SWkWwnzJR7Y+vFiRx+AAcYY9rBuq4CA0Oj\\\\\\\/kzag3fkfuDc7n1HLXibPoKWI2weglegqlV3ntnirYRo6DGjoGl3Imcfd72KvT0tvq3hcmW8dr2BzPf9gcXjFWzRT3ZiIYOmhwLtK6KDMHvLzly9LnutIXYxUMgsnc+8MkFLoZJPTTSiGYnPG1MoBXv9+AE6QVdNYDpJdEYiBQXvvxYrwaQl4lmnFMz0CaEPslj9jVx3yxZ9VXbVJtgl16NzbvF1TVxJBURJnS4qwNzhhO6lijaAYcDwDR2BQlo7s3LOb8gd2zxcPuLoA7r\\\\\\\/l4h4gkXKU80q\\\\\\\/8hCKq9V+hvzbaRlxGYgx+oT5JijyCFLXOkuKDttIUL0fNvanbe0qKlmYBiFOfoxpLqTmiOOQ60O9uP1gNM7xsiaJKHxGRAJLfBsHSlgYYMpBgRsp0wR5NVPLeVcoMmzERarovgk9OBHTFbaIML5V79FnmrChiX6q55uBkzwVSt6\\\\\\\/kEuEyYeF4neZdSiQuYip0Z0h4sAWjOFoDVXiEmu1lwC+5X3jJUOhNrynCOwN\\\\\\\/SFkeR0G9howiVX1\\\\\\\/tkV0KVEVlQhv+k8sNZFQ19IyBQK91kDjrjTsiFFAy2tsmcM15m1PNVyV2NnfdnWGTavTRRK3k7oQMcNTmprQ6TIzsqEOYkNitYenMKlXCfzI3tZFvBVRbhxGkF1ihxhk0fTrXf2nY9KC6GXavAIkf2PqFXzWzW385TFl2vtx92MTAUg9Jw125D1r4StDDLhNNDcdpo0lYeAwZb4NAS7N1+tYOqdM2DLJ+FxcvlxhB8oMWa\\\\\\\/YwKuRzc24Q6dq4+7T+mmzWqNXkZ8m6WTuBqgg4iEi3DILKxcIlKDe3ITl4qUXqa3eJnQq2z29Z7XKpO0LhrU69aPvKar+x5LOrktS4QXUnF7AzV3jqd0IKomYNQSNP5NP1Aa\\\\\\\/Rv9beEjMTgIMVO8WN4M2vKqYxNzHFdOZmcAAD0346ZPrcOxiplhPaI7hECgY9y+0KorGnNA8KCeTRwcCPi8H7JNkqv2\\\\\\\/krftpoEDpjVLrgzu+6YAkNkuMrO9HX4j3zKSHA8LIXB8nKg84FzG29PTpi5I2SU7+9sKgaoPh\\\\\\\/30A+39bJCP0vVhu+pXKXUoTXrM2Ci4r1ZwjZgfTR7xPS12zQjb74Ol5LeHAxTholIiZp1K4IA7cw1rCGM8PQw5CArqp20dRvK2OC2ZEif4gdDfSw80uJjFluAXL7cS\\\\\\\/b2bpOo3WO57F6XdPgWE5JlkUphCY4Y8B5sL7oWipgaEWHbp39gUOLCUkP2XRVZ7iQ4wWIJ2LbWytgoy22r3\\\\\\\/rmvgkU1cq7le0K0v9s\\\\\\\/9932fnJfGT+S5Uo03qce+H7d5lryFTMF0Nj\\\\\\\/vhNtLRLAxv2Z0LV80gkTStaeeWOzavOxmjvEQlhUW1jfubow5dHiJJtKGfqo4MusMZMmtqvwGmHGLbYvMAH8shBryc9X4YUfEE\\\\\\\/Ol9kKTsodxRHwwcWULDNYw2EuogeEEonIC1evVBvwtSliX1xEXeXnVVjFWSW3KPnxFgjmx1aBEjvjnFJI9flSdRAhfEuqYXErOzb9+OCwBgjRC1+rj6fDIWJ8S2cLLeMA+cEft9MhyCzUWIUcMlidahNxXIs4cEle9\\\\\\\/RUdJf+71nuxgPzQBysyYVAjAD+jq9coty\\\\\\\/K49AI+6JPRQgGxF1JVBheAVjbgvCqAedvL0nvO+g8I4YQAntK6JUkU4yAnp5KatJkiv5gSJLHKtzqj8d7flQanwOq\\\\\\\/D2p\\\\\\\/ZXPvf05Fd+RXzMFp+c2GhwclWPzOqhkYQp\\\\\\\/+w+26aJT4XfowtZY2Ipy3ZtmOQMv0k+cw68aGo3VZzkfAj67hsbozh8qgRNKRcc67QKCOddvlAbmI3Mdzf5VJlE+X74X2uwSU+EVDaHYbqpUubs+RSvcxCHiUMnOIJI7uCF8UpTb78JYl3G0KDBmml1UAeMhknDWNJPNfK\\\\\\\/5hiXG9+ICiQk4Q25vPgTebQMdHCHIPrz518NjhjN5TYMTByaDwMGuSPvIUMjCFP3RvCTY7l4hT5xLGBO2iJghVD\\\\\\\/44VMsuvLWEDyZcWlK0TfPo40YfKHrk7xDzPd7ILZ5zvpDw+eZ5DgzQ2jar3+\\\\\\\/W7PeqFQsIfmLxXs8VpHmNZ5hLYv4d1wpX4RCdiTvepS4wV5YEgu8gIvtgDL9Hpg0XN9JBlNmEhoy6vn4iaL53qwxhF1fa0wKTmI8v9EWE3cXaNLLEMFbeHJ8tOCKLiqBmbJVfseow0L5biRj6WQLLZyKyh\\\\\\\/NnPmg1jx9uWr21bl37x5WlXTNxFTunk8SckwfcTJHCNjFy7zOuWFHwAh7eRIySmaO2MO\\\\\\\/57Ka3wl6BbxsItQjYAD70vN3GUwY2aYzAiJpH22UWxa66CAE6dzW2u2UUQYD8sRvqQ3bmGloqAvYn+APKXjHqshKKBbPJkA57AsM1icMiLH6JrVHio1xUTTszVDUTOF9529hBIs9667a9R6vMlWs\\\\\\\/y7Z7CAMplry69DilyhPrCIEhYTJXGTIh\\\\\\\/paPvRmMZodGjcjcadQXr0Xlt7JN2HKYMKfgMbhCcZYUVJ+zrB12x9ZX84YnPaKLXnctptAJgxngSRFLFIxpjZvnXarn8kXMQ03HLJKu3QBN5fXVgT0P4GUyGkJDmfaoTDitKsRQKlCGTTGs+sfj3ypW2aZdEZ55b1QBgCvUuFcspoZw86DLYoZ+EWuOtvwHRzD0mCaUAHcaLct1fplOVaNjC9OnD0cv9a9\\\\\\\/0U50KFdP0pq13xuFSala5MkKbNNSoXE6ygm7CgMyNzsK4gUKv\\\\\\\/XcGPeY3\\\\\\\/r++sQOltgkZgxWWjXY9OtmVv0KzP4LdxXGYgkTEcTdaNsJAorh3zt5fLwm88xSaoWYt5P6Peb1lEk5rABDsCERbVgMOk7IMIZ1rGkw2Cu1keCmwm0witX5uXvf2QXMfPlBlYqH9fwq1rOkT2Z8jajfP+zngHdnsw4CxBwrwS7yphVGe6c97PBpOdDRo7I3P0U5EASiZ8EwI\\\\\\\/cpFdkvVvYCsyUdl3h5X22P53estVHz62xcD+gYGSCZ3AV5CJ6n47yUHlAgSTxT20DDS2Y1tOXsKA2Q6IsLJW3h9LVZZH7+fwofAPhlCUV3g0f+DROhTpnK\\\\\\\/yNET62sQRikEa8f4DG0ijr5QzWbyI9SYDY068kHZv32pcCeyLJC+VkKgLcjxnuIQKbTVhnTF8oDqJaEg3465bcxmvJI8Dys8yY2C6NPYteofnO7tGb8jTwCAgUhrJSAPnokcDUsUnMfv4n1iE4uJSYmR1UYMz3sAOB2Obm4sa\\\\\\\/+xyJ1311b0WwzO8o9dx7C2zm90y0f10cxhQpJPrl\\\\\\\/Q3iyZA77neTmD+pagmo8CHgZ2FQTTW9mYQTD4a8x7yc8BaMpj5m0AnHN93n6PR7EoN\\\\\\\/7kou+YVfy1UIA5xXMxXcpFGKbA8VuhtFvrA3otm6VhZE6FBieQ6RWb8e+luzllnVgF3io2Hgii3gnWgd7ArfrKmetAZ4QxybF4bt1Cdcgj6STzEOnq\\\\\\\/qHRFtL0YJ2k8yDwqntKwUzIfhznXL72acpWUtA147e1TadRAJbD9nLi7Za8f2huaa9aw\\\\\\\/Jf9jdnj9jmkLb003xyNPKtyWYpvoUQQQQTPHksaK5nvGkj89bcICP1sexb+SOxz6mptllHjnv0MYWFvbFte6LUtGjIDRKJO8SJ2wqEm3iZyjlZbi1He3BIpo09+RIf1oBMVEvDv1FLDZjvdUoRpg8OL2eszx0+BPakMe8vtgvsF6U0infTvLq6pVdCaWpC55896vyoirMQkC7Zyukzt8wCvADIxOsP69U0goaJHiB\\\\\\\/jmPXlukF1Q7u5ITnOAoK6S1tkO79vUR5GznPQTqooyQo+omt+t9D\\\\\\\/nJGr4XKGKm4QXeltjsPXCv4CA3rWwebFevbrZKJ61EJB\\\\\\\/THuBGe5VfM8NFSqrpjIxtLK\\\\\\\/bJrhI8WvHP2bwNesQ5F1RUeue2TJ8ksp6c1qpVKm7eoVHj+r5gff7IvfJGLuSklnfR\\\\\\\/9tbtshS1VKX0gvrCn\\\\\\\/zNjpW6DoD7OXAdoBbRcmnvJ6N3ZKZ0CiYgM2xT4vxj4fjgM58ZqaJZ+vUIfLkEWr6+3y1qMi\\\\\\\/ezAn1RJZvoVzLeEdwdejKFFE8j3GhGqtaxJGpQJkq3fbm47svcvnLkWpfDXIi8Rbjo83SBv6PKgUO4aZllcYqycZxqTKPFhniRY+81yIQMQHGpGS179\\\\\\\/nqcOxvkvZgk+j3wY4FnSth4tWvYNMRxPUHRC2pNTaHpigX6jh+BAQvwRDwflbh2MFGuVcytE0BOZxyYU+CegFojP4xpt6jbspC5ouStJn3fMMd+3wpNRF6x6H6CRCuTkxLOzEUnIpi04z6w6F0PglrD6iOH\\\\\\\/qU55hIZcqozqYPRVlBwMh8USVNV5C\\\\\\\/8qjUtxISQLzmlTg4dzpBh03WnEyIyrK2NN6xeN\\\\\\\/O+FtL1D+URo84k25IZ7ISq4shOhSRkPf3lP+rtLZF295ngncfQxRq8h2TU4l3BAP395obLZNj6rUH878UexfCBgsfjAtcK9319VmZNJkIMbJj9Au4S3rCYzRzzUhPgWbE6I9OQ9aE+YGRYCLCEytISqV9i7PzDw8FmBwN7CS2h7Akzeb5kjxfZiy4XJS0Yb31ojXolY9gMAiz2emI+40F\\\\\\\/iy0RR1OkfhHg7qvHSB10anDtwVtK8jkULgVLLbu8EE64MHXn3z72l8E7ie2tv1\\\\\\\/R8W249qJE\\\\\\\/F7k\\\\\\\/MJGvpq821A\\\\\\\/0W72MUyGnEONf1M+mYyoKs8jhjENcI9Oj70CjiHxBhczeS9g7rCzWm1OL0bwduw1JWSk2djy539sMkb77a4kRNRLQvxjhpZ9po84Gfm87Gfyv2s8ixk4\\\\\\\/ZNyfLQXgP4LvldFXUc9GKQfhUFOTyWYco6eqCqbSEgSzw80bg1HBlL2xSCyjSGa5cZgLFiQy1H663aRL\\\\\\\/J2iCSm2DXJi5LVfd3C68hTwIqQ7DnON2U8DPGAiDxd2mfXucV3VLIE+DyqOIC9oEPHMV2Twc83iDoVXNx5aaFpkW1\\\\\\\/++Bq41mwCxqoWGhHqZkCW33GupsKkK5Md8B\\\\\\\/GVn+UTM4bWBxlWgGTM0QLl1SdmuYSe1uTvw2uOj0bujvQ7Y4zx8VrkKSAP3mng+SF9Qc4K+VpasZMkjb4+B1uz\\\\\\\/ScAJQ7sfWK6CwT6fbpOrhmzQpsQ5AWSQ5s+LQcIu8ofvVd+uK+tA66\\\\\\\/iKb+Ju5z3u7v1GHLBnMnuhBbqhs\\\\\\\/P3a9gtCk8v61l6QBN0yUfCkA0oHCyjehim59x4sLV+WxkdzQW4xeTAKR12DCAO7q2wkEjeMK1v0aJ71OuqOlwGe87YbwRW1UHju0xav3vK9OiGobxR\\\\\\\/lpUxbjQEs3ahvqwCPStottE5PQ0KXR7Xj9L0soqoCuda\\\\\\\/kOkno6TqqLVnJSwwiDHlcntqsu1H\\\\\\\/tHwBIwCZnQzjCIKPnUE23ibI4kBdepluIWyG5dgIv9\\\\\\\/\\\\\\\/5JP60mrTH9sefIds2Hb+nyaouQqHcPpXPTv0m4rm5v9d5\\\\\\\/pLGGqf7KEwaxQSRUDx7eCVQGvHZz5itdqhKCt2RGe4qM77PSYDaWlf3\\\\\\\/AszewykzsDwp+OIGPEUoVHikpurUkj+wN\\\\\\\/\\\\\\\/UFb2cAcOzUrOFWE3Or484QWaYDh2c4BSyVyIHeSDvBibveD7yShy+mJ8OSkFgm\\\\\\\/AMg\\\\\\\/tpU1WooVDMak3mGW\\\\\\\/6\\\\\\\/HGYx0NRYjZMy9nGJ9rfOEafouVAThJnb\\\\\\\/WjIOorMTmTULGVGLWUMu579oOq1sMWqDLggW1BDKH0xCiikjpC8eGlgPOZQdyRaYzLPJULZK8KDzTNCADpzBDW3TShvR\\\\\\\/W19aENWuAlJMSKRCe\\\\\\\/DYTxnoNyzerUj\\\\\\\/NXRLn6zVXDta0AWj9bJ7E\\\\\\\/IP4YOEvt80+HLaUpktRlNFYTK3rPPS7AUBPjnwkaeLc8kxRdXbaVMmOCp6XCtcpb7VDJbGzKL2yj4AJ+4ctrpN64yzMGq5\\\\\\\/27j2SV9Jyaj11AEKOSbB4qWHGCPDQdYTPP75Q9BjPqI\\\\\\\/zSKIgpcg4A\\\\\\\/FuJ\\\\\\\/ZRNvMZz\\\\\\\/fjsL5Guk2JL57Hh+vpM85XnCAvthGRRT3ybYBQPj5113iuE\\\\\\\/z2flVYqzSsCaJfoskjq5hxTzV1NvzUDnX8dSZgxV7U3TE3O\\\\\\\/1bnvlodj53MwMD71A2TONjFTdqBjs\\\\\\\/7R27g6AP4vxcTldaKB6y6etjmmyZjSAy7soXhbe0WRRX2t8UGHxMJX+77TanN+oaUqIYGgktM8LOU7fElAmSQA59fY6IlDCUlM4mUGjn2wJhBLFwrz+ecVsmp1RjOxX20vldPfVMySBlaOrBhgfNnCCasd3UBgZoPxSLN7v35lB09gkJUUeUbNTYHFUcb22oI5gIYN3ogogapiLIgwf9KYJ4gdKysoqNMjxG+0gFHidfrrPoEUYmda1MB1Fcak4WZcwa9GIp5fiOU3TiF1xi4VODWHs8TAUTv3sF8I3u7r6PU8dmo6opf307rXb5+PrWgLjJ\\\\\\\/lSjxIQoNwtodfznCxL8IBzcjm\\\\\\\/H1oNnru1wugPMJBa2WRX9RssdjZySvSpSDbFg0\\\\\\\/+ewdUko6yCR4I4AlYXfejWvOf+xeKwsApgySmaltaNjjK+Oa1qjui1IN4Hwb8ayHSppWvA71JDKLwoxB+LE79zxjDf2f11vq5IpTlvYT9unJqp7p7R3fjUUbsiB5Mm5GbPvJPQPRpgpGxBdd\\\\\\\/opTqo6pT6MHbopnYq\\\\\\\/AHYA84Wi66JGg711MgRXPQPu+CQHayWBIIoBVQN72KMrRTn+i1G05PVHwjJQhBzB+8SXsGkQrkKew9Fnwz4VqTw4DxNnOFtVIlIWSKGYJUMChofQIH\\\\\\\/Pq1Qn22ExslbFnilCmzsZ9ujn4CmIZz45aSGTaix5sC3onaHiP7KQJ9me9b5nCfif1suIIFzqCR7JfV4w\\\\\\\/TGQD+7aIfs\\\\\\\/8xm2CkGJ1zieAf9xaw6LAIezukNipP2t50h818VI4iJ1VIyA9BoPhFieBKrU7uScpiPdp1Z\\\\\\\/MTKNwtepBzTiZd4W5u49E33n3G4ck1FbPRPmFIxfXm7pk\\\\\\\/wa9U5G926jkfvr3uUJ\\\\\\\/NFbgPMNnh+Mw+0AcI5ZKZUBZGLzeYegzAWHuMNcMN5vJykDQPrF+wv5MGzQ5Vdk7F3LvB+6QC8JPWSgDO\\\\\\\/wcNI9T7pPt7Zgwvv9RpBisAxfXwmzagwaubv0WVVfCQOBPS4wx5LeqTPRZRAPDL5J+oDfq2C\\\\\\\/nAZr4G7DOI+kqRFtPqSJ\\\\\\\/G9dtbjcVSXpcRG11yPJ+MXKuFtBjXnakBbVq8jfWaWEVp\\\\\\\/7xXa4uhOg5E4vtcr4or0wN1FDRcJqV3HI8wZVb8vH+2XfeC3oFfJXxYBtoFfeOd\\\\\\\/zKfmlM\\\\\\\/NT3BmesM6zfxMy9QYCQpgYFA0qBcqmdy2Im\\\\\\\/IUBlMBChDPE2vfQbEt+q8RKdFGNjZKR2Vk2pCQ1JSc3usazvYmIbRTd9mmQwRVnudFfih2A9rn8Nd\\\\\\\/U34B4AvCLnDW+4eCHxdBsDakDlX0VOsUKDubGXwLofB+IK+jjsp+IDvaD3\\\\\\\/Nfr7y5LFytVeHsY6b7E765SP54HjdpgYb0+RQi4QBTnUSNH2e1NcIymnccVCQz1XBvQAZbXmlfJ+kyIkEhzuzbGxbDe01myMNvEO83rHBj6A0gGKxWm8Uk8NVJtdiRt0tM5jflivoBOMQUpi6DoUJD\\\\\\\/GLr2\\\\\\\/CryqJNJ2VEQTt1sczu9C31Xw7qKvyWRnnDfs56mPZBJmemUad7DZSfBo\\\\\\\/cB2Bgs8F6NbRvsDU\\\\\\\/kgYosH01IhjwGwuAO82If1SsFF+cF556uSzS5mJkL\\\\\\\/yDZZH8sUaScXB+CTeNtOcYqDM4min2TK+\\\\\\\/2mbmA\\\\\\\/GHykZt\\\\\\\/VAr2WG4hPtgYQ6bxU1YbWDpJINzBWvRhEXLDGTMRvWPZebYgqQFsbQ5\\\\\\\/50QW9wbzh7lYlkwaVqfy\\\\\\\/OZ0lsmXXIFdTAleIoVi9ar8Dt8j7h7BtOL+QM3hH\\\\\\\/GCU0pMc9CmNPu+oWLizHof406G3PvutDYGyJ9vnDhwc2JOs9Ih10x4u8qGn7jhPilbKjaYjDmzrPn61iOFL2unSq4tNYy0E0aowzzVscwEO358KA6Pa4bKFDVRX9X\\\\\\\/8NsWyn+hucOL017w6hp88OvII3oF99+LpV94aqI\\\\\\\/dzrFGY24J94Cxjuil2yEdG9Dg5oLJVo+cWbn01FN8S5ywC4py84nqvLVm4Az1RVog8+xPhQpRa1Cq4F\\\\\\\/Ta+xTqcY8enEQpOopWyg3bvNjK4QXQ5EKzElG+R3G0XrDWjVeYYBOdDA8lISZOLAHEWCjn+wA7IhkugUzammj3Y7VKrur7H+NQCuYT30Lo4kM3FHP11icloh7ZXvqOHVGQ2mjae0fl5Q9eZIQzGmaqQx50pXExA2uNuIZxXO9Io40S54\\\\\\\/6oaerRRFLuT91rrmR3Efg7vusFfgX9FwdO2Qwa+Lyn5OLi4MBl24lCODW\\\\\\\/u1wesVKr1+2luDcLzKfY7LZX1xFuD\\\\\\\/2WJWutDiPkPFxH4z6Xty7vJ0Tt1HZzBKjYnLzTO2RqLwGGlMth+FlShIP1wrP+GbB1B01cGM4\\\\\\\/10G6ioo3bA9Uxzt\\\\\\\/MmCP75QnHJC4C2sOrgWnogf2MVaV+ddWSV\\\\\\\/Qumrqj69LxSoViyRatNMQRAbL4RHXF\\\\\\\/fy2IeMnHJg6gCFGRxYJBAGLvUDD3OyZn3lS7zMrIUwFznW3wt3m3GIT+054LBgKB0X\\\\\\\/g9XKI3DflPBFJfi6cQqPAiBKIUFx4EIVcp7EtLN8CbQbN8SyPI28BZvImwAYNXLEBIc+qAWFhqHSpP5xypQfXRpPpg995LtxBwrNq3tSV3fKYqtHryp1nIvV27vOE\\\\\\\/kiWhfwc7pvFAZNgDj0N+Fe1fz9hdZnpXDcEzHYhlcDvp7y1ybIALqdPyQb56gOLri4dRudkc18c7AoZXFn2rFkQBsN3KXg\\\\\\\/DVnU2gdVgrfBAq+meSgdjWd1vOtXA2LnUyg75EVspZXowNEIyd9q0Ro3Ni3vGX1fDCWzIRr1Mugadno2z2Oo96lzmD09I1xdtkfNeiF0pA3Q+pd6xWcTHX\\\\\\\/95MWCX5v76tVoXsfhfpwyuVJUT2496JFZ0R1r5YHqmjLQn8pGWc7iR499yESQxE2GCaWZ0PK2pHWD\\\\\\\/9fSgVzEvmx\\\\\\\/7oQzbmERqv8AV\\\\\\\/CIvBfW5KwI6tDPNeI4He62epbhg1WQijnQfyGq0ABEKirUj9aWdluVU1QqxkAUyhWs9fGYP286r2QDl3jHs5FutRtHUfZ2zihFmvYabkpxCtSnw3t\\\\\\\/M3b16a9JrXU+y4CCfHUWGpJapngjKnxTTzXytmnX\\\\\\\/CyXAXVv5lTCPPwxYG87LqjfRm9h6UYdX4UA+kk3JCfAk27lECXSCWHbY8i4fOKYdW\\\\\\\/L+vpxA+JpyYVcaXxDnp\\\\\\\/jF8HBH3bCiaH24\\\\\\\/DzvT+adhH2f1kA7ePXPh5jkxyd1jXl\\\\\\\/92MzOfAdDdjdCRDpEogF2ECQRzA3KjNNDw+hQeMideATn78LQcL9\\\\\\\/OR3LGIttKpLTNDHI2nkwTnZ1dRSELJra8NOMWITpX5aM+zD8Vx+pkWN4bwLkLUtkX0EwAZSz0kWpdbm4N41nGpEgxKWo1o9hT3NhIQ+z8rMioNJdQ0dgNXvJ5wPFvNQmHcPV9wSZDJ\\\\\\\/WIhGRgXAe5QOACxYu6M1SaZR5CZS6UMGBVtXkAKQhzTuDs88E2TZPIIxOTOEPx\\\\\\\/ewvWjCJ92SR1yoCfg3trom7vyMBIvVBGN16r5JsntnwPHrPiU1QptjT2PuVh68cFYwfIj8h61tlF5u2oEUmaRda7qztDJp0M3A4jF\\\\\\\/W0mdBYwQJi5qZAhF\\\\\\\/yq6ofmiWVe2VjbX28CBybTuS\\\\\\\/cBL63Atop32gln8yAgCdr6So06iWzY+GxKBH3kv+RrRTUmsrjOgZ07phHQYOIij+fKDNOKei+L+d+ts8hGv1guuWAe8YJ5qXGFoQQxBvHClvKnuY9cso1l7wywHSD0DMvvY93REKWUDtyUSgfA38dkN0HQaLb4q\\\\\\\/KcI6C7jKVq3PVcYbAl\\\\\\\/qZPTOpW80oNTbG4I1O6\\\\\\\/DJhSrVeit8189Anr6Vpzlc2aJdfyrNQpas\\\\\\\/acM4EfSPpLMviF\\\\\\\/W2TfXzyoI0vogmuKVi8W+9C80oOyJ1sYOhw\\\\\\\/ZEG6\\\\\\\/THQ3CaTSBFzS9ydhkt7rcLVwz5XQxr8tjOl6jdNNc0q7\\\\\\\/fC48CHD397xBPkdwL8BTMviXnW9Tkhm3hP1qt7U2AnR71HOWdSQBIpQZTqnRFwDqd5dYhveZupY3NmyjHNrfq97XVBy8YxOW499AW8opoLR2j9LWDPbm3hCoHsfqenezzhA3VopD7ign3AkURPNEksnkdy9\\\\\\\/bLjc4y3g3Ck7tsdgq\\\\\\\/DDkWnOIPi29+OuBnYJxTVszZDx+NDmj+7eXPHm+3Q0h3l6GNFhvjvoq5VSQZ3awNJUwlOw+gzqKrJfPDHyKYxkfCL4HRm8bHfEygu+rwJpZcTnL4UUp50y4lMKSvZ4TFcrTXcm7BSx2ic\\\\\\\/4\\\\\\\/47UUxfo9a2bVZL0lbZR24Noz70banEhmKkCAtd8m9Zl1ADiBdDkaESc1bkIOeOjKf41nJhlfHwKO7LABP80OccdzJb77ilfLj0k3w7COXN8tiCUJei94560F+3kQk8R8fhIo12z4N6UvR84iM0VJfxnHd61EUTUVRit8eWREGjNBkoOSmOwQG6S6AJReKHbujHJoQBqQ3a0gVkoeWs3Fdsg\\\\\\\/b\\\\\\\/0ydxcHCJ2pbysBBewc6pZpLXN1xLqfsX65xmaODf8jeT8zKFcDuJwcdE\\\\\\\/5H5I9npY1e3VbDnmcnfctiPO4ypUl7aV5deku4IORJ2XfcPxylcUBixQyCOx\\\\\\\/Dz9Xdrb7p\\\\\\\/oAepXN7td9cLeEVp4mk1Wp2lIqKzDnZ9\\\\\\\/yvupIZsmgWRqCuM\\\\\\\/r9I9Y+XysC55\\\\\\\/QiZZBeiARX2qXqzXRvlbomwWyyYyNC7peJN\\\\\\\/fm4E0SqNPsuSaBGeSZ48z1m+a\\\\\\\/4ue8\\\\\\\/9g77UJ9lVBCeW0v49gJfSZCq2DN4ugQEewnVCvK5IACCo9mpDMCZaFrWc4fMQpHUqJei529hlZniLCGAVOaNWl96soEX66tEe1SxJmd6NcIk1olf0pnhmImdUdJvVn9V9PD94lFJuE0CLTxpyOuGkXbpbyUkF5BS3xpGxKu4apTFFeNlCwgRfNlBuUAX175GcbruQrIp7oFgHvxB2VRxWHPPYLrd6SEERWyac2ushRNC+uW1XNaAwtZyyPBwaO4Pa\\\\\\\/55kKdFGpssJZdpJlYn8Buyl9QZ\\\\\\\/0aECBv6wkgNtY\\\\\\\/HlsKaCnR4DxbINxzO6gd6kh7+ea1ifOt0pJhM9yN+WlGDVRa5lEH3Qo82LXJritADBFkhpb5ltk8G1tjy4jpNRVkLJa9EubOxBiUVG6LJTW43RebIaZq8qNA9N3a7SvaWf1bJUNXnmT\\\\\\\/ajyqyDjlSvEh9V+3hgpOhtjUGyqkRc6FbTBYH0\\\\\\\/i\\\\\\\/KWqH4GStqls98GH1lq3RnoOwwrGlAMNRhOgUDxVkv8JJC1xKLUDu+yG2f2QuiVDMmtWvSj2wvsWzZ+qvJjRnJV5SrVCeMEmzWBPFx4mE7G1Z+IKgeRw\\\\\\\/yiDzyjKNOP9NG9BaYOmsNlA9Mg0PP1DIWdh8tG6Z3qg\\\\\\\/FreGaCHgj9h3TGJ1Zm\\\\\\\/Mg6kVa9wW+e+7fwkAN7injSx0bZidL3RXdJb04ve4rC2oXfjrD8KADMFKixEN+HF4Pfbx6tXn4GB0d2neFZyfQzLu5bamAdiM2YW+HJ12J6+z0+4n6mAgxvSs1xf67wFAylE1l7qNCZ0a0F50hs91+IZsWgE0aQIcur8kBxG3PN86bmrhUc88jqpBMCfbgdvSvsFOSvu3H4bjh50g7v41YSAAHkK\\\\\\\/+yKMGxyQLkmjRlctzpmwPIsNO6mTwV5XQ1BdT5AQAVY9NqIX1mB7zc2oW\\\\\\\/hqwfVqAnmuZpU8XVxWe+V48Wm+ObaAegCwSdO3nj1+xwFCP1tee2mJ84z4xPx8nNuNfNkwY2q56tuYUL9D71TFQgn0dnodS+GOsy9vNsA8nRoD1hL9RvkZpxHhviRQU818J3Zx1FG9ZFFW1k02Lhe7hvHoi9+rKVlDop9lvwrcZEauq\\\\\\\/07\\\\\\\/n\\\\\\\/KXlO3tGhi+rL2rg7kaKbOlwaULBFYbZl8t23BDZVSH34WIyikXSM6S7VorCxiUleUjlrpK5TwkKpuaPaB6bieRxX9jH+bOnRtwV+zjGT3TJuHPVCKRSIL1R\\\\\\\/tQ9LaZrQgOEO+SYtv35+WwpE+JpNLouHLisM+d3yqerykqJUNrDodHoDtnzPFg7N8OVLvus\\\\\\\/ETOVl3yUzh1iJeTcOOycoEhmtiptTYMCVpLFrP7rA7NFDQ3v5kAOZzh4qr6lP8y0ZOHCCAG1LcaNw4+LwtahKBLDQyySMY0K9YlCYFyDBHBHwBhNFtOxlwh4lhe+pB3R85wcaeLMiWOYOK5cCwB8c3UoM3rm037lPRXLdw14y+9OmSSFVpLSptWNK+De2u5STbo7dAHgSQlcflWffffQe3isrrmCGCs7VhheGRV1uvOvVvM4SaJzSsFWuB5rv8ehXmjlQIFWVkuKR08qmRQLh5Aqpj3+6NGoFSoBIiUurrF9H1WdOFcxAOBVd3NZScQhbENwKfEX8DBrlO7BarkyhIxUisMC8ci+1iRNBnP009pasNt2o3wxTE99NH2LUALM+ik+uYkvtTG7Rjx1fFwAT+bMfcHDMjbp6Nl6QOMMjvCW0snNTQ7yKcirjEqNPnI3SAM6+J7HVIibEm5IagUlSiXkhCofCDEnGFVTbQsQC1n0DBavy4Fogn6wI5nIfCs49okG4q375aQxhM+9dCv0ffuO2uQk7WNBGJNRp2EEoNtsNNRq+Y6q2T21ADzxo2vLGK3XWf4XjYqw5aJv0hjn+xgtSctJMCkEAEG2TyCWz72\\\\\\\/Z\\\\\\\/KOLHemFv3rd8MQEHbuwCRES0VBMlfbHhdHGAR1+gqmIKQuIpdkAN6M+HgBxnzJMuBC0injP9hF2pWMJscipE8XVMsESA3ok\\\\\\\/XmwEQOuF\\\\\\\/VS5GT3uNRZtVdsgISjbgwF7O5qAYhehgcCb3W1j8MUVlTwoczST9gUKxrfQdkjfy8L0RG0QUlT9kE\\\\\\\/44x\\\\\\\/UCyctSQDMCXZJ8SW0FgAHL2lkiqbUZ6KCgDwijmO50eXtFpSlpMZemYqBwmFhTIf6GK5UwJ54zkRLGrYJTJh3gfceBoX72Zvp8WrCSCCHWn8rMDH3L0cJE3X81u\\\\\\\/4WF8V915Cj4NeTa0zWeTZtd+ZSdO7KedAFpjGNn3Qo4Q00Xe3VCONaXcbqdPjmzcynUnm19rZAoRO6oVTjTRPsLjWg9z\\\\\\\/4SU\\\\\\\/sWoCxQSCHLWCoUCRCzBt+Eo7O5SuKG524tpGKKlbeCI1zrWvKsw7opZjievfoHzs8AK9HIvPJFG9sf2QkpAyydf9KNdbJzeOmfVfJULQsHQ5\\\\\\\/Iy2MmE9TXMhBoLhQTHESlB5xotn1xvwLHDsAIW5jT3gRKWM0qJ3zRU7Nkm35uHqPP09KZDsT1V1KxoHzMbmey+1QTmKEBR2A\\\\\\\/JAC6q4qHzPa7tR9HpX3yV\\\\\\\/wGzXt7sgQE5\\\\\\\/cQoZm+W6CUQ15wXYh\\\\\\\/i9PiFV1KXP8uO8yFYdTAAFE4aiCDLy5ryIIugNszH9nEPqc+4LIC0J7wNGJhnwCO+ML9gZH1pTvMuR9snKpd6opSaekZe1PX+EVd8Wr3bSep7HfCUgRqVf51kCizOPIJT47Yvlg1X7r\\\\\\\/hTlkTHKiwF9GsUcqyr\\\\\\\/GCL2tOtP7sDeiG6viUAcpocybTZWnpmGKQheiPfR25f6H72eAYaH\\\\\\\/3VYC7JR95RP02hTn7bSt9Y9vEHNhNHgJo+tP\\\\\\\/bJ2QRkd18ISSLDsUsVjCuArPci3FzXW5lBfRIL5NMpjFlZzpFnOBoaAXoXLkcTGozW10VRP2MNCQ2mhvvm1U4dw\\\\\\\/LxoCesgJB6f7FY6TLI9nanG8TWLZ\\\\\\\/RzDkuxekVfB8Ha2dPwGThC\\\\\\\/p0w+ybp7doHw1PcQiCUFm8Gx0ZrvSuIiCL\\\\\\\/OzePX8fsEngHwi+XDlx\\\\\\\/tQ7pMmGCnFC9\\\\\\\/PuXFq1M2c4eonAmy7RTWxYuYdib2PsCot7EAYfeFJxf6n41aPdzIZGR62QNv4tLR\\\\\\\/F+cUTB0t8tFLmqyMS6PuNXs6ClXRZ9vdm8hpGu343Iv9bzZ9XhHgaAf369hrRkPu4jtcuetrdZPkPYnqvfr3jC\\\\\\\/vuqiOLWckjnptm0OqVTYIMo8mfPGKzhf+t7D3tauW2KpvhZmc2ZL1GeUtLuDP17Bv1CnF+jMcqLSpevAq5mFUyl4Qe1K42aBlz04HzaVwesm83i6RvD8ERcGkjyGcuhXa5pe4CV\\\\\\\/it4oY0L+cy46XZECbMkWzOVuePM4wMH9e0K2GUMx83vFBUvI7BN78MPYBP2m8001prBxJDacz9UbOupDv9\\\\\\\/b1+juwhSxuRoZn7DcP38nraXSCjj73R\\\\\\\/iswE+vswEMD7b7o+cnGydzDrCHIE7Bln7pJ4AFa1sqcZXUZGGKhQdVYN5RJ91ladEY+ErtLNkdKcVdS33WiXCMBnG4c9lL83UJjLqDhF3UxDheLmsgkW+kTqqPyEynMhHAHhgspP5kenrGj87utGdHDCfNrBqFHUPVZ1pj3qWIAvqd87X2l2H0V3GIDPpzsH3ffrwZYJ5PmTU\\\\\\\/zuKT\\\\\\\/EeEKz0Rm17AbEolPNw6Aw0mmzHmsrN9kpJFp5h2qncxbuPQWn4+BBZsoqm7UxGzeNaxuD6Z1gGSb8HLSeP31XKHbO7zY2c1ezu1vba7zBAT+FsChhgQ6jTGp9sDEFiReeYoyLK0FQ3CynIpcLMuxNAGwmDeVdoPs\\\\\\\/wOGhQ5SFRRPd\\\\\\\/zLFjItty0w28BO0aQBFqY98kWTptyrS0fNt83CcrtiDiQqL30+X5\\\\\\\/LeHxQc0LWhSGHgDIlrO+tV1lAZnNKcOAceJFEVNYS3XyGLs3rJWAa7wVzi+ANgFfofryA4DY503BOqkrqnJTnpwb4o7abKOKTV4n3Zy\\\\\\\/lh3lj+8pOIIXFiejJu5CtkdIkJf0DmRXk84L3QHXUjYZgDOvRZNu9rLMlpKYZ\\\\\\\/cnz9MFTg0S0iRY9BSaAyKPeCzu4ssZd+lf5MjS0ahKwMvHy5jIdsGHAMf+LZ9th+MniTn5Ojzj2zDC8iz3\\\\\\\/ndC4+Dtj9oV\\\\\\\/YiwI547hD76Qb1HS9q6Ni3g53cZzC4i22fz8WXj8\\\\\\\/7MLNxVNOs\\\\\\\/R\\\\\\\/\\\\\\\/qJNcwRdWWV2C0Q+Kaa9D96dfg67fV9oA2MueLgt3CAbVLpOrL1Kg5uB50NLQgwi3yvJ8q5kWSRE3JD7zQoTnVN\\\\\\\/Af24ymrAogYO5PvEDmOtmNWcrohB0OpocBu0ITtlpGCJqLRLaAuo+kxd+nyWVI4+h81LTurS+QkKl8HVUFKVF0IOwbeRatvteLBVGf0Rk7xi+mYTG3zue2Duq1muoJmKG224wJw8yZDmPsl8MK415Zh+lxiLcUhAWpq9qlumPs2hCLpiRYuRm9\\\\\\\/7g+VVBVFUbcRGBZ5ruS8bOWQT9YR19LBHQgRwtOO8duXPqkUkx0Ed+p\\\\\\\/LbA\\\\\\\/upcNoiW5VID1a7pVvmgLjiKHDlZy2Un7YKLmGZ6BiPM7HtRng0jkNslRq+WhsWuGb55b49hWbnSP14bMwNNhN+kIDxDaWGoPCodFX7MuurB5eqB7poNPRWmehOVf2PnROahCVcuxIQV0Ju+WI39xJj9KL+FHCjzbG54Jgf0s1OXL2ezmdHUBEANrjkT6p9H7eXWqOJzx7gj2lSPAJig\\\\\\\/5biOQsydUAMWdwm0Ff15An\\\\\\\/LtsdDPh2cbehE8k96QscopwbS71Vl47O2J0\\\\\\\/S\\\\\\\/QK+BkVUIIxdkouATmckMzQvZhYBA0xDQQW+cgqyoRTA+JmsdJw8TJF0OzwLdwfj1KkLP+SzAAohTaIl9nbcXdoehV7rg2f0MbyfEjNy718mHlkLYMmLmSISHJv\\\\\\\/5QUFM3UMaa0sju5Vx5tkZZzTQS8BTrT5dPO9PIYcRurQjW2qpnZdUVCls+HpU18R8NfGGQGljEuD+bdXibw\\\\\\\/Cnq711XabdvvdFynyi\\\\\\\/s+Nfce2vMCB1iELsomZntHlclxljVqXwh\\\\\\\/m9Rm8kMxXzaiC8vp3FuwMZrsuR+K\\\\\\\/Sp2hAAGLbA1jtIf8xBr4KK3\\\\\\\/SXFvpGIMSF38SHHiqS44mNYMAMtnp5nffVlIH0KYl9+HF0pMKxN\\\\\\\/PUeRuplw0yADanmfeGk2JFU1TqDP82lhvdaIxHf+2o3wtWsfubKRx1ltGp8om0BpzSkE5WYygWK5ZmQxXb2BErCpX863lr7cLEAFgkPjyjYAUHYeOXTi2rU\\\\\\\/CWFWuB33lZ5GESiK3oK5AQPVD\\\\\\\/QyQT7+Fv9OsGTWLmLaX\\\\\\\/9ALAycqGKKV5THiQHX2BP3QGCCTH2JPGzZ99bSBXVBORHdSxDgTsoLYprM7Txs4+qJJmPsbOXygjcbo2hxaBgKz28w9aYamqNy5b2GkxNabKczecJy5kRF2sB\\\\\\\/TeWO45y7hxuWWEkERvS8DeYTOym7ps9JLcSdKNqt+GCcA0YdlFKzGp17JN4Z3zA8QIWwyEBoAjkI9rDXIup7CyrMCw1Lz5ZvoXauNI\\\\\\\/jr1Z4N4sXRirZU51NrP1n2MVRYg9xBqqtAnAqCTI2l1m+YGXx+K7gN69WaAU65yATRN+rNe7JCSCOUSuCKgnpzFyWu59lxwKx99fTWDWidrQNIiGpcW2pxPM8n82ZDgn3LWydXcrQXsVrfOKZFv4dLXeG3ZHpI45iWSmyOy4lCTEZOPydazwueuZsh55BD+\\\\\\\/NH3GhnTt0Phugt30RYsMdykecFsncCFWE1mRePGJoH\\\\\\\/+qT3XvSt8NR9VjRvNiqKLNG5gZzliiuO04wixYSZasgnw3ezjVMibgrjZ5tkDozM+o3I2++vZr1z7gY1l6jC7SeOYoPcrjxcxaW\\\\\\\/jqnUm\\\\\\\/J6Gd3r4OUxvllkUKll1gZMh7nsiQeR+kW6NbK8tmfij0vqZWW3on354yBrcEypAkI6\\\\\\\/ul315TXfhddHpXnRadnBn7UqE\\\\\\\/C0CycginLEMxgvdr5k8jmRVi0qUN3Ipzfn4XwEbPdDO9nly0+AYhRxRC7nK\\\\\\\/fPxr7G1uK6NP2F8JVS1\\\\\\\/jWej6M8PG\\\\\\\/KzaV9yPVdw4WTo7Uw+vesHE84GsQr\\\\\\\/fHqFxY62BVb9Y9wi9DY6PM1oQCKRVHNGZNuMvwdb6tC12gA5XQgVbfpLQ9tmt7T37OKiqHpO8FY6dHWuOSjz3sCLi6lrpBA7GZIjE+sW3pQG3UQndbEh+xz5Ky5MnT3iYRKGywjk5QpS2VGFrXifsXY4qutR28Myadf7CnaLSQZabAVc3jTOA0HcgfEtSCEIkp3okd7wGpyT21nFRjDTmv+gI9xfN6SgtuDCPaBjgmNRvTwMGEkg4+AvZLKrNQN6zOZYeG0fWa3tHa6c88uQT1J4+U2qnVUForiaDC7jwC5D0afvNsdz8QtkUgh97APu1vcba1yXC9Z6H9J6Ci3sd\\\\\\\/R+OdQTGW2D5zuX4E8PkasIiTRKCa7KpgojGKymUADQlWyS6BHStujNLwcxM3YqYacJd7PAfXnmifmUI+0zelV4\\\\\\\/8EHy9WMOsM6VjAJlJTQy2Tr13Ktzx8mzdgfqtnQgb9wfLrPXXzDyJ+eGxPOrWbe5ag+QqXzr0ZvWMkWvwCCNwFTsL+Vv1EZH223cOd3wXWe+2mgJDcd+JBTg\\\\\\\/CXPQcvG71jX60OywvYrdGVAM8uD+SX7KHw+UGK2TGhQS4c2VAlFAtzieMpRS4DhNsbYCUnfN++Gkb\\\\\\\/Vpri07WYQLeLxC0YNEmnZYHmhQ0CsNxKMwKdUUpQ4y3IpmQHYA0yMAO2RkNlsAj6lSIySjX9+doGLe5ZY8yZLE045TIMKpqcLWPztCJmieFlRinR+jGK2LAZPqTPyN+ISZiyEAIPJsQcH5tUZmQfp8KBwcV\\\\\\\/GjaO8oFLOIkD3t10UOEb1xER2bWG2V8pXtiwUzhpgaym\\\\\\\/OPPmAhigE0+DT6uq\\\\\\\/3EM0et9wRyYNx3Tsm1pBrOz95tZhz1nICxoX0ghcPQ7sM51LYL8j\\\\\\\/iXiBA+TJzaFiXTGAjCB+DeOpOJ0HtofY\\\\\\\/yEFiunRbvj\\\\\\\/YGjzyxtdj+UJrfb\\\\\\\/EkCcN7Dx3dVLYUKMttJq0lr2TpU+ZrDyuK+Woex+LAsO9mebWFJocHU48+r6PqlE\\\\\\\/AIMzX6UYPRZoWiYZWIv72s0yQH8s4VSNpkiOs\\\\\\\/xNV7G6OKNyoCJ0JYOfw4eOEn4SNZHjcM2oEapaDNVYeafiIH4CVnh82tV8Tm0vURfTa3x+6AM83t0jW2qFD+B03grAiMT8Ns9H84cJbwq1D6hXDO\\\\\\\/p7Fd2rnJmEuBVh4Oy+NWQO8tg2XB0C8qUM1lEO640TGVnSfiUgEhh9iOgIYvHcg2besBYTy056lYANbDEfgXXpZT\\\\\\\/oK+3IMOaosJ8Z\\\\\\\/l0zDrn\\\\\\\/rw4EBBArCgvuil20hKieOpu2h\\\\\\\/2109pWIBPL2PqLK02wdeX39fMnSgLcAglRWaBTWYYI6nln1s+8Gu9yL0JhOnLbA\\\\\\\/OGxOjY+BKq9Ot9ZRLyIeD2dOAznI3FFdq97KVTSpep7\\\\\\\/eAvWPRwXEbq2psVtXAKJEnXi5SBcH9d5GTHD47MlgkP4+Bdz4dSbAi6IyIG6Brx\\\\\\\/0C\\\\\\\/Q+c6tyaRy7Dab26CsXxwK2phxR3oo+JJi3+oOtyJIMjWJuAUTyskhQNVUIhMAW3T3wDyg23xEGi8bFuQ8uDqEK894QGZmVYqWMOIYxaO+WGVMTN9u\\\\\\\/Z8RMy7KE5iOCNdLGMsRkavfGDz8miJHjzUEinC4Q2QCjaziyGFTOAYyzA8w9XL7vEwLhd6NTDFsLOnSRIPkmfh3bxfWEtjLZ6\\\\\\\/pUWF1zhwe791p1gIDM3K\\\\\\\/TjTqGzGWdkViDW6uA4f5zamMiUiJ9ffW2j6W5iLQxQaJ9I6elBf6vuQlWZg5PkS9z3rN\\\\\\\/OC0SV9aAT4qP+3QJQB5x0ZTKaPvxk5srcnh0QxYLgF5ePqM+nGSzHXeQKTYQEFZhXsYnOwDA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022US10PJ7aorcb7Ryw4Ra7Kw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00220eb7d46af7dea550859114f847ffdb8b216c5914d67bb4502ea549ff60f1dcb1\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022foIhP2v+FK7H7fmc8Q4sOCi0KY8Bj0STdvjDPJwiAdmebUbf2195StzZzdRBHlUWNCeJ1QTDxVJpsG6EwJvfKTMTsFf59C9YmA4ItLz2YZtI+P5O+CpSRqNzC8a4laU51GSyZ9lL6\\\\\\\/3pswS5xhUy67euKgzKdnKrgEfhHIXl1IOI1oVZ0cn736Mu2AIu4av0YxiDb9Y7mANjJEiDvGWLqCVFMFcvAun2r37DfvPDblW7h7rL7vN2NrtZ6WMxr+IoCpGTgjOr36m5jzVYpXJSsozsL1ekAKDxmpd0LtQytFHmE\\\\\\\/P4JC8Ml8N3CgJ90uZg7l2ZYUFp8iyMnPIDcXrl9lbGZa5hEre4h29GVY1osDbGvZ3LngJQFQ93c6fGEY2jkE3rn21C\\\\\\\/MWxub13mHRXHbZVTVDMaoQR8ca7SXmPzeYPJLlNN6FEwhaG0er5obwqk+80aN+YXSh\\\\\\\/wL2Y6Q63qZtidZMSrw0EM7xiAVPSS8PwvhYNGuozRDJH\\\\\\\/iPAGkICa7bosMHEUWsDEvRsXvXfoDT5seOEQBwJoNzJ3JRZ8Yeo2Hm\\\\\\\/RoQnL4uF715uUwWjreRtJ8eRbusnKx6veko64+JYvlHdYv61N9UiMZxTFkA3myAITstla\\\\\\\/xvWLR7bC7meOnBSVJuJishoqjJ3BygFJUTJZ0snf1uhtAyJG\\\\\\\/J\\\\\\\/OlWFIimk3jIh27ZBCohoB6V98KPOXf7mAt56ld1nKSi7JnDVS1zV6whtJNzIRR\\\\\\\/c0nojTlJzOMdsuicEmWtq1GNnmtonBbzsSIdLiWvbplIHXM9UgjdcVtvmEilAX9RbkP6x0FC44GzuOIWY0rwyGY8yWWmV5uiijGPStiFx8wHsO\\\\\\\/jwYSwP7KPXUYV6qzCzg\\\\\\\/cU97XIMQaP9p80VcRZDpMDoBYewJZzZY5zLzgkUUeSx3Ecy1eS8bHxph4CGjeqp3ZppeFIlBMXQVLEb25CM8fch3KefxmVqDhx+EX2BBnSCK5pkNurzY1vr6CEG8mgrvcwA9XnBnIUhLADucDH4HBl8ddsWeGUlPQw+t7Qh23+RQGdUNf7WRfOYnfeyN7jvOCPGdoYGAd2k7mI9558lJNIpFjYWLz9CDulAR07T10uktFOWmgO2wXfM9sxYCJCGS6Qa0Af1NKUCCP8OF0GfC6wCYwE2OvcmyZNNFfPMqqk28fR\\\\\\\/YH7klB4S5jtDS+AQ\\\\\\\/7nU\\\\\\\/i5Iptw4w8fquVgk2mlZLuMpbDweqfuON5vxI3x4AWpEuZBWQF\\\\\\\/ApqgRASf6OdHkCF+YcAkQGdmeaglT7cC2qltNztrflcIh2xNAe+0T2r1nZVC8EOeQgVwAoL0Go\\\\\\\/r+RlhIIWHy35im2uh3nPyHisiayYVGEN0l6dhfvbbnzZVdJMQcNbOpQYMxqn206qcwTqK2aHnZ6yRfXiU\\\\\\\/KGMbvsuv8bOh9fXs9M3u5JDROQBfzaQamwgdfe1dLvuZfj0P56\\\\\\\/LMi\\\\\\\/0e9moQFTHPtciDf5sMfY2uFXLeLFOKThwEL3tK9YQiqg7k5TcoSwwudlbLg\\\\\\\/WEI7pMYord6Md6kfYeZkaAnuwt\\\\\\\/F6Pe6VUwxaiuqlZZVNjcqwAbzej2JP5dOhBNay2A6m1NnPGEiUgMEHyklC4b+LQsWqqwkcDWzb3lUw0C34ZElOGhVZQ0hGsAY1QKg+XDjLYYVLmq8fQJloHoH77Ym8njCv3fhEjW\\\\\\\/dlRw1w9fz+fk\\\\\\\/ICh0aiATTiYBWa9A0RNYzCirOzmf8qStnT7I0G7gUzwb0sxhaHqTEWi5R4lHaGDh5vFRtPg78kjpt3Zk0Jns\\\\\\\/RJhJ6SYCB9w\\\\\\\/lmt958BJ\\\\\\\/gC3zyggaWibS8ZzQJGwfc9DjXUTA4JyxmLsBIN3\\\\\\\/c2WfmfoCX7L3hW3uzggaueyFw94c+V2ba0ZrJbjbsUAqlxnKNTVjs1nmrKI4dmlUkvWlzM3ubsJq7KUpe8PtPBeubjJsZ3UfZueSJbERXds3AkGJ54HAUJvRneDUzkOGLN7OsujSXFK4n3h4iYtPCFpxi+IyCa5zOg==\\\u0022,\\\u0022iv\\\u0022:\\\u0022wZmfN0pleLCyUQC470QQDg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022d666439fcc7550e2c7ac9bc10f1b0d255e1116d73201a4d054972e16b85059b1\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-487830 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/1/219" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-d0D2WabsrsAR2oLBEt1K_JRLpnvwBM3VuuvuXYHStPw" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/1/219" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-vqH8rPvGgcro77aII-yydWu3PUbw-Bil5oYor16veHw" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/1/207" title="Global Transcriptome Sequencing Reveals Molecular Profiles of Summer Diapause Induction Stage of Onion Maggot, Delia antiqua (Diptera: Anthomyiidae)" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/1/231" title="Egg Case Silk Gene Sequences from Argiope Spiders: Evidence for Multiple Loci and a Loss of Function Between Paralogs" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="487830" id="node487830" data-pisa="ggg;8/1/219" data-pisa-master="ggg;g3.117.300296" data-apath="/ggg/8/1/219.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Comparison of ChIP-Seq Data and a Reference Motif Set for Human KRAB C2H2 Zinc Finger Proteins</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Marjan Barazandeh</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-8222-008X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Samuel A. Lambert</span>, <span class="highwire-citation-author" data-delta="2">Mihai Albu</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-8721-4719" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Timothy R. Hughes</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">219-229;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300296 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node487830" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Marjan Barazandeh </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Terrence Donnelly Centre for Cellular and Biomolecular Research, University of Toronto, Ontario M5S 1A8, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marjan%2BBarazandeh%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Barazandeh%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMarjan%2BBarazandeh%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Samuel A. Lambert </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Molecular Genetics, University of Toronto, Ontario M5S 1A8, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Samuel%2BA.%2BLambert%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lambert%20SA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASamuel%2BA.%2BLambert%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-8222-008X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Samuel A. Lambert</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Mihai Albu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Terrence Donnelly Centre for Cellular and Biomolecular Research, University of Toronto, Ontario M5S 1A8, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mihai%2BAlbu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Albu%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMihai%2BAlbu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Timothy R. Hughes </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Terrence Donnelly Centre for Cellular and Biomolecular Research, University of Toronto, Ontario M5S 1A8, Canada</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Molecular Genetics, University of Toronto, Ontario M5S 1A8, Canada</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Canadian Institutes for Advanced Research, Toronto, Ontario M5G 1M1, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Timothy%2BR.%2BHughes%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hughes%20TR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ATimothy%2BR.%2BHughes%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-8721-4719" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Timothy R. Hughes</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:t.hughes@utoronto.ca" class="" data-icon-position="" data-hide-link-title="0">t.hughes@utoronto.ca</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/8/1/219" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:487830" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:487830/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/1/219.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:487830" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:487830/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/1/219.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:487830" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:487830/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/1/219.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:487830" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:487830/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/8/1/219.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:487830" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:487830/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">KRAB C2H2 zinc finger proteins (KZNFs) are the largest and most diverse family of human transcription factors, likely due to diversifying selection driven by novel endogenous retroelements (EREs), but the vast majority lack binding motifs or functional data. Two recent studies analyzed a majority of the human KZNFs using either ChIP-seq (60 proteins) or ChIP-exo (221 proteins) in the same cell type (HEK293). The ChIP-exo paper did not describe binding motifs, however. Thirty-nine proteins are represented in both studies, enabling the systematic comparison of the data sets presented here. Typically, only a minority of peaks overlap, but the two studies nonetheless display significant similarity in ERE binding for 32/39, and yield highly similar DNA binding motifs for 23 and related motifs for 34 (MoSBAT similarity score &gt;0.5 and &gt;0.2, respectively). Thus, there is overall (albeit imperfect) agreement between the two studies. For the 242 proteins represented in at least one study, we selected a highest-confidence motif for each protein, utilizing several motif-derivation approaches, and evaluating motifs within and across data sets. Peaks for the majority (158) are enriched (96% with AUC &gt;0.6 predicting peak <em>vs.</em> nonpeak) for a motif that is supported by the C2H2 “recognition code,” consistent with intrinsic sequence specificity driving DNA binding in cells. An additional 63 yield motifs enriched in peaks, but not supported by the recognition code, which could reflect indirect binding. Altogether, these analyses validate both data sets, and provide a reference motif set with associated quality metrics.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/krab-c2h2-zinc-finger-proteins" class="hw-term hw-article-keyword hw-article-keyword-krab-c2h2-zinc-finger-proteins" rel="nofollow">KRAB C2H2 zinc finger proteins</a></li><li class="kwd"><a href="/keyword/endogenous-retroelements" class="hw-term hw-article-keyword hw-article-keyword-endogenous-retroelements" rel="nofollow">endogenous retroelements</a></li><li class="kwd"><a href="/keyword/dna-binding-motif" class="hw-term hw-article-keyword hw-article-keyword-dna-binding-motif" rel="nofollow">DNA-binding motif</a></li><li class="kwd"><a href="/keyword/chip-seq" class="hw-term hw-article-keyword hw-article-keyword-chip-seq" rel="nofollow">ChIP-seq</a></li><li class="kwd"><a href="/keyword/c2h2-recognition-code" class="hw-term hw-article-keyword hw-article-keyword-c2h2-recognition-code" rel="nofollow">C2H2 recognition code</a></li></ul><p id="p-3">The human genome encodes ∼350 KRAB C2H2 zinc finger proteins (KZNFs), which encode a Kruppel-Associated Box (KRAB) domain, which is best known for its repressor activity (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Schultz <em>et al.</em> 2002</a>), followed by a tandem array of C2H2 zinc finger (ZNF) domains (up to 40), which mediate sequence-specific DNA binding. The ZNFs each contact three or more bases, and typically bind in tandem with an offset of three bases. The DNA sequence motifs recognized by the array of ZNFs therefore often resemble concatenation of the base preferences for the individual ZNFs (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Wolfe <em>et al.</em> 2000</a>), which can be predicted to some degree on the basis of “specificity residues” at positions –1, 2, 3, and 6 of the DNA-contacting α helix (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Najafabadi <em>et al.</em> 2015a</a>). The modular fashion of DNA recognition by C2H2-ZF proteins apparently facilitates adaptation, with evidence for positive selection on the specificity residues of many KZNFs (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Emerson and Thomas 2009</a>), such that many encode a unique sequence specificity (<a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Najafabadi <em>et al.</em> 2015a</a>).</p><p id="p-4">The best-characterized function of the KRAB domain is to recruit TRIM28 (aka KAP1), which represses transcription by subsequent recruitment of SETDB1, a histone H3 lysine 9 (H3K9) trimethylase (<a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Schultz <em>et al.</em> 2002</a>). TRIM28 is involved in silencing endogenous retroelements (ERE); this observation led to the now widely accepted theory that KZNFs evolve rapidly to silence EREs (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Matsui <em>et al.</em> 2010</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Rowe and Trono 2011</a>). A KZNFs <em>vs.</em> EREs “arms race” model (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Jacobs <em>et al.</em> 2014</a>) provides a readily understood mechanism for the evolution of new KZNFs, but does not explain the retention of so many of them; presumably, they take on other host functions. The KRAB domains vary in primary sequence, with an average sequence identity of ∼40%, and KZNFs vary in their protein–protein interactions (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Schmitges <em>et al.</em> 2016</a>), suggesting that they may also vary in effector function.</p><p id="p-5">Detailed study of the KZNFs requires knowledge of their intrinsic DNA binding preferences (<em>i.e.</em>, motifs), and their genomic binding sites. General characteristics of their motifs can be gleaned from their protein sequences, <em>e.g.</em>, that two proteins should bind distinct motifs, but current “recognition codes” are too error-prone to obtain high-confidence motifs (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Najafabadi <em>et al.</em> 2015b</a>). Furthermore, it remains difficult to determine which C2H2 domains in a large array are likely to bind DNA (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Brayer and Segal 2008</a>); KZNFs typically carry many more tandem ZNFs than needed to specify individual loci in the human genome (2–40; median 12), and known motifs for human ZNF proteins often correspond to only part of the ZNF array (<a id="xref-ref-23-3" class="xref-bibr" href="#ref-23">Najafabadi <em>et al.</em> 2015a</a>). An additional challenge with ZNF proteins is that they often fail to yield motifs from <em>in vitro</em> analyses such as Protein Binding Microarrays (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Badis <em>et al.</em> 2009</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Wang 2014</a>) and HT-SELEX (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Jolma <em>et al.</em> 2013</a>), possibly due to misfolding, lack of obligate cofactors, or the fact that many KZNFs have long binding sites, which would be poorly represented among the sequences in these experiments. Systematic application of these approaches has consequently yielded motifs for very few of the KZNFs (<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Badis <em>et al.</em> 2009</a>; <a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Jolma <em>et al.</em> 2013</a>, <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">2015</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Weirauch <em>et al.</em> 2013</a>). Other approaches to determining sequence specificity of C2H2 proteins have been described, including SMILE-seq (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Isakova <em>et al.</em> 2017</a>), B1H (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Noyes <em>et al.</em> 2008</a>), and analysis of selected sites by context-dependent models (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Zhao <em>et al.</em> 2009</a>), but, to our knowledge, none has been tested on a large number of human proteins.</p><p id="p-6">ChIP-seq and related methods [<em>e.g.</em>, ChIP-exo (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Imbeault <em>et al.</em> 2017</a>)] represent an alternative to rapidly obtain motifs for C2H2 zinc finger proteins. ChIP-seq is a relatively challenging means to obtain high-confidence motifs for TFs, due to the fact that the number of binding sites is relatively small (in comparison to, <em>e.g.</em>, HT-SELEX), that the genome sequence is highly nonrandom (with many sequence biases and repeated sequences), and that proteins can associate with DNA indirectly, such that cofactor motifs are often obtained (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Encode-Project-Consortium 2012</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Mathelier and Wasserman 2013</a>). For KZNF proteins, the fact that EREs are often bound represents an additional confounding variable, because motif-finding tools [<em>e.g.</em>, MEME (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Bailey <em>et al.</em> 2009</a>)] typically assume that different binding sites are drawn from independent random sequences, while EREs are related by common origin. To address these issues, we have previously described Recognition Code-Assisted Discovery of regulatory Elements (RCADE), a computational method that employs the zinc finger recognition code to predict primary binding motifs from ChIP-seq data specifically for C2H2-ZF proteins (<a id="xref-ref-23-4" class="xref-bibr" href="#ref-23">Najafabadi <em>et al.</em> 2015a</a>). We utilized RCADE in two large-scale analyses of human C2H2-ZFs proteins (using tagged, inducible heterologous expression constructs in HEK293 cells) to produce motifs for dozens of human C2H2-ZFs (<a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Najafabadi <em>et al.</em> 2015b</a>; <a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Schmitges <em>et al.</em> 2016</a>).</p><p id="p-7">A recent study described a similar ChIP-exo analysis of 221 human KZNFs (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Imbeault <em>et al.</em> 2017</a>), also in HEK293 cells (but using different constructs). This study did not describe DNA binding motifs, however. Here, we applied our existing data analysis pipeline to the data from the <a id="xref-ref-8-3" class="xref-bibr" href="#ref-8">Imbeault <em>et al.</em> (2017</a>) study, and compared the results to our own, and also to HT-SELEX, SMILE-seq, and other motifs from the literature, where available. Overall, the comparison reveals that the two data sets usually produce comparable motifs, and also largely agree on the sets of EREs bound by the individual KZNFs. This agreement is obtained even though most of the individual peaks do not overlap, suggesting that both studies sample from a larger set of genomic target sites. We introduce a web portal that summarizes the results of our analyses (<a href="http://kznfmotifs.ccbr.utoronto.ca/">http://kznfmotifs.ccbr.utoronto.ca/</a>), and produce a set of high-confidence motifs, each together with a confidence statistic that represents its ability to predict binding sites in living cells.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Reprocessing the ChIP-exo data</h3><p id="p-8">Primary and processed ChIP-seq and ChIP-exo data were obtained from GEO accession numbers GSE76496 (<a id="xref-ref-27-3" class="xref-bibr" href="#ref-27">Schmitges <em>et al.</em> 2016</a>), and GSE78099 (<a id="xref-ref-8-4" class="xref-bibr" href="#ref-8">Imbeault <em>et al.</em> 2017</a>). The Trono ChIP-exo data were reprocessed as previously described (<a id="xref-ref-24-3" class="xref-bibr" href="#ref-24">Najafabadi <em>et al.</em> 2015b</a>; <a id="xref-ref-27-4" class="xref-bibr" href="#ref-27">Schmitges <em>et al.</em> 2016</a>). Briefly, the ChIP-exo reads were trimmed to 50 nucleotides and mapped to the human genome (hg19; build GRCh37) using Bowtie 2 (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Langmead and Salzberg 2012</a>). We used the “–very-sensitive” preset option, which allows the retention of one alignment for the multi-mapped reads; otherwise many ERE instances cannot be detected. The multi-mapping may affect the reads containing EREs due to their similar sequences by aligning them to the genomic repetitive regions other than their origin, and could result in erroneous peak calling. However, as we and others have described before (<a id="xref-ref-24-4" class="xref-bibr" href="#ref-24">Najafabadi <em>et al.</em> 2015b</a>), this is unlikely to be a problem regarding the enrichment of EREs, since these reads typically map to another instance of the same ERE, and vice versa (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Day <em>et al.</em> 2010</a>).</p><p id="p-9">MACS 1.4.2 was used for peak calling (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Zhang <em>et al.</em> 2008</a>), where the provided input DNA data set (a combination of randomly selected peaks from all ChIP-exo) was used as the control. The biological replicates were merged into single samples, retaining all the peaks from all replicates, and merging the peaks in a maximum of 50 bp distance from each other into a single peak. For further analyses, the top 500 peaks with the highest enrichment scores were picked.</p></div><div id="sec-3" class="subsection"><h3>Motif analysis</h3><p id="p-10">Non-ERE peaks were filtered using RepeatMasker track of the UCSC Table Browser (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Karolchik <em>et al.</em> 2004</a>). We used RCADE (<a id="xref-ref-23-5" class="xref-bibr" href="#ref-23">Najafabadi <em>et al.</em> 2015a</a>) using default settings and MEME (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Bailey <em>et al.</em> 2009</a>) with CentriMo, on both all and non-ERE peaks, giving priority to those obtained by RCADE and in non-ERE peaks, as will be described in the text. For further detail, see <a id="xref-ref-27-5" class="xref-bibr" href="#ref-27">Schmitges <em>et al.</em> (2016)</a>. Enrichment of motifs from ChIP data and other sources were tested by calculating AUROC for differentiating the top 500 ChIP peaks in a range of ±250 bp from the summit from dinucleotide shuffled sequences, using the single maximum PWM scoring match for each 501-base sequence as its score.</p></div><div id="sec-4" class="subsection"><h3>Motif similarity analysis</h3><p id="p-11">Motif similarity was measured using the Motif Similarity Based on Affinity of Targets (MoSBAT) method (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Lambert <em>et al.</em> 2016</a>) using the energy scores option. To make similarity measurements consistent across datasets we set the sequence background to <em>n</em> = 100,000 and <em>l</em> = 100.</p></div><div id="sec-5" class="subsection"><h3>ERE enrichment</h3><p id="p-12">The ERE enrichment was calculated as the proportion of the top 500 peaks overlapping any transposon and retroelement instances present in the RepeatMasker. The Pearson correlations (<em>r</em>) between the overlapped EREs of all possible Hughes and Trono pairs (matching and unmatching within the same set of 39) were calculated using R (v. 3.2.3).</p></div><div id="sec-6" class="subsection"><h3>Data availability</h3><p id="p-13">The detailed results of our analyses for the 242 KZNFs are available at the web portal of the paper (<a href="http://kznfmotifs.ccbr.utoronto.ca/">http://kznfmotifs.ccbr.utoronto.ca/</a>). For each KZNF, it contains motifs from different sources, a heatmap of the pairwise motif similarity scores between different motifs, the overlap between all peaks and top 500 peaks of the Hughes and Trono ChIP data, where available, and the ERE enrichment of the peaks.</p></div></div><div class="section results" id="sec-7"><h2 class="">Results</h2><p id="p-14"><a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a> presents an overview of the data analyzed in this study and the analysis steps, the results of which are detailed below. Briefly, we compiled ChIP-seq (“Hughes”) or ChIP-exo (“Trono”) data for KZNF proteins from two different studies. Hughes data (60 KZNFs) is from <a id="xref-ref-27-6" class="xref-bibr" href="#ref-27">Schmitges <em>et al.</em> 2016</a>, while Trono data (221 KZNFs) is from <a id="xref-ref-8-5" class="xref-bibr" href="#ref-8">Imbeault <em>et al.</em> (2017)</a>. We also identified motif data from other sources that correspond to each of the proteins in the combined list of 242 KZNFs (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Matys <em>et al.</em> 2006</a>; <a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Encode-Project-Consortium 2012</a>; <a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">Jolma <em>et al.</em> 2013</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Kulakovskiy <em>et al.</em> 2013</a>; <a id="xref-ref-24-5" class="xref-bibr" href="#ref-24">Najafabadi <em>et al.</em> 2015b</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Mathelier <em>et al.</em> 2016</a>; <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Isakova <em>et al.</em> 2017</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Yin <em>et al.</em> 2017</a>). The Venn diagram in Supplemental Material, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300296/-/DC1/FigureS1.pptx">Figure S1</a> shows the overlaps among the data types.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/1/219/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Overview of the data analysis steps and methods utilized in this study." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-426255469" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Overview of the data analysis steps and methods utilized in this study.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/1/219/F1.medium.gif" width="440" height="310"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/8/1/219/F1.medium.gif" width="440" height="310"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/1/219/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/1/219/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/487974" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-15" class="first-child">Overview of the data analysis steps and methods utilized in this study.</p><div class="sb-div caption-clear"></div></div></div><p id="p-16">For all analyses, we used two versions of the peaks from the Trono data. The first is the set of peaks reported in the original paper (<a id="xref-ref-8-6" class="xref-bibr" href="#ref-8">Imbeault <em>et al.</em> 2017</a>), and the second is the set of peaks obtained by rerunning the raw Trono reads using the same peak-calling system employed for the Hughes data. These “Trono reprocessed” peaks are, on average, 10-fold more numerous than the “Trono original” peaks, but they are only 1.8-fold less compared to those from the Hughes data, for the 39 KZNFs that overlap (average 40,660 for Hughes and 22,483 for Trono reprocessed).</p><p id="p-17">For each KZNF, we computed the overlap in peaks (for the 39 present in both Hughes and Trono data), and the proportion of the Top 500 peaks (or maximum number of peaks where the number of peaks was &lt;500) that overlap with each of 934 types of transposons and retroelements cataloged by the RepeatMasker track of the UCSC Table Browser (<a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Karolchik <em>et al.</em> 2004</a>). We also generated new motifs (using both MEME and RCADE) for both the Trono original and Trono reprocessed peak sets. As in our previous studies, we favored RCADE motifs, which are supported by the recognition code and therefore more likely to represent primary binding sites, but retained MEME motifs if RCADE produced no results [(a result of the algorithm failing to converge, or nonenrichment of any of the predicted motifs in the ChIP-seq peaks; see <a id="xref-ref-23-6" class="xref-bibr" href="#ref-23">Najafabadi <em>et al.</em> (2015a)</a>]. We also favored motifs derived from non-ERE peaks. We then scored the similarity among the motifs for each protein (choosing one per data set), and also scored the area under the receiver operating characteristic curve (AUROC) for all available motifs on the Hughes, the Trono original, and the Trono reprocessed data sets. The AUROC scores reflect the ability of the motif to discriminate peak sequences from the background (dinucleotide shuffled peaks). Finally, we selected a representative motif for each protein, favoring those with evidence for direct binding, and those with the highest AUROC on any ChIP data type. The web site accompanying this paper (<a href="http://kznfmotifs.ccbr.utoronto.ca/">http://kznfmotifs.ccbr.utoronto.ca/</a>) contains files used in all the analyses herein, including the numerical data underlying the figures, as well as a visualization of the motifs and all the analyses for each protein considered. It also contains all the motifs as PWMs, and lists the representative motifs, together with confidence metrics for each: AUROC on ChIP data, similarity to most similar independently generated motif (if available), and method generated (RCADE, MEME, or external source).</p><div id="sec-8" class="subsection"><h3>Overlap in peaks and EREs bound between Hughes and Trono data</h3><p id="p-18">We first considered the overlap in peaks and the overlap in EREs bound between the Hughes and Trono data, for the 39 proteins that overlapped between the two studies. <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a> shows the percent peak overlap for each protein, calculated as the percentage of the top 500 Hughes peaks that overlap Trono peaks and vice versa in a range of ±250 bp from the peak summits, with the four bar graphs representing comparisons between Hughes data and Trono original and Trono reprocessed data. On average, 35% of the Top 500 Hughes and Trono reprocessed peaks overlapped in both comparisons, albeit with a considerable spread. At random, the overlap should be zero in all these comparisons, because the 500 peaks encompass a miniscule fraction of the genome. We observed no correlation between the degree of peak overlap and quality scores of the ChIP experiments (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Kharchenko <em>et al.</em> 2008</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Landt <em>et al.</em> 2012</a>) (data not shown).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/1/219/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Peak overlaps for the 39 shared KZNFs between the Trono original and Hughes data (dark blue bars), Hughes data and Trono original data (light blue bars), Trono reprocessed and Hughes data (dark red bars), and Hughes and Trono reprocessed data (light red bars)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-426255469" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Peak overlaps for the 39 shared KZNFs between the Trono original and Hughes data (dark blue bars), Hughes data and Trono original data (light blue bars), Trono reprocessed and Hughes data (dark red bars), and Hughes and Trono reprocessed data (light red bars).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/1/219/F2.medium.gif" width="440" height="330"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/8/1/219/F2.medium.gif" width="440" height="330"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/1/219/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/1/219/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/487945" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-19" class="first-child">Peak overlaps for the 39 shared KZNFs between the Trono original and Hughes data (dark blue bars), Hughes data and Trono original data (light blue bars), Trono reprocessed and Hughes data (dark red bars), and Hughes and Trono reprocessed data (light red bars).</p><div class="sb-div caption-clear"></div></div></div><p id="p-20">Our use of 500 peaks throughout is for convenience and uniformity; different numbers and proportions of peaks will yield slightly different numbers, but with generally similar conclusions (data not shown). For example, we note that the Trono original peaks often have a generally higher proportion of overlap with Hughes, relative to all other comparisons made. This phenomenon is explained by the data processing; there are a substantially lower number of peaks in the Trono original data (&lt;500 in 17/39 KZNFs compared) and they are presumably the highest enriched based on the description in <a id="xref-ref-8-7" class="xref-bibr" href="#ref-8">Imbeault <em>et al.</em> (2017)</a>. This trend is not evident with the Trono reprocessed data, and similar (albeit again not identical) outcomes were obtained with Trono original and Trono reprocessed data in the analyses below.</p><p id="p-21"><a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a> provides a detailed view of the agreement in EREs bound in the two studies. Our comparison statistic is the Pearson correlation across all ERE classes, where the value for each is the proportion represented among the Top 500 peaks. <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3A</a> shows that the distribution of values for the 39 matched KZNFs between the Hughes and Trono reprocessed data sets is much different from that for mismatched KZNFs; 82% (32/39) of the matched pairs exceed a correlation achieved by only 8% of the mismatched pairs. The distribution is quite similar when comparing Hughes and Trono original data, with 82% of the matched pairs exceed a correlation achieved by 5% of the mismatched pairs (data not shown). <a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3B</a> provides a visual confirmation that the individual transposons and EREs types represented in the three peak sets for each of the 39 proteins are largely in agreement.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/1/219/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Overview of the ERE enrichment in Hughes and Trono ChIP data. (A) Pearson Correlation between the 39 Hughes and Trono reprocessed overlapping KZNFs (matched pairs; red bars) and nonoverlapping KZNFs (unmatched pairs: 2964 comparisons; blue bars) and the frequency of the KZNF pairs at each given correlation. The arrow indicates the correlation beyond which 82% of the matched pairs and 8% of the unmatched pairs lie. The percentage of the peak overlap between the Hughes and Trono reprocessed (yellow dots) and Trono reprocessed and Hughes (green dots) at corresponding correlations are also presented. (B) Fraction of the top 500 overlapping KZNFs enriched in TEs (ERE instances and transposons). In total, 51 single TE instances were enriched with a fraction of &gt;0.1. H, Hughes; O, Trono Original; R, Trono Reprocessed." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-426255469" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Overview of the ERE enrichment in Hughes and Trono ChIP data. (A) Pearson Correlation between the 39 Hughes and Trono reprocessed overlapping KZNFs (matched pairs; red bars) and nonoverlapping KZNFs (unmatched pairs: 2964 comparisons; blue bars) and the frequency of the KZNF pairs at each given correlation. The arrow indicates the correlation beyond which 82% of the matched pairs and 8% of the unmatched pairs lie. The percentage of the peak overlap between the Hughes and Trono reprocessed (yellow dots) and Trono reprocessed and Hughes (green dots) at corresponding correlations are also presented. (B) Fraction of the top 500 overlapping KZNFs enriched in TEs (ERE instances and transposons). In total, 51 single TE instances were enriched with a fraction of &gt;0.1. H, Hughes; O, Trono Original; R, Trono Reprocessed.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/1/219/F3.medium.gif" width="312" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/8/1/219/F3.medium.gif" width="312" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/1/219/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/1/219/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/487980" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-22" class="first-child">Overview of the ERE enrichment in Hughes and Trono ChIP data. (A) Pearson Correlation between the 39 Hughes and Trono reprocessed overlapping KZNFs (matched pairs; red bars) and nonoverlapping KZNFs (unmatched pairs: 2964 comparisons; blue bars) and the frequency of the KZNF pairs at each given correlation. The arrow indicates the correlation beyond which 82% of the matched pairs and 8% of the unmatched pairs lie. The percentage of the peak overlap between the Hughes and Trono reprocessed (yellow dots) and Trono reprocessed and Hughes (green dots) at corresponding correlations are also presented. (B) Fraction of the top 500 overlapping KZNFs enriched in TEs (ERE instances and transposons). In total, 51 single TE instances were enriched with a fraction of &gt;0.1. H, Hughes; O, Trono Original; R, Trono Reprocessed.</p><div class="sb-div caption-clear"></div></div></div><p id="p-23">Overall, we take this outcome to indicate that a large majority of data in both data sets correctly identifies the spectrum of EREs recognized, assuming that the overlapping KZNFs are an unbiased sample from each of the two studies. There can be good agreement on the EREs bound even when the peak overlap is relatively low; however, the higher peak overlap is usually associated with higher EREs correlation (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3A</a>). One interpretation of these observations is that both data sets are drawing from a substantially larger set of <em>bona fide</em> genomic binding sites, but both are subject to noise, and neither has been sequenced to saturation.</p></div><div id="sec-9" class="subsection"><h3>Similarity of motifs from Hughes, Trono, and external data</h3><p id="p-24">We next compared the motifs obtained from the two ChIP data sets for the 39 overlapping proteins, as well as against the motifs obtained from other sources. For Hughes data, we used the motifs directly from the publication <a id="xref-ref-27-7" class="xref-bibr" href="#ref-27">Schmitges <em>et al.</em> (2016</a>). For the Trono data, we generated motifs for both the Trono original and Trono reprocessed peaks, using the same procedure employed in <a id="xref-ref-27-8" class="xref-bibr" href="#ref-27">Schmitges <em>et al.</em> (2016)</a>. Briefly, we used either RCADE (<a id="xref-ref-23-7" class="xref-bibr" href="#ref-23">Najafabadi <em>et al.</em> 2015a</a>) or MEME (<a id="xref-ref-2-3" class="xref-bibr" href="#ref-2">Bailey <em>et al.</em> 2009</a>) to identify motifs based on the ±250 bp sequences around the summits of the top 500 peaks of each protein. We used motifs obtained by RCADE, using non-ERE peaks, if the algorithm was successful, since the fact that EREs within a given class are related by common descent can confound motif-finding algorithms. If non-ERE peaks did not generate significant motifs, we included the ERE peaks in the analysis. We used MEME to generate motifs if no RCADE motifs were obtained, selecting the top-scoring motif using CentriMo; similar to the RCADE motifs, priority was given to the motifs obtained by non-ERE peaks. Overall, 80% (48/60) of the motifs from the Hughes data, 74% (164/221) from the Trono original data, and 77% (170/221) from the Trono reprocessed were obtained from RCADE. RCADE was successful in at least one of the three data sets in the 39 shared proteins, and in many cases, on the same proteins from the Hughes and Trono laboratories (30/39 in common between Hughes, and at least one of the Trono original or Trono reprocessed data sets, 1/39 from Hughes only, 1/39 from Trono original only, and 7/39 from both Trono original and Trono reprocessed, but not Hughes).</p><p id="p-25">To score similarity of motifs, we used MoSBAT (<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Lambert <em>et al.</em> 2016</a>), which computes the Pearson correlation of the predicted affinities for two different motifs to thousands of randomly generated short sequences. MoSBAT is appropriate for this analysis because it is nonparametric, and requires no adjustments for differences in motif length. <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4A</a> shows an example confirming similarity among multiple published motifs for ZIM3 (similar figures are shown for all KZNFs at <a href="http://kznfmotifs.ccbr.utoronto.ca/">http://kznfmotifs.ccbr.utoronto.ca/</a>). In this example, only 35% of the top 500 Hughes peaks overlap Trono original and Trono reprocessed peaks, but RCADE produces very similar motifs with both data sets (<em>r</em> = 0.90 and <em>r</em> = 0.89, respectively). <a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4B</a> shows the motif similarity scores between the Hughes data and the Trono original and reprocessed data, and the corresponding motifs for the 39 overlapping KZNFs, illustrating that virtually all bear a clear visual resemblance, with only 3/39 (8%) and 7/39 (18%) of the KZNFs yielding similarity scores &lt;0.10 between Hughes motifs and Trono original and Trono reprocessed motifs, respectively. Overall, roughly half of the overlapping KZNFs between the Hughes and Trono data (19/39; 49% for Trono Original and 18/39; 46% for Trono Reprocessed) yielded motif similarity scores &gt;0.50, a value typically obtained from different experiments using the same transcription factor (TF) (<a id="xref-ref-16-3" class="xref-bibr" href="#ref-16">Lambert <em>et al.</em> 2016</a>). Furthermore, the majority of the overlapping KZNFs (33/39; 85% for Trono Original and 27/39; 69% for Trono Reprocessed) are clearly comparable by visual inspection (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4B</a>), and bear a similarity score of &gt;0.20, which exceeds the score of the 98% of the mismatched KZNFs. As with the ERE overlap, in most cases the motifs tend to be similar, even when peak overlap between the two data sets is low. Higher peak overlap usually results in higher motif similarity, however (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4C</a>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/1/219/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Similarity between ChIP-derived motifs. (A) Similarity between the Hughes and Trono motifs for ZIM3. The heat map on the left indicates the MoSBAT similarity e-scores between each pair compared. The motifs and the motif-finding methods are represented on the right. H, Hughes; TO, Trono Original; TR, Trono Reprocessed. (B) The MoSBAT e-scores between Hughes motifs and Trono original and Trono reprocessed motifs and the corresponding aligned motifs for the 39 overlapping KZNFs. H, Hughes; TO, Trono Original; TR, Trono Reprocessed; R, RCADE; M, MEME; a, all peaks; nE, nonERE peaks. (C) MoSBAT similarity e-scores for the 39 overlapping KZNFs between the Hughes data and Trono original (blue) and Trono reprocessed (red). The dots indicate the percentage of the overlap between the Hughes and Trono original peaks (blue) and Hughes and Trono reprocessed peaks (red)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-426255469" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Similarity between ChIP-derived motifs. (A) Similarity between the Hughes and Trono motifs for ZIM3. The heat map on the left indicates the MoSBAT similarity e-scores between each pair compared. The motifs and the motif-finding methods are represented on the right. H, Hughes; TO, Trono Original; TR, Trono Reprocessed. (B) The MoSBAT e-scores between Hughes motifs and Trono original and Trono reprocessed motifs and the corresponding aligned motifs for the 39 overlapping KZNFs. H, Hughes; TO, Trono Original; TR, Trono Reprocessed; R, RCADE; M, MEME; a, all peaks; nE, nonERE peaks. (C) MoSBAT similarity e-scores for the 39 overlapping KZNFs between the Hughes data and Trono original (blue) and Trono reprocessed (red). The dots indicate the percentage of the overlap between the Hughes and Trono original peaks (blue) and Hughes and Trono reprocessed peaks (red).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/1/219/F4.medium.gif" width="322" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/8/1/219/F4.medium.gif" width="322" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/1/219/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/1/219/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/487907" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-26" class="first-child">Similarity between ChIP-derived motifs. (A) Similarity between the Hughes and Trono motifs for ZIM3. The heat map on the left indicates the MoSBAT similarity e-scores between each pair compared. The motifs and the motif-finding methods are represented on the right. H, Hughes; TO, Trono Original; TR, Trono Reprocessed. (B) The MoSBAT e-scores between Hughes motifs and Trono original and Trono reprocessed motifs and the corresponding aligned motifs for the 39 overlapping KZNFs. H, Hughes; TO, Trono Original; TR, Trono Reprocessed; R, RCADE; M, MEME; a, all peaks; nE, nonERE peaks. (C) MoSBAT similarity e-scores for the 39 overlapping KZNFs between the Hughes data and Trono original (blue) and Trono reprocessed (red). The dots indicate the percentage of the overlap between the Hughes and Trono original peaks (blue) and Hughes and Trono reprocessed peaks (red).</p><div class="sb-div caption-clear"></div></div></div><p id="p-27">We also compared the ChIP-derived motifs with those from other sources, taken from the initial publications or TF databases. If there was more than one motif from the same source available for a KZNF, the one with the highest AUROC was selected for further analysis (41 in total; <a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Matys <em>et al.</em> 2006</a>; <a id="xref-ref-7-3" class="xref-bibr" href="#ref-7">Encode-Project-Consortium 2012</a>; <a id="xref-ref-11-4" class="xref-bibr" href="#ref-11">Jolma <em>et al.</em> 2013</a>; <a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Kulakovskiy <em>et al.</em> 2013</a>; <a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Mathelier <em>et al.</em> 2016</a>; <a id="xref-ref-9-3" class="xref-bibr" href="#ref-9">Isakova <em>et al.</em> 2017</a>; <a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Yin <em>et al.</em> 2017</a>), and 5/22 of Hughes KZNFs, 14/22 of Trono KZNFs, and 3/22 of both data sets overlapped with at least one external motif (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300296/-/DC1/FigureS1.pptx">Figure S1</a>). For 23/41, the external in-vitro motif was similar (MoSBAT &gt;0.2) to at least one ChIP-derived motif. Surprisingly, 17 of the external motifs outperformed the ChIP-derived motifs in predicting ChIP peaks (see the section <em>A reference motif set for human KZNFs</em> for details), among which six belonged to the SMiLE-seq motif sets (seven in total; <a id="xref-ref-9-4" class="xref-bibr" href="#ref-9">Isakova <em>et al.</em> 2017</a>), six to the HT-SELEX motif sets (12 in total; <a id="xref-ref-11-5" class="xref-bibr" href="#ref-11">Jolma <em>et al.</em> 2013</a>; <a id="xref-ref-33-3" class="xref-bibr" href="#ref-33">Yin <em>et al.</em> 2017</a>), and five to one of the other sources (10 in total; <a id="xref-ref-22-3" class="xref-bibr" href="#ref-22">Matys <em>et al.</em> 2006</a>; <a id="xref-ref-7-4" class="xref-bibr" href="#ref-7">Encode-Project-Consortium 2012</a>; <a id="xref-ref-15-3" class="xref-bibr" href="#ref-15">Kulakovskiy <em>et al.</em> 2013</a>). This outcome not only confirms that motifs derived <em>in vitro</em> are often consistent with <em>in vivo</em> genomic binding sites, but illustrates that the greater depth of the assays can produce more accurate motifs. Some cases, however, are discrepant (11/41 cases score &lt;0.1 in MoSBAT compared to all corresponding ChIP-derived motifs), and will require resolution (see <em>Discussion</em>).</p><p id="p-28">We also found that the vast majority of motifs obtained from one of the ChIP data sets were significantly enriched in peaks from the other. <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a> depicts AUROC values (Top 500 <em>vs.</em> dinucleotide shuffled sequences) for ChIP data and other sources tested on the three data sets: Hughes, Trono original, and Trono reprocessed. The left third of <a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a> shows that, except for a few cases, the ChIP-derived motifs are comparable at predicting the ChIP peaks of the other data set, as well as the one from which they are derived. The right two-thirds of <a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5</a> show that motifs from other sources are also generally good predictors of the ChIP peaks.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/1/219/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="AUROC of the Hughes and Trono motifs and external motifs overlapping any of the two data sets. Heat map represents the AUROC value of each motif tested on Hughes, Trono original, or Trono reprocessed peaks. The first row at the top indicates the source of the motif, and the second row indicates the test data set. TO, Trono Original; TR, Trono Reprocessed; H, Hughes. White indicates no data is available. A full version of the figure that includes the KZNFs IDs is available at the web portal of the paper (http://kznfmotifs.ccbr.utoronto.ca/figures.html)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-426255469" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;AUROC of the Hughes and Trono motifs and external motifs overlapping any of the two data sets. Heat map represents the AUROC value of each motif tested on Hughes, Trono original, or Trono reprocessed peaks. The first row at the top indicates the source of the motif, and the second row indicates the test data set. TO, Trono Original; TR, Trono Reprocessed; H, Hughes. White indicates no data is available. A full version of the figure that includes the KZNFs IDs is available at the web portal of the paper (http://kznfmotifs.ccbr.utoronto.ca/figures.html).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/1/219/F5.medium.gif" width="335" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/8/1/219/F5.medium.gif" width="335" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/1/219/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/1/219/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/487861" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-29" class="first-child">AUROC of the Hughes and Trono motifs and external motifs overlapping any of the two data sets. Heat map represents the AUROC value of each motif tested on Hughes, Trono original, or Trono reprocessed peaks. The first row at the top indicates the source of the motif, and the second row indicates the test data set. TO, Trono Original; TR, Trono Reprocessed; H, Hughes. White indicates no data is available. A full version of the figure that includes the KZNFs IDs is available at the web portal of the paper (<a href="http://kznfmotifs.ccbr.utoronto.ca/figures.html">http://kznfmotifs.ccbr.utoronto.ca/figures.html</a>).</p><div class="sb-div caption-clear"></div></div></div><p id="p-30">These analyses show that the two data sets typically identify similar binding motifs, most of which are also supported by the recognition code and/or by independent data.</p></div><div id="sec-10" class="subsection"><h3>A reference motif set for human KZNFs</h3><p id="p-31">The comparisons above provide confidence metrics for the motifs obtained for each of the 242 KZNFs examined, representing reproducibility and predictive power both within and among data types, consistency with the zinc finger recognition code, and quality statistics for the data used to derive the motifs. We used these metrics to choose a single best current motif for each protein, as such a motif set is useful for many types of analyses (<a id="xref-ref-22-4" class="xref-bibr" href="#ref-22">Matys <em>et al.</em> 2006</a>; <a id="xref-ref-7-5" class="xref-bibr" href="#ref-7">Encode-Project-Consortium 2012</a>; <a id="xref-ref-15-4" class="xref-bibr" href="#ref-15">Kulakovskiy <em>et al.</em> 2013</a>; <a id="xref-ref-20-3" class="xref-bibr" href="#ref-20">Mathelier <em>et al.</em> 2016</a>).</p><p id="p-32">The ranking system, intended to capture motifs that correspond to both ChIP data and external information, if available, is shown in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300296/-/DC1/FigureS2.pptx">Figure S2</a>. We gave highest confidence to any motif (<em>in vitro</em> or ChIP-derived) that predicts “test” ChIP data (the data that they were not trained on: one of the Hughes, Trono original, or Trono reprocessed data sets) with AUROC at least 0.1 greater than all other motifs (Motif is uniquely predictive of test peaks = Class A). In the absence of a Class A motif, we then favored <em>in vitro</em> derived motifs that predict the test ChIP data better than, or almost as well as, the ChIP motifs (AUROC difference &lt;0.1; <em>In vitro</em> motif predictive of test peaks = Class B), since <em>in vitro</em> data are not impacted by extraneous factors such as indirect binding. If no motifs satisfied these criteria, we selected ChIP motifs that are supported by the recognition code (<em>i.e.</em>, RCADE) (RCADE motifs = Class C), followed by the ChIP motifs that predict their training data better than all other motifs, where the AUROC was at least 0.1 greater than the others (ChIP motif is uniquely predictive of training peaks = Class D), followed by those that predict the test ChIP data sets even slightly better than the others (AUROC = 0.01–0.09 greater than the others) (Motif is most predictive of test peaks = Class E), and, finally, all remaining motifs including single motifs and motifs for three KZNFs (ZIM2, ZNF445 and ZNF785) that cannot be discriminated by any of the above criteria (Others = Class F).</p><p id="p-33"><a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6A</a> shows the number and proportion of KZNFs currently falling into each class, and <a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6B</a> shows the motifs. In this scheme, the largest classes of motifs are Class A (in which one motif clearly outperforms all others on test data), and Class E (where all motifs are roughly equivalent). The median AUROC for each class does not vary greatly (<a id="xref-fig-6-3" class="xref-fig" href="#F6">Figure 6A</a>). Most of the motifs are supported by the recognition code: 158/242 (65%) are RCADE motifs, 96% of which have AUC &gt;0.6 predicting peak <em>vs.</em> nonpeak. As previously reported for KZNFs, these reference motifs are highly diverse, and tend to be longer and more information-rich than motifs for other TF types, which tend to be only 6–12 bases long (<a id="xref-ref-1-3" class="xref-bibr" href="#ref-1">Badis <em>et al.</em> 2009</a>; <a id="xref-ref-11-6" class="xref-bibr" href="#ref-11">Jolma <em>et al.</em> 2013</a>).</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/1/219/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The reference motif set for the 242 KZNFs. (A) Percentage and number of the motifs (in parentheses) fit into classes A&#x2013;F and the median AUROC values of each group. (B) The reference motif for each of the 242 KZNFs. Source refers to motif origin (TO, Trono Original; TR, Trono Reprocessed; H, Hughes; Naj, ChIP-seq (Najafabadi et al. 2015b); SM, SMiLE-seq (Isakova et al. 2017); SelY, HT-SELEX (Yin et al. 2017); MSelY, Methyl-HT-SELEX (Yin et al. 2017); SelJ, HT-SELEX (Jolma et al. 2013); EN, ENCODE; Trans, TRANSFAC; HM, HocoMoco). The class is the selection class that each motif falls into. For ZIM2, ZNF445 and ZNF785, both motifs from class F are represented." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-426255469" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The reference motif set for the 242 KZNFs. (A) Percentage and number of the motifs (in parentheses) fit into classes A&#x2013;F and the median AUROC values of each group. (B) The reference motif for each of the 242 KZNFs. Source refers to motif origin (TO, Trono Original; TR, Trono Reprocessed; H, Hughes; Naj, ChIP-seq (Najafabadi &lt;em&gt;et al.&lt;/em&gt; 2015b); SM, SMiLE-seq (Isakova &lt;em&gt;et al.&lt;/em&gt; 2017); SelY, HT-SELEX (Yin &lt;em&gt;et al.&lt;/em&gt; 2017); MSelY, Methyl-HT-SELEX (Yin &lt;em&gt;et al.&lt;/em&gt; 2017); SelJ, HT-SELEX (Jolma &lt;em&gt;et al.&lt;/em&gt; 2013); EN, ENCODE; Trans, TRANSFAC; HM, HocoMoco). The class is the selection class that each motif falls into. For ZIM2, ZNF445 and ZNF785, both motifs from class F are represented.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/1/219/F6.medium.gif" width="312" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/8/1/219/F6.medium.gif" width="312" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/1/219/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/1/219/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/487899" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-34" class="first-child">The reference motif set for the 242 KZNFs. (A) Percentage and number of the motifs (in parentheses) fit into classes A–F and the median AUROC values of each group. (B) The reference motif for each of the 242 KZNFs. Source refers to motif origin (TO, Trono Original; TR, Trono Reprocessed; H, Hughes; Naj, ChIP-seq (<a id="xref-ref-24-6" class="xref-bibr" href="#ref-24">Najafabadi <em>et al.</em> 2015b</a>); SM, SMiLE-seq (<a id="xref-ref-9-5" class="xref-bibr" href="#ref-9">Isakova <em>et al.</em> 2017</a>); SelY, HT-SELEX (<a id="xref-ref-33-4" class="xref-bibr" href="#ref-33">Yin <em>et al.</em> 2017</a>); MSelY, Methyl-HT-SELEX (<a id="xref-ref-33-5" class="xref-bibr" href="#ref-33">Yin <em>et al.</em> 2017</a>); SelJ, HT-SELEX (<a id="xref-ref-11-7" class="xref-bibr" href="#ref-11">Jolma <em>et al.</em> 2013</a>); EN, ENCODE; Trans, TRANSFAC; HM, HocoMoco). The class is the selection class that each motif falls into. For ZIM2, ZNF445 and ZNF785, both motifs from class F are represented.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-11" class="subsection"><h3>Web portal for KZNFs</h3><p id="p-35">Finally, we generated a web interface that assembles all the data described herein, providing all the relevant data files, as well as a graphical interface for each KZNF. <a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7</a> provides an illustration for ZNF549, a KZNF with a Class A motif. Motifs from different sources are shown, if available (<a id="xref-fig-7-2" class="xref-fig" href="#F7">Figure 7A</a>). A heatmap of the pairwise motif similarity scores between different motifs is then shown (<a id="xref-fig-7-3" class="xref-fig" href="#F7">Figure 7B</a>). The peak overlap between the Hughes and Trono ChIP data are illustrated, where available, for all peaks and top 500 peaks (<a id="xref-fig-7-4" class="xref-fig" href="#F7">Figure 7C</a>). Finally, the ERE enrichment of the peaks is given (<a id="xref-fig-7-5" class="xref-fig" href="#F7">Figure 7D</a>).</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/1/219/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Web portal of ZNF549 containing all the analyses described (http://kznfmotifs.ccbr.utoronto.ca/report.php?name=ZNF549). (A) Motifs for the same KZNF derived from different sources. (B) MoSBAT similarity heat maps between all motifs. (C) Overlap between Hughes peaks and Trono reprocessed (left) and Trono original (right) peaks for all peaks and top 500 peaks. (D) ERE enrichment for the Hughes and Trono ChIP peaks." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-426255469" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Web portal of ZNF549 containing all the analyses described (http://kznfmotifs.ccbr.utoronto.ca/report.php?name=ZNF549). (A) Motifs for the same KZNF derived from different sources. (B) MoSBAT similarity heat maps between all motifs. (C) Overlap between Hughes peaks and Trono reprocessed (left) and Trono original (right) peaks for all peaks and top 500 peaks. (D) ERE enrichment for the Hughes and Trono ChIP peaks.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/1/219/F7.medium.gif" width="395" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/8/1/219/F7.medium.gif" width="395" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/1/219/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/1/219/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/487908" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-36" class="first-child">Web portal of ZNF549 containing all the analyses described (<a href="http://kznfmotifs.ccbr.utoronto.ca/report.php?name=ZNF549">http://kznfmotifs.ccbr.utoronto.ca/report.php?name=ZNF549</a>). (A) Motifs for the same KZNF derived from different sources. (B) MoSBAT similarity heat maps between all motifs. (C) Overlap between Hughes peaks and Trono reprocessed (left) and Trono original (right) peaks for all peaks and top 500 peaks. (D) ERE enrichment for the Hughes and Trono ChIP peaks.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-12"><h2 class="">Discussion</h2><p id="p-37">The overall conclusion of these analyses is that there is reasonable agreement between the Hughes and Trono lab ChIP data, and also external data, on the motifs, genomic binding sites in HEK293 cells, and EREs enriched among those sites. This finding enables us to conclude that ChIP-seq—despite being a notoriously difficult method from which to derive motifs and to predict primary regulatory “targets”—is a viable strategy for both obtaining motifs and for identifying a potential physiological role for these proteins, or at least an initial driving evolutionary force (repression of EREs). These analyses also indicate that most human KZNFs are <em>bona fide</em> sequence-specific DNA-binding proteins.</p><p id="p-38">Despite general overall agreement, there are discrepancies between the data sets. When considering both Trono original and Trono reprocessed data together, 23% of the experiments disagree on at least one of these parameters (EREs bound, <em>r</em> &lt; 0.3 and motifs, <em>r</em> &lt; 0.2); 3/39 (8%) of the experiments disagree on both. Because disagreement can stem from noise or error in only one of the two data sets, it is possible that these issues are less prevalent in the individual data sets (<em>e.g.</em>, 11.5% error in each of them would lead to an overall error of 23%). KZNFs for which the data sets disagree, and for which there are no motifs that relate to the recognition code (44/242), might be considered prime candidates for rerunning the ChIP assays, possibly in other cell types, and/or analysis by alternative assays such as SMILE-seq. Ideally, an <em>in vitro</em>-derived motif for each protein will eventually also be available, enabling confirmation that the motifs enriched in ChIP data represent <em>bona fide</em> direct sequence recognition events. In the meantime, the recognition code appears to provide a reasonable substitute.</p><p id="p-39">The absence of a complete set of motifs for human TFs (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Weirauch <em>et al.</em> 2014</a>) remains a glaring shortcoming in the study of human gene regulation. Our collection of reference motifs includes 166 proteins for which there was previously no known motif. We anticipate that the reference set will be of general utility in the study of regulatory mechanisms and networks. It will also facilitate exploration of the targeting mechanisms of these proteins to EREs—the motifs alone are largely insufficient (<a id="xref-ref-24-7" class="xref-bibr" href="#ref-24">Najafabadi <em>et al.</em> 2015b</a>; <a id="xref-ref-27-9" class="xref-bibr" href="#ref-27">Schmitges <em>et al.</em> 2016</a>)—as well as their roles in human genetics and disease, because SNPs in noncoding regions often impact motif scores for TFs (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Deplancke <em>et al.</em> 2016</a>).</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-40">We are grateful to Hamed Najafabadi, Laura Campitelli, Arttu Jolma, and Matt Weirauch for input on the methodology and web site, and grants from the Canadian Institutes of Health Research (FDN-148403) and the National Science and Engineering Research Council (NSERC) (RGPIN-2016-05643) for financial support. M.B. was supported in part by a C.H. Best Fellowship. S.L. is supported by a doctoral scholarship from NSERC.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-41">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300296/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300296/-/DC1</a>.</p></li><li class="fn" id="fn-2"><p id="p-42"><em>Communicating editor: D. Gresham</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2017-09-22"><span class="received-label">Received </span>September 22, 2017.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2017-11-08"><span class="accepted-label">Accepted </span>November 8, 2017.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2018 Barazandeh <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.219.1" data-doi="10.1126/science.1162327"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Badis</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berger</span>  <span class="cit-name-given-names">M. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Philippakis</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Talukder</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gehrke</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Diversity and complexity in DNA recognition by transcription factors.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">324</span>: <span class="cit-fpage">1720</span>–<span class="cit-lpage">1723</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1162327%26rft_id%253Dinfo%253Apmid%252F19443739%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMjQvNTkzNS8xNzIwIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8xLzIxOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.219.2" data-doi="10.1093/nar/gkp335"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bailey</span>  <span class="cit-name-given-names">T. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boden</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buske</span>  <span class="cit-name-given-names">F. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Frith</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grant</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">MEME SUITE: tools for motif discovery and searching.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">W202</span>–<span class="cit-lpage">W208</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkp335%26rft_id%253Dinfo%253Apmid%252F19458158%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkp335&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19458158&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F219.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000267889100037&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.219.3" data-doi="10.1007/s12013-008-9008-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brayer</span>  <span class="cit-name-given-names">K. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Segal</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Keep your fingers off my DNA: protein-protein interactions mediated by C2H2 zinc finger domains.</span> <abbr class="cit-jnl-abbrev">Cell Biochem Biophys</abbr> <span class="cit-vol">50</span>: <span class="cit-fpage">111</span>–<span class="cit-lpage">131</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2Bbiochemistry%2Band%2Bbiophysics%26rft.stitle%253DCell%2BBiochem%2BBiophys%26rft.aulast%253DBrayer%26rft.auinit1%253DK.%2BJ.%26rft.volume%253D50%26rft.issue%253D3%26rft.spage%253D111%26rft.epage%253D131%26rft.atitle%253DKeep%2Byour%2Bfingers%2Boff%2Bmy%2BDNA%253A%2Bprotein-protein%2Binteractions%2Bmediated%2Bby%2BC2H2%2Bzinc%2Bfinger%2Bdomains.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs12013-008-9008-5%26rft_id%253Dinfo%253Apmid%252F18253864%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s12013-008-9008-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18253864&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F219.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000253765000001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.219.4" data-doi="10.1186/gb-2010-11-6-r69"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Day</span>  <span class="cit-name-given-names">D. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luquette</span>  <span class="cit-name-given-names">L. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Park</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kharchenko</span>  <span class="cit-name-given-names">P. V.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Estimating enrichment of repetitive elements from high-throughput sequence data.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">R69</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DDay%26rft.auinit1%253DD.%2BS.%26rft.volume%253D11%26rft.issue%253D6%26rft.spage%253DR69%26rft.epage%253DR69%26rft.atitle%253DEstimating%2Benrichment%2Bof%2Brepetitive%2Belements%2Bfrom%2Bhigh-throughput%2Bsequence%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2010-11-6-r69%26rft_id%253Dinfo%253Apmid%252F20584328%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2010-11-6-r69&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20584328&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F219.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.219.5" data-doi="10.1016/j.cell.2016.07.012"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Deplancke</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alpern</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gardeux</span>  <span class="cit-name-given-names">V.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">The genetics of transcription factor DNA binding variation.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">166</span>: <span class="cit-fpage">538</span>–<span class="cit-lpage">554</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D166%26rft.spage%253D538%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2016.07.012%26rft_id%253Dinfo%253Apmid%252F27471964%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2016.07.012&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27471964&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F219.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.219.6" data-doi="10.1371/journal.pgen.1000325"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Emerson</span>  <span class="cit-name-given-names">R. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thomas</span>  <span class="cit-name-given-names">J. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Adaptive evolution in zinc finger transcription factors.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e1000325</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DEmerson%26rft.auinit1%253DR.%2BO.%26rft.volume%253D5%26rft.issue%253D1%26rft.spage%253De1000325%26rft.epage%253De1000325%26rft.atitle%253DAdaptive%2Bevolution%2Bin%2Bzinc%2Bfinger%2Btranscription%2Bfactors.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000325%26rft_id%253Dinfo%253Apmid%252F19119423%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000325&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19119423&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F219.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.219.7" data-doi="10.1038/nature11247"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">Encode-Project-Consortium</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">An integrated encyclopedia of DNA elements in the human genome.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">489</span>: <span class="cit-fpage">57</span>–<span class="cit-lpage">74</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DBernstein%26rft.auinit1%253DB.%2BE.%26rft.volume%253D489%26rft.issue%253D7414%26rft.spage%253D57%26rft.epage%253D74%26rft.atitle%253DAn%2Bintegrated%2Bencyclopedia%2Bof%2BDNA%2Belements%2Bin%2Bthe%2Bhuman%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature11247%26rft_id%253Dinfo%253Apmid%252F22955616%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature11247&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22955616&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F219.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000308347000039&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.219.8" data-doi="10.1038/nature21683"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Imbeault</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Helleboid</span>  <span class="cit-name-given-names">P. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trono</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">KRAB zinc-finger proteins contribute to the evolution of gene regulatory networks.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">543</span>: <span class="cit-fpage">550</span>–<span class="cit-lpage">554</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D543%26rft.spage%253D550%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature21683%26rft_id%253Dinfo%253Apmid%252F28273063%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature21683&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28273063&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F219.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.219.9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Isakova</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Groux</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Imbeault</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rainer</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alpern</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">SMiLE-seq identifies binding motifs of single and dimeric transcription factors.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">316</span>–<span class="cit-lpage">322</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BMethods%26rft.volume%253D14%26rft.spage%253D316%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.219.10" data-doi="10.1038/nature13760"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jacobs</span>  <span class="cit-name-given-names">F. M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Greenberg</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nguyen</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haeussler</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ewing</span>  <span class="cit-name-given-names">A. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">An evolutionary arms race between KRAB zinc-finger genes ZNF91/93 and SVA/L1 retrotransposons.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">516</span>: <span class="cit-fpage">242</span>–<span class="cit-lpage">245</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D516%26rft.spage%253D242%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature13760%26rft_id%253Dinfo%253Apmid%252F25274305%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature13760&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25274305&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F219.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000346383500044&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.219.11" data-doi="10.1016/j.cell.2012.12.009"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jolma</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Whitington</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Toivonen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nitta</span>  <span class="cit-name-given-names">K. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">DNA-binding specificities of human transcription factors.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">152</span>: <span class="cit-fpage">327</span>–<span class="cit-lpage">339</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D152%26rft.spage%253D327%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2012.12.009%26rft_id%253Dinfo%253Apmid%252F23332764%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2012.12.009&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23332764&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F219.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000313719800027&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.219.12" data-doi="10.1038/nature15518"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jolma</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yin</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nitta</span>  <span class="cit-name-given-names">K. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dave</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Popov</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">DNA-dependent formation of transcription factor pairs alters their binding specificity.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">527</span>: <span class="cit-fpage">384</span>–<span class="cit-lpage">388</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D527%26rft.spage%253D384%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature15518%26rft_id%253Dinfo%253Apmid%252F26550823%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature15518&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26550823&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F219.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.219.13" data-doi="10.1093/nar/gkh103"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Karolchik</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hinrichs</span>  <span class="cit-name-given-names">A. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Furey</span>  <span class="cit-name-given-names">T. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roskin</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sugnet</span>  <span class="cit-name-given-names">C. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">The UCSC Table Browser data retrieval tool.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">D493</span>–<span class="cit-lpage">D496</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DKarolchik%26rft.auinit1%253DD.%26rft.volume%253D32%26rft.issue%253D90001%26rft.spage%253DD493%26rft.epage%253D496%26rft.atitle%253DThe%2BUCSC%2BTable%2BBrowser%2Bdata%2Bretrieval%2Btool%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkh103%26rft_id%253Dinfo%253Apmid%252F14681465%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkh103&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14681465&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F219.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000188079000117&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.219.14" data-doi="10.1038/nbt.1508"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kharchenko</span>  <span class="cit-name-given-names">P. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tolstorukov</span>  <span class="cit-name-given-names">M. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Park</span>  <span class="cit-name-given-names">P. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Design and analysis of ChIP-seq experiments for DNA-binding proteins.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">1351</span>–<span class="cit-lpage">1359</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DKharchenko%26rft.auinit1%253DP.%2BV.%26rft.volume%253D26%26rft.issue%253D12%26rft.spage%253D1351%26rft.epage%253D1359%26rft.atitle%253DDesign%2Band%2Banalysis%2Bof%2BChIP-seq%2Bexperiments%2Bfor%2BDNA-binding%2Bproteins.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.1508%26rft_id%253Dinfo%253Apmid%252F19029915%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.1508&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19029915&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F219.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000261591300019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.219.15" data-doi="10.1093/nar/gks1089"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kulakovskiy</span>  <span class="cit-name-given-names">I. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Medvedeva</span>  <span class="cit-name-given-names">Y. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schaefer</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kasianov</span>  <span class="cit-name-given-names">A. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vorontsov</span>  <span class="cit-name-given-names">I. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">HOCOMOCO: a comprehensive collection of human transcription factor binding sites models.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">D195</span>–<span class="cit-lpage">D202</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgks1089%26rft_id%253Dinfo%253Apmid%252F23175603%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gks1089&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23175603&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F219.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000312893300027&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.219.16" data-doi="10.1093/bioinformatics/btw489"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lambert</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Albu</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hughes</span>  <span class="cit-name-given-names">T. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Najafabadi</span>  <span class="cit-name-given-names">H. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Motif comparison based on similarity of binding affinity profiles.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">3504</span>–<span class="cit-lpage">3506</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtw489%26rft_id%253Dinfo%253Apmid%252F27466627%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btw489&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27466627&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F219.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.219.17" data-doi="10.1101/gr.136184.111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Landt</span>  <span class="cit-name-given-names">S. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marinov</span>  <span class="cit-name-given-names">G. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kundaje</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kheradpour</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pauli</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">ChIP-seq guidelines and practices of the ENCODE and modENCODE consortia.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">1813</span>–<span class="cit-lpage">1831</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DLandt%26rft.auinit1%253DS.%2BG.%26rft.volume%253D22%26rft.issue%253D9%26rft.spage%253D1813%26rft.epage%253D1831%26rft.atitle%253DChIP-seq%2Bguidelines%2Band%2Bpractices%2Bof%2Bthe%2BENCODE%2Band%2BmodENCODE%2Bconsortia%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.136184.111%26rft_id%253Dinfo%253Apmid%252F22955991%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjIyLzkvMTgxMyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzgvMS8yMTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.219.18" data-doi="10.1038/nmeth.1923"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Langmead</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salzberg</span>  <span class="cit-name-given-names">S. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Fast gapped-read alignment with Bowtie 2.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">357</span>–<span class="cit-lpage">359</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DLangmead%26rft.auinit1%253DB.%26rft.volume%253D9%26rft.issue%253D4%26rft.spage%253D357%26rft.epage%253D359%26rft.atitle%253DFast%2Bgapped-read%2Balignment%2Bwith%2BBowtie%2B2.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1923%26rft_id%253Dinfo%253Apmid%252F22388286%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1923&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22388286&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F219.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000302218500017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.219.19" data-doi="10.1371/journal.pcbi.1003214"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mathelier</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wasserman</span>  <span class="cit-name-given-names">W. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The next generation of transcription factor binding site prediction.</span> <abbr class="cit-jnl-abbrev">PLOS Comput. Biol.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e1003214</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLOS%2BComput.%2BBiol.%26rft.volume%253D9%26rft.spage%253De1003214%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.1003214%26rft_id%253Dinfo%253Apmid%252F24039567%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.1003214&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24039567&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F219.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.219.20" data-doi="10.1093/nar/gkv1176"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mathelier</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fornes</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arenillas</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">C. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Denay</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">JASPAR 2016: a major expansion and update of the open-access database of transcription factor binding profiles.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">D110</span>–<span class="cit-lpage">D115</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkv1176%26rft_id%253Dinfo%253Apmid%252F26531826%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkv1176&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26531826&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F219.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.219.21" data-doi="10.1038/nature08858"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Matsui</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leung</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miyashita</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maksakova</span>  <span class="cit-name-given-names">I. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miyachi</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Proviral silencing in embryonic stem cells requires the histone methyltransferase ESET.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">464</span>: <span class="cit-fpage">927</span>–<span class="cit-lpage">931</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DMatsui%26rft.auinit1%253DT.%26rft.volume%253D464%26rft.issue%253D7290%26rft.spage%253D927%26rft.epage%253D931%26rft.atitle%253DProviral%2Bsilencing%2Bin%2Bembryonic%2Bstem%2Bcells%2Brequires%2Bthe%2Bhistone%2Bmethyltransferase%2BESET.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature08858%26rft_id%253Dinfo%253Apmid%252F20164836%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature08858&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20164836&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F219.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000276397300045&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.219.22" data-doi="10.1093/nar/gkj143"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Matys</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kel-Margoulis</span>  <span class="cit-name-given-names">O. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fricke</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liebich</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Land</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">TRANSFAC and its module TRANSCompel: transcriptional gene regulation in eukaryotes.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">D108</span>–<span class="cit-lpage">D110</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DMatys%26rft.auinit1%253DV.%26rft.volume%253D34%26rft.issue%253Dsuppl_1%26rft.spage%253DD108%26rft.epage%253DD110%26rft.atitle%253DTRANSFAC%2528R%2529%2Band%2Bits%2Bmodule%2BTRANSCompel%2528R%2529%253A%2Btranscriptional%2Bgene%2Bregulation%2Bin%2Beukaryotes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkj143%26rft_id%253Dinfo%253Apmid%252F16381825%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkj143&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16381825&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F219.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000239307700023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.219.23" data-doi="10.1093/bioinformatics/btv284"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Najafabadi</span>  <span class="cit-name-given-names">H. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Albu</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hughes</span>  <span class="cit-name-given-names">T. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span><span class="cit-comment">a</span> <span class="cit-article-title">Identification of C2H2-ZF binding preferences from ChIP-seq data using RCADE.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">2879</span>–<span class="cit-lpage">2881</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtv284%26rft_id%253Dinfo%253Apmid%252F25953800%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btv284&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25953800&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F219.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.219.24" data-doi="10.1038/nbt.3128"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Najafabadi</span>  <span class="cit-name-given-names">H. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mnaimneh</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schmitges</span>  <span class="cit-name-given-names">F. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garton</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lam</span>  <span class="cit-name-given-names">K. N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span><span class="cit-comment">b</span> <span class="cit-article-title">C2H2 zinc finger proteins greatly expand the human regulatory lexicon.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">555</span>–<span class="cit-lpage">562</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BBiotechnol.%26rft.volume%253D33%26rft.spage%253D555%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.3128%26rft_id%253Dinfo%253Apmid%252F25690854%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.3128&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25690854&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F219.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.219.25" data-doi="10.1093/nar/gkn048"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Noyes</span>  <span class="cit-name-given-names">M. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meng</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wakabayashi</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sinha</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brodsky</span>  <span class="cit-name-given-names">M. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">A systematic characterization of factors that regulate <em>Drosophila</em> segmentation via a bacterial one-hybrid system.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">2547</span>–<span class="cit-lpage">2560</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkn048%26rft_id%253Dinfo%253Apmid%252F18332042%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkn048&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18332042&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F219.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000255759600011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.219.26" data-doi="10.1016/j.virol.2010.12.007"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rowe</span>  <span class="cit-name-given-names">H. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trono</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Dynamic control of endogenous retroviruses during development.</span> <abbr class="cit-jnl-abbrev">Virology</abbr> <span class="cit-vol">411</span>: <span class="cit-fpage">273</span>–<span class="cit-lpage">287</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirology%26rft.volume%253D411%26rft.spage%253D273%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.virol.2010.12.007%26rft_id%253Dinfo%253Apmid%252F21251689%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.virol.2010.12.007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21251689&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F219.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000288343000013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.219.27" data-doi="10.1101/gr.209643.116"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schmitges</span>  <span class="cit-name-given-names">F. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Radovani</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Najafabadi</span>  <span class="cit-name-given-names">H. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barazandeh</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Campitelli</span>  <span class="cit-name-given-names">L. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Multiparameter functional diversity of human C2H2 zinc finger proteins.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">1742</span>–<span class="cit-lpage">1752</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.209643.116%26rft_id%253Dinfo%253Apmid%252F27852650%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIyNi8xMi8xNzQyIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8xLzIxOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.219.28" data-doi="10.1101/gad.973302"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schultz</span>  <span class="cit-name-given-names">D. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ayyanathan</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Negorev</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maul</span>  <span class="cit-name-given-names">G. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rauscher</span>  <span class="cit-name-given-names">F. J.</span>  <span class="cit-name-suffix">III</span>.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">SETDB1: a novel KAP-1-associated histone H3, lysine 9-specific methyltransferase that contributes to HP1-mediated silencing of euchromatic genes by KRAB zinc-finger proteins.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">919</span>–<span class="cit-lpage">932</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DSchultz%26rft.auinit1%253DD.%2BC.%26rft.volume%253D16%26rft.issue%253D8%26rft.spage%253D919%26rft.epage%253D932%26rft.atitle%253DSETDB1%253A%2Ba%2Bnovel%2BKAP-1-associated%2Bhistone%2BH3%252C%2Blysine%2B9-specific%2Bmethyltransferase%2Bthat%2Bcontributes%2Bto%2BHP1-mediated%2Bsilencing%2Bof%2Beuchromatic%2Bgenes%2Bby%2BKRAB%2Bzinc-finger%2Bproteins%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.973302%26rft_id%253Dinfo%253Apmid%252F11959841%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6ODoiMTYvOC85MTkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy84LzEvMjE5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.219.29" data-doi="10.1186/1471-2105-15-289"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Quality <em>vs.</em> accuracy: result of a reanalysis of protein-binding microarrays from the DREAM5 challenge by using BayesPI2 including dinucleotide interdependence.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">289</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BBioinformatics%26rft.volume%253D15%26rft.spage%253D289%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-15-289%26rft_id%253Dinfo%253Apmid%252F25158938%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-15-289&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25158938&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F219.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.219.30" data-doi="10.1038/nbt.2486"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Weirauch</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cote</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Norel</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Annala</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Evaluation of methods for modeling transcription factor sequence specificity.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">126</span>–<span class="cit-lpage">134</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BBiotechnol.%26rft.volume%253D31%26rft.spage%253D126%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.2486%26rft_id%253Dinfo%253Apmid%252F23354101%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.2486&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23354101&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F219.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.219.31" data-doi="10.1016/j.cell.2014.08.009"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Weirauch</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Albu</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cote</span>  <span class="cit-name-given-names">A. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Montenegro-Montero</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Determination and inference of eukaryotic transcription factor sequence specificity.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">158</span>: <span class="cit-fpage">1431</span>–<span class="cit-lpage">1443</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D158%26rft.spage%253D1431%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2014.08.009%26rft_id%253Dinfo%253Apmid%252F25215497%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2014.08.009&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25215497&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F219.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000343094200022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.219.32" data-doi="10.1146/annurev.biophys.29.1.183"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wolfe</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nekludova</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pabo</span>  <span class="cit-name-given-names">C. O.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">DNA recognition by Cys2His2 zinc finger proteins.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Biophys. Biomol. Struct.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">183</span>–<span class="cit-lpage">212</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bbiophysics%2Band%2Bbiomolecular%2Bstructure%26rft.stitle%253DAnnu%2BRev%2BBiophys%2BBiomol%2BStruct%26rft.volume%253D29%26rft.spage%253D183%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.biophys.29.1.183%26rft_id%253Dinfo%253Apmid%252F10940247%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.biophys.29.1.183&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10940247&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F219.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000088492300007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.219.33"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yin</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morgunova</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jolma</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kaasinen</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sahu</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Impact of cytosine methylation on DNA binding specificities of human transcription factors.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">356</span>: pii: eaaj2239.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.219.34" data-doi="10.1186/gb-2008-9-9-r137"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meyer</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eeckhoute</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">D. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Model-based analysis of ChIP-Seq (MACS).</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">R137</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DZhang%26rft.auinit1%253DY.%26rft.volume%253D9%26rft.issue%253D9%26rft.spage%253DR137%26rft.epage%253DR137%26rft.atitle%253DModel-based%2Banalysis%2Bof%2BChIP-Seq%2B%2528MACS%2529.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2008-9-9-r137%26rft_id%253Dinfo%253Apmid%252F18798982%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2008-9-9-r137&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18798982&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F219.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.219.35" data-doi="10.1371/journal.pcbi.1000590"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Granas</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stormo</span>  <span class="cit-name-given-names">G. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Inferring binding energies from selected binding sites.</span> <abbr class="cit-jnl-abbrev">PLOS Comput. Biol.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e1000590</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BComput%2BBiol%26rft.aulast%253DZhao%26rft.auinit1%253DY.%26rft.volume%253D5%26rft.issue%253D12%26rft.spage%253De1000590%26rft.epage%253De1000590%26rft.atitle%253DInferring%2Bbinding%2Benergies%2Bfrom%2Bselected%2Bbinding%2Bsites.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.1000590%26rft_id%253Dinfo%253Apmid%252F19997485%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.1000590&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19997485&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F219.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/8/1/219.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/1/207" title="Global Transcriptome Sequencing Reveals Molecular Profiles of Summer Diapause Induction Stage of Onion Maggot, Delia antiqua (Diptera: Anthomyiidae)" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/1/231" title="Egg Case Silk Gene Sequences from Argiope Spiders: Evidence for Multiple Loci and a Loss of Function Between Paralogs" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/8/1"><span>Volume 8 Issue 1, January 2018</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/8/1.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/8/1.cover-source.jpg?itok=vyoqxep2" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 8 (1)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/1/219" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-KI1J1XXenZ3usozj-J8z6j6yfTpf3IPkfCuKZCBQnic" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/8/1/219" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/8/1/219" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/8/1/219" class="active" data-icon-position="" data-hide-link-title="0">Comparison of ChIP-Seq Data and a Reference Motif Set for Human KRAB C2H2 Zinc Finger Proteins</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/487830" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-DwUAIVaijZ1Nqkx0zaytRf1pu_61XRTAs7x_auRqPRE" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/1/219.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/8/1/219" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-J3NftoC2pFa7Ld5AzEBTWh6ID41t2N7HiFJ5wzSwK0w" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/8/1/219" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/1/219.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="487830" id="node487830--2" data-pisa="ggg;8/1/219" data-pisa-master="ggg;g3.117.300296" data-apath="/ggg/8/1/219.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Comparison of ChIP-Seq Data and a Reference Motif Set for Human KRAB C2H2 Zinc Finger Proteins</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Marjan Barazandeh</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-8222-008X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Samuel A. Lambert</span>, <span class="highwire-citation-author" data-delta="2">Mihai Albu</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-8721-4719" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Timothy R. Hughes</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">219-229;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300296 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node487830--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Marjan Barazandeh </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Terrence Donnelly Centre for Cellular and Biomolecular Research, University of Toronto, Ontario M5S 1A8, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marjan%2BBarazandeh%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Barazandeh%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMarjan%2BBarazandeh%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Samuel A. Lambert </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Molecular Genetics, University of Toronto, Ontario M5S 1A8, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Samuel%2BA.%2BLambert%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lambert%20SA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASamuel%2BA.%2BLambert%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-8222-008X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Samuel A. Lambert</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Mihai Albu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Terrence Donnelly Centre for Cellular and Biomolecular Research, University of Toronto, Ontario M5S 1A8, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mihai%2BAlbu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Albu%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMihai%2BAlbu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Timothy R. Hughes </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Terrence Donnelly Centre for Cellular and Biomolecular Research, University of Toronto, Ontario M5S 1A8, Canada</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Molecular Genetics, University of Toronto, Ontario M5S 1A8, Canada</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Canadian Institutes for Advanced Research, Toronto, Ontario M5G 1M1, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Timothy%2BR.%2BHughes%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hughes%20TR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ATimothy%2BR.%2BHughes%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-8721-4719" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Timothy R. Hughes</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:t.hughes@utoronto.ca" class="" data-icon-position="" data-hide-link-title="0">t.hughes@utoronto.ca</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/8/1/219" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/8/1/219&amp;title=Comparison%20of%20ChIP-Seq%20Data%20and%20a%20Reference%20Motif%20Set%20for%20Human%20KRAB%20C2H2%20Zinc%20Finger%20Proteins" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/8/1/219&amp;title=Comparison%20of%20ChIP-Seq%20Data%20and%20a%20Reference%20Motif%20Set%20for%20Human%20KRAB%20C2H2%20Zinc%20Finger%20Proteins" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/8/1/219&amp;title=Comparison%20of%20ChIP-Seq%20Data%20and%20a%20Reference%20Motif%20Set%20for%20Human%20KRAB%20C2H2%20Zinc%20Finger%20Proteins" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/8/1/219&amp;text=Comparison%20of%20ChIP-Seq%20Data%20and%20a%20Reference%20Motif%20Set%20for%20Human%20KRAB%20C2H2%20Zinc%20Finger%20Proteins" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/8/1/219&amp;title=Comparison%20of%20ChIP-Seq%20Data%20and%20a%20Reference%20Motif%20Set%20for%20Human%20KRAB%20C2H2%20Zinc%20Finger%20Proteins" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/8/1/219&amp;t=Comparison%20of%20ChIP-Seq%20Data%20and%20a%20Reference%20Motif%20Set%20for%20Human%20KRAB%20C2H2%20Zinc%20Finger%20Proteins" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/8/1/219&amp;title=Comparison%20of%20ChIP-Seq%20Data%20and%20a%20Reference%20Motif%20Set%20for%20Human%20KRAB%20C2H2%20Zinc%20Finger%20Proteins" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/8/1/219&amp;title=Comparison%20of%20ChIP-Seq%20Data%20and%20a%20Reference%20Motif%20Set%20for%20Human%20KRAB%20C2H2%20Zinc%20Finger%20Proteins" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/487830/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="487830" id="node487830--3" data-pisa="ggg;8/1/219" data-pisa-master="ggg;g3.117.300296" data-apath="/ggg/8/1/219.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Comparison of ChIP-Seq Data and a Reference Motif Set for Human KRAB C2H2 Zinc Finger Proteins</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Marjan Barazandeh</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-8222-008X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Samuel A. Lambert</span>, <span class="highwire-citation-author" data-delta="2">Mihai Albu</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-8721-4719" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Timothy R. Hughes</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">219-229;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300296 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node487830--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Marjan Barazandeh </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Terrence Donnelly Centre for Cellular and Biomolecular Research, University of Toronto, Ontario M5S 1A8, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marjan%2BBarazandeh%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Barazandeh%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMarjan%2BBarazandeh%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Samuel A. Lambert </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Molecular Genetics, University of Toronto, Ontario M5S 1A8, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Samuel%2BA.%2BLambert%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lambert%20SA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASamuel%2BA.%2BLambert%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-8222-008X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Samuel A. Lambert</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Mihai Albu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Terrence Donnelly Centre for Cellular and Biomolecular Research, University of Toronto, Ontario M5S 1A8, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mihai%2BAlbu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Albu%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMihai%2BAlbu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Timothy R. Hughes </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Terrence Donnelly Centre for Cellular and Biomolecular Research, University of Toronto, Ontario M5S 1A8, Canada</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Molecular Genetics, University of Toronto, Ontario M5S 1A8, Canada</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Canadian Institutes for Advanced Research, Toronto, Ontario M5G 1M1, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Timothy%2BR.%2BHughes%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hughes%20TR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ATimothy%2BR.%2BHughes%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-8721-4719" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Timothy R. Hughes</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:t.hughes@utoronto.ca" class="" data-icon-position="" data-hide-link-title="0">t.hughes@utoronto.ca</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=219&amp;rft.epage=229&amp;rft.atitle=Comparison%20of%20ChIP-Seq%20Data%20and%20a%20Reference%20Motif%20Set%20for%20Human%20KRAB%20C2H2%20Zinc%20Finger%20Proteins&amp;rft.volume=8&amp;rft.issue=1&amp;rft.date=2018-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Barazandeh%2C+Marjan&amp;rft.au=Lambert%2C+Samuel+A.&amp;rft.au=Albu%2C+Mihai&amp;rft.au=Hughes%2C+Timothy+R."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/487830/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/487830/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/487830/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/487830/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/487830/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/487830/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/487830/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/487830/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/487830/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/487830/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/487830/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/487830/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/8/1/219" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-7" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-12" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/8/1/219.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/8/1/219.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/8/1/219.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/8/1/219.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__c-64NG7CB_fwZbyex-B49GkX1AvVESfu7M9yxTNDG_o__Uvi1liJMEKbHtHHv_1fo_FCLvimvbq9jMwPcCPR364Q__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
